<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Software demonstrations</font></strong></p>
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Thursday<br/>
      September 29, 2005</strong></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D1" id="D1"></a></strong></font><span class="titular">9:00/10:00 - Room 9 Bis 1st Floor</font></span><br>
                <strong class="textomoradoGrandeB">                D1.1: High Performance Computing</strong></font> <span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.ibm.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.ibm.com'" target="_blank">IBM</a>) </span><br>
                 <span class="textomoradoGrande">IBM speaker</span><br>
                </font></font></p>
              <p align="justify"><span class="textomoradoGrandeB"><b>D1.2: Modeling and Simulations on Linux: The Accelrys Advantage</b></span> <span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.accelrys.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.accelrys.com/'" target="_blank">ACCELRYS</a>)</span><b><br>
              </b></font><span class="textomoradoGrande">Dr. Hugues Olivier Bertrand, EMEA Life Sciences Senior Manager</span></p>
              <p align="justify" class="textonegro">Accelrys has teamed up with IBM to develop and optimize its suite of software for drug discovery on the Linux platform. Leveraging the cost-performance benefits and scalability of the Linux platform and IBM hardware, scientists can focus on solving critical problems. The sequence and structure of proteins provides valuable information to guide the discovery process. Accelrys provides tools for protein engineering, homology modeling, molecular simulations and structure-based drug design that allow users to visualize and analyze the structure, function and properties that provide new insights and help design new drugs and suggest further experiments.</font></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D2" id="D2"></a></strong><span class="textomorado">10</font>:00/11:00</font> - Room 9 Bis 1st Floor</span><br>
                <span class="textomoradoGrandeB"></font><b>D2: A total scientific information management solution for collecting, managing, visualizing and analyzing DNA sequence data</b></font></span> <span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('https://www.appliedbiosystems.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='https://www.appliedbiosystems.com/'" target="_blank">APPLIED BIOSYSTEMS</a>)</span><br>
                <span class="textomoradoGrande">Carmelo Volpe Ph.D.<br>
                Global Solution Design Consultant</font></font></span></p>
              <div align="justify">
                <p class="textonegro">Learn how you can improve productivity, increase efficiency, and reduce costs using the industry-proven Geospiza Finch Suite.</font></p>
                <p><br>
                  </font></p>
              </div></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <b><strong><a name="D3" id="D3"></a></strong></b></font><span class="textomorado">11</font>:00/12:00</font> - Room 9 Bis 1st Floor</span><br>
                </font><span class="textomoradoGrandeB"><b>D3: Analysing genome-scale experiments through web servers</b></font> </span><span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.amd.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.amd.com/'" target="_blank">AMD</a>) </span><br>
                 <span class="textomoradoGrande">Joaqu&iacute;n Dopazo</font></font></span></p>
              <div align="justify">
                <p class="textonegro">Bioinformatics Platform for the representation and analysis of genetic diversity: SNP analysis, polymorphism, linkage disequilibrium, association studies, etc. Polymorphism studies are one of the primary areas in the post-genomic era for basic and applied research. Biomedical research is<br>
                  actively screening human variation to detect genetic changes affecting human traits. ebioSNP is the most complete software solution for the storage, representation and analysis of SNPs. It integrates a comprehensive set of analysis programs, management utilities and display tools in a transparent and intuitive interface, which greatly facilitates SNP research.</font></p>
                <p>&nbsp;</p>
              </div></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <b><strong><a name="D4" id="D4"></a></strong></b></font><span class="textomorado">12</font>:00/13:00</font> - Room 9 Bis 1st Floor</span><br>
                <span class="textomoradoGrandeB"><b>D4: ebioSNP, an essential tool in the study of the genetic diversity and its applications</b></font></span> <span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.ebiointel.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.ebiointel.com/'" target="_blank">EBIOINTEL</a>)</span><br>
                <span class="textomoradoGrande">Natalia Arroyo y Bruno Aranda</span></font></font></p>
              <div align="justify">
                <p class="textonegro">Bioinformatics Platform for the representation and analysis of genetic diversity: SNP analysis, polymorphism, linkage disequilibrium, association studies, etc. Polymorphism studies are one of the primary areas in the post-genomic era for basic and applied research. Biomedical research is<br>
                  actively screening human variation to detect genetic changes affecting human traits. ebioSNP is the most complete software solution for the storage, representation and analysis of SNPs. It integrates a comprehensive set of analysis programs, management utilities and display tools in a transparent and intuitive interface, which greatly facilitates SNP research.</font></p>
                <p>&nbsp;</p>
              </div></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <b><strong><a name="D5" id="D5"></a></strong></b></font><span class="textomorado">13</font>:00/14:00</font> - Room 9 Bis 1st Floor</span><br>
                <span class="textomoradoGrandeB"><b>D5: Science: Make your next discovery on a Mac</b></font></span> <span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.apple.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.apple.com/'" target="_blank">APPLE</a>) </span><br>
                <span class="textomoradoGrande">Iv&aacute;n Pe&ntilde;a - Technical Consulting</font></font></span></p>
              <p align="justify" class="textonegro">For 25 years, Apple has provided powerful tools to enable scientific discovery. Today, Apple continues to innovate. With the world's first 64-bit desktop computer, robust cluster, and storage solutions, and thousands of scientific applications, scientists everywhere are moving to Mac - and with good reason. An open source, UNIX-based operating system, industry-leading hardware, and unparalleled price/performance make the Mac the ideal platform for modern scientific research.</font></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <b><strong><a name="D6" id="D6"></a></strong></b></font><span class="textomorado">14</font>:00/15:00</font> - Room 9 Bis 1st Floor</span><br>
                <span class="textomoradoGrandeB"><b>D6: Text Mining for the Life Sciences</b></font></span> <span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.bioalma.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.bioalma.com/'" target="_blank">BIOALMA</a>) </span><br>
                <span class="textomoradoGrande">Christian Blaschke</font></font></span></p>
              <p align="justify" class="textonegro">AlmaKnowledgeServer (AKS) is an integrated text mining solution that makes access to knowledge easier for the biomedical researcher. It reduces significantly the time spent by researchers on reviewing the literature and at the same time it provides access to much more facts related to the users interest.<br>
                <br>
                This demo will show users how to: analyze large document sets, extract keywords and rank documents and sentences according to specific criteria, explore relationships between biological entities and visualize biological networks efficiently.<br>
                <br>
                Cases studies: <br>
                - Integration of AKS in CodeQuest, the TimeLogic's desktop bioinformatics workstation. <br>
                - GeneCards capabilities improved with AKS technology. <br>
                - Use of the system in the EMBL (Germany) for research.</font></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <b><strong><a name="D7" id="D7"></a></strong></b></font><span class="textomorado">16:00/17:00 - Room 9 Bis 1st Floor</span><br>
                <b class="textomoradoGrandeB">D7: Comparative genomics EnsEMBL</b></font> <span class="textonegroGrande">(EMBL-EBI) </span><br>
                <span class="textomoradoGrande">Javier Herrero </font></font></span></p>
              <div align="justify" class="textonegro"><a href="javascript:if(confirm('http://www.ensembl.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.ensembl.org'" target="_blank">Ensembl</font></a> aims to provide a useful access to metazoan genomes. It is an open source project which aims to cover all the steps in the genome annotation. Ensembl Compara is the comparative genomic part of Ensembl and provides analysis at both the nucleotide and peptide levels. </font> </div>
              <p align="justify" class="textonegro">This demo will give an overview of the EnsEMBL genome browser and will focus on the comparative genomics data stored in EnsEMBL and how to extract them using both the web views and the Mart datamining system.</font></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FFEACE"><p align="justify"> <b><strong><a name="D8" id="D8"></a></strong></b></font><span class="textomorado">17</font>:00/18:00 - Room 9 Bis 1st Floor</font></span><br>
                <span class="textomoradoGrandeB"><b>D8: Toward the Modeling and Prediction of the Preclinical Profile of Molecular Entities</b></font> </span><b><span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.prous.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.prous.com/'" target="_blank">PROUS INSTITUTE</a>)</span></b><br>
                <span class="textomoradoGrande">Ann Wescott</font></font></span></p>
              <div align="justify">
                <p class="textonegro">The BioEpisteme&reg; Project, a recent Prous Science endeavor, is a computational technology based on molecular descriptors and binding profiles, gene expression, and experimental analysis designed to facilitate discovering new medicines and new indications for existing drugs. It is oriented to predicting pharmacological properties of molecules on the basis of their chemical structure. The talk will discuss how, using factual data from a variety of sources, the system evaluates the probability that a compound will display a certain combination of molecular events.</font></p>
                <p>&nbsp;</p>
              </div></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <br>
        <br>
      </div>
      <br>
      <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
        <tr bgcolor="#D2F4FF">
          <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Friday<br/>
      September 30, 2005</strong></p></td>
        </tr>
        <tr valign="top">
          <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D9" id="D9"></a></strong><span class="textomorado">9</font>:00/10:00 - Room 9 Bis 1st Floor</span><br>
              <span class="textomoradoGrandeB"><b>D9: EBIMed, Whatizit and Paella: Text Mining services at the EBI </b></font></span><b class="textonegroGrande">(European Bioinformatics Institute)</b><br>
              <span class="textomoradoGrande">Miguel Arregui<br>
              Dietrich Rebholz-Schuhmann </font></font></span></p>
            <div align="justify">
              <p class="textonegro">EBI now offers text mining services to the public, 
                which process Medline abstracts as well as full papers in PDF or HTML format.<br>
                EBIMed is EBI's frontend to Medline, accepts keywords to retrieve a list of abstracts and then processes them to identify associations between UniProt proteins and other concepts.<br>
                All concepts are linked to EBI's and NCBI's databases for inspection.<br>
                The text analysis profits from a set of modules on EBI's compute servers, which are as well used by Whatizit and Paella to process ASCII, PDF or HTML documents and to offer processing via http request. </font></p>
              <p>&nbsp;</p>
            </div></td>
        </tr>
        <tr valign="top">
          <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D10" id="D10"></a></strong><span class="textomorado">10:00/11:00</font> - Room 9 Bis 1st Floor</span><br>
              <span class="textomoradoGrandeB"><b>D10: New developments in Swiss-Prot</b></font> </span><b class="textonegroGrande">(Swiss Institute of Bioinformatics)</b><br>
              <span class="textomoradoGrande">Brigitte Boeckmann<br>
              Isabelle Phan </font></font></span></p>
            <div align="justify">
              <p class="textonegro">The first part of the demo will present Swiss-Prot annotation in its biological context. Particular emphasis will be laid on new information concerning protein variety such as the formation of alternative sequences, structures, chemical modifications and protein complexes.<br>
                The second part will consist of a quick tour of latest developments in the fields of semi-automated annotation procedures, standardized vocabularies, and documentation. Of special interest for data mining and cross-database analysis, practical issues on where and how to access and interpret the data will be presented.</font></p>
              <p>&nbsp;</p>
            </div></td>
        </tr>
        <tr valign="top">
          <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D11" id="D11"></a></strong><span class="textomorado">11</font>:00/12:00 - Room 9 Bis 1st Floor</span><br>
              <b class="textomoradoGrandeB">D11: Molecular Dynamics Extended Library </b></font><b class="textonegroGrande">(Universitat de Barcelona-Parc Cient&iacute;fic)</b><br>
              <span class="textomoradoGrande">Josep LLu&iacute;s Gelp&iacute;<br>
              Manuel Rueda </font></font></span></p>
            <div align="justify" class="textonegro">MoDeL is a public database of molecular dynamics of proteins implemented on a web server.<br>
              In the demo we will show how to navigate around the different web pages and explain how to extract information for a given set of proteins simulations. We will also remark what are the current aplications and future perpectives of work. </font> </div>
            <p align="justify" class="textonegro">The database is accesible via www at the URL <a href="javascript:if(confirm('http://mmb.pcb.ub.es  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://mmb.pcb.ub.es'"%20" target="_blank">http://mmb.pcb.ub.es </a></font></p>
            <p align="justify">&nbsp; </p></td>
        </tr>
        <tr valign="top">
          <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D12" id="D12"></a></strong><span class="textomorado">12</font>:00/13:00 - Room 9 Bis 1st Floor</span><br>
              </font><span class="textomoradoGrandeB"><b>D12: M-GCAT: Multiple Genome Comparison and Alignment Tool </b></font></span><b class="textonegroGrande">(Universitat Polit&egrave;cnica de Catalunya)</b><br>
              <span class="textomoradoGrande">Todd Treangen<br>
              Xavier Messeguer </font></font></span></p>
            <div align="justify" class="textonegro">M-GCAT is genome comparison software that can quickly analyze and align conserved features among multiple organisms. M-GCAT merges several techniques into an intelligent process that is capable of aligning multiple large genomic sequences while identifying rearrangements. </font> </div>
            <p align="justify" class="textonegro">M-GCAT incorporates a robust and interactive graphical user interface which allows the user to efficiently analyze and compare various regions of the genomes. This visualization tool is full-featured and easy-to-use, allowing for interactive multiple genome comparison that aides in detecting biological patterns such as evolution and mutations among the genomes.</font></p>
            <p align="justify" class="textonegro">M-GCAT relies on the popular multiple sequence alignment tools CLUSTALW and MUSCLE for aligning the genomes, so they must be successfully installed before M-GCAT can perform alignments. Once installed, M-GCAT can multi-align and calculate alignment scores for any combination of regions in the provided genomes. Additionally, the user interface allows for BLAST-ing of significant regions among the genomes.</font></p>
            <p align="justify" class="textonegro">M-GCAT can be downloaded for non-commercial use at: <a href="javascript:if(confirm('http://alggen.lsi.upc.es  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://alggen.lsi.upc.es'" target="_blank"><br>
            http://alggen.lsi.upc.es</a> (under the Research link).</font></p>
            <p align="justify"><br>
            </p></td>
        </tr>
        <tr valign="top">
          <td 65535 bgcolor="#FFEACE"><p align="justify"> <strong><a name="D13" id="D13"></a></strong><span class="textomorado">13</font>:00/14:00 - Room 9 Bis 1st Floor</span><br>
              <b>              </font></b><b class="textomoradoGrandeB">D13: Information Based Medicine: The Impact of Science and IT on Healthcare </b><span class="textonegroGrande">(Sponsored by: <a href="javascript:if(confirm('http://www.ibm.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.ibm.com'" target="_blank">IBM</a>) </span><br>
              <span class="textomoradoGrande">Dr. Michael Hehenberger, Global Solutions Executive, IBM Healthcare &amp; Life Sciences.</font></font></span></p>
            <p align="justify" class="textonegro">The convergence of information technology (IT) with molecular biology, drug discovery and the practice of medicine have led to emergence of a new discipline that IBM has named Information Based Medicine. <br>
              </font>IBM&rsquo;s Healthcare &amp; Life Sciences Clinical Genomics initiative is targeting the IT challenges created by the integration of clinical (phenotypic) and genotypic patient data. We are building patient data warehouses, and we are developing query and analysis tools for medical research, bio-pharmaceutical clinical development, and ultimately the care of patients.<br>
              Medical research centers will break new ground by using diagnostic and clinical care decision support tools on top of those patient data warehouses. Advanced diagnostic tests based on genomics and biomedical imaging will enable targeted, &ldquo;personalized&rdquo; treatments. Preventive care will reduce hospitalization and invasive procedures, thereby improving quality of life and cutting healthcare expenses.<br>
              The Mayo Clinic and Karolinska Institute projects are briefly described as examples of leading medical research centers that are embracing Clinical Genomics. <br>
              Emerging Biobank projects are clearly pointing the way towards the future of healthcare, and Biomarker based Pharmaceutical R&amp;D will create the medicines needed to deliver the promise of Information Based Medicine. <br>
              In addition to the use of IT to manage Data and to provide decision support, there is an increasing opportunity to support advanced life sciences projects with high performance computing (HPC) resources. We will show how recent advances in HPC are enabling new breakthroughs in the Life Sciences.</font></p>
            <p align="justify">&nbsp;</p></td>
        </tr>
        <tbody>
        </tbody>
      </table>
      <p>&nbsp;</p>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
