<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Thursday<br/>
      September 29, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><span class="textomoradoGrandeB">AREA 2: DATABASES</span> <span class="textomorado"><br>
            </span><span class="textomoradoGrande">Area chairs: Rolf Apweiler and Jos&eacute; Mar&iacute;a Carazo </span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p align="justify" class="textomoradoGrandeB">OP-5 Predicting Post-Synaptic Activity in Proteins with Data Mining<br>
                  <span class="textomoradoGrande">Alex A. Freitas (1), Anthony Baines (1), Gisele Pappa (1)<br>
  1) University of Kent</span></p>
              <p class="textonegro">ABSTRACT:<br>
  The bioinformatics problem being addressed in this paper is to predict whether or not a protein has post-synaptic activity. This problem is of great intrinsic interest because proteins with postsynaptic activities are connected with functioning of the nervous system. Indeed, many proteins having post-synaptic activity have been functionally characterized by biochemical, immunological and proteomic exercises. They represent a wide variety of proteins with functions in extracellular signal reception and propagation through intracellular apparatuses, cell adhesion molecules and scaffolding proteins that link them in a web. The challenge is to automatically discover features of proteins' primary sequences that typically occur in proteins with post-synaptic activity but rarely (or never) occur in proteins without post-synaptic activity, and vice-versa. In this context, we used data mining to automatically discover classification rules that predict whether or not a protein has post-synaptic activity. The discovered rules were analyzed with respect to their predictive accuracy (generalization ability) and with respect to their interestingness to biologists (in the sense of representing novel, unexpected knowledge).<br>
  CONTACT: glp6@kent.ac.uk</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-6 DPDB: a database for the storage, representation and analysis of polymorphism in the Drosophila genus</span><br>
                  <span class="textomoradoGrande">S&ograve;nia Casillas (1), Natalia Petit (1), Antonio Barbadilla (1)<br>
  1) Departament de Gen&egrave;tica i de Microbiologia, Universitat Aut&ograve;noma de Barcelona, 08193 Bellaterra (Barcelona).</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: Polymorphism studies are one of the main research areas of this genomic era. To date, however, no comprehensive secondary databases have been designed to provide searchable collections of polymorphic sequences with their associated diversity measures.<br>
  Results: We define a data model for the storage, representation and analysis of genotypic and haplotypic data. Under this model we have created DPDB, Drosophila Polymorphism Database, a web site that provides a daily updated repository of all wellannotated polymorphic sequences in the Drosophila genus. It allows the search for any polymorphic set according to different parameter values of nucleotide diversity, linkage disequilibrium and codon bias. For data collection, analysis and updating we use PDA, a pipeline that automates the process of sequence retrieving, grouping, aligning and estimation of nucleotide diversity from Genbank sequences in different functional regions. The web site also includes analysis tools for sequence comparison and the estimation of genetic diversity, a page with real-time statistics of the database contents, a help section and a collection of selected links. DPDB is freely available at http://dpdb.uab.es and can be downloaded via FTP.<br>
  CONTACT: sonia.casillas@uab.es</p>
              <p class="textomoradoGrandeB">OP-7 A Query Language for Biological Networks Ulf Leser, Humboldt-Universit&auml;t zu Berlin</p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: Many areas of modern biology are concerned with the management, storage, visualization, comparison, and analysis of networks, but yet no appropriate query language for such complex data structures exists. Results: We have designed and implemented the pathway query language (PQL) for querying large protein interaction or pathway databases. PQL is based on a simple graph data model with extensions reflecting properties of biological objects. Queries match subgraphs in the database based on node properties and paths between nodes. The syntax is easy to learn for anybody familiar with SQL. As an important feature, a query may require a certain structure in the database to exist, but return a different subgraph. We have tested PQL queries on networks of up to 16.000 nodes and found it to scale very well.<br>
  CONTACT: leser@informatik.hu-berlin.de</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-8 SIMAP - The similarity matrix of proteins</span><br>
                  <span class="textomoradoGrande">Roland Arnold (1), Thomas Rattei (2), Patrick Tischler (1), Volker St&uuml;mpflen (1), Werner Mewes (1)<br>
  1) Institute for Bioinformatics, GSF-National Research Center for Environment and Health, Ingolst&auml;dter Landstr. 1, 85764 Neuherberg, 2) Department of Genome Oriented Bioinformatics, Technical University of Munich, Wissenschaftszentrum Weihenstephan, 85350 Freising</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: Sequence similarity searches are of great importance in bioinformatics. Exhaustive searches for homologous proteins in databases are computationally expensive and may be replaced by a database of pre-calculated homologies in many cases. Retrieving similarities from an incrementally updated database instead of repeatedly recalculating them should provide homologs much faster and frees computational resources for other purposes.<br>
  Results: We have implemented SIMAP - a database containing the similarity space formed by about all amino-acid sequences from public databases and completely sequenced genomes. The database is capable of handling very large datasets and allows incremental updates. We implemented a powerful backbone for similarity computation, which is based on FASTA heuristics. By providing WWW interfaces as well as web-services, we make our data accessible to the world-wide community. We also adapted procedures to detect putative orthologs as example applications. Availibility.The SIMAP portal page providing links to SIMAP services is publicly available: http://mips.gsf.de/services/analyses/simap/ The web-services can be accessed under: http://mips.gsf.de/proj/hobitws/services/RPCSimapService?wsdl and http://mips.gsf.de/proj/hobitws/services/DocSimapService?wsdl<br>
  CONTACT: t.rattei@wzw.tum.de</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-9 Design of a Description Language for Generating Wrapper to Collect Biological Data</span><br>
                  <span class="textomoradoGrande">Myungeun Lim (1), Myungguen Chung (1), Myungnam Bae (1), Sunhee Park (1).<br>
  1) Electronics and Telecommunications Research Institute</span></p>
              <p class="textonegro">ABSTRACT:<br>
  As biological data are scattered in various area with various format and they are changing continuously, data integration becomes an important issue to provide researcher a dynamic access of the data. In the data integration process, the method to extract heterogeneous data dynamically from the data source is an essential part. Data extraction method using wrapper can provide flexibility and extensibility to the integration system.<br>
  CONTACT: melim@etri.re.kr</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-10 Adding Some SPICE to DAS </span><br>
                  <span class="textomoradoGrande">Andreas Prlic (1), Thomas Down (1), Tim Hubbard (1)<br>
  1) The Wellcome Trust Sanger Institute.</span></p>
              <p class="textonegro">ABSTRACT:<br>
  The Distributed Annotation System (DAS) defines a communication protocol used to exchange biological annotations. It is motivated by the idea that annotations should not be provided by single centralized databases, but instead be spread over multiple sites. Data distribution, performed by DAS servers, is separated from visualization, which is done by DAS clients. The original DAS protocol was designed to serve annotation of genomic sequences. We have extended the protocol to be applicable to macromolecular structures. Here we present SPICE, a new DAS client that can be used to visualize protein sequence and structure annotations.<br>
  Availability: http://www.efamily.org.uk/software/dasclients/spice/<br>
  CONTACT: ap3@sanger.ac.u</p>
              <p class="textonegro">&nbsp;</p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
