<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Friday<br/>
      September 30, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><strong><span class="textomoradoGrandeB">AREA 5: MICROARRAYS</span> <br>
            </strong><span class="textonegroGrande">Sponsorized by: <a href="javascript:if(confirm('http://www.bio-rad.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.bio-rad.com'">Bio-rad</a></span><strong> <span class="textomorado"><br>
            </span></strong><span class="textomoradoGrande">Area chairs: Alvis Brazma and Ximo Dopazo </span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p class="textonegro"><span class="textomoradoGrandeB">OP-19 Data-adaptive test statistics for microarray</span><br>
                <span class="textomoradoGrande">Sach Mukherjee (1), Stephen Roberts (1), Mark<br>
1) University of California, Berkeley</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: A vitally important task in microarray is the selection of genes which are differentially different kinds of tissue samples, such as healthy However, microarray data contain an enormous dimensions (genes), and very few samples (arrays), which poses fundamental statistical problems process which have defied easy resolution.<br>
  Results: In this paper, we present a novel selection in which test statistics are learned simple notion of reproducibility in selection learning criterion. Reproducibility, as we computed without any knowledge of the `ground- advantage of certain properties of microarray asymptotically valid guide to expected loss under datagenerating distribution. We are therefore minimize expected loss, and produce results robust than conventional methods. We apply simulated and oligonucleotide array data.<br>
  CONTACT: sach@robots.ox.ac.uk</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-20 Identifying active transcription factors and kinases from expression data using Pathway Queries</span><br>
                  <span class="textomoradoGrande">Florian Sohler (1), Ralf Zimmer (1)<br>
  1) Department of Informatics, Ludwig-Maximilians-Universit&auml;t M&uuml;nchen</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Although progress has been made identifying regulatory relationships from expression data in general, only few methods have focused on detecting biological mechanisms like active pathways using a single measurement. This is of particular importance when only few measurements are available, e.g. if special cell types or conditions are under investigation. Here we present a method to test user specified hypotheses (pathway queries) on expression data where prior knowledge is given in the form of networks. Based on that method, we develop a scoring function to identify active transcription factors or kinases, thus making a first step towards explaining the measured expression data.<br>
  We apply the algorithm to the Rosetta Yeast Compendium dataset, finding that in many cases the results are in concordance with biological knowledge. We were able to confirm that transcription factors and to a lesser degree kinases identified by our method play an important role in the biological processes affected by the respective knock-outs. Furthermore, we show that correlation of inferred activities can provide evidence for a physical interaction or cooperation of transcription factors where correlation of plain expression data fails to do so.<br>
  CONTACT: florian.sohler@ifi.lmu.de</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-21 Analyzing Microarray Data Using Quantitative Association Rules</span><br>
                  <span class="textomoradoGrande">Elisabeth Georgii (1), Lothar Richter (1), Ulrich R&uuml;ckert (1), Stefan Kramer (1)<br>
  1) TU M&uuml;nchen</span></p>
              <p class="textonegro"><br>
  ABSTRACT:<br>
  We tackle the problem of finding regularities in microarray data. A wide variety of data mining tools such as clustering, classification, Bayesian networks and association rules have been applied so far to gain insight into gene expression data. Association rule mining techniques used so far work on discretizations of the data and cannot account for cumulative effects. In this paper, we investigate the use of quantitative association rules that can operate directly on numeric data and represent cumulative effects of variables. Technically speaking, this type of quantitative association rules based on half-spaces can find non-axis-parallel regularities.<br>
  Results: We performed a variety of experiments testing the utility of quantitative association rules for microarray data. First of all, the results should be statistically significant and robust against fluctuations in the data. Next, the approach should be scalable in the number of variables, which is important for such high-dimensional data. Finally, the rules should make sense biologically and be sufficiently different from rules found in regular association rule mining working with discretizations. In all of these dimensions, the proposed approach performed satisfactorily. Therefore, quantitative association rules based on half spaces should be considered as a tool for the analysis of microarray gene expression data.<br>
  CONTACT: e.georgii@gmx.de</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-22 A fully Bayesian model to cluster gene expression profiles</span><br>
                  <span class="textomoradoGrande">Claus Vogl (1), Fatima Sanchez-Cabo (2), Gernot Stocker (3), Simon Hubbard (4), Olaf Wolkenhauer (5), Zlatko Trajanoski (3)<br>
  1) Institute for animal breeding and genomics, Veterin&auml;rmedizinische universit&auml;t Wien, 2) Institute for Genomics and Bioinformatics, Graz University of Technology, Graz, 3) Institute for Genomics and Bioinformatics, Graz University of Technology, Graz, 4) Faculty of Life sciences, University of Manchester, Manchester, 5) Institute of informatics, University of Rostock</span></p>
              <p class="textonegro">ABSTRACT: With cDNA or oligonucleotide chips, gene expression levels of essentially all genes in a genome can be simultaneously monitored over a timecourse or under different experimental conditions. After proper normalization of the data, genes are often classified into co-expressed classes (clusters) to identify subgroups of genes that share common regulatory elements, a common function, or a common cellular origin, or to impute missing values. With most ad-hoc methods, e.g. k-means, the number of clusters needs to be specified in advance; results depend strongly on this choice. Even with likelihood based methods, estimation of this number is difficult. Furthermore, missing values often cause problems and lead to the loss of data.<br>
  Results: We propose a fully probabilistic Bayesian model to cluster gene expression profiles. The number of classes does not need to be specified in advance; instead it is adjusted dynamically using a Reversible Jump Markov Chain Monte Carlo (RJMCMC) sampler. Imputation of missing values is integrated into the model. With simulations, we determined the speed of convergence of the sampler as well as the accuracy of the inferred variables. Results were compared to the widely used k-means algorithm. With our method, biologically related co-expressed genes could be identified in a yeast transcriptome data set, even when some values were missing.<br>
  CONTACT: claus.vogl@vu-wien.ac.at</p>
              <p class="textomoradoGrandeB">OP-23 Fusing microarray experiments with multivariate regression</p>
              <p class="textomoradoGrande">Walter Gilks (1), Brian Tom (1), Alvis Brazma (2),<br>
  1) Medical Research Council, 2)EMBL-EBI</p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: It is widely acknowledged that microarray data are subject to high noise levels and results are often platform dependent. Therefore, microarray experiments should be replicated several times and in several laboratories before results can be relied upon. To make best use of such extensive datasets, methods for microarray data fusion are required. Ideally, the fused data should distil important aspects of the data whilst suppressing unwanted sources of variation, and be amenable to further informal and formal methods of analysis. Also, account should be taken of variability in the quality of experimentation.<br>
  Results: We present such an approach to data fusion, based on multivariate regression. We apply our methodology to data from Rustici et al(2004) on cell-cycle control in S.pombe. Availability: The algorithm implemented in R is freely available from the authors on request.<br>
  CONTACT: wally.gilks@mrc-bsu.cam.ac.uk</p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
