<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Friday<br/>
      September 30, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><span class="textomoradoGrandeB">AREA 7: SNPs</span> <span class="textomoradoGrande"><br>
Area chairs: Guillaume Laval and Arcadi Navarro </span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p><span class="textomoradoGrandeB">OP-28 Recovering haplotype structure through recombination and gene conversion</span><br>
                  <span class="textomoradoGrande">Mathieu Lajoie (1), Nadia El-Mabrouk (1)<br>
  1) Universit&eacute; de Montr&eacute;al</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: Understanding haplotype evolution subject to mutation, recombination and gene conversion is fundamental to understand genetic specificities of human populations and hereditary bases of complex disorders. The goal of this project is to develop new algorithmic tools assisting the reconstruction of historical relationships between haplotypes, and the inference of haplotypes from genotypes.<br>
  Results: We present two new algorithms. The first one finds an optimal pathway of mutations, recombinations and gene conversions leading to a given haplotype of size m from a population of h haplotypes. It runs in time (mhs2), where s is the maximum number of contiguous sites that can be exchanged in a<br>
  single gene conversion. The second one finds an optimal pathway of mutations and recombinations leading to a given genotype, and runs in time (mh2). Both algorithms are based on a penalty score model and use a dynamic programming<br>
  approach. We apply the second algorithm to the problem of inferring haplotypes from genotypes, and show how it can be used as an independent tool, or to improve the performance of existing methods.<br>
  Availability: The algorithms have been implemented in JAVA,<br>
  and are available on request.<br>
  CONTACT: Mathieu.lajoie@umontreal.ca</p>
              <p class="textonegro"><br>
                  <span class="textomoradoGrandeB">OP - 29 Model-P: A basecalling method for resequencing microarrays of diploid samples</span><br>
                  <span class="textomoradoGrande">David Kulp (1), Yiping Zhan (1)<br>
  1) University of Massachusetts</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: Basecalling is a critical step of the analysis of DNA resequencing microarray data for SNP discovery and genotyping. For microarrays hybridized with DNA derived from diploid organisms, basecalling with high accuracy at high call rates is a challenging task. Current methods sometimes do not<br>
  produce satisfactory results. <br>
  Results: We explored using physical models based on the sequences of the probe and the target to predict feature intensities in resequencing microarrays. Based on these intensity-predicting models, a new basecalling method (Model-P), which takes into consideration the expected feature intensities for different potential genotypes, was developed. Model-P is shown to have better performance at high call rates compared with ABACUS, the current state-of-the-art method, on a test dataset and on relatively AT-rich regions.<br>
  Availability: Model-P is available upon request.<br>
  CONTACT: dkulp@cs.umass.edu</p>
              <p><span class="textomoradoGrandeB">OP-30 Association Cluster Detector: A tool for heuristic detection of significance clusters in whole-genome scans</span><br>
                  <span class="textomoradoGrande">Tom&aacute;s Marqu&egrave;s-Bonet (1), Oscar Lao (1), Robert Goertsches (2), Manuel Comabella (2), Xavier Montalban (2), Arcadi Navarro (1)<br>
              1) Departament de Ci&egrave;ncies Experimentals i de la Salut, Universitat Pompeu Fabra, 2) Unitat de Neuroimmunologia Cl&iacute;nica, Hospital Universitari Vall d'Hebron</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Whole genome scans analyze large sets of genetic markers, mainly Single Nucleotide Polymorphisms (SNPs), all over the genome in order to find variants and regions associated to complex traits so these can be further investigated. Analyzing the results of such scans becomes difficult due to multiple testing problems and to the genomic distributions of recombination, linkage disequilibrium and true associations, which generate an extremely complex network of dependences between markers. Here we present Association Cluster Detector (ACD), a simple tool aiming to ease the analysis of the results of Whole Genome Scans. ACD facilitates correction for multiple tests by several standard procedures and implements a sliding-window, heuristic method that helps detecting potentially interesting candidate regions by exploiting the property of non random distribution of significantly associated markers.<br>
  Availability: The tool can be downloaded from http://www.upf.es/cexs/recerca/bioevo/index.htm<br>
  CONTACT: arcadi.navarro@upf.edu</p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
