<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Saturday<br/>
      October 1, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><span class="textomoradoGrandeB">AREA 8: SYSTEMS BIOLOGY</span> <span class="textomoradoGrande"><br>
Area chairs: Martin Kuiper and Luis Serrano</span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p class="textonegro"><span class="textomoradoGrandeB">OP-31 Qualitative Modeling of Regulated Metabolic Pathways: Application to the Tryptophan Biosynthesis in E.Coli</span><br>
                  <span class="textomoradoGrande">Claudine Chaouiya (1), Eugenio Simao (1), Elisabeth Remy (2), Denis Thieffry (1)<br>
  1) IBDM,LGPD, 2) IML, CNRS, remy@iml.univ-mrs.fr</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: The integrated dynamical modelling of mixed metabolic/genetic networks constitutes one of the challenges of systems biology. Furthermore, as most of available data about genetic and metabolic regulations are qualitative, there is a pressing need for rigorous qualitative mathematical approaches.<br>
  Results: On the basis of two established formalisms, the logical modelling of genetic regulatory networks and the Petri net modelling of metabolic networks, we propose a systematic approach for the modelling of regulated metabolic networks. This approach leans on previous work defining a systematic procedure to translate logical regulatory graphs into standard (discrete) Petri nets. This approach is illustrated by the qualitative modelling of the biosynthesis of tryptophan (Trp) in E. coli, taking into account two types of regulatory feedbacks: the direct inhibition of the first enzyme of the pathway by the final product of the pathway, and the transcriptional inhibition of the Trp operon by the Trprepressor complex. On the basis of this integrated PN model, we further indicate how available dynamical analysis tools can be applied to obtain significant insights in the behaviour of the system. Availability: The software GINsim for the logical modelling of Genetic regulatory networks can be downloaded from the url: htpp://www.esil.univ-mrs.fr/~chaouiya/GINsim . Our Petri net model of the regulated Trp biosynthesis pathway is available at the url: htpp://www.esil.univ-mrs.fr/~chaouiya/BioPN/trpregbiosyn.html<br>
  CONTACT: chaouiya@ibdm.univ-mrs.fr</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-32 Genome-Wide Decoding of Hierarchical Modular Structure of Transcriptional Regulation by Cis- Element and Expression Clustering Dmitriy</span><br>
                  <span class="textomoradoGrande">Leyfer (1), Zhiping Weng (2)<br>
  1) Gene Network Sciences, 2) Boston University</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: A holistic approach to the study of cellular processes is identifying both gene expression changes and regulatory elements promoting such changes. Cellular regulatory processes can be viewed as transcriptional modules (TM), groups of coexpressed genes regulated by groups of transcription factors (TF). We devised a method that identifies TMs while avoiding arbitrary thresholds on TM sizes and number.<br>
  Method: Assuming that gene expression is determined by TFs that bind to the gene's promoter, clustering of genes based on TF binding sites (cis-elements) creates gene groups similar to those obtained by expression clustering. Intersections between expression and cis-elementbased gene clusters reveal TMs. Statistical significance assigned to each TM allows identification of regulatory units of any size.<br>
  Results: Our method correctly identifies the number and sizes of TMs on simulated datasets. We demonstrate that yeast experimental TMs are biologically relevant by comparing them with MIPS and GO categories. Our modules are in statistically significant agreement with TMs from other research groups. This work suggests that there is no preferential division of biological processes into regulatory units; each degree of partitioning exhibits a slice of biological network revealing hierarchical modular organization of transcriptional regulation.<br>
  CONTACT: dleyfer@gnsbiotech.com</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-33 Estimating gene regulatory networks and proteinprotein interactions of Saccharomyces cerevisiae from multiple genome-wide data</span><br>
                  <span class="textomoradoGrande">Naoki Nariai (1), Yoshinori Tamada (2), Seiya Imoto (1), Satoru Miyano (1)<br>
  1) Human Genome Center, Institute of Medical Science, University of Tokyo, 2) Bioinformatics Center, Institute for Chemical Research, Kyoto University</span></p>
              <p class="textonegro">ABSTRACT: <br>
  Motivation: Biological processes in cells are properly performed by gene regulations, signal transductions and interactions between proteins. To understand such molecular networks, we propose a statistical method to estimate gene regulatory networks and protein-protein interaction networks simultaneously from DNA microarray data, protein-protein interaction data and other genome-wide data.<br>
  Results: We unify Bayesian networks and Markov networks for estimating gene regulatory networks and protein-protein interaction networks according to the reliability of each biological information source. Through the simultaneous construction of gene regulatory networks and protein-protein interaction networks of Saccharomyces cerevisiae cell cycle, we predict the role of several genes whose functions are currently unknown. By using our probabilistic model, we can detect false positives of high through-put data, such as yeast two hybrid data. In a genome-wide experiment, we find possible gene regulatory relationships and protein-protein interactions between large protein complexes that underlie complex regulatory mechanisms of biological processes.<br>
  CONTACT: nariai@ims.u-tokyo.ac.jp</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-34 Knowledge-Based Framework for Hypothesis Formation in Biochemical Networks</span><br>
                  <span class="textomoradoGrande">Nam Tran (1), Chitta Baral (1), Vinay Nagaraj (1),Lokesh Joshi (1)<br>
  1) Arizona State University</span></p>
              <p class="textonegro">ABSTRACT:<br>
  Motivation: The current knowledge about biochemical networks is largely incomplete. Thus biologists constantly need to revise or extend existing knowledge. The revision and extension are first formulated as theoretical hypotheses, then verified experimentally. Recently, biological data have been produced in great volumes and diverse formats. It is a major challenge for biologists to process these data to reason about hypotheses. Many computer-aided systems have been developed to assist biologists in undertaking this challenge. Most of the systems help in finding &quot;pattern&quot; in data and leave the reasoning to biologists. Few systems have tried to automate the reasoning process of hypothesis formation. These systems generate hypotheses from a knowledge base and given observations. A main drawback of these knowledge-based systems is their knowledge representation formalisms. These formalisms are mostly monotonic and now known to be not quite suitable for knowledge representation, especially in dealing with the inherently incomplete knowledge such as the knowledge about biochemical networks.<br>
  Results: We present a knowledge based framework for hypothesis formation in biochemical networks. The framework has been implemented by extending BioSigNet-RR - a knowledge based system that supports elaboration tolerant representation and non-monotonic reasoning. Features of the extended system are illustrated by a case study of p53 signal network.<br>
  Availability: http://www.biosignet.org<br>
  CONTACT: namtran@asu.edu</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-35 Horizontal gene transfer depends on gene content of the host</span><br>
                  <span class="textomoradoGrande">Martin Lercher (1), Csaba Pal (1), Balazs Papp (2) <br>
  1) European Molecular Biology Laboratory, 69012 Heidelberg, Germany, and MTA, E&ouml;tv&ouml;s Lor&aacute;nd University, Budapest H- 1117, Hungary, 2) School of Biological Sciences, University of Manchester, Manchester M13 9PT</span></p>
              <p class="textonegro"> ABSTRACT:<br>
  Horizontal gene transfer is a major contributor to the evolution of bacterial genomes. We examine this process through a combination of comparative genomics and in silico analysis of the Escherichia coli metabolic network.<br>
  We validate our horizontal transfer estimates by confirming the predicted gradual amelioration of GC content over time. We find that the chance of acquiring a gene by horizontal transfer is up to six times higher if an enzyme that catalyses a coupled metabolite flux is already encoded in the host genome.<br>
  CONTACT: m.j.lercher@bath.ac.uk</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-36 Decomposing protein networks into domain-domain interactions</span><br>
                  <span class="textomoradoGrande">Mario Albrecht (1), Carola Huthmacher (1), Silvio C.E. Tosatto (2), Thomas Lengauer (1)<br>
  1) Max-Planck-Institute for Informatics, 2) CRIBI Biotechnology Center, University of Padova</span></p>
              <p class="textonegro">ABSTRACT:<br>
  The application of novel experimental techniques has generated large networks of protein-protein interactions. Important information on the structure and cellular function of proteinprotein interactions can often be gained from the domains of interacting proteins. We designed a Cytoscape plugin that decomposes interacting proteins into their respective domains and computes a putative network of corresponding domaindomain interactions. To this end, the network graph of proteins has been extended by additional node and edge types for domain interactions, including different node and edge shapes and coloring schemes used for visualization. An additional plugin provides supplementary web links to Internet resources on domain function and structure.<br>
  Availability: Both Cytoscape plugins can be downloaded fromhttp://www.cytoscape.org<br>
  CONTACT: mario.albrecht@mpi-sb.mpg.de</p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
