<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Thursday<br/>
      September 29, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><strong><span class="textomoradoGrandeB">AREA 4: GENES AND GENOMES</span> <br>
            </strong><span class="textonegroGrande">Sponsorized by: <a href="javascript:if(confirm('http://www.sgi.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sgi.com/'">Silicon Graphics</a></span><strong> <span class="textomoradoGrande"><br>
            </span></strong><span class="textomoradoGrande">Area chairs: Mar Alba and Pierre Rouz&eacute;</span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p class="textonegro"><span class="textomoradoGrandeB">OP-15 Building Fragment Assembly String Graphs</span><br>
                <span class="textomoradoGrande">Gene Myers, UC Berkeley</span></p>
              <p class="textonegro">ABSTRACT:<br>
  We present a concept and formalism, the string graph, that represents all that is inferable about a DNA sequence from a collection of shotgun sequencing reads collected from it. We give time and space efficient algorithms for constructing a string graph given the collection of overlaps between the reads and in particular, present a novel linear expected time algorithm for transitive reduction in this context. The result demonstrates that the decomposition of reads into k-mers employed in the de Bruijn graph approach of Pevzner et al is not essential and in fact creates both efficiency problems and unecessary conceptual complexities. The current paper is the first in a series and presents the basic algorithm and preliminary results that demonstrate the efficiency and scalability of the method. The result is a step toward a next-generation whole genome shotgun assembler that will easily scale to mammalian genomes.<br>
  CONTACT: gene@eecs.berkeley.edu</p>
              <p class="textonegro">                  <span class="textomoradoGrandeB">OP-16 Finding Exact Optimal Motif in Matrix Representation by Partitioning</span><br>
                <span class="textomoradoGrande">Henry Leung (1), Francis Chin (1)<br>
  1) The University of Hong Kong</span></p>
              <p class="textonegro">ABSTRACT: Motivation: Finding common pattern, motif, in the promoter regions of coexpressed genes is an important problem in bioinformatics. A common representation of the motif is by probability matrix or PSSM (Position Specific Scoring Matrix). However, even for a motif of length 6 or 7, there is no algorithm that can guarantee finding the exact optimal matrix from an infinite number of possible matrices.<br>
  Results: This paper introduces the first algorithm called EOMM for finding the exact optimal matrix-represented motif, or simply optimal motif. Based on branch-and-bound search by partitioning the solution space recursively, EOMM can find the optimal motif of size of up to 8 or 9, and a motif of larger size with any desired accuracy on the principle that the smaller the error bound, the longer the running time. Experiments show that for some real and simulated data sets, EOMM finds the motif with very weak signals when the existing software, such as MEME, MITRA-PSSM, fails to do so.<br>
  CONTACT: cmleung2@cs.hku.hk</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-17 Prediction of regulatory modules comprising microRNAs and target genes</span><br>
                  <span class="textomoradoGrande">Sungroh Yoon (1), Giovanni De Micheli (1)<br>
  1) Stanford University, nanni@stanford.edu</span></p>
              <p class="textonegro">ABSTRACT: Motivation: MicroRNAs are small endogenous RNAs that can play important regulatory roles via the RNA-interference pathway by targeting mRNAs for cleavage or translational repression. We propose a computational method to predict miRNA regulatory modules or groups of miRNAs and target genes that are believed to participate cooperatively in posttranscriptional gene regulation.<br>
  Results: We tested our method with the human genes and miRNAs, predicting 431 miRNA regulatory modules. We analyze a module with genes: BTG2, WT1, PPM1D, PAK7 and RAB9B, and miRNAs: miR-15a and miR-16. Review of the literature and annotation with Gene Ontology terms reveal that the roles of these genes can indeed be closely related in specific biological processes, such as gene regulation involved in breast, renal, and prostate cancers. Furthermore, it has been reported that miR-15a and miR-16 are deleted together in certain types of cancer, suggesting a possible connection between these miRNAs and cancers. Given that most known functionalities of miRNAs are related to negative gene regulation, extending our approach and exploiting the insight thus obtained may provide clues to achieving practical accuracy in the reverse-engineering of gene regulatory networks.<br>
  Availability: A list of predicted modules is available upon request.<br>
  CONTACT: sryoon@stanford.edu</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-18 Computational Discovery of Transcriptional Regulatory Rules</span><br>
                  <span class="textomoradoGrande">Tho Hoan Pham (1), Jose Carlos Clemente (1), Kenji Satou (1) (2), Tu Bao Ho (1) (2)<br>
  1) Japan Advanced Institute of Science and Technology, 2) Institute for Bioinformatics Research and Development (BIRD)</span></p>
              <p class="textonegro">ABSTRACT: Motivation: Even in a simple organism like yeast S. cerevisiae, Transcription is an extremely complex process. The expression of sets of genes can be turned on or off by binding of specific transcription factors to the promoter regions of genes. Experimental and computational approaches have been proposed to establish mappings of DNA-binding locations of transcription factors. However, while location data obtained from experimental methods is noisy due to imperfections in the measuring methods, computational approaches suffer from over-prediction problems due to the short length of the sequence motifs bound by the transcription factors. Also, these interactions are usually environment-dependent: many regulators only bind to the promoter region of genes under specific environmental conditions. Even more, the presence of regulators at a promoter region indicates binding but not necessarily function: the regulator may act positively, negatively or not act at all. Therefore, identifying true and functional interactions between transcription factors and genes in specific environment conditions and describing the relationship between them are still open problems.<br>
  Results: We developed a method that combines expression data with genomic location information to discover (1) relevant transcription factors from the set of potential transcription factors of a target gene; and (2) the relationship between the expression behavior of a target gene and that of its relevant transcription factors. Our method is based on rule induction, a machine learning technique that can efficiently deal with noisy domains. When applied to genomic location data with a confidence criterion relaxed to p-value=0.005, and three different expression datasets of yeast S. cerevisiae, we obtained a set of regulatory rules describing the relationship between the expression behavior of a specific target gene and that of its relevant transcription factors. The resulting rules provide strong evidence of true positive gene-regulator interactions, as well as of protein-protein interactions that could serve to identify transcription complexes.<br>
  Availability: Supplementary files are available from http://www.jaist.ac.jp/&tilde;h-pham/regulatory-rules<br>
  CONTACT: h-pham@jaist.ac.jp</p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
