<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Saturday<br/>
      October 1, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><span class="textomoradoGrandeB">AREA 9: ALGORITHMS </span><span class="blanco"><strong><br>
            </strong></span><span class="textomoradoGrande">Area chairs: Pedro Larra&ntilde;aga and Martin Vingron </span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p class="textonegro"><span class="textomoradoGrandeB">OP-37 Growing Bayesian Network Models of Gene Networks from Seed Genes</span><br>
                <span class="textomoradoGrande">Jose M Pe&ntilde;a (1), Johan Bj&ouml;rkegren (2), Jesper Tegn&eacute;r (1) <br>
1) Linkoping University, 2) Karolinska Institutet</span></p>
              <p class="textonegro"> ABSTRACT: Motivation: For the last few years, Bayesian networks (BNs) have received increasing attention from the computational biology community as models of gene networks, though learning them from gene expression data is problematic: Most gene expression databases contain measurements for thousands of genes, but the existing algorithms for learning BNs from data do not scale to such highdimensional databases. This means that the user has to decide in advance which genes are included in the learning process, typically no more than a few hundreds, and which genes are excluded from it. This is not a trivial decision. We propose an alternative approach to overcome this problem.<br>
  Results: We propose a new algorithm for learning BN models of gene networks from gene expression data. Our algorithm receives a seed gene S and a positive integer R from the user, and returns a BN for those genes that depend on S such that less than R other genes mediate the dependency. Our algorithm grows the BN, which initially only contains S, by repeating the following step R+1 times and, then, pruning some genes: Find the parents and children of all the genes in the BN and add them to it. Intuitively, our algorithm provides the user with a window of radius R around S to look at the BN model of a gene network without having to exclude any gene in advance. We prove that our algorithm is correct under the faithfulness assumption. We evaluate our algorithm on simulated and biological data (Rosetta compendium) with satisfactory results.<br>
  CONTACT: jmp@ifm.liu.se</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-38 Reconsidering Complete Search Algorithms for Protein Backbone NMR Assignment</span><br>
                  <span class="textomoradoGrande">Olga Vitek (1), Chris Bailey Kellogg (2), Bruce Craig (1), Jan Vitek (1)<br>
  1) Purdue University, 2) Dartmouth College</span></p>
              <p class="textonegro">ABSTRACT: Motivation: Nuclear magnetic resonance (NMR) spectroscopy is widely used to determine and analyze protein structures. An essential step in NMR studies is determining the backbone resonance assignment, which maps individual atoms to experimentally measured resonance frequencies. Performing assignment is challenging due to the noise and ambiguity in NMR spectra. While automated procedures have been investigated, by-and-large they are still struggling to gain acceptance due to inherent limits in scalability and/or unacceptable levels of assignment error. To have confidence in the results, an algorithm should be complete, i.e., able to identify all solutions consistent with the data, including all arbitrary configurations of extra and missing peaks. The ensuing combinatorial explosion in the space of possible assignments has led to the perception that complete search is hopelessly inefficient and cannot scale to realistic data sets.<br>
  Results: This paper presents a complete branch-contract-and-bound search algorithm for backbone resonance assignment. The algorithm controls the search space by hierarchically agglomerating partial assignments and employing statistically sound pruning criteria. It considers all solutions consistent with the data, and uniformly treats all combinations of extra and missing data. We demonstrate our approach on experimental data from 5 proteins ranging in size from 70 to 154 residues. The algorithm assigns over 95% of the positions with over 98% accuracy. We also present results on simulated data from 259 proteins from the RefDB database, ranging in size from 25 to 257 residues. The median computation time for these cases is 1 minute, and the assignment accuracy is over 99%. These results demonstrate that complete search not only has the advantage of guaranteeing fair treatment of all feasible solutions, but is efficient enough to be employed effectively in practice.<br>
  Availability: The MBA2 software package is made available under an open-source software license. The data sets featured in the result section can also be obtained from the contact author.<br>
  CONTACT: ovitek@stat.purdue.edu</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-39 RNA Secondary Structural Alignment with Conditional Random Fields</span><br>
                  <span class="textomoradoGrande">Kengo Sato (1), Yasubumi Sakakibara (1)<br>
  1) Keio University, Department of Biosciences and Informatics</span></p>
              <p class="textonegro">ABSTRACT: Motivation: Computational identification of non-coding RNA regions on the genome has much attention to be investigated. However, it is essentially harder than gene-finding problems for protein-coding regions because non-coding RNA sequences do not have strong statistical signals. Since comparative sequence analysis is effective for non-coding RNA detection, efficient computational methods are expected for structural alignment of RNA sequences. Several methods have been proposed to accomplish the structural alignment tasks for RNA sequences, and we found that one of the most important points is to estimate an accurate score matrix for calculating structural alignments.<br>
  Results: We propose a novel approach for RNA structural alignment based on Conditional Random Fields (CRFs). Our approach has a specific feature compared with previous methods in the sense that the parameters for structural alignment are estimated such that the model can discriminate between correct alignments and incorrect alignments most likely, and has the generalization ability so that a satisfiable score matrix can be obtained even with a small number of sample data without overfitting. Experimental results clearly show that the parameter estimation with CRFs can outperform all the other existing methods for structural alignments of RNA sequences. Further, structural alignment search based on CRFs is more efficient for predicting non-coding RNA regions than the other scoring methods. These experimental results strongly support our discriminative method employing CRFs to estimate the score matrix parameters.<br>
  Availability: The source code of PHMMTS based on CRFs is available at http://phmmts.dna.bio.keio.ac.jp/ under the GNU public licence.<br>
  CONTACT: satoken@bio.keio.ac.jp</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-40 Fast maximum-likelihood refinement of electron microscopy images</span><br>
                  <span class="textomoradoGrande">Sjors H.W. Scheres (1), Mikel Valle (1), Jose-Maria Carazo (1)<br>
  1) Centro Nacional de Biotecnologia - CSIC</span></p>
              <p class="textonegro">ABSTRACT: Motivation: Maximum-likelihood image refinement is a promising candidate to improve attainable resolution limits in 3D-EM. However, its large CPU-requirements may prohibit its application to 3D-structure optimization.<br>
  Results: We speeded-up ML image refinement by reducing its search space over the alignment parameters. Application of this reduced-search approach to a cryo-EM dataset yielded practically identical results as the original approach, but in approximately one day instead of one week of CPU.<br>
  Availability: This work has been implemented in the publicdomain package Xmipp. Documentation and download instructions may be found at http://www.cnb.uam.es/~bioinfo<br>
  CONTACT: scheres@cnb.uam.es</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-41 A quadratic programming approach for decomposing steady-state metabolic flux distributions onto elementary modes</span><br>
                  <span class="textomoradoGrande">Jean-Marc Schwartz (1), Minoru Kanehisa (1)<br>
  1) Bioinformatics Center, Kyoto University</span></p>
              <p class="textonegro">ABSTRACT: Motivation: Steady-state flux distributions in metabolic networks can be expressed as non-negative combinations of elementary modes. However little understanding has been achieved so far in how individual elementary modes contribute to the reconstruction of actual physiological flux distributions. Results:We introduce an approach for decomposing steady-state flux distributions onto elementary modes based on quadratic programming. The decomposition is performed so as to favour modes that are closest to the actual state of the system, i.e. most relevant for biological interpretation. As an illustration, an application of this approach to a model of yeast glycolysis is presented.<br>
  Availability: Software is available upon request from the authors.<br>
  CONTACT: jean@kuicr.kyoto-u.ac.jp</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-42 Web Servicing the Biological Office </span><br>
                  <span class="textomoradoGrande">Martin Szugat (1), Daniel G&uuml;ttler (1), Katrin Fundel (1), Florian Sohler (1), Ralf Zimmer (1)<br>
  1) Department of Informatics, Ludwig-Maximilians-Universit&auml;t M&uuml;nchen</span></p>
              <p class="textonegro">ABSTRACT: Biologists routinely use standard Microsoft Office applications for standard analysis tasks. Despite ubiquitous internet resources, information needed for everyday work is often not directly and seamlessly available. Here we describe a very simple and easily extendable mechanism using Web Services to enrich standard MS Office applications with internet resources. The feature is similar in handling to spell-checker annotations. We demonstrate it with a thesaurus for biological objects, which maps names to database identifiers and vice versa via an appropriate synonym list. It is based on a Web Service providing synonyms and database identifiers for biological objects. The client application ProTag makes these features available in MS Office applications using Smart Tags and Add-Ins. This increases user efficiency and comfort and could also improve the currently awkward situation of naming biological objects. The features proposed here are directly available with only minor installation and learning efforts.<br>
  Availability: http://services.bio.ifi.lmu.de/prothesaurus/<br>
  CONTACT: Martin.Szugat@GMX.net</p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
