<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Sessions</font></strong></p>
        <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index
        </a>        
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr bgcolor="#D2F4FF">
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Thursday<br/>
      September 29, 2005</strong></p></td>
          </tr>
          <tr>
            <td valign="top" 65535 bgcolor="#CCCCCC"><p align="center" class="textonegroGrandeB"><span class="textomoradoGrandeB">AREA 1: SPECIAL SESSIONS</span><br>
              <span class="textomorado"><strong>                </strong></span><span class="textomoradoGrande">Area chairs: Angel R. Ortiz and Anna Tramontano </span></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#D2F4FF"><p align="justify" class="textomoradoGrandeB"><strong> OP-1 </strong>Augmented cell-graphs for automated cancer
                diagnosis<br>
                <span class="textomoradoGrande">Cigdem Demir (1), S. Humayun Gultekin(2), Bulent Yener (1)<br>
                1) Rensselaer Polytechnic Institute, 2) Oregon Health and
                Science University.</span></p>
              <p class="textonegro">ABSTRACT:<br>
                This work reports a novel computational method based on
                Augmented Cell Graphs (ACG), which are constructed from low
                magnification tissue images for the mathematical diagnosis of
                brain cancer (malignant glioma). An ACG is a simple,
                undirected, weighted, and complete graph in which a node
                represents a cell cluster and an edge between a pair of nodes
                defines a binary relationship between them. Both the nodes and
                the edges of an ACG are assigned weights to capture more
                information about the topology of the tissue.
                In this work, the experiments are conducted on a data set that
                comprised of 646 human brain biopsy samples from 60 different
                patients. It is shown that the ACG approach yields sensitivity of
                97.53% and specificities of 93.33% and 98.15% (for the
                inflamed and healthy, respectively) at the tissue level in glioma
                diagnosis.</p>
              <p class="textonegro">CONTACT: demir@cs.rpi.edu</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-2 A framework for computational and experimental
                methods: Identifying dimerization residues in CCR
                chemokine receptors</span><br>
                <span class="textomoradoGrande">Ana Rojas (1), Mario Mellado (1), Jos&eacute; Miguel Rodr&iacute;guez-Frade
                (1), David de Juan(1), Patricia Hernanz-Falc&oacute;n (1), Antonio
                Serrano (1), Antonio del Sol (2), Carlos Mart&iacute;nez (3), Alfonso
                Valencia (1)<br>
                1) CNB, 2) INC, 3) DIO.</span></p>
              <p class="textonegro">ABSTRACT:<br>
                Solving relevant biological problems requires answering
                complex questions. Addressing such questions traditionally
                implied the design of time-consuming experimental procedures
                which most of the times are not accessible to average-sized
                laboratories. The current trends in the actual scientific scenario
                move towards the multidisciplinary approach integrating both
                theoretical knowledge and experimental approaches. This
                combination is a powerful tool that a priori will be sufficient to
                successfully achieve the goal of shedding light to biological
                problems.<br>
                To illustrate this concept, here we show a descriptive example
                where the computational methods showed to be a key aspect to
                detect crucial players in a very important biological problem: the
                chemokine receptors dimerization. Experimental validation of
                computational procedures provide with a wealth of valuable
                information not obtainable by any of the individual approaches
                alone.<br>
                CONTACT: arojas@cnb.uam.es</p>
              <p class="textomoradoGrandeB">OP-3 ARTS: Alignment of RNA Tertiary Structures<br>
                <span class="textomoradoGrande">Oranit Dror (1), Ruth Nussinov (1), Haim Wolfson (1)<br>
              Tel Aviv University</span></p>
              <p class="textonegro">ABSTRACT:<br>
                Motivation: A fast growing number of non-coding RNAs have
                recently been discovered to play essential roles in many cellular
                processes. Similarly to proteins, understanding the functions of
                these active RNAs requires methods for analyzing their tertiary
                structures. However, in contrast to the wide range of structurebased
                approaches available for proteins, there is still a lack of
                methods for studying RNA structures.<br>
                Results: We present a new computational method named ARTS
                (Alignment of RNA Tertiary Structures). The method compares
                two nucleic acid structures (RNAs or DNAs) and detects a-priori
                unknown common substructures. These substructures can be
                either large global folds containing hundreds and even thousands
                of nucleotides or small local tertiary motifs with at least two
                successive base pairs. To the best of our knowledge, this is the
                first method of this type. The method is highly-efficient and was
                used to conduct an all-against-all comparison of all the RNA
                structures currently available in the Protein Data Bank.
                Keywords: Structural Bioinformatics, Nucleic Acid 3D
                structurecomparison, an all-against-all comparison.<br>
                Availability: The program, a web-server and supplementary
                informationare available on http://bioinfo3d.cs.tau.ac.il/ARTS.<br>
                CONTACT: oranit@post.tau.ac.il</p>
              <p class="textonegro">RNA 3D Structure and Function</p>
              <p class="textonegro"><span class="textomoradoGrandeB">OP-4 Non-coding RNAs in Ciona intestinalis</span>                <br>
                <span class="textomoradoGrande">Kristin Missal (1), Dominic Rose (1), Peter F. Stadler (1)<br>
                1) Bioinformatics Group, Dep. of Computer Science, University
                of Leipzig</span></p>
              <p class="textonegro">ABSTRACT:<br>
                Motivation: The analysis of animal genomes showed that only
                a minute part of their DNA codes for proteins. Recent
                experimental results agree, however, that a large fraction of these
                genomes is transcribed and hence is probably functional at the
                RNA level. A computational survey of vertebrate genomes has
                predicted thousands of previously unknown ncRNAs with
                evolutionary conserved secondary structures. An extension of
                these comparative studies beyond vertebrates is difficult,
                however, since most ncRNAs evolve relatively fast at the
                sequence level while conserving their characteristic secondary
                structures.<br>
                Results: Here we report on a computational screen of structured
                ncRNAs in the urochordate lineage based on a comparison of the
                genomic data from Ciona intestinalis, Ciona savignyi, and
                Oikopleura dioica. We predict more than 1000 ncRNAs with an
                evolutionarily conserved RNA secondary structure. Of these,
                about a quarter is located in introns of known protein coding
                sequences. Only a small fraction of the RNA motifs can be
                identified as known RNAs, including about 300 tRNAs, some
                100 snRNA genes, and a few microRNAs and snoRNAs.<br>
                CONTACT: kristin@bioinf.uni-leipzig.de              </p>
              <p class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><br>
            <a href="sessionSchedule.htm" class="textomoradoGrande">Back to Session Index </a></p>
        <p>&nbsp; </p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
