<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <table width="90%"  border="0" cellspacing="0" cellpadding="0">
          <tr>
            <td><p><strong><font size="3" color="5d5d5d"><span class="textomorado"><a href="javascript:if(confirm('http://www.biomedcentral.com/bmcbioinformatics  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biomedcentral.com/bmcbioinformatics'" target="_blank"><img src="images/bmcBioinformaticsLogo.gif" width="212" height="60" border="0" align="right"></a></span>1st International Student Symposium in<br>
  Computational Biology</font></strong></p>
              <p class="textomorado">&nbsp;</p>
              <p class="textomorado">INTERNATIONAL STUDENT SYMPOSIUM </p>
              <p class="textonegro">The ISCB Student Council is organizing an International Symposium to be held on Wednesday, September 28, immediately preceding ECCB 2005 in Madrid. This symposium is targeted to young computational biology researchers who are keen to develop their research communication skills, want to meet like minded colleagues, network with accomplished scientists and learn about career opportunities available in Europe and globally. </p>
              <p class="textomorado">REGISTRATION </p>
              <p class="textonegro">Register for ECCB 2005 at <a href="javascript:if(confirm('http://www.eccb05.org/registration.html  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org/registration.html'">http://www.eccb05.org/registration.html</a> and sign up for the symposium at the same time.&nbsp;If you have already registered with ECCB, but did not sign up with the symposium,&nbsp;you can still attend the symposium if you register at: <a href="javascript:if(confirm('http://www.iscbsc.org/marketing/symposium_reg.htm  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscbsc.org/marketing/symposium_reg.htm'" target="_blank">http://www.iscbsc.org/marketing/symposium_reg.htm</a>. The symposium is free of charge for all participants of ECCB.</p>
              <p class="textomorado">ECCB BEST STUDENT PRESENTATION AWARD</p>
              <p class="textonegro">Oral presentations on the symposium day will all be eligible for the best student presentation competition if the main contributor is a student. The process for selection of the best presentation will be based on the quality of the research presented, the clarity and style of the ideas and on originality. All in attendance at the symposium will be asked to submit a paper ballot to select the winner. The Student Council is grateful to the on line journal <a href="javascript:if(confirm('http://www.biomedcentral.com/bmcbioinformatics  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biomedcentral.com/bmcbioinformatics'" target="_blank">BMC Bioinformatics</a> for kindly sponsoring our &lsquo;Best Student Presentation Award&rsquo;.&nbsp;</p>
              <p class="textonegro">&nbsp;</p>
              <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
                <tr>
                  <td valign="top" bgcolor="#666666" 65535><div align="center"><span class="textoblancoGrande"><strong>Wednesday<br/>
                      September 28, 2005<br>
                      Room 7 (2nd Floor) </strong></span></div></td>
                </tr>
                <tr>
                  <td valign="top" bgcolor="#99CCCC" 65535><p align="left" class="textonegro"><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6">Keynote Talk: </b></span><b><br>
                      </b><span class="textomorado">SS-1: <a href="StudentAbstracts/1.pdf" target="_blank">How old is your fold? </a></span><br>
                      <span class="textomoradoCopy">Charlotte M. Deane </span></p>
                      
                    <p align="center" ><span class="textonegroB"><b>Abstracts selected 
                      for Presentation: </b></span></p>
                      
                    <p align="left"> <span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6">Algorithms</b> 
                      </span><br>
                      <span class="textomorado">SS-2: <a href="StudentAbstracts/2.pdf" target="_blank">Statistical 
                      alignment with a sequence evolution model allowing heterogeneous 
                      evolution behaviors along the sequence</a> </span><br>
                      <span class="textomoradoCopy">Ana Arribas Gil</span> <br>
                      <span class="textomorado">SS-3: <a href="StudentAbstracts/3.pdf" target="_blank">Hybrid 
                      approach based on a combination of methods for operon prediction 
                      </a></span><br>
                      <span class="textomoradoCopy">C. Hubans1,2, D. Hot2, Y. 
                      Lemoine2,4, E. Mephu-Nguifo3 </span><br>
                      <span class="textomorado">SS-4: <a href="StudentAbstracts/4.pdf" target="_blank">An 
                      Alternative Method for Detecting Conserved Elements in Multiple 
                      Sequence Alignments </a></span><br>
                      <span class="textomoradoCopy">Pavol Hanus1, Janis Dingel1, 
                      Joachim Hagenauer1 and Jakob C. Mueller2 <br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"><strong>Databases</strong> 
                      </span><br>
                      <span class="textomorado">SS-5: <a href="StudentAbstracts/5.pdf" target="_blank">Save 
                      a thousand words: Advanced visualisations for Pfam and iPfam 
                      </a></span><br>
                      <span class="textomoradoCopy">Benjamin Schuster-B&ouml;ckler, 
                      Robert D. Finn, Alex Bateman </span><br>
                      <span class="textomorado">SS-6: <a href="StudentAbstracts/6.pdf" target="_blank">High-throughput 
                      computational proteomics - challenges and solutions in the 
                      creation of the Genome Annotating Proteome Pipeline (GAPP)</a> 
                      </span><br>
                      <span class="textomoradoCopy">Ian Shadforth </span></p>
                      
                    <p align="left"><b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>Gene 
                      Expression </b><br>
                      <span class="textomorado">SS-7: <a href="StudentAbstracts/7.pdf" target="_blank">Highly 
                      Expressed Proteins Contain Increased Frequency of Alanine 
                      in the Second Amino Acid Position </a></span><br>
                      <span class="textomoradoCopy">Age Tats, Maido Remm, Tanel 
                      Tenson<br>
                      </span><span class="textomoradoCopy"><br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"><strong>Genes 
                      and Genomes</strong> </span><br>
                      <span class="textomorado">SS-8: <a href="StudentAbstracts/8.pdf" target="_blank">discoveryBASE. 
                      Virtual screening for disease associated proteins </a></span><br>
                      <span class="textomoradoCopy">Ronald Rapberger, Martin Haiduk, 
                      Paul Perco, Arno Lukas, Christian Siehs and Bernd Mayer 
                      <br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"><strong>Phylogenetics</strong> 
                      </span><br>
                      <span class="textomorado">SS-9: <a href="StudentAbstracts/9.pdf" target="_blank">Long 
                      branch attraction and topology testing using weighted least-squares</a></span><a href="StudentAbstracts/9.pdf" target="_blank"> 
                      </a><br>
                      <span class="textomoradoCopy">Borys Wrobel <br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"> 
                      <strong>Protein Folding and Function</strong> </span><br>
                      <span class="textomorado">SS-10: <a href="StudentAbstracts/10.pdf" target="_blank">False 
                      occurrences of functional motifs on protein sequences highlight 
                      evolutionary constraints</a></span><br>
                      <span class="textomoradoCopy">Allegra Via 1, Federico Gherardini, 
                      Enrico Ferraro, Gianpaolo Scalia Tomba, Gabriele Ausiello 
                      and Manuela Helmer-Citterich </span><br>
                      <span class="textomorado">SS-11: <a href="StudentAbstracts/11.pdf" target="_blank">Visualization 
                      and clustering of Protein Local Conformational Space using 
                      Geometric Invariant Theory </a></span><br>
                      <span class="textomoradoCopy">Ashish V. Tendulkar, Pramod 
                      P. Wangikar</span> <br>
                      <span class="textomorado">SS-12: <a href="StudentAbstracts/12.pdf" target="_blank">Structural 
                      Bioinformatics of Protein-Bound </a></span><br>
                      <span class="textomoradoCopy">Water Christopher Bottoms, 
                      Tommi White, John J. Tanner </span><br>
                      <span class="textomorado">SS-13: <a href="StudentAbstracts/13.pdf" target="_blank">Simple 
                      fold composition and modular architecture of the nuclear 
                      pore complex </a></span><br>
                      <span class="textomoradoCopy">Damien Devos, Svetlana Dokudovskaya, 
                      Rosemary Williams, Frank Alber, Narayanan Eswar, Brian T. 
                      Chait, Michael P. Rout, and Andrej Sali </span><br>
                      <span class="textomorado"> SS-14: <a href="StudentAbstracts/14.pdf" target="_blank">REPPER&nbsp;repeats&nbsp;and&nbsp;their&nbsp;periodicities&nbsp;in&nbsp;fibrous&nbsp;proteins</a></span><br>
                      <span class="textomoradoCopy">Markus&nbsp;Gruber,Johannes&nbsp;Soding&nbsp;and&nbsp;Andrei&nbsp;N.&nbsp;Lupas&nbsp; 
                      </span><br>
                      <span class="textomorado">SS-15: <a href="StudentAbstracts/15.pdf" target="_blank">A 
                      novel pair-to-pair substitution matrix improves contact 
                      prediction in protein cores </a></span><br>
                      <span class="textomoradoCopy">Milana Frenkel-Morgenstern, 
                      Eran Eya, Vladimir Sobolev, Shmuel Pietrokovski</span><br>
                      <span class="textomorado">SS-16: <a href="StudentAbstracts/16.pdf" target="_blank">Molecular 
                      Recognition and Partner Prediction for Transient Protein 
                      Complexes: CDK-Cyclin Homologue Interactions </a></span><br>
                      <span class="textomoradoCopy">Xueping Quan, Peter Doerner, 
                      and Dietlind L.Gerloff <br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"> 
                      <strong>Systems Biology</strong></span> <br>
                      <span class="textomorado">SS-17: <a href="StudentAbstracts/17.pdf" target="_blank">Discovering 
                      regulatory modules from heterogeneous information sources</a></span><br>
                      <span class="textomoradoCopy">Karen Lemmens T. De Bie, P. 
                      Monsieurs, K. Engelen, B. De Moor, N. Cristianini, and K. 
                      Marchal</span><br>
                      <span class="textomorado">SS-18: <a href="StudentAbstracts/18.pdf" target="_blank">A 
                      semi-automated method to study the intersection of metabolic 
                      network and interactome </a></span><br>
                      <span class="textomoradoCopy">Joan Planas Iglesias </span><br>
                      <span class="textomorado">SS-19: <a href="StudentAbstracts/19.pdf" target="_blank">Is 
                      there a relationship between protein connectivity and evolutionary 
                      rate? </a></span><br>
                      <span class="textomoradoCopy">Ramazan Saeed </span><br>
                      <span class="textomorado">SS-20: <a href="StudentAbstracts/20.pdf" target="_blank">BioQSpace: 
                      An interactive visualisation tool for clustering MEDLINE 
                      abstracts</a></span><br>
                      <span class="textomoradoCopy">Anna Divoli, Rasmus Winter, 
                    Steve Pettifer&amp; Terri Attwood<br>
                    <br>
                      
                    <p align="center"><span class="textonegroB"><b>Abstracts selected 
                      for Posters:</b></span></p>
                    </span>
<p align="left"><b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>Algorithms</b><br>
                      <span class="textomorado">SS-21: <a href="StudentAbstracts/21.pdf" target="_blank">Rigid 
                      Body Protein-Protein Docking Using Biochemical Data </a></span><br>
                      <span class="textomoradoCopy">Domantas Motiejunas, Razif 
                      Gabdoulline, Rebecca Wade</span><br>
                      <span class="textomorado">SS-22: <a href="StudentAbstracts/22.pdf" target="_blank">Guide 
                      structure calculation: a critical step for the accuracy 
                      of progressive multiple sequence alignment algorithms </a></span><br>
                      <span class="textomoradoCopy">Laure Vescovo Jean-Christophe 
                      Aude, Geraldine Polaillon </span><br>
                      <br>
                      <b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>Databases</b> 
                      <br>
                      <span class="textomorado">SS-23: <a href="StudentAbstracts/23.pdf" target="_blank">IGD: 
                      A Resssource for Human intronless genes </a></span><br>
                      <span class="textomoradoCopy">Amel Louhichi</span> <br>
                      <span class="textomorado">SS-24: <a href="StudentAbstracts/24.pdf" target="_blank">Developing 
                      a knowledge base for PID-expert system </a></span><br>
                      <span class="textomoradoCopy">Crina Samarghitean, Jouni 
                      Vliaho, Mauno Vihinen</span><br>
                      <span class="textomorado">SS-25: <a href="StudentAbstracts/25.pdf" target="_blank">Codon 
                      usage analysis based on web services </a></span><br>
                      <span class="textomoradoCopy">Denis Shestakov, Tapio Salakoski, 
                      Mauno Vihinen. </span><br>
                      <span class="textomorado">SS-26: <a href="StudentAbstracts/26.pdf" target="_blank">VITAMIN: 
                      An Ontology-Driven Information System for Mutant Annotation</a></span><a href="StudentAbstracts/26.pdf" target="_blank"> 
                      </a><br>
                      <span class="textomoradoCopy">&Oacute;scar D. S&aacute;nchez, 
                      Ana Conesa, Jos&eacute; Alberto Maldonado, Montserrat Robles, 
                      Manuel Tal&oacute;n</span><br>
                      <span class="textomorado">SS-27: <a href="StudentAbstracts/27.pdf" target="_blank">MicrobialTREE: 
                      A user friendly tool for bacterial taxonomy navigation</a></span><br>
                      <span class="textomoradoCopy">Pablo Pareja-Tobes, Marina 
                      Manrique, Eduardo Pareja-Tobes, Raquel Tobes y Eduardo Pareja 
                      <br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"><strong> 
                      Genes and Genomes </strong></span><br>
                      <span class="textomorado"> SS-28: <a href="StudentAbstracts/28.pdf" target="_blank">Bioinformatic 
                      sequence identification from sequence family databases </a></span><br>
                      <span class="textomoradoCopy">Anne-Muriel Arigon, Guy Perriere 
                      and Manolo Gouy </span><br>
                      <span class="textomorado">SS-29: <a href="StudentAbstracts/29.pdf" target="_blank">In 
                      silico markovian prediction of variable and constant regions 
                      of lentivirus genomes </a></span><br>
                      <span class="textomoradoCopy">Aur&eacute;lia Boissin-Quillon, 
                      Didier Piau, Caroline Leroux<br>
                      </span><span class="textomorado">SS-30: <a href="StudentAbstracts/30.pdf" target="_blank">Aligning 
                      intron-exon structures of alternatively spliced genes </a></span><br>
                      <span class="textomoradoCopy">Michael Sammeth, Eduardo Eyras 
                      and Roderic Guigo</span><br>
                      <span class="textomorado">SS-31: <a href="StudentAbstracts/31.pdf" target="_blank">Addressing 
                      Drosophila Gene Duplications: Protein Sequence Redundancy 
                      and Genome Evolution</a></span><a href="StudentAbstracts/31.pdf" target="_blank"> 
                      </a><br>
                      <span class="textomoradoCopy">Carlos Quijano </span><br>
                      <span class="textomorado">SS-32: <a href="StudentAbstracts/32.pdf" target="_blank">ELEANALYZER: 
                      an automated method for analysis of retrotransposons in 
                      genomes and prediction of their insertion sites </a></span><br>
                      <span class="textomoradoCopy">Kamal Rawal, Alok Bhattacharya 
                      Sudha Bhattacharya and Ram Ramaswamy</span><br>
                      <span class="textomorado">SS-33: <a href="StudentAbstracts/33.pdf" target="_blank">BlastXtract: 
                      a web-based tool for managing and visualizing results from 
                      large translated BLAST and FastA searches </a></span><br>
                      <span class="textomoradoCopy">Marcus Claesson </span><br>
                      <span class="textomorado">SS-34: <a href="StudentAbstracts/34.pdf" target="_blank">Measuring 
                      Selection in RNA molecules</a></span><br>
                      <span class="textomoradoCopy">Naila Mimouni, Rune lyngsoe 
                      and Jotun Hein. </span><br>
                      <span class="textomorado">SS-35: <a href="StudentAbstracts/35.pdf" target="_blank">Prediction 
                      of Heat-Induced Replication Sites in Replication Origins 
                      from Escherichia Coli and Other Plasmidic Replicons </a></span><br>
                      <span class="textomoradoCopy">Gonz&aacute;lez-Soltero, R., 
                      Botello, E., and Jim&eacute;nez-S&aacute;nchez, A. </span><br>
                      <span class="textomorado">SS-36: <a href="StudentAbstracts/36.pdf" target="_blank">In 
                      silico modelling of the PCR process for detection of bacteria 
                      from clinical samples </a></span><br>
                      <span class="textomoradoCopy">Triinu K&otilde;ressaar, Maido 
                      Remm </span><br>
                      <span class="textomorado">SS-37: <a href="StudentAbstracts/37.pdf" target="_blank">Understanding 
                      Promoter Logic </a></span><br>
                      <span class="textomoradoCopy">Utz J. Pape</span></p>
                      <p align="left"><b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>Microarrays</b> <br>
                          <span class="textomorado">SS-38: <a href="StudentAbstracts/38.pdf" target="_blank">Accounting for Probe-level Noise in Principal Component Analysis of Microarray Data</a></span><br>
                          <span class="textomoradoCopy">Guido Sanguinetti </span><br>
                          <span class="textomorado">SS-39: <a href="StudentAbstracts/39.pdf" target="_blank">Data-adaptive test statistics for microarray data </a></span><br>
                          <span class="textomoradoCopy">Sach Mukherjee </span><br>
                          <span class="textomorado">SS-40: <a href="StudentAbstracts/40.pdf" target="_blank">GALGO: Software for building statistical models from microarray data </a></span><br>
                          <span class="textomoradoCopy">Victor Trevino and Francesco Falciani </span></p>
                      
                    <p align="left"><b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>Phylogenetics</b> 
                      <br>
                      <span class="textomorado">SS-41: <a href="StudentAbstracts/41.pdf" target="_blank">Exploring 
                      the evolution and the diversity of bacterial genes </a></span><br>
                      <span class="textomoradoCopy">Zara Ghazoui </span><br>
                      <span class="textomorado">SS-42: <a href="javascript:if(confirm('http://bioinfo.cnio.es/eccb05/StudentAbstracts/42..pdf  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://bioinfo.cnio.es/eccb05/StudentAbstracts/42..pdf'" target="_blank">Biological/Chemical/ 
                      and Physical behavioral analyses for Protein Classifications</a></span><br>
                      <span class="textomoradoCopy">Rauf Ahmed Shams Malick, Dr. 
                      Kamran Azim </span><br>
                      <span class="textomorado">SS-43: <a href="StudentAbstracts/43.pdf" target="_blank">More 
                      than 140 genes have been acquired through horizontal gene 
                      transfer from Bacteria to rumen ciliates</a></span><br>
                      <span class="textomoradoCopy">Guanola Ricarda, Neil R. McEwanb, 
                      Johannes H.P. Hacksteinc and Martijn A. Huynena. </span><br>
                      <span class="textomorado">SS-44: <a href="StudentAbstracts/44.pdf" target="_blank">Bioinformatics 
                      tools to encode and integrate microscopy time-lapse sequences 
                      for drug discovery: Lineage analysis the basis for novel 
                      cell-based assays </a></span><br>
                      <span class="textomoradoCopy">Imtiaz A. Khan, Lee Campbell, 
                      Paul J Smith and Rachel J Errington<br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"><strong>Protein 
                      Structure and Function </strong></span><br>
                      <span class="textomorado">SS-45: <a href="StudentAbstracts/45.pdf" target="_blank">The 
                      ANNOTATOR sequence analysis suite </a></span><br>
                      <span class="textomoradoCopy">Georg Schneider, Michael Wildpaner, 
                      Florian Leitner, Birgit Eisenhaber, Miklos Kozlovsky, Werner 
                      Kubina, Sebastian Maurer-Stroh, Maria Novatchkova, Alexander 
                      Schleiffer, Sun Tian and Frank Eisenhaber </span><br>
                      <span class="textomorado">SS-46: <a href="StudentAbstracts/46.pdf" target="_blank">Sequence 
                      and Structural Analysis of Extracellular Matrix and Cell 
                      Adhesion Molecules</a></span><a href="StudentAbstracts/46.pdf" target="_blank"> 
                      </a><br>
                      <span class="textomoradoCopy">Hosil Park, Terri Attwood, 
                      Lydia Tabernero, and Jordi Bella </span><br>
                      <span class="textomorado">SS-47: <a href="StudentAbstracts/47.pdf" target="_blank">FILTREST3D: 
                      program for discrimination of protein structure models that 
                      match the restraints from experimental data </a></span><br>
                      <span class="textomoradoCopy">Michal J. Gajda, Marta Kaczor, 
                      Anastasia Bakulina, Janusz M. Bujnicki </span><br>
                      <span class="textomorado">SS-48: <a href="StudentAbstracts/48.pdf" target="_blank">Computational 
                      approach for understanding substrate specificity of nonribosomal 
                      peptide synthetases </a></span><br>
                      <span class="textomoradoCopy">Mohd. Zeeshan Ansari, Rajesh 
                      S. Gokhale and Debasisa Mohanty </span><br>
                      <span class="textomorado">SS-49: <a href="StudentAbstracts/49.pdf" target="_blank">Discovery 
                      of Protein Substructures in EM Maps </a></span><br>
                      <span class="textomoradoCopy">Keren Lasker, Oranit Dror, 
                      Ruth Nussinov, Haim J.Wolfson</span><br>
                      <span class="textomorado">SS-50: <a href="StudentAbstracts/50.pdf" target="_blank">Amino 
                      Acid Dependent Weighting Factors for FFT-based Unbound Protein 
                      Docking </a></span><br>
                      <span class="textomoradoCopy">Philipp Heuser, Dietmar Schomburg 
                      </span><br>
                      <span class="textomorado">SS-51: <a href="StudentAbstracts/51.pdf" target="_blank">Molecular 
                      Modelling and Dynamics of Membrane Proteins </a></span><br>
                      <span class="textomoradoCopy">Shiva Amiri </span><br>
                      <span class="textomorado">SS-52: <a href="StudentAbstracts/52.pdf" target="_blank">B7 
                      and CD28 family immunoglobulins structure-function analysis 
                      </a></span><br>
                      <span class="textomoradoCopy">Xiaoxia Ge </span><br>
                      <span class="textomorado">SS-53: <a href="StudentAbstracts/53.pdf" target="_blank">Amino-Acid 
                      Substitutions In Membrane Proteins: Applications To Homology 
                      Recognition And Comparative Modelling </a></span><br>
                      <span class="textomoradoCopy">Younes MOKRAB and Kenji MIZUGUCHI 
                      </span></p>
                      
                    <p align="left"><b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>SNPs</b> 
                      <br>
                      <span class="textomorado">SS-54: <a href="StudentAbstracts/54.pdf" target="_blank">Prediction 
                      of the insurgence of human genetic diseases due to single 
                      point protein mutation</a></span><br>
                      <span class="textomoradoCopy">Calabrese Remo, Capriotti 
                      Emidio, Fariselli Piero and Casadio Rita <br>
                      <br>
                      </span><span class="textonegroB"><b><img src="images/icono_simple.gif" width="14" height="6"></b></span><span class="textonegroB"><strong>Systems 
                      Biology</strong></span> <br>
                      <span class="textomorado">SS-55: <a href="StudentAbstracts/55.pdf" target="_blank">How 
                      system biology can improve the production of L-carnitine 
                      using Escherichia coli? </a></span><br>
                      <span class="textomoradoCopy">Angel Sevilla Camins </span></p>
                      <p align="left"><b class="textonegroB"><span class="textonegroB"><img src="images/icono_simple.gif" width="14" height="6"></span>Text Mining </b><br>
                          <span class="textomorado">SS-56: <a href="StudentAbstracts/56.pdf" target="_blank">Natural Language Processing and Information Retrieval Techniques Applied to the Reproductive Biology Literature </a></span><br>
                          <span class="textomoradoCopy">R. Farhan, J. D. Aplin, T. K. Attwood, M. McGee Wood and C. P. Sibley<br>
        University of Manchester, UK</span></p></td>
                </tr>
                <tbody>
                </tbody>
              </table>
            <p>&nbsp;</p></td>
          </tr>
        </table>
      </div>
      <p>&nbsp;</p>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
