<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Tutorials</font></strong></p>
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr>
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Wednesday<br/>
      September 28, 2005</strong></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><p align="justify"> <span class="textomorado">9:00/13:15 - Room 9 (2nd Floor)</span><br>
                <span><strong><a name="T1" id="T1"></a></strong><span class="textomoradoGrandeB"><b>T1: Mining Structured Data in Bioinformatics</b></span><b><br>
                </b></span> <span class="textomoradoGrande"> Prof. Stefan Kramer</span></p>
              <p align="justify" class="textonegro">The topic of the tutorial is mining in structured data. This is particularly relevant for data mining applications in bioinformatics, since the majority of biological data is not kept in databases consisting of a single, flat table. Instead, we are frequently dealing with databases of structured and linked objects. In other words, the &ldquo;objects&rdquo; in bioinformatics databases often have a rich internal structure and are connected by some relation. (Consider, for instance, databases of proteins, small molecules, metabolic and regulatory networks, text databases, etc.) The tutorial will give an overview of data mining techniques for sequences, trees, graphs and relational databases. We will present techniques for both descriptive and predictive data mining in this context. In descriptive mining, we are looking for local patterns to characterize the data. In predictive mining, we are looking for models that can be used to make predictions for new, unseen cases. Along the two dimensions (types of data and predictive/descriptive), the tutorial is organized as follows: the first two parts of the tutorial are devoted to descriptive mining in databases of itemsets, strings and sequences, trees, graphs and relational databases. The third part of the tutorial deals with predictive mining based on propositionalization (i.e., feature construction using patterns), instance-based learning and kernel methods for graph and relational databases.</p>
              <p align="justify"><a href="tuto_kramer.htm" class="textomoradoGrande">More Information </a></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><span class="textomorado">9:00/13:15 - Room 9 Bis (1st Floor)</span><br>
              <span><strong><a name="T2" id="T2"></a></strong><span class="textomoradoGrandeB"><b>T2: GEPAS: New challenges in microarray data analysis</b></span><b><br>
              </b></span> <span class="textomoradoGrande"> Dr. Joaquin Dopazo</span>
              <p align="justify" class="textonegro">The need to associate both biochemical function and cellular roles to protein domains, rather than the individual proteins, has been recognized for a number of years. However, the ability to identify functional and/or evolutionary domains in proteins from the many full genomic sequence projects is still somewhat problematic. The problem is compounded by the large number of proteins and domains with no obvious currently recognized homologues. In research on these and related problems, our group has developed a number of tools and discovered some interesting examples of domain functions and deep evolutionary histories. Two cases will be discussed demonstrating both the utility of various bioinformatics approaches and the importance of very large-scale data integration.</p>
              <p align="justify"><a href="tuto_dopazo.htm" class="textomoradoGrande">More Information </a></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><p align="justify"><span class="textomorado">14:15/18:00 - Room 9 (2nd Floor)</span><br>
                <span><strong><a name="T3" id="T3"></a></strong><span class="textomoradoGrandeB"><b>T3: InterPro, exploring a powerful protein diagnostic tool</b></span><b><br>
                </b></span> <span class="textomoradoGrande"> Dr. Jennifer McDowall</span></p>
              <p align="justify" class="textonegro">InterPro is an integrated protein resource that provides protein annotation and classification at family and domain levels. InterPro combines the major signature databases, PROSITE, PRINTS, PFAM, PRODOM, SMART, TIGRFAM, PIR Superfamily, GENE3D, SUPERFAMILY, and PANTHER, as well as structural information from PDB, MSD, CATH, SCOP and SWISS-MODEL, into a unified database. This tutorial is designed to allow users to get the most value out of the database, and will focus on the type and organisation of annotated data, the different query methods possible, understanding the different visualisations of the data, as well as exploring the multiple external links and cross-references available.</p>
              <p align="justify"><a href="tuto_mcdowall.htm" class="textomoradoGrande">More Information </a></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><span class="textomorado">14:15/18:00 - Room 9 Bis (1st Floor)<br>
              <strong><a name="T4" id="T4"></a></strong></span><span><span class="textomoradoGrandeB"><b>T4: Computational proteomics</b></span><b><br>
              </b></span> <span class="textomoradoGrande"> Prof. Colinge Jacques</span>
              <p align="justify" class="textonegro">Proteomics has become an important approach to analyze biological samples and it extensively uses mass spectrometry to identify and characterize proteins. This tutorial will introduce the audience to the central problem of searching mass spectrometry data against a database of proteins. This presentation should stimulate the interest of bioinformatics researchers in other fields and provide a concise though accessible introduction to life scientists. The last part of the tutorial will rapidly cover other important problems in mass spectrometry data analysis such as peptide de novo sequencing, eukaryote genome search and protein quantification and characterization.</p>
              <p align="justify"><a href="tuto_jacques.htm" class="textomoradoGrande">More Information </a></p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p>&nbsp;</p>
      </div>
      <p>&nbsp;</p>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
