<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Tutorials</font></strong></p>
        <a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a>
<table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr>
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Wednesday<br/>
      September 28, 2005</strong></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><p align="justify"><span class="textomorado">14:15/18:00 - Room 9 (2nd Floor)</span><br>
                <span><strong><a name="T3" id="T3"></a></strong><span class="textomoradoGrandeB"><b>T3: InterPro, exploring a powerful protein diagnostic tool</b></span><b><br>
                </b></span> <span class="textomoradoGrande"> Dr. Jennifer McDowall</span></p>
              <p class="textonegro">InterPro is an integrated protein resource that provides protein annotation and classification at family and domain levels. InterPro combines the major signature databases, PROSITE, PRINTS, PFAM, PRODOM, SMART, TIGRFAM, PIR Superfamily, GENE3D, SUPERFAMILY, and PANTHER, as well as structural information from PDB, MSD, CATH, SCOP and SWISS-MODEL, into a unified database. This tutorial is designed to allow users to get the most value out of the database, and will focus on the type and organisation of annotated data, the different query methods possible, understanding the different visualisations of the data, as well as exploring the multiple external links and cross-references available.</p>
              <p><span class="textomorado">Synopsis:</span><span class="textomorado"></span></p>
              <p class="textonegro">The aim of this tutorial is to familiarise users with the wealth of annotated protein data available within the InterPro database, how to extract this information, and how to use InterPro to analyse and annotate protein sequences using the web interface. By the end of the tutorial, participants should be able to confidently navigate InterPro, be able to carry out a variety of search queries, be aware of all the different data presentations available, what information they yield, and what external links and cross-references are available.</p>
              <p class="textonegro"><b>OUTLINE OF TOPICS COVERED</b><br>
  Introduction to InterPro and fundamentals of the member databases<br>
  Overview of the InterPro entry<br>
  Understanding the graphical views<br>
  InterPro information mining<br>
  InterPro sequence analysis</p>
              <p class="textonegro"><b>INTRODUCTION TO INTERPRO AND FUNDAMENTALS OF THE MEMBER DATABASES</b><br>
  InterPro is a searchable database that provides information on the function, annotation and classification of proteins, with over 80% of the proteins in UniProt being represented in InterPro. InterPro combines the major signature databases into a unified protein database, with each member database using different methods to derive their signatures: Prosite (patterns and profiles), Prints (motifs), Prodom (sequence clustering), Pfam (HMM), Smart (HMM), Tigrfam (HMM), PIR Superfamily (HMM), Gene3D (HMM), Superfamily (HMM), and Panther (HMM). Furthermore, each member database refines their methods through their choice of seed alignments and post-processing techniques, in order to attain specific goals. Prosite patterns, Prints, Tigrfam, PIR Superfamily and Panther tend to concentrate on identifying proteins families, grouping them with regard to sequence or functional restraints. Prosite profiles, Prodom, Smart, Pfam, Gene3D and Superfamily are often used to identify domains, grouping them by functional, sequence or structural relatedness. By combining the different methods, InterPro is able to produce a hierarchical classification scheme that provides information on the relationships between different proteins families in terms of sequence, structural and functional divergence, as well as providing information on the domain architecture of the proteins within these families. The tutorial will give an overview of the different signature methods used by the member databases, and the contribution each makes to InterPro.</p>
              <p class="textonegro"><b>OVERVIEW OF THE INTERPRO ENTRY</b><br>
  Protein sequences are grouped together into InterPro entries based on the protein signatures from the member databases. The groups are defined as families, domains, repeats or sites. Each InterPro entry has a unique accession number, name, abstract describing features of the proteins associated with the entry, and literature references with links to PubMed. The signatures that define the entry are listed along with links to the relevant entries in the member databases, providing direct access to the documentation for each signature. Relationships with other entries are listed with their links. These relationships can be either parent/child type, which divide families or domains into more closely related sub-groups to produce a hierarchical classification scheme, or contains/found in type, which describe the organisation of domains, repeats and sites within families. Entries are also annotated with respect to GO terms, providing information on the process, function and component of the proteins within an entry. InterPro entries contain a variety of external links, including those to the structural databases PDB, EMSD, CATH and SCOP, and to the databases MEROPS, PANDIT, Blocks, IntEnz, CAZy, IUPHAR, COMe and CluSTr. The taxonomic coverage of an entry is displayed by a descriptive wheel, which permits the user to select all the sequences from specific taxonomic groups. The tutorial will cover all the features of the InterPro entry, and the external links and cross-references provided, showing participants how to extract the information they need.</p>
              <p class="textonegro"><b>UNDERSTANDING THE GRAPHICAL VIEWS</b><br>
  InterPro provides a number of different graphical views to display the protein matches making up an entry. The detailed view provides a graphical description for each of the proteins in an entry ordered by either accession number or name, or restricts the display to only those proteins whose structure is known. All the signatures that hit the proteins are displayed, thereby providing a comprehensive view of each protein, with links provided to the member databases and to related InterPro entries. The structural features of proteins as described by PDB, CATH, SCOP and SWISS-MODEL are displayed two-dimensionally in relation to the signatures, as well as three-dimensionally using AstexViewer. InterPro also includes an overview, where the signatures and structural features are condensed into a simplified graphical view, as well as a tabular format of the protein data. The different domain combinations found in the set of proteins within an entry can be viewed using the InterPro Domain Architecture. The tutorial will provide an in-depth look at the different ways to view the signature information, how to mine the information contained within the views, and how to navigate between the different views and their external links. The use of AstexViewer to visualise the structural features of the proteins will also be explored.</p>
              <p class="textonegro"><b>INTERPRO INFORMATION MINING </b><br>
  InterPro can be searched in a number of different ways. The simple text search facility allows queries using keywords, UniProt accession numbers, GO terms, or InterPro entry numbers. The simple InterPro SRS search enables more complex queries, providing two field queries, one from InterPro and the other from the list of protein matches. InterPro can also be queried through SRS either directly or indirectly as a database linked to other databases, with the possibility of creating different views, as well as recovering FASTA-format sequences. The tutorial will review the different methods of querying and the syntax that can be used, and will provide an opportunity to see the results of different types of searches.</p>
              <p class="textonegro"><b>INTERPRO SEQUENCE ANALYSIS </b><br>
  There is also a sequence search facility using the web-based server of InterProScan, which permits the sequence analysis and characterisation of unknown protein sequences. Nucleotide sequences, both DNA and RNA, can also be used to query InterProScan, where the sequence used in the query is translated in all six frames. Using InterProScan, InterPro takes each sequence and analyses it against one or more of the member databases using preconfigured cut-off thresholds. Following analysis, each result is returned and combined, and then the InterPro entries and sequence signatures are returned to the submitter as a graphical view with links to both InterPro and SRS. The tutorial will familiarise participants with using InterProScan as a tool to annotate and characterise sequences, using an example sequence to search InterPro via InterProScan, and to analyse the results using the InterPro view and the SRS view.</p>              <p align="justify" class="textonegro">&nbsp;</p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><span class="textomorado">14:15/18:00 - Room 9 Bis (1st Floor)<br>
              <strong><a name="T4" id="T4"></a></strong></span><span><span class="textomoradoGrandeB"><b>T4: Computational proteomics</b></span><b><br>
              </b></span> <span class="textomoradoGrande"> Prof. Colinge Jacques</span>
              <p align="justify" class="textonegro">Proteomics has become an important approach to analyze biological samples and it extensively uses mass spectrometry to identify and characterize proteins. This tutorial will introduce the audience to the central problem of searching mass spectrometry data against a database of proteins. This presentation should stimulate the interest of bioinformatics researchers in other fields and provide a concise though accessible introduction to life scientists. The last part of the tutorial will rapidly cover other important problems in mass spectrometry data analysis such as peptide de novo sequencing, eukaryote genome search and protein quantification and characterization.</p>
              <p align="justify" class="textonegro">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a> </p>
        <p>&nbsp;</p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
