<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Tutorials</font></strong></p>
        <a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a>
<table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr>
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Wednesday<br/>
      September 28, 2005</strong></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><span class="textomorado">9:00/13:15 - Room 9 Bis (1st Floor)</span><br>
              <span><strong><a name="T2" id="T2"></a></strong><span class="textomoradoGrandeB"><b>T2: GEPAS: New challenges in microarray data analysis</b></span><b><br>
              </b></span> <span class="textomoradoGrande"> Dr. Joaquin Dopazo</span>
              <p class="textonegro">DNA microarray technology is an essential tool for studying biological processes at the genomic level. Nevertheless, with the advent of genome-wide methodologies, new challenges have arisen related to the analysis of huge amounts of data being produced. Important topics in microarray data analysis include: data processing, normalization and data transformation procedures; clustering; supervised classification and gene selection; and functional annotation. The proposed tutorial will address these issues by combining theoretical lectures, with practical sessions. The practical sessions will ultimately demonstrate the use of our web-based suite of tools, GEPAS, for microarray data analysis.</p>
              <p><span class="textomorado">Synopsis:</span><span class="textomorado"></span></p>
              <p class="textonegro"><b>1. Outline.</b><br>
  In the tutorial several highly relevant topics in the analysis of cDNA microarray data will be taught. The tutorial will include theoretical lectures and practical sessions during which state-of-the-art methods will be used, as implemented in GEPAS (Herrero et al., 2003, NAR 31, 3461-3467; http://www.gepas.org). Our group has developed this suite of tools for DNA array data analysis that, in addition to being used for training purposes during the tutorial, is also becoming standard, with more than 300 experiments analyzed per day.The tutorial will illustrate the most common steps of data preprocessing followed by different methodologies used to answer different scientific questions for which microarray are used. These include clustering, identification of differentially expressed genes, supervised classification, gene expression signatures and functional annotation of the results.</p>
              <p class="textonegro"><b>2.1. Data processing</b><br>
  2.1.1 Normalization of cDNA arrays<br>
  The course will cover: reading GenePix data, initial diagnostic plots, diagnostic plots for print-order normalization, print-tip loess normalization, and evaluation of normalization results. Our web server, GEPAS, through the DNMAD tool (Vaquerizas et al., Bioinformatics 2004; Herrero et al., NAR, 2004) gives access to print-tip normalization and print order normalization using loess, as implemented in the limma and marrayNorm packages in Bioconductor. Print-tip normalization is one of the most commonly used normalization methods and has repeatedly shown, to perform well (Yang et al., 2002 NAR 30:e15; Smyth &amp; Speed, 2003 submitted). </p>
              <p class="textonegro"><b>2.1.2. Data transformation</b><br>
  In addition to normalization, data must be cleaned and transformed. This theoretical/practical session will deal with the most common transformation steps. GEPAS implements two interfaces for data transformation, the PREPROCESSOR (Herrero et al., 2003, Bioinformatics 19, 655-656) and Knowledge Filtering, which provides the user with the most common transformations: logarithmic, merging of replicates, filtering and/or imputation of missing values by different methods, filtering of flat patterns, filtering by biological information on genes and standardization of patterns. </p>
              <p class="textonegro"><b>2.2. Clustering</b><br>
  The identification of co-expressing genes or the definition of subgroups of experimental conditions based on the comparison of gene expression profiles is achieved by means of the application of clustering techniques. The tutorial will review different clustering techniques. Aggregative hierarchical clustering in its different variants (average-linkage, single-linkage, complete-linkage, etc.) (Sneath &amp; Sokal, 1973, Numerical Taxonomy. W. H. Freeman, San Francisco) is still one of the preferred choices for the analysis of gene expression patterns. As an alternative to hierarchical clustering, other non-hierarchical methods, such as k&shy;means (Hartigan, 1975 Clustering algorithms. Wiley, New York), have been used. Other authors have proposed the use of neural networks as an alternative (Tamayo et al., 1999 Proc. Natl. Acad. Sci. USA 96:2907-2912; Herrero et al., 2001 Bioinformatics. 17:126-136). Unsupervised neural networks, such as Self-Organising Maps (SOM) (Kohonen, 1997 Self-organizing maps, Berlin. Springer-Verlag) or the Self-Organising Tree Algorithm (SOTA) (Dopazo and Carazo, 1997 J. Mol. Evol 44:226-233), provide a more robust framework, appropriate for clustering large amounts of noisy data. The GEPAS package implements some of the most used clustering techniques, including average linkage, k-means, SOM, SOTA (Herrero et al., 2001 Bioinformatics. 17:126-136), SOM-UPGMA (Herrero and Dopazo, 2002 Journal of Proteome Research. 1(5), 467-470) and others. </p>
              <p class="textonegro"><b>2.3. Identification of differentially expressed genes</b><br>
  The tutorial will revise some simple methods for finding differentially expressed genes or genes related to a given parameter or to survival data. The interpretation of the adjusted p-values will be discussed. GEPAS, through the Pomelo tool, implements several basic statistical tests (t-test, one-way ANOVA, linear regression, Cox survival regression, and contingency tables) to identify differentially expressed genes; we provide both unadjusted p-values and adjusted p-values, using control of the Family Wise Error Rate and the False Discovery Rate (see reviews in Dudoit et al., Statistica Sinica, 12:111-139 and Technical Report #110, Division of Biostatistics, UC Berkeley). </p>
              <p class="textonegro"><b>2.4. Discrimination methods for classification of samples from microarray data.</b><br>
  The tutorial will cover several methods that have repeatedly shown an excellent performance in terms of predicting error rates. These methods are K-Nearest Neighbor, Diagonal Linear Discriminant Analysis, Support Vector Machines, and Random Forests (Dudoit et al., 2002 JAMA 97:77-87; Furey et al., 2000 Bioinformatics 2000 16: 906-914; Breiman, 2001 Machine Learning 45:5-32). For the first three, we include methods that allow cross-validating the complete process, including gene selection, to avoid selection bias (Ambroise &amp; McLachlan, 2002 PNAS. 99: 6562-6566); for random forests, we include cross-validation of &quot;important genes&quot;, as recently proposed in Svetnik et al. (2003, submitted). GEPAS will allow access, through its TNASAS tool (Herrero et al., NAR 2004), to these methods. The tools in TNASAS not only return a predictive model, but provide an unbiased assessment of its predictive performance.</p>
              <p class="textonegro"><b>2.5. Functional annotation of the results</b><br>
  Information extraction and textmining techniques have been applied to the analysis of gene expression data (Jenssen, et al., 2000 Nat. Genet. 28: 21-28). Nevertheless, textmining methodologies still have many drawbacks (Blaschke, et al. 2002 Briefings in Bioinformatics 3:154-165) and few users have access to software implementations. An alternative to extracting information from scientific text sources is to use ontologies. In their most simple representation, ontologies provide a structured description of biological knowledge that is extremely convenient for computational management. Gene Ontology (GO; Ashburner, et al. 2000 Nat. Genet. 25:25-29.), which implements information for biochemical function, biological processes and cellular components for a number of different organisms, is widely used. These ontologies can be used as a quick and efficient functional annotation tool for the identification and interpretation of clusters of co-expressing genes studied. In the tutorial different methods for automatic functional annotation will be described in depth. The practical session will include the use the FatiGO tool (Al-Shahrour et al., 2004 Bioinformatics, 20: 578-580; http://www.fatigo.org), which allows the exploration of the biological meaning of groups of genes defined by any of the previously mentioned methodologies, by extracting the list of over and underrepresented GO terms. Significance is provided both with adjusted p-values, using control of the Family Wise Error Rate and the False Discovery Rate. The Babelomics suite (Al-Shahrour et al., 2005, NAR, in press: http://www.babelomics.org) contains different extensions of FatiGO, which includes information from Interpro motifs, pathways (KEGG) and keywords from SwissProt, TFBS, etc.</p>
              <p class="textonegro">&nbsp;</p>
              <p align="justify">&nbsp;</p></td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a> </p>
        <p>&nbsp;</p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
