<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Tutorials</font></strong></p>
        <a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a>
<table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr>
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Wednesday<br/>
      September 28, 2005</strong></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><span class="textomorado">14:15/18:00 - Room 9 Bis (1st Floor)<br>
              <strong><a name="T4" id="T4"></a></strong></span><span><span class="textomoradoGrandeB"><b>T4: Computational proteomics</b></span><b><br>
              </b></span> <span class="textomoradoGrande"> Prof. Colinge Jacques</span>
              <p class="textonegro">Proteomics has become an important approach to analyze biological samples and it extensively uses mass spectrometry to identify and characterize proteins. This tutorial will introduce the audience to the central problem of searching mass spectrometry data against a database of proteins. This presentation should stimulate the interest of bioinformatics researchers in other fields and provide a concise though accessible introduction to life scientists. The last part of the tutorial will rapidly cover other important problems in mass spectrometry data analysis such as peptide de novo sequencing, eukaryote genome search and protein quantification and characterization.</p>
              <p><span class="textomorado">Synopsis:</span><span class="textomorado"></span></p>
              <p class="textonegro"><b>Part 1 (20 min):</b> introduce the main problems and technologies in proteomics.<br>
                  <b>Part 2 (40 min):</b> Peptide mass fingerprinting is presented as a first &#150; simple &#150; type of data to analyze and also as a mean to introduce the basic concepts of database searching with mass spectrometry (MS) data.<br>
                  <b>Part 3 (20 min):</b> Raw spectrum processing is rapidly covered to actually link the somewhat abstract mass lists used for searching databases with the signal generated by the MS instruments.<br>
                  <b>Part 4 (60 min):</b> Tandem mass spectrometry (MS/MS) technologies are introduced and MS/MS database searching is discussed in details.<br>
                  <b>Part5 (40 min):</b> Although the main focus of the tutorial is on database searching, other current and important topics in MS-related bioinformatics are presented such as peptide de novo sequencing, protein quantification and differential proteomics. The material introduced for database searching allows the audience to realize the importance of these topics and the challenges they offer for bioinformatics.</p>
              <p class="textonegro"><b>Part 1: Introduction to proteomics</b><br>
              </p>
              <p class="textonegro">We start by introducing the main problems in proteomics: identify proteins in a sample, characterize modified proteins, compare samples and quantify proteins.</p>
              <p class="textonegro">We point out the difficulty of excessively complex samples with high dynamic range of protein concentrations. Then we introduce the main technologies for reducing initial sample complexity: liquid chromatography and gels.</p>
              <p class="textonegro">We then rapidly introduce the concept of mass spectrometry as a technology to measure masses that constitute specific data sets, which may be used for obtaining information about the proteins.</p>
              <p class="textonegro">Top-down versus bottom-up proteomics: explain the two approaches, respective advantages for specific problems. In particular protein characterization (top-down) and sample analysis (bottom-up) are used as typical problems to illustrate the two opposite approaches.</p>
              <p class="textonegro">For the rest of the tutorial, bottom-up proteomics with enzymatic digestion is the standard case we consider. Explain classical strategies to analyze samples: protein separation and late digestion (gel or LC) versus shot-gun peptide sequencing and early digestion.</p>
              <p class="textonegro">Put the presented material of this first part in perspective from the point of view of data processing. Make obvious the central role of the protein/peptide identification problem.</p>
              <p class="textonegro"><b><br>
  Part 2: Peptide mass fingerprinting and MALDI instruments<br>
              </b> </p>
              <p class="textonegro">On the basis of the general context presented in Part 1, we introduce and detail a first proteomics method: peptide mass fingerprinting (PMF).</p>
              <p class="textonegro">Present the combination 2D gels + MALDI-TOF mass spectrometry. Explain that enzymatic, typically tryptic, peptide masses certainly constitute a data set specific to the protein.</p>
              <p class="textonegro">Show a first example with a spectrum and a database search result. Mention the problem of peak detection in the spectrum, which will be detailed later in Part 3.</p>
              <p class="textonegro">Explain a basic algorithm for searching PMF data against a database of protein sequences. Introduce the notion of scoring function aimed at measuring the correlation between theoretical and experimental spectra, the theoretical spectrum being the list of peptide masses computed from the protein sequences. First scoring function: &#147;shared peak count&#148;, i.e. number of masses in common between theoretical and experimental spectra, given a mass error tolerance.</p>
              <p class="textonegro">Show examples of database searches with shared peak count. Comment the results: impact of mass precision, database size, and scoring function; importance of estimating p-values.</p>
              <p class="textonegro">Before we examine some important scoring functions, we describe the problem of database searching with more details. Namely, we explain the extra difficulties caused by missed cleavages, variable modifications, and imperfect instrument calibration. For each problem we explain the classical solution: missed cleavages -&gt; introduce them in the theoretical spectrum, variable modifications -&gt; generate all possible peptide masses, calibration -&gt; re-calibrate by using the theoretical masses.</p>
              <p class="textonegro">We conclude Part 2 by detailing MOWSE, ProFound, MSA and OLAV-PMF scoring functions.</p>
              <p class="textonegro">MOWSE is used by the commercial package MASCOT and it is based on a simple model of &#147;typical&#148; peptide masses probabilities, which is computed by averaging theoretical spectra of a large population of proteins.</p>
              <p class="textonegro">ProFound is a commercial package that score spectrum matches by using a Bayesian combinatorial model of &#147;how many masses are expected to be found in the experimental spectrum&#148;.</p>
              <p class="textonegro">MSA is a successful heuristic model that explicitly uses TOF mass spectrometer properties to distinguish between spurious and correct PMF identification. It has been developed at Max Plank Institute, Berlin.</p>
              <p class="textonegro">OLAV-PMF is a scoring function that is based on a likelihood ratio of two statistical models (correct and random identification) that try to make use of several properties of correct identifications such as protein coverage, amino acid composition, and peptide modifications.</p>
              <p class="textonegro">State the importance of statistical modeling.</p>
              <p class="textonegro"><br>
                <b>Part 3: Peak detection</b><br>
              </p>
              <p class="textonegro">In this third part we rapidly detail the problem of peak detection as illustrated by MALDI-TOF spectra used for PMF.</p>
              <p class="textonegro">Examples of spectra to present electronic and chemical noises, baseline, isotope peaks, and overlapping peptide signals.</p>
              <p class="textonegro">Explain the main strategies for performing peak detection and present a very simple one as well as an advanced one. Give examples showing the practical difficulty to write very good programs.</p>
              <p class="textonegro"><br>
                <b>Part 4: Complex samples and tandem mass spectrometry</b><br>
              </p>
              <p class="textonegro">Database sizes and sample complexity may limit the usage of PMF. The solution is to get more information on the peptides such as we can identify them directly. Tandem mass spectrometry is a manner to obtain such additional information via fragmentation. Explain the principle of fragmentation in a very general way.</p>
              <p class="textonegro">Present a schematic abstract mass spectrometer with ion source, fragmentation cell and mass analyzer. Present different technologies (collision induced fragmentation, post-/in-source decay).</p>
              <p class="textonegro">Explain the benefit of having more information on peptides. On consequence is that peptide separation before mass spectrometry analysis is possible: LC-MS is introduced with electro-spray ionization (ESI) and LC-MALDI.</p>
              <p class="textonegro">Explain fragmentation more precisely and introduce the various types of ions. Each type of instrument and parameters setting yield certain ions only. Ions actually generated also depend on the properties of the peptide. These observations together are the basis for designing MS/MS scoring functions. Modified peptides yield modified MS/MS spectra.</p>
              <p class="textonegro">Before we examine important scoring functions, we list typical experimental conditions and the type of data generated.</p>
              <p class="textonegro">Scoring functions: MASCOT, SEQUEST, post-processing of SEQUEST, OMSSA, OLAV.</p>
              <p class="textonegro">MASCOT, a commercial program, uses an adapted MOWSE score to score peptide identifications. MASCOT includes spectrum pre-processing to determine noise level.</p>
              <p class="textonegro">SEQUEST, another commercial program, compares the experimental and theoretical spectrum by first generating an artificial spectrum from the theoretical mass lists and then by comparing the two spectra directly. A second step involves the computation of a heuristic score for the top-ranked peptides identified during the first step.</p>
              <p class="textonegro">Many efforts have been made to design filters for SEQUEST results in order to eliminate false positive identifications. The standard design of such post-processing tools will be presented.</p>
              <p class="textonegro">OLAV is a scoring function that models various properties of correct versus random matches in the framework of likelihood ratios and na&iuml;ve Bayesian classification. It is implemented in the commercial program Phenyx.</p>
              <p class="textonegro">Reiterate the importance of statistical modeling and p-value computations.</p>
              <p class="textonegro"><br>
                <b>Part 5: Other problems, other approaches</b><br>
              </p>
              <p class="textonegro">We list several problems which are of great importance in proteomics today. For each of them, we rapidly describe the analytical problem, the type of data and bioinformatics challenges. The notions introduced for database searching and proteomics in general allow the audience to easily understand the problems, their importance and the associated bioinformatics.</p>
              <p class="textonegro">The problems we will present are:</p>
              <ul>
                <li class="textonegro">Genome not sequenced or limited database</li>
                <li class="textonegro">Peptide de novo sequencing: directly infer the peptide sequence from the tandem mass spectrum (no database search) and use the predicted sequence per se or for homology searches in closely related organisms.</li>
                <li class="textonegro">Homology searches: search against closely related organisms sequences.</li>
              </ul>
              <p class="textonegro">-Genome annotation via proteomics<br>
                <br>
  Gene predictions and annotation by homology have their limits and there is a demonstrated potential to gain extra information about genes and their structures by using experimental proteomics data. We will explain how this can be done on a large-scale and give examples to actually demonstrate the potential of the approach.</p>
              <p class="textonegro">-Protein characterization<br>
              </p>
              <ul>
                <li class="textonegro">Top-down analyses: find modification sites, typically phosphorylations, by analyzing the intact protein.</li>
                <li class="textonegro">Identification of modified peptides. Database search programs can be used to find modified peptides in an iterative way.</li>
                <li class="textonegro">Peptide de novo sequencing of modified peptides. Besides its ability to identify new peptides, peptide de novo sequencing can also find modified peptides.</li>
              </ul>
              <p class="textonegro">-Quantification and differential proteomics<br>
              </p>
              <ul>
                <li class="textonegro"> 2D Gels and image analysis. Mention challenging problems in 2D gel images processing and comparisons. </li>
                <li class="textonegro">Semi-quantitative methods: identification scores or the number of identified peptides can be used to infer semi-quantitative and relative information about protein concentration.</li>
                <li class="textonegro">Stable isotope labeling methods allow to obtain precise quantitative information on protein concentrations and thus to compare samples.</li>
                <li class="textonegro">Important individual variations calls for adapted differential expression analysis methods.</li>
                <li class="textonegro">Profiling: direct mass spectrometry measurements of samples allow detecting reliable differential masses between sample classes.</li>
              </ul>
              <p class="textonegro">-Protein structure prediction<br>
                <br>
  Mass spectrometry may be used to assist protein structure prediction by detecting di-sulfur bonds for instance.</p>
              <p class="textonegro"><br>
  -Proteomics data integration<br>
  <br>
  From laboratory data to identification results via quantification: Due to possibly complex sample preparation it is especially important in proteomics to be able to make use of intermediary laboratory information such as chromatography profiles. Also, since proteins are elaborated products of a cascade of processes in the cell, it is often necessary to relate proteomics analyses results to known facts concerning gene structure (alternative splice forms), enzymes (degradation products) and literature.</p>
              <p align="justify" class="textonegro">&nbsp;</p>
              </td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a> </p>
        <p>&nbsp;</p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
