<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/main_menu.dwt" --><!-- DW6 -->
<head>
<!-- #BeginEditable "doctitle" -->
<title>ECCB</title>
<!-- #EndEditable -->
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link href="eccb.css" rel="stylesheet" type="text/css">
<link href="scroll.css" rel="stylesheet" type="text/css">
</head>
<script language="JavaScript" type="text/javascript">
//Begin CCS script
//Include Common JSFunctions @1-7AE8D3F1
</script>
<script language="JavaScript" type="text/javascript" charset="iso-8859-1" src="/DatePicker.js"></script>
<script language="JavaScript" type="text/javascript">
//End Include Common JSFunctions
//Date Picker Object Definitions @1-7EF4584D
var registration_DatePicker1 = new Object(); 
registration_DatePicker1.format           = "mm/dd/yyyy";
registration_DatePicker1.style            = "Themes/Somatic/Style.css";
registration_DatePicker1.relativePathPart = "";
var registration_DatePicker2 = new Object(); 
registration_DatePicker2.format           = "mm/dd/yyyy";
registration_DatePicker2.style            = "Themes/Somatic/Style.css";
registration_DatePicker2.relativePathPart = "";
//End Date Picker Object Definitions
//End CCS script
</script>
<body leftmargin="0" topmargin="0" bgcolor="A1A1A1">
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
        <tr> 
          
    <td width="33%" background="images/tira_izq.gif" bgcolor="650C53" height="131"></td>       
    <td width="750" height="131" bgcolor="650C53"><img src="images/cabecera_m.jpg" width="750" height="131" usemap="#Map" border="0"></td>
    <td width="33%" background="images/tira_dcha.gif" bgcolor="650C53" height="131"></td>
        </tr>
        <tr> 
          <td bgcolor="#527475"></td>
          <td width="750" bgcolor="#527475"> 
<!--** START NAVSTUDIO MENU TAGS (76067) **-->
<div id="vqp_about" style="position:absolute;visibility:hidden;"></div>
<script language="JavaScript" vqptag="doc_level_settings" vqp_datafile0="menu.js" vqp_uid0=76067>cdd__codebase = "http://bioinfo.cnio.es/eccb05/navigator/";cdd__codebase76067 = "http://bioinfo.cnio.es/eccb05/navigator/";</script>
<script language="JavaScript" vqptag="datafile" src="navigator/menu.js"></script><script vqptag="placement" vqp_menuid="76067" language="JavaScript">create_menu(76067)</script>
<!--** END NAVSTUDIO MENU TAGS **-->
          </td>
          <td bgcolor="#527475"></td>
        </tr>
</table>
<map name="Map"> 
  <area shape="rect" coords="427,109,467,125" href="index.htm" alt="HOME" title="HOME">
  <area shape="rect" coords="476,110,565,124" href="registration.html" alt="REGISTRATION" title="REGISTRATION">
  <area shape="rect" coords="575,112,614,124" href="news.htm" alt="NEWS" title="NEWS">
  <area shape="rect" coords="620,110,679,125" href="contact.htm" alt="CONTACT" title="CONTACT">
  <area shape="rect" coords="685,111,743,124" href="sitemap.htm" alt="SITEMAP" title="SITEMAP">
</map>
<table width="750" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#DBDBDB">
  <tr>
    <td valign="top" class="negro">
	<!-- #BeginEditable "main" -->
      <div align="center">
        <p>&nbsp;</p>
        <p><strong><font size="3" color="5d5d5d">Tutorials</font></strong></p>
        <a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a>
        <table width="90%" border="1" align="center" cellpadding="1" cellspacing="1" bordercolor="#660066" _moz_resizing="true">
          <tr>
            <td valign="top" 65535 bgcolor="#666666"><p align="center" class="textoblancoGrandeB"><strong>Wednesday<br/>
      September 28, 2005</strong></p></td>
          </tr>
          <tr valign="top">
            <td 65535 bgcolor="#FEFFDF"><p align="justify"> <span class="textomorado">9:00/13:15 - Room 9 (2nd Floor)</span><br>
                <span><strong><a name="T1" id="T1"></a></strong><span class="textomoradoGrandeB"><b>T1: Mining Structured Data in Bioinformatics</b></span><b><br>
                </b></span> <span class="textomoradoGrande"> Prof. Stefan Kramer</span></p>
              <p class="textonegro">The topic of the tutorial is mining in structured data. This is particularly relevant for data mining applications in bioinformatics, since the majority of biological data is not kept in databases consisting of a single, flat table. Instead, we are frequently dealing with databases of structured and linked objects. In other words, the &#147;objects&#148; in bioinformatics databases often have a rich internal structure and are connected by some relation. (Consider, for instance, databases of proteins, small molecules, metabolic and regulatory networks, text databases, etc.) The tutorial will give an overview of data mining techniques for sequences, trees, graphs and relational databases. We will present techniques for both descriptive and predictive data mining in this context. In descriptive mining, we are looking for local patterns to characterize the data. In predictive mining, we are looking for models that can be used to make predictions for new, unseen cases. Along the two dimensions (types of data and predictive/descriptive), the tutorial is organized as follows: the first two parts of the tutorial are devoted to descriptive mining in databases of itemsets, strings and sequences, trees, graphs and relational databases. The third part of the tutorial deals with predictive mining based on propositionalization (i.e., feature construction using patterns), instance-based learning and kernel methods for graph and relational databases.</p>
              <p><span class="textomorado">Synopsis:</span><span class="textomorado"></span></p>
              <p class="textonegro">The tutorial will give an overview of data mining techniques for sequences, trees, graphs and relational databases. Tree mining is of interest for analyzing text or linguistic data (XML documents, parse trees, etc.), RNA data and phylogenetic trees. Graph mining is useful for finding network motifs in metabolic and regulatory networks, and substructural fragments in small molecules. Mining in relational databases is important for analyzing data from heterogeneous sources in the life sciences. </p>
              <p class="textonegro">In the <b>first part</b> of the tutorial, I will explain, in ascending order of complexity, mining in itemset (transaction) data, in string or sequence data, and in tree data. Typical strategies for searching the space of patterns will be presented for the simple case of itemsets (transaction data): breadth-first (APriori and levelwise search), depth-first, and combined depth-first/breadth-first. Subsequently, the basic ideas will be transferred to mining tree and graph data. Another focus will be on the use of index structures for organizing both the data and the patterns, enabling fast access to information about them. String mining will again be explained in ascending order of complexity: first, we will explain mining interesting substrings (without wildcards) in string or sequence databases, then will extend the approach to substrings containing wildcards. Finally, we will explain serial episodes, i.e., substrings with arbitrary gaps between the characters. In tree mining, different classes of tree patterns can be chosen depending on the application in mind (from embedded to free trees). In the presentation, we will focus on the basic, underlying ideas more than on minor technical details.</p>
              <p class="textonegro">In the <b>second part</b>, we will continue with mining general graph data. Several of the previously developed concepts can be applied to graph mining as well. After a brief introduction to the topic, we will sketch two of the most important approaches so far: AGM (performing breadth-first search), and gSpan (performing depth-first search). Subsequently, we will give a gentle introduction to mining relational databases. For convenience, we will restrict ourselves to simple conjunctive datalog queries. The Warmr algorithm for finding frequently successful datalog queries will be explained in terms of inputs and outputs.</p>
              <p class="textonegro">In the <b>third part</b> of the tutorial, we will give an overview of predictive mining in structured data. First, we will explain how patterns found in descriptive mining can be used to reformulate data for predictive purposes. Second, instance-based learning techniques for structured data will be presented. Last, but not least, a survey of kernel methods for structured data, in particular graphs, is given.</p>
              <p class="textonegro">All theoretical concepts and algorithms will be illustrated by running examples from data mining applications in bioinformatics. A comprehensive reference list will be included, with links to online resources and literature where available.</p>
              <p class="textonegro"><b>Part I</b>: Descriptive Mining:<br>
  From Itemsets to Tree Mining<br>
  (90 minutes)</p>
              <p class="textonegro">- Transaction data and frequent itemsets<br>
  - APriori and levelwise search<br>
  - String mining<br>
  - Introduction to string mining<br>
  - Version space trees<br>
  - Mining with wildcards<br>
  - Mining serial episodes<br>
  - Tree mining<br>
  - Introduction to tree mining<br>
  - Mining embedded subtrees (TreeMiner)<br>
  - Mining induced subtrees (FREQT)<br>
  - Mining unordered subtrees (Unot)<br>
  - Mining free trees (FreeTreeMiner)</p>
              <p class="textonegro">Coffee Break (30 minutes)</p>
              <p class="textonegro"><b>Part II</b>: Descriptive Mining:<br>
  Graph Mining and Mining Relational Databases<br>
  (60 minutes)</p>
              <p class="textonegro">- Graph mining<br>
  - Introduction to graph mining<br>
  - AGM<br>
  - gSpan<br>
  - Mining relational databases<br>
  - SQL, datalog, conjunctive queries, query containment, theta-subsumption<br>
  - Warmr algorithm (inputs and outputs)</p>
              <p class="textonegro"><b>Part III</b>: Predictive Mining:<br>
  Propositionalization, Instance-Based Learning and<br>
  Kernel Methods<br>
  (60 minutes)</p>
              <p class="textonegro">- Propositionalization: using frequent patterns for predictive mining<br>
  - Instance-based learning: distance measures for graphs and relational databases<br>
  - Kernel methods: graph kernels</p>              <p align="justify" class="textonegro">&nbsp;</p>
              </td>
          </tr>
          <tbody>
          </tbody>
        </table>
        <p><a href="tutorialSchedule.htm" class="textomoradoGrande">Back to Tutorial Index</a> </p>
        <p>&nbsp;</p>
      </div>
      <!-- #EndEditable -->
</td>
  </tr>
</table>
<table width="100%" border="0" align="center" cellspacing="0" cellpadding="0">
  <tr> 
    <td width="33%" background="images/tira_izq_abajo.gif" bgcolor="650C53" height="55"></td>
    <td height="55" bgcolor="650C53"><img src="images/tira_abajo_home_c1.gif" width="300" height="55" usemap="#MapMap" border="0"><a href="javascript:if(confirm('http://www.inab.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.inab.org/'"><img src="images/tira_abajo_home_c2.gif" width="95" height="55" border="0" alt="INB"></a><a href="javascript:if(confirm('http://www.eccb05.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.eccb05.org'"><img src="images/tira_abajo_home_c3.gif" width="56" height="55" border="0" alt="ECCB"></a><a href="javascript:if(confirm('http://www.sebiot.org  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sebiot.org'"><img src="images/tira_abajo_home_c4.gif" width="55" height="55" border="0" alt="SEBIOT"></a><a href="javascript:if(confirm('http://www.biosapiens.info  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.biosapiens.info'"><img src="images/tira_abajo_home_c5.gif" width="29" height="55" alt="BIOSAPIENS" border="0"></a><a href="javascript:if(confirm('http://www.iscb.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.iscb.org/'"><img src="images/tira_abajo_home_c6.gif" width="58" height="55" border="0" alt="ISBC"></a><a href="javascript:if(confirm('http://www.gen-es.org/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.gen-es.org/'"><img src="images/tira_abajo_home_c7.gif" width="157" height="55" border="0" alt="GENOMA ESPA&Ntilde;A"></a></td>
    <td width="33%" background="images/tira_dcha_abajo.gif" bgcolor="650C53" height="55"><img src="images/tira_dcha_abajo_cc.gif" width="1" height="47"></td>
  </tr>
</table>
<div align="center" class="textomoradoCopy">Developed by <a href="javascript:if(confirm('http://www.softactiva.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.softactiva.com'" target="_blank">SoftActiva</a></div>
</body>
<!-- #EndTemplate --></html>
