<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/highlights.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:25:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent">




<br><br><div class=smhead1><strong>Highlights Track Presentation Schedule</strong></div> <div class=style1><p><strong>Highlights Track:</strong> <strong><a name="HL01" >HL01</a></strong><br><strong><em>The Plurality of Prognostic Gene Signatures for Cancer</em></strong><br> Monday, June 29 - 10:45 a.m. - 11:10 a.m.<br>Room: T1<br />Presenting author: Paul Boutros, Ontario Institute for Cancer Research, Canada<br>Presentation Overview: <br>Many diseases exhibit highly variable prognosis, with some individuals responding to therapy and others not. Despite this variability, patients are often treated identically. One major goal of modern medicine is to identify biomarkers that predict the optimal therapy for each patient. Although diverse computational techniques have been developed, existing methods have two key weaknesses: they overfit training datasets and they yield non-overlapping biomarkers that complicate clinical validation. We addressed these weaknesses in the context of lung cancer. First, we developed a non-linear biomarker-identification technique by coupling unsupervised machine-learning to gradient-descent optimization. This algorithm identified a six-gene biomarker that was validated on eight independent datasets comprising 589 patients. Second, we devised a technique for estimating the null distribution of biomarkers that can be used to obtain an unbiased ranking of biomarkers; our six-gene biomarker is in the 99.98 percentile. More importantly, this analysis reveals that over 500,000 unique lung cancer biomarkers exist. Thus, our work resolves two key questions in the field of biomarker identification: the over-fitting and non-overlapping biomarker problems. Although we focus on lung cancer, our techniques are directly applicable to other diseases. We are currently applying these techniques to other diseases. Preliminary results from breast cancer and schizophrenia will be presented.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL01: Paul Boutros - The Plurality of Prognostic Gene Signatures for Cancer </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ac11e2ce-69fb-4b83-b657-49ec222d77ff" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ac11e2ce-69fb-4b83-b657-49ec222d77ff" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ac11e2ce-69fb-4b83-b657-49ec222d77ff" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">lung cancer is highly heterogenous in classifications and outcomes -- how to separate those patients that would and would not benefit from chemotherapy after surgery - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">previous studies based on gene expression analysis have not replicated.  why?  weak statistical methods - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">method: "modified steepest descent" - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">first assess genes individually, then take best gene and evaluate best pair, take top pair and screen for top triple, etc. - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">identified 6 -gene marker set -- stx1a, hif1a, cct3, mafk, rnf5, hla_dpb1 - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">classier in training data, survival difference 1E-5 (not surprising) - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">8 separate studies: p = 0.02 (409 patients - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">evaulation: take 10 million random six-gene sets, better than 99.9999% of backgorund, drops to ~90% on other training data, merged 99.98% - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">but, what are the ~450,000 random sets that are better?  Could this explain non-overlap of previous microarray studies? - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">evaulate gene based on marker pluraily -- how many good predictors does a gene occur in? - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/ac11e2ce-69fb-4b83-b657-49ec222d77ff">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL02" >HL02</a></strong><br><strong><em>Modeling Ecological and Genetic Diversity in Bacteria</em></strong><br> Monday, June 29 - 10:45 a.m. - 11:10 a.m.<br>Room: T2<br />Presenting author: Eric Alm, MIT, United States<br>Presentation Overview: <br>Defining species boundaries is a major challenge for microbiologists. This is because bacteria are difficult to study in their natural habitat, and they can exchange genes across species boundaries (however these may be defined). One promising direction is to combine environmental DNA sequence data together with ecological metadata to infer: (i) populations adapted to a habitat/niche; (ii) the structure of the habitat underlying a series of ecological measurements; and (iii) the history of ecological adaptation within a lineage. We describe a quantitative model (AdaptML) that infers the evolutionary
history of ecological differentiation for a collection of ocean bacteria, revealing populations specific for different seasons and life-styles, and in more recent work we have used the predictions of the AdaptML algorithm to target entire populations of bacteria for complete genome sequencing. This population genomic approach is helping to elucidate the underlying microevolutionary processes shaping bacterial species.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL02: Eric Alm - Modeling Ecological and Genetic Diversity in Bacteria </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/6b216e09-8621-4cbc-b992-c056fa9049cd" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/6b216e09-8621-4cbc-b992-c056fa9049cd" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/6b216e09-8621-4cbc-b992-c056fa9049cd" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">So far this is similar to <a style="text-decoration:none" href="http://ff.im/4x4Vg">http://ff.im/4x4Vg</a> of the BioPathways SIG yesterday - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">this lets us study sympatric speciation in the ocean - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">inspired by Mandel, ..., Ruby, "A single regulatory gene is sufficient to alter bacterial host range": what is the genetic basis for adaptation to the niches we found (zooplankton-associated vs small-particle associated)? - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to do ecologically-driven population genomics - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">wanted to sequence all genomes in each of the subtrees - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">asked for 20 genomes in grant application, was met with "?!" - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">sat next to Eric Lander at a dinner, who said "20 genomes? why not 100?", so got funding from Broad Institute - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to model changes within a population of genomes; gene flow and recombination within the population - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">wanted to find recombination breakpoints - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">dimorphic sites, with one nucleotide changing: supports splits, leading to tree with single mutation - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/6b216e09-8621-4cbc-b992-c056fa9049cd">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL03" >HL03</a></strong><br><strong><em>Whole genome analysis of mtDNA natural evolution in human and in cancer</em></strong><br> Monday, June 29 - 11:15 a.m. - 11:40 a.m.<br>Room: T1<br />Presenting author: Eitan Rubin, Ben Gurion University, Israel<br>Presentation Overview: <br>mtDNA is an exceptional model genome for bioinformatics. It is the only genome sequenced in 2400 different individuals. In this work we demonstrate how comparative whole-genome analysis reveals strong patterns the recur in human evolution and in cancer.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL03: Eitan Rubin - Whole genome analysis of mtDNA natural evolution in human and in cancer </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/10bf6ce4-9843-4542-b880-563df25e38ef" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/10bf6ce4-9843-4542-b880-563df25e38ef" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/10bf6ce4-9843-4542-b880-563df25e38ef" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/allyson" target="_top" style="text-decoration:none">Allyson Lister</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">in advance of 1000 genomes, can we use mitochondrial dna sequencing?  advantages: small size (16kb), and maternally inherited (allows human phylogeny) - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">mitochondria have known role in cancer, but unclear if the mito genome (and mutations therein) are relevant - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">method: compare tumor and normal mtDNA -- patient-matched.  --> mutational landscape --> eliminate SNPs in normals --> find high recurrence.  BUT, nothing found... - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">method 2: from mutational landscape, look for co-mutations: two patients with multiple mutations observed - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">one "flip-flop" observed -- mutation in tumor in one patient and in normal in another -- bi-stability? - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">compare evolution in normal population as reference; create trees via nieghbor joining, create inner nodes, find deepest branch with mutation; conclusion: cancer uses snps that arise early in human evolution - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">relates to african vs european haplotypes? - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">conclusion: mtDNA is under selection in cancer; strategy of finding multiple SNPs in subset of tumors; functional prediction of 25 mtDNA snps - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">philosophilcal question -- are tumors different species?  unicellular parasites? - <a style="text-decoration:none" href="http://friendfeed.com/asu" target="_top">Andrew Su</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">A tumor is like a parasite branching off as a separate species from the human - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/10bf6ce4-9843-4542-b880-563df25e38ef">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL04" >HL04</a></strong><br><strong><em>The role of the RNA folding free energy in the evolution of the influenza virus</em></strong><br> Monday, June 29 - 11:15 a.m. - 11:40 a.m.<br>Room: T2<br />Presenting author: Panayiotis Benos, University of Pittsburgh, United States<br>Presentation Overview: <br>The influenza A virus genome consists of eight single stranded RNA segments of negative polarity. Current efforts to understand viral host-specificity have largely focused on the amino acid differences between avian and human isolates. The results presented here demonstrate that the RNA folding free energy (FFE) of the influenza polymerase genes plays a key role in the evolution and host specificity of the virus. In particular, we found that the distribution of the FFEs is significantly different between human and avian isolated strains, with human isolates having generally higher FFEs (less stringent RNA structures). When avian polymerase genes are introduced in the human population, their FFEs shift toward higher values over the years. Infection experiments in mammalian cells growing at different temperatures show that human isolated viruses cannot propagate efficiently at higher temperatures and more recent results (not in the paper) show the opposite: i.e., avian isolates cannot propagate efficiently in lower temperatures.  Taken together, our data suggest for the first time that RNA structure stability is important for the emergence and host shift of influenza A virus.  The fact that cellsÕ temperature affects virus propagation in mammalian cells has important consequences for the prevention and therapeutic strategies.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL04: Panayiotis Benos - The role of the RNA folding free energy in the evolution of the influenza virus </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/9da09ed6-e879-4474-a709-7a5f90bfeff6" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9da09ed6-e879-4474-a709-7a5f90bfeff6" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9da09ed6-e879-4474-a709-7a5f90bfeff6" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL05" >HL05</a></strong><br><strong><em>Proteomics first approach for discovering sub-network targets in cancer</em></strong><br> Monday, June 29 - 11:45 a.m. - 12:10 p.m.<br>Room: T1<br />Presenting author: Rod Nibbe, Case Western Reserve University, United States<br>Presentation Overview: <br>Using a proteomics first approach we identified many targets significant for late stage human colon cancer. These targets were used to seed a search in a well-annotated PPI for subnetworks possibly significant for the late stage phenotype. We devised  a method to score certain of the subnetworks found using an information theoretic (mutual information) approach based on a complement of transcription data (microarray) as a surrogate for subnetwork activity. The subnetworks were pruned to leave significant targets, and extended one hop to infer functional relevance and inform follow-on experiments. The significant targets in one subnetwork were validated by label-free mass spectrometry or western blot, and found to be coordinately regulated at the level of protein and mRNA. Overall, the work outlines a novel quantitative approach for extending the results of proteomic profiling for finding disease discriminators at the level of protein subnetworks (and thus function), and drives target selection for in vitro/in vivo verification.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL05: Rod Nibbe - Proteomics first approach for discovering sub-network targets in cancer </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/66f0622c-08cb-4138-a741-ce5c1858a4af" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/66f0622c-08cb-4138-a741-ce5c1858a4af" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/66f0622c-08cb-4138-a741-ce5c1858a4af" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Move beyond single targets, an aim to identify concerted effects - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Understanding the patophysiology of a late-stage colon cancer phenotype - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">proteomics data _ legacy data + transcriptomics --> quantitative sub-network specific to disease phenotype - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Top-down proteomics approach: paired normal / tumour biopsies from 12 patients - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Differential image analysis to identify significantly changing proteins, MS/MS of excised spots results in lists of significant proteins associated with the phenotype - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">the paper: <a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/19098285">http://www.ncbi.nlm.nih.gov/pubmed/19098285</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">map cancer-significant genes onto PPI db, look for sub-networks enriched in cancer proteins - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">find 4 significant subnetworks, extend by one hop and look for evidence in lit - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Pruned to core network members, expanded by one step for functional inference - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Looks like a fairly standard MetaCore Meta-analysis to me so far. Not quite sure how they scored the subnetworks with the array data, need to check the paper - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/66f0622c-08cb-4138-a741-ce5c1858a4af">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL06" >HL06</a></strong><br><strong><em>Bayesian Inference of Selection Histories in Six Mammalian Genomes</em></strong><br> Monday, June 29 - 11:45 a.m. - 12:10 p.m.<br>Room: T2<br />Presenting author: Tomas Vinar, Comenius University in Bratislava, Slovak Republic<br>Presentation Overview: <br>Genome-wide scans for positively selected genes (PSGs) in mammals haveprovided insight into the dynamics of genome evolution, the geneticbasis of differences between species, and the functions of individualgenes. Here we present the most comprehensive examination of mammalianPSGs to date, using the six high-coverage genome assemblies nowavailable for eutherian mammals. The increased phylogenetic depth ofthis dataset results in substantially improved statistical power, andpermits several new lineage- and clade-specific tests to beapplied. Of ~16,500 human genes with high-confidence orthologs in atleast two other species, 400 genes showed significant evidence ofpositive selection (FDR<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL06: Tomas Vinar - Bayesian Inference of Selection Histories in Six Mammalian Genomes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/0e317dbf-9b12-4052-95ee-b7c8e5bbd95a" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0e317dbf-9b12-4052-95ee-b7c8e5bbd95a" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0e317dbf-9b12-4052-95ee-b7c8e5bbd95a" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The ratio of nonsynonymous to synonymous mutations may identify functional selection. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Detect positive selection by comparing orthologous genes in six mammalian genomes: human, chimp, macaque, mouse, rat and dog. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">One would like to identify which branches of the phylogenetic tree are under positive selection for a given gene. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Algorithm summary:
1) take genes under positive selection under likelihood-ratio tests (544);
2) pre-compute the likelihood of every gene under all 511 possible histories of the tree (using PAML);
3) compute many iterations of Gibbs sampling (very fast);
4) compute history posterior probabilities from the samples. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Most genes (from the 544 originally selected) appear to have switched between evolutionary models (positive selection and nonselection): 95% at least once, 53% at least twice. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL07" >HL07</a></strong><br><strong><em>Computational approach to model peptide antigenicity</em></strong><br> Monday, June 29 - 12:15 p.m. - 12:40 p.m.<br>Room: T1<br />Presenting author: Carlos Camacho, University Of Pittsburgh, United States<br>Presentation Overview: <br>It is well known that relatively unstable peptides bearing only partial structural resemblance to native protein can trigger antibodies recognizing higher order structures found in native protein.  Based on sound thermodynamic principles and computational modeling, this work reveals that stability of immunogenic protein-like motifs is a critical parameter rationalizing the diverse humoral immune responses induced by different linear peptide epitopes.  In this paradigm, peptides with a minimal amount of stability (&#8710;GX < 0 kcal/mol) around a protein-like motif (X) are capable to induce antibodies with similar affinity for both peptide and native protein, more weakly stable peptides (&#8710;GX > 0 kcal/mol) trigger antibodies recognizing full protein but not peptide, and unstable peptides (&#8710;GX > 8 kcal/mol) fail to generate antibodies against either peptide or protein.  Immunization experiments involving peptides derived from the autoantigen histidyl-tRNA synthetase verify that selected peptides with varying relative stabilities predicted by molecular dynamics simulations induce antibody responses consistent with this theory.  Collectively, these studies provide insight pertinent to the structural basis of immunogenicity and, at the same time, validate this form of thermodynamic and molecular modeling as an approach to probe the development/evolution of humoral immune responses.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL07: Carlos Camacho - Computational approach to model peptide antigenicity </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/e23e459a-ff44-4efd-9274-8ba6433bce50" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e23e459a-ff44-4efd-9274-8ba6433bce50" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e23e459a-ff44-4efd-9274-8ba6433bce50" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL08" >HL08</a></strong><br><strong><em>Insights into corn genes derived from large-scale cDNA sequencing</em></strong><br> Monday, June 29 - 12:15 p.m. - 12:40 p.m.<br>Room: T2<br />Presenting author: Nickolai Alexandrov, Ceres, Inc., United States<br>Presentation Overview: <br>We present a large portion of the transcriptome of Zea mays, including 31,552 fully sequenced non-redundant cDNA clones. These and other previously sequenced transcripts have been aligned with genome sequences and have provided new insights into the characteristics of gene structures and promoters within this major crop species. We found that although the average number of introns per gene is about the same in corn and Arabidopsis, corn genes have more alternatively spliced isoforms. Corn genes, as well as genes of other Poaceae (Grass family), can be divided into two classes according to the GC content at the third position in codons. Many transcripts that have lower GC content have dicot homologs but the high GC transcripts are more specific to the grasses. The high GC content class is also enriched with intronless genes. This evolutionary divergence may be the result of horizontal gene transfer from species not only with different GC content but possibly that did not have introns, perhaps outside of the plant kingdom. By comparing the cDNAs described herein with the non-redundant set of corn mRNAs in GenBank, we estimate that there are about 50,000 different protein coding genes in Zea.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL08: Nickolai Alexandrov - Insights into corn genes derived from large-scale cDNA sequencing </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/e2338e03-c2a2-4431-ab52-c0670b74ba64" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e2338e03-c2a2-4431-ab52-c0670b74ba64" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e2338e03-c2a2-4431-ab52-c0670b74ba64" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ab initio gene prediction with accuracy of only 45% for protein coding genes - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">many more discoveries by mapping transcript to genomic sequences - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">corn genes divided into two groups by GC content, unknown why - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Corn genome several times bigger than rice genome - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">2400 Mb - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">There are a couple of million ESTs available - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">less than 50000 full length cDNA, around 10000 high quality - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">thorough quality checking was done - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">longer 5'UTR than in Arab and rice - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">estimation of no. of genes by same method that was used for human genome, result: approx. 50000 genes - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/e2338e03-c2a2-4431-ab52-c0670b74ba64">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL09" >HL09</a></strong><br><strong><em>Inferring pathway activity toward precise disease classification</em></strong><br> Monday, June 29 - 2:15 p.m. - 2:40 p.m.<br>Room: T1<br />Presenting author: Eunjung Lee, KAIST, Korea, Dem. Rep.<br>Presentation Overview: <br>The advent of microarray technology has made it possible to classify disease states based on gene expression profiles of patients.  Typically, marker genes are selected by measuring the power of their expression profiles to discriminate among patients of different disease states.  However, expression-based classification can be challenging in cancer due to factors such as cellular heterogeneity within a tissue sample and genetic heterogeneity across patients.  A promising technique for coping with these challenges is to incorporate pathway information into the disease classification procedure in order to classify disease based on the activity of entire signaling pathways or protein complexes rather than the expression levels of individual genes or proteins.  We propose a new pathway-based classification procedure in which markers are encoded not as individual genes, nor as the set of genes making up a known pathway, but as subsets of Òcondition-responsive genes (CORGs)Ó within those pathways.  Using expression profiles from seven different microarray studies, we show that the accuracy of this method is significantly better than both the conventional gene- and pathway- based diagnostics.  Furthermore, the identified CORGs may facilitate the development of effective diagnostic markers and the discovery of molecular mechanisms underlying disease.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL09: Eunjung Lee - Inferring pathway activity toward precise disease classification </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/b88ccd0a-8c2b-4d7e-aa18-32c914ed780d" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b88ccd0a-8c2b-4d7e-aa18-32c914ed780d" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b88ccd0a-8c2b-4d7e-aa18-32c914ed780d" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL10" >HL10</a></strong><br><strong><em>Evolution of genome structure: what statistics can tell us about the biology of chromosomes</em></strong><br> Monday, June 29 - 2:15 p.m. - 2:40 p.m.<br>Room: T2<br />Presenting author: Aaron Darling, University of California-Davis, United States<br>Presentation Overview: <br>The profound effect recombination has on genome evolution remains poorly understood.  Recombination can give rise to myriad genomic mutations, including large-scale deletions, lateral gene transfers, duplications, and even genome rearrangements.  In this talk, I examine patterns in how homologous recombination gives rise to genomic inversions.  The process of evolution by inversion has been modeled using a Bayesian statistical framework, and under that model we infer the phylogenetic history of inversions among nine Yersinia genomes.  We statistically confirm that inversions are generally short and that bacteria exhibit a Òbilateral genomic symmetryÓ between the origin and terminus of chromosome replication.  We extend previous controversy over breakpoint reuse rates by discovering that hotspots of breakpoint reuse localize near the origin of replication.  We illustrate how statistical confidence intervals can be derived for breakpoint reuse rates.  Finally, we discover a canonical configuration for the origin and terminus of replication in Yersinia.  Our work highlights how advances in combinatorial theory of genome rearrangement can lead to novel statistical inference methods, which can in turn offer new insight into genomic biology.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL10: Aaron Darling - Evolution of genome structure: what statistics can tell us about the biology of chromosomes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/62d7cc53-d297-4792-a33c-bc6b6789cba1" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/62d7cc53-d297-4792-a33c-bc6b6789cba1" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/62d7cc53-d297-4792-a33c-bc6b6789cba1" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Got into this area studying Yersinia pestis - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">However, he thinks there is a lot of structure in the process, even if we get a pattern that suggests uniform breakage - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Inversion events in bacteria are generally scattered across the chromosome. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Mauve is an excellent package for visualizing rearrangements - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">"Seevolution" visualizes inversions - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Breakpoints near the origin of the replication are 5-6x more frequent than at the terminus of replication. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Blog post: <a style="text-decoration:none" href="http://themindwobbles.wordpress.com/2009/06/29/hl10-evolution-of-genome-structure-what-statistics-can-tell-us-about-the-biology-of-chromosomes/">http://themindwobbles.wordpress.com/2009/06/29/hl10-evolution-of-genome-structure-what-statistics-can-tell-us-about-the-biology-of-chromosomes/</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL11" >HL11</a></strong><br><strong><em>Using side effects of medicines to identify drug targets</em></strong><br> Monday, June 29 - 2:45 p.m. - 3:10 p.m.<br>Room: T1<br />Presenting author: Michael Kuhn, EMBL Heidelberg, Germany<br>Presentation Overview: <br>This talk is based on the study ÒDrug target identification using side-effect similarityÓ, published in Science (July 2008). It describes a computational method to predict whether two drugs share targets based on phenotypic side-effect similarities. While earlier studies focused on molecular or cellular features, we employed a global analysis of drugs and their side effects to predict sharing of drug targets. Applied to 746 marketed drugs, a network of 1018 side effect-driven drug-drug relations became apparent, 261 of which are formed by chemically dissimilar drugs from different therapeutic indications. We experimentally tested 20 of these unexpected drug-drug relations and validated 13 implied drug-target relations by in vitro binding assays, of which 11 reveal inhibition constants equal to less than 10 micromolar. Nine of these were tested and confirmed in cell assays, documenting the feasibility of using phenotypic information to infer molecular interactions and hinting at new uses of marketed drugs.   <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL11: Michael Kuhn - Using side effects of medicines to identify drug targets </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/b7c1257d-c1cd-4b5f-a4c7-49e2ecc25911" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b7c1257d-c1cd-4b5f-a4c7-49e2ecc25911" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b7c1257d-c1cd-4b5f-a4c7-49e2ecc25911" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/egonw" target="_top" style="text-decoration:none">Egon Willighagen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">By drugs he means small molecules - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">side effects of drugs are human phenotypes - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">drugs with similar side effects may share targets - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">some side effects are common and uninformative - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">sideeffects.embl.de - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL12" >HL12</a></strong><br><strong><em>Gene Loss Under Neighbourhood Selection Following Whole Genome  Duplication And The Reconstruction Of The Ancestral Diploid</em></strong><br> Monday, June 29 - 2:45 p.m. - 3:10 p.m.<br>Room: T2<br />Presenting author: David Sankoff, University of Ottawa, Canada<br>Presentation Overview: <br>How can we construct a phylogeny based on gene order if some of the genomes under study are descendents of whole genome doubling events?  We have integrated a \\\\\\\\\\\\\\\"guided genome halving algorithm\\\\\\\\\\\\\\\" and a median genome routine to heuristically solve the small phylogeny problem, and have applied it to data containing thousands of sets of homologs among the poplar (tetraploid), grapevine (diploid) and papaya (diploid) genomes.  We have been able to reconstruct the last diploid ancestor of poplar before its genome was doubled.  We can then follow the evolution of duplicate genes pairs, and assess the mechanism that determines which of the two is likely to be lost and whether there is a bias towards losing adjacent genes on the same strand.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL12: David Sankoff - Gene Loss Under Neighbourhood Selection Following Whole Genome  Duplication And The Reconstruction Of The Ancestral Diploid </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/92b8d479-436c-449c-8fd2-2396ed0da8c7" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/92b8d479-436c-449c-8fd2-2396ed0da8c7" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/92b8d479-436c-449c-8fd2-2396ed0da8c7" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL13" >HL13</a></strong><br><strong><em>The Human Phenotype Ontology </em></strong><br> Monday, June 29 - 3:15 p.m.- 3:40 p.m.<br>Room: T1<br />Presenting author: Peter Robinson, CharitŽ - UniversitŠtsmedizin Berlin, Germany<br>Presentation Overview: <br>We will describe the Human Phenotype Ontology and explain how to use it to annotate and analyze human disease. In addition, we will present new methods for performing clinical diagnostics using ontological similarity measures in the HPO and methods for calculating P-values for the scores, as well as new results on the relationship between phenotypic modules (all genes/proteins related to phenotypic features of the HPO) and the network characteristics of these proteins in the protein interactome.   <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL13: Peter Robinson - The Human Phenotype Ontology </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/f1162d8e-9696-4675-9d1f-f304d0b979f0" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f1162d8e-9696-4675-9d1f-f304d0b979f0" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f1162d8e-9696-4675-9d1f-f304d0b979f0" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/mn3" target="_top" style="text-decoration:none">Mitsuteru Nakao</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">problem with OMIM: no controlled vocabulary - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">therefore: build ontology for OMIM for all descriptions used at least twice - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">synonyms were merged - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">refinement purely manual - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">~8800 terms in HPO - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top"><a style="text-decoration:none" href="http://www.human-phenotype-ontology.org/">http://www.human-phenotype-ontology.org</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">They have a procedure which calculates phenotypic similarity by finding their most-specific common ancestor. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">can calculate phenotypic similarity with the HPO, using information content of the most specific ancentor (root: info cont. of 0) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">phenotypic similarity of disease: take average over similarities of the phenotypes - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">can cluster the human phenome - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/f1162d8e-9696-4675-9d1f-f304d0b979f0">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL14" >HL14</a></strong><br><strong><em>Visualizing Genomic Dark Matter: Repeat Probability Clouds in the Human Genome</em></strong><br> Monday, June 29 - 3:15 p.m.- 3:40 p.m.<br>Room: T2<br />Presenting author: David Pollock, University of Colorado School of Medicine, United States<br>Presentation Overview: <br>We have no clear idea about where half the human genome comes from. This is the dark matter of the genome. The part of the genome we do know about is about 90% derived from repetitive elements (mostly transposable elements) and 10% derived from protein- or RNA-encoding genes. It is reasonable to presume that the dark matter is made up of the same things in similar proportions, but that the original sequences have mutated so much that we can no longer identify them easily. The newly developed Òrepeat probability cloudÓ approach is a means of identifying the signature of repeat structure in large genomes that is invisible to standard approaches. In a similar manner to how dark matter is visualized in the physical universe, we visualize genomic dark matter by its perturbing effect on sequence space. Other advantages of this method are that it is extremely fast and that it does not require prior knowledge of transposable element structure. The application of this technique is expected to revolutionize our understanding of the human genome and its origins in the evolution of ancestral mammalian genomes.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL14: David Pollock - Visualizing Genomic Dark Matter: Repeat Probability Clouds in the Human Genome </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/66068c16-9521-4f1e-b409-b94d1448f765" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/66068c16-9521-4f1e-b409-b94d1448f765" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/66068c16-9521-4f1e-b409-b94d1448f765" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/mn3" target="_top" style="text-decoration:none">Mitsuteru Nakao</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Distribution of sequence types in the human genome: 1.5% exons; 44% repetitive sequences; 54.5% unknown (dark matter). - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">RepeatMasker is not performing too well at identifying repeats; it misses ~50% of MIR fragments in the human genome, for example. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The dominant approach to repeat identification is based on the search of consensus sequences for repeat families (e.g. RepBase). Drawbacks: slow; not very sensitive to fragmentary matches; unable to identify novel repeats. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">De novo methods: RECON, PILER, RepeatFinder, RepeatScout. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Evolutionary de novo repeat analysis: better detection of imperfect repeats and fragment left over from extensive duplication and divergence. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Consensus based methods reduce multiple sequences to a single one; P-clouds group similar, high-copy L-mers together (16-mers for mammals, E<1 per genome). - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Next step: mapping P-clouds onto the genome using a sliding window. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Example: human chromosome X. Time required on a single workstation: 45 min (vs. >10 days for RECON and 8hrs for RepeatScout). - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Application to the whole human genome. 38% repetitive (P-clouds and RepeatMasker), 7% (only RM), 27% (PC only), 28% non-repetitive. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL15" >HL15</a></strong><br><strong><em>Analyzing risk factor of heart disease by a computational lipidology approach</em></strong><br> Monday, June 29 - 3:45 p.m. - 4:10 p.m.<br>Room: T1<br />Presenting author: Katrin Huebner, University Heidelberg, Germany<br>Presentation Overview: <br>In our work clinical investigation meets computational simulation to analyze blood lipid values beyond “bad“ and “good“ cholesterol to understand in vivo mechanisms leading to atherosclerosis and heart disease. Following an introduction into the clinical relevance of lipoproteins the experimental and a novel modeling setup is explained. As main results virtual lipoprotein profiles that closely matches clinical values from healthy subjects, model-based predictions of mimicked disorders in underlying molecular processes and alterations in high-resolution lipoprotein profiles are presented.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL15: Katrin Huebner - Analyzing risk factor of heart disease by a computational lipidology approach </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/9cc542c1-4df6-4e6c-9c33-9e159d0ec36f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9cc542c1-4df6-4e6c-9c33-9e159d0ec36f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9cc542c1-4df6-4e6c-9c33-9e159d0ec36f" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">model with 6 lipoprotein components, each one may take part in up to 20 kinetic processes - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e.g. synthesis, uptake, exchange with tissue/lipids, ... - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">simulation of lipoprotein particles, particles have an internal state --> observed kinetic heterogeneity - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">test if simulation results correspond to clinical data - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">virtual blood profiles closely match experimental values - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">hypercholestrolemia: accumulation of cholesterol, especially LDL cholesterol. decrease parameter of LDL receptor to mimick malfunction of the receptor in vivo. simulated lipids follow clinical observations - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">lipoprotein profile depends on initial composition (e.g. influence by diet) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">however, there are some dissimilar initial compositions which result in similar lipid profiles - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">increase resolution to make sure this is not a sample artifact - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">look for alterations in the high-density profile, but no difference in the low-resolution profile - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/9cc542c1-4df6-4e6c-9c33-9e159d0ec36f">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL16" >HL16</a></strong><br><strong><em>MotifMap: a human genome-wide map of candidate regulatory motif sites.</em></strong><br> Monday, June 29 - 3:45 p.m. - 4:10 p.m.<br>Room: T2<br />Presenting author: Pierre Baldi, UC Irvine, United States<br>Presentation Overview: <br>Comprehensive identification of all regulatory elements encoded in the human genome is a fundamental need in biomedical research. So far, only a small fraction of these elements have been identified experimentally. There is great interest in systematically discovering regulatory elements through computational means. We describe how to use comparative genomics to derive the first comprehensive map of regulatory elements in the human genome, taking advantage of the recent availability of 18 mammalian genomes. We developed a new scoring scheme for detecting regulatory elements, called Bayesian branch length score (BBLS), which can account for phylogenetic relationship between the species being compared, and motif-matching score at each individual species, while at the same time being flexible to alignment errors and missing sequences.  Using BBLS, we were able to predict 1.5 million regulatory sites in the human genome with FDR less than 50%, corresponding to 380 regulatory motifs in the Transfac database. The method is particularly effective for 155 motifs, for which over 121 thousands sites can be mapped with FDR less than 10%.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL16: Xiaohui Xie - MotifMap: a human genome-wide map of candidate regulatory motif sites. </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/c2ea1e3e-b33e-406e-8c02-bc9e12f8072e" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/c2ea1e3e-b33e-406e-8c02-bc9e12f8072e" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/c2ea1e3e-b33e-406e-8c02-bc9e12f8072e" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL17" >HL17</a></strong><br><strong><em>Learning from Resequencing Data: What To Do When the $1000 Genome Arrives?</em></strong><br> Tuesday, June 30 - 10:45 a.m. - 11:10 a.m.<br>Room: T1<br />Presenting author: Gregory Kryukov, Brigham & Women's Hospital / Harvard Medical School , United States<br>Presentation Overview: <br>We investigated the potential of resequencing all exons in a clinical  population to discover genes underlying human complex phenotypes. Computer simulations based on currently available deep resequencing data show that  genes meaningfully affecting a human trait can be identified in an unbiased  fashion, although large sample sizes would be required to achieve substantial power.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL17: Gregory Kryukov - Learning from Resequencing Data: What To Do When the $1000 Genome Arrives? </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ffeaebfe-079c-44a9-ac5a-32b54db79864" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ffeaebfe-079c-44a9-ac5a-32b54db79864" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ffeaebfe-079c-44a9-ac5a-32b54db79864" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/rklancer" target="_top" style="text-decoration:none">Richard Klancer</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">If it were not for the great variability among individuals medicine might as well be a science and not an art (Ostler 1892) (did I get this right? :)) - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Mendelian diseases can be characterized by linkage analysis (classical association studies) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Currently method of choice is GWAS. Today, talk about another type of AS that is based on sequencing and is based on rare rather than common SNPs. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Association studies based on sequencing will allow for understanding rare SNPs - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">New sequencing technologies, ways to collect clinical populations and exon capturing approaches can revolutionize the search for genes underlying human phenotypes - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">@Allyson: sounds about right :) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Sequencing will uncover low frequency variants - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">All genes have rare coding variants, and while sequencing will uncover low frequency variants the power to detect their associations is reduced and the multiple testing correction becomes very stringent - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">resequencing AS (RAS). The theory here is that most new missense mutations are functional, most new missense mutations are only weakly deleterious, and most functional missense mutations are likely to influence phenotype in the same direction. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">How many individuals need to be sequenced to find variants with minimal effect. - <a style="text-decoration:none" href="http://friendfeed.com/sextond" target="_top">David Sexton</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/ffeaebfe-079c-44a9-ac5a-32b54db79864">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL18" >HL18</a></strong><br><strong><em>Phylogenetic and Functional Assessment of Orthologs Inference Projects and Methods</em></strong><br> Tuesday, June 30 - 10:45 a.m. - 11:10 a.m.<br>Room: T2<br />Presenting author: Christophe Dessimoz, ETH Zurich, Switzerland<br>Presentation Overview: <br>The identification of orthologs, pairs of homologous genes in different species that started diverging through speciation events, is a central problem in genomics with applications in many research areas, including comparative genomics, phylogenetics, protein function annotation, or genome rearrangement.An increasing number of projects aim at inferring orthologs from complete genomes, but little is known about their relative accuracy or coverage. Since the exact evolutionary history of entire genomes remains largely unknown, predictions can only be validated indirectly, that is, in the context of the different applications of orthology. The few comparison studies published so far have asssessed orthology exclusively from the expectation that orthologs have conserved protein function.In the present work, we have introduced methodology to verify orthology in terms of phylogeny, and perform a comprehensive comparison of nine leading orthologs inference projects and two methods using both phylogenetic and functional tests. The results show large variations among the different projects in terms of performances, which indicates that the choice of orthology database can have a strong impact on any downstream analysis.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL18: Christophe Dessimoz - Phylogenetic and Functional Assessment of Orthologs Inference Projects and Methods </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/f1594bd7-b733-4bc4-85ca-ce67209e0ff5" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f1594bd7-b733-4bc4-85ca-ce67209e0ff5" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f1594bd7-b733-4bc4-85ca-ce67209e0ff5" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Types of homology according to Fitch, 1970 - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Orthologous relationships are not transitive - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">In-paralogues and out-paralogues, defined by the relation between speciation and gene duplication - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Genes with the same function: equivalogs or isofunctional homologues - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Homologues in the same genomic context isopositional orthologues - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">No biological meaning in the COG clustering algorithm - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">InParanoid, only pairs of genomes - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Orthology inferece: by best bi-directional hits or species/gene tree reconciliation - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">assessment of orthology inference methods - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">assessment; previous work: Hulsen, Huynen et al., Genome Biology, 2006 <a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/16613613">http://www.ncbi.nlm.nih.gov/pubmed/16613613</a> and Chen, Mackey et al., PLoS ONE, 2007 <a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/17440619">http://www.ncbi.nlm.nih.gov/pubmed/17440619</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/f1594bd7-b733-4bc4-85ca-ce67209e0ff5">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL19" >HL19</a></strong><br><strong><em>A Mathematical Framework for the Selection of an Optimal Set of Peptides for Epitope-Based Vaccines</em></strong><br> Tuesday, June 30 - 11:15 a.m. - 11:40 a.m.<br>Room: T1<br />Presenting author: Oliver Kohlbacher, Eberhard-Karls-UniversitŠt, Germany<br>Presentation Overview: <br>Due to their manifold advantages (e.g., safety, ease of production,analytical control) and their applicability in personalized medicineepitope-based vaccines (EVs) have recently been attracting significantinterest, in particular as a therapeutic strategy for cancer andinfectious diseases. EVs trigger an immune response via target-specificimmunogenic peptides (epitopes).  A crucial step in the design of an EVis the selection of the epitopes to be included. Depending on the numberof candidate epitopes, the diversity of the target population, and theimmunological requirements, the epitope selection can become a verycomplex problem. The epitope selection problem poses an interesting and novelbioinformatics problem. We present a mathematical framework to find anoptimal set of epitopes for an EV. Given a set of epitopes, theframework efficiently identifies the epitopes most likely to elicit abroad and potent immune response in the target population.We can translate the epitope selection problem into an integer linearprogram which allows an easy adaptation to different variants of the EVdesign problem. Among the few published computational approaches ourapproach is the first to identify an optimal epitope set.  Thismathematical framework will prove to be a valuable tool in vaccine design.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL19: Oliver Kohlbacher - A Mathematical Framework for the Selection of an Optimal Set of Peptides for Epitope-Based Vaccines </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/9cbebb87-b223-4737-b77e-1f10ded2dbb1" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9cbebb87-b223-4737-b77e-1f10ded2dbb1" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9cbebb87-b223-4737-b77e-1f10ded2dbb1" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">OptiTope webserver:  www.epitoolkit.org/optitope - <a style="text-decoration:none" href="http://friendfeed.com/jfkelso" target="_top">Janet Kelso</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Still need to: get real immunogenicity data rather than using MHC binding as a proxy, confirm that additivity holds - <a style="text-decoration:none" href="http://friendfeed.com/jfkelso" target="_top">Janet Kelso</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL20" >HL20</a></strong><br><strong><em>Prediction of Binding Sites on Proteins Using the Gaussian Network Model</em></strong><br> Tuesday, June 30 - 11:15 a.m. - 11:40 a.m.<br>Room: T2<br />Presenting author: Burak Erman, Koc University, Turkey<br>Presentation Overview: <br>Residues at the binding sites of the ligand and receptor of several enzyme-inhibitor and antibodyantigencomplexes are predicted from the slowest (for the ligand) and fastest (for the receptor) modes ofmotion by the Gaussian Network Model applied to unbound molecules.   <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL20: Burak Erman - Prediction of Binding Sites on Proteins Using the Gaussian Network Model </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/65f7b732-cb80-4cd2-a06b-9d4308843ef4" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/65f7b732-cb80-4cd2-a06b-9d4308843ef4" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/65f7b732-cb80-4cd2-a06b-9d4308843ef4" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Energy and environment are a closed system, thermodynamic modeling of system with entropy, energy, volume and position of the residues - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">A probability function over the average values of the system can be used to formulate the average of the extensive variables and the conjugate variables. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Harmonic fluctuations models the relationship between the residues in the system. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Higher order correlation can be used to obtain the energy fluctiuations and the residues, which is proportional. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The highest mode is the interesting end of the decomposition. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">HIV protease as a model system, active site and binding interfaces are identified as important residues as fast modes. System behaves as an energy sieve. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Residues at the surface are energy gates, residues connecting two surfaces are hubs. Residues along the interaction pathway are anticorrelated to the important ones. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Example of the peptide binding HLA protein (1OGT) shows 6 important residues by highest mode analysis. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">One AA is binding the peptide, other stabilizes the interaction. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">[Structure pictures not shown] - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/65f7b732-cb80-4cd2-a06b-9d4308843ef4">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL21" >HL21</a></strong><br><strong><em>The miRNA/siRNA saturation effect - transfection of small RNAs compromise gene regulation by endogenous microRNAs</em></strong><br> Tuesday, June 30 - 11:45 a.m. - 12:10 p.m.<br>Room: T1<br />Presenting author: Debora Marks, Harvard Medical School, United States<br>Presentation Overview: <br>Transfection of siRNAs or miRNAs (microRNAs) into cells typically lowers gene expression of hundreds of genes, assessed by decreases in protein or mRNA levels, but increases in gene expression have also been observed. One explanation for unexpected up-regulation upon miRNA or siRNA over-expression is a reduction in the effective function of the endogenous microRNAs. This may result from competition between exogenous and endogenous RNAs for the intracellular small RNA protein machinery. We tested the validity of this explanation by computational analysis of more than 150 si/miRNA transfection experiments in 7 different cell types for which genome-wide mRNA changes were measured using microarrays. After verifying the expected down-regulation of genes with UTRs that contain target sites for the exogenously introduced small RNAs, we show that genes with target sites for endogenous miRNAs are significantly up-regulated. Confirming this result we observe this competition effect with protein expression changes and a striking correlation of the dose response and temporal response of up-regulated genes with down-regulated genes after siRNA transfections. These findings have broad implications for the design and interpretation of experiments using small RNAs and for the design of clinical trials using siRNA therapeutics.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL21: Debora Marks - The miRNA/siRNA saturation effect - transfection of small RNAs compromise gene regulation by endogenous microRNAs </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ec55850a-b287-4e9e-8481-65dcfad57f17" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ec55850a-b287-4e9e-8481-65dcfad57f17" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ec55850a-b287-4e9e-8481-65dcfad57f17" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Debora Marks has had to cancel her presentation due to a family emergency. - <a style="text-decoration:none" href="http://friendfeed.com/ffbb" target="_top">bb</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL22" >HL22</a></strong><br><strong><em>Sequence Similarity Network Reveals Common Ancestry of Multidomain Proteins </em></strong><br> Tuesday, June 30 - 11:45 a.m. - 12:10 p.m.<br>Room: T2<br />Presenting author: Dannie Durand, Carnegie Mellon University, United States<br>Presentation Overview: <br>The challenge of homology identification in multidomain families with varied domain architectures is to distinguish sequence pairs that share common ancestry from pairs that share an inserted domain but are otherwise unrelated. This distinction is essential for accuracy in gene annotation, function prediction, and comparative genomics. We first present an extension of the traditional model of homology to include domain insertions and a manually curated benchmark of well-studied mammalian families. We next introduce Neighborhood Correlation, a novel method that identifies homologs with great accuracy based on the observation that gene duplication and domain shuffling leave distinct patterns in the sequence similarity network. In a rigorous, empirical comparison, Neighborhood Correlation outperforms sequence similarity, alignment length, and domain architecture comparison. Neighborhood Correlation is easy to compute, does not require explicit knowledge of domain architecture, and classifies both single and multidomain homologs with high accuracy. Our work represents a departure from the prevailing view that the concept of homology cannot be applied to genes that have undergone domain shuffling. We show that homology can be rationally defined for multidomain families with diverse architectures by considering the genomic context of the genes that encode them.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL22: Dannie Durand - Sequence Similarity Network Reveals Common Ancestry of Multidomain Proteins </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/5fe14f75-bc88-4869-9947-68f4b0f8a205" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5fe14f75-bc88-4869-9947-68f4b0f8a205" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5fe14f75-bc88-4869-9947-68f4b0f8a205" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/ruchira" target="_top" style="text-decoration:none">Ruchira S. Datta</a>, <a href="http://friendfeed.com/larsjuhljensen" target="_top" style="text-decoration:none">Lars Juhl Jensen</a>, <a href="http://friendfeed.com/stajich" target="_top" style="text-decoration:none">Jason Stajich</a> and <a href="http://friendfeed.com/arne1" target="_top" style="text-decoration:none">arne</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Multidomain proteins are difficult to categorize because different parts have different histories. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Multidomain homolgs - finding homologs is an important aspect of functional genomics. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Song et al 2008 PLoS Computational Biology 4(5) - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top"><a style="text-decoration:none" href="http://www.neighborhoodcorrelation.org/">http://www.neighborhoodcorrelation.org/</a> - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Paper: <a style="text-decoration:none" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2377100">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2377100</a> - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Genes that share common ancestry tend to have similar structure and function. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">also use to build comparative map of synteny - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Sequence comparison can be used to identify chromosomal regions that share common ancestry. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">this is called "spatial genomics" - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">How multidomain proteins fit this picture? - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/5fe14f75-bc88-4869-9947-68f4b0f8a205">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL23" >HL23</a></strong><br><strong><em>Models from experiments: combinatorial perturbations of cancer cells</em></strong><br> Tuesday, June 30 - 12:15 p.m. - 12:40 p.m.<br>Room: T1<br />Presenting author: Sven Nelander, University of Gothenburg , Sweden<br>Presentation Overview: <br>We present a novel method for deriving network models from molecular profiles of perturbed cellular systems. The network models aim to predict quantitative outcomes of combinatorial perturbations, such as drug pair treatments or multiple genetic alterations. Mathematically, we represent the system by a set of nodes, representing molecular concentrations or cellular processes, a perturbation vector and an interaction matrix. After perturbation, the system evolves in time according to differential equations with built-in nonlinearity, similar to Hopfield networks, capable of representing epistasis and saturation effects. For a particular set of experiments, we derive the interaction matrix by minimizing a composite error function, aiming at accuracy of prediction and simplicity of network structure. To evaluate the predictive potential of the method, we performed 21 drug pair treatment experiments in a human breast cancer cell line (MCF7) with observation of phospho-proteins and cell cycle markers. The best derived network model rediscovered known interactions and contained interesting predictions. Possible applications include the discovery of regulatory interactions, the design of targeted combination therapies and the engineering of molecular biological networks.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL23: Sven Nelander - Models from experiments: combinatorial perturbations of cancer cells </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/1b684d3a-ffc5-4b54-951a-558a65a42a86" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/1b684d3a-ffc5-4b54-951a-558a65a42a86" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/1b684d3a-ffc5-4b54-951a-558a65a42a86" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">What happens when you change more than one thing in a cell? - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">A defining feature of cancer cells is a breakdown in regulation. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">A defining feature of cancer cells is a breakdown in regulation by aquired GOF (gain of function) or LOF (loss of function) mutations. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">One of the advances in cancer pharmacology are compounds that target the function of the proteins in these pathways. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Approaches to CP (combinatorial perturbations) include metabolic flux models, and what we need is an equivalent for the cancer cells. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Each perturbation pattern consists of 1, 2 or 3 perturbations in sequence. This pattern is converted into a phosphoprotein and phenotype pattern. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">use a linear differentiation equation - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Start with good tradeoffs between model fit and simplicity. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">They use a MC simulation, which results in the probabilities of functional interaction. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">They adapted CoPIA to cancer genomics data, and the main issue was speed. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/1b684d3a-ffc5-4b54-951a-558a65a42a86">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL24" >HL24</a></strong><br><strong><em>Discovery of a hidden sequence motif conserved in the bacterial type III secretion signal: implications for structure, drug discovery and host-pathogen systems models.</em></strong><br> Tuesday, June 30 - 12:15 p.m. - 12:40 p.m.<br>Room: T2<br />Presenting author: Jason McDermott, Pacific Northwest National Laboratory, United States<br>Presentation Overview: <br>We describe a novel computational method for identification of type III secreted virulence effectors in bacteria and characterization of a putative secretion signal from apparently unrelated sequences. We will discuss preliminary experimental results supporting our model of a partially disordered structure that may be widely conserved in the absence of sequence similarity and show how these predictions support development of systems models of host-pathogen interactions.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL24: Jason McDermott - Discovery of a hidden sequence motif conserved in the bacterial type III secretion signal: implications for structure, drug discovery and host-pathogen systems models. </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/c8aebd23-1e9f-48b3-9435-3918f2135626" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/c8aebd23-1e9f-48b3-9435-3918f2135626" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/c8aebd23-1e9f-48b3-9435-3918f2135626" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Salmonella typhimurium is the main model organism for this work. It builds its own niche using the T3SS in the macrophages, kills many people each year. In the US, it's mainly a food pathogen. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Good model system for Salmonella typhi, the agent of typhus, a narrow range human pathogen. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Includes two T3SS, injecting effectors into the host cell. 47 known effectors. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Problems in T3SS, consisting of the injectosome and the secreted effectors, is the remote homogy of the efffectors. N-terminus of the effectors required for secretion but no similarity could be found. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Typically not part of the 3D structure, appears to be disordered. How could the diverse sequences have the same function? Different mechanism ´for every effector? Cognate chaperones help the effectors to thread the protein into the injectosome. But you can delete the chaperone. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Hypothesis for this talk: There is a common mechanism in the effectors. Typically BLAST was used to identify the secretion family. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Previous analysis have been working for some organisms but not across species. The group developed a machine learning method (SVMs) from the primary sequence, including the nt-.sequence and used an unaligned input of the sequence. Used Pseudomonas syringae with 35 known effectors. Used the rest of the genome as negative examples. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ROC is in the 0.05 range. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Applied this to Chlamydia trachomatis, an almost untractable obligate intracellular pathogen. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">A similar paper appeared back to back with this paper, another paper was published recently. <a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/19526054">http://www.ncbi.nlm.nih.gov/pubmed/19526054</a> - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/c8aebd23-1e9f-48b3-9435-3918f2135626">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL25" >HL25</a></strong><br><strong><em>Re-examining the connection between the network topology and essentiality</em></strong><br> Tuesday, June 30 - 2:15 p.m. - 2:40 p.m.<br>Room: T1<br />Presenting author: Teresa Przytycka, NIH, United States<br>Presentation Overview: <br>We instigate the reason for correlation between degree and essentiality, observed in a number of yeast networks.  Based on an analysis of six genome-wide protein interaction networks compiled from diverse sources of interaction data, we rejected the previously proposed hypotheses and put forward an alternative explanation. We argued that the majority of hubs are essential due to their involvement in groups densely connected proteins, many presumably protein complexes enriched in essential proteins. <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL25: Teresa Przytycka - Re-examining the connection between the network topology and essentiality </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/0c20a965-a51c-4c84-be5d-76cf15dbe499" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0c20a965-a51c-4c84-be5d-76cf15dbe499" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0c20a965-a51c-4c84-be5d-76cf15dbe499" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL26" >HL26</a></strong><br><strong><em>Built-in loops allow versatility in domainÐdomain interactions: Lessons from self-interacting domains</em></strong><br> Tuesday, June 30 - 2:15 p.m. - 2:40 p.m.<br>Room: T2<br />Presenting author: Eyal Akiva, The Hebrew University, Israel<br>Presentation Overview: <br>The function of most proteins depends on their interaction with other proteins. It was shown that many proteinÐprotein interactions are mediated by protein domains, and that there are distinct domain pairs that are used repeatedly as interaction mediators in various protein contexts. However, not all protein pairs with the corresponding domains that may mediate interaction do interact. It is conceivable that there are intra-domain structural and sequence features that play a role in determining the interaction potential of domains. Here, we discover such features by comparing domains that, on the one hand, mediate homodimerization of proteins and, on the other, occur in different proteins that are monomeric. This comparison uncovered surface loops that can be considered as determinants of the interactions. There are enabling loops, which mediate the domain interactions, and disabling loops that prevent the interactions. The presence of the enabling/disabling loops is consistent with the fulfillment/prevention of the interaction and is highly preserved in evolution. Thus, along with the preservation of structural elements that enable interaction, evolution maintains elements intended to prevent unwanted interactions. Our results extend the hierarchy of attributes that establish the modularity of domain-mediated protein-protein interactions, and provide a novel approach for predicting domain-domain interactions.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL26: Eyal Akiva - Built-in loops allow versatility in domain-domain interactions: Lessons from self-interacting domains </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/0268bd7e-2421-49e2-a2a8-f63da233c7ae" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0268bd7e-2421-49e2-a2a8-f63da233c7ae" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0268bd7e-2421-49e2-a2a8-f63da233c7ae" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL27" >HL27</a></strong><br><strong><em>Regulatory networks define phenotypic classes of human stem cell lines</em></strong><br> Tuesday, June 30 - 2:45 p.m. - 3:10 p.m.<br>Room: T1<br />Presenting author: Igor Ulitsky, Tel Aviv University, Israel<br>Presentation Overview: <br>Hundreds of different human cell lines from embryonic, fetal and adult sources are referred to as stem cells, even though they range from pluripotent cellsÑtypified by embryonic stem cells, which are capable of virtually unlimited proliferation and differentiationÑto adult stem cell lines, which can generate a far more limited repertoire of differentiated cell types. The rapid increase in reports of new sources of stem cells and their anticipated value to regenerative medicine calls for a general, reproducible method for classification of these cells. We have created and analyzed a database of global gene expression profiles that enables the analysis of cultured human stem cells in the context of a wide variety of pluripotent, multipotent and differentiated cell types. We categorized a collection of  150 cell samples, and discovered that pluripotent stem cell lines group together, whereas other cell types, including brain-derived neural stem cell lines, are very diverse. In addition, we uncovered a proteinÐprotein network that is shared by the pluripotent stem cells. Our results offer a new strategy for classifying stem cells and support the idea that pluripotency and self-renewal are under tight control by specific molecular networks.   <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL27: Igor Ulitsky - Regulatory networks define phenotypic classes of human stem cell lines </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/02781394-609a-4268-8d96-7ca1778d9933" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/02781394-609a-4268-8d96-7ca1778d9933" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/02781394-609a-4268-8d96-7ca1778d9933" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to understand molecular basis of pluripotency; need to ensure safety of stem cell therapy (don't want to create cancers) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">mRNA profiling of pluripotent and multipotent stem cells. using negative matrix factorization to group the cell types - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">12 clusters work well, get mostly homogenous clusters - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Matisse approach, Ulitsky & Shamir BMC Systems Biol, 2007 - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">create subnetwork of genes upregulated in pluripotent cells, have reason to believe that the subnetwork contains most of the pluripotency machinery. highly expressed in plurip. cells, not highly expressed in differentiated cells - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">also look for microRNAs; profile 800 miRNAs in 26 cell lines: Laurent et al., Stem Cells 2008. stem cell groups share similar miRNA profiles - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">look for differences between embryonic and other cells lines. clustered miRNA had a tendency to be differentially regulated in embr. cell lines - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">major seed sequence (AAGUGC) contained in miRNAs from 4 clusters (21 miRNAs, 18 are upregulated in embr. cells), also have such families in other species (non-homologous to human) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">find complement of seed sequences in differentially regulated genes; look for pathways co-regulated by miRNAs, identify 57 such "PRMs"; find subnetwork of proteins which are jointly regulated, e.g. in cell cycle progression; find new submodule, which could be important in protection from neurodegeneration - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Login servers still down. Talk notes at <a style="text-decoration:none" href="http://www.fiamh.info/articles/17/ismbeccb-2009-day-2">http://www.fiamh.info/articles/17/ismbeccb-2009-day-2</a> - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL28" >HL28</a></strong><br><strong><em>Confirming alternative protein isoforms in Drosophila</em></strong><br> Tuesday, June 30 - 2:45 p.m. - 3:10 p.m.<br>Room: T2<br />Presenting author: Michael Tress, Spanish National Cancer Research Center (CNIO), Spain<br>Presentation Overview: <br>Alternative splicing of messenger RNA permits the formation of a wide range of mature RNA transcripts and has the potential to generate a diverse spectrum of functional proteins. While there is extensive evidence for large scale alternative splicing at the transcript level there have been no comparable studies validating the existence of alternatively spliced protein isoforms.Two recent large scale proteomics studies generated extensive, high quality peptide catalogs from the Drosophila melangaster proteome. The analysis of this proteomic data confirmed the presence of multiple alternative gene products for over a hundred Drosophila genes and for the first time demonstrated the large-scale expression of alternatively spliced gene products.  The fact that evidence for alternatively spliced isoforms came from proteomics studies confirms that these alternative isoforms must be expressed in sufficient quantity and be stable enough in vivo to be detected.  However, the study suggested that many of the alternative gene products are likely to have regions that are disordered in solution, and that specific proteomics methodologies may be required to identify these isoforms.The analysis highlights the growing importance of proteomics in the validation of predicted proteins and points the way towards further research in this area.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL28: Michael Tress - Confirming alternative protein isoforms in Drosophila </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/f3bb2c53-080e-44f7-b88d-1a753cf991ac" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f3bb2c53-080e-44f7-b88d-1a753cf991ac" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f3bb2c53-080e-44f7-b88d-1a753cf991ac" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL29" >HL29</a></strong><br><strong><em>Biomedical Discovery Acceleration</em></strong><br> Tuesday, June 30 - 3:15 p.m.- 3:40 p.m.<br>Room: T1<br />Presenting author: Lawrence Hunter, University of Colorado School of Medicine, United States<br>Presentation Overview: <br>Recent technology has made it possible to do experiments that show hundreds or even thousands of genes play a role in a disease or other biological phenomena. Interpreting these experimental results in the light of everything that has ever been published about any of those genes is often overwhelming, and the failure to take advantage of all prior knowledge may impede biomedical research. The computer program described in this paper ÒreadsÓ the biomedical literature and molecular biology databases, ÒreasonsÓ about what all that information means to this experiment, and ÒreportsÓ on its findings in a way that makes digesting all of this information far more efficient than ever before possible. Analysis of a large, complex dataset with this tool led rapidly to the creation of a novel hypothesis about the role of several genes in the development of the tongue, which was then confirmed experimentally.   <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL29: Lawrence Hunter - Biomedical Discovery Acceleration </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/3786275a-c4f9-4f28-ae70-3314412e4689" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/3786275a-c4f9-4f28-ae70-3314412e4689" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/3786275a-c4f9-4f28-ae70-3314412e4689" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/colmr" target="_top" style="text-decoration:none">Colm Ryan</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">single genes don't explain phenotypes, have to understand gene sets. genes work together in complex, dynamics groups. need to be able to apply data from other disciplines to our data. but: knowledge has increased so much that it's impractical to screen all potentially relevant papers. - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">example : Relaxin I. Why do some people not respond to beta blockers during heart failure? Cardiologists didn't know about relaxin I, well known in context of child birth. - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">"Genes do not respect disciplinary boundaries." - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">3R system: "Hanalyzer", Reading (import dbs, extract knowledge)/Reasoning (infer additional knowledge)/Reporting (help biologists explain data / generate new hypothesis). - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">idea: align knowledge networks with data networks. nodes: fixed entities, e.g. gene db ids or ontology terms. links: potentially uncertain relations between the nodes. (i.e. probabilistic network). compare multiple sources of info into combined edge - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">data network: data from experiments; knowledge network: data from literature. combine data and knowledge, visualize in two ways: either equal weight, or logistics (emphasizes data) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">cyotscape plugin "common attributes", also see Hanalyzer demo on YouTube, have links to data sources - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">can look at correlated genes, first check out which genes are known and then extend the network to find new genes that may be developed in the process: get new hypothesis. check localization of proteins: could validate predictions. - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">example question: What is the roloe of CAV3 in muscle. create network of genes, GO terms and other ontology terms - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL30" >HL30</a></strong><br><strong><em>Automated Analysis of Patterns in Human Protein Atlas Images</em></strong><br> Tuesday, June 30 - 3:15 p.m.- 3:40 p.m.<br>Room: T2<br />Presenting author: Robert Murphy, Carnegie Mellon University, United States<br>Presentation Overview: <br>This paper describes the first approach to automatically analyze all major subcellular patterns in tissue images from the Human Protein Atlas.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL30: Robert Murphy - Automated Analysis of Patterns in Human Protein Atlas Images </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/c210742b-cbae-477c-9507-de060ba0caaa" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/c210742b-cbae-477c-9507-de060ba0caaa" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/c210742b-cbae-477c-9507-de060ba0caaa" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL31" >HL31</a></strong><br><strong><em>Toward Automated, Practical Provision of Need-Based, High-Utility Text to Diverse Biomedical Users and Database Curators</em></strong><br> Tuesday, June 30 - 3:45 p.m. - 4:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Hagit Shatkay, Queen's University , Canada<br>Presentation Overview: <br>This work is concerned with bridging the gap between actual text-needs of biomedical users (database curators being one example), and text-mining methods. Biomedical text-mining aims to serve a diverse community of scientists by identifying relevant information within scientific text. We note that there is no Òaverage biologistÓª client; different users have distinct needs. Specifically, evaluation efforts (BioCreative, TREC) noted that database curators are often interested in sentences showing experimental evidence and methods. Conversely, lab scientists often search for high-confidence facts about genes and proteins. We have recently introduced a multi-dimensional categorization and annotation scheme, applicable to a wide variety of biomedical text, while supporting specific biomedical retrieval and extraction tasks, including the identification of methods and experimental evidence. Along with it, we developed a large annotated corpus, (10,000 sentences tagged by eight annotators), and trained and tested machine learning classifiers to automatically categorize text based on the tagging scheme. We have also developed models to handle noise and disagreements among annotators. We show that automatic annotation of text along multiple useful dimensions is highly feasible, and that our new framework for scientific sentence categorization is applicable in practice. Among other categories, our classifier accurately identifies methodology and experimental statements.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL31: Hagit Shatkay - Toward Automated, Practical Provision of Need-Based, High-Utility Text to Diverse Biomedical Users and Database Curators </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/86fff92f-6d72-4faa-90e2-b0cf1dcfaee6" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/86fff92f-6d72-4faa-90e2-b0cf1dcfaee6" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/86fff92f-6d72-4faa-90e2-b0cf1dcfaee6" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL32" >HL32</a></strong><br><strong><em>Network modeling of human interactome and phenome</em></strong><br> Tuesday, June 30 - 3:45 p.m. - 4:10 p.m.<br>Room: T1<br />Presenting author: Rui Jiang, Tsinghua University, China<br>Presentation Overview: <br>We model the genome- and phenome-wide human gene-disease relationship using simple regression models, and show that the correlation between protein network distance and disease phenotype similarity accurately predicts disease genes. We perform genome-wide candidate gene prioritization for over 5000 diseases, revealing a comprehensive and modular genetic landscape of human disease. We also provide quantitative evidence supporting the correlation between phenotypic overlap and genetic overlap in human diseases, which implies a concordance between the topology of disease network and gene network. We then introduce the network alignment technique to compare the topology of the protein network and disease network, leading to the discovery of 39 disease families and corresponding causal gene networks, as well as a novel network alignment-based disease gene prediction approach and the high-quality predictions for 70 human diseases. Related paper has been featured by Nature Publishing Group in four areas: Genetics, Pathology, Systems Biology and Biotechnology, and the paper is also highlighted in Nature China. The paper became the journal\'s most accessed article by September of 2008, and is now cited by 7 papers according to ISI. The predicted disease genetic landscape is publicly available, and has been visited by over 1000 researchers around the world.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL32: Rui Jiang - Network modeling of human interactome and phenome </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/46c330bf-f84f-4782-adb2-70997d32c4b5" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/46c330bf-f84f-4782-adb2-70997d32c4b5" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/46c330bf-f84f-4782-adb2-70997d32c4b5" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/woodydon" target="_top" style="text-decoration:none">Kuan-Ting Lin</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">most current methods to predict disease genes are based on "guilt by association" - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e.g. approach based on protein-protein interaction networks - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">these methods depend on seed genes, which limits the power of such methods - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Question: can one borrow seed genes of other diseases for predicting the basis of a disease without known genetic basis? - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">consider disease as a multi-layered network: sequence -- molecules/proteins -- clinical traits -- disease - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">sharing of nodes and edges between diseases - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">can calculate similarity of diseases in their clinical traits - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">linear regression model to explain similarity between diseases based on contributions from genes (that are close in the PPI network) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">approach called CIPHER - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">concordance score for gene--disease links - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/46c330bf-f84f-4782-adb2-70997d32c4b5">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL33" >HL33</a></strong><br><strong><em>Mitochondrial beta-barrel Outer Membrane Proteins, All Accounted For?</em></strong><br> Tuesday, June 30 - 3:45 p.m. - 4:10 p.m.<br>Room: T2<br />Presenting author: Paul Horton, AIST, Computational Biology Research Center, Japan<br>Presentation Overview: <br>Mitochondrial  -barrel Outer Membrane Proteins (MBOMPs) are an important class of proteins which include the essential proteins Tom40, Sam50 and the highly abundant VDAC protein. (Until our challenge) it has been thought that Eukaryotic genomes such as yeast would have 100\'s of such proteins [Wimley, Cur. Opin. Struct. Bio. 2003], an estimate largely based on analogy to bacteria, which share a common ancestor with mitochondria and have numerous families of  -barrel Outer Membrane Proteins. Interestingly, despite these high estimates and the availability of the complete genomes of many Eukaryotic organisms; only 5 families of MBOMPs are currently known: (Tom40, Sam50, VDAC, Mdm10, Mmm2).In our talk we will present substantial evidence to support the provocative hypothesis that the 5 known families of MBOMPs represent all or nearly all of the MBOMPs in the entire Eukaryotic world. This conclusion is based on 1) our initial analysis [Imai et al. Cell 2008] of the recently discovered  -signal ([Kutik et al. Cell 2008]) for MBOMP membrane integration, and 2) further analysis covering all uniprot Eukaryotic proteins as well as a search for possible \"bacteria-like\" MBOMPs without a  -signal.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL33: Paul Horton - Mitochondrial beta-barrel Outer Membrane Proteins, All Accounted For? </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/6459bc7d-2782-48a7-bd86-054d98d4b946" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/6459bc7d-2782-48a7-bd86-054d98d4b946" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/6459bc7d-2782-48a7-bd86-054d98d4b946" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL34" >HL34</a></strong><br><strong><em>Network-based prediction of human tissue-specific metabolism</em></strong><br> Wednesday, July 1 - 10:45 a.m. - 11:10 a.m.<br>Room: T1<br />Presenting author: Tomer Shlomi, Technion, Israel<br>Presentation Overview: <br>A major challenge in studying metabolic processes in mammals is that different tissues are characterized by distinct metabolic functions whose direct in vivo investigation is difficult. Here we present the first computational method that successfully obtains a large-scale, tissue-specific description of human metabolism. Our approach is based on integrating tissue-specific gene and protein expression data with an existing comprehensive reconstruction of the global metabolic network. Applying the method to predict tissue-specific metabolic activity for 10 human tissues reveals that post-transcriptional regulation plays a central role in shaping tissue-specific metabolic activity profiles. The predicted tissue specificity of metabolic disease-causing genes and of metabolite exchange with biofluids are shown to go markedly beyond that manifested in the enzyme expression data, and are validated via large-scale mining of tissue-specificity data. Our results lay down the computational basis for the genome-wide study of normal and abnormal human metabolism in a tissue-specific manner.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL34: Tomer Shlomi - Network-based prediction of human tissue-specific metabolism </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/2457b3d8-2bda-45f2-ac89-b7a04ce9ef2e" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/2457b3d8-2bda-45f2-ac89-b7a04ce9ef2e" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/2457b3d8-2bda-45f2-ac89-b7a04ce9ef2e" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">so far similar to Sunday's biopathways SIG talk: <a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009/59604510/biopathways-sig-tomer-shlomi-predicting">http://friendfeed.com/ismbeccb2009/59604510/biopathways-sig-tomer-shlomi-predicting</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to predict metabolic activity of human genes - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">problem: expression level does not reflect metabolic activity: has post-translational regulation - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">(4% up-regulated, 16% down-regulated) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">large-scale validation using tissue-specific literature data - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">tissue-specificity of disease genes: if a gene causes a disease in a certain tissue, it is likely to be active in this tissue - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">method is capable of predicting tissues where the genes are post-transcriptionally up-regulated - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">highly expressed neighboring genes give hints that a certain gene will be active - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">MEAT: metabolic expression analysis tool - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">follow-up work: biomarkers from genetic disorders, see SIG talk linked at the top and the paper: <a style="text-decoration:none" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2683725">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2683725</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL35" >HL35</a></strong><br><strong><em>Context-specific BLAST detects twice as many homologous proteins as BLAST</em></strong><br> Wednesday, July 1 - 10:45 a.m. - 11:10 a.m.<br>Room: T2<br />Presenting author: Johannes Soeding, University of Munich, Germany<br>Presentation Overview: <br>We present a context-specific approach to sequence comparison that allows to drastically improve the sensitivity and alignment quality in comparison with conventional search methods. Our context-specific version of BLAST, CS-BLAST, achieves over two-fold increase in sensitivity at the same specificity and speed, the iterative version CSI-BLAST finds as many homologs after two iterations as PSI-BLAST after five iterations.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL35: Johannes Soeding - Context-specific BLAST detects twice as many homologous proteins as BLAST </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ec8fab00-c3e6-4372-ad2a-84cad6f709a8" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ec8fab00-c3e6-4372-ad2a-84cad6f709a8" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ec8fab00-c3e6-4372-ad2a-84cad6f709a8" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/gbrsales" target="_top" style="text-decoration:none">Gabriele Sales</a> and <a href="http://friendfeed.com/diriano" target="_top" style="text-decoration:none">Diego M. Riaño-Pachón</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Not only about BLAST, but also about models of protein evolution. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Importance of BLAST searches: 400,000 searches run at NCBI per day. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">BLAST (1990) and PSI-BLAST (1997) cited 45,000 times. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Protein bioinformatics relies heavily on sequence searching. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Sequence alignments are a special case of profile alignments. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The score of an alignment can be thought as the log of the ratio of the probability of the mutations needed to go from sequence X to Y over the average probability of Y. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Context specific substitution matrices used with success, among others, for protein structure prediction (Rice & Eisenberg 1997, Huang & Bystroff 2006). - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Their approch: take 6 neighbours to the left and to the right of a nucleotide. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The search uses a sliding window over the sequence; the mutation probability is computed by looking at a precalculated library of profiles. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Profiles are built out of homology relations found with BLAST. The library contains 1 million profiles. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/ec8fab00-c3e6-4372-ad2a-84cad6f709a8">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL36" >HL36</a></strong><br><strong><em>Leveraging the context-specific coordination of transcript and metabolite concentrations to discover gene-metabolite interactions.</em></strong><br> Wednesday, July 1 - 11:15 a.m. - 11:40 a.m.<br>Room: T1<br />Presenting author: Patrick Bradley, Princeton University , United States<br>Presentation Overview: <br>Metabolite concentrations can regulate gene expression, which can in turn regulate metabolic activity. The extent to which functionally related transcripts and metabolites show similar patterns of concentration changes, however, remains unestablished. We measure and analyze the metabolomic and transcriptional responses of Saccharomyces cerevisiae to carbon and nitrogen starvation. Our analysis demonstrates that transcripts and metabolites show coordinated response dynamics. Furthermore, metabolites and gene products whose concentration profiles are alike tend to participate in related biological processes. To identify specific, functionally related genes and metabolites, we develop an approach based on Bayesian integration of the joint metabolomic and transcriptomic data. This algorithm finds interactions by evaluating transcript-metabolite correlations in light of the experimental context in which they occur and the class of metabolite involved. It effectively predicts known enzymatic and regulatory relationships, including a gene-metabolite interaction central to the glycolytic-gluconeogenetic switch. This work provides quantitative evidence that functionally related metabolites and transcripts show coherent patterns of behavior on the genome scale and lays the groundwork for building gene-metabolite interaction networks directly from systems-level data.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL36: Patrick Bradley - Leveraging the context-specific coordination of transcript and metabolite concentrations to discover gene-metabolite interactions. </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/91c826bf-e717-4689-8dcb-fbc796b82e2d" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/91c826bf-e717-4689-8dcb-fbc796b82e2d" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/91c826bf-e717-4689-8dcb-fbc796b82e2d" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">[apologizes to bloggers for the long title we'd have to write down :) ] - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to find missing edges in metabolic maps, especially for non-model organisms - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e.g. pathogens which have evolved novel p/ws - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to find different types of edges (e.g. regulation) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">gene-metabolite interactions: AMP kinase senses concentrations of AICAR, AMP and ATP - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">how do transcription and metabolism influence each other on a global scale? - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">paper: Bradley PH, Brauer MJ et al, PLoS Comp Bio, Jan 2009. <a style="text-decoration:none" href="http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000270">http://www.ploscompbiol.org/article/info:doi/10.1371/journal.pcbi.1000270</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">starve yeast for etiehr carbon or nitrogen, look at transcriptome and metabolomic response - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">SVD (singular value decomposition) suggests that transcr. and metab. responses are coordinated - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">4 broad classes of metabolites: glycolysis; TCA cycle; biosynthetic intermediates; amino acids - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/91c826bf-e717-4689-8dcb-fbc796b82e2d">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL37" >HL37</a></strong><br><strong><em>Classification, Evolution, and Assembly of Protein Complexes</em></strong><br> Wednesday, July 1 - 11:15 a.m. - 11:40 a.m.<br>Room: T2<br />Presenting author: Emmanuel Levy, Universite de Montreal, Canada<br>Presentation Overview: <br>A homomer is formed by self-interacting copies of a protein unit. This is functionally important, as in allostery, and structurally crucial because mis-assembly of homomers is implicated in disease. Homomers are widespread, with 50Ð70% of proteins with a known quaternary state assembling into such structures. Despite their prevalence, little is known about the mechanisms that drive their formation, both at the level of evolution and assembly in the cell. Here we present an analysis of over 5,000 unique atomic structures and show that the quaternary structure of homomers is conserved in over 70% of protein pairs sharing as little as 30% sequence identity. Where quaternary structure is not conserved, a detailed investigation revealed well-defined evolutionary pathways by which proteins transit between different quaternary structure types. Furthermore, we show by perturbing subunit interfaces within complexes and by mass spectrometry analysis, that the (dis)assembly pathway mimics the evolutionary pathway. These data represent a molecular analogy to Haeckel\'s evolutionary paradigm of embryonic development, where an intermediate in the assembly of a complex represents a form that appeared in its own evolutionary history. Our model of self-assembly allows reliable prediction of evolution and assembly of a complex solely from its crystal structure.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL37: Emmanuel Levy - Classification, Evolution, and Assembly of Protein Complexes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/aaacbf83-026d-40ac-aa26-2553816d91c4" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/aaacbf83-026d-40ac-aa26-2553816d91c4" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/aaacbf83-026d-40ac-aa26-2553816d91c4" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL38" >HL38</a></strong><br><strong><em>MeltDB: A software platform for the analysis and integration of Metabolomics Experiment Data</em></strong><br> Wednesday, July 1 - 11:45 a.m. - 12:10 p.m.<br>Room: T1<br />Presenting author: Heiko Neuweger, Bielefeld University, Germany<br>Presentation Overview: <br>The recent advances in metabolomics have created the potential to measure the levels of hundreds of metabolites which are the end products of cellular regulatory processes. The automation of the sample acquisition and subsequent analysis in high-throughput instruments that are capable of measuring metabolites is posing a challenge on the necessary systematic storage and computational processing of the experimental datasets. Whereas a multitude of specialized software systems for individual instruments and preprocessing methods exists, there is clearly a need for a free and platform-independent system that allows the standardized and integrated storage and analysis of data obtained from metabolomics experiments. Here, we present the web based and platform independent MeltDB systems that provides functionality to consistently store, organize and annotatethe datasets generated in metabolomics experiments. The system offersfunctionality for the preprocessing of mass spectrometry datasets in the file formats netCDF, mzXML and mzData. The results of the preprocessing are visualized and integrated within a functional genomics context and access to higher level statistical analysis are provided via the MeltDB web interface.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL38: Heiko Neuweger - MeltDB: A software platform for the analysis and integration of Metabolomics Experiment Data </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/fc38f610-89a0-4b3f-acfa-8ce01ecb2e4f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/fc38f610-89a0-4b3f-acfa-8ce01ecb2e4f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/fc38f610-89a0-4b3f-acfa-8ce01ecb2e4f" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL39" >HL39</a></strong><br><strong><em>Evolutionary potentials for protein structure and function prediction</em></strong><br> Wednesday, July 1 - 11:45 a.m. - 12:10 p.m.<br>Room: T2<br />Presenting author: Francisco Melo, P. Universidad Catolica de Chile, Chile<br>Presentation Overview: <br>We describe a new type of potentials for protein structure prediction, which are called \'evolutionary potentials\' (EvPs). In contrast to current potentials, which are derived from a set of non-redundant protein structures, the EvPs described here exploit the evolutionary record of all known proteins that adopt a specific fold.This study involved large-scale computations such as the structural comparison and clustering of the complete Protein Data Bank, the comparison at the sequence level of the non-redundant proteins against all known proteins at the NCBI database (about 7 million proteins), the building of 3D structure models for all proteins present in each multiple sequence alignment and the derivation of about 19,000 EvPs. The performance of EvPs was assessed for the task of fold assessment. It was demonstrated that EvPs outperform a typical representative potential and that the increase in performance is not a consequence of the amount of information retrieved.As an extension of the already published paper, recent results when using EvPs in the detection of distantly related proteins that would adopt a similar structure, as well as in the detection of key residues for protein function, will be presented for specific and biologically relevant example cases.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL39: Francisco Melo - Evolutionary potentials for protein structure and function prediction </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/f29e96c7-8e27-4fdf-b3bb-d6ea11101ff9" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f29e96c7-8e27-4fdf-b3bb-d6ea11101ff9" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f29e96c7-8e27-4fdf-b3bb-d6ea11101ff9" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/ruchira" target="_top" style="text-decoration:none">Ruchira S. Datta</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">protein structure prediction through: fold recognition and comparative modeling, or ab initio structure prediction - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">after making a model, its quality is assessed to decide whether to use it or discard it - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">usually have several iterations of comparative modeling - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">will speak about model quality assessment, an important ingredient in this process - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">below 40% sequence identity, have larger errors in predicted structure, mostly due to sequence alignment error - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">one way of dealing with this is to produce many alignments and use model quality assessment to assess the errors of the produced models - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">detect errors due to incorrect template and misalignment, which can occur with template-target identities of 20%, 25%, or 30% id - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">knowledge-based potentials, mean force potentials, or statistical potentials are scoring functions for model assessment - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">states are represented using geometrical descriptors - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">obtained scores represent pseudo-energies - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/f29e96c7-8e27-4fdf-b3bb-d6ea11101ff9">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL40" >HL40</a></strong><br><strong><em>Histone modifications at human enhancers reflect global cell-type-specific gene expression</em></strong><br> Wednesday, July 1 - 12:15 p.m. - 12:40 p.m.<br>Room: T1<br />Presenting author: Gary Hon, University of California, San Diego, United States<br>Presentation Overview: <br>Although it is known that gene expression is driven by promoters, enhancers, and insulators, the relative roles of these regulatory elements in this process are not clear. Here we identify these elements in multiple cell types and investigate their roles in cell-type-specific gene expression. We observed that the chromatin state at promoters and CTCF-binding at insulators is largely invariant across diverse cell types. In contrast, enhancers are marked with
highly cell-type-specific histone modification patterns, strongly correlate to cell-type-specific gene expression programs on a global scale, and are functionally active in a cell-type-specific manner. Our results define over 100,000 potential transcriptional enhancers in the human genome, significantly expanding the current catalogue of human enhancers and highlighting the role of these elements in cell-type-specific gene expression.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL40: Gary Hon - Histone modifications at human enhancers reflect global cell-type-specific gene expression </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/44e2da48-e14c-4483-98e3-2ef0a86a85d4" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/44e2da48-e14c-4483-98e3-2ef0a86a85d4" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/44e2da48-e14c-4483-98e3-2ef0a86a85d4" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Different cells are different because they express different genes. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Gene expression profiles from microarray studies can be used to identify cell types - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Regulation of transcription still not really understood. Coding regions are well studied, but most of the genome is non-coding. Presumably these regions contain regulatory elements. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Some regulatory elements are known. Promoters, Enhancers, Insulators. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Enhancers are the least understood elements in transcriptional regulation. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">How to find enhancers genome wide? - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Enhancers can be found by: sequence elements (>2500 TF motifs). - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">sequence based searches? >2500 transcription factor motifs... - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">can look at motifs, but they are highly degenerate - <a style="text-decoration:none" href="http://friendfeed.com/mikhail1" target="_top">Mikhail Spivakov</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">But such motives are short and highly degenerate. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/44e2da48-e14c-4483-98e3-2ef0a86a85d4">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL41" >HL41</a></strong><br><strong><em>Comparative analysis of crystal interfaces of homologous proteins</em></strong><br> Wednesday, July 1 - 12:15 p.m. - 12:40 p.m.<br>Room: T2<br />Presenting author: Roland Dunbrack, Fox Chase Cancer Center, United States<br>Presentation Overview: <br>Many proteins act as homooligomers. Often there is little direct evidence of what interfaces are present in the biologically active oligomer(s). Most such oligomers are derived from examination of crystal interfaces, but these are mostly hypothetical, based on surface area and observation of specific kinds of interactions. We have compared interfaces in crystals across different crystal forms of proteins, and correlated the existence of an interface in all or most crystal forms as evidence in favor of biological relevance.  We used previously published benchmarks as well as monomeric and oligomeric structures solved by solution NMR to validate this assumption. We find that presence of a similar interface in two or more crystals when sequence identity is less than 90% correlates highly with the benchmark data. The data indicate that of three publicly available sources of biological assemblies (PDB, PQS, and PISA) that PISA is most consistent with interfaces observed in large numbers of crystal forms. Comparative crystal analysis is better at identifying likely monomers, when there are large interfaces present in crystals which automated methods tend to identify as biologically relevant. The cytosolic sulfotransferases provide a particularly interesting example of this kind of analysis.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL41: Roland Dunbrack - Comparative analysis of crystal interfaces of homologous proteins </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/e03df29b-6e76-42f4-b034-2d23e387c175" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e03df29b-6e76-42f4-b034-2d23e387c175" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e03df29b-6e76-42f4-b034-2d23e387c175" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">two papers: Xu JMB 381, 487-507 2008: <a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/18599072">http://www.ncbi.nlm.nih.gov/pubmed/18599072</a> , missed 2nd one - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">homooligomers: 45-60% of all proteins in the PDB are homolooligomers - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">is a protein an oligomer in solution? - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">can determine using gel filtration, analytical ultracentrification, cross-linking + SDS-PAGE - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">sources of information: PDB/PQS/PISA - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">need to distinguish between asymmetric  and biological units - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Xu, Canutescu, Dunbrack, Bioinformatics, 2006: look at overlap of the 3 info sources - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">66% total intersection - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">see different annotations for the same protein family - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">shows an example where a dimer is formed by a small interface, the biounit dbs often don't get this correct - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/e03df29b-6e76-42f4-b034-2d23e387c175">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL42" >HL42</a></strong><br><strong><em>Exploring the human genome with functional maps</em></strong><br> Wednesday, July 1 - 2:15 p.m. - 2:40 p.m.<br>Room: T1<br />Presenting author: Curtis Huttenhower, Princeton University, United States<br>Presentation Overview: <br>Human genomic data of many types are readily available, but the complexity and scale of human molecular biology make it difficult to integrate this body of data, understand it from a systems level, and apply it to the study of specific pathways or genetic disorders. An investigator could best explore a particular protein, pathway, or disease if given a functional map summarizing the data and interactions most relevant to his or her area of interest. Using a regularized Bayesian integration system, we provide maps of functional activity and interaction networks in over 200 areas of human cellular biology, each including information from ~30,000 genome-scale experiments. Key to these analyses is the ability to efficiently summarize this large data collection from a variety of biologically informative perspectives: prediction of protein function and functional modules, cross-talk among biological processes, and association of novel genes and pathways with known genetic disorders. Experimental investigation of five specific genes (AP3B1, ATP6AP1, BLOC1S1, LAMP2 and RAB11A) has confirmed novel roles for these proteins in the proper initiation of macroautophagy in human fibroblasts. Our functional maps can be explored using HEFalMp, a web interface allowing interactive visualization and investigation of this large body of information.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL42: Curtis Huttenhower - Exploring the human genome with functional maps </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/bbbc0000-1589-4046-9a6a-984335caade1" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/bbbc0000-1589-4046-9a6a-984335caade1" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/bbbc0000-1589-4046-9a6a-984335caade1" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/woodydon" target="_top" style="text-decoration:none">Kuan-Ting Lin</a>, <a href="http://friendfeed.com/setar" target="_top" style="text-decoration:none">Oliver Hofmann</a> and <a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">HEFalMp: data integration for human genomic data. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Integration of genomic data and prior knowledge using bayesian integration. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Each resulting network is based on a specific functional context. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Input data is huge: 10,000s of experiments, billions of data points. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">But also very sparse. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Our prior knowledge comes from GeneOntology - <a style="text-decoration:none" href="http://friendfeed.com/satagopam" target="_top">Venkata P. Satagopam</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Human case sources: genomic data (interactions, microarrays, sequences); prior knowledge (229 biological processes from GO and KEGG). - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">There is so much information about humans that the independence assumption at the base of naive bayesian classification is violated. Used mutual information for regularization. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">start with predicted gene or gene product interaction network - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Functional mapping methodology. Start from gene interaction network. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/bbbc0000-1589-4046-9a6a-984335caade1">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL43" >HL43</a></strong><br><strong><em>Disordered flanks prevent peptide aggregation</em></strong><br> Wednesday, July 1 - 2:15 p.m. - 2:40 p.m.<br>Room: T2<br />Presenting author: Sanne Abeln, FOM Institute for Atomic and Molecular Physics [AMOLF], Netherlands<br>Presentation Overview: <br>In their natural cellular environment proteins are dissolved in a concentrated aqueous solution of biomolecules. Even under such crowded conditions, proteins must not aggregate; aggregates may be cytotoxic or compromise the biological function of the peptide. Evolutionary pressure generally ensures that proteins do not aggregate in their natural biochemical environment. A well-known mechanism to prevent aggregation is the folding of proteins. Here we report a different mechanism that can prevent the aggregation of proteins. Recently, it was discovered that many proteins contain regions that are disordered (not folded) in their natural environment. We show with coarse-grained simulations that embedding small hydrophobic binding motifs in disordered regions can prevent aggregation: the disordered regions of different proteins sterically hinder the formation of aggregates. Moreover, our simulations show that the disordered regions have no adverse effect on the biological (signalling) function of the binding motifs, because they do not obstruct the binding and folding of the binding motif on its specific substrate.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL43: Sanne Abeln - Disordered flanks prevent peptide aggregation </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/cc220c7b-6572-4ee6-b129-cb284c7913ca" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/cc220c7b-6572-4ee6-b129-cb284c7913ca" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/cc220c7b-6572-4ee6-b129-cb284c7913ca" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL44" >HL44</a></strong><br><strong><em>Benchmarking tools in Metabolic Pathway Analysis</em></strong><br> Wednesday, July 1 - 2:45 p.m. - 3:10 p.m.<br>Room: T1<br />Presenting author: Luis de Figueiredo, Friedrich-Schiller-UniversitŠt Jena, Germany<br>Presentation Overview: <br>Metabolic Pathway Analysis is a growing field within Systems Biology. For thereconstruction and prediction of metabolic pathways, the concept of elementaryflux modes has turned out to be very useful. It takes into accountstoichiometry and mass balance at steady state not only for monomolecularreactions but also for reactions of higher molecularity. Alternative approacheshave been proposed, which are based on graph theory and neglect the mass balanceof co-substrates and byproducts. Here, we present three benchmark examples bywhich pathway analysis tools can be compared. They concern the question whethereven-chain fatty acids can be converted into sugars in animals, whetherMycoplasma hominis can convert glucose into pyruvate and whether human redblood cells can salvage hypoxanthine. Moreover, new results and directions willbe given for improving stoichiometry-based tools in order to deal withgenome-scale metabolic networks.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL44: Luis de Figueiredo - Benchmarking tools in Metabolic Pathway Analysis </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/e4a30192-7d59-41e0-9c3a-16c7496111f7" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e4a30192-7d59-41e0-9c3a-16c7496111f7" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e4a30192-7d59-41e0-9c3a-16c7496111f7" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">how to represent a metabolic system?  it's a thermodynamically open system, we need to draw a boundary around it - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">we approximate that the system reaches a steady state - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">certain metabolites form a pooll and are removed from the representation of the steady state - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Metabolic system can be represented as stoichiometric matrix - <a style="text-decoration:none" href="http://friendfeed.com/satagopam" target="_top">Venkata P. Satagopam</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">get the stoichiometric matrix; the solution set is a polyhedral convex cone.  similarly if using other constrained representations - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Elementary Flux Mode: minimal set of enzymes that can operate at steady state - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">See also here: <a style="text-decoration:none" href="http://ff.im/4vaOK">http://ff.im/4vaOK</a> - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">1st bench mark system - conversion of fatty acids to carbohydrate - <a style="text-decoration:none" href="http://friendfeed.com/satagopam" target="_top">Venkata P. Satagopam</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">how to convert fatty acids to carbohydrate?  Weinman et al 1957 showed a certain pathway is not possible - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">model used by Weinman et al in 1957 - <a style="text-decoration:none" href="http://friendfeed.com/satagopam" target="_top">Venkata P. Satagopam</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/e4a30192-7d59-41e0-9c3a-16c7496111f7">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL45" >HL45</a></strong><br><strong><em>From the detection of functional regions towards function annotation in proteins</em></strong><br> Wednesday, July 1 - 2:45 p.m. - 3:10 p.m.<br>Room: T2<br />Presenting author: Nir Ben-Tal, Tel Aviv University, Israel<br>Presentation Overview: <br>The identification of functional regions in proteins may aid in function annotation, mutation analysis and drug discovery. In the talk I will present PatchFinder, a method for the identification of functionally important regions in proteins with known three-dimensional structure. The method is based on the assumption that these regions are often evolutionarily conserved in order to retain the effectiveness of the protein. My colleagues and I compiled the N-Func database of 757 proteins of unknown function, whose structure is known, and used PatchFinder to predict their functional regions. In some cases we suggested what the function might be. N-Func and PatchFinder are available as a webserver.Obviously, some of N-Func\\\'s protein bind DNA. In my talk I will also present a new method that we developed for discrimination between proteins that do and do not bind DNA. The method is based on various characteristics of the protein and the (predicted) functional region. We used it to predict DNA-binding proteins in the N-Func database.   <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL45: Nir Ben-Tal - From the detection of functional regions towards function annotation in proteins </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/77344754-0fa2-4793-9bb7-64541318f0bc" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/77344754-0fa2-4793-9bb7-64541318f0bc" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/77344754-0fa2-4793-9bb7-64541318f0bc" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL46" >HL46</a></strong><br><strong><em>Prioritizing functional modules mediating genetic perturbations and phenotypic effects</em></strong><br> Wednesday, July 1 - 3:15 p.m.- 3:40 p.m.<br>Room: T1<br />Presenting author: Li Wang, University of Southern California, United States<br>Presentation Overview: <br>How variation in DNA leads to variation in phenotypes is a question open to interpretation. Here we present a global strategy based on the Bayesian network framework to prioritize the functional modules mediating genetic perturbations and their phenotypic effects among a set of overlapping candidate modules. We take lethality in Saccharomyces cerevisiae and human cancer as two examples to show the superiority of this approach over the traditional hypergeometric enrichment test, which ignores the interrelationships among modules. <br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL46: Li Wang - Prioritizing functional modules mediating genetic perturbations and phenotypic effects </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/9865447a-f9cd-4699-a11b-acf8ddee57b2" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9865447a-f9cd-4699-a11b-acf8ddee57b2" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9865447a-f9cd-4699-a11b-acf8ddee57b2" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">validation: ortholog lethal ration: the proportion of yeast genes whose orthologs in another species, specifically, C. elegans, are lethal - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">hypothesis about ortholog lethal ratio: nonlethal genes in lethal complexes > nonlethal genes in nonlethal complexes - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">hypergeometric enrichment test was not significantly different, but the Bayesian network model was validated.  they think this is because the hypergeometric test produced many false positives - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">gene lethality is more conserved at the module level - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">use module-based mapping to transfer phenotypic mapping across species, rather than gene-based - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL47" >HL47</a></strong><br><strong><em>Fitting multiple components into a cryoEM map of their assembly</em></strong><br> Wednesday, July 1 - 3:15 p.m.- 3:40 p.m.<br>Room: T2<br />Presenting author: Keren Lasker, Tel Aviv University, Israel<br>Presentation Overview: <br>Models of macromolecular assemblies are essential for a mechanistic description of cellular processes. Such models are increasingly obtained by fitting atomic-resolution structures of components into a density map of the whole assembly. Yet, current density-fitting techniques are frequently insufficient for an unambiguous determination of the positions and orientations of all components. In the first part of the talk, we will describe MultiFit, a method for simultaneously fitting atomic structures of components into their assembly density map at resolutions of as low as 25 Å. In MultiFit, the positions and orientations of the components are optimized with respect to a scoring function that includes the quality-of-fit of components in the map, the protrusion of components from the map envelope, and the shape complementarity between pairs of components. The scoring function is optimized by our new exact inference optimizer DOMINO (Discrete Optimization of Multiple INteracting Objects) that efficiently finds the global minimum in a discrete sampling space.  In the second part of the talk, we will demonstrate the utility of MultiFit for modeling the configuration of large macromolecular assemblies.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL47: Keren Lasker - Fitting multiple components into a cryoEM map of their assembly </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/aa1c7565-05fa-47b2-b130-5dbc08e1cb12" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/aa1c7565-05fa-47b2-b130-5dbc08e1cb12" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/aa1c7565-05fa-47b2-b130-5dbc08e1cb12" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/neilfws" target="_top" style="text-decoration:none">Neil Saunders</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">cryoEM has become a standard tool for structural characterization of large protein complexes - <a style="text-decoration:none" href="http://friendfeed.com/anniina" target="_top">Anne Tuukkanen</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Complex modeling by using em maps and minimizing an objective function. The objective function includes terms for geometric complementary, a fitting score and term for envelope penetration. - <a style="text-decoration:none" href="http://friendfeed.com/anniina" target="_top">Anne Tuukkanen</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL48" >HL48</a></strong><br><strong><em>Architecture of CpG methylation in the human genome</em></strong><br> Wednesday, July 1 - 3:45 p.m. - 4:10 p.m.<br>Room: K2<br />Presenting author: Israel Steinfeld, Technion - Israel Institute of Technology, Israel<br>Presentation Overview: <br>Constitutively unmethylated regions in the genome significantly contribute to open chromatin domains within a sea of global transcriptional repression. This constitutive unmethylated status is commonly thought to be directly associated to the density of CpG dinucleotides. We will present data and analysis results from genome-wide CpG-island methylation profiling of multiple human tissue samples. We will show how unmethylated regions (UMRs) seem to be formed during early embryogenesis, not as a result of CpG-ness, but rather through the recognition of specific sequence motifs closely associated with transcription start sites. We will describe the computational methods used in the analysis, including methylation status calling, motif discovery, GO enrichment and machine learning techniques. We will introduce a new class of nonpromoter UMRs that become de novo methylated in a tissue-specific manner during development and present experimental validation of our findings. In short, we show that UMRs influence genome structure and have a dynamic role in development.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL48: Israel Steinfeld - Architecture of CpG methylation in the human genome </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/a5b8d7e8-f53f-41b3-9e29-57b83e8c0f6c" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/a5b8d7e8-f53f-41b3-9e29-57b83e8c0f6c" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/a5b8d7e8-f53f-41b3-9e29-57b83e8c0f6c" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">DNA methylation: modification of cytosines in CpG dinucleotides, maintained across cell divisions - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">CG dinucleotide content in HG: 1%, expected: 4.5% - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">CpG islands: regions on DNA that contain many CpGs. 28000 islands annotated in HG. almost all of them are near gene promoters - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">mDIP: methyl-DNA immunoprecipitation assay, similar to ChIP-chip. 244k DNA methylation array - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">array methylation score (IMS): average signal for all probes mapped to it. bimodal distribution. house keeping genes are methylated (ie, on one side of the distribution) - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">approx  15 samples (different tissues). almost all are not methylated (~70%) - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Nature: Sp1 elements protect a CpG island from de novo methylation, Michael Brandeis et al, Nature 371, September 1994 - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">DRIM Discovering Rank Imbalanced Motifs: <a style="text-decoration:none" href="http://bioinfo.cs.technion.ac.il/drim">http://bioinfo.cs.technion.ac.il/drim</a> - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">use machine learning to distinguish between methyl. and nonmethyl. islands. - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">UMR: undermethylated region (?) - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/a5b8d7e8-f53f-41b3-9e29-57b83e8c0f6c">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL49" >HL49</a></strong><br><strong><em>Robust simplifications of multiscale biochemical networks</em></strong><br> Wednesday, July 1 - 3:45 p.m. - 4:10 p.m.<br>Room: T1<br />Presenting author: Andrei Zinovyev, Institut Curie, France<br>Presentation Overview: <br>Model reduction, i.e. simplifying complex models to simpler ones, preserving some important (dynamical) features is a necessary technique in almost all projects on biochemicalnetworks modeling, but unfortunately there is no yet solid methodology which can be systematically appliedfor systems biology models.There exist several model reduction directions, connected with time scale separation.A textbook notion of the \'limiting\' reaction can be applied to few simple networkstructures. In this paper we show how the notion of the \'limiting\' reaction rate can be applied forlarge and complex networks.The general view that we develop is that at a particular time window, only a small dominant subsetof reactions determine the behaviour of the biochemical model, but this subset can change with time.We develop an algorithm of model reduction in linear networks, for which the dominant system is unique. We show that the case of hierarchical linear biochemical networks is the only one when the topology of the network determinescompletely its dynamical features.We develop a notion of the dominant subsystem for non-linear networks and demonstrate howit can be applied for simplifying a complex model of NFkB signalling. Our approach generatesa hierarchy of simplified models some levels of which can be compared with existing models of NFkB signalling.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL49: Andrei Zinovyev - Robust simplifications of multiscale biochemical networks </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/afc57a60-6d9b-4c0a-9442-71b93049a1a8" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/afc57a60-6d9b-4c0a-9442-71b93049a1a8" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/afc57a60-6d9b-4c0a-9442-71b93049a1a8" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">model reduction: simplify models in order to understand - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e.g., "Consider a spherical cow..." A farmer hires a physicist to help with milk production, who takes  half a year to produce a paper beginning thusly. - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">complexity of data and complexity of models: non-identifiable models still have robust properties - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">See e.g. Chen et al, Molecular Systems Biology 5:239, 2009 - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">given only the order relations between the model parameters, can we provide robust first and second order solutions? - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">biological systems are hierarchical and multiscale (an observation, not a theorem) - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">structure: functional modules, motifs; scales: time scales, concentratio scales - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">this makes it possible to neglect small quantities in favor of larger ones, given proper theory - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">aymptotic approximations in chemical kinetics: quasi-equilibrium (fast reactions), quasi-steady state, ..., quasistationary - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">enzymatic catalysis in quasiequilibrium vs quasistationary approximations gives very different results - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/afc57a60-6d9b-4c0a-9442-71b93049a1a8">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL50" >HL50</a></strong><br><strong><em>Rapid sampling of molecular motion with prior information constraints: Insights into channel gating and domain swapping</em></strong><br> Wednesday, July 1 - 3:45 p.m. - 4:10 p.m.<br>Room: T2<br />Presenting author: Ora Schueler-Furman, The Hebrew University of Jerusalem, Israel<br>Presentation Overview: <br>Proteins are active, flexible machines that perform a range of different functions. Innovative experimental approaches may now provide limited partial information about conformational changes along motion pathways of proteins. There is therefore a need for computational approaches that efficiently incorporate prior information into motion prediction schemes. We present PathRover, a framework designed for the integration of prior information into the motion-planning algorithm of Rapidly-exploring Random Trees (RRT). Each suggested motion pathway comprises a sequence of low-energy, clash-free conformations that satisfy a number of prior information constraints, derived from experimental data or from expert intuition. The incorporation of prior information in an outright fashion narrows down the vast search in the typically high-dimensional conformational space, leading to dramatic reduction in running time. Hybridization of low-energy pathways is then performed using a novel algorithm for the efficient alignment and comparison of molecular motion pathways (similar to string matching algorithms). The suggested framework can serve as an effective, complementary tool for Molecular Dynamics, Normal Mode Analysis, and other prevalent techniques for predicting motion in proteins. We used PathRover to explore in detail molecular motions of domain swapping, substrate binding, and ion channel gating.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL50: Ora Schueler-Furman - Rapid sampling of molecular motion with prior information constraints: Insights into channel gating and domain swapping </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/8f5d4547-54ea-4c93-ab48-af0bb1412d9c" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8f5d4547-54ea-4c93-ab48-af0bb1412d9c" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8f5d4547-54ea-4c93-ab48-af0bb1412d9c" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL51" >HL51</a></strong><br><strong><em>GraphWeb: functional analysis of genomic networks</em></strong><br> Thursday, July 2 - 10:45 a.m. - 11:10 a.m.<br>Room: Victoria Hall<br />Presenting author: JŸri Reimand, University of Tartu, Estonia<br>Presentation Overview: <br>Deciphering heterogeneous cellular networks of transcriptional regulation, protein interactions and metabolism is a great challenge of current systems biology. Such networks contain modules of interacting genes and proteins that potentially share regulatory mechanisms and common function. Here, we present GraphWeb (http://biit.cs.ut.ee/graphweb/), a web server for network analysis and module discovery. GraphWeb (i) integrates heterogeneous and multispecies data into networks; (ii) discovers topological network modules and (iii) interprets the modules using Gene Ontology, pathways, regulatory motifs and microRNA targets.
<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL51: Jüri Reimand - GraphWeb: functional analysis of genomic networks </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/67e757e0-897b-4792-8362-f47635e9523b" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/67e757e0-897b-4792-8362-f47635e9523b" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/67e757e0-897b-4792-8362-f47635e9523b" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">A tool for large, genome scale networks is applied to yeast data. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The modular nature of molecular cell biology can be seen in today's large networks. A broad overview over the different, complex networks. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Many biological entities can be seen as networks, e.g. proteins and genes, measured by DNA binding, physical and genetic interactions, which can be combined into heterogenous networks. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Edges are not born equal, we need edge-weights, easy in gene expression networks. SAGA and SWI/SNF example. Assign local weights to rank edges in the network - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Data set are not born equal. Small scale data set are more reliable than global approaches. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top"># Not sure I agree fully here, but he is merely motivating the weights approach. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Some data set are trusted more than others. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Different scores for global and local weights can be combined. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Hairball. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Everything is interconnected, no way to interpret. Needs to be dissected. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/67e757e0-897b-4792-8362-f47635e9523b">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL52" >HL52</a></strong><br><strong><em>Characterizing transcriptome plasticity using whole-genome tiling arrays and machine learning</em></strong><br> Thursday, July 2 - 10:45 a.m. - 11:10 a.m.<br>Room: T2<br />Presenting author: Georg Zeller, Friedrich Miescher Laboratory of the Max Planck Society, Germany<br>Presentation Overview: <br>Currently, whole-genome tiling arrays are still a cost-effective technology to quantitatively monitor transcriptomes. We present machine learning methods for data normalization and de novo transcript identification and show substantially improved accuracy of the resulting predictions compared to competing methods. Application to Arabidopsis tiling array data revealed thousands of new transcripts missing in current annotations, including ones with a stress-dependant expression pattern. We moreover characterized the transcriptomes of mutants impaired in various steps of RNA processing.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL52: Georg Zeller - Characterizing transcriptome plasticity using whole-genome tiling arrays and machine learning </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/694e6c9e-4fbe-40d4-bdf7-1450b6a8d463" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/694e6c9e-4fbe-40d4-bdf7-1450b6a8d463" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/694e6c9e-4fbe-40d4-bdf7-1450b6a8d463" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/neilfws" target="_top" style="text-decoration:none">Neil Saunders</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Motivation: find all genes in the model plant Arabidopsis thaliana. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">profile transcriptome changes under stress - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">tiling arrays: cost-effective, but noisy - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">normalization pipeline: background correction to reduce imaging artifacts - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">quantile normalization: comparison between arrays - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Normalization: background correction, quantile normalization between arrays, transcript normalization (to remove probe sequence bias) - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">transcript normalization: probe sequence bias reduced - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Not all exon probes show the same level. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">assume that ideal transcripts have constant level of expression, try to predict deviation between actual and ideal signal intensities - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Sequence-based quantile normalization: exon and intron sequences are normalized to 2 distinct levels - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/694e6c9e-4fbe-40d4-bdf7-1450b6a8d463">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL53" >HL53</a></strong><br><strong><em>Promoting coherent minimum reporting guidelines for biological and biomedical investigations: the MIBBI project</em></strong><br> Thursday, July 2 - 10:45 a.m. - 11:10 a.m.<br>Room: T1<br />Presenting author: Chris Taylor, EMBL-EBI, United Kingdom<br>Presentation Overview: <br>Minimum information checklists specify the information that should be provided when reporting research. They promote transparency and data accessibility, and support more thorough quality assessment, increasing the value of data set, and by extension the competitiveness of the originators and the host database(s). However, with no mechanisms to coordinate checklist development, to establish the number of extant checklists and to track their evolution were both challenging exercises. Furthermore, overlaps in scope between checklists and arbitrary decisions on wording and structure almost guaranteed significant incompatibilities.Consequently, representatives of several checklist development projects began the MIBBI (Minimum Information for Biological and Biomedical Investigations) project (http://mibbi.org/). MIBBI has two broad goals: To provide access to checklists and their developers (a Ôone-stop shopÕ), and to foster the development of new, integrated checklist ÔsuiteÕ by the participant communities.Since publication last year the MIBBI project has found increasing favour with both journals and funders. The Portal now lists twenty-nine projects and the Foundry is generating new modules. Tools are also emerging to help researchers follow MIBBI guidelines. Overall, we have much of interest to the bioinformatics community: as significant consumers of research data, they stand to benefit greatly from the standardisation that MIBBI is encouraging.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL53: Chris Taylor - Promoting coherent minimum reporting guidelines for biological and biomedical investigations: the MIBBI project </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/098259e2-ae7e-454e-ada9-7d54db04a70c" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/098259e2-ae7e-454e-ada9-7d54db04a70c" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/098259e2-ae7e-454e-ada9-7d54db04a70c" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Standards are hugely dependent on their respective communities - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Much of biological experiments can fall under an ISA (Investigation-Study-Assay) structure. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">You should then use three types of standards: syntax (FuGE, ISA-TAB etc), semantics, and scope. MIBBI is all about scope. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Why do we care about standards? Data exchange, comprehensibility, and scope for reuse. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">"Metaprojects": FuGE, OBI, ISA-TAB - draw together many different domains and present in structure/semantics useful across all. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">When the independent MI projects overlap, arbitrary decisions on wording and substructuring make integration difficult. This makes it hard to take parts of different guidelines - not very modular. This is what MIBBI helps with. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">MIBBI promotes gradual integration of checklists. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Nature Biotechnology 26, 889 - 896 (2008)
doi:10.1038/nbt0808-889 - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Blog post: <a style="text-decoration:none" href="http://themindwobbles.wordpress.com/2009/07/02/hl53-promoting-coherent-minimum-reporting-guidelines-for-biological-and-biomedical-investigations-the-mibbi-project-ismb-2009/">http://themindwobbles.wordpress.com/2009/07/02/hl53-promoting-coherent-minimum-reporting-guidelines-for-biological-and-biomedical-investigations-the-mibbi-project-ismb-2009/</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL54" >HL54</a></strong><br><strong><em>Search and discovery of recurring patterns with interactomes</em></strong><br> Thursday, July 2 - 11:15 a.m. - 11:40 a.m.<br>Room: Victoria Hall<br />Presenting author: Mona Singh, Princeton University, United States<br>Presentation Overview: <br>Searching for recurring patterns in biological data has been the backbone of much research and analysis in bioinformatics. For example, within the realm of sequence analysis, the search for recurring or similar patterns has given rise to extensive work on sequence alignments and sequence motif discovery, and has resulted in large sequence motif libraries. Our goal is to begin to develop the analogous techniques for biological networks, by building a framework for searching and mining interaction networks in order to reveal and systematize the recurring protein and interaction patterns within them.I will describe:(1) Our formalism--network schemas--for representing recurring patterns within interactomes.  Network schemas specify descriptions of proteins (e.g., their molecular functions or putative domains) and the topology of interactions among them. They can describe domain-domain interactions, signaling and regulatory pathways or more complex network patterns.(2) Our fast algorithms for searching for user-supplied network schemas in arbitrary biological networks.(3) Our framework for systematically uncovering recurring, over-represented schemas in physical interaction networks.(4) An application of our methods to the yeast and human interactomes, where we identify hundreds of recurring and over-represented network schemas of various complexity and provide several lines of evidence of their functional importance.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL54: Mona Singh - Search and discovery of recurring patterns with interactomes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/f188c94b-2227-4a71-aead-ad7f5312bb18" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f188c94b-2227-4a71-aead-ad7f5312bb18" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f188c94b-2227-4a71-aead-ad7f5312bb18" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/neilfws" target="_top" style="text-decoration:none">Neil Saunders</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Hairballs. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Different large scale data sets, genetic, phosphorylation etc. exist but how to interpret? - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Add protein annotatons, sequence, structure, motifs, domains, functional characterization. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Particular interested in interaction domains, analyzing cellular organization and interactomes. Can we discover and analyze recurring patterns. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Analogy to multiple alignment of protein sequences with PROSITE pattern - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">For networks schemas, nodes are description of proteins, e.g. domains. An extension of network by attributes, e.g. PFAM or PROSITE. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Examples for network schema are homologous pathways. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Netgrep has visual interface. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">No details of the algorithm, tricky problem but can solve it fast. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Triangles, quad topology (linear combination of four), Y star topology of four. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/f188c94b-2227-4a71-aead-ad7f5312bb18">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL55" >HL55</a></strong><br><strong><em>The Computational Exploration of (Alternative) Splicing Mechanisms</em></strong><br> Thursday, July 2 - 11:15 a.m. - 11:40 a.m.<br>Room: T2<br />Presenting author: Michael Sammeth, CRG/IMIM/UPF, Spain<br>Presentation Overview: <br>The recent advent of high-throughput methods that allow for sequencing the whole RNA complement of cell populations has given a first impression on the amount of novel exons and spliceforms that is expected to be added to transcriptome annotations over the next years. In this respect alternative splicing (AS) is more and more coming to the fore as the molecular mechanism that is mainly responsible for creating the plethora of different combinations Ð which finally account for organsim complexity Ð from the limited reservoir of genetically inherited information. In order to keep step with the flood of data, we have developed a generic model of AS based on an universal defenition of its atomary unit called \\\\\\\'event\\\\\\\' and a method which can efficiently retrieve all such events from huge datasets (i.e., millions of annotations) containing a high level of noise (i.e., sequencing errors). In the highlighted publication we focus on comparing these generic events to the ones that have usually been considered in computational analyzes across 12 different species. Additionally, we now show several examples in which characterization of such events provides bona fide propositions for kinetic models to explain the molecular mechanism of splicing in general.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL55: Michael Sammeth - The Computational Exploration of (Alternative) Splicing Mechanisms </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/bcd5f9ec-dc61-4e04-a50b-5484c4820889" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/bcd5f9ec-dc61-4e04-a50b-5484c4820889" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/bcd5f9ec-dc61-4e04-a50b-5484c4820889" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/mn3" target="_top" style="text-decoration:none">Mitsuteru Nakao</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Numbering all possible splice variant sites, assign ASCII symbol to each site, "alternative splicing code" that captures all differences between alternative splicing variation - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ASTALAVISTA web service generates these strings at <a style="text-decoration:none" href="http://genome.imim.es/astalavista">http://genome.imim.es/astalavista</a> - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ESTs can be truncated, and still splicing events can be detected as a sub-structure - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">"Bubbles" are cyclic graphs, complete events of splicing. Can construct bubble hierarchies for very complex splicing loci. - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Workflow: Gene annotation -> database of events -> visualization with ASTALAVISTA; optionally add RNA-Seq data for expression levels of events (even simulated data from the Flux Capacitator) - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL56" >HL56</a></strong><br><strong><em>Nominalization and alternations in the language of molecular biology: Implications for text mining</em></strong><br> Thursday, July 2 - 11:15 a.m. - 11:40 a.m.<br>Room: T1<br />Presenting author: Kevin Cohen, University of Colorado School of Medicine, United States<br>Presentation Overview: <br>We present data on the understudied phenomenon of nominalization in the language of molecular biology and demonstrate its implications for the design of biomedical text mining systems.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL56: Kevin Cohen - Nominalization and alternations in the language of molecular biology: Implications for text mining </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/b03a7e02-b5d9-4d64-8096-d6aafe358507" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b03a7e02-b5d9-4d64-8096-d6aafe358507" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b03a7e02-b5d9-4d64-8096-d6aafe358507" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">nominalizations are dominant in biomedical texts, e.g. "expression" is much more used than "express" - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Cohen et al, 2008 [not a very useful reference]: nominalizations are more difficult to handle than verbs, but can yield higher precision - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top"><a style="text-decoration:none" href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0003158">http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0003158</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">alternation: variations like active/passive. much less characterized for nouns than for verbs - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e.g. pre-nominal arguments: agent: "phenobarbital treatment" or patient: "cancer treatment" - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">previous work has tried to handle nominalizations, e.g. Ono et al (2001): interactions, association, complex and binding - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">genescene: <a style="text-decoration:none" href="http://doi.ieeecomputersociety.org/10.1109/JCDL.2003.10011">http://doi.ieeecomputersociety.org/10.1109/JCDL.2003.10011</a> [?} - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">use in-house tool to do annotation of nominalizations - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">alternations are very diverse, contrary to previous prediction that there would only be a limited number of alternations in scientific literature (sub-language model) - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL57" >HL57</a></strong><br><strong><em>FunCoup: global networks of functional coupling in eukaryotes</em></strong><br> Thursday, July 2 - 11:45 a.m. - 12:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Erik Sonnhammer, Stockholm Bioinformatics Centre, Sweden<br>Presentation Overview: <br>Interactomes computationally predicted via data integration are becoming an increasingly popular tool and context for biological research. However merging disparate data sources and presenting relevant parts of a global network is not trivial.� FunCoup, an optimised Bayesian framework and a web resource, was developed to resolve these issues. Because interactomes comprise functional coupling of many types, FunCoup annotates network edges with confidence scores in support of different kinds of interactions � physical interaction, protein complex member, metabolic or signalling link. This capability boosted overall accuracy. On the whole, the constructed framework was comprehensively tested to optimise the overall confidence and ensure seamless, automated incorporation of new data sets of heterogeneous types. Using over 50 datasets in seven organisms, and extensively transferring information between orthologs, FunCoup predicted global networks in eight eukaryotes. For the Ciona intestinalis network only orthologous information was used, and it recovered a significant number of experimental facts. FunCoup predictions were validated on independent cancer mutation data. The networks, which are the largest interactome reconstructions to date, are freely available for download and query at http://FunCoup.sbc.su.se. The site allows detailed graphical and tabular analysis of subnetworks around query genes, as well as comparative analysis of orthologous networks in multiple species.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL57: Erik Sonnhammer - FunCoup: global networks of functional coupling in eukaryotes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/185f2a01-a0cc-4634-828d-ef7207566dbe" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/185f2a01-a0cc-4634-828d-ef7207566dbe" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/185f2a01-a0cc-4634-828d-ef7207566dbe" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">How to reconstruct networks - experimental networks are incomplete. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Up to 300.000 interactions are proposed for human - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">only 35000 known - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Each experimental method can give you more than 20% of the interactions. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">experiments have high false negative and false positive rates - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e.g., false positives from in vitro experiments: the interaction may never happen in a living cell - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Interactions have to be combined and evaluated. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">FunCoup - <a style="text-decoration:none" href="http://friendfeed.com/satagopam" target="_top">Venkata P. Satagopam</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">there are many kinds of evidence for functional coupling - <a style="text-decoration:none" href="http://friendfeed.com/ruchira" target="_top">Ruchira S. Datta</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Lots of evidence for functional coupling, not only from PPI but als from localization, gene expresson, interacting domain, TFBS; miRNAs. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/185f2a01-a0cc-4634-828d-ef7207566dbe">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL58" >HL58</a></strong><br><strong><em>A new method for high-resolution gene expression analysis</em></strong><br> Thursday, July 2 - 11:45 a.m. - 12:10 p.m.<br>Room: T2<br />Presenting author: Caroline Friedel, Ludwig Maximilians - University Munich, Germany<br>Presentation Overview: <br>We present a novel approach for measuring both RNA transcription and decay in a single experimental setting. We show that this approach increases the sensitivity for differentially expressed genes and temporal kinetics of transcriptional regulation. Furthermore, alterations in transcription and decay can be distinguished and rates of RNA turnover can be determined with superior accuracy. This provides new insights into gene regulation and is important for quantitative systems biology modelling.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL58: Caroline Friedel - A new method for high-resolution gene expression analysis </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/a2226131-1670-4c49-908d-d339e708c5b9" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/a2226131-1670-4c49-908d-d339e708c5b9" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/a2226131-1670-4c49-908d-d339e708c5b9" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/diriano" target="_top" style="text-decoration:none">Diego M. Riaño-Pachón</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Metabolic tagging of newly transcribed RNA - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Able to measure decay and de novo synthesis - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">this methods tries to solve a long standing problem in transcriptomics, ] - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">i.e., bias for differential expression of short-lived RNAs - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">and thus in subsequent analysis - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">solution is to perform gene expression analysis of newly transcribed RNA - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">using 4-thiouridine - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Kenzelmann et al PNAS 2007 <a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/17405863">http://www.ncbi.nlm.nih.gov/pubmed/17405863</a> - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">after tagging,normal analysis techniques for microarray or RNA-seq can be sused - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">bur of course new methods could exploit features specific for the tagging of newly transcribed RNA - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/a2226131-1670-4c49-908d-d339e708c5b9">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL59" >HL59</a></strong><br><strong><em>Comparative community assessments for applied biomedical text mining: BioCreative II challenge and metaservices.</em></strong><br> Thursday, July 2 - 11:45 a.m. - 12:10 p.m.<br>Room: T1<br />Presenting author: Florian Leitner, Spanish National Cancer Research Centre (CNIO), Spain<br>Presentation Overview: <br>Comparative evaluation of computational tools applied to biological data is crucial to enable monitoring improvements over time and identify competitive strategies, while promoting the tools\' availability through service encapsulation and unification supports their usage. In the interest of biomedical text mining, the BioCreative initiative has contributed substantially to the assessment of text mining tools applied to biologically relevant tasks. The second BioCreative initiative not only had a considerable impact in the development of text mining systems for the extraction of biological entities and protein interactions, but also motivated the implementation of the first text mining meta-server for biology, the BioCreative MetaServer (BCMS). The BCMS is a key infrastructure for the upcoming BioCreative II.5 event, where efforts of text mining developers, biological annotation databases, article authors and publishers contribute to the improvement of information access to full text articles.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL59: Florian Leitner - Comparative community assessments for applied biomedical text mining: BioCreative II challenge and metaservices. </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/378e5841-f0f5-464f-82e1-fafef1f7466f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/378e5841-f0f5-464f-82e1-fafef1f7466f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/378e5841-f0f5-464f-82e1-fafef1f7466f" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL60" >HL60</a></strong><br><strong><em>Studying alternative splicing regulatory networks through partial correlation analysis</em></strong><br> Thursday, July 2 - 12:15 p.m. - 12:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Liang Chen, University of Southern California, United States<br>Presentation Overview: <br>Alternative pre-mRNA splicing is an important gene regulation mechanism for expanding proteomic diversity in higher eukaryotes.  The rapid accumulation of high-throughput data provides us an unprecedented opportunity to understand the complicated alternative splicing regulatory network.  However, existing statistical and computational methods are still lagging behind the advanced technologies.  Sorting out the coordinate and combinatorial alternative splicing regulatory network proposes a major challenge for post-genomic era.  We developed statistical methods to derive signals from high-throughput exon array data or high-throughput sequencing data to understand the Òsplicing codesÓ in gene regulation.  Partial correlation analysis was proposed to identify the association links between co-spliced exons and links between alternative exons and their regulators. The reconstructed splicing regulatory networks can help us better understand the coordinate and combinatorial nature of the alternative splicing regulation.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL60: Liang Chen - Studying alternative splicing regulatory networks through partial correlation analysis </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/06010ddc-bfbf-4736-9b31-bb1031b0fdbc" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/06010ddc-bfbf-4736-9b31-bb1031b0fdbc" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/06010ddc-bfbf-4736-9b31-bb1031b0fdbc" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL61" >HL61</a></strong><br><strong><em>Global Measures of Uncertainty: Long Overdue in Computational Molecular Biology</em></strong><br> Thursday, July 2 - 12:15 p.m. - 12:40 p.m.<br>Room: T2<br />Presenting author: Lee Newberg, Wadsworth Center, United States<br>Presentation Overview: <br>High-dimensional (high-D) discrete prediction and estimation problems are arguably the most common inference problem in computational biology, covering a range from sequence alignment to network inference. Regardless of the procedure employed, when it delivers a single answer, that answer is a point estimate selected from the set of all possible solutions, the solution ensemble. For high-D discrete spaces the immense size of these ensembles almost always portends considerable uncertainty in estimation.  Nevertheless our field has focused little attention on global uncertainty measures of these estimates.  Specifically, the almost complete absence of confidence limits is a major oversight of our community, most embarrassingly including me.  In this talk I describe credibility limits, Bayesian confidence limits (Webb), and two procedures for efficiently obtaining these limits: a sampling based procedure (Webb) and a more general DP algorithm (Newberg).  This intentionally provocative talk will focus on the statistical underpinnings of prediction and estimation in discrete high-D spaces, the glaring need to delineate the uncertainty of estimates in these spaces, credibility limits for this delineation, the value of such measures in the comparison of estimators in the absence of Ògold standardsÓ, and an illustration of these principles on sequence alignment.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL61: Lee Newberg - Global Measures of Uncertainty: Long Overdue in Computational Molecular Biology </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/a3da1920-f1eb-4529-b971-af7f4b07fa7f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/a3da1920-f1eb-4529-b971-af7f4b07fa7f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/a3da1920-f1eb-4529-b971-af7f4b07fa7f" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/neilfws" target="_top" style="text-decoration:none">Neil Saunders</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">two papers: webb-robertson & lawrence; newberg ... lawrence - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">what is a good e-value? - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">E- and p-values are small when random data in unlikely to do well - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">E-values and p-values tell you about random data but not whether there are other solutions. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">but there still might be other solutions which equally good E or p values - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">need to define confidence / credibility limits - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">e-values and p-values are poor proxies to credibility - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">an example from RNA 2D prediction, MFE is not the best representative - <a style="text-decoration:none" href="http://friendfeed.com/diriano" target="_top">Diego M. Riaño-Pachón</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">today's goal: compute a global measure of representativeness of a point estimate - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">solution spaces are immense, but we often choose a particular point estimate... misleading e.g. if there's a bimodal solution - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/a3da1920-f1eb-4529-b971-af7f4b07fa7f">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Highlights Track:</strong> <strong><a name="HL62" >HL62</a></strong><br><strong><em>A Complete Neandertal Mitochondrial Genome Sequence Determined by High-Throughput Sequencing</em></strong><br> Thursday, July 2 - 12:15 p.m. - 12:40 p.m.<br>Room: T1<br />Presenting author: Richard E. Green, Max Planck Institute for Evolutionary Anthropology, Germany<br>Presentation Overview: <br>Recent advances in high-throughput sequencing have opened new vistas in ancient DNA genomics. Following these advances, we have embarked on an effort to retrieve and assemble the genome of our closest, extinct relative, the Neandertal. As a preamble to the complete nuclear genome, we have recovered, sequenced and assembled several complete mitochondrial genomes from Neandertal fossil bones. The extreme depth of sequencing coverage in these assemblies allows for a comprehensive and qualitative assessment of the issues inherent in ancient DNA sequencing, alignment, and assembly. We encoded this knowledge into a custom assembler for ancient DNA. Using this assembler, we generated six complete Neandertal mtDNA genomes. Comparison within Neandertals and to human and other great ape mtDNA sequences confirms that Neandertal the mtDNA lineage diverged roughly 600,000 years ago. Furthermore, these data reveal the low genetic diversity among late Neandertals and imply an effective population size of fewer than 3,500 females.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>HL62: Richard Green - A Complete Neandertal Mitochondrial Genome Sequence Determined by High-Throughput Sequencing </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/3c95b6c4-bbc9-433e-b7ee-0372a2222b8b" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/3c95b6c4-bbc9-433e-b7ee-0372a2222b8b" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/3c95b6c4-bbc9-433e-b7ee-0372a2222b8b" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/allyson" target="_top" style="text-decoration:none">Allyson Lister</a> and <a href="http://friendfeed.com/sebi0" target="_top" style="text-decoration:none">sebi</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Neandertals: closest extinct relative. existed 400000-30000 yrs ago - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">chimps are closest living relatives, but deviated longer ago - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">first DNA from extinct species was in 1985 from the Quagga - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Neandertal mitochondrial genome, as it is present about 1000 copies per cell vs 2 copies of genomic DNA - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">mitochondrial genome is easier to recover since there are many copies per cell - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top"><a style="text-decoration:none" href="http://www.mitomap.org/WorldMigrations.pdf">http://www.mitomap.org/WorldMigrations.pdf</a> - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Mitochondial genome is useful for tracking maternal lineages, and it accumulates mutations slowly -- ideal for building trees - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">seems like there was no interbreeding between Neandertals and modern humans - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Deep sequencing with high-throughput next generation sequencing, used to be direct PCR - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ancient DNA fragments are just 60nt in length - <a style="text-decoration:none" href="http://friendfeed.com/marcelmartin" target="_top">Marcel Martin</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/3c95b6c4-bbc9-433e-b7ee-0372a2222b8b">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br></div>	<!-- end #mainContent --></div>
	<!-- This clearing element should immediately follow the #mainContent div in order to force the #container div to contain all child floats --><br class="clearfloat" />
<!--  <div id="footer">
    <p>Footer</p>
     </div> -->
<!-- end #container -->
</body>

<!-- Mirrored from www.iscb.org/ismbeccb2009/highlights.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:25:32 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
</html>
