<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/keynotes_details.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:25:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent">


﻿
<br>
<div class="style1">

<strong class="smhead1">Keynotes Details</strong> <br /><br />


  <div class="subheading1">
<a name="ideker"  href="keynotes_details.html#ideker"><strong>Trey Ideker, PhD</strong></a><br />
<font color="blue"><em><strong>ISCB Overton Prize</strong></em></font><br />
Associate Professor<br />
Department of Bioengineering<br />
UC San Diego, La Jolla, California, USA<br /></div>
<br>
<strong>Presentation Title:</strong> New Challenges and Opportunities in Network Biology<br>
<strong>Date:</strong> Tuesday, June 30<br>
<strong>Time:</strong> 4:45 p.m.<br>
<strong>Room:</strong> Victoria Room<br>
<br />
 <img border="2" style="margin-right: 8px" align="right" width="100" height="120" alt="Trey Ideker" vspace="2" src="includes/images/people/ideker.jpg" />
<strong>Presentation Abstract:</strong>
<p>
The past decade has seen an explosion in “genome-era” technologies which profile genes, proteins, metabolites and the intricate web of interconnections among them.  Much of bioinformatics and functional genomics is now focused on methods to assemble these diverse measurements into models of functional networks and pathways within the cell.  As important, these pathway maps will provide essential information as doctors struggle to interpret the flood of genetic and clinical data that can now be collected for a patient. </p>

<p>We are working in several areas that we believe will be critical for assembling network models and for using them in a clinical setting:</p>

<p><strong>1. Mapping the genetic network underlying the response to DNA damage.</strong> Failure of cells to respond to DNA damage is a primary step in the onset of cancer and is a key mechanism of environmental toxicity. Consequently, cells have evolved complex repair and stress responses that are highly conserved across the eukaryotic kingdom, from yeast to humans. We will describe our ongoing efforts to apply ChIP-sequencing and synthetic-lethal screens to map how the cell's transcriptional network is remodeled by DNA-damaging conditions. </p>

<p><strong>2. Network-based biomarkers for disease diagnosis and personalized medicine.</strong> Genetic biomarkers are typically thought of as individual genes and proteins— for example using prostate specific antigen (PSA) as a marker for prostate cancer.  Recently, we have shown that networks can also serve as powerful biomarkers and in many cases are more predictive than any individual gene. Our approach is to project gene and/or protein expression profiles of each patient onto the known human genetic network map to identify pathways that are predictive of disease. This “network-based” biomarker approach has shown improved accuracy in diagnosis of breast and lung cancer as well as NF-kB activation state.</p>

<p><strong>3. Protein network comparative genomics.</strong> We are developing a library of standard approaches for comparing protein interaction networks across species, conditions, and network types. We will describe network comparisons to study the protein interaction network of Plasmodium, the pathogenic protozoan that causes malaria, which surprisingly is quite divergent from other known networks.  We are also working with Dr. Sumit Chanda at the Burnham Institute to identify protein networks essential for HIV infection and how these differ from RNA and DNA viruses.</p>

<p>For professional distribution of our network-based technologies, we are developers of the Cytoscape platform, an Open-Source software environment for visualization and analysis of biological networks and models (<a target="_blank" href="http://www.cytoscape.org/">http://www.cytoscape.org/</a>).</p>



<br />
<strong>Biography:</strong><br />

<p><strong>Trey Ideker</strong>, Ph. D. is Associate Professor at the UC San Diego School of Medicine and the Department of Bioengineering.  Ideker received Bachelor’s and Master’s degrees from MIT in Electrical Engineering and Computer Science and his Ph.D. from the University of Washington in Molecular Biology under the supervision of Dr. Leroy Hood.  His research centers on genome-scale approaches to construct and compare network models of cellular processes and disease.  Ideker serves on the Editorial Boards for Bioinformatics and PLoS Computational Biology, the Board of Directors for US-HUPO and the Cytoscape Consortium, and is a regular consultant for companies such as Monsanto, Genstruct, and Mendel Biotechnology.  He was named one of the Top 10 Innovators of 2006 by Technology Review magazine and is this year’s recipient of the Overton Price from the International Society for Computational Biology.  His work has been featured in news outlets such as The Scientist, the San Diego Union Tribune, and Forbes magazine. </p> <br />




<strong>Website: </strong><a target="_blank" href="http://chianti.ucsd.edu/idekerlab/">http://chianti.ucsd.edu/idekerlab/</a><br />

<br />



  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />


  <div class="subheading1">
  <strong><a name="miller"></a>Webb Miller</strong><br />
	Pennsylvania State University<br />
	University Park, PA, USA<br /></div><br />

<strong>Presentation Title:</strong> Bioinformatics Methods to Study Species Extinctions <br>
<strong>Date:</strong> Thursday, July 2 <br>
<strong>Time:</strong> 2:00 p.m. <br>
<strong>Room:</strong> Victoria Room <br>
<br>



 <img border="2" style="margin: 5px;" align="right" alt="Webb Miller" vspace="2" src="includes/images/people/miller.jpg" /><br />

<strong>Presentation Abstract:</strong><br />

We will discuss several fundamental principles behind our approach to research in bioinformatics. These principles will be illustrated with our current research project, which aims to better understand the genomic properties that affect, or are affected by, extinction of a species. We have sequenced the woolly mammoth and the so-called Tasmanian Tiger (Thylacinus cynocephalus) using new methods for handling ancient DNA. Currently, we are using high-throughput sequencing and genotyping methods to help save the Tasmanian Devil, an iconic Australian marsupial carnivore that is greatly endangered by Devil Facial Tumor Disease. This project has led to the formulation of a novel computational problem concerning the analysis of DNA sequences.<br />
  
  <br />
 <strong>Website:</strong> <a href="http://www.bx.psu.edu/miller_lab/" target="_blank">http://www.bx.psu.edu/miller_lab/</a><br />


<br />

                          
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />

<br />
    <!---------------------------Pierre-Henri Gouyon----------------------------------------- -->      
    
    
  <div class="subheading1">
		      	<strong><a  name="gouyon" target="_blank" href="http://www.mnhn.fr/oseb/GOUYON-Pierre-Henri">Pierre-Henri Gouyon</a></strong><br />
               Muséum National d’Histoire Naturelle<br>
				Equipe Botanique<br>
				MNHN/CNRS Origine, Structure et Evolution de la Biodiversité<br>
				Paris, France<br>
         	 </div>                             
                <br />          
<strong>Presentation Title:</strong> Information and Biology<br>
<strong>Date:</strong> Monday, June 29<br>
<strong>Time:</strong> 9:00 a.m.<br>
<strong>Room:</strong> Victoria Room<br>
                 
				
                      
               <p> <strong>Presentation Abstract:</strong><br />
                Biology has been based on a number of different perceptions of nature. These different approaches have given very different types of science. In the XIXth century, biology, like all other sciences became based on the study of matter and energy. Physiology, biochemistry and functional ecology (studying geochemical cycles) all treat of living beings as entities where fluxes of matter and energy are organized into a metabolism. The XXth century, with the birth of genetics and its integration with Darwinian evolution, took a very different point of view. Namely, living beings are the products of a certain type of information having reproduced and changed through the process of natural selection over long periods of time. These two points of view result in different definitions of life (disagreeing in particular about viruses and Gaïa). Evolutionary genetics is thus based on the idea that genes are of informational nature and have progressively built organisms through the evolutionary process. On the other hand, it is recognized that organisms are built from three kinds of information: genetic, epigenetic and environmental. A point of view developed by Richard Lewontin in “The Triple Helix”. A major difficulty is that while biology is more and more based on the concept of information, this very concept is very poorly defined.</p>
                
                
			
	
                 <img border="2" style="margin: 5px;" align="right" alt="" vspace="2" src="includes/images/people/gouyon.jpg" />
                           
                  <p> <strong>Biography:</strong><br>
                  <strong>Pierre-Henri Gouyon</strong> was for almost 20 years full professor in Evolution &amp; Systematics at the University of Paris-Sud (Orsay) where he was also head of the Department of "Ecology, Systematics and Evolution" (1997-2005), as well as professor at the "Ecole Polytechnique" (1996-2008) where he launched the first course on Ecology &amp; Evolution. He is presently Professor at the Museum National d’Histoire Naturelle in Paris as well as Professor at the AgroParisTech (since 1988). He is currently the head of the team of Botany in the Unit of "Origin, Structure &amp; Evolution of Biodiversity" which is a joint unit between the Museum and the CNRS. He is the curator of the European part of the National Herbarium. He has been over the years deeply involved in the scientific and ethics committees of the department of life sciences of the Centre National de la Recherche Scientifique (CNRS) where he was Deputy-Director in 2000-2001, in the ethics committee of the INSERM (Medical Sciences) and in  diverse committees of the ANR (funding agency for the sciences). He has been as well Editor-in-Chief of the "Journal of Evolutionary Biology" of the European Society of Evolutionary Biology.
                   </p>
                   <p>
                   His research has been devoted to the mechanistic study of evolution, from genetics to ecology. It includes on one side a theoretical part on the mechanisms concerning the major traits of living organisms (sex, death…), trying to combine the forces acting at different levels of integration (genes, individuals, metapopulations, species) and integrating chance, selection and historical contingency in the evolutionary processes. On the other side, a more experimental part concerns plants. His research works were published in numerous (# 100) publications in the greatest international journals (Nature, Science…) as well as in academic books (such as, for instance "Gene avatars" Belin – Plenum for the English translation) as well as in numerous popularisation papers. At present, his main research areas concern the dispersal mechanisms linked with metapopulation dynamics (this subject is applied to the impact of GMO’s cultivation), the evolution of sex and interphyletic selection, and the evolution of shapes integrating the selective and morphogenetic factors (for instance, of pollen grain in Angiosperms).
                  </p>
                  <p>He has extensively participated in the debates concerning the relations between science and society. In particular, he has been involved in the debates over the social impact of the neoDarwinian theory on our perception of Life on one side and on the impact of biotechnologies (GMOs, RAMPs) on the environment, human health and ethics (debates with French and European experts or deputies; participation in national and international Boards and Conferences in diverse places in France and in other countries – USA, GB, Italy, Ukraine, Romania, Tunisia, Senegal, Denmark, Bolivia; interviews…). He belongs to various committees in the fields of biosafety, sustainable development, Ethics….</p>
              
                         
Laboratory Website: <a target="_blank" href="http://www.mnhn.fr/oseb/">http://www.mnhn.fr/oseb/</a><br />
Personal Website: <a target="_blank" href="http://www2.mnhn.fr/oseb/spip/GOUYON-Pierre-Henri">http://www2.mnhn.fr/oseb/spip/GOUYON-Pierre-Henri</a>        <br /> <br />

                          
                          
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
    <!---------------------------Daphne Koller----------------------------------------- -->                      
                          
                          
                          

             <div class="subheading1">
		      	<strong><a name="koller">Daphne Koller</a></strong><br />
                 Department  of Computer Science, Stanford  University<br />
				Stanford,  USA<br />
         	 </div>                             
                <br />         
				
				<strong>Presentation Title:</strong> Individual Genetic Variation: From Networks to Mechanisms<br>
<strong>Date:</strong> Tuesday, June 30<br>
<strong>Time:</strong> 9:00 a.m.<br>
<strong>Room:</strong> Victoria Room<br>

				
               
                <strong><br>Presentation Abstract:</strong>  <br />
				<p>Genetic variations underlie multiple phenotypes, both at the cellular and at the clinical level.  However, the complexity of most traits and the burden of multiple hypotheses make it difficult to uncover the genetic mechanisms that cause phenotypic diversity.  In this talk, I will describe computational methods for using biological networks to model and understand the effect of genetic variations.  We show how gene expression data from a population of genetically diverse individuals (eQTL data) can be used both to uncover regulatory networks and to understand the mechanisms by which genetic variations perturbs those networks.  We also show how a prior knowledge and data regarding biological networks and mechanisms can be used to guide the construction of regulatory networks and allow the robust selection of genetic variations that are causal for phenotypic change.   We demonstrate the applicability of these methods to problems ranging from understanding mRNA degradation pathways to elucidating mechanisms involved in human disease. </p>
			
	
                 <img border="2" style="margin: 5px;" align="right" alt="Daphne Koller" vspace="2" src="includes/images/people/koller.jpg" />
                           
                  <p> <strong>Biography:</strong><br>
                  <strong>Daphne Koller</strong> is a Professor of Computer Science at Stanford
                    University.  She received her BSc and MSc degrees from the Hebrew
                    University of Jerusalem, Israel, and her PhD from Stanford in 1993.
                    She is the author of over 150 refereed papers in a variety of topics,
                    spanning machine learning, statistical modeling, computational systems
                    biology, computational game theory, and computer vision.  She is
                    interested  in using machine learning and probabilistic methods to
                    model and analyze complex systems, such as cellular networks.  She has
                    worked extensively on the reconstruction of regulatory modules and
                    regulatory networks from gene expression data, and in understanding
                    how the activity of these networks and modules changes across tissue
                    types, across organisms, and across individuals within a population.
                   </p>
                   <p>
                    She has served the community in a variety of roles, including as the
                    program chair of the NIPS 2007 and UAI 2001 conferences, as a member
                    of numerous program committees and as associate editor of several
                    journals.  She has been the recipient of a number of prestigious
                    awards, including: the Sloan Foundation Faculty Fellowship in 1996,
                    the ONR Young Investigator Award in 1998, the Presidential Early
                    Career Award for Scientists and Engineers (PECASE) in 1999, the 
                    IJCAI Computers and Thought Award in 2001, the Cox Medal for
                    excellence in fostering undergraduate research at Stanford in 2003,
                    the MacArthur Foundation Fellowship in 2004, and the ACM/Infosys award
                    in 2008.
                  </p>
                  
                  <br>

                  
                  
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
    <!---------------------------Dr. Thomas Lengauer----------------------------------------- -->                          
                  
                <div class="subheading1">
		      	<strong><a name="lengauer">Dr. Thomas Lengauer</a></strong><br />
                 	Max-Planck Institute for Informatics<br />
					Saarbrücken, Germany<br />				
         	 </div>                            
                <br />                
                <strong>Presentation Title:</strong> Chasing the AIDS Virus<br>
<strong>Date:</strong> Wednesday, July 1<br>
<strong>Time:</strong> 4:45 p.m.<br>
<strong>Room:</strong> Victoria Room<br>


                <strong><br>Presentation Abstract:</strong> 
				<br>
				
				<p>AIDS is one of the major plights of humanity. The prime problem in applying antiviral drug therapy to AIDS patients is the variability of the AIDS virus HIV. Presented with a given drug therapy the virus follows evolutionary escape paths to drug resistance, which renders the drug therapy ineffective. Statistical computer models can help estimate the level of resistance of a given viral variant against a given antiviral drug. Furthermore, such models can estimate the evolutionary path to resistance followed by the virus. </p>
<p>				
We describe the science behind the geno2pheno Server www.geno2pheno.org which offers such models via the internet: Given the relevant portion of an HIV genome, geno2pheno predicts the resistance of HIV to any of a number of antiviral drugs that are in clinical use. Furthermore the server ranks combination drug therapies with respect to their expected effectiveness against the given HIV variant.</p>
<p>
A new class of antiviral drugs – the so-called entry inhibitors – aims at preventing HIV from entering the host immune cell. Here, we are confronted with a different computational challenge: When entering the human host cell, HIV uses one of two coreceptor molecules on the cell surface. We present a statistical model offered via the geno2pheno server that predicts which of the two coreceptors the virus uses. Determining this “viral tropism” is essential for disease prognosis and in the context of applying the new entry inhibitors.
</p>
<p>
Both models are trained using various linear and nonlinear statistical learning procedures. The training data are carefully assembled databases comprising relevant genotypic, phenotypic and clinical parameters. While the resistance model only incorporates sequence features, one version of the tropism model also involves information on the structure of the relevant portion of the viral gp120 protein that docks to the human cell. 
</p>
<p>
Both models are available via the geno2pheno webserver. The server has been developed in the context of the Arevir consortium, a German National research consortium targeted at the bioinformatical analysis of HIV resistance data, and is currently in prototypical use for research purposes. Members of the consortium and their associated practices treat about two thirds of the AIDS patients in Germany.
</p>
<p>
The talk also reports on the extension of the software offer on the Euresist server (
<a target=euresist href="http://www.euresist.org/">www.euresist.org</a>) which has been developed within the EU Project Euresist. 
</p>
<p>
Furthermore several lines of ongoing research are addressed.</p>
<b>References:</b><br>

Beerenwinkel, N., Sing, T., Lengauer, T., Rahnenführer, J., Roomp, K., Savenkov, I., Fischer, R., Hoffmann, D., Selbig, J., Korn, K., et al. 2005. Computational methods for the design of effective therapies against drug resistant HIV strains. <em>Bioinformatics</em> <b>21:</b> 3943-3950.
<br><br>
Lengauer, T., and Sing, T. 2006. Bioinformatics-assisted anti-HIV therapy. <em>Nat Rev Microbiol</em> <b>4:</b> 790-797.
<br><br>
Sander, O., T. Sing, et al. (2007). Structural Descriptors of gp120 V3 Loop for the Prediction of HIV-1 Coreceptor Usage. <em>PLoS Comput Biol </em> <b>3</b>(3): e58.
<br><br>
Altmann, A., Beerenwinkel, N. et al. (2007), Improved prediction of response to antiretroviral combination therapy using the genetic barrier to drug resistance. <em>Antivir Ther,</em> <b>12</b>(2): p. 169-78.
<br><br>
Lengauer, T., O. Sander, et al. (2007). "Bioinformatics prediction of HIV coreceptor usage." <em>Nat Biotechnol</em> <b>25</b>(12): 1407-10.
<br><br>
Rosen-Zvi, M., et al., <em>Selecting anti-HIV therapies based on a variety of genomic and clinical factors.</em> Bioinformatics, 2008. <b>24</b>(13): p. i399-406.
		<br><br>
		
			 
	
                 <img border="2" style="margin: 5px;" align="right" alt="Thomas Lengauer" vspace="2" src="includes/images/people/Lengauer.jpg" />
                  <p> <strong>Biography:</strong><br>
                 <strong>Prof. Dr. Thomas Lengauer</strong>, Ph.D. (born 1952) is Director at the Max Planck Institute for Informatics and a professor of Informatics at Saarland University in Saarbrucken, Germany. 
                 His background is in Math (Ph.D. Berlin, Germany 1976) and Computer Science (Ph.D., Stanford 1979). In the seventies he performed research in theoretical computer science, in the 80s on 
                 design methods for integrated circuits. He has been engaged in research in computational biology since the beginning of the 90s. 
                 His major focuses of research are protein bioinformatics, computational drug screening and design and bioinformatics for understanding and curing diseases. 
                 Previously, he held the positions of a full professor at the University of Paderborn, Germany (1984-1992) and of a 
                 Director of the Institute for Algorithms and Scientific Computing at the German National Research Center for Computer Science in 
                 Sankt Augustin, near Bonn Germany (1992-2001). Dr. Lengauer is a founding member of the International Society for Computational Biology 
                 (ISCB), a member of the steering board of the international conference series RECOMB, and he headed the steering board of the European 
                 bioinformatics conference series ECCB since its foundation in 2002 until 2005. In 2001 he co-founded the BioSolveIT GmbH, Sankt Augustin, 
                 Germany, which develops and distributes Cheminformatics software. In 2003 he received the Konrad Zuse Medal of the German Informatics Society 
                 and the Karl Heinz Beckurts Award. He is a member and a senator of the German Academy of Sciences Leopoldina as well as a member of acatech – 
                 German Academy of Science and Engineering.
                   </p>
<b>Website:</b>	<a target=lengauer href="http://bioinf.mpi-inf.mpg.de/homepage/index.php?&amp;account=lengauer">http://bioinf.mpi-inf.mpg.de/homepage/index.php?&account=lengauer</a>
                <br>

  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
          
 
 
    <!---------------------------Eugenia M. del Pino----------------------------------------- -->                       
          
    <div class="subheading1">
		      	<strong><a name="Pino">Eugenia M. del Pino, Ph.D.</a></strong><br />
                Professor, School of Biological Sciences,<br>
               	Pontifical Catholic University of Ecuador (PUCE)<br />
				Quito, Ecuador<br />
         	 </div>
                <br /> 
				  <strong>Presentation Title: </strong> The comparative analysis reveals independence of developmental processes during early development in frogs<br>
  <strong>Date:</strong> Wednesday, July 1<br>
<strong>Time:</strong> 9:00 a.m.<br>
<strong>Room:</strong> Victoria Room<br>

				
				 
                <strong><br>Presentation Abstract:</strong>  <br />
				The development of the frog Xenopus laevis is well characterized at the cellular and molecular levels and one can wonder whether other frogs develop in the same way. Many frogs reduced or abandoned the water dependence for early development, and accordingly there are 20 or so modes of frog reproduction. I will explain about the reproductive strategies of marsupial frogs, dendrobatid frogs and foam nesting frogs. There are changes in the speed of development among frog species. Some frogs develop at about the same rate found in X. laevis, whereas other frogs develop slowly. For example the time from fertilization to the end of gastrulation varies between 14 hrs in X. laevis to 4 days in a dendrobatid frog and to 14 days in marsupial frogs. There are modifications in morphological events associated with the rate of development: Elongation of the archenteron and of the notochord is retarded in slow developing frogs. In rapidly developing frogs, several independent processes occur simultaneously during gastrulation, whereas in frogs that develop more slowly these processes are separated in time. This means that gastrulation in frogs is modular. Associated with these modifications there are differences in gene expression. Development of the notochord and expression of the proteins Brachyury and Lim1 in the notochord is retarded in slow developing frogs. This means that there is retardation in the development of the trunk. This comparative analysis reveals certain independence of developmental processes.
 
				
                 <img border="2" style="margin: 5px;" align="right" alt="Eugenia M. del Pino" vspace="2" src="includes/images/people/pino.jpg" /> 
                  <p> <strong>Biography:</strong><br>
                 Eugenia M. del Pino works on comparative aspects of frog early development at the School of Biological Sciences of the Pontificia Universidad Católica del Ecuador (PUCE), in Quito, since 1972. She was trained in developmental biology at Emory University in the USA, and after completing the doctorate she returned to Ecuador, to teach Biology at PUCE, her Alma Matter in Quito. At Emory she used the frog Xenopus laevis for her research; in Quito, however, she had no access to this frog. She decided to find a local frog, and found a marsupial frog, Gastrotheca riobambae, in the gardens of PUCE, and began an analysis of the reproductive biology and early development of this marsupial frog. More recently she and her collaborators analyzed aspects of early development in poison-arrow frogs (Dendrobatidae) and in foam-nesting frogs (genus Engystomops). She compares the developmental strategies of these frogs with the well studied X. laevis. The comparison reveals the individuality of developmental events during gastrulation, such as blastopore closure and notochord elongation. These processes occur simultaneously in rapidly developing gastrulae of X. laevis. The natural separation of developmental events that occur in slow-developing marsupial and dedrobatid frogs is of interest to model the complex processes of early embryonic development. Eugenia del Pino received the L’Oréal/UNESCO Award for Women in Science for Latin America in 2000, and Emory University, her Alma Matter in the USA, awarded her the Sheth Distinguished International Emory Alumni Award in 2003. She was elected to the membership of the Latin American Academy of Sciences (ACAL) in 1987 and to the Academy of Sciences for the Developing World (TWAS) in 1989. In the year 2006 she was elected as honorary foreign member of the American Academy of Arts and Sciences, and as foreign associate of the National Academy of Sciences (USA). 
                   </p>                  
                  
 
 
 
   <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br /> 
  

<!---------------------------Poggio----------------------------------------- -->    
<div class="subheading1">
<strong><a name="poggio">Tomaso A. Poggio</a></strong><br />                   
               
Massachusetts Institute of Technology<br />
Cambridge, USA<br /> 
      	 </div>
		  <br /> 
          <strong>Presentation Title: </strong>  Computational Neuroscience: Models of the Visual System<br />   
<strong>Date:</strong> Monday, June 29<br />   
<strong>Time:</strong> 4:45 p.m.<br />   
<strong>Room:</strong> Victoria Room<br />   <br />   

                   
                <strong>Presentation Abstract:</strong> 
                <p>Understanding the processing of information in our cortex is a significant part of understanding how the brain works and of understanding intelligence itself, arguably one of the greatest
problems in science today.  In particular, our visual abilities are computationally amazing -- and we are still far from imitating them with computers.  Thus, visual cortex may well be a good proxy for the rest of the cortex and indeed for intelligence itself.  But despite enormous progress in the physiology and anatomy of the visual cortex, our understanding of the underlying computations remains fragmentary.
</p>
<p>
I will briefly review the anatomy and the physiology of primate visual cortex and then describe a class of quantitative models of the ventral stream for object recognition, which, heavily constrained by physiology and biophysics, have been developed during the last two decades and which have been recently shown to be quite successful in explaining several physiological data across different visual areas. </p>

                
                
                
                
                 <img border="2" style="margin: 5px;" align="right" alt="Tomaso A. Poggio" vspace="2" src="includes/images/people/poggio.jpg" /> 
                  <p> <strong>Biography:</strong><br>
               <strong>Tomaso A. Poggio</strong>, is the Eugene McDermott Professor at the Department of Brain and Cognitive Sciences; Co-Director, Center for Biological and Computational Learning; Member for the last 25 years of the Computer Science and Artificial Intelligence Laboratory at MIT; since 2000, member of the faculty of the McGovern Institute for Brain Research. Born in Genoa, Italy in 1947 (and naturalized in 1994), he received his Doctor in Theoretical Physics from the University of Genoa in 1971 and was a Wissenschaftlicher Assistant, Max Planck Institut für Biologische Kybernetik, Tüebingen, Germany from 1972 until 1981 when he became Associate Professor at MIT. He is an honorary member of the Neuroscience Research Program, a member of the American Academy of Arts and Sciences and a Founding Fellow of AAAI. He received several awards such as the Otto-Hahn-Medaille Award of the Max-Planck-Society, the Max Planck Research Award (with M. Fahle), from the Alexander von Humboldt Foundation, the MIT 50K Entrepreneurship Competition Award, the Laurea Honoris Causa from the University of Pavia in 2000 (Volta Bicentennial) and the 2003 Gabor Award. He is one of the most cited computational neuroscientists (with a H-index greater than 70 – based on Google Scholar).
</p>

                  
 
 <br />
                

  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
    <!--------------------------- Uhlen ----------------------------------------- -->                            
    <div class="subheading1">
		      	<strong><a name="Uhlen"></a>Mathias Uhlen</strong><br />
                Royal Institute of Technology (KTH)<br />
				Stockholm, Sweden		
     </div>
                <br />                
                <strong>Presentation Title: </strong> A global view on protein expression based on the Human Protein Atlas <br>
 <strong>Date:</strong> Thursday, July 2<br /> 
<strong>Time:</strong> 9:00 a.m.<br /> 
<strong>Room:</strong> Victoria Room<br /> 
				
                <strong><br>Presentation Abstract:</strong> 
				<p>
				The new version 4.0 of the Human Protein Atlas (<a href="http://www.proteinatlas.org/" target=_blank>www.proteinatlas.org</a>) has been generated with more than 6,000 validated antibodies corresponding to 5,000 human genes. The portal contains more than 5 million high-resolution images generated by immunohistochemistry and confocal microscopy. Each image has been manually annotated and curated by a certified pathologist to provide a knowledge base for functional studies and to allow searches and queries about protein profiles in normal and disease tissue. A new structure has been implemented with the inclusion of all predicted genes (approximately 20,400) with a visualization of the encoded protein characteristics for all genes. A new search tool is also launched in which advance queries can be performed, including searches for chromosome location, protein class and/or tissue specificity. We have used the protein atlas as a discovery tool to find potential biomarkers for cancer diagnostics. Some examples of biomarkers in the field of breast cancer, colon cancer and prostate cancer will be discussed.			
</p>
<p>
Uhlen, Gräslund and Sundström (2008) “ A pilot project for generating affinity reagents to human proteins” Nature Methods 5 (10): 854-855.<br><br>

Berglund et al (2008) “The epitope space of the human proteome” Protein Science 17, 606-613.<br><br>

Rockberg, Löfbom, Hjelm, Uhlen and Ståhl (2008) “Epitope mapping of antibodies using bacterial surface display” Nature Methods, 5(12):1039-45<br><br>

Björling et al (2008) “A web-based tool for in silico biomarker discovery based on tissue-specific protein profiles in normal and cancer tissues” Mol Cell Proteomics 7(5): 825-44.<br><br>

Barbe et al (2008) “Toward a confocal subcellular atlas of the human proteome” Mol Cell Proteomics. 7(3):499-508<br><br>

Berglund et (2008) “A gene-centric protein atlas for expression profiles based on antibodies” Mol Cell Proteomics 7: 2019-202<br><br>
 
Björling and Uhlen (2008) “Antibodypedia – a portal for sharing antibody and antigen validation data, Mol Cell Proteomics 7(10): 2028-37

</p>
				
				
            <img border="2" style="margin: 5px;" align="right" alt="Mathias Uhlen" vspace="2" src="includes/images/people/uhlen.jpg" />
                  <p> <strong>Biography:</strong><br>
<strong>Mathias Uhlen</strong> is Professor of Microbiology at the Royal Institute of Technology (KTH), Stockholm, Sweden. Dr Uhlen is member of the Royal Swedish Academy of Engineering Science (IVA), the Royal Swedish Academy of Science (KVA), the European Molecular Biology Organization (EMBO) and member of the Human Proteome Organization (HUPO) council. He was Vice-President of the Royal Institute of Technology (KTH), responsible for external relations, from 1999 to 2001. He was the chairman of the Swedish Biochemical and Molecular Biology Society (SFMB) from 1994 to 1999. Dr Uhlen has more than 300 publications in bioscience with the focus on the development and use of affinity reagents in biotechnology and biomedicine.
<br /><br />
Dr Uhlen is currently working on the Human Protein Resource Project (HPR), with 
the aim to systematically map the human proteome. At present, the Human Protein Atlas portal 
(<a href="http://www.proteinatlas.org/" target="_blank">www.proteinatlas.org</a>) 
contains more than 5 million high-resolution images representing 5000 human proteins.
</p>


 
 
 
 
 
 
 
                          <br />
         <div align="left">
         <a href="#top" style="font-size:9px">&uarr; TOP</a>
         </div>				<br />
 

</div>






	<!-- end #mainContent --></div>
	<!-- This clearing element should immediately follow the #mainContent div in order to force the #container div to contain all child floats --><br class="clearfloat" />
<!--  <div id="footer">
    <p>Footer</p>
     </div> -->
<!-- end #container -->
</body>

<!-- Mirrored from www.iscb.org/ismbeccb2009/keynotes_details.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:25:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
</html>