<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/poster_list.php?cat=I by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:28:29 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><br><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1"> 
 <tr> 
  <td height="12" valign="top" align="right"><b>Accepted Posters</b></td>
 </tr>
</table> 


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'I'- Genome Annotation'</b></td>
 </tr>
</table>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I01</td>
 </tr>
 <tr> 
  <td><b>Automatic Functional Annotation in a Distributed Web Service Environment</b></td>
 </tr>
 <tr> 
  <td>Anika Joecker- <i>Max-Planck Institute for Plant Breeding Research</i></td>
 </tr>
 <tr> 
  <td>Andreas Joecker (Max-Planck Institute for Plant Breeding Research, Plant Computational Biology); Ulrike Göbel (Max-Planck Institute for Plant Breeding  Research, Bioinformatics Support); Heiko Schoof (Max-Planck Instiute for Plant Breeding Research, Plant Computational Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a phylogenomics approach integrated in a web service workflow that is able to predict the function of a gene very precisely. The workflow is integrated in AFAWE, a tool for the automatic and manual annotation of genes, and used in the Medicago and tomato genome projects.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10322.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I02</td>
 </tr>
 <tr> 
  <td><b>PREDICTION OF NOVEL SHORT PROTEINS USING COMPARATIVE GENOMICS AND LOCAL PROTEIN STRUCTURE PROPERTIES</b></td>
 </tr>
 <tr> 
  <td>Josue Samayoa- <i>UC Santa Cruz</i></td>
 </tr>
 <tr> 
  <td>Kevin  Karplus (UC Santa Cruz, Biomolecular Engineering); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Short proteins (50 amino acids or less) are increasingly being characterized as functional biological components. However, accurate prediction of short proteins remains an open problem in bioinformatics. We developed a method that incorporates local protein structure properties into a comparative genomic analysis to generate whole-genome short protein predictions.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10338.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I03</td>
 </tr>
 <tr> 
  <td><b>Predicting RNA-binding proteins on a genomic scale</b></td>
 </tr>
 <tr> 
  <td>Susan Jones- <i>University of Sussex</i></td>
 </tr>
 <tr> 
  <td>Ruth Spriggs (University of Sussex, Biochemistry); Yoichi Murakami (National Institute of Biomedical Innovation, Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A two-stage support vector machine has been developed that predicts RNA-binding function at the protein level based on the prediction of RNA binding at the residue level; using only sequence information. The method achieves an MCC of 0.5 and has the potential to identify novel RNA-binding proteins on a genomic scale.  </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10444.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I04</td>
 </tr>
 <tr> 
  <td><b>Comprehensive Solexa Based Annotation of Alternatively Spliced Transcripts in Drosophila melanogaster Male, Female, and tra Mutant Transcriptomes</b></td>
 </tr>
 <tr> 
  <td>Joyce Kao- <i>University of Southern California </i></td>
 </tr>
 <tr> 
  <td>Joseph Dunham (University of Southern California, Molecular Computational Biology); Frances  Sung (University of Southern California, Molecular Computational Biology); Mazin Elhadary (University of Southern California, Molecular Computational Biology); Michelle Arbeitman (University of Southern California, Molecular Computational Biology); Sergey Nuzdhin (University of Southern California, Molecular Computational Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have Solexa sequenced male, female, and tra mutant D. melanogaster transcriptomes and are seeking to annotate alternatively spliced transcripts between those specimens.  The methods we develop to do so will also be applied across species to D. simulans and D. yakuba.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10519.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I05</td>
 </tr>
 <tr> 
  <td><b>PromoterSweep: Identification of Transcription Factor Binding Sites</b></td>
 </tr>
 <tr> 
  <td>Agnes Hotz-Wagenblatt- <i>German Cancer Research Center (DKFZ)</i></td>
 </tr>
 <tr> 
  <td>Karl-Heinz Glatting (German Cancer Research Center (DKFZ), Mol. Biophysics ); Oliver Pelz (German Cancer Research Center (DKFZ), Mol. Biophysics ); Coral del Val (University Granada, ETS Ingenieria Informatica); Sándor Suhai (German Cancer Research Center (DKFZ), Mol. Biophysics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A new pipeline is available for the prediction of transcription factor binding sites (TFBS), and potential promoter regions using a combination of different methods to enhance prediction accuracy. Methods employed include searches to promoter databases, the search against TFBS databases, and motif discovery tools using orthologous sequences.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10565.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I06</td>
 </tr>
 <tr> 
  <td><b>Maximum likelihood estimation for targeted homology search</b></td>
 </tr>
 <tr> 
  <td>Peter Menzel- <i>University of Copenhagen / Faculty of Life Sciences</i></td>
 </tr>
 <tr> 
  <td>Jan Gorodkin (University of Copenhagen / Faculty of Life Sciences, Department of Animal and Veterinary Basic Sciences); Peter Stadler (University of Leipzig, Institute for Computer Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Modeling the characteristic motifs of genes is still a manual task that requires expertise and constrains large scale genome annotations. We suggest an approach for creating models based on multiple sequence alignments that are suitable for searching in a particular phylogenetic branch by calculating residue probabilities at each alignment position.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10572.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I07</td>
 </tr>
 <tr> 
  <td><b>Blannotator - making sense of Blast annotations</b></td>
 </tr>
 <tr> 
  <td>Matti Kankainen- <i>Institute of Biotechnology, University of Helsinki</i></td>
 </tr>
 <tr> 
  <td>Liisa Holm (University of Helsinki, Institute of Biotechnology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: One line descriptions are the most popular way to represent the function of a gene. Blannotator creates a concise summary from the match list generated by Blast. Blannotator is a highly accurate tool for the annotation of thousands of gene or protein sequences in a matter of minutes.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10574.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I08</td>
 </tr>
 <tr> 
  <td><b>Xanthomonas vasicola pathovar musacearum draft genome and the putative virulence factors</b></td>
 </tr>
 <tr> 
  <td>Mtakai Ngara- <i>International Institute of Tropical Agriculture(IITA)</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Xanthomonas vasicola pathovar musacearum (XVM) is an etiological agent of Xanthomonas wilt, a disease affecting all genome groups and varieties of the Musa species. Using in silico approaches, XVM draft genome has been annotated and the genes responsible for it's virulence and pathogenesis characterized. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10676.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I09</td>
 </tr>
 <tr> 
  <td><b> MaDAS a collaborative tool for genome annotation.</b></td>
 </tr>
 <tr> 
  <td>Victor Russis- <i>Spanish National Bioinformatics Institute (INB), Spanish National Cancer Research Centre (CNIO)</i></td>
 </tr>
 <tr> 
  <td>Alfonso Valencia (Spanish National Cancer Research Centre (CNIO), Structural Biology and Biocomputing); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: MaDAS is a collaborative annotation environment for the annotation of genomes (or set of sequences) by the end users of the information. The system is oriented to the use by groups of geographyically disperse groups of users, typically experimental biologist, organized in research teams.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10696.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I10</td>
 </tr>
 <tr> 
  <td><b>Annotating Principal Isoforms with APPRIS</b></td>
 </tr>
 <tr> 
  <td>Jose Manuel Rodriguez- <i>Spanish National Cancer Research Center (CNIO)</i></td>
 </tr>
 <tr> 
  <td>Iakes Ezkurdia (Spanish National Cancer Research Center (CNIO), Structural and Computational Biology Programme); Gonzalo Lopez (Spanish National Cancer Research Center (CNIO), Structural and Computational Biology Programme); Alessandro Pietrelli (Spanish National Cancer Research Center (CNIO), Structural and Computational Biology Programme); Jan-Jaap  Wesselink (Spanish National Cancer Research Center (CNIO), Structural and Computational Biology Programme); Alfonso Valencia (Spanish National Cancer Research Center (CNIO), Structural and Computational Biology Programme); Michael Tress (Spanish National Cancer Research Center (CNIO), Structural and Computational Biology Programme); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a server to annotate principal functional variants for well-annotated genes. The server deploys a range of computational methods including the conservation of exonic structure and the conservation of protein structure and function. The server is currently being used in ENCODE project to annotate the human genome.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10709.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I11</td>
 </tr>
 <tr> 
  <td><b>BASys 2: Distributed laboratory software for automated bacterial genome annotation, analysis and visualization</b></td>
 </tr>
 <tr> 
  <td>Matthew Stuart-Edwards- <i>Public Health Agency of Canada - National Microbiology Laboratories</i></td>
 </tr>
 <tr> 
  <td>Aaron Petkau (Public Health Agency of Canada - National Microbiology Laboratory, Bioinformatics Core); Gary Van Domselaar (Public Health Agency of Canada - National Microbiology Laboratory, Bioinformatics Core); Paul Stothard (University of Alberta, Department of Agricultural, Food and Nutritional Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: BASys 2 is a prokaryotic genome annotation, analysis and visualization software platform designed to provide a powerful and flexible analysis tools for biologists. Annotation functionality is extendable through the use of plugins, and computation annotation workload can be distributed across multiple computers.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10742.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I12</td>
 </tr>
 <tr> 
  <td><b>Comparative Multi Genome Annotation with Gnomon</b></td>
 </tr>
 <tr> 
  <td>Alexandre Souvorov- <i>National Center for Biotechnology Information</i></td>
 </tr>
 <tr> 
  <td>Yuri Kapustin (National Center for Biotechnology Information, IEB); Boris Kiryutin (National Center for Biotechnology Information, IEB); Vyacheslav Chetvernin (National Center for Biotechnology Information, IEB); Tatiana Tatusova (National Center for Biotechnology Information, IEB); David Lipman (National Center for Biotechnology Information, IEB); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Genome sequencing from Fungi and Protozoa are focusing on closely related species for comparative studies. These genomes often lack transcript data. Our multi-genome approach is an iterative process that uses protein similarity between predicted genes to gradually improve the annotation. The results for Theileria  and Aspergillus are presented.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10743.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I13</td>
 </tr>
 <tr> 
  <td><b>Transcript Identification from Tiling Array Data</b></td>
 </tr>
 <tr> 
  <td>Georg Zeller- <i>Friedrich Miescher Laboratory of the Max Planck Society</i></td>
 </tr>
 <tr> 
  <td>Sascha Laubinger (Max Planck Isntitute for Developmental Biology, Molecular Biology); Jonas Behr (Friedrich Miescher Laboratory of the Max Planck Society, Machine Learning in Biology); Timo Sachsenberg (Max Planck Institute for Developmental Biology, Molecular Biology); Detlef Weigel (Max Planck Institute for Developmental Biology, Molecular Biology); Gunnar Rätsch (Friedrich Miescher Laboratory of the Max Planck Society, Machine Learning in Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Whole-genome tiling arrays allow characterizing the transcriptome under various conditions. For the central analysis task of identifying expressed transcripts, we present machine learning methods that exploit a combination of hybridization and sequence features. Evaluation on Arabidopsis tiling array data and annotated transcripts indicates greatly improved accuracy compared to other methods.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10575.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I14</td>
 </tr>
 <tr> 
  <td><b>BLASTScanner: Fast BLAST data processing with  database output</b></td>
 </tr>
 <tr> 
  <td>Detlef Groth- <i>University Potsdam</i></td>
 </tr>
 <tr> 
  <td>Joachim Selbig (University Potsdam, AG Bioinformatics); Albert J. Poustka (MPIMG Berlin, Evolution and Development); Georgia Panopoulou (MPIMG Berlin, Evolution and Development); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Scanning BLAST files is still not on solved problem. We use crossplatform single file C-applications generated with the scanner generator re2c developing scanners of biological data. They are easy to install, download and run. The output is database code suitable for piping into standard relational database systems.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/45/10865.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I15</td>
 </tr>
 <tr> 
  <td><b>Anchors based annotation of subsystem: Application to oxidative stress response in prokaryotic genomes</b></td>
 </tr>
 <tr> 
  <td>David Thybert- <i>CNRS UMR 6026 </i></td>
 </tr>
 <tr> 
  <td>Stephane Avner (CNRS UMR 6026 , Equipe B@sic); CÃ©line Lucchetti-Miganeh (CNRS UMR 6026 , Equipe B@sic); AngÃ©lique ChÃ©ron (CNRS UMR 6026 , Equipe B@sic); FrÃ©dÃ©rique Barloy-Hublercorresponding author (CNRS UMR 6026 , Equipe B@sic); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: To improve annotation accuracy and homogeneity we developed OxyGene a platform implementing a new annotation approach based on subsystems and ab-initio genome annotation using anchors. OxyGene improves the functional annotation of ROS-RNS detoxification subsystems in prokaryotic genomes by detecting forgotten loci and by simplifying, homogenizing and correcting functions description. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/48/11549.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I16</td>
 </tr>
 <tr> 
  <td><b>Coiled coil genome annotation: prediction, structure, and evolutionary Past</b></td>
 </tr>
 <tr> 
  <td>Owen Rackham- <i>University of Bristol</i></td>
 </tr>
 <tr> 
  <td>Julian Gough (University of Bristol, Computer Science); Martin Madera (University of Bristol, Computer Science); Dek  Woolfson (University of Bristol, Biochemistry); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We will present a novel approach to coiled coil prediction, the analysis of all genomes and the resulting Spiricoil database, and most importantly, the results of our evolutionary investigations based on this new resource.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11706.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I17</td>
 </tr>
 <tr> 
  <td><b>ENSEMBL ANNOTATION OF HUMAN GRCh37. Better algorithms for a better human.</b></td>
 </tr>
 <tr> 
  <td>Julio Fernandez Banet- <i>Wellcome Trust Sanger Institute</i></td>
 </tr>
 <tr> 
  <td>Bronwen Aken (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Susan Fairley (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Magali Ruffier (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Stephen Searle (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Amy Tang (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Jan-Hinnerk Vogel (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Simon White (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Amonida Zadissa (Wellcome Trust Sanger Institute, Vertebrate Genome Analysis); Michael Schuster (European Bioinformatics Institute, Panda Coordination and Outreach); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: After the release of the new human genome assembly in February 2009,  EnsEMBL has created an annotation set introducing new methods developed recently.We present how these methods improved the consistency of our annotation set compared to the protein sets provided by RefSeq and SwissProt.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12007.doc">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I18</td>
 </tr>
 <tr> 
  <td><b>HMMerThread Database: a resource for remotely conserved domains</b></td>
 </tr>
 <tr> 
  <td>Charles Bradshaw- <i>MPI-CBG</i></td>
 </tr>
 <tr> 
  <td>Robert Henschel (TU-Dresden, ZIH); Matthias Mueller (TU-Dresden, ZIH); Bianca Habermann (Scionics c/o MPI-CBG, Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The HMMerThread Database combines structure and sequence based similarity searches to provide weakly conserved domains for 8 species along with the associated functional predictions within the twilight zone of sequence similarity. The domains are validated and are presented along with extensive annotation. 1,180 validated domains are found in human alone.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12014.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I19</td>
 </tr>
 <tr> 
  <td><b>Annotating the Zebra Finch Genome</b></td>
 </tr>
 <tr> 
  <td>Susan Fairley- <i>Wellcome Trust Sanger Institute</i></td>
 </tr>
 <tr> 
  <td>Bronwen Aken (Wellcome Trust Sanger Institute, Informatics); Julio Fernandez-Banet (Wellcome Trust Sanger Institute, Informatics); Magali Ruffier (Wellcome Trust Sanger Institute, Informatics); Amy Tang (Wellcome Trust Sanger Institute, Informatics); Jan-Hinnerk Vogel (Wellcome Trust Sanger Institute, Informatics); Simon White (Wellcome Trust Sanger Institute, Informatics); Amonida Zadissa (Wellcome Trust Sanger Institute, Informatics); Steve Searle (Wellcome Trust Sanger Institute, Informatics); Tim Hubbard (Wellcome Trust Sanger Institute, Informatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Assembly taeGut3.2.4 is the first public release of the 6x coverage Taeniopygia guttata (zebra finch) genome. It was annotated using a variant of the Ensembl genebuild pipeline. The resulting annotation includes 17,148 protein coding genes and 398 pseudogenes, along with other genomic features. This annotation is available at www.ensembl.org.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12019.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I20</td>
 </tr>
 <tr> 
  <td><b>Analysis of automatic gene annotation of low coverage genome assemblies in Ensembl</b></td>
 </tr>
 <tr> 
  <td>Amonida Zadissa- <i>Wellcome Trust Sanger Institute</i></td>
 </tr>
 <tr> 
  <td>Bronwen Aken (Wellcome Trust Sanger Institute, Informatics); Susan Fairley (Wellcome Trust Sanger Institute, Informatics); Juilo Fernandez-Banet (Wellcome Trust Sanger Institute, Informatics); Magali Ruffier (Wellcome Trust Sanger Institute, Informatics); Amy Tang (Wellcome Trust Sanger Institute, Informatics); Jan Vogel (Wellcome Trust Sanger Institute, Informatics); Simon White (Wellcome Trust Sanger Institute, Informatics); Val Curwen (Wellcome Trust Sanger Institute, Informatics); Tim Hubbard (Wellcome Trust Sanger Institute, Informatics); Steve Searle (Wellcome Trust Sanger Institute, Informatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The standard evidence-based Ensembl annotation pipeline is not suitable for annotating low coverage (2x) genomes because of the highly fragmented nature of the assemblies. We present this highly automated method for producing gene annotation and analyse its application on 21 low coverage genome assemblies, using the cat genome as example.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12021.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I21</td>
 </tr>
 <tr> 
  <td><b>Large duplications in the human genome reveal details of pseudogene formation</b></td>
 </tr>
 <tr> 
  <td>Ekta Khurana- <i>Yale University</i></td>
 </tr>
 <tr> 
  <td>Hugo Lam (Yale University, Program in Computational Biology and Bioinformatics); Chao Cheng (Yale University, Molecular Biophysics and Biochemistry); Mark Gerstein (Yale University, Molecular Biophysics and Biochemistry); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have performed a detailed analysis of pseudogenes (?genes) present in large duplicated regions of the human genome. Comparisons of the substitution rates of a ?gene and its parent gene with the larger duplicated segments that contain them reveal details about the origin and time of ?gene formation.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12071.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster I22</td>
 </tr>
 <tr> 
  <td><b>An integrative approach to support phenotype-genotype research on bioenergy crops </b></td>
 </tr>
 <tr> 
  <td>Keywan Hassani-Pak- <i>Rothamsted Research</i></td>
 </tr>
 <tr> 
  <td>Shao Chih Kuo (Rothamsted Research, Mathematical and Computational Biology ); Jan Taubert (Rothamsted Research, Mathematical and Computational Biology ); Steve Hanley (Rothamsted Research, Bioenergy and Climate Change ); Chris Rawlings (Rothamsted Research, Mathematical and Computational Biology ); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The functional annotation of genomes is still a major challenge in bioinformatics. We are developing methods for automated genome annotation and systematic QTL analysis using the Ondex system. Our methods will be used to annotate the poplar genome and identify candidate genes to improve bioenergy production for willow.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12017.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<br><br>


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="12" valign="top" align="left"><b><span class=smhead1>Accepted Posters</span></b></td>
 </tr>
</table>
<div class="style1">
<a href="#prep">Preparing your Poster - Information and Poster Size</a><br />
<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a>
<a href="#schedule">Poster Schedule</a><br />
<a href="#cat">Poster Categories</a><br />
<a href="#search">Search for a Poster</a><br />

<br />
</div>



<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="poster_list950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="poster_list3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="poster_listf77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="poster_list2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="poster_list1685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="poster_list4158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="poster_list4238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="poster_listbb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="poster_list711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="poster_list39b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="poster_list7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="poster_listfdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="poster_listb2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="poster_lista34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="poster_list53f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="poster_listafd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="poster_listedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="poster_list27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="poster_list5c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="poster_liste35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="poster_listd227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="poster_lista946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="poster_listdc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="poster_list8ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="poster_list718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="poster_list8001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="http://www.iscb.org/ismbeccb2009/poster_list.php" method="POST">
<input type="hidden" name="search" value="1">
<a name="search"></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300;" name="slot"></td>
 </tr>
 <tr> 
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="last_name"></td>
 </tr>
 <tr> 
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="co_auth"></td>
 </tr>
 <tr> 
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="pres_title"></td>
 </tr>
 <tr> 
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>
 
</table>
<br><br>
</form>

<br />
<br />
<br />
<br />
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




