<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/poster_list.php?cat=M by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:26:01 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><br><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1"> 
 <tr> 
  <td height="12" valign="top" align="right"><b>Accepted Posters</b></td>
 </tr>
</table> 


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'M'- Machine Learning'</b></td>
 </tr>
</table>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M01</td>
 </tr>
 <tr> 
  <td><b>Discovering the Rules of Reversible Membrane Binding: A Machine Learning Protocol for identifying C1, C2, and PH Domain Properties.</b></td>
 </tr>
 <tr> 
  <td>Morten Källberg- <i>University of Illinois</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a machine learning protocol for determining membrane-targeting properties achieving 85-90% accuracy in separating binding and non-binding domains within families. The developed model is represented as an interpretable tree of rules showing good agreement between statistically discovered binding properties and those reported in experimental work.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10319.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M02</td>
 </tr>
 <tr> 
  <td><b>A new kernel function for clinical data</b></td>
 </tr>
 <tr> 
  <td>Anneleen Daemen- <i>KULeuven</i></td>
 </tr>
 <tr> 
  <td>Bart De Moor (KULeuven, ESAT-SCD); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: To fully exploit clinical information, appropriate modeling is required. We propose a new kernel function that distinguishes between continuous, ordinal and nominal variables. Evidently, a Least Squares Support Vector Machine based on this kernel function significantly outperformed the widely used linear kernel function when tested on three data sets.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10411.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M03</td>
 </tr>
 <tr> 
  <td><b>Identifying essential genes in metabolic networks of bacteria in silico</b></td>
 </tr>
 <tr> 
  <td>Rainer Koenig- <i>IPMB</i></td>
 </tr>
 <tr> 
  <td>Kitiporn Plaimas (IPMB, University of Heidelberg, Bioinformatics); Roland Eils (DKFZ Heidelberg, TBI Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a machine learning algorithm that infers essential reactions in a metabolic network from the topology of the network and experimental data from genomic sequences and gene expression of the corresponding coding genes. With this we support and extend experiments of high throughput genome wide knock screens. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10471.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M04</td>
 </tr>
 <tr> 
  <td><b>Predicting Protein Subcellular Localization Using Abstract Sequential Features</b></td>
 </tr>
 <tr> 
  <td>Cornelia Caragea- <i>Iowa State University </i></td>
 </tr>
 <tr> 
  <td>Adrian Silvescu (Yahoo! Labs, CA); Vasant Honavar (Iowa State University, Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present an approach to predicting protein subcellular localization from amino acid sequences that exploits the complementary strengths of feature construction (constructing complex features from existing features) and feature abstraction (grouping similar features to generate more abstract features) or feature selection to adapt the data representation used by the learner.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10513.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M05</td>
 </tr>
 <tr> 
  <td><b>Kernel Alignment K-NN for the Identification of Human Cancer Samples using the Gene Expression Profiles</b></td>
 </tr>
 <tr> 
  <td>Manuel Martin-Merino- <i>Universidad Pontificia de Salamanca</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: kNN classifier has been applied to the identification of cancer samples with encouraging results. We present a new method to learn a linear combination of dissimilarities for the kNN classifier that is robust to overfitting and solves a semi-definite programming approach. Our algorithm outperforms otheralternatives in several cancer datasets.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10544.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M06</td>
 </tr>
 <tr> 
  <td><b>A Kernel PCA Biplot method applied to gene expression data</b></td>
 </tr>
 <tr> 
  <td>Ferran Reverter- <i>Universitat de Barcelona</i></td>
 </tr>
 <tr> 
  <td>Esteban Vegas Lozano (Universitat de Barcelona, Dept. Estadística); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We describe a computational graphical tool to visualize genes and samples. We develop a biplot technique based on kernel PCA. We analyze two genomic datasets. Results suggest that our technique is a useful tool to find genes that have a similar pattern of up/down regulation for the samples</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10561.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M07</td>
 </tr>
 <tr> 
  <td><b>Automatic classification of P-type ATPases using Structured Logistic Regression</b></td>
 </tr>
 <tr> 
  <td>Poul Liboriussen- <i>Aarhus University</i></td>
 </tr>
 <tr> 
  <td>Bjørn Panyella Pedersen (Aarhus University, Centre for Membrane Pumps in Cells and Disease (PUMPKIN)); Poul Nissen (Aarhus University, Centre for Membrane Pumps in Cells and Disease (PUMPKIN)); Christian Nørgaard Storm Pedersen (Aarhus University, Bioinformatics Research Center (BiRC)); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: P-type ATPases are a very large family of ATP-driven membrane pumps involved in transmembrane transport of charged substrates. We have constructed a classifier that can distinguish between the 11 subfamilies with high accuracy. The classified it applied to Swiss-Prot/TrEMBL,  and finds 6.624 P-Type ATPases.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10596.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M08</td>
 </tr>
 <tr> 
  <td><b>Discovering biomarker panels in experiments with pooling or heterogeneous tissues</b></td>
 </tr>
 <tr> 
  <td>Dirk Repsilber- <i>Research Institute for the Biology of Farm Animals</i></td>
 </tr>
 <tr> 
  <td>Anna Telaar (FBN Dummerstorf, Genetics and Biometry); Gerd Nürnberg (FBN Dummerstorf, Genetics and Biometry); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A plethora of statistical learning approaches is being applied to find  biomarkers and also true multivariate biomarker signatures. We show how pooling and tissue heterogeneity influence the possibility of detection of biomarker signatures and compare statistical learning algorithms with respect to robustness to these stumbling blocks of biomarker discovery.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10622.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M09</td>
 </tr>
 <tr> 
  <td><b>Combining evidence from ranked gene lists</b></td>
 </tr>
 <tr> 
  <td>Raivo Kolde- <i>University of Tartu</i></td>
 </tr>
 <tr> 
  <td>Sven Laur (University of Tartu, Institute of Computer Science); Priit Adler (University of Tartu, Institute of Molecular and Cell Biology); Jaak Vilo (University of Tartu, Institute of Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose a strategy for combining evidence from ranked lists of genes. In addition to the ranking of genes, the algorithm assigns significance probability for each gene. The method can be applied in network reconstruction, meta-analysis of microarray studies, etc.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10653.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M10</td>
 </tr>
 <tr> 
  <td><b>Missing Value Imputation for Epistasis Maps</b></td>
 </tr>
 <tr> 
  <td>Colm Ryan- <i>University College Dublin</i></td>
 </tr>
 <tr> 
  <td>Derek Greene (University College Dublin, School of Computer Science and Informatics); Nevan  Krogan (University of California, San Francisco, Quantitiative Biology Institute); Gerard Cagney (University College Dublin, Conway Institute of Biomolecular and Biomedical Research); Pádraig Cunningham (University College Dublin, School of Computer Science and Informatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We introduce the problem of missing value imputation for Epistasis miniarray profiles(E-MAPS) and show the results of adapting two existing techniques to address the problem. In doing so we highlight some unique aspects of the problem – the pairwise nature of the data and the high percentage of missing values.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10663.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M11</td>
 </tr>
 <tr> 
  <td><b>KIRMES: Kernel-based Identification of Regulatory Modules in Euchromatic Sequences</b></td>
 </tr>
 <tr> 
  <td>Sebastian Schultheiss- <i>Friedrich Miescher Laboratory of the Max Planck Society</i></td>
 </tr>
 <tr> 
  <td>Wolfgang Busch (Duke University, Biology Department); Jan Lohmann (University of Heidelberg, Center for Organismal Studies); Oliver Kohlbacher (University of Tuebingen, Wilhelm Schickard Institute for Computer Science); Gunnar Raetsch (Friedrich Miescher Laboratory of the Max Planck Society, Machine Learning in Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We identify genes regulated by the same transcription factor by analyzing sets of co-expressed genes from microarrays. KIRMES infers all genes regulated by the same mechanism as the ones in the input set. KIRMES makes use of motif sampling and newly developed kernel methods for this task.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10553.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M12</td>
 </tr>
 <tr> 
  <td><b>Combining Support Vector Machines to predict novel angiogenesis genes </b></td>
 </tr>
 <tr> 
  <td>Kaur Alasoo- <i>University of Tartu</i></td>
 </tr>
 <tr> 
  <td>Phaedra Agius (Memorial Sloan-Kettering Cancer Center, .); Jaak Vilo (Quretec Ltd, .); Hedi Peterson (Quretec Ltd, .); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Angiogenesis is the natural process of growing new blood vessels in human body, that also plays an important role in cancer development. We have identified candidate genes based on 274 known angiogenesis genes using a new machine learning method employing Support Vector Machine (SVM) classification.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10705.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M13</td>
 </tr>
 <tr> 
  <td><b>Natural Kernel-Induced Bayesian Learning for Microarray Data Analysis</b></td>
 </tr>
 <tr> 
  <td>Leo Cheung- <i>Loyola University Medical Center</i></td>
 </tr>
 <tr> 
  <td>Xin Zhao (Sanjole Inc., Computer Engineering); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Incorporating a novel natural kernel building procedure under a general unifying Bayesian framework, we propose a Natural Kernel-Imbedded Gaussian Process (NKIGP) for microarray data analysis. Based on simulated and real data studies, NKIGP performed very well consistently in both linear and non-linear cases without the need of parameter tuning. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10769.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M14</td>
 </tr>
 <tr> 
  <td><b>Using support vector machines for the evaluation of computationally developed lipoxygenase structures</b></td>
 </tr>
 <tr> 
  <td>Aditya Jitta- <i>University of Hyderabad</i></td>
 </tr>
 <tr> 
  <td>Aparoy P (University of Hyderabad, School of Life Sciences); Reddanna  P (University of Hyderabad, School of Life Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Lipoxygenases are a group of structurally related family of non-heme, iron-containing dioxygenases,the geometry and composition at metal binding site in 3D models of lipoxygenases is very important.Based on these features,a tool was developed using support vector machines to evaluatecomputationally developed lipoxygenase structures.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10779.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M15</td>
 </tr>
 <tr> 
  <td><b>I/NI-calls: a novel unsupervised feature selection criterion</b></td>
 </tr>
 <tr> 
  <td>Sepp  Hochreiter - <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>Djork-Arné Clevert (Johannes Kepler University Linz, Institute of Bioinformatics); Willem Talloen (Johnson & Johnson Pharmaceutical Research & Development, Pharmaceutical Research & Development); Hinrich  Göhlmann (Johnson & Johnson Pharmaceutical Research & Development, Pharmaceutical Research & Development); Sepp Hochreiter (Johannes Kepler University Linz, Institute of Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose a novel unsupervised gene selection criterion that is based on a probabilistic latent variable model that takes probe level information -- probe correlations that cannot be explained by noise -- into account to filter out inconsistent probe sets.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10787.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M16</td>
 </tr>
 <tr> 
  <td><b>An integrative pipeline for automated data analysis and gene function annotation for genome wide high content RNAi screening</b></td>
 </tr>
 <tr> 
  <td>Stephen Wong- <i>Center for Biotechnology and Informatics, The Methodist Hospital </i></td>
 </tr>
 <tr> 
  <td>Xiaobo Zhou (Center for Biotechnology and Informatics, The Methodist Hospital, The Methodist Hospital Research Institute and Department of Radiology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose an integrated pipeline of automated data analysis for high-content screening of genome-wide RNA interference on Drosophila cell assays.  Millions of cells are efficiently segmented, and previously un-scored phenotypes are identified. This image bioinformatics pipeline is especially helpful in predicting the roles of genes in complex biological processes.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/48/11560.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M17</td>
 </tr>
 <tr> 
  <td><b>Computational Linguistic Analyses of Unknown Metagenome Sequences</b></td>
 </tr>
 <tr> 
  <td>Victor Seguritan- <i>San Diego State University</i></td>
 </tr>
 <tr> 
  <td>Anca Segall (San Diego State University, Biology); Rob Edwards (San Diego State University, Computer Science); Forest Rohwer (San Diego State University, Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A method is needed to assign functions to unknown sequences which does not rely on sequence homology alone.  The linguistic elements, syntax and semantics, of several model proteins will be used to assign functions to unknown metagenomes in a manner similar to the concept of understanding human language.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11639.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M18</td>
 </tr>
 <tr> 
  <td><b>Neural Network Pairwise Interaction Fields for protein model quality assessment</b></td>
 </tr>
 <tr> 
  <td>Alberto Jesus Martin- <i>University College</i></td>
 </tr>
 <tr> 
  <td>Gianluca Pollastri (Complex and Adaptive Systems Laboratory, University College Dublin, School of Computer Science and Informatics); Alessandro Vullo (Complex and Adaptive Systems Laboratory, University College Dublin, School of Computer Science and Informatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a new knowledge-based Model Quality Assessment Program (MQAP) at the residue level which evaluates single protein structure models. We use a tree representation of the C-alpha trace to train a novel Neural Network Pairwise Interaction Field (NN-PIF) to predict the global quality of a model.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11835.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M19</td>
 </tr>
 <tr> 
  <td><b>BayesCall: A model-based basecalling algorithm}{BayesCall: A model-based basecalling algorithm</b></td>
 </tr>
 <tr> 
  <td>Wei-Chun Kao- <i>UC Berkeley</i></td>
 </tr>
 <tr> 
  <td>Kristian Stevens (UC Davis, Computer Science); Yun Song (UC Berkeley, EECS); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A novel model-based basecalling algorithm BayesCall is introduced for the Illumina sequencing platform. This new approach significantly improves the accuracy over Illumina's basecaller Bustard. For the 76-cycle PhiX174 data from Genome Analyzer II, BayesCall improves Bustard's per-base error rate by about 47%.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11885.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M20</td>
 </tr>
 <tr> 
  <td><b>A Bayesian Monte Carlo Hidden Markov Model Approach to Transmembrane Protein Structure Prediction</b></td>
 </tr>
 <tr> 
  <td>Takashi Kaburagi- <i>Waseda University</i></td>
 </tr>
 <tr> 
  <td>Takashi Matsumoto (Waseda University, Electrical Engineering and Bioscience); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present the preliminary results of a novel scheme for transmembrane protein structure prediction using a Bayesian hidden Markov model. We applied a Bayesian learning method via the Markov chain Monte Carlo (MCMC) sampling scheme to evaluate posterior distribution of Hidden Markov Model (HMM) parameters given the training data set.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11910.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M21</td>
 </tr>
 <tr> 
  <td><b>A method for analyzing gene expression profiles based on the underlying structures</b></td>
 </tr>
 <tr> 
  <td>Shigeto Seno- <i>Dept. Bioinfo. Eng., Grad. Sch. Info. Sci. Tech., Osaka Univ. </i></td>
 </tr>
 <tr> 
  <td>Yoichi Takenaka (Graduate School of Information Science and Technology, Osaka University, Bioinfomatic Engineering); Hideo Matsuda (Graduate School of Information Science and Technology, Osaka University, Bioinfomatic Engineering); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Clustering is a powerful tool for elucidating relationships among genes, and one of the first steps in analysis. Meanwhile choice of suitable method for a given dataset is still difficult. Our approach discovers the underlying structure of a gene expression profile and provides a more intuitive understanding.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11937.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M22</td>
 </tr>
 <tr> 
  <td><b>Monte Carlo-Based Bayesian Prediction of Gene Regulatory Networks with Zipf Distribution: Mouse Nuclear Receptor Superfamily</b></td>
 </tr>
 <tr> 
  <td>Haruka Miyachika- <i>Waseda University</i></td>
 </tr>
 <tr> 
  <td>Yusuke Kitamura (Waseda University, Electrical Engineering and Bioscience); Tomomi Kimiwada (National Center of Neurology and Psychiatry, Neurosurgery); Jun Maruyama (Waseda University, Electrical Engineering and Bioscience); Takashi Kaburagi (Waseda University, Electrical Engineering and Bioscience); Takashi Matsumoto (Waseda University, Electrical Engineering and Bioscience); Keiji Wada (National Center of Neurology and Psychiatry, Neurosurgery); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a Monte Carlo-based algorithm to predict gene regulatory network structure within a Bayesian framework. The algorithm assumes that prior distribution follows the Zipf law, and is implemented using the Exchange Monte Carlo method. We applied the algorithm to a mouse nuclear receptor superfamily.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11948.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M23</td>
 </tr>
 <tr> 
  <td><b>Improving the prediction of protein-protein interactions by combining different biological sources</b></td>
 </tr>
 <tr> 
  <td>Herman van Haagen- <i>LUMC</i></td>
 </tr>
 <tr> 
  <td>Peter-Bram 't Hoen (LUMC, Human Genetics); Barend Mons (LUMC, Human Genetics); Martijn Schuemie (Erasmus MC, Biosemantics group); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Protein-protein interactions (PPIs) can be predicted based on different databases. In this study we investigate if combining those databases increases prediction power. In addition we investigate if the combined system covers more PPIs that can be evaluated. First results are promising both on coverage and prediction improvement.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11950.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M24</td>
 </tr>
 <tr> 
  <td><b>Two-way Analysis of High-Dimensional Metabolomic Datasets</b></td>
 </tr>
 <tr> 
  <td>Ilkka Huopaniemi- <i>Helsinki University of Technology</i></td>
 </tr>
 <tr> 
  <td>Tommi Suvitaival (Helsinki University of Technology, Department of Information and Computer Science); Janne Nikkilä (Helsinki University of Technology, Department of Information and Computer Science); Matej Oresic (VTT Technical Research Centre of Finland, Quantitative Biology and Bioinformatics); Samuel Kaski (Helsinki University of Technology, Department of Information and Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a Bayesian machine learning method for multivariate two-way ANOVA-type analysis ofhigh-dimensional, small sample-size metabolomic datasets. The method assumes clustered metabolites and presents confidence intervals of main and interaction up/down-regulation effects of the clusters.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11997.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M25</td>
 </tr>
 <tr> 
  <td><b>Prediction of antifreeze protein from protein sequence</b></td>
 </tr>
 <tr> 
  <td>Chin-Sheng Yu- <i>Feng Chia University</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: By overall screening in current databases, there are very few homologs of anti-freeze protein in any other species with similar protein sequence and structure identified. We present an approach to recognize AFP from protein sequence. For a nonredundant data set, the overall prediction accuracy reaches 88%. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12041.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M26</td>
 </tr>
 <tr> 
  <td><b>Bioinformatic analyses of mammalian 5'-UTR sequence properties of mRNAs predicts alternative translation initiation sites</b></td>
 </tr>
 <tr> 
  <td>Jill Wegrzyn- <i>University of California at San Diego</i></td>
 </tr>
 <tr> 
  <td>Thomas Drudge (University of California at San Diego, Skaggs School of Pharmacy and Pharmaceutical Sciences); Farmarz Valafar (San Diego State University, Bioinformatics and Medical Informatics Research Center (BMIRC)); Vivian Hook (University of California at San Diego, Skaggs School of Pharmacy and Pharmaceutical Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: This study conducted a bioinformatic evaluation of the 5'-UTR of mammalian mRNA sequences.  Machine learning techniques were applied for the classification and identification of non-AUG initiation sites in a group of mRNAs that have been experimentally demonstrated to utilize alternative sites for protein translation.  </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12103.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M27</td>
 </tr>
 <tr> 
  <td><b>Cognitive State Classification with Magnetoencephalography Data</b></td>
 </tr>
 <tr> 
  <td>andrej savol- <i>University of Pittsburgh</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We train a Support Vector Machine (SVM) soft-margin classifier on magnetoencephalography (MEG) brain-activation trajectories generated by human subjects viewing 60 common nouns divided into 12 noun groups. Semantic groupings and sensor information content are addressed. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10418.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster M28</td>
 </tr>
 <tr> 
  <td><b>The evaluation of common 1H-NMR metabolomics data preprocessing procedures reveals unanticipated side-effects</b></td>
 </tr>
 <tr> 
  <td>Tim De Meyer- <i>Ghent University</i></td>
 </tr>
 <tr> 
  <td>Bjorn Van Gasse (Ghent University, Dept. Organic Chemistry); Davy Sinnaeve (Ghent University, Dept. Organic Chemistry); Sofie Bekaert (Ghent University, Dept. Molecular Biotechnology); José Martins (Ghent University, Dept. Organic Chemistry); Wim Van Criekinge (Ghent University, Dept. Molecular Biotechnology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: 1H-NMR metabolomics provides a high-throughput methodology capable of acquiring high-resolution profiles of low-molecular weight metabolites. However, the complicated data-analysis forms a major drawback, requiring numerous data preprocessing procedures (particularly normalization, reduction and scaling steps). Here, we evaluate the most common procedures and demonstrate several unanticipated side-effects.     </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11955.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<br><br>


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="12" valign="top" align="left"><b><span class=smhead1>Accepted Posters</span></b></td>
 </tr>
</table>
<div class="style1">
<a href="#prep">Preparing your Poster - Information and Poster Size</a><br />
<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a>
<a href="#schedule">Poster Schedule</a><br />
<a href="#cat">Poster Categories</a><br />
<a href="#search">Search for a Poster</a><br />

<br />
</div>



<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="poster_list950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="poster_list3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="poster_listf77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="poster_list2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="poster_list1685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="poster_list4158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="poster_list4238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="poster_listbb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="poster_list711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="poster_list39b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="poster_list7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="poster_listfdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="poster_listb2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="poster_lista34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="poster_list53f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="poster_listafd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="poster_listedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="poster_list27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="poster_list5c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="poster_liste35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="poster_listd227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="poster_lista946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="poster_listdc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="poster_list8ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="poster_list718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="poster_list8001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="http://www.iscb.org/ismbeccb2009/poster_list.php" method="POST">
<input type="hidden" name="search" value="1">
<a name="search"></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300;" name="slot"></td>
 </tr>
 <tr> 
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="last_name"></td>
 </tr>
 <tr> 
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="co_auth"></td>
 </tr>
 <tr> 
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="pres_title"></td>
 </tr>
 <tr> 
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>
 
</table>
<br><br>
</form>

<br />
<br />
<br />
<br />
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




