<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/poster_list.php?cat=D by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><br><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1"> 
 <tr> 
  <td height="12" valign="top" align="right"><b>Accepted Posters</b></td>
 </tr>
</table> 


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'D'- Comparative Genomics'</b></td>
 </tr>
</table>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D01</td>
 </tr>
 <tr> 
  <td><b>Nearest neighbor spacing distributions of basis in different species within the same genus.  </b></td>
 </tr>
 <tr> 
  <td>Fernanda Higareda- <i>Universidad Nacional Autonoma de Mexico</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We analyze the nearest neighbor spacing distribution between basis in some bacterial genomes. The distributions are similar for species of the same genus. This is not true for species belonging to different genuses. In this study we use some species of Burkholderia, Bacillus and Clostridium. Work supported by PAPIIT project-IN111308.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10172.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D02</td>
 </tr>
 <tr> 
  <td><b>cn.FARMS - a probabilistic model to detect DNA copy numbers </b></td>
 </tr>
 <tr> 
  <td>Djork Clevert- <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>Djork-Arné Clevert (Johannes Kepler University Linz, Institute of Bioinformatics); Marianne  Tuefferd (Johnson & Johnson Pharmaceutical Research & Development, Pharmaceutical Research & Development); An  De Bondt (Johnson & Johnson Pharmaceutical Research & Development, Pharmaceutical Research & Development); Willem  Talloen (Johnson & Johnson Pharmaceutical Research & Development, Pharmaceutical Research & Development); Hinrich W.H.  Göhlmann (Johnson & Johnson Pharmaceutical Research & Development, Pharmaceutical Research & Development); Sepp Hochreiter (Johannes Kepler University Linz, Institute of Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: High-density oligonucleotide microarrays, and in particular Affymetrix Mapping or SNP arrays offer the opportunity to get a genome-wide view on copy number alterations and are increasingly used in oncology. We present a probabilistic latent variable model, called cn.FARMS, that takes probe level information to model the the correlation in the observed data.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10327.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D03</td>
 </tr>
 <tr> 
  <td><b>Rearrangement phylogeny of genomes in contig form</b></td>
 </tr>
 <tr> 
  <td>Adriana Munoz- <i>University of Ottawa </i></td>
 </tr>
 <tr> 
  <td>David Sankoff (University of Ottawa, Mathematics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Many genomes are being published in contig form. They are thus not directly usable as input to genome rearrangement algorithms.  We show how to use the contigs directly in the rearrangement algorithms as if they were chromosomes.  The method is applied to Drosophila phylogeny.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10375.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D04</td>
 </tr>
 <tr> 
  <td><b>De novo detection and evolution rate of regulatory motifs based on conservation in a dual space.</b></td>
 </tr>
 <tr> 
  <td>Marleen Claeys- <i>K.U.Leuven</i></td>
 </tr>
 <tr> 
  <td>Valerie Storms (K.U.Leuven, CMPG-bioi); Kathleen Marchal (K.U.Leuven, CMPG-bioi); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Regulatory motifs are characterized as conserved sites in a non-functional background. In motif detection methods, conservation is typically quantified by overrepresentation, or by an evolutionary relation to a common ancestor. We develop an algorithm that searches motifs in both spaces of conservation simultaneously. The method enables to study evolution of motifs.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10381.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D05</td>
 </tr>
 <tr> 
  <td><b>HCOP: A one stop orthology shop</b></td>
 </tr>
 <tr> 
  <td>Michael Lush- <i>European Bioinformatics Institute, </i></td>
 </tr>
 <tr> 
  <td>Susan Gordon (European Bioinformatics Institute,, HGNC); Ruth Seal (European Bioinformatics Institute,, HGNC); Matt Wright (European Bioinformatics Institute,, HGNC); Elspeth Bruford (European Bioinformatics Institute,, HGNC); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The HUGO Gene Nomenclature Committee (HGNC) promotes the use of the samename and symbol for orthologous genes in other species. We have developedthe HCOP search tool, which allows rapid survey of orthology assertionsfor a given gene or group of genes .</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10450.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D06</td>
 </tr>
 <tr> 
  <td><b>Microbial genome sequence analysis in a high-speed process</b></td>
 </tr>
 <tr> 
  <td>Jessica Schneider- <i>Center for Biotechnology (CeBiTec)</i></td>
 </tr>
 <tr> 
  <td>Jochen Blom (Center for Biotechnology, Computational Genomics); Eva Trost (Center for Biotechnology, Systems Biology of Regulatory Networks); Andreas  Tauch (Center for Biotechnology, Systems Biology of Regulatory Networks); Alexander Goesmann (Center for Biotechnology, Computational Genomics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: New sequencing strategies provide ultrafast access to microbial genome sequences. For their interpretation an efficient bioinformatics pipeline based on comparative genome approaches is required. Starting with the assembly and annotation of a new sequenced genome using GenDB, the software tools CARMEN and EDGAR facilitate further functional and comparative genome analysis.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10452.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D07</td>
 </tr>
 <tr> 
  <td><b>Phylogenomic inference of functional divergence</b></td>
 </tr>
 <tr> 
  <td>Tom Williams- <i>Trinity College Dublin</i></td>
 </tr>
 <tr> 
  <td>Brian E. Caffrey (Trinity College Dublin, Smurfit Institute of Genetics); Xiaowei Jiang (Trinity College Dublin, Smurfit Institute of Genetics); Christina Toft (Trinity College Dublin, Smurfit Institute of Genetics); Mario A. Fares (Trinity College Dublin, Smurfit Institute of Genetics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The identification of proteins under functional divergence is of broad interest. We present a fast new method for detecting these changes at the whole-genome level across a complex phylogenetic tree. We demonstrate the usefulness of the method through application to the evolution of pathogenicity in divergent bacterial lineages.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10493.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D08</td>
 </tr>
 <tr> 
  <td><b>Comparative Analysis of the Burkholderia pseudomallei Core Genome </b></td>
 </tr>
 <tr> 
  <td>Tannistha Nandi- <i>Genome Institute of Singapore</i></td>
 </tr>
 <tr> 
  <td>Catherine Ong Ee Ling  (Defense Medical and Environmental Research Institute, Infectious Diseases); Hui Hoon Chua (Genome Institute of Singapore, Infectious Diseases); Jason Kriesberg (Genome Institute of Singapore, Infectious Diseases); Patrick Tan (Genome Institute of Singapore, Infectious Diseases); Paul Keim (Northern Arizona University, Center for Microbial Genetics and Genomics); Talima Ross Pearson (Northern Arizona University, Center for Microbial Genetics and Genomics); William Nierman (J. Craig Venter Institute (JCVI), Infectious Diseases); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Multi-genome comparison of Burkholderia pseudomallei strains. This  often causes a fatal disease called meliodosis in humans, endemic in Southeast Asia and Northern Australia.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10317.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D09</td>
 </tr>
 <tr> 
  <td><b>Evolution of structure and sequence in alternatively spliced Drosophila genes</b></td>
 </tr>
 <tr> 
  <td>Ekaterina Ermakova- <i>Institute for Information Transmission Problems (The Kharkevich Institute)</i></td>
 </tr>
 <tr> 
  <td>Dmitry Malko (State Research Institute of Genetics and Selection of Industrial Microorganisms, Laboratory of Bioinformatics); Mikhail Gelfand (Institute for Information Transmission Problems (The Kharkevich Institute), Research and Training Center on Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Gain and loss of introns, constitutive and alternatively spliced coding regions, and nucleotide substitutions in constitutively and alternatively spliced coding regions were considered in twelve Drosophila genomes. The rearrangement rates may differ dramatically even in recently diverged species. The substitution rates depend on the type of alternative regions.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10535.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D10</td>
 </tr>
 <tr> 
  <td><b>Browsing CRISPR-cassettes in the Sorcerer II metagenome</b></td>
 </tr>
 <tr> 
  <td>Irena Artamonova- <i>Vavilov Institute Of General Genetics RAS</i></td>
 </tr>
 <tr> 
  <td>Valery A. Sorokin (Lomonosov Moscow State University, Faculty of Bioengineering and Bioinformatics); Mikhail S. Gelfand (Kharkevich Institute for Information Transmission Problems, Russian Academy of Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: To address the problem of CRISPR-cassettes identification in metagenomes we developed a computational technique based on publicly available programs. The scheme was applied to the Sorcerer II data. The results were collected in the MeCRISPR database (http://iitp.bioinf.fbb.msu.ru/vsorokin/crispr). For each family of related CRISPR-cassettes we reconstructed the evolutionary history.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10536.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D11</td>
 </tr>
 <tr> 
  <td><b>Detecting Cis Regulatory Modules in Networks: Towards better Candidate Selection</b></td>
 </tr>
 <tr> 
  <td>Ernesto Iacucci- <i>K. U. Leuven</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A significant challenge in the field of Cis Regulatory Module (CRM) detection remains the selection of appropriate training genes for CRM detection algorithms. We look to address this problem through the use of orthologously co-expressed genes, protein-protein interaction, and sequence information.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10540.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D12</td>
 </tr>
 <tr> 
  <td><b>The ABCG tranporter proteins in Plasmodium: Phylogenomic Analises</b></td>
 </tr>
 <tr> 
  <td>Ricardo Gonzalez Mendez- <i>University of Puerto Rico School of Medicine</i></td>
 </tr>
 <tr> 
  <td>Adelfa Serrano (University of Puerto Rico School of Medicine, Microbiology); Hugh Nicholas, Jr. (Pittsburgh Supercomputing Center, National Resource for Biomedical Supercomputing); Alexander Ropelewski (Pittsburgh Supercomputing Center, National Resource for Biomedical Supercomputing); Roxana Cintron Moret (University of Georgia, Department of Cellular Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The ATP-binding cassette, subfamily G (ABCG family), has five members. Sequence analyses and phylogenetic studies were done in Plasmodium species. These analyses show that the ABCG protein found in Plasmodium is a member of the ABCG1 subfamily. These results have important implications for the theories of drug resistance in Plasmodium.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10617.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D13</td>
 </tr>
 <tr> 
  <td><b>In-Silico Development of PCR Primers for Detection of Endogenous Para-Retroviruses & Episomal Viral Sequences </b></td>
 </tr>
 <tr> 
  <td>Angela Eni- <i>Covenant University</i></td>
 </tr>
 <tr> 
  <td>Ijeoma Dike (Covenant University, Biological Sciences); Conrad Omonhinmin (Covenant University, Biological Sciences); Shalom Chinedu (Covenant University, Biological Sciences); Olubanke  Ogulana (Covenant University, Biological Sciences); Segun Fatumo (Covenant University, Computer and Information Sciences); Ezekiel  Adebiyi (Covenant University, Computer and Information Sciences); Abiodun  Adebayo (Covenant University, Biological Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Dioscorea spp serve as food for millions of West-Africans yet yam-viruses are a constraint to yam production.  In-silico PCR-primer designing employing sequences of the conserved regions for the detection of EPRVs and episomal badnavirus sequences is proposed and will be implemented using ClustalX, Expression software and primer efficiency tested In-Silico.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10673.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D14</td>
 </tr>
 <tr> 
  <td><b>Human microRNAs with similarity to box H/ACA small nucleolar RNAs</b></td>
 </tr>
 <tr> 
  <td>Michelle Scott- <i>University of Dundee</i></td>
 </tr>
 <tr> 
  <td>Fabio Avolio (Univeristy of Dundee, GRE); Motoharu Ono (Univeristy of Dundee, GRE); Angus I Lamond (Univeristy of Dundee, GRE); Geoffrey J Barton (Univeristy of Dundee, BCDD); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: By computational analysis, we identified miRNA precursors with significant similarity to snoRNAs, both on the level of their genomic context and predicted folded structure. Some of these miRNA precursors bind dyskerin, a protein that functionally associates with snoRNAs, strengthening the possibility of an evolutionary relationship between some miRNAs and snoRNAs.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10723.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D15</td>
 </tr>
 <tr> 
  <td><b>Genome-wide annotation of human microRNAs under long-range developmental regulation</b></td>
 </tr>
 <tr> 
  <td>YING SHENG- <i>Bergen Center for Computational Science and Sars International Centre for Marine Molecular Biology</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: MicroRNAs(miRNAs) are small non-coding RNAs that modulate gene expression at post/transcriptional level, often play important roles in fine-tuning of developmental processes. A subset of self-transcribed miRNA loci exhibits typical characteristics of developmental genes controlled by multiple long-range enhancers. We present an approach for genome-wide annotation of this class of miRNAs.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10739.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D16</td>
 </tr>
 <tr> 
  <td><b>Development of a simple method for representation of genomics features as file system objects with extended attributes</b></td>
 </tr>
 <tr> 
  <td>Steven Karcz- <i>Agriculture and Agri-Food Canada</i></td>
 </tr>
 <tr> 
  <td>Matthew Links (Agriculture and Agri-Food Canada, Biotechnology and Bioprocesses); Isobel Parkin (Agriculture and Agri-Food Canada, Biotechnology and Bioprocesses); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The performance and scalabilty of monolithic relational database systems are beginning to limit comparative genomics feature visualization. We have developed a system of data representation for genomics features based on open source standards using extended attributes on file system objects to overcome query latency in complex comparative genomics datasets.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10755.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D17</td>
 </tr>
 <tr> 
  <td><b>Efficient Classification of Orthologues and in-Paralogues based on Super-Partitions</b></td>
 </tr>
 <tr> 
  <td>Fredj Tekaia- <i>Institut Pasteur</i></td>
 </tr>
 <tr> 
  <td>Edouard Yeramian (Institut Pasteur, Structural Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We introduce a method involving reasonable shortcuts for the detection of orthologues and their classification based on partitioning and mcl clusterings of Reciprocal Best Hits in closely related species. The partitioning-output (SuperPartitions) of this simple and efficient procedure recovers at least 75% of the potential sets of orthologues. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10554.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D18</td>
 </tr>
 <tr> 
  <td><b>Structure-function conservation of steroidogenic systems of plants and animals: phylogeny of the main components</b></td>
 </tr>
 <tr> 
  <td>George Shpakovski- <i>Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences</i></td>
 </tr>
 <tr> 
  <td>Irina N. Berdichevets (Institute of Genetics and Cytology, National Academy of Sciences of Belarus, Laboratory of Molecular Genetics); Nikolai A. Kartel (Institute of Genetics and Cytology, National Academy of Sciences of Belarus, Laboratory of Molecular Genetics); Elena K. Shematorova (Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Laboratory of Mechanisms of Gene Expression); Dmitry G. Shpakovski (Shemyakin-Ovchinnikov Institute of Bioorganic Chemistry, Russian Academy of Sciences, Laboratory of Mechanisms of Gene Expression); Svetlana G. Spivak (Institute of Bioorganic Chemistry, National Academy of Sciences of Belarus, 3Laboratory of Lipids Chemistry); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Phylogenetic analysis of all main components of sterol biogenesis in plants and animals was used to uncover the common stages and catalysed reactions. Steroid-5[alpha]-reductase, 3[beta]-hydroxysteroiddehydrogenase/[delta]5-4isomerase, 11[beta]</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10822.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D19</td>
 </tr>
 <tr> 
  <td><b>Phylogenetic and Functional Assessment of Orthologs Inference Projects and Methods</b></td>
 </tr>
 <tr> 
  <td>Christophe Dessimoz- <i>ETH Zurich</i></td>
 </tr>
 <tr> 
  <td>Adrian Altenhoff (ETH Zurich, Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: This work introduces methodology to verify orthology in terms of phylogeny and function, and performs a comprehensive comparison of nine leading ortholog inference projects using both phylogenetic and functional tests. The results show large variations in terms of performances, indicating that the choice of orthology database can have a strong impact on downstream analysis.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/48/11562.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D20</td>
 </tr>
 <tr> 
  <td><b>Viral Bioinformatics Resource Center Tools</b></td>
 </tr>
 <tr> 
  <td>Chris Upton- <i>University of Victoria</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The VBRC was established by the NIH to provide virus-centric databases and bioinformatics tools.  We support 7 virus families, including pathogens considered to be potential threats as agents of bioterrorism, as well as pathogens classified as causing emerging or re-emerging infectious diseases.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11781.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D21</td>
 </tr>
 <tr> 
  <td><b>Pathway Signature Genes</b></td>
 </tr>
 <tr> 
  <td>Lucas Brouwers- <i>NCMLS</i></td>
 </tr>
 <tr> 
  <td>Martijn Huynen (NCMLS, Centre for Molecular and Biomolecular Informatics); Bas Dutilh (NCMLS, Centre for Molecular and Biomolecular Informatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We define pathway signature genes that reliably predict the presence or absence of pathways in incomplete (metagenomic) sequencing data by overlapping presence/absence profiles of genes with pathways. Cross-validation on partial genomes yields high precision (85,3%) and accuracy (80,6%), while application to metagenomic data confirms known pathways and identifies new processes.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11945.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D22</td>
 </tr>
 <tr> 
  <td><b>Benchmarking of Methods for the Identification of Orthologs</b></td>
 </tr>
 <tr> 
  <td>Sabine Thuss- <i>Heinrich Heine Universitaet Duesseldorf</i></td>
 </tr>
 <tr> 
  <td>Nina Levar (Heinrich Heine Universitaet Duesseldorf, - ); Christian Esser (Heinrich Heine Universitaet Duesseldorf , Institut fuer Botanik III ); Dr. Tal Dagan (Heinrich Heine Universitaet Duesseldorf , Institut fuer Botanik III ); Prof. Dr. Martin J. Lercher (Heinrich Heine Universitaet Duesseldorf, Lehrstuhl fuer Bioinformatik); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We compare four methods for the identification of orthologous genes. A non-standard application of the Markov Cluster Algorithm (MCL), as well as a method combining sequence similarity and synteny, lead to many more complete clusters and fewer singletons than the other tested methods.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11957.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D23</td>
 </tr>
 <tr> 
  <td><b>A High Performance E-Cluster BLAST System</b></td>
 </tr>
 <tr> 
  <td>Tae-Kyung Kim- <i>Chungbuk National University</i></td>
 </tr>
 <tr> 
  <td>Chi-Whan Choi (Chungbuk National University, Bioinformatics); Hun-Gi Kim (Chungbuk National University, Bioinformatics); Wan-Sup Cho (Chungbuk National University, MIS/BK21 Team); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose a novel BLAST cluster architecture on top of E-Cluster, which consists of dynamic number of PCs. We adopt a logical partitioning and intra-query parallelism on E-Cluster. A proposed BLAST is much superior to conventional grid or cluster BLAST systems in terms of manageability, performances and expenses.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12015.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D24</td>
 </tr>
 <tr> 
  <td><b>Visual and Statistical Comparison of Metagenomes with MEGAN 3</b></td>
 </tr>
 <tr> 
  <td>Suparna Mitra- <i>Center for Bioinformatics (ZBIT), Tuebingen University</i></td>
 </tr>
 <tr> 
  <td>Daniel Richter (Center for Bioinformatics (ZBIT), Tuebingen University, Computer Science Department); Alexander Auch (Center for Bioinformatics (ZBIT), Tuebingen University, Computer Science Department); Stephan Schuster (Center for Comparative Genomics, Center for Infectious Disease Dynamics, PennState University); Daniel Huson (Center for Bioinformatics (ZBIT), Tuebingen University, Computer Science Department); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: MEGAN (MEtaGenome Analyzer) is a powerful computer program that allows laptop analysis of large metagenomics datasets providing taxonomic and functional analysis. It has an interactive and fully customizable chart viewer and also allows visual and statistical comparative analysis of different metagenomes.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12034.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D25</td>
 </tr>
 <tr> 
  <td><b>Convergent evolution of domain architectures is rare after all</b></td>
 </tr>
 <tr> 
  <td>Martin Madera- <i>University of Bristol</i></td>
 </tr>
 <tr> 
  <td>Julian Gough (University of Bristol, Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The domain architecture of a protein is the sequential order of its domains. (Gough, 2005) concluded that only 0.2-2% of domain architectures have evolved more than once. Using an automated method, (Forslund et al., 2008) found 12.4%. We examine the Forslund predictions and present several large classes of false positives.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12045.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster D26</td>
 </tr>
 <tr> 
  <td><b>Fine-Structured Segmental Variation in Eukaryote Genomes</b></td>
 </tr>
 <tr> 
  <td>Jonathan Keith- <i>Queensland University Of Technology</i></td>
 </tr>
 <tr> 
  <td>Christopher Oldmeadow (Queensland University of Technology, School of Mathematical Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: As an initial step towards delineating new functional elements in eukaryote genomes, we present a segmentation of human and fruit fly genomes, and classify the segments according to properties suggestive of function. We further investigate the motif content and GO term enrichment of these classes.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10515.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<br><br>


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="12" valign="top" align="left"><b><span class=smhead1>Accepted Posters</span></b></td>
 </tr>
</table>
<div class="style1">
<a href="#prep">Preparing your Poster - Information and Poster Size</a><br />
<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a>
<a href="#schedule">Poster Schedule</a><br />
<a href="#cat">Poster Categories</a><br />
<a href="#search">Search for a Poster</a><br />

<br />
</div>



<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="poster_list950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="poster_list3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="poster_listf77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="poster_list2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="poster_list1685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="poster_list4158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="poster_list4238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="poster_listbb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="poster_list711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="poster_list39b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="poster_list7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="poster_listfdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="poster_listb2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="poster_lista34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="poster_list53f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="poster_listafd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="poster_listedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="poster_list27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="poster_list5c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="poster_liste35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="poster_listd227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="poster_lista946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="poster_listdc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="poster_list8ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="poster_list718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="poster_list8001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="http://www.iscb.org/ismbeccb2009/poster_list.php" method="POST">
<input type="hidden" name="search" value="1">
<a name="search"></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300;" name="slot"></td>
 </tr>
 <tr> 
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="last_name"></td>
 </tr>
 <tr> 
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="co_auth"></td>
 </tr>
 <tr> 
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="pres_title"></td>
 </tr>
 <tr> 
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>
 
</table>
<br><br>
</form>

<br />
<br />
<br />
<br />
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




