<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/poster_list.php?cat=G by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><br><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1"> 
 <tr> 
  <td height="12" valign="top" align="right"><b>Accepted Posters</b></td>
 </tr>
</table> 


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'G'- Functional Genomics'</b></td>
 </tr>
</table>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G01</td>
 </tr>
 <tr> 
  <td><b>Tissue-specific repression of certain housekeeping genes is essential to allow specialized tissue function</b></td>
 </tr>
 <tr> 
  <td>Lieven Thorrez- <i>K.U.Leuven</i></td>
 </tr>
 <tr> 
  <td>Ilaria Laudadio (Universite Catholique de Louvain, de Duve Institute); Katrijn Van Deun (K.U.Leuven, Department of Psychology); Roel Quintens (K.U.Leuven, Gene expression unit); Nico Hendrickx (K.U.Leuven, Gene expression unit); Mikaela Granvik (K.U.Leuven, Gene expression unit); Katleen Lemaire (K.U.Leuven, Gene expression unit); Anica Schraenen (K.U.Leuven, Gene expression unit); Leentje Van Lommel (K.U.Leuven, Gene expression unit); Stefan Lehnert (K.U.Leuven, Gene expression unit); Cristina Aquayo-Mazzucato (Harvard University, Section of islet transplantation and cell biology); Susan Bonner-Weir (Harvard University, Section of islet transplantation and cell biology); Rui Cheng-Xue (University of Louvain, Unite d\\\'endocrinologie et metabolisme); Patrick Gilon (University of Louvain, Unite d\\\'endocrinologie et metabolisme); Ivan Van Mechelen (K.U.Leuven, Psychology); Frederic Lemaigre (Universite Catholique de Louvain, de Duve Institute); Frans Schuit (K.U.Leuven, Gene expression unit); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose that specific tissue functions not only rely on tissue-specific gene expression but also on tissue-specific gene repression, which is established during maturation of these tissues.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10309.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G02</td>
 </tr>
 <tr> 
  <td><b>BRISKA: Brassica Seed Knowledge Application</b></td>
 </tr>
 <tr> 
  <td>Hugo Berube- <i>National Research Council Canada</i></td>
 </tr>
 <tr> 
  <td>Alain Tchagang (National Research Council Canada, Institute for Information Technology); Yunli Wang (National Research Council Canada, Institute for Information Technology); Ziying Liu (National Research Council Canada, Institute for Information Technology); Sieu Phan (National Research Council Canada, Institute for Information Technology); Fazel Famili (National Research Council Canada, Institute for Information Technology); Youlian Pan (National Research Council Canada, Institute for Information Technology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The Brassica Seed Knowledge Application (BRISKA) is an interactive web-based application providing, via interactive tools, useful biological information for functional genomics studies of oilseed crops with emphasis on seed development and fatty acid metabolism in Brassica napus and related species.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10362.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G03</td>
 </tr>
 <tr> 
  <td><b>Building your own gene sets with WhichGenes, a new webtool for creating biological hypothesis to test in gene set based methods.</b></td>
 </tr>
 <tr> 
  <td>Gonzalo Gómez-López- <i>CNIO</i></td>
 </tr>
 <tr> 
  <td>Daniel Gonzalez-Peña (University of Vigo, Higher Technical School of Computer Engineering); Florentino Fernandez-Riverola (University of Vigo, Informatics Department); David Gonzalez-Pisano (CNIO, Bioinformatics Unit); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Whichgenes is a new web-tool to build gene sets from multiple and scattered biological databases. The aim of this user-friendly server is to facilitate the creation of biological hypothesis to test using gene set based methods.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10390.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G04</td>
 </tr>
 <tr> 
  <td><b>Functional annotation of orthologous groups by using hierarchical multi label classification</b></td>
 </tr>
 <tr> 
  <td>Nives Skunca- <i>Rudjer Boskovic Institute</i></td>
 </tr>
 <tr> 
  <td>Fran Supek (Rudjer Boskovic Institute, Department of Electronics); Tomislav Smuc (Rudjer Boskovic Institute, Department of Electronics); Pance Panov (Jozef Stefan Institute, Department of Knowledge Technologies); Saso Dzeroski (Jozef Stefan Institute, Department of Knowledge Technologies); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The object of research was assessing the success of phylogenetic profiling in predicting the function of protein groups in the Orthologous Matrix project, when using decision-trees for Hierarchical Multilabel Classification. Performance analysis will be presented, broken down per Gene ontology category, discussing the effect of different features used in prediction.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10438.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G05</td>
 </tr>
 <tr> 
  <td><b>A systematic analysis of alternative 3’ UTR of human transcripts: improvement of detection of microRNA targets</b></td>
 </tr>
 <tr> 
  <td>Gabriele Sales- <i>University of Padua</i></td>
 </tr>
 <tr> 
  <td>Marta Biasiolo (University of Padua, Department of Biology); Silvio Bicciato (University of Modena,  Department of Medical Biosciences); Stefania Bortoluzzi (University of Padua, Department of Biology); Chiara Romualdi (University of Padua, Department of Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: MicroRNAs are small noncoding RNAs that serve as post-transcriptional regulators of gene expression in Eukaryotes. DNA microarray technology can be used to detect the interaction between microRNAs and their targets. Given that microRNAs interact with 3\\\'UTR regions only, we propose a custom annotation scheme based on custom 3'UTR transcript annotation.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10442.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G06</td>
 </tr>
 <tr> 
  <td><b>ConceptGen: A gene set enrichment and concept mapping tool</b></td>
 </tr>
 <tr> 
  <td>Maureen Sartor- <i>University of Michigan</i></td>
 </tr>
 <tr> 
  <td>Vasudeva Mahavisno (Univ of Michigan, CCMB); Zach Wright (Univ of Michigan, CCMB); Alla  Karnovsky (Univ of Michigan, CCMB); Gilbert Omenn (Univ of Michigan, CCMB); Brian Athey (Univ of Michigan, CCMB); James Cavalcoli (Univ of Michigan, CCMB); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed ConceptGen, a web-based gene set enrichment and concept mapping tool that allows researchers to identify biological concepts, derived from 13 knowledge sources, in their experimental results or collaborative analyses. Networks of relationships among these diverse concepts, including several hundred experimentally-derived gene sets, can be explored and visualized. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10465.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G07</td>
 </tr>
 <tr> 
  <td><b>Designing, Executing, and Sharing Scientific Workflows: Taverna and myExperiment</b></td>
 </tr>
 <tr> 
  <td>Paul Fisher- <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>Alan  Williams (University of Manchester, Computer Science); Aleksandra  Nenadic (University of Manchester, Computer Science); Constantinos  Astreos (University of Manchester, Computer Science); Danius  Michaelides (University of Southampton, Computer Science); Don  Cruickshank (University of Southampton, Computer Science); David  De Roure (University of Southampton, Computer Science); David  Withers (University of Manchester, Computer Science); Franck  Tanoh (University of Manchester, Computer Science); Ian  Dunlop (University of Manchester, Computer Science); Jiten  Bhagat (University of Manchester, Computer Science); Katy  Wolstencroft (University of Manchester, Computer Science); Paolo  Missier (University of Manchester, Computer Science); Sergejs  Aleksejevs (University of Manchester, Computer Science); Stian  Soiland-Reyes (University of Manchester, Computer Science); Stuart  Owen (University of Manchester, Computer Science); Tom Oinn (Contractor, Computer Science); Carole  Goble (University of Manchester, Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The Taverna workbench allows scientists to design and execute scientific workflows, combining distributed services and data resources into a single experimental protocol. Workflows provide a means of conducting systematic and explicit data analyses. Workflows developed in Taverna can be shared through myExperiment, a social networking site for scientists.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10577.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G08</td>
 </tr>
 <tr> 
  <td><b>Global modeling of cancer gene expression signatures </b></td>
 </tr>
 <tr> 
  <td>Leo Lahti- <i>Helsinki University of Technology</i></td>
 </tr>
 <tr> 
  <td>Samuel Myllykangas (Stanford University School of Medicine, Division of Oncology); Sakari Knuutila (University of Helsinki, Haartman Institute); Samuel Kaski (Helsinki University of Technology, Information and Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Heterogeneous cancer types are coupled through common functionalchanges and the corresponding gene expressionsignatures. Computational modeling of these functional relationshipsis used to build a comprehensive view of the functional landscape ofhuman cancer.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10578.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G09</td>
 </tr>
 <tr> 
  <td><b>Generalised Linear Models of yeast transcriptional regulation</b></td>
 </tr>
 <tr> 
  <td>Juri Reimand- <i>University of Tartu</i></td>
 </tr>
 <tr> 
  <td>Juan Vaquerizas (EMBL , European Bioinformatics Institute); Annabel Todd (EMBL , European Bioinformatics Institute); Jaak Vilo (University of Tartu, Insitute of Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Transcriptional regulation is a complex and poorly characterised process. We present a framework for predicting transcriptional regulatorsof selected cellular processes, using protein-DNA interactions and expressionprofiles of transcription factor (TF) knockouts. Our model successfully recovers the yeast cell cycle, as 7 of the 10 key cell cycle TFs are detected in top10 list of candidates.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10593.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G10</td>
 </tr>
 <tr> 
  <td><b>Cross-platform microarray compendium and a corresponding web interface</b></td>
 </tr>
 <tr> 
  <td>Qiang FU- <i>Katholieke Universiteit Leuven</i></td>
 </tr>
 <tr> 
  <td>kristof engelen (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); Pieter Meysman (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); Karen Lemmens (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); Riet De Smet (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); Carolina Fierro (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); Inge Thijs (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); Kathleen Marchal (Katholieke Universiteit Leuven, Centre of Microbial and Plant Genetics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have designed a (semi-automatic) system to create cross-platform expression compendia, offering the advantage of maximally exploiting publicly available information.  A web interface exists for public access of the data, including tools for performing query-based bi-clustering and visualization of overlapping bi-clusters. Compendia are created for Escherichia coli, Salmonella enterica and Bacillus subtilis.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10681.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G11</td>
 </tr>
 <tr> 
  <td><b>Discovery of functional human ncRNAs by expression profiling using new mapping of microarray probes to the non protein coding transcriptome</b></td>
 </tr>
 <tr> 
  <td>Alberto Risueño- <i>Cancer Research Center (CIC, CSIC/USAL)</i></td>
 </tr>
 <tr> 
  <td>Carlos Prieto (Cancer Research Center (CIC, CSIC/USAL), Bioinformatics and Functional Genomics Research Group); Celia Fontanillo (Cancer Research Center (CIC, CSIC/USAL), Bioinformatics and Functional Genomics Research Group); Javier De Las Rivas (Cancer Research Center (CIC, CSIC/USAL), Bioinformatics and Functional Genomics Research Group); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Transcriptomic profiling provided by high density microarrays includes a significant signal coming from non-protein coding RNAs (like miRNAs, snRNA, etc). However, specific expression signal produced by such bio-entities is usually neglected. We use significant differential expression of probes remapped to non-protein coding transcriptome to discover functional ncRNAs.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10691.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G12</td>
 </tr>
 <tr> 
  <td><b>Identification of functional related gene in Malaria parasites using Computational Pipeline Technique</b></td>
 </tr>
 <tr> 
  <td>Jelili Oyelade- <i>Covenant University</i></td>
 </tr>
 <tr> 
  <td>Ezekiel  Adebiyi (Covenant University, Computer and Information Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: P. falciparum, is the most deadly form of malaria. Several computational methods have been used to identify genes clustering. Grouping of genes that are co-regulated in a metabolic pathway is very important in drug prediction. We applied k-means clustering tool to classify genes into their various metabolic pathways.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10699.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G13</td>
 </tr>
 <tr> 
  <td><b>Genome-wide DNA Methylation Analysis by sequencing of reduced complexity bisulfite-treated genomic fragments</b></td>
 </tr>
 <tr> 
  <td>Irina Khrebtukova- <i>Illumina, Inc.</i></td>
 </tr>
 <tr> 
  <td>Lu Zhang (Illumina, Inc., Expression Applications R&D); Raymond McCauley (Illumina, Inc., Expression Applications R&D); Juying Yan (Illumina, Inc., Expression Applications R&D); Gary P Schroth (Illumina, Inc., Expression Applications R&D); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Next-generation sequencing with the Illumina GAII system allows study of DNA methylation at single base resolution on the whole genome scale. We have developed a bioinformatics pipeline for alignment of bisulfite-converted methylated fragments to the genome followed by scoring of methylation levels of CpG dinucleotides on a genome-wide scale.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10701.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G14</td>
 </tr>
 <tr> 
  <td><b>Meta-analysis of Chronic obstructive pulmonary disease public gene expression datasets</b></td>
 </tr>
 <tr> 
  <td>Ketan Patel- <i>Pfizer Ltd</i></td>
 </tr>
 <tr> 
  <td>Sari Ward (Pfizer Ltd, eBiology); Iain Kilty (Pfizer Ltd, Allergy and Respiratory); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A meta-analysis of several publically available Chronic obstructive pulmonary disease (COPD) gene expression datasets was performed.  To understand the consistent gene expression changes in lung tissue we used  gene set enrichment techniques and the 'Connectivity map' of small molecule gene expression signatures to identify pathways for drug targeting.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10721.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G15</td>
 </tr>
 <tr> 
  <td><b>Understanding the consistency of molecular changes in Endometriosis through meta-analysis of public gene expression datasets</b></td>
 </tr>
 <tr> 
  <td>Roddy Walsh- <i>Pfizer Ltd</i></td>
 </tr>
 <tr> 
  <td>Ketan Patel (Pfizer Ltd., Computational Sciences); Anneli Sullivan (Pfizer Ltd, eBiology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: A meta-analysis of public Endometriosis gene expression datasets was performed.  Due to the large amount of gene changes, we have used a comprehensive approach to tease apart underlying consistent changes in biological processes.  Our results provide an understanding of data consistency between the various published datasets and provide guidance for designing future studies.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10725.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G16</td>
 </tr>
 <tr> 
  <td><b>An Ensemble Model of Competitive Multi-factor Binding of the Genome</b></td>
 </tr>
 <tr> 
  <td>Todd Wasson- <i>Duke University</i></td>
 </tr>
 <tr> 
  <td>Alexander Hartemink (Duke University, Department of Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: DNA occupancy by various proteins and protein complexes is the result of thermodynamic competition amongst them. This competition is driven by the sequence preferences and concentration of each of these DNA binding factors. We present a model that explicitly considers competition to produce a probabilistic representation of DNA occupancy.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10746.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G17</td>
 </tr>
 <tr> 
  <td><b>MouseCyc: a pathways approach to integration of mouse functional, phenotype and expression data</b></td>
 </tr>
 <tr> 
  <td>Judith Blake- <i>The Jackson Laboratory</i></td>
 </tr>
 <tr> 
  <td>Alexei Evsikov (The Jackson Laboratory, Mouse Genome Informatics); Mary E. Dolan (The Jackson Laboratory, Mouse Genome Informatics); Carol J. Bult (The Jackson Laboratory, Mouse Genome Informatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: MouseCyc is a database of curated biochemical pathways for the laboratory mouse. We have developed a resource based on pathway genes sets that integrates functional, phenotype and expression data for MouseCyc pathways based on mouse genetic and genomic data available at Mouse Genome Informatics. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10757.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G18</td>
 </tr>
 <tr> 
  <td><b>Subtractive Genomic Context and the Gene Modules of Patho-genic strains of Escherichia coli</b></td>
 </tr>
 <tr> 
  <td>Gabriel Moreno-Hagelsieb- <i>Wilfrid Laurier University</i></td>
 </tr>
 <tr> 
  <td>Anis  Karimpour-Fard (University of Colorado School of Medicine, Center for Computational Pharmacology); Lawrence  Hunter (University of Colorado School of Medicine, Center for Computational Pharmacology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: This work demonstrates the use of subtractive interactomics for examining the functional interactions that differentiate evolutionarily closely related Prokaryotes, such as non-pathogenic and pathogenic strains of Escherichia coli. We focused on distinct interactions responsible for pathogenicity by comparing two pathogenic strains against each other, and against a non-pathogenic strain.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/45/10867.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G19</td>
 </tr>
 <tr> 
  <td><b>Using related functions to improve the construction of composite functional linkage networks</b></td>
 </tr>
 <tr> 
  <td>Sara Mostafavi- <i>University of Toronto</i></td>
 </tr>
 <tr> 
  <td>Quaid Morris (University of Toronto, Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We use related Gene Ontology categories to improve the construction of composite functional linkage networks. In doing so, we show that we can considerably improve the accuracy of gene function prediction on several benchmark datasets.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/45/10872.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G20</td>
 </tr>
 <tr> 
  <td><b>Meta - Analysis</b></td>
 </tr>
 <tr> 
  <td>Mahesh Visvanathan- <i>KU</i></td>
 </tr>
 <tr> 
  <td>Mahesh Visvanathan (KU, BCF); Gerald Lushington (KU, BCF); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Meta-analysis of microarray data coming from a number of microarray experiments can be attempted with two systematically different approaches. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11673.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G21</td>
 </tr>
 <tr> 
  <td><b>Design and implementation of image and data analysis strategies for assessing the relationship between cell state and endocytic system in a high-content genome-wide screening</b></td>
 </tr>
 <tr> 
  <td>Giovanni Marsico- <i>Max Planck Institut of Molecular Cell Biology and Genetics </i></td>
 </tr>
 <tr> 
  <td>Yannis Kalaidzidis (Max Planck Institut of Molecular Cell Biology and Genetics , MPG); Marino Zerial (Max Planck Institut of Molecular Cell Biology and Genetics , MPG); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose a new methodology for the analysis of cell-based, high-content assays. We design and implement image and data analysis strategies for assessing the relationship between cell state and endocytic system. We show as a proof of principle how this approach can lead to the formulation of new biological hypothesis.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11884.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G22</td>
 </tr>
 <tr> 
  <td><b>Supervised learning for detection of transcription factor binding sites</b></td>
 </tr>
 <tr> 
  <td>Justin Bedo- <i>NICTA</i></td>
 </tr>
 <tr> 
  <td>Geoff Macintyre (NICTA, Life Sciences); Izhak Haviv (Baker IDI,  ); Adam Kowalczyk (NICTA, Life Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a supervised prediction method for genome-wide discovery oftranscription factor (TF) binding sites that uses sequence information only.Using a single chromosome TF binding profile, we trained a support vectormachine (SVM) which predicts TF binding with an accuracy superior to standardposition-weight matrix (PWM) approaches.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11982.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G23</td>
 </tr>
 <tr> 
  <td><b>Nucleosome-free Regions are Associated with Coding and Non-coding Transcripts</b></td>
 </tr>
 <tr> 
  <td>Karin Schwarzbauer- <i>Johannes Kepler University</i></td>
 </tr>
 <tr> 
  <td>Ulrich Bodenhofer (Johannes Kepler University, Institute of Bioinformatics); Mihaela Ionescu (Johannes Kepler University, Institute of Bioinformatics); Sepp Hochreiter (Johannes Kepler University, Institute of Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We analyze sequence characteristics of long nucleosome-free regions (lNFRs) in the human genome obtained from next-generation sequencing. lNFR sequences in promoter regions and the remaining lNFR sequences share the same sequence patterns, hence long NFR sequences not lying in known promoter regions potentially reveal previously unknown transcripts. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11954.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G24</td>
 </tr>
 <tr> 
  <td><b>Novel RNA Paired-End Sequencing Technique Generates High-Resolution Map of Drosophila Melanogaster Transcription Start Sites During Embryogenesis </b></td>
 </tr>
 <tr> 
  <td>David Corcoran- <i>Duke University</i></td>
 </tr>
 <tr> 
  <td>Ting Ni (Duke University, Institute for Genome Sciences and Policy); Yuan Gao (Virginia Commonwealth University, School of Engineering); Elizabeth Rach (Duke University, Institute for Genome Sciences and Policy); Eric Spana (Duke University, Department of Biology); Jun Zhu (Duke University, Institute for Genome Sciences and Policy); Uwe Ohler (Duke University, Institute for Genome Sciences and Policy); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Our study utilizes a novel high-throughput sequencing technique for the identification of transcription start sites, and we present a high-resolution map of start sites utilized by Drosophila melanogaster during embryogenesis.  This map makes it possible to identify sequence specific features that are associated with different types of transcriptional initiation.  </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12026.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster G25</td>
 </tr>
 <tr> 
  <td><b>Mechanisms for Action of the Ultraconserved Elements in Embryonic Stem Cell Differentiation</b></td>
 </tr>
 <tr> 
  <td>Courtney Onodera- <i>University of California, Santa Cruz</i></td>
 </tr>
 <tr> 
  <td>Jason Underwood (University of California, Santa Cruz, Howard Hughes Medical Institute); Sol Katzman (University of California, Santa Cruz, Biomolecular Engineering); Bryan King (University of California, Santa Cruz, Howard Hughes Medical Institute); Sara Sowko (University of California, Santa Cruz, Biomolecular Engineering); Andre Love (University of California, Santa Cruz, Howard Hughes Medical Institute); Sofie Salama (University of California, Santa Cruz, Howard Hughes Medical Institute); David Haussler (University of California, Santa Cruz, Howard Hughes Medical Institute); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We examine the ultraconserved elements as transcriptional regulators in neural differentiation of mouse embryonic stem cells.  We demonstrate with a luciferase reporter that several UCEs possess activity as cis-regulatory DNA elements and present current RNA-sequencing efforts to uncover novel UCE-derived ncRNAs that may be involved in transcriptional regulation.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12095.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<br><br>


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="12" valign="top" align="left"><b><span class=smhead1>Accepted Posters</span></b></td>
 </tr>
</table>
<div class="style1">
<a href="#prep">Preparing your Poster - Information and Poster Size</a><br />
<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a>
<a href="#schedule">Poster Schedule</a><br />
<a href="#cat">Poster Categories</a><br />
<a href="#search">Search for a Poster</a><br />

<br />
</div>



<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="poster_list950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="poster_list3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="poster_listf77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="poster_list2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="poster_list1685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="poster_list4158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="poster_list4238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="poster_listbb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="poster_list711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="poster_list39b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="poster_list7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="poster_listfdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="poster_listb2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="poster_lista34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="poster_list53f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="poster_listafd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="poster_listedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="poster_list27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="poster_list5c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="poster_liste35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="poster_listd227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="poster_lista946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="poster_listdc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="poster_list8ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="poster_list718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="poster_list8001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="http://www.iscb.org/ismbeccb2009/poster_list.php" method="POST">
<input type="hidden" name="search" value="1">
<a name="search"></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300;" name="slot"></td>
 </tr>
 <tr> 
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="last_name"></td>
 </tr>
 <tr> 
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="co_auth"></td>
 </tr>
 <tr> 
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="pres_title"></td>
 </tr>
 <tr> 
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>
 
</table>
<br><br>
</form>

<br />
<br />
<br />
<br />
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




