<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/poster_list.php?cat=N by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><br><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1"> 
 <tr> 
  <td height="12" valign="top" align="right"><b>Accepted Posters</b></td>
 </tr>
</table> 


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'N'- Microarrays'</b></td>
 </tr>
</table>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N01</td>
 </tr>
 <tr> 
  <td><b>iNNfovis: Neural Network Enhanced Information Visualization of High-Dimensional Microarray Data</b></td>
 </tr>
 <tr> 
  <td>Marjan Trutschl- <i>Louisiana State University and Louisiana State University Health Sciences Center in Shreveport</i></td>
 </tr>
 <tr> 
  <td>Urska Cvek (Louisiana State University and Louisiana State University Health Sciences Center in Shreveport, Computer Science and Center for Molecular and Tumor Virology); John Clifford (Louisiana State University Health Sciences Center in Shreveport, Department of Biochemistry and Molecular Biology); Rona Scott (Louisiana State University Health Sciences Center in Shreveport, Department of Microbiology and Immunology); Evan Boswell (Louisiana State University in Shreveport, Department of Computer Science); Phillip Kilgore (Louisiana State University in Shreveport, Department of Computer Science); John Wessler (Louisiana State University in Shreveport, Department of Computer Science); Zanobia Syed (Louisiana State University Health Sciences Center in Shreveport, Department of Biochemistry and Molecular Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: High-dimensional microarray data does not only create the need for the analysis of the data and interpretation of results, but also the need for the development of tools and methods that can handle such data. We present techniques that combine well-understood classic visualizations and neural-network algorithms, creating meaningful visual representations.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10377.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N02</td>
 </tr>
 <tr> 
  <td><b>Bayesian Gene Set Enrichment Analysis</b></td>
 </tr>
 <tr> 
  <td>David Rossell- <i>IRB Barcelona</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We formulate a GSEA generalization within the Bayesian paradigm which provides extra flexibility in testing the right hypothesis and considering more than 2 biological states and 2 hypotheses,and delivers easy to interpret results.The framework is general and can be used with any Bayesian hypothesis-testing probability model.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10396.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N03</td>
 </tr>
 <tr> 
  <td><b>An evaluation framework for statistical tests on microarray data</b></td>
 </tr>
 <tr> 
  <td>Dominik Mertens- <i>Center for Biotechnology</i></td>
 </tr>
 <tr> 
  <td>Michael Dondrup (Center for Biotechnology, Computational Genomics); Andrea Hueser (Center for Biotechnology, Genetics); Alexander Goesmann (Center for Biotechnology, Computational Genomics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Microarray experiments characteristically involve a small number of replicates causing unreliable estimates of the sample variance. We evaluate the performance of widely used statistical tests for generating ranked gene lists from two-channel direct comparisons for a variable number of replicates based on a highly replicative oligonucleotide microarray experiment.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10400.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N04</td>
 </tr>
 <tr> 
  <td><b>Gene expression analysis of oral tongue squamous cell carcinoma between different macroscopic appearances</b></td>
 </tr>
 <tr> 
  <td>Afsaneh Eslami- <i>Tokyo Medical and Dental University</i></td>
 </tr>
 <tr> 
  <td>Mayuko  Ishikawa (Tokyo Medical and Dental University, Department of Oral Restitution); Akiko  Hatano (Tokyo Medical and Dental University, Information Center for Medical Sciences); Ken  Miyaguchi (Tokyo Medical and Dental University, Information Center for Medical Sciences); Kaoru  Mogushi (Tokyo Medical and Dental University, Information Center for Medical Sciences); Hiroshi   Mizushima (Tokyo Medical and Dental University,  Information Center for Medical Sciences); Hiroshi  Watanabe (Tokyo Medical and Dental University, Department of Oral Restitution); Norihiko   Okada (Tokyo Medical and Dental University,  Department of Oral Restitution); Masahiko  Miura (Tokyo Medical and Dental University, Department of Oral Restitution); Hitoshi   Shibuya (Tokyo Medical and Dental University, Department of Head and Neck Reconstruction); Hiroshi Tanaka (Tokyo Medical and Dental University, Information Center for Medical Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Oral tongue squamous cell carcinoma has different macroscopic appearances; superficial, exophytic and invasive types. We investigated gene expression related to distinction of three types by microarray. Different gene expression was found between invasive and superficial types. Results suggest that different macroscopic appearances may be related to unusual gene expression.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10422.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N05</td>
 </tr>
 <tr> 
  <td><b>A modified LOESS normalization applied to miRNA arrays: a comparative evaluation</b></td>
 </tr>
 <tr> 
  <td>Davide Risso- <i>University of Padua</i></td>
 </tr>
 <tr> 
  <td>Maria Sofia Massa (University of Padua, Department of Statistical Sciences); Chiara Romualdi (University of Padua, Department of Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose a novel normalization (loessM) applied to microRNA arrays, based on loess algorithm, that scales data on the median expression values. LoessM is able to outperform other techniques in most experimental scenarios, giving the best results in term of specificity and sensitivity either on simulated and on real data.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10460.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N06</td>
 </tr>
 <tr> 
  <td><b>Time-resolved monitoring of the transcriptome of MCF-7 breast cancer cells during the emergence of cisplatin resistance </b></td>
 </tr>
 <tr> 
  <td>Martin Koch- <i>University Bonn</i></td>
 </tr>
 <tr> 
  <td>Niels Eckstein (University Düsseldorf, Institute of Human Genetics and Anthropology); Norbert Brenner (Caesar, Neurosensorik); Hans-Dieter Royer (University Düsseldorf, Institute of Human Genetics and Anthropology); Michael Wiese (University Bonn, Pharmaceutical Chemistry); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Cisplatin is an emerging new treatment modality of breast cancer, after failure of chemotherapy. Development of a resistant phenotype resembles a major obstacle in clinical cisplatin therapy. We monitored the gene expression of cisplatin treated MCF-7 cells. Microarray time series can resolve the development of a cisplatin resistant phenotype.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10467.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N07</td>
 </tr>
 <tr> 
  <td><b>Comparative analysis of mRNA isoforms features using statistical and learning methods  </b></td>
 </tr>
 <tr> 
  <td>Murlidharan Nair- <i>Indiana University South Bend</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: mRNA isoforms reflect the integrated outcome of molecular regulation and is thus a more effective measure towards understanding the state of the cell.  We have identified mRNA isoform features using statistical and recursive-SVM based feature selection methods. We address the question of understanding which features best represent class separation biologically.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10469.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N08</td>
 </tr>
 <tr> 
  <td><b>Integrating miRNAs and mRNAs data from microarray experiments. Testing the influence of miRNAs signatures into gene expression profiles.</b></td>
 </tr>
 <tr> 
  <td>David Gonzalez-Pisano- <i>Spanish National Cancer Research Centre (CNIO)</i></td>
 </tr>
 <tr> 
  <td>Marcos Malumbres (Spanish National Cancer Research Centre (CNIO), Molecular Oncology); Miguel Angel Piris (Spanish National Cancer Research Centre (CNIO), Molecular Pathology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We introduce a new approach to determine statistically relevant relationships between microRNAs profiles and gene expression signatures integrating both types of microarray experiments. The results obtained by this approach were experimentally validated.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10478.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N09</td>
 </tr>
 <tr> 
  <td><b>Probabilistic Search for Relevant Microarray Experiments</b></td>
 </tr>
 <tr> 
  <td>Jose Caldas- <i>Helsinki University of Technology</i></td>
 </tr>
 <tr> 
  <td>Nils Gehlenborg (European Bioinformatics Institute, Microarray Team); Ali Faisal (Helsinki University of Technology, Department of Information and Computer Science); Alvis Brazma (European Bioinformatics Institute, Microarray Team); Samuel Kaski (Helsinki University of Technology, Department of Information and Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Search for data sets in gene expression data repositories is commonly based on textual descriptions of the experimental setup. We introduce novel retrieval methods that incorporate the gene expression measurements into the search process in order to retrieve data sets in which similar biological processes are activated.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10496.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N10</td>
 </tr>
 <tr> 
  <td><b>METASIS: The mata-analysis tool for expression microarray</b></td>
 </tr>
 <tr> 
  <td>Mi-Kyung Lee- <i>KyungHee University</i></td>
 </tr>
 <tr> 
  <td>YangSeok Kim (KyungHee University, Department of Physiology); JinHo Yoo (Yonsei University College of Medicine, Cancer Metastasis Research Center); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed meta-analysis software for expression array, METASIS. Many different types of expression array data can be used in METASIS such as Affymetrix, Illumina, Agilent, ABI and two-dye style. For the meta-analysis, t-based modeling, parametric approach, and rank product, non-parametric approach, were implemented in METASIS.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10511.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N11</td>
 </tr>
 <tr> 
  <td><b>A comparison of Affymetrix exon expression values when preprocessed with different library files</b></td>
 </tr>
 <tr> 
  <td>Lingjia Kong- <i>Tampere University of Technology</i></td>
 </tr>
 <tr> 
  <td>Olli Yli-Harja (Tampere University of Technology, Department of Signal Processing); Reija Autio (Tampere University of Technology, Department of Signal Processing); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: The selection of the library files has an effect on the values of several exons and genes. In addition, this study may offer a reliable solution for the selection of suitable library files, and for the development of more advanced methods to be used in the exon expression data analysis.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10530.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N12</td>
 </tr>
 <tr> 
  <td><b>Filtering and Identifying non-reliable probes in Affymetrix GeneChip® platforms </b></td>
 </tr>
 <tr> 
  <td>Noura Chelbat- <i>Johannes Kepler University</i></td>
 </tr>
 <tr> 
  <td>Ulrich Bodenhofer (Johannes Kepler University, Institute of Bioinformatics); Sepp Hochreiter (Johannes Kepler University, Institute of Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Non-reliable/bad probes of oligonucleotide microarrays fail to spot signals which are detected and by the majority of probes in a probe set. We predict “bad” probes from the nucleotide sequence using SVMs and spectrum kernels with accuracies between 60 and 75% where, surprisingly, the models generalize to other platforms.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10483.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N13</td>
 </tr>
 <tr> 
  <td><b>Characterization of gene-specific intrinsic expression patterns by global analysis of microarray data</b></td>
 </tr>
 <tr> 
  <td>Changsik Kim- <i>Sookmyung Women's University</i></td>
 </tr>
 <tr> 
  <td>Jiwon Choi (Sookmyung Women's University, Department of Biological Sciences); Yanghee Jang (Sookmyung Women's University, Department of Biological Sciences); Sukjoon Yoon  (Sookmyung Women's University, Department of Biological Sciences); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a method to integrate heterogeneous microarray global data for physiome-wide analysis of gene-expression. We found that individual genes have unique levels of average expression and expressional variation in thousands of different tissues and experimental conditions. This feature was used to identify novel tissue (or disease)-selective gene expression.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10549.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N14</td>
 </tr>
 <tr> 
  <td><b>A database for meta-analysis of clinical microarray studies</b></td>
 </tr>
 <tr> 
  <td>Ian Hsu- <i>National Tsing Hua University</i></td>
 </tr>
 <tr> 
  <td>Wei-Chung Cheng (National Tsing Hua University, Biomedical Engineering and Environmental Sciences); Min-Lung Tsai (National Tsing Hua University, Biomedical Engineering and Environmental Sciences); Chung-Wei Chang (National Tsing Hua University, Biomedical Engineering and Environmental Sciences); Ching_Lung Huang (National Tsing Hua University, Biomedical Engineering and Environmental Sciences); Chaang-Ray Chen (National Tsing Hua University, Biomedical Engineering and Environmental Sciences); Wun-Yi Shu (National Tsing Hua University, Statistics); Yi-Chun Lin   (National Tsing Hua University, Statistics); Tzu-Hao Wang (Chang Gung Memorial Hospital and Chang Gung University, Obstetrics and Gynecology); Ji-Hong Hong (Chang Gung Memorial Hospital, Radiation Oncology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Features offered by this database can efficiently facilitate the searching and retrieval process to assure the reliability of human clinical microarray metadata. The database provides uniformly preprocessed data along with sets of QC metrics that can significantly improve the data quality and comparability of microarray data generated among different laboratories.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10582.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N15</td>
 </tr>
 <tr> 
  <td><b>Comparative analysis of gene expression across species</b></td>
 </tr>
 <tr> 
  <td>Ana Carolina Fierro Gutierrez- <i>K.U. Leuven</i></td>
 </tr>
 <tr> 
  <td>Peyman Zarrineh (Katholieke Universiteit Leuven, Department of Microbial and Molecular system); Kristof Engelen (Katholieke Universiteit Leuven, Department of Microbial and Molecular system); Lieve Verlinden (Katholieke Universiteit Leuven, Laboratorium voor Experimentele Geneeskunde en Endocrinologie); Guy Eelen (Katholieke Universiteit Leuven, Laboratorium voor Experimentele Geneeskunde en Endocrinologie); Els Vanoirbeek (Katholieke Universiteit Leuven, Laboratorium voor Experimentele Geneeskunde en Endocrinologie); Annemieke Verstuyf (Katholieke Universiteit Leuven, Laboratorium voor Experimentele Geneeskunde en Endocrinologie); Kathleen Marchal (Katholieke Universiteit Leuven, Department of Microbial and Molecular system); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: In this study we compared the response to vitaminD in human and mouse based on gene coexpression derived from microarray experiments. By applying a differential clustering approach we get a better view on which gene expression changes are species-specific and which are conserved between human and mouse.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10615.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N16</td>
 </tr>
 <tr> 
  <td><b>TAFFEL: A Tool for Parts Based Analysis of Differentially Expressed Genes</b></td>
 </tr>
 <tr> 
  <td>Petri Pehkonen- <i>University of Kuopio</i></td>
 </tr>
 <tr> 
  <td>Mitja Kurki (University of Kuopio, Department of Biosciences); Garry Wong (University of Kuopio, Department of Biosciences); Jussi Paananen (University of Kuopio, Department of Biosciences); Markus Storvik (University of Kuopio, Department of Biosciences); Mikael von und zu Fraunberg (Kuopio University Hospital, Department of Neurosurgery); Juha Jääskeläinen (Kuopio University Hospital, Department of Neurosurgery); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a software tool TAFFEL that performs analysis of differentially expressed genes by finding co-regulated and co-functional gene sub groups and their internecine correlations. Analysis of data from forskolin treated human hepatocytes and ruptured saccular cerebral artery aneurysm reveal the key processes driven by different sets of regulatory proteins.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10619.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N17</td>
 </tr>
 <tr> 
  <td><b>A NEW MANIFOLD LEARNING APPROACH FOR ANALYSIS AND VISUALISATION OF DIFFERENTIAL GENE EXPRESSION </b></td>
 </tr>
 <tr> 
  <td>Jitender Cheema- <i>John Innes Centre</i></td>
 </tr>
 <tr> 
  <td>Jo Dicks  (John Innes Centre, Computational and System Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a new method to create, visualise and validate clusters of gene expression profiles. Our approach combines ideas from machine learning and graph partitioning (manifold Laplacian embedding) and, through a 3D visualisation process, allows the user to interact with their results, using their expert knowledge to refine them.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10629.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N18</td>
 </tr>
 <tr> 
  <td><b>Filtering low-signal probesets improves enrichment analysis in microarray studies</b></td>
 </tr>
 <tr> 
  <td>Krzysztof Goryca- <i>Medical Center for Postgraduate Education and the Maria Sk&#322;odowska-Curie Memorial Cancer Center</i></td>
 </tr>
 <tr> 
  <td>Tymon Rubel (Maria Sk&#322;odowska-Curie Memorial Cancer Center, Department of Gastroenterology and Hepatology); Lucjan Wyrwicz (Maria Sk&#322;odowska-Curie Memorial Cancer Center, Department of Gastroenterology and Hepatology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Statistical techniques for analysis of microarrays data sets possess limitations related to low signal-to-noise ratio, especially for least abundant mRNAs. Here we report, that a simple procedure - filtering of genes with unfavorable signal-to-noise ratio in a given microarray experiment- can result in better functional description of microarray data.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10635.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N19</td>
 </tr>
 <tr> 
  <td><b>Bicluster-based meta-analysis of microarray data</b></td>
 </tr>
 <tr> 
  <td>Edward Curry- <i>University of Edinburgh</i></td>
 </tr>
 <tr> 
  <td>Simon Tomlinson (University of Edinburgh, MRC Centre for Regenerative Medicine); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a meta-analysis technique to identify, in large gene expression datasets, subsets of the data that facilitate prediction of different roles a gene or group of genes may have in different biological contexts. We demonstrate use of our approach on publicly available microarray data to explore observed effects in novel datasets.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10700.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N20</td>
 </tr>
 <tr> 
  <td><b>Clustering of Temporal Gene Expression Data Across Multiple Treatments with a Bayesian Nonparametric Mixture Model</b></td>
 </tr>
 <tr> 
  <td>Ana Paula Sales- <i>Duke University</i></td>
 </tr>
 <tr> 
  <td>Thomas Kepler (Duke University, Biostatistics and Bioinformatics); Feng Feng (Duke University, Biostatistics and Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Time-course gene expression data from dendritic cells based on four distinct treatments is modeled  with Gaussian processes and clustered with a Dirichlet process.  This mixture model does not impose any parametric form for the time-trajectories nor requires specification of the number of clusters, inferring it from the data.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10708.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N21</td>
 </tr>
 <tr> 
  <td><b>Ecological Genomics: Construction of Molecular Pathways Responsible for Gene Regulation and Adaptation to Heavy Metal Stress in Arabidopsis thaliana and Raphanus sativus. </b></td>
 </tr>
 <tr> 
  <td>Lynda Villagomez- <i>Loyola Marymount University</i></td>
 </tr>
 <tr> 
  <td>Tatiana Tatarinova (Loyola Marymount University, Mathematics); Gary Kuleck (Loyola Marymount University, Biology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Using Pathway Studio, we explored molecular pathways for heavy metal stress responsiveness in Arabidopsis thaliana and Raphanus sativus. We built a prototype response network in Arabidopsis. 8,059 ortholog pairs were identified between Raphanus and Arabidopsis (>95% match). The 50 best candidate genes in Raphanus were nominated for RT-PCR validation.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10714.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N22</td>
 </tr>
 <tr> 
  <td><b>Multi Experiment Matrix - Webtool for finding co-expressed genes over hundreds of datasets</b></td>
 </tr>
 <tr> 
  <td>Priit Adler- <i>University of Tartu</i></td>
 </tr>
 <tr> 
  <td>Raivo Kolde (University of Tartu, Institute of Computer Science); Meelis Kull (University of Tartu, Institute of Computer Science); Aleksandr Tkatchenko (University of Tartu, Institute of Computer Science); Hedi Peterson (University of Tartu, Institute of Molecular and Cell Biology); Jüri Reimand (University of Tartu, Institute of Computer Science); Jaak Vilo (University of Tartu, Institute of Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a query engine atop of the microarray experiments from ArrayExpress that performs search for co-expressed genes over hundreds of datasets at a time. Given the query gene MEM finds a list of genes that have similar expression in many datasets. (MEM, http://biit.cs.ut.ee/mem)</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10719.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N23</td>
 </tr>
 <tr> 
  <td><b>Chipster microarray data analysis software - new release with extended functionality</b></td>
 </tr>
 <tr> 
  <td>Eija Korpelainen- <i>CSC - the Finnish IT Center for Science</i></td>
 </tr>
 <tr> 
  <td>Jarno Tuimala (CSC - IT Centre for Science, Software solutions); Aleksi Kallio (CSC - IT Centre for Science, Software engineering); Taavi Hupponen (CSC - IT Centre for Science, Software engineering); Petri Klemelä (CSC - IT Centre for Science, Software engineering); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Chipster (http://chipster.csc.fi/) offers an intuitive GUI to a comprehensive collec¬tion of up-to-date microarray data analysis methods, such as those developed in the R/Bioconductor project. Chipster supports Affymetrix, Illumina, Agilent and cDNA arrays, and it is open source. The new release has many new analysis tools, visualizations and supported chip types.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10636.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N24</td>
 </tr>
 <tr> 
  <td><b>Gene Expression data Classification using Filter method and Discrete Wavelet Feature Selection</b></td>
 </tr>
 <tr> 
  <td>Ho Sun Shon- <i>CBITRC, PTERC, Chungbuk National University</i></td>
 </tr>
 <tr> 
  <td>Dong Gyu Lee (CBITRC, PTERC, Chungbuk National University, Computer Sciencce);  Keun Ho Ryu (CBITRC, PTERC, Chungbuk National University, Computer Sciencce); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: In this research, the gene selection method that well-reflects the characteristics of microarray data was applied. That is, the performance was compared and evaluated by using the Wavelet method applicable to the high-dimensional data through similar gene selection methods by making classifiers with the extracted genes and applying the test data set. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10785.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N25</td>
 </tr>
 <tr> 
  <td><b>Splicing prediction is dominated by gene expression changes</b></td>
 </tr>
 <tr> 
  <td>Axel Rasche- <i>MPI for Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>Ralf Herwig (MPI for Molecular Genetics, Vertebrate Genomics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a systematic evaluation of available splicing prediction methods advancing the theoretical part of alternative splicing analysis. The compared methods focus on high-throughput screening methods. The evaluation shows a lack of performance for not differentially expressed genes. This is particularly interesting for the proposed coupling of transcription and splicing.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11621.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N26</td>
 </tr>
 <tr> 
  <td><b>High accuracy for NaÃ¯ve Bayesian Tree classifying disease-associated copy number variation in mental retardation</b></td>
 </tr>
 <tr> 
  <td>Jayne Hehir-Kwa- <i>Microarray Facility</i></td>
 </tr>
 <tr> 
  <td>Nienke Wieskamp (UMCN, Human Genetics); Caleb Webber (Oxford University, Functional Genomics); Christian Gilissen (UMCN, Human Genetics); Rolph Pfundt (UMCN, Human Genetics); Chris Ponting (Oxford University, Functional Genetics); Joris Veltman (UMCN, Human Genetics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have discovered several structural and functional features that significantly differ between rare de novo CNVs associated with mental retardation (MR) and benign CNVs found in healthy individuals. We train a classifier with these features and achieve a high accuracy in a replication study with individuals suffering from unexplained MR. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11745.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N27</td>
 </tr>
 <tr> 
  <td><b>Microarray Analysis to investigate Mechanisms of Metabolic Syndrome</b></td>
 </tr>
 <tr> 
  <td>Tiffany Morris- <i>University of Cambridge</i></td>
 </tr>
 <tr> 
  <td>Mark Vickers (University of Auckland, Liggins Institute); Peter Gluckman (University of Auckland, Liggins Institute); Stewart Gilmour (University of Cambridge, Pathology Department); Nabeel Affara (University of Cambridge, Pathology Department); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Microarray was used to investigate the effects of neonatal leptin treatment on the metabolic phenotype of adult female offspring of undernourished mothers. Liver samples from eight treatment groups were hybridised to the Illumina array and R/bioconductor was used to analyse the results.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11763.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N28</td>
 </tr>
 <tr> 
  <td><b>Fusion of sequence data and microarray data, a systematic approach toward cross-species comparison </b></td>
 </tr>
 <tr> 
  <td>Peyman Zarrineh- <i>Katholieke Universiteit Leuven</i></td>
 </tr>
 <tr> 
  <td>Carolina Fierro (Postdoc researcher, Department of Microbial and Molecular systems, Katholieke Universiteit Leuven, Kasteelpark Arenberg 20, 3001 Leuven); Bart De Moor (Professor, Department of Electrical engineering, Katholieke Universiteit Leuven, Kasteelpark Arenberg 10, 3001 Leuven); Kathleen Marchal (Professor, Department of Microbial and Molecular systems, Katholieke Universiteit Leuven, Kasteelpark Arenberg 20, 3001 Leuven); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: In this study functional conservation between bacterial species has been studied by utilizing a new methodology developed for cross-species comparison. This methodology combines microarray data and orthology data in an elaborated manner.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11877.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N29</td>
 </tr>
 <tr> 
  <td><b>Combining Semantic Relations from the Literature and DNA Microarray Data for Novel Hypotheses Generation</b></td>
 </tr>
 <tr> 
  <td>Dimitar Hristovski- <i>University of Ljubljana, Medical Faculty</i></td>
 </tr>
 <tr> 
  <td>Andrej  Kastrin (University Medical Centre, Institute of Medical Genetics); Borut Peterlin (University Medical Centre, Institute of Medical Genetics); Thomas Rindflesch (National Institutes of Health, National Library of Medicine); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Although microarray experiments have great potential to support progress in biomedicine, results are not easy to interpret. We describe a method and an application that integrates semantic relations extracted from the literature with microarray results and show the benefits for interpretation of results and novel hypotheses generation.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11889.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N30</td>
 </tr>
 <tr> 
  <td><b>GeSETbench: Gene SET analysis workbench for microarray data </b></td>
 </tr>
 <tr> 
  <td>Jaeyoung Kim- <i>Kyungpook National University</i></td>
 </tr>
 <tr> 
  <td>Miyoung Shin (Kyungpook National University, School of Electrical Engineering & Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: GeSETbench is a gene-set analysis and visualization for identifying significant gene-sets in a parametric way and in a nonparametric way, based on two sample groups of gene expression data and biological resources. In particular, we consider distribution model of gene ranking scores produced by several different importance ranking methods.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11912.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N31</td>
 </tr>
 <tr> 
  <td><b>Robust Extraction of Functional Signals from Gene Set Analysis using a Generalized Threshold Free Scoring Function</b></td>
 </tr>
 <tr> 
  <td>Petri Törönen- <i>Helsinki University</i></td>
 </tr>
 <tr> 
  <td>Pauli Ojala (Finnish Red Cross, research unit); pekka marttinen (University of Helsinki, Statistics department); liisa holm (Helsinki University, Institute of Biotechnology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose a new scoring function, Gene Set Z-score (GSZ), for threshold free gene set enrichment analysis. GSZ performs over-representation analysis that takes the actual differential expression scores into account. The method surpasses other scoring functions in artificial and real data comparisons.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11924.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N32</td>
 </tr>
 <tr> 
  <td><b>Improving differential expression detection using difference normalization</b></td>
 </tr>
 <tr> 
  <td>Marc Hulsman- <i>Delft University of Technology</i></td>
 </tr>
 <tr> 
  <td>Anouk Leusink (University of Twente, Department of Tissue Regeneration); Eugene van Someren (Department of Applied Biology, Radboud Universiteit Nijmegen); Koen J. Dechering (Schering-Plough Research Institute, Department of Molecular Pharmacology); Jan de Boer (University of Twente, Department of Tissue Regeneration); Marcel J.T.  Reinders (Delft University of Technology, Information and Communication Theory Group); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Microarrays are particularly sensitive to experimental conditions, causing unwanted signal differences between arrays. These differences can be attributed to amplification, hybridization and array location effects. These technical effects are not adequately removed by existing methods. We propose a new normalization method, showing that it significantly improves differential expression detection.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11965.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N33</td>
 </tr>
 <tr> 
  <td><b>Sparse Factor Analysis for Detecting Copy Number Variations (CNVs)</b></td>
 </tr>
 <tr> 
  <td>Andreas Mitterecker- <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>Djork-Arné Clevert (Johannes Kepler University Linz, Institute of Bioinformatics); Andreas Mayr (Johannes Kepler University Linz, Institute of Bioinformatics); An De Bondt (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Willem Talloen (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Marianne Tuefferd (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Hinrich Göhlmann (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Sepp Hochreiter (Johannes Kepler University, Institute of Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Most reported CNVs affect less than three HapMap samples. We model these sparse CNVs by Laplace or multimodal distributions, where learning is based on variational and EM approaches. With Affymetrix SNP6 chips on the HapMap data we found novel CNVs. Moreover many known CNVs seem to be false positives.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12028.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N34</td>
 </tr>
 <tr> 
  <td><b>Construction of Metagenes by Conditional Factor Analysis</b></td>
 </tr>
 <tr> 
  <td>Andreas Mayr- <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>Djork-Arne Clevert (Johannes Kepler University Linz, Institute of Bioinformatics); Andreas Mitterecker (Johannes Kepler University Linz, Institute of Bioinformatics); An De Bondt (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Willem Talloen (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Hinrich Göhlmann  (Janssen Pharmaceutica, Johnson & Johnson Pharmaceutical Research & Development); Sepp Hochreiter (Johannes Kepler University Linz, Institute of Bioinformatics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: To combine microarray gene selections and to transfer gene signatures across platforms, we construct "metagenes" by conditional factor analysis. A hidden factor in expression values that is correlated to a chosen genes' expression value is its "metagene", where its variance is the hyperparameter determining robustness and number of included genes.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12053.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N35</td>
 </tr>
 <tr> 
  <td><b>Multivariate Methods for Genomic Data Integration and Visualization</b></td>
 </tr>
 <tr> 
  <td>Alex Sanchez- <i>University of Barcelona</i></td>
 </tr>
 <tr> 
  <td>Francesc Carmona (University of Barcelona, Statistics); Ferran Reverter (University of Barcelona, Statistics); Esteban Vegas (University of Barcelona, Statistics); José Fernández-Real (Hospital Universitari de Girona, Diabetes, Endocrinología y Nutrición ); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We discuss the application of two multivariate statistics approaches to integrate bio-molecular information: Multiple Factorial Analysis and Ecological Data Analysis, each combining several traditional or new multivariate statistical methods. The techniques are applied to an unpublished dataset consisting of three different data types: DGGE, microarrays and clinical variables.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12024.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster N36</td>
 </tr>
 <tr> 
  <td><b>Visualization of Large Microarray Experiments with Space Maps </b></td>
 </tr>
 <tr> 
  <td>Nils Gehlenborg- <i>European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>Nils Gehlenborg (European Bioinformatics Institute, Microarray Team); Alvis Brazma (European Bioinformatics Institute, Microarray Team); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Microarray studies that include a large number of samples have become increasingly common over the last few years. We present the Space Maps visualization technique, which can visualize data sets with hundreds or thousands of samples, a task at which state-of-the-art techniques such as heatmaps fail.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/12093.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<br><br>


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="12" valign="top" align="left"><b><span class=smhead1>Accepted Posters</span></b></td>
 </tr>
</table>
<div class="style1">
<a href="#prep">Preparing your Poster - Information and Poster Size</a><br />
<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a>
<a href="#schedule">Poster Schedule</a><br />
<a href="#cat">Poster Categories</a><br />
<a href="#search">Search for a Poster</a><br />

<br />
</div>



<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="poster_list950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="poster_list3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="poster_listf77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="poster_list2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="poster_list1685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="poster_list4158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="poster_list4238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="poster_listbb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="poster_list711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="poster_list39b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="poster_list7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="poster_listfdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="poster_listb2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="poster_lista34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="poster_list53f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="poster_listafd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="poster_listedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="poster_list27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="poster_list5c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="poster_liste35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="poster_listd227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="poster_lista946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="poster_listdc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="poster_list8ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="poster_list718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="poster_list8001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="http://www.iscb.org/ismbeccb2009/poster_list.php" method="POST">
<input type="hidden" name="search" value="1">
<a name="search"></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300;" name="slot"></td>
 </tr>
 <tr> 
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="last_name"></td>
 </tr>
 <tr> 
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="co_auth"></td>
 </tr>
 <tr> 
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="pres_title"></td>
 </tr>
 <tr> 
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>
 
</table>
<br><br>
</form>

<br />
<br />
<br />
<br />
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




