<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/poster_list.php?cat=R by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><br><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1"> 
 <tr> 
  <td height="12" valign="top" align="right"><b>Accepted Posters</b></td>
 </tr>
</table> 


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'R'- Proteomics'</b></td>
 </tr>
</table>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R01</td>
 </tr>
 <tr> 
  <td><b>MRMaid: automating the design of multiple reaction monitoring (MRM) experiments using expert knowledge and MS/MS data-mining</b></td>
 </tr>
 <tr> 
  <td>Jennifer Mead- <i>Cranfield University</i></td>
 </tr>
 <tr> 
  <td>Luca Bianco (Cranfield University, Bioinformatics Group); Conrad Bessant (Cranfield University, Bioinformatics Group); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Multiple reaction monitoring (MRM) is a popular technique that employs tandem MS to quantify multiple proteins in a single experiment.  MRMaid (pronounced ‘mermaid’) is a new service (www.mrmaid.info) that answers the challenging question faced when designing MRM experiments: which peptide/product ions should I monitor for my protein of interest?</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10332.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R02</td>
 </tr>
 <tr> 
  <td><b>spectral clustering in peptidomics studies helps to unravel modification profile of biologically active peptides, and enhances peptide identification rate.</b></td>
 </tr>
 <tr> 
  <td>Gerben Menschaert- <i>University of Ghent</i></td>
 </tr>
 <tr> 
  <td>Tom Vandekerckhove (Ghent University, Department of Molecular Biotechnology); Wim Vancriekinge (Ghent University, Department of Molecular Biotechnology); Eisuke Hayakawa (K.U. Leuven, Research group of Functional Genomics and Proteomics); Bart  Landuyt (K.U. Leuven, Research group of Functional Genomics and Proteomics); Liliane Schoofs (K.U. Leuven, Research group of Functional Genomics and Proteomics); Walter Luyten (K.U. Leuven, Department of Woman & Child, Biomedical Science Group); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Identification of bio-active peptides (the peptidome) is mostly attempted by mass spectrometry. However, the identification rates are often unsatisfactory, mainly caused by the wealth of peptide modifications. Including a spectral clustering step into a peptidomics identification pipeline results in doubled identification rates and a complete modification profile (including new ones).</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11952.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R03</td>
 </tr>
 <tr> 
  <td><b>FOLD SPACE GRAPHS &#8211; A NEW METHOD TO EXPLORE EVOLUTIONARY RELATIOSHIPS BETWEEN PROTEIN STRUCTURES </b></td>
 </tr>
 <tr> 
  <td>Natalja Kurbatova- <i>EBI</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We have developed a new method for exploring evolutionary relations between protein structures by navigating through the fold space under assumption that protein structures have evolved by a stepwise mutation processes. This method is based on the ESSM algorithm for detecting structural mutations and construction of fold space graphs. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10387.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R04</td>
 </tr>
 <tr> 
  <td><b>Prediction of glycosylation sites in proteins</b></td>
 </tr>
 <tr> 
  <td>Karin Julenius- <i>Karolinska Institutet</i></td>
 </tr>
 <tr> 
  <td><i>No additional authors</i></td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Protein glycosylation is more abundant and structurally diverse than all other types of post-translational modifications. We develop prediction methods that recognize glycosylation sites in proteins from amino acid sequence alone. All our prediction servers are made publically available at www.cbs.dtu.dk/services, the most recent being NetPGlyc, a predictor of proteoglycan sites.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10454.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R05</td>
 </tr>
 <tr> 
  <td><b>A new systematic approach to identify functions of human extracellular proteins using yeast</b></td>
 </tr>
 <tr> 
  <td>Solip  Park- <i>POSTECH</i></td>
 </tr>
 <tr> 
  <td>Ok-Kyu Song (POSTECH, Life science); Hyung-Jin Lee (POSTECH, Life science); Vit Kim (POSTECH, Life science); Jae-Seong  Yang (POSTECH, I-Bio); Sanguk Kim (POSTECH, Life science); Sung Key Jagn (POSTECH, Life science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Cell signaling, differentiation and proliferation are governed by extracellular proteins.  Despite their functions importance, a large part of the proteins are remained unknown.  Here, we present a new approach to determine functions of mammalian proteins using co-cultivation of yeast and mammalian cells named Zymogand system. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10457.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R06</td>
 </tr>
 <tr> 
  <td><b>X-Tracker: A Generic Quantitation Tool for MS-based Proteomics</b></td>
 </tr>
 <tr> 
  <td>Luca Bianco- <i>Cranfield University</i></td>
 </tr>
 <tr> 
  <td>Luca Bianco (Bioinformatics Group, Cranfield Health); Conrad Bessant (Bioinformatics Group, Cranfield Health); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: X-Tracker is a new piece of software allowing Mass Spectrometry-based protein quantitation.  Through an abstraction of the main steps involved in quatitation, X-Tracker is able to support all the current quantitative protocols, both at  MS or Tandem-MS level, and provide a flexible, platform-independent and easy-to-use quantitation environment.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10472.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R07</td>
 </tr>
 <tr> 
  <td><b>Model-based Imputation of Missing Data in Bottom-Up MS-Based Proteomics</b></td>
 </tr>
 <tr> 
  <td>Yuliya Karpievitch- <i>Texas A&M University</i></td>
 </tr>
 <tr> 
  <td>Jeff Stanley (Texas A&M, Statistics); Thomas Taverner (PNNL, Biological Sciences); Jianhua Huang (Texas A&M, Statistics); Joshua Adkins (PNNL, Biological Sciences); Charles Ansong (PNNL, Biological Sciences); Fred Heffron (Oregon Health and Science University, Microbiology); Hyunjin Yoon (Oregon Health and Science University, Microbiology); Thomas Metz (PNNL, Biological Sciences); Wei-Jun Qian (PNNL, Biological Sciences); Dick Smith (PNNL, Biological Sciences); Alan Dabney (Texas A&M, Statistics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a model-based approach for filtering low quality proteins and peptides and then imputing the missing values. Our likelihood model accounts for the fact that many peptide measurements will be unobserved, due to (1) random “missingness” and (2) falling below the detection limit, i.e. censoring.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10521.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R08</td>
 </tr>
 <tr> 
  <td><b>Understanding phytocystatin structure/function relationships through homology modeling</b></td>
 </tr>
 <tr> 
  <td>Juan Vorster- <i>University of Pretoria</i></td>
 </tr>
 <tr> 
  <td>Özlem Tastan Bishop (University of Pretoria, Biochemistry); Urte Schlüter (University of Pretoria, Plant Science); Karl Kunert (University of Pretoria, Plant Science); Dominique Michaud (Université Laval, Département de Phytologie); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Amino acid substitutions at positively selected sites were performed on a model cystatin and shown to modulate the inhibitory potency against. Using a homology modeling approach we investigate structure/function relationships between the mutant cystatins and various cysteine proteases, with the aim of better understanding their mode of action.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10523.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R09</td>
 </tr>
 <tr> 
  <td><b>Detecting specificity residues in protein interactions</b></td>
 </tr>
 <tr> 
  <td>Qiang Luo- <i>University of Oxford</i></td>
 </tr>
 <tr> 
  <td>Rebecca Hamer (University of Oxford, Department of Statistics); Charlotte Deane (University of Oxford, Department of Statistics); Gesine Reinert (University of Oxford, Department of Statistics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Detection of residues which determine the specificity of protein-protein interactions is currently non trivial. We propose a novel algorithm which considers a protein as a network with 20 types of nodes and show improved results over existing methods.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10609.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R10</td>
 </tr>
 <tr> 
  <td><b>Adaptation of viruses towards their hosts -A proteome scale analysis</b></td>
 </tr>
 <tr> 
  <td>Iris Bahir- <i>Hebrew University</i></td>
 </tr>
 <tr> 
  <td>Michal Linial (Hebrew University, Biological Chemistry); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Viral evolution is dominated by an extreme high mutation rate, a large population sizeand by the inherent ability for fast exchange of genetic material. We tested the forces that shape viruses by monitoring all pairs of amino acid distribution and codon usage for viruses and hosts.  The significant findings are discussed.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10718.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R11</td>
 </tr>
 <tr> 
  <td><b>Iterative precursor ion selection for LC-MS/MS based shotgun proteomics</b></td>
 </tr>
 <tr> 
  <td>Alexandra Zerck- <i>Max Planck Institute for Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>Johan Gobom (University of Gothenborg/Sahlgrenska University Hospital, Department of Neuroscience and Physiology); Hans Lehrach (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Knut Reinert (FU Berlin, Department for Computer Science); Eckhard Nordhoff (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We present a result-driven, iterative approach for precursor ion selection forLC-MS/MS based shotgun proteomics. In our approach identification results of previous iterations guide the precursor ionselection. This way the efficiency of an MS/MS analysis is significantlyincreased, as data redundancy and analysis time are reduced.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10731.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R12</td>
 </tr>
 <tr> 
  <td><b>Graph theoretic properties of known protein complexes</b></td>
 </tr>
 <tr> 
  <td>Suzanne Gallagher- <i>University of Colorado</i></td>
 </tr>
 <tr> 
  <td>Debra Goldberg (University of Colorado, Computer Science); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We examined protein complexes in the protein-protein interaction network in order to determine what graph theoretic properties correspond to complexes. We examined approximately 150 known protein complexes from MIPS and iPFam and looked at edge density, clustering coefficient, betweenness centrality, vertex and edge connectivity, and subgraph properties for each.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10730.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R13</td>
 </tr>
 <tr> 
  <td><b>A Bayeisan Approach to the Quantification of Overlapping Peptides in a MALDI-TOF Mass Spectrum</b></td>
 </tr>
 <tr> 
  <td>Qi Zhu- <i>I-Biostat</i></td>
 </tr>
 <tr> 
  <td>Adetayo Kasim (I-Biostat, Universiteit Hasselt); Dirk  Valkenborg (I-Biostat, Katholieke Universiteit Leuven); Ivy Jansen (I-Biostat, Universiteit Hasselt); Tomasz  Burzykowski (I-Biostat, Universiteit Hasselt); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: In MALDI-TOF mass spectra, it occurs that peptide peaks coincide with each other. The quantification of relative abundances and exact masses of these overlapping peptides is problematic. We propose a Bayesian model to address this problem and apply the method to real-life data sets from a controlled mass spectrometry experiment.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/45/10868.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R14</td>
 </tr>
 <tr> 
  <td><b>Confirming alternative protein isoforms in Drosophila</b></td>
 </tr>
 <tr> 
  <td>Michael Liam Tress- <i>Cnio</i></td>
 </tr>
 <tr> 
  <td>Michael Tress (Spanish National Cancer Centre (CNIO), Structural and Biological Computation); Alfonso Valencia (Spanish National Cancer Centre (CNIO), Structural and Biological Computation); Bernd Bodenmiller (ETH , Institute of Molecular Systems Biology ); Ruedi Aebersold (ETH , Institute of Molecular Systems Biology ); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Two recent large scale proteomics studies generated extensive peptide catalogs from the Drosophila melangaster proteome. The analysis of this proteomic data confirmed the presence of multiple alternative gene products for over a hundred Drosophila genes.  The analysis highlights the growing importance of proteomics in the validation of predicted proteins.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/48/11550.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R15</td>
 </tr>
 <tr> 
  <td><b>Crystal structures of thymidylate kinase from Sulfolobus tokodaii and Aquifex aeolicus</b></td>
 </tr>
 <tr> 
  <td>Jigisha Darbha- <i>Vellore Institute of Technology</i></td>
 </tr>
 <tr> 
  <td>J Jeyakanthan (National Synchrotron Radiation Research Center, 101 Hsin-Ann Road, Hsinchu Science Park, Hsinchu 30076, -); S.P Kanaujia (Indian Institute of Science, Bangalore 560 012, Bioinformatics Centre, Supercomputer Education and Research Centre); Z.A Rafi (Madurai Kamaraj University, Madurai 625 021, Bioinformatics Centre, School of Biotechnology ); N Nakagawa (RIKEN SPring-8 Center, Harima Institute, 1-1-1 Kouto, Sayo, Hyogo 679-5148, Graduate School of Science Osaka Univ., 1-1 Yamadaoka, Suita, Osaka, 565-0871, -); A Shinkai (RIKEN SPring-8 Center, Harima Institute, 1-1-1 Kouto, Sayo, Hyogo 679-5148, -); S Kuramitsu (RIKEN SPring-8 Center, Harima Institute, 1-1-1 Kouto, Sayo, Hyogo 679-5148, Graduate School of Science, Osaka University, Toyonaka, Osaka 560-0043, -); K Sekar (Indian Institute of Science, Bangalore 560 012, Bioinformatics Centre, Supercomputer Education and Research Centre); S Yokoyama (RIKEN SPring-8 Center, Harima Institute, 1-1-1 Kouto, Sayo, Hyogo 679-5148, Genomic Sciences Center, Yokohama Institute, RIKEN, 1-7-22 Suehiro-cho, Tsurumi, Yokohama 230-0045, Department of Biophysics and Biochemistry, Graduate School of Science, University of Tokyo, 7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033, -); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Our report provides insights into the structural features of thymidylate kinase from two thermophilic microorganisms, Sulfolobus tokodaii and Aquifex aeolicus. </td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11567.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R16</td>
 </tr>
 <tr> 
  <td><b>Improving protein identification by MsPI using peak intensity</b></td>
 </tr>
 <tr> 
  <td>Alessandra Tiengo- <i>UniversitÃ  degli Studi di Pavia</i></td>
 </tr>
 <tr> 
  <td>Nicola Barbarini (UniversitÃ  degli Studi di Pavia, Dipartimento di Informatica e Sistemistica); Sonia Troiani (Nerviano Medical Sciences, Biotechnology Department); Luisa Rusconi (Nerviano Medical Sciences, Biotechnology Department); Paolo Magni (UniversitÃ  degli Studi di Pavia, Dipartimento di Informatica e Sistemistica); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: This work presents MsPI, a software tool developed to perform protein identification by Peptide Mass Fingerprinting approach. Compared with other software tools, MsPI shows better performance, especially when are not considered only the masses of the peak list but also the intensity values.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11642.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R17</td>
 </tr>
 <tr> 
  <td><b>A Robust Statistical Approach for Detecting Differential Protein Expression in iTRAQ MS-Experiments</b></td>
 </tr>
 <tr> 
  <td>Holger Froehlich- <i>Cellzome AG</i></td>
 </tr>
 <tr> 
  <td>Marcus Bantscheff (Cellzome AG,  ); Gerard Joberty (Cellzome AG,  ); Yann Abraham (Cellzome AG,  ); Judith Schlegl (Cellzome AG,  ); Seon-Hi Jang (Max-Planck Institute for Molecular Genetics,  ); Bodo Lange (Max-Planck Institute for Molecular Genetics,  ); Gerard Drewes (Cellzome AG,  ); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We devise a robust statistical framework for iTRAQ MS data normalization and assessment of differentially expressed proteins. Normalization is carried out via a median polish procedure followed by a robust linear model fitted to the residuals. For differential protein expression we use limma.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11917.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R18</td>
 </tr>
 <tr> 
  <td><b>On the beta binomial model for spectral count data in label-free tandem mass spectrometry-based proteomics</b></td>
 </tr>
 <tr> 
  <td>Thang Pham- <i>VU University Medical Center</i></td>
 </tr>
 <tr> 
  <td>Sander Piersma (VU University Medical Center, OncoProteomics); Marc Warmoes (VU University Medical Center, OncoProteomics); Connie Jimenez (VU University Medical Center, OncoProteomics); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: We propose to use the beta-binomial distribution for differential analysis of protein abundances expressed in spectral counts in label-free mass spectrometry-based proteomics. The beta binomial test performs favorably in comparison to other methods on several datasets. A software package is implemented for parameter estimation and inference of beta-binomial models.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11938.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R19</td>
 </tr>
 <tr> 
  <td><b> Novel cancer biomarkers identified by a high through-put immuno-proteomic approach</b></td>
 </tr>
 <tr> 
  <td>Andrea Pierleoni- <i>Externautics</i></td>
 </tr>
 <tr> 
  <td>Renata Grifantini (Externautics, R&D); Alberto  Grandi (Externautics, R&D); Susanna Campagnoli (Externautics, R&D); Renzo Nogarotto (Externautics, R&D); Piero Pileri (Externautics, R&D); Elena Canidio (Primm, R&D); Davide Cattaneo (Primm, R&D); Massimiliano Pagani (Primm, R&D); Paolo Sarmientos (Externautics - PRIMM, R&D); Sergio Abrignani (INGM, Istituto Nazionale Genetica Molecolare); Guido Grandi (Externautics, Scientific Advisory Board); Giuseppe Viale (European Institute of Oncology and University of Milan , Division of Pathology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: 8 novel cancer-specific biomarkers were identified by a high through-put immuno-proteomic approach, based on the direct detection of tumor-associated-proteins on clinical tumor tissues by immunohistochemistry. The screening is sill in progress and a  library of 1600 mouse antisera is currently been tested versus the most common human cancer tissues.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/43/11858.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R20</td>
 </tr>
 <tr> 
  <td><b>ProSE - a Rich Internet Application to securely Store, Organise, and Analyse Quantitative Proteomics Experiments</b></td>
 </tr>
 <tr> 
  <td>Stefan Albaum- <i>Bielefeld University</i></td>
 </tr>
 <tr> 
  <td>Heiko Neuweger (Bielefeld University, Computational Genomics, Center for Biotechnology); Sita Lange (Bielefeld University, Computational Genomics, Center for Biotechnology); Dominik Mertens (Bielefeld University, Computational Genomics, Center for Biotechnology); Jörn Kalinowski (Bielefeld University, Institute for Genome Research and Systems Biology (IGS), Center for Biotechnology); Tim W. Nattkemper (Bielefeld University, Biodata Mining & Applied Neuroinformatics Group, Faculty of Technology); Alexander Goesmann (Bielefeld University, Computational Genomics, Center for Biotechnology); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: Tandem mass spectrometry coupled to liquid chromatography in combination with stable isotope labeling is able to measure the expression of hundreds of peptides in one experiment. To cope with the arising amounts of data and help in the conduction of these experiments we have developed the web application ProSE.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10543.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster R21</td>
 </tr>
 <tr> 
  <td><b>Epitope mapping of monospecific polyclonal antisera raised against KRAB zink finger proteins</b></td>
 </tr>
 <tr> 
  <td>Hans Thiesen- <i>Institute of Immunology</i></td>
 </tr>
 <tr> 
  <td>Peter Lorenz (University of Rostock, Institute of Immunology); Cristina Al-Khalili Szigyarto (Royal Institute of Technology (KTH), AlbaNova University Center); Mathias Uhlén (Royal Institute of Technology (KTH), AlbaNova University Center); </td>
 </tr>
 <tr> 
  <td><b>Short Abstract</b>: KRAB C2H2 zinc finger (ZNF) proteins constitute the largest mammalian class of transcriptional regulators with repression potential, see database SysZNF. High affinity antibody tools are essential prerequisites to establish tissue expression profiles. Peptide microarrays were used to pinpoint specificities and cross-reactivities of monovalent affinity-purified antibodies against 54 KRAB ZNF proteins.</td>
 </tr>
 <tr> 
  <td><b>Long Abstract</b>: <a href="http://www.iscb.org/uploaded/css/38/10437.pdf">Click Here</a></td>
 </tr>
 <tr> 
  <td><b></b></td>
 </tr>
</table><br>
<br><br>


<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="12" valign="top" align="left"><b><span class=smhead1>Accepted Posters</span></b></td>
 </tr>
</table>
<div class="style1">
<a href="#prep">Preparing your Poster - Information and Poster Size</a><br />
<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a>
<a href="#schedule">Poster Schedule</a><br />
<a href="#cat">Poster Categories</a><br />
<a href="#search">Search for a Poster</a><br />

<br />
</div>



<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="poster_list950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="poster_list3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="poster_listf77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="poster_list2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="poster_list1685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="poster_list4158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="poster_list4238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="poster_listbb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="poster_list711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="poster_list39b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="poster_list7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="poster_listfdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="poster_listb2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="poster_lista34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="poster_list53f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="poster_listafd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="poster_listedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="poster_list27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="poster_list5c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="poster_liste35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="poster_listd227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="poster_lista946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="poster_listdc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="poster_list8ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="poster_list718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="poster_list8001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="http://www.iscb.org/ismbeccb2009/poster_list.php" method="POST">
<input type="hidden" name="search" value="1">
<a name="search"></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr> 
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300;" name="slot"></td>
 </tr>
 <tr> 
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="last_name"></td>
 </tr>
 <tr> 
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="co_auth"></td>
 </tr>
 <tr> 
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300;" name="pres_title"></td>
 </tr>
 <tr> 
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>
 
</table>
<br><br>
</form>

<br />
<br />
<br />
<br />
  <a href="#top" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




