<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/proceedingstrackbytime.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent">




<br><br><div class=smhead1><strong>Proceedings Track Presentation Schedule</strong></div> <div class=style1><p><strong>Proceedings Track:</strong> <strong><a name="PT01" >PT01</a></strong><br><strong><em><a href="papers4365.html?id=PT01" >PICKY: A Novel SVD-Based NMR Spectra Peak Picking Method</a></em></strong><br> Monday, June 29 - 10:45 a.m. - 11:10 a.m.<br>Room: Victoria Hall<br />Presenting author: Babak Alipanahi Ramandi, University of Waterloo<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT01: Babak Alipanahi Ramandi - PICKY: A Novel SVD-Based NMR Spectra Peak Picking Method </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/08317231-220b-46d6-a8f2-5284c079d91e" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/08317231-220b-46d6-a8f2-5284c079d91e" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/08317231-220b-46d6-a8f2-5284c079d91e" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT02" >PT02</a></strong><br><strong><em><a href="papersd514.html?id=PT02" >From Disease Ontology (DO) to Disease-Ontology Lite (DOLite): Statistical Methods to Adapt a General-Purpose Ontology for the Test of Gene-Disease Associations</a></em></strong><br> Monday, June 29 - 10:45 a.m. - 11:10 a.m.<br>Room: K2<br />Presenting author: Warren A. Kibbe, Northwestern University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT02: Warren Kibbe - From Disease Ontology (DO) to Disease-Ontology Lite (DOLite): Statistical Methods to Adapt a General-Purpose Ontology for the Test of Gene-Disease Associations </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/dc472446-2341-4fb1-9960-1f49fe574f59" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/dc472446-2341-4fb1-9960-1f49fe574f59" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/dc472446-2341-4fb1-9960-1f49fe574f59" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/yannabraham" target="_top" style="text-decoration:none">Yann Abraham</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">paper: <a style="text-decoration:none" href="http://bioinformatics.oxfordjournals.org/cgi/content/full/25/12/i63">http://bioinformatics.oxfordjournals.org/cgi/content/full/25/12/i63</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want to simplify the DO to make results more easy to interprete - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">DOLite is a CV whereas the DO is an ontology. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">using two parallel clustering steps and expert curation - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">DO: ~12,000 terms, DOlite: ~600 terms - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">FunDO: <a style="text-decoration:none" href="http://django.nubic.northwestern.edu/fundo/">http://django.nubic.northwestern.edu/fundo/</a> - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">"FunDO takes a list of genes and finds relevant diseases based on statistical analysis of the Disease Ontology annotation database. " - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The same query using a gene list with DO and DOLite gets better clustering with DOLite (where better == more distinct clusters, greater number of clusters in the example we were shown - 2 clusters rather than 1). - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">future plan: reduce GO (Gene Ontology) in a similar way - <a style="text-decoration:none" href="http://friendfeed.com/biocs" target="_top">Michael Kuhn</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Blog post: <a style="text-decoration:none" href="http://themindwobbles.wordpress.com/2009/06/29/pt02from-disease-ontology-do-to-disease-ontology-lite-dolite/">http://themindwobbles.wordpress.com/2009/06/29/pt02from-disease-ontology-do-to-disease-ontology-lite-dolite/</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT03" >PT03</a></strong><br><strong><em><a href="papers4912.html?id=PT03" >Prediction of Sub-Cavity Binding Preferences Using an Adaptive Physicochemical Structure Representation</a></em></strong><br> Monday, June 29 - 11:15 a.m. - 11:40 a.m.<br>Room: Victoria Hall<br />Presenting author: Izhar Wallach, University of Toronto<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT03: Izhar Wallach - Prediction of Sub-Cavity Binding Preferences Using an Adaptive Physicochemical Structure Representation </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/54abcf5c-f2ab-4d26-bf00-89f8ef9779c0" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/54abcf5c-f2ab-4d26-bf00-89f8ef9779c0" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/54abcf5c-f2ab-4d26-bf00-89f8ef9779c0" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Seemingly unrelated proteins may share similar subcavities with similar chemical features - <a style="text-decoration:none" href="http://friendfeed.com/anniina" target="_top">Anne Tuukkanen</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT04" >PT04</a></strong><br><strong><em><a href="papers97d1.html?id=PT04" >Alignment of the UMLS Semantic Network with BioTop Methodology and Assessment</a></em></strong><br> Monday, June 29 - 11:15 a.m. - 11:40 a.m.<br>Room: K2<br />Presenting author: Stefan Schulz, Freiburg University Hospital<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT04: Stefan Schulz - Alignment of the UMLS Semantic Network with BioTop Methodology and Assessment </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/cb2b4633-dfc3-4349-a4d1-d4766a87b8db" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/cb2b4633-dfc3-4349-a4d1-d4766a87b8db" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/cb2b4633-dfc3-4349-a4d1-d4766a87b8db" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/mn3" target="_top" style="text-decoration:none">Mitsuteru Nakao</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ontology alignment - link two ontologies together semantically - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Why decide to create BioTop and not use BFO or DOLCE lite directly, I wonder? It's not that I would necessarily suggest that these be used, I am just curious... - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The methodology is first to provide DL semantics to the UMLS SN, and second build the bridge between BioTop and UML SN. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Interestingly, semantic relations were reified as classes, NOT represented as OWL object properties. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">subsumption hierarchies are assumed to be is_a hierarchies, but is that a safe assumption in UMLS SN? I don't know, as I am not familiar with UMLS SN. For instance, in older versions of GO this would have been a problem (some things marked as subsumption were not in fact is_a, though I am pretty sure GO has fixed all of this now). - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">There were inconsistent categorizations of UMLS SN objects which exposed hidden ambiguities (e.g. that Hospital was both a building and an organisation). - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Blog post: <a style="text-decoration:none" href="http://themindwobbles.wordpress.com/2009/06/29/alignment-of-the-umls-semantic-network-with-biotop/">http://themindwobbles.wordpress.com/2009/06/29/alignment-of-the-umls-semantic-network-with-biotop/</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT05" >PT05</a></strong><br><strong><em><a href="papers7de1.html?id=PT05" >Pokefind: A Novel Topological Filter for use With Protein Structure Prediction</a></em></strong><br> Monday, June 29 - 11:45 a.m. - 12:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Firas Khatib, University of Washington<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT05: Firas Khatib - Pokefind: A Novel Topological Filter for use With Protein Structure Prediction </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/2a43a42d-e7e0-4055-b4ce-0cbaa5efe36b" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/2a43a42d-e7e0-4055-b4ce-0cbaa5efe36b" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/2a43a42d-e7e0-4055-b4ce-0cbaa5efe36b" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT06" >PT06</a></strong><br><strong><em><a href="papers6a09.html?id=PT06" >Ontology Quality Assurance Through Analysis of Term Transformations</a></em></strong><br> Monday, June 29 - 11:45 a.m. - 12:10 p.m.<br>Room: K2<br />Presenting author: Karin Verspoor, University of Colorado Denver<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT06: Karin Verspoor - Ontology Quality Assurance Through Analysis of Term Transformations </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/209b7ede-826a-4e06-8b82-e424590425d7" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/209b7ede-826a-4e06-8b82-e424590425d7" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/209b7ede-826a-4e06-8b82-e424590425d7" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/dullhunk" target="_top" style="text-decoration:none">Duncan Hull</a>, <a href="http://friendfeed.com/mn3" target="_top" style="text-decoration:none">Mitsuteru Nakao</a> and <a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">univocality: consistency of expression of concepts within an ontology. This facilitates human usability and computational tools can utilize this regularity. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Try to identify cases where there were violations of univocality: two semantically similar terms with different structure in their term labels. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Steps: abstraction, stopword removal and alphabetic reordering - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">In the end, found 237 clusters that may contain a univocality violation. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Discovered 67 true positive violations (35% ) of univocality. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Blog post: <a style="text-decoration:none" href="http://themindwobbles.wordpress.com/2009/06/29/pto6-ontology-quality-assurance/">http://themindwobbles.wordpress.com/2009/06/29/pto6-ontology-quality-assurance/</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT07" >PT07</a></strong><br><strong><em><a href="papers188f.html?id=PT07" >REPETITA: Detection and Discrimination of the Periodicity of Protein Solenoid Repeats by Discrete Fourier Transform</a></em></strong><br> Monday, June 29 - 12:15 p.m. - 12:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Silvio Tosatto, University of Padova<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT07: Silvio Tosatto - REPETITA: Detection and Discrimination of the Periodicity of Protein Solenoid Repeats by Discrete Fourier Transform </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ce0ddb39-a731-45db-a35f-7c30f7f51739" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ce0ddb39-a731-45db-a35f-7c30f7f51739" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ce0ddb39-a731-45db-a35f-7c30f7f51739" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT08" >PT08</a></strong><br><strong><em><a href="papersd349.html?id=PT08" >Modeling Stochasticity and Robustness in Gene Regulatory Networks</a></em></strong><br> Monday, June 29 - 12:15 p.m. - 12:40 p.m.<br>Room: K2<br />Presenting author: Abhishek Garg, Ecole Polytechnique Federale de Lausanne<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT08: Abhishek Garg - Modeling Stochasticity and Robustness in Gene Regulatory Networks </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/9679a252-37f0-49c8-831e-ab17abb486ee" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9679a252-37f0-49c8-831e-ab17abb486ee" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9679a252-37f0-49c8-831e-ab17abb486ee" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/cassj" target="_top" style="text-decoration:none">Cass Johnston</a> and <a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Looking at stochasticity in nodes and in functions within gene regulartory (GR) networks. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">How does the system behave when it is subjected to stochastic behaviour? - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">uses boolean modeling for this work - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Boolean model: each protein can either be present or absent. The interaction network can be encoded with a set of boolean functions exhibiting steady states. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Previous work: some nodes behave stochastically. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">With SIN (stochasticity in nodes), they had an example of stochastic cellular differentiation. You understand what the probabilities are in SIN by looking at a population of cells. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Example: stochasticity introduces cellular differentiation into a popolation. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The SIN method overrepresents noise in the system. - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">This work employs a different approach: stochasticity in functions. The boolean function itself becomes noisy. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Advantage of this approach: the noise can be selected depending on the biological function it applies to. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/9679a252-37f0-49c8-831e-ab17abb486ee">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT09" >PT09</a></strong><br><strong><em><a href="papersfb73.html?id=PT09" >A Framework to Refine Particle Clusters Produced by EMAN</a></em></strong><br> Monday, June 29 - 2:15 p.m. - 2:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Liya Fan, Institute of Computing Technology<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT09: Liya Fan - A Framework to Refine Particle Clusters Produced by EMAN </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/4068314b-7b25-437f-a301-cb0e60f8062f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/4068314b-7b25-437f-a301-cb0e60f8062f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/4068314b-7b25-437f-a301-cb0e60f8062f" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT10" >PT10</a></strong><br><strong><em><a href="papers11b3.html?id=PT10" >Clustered Alignments of Gene-Expression Time Series Data</a></em></strong><br> Monday, June 29 - 2:15 p.m. - 2:40 p.m.<br>Room: K2<br />Presenting author: Adam Smith, University of Wisconsin, Madison<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT10: Adam Smith - Clustered Alignments of Gene-Expression Time Series Data </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/812aadea-8499-4d14-b263-b5335a2d791b" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/812aadea-8499-4d14-b263-b5335a2d791b" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/812aadea-8499-4d14-b263-b5335a2d791b" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Usually restricted to discrete measurements of a continues attribute, requires reconstruction or interpolation. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Time warping/alignment to maximize the similarity of the compared point helps. Warpspace diagrams are an alternative way of showing alignment between two time series (think two sequence alignment matrices) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">SCOW: efficient method for sparse time series - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Known algorithm (1978) for dynamic time warping (DTW), minimize sum of euclidean distances. Parametric time warping (2005, Eilers, PTW) fits alignment function from given family. More limited in expressiveness than DTW - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Segment-based warping splits warps into individually scored segments, sits in a happy intermediate between DTW, PTW, but very slow (n^5 complexity). Correlation-optimized warping (COW, Nielsen 1998) looks for good 'knots' (one dimensional). SCOW searches in each dimension independently until convergence. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Evaluation with EDGE toxicology database, SCOW does better than other algorithms (little less so for distorted test data) in identifying the correct matching time series. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Cluster gene time series to get a regularization effect; algorithm based on k-means. Each cluster defined by an alignment, initial clusters calculates by a greedy algorithm, assign genes to cluster, re-calculate alignment for each cluster, iterate until convergence. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Does clustering help? Test with Mop3 knockout (circadian cycle gene); five clusters with exemplar genes identifies genes with a strong phase shift. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Talk notes at <a style="text-decoration:none" href="http://www.fiamh.info/articles/16/ismbeccb-2009-day-1">http://www.fiamh.info/articles/16/ismbeccb-2009-day-1</a> - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT11" >PT11</a></strong><br><strong><em><a href="papersc8ee.html?id=PT11" >Domain-oriented Edge-based Alignment of Protein Interaction Networks</a></em></strong><br> Monday, June 29 - 2:45 p.m. - 3:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Xin Guo, Duke University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT11: Xin Guo - Domain-oriented Edge-based Alignment of Protein Interaction Networks </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/0fa85f40-cf85-4c28-a841-1dcd7449fdce" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0fa85f40-cf85-4c28-a841-1dcd7449fdce" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/0fa85f40-cf85-4c28-a841-1dcd7449fdce" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Introducing... the DOMain-oriented Alignment of Interaction Networks (DOMAIN). - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">DOMAIN is the first algorithm to introduce domains to PPI network alignment problem, and the first attempt to align PPIs directly. (They think) - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Blog post: <a style="text-decoration:none" href="http://themindwobbles.wordpress.com/?p=735">http://themindwobbles.wordpress.com/?p=735</a> - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT12" >PT12</a></strong><br><strong><em><a href="papers7da3.html?id=PT12" >KELLER: Estimating Time-Evolving Interactions Between Genes</a></em></strong><br> Monday, June 29 - 2:45 p.m. - 3:10 p.m.<br>Room: K2<br />Presenting author: Le Song, Carnegie Mellon University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT12: Le Song - KELLER: Estimating Time-Evolving Interactions Between Genes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/9c8e41aa-a38f-4438-82be-1164ed326619" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9c8e41aa-a38f-4438-82be-1164ed326619" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/9c8e41aa-a38f-4438-82be-1164ed326619" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT13" >PT13</a></strong><br><strong><em><a href="paperse2bc.html?id=PT13" >Network-based Prediction of Metabolic Enzymes Subcellular Localization</a></em></strong><br> Monday, June 29 - 3:15 p.m.- 3:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Shira Mintz-Oron, Weizmann Institute of Science<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT13: Shira Mintz-Oron - Network-based Prediction of Metabolic Enzymes Subcellular Localization </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/1149f0f1-762d-4211-a8db-7533ed5bbbe7" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/1149f0f1-762d-4211-a8db-7533ed5bbbe7" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/1149f0f1-762d-4211-a8db-7533ed5bbbe7" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT14" >PT14</a></strong><br><strong><em><a href="papers2a12.html?id=PT14" >Predicting Functionality of Protein-DNA Interactions by Integrating Diverse Evidence</a></em></strong><br> Monday, June 29 - 3:15 p.m.- 3:40 p.m.<br>Room: K2<br />Presenting author: Duygu Ucar, Ohio State University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT14: Duygu Ucar - Predicting Functionality of Protein-DNA Interactions by Integrating Diverse Evidence </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/743f82e9-3b22-422a-b24a-d31e8573207d" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/743f82e9-3b22-422a-b24a-d31e8573207d" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/743f82e9-3b22-422a-b24a-d31e8573207d" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/cassj" target="_top" style="text-decoration:none">Cass Johnston</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">background about TFs. ChIPchip, ChIPseq can define binding (but noisy data). Even when we know binding occurs, knowing when binding is functional is less well characterized. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Detecting interaction events between TFs and targets is important, can be observed via chip-chip/seq - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Semantics of interactions is important, but difficult to characterize (context dependency of TF binding event) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Binding and gene regulation are context dependent - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Estimating TF binding and gene expression response based on three data sets (chip-chip, PSSM and nucleosome occupancy). Binding considered functional if it changes gene expression (determined from microarray data) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">combined into a probabilistic bayesian model - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Integrated model outperforms individual three data sets in a five-fold cross validation - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Used two expression data sets to distinguish putatively functional and non-functional binding events - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Data is from yeast. Many TFs. Conditions are Normal Growth and various Stress. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Functional binding rate for a TF was context dependent. They could rank the TFs in order of their impact in different conditions - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/743f82e9-3b22-422a-b24a-d31e8573207d">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT15" >PT15</a></strong><br><strong><em><a href="papers3b44.html?id=PT15" >IsoRankN: Spectral Methods For Global Alignment of Multiple Protein Networks</a></em></strong><br> Monday, June 29 - 3:45 p.m. - 4:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Michael Baym, Massachusetts Institute of Technology<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT15: Michael Baym - IsoRankN: Spectral Methods For Global Alignment of Multiple Protein Networks </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/42c20caf-01bb-446b-a444-a9176e2e472f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/42c20caf-01bb-446b-a444-a9176e2e472f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/42c20caf-01bb-446b-a444-a9176e2e472f" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT16" >PT16</a></strong><br><strong><em><a href="papers0dd4.html?id=PT16" >Grouped Graphical Granger Modeling for Gene Expression Regulatory Networks Discovery</a></em></strong><br> Monday, June 29 - 3:45 p.m. - 4:10 p.m.<br>Room: K2<br />Presenting author: Saharon Rosset, Tel Aviv University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT16: Saharon Rosset - Grouped Graphical Granger Modeling for Gene Expression Regulatory Networks Discovery </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/8bf1aadb-db16-49cd-ad59-d6ad87d8c808" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8bf1aadb-db16-49cd-ad59-d6ad87d8c808" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8bf1aadb-db16-49cd-ad59-d6ad87d8c808" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Friendfeed blocked access temporarily. Notes at <a style="text-decoration:none" href="http://www.fiamh.info/articles/16/ismbeccb-2009-day-1">http://www.fiamh.info/articles/16/ismbeccb-2009-day-1</a> - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT17" >PT17</a></strong><br><strong><em><a href="papers7da8.html?id=PT17" >A Unified Statistical Model to Support Local Sequence Order Independent  Similarity Searching for Ligand Binding Sites and its Application to Genome-Based Drug Discovery</a></em></strong><br> Tuesday, June 30 - 10:45 a.m. - 11:10 a.m.<br>Room: Victoria Hall<br />Presenting author: Lei Xie, University of California, San Diego<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT17: Lei Xie - A Unified Statistical Model to Support Local Sequence Order Independent  Similarity Searching for Ligand Binding Sites and its Application to Genome-Based Drug Discovery </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/094993d0-a207-4044-bbfc-10380b9fbac4" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/094993d0-a207-4044-bbfc-10380b9fbac4" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/094993d0-a207-4044-bbfc-10380b9fbac4" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT18" >PT18</a></strong><br><strong><em><a href="papersca26.html?id=PT18" >Multi-Locus Match Probability in a Finite Population: A Fundamental Difference Between the Moran and Wright-Fisher Models</a></em></strong><br> Tuesday, June 30 - 10:45 a.m. - 11:10 a.m.<br>Room: K2<br />Presenting author: Anand Bhaskar, University of California, Berkeley<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT18: Anand Bhaskar - Multi-Locus Match Probability in a Finite Population: A Fundamental Difference Between the Moran and Wright-Fisher Models </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/87c5def3-06c8-491b-9d97-84bed0f4f6db" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/87c5def3-06c8-491b-9d97-84bed0f4f6db" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/87c5def3-06c8-491b-9d97-84bed0f4f6db" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT19" >PT19</a></strong><br><strong><em><a href="paperse965.html?id=PT19" >Global Alignment of Protein-Protein Interaction Networks by Graph Matching Methods</a></em></strong><br> Tuesday, June 30 - 11:15 a.m. - 11:40 a.m.<br>Room: Victoria Hall<br />Presenting author: Mikhail Zaslavskiy, Mines ParisTech<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT19: Mikhail Zaslavskiy - Global Alignment of Protein-Protein Interaction Networks by Graph Matching Methods </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/8cb04f7c-c32c-4020-9476-1effcfd2b90c" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8cb04f7c-c32c-4020-9476-1effcfd2b90c" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8cb04f7c-c32c-4020-9476-1effcfd2b90c" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/stajich" target="_top" style="text-decoration:none">Jason Stajich</a>, <a href="http://friendfeed.com/yannabraham" target="_top" style="text-decoration:none">Yann Abraham</a> and <a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Motivation: identification of protein functional orthologs - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Standard approach like reciprocal best BLAST hits have problems when several top hits have similar scores -- which pair to chose? Additional information helps to resolve ambiguity. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">PPI networks (the usual hairball) can be used to resolve this. If ortholog assignments conserve PPI interactions are ranked higher - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Maximize number of conserved interaction and the sum of BLAST pair scores - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Used message passing clustering approach for the InParanoid/protein cluster approach; each node is a protein cluster - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Balanced alignment uses graph matching approximation (gradient ascent and path algorithm) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">PPI networks and InParanoid clusters from the Ideker paper, 2244 clusters (1552 with only 2 proteins / orthologs, 692 ambiguous clusters that need to be resolved). No cycles in this graph means constrained alignment approach can be used, results in 238 conserved interactions - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Constrained algorithm with message passing is an exact solution, graph matching algorithms deliver good performance for balanced alignment problems - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Additional notes: <a style="text-decoration:none" href="http://www.fiamh.info/articles/17/ismbeccb-2009-day-2">http://www.fiamh.info/articles/17/ismbeccb-2009-day-2</a> - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT20" >PT20</a></strong><br><strong><em><a href="papersccfd.html?id=PT20" >Speeding Up HMM Algorithms for Genetic Linkage Analysis via Chain Reductions of the State Space</a></em></strong><br> Tuesday, June 30 - 11:15 a.m. - 11:40 a.m.<br>Room: K2<br />Presenting author: Ydo Wexler, Microsoft Research<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT20: Ydo Wexler - Speeding Up HMM Algorithms for Genetic Linkage Analysis via Chain Reductions of the State Space </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/7c5afc6a-1479-4682-b8cf-b1838a07d032" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/7c5afc6a-1479-4682-b8cf-b1838a07d032" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/7c5afc6a-1479-4682-b8cf-b1838a07d032" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT21" >PT21</a></strong><br><strong><em><a href="papers9787.html?id=PT21" >Probabilistic retrieval and Visualization of Biologically Relevant Microarray Experiments</a></em></strong><br> Tuesday, June 30 - 11:45 a.m. - 12:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Jos Caldas, Helsinki University of Technology<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT21: Jose Caldas - Probabilistic retrieval and Visualization of Biologically Relevant Microarray Experiments </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/f3a1848d-3b19-4282-873f-4104e764020f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f3a1848d-3b19-4282-873f-4104e764020f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/f3a1848d-3b19-4282-873f-4104e764020f" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Trying to find a method to relate results in large array databases based on expression information rather than annotation. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Standard approaches like spearman correlation coefficients, but it would be interested to use sets of experiments as a query rather than a single array. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Query with a binary phenotype comparison and try to get back other, similar comparisons. Requires encoding of the phenotype comparison such as a vector of t-tests (0/1 vector for differentially expressed genes) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Using differential GSEA (shorter vector, more robust) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">GSEA is gene set enrichment analysis and compares gene sets with -1,0,+1 vectors - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Uses standard GSEA, number of genes in leading edge as vector value, ignoring the directionality - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Latent Dirichlet Allocation for the retrieval algorithm - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Latent Drichilet Allocation (LDA) from text analysis works intuitively: uses documents to build up topics -- in bag-of-words text data. - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Bag-of-words from gene sets, represented as combinations of sets, relevant distance measures can be applied - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">750+ binary phenotype comparisons from 288 experiments, focus on 105 comparisons for this analysis. Gene sets assigned to topics are coherent across a wide range of biological processes - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/f3a1848d-3b19-4282-873f-4104e764020f">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT22" >PT22</a></strong><br><strong><em><a href="papersa63a.html?id=PT22" >A Multivariate Regression Approach to Association Analysis of Quantitative Trait Network</a></em></strong><br> Tuesday, June 30 - 11:45 a.m. - 12:10 p.m.<br>Room: K2<br />Presenting author: Seyoung Kim, Carnegie Mellon University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT22: Seyoung Kim - A Multivariate Regression Approach to Association Analysis of Quantitative Trait Network </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/71e070b9-2395-478a-9ac1-cb37731c1fc2" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/71e070b9-2395-478a-9ac1-cb37731c1fc2" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/71e070b9-2395-478a-9ac1-cb37731c1fc2" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT23" >PT23</a></strong><br><strong><em><a href="papers5024.html?id=PT23" >Proteome Coverage Prediction with Infinite Markov Models</a></em></strong><br> Tuesday, June 30 - 12:15 p.m. - 12:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Manfred Claassen, ETH Zrich<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT23: Manfred Claassen - Proteome Coverage Prediction with Infinite Markov Models </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/bcd63fb7-2a7f-4e9f-9e9c-350c3694020d" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/bcd63fb7-2a7f-4e9f-9e9c-350c3694020d" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/bcd63fb7-2a7f-4e9f-9e9c-350c3694020d" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Predticiton of proteome coverage in shotgun proteomics for maximum coverage, to find a stop criterion, and to direct study design - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">estimating the limit of theoretical coverage first requires a quantification of the distribution of observable peptides. The computer science analogy used is the chinese restaurant process - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Chinese Restaurant Processes, i.e. the rich get richer, "dish" distribution: many proteins are observed over and over again - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">customers map to spectra, dishes map to peptides - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">a hierarchy of chinese restaurants (== dirichlet processes) ensuring support for likely observed peptides in the root distribution of observable peptides - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">adapting the infinite markov model to the case where identifications are false positives (account for target decoy strategy based FPR estimation) - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">(... simply add an extra dimension that samples over the identification database - ie model the FPR random chance distribution) - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">maximal coverage!=saturation coverage - more experiments means the quality of discoveries are worse, ie the less certain you can be about your real matches - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Answers that the proteome coverage prediction can provide: How many peptides can be identified?
More experiments deteriorate quality of discoveries: quality constraint restricts maximum coverage - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">I think this is really important - but I am not expert enough to understand how this diriSim model can be applied to the 'limits' of proteomics studies, where one is looking for very specific types of proteins (e.g. PTM characterisation, etc). - <a style="text-decoration:none" href="http://friendfeed.com/foreveremain" target="_top">Jim Procter</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT24" >PT24</a></strong><br><strong><em><a href="papersda1a.html?id=PT24" >Inference of Locus-Specific Ancestry in Closely Related Populations</a></em></strong><br> Tuesday, June 30 - 12:15 p.m. - 12:40 p.m.<br>Room: K2<br />Presenting author: Sriram Sankararaman, University of California, Berkeley<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT24: Sriram Sankararaman - Inference of Locus-Specific Ancestry in Closely Related Populations </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/3b5d79b1-3a4b-4b5e-8c19-603d3db9fdac" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/3b5d79b1-3a4b-4b5e-8c19-603d3db9fdac" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/3b5d79b1-3a4b-4b5e-8c19-603d3db9fdac" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT25" >PT25</a></strong><br><strong><em><a href="papers09d6.html?id=PT25" >Fewer permutations, More Accurate P-values</a></em></strong><br> Tuesday, June 30 - 2:15 p.m. - 2:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Theo Knijnenburg, Institute for Systems Biology<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT25: Theo Knijnenburg - Fewer permutations, More Accurate P-values </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/da05b25d-9cd1-466c-86a9-541e6c7e1cf3" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/da05b25d-9cd1-466c-86a9-541e6c7e1cf3" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/da05b25d-9cd1-466c-86a9-541e6c7e1cf3" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Website: <a style="text-decoration:none" href="http://informatics.systemsbiology.net/prototypes/tknijnenburg/permutations">http://informatics.systemsbiology.net/prototypes/tknijnenburg/permutations</a> (also via <a style="text-decoration:none" href="http://tinyurl.com/permutation-test)">http://tinyurl.com/permutation-test)</a> - <a style="text-decoration:none" href="http://friendfeed.com/theoknijnenburg" target="_top">Theo Knijnenburg</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT26" >PT26</a></strong><br><strong><em><a href="papersaedd.html?id=PT26" >A Geometric Approach for Classification and Comparison of Structural Variants</a></em></strong><br> Tuesday, June 30 - 2:15 p.m. - 2:40 p.m.<br>Room: K2<br />Presenting author: Suzanne Sindi, Brown University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT26: Suzanne Sindi - A Geometric Approach for Classification and Comparison of Structural Variants </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/abee26b2-f74b-406a-8e52-f425b67ba676" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/abee26b2-f74b-406a-8e52-f425b67ba676" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/abee26b2-f74b-406a-8e52-f425b67ba676" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT27" >PT27</a></strong><br><strong><em><a href="papers958b.html?id=PT27" >A General Computational Method for Robustness Analysis with Applications to Synthetic Gene Networks</a></em></strong><br> Tuesday, June 30 - 2:45 p.m. - 3:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Aurlien Rizk, INRIA Paris-Rocquencourt<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT27: Aurelien Rizk - A General Computational Method for Robustness Analysis with Applications to Synthetic Gene Networks </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/8855f7ca-e0e8-4e5b-83be-9818c580dc06" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8855f7ca-e0e8-4e5b-83be-9818c580dc06" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8855f7ca-e0e8-4e5b-83be-9818c580dc06" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT28" >PT28</a></strong><br><strong><em><a href="papers825f.html?id=PT28" >Joint Estimation of Gene Conversion Rates and Mean Conversion Tract Lengths from Population SNP Data</a></em></strong><br> Tuesday, June 30 - 2:45 p.m. - 3:10 p.m.<br>Room: K2<br />Presenting author: Junming Yin, University of California, Berkeley<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT28: Junming Yin - Joint Estimation of Gene Conversion Rates and Mean Conversion Tract Lengths from Population SNP Data </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/eb976da5-cb92-4753-be43-ce11c8585396" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/eb976da5-cb92-4753-be43-ce11c8585396" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/eb976da5-cb92-4753-be43-ce11c8585396" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT29" >PT29</a></strong><br><strong><em><a href="papersd4db.html?id=PT29" >E-zyme: Predicting Potential EC Numbers from the ChemicalTransformation Pattern of Substrate-product Pairs</a></em></strong><br> Tuesday, June 30 - 3:15 p.m. - 3:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Yoshihiro Yamanishi, Ecole des Mines de Paris<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT29: Yoshihiro Yamanishi - E-zyme: Predicting Potential EC Numbers from the Chemical
Transformation Pattern of Substrate-product Pairs </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/299936b8-5d7c-4f7a-811f-79f1b63a04eb" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/299936b8-5d7c-4f7a-811f-79f1b63a04eb" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/299936b8-5d7c-4f7a-811f-79f1b63a04eb" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT30" >PT30</a></strong><br><strong><em><a href="papersf9e5.html?id=PT30" >Computing Galled Networks from Real Data</a></em></strong><br> Tuesday, June 30 - 3:15 p.m. - 3:40 p.m.<br>Room: K2<br />Presenting author: Daniel Huson, University of Tuebingen<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT30: Daniel Huson - Computing Galled Networks from Real Data </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/7b8d55c3-b46f-4bc2-b99b-1d380de8751e" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/7b8d55c3-b46f-4bc2-b99b-1d380de8751e" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/7b8d55c3-b46f-4bc2-b99b-1d380de8751e" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Two different types, networks that provide an explicit picture with hybridization etc, or an abstract network. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">An early approach using a mathematical generalization were  split networks, which provide an abstraction but no explanation. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">See dendroscope.org - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Go to an explicit networks where you have hybridization and speciation events, e.g. in plant evolution, horizontal gene transfer in bacteria. Both appear similar in this framework. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">When modelling reticulate evolution a node can have a variable input from different parents, e.g. 50/50 for plants, 95/5 for horizontal gene transfer settings. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT32" >PT32</a></strong><br><strong><em><a href="papersd1e1.html?id=PT32" >Genotype-Phenotype Associations: Substitution Models to Detect Evolutionary Associations Between Phenotypic Variables and Genotypic Evolutionary Rate</a></em></strong><br> Tuesday, June 30 - 3:45 p.m. - 4:10 p.m.<br>Room: K2<br />Presenting author: Timothy OConnor, University of Cambridge<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT32: Timothy O'Connor - Genotype-Phenotype Associations: Substitution Models to Detect Evolutionary Associations Between Phenotypic Variables and Genotypic Evolutionary Rate </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/74f26784-0fc4-4e8b-8e28-3ec17166a158" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/74f26784-0fc4-4e8b-8e28-3ec17166a158" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/74f26784-0fc4-4e8b-8e28-3ec17166a158" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Can we detect a change of rate in evolution. Model is sperm competition in apes. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Phylogeny is important to look at the independent signal, also use the past nodes for inference. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Problems arise from the use of extant taxa and neglecting internal states. Trying to map the rate of evolution using Markov Models, including the error of the internal state reconstruction. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Independent and dependent Q-matrices are introduced. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Several szenarios for primate sperm competition, split into low and high cases, looked at 13 candidate genes and 2 controls, more than in the submitted paper. Model slightly modified by setting W1> W0. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The known gene SEMG2 stands out after multiple testing corrections, other genes not that dramatically. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Current work on the electron transport chain to look into max lifespans and body size, referring to the metabolic and the longevity hypotheses. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Metabolic hypothesis slightly supported, ongoing work. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">It's a first model that incorporates rate changes in genotype-phenotype associations. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Q: For this method, what is the statistical power (sensitivity)? A: Typically, its fairly sensitive. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/74f26784-0fc4-4e8b-8e28-3ec17166a158">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT33" >PT33</a></strong><br><strong><em><a href="papers7e97.html?id=PT33" >Towards a Gold Standard for Promoter Prediction Evaluation</a></em></strong><br> Wednesday, July 1 - 10:45 a.m. - 11:10 a.m.<br>Room: Victoria Hall<br />Presenting author: Thomas Abeel, Ghent University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT33: Thomas Abeel - Towards a Gold Standard for Promoter Prediction Evaluation </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/e5bf2dd7-9c48-45aa-a9d3-b42ed3cccb9c" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e5bf2dd7-9c48-45aa-a9d3-b42ed3cccb9c" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e5bf2dd7-9c48-45aa-a9d3-b42ed3cccb9c" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">definition of a promoter. Multiple transcription start sites in a given transcriptional unit. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">1997: first paper evaluating promoter prediction. 9 prediction programs. 15-30% success - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">2004 - whole genome. Precision 5-87% and recall 27-29%. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">2006 - CAGE tags provide known results to test predictions - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Need a gold standard as there is no consensus method at the moment, and different tools all claim to be the best. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">They use data from CAGE (clustered into transcription start regions) and RefSeq (5' end as true promoter site) to test predictors - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">17 programs. All free for academic use and capable of being run on the whole human genome. Ran all programs over various thresholds - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Plot precision v recall and calc AUC over threshold range for each of the programs. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Four programs scoring over 20% in prediction: ARTS, EP3, Eponine, ProSOM - <a style="text-decoration:none" href="http://friendfeed.com/sebi0" target="_top">sebi</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">considered 4 best scoring programs. Found classes of promoters - some with a single peak of tss, some with a couple, a few with many tss all over the region - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/e5bf2dd7-9c48-45aa-a9d3-b42ed3cccb9c">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT34" >PT34</a></strong><br><strong><em><a href="papersca6e.html?id=PT34" >Constrained Mixture Estimation for Analysis and Robust Classification of Clinical Time Series</a></em></strong><br> Wednesday, July 1 - 10:45 a.m. - 11:10 a.m.<br>Room: K2<br />Presenting author: Ivan Gesteira Costa, Federal University of Pernambuco<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT34: Ivan Gesteira Costa - Constrained Mixture Estimation for Analysis and Robust Classification of Clinical Time Series </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ed13ff3a-d533-48de-99b9-f10c523962b6" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ed13ff3a-d533-48de-99b9-f10c523962b6" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ed13ff3a-d533-48de-99b9-f10c523962b6" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT35" >PT35</a></strong><br><strong><em><a href="papers0502.html?id=PT35" >DISCOVER: A Feature-Based Discriminative Method for Motif Search in Complex Genomes</a></em></strong><br> Wednesday, July 1 - 11:15 a.m. - 11:40 a.m.<br>Room: Victoria Hall<br />Presenting author: Pradipta Ray, Carnegie Mellon University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT35: Pradipta Ray - DISCOVER: A Feature-Based Discriminative Method for Motif Search in Complex Genomes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/76d333fb-52a9-4037-9c4c-22115c3c1367" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/76d333fb-52a9-4037-9c4c-22115c3c1367" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/76d333fb-52a9-4037-9c4c-22115c3c1367" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The problem: find transcription factor binding sites. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">In higher organisms, binding sites operate in clusters. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Two approaches: supervides / unsupervised. They used a supervised search. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Traditionally used PWMs to model motifs - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">PWM: position weight matrix. Used to score sequence windows. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Problem: high false positive rate. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">An evolution of this idea: hidden markov models. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Two states: motif and background states / distributions. You learn parameters of these models from known data. - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The performance of such models has saturated in recent years - <a style="text-decoration:none" href="http://friendfeed.com/gbrsales" target="_top">Gabriele Sales</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">problems with HMM / generative models: may tune to noise, rather than the signal; Seem to have hit peak performance; Difficult to incorporate other sources of information to improve predictions - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/76d333fb-52a9-4037-9c4c-22115c3c1367">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT36" >PT36</a></strong><br><strong><em><a href="papersa8ba.html?id=PT36" >Graph Theoretical Approach To Study eQTL: A Case Study of Plasmodium Falciparum</a></em></strong><br> Wednesday, July 1 - 11:15 a.m. - 11:40 a.m.<br>Room: K2<br />Presenting author: Yang Huang, National Institutes of Health<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT36: Yang Huang - Graph Theoretical Approach To Study eQTL: A Case Study of Plasmodium Falciparum </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ce2e93d0-f389-462f-9bda-dc18eae83810" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ce2e93d0-f389-462f-9bda-dc18eae83810" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ce2e93d0-f389-462f-9bda-dc18eae83810" target="_top">View in FriendFeed</a></b></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">P.falciparum is the most deadly human malaria pathogen - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Little information about gene regulation so far, eQTL might be able to shed some light on this regulation and drug resistance - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">SNPs might affect gee expression. Consider expression as a quantitative trait like height, weight. Identify the associated locus by statistical methods. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Traditional tests between multiple loci, all expression. Comprehensive and without biast, but does not use the inherent data structure, computationally expensive and a problem of statistical power. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Alternative approach GeD, Graph-based eQTL decomposition. Include strain data in the association graph - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Graph structure: Three types of vertices: gene linked to strain linked to locus - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Find cliques to reduce data complexity - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Each clique has 3 vertices (G/S/L) that are fully connected, in addition each clique is a maximal subgraph that cannot be extended further - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Represent inherent data structures - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Heuristic approach on eQTL cliques to look for (Locus,gene) pairs with certain patterns; refer to graph/diagram in paper - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/ce2e93d0-f389-462f-9bda-dc18eae83810">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT37" >PT37</a></strong><br><strong><em><a href="papers3509.html?id=PT37" >Predictions of RNA Secondary Structure by Combining Homologous Sequence Information</a></em></strong><br> Wednesday, July 1 - 11:45 a.m. - 12:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Michiaki Hamada, Mizuho Information & Research Institute, Inc.<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT37: Michiaki Hamada - Predictions of RNA Secondary Structure by Combining Homologous Sequence Information </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/5c840900-c6cd-4afa-9cda-768df648904f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5c840900-c6cd-4afa-9cda-768df648904f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5c840900-c6cd-4afa-9cda-768df648904f" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/diriano" target="_top" style="text-decoration:none">Diego M. Riao-Pachn</a>, <a href="http://friendfeed.com/mn3" target="_top" style="text-decoration:none">Mitsuteru Nakao</a> and <a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Need for better RNA secondary structure prediction with increasing awareness of functional ncRNAs - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Most algorithms at the moment don't allow pseudoknots - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Minimum Free Energy approaches: Mfold, RNAfold etc. But many structures close to MFE - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Maximizing expected accuracy CONTRAfold etc. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">CentroidFold (their algorithm) is an MEA tool. Performs better than RNAFold, Sfold, Contrafold... (not sure what the test set was) - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Using homology to further improve accuracy of structure prediction (previous approaches: RNAalifold, McCaskill) - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Sankoff sequence/structure alignment of sets of homologous sequences plus MEA. Computationally unfeasible. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Approximate the Sankoff method such that it is practical to run the method even for long RNA sequences - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Compared CentroidAliFold to other state of the art methods. Outperforms conventional secondary structure prediction (ie. MFE-based) and outperforms everything except RAF (comparable) for approaches using homology too. - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Much quicker than RAF - <a style="text-decoration:none" href="http://friendfeed.com/cassj" target="_top">Cass Johnston</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/5c840900-c6cd-4afa-9cda-768df648904f">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT38" >PT38</a></strong><br><strong><em><a href="papers87ab.html?id=PT38" >A Classifier-based Approach to Identify Genetic Similarities Between Diseases</a></em></strong><br> Wednesday, July 1 - 11:45 a.m. - 12:10 p.m.<br>Room: K2<br />Presenting author: Marc A. Schaub, Stanford University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT38: Marc Schaub - A Classifier-based Approach to Identify Genetic Similarities Between Diseases </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/27048317-1517-4ad5-8753-6603c6951364" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/27048317-1517-4ad5-8753-6603c6951364" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/27048317-1517-4ad5-8753-6603c6951364" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT39" >PT39</a></strong><br><strong><em><a href="papers77af.html?id=PT39" >Assessing Phylogenetic Motif Models For Predicting Transcription Factor Binding Sites</a></em></strong><br> Wednesday, July 1 - 12:15 p.m. - 12:40 p.m.<br>Room: Victoria Hall<br />Presenting author: John Hawkins, University of Queensland<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT39: John Hawkins - Assessing Phylogenetic Motif Models For Predicting Transcription Factor Binding Sites </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/8a49a02c-e36a-459b-a8ac-860ec70d74b6" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8a49a02c-e36a-459b-a8ac-860ec70d74b6" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8a49a02c-e36a-459b-a8ac-860ec70d74b6" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT40" >PT40</a></strong><br><strong><em><a href="papers9516.html?id=PT40" >Model Based Clustering of Array CGH Data</a></em></strong><br> Wednesday, July 1 - 12:15 p.m. - 12:40 p.m.<br>Room: K2<br />Presenting author: Sohrab Shah, University of British Columbia<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT40: Sohrab Shah - Model Based Clustering of Array CGH Data </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/5c1293db-5016-4656-9e3d-f2aa2c31cede" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5c1293db-5016-4656-9e3d-f2aa2c31cede" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5c1293db-5016-4656-9e3d-f2aa2c31cede" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT41" >PT41</a></strong><br><strong><em><a href="papersa75d.html?id=PT41" >Modeling Interactions between Adjacent Nucleosomes Improves Genome-wide Predictions of Nucleosome Occupancy</a></em></strong><br> Wednesday, July 1 - 2:15 p.m. - 2:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Shai Lubliner, Weizmann Institute of Science<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT41: Shai Lubliner - Modeling Interactions between Adjacent Nucleosomes Improves Genome-wide Predictions of Nucleosome Occupancy </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/b9cfbf0d-2d3b-4b88-96cb-66c06d3083c4" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b9cfbf0d-2d3b-4b88-96cb-66c06d3083c4" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/b9cfbf0d-2d3b-4b88-96cb-66c06d3083c4" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/jxtx" target="_top" style="text-decoration:none">James Taylor</a>, <a href="http://friendfeed.com/mikechelen" target="_top" style="text-decoration:none">Mike Chelen</a> and <a href="http://friendfeed.com/spitshine" target="_top" style="text-decoration:none">Roland Krause</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Nucleosome position affects transcriptional regulation, models for position are required for understanding of TR. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">75-90% of the DNA is associated with nucleosomes, play an important regulatory role. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Use a recently published model to produce the affinity landscape, showing the suitability of nucleosome occupancy. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Genomic nuceosome affinity landscape based on a thermodynamical model - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The model is probably published here (<a style="text-decoration:none" href="http://www.ncbi.nlm.nih.gov/pubmed/19451592)">http://www.ncbi.nlm.nih.gov/pubmed/19451592)</a> - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Can interactions between nucleosomes introduced into the model? - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Two kinds of interactions: direct or bridged by transcription factors or other proteins. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Additional interactions are important for chromatin organization. DNA bending proteins, TF, histone modifications, etc. - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">First approach focuses on direct interaction, including the distance between the nucleosomes. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Trying to capture interactions between adjacent nucleosomes (cooperative effects) - <a style="text-decoration:none" href="http://friendfeed.com/setar" target="_top">Oliver Hofmann</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/b9cfbf0d-2d3b-4b88-96cb-66c06d3083c4">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT42" >PT42</a></strong><br><strong><em><a href="papers19af.html?id=PT42" >Viruses Selectively Mutate Their Cd8  T Cell Epitopes  A Large Scale Immunomic Analysis</a></em></strong><br> Wednesday, July 1 - 2:15 p.m. - 2:40 p.m.<br>Room: K2<br />Presenting author: Yoram Louzoun, Bar Ilan University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT42: Yoram Louzoun - Viruses Selectively Mutate Their Cd8  T Cell Epitopes - A Large Scale Immunomic Analysis </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/ce78d0c4-e4b7-4280-b1a4-622aa195220f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ce78d0c4-e4b7-4280-b1a4-622aa195220f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/ce78d0c4-e4b7-4280-b1a4-622aa195220f" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT43" >PT43</a></strong><br><strong><em><a href="papers3f6d.html?id=PT43" >Efficient Exact Motif Discovery</a></em></strong><br> Wednesday, July 1 - 2:45 p.m. - 3:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Tobias Marschall, TU Dortmund<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT43: Tobias Marschall - Efficient Exact Motif Discovery </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/e77b8fb0-e924-43c8-97f0-dc5fe7133921" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e77b8fb0-e924-43c8-97f0-dc5fe7133921" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/e77b8fb0-e924-43c8-97f0-dc5fe7133921" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/marcelmartin" target="_top" style="text-decoration:none">Marcel Martin</a> and <a href="http://friendfeed.com/spitshine" target="_top" style="text-decoration:none">Roland Krause</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Unsupervised motif discovery on a string with no previous knowledege in an automated fashion. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">issues: How to measure over-representation and how to find them. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">184 publications in pubmed for "motif discovery algorithm" - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">aim to establish an (almost) exact method based on a rigorous motif statistics - <a style="text-decoration:none" href="http://friendfeed.com/mikhail1" target="_top">Mikhail Spivakov</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">given: query text, IUPAC motifs, random text model (background) - for now, iid - <a style="text-decoration:none" href="http://friendfeed.com/mikhail1" target="_top">Mikhail Spivakov</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Calculating a p-value of a given query text, a IUPAC motif and a random text model. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">want: a p-value for a motif - <a style="text-decoration:none" href="http://friendfeed.com/mikhail1" target="_top">Mikhail Spivakov</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">use a novel device called probabilistic arithmetic automata  - won't go into details - <a style="text-decoration:none" href="http://friendfeed.com/mikhail1" target="_top">Mikhail Spivakov</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Need to compute the distribution of occurrences by chance. Not a straight forward task, recently proposed a new approach by building a probabilistic arithmetic automata. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">an exact calculation - <a style="text-decoration:none" href="http://friendfeed.com/mikhail1" target="_top">Mikhail Spivakov</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/e77b8fb0-e924-43c8-97f0-dc5fe7133921">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT44" >PT44</a></strong><br><strong><em><a href="papers11f8.html?id=PT44" >Family Classification Without Domain Chaining</a></em></strong><br> Wednesday, July 1 - 2:45 p.m. - 3:10 p.m.<br>Room: K2<br />Presenting author: Jacob Joseph, Carnegie Mellon University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT44: Jacob Joseph - Family Classification Without Domain Chaining </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/4d1946da-b1a2-4e19-9e54-518376351e2f" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/4d1946da-b1a2-4e19-9e54-518376351e2f" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/4d1946da-b1a2-4e19-9e54-518376351e2f" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT45" >PT45</a></strong><br><strong><em><a href="papers19b2.html?id=PT45" >A Partition Function Algorithm for Interacting Nucleic Acid Strands</a></em></strong><br> Wednesday, July 1 - 3:15 p.m.- 3:40 p.m.<br>Room: Victoria Hall<br />Presenting author: Hamid Reza Chitsaz, Simon Fraser University<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT45: Hamid Reza Chitsaz - A Partition Function Algorithm for Interacting Nucleic Acid Strands </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/8d60db32-c606-4a42-9cc6-fe97716e17ac" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8d60db32-c606-4a42-9cc6-fe97716e17ac" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/8d60db32-c606-4a42-9cc6-fe97716e17ac" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/dswan" target="_top" style="text-decoration:none">Daniel Swan</a> and <a href="http://friendfeed.com/spitshine" target="_top" style="text-decoration:none">Roland Krause</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">First part presented by Raheleh Salari - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Increased interest in RNA-RNA interaction prediction requires computational target prediction. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">ncRNAs bind to mRNA and regulate translation, including the specificity etc. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Several models have been described, e.g. PairFold, RNAhybrid, RNAup, IRIS, InteRNA. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Problem np-complete - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">All current approaches do not include the probability and stability of the joint secondary structure. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Interaction energy model and interaction partition function over all SS without pseudoknots or crossing interactions and zigzags. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">The standard model by Matthews et al (1999) assumes an energy model with independence between hairpins, bulges etc. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">More than one loop-loop interaction in a real example, ignore interhybrid loop, other loops termed kissing loops. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top"># that was a little fast - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/8d60db32-c606-4a42-9cc6-fe97716e17ac">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT46" >PT46</a></strong><br><strong><em><a href="papers1553.html?id=PT46" >Identifying Novel Constrained Elements by Exploiting Biased Substitution Patterns</a></em></strong><br> Wednesday, July 1 - 3:15 p.m.- 3:40 p.m.<br>Room: K2<br />Presenting author: Xiaohui Xie, University of California Irvine<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT46: Xiaohui Xie - Identifying Novel Constrained Elements by Exploiting Biased Substitution Patterns </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/5dd93b37-9c6d-4782-8ff2-bc65ee8f53a6" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5dd93b37-9c6d-4782-8ff2-bc65ee8f53a6" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/5dd93b37-9c6d-4782-8ff2-bc65ee8f53a6" target="_top">View in FriendFeed</a></b></div></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br><p><strong>Proceedings Track:</strong> <strong><a name="PT47" >PT47</a></strong><br><strong><em><a href="papers8ca7.html?id=PT47" >Predicting and Understanding the Stability of G-Quadruplexes</a></em></strong><br> Wednesday, July 1 - 3:45 p.m. - 4:10 p.m.<br>Room: Victoria Hall<br />Presenting author: Oliver Stegle, University of Cambridge<br><br/><div style="font-family:Arial,sans-serif;border-width:1px;border-width-value: 1px;border-width-ltr-source: physical;border-width-rtl-source: physical;border-style: solid;border-color: #437ec7;overflow-x: hidden;overflow-y: hidden;font-size: 13px;margin-right: 10px;margin-left:0px;" ><div style="float:left;line-height:0;padding:5px"><img src="http://www.iscb.org/cms_addon/friendfeeds/logoismbmedium.jpg" style="width:30px;height:30px" /></div><div style="padding-top:5px;margin-left:40px;font-size:13px"><a style="text-decoration:none" href="http://friendfeed.com/ismbeccb2009"" target="_top"><b>ISMB/ECCB 2009 Blog</b></font></a><div class="style1" style="padding:5px 2px;"><b>PT47: Oliver Stegle - Predicting and Understanding the Stability of G-Quadruplexes </b><div style="padding:0px 0px 5px 10px"><b><a style="text-decoration:none" href="http://friendfeed.com/e/1f47869a-cd9c-4406-9621-59aee1d702b4" target="_top">Add a Comment</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/1f47869a-cd9c-4406-9621-59aee1d702b4" target="_top">Like</a> - <a  style="text-decoration:none" href="http://friendfeed.com/e/1f47869a-cd9c-4406-9621-59aee1d702b4" target="_top">View in FriendFeed</a></b></div><div class="likes" style="padding:0px 5px 5px 10px"><a href="http://friendfeed.com/spitshine" target="_top" style="text-decoration:none">Roland Krause</a> liked this </div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">G-Quadruplexes are stable structures of RNA and DNA - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Typically of the from GGGACTAAGGGACTTCCCACTTGG - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Will form spontaneously, have role in transcriptional control and telomeres. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Are these patterns really stable? It's the first indicator of a functional role. Melting temp will be a proxy for stability - <a style="text-decoration:none" href="http://friendfeed.com/allyson" target="_top">Allyson Lister</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Overrepresented in promoter regions [Hupper and Balasumbramanian, 2007] - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Melting temperature can be predicted and experimentally verified. It's low throughput though, rules are limited, complicated non-linear relationships. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Gaussian processes (GP) regression with different error rates across the sequence. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Gives the posterior distribution of function values given a training set. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Needs a covariance function (kernel), a likelihood model and hyperparameters. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><div class="comment" style="padding:0px 5px 5px 10px;"><img src="http://www.iscb.org/cms_addon/friendfeeds/comment.gif" style="vertical-align:middle;padding-top:0px;padding-right:3px" /><font style="vertical-align:top">Product ansatz to construct a joint covariance function of concentration and sequence. - <a style="text-decoration:none" href="http://friendfeed.com/spitshine" target="_top">Roland Krause</a></font></div><a style="font-style:italic;padding:0px 5px 5px 10px;text-decoration:none;" href="http://friendfeed.com/e/1f47869a-cd9c-4406-9621-59aee1d702b4">...more...</a></span></div></div></div><a href="#top" style="font-size:9px;">&uarr; TOP</a><br><br></div>	<!-- end #mainContent --></div>
	<!-- This clearing element should immediately follow the #mainContent div in order to force the #container div to contain all child floats --><br class="clearfloat" />
<!--  <div id="footer">
    <p>Footer</p>
     </div> -->
<!-- end #container -->
</body>

<!-- Mirrored from www.iscb.org/ismbeccb2009/proceedingstrackbytime.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:22:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
</html>





