<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/sigs.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent"><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/sigs.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Untitled Document</title>
<style type="text/css">
<!--
p.MsoNormal {
	color:#333333;
	font-family:Arial,Verdana,Helvetica,sans-serif;
	font-size: 12.8px;
	line-height: 18px;
	text-align:left;
}
li.MsoNormal {
margin:0in;
margin-bottom:.0001pt;
	color:#333333;
	font-family:Arial,Verdana,Helvetica,sans-serif;
	font-size: 12.8px;
	line-height: 18px;
	text-align:left;
}
-->
</style>
</head>

<body>

<p class="smhead1" ><strong><span style="font-family:Helvetica; color:#823334; ">Satellite  Meetings and SIGs</span></strong><span style="font-family:Helvetica; color:#823334; "> </span></p>

<div class="style1">
<p class="MsoNormal"><span >ISMB/ECCB 2009 will hold a number of one-day and two-day  specialized meetings in computational biology. These meetings will be held on  Saturday, June 27 and Sunday, June 28. All meetings will be held at the  Stockholmsmässan (Stockholm International Fairs). Program information and registration details are  noted below.</span></p>

<p class="MsoNormal"><span >To inquire about the possibility of forming a new SIG for ISMB  2010 or beyond, please address your inquiry to </span><span style="font-family:ArialMT; color:#333333; "><a href="mailto:steven@iscb.org"><span style="font-family:Helvetica; color:#3C68C8; ">steven@iscb.org</span></a></span><br />
  <br />
</p>

<div class="subheading1"><strong>SATELLITE  MEETINGS</strong></div>

<p class="MsoNormal"><span >Registrants are not required to register as delegates for the  ISMB/ECCB 2009 conference to attend a Satellite Meeting. You may select  &quot;Satellite only&quot; to register just for the 3Dsig Satellite Meeting.  Please note Satellite Meeting registration <strong>does  not</strong> allow access to the SIG Meetings.</span></p>

<p class="MsoNormal"><span >The registration fee for the Satellite Meeting includes daily  lunches and the 3Dsig Evening Session and Dinner. Each Satellite Meeting  participant will receive a booklet that includes meeting notes.</span><br />
</p>


<div class="subheading1"><strong>Two-Day Satellite Meeting - Saturday, June 27 and Sunday, June 28</strong></div><br />

<a href="#3dsig">3Dsig: Structural Bioinformatics  &amp; Computational Biophysics Satellite Meeting</a><br />
<br />
<br />


<div class="subheading1"><strong>SPECIAL INTEREST GROUP MEETINGS (SIGs)</strong></div>

<p class="MsoNormal">SIG Registrants are not required to register as delegates for  the ISMB/ECCB 2009 conference to attend a SIG meeting.
 You may select &quot;SIG only&quot; to register just for a SIG.</p>

<p class="MsoNormal">Registering for a SIG allows you to move freely between all SIGs  that take place on the same day as the meeting for which you are registered, to  the extent that the room capacities can accommodate. SIG registration <strong>does not</strong> allow access to the Satellite  Meeting.</p> 

<p class="MsoNormal"><span >Each SIG participant will receive a booklet that includes  handouts from all SIGs and daily lunches are included.</span></p>

<p class="MsoNormal"><span >Please register for the SIG in which you are most interested so  that we can assign SIGs to larger and smaller rooms as appropriate.</span></p>

<p class="MsoNormal"><span >NOTE: One-Day SIG registrants are able to participate in SIG  meetings only on the day for which they have registered unless registering for  two one-day SIG meetings.</span><br />
</p>
  
 <blockquote>

<strong>One-Day  SIGs (Pre-conference) - Saturday, June 27, 2009</strong><br />
<a href="#DAM">DAM: Data  and Analysis Management</a><br />
<a href="#M3">M3:  Metagenomics, Metadata and Metaanalysis</a><br />

<strong><span >One-Day  SIG (Pre-conference) - Sunday, June 28, 2009</span></strong><br />
<a href="#BioOntologies">Bio-Ontologies</a><br />
<a href="#Short-Read">Short-SIG: Next Generation Sequencing and Algorithms for Short Read Analysis</a><br />

<br />
<strong>Two-Day  SIG - Saturday, June 27 and Sunday, June 28, 2009</strong><br />
<a href="#Alternative">Alternative  Splicing AS-SIG</a><br />
<a href="#BOSC">Bioinformatics  Open Source Conference (BOSC)</a><br />
<a href="#BioPathways">BioPathways</a><br />
 
<strong><span >Two-Day  SIGs - Sunday, June 28 and Monday, June 29, 2009</span></strong><br />
<a href="#BioLINK">BioLINK</a> <br />

<span >
*BioLINK – Day 2 of the  SIG runs in parallel with the main conference on Monday, June 29.
The afternoon  program on Monday, June 29 is open to all delegates of ISMB/ECCB
</span>
</blockquote>
<br />
<br />



<div class="subheading1"><strong>SATELLITE  MEETINGS</strong></div>
<hr />

<a name="3dsig"></a>
<p class="MsoNormal"><strong><span >3Dsig:  Structural Bioinformatics &amp; Computational Biophysics Satellite Meeting</span></strong></p>

<p class="MsoNormal"><span >URL: </span><a href="https://www.ebi.ac.uk/~rafi/3dsig09" target="_blank">https://www.ebi.ac.uk/~rafi/3dsig09</a></p>

<p class="MsoNormal"><span ><strong>Date:</strong> Saturday, June 27 and Sunday, June 28</span></p>
<p class="MsoNormal"><span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span></p>

<span ><strong>Room Location:</strong> TBD</span><br /><br />

<p class="MsoNormal"><span >(*Please note that the program ends Sunday, June 28 at  6:00 p.m. to allow movement to Stockholm City Hall for the Opening Reception  hosted by the City of Stockholm)</span></p>

<span >*Saturday, June 27 - 3Dsig Dinner and Evening  Presentation</span><br />
<span >7:00 p.m. Reception (Cash bar)</span><br />
<span >7:30 p.m. Dinner</span><br /><br />

<p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=-DAM&amp;intitle=3d+SIG&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>

<iframe src="http://friendfeed.com/iphone/search?q=group%3Aismbeccb2009+intitle%3A3d%2BSIG+-DAM" height="250" width="100%"></iframe>
<p class="MsoNormal" ><span >The  2009 3DSIG: Structural Bioinformatics and Computational Biophysics satellite  meeting is in its fifth instalment &nbsp;in what has become the most important  annual meeting serving the computational structural molecular biology  community. Over the years 3DSIG has presented a very dynamic program comprising  two full days bringing together the established as well as up-and-coming  leaders in the field in a series of 6 keynote lectures, 20 oral presentations  chosen from within the contributed abstracts, a reception followed by a dinner  and a post-dinner presentation and our very own signature poster/laptop session  where all accepted contributors are invited to present their work as a  traditional poster but also as an interactive laptop presentation and  demonstration.</span></p>

<p class="MsoNormal" ><span >In  2008, the best two student presentations were awarded a prize (sponsored by  MGMS). The list of past keynote speakers includes but is not restricted to: Rob  Russell, Ron Unger, Gunnar von Heijne, Cyrus Chothia, Phil Bourne, Nick  Grishin, Wah Chiu, John Moult, Jeff Saven, Janet Thornton, Steven Brenner, Anna  Tramontano, Thomas Lengauer, Kevin Karplus, Sandor Vajda, Burkhard Rost, Ivet  Bahar, Barry Honig, Alfonso Valencia, Tamar Schlick, Tanja Kortemme, Brian  Shoichet. All aspects of computational molecular biology are welcome. The  deadline for submissions is June 4th. For more information, including key dates  and the list of confirmed keynote speakers for 2009, please visit: <a href="https://www.ebi.ac.uk/~rafi/3dsig09"><span style="color:#001FE8; ">https://www.ebi.ac.uk/~rafi/3dsig09</span></a></span><br />
</p>


<a href="#top">top</a><br />
<br />
<br />
<div class="subheading1"><strong>3DSig  Organizing Committee:</strong></div><br />
John Moult<br />
Biotechnology  Institute<br />
University of  Maryland, USA<br />
&nbsp;<br />
Ilan Samish<br />
Dept of Biochemistry  &amp; Biophysics&nbsp;and Dept. of Chemistry<br />
University of  Pennsylvania, USA<br />
&nbsp;<br />
Rafael Najmanovich<br />
Département de  Biochimie<br />
Faculté de Médecine<br />
Université de  Sherbrooke, Canada<br />
<br />

<strong>Contact:</strong> <a href="mailto:3dsig09@googlemail.com">3dsig09@googlemail.com</a><br />
<br />



<strong>Keydates:</strong>

<p class="MsoNormal"><span ><strong>February 16, 2009</strong> -  Registration opens for 3Dsig and ISMB/ECCB</span></p>

<p class="MsoNormal"><span ><strong>March 18, 2009</strong> -  Deadline for submitting an abstract to 3Dsig to be considered for oral  presentation.</span></p>

<p class="MsoNormal"><span ><strong>April 2, 2009</strong> - Deadline  for travel fellowship application for the main ISMB/ECCB conference. Remember  to be eligible to receive an ISMB/ECCB travel fellowship you must have an  accepted paper or poster for the 2009 conference.</span></p>

<p class="MsoNormal"><span ><strong>April 15, 2009</strong> -  Tentative date for the announcement of the abstracts selected for oral  presentation and laptop/poster session (full program and online abstracts will  be posted soon afterwards).</span></p>

<p class="MsoNormal"><span ><strong>April 29, 2009</strong> - Late  3Dsig abstract submission deadline. Late submissions will not be considered for  oral presentation. Late submission deadline for ISMB/ECCB posters. </span></p>

<p class="MsoNormal"><span ><strong>May 15, 2009</strong> - Early  registration deadline to the main ISMB/ECCB conference. Note: you may register  only for 3Dsig.</span></p>

<p class="MsoNormal"><span ><strong>June 27-28, 2009</strong> - 3Dsig  2009</span></p>



<a href="#top">top</a><br />
<br />
<br />

<div class="subheading1"><strong>SPECIAL  INTEREST GROUP MEETINGS (SIGs)</strong></div>
<hr />
<br />
<a name="Alternative"></a>

<div class="subheading1"><strong><span >Alternative Splicing (AS-SIG)</span></strong></div><br />

To register by fax for the AS-SIG informal dinner on Saturday, June 27 click <a href="downloads/ASSIG_Dinner09.doc">here</a> <br /><br />

URL: <a href="http://www.alternative-splicing.org/as-sig-09/">http://www.alternative-splicing.org/as-sig-09/</a><br />
<strong>Date:</strong> <strong>Saturday,  June 27 and Sunday, June 28, 2009</strong> <br />
<strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m. <br />
<strong>Room Location:</strong> TBD <br /><br />

(*Please note that the program ends Sunday, June 28 at  6:00 p.m. to allow movement to Stockholm City Hall for the Opening Reception  hosted by the City of Stockholm)<br />
   <br />
   <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=intitle%3AAlternative+Splicing+SIG+group%3Aismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
  <iframe src="http://friendfeed.com/iphone/search?q=intitle%3A%28alternative+splicing%29+SIG+group%3Aismbeccb2009" height="250" width="100%"></iframe>
<br />

<p class="MsoNormal"><strong>Deciphering the splicing code: the role of  protein-RNA interactions</strong></p>

<p>
In order to unravel the complete regulatory expression network, it is necessary to systematically identify protein-RNA interactions. In vitro techniques have been successful in identifying the motifs bound by regulators. However, these motifs are relatively short and bioinformatics searches produce a large number of false positives. Recent advances in sequencing technologies and systematic structural studies have provided new insight in how proteins interact with RNA. These technological advances provide an unprecedented opportunity to explore how RNA is regulated at a global level. These advances also pose new computational challenges. An extensive interaction between computational and experimental researchers is therefore required, aimed at building a community of shared resources and tools.
</p>
<p>
The AS-SIG meeting provides a forum for the discussion of the latest findings and to establish new collaborations in the field of splicing and RNA biology. AS-SIG 2009 format follows previous successful AS-SIG meetings at Detroit (ISMB2005), Fortaleza (ISMB2006), Vienna (ISMB2007) and Toronto (ISMB2008). AS-SIG 2009 will take place immediately before ISMB/ECCB 2009. This year's location, Stockholm, Sweden, provides an excellent opportunity for attracting scientists from all over Europe and the world, and will facilitate broadening the scope of the AS-SIG meeting, as well as the interactions with the international community.
</p>
<p>
This year there will be special emphasis on high-throughput technologies in the study of RNA processing and protein-RNA interactions. Planned topics are:</p>

<p>
<strong>RNA processing and protein-RNA interactions</strong><br />
Protein-RNA and RNA-RNA interactions in gene regulation and the pathogenesis of cancer and other diseases. Coupling of transcription and splicing and regulation of splicing by chromatin modification. Special emphasis will be given on structure-function relationships.</p>

<p>
<strong>RNA regulatory maps</strong><br />
Splicing factors and their cognate cis-regulatory elements; next-generation sequencing for the analysis of protein-RNA interactions, RNA expression, and the detection of regulatory RNAs.
</p>
<br />


<strong>Keydates</strong><br />


<strong>February 2009</strong>		AS-SIG call for papers<br />
<strong>March 2009</strong>			AS-SIG announcement of accepted papers<br />
<strong>April 2009</strong>			AS-SIG selection (late) of posters<br />
<strong>June 27-28 2009</strong>		AS-SIG meeting<br />

<br />

<strong>Contacts:</strong>  <br />


Eduardo Eyras, Pompeu Fabra University, PRBB, Barcelona, Spain<br />
Email: <a href="mailto:eduardo.eyras@upf.edu">eduardo.eyras@upf.edu</a><br />
<br />

Dirk Holste, Austria Research Centers – ARC and University of Vienna, Austria <br />
Email: <a href="mailto:holste@imp.ac.at">holste@imp.ac.at</a><br /><br />


Frederic Allain, ETH Zurich, Institute of Molecular Biology and Biophysics, Zurich, Switzerland<br />
E-mail: <a href="mailto:frederic.allain@mol.bio.ethz.ch">frederic.allain@mol.bio.ethz.ch</a><br />
<br />



Rickard Sandberg, Karolinska Insititute, Stockholm, Sweden<br />
E-mail:  <a href="mailto:rickard.sandberg@ki.se">rickard.sandberg@ki.se</a><br />
<br />

Graziano Pesole, University of Bari and Istituto Tecnologie Biomediche del CNR (sede di Bari) via Orabona, Italy<br />
E-mail: <a href="mailto:graziano.pesole@biologia.uniba.it">graziano.pesole@biologia.uniba.it</a><br />
<br />





<br />

<a name="BOSC"></a>
<div class="subheading1"><strong><span >Bioinformatics Open Source Conference (BOSC)</span></strong></div><br />


<span >URL: </span><a href="http://www.open-bio.org/wiki/BOSC_2009">http://www.open-bio.org/wiki/BOSC_2009</a><br />

<span ><strong>Date:</strong> <strong>Saturday,  June 27 and Sunday, June 28, 2009</strong></span><br />
<span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span><br />
<span ><strong>Room Location:</strong> TBD</span><br /><br />

<span>(*Please note that the program ends  Sunday, June 28 at 6:00 p.m. to allow movement to Stockholm City Hall for the  Opening Reception hosted by the City of Stockholm)</span><br />
               <br />
                <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=BOSC&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
<iframe src="http://friendfeed.com/iphone/search?q=group%3Aismbeccb2009+intitle%3ABOSC%2BSIG" height="250" width="100%"></iframe>
    <br />

               <br />

<p class="MsoNormal"><span >The Bioinformatics Open  Source Conference (BOSC) is sponsored by the Open Bioinformatics Foundation  (O|B|F), a non-profit group dedicated to promoting the practice and philosophy  of Open Source software development within the biological research community.  To be considered for acceptance, software systems representing the central  topic in a presentation submitted to BOSC must be licensed with a recognized Open  Source License, and be freely available for download in source code form.</span></p>

<p class="MsoNormal">Many open source bioinformatics packages are widely used by the research community across many  application areas and form a cornerstone in enabling research in the genomic  and post-genomic era. Open source bioinformatics software has facilitated rapid  innovation, dissemination, and wide adoption of new computational methods,  reusable software components, and standards. One of the hallmarks of BOSC is  the coming together of the open source developer community in one  location to meet face-to-face.  This creates synergy where  participants can work together to create use cases, prototype working code, or  run bootcamps for developers from other projects as short, informal, and  hands-on tutorials in new software packages and emerging technologies.&nbsp; In  short, BOSC is not just a conference for presentations of completed work, but  is a dynamic meeting where collaborative work gets done and attendees can learn  about new or on-going developments that they can directly apply to their own  work.
</p>

<p class="MsoNormal"> This year’s conference  will mark the 10th anniversary of BOSC.&nbsp; To celebrate the special  occasion, the theme of this year’s conference is “Looking Back and Looking  Ahead:  Open Source Solutions to Grand  Challenges in Bioinformatics.”  We are  inviting abstracts for two different types of talks: </p>

<ul type="disc">
  <li class="MsoNormal"> Descriptions of a particular open source software implementation by a member of the development team, especially in the areas listed below.</span></li>
  <li class="MsoNormal"> Reviews of open source software that compares and contrasts different solutions to the same bioinformatics problem.</span></li>
</ul>

<p class="MsoNormal">
In  addition, we are asking all speakers to come prepared to lead an informal
tutorial on their software during a Birds of a Feather/hackathon session.  This year’s topics include:</p>

<ul type="disc">
  <li class="MsoNormal"><span >Design Patterns in Bioinformatics</span></li>
  <li class="MsoNormal"><span >Regulatory Genomics</span></li>
  <li class="MsoNormal"><span >Multicore and GPGPU computing</span></li>
  <li class="MsoNormal"><span >Data &amp; Analysis Management (shared session with DAM SIG)</span></li>
  <li class="MsoNormal"><span >Computational Grids</span></li>
  <li class="MsoNormal"><span >Visualization</span></li>
</ul>

<p class="MsoNormal">There will also be  updates from O|B|F-sponsored projects, and as always,
abstracts may be  submitted for open source software that does not fit neatly into the above categories.
Lightning Talks will also highlight very recent developments.
</p>

<p class="MsoNormal"><strong> Abstract deadline is April 13, 2009 </strong>. </p>

<p class="MsoNormal"><strong>BOSC Coordinator:</strong></p>

<span >Kam D. Dahlquist, Ph.D.</span><br />
<span >Department of Biology</span><br />
<span >Loyola Marymount  University</span><br />
<span >Los Angeles, California,  USA</span><br />
<span >Email: <a href="mailto:kdahlquist@lmu.edu">kdahlquist@lmu.edu</a> or <a href="mailto:bosc@open-bio.org">bosc@open-bio.org</a></span><br />


<br />
<a href="#top">top</a><br /><br /><br />


<a name="BioOntologies"></a>
<div class="subheading1"><strong>Bio-Ontologies</strong></div><br />

<span >URL:</span> <a href="http://bio-ontologies.org.uk/">http://bio-ontologies.org.uk/</a> <br />
<span ><strong>Date:</strong> <strong>Sunday,  June 28, 2009</strong></span><br>
<span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span><br>
<span ><strong>Room Location:</strong> TBD</span><br>
<br />
            <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=bio-ontologies&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
<iframe src="http://friendfeed.com/iphone/search?q=SIG+group%3Aismbeccb2009+intitle%3AOntologies" height="250" width="100%"></iframe>
<br />
<br />

<span style="font-family:Helvetica; font-size:10.0pt; color:#333333; ">
(*Please note that the program ends  Sunday, June 28 at 6:00 p.m. to allow movement to Stockholm City Hall for the  Opening Reception hosted by the City of Stockholm)</span><br>


<p>
Bio-Ontologies has been a Special Interest Group (SIG) at ISMB for the last 11 years, providing an environment for sharing knowledge and learning about ontologies and their application to biology. Over the years, the Bio-Ontologies SIG has provided a forum for discussion on the latest and most innovative topics in this area. Last year, the scope was increased to include formal and informal approaches to organizing, curating, representing and disseminating knowledge in biology. Papers will be welcomed in traditional areas, such as the biological applications of ontologies, newly developed Bio-Ontologies, and the use of ontologies in data communication standards. In addition, papers will also be welcomed in the following areas:
<ul>
<li>Semantic and/or scientific wikis </li>
<li>Folksonomies and tag clouds </li>
<li>Collaborative curation platforms </li>
<li>Collaborative ontology authoring and peer-review mechanisms </li>
<li>Automated ontology learning </li>
<li>Mapping between ontologies  </li>
</ul>
</p>
<p>
Each year, we focus on a specific theme in our keynote and panel sessions. For 2009 the theme will be: “The emergence, use and success of wikis for collaborative knowledge capture for biology”.</p>

<strong>Key Dates</strong><br /><br />


Friday, April 10, 2009			Paper Submissions Due<br />
Friday, May 1 2009			Notification of Accepted Papers<br />
Friday, May 8, 2009			Final Version Due<br />
<br />

<strong>Contacts</strong><br /><br />
General Contact: bioontologies2009@gmail.com <br /><br />

Nigam Shah, Stanford University<br />
Email: <a href="mailto:nigam@stanford.edu">nigam@stanford.edu</a> <br /><br />


Phillip Lord, Newcastle University <br />
Email: <a href="mailto:phillip.lord@newcastle.ac.uk">phillip.lord@newcastle.ac.uk</a> <br /><br />


Susanna-Assunta Sansone, EBI <br />
Email: <a href="mailto:sansone@ebi.ac.uk">sansone@ebi.ac.uk</a> <br /><br />


Larisa Soldatova, University of Aberswyth <br />
Email: <a href="mailto:lss@aber.ac.uk">lss@aber.ac.uk</a> <br /><br />

Susie M Stephens, Eli Lilly <br />
Email: <a href="mailto:stephens_susie_m@lilly.com">stephens_susie_m@lilly.com</a> <br /><br />

SIG URL <br />
<a target="_blank" href="http://www.bio-ontologies.org.uk/">www.bio-ontologies.org.uk</a> <br />







<br />
<a href="#top">top</a><br /><br /><br />

<a name="BioPathways"></a>
<div class="subheading1"><strong>BioPathways</strong></div><br />


URL: </span><a href="http://www.biopathways.org/"><span >http://www.biopathways.org</span></a><br />
<span ><strong>Date:</strong> <strong>Saturday,  June 27 and Sunday, June 28, 2009</strong></span><br />
<span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span><br />
<span ><strong>Room Location:</strong> TBD</span><br /><br />


<span style="font-family:Helvetica; font-size:10.0pt; color:#333333; ">(*Please note that the program ends  Sunday, June 28 at 6:00 p.m. to allow movement to Stockholm City Hall for the  Opening Reception hosted by the City of Stockholm)</span><br />
 <br />
             <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=BioPathways&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
 <iframe src="http://friendfeed.com/iphone/search?q=SIG+group%3Aismbeccb2009+intitle%3ABioPathways" height="250" width="100%"></iframe>

<br />

<p class="MsoNormal"><span >The 10th BioPathways  meeting is organized by the BioPathways Consortium, an open forum aimed at  fostering computational approaches to the understanding of biological networks  in biotechnology and biomedicine.</span></p>
<p class="MsoNormal"><span >Previous BioPathways  meetings have focused on a variety of themes, such as computational  reconstruction of molecular networks, pathway evolution, integration of models  and experiments, models and ontologies for pathways, modeling of interactions  and regulation on a systems scale and network approaches for discovering  disease associations, among others.</span></p>
<p class="MsoNormal"><span >This year’s meeting will  remain within the broad spectrum of computational and semantic approaches for  molecular network analysis and systems modeling, featuring also presentations  on pathway analysis tools, metabolism, signaling or regulation, selected from  the most notable recently published work or from yet unpublished work. In  addition, one session each will be dedicated to the three specific thrusts,  namely, </span></p>

<span >a) Design Tools for  Synthetic Biology,  </span><br />
<span >b) Network Analysis of  Metagenomes,</span><br />
<span >c) Computational Pathway  Methods for Translational Medicine. </span><br />

<p class="MsoNormal"><span >Brief descriptions of  the scope of each of these themes are available at the</span> <span >BioPathways site (</span><a href="http://www.biopathways.org/"><span >www.biopathways.org</span></a><span >).</span><span > </span><span >As  customary, the meeting will consist of two full days with a </span><span >balance of invited talks, short oral  presentations, and critical, topic-focused panel discussions. We strongly  encourage the presentation of </span><span >work-in-progress in cutting-edge research.</span></p>

<p class="MsoNormal"><span >If you wish to present a  contributed talk at BioPathways 2009, please send a 250-word abstract  describing your work. Authors are encouraged to add URLs of web-based tools,  and selected tools will be featured on the BioPathways web site. Abstracts  should arrive by April 21, 2009, and presenters will be notified no later than  May 21, 2009. Submissions should be sent to: </span><span style="color:blue; "><a href="mailto:BioPathways@Helmholtz-HZI.de"><span style="text-underline:none; font-family:Helvetica; text-decoration:none; ">BioPathways@Helmholtz-HZI.de</span></a></span><span > </span></p>

<p class="MsoNormal"><span >We look forward to  seeing you at BioPathways 2009 - from the organizers: Vítor Martins dos Santos,  Helmholtz Centre for Infection Research, Germany (chair), Vincent Schächter,  Genoscope, France, Vincent Danos, University Edinburgh, Joanne Luciano, MITRE  Corp. and Predictive Medicine Inc, Eric Neumann, Clinical Semantics Group and  Aviv Regev, Broad Institute and MIT.</span></p>

<strong><span >Contact:</span></strong><br />
<span >Vítor  Martins dos Santos</span><br />
<span >Helmholtz  Centre for Infection Research</span><br>
<span >Germany</span><span > (chair)</span><span > </span><br>
<span >Email: </span><a href="mailto:vds@helmholtz-hzi.de"><span >vds@helmholtz-hzi.de</span></a><span > </span><br>



<br /><a href="#top">top</a><br /><br /><br />

<a name="BioLINK"></a>
<div class="subheading1"><strong>BioLINK:  Linking Literature, Information and Knowledge for Biology</strong></div><br />


URL: <a href="http://www.cs.queensu.ca/biolink09">http://www.cs.queensu.ca/biolink09</a></span><br />
<strong>Date:</strong> <strong>Sunday,  June 28 and Monday, June 29, 2009</strong></span><br />

<strong>Start time:</strong> 8:30 a.m. - End time: *6:00 p.m.<br />

<strong>Room Location:</strong> TBD<br />

<p class="MsoNormal"><span >(Note: BioLINK – Day 2  of the SIG runs in parallel with the main conference on Monday, June 29. The  afternoon program on Monday, June 29 is open to all delegates of ISMB/ECCB)</span></p>
     <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=BioLink&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>

    <iframe src="http://friendfeed.com/iphone/search?q=SIG+group%3Aismbeccb2009+intitle%3ABioLINK" height="250" width="100%"></iframe>

<br />

<p class="MsoNormal"><span >The Annual Meeting of  the ISMB BioLINK Special Interest Group:   Text Mining, Image Analysis, and the future of Scientific Publishing</span></p>

<p class="MsoNormal"><span >The BioLINK SIG meeting  has been regularly held in association with the ISMB conference since 2001,  focusing on the development and application of resources and tools for  biomedical text mining. The SIG is interdisciplinary in nature, and brings  together researchers applying natural language processing, text mining,  information extraction and retrieval in the biomedical domain, with scientists  from bioinformatics and biology.</span></p>

<p class="MsoNormal"><span >This year the SIG will  go beyond text to take into account the analysis of images and figures in  scientific publications because of their critical information content. Furthermore,  we will include a session about the future of scientific publishing, and the  impact of information extraction methods (be they based on text or on images)  on producers and consumers of scientific information. The SIG will feature a  special session (on Monday), including presentations and discussions by  representatives from scientific journals and community members. This session is  jointly organized with PLoS, and is open to all ISMB attendees.</span></p>

<p class="MsoNormal">The meeting includes  invited talks, presentations of peer-reviewed contributed papers, reports from  recent evaluations and workshops, as well as a poster session. The SIG solicits  papers that discuss current biological image- and text-related needs, the  challenges of meeting them, and the tools that may help address them. Specific  topics include: biological image analysis and its relationship to biomedical  text mining, application of text-mining tools to biomedical research, automated  indexing of the biomedical literature, generation of structured digital abstracts,  new evaluation measures to measure utility and usability of text mining tools  integrated into the biologists’ workflow, as well as new tools and  applications, and future directions in biomedical text mining.
</p>

<p class="MsoNormal">We anticipate a  productive workshop that will facilitate discussion and exchange of ideas by  bringing together multiple communities:   the users of text mining tools, including curators of biological  databases, bench scientists and bioinformaticians; and the researchers applying  natural language processing, ontologies, text mining, image analysis,  information extraction and retrieval to problems in the biomedical domain.</p>

Information about last year's meeting is available at the <a target="_blank" href="http://www.cs.queensu.ca/biolink08">BioLINK08 web site</a><br />


<p class="MsoNormal"><strong>Keydates</strong>:</p>

<p class="MsoNormal"><span ><strong>April 5:</strong> Short papers  due </span><br />
    <span ><strong>April 15:</strong> Poster Abstracts (up 350 words)  due </span><br />
    <span ><strong>May 5:</strong> Notification to authors</span><br />
    <span ><strong>May 15:</strong> Final version due</span></p>

<p class="MsoNormal"><strong><span >Contacts</span></strong><span >:</span><br /></p>
<span >Christian Blaschke</span><br />
<span >Bioalma, Spain</span><br />

<span >Email: </span><a href="mailto:blaschke@almabioinfo.com">blaschke@almabioinfo.com</a><br />

<br />
Hagit Shatkay<br />
Queen's University,  Canada<br />


<span >Email: </span>
<a href="mailto:shatkay@cs.queensu.ca">shatkay@cs.queensu.ca</a>
<br />
<br />

<a href="#top">top</a><br /><br /><br />

<a name="DAM"></a>
<div class="subheading1"><strong>DAM: Data and Analysis Management</strong></div><br />


URL: <a href="http://dam-sig.org/">http://dam-sig.org</a><br />

<span ><strong>Date:</strong> <strong>Saturday,  June 27, 2009</strong></span><br />

<span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span><br />

<span ><strong>Room Location:</strong> TBD</span><br /><br />

(*Please note that the program ends Sunday, June 28 at 6:00 p.m. to allow movement to Stockholm City Hall for the Opening Reception hosted by the City of Stockholm)<br />
<br />
     <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=DAM&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
      <iframe src="http://friendfeed.com/iphone/search?q=SIG+group%3Aismbeccb2009+intitle%3ADAM" height="250" width="100%"></iframe>
<br />


<p class="MsoNormal"><span >Over the  last few years we have seen another explosion in the data generation capacity  of experimental biology. New DNA sequencing and high-density array technologies  have enabled a variety of novel genome scale experimental techniques. What is  perhaps most unique about this recent data explosion is that it is distributed  – relatively inexpensive instruments allow any lab or institution to produce  enormous amounts of data. Because of this change, the traditional model of  centralized data warehouses for biological data is no longer sufficient. This  has led to a new generation of systems for data management and analysis which  are designed to put make it possible for experimentalists to effectively manage  and analyze the wealth of data they can now produce, and integrate that data  with existing date warehouses and literature, with little or no expert  informatics support.</span></p>


<p class="MsoNormal"><span >These new  approaches are critical if the promise of next generation experimental  techniques is to be realized. This session will serve as an open forum for  introduction of cutting edge applications that mold data sources and analysis  tools into a singular medium benefitting both extremes on the user spectrum:  experimentalists and computational researchers.</span></p>
<br />
<strong>Confirmed Speakers:</strong><br />
Carole Goble (University of Manchester)<br />
Christopher Lee (UCLA)<br />
Jill Mesirov (Broad Institute)<br />
Gos Micklem (Cambrigde)<br />
<br />


<strong><span >Contacts:</span></strong><br />

<span >James  Taylor</span><br />

<span >Emory  University, Atlanta, UA</span><br />
<span >Email: </span><a href="mailto:james.taylor@emory.edu">james.taylor@emory.edu</a><br />
<a href="http://bx.mathcs.emory.edu/">http://bx.mathcs.emory.edu</a><br />
<br />


Anton  Nekrutenko<br />
Penn State  University<br />
University  Park, USA<br />

Email: <a  href="mailto:anton@bx.psu.edu">anton@bx.psu.edu</a><br />
<a target="_blank" href="http://nekrut.bx.psu.edu/">http://nekrut.bx.psu.edu</a><br />
<br />




<br />
<a href="#top">top</a><br /><br /><br />




<div class="subheading1"><a name="M3"></a><strong>M3: Metagenomics, Metadata and Metaanalysis</strong></span></div>

<br />

URL: <a href="http://gensc.org/">http://gensc.org/</a><br />

<span ><strong>Date:</strong> <strong>Saturday,  June 27, 2009</strong></span><br />

<span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span><br />

<span ><strong>Room Location:</strong> TBD</span><br /><br />

(*Please note that the program ends Sunday, June 28 at 6:00 p.m. to allow movement to Stockholm City Hall for the Opening Reception hosted by the City of Stockholm)<br />
<br />          <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=M3&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
            <iframe src="http://friendfeed.com/iphone/search?q=SIG+group%3Aismbeccb2009+intitle%3AM3" height="250" width="100%"></iframe>
<br />


<p class="MsoNormal" ><span >While genomes  represent the full genetic (DNA) complement of a single organism, metagenomes  represent the DNA of an entire community of organisms. Metagenomes are partial  samples of complex and largely unknown communities that can only be poorly  assembled. This SIG will explore the latest concepts, algorithms, tools,  informatics pipelines, databases and standards being developed to cope with the  analysis of vast quantities of metagenomic data.&nbsp; Contributed submissions  based on talks and posters that describe studies that demonstrate the power of  using curated (e.g. habitat or host) and measured (e.g. geographic location,  salinity, temperature, or pH) contextual data in comparative metagenomic  studies of large numbers of samples are encouraged. Likewise, papers describing  new approaches, tools, databases, standards, ontologies or substantial new sets  of curated metadata that aid in the integration and inter-operablility of  disparate datasets are welcomed.</span></p>
<p class="MsoNormal" ><span >The  M3 meeting will include 3 sessions, presenting talks on Metagenomics, Metadata  and MetaAnalysis.&nbsp; The meeting will also include a Panel discussion and  lightening talks (5) minutes from selected poster submissions.</span><span > </span></p>
<p class="MsoNormal" ><span >If  you wish to present a contributed talk or poster at M3 2009, please prepare an  abstract describing your work. Abstracts should arrive by April 1, 2009 and  presenters will be notified no later than April 15, 2009. Further details can be  found on the M3 homepage at
<a href="http://gensc.org/">http://gensc.org</a>.</p>
<p class="MsoNormal" >We  look forward to welcoming you to the GSC's M3 in 2009.</p>

Contacts:<br />
Dawn  Field<br />
Email: <a href="mailto:dfield@ceh.ac.uk">dfield@ceh.ac.uk</a><br />


Iddo Friedberg
Email: <a href="mailto:idoerg@gmail.com">idoerg@gmail.com</a><br />
<br />

<a href="#top">top</a><br /><br />


<a name="Short-Read"></a>
<div class="subheading1"><strong>Short-SIG: Next Generation Sequencing and Algorithms for Short Read Analysis</strong></div>
<br />


<span >URL:</span><span ><strong>Date:</strong> <strong>Sunday,  June 28, 2009</strong></span><br />

<span ><strong>Start time:</strong> 8:30 a.m. - End time: *6:30 p.m.</span><br />

<span ><strong>Room Location:</strong> TBD</span><br /><br />

 <p style="text-align:right"><b>Click <a href="http://friendfeed.com/search?q=SIG&amp;intitle=Short-Read&amp;group=ismbeccb2009">
 here</a> to view the corresponding feed entries in FriendFeed.</b></p>
             <iframe src="http://friendfeed.com/iphone/search?q=SIG+group%3Aismbeccb2009+intitle%3AShort-Read" height="250" width="100%"></iframe>
           <br />


<p>
While the promise of next generation sequencing (NGS) technologies has become a reality, and in the past year several computational methods for assembly and alignment with short reads have been developed, realizing the full promise of these technologies requires the development of methods that can analyze the resulting assemblies and mappings of the reads to infer biological meaning – genome variation, novel transcript discovery, microRNA expression analysis, and metagenomic analysis.
</p>
<p>
In particular, we are interested in presentations describing methodology to infer various polymorphisms (SNPs, large insertions/deletions, copy number variations) with short read data.. As the HapMap project has increased our understanding of SNPs, structural variation (including insertions, deletions, translocation, inversions, and CNVs) have come to the forefront as one of the main sources of variation within a species. NGS technologies offer the potential of high resolution detection of most types of genomic variants that are just beginning to be explored.
</p>

<p>
Another exciting application of NGS technologies is RNA sequencing. RNA sequencing is currently used for several applications, including RNA expression, de-novo transcriptome sequencing for  non-model organisms, and novel transcript discovery, however computational methods for analysis of this data is in its infancy. For RNA and micro-RNA expression profiling, NGS has significant advantages compared to microarray methods in that it is better able to identify quantities of very common and very rare transcripts. Transcriptome profiling is also increasingly used for sequencing the RNA of non-model organisms for phylogenetic and population studies, and methods for transcriptome assembly and co-assembly need to be improved.
</p>
<p>
NGS Sequencing is also commonly used for discovery of transcription factor binding sites (and nucleosome positioning) using CHiP-SEQ, for methylation profiling, and for analysis of metagenomic samples. We will certainly welcome presentations on these topics, as well as any other important methods related to NGS technologies.
</p>
<p>
Submission information: TBA, watch website.</p>


<strong>Coordinator: </strong><br>

Michael Brudno, <a href="mailto:brudno@cs.toronto.edu">brudno@cs.toronto.edu</a><br>
Francisco M De La Vega, <a href="mailto:francisco.delavega@appliedbiosystems.com">francisco.delavega@appliedbiosystems.com</a>
<br>
Jens Stoye, <a href="mailto:stoye@techfak.uni-bielefeld.de">stoye@techfak.uni-bielefeld.de</a>
<br><br>

SIG URL:
<a href="http://compbio.cs.toronto.edu/ismb09-sig/" target="_blank">http://compbio.cs.toronto.edu/ismb09-sig/</a>

<br />
<br />
<a href="#top">top</a><br />

<br />
<br /></div>

</body>

<!-- Mirrored from www.iscb.org/ismbeccb2009/sigs.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
</html>



	<!-- end #mainContent --></div>
	<!-- This clearing element should immediately follow the #mainContent div in order to force the #container div to contain all child floats --><br class="clearfloat" />
<!--  <div id="footer">
    <p>Footer</p>
     </div> -->
<!-- end #container -->
</body>

<!-- Mirrored from www.iscb.org/ismbeccb2009/sigs.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:19:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
</html>