<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<!-- Mirrored from www.iscb.org/ismbeccb2009/tutorials.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:18:08 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
        
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>ISMB ECCB 2009</title>
<link href="includes/css/twoColFixLtHdr.css" rel="stylesheet" type="text/css" />
</head>
<body class="twoColFixLtHdr">  
<div id="container" onclick="MM_clearmenu()">
 <map name="MyMap" id="MyMap">
        <area shape="rect" alt="ISMB 2008" href="index-2.html" coords="8,7,168,155" />
        <area shape="rect" alt="ISCB" href="http://www.iscb.org/" coords="670,10,790,62" />
        </map>
 
  <div id="header">
  <table style="background:#0072b9" border="0" cellpadding="0" cellspacing="0">
  <tr><td colspan="6"><a name="top"><img border="0" usemap="#MyMap" src="includes/images/header/bannertop.jpg" /></a></td>
  <tr>
  <tr><td>
  <img src="includes/images/header/b1.gif" /></td>
  <td><a href="http://www.iscb.org/index.php/iscb-membership-benefits" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b2','','includes/images/header/b2_down.gif','');">
  <img src="includes/images/header/b2.gif" name="b2" border="0"  /></a></td>
  <td><a href="deadlines.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b3','','includes/images/header/b3_down.gif','');">
  <img src="includes/images/header/b3.gif" name="b3" border="0"  /></a></td>
  <td><a href="news.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b4','','includes/images/header/b4_down.gif','');">
  <img src="includes/images/header/b4.gif" name="b4" border="0"  /></a></td>
  <td><a href="registration.html" onMouseOut="MM_swapImgRestore()" onmouseover="MM_swapImage('b5','','includes/images/header/b5_down.gif','');">
  <img src="includes/images/header/b5.gif" name="b5" border="0"  /></a></td>
  <td>
  <img src="includes/images/header/b6.gif" name="b6" border="0"  /></td>
  </tr>
  </table>
  
  </div>
  <div id="sidebar1">
  <script src="includes/js/menu.js">
window.onload =  MM_clearmenu();
window.onunload = MM_clearmenu();
</script>


<script>
function custom_swap(picName,picFileName,obj,layer){
	
	var local = 'includes/images/menu/';
	
	if (picName != ""){
	MM_clearmenu();
	MM_swapImage(picName,'',local + picFileName,1);
	}
	findPos(obj);
	if(layer=="Layer1"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer2"){
	MM_showHideLayers('Layer1','','hide','Layer2','','show','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer3"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','show','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer4"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','show','Layer5','','hide','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer5"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','show','Layer6','','hide','Layer7','','hide')
	}else if(layer=="Layer6"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','show','Layer7','','hide')
	}else if(layer=="Layer7"){
	MM_showHideLayers('Layer1','','hide','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','show');
	}else if (layer=="Layer1a"){
	MM_showHideLayers('Layer1','','show','Layer2','','hide','Layer3','','hide','Layer4','','hide','Layer5','','hide','Layer6','','hide','Layer7','','hide','Layer1a','','show');
	}
}
</script>

<style>
a.smenu{
color:#EAA800;
font-family:Verdana,Arial,Helvetica,sans-serif;
/*font-style:normal;*/
font-size:10px;
}
a.smenu:link, a.smenu:visited {
color:#3366CC;
font-family:Verdana,Arial,Helvetica,sans-serif;
font-style:normal;
text-decoration:underline;
font-size:10px;
}
a.smenu:hover {
color:#EAA800;
font-family: Verdana,Arial,Helvetica,sans-serif;
font-size:10px;
}
</style>
<br><br>
      

    <a href="index-2.html" onMouseOut="MM_swapImgRestore()" onMouseOver="MM_clearmenu();MM_swapImage('home1','','includes/images/menu/down_home.gif',1);">
  <img src="includes/images/menu/up_home.gif" name="home1" border="0"  /></a><br>

    <a href="general_info.html" onMouseOut="MM_swapImgRestore();" onMouseOver="custom_swap('generalinformation1','down_general.gif',this,'Layer3')">
    <img src="includes/images/menu/up_general.gif" name="generalinformation1"  border="0" id="generalinformation1" /></a><br>
    
  <a href="program.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('program1','down_program.gif',this,'Layer7');">
    <img src="includes/images/menu/up_program.gif" alt="" name="program1" border="0" ></a><br>

   
    <a href="exhibitors_sponsors.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('sponsorsexhibitors1','down_sponsors.gif',this,'Layer4');">
    <img src="includes/images/menu/up_sponsors.gif" name="sponsorsexhibitors1"  border="0" id="sponsorsexhibitors1" /></a><br>
  
        <a href="submissions_details.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('submissionsdetails1','down_submission.gif',this,'Layer2')">
    <img src="includes/images/menu/up_submission.gif" name="submissionsdetails1"  border="0" id="submissionsdetails1" /></a><br>
    
    <a href="committee_organizing.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('committees1','down_committees.gif',this,'Layer5');">
    <img src="includes/images/menu/up_committees.gif" name="committees1"  border="0" id="committees1" /></a><br>
    
    <a  href="contacts.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('contactus1','down_contact.gif',this,'');">
    <img src="includes/images/menu/up_contact.gif" name="contactus1" border="0" id="contactus1" /></a><br>
    
    <a href="promotion.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('conferencepromotion1','down_conferencepromo.gif',this,'');">
    <img src="includes/images/menu/up_conferencepromo.gif" name="conferencepromotion1"  border="0" id="conferencepromotion1" /></a><br> 

    <a href="presspass.html" onMouseOut="MM_swapImgRestore()" onMouseOver="custom_swap('presspass1','down_presspass.gif',this,'');"> 
	    <img src="includes/images/menu/up_presspass.gif" name="presspass1" border="0" id="presspass1" /></a>
	<br />
	
<!--       <form action="/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search ISMB 2008" onfocus="value=''" name="query" id="query" size="15" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form>-->   
   <br /> <div align="center">
   <a border=0 href="scs5.html"><img align=center  border=0  src="includes/images/menu/scs5c.gif"></a>
   <br>
   <a border=0 href="sigs.html"><img align=center  border=0  src="includes/images/menu/sigsata.gif"></a>
       <br />
	   <br />
    
   <form action="http://www.iscb.org/ismbeccb2009/search/search.php?" method="get">
         <input type="text" value="Search Conference Site" onfocus="value=''" name="query" id="query" size="20" />
      <input type="submit" value="Search" />
   <input type="hidden" name="search" value="1"> 
   </form></div>
   
   </td>
  </tr>

 



<div id="Layer1" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 210px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td width="125" bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="demonstrations.html">Demonstrations</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights_track.html">Highlights Track</a></td>
          </tr>
	  

	
      </table></td>
    </tr>
  </table>
</div>


<div id="Layer7" style="position:absolute; width:auto; height:auto; z-index:1; left: 181px; top: 262px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
 <tr> 
      <td class="para"><table width="131" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="artscience.html">Art &amp; Science Exhibition</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="bof.html">Birds of a Feather</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="highlights.html">Highlights Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="keynotes.html">Keynotes</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="poster_list.html">Posters</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="proceedingstrackbytime.html">Proceedings Track</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="http://spreadsheets.google.com/pub?key=rYyCOSAzYakAaffLa03lxRg&amp;single=true&amp;gid=0&amp;output=html">Schedule</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="specialsessions.html">Special Sessions</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="socialevents.html">Social Events</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sigs.html">Satellite Meetings &amp; SIGs</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="scs5.html">Student Council Symposium</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="technologytrack.html">Technology Track</a></td></tr><tr><td width=125 bgcolor="#FFFFFF" class=para><a class=smenu href="tutorials.html">Tutorials</a></td></tr><tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="workshops.html">Workshops</a></td></tr>        </table></td>
    </tr>
  </table>
</div>


<div id="Layer2" style="position:absolute; width:auto; height:auto; z-index:2; left: 181px; top: 282px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
        

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="call_for_arts_exhibition.html">Art &amp; Science Exhibition</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="birds.html">Birds of a Feather</a></td>
          </tr>
            <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_highlights.html">Highlights Track</a></td>
          </tr>
		       <tr> 
            <td bgcolor="#FFFFFF" class="para"><a  class="smenu"  href="submit_track_latehighlights.html">Highlights Track - Late</a></td>
          </tr>
		 
  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_proceedings.html">Proceedings</a></td>
          </tr> 

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_posters.html">Posters</a></td>
          </tr>
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_lateposters.html">Posters - Late</a></td>
          </tr>
		   <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_sigs.html">Special Interest Groups (SIGs)</a></td>
          </tr>
		  
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_specialsessions.html">Special Sessions</a></td>
          </tr>
		  
		  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_scs5.html">Student Council Symposium</a></td>
          </tr>
		 
 <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submit_track_technology.html">Technology Track</a></td>
          </tr>
		    <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="travel_fellowships.html">Travel Fellowship</a></td>
          </tr> 

    	    <tr>  
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="call_for_tutorials.html">Tutorials</a></td>
          </tr>
          
        </table></td>
    </tr>
  </table>
</div>


<div id="Layer3" style="position:absolute; width:auto; height:auto; z-index:5; left: 181px; top: 200px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1" bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
 <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="airtravel.html">Air Travel</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="faq.html">FAQ</a></td></tr>
 <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="housing.html">Housing</a></td></tr>
  <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="speakerinformation.html">Speaker Information</a></td></tr>
  <tr> <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="volunteers.html">Volunteers</a></td></tr> 
        </table></td> 
    </tr>
  </table>
</div>

<div id="Layer4" style="position:absolute; width:auto; height:auto; z-index:4; top: 389px; left: 181px; overflow: auto; visibility: hidden;"> 
  <table width="135" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="133" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsors.html">Conference Sponsors</a></td>
          </tr>
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="floorplan.html">Exhibit Floor Plan</a></td></tr>
	   
	   <tr><td bgcolor="#FFFFFF" class="para"><a class="smenu" href="exhibitor_info.html">Exhibitor Info</a></td>
          </tr>
 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="past_sponsors.html">Past Sponsors</a></td>
          </tr>

          <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsoropps.html">Sponsor Opportunities</a></td>
          </tr>


 <tr>
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_packages.html">Sponsorship Benefits</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsorbrochure.html">Sponsor Brochure</a></td>
          </tr>

          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="sponsor_signup.html">Sponsor &amp; Exhibitor Sign-up</a></td>
          </tr>

        </table></td>
    </tr>
  </table>


</div>
<div id="Layer5" style="position:absolute; width:auto; height:auto; z-index:3; left: 181px; top: 467px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
	
	  <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="conference_chairs.html">Conference Chairs</a></td>
          </tr>
<tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="committee_organizing.html">Committees</a></td>
          </tr>


      
        </table></td>
    </tr>
  </table>
</div>

<div id="Layer6" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr> 
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Contacts</a></td>
          </tr>

        </table></td>
    </tr>
  </table>
</div>
<div id="Layer1a" style="position:absolute; width:auto; height:auto; z-index:6; left: 181px; top: 386px; overflow: auto; visibility: hidden;"> 
  <table width="132" border="0" cellpadding="0" cellspacing="1"  bgcolor="#EAA800">
    <tr>
      <td class="para"><table width="130" border="0" cellpadding="2" cellspacing="1"  bgcolor="#FFFFFF">
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="contacts.html">Scientific Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Institute Posters</a></td>
          </tr>
          <tr> 
            <td bgcolor="#FFFFFF" class="para"><a class="smenu" href="submission_contacts.html">Industry Posters</a></td>
          </tr>
         
        </table></td>
    </tr>
  </table>
</div>
<br />
<br />
<br />
<br />
<br />

  
  
  
  
  
  
  
  <!-- end #sidebar1 -->
    
  </div>  
  <div id="mainContent">



<br>  <br>    <span class="smhead1"><strong>Tutorials Program</strong></span>
<div class="style1">
<p>ISMB/ECCB 2009 features ten (10) half-day introductory to advanced tutorial sessions. The tutorials will be given on Sunday, June 28, 2009 one day prior to the conference scientific program. Tutorials are held on the same day as the second day of the SIG and Satellite meetings. Tutorial programs provide participants with lectures and instruction, on either well-established or new "cutting-edge" topics, relevant to the bioinformatics field. It offers participants an opportunity to learn about new areas of bioinformatics research, to get an introduction to important established topics, or to develop higher skill levels in areas in which they are already knowledgeable.<br />
  Tutorial attendees should register using the on-line registration system when registration opens on February 6, 2009.<br />
  Attendees will receive a Tutorial Entry Pass (coupon) at the time they register on site. Tutorial handouts and CD can be picked up at the door of each tutorial session in exchange for the coupon. Lunch is included in the registration fee for attendees registering for two tutorials. Those attending one tutorial only have the option to purchase a lunch ticket during on-line registration.<br />
  Tutorial participants must be registered for the conference.</p>
<p>*please note start and finish times are subject to change</p>
<p>
<div class="subheading1">
<strong>Morning Tutorials: 8:30 a.m. - 12:30 p.m.</strong>
</div>

<br />
  <a href="tutorials.html#AM1">AM 1: Text mining for non text miners: state of the art for practical use</a><br />
  <a href="tutorials.html#AM2">AM 2: From PPIs to networks and pathways</a><br />
  <a href="tutorials.html#AM3">AM 3: Support Vector Machines and Kernels for Computational Biology</a><br />
  <a href="tutorials.html#AM4">AM 4: Immunological Bioinformatics: From fundamental bioinformatics to clinical validation</a><br />
  <a href="tutorials.html#AM5">AM 5: High-throughput genome-scale sequence analysis and mapping using compressed data structures</a></p> 
  <br /><br /><hr />
<strong>Lunch Break: 12:30 p.m. - 1:30 p.m.</strong><br />
<hr />
<br />
<div class="subheading1"><strong>Afternoon Tutorials 1:30 p.m. - 5:30 p.m.</strong></div>
  <a href="tutorials.html#PM6">PM 6: Metabolic Engineering - Metabolic Reconstruction, Strain Optimization, and Flux Measurements</a><br />
  <a href="tutorials.html#PM7">PM 7: Comparative structural modeling of proteins and protein-DNA complexes: What can and what can’t be predicted</a><br />
  <a href="tutorials.html#PM8">PM 8: IBM Systems Update 2009</a> <font color=red>(Cancelled)</font><br />
  <a href="tutorials.html#PM9">PM 9: How to understand the cell by breaking it - computational analysis of single and combinatorial gene perturbations</a><br />
  <a href="tutorials.html#PM10">PM 10: Ensemble Based Statistical Inference in Genomics and Molecular Biology</a><br />
</p>
<p>
<hr />
 <span class="smhead1"><strong>Tutorial Details</strong></span>
<hr />
</p>

<div class="subheading1"><strong><a name="AM1"></a>AM 1: Text mining for non text miners: state of the art for practical use</strong></div><br />

<p><strong>Presenter(s):</strong> Raul Rodriguez-Esteban, Pfizer, Cambridge, MA, USA, <a href="mailto:Raul.Rodriguez-Esteban@pfizer.com">Raul.Rodriguez-Esteban@pfizer.com</a><br /><br />

  <strong><strong>Abstract:</strong></strong> Familiarity with the tools and methods available in text mining offers computational biologists an opportunity to develop new approaches in their area of expertise. This tutorial will offer a broad survey of practical text mining. <br /><br />

  Tutorial level: Introductory - Basic knowledge of biology and statistics is suggested - no prior knowledge of text mining needed.<br />
</p>
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />
<div class="subheading1"><strong><a name="AM2"></a>AM 2: From PPIs to networks and pathways</strong></div><br />

<p><strong>Presenter(s):</strong> Javier De Las Rivas, Consejo Superior de Investigaciones Cientificas (CSIC), Salamanca, Spain, <a href="mailto:jrivas@usal.es">jrivas@usal.es</a><br />
  Carlos Prieto, Consejo Superior de Investigaciones Cientificas (CSIC) , Salamanca, Spain<br /><br />

  <strong>Abstract:</strong> The tutorial aims to show how to explore and analyse data and information about protein-protein interactions (PPIs) to build complete or partial interactomes as networks and correlate them with specific cellular states, mapping pathways and specific biological processes. The information about the interaction between proteins is present in several major biological databases (BIND, BioGRID, DIP, HPRD, IntAct and MINT) and there are some very useful bioinformatic tools that allow adequate integrated exploration and analysis of such data (APID, APID2NET, NetworkAnalyzer, MCODE).</p>
<p>The tutorial is aimed to both biologists and bioinformaticians - a basic background in biology will be needed, but programming expertise is not required.<br />
</p>
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />

<div class="subheading1"><strong><a name="AM3"></a>AM 3: Support Vector Machines and Kernels for Computational Biology</strong></div>
<br />

<p><strong>Presenter(s):</strong> Gunnar Ratsch, Friedrich Miescher Laboratory of the Max Planck Society, Tubingen, Germany, <a href="mailto:Gunnar.Raetsch@tuebingen.mpg.de">Gunnar.Raetsch@tuebingen.mpg.de</a><br />
  Sören Sonnenburg, Fraunhofer Institute First, Berlin, Germany, Soeren.Sonnenburg@first.fraunhofer.de<br />
  Cheng Soon Ong, Max Planck Institute for Biological Cybernetics and<br />
  Friedrich Miescher Laboratory, Tübingen, Germany, <a href="mailto:chengsoon.ong@inf.eth.ch">chengsoon.ong@inf.eth.ch</a><br /><br />

  <strong>Abstract:</strong> SVMs are very popular in data mining and bioinformatics. This tutorial introduces SVMs and kernel algorithms and illustrates their application to typical problems in computational biology. It covers advances in kernels on strings and graphs, predicting structured outputs and discusses how to derive biological insight from the classifiers.<br />
</p>

 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />


<div class="subheading1"><strong><a name="AM4"></a>AM 4: Immunological Bioinformatics: From fundamental bioinformatics to clinical validation</strong></div>

<p><strong>Presenter(s):</strong> Morten Nielsen, The Technical University of Denmark, Lyngby, Denmark, <a href="mailto:mniel@cbs.dtu.dk">mniel@cbs.dtu.dk</a><br />

Claus Lundegaard, The Technical University of Denmark, Lyngby, Denmark, <a href="mailto:lunde@cbs.dtu.dk">lunde@cbs.dtu.dk</a><br />
</p>
<p>
  <strong>Abstract:</strong> A major task is to develop methods identifying epitopes in pathogen proteomes. Bioinformatics and high throughput screenings can be used to develop reliable prediction systems using artificial neural networks, Gibbs sampling, weight matrices, integrative models and protein structural analysis. Predictions are verified clinically against pathogens like Flu, Tuberculosis and HIV.</p>
<p>This tutorial is suitable for those already working with epitope data with prior knowledge of basic and statistical bioinformatics.<br />
</p>
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />

<div class="subheading1"><strong><a name="AM5"></a>AM 5: Genome-scale sequence analysis using compressed index structures</strong></div>
<br />

<p><strong>Presenter(s):</strong> Veli Mäkinen, University of Helsinki, Finland, <a href="mailto:vmakinen@cs.helsinki.fi">vmakinen@cs.helsinki.fi</a>
<br /></p>

   <p> <strong>Abstract:</strong> 
  

  The tutorial focuses on tools and techniques that make genome-scale
 sequence analysis and mapping feasible with reasonable computing
 resources. The classical algorithms exploiting generalized suffix 
 trees are reviewed and then modern data structure compression techniques are
 introduced that enable the simulation of those algorithms on
 genome-scale sequence collections.</p>
<p>
 Applications on high-throughput sequence mapping for short sequence
 reads (e.g. ChIP-seq data) are covered, including the backward 
 search backtracking mechanism on Burrows-Wheeler-based indexes, which is the
 principle behind the state-of-the-art software tools for the task.
 On genome-scale sequence analysis tasks, the tutorial covers the
 technique of mining discriminative patterns, that can be used to 
 produce good candidate "seeds" for the discovery of regulatory motifs or as a 
 tool for comparative genomics. Future applications in the storage,
 retrieval, and analysis of collections of individual genomes are 
 discussed.</p>
<p>
 In addition to the theoretical foundations, the practical 
 performance of the data structures and algorithms are demonstrated by experiments on
 genome-scale tasks.</p>
<p>
 The tutorial is suitable for those familiar with basics of sequence
 analysis. Earlier experience on compression is not necessary; there 
 will be a brief introductory part on the elementary data compression and
 information theory concepts as a prelude to the data structure
 compression realm.</p>

  
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />


<div class="subheading1"><strong><a name="PM6"></a>PM 6: Metabolic Engineering - Metabolic Reconstruction, Strain Optimization, and Flux Measurements</strong></div>
<br />

<p><strong>Presenter(s):</strong> Isabel Rocha, Portuguese Institute of Biotechnology and Bioengineering (IBB), Braga, Portugal, <a href="mailto:irocha@deb.uminho.pt">irocha@deb.uminho.pt</a><br />
Marcelinus Pont, Dupont Inc.</p>

<p><strong>Abstract:</strong> 

<!--
The material to be covered in this tutorial is considered of intermediate level and is targeted to researchers willing to apply 
Bioinformatics and Systems Biology tools to the field of Metabolic Engineering and Industrial Biotechnology. Therefore, participants 
should be familiar with mathematical tools including Dynamic Equations and Linear Algebra. Also, basic knowledge on microbial metabolism is required.<br />
-->
This tutorial is targeted to researchers willing to apply Bioinformatics and Systems Biology tools to the field of Metabolic Engineering and Industrial Biotechnology. 
The main topics to be covered are Metabolic Reconstruction and Analysis (including Metabolic Flux Analysis, Flux Balance Analysis and Elementary Flux Modes) and 

In silico Metabolic Engineering tools. Therefore, participants should be familiar with mathematical tools including Linear Algebra. Also, basic knowledge on 
microbial metabolism is required.


</p>
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />


<div class="subheading1"><strong><a name="PM7"></a>PM 7: Comparative structural modeling of proteins and protein-DNA complexes: What can and what can’t be predicted</strong></div><br />

<p><strong>Presenter(s):</strong> Jan Kosinski, International Institute of Molecular and Cell Biology, Warsaw, Poland, <a href="mailto:kosa@genesilico.pl">kosa@genesilico.pl</a></p>

<p><strong>Abstract:</strong> The increasing number of complexes with known structures allows modeling whole biological assemblies using comparative methods. This tutorial will provide detailed overview of the state-of-the-art methods for constructing comparative structural models of proteins and protein-DNA complexes. Several alternative methods will be presented for protein comparative modeling, editing DNA molecules and optimization of protein-DNA interfaces.</p>
<p>The tutorial is especially targeted to the audience with general knowledge about protein and nucleic acid structure, but little or no experience in protein structure modeling. Basic understanding of sequence similarity and sequence alignments, general knowledge about sequence and structural databases, and familiarity with structural molecular viewers will be advantageous, although not critical.<br />
</p>

 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />


<div class="subheading1"><strong><a name="PM8"></a>PM 8: IBM Systems Update 2009</strong>  <font color=red>(Cancelled)</font></div><br />
<!--
<p><strong>Presenter(s):</strong> Kirk E. Jordan, Emerging Solution Executive, Deep Computing, IBM Systems and Technology Group, USA<br />
</p>
-->
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />


<div class="subheading1"><strong><a name="PM9"></a>PM 9: How to understand the cell by breaking it - computational analysis of single and combinatorial gene perturbations</strong></div><br />

<p><strong>Presenter(s):</strong>Florian Markowetz, Cancer UK Cambridge Research Institute, Cambridge, UK, <a href="mailto:florian@genomics.princeton.edu">florian@genomics.princeton.edu</a><br />

Chad Myers, University of Minnesota, Minneapolis, USA, <a href="cmyers_cs.umn.html">cmyers@cs.umn.edu</a></p>

<p><strong><strong>Abstract:</strong></strong> 

The analysis of genome-wide single or combinatorial gene perturbation screens is moving to the center-stage of computational systems 
biology as more and better experimental systems are established in humans and model organisms. This tutorial will provide a comprehensive 
overview of recent advances both in experimental technologies and computational analysis methods. 
The tutorial will serve as a thorough introduction to gene perturbation screens and their analysis, but some advanced issues will also be introduced.

</p>
<br />
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />


<div class="subheading1"><strong><a name="PM10"></a>PM 10: Ensemble Based Statistical Inference in Genomics and Molecular Biology</strong></div>
<br />

<p><strong>Presenter(s):</strong> Charles (Chip) E. Lawrence, Brown University, Providence, RI, USA, <a href="mailto:Charles_Lawrence@brown.edu">Charles_Lawrence@brown.edu</a><br />
<br />

  <strong><strong>Abstract:</strong></strong> <br />
  In the past decade, high-throughput data-acquisition technologies have rendered datasets with sizes unimaginable to our predecessors, including the sequence of many genomes and the products of numerous high-throughput technologies of the post-genome era. Although the emergence of such large datasets seems to imply more precise parameter estimates, paradoxically just the opposite is becoming increasingly common. This paradox emerged because these technologies simultaneously opened opportunities to draw inferences on previously unanswerable high-dimensional (high-D) questions. Novel ensemble based statistical inference procedures based on statistical decision theory addressing this paradox have recently been developed. The resulting centroid and maximum expected accuracy estimators have been shown to outperform traditional mode based procedures in the prediction of RNA secondary structure, protein structure prediction by homology, and prediction of transcription factor binding sites. In all three of these cases the identical probabilistic models were used for mode based estimates and ensemble-based estimates. Thus the reported improvements stem exclusively from the novel means that these procedures employ to extract information from posterior ensembles. Also, these ensemble-based procedures provide the means to describe the shape of posterior distributions, and interestingly they have shown that these distributions are frequently multimodal.</p>
<p>Arguably the best description of the importance and novelty of these ensemble-based approaches has been presented in independent reviews. Specifically, in a review entitled “Revolutions in RNA Secondary Structure Prediction” Mathews (J. Molecular Biology. (2006) 359, 526-532) describes the ensemble based sampling approach of Ding et al. (J. of Molecular Biology, (2006) 359: 554-571) as follows. “Recent results show the power of sampling in predicting regions in an RNA that are most likely to be accessible to hybridization, in predicting secondary structures with fewer false positive base-pairs, and to understanding the folding landscape. The impact of this is enormous because, for the first time, the set of predicted secondary structures is a statistical sample of the complete ensemble of structures. The probability of sampling any given structure is exactly its probability of occurring in the thermodynamic ensemble.” In the “Faculty of 1000” Michael Zhang (July 16, 2008) reported that the paper of Carvalho and Lawrence (PNAS: USA, 105: 3209-3214, 2008) is a “must read article” and stated that “I found this article interesting because it challenges common practice in many areas of computational genomics data analyses. This centroid estimation method was first discovered to give substantial improvement in RNA structure prediction. Now the authors have provided further theoretical foundations and demonstrate its successful application to other genomics problems.”<br />
  The goal of this tutorial is to raise serious questions about the conceptual foundations of the<br />
  traditional mode based estimation procedure that now dominate our field, and to introduce students to the theory, methods, and algorithms of alternative ensemble based estimation procedures.</p>
<p>This tutorial will be at an intermediate level on the theory and methods for prediction and<br />
  estimation in computational molecular biology and computational genomics, and at a beginning level on ensemble based statistical inference procedures.</p>
</div>
<br />
 <div align="left" class="style1">
         <a href="#top" style="font-size:8px">&uarr; TOP</a>
         </div> <br />
<br />
<br />





	<!-- end #mainContent --></div>
	<!-- This clearing element should immediately follow the #mainContent div in order to force the #container div to contain all child floats --><br class="clearfloat" />
<!--  <div id="footer">
    <p>Footer</p>
     </div> -->
<!-- end #container -->
</body>

<!-- Mirrored from www.iscb.org/ismbeccb2009/tutorials.php by HTTrack Website Copier/3.x [XR&CO'2008], Tue, 07 Jul 2009 10:18:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
</html>