<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Sequence Analysis, Alignment and Next Generation Sequencing</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Sequence Analysis, Alignment and Next Generation Sequencing			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="text-transform: uppercase;" mce_style="text-transform:uppercase">Sequence
Analysis, Alignment and Next Generation Sequencing</span><span style="text-transform: uppercase;" mce_style="text-transform: uppercase;"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Geoff Barton and Des Higgins<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741004" mce_href="#_Toc269741004">Sequence Analysis, Alignment
and Next Generation Sequencing<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741004 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000300034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741005" mce_href="#_Toc269741005">A-1. Resolving the genomic
breakpoints of a microdeletion using next-generation sequencing in a family<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741005 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">4<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000300035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741006" mce_href="#_Toc269741006">A-2. Identification of unknown
environmental sequences by biased distribution of signature oligonucleotides<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741006 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000300036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741007" mce_href="#_Toc269741007">A-3. Identification of conserved
repeats with advanced background models<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741007 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000300037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741008" mce_href="#_Toc269741008">A-4. Searching for structural
homologues of LeuT-like secondary transporters beyond sequence similarity<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741008 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000300038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741009" mce_href="#_Toc269741009">A-5. Why inverse proteins are
relatively abundant<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741009 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000300039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741010" mce_href="#_Toc269741010">A-6. HERD: the highest expected
reward decoding for HMMs with application to recombination detection<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741010 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">9<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741011" mce_href="#_Toc269741011">A-7. Visualizing the next generation
of sequencing data with GenomeView<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741011 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741012" mce_href="#_Toc269741012">A-8. IsUnstruct: a method based on
Ising’s model for prediction of disordered residues from protein sequence alone<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741012 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741013" mce_href="#_Toc269741013">A-9. MASiVE: Mapping and Analysis of
SireVirus Elements in plant genome sequences<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741013 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741014" mce_href="#_Toc269741014">A-10. Signals in human promoter
sequences<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741014 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741015" mce_href="#_Toc269741015">A-11. Iterative read mapping and
assembly allows the use of more distant reference in metagenome assembly<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741015 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741016" mce_href="#_Toc269741016">A-12. APPRIS: a database of
annotated principal splice variants for the human genome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:
windowtext;display:none;mso-hide:screen;text-decoration:none;text-underline:
none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741016 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741017" mce_href="#_Toc269741017">A-13. Accurate long read mapping
using enhanced suffix arrays<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741017 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741018" mce_href="#_Toc269741018">A-14. Detection and architecture of
small heat shock protein monomers<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741018 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741019" mce_href="#_Toc269741019">A-15. Assembly of mitochondrial
genomes using next-generation sequencing data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741019 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">18<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741020" mce_href="#_Toc269741020">A-16. AnsNGS: Annotation system of
structural variation for Next Generation Sequencing data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:
windowtext;display:none;mso-hide:screen;text-decoration:none;text-underline:
none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741020 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">19<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741021" mce_href="#_Toc269741021">A-17. Assessing the accuracy and
completeness of the bonobo genome sequence<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741021 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">20<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741022" mce_href="#_Toc269741022">A-18. Estimating effective DNA
database size via compression<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741022 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">21<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741023" mce_href="#_Toc269741023">A-19. ChIP-seq analysis: from peaks
to motifs<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741023 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">22<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741024" mce_href="#_Toc269741024">A-20. Applying mass spectrometry
data to improve annotation of the reference mouse genome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:
windowtext;display:none;mso-hide:screen;text-decoration:none;text-underline:
none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741024 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">23<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741025" mce_href="#_Toc269741025">A-21. Prediction of splice-modifying
SNPs in human genes using a new analysis pipeline called AASsites<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741025 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">24<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741026" mce_href="#_Toc269741026">A-22. Structator: a tool for fast
index-based matching of RNA structural motifs<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF _Toc269741026
\h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">25<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741027" mce_href="#_Toc269741027">A-23. Project HOPE: Providing the
last piece of the puzzle...<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741027 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">26<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741028" mce_href="#_Toc269741028">A-24. Simple not-PWM-based approach
for transcription factor binding site prediction<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741028 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">27<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741029" mce_href="#_Toc269741029">A-25. PICMI: mapping point
variations on genomes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741029 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">28<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000320039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741030" mce_href="#_Toc269741030">A-26. Detection of alternative
splice isoforms in the human genome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741030 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">29<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741031" mce_href="#_Toc269741031">A-27. Boundaries for short-read,
low-coverage de novo assembly and resequencing<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741031 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">30<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741032" mce_href="#_Toc269741032">A-28. TAPIR: a web server for the
prediction of plant microRNA targets, including target mimics<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741032 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">31<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741033" mce_href="#_Toc269741033">A-29. GRID/CPCA analysis of the
SULTs superfamily: multivariate characterization of the most relevant
structural and physicochemical differences<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741033 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">32<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741034" mce_href="#_Toc269741034">A-30. Theoretical and empirical
quality assessment of transcription factor binding motifs<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741034 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">33<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741035" mce_href="#_Toc269741035">A-31. Characterization and
prediction of protein nucleolar localization sequences<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741035 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">35<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741036" mce_href="#_Toc269741036">A-32. Jalview: past, present, future<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741036 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">36<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741037" mce_href="#_Toc269741037">A-33. Enrichment of eukaryotic
linear motifs in viral proteins<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741037 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">37<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741038" mce_href="#_Toc269741038">A-34. Analyzing ChIP-Seq data using
qips<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741038 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">38<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741039" mce_href="#_Toc269741039">A-35. COMA server for protein
distant homology search<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span> PAGEREF _Toc269741039 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">39<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000330039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741040" mce_href="#_Toc269741040">A-36. Protein disorder and short
motifs in disordered regions are enriched near the cytoplasmic side of
single-pass transmembrane proteins<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741040 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">40<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741041" mce_href="#_Toc269741041">A-37. Variation in positional
preferences by 20 amino acids in alpha helices with different size<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741041 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">41<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741042" mce_href="#_Toc269741042">A-38. Comparison of mapping and
variant calling accuracies for next-generation sequence data in relation to
coverage depth: a case study in leukemia cell lines<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741042 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">42<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741043" mce_href="#_Toc269741043">A-39. Monitoring strain diversity in
metagenomics data using meta-MLST SNP profiling<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741043 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">43<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741044" mce_href="#_Toc269741044">A-40. A variance stabilizing
parametrization to detect differentially expressed genes in RNA-Seq data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741044 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">44<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741045" mce_href="#_Toc269741045">A-41. DNA annotation induction: from
RefGene on human chr. 22 to genome-wide CAGE for human and mouse<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741045 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">45<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741046" mce_href="#_Toc269741046">A-42. A new resampling method<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741046 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">46<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741047" mce_href="#_Toc269741047">A-43. Clustering massive sequence
databases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741047 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">47<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741048" mce_href="#_Toc269741048">A-44. WordCluster: detecting
clusters of DNA words and genomic elements<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741048 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">48<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741049" mce_href="#_Toc269741049">A-45. Analysis of copy loss and gain
variations in Holstein cattle autosomes using BeadChip SNPs<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741049 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">49<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000340039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741050" mce_href="#_Toc269741050">A-46. The role of the Occludin
MARVEL domain in intracellular targeting and clustering<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741050 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">50<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741051" mce_href="#_Toc269741051">A-47. Multi-Harmony: detecting
functional specificity from sequence alignment<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741051 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">51<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741052" mce_href="#_Toc269741052">A-48. Fast approximate statistical
ranking for sequence motif discovery under higher order Markov background model<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741052 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">52<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741053" mce_href="#_Toc269741053">A-49. Networking interacting
residues, evolutionary conservation and energetics in protein structures<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741053 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">53<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741054" mce_href="#_Toc269741054">A-50. An iterative scheme for making
long structural alignments of ncRNAs using FOLDALIGN<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">&nbsp;&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741054 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">54<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741055" mce_href="#_Toc269741055">A-51. A multi-objective approach to
the prediction of sRNAs in bacteria<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741055 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">55<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741056" mce_href="#_Toc269741056">A-52. Scalability of large-scale
protein domain family inference<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741056 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">56<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741057" mce_href="#_Toc269741057">A-53. Evolutionary study of
eye-developmental gene expression across multiple Drosophila species by
high-throughput tag sequencing<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741057 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">57<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741058" mce_href="#_Toc269741058">A-54. Higher order repeat with
largest primary repeat unit (~2.4 kb) and located within a gene in human genome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741058 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">58<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741059" mce_href="#_Toc269741059">A-55. Fast and accurate digging for
binding motifs in ChIP-Seq data using ChIPMunk software<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741059 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">59<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000350039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741060" mce_href="#_Toc269741060">A-56. Mebitoo: a sequence analysis
toolbox framework<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741060 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">60<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741061" mce_href="#_Toc269741061">A-57. Practical aspects of RNA-Seq
data analysis that you should know and they don't tell you<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741061 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">61<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741062" mce_href="#_Toc269741062">A-58. MutaBase: a framework and
web-interface for the archival, management and interpretation of single
nucleotide variants obtained by next generation sequencing<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741062 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">62<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741063" mce_href="#_Toc269741063">A-59. Mining cleavage sites of the
mouse peptidome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741063 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">63<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741064" mce_href="#_Toc269741064">A-60. Properties of plant microRNAs
affect miRDeep statistical scoring as well as accuracy<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741064 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">64<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741065" mce_href="#_Toc269741065">A-61. Using the Amazon elastic cloud
computing resource for the analysis of next-generation sequencing data in
Ensembl<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741065 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">65<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#_Toc269741066" mce_href="#_Toc269741066">A-62. Automated sequence extraction
of relevant genomic features for targeted resequencing<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269741066 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">66<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340031003000360036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741005">A-1. Resolving the
genomic breakpoints of a microdeletion using next-generation sequencing in a
family</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">de Ligt J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;de Ligt J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Vissers LELM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vissers LELM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kloosterman W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kloosterman W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Hehir-Kwa JY<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hehir-Kwa JY&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bruijn E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bruijn E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Gilissen C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gilissen C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Guryev V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Guryev V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Cuppen E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cuppen E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3) , Brunner HG<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brunner HG&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Veltman JA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Veltman JA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Some genomic regions are particularly difficult to map
and analyse due to repeat rich content, inversion prone regions and
polynucleotide stretches. One such region has a commonly occurring alternative
locus which is located on chromosome 17q21 and is associated to different
disorders. The common haplotype is H1, the H2 haplotype occurs in approximately
20% of the Caucasian population and is characterized by an 900kb inversion. The
H2 haplotype predisposes to a microdeletion that causes a frequent mental
retardation syndrome.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this study different strategies are used to fine
map the breakpoints of a deletion within the complex region of 17q21. We use a
combination of genomic microarrays and Next Generation Sequencing (NGS)
strategies; mate-pair sequencing and coverage depth comparison. Data from one
pedigree was analysed; parents (? homozygous H2, ? homozygous H1), child
(heterozygous H1/H2) with a deletion within the inversion region of H2 which
resulted in the above mentioned syndrome. NGS should be able to provide the
data to pinpoint the exact position and orientation of the inversion and the
deletion.</p>

<h3>Results</h3>

<p class="abresults">While microarray data provided a breakpoint in the patient,
the mate-pair data was inconclusive in detecting these breakpoints for both the
inversion and the deletion. The coverage values did show a deletion region when
compared with a general reference, but the signal was too noisy to determine
the breakpoints. Pedigree data provided a more reliable reference sequence and
enabled better coverage comparison, which allowed us to capture the elusive
breakpoints of the causative deletion. NGS predicted a more proximal start
position (~40kb apart), while the range for the distal breakpoint is 2kb.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The identification of the deletion breakpoints would not
have been possible if the currently available reference genome would have been
used. We conclude that many structural variants in variable or complex genomic
regions will go un-noticed in high throughput approaches due to the fact that
mapping errors will cause too much noise or result in un-mapped regions.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Joep de Ligt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65681 = 'j.l&#105;gt' + '&#64;';
 addy65681 = addy65681 + '&#97;ntrg' + '&#46;' + '&#117;mcn' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy65681 + suffix + '\'' + attribs + '>' );
 document.write( addy65681 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18333 = 'j.l&#105;gt' + '&#64;';
 addy18333 = addy18333 + '&#97;ntrg' + '&#46;' + '&#117;mcn' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy18333 + suffix + '\'' + attribs + '>' );
 document.write( addy18333 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67362 = 'j.l&#105;gt' + '&#64;';
 addy67362 = addy67362 + '&#97;ntrg' + '&#46;' + '&#117;mcn' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy67362 + suffix + '\'' + attribs + '>' );
 document.write( addy67362 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Human Genetics UMC St Radboud,
Nijmegen, The Netherlands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Human Genetics UMC St Radboud,
Nijmegen, The Netherlands 2 Department of Medical Genetics, UMC Utrecht, The
Netherlands 3 Hubrecht Institute for Developmental Biology and Stem Cell
Research, Utrecht, The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abfund">TechGene</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741006">A-2. Identification of
unknown environmental sequences by biased distribution of signature
oligonucleotides</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Labuschange P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Labuschange P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Emmett W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Emmett W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Davenport CF<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Davenport CF&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Reva O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reva O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reva ON&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">The next generation sequencing technologies make it
possible to sequence all the DNA in given samples, but data mining remains a
challenge. While k-mer classification of unknown sequences is widely discussed
in the literature, the sensitivity of the approaches based on short k-mers is
still low. Further, the analysis of frequencies of longer words is computer
intensive. Frequencies of 8 to 14-mer oligonucleotides are loaded with
significant taxonomic information, but there is a lack of computer algorithms
to exploit this information in large-scale studies.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The distribution of oligonucleotides was analysed
in DNA sequences of bacterial chromosomes available in the NCBI database. A
template of 172,636 signature words was created. Each oligonucleotide and its
reverse complement were considered as the same word so that the two different
stands of the DNA molecule will be assigned identical scores. To keep the size
of the database small enough for easy use, the numeric frequencies of word
occurrence were replaced by percentiles. For case studies several metagenome
datasets were simulated by MetaSim and clustered by LikelyBin.</p>

<h3>Results</h3>

<p class="abresults">Here we present new genome linguistics algorithms and a
portable program with a database of signature oligonucleotides. Frequencies of
signature oligonucleotides were calculated in 768 bacterial chromosomes and
this database can be downloaded from the project web-site. The program includes
several algorithms that help one to find the words that discriminate best
between certain bacterial genomes or taxonomic units. Once such words have been
found, another algorithm implemented in the program allows one to make
inferences about the phylogeny of unknown DNA sequences.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The case studies showed how the program and signature
words perform in practice; however, the quality depends on sequence length. Our
current work focuses on the development of a Web-based interface that will make
it possible to perform a semi-automated identification in several hierarchical
steps. First, a cluster of reads will be associated with a high level taxonomic
groups sharing compositional sequence similarity based on pre-selected
signature words. Then, the cluster will be assigned to a smaller group of
organisms until the confidence in the assignment drop below a certain threshold.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.bi.up.ac.za/SeqWord/oligodb/index.html" mce_href="http://www.bi.up.ac.za/SeqWord/oligodb/index.html">http://www.bi.up.ac.za/SeqWord/oligodb/index.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Oleg N. Reva (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8904 = '&#111;l&#101;g.r&#101;v&#97;' + '&#64;';
 addy8904 = addy8904 + '&#117;p' + '&#46;' + '&#97;c' + '&#46;' + 'z&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy8904 + suffix + '\'' + attribs + '>' );
 document.write( addy8904 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24936 = '&#111;l&#101;g.r&#101;v&#97;' + '&#64;';
 addy24936 = addy24936 + '&#117;p' + '&#46;' + '&#97;c' + '&#46;' + 'z&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy24936 + suffix + '\'' + attribs + '>' );
 document.write( addy24936 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44362 = '&#111;l&#101;g.r&#101;v&#97;' + '&#64;';
 addy44362 = addy44362 + '&#117;p' + '&#46;' + '&#97;c' + '&#46;' + 'z&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy44362 + suffix + '\'' + attribs + '>' );
 document.write( addy44362 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Pretoria</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics and Computational Biology Unit,
Department of Biochemistry, University of Pretoria,Pretoria 0002, Republic of
South Africa; (2) Klinische Forschergruppe, OE 6711, Hannover Medical School,
Carl-Neuberg-Strasse 1, D-30625 Hanover, Germany</p>

<h3>Acknowledgements</h3>

<p class="abfund">Funding for this research was provided by the National Research
Foundation of South Africa (NRF) grant for the National Bioinformatics and
Functional Genomics Programme.</p>

</div><p>

<b><span style="font-size: 13pt; line-height: 115%;" mce_style="font-size:13.0pt;line-height:115%;font-family:" ,="" serif="" ;="" mso-fareast-font-family:="" times="" new="" roman="" ;color:black;mso-ansi-language:en-us;="" mso-fareast-language:en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before: always;" clear="all">
</span></b></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741007">A-3. Identification of
conserved repeats with advanced background models</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Labaj P-P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Labaj P-P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sykacek P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sykacek P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kreil D-P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kreil D-P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The functional interpretation of sequencing data remains
a key challenge, especially the identification of biologically relevant patterns
against a backdrop of functionally meaningless variation. Considering the
unexpected abundance of single amino acid repeats, a detection of repeats with
potential biological functions is of interest. As their occurrence cannot be
assessed by established models, an exploration of more complex models is
necessary. We demonstrate the efficacy of an application specific background
model for the identification of novel conserved sequence patterns.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">An empirical model was developed for the study of
repeat abundance in signal peptides compared to mature proteins. It was trained
on a comprehensive unbiased reference sample from UniProt. Input parameters
comprised protein length, amino acid composition, and repeat length. A
two-stage approach was found to be efficient. First, a logistic regression is
used to model the probability that a given protein has at least one repeat.
Then a relevance vector machine predicts the conditionally expected number of repeats.</p>

<h3>Results</h3>

<p class="abresults">Considering that the abundance of single amino acid repeats
cannot be assessed by standard models (including higher order HMMs) we have
developed an application specific background model for their analysis. In a
survey of these repeats in signal peptides of eukaryotes, a significant
enrichment was only detected for leucine repeats. It explains their unexpected
abundance in general. We could then show that these repeats were conserved in
tetrapods, indicating a functional role. This was particularly clear in
mammals.</p>

<h3>Discussion</h3>

<p class="abdiscussion">While standard models perform well for globular protein
domains, they are known to break down in regions of stronger compositional bias
or low complexity. As demonstrated, to meaningfully study such regions,
application specific background models may be required. As repeats in general
seem not to be conserved but rather tolerated as neutral, these features are
filtered and excluded in regular sequence analysis such as similarity searches.
We identify leucine repeats in signal peptides that are more strongly conserved
than their host sequence, indicating novel functional roles.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Pawel P. Labaj (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11833 = '&#101;ccb2010' + '&#64;';
 addy11833 = addy11833 + 'b&#111;k&#117;' + '&#46;' + '&#97;c' + '&#46;' + '&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy11833 + suffix + '\'' + attribs + '>' );
 document.write( addy11833 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62843 = '&#101;ccb2010' + '&#64;';
 addy62843 = addy62843 + 'b&#111;k&#117;' + '&#46;' + '&#97;c' + '&#46;' + '&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy62843 + suffix + '\'' + attribs + '>' );
 document.write( addy62843 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4303 = '&#101;ccb2010' + '&#64;';
 addy4303 = addy4303 + 'b&#111;k&#117;' + '&#46;' + '&#97;c' + '&#46;' + '&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy4303 + suffix + '\'' + attribs + '>' );
 document.write( addy4303 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Chair of Bioinformatics, BOKU University Vienna, Austria</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Chair of Bioinformatics, BOKU University Vienna,
Austria</p>

<h3>Acknowledgements</h3>

<p class="abfund">This work was supported by the Vienna Science and Technology
Fund (WWTF), Baxter AG, Austrian Research Centres Seibersdorf, and the Austrian
Centre of Biopharmaceutical Technology.</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741008">A-4. Searching for
structural homologues of LeuT-like secondary transporters beyond sequence
similarity</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Khafizov K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Khafizov K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Staritzbichler R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Staritzbichler R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Stamm M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Stamm M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Forrest LR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Forrest LR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">X-ray structures have recently revealed that numerous
secondary transporter proteins belonging to different sequence families
unexpectedly share the so-called LeuT fold. The core of this fold consists of
two units of five transmembrane helices (TM). That these two units share
similar structures implies that LeuT-like transporters arose from gene
duplication and fusion events. The evolutionary origins of this fold may be
relevant to their function. However, the repeat units have no sequence
similarity, so very sensitive methods are required to search for relationships
between such 5TM proteins.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed AlignMe, which can use various types
of input information to perform pair-wise alignments of (membrane) protein
sequences. Aside from conventional similarity matrices, AlignMe can also use
specific amino-acid properties (e.g., hydrophobicity); predictions of secondary
structure and/or membrane-spanning segments; as well as combinations of these
data. We used AlignMe to search for 5TM homologues of LeuT-fold transporters by
screening hydropathy profiles of 5TM repeats from known structures against
hydropathy profiles constructed for a large dataset of membrane protein
sequences.</p>

<h3>Results</h3>

<p class="abresults">We first demonstrate that AlignMe hydropathy profile
alignments accurately match up the TM segments of LeuT-like secondary
transporter proteins, which share similar structures but lack significant
sequence similarity. We then report the identification of 5TM proteins that possess
similar hydropathy profiles to 5TM repeats of LeuT-like transporters and,
therefore, may share a common ancestor to the 5TM repeat units in the LeuT
fold.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The comparison of the LeuT-fold transporters shows that
AlignMe is a useful tool for alignment of membrane proteins that are remote
homologues. By identification of 5TM proteins that may share a common ancestor
with the LeuT-fold transporters using AlignMe we hope to provide some insight
into the evolution of these remarkable structural features, whose symmetry is
thought to allow the formation of two alternate states during the alternating
access mechanism of secondary transport.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.forrestlab.org/alignme" mce_href="http://www.forrestlab.org/alignme">http://www.forrestlab.org/alignme</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Kamil Khafizov (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88040 = 'k&#97;m&#105;l.kh&#97;f&#105;z&#111;v' + '&#64;';
 addy88040 = addy88040 + 'b&#105;&#111;phys' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy88040 + suffix + '\'' + attribs + '>' );
 document.write( addy88040 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54670 = 'k&#97;m&#105;l.kh&#97;f&#105;z&#111;v' + '&#64;';
 addy54670 = addy54670 + 'b&#105;&#111;phys' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy54670 + suffix + '\'' + attribs + '>' );
 document.write( addy54670 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34087 = 'k&#97;m&#105;l.kh&#97;f&#105;z&#111;v' + '&#64;';
 addy34087 = addy34087 + 'b&#105;&#111;phys' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy34087 + suffix + '\'' + attribs + '>' );
 document.write( addy34087 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Max Planck Institute of Biophysics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Computational Structural Biology group, Max Planck
Institute of Biophysics, 60438 Frankfurt am Main, Germany</p>

<h3>Acknowledgements</h3>

<p class="abfund">SFB807</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741009">A-5. Why inverse
proteins are relatively abundant</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nebel J-C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nebel J-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*</p>

<p class="abmotivation">Studies of peptide chains created by inversing protein
sequences have established there is no correlation between the structures of a
protein and its inverse. However, inverse sequences are more common in nature
than one would expect by chance. They must display some characteristics that
are not present in random sequences. However, to date, the relative abundance
of inverse proteins remains unexplained. Here, we investigate the proposition
that inverse peptide chains are more common than random ones mainly because
they display periodicity and repeat patterns present in protein sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We designed a new artificial peptide dataset, i.e.
‘opprotein’, sharing repeat patterns with reference proteins, but displaying
very different amino acid (aa) environments. An ‘opprotein’ is the peptide
chain which is the most unlike a given protein sequence. More specifically, the
opposite of a protein sequence is defined as the aa sequence where each aa is
replaced by an aa with the most opposite physico-chemical properties. 3
artificial datasets, i.e. inverse, opprotein and random, containing 5489
sequences each were analysed using BLAST to establish similarity to existing
proteins.</p>

<h3>Results</h3>

<p class="abresults">Inverse, opprotein and random sets had, respectively, 16, 5
and 0 significant hits. Consequently, opproteins share some properties with
inverse and real sequences, which are not found in random sequences.
Remarkably, 4 protein sequences produced both inverse and opprotein hits. In
most cases (15), matching proteins display repeats or repeating structural
elements, i.e. beta-barrel, beta propeller and duplicated folds.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our study suggests that repeats are the main contributor
of the abundance of inverse proteins. However, since inverse proteins are more
common than opproteins that share the same repeat patterns, other factors are
involved. Since, unlike opproteins, inverse peptide sequences have the same
residue propensity as known proteins, amino acid distribution must also play a
part in the similarity between a peptide chain and a protein. A consequence of
this work is that the use of inverse sequences as a negative set in experiments
should be done with caution as they cannot be considered as random.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://staffnet.kingston.ac.uk/%7Eku33185/ProteinSequenceConverter/OpProtein.html" mce_href="http://staffnet.kingston.ac.uk/%7Eku33185/ProteinSequenceConverter/OpProtein.html">http://staffnet.kingston.ac.uk/~ku33185/ProteinSequenceConverter/OpProtein.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jean-Christophe Nebel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4583 = 'j.n&#101;b&#101;l' + '&#64;';
 addy4583 = addy4583 + 'k&#105;ngst&#111;n' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy4583 + suffix + '\'' + attribs + '>' );
 document.write( addy4583 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83544 = 'j.n&#101;b&#101;l' + '&#64;';
 addy83544 = addy83544 + 'k&#105;ngst&#111;n' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy83544 + suffix + '\'' + attribs + '>' );
 document.write( addy83544 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43198 = 'j.n&#101;b&#101;l' + '&#64;';
 addy43198 = addy43198 + 'k&#105;ngst&#111;n' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy43198 + suffix + '\'' + attribs + '>' );
 document.write( addy43198 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Kingston University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Kingston University, London</p>

<h3>Acknowledgements</h3>

<p class="abfund">Why inverse proteins are relatively abundant, J.-C. Nebel and
C. Walawage, Protein &amp; Peptide Letters, 17(7), 2010</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741010">A-6. HERD: the highest
expected reward decoding for HMMs with application to recombination detection</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nánási M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nánási M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Vinar T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vinar T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Brejová B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brejová B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Hidden Markov models (HMMs) are an important tool for
modeling biological sequences and their annotations. By sequence annotation we
mean assignment of labels to each symbol according to its function. The Viterbi
algorithm, commonly used for HMM decoding, finds the most probable annotation
some HMMs. In general, the sequence annotation is NP-hard (Brejová et al. 2007)
and the Viterbi algorithm is used as a heuristic. Recently it has been shown
that other decoding methods lead to more accurate results in specific
applications (Kall et al 2005, Gross et al. 2007, Brown and Truszkowski 2010).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We propose a new efficient HMM decoding algorithm
called the highest expected reward decoding (HERD). It is appropriate, when we
want to partition sequence into features, but allow some tolerance in the
placement of each feature boundary. We define our objective function in the
terminology of gain functions (Hamada et al. 2009), where the gain
characterizes the similarity between a predicted and the correct annotation.
Then we seek the annotation with the highest expected gain where the
expectation is taken over all annotations in the HMM, since we do not know the
true annotation.</p>

<h3>Results</h3>

<p class="abresults">We evaluate our approach on the problem of detecting
recombination in the HIV genome and compare it with an existing tool called
jumping HMM which uses the Viterbi algorithm (Schultz 2009). On artificial
recombinants created from real HIV sequences, HERD has significantly higher
feature specificity and sensitivity than the Viterbi algorithm using the same
HMM. Here, a feature is a predicted block between two adjacent recombination
points, and we count it as correctly predicted if its boundaries are misplaced
by at most 10 symbols.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results show that the HERD predicts viral
recombination with higher accuracy than the Viterbi algorithm. Here, the
Viterbi algorithm finds the highest scoring alignment of a query to a profile
HMM. In contrast, we marginalize over all alignments and over nearby placements
of recombination points. Our algorithm can be easily extended to other similar
gain functions, and therefore it can be used in other application domains. Our
method has several parameters that have significant impact on the prediction
accuracy. It remains an open question how to systematically choose their
values.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.compbio.fmph.uniba.sk/herd/" mce_href="http://www.compbio.fmph.uniba.sk/herd/">http://www.compbio.fmph.uniba.sk/herd/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Michal Nánási (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99687 = 'm&#105;c' + '&#64;';
 addy99687 = addy99687 + '&#101;ccb' + '&#46;' + 'ksp' + '&#46;' + 'sk';
 document.write( '<a ' + path + '\'' + prefix + addy99687 + suffix + '\'' + attribs + '>' );
 document.write( addy99687 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91382 = 'm&#105;c' + '&#64;';
 addy91382 = addy91382 + '&#101;ccb' + '&#46;' + 'ksp' + '&#46;' + 'sk';
 document.write( '<a ' + path + '\'' + prefix + addy91382 + suffix + '\'' + attribs + '>' );
 document.write( addy91382 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83691 = 'm&#105;c' + '&#64;';
 addy83691 = addy83691 + '&#101;ccb' + '&#46;' + 'ksp' + '&#46;' + 'sk';
 document.write( '<a ' + path + '\'' + prefix + addy83691 + suffix + '\'' + attribs + '>' );
 document.write( addy83691 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Comenius University in Bratislava</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Faculty of Mathematics, Physics and Informatics,
Comenius University in Bratislava, Slovakia</p>

<h3>Acknowledgements</h3>

<p class="abfund">This research is funded by European Community FP7 grants
IRG-224885 and IRG-231025 and VEGA grant 1/0210/10.</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741011">A-7. Visualizing the
next generation of sequencing data with GenomeView</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Abeel T<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Abeel T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van Parys T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Parys T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Galagan J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Galagan J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">High-throughput sequencing instruments are widely
available. As costs decrease and volumes increase, sequencing is rapidly moving
to become a general purpose assay as a read-out for a specific biological
question. Thus, applications including RNA-seq, Chip-Seq and SNP discovery have
become common. Moreover, as sequencing a single genome has become more
affordable, it is no longer unusual to have sequences for several closely
related genomes. Yet, although it is relatively easy to acquire HTS data, there
is still a critical need for tools that allow scientists to effectively use
these data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Data types that can be visualized include reference
sequences, annotation, multiple alignments, high-throughput sequencing mappings
and synteny. Multiple file format are supported for each data type and we cover
most of the common file formats. Furthermore it is very easy to link GenomeView
with any existing data sources,because GenomeView supports a broad range of
widely used data formats. Integrating our tool with a website consists of two
steps: (i) put the data on a webserver, (ii) make a URL pointing to GenomeView
and the data. Creating this URL is easy and documented in the manual.</p>

<h3>Results</h3>

<p class="abresults">To address this need for interactive and dynamic
visualization of gigantic datasets coming from HTS, we have developed a
user-friendly and intuitive tool, called GenomeView. GenomeView can handle a
broad range of the emerging information that results from NGS. In particular,
GenomeView is the only tool currently available that allows biologists to
rapidly and interactively browse, edit, and analyze genome wide datasets of
mapped short read sequences and comparative data. Website:
http://genomeview.org/ (free download, tutorials, manual, screencasts, etc.)</p>

<h3>Discussion</h3>

<p class="abdiscussion">GenomeView provides biologists with a unique tool to
quickly explore their data in an intuitive and visually appealing way. It has
value throughout any genomics project. At the start of a project it can act as
a portal to get an overview of the data and to generate hypotheses. Later on,
it provides valuable visual sanity checks on any preliminary results. And
finally, it is also valuable at the end of the project, to present the results
and data in an appealing way to the scientific community.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://genomeview.org" mce_href="http://genomeview.org">http://genomeview.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Thomas A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thomas A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->beel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64582 = 'th&#111;m&#97;s' + '&#64;';
 addy64582 = addy64582 + '&#97;b&#101;&#101;l' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy64582 + suffix + '\'' + attribs + '>' );
 document.write( addy64582 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62112 = 'th&#111;m&#97;s' + '&#64;';
 addy62112 = addy62112 + '&#97;b&#101;&#101;l' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy62112 + suffix + '\'' + attribs + '>' );
 document.write( addy62112 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28766 = 'th&#111;m&#97;s' + '&#64;';
 addy28766 = addy28766 + '&#97;b&#101;&#101;l' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy28766 + suffix + '\'' + attribs + '>' );
 document.write( addy28766 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB-UGent</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) VIB Department of Plant systems biology, Ghent
University, Gent, Belgium (2) Broad Institute of MIT and Harvard, Cambridge,
MA, USA</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741012">A-8. IsUnstruct: a
method based on Ising’s model for prediction of disordered residues from
protein sequence alone</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lobanov MY<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Lobanov MY&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Galzitskaya OV<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Galzitskaya OV&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Intrinsically disordered regions serve as molecular
recognition elements, play an important role in the control of many cellular
processes and signaling pathways. It is useful to be able to predict positions
of disordered residues and disordered regions in protein chains.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this work, a statistical analysis of disordered
residues was made by considering 28727 unique protein chains taken from the PDB
database. In this database, 4.65% of residues are disor-dered. The statistics
was obtained separately for the N- and C-termini as well as for the central
part of the protein chain. The optimal parameters have been sorted out for
predictions using prepared databases.</p>

<h3>Results</h3>

<p class="abresults">A new method based on the dynamic programming was developed
for searching not only disordered regions but also individual disordered
residues in protein chain. This method correctly finds 77% of disordered
residues as well as 87% of ordered residues in the CASP8 database.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Comparison of our method with other methods has shown
that our method is one of the best.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://antares.protres.ru/IsUnstruct/" mce_href="http://antares.protres.ru/IsUnstruct/">http://antares.protres.ru/IsUnstruct/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Oxana V. Galzitskaya (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89172 = '&#111;g&#97;lz&#105;t' + '&#64;';
 addy89172 = addy89172 + 'v&#101;g&#97;' + '&#46;' + 'pr&#111;tr&#101;s' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy89172 + suffix + '\'' + attribs + '>' );
 document.write( addy89172 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13321 = '&#111;g&#97;lz&#105;t' + '&#64;';
 addy13321 = addy13321 + 'v&#101;g&#97;' + '&#46;' + 'pr&#111;tr&#101;s' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy13321 + suffix + '\'' + attribs + '>' );
 document.write( addy13321 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34391 = '&#111;g&#97;lz&#105;t' + '&#64;';
 addy34391 = addy34391 + 'v&#101;g&#97;' + '&#46;' + 'pr&#111;tr&#101;s' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy34391 + suffix + '\'' + attribs + '>' );
 document.write( addy34391 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of protein research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Protein Research, Russian Academy of
Sciences</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Russian Academy of
Sciences (“Molecular and Cell Biology” and "Fundamental Sci-ence to
Medicine" programs), by the Russian Foundation for Basic Research
(08-04-00561). We are grateful to Prof. J.L. Sussman for providing us the
materials from CASP8. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741013">A-9. MASiVE: Mapping and
Analysis of SireVirus Elements in plant genome sequences</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Darzentas N<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Darzentas N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Bousios A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bousios A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Apostolidou V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Apostolidou V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tsaftaris AS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tsaftaris AS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">The availability of large plant genome sequences has
necessitated the development of tools for high throughput discovery and
annotation of transposable elements (TE), which comprise the majority of these
genomes. Such algorithms are available, however they are based on general TE
structural characteristics that lead to misannotations or incomplete
identification. Our discovery that the abundant (20% of the maize genome)
Sireviruses contain highly conserved motifs in their genome, provided the
opportunity to develop MASiVE, a tool able to identify with high precision
intact Sirevirus elements.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">MASiVE is written in Perl and makes use of external
algorithms (LTRharvest, Vmatch and GeneWise) and of the highly conserved
Sirevirus multiple polypurine tract (PPT) signature and primer binding site
(PBS) to build a pipeline of filters applied sequentially. Initially,
LTRharvest scans the genomic sequence for LTR retrotransposons (LTR-RTN) and
only elements that overlap with a multiple PPT signature are retained. Next, a
cascade of distance requirements, LTR-RTN specific gene hits and within-element
homology-based searches result in the accurate annotation of intact Sirevirus
elements.</p>

<h3>Results</h3>

<p class="abresults">We demonstrate the specificity of the multiple PPT signature
and its capacity to be used by MASiVE. The stepwise application of the filters
rejected 2086 elements from a total of 3870 LTRharvest-predicted Sireviruses
that overlapped with a multiple PPT signature. The final set totals 1784 high
quality intact Sireviruses in maize chromosome one. Comparison with the
annotation from the Maize Transposable Element Consortium (MTEC) shows that
MASiVE not only recovered 76% of the MTEC set, but also added another 50%
intact elements to the pool.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Based on the highly conserved genome structure of
Sireviruses, we developed MASiVE, a novel method able to discover with high
sensitivity and precision intact Sireviruses in plant genomic sequences. MASiVE
also provides detailed information on elements that failed the filters, thus
expanding the view on the genome distribution of Sireviruses. Since Sireviruses
are among the most abundant LTR-RTN genera of plant genomes, we believe MASiVE
to be a valuable addition to the toolbox of scientists trying to untangle the
intriguingly complex genomic landscape of plants.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Nikos Darzentas (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90399 = 'nd&#97;rz' + '&#64;';
 addy90399 = addy90399 + 'c&#101;rth' + '&#46;' + 'gr';
 document.write( '<a ' + path + '\'' + prefix + addy90399 + suffix + '\'' + attribs + '>' );
 document.write( addy90399 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11346 = 'nd&#97;rz' + '&#64;';
 addy11346 = addy11346 + 'c&#101;rth' + '&#46;' + 'gr';
 document.write( '<a ' + path + '\'' + prefix + addy11346 + suffix + '\'' + attribs + '>' );
 document.write( addy11346 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51655 = 'nd&#97;rz' + '&#64;';
 addy51655 = addy51655 + 'c&#101;rth' + '&#46;' + 'gr';
 document.write( '<a ' + path + '\'' + prefix + addy51655 + suffix + '\'' + attribs + '>' );
 document.write( addy51655 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Agrobiotechnology, Centre for Research
and Technology Hellas (CERTH)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Institute of Agrobiotechnology, Centre for Research
and Technology Hellas (CERTH), Thessaloniki, Greece 2 Department of Genetics
and Plant Breeding, Aristotle University of Thessaloniki (AUTh), Thessaloniki,
Greece</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the FP6 Biosapiens Network
of Excellence (contract number LSHG-CT-2003-503265). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741014">A-10. Signals in human
promoter sequences</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Putta P<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Putta P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*</p>

<p class="abmotivation">It is believed that there are no universal or conserved
core promoter sequence in eukaryotes as that of clear signals like TATAAT and
TTGACA around -10 and -35 from TSS in prokaryotes.Eukaryotic promoters are
structurally complex and so it is believed that transcription in more complex
than currently believed.This has motivated us to understand the complexity in
sequence-specific binding nature of TFs in promoter recognition during initial
stages of transcription based on the 6-nts that are present within the promoter
sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have looked at functional promoter sequences in
Homo sapiens from Eukaryotic Promoter Database (EPD). We have located
relatively conserved GC-rich 6-nt sequences that are common in the promoter
sequences. Next looked for their distribution with in the promoter sequences
and also in human miRNA datasets (from miRBase database) and transcription
factor binding sites from JASPAR database.</p>

<h3>Results</h3>

<p class="abresults">Sigmoidal distribution of 6-nt sequences in upstream and
downstream of TSS suggests existence of some internal co-operativity in their
distribution.We also noticed that more than half of the promoter sequences
contain multiple signals that favor strong binding signals for TFs.Many of
these 6-nts were identified in miRNA datasets suggesting that miRNAs play
significant role in recognition of promoters.We have identified 6-nts sequences
in transcription factor binding sites of TFs.We correlated the promoters, miRNA
and TFs based on common 6-nts.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have located GC-rich 6-nt signals around TSS in human
promoter sequences.Sigmoidal distributions of 6-nts suggest that at least many
of them are likely to have important biochemical roles.TFs must straddle on
both strands at or near the TSS though they must be relived for continuation of
transcription.Correlation between promoters and miRNAs suggest that a group of
promoters or genes could be regulated by few miRNAs.We postulate that miRNAs
play significant role in promoter recognition.We aim to classify TFs based on
6-nt sequences and also studying their biological significance.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Padmavathi Putta (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87914 = 'p&#97;dm&#97;_b&#105;&#111;&#105;nf&#111;' + '&#64;';
 addy87914 = addy87914 + 'y&#97;h&#111;&#111;' + '&#46;' + 'c&#111;' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy87914 + suffix + '\'' + attribs + '>' );
 document.write( addy87914 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22890 = 'p&#97;dm&#97;_b&#105;&#111;&#105;nf&#111;' + '&#64;';
 addy22890 = addy22890 + 'y&#97;h&#111;&#111;' + '&#46;' + 'c&#111;' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy22890 + suffix + '\'' + attribs + '>' );
 document.write( addy22890 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95286 = 'p&#97;dm&#97;_b&#105;&#111;&#105;nf&#111;' + '&#64;';
 addy95286 = addy95286 + 'y&#97;h&#111;&#111;' + '&#46;' + 'c&#111;' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy95286 + suffix + '\'' + attribs + '>' );
 document.write( addy95286 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dept of Biochemistry, University of Hyderabad</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Biochemistry, University of Hyderabad,
Hyderabad - 500046 Andhra Pradesh, India. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741015">A-11. Iterative read
mapping and assembly allows the use of more distant reference in metagenome
assembly</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dutilh BE<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Dutilh BE&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Huynen MA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Huynen MA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gloerich J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gloerich J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Strous M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Strous M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4,5)</p>

<p class="abmotivation">Most microbial species can not be cultured in the lab.
Metagenomic sequencing may still yield a complete genome if the sequenced
community is enriched and the sequencing coverage is high. However, a natural
population may contain multiple related strains. Moreover, it is not uncommon
that these strains represent a quasispecies that is only distantly related to
the closest available reference genome. This can confound strict assembly
programs and lead to a fragmented assembly.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The full methods are explained in [Dutilh et al.
2009, Bioinformatics 25:2878]. Briefly, we used a permissive (BlastN) and a strict
(Maq) mapping algorithm to assemble a majority consensus. As the initial
assembly better represents the sequenced genomes than the reference genome
does, we iterated the mapping and assembly several times.</p>

<h3>Results</h3>

<p class="abresults">We show that by iteratively mapping short metagenomic
sequencing reads from a population of strains to a related reference genome, we
can create a genome that captures the consensus of the population’s sequences.
Iteration allows us to map more of the reads, leading to a higher coverage and
depth of the assembled consensus genome. At the same time, the similarity with
the reference genome decreases, while the number of metaproteomic peptides that
can be explained by the assembly increases.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This indicates that the assembly becomes less dependent
on the reference genome and approaches the consensus genome of the multi-strain
population. Thus, by exploiting the homology offered by a reference genome in
combination with permissive, iterative read mapping, we get a better view of
both the consensus genome sequence of the quasispecies present in the sample
and of the sequence diversity between the strains.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.cmbi.ru.nl/%7Edutilh/" mce_href="http://www.cmbi.ru.nl/~dutilh/">http://www.cmbi.ru.nl/~dutilh/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Bas E. Dutilh (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12494 = 'd&#117;t&#105;lh' + '&#64;';
 addy12494 = addy12494 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy12494 + suffix + '\'' + attribs + '>' );
 document.write( addy12494 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88570 = 'd&#117;t&#105;lh' + '&#64;';
 addy88570 = addy88570 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy88570 + suffix + '\'' + attribs + '>' );
 document.write( addy88570 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13619 = 'd&#117;t&#105;lh' + '&#64;';
 addy13619 = addy13619 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy13619 + suffix + '\'' + attribs + '>' );
 document.write( addy13619 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CMBI / NCMLS / Radboud University Nijmegen Medical
Centre</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Centre for Molecular and Biomolecular Informatics,
Nijmegen Centre for Molecular Life Sciences, Radboud University Nijmegen
Medical Centre, Geert Grooteplein 28, 6525 GA Nijmegen, The Netherlands (2)
Nijmegen Proteomics Facility, Radboud University Nijmegen Medical Centre, Geert
Grooteplein-Zuid 10, 6525 GA, Nijmegen, The Netherlands (3) Department of
Microbiology, Radboud University Nijmegen, Heyendaalsweg 135, 6525 AJ,
Nijmegen, The Netherlands (4) MPI for Marine Microbiology, Celsiusstrasse 1,
D-28359, Bremen, Germany (5) Centre for Biotechnology, University of Bielefeld,
Bielefeld, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research was funded in part by Dutch Science
Foundation (NWO) Horizon Project 050-71-058. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741016">A-12. APPRIS: a database
of annotated principal splice variants for the human genome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rodríguez J-M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Rodríguez J-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Ezkurdia I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ezkurdia I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lopez G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lopez G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Pietrelli A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pietrelli A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Maietta P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Maietta P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Wesselink J-J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wesselink J-J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Valencia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valencia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Tress M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tress M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The role played by alternative splicing in the modulation
of cellular function is some way from being clarified. Recent works have
suggested that many alternative isoforms are likely to have altered structure,
localisations and cellular functions. Given the likely ubiquity in the cell of
alternative isoforms, the role of these alternatively spliced gene products is
becoming an increasingly important question. Determining principal functional
variants is a critical first step in the study of the implications of
alternative splicing.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have developed a database to aid in the
annotation of any set of well-annotated genes. The database deploys a range of
computational methods. For example, variants are mapped to known structures
using the module MATADOR and Firestar predicts individual functionally
important residues. The principal isoform for each gene is selected based on
the annotations from these methods. The database is currently being used in
collaboration with the GENCODE consortium to annotate the human genome and is part
of the scale up of the ENCODE project to annotate 100% of the human genome.</p>

<h3>Results</h3>

<p class="abresults">We have annotated the 22,304 genes in the current HAVANA
release of the human genome (release 3C). Based on this information we have
been able to select a principal isoform for over 75% of the genes in the human
genome. Many of the alternative variants are likely to have changed structure,
localisation or function. Over 35% of the annotated alternative splice variants
have damaged Pfam functional domains, while at least 30% of the alternative
variants are likely to have substantially altered protein structure.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Alternative splicing has the potential to expand the
cellular protein repertoire by altering the biological function of the
expressed proteins. Many of the alternative splice variants that we see here
are predicted to have changed structure, localisation or function in relation
to the selected principal variant. These are results are an interesting first
step. What is required now is experimental evidence of the expression of these
isoforms as proteins and the characterization of the structure and in vivo
function and localisation of the alternative isoforms.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://appris.bioinfo.cnio.es" mce_href="http://appris.bioinfo.cnio.es">http://appris.bioinfo.cnio.es</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jose M. Rodriguez Carrasco (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79855 = 'jmr&#111;dr&#105;g&#117;&#101;z' + '&#64;';
 addy79855 = addy79855 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy79855 + suffix + '\'' + attribs + '>' );
 document.write( addy79855 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97474 = 'jmr&#111;dr&#105;g&#117;&#101;z' + '&#64;';
 addy97474 = addy97474 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy97474 + suffix + '\'' + attribs + '>' );
 document.write( addy97474 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38554 = 'jmr&#111;dr&#105;g&#117;&#101;z' + '&#64;';
 addy38554 = addy38554 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy38554 + suffix + '\'' + attribs + '>' );
 document.write( addy38554 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Spanish National Bioinformatics Institute (INB)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)-Structural Biology and Biocomputing Programme,
Spanish National Cancer Research Centre (CNIO). Madrid, Spain. (2)-Spanish
National Bioinformatics Institute (INB).</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This is a NIH-funded ENCODE project and was supported
in part by Spanish National Bioinformatics Institute (INB). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741017">A-13. Accurate long read
mapping using enhanced suffix arrays</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vyverman M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vyverman M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Fack V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fack V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Schrijver J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Schrijver J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Dawyndt P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dawyndt P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">With the rise of high throughput sequencing, new programs
have been developed for dealing with the alignment of a huge amount of short
read data to reference genomes. Recent developments in sequencing technology
allow longer reads, but the mappers for short reads are not suited for reads of
several hundreds of base pairs. We propose an algorithm that can handle large
reads with mutations and long insertions and deletions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our algorithm is based on chaining together maximal
exact matches (MEMs) and uses heuristics to speed up the search. To compute the
MEMs we use a specialized index structure, called enhanced suffix array. The
MEMs are used as anchors in the alignment and are combined to a full local
alignment using a combination of the Needleman-Wunsch algorithm and some
heuristics.</p>

<h3>Results</h3>

<p class="abresults">The proposed algorithm has been tested on simulated BRCA1
data. BRCA1 is a gene known to be involved in the development of breast cancer
when mutated. A reference sequence of 80000bp was used, together with about
80000 queries of length between 8bp and 1021bp. We compared our results with
results obtained by Bowtie. Our algorithm was able to find an optimal alignment
for 95% of the queries, whereas Bowtie found only 31% of them. Moreover, our
algorithm is very fast, taking only a few seconds.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We presented an algorithm for mapping large reads to a
reference genome which is fast and accurate in finding an optimal local
alignment. The algorithm is easy to understand and has few parameters. Our
first results show that the algorithm is able to map reads with insertions,
deletions and mutations and with a length of several hundreds of base pairs
successfully to a reference sequence.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Michaël Vyverman (<a class="moz-txt-link-abbreviated" href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24217 = 'M&#105;ch&#97;&#101;l.Vyv&#101;rm&#97;n' + '&#64;';
 addy24217 = addy24217 + 'Ug&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy24217 + suffix + '\'' + attribs + '>' );
 document.write( addy24217 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9306 = 'M&#105;ch&#97;&#101;l.Vyv&#101;rm&#97;n' + '&#64;';
 addy9306 = addy9306 + 'Ug&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9306 + suffix + '\'' + attribs + '>' );
 document.write( addy9306 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1396 = 'M&#105;ch&#97;&#101;l.Vyv&#101;rm&#97;n' + '&#64;';
 addy1396 = addy1396 + 'Ug&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy1396 + suffix + '\'' + attribs + '>' );
 document.write( addy1396 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University, Dept. Applied Math. &amp; Comp.
Sci.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Applied Mathematics and Computer
Science, Ghent University, Krijgslaan 281-S9, 9000 Ghent, Belgium (2)
Department of Molecular Biotechnology, Ghent University, Coupure Links 653,
9000 Ghent, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741018">A-14. Detection and
architecture of small heat shock protein monomers</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Poulain P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Poulain P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gelly J-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gelly J-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Flatters D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Flatters D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">sHSPs are chaperone-like proteins able to prevent
aggregation of misfolded proteins. Found in most organisms, these proteins form
large oligomeric complexes. Their chaperone function is tightly associated to
their highly dynamics quaternary structures. The features that regulate the
sequence - structure - function relationships remain still unclear. Here, we
propose a better understanding of the architecture, organization and properties
of the sHSP family through structural and functional annotations of monomers.
We focused on ACD, a beta-sandwich fold, that is the hallmark of the sHSP
family.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">sHSP proteins are characterized by low sequence
identity, few available structures but with a strong structural pattern. We
developed a new approach for detecting protein sequences of sHSPs and
delineating ACDs. This approach is based on an iterative Hidden Markov Model
algorithm using a multiple alignment profile generated from structural data on
ACD. Protein sequences identified as sHSPs were found in the UniProt databank
that contains more than 10 million protein sequences. sHSPs were grouped based
on their taxonomic origin.</p>

<h3>Results</h3>

<p class="abresults">We found 4478 sequences identified as sHSP showing a very
good coverage with the corresponding PROSITE and Pfam profiles. We showed that
taxonomic-based groups of sHSPs have unique features regarding their ACD. We
detailed highly conserved residues and patterns specific to the whole family or
to some groups of sHSPs. For 96% of studied sHSPs, we identified in the
C-terminal region a conserved I/V/L-X-I/V/L motif that acts as an anchor in the
oligomerization process. The fragment defined from the end of ACD to the end of
this motif is named the C-terminal Anchoring Module (CAM).</p>

<h3>Discussion</h3>

<p class="abdiscussion">In this work, we annotated structural components of ACD
and quantifies properties of thousands sHSPs. Structural elements identified as
essential to the oligomerization process are associated with specific sequence
properties in each group. We showed the relevance of the iterative HMM approach
based on structural data, learned on sequences but aimed toward elucidating
structural/functional properties of sHSPs. As a perspective, we intend to make
a web server dedicated to this family.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Delphine Flatters (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28519 = 'd&#101;lph&#105;n&#101;.fl&#97;tt&#101;rs' + '&#64;';
 addy28519 = addy28519 + '&#117;n&#105;v-p&#97;r&#105;s-d&#105;d&#101;r&#111;t' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy28519 + suffix + '\'' + attribs + '>' );
 document.write( addy28519 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97345 = 'd&#101;lph&#105;n&#101;.fl&#97;tt&#101;rs' + '&#64;';
 addy97345 = addy97345 + '&#117;n&#105;v-p&#97;r&#105;s-d&#105;d&#101;r&#111;t' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy97345 + suffix + '\'' + attribs + '>' );
 document.write( addy97345 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56065 = 'd&#101;lph&#105;n&#101;.fl&#97;tt&#101;rs' + '&#64;';
 addy56065 = addy56065 + '&#117;n&#105;v-p&#97;r&#105;s-d&#105;d&#101;r&#111;t' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy56065 + suffix + '\'' + attribs + '>' );
 document.write( addy56065 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">INSERM UMR-S 665 and Université Paris Diderot</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) DSIMB, Inserm UMR-S 665 and Université Paris
Diderot – Paris 7, INTS, Paris, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Financial support was provided by grants from the
French Ministry of Research, the Paris Diderot - Paris 7 University, the
National Institute for Blood Transfusion (INTS) and the Institute for Health
and Medical Research (INSERM). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741019">A-15. Assembly of
mitochondrial genomes using next-generation sequencing data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Felder M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Felder M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Taudien S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Taudien S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Groth M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Groth M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Münsterkötter M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Münsterkötter M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Navarro-Quezada A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Navarro-Quezada A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Knogge W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Knogge W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Platzer M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Platzer M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Classical approaches to sequence and analyse
mitochondrial DNA are based on the separate isolation and/or amplification of
mitochondrial DNA. The aim of our approach was to identify and analyse
mitochondrial DNA using next-generation data derived from whole-genome shotgun
sequencing of Rhynchosporium secalis UK7 and 6a.1. These two isolates are of
particular interest as pathogens of barley and rye, respectively.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For both isolates whole-genome 454 libraries were
generated according to the manufacturer's protocol and sequenced using a GS FLX
(Roche). In addition, whole-genome paired-end DNA libraries with insert sizes
of about 330 bp were constructed and sequenced on a Solexa/Illumina Genome
Analyser II using standard procedures. In total, for each of the isolates more
than 2 Gb of usable sequence were generated. The nucleotide sequences were
assembled using the Newbler software (Roche).</p>

<h3>Results</h3>

<p class="abresults">De novo assemblies using datasets with reduced read counts
resulted in putative mitochondrial sequences of the R. secalis isolates. By
remapping reads to these sequences the complete circular mitochondrial genomes
of R. secalis UK7 (69,581 bp) and 6a.1 were obtained (68,729 bp). A comparison
of the R. secalis mitochondrial DNAs with fungal mitochondrial reference
sequences confirmed the high degree of conservation of sequence and structure
within these fungi.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Despite the extremely high mitochondrial and the low
nuclear sequencing depths the mitochondrial genomes of two strains of the plant
pathogenic fungus R. secalis were identified and analyzed from whole-genome
shotgun sequencing data.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Marius Felder (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62606 = 'mf&#101;ld&#101;r' + '&#64;';
 addy62606 = addy62606 + 'fl&#105;-l&#101;&#105;bn&#105;z' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy62606 + suffix + '\'' + attribs + '>' );
 document.write( addy62606 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1927 = 'mf&#101;ld&#101;r' + '&#64;';
 addy1927 = addy1927 + 'fl&#105;-l&#101;&#105;bn&#105;z' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy1927 + suffix + '\'' + attribs + '>' );
 document.write( addy1927 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39609 = 'mf&#101;ld&#101;r' + '&#64;';
 addy39609 = addy39609 + 'fl&#105;-l&#101;&#105;bn&#105;z' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy39609 + suffix + '\'' + attribs + '>' );
 document.write( addy39609 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Genome Analysis, Leibniz Institute for Age Research -
Fritz Lipmann Institute (FLI)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Genome Analysis, Leibniz Institute for Age Research
- Fritz Lipmann Institute (FLI), Beutenbergstr. 11, D-07745 Jena, Germany (2)
MIPS - Institute of Bioinformatics and Systems Biology, Ingolstädter Landstr.
1, D-85764 Neuherberg, Germany (3) IPB - Leibniz Institute of Plant
Biochemistry, Weinberg 3, D-06120 Halle, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Collaborative PAKT project of IPB Halle, Germany</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741020">A-16. AnsNGS: Annotation
system of structural variation for Next Generation Sequencing data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Na Y-J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Na Y-J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Park CH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Park CH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Cho Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cho Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kim JH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kim JH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">The ongoing revolution in sequencing technology has led
to the production of sequencing machines with dramatically low costs and high
throughput. While the promise of next-generation sequencing technologies has
become a reality, they also present substantial informatics challenges. Most
current informatics methods have focused on preprocessing such as alignment and
assembly or visualization of mapping read sequences. However, in order to
satisfy the impending need for deciphering the large-scale data generated from
next-generation sequencing, it is essential to the development of an integrated</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">AnsNGS can import data from MAQ and SOAP which are
the most popular read mapping methods. It supports annotated sequence
information by integrating genome feature annotation databases such as genes,
non-coding RNAs and structural variations. To speed up query, R-Tree with
quadratic splitting to index spatial columns was used to retrieve annotation
information from the integrated database. The use of a parallel computing
system based on multi-core processor architectures will scale its performance.
AnsNGS can export the annotation results for post-analysis.</p>

<h3>Results</h3>

<p class="abresults">To determine the effectiveness of AnsNGS, we have tested
AnsNGS on several Roche 454, Illumina Solexa and ABI SOLiD data sets with data
size ranging from 10 to 50 million reads. All mapped reads are annotated, using
dual-core CPU processors on a Linux machine with 16G RAM.</p>

<h3>Discussion</h3>

<p class="abdiscussion">AnsNGS supports correlations between aligned read
sequences and transcriptome features. Moreover, it provides known structural
variation as well as sequence annotation information. Elucidation and
utilization of individual genetic differences including various structural
variations would be a major hallmark of personalized medicine.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://clara.snubi.org:8080/shortRead/" mce_href="http://clara.snubi.org:8080/shortRead/">http://clara.snubi.org:8080/shortRead/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Young-Ji Na (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5803 = 'yjn&#97;01' + '&#64;';
 addy5803 = addy5803 + 'sn&#117;' + '&#46;' + '&#97;c' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy5803 + suffix + '\'' + attribs + '>' );
 document.write( addy5803 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1613 = 'yjn&#97;01' + '&#64;';
 addy1613 = addy1613 + 'sn&#117;' + '&#46;' + '&#97;c' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy1613 + suffix + '\'' + attribs + '>' );
 document.write( addy1613 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy30990 = 'yjn&#97;01' + '&#64;';
 addy30990 = addy30990 + 'sn&#117;' + '&#46;' + '&#97;c' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy30990 + suffix + '\'' + attribs + '>' );
 document.write( addy30990 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Seoul National University Biomedical Informatics
(SNUBI) Seoul National University College of Medicine</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1Seoul National University Biomedical Informatics
(SNUBI) Seoul National University College of Medicine 28 Yongon-dong Chongno-gu
Seoul 110-799, Korea 2Interdisciplinary Program in Bioinformatics, Seoul
National University, Seoul 151-747, Korea 3Division of Biomedical Informatics
Seoul National University College of Medicine 28 Yongon-dong Chongno-gu Seoul
110-799, Korea</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741021">A-17. Assessing the
accuracy and completeness of the bonobo genome sequence</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Prüfer K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Prüfer K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Ptak SE<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ptak SE&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Fischer A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fischer A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Good JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Good JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mullikin JC<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mullikin JC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Miller J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Miller J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Kodira CD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kodira CD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Knight JR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Knight JR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), The Bonobo Genome Consortium<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;The Bonobo Genome Consortium&quot;
<![endif]--><!--[if supportFields]><span style="mso-element:field-end" mce_style="mso-element:field-end"></span><![endif]-->,
Kelso J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Kelso J&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1), Pääbo
S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Pääbo S&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1)</p>

<p class="abmotivation">We analyze the quality of the bonobo genome assembly,
generated from 25-fold coverage of 454 SequencingTM data, encompassing both
shotgun and paired end reads. This assembly has a N50 scaffolds size of 9.6
megabases and a N50 contig size of 67 kilobases.The bonobo genome sequence is
particularly suited for the analysis of assembly quality since the genome
sequences of two closely related species (human and chimpanzee) are available
for comparison.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use whole genome alignments between the genomes
of human, chimpanzee and bonobo to estimate the completeness and sequence
accuracy of the Bonobo genome in comparison with the chimpanzee genome.</p>

<h3>Results</h3>

<p class="abresults">Our analysis shows that the bonobo assembly achieves
comparable quality to the chimpanzee assembly in terms of sequence accuracy and
genome completeness. Using the high quality of the finished chimpanzee
chromosome 21 we are able to estimate a rate of approximately two errors in
10,000 base pairs for the bonobo genome.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The quality of the Bonobo X chromosome assembly is as
good as the other autosomes since a female individual was selected for
sequencing. This places this bonobo's X and all her autosomes in an excellent
position for comparative analysis to other primates. We conclude that large and
complex genomes can be de novo assembled from next generation sequencing data.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Kay Prüfer (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89494 = 'pr&#117;&#101;f&#101;r' + '&#64;';
 addy89494 = addy89494 + '&#101;v&#97;' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy89494 + suffix + '\'' + attribs + '>' );
 document.write( addy89494 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66195 = 'pr&#117;&#101;f&#101;r' + '&#64;';
 addy66195 = addy66195 + '&#101;v&#97;' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy66195 + suffix + '\'' + attribs + '>' );
 document.write( addy66195 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93102 = 'pr&#117;&#101;f&#101;r' + '&#64;';
 addy93102 = addy93102 + '&#101;v&#97;' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy93102 + suffix + '\'' + attribs + '>' );
 document.write( addy93102 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Max-Planck Institute for Evolutionary Anthropology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Max Planck Institute for Evolutionary Anthropology,
Leipzig, Germany 2 National Institute of Health, Bethesda, MD, USA 3 J. Craig
Venter Institute, Rockville, MD, USA 4 454 Life Sciences, Branford, CT, USA</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741022">A-18. Estimating
effective DNA database size via compression</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Visnovska M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Visnovska M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Nanasi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nanasi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vinar T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vinar T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Brejova B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brejova B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Recent progress in genome sequencing technologies led to
a rapid increase in the size of DNA sequence databases. In large databases,
many query sequences will have high scoring matches purely by chance. To
distinguish real matches from spurious ones, we need to assess their
statistical significance. One of the main parameters used in P-value
computation is the database size n. Instead of the real database size, we
propose to use an effective database size which will account for redundancies
in the database.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">One way of describing the information content of a
database D is its Kolmogorov complexity (Li and Vitanyi, 2008). The Kolmogorov
complexity is not computable, but the compressed size of D obtained by a fixed
compression algorithm can be used as an upper bound. We use the compressed size
of D as its effective size. We have implemented an algorithm that computes
exact P-values for a given database D and all of its prefixes in the simple
scenario of finding the closest match to a short query under Hamming distance.
It can be used on real or randomly generated databases.</p>

<h3>Results</h3>

<p class="abresults">We tested our approach on databases of three different
types: randomly generated databases with GC-contents 75% and 90%, artificial
databases that are concatenation of many mutually similar sequences of the same
length, and real genomic data from human, chimpanzee, and rhesus macaque.
Experimental results suggest that P-value estimates based on effective size are
appropriate for large databases, but not conservative for the small. However,
conditional entropy as a correction factor for small databases seems promising.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have considered methods for more accurate estimation
of P-values for sequence homology search. In particular, we propose to adjust
the size of the database to compensate for the structure present in the
database due to the fact that individual sequences are related by evolution.
Our method is flexible and easy to implement. In future, we would like to
extend our work to more complex scenarios of homology search. Longer query
sequences will require handling of insertions and deletions. More complex
scoring schemes on both nucleotide and protein sequences also need to be
examined.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Martina Visnovska (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18260 = 'v&#105;sn&#111;vsk&#97;' + '&#64;';
 addy18260 = addy18260 + '&#105;&#105;' + '&#46;' + 'fmph' + '&#46;' + '&#117;n&#105;b&#97;' + '&#46;' + 'sk';
 document.write( '<a ' + path + '\'' + prefix + addy18260 + suffix + '\'' + attribs + '>' );
 document.write( addy18260 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55366 = 'v&#105;sn&#111;vsk&#97;' + '&#64;';
 addy55366 = addy55366 + '&#105;&#105;' + '&#46;' + 'fmph' + '&#46;' + '&#117;n&#105;b&#97;' + '&#46;' + 'sk';
 document.write( '<a ' + path + '\'' + prefix + addy55366 + suffix + '\'' + attribs + '>' );
 document.write( addy55366 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6422 = 'v&#105;sn&#111;vsk&#97;' + '&#64;';
 addy6422 = addy6422 + '&#105;&#105;' + '&#46;' + 'fmph' + '&#46;' + '&#117;n&#105;b&#97;' + '&#46;' + 'sk';
 document.write( '<a ' + path + '\'' + prefix + addy6422 + suffix + '\'' + attribs + '>' );
 document.write( addy6422 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Applied Informatics, Faculty of
Mathematics, Physics, and Informatics, Comenius University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Faculty of Mathematics, Physics and Informatics,
Comenius University, Bratislava</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is funded by European Community FP7
grants IRG-224885 and IRG-231025, VEGA grant 1/0210/10, and Comenius University
grant UK/151/2010. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741023">A-19. ChIP-seq analysis:
from peaks to motifs</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Thomas-Chollier M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thomas-Chollier M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Defrance M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Defrance M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Sand O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sand O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Herrmann C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Herrmann C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Thieffry D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thieffry D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), van Helden J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Helden J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,5)</p>

<p class="abmotivation">The treatment of the raw ChIP-seq data proceeds in three
steps :(i) read mapping; (ii) peak calling; (iii) detecting regulatory motifs
in peak sequences. Whereas many software tools are being developed to address
the two first steps, most classical methods for analyzing regulatory motifs are
not able to cope with the unprecedented amounts of sequences to be processed:
computing time and memory requirements constitute the main bottleneck for peak
sequence analysis. Consequently, a common strategy is to restrict the analysis
to a subset of the top-scoring peak sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We present ChIP-motifs, a pipe-line combining
several motif discovery approaches that are scalable to genome-wide datasets.
Those approaches rely on distinct statistical criteria to detect exceptional
words: global over-representation, local over-representation and positional
bias. The analysis covers oligonucleotides as well as spaced pairs (dyads),
which are particularly suited for dimeric transcription factors. Discovered
motifs are compared to collections of previously known motifs. The pipe-line
also analyses mono- and di-nucleotide compositions of the peaks.</p>

<h3>Results</h3>

<p class="abresults">When applied to 13 ChIP-seq peak sets from mouse
transcription factors controlling stem cell pluripotency [Chen et al. (2008)
Cell 133:1106-17], ChIP-motifs successfully recovers the reference motifs, and
reveals potential co-factors. In most cases, several algorithms of the
pipe-line independently discover the same motifs, revealing both
over-representation and bias towards the center of the peaks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The approach proposed here answers a crucial need for the
treatment of full datasets resulting from ChIP-seq and ChIP-chip experiments.
The program chip-motifs is integrated in the Regulatory Sequence Analysis Tools
(RSAT), and can be accessed via a Web form, SOAP/WSDL Web services, or as a
stand-alone application. Its low time and memory requirements enable to treat
datasets comprising tens of thousands peaks on a personal computer.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://rsat.ulb.ac.be/rsat/" mce_href="http://rsat.ulb.ac.be/rsat/">http://rsat.ulb.ac.be/rsat/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jacques van Helden (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52650 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy52650 = addy52650 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy52650 + suffix + '\'' + attribs + '>' );
 document.write( addy52650 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45765 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy45765 = addy45765 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy45765 + suffix + '\'' + attribs + '>' );
 document.write( addy45765 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17768 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy17768 = addy17768 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy17768 + suffix + '\'' + attribs + '>' );
 document.write( addy17768 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Computational Molecular Biology, Max
Planck Institute for Molecular Genetics, Ihnestrasse 73, 14195 Berlin, Germany.
Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4304 = 'm&#111;rg&#97;n&#101;' + '&#64;';
 addy4304 = addy4304 + 'b&#105;gr&#101;' + '&#46;' + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy4304 + suffix + '\'' + attribs + '>' );
 document.write( addy4304 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 2. Centro de Ciencias Genomicas, Universidad
Nacional Autónoma de México. Av. Universidad, Cuernavaca, Morelos 62210,
Mexico. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33678 = 'd&#101;fr&#97;nc&#101;' + '&#64;';
 addy33678 = addy33678 + 'ccg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy33678 + suffix + '\'' + attribs + '>' );
 document.write( addy33678 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 3. CNRS-UMR8199 Institut de Biologie de
Lille. Génomique et maladies métaboliques. 1, rue du Pr Calmette, 59000 Lille,
France. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40657 = 's&#97;nd' + '&#64;';
 addy40657 = addy40657 + 'g&#111;&#111;d' + '&#46;' + '&#105;bl' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy40657 + suffix + '\'' + attribs + '>' );
 document.write( addy40657 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 4. Technological Advances for Genomics and
Clinics (TAGC), INSERM U928 &amp; Université de la Méditerranée. Campus de
Luminy, F - 13288 Marseille, France. Email:
{thieffry,herrmann}@tagc.univ-mrs.fr 5. Laboratoire de Bioinformatique des
Génomes et des Réseaux (BiGRe). Université Libre de Bruxelles, Campus Plaine,
CP 263. Bld du Triomphe. B-1050 Bruxelles, Belgium. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99589 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy99589 = addy99589 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy99589 + suffix + '\'' + attribs + '>' );
 document.write( addy99589 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Belgian Program on
Interuniversity Attraction Poles (IAP), initiated by the Belgian Federal
Science Policy Office, project P6/25 (BioMaGNet). The BiGRe laboratory is
supported by the MICROME Collaborative Project funded by the European
Commission within its FP7 Programme, under the thematic area
"BIO-INFORMATICS - Microbial genomics and bio-informatics", contract
number 222886-2." MTC is supported by the Alexander von Humbold Stiftung. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741024">A-20. Applying mass
spectrometry data to improve annotation of the reference mouse genome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Saunders GI<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Saunders GI&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Brosch M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brosch M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Frankish A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Frankish A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Collins MO<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Collins MO&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Yu L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yu L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Harrow J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Harrow J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Choudhary JS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Choudhary JS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Hubbard T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hubbard T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">With the advent of the next-generation DNA sequencing
technologies, our understanding of the transcriptome has rapidly improved very
recently; however, the same cannot be said of the proteome. Recent advances in
protein Mass Spectrometry (MS) are of great excitement to genome annotators,
offering the chance to marry high-throughput peptide sequencing to predicted
coding sequences, and to identify new protein-coding loci. For the first time,
we have systematically validated and improved mouse genome annotation with
tandem MS data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">This study is based on 10,465,149 tandem MS
spectra, of which 9,735,566 were obtained from the Peptide Atlas project and
729,583 from in-house experiments on nuclear protein extracts of murine
embryonic stem cells and murine brain membrane fractions. All MS peaklist data
were searched with Mascot and post processed with Mascot Percolator. The processing
parameters used in each programme were identical to those employed by Brosch
2009 (Brosch et al. Journal of Proteome Research 8: 3176-3181). We used the
Ensembl Perl API to unambiguously map the resulting peptides to the reference
mouse genome.</p>

<h3>Results</h3>

<p class="abresults">We have validated 32% of all mouse protein-coding genes, 17%
of all protein-coding exons and 7% of all protein-coding splice boundaries. In
addition we present evidence for the identification of 53 high confidence genes
with multiple alternative translations. Moreover, we have uncovered 10 novel
protein-coding loci that are not present in any mouse annotation data sources.
Our study also provides 24 high confidence peptide spectrum matches that map to
and support the translation of 10 processed pseudogenes, indicating their
resurrection into coding forms.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In spite of the mouse proteome being far from saturated
by MS based peptide identifications, we have applied our novel annotation
pipeline to enable the first proteogenomic study to both validate and improve
the annotation of this genome. We believe that this method to identify
protein-coding loci will become a mainstay of genome annotation pipelines in
the near future and we provide results to support our view that, in order to
avoid erroneous annotation, it should be our high stringency approach that is
adopted.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://das.sanger.ac.uk/das/ms_das" mce_href="http://das.sanger.ac.uk/das/ms_das">http://das.sanger.ac.uk/das/ms_das</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Gary I. Saunders (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46172 = 'gs6' + '&#64;';
 addy46172 = addy46172 + 's&#97;ng&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy46172 + suffix + '\'' + attribs + '>' );
 document.write( addy46172 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29227 = 'gs6' + '&#64;';
 addy29227 = addy29227 + 's&#97;ng&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy29227 + suffix + '\'' + attribs + '>' );
 document.write( addy29227 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13207 = 'gs6' + '&#64;';
 addy13207 = addy13207 + 's&#97;ng&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy13207 + suffix + '\'' + attribs + '>' );
 document.write( addy13207 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Wellcome Trust Sanger Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Wellcome Trust Sanger Institute, Wellcome Trust Genome
Campus, Cambridge, UK, CB10 ISA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by the Wellcome Trust. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741025">A-21. Prediction of splice-modifying
SNPs in human genes using a new analysis pipeline called AASsites</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hotz-Wagenblatt A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hotz-Wagenblatt A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Faber K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Faber K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Risch A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Risch A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Glatting K-H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Glatting K-H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Single nucleotide polymorphisms (SNPs) become more and
more important in disease research. The aim of most existing SNP tools is the
annotation of SNPs mainly by analyzing variances in the coding region of the
affected gene and possible effects on the transcriptome and proteome levels. In
contrast, our AASsites pipeline (Automatic Analysis of SNP sites) indicates if
a change in the splice pattern due to a SNP is likely to occur.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Implemented in the W3H-pipeline system, AASsites
uses a combination of different gene prediction methods (GenScan, GeneID,
HMMgene, GlimmerHMM and GrailExp), and an ESE (Exonic Splice Enhancer)
detection to predict changes in the elements which are relevant for splicing.
Additionally, Genewise analysis determines changes in the ORF (open reading
frame). The rule based classification system ranks the SNPs and their effects
into different categories.</p>

<h3>Results</h3>

<p class="abresults">Modifications in the exon-intron structure and variances in
ESE and ORF are shown in the final result of the pipeline including new, disappeared
and modified exons and introns. 37 SNPs with known effects on splicing and 23
negative SNPs were used for testing AASsites. 73% of the positive SNPs and 100%
of the negative SNPs were classified correctly. We checked 80,000 SNPs from the
human genome which are located near splice sites for their ability to change
the splicing pattern of the gene. We identified 301 “likely” and 985 “probable”
classified SNPs with such characteristics.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have shown with a set of SNPs with known changes, that
the pipeline can predict the change in splicing caused by the SNP in 83% of 60
cases correctly. The problem of testing and improving the rule system for
combining the results is caused by the small number of experimentally proven
SNP-derived modifications in splicing. With more experimental data available we
could replace the rule system by a knowledge system based on machine learning
algorithms.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Agnes Hotz-Wagenblatt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26027 = 'h&#111;tz-w&#97;g&#101;nbl&#97;tt' + '&#64;';
 addy26027 = addy26027 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy26027 + suffix + '\'' + attribs + '>' );
 document.write( addy26027 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26700 = 'h&#111;tz-w&#97;g&#101;nbl&#97;tt' + '&#64;';
 addy26700 = addy26700 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy26700 + suffix + '\'' + attribs + '>' );
 document.write( addy26700 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51760 = 'h&#111;tz-w&#97;g&#101;nbl&#97;tt' + '&#64;';
 addy51760 = addy51760 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy51760 + suffix + '\'' + attribs + '>' );
 document.write( addy51760 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">German Cancer Research Center (DKFZ)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Bioinformatik (HUSAR), Core Facility Genomics
Proteomics, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 580,
69120 Heidelberg, Germany 2 Division of Epigenomics and Cancer Risk Factors,
German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120
Heidelberg, Germany</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741026">A-22. Structator: a tool
for fast index-based matching of RNA structural motifs</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Meyer F<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Meyer F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Will S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Will S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Beckstette M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beckstette M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The exponential growth rate of biological sequence
databases asks for tools that allow for fast search of biologically relevant
motifs. While several sequence-pattern matching problems can be vastly
accelerated by index data structures like suffix trees and suffix arrays,
matching RNA structural motifs is more efficient with affix arrays. Despite not
being a new data structure, its potential was neither recognized by the
scientific community, nor was there a public implementation of it.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Structator matches sets of patterns employing the
affix array index data structure introduced by [Strothmann, 2007]. The index is
the combination of the enhanced suffix array of [Abouelhoda et al., 2004] and
the enhanced reverse prefix array of a given sequence. Patterns are matched in
a bidirectional way, i.e. possibly beginning with a character in the middle
with consecutive left and right extensions. In a second step, Structator
performs RNA family classification by building chains of matches where the
order of the patterns is preserved using the algorithm of [Abouelhoda et al.,
2005].</p>

<h3>Results</h3>

<p class="abresults">We measured the running time to search with 392 motifs
describing families of Rfam rel. 10.0 where the consensus structure contained
at least 5 stem-loop motifs. On the precomputed affix array of the database,
Structator performed all searches in 57 sec., whereas a simple online algorithm
required 105 min. 12 sec. In a second experiment we searched with eight motifs
from family RF00193 of Rfam rel. 9.1. Via chaining of the thousands of matches,
Structator removed all spurious hits, thus identifying the 26 members of the
family in the full alignment.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Structator's main features are persistent affix array
construction, flexible alphabet handling (DNA, RNA, proteines) including
alphabet transformation, matching on forward and reverse strand, support of
various pattern types including IUPAC wildcard symbols, support of standard and
user-defined base pairing rules, integrated chaining algorithms, output of
results in different formats e.g. BED for visualization in the UCSC Genome
Browser.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Fernando Meyer (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50243 = 'm&#101;y&#101;r' + '&#64;';
 addy50243 = addy50243 + 'zbh' + '&#46;' + '&#117;n&#105;-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy50243 + suffix + '\'' + attribs + '>' );
 document.write( addy50243 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36005 = 'm&#101;y&#101;r' + '&#64;';
 addy36005 = addy36005 + 'zbh' + '&#46;' + '&#117;n&#105;-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy36005 + suffix + '\'' + attribs + '>' );
 document.write( addy36005 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53156 = 'm&#101;y&#101;r' + '&#64;';
 addy53156 = addy53156 + 'zbh' + '&#46;' + '&#117;n&#105;-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy53156 + suffix + '\'' + attribs + '>' );
 document.write( addy53156 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Bioinformatics, University of Hamburg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Center for Bioinformatics, University of Hamburg
(2) Computation and Biology Group, Massachusetts Institute of Technology</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741027">A-23. Project HOPE:
Providing the last piece of the puzzle...</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Venselaar H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Venselaar H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*)</p>

<p class="abmotivation">Recent developments in next-generation sequencing
increased the detection of disease-related mutations, a significant part
affecting the protein’s 3D-structure. Explaining their structural effect
requires knowledge of the protein, obtained from a series of sources including
the 3D-structure and several databases such as the uniprot database. Existing
online prediction-servers often do not focus on the structural effect of the
mutation. Therefore, we developed project HOPE, a web-server for automatic
point-mutant analysis that is easy-to-understand for anyone in the (bio)medical
field.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The core of Project HOPE is a Java-implemented
module that collects information from a series of sources and stores this in a
database. This information consists of calculations on the real structure or a
homology model by WHAT IF web-services, sequence based predictions by
DAS-servers, and structural features annotated in Uniprot. All information is
stored per residue in a PostGreSQL database. A decision scheme is used to reach
a conclusion about the effect of the mutation that focuses on the structural
changes. The conclusion is shown on a website and illustrated with figures and
animations.</p>

<h3>Results</h3>

<p class="abresults">Our approach was tested in numerous collaborative projects
with researchers from several (bio)medical departments. Our contribution
consisted mainly of mutation studies (non-sense and missense mutations) and
suggestions for experimental design. Often, our findings provided insight that
was necessary to understand the mutation and can therefore be seen as the last
piece that completed the molecular puzzle.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our examples show that a structural analysis provides
essential insight in the effect of a mutation. We developed Project HOPE to
provide easy accessible and understandable structural information that can be
beneficial for research in the (bio)medical field.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.cmbi.ru.nl/hope" mce_href="http://www.cmbi.ru.nl/hope">http://www.cmbi.ru.nl/hope</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Hanka Venselaar (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78762 = 'h.v&#101;ns&#101;l&#97;&#97;r' + '&#64;';
 addy78762 = addy78762 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy78762 + suffix + '\'' + attribs + '>' );
 document.write( addy78762 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33350 = 'h.v&#101;ns&#101;l&#97;&#97;r' + '&#64;';
 addy33350 = addy33350 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy33350 + suffix + '\'' + attribs + '>' );
 document.write( addy33350 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9221 = 'h.v&#101;ns&#101;l&#97;&#97;r' + '&#64;';
 addy9221 = addy9221 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy9221 + suffix + '\'' + attribs + '>' );
 document.write( addy9221 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CMBI, NCMLS, Radboud University Nijmegen Medical
Centre</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Centre for Molecular and Biomolecular Informatics
(CMBI) NCMLS, Radboud University Nijmegen Medical Centre</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741028">A-24. Simple
not-PWM-based approach for transcription factor binding site prediction</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Fazius E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Fazius E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Shelest V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shelest V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Shelest E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shelest E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*)</p>

<p class="abmotivation">The prediction of transcription factor binding sites
(TFBS) is crucial for promoter modeling and consequent network inference.
Despite the huge number of tools for the TFBS prediction, no satisfactory
solution for the high number of false predictions is found so far. All
available tools are PWM-based and differ only in the ways of calculating the
cut-offs. Recent new approaches tend to use more mathematically sofisticated
methods, without breakthrough results. We decided to develop a straightforward
non-heuristic metod based on direct comparison of training set motifs with the
query sequence.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The tool is implemented in PHP.</p>

<h3>Results</h3>

<p class="abresults">In our approach, the query sequence is scanned for motifs
with a sliding window. Each motif is assigned a weight that takes into account
the probabilty of the motif occurrence by chance, the number of matching motifs
of the training set, and the number of mismatches. The method does not require
any preliminary alignment of motifs of the training set. The comparison of our
method with Jaspar database searching tool and Match (TRANSFAC) reveals its
better performance in both sensitivity and specificity. Most important, it
allows to significantly reduce the number of false positive predictions.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We suggest a simple and transparent not-PWM-based
approach that is implemented in a tool called SiteTracker. Not requiring an
alignment of the training set motifs, we avoid the uncertainties introduced by
the alignment procedure, which is especially problematic for very degenerate
motifs, and the following trimming, which can lead to a loss of potentially
important sequence information.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Ekaterina Shelest (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41367 = '&#101;k&#97;t&#101;r&#105;n&#97;.sh&#101;l&#101;st' + '&#64;';
 addy41367 = addy41367 + 'hk&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy41367 + suffix + '\'' + attribs + '>' );
 document.write( addy41367 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35276 = '&#101;k&#97;t&#101;r&#105;n&#97;.sh&#101;l&#101;st' + '&#64;';
 addy35276 = addy35276 + 'hk&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy35276 + suffix + '\'' + attribs + '>' );
 document.write( addy35276 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy48829 = '&#101;k&#97;t&#101;r&#105;n&#97;.sh&#101;l&#101;st' + '&#64;';
 addy48829 = addy48829 + 'hk&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy48829 + suffix + '\'' + attribs + '>' );
 document.write( addy48829 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Hans Knoell Institute, HKI, Jena</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Leibniz institute for Natural Product Research and
Infection Biology, - Hans Knoell Institute, Jena</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741029">A-25. PICMI: mapping
point variations on genomes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Le Pera L<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Le Pera L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Marcatili P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marcatili P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tramontano A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tramontano A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Several international collaborations and local projects
are producing extensive catalogues of genomic variations which are
supplementing existing collections such as the OMIM catalogue. This flood of
data will keep increasing and, especially, it will be used by a wider user
base, including also clinical researchers. Mapping the observed variations on a
genome, identifying whether it affects a gene and - if so - whether it also
affects different isoforms of the same gene, is not a straightforward task. To
face this issue we developed a web server called PICMI.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Multiple nucleotidic and amino acidic variations
can be used as input of the PICMI server, which finds for each of them
(querying different releases of the Ensembl database) whether it lies in a gene
and if it is in a non-coding region (upstream or downstream, in the 5' or 3'
untranslated region, in a stop-codon or in a skipped-exon) or in a coding
region of each transcript. In this last case, the mutation is mapped and
classified for all the isoforms. Notably, when the input is an amino acidic
mutation, the tool verifies if a corresponding single base mutation can be
unambiguously inferred.</p>

<h3>Results</h3>

<p class="abresults">The PICMI web server provides a user-friendly, extremely
easy to use tool for mapping single site variations on a genome and its
products, including alternative spliced isoforms. The server can map nucleotide
variations, but also amino acidic mutations when the corresponding nucleotide
substitution can be unambiguously identified.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We believe that this easy-to-use tool can reveal to be
very useful both to simplify the mapping of nucleotidic variations, for example
it is set to get the SNPs from the 1000 genomes project as input, and,
especially, to analyse a number of pathological and physiological variations at
the nucleotide level when they are only available at the protein level, as it
is often the case for reports recorded in OMIM and in SwissProt variations.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Loredana Le Pera (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy47170 = 'l&#111;r&#101;d&#97;n&#97;l&#101;p&#101;r&#97;' + '&#64;';
 addy47170 = addy47170 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy47170 + suffix + '\'' + attribs + '>' );
 document.write( addy47170 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36889 = 'l&#111;r&#101;d&#97;n&#97;l&#101;p&#101;r&#97;' + '&#64;';
 addy36889 = addy36889 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy36889 + suffix + '\'' + attribs + '>' );
 document.write( addy36889 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79819 = 'l&#111;r&#101;d&#97;n&#97;l&#101;p&#101;r&#97;' + '&#64;';
 addy79819 = addy79819 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy79819 + suffix + '\'' + attribs + '>' );
 document.write( addy79819 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Biochemical Sciences, Sapienza
University of Rome</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Biochemical Sciences, Sapienza
University of Rome, P.le A. Moro, 5 - 00185 Rome</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">FIRB ITALBIONET</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741030">A-26. Detection of
alternative splice isoforms in the human genome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ezcurdia I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ezcurdia I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), del Pozo A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;del Pozo A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rodriguez JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rodriguez JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ashman K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ashman K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Valencia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valencia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Tress M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tress M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tress ML&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Recent studies have estimated that almost all multi-exon
human genes can produce at least two differently spliced mRNA transcripts by
alternative splicing. Although multiple transcripts are strongly supported by
mRNA and EST sequence evidence, the expression of multiple protein isoforms is
less well studied. Proteomics technologies ought to provide conclusive evidence
for alternative protein variants, but MS/MS experiments can only identify a
fraction of the peptide ions present in protease digests and it remains a
technical challenge to detect proteins that are expressed at low levels.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have carried out a comprehensive re-mapping of
experimental spectra to the annotated sequences from the Human proteome. We
used as data all the relevant experiments stored in two huge proteomics data
repositories, the GPM and PeptideAtlas. We reanalysed the mass spectra using
X!Tandem and mapped the results to the HAVANA release 3C annotations.</p>

<h3>Results</h3>

<p class="abresults">We confirmed that many human genes do indeed express
multiple alternative protein isoforms that are stable enough and expressed in
sufficient quantities to be detected in proteomics experiments. In addition to
demonstrating the expression of alternatively spliced isoforms in the human
proteome we have also been able to provide some insight into the functional
role of alternative splicing in the cell.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This work highlights the growing importance of proteomics
in the validation of predicted proteins, especially as a complement to
large-scale annotation efforts such as the ENCODE project. However, we also
show that careful statistical testing is a necessary part of validating any
proteomics results.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Iakes Ezcurdia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36663 = '&#105;&#101;zk&#117;rd&#105;&#97;' + '&#64;';
 addy36663 = addy36663 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy36663 + suffix + '\'' + attribs + '>' );
 document.write( addy36663 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3083 = '&#105;&#101;zk&#117;rd&#105;&#97;' + '&#64;';
 addy3083 = addy3083 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy3083 + suffix + '\'' + attribs + '>' );
 document.write( addy3083 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72921 = '&#105;&#101;zk&#117;rd&#105;&#97;' + '&#64;';
 addy72921 = addy72921 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy72921 + suffix + '\'' + attribs + '>' );
 document.write( addy72921 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Structural Biology and Biocomputing Programme,
Spanish National Cancer Research Centre (CNIO)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Structural Biology and Biocomputing Programme,
Spanish National Cancer Research Centre (CNIO), C./ Melchor Fernandez Almagro,
Madrid, Spain. (2) Spanish National Bioinformatics Institute (INB), Spanish
National Cancer Research Centre (CNIO), C./ Melchor Fernandez Almagro, Madrid,
Spain. (3) Proteomics Division, Spanish National Cancer Research Centre (CNIO),
C./ Melchor Fernandez Almagro, 3, Madrid 28029, Spain. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741031">A-27. Boundaries for
short-read, low-coverage de novo assembly and resequencing</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schatz F<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Schatz F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Schimmler M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schimmler M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">For resequencing and deNovo assembly the amount of input
data is crucial. On the one hand, too little input data results in missing
information about parts of the DNA sequence; and on the other hand, too much
information makes the computations take too much time. This conflict is why
this work is important for the sequencing community, because the results of
this work can be used prior to the sequencing process to estimate the amount of
data needed, which are input parameters for sequencing.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">It is assumed that the positions of reads within
the genome of a sequencing process are distributed equally over the genome.
Although this assumption might not fit to all sequencing techniques, this
simplification allows us to deduce manageable formulas for calculating the
contig length distribution using only the parameters of sequence length, read
count, and read length. From this general formula reflecting general knowledge
about a possibly optimal assembly, a formula calculating N50 is deduced.</p>

<h3>Results</h3>

<p class="abresults">Theoretical results have been shown to be the same as random
data for fixed input parameters. Furthermore, these results have been compared
to real data from the human genome. From this it was deduced that real data are
much worse in terms of having a higher number of sections with no coverage.
Finally, it is shown that the given formulas can estimate the expected contig
length distribution very precisely. Even the calculated N50 value was shown to
give good estimations of real data or, depending on the read data’s quality,
give an upper bound for a best-case assembly.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A mathematical model to precalculate the contig length
distribution for equally distributed reads has been proposed and been proven to
be correct by simulation. It has been shown that good predictions even for real
biological data can be made. Because read positions generally are not
distributed equally, another approach is to take different distributions to fit
real data even more exactly than the proposed model. Besides standard
distribution functions an interesting point of research would be to try to
define a distribution function by the inverse transformation method.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Florian Schatz (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54922 = 'm&#97;&#105;l' + '&#64;';
 addy54922 = addy54922 + 'fl&#111;r&#105;&#97;nsch&#97;tz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy54922 + suffix + '\'' + attribs + '>' );
 document.write( addy54922 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58449 = 'm&#97;&#105;l' + '&#64;';
 addy58449 = addy58449 + 'fl&#111;r&#105;&#97;nsch&#97;tz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy58449 + suffix + '\'' + attribs + '>' );
 document.write( addy58449 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79343 = 'm&#97;&#105;l' + '&#64;';
 addy79343 = addy79343 + 'fl&#111;r&#105;&#97;nsch&#97;tz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy79343 + suffix + '\'' + attribs + '>' );
 document.write( addy79343 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Christian-Albrechts-Universität zu Kiel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Christian-Albrechts-Universität zu Kiel, Department
of Computer Science</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection29">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741032">A-28. TAPIR: a web
server for the prediction of plant microRNA targets, including target mimics</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bonnet E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bonnet E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), He Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;He Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Billiau K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Billiau K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">MicroRNAs (miRNAs) constitute a prominent class of small
non-coding RNAs that regulate gene expression at the post-transcriptional level.
MiRNA targets can be identified through base pair complementarity of the miRNA
sequence to mRNA sequences. Recent reports have also revealed the existence in
plants of a phenomenon called miRNA target mimicry, where duplexes with a large
bulge around the cleavage site actually sequester miRNAs, inhibiting the miRNA
activity. We have designed and implemented TAPIR, a novel web server dedicated
to the prediction of plant miRNAs, including miRNA target mimics.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For the detection of miRNA:mRNA duplexes, we use
two different previously published algorithms. The first is the classical FASTA
local alignment program, which is very fast but cannot detect the duplexes
having a lot of bulges and/or mismatches. The second algorithm is RNAhybrid, an
algorithm for a precise detection of the miRNA:mRNA duplexes.</p>

<h3>Results</h3>

<p class="abresults">We tested the sensitivity and specificity of the TAPIR web
server using a refernce set of 102 experimentally verified plant miRNA-target
pairs. We also compared the results to existing web servers performing the same
task. The TAPIR web server is performing equally well or even slightly better
than the other tools, and seems to have a more flexible and user-friendly
interface. The other tools don’t offer the ability to predict target mimics.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The TAPIR web server is bringing new features compared to
existing solutions. The ability to use two different search engines, the rich
output results featuring a precise calculation of the free energy and free
energy ratio, the possibility to look for target mimics and to use pre-computed
results should make TAPIR a useful resource for the plant research community.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinformatics.psb.ugent.be/webtools/tapir" mce_href="http://bioinformatics.psb.ugent.be/webtools/tapir">http://bioinformatics.psb.ugent.be/webtools/tapir</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Eric Bonnet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7571 = '&#101;r&#105;c.b&#111;nn&#101;t' + '&#64;';
 addy7571 = addy7571 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy7571 + suffix + '\'' + attribs + '>' );
 document.write( addy7571 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4214 = '&#101;r&#105;c.b&#111;nn&#101;t' + '&#64;';
 addy4214 = addy4214 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy4214 + suffix + '\'' + attribs + '>' );
 document.write( addy4214 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97110 = '&#101;r&#105;c.b&#111;nn&#101;t' + '&#64;';
 addy97110 = addy97110 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy97110 + suffix + '\'' + attribs + '>' );
 document.write( addy97110 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Plant Systems Biology, VIB,
Technologiepark 927, B-9052 Gent, Belgium. (2) Department of Molecular
Genetics, Ghent University, Technologiepark 927, B-9052 Gent, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by an Innovatie door Wetenschap
en Technologie (IWT) grant for the Bioframe project and by an IUAP grant for
the BioMaGNet project (ref. p6/25). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection30">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741033">A-29. GRID/CPCA analysis
of the SULTs superfamily: multivariate characterization of the most relevant
structural and physicochemical differences</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rebollido-Rios R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rebollido-Rios R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2*), Zamora-Rico I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zamora-Rico I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Sulfation plays an important role in the second phase
metabolism. In this work provide a multivariate characterization of the SULTs
superfamily, in order to identify the most relevant structural differences that
could be responsible for the selective biding affinity and could be used in the
design of new potential ligands determinants for drug metabolism. To achieve
this goal, we employed the GRID/CPCA computational procedure; which has shown
to be an able tool to distinguish selectivity differences from the point of
view of the receptor. Consequently, it is independent of the availability of
appropriate ligands for a based QSAR analysis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The methodology involves the following
steps:(1)Obtain 3D structures of the target proteins.(2)Produce a superposition
of the well known conserved regions, in this particular case the (TYPKSTTW)
sequence motif, and also of the binding sites.(3) Get a multivariate
characterization of these regions derived from the GRID force field.(4)Use CPCA
to handle the multivariate data.(5)Graphical analysis and interpretations of
the results of the CPCA.</p>

<h3>Results</h3>

<p class="abresults">CPCA analysis of the SULT superfamily: In general, the CPCA
plot reveals the different subfamilies of SULTs that were included in this
work: SULT1, SULT2, SULT4. CPCA analysis interspecies (mSULTs and hSULTs): The
replacement of a few amino acids that are involved in the ligand binding site
can make a large difference between mouse and human SULTs. The aromatic
residues Phe 81 y Phe 142 have a major role in the phenol recognition. CPCA
analysis of hSULT1 and hSULT2 families: The CPCA procedure was able to
distinguish between the isoforms of both families.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The overall aim of this study was to compare the SULTs
superfamily structurally. This was done in order to identify the most relevant
structural differences that could be responsible for the selective binding
affinity. Despite of mSULT and hSULT sharing some sequence identity, CPCA shows
important structural differences regarding to some key amino acids that are
involved in the ligand recognition. The presence of the substrate in the active
site plays an important role in the determination of some structural
differences between SULTs isoforms. We can conclude that it is a computational
tool that allows distinguishing structural differences between these enzymes.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Rocio Rebollido-Rios (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11875 = 'r&#111;c&#105;&#111;.r&#101;b&#111;ll&#105;d&#111;-r&#105;&#111;s' + '&#64;';
 addy11875 = addy11875 + 'st&#117;d' + '&#46;' + '&#117;n&#105;-d&#117;&#101;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy11875 + suffix + '\'' + attribs + '>' );
 document.write( addy11875 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37891 = 'r&#111;c&#105;&#111;.r&#101;b&#111;ll&#105;d&#111;-r&#105;&#111;s' + '&#64;';
 addy37891 = addy37891 + 'st&#117;d' + '&#46;' + '&#117;n&#105;-d&#117;&#101;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy37891 + suffix + '\'' + attribs + '>' );
 document.write( addy37891 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37766 = 'r&#111;c&#105;&#111;.r&#101;b&#111;ll&#105;d&#111;-r&#105;&#111;s' + '&#64;';
 addy37766 = addy37766 + 'st&#117;d' + '&#46;' + '&#117;n&#105;-d&#117;&#101;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy37766 + suffix + '\'' + attribs + '>' );
 document.write( addy37766 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dept. of Bioinformatics, Center of Medical
Biotechnology.University of Duisburg-Essen</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1.Department of Molecular Design and Synthesis. Higher
Institute of Technologies and Applied Sciences. Ciudad de La Habana. CP
10600,AP 6163,Cuba. 2.Dept. of Bioinformatics.Center of Medical Biotechnology.University
of Duisburg-Essen. 45117. Essen. Germany 3.Lead Molecular Design, S.L., Vallés
96-102 (27). 08190. San Cugat del Vallés. Municipal Institute for Medical
Research. IMIM. Pompeu Fabra University. 08003. Barcelona. Spain. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection31">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741034">A-30. Theoretical and
empirical quality assessment of transcription factor binding motifs</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Medina-Rivera A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Medina-Rivera A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3*), Abreu-Goodger C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Abreu-Goodger C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Thomas-Chollier M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thomas-Chollier M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Salgado-Osorio H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Salgado-Osorio H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Collado-Vides J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Collado-Vides J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), van Helden J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Helden J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">Position-specific scoring matrices are routinely used to
predict transcription factor (TF) binding sites in genome sequences. However,
their reliability to predict novel binding sites can be far from optimum, due
to the use of a small number of training sites or the inappropriate choice of
parameters when building the matrix or when scanning sequences with it.
Measures of matrix quality such as E-value and information content rely on
theoretical models, and may fail in the context of full genome sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The program matrix-quality combines theoretical and
empirical score distributions to assess the predictive capability of
position-specific matrices. The theoretical distribution provides an estimate
of the false prediction rate. Empirical distributions indicate the enrichment
of binding sites in various collections of sequences: known binding sites
(positive control), all upstream regions of a genome, microarray clusters,
ChIP-seq peaks. Negative controls are performed by analyzing the same sequence
collections with column-permuted matrices.</p>

<h3>Results</h3>

<p class="abresults">We applied the method to estimate the predictive capacity of
matrices for bacterial, yeast and mouse transcription factors. The evaluation
of 60 matrices from RegulonDB revealed some poorly predictive motifs, and
allowed us to quantify the improvements obtained by applying multi-genome motif
discovery. Interestingly, the results reveal differences between global and
specific regulators. It also highlights the enrichment of binding sites in
sequence sets obtained from high-throughput ChIP-chip (bacterial and yeast
TFs), and ChIP-seq and experiments (mouse TFs).</p>

<h3>Discussion</h3>

<p class="abdiscussion">Users are often restricted by the available databases,
which contain motifs of variable quality. In this context, the method presented
here has many applications: (i) selecting reliable motifs before scanning
sequences; (ii) improving motif collections in transcription factor databases;
(iii) evaluating motifs discovered in massive datasets resulting from new
sequencing technologies (ChIP-seq, ChIP-chip), to cite a few.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://rsat.ulb.ac.be/rsat/" mce_href="http://rsat.ulb.ac.be/rsat/">http://rsat.ulb.ac.be/rsat/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Alejandra E Medina Rivera (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11463 = '&#97;m&#101;d&#105;n&#97;' + '&#64;';
 addy11463 = addy11463 + 'lcg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy11463 + suffix + '\'' + attribs + '>' );
 document.write( addy11463 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84063 = '&#97;m&#101;d&#105;n&#97;' + '&#64;';
 addy84063 = addy84063 + 'lcg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy84063 + suffix + '\'' + attribs + '>' );
 document.write( addy84063 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66992 = '&#97;m&#101;d&#105;n&#97;' + '&#64;';
 addy66992 = addy66992 + 'lcg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy66992 + suffix + '\'' + attribs + '>' );
 document.write( addy66992 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Genomic Sciences, Universidad Nacional
Autonoma de Mexico</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Centro de Ciencias Genómicas, Universidad Nacional
Autónoma de México. Av. Universidad s/n. Cuernavaca, Col. Chamilpa, Morelos
62210; Mexico. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24670 = '&#97;m&#101;d&#105;n&#97;' + '&#64;';
 addy24670 = addy24670 + 'lcg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy24670 + suffix + '\'' + attribs + '>' );
 document.write( addy24670 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>, 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10090 = 'h&#101;l&#97;d&#105;&#97;' + '&#64;';
 addy10090 = addy10090 + 'ccg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy10090 + suffix + '\'' + attribs + '>' );
 document.write( addy10090 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>,

 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93691 = 'c&#111;ll&#97;d&#111;' + '&#64;';
 addy93691 = addy93691 + 'ccg' + '&#46;' + '&#117;n&#97;m' + '&#46;' + 'mx';
 document.write( '<a ' + path + '\'' + prefix + addy93691 + suffix + '\'' + attribs + '>' );
 document.write( addy93691 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 2. EMBL - European Bioinformatics Institute, Wellcome Trust
Genome Campus, Hinxton, Cambridge, CB10 1SD, UK. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76430 = 'c&#101;&#105;' + '&#64;';
 addy76430 = addy76430 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy76430 + suffix + '\'' + attribs + '>' );
 document.write( addy76430 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 3.
Laboratoire de Bioinformatique des Génomes et des Réseaux (BiGRe). Université
Libre de Bruxelles, Campus Plaine, CP 263. Bld du Triomphe. B-1050 Bruxelles,
Belgium. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60332 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy60332 = addy60332 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy60332 + suffix + '\'' + attribs + '>' );
 document.write( addy60332 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 4. Department of Computational
Molecular Biology. Max Planck Institute for Molecular Genetics. Ihnestrasse 73.
14195 Berlin. Germany. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29696 = 'th&#111;m&#97;s-c' + '&#64;';
 addy29696 = addy29696 + 'm&#111;lg&#101;n' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy29696 + suffix + '\'' + attribs + '>' );
 document.write( addy29696 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge the members of the BiGRe laboratory for
useful comments on the manuscript. Funding: A.M-R. was supported during her
Ph.D. studies (Programa de Doctorado en Ciencias Biomédicas, Universidad
Nacional Autónoma de México) by a fellowship from the Consejo Nacional de
Ciencia y Tecnología (Mexico). C.A-G. was supported by a Sanger Institute
Postdoctoral Fellowship. The BiGRe laboratory is supported by the BioSapiens
Network of Excellence funded under the sixth Framework program of the European
Communities (LSHG-CT-2003-503265), by the Belgian Program on Interuniversity
Attraction Poles, initiated by the Belgian Federal Science Policy Office,
project P6/25 (BioMaGNet) and by the MICROME Collaborative Project funded by
the European Commission within its FP7 Programme, under the thematic area
"BIO-INFORMATICS - Microbial genomics and bio-informatics", contract
number 222886-2.". Travel costs for A.M.-R. were partly supported by the
Actions de Recherches Concertées de la Communauté Française de Belgique (ARC
grant number 04/09-307), the Bureau des Relations Internationales et de
Coopération (BRIC, Université Libre de Bruxelles) and UNAM. J.C.-V.
acknowledges support by the National Institutes of Health, grant number R01
GM071962-05. MTC is supported by the Alexander von Humbold Stiftung. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection32">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741035">A-31. Characterization
and prediction of protein nucleolar localization sequences</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Scott MS<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Scott MS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Boisvert F-M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Boisvert F-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), McDowall MD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;McDowall MD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lamond AI<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lamond AI&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Barton GJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Barton GJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The nucleolus is the cellular site of ribosomal subunit
assembly but is also known to be a key player in numerous other fundamental
cellular activities. Although the nucleolar localization of proteins is often
believed to be mediated primarily by non-specific retention to core nucleolar
components, many examples of short nucleolar targeting sequences have been
reported in recent years. These motifs are referred to as NoLSs (Nucleolar
Localization Sequences).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To characterize NoLSs, we performed an extensive
curation of the literature, uncovering 46 human NoLSs, and performed
statistical analysis on these motifs. Then the sequence and predicted protein
secondary structure of the curated NoLSs and a carefully generated negative set
which included nuclear localization signals (NLSs) were used to train an
artificial neural network for the prediction of NoLSs. The predictor was
applied to the complete human proteome and ten of the highest scoring
previously unknown NoLSs were experimentally confirmed by fusion to GFP and
visualization by microscopy.</p>

<h3>Results</h3>

<p class="abresults">Statistical analysis of the NoLSs revealed that 48% of their
residues are basic, while 99% of their residues are predicted as
solvent-accessible. These and other features were used to train an artificial
neural network NoLS predictor. At a true positive rate of 54%, the predictor’s
overall false positive rate (FPR) was estimated to be 1.52%, which can be
broken down to FPRs of 0.26%, 0.80% and 12% for randomly chosen cytoplasmic,
nucleoplasmic and NLS sequences respectively. All previous unknown predicted NoLSs
chosen for validation were experimentally confirmed as localizing to the
nucleolus.</p>

<h3>Discussion</h3>

<p class="abdiscussion">NoLSs are a prevalent type of targeting motif that can be
computationally predicted. We predict 19% of human proteins to encode NoLSs.
Although recognized as different in the cell, the NoLS is a targeting signal
that has a similar composition to NLSs and both signals are often confused or
used interchangeably. This study should help define the differences between the
two types signals. The proteome-wide prediction of NoLSs may be searched and
downloaded from http://www.compbio.dundee.ac.uk/www-nod/.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.compbio.dundee.ac.uk/www-nod/" mce_href="http://www.compbio.dundee.ac.uk/www-nod/">http://www.compbio.dundee.ac.uk/www-nod/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Michelle S. Scott (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29585 = 'm&#105;ch&#101;ll&#101;' + '&#64;';
 addy29585 = addy29585 + 'c&#111;mpb&#105;&#111;' + '&#46;' + 'd&#117;nd&#101;&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy29585 + suffix + '\'' + attribs + '>' );
 document.write( addy29585 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39094 = 'm&#105;ch&#101;ll&#101;' + '&#64;';
 addy39094 = addy39094 + 'c&#111;mpb&#105;&#111;' + '&#46;' + 'd&#117;nd&#101;&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy39094 + suffix + '\'' + attribs + '>' );
 document.write( addy39094 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63045 = 'm&#105;ch&#101;ll&#101;' + '&#64;';
 addy63045 = addy63045 + 'c&#111;mpb&#105;&#111;' + '&#46;' + 'd&#117;nd&#101;&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy63045 + suffix + '\'' + attribs + '>' );
 document.write( addy63045 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Division of Biological Chemistry and Drug Discovery ,
College of Life Sciences, University of Dundee</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Division of Biological Chemistry and Drug Discovery
and (2) Wellcome Trust Centre for Gene Regulation and Expression, College of
Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, United Kingdom</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">MSS is a recipient of a post-doctoral fellowship from
the Caledonian Research Foundation. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection33">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741036">A-32. Jalview: past,
present, future</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Procter JB<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Procter JB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Troshin PV<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Troshin PV&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Martin MAM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Martin MAM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Barton GJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Barton GJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Sequence alignments are essential to comparative sequence
analysis, and they provide a scaffold for highlighting variation. Jalview
provides a light weight and flexible sequence alignment visualization
environment, either as a web based or stand-alone system. However, new
annotation capabilities are needed to better visualize quantitative annotation
- such as amino acid profiles and positional scores derived from computational
prediction and experimental observations. Furthermore, a new web service
infrastructure is needed to enable users to exploit their own computational
resources.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The Jalview visualization pipeline and datamodel
were extended to support the display of alignments overlaid with quantitative
sequence annotation. A new program execution engine and parameter specification
model was developed in Java 1.6, which is able to manage local and distributed
job execution. The engine provides a web service interface layer and its client
was embedded in the Jalview desktop. Jalview's user interface was also extended
to support the configuration of multiple web service installations.</p>

<h3>Results</h3>

<p class="abresults">Jalview's new annotation visualization capabilities were
evaluated with data retrieved from public servers via the Distributed
Annotation System (DAS), and General Feature Format (GFF) files. The new
interface enables regions to be shaded according to score or any associated
text to better distinguish annotation of the same type. Several multiple
sequence alignment programs were configured for execution using the new web
service execution engine. Jalview users are able to access a public
installation of the engine, or download them for local installation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The extensions to Jalview's alignment, visualization, and
analysis capabilities that we described enable it to be a more useful system
for biologists, and expert bioinformaticians. We wish to extend the repertoire
of analysis types accessible via the new Jalview web services system, to
include sequence database searches and automated alignment analysis and
annotation tools. The new capabilities also introduce new levels of complexity
for the user. Therefore, we are now developing new training materials to help
biologists and bioinformaticians effectively exploit Jalview's new
capabilities.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.jalview.org" mce_href="http://www.jalview.org">http://www.jalview.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">James B. Procter (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38805 = 'j.pr&#111;ct&#101;r' + '&#64;';
 addy38805 = addy38805 + 'd&#117;nd&#101;&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy38805 + suffix + '\'' + attribs + '>' );
 document.write( addy38805 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy80461 = 'j.pr&#111;ct&#101;r' + '&#64;';
 addy80461 = addy80461 + 'd&#117;nd&#101;&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy80461 + suffix + '\'' + attribs + '>' );
 document.write( addy80461 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98321 = 'j.pr&#111;ct&#101;r' + '&#64;';
 addy98321 = addy98321 + 'd&#117;nd&#101;&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy98321 + suffix + '\'' + attribs + '>' );
 document.write( addy98321 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Dundee</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. School of Life Sciences Research, College of Life
Sciences, University of Dundee, UK.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Jalview’s core development is supported by the UK’s
Biotechnology and Biological Sciences Research Council (BBSRC). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection34">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741037">A-33. Enrichment of
eukaryotic linear motifs in viral proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Pushker R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Pushker R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Edwards R-J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Edwards R-J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Shields D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shields D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shields D-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Eukaryotic Linear Motifs (ELMs) in proteins are short and
linear motifs spanning 3 to 10 amino acids that may play an important role in
virus-host interaction and subversion of host cell signaling. These ELMs have been
found to be more prevalent in the disordered regions (lacking secondary or
tertiary structure) of proteins. Since many viral genomes display a propensity
for intrinsic disorder, we expect that there would be an enrichment of these
ELMs in viruses. We were also interested to see whether disordered regions are
more enriched with ELMs than ordered regions of viral proteins.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">119 ELMs were searched in the ordered and
disordered regions of 1,646 viruses consisting of 33,269 proteins. For each
ELM, the number of occurrences in all proteins was recorded and compared with
that of control ELMs; typically the reversed sequences. Chi-square test with
Bonferroni correction (p &lt; 4.2E-4; this threshold allows for multiple
testing) was performed to test whether certain ELMs are over-represented or
under-represented in viruses. Since the statistical test relies on
non-independence of the data; we also searched for these ELMs in evolutionary
Unrelated Protein Clusters (UPCs).</p>

<h3>Results</h3>

<p class="abresults">We found that 23 ELMs were over-represented in the ordered
region of viral proteins. Proprotein convertase subtilisin/kexin (PCSK)
cleavage site motif, TRAF2 binding site motif and SH2 ligand binding motif were
the most significantly over-represented ELMs. Integrin binding site motif was
the most significant one among the 12 under-represented ELMs. After searching
for ELMs in the UPCs, we found that the ordered regions of viral proteins were
enriched with an ELM that binds to Proliferating Cell Nuclear Antigen (PCNA)
protein and forms an alpha helix on binding.</p>

<h3>Discussion</h3>

<p class="abdiscussion">There were 14 ELMs significantly over-represented;
coincidently, PCSK cleavage site motif, being the most significant one in the
disordered regions of viral proteins. Analyzing the disordered regions of UPCs,
we found that two PDZ ligand binding motifs were still significantly
over-represented in these UPCs. Interestingly, the N-myristoylation site motif
was found to be under-represented in the disordered region of UPCs. We conclude
that there is an evidence for enrichment of PCNA binding motif in the ordered
region and PDZ ligand binding motifs in the disorder region of viral proteins.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Ravindra Pushker (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87634 = 'p&#117;shk&#101;r' + '&#64;';
 addy87634 = addy87634 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy87634 + suffix + '\'' + attribs + '>' );
 document.write( addy87634 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27630 = 'p&#117;shk&#101;r' + '&#64;';
 addy27630 = addy27630 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy27630 + suffix + '\'' + attribs + '>' );
 document.write( addy27630 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35210 = 'p&#117;shk&#101;r' + '&#64;';
 addy35210 = addy35210 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy35210 + suffix + '\'' + attribs + '>' );
 document.write( addy35210 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University College Dublin, Ireland</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. UCD Complex and Adaptive Systems Laboratory; UCD
Conway Institute of Biomolecular &amp; Biomedical Research; School of Medicine
and Medical Science, University College Dublin, Belfield, Dublin 4, Ireland. 2.
School of Biological Sciences, University of Southampton, Southampton, UK.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by Science Foundation Ireland and
University College Dublin, Ireland. We also wish to acknowledge the SFI/HEA
Irish Centre for High-End Computing (ICHEC) for providing computational
facilities and support. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection35">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741038">A-34. Analyzing ChIP-Seq
data using qips</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gogol-Döring A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gogol-Döring A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Chen W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chen W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Chromatin immunoprecipitation followed by high-throughput
DNA sequencing (ChIP-Seq) is a common method for the genome-wide analysis of
protein-DNA interactions. Specific antibodies select the proteins of interest
together with any piece of randomly fragmented DNA bound to them. The origin of
the selected DNA fragments is then determined by sequencing and mapping to a
reference genome; genomic regions binding to proteins feature an increased
number of mapped sequencing reads.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">qips computes a set I of nonoverlapping and high-scoring
intervals which is optimal, i.e. for each alternative interval j exists an
overlapping interval in I having at least the same score as j. The scores are
derived from a statistical model that implies the mappability of genomic
regions and, if available, the read distribution of a control data set. Our
software also provides a new and accurate way for estimating the length
distribution of single-end sequenced DNA-fragments which does not require
distinctive peaks and is robust with respect to noise and mapping biases.</p>

<h3>Results</h3>

<p class="abresults">Calculating p-Values for intervals of arbitrary length and
without fixed starting positions makes our method more precise than alternative
approaches.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our software generates several figures and statistical
output valuable for evaluating the overall quality of ChIP-Seq data. It will be
published free and open source.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Andreas Gogol-Döring (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67453 = '&#97;ndr&#101;&#97;s.d&#111;&#101;r&#105;ng' + '&#64;';
 addy67453 = addy67453 + 'mdc-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy67453 + suffix + '\'' + attribs + '>' );
 document.write( addy67453 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64292 = '&#97;ndr&#101;&#97;s.d&#111;&#101;r&#105;ng' + '&#64;';
 addy64292 = addy64292 + 'mdc-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy64292 + suffix + '\'' + attribs + '>' );
 document.write( addy64292 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38292 = '&#97;ndr&#101;&#97;s.d&#111;&#101;r&#105;ng' + '&#64;';
 addy38292 = addy38292 + 'mdc-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38292 + suffix + '\'' + attribs + '>' );
 document.write( addy38292 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Berlin Institute for Medical Systems Biology,
MDC-Berlin</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Berlin Institute for Medical Systems Biology,
MDC-Berlin</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection36">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741039">A-35. COMA server for
protein distant homology search</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Margelevicius M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Margelevicius M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Laganeckas M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laganeckas M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Venclovas C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Venclovas C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The concept of homology (common evolutionary origin) is
at the heart of most studies dealing with protein sequence, structure and
function. In the absence of protein structure, inference of homology usually
has to rely exclusively on sequence data. At present, most sensitive
sequence-based methods use comparison of multiple sequence alignments
represented as either Hidden Markov Models or sequence profiles.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">With the goal of improving distant homology detection,
we have recently developed a new method COMA, based on sequence profile-profile
comparison (Margelevicius and Venclovas, 2010, BMC Bioinformatics 11, 89). The
new method has at least two major features distinguishing it from other
profile-profile comparison methods. They comprise a position-specific variable
gap penalties, and a global score system leading to improved estimation of
statistical significance of detected similarities. To make this method
accessible for larger biological community, we have developed a web server.</p>

<h3>Results</h3>

<p class="abresults">The server has a simple and intuitive user interface for
setting up homology search jobs, and uses the latest version of the COMA
method. The actual homology search is done by comparing query-based sequence
profile against the selected profile database (SCOP, PDB, and PFAM). The
structured output of the homology search is enriched with useful functionality.
The user may extract the combined alignment between the query and any number of
hits for further analysis. The server also provides a possibility to generate a
3D model of the query based on the corresponding alignment.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To illustrate the utility of the COMA server for distant
homology detection, we present newly discovered evolutionary links to the
highly diverse PD-(D/E)XK nuclease superfamily. We also provide some examples
where COMA performs better than a structure comparison method. The provided
illustrations demonstrate the COMA server's utility in inferring structure and
function of specific protein families, and non-trivial evolutionary
relationships.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinformatics.ibt.lt/coma/" mce_href="http://bioinformatics.ibt.lt/coma/">http://bioinformatics.ibt.lt/coma/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Mindaugas Margelevicius (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40374 = 'm&#105;nm&#97;r' + '&#64;';
 addy40374 = addy40374 + '&#105;bt' + '&#46;' + 'lt';
 document.write( '<a ' + path + '\'' + prefix + addy40374 + suffix + '\'' + attribs + '>' );
 document.write( addy40374 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19214 = 'm&#105;nm&#97;r' + '&#64;';
 addy19214 = addy19214 + '&#105;bt' + '&#46;' + 'lt';
 document.write( '<a ' + path + '\'' + prefix + addy19214 + suffix + '\'' + attribs + '>' );
 document.write( addy19214 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96741 = 'm&#105;nm&#97;r' + '&#64;';
 addy96741 = addy96741 + '&#105;bt' + '&#46;' + 'lt';
 document.write( '<a ' + path + '\'' + prefix + addy96741 + suffix + '\'' + attribs + '>' );
 document.write( addy96741 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Biotechnology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Institute of Biotechnology, Graiciuno 8, LT-02241
Vilnius, Lithuania</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by Lithuanian State Science and
Studies Foundation and Howard Hughes Medical Institute. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection37">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741040">A-36. Protein disorder
and short motifs in disordered regions are enriched near the cytoplasmic side
of single-pass transmembrane proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Stavropoulos I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Stavropoulos I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2*), Khaldi N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Khaldi N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2), Davey NE<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Davey NE&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Finian M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Finian M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Shields D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shields D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shields DC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2)</p>

<p class="abmotivation">We investigated the distribution of predicted intrinsic
disorder in human single-pass transmembrane proteins in relation to the plasma
membrane. The proteins were also studied to identify potentially functional
short linear motifs, by searching for regions in the protein sequences that are
both intrinsically disordered and relatively conserved compared to adjacent
residues in orthologous proteins. The distribution of the predicted motifs was
also investigated in relation to the plasma membrane.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">All 803 single-pass human transmembrane proteins
were analyzed using IUPred to predict disorder. Gopher algorithm calculated the
relative local conservation for every residue in the proteins and by combining
information from disorder prediction identified regions of relative
conservation in disordered regions. As disordered regions are devoid of
structural constraints, conservation of residues relative to a background of
random mutation is generally due to functional relevance. Using this method,
the protein sequences were searched for statistically over conserved groupings
of residues.</p>

<h3>Results</h3>

<p class="abresults">Human single-pass proteins have higher mean disorder in
their cytoplasmic segments than their extracellular parts. Since the region
immediately adjacent to the membrane is less disordered, the disorder peaks at
around 30 residues from the membrane. We noted a marked increase in the
incidence of conserved motifs within the disordered regions at the same
location, suggesting that this area may be extensively used in protein-protein
interactions. We noted that this excess of conserved motifs is seen even after
adjusting for the disorder level.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We conclude that many transmembrane proteins have highly
disordered cytoplasmic tails, and that these may play roles in cellular
signalling.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Ilias Stavropoulos (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19716 = '&#105;l&#105;&#97;s.st&#97;vr&#111;p&#111;&#117;l&#111;s' + '&#64;';
 addy19716 = addy19716 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy19716 + suffix + '\'' + attribs + '>' );
 document.write( addy19716 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26785 = '&#105;l&#105;&#97;s.st&#97;vr&#111;p&#111;&#117;l&#111;s' + '&#64;';
 addy26785 = addy26785 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy26785 + suffix + '\'' + attribs + '>' );
 document.write( addy26785 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy954 = '&#105;l&#105;&#97;s.st&#97;vr&#111;p&#111;&#117;l&#111;s' + '&#64;';
 addy954 = addy954 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy954 + suffix + '\'' + attribs + '>' );
 document.write( addy954 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University College Dublin</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1-UCD Conway Institute of Biomolecular and Biomedical
Research, School of Medicine and Medical Sciences, University College Dublin,
Dublin 4, Republic of Ireland. 2-UCD Complex and Adaptive Systems Laboratory, University
College Dublin, Dublin 4, Republic of Ireland. 3-European Molecular Biology
Laboratory, Heidelberg, Germany.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by the Irish Research Council for
Science, Engineering &amp; Technology (IRCSET) Graduate Education Research
Programme (http://bioinformatics.ucd.ie/PhD). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection38">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741041">A-37. Variation in
positional preferences by 20 amino acids in alpha helices with different size</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Fallahi H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Fallahi H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Dokanehiifard S-A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dokanehiifard S-A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Hosseini- Nasab S-M-E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hosseini- Nasab S-M-E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Many studies indicated that different amino acids have
different preferences for taking part in alpha helix conformation. As a result
of these studies, substitution matrices have been prepared to compare the
proteins secondary and tertiary structures. Unfortunately, in many cases these
matrixes are unable to produce reliable results due to the complexity of the
conformation. In order to improve prediction of the secondary structures based
on the peptide sequences, a more detailed look at these helices is inevitable.
Here we present the result of dissecting different alpha helixes by position.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The amino acid sequences of proteins containing
alpha helixes were obtained from PDB. Alpha helixes in these proteins were then
categorized based on the number of amino acids into classes with 3 to 25 amino
acids. The amino acid contents of each class and each position within the
classes were then determined. Amino acid contents of the same position in different
classes were also compared. Using PERL language programming all the scripts
created and tested locally on a Mac operating system (Mac OSX ver.10.5.8 ).</p>

<h3>Results</h3>

<p class="abresults">The results indicate differences in the preferences of
different amino acids for each position. For example, while alanine did not
show significant tendency toward any position, histidine were mostly present in
positions 18 and afterwards. Besides, the same positions in different classes
of helixes were occupied by different amino acids. As an example, position two
in the 4-aa classes of helixes was mostly occupied by proline, followed by
alanine. Interestingly, in the 5-aa helixes the same position was contained
mostly glutamic acid followed by alanine and proline was the third.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The result of the current study might be an indication of
greater differences among helices with different number of amino acids.
Therefore, one might need separate substitution matrices for each class for
alignment and comparison studies. Currently used matrices are not effective,
because AA substitutions are regarded to be the same in all helixes. These
results might also imply alternative evolutionary route for different sized
helixes. The longer helixes could be a result of merging the smaller one
together in different combinations, rather than simple expansion of the smaller
helices.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Hossein Fallahi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16826 = 'f&#97;ll&#97;h&#105;.h&#111;ss&#101;&#105;n' + '&#64;';
 addy16826 = addy16826 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy16826 + suffix + '\'' + attribs + '>' );
 document.write( addy16826 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38659 = 'f&#97;ll&#97;h&#105;.h&#111;ss&#101;&#105;n' + '&#64;';
 addy38659 = addy38659 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy38659 + suffix + '\'' + attribs + '>' );
 document.write( addy38659 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38845 = 'f&#97;ll&#97;h&#105;.h&#111;ss&#101;&#105;n' + '&#64;';
 addy38845 = addy38845 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy38845 + suffix + '\'' + attribs + '>' );
 document.write( addy38845 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Razi Unversity, Kermanshah, IRAN (I.R. of))</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Dep. of Biology, School of Science, Razi University,
Kermanshah, I.R. of Iran 2. Dep. of Statistics, Shahid Beheshti University,
Evin, Tehran, I.R. of Iran * corresponding author</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by Razi University,
through a research fund provided to Dr. Hossein Fallahi. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection39">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741042">A-38. Comparison of
mapping and variant calling accuracies for next-generation sequence data in
relation to coverage depth: a case study in leukemia cell lines</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kalender Z<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kalender Z&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Geerdens E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Geerdens E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Cuppens H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cuppens H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Cools J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cools J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Aerts S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aerts S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">We found that combinations of different mapping and
variant calling algorithms lead to significantly different variant predictions.
Moreover, we have observed that most of the variant caller algorithms have a
coverage depth threshold of 10 reads, yet there might be high quality reads
with low coverages reflecting true variations, especially in targeted
sequencing studies. Thus, we aimed to identify the most accurate combination of
mapping and variation calling methods and to study how the performance of
different methods varies with coverage depth.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We re-sequenced the exons of 97 genes in eight
leukemia cell lines on the Roche 454 platform. Nine analysis pipelines are
constructed using different combinations of existing mapping and variant
algorithms. Sequence reads are aligned to the human reference genome using one
of the four mapping algorithms (BLAT,BWA-SW, SSAHA2, and GsMapper). Then,
single nucleotide variations are predicted using one of the four variant
calling algorithms (SAMTools,VarScan,Atlas-SNP2, and GsMapper). Furthermore, a
set of 192 predicted variations are sequenced with capillary sequencing to
create a validation set.</p>

<h3>Results</h3>

<p class="abresults">To assess how the coverage is affecting variant calling,
separate variant calling is performed for low coverage (3 to 10 reads per base,
LC) and high coverage (more than 10 reads per base, HC) regions. By comparing
the sensitivity (SN) and specificity (SP) between pipelines, we found that
different pipelines perform good in different coverages
(HC:BWA-SW+SAMTools;SN=99%,SP=87% and LC:BLAT+Atlas-SNP2 and SSAHA2+Atlas-SNP2;
SN=71%, SP=100%). Moreover, higher performance can be achieved using
combinations of high specificity pipelines in both LC and HC (HC:SN=92%, SP=98%
and LC:SN=79%, SP=100%).</p>

<h3>Discussion</h3>

<p class="abdiscussion">A new computational pipeline that combines different
combinations of mapping and variant calling algorithms leads to more accurate
and specific identification of genomic variants in cancer genomes. We show that
it is possible to extract valuable information even in LC regions. We believe
these results will contribute to the identification of high-confidence
mutations in cancer genomes, and we are currently using the SNV predictions
from our best pipeline to identify driver mutations in a cohort of re-sequenced
leukemia cell lines and patient samples.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Zeynep Kalender (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54592 = 'z&#101;yn&#101;p.k&#97;l&#101;nd&#101;r' + '&#64;';
 addy54592 = addy54592 + 'm&#101;d' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy54592 + suffix + '\'' + attribs + '>' );
 document.write( addy54592 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50122 = 'z&#101;yn&#101;p.k&#97;l&#101;nd&#101;r' + '&#64;';
 addy50122 = addy50122 + 'm&#101;d' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy50122 + suffix + '\'' + attribs + '>' );
 document.write( addy50122 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22907 = 'z&#101;yn&#101;p.k&#97;l&#101;nd&#101;r' + '&#64;';
 addy22907 = addy22907 + 'm&#101;d' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy22907 + suffix + '\'' + attribs + '>' );
 document.write( addy22907 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Laboratory of Computational Biology, Center for Human
Genetics, KULeuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Laboratory of Computational Biology, Center for Human
Genetics, KULeuven 2 Laboratory for Molecular Pathogenesis of Leukemia, Center
for Human Genetics, KULeuven 3 Laboratory for Cytogenetics and Genome Research,
Center for Human Genetics, KULeuven</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project was supported by the National Taskforce on
Cancer from the Belgian government. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection40">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741043">A-39. Monitoring strain
diversity in metagenomics data using meta-MLST SNP profiling</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Jager VCL<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Jager VCL&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,4,5*), Van der Sijde MR (1,2,3) Hazelwood
LA (1,3) Kutahya O (1,3), Kleerebezem M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kleerebezem M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,5) Smid EJ (1,3), Siezen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen RJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4), Van Hijum S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Hijum S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->AFT<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Hijum SAFT&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4)</p>

<p class="abmotivation">The diversity in many industrial and naturally occurring
metagenomes (e.g. a complex starter culture) with a limited set of species is
mostly represented by strain variants. Properties of such metagenomes may be
explained by the presence of combinations of such strains. It is therefore it
is important in these metagenomes to profile the diversity at the bacterial
strain level.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The method allows following bacterial diversity in
metagenomes at the strain level. It involves selecting “core” genes expected to
be present in all strains of interest. Next, single nucleotide polymorphisms
(SNPs) are determined that support strain classification. These SNPs are
subsequently used to assign strain classes to indvidual reads or contigs
obtained from metagenomic samples, similar to multi locus sequence typing
(MLST). We apply our method to follow strain level diversity of Lactococcus lactis
in multiple-timepoint metagenomics data obtained during the cheese making
process.</p>

<h3>Results</h3>

<p class="abresults">We obtained core clusters of genes from sequenced species
expected to be present in the metagenome of interest and from sequenced
individual isolated strains. SNPs were detected on these core genes and
separating SNPs were selected for read classification. We were able to assign
strain classes to individual reads and contigs from a multiple-timepoint
metagenomics dataset obtaining known and new varieties.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The ability to follow the presence of individual but
closely related bacterial strains in metagenomes allows to gain insight in the
dynamics of such populations. The combination of metagenome data and sequences
of selected individual isolates allows detection of known classes and new
strain variants.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Victor C.L. de Jager (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21583 = 'v&#105;ct&#111;r.d&#101;.j&#97;g&#101;r' + '&#64;';
 addy21583 = addy21583 + 'nb&#105;c' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy21583 + suffix + '\'' + attribs + '>' );
 document.write( addy21583 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74792 = 'v&#105;ct&#111;r.d&#101;.j&#97;g&#101;r' + '&#64;';
 addy74792 = addy74792 + 'nb&#105;c' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy74792 + suffix + '\'' + attribs + '>' );
 document.write( addy74792 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32996 = 'v&#105;ct&#111;r.d&#101;.j&#97;g&#101;r' + '&#64;';
 addy32996 = addy32996 + 'nb&#105;c' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy32996 + suffix + '\'' + attribs + '>' );
 document.write( addy32996 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">NBIC / CMBI / TIFN / WUR</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">De Jager V-C-L (2,4,*), Van der Sijde M-R (1,2,3)
Hazelwood L-A (1,3) Kutahya O (1,3), Kleerebezem M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kleerebezem M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,5) Smid E-J (1,3), Siezen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->-J (1,2,3,4), Van Hijum S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Hijum S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->-A-F-T (1,2,3,4) 1) NIZO food
research, P.O. Box 20, 6710 BA Ede, the Netherlands 2) Center for Molecular and
Biomolecular Informatics, Nijmegen Center for Molecular Life Sciences, Radboud
University Medical Centre, P.O. Box 9101, 6500 HB Nijmegen, the Netherlands 3)
TI Food and Nutrition, P.O. Box 557, 6700 AN Wageningen, the Netherlands 4)
Netherlands Bioinformatics Centre, 260 NBIC, P.O. Box 9101, 6500 HB Nijmegen,
the Netherlands 5) Wageningen University, Laboratory of Microbiology,
Dreijenplein 10, Buildingnumber 316, 6703 HB, Wageningen, The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Netherlands Bioinformatics Centre, Geert Grooteplein
28, 6525 GA Nijmegen, The Netherlands, programme: BioRange 2.4</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection41">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741044">A-40. A variance
stabilizing parametrization to detect differentially expressed genes in RNA-Seq
data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sales G<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Sales G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Risso D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Risso D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Chiogna M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chiogna M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Romualdi C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Romualdi C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Several studies have shown that deep sequencing provides
a new level of detail for the analysis of the transcriptome. Current protocols
for the processing of RNA-Seq data, however, are affected by a transcript
length bias in the calling of differentially expressed genes. This effect
introduces a distortion in the ranking of differentially expressed genes and
may affect adversely studies comparing the behavior of multiple genes (for
example, in the context of pathway analysis).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The Poisson regression offers a simple model to
deal with count data, such as that obtained from deep sequencing technologies.
In the context of the Generalized Linear Model framework, the choice of the
link function represents a crucial issue. Although the canonical link is often
the default choice, other links can present favorable properties. Here, we
explore the use of the square root as the variance stabilizing link for the
Poisson regression. We propose a Wald type test (equivalent to the classical
t-test within the Poisson model) to test for differentially expressed genes.</p>

<h3>Results</h3>

<p class="abresults">By means of simulations and real data (from the MAQC-III
project), we show that, in the Poisson model with the canonical link function,
the variance of the estimators of the parameters strongly depends on the mean
count level. By exploiting a variance stabilizing transformation of the
parameters, this dependency is removed without affecting the biological
interpretation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We propose a novel approach for the detection of
differentially expressed genes in RNA-Seq data. In particular, we have observed
that the transcript length dependency reported in other studies can be
explained almost completely by the more fundamental link between the intercept
and the variance of the estimators in the Poisson regression used to model
short reads data. Building on this conclusion, we have developed a variance stabilizing
parametrization compensating the bias and allowing the use of the standard
t-test for the identification of differentially expressed genes.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Gabriele Sales (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15274 = 'gbrs&#97;l&#101;s' + '&#64;';
 addy15274 = addy15274 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy15274 + suffix + '\'' + attribs + '>' );
 document.write( addy15274 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51221 = 'gbrs&#97;l&#101;s' + '&#64;';
 addy51221 = addy51221 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy51221 + suffix + '\'' + attribs + '>' );
 document.write( addy51221 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93328 = 'gbrs&#97;l&#101;s' + '&#64;';
 addy93328 = addy93328 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy93328 + suffix + '\'' + attribs + '>' );
 document.write( addy93328 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Biology, University of Padua</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Department of Biology, University of Padua, via U.
Bassi 58/B, 35121 Padova, Italy. 2) Department of Statistical Sciences,
University of Padua, via C. Battisti 241, 35121 Padova, Italy.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">University of Padua. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection42">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741045">A-41. DNA annotation
induction: from RefGene on human chr. 22 to genome-wide CAGE for human and
mouse</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bedo J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Bedo J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Steininger A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Steininger A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Izhak Haviv<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Izhak Haviv&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,5), Adam Kowalczyk<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Adam Kowalczyk&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Transcriptions start site (TSS) prediction on the human
genome (using RefGene) has recently been extensively studied as a benchmark
problem for such applications, and supervised learning techniques were demonstrated
as a very robust approach that outperformed other competing approaches. We show
that supervised techniques are capable of even more: they can extract novel
knowledge from a small sample of annotated DNA and build predictive models
capable of accurate annotation of the same or even another species,
generalising beyond the apparent scope of the initial annotation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach is to segment the genome into small
tiles assuming that some corresponding sets of binary labels are available for
the tiles. A small subset of tiles (e.g., a single chromosome) is used to train
a predictive model. The model is then applied to the rest of the genome. We
used different genome annotations: RefGene genes, CAGE tags and also broad,
unedited ChIP-Seq for Pol-II binding from the ENCODE project. Five different
predictive model were used -- three developed for human and two for mouse --
with each tested on both mouse and human.</p>

<h3>Results</h3>

<p class="abresults">We have shown that the knowledge of DNA sequences and
functional annotation on the smallest autosomal human ch. (22) is sufficient to
build robust genome-wide models for CAGE tag prediction on human and mouse. For
human we detect the top 10^4 tags with precision ~95%. Similar performance was
observed for four additional models: two human models from ChIP-Seq peaks for
Pol-II binding and CAGE tags, and two mouse models from RefGene and CAGE
annotations. We conclude that a significant fraction of TSS and CAGE tag
locations are associated with very similar local DNA properties.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We believe our algorithm provides a good baseline
in-silico tool for extending empirical data obtained during phase I of ENCODE
through to the rest of the genome, further to the TSS task we explored.
Furthermore, our predicted TSS annotations merits consideration by the human
ENCODE Genome Annotation Assessment Project (EGASP), and could improve our
annotation of functional elements in the context of interpretation of genetic
studies, such as genome wide disease-allelic associations.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Justin Bedo (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44970 = 'j&#117;st&#105;n.b&#101;d&#111;' + '&#64;';
 addy44970 = addy44970 + 'n&#105;ct&#97;' + '&#46;' + 'c&#111;m' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy44970 + suffix + '\'' + attribs + '>' );
 document.write( addy44970 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy80806 = 'j&#117;st&#105;n.b&#101;d&#111;' + '&#64;';
 addy80806 = addy80806 + 'n&#105;ct&#97;' + '&#46;' + 'c&#111;m' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy80806 + suffix + '\'' + attribs + '>' );
 document.write( addy80806 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32421 = 'j&#117;st&#105;n.b&#101;d&#111;' + '&#64;';
 addy32421 = addy32421 + 'n&#105;ct&#97;' + '&#46;' + 'c&#111;m' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy32421 + suffix + '\'' + attribs + '>' );
 document.write( addy32421 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">NICTA</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1: NICTA, Victoria Research Laboratory 2: Dept. of
Electrical and Electronic Engineering and 4: Dept. of Biochemistry and Molecular
Biology of The University of Melbourne VIC 3010, Australia 3: University of
Applied Sciences Technikum Wien, Vienna, Austria 5: The Alfred Medical Research
and Education Precinct, Baker Medical Research</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">JB and AK acknowledge the support of NICTA in
conducting this research. NICTA is funded by the Australian Government's
Department of Communications, Information Technology and the Arts, the
Australian Research Council through Backing Australia's Ability, and the ICT
Centre of Excellence programs. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection43">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741046">A-42. A new resampling
method</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bakis Y<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Bakis Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Sezerman OU<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sezerman OU&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Increase in molecular data has caused diversity of
methods in analysing this data. Among them, some recently introduced approaches
does not require an alignment procedure. However, it wouldn't be possible to
test the robustness of the resulting phylogenies. In this work, we propose a
new resampling method to use with those programs.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have chosen a biological way, mutation, to
create resamples from the original sequence. A mutation can be performed by
three different ways; insertion, deletion and substitution. Mutations were
performed at random positions and with random rates. A resampling is result of
random mutations on certain percent of original sequences. We have used
Relative Complexity Measure method to construct the phylogenies and TreeDist
routine of PHYLIP 3.65 to calculate tree distances. We have used molecular
sequences from ITS region of DNA to test for.</p>

<h3>Results</h3>

<p class="abresults">We have performed our method on different set of molecular
sequences. Results showed that proposed resampling method produced reasonable
resamples at 10% mutation level for ITS region of DNA. The resulting resamples
were produced very diverse phylogenies, some have very distant topology while
some others were very close to the original topology.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Proposed method has produced resamples successfully.
Since all other methods are based on aligned sequences, presence of a method
which does not require multiple alignment is crucial. The method is also unique
by its way of creating resamples by means of biological mutations.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Yasin Bakis (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8014 = 'b&#97;k&#105;s_y' + '&#64;';
 addy8014 = addy8014 + '&#105;b&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy8014 + suffix + '\'' + attribs + '>' );
 document.write( addy8014 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19610 = 'b&#97;k&#105;s_y' + '&#64;';
 addy19610 = addy19610 + '&#105;b&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy19610 + suffix + '\'' + attribs + '>' );
 document.write( addy19610 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12882 = 'b&#97;k&#105;s_y' + '&#64;';
 addy12882 = addy12882 + '&#105;b&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy12882 + suffix + '\'' + attribs + '>' );
 document.write( addy12882 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Abant Izzet Baysal University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Biology, Abant Izzet Baysal University,
Bolu, 14280 Turkey 2 Biological Sciences and Bioengineering, Sabanci
University, Tuzla, Istanbul, 34956 Turkey</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection44">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741047">A-43. Clustering massive
sequence databases</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vu D<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Vu D&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*),
Robert V<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Robert V&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,2)</p>

<p class="abmotivation">The amount of genomic data grows significantly. Currently
there are more than 6 millions fungal sequences in GENBANK databases.
Unfortunately, many sequences have been submitted wrongly or without any
indication on the species name. Thus there is a need for tools to classify
sequences. Existing tools such as UPGMA require heavy computations as they aim
to construct phylogenetic trees. Thus, it is not feasible to use them for
clustering very large databases. We aim to develop a clustering tool that can
group sequences from the same species which is is able to handle large numbers
of sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use a greedy algorithm as the starting point of
our approach. For each sequence, we find its cluster by searching for the
nearest matching sequence in a sequence database obtained from the existing
clusters. If the similarity between the sequence and its nearest match is
sufficient, the sequence is placed in the cluster of its match. Otherwise a new
cluster is created for the sequence. To determine a sufficient similarity, we
created a database of identified sequences to learn this number. It is the
similarity that gives the closest clustering to the species of the given sequences.</p>

<h3>Results</h3>

<p class="abresults">We have implemented the algorithm in BioloMICS, a software
solution for biological data management. We have experimented with more than 5
millions fungal sequences from GENBANK databases. The sufficient similarity of
sequences in the same cluster for the most used regions such as ITS, 26S has
been predicted. Preliminary experiments of clustering show promising results as
a number of ITS and 26S fungal sequences have been detected as incorrectly
identified.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The advantage of our approach is that it is time
efficient since we do not need to take into account the relationship between
organisms. Another approach, which is similar to ours, is the tool called
UCLUST by C. Edgar. However, in our algorithm, a cluster can have more than one
representative sequences, and therefore, it respects a transitive closure of
sequences with respect to similarity. Moreover, we use BLAST searching for
local matches of a sequence, while UCLUST searches for global matches. BLAST
may find less accurate matches, but it is more time efficient than other global
alignments.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Duong Vu (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6335 = 'd.v&#117;' + '&#64;';
 addy6335 = addy6335 + 'cbs' + '&#46;' + 'kn&#97;w' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy6335 + suffix + '\'' + attribs + '>' );
 document.write( addy6335 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7244 = 'd.v&#117;' + '&#64;';
 addy7244 = addy7244 + 'cbs' + '&#46;' + 'kn&#97;w' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy7244 + suffix + '\'' + attribs + '>' );
 document.write( addy7244 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40511 = 'd.v&#117;' + '&#64;';
 addy40511 = addy40511 + 'cbs' + '&#46;' + 'kn&#97;w' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy40511 + suffix + '\'' + attribs + '>' );
 document.write( addy40511 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioinformatics group, CBS-KNAW fungal biodiversity
centre</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics group, CBS-KNAW fungal biodiversity
centre, Utrecht, The Netherlands (2) BioAware, www.bio-aware.com</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection45">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741048">A-44. WordCluster:
detecting clusters of DNA words and genomic elements</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Barturen G<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Barturen G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, Oliver JL<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Oliver JL&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Hackenberg M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hackenberg M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Many k-mers (DNA words) and genomic elements are known to
be clustered in the genome. Well established examples are the genes, TFBSs, CpG
dinucleotides and ultra-conserved non-coding regions. Currently, no algorithm
exists to find these clusters in a statistically comprehensible way. The
detection of clustering often relies on densities and sliding window approaches
or arbitrarily chosen distance thresholds.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The algorithm is based on entity distances and an
assigned p value. Two different types of input data can be supplied: 1) a group
of k-mers and a genomic sequence to be scanned by the program (user supplied or
chosen among the 19 genome assemblies available in our database); and 2) a file
in BED format with the coordinates of the genomic elements whose clustering
properties should be analyzed. The output includes a basic statistics of the
clusters, the co-localization with gene annotation, and GO enrichment/depletion
analysis for the genes overlapped by the predicted clusters.</p>

<h3>Results</h3>

<p class="abresults">WordCluster detects clusters of DNA words (k-mers) or
genomic elements, based on the distance between neighboring copies and an
assigned p-value. The method was implemented into a web server which also
determines the co-localization with gene annotations. We demonstrate the
usefulness of this approach by detecting the clusters of CAG/CTG (cytosine
contexts that can be methylated), showing that the degree of methylation varies
drastically inside/outside of the clusters. As a second example, we search for
statistically significant clusters of olfactory receptor (OR) genes in the
human genome.</p>

<h3>Discussion</h3>

<p class="abdiscussion">WordCluster generalizes the previous CpGcluster algorithm
to any word or genomic element in the genome, at the same time associating a
statistical significance to the detected clusters. It outperforms current
methods relying on densities and sliding window approaches or arbitrarily
chosen distance thresholds. The implementation as a web server connected to
online databases allows for co-localization studies with different gene
regions, as well as for genome wide enrichment/depletion analysis in GO terms
of the predicted clusters, which may facilitate relating them to biological
function.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinfo2.ugr.es/wordCluster/wordCluster.php" mce_href="http://bioinfo2.ugr.es/wordCluster/wordCluster.php">http://bioinfo2.ugr.es/wordCluster/wordCluster.php</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Guillermo G.B. Barturen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41544 = 'b&#97;rtg01' + '&#64;';
 addy41544 = addy41544 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy41544 + suffix + '\'' + attribs + '>' );
 document.write( addy41544 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74697 = 'b&#97;rtg01' + '&#64;';
 addy74697 = addy74697 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy74697 + suffix + '\'' + attribs + '>' );
 document.write( addy74697 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4803 = 'b&#97;rtg01' + '&#64;';
 addy4803 = addy4803 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy4803 + suffix + '\'' + attribs + '>' );
 document.write( addy4803 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Granada</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Dpto. de Genética, Facultad de Ciencias, Universidad de
Granada, 18071-Granada &amp; Lab. de Bioinformática, Centro de Investigación
Biomédica, PTS, 18100-Granada, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Spanish Government Grant No. BIO2008-01353 to JLO.
Spanish ‘Juan de la Cierva’ grant to MH and Basque Country ‘Programa de
formación de investigadores del Departamento de Educación, Universidades e
Investigación’ grant to GB. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection46">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741049">A-45. Analysis of copy
loss and gain variations in Holstein cattle autosomes using BeadChip SNPs</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Seroussi E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Seroussi E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Glick G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Glick G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Shirak A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shirak A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Yakobson E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yakobson E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Weller JI<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weller JI&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ezra E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ezra E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Zeron Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zeron Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Copy number variation (CNV) has been recently identified
in human and other mammalian genomes and there is a growing awareness of CNV's
potential as a major source for heritable variation in complex traits. Genomic
selection is a newly developed tool for the estimation of breeding values for
quantitative traits through the use of genome-wide genotyping of SNPs. The
Illumina BovineSNP50 BeadChip has been used worldwide to genotype over 30,000
Holstein cattle. On this chip, 54,001 SNPs are represented (~SNP/50,000 bp),
and some of them fall within CNV regions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used the BeadChip data obtained for 912 Israeli
bulls to investigate the effects of CNV on SNP calls.For each of the SNPs, we
estimated the frequencies of occurrence of loss of heterozygosity (LOH) and of
gain based either on deviation from the expected Hardy-Weinberg equilibrium
(HWE) or on signal intensity (SI) using the PennCNV “detect” option.</p>

<h3>Results</h3>

<p class="abresults">Correlations between LOH/CNV frequencies predicted by the
two methods were low (up to r = 0.08). Nevertheless, 418 locations displayed
significantly high frequencies by both methods. Efficiency of designating large
genomic clusters of olfactory receptors as CNVs was 29%. Frequency values for
copy loss were distinguishable in non-autosomal regions, indicating
misplacement of a region in the current BTA7 map. Analysis of BTA18 placed
important net merit QTLs in regions rich in segmental duplications and CNVs.
Enrichment of transporters in CNV loci suggested their potential effect on
milk-production traits.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Expansion of HWE and PennCNV analyses allowed estimating
LOH/CNV frequencies, and combining the two methods yielded more sensitive
detection of inherited CNVs and better estimation of their possible effects on
cattle genetics. Although this approach was more effective than methodologies
previously applied in cattle, it has severe limitations. Thus the number of
CNVs reported here for the Holstein breed may represent as little as one-tenth
of inherited common structural variation.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Eyal Seroussi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79835 = 's&#101;r&#111;&#117;ss&#105;' + '&#64;';
 addy79835 = addy79835 + '&#97;gr&#105;' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy79835 + suffix + '\'' + attribs + '>' );
 document.write( addy79835 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15070 = 's&#101;r&#111;&#117;ss&#105;' + '&#64;';
 addy15070 = addy15070 + '&#97;gr&#105;' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy15070 + suffix + '\'' + attribs + '>' );
 document.write( addy15070 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24016 = 's&#101;r&#111;&#117;ss&#105;' + '&#64;';
 addy24016 = addy24016 + '&#97;gr&#105;' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy24016 + suffix + '\'' + attribs + '>' );
 document.write( addy24016 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Agricultural Research Organization (ARO), Volcani
Center</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Institute of Animal Sciences, ARO, The Volcani
Center, Bet Dagan 50250, Israel 2 Israel Cattle Breeders Association, Caesaria
Industrial Park 38900, Israel 3 Sion, AI Institute, Shikmim 79800, Israel</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research was supported by grants from the Israel
Milk Marketing Board; the European Sixth Research and Technological Development
Framework Programme, Proposal No. 016250-2 SABRE, and Research Grant Award No.
IS-4201-09 from BARD, The United States - Israel Binational Agricultural
Research and Development Fund. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection47">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741050">A-46. The role of the Occludin
MARVEL domain in intracellular targeting and clustering</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Yeheskel A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Yeheskel A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Yaffe Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yaffe Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Hirschberg K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hirschberg K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Pasmanik-Chor M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pasmanik-Chor M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Tight junctions (TJs) are complex proteinatious integral
membrane assemblies indispensable for epithelial function. Occludin is a MARVEL
domain containing protein, found in all TJs and is associated with regulating
it functions. The MARVEL domain contains 4 transmembrane (TM) segments and its
function is largely unclear. We propose that this lipid membrane interacting
motif is associated with a number of central functions such as targeting to the
tight junctions and clustering. The purpose of this study is to gain insight
into Ocln MARVEL function using protein sequence analysis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">94 SwissProt MARVEL homologues were obtained from
PFAM. Alignment was performed by MAFFT and phylogeny tree was built by RaxML.
The TM boundaries of Ocln were determined using secondary structure prediction,
conservation and hydrophobicity profile. Consensus prediction of TM helices was
used to produce helical wheel presentation of each helix, using Textopo.
Bi-molecular fluorescent complementation analysis was applied to study Ocln
oligomerization. Site directed mutagenesis of conserved aromatic residues was
applied. For cell microscopy, a fluorescent protein-tagged Ocln-MARVEL motif
was used</p>

<h3>Results</h3>

<p class="abresults">Phylogenetic tree of the MARVEL family illustrates distinct
subgroups, with Ocln as one of them having unique properties. The first
extracellular loop contains a glycine-tyrosine rich motif. The second
extracellular loop has a distinctive secondary structure. Live cell confocal
microscopy shows that Ocln-MARVEL is correctly sorted to the basolateral
membrane and TJs. Alanine substitutions had little effect on the surface
expression of Ocln-MARVEL. However 4 residue mutants resulted in reduced
oligomerization, increased tendency to aggregate and slowed down secretory
transport.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We propose a mechanism for the binding of 2 Ocln
molecules from opposing membranes of adjacent cells using the first
extracellular loop, previously suggested to contribute to protein flexibility
and organization. The second extracellular loop was predicted to have a
distinctive secondary structure which may contribute to paracellular transport
and cell-cell adhesion. Site directed mutagenesis showed a role for conserved
aromatic amino acids in cluster formation. These data shed light on the
potential role of protein–lipid interactions of the Ocln MARVEL domain and
membrane in TJ functions.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.tau.ac.il/lifesci/bioinformatics.html" mce_href="http://www.tau.ac.il/lifesci/bioinformatics.html">http://www.tau.ac.il/lifesci/bioinformatics.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Adva Yeheskel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76576 = 's&#117;&#101;z&#97;dv&#97;' + '&#64;';
 addy76576 = addy76576 + 't&#97;&#117;&#101;x' + '&#46;' + 't&#97;&#117;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy76576 + suffix + '\'' + attribs + '>' );
 document.write( addy76576 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34786 = 's&#117;&#101;z&#97;dv&#97;' + '&#64;';
 addy34786 = addy34786 + 't&#97;&#117;&#101;x' + '&#46;' + 't&#97;&#117;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy34786 + suffix + '\'' + attribs + '>' );
 document.write( addy34786 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50801 = 's&#117;&#101;z&#97;dv&#97;' + '&#64;';
 addy50801 = addy50801 + 't&#97;&#117;&#101;x' + '&#46;' + 't&#97;&#117;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy50801 + suffix + '\'' + attribs + '>' );
 document.write( addy50801 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioinformatics Unit, G.S.W. Faculty of Life Sciences,
Tel Aviv University, Tel Aviv 69978, Israel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Bioinformatics Unit, G.S.W. Faculty of Life
Sciences, Tel Aviv University, Tel Aviv 69978, Israel 2.Department of
Pathology, Sackler School of Medicine, Tel Aviv University, Tel Aviv 69978,
Israel</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection48">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741051">A-47. Multi-Harmony:
detecting functional specificity from sequence alignment</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Feenstra KA<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Feenstra KA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Brandt BW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brandt BW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Heringa J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Heringa J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Many protein families contain sub-families with
functional specialization, such as binding different ligands or being involved
in different protein-protein interactions. A small number of amino acids
generally determine functional specificity. The identification of these
residues can aid the understanding of protein function and help finding targets
for experimental analysis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Here, we present multi-Harmony, an interactive web
sever for detecting sub-type-specific sites in proteins starting from a
multiple sequence alignment. Combining our Sequence Harmony (SH) and
multi-Relief (mR) methods in one web server allows simultaneous analysis and
comparison of specificity residues; furthermore, both methods have been significantly
improved and extended. SH has been extended to cope with more than two
sub-groups. mR has been changed from a sampling implementation to a
deterministic one, making it more consistent and user friendly. For both
methods Z-scores are reported.</p>

<h3>Results</h3>

<p class="abresults">The Sequence Harmony and multi-Relief methods are
benchmarked on a test-set comprising 23 protein families with known specificity
sites and functional sub-grouping. Overall performance is shown to be
comparable to or better than the best of alternative available methods, such as
SDPpred, ProteinKeys, PROUST-II and Xdet. Performance at highest precision
continues up to higher recall compared to all other methods tested.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The multi-Harmony web server provides state-of-the art
specificity detection, and is easily accessible to non-expert users. An example
script for remote access for expert users is available on request. The
webserver produces a dynamic output page, which includes interactive
connections to the Jalview and Jmol applets, thereby allowing interactive
analysis of the results. Multi-Harmony is available at
http://www.ibi.vu.nl/programs/shmrwww.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ibi.vu.nl/programs/shmrwww" mce_href="http://www.ibi.vu.nl/programs/shmrwww">http://www.ibi.vu.nl/programs/shmrwww</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">K. Anton Feenstra (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77529 = 'f&#101;&#101;nstr&#97;' + '&#64;';
 addy77529 = addy77529 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy77529 + suffix + '\'' + attribs + '>' );
 document.write( addy77529 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51611 = 'f&#101;&#101;nstr&#97;' + '&#64;';
 addy51611 = addy51611 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy51611 + suffix + '\'' + attribs + '>' );
 document.write( addy51611 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89460 = 'f&#101;&#101;nstr&#97;' + '&#64;';
 addy89460 = addy89460 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy89460 + suffix + '\'' + attribs + '>' );
 document.write( addy89460 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">IBIVU / Free University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Centre for Integrative Bioinformatics (IBIVU), VU
University Amsterdam, De Boelelaan 1081A, 1081HV Amsterdam, The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">ENFIN, a Network of Excellence funded by the European
Commission within its FP6 Programme, under the thematic area ‘Life sciences,
genomics and biotechnology for health’ (LSHG-CT-2005-518254). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection49">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741052">A-48. Fast approximate
statistical ranking for sequence motif discovery under higher order Markov
background model</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Shida K<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Shida K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*)</p>

<p class="abmotivation">Sequence motif discovery is still a difficult problem in
bioinformatics. The motif candidates found by motif discovery programs are
routinely ranked by their p-values to ensure that the most statistically
significant answer is presented. However, accurate p-value evaluation is a
complicated and time consuming computation and there is considerable difficulty
in taking higher order Markov background models into account precisely.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For a motif discovery problem under k-th order
Markov background, we propose (1) a novel polynomial approximation scheme for
the motif p-value (2) a MCMC method to optimize the coefficients of the
polynomial to maximize the accuracy of the resultant p-value and (3) a system
to validate the obtained approximation using a motif discovery problem on
random binary sequences as an example. The optimization of polynomial
parameters by means of MCMC sampling is surely time consuming but reusable:
such a sampling is necessary only once for a given condition.</p>

<h3>Results</h3>

<p class="abresults">First, the numerical precision of the p-value estimated by
this method is merely acceptable. However, unlike any previously proposed
method, this method is applicable for higher order (k=&gt;2) Markov background
without excessive loss of precision nor catastrophic increase of computational
complexity. Second, because this method is just a calculation of polynomial,
the approximate evaluation of p-value is almost comparable to some of typical
motif score functions, such as the relative entropy, in speed. Third, a
prototype of this method is successfully tested on a small Human dataset.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The previous methods of motif p-value estimation are
focused on near-exact precision for limited level of correlation in the
background requiring large computational complexity. On the contrary, this new
method can handle increased level of background correlation with minimum
sacrifice of p-value precision. The computational burden is increased only in
the pre-processing stage. Some other problems in bioinformatics can be heavily
affected by background correlations that are hidden or neglected. This method
can be extended to analyze and cancel the effect of such correlations.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Kazuhito Shida (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16374 = 'sh&#105;d&#97;' + '&#64;';
 addy16374 = addy16374 + '&#105;mr' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy16374 + suffix + '\'' + attribs + '>' );
 document.write( addy16374 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6202 = 'sh&#105;d&#97;' + '&#64;';
 addy6202 = addy6202 + '&#105;mr' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy6202 + suffix + '\'' + attribs + '>' );
 document.write( addy6202 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39582 = 'sh&#105;d&#97;' + '&#64;';
 addy39582 = addy39582 + '&#105;mr' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy39582 + suffix + '\'' + attribs + '>' );
 document.write( addy39582 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute for materials research, Tohoku university</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Institute for materials research, Tohoku university</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection50">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741053">A-49. Networking interacting
residues, evolutionary conservation and energetics in protein structures</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vidotto M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vidotto M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Martin AJM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Martin AJM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Boscariol F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Boscariol F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Walsh I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Walsh I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tosatto S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tosatto S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->CE<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tosatto SCE&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Recently, there has been a growing interest in
representing protein structures as so-called residue interaction networks
(RINs). RINs consider single amino acids in the protein structure as nodes and
connections as physico-chemical interactions, such as covalent bonds and
non-covalent contacts. The intuitive idea is to analyze a structure with the
same approach as a protein interaction network in order to investigate whether
the same rules apply. RINs have already been used to analyze protein stability
and folding, allosteric communication, enzyme catalysis or mutation effect
prediction.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Here, we present RING (URL:
http://protein.bio.unipd.it/ring/) as a novel tool to generate RINs for use in
Cytoscape. The tools was conceived to yield a simple, intuitive representation
that is also physico-chemically meaningful, i.e. distinguish salt bridges,
hydrogen bonds and aromatic interactions from generic van-der-Waals contacts.
Structural features are generated for each node. Protein sequence conservation,
determined by PSI-BLAST, and local conformational energy preferences to the
best of our knowledge represent two novel features for RINs.</p>

<h3>Results</h3>

<p class="abresults">The web server provides two user interfaces. A simple user
interface with meaningful default values for most parameters and a more complex
interface where the user can specify how the RIN parameters. The output file
contains the network definition files for Cytoscape. Online help pages are
provided with extensive documentation. The generated RINs contain information
regarding different interaction types as well as secondary structure and
energetics of each residue. Once generated, the RIN can be easily visualized in
Cytoscape and network properties analyzed with available plugins.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A special feature of RING is to generate meaningful
sub-networks. RING allows the user to limit the generated network to buried
residues, conserved residues or both. Intuitively, limiting RIN analysis to a
network of conserved residues will help to focus on the essential interactions
for a protein fold. In summary, RING is a novel web tool for use with Cytoscape
designed for the analysis of protein structures in terms of physico-chemical
interactions, evolutionary conservation and energetics while taking advantage
of the powerful network paradigm.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://protein.bio.unipd.it/ring/" mce_href="http://protein.bio.unipd.it/ring/">http://protein.bio.unipd.it/ring/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Silvio C.E. Tosatto (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39281 = 's&#105;lv&#105;&#111;.t&#111;s&#97;tt&#111;' + '&#64;';
 addy39281 = addy39281 + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy39281 + suffix + '\'' + attribs + '>' );
 document.write( addy39281 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27785 = 's&#105;lv&#105;&#111;.t&#111;s&#97;tt&#111;' + '&#64;';
 addy27785 = addy27785 + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy27785 + suffix + '\'' + attribs + '>' );
 document.write( addy27785 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14373 = 's&#105;lv&#105;&#111;.t&#111;s&#97;tt&#111;' + '&#64;';
 addy14373 = addy14373 + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy14373 + suffix + '\'' + attribs + '>' );
 document.write( addy14373 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dept. of Biology, University of Padua</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 - Dept. of Biology, University of Padua, Italy</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection51">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741054">A-50. An iterative
scheme for making long structural alignments of ncRNAs using FOLDALIGN</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Havgaard J H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Havgaard J H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Gorodkin J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gorodkin J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Structured non-coding RNAs (ncRNAs) are often more
conserved at the structural level than in the primary sequence. Sequence based
alignments of ncRNAs are therefore often inaccurate. Specialized approaches,
such as FOLDALIGN, which also takes the structures of the molecules into
account are needed to obtain the proper alignment. However, due to the high run
times of such methods heuristics are needed to lower the time and memory
requirements to make the algorithms practical.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The FOLDALIGN algorithm (Sankoff based) makes
pairwise alignments of RNA sequences using both sequence and structure
information. Pre-aligned positions or whole subalignments are introduced as
seed constraints to FOLDALIGN. The algorithm builds longer alignments from the
shorter seed constraints. At each step in the iteration a subset of the
generated (length constrained) alignments are used as seeds which are extended
into longer alignments. A subset of these alignments are then used as seeds in
the next step of the iteration.</p>

<h3>Results</h3>

<p class="abresults">We present putative results of test runs. These preliminary
results indicates that the use of seed constraints makes it possible to make
global structural alignments of ribosomal sequences on a modern computer, and thus
promise to make de novo search for similar size ncRNAs. The seed constraints
also result in lower run time for local alignments. We are currently working a
more systematic benchmark. An effort has also been made to improve the
statistical evaluation of the local alignments.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Tools which simultaneously can fold and align RNAs have
been shown to be an essential component in do novo searches for ncRNAs.
However, a main limitation of these methods are their time and memory
requirements. With the implementation of seed constraints FOLDALIGN can now
make alignments of size 1500 - 2000 nt. compared to the previous 300 - 500 nt.
This makes it possible to find not only the small but also some of the larger
ncRNAs. With the improvement in the evaluation of the significans of the
alignments the selection of candidates for experimental verification have
become more feasible.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://foldalign.ku.dk" mce_href="http://foldalign.ku.dk">http://foldalign.ku.dk</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jakob H. Havgaard (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72277 = 'h&#117;ll' + '&#64;';
 addy72277 = addy72277 + 'g&#101;n&#111;m&#101;' + '&#46;' + 'k&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy72277 + suffix + '\'' + attribs + '>' );
 document.write( addy72277 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43059 = 'h&#117;ll' + '&#64;';
 addy43059 = addy43059 + 'g&#101;n&#111;m&#101;' + '&#46;' + 'k&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy43059 + suffix + '\'' + attribs + '>' );
 document.write( addy43059 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65593 = 'h&#117;ll' + '&#64;';
 addy65593 = addy65593 + 'g&#101;n&#111;m&#101;' + '&#46;' + 'k&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy65593 + suffix + '\'' + attribs + '>' );
 document.write( addy65593 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Copenhagen</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Center for non-coding RNA in Technology and Health,
Genetics and Bioinformatics IBHV, Faculty of Life Sciences University of
Copenhagen</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Danish Council for Independent Research (Technology
and Production Sciences) The Danish Council for Strategic Research (Strategic
growth Technologies) The Danish Center for Scientific Computing</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection52">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741055">A-51. A multi-objective
approach to the prediction of sRNAs in bacteria</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Arnedo J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Arnedo J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Romero-Zaliz R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Romero-Zaliz R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), del Val C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;del Val C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*)</p>

<p class="abmotivation">Bacterial small non-coding RNAs (sRNAs) are recognized as
novel widespread regulators of gene expression. There are several algorithms
available for their ab initio prediction, but all of them present intrinsic
limitations. We present in this work a methodology, which uses a
multi-objective approach approach to extract the best methods’ aggregations by
maximizing the specificity and sensitivity of the program´s individual
predictions</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">All potential aggregations, methods’, form a space
of potential hypotheses, which can be represented as a lattice structure. We
search for the best methods’ aggregations, moving from hypothesis to hypothesis
towards the most general, the union of all methods, and the most specific,
their intersection, which are located at the top and the bottom of the lattice,
respectively. Salmonella typhimurium LT2 was used reference genome, and the
algorithms: zMFold, eQRNA, RNAz, Alifoldz, Dynalign and MSARi.</p>

<h3>Results</h3>

<p class="abresults">Results show a major improvement in specificity and
sensitivity when our methodology is compared to the performance of individual
methods. The use of methods’ aggregations increase either prediction’s sensitivity
or specificity, and a few aggregations increase both objectives when compared
with the individual methods. The best results are obtained in aggregations
containing 2 or 3 methods via either union or intersection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The here proposed methodology is an automatic method
generator, and a step forward to exploit all already existing methods, by
providing optimal methods’ aggregations to answer concrete queries for a
certain biological problem with a maximized accuracy of the prediction. As more
approaches are integrated for each of the presented problems, de novo accuracy
can be expected to improve further.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Coral del Val (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65604 = 'd&#101;lv&#97;l' + '&#64;';
 addy65604 = addy65604 + 'd&#101;cs&#97;&#105;' + '&#46;' + '&#117;gr' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy65604 + suffix + '\'' + attribs + '>' );
 document.write( addy65604 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88028 = 'd&#101;lv&#97;l' + '&#64;';
 addy88028 = addy88028 + 'd&#101;cs&#97;&#105;' + '&#46;' + '&#117;gr' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy88028 + suffix + '\'' + attribs + '>' );
 document.write( addy88028 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46399 = 'd&#101;lv&#97;l' + '&#64;';
 addy46399 = addy46399 + 'd&#101;cs&#97;&#105;' + '&#46;' + '&#117;gr' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy46399 + suffix + '\'' + attribs + '>' );
 document.write( addy46399 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Granada</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Dpt. of Computer Science and Artificial Inteligence.
University of Granada, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Consejeria de
Innovacion, Investigacion y Ciencia de la Junta de Andalucia under project
TIC-02788</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection53">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741056">A-52. Scalability of
large-scale protein domain family inference</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rezvoy C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Rezvoy C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Vivien F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vivien F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Kahn D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kahn D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">The ProDom database is a repository of protein domain
families inferred automatically from homologies between protein sequences of
the Uniprot database. Since 1999, ProDom has been built using MkDom2, a
sequential algorithm of quadratic complexity. Because of its sequential nature,
MkDom2 cannot keep up with the exponential increase of Uniprot over the years,
to the point that it is no longer possible to envision a new release of ProDom
with this sequential program. given past records, running MkDom2 on a recent
release of Uniprot would have taken over 10 years.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Mpi_MkDom2 is a master-worker greedy algorithm. At
each iteration, the shortest sequences are considered potential domains and
used as queries to recruit homologous domains using PSIBLAST. If two sequences
share an homology in an all-against-all BLAST, only the shortest is used as
query. New families are validated by checking if they do not overlap with each
other. Valid families are removed from the database before the next iteration.
To assess the performance of the algorithm, MPI_MkDom2 was tested with nested
databases (27MB and 210MB) and different numbers of computing nodes.</p>

<h3>Results</h3>

<p class="abresults">Experiments show that MPI_MkDom2 is able to achieve a
reasonable speedup on a small database (27 MB), but starts loosing efficiency
above 40 workers. A larger database, however, allows a better load balancing.
MPI_MkDom2 is thus able to efficiently use more workers when processing larger
databases. While allowing to substantially decrease the construction time of
ProDom, the new algorithm still creates results that are consistent with those
of the original algorithm: on average domain families created by MkDom2 are
more than 90% similar to those created by MPI_MkDom2.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The distributed algorithm presented here is able to
provide reasonable speed-ups while retaining the structure of the protein
domain families built buy the sequential algorithm. Even when achieving maximal
speedup the clustering stays consistent with the sequential result. The speedup
being dependent of the size of the database processed, this new algorithm will
provide a mean to efficiently construct new releases of ProDom while staying
consistent with the original sequential heuristic.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://prodom.prabi.fr" mce_href="http://prodom.prabi.fr">http://prodom.prabi.fr</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Clément Rezvoy (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98025 = 'Cl&#101;m&#101;nt.R&#101;zv&#111;y' + '&#64;';
 addy98025 = addy98025 + '&#101;ns-ly&#111;n' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy98025 + suffix + '\'' + attribs + '>' );
 document.write( addy98025 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96042 = 'Cl&#101;m&#101;nt.R&#101;zv&#111;y' + '&#64;';
 addy96042 = addy96042 + '&#101;ns-ly&#111;n' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy96042 + suffix + '\'' + attribs + '>' );
 document.write( addy96042 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66008 = 'Cl&#101;m&#101;nt.R&#101;zv&#111;y' + '&#64;';
 addy66008 = addy66008 + '&#101;ns-ly&#111;n' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy66008 + suffix + '\'' + attribs + '>' );
 document.write( addy66008 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Éns Lyon</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">´ 1 - ENS Lyon, Universite de Lyon, LIP, UMR 5668, ENS
Lyon - CNRS - INRIA - UCBL, Lyon, FRANCE 2 - Université de Lyon, Laboratoire de
Biometrie et Biologie Evolutive, UMR 5558, UCBL - CNRS, INRA, Villeurbanne,
FRANCE 3 - INRIA, Universite de Lyon, LIP, UMR 5668, ENS Lyon - CNRS - INRIA -
UCBL, Lyon, FRANCE</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The ProDom project was supported by the FP6 EMBRACE
Network of Excellence, the FP7 IMPACT Research Infrastructure programme and the
France-Israel Research Network Program in Bioinformatics. Experiments presented
in this paper were carried out using the Grid’5000 experimental testbed, being
developed under the INRIA ALADDIN development action with support from CNRS,
RENATER and several Universities as well as other funding bodies (see
https://www.grid5000.fr). This work was granted access to the HPC resources of
CINES under the allocation 2010-c2010076425 made by GENCI (Grand Equipement
National de Calcul Intensif).</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection54">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741057">A-53. Evolutionary study
of eye-developmental gene expression across multiple Drosophila species by
high-throughput tag sequencing</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Naval-Sanchez M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Naval-Sanchez M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Aerts S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aerts S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Retinal differentiation is initiated in the eye imaginal
disc during third instar larval development. During this process the genes
eyeless and atonal activate a cascade of signaling events that end in retina
formation. Drosophila eye-development is a well-studied system but information
about gene expression and gene regulatory interactions remains sparse. Here, we
aim to gain further insight into the gene regulatory network underlying retinal
differentiation by determining conserved and divergent patterns of gene
expression in various Drosophila species using next-generation sequencing.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">RNA was extracted from eye-antennal imaginal discs
and wing imaginal discs (controls) during third instar larval stage from three
species, namely D.melanogaster, D.yakuba and D.virilis. Tag libraries for
digital gene expression were created using the NlaIII restriction enzyme and
were sequenced on the Illumina GAII platform in six separate lanes. Reads were
mapped against their reference genome using bowtie, considering only uniquely
mapped tags. Gene expression levels were normalized as tags per million (TPM).
Gene set and functional enrichment analyses were performed using GSEA.</p>

<h3>Results</h3>

<p class="abresults">First, we show that DGE yields meaningful expression values
in D.melanogaster through a comparison with microarray data on the same tissue.
Next, we confirm that gene expression divergence increases over evolutionary
time, being larger for Dmel-Dvir than for Dmel-Dyak. Finally, we identify genes
with conserved expression in the eye disc. Surprisingly, only a small fraction
of eye specific genes in Dmel is also eye specific in the other species (30%
and 17% overlap with Dyak and Dvir respectively). These conserved genes are
mostly involved in photoreceptor cell fate specification.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To our knowledge this is the first time that digital gene
expression is used to measure tissue-specific gene expression in multiple
Drosophila species. We conclude that DGE-based gene expression measurements are
accurate and appropriate for determining gene expression levels for species
without a microarray platform. Finally, we are able to find expression
conservation and divergence patterns across Drosophila species allowing us to
investigate further the evolutionary differences in the mechanisms governing
gene expression.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Marina Naval Sanchez (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10906 = 'M&#97;r&#105;n&#97;.N&#97;v&#97;lS&#97;nch&#101;z' + '&#64;';
 addy10906 = addy10906 + 'cm&#101;' + '&#46;' + 'v&#105;b-k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy10906 + suffix + '\'' + attribs + '>' );
 document.write( addy10906 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2376 = 'M&#97;r&#105;n&#97;.N&#97;v&#97;lS&#97;nch&#101;z' + '&#64;';
 addy2376 = addy2376 + 'cm&#101;' + '&#46;' + 'v&#105;b-k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy2376 + suffix + '\'' + attribs + '>' );
 document.write( addy2376 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73252 = 'M&#97;r&#105;n&#97;.N&#97;v&#97;lS&#97;nch&#101;z' + '&#64;';
 addy73252 = addy73252 + 'cm&#101;' + '&#46;' + 'v&#105;b-k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy73252 + suffix + '\'' + attribs + '>' );
 document.write( addy73252 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Laboratory of Computational Biology, Center for Human
Genetics, KULeuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Laboratory of Computational Biology, Center for Human
Genetics, KULeuven. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection55">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741058">A-54. Higher order
repeat with largest primary repeat unit (~2.4 kb) and located within a gene in
human genome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gluncic M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gluncic M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Paar V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Paar V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Paar P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Paar P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Rosandic M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rosandic M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vlahovic I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vlahovic I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Using our novel computational method Global Repeat Map
(GRM), based on Key String Algorithm (KSA), which is particularly convenient
for identification and analysis of repeats and higher order repeats (HORs)
characterized by very long repeat units, we performed a case study of Build
37.1 genomic ensemble of Y chromosome, addressing the following questions: (i)
Is there a HOR in chromosome Y in addition to the known HORs based on 5-bp and
alphoid 171-bp primary repeat units? (ii) Is there a HOR with very large basic
repeat unit far beyond the size of all those found so far in human genome?
(iii) Is there a HOR fully contained within a gene for which no evidence was
reported so far?</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The Key String Algorithm (KSA) framework is based
on the use of a freely chosen short sequence of nucleotides, a key string,
which cuts a genomic sequence at each location of the key string. The lengths
of ensuing KSA fragments form KSA length array. Any periodicity appearing in
the KSA length array enables identification and location of repeat in a given
sequence. Analysis of repeat sequences at position of any periodicity gives
consensus repeat unit. Any presence of higher order periodicity in the KSA length
array reveals the presence of HOR at that position. The GRM algorithm is a
graphical extension of KSA to an ensemble of all key strings of the same length
and superposition of their segmentation results.</p>

<h3>Results</h3>

<p class="abresults">We discovered in Y chromosome a new Higher Order Repeat
(HOR), based on ~2.4 kb primary repeat units. We classified an ensemble of ~2.4
kb monomers into five highly homologous monomer families denoted m01, m02, m03,
m04, and m05. Three monomer families, m02, m03, and m04 form the basic repeat
units of 2mer HOR copies and 3mer HOR copies which have the largest primary
repeat units discovered so far in the whole human genome. We found that each of
four tandem sequences involving HOR copies is contained within one of the genes
DAZ1 – DAZ4. Highly homologous simple repeats of m05 monomer family, are
organized into two distinct tandem arrays; they are wholly contained within the
DAZ2 and DAZ4 gene, respectively.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Using our novel computational method GRM we discovered in
Y chromosome a new HOR with largest known basic repeat unit, and wholly
contained within genes. That HOR may shed a new light at a unique role of Y
chromosome. Here we open the question on a possible role of HOR structure
within genes, and in particular if the basic repeat unit is very large. It is
therefore of interest to search for another examples of HOR structure within
genes. The use of Global Repeat Map is especially suited for bioinformatics
identification and analysis of repeats with very large monomeric repeat units
and for the case of periodic sequencing of several families of basic monomeric
repeat units as in the case of higher order repeats.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.hazu.hr/KSA/app/tools.html" mce_href="http://www.hazu.hr/KSA/app/tools.html">http://www.hazu.hr/KSA/app/tools.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Matko Gluncic (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51417 = 'm&#97;tk&#111;' + '&#64;';
 addy51417 = addy51417 + 'phy' + '&#46;' + 'hr';
 document.write( '<a ' + path + '\'' + prefix + addy51417 + suffix + '\'' + attribs + '>' );
 document.write( addy51417 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43919 = 'm&#97;tk&#111;' + '&#64;';
 addy43919 = addy43919 + 'phy' + '&#46;' + 'hr';
 document.write( '<a ' + path + '\'' + prefix + addy43919 + suffix + '\'' + attribs + '>' );
 document.write( addy43919 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy47948 = 'm&#97;tk&#111;' + '&#64;';
 addy47948 = addy47948 + 'phy' + '&#46;' + 'hr';
 document.write( '<a ' + path + '\'' + prefix + addy47948 + suffix + '\'' + attribs + '>' );
 document.write( addy47948 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Zagreb</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Faculty of Science, University of Zagreb, Zagreb,
Croatia (2) Faculty of Electrical Engineering and Computing, University of
Zagreb, Zagreb, Croatia</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection56">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741059">A-55. Fast and accurate
digging for binding motifs in ChIP-Seq data using ChIPMunk software</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kulakovskiy I V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kulakovskiy I V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Boeva VA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Boeva VA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4,5), Favorov AV<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Favorov AV&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,6), Makeev V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Makeev V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Makeev VJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,7)</p>

<p class="abmotivation">The task of identification of transcription factor
binding motifs in a limited number of short DNA sequences has a long history.
Recently upcoming ChIP-Seq data provided a new challenge for motif discovery.
Such data consist of thousands of sequences where a short overrepresented motif
is to be found. Fortunately, in the case of ChIP-Seq data one has additional
information, which helps to select the correct signal. This information is the
coverage profile constructed for DNA fragments obtained from ChIP-Seq
experiments.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We present a ChIP-Seq 'peak mode' of our efficient
motif discovery tool "ChIPMunk". It is an
expectation-maximization-with-bootstrapping algorithm which uses coverage
profile information by setting a specific weight profile over each sequence to
represent a preference for the sequence position to overlap with a binding
site.</p>

<h3>Results</h3>

<p class="abresults">We have tested our algorithm on different ChIP-Seq data
sets, including those for NRSF, GABP and for the oncogenic protein EWS-FLI1. We
successfully identified the correct motifs without traditional strict
truncation of large enriched regions. This was possible because of the coverage
profile information used as the motif positional preference prior. We show that
ChIPMunk motif recognition quality is the same or better than that of the
traditional (MEME, Multiple EM for Motif Elicitation) or ChIP-Seq-oriented
(HMS, Hybrid Motif Sampler) tools while the speed is dramatically better.</p>

<h3>Discussion</h3>

<p class="abdiscussion">ChIPMunk can be effectively used to analyze large
sequence sets and therefore is a helpful tool for prediction of transcription
factor binding motifs in ChIP-Seq data. ChIPMunk is based on Java. The source
code is freely available on the web.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://line.imb.ac.ru/ChIPMunk/" mce_href="http://line.imb.ac.ru/ChIPMunk/">http://line.imb.ac.ru/ChIPMunk/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Ivan V. Kulakovskiy (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56219 = '&#105;v&#97;n.k&#117;l&#97;k&#111;vsk&#105;y' + '&#64;';
 addy56219 = addy56219 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy56219 + suffix + '\'' + attribs + '>' );
 document.write( addy56219 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23754 = '&#105;v&#97;n.k&#117;l&#97;k&#111;vsk&#105;y' + '&#64;';
 addy23754 = addy23754 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy23754 + suffix + '\'' + attribs + '>' );
 document.write( addy23754 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63018 = '&#105;v&#97;n.k&#117;l&#97;k&#111;vsk&#105;y' + '&#64;';
 addy63018 = addy63018 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy63018 + suffix + '\'' + attribs + '>' );
 document.write( addy63018 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Research Institute for Genetics and Selection of
Industrial Microorganisms</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Research Institute for Genetics and Selection of
Industrial Microorganisms, Moscow, 117545 Russia (2) Institut Curie, 26 rue
d'Ulm, Paris, F-75248 France (3) INSERM, U900, Paris, F-75248 France (4)
INSERM, U830, Paris, F-75248 France (5) Mines ParisTech, Fontainebleau, F-77300
France (6) Johns Hopkins University, Baltimore MD 21205, US (7) Engelhardt
Institute of Molecular Biology, Russian Academy of Sciences, Moscow, 119991
Russia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by RFBR#07-04-01623 and
#07-04-01584 and RSASI#02.740.11.5008. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection57">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741060">A-56. Mebitoo: a
sequence analysis toolbox framework</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Spaniol C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Spaniol C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Helms V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Helms V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Since we want to deploy our methods for public use and
instead of implementing a variety of independent services, like online
services, we want to speed up both the analysis and development workflow
providing an incorporated framework that allows for: * module development
independent from the core framework * storage of arbitrary data, based on
module-individual XML concepts * a task processing schedule that each
concurrently processes any set of data with any set of methods incremental
analysis workflows, i.e. one method can make use of previously obtained
results, gained by other methods.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Mebitoo is a platform-independent desktop
application written in Java an based on the Netbeans Rich Client Platform. The
software allows to import sequences persistently into an incorporated database
engine, namely HSQL. A combined storage concept based on XML-files enables
software extensions – plugins – to archive information in customized data
structures. Automated data processing using those extensions can be invoked
using a task execution interface, which enables to queue multiple operations
and process multiple datasets in parallel.</p>

<h3>Results</h3>

<p class="abresults">We started to incorporate our own existing prediction
methods TMX and MINS as well as upcoming tools to predict helix cooperativitiy,
and the HMM-based Beta-Barrel TMX (BTMX). Moreover, we started to adapt the
MEME framework for motif search as a plugin. We aim to release the software
including these tools combined with utilities like PFAM-based sequence
alignments and visualization with TopoView and JMol.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Mebitoo is intended to be framework software which offers
researchers to develop new methods easily without getting lost in details of
data input and output or GUI representation. Novice users may implement based
on the interface we provide while advanced users are able to use the entire
Netbeans RCP and customize their own data storage. Because of the task concept,
researchers can import and automatically process large datasets easily with
different methods by queuing several jobs.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Christian Spaniol (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy80235 = 'chr&#105;st&#105;&#97;n.sp&#97;n&#105;&#111;l' + '&#64;';
 addy80235 = addy80235 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy80235 + suffix + '\'' + attribs + '>' );
 document.write( addy80235 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy329 = 'chr&#105;st&#105;&#97;n.sp&#97;n&#105;&#111;l' + '&#64;';
 addy329 = addy329 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy329 + suffix + '\'' + attribs + '>' );
 document.write( addy329 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97803 = 'chr&#105;st&#105;&#97;n.sp&#97;n&#105;&#111;l' + '&#64;';
 addy97803 = addy97803 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy97803 + suffix + '\'' + attribs + '>' );
 document.write( addy97803 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Bioinformatics, Saarland University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Center for Bioinformatics, Saarland University</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection58">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741061">A-57. Practical aspects
of RNA-Seq data analysis that you should know and they don't tell you</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Garcia F<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Garcia F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Tarazona S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tarazona S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Zuñiga S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zuñiga S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Santoyo J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Santoyo J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Dopazo J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dopazo J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Conesa A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Conesa A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">RNA-Seq has recently emerged as a powerful tool for the
analysis of the transcriptome as it is able to uncover novel transcriptionally
active genome regions, and can characterize with an unprecendeted detail the
complexity of the alternatively spliced transcriptional landscape.
Unfortunately, the analysis of these data is complex and includes many
"tricky" points. In this poster we report and comment some practical
aspects concerningr RNA-seq data analysis that are not usually found in
manuals/documentations, but should be taken into account when anlysing
high-throughput count data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We made use of published RNA-Seq data generated
with each of the three current NGS platforms (Roche, Illumina, and SOLiD). We
considered sequencing data of different properties (read length, pair-ends,
strand-specificity, etc.), and applied state-of-the-art software (Tophat,
Cufflinks, BFAST, AB WT, etc.), and in-house scripts for the analysis of the
data.</p>

<h3>Results</h3>

<p class="abresults">In general, we found that available documentation was
inssuficient to clearly understand the effect of parameters in the analysis
results. Moreover, provided examples are nearly impossible to reproduce. Also
data quality is an issue. Some algorithms ignore it at all or are accompained
by example data with strong biases. Finally we have detected problems with the
scalability of the algorithms and different bugs. These include failure in
processing parameter values, miss-match position misplacing resulting in false
discoveries and significance divergences when using or not reference genome
data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">RNA-Seq technologies are extremely powerful for
transcriptome studies. However, analysis technologies for this kind of data are
still at their infancy and users should use them critically. As more datasets
become available, algorithms will gain robustness and quality standards for
RNA-Seq analysis will be established.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Fernando Garcia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31036 = 'fdg&#97;rc&#105;&#97;' + '&#64;';
 addy31036 = addy31036 + 'c&#105;pf' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy31036 + suffix + '\'' + attribs + '>' );
 document.write( addy31036 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77858 = 'fdg&#97;rc&#105;&#97;' + '&#64;';
 addy77858 = addy77858 + 'c&#105;pf' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy77858 + suffix + '\'' + attribs + '>' );
 document.write( addy77858 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49414 = 'fdg&#97;rc&#105;&#97;' + '&#64;';
 addy49414 = addy49414 + 'c&#105;pf' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy49414 + suffix + '\'' + attribs + '>' );
 document.write( addy49414 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioinformatics Department, Centro de Investigacion
Principe Felipe</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics Department, Centro de Investigacion
Principe Felipe, Valencia, Spain (2) Bioinformatics Department, Sistemas
Genomicos, Valencia, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">MICINN Projects BIO2009-10799 y BIO2008-05266-E</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection59">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741062">A-58. MutaBase: a
framework and web-interface for the archival, management and interpretation of
single nucleotide variants obtained by next generation sequencing</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sifrim A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Sifrim A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Van Houdt J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Houdt J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vermeesch J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vermeesch J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Next generation sequencing allows the detection of novel
variant sites which are elusive in array-based methods of genotyping using
known common variants. As technology develops the throughput increases and the
management and interpretation of tens of thousands of variants per sample
becomes the new experimental bottleneck. We developed a flexible framework with
an easy-to-use interface which allows the archival of variants for every sample
and which integrates existing knowledge and prediction software for functional
impacts of variants.This allows for the rapid interpretation of NGS results</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our framework is based on a Ruby on Rails system
which includes parsing capabilities of most common single nucleotide variant
callers. This system is linked to a local Ensembl database to gather exisiting
knowledge about the given variants and genes. We use Polyphen2, FoldX, Waltz,
Tango and other software to computationally predict the impact of variants on
the phenotype. In order to make the task feasible in a reasonable time frame we
provide an easy-to-setup parallelized and distributed task management system.
The web-interface provides a multi-level complex query system for the archive.</p>

<h3>Results</h3>

<p class="abresults">We present the application of MutaBase in a case-study of
full-exome sequencing of 4 patients with Nicolaides-Baraitser syndrome. Doing
this we predict the genetic consequences and known information of each variant
and predict their possible functional impacts. By doing gene-centered
cross-patients queries and taking into account existing knowledge,
automatically retrieved by MutaBase, we can postulate candidate genes for
downstream validation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Due to the sheer magnitude of the datasets produced by NGS
experiments, managing and interpreting variant lists becomes a cumbersome and
time-consuming task. As more and more data is generated, interesting complex
cross-sample questions become more relevant but are hard to achieve. With
MutaBase, we aimed to deliver a tool which handles such data conveniently and
easily and which provides a front-end for the biologist or physician to quickly
gain new insights from the data. In the future we plan to broaden the scope of
MutaBase further by integrating more knowledge and prediction software.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://homes.esat.kuleuven.be/%7Easifrim/mutabase" mce_href="http://homes.esat.kuleuven.be/~asifrim/mutabase">http://homes.esat.kuleuven.be/~asifrim/mutabase</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Alejandro Sifrim (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy20495 = '&#97;l&#101;j&#97;ndr&#111;.s&#105;fr&#105;m' + '&#64;';
 addy20495 = addy20495 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy20495 + suffix + '\'' + attribs + '>' );
 document.write( addy20495 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94231 = '&#97;l&#101;j&#97;ndr&#111;.s&#105;fr&#105;m' + '&#64;';
 addy94231 = addy94231 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy94231 + suffix + '\'' + attribs + '>' );
 document.write( addy94231 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55615 = '&#97;l&#101;j&#97;ndr&#111;.s&#105;fr&#105;m' + '&#64;';
 addy55615 = addy55615 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy55615 + suffix + '\'' + attribs + '>' );
 document.write( addy55615 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics Group, Department of Electrical
Engineering (ESAT-SCD), University of Leuven, Belgium. (2) Center for Human
Genetics, University Hospital Gasthuisberg Leuven, Belgium. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection60">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741063">A-59. Mining cleavage
sites of the mouse peptidome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Grave K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Grave K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Guns T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Guns T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vandekerckhove TTM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandekerckhove TTM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Landuyt B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Landuyt B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Menschaert G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Menschaert G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Van Criekinge W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Criekinge W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Schoofs L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schoofs L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Luyten W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Luyten W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4)</p>

<p class="abmotivation">Recently, advances in peptidomics technology [1] have
permitted identification from MALDI- or ESI-TOF/TOF mass spectra of over 100
novel peptides in tissues or body fluids of mice. The precursor proteins of all
observed peptides could be identified. This enables data analysis of the
cleavage sites, with the prospect of obtaining clues about novel cleavage
enzymes or mechanisms. [1] K. Boonen, B. Landuyt, G. Baggerman, S.J. Husson, J.
Huybrechts, L. Schoofs. Peptidomics: the integrated approach of MS, hyphenated
techniques and bioinformatics for neuropeptide analysis. J Sep Sci. 2008.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The problem of finding patterns in the cleavage
locations is formulated as an item set mining task and solved using the CP4IM
constraint-based itemset miner [2]. Ranking and pattern redundancy is
controlled using the statistical methods described in Mining Informative
Non-redundant Itemsets (MINI) [3]. We also apply established bioinformatics
tools such as sequence logos. [2] L. De Raedt, T. Guns and S. Nijssen.
Constraint programming for itemset mining. In ACM SIGKDD, 2008. [3] A. Gallo,
T. De Bie and N. Cristianini. MINI: Mining informative non-redundant itemsets.
In PKDD, 2007.</p>

<h3>Results</h3>

<p class="abresults">[Preliminary analysis.] The cleavage sites are significantly
different from the background model (independent and identically distributed
amino acids according to the abundance in the mouse proteome). There is also a
clear difference between previously known peptides and the ones observed by
mass spectrometry during the project. Some patterns found confirm secondary
cleavage documented in the literature, such as C-terminal dibasic
KR-elimination. We are in the process of evaluating a conjectured leucine
pattern.</p>

<h3>Discussion</h3>

<p class="abdiscussion">While distant (&gt;2AA from the cleavage point) positions
contain hardly any information individually, longer combinatorial patterns can be
found that remain significant after conservative Bonferroni correction,
revealing potential recognition sites for endopeptidases.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.peptidomics.be" mce_href="http://www.peptidomics.be">http://www.peptidomics.be</a><span style=""> and </span><a href="http://dtai.cs.kuleuven.be/CP4IM/" mce_href="http://dtai.cs.kuleuven.be/CP4IM/">http://dtai.cs.kuleuven.be/CP4IM/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Kurt De Grave (<a href="/Rebholz-srv/eccb2010/../../Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60076 = 'k&#117;rt.d&#101;gr&#97;v&#101;' + '&#64;';
 addy60076 = addy60076 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy60076 + suffix + '\'' + attribs + '>' );
 document.write( addy60076 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="http://mce_host/Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33512 = 'k&#117;rt.d&#101;gr&#97;v&#101;' + '&#64;';
 addy33512 = addy33512 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy33512 + suffix + '\'' + attribs + '>' );
 document.write( addy33512 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83400 = 'k&#117;rt.d&#101;gr&#97;v&#101;' + '&#64;';
 addy83400 = addy83400 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy83400 + suffix + '\'' + attribs + '>' );
 document.write( addy83400 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">K.U.Leuven, DTAI</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) DTAI, Department of Computer Science, Faculty of
Engineering, K.U.Leuven, Leuven, Belgium (2) Department of Molecular
Biotechnology, Faculty of Bioscience Engineering, Laboratory for Bioinformatics
and Computational Genomics, Ghent University, Ghent, Belgium (3) Biology
Department, K.U.Leuven, Leuven, Belgium (4) Biomedical Sciences Group,
K.U.Leuven, Leuven, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Kurt De Grave is funded by GOA/08/008
"Probabilistic Logic Learning". Tias Guns is funded by the Agency for
Innovation by Science and Technology in Flanders (IWT). The project was
supported by IWT grant SBO 50164 to L. Schoofs. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection61">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741064">A-60. Properties of
plant microRNAs affect miRDeep statistical scoring as well as accuracy</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Thakur V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Thakur V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Wanchana S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wanchana S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Xu M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Xu M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2, Bruskiewich R (1), Mosig A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mosig A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Zhu X<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zhu X&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">There are several tools which can identify miRNAs from
deep-sequencing, however only a few of them like miRDeep, which can identify
novel ones and also being available as standalone application. Given the
difference between plant and animal miRNAs, particularly in terms of
distribution of precursor length and complementarity, it’s likely that the
underlying prediction measures get affected. We studied the key prediction
measures which get affected for plant specific miRNA prediction, in particular,
those employed by miRDeep, and also attempted to estimate the new set of
parameters.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The said effect on measures like minimum free
energy, stability of secondary structures, excision length was examined, and
the parameters of those, which displayed sizable changes, were estimated for
plant specific miRNAs. Further, the fine-tuned miRDeep was tested on a Illumina
dataset from maize leaves and compared the prediction with those obtained using
the default parameters. The targets of these candidates was also predicted
followed by their functional enrichment analysis.</p>

<h3>Results</h3>

<p class="abresults">We found majority of the measures to take new set of
values/distributions for plant specific miRNAs. While nucleus or seed region
was relatively longer, the difference between distribution of minimum free
energy was marginal. We identified 43 miRNA candidates; 13 of them were
apparently novel, not showing any homology with any of the known plant miRNAs.
The target prediction and its enrichment analysis, however, showed mixed
results</p>

<h3>Discussion</h3>

<p class="abdiscussion">The present study characterizes the effect of precursor
length on MFE, and several other measures, which provide insight into their use
in plant miRNA prediction. Further, researchers applying miRDeep in plants, can
improve their results by using the new set of parameters.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Vivek Thakur (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74448 = 'v.th&#97;k&#117;r' + '&#64;';
 addy74448 = addy74448 + '&#105;rr&#105;' + '&#46;' + '&#111;rg';
 document.write( '<a ' + path + '\'' + prefix + addy74448 + suffix + '\'' + attribs + '>' );
 document.write( addy74448 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5788 = 'v.th&#97;k&#117;r' + '&#64;';
 addy5788 = addy5788 + '&#105;rr&#105;' + '&#46;' + '&#111;rg';
 document.write( '<a ' + path + '\'' + prefix + addy5788 + suffix + '\'' + attribs + '>' );
 document.write( addy5788 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26458 = 'v.th&#97;k&#117;r' + '&#64;';
 addy26458 = addy26458 + '&#105;rr&#105;' + '&#46;' + '&#111;rg';
 document.write( '<a ' + path + '\'' + prefix + addy26458 + suffix + '\'' + attribs + '>' );
 document.write( addy26458 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">International Rice Research Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. International Rice Research Institute (IRRI), Los
Banos, Philippines. 2. MPG-CAS Partner Institute of Computational Biology
(PICB), 320 Yue Yang Road, Shanghai, China.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work has been done under C4 Rice project funded by
Bill and Melinda Gates Foundation. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection62">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741065">A-61. Using the Amazon
elastic cloud computing resource for the analysis of next-generation sequencing
data in Ensembl</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vogel J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Vogel J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Aken B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aken B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Chiang G-T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chiang G-T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Clapham P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Clapham P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Coates G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Coates G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Fairley S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fairley S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Hourlier T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hourlier T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Ruffier M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ruffier M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Tang A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tang A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), White S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;White S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Zadissa A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zadissa A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Searle S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Searle S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Hubbard T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hubbard T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Ensembl is one of the leading sources of genome sequence
annotation data for a variety of different genomes. Ensembl genome annotation is
currently available for more than 50 species, ranging from high-coverage
reference genomes to genomes with highly fragmented, low-coverage assemblies.
With the arrival of the Next-Generation transcriptome sequencing, a novel,
valuable data source has become available to further extend our ongoing effort
in annotating genome sequences. This new data type provides an unprecedented
view of the whole transcriptome for coding- and non-coding transcripts.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Such data is available for different tissues, cell
compartments and different developmental stages. The analysis of transcript
expression levels and alternative splice forms can now be integrated into our
gene annotations. The large amount of available data produced by NGS platforms
sets new challenges to the development of bioinformatic tools and requires the
access to a powerful compute infrastructure for a short time to process, map
and analyze the sequence data. A cloud computing environment like the Amazon
EC2 offers an affordable way to analyze large scale data sets.</p>

<h3>Results</h3>

<p class="abresults">With the development of our Ensembl RNA sequence analysis
pipeline (publication in preparation), we are now to able to handle and analyze
large NGS based transcriptome data sets. This enables us to quantify short-read
support for exons and introns of existing Ensembl annotation and, depending on
the NGS input data, to produce complete de-novo gene sets for different
tissues. Here, we give a detailed overview of how various Ensembl analysis
modules can be configured and run in the Amazon EC2 environment using Sun
GridEngine as a Job Submission System.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We provide a practical example of how to use our
pre-configured Amazon machine images to process and analyze NGS data with the
Ensembl RNA sequence pipeline. With this infrastructure in place, the
adaptation of further standard Ensembl analyses to the EC2 environment is
facilitated. Additionally, we show how the Distributed Annotation System DAS
can be used to display the obtained results in Ensembl. The analysis of NGS
transcriptome data in the Amazon cloud with the Ensembl analysis pipeline
offers a modular solution to analyse large-scale data sets without in-house
compute infrastructures.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ensembl.org" mce_href="http://www.ensembl.org">http://www.ensembl.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jan H Vogel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40041 = 'j&#97;n.v&#111;g&#101;l' + '&#64;';
 addy40041 = addy40041 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy40041 + suffix + '\'' + attribs + '>' );
 document.write( addy40041 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71392 = 'j&#97;n.v&#111;g&#101;l' + '&#64;';
 addy71392 = addy71392 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy71392 + suffix + '\'' + attribs + '>' );
 document.write( addy71392 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14485 = 'j&#97;n.v&#111;g&#101;l' + '&#64;';
 addy14485 = addy14485 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy14485 + suffix + '\'' + attribs + '>' );
 document.write( addy14485 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Wellcome Trust Sanger Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Wellcome Trust Sanger Institute (2) European
Bioinformatics Institute</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection63"></div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection64">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269741066">A-62. Automated sequence
extraction of relevant genomic features for targeted resequencing</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Wilde B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Wilde B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), D’Hont B (1), Lefever S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lefever S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Hellemans J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hellemans J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Speleman F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Speleman F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Pattyn F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pattyn F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vandesompele J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandesompele J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Massively parallel sequencing technologies enable
researchers to determine the nucleotide sequence of DNA at ever-increasing
throughput and decreasing cost. Focusing this sequencing power to a genomic
region of interest to perform targeted resequencing is an important challenge
in genetics. We created a web tool to assist in extracting sequences from
relevant genomic features (e.g. coding exons) in large regions typically
defined by by linkage analysis, homozygosity mapping or arrayCGH.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The tool is based on locally stored annotation
tables originating from the genome databases of NCBI, UCSC, Ensembl, and other
specialized sources of annotation information. The database content is managed
by automated update scripts to allow version tracking and re-formatting of the
data into a common pre-defined annotation format. This enables easy comparison
and merging of the tables resulting in an integrated resource of high quality
genome annotation information. Depending on the downstream application a
reference table can be built containing only the required annotation
information.</p>

<h3>Results</h3>

<p class="abresults">The reference tables form the basis for an easy to use interface
for genomic target selection. This tool allows a scientist without
bioinformatics skills to select annotation information for a large genomic
region based upon its characteristic features (e.g. protein coding, pseudogene
or a non-coding element; a transcribed, untranscribed or translated part of an
element; etc.). The annotated information can be visualized in a genome browser
and outputted to a format ready for DNA chip or PCR based target selection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We present an easy to use web tool for genomic target
selection based upon the most complete and uniform source of annotation
information currently available. The tool itself has a focus on usability and
should allow biologists to easily prepare their next generation sequencing
experiments. We present results on the practical application of this tool for
the selection of highly dispersed short non coding RNA genes for PCR based
target selection as well as Nimblegen chip based capture of regions of several
megabases in size for some mendelean human disorders.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.mellfire.ugent.be/NGSF" mce_href="http://www.mellfire.ugent.be/NGSF">http://www.mellfire.ugent.be/NGSF</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Bram De Wilde (<a href="/Rebholz-srv/eccb2010/../../Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86439 = 'br&#97;m.d&#101;w&#105;ld&#101;' + '&#64;';
 addy86439 = addy86439 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86439 + suffix + '\'' + attribs + '>' );
 document.write( addy86439 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="http://mce_host/Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69416 = 'br&#97;m.d&#101;w&#105;ld&#101;' + '&#64;';
 addy69416 = addy69416 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy69416 + suffix + '\'' + attribs + '>' );
 document.write( addy69416 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10526 = 'br&#97;m.d&#101;w&#105;ld&#101;' + '&#64;';
 addy10526 = addy10526 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy10526 + suffix + '\'' + attribs + '>' );
 document.write( addy10526 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Medical Genetics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Center for Medical Genetics, Ghent University,
Ghent, Belgium (2) NXTGNT, Ghent University, Ghent, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Research Foundation - Flanders (FWO) Special Research
Fund Ghent University (BOF-UGent)<a class="mceItemAnchor" name="_Toc269741067"></a><br mce_bogus="1"></p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="113-sequence-analysis-alignment-and-next-generation-sequencing.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>