<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Comparative Genomics, Phylogeny and Evolution</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Comparative Genomics, Phylogeny and Evolution			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="text-transform: uppercase;" mce_style="text-transform:uppercase">Comparative
Genomics, Phylogeny and Evolution</span><span style="text-transform: uppercase;" mce_style="text-transform:uppercase"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Martijn Huynen and Yves Van de Peer<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Comparative Genomics,
Phylogeny and Evolution<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span> PAGEREF _Toc269742381 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742382" mce_href="#_Toc269742382">B-1. The role of transposable
elements in the evolution of non-mammalian vertebrates and invertebrates<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742382 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">3<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742383" mce_href="#_Toc269742383">B-2. A tool for comparison of
complete proteomes between pairs of organisms<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742383 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">4<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742384" mce_href="#_Toc269742384">B-3. Toward an unified measure of
intraspecific selective pressure<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742384 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742385" mce_href="#_Toc269742385">B-4. Assessment of genetic structure
within and among Normandie (Northern France) populations of wild beet (Beta
vulgaris sps. maritima Arcang.)<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742385 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742386" mce_href="#_Toc269742386">B-5. Trees inside trees for genome
wide association studies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span> PAGEREF _Toc269742386 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742387" mce_href="#_Toc269742387">B-6. Validation of single cell
arrayCGH on a 60-mer oligo microarray platform for preimplantation genetic
diagnosis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742387 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742388" mce_href="#_Toc269742388">B-7. Polymorphisms associated with
mtDNA of elite Kenyan endurance athletes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742388 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742389" mce_href="#_Toc269742389">B-8. Bayesian inference of community
average gene copy numbers and its application for the metagenomic
characterization of bacterioplankton community types in the Sargasso Sea<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742389 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300380039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742390" mce_href="#_Toc269742390">B-9. Phylogenetic mapping of
non-model organism RNA-seq reads using a graph algorithm<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:
windowtext;display:none;mso-hide:screen;text-decoration:none;text-underline:
none"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742390 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742391" mce_href="#_Toc269742391">B-10. GEI-DB: a database of atypical
genomic elements in bacteria<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742391 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742392" mce_href="#_Toc269742392">B-11. The CNS twilight zone:
limitations in comparing upstream regions in plants<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742392 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742393" mce_href="#_Toc269742393">B-12. TurboOrtho: a high performance
alternative to OrthoMCL<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span> PAGEREF _Toc269742393 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742394" mce_href="#_Toc269742394">B-13. i-ADHoRe 3.0 Detection of
collinearity in large scale datasets<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742394 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742395" mce_href="#_Toc269742395">B-14. Comparative expression
analysis of orthologous genes between Arabidopsis and rice<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742395 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742396" mce_href="#_Toc269742396">B-15. Phylogenomics and robust construction
of prokaryotic evolutionary trees<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742396 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">18<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742397" mce_href="#_Toc269742397">B-16. Mapping the human membrane
proteome: a majority of the human membrane proteins can be classified according
to function and evolutionary origin<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742397 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">19<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742398" mce_href="#_Toc269742398">B-17. Towards molecular trait-based ecology
through integration of biogeochemical, geographical and metagenomic data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742398 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">20<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742399" mce_href="#_Toc269742399">B-18. Orthologous proteins
associated with yeast telomeric complex identified by synteny and sequence
similarity<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742399 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">21<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003300390039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742400" mce_href="#_Toc269742400">B-19. Estimating average
heterozygosity using multilocus dominant marker data: a maximum likelihood
approach via the EM algorithm.<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742400 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">22<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742401" mce_href="#_Toc269742401">B-20. PLAZA 2.0 : a resource for plant
comparative genomics<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742401 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">23<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742402" mce_href="#_Toc269742402">B-21. Branch testing in phylogenetic
trees: comparison of computationally efficient methods<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742402 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">24<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742403" mce_href="#_Toc269742403">B-22. Parsimonious higher-order
hidden Markov models for enhanced comparative genomics<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742403 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">25<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742404" mce_href="#_Toc269742404">B-23. Correction of bootstrap
confidence levels using an iterated bootstrap procedure with computational
efficient methods<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742404 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">26<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742405" mce_href="#_Toc269742405">B-24. Molecular evolution, structure
and functional divergence of Lipoxygenase gene family in vertebrate<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742405 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">27<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742406" mce_href="#_Toc269742406">B-25. Cassis: detection of genomic
rearrangement breakpoints<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span> PAGEREF _Toc269742406 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">28<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742407" mce_href="#_Toc269742407">B-26. Methylated cytosines are less
likely to mutate within CpG islands<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742407 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">29<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742408" mce_href="#_Toc269742408">B-27. Entropy approach reveals new
features of genes and genomes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742408 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">30<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742409" mce_href="#_Toc269742409">B-28. The hidden duplication past of
the plant pathogen Phytophthora and its consequences for infection<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742409 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">31<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400300039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742410" mce_href="#_Toc269742410">B-29. Protein linkages between
African Trypanosoma and 8 pathogenetic eukaryotic organisms<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742410 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">32<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742411" mce_href="#_Toc269742411">B-30. Reconstructing phylogenetic
trees from clustering trees<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742411 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">33<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742412" mce_href="#_Toc269742412">B-31. Predictive modeling of
psychrophilic adaptation on the proteome sequence level<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742412 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">34<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742413" mce_href="#_Toc269742413">B-32. Microbial phenotype prediction
based on protein domain profiles<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742413 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">35<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742414" mce_href="#_Toc269742414">B-33. New insights into the metazoan
evolution of cadherins: from basal to modern<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742414 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">36<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742415" mce_href="#_Toc269742415">B-34. Comparative mapping of
transcription factor binding sites in plant genomes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742415 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">37<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742416" mce_href="#_Toc269742416">B-35. Protein model selection with
ProtTest increases phylogenetic performance<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742416 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">38<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742417" mce_href="#_Toc269742417">B-36. cn.FARMS: a probabilistic
latent variable model to detect copy number variations<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742417 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">39<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742418" mce_href="#_Toc269742418">B-37. Genome-wide heterogeneity of
the substitution process<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span> PAGEREF _Toc269742418 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">40<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742419" mce_href="#_Toc269742419">B-38. Comparative microarray
analysis to elucidate TF-networks in activated T-cells<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742419 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">41<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742420" mce_href="#_Toc269742420">B-39. Gene-trait matching analysis
of Lactobacillus plantarum strains<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742420 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">42<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742421" mce_href="#_Toc269742421">B-40. ProtTest-HPC: fast selection
of best-fit models of protein evolution<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742421 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">43<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742422" mce_href="#_Toc269742422">B-41. NTRFinder: an algorithm to
find nested tandem repeats<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742422 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">44<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742423" mce_href="#_Toc269742423">B-42. Computational epigenomics of
plant SNF2 genes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742423 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">45<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="115-comparative-genomics-phylogeny-and-evolution.html#_Toc269742424" mce_href="#_Toc269742424">B-43. Comprehensive analysis of
splice site evolution in primates using whole genome alignments and RNA-Seq
data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> PAGEREF
_Toc269742424 \h <span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">46<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340032003400320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<b><span style="font-size: 13pt; line-height: 115%;" mce_style="font-size:13.0pt;line-height:115%;font-family:" ,="" serif="" ;="" mso-fareast-font-family:="" times="" new="" roman="" ;color:black;mso-ansi-language:en-us;="" mso-fareast-language:en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before: always;" clear="all">
</span></b></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742382">B-1. The role of
transposable elements in the evolution of non-mammalian vertebrates and
invertebrates</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sela N<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Sela N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Kim E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kim E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ast G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ast G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Transposable elements (TEs) have played an important role
in the diversification and enrichment of mammalian transcriptomes through
various mechanisms such as exonization and intronization (the birth of new
exons/introns from previously intronic/exonic sequences, respectively), and
insertion into first and last exons. However, no extensive analysis has
compared the effects of TEs on the transcriptomes of mammalian, non-mammalian
vertebrates and invertebrates.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We determined the TE fraction within intronic sequences
using the UCSC genome browser and GALAXY. Introns of chicken (G. gallus, Build
1.1), zebrafish (D. rerio, release Zv4), C. elegans (Release 2003) and D.
melanogaster (Build 4.1) were extracted from the Exon-Intron Database
(http://hsc.utoledo.edu/depts/bioinfo/database.html). When alternatively
spliced isoforms of the same gene were present, only the first annotated
isoform was extracted; all other isoforms were excluded in order to avoid
redundancy. The analysis of the TE content was done using RepeatMasker software
and rep</p>

<h3>Results</h3>

<p class="abresults">We analyzed the influence of TEs on the transcriptomes of
five species, three invertebrates and two non-mammalian vertebrates. Compared
to previously analyzed mammals, there were lower levels of TE introduction into
introns, significantly lower numbers of exonizations originating from TEs and a
lower percentage of TE insertion within the first and last exons. Although the
transcriptomes of vertebrates exhibit a significant level of exonizations of
TEs, only anecdotal cases were found in invertebrates. In vertebrates, as in
mammals, the exonized TEs are mostly alternatively spliced, indicating</p>

<h3>Discussion</h3>

<p class="abdiscussion">Exonization of TEs is wide-spread in mammals, less so in
non-mammalian vertebrates, and very low in invertebrates. We assume that the
exonization process depends on the length of introns. Vertebrates, unlike
invertebrates, are characterized by long introns and short internal exons. Our
results suggest that there is a direct link between the length of introns and
exonization of TEs and that this process became more prevalent following the
appearance of mammals.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Noa Sela (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90730 = 'n&#111;&#97;.s&#101;l&#97;5' + '&#64;';
 addy90730 = addy90730 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy90730 + suffix + '\'' + attribs + '>' );
 document.write( addy90730 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4659 = 'n&#111;&#97;.s&#101;l&#97;5' + '&#64;';
 addy4659 = addy4659 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy4659 + suffix + '\'' + attribs + '>' );
 document.write( addy4659 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32369 = 'n&#111;&#97;.s&#101;l&#97;5' + '&#64;';
 addy32369 = addy32369 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy32369 + suffix + '\'' + attribs + '>' );
 document.write( addy32369 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ludwig-maximilans university Munich</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department Biology I, Ludwig-Maximilians-University
Munich (LMU) Großhaderner Str. 2, D-82152, Planegg-Martinsried, Germany 2.
1Department of Human Molecular Genetics, Sackler Faculty of Medicine, Tel Aviv
University, Tel Aviv 69978, Israel</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Cooperation Program in
Cancer Research of the Deutsches Krebsforschungszentrum (DKFZ) and Israel’s
Ministry of Science and Technology (MOST) and by a grant from the Israel
Science Foundation (40/05), ICRF, DIP and EURASNET. NS is supported by the LMU
excellence fellowship. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742383">B-2. A tool for
comparison of complete proteomes between pairs of organisms</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Karathia H (1, *), Alves R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alves R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Assignment of protein function to newly discovered or
sequenced proteins based on the similarity of their sequence to that of other
proteins of well-known function is a critical and fundamental problem in
molecular system biology, comparative genomics and molecular biology. Although
there are many tools that permit doing such an assignment on a protein by
protein basis, very few tools are available to compare full proteomes between
organisms.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a tool that has been developed to
perform: (a) a full comparison of the proteomes between two organisms (b)
discrimination between orthologs and in-paralogs in each of the organisms being
compared (c) full comparison of the functional pathways and processes in the
two organisms to a third model organism in order to assess how similar or
different specific physiological responses for the same biological processes
may be between the organisms. The tool is implemented in PERL and graphical
representations are generated in Mathematica.</p>

<h3>Results</h3>

<p class="abresults">As a case study, we apply the tool to compare the proteome
of 59 eukaryotic organisms with fully sequenced genomes to that of
Saccharomyces cerevisiae. We identify different gene duplication and deletion
events between each organism and the yeast. We use the results to predict how
good a model organism is S. cerevisiae for the study of different biological
processes and pathways in other eukaryotes. We rank both organisms and
processes/pathways as to their probable closeness to those in S. cerevisiae.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our analysis and results are supported by the fact that
fungi organisms are predicted to be the closest to S. cerevisiae. Candida
albicans in particular is an exceptional species in fungi, because it has the
highest proportion of duplicated proteins relative to S. cerevisiae. Our tool
accurately classifies the duplications and deletions of genes between pairs of
organisms. It also appears to be a good predictor of qualitative differences in
the adaptive responses of corresponding biological processes in different
organisms.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Hiren Karathia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97983 = 'h&#105;r&#101;n' + '&#64;';
 addy97983 = addy97983 + 'cmb' + '&#46;' + '&#117;dl' + '&#46;' + 'c&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy97983 + suffix + '\'' + attribs + '>' );
 document.write( addy97983 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33387 = 'h&#105;r&#101;n' + '&#64;';
 addy33387 = addy33387 + 'cmb' + '&#46;' + '&#117;dl' + '&#46;' + 'c&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy33387 + suffix + '\'' + attribs + '>' );
 document.write( addy33387 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21210 = 'h&#105;r&#101;n' + '&#64;';
 addy21210 = addy21210 + 'cmb' + '&#46;' + '&#117;dl' + '&#46;' + 'c&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy21210 + suffix + '\'' + attribs + '>' );
 document.write( addy21210 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Departament Ciències Mèdiques Bàsiques, Universitat
de Lleida and IRBLleida</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Departament Ciències Mèdiques Bàsiques, Universitat de
Lleida and IRBLleida, Montserrat Roig 2, 25008 Lleida, Spain (1)</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially funded by a Ramon y Cajal
Research Award and by grant BFU2007-62772/BMC from the Spanish Ministry of
Science and Innovation to RA and by a Generalitat de Catalunya Ph. D.
fellowship to HK. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742384">B-3. Toward an unified
measure of intraspecific selective pressure</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Amato R<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Amato R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Miele G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Miele G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Pinelli M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pinelli M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Cocozza S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cocozza S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">During recent decades the study of human evolution has
been of increasing interest, also due to the large amount of data now
available. In the mean time, new applications of evolutionary biology to
medical problems are being discovered at an accelerating rate. Several
estimators for the selective pressure have been proposed. Being introduced to
face different aspects of selective pressure, each measure has its own pros and
cons.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We focused on the most widely used intraspecific
estimators of selective pressure. In particular, to cope with both inter- and
intra-group phenomena, we analysed the fixation index (FST) and some measures
based on the extended haplotype homozygosity (namely REHH, iHS, XP-EHH). We
assessed the performances of each measure on simulated data produced using
Fregene, a tool developed and calibrated to reproduce, in a biologically sound
manner, our evolutionary history, also taking into account for complex
demographic, selection and recombination scenarios.</p>

<h3>Results</h3>

<p class="abresults">By using GRID facilities to produce a large enough amount of
data, we were able to exhaustively assess the performances of each estimator
with regard to several biological parameters of the sites under selection (e.g.
selection coefficient, dominance model, age of selection).</p>

<h3>Discussion</h3>

<p class="abdiscussion">The study of human evolution is obtaining increasing
interest also for its potential medical applications and until now several
measure of selective pressure have been proposed. Our analysis highlighted a
complementary behaviour of these estimators suggesting a possible strategy of
merging them in an unified and versatile measure of selective pressure.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Roberto Amato (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8844 = 'r&#111;&#97;m&#97;t&#111;' + '&#64;';
 addy8844 = addy8844 + 'n&#97;' + '&#46;' + '&#105;nfn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy8844 + suffix + '\'' + attribs + '>' );
 document.write( addy8844 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy70649 = 'r&#111;&#97;m&#97;t&#111;' + '&#64;';
 addy70649 = addy70649 + 'n&#97;' + '&#46;' + '&#105;nfn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy70649 + suffix + '\'' + attribs + '>' );
 document.write( addy70649 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66439 = 'r&#111;&#97;m&#97;t&#111;' + '&#64;';
 addy66439 = addy66439 + 'n&#97;' + '&#46;' + '&#105;nfn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy66439 + suffix + '\'' + attribs + '>' );
 document.write( addy66439 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dipartimento di Scienze Fisiche - Universita' degli
Studi di Napoli "Federico II"</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Gruppo Interdipartimentale di Bioinformatica e
Biologia Computazionale, Università di Napoli "Federico II" &amp;
Università di Salerno (2) Dipartimento di Scienze Fisiche, Università di Napoli
"Federico II" - INFN Sezione di Napoli (3) Dipartimento di Biologia e
Patologia Cellulare e Molecolare "L. Califano", Università di Napoli
"Federico II"</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742385">B-4. Assessment of
genetic structure within and among Normandie (Northern France) populations of
wild beet (Beta vulgaris sps. maritima Arcang.)</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tran HT<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Tran HT&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Cuguen J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cuguen J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Touzet P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Touzet P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Saumitou-Maprade P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Saumitou-Maprade P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The examination of coastal northern France (Normandie)
beet populations is of particular concern for competent authorities for the
regulation of transgenic organisms (GMO) due to the close proximity of wild
beet to sugar beet fields. Gene flow from cultivated beets in this area is
theoretically possible, if fields near wild beet habitats contain vernalized
sugar beets or annual weed types. For purpose of in situ conservation of this
genetic resource . For crop improvement, it is very important to examine
genetic diversity of sea beet population in Normandie where still is a little
information about this.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Young leafs gathered from from 396 plants in 9
populations coastal northern France. Nuclear microsatellite loci (codominant
marker)used to analysis genetic variations.</p>

<h3>Results</h3>

<p class="abresults">The global inbreeding coefficient Fit is 0.3336 , meaning
that overall there is evidence for quite substantial inbreeding. We see that
17% of genetic variation is found among populations and 83% within
popupulations. The Fst/(1-Fst) ratio for pairs of populations increased no
linearly with the natural logarithm of the geographical distance. (Mantel test:
P = 0.0005, R 2 = 0.0099), showing a pattern of no isolation by distance
(Rousset 1997). There are two groups of populations widely separated: one group
of three populations, fec4, veu5 and val6; the other with all 6 other
populations. Within the latter group, mars-08 and pou9 are very closely related
to each other.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Information of biological characteristics from this study
is usseful for consevation management such as the level of inbreeding, the population
bottlenecks and the variance of gene dispersal distances.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://gepv.univ-lille1.fr/english/perso_pages_en" mce_href="http://gepv.univ-lille1.fr/english/perso_pages_en">http://gepv.univ-lille1.fr/english/perso_pages_en</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Hoa Thi Tran (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14160 = 'tr&#97;nth&#105;h&#111;&#97;' + '&#64;';
 addy14160 = addy14160 + '&#97;g&#105;' + '&#46;' + 'v&#97;&#97;s' + '&#46;' + 'vn';
 document.write( '<a ' + path + '\'' + prefix + addy14160 + suffix + '\'' + attribs + '>' );
 document.write( addy14160 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16604 = 'tr&#97;nth&#105;h&#111;&#97;' + '&#64;';
 addy16604 = addy16604 + '&#97;g&#105;' + '&#46;' + 'v&#97;&#97;s' + '&#46;' + 'vn';
 document.write( '<a ' + path + '\'' + prefix + addy16604 + suffix + '\'' + attribs + '>' );
 document.write( addy16604 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33874 = 'tr&#97;nth&#105;h&#111;&#97;' + '&#64;';
 addy33874 = addy33874 + '&#97;g&#105;' + '&#46;' + 'v&#97;&#97;s' + '&#46;' + 'vn';
 document.write( '<a ' + path + '\'' + prefix + addy33874 + suffix + '\'' + attribs + '>' );
 document.write( addy33874 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Agricultural Genetics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1*.Institute of Agricultural Genetics, Laboratory for
Forest Genetics and Conservation, Hanoi 10000, Vietnam 2.Université des
Sciences et Technologies de Lille I, Laboratoire de Génétique et Evolution des
Populations Végétales, UPRESA 8016 du CNRS, Bâtiment SN2, 59655 Villeneuve
d'Ascq Cedex, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">AUPEL-UREF- Bourses d’excellence</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742386">B-5. Trees inside trees
for genome wide association studies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Botta V<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Botta V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Geurts P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Geurts P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Wehenkel L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wehenkel L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In the field of genome wide association studies, the
individuals are described by hundreds of thousands of SNPs. Due to linkage
disequilibrium, these descriptors are correlated. We propose here to exploit
these correlations inside Random Forest by treating attribute groups instead of
single variables within test-nodes. We expect that grouped variables can be
exploited to construct more efficient tests and thus better predictors. Indeed,
when a SNP is associated with the disease, its neighbors are expected to be
disease associated too.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">More precisely, each test-node becomes a node
limited totally randomized tree learned from a subgroup of variables, which, in
this case, correspond to a group of adjacent SNPs. During the learning process,
at each node, K decision trees are constructed on K randomly picked groups, the
most informative one is kept to split the learning set given a threshold on the
probabilities generated by the resulting tree. Inside these tree test-nodes,
the best nodes are expanded first, then tree complexity (called here the
internal complexity) can be limited to use only some of the SNPs from a group.</p>

<h3>Results</h3>

<p class="abresults">This approach has been applied on simulated data and on a
real dataset related to Crohn’s disease. The results are compared to
Random-Forest testing individual SNPs. The resulting models are far less
complex due to the dimension reduction implied by arbitrarily grouping adjacent
SNPs. Also, we observe a predictive power increase with bigger values of the
internal complexity. Furthermore, we extended variable importances to block
importances which allow us to directly highlight chro- mosomal regions of
interests.</p>

<h3>Discussion</h3>

<p class="abdiscussion">More generally, facing the increasing amount of data in
GWAS, what we propose here is to use weak learners as test-nodes into decision
trees in order to reduce the problem dimensionnality given the data structure.
Further works will focus our efforts on exploring the internal complexity
parameter space and on testing variant weak learners (like ensemble of stumps,
ensemble of trees or HMM models) as test- nodes.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Vincent Botta (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy878 = 'v&#105;nc&#101;nt.b&#111;tt&#97;' + '&#64;';
 addy878 = addy878 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy878 + suffix + '\'' + attribs + '>' );
 document.write( addy878 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61067 = 'v&#105;nc&#101;nt.b&#111;tt&#97;' + '&#64;';
 addy61067 = addy61067 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy61067 + suffix + '\'' + attribs + '>' );
 document.write( addy61067 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32610 = 'v&#105;nc&#101;nt.b&#111;tt&#97;' + '&#64;';
 addy32610 = addy32610 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy32610 + suffix + '\'' + attribs + '>' );
 document.write( addy32610 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Liege : Department of Electrical
Engineering and Computer Science &amp; GIGA-Research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) University of Liege - Department of Electrical
Engineering and Computer Science and GIGA-Research</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This paper presents research results of the Belgian
Network BIOMAGNET (Bioinformatics and Modeling: from Genomes to Networks),
funded by the Interuniversity Attraction Poles Programme, initiated by the
Belgian State, Science Policy Office. VB is recipient of a F.R.I.A. fellowship
and PG is a Research Associate of the F.R.S.-FNRS. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742387">B-6. Validation of
single cell arrayCGH on a 60-mer oligo microarray platform for preimplantation
genetic diagnosis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cheng J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Cheng J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Vanneste E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vanneste E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Konings P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Konings P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Van Eyndhoven W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Eyndhoven W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Voet T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Voet T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ampe M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ampe M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Verbeke G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Verbeke G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Vermeesch J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vermeesch J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Preimplantation Genetic Diagnosis with Aneuploidy
Screening (PGD-AS) is used for the selection of genetically normal embryos
before implantation, thus increasing both baby take home rate and health of the
fetuses. Array CGH technologies have been employed to detect chromosomal
anerploidies in single blastomeres on a genome-wide level. The aim of this
study was to explore whether the use of high resolution Agilent 244K Human CGH
arrays would improve the detection of segmental aneuploidies in single cells compared
to the previously used BAC and SNP arrays (Vanneste et al., Nat Med, 2009).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In total, 8 Epstein Barr Virus (EBV) transformed
lymphoblastoids and 43 blastomeres derived from human embryos were analyzed.
Data were normalized by the new developed channel based method. Subsequently,
mean-median calculation, Circular binary segmentation (CBS) and Haarseg
algorithms have been optimized and employed to analyze the accuracy of the data
to call for a priori known and/or de novo whole chromosome and segmental
aberrations in single blastomeres.</p>

<h3>Results</h3>

<p class="abresults">We succeeded in developing a new normalization method to
preprocess Agilent 244K human mircroarray data, enabling single cell data
analysis on oligo platforms. We proved that this channel based method can
effectively correct for artifacts/biases introduced by single cell
amplification while maintaining a similar dynamic range of the log-ratios as
compared to unamplified genomic DNA (Cheng et al., BMC Genomics, submitted).
The mean-median calculation method can identify a priori known aberrations
while the CBS and the Haarseg algorithms can detect both a priori known and de
novo aberrations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In conclusion, whole chromosome and segmental imbalances
can be detected at the single cell level using Agilent 244K human CGH
microarrays. However, the aberration calling accuracy of the three detection
algorithms optimized for Agilent is in the same order of magnitude as compared
to BAC and SNP based arrays. Further validation studies are needed before any
array CGH technology can be introduced in the clinic as a standardized assay
for PGD.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jiqiu Cheng (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86374 = 'j&#105;q&#105;&#117;.ch&#101;ng' + '&#64;';
 addy86374 = addy86374 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86374 + suffix + '\'' + attribs + '>' );
 document.write( addy86374 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81569 = 'j&#105;q&#105;&#117;.ch&#101;ng' + '&#64;';
 addy81569 = addy81569 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy81569 + suffix + '\'' + attribs + '>' );
 document.write( addy81569 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32148 = 'j&#105;q&#105;&#117;.ch&#101;ng' + '&#64;';
 addy32148 = addy32148 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy32148 + suffix + '\'' + attribs + '>' );
 document.write( addy32148 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Katholieke Universiteit Leuven, ESAT-SCD:
SISTA/COSIC/DOCARCH, Leuven. 2. Katholieke Universiteit Leuven, Center for
Human Genetics, Leuven, Belgium. 3. Katholieke Universiteit Leuven, Center for
Biostatistics, Leuven, Belgium. 4. Agilent Technologies.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by Research Council K.U.Leuven:
ProMeta, GOA Ambiorics, GOA MaNet, CoE EF/05/007 SymBioSys, START 1, several
PhD/postdoc &amp; fellow grants; Flemish government: FWO-PhD/postdoc grants,
FWO-projects, G.0318.05 (subfunctionalization), G.0553.06 (VitamineD),
G.0302.07 (SVM/Kernel), FWO- research communities (ICCoS, ANMMM, MLDM),
G.0733.09 (3UTR), G.082409 (EGFR), IWT- PhD Grants, Silicos, SBO-BioFrame,
SBO-MoKa, TBM-IOTA3, FOD-Cancer plans; Belgian Federal Science Policy Office:
IUAP P6/25 (BioMaGNet, Bioinformatics and Modeling: from Genomes to Networks,
2007-2011); EU-RTD: ERNSI: European Research Network on System Identification;
FP7-HEALTH CHeartED. We would like to thank Agilent for providing us with the
Agilent 244K arrays. We would like to thank Pascal Yazbeck and Sigrun Jackmaert
for preparing and performing the array experiments. We appreciated technical
discussions with Kristof Engelen, Inge Thijs, Marijke Bauters, Guy Froyen and
Ernesto Iacucci. E.V. was supported by the Institute for the Promotion of
Innovation through Science and Technology in Flanders (IWT-Vlaanderen). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742388">B-7. Polymorphisms
associated with mtDNA of elite Kenyan endurance athletes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Seiler M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Seiler M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Fuku N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fuku N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Diao L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Diao L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Solovyov A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Solovyov A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Seiler S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Seiler S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Tanaka M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tanaka M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Bhanot G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bhanot G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,5,6,7)</p>

<p class="abmotivation">Kenyan dominance in endurance running events in recent
decades is now well-established. Of the top 500 outdoor marathon official
running times, 42% belong to Kenyans, including the top 4. It is natural to
suspect that dominance by a single ethnic group may have a partial genetic
basis. It has also been demonstrated that perhaps the most important
contributor to endurance running success, aerobic capacity, can be partially
attributed to matrilineal inheritance. Since mitochondria are strictly
maternally inherited, mtDNA may harbor polymorphisms associated with endurance
performance.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">5023 mitochondrial sequences were obtained from
MITOMAP and aligned to rCRS using the stretcher algorithm as implemented in the
EMBOSS package. An additional 74 mtDNA samples from elite Kenyan endurance
athletes were sequenced and added to the dataset. All sequences were organized
into mitochondrial haplogroups using known sequence markers and recursive
consensus clustering as implemented in the ConsensusCluster package.
Polymorphisms enriched in the 74 medalists were identified within haplogroups using
a strict permutation test for significance.</p>

<h3>Results</h3>

<p class="abresults">The consensus clustering procedure identified 7 robust
clades (L0/1/2/3/5/6/7) in the data as well as polymorphisms distinguishing
these clades with 90% or higher accuracy under sample bootstrap. Our assignments
agreed with the mtDNA tree in Behar et al, 2008 in common samples. We found 4
SNPs enriched in athletes of the L0 clade, two of which result in
non-synonymous mtDNA mutations. We did not find any SNPs to be significantly
enriched in African samples of the same haplogroup yet absent in runners,
suggesting that SNPs which reduce endurance performance are not under
selection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A previous study of elite Kenyan endurance athletes found
a significant representation of L0 samples and a dearth of L3 samples (Scott et
al., 2008), though no study was made of the polymorphisms which might
contribute to enhanced performance. The SNPs we have identified in the L0 elite
endurance athlete samples studied are virtually absent in the L0 clade members of
the general population, suggesting these polymorphisms are very recent.
Experiments on cell lines derived from samples with these SNPs can be used to
show whether these cells show increased mitochondrial efficiency.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Michael W Seiler (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73971 = 'm&#105;s&#101;&#105;l&#101;r' + '&#64;';
 addy73971 = addy73971 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy73971 + suffix + '\'' + attribs + '>' );
 document.write( addy73971 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94440 = 'm&#105;s&#101;&#105;l&#101;r' + '&#64;';
 addy94440 = addy94440 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy94440 + suffix + '\'' + attribs + '>' );
 document.write( addy94440 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72284 = 'm&#105;s&#101;&#105;l&#101;r' + '&#64;';
 addy72284 = addy72284 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy72284 + suffix + '\'' + attribs + '>' );
 document.write( addy72284 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Rutgers University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) BioMaPs Institute, Rutgers University, Piscataway,
NJ, 08854, USA (2) Physics Department, Princeton University, Princeton, NJ,
08544, USA (3) Sarah W. Stedman Nutrition and Metabolism Center, Durham, North
Carolina 27710, USA (4) Department of Genomics for Longevity and Health, Tokyo
Metropolitan Institute of Gerontology, 35-2 Sakae-cho, Itabashi-ku, Tokyo
173-0015, Japan (5) Simons Center for Systems Biology, Institute for Advanced
Study, Princeton, NJ, 08540, USA (6) Department of Physics; Department of
Molecular Biology &amp; Biochemistry, Rutgers University, Piscataway, NJ,
08854, USA (7) Cancer Institute of New Jersey, 195 Little Albany Street, New Brunswick,
NJ, 08903, USA</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742389">B-8. Bayesian inference
of community average gene copy numbers and its application for the metagenomic
characterization of bacterioplankton community types in the Sargasso Sea</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Beszteri B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Beszteri B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Frickenhaus S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Frickenhaus S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Giovannoni SJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Giovannoni SJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">We recently drew attention to the fact that average
genome sizes (GS) of communities are expected to influence apparent relative
gene abundances in comparative metagenomics (doi: 10.1038/ismej.2010.29). We
suggested a method to account for this possible bias by estimating average GSs
and incorporating them into gene count normalization in a generalized linear
modelling framework. This approach, however, did not account for the
uncertainty in estimated GSs, and led to an arbitrary scaling of inferred
relative gene abundances. The methods presented here remedy these weaknesses.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a fully Bayesian framework for
incorporating gene length, sampling effort and community-averaged genome size
into estimating gene abundances from metagenomic counts. We implemented an
adaptive normal jump Metropolis-Hastings scheme to fit the model which scales
well to real life metagenomic data set sizes, and used an ANOVA variant of this
model to identify genes characteristic of previously described
(doi:10.1038/ismej.2009.60), seasonally and vertically differentiated
planktonic microbial community types at the Bermuda Atlantic Time Series Site.</p>

<h3>Results</h3>

<p class="abresults">Besides identifying some biological processes potentially
relevant for adaptive differentiation across the communities compared, our
results also highlight some inherent limitations of gene centric community
-omics approaches. In several cases, average copy numbers of different gene
families with similar fuctions showed complementary changes among community
types. The inferred differences in gene copy numbers often do not directly
reflect functional adaptation but rather differences in the taxonomic
composition between communities.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The method introduced represents a next step towards more
fully accounting for biases impacting gene centric comparative metagenomics.
The Bayesian framework we propose is flexible and can be extended to
incorporate further biasing factors. However, our results also highlight that
gene abundance differences among communities do not necessarily reflect
differential environmental adaptation of community functional repertoire as
often implicitly assumed in similar studies.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Bank M Beszteri (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy538 = 'b&#97;nk.b&#101;szt&#101;r&#105;' + '&#64;';
 addy538 = addy538 + '&#97;w&#105;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy538 + suffix + '\'' + attribs + '>' );
 document.write( addy538 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83258 = 'b&#97;nk.b&#101;szt&#101;r&#105;' + '&#64;';
 addy83258 = addy83258 + '&#97;w&#105;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy83258 + suffix + '\'' + attribs + '>' );
 document.write( addy83258 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34222 = 'b&#97;nk.b&#101;szt&#101;r&#105;' + '&#64;';
 addy34222 = addy34222 + '&#97;w&#105;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy34222 + suffix + '\'' + attribs + '>' );
 document.write( addy34222 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Alfred Wegener Institute for Polar and Marine
Research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1: Alfred Wegener Institute for Polar and Marine
Research, Bremerhaven, Germany 2: Department of Microbiology, Oregon State
University, Corvallis, OR, USA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by the Gordon and Betty Moore
Foundation</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742390">B-9. Phylogenetic
mapping of non-model organism RNA-seq reads using a graph algorithm</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vilella A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vilella A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Massingham T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Massingham T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Loytynoja A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Loytynoja A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">RNA-seq analyses of non-model organisms suffer from the
lack of a reference gene set to map the reads against. We propose to tackle
this problem by aligning the reads to multiple alignments of gene phylogenies
from related species using the PAGAN sequence graph aligner. Our simulation
study and analyses of real data indicate better performance than simply using
current de novo assembly approaches. Our PAGAN graph aligner can take graphs
from de novo assemblers and phylogenetically map them to reference gene family
alignments, increasing the bridging between contigs and separating paralogs.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach builds profile-HMM models from
EnsemblCompara GeneTrees codon alignments, scans the RNA-seq reads using HMMER3
and clusters reads as putatively homologous to the highest scoring gene tree
with evalue</p>

<h3>Results</h3>

<p class="abresults">The aligned reads from each species reflect the underlying
paralogous haplotypes in duplicating genetrees and indicates the inferred
missing site info. We performed simulations to study the expected coverage of a
given RNA-seq run for a non-model species with no close sequenced genome Our
method performs well even for species 300 MYA to the closest reference genome.
We show that there is a benefit in using multiple references even when a single
closest reference genome is available, due to the incompleteness of genomic
annotation and the wider scope of gene families in representing evolution.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We applied our approach to reconstruct the brain
transcriptome of several publicly available non-model transcriptomes in the
NCBI SRA against the available gene families in EnsemblCompara. Even for
low-coverage runs (1x-4x of 454 Ti sequencing) we already obtained an average
coverage of 50% transcripts at 60% median of full length. Considering current
NGS technologies, we predict that our phylogenetic approach is be able to
deliver comprehensive transcript models for a vertebrate species in a sampled
subgroup for less than $10k in sequencing costs.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="https://www.ebi.ac.uk/%7Eavilella/talks/20100901_ECCB" mce_href="https://www.ebi.ac.uk/~avilella/talks/20100901_ECCB">https://www.ebi.ac.uk/~avilella/talks/20100901_ECCB</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Albert J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Albert J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->. Vilella Bertran (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63531 = '&#97;v&#105;l&#101;ll&#97;' + '&#64;';
 addy63531 = addy63531 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy63531 + suffix + '\'' + attribs + '>' );
 document.write( addy63531 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92887 = '&#97;v&#105;l&#101;ll&#97;' + '&#64;';
 addy92887 = addy92887 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy92887 + suffix + '\'' + attribs + '>' );
 document.write( addy92887 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22276 = '&#97;v&#105;l&#101;ll&#97;' + '&#64;';
 addy22276 = addy22276 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy22276 + suffix + '\'' + attribs + '>' );
 document.write( addy22276 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">EBI-EMBL</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 EMBL, EBI, WTGC, Hinxton, CB10 1SD, United Kingdom</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742391">B-10. GEI-DB: a database
of atypical genomic elements in bacteria</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bezuidt O<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bezuidt O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Lima-Mendez G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lima-Mendez G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Reva O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reva O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Comparative genome analysis revealed lateral transfer as
one of the dominant factors in the evolution of bacteria. Analysis of these
organisms showed that mobile genomic elements (MGE) appear as atypical genomic
entities that influence the dissemination of genes that contribute to virulence
and adaptation to environmental selection pressures. Developments of tools are
desperately needed to aid the identification of such elements to fully
understand their role and influence in bacterial evolution. This work focuses
on the analysis of pathways of distribution of MGE in bacteria.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">SeqWord genomic islands sniffer (SWGIS) was used in
the study to search for genomic islands (GI) in bacteria. SWGIS detects GI by
the analysis of intragenomic variations and compositonal biases in the
genome-wide distribution of tetranucleotide usage patterns. It evaluates
variances of oligonucleotide frequencies (OUV) and calculates distances (D)
between local pattern deviations from the patterns calculated for the whole
chromosomal DNA.</p>

<h3>Results</h3>

<p class="abresults">The current version of GEI-DB offers an array of 3518
precalculated genomic islands (GI) identified in 637 prokaryotic genomes by
SWGIS. Annotation of genes that are contained within each islands is provided.
The resource also provides the tRNAs that each island is associated with
identified by tRNA-scan tool; the list of homologous genomic regions which were
acquired upon a comparison of all the genomic islands against one another using
BLASTN and classifications of functional related genomic islands in the form of
protein families identified by a Markov clustering algorithm.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The GEI-DB serves as a resource for MGE that could
further be used for more analysis by the scientific community. It has available
in it a collection of MGE from prokaryotes of various backgrounds that are of
medical and environmental importance. It allows browsing of genomic elements
that were classified according to DNA and protein compositional features that
they have in common and also gene entities that are associated with these
elements, which also allow the study of evolutionary profiles that are shared
among MGE.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://anjie.bi.up.ac.za/geidb/geidb-home.php" mce_href="http://anjie.bi.up.ac.za/geidb/geidb-home.php">http://anjie.bi.up.ac.za/geidb/geidb-home.php</a><span style=""> (firefox, konqueror and safari only)</span></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Oliver Bezuidt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4419 = 'b&#101;z&#117;&#105;dt' + '&#64;';
 addy4419 = addy4419 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy4419 + suffix + '\'' + attribs + '>' );
 document.write( addy4419 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8239 = 'b&#101;z&#117;&#105;dt' + '&#64;';
 addy8239 = addy8239 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy8239 + suffix + '\'' + attribs + '>' );
 document.write( addy8239 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54228 = 'b&#101;z&#117;&#105;dt' + '&#64;';
 addy54228 = addy54228 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy54228 + suffix + '\'' + attribs + '>' );
 document.write( addy54228 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of pretoria, Department of biochemistry</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">University of Pretoria, Department of Biochemistry,
Bioinformatics and Computational Biology Unit, Lynnwood Rd, Hillcrest,
Pretoria, South Africa (1*), Laboratoire de Bioinformatique des Genome et des
Reseaux, Universite de Bruxelles, 1050 Bruxelles, Belgium (2)</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The work was funded by the National Bioinformatics
Network of South Africa. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742392">B-11. The CNS twilight
zone: limitations in comparing upstream regions in plants</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Reineke AR<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Reineke AR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Bornberg-Bauer E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bornberg-Bauer E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gu J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gu J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Two main problems challenge the investigation of upstream
regions in plants through comparative genomic analysis. First, the search for
clear orthologous gene pairs between plants is difficult due to many in- and
outparalogs arisen from frequent duplication events. Second, the search for
conserved non-coding sequences (CNS) is difficult because they are less
frequent and much shorter in upstream regions of plants compared to animals.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">With Inparanoid (Remm et al.,2001), orthologous
genes were found in plant pairs with divergence times between 5 and 108 mya.
Upstream regions of genes were extracted in 500 bp segments up to 3 kb upstream
of the transcription start site. With DIALIGN (Morgenstern et al., 1998), a
tool combining global and local alignments, similarities and CNS between
orthologous upstream regions were analyzed. The missing plant divergence times
were estimated using BEAST (Drummond et al., 2007).</p>

<h3>Results</h3>

<p class="abresults">Upstream region similarities of monocots are significantly
lower compared to dicots. In both plant groups, CNS frequency decreases with
increasing plant divergence time. The decay rate of UTR similarity is estimated
using the exponential function. CNSs are found to be mostly embedded in the
first +500 bp upstream, followed by -500 bp downstream. The similarity between
UTR decreases with distance from the transcription start site (TSS). In
monocots, the signal for significant similarity is not detectable for a
distance of &gt;+1kb in monocots and +1.5kb in dicots from the TSS.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our findings identifies the twilight zone, where
significant similarity of upstream regions becomes difficult to distinguish, is
estimated to be around 70 mya divergence time for plants. Furthermore, independent
duplication events reduce this time of useful divergence. In regions with a
distance over +1.5 kb from transcription starts site, CNS detection is
difficult because the weak CNS signal cannot be distinguished from the
background noise. Upstream region comparisons between monocots and dicots have
to be done with caution, due to different CNS properties.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Anna R. Reineke (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18593 = '&#97;.r&#101;&#105;n&#101;k&#101;' + '&#64;';
 addy18593 = addy18593 + '&#117;n&#105;-m&#117;&#101;nst&#101;r' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy18593 + suffix + '\'' + attribs + '>' );
 document.write( addy18593 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98968 = '&#97;.r&#101;&#105;n&#101;k&#101;' + '&#64;';
 addy98968 = addy98968 + '&#117;n&#105;-m&#117;&#101;nst&#101;r' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy98968 + suffix + '\'' + attribs + '>' );
 document.write( addy98968 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58887 = '&#97;.r&#101;&#105;n&#101;k&#101;' + '&#64;';
 addy58887 = addy58887 + '&#117;n&#105;-m&#117;&#101;nst&#101;r' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy58887 + suffix + '\'' + attribs + '>' );
 document.write( addy58887 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute for Evolution and Biodiversity, University
of Münster</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Institute for Evolution and Biodiversity, University of
Münster</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by HFSP R6P0033/2006-C and the
Alexander von Humboldt Foundation. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742393">B-12. TurboOrtho: a high
performance alternative to OrthoMCL</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ekseth O<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ekseth O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lindi B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lindi B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kuiper M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kuiper M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mironov V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mironov V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Establishing genuine orthology relations among species is
a hard problem, therefore in practice some heuristic approaches are used. In
most cases BLAST, TerraBLAST or mpiBLAST are used in the all-against-all mode
to generate a list of gene/protein similarities which serves as the starting
point for elucidating putative orthology relationships. A number of software
packages are available for this purpose, OrthoMCL being one of the most
popular. However, its performance is limited by its implementation based on
Perl. Therefore, we developed a High Performance alternative to OrthoMCL.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">TurboOrtho was implemented in C++. For benchmarking
TurboOrtho we used a set of 144829 protein sequences from four species: H.
sapiens, A. thaliana, S. cerevisiae and S. pombe. The BLAST output file
contained 30841618 hits and was 2.2 G large. The analysis was performed on a
Hewlett-Packard machine with 125 G memory and 2.66 GHz speed.</p>

<h3>Results</h3>

<p class="abresults">The execution time for the benchmarking set was 1080 min for
OrthoMCL v.1, 70 min for OrthoMCL v.2 and below 5 min for TurboOrtho.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have re-implemented the OrthoMCL algorithm (with minor
amendments) in C++ and demonstrate a many-fold increase in the performance. In
particular, TurboOrtho outperforms OrthoMCL v.1 and OrthoMCL v.2 by the factor
of 216 and 14 respectively. TurboOrtho is also easier to use than OrthoMCL v.2
because it combines four steps of the OrthoMCL procedure in a single program
that does not require a relational database.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Vladimir N. Mironov (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50961 = 'm&#105;r&#111;n&#111;v' + '&#64;';
 addy50961 = addy50961 + 'b&#105;&#111;' + '&#46;' + 'ntn&#117;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy50961 + suffix + '\'' + attribs + '>' );
 document.write( addy50961 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96951 = 'm&#105;r&#111;n&#111;v' + '&#64;';
 addy96951 = addy96951 + 'b&#105;&#111;' + '&#46;' + 'ntn&#117;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy96951 + suffix + '\'' + attribs + '>' );
 document.write( addy96951 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92273 = 'm&#105;r&#111;n&#111;v' + '&#64;';
 addy92273 = addy92273 + 'b&#105;&#111;' + '&#46;' + 'ntn&#117;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy92273 + suffix + '\'' + attribs + '>' );
 document.write( addy92273 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Norwegian University for Science and Technology
(NTNU) Trondheim, Norway</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Norwegian University for Science and Technology
(NTNU),Trondheim, Norway</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742394">B-13. i-ADHoRe 3.0
Detection of collinearity in large scale datasets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Proost S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Proost S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,+,*), Fostier J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fostier J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,+,*), Dhoedt B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dhoedt B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Demeester P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Demeester P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Vandepoele K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandepoele K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Though several tools to detect collinearity exist, few
are able to go beyond a pairwise comparison and harvest the information from
multiple genomes to detect additional diverged regions. Current tools
supporting multiple genomes have memory and run-time issues when scaled up to a
dozen or more genomes. As recently several new genomes have been sequenced, and
many more can be expected in the near future, there is a clear need for a novel
tool able to cope with this abundance of data in an efficient and accurate way.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Several algorithmic improvements are featured in
this version, the most important one being a novel, graph-based alignment of
gene lists. Also, support for multi-core CPUs and computer clusters is
implemented using multi-threading and MPI support for parts of the algorithm
that could run in parallel. Statistical improvements have been thoroughly
benchmarked using permutation tests and different datasets derived from PLAZA
(http://bioinformatics.psb.ugent.be/plaza/) and Ensembl release 57
(http://www.ensembl.org/index.html).</p>

<h3>Results</h3>

<p class="abresults">The new version of i-ADHoRe made it possible to analyze an
unprecedented large dataset containing 49 genomes in less then 5 hours (using
32 CPUs). We show the merits of the algorithmic improvements and the technical
adjustments that greatly reduce the runtime and memory requirements, compared
to previous versions and other tools. Furthermore, a few preliminary results
from the analysis of this large dataset are shown, including a large alignment
of several Hox-custers found in a variety of species.</p>

<h3>Discussion</h3>

<p class="abdiscussion">As new sequencing technologies greatly reduce the time
and costs to sequence new genomes, the downstream analysis of this data will
become the bottleneck in the absence of fast and accurate tools to analyze
them. In this study, it was demonstrated that the combination of fast
heuristics, efficient implementation and support for modern hardware can be a
significant improvement over the current state-of-the-art.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Sebastian J.J. Proost (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72171 = 's&#101;b&#97;st&#105;&#97;n.pr&#111;&#111;st' + '&#64;';
 addy72171 = addy72171 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy72171 + suffix + '\'' + attribs + '>' );
 document.write( addy72171 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5795 = 's&#101;b&#97;st&#105;&#97;n.pr&#111;&#111;st' + '&#64;';
 addy5795 = addy5795 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy5795 + suffix + '\'' + attribs + '>' );
 document.write( addy5795 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62921 = 's&#101;b&#97;st&#105;&#97;n.pr&#111;&#111;st' + '&#64;';
 addy62921 = addy62921 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy62921 + suffix + '\'' + attribs + '>' );
 document.write( addy62921 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Plant Systems Biology, Bioinformatics
and Systems Biology Division, VIB</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Plant Systems Biology, Bioinformatics
and Systems Biology Division, VIB, Technologiepark 927, 9052 Ghent, Belgium 2
Department of Plant Biotechnology and Genetics, Ghent University,
Technologiepark 927, 9052 Ghent, Belgium 3 Ghent University - IBBT, Department
of Information Technology (INTEC), Gaston Crommenlaan 8, Bus 201, 9050 Ghent,
Belgium +contributed equally</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">S.P. thanks the Institute for the Promotion of
Innovation by Science and Technology in Flanders for a predoctoral fellowship.
K.V. is a Postdoctoral Fellow of the Research Foundation–Flanders. This project
is funded by the Research Foundation–Flanders and the Belgian Federal Science
Policy Office: IUAP P6/25 (BioMaGNet). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742395">B-14. Comparative
expression analysis of orthologous genes between Arabidopsis and rice</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Movahedi S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Movahedi S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Vandepoele K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandepoele K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Recently microarray experiments have yielded massive
amounts of expression information for many genes under various conditions or in
different tissues for different model species. Expression compendia grouping
multiple microarray experiments performed in similar (or different)
experimental condition make it possible to define correlated expression
patterns between genes or go a bit further and see how tissue-specific or
constitutive expression is linked with expression context conservation and
protein evolution.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this study we use three main data bases
including microarray expression data, GO functional annotations and orthologous
gene families to calculate Expression Coherence (EC) and Expression Context
Conservation (ECC) measures. We also calculate protein evolution rate (Ka) in
order to study correlations between tissue specificity, expression context
conservation and protein evolution.</p>

<h3>Results</h3>

<p class="abresults">Among 4,630 1:1 orthologous Arabidopsis-rice gene pairs, 60%
are ECC conserved, 21% ECC diverged and 19% have non-significant ECC . ECC
conserved genes are evolving slightly slower at the protein level compared to
genes with non-conserved coexpression contexts. ECC conserved genes overall
show a higher expression breadth compared to non-conserved ECC genes. Colinear
genes do not show a different ECC conservation pattern but Core genes are
slightly more conserved. Finally genes with increased rates of protein
evolution show reduced expression breadth.</p>

<h3>Discussion</h3>

<p class="abdiscussion">It is not easy to find a clear correlation pattern
between tissue specificity, expression context conservation and protein
evolution, most of the previous attempts gave conflicting results. In this
study we also found tissue specificity and protein evolution are weakly linked
with the conservation of gene expression. ECC conservation pattern of colinear
genes and core genes is not striking as well but we have strong evidences
showing tissue specific genes are fast-evolving at the protein level.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Sara Movahedi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38610 = 's&#97;r&#97;.m&#111;v&#97;h&#101;d&#105;' + '&#64;';
 addy38610 = addy38610 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38610 + suffix + '\'' + attribs + '>' );
 document.write( addy38610 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19954 = 's&#97;r&#97;.m&#111;v&#97;h&#101;d&#105;' + '&#64;';
 addy19954 = addy19954 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy19954 + suffix + '\'' + attribs + '>' );
 document.write( addy19954 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79525 = 's&#97;r&#97;.m&#111;v&#97;h&#101;d&#105;' + '&#64;';
 addy79525 = addy79525 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy79525 + suffix + '\'' + attribs + '>' );
 document.write( addy79525 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Plant Systems Biology, Bioinformatics
and Systems Biology Division, VIB</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Plant Systems Biology, VIB,
Technologiepark 927, B-9052 Ghent, Belgium. (2) Department of Plant
Biotechnology and Genetics, Ghent University, Technologiepark 927, B-9052
Ghent, Belgium. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742396">B-15. Phylogenomics and
robust construction of prokaryotic evolutionary trees</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Korenblat K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Korenblat K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Volkovich Z<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Volkovich Z&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bolshoy A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bolshoy A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,*)</p>

<p class="abmotivation">Usually, the phylogeny produced by means of molecular
evolution is based on the comparison of the highly conserved small subunit rRNA
sequences (ssu-rRNA) as originally proposed by Woese and colleagues. We wanted
to demonstrate the power of the phylogenomics' approach. For this purpose we
constructed an evolutionary tree of a few hundreds of prokaryotes. We wanted to
show that our method of construction of prokaryotic evolutionary trees produces
very robust results. We wanted to show that more genes are considered for tree
construction better trees are obtained.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Data consist of median protein lengths related to
several thousands of COGs from more than 600 genomes. To demonstrate robustness
we selected 60 genomes. To evaluate robustness of trees we used two approaches.
The first way was to compare distances between trees constructed on real data
with those obtained for randomized data. The second method was to compare
distances among trees based on different subsets of all COGs set. Partition
distance between trees was chosen to measure distances between obtained
evolutionary trees.</p>

<h3>Results</h3>

<p class="abresults">1) An average distance between trees obtained on different
COG-subsets (randomly selected 300 COGs out of 900 COGs) is significantly
smaller than an average distance between trees based on partially randomized
data. 2) An average distance between trees obtained on different COG-subsets is
negatively correlated with a number of randomly selected N COGs out of 900
COGs. 3) Robustness of the method allows obtaining reliable consensus phylogenomic
trees of hundreds of prokaryotes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our results make a positive contribution to resolve an
ongoing discussion whether phylogenomics can successfully assist in
construction of a "species tree" as opposite to a "gene
tree".</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Alexander Bolshoy (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72483 = 'b&#111;lsh&#111;y' + '&#64;';
 addy72483 = addy72483 + 'r&#101;s&#101;&#97;rch' + '&#46;' + 'h&#97;&#105;f&#97;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy72483 + suffix + '\'' + attribs + '>' );
 document.write( addy72483 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy20832 = 'b&#111;lsh&#111;y' + '&#64;';
 addy20832 = addy20832 + 'r&#101;s&#101;&#97;rch' + '&#46;' + 'h&#97;&#105;f&#97;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy20832 + suffix + '\'' + attribs + '>' );
 document.write( addy20832 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40591 = 'b&#111;lsh&#111;y' + '&#64;';
 addy40591 = addy40591 + 'r&#101;s&#101;&#97;rch' + '&#46;' + 'h&#97;&#105;f&#97;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy40591 + suffix + '\'' + attribs + '>' );
 document.write( addy40591 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Haifa</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Software Engineering Department, ORT Braude Academic
College, Karmiel, Israel 2 Department of Evolutionary and Environmental
Biology, University of Haifa, Haifa, Israel 3 Genome Diversity Center of the
Institute of Evolution, University of Haifa, Haifa, Israel</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742397">B-16. Mapping the human
membrane proteome: a majority of the human membrane proteins can be classified
according to function and evolutionary origin</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Almén MS<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Almén MS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Nordström KJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nordström KJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Fredriksson R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fredriksson R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Schiöth HB<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schiöth HB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The transmembrane proteins form a link between the cell
and the surroundings, allowing it to interact and respond to environmental
changes and communicate with neighboring cells. Large research efforts have
concerned these proteins over the years as they include the majority of our
drug targets. Although this numerous group of proteins has received a lot of
attention it is largely unexplored and a comprehensive overview is missing.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We mined the human proteome and identified the
membrane proteome subset using three prediction tools for alpha-helices:
Phobius, TMHMM, and SOSUI. This dataset was reduced to a non-redundant set by
aligning it to the human genome and then clustered with our own implementation
of the ISODATA algorithm. The genes were classified and each protein group was
manually curated, virtually evaluating each sequence of the clusters, applying
systematic comparisons with a range of databases and other resources.</p>

<h3>Results</h3>

<p class="abresults">We identified 6,718 human membrane proteins and classified
the majority of them into 234 families of which 151 belong to the three major
functional groups: receptors (63 groups, 1,352 members), transporters (89
groups, 817 members) or enzymes (7 groups, 533 members). Also, 74 miscellaneous
groups with 697 members were determined. Interestingly, we find that 41% of the
membrane proteins are singlets with no apparent affiliation or identity to any
human protein family. Further, we have extended our study with a thorough
evolutionary investigation of the understudied human 4TM proteins.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In conclusion, we estimate that 27% of the total human
proteome are alpha-helical transmembrane proteins and have created a complete
overview for all major families of transmembrane proteins and functionally
classified them. This roadmap will aid future identification and classification
of novel transmembrane proteins by providing a functional context of the known
human transmembrane repertoire.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Markus Sällman Almén (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5092 = 'm&#97;rk&#117;s.s&#97;llm&#97;n-&#97;lm&#101;n' + '&#64;';
 addy5092 = addy5092 + 'n&#101;&#117;r&#111;' + '&#46;' + '&#117;&#117;' + '&#46;' + 's&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy5092 + suffix + '\'' + attribs + '>' );
 document.write( addy5092 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7205 = 'm&#97;rk&#117;s.s&#97;llm&#97;n-&#97;lm&#101;n' + '&#64;';
 addy7205 = addy7205 + 'n&#101;&#117;r&#111;' + '&#46;' + '&#117;&#117;' + '&#46;' + 's&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy7205 + suffix + '\'' + attribs + '>' );
 document.write( addy7205 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22159 = 'm&#97;rk&#117;s.s&#97;llm&#97;n-&#97;lm&#101;n' + '&#64;';
 addy22159 = addy22159 + 'n&#101;&#117;r&#111;' + '&#46;' + '&#117;&#117;' + '&#46;' + 's&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy22159 + suffix + '\'' + attribs + '>' );
 document.write( addy22159 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dept. of Neuroscience, Uppsala Universitet</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Neuroscience, Functional Pharmacology,
Uppsala University, Uppsala, Sweden</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The studies were supported by the Swedish Research Council,
The Novo Nordisk Foundation, Swedish Royal Academy of Sciences, and Magnus
Bergvall Foundation. RF was supported by the Göran Gustafssons foundation. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742398">B-17. Towards molecular
trait-based ecology through integration of biogeochemical, geographical and
metagenomic data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Raes J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Raes J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Letunic I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Letunic I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Yamada T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yamada T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Jensen LJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jensen LJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Bork P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bork P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,4)</p>

<p class="abmotivation">Meta-omics (metagenomics, metatranscriptomics, metaproteomics)
are powerful tools for the analysis of complete microbial communities in both
environmental (e.g. ocean, soil) and medical (e.g. the human microbiome)
context. Because of its complexity, meta-omics data has required the
development of novel computational approaches to move from 'parts lists' to
ecosystem functioning and and understanding of the ecology of these
communities.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To deduce important ecological indicators such as
environmental adaptation, molecular trait dispersal, diversity variation and
primary production from the gene pool of an ecosystem, we integrated 25 ocean
metagenomes with geographical, meteorological and geo-physicochemical data.</p>

<h3>Results</h3>

<p class="abresults">We find that climate (temperature, sunlight) is the major
determinant of the biomolecular repertoire of each sample and the main limiting
factor on functional trait dispersal (absence of biogeographic provincialism).
Molecular functional richness and diversity show a distinct latitudinal
gradient peaking at 20°N and correlate with primary production. The latter can
also be predicted from the molecular functional composition of an environmental
sample.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Together, our results show that the functional community
composition derived from metagenomes can be used as quantitative predictor for
molecular trait-based biogeography and ecology. We see this study as
proof-of-principle that molecular functional composition can be used in various
other environmental settings such as the human microbiome where it could be integrated
with clinical data to study the molecular ecology and temporo-spatial variation
of the ‘human’ ecosystem.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jeroen Raes (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37240 = 'j&#101;r&#111;&#101;n.r&#97;&#101;s' + '&#64;';
 addy37240 = addy37240 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy37240 + suffix + '\'' + attribs + '>' );
 document.write( addy37240 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81175 = 'j&#101;r&#111;&#101;n.r&#97;&#101;s' + '&#64;';
 addy81175 = addy81175 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy81175 + suffix + '\'' + attribs + '>' );
 document.write( addy81175 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16598 = 'j&#101;r&#111;&#101;n.r&#97;&#101;s' + '&#64;';
 addy16598 = addy16598 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy16598 + suffix + '\'' + attribs + '>' );
 document.write( addy16598 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB - Vrije Universiteit Brussel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Structural and Computational Biology Unit, European
Molecular Biology Laboratory, 69117 Heidelberg, Germany 2 Molecular and
Cellular Interactions Department, VIB – Vrije Universiteit Brussel, 1040
Brussels, Belgium 3 NNF Center for Protein Research, 2200 Copenhagen, Denmark 4
Max Delbrück Center for Molecular Medicine, 13125 Berlin-Buch, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is supported by the European Union FP7
Program Contract no. HEALTH-F4–2007-201052. JR is supported by the FWO Oddyseus
programme. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742399">B-18. Orthologous
proteins associated with yeast telomeric complex identified by synteny and
sequence similarity</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Macko M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Macko M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Tomáška L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tomáška L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vinař T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vinař T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Proteins of telomeric complex protect integrity of
chromosome and thus they have an important function during the replication
process. These proteins have been previously identified in yeast Saccharomyces
cerevisiae, but not in other yeast species e.g. Candida parapsilosis. Search
for orthologous proteins in other species is complicated by faster evolution of
these proteins and species specific sequences on chromosome ends where these
proteins bind. Identification of proper orthologs in several yeast species will
enable further analysis of evolution of this complex.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We searched for the putative orthologs by a
combination of sequence comparison and synteny. The first batch of orthologs
have been identified by the reciprocal best BLAST hits between proteins of
Saccharomyces cerevisiae and several other yeast species. Next, we used
orthologs located by our search and orthologs already identified in KEGG
database, to create pHMM and motifs by software HMMER and MEME. These models
and motifs were used to identify additional orthologs. Confidence for new
orthologs is evaluated by the naive Bayes classifier with use of the sequence
and synteny information.</p>

<h3>Results</h3>

<p class="abresults">We have searched for orthologous proteins of 121
Saccharomyces cerevisiae proteins associated with telomeres in genomes of 6
related yeast species. Higher sensitivity of search was accomplished by
combining information from known orthologs and creating profile hidden markov
models for each group of orthologous proteins. For example, significant matches
for EST3 protein were not found by BLAST search but with motifs and pHMM
searches. On the other hand protein CDC13 had no significant matches even
though most proteins that interact with it according to database UniPROT were
found.</p>

<h3>Discussion</h3>

<p class="abdiscussion">After obtaining putative orthologs for most proteins of
the telomeric complex, protein interaction networks could be reconstructed, by
assuming conserved protein interactions between orthologs of proteins which
have interaction identified in UNIProt database. This could help locate
“missing” nodes in these putative networks and point out cases where further
analysis is needed.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Martin Macko (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9523 = 'm&#97;rt&#105;n.m&#97;ck&#111;1' + '&#64;';
 addy9523 = addy9523 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy9523 + suffix + '\'' + attribs + '>' );
 document.write( addy9523 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81712 = 'm&#97;rt&#105;n.m&#97;ck&#111;1' + '&#64;';
 addy81712 = addy81712 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy81712 + suffix + '\'' + attribs + '>' );
 document.write( addy81712 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99856 = 'm&#97;rt&#105;n.m&#97;ck&#111;1' + '&#64;';
 addy99856 = addy99856 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy99856 + suffix + '\'' + attribs + '>' );
 document.write( addy99856 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">doctoral student, Department of Applied Informatics,
Faculty of Mathematics, Physics, and Informatics Comenius University in
Bratislava</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)doctoral student,Department of Applied Informatics,
Faculty of Mathematics, Physics, and Informatics Comenius University (2) head
of the Department of Genetics, Faculty of Natural Sciences, Comenius University
(3) Assistant Professor,Department of Applied Informatics, Faculty of
Mathematics, Physics, and Informatics Comenius University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work has been supported by a grant VEGA 1/0210/10,
Marie Curie Fellowship to Tomas Vinar IRG-224885, and Comenius University young
researcher grant G-10-229-00</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742400">B-19. Estimating average
heterozygosity using multilocus dominant marker data: a maximum likelihood
approach via the EM algorithm.</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Khang TF<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Khang TF&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Yap VB<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yap VB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Many preliminary studies of natural plant and animal
populations use molecular methods such as amplified fragment length
polymorphism (AFLP) and random amplified polymorphic DNA (RAPD) to assess
average heterozygosity in a population. For small sample sizes, ascertainment
bias can lead to inaccurate estimates when the distribution of null homozygote
proportions for dominant loci is J-shaped. We show how an EM algorithm can be used
effectively to correct for ascertainment bias when estimating average
heterozygosity under Hardy-Weinberg equilibrium assumption for all dominant
loci.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">From the full likelihood function, which is a
product of beta-binomial functions, we derived the Expectation and Maximization
steps of the EM algorithm. We used simulation to check the error distribution
of maximum likelihood estimates of average heterozygosity via the EM algorithm.
We did this for several different beta distribution profiles for the null
homozygote proportions. Subsequently, We compared its performance with two
other maximum likelihood approaches: one that does not correct for
ascertainment bias, and another that corrects for it using a truncated
beta-binomial likelihood.</p>

<h3>Results</h3>

<p class="abresults">Our results show that ML estimates via the EM algorithm as
well as the truncated beta-binomial likelihood lead to subtantial improvement
in accuracy and root mean square error for J-shaped beta profiles. For U-shaped
profiles, the improvement is nontrivial; and for inverse-J shaped profiles,
there is negligible improvement as expected.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Many published results do not incorporate correction for
ascertain bias when estimating average heterozygosity. As we have shown, this
can lead to estimates that are biased upwards, with magnitude dependent on the
beta profile of the null homozygote proportions. This finding is of concern
because it may mislead researchers into thinking that a particular population
is genetically diverse. Therefore, to properly interpret the results of
published work that do not correct for ascertanment bias, we propose to do a
simple graphical check of the beta profiles.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Tsung Fei Khang (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43745 = 'tfkh&#97;ng' + '&#64;';
 addy43745 = addy43745 + '&#117;m' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'my';
 document.write( '<a ' + path + '\'' + prefix + addy43745 + suffix + '\'' + attribs + '>' );
 document.write( addy43745 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45243 = 'tfkh&#97;ng' + '&#64;';
 addy45243 = addy45243 + '&#117;m' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'my';
 document.write( '<a ' + path + '\'' + prefix + addy45243 + suffix + '\'' + attribs + '>' );
 document.write( addy45243 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92743 = 'tfkh&#97;ng' + '&#64;';
 addy92743 = addy92743 + '&#117;m' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'my';
 document.write( '<a ' + path + '\'' + prefix + addy92743 + suffix + '\'' + attribs + '>' );
 document.write( addy92743 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Biological Sciences, University of
Malaya</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Institute of Biological Sciences, Faculty of
Science, University of Malaya, 50603 Kuala Lumpur, Malaysia. 2. Department of
Statistics &amp; Applied Probability, Faculty of Science, National University
of Singapore, Singapore 117546.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The National University of Singapore supported the
first author with a research scholarship while the work was carried out as part
of his PhD thesis. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742401">B-20. PLAZA 2.0 : a
resource for plant comparative genomics</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Van Bel M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Van Bel M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Proost S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Proost S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Wischnitzki E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wischnitzki E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Vandepoele K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandepoele K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Comparative sequence analysis has significantly altered
our view on the complexity of genome organization and gene function. To explore
all this genome information, a centralized infrastructure is required where all
data generated by different sequencing initiatives is integrated and combined
with advanced methods for data mining. Here we describe PLAZA2.0, an update to
the PLAZA platform. This resource integrates structural and functional
annotation of published plant genomes together with a large set of interactive
tools to study gene function and gene and genome evolution.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The update PLAZA version contains the nuclear and
organelle genomes of twenty three species within the Viridiplantae kingdom:
eleven eudicots, five monocots, two (club-)mosses and five green algae. The
integration of all gene annotations provided by the different sequencing
centers yielded a data set of 843,854 gene models, of which 88.3% represent
protein-coding genes. The remaining genes are classified as transposable
elements, RNA, and pseudogenes (11.3%, 0.1%, and 0.3%, respectively).</p>

<h3>Results</h3>

<p class="abresults">Gene families were automatically generated using MCL
clustering, resulting in far fewer single species gene families compared with
PLAZA 1.0, thus providing evidence that the growing number of species leads to
a more balanced view of species-specific gene families. Automated whole-genome
statistical comparisons give further insights in variations in codon usage
between species, and the new and improved tools allow users to gain knowledge
faster and more efficiently.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We provide an update to the PLAZA platform, where users
can easily browse genes, gene families and the associated functional
annotation.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinformatics.psb.ugent.be/plaza" mce_href="http://bioinformatics.psb.ugent.be/plaza">http://bioinformatics.psb.ugent.be/plaza</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Michiel Van Bel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66021 = 'M&#105;ch&#105;&#101;l.v&#97;nb&#101;l' + '&#64;';
 addy66021 = addy66021 + 'psb' + '&#46;' + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy66021 + suffix + '\'' + attribs + '>' );
 document.write( addy66021 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49661 = 'M&#105;ch&#105;&#101;l.v&#97;nb&#101;l' + '&#64;';
 addy49661 = addy49661 + 'psb' + '&#46;' + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy49661 + suffix + '\'' + attribs + '>' );
 document.write( addy49661 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy981 = 'M&#105;ch&#105;&#101;l.v&#97;nb&#101;l' + '&#64;';
 addy981 = addy981 + 'psb' + '&#46;' + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy981 + suffix + '\'' + attribs + '>' );
 document.write( addy981 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universiteit Gent</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Plant Systems Biology, VIB, B-9052
Ghent, Belgium. (2) Department of Plant Systems Biology, Ghent University,
B-9052 Ghent, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by European Union EUFP6 Food
Safety and Quality Contract FOOD-CT-2006-016214. This project is funded by the
Research Foundation?Flanders and the Belgian Federal Science Policy Office:
IUAP P6/25 (BioMaGNet). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742402">B-21. Branch testing in
phylogenetic trees: comparison of computationally efficient methods</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Czarna A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Czarna A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Wróbel B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wróbel B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">It has become standard procedure to assess uncertainty in
phylogenetic reconstruction by calculating bootstrap support or Bayesian
posterior probability for clades. However, when the tree is reconstructed using
maximum likelihood, the computational cost of calculating bootstrap support may
be prohibitive, especially for large datasets. The same goes for Bayesian
analysis. Other approaches to branch testing were proposed, but they have not
been compared thus far in a systematic manner.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used nucleotide sequences of yeast species for
which correct topology is known. The alignment was separated into 1000-nt
subsets. We have first analyzed how the bootstrap support values are affected
by the method used for the reconstruction (ML, NJ/BIONJ, PhyML). Then, the
branches were tested using the quartet puzzling (QP) method, the Dopazo test,
aLRT and WLS-LRT. Finally, we have investigated how the results are affected by
simplifying the substitution model (using JC+Γ+I model instead of GTR+Γ+I) or
by not addressing the rate heterogeneity among sites (GTR instead of GTR+Γ+I).</p>

<h3>Results</h3>

<p class="abresults">The PhyML bootstrap support values were almost the same as
the values obtained when a computationally expensive ML method was used for the
reconstruction of pseudotrees. Using PhyML, NJ or BIONJ gave results similar to
ML in terms of false positives. On the other hand, the Bayesian approach, aLRT
and Dopazo test resulted in less false negatives and more false positives. The
QP method and WLS-LRT gave intermediate results. Using GTR instead of JC+Γ+I
resulted a higher support for true branches. Both simplified models caused an
increase of false positives.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Calculating bootstrap support or Bayesian posterior
probability for clades may not be possible for large data sets. Our results
show that more computationally efficient methods can be used to find support
for branches, although some result in more false positives, especially when the
model is too simple. Using PhyML for pseudotrees reconstruction was the most
robust to model misspecification; the QP method the least robust. However,
bootstrapping trees with PhyML took noticeably more time than using WLS-LRT,
aLRT or Dopazo test.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.evosys.org" mce_href="http://www.evosys.org">http://www.evosys.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Aleksandra Czarna (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy20248 = '&#97;cz&#97;rn&#97;' + '&#64;';
 addy20248 = addy20248 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy20248 + suffix + '\'' + attribs + '>' );
 document.write( addy20248 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy68253 = '&#97;cz&#97;rn&#97;' + '&#64;';
 addy68253 = addy68253 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy68253 + suffix + '\'' + attribs + '>' );
 document.write( addy68253 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99948 = '&#97;cz&#97;rn&#97;' + '&#64;';
 addy99948 = addy99948 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy99948 + suffix + '\'' + attribs + '>' );
 document.write( addy99948 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Computational Biology Group, Institute of Oceanology,
Polish Academy of Sciences</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 – Computational Biology Group, Institute of
Oceanology, Polish Academy of Sciences, Sopot, Poland 2 – Evolutionary Systems
Group, Laboratory of Bioinformatics, Adam Mickiewicz University in Poznań,
Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Polish Ministry of Science and Education (project N303
291234). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742403">B-22. Parsimonious
higher-order hidden Markov models for enhanced comparative genomics</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Seifert M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Seifert M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Banaei A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Banaei A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gohr A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gohr A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Strickert M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Strickert M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Grosse I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Grosse I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Arabidopsis thaliana is an important model organism in
plant biology with a broad geographic distribution. Precise knowledge of
genomic differences between ecotypes is a fundamental component for a better
understanding of their phenotypic variation. For measuring genomic differences,
the method of Array-Comparative Genomic Hybridization (Array-CGH) is applied
routinely nowadays. The analysis of the resulting huge data sets requires
efficient bioinformatics tools. Long-range dependencies between measurements on
the chromosomes stimulate an extension of standard Hidden Markov Models.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Array-CGH has been applied to compare the genomes
of the two important ecotypes Col and C24 of the model plant Arabidopsis
thaliana. Sequence polymorphisms have been identified by a newly developed
parsimonious higher-order Hidden Markov Model (PHHMM) modeling dependencies
between more than two directly neighboring probes on a chromosome. The PHHMM is
compared to a standard HMM and to other methods for Array-CGH data analysis.
This is done based on sequence polymorphisms identified in publicly available
re-sequencing experiments of C24.</p>

<h3>Results</h3>

<p class="abresults">The identification of sequence polymorphisms by the PHHMM is
noticeably higher than for the standard HMM and other methods for Array-CGH
data analysis. Sequence polymorphisms have been identified widespread over all
chromosomes in good accordance with polymorphisms known from re-sequencing
experiments. A significantly large proportion of the identified sequence
polymorphisms in C24 is covered by transposable elements in the reference
genome of Col. Genes and their functional components are significantly less
affected.</p>

<h3>Discussion</h3>

<p class="abdiscussion">By modeling higher-order dependencies in a data-dependent
manner, the newly developed PHHMM overcomes the limitation of the commonly used
HMM that only models dependencies between two adjacent probes on a chromosome.
This leads to an improved identification of sequence polymorphisms also in
comparison to other methods. The biological relevance of the identified
sequence polymorphisms is emphasized by the over-representation of transposable
elements that are known to be involved in driving the variation among ecotypes.
The PHHMM turns out to be a useful tool for the analysis of ACGH data sets.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Michael Seifert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79134 = 's&#101;&#105;f&#101;rt' + '&#64;';
 addy79134 = addy79134 + '&#105;pk-g&#97;t&#101;rsl&#101;b&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy79134 + suffix + '\'' + attribs + '>' );
 document.write( addy79134 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19214 = 's&#101;&#105;f&#101;rt' + '&#64;';
 addy19214 = addy19214 + '&#105;pk-g&#97;t&#101;rsl&#101;b&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy19214 + suffix + '\'' + attribs + '>' );
 document.write( addy19214 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96899 = 's&#101;&#105;f&#101;rt' + '&#64;';
 addy96899 = addy96899 + '&#105;pk-g&#97;t&#101;rsl&#101;b&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy96899 + suffix + '\'' + attribs + '>' );
 document.write( addy96899 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">IPK Gatersleben</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Leibniz Institute of Plant Genetics and Crop Plant Research,
Gatersleben, Germany (2) Leibniz Institute of Plant Biochemistry, Halle
(Saale), Germany (3) Martin Luther University Halle, Halle (Saale), Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Development of the PHHMM is funded by the Ministry of
Culture Saxony-Anhalt (XP3624HP/0606T), and the Array-CGH experiments were
funded by the DFG (HO1779/7-2). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742404">B-23. Correction of
bootstrap confidence levels using an iterated bootstrap procedure with
computational efficient methods</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Czarna A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Czarna A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Wróbel B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wróbel B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*)</p>

<p class="abmotivation">The uncertainty in phylogenetic analysis is affected by
many factors which cause that the bootstrap support values are often
underestimated or overestimated. Using the method proposed by Rodrigo (1993)
allows for correction of bootstrap confidence levels. The method requires
bootstrapping at three levels. Howver, even with only 20 pseudosamples at each
level (8000 in total), the computational cost is prohibitive when maximum
likelihod is used for the reconstruction of pseudotrees. Here we investigate if
the NJ or PhyML methods can be used to approximate the correct bootstrap
confidence levels.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The iterated bootstrap procedure was used for
nucleotide sequences of yeast species for which correct topology was known. We
separated the alignment of over 100000 nt into 100 datasets, 20 replicates were
obtained for each. Pseudotrees were reconstructed using NJ and PhyML. 20
2nd-level bootstrap replicates were obtained for each 1st-level replicate and
again 20 3rd-level bootstrap replicates for each 2nd-level replicate. Finally,
8000 3rd-level pseudotrees were reconstructed to allow for the correction of
bootstrap confidence levels.</p>

<h3>Results</h3>

<p class="abresults">We analyzed 5 true and 3 wrong branches for one hundred
1000-nt datasets. True branches got support in 47-99% of datasets when PhyML
was used for pseudotree reconstruction (depending on the branch). When NJ was
used, they got support in 22-92% of datasets. Wrong branches got support in
9-21% (PhyML) and 7-17% (NJ). The results were similar to using 70% threshold
in the standard bootstrap procedure: in 32-100% of datasets (PhyML) and 17-85%
(NJ) the bootstrap proportions (BP) were above 70% for true branches. In 2-13%
(PhyML) and 3-12% (NJ) datasets wrong branches had BP above 70%.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The use of iterated bootstrap resulted in a much lower
number of false negatives than the standard bootstrap method. However, the
correction of the bootstrap support values also caused undesirable increase of
support for wrong branches. When PhyML was used for the reconstruction of
pseudotrees, all the branches received higher support than when NJ was applied.
However, the computations using PhyML took 750 times longer. Our results
support the notion that iterated bootstrap using computationally efficient methods
can be applied as an approximate method to correct bootstrap confidence levels</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.evosys.org" mce_href="http://www.evosys.org">http://www.evosys.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Borys Wróbel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71407 = 'bwr&#111;b&#101;l' + '&#64;';
 addy71407 = addy71407 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy71407 + suffix + '\'' + attribs + '>' );
 document.write( addy71407 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91385 = 'bwr&#111;b&#101;l' + '&#64;';
 addy91385 = addy91385 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy91385 + suffix + '\'' + attribs + '>' );
 document.write( addy91385 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2693 = 'bwr&#111;b&#101;l' + '&#64;';
 addy2693 = addy2693 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy2693 + suffix + '\'' + attribs + '>' );
 document.write( addy2693 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Computational Biology Group, Institute of Oceanology,
Polish Academy of Sciences</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 – Computational Biology Group, Institute of
Oceanology, Polish Academy of Sciences, Sopot, Poland 2 – Evolutionary Systems
Group, Laboratory of Bioinformatics, Adam Mickiewicz University in Poznań,
Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Polish Ministry of Science and Education (project N303
291234). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742405">B-24. Molecular
evolution, structure and functional divergence of Lipoxygenase gene family in
vertebrate</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Padmanabhan R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Padmanabhan R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*), Kuhn H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kuhn H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Reddanna P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reddanna P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Lipoxygenase are a group of non-heme, iron containing
enzymes involved in oxygenation of PUFA. The products of LOX reactions are
hydroperoxides (eicosanoids) which are powerful mediators of inflammation and
other medically important states, like asthma, atherosclerosis, cancer
etc.Although discoveries about LOX family in mammals and other species have
been reported in some studies, the evolution and functional and structural
divergence of LOX in mammals are not clearly understood.Hence to study the
phylogenetic relationships, structural and functional divergence of
lipoxygenas.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Sequence data sets of the LOX protein family were obtained
by Blast-Explorer and genomic databases at Ensembl.Amino acid sequences were
aligned with muscle and refined in Bioedit and Gblock. Phylogentic trees were
constructed with ML, NJ programs implemented in the PHYLIP program package and
PhyML.GeneTree was used to resolve the incongruence between the gene and
species trees by predicting duplication and losses.Estimation of substitution
rates and testing positive selection was done using PAML.Site-specific changes
in evolution rate after gene duplication (type I) was estimated using DIVERGE</p>

<h3>Results</h3>

<p class="abresults">Sequence similarity searches using BLAST-EXPLORER was used
to identified the homologus sequences of LOX in all the kingdoms. A total of
152 sequeces from 46 species representing four major lineages (fishes,amphibeia,
sauria, mammalia) were included in the study.Phylogenetic analysis was used to
characterize the family evolutionary history by identifying two major
duplications early in vertebrate evolution., to further analyse this GeneTree
software was used.Estimates of synonymous and nonsynonymous substitution rates
suggest that the duplication events were associated with positive selection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We present a phylogeny describing the evolutionary
history of the Lipoygenase gene family and shows that the genes have evolved
through duplications in animalia followed by positive selection., even though
many sites are higly conserved in the gene familiy and preserve an over all
structural feature of these enzymes. The critical amino acid residues likely
relevant for the distinct functional properties of the prologues has been
identified. Intron-Exon structure evolution of these genes showed well
conservation of exons and intron expansion in mammals.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Roshan Padmanabhan (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34327 = 'r&#111;s&#97;&#97;k' + '&#64;';
 addy34327 = addy34327 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy34327 + suffix + '\'' + attribs + '>' );
 document.write( addy34327 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29994 = 'r&#111;s&#97;&#97;k' + '&#64;';
 addy29994 = addy29994 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy29994 + suffix + '\'' + attribs + '>' );
 document.write( addy29994 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22646 = 'r&#111;s&#97;&#97;k' + '&#64;';
 addy22646 = addy22646 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy22646 + suffix + '\'' + attribs + '>' );
 document.write( addy22646 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The School of Life Sciences, University of Hyderabad,
Hyderabad-500046, Andhra Pradesh, India</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) The School of Life Sciences, University of
Hyderabad, Hyderabad-500046, Andhra Pradesh, India (2)Institute of
Biochemistry, University Medicine Berlin-Charité, Monbijoustrasse 2, D-10117
Berlin, Germany</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742406">B-25. Cassis: detection
of genomic rearrangement breakpoints</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Baudet C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Baudet C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,*), Lemaitre C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lemaitre C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Dias Z<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dias Z&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Gautier C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gautier C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tannier E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tannier E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sagot M-F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sagot M-F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Genomic regions which have undergone a rearrangement are
called breakpoints. Current methods for breakpoint identification are in fact
strategies for detecting synteny blocks or conserved regions (that are
orthologous regions which have not been rearranged): breakpoints are obtained
only as a byproduct, simply by returning regions that are not found in a
conserved synteny. Cassis aims to go one step further and to extend the synteny
blocks by focusing on the breakpoints themselves in order to improve the
precision of the breakpoints.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Cassis receives as input a list of one-2-one
orthologous genes. It processes the given list to determine the breakpoint
regions. For each breakpoint, its sequence, called SR (in the reference
genome), is aligned against its orthologous sequences SoA and SoB (in the
second genome). A sequence of discrete values is defined along the sequence SR:
+1 (-1) for the position that has a hit only with the sequence SoA (SoB) and 0
for the position that has hits with both sequences. A segmentation algorithm
analyses this sequence to narrow the breakpoint, and its result is then
statistically assessed.</p>

<h3>Results</h3>

<p class="abresults">We compared the genomes of human and mouse with Cassis.
First, we used the orthologous genes between the two species available in
Ensembl and we identified 369 breakpoints. A total of 340 breakpoints were
narrowed with an average length reduction of 62%. The median size of the
refined breakpoints is 54.9 kbp. Secondly, we ran Cassis with lists of synteny
blocks obtained by other methods (Compara and Mauve). We show that Cassis is
able to refine most of the breakpoints identified by the other methods.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Cassis is a method that allows the identification and
refinement of breakpoints by a comparison between two genomes. Our results
showed that it is able to narrow a large number of breakpoints, and that it can
work also with the synteny blocks identified by other methods. Cassis thus
enables to better characterise breakpoint sequences and their distribution
along the genomes, which could lead to a better understanding of the mechanisms
and evolutionary properties of chromosomal rearrangements.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://pbil.univ-lyon1.fr/software/Cassis/" mce_href="http://pbil.univ-lyon1.fr/software/Cassis/">http://pbil.univ-lyon1.fr/software/Cassis/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Christian Baudet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13851 = 'chr&#105;st&#105;&#97;n.b&#97;&#117;d&#101;t' + '&#64;';
 addy13851 = addy13851 + '&#117;n&#105;v-ly&#111;n1' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy13851 + suffix + '\'' + attribs + '>' );
 document.write( addy13851 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2476 = 'chr&#105;st&#105;&#97;n.b&#97;&#117;d&#101;t' + '&#64;';
 addy2476 = addy2476 + '&#117;n&#105;v-ly&#111;n1' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy2476 + suffix + '\'' + attribs + '>' );
 document.write( addy2476 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43478 = 'chr&#105;st&#105;&#97;n.b&#97;&#117;d&#101;t' + '&#64;';
 addy43478 = addy43478 + '&#117;n&#105;v-ly&#111;n1' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy43478 + suffix + '\'' + attribs + '>' );
 document.write( addy43478 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Equipe BAMBOO, INRIA Grenoble Rhône-Alpes et
Laboratoire de Biométrie et Biologie (UMR 5558) CNRS, Université Lyon 1</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Equipe BAMBOO, INRIA Grenoble Rhône-Alpes et
Laboratoire de Biométrie et Biologie Évolutive (UMR 5558) CNRS, Université Lyon
1, F-69100 Villeurbanne, France (2) Université de Bordeaux, Centre de
Bioinformatique - Génomique Fonctionnelle Bordeaux, F-33000 Bordeaux, France 3
(3) Institute of Computing, University of Campinas (Unicamp), Av. Albert
Einstein, 1251 - Cidade Universitária, Caixa Postal 6176 - CEP 13083-970,
Campinas - São Paulo, Brazil</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by Coordenação de
Aperfeiçoamento de Pessoal de Nível Superior [4676/08-4] and partially
supported by Conselho Nacional de Desenvolvimento Científico e Tecnológico
[472504/2007-0, 479207/2007-0 and 483177/2009-1]. This work was funded by the
ANR [project BLAN08-1335497], ANR-BBSRC [project ANR-07-BSYS 003 02] and the
ERC AdG (project Sisyphe). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742407">B-26. Methylated
cytosines are less likely to mutate within CpG islands</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Medvedeva Y<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Medvedeva Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Panchin A (2,3) Mitrofanov S (2),
Makeev V<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Makeev V&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1)</p>

<p class="abmotivation">CpG islands (CGIs) in mammals are regions with about 7-10
times higher CpG dinucleotide frequency comparing to the rest of the genome.
The main cause of this effect is though to be the decreased level of cytosine
methylation within the CG context inside CGIs. Methyl-cytosines followed by
guanines (5mCpGs) are about 10 times more likely to mutate into TpGs than
nonmethylated CpGs. However, recent studies show that 5mCpGs are not very rare
in CGIs. We decided to compare the frequency of 5mCpG to TpG mutations inside
CGIs and across the remaining parts of the human genome.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">From a dataset of human SNPs with cytosine/thymine
variants we selected those containing cytosine in orthologous positions of
chimp and orangutan genomes. These SNPs represent C to T mutations in human
population. We took into consideration only SNPs with double strand cytosine
methylation in the human embryonic stem cell line. For each 5mCpG within CGIs a
control 5mCpG was selected outside CGIs with the same degree of methylation and
coverage depths. We compared the percentage of mutated 5mCpGs in both groups
using Fisher’s exact test. to check for statistical significance.</p>

<h3>Results</h3>

<p class="abresults">5mCpGs are over 1.5 fold less likely to mutate into TpGs
inside CGIs comparing to the rest of the genome. This effect is statistically
significant (p &lt; 10e-20).</p>

<h3>Discussion</h3>

<p class="abdiscussion">We propose two possible explanations for the observed
decrease of 5mCpG mutation percentage in CGIs. First, deleterious variants are
expected to be underrepresented among SNPs. CGIs have been shown to carry
certain functions; therefore some SNPs within CGIs might have been eliminated
from the population as a result of natural selection. Second, the mutation
frequency of 5mCpG-&gt;TpG itself or the effectiveness of reparation systems
could be subject to local GC content influence. Several studies show that
adjacent methylated cytosines increase the effectiveness of mismatched thymine
replacement.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Yulia Medvedeva (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54442 = 'j&#117;.m&#101;dv&#101;d&#101;v&#97;' + '&#64;';
 addy54442 = addy54442 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy54442 + suffix + '\'' + attribs + '>' );
 document.write( addy54442 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7568 = 'j&#117;.m&#101;dv&#101;d&#101;v&#97;' + '&#64;';
 addy7568 = addy7568 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy7568 + suffix + '\'' + attribs + '>' );
 document.write( addy7568 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50682 = 'j&#117;.m&#101;dv&#101;d&#101;v&#97;' + '&#64;';
 addy50682 = addy50682 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy50682 + suffix + '\'' + attribs + '>' );
 document.write( addy50682 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Research Institute for Genetics and Selection of
Industrial Microorganisms</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Research Institute for Genetics and Selection of
Industrial Microorganisms, Moscow, Russia (2) Department of Bioengineering and
Bioinformatics, Moscow State University, Moscow, Russia (3) Institute for
Information Transmission Problems RAS, Moscow, Russia</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742408">B-27. Entropy approach
reveals new features of genes and genomes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Putintseva Y<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Putintseva Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sadovsky M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sadovsky M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Investigation of statistical properties of the nucleotide
sequence brings new facts, new knowledge, and inspires a researcher with new
methodology. Our approach is mathematically oriented and aims to retrieve as
much knowledge, as possible, from a sequence itself, avoiding an implementation
of any additional, external information. Here we proposed three new indices of codon
usage bias, as well as some other issues. All the indices are based on mutual
entropy calculation. They differ in the codon frequency distribution supposed
to be "quasi-equilibrium" one.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">1031 bacterial, 89 archaeal and 8 yeast annotated
genome sequences were retrieved from the European Bioinformatics Institute
Database (https://www.ebi.ac.uk/genomes/). S. cerevisiae nucleotide sequences
were taken from Saccharomyces Genome Database (http://www.yeastgenome.org/).
Sequences containing letters other than A, C, G, T were excluded from the
analysis. Frequency dictionary (including expected frequencies) and its entropy
figures (including mutual entropy) are the key tools of the studies.</p>

<h3>Results</h3>

<p class="abresults">Three indices of mutual entropy and absolute entropy were
calculated at the level of genomes and genes. Correlations between the figures
and the taxonomic position of the organism were observed with varying levels of
success. Genome regions, which differ considerably in their entropy
characteristics from the average of the genome, were identified (for example,
in S. cerevisiae genome a group of genes involved into flocculation process has
remarkably specific entropy characteristics). Numerous examples of statistically
identified sites have been studied, as well.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed a new index measuring codon usage bias. The
index, designated as mutual entropy, aims to retrieve as much knowledge, as
possible, from the genome sequence itself. Representation of all of the
information for a gene and genome by a single statistic is essentially a
reduction in information; therefore, no single measure reflects all aspects of
codon usage. Although measures, proposed here, also suffers from some
limitations we can conclude that it is a promising method for obtaining
quantitative information about the degree of overall synonymous codon usage of
a gene and genomes.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Yuliya A. Putintseva (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76601 = 'y&#117;l&#105;y&#97;-p&#117;t&#105;nts&#101;v&#97;' + '&#64;';
 addy76601 = addy76601 + 'r&#97;mbl&#101;r' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy76601 + suffix + '\'' + attribs + '>' );
 document.write( addy76601 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44807 = 'y&#117;l&#105;y&#97;-p&#117;t&#105;nts&#101;v&#97;' + '&#64;';
 addy44807 = addy44807 + 'r&#97;mbl&#101;r' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy44807 + suffix + '\'' + attribs + '>' );
 document.write( addy44807 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31857 = 'y&#117;l&#105;y&#97;-p&#117;t&#105;nts&#101;v&#97;' + '&#64;';
 addy31857 = addy31857 + 'r&#97;mbl&#101;r' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy31857 + suffix + '\'' + attribs + '>' );
 document.write( addy31857 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Siberian Federal University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 - Siberian Federal University 2 - Institute of
Computational Modelling of Russian Academy of Sciences</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection29">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742409">B-28. The hidden
duplication past of the plant pathogen Phytophthora and its consequences for
infection</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Martens C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Martens C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Oomycetes of the genus Phytophthora are pathogens that
infect a wide range of plant species. For dicot hosts such as tomato, potato
and soybean, Phytophthora is even the most important pathogen. Previous
analyses of Phytophthora genomes uncovered many genes, large gene families and
large genome sizes that can partially be explained by significant repeat
expansion patterns.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Analysis of the complete genomes of three different
Phytophthora species, using a newly developed approach, unveiled a large number
of small duplicated blocks, mainly consisting of two or three consecutive
genes. The detected number of duplicated genes was further compared with ten
other eukaryotes including parasites, algae, plants, fungi, vertebrates and
invertebrates, of which the gene and genome duplication history is known.</p>

<h3>Results</h3>

<p class="abresults">The performed analyses suggest that the ancestor of P.
infestans, P. sojae and P. ramorum most likely underwent a whole genome
duplication (WGD). Genes that have survived in duplicate are mainly genes that
are known to be preferentially retained following WGDs, but also genes
important for pathogenicity and infection of the different hosts seem to have
been retained in excess. As a result, the WGD might have contributed to the
evolutionary and pathogenic success of Phytophthora.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The fact that we find many small blocks of duplicated
genes indicates that the genomes of Phytophthora species have been heavily
rearranged following the WGD. Most likely, the high repeat content in these
genomes have played an important role in this rearrangement process. As a
consequence, the paucity of retained larger duplicated blocks has greatly
complicated previous attempts to detect remnants of a large-scale duplication
event in Phytophthora.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinformatics.psb.ugent.be" mce_href="http://bioinformatics.psb.ugent.be">http://bioinformatics.psb.ugent.be</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Cindy Martens (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93199 = 'c&#105;ndy.m&#97;rt&#101;ns' + '&#64;';
 addy93199 = addy93199 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy93199 + suffix + '\'' + attribs + '>' );
 document.write( addy93199 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54330 = 'c&#105;ndy.m&#97;rt&#101;ns' + '&#64;';
 addy54330 = addy54330 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy54330 + suffix + '\'' + attribs + '>' );
 document.write( addy54330 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13569 = 'c&#105;ndy.m&#97;rt&#101;ns' + '&#64;';
 addy13569 = addy13569 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy13569 + suffix + '\'' + attribs + '>' );
 document.write( addy13569 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Plant Systems Biology, VIB,
Technologiepark 927, B-9052 Ghent, Belgium (2) Bioinformatics and Systems
Biology, Department of Plant Biotechnology and Genetics, Technologiepark 927,
Ghent University, B-9052 Ghent, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">IWT</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection30">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742410">B-29. Protein linkages
between African Trypanosoma and 8 pathogenetic eukaryotic organisms</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dimitriadis D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Dimitriadis D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Trimpalis P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Trimpalis P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Choli-Papadopoulou T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Choli-Papadopoulou T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Anagnou NP<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Anagnou NP&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Kossida S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kossida S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Biological cells are regulated and controlled by
interacting proteins in metabolic and signaling pathways. Moreover, as the
processing power of computers is constantly being improved, the implementation
of computational methodologies studying biological systems and quantitatively
understanding their functionality and complexity is feasible and necessary.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The methodology we followed for the domain fusion
analysis is described in the below given steps: 1.1.The protein sequences of
the African trypanosome, Trypanosoma brucei,were retrieved. 1.2.The translated
sequences of 8 genomes (say which ones in this paranthesis) from NCBI &amp;
UniProt databases were also retrieved. 2.Trypanosoma brucei protein sequences
were compared (blastp) against the 8 retrieved proteomes by the “in-house”
software to predict functional associations based on fusion events.
3.Validation of the final predictions based on some criteria’s (see figure)</p>

<h3>Results</h3>

<p class="abresults">We identified 176 component of which 12 were excluded for
further analysis: 1.Functional annotation prediction. 2.Biological pathways through
the evolution, based on domain fusion analysis method.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We compared the 5 fused events against all the family of
eukaryotic organisms in each subkingdom, using BlastP. The scope of this step
was to make an observation and prediction for the evolution of the fusion
domains based on fusion or fission events within other eukaryotic organisms.</p>

<p class="abdiscussion">The in vivo confirmation of the fused predicted events
have being undergone. Future efficient results may </p>

<p class="abdiscussion">a.surface more details about the biological function and
the metabolic pathways of compounds.</p>

<p class="abdiscussion">b.Motivate the development of new drugs. </p>

<p class="abdiscussion">•we have supported the accuracy of the “in-house”
software</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Dimitris D.D Dimitriadis (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93054 = 'd&#105;m&#105;tr&#105;sd&#105;m&#105;tr&#105;&#97;d&#105;s' + '&#64;';
 addy93054 = addy93054 + 'm&#97;c' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy93054 + suffix + '\'' + attribs + '>' );
 document.write( addy93054 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98074 = 'd&#105;m&#105;tr&#105;sd&#105;m&#105;tr&#105;&#97;d&#105;s' + '&#64;';
 addy98074 = addy98074 + 'm&#97;c' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy98074 + suffix + '\'' + attribs + '>' );
 document.write( addy98074 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58811 = 'd&#105;m&#105;tr&#105;sd&#105;m&#105;tr&#105;&#97;d&#105;s' + '&#64;';
 addy58811 = addy58811 + 'm&#97;c' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy58811 + suffix + '\'' + attribs + '>' );
 document.write( addy58811 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioinformatics and Medical Informatics Team of the
Biomedical Research Foundation of the Academy of Athens.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1)Bioinformatics and Medical Informatics Team of the
Biomedical Research Foundation of the Academy of Athens. 2)department of
chemistry-Aristotle University of Thessaloniki 3)department of
medicine-National and Kapodistrian University of Athens</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection31">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742411">B-30. Reconstructing
phylogenetic trees from clustering trees</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Costa E<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Costa E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Vens C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vens C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Blockeel H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blockeel H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">In the context of phylogenetic tree reconstruction,
divisive clustering methods can be used to infer phylogenetic trees in a
top-down way. These methods have the important advantage of providing an
explanation for the resulting topology, since the splits are described by
polymorphic locations in the sequences. However, the quality of the resulting
trees is rather variable. In this work, we argue that trees induced by top-down
methods can be viewed not just as phylogenetic trees, but also as identifying
constraints that the real phylogenetic tree must satisfy.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We analyzed trees inferred by Clus-φ, a distance
based method for phylogenetic tree reconstruction based on a conceptual
clustering method that extends the well-known decision tree learning approach.
Each split defines two subclusters, such that the total branch length of the
tree is minimized. However, the split does not define how the subclusters have
to be connected. We propose a post-processing method that processes the
clustering tree bottom-up, at each split finding the internal branch that
connects the two subclusters with a minimal number of mutations.</p>

<h3>Results</h3>

<p class="abresults">To evaluate this method we used a number of synthetic datasets
generated by an evolutionary process simulator. In general, the post-processed
Clus-φ trees are more similar to the underlying target trees of the synthetic
datasets than the original Clus-φ trees, which shows that the post-processing
step yields a better approximation of the target tree. When we consider
Neighbor Joining and Parsimony results in this comparative analysis, we observe
that the post-processed Clus-φ trees tend to be better than the NJ trees and
are comparable to the parsimony trees.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results show that trees resulting from top-down
phylogenetic tree construction can be improved by post-processing them. This
post-processing is based on the viewpoint that the methods do not necessarily
return the correct tree, but return constraints that the correct tree must
satisfy. These constraints allow to guide the search for the tree with a
minimal number of mutations in a more exhaustive way than the greedy search
performed by parsimony methods. In general, the quality of post-processed
Clus-φ trees are comparable to that of parsimony trees.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Eduardo Costa (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy85796 = '&#101;d&#117;&#97;rd&#111;.c&#111;st&#97;' + '&#64;';
 addy85796 = addy85796 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy85796 + suffix + '\'' + attribs + '>' );
 document.write( addy85796 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64094 = '&#101;d&#117;&#97;rd&#111;.c&#111;st&#97;' + '&#64;';
 addy64094 = addy64094 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy64094 + suffix + '\'' + attribs + '>' );
 document.write( addy64094 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8472 = '&#101;d&#117;&#97;rd&#111;.c&#111;st&#97;' + '&#64;';
 addy8472 = addy8472 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy8472 + suffix + '\'' + attribs + '>' );
 document.write( addy8472 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 - Department of Computer Science, Katholieke
Universiteit Leuven, Celestijnenlaan 200A, 3001 Leuven, Belgium 2 - Institut
National de Recherche Agronomique, UMR 1301, 400 Route des Chappes, 06903
Sophia-Antipolis, France 3 - Leiden Institute of Advanced Computer Science,
Universiteit Leiden, Niels Bohrweg 1, 2333 CA Leiden, The Netherlands
{eduardo.costa,celine.vens,hendrik.blockeel}@cs.kuleuven.be</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Eduardo Costa is supported by the Research Foundation –
Flanders (FWO) and the GOA Probabilistic Logic Learning. Celine Vens is a
Postdoctoral Fellow of the Research Foundation –Flanders (FWO). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection32">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742412">B-31. Predictive
modeling of psychrophilic adaptation on the proteome sequence level</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Frickenhaus S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Frickenhaus S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Beszteri B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beszteri B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Psychrophilic sequence adaptation of cold-active
sequences is nowdays commonly analysed under the hypothesis that
protein-flexibility must be maintained in the cold. Based on structural models
and evaluation of local free-energy contributions the existence of families of
locally as well as globally cold-adapted protein sequences were found. The
present approach evaluates amino-acid properties numerically, allowing for
discriminative analyses of families of protein-homologues, negelcting
structural details.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Protein sequence alignments of homologues derived
from completed genome sequencing projects of six psychrophilic and six
mesophilic species are computed. The aligned aminoacid-sequences are translated
residue-wise into a matrix by a set of five biopysically meaningful
factor-scores. In a windowing approach along the alignment matrix linear
discriminant models are computed for each factor-score independently to explain
psychrophilic origin.</p>

<h3>Results</h3>

<p class="abresults">Statistical analyses of the resulting models demonstrate a
slight advantage in the predictive power for the factor-score IV corresponding
to amino-acid heat-capacities. However, it is not possible to derive a clear
relation between successful discrimination models and supersets of protein
function.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The result is consistent with the interpretation of
flexibility as amplitude of positional fluctuation, as experimentally shown by
neutron-scattering. A similar trend of heat-capacity is found from
proteome-average scores in a linear discriminant analysis.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.awi.de/en/go/bioinformatics" mce_href="http://www.awi.de/en/go/bioinformatics">http://www.awi.de/en/go/bioinformatics</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Stephan Frickenhaus (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86776 = 'st&#101;ph&#97;n.fr&#105;ck&#101;nh&#97;&#117;s' + '&#64;';
 addy86776 = addy86776 + '&#97;w&#105;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86776 + suffix + '\'' + attribs + '>' );
 document.write( addy86776 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84342 = 'st&#101;ph&#97;n.fr&#105;ck&#101;nh&#97;&#117;s' + '&#64;';
 addy84342 = addy84342 + '&#97;w&#105;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy84342 + suffix + '\'' + attribs + '>' );
 document.write( addy84342 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76724 = 'st&#101;ph&#97;n.fr&#105;ck&#101;nh&#97;&#117;s' + '&#64;';
 addy76724 = addy76724 + '&#97;w&#105;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy76724 + suffix + '\'' + attribs + '>' );
 document.write( addy76724 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Alfred-Wegener-Institute for Polar and Marine
Research Bremerhaven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1: Alfred-Wegener-Institute for Polar and Marine
Research, 2: Oregon State University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The project is funded within the AWI-research programme
PACES. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection33">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742413">B-32. Microbial
phenotype prediction based on protein domain profiles</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lingner T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Lingner T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Mühlhausen S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mühlhausen S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Meinicke P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Meinicke P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Accurately predicting microbial phenotypes solely based
on genomic features will allow us to infer relevant phenotypic characteristics
when the availability of a genome sequence precedes experimental
characterization. Although this scenario is favored by the advent of novel
high-throughput and single-cell sequencing techniques, no practical solution
exists. A particular challenge arises from the growing need to establish
genotype-phenotype associations without the application of an orthology search
step that is usually required by current phylogenomics approaches.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach for phenotype prediction is based on
the complete genomic sequences and the NCBI phenotype annotation of more than
1000 prokaryotic organisms. Pfam domain family occurrence profiles are inferred
from all organisms' genomic sequences and are then represented as
high-dimensional vectors of counts reflecting the organism-specific domain
abundance. The phenotype annotation is used to construct phenotype-specific
binary classification problems, which are then evaluated using a discriminative
machine learning technique.</p>

<h3>Results</h3>

<p class="abresults">Our approach provides high prediction accuracy regarding the
phenotype categories motility, Gram stain, oxygen requirement and spore
formation without the need for any orthology assignments. Our method
substantially outperforms an approach that is based on inferred metabolic
pathways and yields an average area under ROC curve of 0.972. Furthermore, the
set of discriminative domains provides biological insight into the underlying
mechanisms for a given phenotype and enables deriving hypotheses on the
possible functions of uncharacterized domains.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Fast and accurate orthology-free prediction of microbial
phenotypes based on genomic protein domain content is feasible and has the
potential to provide novel biological insights. First results of a systematic
check for annotation errors indicate that our approach may also be applied to
semi-automatic correction and completion of the existing phenotype annotation.
Furthermore, the analysis of a hierarchical clustering of discriminative domain
families indicate that the method can be used to suggest phenotype-specific
functions for uncharacterized protein families.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.gobics.de" mce_href="http://www.gobics.de">http://www.gobics.de</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Thomas Lingner (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86724 = 'th&#111;m&#97;s' + '&#64;';
 addy86724 = addy86724 + 'g&#111;b&#105;cs' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86724 + suffix + '\'' + attribs + '>' );
 document.write( addy86724 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63476 = 'th&#111;m&#97;s' + '&#64;';
 addy63476 = addy63476 + 'g&#111;b&#105;cs' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy63476 + suffix + '\'' + attribs + '>' );
 document.write( addy63476 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95938 = 'th&#111;m&#97;s' + '&#64;';
 addy95938 = addy95938 + 'g&#111;b&#105;cs' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy95938 + suffix + '\'' + attribs + '>' );
 document.write( addy95938 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Bioinformatics, Göttingen, Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Bioinformatics, Georg-August-University
Göttingen, Göttingen, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">DAAD PostDoc Fellowship to Thomas Lingner FUGE Vest
fellowship to Thomas Lingner</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection34">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742414">B-33. New insights into
the metazoan evolution of cadherins: from basal to modern</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hulpiau P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Hulpiau P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), van Roy F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Roy F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Specific cell-cell adhesion and intracellular communication
are key processes in multicellular animals. Members of the cadherin superfamily
are essential players in these processes. The details of premetazoan and
metazoan evolution of the cadherin superfamily are largely unknown, but the
many newly sequenced genomes of key metazoan organisms provide a rich resource
for unraveling evolution of such complex protein families.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used a combination of tBLASTn and profile HMM to
identify the cadherin repertoires in amphioxus, sea anemone and the placozoan
Trichoplax. To determine the evolution within the cadherin superfamily, we
extensively compared domain organization and sequences of cadherin and
cadherin-related proteins in more than 10 organisms from different metazoan
lineages. Using bl2seq, we compared extracellular cadherin (EC) domain blocks,
individual EC repeats and also non-EC domains. To complete the phylogenetic
study we constructed a Neighbor-Joining and Bayesian Inference tree of
cadherins all across the metazoan kingdom.</p>

<h3>Results</h3>

<p class="abresults">Classical cadherins, such as E-cadherin, arose from an
Urmetazoan cadherin, which progressively lost N-terminal extracellular cadherin
repeats while its cytoplasmic domain, which binds p120ctn and β-catenin,
remained quite conserved from placozoa to man. Protocadherins, until now
considered a chordate innovation, predate the Bilateria and are likely rooted
in an ancestral FAT cadherin. Flamingo,FAT-like and dachsous cadherins have
remained essentially unchanged. The remarkable ancient origin of several
cadherin types suggests that each of them has fulfilled separate and essential
needs throughout metazoan evolution.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The last common ancestor of animals expressed at least
five types of cadherins found in nearly all its descendants: a ‘classical’
cadherin, a Flamingo cadherin and three cadherin-related members FAT, FAT-like
and dachsous. Their similar domain composition suggests that they were
originally paralogs. Both modern ‘classical’ cadherins and protocadherins have
short ectodomains suggesting that the role of particular cadherins has changed
from intercellular signaling via loose contacts to tight interactions. This
might reflect the evolution from the limited requirements of a colonial
unicellular eukaryote to the strict morphogenetic programs essential for
metazoan organs.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Paco Hulpiau (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83622 = 'p&#97;c&#111;.h&#117;lp&#105;&#97;&#117;' + '&#64;';
 addy83622 = addy83622 + 'dmbr' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy83622 + suffix + '\'' + attribs + '>' );
 document.write( addy83622 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34882 = 'p&#97;c&#111;.h&#117;lp&#105;&#97;&#117;' + '&#64;';
 addy34882 = addy34882 + 'dmbr' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy34882 + suffix + '\'' + attribs + '>' );
 document.write( addy34882 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87322 = 'p&#97;c&#111;.h&#117;lp&#105;&#97;&#117;' + '&#64;';
 addy87322 = addy87322 + 'dmbr' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy87322 + suffix + '\'' + attribs + '>' );
 document.write( addy87322 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB &amp; Ghent Uiniversity</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department for Molecular Biomedical Research, VIB, Ghent,
Belgium (2) Department of Biomedical Molecular Biology, Ghent University,
Ghent, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Supported by the Research Foundation Flanders (FWO) and
the Geconcerteerde Onderzoeksacties of Ghent University. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection35">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742415">B-34. Comparative mapping
of transcription factor binding sites in plant genomes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wischnitzki E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Wischnitzki E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Vandepoele K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandepoele K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">The growing number of sequenced genomes provides an
unique opportunity to study the conservation of transcription factor binding
sites in plant species. The detection and evaluation of functional
transcription binding sites is essential but unfortunately not trivial. It is
especially hindered in plants by several small and large scale duplication
events, leading to a very diverse number of orthologous or in many cases
inparalogous genes. Our comparative mapping approach incorporates the
individual information of each related set of genes and is applicable to any
combination of genomes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To estimate the evolutionary conservation the
candidate motifs are evaluated using regulatory regions from orthologous genes.
For each gene set and motif the conservation value is calculated as a
probability for this event (p-value). The calculation is performed individually
for each combination of gene set and motif and takes the individual composition
and size of the set into consideration. Using this approach the p-value can be
assigned and the significance of the result is defined. The results are
evaluated using published TF target sets and enrichment for functional
categories.</p>

<h3>Results</h3>

<p class="abresults">Genes with significantly conserved binding sites show higher
enrichment of functional categories than a simple genome wide mapping or a
naive conservation filtering. Additionally known binding sites lacking
functional enrichment using a naive mapping approach can now be linked with the
underlying biological process they regulate. The comparison with experimental
data shows a clear reduction of false positive instances and increased
enrichment for the target sets. Our comparative mapping performed essentially
better than the other two methods which sometimes were equal to random.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results demonstrate that comparative mapping is a
powerful approach to identify functional binding sites and to distinguish them
from false positives instances. Our approach can also be applied to study gene
regulatory network inference and promoter evolution after speciation and/or
duplication. This makes it a very effective tool to detect conserved and
potentially functional transcription factor binding sites. Especially with the
growing number of sequenced genomes the applications of our approach become
more distinct and more detailed evolutionary conservation patterns can be
studied.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Elisabeth Wischnitzki (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86315 = '&#101;lw&#105;s' + '&#64;';
 addy86315 = addy86315 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86315 + suffix + '\'' + attribs + '>' );
 document.write( addy86315 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69209 = '&#101;lw&#105;s' + '&#64;';
 addy69209 = addy69209 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy69209 + suffix + '\'' + attribs + '>' );
 document.write( addy69209 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17315 = '&#101;lw&#105;s' + '&#64;';
 addy17315 = addy17315 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy17315 + suffix + '\'' + attribs + '>' );
 document.write( addy17315 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB/UGent</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Plant Systems Biology, Bioinformatics
and Systems Biology Division, VIB, Technologiepark 927, 9052 Ghent, Belgium 2
Department of Plant Biotechnology and Genetics, Ghent University,
Technologiepark 927, 9052 Ghent, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection36">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742416">B-35. Protein model
selection with ProtTest increases phylogenetic performance</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Patricio M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Patricio M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Abascal F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Abascal F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Zardoya R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zardoya R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2, Posada D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Posada D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Maximum likelihood and Bayesian phylogenetics rely on the
use of explict probabilistic models of evolution. Best-fit models of evolution
can be selected for the data at hand using statistical techniques like
hierarchical likelihood ratio tests, or information criteria. While the
connection between model selection and phylogenetic performance has been
systematically studied for DNA sequence alignments, we do not much about this
relationship in the case of protein alignments.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We simulated 5000 protein alignments (40 sequences
with 1000 amino acids) using 5000 non-clock trees. For each alignment, we
estimated the ML tree for every competing models (112 in total), and identified
the best-fit model using the AIC and BIC criteria as implemented in ProtTest.
To compare the phylogenetic performance of every competing model against the
best-fit model we measured the differences between the generating trees and the
estimated trees using the Robinson-Foulds symmetric distance (RF), the
Branch-Score (BS) and the K-tree score (K) with its associated scale factor
(SF).</p>

<h3>Results</h3>

<p class="abresults">ProtTest correctly identified the true empirical model of
amino acid replacement, or a very close one, most of the time. In general, the
best-fit model was selected with little uncertainty and therefore receiving
large weights, while model-averaged parameter estimates were also very precise.
Phylogenetic trees reconstructed under the best-fit model selected by ProtTest
were most accurate, together with the trees estimated under the true model used
in the simulations. This was so regardless of the criteria used for model
selection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Protein model selection using ProtTest increased
phylogenetic accuracy independently of the criteria used. Not surprisingly, the
most influential parameter was the shape of the gamma distribution, since the
performance of the +G models was always closer to that of the generating model.
A few particular models seem to work only slightly worse, like WAG and LG,
which were derived from comprehensive datasets using ML estimationFurther
simulations should explore whether these results hold under different
generating models, including more time-consuming mechanistic models not
explored here.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://darwin.uvigo.es" mce_href="http://darwin.uvigo.es">http://darwin.uvigo.es</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Mateus Patricio (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8961 = 'm&#97;t&#101;&#117;s' + '&#64;';
 addy8961 = addy8961 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy8961 + suffix + '\'' + attribs + '>' );
 document.write( addy8961 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83060 = 'm&#97;t&#101;&#117;s' + '&#64;';
 addy83060 = addy83060 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy83060 + suffix + '\'' + attribs + '>' );
 document.write( addy83060 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19791 = 'm&#97;t&#101;&#117;s' + '&#64;';
 addy19791 = addy19791 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy19791 + suffix + '\'' + attribs + '>' );
 document.write( addy19791 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Vigo</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Departament of Biochemistry, Genetics and
Immunology, University of Vigo, Vigo, Spain. (2) Departament of Biodiversity
and Evolutionary Biology, National Museum of Natural Sciences (CSIC), Madrid,
Spain.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This study was partially funded by the Spanish Ministry
of Science, Xunta de Galicia and European Research Council. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection37">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742417">B-36. cn.FARMS: a
probabilistic latent variable model to detect copy number variations</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Clevert D-A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Clevert D-A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2+,*), Mitterecker A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mitterecker A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mayr A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mayr A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Klambauer G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Klambauer G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tuefferd M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tuefferd M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), De Bondt A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Bondt A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Talloen W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Talloen W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Goehlmann HWH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Goehlmann HWH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Hochreiter S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hochreiter S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Existing pre-processing methods for DNA microarrays
designed to detect copy-number variations (CNVs) lead to high false discovery
rates (FDRs). High FDRs misguide researchers especially in the medical context
where CNVs are wrongly associated with diseases. Existing pre-processing
methods for DNA microarrays designed to detect copy-number variations (CNVs)
lead to high false discovery rates (FDRs). High FDRs misguide researchers
especially in the medical context where CNVs are wrongly associated with
diseases.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We suggest modeling the DNA copy numbers as a
latent variable in our factor analysis model, which is optimized by Bayesian maximum
a posteriori estimation. We originally developed this model for gene expression
arrays but we extend our approach for copy number analysis. In comparison to
other approaches we don’t need to correct for sequence effects to remove the
typical chromosome specific wave pattern correlated with GC content. cn.FARMS
can be used for single-locus and multi-loci CNV analysis. The latter constructs
so-called "meta-probe sets" combining probes from adjacent probe sets
to one</p>

<h3>Results</h3>

<p class="abresults">We compared cn.FARMS on HapMap Mapping250K_Nsp and SNP6.0
benchmark data sets to CRMA, dChip, CNAG and CNAT v4. The aim, described in
this presentation, is to distinguish males from females based on the X
chromosome copy numbers, where males possess one copy and females two. The ROC
curve serves to compare the FDR for different true positive rates. cn.FARMS
clearly outperforms its competitors. In addition we have assessed the CNVs
detection performance under the objective of the area under the
precision-recall curve. We found that, cn.FARMS performs superior to all other
compared methods.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Of course cn.FARMS is not limited to the Affymetrix
platform and can be applied to other platforms like Illumina bead arrays. The
concept remains the same: do genomic adjacent measurements tell the same story
about copy numbers?</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.bioinf.jku.at/software/farms/farms.html" mce_href="http://www.bioinf.jku.at/software/farms/farms.html">http://www.bioinf.jku.at/software/farms/farms.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Djork-Arne Clevert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52438 = '&#111;kk&#111;' + '&#64;';
 addy52438 = addy52438 + 'cl&#101;v&#101;rt' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy52438 + suffix + '\'' + attribs + '>' );
 document.write( addy52438 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37501 = '&#111;kk&#111;' + '&#64;';
 addy37501 = addy37501 + 'cl&#101;v&#101;rt' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy37501 + suffix + '\'' + attribs + '>' );
 document.write( addy37501 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27359 = '&#111;kk&#111;' + '&#64;';
 addy27359 = addy27359 + 'cl&#101;v&#101;rt' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy27359 + suffix + '\'' + attribs + '>' );
 document.write( addy27359 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Bioinformatics, Johannes Kepler
University Linz</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Bioinformatics, Johannes Kepler
University Linz, Linz, Austria (2) Department of Nephrology and Internal
Intensive Care, Charit e University Medicine, Berlin, Germany (3) Johnson &amp;
Johnson Pharmaceutical Research &amp; Development, a Division of Janssen P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Janssen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->harmaceutica, Beerse, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work has been funded by Janssen P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Janssen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->harmaceutica N.V. and the
Belgium IWT project 80536. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection38">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742418">B-37. Genome-wide
heterogeneity of the substitution process</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Arbiza L<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Arbiza L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Dopazo H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dopazo H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Posada D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Posada D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">At genomic scales, the patterns that have shaped
molecular evolution are largely heterogeneous. In particular, modern
phylogenetic approaches should use appropriate probabilistic substitution
models that capture the main features under which different genomic regions
have evolved. While efforts have concentrated in the development and understanding
model selection techniques, no descriptions of overall model fit at a complete
genomic scale have been reported. Because NGS technologies are providing vast
arrays of biological data, the understanding of model heterogeneity is
fundamental.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The longest transcripts of orthologous coding genes
from the complete genomes of 5 mammals and 15 vertebrates were obtained from
Ensembl v54. Sequences were aligned using Muscle and filtered with Gblocks. In
addition, filtered alignments for each of the 12 Drosophila genomes were
obtained from the Drosophila 12 Genomes Consortium. The jModelTest and Phyml
programs were used to estimate best-fit models of nucleotide substitution and
ML trees. Putative functional association with model parameters was studied
using the GO database, the program FatiScan and logit regressions using R.</p>

<h3>Results</h3>

<p class="abresults">Out of the 88 models considered, 82 were selected as
best-fit models at least in one occasion, although with very different
frequencies. Most parameter estimates also varied broadly among genes. The
patterns found for vertebrates and Drosophila were quite similar, and usually
more complex than those found in mammals. Phylogenetic trees derived from
models in the 95% CI set showed much less variance and were significantly
closer to the tree estimated under the best-fit model than trees derived from
models outside this interval. BIC selected simpler models, but suggested
similar patterns.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The analysis of ~20,000 alignments from three different
genomic sets clearly shows that different genes are best explained by different
models of nucleotide substitution, suggesting that the reasonably large variety
of substitution models available is justified in order to maximize statistical
model fit. The relevance of taking model selection uncertainty into account in
phylogenetic analysis is clear from our results. We conclude that the use of
model selection techniques is necessary to obtain accurate phylogenetic
estimates from real data at a genomic scale.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://darwin.uvigo.es" mce_href="http://darwin.uvigo.es">http://darwin.uvigo.es</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">David Posada (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3120 = 'dp&#111;s&#97;d&#97;' + '&#64;';
 addy3120 = addy3120 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy3120 + suffix + '\'' + attribs + '>' );
 document.write( addy3120 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14101 = 'dp&#111;s&#97;d&#97;' + '&#64;';
 addy14101 = addy14101 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy14101 + suffix + '\'' + attribs + '>' );
 document.write( addy14101 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72165 = 'dp&#111;s&#97;d&#97;' + '&#64;';
 addy72165 = addy72165 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy72165 + suffix + '\'' + attribs + '>' );
 document.write( addy72165 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Vigo</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Biochemistry, Genetics and Immunology,
University of Vigo, Spain (2) Comparative Genomics Unit, Bioinformatics Dept.,
CIPF, Valencia, Spain.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by the European
Research Council, Spanish Ministry of Science and Education and Xunta de Galicia</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection39">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742419">B-38. Comparative
microarray analysis to elucidate TF-networks in activated T-cells</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kröger S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kröger S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Scheel T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Scheel T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Leser U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Leser U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2) , Baumgrass R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Baumgrass R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">A better knowledge of the transcriptional regulation
during T cell activation is essential to understand the physiology and
pathophysiology of this important hub of the adaptive immune system. Therefore,
we created a basic gene regulatory network of T cell activation using public available
data sets of gene expression arrays. The major challenges therein were the
selection and heterogeneity of the available data sets. Eventually, we had to
combine data sets across experimental techniques, species, stimulation time,
and further experimental context.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">From GEO [Geo2009] a pre-selected set of of gene
expression sets were retrieved and curated by a immune biologists. All data
sets were subjected to local (data set specific) and global (over all sets)
RMA. The subsequent mapping of probe sets to Ensembl Gene IDs enables the
comparison across microarray platforms. Mapping was improved by ANOVA sibling
consolidation [Li2008]. We then used Network Component Analysis (NCA)
[Liao2003] to construct a regulatory network of T cell activation.</p>

<h3>Results</h3>

<p class="abresults">We present a large data set of T cell activation experiments
consisting of more than 300 microarrays. Using NCA we constructed a basic gene
regulatory network of differentially expressed transcription factors during T
cell activation. Thereby, we confirm that the joint analysis of different
microarray data sets is possible and helps to re-engineer regulatory networks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Further efforts are necessary to improve the process of
data selection. In addition, validation and expansion of the basic
transcription factor regulatory network is planned using analysis of global
transcription factor binding [Lee2008], DNA methylation and ChIP-sequencing.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://informatik.hu-berlin.de/%7Ekroeger/eccb2010/" mce_href="http://informatik.hu-berlin.de/~kroeger/eccb2010/">http://informatik.hu-berlin.de/~kroeger/eccb2010/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Stefan Kroeger (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34976 = 's.kr&#111;&#101;g&#101;r' + '&#64;';
 addy34976 = addy34976 + 'drfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy34976 + suffix + '\'' + attribs + '>' );
 document.write( addy34976 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7299 = 's.kr&#111;&#101;g&#101;r' + '&#64;';
 addy7299 = addy7299 + 'drfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy7299 + suffix + '\'' + attribs + '>' );
 document.write( addy7299 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26495 = 's.kr&#111;&#101;g&#101;r' + '&#64;';
 addy26495 = addy26495 + 'drfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy26495 + suffix + '\'' + attribs + '>' );
 document.write( addy26495 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">German Rheumatism Research Center Berlin</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 German Rheumatism Research Center Berlin 2 Humboldt Universität
zu Berlin</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection40">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742420">B-39. Gene-trait
matching analysis of Lactobacillus plantarum strains</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bayjanov JR<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bayjanov JR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Siezen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen RJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4,5), van Hijum SAFT (1,2,3,4,5)</p>

<p class="abmotivation">Identifying genotype-phenotype (gene-trait) relations
serves multiple purposes: (i) screen microorganisms for desired traits using
gene content, (ii) gain insight in gene function, and (iii) gain knowledge on
the effects of environmental factors that lead to gain or loss of a certain
trait in an organism. A very cost-effective way for inferring gene-trait
relations is to use high-throughput techniques such as comparative genome
hybridization, transcriptomics, metabolomics, and phenotypic microarrays. A major
bottleneck lies in the integration of these complex multivariate data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We are developing a method and software that allows
for non-parametric integration of multivariate data to identify many-to-many
relationships. Genes linking to phenotypes are presented to the user, allowing
for in-depth mining of such complex datasets. The software will be made
available as a web-tool.</p>

<h3>Results</h3>

<p class="abresults">The method is being tested on available genotype data
(presence/absence of genes based on comparative genome hybridization results)
and phenotype data (growth of strains on different sugars) for different
Lactobacillus plantarum strains. Our method allowed determining (i) gene
clusters associated to a single phenotype and (ii) gene clusters that were
associated to multiple phenotypes. Additionally, there were leads to probable
re-annotation of some genes in number of gene-phenotype relations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Current methods in multivariate data analysis have in
general the following shortcomings: (i) they correlate only one observed
variable (e.g., phenotypic measurement) with predictors (e.g., presence or
absence of genes), and (ii) the actual relation between gene presence and
phenotype is difficult to extract.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Jumamurat R. Bayjanov (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy48545 = 'j&#117;m&#97;m&#117;r&#97;t' + '&#64;';
 addy48545 = addy48545 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy48545 + suffix + '\'' + attribs + '>' );
 document.write( addy48545 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy353 = 'j&#117;m&#97;m&#117;r&#97;t' + '&#64;';
 addy353 = addy353 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy353 + suffix + '\'' + attribs + '>' );
 document.write( addy353 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24569 = 'j&#117;m&#97;m&#117;r&#97;t' + '&#64;';
 addy24569 = addy24569 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy24569 + suffix + '\'' + attribs + '>' );
 document.write( addy24569 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Centre for Molecular and Biomolecular Informatics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Center for Molecular and Biomolecular Informatics,
Radboud University Nijmegen Medical Center, the Netherlands. 2) NBIC, Toernooiveld
1, 6525 ED, Nijmegen, the Netherlands 3) TI Food and Nutrition, Wageningen, the
Netherlands 4) NIZO food research BV, Ede, the Netherlands 5) Kluyver Centre
for Genomics of Industrial Fermentation, Delft, the Netherlands.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">BSIK grant [through the Netherlands Genomics Initiative
(NGI)]; BioRange programme [as part of, the Netherlands Bioinformatics Centre
(NBIC)]; NGI (as part of the Kluyver Centre for Genomics of Industrial
Fermentation). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection41">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742421">B-40. ProtTest-HPC: fast
selection of best-fit models of protein evolution</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Darriba D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Darriba D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Taboada G-L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Taboada G-L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Doallo R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Doallo R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Posada D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Posada D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Using appropriate models of amino acid replacement is key
for the study of protein evolution and can increase phylogenetic performance.
ProtTest is a very popular tool for the selection of best-fit empirical models
of amino acid replacement. The maximum likelihood calculations required for
model selection can take a considerable time and can be unfeasible for large
alignments. Because the latter are becoming increasingly common, it is
necessary to develop computationally efficient solutions for protein model
selection.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a new version called ProtTest-HPC that
is able to distribute the workload (i.e., ML optimization) into the available
computational resources. We implemented and compared different strategies using
shared (available cores in a machine), distributed (message-passing version on
a whole cluster) and hybrid shared/distributed (two-level parallelism, relying
on message-passing for inter-node communications and on a thread-based approach
to exploit the available cores within each node) memory using MPI and OpenMP.</p>

<h3>Results</h3>

<p class="abresults">We have redesigned ProtTest for high performance computing.
ProtTest-HPC showed almost linear scalability with large input data in shared
memory architectures with a low number of cores. For the distributed memory
benchmarking, a two-step workload distribution was most efficient, scattering
tasks among cluster nodes and taking advantage of the available processor cores
per node. Moreover, we included new functionalities like the ability of
building consensus trees and fault tolerance.</p>

<h3>Discussion</h3>

<p class="abdiscussion">ProtTest-HPC provides a user-friendly graphic interface
for model selection tasks on multicore commodity/desktop systems, for current
multicore processors, and also an interface for queueing systems (script-based)
for distributed memory clusters. The speed up obtained with this version can be
considerable. For instance, a large alignment analysis could be reduced from
about 7 days using ProtTest to around 1 hour with ProtTest-HPC.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://darwin.uvigo.es/software/prottesthpc/" mce_href="http://darwin.uvigo.es/software/prottesthpc/">http://darwin.uvigo.es/software/prottesthpc/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">David Posada (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7356 = 'dp&#111;s&#97;d&#97;' + '&#64;';
 addy7356 = addy7356 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy7356 + suffix + '\'' + attribs + '>' );
 document.write( addy7356 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86148 = 'dp&#111;s&#97;d&#97;' + '&#64;';
 addy86148 = addy86148 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy86148 + suffix + '\'' + attribs + '>' );
 document.write( addy86148 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88662 = 'dp&#111;s&#97;d&#97;' + '&#64;';
 addy88662 = addy88662 + '&#117;v&#105;g&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy88662 + suffix + '\'' + attribs + '>' );
 document.write( addy88662 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Vigo</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Biochemistry, Genetics and
Immunology, University of Vigo, 36310 Vigo, Spain. (2) Department of
Electronics and Systems, University of A Coruna, 15071 A Coruna, Spain.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by the European
Research Council, Spanish Ministry of Science and Education and Xunta de
Galicia. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection42">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742422">B-41. NTRFinder: an
algorithm to find nested tandem repeats</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Matroud A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Matroud A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,2,*), Hendy M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hendy M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,2), Tuffley C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tuffley C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2)</p>

<p class="abmotivation">The ITS region of rRNA from Colocasia esculenta (taro), a
plant of ethnobotanical interest, has been found to contain a 1600 bp complex
repetitive structure consisting of two distinct tandem repeat motifs
interspersed with one another. We propose that such "nested tandem
repeats" are significant population markers and have developed the
algorithm NTRFinder to search for NTRs in other sequences. A major issue is
parsing, the determination of the optimal boundaries of the motifs, which is a
significant problem in the analysis of NTRs.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">NTRFinder adapts Wexler et al's heuristic for
finding tandem repeats to the problem of finding NTRs, and aligns and analyses
them using an extension of Fischetti et al's wrap-around dynamic programming.
The algorithm has been implemented in java. We propose a model of motif
duplication, mutation and deletion to explain the observed patterns. Under this
model the parsing problem is solved using the most parsimonious tree linking
the observed variants approximating each motif, minimising the number of
duplicate substitutions in the development of the NTR.</p>

<h3>Results</h3>

<p class="abresults">We have implemented and verified the algorithm with
extensive testing on simulated NTRs. We have begun searching real sequence
data. To date two other significant NTR regions of interest have been found.
Variations in the NTRs of taro varieties from different regions are indicating
its potential as a marker, and the rate of duplication appears to be
approximately twice the rate of nucleotide substitution.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have begun the search for NTRs from other genomes
using next generation sequencing. As the ITS region of rRNA is repeated hundreds
of times throughout the taro genome, the consistency of the NTR is a result of
concerted evolution, and further NTRs offer another window to understand this
process. Comparisons of these additional NTRs will both give a more detailed
pointer to the global spread of this prehistoric crop.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://arxiv.org/abs/1006.1730" mce_href="http://arxiv.org/abs/1006.1730">http://arxiv.org/abs/1006.1730</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Atheer Matroud (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15827 = '&#97;.&#97;.m&#97;tr&#111;&#117;d' + '&#64;';
 addy15827 = addy15827 + 'm&#97;ss&#101;y' + '&#46;' + '&#97;c' + '&#46;' + 'nz';
 document.write( '<a ' + path + '\'' + prefix + addy15827 + suffix + '\'' + attribs + '>' );
 document.write( addy15827 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72924 = '&#97;.&#97;.m&#97;tr&#111;&#117;d' + '&#64;';
 addy72924 = addy72924 + 'm&#97;ss&#101;y' + '&#46;' + '&#97;c' + '&#46;' + 'nz';
 document.write( '<a ' + path + '\'' + prefix + addy72924 + suffix + '\'' + attribs + '>' );
 document.write( addy72924 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73004 = '&#97;.&#97;.m&#97;tr&#111;&#117;d' + '&#64;';
 addy73004 = addy73004 + 'm&#97;ss&#101;y' + '&#46;' + '&#97;c' + '&#46;' + 'nz';
 document.write( '<a ' + path + '\'' + prefix + addy73004 + suffix + '\'' + attribs + '>' );
 document.write( addy73004 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Fundamental Sciences , Massey University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Allan Wilson Centre for Molecular Ecology and
Evolution, Massey University, Private Bag 11 222 Palmerston North, New Zealand.
2 Institute of Fundamental Sciences , Massey University, Private Bag 11 222 Palmerston
North, New Zealand.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project was funded by the Allan Wilson Centre for
Molecular Ecology and Evolution. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection43">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742423">B-42. Computational
epigenomics of plant SNF2 genes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bargsten JW<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bargsten JW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*,1,2,4), Mlynárová L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mlynárová L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,5), Nap JPH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nap JPH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,5)</p>

<p class="abmotivation">Chromatin remodeling controls the accessibility of DNA
for transcription via DNA methylation and histone modification, mediated by
ATPases of the so-called Snf2 family. Arabidopsis contains at least 41 members
of this family, some of which have already been associated with desirable
traits such as tolerance to biotic or abiotic stress. The Snf2 family is at the
same time conserved and highly diversified, posing computational challenges for
identification, characterization and classification of such genes in other
plant genomes as means to infer the function of individual family members.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Public domain genome resources of various plant
genomes (arabidopsis, poplar, rice, cucumber, tomato and more) and data in
different data warehouses (phytozome, UniRef, GenBank, ChromDB and more) were
used for the comparative analyses. As soon as allowed, whole genome analyses of
the soon-to-be-released tomato and potato genomes will be included in the comparisons.
Analysis and modeling involved the appropriate software available for
identification and classification, such as Blast, HMMER, MUSCLE, MEME and
PHYLIP, in addition to Perl-based custom approaches for sequence masking and
clustering.</p>

<h3>Results</h3>

<p class="abresults">The Snf2 family consists of genes with conserved ATPase and
helicase domains, interspersed with more diverse sequences. No less than 19
subfamilies in Arabidopsis may indicate the diverse functions of subfamily
members. We screened the publicly available genome data with the conserved
domains to identify family members and clustered genes on the basis of the
remaining domain-spanning sequences. Results show that the total number of
putative family members is surprisingly different in different plant genomes and
that various subfamilies differ in the number of individual members.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The distribution over subfamilies and the differences in
absolute number of Snf2 family members both hint at diversified functions
within plant species. These functions could go beyond the functions already
identified in or suggested for Arabidopsis. Further analyses of the structural
details of the subfamily members that differ between plant species, as well as
comparative expression analyses of the family members in targeted cells and/or
issues and/or mutational analyses for further functional characterization of
this gene family, will be presented at the meeting.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Joachim W. Bargsten (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92551 = 'j&#111;&#97;ch&#105;m.b&#97;rgst&#101;n' + '&#64;';
 addy92551 = addy92551 + 'w&#117;r' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy92551 + suffix + '\'' + attribs + '>' );
 document.write( addy92551 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy59648 = 'j&#111;&#97;ch&#105;m.b&#97;rgst&#101;n' + '&#64;';
 addy59648 = addy59648 + 'w&#117;r' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy59648 + suffix + '\'' + attribs + '>' );
 document.write( addy59648 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36479 = 'j&#111;&#97;ch&#105;m.b&#97;rgst&#101;n' + '&#64;';
 addy36479 = addy36479 + 'w&#117;r' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy36479 + suffix + '\'' + attribs + '>' );
 document.write( addy36479 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Applied Bioinformatics, Plant Research International,
Plant Sciences Group, Wageningen University and Research Centre, The
Netherlands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Applied Bioinformatics, Plant Research
International, Plant Sciences Group, Wageningen University and Research Centre,
The Netherlands 2. Laboratory for Plant Breeding, Plant Sciences Group,
Wageningen University, The Netherlands 3. Laboratory for Molecular Biology,
Plant Sciences Group, Wageningen University, The Netherlands 4. Netherlands
Bioinformatics Centre (NBIC), Nijmegen, The Netherlands 5. Centre for
BioSystems Genomics 2012 (CBSG2012), Wageningen, The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project was cofinanced by the Netherlands
Bioinformatics Centre (NBIC), and was carried out within the research program
of the Centre of BioSystems Genomics (CBSG), both of which are part of the
Netherlands Genomics Initiative / Netherlands Organization for Scientific
Research. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection44">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2 style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1"><a class="mceItemAnchor" name="_Toc269742424">B-43. Comprehensive
analysis of splice site evolution in primates using whole genome alignments and
RNA-Seq data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ongyerth M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ongyerth M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Prüfer K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Prüfer K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kelso J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kelso J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In recent years many new closely related genomes became
available for the comparison with the human genome. Next generation sequencing
is often used to supplement the genomic information with RNASeq data to
quantify expression levels and discover differences in transcript structure.
Splice sites, as one of the determinants of transcript structure, are encoded
in the majority of cases through a well characterized sequence motif. We use
this property to determine potential non-functional sites in non-human species
and verify our findings with RNASeq data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use the Ensembl and Vega gene annotation of the
human genome and a whole genome alignment between the human, chimpanzee,
bonobo, orangutan and rhesus macaque genomes to identify orthologous positions
of functional human splicing motifs in several primate species. To evaluate the
functional impact of changes in splice sites, we score motifs using a profile
weight matrix trained on human splice motifs. We use the score differences to
the non-human species to analyze correlations to several features of the splice
site and gene, and to identify potential non-functional sites.</p>

<h3>Results</h3>

<p class="abresults">We show that high splice score differences between non-human
and human species are more often observed with higher divergence between
species. Sites in protein coding transcripts are generally more conserved than splice
sites in pseudogenes or noncoding transcripts. Similarly, constituative splice
sites are more conserved than alternative splice sites. When using RNASeq data,
we observe that score differences are highly predictive of splice site usage.
However this result applies only for high score differences. Minor score
changes are not correlated with splice site usage.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our analysis shows that a simple model of splice sites
can be used to identify non-functional splice sites in closely related primate
species using solely human annotation. This result paves the way to a
comprehensive analysis of splice site overturn over the cause of evolution in
primate species. The analysis of individual candidates may uncover functionally
important differences caused by a difference in transcriptome structure between
human and other primates.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor" style="background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="background:#C6D9F1">Presenting Author</p>

</div>

<p class="presentingauthor">Matthias Ongyerth (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88488 = '&#111;ngy&#101;rth' + '&#64;';
 addy88488 = addy88488 + '&#101;v&#97;' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy88488 + suffix + '\'' + attribs + '>' );
 document.write( addy88488 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43270 = '&#111;ngy&#101;rth' + '&#64;';
 addy43270 = addy43270 + '&#101;v&#97;' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy43270 + suffix + '\'' + attribs + '>' );
 document.write( addy43270 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71361 = '&#111;ngy&#101;rth' + '&#64;';
 addy71361 = addy71361 + '&#101;v&#97;' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy71361 + suffix + '\'' + attribs + '>' );
 document.write( addy71361 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Max-Planck Institute for Evolutionary Anthropology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Max Planck Institute for Evolutionary Anthropology,
Leipzig, Germany</p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="115-comparative-genomics-phylogeny-and-evolution.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>