<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Protein and Nucleotide Structure</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Protein and Nucleotide Structure			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Protein and Nucleotide Structure</span><span style="font-variant: normal ! important; text-transform: uppercase;"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Anna Tramontano and Jan Gorodkin<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">Protein and Nucleotide Structure</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743008 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">1<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000300038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743009"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-1. Developing a methodology for protein-ligand docking
based on genetic algorithm and normal modes</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;display:none;mso-hide:screen;="" mso-no-proof:yes;text-decoration:none;text-underline:none=""><span style="">&nbsp; </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743009 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">3<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000300039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743010"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-2. A statistical potential for modelling of protein-RNA
complexes</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;="" display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:="" none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743010 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">4<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743011"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-3. ProBiS: a web server for detection of structurally
similar protein binding sites</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743011 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">5<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743012"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-4. Toward the prediction of the absolute quality of single
protein structure models</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743012 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">6<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743013"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-5. NMR data and protein structure</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743013 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">7<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743014"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-6. Predicting the effect of mutations on the thermal
stability of proteins with statistical potentials and artificial neural
networks</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;="" display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:="" none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743014 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">8<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743015"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-7. BioShell: a universal utility library for structural
bioinformatics</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743015 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">9<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743016"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-8. Molecular dynamic simulation studies of membrane bound
fully solvated β3 Adrenergic Receptor</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743016 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">10<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743017"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-9. The significance of the ProtDeform score</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743017 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">11<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743018"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-10. Exploiting synergy between computational biology and
X-ray crystallography for solving challenging macromolecular structures</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743018 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">12<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743019"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-11. Pattern recognition with moment invariants for
interpretation of very low resolution macromolecular density maps</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743019 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">13<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743020"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-12. PTools: an open source molecular docking library</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743020 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">14<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743021"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-13. Development of a structural alignment tool for protein
local surfaces</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743021 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">15<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743022"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-14. Blender Game Engine as a tool to navigate the
conformational space of proteins</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743022 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">16<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743023"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-15. Using data mining to identify structural rules in
proteins</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;="" display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:="" none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743023 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">17<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743024"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-16. Detecting protein domains of biological assemblies
using TopDomain-web</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743024 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">18<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743025"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-17. Metals in protein structures: classification and
functional prediction</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743025 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">19<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743026"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-18. Intuitive visualization of surface properties of moving
proteins</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;="" display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:="" none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743026 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">20<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743027"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-19. Hierarchical classification of helical distortions
related to proline</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743027 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">21<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743028"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-20. A “smooth” knowledge based potential for protein
structure refinement</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted"> </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743028 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">22<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743029"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-21. PROTEIN PEELING in 2010: recent developments and
applications</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743029 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">23<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000320039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743030"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-22. Local moves for efficient sampling of protein
conformational space</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743030 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">24<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743031"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-23. Self-organising maps applied to protein structure
classification</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743031 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">25<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743032"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-24. Construction of a sequence- and backbone-dependent
rotamer library by hidden Markov model</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743032 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">26<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743033"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-25. Codon usage and protein structure in prokaryotes: old
stories, new findings and puzzling questions</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;display:none;mso-hide:screen;="" mso-no-proof:yes;text-decoration:none;text-underline:none=""><span style="">&nbsp; </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743033 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">27<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743034"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-26. Analysis of FTICR mass spectra with H/D exchange of
deprotonated dinucleotides</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743034 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">28<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743035"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-27. Voroprot: an interactive tool for the analysis of
geometric features of protein structure</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743035 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">29<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743036"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-28. BBP - Beta-Barrel Predictor: a web server for the
prediction of the super-secondary structure of transmembrane β-barrel proteins</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743036 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">30<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743037"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-29. Prediction of three dimensional structure of protein
complexes</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;="" display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:="" none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743037 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">31<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743038"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-30. Investigating differences in the structural environment
of parallel and anti-parallel beta sheets</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743038 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">32<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743039"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-31. Protein model quality assessment based on structural
and functional similarities</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743039 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">33<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000330039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743040"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-32. Data driven structure prediction: calculating accurate
small angle X-ray scattering curves from coarse-grained protein models</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743040 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">34<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743041"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-33. OpenStructure: A flexible software framework for
computational structural biology</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743041 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">35<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743042"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-34. Modeling structure of ionic channels from rigid network
of contact sites</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743042 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">36<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743043"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-35. Multi-task sequence labeling for protein annotation</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743043 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">37<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743044"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-36. A convex programming model for protein structure
prediction</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;color:windowtext;="" display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:="" none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743044 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">38<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743045"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-37. Predicting protein function with the relative backbone
position kernel</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted"> </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743045 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">39<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743046"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-38. The Torsional Network Model: improvements in modelling
of protein flexibility</span><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:="" none;text-underline:none=""><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743046 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">40<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743047"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-39. MEDELLER: coordinate generation for membrane proteins</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743047 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">41<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><a href="116-protein-and-nucleotide-structure.html#_Toc269743048"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes="">C-40. Structure-based predictor of HIV coreceptor tropism</span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;"><span style="">.. </span></span><!--[if supportFields]><span
style='font-family:"Times New Roman","serif";color:windowtext;display:none;
mso-hide:screen;mso-no-proof:yes;text-decoration:none;text-underline:none'><span
style='mso-element:field-begin'></span> PAGEREF _Toc269743048 \h <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;; color: windowtext; display: none; text-decoration: none;">42<!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003000340038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none" mce_style="font-family:"Times New Roman","serif";
color:windowtext;display:none;mso-hide:screen;mso-no-proof:yes;text-decoration:
none;text-underline:none"><span style="mso-element:field-end" mce_style="mso-element:field-end"></span></span><![endif]--></a><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p></o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743009">C-1. Developing a methodology for protein-ligand
docking based on genetic algorithm and normal modes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lima AN<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Lima AN&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Philot EA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Philot EA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Scott LPB<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Scott LPB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Perahia D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Perahia D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The inconvenience of the fully flexible docking is the
high computational cost. Consequently, few method. Different approximations
exist depending which type of degrees of freedom are to be considered , and
this includes rigid docking where protein and ligand are rigid, semi-flexible
where only a selected set of side-chains of the protein are allowed to move, or
fully flexible where all the protein (including the backbone) and the ligand
are allowed to change their conformation. A methodology that allows a fully
flexible docking is very interisting.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">GANM combines Genetic Algorithm and Normal Modes
and uses a amino-acid Rotamer Library to simulate protein-ligand docking.
Within the framework of GA the degrees of freedom used include the rigid
translational and rotational motions of the ligand as well as its internal flexibility,
the rotameric states of protein side-chains within and around the binding
cavity, and the lowest frequency modes of the protein that describe its global
conformational changes. We are implementing and testing the methodology in four
versions.</p>

<h3>Results</h3>

<p class="abresults">We present here the docking results obtained with the
application of this two first version of GANM to different protein-ligand syste
and we compare our results wiht the reuslts of ohters docking programs.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We present the GANM method using a rotamer library to
simulate “semi-flexible” protein-ligand docking which consider a rigid backbone
and a rigid ligand but flexible side chains and the second version with the
backbone semi-flexible. Although this two first versions do not treat the
protein as being completely flexible, it can however be interesting to use it
as a first approach to rapidly dock a ligand to a protein or to achieve a
virtual screening for searching putative ligands.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Luis P. B. Scott (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26963 = 'l&#117;&#105;s.sc&#111;tt' + '&#64;';
 addy26963 = addy26963 + '&#117;f&#97;bc' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'br';
 document.write( '<a ' + path + '\'' + prefix + addy26963 + suffix + '\'' + attribs + '>' );
 document.write( addy26963 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">UFABC - CMCC</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Universidade Federal do ABC, CMCC. (2)Universidade
Federal do ABC , Program of Master in Information Eginnering (3)Université
Paris-Sud, Orsay, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The authors aknowledge the Fapesp (Brazil) and the CNRS
(France) for the support to this work</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743010">C-2. A statistical potential for modelling of
protein-RNA complexes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tuszynska I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Tuszynska I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2*), Rother K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rother K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Bujnicki JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bujnicki JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">RNA-protein interactions are extremely essential to gene
expression in the cell. Methods which are able to predict protein-RNA complexes
are needed to understand the principles of protein-RNA recognition and next to
design new RNA-binding proteins.There are two statistical potentials for
protein-RNA interaction prediction, which are developed by the Varani and the
Fernandez groups. But first potential recognised very near native protein-RNA
structures, with RMS</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our potentials comprise a distance-dependent energy
term (Er),angular-dependent energy component (Ea),site-dependent energy term
(Es) and penalty for sterical clashes (Ep). Before the calculation, potentials
reduce the all atom representation to the reduced representation, which is used
in the Refiner program for a protein and the RedRNA program for RNA developed
in our laboratory.</p>

<h3>Results</h3>

<p class="abresults">Score-rmsd plots as results of scoring of four decoys from
bound docking. There are two testing sets generated by our group with GRAMM
program and by the Varani group. Each test set was scored by our quasichemical
potential and the potential developed by the Varani group. The first best
scored decoy of our quasichemical potential function for each complex from the
testing set compared to the native complex are presented. Only proteins are
superimposed.</p>

<h3>Discussion</h3>

<p class="abdiscussion">1.Our potentials discriminate native-like (with RMS &lt;
10 Å) structures of protein-RNA complexes, while the potential developed by the
Varani group recognises structures very close to the native (RMS &lt; 5 Å) 2.Potentials
give better results for thetest setobtained bytheVarani group, where
perturbation was applied to the native structures of complexes. Hence, the
using of refinement for GRAMM rigid body docking decoys before the application
ofour potentials may improve the discrimination power of potentials. 3.To
eliminate false positives in the final stage of the prediction of native
structure o</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Irina Tuszynska (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16134 = '&#105;r&#101;n&#97;' + '&#64;';
 addy16134 = addy16134 + 'g&#101;n&#101;s&#105;l&#105;c&#111;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy16134 + suffix + '\'' + attribs + '>' );
 document.write( addy16134 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93473 = '&#105;r&#101;n&#97;' + '&#64;';
 addy93473 = addy93473 + 'g&#101;n&#101;s&#105;l&#105;c&#111;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy93473 + suffix + '\'' + attribs + '>' );
 document.write( addy93473 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36850 = '&#105;r&#101;n&#97;' + '&#64;';
 addy36850 = addy36850 + 'g&#101;n&#101;s&#105;l&#105;c&#111;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy36850 + suffix + '\'' + attribs + '>' );
 document.write( addy36850 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">International Institute of Molecular and Cell Biology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Laboratory of Bioinformatics and Protein
Engineering, International Institute of Molecular and Cell Biology, Trojdena 4,
02-109 Warsaw, Poland 2. Laboratory of Bioinformatics and Protein Engineering,
International Institute of Molecular and Cell Biology, Trojdena 4, 02-109
Warsaw, Poland 3. Bioinformatics Laboratory, Institute of Molecular Biology and
Biotechnology, Adam Mickiewicz University, Umultowska 89, 61-614 Poznan, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by following
grants:HISZPANIA/152/2006from the Polish Ministry of Scienceand the EU 6FP
Network of Excellence EURASNET (grant LSHG-CT-2005-518238) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743011">C-3. ProBiS: a web server for detection of
structurally similar protein binding sites</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Konc J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Konc J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Janežič D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Janežič D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Exploitation of locally conserved three-dimensional
patterns of physicochemical properties on the surface of a protein for
detection of binding sites that may lack sequence and global structural
conservation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Web server ProBiS compares the query protein to
members of a database of protein 3D structures and detects with sub-residue
precision, structurally similar sites as patterns of physicochemical properties
on the protein surface. Using an efficient maximum clique algorithm, the server
identifies proteins which share local structural similarities with the query
protein and generates structure-based alignments of these proteins with the
query. Structural similarity scores are calculated for the query protein’s
surface residues, and are expressed as different colors on the query protein
surface.</p>

<h3>Results</h3>

<p class="abresults">Given a structure of a protein with unknown binding sites,
ProBiS suggests the regions on its surface which may be involved in binding
with small ligands, proteins or DNA/RNA. Alternatively, given a protein with an
identified binding site, ProBiS finds other proteins with structurally or
physicochemically similar binding sites. If used as a pairwise structure
alignment program, ProBiS detects and superimposes similar functional sites in
a pair of submitted protein structures, even when these do not have similar
folds.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Detection of binding sites which depends on structural
similarity of protein surfaces is useful and accurate and can enjoy success in
structures with dissimilar folding patterns. Structural similarity can provide
additional advantages over sequence conservation in the detection of functional
regions such as binding sites. Detailed instructions and user guidelines for
use of ProBiS web server are available at http://probis.cmm.ki.si under 'HELP'
and selected examples are provided under 'EXAMPLES'.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://probis.cmm.ki.si" mce_href="http://probis.cmm.ki.si">http://probis.cmm.ki.si</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Janez Konc (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97527 = 'k&#111;nc' + '&#64;';
 addy97527 = addy97527 + 'cmm' + '&#46;' + 'k&#105;' + '&#46;' + 's&#105;';
 document.write( '<a ' + path + '\'' + prefix + addy97527 + suffix + '\'' + attribs + '>' );
 document.write( addy97527 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67003 = 'k&#111;nc' + '&#64;';
 addy67003 = addy67003 + 'cmm' + '&#46;' + 'k&#105;' + '&#46;' + 's&#105;';
 document.write( '<a ' + path + '\'' + prefix + addy67003 + suffix + '\'' + attribs + '>' );
 document.write( addy67003 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24652 = 'k&#111;nc' + '&#64;';
 addy24652 = addy24652 + 'cmm' + '&#46;' + 'k&#105;' + '&#46;' + 's&#105;';
 document.write( '<a ' + path + '\'' + prefix + addy24652 + suffix + '\'' + attribs + '>' );
 document.write( addy24652 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">National Institute of Chemistry</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) National Institute of Chemistry, Hajdrihova 19,
1000 Ljubljana, Slovenia (2) University of Primorska, Faculty for Mathematics,
Natural Sciences and Information Technologies, Glagoljaška 8, 6000 Koper,
Slovenia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Ministry of Higher Education, Science and Technology of
Slovenia; Slovenian research agency (Z1-31666). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743012">C-4. Toward the prediction of the absolute quality
of single protein structure models</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Benkert P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Benkert P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Biasini M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Biasini M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Schwede T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schwede T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Estimating the quality of predicted structural models is
a vital step in homology modeling since the individual models may contain
considerable errors. To identify such inaccuracies, scoring functions have been
developed which analyse different structural features of the protein models in
order to generate a quality estimate. Most state-of-the-art scoring functions
are primarily designed to rank alternative models of the same sequence. However,
the relative quality of a model within an ensemble of models is not sufficient
for determining its usefulness for different biomedical applications.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Recently, we have introduced the composite scoring
function QMEAN which consists of four statistical potential terms (two
non-bonded interaction terms, a torsion angle term and a solvation term) and
two components describing the agreement between predicted and observed
secondary structure and solvent accessibility. QMEAN has been successfully used
in model selection and has now been extended to provide an estimate of the
absolute quality of a single model. This QMEAN Z-score is a measure of the
likelihood that a given model is of a quality comparable to experimental
structures.</p>

<h3>Results</h3>

<p class="abresults">We calculated the QMEAN Z-score for all protein chains from
the PDB solved by X-ray crystallography and identified outliers with very
high/low score. On the one side of the spectrum we observe positive QMEAN
Z-score &gt; 3 standard deviations for extremely stable proteins from
(hyper-)thermophilic organisms. On the other side, membrane proteins which are
known to require a lipid bilayer to maintain their structural integrity, result
in negative QMEAN Z-score &gt; 3 standard deviations. We also identified several
structures in the PDB database with very low Z-scores, some known to be
fabricated.</p>

<h3>Discussion</h3>

<p class="abdiscussion">These results indicate that the new QMEAN score is
reasonable measure of the stability and integrity of protein structures. We
show that the QMEAN Z-score can not only be used to identify experimental
structures of poor quality but also to assess the quality of theoretical
protein models and thereby serve as an absolute measure of model quality.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://swissmodel.expasy.org/qmean" mce_href="http://swissmodel.expasy.org/qmean">http://swissmodel.expasy.org/qmean</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pascal Benkert (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38579 = 'p&#97;sc&#97;l.b&#101;nk&#101;rt' + '&#64;';
 addy38579 = addy38579 + '&#117;n&#105;b&#97;s' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy38579 + suffix + '\'' + attribs + '>' );
 document.write( addy38579 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Swiss Institute of Bioinformatics, Biozentrum,
University of Basel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Biozentrum, University of Basel, Switzerland 2 SIB
Swiss Institute of Bioinformatics, Basel, Switzerland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">SIB Swiss Institute of Bioinformatics</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743013">C-5. NMR data and protein structure</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vranken W<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vranken W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Nuclear Magnetic Resonance (NMR) is one of the key
experimental techniques to determine the structure of biomolecules. Compared to
X-ray crystallography, NMR data is complex and the software to analyse it
fragmented. At the PDBe we have been working with the Collaborative Computing
Project for the NMR community (CCPN) framework to make archived NMR data
consistent with archived Protein Data Bank (PDB) coordinate data. We make this
extensive data available to the community and have started to analyse it to
gain new insights into the relation between protein structure and NMR
paramaters.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">All software is developed in Python. The CCPN
framework is used to store all NMR data, the CcpNmr FormatConverter to parse
all archived data files. The NMR data is analysed and graphs are generated with
the RPy package (connecting Python to R). Django is used for dynamic display of
the web pages.</p>

<h3>Results</h3>

<p class="abresults">1. We now capture as much NMR data as possible at the time
the coordinates are deposited by accepting full CCPN projects. 2. We continue
to provide reference data for CCPN and, in collaboration, consistent NMR
restraints and recalculated NMR structure coordinates. 3. We develop software
for the community using the CCPN framework, for example for NMR data format
data conversions (CcpNmr FormatConverter). 4. We collaborate to relate chemical
shift and NMR restraint data to coordinate information, and work on analysing
NMR structure ensembles.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Researchers who want to use structures determined by NMR
often find it difficult to make sense of the way the structures are presented,
or have difficulty locating the information they require in a standardized
form. At the PDBe, we continue to work with the NMR community to provide
services and analyses to help the user and make NMR data more accessible.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.pdbe.org/nmr" mce_href="http://www.pdbe.org/nmr">http://www.pdbe.org/nmr</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Wim F. Vranken (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74502 = 'w&#105;m' + '&#64;';
 addy74502 = addy74502 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy74502 + suffix + '\'' + attribs + '>' );
 document.write( addy74502 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46714 = 'w&#105;m' + '&#64;';
 addy46714 = addy46714 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy46714 + suffix + '\'' + attribs + '>' );
 document.write( addy46714 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19923 = 'w&#105;m' + '&#64;';
 addy19923 = addy19923 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy19923 + suffix + '\'' + attribs + '>' );
 document.write( addy19923 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">European Bioinformatics Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Protein Data Bank in Europe (PDBe) European Bioinformatics
Institute (EBI) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743014">C-6. Predicting the effect of mutations on the
thermal stability of proteins with statistical potentials and artificial neural
networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Folch B<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Folch B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rooman M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rooman M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Dehouck Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dehouck Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Proteins are exploited in applications such as the design
of therapeutic agents, or in the agro-food or biotechnology sector. To improve
yield or specificity, it is interesting to tune their properties via amino acid
substitutions. In particular, maintaining its activity in unusual temperature
conditions is often crucial. Although several methods have been designed to
predict changes in thermodynamic stability at room temperature upon mutation
(DDG), there is still a stringent need for predictive models that focus
specifically on the effect of mutations on the thermal resistance of proteins.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our method relies on a set of statistical
potentials, extracted protein structures, describing the couplings between four
protein descriptors (sequence, distance, torsion angles and solvent
accessibility). Terms accounting for volume variations upon mutation are also
considered. The change in melting temperature is expressed as a linear
combination of these potentials, whose weights are functions of the solvent
accessibility of the mutated residue, identified with the help of a neural
network. The network is trained and validated on a dataset on 1601
experimentally characterized mutations.</p>

<h3>Results</h3>

<p class="abresults">We used a 5-fold validation procedure to assess the
performances of our model. The correlation coefficient (R) between computed and
measured changes in melting temperature (DTm) is 0.73, for 90% of the
mutations. In contrast, when DDG predictions are used to estimate DTm values,
the performances drop significantly (R=0.67). Interesting differences are
observed between DTm and DDG predictors. In particular, to predict thermal
stability changes, local interactions are more important in the core than on
the surface, although the opposite trend is observed for DDG predictions.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We showed that accurate predictions of thermal stability
changes (DTm) upon mutation can only be achieved through predictive models
specifically focused on thermal stability. Indeed, even though thermodynamic stability
changes (DDG) are related to thermal stability changes, DDG predictions only
allow a poor evaluation of DTm values. Moreover, our model reveals that for
some types of interactions, the weighting functions strongly differ between DDG
and DTm predictors. This indicates differences between the mechanisms that rule
protein stability at room temperature and at higher temperatures.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Yves Dehouck (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86846 = 'yd&#101;h&#111;&#117;ck' + '&#64;';
 addy86846 = addy86846 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86846 + suffix + '\'' + attribs + '>' );
 document.write( addy86846 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15499 = 'yd&#101;h&#111;&#117;ck' + '&#64;';
 addy15499 = addy15499 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy15499 + suffix + '\'' + attribs + '>' );
 document.write( addy15499 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73121 = 'yd&#101;h&#111;&#117;ck' + '&#64;';
 addy73121 = addy73121 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy73121 + suffix + '\'' + attribs + '>' );
 document.write( addy73121 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Unité de Bioinformatique génomique et structurale,
Université Libre de Bruxelles, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge support from the Belgian State Science
Policy Office through an Interuniversity Attraction Poles Programme (DYSCO),
the Belgian Fund for Scientific Research (FNRS) (FRIA grant to BF and FRFC
project), the Héger-Massa Fund and the Brussels Region (TheraVip project). MR
is Research Director at the FNRS. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743015">C-7. BioShell: a universal utility library for structural
bioinformatics</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gront D<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Gront D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*)</p>

<p class="abmotivation">The amount of biological data can be overwhelming,
therefore automated processing methods are indispensable in bioinformatical
research. BioShell package is one of the very few tools that cover very wide range
of applications, providing keeping intuitive design and great ease of use.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">BioShell project has been started in 2005 as a set
of stand-alone programs. It has since then evolved to a fully featured
scripting language for biomolecular modeling and structural bioinformatics. It
is addressed to a wide audience of users. It can be used in three ways: • by
calling command-line programs to do simple tasks as statistical data analysis,
ﬁle parsing, sequence and structure alignment and rms calculations • as a
library of modules for scripting languages running on JVM, such as Python or
Ruby implementations • as a library for bioinformatics software development in
Java.</p>

<h3>Results</h3>

<p class="abresults">BioShell has already played the crucial role in several scientific
projects. For example, its graph library combined with a PDB parser were used
to scan the whole PDB to find a particular topology of interdigitated
b-strands. In a another case a large set of binding pockets was automatically
detected and compared. Other applications comprises various calculations on
biomolecular structures and processing trajectories from molecular simulations.
The newest features include clustering of protein structures, optimal structure
alignment (for any arbitrary coverage) and a PsiBlast parser/filter.</p>

<h3>Discussion</h3>

<p class="abdiscussion">One of the biggest challenges in designing a
computational tool is to combine the variety of different modules and options
with the ease to learn and to use. BioShell users may begin with simple
command-line tools and later switch to writing scripts. The combination of Java
with Jython gives an easy access to any fragment of the BioShell code. The
project's website provides many examples that cover almost the whole range of
BioShell applications.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioshell.chem.uw.edu.pl/" mce_href="http://bioshell.chem.uw.edu.pl/">http://bioshell.chem.uw.edu.pl/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Dominik Gront (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24941 = 'dgr&#111;nt' + '&#64;';
 addy24941 = addy24941 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy24941 + suffix + '\'' + attribs + '>' );
 document.write( addy24941 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90186 = 'dgr&#111;nt' + '&#64;';
 addy90186 = addy90186 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy90186 + suffix + '\'' + attribs + '>' );
 document.write( addy90186 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54754 = 'dgr&#111;nt' + '&#64;';
 addy54754 = addy54754 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy54754 + suffix + '\'' + attribs + '>' );
 document.write( addy54754 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Warsaw, Faculty of Chemistry</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">University of Warsaw, Faculty of Chemistry ul. Pasteura
1, 02-093 Warsaw, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Support from Marie Curie fellowship (FP7-people-IOF) is
greatly acknowledged. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743016">C-8. Molecular dynamic simulation studies of
membrane bound fully solvated β3 Adrenergic Receptor</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tewatia P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Tewatia P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Malik BK<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Malik BK&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sahi S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sahi S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">β3-adrenergic receptors (β3- ARs) belong to the widely
studied class A of the G-protein coupled receptor (GPCR) super family. β3-ARs
are located in the plasma membrane of both white and brown adipocytes where
they mediate metabolic effects such as lipolysis and thermogenesis. They are
reported to exist in human heart, gastrointestinal tract, urinary bladder
detrusor, prostate, brain and in near term myometrium. Researchers are working
to understand their function and regulation in different human tissues as they
are also attractive drug target for treatment of obesity &amp; adult-onset
diabetes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A 5.0 ns Molecular dynamics (MD) simulation was performed
of the 3D-model of β3-AR using the DESMOND version 2.2. The 3D-model of β3 AR
was solvated with three point simple point charge (SPC) water molecules,
1-palmitoyl-2-oleoyl-phosphatidylcholine (POPC) membrane was also incorporated.
The system was subjected to 20,000 steps of conjugate gradient energy
minimization. Simulation was performed in the NVT ensemble with thermostat set
at 300 K. Conformations were saved every 10 ps. All calculations were performed
on a Linux cluster of 8 computers with 16 processors.</p>

<h3>Results</h3>

<p class="abresults">Comparing the initial structure and the structure after 5.0
ns simulation dynamic perturbations in the structure of the protein especially
in the loop regions were observed. Matching the Cα of the transmembrane helices
give RMSD of 3.2 Å and for the loops the Cα RMSD is 6.8 Å. The loop regions: I1
(intracellular loop 1 residues 64 to 72), I2 (residues 134 to 155), I3
(residues 226 to 292), E1 (Extracellular loop 1) and some regions of the
helices showed major dynamic perturbations. These were further analyzed as I2
and I3 loops are important for selectivity and affinity for G proteins.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Comparing the initial structure to the structure after 5
ns simulation gives RMSD of 3.85 Å with major conformational changes in the
loop regions. The helices are tilted a little closer to each other. On
superimposing and comparing the trajectories obtained during production phase
an average RMSD of 1.3Å was observed indicating fair overall stability of the
different conformations in the helical region and high flexibility in the loop
regions. The trajectory was analyzed and the average coordinate file generated
during production phase of simulation was taken as the final model of β3 AR.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Parul Tewatia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51904 = 'pt&#101;w&#97;t&#105;&#97;' + '&#64;';
 addy51904 = addy51904 + '&#97;m&#105;ty' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy51904 + suffix + '\'' + attribs + '>' );
 document.write( addy51904 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28764 = 'pt&#101;w&#97;t&#105;&#97;' + '&#64;';
 addy28764 = addy28764 + '&#97;m&#105;ty' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy28764 + suffix + '\'' + attribs + '>' );
 document.write( addy28764 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97639 = 'pt&#101;w&#97;t&#105;&#97;' + '&#64;';
 addy97639 = addy97639 + '&#97;m&#105;ty' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy97639 + suffix + '\'' + attribs + '>' );
 document.write( addy97639 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Amity Institute of Biotechnology, Amity University,
Noida, UP, India</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Amity Institute of Biotechnology, Amity University,
Expressway, Sec 125, Noida, UP, India (2) School of Biotechnology, Gautam
Buddha University, Greater Noida, UP, India</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743017">C-9. The significance of the ProtDeform score</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rocha J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Rocha J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(*), Alberich R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alberich R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">When a researcher uses a program to align two proteins
and gets a score, one of her main concerns is how often the program gives a
similar score to pairs that are or are not in the same fold. This issue was
analysed in detail recently for the program TM-align with its associated
TM-score. It was shown that because the TM-score is length independent, it allows
a P-value and a hit probability to be defined depending only on the score.
Also, they found that the TM-scores of gapless alignments closely follow an
Extreme Value Distribution. ProtDeform score (PDscore) needs to be analysed.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this study, we define the PD-score in a coarser
version based on backbone fragments instead of amino acids (Bioinformatics, 25:
1625-1631, 2009). We use the Gold Standard Benchmark (BMC Structural Biology,
9:23, 2009) and the one used for the TM-score analysis and one for gapless
score analysis. We fit PD-scores frequencies to EVD and show PD-score
independence from protein length. We calculate the a posteriori probability of
a pair in the same fold given a PD-score and found the score thresholds also
for homology discrimination. We plot prob. of Type I errors vs prob. of Type II
errors.</p>

<h3>Results</h3>

<p class="abresults">This study on the ProtDeform score reveals that it is also
length independent and that PD-scores of gapless alignments also follow
approximately an EVD. Using the Gold Standard benchmark, PD-scores have lower
probabilities of error than TM-scores all at a similar speed. The analysis is
extended to homology discrimination showing that, again, ProtDeform gives
higher hit probabilities than TM-align. We suggest using three different
P-values according to the three different contexts: Gapless alignments, optimised
alignments for fold discrimination and that for superfamily discrimination.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We estimated three different P-values for the three
different discrimination problems we are faced with: one is for the scores
obtained for gapless alignments between domains of the same length; a second
one is for the scores produced by optimisation programs on domains of different
topology; and the third one is for the scores produced by the same programs on
domains of different homology. Mixing these hypotheses can under-or
over-estimate the significance of the scores seen. We have found that PD-scores
are length independent, discriminant and with a known significance.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinfo.uib.es/%7Erecerca/ProtDeform/" mce_href="http://bioinfo.uib.es/~recerca/ProtDeform/">http://bioinfo.uib.es/~recerca/ProtDeform/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jairo E. Rocha (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42451 = 'j&#97;&#105;r&#111;' + '&#64;';
 addy42451 = addy42451 + '&#117;&#105;b' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy42451 + suffix + '\'' + attribs + '>' );
 document.write( addy42451 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42193 = 'j&#97;&#105;r&#111;' + '&#64;';
 addy42193 = addy42193 + '&#117;&#105;b' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy42193 + suffix + '\'' + attribs + '>' );
 document.write( addy42193 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39047 = 'j&#97;&#105;r&#111;' + '&#64;';
 addy39047 = addy39047 + '&#117;&#105;b' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy39047 + suffix + '\'' + attribs + '>' );
 document.write( addy39047 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of the Balearic Islands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Mathematics and Computer Science and University
Institute for Health Research University of the Balearic Islands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Spanish Ministry of Science and Technology
[MTM2009-07165]. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743018">C-10. Exploiting synergy between computational biology
and X-ray crystallography for solving challenging macromolecular structures</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wiegels T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Wiegels T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Lamzin V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lamzin V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Solving structures of large macromolecular complexes is a
challenging task in macromolecular crystallography (MX). Such crystals rarely
diffract to high resolution, resulting in a scarce amount of data for
generating electron density maps. Automated modelling approaches have been
focused on high-resolution and their application to data extended to less than
3.0 Å typically results in incomplete and highly fragmented models. Therefore a
use of complementary information for structure completion is highly desirable
for robust and automated solution of low-resolution MX structures.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">ARP/wARP - one of the leading MX software projects
- uses small protein-specific building blocks (residues or short poly-peptides)
to build a model from MX electron density maps in an iterative approach. At
least 50% of all protein structures in the PDB contain NCS-related fragments
(subunits or whole molecules). By using an all-vs-all least squares
superposition of the built fragments, we identify NCS relations. Typically,
fragments are built differently in different NCS-related copies. Thus the
identified NCS relations are used to extend and connect fragments in subsequent
building cycles.</p>

<h3>Results</h3>

<p class="abresults">The method has been successfully tested on several cases. In
a proof of principle 30% of residues of a homodimer (pdbID 1c48:A,B) were
artificially removed. Starting from this 70% complete model at 1.6Å resolution
ARP/wARP improved it to 78% (still highly fragmented) where only 50% of the
sequence could be docked. However, by the aid of the NCS-based fragment
extension, the completeness was improved to 96% and 93% of the sequence could
be docked already after two building cycles. We will also present results for a
large-scale test with data from our remote web-based MX computational facility.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The use of NCS provides a significant improvement in many
cases, often in less model-building cycles. Since 50% of all structures contain
NCS, the presented development may become an important contribution in the
automated derivation of 3D structures from low-resolution MX data. The
possibility to determine more structures in an automated manner would be a
valuable asset to biomedical and pharmaceutical research.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.embl-hamburg.de/research/unit/lamzin/index.html">http://www.embl-hamburg.de/research/unit/lamzin/index.html</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tim Wiegels (<a href="/Rebholz-srv/eccb2010/../../Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60342 = 'w&#105;&#101;g&#101;ls' + '&#64;';
 addy60342 = addy60342 + '&#101;mbl-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy60342 + suffix + '\'' + attribs + '>' );
 document.write( addy60342 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39821 = 'w&#105;&#101;g&#101;ls' + '&#64;';
 addy39821 = addy39821 + '&#101;mbl-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy39821 + suffix + '\'' + attribs + '>' );
 document.write( addy39821 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">EMBL, Hamburg Outstation</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) European Molecular Biology Laboratory (EMBL),
Hamburg Outstation</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">EMBL International PhD Programme (EIPP) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743019">C-11. Pattern recognition with moment invariants
for interpretation of very low resolution macromolecular density maps</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Heuser P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Heuser P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, Lamzin V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lamzin V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lamzin VS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">3D structural studies of macromolecular complexes often
yield data to only very low resolution (cryo EM/X-ray Crystallography). The
interpretation of such data usually starts with the segmentation of the map
(e.g., with Watershed algorithm), which does not always give satisfactory
results (over-segmentation/incorrect structural borders). Overall, fitting of
known structures currently requires a lot of human expert knowledge and
interaction so that an automated procedure is a highly desirable.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use 3rd order moment invariants to identify
regions in density maps of macromolecular complexes that match the
corresponding regions in the known structures of the constituting fragments.
False positives are eliminated using difference distance matrices. Finally the
structures of the fragments are placed into the map. Third order moment
invariants give a concise but comprehensive description of 3D objects in only
11 numerical values, providing convenient means for fast search through a large
amount of 3D data.</p>

<h3>Results</h3>

<p class="abresults">The method has been tested on calculated structure factors
for large macromolecular complexes (genotoxin and GroEL) with 10 or 15 Å
high-resolution limit. The individual subunits were fitted in the
low-resolution density maps with an average r.m.s.d. on Cα atoms of 1.3 Å. New
results obtained for experimental EM data will also be presented.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Since the last decade there have been many attempts to
develop reliable 3D map segmentation algorithms with varying success, in order
to reduce the complexity of the challenging task of low-resolution density map
interpretation. The method presented here does not require a map segmentation
step and provides accurate results without human interaction in reasonable
time, due to the use of sophisticated pattern recognition algorithms.
Implementation of real-space refinement procedures is expected to improve the
results even further.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.embl-hamburg.de/research/unit/lamzin/index.html">http://www.embl-hamburg.de/research/unit/lamzin/index.html</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Philipp Heuser (<a href="/Rebholz-srv/eccb2010/../../Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56368 = 'ph&#105;l&#105;pp.h&#101;&#117;s&#101;r' + '&#64;';
 addy56368 = addy56368 + '&#101;mbl-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy56368 + suffix + '\'' + attribs + '>' );
 document.write( addy56368 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6264 = 'ph&#105;l&#105;pp.h&#101;&#117;s&#101;r' + '&#64;';
 addy6264 = addy6264 + '&#101;mbl-h&#97;mb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy6264 + suffix + '\'' + attribs + '>' );
 document.write( addy6264 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">EMBL-Hamburg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">EMBL-Hamburg</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">EMBL for a postdoctoral fellowship</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743020">C-12. PTools: an open source molecular docking
library</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Poulain P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Poulain P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Saladin A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Saladin A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Fiorucci S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fiorucci S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Zacharias M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zacharias M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Prévost C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Prévost C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5)</p>

<p class="abmotivation">Most biological processes in the cell involve
macromolecules interacting with one or several partners. Knowledge of the
overall structures of these assemblies as well as the details of the
interactions is essential for understanding the underlying biological
mechanisms. Macromolecular docking is a challenging field of bioinformatics
that tackles this question. PTools is an open source library dedicated to
molecular docking simulations.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">PTools is an object-oriented Python/C++ library
that contains low-level routines like PDB-format manipulation functions as well
as high-level tools to perform docking simulations and analyze results. PTools
deals with reduced (coarse-grained) representation for proteins and DNAs
allowing simulations with large size biomolecules. The development framework
relies on the collaborative server launchpad.net together with the distributed
version control system bazaar.</p>

<h3>Results</h3>

<p class="abresults">ATTRACT is the rigid-body docking program based on PTools.
It shows good results with protein-protein and protein-DNA complexes as well as
3-body protein docking. We recently studied the RecA nucleofilament formed on
single-stranded DNA (ssDNA) and involved in the homologous recombination of
prokaryotes. Docking simulations of double-stranded DNA (dsDNA) on RecA
indicate that highly curved dsDNA can access the RecA-bound ssDNA while
initially retaining its Watson-Crick pairing.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The PTools library is freely available under the GNU GPL
license, together with detailed documentation and a step-by-step tutorial. In
addition to its ease of use, PTools and its docking program ATTRACT are
efficient tools to study large and heterogeneous biomolecular complexes. Our
next developments aim to embed the partner flexibility during the docking
process and to enhance the multicomponent ability.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://ptoolsdocking.sourceforge.net/" mce_href="http://ptoolsdocking.sourceforge.net/">http://ptoolsdocking.sourceforge.net/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pierre Poulain (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42886 = 'p&#105;&#101;rr&#101;.p&#111;&#117;l&#97;&#105;n' + '&#64;';
 addy42886 = addy42886 + '&#117;n&#105;v-p&#97;r&#105;s-d&#105;d&#101;r&#111;t' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy42886 + suffix + '\'' + attribs + '>' );
 document.write( addy42886 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Univ. Paris Diderot - Paris 7 &amp; Inserm U665</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) DSIMB, Inserm UMR-S 665 et Université Paris Diderot
- Paris 7, INTS, 6 rue Alexandre Cabanel, 75015 Paris, France. (2) MTI, Inserm
UMR-M 973, Université Paris Diderot-Paris 7, Bâtiment Lamarck, 35 rue Hélène
Brion, 75205 Paris Cedex 13, France. (3) LCMBA, UMR-CNRS UNSA 6001, Faculté des
Sciences, Université de Nice-Sophia Antipolis, 06108 Nice Cedex 2, France. (4)
Physik-Department (T38), Technische Universität München, James-Franck-Str. 1,
85748 Garching, Germany. (5) Laboratoire de Biochimie Théorique - UPR 9080
CNRS, Institut de Biologie Physico-Chimique, 13 rue Pierre et Marie Curie,
F-75005 Paris, France.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is funding by the Paris Diderot-Paris 7
University; the National Institute for Blood Transfusion (INTS); the Institute
for Health and Medical Care (Inserm); the National Center for Scientific
Research (CNRS); research grant ANR-07-CIS7-003-01. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743021">C-13. Development of a structural alignment tool
for protein local surfaces</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Minai R<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Minai R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Horiike T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Horiike T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Small molecule drugs generally bind to protein surface
regions that have geometrical and physicochemical properties suitable for the
structure of each molecule. It is therefore possible that nonhomologous
proteins with different overall three-dimensional folds could bind an identical
small molecule if they have similar surface regions. Since these cause the
cross-activity of drugs, to detect similar local surface regions would be
valuable in drug discovery. Here, we have developed an alignment tool that can
compare the entire local surface of proteins and identify similar regions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">This tool uses a feature vector to define the
geometrical and physicochemical properties of solvent-accessible atoms of a
protein. The physicochemical property is defined 6 types (cation, anion,
hydrogen-bond donor, hydrogen-bond acceptor, hydrophobic, and none of these
types) according to the PATTY (programmable atom typer) algorithm. The feature
vectors are used for the coordinate representation, the sampling optimization
(together with geometric hashing), and the similarity scoring of local surface
areas. Similar regions are obtained by single linkage clustering of local
similar areas.</p>

<h3>Results</h3>

<p class="abresults">The proposed method runs on a personal computer with the
input of a pair of PDB files and outputs similarity scores and superposed
coordinates. It was tested on several protein pairs that have the same function
with different folds, for instance, phosphoenolpyruvate carboxykinase (PDB
code: 1k3c, SCOP fold: PEP carboxykinase-like) and ABC transporter (1oxu,
P-loop containing nucleoside triphosphate hydrolases), and trypsin (1tpo,
Trypsin-like serine proteases) and subtilisin (2prk, Subtilisin-like). As a
result, the function sites (e.g., the ATP-binding site) were detected as
similar regions.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This tool generates not only a similarity score but also
a statistical score (Z-score) that describes the degree of similarity between
the identified similar regions. It can thus be used for the efficient
comparison of similar regions. A high Z-score suggests a high probability that
the identified regions are important similar regions. In practice, the number
of regions having high Z-scores (≥4) obtained in the test calculations is
relatively small (approximately ten).</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://d-search.atnifty.com/research.html" mce_href="http://d-search.atnifty.com/research.html">http://d-search.atnifty.com/research.html</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ryoichi Minai (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90548 = 'drm&#105;n&#97;&#105;' + '&#64;';
 addy90548 = addy90548 + '&#105;pc' + '&#46;' + 'sh&#105;z&#117;&#111;k&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy90548 + suffix + '\'' + attribs + '>' );
 document.write( addy90548 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Graduate School of Science and Technology, Shizuoka
University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Graduate School of Science and Technology, Shizuoka
University</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743022">C-14. Blender Game Engine as a tool to navigate the
conformational space of proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Zini M-F<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Zini M-F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Andrei R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andrei R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Loni T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Loni T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Zoppè M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zoppè M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Proteins and other macromolecules typically exert their
function through motion. Glimpse of these movements can be acquired, e.g., by
NMR spectroscopy: this information is delivered in pdb files containing many
models of the same molecule. We reasoned that if a protein can acquire many conformations,
there must be a series of intermediates connecting them. The tools developed
for computer graphics (CG) are now powerful enough to be used in complex
elaborations such as those involved in molecular motion, which comprise many
1000s atoms. Blender is a CG program open-source and freely available.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Blender is a complete package for 3D computer
graphics and video games. The program is open source and can be scripted in
Python. We prepared scripts to read pdb files, introduce atoms and bonds as
specific constraints, record the motion elaborated by the game engine (which
also considers collisions), and export all steps of motions as new pdb files. We
also use Swiss-PdbViewer and Gromos force filed to evaluate and (eventually)
adjust Blender-calculated conformations. All our scripts and programs will be
made available after publication, and distributed along with the version 2.5 of
Blender</p>

<h3>Results</h3>

<p class="abresults">With Calmodulin as case study, we report that using Blender
we can generate new conformations that smoothly transit the protein between
different models of the NMR file. The intermediate models are calculated in few
seconds, and are very quickly adjusted according to Gromos (see Methods) with
few minimization cycles. The procedure consists of a reiterative series of
steps to find conformations along a single path. By repeating the procedure
with different starting points, the navigation map is generated. The resulting
series of models is used to observe the protein in motion (v. poster Andrei)</p>

<h3>Discussion</h3>

<p class="abdiscussion">The availability of different conformations for the same
macromolecule, opens the possibility of studying transition between the various
forms. The use of computer graphics tools conbines the advantages of their
flexibility and speed with the programmability, so that features can be
introduced and handled by the game engine. We humans are especially equipped to
grasp concepts and extract infomration through visual exposure, the possibility
of animating molecules, and representing them in photorealistic way (see poster
Andrei et al) is likely to facilitate comprehension of protein functioning.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.scivis.ifc.cnr.it" mce_href="http://www.scivis.ifc.cnr.it">http://www.scivis.ifc.cnr.it</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Monica Zoppè (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41106 = 'mz&#111;pp&#101;' + '&#64;';
 addy41106 = addy41106 + '&#105;fc' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy41106 + suffix + '\'' + attribs + '>' );
 document.write( addy41106 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13115 = 'mz&#111;pp&#101;' + '&#64;';
 addy13115 = addy13115 + '&#105;fc' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy13115 + suffix + '\'' + attribs + '>' );
 document.write( addy13115 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58933 = 'mz&#111;pp&#101;' + '&#64;';
 addy58933 = addy58933 + '&#105;fc' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy58933 + suffix + '\'' + attribs + '>' );
 document.write( addy58933 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Insitute of Clinical Physiology. CNR</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Scientific Visualization Unit, IFC - CNR, Pisa. Italy
2 Lab of Molecular Biology, Scuola Normale Superiore, Pisa. Italy 3 Big Bang
Solutions, Navacchio (Pi). Italy</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">work funded by the Regione Toscana (Italy) grant '3d
animation' to MZ. RA supported by Scuola Normale Superiore Pisa. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743023">C-15. Using data mining to identify structural
rules in proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Stelle D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Stelle D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Scott LPB<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Scott LPB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Barioni MC<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Barioni MC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Proteins are, among others, the macromolecules that
perform all important task in organism. The three-dimensional structure
(tertiary structure) of proteins determines their function it is necessary and
important for the specific binding of substrates or other ligands. The number
of primary structures deposited in databases is growing faster than our ability
to solve the tertiary structures using experimental methods. Efficient
computational techniques can aid to predict the protein structures and they can
help to understand the folding process.Several works have explored this
subject.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We designed and implemented a local database (DB),
consisting around 20.000 protein extracted from the Protein Data Base (PDB).
The data base is composed by protein of five folding class. Are stored in the
DB information regarding the size and class of each protein, in addition to
amino acids (AA) that compose it, together with the physico-chemical properties
of each AA. The AA that compose these proteins were organized in windows of different
sizes (7, 11, 15, 21) in the DB, each window is associated with its structural
motif (this classification was obtained from the DDSP).</p>

<h3>Results</h3>

<p class="abresults">We can observe that the experiments with windows with length
7 and 15 obtained a great number of rules with positive lift.It indicated that
smaller structural motifs are more easy to be formed. This is a important
result that we are investigating with more details in this moment. We need to test
this result with a data base bigger with the other class folding. We can
observe that there are not rules with lift negative . Thus, our algorithm did
not detect sequences (in this data base version) that forbid a structure motif.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This is the first step and the first results of our
project and software to extract rules to secondary structures of proteins. We
observed that data mining is a effcient thechinique to extract this kind of
rule in PDB. We obtained three interesting results: the small structural motifs
are easier to be formed in proteins of α-helix folding class, there are a
bigger number of small sequences that converge to the same structural motif and
the algorithm could not detect similar sequences that forbid a specific motif.
We are inlcuding transmembrane protein in data base.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Luis Paulo Barbour Scott (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13429 = 'l&#117;&#105;sp37' + '&#64;';
 addy13429 = addy13429 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy13429 + suffix + '\'' + attribs + '>' );
 document.write( addy13429 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">UFABC - CMCC</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1-CMCC - Universidade Federal do ABC (UFABC), Brasil 2-Universidade
Federal do ABC (UFABC), Brasil</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Fapesp, UFABC</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743024">C-16. Detecting protein domains of biological
assemblies using TopDomain-web</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Senn S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Senn S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Sippl MJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sippl MJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Examining asymmetric units (ASUs) stored in the PDB
reveals that a lot of those ASUs don't hold enough information to fully
understand the protein structure. A frequent case are missing chains to
establish a homomer. In such a case it's important to gather all of the
possible structural information to assign domains to the protein. The
biological assembly (BA) defined in the PDB file can provide said structural
information. A lot of the frequently encountered problems with domains in the
protein classifications are based on missing structural information.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">TopDomain-web is a tool to analyze and decompose
different structural entities of proteins. It's an easy to use web application
that can handle even biological assemblies up to 6000 residues (99.8 % of all
BAs currently defined in the PDB). The application allows instant access to
TopMatch-web to perform structural alignments and the Classification of Protein
Structures (COPS) to examine quantified relations of the examined protein
structure. It is possible to upload files in PDB format.</p>

<h3>Results</h3>

<p class="abresults">Examination of the PDB shows that some ASUs do not offer
enough structural information to assign all domains of the proteins correctly. For
example the ASU of the PDB-file 3g0o is a single chain monomer. The biological
assembly is a homodimer which consists of 3 domains with a central domain
consisting of parts of both monomeric chains. The central domain can't be
assigned using only the ASU. We observe a gain of compactness for the central
dimer domain. We also observe that decomposing asymmetric units yields 7.4%
more domains per starting entity than decomposing biological assemblies</p>

<h3>Discussion</h3>

<p class="abdiscussion">Examining protein structures with TopDomain-web
illustrates the fact that using the correct molecule entity, from which to
derive a domain decomposition, is important to yield a stable and consistent
domain decomposition. A domain decomposition algorithm analyzing the
interaction of atoms in a protein structure will interpret missing parts of
structures as lack of interaction and deliver inconsequent results. The loss of
domains per entity for large scale automated domain decomposition can be
explained with rudimentary domains derived from molecules with insufficient
structural information.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://topdomain.services.came.sbg.ac.at" mce_href="http://topdomain.services.came.sbg.ac.at">http://topdomain.services.came.sbg.ac.at</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Stefan Senn (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44522 = 'st&#101;ff' + '&#64;';
 addy44522 = addy44522 + 'c&#97;m&#101;' + '&#46;' + 'sbg' + '&#46;' + '&#97;c' + '&#46;' + '&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy44522 + suffix + '\'' + attribs + '>' );
 document.write( addy44522 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28544 = 'st&#101;ff' + '&#64;';
 addy28544 = addy28544 + 'c&#97;m&#101;' + '&#46;' + 'sbg' + '&#46;' + '&#97;c' + '&#46;' + '&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy28544 + suffix + '\'' + attribs + '>' );
 document.write( addy28544 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy59025 = 'st&#101;ff' + '&#64;';
 addy59025 = addy59025 + 'c&#97;m&#101;' + '&#46;' + 'sbg' + '&#46;' + '&#97;c' + '&#46;' + '&#97;t';
 document.write( '<a ' + path + '\'' + prefix + addy59025 + suffix + '\'' + attribs + '>' );
 document.write( addy59025 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Division of Bioinformatics, Department of Molecular
Biology, University of Salzburg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Division of Bioinformatics, Department of Molecular
Biology, University of Salzburg. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743025">C-17. Metals in protein structures: classification
and functional prediction</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Andreini C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Andreini C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Cavallaro G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cavallaro G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The PDB contains many metal-binding structures reflecting
the importance that metals play in proteins. It was estimated that a large
fraction of proteins encoded by living systems are metalloproteins, and that
about 40% of all structurally characterized enzymes perform metal-dependent
reactions. Notwithstanding, bioinformatics resources devoted to the study of
metals in biological systems have been scarce so far, most likely due to the
difficulty to establish formal criteria to describe the exceptional variety of
metalloproteins. Our research is intended to fill this gap.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Metal sites present in PDB protein structures are
described as 3D templates composed of the metal(s) in the site and the residues
forming the first and the second coordination sphere of the metal(s). The sites
are grouped based on CATH and SCOP classifications, and the relevant literature
is examined on a per-group basis to annotate metal functions and to identify
non-physiological metals. Representative sites are selected for each group and
compared via structural alignment.</p>

<h3>Results</h3>

<p class="abresults">The method was applied to non-heme iron (Fe), copper (Cu)
and zinc (Zn) sites. It resulted in the definition of 86 groups of Fe sites, 35
of Cu sites, and 368 of Zn sites. The most common metal function was electron
transfer and catalytic for Fe (30% each), catalytic for Cu (ca. 50%), and
structural for Zn (55%). The comparison of sites taken from different groups
led to cluster together sites which, though present in unrelated proteins, are
similar and have the same function. The functionally annotated datasets of
metal sites were used to predict the function of structural genomics proteins.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The systematic description of metal sites as 3D templates
lends itself to perform several analyses that are useful to understand the
properties, functions and evolution of metalloproteins. They range from the
analysis of metal coordination to the investigation of the evolutionary
pathways of metal usage in proteins. Also, structural similarity searches
focused on metal sites can be advantageously combined with classical
whole-domain comparisons to predict protein function. Finally, this description
represents an ideal framework for the systematic classification of
metalloproteins in databases.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Claudia Andreini (<a href="/Rebholz-srv/eccb2010/../../Documents%20and%20Settings/pkonings/Application%20Data/Microsoft/Word/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60656 = '&#97;ndr&#101;&#105;n&#105;' + '&#64;';
 addy60656 = addy60656 + 'c&#101;rm' + '&#46;' + '&#117;n&#105;f&#105;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy60656 + suffix + '\'' + attribs + '>' );
 document.write( addy60656 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22016 = '&#97;ndr&#101;&#105;n&#105;' + '&#64;';
 addy22016 = addy22016 + 'c&#101;rm' + '&#46;' + '&#117;n&#105;f&#105;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy22016 + suffix + '\'' + attribs + '>' );
 document.write( addy22016 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Florence</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. CERM - University of Florence (Italy)</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Italian Ministry of University and Research (MIUR)
through the project FIRB "Futuro in Ricerca"</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743026">C-18. Intuitive visualization of surface properties
of moving proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Andrei R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Andrei R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Callieri M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Callieri M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Zini M-F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zini M-F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Loni T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Loni T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,4), Zoppè M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zoppè M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Surface properties of macromolecules, such as
Electrostatic Potential and hydropathy are important features that determine their
interaction with the medium or other macromolecules. Proteins are usually in
continuous movement, changing shape as they interact with ligands and other
molecules, therefore their surface characteristics change in time. A method
that permits the visualization of moving molecules simultaneously with their
surface properties should facilitate molecular behavior.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A harmonious combination of chemico-physical
programs, computer graphics software and custom scripts and programs was used
for calculating and graphically representing the surface properties of proteins
and glycoproteins. Electrostatic Potential and Molecular Lipophilic Potential
are visualized on the molecular surface, taking advantage of a free,
open-source software, Blender. For their representation we combine computer
graphics features such as materials, textures, lights to obtain a
photorealistic rendering that give substance and consistency impression.</p>

<h3>Results</h3>

<p class="abresults">EP, calculated using APBS, is visualized as animated line particles
flowing along field lines from positive to negative, respecting the convention
in physics. A novel code is introduced for the MLP visualization: a range of
optical features that goes from dark-dull-rough surfaces for the most
hydrophilic areas to bright-shiny-smooth surfaces for the most hydrophobic
ones. MLP calculation is done using Testa formula based on atomic properties:
this permits a spatial distribution of hydropathy more accurate than the
well-known Kyte-Doolittle values.</p>

<h3>Discussion</h3>

<p class="abdiscussion">EP and MLP are shown simultaneously for moving proteins
avoiding the use of color, which cannot be interpreted without a legend. Using
real world tactile/sight feelings, the nanoscale world of proteins becomes more
understandable, familiar to our everyday life, making it easier to introduce
“un-seen” phenomena (concepts) such as hydropathy or charges, while leaving the
utilization of color space for the description of other biochemical
information. The proteins motion is obtained using the method described in Zini
et al.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.scivis.ifc.cnr.it" mce_href="http://www.scivis.ifc.cnr.it">http://www.scivis.ifc.cnr.it</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Raluca M Andrei (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95874 = 'r.&#97;ndr&#101;&#105;' + '&#64;';
 addy95874 = addy95874 + 'sns' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy95874 + suffix + '\'' + attribs + '>' );
 document.write( addy95874 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58183 = 'r.&#97;ndr&#101;&#105;' + '&#64;';
 addy58183 = addy58183 + 'sns' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy58183 + suffix + '\'' + attribs + '>' );
 document.write( addy58183 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89018 = 'r.&#97;ndr&#101;&#105;' + '&#64;';
 addy89018 = addy89018 + 'sns' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy89018 + suffix + '\'' + attribs + '>' );
 document.write( addy89018 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Scuola Normale Superiore</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Scientific Visualization Unit, IFC - CNR, Pisa. Italy
2 Lab of Molecular Biology, Scuola Normale Superiore, Pisa. Italy 3 Visual
Computing Lab, ISTI - CNR, Pisa. Italy 4 Big Bang Solutions, Navacchio (Pi).
Italy</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Funded by the Regione Toscana (Italy) grant '3d
animation' to MZ. RA supported by Scuola Normale Superiore Pisa. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743027">C-19. Hierarchical classification of helical
distortions related to proline</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rey J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Rey J&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]-->, Devillé J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Devillé J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Chabbert M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chabbert M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*)</p>

<p class="abmotivation">The presence of proline in a helix may be accommodated by
several types of distortions. Tools able to predict these distortions would be
of great help in molecular modeling. To develop such tools, a systematic
classification of proline-related helical distortions is necessary. We have previously
shown that helical distortions can be described in terms of structural motifs
in which two helices are joined by a few linker residues (Devillé et al., 2008;
Rey et al., 2010). This description provides a clue for data mining and can be
used to develop a classification method.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We analyzed a representative subset of the Protein
Data Bank including 4323 protein chains and 26380 helices. Using DSSP, we
searched this subset for both contiguous helices and helical HXnH motifs containing
proline. Helical HXnH motifs consist of two helices joined by n residues whose
dihedral angles were located in the generous alpha conformation. These motifs
were classified as a function of the number of linker residues, of their
dihedral angles and of the proline position. Analyses were carried out using
home-developed scripts.</p>

<h3>Results</h3>

<p class="abresults">“Typical” or “non-typical” distortions correspond to proline
in contiguous helices and in helical HXnH motifs. They can be differentiated by
DSSP. “Non-typical” distortions can be further classified as a function of the
dihedral angles of the linker residues. This classification is equivalent to
classification based on main chain – main chain H-bonds and differentiates pi
bulges and tight turns. Sub-division of bulges and turns is based on the number
of linker residues and on the position of proline in the second helix. Using
these parameters, about 85% of “non-typical” proline distortions are described
by only five canonical structures.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed a hierarchical method that allows
classifying proline distortions into a limited number of canonical structures
in three steps. The first step is based on DSSP whose algorithm yields a robust
evaluation of H-bonds. The following steps are based on discrete criteria,
resulting in an efficient classification scheme. This one can be easily
implemented on a large scale and will help to develop predictive tools for
molecular modeling.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Marie Chabbert (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41409 = 'm&#97;r&#105;&#101;.ch&#97;bb&#101;rt' + '&#64;';
 addy41409 = addy41409 + '&#117;n&#105;v-&#97;ng&#101;rs' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy41409 + suffix + '\'' + attribs + '>' );
 document.write( addy41409 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">CNRS UMR 6214 - INSERM U771</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">CNRS UMR 6214 - INSERM U771</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We thank NEC Computers Services SARL (Angers, France)
for the kind provision of a multiprocessor server. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743028">C-20. A “smooth” knowledge based potential for
protein structure refinement</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Røgen P<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Røgen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*(1), Koehl P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Koehl P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2)</p>

<p class="abmotivation">The accuracy of a predicted protein structure is
essential for its biological usefulness. It is first within the last few years
that positive results on structure refinement have been reported in the
literature. Phenomenological, it seems that the local roughness of potential
energy functions causes a refinement algorithm to go off in a random direction
rather than bringing the model closer to the native structure. To enable the
use of gradient based refinement methods the potential is modeled using splines
and is smooth except for the salvation term which is only piecewise smooth.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our carbon-alpha knowledge based potential contains
3 different types of terms: A local term, which for each 7-mer in the quarry
sequence expresses its structural preferences and flexibility as extracted from
a set of known 7-mer structures. The local term mainly expresses the shortest distance
to known configurations. A non-local pair potential. And finally solvent
effects depending on solvent accessibility and occupied volume. We present a
novel method for constructing a knowledge based potential by minimizing the
roughness of minima around native structures using 400 models for each of 1326</p>

<h3>Results</h3>

<p class="abresults">Comparative modeling techniques can give relatively good
models if a homologous protein has known structure. In such cases our potential
and the RMSD to the native structure have a correlation of 0.8 to 0.9 which
seems promising for structure refinement. This very high correlation stems from
a sampling of the local 7-mer geometry of known structures. In the range of
free modeling where no prior data is available the average energy-RMSD correlation
drops to 0.72 but stays in the range of 0.8-0.9 for most folds.</p>

<h3>Discussion</h3>

<p class="abdiscussion">It is well known that a knowledge based potential based
on decoy structures have a tendency to learn this specific type of decoys.
Hence our method needs testing on other decoy sets but mostly when used for
structure refinement. In the absence of prior knowledge the performance often
remains good. It’s an interesting problem to determine why and when the
performance occasionally is poor. The functional form of the potential makes it
highly suited for threading which in an interesting application to pursue.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Peter Røgen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy20526 = 'P&#101;t&#101;r.R&#111;&#101;g&#101;n' + '&#64;';
 addy20526 = addy20526 + 'm&#97;t' + '&#46;' + 'dt&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy20526 + suffix + '\'' + attribs + '>' );
 document.write( addy20526 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32684 = 'P&#101;t&#101;r.R&#111;&#101;g&#101;n' + '&#64;';
 addy32684 = addy32684 + 'm&#97;t' + '&#46;' + 'dt&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy32684 + suffix + '\'' + attribs + '>' );
 document.write( addy32684 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35731 = 'P&#101;t&#101;r.R&#111;&#101;g&#101;n' + '&#64;';
 addy35731 = addy35731 + 'm&#97;t' + '&#46;' + 'dt&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy35731 + suffix + '\'' + attribs + '>' );
 document.write( addy35731 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Technical University of Denmark Departmant of
Mathematics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Technical University of Denmark, Department of
Mathematics (2) Department of Computer Science, Genome Center, University of
California Davis, USA</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743029">C-21. PROTEIN PEELING in 2010: recent developments
and applications</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gelly J-C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gelly J-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), de Brevern A G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;de Brevern A G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">To better understand the architecture and anatomy of
proteins we developed Protein Peeling web server to sub-divide a protein
structure in smaller sub-units. These elements called Protein Units (PUs) allow
analysis of protein structure organization in simple and detailed description.
We propose here a new version of Protein Peeling web server incorporating new
progress: unstructured terminal segments recognition, novel scoring function
for PUs characterization and lastly structural domains identification. PUs can
be used to study protein structure folding, stability or evolution.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Based on a new refined non-redundant protein
structure databank, we have defined a new assignment method to identify
unstructured N or C termini segments: These PUs are identified as unstructured
if they are isolated at the first cutting event and no more thereafter.
Additionally, a characterization of PU by pseudo-energetic criterion based on
statistical potentials computation on carbon alpha is proposed. Lastly a new
bottom up algorithm called Domain Reconstruction (DR) provides domain
identification. Results have been compared to classical protein domain
benchmark datasets.</p>

<h3>Results</h3>

<p class="abresults">Protein Peeling method had been implemented in a webserver.
It takes a protein structure (PDB format) and gives cutting events of PU using
various visual outputs: cutting dendrogram with the different generated PUs,
secondary structure contents, visual representation of PUs. An update of the
interface that provides tools for viewing and detailed new analyses was
developed. Hence, unstructured N or C termini segments, the novel
characterization of PU by pseudo-energetic criterion and domain identification
are all available. Some practical cases are shown.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Protein structures are often described as series of
alpha-helices and beta-sheets, or at a higher level as an arrangement of
protein domains. We have proposed an intermediate view, the Protein Units
(PUs). They are novel level of protein structure description between secondary
structures and domains. PUs are linear and compact sub-region of the structure
defined by a high number of intra-PU contacts and a low number of inter-PU
contacts. Our new developments can help in defining (i) mobile extremities,
(ii) pertinent energetic assessment of PUs, and (iii) given a new view of
Protein Domains.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.dsimb.inserm.fr/dsimb_tools/peeling3" mce_href="http://www.dsimb.inserm.fr/dsimb_tools/peeling3">http://www.dsimb.inserm.fr/dsimb_tools/peeling3</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jean-Christophe Gelly (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17891 = 'j&#101;&#97;n-chr&#105;st&#111;ph&#101;.g&#101;lly' + '&#64;';
 addy17891 = addy17891 + '&#117;n&#105;v-p&#97;r&#105;s-d&#105;d&#101;r&#111;t' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy17891 + suffix + '\'' + attribs + '>' );
 document.write( addy17891 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">DSIMB, Inserm UMR-S 665 and Université Paris Diderot
– Paris 7, INTS, Paris, France</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">DSIMB, Inserm UMR_S 665 and Université Paris Diderot –
Paris 7, INTS, Paris, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Financial support was provided by grants from the
French Ministry of Research, Paris Diderot - Paris 7 University, the National
Institute for Blood Transfusion (INTS), the Institute for Health and Medical
Research (INSERM) and Parternariat Hubert Curien (PHC) Orchid. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743030">C-22. Local moves for efficient sampling of protein
conformational space</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bottaro S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bottaro S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*),Boomsma W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Boomsma W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1) ,Enøe Johansson K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Enøe Johansson K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2) , Andreetta C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andreetta C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2 ), Hamelryck T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hamelryck T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ferkinghoff-Borg J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ferkinghoff-Borg J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">During their function proteins often undergo local
conformational rearrangements. Standard molecular dynamics simulations cannot
address the typical time scale at which these processes occur (10^-6 - 10^-3
s). Monte Carlo simulation is an alternative methodology to compute pathways
and thermodynamic properties of proteins. To probe the conformational space
around the densely packed native state a method to perform local deformations
of protein backbone is required</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We divide the local move into two steps. During the
first step new angles are proposed for a small segment of the chain,
introducing a break at the end of the segment. In the second step are found the
changes in the remaining degrees of freedom of the segment in order to return
to a closed state. We found an analytical solution for the second step (loop or
chain closure), which is used, in turn, to derive an efficient strategy to
ensure the success the chain closure. The move is used in Monte Carlo
simulations of proteins using OPLSaa potential in combination with the GB/SA
solvation model.</p>

<h3>Results</h3>

<p class="abresults">We verify the formal correctness of the method, showing the
geometrical validity of the approach. A comparison with the existing state of
art methodology is performed , and an improvement of a factor 30 in simulation
efficiency was found. A set of Monte Carlo runs on ubiquitin were able to
reproduce with statistical significance data obtained from NMR experiments.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results obtained suggest that the move is suitable
for establishing a correspondence between MC trials and real molecular time
scales, thus providing dynamic information within a MC framework. Although our
research was focused on MC simulations, it should be noted that this method may
be useful for any application where the resampling of protein conformations is
needed, such as loop modeling or structure refinement.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.phaistos.org/" mce_href="http://www.phaistos.org/">http://www.phaistos.org/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sandro Bottaro (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40449 = 'sb&#111;' + '&#64;';
 addy40449 = addy40449 + '&#101;l&#101;ktr&#111;' + '&#46;' + 'dt&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy40449 + suffix + '\'' + attribs + '>' );
 document.write( addy40449 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19529 = 'sb&#111;' + '&#64;';
 addy19529 = addy19529 + '&#101;l&#101;ktr&#111;' + '&#46;' + 'dt&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy19529 + suffix + '\'' + attribs + '>' );
 document.write( addy19529 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51230 = 'sb&#111;' + '&#64;';
 addy51230 = addy51230 + '&#101;l&#101;ktr&#111;' + '&#46;' + 'dt&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy51230 + suffix + '\'' + attribs + '>' );
 document.write( addy51230 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Techincal University of Denmark - DTU Elektro</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) DTU Elektro, Technical University of Denmark, 2800
Lyngby, Denmark, and (2) Bioinformatics Centre, Department of Biology,
University of Copenhagen, Ole Maaloes Vej 5, 2200 Copenhagen N, Denmark</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">AS Radiometer</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743031">C-23. Self-organising maps applied to protein
structure classification</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Alanis-Lobato G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alanis-Lobato G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*)</p>

<p class="abmotivation">Protein Structure Classification is very important for
pharmacists, medical doctors, biologists, chemists, and others. Because of
this, several databases, methods and algorithms are devoted to the organisation
of protein structures according to its architecture, functionality, and other
features. These tools normally require a lot of manual input to classify which
makes the task tedious. The main motivation of this work is to ease the
classification of protein structures by making use of the unsupervised neural
network called SOM, which only requires the protein PDB or its amino acid
sequence</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The sample set for the SOM (based on 8192 representative
proteins from the SCOP classification) was generated using BioPerl which
obtains the amino acid sequence of proteins from PDB files. Once obtained, the
sequences were represented with 3 different type of vectors, taking into
account amino acid properties such as solubility and volume (linked to the
hydrophobic and steric effects respectively), and position in the sequence. The
Matlab’s SOM Toolbox was used to apply the algorithm and obtain visual results.</p>

<h3>Results</h3>

<p class="abresults">Different SOM sizes were used to cluster the 8192 sample
proteins. All validations were made by comparing proteins organised in one
group by the SOM with the classes proposed by SCOP. Interestingly, no matter
the SOM size, around 6 clusters were generated, which pretty much matched the
SCOP classes (all alpha, all beta, a/b, a+b, multi-domain, and membrane
proteins). There were, of course, some not very large extra clusters whose
proteins could represent either some of the other SCOP classes (like coiled
coil, peptides, or designed proteins) or totally new categories.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results from this work have shown that SOMs are very
powerful tools to cluster proteins with similar structures starting from
something as simple as the vector representation of their amino acid sequence. Once
trained, SOMs are able to cluster a new protein in a group with similar
characteristics. This can be exploited to predict functionality of proteins,
given the relation structure-function that holds for lots of them. Since this
prediction might be unreliable given the simplicity of the vector
representations, SOMs can be part of preprocessing steps in more powerful
predicting tools.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Gregorio Alanis-Lobato (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13569 = 'gr&#101;g&#111;r&#105;&#111;.&#97;l&#97;n&#105;sl&#111;b&#97;t&#111;' + '&#64;';
 addy13569 = addy13569 + 'k&#97;&#117;st' + '&#46;' + '&#101;d&#117;' + '&#46;' + 's&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy13569 + suffix + '\'' + attribs + '>' );
 document.write( addy13569 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>%5C">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44470 = 'gr&#101;g&#111;r&#105;&#111;.&#97;l&#97;n&#105;sl&#111;b&#97;t&#111;' + '&#64;';
 addy44470 = addy44470 + 'k&#97;&#117;st' + '&#46;' + '&#101;d&#117;' + '&#46;' + 's&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy44470 + suffix + '\'' + attribs + '>' );
 document.write( addy44470 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">King Abdullah University of Science and Technology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Mathematical and Computer Sciences and Engineering
Division, King Abdullah University of Science and Technology, Thuwal, Saudi
Arabia (2) ESCOM - National Polytechnic Institute, Mexico City, Mexico</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">King Abdullah University of Science and Technology</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743032">C-24. Construction of a sequence- and
backbone-dependent rotamer library by hidden Markov model</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wen W<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Wen W&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]-->(1),Lv Q<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lv Q&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,2,*),Yang P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yang P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1),Yang L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yang L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->-Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yang L-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1),Wu J-Z<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wu J-Z&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1),Huang X<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Huang X&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">However, the state-of-the-art rotamer libraries focus on
the statistical information of individual amino acids, ignoring the direct
affection of its adjacent amino acids. This article presents a sequence- and
backbone-dependent rotamer library. Both the conformation information of
adjacent amino acids and torsion angle of the current residue are taken into
account to construct a sequence- and backbone-dependent library by HMM.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The sequence- and backbone-dependent model
considers not only amino acids sequence information but also backbone dihedral
angles as observed data. The model is trained to learn the relationships among
the observed data and side-chain conformation. After being trained based on
comparative training data, this model can produce the specific rotamer library
given the target amino acids sequence by sampling the two trained models.</p>

<h3>Results</h3>

<p class="abresults">Comparing to rotamer library produced by the other popular
side-chain modeling methods, ours is more close to native conformation.We find
that the prediction accuracy outperforms on all the test targets to a certain
extent comparing with that based on traditional backbone dependent rotamer
libraries.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This thesis verifies that the sequence- and
backbone-dependent model will enable fully taking into account the possible
affection of sequence amino acids to the specific amino acid side-chain. The
rotamer libraries generated by sequence- and backbone-dependent models provide
solid support to protein side-chain prediction.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Qiang Lv (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41416 = 'q&#105;&#97;ng' + '&#64;';
 addy41416 = addy41416 + 's&#117;d&#97;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'cn';
 document.write( '<a ' + path + '\'' + prefix + addy41416 + suffix + '\'' + attribs + '>' );
 document.write( addy41416 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy68322 = 'q&#105;&#97;ng' + '&#64;';
 addy68322 = addy68322 + 's&#117;d&#97;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'cn';
 document.write( '<a ' + path + '\'' + prefix + addy68322 + suffix + '\'' + attribs + '>' );
 document.write( addy68322 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96373 = 'q&#105;&#97;ng' + '&#64;';
 addy96373 = addy96373 + 's&#117;d&#97;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'cn';
 document.write( '<a ' + path + '\'' + prefix + addy96373 + suffix + '\'' + attribs + '>' );
 document.write( addy96373 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">School of Computer Science and Technology, Soochow
University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 School of Computer Science and Technology, Soochow
University, Suzhou, 215006, China 2 Jiangsu Provincial Key Lab for Computer
Information Processing Technology, Suzhou, 215006, China</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by National Natural Science
Foundation of China under the grant number 60970055. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743033">C-25. Codon usage and protein structure in
prokaryotes: old stories, new findings and puzzling questions</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cozzetto D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Cozzetto D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Ward S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ward S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Jones DT<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jones DT&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The community has long discussed whether synonymous codon
usage and protein secondary structure are related or not, by supporting or
refuting it based on diverse datasets and methodologies. Early work suggested
that short segments of more/less common codons strongly correlates with the
observation of helices and strands respectively in E. coli highly expressed
genes. Though challenging this view, subsequent studies did not clearly
disprove it, thus leaving the question still unanswered.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We collected 408 E. coli globular single domain
protein structures that are confidently and completely mapped to their coding
sequences. We used codon frequencies as a proxy for their respective tRNA
abundance (and hence translation speeds) and calculated their sums for all
tricodons – segments of 3 consecutive codons. We used chi-squared tests to
compare the distributions of observed tricodons with those simulated by 250
independent random samples of the codons associated with individual amino acids
in our dataset. We partly extended this study to 1024 completely sequenced
bacterial genomes.</p>

<h3>Results</h3>

<p class="abresults">i) Coding sequences include tricodons that are of much
higher frequency than expected by chance; ii) In the set of lowest frequency
tricodons,, helices are significantly underrepresented but strands are not
overrepresented. Highly expressed genes show no significant bias; iii) Strands
are significantly overrepresented in the set of highest frequency tricodons,
while helices are significantly underrepresented. Similar results hold for
highly expressed genes, too; iv) Analysis of genome-wide data confirms these
results and highlights evolutionary conservation of codon usage patterns.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Protein secondary structure does not correlate with codon
usage, once the structural biases of the amino acids is considered. Other
protein-wide properties like contact order, expression level, and overall
secondary structure do not explain the variance in codon usage frequencies in
and across prokaryotes. Yet, codon frequencies capture an important signal,
being conserved at significant mutational distances and in a number of
different folds. Additional work is needed to unveil it, and small (10-30 aa)
fragment folding kinetics looks promising to explain differences at the fold
level.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Domenico Cozzetto (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97629 = 'd.c&#111;zz&#101;tt&#111;' + '&#64;';
 addy97629 = addy97629 + 'cs' + '&#46;' + '&#117;cl' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy97629 + suffix + '\'' + attribs + '>' );
 document.write( addy97629 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Dept. of Computer Science - University College London</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Computer Science University College
London Gower Street, London, WC1E 6BT</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">EU-funded Marie Curie IEF Project ProtDNABindSpec
(PIEF-GA-2009-237292); BBRSC</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743034">C-26. Analysis of FTICR mass spectra with H/D
exchange of deprotonated dinucleotides</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Claesen J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Claesen J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Valkenborg D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valkenborg D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Burzykowski T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Burzykowski T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Hydrogen/Deuterium exchange (HDX) combined with mass
spectrometry can be used to study the conformational speciation of small
biomolecules. By monitoring the level of deuterium incorporation over time,
information about the accessibility of hydrogens for the exchange in various
parts of the molecule is gathered. Determination of the number of exchanged
hydrogen atoms and their exchange rates allows drawing conclusions about the
conformation and dynamics of proteins.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We propose the following method to estimate the
exchange rates (l_1,..,l_n), of n hydrogens: 1)Read mass spectra for all time
points in 2)For spectrum k, determine the intensities of observed peaks
(O_k,1,...,O_k,n+p) 3)Determine the isotopic distribution of molecule M before
HDX has taken place (F_0) 4)For spectrum k, calculate the expected intensities
by means of summing the “shifts” of F_0, which occur with probabilities modeled
as a function of time t and exchange rates l_i 5) Estimate l’s by minimizing
the Residual Sum of Squares for the observed and expected intensities over all
spectra.</p>

<h3>Results</h3>

<p class="abresults">The proposed model and algorithm are evaluated on dinucleotides
dTT, dTG and dAA. -dTT: 4 protons are potentially available for HDX; according
to our model, 2 exchanged. -dTG: 6 protons are potentially available for HDX;
according to our model, all 6 exchanged, but one exchanged at a very slow rate.
-dAA: 6 protons are potentially available for HDX; all exchanged at the same
rate. These results are in accordance with the results reported by Balbeur et
al 2007. However, not all observed intensities can be explained by our model.
This might be due to the fact that some protons exchanged before</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our method allows predicting the number of HDXs of H
atoms and estimating exchange rates for the available protons of a dinucleotide
or protein.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jürgen Claesen (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17322 = 'j&#117;rg&#101;n.cl&#97;&#101;s&#101;n' + '&#64;';
 addy17322 = addy17322 + '&#117;h&#97;ss&#101;lt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy17322 + suffix + '\'' + attribs + '>' );
 document.write( addy17322 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Interuniversity Institute for Biostatistics and
statistical Bioinformatics, Universiteit Hasselt</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Interuniversity Institute for Biostatistics and
statistical Bioinformatics, Universiteit Hasselt, Belgium (2) VITO</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743035">C-27. Voroprot: an interactive tool for the
analysis of geometric features of protein structure</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Olechnovič K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Olechnovič K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Margelevičius M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Margelevičius M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Venclovas Č<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Venclovas Č&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The ability to analyze and visualize geometric features
of the three-dimensional protein structure is fundamental for studies of
protein folding, residue packing, docking, protein-protein interactions, etc.
Some of the advanced geometric data structures, like Apollonius diagram (also
known as additively weighted Voronoi diagram), make it possible to perform such
an analysis in a comprehensive manner. With this goal in mind, we have set out
to develop an interactive protein structure analysis and visualization tool.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In Voroprot we used our own algorithm
implementation to construct and display Apollonius diagram and Apollonius graph
of protein atoms, considering each atom to be a 3D ball of Van der Waals
radius. We subdivide a protein structure into quadruples of atoms, making sure
that no tangent sphere of any quadruple overlaps any atom. This subdivision is
used to find internal cavities and to construct Voronoi cells and various
surfaces for each atom. Voroprot is written in C++ using Qt library and OpenGL
API.</p>

<h3>Results</h3>

<p class="abresults">We developed Voroprot, an easy-to-use cross-platform
software tool, that provides a unique set of capabilities for exploring
geometric features of protein structure. Our tool concentrates on geometric
methods not included in other software packages used for protein structure
analysis. Using Apollonius diagram, Voroprot allows construction and
visualization of protein alpha shapes, atoms contact surfaces, solvent
accessible surfaces and cavities inside protein structure.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The use of additively weighted Voronoi diagram offers a
powerful approach in studies of protein structure. However, this geometric data
structure so far has been exploited by software developers only to a limited
extent. Voroprot is a significant advancement in effective utilization of
Apollonius diagram for protein structure analysis and visualization.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ibt.lt/en/laboratories/bioinfo_en/software/voroprot-2.html">http://www.ibt.lt/en/laboratories/bioinfo_en/software/voroprot-2.html</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Kliment Olechnovič (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy70179 = 'kl&#105;m&#101;nt' + '&#64;';
 addy70179 = addy70179 + '&#105;bt' + '&#46;' + 'lt';
 document.write( '<a ' + path + '\'' + prefix + addy70179 + suffix + '\'' + attribs + '>' );
 document.write( addy70179 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65961 = 'kl&#105;m&#101;nt' + '&#64;';
 addy65961 = addy65961 + '&#105;bt' + '&#46;' + 'lt';
 document.write( '<a ' + path + '\'' + prefix + addy65961 + suffix + '\'' + attribs + '>' );
 document.write( addy65961 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38824 = 'kl&#105;m&#101;nt' + '&#64;';
 addy38824 = addy38824 + '&#105;bt' + '&#46;' + 'lt';
 document.write( '<a ' + path + '\'' + prefix + addy38824 + suffix + '\'' + attribs + '>' );
 document.write( addy38824 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Biotechnology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Biotechnology, Vilnius, Lithuania</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Howard Hughes Medical
Institute grant to ČV. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection29">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743036">C-28. BBP - Beta-Barrel Predictor: a web server for
the prediction of the super-secondary structure of transmembrane β-barrel
proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tran V D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Tran V D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Chassignet P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chassignet P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Steyaert J-M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Steyaert J-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The transmembrane β-barrel (TMB) proteins perform diverse
important functions in cells. Computational structure prediction methods based
on learning are poorly tractable for these proteins since they are difficult to
observe by standard experimental techniques. Long range structure is also a
challenge. Generally, those structures are not only a series of β-strands where
each is bonded to the preceding and following ones in the primary sequence, but
they may contain Greek key or Jelly roll motifs as well. This may be described
as a permutation on the order of the bonded segments.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We model the protein folding problem with minimum
energy into the search of the longest closed path in a weighted graph with
respect to some given permutation. A trained probabilistic model has been
introduced and acts as a primary filter to identify substrings as potential
β-strands (vertices) or coils (edges). The weights are related to energy terms
of electrostatic, hydrophobic and hydrogen bonding interactions. By dynamic
programming, the algorithm runs in O(N^2) for the identity permutation, and in
at most O(N^4) for the Greek key motifs, where N is the number of amino acids.</p>

<h3>Results</h3>

<p class="abresults">We develop a web-based application which is capable to
predict the super- secondary structure of transmembrane β-barrel proteins with
or without permuted structure. The input is a sequence, a pattern for the
strand per- mutation and a training set for the short range statistics. The
output also gives the side-chain orientation of β-strand residues, the
inter-strand residue contacts and the folding pseudo-energy. Thus, most of
α-bundles (97%) and globular β-barrels (81%) are easily rejected and BBP may be
used as an effective tool for genome screening.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have implemented a fast pseudo-energy minimization
method for trans- membrane protein super-secondary structure prediction based
on a variety of potential structures. The model has been tested with TMB
proteins and it performs with an efficiency comparable to those of previous
approaches. Yet our approach does not rely only on the known structure
families; it is more flexible and allows the modeling and exploration of a
large variety of structures. We are validating the results on the well known yeast
genome and further results are expected for other genomes.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.lix.polytechnique.fr/Labo/Van-Du.Tran/bbp">http://www.lix.polytechnique.fr/Labo/Van-Du.Tran/bbp</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Van Du T. Tran (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45923 = 'v&#97;nd&#117;' + '&#64;';
 addy45923 = addy45923 + 'l&#105;x' + '&#46;' + 'p&#111;lyt&#101;chn&#105;q&#117;&#101;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy45923 + suffix + '\'' + attribs + '>' );
 document.write( addy45923 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Laboratoire d'Informatique de l'Ecole Polytechnique</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Laboratoire d'Informatique de l'Ecole Polytechnique</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Ecole Polytechnique</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection30">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743037">C-29. Prediction of three dimensional structure of
protein complexes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Plewczyński D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Plewczyński D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1),Łaźniewski M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Łaźniewski M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2*), Augustyniak R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Augustyniak R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ginalski K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ginalski K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Docking is one of the most commonly used techniques in
drug design. It is employed for both identifying correct poses of a ligand in
the binding site of a protein as well as for the estimation of the
protein-ligand interaction strength. Since millions of compounds must be
screened, before identifying suitable target for biological testing, all
calculations should be done in a reasonable time frame. Since almost every year
new software is released it should be evaluated by the independent group, to
confirm advantage of new version. It is also crucial to compare all programs on
same data set.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Seven popular docking programs (Surflex, LigandFit,
Glide, GOLD, FlexX, eHiTS and AutoDock) were tested on the extensive dataset
composed of 1300 protein-ligands complexes from PDBbind 2007 database, where experimentally
measured binding affinity values are also available. We compared independently
the ability of proper posing (according to RMSD of predicted conformations
versus the corresponding native one) and scoring (by calculating correlation
between docking score and ligand binding strength). To our knowledge it is the
first large-scale docking evaluation.</p>

<h3>Results</h3>

<p class="abresults">Based on our results we can order tested programs in the
following way: GOLD ~ eHITS &gt; Surflex &gt; Glide &gt; LigandFit &gt; FlexX ~
AutoDock. The best programs have the average RMSDtop score around 2.7Å, and it
rises to nearly 4.5Å for the weakest FlexX. Percentage of successfully docked
pairs, i.e. complexes for which top scored conformation had RMSD below 2Å, is
around 60% for best program GOLD, while for weakest FlexX it drops below 40%.
As for scoring capabilities Pearson correlation between docking score and
ligands true binding affinities is 0.36 for best in this test eHiTS scoring
function.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our results clearly show that using existing software
ligand binding conformation could be identified in most cases. Moreover
programs produce similar docking solutions regardless of ligands starting
conformations used for docking. Still we observe the lack of universal scoring
function for all types of molecules and protein families. Most promising
function in our test eHiTS_score is classified to knowledge-based type of
functions and it clearly outperforms functions representing different
approaches in scoring like force-field based GoldScore or empirically based
ChemScore.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michał Łaźniewski (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12372 = 'm&#105;ch&#97;ll' + '&#64;';
 addy12372 = addy12372 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy12372 + suffix + '\'' + attribs + '>' );
 document.write( addy12372 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5007 = 'm&#105;ch&#97;ll' + '&#64;';
 addy5007 = addy5007 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy5007 + suffix + '\'' + attribs + '>' );
 document.write( addy5007 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99166 = 'm&#105;ch&#97;ll' + '&#64;';
 addy99166 = addy99166 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy99166 + suffix + '\'' + attribs + '>' );
 document.write( addy99166 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Warsaw</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. 1Interdisciplinary Centre for Mathemtaical and
Computetiobal Modelling, University of Warsaw, Pawińskiego 5a Street, 02-106
Warsaw, Poland 2. 2Department of Physical Chemistry, Faculty of Pharmacy,
Medical University of Warsaw, Banacha 1 Street, 02-097 Warsaw, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Calculations were performed at the Interdisciplinary
Center for Mathematical and Computational Modelling. This work was supported by
Polish Ministry of Science and Higher Education N301 159735 grant</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection31">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743038">C-30. Investigating differences in the structural
environment of parallel and anti-parallel beta sheets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bawono P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bawono P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Abeln S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Abeln S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Beta sheet is the second most common form of regular
secondary structure in proteins. Despite numerous studies into the structural
specifics of beta sheets, the stabilising factors of parallel versus anti
parallel beta sheets are not yet well understood. Further understanding of the
stability of beta sheets is important in the analysis of amyloid formation.
This project investigates the differences in the environmental structure of
parallel and anti parallel beta sheets, and whether these differences account
for the stability and relative abundance of parallel and anti parallel beta
sheets.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this research we focus on two major structural
features: the solvent accessibility and hydrogen bonds. We use DSSP to
calculate the solvent exposed surface of each residue. A residue is considered
buried if its solvent exposed surface is less than 7 %. We categorize beta
sheets into local and non local based on the sequence distance between two
adjacent strands, in order to see whether the locality can explain the
difference in solvent accessibility. If the distance is greater than 15
residues then the beta sheet is non local and it is local if otherwise.</p>

<h3>Results</h3>

<p class="abresults">We found that there is a significant difference in solvent
accessibility. There are more buried residues in parallel beta sheets than in
anti parallel beta sheets. Typically, anti parallel sheets have half of the
residues buried and the other half exposed. We also found that almost all
parallel sheets are non local while anti parallel are more local. The
difference in solvent accessibility remains when we account for the difference
in locality between parallel and anti-parallel sheets. We are currently
analyzing the hydrogen bonding patterns for further environmental analysis.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We show that there is an unexplained difference in the
structural environment of parallel and anti-parallel beta sheets, with
anti-parallel beta sheets being more exposed to the solvent. It is well known
that anti-parallel strands are generally more local than parallel strands.
However, comparisons between parallel sheets with both local and non local anti
parallel sheets still show significant difference, leaving the environmental
difference between parallel and anti-parallel sheets still unexplained.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Punto Bawono (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52192 = 'pb&#111;300' + '&#64;';
 addy52192 = addy52192 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy52192 + suffix + '\'' + attribs + '>' );
 document.write( addy52192 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61375 = 'pb&#111;300' + '&#64;';
 addy61375 = addy61375 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy61375 + suffix + '\'' + attribs + '>' );
 document.write( addy61375 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5430 = 'pb&#111;300' + '&#64;';
 addy5430 = addy5430 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy5430 + suffix + '\'' + attribs + '>' );
 document.write( addy5430 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Vrije Universiteit, the Netherlands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Centre for Integrative Bioinformatics Vrije
Universiteit Amsterdam</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection32">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743039">C-31. Protein model quality assessment based on
structural and functional similarities</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Konopka BM<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Konopka BM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Nebel J-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nebel J-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Kotulska M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kotulska M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Experimental determination of 3D structures is
time-consuming, expensive and has technical limitations. Consequently,
computational prediction of 3D structures is a very attractive alternative. If
homologue structures are available, computer-based models can be very accurate.
Otherwise, threading and ab initio methods can be applied to approximate the
structure of the target. Since these approaches generally produce several
models per target, the ability to evaluate their quality is essential. Thus
quality assessment procedures should be developed alongside new modelling
techniques.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Since there is a strong structure-function
relationship in proteins, knowledge about function should allow discriminating
against poor structure candidates. First, the Distance matrix ALIgnment method
is used to find molecules that are structurally similar to the investigated
model. Next the description of protein functions from Gene Ontology is used to
evaluate the functional similarity of the model's structural neighbours and the
prediction target. The gathered information is processed in a modified Receiver
Operating Characteristic curve framework to estimate the global quality of the
model</p>

<h3>Results</h3>

<p class="abresults">The application was validated using models submitted to
CASP8 (The Critical Assessment of protein Structure Predictions) contest. The
observed quality of models was measured with GDT_TS, which is one of the
standard metrics used by CASP assessors. Out of 127 released targets, 75 were
found to have Gene Ontology annotations, thus these were used in the validation
process. Correlation coefficients of proposed measures and GDT_TS, pooled for
all targets and for each target separately were calculated: best Rpooled=0.511,
best Raverage=0.521.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The method performed well discriminating between good and
bad models. However, its performance in the task of ranking high quality models
was less satisfactory. Since the approach is novel and benefits from sources of
information that are not explored by other Model Quality Assessment Programmes,
there is a chance that in conjunction with other methods it can significantly
increase the accuracy of estimating the quality of protein model-structures.
The method can be efficiently employed when neither experimental structure of a
protein nor its homologues are known.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Bogumil M. Konopka (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1285 = 'b&#111;g&#117;m&#105;l.k&#111;n&#111;pk&#97;' + '&#64;';
 addy1285 = addy1285 + 'pwr' + '&#46;' + 'wr&#111;c' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy1285 + suffix + '\'' + attribs + '>' );
 document.write( addy1285 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Wroclaw University Of Technology, Faculty of
Fundamental Problems of Technology, Institute of Biomedical Engineering and
Instrumentation</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Institute of Biomedical Engineering and
Instrumentation, Faculty of Fundamental Problems of Technology Wroclaw
University Of Technology, Wroclaw, Poland 2 Faculty of Computing, Information
Systems and Mathematics, Kingston University, Kingston Upon Themes, United
Kingdom</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The project was supported by the Department of Student
Affairs at Wroclaw University of Technology (grant no. 273/2010) and Wroclaw
Centre for Networking and Supercomputing (computing grant. no 98) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection33">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743040">C-32. Data driven structure prediction: calculating
accurate small angle X-ray scattering curves from coarse-grained protein models</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Stovgaard K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Stovgaard K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Andreetta C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andreetta C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ferkinghoff-Borg J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ferkinghoff-Borg J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Hamelryck T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hamelryck T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Genome sequencing projects are currently expanding the
gap between the amount of known protein sequences and structures. The
limitations of current high resolution structure determination methods, like
X-ray crystallography and NMR spectroscopy, make it unlikely that this gap will
disappear in the near future. Small angle X-ray scattering (SAXS) is an
established method for routinely determining the structure of proteins in
solution, but due to the lower structural information content in this data,
additional model constraints are needed for the structural interpretation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In order to calculate the scattering curve for a
coarse-grained protein structure, we used the well-known Debye formula.
Scattering form factors for each amino acid were needed in this description for
computational efficiency. The Top500 set of protein structures was used for
training. A maximum-likelihood based MCMC method implemented in the Muninn
program was applied to obtain a set of samples from the posterior form factor
distributions. TorusDBN, a probabilistic model of protein structure formulated
as a dynamic Bayesian network was used to calculate the likelihood of the decoy
structures.</p>

<h3>Results</h3>

<p class="abresults">Two coarse-grained representations of protein structures
were investigated; using two scattering bodies per residue gave significantly
better results than using a single body. Obtained form factor point estimates
were evaluated for a set of diverse proteins, with resulting SAXS curves on par
with the current state-of-the-art program CRYSOL. Furthermore, our method was
validated in a Z-score comparison for a set of native-like protein decoys from
the TASSER program, showing excellent results. Including prior information from
the probabilistic model TorusDBN further improved decoy recognition.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have demonstrated that it is possible to obtain
accurate SAXS curves from coarse-grained protein structures and matching
estimated form factors without the use of ad hoc correction factors. SAXS curve
calculations and decoy recognition experiments show that our method performs on
par with the current state-of-the-art program CRYSOL, which requires
full-atomic detail. The presented method shows great promise for use in
statistical inference of protein structures from SAXS data. Such an approach is
currently being implemented in the PHAISTOS software package.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.phaistos.org" mce_href="http://www.phaistos.org">http://www.phaistos.org</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Kasper Stovgaard (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56747 = 'st&#111;vg&#97;&#97;rd' + '&#64;';
 addy56747 = addy56747 + 'b&#105;nf' + '&#46;' + 'k&#117;' + '&#46;' + 'dk';
 document.write( '<a ' + path + '\'' + prefix + addy56747 + suffix + '\'' + attribs + '>' );
 document.write( addy56747 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">The Bioinformatics Centre, University of Copenhagen</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) The Bioinformatics Centre, Department of Biology,
University of Copenhagen, Denmark (2) Department of Electrical Engineering,
Technical University of Denmark, Lyngby, Denmark</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Danish Council for Strategic Research (Program
Commission on Nanoscience, Biotechnology and IT; project: Simulating proteins
on a millisecond time scale, 2106-06-0009) and the Danish Council for
Independent Research (Danish Research Council for Technology and Production
Sciences; project: Data driven protein structure prediction, 274-06-0380) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection34">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743041">C-33. OpenStructure: A flexible software framework
for computational structural biology</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Biasini M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Biasini M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Mariani V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mariani V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Haas J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Haas J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Scheuber S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Scheuber S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Schenk A-D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schenk A-D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Schwede T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schwede T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Philippsen A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Philippsen A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">We introduce OpenStructure, a flexible software framework
for computational structural biology, which emerged out of the need for a
solid, yet flexible and versatile toolkit for method development. It combines a
C++ based library of commonly with a Python layer and powerful visualization
tools. OpenStructure is also designed to easily accommodate interfaces to
already existing software. This allows for rapid visually-enhanced prototyping
of new functionality, making OpenStructure an ideal environment for the development
of next-generation structural biology algorithms.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The functionality is grouped into modules. Each of
these modules consists of a computational core in the form of a shared library
of C++ code, a dynamic module that exposes the C++ API to Python using
Boost.Python and a set of Python modules built on top of the exported API.
Parts of the computational core and the GUI of the Image Processing Library and
Toolkit IPLT have been incorporated into the OpenStructure geom, img and gui
modules to offer versatile handling of image data with support for various
algorithms. Other modules have been written anew to manipulate sequence and
structural data.</p>

<h3>Results</h3>

<p class="abresults">Typically, users interact with a high-level Python
interface, while power users with high computational requirements access the
API directly at the level of C++ (optionally exporting their algorithms to
Python). The framework additionally features capabilities for real-time
rendering of molecules, density maps and molecular surfaces which can be
visualized in a graphical user interface.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The flexibility and expressive API of OpenStructure make
it an ideal basis for the development of computational methods. For example,
new versions of the QMean tools for model quality assessment are based on
OpenStructure, as well as the structural variation analysis tools in
ProteinModelPortal. Further, work is on the way to implement the next
generation of the SWISS-MODEL pipeline using the OpenStructure framework.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.openstructure.org" mce_href="http://www.openstructure.org">http://www.openstructure.org</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Marco Biasini (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52669 = 'm&#97;rc&#111;.b&#105;&#97;s&#105;n&#105;' + '&#64;';
 addy52669 = addy52669 + '&#117;n&#105;b&#97;s' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy52669 + suffix + '\'' + attribs + '>' );
 document.write( addy52669 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Biozentrum, University of Basel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Biozentrum, Universität Basel, Basel, Switzerland. 2
SIB Swiss Institute of Bioinformatics, Basel, Switzerland. 3 Department of Cell
Biology, Harvard Medical School, Boston, MA 02115 , USA</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection35">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743042">C-34. Modeling structure of ionic channels from
rigid network of contact sites</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kotulska M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kotulska M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2, *), Gerstein M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gerstein M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The ratio of experimental structures of protein channels
to their total number is disastrously small compared to other proteins. Lipid
environment of the channels and their great size impede the development.
Improvement could contribute to drug design and resolving mechanisms of
channelopathies. Modeling 3D structure of transmembrane channels will be
enhanced by constraints defined by their contact sites distribution.
Representing protein as a network of contact sites and characterizing topology
of such a channel graph could be employed for limiting protein decoys in
modeling process.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Protein is represented as a network of non-bond
contact sites. As a contact site we consider residues within a defined distance
between protein C alpha atoms that are separated by a certain number of amino
acids. We test how definition of the contact site, in terms of the distance and
separation threshold values, affects topology of the network and which graph
model characterizes the protein channel best. The tests are provided for
helical channels and barrels. Transmembrane and soluble parts are compared.</p>

<h3>Results</h3>

<p class="abresults">Compared to random (Erdős–Rényi) network, graph of a protein
shows high clustering coefficient. The average shortest path length is small
but greater than in the random model, as in “small-world” graphs, which
describe all real world networks. The node degree in protein graphs changes
from Poisson distribution ( “small-world” ) to power-law (random graphs) as the
residue separation increases. Graphs with high value of the distance threshold
also tend from power-law to Poisson distribution. Transmembrane helices
contribute power-law, while other components Poisson distribution.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The graph model of ionic channel shows distinct features,
related also to its secondary structure and location. This could be utilized in
protein structure modeling process with regard to contact site distribution.
The network topology proved sensitive to the contact site definition, showing
that protein graph obeys typical real world network characteristics only if the
distance is below certain threshold and the separation above a limit number,
and follows random network characteristics otherwise. The thresholds should be
incorporated into optimum definition of a contact site.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Malgorzata Kotulska (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3417 = 'm&#97;lg&#111;rz&#97;t&#97;.k&#111;t&#117;lsk&#97;' + '&#64;';
 addy3417 = addy3417 + 'pwr' + '&#46;' + 'wr&#111;c' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy3417 + suffix + '\'' + attribs + '>' );
 document.write( addy3417 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Wroclaw University Of Technology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Biomedical Engineering and
Instrumentation, Wroclaw University of Technology, Wroclaw, Poland; (2)
Department of Molecular Biophysics and Biochemistry, Yale University, New
Haven, USA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">MK would like to acknowledge the Fulbright scholarship
to Yale University . </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection36">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743043">C-35. Multi-task sequence labeling for protein
annotation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Maes F<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Maes F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Becker J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Becker J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Wehenkel L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wehenkel L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Many tasks related to protein annotation can be
formalized as sequence labeling problems where the input is a sequence of amino
acids and the output is a corresponding sequence of labels (secondary
structure, solvent accessibility, disordered region, etc.). To solve $k$ such
tasks with machine learning techniques, we can independently learn $k$ sequence
labeling models. Several textit{multi-task} approaches that {em jointly} learn
multiple related problems have been proposed in the machine learning community
recently and have been shown to often outperform single-task approaches. We
propose one such model for protein annotation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We introduce an original multi-task approach for
sequence labeling problems that iteratively re-estimates each target sequence
several times, using the input and the current state of all other predicted
sequences. * Each label is estimated based on a surrounding window of input
elements and of predicted elements in other target sequences. The
label-predictor is a linear support vector machine, learned with stochastic
gradient descent. * Our study relies on two datasets. The first one includes 500
proteins extracted from PDB (at 70% identity). The second one is the CB513
dataset which enables to compare our results with the state-of-the-art of
secondary structure.</p>

<h3>Results</h3>

<p class="abresults">To illustrate our novel approach, we focus on the joint
prediction of the secondary structure (with both three and eight classes), the solvent
accessibility (with two classes) and the disorder regions (with two classes).
We show experimentally that the multi-task approach nearly always outperforms
single-task sequence labeling on the PDB dataset.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We show that solving simultaneously multiple protein
annotation problems leads to an improvement over methods that treat each
problem independently. When combining two tasks, the results are nearly always
better than corresponding single-task models. However, when treating more than
two tasks together, we often observe a slight degradation of results. This
might be due to an over-fitting problem. Our future work includes making
experiments on larger sets of proteins.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Julien Becker (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93298 = 'J.B&#101;ck&#101;r' + '&#64;';
 addy93298 = addy93298 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy93298 + suffix + '\'' + attribs + '>' );
 document.write( addy93298 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74490 = 'J.B&#101;ck&#101;r' + '&#64;';
 addy74490 = addy74490 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy74490 + suffix + '\'' + attribs + '>' );
 document.write( addy74490 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64362 = 'J.B&#101;ck&#101;r' + '&#64;';
 addy64362 = addy64362 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy64362 + suffix + '\'' + attribs + '>' );
 document.write( addy64362 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Systems and Modeling - Department of EE and CS,
University of Liege, Belgium</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Systems and Modeling - Department of EE and CS, Systems
Biology and Chemical Biology - GIGA-Research, University of Liege, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This paper presents research results of the Belgian
Network BIOMAGNET (Bioinformatics and Modelling: from Genomes to Networks),
funded by the Interuniversity Attraction Poles Programme, initiated by the
Belgian State and of the PASCAL network. Julien Becker is recipient of a F.R.I.A.
fellowship. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection37">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743044">C-36. A convex programming model for protein
structure prediction</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kauffman C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kauffman C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Karypis G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Karypis G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Most current protein structure prediction methods involve
high computational cost, mostly due to the nonconvex function used to represent
the protein's potential energy surface. Though thought to be chemically
accurate, such functions result in may local minima which must be searched to
find the global minimum energy conformation, generally regarded as where the
native state of the protein will reside. We propose an alternative in which we
can guarantee protein structures are at the global minimum and find them
quickly, the cost being interpretability of the energy function.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach is to represent protein structure
conformations in a semidefinite program (SDP), a convex optimization problem in
which a global minimum can be found efficiently. Bond lengths become linear constraints
in the program and atom clashes are avoided using inequality constraints. The
energy of the conformation is represented as a weighted sum of the squared
distances between unbonded atoms. Inverse optimization may be used to
efficiently determine energy function parameters which place a structure at the
energy minimum.</p>

<h3>Results</h3>

<p class="abresults">One application of our convex framework is to recover coordinates
from distance information on structures, potentially a useful capability for
NMR experiments. In an evaluation study on 124 known structures, our method
produced statistically lower RMSDs than two standard distance geometry
techniques, though our method needs to be adapted to noisy and sparse distance
matrices which are usually the case for NMR experiments. We are also conducting
a study of sequence-based structure prediction on a set of 2748 proteins using
convex optimization.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our work is an attempt to determine how well protein
structure can be estimated using convex optimization techniques. Restricting
ourselves to the class of convex functions ensures that at the very least the
resulting technique will be fast enough for most desktop workstations to
produce structure predictions in a matter of minutes to hours. Our preliminary
results indicate that there is potential for convex optimization to produce
accuracy that is competitive with standard homology modeling and ab initio
structure prediction techniques.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinfo.cs.umn.edu/supplements/eccb2010" mce_href="http://bioinfo.cs.umn.edu/supplements/eccb2010">http://bioinfo.cs.umn.edu/supplements/eccb2010</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Chris Kauffman (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37819 = 'k&#97;&#117;ffm&#97;n' + '&#64;';
 addy37819 = addy37819 + 'cs' + '&#46;' + '&#117;mn' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy37819 + suffix + '\'' + attribs + '>' );
 document.write( addy37819 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3033 = 'k&#97;&#117;ffm&#97;n' + '&#64;';
 addy3033 = addy3033 + 'cs' + '&#46;' + '&#117;mn' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy3033 + suffix + '\'' + attribs + '>' );
 document.write( addy3033 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23387 = 'k&#97;&#117;ffm&#97;n' + '&#64;';
 addy23387 = addy23387 + 'cs' + '&#46;' + '&#117;mn' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy23387 + suffix + '\'' + attribs + '>' );
 document.write( addy23387 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Minnesota, Twin Cities</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">University of Minnesota, Twin Cities</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">National Institute of Health [T32GM008347, RLM008713A],
National Science Foundation [IIS-0431135], and the University of Minnesota
Digital Technology Center</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection38">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743045">C-37. Predicting protein function with the relative
backbone position kernel</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schietgat L<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Schietgat L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Aryal S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aryal S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Ramon J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ramon J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Proteins are macromolecules that play crucial roles in
many biological processes. As more data about proteins become available, the
automatic classification of their function is an important challenge in
bioinformatics. A lot of techniques have been proposed that predict function
based on the primary and secondary structure of the proteins. However, little
attention has gone to 3D structures of proteins, while these carry a lot of
additional information. Recently, it was shown that kernel methods provide good
results on protein classification tasks. However, none of them use 3D
information.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We propose a new kernel for proteins called the
relative backbone position kernel (RBPK). It makes use of 3D information by
comparing Euclidean distances between the residue atoms and the backbone atoms
of the protein. In this way, the kernel can select spatial features that are
important for interactions with ligands or other proteins and will influence
protein function. We evaluate our kernel through two datasets: one contains
protein structures that have to be classified into enzymes and non-enzymes,
while for the second one, the task is to predict the resistance of HIV protease
structures.</p>

<h3>Results</h3>

<p class="abresults">We compare the performance of RBPK with the Fast Subtree
Kernel (FSTK), which is a state-of-the-art kernel for protein function
classification. FSTK uses a graph with amino acids as vertices and distances as
edges for the representation of the proteins. Although FSTK is more efficient
to compute than RBPK, the latter obtains a higher predictive accuracy,
resulting in state-of-the-art results for the two datasets.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our experimental results show that RBPK, wich exploits 3D
information of the protein, leads to more accurate predictions over a recently
proposed graph-based kernel. The accuracy of above 85% for the first dataset
(D&amp;D) is an encouraging result. However, the computational efficiency of
the kernel still remains an important issue, especially for large proteins.
There are several ideas to improve this, by limiting the amount of distances
that are computed by the kernel.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Leander Schietgat (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15429 = 'l&#101;&#97;nd&#101;r.sch&#105;&#101;tg&#97;t' + '&#64;';
 addy15429 = addy15429 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy15429 + suffix + '\'' + attribs + '>' );
 document.write( addy15429 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Computer Science, Katholieke Universiteit
Leuven</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is funded by the ERC Starting Grant
240186: Mining Graphs and Networks: a Theory-based approach. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection39">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743046">C-38. The Torsional Network Model: improvements in
modelling of protein flexibility</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Méndez Giráldez R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Méndez Giráldez R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Bastolla U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bastolla U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Proteins undergo significant conformational
rearrangements to achieve their function, which often pose serious limitations
to protein structure prediction and docking methods. Those collective motions
have been assessed through normal mode computations, using typical all atom
force fields. Elastic Network Models in Cartesian space simplify computations
but still yield reasonable results, at expense of distorting protein secondary
structure. Our TNM increases protein structure accuracy while reducing the
number of degrees of freedom, by only allowing main backbone torsion angles to
change.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The matrices of the second derivatives of the
Elastic Network potential and kinetic energy with respect to torsion angles are
calculated as linear transformations of the corresponding matrices in Cartesian
space through the Jacobian. The latter is restricted to internal degrees of
freedom imposing Eckart conditions. Normal modes in torsional space are
obtained by solving a generalized eigenvalue problem. TNM modes are compared to
ANM ones in terms of: tolerance to elongation, their ability to account for
observed conformational changes and the energy barrier reduction in such
processes.</p>

<h3>Results</h3>

<p class="abresults">TNM modes are in average more robust with than ANM ones, as
witnessed by Lennard Jones energy calculations, in a set of 500 protein
structures. Low frequency modes are more deformable than high frequency ones,
account better for conformational changes in a different set of 50 protein structures
and are localized in torsional space, but collective in Cartesian space.
Transport and enzyme proteins seem to reduce significantly their conformational
change barriers as a result of the selective pressure. Their collective motions
can be described by few modes (fewer for the TNM than the ANM).</p>

<h3>Discussion</h3>

<p class="abdiscussion">The TNM allows better protein motion characterization
than conventional Cartesian Elastic Network Models, and reduces its
computational cost. TNM modes (i.e. low frequency ones) localized in torsional
space and collective in Cartesian space may be reminiscent of hinge motions. In
general, a smaller number of (low frequency) TNM modes than ANMs, is needed to
describe the same conformational change. Evolution seems to reduce energy
barriers via few low frequency modes that contribute significantly to the
change. The results make TNM ideal to model protein flexibility in
Computational Biology.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Raúl Méndez Giráldez (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93 = 'r&#97;&#117;lm&#101;nd&#101;z' + '&#64;';
 addy93 = addy93 + 'cbm' + '&#46;' + '&#117;&#97;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy93 + suffix + '\'' + attribs + '>' );
 document.write( addy93 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Centre for Molecular Biology "Severo Ochoa"</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Bioinformatics Unit Centre for Molecular Biology
"Severo Ochoa" Universidad Autónoma de Madrid C/ Nicolás Cabrera, 1 Cantoblanco
28049, Madrid SPAIN</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Centrosome 3D (Spanish Ministry of Science), CSIC
(Spanish Supreme Council for Scientific Research) through the JAE program</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection40">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743047">C-39. MEDELLER: coordinate generation for membrane
proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kelm S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Kelm S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Shi J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shi J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Deane C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Deane C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Deane CM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Membrane proteins are important drug targets but
knowledge of their exact structure is limited to relatively few examples.
Existing homology-based structure prediction methods are designed for globular,
water-soluble proteins. However, we are now beginning to have enough membrane
protein structures to justify the development of a homology-based approach
specifically for them. We present a homology-based coordinate generation
method, MEDELLER, which uses membrane protein-specific information, and compare
it to the popular software Modeller.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Comparison between the two methods was performed on
a set of over 1000 target-template pairs of membrane proteins, which were
classified into four test sets by their sequence identity. MEDELLER first
builds a core model, using membrane protein-specific information. Then, loops
are added using the FREAD loop structure prediction algorithm, to generate
high-accuracy models.</p>

<h3>Results</h3>

<p class="abresults">MEDELLER's accuracy distribution over all test sets is
clearly shifted towards the more accurate end, when compared to Modeller, with
an average backbone RMSD of 2.61Å versus 3.02Å for Modeller. On our “easy” test
set, MEDELLER achieves an average accuracy of 0.89Å backbone RMSD, versus 1.55Å
for Modeller.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The MEDELLER algorithm is conceptually simple and
outperforms the popular structure prediction method Modeller on all test sets.
Its major advantage lies in the generation of an accurate core model. Future
versions of MEDELLER will also address the problem of helix kinks and twists,
promising a better overall model quality in cases where the template differs
slightly from the target structure.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://imembrane.info" mce_href="http://imembrane.info">http://imembrane.info</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sebastian Kelm (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98475 = 'k&#101;lm' + '&#64;';
 addy98475 = addy98475 + 'st&#97;ts' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy98475 + suffix + '\'' + attribs + '>' );
 document.write( addy98475 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy25049 = 'k&#101;lm' + '&#64;';
 addy25049 = addy25049 + 'st&#97;ts' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy25049 + suffix + '\'' + attribs + '>' );
 document.write( addy25049 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19261 = 'k&#101;lm' + '&#64;';
 addy19261 = addy19261 + 'st&#97;ts' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy19261 + suffix + '\'' + attribs + '>' );
 document.write( addy19261 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Oxford</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Statistics, University of Oxford,
Oxford, UK (2) UCB Celltech, Slough, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Biotechnology and Biological Sciences Research Council</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection41">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743048">C-40. Structure-based predictor of HIV coreceptor
tropism</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bozek K<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Bozek K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Lengauer T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lengauer T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Domingues FS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Domingues FS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Human immunodeficiency virus (HIV) cell entry is the
first step of infection and conditions further successful replication. In
addition to the main cellular receptor CD4 the virus attaches to one of the
coreceptors CCR5 or CXCR4. Viruses binding to CCR5 have been shown to be
predominantly present during the early asymptomatic stage of infection.
CXCR4-binding viruses are associated with disease progression. New HIV
treatment strategies block CCR5 coreceptor. Monitoring of the virus coreceptor
usage and understanding the mechanisms driving coreceptor switch are therefore
of crucial importance.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We built a structural descriptor of the V3 loop,
the main determinant of viral tropism, to predict coreceptor usage and to gain
more insight into its molecular background. The amino acid properties of V3
loops were mapped on the published reference crystal structures. Each V3 loop
was described as a concatenated vector of amino acid properties in its
structural parts. We next performed feature selection to reduce the redundancy
of highly correlated properties and to extract biologically meaningful
structural and physicochemical determinants of coreceptor tropism.</p>

<h3>Results</h3>

<p class="abresults">This systematic search for structural determinants of the
HIV tropism resulted in an improved prediction method allowing for biological
interpretation. The initial prediction based on the full set of descriptors had
a similar accuracy to the sequence-based method with lower efficiency due to
the long vector representation of each of the sequence. Reducing the feature
set increased the efficiency of the method and improved prediction accuracy
both in the clonal and clinical datasets. The selected features point to
structural parts of the loop and its properties crucial for the coreceptor
usage.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Structural descriptors are a more accurate measure of the
actual physicochemical similarities among amino acids than the 20 letter
representation used in the sequence-based methods. The proposed coreceptor
prediction method operates on a relatively simple and efficient algorithm not
involving structure modeling. The features selected out of the full set of
structural descriptors distributed along the V3 loop, not only improved the
computational efficiency of the prediction procedure but also indicated which
of the biochemical properties on which part of the loop are crucial for the
tropism.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Katarzyna Bozek (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56657 = 'b&#111;z&#101;k' + '&#64;';
 addy56657 = addy56657 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy56657 + suffix + '\'' + attribs + '>' );
 document.write( addy56657 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14066 = 'b&#111;z&#101;k' + '&#64;';
 addy14066 = addy14066 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy14066 + suffix + '\'' + attribs + '>' );
 document.write( addy14066 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39786 = 'b&#111;z&#101;k' + '&#64;';
 addy39786 = addy39786 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy39786 + suffix + '\'' + attribs + '>' );
 document.write( addy39786 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Max Planck Institute for Informatics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Max Planck Institute for Informatics, Computational
Biology and Applied Algorithmics</p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="116-protein-and-nucleotide-structure.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>