<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Prediction and Annotation of Molecular Function</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Prediction and Annotation of Molecular Function			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Prediction and Annotation of Molecular Function</span><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant: normal ! important; text-transform: uppercase;"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Nir Ben-Tal and Frederick Roth<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Prediction and Annotation of
Molecular Function<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743770 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span><span style=""><o:p></o:p></span></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743771" mce_href="#_Toc269743771">D-1. Validating the structural
information exchange within the domain Fyn SH2<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743771 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">3</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743772" mce_href="#_Toc269743772">D-2. Discriminant functions for
classification and prediction of T1SS, T3SS, T4SS and T6SS secreted proteins
from proteobacteria<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743772 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">4</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743773" mce_href="#_Toc269743773">D-3. Computing fragmentation trees
from tandem mass spectrometry data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743773 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743774" mce_href="#_Toc269743774">D-4. Computational identification of
new miRNAs in genomic regions associated with psychiatric diseases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743774 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743775" mce_href="#_Toc269743775">D-5. CatANalyst: a web server for
predicting catalytic residues<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743775 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743776" mce_href="#_Toc269743776">D-6. Functional characterization of
Parkinson by high-throughput data analysis with l1l2 regularization<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743776 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743777" mce_href="#_Toc269743777">D-7. Feature elimination for
one-class microRNA target prediction and gene identification<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743777 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">9</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743778" mce_href="#_Toc269743778">D-8. Computational identification of
synonymous SNPs in the human genome and their potential role in disease<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743778 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743779" mce_href="#_Toc269743779">D-9. The potential functional role
of chimeric transcripts in higher eukaryotes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743779 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700370039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743780" mce_href="#_Toc269743780">D-10. A "Functional
Signature" for analysing annotation space<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743780 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743781" mce_href="#_Toc269743781">D-11. Predicting small molecule
ligand binding sites and catalytic residues with firestar/FireDB<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743781 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743782" mce_href="#_Toc269743782">D-12. Classifying substrate
specificities of membrane transporters from Arabidopsis thaliana<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743782 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743783" mce_href="#_Toc269743783">D-13. Molecular modeling and
comparison by docking and molecular dynamics of the interaction between fatty
acids and proteins and EgFABP2 EgFABP1<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743783 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743784" mce_href="#_Toc269743784">D-14. GreenPhylDB version 2: web
resources for comparative and functional genomics in plants<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743784 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743785" mce_href="#_Toc269743785">D-15. Collaboration-based function
prediction in protein-protein interaction networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743785 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743786" mce_href="#_Toc269743786">D-16. SVM feature selection applied
to lignocellulose degradation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743786 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">18</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743787" mce_href="#_Toc269743787">D-17. Substrate-specific HMMs for
the classification of adenylation and acyltransferase domains of NRPS/PKS
systems<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743787 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">19</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743788" mce_href="#_Toc269743788">D-18. Stochastic modeling of
proteolytic activity from LC-MS data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743788 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">20</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743789" mce_href="#_Toc269743789">D-19. Mining posttranscriptional
regulatory features by comparing total and polysomal mRNA profilings<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743789 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">21</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700380039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743790" mce_href="#_Toc269743790">D-20. Prototypes of elementary
functional loops unravel evolutionary connections between protein functions<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743790 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">22</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743791" mce_href="#_Toc269743791">D-21. Utilization of proteins and
nucleic acids in the study of gene function: a comparative review<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp;&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743791 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">23</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743792" mce_href="#_Toc269743792">D-22. ClanTox: A predictor tool for
toxin-like proteins reveals 500 such proteins within viral genomes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743792 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">24</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743793" mce_href="#_Toc269743793">D-23. I-Patch web service:
inter-protein contact prediction using local network information<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743793 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">25</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743794" mce_href="#_Toc269743794">D-24. Predicting candidate genes for
biomass traits<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743794 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">26</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743795" mce_href="#_Toc269743795">D-25. Exploring residue interaction
networks to understand protein structure-function relationships<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743795 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">27</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="117-prediction-and-annotation-of-molecular-function.html#_Toc269743796" mce_href="#_Toc269743796">D-26. BioXSD: the canonical XML-Schema
data model for bioinformatics web services<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269743796 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">28</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340033003700390036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743771">D-1. Validating the structural information exchange
within the domain Fyn SH2</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Huculeci R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Huculeci R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Buts L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Buts L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Rousseau F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rousseau F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4), Schymkowitz J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schymkowitz J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4), van Nuland N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Nuland N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Lenaerts T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lenaerts T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5,6)</p>

<p class="abmotivation">In recent years, a variety of in silico methods have been
developed that try to predict, starting from sequential or structural
information, the long-range (allosteric) effects of peptide binding on protein
structures. These predictions are highly relevant since they expose the
functionally relevant parts of the protein, allowing one to gain scientific
insight into highly important regions and their possible relations to known
diseases.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our own approach uses Shannon’s information theory
to quantify the conformation coupling between the residue sidechains [1]. When
comparing the mutual information profiles of the bound and unbound states of a
domain, regions can be identified that may play a role in signal propagation. In
order to validate our predictions we have initiated an extensive NMR study to
analyze the structural and dynamic features of both isolated and peptide bound
Fyn SH2.</p>

<h3>Results</h3>

<p class="abresults">In a first stage two new structural ensembles, representing
the two unbound and bound state, have been determined and these were used to
predict the network of residues affected by binding. In a second stage, the
binding dynamics are experimentally examined through an NMR analysis of the
methyl dynamics. This analysis provides the experimental validation for our
predictions. In this poster we will present the results that are obtained so
far : novel Fyn SH2 structures and the newly predicted and experimental results</p>

<h3>Discussion</h3>

<p class="abdiscussion">This work clearly shows that the predicted residues are
indeed involved in the information processing by the domain structure. Moreover,
it provided the first experimental proof of our results. As such it opens the
road to an new understanding of domain structure and domain engineering in the
longer term.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ulb.ac.be/di/map/tlenaert/" mce_href="http://www.ulb.ac.be/di/map/tlenaert/"><span style="">http://www.ulb.ac.be/di/map/tlenaert/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tom Lenaerts (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy59945 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy59945 = addy59945 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy59945 + suffix + '\'' + attribs + '>' );
 document.write( addy59945 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73404 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy73404 = addy73404 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy73404 + suffix + '\'' + attribs + '>' );
 document.write( addy73404 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23739 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy23739 = addy23739 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy23739 + suffix + '\'' + attribs + '>' );
 document.write( addy23739 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">MLG, DI, Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Structural Biology Brussels, Vrije Universiteit
Brussel, Pleinlaan 2, 1050 Brussel, Belgium 2 Department of Molecular and
Cellular Interactions, VIB, Pleinlaan 2, 1050 Brussel, Belgium 3 Switch,
VIB,Pleinlaan 2, 1050 Brussel, Belgium 4 Vrije Universiteit Brussel, Pleinlaan
2, 1050 Brussel, Belgium 5 MLG, Département d’informatique, Université Libre de
Bruxelles, Boulevard du Triomphe CP212, 1050 Brussel, Belgium 6 Department of
Computer Science, Vrije Universiteit Brussel, Pleinlaan 2, 1050 Brussel,
Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The authors acknowledge the support of the FWO, FNRS
and the VIB for this project</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743772">D-2. Discriminant functions for classification and
prediction of T1SS, T3SS, T4SS and T6SS secreted proteins from proteobacteria</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kampenusa I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kampenusa I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Zikmanis P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zikmanis P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">There are at least four systems being responsible for
leaderless secretion in proteobacteria. Nevertheless, exoproteins
experimentally annotated to the definite secretion type, still remain as a
trifling part of sequenced exoproteome. The available prediction tools can only
attribute leaderless proteins as a whole group without specifications in
respect to appropriate secretion types. Recently some computational approaches
have been proposed to identify substrates of leaderless secretion within the
bacterial exoproteome, however representing the very narrow range of organisms.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Data set: 111 proteins from 30 genera of
proteobacteria annotated as secreted by T1SS, T3SS, T4SS or T6SS
(Swiss-Prot/TrEMBL ID numbers are summarized in: Kampenusa and Zikmanis 2010).
The frequencies of amino acids and codon usage were computed by
ExPASy/ProtParam and Emboss/Cusp tools, respectively. Multiple Discriminant
analysis and Press's Q test were performed by SPPS 11.0 for Windows. The
prediction success of obtained sets of predictor variables (i.e., the
classifiers) were compared by means of McNemar’s test.</p>

<h3>Results</h3>

<p class="abresults">Frequencies of 24 codons, partially representing 16 amino acids
(C D G I K L M N P Q R S T V W Y), and 10 amino acids (G K M N P Q R W Y E)
were selected as two sets of predictor variables from the sets of codon usage
and amino acid frequencies for multiple-4-group discrimination of T1SS, T3SS,
T4SS and T6SS secreted proteins (Q 309.43 &amp; 272.08, p&lt; 0.001;
respectively). The both classifiers exhibit comparable error rates (p&gt;0.05,
McNemar's test) as well as the equal prediction success (81%) for the
affiliation of proteins from independent testing-sets.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The 10 sequences from the training set of proteins being
misclassified by the both classifiers exhibit an opposite trend of
compositional characteristics as compared to the whole set of proteins.
Therefore it is likely, that restrictions of proposed discriminant functions in
respect of an adequate classification reflect somehow different evolutionary
routes of the protein sequences. Nevertheless, the both classifiers can be used
as either independent or complementary tools to specify the substrates of
leaderless secretion with potent more widespread applications outside the
proteobacteria.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ināra Kampenusa (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3038 = '&#105;n&#97;r&#97;.k&#97;mp&#101;n&#117;s&#97;' + '&#64;';
 addy3038 = addy3038 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy3038 + suffix + '\'' + attribs + '>' );
 document.write( addy3038 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99129 = '&#105;n&#97;r&#97;.k&#97;mp&#101;n&#117;s&#97;' + '&#64;';
 addy99129 = addy99129 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy99129 + suffix + '\'' + attribs + '>' );
 document.write( addy99129 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1893 = '&#105;n&#97;r&#97;.k&#97;mp&#101;n&#117;s&#97;' + '&#64;';
 addy1893 = addy1893 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy1893 + suffix + '\'' + attribs + '>' );
 document.write( addy1893 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Latvia, Faculty of Biology, Institute
of Microbiology and Biotechnology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Faculty of Biology, University of Latvia 2 Institute
of Microbiology and Biotechnology, University of Latvia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Support for the research has been granted by ESF
project “Support of doctoral studies at the University of Latvia”. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743773">D-3. Computing fragmentation trees from tandem mass
spectrometry data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rasche F<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Rasche F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Svatos A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Svatos A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Böcker S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Böcker S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Plants and other organisms produce enormous numbers of
unknown metabolites. Identification of such substances is a common task
metabolomics and pharmaceutical chemistry. Tandem mass spectrometry is used to
achieve this, but computational analysis of the spectra is still in its
infancy. Here we introduce fragmentation trees representing the fragmentation
reactions of the molecule. We also present a method for the calculation of
hypothetical fragmentation trees. This method does not require a molecular
structure or mass spectral database.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To calculate the tree we first generate a
fragmentation graph containing all fragmentation trees that are in accordance
with the measured spectra. In the second step, this graph is scored using
properties such as peak intensities, mass deviations, typical neutral losses,
etc. Finally, we search the highest-scoring fragmentation tree inside the
graph. Since we want to explain every peak only once, we have to apply
restrictions that render the problem NP-hard. A fixed parameter tractable
algorithm is used to solve the problem in reasonable running time.</p>

<h3>Results</h3>

<p class="abresults">We tested the method on spectra of 180 compounds from 3
different instruments. Mass spectrometry experts evaluated the resulting trees.
They considered 79% of the fragmentation reactions to be "correct".
The experts used multiple MS spectra of some compounds to support their
assignment. Additionally, we used the commercial software MassFrontier (MF) to
assess our peak annotation. MF annotates a spectrum when given a compound
structure. MF and our tool assigned the same molecular formula to 97% of the
peaks annotated by MF, but our tool annotates four times as many peaks as MF.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have demonstrated that the calculated fragmentation
trees agree well with expert-knowledge and rule-besed fragmentation prediction.
Our method is fully automated and does not require any previous knowledge about
the compound. Fragmentation trees reveal more about the molecular structure
than the spectrum alone. Therefore, in future, we might discover structural
similarities between known and unknown compounds by comparing their
fragmentation trees. We are currently developing a method for this comparison.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bio.informatik.uni-jena.de/software/starburst" mce_href="http://bio.informatik.uni-jena.de/software/starburst"><span style="">http://bio.informatik.uni-jena.de/software/starburst</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Florian Rasche (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89525 = 'fl&#111;r&#105;&#97;n.r&#97;sch&#101;' + '&#64;';
 addy89525 = addy89525 + '&#117;n&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy89525 + suffix + '\'' + attribs + '>' );
 document.write( addy89525 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91434 = 'fl&#111;r&#105;&#97;n.r&#97;sch&#101;' + '&#64;';
 addy91434 = addy91434 + '&#117;n&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy91434 + suffix + '\'' + attribs + '>' );
 document.write( addy91434 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14656 = 'fl&#111;r&#105;&#97;n.r&#97;sch&#101;' + '&#64;';
 addy14656 = addy14656 + '&#117;n&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy14656 + suffix + '\'' + attribs + '>' );
 document.write( addy14656 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Friedrich-Schiller-University Jena</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Chair of Bioinformatics,
Friedrich-Schiller-University Jena, Germany (2) Research Group Mass
Spectrometry, Max Planck Institute for Chemical Ecology, Jena, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Max Planck Society supported this work financially.
</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743774">D-4. Computational identification of new miRNAs in
genomic regions associated with psychiatric diseases</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Aelterman B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Aelterman B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), De Rijk P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Rijk P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Del-Favero J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Del-Favero J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">miRNAs are small (~22 nt) noncoding RNAs that are
important regulators of gene expression. The latest release of miRBase (15.0)
contains 940 human miRNA sequences of which several are already implicated in
human diseases such as Tourette's symdrome, autism, schizophrenia, ....</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a miRNA prediction pipeline that
integrates different programs to predict novel miRNAs in genomic regions
associated with psychiatric diseases. In total we used 8 different miRNA
prediction programs for the construction of our miRNA prediction pipeline.
Predictions of all programs were clustered to form consensus predictions and a
score was allocated to each prediction based on the number of different
programs supporting the prediction. We used available Next Generation
Sequencing data to validate the obtained predictions.</p>

<h3>Results</h3>

<p class="abresults">We ran our pipeline on 15 genomic regions, all of which are
associated or linked to psychiatric diseases. The total size of these regions
is approximately 1,5% of the human genome. Over 300.000 miRNAs were predicted
with prediction scores ranging from 1 to 7. To add experimental evidence to the
predictions, we developed an in silico validation platform based on sequence
similarity between the miRNA predictions and sequence reads from RNA-Seq data
available via NCBI's Sequence Read Archive. When selecting all miRNA
predictions with a prediction score above 5 (n=163) we were able to show that
21 predicted miRNAs were present in the RNA-seq data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We were able to predict novel miRNAs and could validate
21 novel miRNAs. As these are located in genomic regions associated with
schizophrenia or bipolar disorder, they are potentially genetic risk factors
for these diseases. Since we validated 21 novel miRNAs from 1,5% of the human
genome it is likely that many more miRNAs are to be discovered.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Bart Aelterman (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93425 = 'b&#97;rt.&#97;&#101;lt&#101;rm&#97;n' + '&#64;';
 addy93425 = addy93425 + 'm&#111;lg&#101;n' + '&#46;' + 'v&#105;b-&#117;&#97;' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy93425 + suffix + '\'' + attribs + '>' );
 document.write( addy93425 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy68061 = 'b&#97;rt.&#97;&#101;lt&#101;rm&#97;n' + '&#64;';
 addy68061 = addy68061 + 'm&#111;lg&#101;n' + '&#46;' + 'v&#105;b-&#117;&#97;' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy68061 + suffix + '\'' + attribs + '>' );
 document.write( addy68061 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90265 = 'b&#97;rt.&#97;&#101;lt&#101;rm&#97;n' + '&#64;';
 addy90265 = addy90265 + 'm&#111;lg&#101;n' + '&#46;' + 'v&#105;b-&#117;&#97;' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy90265 + suffix + '\'' + attribs + '>' );
 document.write( addy90265 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Applied Molecular Genomics Group, VIB Department of
Molecular Genetics, University of Antwerp</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Applied Molecular Genomics group, VIB Department of
Molecular Genetics, VIB, Antwerp, Belgium (2) University of Antwerp (UA),
Antwerp, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The authors would like to thank IWT, VIB and UA for
their support. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743775">D-5. CatANalyst: a web server for predicting
catalytic residues</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cilia E<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Cilia E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Passerini A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Passerini A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The identification of the residues involved in the
catalytic processes is a key step for fully characterizing an enzyme function.
Understanding the molecular mechanisms of protein functioning is difficult even
when its 3D structure is known. The process is time consuming and goes through
several hypothesis formulation and verification steps by targeted experiments
(e.g. site-directed mutagenesis). Machine learning approaches can significantly
speed up the whole process by suggesting candidates to be experimentally
verified and potentially allowing automatic annotation of functional residues.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">CatANalyst takes a FASTA sequence or a PDB
structure and builds a residue representation that is fed to an SVM classifier.
Sequential features are based on multiple alignment profiles. Structural
features are extracted from spherical regions around residues and include features
like solvent accessibility, relative position on the protein surface, hydrogen
bonds, secondary structure. CatANalyst outputs the probability of being
catalytic for residues exceeding an adjustable threshold. All residues are also
rendered with different colors and sizes, reflecting how likely they are to be
catalytic.</p>

<h3>Results</h3>

<p class="abresults">The structured-based CatANalyst was previously shown to
achieve consistent improvements over existing approaches on several benchmark datasets.
The online version of the predictors was trained on a dataset of 137,116
residues of 505 enzymes from PDBselect and deposited in PDB before 2008.
Results on a test set of 88 newly deposited enzymes are consistent with the
cross-validation results obtained on the benchmark datasets. As a case study we
also tested CatANalyst on a predicted model of an amidase: predictions include
the known catalytic residues and also putative ones reported in literature.</p>

<h3>Discussion</h3>

<p class="abdiscussion">CatANalyst provides both sequence- and structure- based
functional residue predictions. The classifier shows a slight positive bias on
residues having a high catalytic propensity that bind catalytic cofactors.
These residues do not have a direct role in the catalysis according to the CSA
annotation. On the other hand, information on nearby heterogens helps in
identifying functional residues with low catalytic propensities. Planned
extensions include the joint prediction of both catalytic and binding site
residues and the collective prediction of residues belonging to the same catalytic
site.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://catanalyst.disi.unitn.it/" mce_href="http://catanalyst.disi.unitn.it/"><span style="">http://catanalyst.disi.unitn.it/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Elisa Cilia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92430 = '&#101;l&#105;s&#97;.c&#105;l&#105;&#97;' + '&#64;';
 addy92430 = addy92430 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy92430 + suffix + '\'' + attribs + '>' );
 document.write( addy92430 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56951 = '&#101;l&#105;s&#97;.c&#105;l&#105;&#97;' + '&#64;';
 addy56951 = addy56951 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy56951 + suffix + '\'' + attribs + '>' );
 document.write( addy56951 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43872 = '&#101;l&#105;s&#97;.c&#105;l&#105;&#97;' + '&#64;';
 addy43872 = addy43872 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy43872 + suffix + '\'' + attribs + '>' );
 document.write( addy43872 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Fondazione Edmund Mach - Istituto Agrario S. Michele
all'Adige</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Engineering and Computer Science,
University of Trento via Sommarive, 14, I-38123 Trento, Italy {cilia,passerini}@disi.unitn.it
(2) Fondazione Edmund Mach - Istituto Agrario S. Michele all'Adige via Edmund
Mach 1, I-38010 S. Michele all'Adige (TN), Italy 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87439 = '&#101;l&#105;s&#97;.c&#105;l&#105;&#97;' + '&#64;';
 addy87439 = addy87439 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy87439 + suffix + '\'' + attribs + '>' );
 document.write( addy87439 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Provincia Autonoma di Trento</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743776">D-6. Functional characterization of Parkinson by
high-throughput data analysis with l1l2 regularization</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Squillario M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Squillario M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Masecchia S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Masecchia S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Barla A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Barla A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Parkinson disease (PD) is a neurodegenerative disorder
that impairs the motor skills at the onset and the cognitive and the speech
functions successively. Like other neurological diseases, once the first
symptoms appear, a great loss of neurons has already occurred and so far no
clinical tests are able to early diagnose the disease long time before the
first symptoms appear. To develop a reliable clinical test, it is necessary to
understand the disease at a molecular level. In this context, we analyze
microarray data by means of the l1l2 regularization framework.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">From the statistical viewpoint, we use the l1l2
regularization framework, a very interesting method when dealing with
high-throughput data, given its ability of providing sparse predictive models.
Based on soft thresholding, this approach is consistent, multivariate and
capable of dealing with correlated variables. The double loop of cross
validation guarantees an unbiased result. We also characterize the list of
relevant probesets with a functional analysis performed by WebGestalt, a
toolkit that allows for enrichment analysis on the Gene Ontology and other
digital repositories.</p>

<h3>Results</h3>

<p class="abresults">The feature selection analysis results in a 64% prediction
accuracy, associated to a signature composed by 378 selected probesets for the
maximum correlation allowed. The functional analysis shows that many of the
genes are involved in pathways related to the immune system, to some particular
cell basic processes (apoptosis, cell cycle, motility, communication) but also
to the nervous system, to the metabolism and to some diseases/infections.
Moreover, some interesting genes like BCL2, SNCA, CASP1 are selected and a
significant overlap exists with the signature by Scherzer et al.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The reliability of the statistical method is confirmed by
the signature, that includes genes known to be involved in PD (like SNCA) and
genes that are known to be implicated in other neurodegenerative diseases (like
BCL2, CASP1). The signature is also intersecting with the one by Scherzer et
al. (e.g. ST13, BCL11B). The functional characterization of the signature
confirmed that the majority of the genes are included in categories already
known to be affected by Parkinson (metabolism, cell related pathways as signaling
and apoptosis, the nervous system and immune system related pathways).</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Margherita Squillario (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42333 = 'm&#97;rgh&#101;r&#105;t&#97;.sq&#117;&#105;ll&#97;r&#105;&#111;' + '&#64;';
 addy42333 = addy42333 + '&#117;n&#105;g&#101;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy42333 + suffix + '\'' + attribs + '>' );
 document.write( addy42333 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy48376 = 'm&#97;rgh&#101;r&#105;t&#97;.sq&#117;&#105;ll&#97;r&#105;&#111;' + '&#64;';
 addy48376 = addy48376 + '&#117;n&#105;g&#101;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy48376 + suffix + '\'' + attribs + '>' );
 document.write( addy48376 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64372 = 'm&#97;rgh&#101;r&#105;t&#97;.sq&#117;&#105;ll&#97;r&#105;&#111;' + '&#64;';
 addy64372 = addy64372 + '&#117;n&#105;g&#101;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy64372 + suffix + '\'' + attribs + '>' );
 document.write( addy64372 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dipartimento di Informatica, Verona and DISI, Genova</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Dipartimento di Informatica, Universita degli Studi
di Verona (2) Dipartimento di Informatica e Scienze dell'Informazione,
Universita degli Studi di Genova</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743777">D-7. Feature elimination for one-class microRNA
target prediction and gene identification</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Khalifa W<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Khalifa W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Yousef M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yousef M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">Different studies have described the use of two-class
machine learning to predict microRNAs (miRNAs) gene target or gene
identification. We present study using one-class machine learning for miRNA
target discovery and gene identification. We compare one-class to two-class
approaches using a Backward zero-norm for feature selection to Improve model
performance, provide faster and more cost-effective models and gain a deeper
insight into the underlying processes that generated the data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">we design duplex structure and sequence features in
order to represent each example for the learning process. For each feature’s
vector we define a zero-norm to be the non-zero values over the given positive
examples and chose different levels of threshold to determine a feature as
irrelevant. We chose four one class methods to compare for miRNA discovery and
gene identification: Support Vector Machines (SVM), Gaussian, Kmeans and
K-Nearest Neighbor. Also, four two class methods: Random forest, Naïve Bayes,
C4.5 and SVM. The WEKA software was used as implementation of the two-class
classifier.</p>

<h3>Results</h3>

<p class="abresults">Of all the one-class (OC)methods tested based on the all
features, we found that most of them gave similar accuracy that range from 0.81
to 0.89 while the two-class gave 0.93-0.99 accuracy. the zero-norm feature
selection with the OC improves the performance dramatically and makes the OC
compete with two-class. It is clear from the accuracy plot of OC classifiers
with different zero-norm thresholds that the performance is improving as the
number of features is decreasing until a specific level. Interestingly, using
zero-norm feature selection improves the results to reach accuracy of 0.96</p>

<h3>Discussion</h3>

<p class="abdiscussion">The current results show that it is possible to build up
a classifier based only on positive examples yielding a reasonable performance.
Moreover, using the zero-norm feature selection with the one-class approaches
is essential to improve the performance to reach the two-class performance.
Clearly the one-class is more sensitive to non-relevant features than the
two-class. It is known that obtaining a biological data is an expensive
process. The successful of the one-class classifier over a biological data can
reduce the cost of generat-ing a such data by not requiring the control data.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Waleed Khalifa (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24718 = 'khw&#97;l&#105;d' + '&#64;';
 addy24718 = addy24718 + 'h&#111;tm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy24718 + suffix + '\'' + attribs + '>' );
 document.write( addy24718 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy80514 = 'khw&#97;l&#105;d' + '&#64;';
 addy80514 = addy80514 + 'h&#111;tm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy80514 + suffix + '\'' + attribs + '>' );
 document.write( addy80514 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63282 = 'khw&#97;l&#105;d' + '&#64;';
 addy63282 = addy63282 + 'h&#111;tm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy63282 + suffix + '\'' + attribs + '>' );
 document.write( addy63282 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Institute of Applied Research- the Galilee
Society</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1.The Institute of Applied Research- the Galilee
Society, P.O. Box 437 Shefa Amr,ZIP 20200, Israel 2. Computer Science, The
College of Sakhnin, Sakhnin, ZIP30810, Israel. 3. Al-Qasemi Academic College,
Baqa Algharbiya, ZIP 30100, Israel.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project is funded in part under a grant with the
ESHKOL scholarship program of the ministry of science culture and sport to
support KW. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743778">D-8. Computational identification of synonymous
SNPs in the human genome and their potential role in disease</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wood L<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Wood L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Ramsay M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ramsay M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The potential phenotypic effects of synonymous SNPs
(sSNPs) have long been overlooked. Although several sSNPs are no longer thought
to be silent no one has identified which sSNPs may contribute to disease
phenotypes on a genome-wide scale, nor has a tool been developed that may
identify a functional role for sSNPs in disease. Using available bioinformatics
tools we have predicted the potential functional impact of many sSNPs within
the human genome, and shown that more sSNPs could contribute to phenotypic
variation than originally thought. The analysis tool will aid in the discovery
of potential phenotypic effects of sSNPs within genes associated with disease.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Galaxy was used to retrieve all synonymous SNPs
(sSNPs) from dbSNP130. Python scripts were written to calculate the change in
codon-usage frequency caused by each sSNP. sSNPs that could potentially cause a
change in mRNA secondary structure will be identified using RNAfold and
RNAdist. ESEfinder 3.0 and MaxEntScan were used to determine which sSNPs could
potentially result in aberrant splicing. BioMart was used to retrieve genes
associated with disease from OMIM. sSNPs predicted to have a potential
functional impact were then assessed to determine whether they were within
genes previously associated with disease. MySQL will be used to create a
searchable on-line database and analysis tool.</p>

<h3>Results</h3>

<p class="abresults">Of the 72 504 synonymous SNPs (sSNPs) identified
approximately 70% had no predicted change in function. The remaining sSNPs were
predicted to have a potential functional impact by one or more of the molecular
mechanisms. 35% of the sSNPs were predicted to cause a significant change in
codon-usage frequency, whereas approximately 13% of the sSNPs were predicted to
cause aberrant splicing. sSNPs that could potentially cause a change in mRNA
secondary structure are currently being identified. 20% of the sSNPs were
predicted to be within genes associated with disease, many of which were
predicted to contribute to at least one of the molecular mechanisms.</p>

<h3>Discussion</h3>

<p class="abdiscussion">sSNPs that cause a change in codon-usage frequency or
mRNA secondary structure may alter translational-kinetics and protein folding. In
addition, sSNPs that disrupt splice-site consensus sequences may cause aberrant
splicing, changing the protein product. Thus, a sSNP that contributes to any of
these molecular mechanisms may cause a change in protein structure and
function. Of the sSNPs published within dbSNP130, about 30% were predicted to
have a potential functional impact, many of which were within genes associated
with disease. Thus, more sSNPs may have a functional impact than originally
thought and the potential role of sSNPs in disease should therefore not be
underestimated or neglected.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Lee-Ann Wood (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99281 = '123456.w&#111;&#111;d' + '&#64;';
 addy99281 = addy99281 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy99281 + suffix + '\'' + attribs + '>' );
 document.write( addy99281 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy20562 = '123456.w&#111;&#111;d' + '&#64;';
 addy20562 = addy20562 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy20562 + suffix + '\'' + attribs + '>' );
 document.write( addy20562 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94773 = '123456.w&#111;&#111;d' + '&#64;';
 addy94773 = addy94773 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy94773 + suffix + '\'' + attribs + '>' );
 document.write( addy94773 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Wits Bioinformatics, University of the Witwatersrand,
Johannesburg, South Africa</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1). Wits Bioinformatics, University of the
Witwatersrand, Johannesburg, South Africa (2). Division of Human Genetics,
National Health Laboratory Service, School of Pathology, University of the
Witwatersrand, Johannesburg, South Africa</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Cape Biotech and National Research Foundation</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743779">D-9. The potential functional role of chimeric
transcripts in higher eukaryotes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Frenkel Morgenstern M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Frenkel Morgenstern M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Valencia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valencia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Chimeric RNAs which are produced by trans-splicing of two
or more distinct transcripts, have been reported in different organisms
including fruit fly, mouse, and human [1,2]. Thousands of chimeric transcripts
were identified among the variety of ESTs for these organisms. In theory, these
transcripts can greatly contribute to the complexity of the transcriptome and
proteome of organisms [1, 2], and nowadays are validated by the new generation
sequencing technologies. However, a question of how the trans-splicing
machinery if exists can be used to design new functional proteins remains
unclear.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have studied the potential functional role of
the chimeric proteins in human and mouse. For this purpose, we annotated both
of the two proteins participating in creation of the chimeric transcripts
described by Li et al [2].</p>

<h3>Results</h3>

<p class="abresults">We found that chimeras might integrate the transmembrane
domains of some membrane proteins in their formation. Moreover, chimeras seem
to contain full exons of the translated proteins, where the activation domain
of one protein might be missing, but DNA binding domain remains in the
resulting chimeric protein. In addition, for proteins, which function as
dimers, the functional domains sometimes are missing, but dimerization domain
is still present. Chimeras also integrate signal peptides from the proteins
forming the chimera that can allow changes in its localization.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To conclude, it seems that chimeric RNAs resulting from
trans-splicing, if they are translated to proteins, could have a functional
role suggested by our analysis, and may produce in some cases the dominant
negative effect in the cells.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Milana Frenkel-Morgenstern (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy59997 = 'mm&#111;rg&#101;nst&#101;rn' + '&#64;';
 addy59997 = addy59997 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy59997 + suffix + '\'' + attribs + '>' );
 document.write( addy59997 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71943 = 'mm&#111;rg&#101;nst&#101;rn' + '&#64;';
 addy71943 = addy71943 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy71943 + suffix + '\'' + attribs + '>' );
 document.write( addy71943 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5006 = 'mm&#111;rg&#101;nst&#101;rn' + '&#64;';
 addy5006 = addy5006 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy5006 + suffix + '\'' + attribs + '>' );
 document.write( addy5006 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Spanish National Research Centre (CNIO)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Spanish National Research Centre (CNIO), Madrid,
Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The work of Milana Frenkel-Morgenstern is supported by
the CNIO-Caja Navarra International Postdoctoral fellowships. References: [1]
Gingeras TR, Implications of chimeric non-co-linear transcripts, 2009, Nature,
461, pp 206-211. [2] Li X, Zhao L, Jiang H, Wang W, Short homologous sequences
are strongly associated with the generation of chimeric RNAs in eukaryotes,
2009, J Mol Evol, 68:56-65. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743780">D-10. A "Functional Signature" for
analysing annotation space</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">del Pozo A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;del Pozo A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*) , Tress M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tress M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Valencia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valencia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The hypotheses generated by homology-based functional
annotation methods are formulated under the assumption that similar sequences
have evolved from common ancestors and that functional roles are likely to have
been conserved. Many studies have attempted to establish pairwise sequence
identity cutoffs that guarantee homology. What is missing from these analyses
is an exhaustive investigation of protein annotation space. Here we describe
the current state of functional annotations in protein sequence databases with
emphasis on those cases where reliable functional transfer should be possible</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We selected a sample set of Uniref90 entries and
obtained a subset of 6385 homologous clusters with at least one confident
UniprotKB sequence. We used this subset to study the associated ’annotation
space’ as snapshot of the more complex and generalized ’functional space’. We
calculated a 'Functional Signature' for each cluster based on the quality of
the annotations, the density of the local annotation space and the consensus of
the functional terms (GO terms and EC numbers) with which the member sequences
of each cluster are annotated.</p>

<h3>Results</h3>

<p class="abresults">The 'Functional Signatures' generated for each cluster
allowed us to define the annotation space of the clusters from the Uniref90
sample set. Over a third of the clusters were annotated with high quality
functional terms and 20% of the clusters had at least one unannotated member,
which indicated that homology-based functional annotation was possible in an
important percentage of the clusters.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Here we have introduced a new term, the 'Functional
Signature' as a means of analysing the state of the annotation space of protein
sequences. The analysis suggests that annotation space is unevenly populated
with a large number of gaps that will need to be filled by non-homology based
methods. For those clusters that are annotated with good quality functional
annotations the functional information is not homogeneous, which shows that
methods for homology-based functional transfer are still relevant.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Angela del Pozo (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46866 = '&#97;d&#101;lp&#111;z&#111;' + '&#64;';
 addy46866 = addy46866 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy46866 + suffix + '\'' + attribs + '>' );
 document.write( addy46866 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82716 = '&#97;d&#101;lp&#111;z&#111;' + '&#64;';
 addy82716 = addy82716 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy82716 + suffix + '\'' + attribs + '>' );
 document.write( addy82716 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86770 = '&#97;d&#101;lp&#111;z&#111;' + '&#64;';
 addy86770 = addy86770 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy86770 + suffix + '\'' + attribs + '>' );
 document.write( addy86770 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Structural Bioinformatics Group, Spanish National
Cancer Research Centre (CNIO)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Structural Bioinformatics Group, Spanish National
Cancer Research Centre (CNIO). Madrid, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">ENFIN (Enabling System Biology). European Network of
Excellence with contract number LSHG-CT-2005-518254 funded by the EU within its
FP6 Programme. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743781">D-11. Predicting small molecule ligand binding
sites and catalytic residues with firestar/FireDB</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">López G<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;López G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Maietta P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Maietta P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), del Pozo A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;del Pozo A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rodríguez JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rodríguez JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Valencia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valencia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Tress M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tress M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tress ML&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Genome sequencing projects have lead to a surge in the
number of protein sequences that lack experimental functional data.
Experimental approaches for function characterization are expensive and
difficult to automate and this has meant that researchers have turned
increasingly to computational methods to try to close the gap between the
number of new unannotated sequences and the number of sequences with known
function. Frequently the most interesting functional information can be found
at the level of the amino acid residues implicated in molecular interactions,
catalysis or regulation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The basic principle of the pipeline is the
homology-based transfer of validated functional residues, supported by local
sequence conservation. Previous versions of firestar required human
interpretation of the results. Now the whole process has been automatized and a
new web interface has been made available. Likely biological ligands have been
separated from ligands present only as “contaminants”. Additionally our in
house automatic pipeline is able to produce high quality results in
high-throughput mode, which has allowed firestar to be incorporated into
several prediction pipelines.</p>

<h3>Results</h3>

<p class="abresults">We carried out testing of the server during the CASP8
experiment. We made predictions for the ligand binding prediction category, in
which predictors have to predict residues in contact with small ligands.
Although we were not allowed to compete officially in this category, testing
showed that firestar would have been the best predictor in CASP8. The method
was able to correctly detect 95% of the binding sites from the manually curated
datasets. We have made improvements to the server accuracy for CASP9, in which
firestar is currently participating and we hope to repeat the result from
CASP8.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We are currently using firestar to annotate functional
residues for the human genome. The server is predicting functional residues for
all annotated splice variants in order to evaluate whether functional residues
are lost as a result of alternative splicing. In addition firestar has allowed
us to analyse functional sites in protein coding genes, focusing on metal
binding sites and cofactor binding sites. Finally we hope to extend reliable
annotations from the Catalytic Site Atlas to all proteins in the human genome.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://firedb.bioinfo.cnio.es" mce_href="http://firedb.bioinfo.cnio.es"><span style="">http://firedb.bioinfo.cnio.es</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michael L Tress (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6811 = 'mtr&#101;ss' + '&#64;';
 addy6811 = addy6811 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy6811 + suffix + '\'' + attribs + '>' );
 document.write( addy6811 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56119 = 'mtr&#101;ss' + '&#64;';
 addy56119 = addy56119 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy56119 + suffix + '\'' + attribs + '>' );
 document.write( addy56119 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10509 = 'mtr&#101;ss' + '&#64;';
 addy10509 = addy10509 + 'cn&#105;&#111;' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy10509 + suffix + '\'' + attribs + '>' );
 document.write( addy10509 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Structural Biology and Biocomputing Programme,
Spanish National Cancer Research Centre (CNIO)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Structural Biology and Biocomputing Programme,
Spanish National Cancer Research Centre (CNIO), C./ Melchor Fernandez Almagro,
Madrid, Spain. (2) Spanish National Bioinformatics Institute (INB), Spanish
National Cancer Research Centre (CNIO), C./ Melchor Fernandez Almagro, Madrid,
Spain. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743782">D-12. Classifying substrate specificities of
membrane transporters from Arabidopsis thaliana</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schaadt NS<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Schaadt NS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Christoph J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Christoph J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Helms V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Helms V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The transport of molecules across biological barriers
such as lipid membranes is catalyzed by integral transmembrane proteins which
act either as passive channels or as active carriers selective for a particular
substrate or substrate group. Although the amino acid sequences of many
putative membrane transporters are known, their specific functions remain often
unknown due to the large experimental effort involved in determining substrate
specificities. Therefore, it is highly desirable to develop computational
methods that may aid in the substrate annotation of membrane transport
proteins.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For all gene products from Arabidopsis thaliana
that are annotated in the Aramemnon database as membrane transporters, we
constructed subsets of those transporter sequences that are annotated to
transport either amino acids, oligopeptides, phosphates, or hexoses. We ranked
all transporters according to their similarity to the considered substrate
class. The similarity was estimated by the Euclidean distance between the amino
acid frequency of the transporters and the average value of the substrate
class.</p>

<h3>Results</h3>

<p class="abresults">Substrate classification based on the simple amino acid
frequency had an accuracy of 75% or higher. Integrating higher-order sequence
information, the frequencies of amino acid pairs, or from profile analysis
further improved the prediction performance to 80-90%. For comparison, the accuracy
for randomly compiled substrate classes was only around 30%.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed a novel computational approach to predict
substrate specificities of membrane transport proteins from their amino acid
sequence. We found that membrane transporters belonging to different
transporter families but transport the same substrate have a significantly
higher similarity of their amino acid frequencies that random pairs of
transporters. Thus, the amino acid characteristics of membrane transporters
appear related to the physicochemical properties of the transported substrates.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Nadine S. Schaadt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9848 = 'nsch&#97;&#97;dt' + '&#64;';
 addy9848 = addy9848 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9848 + suffix + '\'' + attribs + '>' );
 document.write( addy9848 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55248 = 'nsch&#97;&#97;dt' + '&#64;';
 addy55248 = addy55248 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy55248 + suffix + '\'' + attribs + '>' );
 document.write( addy55248 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12402 = 'nsch&#97;&#97;dt' + '&#64;';
 addy12402 = addy12402 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy12402 + suffix + '\'' + attribs + '>' );
 document.write( addy12402 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Bioinformatics, Saarland University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Center for Bioinformatics, Saarland University</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743783">D-13. Molecular modeling and comparison by docking
and molecular dynamics of the interaction between fatty acids and proteins and
EgFABP2 EgFABP1</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Paulino Z-M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Paulino Z-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Esteves A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Esteves A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The fatty acid binding proteins (FABPs) are cytosolic
proteins of low molecular weight (14-15 kDa) and very abundant. They are able
to bind fatty acids and other small hydrophobic molecules. Several types of
tissue-specific FABPs have been identified in vertebrates and more recently, in
two FABPS - EgFABP1 and EgFABP2 - in the flatworm parasite Echinococcus
granulosus. Both proteins have a similarity of 96% at amino acid level and a
significant sequence similarity with mammalian members of the subfamily of
cardiac FABPs.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In determining whether there are functional
differences between the two parasite proteins, and considering that the
functional specificity could be linked to differential ligand affinities, we
have initiated the in silico analysis of the binding energies of each protein
by different ligands (arachidonic, palmitic, oleic and linoleic acids), using
the software package MOE 2007 -2009.</p>

<h3>Results</h3>

<p class="abresults">The energetic docking results (scores) were not conclusive,
giving similar interaction energies for all ligands. Then, the optimal energy
structures were taken as the starting point of 4 nanoseconds molecular dynamics
procedures in which the solvent was included explicitally and a full
flexibility allowed. After MD simulation, all results were averaged and the
standard deviations evaluated. All results showed low standard deviations,
being the differences between different ligands, significants.</p>

<h3>Discussion</h3>

<p class="abdiscussion">EgFABP1 showed statistically better interaction energy
with arachidonic acid by the other ligands, while EgFABP2 showed better
interaction energy for oleic acid. Three main contacts of carboxylic head of
all acids were made with side chains of the conserved residues Tyr 129 Arg 107
and Arg 127. Variations between different affinities are given through the
effect of hydrogen bonding of those residues as well as a water cluster around
the polar head of ligands. This observation evidencenced the importance of the
influence of solvent in the network of contacts to strengthen the interaction.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.fq.edu.uy" mce_href="http://www.fq.edu.uy"><span style="" mce_style="mso-no-proof:
yes">http://www.fq.edu.uy</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Margot Paulino Zunini (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99372 = 'm&#97;rg&#111;t' + '&#64;';
 addy99372 = addy99372 + 'fq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#117;y';
 document.write( '<a ' + path + '\'' + prefix + addy99372 + suffix + '\'' + attribs + '>' );
 document.write( addy99372 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46681 = 'm&#97;rg&#111;t' + '&#64;';
 addy46681 = addy46681 + 'fq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#117;y';
 document.write( '<a ' + path + '\'' + prefix + addy46681 + suffix + '\'' + attribs + '>' );
 document.write( addy46681 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27057 = 'm&#97;rg&#111;t' + '&#64;';
 addy27057 = addy27057 + 'fq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#117;y';
 document.write( '<a ' + path + '\'' + prefix + addy27057 + suffix + '\'' + attribs + '>' );
 document.write( addy27057 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Facultad de Química</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. LaBioFarMol, DETEMA, Facultad de Química, UdelaR,
General Flores 2124, 11600, Montevideo, Uruguay 2. Sección Bioquímica,
Instituto de Biología, Facultad de Ciencias UdelaR, Iguá 4225, Montevideo,
Uruguay</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Agencia National de Investigación e Innovación (ANII).
Uruguay. Programa de Desarrollo de Ciencias Básicas (PEDECIBA), Uruguay. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743784">D-14. GreenPhylDB version 2: web resources for
comparative and functional genomics in plants</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rouard M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Rouard M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Guignon V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Guignon V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Aluome C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aluome C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Laporte MA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laporte MA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Droc G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Droc G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Walde C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Walde C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Zmasek CM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zmasek CM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(3), Perin C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Perin C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Conte MG<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Conte MG&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">With the increasing number of plant genomes being
sequenced, a major objective is to transfer annotation between genome models
and other species of interest. Orthology inference is one of the most reliable
strategies for functional annotation</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">GreenPhylDB contains gene families being annotated,
computational analyzes and external references (e.g. InterPro, KEGG,
Swiss-Prot, Pubmed) related to all sequences. Once manually annotated (i.e.
properly named and classified), gene families are finally processed by
phylogenetic analyses to distinguish orthologous and paralogous gene.</p>

<h3>Results</h3>

<p class="abresults">The website offers a range of user-friendly tools to query
the data. These resources will be particularly helpful to molecular biologist
for gene discovery and gene function inference.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A better understanding of genome evolution will
contribute to elucidate the molecular basis of important agronomic traits and
therefore facilitate ongoing plant breeding efforts.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://greenphyl.cirad.fr" mce_href="http://greenphyl.cirad.fr"><span style="" mce_style="mso-no-proof:
yes">http://greenphyl.cirad.fr</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Mathieu Rouard (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92797 = 'm.r&#111;&#117;&#97;rd' + '&#64;';
 addy92797 = addy92797 + 'cg&#105;&#97;r' + '&#46;' + '&#111;rg';
 document.write( '<a ' + path + '\'' + prefix + addy92797 + suffix + '\'' + attribs + '>' );
 document.write( addy92797 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14741 = 'm.r&#111;&#117;&#97;rd' + '&#64;';
 addy14741 = addy14741 + 'cg&#105;&#97;r' + '&#46;' + '&#111;rg';
 document.write( '<a ' + path + '\'' + prefix + addy14741 + suffix + '\'' + attribs + '>' );
 document.write( addy14741 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17321 = 'm.r&#111;&#117;&#97;rd' + '&#64;';
 addy17321 = addy17321 + 'cg&#105;&#97;r' + '&#46;' + '&#111;rg';
 document.write( '<a ' + path + '\'' + prefix + addy17321 + suffix + '\'' + attribs + '>' );
 document.write( addy17321 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioversity International</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Bioversity International - CfL programme Parc
Scientifique Agropolis II, 34397 Montpellier, France 2 CIRAD, Department BIOS,
UMR DAP - TA40/03, 34398 Montpellier, France 3 Sanford-Burnham Medical Research
Institute, 10901 North Torrey Pines Road, La Jolla, CA 92037, USA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Generation Challenge Programme (GCP) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743785">D-15. Collaboration-based function prediction in
protein-protein interaction networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rahmani H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Rahmani H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Blockeel H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blockeel H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Bender A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bender A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Existing techniques assume that proteins that are
topologically close in the network tend to have similar functions. We
hypothesize that better predictive accuracy can be obtained by generalizing
this assumption. We call two functions collaborative if proteins with one
function often interact with proteins performing the other function. Our
hypothesis is that techniques that extract such function collaboration
information from networks, and exploit it, can yield better predictions. We
propose and evaluate two such techniques.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We propose two methods implementing the function
collaboration hypothesis. In the first method, first we extract the
collaborative function pairs from the whole network. Then, in order to make a
prediction for an unclassified protein, we extract the candidate functions
based on the position of the protein in the network. Finally, we calculate the
score of each candidate function. High score candidate functions are those
which collaborate more with the neighborhood of unclassified protein. The
second method adopts Self Organizing Map (SOM) for modeling the function
collaboration in PPI networks</p>

<h3>Results</h3>

<p class="abresults">We selected two methods, Majority Rule and Functional
Clustering, as representatives of the similarity based approaches. We compared
our collaboration based methods with these similarity based methods on three
interaction datasets: Krogan, DIP-Core and VonMering. We examined up to five
different function levels and we found classification performance according to
F-measure values indeed improved, sometimes by up to 17 percent, over the
benchmark methods employed.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To our knowledge, this is the first study that considers
function collaboration for the task of function prediction in PPI networks. We
view biological process as an aggregation of individual protein functions and
our hypothesis is that topologically close proteins have collaborative functions.
We propose two methods that predict protein functions based on function
collaboration. We perform a comprehensive set of experiments that reveal a
significant improvement (ranging from 3% to 17%) of F-measure values compared
to existing methods.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Hossein Rahmani (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy85226 = 'hr&#97;hm&#97;n&#105;' + '&#64;';
 addy85226 = addy85226 + 'l&#105;&#97;cs' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy85226 + suffix + '\'' + attribs + '>' );
 document.write( addy85226 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71692 = 'hr&#97;hm&#97;n&#105;' + '&#64;';
 addy71692 = addy71692 + 'l&#105;&#97;cs' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy71692 + suffix + '\'' + attribs + '>' );
 document.write( addy71692 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61193 = 'hr&#97;hm&#97;n&#105;' + '&#64;';
 addy61193 = addy61193 + 'l&#105;&#97;cs' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy61193 + suffix + '\'' + attribs + '>' );
 document.write( addy61193 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Leiden Institute of Advanced Computer Science</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Leiden Institute of Advanced Computer Science,
Universiteit Leiden Niels Bohrweg 1, 2333 CA Leiden, The Netherlands (2)
Department of Computer Science, Katholieke Universiteit Leuven, Celestijnenlaan
200A, 3001 Leuven, Belgium (3) Division of Medicinal Chemistry,
Leiden/Amsterdam Center for Drug Research, Universiteit Leiden, 2333 CC Leiden,
The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is funded by the Dutch Science Foundation
(NWO) through a VIDI grant. Andreas Bender was funded by the Dutch Top
Institute Pharma, project number: D1-105. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743786">D-16. SVM feature selection applied to
lignocellulose degradation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Trukhina Y<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Trukhina Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), McHardy A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;McHardy A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The study of the lignocellulose degradation process is an
area of substantial scientific interest, not only because of the general
scientific importance of this topic, but also because of the possible
commercial and ecological benefits from controllable biodegradation of
cellulosic waste.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this work we investigate this problem, and more
generally, phenotype prediction from genomic data, which is one of the most
important tasks in computational genomics, with an approach based on support
vector machine (SVM) models. More specifically, we apply feature selection with
an L1-regularized linear SVM to our data set, which we created based on
information provided by the IMG/M database.</p>

<h3>Results</h3>

<p class="abresults">Our approach results in a good phenotype prediction accuracy
and a meaningful set of selected features.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our results are consistent with the current knowledge of
microbial cellulose degradation.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Yulia Trukhina (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72665 = 'tr&#117;kh&#105;n&#97;' + '&#64;';
 addy72665 = addy72665 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy72665 + suffix + '\'' + attribs + '>' );
 document.write( addy72665 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14024 = 'tr&#117;kh&#105;n&#97;' + '&#64;';
 addy14024 = addy14024 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy14024 + suffix + '\'' + attribs + '>' );
 document.write( addy14024 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9569 = 'tr&#117;kh&#105;n&#97;' + '&#64;';
 addy9569 = addy9569 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9569 + suffix + '\'' + attribs + '>' );
 document.write( addy9569 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Computational Genomics and Epidemiology Group, Max
Planck Institute for Computer Science, Saarbruecken, Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Computational Genomics and Epidemiology Group, Max
Planck Institute for Computer Science, Campus E1 4, 66123 Saarbruecken, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Max-Planck Society</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743787">D-17. Substrate-specific HMMs for the
classification of adenylation and acyltransferase domains of NRPS/PKS systems</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Khayatt B I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Khayatt B I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Siezen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3), Francke C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Francke C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">NRPS/PKS systems produce bioactive peptides and
polyketides in a variety of microbes and plants. The molecules range from
antibiotics to kill competitors (e.g. penicillin and erythromycin) to
surfactants produced to escape a biofilm environment (e.g. surfactin). The
ability to identify substrate specificity of the adenylation (A) and
acyltransferase (AT) domains will be very helpful in the rapid discovery of new
products of these systems.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">NCBI, UniProt and NRPS/PKS-specialized databases were
used to retrieve A and AT sequences of experimentally verified NRPSs and PKSs.
The retrieved sequences were aligned and the residues relating to substrate
specificity were extracted. Using these residues, substrate-specific Hidden
Markov Models were generated and applied and ‘the recognition power’ of these
different A and AT substrate models was evaluated.</p>

<h3>Results</h3>

<p class="abresults">We collected a set of 469 A and 164 AT domain sequences and
determined the substrate-specific residues by splitting the set on basis of substrate
specificity after alignment. We found that there was more variation in the AT
domains. An initial classification procedure using Neighbor Joining trees, as
applied before by others, appeared to limit the power of recognition to around
85%. We therefore created substrate-specific HMMs and tested whether these
would improve performance. Indeed, using these models, the recognition power
improved to 95% and 99% for A and AT domains, respectively.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We found that the substrate specificity of the A and AT
domains is largely captured in a specific set of domain residues and highly
specific HMMs could be generated. The methodology to generate
substrate-specific HMMs will be helpful in detecting substrate specificity of
the sub-classes of other enzymes that have so far no characterized structural
models as references for residue numbering.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Barzan I. Khayatt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37036 = 'b.kh&#97;y&#97;tt' + '&#64;';
 addy37036 = addy37036 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy37036 + suffix + '\'' + attribs + '>' );
 document.write( addy37036 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38742 = 'b.kh&#97;y&#97;tt' + '&#64;';
 addy38742 = addy38742 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy38742 + suffix + '\'' + attribs + '>' );
 document.write( addy38742 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90082 = 'b.kh&#97;y&#97;tt' + '&#64;';
 addy90082 = addy90082 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy90082 + suffix + '\'' + attribs + '>' );
 document.write( addy90082 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Radboud University Nijmegen Medical Centre, NCMLS,
CMBI</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Center for Molecular and Biomolecular Informatics
(CMBI) (260), NCMLS, Radboud University Nijmegen Medical Center, PO Box 9101,
6500HB Nijmegen, The Netherlands 2) NIZO food research, PO Box 20, 6710BA Ede,
The Netherlands 3) TI Food and Nutrition, Kluyver Centre for Genomics of
Industrial Fermentation and Netherlands Bioinformatics Centre, PO Box 557,
6700AN Wageningen, The Netherlands</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743788">D-18. Stochastic modeling of proteolytic activity
from LC-MS data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dittwald P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Dittwald P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Gambin A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gambin A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ostrowski J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ostrowski J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2) , Karczmarski J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Karczmarski J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The motivation for model is a hypothesis that the
activity of specific enzymes present in the serum samples of patients and
healthy donnors differ significantly giving the possibility to distinguish
colorectal cancer and healthy samples.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach integrate the existing knowledge about
proteases’ activity stored in MEROPS database with the efficient procedure for
estimation the model parameters. The proteolytic activity is modelled with the
use of Chemical Master Equation. Assuming the stationarity of the Markov
process we calculate the expected values of digested peptides in the model. The
parameters are fitted to minimize the discrepancy between those expected
amounts and the peptide intensities observed in the tandem mass spectrometry
data. Constrained optimization problem is solved by Levenberg-Marquadt
algorithm.</p>

<h3>Results</h3>

<p class="abresults">We tested our approach on 39 colorectal cancer and healthy
control subjects. The set of proteolytic enzymes probably involved in the
disease process has been identified. Using these enzymatic activities occur to
be meaningful in discriminating between healthy and colorectal cancer samples.
Our preliminary resuts show the efficiency and accuracy of the estimation
procedure.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The model significantly extends those proposed recently
in [Klu08]. In the current approach the parameters of the model inferred from
LC-MS/MS correspond directly to the activity of specific enzymes present in the
serum samples of patients and healthy control subjects. The scanning of
biological literature show that many of enzymes identified by our approach are
regarded to be involved in colorectal cancer process. Bibliography: [Klu08]
Kluge, B., Gambin, A. &amp; Niemiro, W., Modeling Exopeptidase Activity from LC-MS
Data (2009), Journal of Computational Biology, Vol.16, No.2, Pp.395-406</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioputer.mimuw.edu.pl/papers/proteolysis/" mce_href="http://bioputer.mimuw.edu.pl/papers/proteolysis/"><span style="">http://bioputer.mimuw.edu.pl/papers/proteolysis/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Piotr Dittwald (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy318 = 'p&#105;&#111;tr.d&#105;ttw&#97;ld' + '&#64;';
 addy318 = addy318 + 'st&#117;d&#101;nts' + '&#46;' + 'm&#105;m&#117;w' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy318 + suffix + '\'' + attribs + '>' );
 document.write( addy318 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38022 = 'p&#105;&#111;tr.d&#105;ttw&#97;ld' + '&#64;';
 addy38022 = addy38022 + 'st&#117;d&#101;nts' + '&#46;' + 'm&#105;m&#117;w' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy38022 + suffix + '\'' + attribs + '>' );
 document.write( addy38022 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10644 = 'p&#105;&#111;tr.d&#105;ttw&#97;ld' + '&#64;';
 addy10644 = addy10644 + 'st&#117;d&#101;nts' + '&#46;' + 'm&#105;m&#117;w' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy10644 + suffix + '\'' + attribs + '>' );
 document.write( addy10644 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Informatics, University of Warsaw</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1: Institute of Informatics, University of Warsaw,
Banacha 2, 02-097 Warsaw, Poland 2: Department of Gastroenterology and
Hepatology, Medical Center for Postgraduate Education at the Maria
Sklodowska-Curie Memorial Cancer Center, Institute of Oncology, 02-781 Warsaw,
Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is supported in part by Polish Ministry
of Science and Educations grants N301 065236 and N206 356036. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743789">D-19. Mining posttranscriptional regulatory
features by comparing total and polysomal mRNA profilings</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Re A<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Re A&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*),
Tebaldi T<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Tebaldi T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Segata N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Segata N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Passerini A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Passerini A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Blanzieri E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blanzieri E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Quattrone A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Quattrone A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Posttranscriptional events such as export, transcript
stability and translational regulation contribute substantially to the final
protein readout. They underlie the disparity recurrently noted between the
steady-state levels of the mRNAs and those of the proteins. Little is known on how
often each mRNA is sensitive to posttranscriptional activities and how these
activities influence mRNAs. The aim of this study is to quantify the extent of
posttranscriptional control and to identify the relevant features, from the
general ones providing aspecific control to those regulating specific mRNAs.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We combined total and polysome-bound RNA profilings
in eight conditions in order to systematically estimate by what degree the
total and polysomal fractions of an mRNA differ. As the mRNAs actively engaged
in polysomes integrate all the events contributing to the rate of protein
synthesis, they have been used as a proxy for newly synthesized proteins. We
subsequently computed to what extent general and mRNA-specific, structural and
sequential features of 5’ and 3’ UTRs of expressed mRNAs enable to characterize
classes of mRNAs with different translational efficiency estimated as above.</p>

<h3>Results</h3>

<p class="abresults">From a systems-level point of view quantifying the mRNA
engagement with posttranscriptional control uncovers distinct populations. A
general trend among conditions seems to indicate a higher informativity of
3'UTR features with respect to 5'UTR ones. This trend was especially evident
for aspecific features like UTRs length, folding propensity and GC-content.
Furthermore mRNA-specific sequential and structural features have been found to
be required for characterizing the mRNA populations. The specific trans-acting
factors recognizing the cis-acting elements have been partially determined.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The combination of total and polysomal mRNA profiling
uncovers a widespread uncoupling between transcriptome and translatome. General
5’ UTR features are less important than suggested to explain the existence of
weakly translated mRNAs. mRNA specific features should be taken into account
and the responsible corresponding trans-acting factors identified, highlighting
the relative contribution of coding and non-coding factors. A better
characterization of post-transcriptional regulation would also require a more
complex modeling of the interaction among multiple regulatory elements.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Angela Re (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95090 = 'r&#101;' + '&#64;';
 addy95090 = addy95090 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy95090 + suffix + '\'' + attribs + '>' );
 document.write( addy95090 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98019 = 'r&#101;' + '&#64;';
 addy98019 = addy98019 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy98019 + suffix + '\'' + attribs + '>' );
 document.write( addy98019 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82586 = 'r&#101;' + '&#64;';
 addy82586 = addy82586 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy82586 + suffix + '\'' + attribs + '>' );
 document.write( addy82586 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Centre for Integrative Biology, University of Trento</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Centre for Integrative Biology, University of
Trento, Trento, Italy (2) Department of Information Engineering and Computer
Science, University of Trento, Trento, Italy</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743790">D-20. Prototypes of elementary functional loops
unravel evolutionary connections between protein functions</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Goncearenco A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Goncearenco A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Berezovsky IN<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Berezovsky IN&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Earlier studies of protein structure revealed closed
loops with a characteristic size 25-30 residues and ring-like shape as a basic
universal structural element of globular proteins. Elementary functional loops
(EFL) have specific signatures and provide functional residues important for
binding/activation and principal chemical transformation steps of the enzymatic
reaction. The goal of this work is to show how these functional loops evolved
from pre-domain peptides, and to find a set of prototypes from which the EFLs
of contemporary proteins originated.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We describe a computational method for deriving
prototypes of elementary functional loops based on the sequences of complete
genomes. The procedure comprises the iterative derivation of sequence profiles
followed by the hierarchical clustering of profiles. We propose a scoring
function that weights profile positions proportional to the information content
on position, allowing to discriminate between matches that carry a specific
signature from non-specific ones. The statistical significance of the scores is
calculated from the empirical distribution of the reshuffled profile scores.</p>

<h3>Results</h3>

<p class="abresults">We developed a computational procedure for deriving
prototypes of EFLs, obtained prototypes from the set of archaeal proteomes,
considered several prototypes in detail, delineated connections between domains
superfamilies using the most abundant prototypes, and exemplified how
combinations of EFLs result in specific enzymatic function.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The existence of EFLs in different folds and functions
makes it possible to survey subtle evolutionary relations, originating from the
pre-domain evolution of protein structure. It suggests that contemporary
enzymatic functions are constructs of different sets and combinations of
elementary chemical functions. Obtaining the full collection of prototypes with
elementary functions will make it possible to (a) predict enzymatic functions
via determining EFLs corresponding to prototypes; (b) (re)design folds with
desired functions by building constructs from necessary elementary functional
loops.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Alexander Goncearenco (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96 = '&#97;l&#101;x&#97;ndr.g&#111;nc&#101;&#97;r&#101;nc&#111;' + '&#64;';
 addy96 = addy96 + '&#117;n&#105;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy96 + suffix + '\'' + attribs + '>' );
 document.write( addy96 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44885 = '&#97;l&#101;x&#97;ndr.g&#111;nc&#101;&#97;r&#101;nc&#111;' + '&#64;';
 addy44885 = addy44885 + '&#117;n&#105;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy44885 + suffix + '\'' + attribs + '>' );
 document.write( addy44885 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65302 = '&#97;l&#101;x&#97;ndr.g&#111;nc&#101;&#97;r&#101;nc&#111;' + '&#64;';
 addy65302 = addy65302 + '&#117;n&#105;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy65302 + suffix + '\'' + attribs + '>' );
 document.write( addy65302 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Bergen, Bergen Center for Computational
Science</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Computational Biology Unit, Bergen Center for
Computational Science; (2) Department of Informatics, University of Bergen,
N-5008 Norway.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">FUGE-II Norwegian functional genomics platform</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743791">D-21. Utilization of proteins and nucleic acids in
the study of gene function: a comparative review</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Mwololo JK<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Mwololo JK&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*), Karaya HG<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Karaya HG&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Munyua JK<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Munyua JK&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(3), Muturi PW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Muturi PW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Munyiri SW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Munyiri SW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Proteomics is one of the fastest growing areas in areas
of research, largely because the global-scale analysis of proteins is expected
to yield more direct understanding of function and regulation than analysis of
genes. Protein structure characterizes its function and a protein sequence that
relates to a known structure forms a basis for identifying gene function.
Proteins are encoded by the genome (genes), and the set of proteins encoded by
the genome, including the added variation of post-translational modification,
constitute the proteome.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The major role of protein structure is to
characterize function. Proteins are encoded by the genome and are involved in
cellular metabolic activities. DNA can provide more information than proteins
due to the degeneracy of the genetic code and the presence of large non-coding
stretches.The proteins are dynamic and interacting molecules, and their state
of instability can make proteomic snapshots difficult. Use of DNA to understand
such complexity is not adequate because problems of redundancy, pseudogenes and
sequencing create inaccuracies. Measuring the intermediate step between genes
and proteins bridges the gap between the genetic code and the functional
molecules that run cells.</p>

<h3>Results</h3>

<p class="abresults">It is clear that proteins are more amenable to prediction of
gene function and therefore proteomics will increase in importance since
analyses of proteins is expected to yield more direct understanding of function
and regulation of genes than would be achieved through analysis of genes
themselves. However, there is need to integrate genomics, transcriptomics and
proteomics to facilitate understanding of normal cell development, function and
response to diseases.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The challenges encountered in identifying the biochemical
and cellular functions of the many gene products which are currently not
characterized has necessitated the use of the proteome. The development of
automated methods for the annotation of predicted gene products with functional
categories is becoming important. Compared to the study of the genetic code
proteomics may allow greater understanding of the complexity of life and the
process of evolution due to the large number of proteins that can be produced
by an individual organism. A major challenge to proteomics is that proteins are
dynamic and interacting molecules and their variability can complicate detailed
studies on gene function.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">James K. Mwololo (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86866 = 'mw&#111;l&#111;l&#111;j&#97;m&#101;s' + '&#64;';
 addy86866 = addy86866 + 'y&#97;h&#111;&#111;' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy86866 + suffix + '\'' + attribs + '>' );
 document.write( addy86866 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51695 = 'mw&#111;l&#111;l&#111;j&#97;m&#101;s' + '&#64;';
 addy51695 = addy51695 + 'y&#97;h&#111;&#111;' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy51695 + suffix + '\'' + attribs + '>' );
 document.write( addy51695 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21421 = 'mw&#111;l&#111;l&#111;j&#97;m&#101;s' + '&#64;';
 addy21421 = addy21421 + 'y&#97;h&#111;&#111;' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy21421 + suffix + '\'' + attribs + '>' );
 document.write( addy21421 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">MAKERERE UNIVERSITY</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1.Makerere University, Faculty of Agriculture, Crop
Science Department, P.O. Box 7062 Kampala, Uganda; 2.International Maize and
Wheat Improvement Centre (CIMMYT), P.O. Box 1041-00621, Nairobi, Kenya; 3.University
of Nairobi, P.O. Box 30197-00100 Nairobi, Kenya.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Support from my co-authors in preparation of this
manuscript. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743792">D-22. ClanTox: A predictor tool for toxin-like
proteins reveals 500 such proteins within viral genomes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Naamati G<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Naamati G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Askenazi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Askenazi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2,3), Linial M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Linial M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2)</p>

<p class="abmotivation">Animal toxins operate by binding to receptors and ion
channels. These proteins are short and vary in sequence, structure and
function. Sporadic discoveries have also revealed endogenous toxin-like
proteins in non-venomous organisms. Viral proteins are the largest group of
quickly evolving proteomes. We tested the hypothesis that toxin-like proteins
exist in viruses and that they act to modulate functions of their hosts.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Data collection: We collected viral proteins from
UniProt. We started with 773,000 sequences which were reduced to 26,000. Vector
construction: Each protein sequence was represented as a vector of 545 sequence
derived numerical features. Among these features we included amino acid
frequencies, length, and specific amino acids and their spacing, with special
consideration for Cysteine. Training sets: We used a training set of 924
sequences listed as ionic channel inhibitors. Learning algorithm: We used a
meta classifier based on a boosted stump algorithm (Adaboost).</p>

<h3>Results</h3>

<p class="abresults">We updated and improved a classifier for proteins resembling
short animal toxins that is based on a machine learning method. We applied
identify toxin-like proteins among short viral proteins. Among the ~26,000
representatives of such short proteins, 510 sequences were positively identified.
We focused on the 19 highest scoring proteins. Our predictor was shown to
enhance annotation inference for many ‘uncharacterized’ proteins. We conclude
that our protocol can expose toxin-like proteins in unexplored niches and
enhance the systematic discovery of novel cell-modulators for drug development.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Clantox identifies short proteins that function as cell
modulators that are beyond the function of toxins. Among the ~500 viral
positive predictions we identified a surprising number of conotoxin-like
proteins. Finding of a large number of conotoxin-like peptides in viruses is
intriguing. Two scenarios may account for this observation: (i) fast evolving
convergent evolution of short secreted proteins(ii) Genetic material exchange
from hosts to viruses. We postulate that some of the sequences identified in
this study carry a potential for drug development.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.clantox.cs.huji.ac.il/" mce_href="http://www.clantox.cs.huji.ac.il/"><span style="">http://www.clantox.cs.huji.ac.il/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Guy Naamati (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97374 = 'g&#117;y.n&#97;&#97;m&#97;t&#105;' + '&#64;';
 addy97374 = addy97374 + 'm&#97;&#105;l' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy97374 + suffix + '\'' + attribs + '>' );
 document.write( addy97374 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61543 = 'g&#117;y.n&#97;&#97;m&#97;t&#105;' + '&#64;';
 addy61543 = addy61543 + 'm&#97;&#105;l' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy61543 + suffix + '\'' + attribs + '>' );
 document.write( addy61543 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76668 = 'g&#117;y.n&#97;&#97;m&#97;t&#105;' + '&#64;';
 addy76668 = addy76668 + 'm&#97;&#105;l' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy76668 + suffix + '\'' + attribs + '>' );
 document.write( addy76668 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Hebrew University of Jerusalem</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Department of Computer Science, and Engineering, The
Hebrew University of Jerusalem, Israel. 2) Department of Biological Chemistry,
The Hebrew University of Jerusalem, Israel. 3) Blais Proteomics Center,
Dana-Farber Cancer Institute, Boston, MA, USA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This study is supported by the grant agency of the ISF
and BSF. G.N is supported by the SCCB, the Sudarsky Center for Computational
Biology. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743793">D-23. I-Patch web service: inter-protein contact
prediction using local network information</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hamer R<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Hamer R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Luo Q<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Luo Q&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Armitage J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Armitage J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Reinert G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinert G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Deane C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Deane C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Krawczyk K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Krawczyk K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*)</p>

<p class="abmotivation">Prediction of inter-protein contact sites and of specificity-determining
residues is of utmost importance for the pharmaceutical industry. Since the set
of specificity-determining residues should be contained within the set of all
contact sites, identifying this set would be a step forward towards identifying
the specificity-determining contact-sites. I-Patch achieves the goal of
identifying contact-sites with 59% precision and 20% recall - better than the
other existing methods.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Given input of 2 PDB files (proteins) and MSA's of
their homologs (ideally with indication of known inter-homolog complexes), the
propensity score for each residue is calculated. The algorithm relies on
propensity data for a single residue, pair of residues or a triple of residues
to be a contact site. The data was derived from a non-redundant set of 677
protein complexes and 1150 proteins with two domains on a single chain,
yielding a set of 1827 distinct proteins. Fitting and blind data sets consisted
of 31 proteins each - sets were non-overlapping with those used to calculate
propensities.</p>

<h3>Results</h3>

<p class="abresults">I-Patch achieves 59% precision with 20% recall on the
non-redundant blind data set of 31 proteins. It performs better than other
existing methods (ELSC, SCA, OMES, MI and McBASC). The best results from
amongst other algorithms on this data set are by EBMcBASC - 36% precision with
20% recall. I-Patch predictions on proteins involved in bacterial chemotaxis
correctly identified 13 out of 36 contact sites. 27 predicted contact sites do
not correspond to any known contact sites, some of them however, seem to lie on
inter-domain interfaces or previously unknown interfaces.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Method for identifying protein contact sites was
developed by treating the protein complex as a network of possible contact
sites. The algorithm performs better than other existing methods on the same
data set and seems to be identifying previously unknown interaction interfaces.
The program is currently available for MATLAB from the project website but a
web-based service will be available by the end of August/early September
(information about which will be available on the project website).</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.stats.ox.ac.uk/research/bioinfo/resources" mce_href="http://www.stats.ox.ac.uk/research/bioinfo/resources"><span style="">http://www.stats.ox.ac.uk/research/bioinfo/resources</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Konrad Krawczyk (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9775 = 'k&#111;nr&#97;d.kr&#97;wczyk' + '&#64;';
 addy9775 = addy9775 + 'dtc' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy9775 + suffix + '\'' + attribs + '>' );
 document.write( addy9775 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60914 = 'k&#111;nr&#97;d.kr&#97;wczyk' + '&#64;';
 addy60914 = addy60914 + 'dtc' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy60914 + suffix + '\'' + attribs + '>' );
 document.write( addy60914 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23348 = 'k&#111;nr&#97;d.kr&#97;wczyk' + '&#64;';
 addy23348 = addy23348 + 'dtc' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy23348 + suffix + '\'' + attribs + '>' );
 document.write( addy23348 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Oxford, Lady Margaret Hall</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Oxford Centre for Integrative Systems Biology,
University of Oxford, Oxford, UK (2) Department of Statistics, University of
Oxford, Oxford, UK (3) Department of Mathematics and Systems Science, National
University of Defense Technology, Changsha, Hunan, China</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743794">D-24. Predicting candidate genes for biomass traits</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hassani-Pak K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Hassani-Pak K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Kuo SC<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kuo SC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Hanley S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hanley S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rawlings C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rawlings C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The phenotype-genotype problem starts with phenotypic
variation and tries to determine which genes are involved. Quantitative trait
loci (QTL) are chromosomal regions identified through linkage analysis that
assign variation observed in a phenotype to a region on a genetic map. Even
well-defined QTL, however, may encompass many potential candidate genes,
perhaps hundreds. It is therefore hard to objectively choose underlying
candidate(s) that drive the phenotype.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We are developing bioinformatics tools to support
systematic analysis of QTL regions and prioritise genes for experimental
analyses. Prioritisation is generally based on evidence that supports the role
of a gene product in the biological process being investigated. The two most
important bodies of information providing such evidence are bioinformatics
databases and the scientific literature.</p>

<h3>Results</h3>

<p class="abresults">In this work, we first present a knowledge base (KB) for the
Poplar genome created by using comparative genomics, data integration and text
mining methods of the freely available Ondex system (www.ondex.org). Poplar
proteins in this KB are enriched with functional, phenotype and literature
information. Second, we demonstrate our novel web-based tool to query the
Poplar KB and analyse a QTL region to identify and prioritise candidate genes
for complex phenotypes such us biomass production.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We present a generic workflow that can be used for the
functional annotation of any newly sequenced genome. Scientists working on functional
genomics or population genetics, searching for new ways to explore their data
and predicting candidate genes, will be interested in our novel visualization
tool which enables users to examine QTLs and explore the underlying networks.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Keywan Hassani-Pak (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36832 = 'k&#101;yw&#97;n.h&#97;ss&#97;n&#105;-p&#97;k' + '&#64;';
 addy36832 = addy36832 + 'bbsrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy36832 + suffix + '\'' + attribs + '>' );
 document.write( addy36832 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53710 = 'k&#101;yw&#97;n.h&#97;ss&#97;n&#105;-p&#97;k' + '&#64;';
 addy53710 = addy53710 + 'bbsrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy53710 + suffix + '\'' + attribs + '>' );
 document.write( addy53710 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38088 = 'k&#101;yw&#97;n.h&#97;ss&#97;n&#105;-p&#97;k' + '&#64;';
 addy38088 = addy38088 + 'bbsrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy38088 + suffix + '\'' + attribs + '>' );
 document.write( addy38088 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Rothamsted Research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Rothamsted Research, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">BBSRC</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743795">D-25. Exploring residue interaction networks to
understand protein structure-function relationships</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Doncheva N-T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Doncheva N-T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Domingues F-S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Domingues F-S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Albrecht M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Albrecht M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Studying the properties of individual amino acid residues
and their pairwise interactions improves the understanding of protein
structure-function relationships. Recent work showed that 2D residue
interaction networks (RINs) derived from 3D protein structures provide new
insights into the role of biologically important residues for protein function.
In particular, RINs are very suitable for characterizing the effect of residue
mutations on protein structure and function, for instance, if the mutated
residue is not located close to the affected protein site.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A RIN is represented as a network in which the
interacting nodes are the amino acid residues and the interaction edges
represent their atomic contacts. We also use a more sophisticated method for
RIN generation that considers non-covalent interactions between main and side
chains of the amino acids and quantifies the strength of these interactions.
For most protein structures deposited in the Protein Data Bank, we have
generated precomputed RINs for download.</p>

<h3>Results</h3>

<p class="abresults">We developed the Cytoscape plugin RINalyzer, which uses UCSF
Chimera to support simultaneous viewing and exploring of the 2D network of
residue interactions and the corresponding 3D protein structure. Among other
visual analytics tools, RINalyzer offers the computation and visualization of a
comprehensive set of centrality measures for discovering critical residues in
the protein structure. Additionally, comparisons between RINs can be performed
by constructing a combined network that allows for investigating the residue
interaction differences between aligned 3D protein structures.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Due to its rich functionality and user-friendly
interface, RINalyzer is a versatile tool that facilitates the usage of RINs for
investigating and understanding complex protein structure-function
relationships.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Nadezhda T. Doncheva (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54153 = 'd&#111;nch&#101;v&#97;' + '&#64;';
 addy54153 = addy54153 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy54153 + suffix + '\'' + attribs + '>' );
 document.write( addy54153 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38936 = 'd&#111;nch&#101;v&#97;' + '&#64;';
 addy38936 = addy38936 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38936 + suffix + '\'' + attribs + '>' );
 document.write( addy38936 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9780 = 'd&#111;nch&#101;v&#97;' + '&#64;';
 addy9780 = addy9780 + 'mp&#105;-&#105;nf' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9780 + suffix + '\'' + attribs + '>' );
 document.write( addy9780 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Computational Biology and Applied
Algorithmics, Max Planck Institute for Informatics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Computational Biology and Applied
Algorithmics, Max Planck Institute for Informatics, Germany</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269743796">D-26. BioXSD: the canonical XML-Schema data model
for bioinformatics web services</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kalaš M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Kalaš M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Puntervoll P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Puntervoll P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Joseph A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Joseph A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Bartaševičiūtė E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bartaševičiūtė E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Töpfer A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Töpfer A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,5), Ison J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ison J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Blanchet C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blanchet C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Rapacki K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rapacki K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Jonassen I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jonassen I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">The wide community of life-scientific groups offers a
huge amount of public bioinformatics data sources and tools. These resources
cover diverse areas of bioinformatics and follow diverse sets of implementation
designs. More and more of the resources provide a standardised programmatic
Web-service interface. However, we are witnessing a burden to smooth
interoperability among the services: a lack of standard input and output data
formats.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">BioXSD has been developed by EMBRACE project
partners, by analysing existing approaches, tools, data formats, ontologies,
and the requirements of bioinformaticians and bench biologists. We defined the
canonical exchange formats in a pure XML Schema, semantically annotated using
SAWSDL standard by the EMBRACE Data And Methods (EDAM) ontology. EDAM defines
formal semantics of the syntactic BioXSD data objects. Maintenance and further
development of BioXSD will be done in an open collaboration with the world-wide
bioinformatics community, curated by the BioXSD consortium.</p>

<h3>Results</h3>

<p class="abresults">BioXSD is a candidate for standard, canonical exchange
format for basic bioinformatics data types. BioXSD defines syntax for
biological sequences, sequence annotations, alignments, and references to
resources (data, databases, taxonomies, ontologies). We have adapted a set of
Web services to use BioXSD as the input and output format, and implemented a test-case
workflow. This demonstrates that the approach is feasible and provides smooth
interoperability.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Providers of bioinformatics databases and tools are
encouraged to include BioXSD among the other supported input and output
formats. BioXSD types can be further included in custom types, extended, or
restricted, or can be used as an intermediate exchange format. We recommend the
specialised standard XSDs (SBML, PDBML, MAGE-ML, PSI MI MIF, phyloXML, ...) to
be used whenever applicable. BioXSD should be used for the most common, basic
bioinformatics data-types. These basic data types were not previously
standardised in a sufficiently interoperable way applicable to the world-wide
Web services.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioxsd.org" mce_href="http://bioxsd.org"><span style="" mce_style="mso-no-proof:yes">http://bioxsd.org</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Matus Kalas (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15345 = 'm&#97;t&#117;s.k&#97;l&#97;s' + '&#64;';
 addy15345 = addy15345 + 'bccs' + '&#46;' + '&#117;&#105;b' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy15345 + suffix + '\'' + attribs + '>' );
 document.write( addy15345 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11600 = 'm&#97;t&#117;s.k&#97;l&#97;s' + '&#64;';
 addy11600 = addy11600 + 'bccs' + '&#46;' + '&#117;&#105;b' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy11600 + suffix + '\'' + attribs + '>' );
 document.write( addy11600 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23804 = 'm&#97;t&#117;s.k&#97;l&#97;s' + '&#64;';
 addy23804 = addy23804 + 'bccs' + '&#46;' + '&#117;&#105;b' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy23804 + suffix + '\'' + attribs + '>' );
 document.write( addy23804 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Computational Biology Unit, Bergen Center for
Computational Science</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Computational Biology Unit, Bergen Center for
Computational Science, Uni Research, Bergen, Norway. (2) Department of
Informatics, University of Bergen, Bergen, Norway. (3) Institut de Biologie et
Chimie des Protéines, Centre National de la Recherche Scientifique &amp;
Université Claude Bernard Lyon 1, Lyon, France. (4) Center for Biological
Sequence Analysis, Department of Systems Biology, Technical University of
Denmark, Kongens Lyngby, Denmark. (5) Institute for Bioinformatics, Center for
Biotechnology, Bielefeld University, Bielefeld, Germany. (6) European
Bioinformatics Institute, EMBL, Wellcome Trust Genome Campus, Hinxton,
Cambridge, UK.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the European Commission
within FP6 [LHSG-CT-2004-512092, the EMBRACE project]; the Research Council of
Norway within eVITA [178885/V30, eSysbio project, to M.K., P.P., A.T., I.J.];
l'Agence Nationale de la Recherche [ANR-06-CIS6-005, HIPCAL project, to A.J.];
and the Villum Foundation [grant for the Center of Disease Systems Biology, to
E.B. and K.R.].</p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="117-prediction-and-annotation-of-molecular-function.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>