<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Gene Regulation and Transcriptomics</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Gene Regulation and Transcriptomics			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Gene Regulation and Transcriptomics</span><span style="font-variant: normal ! important; text-transform: uppercase;"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Jaak Vilo and Zohar Yakhini<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Gene Regulation and
Transcriptomics<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744870 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">1</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span><span style=""><o:p></o:p></span></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744871" mce_href="#_Toc269744871">E-1. TFactS: a tool to predict
transcription factor regulation from gene expression data<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744871 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">4</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744872" mce_href="#_Toc269744872">E-2. Time- and dose-dependent
effects of cigarette smoke on monocytic cell transcriptome<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744872 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">5</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744873" mce_href="#_Toc269744873">E-3. Combined Monte Carlo and
dynamical modelling of gene regulation enables multiple parameter estimations
and hypothesis generation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span></span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"> PAGEREF
_Toc269744873 \h </span><span style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">6</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744874" mce_href="#_Toc269744874">E-4. Module-based comparative gene
expression analysis: evolutionary conserved coexpression in Bacillus subtilis
and Escherichia coli<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744874 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">7</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744875" mce_href="#_Toc269744875">E-5. Algorithm-driven artifacts in
median polish summarization of microarray data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744875 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">8</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744876" mce_href="#_Toc269744876">E-6. In silico study of the
regulation of sRNAs in Escherichia coli<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744876 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">9</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744877" mce_href="#_Toc269744877">E-7. Systems biology analysis at VIB<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744877 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">10</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744878" mce_href="#_Toc269744878">E-8. A computational framework for
automated analysis of nonlinear dynamics of gene regulatory networks<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744878 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">11</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744879" mce_href="#_Toc269744879">E-9. Exploiting ESTs to infer plant
alternative splicing<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744879 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">12</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800370039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744880" mce_href="#_Toc269744880">E-10. Inferring gene regulatory
networks to identify conserved regulation between Arabidopsis and Populus<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744880 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">13</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744881" mce_href="#_Toc269744881">E-11. BayesPI: a new model to study
protein-DNA interactions; a case study of condition-specific protein binding
parameters for yeast transcription factors<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744881 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">14</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744882" mce_href="#_Toc269744882">E-12. Use of structural DNA
properties for the prediction of regulator binding sites with conditional
random fields<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744882 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">15</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744883" mce_href="#_Toc269744883">E-13. Gene expression analysis in
Scots pine in response to red and far-red light<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744883 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">16</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744884" mce_href="#_Toc269744884">E-14. Transcription regulatory
regions and motifs (TRAM) database: the genomic data layer of the Nencki
regulatory genomics portal<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744884 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">17</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744885" mce_href="#_Toc269744885">E-15. The microRNA body map:
dissecting microRNA function through integrative genomics<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744885 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">18</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744886" mce_href="#_Toc269744886">E-16. D-Light: transcription factor
binding site management and visualization<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744886 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">19</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744887" mce_href="#_Toc269744887">E-17. Expression and chromosomal
clustering of tissue restricted antigens<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744887 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">20</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744888" mce_href="#_Toc269744888">E-18. Community dynamics and
metatranscriptome analysis of lactic acid bacteria ecosystems through
functional gene microarray analysis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744888 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">21</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744889" mce_href="#_Toc269744889">E-19. A functional gene microarray
for lactic acid bacteria exceeding the species level: design, validation, and
data analysis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744889 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">22</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800380039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744890" mce_href="#_Toc269744890">E-20. Combinatorial analysis of
transcription regulation of response to oxidative stresses in E. coli<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744890 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">23</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744891" mce_href="#_Toc269744891">E-21. Time delays in gene
activation: sensitivity to external noise<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744891 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">24</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744892" mce_href="#_Toc269744892">E-22. Discovering regulatory
mechanisms that govern zygotic genome activation in Drosophila<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744892 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">25</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744893" mce_href="#_Toc269744893">E-23. Detection of extended UTRs
with 3' expression microarrays<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744893 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">26</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744894" mce_href="#_Toc269744894">E-24. A dynamic regulatory network
model reveals hard-wired heterogeneity in blood stem cells<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744894 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">27</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744895" mce_href="#_Toc269744895">E-25. Alternative splicing as a
marker of disease<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744895 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">28</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744896" mce_href="#_Toc269744896">E-26. TilSeg: an automated pipeline
to process tiling array expression data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744896 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">29</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744897" mce_href="#_Toc269744897">E-27. Evaluation of computational
miRNA target predictions in human<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744897 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">30</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744898" mce_href="#_Toc269744898">E-28. Model-Based Multifactor
Dimensionality Reduction to detect epistasis for quantitative traits in the
presence of error-free and noisy data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744898 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">31</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744899" mce_href="#_Toc269744899">E-29. Time coherent
three-dimensional clustering: unraveling local transcriptional patterns in
multiple gene expression time series<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744899 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">32</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003800390039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744900" mce_href="#_Toc269744900">E-30. The role of incoherent
microRNA-mediated feedforward loops in noise buffering<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744900 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">33</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744901" mce_href="#_Toc269744901">E-31. Dynamic transcriptomics of
helper T cell differentiation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744901 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">34</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744902" mce_href="#_Toc269744902">E-32. Effect of size and
heterogeneity of samples on biomarker discovery: synthetic and real data
assessment<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744902 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">35</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744903" mce_href="#_Toc269744903">E-33. Analysis of DNA sequence
dependent structural properties in prokaryotic genomes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744903 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">36</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744904" mce_href="#_Toc269744904">E-34. Computational analysis of
transcription factor binding co-localization<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744904 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">37</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744905" mce_href="#_Toc269744905">E-35. Cross-checking experimental
results with publicly available gene expression data: a query-driven approach<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744905 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">38</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744906" mce_href="#_Toc269744906">E-36. Using tiling microarrays to
predict transcription start sites: application to Lactobacillus plantarum WCFS1<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744906 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">39</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744907" mce_href="#_Toc269744907">E-37. Predicting regulatory networks
from large scale time-course microarray data under several stimulation
conditions on MCF-7 cells<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span></span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"> PAGEREF
_Toc269744907 \h </span><span style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">40</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744908" mce_href="#_Toc269744908">E-38. Inferring regulatory networks
from expression data using tree-based methods<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744908 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">41</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744909" mce_href="#_Toc269744909">E-39. Dynamic profile of
transcription factors inferred from mRNA expression time courses in
Gefitinib-treated lung cancer cells<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744909 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">42</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900300039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744910" mce_href="#_Toc269744910">E-40. Deciphering the genomic
plasticity of bacterial species<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744910 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">43</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744911" mce_href="#_Toc269744911">E-41. Expression arrays &amp;
outlying processes: building a database of outliers<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744911 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">44</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744912" mce_href="#_Toc269744912">E-42. Detecting significantly
correlated gene clusters and their interactions in a continuous scale-space<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744912 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">45</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744913" mce_href="#_Toc269744913">E-43. MAMMOD: Multi-Agent Motif and
MOdule Discovery<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744913 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">46</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744914" mce_href="#_Toc269744914">E-44. Modulon identification from
bacterial gene expression array data using an improved supervised approach<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744914 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">47</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744915" mce_href="#_Toc269744915">E-45. GCQM: quality assessment of
microarray studies and samples using gene correlations<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744915 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">48</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744916" mce_href="#_Toc269744916">E-46. Integration of correlation
structure improves performance of pathway-based classification<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744916 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">49</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744917" mce_href="#_Toc269744917">E-47. Multi Experiment Matrix: web
tool for mining co-expressed genes over hundreds of datasets<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744917 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">50</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744918" mce_href="#_Toc269744918">E-48. Linking parallel measurements
of high-throughput miRNA, gene and protein expression data<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744918 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">51</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744919" mce_href="#_Toc269744919">E-49. An extended translational
network involving transcription factors and RNA binding proteins derived fro,
the identification of phylogenetic footprints in 5' and 3' untranslated regions<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744919 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">52</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744920" mce_href="#_Toc269744920">E-50. Unvealing heterogeneity of
stage II and stage III colorectal cancer by defining molecular subtypes<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744920 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">53</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744921" mce_href="#_Toc269744921">E-51. A method for improved
prediction of in vivo transcription factor binding sites by using biophysical
DNA features<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744921 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">54</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744922" mce_href="#_Toc269744922">E-52. Nonlinear dimension reduction
and clustering by minimum curvilinearity unfold neuropathic pain and tissue
embryological classes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744922 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">55</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744923" mce_href="#_Toc269744923">E-53. Seeded cis-regulatory module
discovery using rank based motif scoring<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744923 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">56</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744924" mce_href="#_Toc269744924">E-54. Proximity-based cis-regulatory
module detection using constraint programming for itemset mining<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744924 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">57</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744925" mce_href="#_Toc269744925">E-55. High-throughput detection of
epistasis in studies of the genetics of complex traits<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744925 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">58</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744926" mce_href="#_Toc269744926">E-56. Heavy metal resistance in
Cupriavidus metallidurans: a complex evolutionary and transcriptional process<span style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744926 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">59</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744927" mce_href="#_Toc269744927">E-57. Identification of co-regulated
mRNA/miRNA pairs by the CoExpress software tool<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744927 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">60</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="118-Gene-regulation-and-transcriptomics.html#_Toc269744928" mce_href="#_Toc269744928">E-58. Identification of genes with
preferential expression in the egg cell<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269744928 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;">61</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340034003900320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744871">E-1. TFactS: a tool to predict transcription factor
regulation from gene expression data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Essaghir A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Essaghir A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Pachikian BD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pachikian BD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Delzenne NM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Delzenne NM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Toffalini F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Toffalini F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), van Helden J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Helden J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Demoulin JB<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Demoulin JB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Deciphering transcription factor networks from microarray
data remains difficult. This study presents a simple method to infer the
regulation of transcription factors from microarray data based on their
well-characterized target genes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We generated a catalog containing 343 transcription
factors associated with 2720 target genes and 6401 experimentally validated
regulations. When it was available, a distinction between transcriptional
activation and inhibition was included for each regulation. Next, we built a tool
(TFactS) that compares submitted gene lists with target genes in the catalog to
detect regulated transcription factors. TFactS was validated with published
lists of regulated genes in various models and compared to tools based on in
silico promoter analysis and was used to screen cancer gene expression data.</p>

<h3>Results</h3>

<p class="abresults">On cancer data, TFactS showed the regulation of SOX10, MITF
and JUN in melanomas. We then performed microarray experiments comparing gene
expression response of human fibroblasts stimulated by different growth
factors. TFactS predicted the specific activation of Signal transducer and
activator of transcription factors by PDGF-BB, which was confirmed
experimentally. We also compared in vivo gene expression in the liver of mice
subjected to an omega3 fatty acid deficient diet. TFactS predicted an
activation of LXR and SREBP which was confirmed experimentally.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Altogether, we show here that comparing groups of genes
showing a response in microarray data with experimental target gene signatures
is an efficient way to predict the regulation of well-characterized
trans-acting factors. As a proof of concept, we validated a simple tool
(TFactS) that combines Fisher's test and a curated target catalogs, which do or
do not take into account the sign of the regulations. We also suggest that
TFactS may contribute to the functional analysis of cancer microarray data.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.tfacts.org" mce_href="http://www.tfacts.org">http://www.tfacts.org</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ahmed Essaghir (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37492 = '&#97;hm&#101;d.&#101;ss&#97;gh&#105;r' + '&#64;';
 addy37492 = addy37492 + '&#117;cl&#111;&#117;v&#97;&#105;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy37492 + suffix + '\'' + attribs + '>' );
 document.write( addy37492 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Université Catholique de Louvain</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1- de Duve Institute, Université Catholique de Louvain
(UCL), Brussels. 2- Metabolism and Nutrition Research Group, Louvain Drug
Research Institute, UCL, Brussels. 3- Bioinformatique des Génomes et des
Réseaux (BIGRe), Université Libre de Bruxelles.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Wallonie-Bruxelles International scholarship. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744872">E-2. Time- and dose-dependent effects of cigarette
smoke on monocytic cell transcriptome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Poussin C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Poussin C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Lietz M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lietz M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Schlage W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schlage W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Lebrun S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lebrun S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Hoeng J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hoeng J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Peitsch M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Peitsch M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Atherosclerosis is a chronic inflammatory process
characterized by a series of biological processes resulting in plaque formation
in the vascular wall. The monocyte/macrophage might be an important key player
in the initiation and progression of plaque formation. It has been shown that
exposure to cigarette smoke (CS) increases the adhesion of monocytes to
endothelial cells in vitro, a critical step in the initiation of
atherosclerosis. The purpose of this study was to investigate the molecular
mechanisms underlying the effect of CS exposure on monocytic cells over time.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Human Mono Mac 6 cells were treated with different
doses (0.045 and 0.09 puff/ml) of CS that had been bubbled through phosphate
buffered saline (PBS) or with control PBS for 30, 60, and 120 minutes. RNA from
different cell samples was extracted for further transcriptomic analysis using
the Human Genome U133 Plus 2.0 Affymetrix® gene expression platform. The
statistical analysis, including a multiple linear model approach performed with
the limma package (Smyth GK,2005), identified 502 probesets (~428 genes) with a
significant interaction (FDR≤0.01) between the time and dose variables.</p>

<h3>Results</h3>

<p class="abresults">Genes involved in transcription and transcription
regulation, cell cycle, endoplasmic reticulum- and oxidative-stress response
were over-represented in the list of differentially regulated genes. Indeed,
numerous genes involved in the Nrf2-mediated oxidative stress response and in
the p53-signaling pathway were found to be coordinately activated in a dose-
and time-dependent manner. In fact, the expression of the cycle-dependent
kinase inhibitor p21 (CDKN1A) gene was found to be dose- and time-dependently
increased, whereas the expression of some cyclins, targets of p21, was
repressed by CS.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Interestingly, CDKN1A is an important negative regulator
of cell cycle and has been shown to be involved in monocytic cell
differentiation to macrophages and to protect against atherosclerosis when
knocked out in apoE-/- mice. Overall, the results indicate that upon CS
exposure, monocytic cells undergo stress responses and transcriptional gene
expression regulation in a dose- and time-dependent manner, which may drive these
cells to acquire new biological functionalities that may be involved in the
development of atherosclerosis.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Carine Poussin (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32611 = 'c&#97;r&#105;n&#101;.p&#111;&#117;ss&#105;n' + '&#64;';
 addy32611 = addy32611 + 'pm&#105;ntl' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy32611 + suffix + '\'' + attribs + '>' );
 document.write( addy32611 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Philip Morris International R&amp;D Neuchâtel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Philip Morris International R&amp;D, Philip Morris
Products S.A., Neuchâtel, Switzerland. (2) Philip Morris International R&amp;D,
Philip Morris Research Laboratories GmbH, Cologne, Germany. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744873">E-3. Combined Monte Carlo and dynamical modelling
of gene regulation enables multiple parameter estimations and hypothesis
generation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Herman D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Herman D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Thomas CM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thomas CM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Stekel DJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Stekel DJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">KorA and KorB are global regulators of the broad host
range plasmid RK2 that can either separately or cooperatively autorepress the
central control operon (cco). Separate repression has been known as partial.
However their mechanism and strength have not been examined. Better
understanding of the cco control and a hypothesis formulation required a model
generation and estimation of protein synthesis and monomerization rates. Our
interest was also in the model sensitivity analyses and their consistency with
information provided by the parameter estimation algorithm.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The gene regulation of the cco was mathematically
modeled with a deterministic approach and the steady state of the growth phase
was considered. Five models, which varied by different scenarios of the
mechanism of partial repression, were examined. In order to estimate unknown
parameters, the Metropolis-Hastings algorithm, based on a Markov Chain Monte
Carlo method, was applied. This allowed us to include both known and unknown
parameters into the model fitting by choosing log normal or flat priors as
appropriate. For model sensitivity analyses, the Metabolic Control Analyses was
implemented.</p>

<h3>Results</h3>

<p class="abresults">The stochastic sampling of the parameter has resulted in
multimodal posterior distributions, suggesting a number of hypotheses about the
behavior of the system. Updated posteriors of monomerization rates have been
spread over a broad value range of 50 orders of magnitude, implying an
insignificance of these parameters for the model. This conclusion is consistent
with the results from the MCA. A ratio of protein abundances for the examined
models and the model without repression and experimental results of protein
abundances for the unrepressed operon might be able to eliminate a real model.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The reported results have demonstrated the general
utility of the Monte Carlo Markov Chain (MCMC) approach to systems biology
modeling. Additionally, the consistency in some information from MCMC and
Metabolic Control Analyses calculations is significant. Moreover, based on our
results an experimental verification we might be able to eliminate a model for
the mechanism of partial repression of the central control operon on RK2
plasmid.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Dorota Herman (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35441 = 'dxh885' + '&#64;';
 addy35441 = addy35441 + 'bh&#97;m' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy35441 + suffix + '\'' + attribs + '>' );
 document.write( addy35441 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12539 = 'dxh885' + '&#64;';
 addy12539 = addy12539 + 'bh&#97;m' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy12539 + suffix + '\'' + attribs + '>' );
 document.write( addy12539 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3659 = 'dxh885' + '&#64;';
 addy3659 = addy3659 + 'bh&#97;m' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy3659 + suffix + '\'' + attribs + '>' );
 document.write( addy3659 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Birmingham</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Center for Systems Biology, School of Biosciences,
The University of Birmingham, B15 2TT,United Kingdom (2) School of Biosciences,
The University of Birmingham, B15 2TT,United Kingdom (3) Integrative Systems
Biology, School of Biosciences, The University of Nottingham, LE12 5RD, United
Kingdom</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Darwin Trust of Edinburgh</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744874">E-4. Module-based comparative gene expression
analysis: evolutionary conserved coexpression in Bacillus subtilis and
Escherichia coli</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Zarrineh P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Zarrineh P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Fierro C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fierro C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Sánchez-Rodríguez A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sánchez-Rodríguez A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Moor B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Moor B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Marchal K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marchal K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Increasingly large scale expression compendia for
different species are becoming available. By exploiting the modularity of the
coexpression network, these compendia can be used to identify pathways or
processes for which the expression behavior is conserved between different
species. However, comparing module networks across species is not trivial as
the definition of a biologically true module is not a fixed one, but depends on
the distance threshold that defines the degree of coexpression with the
modules.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a cross-species cocluster approach
COMODO (COnserved MODules across Organisms). COMODO uses as input microarray
data and homology mapping among genes and provides as output pairs of conserved
modules. COMODO searches for conserved coexpression modules in heterogeneous
microarray expression compendia. Module sizes are selected as such that they
maximize the number of orthologs that link up both modules relative to the
total number of genes in each module that contribute to a conserved module
pair.</p>

<h3>Results</h3>

<p class="abresults">We applied our methodology to study coexpression
conservation between Escherichia coli and Bacillus subtilis. We identified
several pathways and processes for which transcriptional coexpression has been
conserved over the long evolutionary distance that separates both prokaryotic
organisms. As was expected, operons seemed to be one of the major mechanisms to
guarantee coexpression behavior amongst genes, not only within a species, but
also across species. The mechanism by which genes were transcriptionally
coregulated seemed to be much less conserved than their coexpression behavior
itself.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed a method to search for conserved modules
between two species. Applying this method allowed detecting both processes for
which the expression behavior is conserved over the wide evolutionary distance
that separates E. coli from B. subtilis or where subparts of the processes
differ in expression As it relies on many-to-many homology mapping, instead of
using bidirectional reciprocal best blast hit it offers the possibility to
search for functionally true orthology relations.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Peyman Zarrineh (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29129 = 'P&#101;ym&#97;n.Z&#97;rr&#105;n&#101;h' + '&#64;';
 addy29129 = addy29129 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy29129 + suffix + '\'' + attribs + '>' );
 document.write( addy29129 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Electrical Engineering, Katholieke
Universiteit Leuven, Kasteelpark Arenberg 10, 3001 Leuven, Belgium. (2)
Department of Microbial and Molecular Systems, Katholieke Universiteit Leuven,
Kasteelpark Arenberg 20, 3001 Leuven, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by: 1) KUL: GOA AMBioRICS,
GOA/08/011, CoE EF/05/007, SymBioSys; CREA/08/023 2) IWT: SBO-BioFrame; 3) IUAP
P6/25 (BioMaGNet); 4) FWO IOK-B9725-G.0329.09; 5) ZKB8933/CREA/08/023/ BOF, 6)
HFSP-RGY0079/2007C. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744875">E-5. Algorithm-driven artifacts in median polish
summarization of microarray data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Giorgi F-M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Giorgi F-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Bolger A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bolger A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Lohse M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lohse M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Usadel B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Usadel B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">High-throughput measurement of transcript intensities
using Affymetrix type oligonucleotide microarrays has produced a massive
quantity of data during the last decade. Different preprocessing techniques
exist to convert the raw signal intensities measured by these chips into gene
expression estimates. Although these techniques have been widely benchmarked
for differential gene expression analysis, there are only few examples where
their performance has been assessed in respect to other contexts like sample
clustering.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We assessed overall correlation between several thousand
Arabidopsis thaliana ATH1 microarrays available from Gene Expression Omnibus
database using RMA, GCRMA and MAS5 preprocessing methods. We permuted the
arrays to have a null-information scenario for correlation. We implemented a
modified RMA version (tRMA) in R using this corrected summarization method.
Finally, we tested clustering performance of our method using publicly
available Arabidopsis and Human datasets.</p>

<h3>Results</h3>

<p class="abresults">We benchmark the most used normalization procedures (MAS5,
RMA and GCRMA) in the context of inter-array correlation analysis, confirming
and extending the finding that RMA and GCRMA overestimate sample similarity.
This artificial behaviour is particularly strong in internally inconsistent,
noise-driven probesets. We offer a mathematical explanation for the artifact
behaviour and relate this to an analysis of probesets’ features. Our proposed
solution, tRMA, modifies the summarization step of RMA/GCRMA and provides
improved clustering performance in biological datasets.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our finding proposes that one of the most relevant
advantages of RMA and GCRMA in microarray preprocessing benchmarks, the low
variance across replicates, is partially the result of an artificial
inter-array correlation. While this doesn’t seem to affect differential gene
expression analyses, it can’t be ignored where unbiased measurements are
needed, like sample clustering. We propose the tRMA method to overcome this
issue, while retaining most of the features of the original RMA procedure.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Federico M. Giorgi (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1504 = 'g&#105;&#111;rg&#105;' + '&#64;';
 addy1504 = addy1504 + 'mp&#105;mp-g&#111;lm' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy1504 + suffix + '\'' + attribs + '>' );
 document.write( addy1504 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Max Planck Institute for Molecular Plant Physiology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Max Planck Institute for Molecular Plant
Physiology, Am Muehlenberg 1, 14476 Golm Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Max Planck Society</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744876">E-6. In silico study of the regulation of sRNAs in
Escherichia coli</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ishchukov I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ishchukov I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Ryan D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ryan D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Zarrineh P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zarrineh P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Cloots L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cloots L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Thijs I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thijs I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Engelen K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Engelen K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Marchal K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marchal K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">In bacteria a large part of the sRNAs regulates gene
expression by pairing to mRNAs and affecting the mRNA stability and/or
translation. This sRNA network and its impact on bacterial regulation is still
unknown: it is not clear how many targets are affected by one particular sRNA,
nor how many sRNAs can regulate a single gene. Moreover, except for the fact
that some known TFs regulate sRNAs, it is still unclear how the sRNA network is
interwoven with that of the known part of the transcriptional network.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this work we aimed at updating the known
Escherichia coli transcriptional network with the sRNA regulatory network. This
includes predicting the regulation of the sRNAs and extending sRNA regulons of
with novel targets. sRNA regulation was studied using de novo motif detection
based on phylogenetic footprinting. Intergenic sequences of orthologous sRNA
families in different related gamma proteobacteria were subjected to the
PhyloGibbs (v. 1.2). Parameters were optimized as described in Storms et al.
For target prediction we combined the intaRNA and targetRNA.</p>

<h3>Results</h3>

<p class="abresults">We updated E. coli sRNA interaction network and integrate it
with the existing transcriptional network.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We further validated the TF-motif assignments in our sRNA
dataset by using a strategy based on GO enrichment, assuming that if the target
genes of respectively the TF predicted to regulate the corresponding sRNA, and
the sRNA itself are involved in similar functions, the assignment of the
cognate TF to the sRNA should be more reliable.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ivan Ishchukov (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28628 = '&#105;v&#97;n.&#105;shch&#117;k&#111;v' + '&#64;';
 addy28628 = addy28628 + 'st&#117;d&#101;nt' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy28628 + suffix + '\'' + attribs + '>' );
 document.write( addy28628 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">K U Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">K U Leuven</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">OE-grant KULeuven</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744877">E-7. Systems biology analysis at VIB</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ren X-Y<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Ren X-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, Bonnet E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bonnet E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Joshi A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Joshi A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Maere S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Maere S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Michoel T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Michoel T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Van Parys T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Parys T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Vermeirssen V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vermeirssen V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Lost in regulation networks? Want to find regulators in
your expression data? Want to visualize the regulation networks? We have
solutions for you!</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have some well benchmarked softwares that are
specialized in inferring gene regulation networks by integrative modeling of
large amounts of high-throughput molecular data using systems biology
approaches.</p>

<h3>Results</h3>

<p class="abresults">LeMoNe: a software package for inferring regulatory modules
and predict their condition-dependent regulators from large-scale compendia of
gene expression data. ENIGMA: a software tool to extract gene expression
modules from perturbational microarray expression data. Pathicular: a Cytoscape
plugin for studying the cellular response to genetic perturbations. MINT: a
Cytoscape plugin for the identification of functional modules in integrated
networks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">If studying regulatory networks from systems biology
perspective is all what you want, just try our tools! You can either download
them from our website (http://bioinformatics.psb.ugent.be/software), or
alternatively, you can bring your expression data to us for the analysis. The
results will aid you greatly in the biological interpretations and in the
assessment of crosstalk between biological processes, as well as speed up your
possible biomarker discovery.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinformatics.psb.ugent.be/software" mce_href="http://bioinformatics.psb.ugent.be/software">http://bioinformatics.psb.ugent.be/software</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Xin-Ying Ren (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77302 = 'x&#105;ny&#105;ng.r&#101;n' + '&#64;';
 addy77302 = addy77302 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy77302 + suffix + '\'' + attribs + '>' );
 document.write( addy77302 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Plant Systems Biology, VIB</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Bioinformatics and Systems Biology, Plant Systems
Biology, VIB</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge the support of Ghent University
(Multidisciplinary Research Partnership “Bioinformatics: from nucleotides to
networks”). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744878">E-8. A computational framework for automated
analysis of nonlinear dynamics of gene regulatory networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ironi L<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Ironi L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(*), Panzeri L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Panzeri L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">A specific class of ODEs adequately describes the
essential features of the complex dynamics of GRNs. But, the qualitative
behaviour of a GRN depends crucially on the parameter values, and bifurcation
phenomena may occur when parameters pass critical values. Then, numerical
investigations of large GRNs are nontrivial and time-consuming, and might also
be impracticable as quantitative information on the biochemical reactions
underlying regulatory interactions, and on parameters are often unknown. Thus,
the need for an automated symbolic procedure to reveal the full range of
network dynamics.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We assume threshold dependent regulation, i.e. only
effective above or below a certain threshold. Switch-like behaviours across
variable thresholds are properly modelled by steep sigmoid functions, and the
ODES model linear and nonlinear dynamics occurring at different time scales.
Unlike other tools, we consider self-regulation and directly deals with
continuous response functions. The algorithm integrates QR methods with
singular perturbation analysis: the former ones allow us to cope with incomplete
knowledge, and the latter one to deal with processes that occur at different
time scales.</p>

<h3>Results</h3>

<p class="abresults">The algorithm works under the assumptions above and that any
two genes are never regulated at the same threshold by a certain variable.
Under these biologically reasonable assumptions, we have worked out the
singular perturbation approach to establish sound rules, computationally
tractable, that determine the trajectories as the sequence of phase space
regions through which the system passes. When suitable conditions are
fulfilled, the simulation outcomes do capture the network dynamical properties
dependent on the model structure and invariant for ranges of model parameter
values.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The tool proposed is applicable to predict the full range
of dynamics of GRNs that fulfill the given assumptions. Thus, it provides
grounds for dry experiments in both Systems and Synthetic Biology. In the
former context, it is useful to formulate new hypotheses that explain data
previously unobserved or not interpretable by the current knowledge; in the
latter one to preliminarly dry design of a synthetic regulatory network. In
particular, we are applying the computational tool to explore the mechanisms
underlying sex determination in Ceratitis capitata, and motility in Bacillus
subtilis.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Liliana Ironi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69117 = '&#105;r&#111;n&#105;' + '&#64;';
 addy69117 = addy69117 + '&#105;m&#97;t&#105;' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy69117 + suffix + '\'' + attribs + '>' );
 document.write( addy69117 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31167 = '&#105;r&#111;n&#105;' + '&#64;';
 addy31167 = addy31167 + '&#105;m&#97;t&#105;' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy31167 + suffix + '\'' + attribs + '>' );
 document.write( addy31167 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42980 = '&#105;r&#111;n&#105;' + '&#64;';
 addy42980 = addy42980 + '&#105;m&#97;t&#105;' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy42980 + suffix + '\'' + attribs + '>' );
 document.write( addy42980 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">IMATI-CNR</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">IMATI-CNR, Pavia</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744879">E-9. Exploiting ESTs to infer plant alternative
splicing</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Potenza E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Potenza E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Cestaro A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cestaro A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Fontana P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fontana P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bianco L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bianco L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Velasco R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Velasco R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The aim of this work is to devise a method to estimate
and analyze alternative splicing events from cDNAs information. In the past, AS
was considered a minor mechanism in plants but recent studies show that AS
accounts for 20 to 40% of the whole plant transcriptome. Given the increase of
available expression data, we focused on plant genomes. The major goal of this
work is the development and validation of a piece of software for predicting AS
events from expressed sequences aligned to a genome.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">FindAS was completely developed in Java, due to its
capability of being cross-platform.The algorithm first reads alignment data
from a gff3 format, then it performs two clustering processes. The first one is
used to group together all the cDNAs sorted by genome start positions, while
the second one is aimed at identifying “alignment blocks” inside the clusters
created by the first process. The ratio between the coverage distribution of
blocks and clusters is afterwards used to identify putative AS events. Finally,
findAS predicts all possible transcripts by traversing a direct acyclic graph.</p>

<h3>Results</h3>

<p class="abresults">Validation of findAS was performed on Arabidopsis thaliana,a
comparison was made with ASIP database and the software has been assessed by
using TAIR predictions.Predictions were compared with several dataset of TAIR
annotation. FindAS has always proved to be slightly more specific and sensitive
than ASIP, when compared on the same alignment.An interesting result is the
number of predicted genes with alternative splicing event. If we consider
intron retention and exon skipping, ASIP predicts 4790 genes while findAS 4439
but TAIR 8 contains only 4325 AS genes with alternative donor/acceptor too.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The main problem for findAS, as well as for all the other
AS predictors, is the low specificity; this is probably due to the lack of
complete AS datasets suitable for a comprehensive analysis that can also
include transcripts targeted by mRNA NMD or RUST surveillance pathway. It is
likely that RNA-seq techniques will increase the number of available data and
this will constitute a benefit for tools like findAS. As future work, we plan
to develop findAS further to use deep sequencing information for improving the
discrimination among false and non-coding alternative spliced transcripts.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Emilio Potenza (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73879 = '&#101;m&#105;l&#105;&#111;.p&#111;t&#101;nz&#97;' + '&#64;';
 addy73879 = addy73879 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy73879 + suffix + '\'' + attribs + '>' );
 document.write( addy73879 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Fondazione Edmund Mach - Istituto Agrario di San
Michele all'Adige</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Fondazione Edmund Mach - Istituto Agrario di San
Michele all'Adige</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Provincia Autonoma di Trento</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744880">E-10. Inferring gene regulatory networks to
identify conserved regulation between Arabidopsis and Populus</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Netotea S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Netotea S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Hvidsten TR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hvidsten TR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Trees are a major source of renewable raw material in
terms of cellulose and lignin biosynthesis, the two most common organic
compounds on Earth. Populus is the only tree with a fully sequenced genome, and
is thus a model organism for trees. The challenge now lies in taking advantage
of the growing amount of 'omics' data, to describe the regulation of growth and
differentiation in trees. The present study aims to develop efficient methods
for inferring the gene regulatory networks of trees and to identify their
unique regulatory mechanism by network alignment to another plant species.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We employ microarray gene expression data derived
from 1024 hybridization experiments using spotted cDNA arrays, mapped to the
second generation of Populus gene models. We infer networks of significant gene
regulations based on several statistical dependency scores, including
correlation, mutual information and a context-relevant approach that takes into
consideration the local network neighborhood. We then compare the inferred
network to Arabidopsis using several local and global network alignment scoring
metrics. The alignment is based on both network topology and network function.</p>

<h3>Results</h3>

<p class="abresults">The performance of various gene regulatory network inference
methods is quantified by using multiple regression based on the network
topology to predict the expression profiles of each gene in a cross validation
framework. We align the Populus networks to the Arabidopsis network, present
global alignment scores, investigate the correlation of various node centrality
measures among the two networks and establish the high scoring network
alignment motifs. We identify conserved regulation of cellular and tissue
function, and focus on unique features related to trees.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We combine function and topology in a method of local and
global regulatory network alignment to compare the transcriptional programs of
Populus and Arabidopsis. Our current goal is to develop methods to predict
functional interaction that can complement traditional sequence based studies
in plant biology. The next step will be integrating the transcriptional data
with protein and metabolic data.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sergiu Netotea (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28497 = 's&#101;rg&#105;&#117;.n&#101;t&#111;t&#101;&#97;' + '&#64;';
 addy28497 = addy28497 + 'pl&#97;ntphys' + '&#46;' + '&#117;m&#117;' + '&#46;' + 's&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy28497 + suffix + '\'' + attribs + '>' );
 document.write( addy28497 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Umea Plant Science Centre, Umea University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Bioinformatics lab, Umeå Plant Science Centre, Dep.
of Plant Physiology, Umeå University, Sweden</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Swedish Research Council (VR) and The Swedish
Governmental Agency for Innovation Systems (VINNOVA). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744881">E-11. BayesPI: a new model to study protein-DNA
interactions; a case study of condition-specific protein binding parameters for
yeast transcription factors</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Junbai Wang<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Junbai Wang&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Morigen<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Morigen&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The application of statistical-mechanical theory in
computational prediction of TF binding sites, using ChIP-chip dataset, has
generated numerous new algorithms. Most of the early developed methods are
based on an assumption of low protein concentration, where the protein binding
probability in the models is approximated by Maxwell-Boltzmann probability. For
a full biophysical modeling of protein-DNA interaction, a term called chemical
potential has been introduced. A novel computational approach by incorporating
the chemical potential with the protein binding probability is developed.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have incorporated Bayesian model regularization
with biophysical modeling of protein-DNA interactions, and of genome-wide
nucleosome positioning to study protein-DNA interactions, using a
high-throughput dataset. The newly developed method (BayesPI) includes the
estimation of a transcription factor (TF) binding energy matrices, the
computation of binding affinity of a TF target site and the corresponding
chemical potential.</p>

<h3>Results</h3>

<p class="abresults">The method was successfully tested on synthetic ChIP-chip
datasets, real yeast ChIP-chip experiments. Subsequently, it was used to
estimate condition-specific and species-specific protein-DNA interaction for
several yeast TFs. The results revealed that the modification of the protein
binding parameters and the variation of the individual nucleotide affinity in
either recognition or flanking sequences occurred under different stresses and
in different species.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our findings suggest that the modifications of protein
binding parameters may be adaptive and play roles in the formation of the
environment-specific binding patterns of yeast TFs and in the divergence of TF
binding sites across the related yeast species.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://folk.uio.no/junbaiw/bayesPI/" mce_href="http://folk.uio.no/junbaiw/bayesPI/">http://folk.uio.no/junbaiw/bayesPI/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Junbai Wang<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Junbai Wang&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58015 = 'j&#117;nb&#97;&#105;.w&#97;ng' + '&#64;';
 addy58015 = addy58015 + 'rr-r&#101;s&#101;&#97;rch' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy58015 + suffix + '\'' + attribs + '>' );
 document.write( addy58015 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy70009 = 'j&#117;nb&#97;&#105;.w&#97;ng' + '&#64;';
 addy70009 = addy70009 + 'rr-r&#101;s&#101;&#97;rch' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy70009 + suffix + '\'' + attribs + '>' );
 document.write( addy70009 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92586 = 'j&#117;nb&#97;&#105;.w&#97;ng' + '&#64;';
 addy92586 = addy92586 + 'rr-r&#101;s&#101;&#97;rch' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy92586 + suffix + '\'' + attribs + '>' );
 document.write( addy92586 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Pathology, Norwegian Radium Hospital,
Oslo University Hospital</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Pathology 2. Department of Cell
Biology, Institute for Cancer Research The Norwegian Radium Hospital, Oslo
University Hospital Montebello 0310 Oslo, Norway</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Norwegian Cancer Society</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744882">E-12. Use of structural DNA properties for the
prediction of regulator binding sites with conditional random fields</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Meysman P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Meysman P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Engelen K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Engelen K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Laukens K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laukens K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Dang TH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dang TH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Marchal K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marchal K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Molecular recognition of genomic target sites by
regulator proteins is a vital process in the transcription regulation of genes
in living cells. The types of physical interactions that contribute to the
recognition of binding sites by a protein can roughly be divided into those
enabling direct read-out and those that allow for indirect read-out. The former
comprises base-specific recognition, such as stabilizing hydrogen bonds between
regulator amino acids and a set of conserved bases in the genomic DNA sequence,
while in the case of the latter variations within the DNA structure will be
used</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The structural DNA properties of the target sites,
needed to construct the model, are derived from their nucleotide sequence using
a number of higher-order value look-up functions, so-called structural scales,
which are based on experi-mental data (e.g. X-ray crystallography of various
DNA molecules). These structural properties were used as input data to train a
model representing the common structural features shared by all known binding
sites of a specified regulator. This was done using conditional random fields
(CRF), a discriminative machine learning method. Two novel extension algorith</p>

<h3>Results</h3>

<p class="abresults">The performance of the presented methodology was
demonstrated on data sets for 27 different Escherichia coli transcription
factors and showed improved performance when compared to previously developed
methods. Further a set of novel target site predictions, resulting from use of
the trained models in a genome wide screening of E. coli, were validated using
a microarray compendium of ca. 1500 arrays, as well as comparison with current
literature.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Using the presented framework, we created regulator
target site models that showed a higher accuracy than similar methods and were
able to find coded structural patterns that had been identified during
crystallographic analysis of the protein-DNA complexes. Further we identified
several new target sites using the structure-based models that were not
previously uncovered using traditional sequence-based methods.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pieter Meysman (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33677 = 'p&#105;&#101;t&#101;r.m&#101;ysm&#97;n' + '&#64;';
 addy33677 = addy33677 + 'b&#105;w' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy33677 + suffix + '\'' + attribs + '>' );
 document.write( addy33677 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">K.U.Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Microbial and Molecular systems,
K.U.Leuven, Kasteelpark Arenberg 20, B-3001 Leuven Heverlee, Belgium. (2)
Intelligent Systems Laboratory, Department of Mathematics and Computer Science,
Middelheimlaan 1, B-2020 Antwerpen, Belgium. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744883">E-13. Gene expression analysis in Scots pine in
response to red and far-red light</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ranade SS<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ranade SS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Abrahamsson S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Abrahamsson S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Niemi J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Niemi J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, García-Gil MR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;García-Gil MR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Light is vital for plant growth, morphogenesis and
metabolism. Genome expression pattern under red light and far-red light is
extensively studied in angiosperms but little is known about the gene
regulation in this aspect with reference to Scots Pine</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Seeds from northern Sweden were grown under
continuous red and far-red light under two separate experiments. Microarray was
performed with Pine cDNA microarrays. Microarray data was normalised by LOWESS
normalisation method and by computing the log ratios of the intensity
measurements. Statistical analysis was carried by calculating the t-test
statistic and the corresponding p-values from the M-values for each gene in the
dye swap experiment. The p-values were adjusted by FDR method using the R-statistical
package. Computational annotation of the genes was done using Blast2GO</p>

<h3>Results</h3>

<p class="abresults">405 genes were found to be up-regulated under red light
treatment and down-regulated under far-red light; and 239 genes were identified
which were down-regulated under red light and up-regulated under far-red light
treatment. Microarray analysis thus reveals that there exist genes, which are
differentially expressed and regulated under the red and far-red light
treatments in the northern population of Scots pine in Sweden</p>

<h3>Discussion</h3>

<p class="abdiscussion">Phytochromes and cryptochromes are the well studied
photoreceptors in plants that perceive red/far-red and blue light respectively.
Phytochromes control the elongation of the hypocotyls and root; while
cryptochromes are involved in the circadian rhythm of plants. In the current
investigation, cryptochromes are found to be down regulated in red light
treatment and upregulated under far-red light. Significant regulation of
phytochromes was not detected in the study. This suggests that phytochrome regulation
does not happen at the transcription level but rather at the more downstream
stages</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sonali S. Ranade (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18629 = 'S&#111;n&#97;l&#105;.R&#97;n&#97;d&#101;' + '&#64;';
 addy18629 = addy18629 + 'g&#101;nfys' + '&#46;' + 'sl&#117;' + '&#46;' + 's&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy18629 + suffix + '\'' + attribs + '>' );
 document.write( addy18629 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Department of Forest Genetics and Plant Physiology,
Umea Plant Science Centre, Swedish University of Agricultural Sciences</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Forest Genetics and Plant Physiology,
Umea Plant Science Centre, Swedish University of Agricultural Sciences (SLU),
SE-901 83 Umeå, Sweden</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744884">E-14. Transcription regulatory regions and motifs
(TRAM) database: the genomic data layer of the Nencki regulatory genomics
portal</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dabrowski M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Dabrowski M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Lenart J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lenart J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Mieczkowski J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mieczkowski J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Kaminska B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kaminska B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The vertebrate putative regulatory regions available in
Ensembl are of several different kinds. Their relative (or joint) usefulness in
understanding gene co-regulation merits further studies. The relative
performance of existing libraries of transcription factor binding motifs, both
public (JASPAR) and commercial (TRANSFAC, Genomatix) on the same genome
wide-set of putative regulatory sequences is also of interest.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We designed and implemented a database schema
unifying representation of the data from Ensembl funcgen, compara and core to
the application layer above it. Several types of putative regulatory regions
(VISTA Enhancers, cisRED promoters, AVID-VISTA or BlastZNet alignments) will be
separately scored with TRANSFAC, Genomatix and JASPAR, using their local
installations. The results will constitute the data layer used by the future
application layer - together with user-supplied expression data, for analysis
of gene co-regulation.</p>

<h3>Results</h3>

<p class="abresults">We have completed AVID-VISTA non-coding alignments in the
genomic sequence flanks of human-rat orthologous genes from Ensembl v.57,
scored with the Genomatix motif library v. 8.2. This section of TRAM contains
43583 conserved non-coding regions for 9217 genes, with nearly 2 mln instances
of 743 motifs from 180 families in either species. Scoring of all types of
putative regulatory regions with TRANSFAC and Genomatix should soon be
finished. We are also working on automation of TRAM updating.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The unifying database schema permits comparison of
different types of predicted regulatory regions with the experimentally verified
VISTA Enhancer set and genome-wide chromatin modification data. The results
obtained with the three motif libraries can be compared with the ChIP data for
individual transcription factors. With the future application layer, it will be
possible to compare the performance of different types of putative regulatory
regions, and of the three motif libraries, in prediction of gene expression
from the genomic sequence.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michal Dabrowski (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39322 = 'm.d&#97;br&#111;wsk&#105;' + '&#64;';
 addy39322 = addy39322 + 'n&#101;nck&#105;' + '&#46;' + 'g&#111;v' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy39322 + suffix + '\'' + attribs + '>' );
 document.write( addy39322 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Nencki Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Nencki Institute, Laboratory of Transcription
Regulation, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project is supported by POIG 02.03.00-00-003/09. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744885">E-15. The microRNA body map: dissecting microRNA
function through integrative genomics</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Mestdagh P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Mestdagh P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,#), Lefever S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lefever S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,#,*), Pattyn F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pattyn F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Ridzon D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ridzon D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Fredlund<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fredlund&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Fieuw A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fieuw A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vermeulen J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vermeulen J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Paepe A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Paepe A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Wong L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wong L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Speleman F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Speleman F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Chen C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chen C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Vandesompele J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandesompele J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">MicroRNAs (miRNAs) are small non-coding RNA molecules
that function as indispensible regulators of an increasing number of cellular
processes. The role of miRNAs is depending on its spatiotemporal expression
pattern and the targeted genes. While miRNA expression profiles have been
established for various normal and diseased tissues, our understanding of miRNA
function remains limited. Here, we report a unique repository of published and
unpublished miRNA expression data from more than 550 samples, which can serve
as a resource for predicting miRNA function through integrative
transcriptomics.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Unlike classical approaches, which rely solely on
in silico miRNA target prediction to infer function, we combined genome wide
mRNA and miRNA expression data in a correlative analysis and used gene set
enrichment analysis (GSEA) to identify significant miRNA – gene set
associations. Besides miRNA function and specific expression, the miRNA body
map also enables the identification of stably expressed reference miRNAs in
specified tissues, differential miRNAs between defined sample groups and
candidate therapeutic miRNAs.</p>

<h3>Results</h3>

<p class="abresults">To assess the validity of the inferred functions,
GSEA-results were analyzed for tissue and sample subset specific miRNAs. These
results show a striking correlation between the miRNA expression in these
samples and the gene set annotation to which they are linked.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The miRNA body map tailors each of the above described
analyses according to user-defined settings in the available datasets. Our
results demonstrate the value of integrated analysis and highlight the
potential of the miRNA body map project as a community resource. To promote
expansion of the current database, additional datasets can be uploaded.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Steve Lefever (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50270 = 'st&#101;v&#101;.l&#101;f&#101;v&#101;r' + '&#64;';
 addy50270 = addy50270 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy50270 + suffix + '\'' + attribs + '>' );
 document.write( addy50270 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24240 = 'st&#101;v&#101;.l&#101;f&#101;v&#101;r' + '&#64;';
 addy24240 = addy24240 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy24240 + suffix + '\'' + attribs + '>' );
 document.write( addy24240 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7081 = 'st&#101;v&#101;.l&#101;f&#101;v&#101;r' + '&#64;';
 addy7081 = addy7081 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy7081 + suffix + '\'' + attribs + '>' );
 document.write( addy7081 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Medical Genetics Ghent</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Center for Medical Genetics, Ghent University
Hospital, Ghent, Belgium. (2) German Cancer Research Center (DKFZ), Heidelberg,
Germany. (3) Life Technologies, Foster City, California, USA. (#) Equally
contributing authors</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744886">E-16. D-Light: transcription factor binding site
management and visualization</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Laimer J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Laimer J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Zuzan C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zuzan C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ehrenberger T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ehrenberger T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Freudenberger M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Freudenberger M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gschwandtner S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gschwandtner S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lebherz C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lebherz C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lirk G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lirk G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lackner P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lackner P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Different databases like JASPAR, TRANSFAC or PAZAR
collect experimental data about transcription factor binding sites (TFBSs) and
provide the data as raw DNA sequences or positional frequency matrices (PFMs)
for hundreds of different transcription factors (TFs). A number of
computational methods have been developed to use these data for the prediction
of potential binding sites on promoter sequences. This results in a large
number of TF annotation data even for a single organism which need to be
stored, queried and combined an efficient manner.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The server imports data from JASPAR (PFMs), UCSC
(promoter sequences) and NCBI (homology information). The corresponding Python
scripts can easily be adopted for other data sources. The server requires a
Unix like OS. The client is written in Java for platform independency. The
visualization is based on the GenoViz SDK.</p>

<h3>Results</h3>

<p class="abresults">We present a client-server system for compiling, managing,
querying and displaying annotation data for transcription factor binding sites,
called 'D-Light'. The system provides simple access to the annotation data.
Combinatorial queries within or between genomes allow for reasonable
localization of target genes controlled by certain TFs. Moreover, custom data
such as PFMs and promoter sequences can be added by the user. Finally, a user
management enables privacy.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We implemented a new client-server system for working
with transcription factor binding sites in a large scale. Users can easily integrate
custom data to be used in simple but also complex cross species combinatorial
queries. D-light is provided as web service on
http://biwww.che.sbg.ac.at/dlight. The server software is also available for
local installation from the same web page.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://biwww.che.sbg.ac.at/dlight" mce_href="http://biwww.che.sbg.ac.at/dlight">http://biwww.che.sbg.ac.at/dlight</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Josef Laimer (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16033 = 'j&#111;s&#101;f' + '&#64;';
 addy16033 = addy16033 + 'l&#97;&#105;m&#101;r' + '&#46;' + 'cc';
 document.write( '<a ' + path + '\'' + prefix + addy16033 + suffix + '\'' + attribs + '>' );
 document.write( addy16033 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60165 = 'j&#111;s&#101;f' + '&#64;';
 addy60165 = addy60165 + 'l&#97;&#105;m&#101;r' + '&#46;' + 'cc';
 document.write( '<a ' + path + '\'' + prefix + addy60165 + suffix + '\'' + attribs + '>' );
 document.write( addy60165 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55247 = 'j&#111;s&#101;f' + '&#64;';
 addy55247 = addy55247 + 'l&#97;&#105;m&#101;r' + '&#46;' + 'cc';
 document.write( '<a ' + path + '\'' + prefix + addy55247 + suffix + '\'' + attribs + '>' );
 document.write( addy55247 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Upper Austria University of Applied Sciences - School
of Informatics, Communications and Media</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Upper Austria University of Applied Sciences -
School of Informatics, Communications and Media (2) University of Salzburg,
Department of Molecular Biology</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Austrian Study Grant Authority</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744887">E-17. Expression and chromosomal clustering of tissue
restricted antigens</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dinkelacker M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Dinkelacker M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*),Pinto S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pinto S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2),Derbinski J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Derbinski J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2),Eils R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Eils R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,3),Kyewski B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kyewski B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2),Brors B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brors B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Eliciting the molecular mechanism of the negative
selection of T cells in the thymus gives raise to help understand and hopefully
cure autoimmune diseases in the future. Tissue restricted antigens are
promiscously expressed by medullary thymic epithelial cells (mTECs) which
mirror virtually all tissues of the body to the T cells so that immune system
will protect the body from pathogens but at the same time tolerate the bodys
own tissues. Chromosomal clustering of TRAs gives a good explanation, how this
monitoring could be regulated on a cellular level.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">TRAs where defined by a bioinformatical approach to
detect tissue specifically expressed genes in gene expression data in the
mouse, chromosomal clustering was tested with a sliding gene window of fixed
size and a sliding 10 gene window method, proposed by Roy et al. 2002 and
Gotter et al. 2004</p>

<h3>Results</h3>

<p class="abresults">With our bioinformatical approach of defining TRAs we could
detect 5800 (25%) of all genes on the microarray to be tissue specific, and
show that tissue restricted antigens are significantly clustered on the
chromosome compared to 1000 times randomly drawn gene lists of the same length.
Comparing this data to gene expression data measured in medullary thymic
epithelial cells (mTECS) in AIRE ko versus wt mice, which suffer from multiple
autoimmune diseases, we could show that our theoretically definition performed
well to detect TRAs in microarray data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Chromosomal clustering of TRAs in the thymus can be an
explanation of how T cells are trained to tolerate the bodys own tissues and
how one single cell type in the thymus is able to express all different tissue
type specific genes at once. A detailed picture of single TRA clusters proofs
that single clusters are intermingled with genes of all different tissue types.
Further analysis will hopefully help to understand how central tolerance is
established and autoimmune diseases can be prevented.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Maria A. Dinkelacker (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88898 = 'm.d&#105;nk&#101;l&#97;ck&#101;r' + '&#64;';
 addy88898 = addy88898 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy88898 + suffix + '\'' + attribs + '>' );
 document.write( addy88898 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">German Cancer Research Center, Heidelberg, DKFZ</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Dept. Theoretical Bioinformatics,German Cancer
Research Center(DKFZ),Heidelberg, Germany (2) Dept. Developmental
Immunology,German Cancer Research Center(DKFZ) Heidelberg, Germany (3)
Department for Bioinformatics and Functional Genomics, Institute for Pharmacy
and Molecular Biotechnology, Bioquant, University of Heidelberg, Heidelberg,
Germany</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744888">E-18. Community dynamics and metatranscriptome
analysis of lactic acid bacteria ecosystems through functional gene microarray
analysis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Weckx S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Weckx S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Allemeersch J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Allemeersch J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Van der Meulen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van der Meulen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vrancken G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vrancken G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Huys G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Huys G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Vandamme P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandamme P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Van Hummelen P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Hummelen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Vuyst L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Vuyst L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">LAB are of industrial importance in the production of
fermented foods, among which sourdough-derived products. LAB contribute
considerably to important characteristics of these fermented foods, such as an
extended shelf-life and microbial safety, improved texture, and enhanced
organoleptic properties. Thanks to the genomic information on LAB that became
available during the last years, transcriptome and by extension
metatranscriptome studies are the exquisite research approaches to study LAB
whole-ecosystem gene expression and community dynamics into more detail.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Microarray hybridizations were performed using
time-related RNA samples, representing the metatranscriptome, of four 10-day
sourdough fermentations, and an in-house developed functional gene LAB
microarray, consisting of 2,269 oligonucleotides targeting 406 genes that play
a key role in sugar and nitrogen metabolism, functional metabolite production,
stress responses, and safety characteristics. For each gene, a net expression
profile was calculated. Microarray hybridization data were used to determine
LAB community dynamics. Results were compared with data of metabolite target
analyses.</p>

<h3>Results</h3>

<p class="abresults">Hybridization data analysis revealed that the sourdough
fermentations could be considered as a microbial ecosystem that achieved
stability after a transition phase of five days, with Lactobacillus plantarum,
Lactobacillus fermentum, and Pediococcus pentosaceus as the ultimate dominating
species. The net expression profiles showed that gene expression was
characterized by the activation of different key metabolic pathways, the
ability to use different energy sources, and the conversion of amino acids as
contribution to redox equilibrium and flavor compound generation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Overall gene expression of four sourdough fermentations
were successfully followed up using a functional gene LAB microarray. The
strains present in the sourdoughs showed an efficient use of the available
substrates, such as the consumption of other carbohydrates than glucose, either
as energy source or as alternative external electron acceptor, thereby
elaborating metabolites of interest from a technological point of view. In
addition, the hybridization patterns showed that the microarray could be used
as alternative tool to monitor population dynamics during sourdough
fermentations.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Stefan Weckx (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11869 = 'st&#101;f&#97;n.w&#101;ckx' + '&#64;';
 addy11869 = addy11869 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy11869 + suffix + '\'' + attribs + '>' );
 document.write( addy11869 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34139 = 'st&#101;f&#97;n.w&#101;ckx' + '&#64;';
 addy34139 = addy34139 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy34139 + suffix + '\'' + attribs + '>' );
 document.write( addy34139 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5810 = 'st&#101;f&#97;n.w&#101;ckx' + '&#64;';
 addy5810 = addy5810 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy5810 + suffix + '\'' + attribs + '>' );
 document.write( addy5810 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Vrije Universiteit Brussel - IMDO</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Research Group of Industrial Microbiology and Food
Biotechnology (IMDO), Faculty of Sciences and Bio-Engineering Sciences, Vrije
Universiteit Brussel, Belgium (2) Microarray Facility, Flanders Institute for
Biotechnology (VIB), Belgium (3) Laboratory of Microbiology, Faculty of
Sciences, Ghent University, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744889">E-19. A functional gene microarray for lactic acid
bacteria exceeding the species level: design, validation, and data analysis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Allemeersch J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Allemeersch J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Van der Meulen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van der Meulen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vrancken G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vrancken G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Huys G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Huys G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Vandamme P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandamme P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Van Hummelen P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Hummelen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Vuyst L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Vuyst L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Weckx S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weckx S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2)</p>

<p class="abmotivation">Functional gene microarrays allow to monitor the
expression of a selected set of genes within a microbial ecosystem as a whole. To
investigate whole-ecosystem gene expression during food fermentations, a
functional gene microarray for LAB was developed targeting 406 genes across 50
species that play a key role in carbohydrate metabolism, in the production of
bacteriocins, exopolysaccharides, and flavour compounds, in probiotic and
biosafety characteristics, and in stress response. Also, genes linked to
negative traits such as antibiotic resistance and virulence were represented.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">As LAB ecosystems can contain a variety of species,
the microarray design focused on functional properties, beyond the species
level. An algorithm was implemented to design gene-specific 30-mer
oligonucleotides that preferably could hybridize multiple LAB species, to allow
for controlled cross-hybridization. Validation hybridizations were performed
using RNA and DNA of 18 LAB strains. Also, a new algorithm was developed to
analyze the hybridization data by calculating a net expression profile for each
gene represented on the microarray, thereby exceeding the species level.</p>

<h3>Results</h3>

<p class="abresults">Applying the oligonucleotide design algorithm on manually
curated public LAB sequence data resulted in 4,851 oligonucleotides. A subset
of 2,269 oligonucleotides was selected for synthesis and hence production of
the microarray based on the LAB microbiota prevailing during sourdough
fermentations. This subset represented 406 key genes in a total of 46 LAB and
four non-LAB species. The validation hybridizations using DNA and RNA from 18
strains of twelve LAB species, representing 86.0% of all oligonucleotides,
showed wide ranges of intensity and high reproducibility for technical
replicates.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A species-independent functional gene LAB microarray was
developed to study gene expression of LAB in fermented food ecosystems. An
algorithm was implemented to design 30-mer oligonucleotides with controlled
cross-hybridization capabilities between species for a given gene. Of the 2,269
oligonucleotides on the microarray, 15.5% could cross-hybridize multiple
species. Validation hybridizations proved that 30-mer oligonucleotides showed
good sensitivity and reproducibility. The net expression profiles allowed to
interpret whole-ecosystem gene expression, exceeding the individual species
level.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Joke Allemeersch (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38842 = 'j&#111;k&#101;.&#97;ll&#101;m&#101;&#101;rsch' + '&#64;';
 addy38842 = addy38842 + 'v&#105;b' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38842 + suffix + '\'' + attribs + '>' );
 document.write( addy38842 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">VIB-MAF</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Microarray Facility, Flanders Institute for
Biotechnology (VIB), Belgium (2) Research Group of Industrial Microbiology and
Food Biotechnology (IMDO), Faculty of Sciences and Bio-Engineering Sciences,
Vrije Universiteit Brussel, Belgium (3) Laboratory of Microbiology, Faculty of
Sciences, Ghent University, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744890">E-20. Combinatorial analysis of transcription
regulation of response to oxidative stresses in E. coli</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Thiele S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Thiele S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Klie S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Klie S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Jozefczuk S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jozefczuk S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Selbig J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Selbig J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2, 3), Blachon S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blachon S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,*)</p>

<p class="abmotivation">Transcriptional regulation is a crucial component of
bacterial stress response. The existence of operons and global transcriptional
regulators allows E. coli to quickly adjust to changing environment. However
the great variability of environmental signals requires the ability to combine
activity of different transcriptional regulators to fine tune the response. By
integrating (anti)correlated gene responses with knowledge on transcription factor
regulation we aim at detecting behaviors that cannot be explained with current
knowledge and can result from more complex yet unknown regulations.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Using microarrays, the time response to various
stresses was measured at the transcript level. Correlations between genes that
vary under stress were computed. When two genes are (anti)correlated, at least
one common transcription factor should be responsible for the coordinated
response. Conversely, if two genes have no common transcription factor, they
should not be (anti)correlated. We propose an original approach based on the
Answer Set Programming (ASP) framework. ASP enables to model the problem,
detect conflicts and propose sets of solutions that resolve the conflicts.</p>

<h3>Results</h3>

<p class="abresults">Focusing on oxidative stress response, 264 genes vary
significantly and have at least one known transcription factor regulating their
activities. Overall, 83 transcription factors regulate these genes. Applying
the ASP framework, conflicts were detected in 4471 gene pairs. Using a feature
of ASP, the cardinality constraint, it is possible to compute the minimal
number of repairs. Here, adding 3 unknown transcription factors is enough to
repair all conflicts. These unknown transcription factors can be identified as
real transcription factors for which the targets remain to be discovered.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Exploring all possible combinations of repairs can be
done only at a high computational cost. We are currently implementing new rules
to refine the search space. These rules are based on the problem properties and
on biological knowledge (common region in promoter sequences). So far, the
approach was applied only to data produced on oxidative stress. Other stresses
(heat shocks, ...) will be studied in the short run.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sylvain Blachon (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49607 = 'sylv&#97;&#105;n.bl&#97;ch&#111;n' + '&#64;';
 addy49607 = addy49607 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy49607 + suffix + '\'' + attribs + '>' );
 document.write( addy49607 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Department of Bioinformatics, University of Potsdam,
Potsdam, Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute for Computer Science, University of
Potsdam, Potsdam, Germany (2) Max Planck Institut for Molecular Plant
Physiology, Potsdam, Germany (3) Institute for Biochemistry and Biology,
University of Potsdam, Potsdam, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">GoFORSYS project</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744891">E-21. Time delays in gene activation: sensitivity
to external noise</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Albert J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Albert J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Rooman M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rooman M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Many biological processes occur through a controlled
sequence of events – a cascade – in which the activation of one gene by another
is delayed. Understanding the mechanisms that cause delays is of great importance
in, among other fields, developmental biology where the ability of cells to
turn on certain genes at specific times plays a crucial role. How the
activation of genes can be delayed with reliable accuracy even in the presence
of external perturbations needs to be addressed to ensure some biological
relevance.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have modeled the dynamics of a genetic switch by
a set of two coupled differential equations, using mathematica. The parameters
of the system were perturbed, one at a time, by Gaussian noise occurring at
time intervals whose probability followed a Poisson distribution. The
statistics of the switch were carried out on a basis of a pool of fifty runs
for several combinations of average amplitude and time intervals of the noise.
The performance of the switch was judged on the basis of how close were its
average delays to the exact delays (without noise) and their standard
deviations.</p>

<h3>Results</h3>

<p class="abresults">We have found that the robustness of the delays against
random parameter perturbations is determined primely by the ratio of protein to
mRNA degradation rates. The larger this ratio, the more sensitive are the
delays to parameter fluctuations. The performance of the switch was observed to
be dependent on the particular parameter that was subjected to noise. It turns
out that the optimal way to generate time-insensitive delays is through the
binding of an activating transcription factor, which does not interact with the
protein synthesized from the gene under consideration.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Considering the range of performances given by the
genetic switch we found that only some combinations of parameters allow the
activation of the switch to be delayed with reliable accuracy. The observations
that the robustness of the delays against noise depends strongly on the
degradation rates of protein and mRNA, and that these rates can differ by
orders of magnitude from eukaryotes to prokaryotes, suggest that the switching
delays caused by the proposed mechanism may be unique to one or the other type
of cell.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jaroslav Albert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16293 = 'j&#97;lb&#101;rt' + '&#64;';
 addy16293 = addy16293 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy16293 + suffix + '\'' + attribs + '>' );
 document.write( addy16293 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21486 = 'j&#97;lb&#101;rt' + '&#64;';
 addy21486 = addy21486 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy21486 + suffix + '\'' + attribs + '>' );
 document.write( addy21486 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17287 = 'j&#97;lb&#101;rt' + '&#64;';
 addy17287 = addy17287 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy17287 + suffix + '\'' + attribs + '>' );
 document.write( addy17287 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Biosystems, Biomodeling and Bioprocesses, Université
Libre de Bruxelles</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744892">E-22. Discovering regulatory mechanisms that govern
zygotic genome activation in Drosophila</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Darbo E<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Darbo E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Thieffry D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thieffry D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lecuit T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lecuit T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), van Helden J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Helden J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,1)</p>

<p class="abmotivation">In Drosophila embryos, no transcription occurs during the
first 7 cell cycles after fertilization. The control of the early steps of development
is ensured by maternal mRNAs loaded in the egg during oogenesis. Zygotic genome
activation (ZGA) occurs in two waves: the first wave involves about 60 genes,
whose transcription is activated during mitotic cycle 8; a second wave
involving more than 300 genes occurs at the 14th mitotic cycle. We aim at
characterizing the mechanisms involved in the control of transcriptional
activation during ZGA.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We integrated transcriptome profiles for 3
independent studies focused on (i) gene transcription during early
embryogenesis [Pilot et al. (2006) Development 133:711]; (ii) respective
contributions of maternal and embryonic genes [De Renzis et al.( 2007) PLoS
Biol 5:e117]; (iii) impact of nucleo/cytoplasmic ratio on transcription [Lu et
al. (2009) Development 136: 2101]. We defined clusters of genes showing
significant changes in expressions at crucial stages, predicted their putative
cis-regulatory motifs, and detected functional enrichment using Gene Ontology
(GO) annotations.</p>

<h3>Results</h3>

<p class="abresults">We analyzed 73 clusters of co-expression and discovered
highly significant motifs. Some of the discovered motifs correspond to the
consensus of known transcription factors (e.g. CAGGTAC is the consensus of
Zelda, involved in the first wave of ZGA). For some clusters, the transcription
factors predicted by motif discovery (Zelda, Dref) were consistent with the
functional enrichment in GO classes related to embryonic development. We also
found novel motifs (e.g., CAGATACA) which do not correspond to any annotated
factor.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed a pipe-line combining the analysis of
transcriptome data, regulatory motifs and functional enrichment. The finding of
known motifs such as CAGGTAG shows that the approach is able to return relevant
results, and increases our interest for the novel motifs discovered. Further
analyses of combination of motifs (motif co-occurrences, detection of
cis-regulatory modules) and multi-genome analyses (conservation and divergence
of putative binding sites) will contribute to select the most promising motifs
as candidate for experimental validation.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Elodie E Darbo (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92256 = 'd&#97;rb&#111;' + '&#64;';
 addy92256 = addy92256 + 't&#97;gc' + '&#46;' + '&#117;n&#105;v-mrs' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy92256 + suffix + '\'' + attribs + '>' );
 document.write( addy92256 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">TAGC INSERM U928</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1.Technological Advances for Genomics and Clinics
(TAGC), INSERM U928 &amp; Université de la Méditerranée (Aix-Marseille II).
Campus de Luminy, Case 928, F - 13288 Marseille, France.

 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57059 = 'd&#97;rb&#111;' + '&#64;';
 addy57059 = addy57059 + 't&#97;gc' + '&#46;' + '&#117;n&#105;v-mrs' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy57059 + suffix + '\'' + attribs + '>' );
 document.write( addy57059 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>, 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11428 = 'th&#105;&#101;ffry' + '&#64;';
 addy11428 = addy11428 + 't&#97;gc' + '&#46;' + '&#117;n&#105;v-mrs' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy11428 + suffix + '\'' + attribs + '>' );
 document.write( addy11428 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 2.Laboratoire de Génétique et
de Physiologie du Développement, UMR 6545 CNRS-Université de la Méditerranée,
Institut de Biologie du Développement de Marseille (IBDM), Campus de Luminy,
case 907, 13288 Marseille CEDEX 09, France. 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16951 = 'l&#101;c&#117;&#105;t' + '&#64;';
 addy16951 = addy16951 + '&#105;bdm' + '&#46;' + '&#117;n&#105;v-mrs' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy16951 + suffix + '\'' + attribs + '>' );
 document.write( addy16951 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 3.Laboratoire
de Bioinformatique des Génomes et des Réseaux (BiGRe). Université Libre de
Bruxelles, Campus Plaine, CP 263. Bld du Triomphe, B-1050 Bruxelles, Belgium.
Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy70983 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy70983 = addy70983 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy70983 + suffix + '\'' + attribs + '>' );
 document.write( addy70983 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by the ANR Program NeMo. The
collaboration between BiGRe and TAGC laboratories is supported by the Interuniversity
Attraction Poles, funded by the Belgian Federal Science Policy Office, project
P6/25 (BioMaGNet). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744893">E-23. Detection of extended UTRs with 3' expression
microarrays</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Thorrez L<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Thorrez L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Tranchevent L-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tranchevent L-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Chang H-J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chang H-J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Schuit F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schuit F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The 3' untranslated regions (UTRs) of transcripts are not
well characterized for many genes and often extend beyond the annotated
regions. Since Affymetrix 3' expression arrays were designed based on expressed
sequence tags, many probesets map to intergenic regions downstream of genes. We
used expression information from these probesets to predict transcript
extension beyond currently known boundaries.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The expression dataset consisting of 70 microarrays
covering 22 different murine tissues with 3-5 replicates per tissue was used as
starting data. Data are accessible through the GEO database, with accession
number GSE9954 [32]. Analysis was carried out with scripts in R, Perl and Java.
Confimation of predicitions was performed by PCR.</p>

<h3>Results</h3>

<p class="abresults">Based on our dataset encompassing expression in 22 different
murine tissues, we identified 845 genes with predicted 3'UTR extensions. These
extensions have a similar conservation as known 3'UTRs, which is distinctly
higher than intergenic regions. We verified 8 of the predictions by PCR and
found all of the predicted regions to be expressed. The method can be extended
to other 3' expression microarray platforms as we demonstrate with human data.
Additional confirming evidence was obtained from public paired end read data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We show that many genes have 3'UTR regions extending
beyond currently known gene regions and provide a method to identify such
regions based on microarray expression data. Since 3' UTR contain microRNA
binding sites and other stability determining regions, identification of the
full length 3' UTR is important to elucidate posttranscriptional regulation.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Lieven Thorrez (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54232 = 'l&#105;&#101;v&#101;n.th&#111;rr&#101;z' + '&#64;';
 addy54232 = addy54232 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy54232 + suffix + '\'' + attribs + '>' );
 document.write( addy54232 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">KULeuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) SymBioSys - KUL Center for computational systems
biology</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research was supported by research grants from the
Fonds voor Wetenschappelijk Onderzoek Vlaanderen (FWO grant G.0733.09N),
Katholieke Universiteit Leuven (GOA/2004/11 to the FS laboratory, GOA AMBioRICS
(2004-2009), GOA 3M080124 and SymBioSys: CoE EF/05/007) and the Belgian Federal
Science Policy Office (IUAP P6/40 and P6/25: BioMaGNet). L. Tranchevent is
funded by EU-RTD: ERNSI: FP6-MC-EST Bioptrain. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744894">E-24. A dynamic regulatory network model reveals
hard-wired heterogeneity in blood stem cells</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bonzanni N<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bonzanni N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Garg A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Garg A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Foster SD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Foster SD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Wilson NK<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wilson NK&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Kinston S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kinston S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Miranda-Saavedra D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Miranda-Saavedra D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Heringa J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Heringa J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Feenstra A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Feenstra A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Xenarios I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Xenarios I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Göttgens B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Göttgens B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Combinatorial interactions of transcription factors with
cis-regulatory elements control the dynamic progression through successive
cellular states and thus underpin all metazoan development. The construction of
regulatory network models based on the functionality of cis- regulatory
elements therefore has the potential to generate fundamental insights into
cellular fate and differentiation. Haematopoiesis has long served as a model
system to study mammalian differentiation, yet modelling based on
experimentally informed interactions has so far been restricted to pairs of
interacting factors.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Here we have generated a network model based on
detailed cis-regulatory functional data connecting 11 haematopoietic stem cell
(HSC) regulators. We used Boolean logic functions to model the interactions
between the cis-regulatory elements. F0 transgenic mouse embryos were generated
and analyzed to test the novel hypotheses.</p>

<h3>Results</h3>

<p class="abresults">Dynamic analysis of our model predicts that HSCs display
heterogeneous expression patterns and possess many intermediate states that
appear to act as ‘stepping stones’ for the HSC to achieve a final
differentiated state. By focussing on intermediate states occurring during
erythrocyte differentiation, we predicted a novel negative regulation of Fli1
by Gata1 which we confirmed experimentally thus validating our model.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In conclusion, we present the most advanced mammalian
regulatory network model based on experimentally validated cis-regulatory interactions
to date. This model has allowed us to make novel, experimentally testable
hypotheses about transcriptional mechanisms that control differentiation of
mammalian stem cells.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Nicola Bonzanni (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40438 = 'b&#111;nz&#97;nn&#105;' + '&#64;';
 addy40438 = addy40438 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy40438 + suffix + '\'' + attribs + '>' );
 document.write( addy40438 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54107 = 'b&#111;nz&#97;nn&#105;' + '&#64;';
 addy54107 = addy54107 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy54107 + suffix + '\'' + attribs + '>' );
 document.write( addy54107 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17517 = 'b&#111;nz&#97;nn&#105;' + '&#64;';
 addy17517 = addy17517 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy17517 + suffix + '\'' + attribs + '>' );
 document.write( addy17517 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Vrije Universiteit Amsterdam</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 IBIVU/Bioinformatics, Free University Amsterdam, De
Boelelaan 1083A - 1081 HV, Amsterdam, The Netherlands 2 Swiss Institute of
Bioinformatics, Quartier Sorge - Batiment Genopode, CH-1015 Lausanne,
Switzerland 3 Cambridge Institute for Medical Research, University of
Cambridge, Hills Road, Cambridge CB2 0XY, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This study was funded by the European Commission FP6
project ENFIN (Experimental Network for Functional INtegration). Computations
were performed at the Vital-IT Center of the Swiss Institute of Bioinformatics.
Experimental work was supported by Leukaemia &amp; Lymphoma Research (UK), the
Leukemia &amp; Lymphoma Fund (USA) and the Medical Research Council (UK). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744895">E-25. Alternative splicing as a marker of disease</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Muro EM<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Muro EM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(* ), Andrade-Navarro MA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andrade-Navarro MA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The goal of this work is to identify alternative splicing
variants depending on tissue and age, classifying them according to a ranked
list of candidate genes for neurodegenerative diseases. In addition, we will
investigate possible regulatory mechanisms for splice variation based on
antisense expression. Here we present our latest advances in these subjects
ending with the analysis of alternative splicing related to disease.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For detecting alternative splicing variants we use
different approaches: for the 3' UTR alternative variants we profit from our
own experience with "Transcriptome Sailor" (see Muro E.M. et al.
2008. Identification of gene 3' ends by automated EST cluster analysis. Proc.
Natl. Acad. Sci. 150:20286-20290). Data from the Alternative Splicing and
Transcript Diversity database 1.1 is used in combination with the corresponding
Ensembl genome annotation. For analyzing tissue, age and some other factors
associated to transcripts we use the Genebank annotations of the cDNA data
(ESTS and mRNA) in combination with MeSH terms.</p>

<h3>Results</h3>

<p class="abresults">In a first analysis we detected 3,875 genes (4,187 splice
variants) related to the central nervous system using the different databases
mentioned above. Independently we observed that some NATs (natural antisense
transcripts) arise from pseudogenes, identifying 87 cases genome wide (human).
In particular, we hypothesize that some of them might be regulating splicing,
because they bind pre-mRNAs. For some exons that are subject of exon skipping
events, we found cis-NATs expressed in the regulatory regions (cis-acting
receptors) of the skipped exon, probably acting as splicing regulators.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A large percentage of human diseases are caused by
aberrant alternative splicing. In some other cases, alternative splicing
isoforms are transcribed as a consequence of a disease but do not cause it. In
the last case the alternative splicing variants would be markers of the
disease. We are collecting splicing variants, classifying them depending on
factors like tissue or age. We hope to obtain splicing variants that are unique
in certain age range tissue and disease. In parallel we are working on the
regulation of the splicing variants always with the aim of finding examples
that could be related to disease.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://cbdm.mdc-berlin.de/" mce_href="http://cbdm.mdc-berlin.de/">http://cbdm.mdc-berlin.de/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Enrique M Muro (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14496 = '&#101;nr&#105;q&#117;&#101;.m&#117;r&#111;' + '&#64;';
 addy14496 = addy14496 + 'mdc-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy14496 + suffix + '\'' + attribs + '>' );
 document.write( addy14496 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Computational Biology and Data Mining Group
Max-Delbrueck-Centrum fuer Molekulare Medizin (MDC). Berlin. Germany.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Computational Biology and Data Mining Group Max-Delbrueck-Centrum
fuer Molekulare Medizin (MDC). Berlin. Germany.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">MSBN, NGFN. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744896">E-26. TilSeg: an automated pipeline to process
tiling array expression data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Barreau D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Barreau D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,4), Gao Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gao Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4), Jaffrezic F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jaffrezic F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4), Hugo K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hugo K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3), Rogel-Gaillard C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rogel-Gaillard C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4), Marthey S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marthey S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,*)</p>

<p class="abmotivation">High density oligonucleotide tiling arrays represent a
powerful approach to decipher gene expression in genomic regions that are
partially or not annotated. However, data analysis remains challenging due to
the lack of free or commercial softwares. Moreover, most of the existing
methods focus on probe expression levels and do not take into account the
information provided by contiguous probes. We thus developed a method to take
advantage of this kind of array design and a tool to process automatically the
data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">TilSeg mainly uses perl for managing files and for
automatized treatment of data. This tool uses standard R packages to perform
statistical analysis of the data, including normalization and segmentation
methods.</p>

<h3>Results</h3>

<p class="abresults">We developed a user friendly tool which can be used directly
on a batch of quantification files. The analysis of tiling arrays data can be
easily processed using the standard parameters defined in TilSeg, but if needed
all the statistical parameters can be easily modified. In addition, output
files can be produced after each step of the statistical pipeline and used
directly as input files for other analysis tools. Finally the data fully
processed through TilSeg can be formated and visualized directly in current
genome browsers as Ensembl or UCSC.</p>

<h3>Discussion</h3>

<p class="abdiscussion">TilSeg can be used as an automated pipeline for basic
statistical analysis of Tiling Array expression data. The new methods of
analysis that are currently being developed will be implemented in TilSeg as
new functionalities, making it a sustainable tool for the analysis of tiling
arrays. Further developments should enlarge the scope of TilSeg, including for
example automatic annotation of segments of interest and the analysis of tiling
arrays for CNV characterization.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sylvain Marthey (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65763 = 'sylv&#97;&#105;n.m&#97;rth&#101;y' + '&#64;';
 addy65763 = addy65763 + 'j&#111;&#117;y' + '&#46;' + '&#105;nr&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy65763 + suffix + '\'' + attribs + '>' );
 document.write( addy65763 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">CRB GADIE / GABI / INRA</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 INRA, CRB GADIE, UMR de Génétique Animale et Biologie
Intégrative, Jouy-en-Josas, France 2 INRA, UMR de Génétique Animale et Biologie
Intégrative, Jouy-en-Josas, France 3 CEA, DSV, iRCM, Laboratoire de
Radiobiologie et Etude du Génome, Jouy-en-Josas, France 4 AgroParisTech, UMR de
Génétique Animale et Biologie Intégrative, Jouy-en-Josas, France</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744897">E-27. Evaluation of computational miRNA target
predictions in human</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gevaert O<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gevaert O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*</p>

<p class="abmotivation">MicroRNAs (miRNAs) are short RNAs that regulate
expression through binding to the 3’UTR of mRNAs. An important shortcoming in
current miRNA research is the lack of experimentally verified mRNA targets.
This initiated a surge in the development of computational methods attempting
to predict miRNA target sites.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Here, we compare the target predictions of seven
often used target prediction tools for human miRNAs: microRNA.org, MicroCosm,
PITA, TargetScan, PicTar, MirZ and MicroT.</p>

<h3>Results</h3>

<p class="abresults">Our results showed that most tools provide target
predictions for the majority of currently known miRNAs but large differences
where observed in the number of targets predicted on average. Comparison with
experimentally verified targets showed that approximately half of the verified
interactions are not predicted by most tools. Finally, we investigated the
coordinated downregulation and showed that MirZ and MicroT captured more
significant miRNA mediated down-regulation compared to the other tools in a
wide range of tissues.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We conclude that many of the tools predict less than half
of the experimental observations. In addition, many tools predict different
targets for the same miRNA. The availability of more experimentally verified
miRNA-mRNA interactions could boost the development of target prediction models
significantly and allow a better characterization of the
miRNA-mRNA-interactome.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.esat.kuleuven.be/%7Ebioiuser/microRNA/" mce_href="http://www.esat.kuleuven.be/~bioiuser/microRNA/">http://www.esat.kuleuven.be/~bioiuser/microRNA/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Olivier Gevaert (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35925 = '&#111;l&#105;v&#105;&#101;r.g&#101;v&#97;&#101;rt' + '&#64;';
 addy35925 = addy35925 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy35925 + suffix + '\'' + attribs + '>' );
 document.write( addy35925 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Stanford/KU Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Stanford University School of Medicine Department of
Radiology/Katholieke Universiteit Leuven, Department of Electrical Engineering</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection29">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744898">E-28. Model-Based Multifactor Dimensionality
Reduction to detect epistasis for quantitative traits in the presence of
error-free and noisy data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Mahachie John JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mahachie John JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van Lishout F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Lishout F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2) , Van Steen K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Steen K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Cattaert et al. (2010) have shown the excellent power of
MB-MDR over MDR (Ritchie et al. 2003) in identifying epistasis effects on
dichotomous traits in the presence of noisy data. Although MB-MDR has been
successfully applied to quantitative traits as well by Mahachie John et al
(2009), the power of MB-MDR in identifying significant epistasis effects on a
quantitative trait in the presence of noisy data has never been explored.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use a simulation study to evaluate the power and
type 1 error rates of MB-MDR for quantitative traits in the presence of noisy
data. Considered sources of error are genotyping errors, missing genotypes,
phenotypic mixtures and genetic heterogeneity. The MB-MDR strategy to tackle
the dimensionality problem involves reducing the dimension to one by pooling
multi-locus genotypes into three groups of risk : High risk (H), Low risk (L)
or No Evidence for risk (O). An association test is then performed with the new
predictor variable X in {H,L,O} on the outcome variable.</p>

<h3>Results</h3>

<p class="abresults">The obtained results show that MB-MDR has excellent power to
identify epistasis effects in the presence of missing genotypes and low
percentages of genotyping errors. Lowest power performances were observed in
the presence of phenotypic mixtures and genetic heterogeneity. For null data
with phenotypes generated completely at random, the type I error percentages
are very close to the nominal level of 5%.</p>

<h3>Discussion</h3>

<p class="abdiscussion">For some time to come, phenotypic mixtures and genetic
heterogeneity seem to remain challenging for epistasis screening methods. This
was also observed by Cattaert et al. (2010) for dichotomous traits. Despite the
flexibility offered by the MB-MDR framework, more work is needed to improve the
performance under phenotypic mixtures and complex genetic heterogeneity
patterns</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jestinah M Mahachie John (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy48167 = 'jm&#97;h&#97;ch&#105;&#101;' + '&#64;';
 addy48167 = addy48167 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy48167 + suffix + '\'' + attribs + '>' );
 document.write( addy48167 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">University of Liege- Institute Montefiore</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Montefiore Institute, University of Liege, Belgium (2)
GIGA-Research, University of Liege, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">J.M. Mahachie John is a doctoral student funded by the
Belgian Network BioMAGNet (Bioinformatics and Modeling: from Genomes to
Networks), within the Interuniversity Attraction Poles Program (Phase VI/4),
initiated by the Belgian State, Science Policy Office. All authors acknowledge
research opportunities offered by the Belgian Network BioMAGNetand partial
support by the IST Program of the European Community, under the PASCAL2 Network
of Excellence (Pattern Analysis, Statistical Modeling and Computational
Learning), IST-2007-216886. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection30">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744899">E-29. Time coherent three-dimensional clustering:
unraveling local transcriptional patterns in multiple gene expression time
series</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gonçalves JP<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gonçalves JP&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,*), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Madeira SC<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Madeira SC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Analyzing temporal expression programs is essential to
the understanding of complex biological processes. In particular, interesting
local patterns revealing coherent behavior of genes in the time frame of a
biological event can be sought using biclustering. However, as the amount of
available data increases, research studies have been progressing towards
comparative analysis of multiple time series expression matrices (TSEMs). Few
methods have been proposed to address this problem, the majority relying on
pseudo simultaneous three-dimensional approaches where one dimension is never
clustered.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Given a set of discretized TSEMs, all maximal
triclusters of genes exhibiting coherent expression patterns in specific time
frames and TSEMs are identified as follows: 1) Alphabet transformation to
ensure time coherence; 2) Expression pattern matching using a generalized
suffix tree of the transformed patterns to identify nodes containing
triclusters; 3) Frequent closed itemset mining to reveal the maximal subsets of
genes and matrices underlying on such nodes; 4) Integration of pattern matching
and itemset mining results into triclusters; Removal of non-pattern-maximal
triclusters.</p>

<h3>Results</h3>

<p class="abresults">Preliminary results analyze a dataset consisting of gene
expression measurements obtained for 2920 genes and 9 time points in a 168 hour
period for 14 multiple sclerosis patients after injection of IFN-beta using
GeneFilters GF211 DNA arrays. Expression levels were normalized by gene to mean
0 and standard deviation 1 and then discretized using a technique expressing
the variations between time points. Triclusters were subsequently identified
using the time coherent three-dimensional clustering algorithm.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Preliminary studies show promising results. Interesting
expression patterns shared by sets of patients could be identi ed unraveling
putative gene modules involved in patients' response to the treatment with
IFN-beta.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Joana P. Gonçalves (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95836 = 'jpg' + '&#64;';
 addy95836 = addy95836 + 'kdb&#105;&#111;' + '&#46;' + '&#105;n&#101;sc-&#105;d' + '&#46;' + 'pt';
 document.write( '<a ' + path + '\'' + prefix + addy95836 + suffix + '\'' + attribs + '>' );
 document.write( addy95836 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">KDBIO / INESC-ID</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) KDBIO / INESC-ID, Rua Alves R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alves R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->edol 9, 1000-029 Lisboa,
Portugal (2) IST / Technical University of Lisbon, 1049-001 Lisboa, Portugal (3)
BIOI / ESAT-SCD, Katholieke Universiteit Leuven, Kasteelpark Arenberg 10,
B-3001 Leuven, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by FCT (INESC-ID
multiannual funding) through the PIDDAC Program funds and project ARN -
Algorithms for the Identi cation of Genetic Regulatory Networks,
PTDC/EIA/67722/2006. The work of JPG was supported by FCT doctoral grant
SFRH/BD/36586/2007. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection31">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744900">E-30. The role of incoherent microRNA-mediated
feedforward loops in noise buffering</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Osella M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Osella M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bosia C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bosia C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Corà D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Corà D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Caselle M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Caselle M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">MicroRNAs are endogenous non coding RNAs that play
important gene regulatory roles in animals and plants by pairing to the
messenger RNAs of protein-coding genes to direct their post-transcriptional
repression. Transcriptional and miRNA regulations are interlinked in a complex
network in which the microRNA-mediated feed forward loop is a remarkably
overrepresented regulatory circuit. The aim of our work is to suggest a
possible role of this network motif in controlling the fluctuations in the
target protein levels</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used both analytical and numerical methods. In
particular we described the circuit using a set of stochastic equations which
we then solved using the generating function approach and then tested our
results using Gillespie simulations.</p>

<h3>Results</h3>

<p class="abresults">We show that with respect to the simple gene activation by a
TF, the introduction of a miRNA-mediated repressing pathway can significantly
dampen fluctuations in the target protein output and that this noise buffering
function is a consequence of the peculiar topolgy of the FFL.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our model predicts that the optimal attenuation of
fluctuations coincides with a modest repression of the target expression. This
feature is coherent with a fine-tuning function and in agreement with
experimental observations of the actual impact of a wide class of microRNAs on
the protein output of their targets.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michele Caselle (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98577 = 'c&#97;s&#101;ll&#101;' + '&#64;';
 addy98577 = addy98577 + 't&#111;' + '&#46;' + '&#105;nfn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy98577 + suffix + '\'' + attribs + '>' );
 document.write( addy98577 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92767 = 'c&#97;s&#101;ll&#101;' + '&#64;';
 addy92767 = addy92767 + 't&#111;' + '&#46;' + '&#105;nfn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy92767 + suffix + '\'' + attribs + '>' );
 document.write( addy92767 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92679 = 'c&#97;s&#101;ll&#101;' + '&#64;';
 addy92679 = addy92679 + 't&#111;' + '&#46;' + '&#105;nfn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy92679 + suffix + '\'' + attribs + '>' );
 document.write( addy92679 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dep. of Theoretical Physics, Torino University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Theoretical Physics, University of
Torino (2) Systems Biology Lab (IRCC) University of Torino</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection32">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744901">E-31. Dynamic transcriptomics of helper T cell
differentiation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">van den Ham HJ<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;van den Ham HJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), de Waal L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;de Waal L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Zaaraoui F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zaaraoui F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Bijl M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bijl M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), van IJcken WF<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van IJcken WF&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(3), Osterhaus ADME<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Osterhaus ADME&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), de Boer RJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;de Boer RJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Andeweg AC<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andeweg AC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Helper T cells can adopt a number of different
phenotypes, including Th1, Th2, Th17, Th9, and Treg. We aim to study the gene
interacting network controlling the induction and development of these Th
phenotypes in time. Because the majority of genes change expression upon
activation of T cells, it is difficult to find Th phenotype associated genes.
The recently developed polar score method enables us to fish for differential
genes in a large set of genes that change expression. Using this dynamic
transcriptomics approach, we identify several candidate genes related to
Th1/Th2 differentiation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We performed microarray assisted mRNA profiling on
antigen specifically stimulated, TCR transgenic murine splenocytes that were
cultured in the presence of polarising cytokines. Transcriptome snapshots of Th
cells differentiating into Th1 and Th2 phenotypes were obtained.</p>

<h3>Results</h3>

<p class="abresults">Principle component analysis shows that: 1) activation, 2)
time since activation, and 3) Th skewing are the largest sources of variance in
our profiling experiments. Multiple immune regulatory pathways are
differentially expressed, including pathways specific to Th differentiation.
The dynamic transcriptomics approach, applying the polar score data analysis
method, enabled us to identify novel clusters of genes associated with Th1 and
Th2 differentiation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Among the Th1 and Th2 specific genes that we identify are
a number of key players previously associated with other T lymphocyte
phenotypes or the maturation of other lymphoid cell types. We hypothesise that
many of these genes play a universal role in cell differentiation, and are not
necessarily restricted to Th cells.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.virgo.nl" mce_href="http://www.virgo.nl">http://www.virgo.nl</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Henk-Jan van den Ham (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73974 = 'h.j.v&#97;nd&#101;nh&#97;m' + '&#64;';
 addy73974 = addy73974 + '&#101;r&#97;sm&#117;smc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy73974 + suffix + '\'' + attribs + '>' );
 document.write( addy73974 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">ErasmusMC, Rotterdam, The Netherlands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Theoretical Biology &amp; Bioinformatics, Utrecht
University, The Netherlands, 2 Dep. of Virology, 3 Erasmus Center for Biomics,
Erasmus MC, Rotterdam, The Netherlands * Presenting &amp; corresponding author:

 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34017 = 'h.j.v&#97;nd&#101;nh&#97;m' + '&#64;';
 addy34017 = addy34017 + '&#101;r&#97;sm&#117;smc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy34017 + suffix + '\'' + attribs + '>' );
 document.write( addy34017 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project was funded by the VIRGO consortium, an
innovative cluster approved by the Netherlands Genomics Initiative and
partially funded by the Dutch Government (BSIK 03012). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection33">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744902">E-32. Effect of size and heterogeneity of samples
on biomarker discovery: synthetic and real data assessment</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Di Camillo B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Di Camillo B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Martini M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Martini M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sanavia T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sanavia T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Jurman G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jurman G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Sambo F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sambo F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Barla A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Barla A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Squillario M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Squillario M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Furlanello C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Furlanello C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2),Toffolo G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Toffolo G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Cobelli C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cobelli C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The identification of robust lists of molecular
biomarkers related to a disease is a fundamental step for early diagnosis and
treatment. However, methodologies for the discovery of biomarkers using
microarray data often provide results with limited overlap. These differences
are imputable to 1) dataset size; 2) heterogeneity of the disease; 3) heterogeneity
of experimental protocols and computational analysis. Addressing issues 1 and
2, we assessed both on simulated and clinical datasets, the consistency of
candidate biomarkers provided by a number of different methods.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Three datasets of breast cancer patients ER+/- were
used to assess list stability. To assess average precision/ recall on a number
of studies, microarray data were simulated based on a gene network simulator,
accounting for intrinsic variability of the population, simulated based on a
model of evolution, and heterogeneity of the disease, simulated by modifying
subjects’ regulatory network. SAM and different methods for binary
classification and feature ranking (SVM, I-Relief, Spectral Regression
Discriminant Analysis), with/without external cross validation, were applied to
data.</p>

<h3>Results</h3>

<p class="abresults">Results on simulated data show that when some tens of
subjects are available per group, performance in terms of features ranking and
feature selection is good independently of the method. However, when available
subjects are equal or lower than 15, the multivariate machine learning methods
improve both the precision and the recall. The stability of the lists of
features obtained using SAM both on simulated and real data, is poorer than the
one given by the other methods. The use of external cross-validation loops
improves results reproducibility rather than precision and recall.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Altogether, these results show that although biomarkers
are difficult to be univocally identified, biological signatures are consistent
across different datasets when appropriate methods are used for the analysis.
In particular, classification based approaches improve both precision and
recall in feature selection, whereas using external cross-validation loops
improves the reproducibility of biomarker lists. The in silico approach allowed
us to outline advantages and drawbacks of different methods in a variety of
experimental conditions. Application to real data confirms these results.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Gianna Toffolo (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5305 = 'd&#105;c&#97;m&#105;ll' + '&#64;';
 addy5305 = addy5305 + 'd&#101;&#105;' + '&#46;' + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy5305 + suffix + '\'' + attribs + '>' );
 document.write( addy5305 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96337 = 'd&#105;c&#97;m&#105;ll' + '&#64;';
 addy96337 = addy96337 + 'd&#101;&#105;' + '&#46;' + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy96337 + suffix + '\'' + attribs + '>' );
 document.write( addy96337 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3134 = 'd&#105;c&#97;m&#105;ll' + '&#64;';
 addy3134 = addy3134 + 'd&#101;&#105;' + '&#46;' + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy3134 + suffix + '\'' + attribs + '>' );
 document.write( addy3134 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Padova</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Information Engineering Department, University of
Padova, via Gradenigo 6A, I-35131 Padova, Italy. (2) Fondazione Bruno Kessler,
via Sommarive 18, I-38123 Povo (Trento), Italy. (3) Department of Computer and
Information Science, University of Genova, via Dodecaneso 35, I-16146 Genova,
Italy.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">FUNDINGS This work was supported by CARIPARO 2008/2010
"Systems biology approaches to infer gene regulation from gene and protein
time series data"; ATENEO 2009/2010 "Methods for the integration of
background knowledge in kernel-based learning algorithms for the robust
identification of biomarkers in genomics"; RF Project 2007 Conv. 42.
ISITAD. ACKNOWLEDGEMENTS We would like to thank Dr. Marco Falda and Roberto
Visintainer for their help in running the simulations. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection34">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744903">E-33. Analysis of DNA sequence dependent structural
properties in prokaryotic genomes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rangannan V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Rangannan V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Bansal M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bansal M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Sequence dependent structural properties and their
variation in genomic DNA are important in controlling several crucial processes
such as transcription, replication, recombination and chromatin compaction. As
there is rapid accumulation of genomic data, quantitative analysis of sequences
motifs as well as structural properties, such as curvature, bendability and
stability in the upstream region of genes, has become very important in order
to automate the annotation pipeline for the newly sequenced genomes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Genomic sequences flanking the Transcription Start
Sites (TSSs) in E. coli, B. subtilis and M. tuberculosis have been taken from
Ecocyc, DBTBS and MtbRegList databases respectively. The stability of a double
stranded DNA molecule has been calculated using the free energy values of
dinucleotides steps. The bendability profiles were calculated using two
trinucleotide model values (DNase I sensitivity and Nucleosomal positioning
preference). Curvature has been calculated using the dinucleotide structural
parameters from crystal structure analysis and relative gel mobility data.</p>

<h3>Results</h3>

<p class="abresults">Promoter regions have a preponderance of AT-rich
tetranucleotides in E. coli and B. subtilis, not in high GC containing M.
tuberculosis. % occurrence of TATA containing sequences is less in promoter
regions as compared to other AT-containing tetramers. Average stability profile
showed a low stability peak near TSS in all three cases. Bendability profile
using both DNase I sensitivity and nucleosomal positioning preference models
showed less bendable region around TSS, but not for M. tuberculosis and other
GC rich genomes. High curvature in promoter regions is more pronounced in E.
coli.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The lower occurrence of AT-containing tetramers in the
vicinity of high GC containing promoters, may account for the absence of low
bendability peak in M. tuberculosis. Analysis of the structural features in the
promoter sequences of these three systems showed that lower bendability and
stability are seen in an equal number of promoters in E. coli and B. subtilis.
Curvature is observed in a smaller number of the promoter sequences as compared
to other two structural properties. Similar conclusions can be drawn from
analysis of regions upstream of genes in all prokaryotic genomes.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://nucleix.mbu.iisc.ernet.in/prombase" mce_href="http://nucleix.mbu.iisc.ernet.in/prombase">http://nucleix.mbu.iisc.ernet.in/prombase</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Vetriselvi Rangannan (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50563 = 'v&#101;tr&#105;' + '&#64;';
 addy50563 = addy50563 + 'mb&#117;' + '&#46;' + '&#105;&#105;sc' + '&#46;' + '&#101;rn&#101;t' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy50563 + suffix + '\'' + attribs + '>' );
 document.write( addy50563 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Molecular Biophysics Unit, Indian Institute of
science, Bangalore, INDIA</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Molecular Biophysics Unit, Indian Institute of
science, Bangalore-560012, INDIA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by Department of Biotechnology,
INDIA. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection35">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744904">E-34. Computational analysis of transcription
factor binding co-localization</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Martinez P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Martinez P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Blanc E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blanc E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Coolen A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Coolen A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Holzwarth J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Holzwarth J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Fraternali F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fraternali F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Transcription factors (TFs) are proteins that regulate
gene expression, therefore cooperativity among them provides a complex mechanism
to accurately control many biological processes. TFs display different modes of
cooperation, either directly binding to DNA or through protein interactions
with various partners and under different conditions. Because of this
variability in cooperative modes and the growing amount of data on
protein-protein interactions and regulatory networks, the analysis and the
validation of TFs' cooperation remains a challenge.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Scanning human and mouse promoter regions for fixed
nucleotide distances between pairs of DNA binding targets led to detection of
clears outliers. These outliers repesented set of genes for whom co-regulation
was evaluated using microarray data. In the last stage of the analysis, the
Reactome database of known pathways was used to characterize the most
interesting results.</p>

<h3>Results</h3>

<p class="abresults">Evidence of transcription factor cooperation based on DNA
sequence in promoter regions was found to bear consequences observed at the
protein expression stage at a statistically significant level. Results shed
light on the regulation of many known pathways and may be interesting hints for
new functional pathways yet to be annotated.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results obtained in our analysis confirm the
efficiency of our method for the detection of over-represented distances
between pairs of DNA binding targets. Furthermore, the use of gene expression
and known pathways data allowed us to suggest cooperation between transcription
factors for regulation known and unknown pathways. Our results would still need
experimental validation and analysis of a larger set of DBTs could provide more
specific information on the regulation of many biological pathways.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pierre Martinez (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4425 = 'p&#105;&#101;rr&#101;.m&#97;rt&#105;n&#101;z' + '&#64;';
 addy4425 = addy4425 + 'kcl' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy4425 + suffix + '\'' + attribs + '>' );
 document.write( addy4425 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">King's College London</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Randall Division of Cell and Molecular Biophysics,
King's College London, UK. 2) MRC Centre for Developmental Neurobiology, King's
College London, UK. 3) Nestlé Research Centre, Lausanne, Switzerland.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Engineering and Physical Sciences Research Council
(EPSRC - UK) Nestlé (Switzerland) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection36">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744905">E-35. Cross-checking experimental results with
publicly available gene expression data: a query-driven approach</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Smet R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Smet R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Hermans K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hermans K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Keersmaecker S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Keersmaecker S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Marchal K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marchal K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Different experimental assays highlight different aspects
of the studied biology. Considering the wide availability of public microarray
data an appealing approach is to interpret the results of experimental assays
in light of these data. The exploration of such compendia for experimental
results requires a query-driven search approach. Existing query-driven
approaches, however, either fail to deal with the potential functional
diversity of the experimental gene set or require the method to be run on each
gene separately, resulting in a large number of solutions that needs to be
evaluated.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We introduce a new computational tool which builds
on an existing query-driven biclustering strategy and that given a list of
genes derived from an experimental assay identifies distinct biclusters in a
gene expression compendium to which the genes within the list belong. The
method implements principles from consensus clustering and graph clustering to
aggregate multiple query-driven biclustering results, each of them derived for
a single query-gene, in post-processing. In particular different ways of
assembling the query-driven biclustering results into a consensus solution are
evaluated.</p>

<h3>Results</h3>

<p class="abresults">To illustrate the effectiveness of this approach the method
was applied to an Escherichia coli (ChIP-chip experiment) and Salmonella
Typhimurium (single cell experiment) case study. The approach efficiently
removes the redundancy amongst the query-driven biclustering results and avoids
defining separate thresholds for each query-driven biclustering solution. In
particular genes from the experimental assays are separated into multiple
consensus biclusters, allowing for evaluation of the experimental results in
terms of the genes and the conditions belonging to the bicluster.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We illustrate that the here introduced approach might
reveal different aspects of the studied biology and as such allow for a more
holistic interpretation of the experimental results. In addition, it might
reveal experimental inconsistencies.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Riet De Smet (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87348 = 'r&#105;&#101;t.d&#101;sm&#101;t' + '&#64;';
 addy87348 = addy87348 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy87348 + suffix + '\'' + attribs + '>' );
 document.write( addy87348 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">KULeuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Microbial and Molecular systems,
Katholieke Universiteit Leuven, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by: 1) KUL: GOA AMBioRICS,
GOA/08/011, CoE EF/05/007, SymBioSys; CREA/08/023 2) IWT: SBO-BioFrame; 3) IUAP
P6/25 (BioMaGNet); 4) FWO IOK-B9725-G.0329.09; 6) HFSP-RGY0079/2007C. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection37">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744906">E-36. Using tiling microarrays to predict
transcription start sites: application to Lactobacillus plantarum WCFS1</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Todt T<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Todt T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Wels M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wels M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,4), Siezen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen RJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,4,5), van Hijum SAFT (1,3,4,5),
Kleerebezem M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Kleerebezem M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4,5)</p>

<p class="abmotivation">Prediction of transcription start sites (TSS) in prokaryotes
allows to accurately determine upstream cis regulatory promoter sequences.
Microarray tiling arrays offer a cost-effective alternative to transcriptome
sequencing to determine approximate TSS locations for a large complement of
genes in a given organism. Our goal is to predict genome-wide TSS by combining
existing gene annotation information of L. plantarum WCFS1 with whole-genome
tiling microarray expression data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In order to calibrate our algorithm, a training
gene set was compiled using the following criteria: (1) gene shows expression
signal in tiling array data, (2) predicted promoter location is upstream (up to
1 kb) of the gene, and (3) a predicted transcription start site is situated
between predicted promoter location and gene.</p>

<h3>Results</h3>

<p class="abresults">In this gene set of 620 genes, we found that 53% of the
predicted transcription start sites are in close proximity (up to 30
nucleotides) to predicted promoter locations. For the other TSS, predicted
promoter locations were further upstream or were not predicted yet.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We are currently adapting our method to predict
genome-wide TSS and to identify new promoter sequences based on these
approximate TSS locations. Current results indicate that the resolution of
tiling arrays limits determining exact TSS locations. Transcriptome sequencing
would be a more suited means to this end.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tilman Todt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36472 = 't&#105;lm&#97;.t&#111;dt' + '&#64;';
 addy36472 = addy36472 + 'h&#97;n' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy36472 + suffix + '\'' + attribs + '>' );
 document.write( addy36472 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39316 = 't&#105;lm&#97;.t&#111;dt' + '&#64;';
 addy39316 = addy39316 + 'h&#97;n' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy39316 + suffix + '\'' + attribs + '>' );
 document.write( addy39316 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61148 = 't&#105;lm&#97;.t&#111;dt' + '&#64;';
 addy61148 = addy61148 + 'h&#97;n' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy61148 + suffix + '\'' + attribs + '>' );
 document.write( addy61148 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">HAN</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Radboud University Nijmegen Medical Centre, Centre
for Molecular and Biomolecular Informatics, P.O. Box 9010, 6500 GL Nijmegen,
the Netherlands; 2 HAN University, P.O. Box 6960, 6503 GL Nijmegen, the
Netherlands; 3 NIZO food research, P.O. Box 20, 6710 BA Ede, the Netherlands; 4
TI Food and Nutrition, P.O. Box 557, 6700 AN Wageningen, the Netherlands 5
Kluyver Centre for Genomics of Industrial Fermentation, P.O. Box 5057 2600 GA
Delft, the Netherlands. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection38">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744907">E-37. Predicting regulatory networks from large
scale time-course microarray data under several stimulation conditions on MCF-7
cells</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Shiraishi Y<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Shiraishi Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Saeki Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Saeki Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Nagashima T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nagashima T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Yumoto N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yumoto N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Takahashi K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Takahashi K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Okada M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Okada M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Distinct extracellular stimulations lead to different
biological regulations in many cell lines. For example, epidermal growth factor
(EGF) induces proliferation whereas heregulin (HRG) induces differentiation in
MCF7 human breast cancer cells. Furthermore, several researches implied that
multiple signaling transduction pathways such as mitogen-activated protein
kinase (MAPK) pathway and phosphoinositide 3-kinase (PI3K) - Akt pathway play
important roles to determine these cell decisions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To understand the relationships between ligands,
pathways and biological regulations, we analyzed large scale time-course microarray
data collected under several stimulation conditions (estrogen, EGF, HRG, MAPK
inhibitors, Akt inhibitors, etc.) using MCF7 cells. First, we extracted several
gene sets with similar expression patterns on many stimulation conditions using
some statistical methods. Then we investigated the feature of each gene set by
performing enrichment analysis of transcription factor binding motifs and gene
function annotations.</p>

<h3>Results</h3>

<p class="abresults">Our approaches found several important gene sets, to which
some transcription factors known to serve significant roles on breast cancer
are associated. Furthermore, we performed wet-lab analysis on some of the
transcription factors identified above and confirmed that their activity levels
change dynamically on corresponding stimulation conditions. Finally, using
extracted gene sets and its characteristics, we constructed a putative network
which describes how each ligand activates transcription factors through
signaling pathways and induces biological regulations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our results will be important implications for biological
researchers, especially those dealing with MCF-7 cells. We believe that using
gene expression patterns with some perturbations, e.g., inhibitors is a
promising approach for inferring gene regulatory networks, because it will
increase the variety of expression patterns and help to classify the gene sets.
Still, dealing with large scale time-course microarray data under multiple
conditions is not a simple task. It is necessary to develop statistical methods
for taking advantage of multiple stimulation conditions further.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Yuichi Shiraishi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6480 = 'ysh&#105;r&#97;' + '&#64;';
 addy6480 = addy6480 + 'r&#105;k&#101;n' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy6480 + suffix + '\'' + attribs + '>' );
 document.write( addy6480 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31901 = 'ysh&#105;r&#97;' + '&#64;';
 addy31901 = addy31901 + 'r&#105;k&#101;n' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy31901 + suffix + '\'' + attribs + '>' );
 document.write( addy31901 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11418 = 'ysh&#105;r&#97;' + '&#64;';
 addy11418 = addy11418 + 'r&#105;k&#101;n' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy11418 + suffix + '\'' + attribs + '>' );
 document.write( addy11418 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">RIKEN RCAI</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) RIKEN Research Center for Allergy and Immunology</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Grant-in-Aid for Young Scientists (B) 21700316 of the
Ministry of Education, Culture, Sports, Science and Technology. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection39">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744908">E-38. Inferring regulatory networks from expression
data using tree-based methods</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Huynh-Thu VA<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Huynh-Thu VA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Irrthum A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Irrthum A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Wehenkel L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wehenkel L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Geurts P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Geurts P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">One of the pressing open problems of computational
systems biology is the elucidation of the topology of genetic regulatory
networks (GRNs) using high throughput genomic data, in particular microarray
gene expression data. The Dialogue for Reverse Engineering Assessments and
Methods (DREAM) challenge aims to evaluate the success of GRN inference
algorithms on benchmarks of simulated data. In this article, we present a new
algorithm for the inference of GRNs that was best performer in the DREAM4 In
Silico Multifactorial challenge.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our GRN inference algorithm decomposes the
prediction of a regulatory network between p genes into p different regression
problems. In each of the regression problems, the expression pattern of one of
the genes (target gene) is predicted from the expression patterns of all the
other genes (input genes), using tree-based ensemble methods. The importance of
an input gene in the prediction of the target gene expression is taken as an
indication of a putative regulatory link. The whole network is then
reconstructed by aggregating putative links over all genes.</p>

<h3>Results</h3>

<p class="abresults">We applied our algorithm on the DREAM4 In Silico
Multifactorial challenge simulated data. The goal of this challenge was to
infer five networks of 100 genes from static steady-state expression profiles
resulting from slight perturbations of all genes. Our method got the best
performance among twelve challengers. Experiments were also carried out using
907 Escherichia coli Affymetrix arrays and 3433 known E. Coli regulatory
interactions from RegulonDB. They show that our algorithm compares favorably
with existing algorithms to decipher the genetic regulatory network of
Escherichia coli.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In conclusion, we propose a new algorithm for GRN
inference that performs well on both synthetic and real gene expression data.
The algorithm, based on feature selection with tree-based ensemble methods,
does not make any assumption about the nature of gene regulation, can deal with
combinatorial and non-linear interactions, produces directed GRNs, and is fast
and scalable. It is furthermore simple and generic, making it adaptable to
other types of genomic data and interactions.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.montefiore.ulg.ac.be/%7Ehuynh-thu/software.html">http://www.montefiore.ulg.ac.be/~huynh-thu/software.html</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Vân Anh Huynh-Thu (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy30720 = 'v&#97;h&#117;ynh' + '&#64;';
 addy30720 = addy30720 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy30720 + suffix + '\'' + attribs + '>' );
 document.write( addy30720 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38982 = 'v&#97;h&#117;ynh' + '&#64;';
 addy38982 = addy38982 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38982 + suffix + '\'' + attribs + '>' );
 document.write( addy38982 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27450 = 'v&#97;h&#117;ynh' + '&#64;';
 addy27450 = addy27450 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy27450 + suffix + '\'' + attribs + '>' );
 document.write( addy27450 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Liège</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Electrical Engineering and Computer
Science, Systems and Modeling, University of Liège, Liège, Belgium (2)
GIGA-Research, Bioinformatics and Modeling, University of Liège, Liège, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially funded by the Interuniversity
Attraction Poles Programme (IAP P6/25 BIOMAGNET), initiated by the Belgian
State, Science Policy Office and by the European Network of Excellence PASCAL2.
VAHT is recipient of a F.R.I.A. fellowship and PG is Research Associate of the
F.R.S.-FNRS, Belgium. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection40">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744909">E-39. Dynamic profile of transcription factors
inferred from mRNA expression time courses in Gefitinib-treated lung cancer
cells</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nagao H<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Nagao H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Yoshida R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yoshida R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Saito M-M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Saito M-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Imoto S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Imoto S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Nagasaki M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nagasaki M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Yamaguchi R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yamaguchi R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Yamauchi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yamauchi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Goto N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Goto N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Miyano S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Miyano S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Higuchi T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Higuchi T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Transcription regulatory pathways respond to stimuli by
changing activities of transcription factors (TFs) and enhancing rates of gene
decoding. DNA microarray experiments enable us to measure transcript levels of
mRNA molecules that reflect underlying activity of TFs. Our study builds on a
reverse-engineering system to reconstruct temporal profiles of activating TFs
using given mRNA time courses. The proposed method is tested on lung cancer
cells, clarifying differences in TFs involving pathways between Gefitinib-sensitive/resistive
lung tumors treated with epidermal growth factor (EGF).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We measure gene expression time courses of
Gefitinib-sensitive/resistive lung tumor cells (PC9 and PC9GRM2), treated with
EGF and/or Gefitinib. To understand differences in TFs involving pathways
between the different types of cells, for each of the experimental conditions,
the proposed method reconstructs temporal profiles of activating TFs called
"TF dynamic maps” by statistically evaluating significances of synchronization
in temporal gene expression patters involving a particular TF. The list of TFs
and their target genes were retrieved from TRANSPATH database.</p>

<h3>Results</h3>

<p class="abresults">By comparisons of the obtained TF dynamic maps, we received
preliminary results as follows; (1) key TFs on Gefitinib-response pathways were
clearly identified from PC9 profiles, (2) the TF dynamic maps uncovered sets of
synchronizing TFs dynamically controlled during propagations of EGF and/or
Gefitinib stimuli, (3) PC9GRM2 showed significantly lower responses than PC9 in
both EGF and EGF+Gefitinib treated cells, (4) Activity levels of most
transcriptional activators and repressors decreased and increased respectively
at the almost same timing (around 20 hours after stimulation) in PC9GRM2.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The constructed TF dynamic maps are utilized in
understanding which TFs play significant regulatory roles in EGF and/or
Gefitinib treated tumor cells and how the Gefitinib acts on TFs dynamically.
More interestingly, conduction of a more detailed analysis can distinguish
controllable and uncontrollable TFs that are able/unable to induce significant
changes in gene regulatory systems by conducting intervention experiments. A
list of controllable and uncontrollable TFs can be vital information in design
of molecular-targeted drug.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Hiromichi Nagao (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90884 = 'hn&#97;g&#97;&#111;' + '&#64;';
 addy90884 = addy90884 + '&#105;sm' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy90884 + suffix + '\'' + attribs + '>' );
 document.write( addy90884 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94229 = 'hn&#97;g&#97;&#111;' + '&#64;';
 addy94229 = addy94229 + '&#105;sm' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy94229 + suffix + '\'' + attribs + '>' );
 document.write( addy94229 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39318 = 'hn&#97;g&#97;&#111;' + '&#64;';
 addy39318 = addy39318 + '&#105;sm' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy39318 + suffix + '\'' + attribs + '>' );
 document.write( addy39318 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Institute of Statistical Mathematics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. The Institute of Statistical Mathematics 2. The
Institute of Medical Science, the University of Tokyo</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is supported by the next-generation
supercomputer project of RIKEN. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection41">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744910">E-40. Deciphering the genomic plasticity of
bacterial species</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Janky R<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Janky R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Harvill ET<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Harvill ET&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Babu MM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Babu MM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The genomes of bacterial species show enormous plasticity
in the function of individual genes, reflected by a rapid evolution in genome
organization and regulatory interactions. One of the major challenges is to
decipher the mechanisms of evolution of gene regulation which leads to
bacterial adaptation. We developed a computational procedure and compared the
predicted regulatory systems from four closely related bacterial species in the
genus Bordetellae, of which three are pathogenic with different host
specificity.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used a phylogenetic footprint discovery method
to detect conserved regulatory motifs in promoter regions of
Beta-proteobacteria. To study the regulatory motifs evolution within Bordetella
species, we defined a conservation profile according to its presence or not in
their promoters. The distribution of predicted motifs in different phylogenetic
profile was compared to those from random selection of Bordetella promoters.</p>

<h3>Results</h3>

<p class="abresults">We found that the distribution of predicted motifs on the
Bordetella promoter sequences is not random. Cis-regulatory motifs matching
pathogenic sequences are significantly over-represented, while others detected
in any combination of non-pathogenic and pathogenic promoter sequences are
significantly under-represented. Two case studies with enriched pathogenic
regulations will be discussed.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Such comparative approach at the level of regulatory
analysis may allow us to predict pathogen-specific regulatory interactions and
provide new insights into bacterial adaptation due to the evolution of the
regulation. We will apply this approach to other taxonomic groups with
pathogenic species.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Rekin's Janky (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11478 = 'rj&#97;nky' + '&#64;';
 addy11478 = addy11478 + 'mrc-lmb' + '&#46;' + 'c&#97;m' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy11478 + suffix + '\'' + attribs + '>' );
 document.write( addy11478 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Medical Research Council - Laboratory of Molecular
Biology.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Structural Studies Division. Medical Research
Council - Laboratory of Molecular Biology. Hills Road, Cambridge, CB2 0QH.
United Kingdom. (2) Department of Veterinary and Biomedical Science. The
Pennsylvania State University, USA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">M.M.B. and R.J. acknowledge the Medical Research
Council for financial support. R.J. was also supported by a post-doctoral grant
from the Wiener-Anspach Foundation (Belgium). M.M.B. thanks Darwin College,
Cambridge and Schlumberger Ltd. for generous support. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection42">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744911">E-41. Expression arrays &amp; outlying processes:
building a database of outliers</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Berger F<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Berger F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Kroll KM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kroll KM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), van Dorp M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Dorp M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Carlon E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Carlon E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Expression arrays are used to study the expression
profile of the whole genome. Focusing on Affymetrix expression arrays, Kroll M.
&amp; Carlon E<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Carlon E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->. recently introduced affyILM,
a package estimating transcript concentrations. Their method makes use of
hybridization free energies, extended Langmuir isotherm, and physical features of
neighboring spots. In contrast to other methods, the output is target
concentrations measured in pM (picoMolar). Based on this approach, we aim to
identify probes that are systematically detected as outliers, as well as to
investigate the associated outlying processes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To develop the database, we used the MySQL database
management system. The interface has been developped using the php scripting
language, and is hosted on the apache http server. Datasets were retrieved from
the EBI ArrayExpress repository (accession ids : E-GEOD-994 and E-TABM-145). The
analyses were performed using R, Bioconductor and the affyILM package. Sequences
alignments were performed using the NCBI blastn methodology (short sequences
algorithm), against GenBank and RefSeq human nucleotide sequences.</p>

<h3>Results</h3>

<p class="abresults">We build an exploratory database of probes features,
focusing on the HG-U133A chip model. Theoretical outliers are retrieved from
previous publications, and from alignment results between probes and targets.
Empirical detection of outliers is performed assuming that concentration values
within a probe-set follow the normal distribution. A php-interface compares the
concentrations associated to transcript-specific probes from publicly available
datasets. By means of our web interface, the database can be used in
combination with public datasets to track the behavior of outlying probes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The outlier database and the web interface will be used
to highlight the outlying processes that are involved. In the future, we plan
to modify affyILM to improve the concentration estimation of probes associated
to specific outlying processes (appropriate estimation of hybridization free
energies). In addition, the current database will be enlarged with other Affymetrix
expression arrays.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Fabrice Berger (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40767 = 'f&#97;br&#105;c&#101;.b&#101;rg&#101;r' + '&#64;';
 addy40767 = addy40767 + 'fys' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy40767 + suffix + '\'' + attribs + '>' );
 document.write( addy40767 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">KULeuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Institute for Theoretical Physics, KULeuven</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection43">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744912">E-42. Detecting significantly correlated gene
clusters and their interactions in a continuous scale-space</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Van Dyk E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Van Dyk E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Reinders M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinders M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Wessels L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wessels L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Biological complexity stems from the large number of
possible interactions between molecular entities occurring at different levels
of modularity. Low-level (i.e. gene-gene) interactions include co-operating
interactions and mutually exclusive interactions. Examples of higher-level
interactions are module (collection of genes) interactions. Like single genes,
modules can be activated or repressed. Modules can also be mutually exclusively
and co-occurrently activated. These gene or module level interactions could
reveal critical steps in tumorigenesis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We are mainly interested in defining gene modules
on a continuous scale space using kernel convolution methods applied to
microarray gene expression data from breast-cancer tumors. We map human genes
on their genomic positions and create a correlation matrix where each entry
represents the Pearson correlation of a gene pair over all samples. In this
newly defined space, we perform kernel smoothing (with different kernel widths)
to reveal significant clusters at different scales. We compensate for the
non-uniformity of the gene positioning in this space to maximize the power of significance.</p>

<h3>Results</h3>

<p class="abresults">We present theoretical methods for finding statistically
significant clusters in the kernel-convoluted space (non-randomly occurring
clusters). These include methods for minimizing false-negative rates (chance of
missing clusters) and approaches for controlling the FWE rate
(false-positives). We compute a significance threshold that in general depends
on the position in space and thus compensates for the non-uniform spacing of
genes. We demonstrate this approach on two breast cancer gene expression data
sets obtained from different micro-array platforms (Agilent and Affymetrix
respectively).</p>

<h3>Discussion</h3>

<p class="abdiscussion">We focus on finding clusters of genes that are
significantly correlated. These clusters can be defined as modules of genes
that are close together on the genome and are identified by significant peaks
on the diagonal of the smoothed correlation matrix. Clusters far from the
diagonal represent interactions between modules (such as co-occurring or
mutually exclusive). This also allows for the discovery of modules at different
scales and inter-scale interactions. Identification of stable modules in
different data sets can be integral for providing robust feature extraction
tools.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ewald van Dyk (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy25022 = '&#101;.v.d&#105;jk' + '&#64;';
 addy25022 = addy25022 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy25022 + suffix + '\'' + attribs + '>' );
 document.write( addy25022 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39 = '&#101;.v.d&#105;jk' + '&#64;';
 addy39 = addy39 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy39 + suffix + '\'' + attribs + '>' );
 document.write( addy39 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55610 = '&#101;.v.d&#105;jk' + '&#64;';
 addy55610 = addy55610 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy55610 + suffix + '\'' + attribs + '>' );
 document.write( addy55610 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Netherlands Cancer Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Bioinformatics and statistics, Division of Molecular
Biology, The Netherlands Cancer Institute, Amsterdam, The Netherlands 2 Delft
Bioinformatics Laboratory, Faculty of EEMCS, Delft University of Technology,
Delft, The Netherlands</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection44">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744913">E-43. MAMMOD: Multi-Agent Motif and MOdule
Discovery</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Van Delm W<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Van Delm W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Ayoubi T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ayoubi T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Moor B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Moor B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Deciphering non-coding DNA is a challenging task. The
first generation of tools searched for binding sites of a single transcription
factor (TF) at a time. A thorough community assessment in 2004 however revealed
that much work remained to be done. Gradually techniques became more advanced,
looking for clusters of binding sites of a collection of TFs. Very recently,
the flood of new methods was accompanied by new test benches to validate and
compare alternatives. Despite the improvements, many predictions still appear
to be false positives.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We adopt a Markov random field (MRF) that connects
the unknown variables with data. Data of sequence conservation and protein
interaction is included. The configuration of bound TFs is modeled with a
semi-hidden Markov model. The other components of the MRF are classical
statistical distributions, of which the parameters are fitted with
expectation-maximization methods. Samples are drawn from the posterior
distribution with a novel sampling scheme, that generalizes a grouped Gibbs
sampling algorithm. We use Friedman tests to statistically compare performance
measures of different algorithms.</p>

<h3>Results</h3>

<p class="abresults">By using both simulated and benchmark data sets of skeletal
muscle and liver tissue, we demonstrate that our novel algorithm for Bayesian
inference can compete with several state-of-the-art methods in predicting
relevant transcription factors and cis-regulatory modules.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have introduced a general computational framework for
Bayesian inference of cis-regulatory motifs and modules in eukaryotic
promoters. The framework allows the integration of alternative algorithms and
data sources. Further investigation of the algorithmic behavior is necessary to
control better prediction specificity, robustness to noise and improper
modeling, and especially computation time.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Wouter M.H.J. Van Delm (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89942 = 'wv&#97;nd&#101;lm' + '&#64;';
 addy89942 = addy89942 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy89942 + suffix + '\'' + attribs + '>' );
 document.write( addy89942 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">VIB - MicroArray Facility</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Dept. Electrical Engineering ESAT-SDC, Katholieke
Universiteit Leuven, (2) MicroArray Facility, VIB</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is funded by a Ph.D. grant of the
Institute for the Promotion of Innovation through Science and Technology in
Flanders (IWT-Vlaanderen). We are grateful to the Department of Electrical
Engineering (ESAT-SCD) of the Katholieke Universiteit Leuven and the Flemish
Institute for Biotechnology (VIB) for their scientific support. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection45">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744914">E-44. Modulon identification from bacterial gene
expression array data using an improved supervised approach</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hedge S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Hedge S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Permina E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Permina E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,*), Medvedeva Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Medvedeva Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Mande S C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mande S C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Makeev V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Makeev V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">Inferring regulatory pathways from microarray data is an
important preparatory stage for system biology modeling of a bacterial cell. A
variety of methods have been developed to solve this problem. Using a
traditional approach, researchers focus on finding clusters of correlated
expression values over the set of various conditions. This leaves us with the
major problem of improving the quality of resulting data</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We suggest a strategy dealing with microarray data,
starting with identifying noise component in expression array, exploiting the
idea that pairs of genes belonging to the same operon make a natural set of
positively correlated pairs of genes. Negatively correlated genes observed in
the set of operon pairs allow calculating the noise component. The next step is
identifying clusters of coexpressed genes starting from a set of known genes
which are divided into two subsets representing the essential and optional
parts of the modulon in order to make the approach more flexible</p>

<h3>Results</h3>

<p class="abresults">We have develop an easy-to-handle software for calculating
the cliques of potentially coexpressed genes and have applied our technique to
LexA, PurR, and heat shock (Sig32) operons in E. coli, verifying questionable
regulon members and predicting the new ones. We succeeded in prediction of
additional modulon participants and have compared the results obtained from
several independent datasets. Our results are consistent with the available
experimental data</p>

<h3>Discussion</h3>

<p class="abdiscussion">The work demonstrates the importance of using reference
sets for which the correlation data can be calculated a priori and the
advantage of supervised method for dealing with microarray data. The use of
structured set of known genes allows to get clusters of potentially coexpressed
genes which are more compact and reasonable from the biological point of view.
Coordinated expressions of genes belonging to different regulons renders
possible revealing the global structure of bacterial regulatory network.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Elizabeth A. Permina (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99665 = '&#101;n&#101;lk&#105;ns&#97;n' + '&#64;';
 addy99665 = addy99665 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy99665 + suffix + '\'' + attribs + '>' );
 document.write( addy99665 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">GosNIIGenetika</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Centre for DNA Fingerprinting and Diagnostics,
India; (2) State Research Institute for Genetics and Selection of Industrial
Microorganims, GosNIIgenetika, Russia; (3) Engelhardt Institute of Molecular
Biology, Russia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by Russian Fund of
Basic Research Project 08-04-91318 and Russian President Young Scientist Grant
MK-4805.2009.4</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection46">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744915">E-45. GCQM: quality assessment of microarray
studies and samples using gene correlations</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Venet D<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Venet D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Detours V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Detours V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Bersini H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bersini H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">A large amount of high-throughput gene expression data is
available in public repositories. However, the quality of those data can vary
dramatically from platform to platform, study to study and sample to sample.
Determining data quality is of the utmost importance. Current quality measures
are platform-specific, and cannot be used to compare studies. We propose here a
biology-motivated quality measure, the Gene Correlation Quality Measure (GCQM),
based on the consistency of gene-gene correlations.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">GCQM allows for the estimation of qualities of both
studies and samples. First, the average of all gene-gene correlation matrices in
our database was calculated. Then, the quality of studies is obtained by
comparing its gene-gene correlation matrix with the expected correlations. The
relative quality of a sample is obtained by comparing the quality of its study
with and without that sample.</p>

<h3>Results</h3>

<p class="abresults">the quality of studies, samples and platforms. This allowed
to discover serious issues with three studies. We show that qualities of both
studies and samples are linked to the statistical significance of the
biological results, hence higher quality studies have a higher signal/noise
ratio. We show that GCQM does give a higher quality to biological outliers than
warranted, so that those will not be discarded. We also determined an average
quality for many platforms.</p>

<h3>Discussion</h3>

<p class="abdiscussion">It could come as a surprise that average gene-gene
correlations allow for the determination of data quality. It works because
gene-gene correlations have a distribution which is meaningful for any study,
being largely based on genes that belong to the main cellular processes (e.g.
respiration, cell cycle, proliferation). Biological variability slightly up- or
down-regulates those processes in every sample, creating those correlations.
The quality of biological outliers is over-valued because GCQM is a
signal/noise measure.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">David Venet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46508 = 'd&#97;v&#101;n&#101;t' + '&#64;';
 addy46508 = addy46508 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy46508 + suffix + '\'' + attribs + '>' );
 document.write( addy46508 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17325 = 'd&#97;v&#101;n&#101;t' + '&#64;';
 addy17325 = addy17325 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy17325 + suffix + '\'' + attribs + '>' );
 document.write( addy17325 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12669 = 'd&#97;v&#101;n&#101;t' + '&#64;';
 addy12669 = addy12669 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy12669 + suffix + '\'' + attribs + '>' );
 document.write( addy12669 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 IRIDIA, Université Libre de Bruxelles, 50 Av. F.
Roosevelt, CP 194/6, B-1050 Brussels, Belgium 2 IRIBHM, Université Libre de
Bruxelles, Campus Hospitalo-Facultaire Erasme, 808 route de Lennik, B-1070
Brussels, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by the ICT Impulse program 2006,
Brussels Capital Region, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection47">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744916">E-46. Integration of correlation structure improves
performance of pathway-based classification</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cho SB<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Cho SB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Kim JH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kim JH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3)</p>

<p class="abmotivation">In a pathway-based classification analysis, considering
the correlation structure between genes is important. We have developed a
boosting kernel density Bayesian classifier (boostKDBC) algorithm to capture
the correlation structure during classification.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The pathway expression profiles from four breast
cancer data sets and simulation data were tested to validate the boostKDBC
method. Multivariate kernel density estimation was plugged in boosting
procedure.</p>

<h3>Results</h3>

<p class="abresults">In real data analysis, the boostKDBC algorithm outper-formed
other benchmark algorithms. Simulation study indicated that the performance of
the boostKDBC algorithm was superior to other algorithms with data having a
gene–gene correlation structure. The pathways listed in the results provide
many invaluable hypotheses on the pathophysiologic mechanism of breast cancer.</p>

<h3>Discussion</h3>

<p class="abdiscussion">It seeemed that boosting reinfored accuracy of
conditional density estimation, which led to better perforamnce than any other
benchmark algorithms. Based on our results, we assume that the boostKDBC
algorithm is appropriate for pathway-based classifica-tion analysis.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">SungBum Cho (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57935 = 'sbch&#111;' + '&#64;';
 addy57935 = addy57935 + 'k&#111;r&#101;&#97;' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy57935 + suffix + '\'' + attribs + '>' );
 document.write( addy57935 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34276 = 'sbch&#111;' + '&#64;';
 addy34276 = addy34276 + 'k&#111;r&#101;&#97;' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy34276 + suffix + '\'' + attribs + '>' );
 document.write( addy34276 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83652 = 'sbch&#111;' + '&#64;';
 addy83652 = addy83652 + 'k&#111;r&#101;&#97;' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy83652 + suffix + '\'' + attribs + '>' );
 document.write( addy83652 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">center for disease control, korea</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Division of biomedical informatics, National
Institute of Health, Korea, Seoul 122-401. (2) Seoul National University
Biomedical Informatics (SNUBI) (3) Division of Biomedical Informatics, Seoul
Na-tional University College of Medicine, Seoul 110-799, Korea</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection48">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744917">E-47. Multi Experiment Matrix: web tool for mining
co-expressed genes over hundreds of datasets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Adler P<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Adler P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Kolde R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kolde R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Kull M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kull M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Peterson H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Peterson H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Reimand J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reimand J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Tkatšenko A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tkatšenko A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Vilo J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vilo J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3)</p>

<p class="abmotivation">Accumulation of gene expression data in public domain has
raised the opportunity to discover new facts by re-analyzing the existing
experiments. Co-expression over many gene expression datasets has been proven
useful in many areas of molecular biology and bioinformatics, such as network
reconstruction and gene function prediction. However, the tools for this are
lacking. Here, we present MEM, a web-based resource for ﬁnding co-expressed
genes over large collections of public microarray experiments.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The search goes over hundreds of gene expression
datasets from ArrayExpress covering wide spectrum of biological conditions from
stem cells to different diseases and cancer. The essence of MEM is a novel rank
aggregation method that combines similarity searches across individual data
sets into a global ordering by selecting automatically the appropriate
experiments and assigning a combined p-value for the significance of the
similarity across all data. For more speciﬁc queries we have various mechanisms
for dataset selection.</p>

<h3>Results</h3>

<p class="abresults">As a result we have developed a web-based tool to query over
many hundreds of gene expression datasets to find similarly expressed genes.
Using known transcription factor NANOG as an example gene, First we demonstrate
how we are able to identify relevant datasets for query. Then looking for known
targets of NANOG among top similarly expressed genes in each individual dataset
separately and using MEM, we show that using our rank-aggregation schema to
combine rankings from each dataset improves the overall yield.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have developed a tool for finding co-expressed genes
over many public gene expression datasets. The tool has many possible
applications, such as gene function prediction or network reconstruction. The
search is powered by a novel rank aggregation algorithm. User interface is
highly interactive and provides additional information about the query and the
results.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://biit.cs.ut.ee/mem" mce_href="http://biit.cs.ut.ee/mem">http://biit.cs.ut.ee/mem</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Priit Adler (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98373 = '&#97;dl&#101;r' + '&#64;';
 addy98373 = addy98373 + '&#117;t' + '&#46;' + '&#101;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy98373 + suffix + '\'' + attribs + '>' );
 document.write( addy98373 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88383 = '&#97;dl&#101;r' + '&#64;';
 addy88383 = addy88383 + '&#117;t' + '&#46;' + '&#101;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy88383 + suffix + '\'' + attribs + '>' );
 document.write( addy88383 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1168 = '&#97;dl&#101;r' + '&#64;';
 addy1168 = addy1168 + '&#117;t' + '&#46;' + '&#101;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy1168 + suffix + '\'' + attribs + '>' );
 document.write( addy1168 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Tartu, IMCB</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Molecular and Cell Biology, University
of Tartu (2) Institute of Computer Science, University of Tartu (3) Quretec Inc</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work has been supported by EU FP6 ENFIN, FunGenES,
Estonian Science Foundation 5724, 7437</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection49">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744918">E-48. Linking parallel measurements of
high-throughput miRNA, gene and protein expression data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gade S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Gade S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Binder H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Binder H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Beißbarth T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beißbarth T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">The parallel measurement of different types of high
throughput expression data, like gene, miRNA and protein data, gives rise to
new tasks of data integration, specifically, when also the combined effect on a
clinical endpoint is of interest. Different strategies have been formulated,
performing the integration on different levels of the analysis process.
Specifically, miRNAs are small non-coding RNAs which play an important role
during cancer development. The details of the miRNA mediated regulation
mechanism are not fully understood. Though it is known that the binding of a
miRNA to its target transcript can lead to degradation and consequently to a measurable
change in the level of the mRNA.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We proposed a graph-based work flow to integrate
miRNA and gene expression data. By combining the correlation structure of the
two data sets with target predictions we were able to construct a bipartite
graph with connections between miRNAs and genes. Meta information were added
and used to distinguish reasonable sub clusters in this graph. This structure
was used to perform an over-representation analysis of targets and effected
pathways. Furthermore, the bipartite graph is interpretable as a pathway
linking different kinds of features. Similar to gene pathways it is suitable as
an additional knowledge for classification tasks.</p>

<h3>Results</h3>

<p class="abresults">We applied this method to a colorectal cancer data set consisting
of 97 samples and measurements of miRNA and gene expressions. With help of the
graph structure we were able to identify clusters of miRNAs with significant
correlations to genes in cancer related pathways like cytokine-cytokine
receptor interaction and the Jak-STAT signaling. Finally, for linking the miRNA
and gene expression data to a clinical endpoint, a componentwise boosting
approach was employed that incorporates the graph information for fitting a
multivariable model and selects a small set of prognostic molecular entities.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Combining different kind of omics data is a challenging
task. With this graph based workflow we presented a method that combines the
different data sets in a graph model giving rise to e.g. pathway over
representation analysis and classification methods for predicting clinical
endpoints. Thereby, not only the features of both data sets but the coherences
between these features are used.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Stephan Gade (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13924 = 's.g&#97;d&#101;' + '&#64;';
 addy13924 = addy13924 + 'dkfz-h&#101;&#105;d&#101;lb&#101;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy13924 + suffix + '\'' + attribs + '>' );
 document.write( addy13924 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">German Cancer Research Center Heidelberg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) German Cancer Research Center Heidelberg (2)
University of Freiburg (3) University of Göttingen</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection50">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744919">E-49. An extended translational network involving
transcription factors and RNA binding proteins derived fro, the identification
of phylogenetic footprints in 5' and 3' untranslated regions</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dassi E<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Dassi E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Tebaldi T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tebaldi T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Zuccotti P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zuccotti P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Riva P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Riva P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Quattrone A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Quattrone A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Sequence motifs found in the untranslated regions (UTRs)
of mRNAs could be cis functional element involved in the post-transcriptional
regulation of gene expression, given that most RNA-binding proteins (RBPs) and
non coding RNAs (ncRNAs) bind to target sequences in either the mRNA 5' or 3'
UTRs to influence the fate of a transcript. If acting in evolutionarily
conserved control networks, these motifs would be subjected to negative
(purifying) selection, so that they could be identified by extracting
orthologous sequences in 5’ and 3’ UTRs for their high similarity among a
variety of species.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To map hyperconserved elements in the 5’ and 3’ UTR
we exploited the UCSC genome-wide alignment of 46 vertebrate species.
Hyperconservation measure was derived both from sequence conservation and
extent of the phylogenetic tree covered by the UTR alignment, each parameter
weighting equally. Hyperconserved elements (hyp-SUBs) were identified by
searching in all UTRs for fully conserved seeds that were then extended
upstream and downstream until the score did not fall under the 85% threshold.
Hyp-SUBs were then characterized by ontologies, motifs and regulatory factors
binding sites search.</p>

<h3>Results</h3>

<p class="abresults">Analysis of hyp-SUBs resulted in the observation of no
matching between them and long ncRNAs or microRNA seed targets. Hyp-5’SUBs were
enriched for HOX gene family members (suggesting the presence of an
unappreciated 5’UTR-targeted mechanism of HOX gene regulation) and for kinase
family genes; hyp-3’SUBs are enriched in genes coding for RBPs of the RRM type,
in line with experimental findings indicating a tendency for RBP expression to
be regulated post-transcriptionally. A benchmark experiment will be presented
to identify the mechanism exploiting these hyp-3'SUBs for the regulation of
RBPs.</p>

<h3>Discussion</h3>

<p class="abdiscussion">As a proof of validity of our approach, we identified a
subset of hyp-3'-SUBs belonging to the 3'UTR of histones mRNAs: a motif search
in these hyp-SUBs revealed that most of them contains the binding motif of the
histone stem loop binding protein (SLBP), which binds to the 3' histones UTR to
stabilize their mRNAs. This protein and its binding sites are very conserved
across evolution, indicating that our method can identify truly conserved
elements in UTRs. Also, a network of RBPs controlled by an upstream master RBPs
has been identified and partially experimentally confirmed in the same way.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Erik Dassi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96949 = 'd&#97;ss&#105;' + '&#64;';
 addy96949 = addy96949 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy96949 + suffix + '\'' + attribs + '>' );
 document.write( addy96949 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81149 = 'd&#97;ss&#105;' + '&#64;';
 addy81149 = addy81149 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy81149 + suffix + '\'' + attribs + '>' );
 document.write( addy81149 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93846 = 'd&#97;ss&#105;' + '&#64;';
 addy93846 = addy93846 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy93846 + suffix + '\'' + attribs + '>' );
 document.write( addy93846 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CIBIO - University of Trento</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Laboratory of Translational Genomics, Centre for
Integrative Biology, University of Trento (2) Department of Biology and
Genetics for Medical Sciences, University of Milan</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection51">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744920">E-50. Unvealing heterogeneity of stage II and stage
III colorectal cancer by defining molecular subtypes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Budinska E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Budinska E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,*), Popovici V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Popovici V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Delorenzi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Delorenzi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,4)</p>

<p class="abmotivation">Despite the continuous efforts for characterization of
the colorectal cancer, a large heterogeneity in survival experience is still
observed. Furthermore,the treatment efficacity varies widely even inside the
subpopulations defined by clinical and mutational status variables. Our
interest was to provide a complementary characterization of stage II and III
patients, built in a bottom-up fashion, that would allow further stratification
of the patient population, better outcome prediction and development of more
effective targeted treatments.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Stage II and III CRC samples from three public
datasets were considered for analysis. The unsupervised consensus clustering of
the patients led to the the definition of an optimal number of subgroups. Gene
modules defining the subgroups of patients were defined by hierarchical
clustering. The robustness of the patient subgroups was assessed across
datasets by cluster reproducibility measures on the gene modules. The subgroups
were characterized by the pathway analysis, the comparison with multiple normal
and cancer cell types and the assessment of clinical parameters.</p>

<h3>Results</h3>

<p class="abresults">Using techniques of meta-analysis we have defined four major
molecular subtypes of stage II and stage III colorectal cancer, reproducible
across the three public datasets. The pathway analysis indicated differences in
major cellular proceses among subgroups of patients, suggesting a novel
perspective on the classification of the colorectal cancer. The identified
subgroups remain, however, heterogeneous in terms of clinical parameters,
idicating a complementarity between the gene expression profiles and clinical
characteristics.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have identified four major molecular subtypes of colorectal
cancer, which are highly robust across studies. The most important outcome
consists in the observation that from a clinical perspective the subgroups
remain heterogeneous. This suggests that the newly proposed stratification
should be considered as a complementary characterization of colorectal cancer.
The limited information does not allow deeper analysis of treatment
implications on the subgroups. Nevertheless, we are confident that the proposed
classification represents true biological differences between patients.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Eva Budinska (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy30966 = '&#101;v&#97;.b&#117;d&#105;nsk&#97;' + '&#64;';
 addy30966 = addy30966 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy30966 + suffix + '\'' + attribs + '>' );
 document.write( addy30966 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86454 = '&#101;v&#97;.b&#117;d&#105;nsk&#97;' + '&#64;';
 addy86454 = addy86454 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy86454 + suffix + '\'' + attribs + '>' );
 document.write( addy86454 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90183 = '&#101;v&#97;.b&#117;d&#105;nsk&#97;' + '&#64;';
 addy90183 = addy90183 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy90183 + suffix + '\'' + attribs + '>' );
 document.write( addy90183 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Swiss Institute of Bioinformatics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Swiss Institute of Bioinformatics, Lausanne,
Switzerland 2 Institute of Biostatistics and Analyses, Brno, Czech Republic 3
Masaryk Memorial Center Institute, Brno, Czech Republic 4 Department of
Research, Lausanne University Hospital, Lausanne, Switzerland</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection52">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744921">E-51. A method for improved prediction of in vivo
transcription factor binding sites by using biophysical DNA features</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hooghe B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Hooghe B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3, +), Broos S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Broos S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,*, +) ,Van Roy F (2,3), De Bleser P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Bleser P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3)</p>

<p class="abmotivation">Transcription factor binding sites (TFBSs) are short
(6-20 bp) DNA sequences that can be regarded as base units of transcription
regulation. The binding interactions with their protein partners, transcription
factors (TFs), determine a big part of the spatiotemporal gene expression
patterns. In silico identification of these basic units has in principal a huge
potential to speed up research of gene regulation. The accurate in silico
identification of TFBSs that do really bind in vivo and not just in vitro has
however proven to be a difficult task.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Prediction by PWMs was used for the initial
alignment of true BSs and in the first step of TFBS identification. PWMs and
the search algorithm MATCH were from TRANSFAC version 2008.4. We linked all
biophysical values to tetra- or pentanucleotides to capture the
sequence-dependent biophysical variation caused by neighboring nucleotides.
Most values were collected from literature, others are own calculations. We
used 3DNA to calculate this for several models of both unbound DNA and either
protein-bound or nucleosome-bound DNA.</p>

<h3>Results</h3>

<p class="abresults">We then compared our approach to currently employed
sequence-based methods that use only nucleotide sequence information. To ensure
that methods and not training data sets were being compared, we trained these
models with sequences sampled from the same data set we used to train our
model. For each TF, we compared the best biophysical model to own constructed
PWMs and the TRANSFAC PWMs for that TF: our approach performed significantly
better for all 20 TFs.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The reported improvements in accuracy make a huge
difference upon identification when identifying TFBSs. We took one trained
biophysical model of Stat1 and used it to search binding sites (BSs) for this
TF in the sequences of the corresponding positive test set. We also used the
TRANSFAC PWM for Stat1 (V$STAT1_01) to identify BSs in these sequences. This
PWM is quite qualitative and the difference of accuracy between the PWM and the
best biophysical model is the smallest of all TFs shown. Yet the biophysical
model is clearly the best choice for reliable identification of in vivo TFBSs.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Stefan Broos (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29011 = 'st&#101;f&#97;n.br&#111;&#111;s' + '&#64;';
 addy29011 = addy29011 + 'dmbr' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy29011 + suffix + '\'' + attribs + '>' );
 document.write( addy29011 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">DMBR, VIB - Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Bioinformatics Core Facility, VIB, B-9052 Ghent,
Belgium 2. Department of Biomedical Molecular Biology, Ghent University, B-9052
Ghent, Belgium 3. Department for Molecular Biomedical Research, VIB, B-9052
Ghent, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was made possible by Flanders Institute for
Biotechnology (VIB) and funding from the Agency for Innovation through Science
and Technology in Flanders (IWT). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection53">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744922">E-52. Nonlinear dimension reduction and clustering
by minimum curvilinearity unfold neuropathic pain and tissue embryological
classes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cannistraci CV<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Cannistraci CV&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4,5,*), Ravasi T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ravasi T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,5), Montevecchi FM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Montevecchi FM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Ideker T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ideker T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5), Alessio M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alessio M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Nonlinear small datasets, which are characterized by low
numbers of samples and very high numbers of measures, occur frequently in
computational biology, and pose problems in their investigation. Unsupervised
Hybrid-two-phase procedures (H2P) - specifically dimension reduction (DR)
coupled with clustering - provide valuable assistance, not only for
unsupervised data classification, but also for visualization of the patterns
hidden in highdimensional feature space.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">‘Minimum Curvilinearity’ (MC) is a principle that -
for small datasets - suggests the approximation of curvilinear sample distances
in the feature space by pairwise distances over their minimum spanning tree
(MST), and thus avoids the introduction of any tuning parameter. MC is used to
design two novel forms of nonlinear machine learning (NML): Minimum Curvilinear
Embedding (MCE) for DR, and Minimum Curvilinear Affinity Propagation (MCAP) for
clustering.</p>

<h3>Results</h3>

<p class="abresults">Compared with several other unsupervised and supervised algorithms,
MCE and MCAP, whether individually or combined in H2P, overcome the limits of
classical approaches. High performance was attained in the visualization and
classification of: 1) pain patients (proteomic measurements) in peripheral
neuropathy; 2) human organ tissues (genomic transcription factor measurements)
on the basis of their embryological origin.</p>

<h3>Discussion</h3>

<p class="abdiscussion">MC provides a valuable framework to estimate nonlinear distances
in small datasets. Its extension to large datasets is prefigured for novel
NMLs. Classification of neuropathic pain by proteomic profiles offers new
insights for future molecular and systems biology characterization of pain.
Improvements in tissue embryological classification refine results obtained in an
earlier study, and suggest a possible reinterpretation of skin attribution as
mesodermal.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="https://sites.google.com/site/carlovittoriocannistraci/">https://sites.google.com/site/carlovittoriocannistraci/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Carlo Vittorio Cannistraci (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37156 = 'k&#97;l&#111;k&#97;g&#97;th&#111;s.&#97;g&#111;n' + '&#64;';
 addy37156 = addy37156 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy37156 + suffix + '\'' + attribs + '>' );
 document.write( addy37156 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Red Sea Integrative Systems Biology Lab,
Computational Bioscience Research Center, Division of Chemical &amp; Life
Sciences and Engineering, King Abdullah University for Science and Technology
(KAUST)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Red Sea Integrative Systems Biology Lab,
Computational Bioscience Research Center, Division of Chemical &amp; Life
Sciences and Engineering, King Abdullah University for Science and Technology
(KAUST), Jeddah, Kingdom of Saudi Arabia (2) Proteome Biochemistry, San
Raffaele Scientific Institute, via Olgettina 58, 20132 Milan, Italy (3)
Department of Mechanics, Politecnico di Torino, c/so Duca degli Abruzzi 24,
10129 Turin, Italy (4) CMP Group, Microsoft Research, Politecnico di Torino,
c/so Duca degli Abruzzi 24, 10129 Turin, Italy (5) Department of Bioengineering
and Department of Medicine, University of California, San Diego, 9500 Gilman
Drive, La Jolla, CA 92093 USA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We thank Ewa Aurelia Miendlarzewska for her generous
assistance and for language revision and Sven Bergmann for precious
suggestions. Funding: This work was supported by Fondazione CARIPLO (NOBEL GuARD
Project) and by MoH RF-FSR-2007-637144. CVC was supported by the Italian Interpolytechnic
School of Doctorate SIPD (http://sipd.polito.it/). TR and CVC was supported by
the MH062261grant from the US National Institute of Mental Health and the King
Abdullah University of Science and Technology. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection54">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744923">E-53. Seeded cis-regulatory module discovery using
rank based motif scoring</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Macintyre G<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Macintyre G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2*), Bailey J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bailey J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Kowalczyk A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kowalczyk A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,1), Haviv I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Haviv I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4,5)</p>

<p class="abmotivation">Transcription factors (TFs) act in concert by binding to
regions of DNA known as cis-regulatory modules (CRMs), usually in proximity to
the genes they regulate. These cooperating transcription factors play a major
role in the molecular mechanisms responsible for complex gene regulatory
programs observed in humans. Limited experimental mapping of CRMs in the human
genome means that in silico CRM discovery methods continue to play an important
role in understanding complex gene regulatory programs. We present a novel and
improved in silico cis-regulatory module detection algorithm called SCRM,</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach does not use species conservation
information to limit the CRM search space. This allows for the discovery of
species specific cis-regulatory elements. In addition, we have developed a
`seeded' search strategy for de novo discovery of interacting TFs, eliminating
the need for prior knowledge of interacting TFs and their position weight
matrices. The `seeded' search strategy is also used to determine CRM locations
using a rank based scoring system that favors clusters of interacting TF
binding sites around the `seed' TF binding site.</p>

<h3>Results</h3>

<p class="abresults">Our method is designed to uncover the regulatory mechanisms
involved in a particular molecular pathway for which there is a set of
putatively co-regulated genes and prior knowledge of a single TF (the `seed')
that may regulate those genes. For example, we provide the location and
composition of CRMs involved in the response of Hela cells to IFN-gamma for
which STAT1 plays a major role.We validate our predictions in vivo using a
mouse c-jun knockout model demonstrating a significant loss in expression of
those genes predicted to be regulated by c-jun.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The ability of transcription factors to bind specific
locations in the genome and control the transcription of a gene is typically
governed by four main factors: DNA site recognition, the gene's regulatory
domain, chromatin structure and co-factor interactions. We use knowledge of
each of these in the development of an in silico cis-regulatory module (CRM)
detection algorithm (called SCRM) for aid in interpretation of gene regulatory
networks in humans. Our biologically motived design and inclusion of a single
known regulating TF results in improved performance over existing CRM
predictors.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Geoff J Macintyre (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46535 = 'gm&#97;c&#105;' + '&#64;';
 addy46535 = addy46535 + 'css&#101;' + '&#46;' + '&#117;n&#105;m&#101;lb' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy46535 + suffix + '\'' + attribs + '>' );
 document.write( addy46535 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Department of Computer Science and Software
Engineering, University of Melbourne</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Department of Computer Science and Software
Engineering, The University of Melbourne, Victoria, Australia (2) NICTA,
Victoria Research Lab, The University of Melbourne, Victoria, Australia (3)
Bioinformatics and Systems Integration, The Blood and DNA Profiling Facility,
Baker IDI Heart and Diabetes Institute, 75 Commercial Rd, Prahran, Victoria,
Australia (4) Metastasis Research Lab, Peter MacCallum Cancer Centre, St
Andrews Place, East Melbourne, Victoria, Australia (5) Department of
Biochemistry and Molecular Biology. The University of Melbourne, Victoria,
Australia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">IH is supported by the following grants: [586649
NHMRC], [BC084667 US DoD BCRP], [566882 Cancer Australia], [BCTR0707358 Komen
for the Cure], and [509292 NBCF]. This work was partially supported by NICTA.
NICTA is funded by the Australian Government's Department of Communications,
Information Technology and the Arts, the Australian Research Council through
Backing Australia's Ability, and the ICT Centre of Excellence programs. GM is
supported by an Australian Postgraduate Award and a NICTA Research Project
Award. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection55">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744924">E-54. Proximity-based cis-regulatory module
detection using constraint programming for itemset mining</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Guns T<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Guns T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sun H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sun H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Nijssen S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nijssen S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sanchez-Rodriguez A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sanchez-Rodriguez A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Raedt L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Raedt L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Marchal K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marchal K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">cis-regulatory modules (CRMs) are combinations of
Transcription Factor Binding Sites involved in the regulation of genes. The
detection of CRMs is key in developing a better understanding of gene
regulation. Identifying significant combinations of binding sites is a
difficult computational problem. Existing techniques use heuristic methods or
strong restrictions to make the problem tractable, and are not very extendible.
We present an extendible technique for enumerating all potential CRMs using
only a biologically well-motivated restriction on the proximity of the binding
sites involved.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our method consists of 3 phases: First, the genomic
sequences under investigation are screened using an existing library of motif
models, namely, position weight matrices. This screening identifies
Transcription Factor Binding Site (TFBS) hits. Secondly, we use constraint
programming for itemset mining to efficiently enumerate all combinations of
TFBS hits that co-occur within a pre-defined distance, while avoiding
undesirable redundancies. This results in an exhaustive list of potential CRMs.
Lastly, the CRMs are ranked using statistical methods.</p>

<h3>Results</h3>

<p class="abresults">We use two previously published datasets which are known to
contain a true CRM composed of 3 transcription factors (TFs). One is a
synthetic dataset consisting of 22 genomic sequences, the other a ChIP-seq
dataset of 25 sequences. We test the scalability of our method by screening an
increasing number of TFs, sampled out of 516 TRANSFAC vertebrate TFs. Our
method finds all CRMs in up to 40 TFs and possibly even more. We are still in
the process of comparing the sensitivity and precision/recall of our method,
including the statistical ranking phase, with other tools.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To make enumerating all potential CRMs feasible, we take
special care in avoiding solutions that are equivalent to others. We believe a
large part of the computational challenge is related to the many equivalent
(and hence uninteresting) CRMs. After mining the potential CRMs, several
alternative statistical significance tests can be used for ranking in phase 3. As
our algorithm imposes no restriction on the size of the CRM, the found CRMs can
be quite large for small datasets. It is hence necessary to provide a
sufficient number of genes to avoid long CRMs containing false positive TFs.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://dtai.cs.kuleuven.be/CP4IM/" mce_href="http://dtai.cs.kuleuven.be/CP4IM/">http://dtai.cs.kuleuven.be/CP4IM/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tias Guns (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34099 = 't&#105;&#97;s.g&#117;ns' + '&#64;';
 addy34099 = addy34099 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy34099 + suffix + '\'' + attribs + '>' );
 document.write( addy34099 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22925 = 't&#105;&#97;s.g&#117;ns' + '&#64;';
 addy22925 = addy22925 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy22925 + suffix + '\'' + attribs + '>' );
 document.write( addy22925 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29498 = 't&#105;&#97;s.g&#117;ns' + '&#64;';
 addy29498 = addy29498 + 'cs' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy29498 + suffix + '\'' + attribs + '>' );
 document.write( addy29498 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">K.U. Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) DTAI, Department of Computer Science, K.U. Leuven,
Belgium. (2) CMPG/BIOI, Department of Microbial and Molecular systems.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">1) Postdoc and project from the Research
Foundation--Flanders, project "Principles of Patternset Mining" 2)
PhD and project from the Agency for Innovation by Science and Technology in
Flanders (IWT), project "A Bioinformatics Framework for Top-Down Systems
Biology" 3) Project IUAP P6/25 (BioMaGNet) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection56">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744925">E-55. High-throughput detection of epistasis in
studies of the genetics of complex traits</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gyenesei A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gyenesei A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Laiho A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laiho A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Semple C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Semple C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Haley C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Haley C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Wei W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wei W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Gene interactions are thought to be important in shaping
complex trait variation in agricultural, model organism and human disease
genetics. They have been poorly explored, however, because of the lack of high
throughput tools to analyze many different traits. The key challenge is how to
effectively reduce the search space without losing power of detection so that
one trait can be analysed in CPU hours.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The developed method is based on a bi-clustering
data mining algorithm named Mining Attribute Profiles that has been
successfully applied in gene expression data analyses.</p>

<h3>Results</h3>

<p class="abresults">We have developed a novel method, called Epicluster, that
can quickly select candidate SNPs with consistent genotype distribution
patterns differentiating phenotypes of interest and then perform comprehensive
epistasis analysis among the candidate SNPs.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In the future we plan to implement the developed
algorithm as a distributed computing tool to allow multiple traits to be
analyzed simultaneously.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Attila Gyenesei (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58139 = '&#97;tt&#105;l&#97;.gy&#101;n&#101;s&#101;&#105;' + '&#64;';
 addy58139 = addy58139 + 'btk' + '&#46;' + 'f&#105;';
 document.write( '<a ' + path + '\'' + prefix + addy58139 + suffix + '\'' + attribs + '>' );
 document.write( addy58139 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Turku Centre for Biotechnology, University of Turku
and Åbo Akademi University, Turku, Finland</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Turku Centre for Biotechnology, University of Turku
and Åbo Akademi University, Turku, Finland (2) MRC Human Genetics Unit,
Edinburgh, UK</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection57">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744926">E-56. Heavy metal resistance in Cupriavidus
metallidurans: a complex evolutionary and transcriptional process</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Monsieurs P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Monsieurs P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Moors H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moors H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Van Houdt R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Houdt R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Janssen P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Janssen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mergeay M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mergeay M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Leys N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Leys N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The soil bacterium Cupriavidus metallidurans CH34 shows
great promise in bioremediation schemes since it contains at least 25 loci
involved in heavy metal resistance, allowing it to thrive in metal-rich
environments. A first step in understanding the molecular mechanisms that
underly heavy metal resistance is to reconstruct the transcriptional regulatory
networks.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We designed a whole-genome microarray to
investigate the full stress response of C. metallidurans CH34 at the
transcriptomic level when it was challenged to a variety of heavy metals
including zinc, copper, cadmium, and lead. By performing a large number of
microarray experiments and developing a novel algorithm, additional heavy metal
response genes were detected. The redundancy of these genes, which often
clustered together within certain genome regions, was assessed by paralogy and
phylogenetic analysis.</p>

<h3>Results</h3>

<p class="abresults">Certain heavy metal response gene clusters showed similar
expression profiles when cells were exposed to varying combinations of heavy
metals, thus pointing to complex cross-talk at the transcriptional level
between the different heavy metal resistance mechanisms. The highly redundant
nature of these heavy metal response gene clusters combined with their
phylogenetic distribution within evolutionary related metal resistant bacteria
sheds light on the recent evolution of this soil bacterium towards a highly
metal-resistant species.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The intricate transcriptional behavior leading to metal
resistance in C. metallidurans CH34 represents an ideal test case to
reconstruct the regulatory networks underlying this response. The complexity of
this regulation is underpinned by the complete genome annotation of the C.
metallidurans CH34 which allowed us to identify several hundreds of regulatory
proteins, many of which are involved in metal detoxification and general metal
resistance.Using a combination of microarray results and DNA and protein
motifs, we integrated these data to get a better view on the complex response.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pieter Monsieurs (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95247 = 'pm&#111;ns&#105;&#101;&#117;' + '&#64;';
 addy95247 = addy95247 + 'sckc&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy95247 + suffix + '\'' + attribs + '>' );
 document.write( addy95247 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29405 = 'pm&#111;ns&#105;&#101;&#117;' + '&#64;';
 addy29405 = addy29405 + 'sckc&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy29405 + suffix + '\'' + attribs + '>' );
 document.write( addy29405 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61399 = 'pm&#111;ns&#105;&#101;&#117;' + '&#64;';
 addy61399 = addy61399 + 'sckc&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy61399 + suffix + '\'' + attribs + '>' );
 document.write( addy61399 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Belgian Nuclear Research Center (SCK•CEN)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Belgian Nuclear Research Center (SCK-CEN), Belgium.
</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection58">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744927">E-57. Identification of co-regulated mRNA/miRNA
pairs by the CoExpress software tool</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nazarov P V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nazarov P V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Khutko V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Khutko V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Schmitz S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schmitz S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Muller A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Muller A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kreis S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kreis S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vallar L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vallar L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Small noncoding miRNAs influence most fundamental
biological processes by ultimately altering the expression levels of proteins.
miRNAs tend to have long half lives and therefore represent promising
candidates to be used as disease markers and therapeutic targets. Insights into
miRNA functions and miRNA target genes can be obtained from simultaneous
analyses of full genome transcription profiles and miRNA levels derived from
the same sample types. Therefore, there is a need for effective and
user-friendly tools for fast analysis of co-regulation between miRNAs and
mRNAs.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The co-regulation study was performed using the
developed stand-alone software tool CoExpress, which allows for interactive
analysis of mRNA and miRNA co-expression by using microarray data. The software
performs microarray data pre-processing, building and visualization of
co-expression between mRNA and miRNA using correlation or mutual information
metrics. Taken together, the software facilitates the analysis and
visualization of miRNA-mRNA, mRNA-mRNA and miRNA-miRNA co-expression events,
some of which were confirmed by real-time quantitative PCR.</p>

<h3>Results</h3>

<p class="abresults">The proper functioning of the software was tested using
public mRNA and miRNA expression data from 14 various cell lines. Data from 42
Affymetrix HGU133plus2 arrays and 14 miRNA custom microarray experiments were
downloaded from public repositories, normalized and analyzed. We have detected
7423 co-expression events between 2533 mRNAs and 199 miRNAs with r2&gt;0.6. 22
of the most prominent mRNA-miRNA co-expression events were validated by qPCR,
which showed good concordance with the results of co-expression analyses.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The computational validation of the results was performed
by permutation of the samples in the mRNA data set. For the threshold of r2 =
0.6 the estimated p-value &lt; 1e–7. Then, the lists of co-expressed genes were
compared to lists of potential miRNA targets, predicted by combination of 6
commonly used algorithms EIMMo, DIANA, Pictar, TargetScan, PITA, and miRanda.
Despite the small concordance between predicted targets and co-expressed gene
lists, we were able to find negatively regulated genes with significant
p-values for most of the 199 considered miRNAs, which showed inverse
correlation</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.bioinformatics.lu/" mce_href="http://www.bioinformatics.lu/">http://www.bioinformatics.lu/</a><br mce_bogus="1"></p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Petr V. Nazarov (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38660 = 'p&#101;tr.n&#97;z&#97;r&#111;v' + '&#64;';
 addy38660 = addy38660 + 'crp-s&#97;nt&#101;' + '&#46;' + 'l&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy38660 + suffix + '\'' + attribs + '>' );
 document.write( addy38660 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Microarray Center, CRP-Santé</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Microarray Center, CRP-Santé, Luxembourg 2. Life
Sciences Research Unit, University of Luxembourg</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection59">

<div style="border-right: medium none; border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269744928">E-58. Identification of genes with preferential
expression in the egg cell</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Köszegi D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Köszegi D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Czhial A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Czhial A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kumlehn J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kumlehn J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Altschmied L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Altschmied L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Baumlein H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Baumlein H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In contrast to animals, the life cycle of higher plants
alternates between a gamete-producing (gametophyte) and a spore-producing
generation (sporophyte). The angiosperm female gametophyte consists of four
distinct cell types, including two gametes, the egg and the central cell, which
give rise to embryo proper and the nutritive endosperm, respectively. To gain
insights into the molecular basis of gamete differentiation and function, genes
with preferential expression in egg and central cell need to be isolated.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A combined subtractive hybridization and virtual
subtraction approach was used to isolate egg cell specific genes from a wheat
egg cell cDNA library. Using microarray hybridization and in silico
subtraction, egg cell expressed genes of Arabidopsis were isolated from a
transcription factor induced proliferating tissue, which exhibits an egg cell-like
transcriptome.</p>

<h3>Results</h3>

<p class="abresults">In total we have isolated seven and nine candidate genes
with preferential expression in the wheat and Arabidopsis egg cell,
respectively. Via single cell RT-PCR we confirm the preferential expression of
three wheat genes. Transgenic Arabidopsis plants transformed with
promoter:reporter constructs confirm egg cell specific promoter activity for
four out of nine genes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We demonstrate the suitability of the combined
subtractive approach for the isolation of gamete specific genes. The approach
is broadly applicable also for other species. Isolation and characterization of
genes with preferential expression in either gamete allow to unravel the
regulatory network which controls specification and differentiation of these
important cell types in plants.</p>

<div style="border-right: medium none; border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">David Köszegi (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy75162 = 'k&#111;sz&#101;g&#105;' + '&#64;';
 addy75162 = addy75162 + '&#105;pk-g&#97;t&#101;rsl&#101;b&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy75162 + suffix + '\'' + attribs + '>' );
 document.write( addy75162 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">IPK-Gatersleben</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 - IPK-Gatersleben</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">BMBF GABI-SEEDII and InnoRegio Program (No.: 03i0602) </p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="118-Gene-regulation-and-transcriptomics.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>