<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Text Mining, Ontologies and Databases</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Text Mining, Ontologies and Databases			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Text Mining, Ontologies and Databases</span><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant: normal ! important; text-transform: uppercase;"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Alfonso Valencia and Guy Cochrane<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Text Mining, Ontologies and
Databases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745307 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300300037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span><span style=""><o:p></o:p></span></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745308" mce_href="#_Toc269745308">F-1. Resource of Asian Primary
Immunodeficiency Diseases (RAPID) update: an open web-based integrated
molecular database on primary immunodeficiencies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745308 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">3</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300300038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745309" mce_href="#_Toc269745309">F-2. Measuring the functional
similarity of drugs<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745309 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300300039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745310" mce_href="#_Toc269745310">F-3. G-language bookmarklet: a
gateway for semantic web, linked data, and web services<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745310 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745311" mce_href="#_Toc269745311">F-4. A graphical view of the world
of protein-protein interaction databases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745311 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745312" mce_href="#_Toc269745312">F-5. StrainInfo: your
microbiological stepping stone<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745312 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745313" mce_href="#_Toc269745313">F-6. Semantic integration of
isolation habitat and location in StrainInfo<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745313 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">9</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745314" mce_href="#_Toc269745314">F-7. Towards an ontology for farm
animal reproduction traits<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745314 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745315" mce_href="#_Toc269745315">F-8. Human gene symbol validation at
the HGNC<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745315 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745316" mce_href="#_Toc269745316">F-9. miRSel: Automated extraction of
associations between microRNAs and genes from the biomedical literature<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745316 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745317" mce_href="#_Toc269745317">F-10. Metarel: a relation metagraph
for inferences in biomedical ontologies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745317 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745318" mce_href="#_Toc269745318">F-11. Predicting disease causing
genes using the information content surrounding the disease and gene in
literature<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745318 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745319" mce_href="#_Toc269745319">F-12. Revealing heterogeneities and
inconsistencies in protein functional annotations<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745319 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745320" mce_href="#_Toc269745320">F-13. A formal framework for
evaluating the significance of peptide-spectrum matches<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745320 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745321" mce_href="#_Toc269745321">F-14. Improving the execution of
bioinformatical workflows<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span></span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"> PAGEREF
_Toc269745321 \h </span><span style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745322" mce_href="#_Toc269745322">F-15. BridgeDb: standardized access
to gene, protein and metabolite identifier mapping services<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745322 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">18</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745323" mce_href="#_Toc269745323">F-16. HGVbaseG2P: an advanced
database for the integration and interrogation of genetic association datasets<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745323 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">19</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745324" mce_href="#_Toc269745324">F-17. Extracting phytogeography
information from species distribution data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745324 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">20</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745325" mce_href="#_Toc269745325">F-18. D2K - data to knowledge: data
integration for biological reasoning<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745325 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">21</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745326" mce_href="#_Toc269745326">F-19. BRENDA text-mining: new developments
for obtaining enzyme-related disease information from scientific literature<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745326 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">22</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745327" mce_href="#_Toc269745327">F-20. Beegle: a new search engine
for discovering novel genes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745327 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">23</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745328" mce_href="#_Toc269745328">F-21. An overview of the
pathogen-host interactions database, PHI-base<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745328 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">24</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745329" mce_href="#_Toc269745329">F-22. Finding disease related genes
by GeneRank algorithm using co-occurrence based network structures<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745329 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">25</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300320039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745330" mce_href="#_Toc269745330">F-23. Identifying variability of
human splicing forms in their interactions by using literature mining<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745330 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">26</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300330030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745331" mce_href="#_Toc269745331">F-24. The meaning behind the choice
of words: a case in cancer metastasis.<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745331 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">27</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300330031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745332" mce_href="#_Toc269745332">F-25. Linked open experimentation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745332 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">28</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300330032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745333" mce_href="#_Toc269745333">F-26. First CALBC challenge: first
results<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745333 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">29</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300330033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="119-text-mining-ontologies-and-databases.html#_Toc269745334" mce_href="#_Toc269745334">F-27. EuroPhenome: a repository for
high-throughput mouse phenotyping data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745334 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">30</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003300330034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745308">F-1. Resource of Asian Primary Immunodeficiency
Diseases (RAPID) update: an open web-based integrated molecular database on
primary immunodeficiencies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kumar Ramadoss S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kumar Ramadoss S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*), Keerthikumar S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Keerthikumar S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Raju R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Raju R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Kandasamy K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kandasamy K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,5), Balakrishnan L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Balakrishnan L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Dhevi Nagarajha Selvan L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dhevi Nagarajha Selvan L&quot;
<![endif]--><!--[if supportFields]><span style="mso-element:field-end" mce_style="mso-element:field-end"></span><![endif]--><span style="">&nbsp;</span>(4), Raja Sekhar N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Raja Sekhar N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Mohan S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mohan S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Bhattacharjee M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bhattacharjee M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Hijikata A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hijikata A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Imai K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Imai K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Kanegane H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kanegane H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7), Miyawaki T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Miyawaki T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7), Nonoyama S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nonoyama S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Pa</p>

<p class="abmotivation">Primary immunodeficiency diseases (PIDs) are a
genetically heterogeneous group of disorders that affect distinct components of
the innate and adaptive immune system. Despite rapid developments in the
science of PID, early diagnosis and effective treatment are posed with greater
challenges to all physicians. In this milieu, a development of freely
accessible dynamic web-based integrated tool on PID has become a paramount need
for different target groups. Thus, we have developed a web-based compendium of
molecular alterations in PID, Resource of Asian Primary Immunodeficiency
diseases (RAPID).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">RAPID, is an object-oriented database, used
three-tier architecture namely Zope, Python and MySQL. It hosts information on
sequence variations of all reported PID genes along with expression profiling,
mouse studies, interaction network and available DNA sequencing protocols. All
mutation data have been linked with a graphical user interface (GUI) enabled
tool, named Mutation@A Glance, to visualize and evaluate reported mutations.
Using RAPID data, we have used Support Vector Machine (SVM) based parameter
classification to predict candidate PID genes by scanning genes in the whole
human genome</p>

<h3>Results</h3>

<p class="abresults">At present, RAPID comprises total of 195 genes that are involved
in PID, out of which 183 PID genes are reported with over 4300 unique mutation
data referred from about 1500 public citations. For prediction of novel PID
gene candidates, we trained SVM with 69 features for both positive and negative
gene datasets and obtained 1,442 candidate PID genes from the human genome -
these can be prioritized further and experimentally validated for its role in
PID pathogenesis. Apart from these, RAPID has been updated with standardized
DNA sequencing protocols for selected PID genes and PID expert page.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our next focus will be on the analysis of annotated PID
data such as correlation of mutation types and effects, influence of mutation
occurrence in its functional domains, identification of hotspot mutations,
disease-causing mutations distribution and frequency in various ethnic groups
and then interpretation of all these analyzed information to a greater
knowledgebase of PID. Also, we are in the process of introducing RAPID gateway
to closed network PID clinical consultation forum for registered users, using
in-built tool developed and maintained by RIKEN Scientists Networking System
(SciNeS)</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://rapid.rcai.riken.jp/" mce_href="http://rapid.rcai.riken.jp/"><span style="">http://rapid.rcai.riken.jp/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Suresh Kumar Ramadoss (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27947 = 's&#117;r&#101;sh' + '&#64;';
 addy27947 = addy27947 + 'rc&#97;&#105;' + '&#46;' + 'r&#105;k&#101;n' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy27947 + suffix + '\'' + attribs + '>' );
 document.write( addy27947 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17965 = 's&#117;r&#101;sh' + '&#64;';
 addy17965 = addy17965 + 'rc&#97;&#105;' + '&#46;' + 'r&#105;k&#101;n' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy17965 + suffix + '\'' + attribs + '>' );
 document.write( addy17965 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy25814 = 's&#117;r&#101;sh' + '&#64;';
 addy25814 = addy25814 + 'rc&#97;&#105;' + '&#46;' + 'r&#105;k&#101;n' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy25814 + suffix + '\'' + attribs + '>' );
 document.write( addy25814 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">RCAI, RIKEN Yokohama Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Research Unit for Immunoinformatics, RIKEN Research
Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama,
Kanagawa 230-0045, Japan. 2. Immunogenomics Research Group, RIKEN Research
Center for Allergy and Immunology, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama,
Kanagawa 230-0045, Japan. 3. Department of Human Genome Research, Kazusa DNA
Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan. 4.
Institute of Bioinformatics, International Tech Park, Bangalore 560 066, India.
5. McKusick-Nathans Institute of Genetic Medicine and the Departments of
Pathology and Oncology, Johns Hopkins University, Baltimore, Maryland 21205,
USA. 6. Department of Medical Informatics &amp; Department of Pediatrics,
National Defense Medical College, Saitama 359-8513, Japan. 7. Department of
Pediatrics, Graduate School of Medicine and Pharmaceutical Sciences, University
of Toyama, Toyama 930-0194, Japan</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by a Grant-in-aid for Special
Coordination Funds for Promoting Science and Technology, from the Ministry of
Education, Culture, Sports, Science and Technology of Japan through an
International collaboration with Institute of Bioinformatics (IOB), India. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745309">F-2. Measuring the functional similarity of drugs</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Götz S<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Götz S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Behrens S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Behrens S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Tarazona S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tarazona S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Marba M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marba M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Dopazo J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dopazo J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,4), Conesa A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Conesa A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">As functional profiling/enrichment methods become the
standard procedure for interpreting functional genomic studies it becomes more
and more necessary to develop bioinformatic methodologies that establish
connections within the functional profiling world. We presents a novel strategy
to assessing the functional similarity of profiles based on the GO topology. We
applied this strategy to an extensive dataset of drug profiles derived from
genome-wide expression data. The dominant biological functions of drugs were
compared to identify and detect similar functional characteristics among them.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Drug expression data from the Connectivity Map
project was analysed by 2 GSEA methods. The resulting functional profile of
1500 drugs were compared against each other by a 3-step approach: Pairwise
“node-based” semantic similarity for each pair of GO terms, the “best-match
average” to combine the similarity of profiles and a random based normalization
to account for the profile's size bias. The resulting matrix of drug
similarities was evaluated by comparing it against the ATC drug classification
scheme to show that similar compounds obtained high semantic similarity scores.</p>

<h3>Results</h3>

<p class="abresults">The strategy, based on an edge-based metric to considers the
topology of the GO hierarchy was used to measure the functional distances of
drug based on a large set of transcriptomics data. A classification scheme for
these compounds made it possible to contrast our results with an external criterion
of similarity. The ATC system codifies drugs at different levels of specificity
and we confirmed that the proposed method captures this specificity and
observed that the mean similarity value increased when compared drugs belong to
more specific ATC level.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The proposed strategy to assess the semantic similarity
of functional profiles combines a edge-based similarity metric with random
null-model comparison to assess absolute distances of functional profiles. The
analysis includes the evaluation and preprocessing of a large gene-expression
drug dataset. To evaluate the functional similarity, drugs had been grouped by
the hierarchical ATC classification scheme. The strategy proposed for measuring
the semantic similarity of functional profiles has proved to be a valid
methodology to study functional proximity between biological samples.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Stefan Götz (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21433 = 'sg&#111;&#101;tz' + '&#64;';
 addy21433 = addy21433 + 'c&#105;pf' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy21433 + suffix + '\'' + attribs + '>' );
 document.write( addy21433 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91072 = 'sg&#111;&#101;tz' + '&#64;';
 addy91072 = addy91072 + 'c&#105;pf' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy91072 + suffix + '\'' + attribs + '>' );
 document.write( addy91072 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49291 = 'sg&#111;&#101;tz' + '&#64;';
 addy49291 = addy49291 + 'c&#105;pf' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy49291 + suffix + '\'' + attribs + '>' );
 document.write( addy49291 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Centro de Investigacion Principe Felipe (CIPF)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics and Genomics Department, Centro de
Investigación Príncipe Felipe (CIPF), Valencia, Spain (2) CIBER de Enfermedades
Raras (CIBERER), Valencia, Spain (3) Department of Genome Oriented
Bioinformatics, Technische Universität München, Wissenschaftszentrum
Weihenstephan, Freising, Germany (4) Functional Genomics Node (National
Institute for Bioinformatics, INB), Centro de Investigaciones Príncipe Felipe
(CIPF), Valencia, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Ministerio de Ciencia e Innovación, Gobierno de España</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745310">F-3. G-language bookmarklet: a gateway for semantic
web, linked data, and web services</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Arakawa K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Arakawa K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1, *), Kido N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kido N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Oshita K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Oshita K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Tomita M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tomita M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">In order to efficiently navigate and query through the
huge masses of biological information, concepts of Linked Data and Semantic Web
are gaining momentum as promising means for data integration. In light of the
advent of these new data representation models, we here present a bookmarklet
that provides an intuitive user interface for accessing the Linked Data and for
querying the resources of Semantic Web, through a graphical ring-shaped menu on
any webpage that the user is browsing. In this way, users can take advantage of
the Semantic Web and Linked Data coherently with existing data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">G-language Bookmarklet is implemented as a
bookmarklet to seamlessly work with regular web browsing, runs on any modern
browsers, and can be invoked from any websites, without the need for
installation of software or any specialized browser plugins. It is implemented
with Javascript and HTML5 with Asynchronous Javascript and XML (AJAX)
programing paradigm, and the source code is available with MIT License.
Supported data sources are Google, PubMed, and Wikipedia for standard web, NCBI
Entrez, EBI EB-eye, KEGG LinkDB for Linked Data, and Bio2RDF for Semantic Web.</p>

<h3>Results</h3>

<p class="abresults">By selecting keywords of interest within any webpage and by
opening the G-language Bookmarklet, an array of icons in the shape of a ring
appears with animation on top of the webpage that the user is currently
browsing. Here the users can select the database to search with the selected
keyword, such as NCBI Entrez, Pubmed, KEGG, and Bio2RDF. Results of the queries
are readily shown as another ring of icons representing the top hits of the
query. Users can also access web services such as BLAST and G-language. The
bookmarklet is freely available at: http://www.g-language.org/wiki/bookmarklet.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Semantic Web and Linked Data are generally accepted as
the promising means for data integration in biology, lead by initiatives such
as Concept Web Alliance, Banff Manifesto, SADI, and DBCLS BioHackathon 2010.
G-language Bookmarklet aims to provide a gateway to these new technologies for
the end-users, by providing an intuitive interface with icons and animations
that works on any web pages without the need for installation. We therefore
provide a unique and consistent intuitive user interface for web search
engines, linked data, semantic web, and web services.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.g-language.org/wiki/bookmarklet" mce_href="http://www.g-language.org/wiki/bookmarklet"><span style="">http://www.g-language.org/wiki/bookmarklet</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Kazuharu Arakawa (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81735 = 'g&#97;&#111;&#117;' + '&#64;';
 addy81735 = addy81735 + 'sfc' + '&#46;' + 'k&#101;&#105;&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy81735 + suffix + '\'' + attribs + '>' );
 document.write( addy81735 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69966 = 'g&#97;&#111;&#117;' + '&#64;';
 addy69966 = addy69966 + 'sfc' + '&#46;' + 'k&#101;&#105;&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy69966 + suffix + '\'' + attribs + '>' );
 document.write( addy69966 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63299 = 'g&#97;&#111;&#117;' + '&#64;';
 addy63299 = addy63299 + 'sfc' + '&#46;' + 'k&#101;&#105;&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy63299 + suffix + '\'' + attribs + '>' );
 document.write( addy63299 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute for Advanced Biosciences, Keio University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Institute for Advanced Biosciences, Keio University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research was supported by the Grant-in-Aid for
Young Scientists No.222681029 from the Japan Society for the Promotion of
Science (JSPS), and by funds from the Yamagata Prefectural Government and
Tsuruoka City. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745311">F-4. A graphical view of the world of
protein-protein interaction databases</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Klingström T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Klingström T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Plewczynski D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Plewczynski D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">The amount of information regarding protein-protein
interactions (PPI) at a proteomic scale is constantly increasing. This is
paralleled with an increase of databases making information available.
Consequently there are diverse ways of delivering information about not only
PPIs but also regarding the databases themselves. This creates a time consuming
obstacle for many researchers working in the field.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The initial selection of investigated databases was
made from the index of databases kept at Pathguide (http://www.pathguide.org/)
and based on their popularity ranking. Most databases have at least one article
describing them in the annual database issue of Nucleic Acids Research (NAR)
journal. Articles published in this annual issue are obligated to contain a
section comparing it to other similar databases. Relevant databases mentioned
in those articles have also been added to complement the initial selection from
Path-guide.</p>

<h3>Results</h3>

<p class="abresults">Our survey provides a valuable tool for researchers to
reduce the time necessary to gain a broad overview of PPI-databases. The
graphical representation of data exchange also provides a novel way for
researchers to quickly assess the origins of their data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The graphical representantion of data exchange between
databases will be made available in cooperation with
http://www.pathguide.org/interactions.php in a new Cytoscape web
implementation.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.pathguide.org/interactions.php" mce_href="http://www.pathguide.org/interactions.php"><span style="">http://www.pathguide.org/interactions.php</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Dariusz Plewczynski (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26289 = 'd&#97;rm&#97;n' + '&#64;';
 addy26289 = addy26289 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy26289 + suffix + '\'' + attribs + '>' );
 document.write( addy26289 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34199 = 'd&#97;rm&#97;n' + '&#64;';
 addy34199 = addy34199 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy34199 + suffix + '\'' + attribs + '>' );
 document.write( addy34199 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46432 = 'd&#97;rm&#97;n' + '&#64;';
 addy46432 = addy46432 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy46432 + suffix + '\'' + attribs + '>' );
 document.write( addy46432 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Interdyscyplinary Centre for Mathematical and
Computational Modelling, University of Warsaw</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Interdisciplinary Centre for Mathematical and
Computational Modelling, University of Warsaw, Pawinskiego 5a Street, 02-106
Warsaw, Poland, E-mail: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84559 = 'd&#97;rm&#97;n' + '&#64;';
 addy84559 = addy84559 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy84559 + suffix + '\'' + attribs + '>' );
 document.write( addy84559 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> (2) Biology Education Centre, Uppsala
University, Norbyvägen 14, 752 36 Uppsala, Sweden.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We would like to thank all the researchers who have
taken the time to personally answer our questions. We would also like to thank
Hagen Blankenburg for taking the time to extensively explain DAS to us, and
Jonas Klingström for his linguistic advice. Funding: This work was supported by
Linnéska stipendiestiftelsen, EC OxyGreen (KBBE-2007-212281) 6FP projects as
well as the Polish Ministry of Education and Science (N301 159735,
PBZ-MNiI-2/1/2005 and others). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745312">F-5. StrainInfo: your microbiological stepping
stone</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Verslyppe B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Verslyppe B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), De Smet W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Smet W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,*), Gillis W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gillis W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Vos P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Vos P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), De Baets B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Baets B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Dawyndt P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dawyndt P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Microbiology traditionally builds upon actual material
which can be deposited in Biological Resource Centers. BRCs assign so-called
strain numbers to identify the accessioned material and are responsible for
long-term preservation and world-wide distribution of the material. As BRCs
mutually exchange these cultured microorganisms, a plethora of equivalent
strain numbers has come into use. This implies that to find all information on
a given strain, all equivalent strain numbers should be used when searching.
Moreover, the information is not suited for electronical processing.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">StrainInfo is a virtual, global catalog integrating
BRC catalogs. It is envisioned as a platform giving uniform access to
microbiological specimen information. It integrates the specimen information
with genomic information, taxonomic resources and literature databases. It
builds upon the proposed Microbiological Common Language (MCL), aimed at
standardizing the electronic exchange of meta-information about microorganisms.
It provides mechanisms to the community to curate the vast amount of
microbiological information readily available.</p>

<h3>Results</h3>

<p class="abresults">StrainInfo is accessible through its web application which
is based on the concepts of "passport pages" and
"browsers". All equivalent numbers, together with additional
information, are listed on so-called "strain passports". The
corresponding "strain browser" enables direct access to the
underlying BRC catalogs and features an overlay panel allowing easy browsing to
equivalent catalog entries. Taxon, sequence and literature passports and
browsers are also available. StrainInfo also offers web services for advanced
users that want to automate queries or build powerful workflows.</p>

<h3>Discussion</h3>

<p class="abdiscussion">StrainInfo is an open, publicly available platform which
is accessed online through a web application and web services. Its integrated
view allows to handle microbiological information on a new scale and prepares
microbiology for an era where new insights can be obtained based on existing
information. StrainInfo provides true globally unique identifiers for
microbiological material, forming the basis for electronical processing of
microbiological information. Integration results are made available in
electronically processable MCL files, allowing consumption by downstream
applications.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.straininfo.net" mce_href="http://www.straininfo.net"><span style="" mce_style="mso-no-proof:
yes">http://www.straininfo.net</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Wim De Smet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44793 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy44793 = addy44793 + 'Ug&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy44793 + suffix + '\'' + attribs + '>' );
 document.write( addy44793 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55865 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy55865 = addy55865 + 'Ug&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy55865 + suffix + '\'' + attribs + '>' );
 document.write( addy55865 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96743 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy96743 = addy96743 + 'Ug&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy96743 + suffix + '\'' + attribs + '>' );
 document.write( addy96743 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Applied Mathematics and Computer
Science, Ghent University, Krijgslaan 281, 9000 Ghent, Belgium. 2. Laboratory
of Microbiology, Ghent University, K.L. Ledeganckstraat 35, 9000 Ghent,
Belgium. 3. BCCM/LMG Bacteria Collection, Ghent University, K.L.
Ledeganckstraat 35, 9000 Ghent, Belgium. 4. Department of Applied Mathematics,
Biometrics and Process Control, Ghent University, Coupure links 653, 9000
Ghent, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The development the StrainInfo platform is funded by
the Belgian Science Policy (contract IUAP VI/06) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745313">F-6. Semantic integration of isolation habitat and
location in StrainInfo</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Verslyppe B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Verslyppe B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), De Smet W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Smet W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,*), De Vos P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Vos P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), De Baets B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Baets B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Dawyndt P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dawyndt P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The adoption of Microbiological Common Language (MCL) XML
synchronization quickly increased the volume of semantic data in StrainInfo.
However, the effective data values of the different semantic fields still are
raw textual entries and therefore are of varying detail, can have different
forms or languages, and sometimes contain inconsistencies. By consequence, in
order to generate a strain level consensus value for each field, a specialized
semantic integration of this data needs to be developed. As a case study, the
focus was put on the isolation habitat and location information fields.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To integrate geographical information, NER is
performed to annotate all geographic names with features from the GeoNames
ontology. This yields a multitude of annotations, each annotation matching a
name with one or more geographical features. Multiple heuristics are used to
cope with ambiguous and broad geographical names to select the most specific
geographical feature. This is the integration result; multiple remaining
annotations or features being too distant indicate inconsistent data. The
habitat fields can be integrated using a similar algorithm (based on other
ontologies).</p>

<h3>Results</h3>

<p class="abresults">The consensus values are made available to end-users by
listing them on the corresponding strain passports. Geographical names can be
visualized on a map. Advanced search functionality is made available to allow
users to perform true semantic search (e.g. find all strains isolated from
dairy products in Europe). The integration results are also available from the
MCL XML exports.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In order to increase the information content of
StrainInfo, it is necessary to add fine-grained semantic information. This
information enters StrainInfo on the culture level (synchronization with BRC
catalogs), but must be integrated on the strain level (i.e. the set of
equivalent cultures) in order to be presented on strain passports. These
results allow to use ontological knowledge when searching and therefore
increase precision and recall compared to full text search. In addition, the
ontologies enrich the data by providing or linking additional information (e.g.
GPS coordinates).</p>

<h3>URL</h3>

<p class="abwebsite"><span style="" mce_style="mso-no-proof:yes"><a href="http://www.straininfo.net" mce_href="http://www.straininfo.net">http://www.straininfo.net</a></span></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Wim De Smet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73312 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy73312 = addy73312 + 'UG&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy73312 + suffix + '\'' + attribs + '>' );
 document.write( addy73312 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44895 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy44895 = addy44895 + 'UG&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy44895 + suffix + '\'' + attribs + '>' );
 document.write( addy44895 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6855 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy6855 = addy6855 + 'UG&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy6855 + suffix + '\'' + attribs + '>' );
 document.write( addy6855 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Applied Mathematics and Computer
Science, Ghent University, Krijgslaan 281, 9000 Ghent, Belgium. 2. Laboratory
of Microbiology, Ghent University, K.L. Ledeganckstraat 35, 9000 Ghent,
Belgium. 3. BCCM/LMG Bacteria Collection, Ghent University, K.L.
Ledeganckstraat 35, 9000 Ghent, Belgium. 4. Department of Applied Mathematics,
Biometrics and Process Control, Ghent University, Coupure links 653, 9000
Ghent, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The development the StrainInfo platform is funded by
the Belgian Science Policy (contract IUAP VI/06) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745314">F-7. Towards an ontology for farm animal
reproduction traits</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hulsegge B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Hulsegge B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Smits MA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Smits MA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), te Pas MFW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;te Pas MFW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Woelders H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Woelders H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Reproduction is an important characteristic of farm
animals. Increasing amounts of information on reproduction traits in farm
animals are becoming available in various formats and media, both
electronically and otherwise. Accessibility and integration of this scattered
information is an essential step not only in the understanding of the biology
of reproduction traits but also for future optimizations. A prerequisite for
this is to set-up an ontology to make relevant reproduction terms explicit and
to define their relationships.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For the development of the Reproduction ontology we
choose to start with the subdomain female fertility traits in cattle. Terms for
this domain were collected from literature, from thesauruses, from existing
ontologies and from experts in this field. The ontology is organized into a
hierarchical structure. The relationships between terms is described as a
class-subclass (“is_a”). relationship. Each term in the ontology has a name, a
unique identifier (ID), a definition and may have synonyms. The ontology is
constructed using Protégé and represented in the Web Ontology Language (OWL).</p>

<h3>Results</h3>

<p class="abresults">A framework for the REProduction Ontology (REPO) was
developed. A first version was provided which includes the major terms for the
domain female fertility traits in cattle, together with their synonyms and, in
most cases, text definitions. The top level terms of REPO include the following
terms: reproductive behaviour, reproductive failure and reproductive
performance. Currently REPO contains 94 terms.</p>

<h3>Discussion</h3>

<p class="abdiscussion">REPO can be used in databases, for literature analysis,
for annotation (e.g. probes on microarrays) and linking high level traits to
molecular information. The current version of REPO is not complete, its
practical value will depend on future additions and corrections. An objective
for future elaboration of REPO could be to include terms that relate to male
fertility and to include other farm animals.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ina Hulsegge (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36346 = '&#105;n&#97;.h&#117;ls&#101;gg&#101;' + '&#64;';
 addy36346 = addy36346 + 'w&#117;r' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy36346 + suffix + '\'' + attribs + '>' );
 document.write( addy36346 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60899 = '&#105;n&#97;.h&#117;ls&#101;gg&#101;' + '&#64;';
 addy60899 = addy60899 + 'w&#117;r' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy60899 + suffix + '\'' + attribs + '>' );
 document.write( addy60899 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46009 = '&#105;n&#97;.h&#117;ls&#101;gg&#101;' + '&#64;';
 addy46009 = addy46009 + 'w&#117;r' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy46009 + suffix + '\'' + attribs + '>' );
 document.write( addy46009 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Animal Breeding and Genomics Centre, Wageningen UR
Livestock Research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Animal Breeding and Genomics Centre, Wageningen UR
Livestock Research</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the European Commission,
within the Sixth Framework Programme, contract No. FOOD-CT-2006-016250 (SABRE)
and from the Knowledge Base grant no KB-01-006-065 of the Dutch Ministry of
Agriculture, Nature and Food security. The text represent the authors views and
does not necessary represent a position of the Commission or Ministry who will
not be liable for the use made of such information. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745315">F-8. Human gene symbol validation at the HGNC</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lush MJ<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Lush MJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>*, Seal RL<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Seal RL&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Gordon SM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gordon SM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Wright MW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wright MW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Bruford EB<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bruford EB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The aim of the HUGO Gene Nomenclature Committee (HGNC
www.genenames.org) is to provide and promote unique and meaningful gene symbols
and names for human genes. Standardisation of gene symbols is important as it
allows researchers to refer to the same gene without ambiguity and facilitates
electronic data retrieval. With the advent of microarraying and high throughput
technologies, we have found there is an increasing demand to resolve large
lists of gene symbols to their corresponding HGNC ID allowing gene data to be
linked to many disparate sources.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The HGNC list search
&lt;http://www.genenames.org/list&gt; is coded in perl 5.8, runs on a CentOS
5.1 server utilizing MySQL 5.0.41 and Apache version 2.0.</p>

<h3>Results</h3>

<p class="abresults">Unfortunately this process is complicated by the use of
symbol aliases (i.e. non HGNC-approved gene symbols) as a single alias may have
been used to refer to multiple genes and may also clash with existing approved
nomenclature. Additionally, approved nomenclature is not completely static; for
example, gene records can be merged or split as new evidence becomes available.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In response to demand we have developed a new "List
Search" facility (www.genenames.org/list) which allows researchers to
upload large lists of symbols for checking against the HGNC database. The
results confirm which symbols are approved, and if the symbol is a known alias
of a gene the corresponding approved symbol is displayed. Each approved symbol
then leads the user to the gene entry in our database, which contains multiple
links to other resources.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.genenames.org/list" mce_href="http://www.genenames.org/list"><span style="">http://www.genenames.org/list</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michael J. Lush (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13879 = 'mjl&#117;sh' + '&#64;';
 addy13879 = addy13879 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy13879 + suffix + '\'' + attribs + '>' );
 document.write( addy13879 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35822 = 'mjl&#117;sh' + '&#64;';
 addy35822 = addy35822 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy35822 + suffix + '\'' + attribs + '>' );
 document.write( addy35822 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82142 = 'mjl&#117;sh' + '&#64;';
 addy82142 = addy82142 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy82142 + suffix + '\'' + attribs + '>' );
 document.write( addy82142 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">HUGO Gene Nomenclature Committee</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">HUGO Gene Nomenclature Committee, EMBL Outstation -
Hinxton, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton,
Cambridge, CB10 1SD, United Kingdom</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The work of the HGNC is supported by National Human
Genome Research Institute (NHGRI) grant P41 HG03345 and Wellcome Trust grant
081979/Z/07/Z. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745316">F-9. miRSel: Automated extraction of associations
between microRNAs and genes from the biomedical literature</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Naeem H<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Naeem H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Küffner R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Küffner R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Csaba G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Csaba G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Zimmer R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zimmer R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">microRNAs (miRNAs) have been discovered as important
regulators of gene expression. To identify the target genes of miRNAs, several
databases and prediction algorithms have been developed. Only few
experimentally confirmed miRNA targets are available in databases. Many of the miRNAs
targets stored in databases were derived from large scale experiments that are
considered not very reliable.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The construction of a database of miRNA-gene
co-occurrences via named entity recognition requires the compilation of miRNA
and gene name dictionaries. The dictionaries for human, mouse and rat are
compiled from several databases: HGNC, MGD, Entrez Gene, miRBase. In case of
miRNAs we found that many miRNA names in the literature are not yet contained
in databases. We detect miRNA names using a regular expression. Gene names are
detected by string matching using the syngrep program. syngrep uses the
Aho-Corasick algorithm for matching and context resolution techniques to
resolve ambiguities.</p>

<h3>Results</h3>

<p class="abresults">The miRNA-gene association database miRSel combines text
mining results with existing databases and computational predictions. Text
mining enables the reliable extraction of miRNA, gene and protein occurrences
as well as their relationships from texts. Thereby, we increased the number of
human, mouse and rat miRNA-gene associations by at least four-fold as compared
to e.g. TarBase. miRSel is updated daily and can be queried using a web-based
interface ( http://services.bio.ifi.lmu.de/mirsel) via miRNA identifiers, gene
and protein names, PubMed queries as well as gene ontology (GO) terms.</p>

<h3>Discussion</h3>

<p class="abdiscussion">miRNA related research depends on knowledge of miRNA
target genes. In contrast to manual curation, we proposed a simple, automated
approach for biological name identification that collects many potential
targets for miRNAs not contained in current databases. Text mining of miRNA,
gene or protein names results in good recall and precision for miRNA-gene
associations detected in single sentences. We thereby extracted many pairs from
human (2875 pairs), mouse (1466 pairs) and rat (595 pairs) abstracts. To keep
the miRSel database up-to-date, newly available PubMed abstracts are included
daily.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://services.bio.ifi.lmu.de/mirsel/" mce_href="http://services.bio.ifi.lmu.de/mirsel/"><span style="">http://services.bio.ifi.lmu.de/mirsel/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Haroon Naeem (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43740 = 'h&#97;r&#111;&#111;n.n&#97;&#101;&#101;m' + '&#64;';
 addy43740 = addy43740 + 'c&#97;mp&#117;s' + '&#46;' + 'lm&#117;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy43740 + suffix + '\'' + attribs + '>' );
 document.write( addy43740 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12919 = 'h&#97;r&#111;&#111;n.n&#97;&#101;&#101;m' + '&#64;';
 addy12919 = addy12919 + 'c&#97;mp&#117;s' + '&#46;' + 'lm&#117;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy12919 + suffix + '\'' + attribs + '>' );
 document.write( addy12919 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36739 = 'h&#97;r&#111;&#111;n.n&#97;&#101;&#101;m' + '&#64;';
 addy36739 = addy36739 + 'c&#97;mp&#117;s' + '&#46;' + 'lm&#117;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy36739 + suffix + '\'' + attribs + '>' );
 document.write( addy36739 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institut für Informatik,
Ludwig-Maximilians-Universität München, Amalienstr. 17 80333 München, Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Lehr- und Forschungseinheit Bioinformatik Institut für
Informatik, Ludwig-Maximilians-Universität München, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work has been supported in part by the German
Federal Ministry of Education and Research (via DLR, FKZ 01GS0801). Haroon
Naeem is supported by the Deutscher Akademischer Austausch Dienst (Referat 442,
Code A/07/96865). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745317">F-10. Metarel: a relation metagraph for inferences
in biomedical ontologies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Blondé W<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Blondé W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Antezana E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Antezana E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Venkatesan A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Venkatesan A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Baets B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Baets B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kuiper M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kuiper M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Mironov V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mironov V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Ontologies start to become ubiquitous in the life
sciences for managing and integrating knowledge and data. They provide a
reference framework where scientific terms are defined and interconnected with
sound methods. Biologists and bioinformaticians need this framework in order to
query the available knowledge and to fill the gaps where necessary. Gene
Ontology Annotations (GOA), containing millions of annotations about proteins,
makes use of this framework. Scalable inferencing tools are necessary for
facilitating queries on the integrated Knowledge Base (KB).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The Semantic Web provides all the tools that are
required for integration, querying and reasoning. We translated all the OBO
ontologies to RDF and stored them in the BioGateway KB, powered by Virtuoso. We
carefully separated all the information about the relations between terms,
because these contain the rules about knowledge that is not explicitly present
in the ontologies. We condensed all these rules into the relation metagraph of
Metarel, which was created for task. It contains rules for reflexivity,
transitivity, super-relation types, composite relations and more.</p>

<h3>Results</h3>

<p class="abresults">By running SPARUL update queries over all the ontologies and
the metagraph, we could infer hundreds of millions of new knowledge statements.
All these statements together contain all the implied knowledge that follows
from the knowledge that was stated explicitly. We call this a total relational
closure. Any queries on the total closures (TC) return all the correct answers,
which was not the case for queries on the original ontologies. We could create
many new queries in BioGateway that are very useful from the perspective of
biological scientists.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This work proves that a sound ontological framework is
necessary and desirable for Knowledge Management (KM), because it enables many
intuitive queries within integrated systems. This sound framework should
contain rules that allow to infer many knowledge statements from a small core
that is well maintained. Metarel in RDF proves to be an excellent knowledge
representation language for executing such inferences. By using the Semantic
Web, the approach remains compatible with Semantic Web reasoning that is more
advanced, though less scalable, like OWL reasoning.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.semantic-systems-biology.org/metarel" mce_href="http://www.semantic-systems-biology.org/metarel"><span style="">http://www.semantic-systems-biology.org/metarel</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ward Blondé (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy25593 = 'w&#97;rd.bl&#111;nd&#101;' + '&#64;';
 addy25593 = addy25593 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy25593 + suffix + '\'' + attribs + '>' );
 document.write( addy25593 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31863 = 'w&#97;rd.bl&#111;nd&#101;' + '&#64;';
 addy31863 = addy31863 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy31863 + suffix + '\'' + attribs + '>' );
 document.write( addy31863 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18512 = 'w&#97;rd.bl&#111;nd&#101;' + '&#64;';
 addy18512 = addy18512 + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy18512 + suffix + '\'' + attribs + '>' );
 document.write( addy18512 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Department of Applied Mathematics, Biometrics and
Process Control, Ghent University, Belgium 2) Department of Biology, Norwegian
University of Science and Technology, Trondheim, Norway</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Blondé Engineering</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745318">F-11. Predicting disease causing genes using the
information content surrounding the disease and gene in literature</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">van Haagen H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;van Haagen H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Aten E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aten E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), ’t Hoen P-B (1), Roos M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Roos M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Messemaker T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Messemaker T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mons B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mons B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), van Ommen G-J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Ommen G-J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Schuemie M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schuemie M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Finding relevant information for putative disease causing
genes is difficult since information almost never is explicitly available for
the gene and the disease. Some bioinformatic approaches circumvent this problem
with the use of alternative information already known for the disease such as
previously found genes. However for the majority of the diseases recorded in
OMIM this information in not available.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this paper we describe a way to extract
information for diseases to find putitave genes even if there is no explicit
information available in text nor in any other database. The information
generated is the surrounded literature content of the disease stored in a
profile. We generated a test set where the disease was related with no gene
before the landmark paper.</p>

<h3>Results</h3>

<p class="abresults">Most prioritizers based on training genes fail, but our
method still was able to prioritize the disease gene on average within the top
13 genes out of 200 genes in a linkage interval. In addition we evaluated our
method on the test set used to evaluate Endeavour and showed that we perform
twice as good in prioritizing genes for polygenic disease. Not only is the gene
predicted, the information content shared between the gene and the disease
explains the relationship.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have proven that our approach of indirect links in
text works. Our text-mining system is implemented in the biosemantics tool
Anni. Anni can be found on http://www.biosemantics.org/anni</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Herman H.H.B.M. van Haagen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53540 = 'hv&#97;nh&#97;&#97;g&#101;n' + '&#64;';
 addy53540 = addy53540 + 'l&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy53540 + suffix + '\'' + attribs + '>' );
 document.write( addy53540 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49827 = 'hv&#97;nh&#97;&#97;g&#101;n' + '&#64;';
 addy49827 = addy49827 + 'l&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy49827 + suffix + '\'' + attribs + '>' );
 document.write( addy49827 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44326 = 'hv&#97;nh&#97;&#97;g&#101;n' + '&#64;';
 addy44326 = addy44326 + 'l&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy44326 + suffix + '\'' + attribs + '>' );
 document.write( addy44326 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">LUMC</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) LUMC, Leiden, Netherlands (2) Erasmus MC,
Rotterdam. Netherlands (3) NBIC, Netherlands</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745319">F-12. Revealing heterogeneities and inconsistencies
in protein functional annotations</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sanavia T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Sanavia T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*) Facchinetti A (1), Di Camillo B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Di Camillo B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Toffolo G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Toffolo G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lavezzo E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lavezzo E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Toppo S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Toppo S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Fontana P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fontana P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4)</p>

<p class="abmotivation">Gene Ontology (GO) is the most widely used annotation
database to transfer biological knowledge to a new raw sequence. However, it is
known that the existing annotations are incomplete and susceptible to several sources
of errors (King, 2003); in this context, one of the most important challenges
is to interpret the accuracy and consistency of the available annotations, in
order to identify or correctly predict the functions of protein sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a new method to analyze
heterogeneities of GO annotations. For groups of proteins sharing a high
sequence similarity, the method performs two agglomerative hierarchical
clustering based on semantic similarity between GO terms and protein annotations,
respectively. Semantic similarity is computed using Lin’s measure (Lin, 1998)
and Best Match Average (Couto, 2007). The algorithm gives as output a
color-coded matrix where each cell (i,j) is colored if the protein j is
annotated with the GO term i, with different color intensities representing the
information content of GO terms.</p>

<h3>Results</h3>

<p class="abresults">Proteins with the same sequence length were first clustered
in order to define groups of proteins at a high sequence similarity level (more
than 95%) and their GO annotations were retrieved from GOA database. The method
was able to reveal inconsistencies highlighting the presence of isolated cells
representing GO terms and/or proteins outliers and indicating the specificity
of the functional annotations through the information content. Several groups
of proteins with high sequence similarity evidenced a lack of homogeneity among
GOA annotations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The proposed method is able to efficiently organize
information on GO annotations of groups of proteins through semantic similarity
measures. Results in the GOA database evidenced unexpected and not easily
traceable heterogeneities on protein annotations. The algorithm can be applied
as a pre-processing step in order to identify incoherencies in biological information,
thus avoiding propagations of potential annotation errors. The resulting output
can be used as a useful guidance to predict functions of new protein sequences
or to re-annotate the existing annotations of known proteins.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tiziana Sanavia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74972 = 's&#97;n&#97;v&#105;&#97;t' + '&#64;';
 addy74972 = addy74972 + 'd&#101;&#105;' + '&#46;' + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy74972 + suffix + '\'' + attribs + '>' );
 document.write( addy74972 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40899 = 's&#97;n&#97;v&#105;&#97;t' + '&#64;';
 addy40899 = addy40899 + 'd&#101;&#105;' + '&#46;' + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy40899 + suffix + '\'' + attribs + '>' );
 document.write( addy40899 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93617 = 's&#97;n&#97;v&#105;&#97;t' + '&#64;';
 addy93617 = addy93617 + 'd&#101;&#105;' + '&#46;' + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy93617 + suffix + '\'' + attribs + '>' );
 document.write( addy93617 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Information Engineering, University of
Padova</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Information Engineering, University
of Padova, via G. Gradenigo, 6/B, 35131 Padova, Italy. (2) Department of
Histology, Microbiology and Medical Biotechnologies, University of Padova, via
Gabelli 63, 35131 Padova, Italy. (3) Department of Biological Chemistry,
University of Padova, viale Colombo, 3, 35131 Padova, Italy. (4) FEM-IASMA Research
Center, Via E. Mach, 1, 38010 San Michele all’Adige(Trento), Italy.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by CARIPARO 2008/2010 “Systems
biology approaches to infer gene regulation from gene and protein time series
data”. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745320">F-13. A formal framework for evaluating the
significance of peptide-spectrum matches</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vandenbogaert M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandenbogaert M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Tandem mass spectrometry (LC-MS/MS) experiments, as a
high-throughput technology for protein identification, generate millions of
observed spectra. To correctly match every spectrum to a theoretical spectrum,
traditionally used target-decoy strategies reveal to be mostly imprecise.
Exploiting all the peaks in the spectra, and using a more formal framework for
assessing the relevance of assignments of peptides in an MS/MS proteomic
dataset, allows to bypass the need of those expensive decoy searches, and in
addition speeds up the decision process for matching an observed spectrum to a
peptide.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A given MS/MS spectrum both gives the sequence of
amino-acids and the fragment masses that constitute the spectrum-prefixes. We
derive from a spectrum all possible explanations, (amino-acid sequences with or
without post-translational modifications), that best explain the correspondence
to putative peptides in a proteome database. For a given set of patterns one
searches for multiple occurrences of the sequences from this set in a proteomic
text. We combinatorially construct the set of sequences explaining a spectrum,
and use the spectrum characteristics to assess a peptide's match propensity.</p>

<h3>Results</h3>

<p class="abresults">A counting function is constructed that counts, given the
length of a random text, the number of occurrences of a motif in this text. To
consider the match score between a spectrum and a peptide, as well as the mass
of the peptide, we consider the dot product as the match score between an
observed spectrum and a theoretically derived spectrum. For the set of peptides
that have an integer number of peaks in common with a spectrum, the set of
possible masses are enumerable, which we integrate in a counting function with
two variables, that is transformed into a probability generating function.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To assess the significance, a probability assumption is
needed to express what is expected and some criteria to establish that the
observation deviates significantly from the expected. Here we consider the
probability that a random peptide matches a spectrum with a score above a
threshold. We consider the Delta score as the match quality for peptides, i.e.
the difference of score of the best PSM and the score of the second best PSM,
for a specific spectrum. Considering specific neutral losses are a natural
extension to this framework, and lead to the mining of MSn/MSA fragmentation
spectra.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Mathias L.C. Vandenbogaert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56707 = 'm&#97;th&#105;&#97;s.v&#97;nd&#101;nb&#111;g&#97;&#101;rt' + '&#64;';
 addy56707 = addy56707 + 'p&#97;st&#101;&#117;r' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy56707 + suffix + '\'' + attribs + '>' );
 document.write( addy56707 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10864 = 'm&#97;th&#105;&#97;s.v&#97;nd&#101;nb&#111;g&#97;&#101;rt' + '&#64;';
 addy10864 = addy10864 + 'p&#97;st&#101;&#117;r' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy10864 + suffix + '\'' + attribs + '>' );
 document.write( addy10864 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56915 = 'm&#97;th&#105;&#97;s.v&#97;nd&#101;nb&#111;g&#97;&#101;rt' + '&#64;';
 addy56915 = addy56915 + 'p&#97;st&#101;&#117;r' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy56915 + suffix + '\'' + attribs + '>' );
 document.write( addy56915 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institut Pasteur, Paris, France.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institut Pasteur, Paris, France.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Benno Schwikowski's Systems Biology lab, Institut Pasteur,
Paris, France. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745321">F-14. Improving the execution of bioinformatical
workflows</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Subramanian S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Subramanian S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Sztromwasser P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sztromwasser P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Puntervoll P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Puntervoll P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Most scientific workflow engines use centralized
coordination as the choice of approach for executing bioinformatical workflows,
requiring the coordinator to send and receive all the input and output data of
component services. Such indirect data communication between the component
services increases the data-traffic of the coordinator and weakens the
performance of the workflow. To optimize this, we propose an approach where
data-flow is dynamically delegated from the coordinator to the component
services, with direct transportation of data between the component services.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The vanilla (e.g., BPEL) and the proposed data-flow
delegation methods are considered for the performance analysis. The
technologies used in the implementation are: (i) Python v2.6.4 language, for
the development of a workflow coordinator and the three component web services,
(ii) ZSI v2.1_a1 library, for the development an deployment of the web
services, and (iii) the suds v 0.3.8 library, for sending and receiving
messages between the web services.</p>

<h3>Results</h3>

<p class="abresults">We have implemented a simple gene annotation workflow to
compare the performance of the proposed and existing approaches. The gene
annotation workflow integrates and coordinates different bioinformatical
resources like UniProt/SwissProt database and BLAST tool, located in different
places. To generalize the proposed approach, the execution semantics are
formally defined, and an XML schema is developed to support the data-flow
delegation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">By comparing the obtained results, the proposed approach
optimizes 32% of workflow runtime, 7 times of bandwidth usage, 36 times of
memory usage, and 8.7 times of CPU usage. This shows that the proposed
data-flow delegating approach improves the performance of the bioinformatical
workflow and its coordinator considerably, compared to the non-delegating
classical approach. We believe that this poster session would be a great
opportunity to identify different data-centric workflows in the domain of
bioinformatics for testing our approach further.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sattanathan Subramanian (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82995 = 's&#97;t' + '&#64;';
 addy82995 = addy82995 + '&#117;n&#105;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy82995 + suffix + '\'' + attribs + '>' );
 document.write( addy82995 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45062 = 's&#97;t' + '&#64;';
 addy45062 = addy45062 + '&#117;n&#105;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy45062 + suffix + '\'' + attribs + '>' );
 document.write( addy45062 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3346 = 's&#97;t' + '&#64;';
 addy3346 = addy3346 + '&#117;n&#105;' + '&#46;' + 'n&#111;';
 document.write( '<a ' + path + '\'' + prefix + addy3346 + suffix + '\'' + attribs + '>' );
 document.write( addy3346 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Uni BCCS</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Uni BCCS, Bergen, Norway</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was carried out as part of the eSysbio
project (No: 178885) funded by the Research Council of Norway. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745322">F-15. BridgeDb: standardized access to gene,
protein and metabolite identifier mapping services</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Van Iersel MP<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Van Iersel MP&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Pico AR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pico AR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Kelder T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kelder T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gao J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gao J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Ho I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ho I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Hanspers K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hanspers K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Conklin BR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Conklin BR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Evelo CT<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Evelo CT&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Many interesting problems in bioinformatics require
integration of data from various sources. For example when combining microarray
data with a pathway database, or merging co-citation networks with
protein-protein interaction networks. Invariably this leads to an identifier
mapping problem, where different datasets are annotated with identifiers that
originate from different databases. Several existing services (e.g. BioMart)
can perform identifier mapping. Applications can be made to flexibly support
multiple mapping services or mapping services could be combined to get broader
coverage.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We implemented a development library that forms an
interface that connects mapping services and bioinformatics tools. This
interface is in the form of both a Java and REST API.</p>

<h3>Results</h3>

<p class="abresults">BridgeDb has already been integrated into several popular
bioinformatics applications, such as Cytoscape, WikiPathways, PathVisio, Vanted
and Taverna. To encourage tool developers to start using BridgeDb, we've
created code examples, online documentation, and a mailinglist to ask
questions. The BridgeDb library is open source and available at The BridgeDb
library is available at http://www.bridgedb.org.</p>

<h3>Discussion</h3>

<p class="abdiscussion">BridgeDb helps to improve the user-friendliness of
existing bioinformatics applications, by making it easy to add important new
capabilities. BridgeDb isn't just yet another mapping service: it tries to
build further on existing work, and integrate existing partial solutions. The
framework is intended for customization and adaptation to any identifier
mapping service. BridgeDb was open source from the very beginning of the
project. We believe the philosophy of open source is closely aligned to
academic values, of building on top of the work of giants.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.bridgedb.org" mce_href="http://www.bridgedb.org"><span style="" mce_style="mso-no-proof:
yes">http://www.bridgedb.org</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Martijn P. van Iersel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67553 = 'm&#97;rt&#105;jn.v&#97;n&#105;&#101;rs&#101;l' + '&#64;';
 addy67553 = addy67553 + 'b&#105;gc&#97;t' + '&#46;' + '&#117;n&#105;m&#97;&#97;s' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy67553 + suffix + '\'' + attribs + '>' );
 document.write( addy67553 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89855 = 'm&#97;rt&#105;jn.v&#97;n&#105;&#101;rs&#101;l' + '&#64;';
 addy89855 = addy89855 + 'b&#105;gc&#97;t' + '&#46;' + '&#117;n&#105;m&#97;&#97;s' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy89855 + suffix + '\'' + attribs + '>' );
 document.write( addy89855 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy59211 = 'm&#97;rt&#105;jn.v&#97;n&#105;&#101;rs&#101;l' + '&#64;';
 addy59211 = addy59211 + 'b&#105;gc&#97;t' + '&#46;' + '&#117;n&#105;m&#97;&#97;s' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy59211 + suffix + '\'' + attribs + '>' );
 document.write( addy59211 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">BiGCaT Bioinformatics, Maastricht University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Bioinformatics - BiGCaT, Maastricht
University, Maastricht, the Netherlands (2) Gladstone Institute of
Cardiovascular Disease, San Francisco, CA 94158, USA (3) Department of Computer
Science, University of Missouri, Columbia, MO 65201, USA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Google Summer of Code
program, transnational University Limburg (tUL), the BioRange program of the
Netherlands Bioinformatics Consortium (NBIC), the Netherlands Consortium for
Systems Biology (NCSB), the National Institutes of Health [GM080223,HG003053],
the European Nutrigenomics Organization (NuGO) and theDutch Scientific
Organisation (NWO) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745323">F-16. HGVbaseG2P: an advanced database for the
integration and interrogation of genetic association datasets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Free R<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Free R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Hastings R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hastings R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Thorisson GA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thorisson GA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Gollapudi VLS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gollapudi VLS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Beck T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beck T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Lancaster O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lancaster O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Brookes AJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brookes AJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Genetic association study data are rarely published in
any comprehensive fashion in journals or databases, and in the case of negative
findings such data are often not reported at all. Consequently, it is difficult
to compare and contrast the results of different studies, and it is completely
impossible to examine a full and unbiased picture of all the data that exist. To
address this deficit, the Human Genome Variation Genotype to Phenotype database
(HGVbaseG2P: http://www.hgvbaseg2p.org) has been constructed.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">HGVbaseG2P is implemented as a traditional
relational database using the MySQL platform. Data import/export tools and
database middleware components are all implemented in the Perl programming
language, leveraging a number of open-source software packages from the CPAN. The
web application is built using the Perl-based Catalyst MVC-framework.</p>

<h3>Results</h3>

<p class="abresults">HGVbaseG2P provides a simple interface and advanced search
capabilities. Researchers are able to identify studies of interest based on
chromosomal regions/genes and markers. Studies of interest can be selected for
comparison at both genome-wide and region-specific levels. Studies can also be
searched based on phenotype, via the Medical Subject Headings (MeSH) and Human
Phenotype Ontology (HPO) hierarchies. We constantly and actively search for new
data available in publications and multiple online public resources. The
database now hosts &gt;18,000,000 p-values and 352 studies.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Researchers are encouraged to directly submit studies and
association data into HGVbaseG2P. A data access control system enables data
submitters to decide what studies and which data elements are to be shared and
with whom (e.g. collaborators, own research group). By summer 2010 the complete
HGVbaseG2P code base will be available for anyone to install on their own
servers so they can run their own genetic association database.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.hgvbaseg2p.org" mce_href="http://www.hgvbaseg2p.org"><span style="" mce_style="mso-no-proof:
yes">http://www.hgvbaseg2p.org</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tim Beck (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64296 = 'tb143' + '&#64;';
 addy64296 = addy64296 + 'l&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy64296 + suffix + '\'' + attribs + '>' );
 document.write( addy64296 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63167 = 'tb143' + '&#64;';
 addy63167 = addy63167 + 'l&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy63167 + suffix + '\'' + attribs + '>' );
 document.write( addy63167 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4105 = 'tb143' + '&#64;';
 addy4105 = addy4105 + 'l&#101;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy4105 + suffix + '\'' + attribs + '>' );
 document.write( addy4105 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Leicester</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Genetics, University of Leicester, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The European Community's Seventh Framework Programme
(FP7/2007-2013) under grant agreement number 200754 – the GEN2PHEN project. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745324">F-17. Extracting phytogeography information from
species distribution data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bakis Y<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Bakis Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sezerman OU<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sezerman OU&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Babaç MT<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Babaç MT&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Today, it is possible to analyze large datasets as large
as species distribution of Turkish flora. Availability of species distributions
in digital format had motivated us to analyze this data. Thus, we have
calculated pair wise similarities among the 30 grid squares by presence of
species in the grid pairs. An algorithm has been de-signed for this purpose
because of known limitations in current multivariate analyzing techniques. By
using the similarity matrix, we constructed a phylogeny for the 30 grids
squares.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The distribution of each taxa had been indicated in
Davis’ “Flora of Turkey and East Aegean Islands” (Davis, et al., 1965).
Recently, Babaç has digitized the species distributions into a database of
Turkish Plants Data Service (TUBIVES) (Babaç, 2003). All occu-rences were coded
as “1”s and non-occurences as “0”s as in figure 2. By using the species
distribution records from the Turkish Plant Data Service (M. Tekin Babaç, 2010)
of all taxa we tried to cluster the Davis’ grids. Some taxa have been
represented by 1 or no dis-tribution record or show a global distribution.</p>

<h3>Results</h3>

<p class="abresults">Cladogram constructed by the cluster analysis has shown that
spe-cies distributions over the grid squares were highly correlated with the
phytogeographic regions. There were 3 main clusters in the cladogram topology
serving as Euro-Siberian, Irano-Turanian and Mediterranean regions. The grids
squares were all correct within these clusters with some exceptions. A2 is
assigned to be a member of Mediterranean region not Euro-Siberian which is an
important result since more than half of the sea shores of the region is
included within the Mediterranean Region.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Species distribution data gas given accurate results with
the phytogeographical regions. The two very important distinction were pointing
already defined problems in phytogeography of Turkey.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Yasin Bakis (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71151 = 'b&#97;k&#105;s_y' + '&#64;';
 addy71151 = addy71151 + '&#105;b&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy71151 + suffix + '\'' + attribs + '>' );
 document.write( addy71151 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99513 = 'b&#97;k&#105;s_y' + '&#64;';
 addy99513 = addy99513 + '&#105;b&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy99513 + suffix + '\'' + attribs + '>' );
 document.write( addy99513 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65003 = 'b&#97;k&#105;s_y' + '&#64;';
 addy65003 = addy65003 + '&#105;b&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy65003 + suffix + '\'' + attribs + '>' );
 document.write( addy65003 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Abant Izzet Baysal University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Biology, Abant Izzet Baysal University,
Bolu, 14280 Turkey 2 Biological Sciences and Bioengineering, Sabanci
University, Tuzla, Istanbul, 34956 Turkey</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745325">F-18. D2K - data to knowledge: data integration for
biological reasoning</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wienecke-Baldacchino AK<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wienecke-Baldacchino AK&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Heinäniemi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Heinäniemi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>, Carlberg C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Carlberg C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Hypothesis driven data mining faces problems, like a high
amount of public databases to evaluate and quality control, in order to extract
information in a computationally processable format and its automating via
interfaces. Additionally, for new types of high throughput data, e.g. ChIP-seq,
no public databases are established yet, which would provide a certain
standardized data format in particular concerning annotation or analysis
aspects. Therefore, we decided to set up a database system integrating
published, in silico generated and experimentally achieved data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our tool is based on a relational database system.
Processing or insert functions are implemented in Python. The selection of
public data sources and ontology was done by hand. Here aspects like data
quality, data structure, formats, automation possibilities and provided
interfaces were considered. Data consistency especially for published data is
provided by inter-database-confirmation, only cross-references confirmed over
different public databases are inserted. We also integrated in silico generated
genome-wide transcription factor binding sites and respective experimental
data.</p>

<h3>Results</h3>

<p class="abresults">The present version of the database consists of 98 tables
and 209 relations. Several public databases, e.g. dbSNP, miRBase, GAD, HGNC,
Entrez and Ensembl, are integrated. Further data sources are under revision.
For 1058 position weight matrices genome-wide data are inserted and the
integration of experimental data is ongoing. Selected current applications are:
Decision making if to evaluate published, non-coding, significantly associated
SNPs; ID mappings for automated text mining; selection of candidate SNPs for
association studies and selection of candidate miRNAs by ChIP-seq data
matching.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The main intention of the presented system is to take
advantage of the huge amount of already available data and to transfer data to
knowledge namely a reasonable biological hypothesis, so that the wet-lab serves
only for validation. Main challenges are to curate the system and keep it
up-to-date, especially in context of genome versions, coordinates and IDs.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anke K. Wienecke-Baldacchino (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17159 = 'Ank&#101;.W&#105;&#101;n&#101;ck&#101;' + '&#64;';
 addy17159 = addy17159 + '&#117;n&#105;' + '&#46;' + 'l&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy17159 + suffix + '\'' + attribs + '>' );
 document.write( addy17159 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13391 = 'Ank&#101;.W&#105;&#101;n&#101;ck&#101;' + '&#64;';
 addy13391 = addy13391 + '&#117;n&#105;' + '&#46;' + 'l&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy13391 + suffix + '\'' + attribs + '>' );
 document.write( addy13391 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy824 = 'Ank&#101;.W&#105;&#101;n&#101;ck&#101;' + '&#64;';
 addy824 = addy824 + '&#117;n&#105;' + '&#46;' + 'l&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy824 + suffix + '\'' + attribs + '>' );
 document.write( addy824 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Life Sciences Research Unit, University of
Luxembourg, Luxembourg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Life Sciences Research Unit, University of Luxembourg,
Luxembourg</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745326">F-19. BRENDA text-mining: new developments for
obtaining enzyme-related disease information from scientific literature</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Soehngen C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Soehngen C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Scheer M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Scheer M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Schomburg I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schomburg I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Chang A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chang A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Grote A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Grote A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Schomburg D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schomburg D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">BRENDA (BRaunschweig ENzyme DAtabase) provides manually
annotated data and additionally information extracted from the scientific
literature by a text mining approach. The part of the text mining component,
which focuses on the search for enzyme and disease related references, has been
revised and augmented with a subsequent classification of mining results to get
an insight on the stressed issues of the reference. The F1 score for
classification, evaluated in a 5fold cross validation, ranges between
0.802+/-0.032 - 0.738+/-0.033 depending on the text class and preprocessing
procedures.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In a text mining approach PubMed references with
enzyme hits in title or abstract are searched for co-occurring disease terms.
The dictionaries are BRENDA enzyme names or synonyms and MeSH disease terms.
The classification is processed by SVMlight, a Support Vector Machine (SVM)
implementation. For training purposes a corpus has been manually annotated,
containing 5,033 sentences derived from PubMed abstract and titles. The aim is
the classification of mining results which may belong to one or more of the
four pre-defined categories or none of them.</p>

<h3>Results</h3>

<p class="abresults">In the summer release 2010.2 910,897 disease entries
(522,720 distinct PubMed references) are added to BRENDA (co-occurrence related
F1 score 0.89). The further SVM classification was evaluated in a 5 fold cross
validation: Categories (F1 scores +/- std. dev.) The enzyme is a therapeutic
agent or target (0.802+/-0.032) The intention of research about the connection
of the enzyme and the disease is stated (0.744+/-0.020) The enzyme is causal
for the disease or part of the disease process (0.743+/-0.009) The
characteristic values of the enzyme are used for diagnostic purpose
(0.738+/-0.033)</p>

<h3>Discussion</h3>

<p class="abdiscussion">BRENDA provides a large number of disease related enzyme
entries which are half yearly updated in the course of every new release of the
database. However, the categorisation through SVM classification will be a
supplemental information to disease entries in future BRENDA releases. A high
value on precision will be set on the choice of parameter settings for the SVM.
Thus, the classification will be a reliable indicator on the stressed issues of
every reference found by disease mining.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.brenda-enzymes.org" mce_href="http://www.brenda-enzymes.org"><span style="">http://www.brenda-enzymes.org</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Carola Söhngen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99300 = 'c.s&#111;&#101;hng&#101;n' + '&#64;';
 addy99300 = addy99300 + 't&#117;-bs' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy99300 + suffix + '\'' + attribs + '>' );
 document.write( addy99300 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57131 = 'c.s&#111;&#101;hng&#101;n' + '&#64;';
 addy57131 = addy57131 + 't&#117;-bs' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy57131 + suffix + '\'' + attribs + '>' );
 document.write( addy57131 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13742 = 'c.s&#111;&#101;hng&#101;n' + '&#64;';
 addy13742 = addy13742 + 't&#117;-bs' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy13742 + suffix + '\'' + attribs + '>' );
 document.write( addy13742 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Technische Universitaet Braunschweig</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Technische Universitaet Braunschweig (2) Enzymeta
GmbH, Erftstadt</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">European Union Projects: FELICS (Free European Life-Science
Information and Computational Services) SLING (Serving Life-science Information
for the Next Generation) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745327">F-20. Beegle: a new search engine for discovering
novel genes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Brohée S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Brohée S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Gonçalves JP<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gonçalves JP&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3), Nitsch D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nitsch D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Observing the high rate of publication and the complexity
of many fields in genetics, it becomes increasingly difficult to keep this
knowledge up-to-date considering a given biological process, to have a quick
overview of the underlying genetic mechanisms or to initiate new studies to
discover novel genes implied in this process. To solve this, we designed a new
user-friendly strategy allowing the easy retrieval of known genes and the
prediction of novel genes related to any query or search term.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach combines two methods. Firstly,
starting from any text query, our tool retrieves the set of linked PUBMED
abstracts and builds a vector whose components reflect the weight of given
terms found in these abstracts. This query vector is then compared to a set of
reference vectors, each one corresponding to a given gene, built using the
GeneRIF resource (which lists references for each gene). In the second step,
wes discover novel genes by applying a network search strategy with the genes
whose word weight vector was the most similar to the query vector.</p>

<h3>Results</h3>

<p class="abresults">The first results delivered by our approach are very
encouraging. We are currently benchmarking and statistically assessing our
approach for its ability to detect relevant known and novel genes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Starting from a simple free text query, our user-friendly
method is able to return relevant genes linked to this query. We are confident
that our method will be of interest: on the one hand, to the wet lab scientist
willing an overview of the underlying genetic processes or to discover new
genes concerning a process of interest; on the other hand, to the
bioinformatician in need of training data sets of genes to run an application
(e.g. prioritization tool).</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://homes.esat.kuleuven.be/%7Ebioiuser/beegle/" mce_href="http://homes.esat.kuleuven.be/~bioiuser/beegle/"><span style="">http://homes.esat.kuleuven.be/~bioiuser/beegle/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sylvain Brohée (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36038 = 'sbr&#111;h&#101;&#101;' + '&#64;';
 addy36038 = addy36038 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy36038 + suffix + '\'' + attribs + '>' );
 document.write( addy36038 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82723 = 'sbr&#111;h&#101;&#101;' + '&#64;';
 addy82723 = addy82723 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy82723 + suffix + '\'' + attribs + '>' );
 document.write( addy82723 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45605 = 'sbr&#111;h&#101;&#101;' + '&#64;';
 addy45605 = addy45605 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy45605 + suffix + '\'' + attribs + '>' );
 document.write( addy45605 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioinformatics group. ESAT-SCD. Katholieke
Universiteit Leuven.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics group, ESAT-SCD, Katholieke
Universiteit Leuven. (2) Knowledge Discovery and Bioinformatics group,
INESC-ID. (3) Instituto Superior Técnico, Technical University of Lisbon. (+)
Equal contribution</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Sylvain Brohée is the recipient of a post-doctoral
grant supported by the EU FP7 CHeartED project (Grant HEALTH-F2-2008-223040).
Bioinformatics group is supported by Research Council KUL: ProMeta, GOA
Ambiorics, GOA MaNet, CoE EF/05/007 SymBioSys, START 1, several PhD/postdoc
&amp; fellow grants. Joana P. Gonçalves is the recipient of a doctoral grant
supported by FCT (SFRH/BD/36586/2007). INESC-ID multiannual funding is
supported by FCT through the PIDDAC Program funds. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745328">F-21. An overview of the pathogen-host interactions
database, PHI-base</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Janowska-Sejda E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Janowska-Sejda E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Defoin-Platel M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Defoin-Platel M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Hammond-Kosack K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hammond-Kosack K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Urban M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Urban M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Tsoka S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tsoka S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Saqi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Saqi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Recent advances in sequencing technologies resulted in a
wealth of full-genome information becoming available for many pathogenic
species. Over the years, the number of publications describing a functional
characterisation of pathogenicity genes, virulence and effector genes has grown
significantly. PHI-base is a database, manually curated by scientists, which
systematically catalogues phenotypic information extracted from peer-reviewed
publications for genes involved in pathogen–host interactions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The sequences in PHI-base version 3.2 were
clustered on the basis of sequence similarity and represented graphically to
give a global overview of the database content in terms of phenotype and
experimental hosts.</p>

<h3>Results</h3>

<p class="abresults">General analysis of PHI-base content revealed the presence
of 807 non-redundant genes for which nucleotide and protein sequences are
available in the database. These genes belong to 75 different pathogenic
species. The total number of interactions within the PHI-base equals 1314.
Overall 111 clusters with two or more genes were identified. The clustering
identified 52 plant specific gene clusters and 25 animal specific gene
clusters. Pathogenicity genes in 33 mixed clusters identified genes important
for host invasion both in animal and plant pathogens.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our comparative gene set analysis of genes involved in
pathogen-host interactions across diverse species identified orthologous genes
conserved in plants and/or animal pathogens. We next plan to use HMM3 profiling
to confirm these findings and identify the molecular themes required for host
specificity.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Elzbieta I Janowska-Sejda (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54550 = '&#101;lzb&#105;&#101;t&#97;.j&#97;n&#111;wsk&#97;-s&#101;jd&#97;' + '&#64;';
 addy54550 = addy54550 + 'bbsrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy54550 + suffix + '\'' + attribs + '>' );
 document.write( addy54550 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36262 = '&#101;lzb&#105;&#101;t&#97;.j&#97;n&#111;wsk&#97;-s&#101;jd&#97;' + '&#64;';
 addy36262 = addy36262 + 'bbsrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy36262 + suffix + '\'' + attribs + '>' );
 document.write( addy36262 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95431 = '&#101;lzb&#105;&#101;t&#97;.j&#97;n&#111;wsk&#97;-s&#101;jd&#97;' + '&#64;';
 addy95431 = addy95431 + 'bbsrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy95431 + suffix + '\'' + attribs + '>' );
 document.write( addy95431 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Rothamsted Research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Biomathematics and Bioinformatics,
Rothamsted Research 2 Department of Plant Pathology and Microbiology,
Rothamsted Research 3 Department of Computer Science, King's College London</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is fully funded by BBSRC</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745329">F-22. Finding disease related genes by GeneRank
algorithm using co-occurrence based network structures</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lee H-M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Lee H-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Shin M-Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shin M-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Hong M-P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hong M-P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2)</p>

<p class="abmotivation">To find significant genes related to specific diseases,
microarray gene expression profiles have been widely used for gene ranking.
Lately, however, the GeneRank algorithm was proposed which employs gene
annotation data from Gene Ontology to construct gene networks and use them for
the gene ranking along with gene expression profiles. To make use of relevant
information from bio-literature, this work presents a method to decide the
ranks of genes by using text-mining based method to construct gene networks for
the GeneRank algorithm.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We assume that two or more genes are likely to be
related, if they co-occur in a sentence. In this approach the correct
recognition of gene entities is of great importance. For this purpose, we
employ a dictionary-based approach and a machine learning approach. Once
co-occurring genes are found in a sentence, they are taken to be related and a
new link is added to the gene network. For the experiment, we used text data
related to prostate cancer from the PubMed. The prostate cancer gene expression
profiles were obtained from the Gene Expression Omnibus.</p>

<h3>Results</h3>

<p class="abresults">Based on the gene networks and gene expression profiles, we
compute the gene ranking scores with GeneRank algorithm. Once the ranking
scores are computed, the genes are sorted in a decreasing order. For biological
validation, we obtained gene lists already known to be related to prostate
cancer. Using these gene lists, we find out how many genes in the list are
identified by our method. The experiments show that our method outperforms the
previous works based on the Gene Ontology. It also shows that the machine
learning approach produces better results than the dictionary based approach.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The gene networks based on text mining produced a better
result than the GO-based gene networks in computing gene ranking scores. Our
experiment shows that it is of great worth to use gene relations found in
bio-literature for disease related gene identification. As a future work, we
are planning to employ a syntactic parser customized for a medical domain to
extract relations between genes in a more sophisticated manner, so that a
directed gene network can be built.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Miyoung Shin (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98876 = 'sh&#105;nmy' + '&#64;';
 addy98876 = addy98876 + 'kn&#117;' + '&#46;' + '&#97;c' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy98876 + suffix + '\'' + attribs + '>' );
 document.write( addy98876 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11234 = 'sh&#105;nmy' + '&#64;';
 addy11234 = addy11234 + 'kn&#117;' + '&#46;' + '&#97;c' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy11234 + suffix + '\'' + attribs + '>' );
 document.write( addy11234 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36329 = 'sh&#105;nmy' + '&#64;';
 addy36329 = addy36329 + 'kn&#117;' + '&#46;' + '&#97;c' + '&#46;' + 'kr';
 document.write( '<a ' + path + '\'' + prefix + addy36329 + suffix + '\'' + attribs + '>' );
 document.write( addy36329 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Kyungpook National University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Kyungpook National University (2) Sungkyunkwan
University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Korea Science and
Engineering Foundation grant (No. R01-2008-000-11089-0) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745330">F-23. Identifying variability of human splicing
forms in their interactions by using literature mining</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kafkas S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kafkas S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Varoglu E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Varoglu E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rebholz-Schuhmann D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rebholz-Schuhmann D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Taneri B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Taneri B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Alternative splicing (AS) is a prominent mechanism
contributing to transcriptome and proteome diversity by generating multiple
protein isoforms from a single gene. Isoforms exhibiting differences in their
structures and functions can be expected to exhibit variations in their
interactions. Although there are several PPI databases providing structured
information on PPIs, they cover only a portion of the interactome and
interaction information on isoforms is sparse. Literature has rich information
content. Our work provides a novel extension both to AS and PPIs including
variability analysis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We analyze the transcript data from HumanSDB3 for
16,826 different genes and collect relevant abstracts from PubMed. Then we
employ an SVM trained on BioCreative-II IAS corpus with a novel and high
performing feature set for selecting interaction abstracts. Another SVM trained
on AIMed corpus by using syntactic features is utilized to extract interacting
proteins from the selected abstracts. We evaluate our DB against PINA. We
utilize it to select the genes having multiple interacting isoforms and
identify their unique and shared interactions to determine the variation in
their interactions.</p>

<h3>Results</h3>

<p class="abresults">We process around 4 million abstracts and find a total of
31,819 distinct interactions belonging to 5,615 isoform from HumanSDB3. 69.04%
of proteins and 9.00% of interactions overlap with PINA. Variability analysis
shows that all isoforms from the majority of genes having multiple interacting
isoforms (81.82%) have unique interaction partners only. Isoforms from 17.04%
of such genes have both; shared and unique interaction partners while isoforms
from the remaining 1.14% genes exhibit shared interactions only.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Evaluation of our PPI DB against PINA yields low
interaction overlap due to different interaction extraction methods used to
build PPI databases and low number of isoform interactions in such databases.
Interaction variability analysis expose that the majority of protein isoforms
exhibit differences in their interaction partners. Our analysis demonstrates
that alternative splicing significantly contributes to the variability in the
isoform interactions. Our findings will be publicly available through a web
interface for further usage.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Senay Kafkas (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92492 = 'k&#97;fk&#97;s' + '&#64;';
 addy92492 = addy92492 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy92492 + suffix + '\'' + attribs + '>' );
 document.write( addy92492 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67940 = 'k&#97;fk&#97;s' + '&#64;';
 addy67940 = addy67940 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy67940 + suffix + '\'' + attribs + '>' );
 document.write( addy67940 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy47193 = 'k&#97;fk&#97;s' + '&#64;';
 addy47193 = addy47193 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy47193 + suffix + '\'' + attribs + '>' );
 document.write( addy47193 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">European Bioinformatics Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1-Department of Computer Engineering, Eastern
Mediterranean University, Famagusta, North Cyprus 2-European Bioinformatics
Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, CB10 1SD, UK 3-Faculty
of Arts and Sciences, Eastern Mediterranean University, Famagusta, North Cyprus</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745331">F-24. The meaning behind the choice of words: a
case in cancer metastasis.</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Divoli A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Divoli A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2, *), Mendonca EA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mendonca EA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,5), Rzhetsky A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rzhetsky A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4)</p>

<p class="abmotivation">We interviewed 28 cancer metastasis experts to investigate
how uniform their views are in a number of topics in the research field. While
studying the audio files and transcripts we noticed several linguistic
phenomena: hesitation, (un)certainty, hedging, use of anecdotes, metaphors and
so on. Based on our initial observations we decided to study the texts more
systematically to detect correlations between specific language use and topics
of discussion and/or types of experts.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We split our data into 14 topics and grouped our
experts in categories based on their doctoral training (PhD and/or MD), their
gender, the number of years since their first doctorate, and if the interviews
were conducted in person or over the phone. We then manually analyzed our data
to pick up language idioms such as metaphors and proceeded with several
automatic methods. We exploited word and N-gram frequencies, applied commercial
linguistic inquiry tools, and performed extensive studies employing
crowd-sourcing.</p>

<h3>Results</h3>

<p class="abresults">Our results exhibited several interesting points, such as
that some discussion topics like “tropism” provoked more strong opinions and
inspired the use of metaphors, and certain expert groups overall used a more
assured tone of voice. The linguistic analysis revealed high use of language
associated with cognitive processes (certainty and insight in particular). When
this language is known to be under-represented in scientific text, in our data
it was over-represented, often, more than one standard deviation over any
control text, including oral speech and even novels.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Unlike the view one might get by looking at textbooks and
the scientific literature, our results show that, in reality, the knowledge is
not as crisp. There is speculation, uncertainty and difference in opinion.
Beyond the value of these data for the generation of hypothesis, researchers
that analyze biomedical text and are concerned with knowledge representation
should examine the effect that this uncertainty has on their work.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anna Divoli (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49407 = 'd&#105;v&#111;l&#105;' + '&#64;';
 addy49407 = addy49407 + '&#117;ch&#105;c&#97;g&#111;' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy49407 + suffix + '\'' + attribs + '>' );
 document.write( addy49407 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50934 = 'd&#105;v&#111;l&#105;' + '&#64;';
 addy50934 = addy50934 + '&#117;ch&#105;c&#97;g&#111;' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy50934 + suffix + '\'' + attribs + '>' );
 document.write( addy50934 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92255 = 'd&#105;v&#111;l&#105;' + '&#64;';
 addy92255 = addy92255 + '&#117;ch&#105;c&#97;g&#111;' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy92255 + suffix + '\'' + attribs + '>' );
 document.write( addy92255 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The University of Chicago</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">The University of Chicago, (1) Department of Medicine,
(2) Institute for Genomics and Systems Biology, (3) Computation Institute (4)
Department of Human Genetics (5) Department of Pediatrics</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Funded by the Ludwig Institute of Cancer Research</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745332">F-25. Linked open experimentation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Roos M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Roos M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van Haagen HHHBM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Haagen HHHBM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Singh B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Singh B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Marshall MS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marshall MS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Mons B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mons B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mons BM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The amount of data and knowledge and the number of
methods for exploring biological data towards a new hypothesis are large and
increasing each day. We investigate the application of a combination of
information technologies that can help us explore knowledge beyond our own
locally produced data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our approach combines (i) Linked Data and the
Semantic Web (RDF/OWL) to capture the relations between data from several
experiments and the biological hypotheses related to them; (ii) the Concept Web
as a reference for disambiguated biological concepts and ‘nano-publications’;
(iii) ‘Concept Mining’ methods to mine knowledge from text and other sources in
terms of the Concept Web; (iv) Taverna, myExperiment.org, BioCatalogue.org, and
the 'AIDA' plugin for Taverna to capture the design and provenance of a
bioinformatics analysis workflow in relation to its results and biological
interpretation.</p>

<h3>Results</h3>

<p class="abresults">We present a workflow that helps interpret the pre-processed
data from genome wide association studies in the context of two case studies:
(i) elucidating the role of epigenetic factors in the progression of
Huntington’s Disease and (ii) finding relations between BioBanks (a pilot by
the Free University of Amsterdam, the Erasmus Medical Centre Rotterdam, and the
Leiden University Medical Centre). We present semantic (SPARQL) queries that
retrieve evidential relations, and putatively novel relations across
experiments.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We conclude that adoption of web standards to expose (in
a controlled manner) all elements of bioinformatics experiments, including the
biological concepts that make a hypothesis, has the potential to enhance
biological research by enabling computational support for reasoning over the
multitude of data and information that may be relevant for a better hypothesis.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Marco Roos (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52752 = 'm.r&#111;&#111;s' + '&#64;';
 addy52752 = addy52752 + 'l&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy52752 + suffix + '\'' + attribs + '>' );
 document.write( addy52752 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18486 = 'm.r&#111;&#111;s' + '&#64;';
 addy18486 = addy18486 + 'l&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy18486 + suffix + '\'' + attribs + '>' );
 document.write( addy18486 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82109 = 'm.r&#111;&#111;s' + '&#64;';
 addy82109 = addy82109 + 'l&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy82109 + suffix + '\'' + attribs + '>' );
 document.write( addy82109 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Leiden University Medical Centre (LUMC) &amp;
University of Amsterdam (UvA)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Leiden University Medical Center 2) University of
Amsterdam 3) Erasmus Medical Center Rotterdam</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported through the Netherlands
BioInformatics Centre (NBIC; BioRange program), the Virtual Laboratory
e-Science Project (Dutch Ministry of Economic Affairs) and the Centre for
Medical Systems Biology (CMSB). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745333">F-26. First CALBC challenge: first results</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Rebholz-Schuhmann D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rebholz-Schuhmann D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Jimeno Yepes A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jimeno Yepes A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Li C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Li C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), van Mulligen E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Mulligen E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Kors J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kors J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Milward D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Milward D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Hahn U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hahn U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4)</p>

<p class="abmotivation">CALBC is a support action project that brings together
the researchers from international biomedical text mining groups to address the
difficult issue of annotating large text corpora with a large set of semantic
types [1, 2]. CALBC introduced a collaborative approach to this annotation task
in the form of an open challenge to the biomedical text mining community. The
task is the annotation of named entities in a large biomedical corpus, for a
variety of semantic categories [3]. CALBC will deliver a large, collaboratively
annotated corpus, marked with the mentions of biomedical entities.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The corpus was a selection of 100k Medline
abstracts. The documents have been automatically annotated by the 16
participants of the first CALBC challenge round (22 submissions overall). The
harmonization of these annotations will be made available to the research
community in Juli 2010 and will serve as a training corpus for the CALBC
challenge II (second half of 2010).</p>

<h3>Results</h3>

<p class="abresults">From the 22 submitted annotations to the first CALBC
challenge, 9 were trained on the 50k abstracts training corpus. 14 of these
submissions annotated disorders, 19 annotated proteins and genes, 13 annotated
species, and 14 annotated chemicals. The best performing solutions for the
annotation of the boundaries in the corpus reached an F-measure above 80%
across all semantic groups.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Submissions of annotations to the CALBC annotation server
contain positional information on the phrases that have been recognized as
biomedical entities and information on the medical entity type. For the
biomedical entity types we used a classification that is a variant of McCray's
classification [4]. Optionally, the text mining systems could provide
annotations for term from additional lexical resources (MeSH, GO, etc.).</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.calbc.eu" mce_href="http://www.calbc.eu"><span style="" mce_style="mso-no-proof:
yes">http://www.calbc.eu</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Dietrich RG Rebholz-Schuhmann (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11962 = 'r&#101;bh&#111;lz' + '&#64;';
 addy11962 = addy11962 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy11962 + suffix + '\'' + attribs + '>' );
 document.write( addy11962 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82782 = 'r&#101;bh&#111;lz' + '&#64;';
 addy82782 = addy82782 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy82782 + suffix + '\'' + attribs + '>' );
 document.write( addy82782 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45276 = 'r&#101;bh&#111;lz' + '&#64;';
 addy45276 = addy45276 + '&#101;b&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy45276 + suffix + '\'' + attribs + '>' );
 document.write( addy45276 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">European Bioinformatics Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 EMBL-EBI, Hinxton, U.K. 2 Erasmus Medical Center, The
Netherlands 3 Linguamatics, U.K. 4 Friedrich-Schiller-Universität Jena, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by the EU Support Action grant
231727 under the 7th EU Framework Programme within Theme “Intelligent Content
and Semantics” (ICT 2007.4.2). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745334">F-27. EuroPhenome: a repository for high-throughput
mouse phenotyping data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Morgan H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Morgan H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Hassan A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hassan A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Blake A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Blake A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Hancock JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hancock JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Mallon A-M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mallon A-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The broad aim of biomedical science in the postgenomic
era is to link genomic and phenotype information to allow deeper understanding
of the processes leading from genomic changes to altered phenotype and disease.
Essential to developing such a linkage are databases which contain information
on inbred mouse strain and mutant phenotypes. EUMODIC is gathering data from
the EMPReSSslim pipeline which is performed on inbred mouse strains and on
knock-out lines arising from the EUCOMM project. The EuroPhenome interface
allows the user to access the data via the phenotype or genotype.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The EuroPhenome database is implemented in MySQL
running on Solaris. The Phenome Data Viewer is implemented as a Java Servlet
and the PhenoMap and Ontology Tree are implemented in PHP. All the web site is
styled using css and uses DHTML and AJAX to provide the relevant functionality.
The images are generated by JFreeChart. Memcached is used to improve
performance. Particular pheodeviants are annotated with terms from the
Mammalian Phenotype Ontology.</p>

<h3>Results</h3>

<p class="abresults">EuroPhenome allows users access the raw data and the
annotated data described above through three new integrated web tools. The
‘Phenome Data Viewer’ allows access to the raw data, presented graphically. All
lines within Europhenome are compared to relevant baseline animals and
significant differences are recorded and are viewable by the ‘PhenoMap’ and the
‘Mine for a Mutant’ Tool.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Europhenome has data for 151 mutant lines, including a
total of over 2 million data points. This will increase to 500 mutant lines
over the next 18 months. This provides useful data for the examination of
individual genes, finding mutant lines if interest to a particular field of
research and analysis of the data set as a whole to find interesting biological
phenomena.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.europhenome.org" mce_href="http://www.europhenome.org"><span style="" mce_style="mso-no-proof:
yes">http://www.europhenome.org</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Hugh L Morgan (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16066 = 'h.m&#111;rg&#97;n' + '&#64;';
 addy16066 = addy16066 + 'h&#97;r' + '&#46;' + 'mrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy16066 + suffix + '\'' + attribs + '>' );
 document.write( addy16066 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53932 = 'h.m&#111;rg&#97;n' + '&#64;';
 addy53932 = addy53932 + 'h&#97;r' + '&#46;' + 'mrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy53932 + suffix + '\'' + attribs + '>' );
 document.write( addy53932 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44788 = 'h.m&#111;rg&#97;n' + '&#64;';
 addy44788 = addy44788 + 'h&#97;r' + '&#46;' + 'mrc' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy44788 + suffix + '\'' + attribs + '>' );
 document.write( addy44788 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">MRC Harwell, Mammalian Genetics Unit</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">MRC Harwell, Mammalian Genetics Unit</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the EUMODIC project and the
MRC. </p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="119-text-mining-ontologies-and-databases.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>