<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Protein Interactions, Molecular Networks and Systems Biology</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Protein Interactions, Molecular Networks and Systems Biology			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Protein Interactions, Molecular Networks and Systems
Biology</span><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;text-transform:
uppercase"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Benno Schwikowski and Denis Thieffry<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Protein Interactions,
Molecular Networks and Systems Biology<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745772 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span><span style=""><o:p></o:p></span></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745773" mce_href="#_Toc269745773">G-1. Phenotypic effects of network
rewiring in regulatory hierarchies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745773 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">4</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745774" mce_href="#_Toc269745774">G-2. Bringing order to disorder:
comparative genomics and genetic interactions uncover three biologically
distinct forms of protein disorder<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745774 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745775" mce_href="#_Toc269745775">G-3. Prediction of genetic
interactions in yeast using machine learning<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745775 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745776" mce_href="#_Toc269745776">G-4. Prioritizing candidate genes by
network analysis of differential expression using machine learning approaches<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745776 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745777" mce_href="#_Toc269745777">G-5. The human E3 ubiquitin ligase
enzyme protein interaction network<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745777 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745778" mce_href="#_Toc269745778">G-6. Fast motif enumeration and
clustering in integrated networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745778 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">9</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745779" mce_href="#_Toc269745779">G-7. Dynamic Deterministic Effects
Propagation Networks: learning signalling pathways from longitudinal protein
array data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745779 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700370039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745780" mce_href="#_Toc269745780">G-8. Systematic mapping of multiple
specificity in peptide recognition modules reveals new binding modes of protein
domains<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745780 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745781" mce_href="#_Toc269745781">G-9. Reverse-engineering gene
regulatory networks for the abiotic stress response in Arabidopsis thaliana<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745781 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745782" mce_href="#_Toc269745782">G-10. Predicting protein-protein
interaction using mirror tree<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745782 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745783" mce_href="#_Toc269745783">G-11. Comparison of Bayesian
networks and its extensions applied to the inference of regulatory networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745783 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745784" mce_href="#_Toc269745784">G-12. Insight into the mechanism of
specific regulation of STAT proteins by atypical dual-specificity phosphatases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745784 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745785" mce_href="#_Toc269745785">G-13. VoteDock: The consensus
docking method for prediction of protein-ligand interactions<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745785 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745786" mce_href="#_Toc269745786">G-14. Inferring genetic regulatory
networks with a hierarchical Bayesian model and a parallel sampling algorithm<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp;&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745786 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745787" mce_href="#_Toc269745787">G-15. Prediction of protein-protein
interactions in the apoptosis pathway<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745787 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">18</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745788" mce_href="#_Toc269745788">G-16. Systematic network analysis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745788 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">19</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745789" mce_href="#_Toc269745789">G-17. Model of trisporic acid
synthesis in Mucorales shows bistable behaviour<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745789 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">20</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700380039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745790" mce_href="#_Toc269745790">G-18. Simulation of immune response
to Mycobacterium tuberculosis using an agent-based model<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:
windowtext;display:none;mso-hide:screen;text-decoration:none;text-underline:
none"><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745790 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">21</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745791" mce_href="#_Toc269745791">G-19. Design, optimization and
predictions of a coupled model of the cell cycle, circadian clock, DNA repair
system, irinotecan metabolism and exposure control under temporal logic
constraints<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745791 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">22</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745792" mce_href="#_Toc269745792">G-20. Adding structural information
to the von Hippel-Lindau (VHL) tumor suppressor interaction network<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745792 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">23</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745793" mce_href="#_Toc269745793">G-21. Computational multiscale
modeling of brain tumor growth<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745793 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">24</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745794" mce_href="#_Toc269745794">G-22. SLIDER: a generic
metaheuristic for the discovery of correlated motifs in protein-protein
interaction networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745794 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">25</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745795" mce_href="#_Toc269745795">G-23. The effective interactions
between proteins via coarse-grained modeling<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745795 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">26</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745796" mce_href="#_Toc269745796">G-24. Predicting cellular
perturbation effects and identifying causal perturbations using a
message-passing based machine learning method<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745796 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">27</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745797" mce_href="#_Toc269745797">G-25. Towards better receptor-ligand
prediction<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745797 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">28</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745798" mce_href="#_Toc269745798">G-26. Extracting and visualising
protein-protein interfaces for data-driven docking<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745798 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">29</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745799" mce_href="#_Toc269745799">G-27. Visualizing spatiotemporal
information in heterogeneous biological networks with Arena3D<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp;&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745799 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">30</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003700390039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745800" mce_href="#_Toc269745800">G-28. Efficient learning of
signaling networks from perturbation time series via dynamic nested effects
models<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745800 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">31</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745801" mce_href="#_Toc269745801">G-29. Discovering patterns of
differentially regulated enzymes in metabolic pathways of tumors<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745801 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">32</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745802" mce_href="#_Toc269745802">G-30. Extending the yeast metabolic
network using a data integration approach<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745802 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">33</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745803" mce_href="#_Toc269745803">G-31. Predicting genetic
interactions by quantifying redundancy in biochemical pathways<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745803 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">34</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745804" mce_href="#_Toc269745804">G-32. Hub protein interfaces and hot
region organization<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745804 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">35</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745805" mce_href="#_Toc269745805">G-33. A functional-genomics
fermentation platform to identify and optimize industrial-relevant properties
of Lactobacillus plantarum<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745805 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">36</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745806" mce_href="#_Toc269745806">G-34. DisGeNET: a Cytoscape plugin
to visualize, integrate, search and analyze gene-disease networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745806 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">37</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745807" mce_href="#_Toc269745807">G-35. Combination of network topology
and pathway analysis to reveal functional modules in human disease<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745807 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">38</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745808" mce_href="#_Toc269745808">G-36. Attacking interface &amp;
interaction networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745808 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">40</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745809" mce_href="#_Toc269745809">G-37. Pathway Projector: web-based
zoomable pathway browser using KEGG Atlas and Google Maps API<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745809 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">41</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800300039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745810" mce_href="#_Toc269745810">G-38. Mapping the
"Farnesylome": structure-based prediction of farnesyltransferase
targets<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745810 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">42</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745811" mce_href="#_Toc269745811">G-39. Integrative analysis of gene
expression and copy number variation data to elucidate a reference human gene
network<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745811 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">43</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745812" mce_href="#_Toc269745812">G-40. Unknown player in modelling of
signal transduction pathway<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745812 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">44</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745813" mce_href="#_Toc269745813">G-41. Assembly of protein complexes
by integrating graph clusterings<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745813 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">45</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745814" mce_href="#_Toc269745814">G-42. How natural host species avoid
CD4+ T cell depletion during SIV infection<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745814 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">46</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745815" mce_href="#_Toc269745815">G-43. Kinase-specific
phosphorylation site prediction in Arabidopsis thaliana<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745815 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">47</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745816" mce_href="#_Toc269745816">G-44. BiologicalNetworks: enabling
systems-level studies of host-pathogen interactions<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745816 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">48</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745817" mce_href="#_Toc269745817">G-45. Genome-wide identification of
disease-related genes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745817 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">49</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745818" mce_href="#_Toc269745818">G-46. Bioinformatics tools for the
investigation of proteomics aspects involved in cardiovascular diseases: from
biomarker discovery to protein-protein interaction networks<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745818 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">50</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745819" mce_href="#_Toc269745819">G-47. ABC database: the Analysing
Biomolecular Contacts database<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745819 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">51</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745820" mce_href="#_Toc269745820">G-48. The core and pan metabolism in
the Escherichia coli species<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745820 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">52</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745821" mce_href="#_Toc269745821">G-49. BioGraph: discovering
biomedical relations by unsupervised hypothesis generation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745821 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">53</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745822" mce_href="#_Toc269745822">G-50. Tuning noise propagation in a
two-step series enzymatic cascade<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745822 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">54</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745823" mce_href="#_Toc269745823">G-51. The effect of interactome
evolution on network alignment<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745823 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">55</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745824" mce_href="#_Toc269745824">G-52. Analyzing compounds’ mode of
action by biological relatedness of proteins<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745824 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">56</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745825" mce_href="#_Toc269745825">G-53. Logical modelling of the
regulatory network controlling the formation of the egg appendages in
Drosophila<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745825 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">57</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745826" mce_href="#_Toc269745826">G-54. A curated database of microRNA
mediated feed-forward loops involving MYC as master regulator<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745826 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">58</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745827" mce_href="#_Toc269745827">G-55. The Biological Connection
Markup Language (BCML): a SBGN compliant data format for visualization,
filtering and analysis of biological pathways<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745827 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">59</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745828" mce_href="#_Toc269745828">G-56. Estimating the size of the
S.cerevisiae interactome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span></span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"> PAGEREF
_Toc269745828 \h </span><span style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">60</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745829" mce_href="#_Toc269745829">G-57. Laplacian eigenmaps, penalized
principal component regression on graphs, and analysis of biological pathways<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745829 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">61</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800320039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745830" mce_href="#_Toc269745830">G-58. Predicting metabolic pathways
from bacterial operons and regulons<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745830 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">62</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745831" mce_href="#_Toc269745831">G-59. Inferring translationally
active RNA binding proteins: mRNA interactions from polysomal profiling data
with a Bayesian inference approach<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745831 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">63</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745832" mce_href="#_Toc269745832">G-60. DASS-GUI: a new data mining
framework<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745832 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">64</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745833" mce_href="#_Toc269745833">G-61. EnrichNet: network-based gene
set enrichment analysis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span></span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"> PAGEREF
_Toc269745833 \h </span><span style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">65</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745834" mce_href="#_Toc269745834">G-62. Interacting copy number
alterations in breast cancer<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745834 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">66</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745835" mce_href="#_Toc269745835">G-63. Compression of mass spectral
imaging data using discrete wavelet transform guided by spatial information<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745835 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">67</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745836" mce_href="#_Toc269745836">G-64. Learning ancestral polytrees
for HIV-1 mutation pathways against nelfinavir<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745836 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">68</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745837" mce_href="#_Toc269745837">G-65. Ranking of genes from RNAi
perturbation data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745837 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">70</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745838" mce_href="#_Toc269745838">G-66. A network centered approach
for genome wide data integration applied to Alzheimers disease prediction<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745838 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">71</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745839" mce_href="#_Toc269745839">G-67. How to efficiently efficiently
hunt the disease causing gene<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745839 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">72</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800330039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745840" mce_href="#_Toc269745840">G-68. Study of the ultrasensitivity
of the BCL-2 apoptotic switch<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745840 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">73</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800340030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745841" mce_href="#_Toc269745841">G-69. Improved genome-wide
protein-protein interaction prediction and analysis of biological process
coordination in Escherichia coli<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745841 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">74</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800340031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745842" mce_href="#_Toc269745842">G-70. A generic gene regulatory
network reconstruction method: application to Lactococcus lactis MG1363<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745842 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">75</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800340032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="120-protein-interactions-molecular-networks-and-systems-biology.html#_Toc269745843" mce_href="#_Toc269745843">G-71. Dissecting the specificity of
E2-E3 interaction in the ubiquitination pathway by a molecular docking approach<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269745843 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">76</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340035003800340033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745773">G-1. Phenotypic effects of network rewiring in
regulatory hierarchies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bhardwaj N<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bhardwaj N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Kim PM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kim PM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4,5), Gerstein M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gerstein M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gerstein MB&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,6,7)</p>

<p class="abmotivation">Tinkering with transcriptional regulatory networks has
been used as a tool in the post-genome era to measure the tolerance of the cell
in response to different kinds of perturbations. This tampering may include
constructing synthetic model networks, knocking out and over-expression of
genes. To obtain further insights into the organizational architecture of these
networks, they have been rearranged into more intuitive structures like
pyramidal hierarchies. In this study, we study the phenotypic effects of
various kinds of network rewiring events in regulatory hierarchies in E. coli
and Yeast.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To study first order effects, we build intuitive
pyramidal hierarchies with the ‘chain-of-command’ pointing downward and
superimpose the phenotypic effects of tampering with nodes and edges related to
various levels of the hierarchy. We find that rewiring events that affect the
upper levels have more dramatic effect on cell growth than those affecting
lower level ones.</p>

<h3>Results</h3>

<p class="abresults">Extending it further, second order effects involve allowing
the hierarchies to change upon deletions and insertions of nodes/edges in the
network. To study these effects, we reconstruct modified hierarchies in
response to various perturbations that change the arrangement of genes in
different levels. From this analysis, we find that more than the absolute
number of changes in the altered hierarchies, it is the location and type of
change that more accurately reflects the phenotypic effect of rewiring; upper
level changes more adversely affect the cell fitness than lower level ones.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Since, connectivity of regulators is not linearly related
with its position in the hierarchy, these results show that approaches based on
the position of regulators in hierarchies can give better predictions about
their importance for cell fitness than those just based on their in- or
out-degrees.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Nitin Bhardwaj (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82249 = 'n&#105;t&#105;n.bh&#97;rdw&#97;j' + '&#64;';
 addy82249 = addy82249 + 'y&#97;l&#101;' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy82249 + suffix + '\'' + attribs + '>' );
 document.write( addy82249 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38649 = 'n&#105;t&#105;n.bh&#97;rdw&#97;j' + '&#64;';
 addy38649 = addy38649 + 'y&#97;l&#101;' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy38649 + suffix + '\'' + attribs + '>' );
 document.write( addy38649 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15488 = 'n&#105;t&#105;n.bh&#97;rdw&#97;j' + '&#64;';
 addy15488 = addy15488 + 'y&#97;l&#101;' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy15488 + suffix + '\'' + attribs + '>' );
 document.write( addy15488 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Molecular Biophysics and Biochemistry, Yale
University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Program in Computational Biology and Bioinformatics,
6 Department of Molecular Biophysics and Biochemistry, and 7 Department of
Computer Science, Yale University, Bass 432, 266 Whitney Avenue, New Haven, CT
06520. 2 Terrence Donnelly Centre for Cellular and Biomolecular Research, 3
Banting and Best Department of Medical Research, 4 Department of Molecular
Genetics, 5 Department of Computer Science, University of Toronto, Toronto, ON
M5S 3E1, Canada. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745774">G-2. Bringing order to disorder: comparative
genomics and genetic interactions uncover three biologically distinct forms of
protein disorder</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bellay J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bellay J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,=), Han S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Han S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,=), Michaut M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Michaut M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,=,*), Constanzo M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Constanzo M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Andrews BJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andrews BJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4), Boone C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Boone C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4), Bader GD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bader GD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4,5), Myers CL<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Myers CL&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kim PM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kim PM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,4,5)</p>

<p class="abmotivation">Intrinsically disordered regions are common in many
proteins, especially in higher eukaryotes. Intrinsically disordered proteins,
which have a large fraction of disordered residues, have been associated with a
large variety of functions and many diseases. However, a detailed understanding
of the role of disordered regions has remained elusive. In this study, we help
to shine some light on this by systematically distinguishing different types of
intrinsic disorder using genetic interactions and comparative genomics.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To investigate disordered proteins in the context
of genetic interactions, we used the most comprehensive genetic interaction
network for S. cerevisiae (Costanzo et al. 2010). To examine their evolutionary
properties we investigated which disordered regions were also disordered in
orthologous proteins across the yeast clade. We defined three distinct classes:
regions of conserved disorder with quickly evolving sequences (“flexible
disorder”), regions where disorder is conserved with highly constrained amino
acid sequence (“constrained disorder”) and, lastly, non-conserved disorder.</p>

<h3>Results</h3>

<p class="abresults">We observed that genes that have numerous genetic
interactions often tend to encode proteins that have a higher percentage of
disordered residues. We found that regions of conserved disorder are strongly
predictive for harboring linear motifs and phosphorylation sites. Flexible
disorder is closest to the canonical notion of protein disorder and is
responsible for its association with signaling pathways and
multi-functionality. Conversely, proteins high in constrained disorder are
involved in RNA binding and protein folding. Non-conserved disorder appears to
be largely non-functional sequence.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We thus conclude that by analyzing evolutionary
signatures of disordered regions we can distinguish three functionally distinct
subdivisions that also correspond to biophysically distinct phenomena.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Magali Michaut (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18760 = 'm&#97;g&#97;l&#105;.m&#105;ch&#97;&#117;t' + '&#64;';
 addy18760 = addy18760 + '&#117;t&#111;r&#111;nt&#111;' + '&#46;' + 'c&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy18760 + suffix + '\'' + attribs + '>' );
 document.write( addy18760 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56231 = 'm&#97;g&#97;l&#105;.m&#105;ch&#97;&#117;t' + '&#64;';
 addy56231 = addy56231 + '&#117;t&#111;r&#111;nt&#111;' + '&#46;' + 'c&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy56231 + suffix + '\'' + attribs + '>' );
 document.write( addy56231 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy85351 = 'm&#97;g&#97;l&#105;.m&#105;ch&#97;&#117;t' + '&#64;';
 addy85351 = addy85351 + '&#117;t&#111;r&#111;nt&#111;' + '&#46;' + 'c&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy85351 + suffix + '\'' + attribs + '>' );
 document.write( addy85351 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Toronto, CCBR</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Computer Science and Engineering,
University of Minnesota, Minneapolis, MN, USA (2) Terrence Donnelly Centre for
Cellular and Biomolecular Research (3) Banting and Best Department of Medical
Research (4) Department of Molecular Genetics (5) Department of Computer
Science University of Toronto, Toronto, ON M5S 3E1, Canada (=) These authors
contributed equally to this work</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745775">G-3. Prediction of genetic interactions in yeast
using machine learning</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schrynemackers M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schrynemackers M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Geurts P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Geurts P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Wehenkel L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wehenkel L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Madan Babu M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Madan Babu M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The inference of the genetic interaction network of an
organism is an important challenge in systems biology. The knowledge of these
interactions is very useful to understand the functions of the genes and their
products. In yeast S.cerevisiae, interactions subnetworks (E-MAPs) on four
subsets of genes have been measured. For the time being, it remains however
impossible to test experimentally the 18 millions potential interactions
between the 6000 genes. In this work, we propose to use computational
techniques based on machine learning to complete the experimentally confirmed
interactions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We proposed several strategies to transform this
problem into one or several standard classification problems and we exploited
two families of supervised learning algorithms: tree-based ensemble methods and
support vector machines. We considered as inputs various feature sets,
including chemo-genomic profiles, expression data, and morphological data. We validated
the approach by using cross-validation on four available E-MAPs. We
experimented with several protocols, including the completion of missing values
in a given E-MAP and the prediction of interactions in one E-MAP from the
others.</p>

<h3>Results</h3>

<p class="abresults">Globally, the best results are obtained with support vector
machines. Cross-validation shows that we are able to predict new interactions
with a reasonable accuracy. As expected, predictions of interactions between
genes from the training E-MAPs are more accurate than predictions of
interactions between genes not present in the training set. Some E-MAPs are
also much easier to predict than others. Among input feature sets, the
chemo-genomic profiles are the most predictive followed by the morphological
data, while we found that expression profiles are not informative.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have mostly focused on the prediction of negative
interactions. Positive interactions are less frequent, which renders their
prediction by machine learning techniques more challenging. We will now focus
on these interactions. Future work will also consider the addition of other
input features (e.g., interaction networks) or further methodological
developments. Our ultimate goal is to make genome-wide predictions with our
algorithms and to prioritise these predictions for an experimental validation.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Marie Schrynemackers (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46491 = 'M&#97;r&#105;&#101;.Schryn&#101;m&#97;ck&#101;rs' + '&#64;';
 addy46491 = addy46491 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy46491 + suffix + '\'' + attribs + '>' );
 document.write( addy46491 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1483 = 'M&#97;r&#105;&#101;.Schryn&#101;m&#97;ck&#101;rs' + '&#64;';
 addy1483 = addy1483 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy1483 + suffix + '\'' + attribs + '>' );
 document.write( addy1483 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41149 = 'M&#97;r&#105;&#101;.Schryn&#101;m&#97;ck&#101;rs' + '&#64;';
 addy41149 = addy41149 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy41149 + suffix + '\'' + attribs + '>' );
 document.write( addy41149 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of EE and CS &amp; GIGA-R, University of
Liège, Belgium</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of EE and CS &amp; GIGA-R, University of
Liège, Belgium (2) MRC Laboratory of Molecular Biology, Cambridge, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This poster presents research results of the Belgian
Network BIOMAGNET (Bioinformatics and Modeling: from Genomes to Networks),
funded by the Interuniversity Attraction Poles Programme, initiated by the
Belgian State, Science Policy Office. MS is recipient of a F.R.I.A. fellowship
and PG is a Research Associate of the F.R.S.-FNRS. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745776">G-4. Prioritizing candidate genes by network
analysis of differential expression using machine learning approaches</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nitsch D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nitsch D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Discovering novel disease genes is still challenging for
diseases for which no prior knowledge - such as known disease genes or
disease-related pathways - is available. Performing genetic studies frequently
result in large lists of candidate genes of which only few can be followed up
for further investigation. We have recently developed a computational method
that identifies the most promising candidate genes by replacing prior knowledge
by experimental data of differential gene expression between affected and
healthy individuals (Nitsch et al. 2009, PlosOne 4(5): e5526).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To improve the performance of our prioritization strategy,
we have extended our previous work by applying different machine learning
approaches that identify promising candidate genes by determining whether a
gene is surrounded by highly differentially expressed genes in a functional
association or protein-protein interaction network. Our novel ranking
strategies, scoring disease candidate genes, rely on network-based machine
learning approaches, such as kernel ridge regression, heat kernel, Arnoldi
kernel approximation, and a local measure based on a direct neighborhood
measure.</p>

<h3>Results</h3>

<p class="abresults">We have used a standard procedure in genetics that ranks
candidate genes based solely on their differential expression level as a
baseline. Our results showed that our four strategies could outperform this
standard procedure and that the best results were obtained using the heat
kernel approach leading to an average ranking position of 8 out of 100 genes,
an AUC value of 92.3% and an error reduction of 52.8% relative to a standard
genetic procedure which ranked the knockout gene in average at position 17 with
an AUC value of 83.7%.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In this study we could identify promising candidate genes
using network-based machine learning approaches even if no knowledge is
available about the disease or phenotype.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Daniela Nitsch (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1270 = 'D&#97;n&#105;&#101;l&#97;.N&#105;tsch' + '&#64;';
 addy1270 = addy1270 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy1270 + suffix + '\'' + attribs + '>' );
 document.write( addy1270 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21510 = 'D&#97;n&#105;&#101;l&#97;.N&#105;tsch' + '&#64;';
 addy21510 = addy21510 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy21510 + suffix + '\'' + attribs + '>' );
 document.write( addy21510 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67626 = 'D&#97;n&#105;&#101;l&#97;.N&#105;tsch' + '&#64;';
 addy67626 = addy67626 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy67626 + suffix + '\'' + attribs + '>' );
 document.write( addy67626 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">K.U.Leuven, ESAT-SCD</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) K.U.Leuven, ESAT-SCD</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745777">G-5. The human E3 ubiquitin ligase enzyme protein
interaction network</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kar G<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Kar G&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*), Keskin
O<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE &quot;Keskin
O&quot; <![endif]--><!--[if supportFields]><span style="mso-element:field-end" mce_style="mso-element:field-end"></span><![endif]--><span style="">&nbsp;</span>(1), Nussinov R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nussinov R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Gursoy A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gursoy A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Ubiquitination is crucial for protein degradation in
eukaryotic cells. It is achieved by a sequential cascade of
ubiquitin-activating (E1), ubiquitin-conjugating (E2) and ubiquitin-ligating
(E3) enzymes. E3 ligases mediate ubiquitin transfer from E2s to substrates and
as such confer substrate specificity. Despite their essential role, current
knowledge of their distinct biological functions and interaction partners is
limited.Here, using structural data, efficient structural comparison algorithms
and appropriate filters,we construct human E3 ubiquitin ligase enzyme protein
interaction network.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We first compile the available structures for E2
and E3 proteins in the human ubiquitination pathway. Second, we apply our
efficient protein-protein interaction prediction algorithm PRISM, which uses
experimental (X-ray, NMR) protein-protein interface templates to model the
interactions of E3 and E2 proteins in a large, proteome-scale docking strategy based
on interface structural motifs. Then, we include flexibility and energetic
considerations in our modeling using FiberDock, a flexible docking refinement
server, to obtain more physical and biologically relevant interactions.</p>

<h3>Results</h3>

<p class="abresults">Analysis of the human E3 ubiquitin ligase enzyme protein
interaction network reveals important functional features and uncovers an a
priori unknown E3-E2 and E3-E3 interactions. Our results show that E3 proteins
such as Mdm2 and Huwe1 share E2 partners, which may explain how both Mdm2 and
Huwe1 ubiquitinate p53 tumor suppressor protein for degradation. In addition,
we discover the mode of E3-E3 interactions such as Mdm2-Siah1, which are known
to enhance the degradation of the Numb protein.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Here, for the first time, we constructed a structural
human E3 ubiquitin ligase enzyme protein interaction network. Our strategy
allows elucidation of both which E3s interact with which E2s in the human
ubiquitination pathway and how they interact. In addition to identifying E3-E2
interactions, our strategy also reveals functionally-relevant E3-E3
interactions in the human ubiquitination pathway that were hitherto unknown.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Gozde Kar (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78074 = 'gk&#97;r' + '&#64;';
 addy78074 = addy78074 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy78074 + suffix + '\'' + attribs + '>' );
 document.write( addy78074 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11797 = 'gk&#97;r' + '&#64;';
 addy11797 = addy11797 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy11797 + suffix + '\'' + attribs + '>' );
 document.write( addy11797 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39626 = 'gk&#97;r' + '&#64;';
 addy39626 = addy39626 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy39626 + suffix + '\'' + attribs + '>' );
 document.write( addy39626 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Koc University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Koc University, Center for Computational Biology
and Bioinformatics, and College of Engineering, Rumelifeneri Yolu, 34450
Sariyer Istanbul, Turkey (2) Basic Research Program, SAIC-Frederick, Inc.,
Center for Cancer Research Nanobiology Program, NCI-Frederick, Frederick, MD 21702,
USA (3) Sackler Inst. of Molecular Medicine, Department of Human Genetics and
Molecular Medicine, Sackler School of Medicine, Tel Aviv University, Tel Aviv
69978, Israel</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745778">G-6. Fast motif enumeration and clustering in integrated
networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Audenaert P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Audenaert P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Van Parys T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Parys T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Pickavet M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pickavet M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Demeester P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Demeester P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Michoel T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Michoel T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3)</p>

<p class="abmotivation">Network motifs, small frequently occuring subgraphs, are
the basic building blocks of complex networks of interactions between DNA, RNA,
proteins and metabolites. Network motifs aggregate into larger, self-contained
units reflecting the organization of a cell into discrete modules carrying out
specific biological functions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">All algorithms were validated on an integrated
network of more than 50,000 transcriptional regulatory, posttranslational
regulatory, and physical protein-protein interactions in the yeast S.
cerevisiae. Our alorithms were developed in a Java environment, running on
stock hardware (2Gb of RAM and a dual core 2.4GHz CPU).</p>

<h3>Results</h3>

<p class="abresults">Enumerating all motif instances is computationally
intensive. We achieved a significant speedup by optimized branch-pruning
allowing us to enumerate larger motifs than was previously possible. Motif
clusters are defined as sets of nodes with a high number of motif instances
between them, relative to their size. To find high-scoring clusters, we first
defined PageRank-like cluster membership weights. Optimal weights are found by
solving a multilinear set of equations and converted to motif clusters by
taking a provably optimal weight cutoff. User interfaces are provided through
Cytoscape.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed novel algorithms and a Cytoscape user
interface for identifying clusters of overlapping network motifs in large-scale
networks composed of multiple interaction types spanning all levels of regulation
in a cell.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pieter Audenaert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3246 = 'P&#105;&#101;t&#101;r.A&#117;d&#101;n&#97;&#101;rt' + '&#64;';
 addy3246 = addy3246 + '&#105;nt&#101;c' + '&#46;' + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy3246 + suffix + '\'' + attribs + '>' );
 document.write( addy3246 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7723 = 'P&#105;&#101;t&#101;r.A&#117;d&#101;n&#97;&#101;rt' + '&#64;';
 addy7723 = addy7723 + '&#105;nt&#101;c' + '&#46;' + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy7723 + suffix + '\'' + attribs + '>' );
 document.write( addy7723 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42389 = 'P&#105;&#101;t&#101;r.A&#117;d&#101;n&#97;&#101;rt' + '&#64;';
 addy42389 = addy42389 + '&#105;nt&#101;c' + '&#46;' + '&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy42389 + suffix + '\'' + attribs + '>' );
 document.write( addy42389 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universiteit Gent</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Ghent University, Faculty of Engineering, Dept. of
Information Technology (2) VIB, Dept. of Plant Systems Biology (3) Ghent
University, Faculty of Sciences, Dept. of Plant Biotechnology and Genetics</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge funding of the IWT (SBO-BioFrame), IUAP
P6/25 (BioMagnet), Ghent University (Multidisciplinary Research Partnership
"Bioinformatics: from nucleotides to networks") and IBBT. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745779">G-7. Dynamic Deterministic Effects Propagation
Networks: learning signalling pathways from longitudinal protein array data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bender C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bender C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Henjes F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Henjes F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Fröhlich H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fröhlich H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Wiemann S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wiemann S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Korf U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Korf U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Beißbarth T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beißbarth T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3)</p>

<p class="abmotivation">Network modelling has become an important tool to study
cancer related molecular interactions, which is the basis for the development
of novel drugs and therapies. We present a new signalling network
reconstruction method using longitudinal protein array data after external
perturbation, measured on Reverse Phase Protein Arrays, and infer a network
among ERBB-signalling related proteins in a human breast cancer cell line.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our method models the signalling dynamics by a
boolean signal propagation mechanism that defines a sequence of state
transitions for a given network structure. A likelihood score is proposed that
describes the probability of our measurements given a particular state
transition matrix. We identify the optimal sequence of state transitions via a
Hidden Markov Model. Network structure search is performed by a genetic
algorithm that optimises the overall likelihood of a population of candidate
networks.</p>

<h3>Results</h3>

<p class="abresults">We test our method on simulated networks and data and show
its increased performance in comparison to other Dynamical Bayesian Network
approaches. The reconstruction of a network in our real data results in several
known signalling chains from the ERBB network, showing the validity and
usefulness of our approach.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our method was able to reconstruct several signalling
chains from the ERBB network known in the literature. Due to restrictions in
the data, all edges must be carefully assessed and validated to proof their
existence. Given that a major part of the inferred edges in our network are
described in previous publications, it is a valuable tool for the generation of
hypotheses on the underlying biological signalling processes in the biological
system which was studied.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.dkfz.de/mga2/ddepn" mce_href="http://www.dkfz.de/mga2/ddepn">http://www.dkfz.de/mga2/ddepn</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Christian Bender (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16210 = 'c.b&#101;nd&#101;r' + '&#64;';
 addy16210 = addy16210 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy16210 + suffix + '\'' + attribs + '>' );
 document.write( addy16210 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63003 = 'c.b&#101;nd&#101;r' + '&#64;';
 addy63003 = addy63003 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy63003 + suffix + '\'' + attribs + '>' );
 document.write( addy63003 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84727 = 'c.b&#101;nd&#101;r' + '&#64;';
 addy84727 = addy84727 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy84727 + suffix + '\'' + attribs + '>' );
 document.write( addy84727 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">German Cancer Research Center</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) German Cancer Research Center, Department of
Molecular Genome Analysis, 69120 Heidelberg, Germany (2) Bonn-Aachen
International Center for IT, Department of Algorithmic Bioinformatics, 53113
Bonn, Germany (3) University of Göttingen, Department of Medical Statistics,
37099 Göttingen, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project was supported through the Helmholtz
Alliance on Systems Biology network SB-Cancer, through the German Federal
Ministry of Education and Science (BMBF) project BreastSys in the platform
Medical Systems Biology and the network IG Cellular Systems Genomics (01GS0864)
in the platform NGFNplus. Further by the Clinical Research Group 179 through
the DFG. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745780">G-8. Systematic mapping of multiple specificity in
peptide recognition modules reveals new binding modes of protein domains</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gfeller D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gfeller D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Ernst A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ernst A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Verschueren E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Verschueren E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vanhee P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vanhee P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Dar N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dar N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Serrano L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Serrano L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Sidhu SS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sidhu SS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bader GD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bader GD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kim PM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kim PM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Accurately modeling binding specificity is a promising
way to understand the properties of protein interaction domains such as SH3 or
PDZ domains and predict new protein interactions, which can then lead to better
understanding of signaling pathways. Current computational models in general
disregard correlations between ligand residues. Using phage display results, we
observe that this assumption is unrealistic. We develop a new model that
provides a more detailed view of binding specificity and enable uncovering
multiple specificity in protein interaction domains.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Experimental datasets: Phage display experiments
for PDZ and SH3 domains (Tonikian et al, 2008, Tonikian et al 2009), as well as
next-generation sequencing phage display data for Erbin PDZ mutants. Computational
tools: Correlations between residues within an alignment of peptide interacting
with a domain are measured using Mutual Information. Multiple specificity was
detected by clustering the peptides and running a Maximum-Likelihood algorithm
based on mixture models.</p>

<h3>Results</h3>

<p class="abresults">We find that peptides interacting with a domain often
exhibited highly significant correlations among their residues, which can be
accurately modeled by considering different specificities for a domain. Our
results reveal widespread occurrence of multiple specificity and enable us to
identify structurally distinct binding modes. In particular, we find a new
binding mode of PDZ domains, where an additional amino acid can be accommodated
at the C-terminus of the ligand. This non-canonical binding mode predicts new
and unexpected protein interactions.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Identifying multiple specificity in peptide recognition
modules both yields more accurate protein interaction predictions and provides
new structural insights. While this kind of approaches have often been hampered
by the limited number of experimental data, new sequencing technologies,
combined with techniques such as phage display, are revolutionizing our
capacities to experimentally detect protein interactions. Efficient
computational tools, such as the one presented here, will be crucial to analyze
these data and accurately model the binding specificity of protein interaction
domains.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">David Gfeller (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17348 = 'd&#97;v&#105;d.gf&#101;ll&#101;r' + '&#64;';
 addy17348 = addy17348 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy17348 + suffix + '\'' + attribs + '>' );
 document.write( addy17348 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86364 = 'd&#97;v&#105;d.gf&#101;ll&#101;r' + '&#64;';
 addy86364 = addy86364 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy86364 + suffix + '\'' + attribs + '>' );
 document.write( addy86364 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45414 = 'd&#97;v&#105;d.gf&#101;ll&#101;r' + '&#64;';
 addy45414 = addy45414 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy45414 + suffix + '\'' + attribs + '>' );
 document.write( addy45414 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Swiss Bioinformatics Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Terrence Donnelly Centre for Cellular and
Biomolecular Research, University of Toronto, Toronto, Ontario M5S 3E1, Canada (2)
EMBL-CRG Systems Biology Unit, CRG-Centre de Regulacio Genomica, Dr. Aiguader
88, 08003 Barcelona, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Canadian Institute of Health Research Swiss National
Science Foundation</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745781">G-9. Reverse-engineering gene regulatory networks
for the abiotic stress response in Arabidopsis thaliana</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vermeirssen V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vermeirssen V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), De Clercq I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Clercq I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Van Parys T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Parys T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Van Breusegem F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Breusegem F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">In order to understand how the plant responds to its
changing environment, we study gene regulatory networks under stress conditions
in Arabidopsis thaliana.Gene regulatory networks describe gene expression as a
function of regulatory inputs specified by interactions between regulators and
DNA. Often these data are not available yet on a genome-wide scale and need to
be predicted from more abundant transcription expression profiles. This is
often a high-dimensional, underdetermined problem, further complicated by noise
and the indirect regulator-gene relations present in the data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this study we used a probabilistic model to
“reverse engineer” microarray expression profile data into biologically
relevant gene regulatory networks. LeMoNe (learning module networks)uses gene
expression profiles to extract ensemble gene regulatory networks of
coexpression modules and their prioritized regulators. Through post-processing
data integration we demonstrate the value of our method.</p>

<h3>Results</h3>

<p class="abresults">We compiled gene expression profiles for 283 stress
conditions in Arabidopsis thaliana and inferred through LeMoNe the stress gene
regulatory networks. Through GO and AraCyc enrichment analysis, comparison with
the gene-gene association network AraNet, cis-regulatory motif analysis and
integration of other biological data, we show that LeMoNe identifies
functionally coherent coexpression modules and prioritizes regulators that
relate to similar biological processes as the module genes. Furthermore, we
predict functional relationships for uncharacterized genes and regulators.</p>

<h3>Discussion</h3>

<p class="abdiscussion">LeMoNe is a highly qualified reverse-engineering
algorithm for generating hypotheses on differential gene expression. In an era
of functional genomics data abundance, it provides a means to put forward
selected follow-up experiments for experimental validation. The knowledge
gained in this systems biology study might lead to the identification of genes
that are potential candidates for innovative molecular breeding strategies to
develop stress-tolerant crops.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Vanessa Vermeirssen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14972 = 'v&#97;n&#101;ss&#97;.v&#101;rm&#101;&#105;rss&#101;n' + '&#64;';
 addy14972 = addy14972 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy14972 + suffix + '\'' + attribs + '>' );
 document.write( addy14972 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2724 = 'v&#97;n&#101;ss&#97;.v&#101;rm&#101;&#105;rss&#101;n' + '&#64;';
 addy2724 = addy2724 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy2724 + suffix + '\'' + attribs + '>' );
 document.write( addy2724 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79952 = 'v&#97;n&#101;ss&#97;.v&#101;rm&#101;&#105;rss&#101;n' + '&#64;';
 addy79952 = addy79952 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy79952 + suffix + '\'' + attribs + '>' );
 document.write( addy79952 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB-UGent</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Plant Systems Biology, VIB, Ghent
University, Technologiepark 927, 9052 Gent, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745782">G-10. Predicting protein-protein interaction using
mirror tree</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Esmaielbeiki R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Esmaielbeiki R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*) , Nebel J-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nebel J-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Protein-protein interactions (PPI) are important for
every process that takes place within a living cell. Although many experimental
techniques can estimate PPI, their cost and inaccuracy have led to the
development of computational methods. Among them, the popular Mirror Tree (MT)
is based on the intuitive assumption that proteins which interact are under
co-evolution pressure. Latest improvements attempt to remove the species
evolution signal (Tree of Life) from co-evolution measurements. Since many
enhancements of the MT have been proposed a comparative study is of great
interest.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">MT relies on measuring co-evolution between two
protein chains. This is achieved by comparing their phylogenetic trees which
are built from the multiple alignment of sequence homologues. We propose to use
mutual information to quantify their similarity. Experiments were conducted
using a dataset containing 179 E.coli proteins where 281 pairs are known to
interact. For each protein, 65 orthologues were extracted to produce multiple
alignments using ClustalW. Quantitative comparison is presented for recent MT
implementations including ours.</p>

<h3>Results</h3>

<p class="abresults">Evaluation of the different methods shows that usually
removing Tree of Life (TOL) information produces more accurate results.
Although the MI metric generates high sensitivity, its specificity is very low.
Finally, our ClustalW based MTs outperform all other implementations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">As a whole, our TOL-TM achieves the best compromise
between sensitivity, specificity, and AUROC. Results suggest that ClustalW
produces more accurate phylogenetic trees and Mutual Information is not a
better measure for the comparison of these trees. In the future, we plan to
extract Tree of Life information using advanced models from information theory
and attempt to detect the specific amino acids which are involved in the protein-protein
interaction.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Reyhaneh Esmaielbeiki (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96212 = 'k0854421' + '&#64;';
 addy96212 = addy96212 + 'k&#105;ngst&#111;n' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy96212 + suffix + '\'' + attribs + '>' );
 document.write( addy96212 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy70324 = 'k0854421' + '&#64;';
 addy70324 = addy70324 + 'k&#105;ngst&#111;n' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy70324 + suffix + '\'' + attribs + '>' );
 document.write( addy70324 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15434 = 'k0854421' + '&#64;';
 addy15434 = addy15434 + 'k&#105;ngst&#111;n' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy15434 + suffix + '\'' + attribs + '>' );
 document.write( addy15434 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Kingston University London</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Kingston University London</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745783">G-11. Comparison of Bayesian networks and its
extensions applied to the inference of regulatory networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Werhli A V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Werhli A V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*)</p>

<p class="abmotivation">The rapid increase in the availability and diversity of
molecular biology data has instigated the investigation of the viability of learning
biological network's structure from these data. Among the various methods
available for the inference of the network's structure, Bayesian networks (BNs)
are very attractive due to its flexibility and probabilistic nature. For
instance, BNs permit interventional experiments and/or extra knowledge from
diverse data sources to be properly included in the inference. The main aim of
this work is to compare how these approaches compare with each other.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The main aim is to devise a BN structure from a set
of data. In the first setting a simple BN is applied. The second investigated
approach explicitly uses a set of interventions in the data, BNi. In the third
method an extra source of prior knowledge is included in the inference through
an hierarchical Bayesian model, BNe. In all the approaches the networks are sampled
with a Markov Chain Monte Carlo. For comparison purposes both synthetic data
and real data (obtained from flow cytometry experiments) are employed. The
network investigated is the raf signalling pathway Sachs et al. 2005.</p>

<h3>Results</h3>

<p class="abresults">In order to compare the methods the area under the ROC curve
(AUC) is computed with two distinct settings. In one setting the directions are
considered (DGE) an in the other they are all discarded (UGE). For all types of
data, namely Gaussian, Netbuilder and flow cytometry, the BNi and BNe approaches
significantly outperform the simple BN when considering the DGE metric. When
the directions are discarded there is no significant differences among the
methods in simulated data but there is a difference in flow cytometry data. The
results are obtained from 5 different data sets.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The methods BNi and BNe clearly outperform the simple BN
specially when considering the edge directions, DGE. This suggests that the
inclusion of extra-knowledge or interventions breaks up symmetries and resolve
the direction of some edges that can not be resolved only on basis of the data
due to the existence of equivalence classes. Therefore the use of more refined
BN methods i.e. BNi and BNe is completely justified. As future research we plan
to combine the addition of interventions and the extra knowledge and also test
the methods in different networks' structures.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Adriano V. Werhli (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13490 = 'w&#101;rhl&#105;' + '&#64;';
 addy13490 = addy13490 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy13490 + suffix + '\'' + attribs + '>' );
 document.write( addy13490 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52573 = 'w&#101;rhl&#105;' + '&#64;';
 addy52573 = addy52573 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy52573 + suffix + '\'' + attribs + '>' );
 document.write( addy52573 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54083 = 'w&#101;rhl&#105;' + '&#64;';
 addy54083 = addy54083 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy54083 + suffix + '\'' + attribs + '>' );
 document.write( addy54083 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universidade federal do Rio Grande</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Centro de Ciências Computacionais, Universidade
Federal do Rio Grande</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project is supported by Conselho Nacional de
Desenvolvimento Científico e Tecnológico, CNPQ. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745784">G-12. Insight into the mechanism of specific
regulation of STAT proteins by atypical dual-specificity phosphatases</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Jardin C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Jardin C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sticht H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sticht H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Protein phosphorylation at tyrosine residues is regulated
by the coordinated activity of protein tyrosine kinases and tyrosine
phosphatases. Dephosphorylation of Signal Transducer and Activator of
Transcription (STAT) proteins by the atypical dual-specificity protein-tyrosine
phosphatases (DSP) is highly specific. To date however, there is no
experimental data that explain this specificity. Our goal was to get an
explanation at the molecular level of the mechanism that regulates the
phosphatase activity of these DSPs toward specific STATs using computational
methods.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">An approach combining docking and molecular dynamic
(MD) simulations was used to investigate the binding between the PTP domain of
the phosphatases and the SH2 domain of STATs, starting from the crystal
structures. Docking conformations between VHR and STAT5 and between VH1 and
STAT1 were generated systematically. For those docking solutions that agree
with the mutagenesis data MD simulations were done. The interactions at the
binding interfaces were compared in sequence alignments of the STAT aminoacid
sequences to understand the origins of the specificity.</p>

<h3>Results</h3>

<p class="abresults">In agreement with the mutagenesis data, we found a
conformation for VHR interacting with STAT5 that involves a phosphorylated
tyrosine of VHR, located apart from the catalytic site, and an arginine of the
STAT5 SH2 domain that is conserved in all STAT members. A similar binding site
was found for VH1 with STAT1. The binding interfaces are stable in MD
simulations and present an intensive network of interactions between the
phosphatases and STATs. The analysis of this network reveals interactions that
are specific of each DSP-STAT complex.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our simulations reveal that the site of dephosphorylation
at the phosphatase PTP domain is, at least alone, not enough to explain the
specificity. The PTP domain exhibits another site, located apart from the
catalytic one, which is responsible both for the binding to STAT SH2 domains
and for the specificity. Thus, atypical DSP phosphatases possess both the
catalytic and recognition sites in the same polypeptide chain that forms the
PTP domain. We thus suggest that the two sites play complementary roles in a
two-sites/two-steps mechanism for the phosphatase activity of atypical DSPs on
STATs.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Christophe Jardin (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28977 = 'Chr&#105;st&#111;ph&#101;.J&#97;rd&#105;n' + '&#64;';
 addy28977 = addy28977 + 'b&#105;&#111;ch&#101;m' + '&#46;' + '&#117;n&#105;-&#101;rl&#97;ng&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy28977 + suffix + '\'' + attribs + '>' );
 document.write( addy28977 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71332 = 'Chr&#105;st&#111;ph&#101;.J&#97;rd&#105;n' + '&#64;';
 addy71332 = addy71332 + 'b&#105;&#111;ch&#101;m' + '&#46;' + '&#117;n&#105;-&#101;rl&#97;ng&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy71332 + suffix + '\'' + attribs + '>' );
 document.write( addy71332 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10313 = 'Chr&#105;st&#111;ph&#101;.J&#97;rd&#105;n' + '&#64;';
 addy10313 = addy10313 + 'b&#105;&#111;ch&#101;m' + '&#46;' + '&#117;n&#105;-&#101;rl&#97;ng&#101;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy10313 + suffix + '\'' + attribs + '>' );
 document.write( addy10313 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioinformatics, Institute for Biochemistry,
University of Erlangen-Nuremberg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Bionformatics Institute for Biochemistry Emil-Fischer
Center University of Erlangen-Nuremberg</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">DFG Sonderforschungsbereich 796 Rechenzentrum Erlangen
(RRZE) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745785">G-13. VoteDock: The consensus docking method for
prediction of protein-ligand interactions</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Plewczynski D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Plewczynski D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Łaźniewski M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Łaźniewski M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), von Grotthuss M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;von Grotthuss M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Rychlewski L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rychlewski L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Ginalski K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ginalski K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Molecular recognition plays a fundamental role in all
biological processes that is why great efforts have been made to understand and
predict protein-ligand interactions. Finding a molecule that can potentially
bind to a target protein is particularly essential in drug discovery but
experimental techniques are still expensive and time-consuming. Thus in silico
tools are frequently used to screen molecular libraries in order to identify
new lead compounds [Lee2009]. If information about protein structure is known,
also various protein-ligand docking programs can be used [Perola2004][Li2010].</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The aim of docking procedure is to predict correct
poses of ligand in the binding site of the protein, as well as to score them
according to the strength of interaction in a reasonable time frame. The
purpose of our studies was to present the novel consensus approach to predict
both protein-ligand complex structure and its binding affinity. Our method,
called VoteDock, uses as the input the results from seven docking programs
(Surflex, LigandFit, Glide, GOLD, FlexX, eHiTS and AutoDock) that are widely
used by community, as most of them are part of popular modeling software
packages.</p>

<h3>Results</h3>

<p class="abresults">Those programs were evaluated on the extensive dataset of
1300 protein-ligands pairs from refined set of PDBbind database [Wang2004], for
which the structural and binding affinity data are available. We compared
independently ability of proper scoring by calculating Pearson correlation
between docking score and experimental binding affinities, and posing by
measuring RMSD of obtained conformations with native structure. That procedure
allows us to compare performance of individual programs with that of VoteDock.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In most cases our consensus-based method was able to dock
properly about 20% of pairs more than docking methods in average, and more than
10 % of pairs more than single best program. Also drop in RMSD of top scored
conformation can be observed, with value 0.5Å lower than that of best
individual program, namely GOLD. Similar increase in overall docking accuracy
can be also observed for subsets created based on PDBbind, that explore various
physico-chemical properties of ligands, their size or hydrophobic potential.
Finally, we are able to boost the Pearson correlation of the predicted binding
a</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://dock.bioinfo.pl" mce_href="http://dock.bioinfo.pl">http://dock.bioinfo.pl</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Dariusz Plewczynski (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14327 = 'd&#97;rm&#97;n' + '&#64;';
 addy14327 = addy14327 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy14327 + suffix + '\'' + attribs + '>' );
 document.write( addy14327 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17823 = 'd&#97;rm&#97;n' + '&#64;';
 addy17823 = addy17823 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy17823 + suffix + '\'' + attribs + '>' );
 document.write( addy17823 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11795 = 'd&#97;rm&#97;n' + '&#64;';
 addy11795 = addy11795 + '&#105;cm' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy11795 + suffix + '\'' + attribs + '>' );
 document.write( addy11795 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Interdyscyplinary Centre for Mathematical and
Computational Modelling, University of Warsaw</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Interdisciplinary Centre for Mathemtaical and
Computetiobal Modelling, University of Warsaw, Pawińskiego 5a Street, 02-106
Warsaw, Poland, (2) Department of Chemistry and Chemical Biology, Harvard
University, 12 Oxford Street, Cambridge, Massachusetts 02138, USA, (3)
BioInfoBank Institute, Poznan, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Calculations were performed at the Interdisciplinary
Center for Mathematical and Computational Modelling. This work was supported by
Polish Ministry of Science and Higher Education N301 159735 grant. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745786">G-14. Inferring genetic regulatory networks with a
hierarchical Bayesian model and a parallel sampling algorithm</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Mendoza M R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Mendoza M R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Werhli A V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Werhli A V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,*)</p>

<p class="abmotivation">Bayesian Networks (BNs) are a particularly promising tool
for inferring the structure of regulatory networks. It is usually assumed that
a molecular biological system may be represented by a single regulatory
network. In a previous work a method for reconstructing the regulatory structure
of a network considering that its active parts can differ under different
experimental conditions was proposed. Unfortunately the proposed method when
sampled with a MCMC suffered problems of mixing and convergence. In the present
work we propose a Parallel Sampling Algorithm to overcome this problem.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In order to integrate information from various
different data sets obtained under distinct experimental conditions we use an hierarchical
Bayesian model. The model is composed of data sets, a hypernetwork which is not
directly associated with the data but encourages the graphs to be similar and
the hyperparameters which associate each data set with the hypernetwork. All
components of the model are sampled with a Metropolis Coupled Markov Chain
Monte Carlo which involves the parallel execution of multiple Markov chains,
some of which are heated, and a state swap proposal between chains.</p>

<h3>Results</h3>

<p class="abresults">The method was tested using data simulated from a linear
Gaussian distribution and with Netbuilder tool, as well as with real data collected
in cytometry flow experiments. The approach was applied in the integration of
five data sets: one composed of totally random values, representing an
unsuccessful experiment, and four others of the same nature, either simulated or
measured data. For synthetic data the method correctly identified the random
data and properly converged. It also outperformed other options of using the
various data sets. The convergence was not observed in real data though.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The method was tested using data simulated from a linear
Gaussian distribution and with Netbuilder tool, as well as with real data collected
in cytometry flow experiments. The approach was applied in the integration of
five data sets: one composed of totally random values, representing an
unsuccessful experiment, and four others of the same nature, either simulated
or measured data. For synthetic data the method correctly identified the random
data and properly converged. It also outperformed other options of using the
various data sets. The convergence was not observed in real data though.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Adriano V. Werhli (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55476 = 'w&#101;rhl&#105;' + '&#64;';
 addy55476 = addy55476 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy55476 + suffix + '\'' + attribs + '>' );
 document.write( addy55476 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19093 = 'w&#101;rhl&#105;' + '&#64;';
 addy19093 = addy19093 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy19093 + suffix + '\'' + attribs + '>' );
 document.write( addy19093 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33304 = 'w&#101;rhl&#105;' + '&#64;';
 addy33304 = addy33304 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy33304 + suffix + '\'' + attribs + '>' );
 document.write( addy33304 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universidade federal do Rio Grande</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Instituto de Informática, Universidade Federal do
Rio Grande do Sul. (2) Centro de Ciências Computacionais, Universidade Federal
do Rio Grande.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is supported by Conselho Nacional de
Desenvolvimento Científico e Tecnológico (CNPQ). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745787">G-15. Prediction of protein-protein interactions in
the apoptosis pathway</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Acuner Ozbabacan SE<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Acuner Ozbabacan SE&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Gursoy A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gursoy A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Keskin O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Keskin O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Nussinov R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nussinov R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3)</p>

<p class="abmotivation">Prediction methods of protein-protein interactions have
been intensively developed by scientists for the past years. From the
structural point of view, proteins interact through their interfaces and the
structure of these regions play crucial roles in the determination of
interaction affinities. Our group constructed a server named PRISM (PRotein
Interactions by Structural Matching), which can predict the protein-protein
interactions in a given target set by structurally matching the target
structures with a template interface set. In this work, the aim is to predict
new protein-protein interactions in the apoptosis pathway by using PRISM server
and the default template set of this server.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The template set includes the interfaces obtained
from the complexes that are known to interact. It was prepared by collecting
all of the interfaces in the PDB (in 2006) and clustering them structurally.
The homologous and nonbiological interfaces in the clusters were eliminated and
158 interfaces were left as the nonredundant representative template set. The
target set is composed of the proteins/genes in the apoptosis pathway with available
nonhomologous PDB structures. Protein-protein interactions, which are thought
to complement the apoptosis pathway, are predicted and corresponding
interaction energies are calculated by using the PRISM server and FiberDock
server, respectively.</p>

<h3>Results</h3>

<p class="abresults">Many interactions were predicted by PRISM but we
concentrated only on around 2000 interactions, which are redundant and contain
interactions that are already in the apoptosis pathway, having energy values
lower than -10 kcal/mol. One of the lowest interaction energies, therefore one
of the most favorable interactions, was found for CASP7-CASP8 interaction,
which is already in the apoptosis pathway. These interactions were found by
using 85 template interfaces out of 158, for structural matching.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The prediction algorithm of PRISM server is based on the
structural resemblance of the potential pairs in the target set to the
templates which are interfaces of the interacting pairs (complexes) taken from
the PDB. Hence, in addition to the prediction and analysis of candidate new
interactions in the apoptosis pathway, a structural dimension has been
introduced since the interactions are found by structural matching.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ece S. Acuner Ozbabacan (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23101 = 's&#111;zb&#97;b&#97;c&#97;n' + '&#64;';
 addy23101 = addy23101 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy23101 + suffix + '\'' + attribs + '>' );
 document.write( addy23101 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97166 = 's&#111;zb&#97;b&#97;c&#97;n' + '&#64;';
 addy97166 = addy97166 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy97166 + suffix + '\'' + attribs + '>' );
 document.write( addy97166 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45101 = 's&#111;zb&#97;b&#97;c&#97;n' + '&#64;';
 addy45101 = addy45101 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy45101 + suffix + '\'' + attribs + '>' );
 document.write( addy45101 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Koc University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Center for Computational Biology and Bioinformatics
and College of Engineering, Koc University Rumelifeneri Yolu, 34450 Sariyer
Istanbul, Turkey (2)Basic Science Program, SAIC-Frederick, Inc. Center for
Cancer Research Nanobiology Program NCI-Frederick, Frederick, MD 21702 (3)Sackler
Inst. of Molecular Medicine Department of Human Genetics and Molecular
Medicine, Sackler School of Medicine, Tel Aviv University, Tel Aviv 69978,
Israel</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745788">G-16. Systematic network analysis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Agarwal S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Agarwal S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Villar G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Villar G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Jones N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jones N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Many real-world systems, including a number of biological
ones, are naturally represented as networks, and a variety of measures exist
for network analysis. However, studies of networks typically employ only a
small, largely arbitrary subset of these, and the lack of a systematic
comparison makes it unclear which metrics are most appropriate for a given
task.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We present a framework for systematic analysis of
networks and network metrics, and use it to analyse a large set of real
networks from a wide variety of domains, utilising several hundred network
metrics or summary statistics thereof.</p>

<h3>Results</h3>

<p class="abresults">We demonstrate the utility of the framework for finding
redundant metrics, classifying networks, and selecting relevant features of
networks, particularly in the context of evolutionary phylogenies on biological
networks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our approach provides a generic, data-driven way of
thinking about networks of different kinds and relating structural and
topological features of networks to phenotypic properties. In particular, it is
useful in inferring what properties of biological networks such as metabolic or
protein interaction networks are most relevant in the context of evolutionary
pressures, and thus in helping to motivate models for network evolution and
function.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sumeet Agarwal (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62727 = 's&#117;m&#101;&#101;t.&#97;g&#97;rw&#97;l' + '&#64;';
 addy62727 = addy62727 + 'phys&#105;cs' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy62727 + suffix + '\'' + attribs + '>' );
 document.write( addy62727 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy411 = 's&#117;m&#101;&#101;t.&#97;g&#97;rw&#97;l' + '&#64;';
 addy411 = addy411 + 'phys&#105;cs' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy411 + suffix + '\'' + attribs + '>' );
 document.write( addy411 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52824 = 's&#117;m&#101;&#101;t.&#97;g&#97;rw&#97;l' + '&#64;';
 addy52824 = addy52824 + 'phys&#105;cs' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy52824 + suffix + '\'' + attribs + '>' );
 document.write( addy52824 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Oxford</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 University of Oxford</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Systems Biology Doctoral Training Centre Oxford, BBSRC,
EPSRC, Clarendon Fund</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745789">G-17. Model of trisporic acid synthesis in
Mucorales shows bistable behaviour</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Werner S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Werner S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Vlaic S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vlaic S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Schroeter A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schroeter A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Schuster S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schuster S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The synthesis pathway of trisporic acid involved in
sexual interactions in Mucorales is very interesting because it includes an
exchange of intermediates between the two mating types. Trisporic acid itself
enhances its synthesis rate by a positiv feedback loop. It is experimentally
observed that a switch from the production of a low-level concentration of
trisporic acid to a high-level state occurs if the fungi approach.the opposite
mating type. This bistable behaviour is modelled in this work.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For the ODE model the pathway of trisporic acid
production is shrinked to 19 essential species, subdivided into 13 metabolites
and 6 enzymes, as well as 41 reactions (conversion, diffusion, degradation and
inflow of metabolites). The reactions obey different kinetics: mass action
kinetics, Michaelis-Menten kinetics, constant flux, and Hill kinetics. To test
the model regarding bistable behaviour we defined an artifical inflow and
outflow of trisporic acid into the growth medium represented by a constant
addition resp. constant removal of trisporic acid over a fixed period.</p>

<h3>Results</h3>

<p class="abresults">In response to mimicking the approach of fungi of the
opposite mating type by an artifical inflow and outflow of trisporic acid, the
model reveals bistable behaviour. Above a threshold value of trisporic acid
constantly fed to the growth medium over a certain period of time the system
switches to the second stable state.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Due to lacking experimental data concerning metabolite
concentrations and (kinetic) parameter values in the trisporic acid production
pathway these have been chosen qualitatively. A more quantitative approach by
integrating measured data is pending.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://pinguin.biologie.uni-jena.de/bioinformatik/" mce_href="http://pinguin.biologie.uni-jena.de/bioinformatik/">http://pinguin.biologie.uni-jena.de/bioinformatik/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sarah Werner (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5115 = 's&#97;r&#97;h.w&#101;rn&#101;r' + '&#64;';
 addy5115 = addy5115 + '&#117;n&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy5115 + suffix + '\'' + attribs + '>' );
 document.write( addy5115 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16621 = 's&#97;r&#97;h.w&#101;rn&#101;r' + '&#64;';
 addy16621 = addy16621 + '&#117;n&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy16621 + suffix + '\'' + attribs + '>' );
 document.write( addy16621 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15827 = 's&#97;r&#97;h.w&#101;rn&#101;r' + '&#64;';
 addy15827 = addy15827 + '&#117;n&#105;-j&#101;n&#97;' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy15827 + suffix + '\'' + attribs + '>' );
 document.write( addy15827 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Friedrich-Schiller-University Jena</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Friedrich-Schiller-University Jena, Department of
Bioinformatics, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project is funded by the Jena School for Microbial
Communication (JSMC) and the German Research Foundation (DFG) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745790">G-18. Simulation of immune response to
Mycobacterium tuberculosis using an agent-based model</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Galvão V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Galvão V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Miranda JGV<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Miranda JGV&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Andrade RFS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Andrade RFS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The knowledge about the progression and control of
tuberculosis is important to identify the factors responsible for the migration
of immune cells to the affected tissue and the organization of these cells into
a granuloma. To clarify these mechanisms, an experiment identified the kinetics
of different cell types during the infectious process. Based on the results of
this experimental study, we have developed an agent-based model to verify some
hypothesis of the immune response to Mycobacterium tuberculosis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our model includes six different cell types: M.
tuberculosis, macrophage, CD4 T cell, B cell, pneumocyte, and necrosis. CD4 T and
B cells can be found in two states: inactivated and activated. Additionally,
macrophages can be found in four states: inactivated, infected, chronically
infected, and activated. It was constructed on a three-dimensional cubic
lattice. The lattice consists of discrete sites, such that each one of them can
be void or occupied by one type of autonomous agent.</p>

<h3>Results</h3>

<p class="abresults">This computational model can reproduce some cellular
properties including migration, activation, phagocytosis and death. Also, there
are the bacillus replication and necrosis formation. This way, it reproduces
the kinetics of M. tuberculosis, macrophages, CD4 T cells, B cell, and necrosis
formation. Our results were quantitatively and qualitatively compared with
experimental data, leading to a good agreement.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our results suggest that a multi-agent-based approach is
a suitable instrument for investigating and modeling the cellular interaction
in the progression of disease. Additionally, our model rules can simulate the
immune response to Mycobacterium tuberculosis.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Viviane Galvão (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89842 = 'v&#105;vg&#97;lv&#97;&#111;' + '&#64;';
 addy89842 = addy89842 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy89842 + suffix + '\'' + attribs + '>' );
 document.write( addy89842 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33969 = 'v&#105;vg&#97;lv&#97;&#111;' + '&#64;';
 addy33969 = addy33969 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy33969 + suffix + '\'' + attribs + '>' );
 document.write( addy33969 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2190 = 'v&#105;vg&#97;lv&#97;&#111;' + '&#64;';
 addy2190 = addy2190 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy2190 + suffix + '\'' + attribs + '>' );
 document.write( addy2190 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universidade Federal da Bahia</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Universidade Federal da Bahia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by Conselho Nacional de
Desenvolvimento Científico e Tecnológico (CNPq) and Fundação de Amparo à
Pesquisa do Estado da Bahia (FAPESB). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745791">G-19. Design, optimization and predictions of a
coupled model of the cell cycle, circadian clock, DNA repair system, irinotecan
metabolism and exposure control under temporal logic constraints</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Maria E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Maria E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Fages F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fages F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rizk A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rizk A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Soliman S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Soliman S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In systems biology, the number of available models of
cellular processes increases rapidly, but re-using models in different contexts
or for different questions remains a challenging issue. In this work, we study
the coupling of the mammalian cell cycle, the circadian clock, the p53/Mdm2
DNA-damage repair system, the metabolism of irinotecan and the control of cell
exposure to it. We show how the formalization of experimental observations in
temporal logic with numerical constraints can be used to compute the unknown
coupling kinetics parameter values agreeing with experimental data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We consider a model of the mammalian cell cycle proposed
by Novak and Tyson (2004), a model of the circadian clock developped by Leloup
and Golbeter (2003), a model of proteins p53/ Mdm2 introduced by Ciliberto et
al. (2005), and a model of the irinotecan metabolism by Dimitrio (2007). After
encoding the models in the rule-based language of Biocham, we add suitable
linking rules and we use an original method based on temporal logic constraint
solving and optimization techniques to find parameter values for the new rules
so that some expected properties are verified by the coupled model.</p>

<h3>Results</h3>

<p class="abresults">The coupling of the composite models has been achieved and
irinotecan exposure times and maximum amount that maintain toxicity low for
healthy cells have been found (in Biocham). The predictive power of the coupled
model was tested with respect to a limited set of mutants of the circadian
clock genes. In the case of genes knock outs, we succeeded in considering
temporal logic constraints over different traces corresponding to the mutations
of different genes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Although preliminary, the results obtained are very
encouraging for our coupling method. They pointed out that mass-entrained
models of the cell-cycle have a limited possibility of entrainment by the
circadian clock and that non mass-entrained models should be preferred in
future studies. They also showed that the p53/Mdm2 model of Ciliberto et al. should
be improved to introduce a threshold above which the DNA is no longer repaired.
Finally, a PK/PD model of irinotecan in the body should be added to link the injection
law to the cell exposure model and optimize the injection law directly.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://contraintes.inria.fr/supplementary_material/TCS-CMSB09/" mce_href="http://contraintes.inria.fr/supplementary_material/TCS-CMSB09/">http://contraintes.inria.fr/supplementary_material/TCS-CMSB09/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Elisabetta De Maria (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35255 = '&#101;l&#105;s&#97;b&#101;tt&#97;.d&#101;_m&#97;r&#105;&#97;' + '&#64;';
 addy35255 = addy35255 + '&#105;nr&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy35255 + suffix + '\'' + attribs + '>' );
 document.write( addy35255 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy48940 = '&#101;l&#105;s&#97;b&#101;tt&#97;.d&#101;_m&#97;r&#105;&#97;' + '&#64;';
 addy48940 = addy48940 + '&#105;nr&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy48940 + suffix + '\'' + attribs + '>' );
 document.write( addy48940 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4914 = '&#101;l&#105;s&#97;b&#101;tt&#97;.d&#101;_m&#97;r&#105;&#97;' + '&#64;';
 addy4914 = addy4914 + '&#105;nr&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy4914 + suffix + '\'' + attribs + '>' );
 document.write( addy4914 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">INRIA Paris-Rocquencourt</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">INRIA Paris-Rocquencourt</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the EU FP6 STREP project
TEMPO on cancer chronotherapies and is now supported by the ERASysBio project
C5Sys concerning circadian and cell cycle clock systems in cancer. We
acknowledge fruitful discussions with the partners of this project, in
particular with Francis Levi, Jean Clairambault and Annabelle Ballesta. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745792">G-20. Adding structural information to the von
Hippel-Lindau (VHL) tumor suppressor interaction network</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Leonardi E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Leonardi E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2, *), Murgia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Murgia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Tosatto S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tosatto S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The von Hippel-Lindau (VHL) tumor suppressor gene is a
protein interaction hub, controlling numerous genes implicated in tumor
progression. Here we focus on structural aspects of protein interactions for a
list of 35 experimentally verified protein VHL (pVHL) interactors.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Regions of pVHL sequence where several proteins
interact were inferred to contain a binding interface. For each class of interactors,
the Pfam and CATH classification codes were compared to search for common
domains or architectures. For each interactor, the GO molecular function and
cellular localization allowed us to determine functions mediated by each
interface region with different space and time patterns. A potential linear
motif which could mediate the interaction with pVHL was searched on the protein
sequences starting from the HIF1a peptide.</p>

<h3>Results</h3>

<p class="abresults">The modular nature of pVHL becomes apparent from the
subdivision in three interaction interfaces corresponding to processing,
substrate recognition and localization. These highlight various protein
interaction types, namely domain-domain (interface A) and domain-peptide
(interface B), with interface C being less clear. Structural characterization
of the putative interface B interaction peptides yielded both a complete list
of hypothetical interaction motifs and the intriguing possibility for the pVHL
N-terminus to auto-inhibit substrate recognition after phosphorylation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our findings, rationalizing pVHL function at the
molecular level through experimental data from the literature, can serve as a
gold standard for the analysis of other putative interaction partners. They can
be useful to design and interpret specific experimental interaction assays for
the 200 or more remaining putative pVHL interaction partners. Addressing these
in different time and space points will provide the ultimate validation for
this complex protein.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Emanuela Leonardi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy15207 = '&#101;m&#97;n&#117;&#101;l&#97;.l&#101;&#111;n&#97;rd&#105;' + '&#64;';
 addy15207 = addy15207 + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy15207 + suffix + '\'' + attribs + '>' );
 document.write( addy15207 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45152 = '&#101;m&#97;n&#117;&#101;l&#97;.l&#101;&#111;n&#97;rd&#105;' + '&#64;';
 addy45152 = addy45152 + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy45152 + suffix + '\'' + attribs + '>' );
 document.write( addy45152 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy75237 = '&#101;m&#97;n&#117;&#101;l&#97;.l&#101;&#111;n&#97;rd&#105;' + '&#64;';
 addy75237 = addy75237 + '&#117;n&#105;pd' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy75237 + suffix + '\'' + attribs + '>' );
 document.write( addy75237 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dept. of Biology, University of Padua</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Biology, University of Padua, 35121
Padua, Italy Department of Pediatrics, University of Padua, 35128 Padua, Italy</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745793">G-21. Computational multiscale modeling of brain
tumor growth</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schütz T A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Schütz T A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Toma A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Toma A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Becker S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Becker S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mang A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mang A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Buzug T M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Buzug T M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The prognosis for patients with malignant brain tumors
remains poor. Consequently, a better understanding of the complex mechanisms
underlying tumor progression is of key interest to design better treatment
strategies. A powerful tool to test hypotheses on tumor evolution for
individual patients and thus, to improve the understanding of the disease, is
the mathematical modeling of tumor growth. A novel multi-scale approach for
coupling our existing hybrid model for brain tumor progression on a microscopic
scale with a molecular model based on ordinary differential equations (ODEs) is
proposed.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The present work introduces a mathematical model
describing the avascular tumor progression on a microscopic and molecular
scale. More precisely, a hybrid model is used, which additionally considers the
nutrient concentration described in terms of reaction-diffusion equations. The
solution is computed using the finite element method (FEM) and itself
influences a molecular network on the subcellular level. This network is
defined by a system of ODEs whose solution reflects the tumor cell processes.
The cellular automata method is used to simulate chemotactic motility and
necrosis.</p>

<h3>Results</h3>

<p class="abresults">Visual inspection of the results demonstrates the
plausibility of the implemented model. The computed dynamics of cancerous cells
follows the well known exponential increase in cell population during the early
stage of cancer progression. For the subsequent cancer growth the model
displays structures exhibiting a necrotic core, a rim of quiescent cells
surrounded by dividing cells.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We report first results for a multiscale simulation
framework of brain cancer dynamics. The model depicts the expected
characteristic spatial patterns. The central aspect of this work is the
coupling of the cellular model including the nutrient concentration with a
molecular network on the subcellular scale. Using the FEM to solve for the
nutrient concentration it is possible to provide an efficient solution. The
standard processes for cancerous cells are controlled by molecular reactions
and protein concentrations. The model furthermore incorporates chemotaxis.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tina Anne Schütz (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy30641 = 'sch&#117;&#101;tz' + '&#64;';
 addy30641 = addy30641 + '&#105;mt' + '&#46;' + '&#117;n&#105;-l&#117;&#101;b&#101;ck' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy30641 + suffix + '\'' + attribs + '>' );
 document.write( addy30641 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58641 = 'sch&#117;&#101;tz' + '&#64;';
 addy58641 = addy58641 + '&#105;mt' + '&#46;' + '&#117;n&#105;-l&#117;&#101;b&#101;ck' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy58641 + suffix + '\'' + attribs + '>' );
 document.write( addy58641 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27809 = 'sch&#117;&#101;tz' + '&#64;';
 addy27809 = addy27809 + '&#105;mt' + '&#46;' + '&#117;n&#105;-l&#117;&#101;b&#101;ck' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy27809 + suffix + '\'' + attribs + '>' );
 document.write( addy27809 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Medical Engineering, University of
Lübeck; Graduate School for Computing in Medicine and Life Sciences, University
of Lübeck</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Institute of Medical Engineering, University of
Lübeck 2 Graduate School for Computing in Medicine and Life Sciences,
University of Lübeck</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Tina Anne Schütz is supported by the Graduate School
for Computing in Medicine and Life Sciences funded by Germany’s Excellence
Initiative [DFG GSC 235/1]. Alina Toma and Stefan Becker are financially
supported by the European Union and the State Schleswig-Holstein (Competence
Center for Technology and Engineering in Medicine (TANDEM): grant no.
122-09-024). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745794">G-22. SLIDER: a generic metaheuristic for the
discovery of correlated motifs in protein-protein interaction networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Boyen P<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Boyen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Van Dyck D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Dyck D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Neven F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Neven F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), van Ham RCHJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Ham RCHJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), van Dijk ADJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Dijk ADJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Correlated motif mining (CMM) is the problem of finding
overrepresented pairs of patterns, called motifs, in sequences of interacting
proteins. Algorithmic solutions for CMM thereby provide a computational method
for predicting binding sites for protein interaction. Large-scale biological
networks describing interactions between proteins are available now for several
organisms. Such data show which proteins interact, but provides no insight into
how interactions are encoded in protein sequences.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">we adopt a motif-driven approach where the support
of candidate motif pairs is evaluated in the network. We present the generic
metaheuristic SLIDER which uses steepest ascent with a neighborhood function
based on sliding motifs and employs the Chi-square-based support measure. We
implemented these methodes using Java. We validated our method on simulated
data as well as the protein-protein interaction networks for yeast and human. The
SLIDER-implementation and the data used in the experiments are available on
http://bioinformatics.uhasselt.be.</p>

<h3>Results</h3>

<p class="abresults">We experimentally establish the superiority of the
Chi-square-based support measure over other support measures. Furthermore, we
obtain that CMM is an NP-hard problem for a large class of support measures
(including Chi-square) and reformulate the search for correlated motifs as a
combinatorial optimization problem. We show that SLIDER outperforms existing
motif-driven CMM methods and scales to large protein-protein interaction
networks. For our results on the human PPI-network, we find significant overlap
with the interface (the physically interacting surface areas of the proteins).</p>

<h3>Discussion</h3>

<p class="abdiscussion">This work establishes an adequate support measure and
determines the complexity of the motif-driven CMM problem. SLIDER outperforms other
motif-driven CMM algorithms and shows promising behavior on real-world
PPI-networks. Directions for future work include investigating candidate
generation for motif pairs. A detailed comparison with interaction-driven
approaches should be done. Maybe ideas from both paradigms can be combined into
a hybrid method. We used the simple model of (l,d)-motifs and our results
suggest that this model suffers from false positives caused by indirect
interactions.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinformatics.uhasselt.be" mce_href="http://bioinformatics.uhasselt.be">http://bioinformatics.uhasselt.be</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Peter Boyen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84723 = 'p&#101;t&#101;r.b&#111;y&#101;n' + '&#64;';
 addy84723 = addy84723 + '&#117;h&#97;ss&#101;lt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy84723 + suffix + '\'' + attribs + '>' );
 document.write( addy84723 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87618 = 'p&#101;t&#101;r.b&#111;y&#101;n' + '&#64;';
 addy87618 = addy87618 + '&#117;h&#97;ss&#101;lt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy87618 + suffix + '\'' + attribs + '>' );
 document.write( addy87618 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99141 = 'p&#101;t&#101;r.b&#111;y&#101;n' + '&#64;';
 addy99141 = addy99141 + '&#117;h&#97;ss&#101;lt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy99141 + suffix + '\'' + attribs + '>' );
 document.write( addy99141 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Hasselt University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Hasselt University and Transnational University of
Limburg (2) Applied Bioinformatics - Plant Research International, Wageningen
UR</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Peter Boyen is funded by a Ph.D grant of the Institute
for the Promotion of Innovation through Science and Technology in Flanders
(IWT-Vlaanderen). A. D. J. van Dijk is supported by an NWO (Netherlands
Organisation for Scientific Research) VENI grant (863.08.027). This research is
supported by the BioRange programme (SP 2.3.1) of the Netherlands
Bioinformatics Centre (NBIC), which is supported through the Netherlands
Genomics Initiative (NGI) and the Research Programme of the Research Foundation
Flanders (FWO) (G030607). This work was also sponsored by the BiG Grid project
for the use of computing and storage facilities, with financial support from
NWO. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745795">G-23. The effective interactions between proteins
via coarse-grained modeling</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Pool R<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Pool R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Feenstra KA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Feenstra KA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Heringa J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Heringa J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The knowledge on protein-protein interactions (PPIs) is
important since they play an important role in many cellular processes.
Aggregation in solution can occur by driven processes or via self-assembly.
Here the effective interaction between the monomers as a function of molecular
separation (potentials of mean force (PMFs)) is of vital importance. Currently,
a systematic exploration of a large number of PPIs is practically unfeasible
due to the large amount of time needed for experiments or computations. Here we
propose a coarse-grained approach that yields PMFs within reasonable times.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use state-of-the-art molecular simulation
techniques (hybrid Molecular Dynamics-Monte Carki), applied to coarse-grained
systems using the MARTINI force field. Sampling efficiency is further
accomplished by reducing the system size to the volume including the
interacting interfaces as well as the volume between them. All interacting
species are simulated explicitly. Forces acting between the interfaces are
averaged. The resulting force profile is then integrated to yield a PMF.</p>

<h3>Results</h3>

<p class="abresults">Using our coarse-grained and reduced system approach we can
succesfully determine PMFs for given protein pairs. As expected the PMFs are
attractive, The obtained PMFs agree with simulation results from complete
coarse-grained systems that include the completely dissolved protein
structures. In addition, the coarse-grained simulation results show a
considerable agreement with more realistic and more expensive full atom
simulations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The results obtained so far are promising, although we
should be aware of physical phenomena we simply cannot capture using a
coarse-grained model. The combination of the coarse-grained molecular model and
the minimal system size is expected to enable large scale exploration of the
possible existence of protein-protein interactions in currently available
databases.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">René Pool (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy95035 = 'r.p&#111;&#111;l' + '&#64;';
 addy95035 = addy95035 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy95035 + suffix + '\'' + attribs + '>' );
 document.write( addy95035 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1944 = 'r.p&#111;&#111;l' + '&#64;';
 addy1944 = addy1944 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy1944 + suffix + '\'' + attribs + '>' );
 document.write( addy1944 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16964 = 'r.p&#111;&#111;l' + '&#64;';
 addy16964 = addy16964 + 'f&#101;w' + '&#46;' + 'v&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy16964 + suffix + '\'' + attribs + '>' );
 document.write( addy16964 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Centre for Integrative Bioinformatics Vrije
Universiteit (IBIVU)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Centre for Integrative Bioinformatics Vrije
Universiteit Amsterdam (IBIVU)</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is funded by the Netherlands Bioinformatics
Centre (NBIC): BioRange 2.3.1. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745796">G-24. Predicting cellular perturbation effects and
identifying causal perturbations using a message-passing based machine learning
method</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hulsman M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Hulsman M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Reinders M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinders M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->JT<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinders MJT&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Perturbations in a cell, as a result of a changing
environment or a gene knockout, are propagated through different physical links
within the cellular network. Changes in expression levels of genes give an
impression of the resulting effects of a given perturbation. Together with direct
and indirect information about the physical links, these observations can be
used to unravel the cellular network and the network information flow. We
present a method that simultaneously reconstructs and simulates a cellular
network capable of propagating perturbations throughout the network.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The network reconstruction is based on a machine
learning method that takes various data sources, such as protein-protein
interaction and protein-DNA interaction, into account to predict the
probability of an edge propagating the perturbation from its parent node to its
child node. The network simulation is realized by a novel propagation model
that passes messages throughout the network. The combined message passing
scheme and learned network generates probabilities expressing whether a gene
shows a change in expression given a perturbation.</p>

<h3>Results</h3>

<p class="abresults">The proposed method has been applied to learn from gene
knock-out data. It generalizes to unseen knockouts, i.e. one is able to predict
genes that change expression levels after a hypothetical gene knockout as well
as propose a hypothetical gene knockout that maximally explains an observed set
of changed gene expression levels. For most knockouts, up to 50% of the
affected genes are correctly predicted. Important data sources included
literature and chip-chip data. Approximately, 12% of the knockout genes were
accurately predicted from perturbed profiles.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Predicting knockout effects is complicated as they
disturb the normal functioning of the cell, leading to many side-effects. Often,
this leads to growth defects, stress responses, etc. In our analysis, we found
that 40% of the effects consisted of genes that were affected in more than 10
knockouts. HSP30, a known stress-response gene, was even affected in 95 of the
179 knockouts. GO analysis relates these genes to metabolic processes, which
would be expected if growth defects occur. This observation suggests that
predictions should be related to each other to characterize the core effects.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Marc Hulsman (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6830 = 'm.h&#117;lsm&#97;n' + '&#64;';
 addy6830 = addy6830 + 't&#117;d&#101;lft' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy6830 + suffix + '\'' + attribs + '>' );
 document.write( addy6830 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57419 = 'm.h&#117;lsm&#97;n' + '&#64;';
 addy57419 = addy57419 + 't&#117;d&#101;lft' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy57419 + suffix + '\'' + attribs + '>' );
 document.write( addy57419 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36056 = 'm.h&#117;lsm&#97;n' + '&#64;';
 addy36056 = addy36056 + 't&#117;d&#101;lft' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy36056 + suffix + '\'' + attribs + '>' );
 document.write( addy36056 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Delft University of Technology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Delft University of Technology Faculty of Electrical
Engineering, Mathematics, and Computer Science Delft Bioinformatics Lab</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745797">G-25. Towards better receptor-ligand prediction</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Iacucci E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Iacucci E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Ojeda F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ojeda F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Moor B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Moor B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Regulation of cellular events is often initiated via
extracellular signaling. Extracellular signaling occurs when a circulating
ligand interacts with one or more membrane-bound receptors. Identification of
receptor-ligand pairs is thus an important and spe-cific form of PPI
prediction. Given a set of disparate data sources (expression data, domain
content, and phylogenetic profile) we seek to predict new receptor-ligand
pairs. We create a combined kernel classifier and assess its performance with
respect to the DLRP ‘golden standard’ as well as the method proposed by Gertz
et al. (2003).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our objective is to predict candidate receptor
ligand pairs; more specifically, seeking to create a better method than what is
currently available (Gertz et al. 2003) to identify known pairs as well as to
determine putative pairs for further research. Our method involves taking
multiple data sources and producing separate kernels for each data type,
creating LS-SVM classifiers and combining the results to predict receptor
ligand pairs.</p>

<h3>Results</h3>

<p class="abresults">Among our findings, we discover that our predictions for the
tgfβ family accurately reconstructs over 95% of the supported edges (0.95
recall and 0.71 precision) of the receptor-ligand bipartite graph defined by
the DLRP “golden standard”. In addition, the combined kernel classifier is able
to relatively out-preformed the Gertz et al. (2003) work by a factor of
approximately two as the Gertz et al. (2003) work reconstruct 44% of the
supported edges (0.44 recall and 0.53 precision) of the receptor-ligand bipartite
graph defined by the DLRP “golden standard”.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The prediction of receptor ligand pairings is a difficult
and complex task. We have demonstrated that using multiple data sources provide
a clear advantage over single data sources in solv-ing this task. As more high
through-put data becomes available, we expect to extend the current methodology
to accommodate it.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ernesto Iacucci (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40134 = '&#101;rn&#101;st&#111;.&#105;&#97;c&#117;cc&#105;' + '&#64;';
 addy40134 = addy40134 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy40134 + suffix + '\'' + attribs + '>' );
 document.write( addy40134 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy80520 = '&#101;rn&#101;st&#111;.&#105;&#97;c&#117;cc&#105;' + '&#64;';
 addy80520 = addy80520 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy80520 + suffix + '\'' + attribs + '>' );
 document.write( addy80520 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33221 = '&#101;rn&#101;st&#111;.&#105;&#97;c&#117;cc&#105;' + '&#64;';
 addy33221 = addy33221 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy33221 + suffix + '\'' + attribs + '>' );
 document.write( addy33221 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">K. U. Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of ESAT-SCD, Kathlieke Universiteit
Leuven, Kasteelpark Arenberg 10, 3001 Leuven, Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745798">G-26. Extracting and visualising protein-protein
interfaces for data-driven docking</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ghoorah AW<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ghoorah AW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Devignes M-D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Devignes M-D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Smaïl-Tabbone M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Smaïl-Tabbone M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Ritchie DW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ritchie DW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">PPIs play a key role in many cellular processes. In order
to understand these processes, we need to be able to analyse and model them at
the molecular level. There now exist several databases which contain almost all
of the available experimentally-determined PPIs. Our aim is to search such
databases for potentially useful information to identify key interface residues
automatically in order to help predict the structures of unknown complexes
using protein docking simulations.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Given a query domain, we retrieve from the 3DID
database all hetero domain-domain interactions involving the query.
Biologically relevant interfaces are distinguished from other crystal contacts
using calculated interface areas, and duplicates are removed using a sequence
similarity threshold. We then annotate and store the 3DID interface residues
according to their location in the interface (i.e. “core” or “rim”). We also
calculate and store overall residue conservation. We use PFAM consensus
sequence to perform 3D superposition of the PDB complexes which contain the
query domain.</p>

<h3>Results</h3>

<p class="abresults">For any given query domain, our approach retrieves
automatically a set of non-redundant hetero interactions. For each interaction,
we use VMD to show the core and rim residues at the interface and to highlight
any conserved residues. In addition we show a superposition of the protein
complexes in the coordinate frame of the query. This provides a convenient way
to view and analyse all of the interactions involving the domain of interest.
Using this approach we modelled CAPRI target T40, a complex between an API-A
and two trypsins, and we obtained 9 out of 10 acceptable predictions.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have developed an automated approach for extracting
and analysing relevant domain-domain interactions involving a query domain. We
aim to make this approach available as a web server. We are also extending our
approach to annotate PPIs according to their biological function and to
distinguish permanent and transient interactions. In the longer term, we aim to
apply machine learning techniques to discover symbolic rules which can identify
key interface residues automatically.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anisah W Ghoorah (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy85235 = '&#97;n&#105;s&#97;h.gh&#111;&#111;r&#97;h' + '&#64;';
 addy85235 = addy85235 + 'l&#111;r&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy85235 + suffix + '\'' + attribs + '>' );
 document.write( addy85235 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43246 = '&#97;n&#105;s&#97;h.gh&#111;&#111;r&#97;h' + '&#64;';
 addy43246 = addy43246 + 'l&#111;r&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy43246 + suffix + '\'' + attribs + '>' );
 document.write( addy43246 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33632 = '&#97;n&#105;s&#97;h.gh&#111;&#111;r&#97;h' + '&#64;';
 addy33632 = addy33632 + 'l&#111;r&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy33632 + suffix + '\'' + attribs + '>' );
 document.write( addy33632 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">INRIA Nancy Grand-Est</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. INRIA Nancy; 2. CNRS Nancy; 3. Nancy Université,
LORIA, Vandoeuvre-lès-Nancy 54506, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project is funded by the Agence Nationale de la
Recherche, France. Grant reference ANR-08-CEXC-017-01. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745799">G-27. Visualizing spatiotemporal information in
heterogeneous biological networks with Arena3D</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Secrier M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Secrier M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Pavlopoulos GA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pavlopoulos GA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Schneider R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schneider R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Biological systems are complex, dynamic entities, the
behavior of which is difficult to capture in a holistic manner over space and
time. Visualization tools enhance the ability to explore the properties of such
systems by exploiting man's capacity to perceive hidden features in the data.
Several tools for the visualization of biological networks exist, but
integrating the spatial and temporal information remains a major bottleneck in
systems biology. We introduce Arena3D, a framework for biological visualization
and analysis, with focus on network linking and dynamic pattern identification.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Arena3D employs a 2.5D framework with separate
layers to distinguish different levels of biological information (genes,
proteins, chemicals, pathways, diseases etc). The different entity layers are
arranged via layout algorithms and the connections between the layers are
visualized. The core graphics visualization in Arena3D is implemented in Java
3D (1.5.1 API). All other parts of Arena3D, including clustering methods and
the GUI, are done in Java (JDK 1.6). Parsing is enabled for SMBL and KEGG file
formats. Networks can be exported in Medusa, Pajek and VRML format.</p>

<h3>Results</h3>

<p class="abresults">Visualizing biochemical reaction networks highlights the
most prominent reactions at different time points and the changes in enzyme
concentrations. The reaction profile of the cell cycle is described. The
succession of phenotypic events upon siRNA knockdown of genes essential for the
cell cycle (www.mitocheck.org) has been traced based on scoring and clustering
of events. Distinct patterns are pinpointed. We observe many positive
correlations between gene knockdowns for the binuclear and polylobed phenotypes
and highly uncorrelated behavior in the large and dynamic phenotypes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The Arena3D software enables the investigation of a wide
variety of networks and connections between systems at different levels, as
well as their evolution in time, by emphasizing changes in gene expression and
highlighting pathways that are activated at different stages. The 3D framework
allows the user to easily understand the connections that appear between
different biological levels. Providing an enriched informational content
through spatiotemporal integration can trigger the discovery of new
relationships between biomolecules, links between processes and patterns in
evolution.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://arena3d.org/" mce_href="http://arena3d.org/">http://arena3d.org/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Maria Secrier (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38058 = 's&#101;cr&#105;&#101;r' + '&#64;';
 addy38058 = addy38058 + '&#101;mbl' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38058 + suffix + '\'' + attribs + '>' );
 document.write( addy38058 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy48361 = 's&#101;cr&#105;&#101;r' + '&#64;';
 addy48361 = addy48361 + '&#101;mbl' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy48361 + suffix + '\'' + attribs + '>' );
 document.write( addy48361 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50252 = 's&#101;cr&#105;&#101;r' + '&#64;';
 addy50252 = addy50252 + '&#101;mbl' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy50252 + suffix + '\'' + attribs + '>' );
 document.write( addy50252 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">European Molecular Biology Laboratory, Heidelberg,
Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Structural and Computational Biology Unit, European
Molecular Biology Laboratory, Heidelberg, Germany (2) Department of Computer
Science and Biomedical Informatics, University of Central Greece, Lamia, Greece</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection29">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745800">G-28. Efficient learning of signaling networks from
perturbation time series via dynamic nested effects models</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Praveen P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Praveen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Tresch A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tresch A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Froehlich H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Froehlich H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Reverse engineering of biological networks is a key to
understand the biological systems. The exact knowledge of interdependencies
among proteins in the living cell is crucial for the drug targets
identification in various diseases. The advent of techniques, like RNAi, opens
new aspects in network reconstruction. NEMs (Markowetz et al, 2005) allow
inferring network hypothesis from high-dimensional, indirect downstream perturbation
effects. The original approach works only with static data. However,
time-resolved data can give better insight into the signal behavior and resolve
feedback loops.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We propose an extension of the NEM approach for
perturbation time series measurements, thus complementing the attempt of
Anchang et al. (2009). It allows resolving the feedback loops in the signaling
cascade and discriminates direct &amp; indirect signaling therein. Our method
works by unrolling the signal flow in the network over time. The signaling is
modeled via a Boolean network with synchronous state updates. The likelihood of
a network hypothesis is determined efficiently in a closed expression. A
network structure prior limits the search space for structure learning via
greedy hill climber.</p>

<h3>Results</h3>

<p class="abresults">Realistic simulations show a high specificity and
sensitivity of our proposed approach. Moreover, it compares favorable against
standard Dynamic Bayesian Networks (DBNs), which do not model the propagation
of a perturbation effects in a signaling network. In contrast to the approach
by Achang et al. our method does not use time consuming Gibbs sampling.
Moreover, our model can estimate feedback loops from data as we allow genes to
change their perturbation state over time. Application of our approach to a
dataset for murine stem cell development leads to biologically interpretable
results.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We proposed an efficient statistical model to reconstruct
features of signaling cascades from perturbation time series. It allows
feedback loop estimation since genes are allowed to change its activation state
over time. We suggested an appropriate prior for network structure learning via
a greedy hill climbing algorithm. Realistic simulations evince the accuracy of
our method, subsequently applied on data for murine stem cell development. A
non-parametric bootstrap verifies the confidence of the learned network. Our
results are in-line with the data and with previous results by Anchang et al.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.abi.bit.uni-bonn.de" mce_href="http://www.abi.bit.uni-bonn.de">http://www.abi.bit.uni-bonn.de</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Paurush Praveen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53885 = 'pr&#97;v&#101;&#101;n' + '&#64;';
 addy53885 = addy53885 + 'b&#105;t' + '&#46;' + '&#117;n&#105;-b&#111;nn' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy53885 + suffix + '\'' + attribs + '>' );
 document.write( addy53885 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38202 = 'pr&#97;v&#101;&#101;n' + '&#64;';
 addy38202 = addy38202 + 'b&#105;t' + '&#46;' + '&#117;n&#105;-b&#111;nn' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy38202 + suffix + '\'' + attribs + '>' );
 document.write( addy38202 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84220 = 'pr&#97;v&#101;&#101;n' + '&#64;';
 addy84220 = addy84220 + 'b&#105;t' + '&#46;' + '&#117;n&#105;-b&#111;nn' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy84220 + suffix + '\'' + attribs + '>' );
 document.write( addy84220 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Algorithmic Bioinformatics, B-IT, University of Bonn</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Algorithmic Bioiformatics, B-IT, University of Bonn 2.
Gene Center Munich, Ludwig-Maximilians-University Munich</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by the state of NRW
via the B-IT Research School</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection30">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745801">G-29. Discovering patterns of differentially
regulated enzymes in metabolic pathways of tumors</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schramm G<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Schramm G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1),Wiesberg S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wiesberg S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Diessl N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Diessl N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kranz A-J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kranz A-J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sagulenko V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sagulenko V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Oswald M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Oswald M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Reinelt G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinelt G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Westermann F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Westermann F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Eils R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Eils R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Konig R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Konig R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Gene expression profiling by microarrays or transcript
sequencing enables observing the pathogenic function of tumors on a mesoscopic
level.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We investigated gene expression profiles of
neuroblastoma and breast cancer tumors. In contrast to common enrichment tests,
we took network topology into account by applying adjusted wavelet transforms
on an elaborated and new 2D grid representation of curated pathway maps from
the Kyoto Enzyclopedia of Genes and Genomes.</p>

<h3>Results</h3>

<p class="abresults">The aggressive form of the neuroblastoma tumors showed
regulatory shifts for purine and pyrimidine biosynthesis as well as
folate-mediated metabolism of the one carbon pool in respect to increased
nucleotide production. We spotted an oncogentic regulatory switch in glutamate
metabolism for which we provided experimental validation, being the first steps
towards new possible drug therapy. For the breast cancer tumors, we found a
regulatory switch a the bile acid biosynthesis pathway which may inhibit
cholesterol degradation and therefore induction of estrogen synthesis.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The pattern recognition method we used complements normal
enrichment tests to de tect such functionally related regulation patterns.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Rainer Konig (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56075 = 'r.k&#111;&#101;n&#105;g' + '&#64;';
 addy56075 = addy56075 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy56075 + suffix + '\'' + attribs + '>' );
 document.write( addy56075 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73457 = 'r.k&#111;&#101;n&#105;g' + '&#64;';
 addy73457 = addy73457 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy73457 + suffix + '\'' + attribs + '>' );
 document.write( addy73457 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33160 = 'r.k&#111;&#101;n&#105;g' + '&#64;';
 addy33160 = addy33160 + 'dkfz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy33160 + suffix + '\'' + attribs + '>' );
 document.write( addy33160 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Bioquant, IPMB</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Bioinformatics and Functional Genomics,
Institute of Pharmacy and Molecular Biotechnology, and Bioquant, University of
Heidelberg, Im Neuenheimer Feld 267, 2 Institute of Computer Science, 3
Interdisciplinary Center for Scientific Computing, University of Heidelberg 4
Department of Tumor Genetics, German Cancer Research Center (DKFZ), Im Neuenheimer
Feld 280, 69120 Heidelberg, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Funding: BMBF-FORSYS Consortium Viroquant (#0313923);
the Helmholtz Alliance on Systems Biology and the Nationales Genom-
Forschungs-Netz (NGFN+) for the neuroblastoma project, ENGINE (#01GS0898). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection31">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745802">G-30. Extending the yeast metabolic network using a
data integration approach</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Fisher P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Fisher P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, Dobson P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dobson P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Brenninkmeijer C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brenninkmeijer C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Canevet C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Canevet C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Taubert J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Taubert J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Rawlings C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rawlings C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Stevens R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Stevens R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">We are working towards the extension of the yeast
metabolic network [PMID: 18846089]. However manual curation is expensive and
therefore a combination of bioinformatics tools is required to generate a
semi-automatic approach which can assist in the curation of the metabolic
network, providing domain experts with relevant publications, and subsequently
reducing curation time. We have adopted an integrated systems biology approach
to exploit a wide range of datasets that are useful to the curators and
bioinformaticians on the Yeast Jamboree Project.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The Yeast metabolic network was imported into Ondex
and a filter applied to remove highly connected metabolites. Unconnected
(orphan) metabolites were identified and used to filter the KEGG Yeast
database. A Taverna workflow was created and run to find these orphans in the
KEGG Yeast database, retrieve enzyme names, and assemble a MEDLINE query
(restricted to the yeast literature) based on those names. Literature
identified by this workflow was re-integrated back into the KEGG Yeast network
stored within Ondex, as a means of identifying novel metabolic reactions for
each orphan metabolite.</p>

<h3>Results</h3>

<p class="abresults">The Taverna and Ondex tools were successfully combined into
a single data integration platform. As a result, we developed a mechanism in
which researchers are able to identify gaps within the yeast metabolism network
by providing them with publications that may be relevant to close such gaps.
New Taverna workflows were created to obtain a set of MEDLINE publications which
allowed us to annotate a subset of the KEGG database relating to gaps in the
Yeast metabolism network. Identified orphans were linked to publications via a
Taverna workflow and re-integrated into the network.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Through the combination of Ondex and Taverna, we were
able to successfully construct an explicit methodology that provides a powerful
data analysis platform for systems biology research. We have developed a system
that considerably reduces the time needed by the curators of the Yeast Jamboree
to identify evidence that links new metabolites into the yeast metabolic
network. This has led to significant progress in extending the Jamboree model.
Publications added to the yeast metabolic network are currently being reviewed
by curators for their suitability to fill further gaps in the network.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ondex.org" mce_href="http://www.ondex.org">http://www.ondex.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Paul R Fisher (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60988 = 'p&#97;&#117;l.f&#105;sh&#101;r' + '&#64;';
 addy60988 = addy60988 + 'm&#97;nch&#101;st&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy60988 + suffix + '\'' + attribs + '>' );
 document.write( addy60988 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88664 = 'p&#97;&#117;l.f&#105;sh&#101;r' + '&#64;';
 addy88664 = addy88664 + 'm&#97;nch&#101;st&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy88664 + suffix + '\'' + attribs + '>' );
 document.write( addy88664 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78312 = 'p&#97;&#117;l.f&#105;sh&#101;r' + '&#64;';
 addy78312 = addy78312 + 'm&#97;nch&#101;st&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy78312 + suffix + '\'' + attribs + '>' );
 document.write( addy78312 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Manchester</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">The University of Manchester Rothamsted Research</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The Ondex SABR project is funded by BBSRC Grants
BBS/B/13640 and BB/F006039/1. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection32">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745803">G-31. Predicting genetic interactions by
quantifying redundancy in biochemical pathways</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Delgado-Eckert E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Delgado-Eckert E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Gill S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gill S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Merdes G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Merdes G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Beerenwinkel N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beerenwinkel N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Biochemical networks consist of interacting proteins that
carry out a function. Most of the proteins in such a network are not essential,
a property called robustness. Systematic double-knockout screens in S.
Cerevisiae have started to shed light on the nature of this robustness by
quantifying the degree of genetic redundancy. A pair of genes is said to
genetically interact if the phenotype of the double-knockout mutant notably
differs from the single-knockout mutants. Screening all pairs represents a huge
effort. Thus, computational methods that help predict genetic interactions are
required.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We devised a computational methodology that uses
the connectivity information contained in the protein interactome to predict
the likelihood of genetic interactions. To account for the uncertainty attached
to experimental conditions, we represent the interactome using probabilistic
networks, i.e., probabilities are assigned to the network’s edges. Within this
framework we use the concept of network reliability to address the stability of
the network under simultaneous perturbation of node pairs.</p>

<h3>Results</h3>

<p class="abresults">We are currently testing our methodology using the vast
amount of protein interaction data and the recently established genetic
interaction data available for S. Cerevisiae. We are focusing on the study of
transcription pathways in S. Cerevisiae due to the large body of knowledge and
experimental results regarding transcription mechanisms.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our first explorations show that one of the biggest
challenges in this approach is to define the connectivity properties of the
biochemical network that are required for the functionality of the pathway.
This step requires deep knowledge about the molecular mechanisms that underlie
a particular cellular function. This insight brings up the question as to what
extent one could explore by means of simulation what connectivity properties of
a network best explain the experimentally observed pattern of genetic
interactions and pairwise redundancies.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Edgar W. Delgado-Eckert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36225 = '&#101;dg&#97;r.d&#101;lg&#97;d&#111;-&#101;ck&#101;rt' + '&#64;';
 addy36225 = addy36225 + 'bss&#101;' + '&#46;' + '&#101;thz' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy36225 + suffix + '\'' + attribs + '>' );
 document.write( addy36225 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19304 = '&#101;dg&#97;r.d&#101;lg&#97;d&#111;-&#101;ck&#101;rt' + '&#64;';
 addy19304 = addy19304 + 'bss&#101;' + '&#46;' + '&#101;thz' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy19304 + suffix + '\'' + attribs + '>' );
 document.write( addy19304 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36952 = '&#101;dg&#97;r.d&#101;lg&#97;d&#111;-&#101;ck&#101;rt' + '&#64;';
 addy36952 = addy36952 + 'bss&#101;' + '&#46;' + '&#101;thz' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy36952 + suffix + '\'' + attribs + '>' );
 document.write( addy36952 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">ETH Zurich</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Biosystems Science and Engineering,
ETH Zürich, Mattenstrasse 26, 4058 Basel, Switzerland.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The authors acknowledge support from the SystemsX
initiative in systems biology of the Swiss National Science Foundation. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection33">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745804">G-32. Hub protein interfaces and hot region organization</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cukuroglu E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Cukuroglu E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Gursoy A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gursoy A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Keskin O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Keskin O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Binary interactions of the proteins constitute the
protein interaction networks. Some proteins are highly connected to the others
(called as hub proteins), whereas some others have a few interactions (called
as non hub proteins). Here, we concentrate on hub proteins from a structural
point of view: protein-protein interfaces. Our proposed method analyzes the
connection between organization of hot spots (hot regions) and being hub
proteins. We define interfaces as the ones between two date hubs (DD), and two
party hubs (PP).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A Hub protein dataset generated from PPIN dataset
is used in this study. Date, party and non hub proteins which have complex
structured PDB file are extracted and interfaces of complexes are fetched from
the interface dataset to eliminate redundancy. After elimination of the
structurally redundant complexes, there are 38 PP and 26 DD interfaces. Hot
spots of the interfaces are defined and the groups of hot spots which have at
least two contacting hotspot neighbors in the interface are called hot regions.
Significant features of the interface are defined with ANOVA test.</p>

<h3>Results</h3>

<p class="abresults">Results reveal that there are significant differences
between DD and PP interfaces. More of the hot spots are organized into the hot
regions in DD interfaces compared to PP ones. A high fraction of the interfaces
are covered by hot regions in DD interfaces. There are more distinct hot
regions in DDs. Since the same (or overlapping) DD interfaces should be used
repeatedly, different hot regions can be used to bind to different partners.
Further, these hot region characteristics can be used to predict whether a
given hub interface is involved in a DD or a PP interface type with 80%
accuracy.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In this work, we conclude that there is a relationship
between organization of hot spots (hot regions) and the status of hub proteins.
The hot region characteristics (Hot spot ratio, average hot region size,
average hot region ASA to interface ASA ratio, Polar amino acid (aa)
frequencies of interfaces, Polar aa frequencies of hot spots, Polar aa
frequencies of hot regions) can be used to predict whether an interface is
formed between a DD or PP type of an interface with 80% accuracy.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Engin Cukuroglu (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64034 = '&#101;c&#117;k&#117;r&#111;gl&#117;' + '&#64;';
 addy64034 = addy64034 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy64034 + suffix + '\'' + attribs + '>' );
 document.write( addy64034 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4026 = '&#101;c&#117;k&#117;r&#111;gl&#117;' + '&#64;';
 addy4026 = addy4026 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy4026 + suffix + '\'' + attribs + '>' );
 document.write( addy4026 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24569 = '&#101;c&#117;k&#117;r&#111;gl&#117;' + '&#64;';
 addy24569 = addy24569 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy24569 + suffix + '\'' + attribs + '>' );
 document.write( addy24569 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">KOC University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Koc University, Center for Computational Biology and
Bioinformatics and College of Engineering, Rumelifeneri Yolu, 34450 Sariyer, Istanbul,
Turkey</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection34">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745805">G-33. A functional-genomics fermentation platform
to identify and optimize industrial-relevant properties of Lactobacillus
plantarum</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wels M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Wels M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,4,*), Bron PA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bron PA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,), Wiersma A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wiersma A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,4), Marco M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marco M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,§), Kleerebezem M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kleerebezem M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,4)</p>

<p class="abmotivation">Probiotics constitute an important growth market for the
food industry. However, further development of the probiotics market is
presently constrained by a lack of knowledge on probiotic cellular components
important for health-promoting effects. We have developed a functional-genomics
fermentation platform approach for the identification and optimization of
expression of specific probiotic functionality parameters. The platform employs
Lactobacillus plantarum WCFS1 as an extensively studied model microbe for which
advanced molecular tools are available.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In the first set of fermentations, L. plantarum was
grown in chemically defined medium according to a combinatorial fermentation
scheme that included variations in medium composition and mild stress
conditions (NaCl, pH, oxygen, amino acids and temperature). The molecular
characteristics of the bacteria in samples harvested from these fermentations
were investigated by transcriptome analysis, while in parallel their functional
parameters were assessed using specific probiotic functionality assays, but
also flavor profile changes that might influence product taste.</p>

<h3>Results</h3>

<p class="abresults">Advanced bioinformatics analyses were performed to correlate
transcriptomics results with the functionality parameters obtained, enabling
the identification of genes involved in specific functions that are relevant
for performance. The function of the identified genes and their relevance for
performance will be further studied by genetic engineering (KO,
overexpression).</p>

<h3>Discussion</h3>

<p class="abdiscussion">This approach will allow optimization strategies for the
improvement of pre-selected target genes through specific modulation of
fermentation conditions. Although developed with the use of a model organism,
this approach is applicable to other bacteria, and the correlation of desired
functional properties to “omics” datasets may assist identification of the
underlying molecular mechanism, which opens avenues towards the design of
fermentation strategies geared to improve the functional properties of bacterial
(health-impact) cultures.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michiel Wels (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63174 = 'm&#105;ch&#105;&#101;l.w&#101;ls' + '&#64;';
 addy63174 = addy63174 + 'n&#105;z&#111;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy63174 + suffix + '\'' + attribs + '>' );
 document.write( addy63174 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy99061 = 'm&#105;ch&#105;&#101;l.w&#101;ls' + '&#64;';
 addy99061 = addy99061 + 'n&#105;z&#111;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy99061 + suffix + '\'' + attribs + '>' );
 document.write( addy99061 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26513 = 'm&#105;ch&#105;&#101;l.w&#101;ls' + '&#64;';
 addy26513 = addy26513 + 'n&#105;z&#111;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy26513 + suffix + '\'' + attribs + '>' );
 document.write( addy26513 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Top Institute Food and Nutrition</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1TIFN/2NIZO food research B.V., the Netherlands;
3Wageningen UR, the Netherlands; 4 Kluyver Centre for Industrial Fermentations,
Netherlands. § Present address UCDavis, USA</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection35">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745806">G-34. DisGeNET: a Cytoscape plugin to visualize,
integrate, search and analyze gene-disease networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bauer-Mehren A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bauer-Mehren A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Rautschka M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rautschka M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sanz F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sanz F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Furlong LI<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Furlong LI&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Most human diseases arise due to interactions among
multiple genetic variants and environmental factors. To get a comprehensive
view of the full spectrum of human diseases with genetic origin including
monogenic, complex and environmental diseases, we developed a comprehensive
database comprising gene-disease associations from expert-curated repositories
and a text-mining source. Network representation allows studying global
properties of genetic origin of human diseases. Hence, we developed DisGeNET, a
Cytoscape plugin to visualize, integrate, search and analyze gene-disease
networks.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed DisGeNET, a plugin for Cytoscape to
access and explore our gene-disease association database. Gene-disease
associations are displayed as bipartite-graphs with multiple edges between
nodes, each representing a unique association found in the original data
sources. Moreover, we color edges according to the association type following
our gene-disease association ontology. Network projections are accessible
presenting gene and disease centric view of the data. Diseases were classified
into disease classes and represented with multiple node colors.</p>

<h3>Results</h3>

<p class="abresults">DisGeNET allows user-friendly access to our database
including queries restricted to the (i) original source, (ii) association type,
(iii) disorder class of interest and (iv) specific disease/gene or set of
diseases/genes. It represents gene-disease associations as bipartite graphs and
additionally provides gene centric and disease centric views of the data. It
assists the user in the interpretation and exploration of human diseases with
respect to their genetic origin by a variety of built-in functions. DisGeNET is
compatible with Cytoscape 2.x.</p>

<h3>Discussion</h3>

<p class="abdiscussion">DisGeNET is a valuable source for biomedical research
with a variety of possible applications including finding new candidate disease
genes, understanding mechanisms underlying diseases, studying influence of
environmental factors such as drugs on human health. In addition, DisGeNET
provides a user-friendly framework and the possibility to make use of the
variety of available Cytoscape functionalities.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://ibi.imim.es/DisGeNET/DisGeNETweb.html" mce_href="http://ibi.imim.es/DisGeNET/DisGeNETweb.html">http://ibi.imim.es/DisGeNET/DisGeNETweb.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anna Bauer-Mehren (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy80138 = '&#97;b&#97;&#117;&#101;r-m&#101;hr&#101;n' + '&#64;';
 addy80138 = addy80138 + '&#105;m&#105;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy80138 + suffix + '\'' + attribs + '>' );
 document.write( addy80138 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5890 = '&#97;b&#97;&#117;&#101;r-m&#101;hr&#101;n' + '&#64;';
 addy5890 = addy5890 + '&#105;m&#105;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy5890 + suffix + '\'' + attribs + '>' );
 document.write( addy5890 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83932 = '&#97;b&#97;&#117;&#101;r-m&#101;hr&#101;n' + '&#64;';
 addy83932 = addy83932 + '&#105;m&#105;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy83932 + suffix + '\'' + attribs + '>' );
 document.write( addy83932 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Integrative Biomedical Informatics Laboratory
Research Group on Biomedical Informatics (GRIB) - IMIM/UPF</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Research Unit on Biomedical Informatics (GRIB)
IMIM/UPF, C/Dr. Aiguader 88, 08003 Barcelona, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was generated in the framework the EU-ADR
project co-financed by the European Commission through the contract no.
ICT-215847 and the e-TOX project from the European Community's Seventh Framework
Programme (FP7/2007-2013) for the Innovative Medicine Initiative under grant
agreement n° 115002. The Research Unit on Biomedical Informatics (GRIB) is a
node of the Spanish National Institute of Bioinformatics (INB) and member of
the COMBIOMED network. We thank the Departament d’Innovació, Universitat i
Empresa (Generalitat de Catalunya) for a grant to author ABM. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection36">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745807">G-35. Combination of network topology and pathway
analysis to reveal functional modules in human disease</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bauer-Mehren A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bauer-Mehren A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Bundschus M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bundschus M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Rautschka M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rautschka M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mayer MA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mayer MA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sanz F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sanz F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Furlong LI<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Furlong LI&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Most human diseases arise due to interactions among
multiple genetic variants and environmental factors. In the last years, several
gene-disease association databases have been developed; nevertheless much
information is still locked in the literature. To get a comprehensive view of
the full spectrum of human diseases with genetic origin including monogenic,
complex and environmental diseases, data integration is required. Network representation
combined with functional analysis can be used to elucidate functional modules
related to human diseases and to understand their underlying mechanisms.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">A comprehensive database comprising gene-disease
associations from expert-curated repositories and a text-mining derived network
was developed. The integration required disease and gene vocabulary mapping and
development of a gene-disease association ontology. We represent gene-disease
associations as bipartite-graphs and generate disease and gene centric views of
the data by means of network projection. Network analysis, graph-clustering
approaches and functional annotation to GO terms and biological pathways were
used to explore global properties of the data or to focus on specific diseases.</p>

<h3>Results</h3>

<p class="abresults">Integration of data from diverse sources highlighted small
overlap of genes, diseases and their associations. Most diseases have more
associated genes than reported in a single database, even for diseases regarded
as monogenic. We identified phenotypically derived gene clusters with varying
degree of functional homogeneity. For most clusters more than one biological
process was found suggesting a putative role of cross-talks of pathways in
disease development. In addition, gene products of clusters characterized by
single biological process were involved in direct physical interactions.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Network analysis confirmed the need of integrating
gene-disease associations from diverse sources to obtain a comprehensive
picture of genetic basis of human diseases. Clustering and pathway analysis of
disease-related processes can give insights into the underlying mechanisms
which can help in finding new candidate disease genes, explanations for drug
toxicity and eventually support development of new treatment strategies. Based
on cluster and pathway enrichment analysis, a possible explanation for the
toxicity on skeletal muscle of the drug Perhexilin is proposed.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://ibi.imim.es/DisGeNET/DisGeNETweb.html" mce_href="http://ibi.imim.es/DisGeNET/DisGeNETweb.html">http://ibi.imim.es/DisGeNET/DisGeNETweb.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anna Bauer-Mehren (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16193 = '&#97;b&#97;&#117;&#101;r-m&#101;hr&#101;n' + '&#64;';
 addy16193 = addy16193 + '&#105;m&#105;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy16193 + suffix + '\'' + attribs + '>' );
 document.write( addy16193 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46024 = '&#97;b&#97;&#117;&#101;r-m&#101;hr&#101;n' + '&#64;';
 addy46024 = addy46024 + '&#105;m&#105;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy46024 + suffix + '\'' + attribs + '>' );
 document.write( addy46024 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64451 = '&#97;b&#97;&#117;&#101;r-m&#101;hr&#101;n' + '&#64;';
 addy64451 = addy64451 + '&#105;m&#105;m' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy64451 + suffix + '\'' + attribs + '>' );
 document.write( addy64451 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Integrative Biomedical Informatics Laboratory
Research Group on Biomedical Informatics (GRIB) - IMIM/UPF</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Research Unit on Biomedical Informatics (GRIB)
IMIM/UPF, C/Dr. Aiguader 88, 08003 Barcelona, Spain (2) Institute for Computer
Science, Ludwig-Maximilians-University Munich, Oettingenstr. 67, 80538 Munich,
Germany (3) Siemes AG, Corporate Technology, Information and Communications,
Otto-Hahn-Ring 6, 81739 Munich, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was generated in the framework the EU-ADR
project co-financed by the European Commission through the contract no.
ICT-215847 and the e-TOX project from the European Community's Seventh
Framework Programme (FP7/2007-2013) for the Innovative Medicine Initiative
under grant agreement n° 115002. The Research Unit on Biomedical Informatics
(GRIB) is a node of the Spanish National Institute of Bioinformatics (INB) and
member of the COMBIOMED network. We thank the Departament d’Innovació,
Universitat i Empresa (Generalitat de Catalunya) for a grant to author ABM. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection37">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745808">G-36. Attacking interface &amp; interaction
networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Engin B<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Engin B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Gursoy A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gursoy A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Keskin O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Keskin O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Protein-protein interactions (PPIs) are crucial in almost
all biological processes such as cell regulation, signal transduction, gene
replication, or translation. Understanding interactions between proteins is
very important to explore biological processes, as well as solving the
mysteries of disease mechanisms in organisms. Representation of interaction
data by PPI networks enables us to utilize well-known and well-studied graph
theoretical approaches to analyze protein-protein interaction networks where
nodes represent proteins and the edges between them represent the interactions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">“Interface &amp; Interaction Networks” result from
integration of binding site information into PPI networks. This representation
depicts proteins as nodes, interactions as edges and interfaces as a different
kind of node. Protein interactions are carried over interface nodes. Here
attacks are performed to interfaces, which mean deleting all similar interfaces
to a chosen target interface (in order to mimic drug mechanism) and associated
interactions from the network.</p>

<h3>Results</h3>

<p class="abresults">We performed our analysis on p53 pathway and observed that
among various target selection strategies the most effective one is the
selection of the interfaces by the number of their occurrences. This is logical
because more the interface is observed, the more the other interfaces will be
affected.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Complex Networks is an important paradigm in biology
since protein-protein interactions are generally abstracted with graph
representation. In order to improve ways of defining multiple targets for
partial attacks we introduced structural information to interaction networks.
Resulting network model details the interaction information, it also includes
the interface(binding site) information which are used by interacting proteins.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Billur Engin (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49414 = 'b&#101;ng&#105;n' + '&#64;';
 addy49414 = addy49414 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy49414 + suffix + '\'' + attribs + '>' );
 document.write( addy49414 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31258 = 'b&#101;ng&#105;n' + '&#64;';
 addy31258 = addy31258 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy31258 + suffix + '\'' + attribs + '>' );
 document.write( addy31258 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7697 = 'b&#101;ng&#105;n' + '&#64;';
 addy7697 = addy7697 + 'k&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy7697 + suffix + '\'' + attribs + '>' );
 document.write( addy7697 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Koc University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Koc University, Istanbul, Turkey</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection38">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745809">G-37. Pathway Projector: web-based zoomable pathway
browser using KEGG Atlas and Google Maps API</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kono N<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Kono N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*), Arakawa K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Arakawa K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ogawa R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ogawa R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kido N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kido N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Oshita K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Oshita K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ikegami K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ikegami K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tamaki S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tamaki S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Tomita M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tomita M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Biochemical pathways provide an essential context for
understanding the experimental data and the systematic workings of a cell.
Therefore, the availability of online pathway browsers will facilitate
post-genomic research, just as genome browsers have contributed to genomics.
Many pathway maps have been provided online as part of public pathway
databases. Most of these maps, however, function as the gateway interface to a
specific database, and the comprehensiveness of their represented entities,
data mapping capabilities, and user interfaces are not always sufficient for
generic usage.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The pathway map of Pathway Projector is based on
the KEGG Atlas map, for the familiarity of its layout and for the availability
of various analysis tools. However, because the KEGG Atlas only represents
metabolite nodes, we added all gene and enzyme nodes semi-automatically on the
reference pathway map. The software was implemented using AJAX (Asynchronous
JavaScript + XML) programming paradigm and the main interface framework was
built with Ext JS library. For the representation of the global pathway map, we
adopted zoomable user interface (ZUI) using Google Maps API through G-language
GAE.</p>

<h3>Results</h3>

<p class="abresults">Here we developed a novel web-based pathway browser named
the Pathway Projector, which allows browsing of the integrated pathway map with
intuitive ZUI. Pathway Projector is based on the integrated metabolic pathway
map of KEGG Atlas by adding gene and enzyme nodes, and uses Google Maps API for
intuitive zooming. Pathway Projector has four types of search functionalities,
including those by keywords and identifiers, by molecular mass, by possible
routes between two metabolites using PathComp, and by sequence similarity using
BLAST. It also has functionality for data mapping and manual editing.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The understanding of omics layers is important for
systems biology, and biochemical pathways provides a necessary context for this
purpose. Since pathways do not exist independently, but are rather
interconnected in vivo, use of an integrated map is desirable, especially for
the mapping of comprehensive experimental data. Pathway Projector has an
intuitive interface for such integrated global pathway maps. Moreover,
capabilities of this software such as searching, editing, annotation, mapping
and links to various databases, will be a useful gateway for pathway analysis.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.g-language.org/PathwayProjector/" mce_href="http://www.g-language.org/PathwayProjector/">http://www.g-language.org/PathwayProjector/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Nobuaki Kono (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83045 = 'c&#105;c&#111;n&#105;&#97;' + '&#64;';
 addy83045 = addy83045 + 'sfc' + '&#46;' + 'k&#101;&#105;&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy83045 + suffix + '\'' + attribs + '>' );
 document.write( addy83045 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69316 = 'c&#105;c&#111;n&#105;&#97;' + '&#64;';
 addy69316 = addy69316 + 'sfc' + '&#46;' + 'k&#101;&#105;&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy69316 + suffix + '\'' + attribs + '>' );
 document.write( addy69316 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56057 = 'c&#105;c&#111;n&#105;&#97;' + '&#64;';
 addy56057 = addy56057 + 'sfc' + '&#46;' + 'k&#101;&#105;&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy56057 + suffix + '\'' + attribs + '>' );
 document.write( addy56057 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute for Advanced Biosciences, Keio University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Institute for Advanced Biosciences, Keio University 2.
Nara Institute of Science and Technology</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research was supported in part by the Grant-in-Aid
for JSPS Fellows and the Grant-in-Aid for Young Scientists (A), No.222681029,
2010, from the Japan Society for the Promotion of Science (JSPS), as well as by
funds from the Yamagata Prefectural Government and Tsuruoka City. The funders
had no role in study design, data collection and analysis, decision to publish,
or preparation of the manuscript. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection39">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745810">G-38. Mapping the "Farnesylome":
structure-based prediction of farnesyltransferase targets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">London N<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;London N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Schueler-Furman N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schueler-Furman N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*)</p>

<p class="abmotivation">Farnesylation by farnesyltransferase (FTase) targets
proteins to the membrane. We propose a structural modeling scheme to account
for peptide specificity. We obtain a very good discrimination (AUC = 0.88/0.91
on training/test sets), and identify 85% of known targets. A genomic scan
reveals 77 novel putative substrates undetected by sequence-based methods. These
are currently validated by experiment. Our approach can easily be adapted to
additional systems, and is expected to contribute significantly to the
elucidation of the cellular network of peptide-mediated interactions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Given a Cxxx sequence, we wish to predict whether
it would bind FTase, using the following, Rosetta-based protocol: Starting from
the solved structure, we find the optimal rotameric state within the binding
pocket for a given peptide sequence. We minimize all of the peptide degrees of
freedom while constraining two conserved hydrogen bonds and the ZN
coordination. The energy contributions of the peptide residues is used as score
for its classification as a binder or non-binder. To identify potential new
targets, we then screen genomes for Cxxx peptides and score these.</p>

<h3>Results</h3>

<p class="abresults">We chose a set of peptide sequences tested for FTase
activity to devise our prediction protocol (AUC=0.88). We defined a threshold which
yields 69% sensitivity with a FPR of only 8%. On an independent test set, we
obtain an even better AUC (0.91), and the learnt threshold provides a TPR of
86% and FPR of 12%. This result gives us confidence in predicting novel
sequences. Using a stringent score threshold, we scanned the Human genome and
identified 167 potential targets, and those not identified by sequence-based
approaches (e.g. PrePS) are now tested experimentally in the Fierke Lab.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Elucidation of FTase targets has significant implications
for signal transduction research and new therapeutics. We present a generic,
robust and accurate protocol for peptide binding specificity prediction, which
complements sequence based approaches and detects novel targets. Our structure
based approach can be easily adapted to other systems and will expand our
knowledge of the cellular peptide-protein interaction network.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ora Schueler-Furman (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33297 = '&#111;r&#97;f' + '&#64;';
 addy33297 = addy33297 + '&#101;kmd' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy33297 + suffix + '\'' + attribs + '>' );
 document.write( addy33297 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23200 = '&#111;r&#97;f' + '&#64;';
 addy23200 = addy23200 + '&#101;kmd' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy23200 + suffix + '\'' + attribs + '>' );
 document.write( addy23200 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94259 = '&#111;r&#97;f' + '&#64;';
 addy94259 = addy94259 + '&#101;kmd' + '&#46;' + 'h&#117;j&#105;' + '&#46;' + '&#97;c' + '&#46;' + '&#105;l';
 document.write( '<a ' + path + '\'' + prefix + addy94259 + suffix + '\'' + attribs + '>' );
 document.write( addy94259 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Hebrew University, Institute for Medical Research
IMRIC</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Microbiology and Molecular Genetics,
Institute for Medical Research IMRIC, The Hebrew University.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Israel Science Foundation Grant No. 306/6</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection40">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745811">G-39. Integrative analysis of gene expression and
copy number variation data to elucidate a reference human gene network</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Orsini M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Orsini M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Pinna A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pinna A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Leo V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Leo V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), de la Fuente A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;de la Fuente A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Understanding diseases requires identifying the
differences between healthy and affected tissues also in terms of regulatory
networks that are dysfunctional in a given disease state. Gene expression and
CNV data are currently available in public DBs. From gene-expression data
alone, it is possible to compile Gene Co-expression Networks for both the
healthy and affected samples, and then compare them to identify subnetworks
that potentially underlie the disease phenotype. Gene Co-expression Networks
are undirected networks, and thus lack the causal directions for the effects
between genes</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Raw measurement, sample properties and clinical
information (healthy, disease state, tumor, biopsy, stadiation, treatment,etc)
were obtained from public repositories. To limit dis-homogeneity, only data
from Affy platforms were recovered. We explicitly considered only those sample
were both CNV and gene expression measurement are available simultaneously. We
stored these data in a database (IntegromicsDB) for integrative biology
studies. Simulated gene expression data were generated with MATLAB using
Ordinary Differential Equations based on biochemical kinetics.</p>

<h3>Results</h3>

<p class="abresults">We drawn algorithms to elucidate Gene Networks thoroughly
evaluated them with simulation studies, and applied these to human data to
compile a “healthy” network. Although correlation does not always implies
causation, if there is significant correlation between the CNV of a gene i and
the gene expression level of gene j, then we can interpret such correlation as
a causal effect of gene i on gene j, as it is unlikely that the gene expression
level of gene j caused the CNV of gene i. They might be correlated due to
confounding by a third factor, but our inference approach recognized such
cases.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The availability of a centralized expression and CNV
data-resource is useful to investigate, common gene expression and CNV profiles
among patients having same disease with similar characteristics. Comparison of
Gene Networks among healthy and sick patients helps to identify genes and
subnetworks involved in a given pathology. Integrative analysis of gene
expression and CNV data allowed us to obtain directed Gene Networks, where
edges represent causal effects between genes. This approach can represent a
starting point to design new integrative and multi-dimensional genetic
strategies of study.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Massimiliano Orsini (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17517 = '&#111;rs&#105;n&#105;' + '&#64;';
 addy17517 = addy17517 + 'crs4' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy17517 + suffix + '\'' + attribs + '>' );
 document.write( addy17517 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79274 = '&#111;rs&#105;n&#105;' + '&#64;';
 addy79274 = addy79274 + 'crs4' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy79274 + suffix + '\'' + attribs + '>' );
 document.write( addy79274 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67715 = '&#111;rs&#105;n&#105;' + '&#64;';
 addy67715 = addy67715 + 'crs4' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy67715 + suffix + '\'' + attribs + '>' );
 document.write( addy67715 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CRS4 bioinformatics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">CRS4 Bioinformatica, c/o Parco Tecnologico della
Sardegna, Edificio 3, Loc. Piscina Manna, 09010 PULA (CA), ITALY</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection41">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745812">G-40. Unknown player in modelling of signal
transduction pathway</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sadeh M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Sadeh M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Anchang B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Anchang B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Spang R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Spang R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Nested Effects Models are a class of probabilistic models
introduced to analyze the effects of gene perturbation screens visible in high-dimensional
phenotypes like microarrays. NEMs require expression profiles of perturbation
experiments of all signalling genes in a pathway. In many cases one does not
know all these genes or one does not even know whether one knows them all.
There is virtually always the possibility that unknown pathway players exist
and that important links of the pathway are missing in our current knowledge.
The propose method is useful to handle this situation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In signallling pathway, there is virtually always
the possibility that unknown pathway players exist and that important links of
the pathway are missing in our current knowledge. This means that we might
model too many genes including some that do not belong to the pathway. We
hypothesize by considering an additional but unknown signalling Silencing gene
in the model for which no data is available, we can still obtain better
predictive performance of the cells response to treatment compared with the
optimal model without this gene.</p>

<h3>Results</h3>

<p class="abresults">We started the investigation of this idea in the context of
a simulated random five gene network. In order to have an comprehensive
conclusion, this method has been applied for 100 random networks (graph) with 5
genes (nodes). Nested Effects Models was applied with for noisy dataset which
are generated from these networks. The results show, after removing the
information of one random gene in networks, we would obtain better prediction
performance by considering an additional but unknown signalling gene in the
model for which no data is available.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We introduced a method to address one specific problems
which are central the analysis of interventional data in biology: Does the
expression data used for Nested Effects Models contain the information that
unknown players must exist? The proposal method tries to find a reasonable
pattern in simulation study in order to use it in a real data scenario. So far
we have investigated this idea in the context of simulated five genes networks,
where one gene is not known. This investigation suggest a couple of new ideas
in signalling network reconstruction.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Mohammad Sadeh (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50677 = 'm&#111;h&#97;mm&#101;d.s&#97;d&#101;h' + '&#64;';
 addy50677 = addy50677 + 'kl&#105;n&#105;k' + '&#46;' + '&#117;n&#105;-r&#101;g&#101;nsb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy50677 + suffix + '\'' + attribs + '>' );
 document.write( addy50677 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40262 = 'm&#111;h&#97;mm&#101;d.s&#97;d&#101;h' + '&#64;';
 addy40262 = addy40262 + 'kl&#105;n&#105;k' + '&#46;' + '&#117;n&#105;-r&#101;g&#101;nsb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy40262 + suffix + '\'' + attribs + '>' );
 document.write( addy40262 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56379 = 'm&#111;h&#97;mm&#101;d.s&#97;d&#101;h' + '&#64;';
 addy56379 = addy56379 + 'kl&#105;n&#105;k' + '&#46;' + '&#117;n&#105;-r&#101;g&#101;nsb&#117;rg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy56379 + suffix + '\'' + attribs + '>' );
 document.write( addy56379 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute for Functional Genomics Computational
Diagnostics Group University of Regensburg</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Institute for Functional Genomics Computational
Diagnostics Group University of Regensburg</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection42">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745813">G-41. Assembly of protein complexes by integrating
graph clusterings</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Chin C-H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Chin C-H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 4, *), Chen S-H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chen S-H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Chen C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chen C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->-Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chen C-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Hsiung C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hsiung C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ho C-W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ho C-W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Ko M-T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ko M-T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lin C-Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lin C-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2, 3, 5)</p>

<p class="abmotivation">Proteins in a complex are organized for particular
functions, such as synthesis of biomolecules or protein degradation. The
complex is similar to the idea of “community” of a network, in which vertices
are joined tightly in between, while loosely out-connected. There exist many
community detection methods. However, there is no suitable method for
integrating them at present. For grasping more comprehensive community
structures, we propose a framework to integrate different clustering methods' results
to generate a better one.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">There are two types of graph clustering algorithms:
global clustering and local clustering. A clustering method is a global
clustering if each vertex of the input graph is assigned a cluster in the output
of the method; and a clustering method is a local clustering if the cluster
assignments are only performed for a certain subset of vertices. Naturally,
each type has both strengths and weaknesses. Here, we design a measure to judge
the quality of a cluster as a community, and propose a framework to integrate
different clustering results.</p>

<h3>Results</h3>

<p class="abresults">First, we applied six clustering methods on a yeast's
protein-protein interaction network downloaded from DIP, and computed their
integration. Then, we validated these results by comparing them with Gene
Ontology annotations and two reference sets from MIPS and Aloy et al. The
validation shows that our framework works. After checking the biological
significance of the integration, we found that the result can successfully
identify undiscovered components and decipher common submodules inside these
complexes like RNA polymerases I, II, III on yeast interactome.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In local clustering methods, some protein components may
not be classified into the resulting clusters due to the incomplete input PPI
network structure. Therefore, the recall of a local clustering method is
usually lower than a global clustering method. However, the precision of a
local clustering method is often higher than a global clustering because a
global clustering method does not classify "intermediate" vertices
well. We show that a better result could be derived when we integrate the two
types of methods properly. A user-friendly web service for this research is
also online.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://hub.iis.sinica.edu.tw/clustering/" mce_href="http://hub.iis.sinica.edu.tw/clustering/">http://hub.iis.sinica.edu.tw/clustering/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Chia-Hao Chin (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28988 = 'j&#111;v&#105;c&#101;' + '&#64;';
 addy28988 = addy28988 + '&#105;&#105;s' + '&#46;' + 's&#105;n&#105;c&#97;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tw';
 document.write( '<a ' + path + '\'' + prefix + addy28988 + suffix + '\'' + attribs + '>' );
 document.write( addy28988 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76486 = 'j&#111;v&#105;c&#101;' + '&#64;';
 addy76486 = addy76486 + '&#105;&#105;s' + '&#46;' + 's&#105;n&#105;c&#97;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tw';
 document.write( '<a ' + path + '\'' + prefix + addy76486 + suffix + '\'' + attribs + '>' );
 document.write( addy76486 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy75682 = 'j&#111;v&#105;c&#101;' + '&#64;';
 addy75682 = addy75682 + '&#105;&#105;s' + '&#46;' + 's&#105;n&#105;c&#97;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tw';
 document.write( '<a ' + path + '\'' + prefix + addy75682 + suffix + '\'' + attribs + '>' );
 document.write( addy75682 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Academia Sinica, Taiwan</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Information Science, Academia Sinica,
No. 128 Yan-Chiu-Yuan Rd., Sec. 2, Taipei 115, Taiwan. (2) Division of
Biostatistics and Bioinformatics, National Health Research Institutes. No. 35
Keyan Rd. Zhunan, Miaoli County 350, Taiwan. (3) Institute of Fisheries Science,
College of Life Science, National Taiwan University, No. 1, Roosevelt Rd. Sec
4, Taipei, Taiwan. (4) Department of Computer Science and Information
Engineering, National Central University, No.300, Jung-da Rd, Chung-li,
Tao-yuan 320, Taiwan. (5) Research Center of Information Technology Innovation,
Academia Sinica, No. 128 Yan-Chiu-Yuan Rd., Sec. 2, Taipei 115, Taiwan.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The authors would like to thank National Science
Council (NSC), Taiwan, for financially supporting this research through NSC
98-2221-E-008-079 to Ho C-W<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ho C-W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, 98-3112-B-400-010- and
98-2221-E-001-018- to Lin C-Y<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Lin C-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection43">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745814">G-42. How natural host species avoid CD4+ T cell
depletion during SIV infection</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Chan ML<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Chan ML&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2*), Petravic J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Petravic J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ortiz AM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ortiz AM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Engram J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Engram J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Paiardini M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Paiardini M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Cromer D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cromer D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Silvestri G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Silvestri G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,5), Davenport MP<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Davenport MP&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Natural hosts of SIV such as Sooty Mangabeys (SM) remain
largely asymptomatic when infected. This is in sharp contrast with non-natural
hosts, such as Rhesus Macaques (RM) and HIV-infected individuals, who experience
a progressive decline of CD4+ T cells in the blood and lymphoid tissues, and
develop AIDS. Currently, the lack of disease progression in natural hosts is
still not clearly understood. A better understanding of why SIV is not
pathogenic in natural hosts will in turn provide valuable insights into the
pathogenesis of AIDS in HIV-infected individuals.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Experimental data show that as CD4+ T cells are
depleted during infection, SM show only a small increase in the proportion of
proliferating CD4+ T cells compared to larger increases in RM and HIV-infected
individuals. This suggests that disease progression is associated with marked
differences in the proliferation level of CD4+ T cells in response to CD4+ T
cell depletion. We have developed a model that demonstrates the relationship
between proliferation and disease outcome to help us understand why SIV is not
pathogenic in SM, and otherwise in RM and HIV-infected individuals.</p>

<h3>Results</h3>

<p class="abresults">We modelled the disease course in a SM- and a human-like
host, which only differed in their maximal CD4+ T cell proliferation level.
Since dividing cells are preferentially infected, the decrease in the
proliferative response to CD4+ T cell depletion in the SM-like host led to a
large increase in the total number of uninfected CD4+ T cells present in
chronic infection. The model therefore suggests that a lower proliferative
response to CD4+ T cell depletion in non-pathogenic SIV infection paradoxically
leads to the preservation of CD4+ T cell counts during chronic infection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">CD4+ T cell proliferation plays a dual role during
infection: Increasing it will act to replace CD4+ T cells and restore immunity,
but this also generates more cells that are susceptible to infection. The rapid
CD4+ T cell proliferation in RM fuels the fire of infection by producing a
large proportion of proliferating cells that are susceptible to infection.
Conversely, the moderate homeostatic proliferation in SM is optimal for the
preservation of CD4+ T cells during chronic infection. Hence SMs may have
adapted to infection by attenuating their proliferative response to CD4+ T cell
depletion</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ming Liang Chan (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65940 = 'd&#97;n&#105;&#101;l_ch&#97;n' + '&#64;';
 addy65940 = addy65940 + 'st&#117;d&#101;nt' + '&#46;' + '&#117;nsw' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy65940 + suffix + '\'' + attribs + '>' );
 document.write( addy65940 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40519 = 'd&#97;n&#105;&#101;l_ch&#97;n' + '&#64;';
 addy40519 = addy40519 + 'st&#117;d&#101;nt' + '&#46;' + '&#117;nsw' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy40519 + suffix + '\'' + attribs + '>' );
 document.write( addy40519 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79708 = 'd&#97;n&#105;&#101;l_ch&#97;n' + '&#64;';
 addy79708 = addy79708 + 'st&#117;d&#101;nt' + '&#46;' + '&#117;nsw' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy79708 + suffix + '\'' + attribs + '>' );
 document.write( addy79708 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">School of Medical Sciences, University of New South
Wales</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Complex Systems in Biology Group, Centre for
Vascular Research, University of New South Wales, Kensington, NSW, Australia 2.
School of Medical Sciences, Faculty of Medicine, University of New South Wales,
Kensington, NSW, Australia 3. Department of Pathology and Laboratory Medicine,
University of Pennsylvania, PA, USA 4. Department of Mathematics and Centre for
Integrative Systems Biology, Imperial College London, London, UK 5. Yerkes
National Primate Research Center, Emory University, Atlanta, GA, USA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">NIH-AI-66998, HL-75766, NHMRC (Australia), ARC
(Australia), UNSW-UIPA (Australia) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection44">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745815">G-43. Kinase-specific phosphorylation site
prediction in Arabidopsis thaliana</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dang T-H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Dang T-H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Verschoren A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Verschoren A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Laukens K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laukens K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Reversible phosphorylation of proteins plays critical
roles in a diverse range of signaling pathways. Phosphorylated sites are
usually experimentally identified by Mass Spectrometry based techniques which
are often time-consuming, labor-intensive and expensive. Although Arabidopsis
thaliana is a model plant and thus a significant focus of international plant
research, the knowledge about phosphorylation processes in Arabidopsis thaliana
is still limited. The Arabidopsis genome encodes at least two times more
protein kinases than the human genome (Manning et al, 2002).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The method uses the InParanoid algorithm (O'Brien
et al., 2005) to find kinases in Arabidopsis thaliana which are orthologous to
extracted kinases belonging to kinase groups in the Phospho.ELM database. It
then uses Pfam (Finn et al., 2010) to discard candidates that do not contain
any kinase catalytical domains. Subsets of known residues phosphorylated by
kinase groups in other species were extracted. From those the CRPhos models
(Dang et al., 2008) were trained and used to make a prediction for all protein
sequences in the Arabidopsis thaliana’s genome.</p>

<h3>Results</h3>

<p class="abresults">The prediction results were compared with those from
previous studies (Sugiyama et al. 2008, Heazlewood et al. 2008). They both do
not provide any information about responsible kinases, and they overlap only
partially. A statistically significant overlap between predicted sites and
those from the two databases was found, and it increases proportionally to the
increased FPR, as expected. Applying the method results in a predicted
phosphorylation network for which is further analysed in detail.</p>

<h3>Discussion</h3>

<p class="abdiscussion">To our knowledge, the proposed method is the first
attempt to predict the genome-wide phosphorylation network in Arabidopsis
thaliana. The first results demonstrate that the approach based on orthologous
protein kinases across multiple species is useful, in particular for organisms
for which existing data on phosphorylations is limited, like in Arabidopsis
thaliana. The predicted results are a solid basis for further computational
analysis and experimental validations.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Thanh Hai Dang (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90509 = 'th&#97;nhh&#97;&#105;.d&#97;ng' + '&#64;';
 addy90509 = addy90509 + '&#117;&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy90509 + suffix + '\'' + attribs + '>' );
 document.write( addy90509 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3693 = 'th&#97;nhh&#97;&#105;.d&#97;ng' + '&#64;';
 addy3693 = addy3693 + '&#117;&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy3693 + suffix + '\'' + attribs + '>' );
 document.write( addy3693 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78768 = 'th&#97;nhh&#97;&#105;.d&#97;ng' + '&#64;';
 addy78768 = addy78768 + '&#117;&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy78768 + suffix + '\'' + attribs + '>' );
 document.write( addy78768 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Intelligent Systems Laboratory (ISLab), Department of
Mathematics and Computer Science, University of Antwerp</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Intelligent Systems Laboratory (ISLab), Department
of Mathematics and Computer Science, University of Antwerp</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">SBO grant (IWT-600450, Bioframe) of the Flemish
Institute supporting Scientific-Technological Research in Industry</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection45">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745816">G-44. BiologicalNetworks: enabling systems-level
studies of host-pathogen interactions</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kozhenkov S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kozhenkov S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sedova M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sedova M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Dubinina Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dubinina Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ponomarenko J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ponomarenko J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Baitaluk M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Baitaluk M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Understanding of immune response mechanisms of
pathogen-infected host requires multi-scale analysis of genome-wide data. Data
integration methods have proved useful to the study of biological processes in
model organisms, but their systematic application to the study of host immune
system response to a pathogen and human disease is still in the initial stage.
To study host-pathogen interaction on the systems biology level, an extension
to the previously described BiologicalNetworks (1) and IntegromeDB (2) is
proposed.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Data integration and mapping to the internal
database is fully automated and based on Semantic Web technologies and Web
Ontology Language. The system represents a general-purpose graph warehouse with
its own data definition and query language, augmented with data types for
biological entities. The list of integrated databases is at
http://www.biologicalnetworks.net/Database/tut5.php. Bioinformatics methods
were implemented that allow to reconstruct/modify phylogenetic trees, get
multiple sequence alignments, identify phylogenetically conserved transcription
factor binding sites, and other.</p>

<h3>Results</h3>

<p class="abresults">The developed system has been applied to the systems-level
analysis of the influenza virus-host interactions, including host molecular
pathways that are induced/repressed during the infections, co-expressed genes,
and conserved transcription factor binding sites. Previously unknown to be
associated with the influenza infection genes were identified and suggested for
further investigation as potential drug targets.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The developed methods and data integration and querying
tools allow simplifying and streamlining the process of integration of diverse
experimental data types, including molecular interactions and phylogenetic
classifications, genomic sequences and protein structure information, gene
expression and virulence data for pathogen-related studies. The data can be
integrated from the databases and user’s files for public use. References: 1)
Baitaluk M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Baitaluk M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, et al. (2006) Nucleic Acids
Res. 34: W466. 2) Baitaluk M, Ponomarenko J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ponomarenko J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2010) Bioinformatics,
doi:10.1093/bioinformatics/btq231.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Julia Ponomarenko (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17021 = 'jp&#111;n' + '&#64;';
 addy17021 = addy17021 + 'sdsc' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy17021 + suffix + '\'' + attribs + '>' );
 document.write( addy17021 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83830 = 'jp&#111;n' + '&#64;';
 addy83830 = addy83830 + 'sdsc' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy83830 + suffix + '\'' + attribs + '>' );
 document.write( addy83830 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy84658 = 'jp&#111;n' + '&#64;';
 addy84658 = addy84658 + 'sdsc' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy84658 + suffix + '\'' + attribs + '>' );
 document.write( addy84658 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of California San Diego</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) San Diego Supercomputer Center, (2) Skaggs School
of Pharmacy and Pharmaceutical Sciences, University of California San Diego,
9500 Gilman Drive, La Jolla, CA, 92093, USA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The US National Institute of Health grants R01GM084881
and R01GM085325. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection46">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745817">G-45. Genome-wide identification of disease-related
genes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Jaeger S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Jaeger S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Leser U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Leser U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">For many human diseases it is not yet known which genes
are involved in the pathogenesis of the diseases. Elucidating the underlying
disease mechanisms is crucial for understanding the onset of diseases, and the
development of specific diagnostic and therapeutic approaches. Several
computational methods have been devised for identifying disease-related genes.
However, the majority focuses on prioritizing candidates within defined
chromosomal regions. These approaches fail short in appropriately handling
diseases where the associated loci are still unknown and diseases without
genetic origin.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We present a computational framework to identify
novel disease genes in a genome-wide setting. For a given disease, we extract
all genes that are known to be associated with this disease. We compile
disease-specific networks by integrating directly and indirectly linked gene
products using protein interaction as well as manually curated and predicted
functional data. Network centrality analysis is applied to rank genes according
to their relevance for the given disease. In addition, a simple normalization
strategy is employed to adjust the ranking for a bias toward global hubs in the
networks.</p>

<h3>Results</h3>

<p class="abresults">We validate our method using disease-gene association data
from OMIM. We show that using predicted functions allows for an improved
ranking of proteins that are uncharacterized in first place by increasing the
recovery rate of disease proteins from 80% to 86%. Cross-validation is used to
simulate a genome-wide candidate search for diseases where no associated
chromosomal regions are known. We show that using predicted functional data and
including indirectly linked genes in combination with a proper score
normalization improves the genome-wide identification of disease genes from 29%
to 51%.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We developed a linkage interval-independent framework for
identifying disease genes using protein-protein interaction data and functional
annotations in combination with network centrality analysis. Our approach does
not depend on the availability of associated chromosomal regions, which makes
it applicable to a much wider range of diseases than previous algorithms, such
as disorders with very few or even only a single known disease protein,
diseases with multiple, very large, or no associated loci, and also diseases
without genetic origin.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Samira Jaeger (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy952 = 'sj&#97;&#101;g&#101;r' + '&#64;';
 addy952 = addy952 + '&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + 'h&#117;-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy952 + suffix + '\'' + attribs + '>' );
 document.write( addy952 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23 = 'sj&#97;&#101;g&#101;r' + '&#64;';
 addy23 = addy23 + '&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + 'h&#117;-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy23 + suffix + '\'' + attribs + '>' );
 document.write( addy23 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy30681 = 'sj&#97;&#101;g&#101;r' + '&#64;';
 addy30681 = addy30681 + '&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + 'h&#117;-b&#101;rl&#105;n' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy30681 + suffix + '\'' + attribs + '>' );
 document.write( addy30681 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Humboldt-Universtität zu Berlin</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Knowledge Management in Bioinformatics,
Humboldt-Universität zu Berlin, Unter den Linden 6, 10099 Berlin, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is funded by an Elsa-Neumann scholarship. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection47">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745818">G-46. Bioinformatics tools for the investigation of
proteomics aspects involved in cardiovascular diseases: from biomarker
discovery to protein-protein interaction networks</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Di Silvestre D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Di Silvestre D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Brambilla F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brambilla F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Brunetti P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brunetti P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lionetti V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lionetti V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Agostini S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Agostini S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Cavallini C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cavallini C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Mauri PL<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mauri PL&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">An approach based on systems biology implies the
examination of a set of multiple elements. In this context, proteins, which
represent the effector molecules for all cell functions, play a central role.
In the last few years, mass spectrometry-based (MS) proteomics, such as MudPIT
(Multidimensional Protein Identification Technology), have become the leading
approach employed in high-throughput analysis. At the same time, the great
amount of data produced by MS experiments, requires the development of
computational tools for data processing and handling.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The MudPIT approach has been applied to Sus Scrofa
cardiac tissues affected by myocardial infarction. Protein profiles of analyzed
samples have been achieved using the SEQUEST algorithm in combination with the
Homo sapiens protein database. In order to identify biomarkers and proteotypic
peptides, the MAProMa and EPPI software have been employed, respectively. In
addition, protein profiles and biomarkers were further processed by means of
cluster analysis and to investigate protein-protein interaction networks using
platforms such as Cytoscape.</p>

<h3>Results</h3>

<p class="abresults">The combination of the data obtained from all MudPIT
experiments allowed the identification of 1500 different proteins and 4800
peptides. The protein profiles identified were found to be useful in cluster
analysis of analyzed tissues, which grouped according to their specific
biological condition. Finally, the investigation of the protein-protein
interaction network allowed the identification of sub-networks potentially
involved in disease state and, at the same time, the visualization of a large
amount of data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Employment of bioinformatics tools to handle and process
the vast amount of data produced by “omics” technologies is fundamental to
increase our knowledge of the biological systems investigated. In particular,
the identification of biomarkers and the characterization of biological systems
as scale free networks proved to be useful in the elucidation of mechanisms
indicative of diseased phenotypes.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Dario Di Silvestre (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65403 = 'd&#97;r&#105;&#111;.d&#105;s&#105;lv&#101;str&#101;' + '&#64;';
 addy65403 = addy65403 + '&#105;tb' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy65403 + suffix + '\'' + attribs + '>' );
 document.write( addy65403 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49436 = 'd&#97;r&#105;&#111;.d&#105;s&#105;lv&#101;str&#101;' + '&#64;';
 addy49436 = addy49436 + '&#105;tb' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy49436 + suffix + '\'' + attribs + '>' );
 document.write( addy49436 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61939 = 'd&#97;r&#105;&#111;.d&#105;s&#105;lv&#101;str&#101;' + '&#64;';
 addy61939 = addy61939 + '&#105;tb' + '&#46;' + 'cnr' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy61939 + suffix + '\'' + attribs + '>' );
 document.write( addy61939 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute for Biomedical Technologies - National
Research Council</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)- Institute for Biomedical Technologies - CNR,
Segrate (Milan), Italy. (2)- Scuola Superiore Sant'Anna - Sector of Medicine,
Pisa, Italy. (3)- University Hospital S. Orsola Malpighi - Cardiology Department,
Bologna, Italy. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection48">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745819">G-47. ABC database: the Analysing Biomolecular
Contacts database</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Walter P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Walter P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Metzger J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Metzger J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Helms V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Helms V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Protein-protein interactions are essential for most
cellular processes such as signal transduction and immune response which makes
them attractive targets for a wide range of pharmaceutical interventions. Thus,
it is highly desirable to understand the molecular details of protein-protein
contacts as well as contacts between proteins and small molecules. Studying
such interactions is greatly facilitated by convenient simultaneous access to
as much data as possible. For this reason we have designed the ABC database
that provides a broad range of different precomputed interface features.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The protein complexes stored in our database are
retrieved from the RCSB Protein Data Bank. Besides, we included some additional
information such as CATH and SCOP classification, downloaded from the
corresponding websites. The data import was done using a JAVA routine. The
database and the web interface were created with non-commercial or freely
available tools. The program logic was written in JAVA using Servlets and JSP.
As web server we use Tomcat for the management of Servlets and JSP pages as
well as Apache for the static web content. The relational database was realised
with MySQL.</p>

<h3>Results</h3>

<p class="abresults">The database can be queried by various types of parameters
including criteria such as CATH denotation, polarity and size. After performing
a query the user obtains a list of interfaces that fit the constraints. For
each entry found further information can be displayed including statistics of
the amino acid composition and a list of the residue-residue contacts. It is
also possible to export the retrieved data into text files. We used the database
to compare protein-protein interfaces with protein-small molecule interfaces
according to features like amino acid composition and interface size.</p>

<h3>Discussion</h3>

<p class="abdiscussion">There exist numerous freely available meta-databases that
provide convenient access to either data on protein-protein complexes or on
protein-small molecule complexes taken from the PDB. To our knowledge, no
meta-database so far combines both types of interfaces into one resource. Our
ABC database is therefore a powerful tool allowing novel statistical analyses.
For example, researchers interested in small-molecule modulators or inhibitors
of protein-protein interactions may wish to analyze shared parts of
protein-protein and protein-ligand interfaces.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://service.bioinformatik.uni-saarland.de/abc" mce_href="http://service.bioinformatik.uni-saarland.de/abc">http://service.bioinformatik.uni-saarland.de/abc</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jennifer Metzger (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73099 = 'jm&#101;tzg&#101;r' + '&#64;';
 addy73099 = addy73099 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy73099 + suffix + '\'' + attribs + '>' );
 document.write( addy73099 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32481 = 'jm&#101;tzg&#101;r' + '&#64;';
 addy32481 = addy32481 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy32481 + suffix + '\'' + attribs + '>' );
 document.write( addy32481 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31254 = 'jm&#101;tzg&#101;r' + '&#64;';
 addy31254 = addy31254 + 'b&#105;&#111;&#105;nf&#111;rm&#97;t&#105;k' + '&#46;' + '&#117;n&#105;-s&#97;&#97;rl&#97;nd' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy31254 + suffix + '\'' + attribs + '>' );
 document.write( addy31254 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for Bioinformatics, Saarland University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Center for Bioinformatics, Saarland University</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection49">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745820">G-48. The core and pan metabolism in the
Escherichia coli species</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vieira G<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vieira G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sabarly V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sabarly V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Bourguignon P-Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bourguignon P-Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Durot M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Durot M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Le Fèvre F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Le Fèvre F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Mornico D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mornico D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Vallenet D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vallenet D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bouvet O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bouvet O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Denamur E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Denamur E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Schachter V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schachter V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5), Médigue C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Médigue C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Escherichia coli has a highly dynamic genome, a
plasticity that is an important factor for evolution and pathogenicity. It is
difficult to link genomic elements to phenotype, due to different layers of interaction
and to evolutive convergence. Since most shared genomic elements are linked to
metabolism, is it possible to use metabolism as an integrative layer?
Unfortunately, no frameworks for metabolic network reconstruction of closely
related strains exist yet. Could we provide highly detailed networks,
highlighting links between evolution and metabolism?</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We reconstructed the metabolic networks of 23 E.
coli (8 commensal and 15 pathogenic strains) and 6 Shigella (pathogens). After
having reannotated all genomes, we built all networks using EcoCyc as a primary
high-quality pivot and MetaCyc. We computed and compared the core/pan genome
and metabolism for all strains. We reconstructed the phylogenetic (metabolic)
tree by maximum of likelihood (neighbour joining). We computed the intra- and
inter-group phylogenetic distance. We conducted a Multiple Correspondence
Analysis and a Classification And Regression Tree on the reactions.</p>

<h3>Results</h3>

<p class="abresults">The quality of the annotation (i) and the introduction of a
pivot (ii) increased the network quality: i) Increasing the number of reactions
by 24%. ii) Decreasing the number of reactions without associated genes by 34
%. On average the networks possessed 1491 reactions, half of them were common
to all the strains, biosynthesis processes being the most conserved. The
different analyses showed a strong correlation between phylogenetic groups and
metabolic networks. We also found reactions linked to pathogenicity, some
already known and others requiring further investigation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">By propagating specific knowledge from annotation and
benefiting from expert curation made on the pivot, we were able to reconstruct
detailed networks. Those networks closely mirror the phylogeny. Metabolism is
thus a way to link genome and phenotype; we found metabolic processes specific
to each phylogenetic group. Among those candidates, some are putative
reactions. Currently, the Pan metabolism is underestimated and it will be more
interesting to focus on filling metabolic gaps than adding new networks. These
networks work will serve as a basis to reconstruct whole cell metabolic models.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.genoscope.cns.fr/agc/metacoli/" mce_href="http://www.genoscope.cns.fr/agc/metacoli/">http://www.genoscope.cns.fr/agc/metacoli/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Gilles Vieira (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29156 = 'gv&#105;&#101;&#105;r&#97;' + '&#64;';
 addy29156 = addy29156 + 'g&#101;n&#111;sc&#111;p&#101;' + '&#46;' + 'cns' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy29156 + suffix + '\'' + attribs + '>' );
 document.write( addy29156 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65778 = 'gv&#105;&#101;&#105;r&#97;' + '&#64;';
 addy65778 = addy65778 + 'g&#101;n&#111;sc&#111;p&#101;' + '&#46;' + 'cns' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy65778 + suffix + '\'' + attribs + '>' );
 document.write( addy65778 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54454 = 'gv&#105;&#101;&#105;r&#97;' + '&#64;';
 addy54454 = addy54454 + 'g&#101;n&#111;sc&#111;p&#101;' + '&#46;' + 'cns' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy54454 + suffix + '\'' + attribs + '>' );
 document.write( addy54454 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CNRS-UMR 8030, Laboratoire d’Analyse Bioinformatique
en Génomique et Métabolisme, Commissariat à l’Energie Atomique (CEA), Direction
des Sciences du Vivant, Institut de Génomique, Genoscope</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1: CNRS-UMR 8030, Laboratoire d’Analyse Bioinformatique
en Génomique et Métabolisme, Commissariat à l’Energie Atomique (CEA), Direction
des Sciences du Vivant, Institut de Génomique, Genoscope, 2 rue Gaston
Crémieux, 91057 Evry Cedex, Evry cedex F-91006 2: INSERM U722 and Université
Paris 7, 16 rue Henri Huchard, 75018 Paris, France 3: INRA, UMR de Génétique
Végétale, INRA/CNRS/Univ Paris-Sud/AgroParistech, Ferme du Moulon, F-91190 Gif
sur Yvette, France: 4: Max Planck Institute for Mathematics in the Sciences,
Inselstr. 22, D-04103 Leipzig, Germany 5: TOTAL Gas &amp; Power 2, place Jean
Miller - La Défense 6 92078 Paris La Défense Cedex - France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by a grant from the French
National Research Agency (ANR): METACOLI project, ref. ANR-08-SYSC-011. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection50">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745821">G-49. BioGraph: discovering biomedical relations by
unsupervised hypothesis generation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Liekens A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Liekens A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), De Knijf J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Knijf J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Daelemans W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Daelemans W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Goethals B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Goethals B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), De Rijk P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Rijk P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Del-Favero J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Del-Favero J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">For the computational identification of suitable targets
among candidate genes in a biomedical context, the intelligence and
intelligibility of the method are of vital importance for evaluating the
prioritizations. Protein-protein interaction networks are often adopted, but
are limited in functional expressivity. The integration with multiple types of
biomedical knowledge can enhance the quality of automatically generated
functional hypotheses relating contexts, e.g., a disease, and target sets,
e.g., a set of candidate genes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We propose a data mining framework that allows for
the automated formulation of comprehensible functional hypotheses relating a
context to targets. The method is based on the integration of heterogeneous
biomedical knowledge bases and yields intelligible and literature-supported
indirect functional relations. By assessing the plausibility and specificity of
these hypothetical functional paths within a user-provided research context,
the unsupervised methodology is capable of appraising and ranking of research
targets, without requiring prior domain knowledge from the user.</p>

<h3>Results</h3>

<p class="abresults">We highlight this methodology’s application in the
prioritization of susceptibility genes for hereditary diseases. We show that
the proposed framework outperforms leading technologies on published benchmarks
(AUC 91.31% on Endeavour benchmark) and is capable of robustly predicting
recently discovered susceptibility genes for a range of hereditary psychiatric
disorders and for suggesting new genome-wide putative candidates. The data
mining method is publicly accessible via a web service at
http://www.biograph.be</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our proposed methodology offers a range of significant
improvements over leading bioinformatics platforms for in silico identification
of susceptibility genes: highly ranked targets are grounded in intelligible
putative functional hypotheses with rich semantics, verifiable by their
references in the literature. The method is unsupervised and does not require
prior domain knowledge from the user. Beyond disease-gene applications, the
method is applicable in various biological research settings requiring the
intelligent and intelligible identification of promising research targets</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://biograph.be" mce_href="http://biograph.be">http://biograph.be</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anthony M.L. Liekens (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23414 = '&#97;nth&#111;ny' + '&#64;';
 addy23414 = addy23414 + 'l&#105;&#101;k&#101;ns' + '&#46;' + 'n&#101;t';
 document.write( '<a ' + path + '\'' + prefix + addy23414 + suffix + '\'' + attribs + '>' );
 document.write( addy23414 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83295 = '&#97;nth&#111;ny' + '&#64;';
 addy83295 = addy83295 + 'l&#105;&#101;k&#101;ns' + '&#46;' + 'n&#101;t';
 document.write( '<a ' + path + '\'' + prefix + addy83295 + suffix + '\'' + attribs + '>' );
 document.write( addy83295 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33728 = '&#97;nth&#111;ny' + '&#64;';
 addy33728 = addy33728 + 'l&#105;&#101;k&#101;ns' + '&#46;' + 'n&#101;t';
 document.write( '<a ' + path + '\'' + prefix + addy33728 + suffix + '\'' + attribs + '>' );
 document.write( addy33728 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB Department of Molecular Genetics, University of
Antwerp, Belgium</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Applied Molecular Genomics group, VIB Department of
Molecular Genetics, Universiteit Antwerpen, Antwerpen, Belgium, (2) Advanced
Database Research and Modelling group, Department of Mathematics and Computer
Science, Universiteit Antwerpen, Antwerpen, Belgium, (3) Computational
Linguistics and Psycholinguistics Research Center, Universiteit Antwerpen,
Antwerpen Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection51">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745822">G-50. Tuning noise propagation in a two-step series
enzymatic cascade</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Dhananjaneyulu V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dhananjaneyulu V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sagar PVN<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sagar PVN&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kumar G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kumar G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Viswanathan GA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Viswanathan GA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">Noise or fluctuations propagate and distribute via
signaling pathways in cells, and thereby affect cellular response. While noise
might have to be minimized in certain systems, it may be used as part of
cellular decision-making process. Characterization of noise propagation in
signaling pathways can provide useful insights into the functioning of cells in
the presence of noise. Signaling pathways consists of several enzymatic
building blocks such as Mitogen Activated Protein Kinase (MAPK) cascades. In
this study, we characterize noise propagation in a two-step series MAPK
enzymatic cascade.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The stochastic dynamics of the proteins involved in
the enzymatic cascade is captured using chemical master equations. Using
Gillespie simulations, we obtained noise dynamics and estimate (intrinsic)
noise propagation in the cascade due to the stochastic nature of the
biochemical reactions. We detected extrinsic noise propagation in the cascade
using linear stability analysis of the stochastic differential equation (SDE)
of the Langevin type and by solving the full SDE. We used global sensitivity
analysis to study the effect of various system parameters on noise propagation
in the cascade.</p>

<h3>Results</h3>

<p class="abresults">Stochastic simulations suggest that intrinsic noise
propagation dominates over the extrinsic noise. Extrinsic noise estimates
obtained using steady state perturbation analysis are comparable to those
obtained using stochastic simulations. Simulations indicate that there exists a
critical upstream enzyme concentration below which noise propagation in the
cascade is amplified. Sensitivity analysis shows that the extent of noise
propagation is mostly controlled by the total concentrations of upstream enzyme
and substrate of the downstream enzymatic reaction cycle.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Steady state perturbation analysis, which is
computationally less intensive compared to the full stochastic simulations,
provide a good extrinsic noise propagation estimate. However, for prediction of
intrinsic noise propagation, stochastic simulations, though tedious and
computationally intensive, has to be performed. Noise propagation in the
enzymatic cascade can be tuned by modulating the upstream enzyme concentration
and/or the substrate of the downstream enzymatic reaction.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ganesh A. Viswanathan (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91129 = 'g&#97;n&#101;sh' + '&#64;';
 addy91129 = addy91129 + 'ch&#101;' + '&#46;' + '&#105;&#105;tb' + '&#46;' + '&#97;c' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy91129 + suffix + '\'' + attribs + '>' );
 document.write( addy91129 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33971 = 'g&#97;n&#101;sh' + '&#64;';
 addy33971 = addy33971 + 'ch&#101;' + '&#46;' + '&#105;&#105;tb' + '&#46;' + '&#97;c' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy33971 + suffix + '\'' + attribs + '>' );
 document.write( addy33971 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73989 = 'g&#97;n&#101;sh' + '&#64;';
 addy73989 = addy73989 + 'ch&#101;' + '&#46;' + '&#105;&#105;tb' + '&#46;' + '&#97;c' + '&#46;' + '&#105;n';
 document.write( '<a ' + path + '\'' + prefix + addy73989 + suffix + '\'' + attribs + '>' );
 document.write( addy73989 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Chemical Engineering, Indian Institute
of Technology Bombay</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Chemical Engineering Indian Institute of
Technology Bombay Powai, Mumbai 400076 India</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Department of Science and Technology, Ministry of
Science and Technology, India</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection52">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745823">G-51. The effect of interactome evolution on
network alignment</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ali W<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Ali W&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*),
Deane C<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE &quot;Deane
C&quot; <![endif]--><!--[if supportFields]><span style="mso-element:field-end" mce_style="mso-element:field-end"></span><![endif]-->M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Deane CM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Local alignments of protein interaction networks have
found little conservation among several species. While this could be a
consequence of the incompleteness of interaction data-sets and presence of
error, an intriguing prospect is that the process of network evolution is
sufficient to erase any evidence of conservation. Here, we aim to test this
hypothesis using models of network evolution and also investigate the role of
error in the results of network alignment.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used the duplication-divergence and geometric
models of network evolution to grow pairs of networks to the size of
experimental datasets. The network evolution was supplemented with a process of
ortholog evolution to model the distribution of orthologous relationships in
real proteomes. Network alignments were then carried out on pairs of real and
modelled data-sets and a distance metric based on graph statistics was used to
quantify the agreement between the alignments. Error was finally introduced in
the modelled networks to explain the discrepancy between real and modelled
alignments.</p>

<h3>Results</h3>

<p class="abresults">Our results indicate that evolution alone is unlikely to
account for poor quality alignments given by real data. Alignments of modelled
networks undergoing evolution are at least 4-5 times larger than real
alignments. We compare several error models in their ability to explain this
discrepancy. Estimates of false negative rates vary from 20 to 60% dependent on
the inclusion of the effect of incomplete proteome sampling. We find that false
positives affect network alignments little compared to false negatives
indicating that incompleteness is the major challenge for interactome
comparisons.</p>

<h3>Discussion</h3>

<p class="abdiscussion">While network evolution does play a considerable role in
network alignment quality, the effect of error cannot be discounted. This is
especially true for false negatives which seems to mostly account for the
extremely low conservation observed in real interactomes. Moreover, using a
quantitative comparison of real and simulated alignments makes it possible to
arrive at independent estimates of network error and evaluate likely error
models.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Waqar Ali (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy47507 = '&#97;l&#105;' + '&#64;';
 addy47507 = addy47507 + 'st&#97;ts' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy47507 + suffix + '\'' + attribs + '>' );
 document.write( addy47507 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62959 = '&#97;l&#105;' + '&#64;';
 addy62959 = addy62959 + 'st&#97;ts' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy62959 + suffix + '\'' + attribs + '>' );
 document.write( addy62959 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50475 = '&#97;l&#105;' + '&#64;';
 addy50475 = addy50475 + 'st&#97;ts' + '&#46;' + '&#111;x' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy50475 + suffix + '\'' + attribs + '>' );
 document.write( addy50475 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Statistics, University of Oxford</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Statistics, University of Oxford,
United Kingdom</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Department of Statistics, University of Oxford</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection53">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745824">G-52. Analyzing compounds’ mode of action by
biological relatedness of proteins</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Schmid R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Schmid R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Baum P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Baum P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Ittrich C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ittrich C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Fundel-Clemens K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Fundel-Clemens K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Lämmle B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lämmle B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Birzele F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Birzele F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Weith A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weith A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Brors B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brors B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Eils R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Eils R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Mennerich D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mennerich D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Quast K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Quast K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Identification of on- as well as off-target effects of
drugs remains a common objective in scientific research. These effects are
mediated by binding of compounds to proteins and subsequently influencing
related regulatory networks within the organism. Discovering key proteins as
well as their meaning in a broader biological context seems to be the most
promising way to answer questions with respect to mode of action and cellular
processes. As more and more data from diverse sources is available, the integration
of this knowledge is an important step to get a deeper insight into biology.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Protein interactions are taken from iRefIndex and
are enriched by literature derived confidence scores, information about
transcription factor binding sites, Gene Ontology annotation as well as gene
expression data. This accumulated information is translated to edge weighted
graphs. Finally, subnets are extracted based on interactions exhibiting high
biological relatedness with respect to the assigned weights. Results are
quantitatively evaluated by a comparison to randomized networks. To elucidate
the biological context of the subnets Fisher’s exact test is conducted on
predefined gene sets.</p>

<h3>Results</h3>

<p class="abresults">We are able to derive protein interactions that are
meaningful in the biological context under consideration. The results were
biologically validated using gene expression measured in TGF-β stimulated
cells. First, an enrichment of TGF-β related gene sets within the extracted
nets could be detected. Second, we were able to detect on- as well as
off-target effects of TGF-β receptor 1 kinase inhibiting compounds. On-target
effects are detected by the direction of deregulation in the extracted nets,
off-target effects are revealed by enrichment of genes related to off-target
signaling cascades.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Despite the extreme flexibility and simplicity of our
approach, we achieve very good and traceable results. Based on the research
objective the scores can be individually weighted and optimized and additional (experimental)
data can be added. These are the main advantages towards other, more abstract
approaches. As next steps, we plan to compare results based on our network to
STRING based results and contrast our method with similar approaches. Finally,
we plan to integrate phenotypic information of genes to gain even more value
for biological analyses and publish our method as an R-package.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ramona Schmid (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23189 = 'r&#97;m&#111;n&#97;.schm&#105;d' + '&#64;';
 addy23189 = addy23189 + 'b&#111;&#101;hr&#105;ng&#101;r-&#105;ng&#101;lh&#101;&#105;m' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy23189 + suffix + '\'' + attribs + '>' );
 document.write( addy23189 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28898 = 'r&#97;m&#111;n&#97;.schm&#105;d' + '&#64;';
 addy28898 = addy28898 + 'b&#111;&#101;hr&#105;ng&#101;r-&#105;ng&#101;lh&#101;&#105;m' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy28898 + suffix + '\'' + attribs + '>' );
 document.write( addy28898 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90993 = 'r&#97;m&#111;n&#97;.schm&#105;d' + '&#64;';
 addy90993 = addy90993 + 'b&#111;&#101;hr&#105;ng&#101;r-&#105;ng&#101;lh&#101;&#105;m' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy90993 + suffix + '\'' + attribs + '>' );
 document.write( addy90993 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Boehringer Ingelheim Pharma GmbH &amp; Co. KG</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Boehringer Ingelheim Pharma GmbH &amp; Co. KG,
Biberach/Riss, Germany, 2 Theoretical Bioinformatics Department, German Cancer
Research Center (DKFZ), Heidelberg, Germany, 3 Department of Bioinformatics and
Functional Genomics, Institute of Pharmacy and Molecular Biotechnology (IPMB)
and BioQuant, University of Heidelberg, Heidelberg, Germany, 4 Boehringer
Ingelheim Pharmaceuticals Inc., Ridgefield, CT 06877, USA. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection54">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745825">G-53. Logical modelling of the regulatory network
controlling the formation of the egg appendages in Drosophila</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Fauré A<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Fauré A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Vreede B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vreede B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Sucena É<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sucena É&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Chaouiya C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chaouiya C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Formation of the dorsal appendages in Drosophila provides
an interesting model to study epithelial morphogenesis. Experimental studies
have identified Gurken and Decapentaplegic as the two main regulatory signals
that are responsible for the patterning of the follicle cell epithelium that
will secrete the eggshell later in oogenesis. However, the pattern of expression
of the markers that characterise the different regions is dynamic in both space
and time, and different groups report contradictory results. We use a modelling
approach to improve our understanding of this system.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The logical formalism offers a simple framework to
study regulatory networks. Briefly, regulatory components are represented by
variables that can take a finite number of discrete values representing their
level of activity with respect to a given threshold. Logical rules determine
the target level of each variable depending on those of its regulators. Beyond
topological properties of the wiring diagram, the logical formalism offers
tools to study dynamic properties such as attractors, and logical models can be
used to simulate experiments in silico and make new predictions.</p>

<h3>Results</h3>

<p class="abresults">We review the litterature to extract a map of the regulatory
network controlling dorsal appendages formation in Drosophila melanogaster.
Using the logical modelling software GINsim, we further simplify this map to
build a dynamic model of the intracellular pathway, with Gurken and
Decapentaplegic as inputs, and Rhomboid, Broad and Fasciclin III as reading
outputs. Our preliminary results support the hypothesis that Broad inhibits
Fas3 in the roof cells.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In spite of several modelling efforts, the precise wiring
of the regulatory network remains elusive. Our map synthesises in an intuitive
format the current knowledge of the control of dorsal appendage patterning in
Drosophila melanogaster. We have yet to confirm experimentally our modelling
predictions. In the near future we also plan to use our cellular model as a
module to build a multicellular model of the antero-dorsal follicle cell
epithelium.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Adrien Fauré (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2896 = '&#97;f&#97;&#117;r&#101;' + '&#64;';
 addy2896 = addy2896 + '&#105;gc' + '&#46;' + 'g&#117;lb&#101;nk&#105;&#97;n' + '&#46;' + 'pt';
 document.write( '<a ' + path + '\'' + prefix + addy2896 + suffix + '\'' + attribs + '>' );
 document.write( addy2896 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19406 = '&#97;f&#97;&#117;r&#101;' + '&#64;';
 addy19406 = addy19406 + '&#105;gc' + '&#46;' + 'g&#117;lb&#101;nk&#105;&#97;n' + '&#46;' + 'pt';
 document.write( '<a ' + path + '\'' + prefix + addy19406 + suffix + '\'' + attribs + '>' );
 document.write( addy19406 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94685 = '&#97;f&#97;&#117;r&#101;' + '&#64;';
 addy94685 = addy94685 + '&#105;gc' + '&#46;' + 'g&#117;lb&#101;nk&#105;&#97;n' + '&#46;' + 'pt';
 document.write( '<a ' + path + '\'' + prefix + addy94685 + suffix + '\'' + attribs + '>' );
 document.write( addy94685 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Instituto Gulbenkian de Ciência</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Instituto Gulbenkian de Ciência, Oeiras, Portugal</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">A. Fauré acknowledges funding from the Fundação para a
Ciência e a Tecnologia, Portugal (project PTDC/EIACCO/099229/2008). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection55">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745826">G-54. A curated database of microRNA mediated
feed-forward loops involving MYC as master regulator</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">El Baroudi M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;El Baroudi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Corà D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Corà D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3,*), Bosia C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bosia C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Osella M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Osella M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Caselle M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Caselle M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">The MYC Transcription Factors (TFs) are one of the most
important family of regulators in human. They may act both as activators or
repressors of their target genes and are involved in a host of key biological
processes, ranging from cell cycle progression, to apoptosis and cellular
transformation. They are also known to be involved in the biology of many human
cancer types. Do date, little is known about the MYC / microRNAs cooperation in
the regulation of genes at the transcriptional and post-transcriptional level.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In order to investigate the interplay between MYC
and microRNAs, we concentrated our attention on the study of mixed microRNA /
TF Feed-Forward Regulatory Loops (FFLs), i.e. elementary regulatory circuits in
which a master TF regulates an microRNA and together with it a set of joint
target protein-coding genes. Employing independent databases with
experimentally validated data, we identified several mixed FFLs regulated by
MYC and characterized completely by experimentally supported regulatory
interactions, in human.</p>

<h3>Results</h3>

<p class="abresults">We were able to identify a total of 130 mixed FFLs having
MYC as master regulator, involving 36 MYC regulated microRNAs and 82 joint
target protein-coding genes. Out of these 130 FFLs, 28 could be classified as
incoherent (type_I) and 42 as coherent (type_II). We then study the statistical
and functional properties of these circuits, showing that this class of loops
is over-represented in the human regulatory network, it is functionally related
to cancer and shows a remarkable redundancy of the microRNA branch. Finally, we
discuss a few examples involving E2F1, PTEN, RB1 and VEGF.</p>

<h3>Discussion</h3>

<p class="abdiscussion">It has become by now clear that the interplay between
transcriptional and post-transcriptional (microRNA mediated) regulation plays a
crucial role in the modulation of gene expression. We report the assembly and
characterization of a catalogue of human mixed TF / microRNA Feed-Forward
Loops, having MYC as master regulator and completely defined by experimentally
verified regulatory interactions. This study allowed us to draw a new and rich
picture of the MYC centred network, revealing complex interplays with potential
important role in defining accurate expression levels for many key genes.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Davide Cora' (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81663 = 'd&#97;v&#105;d&#101;.c&#111;r&#97;' + '&#64;';
 addy81663 = addy81663 + '&#105;rcc' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy81663 + suffix + '\'' + attribs + '>' );
 document.write( addy81663 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36427 = 'd&#97;v&#105;d&#101;.c&#111;r&#97;' + '&#64;';
 addy36427 = addy36427 + '&#105;rcc' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy36427 + suffix + '\'' + attribs + '>' );
 document.write( addy36427 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78515 = 'd&#97;v&#105;d&#101;.c&#111;r&#97;' + '&#64;';
 addy78515 = addy78515 + '&#105;rcc' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy78515 + suffix + '\'' + attribs + '>' );
 document.write( addy78515 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Systems Biology Lab - Institute for Cancer Research
and Treatment (IRCC), School of Medicine, University of Torino.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Department of Theoretical Physics, University of
Torino and INFN, Via P. Giuria, 1. I-10125 Torino, Italy. 2) Center for
Molecular Systems Biology, University of Torino, Via A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Via A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->ccademia Albertina 13, I-10100
Torino, Italy 3) Systems Biology Lab, Institute for Cancer Research and
Treatment (IRCC), School of Medicine, University of Torino, Str. Prov. 142 Km.
3.95, I-10060 Candiolo, Torino, Italy</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection56">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745827">G-55. The Biological Connection Markup Language
(BCML): a SBGN compliant data format for visualization, filtering and analysis
of biological pathways</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Calura E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Calura E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*,=), Beltrame L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beltrame L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, (2,=), Popovici R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Popovici R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4), Rizzetto L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rizzetto L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Rivero Guedez D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rivero Guedez D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Donato M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Donato M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Romualdi C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Romualdi C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Draghici S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Draghici S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Cavalieri D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cavalieri D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Throughout the years, many data models for pathways have
been proposed, although successful, they did not tackle the the underlying biological
problem, especially for signaling pathways. These have interconnected, finely
regulated structures and are affected by environment-dependent changes. Current
models represent pathways regardless of these features. However, an imprecise
model affects the power of the analyses carried out with it. BCML allows an
unambiguous and fully dynamic representation of signaling pathways that is
useful for both biologists and bioinformaticians.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">An XML schema describing the complete SBGN Process
Description version 1.1 was implemented according to the SBGN reference
documentation. An additional schema, was written as an extension of the main
SBGN schema. A set of utilities to validate, filter and extract entities from
the schema was also developed in the Java programming language. In particular,
the utilties support consistency checking, export to analysis formats,
annotation, pathway filtering and visualization of BCML compliant files.</p>

<h3>Results</h3>

<p class="abresults">The model, defined as an XML Schema, incorporates all the
entities and interactions defined by the SBGN PD specification, including the
rules and restraints that apply to them. The model can store additional
information on the entities that compose the network: each entity can have
identifiers linked to public databases and information can be stored for
multiple species in the same entity definition. Data stored in a BCML compliant
file can be manipulated mainly in four ways: discriminative selection of the
pathway features, graphical representation, incorporation of experimental
measurements.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Most pathways present in public databases, although
described for different organisms, lack information on the biological
environment where they have been described. Basing on that we believe that
improvements in the analysis of pathway will come from a better use of the
knowledge present in scientific literature. BCML, which stores deeper
descriptions of biological knowledge, turns out to be a format extremely
suitable for advanced pathway analysis methods and its dynamic nature makes it
an important tool for the dissection of complex, highly specific biological
problems.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Enrica Calura (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66321 = '&#101;nr&#105;c&#97;.c&#97;l&#117;r&#97;' + '&#64;';
 addy66321 = addy66321 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy66321 + suffix + '\'' + attribs + '>' );
 document.write( addy66321 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy37378 = '&#101;nr&#105;c&#97;.c&#97;l&#117;r&#97;' + '&#64;';
 addy37378 = addy37378 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy37378 + suffix + '\'' + attribs + '>' );
 document.write( addy37378 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78537 = '&#101;nr&#105;c&#97;.c&#97;l&#117;r&#97;' + '&#64;';
 addy78537 = addy78537 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy78537 + suffix + '\'' + attribs + '>' );
 document.write( addy78537 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Padova</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1- Department of Biology, University of Padova, Padova,
Italy 2- Department of Pharmacology, University of Firenze, Firenze, Italy 3-
Department of Computer Science, Wayne State University, Detroit, USA 4-
Miravtech ± These authors contributed equally to this work.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was financially supported by the Network of
Excellence DC-THERA, Dendritic cells for novel immunotherapies
(www.dc-thera.org) and by the Network of Excellence SYBARIS (Grant Agreement
242220), University of Firenze and University of Padova. We are grateful to the
DC-Thera curation team specially to Maria Cristina Gauzzi and Sandra Gessani,
Sonja Bushow and Carl Figdor, Philippe Pierre, Walter Reith, Andrea Splendiani
and Jon Austin. We thank Gabriele Sales for helpful discussions. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection57">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745828">G-56. Estimating the size of the S.cerevisiae
interactome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sambourg L<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Sambourg L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Thierry-Mieg N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thierry-Mieg N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">As protein interactions mediate most cellular mechanisms,
protein-protein interaction networks are essential in the study of cellular
processes. Consequently, many large scale interactome mapping projects have
been undertaken, and protein-protein interactions are being distilled into
databases through literature curation; yet protein-protein interaction data are
still far from comprehensive, even in the model organism S.cerevisiae.
Estimating the interactome size is important for evaluating the completeness of
current datasets, in order to measure the remaining efforts that are required.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We show that literature curation has reported many
more interactions for highly studied proteins than for poorly studied ones. So
literature-curated data allows us to precisely assess the size of the
'well-studied' sub-network (sub-network containing proteins cited in more than
125 papers). We then compute the proportion of interactions that HT experiments
detect for this sub-network. Assuming that the HT experiments coverage is
independent of how thoroughly a protein has been studied allows to apply this
coverage to the entire network, and gives an estimate of the interactome size.</p>

<h3>Results</h3>

<p class="abresults">Several estimates of the size of the S. cerevisiae
interactome have been proposed, but none of them directly take into account
information from both literature-curated and high-throughput data. We propose
here a simple and reliable method for estimating the size of an interactome,
combining these two data sources. Our method yields an estimate of at least ~
35,000 direct physical protein-protein interactions in S.cerevisiae.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our method allows to free oneself from the necessity of a
gold standard and takes advantage of almost all available data, leading to a
more reliable estimate. Contrary to several other methods, our estimate is very
robust with respect to the choice of the high-throughput dataset used. Finally,
combining literature-curated and high-throughput data leads to higher estimates
of the S. cerevisiae interactome size. This confirms the complementarity of
these two data sources, and provides a sobering view of the coverage of current
interactomes: extensive efforts and/or new methods are sorely needed.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Laure Sambourg (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97002 = 'l&#97;&#117;r&#101;.s&#97;mb&#111;&#117;rg' + '&#64;';
 addy97002 = addy97002 + '&#105;m&#97;g' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy97002 + suffix + '\'' + attribs + '>' );
 document.write( addy97002 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2781 = 'l&#97;&#117;r&#101;.s&#97;mb&#111;&#117;rg' + '&#64;';
 addy2781 = addy2781 + '&#105;m&#97;g' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy2781 + suffix + '\'' + attribs + '>' );
 document.write( addy2781 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27973 = 'l&#97;&#117;r&#101;.s&#97;mb&#111;&#117;rg' + '&#64;';
 addy27973 = addy27973 + '&#105;m&#97;g' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy27973 + suffix + '\'' + attribs + '>' );
 document.write( addy27973 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CNRS UMR5525</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">CNRS UMR5525</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection58">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745829">G-57. Laplacian eigenmaps, penalized principal
component regression on graphs, and analysis of biological pathways</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Shojaie A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Shojaie A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Michailidis G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Michailidis G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Gene, protein and metabolite networks provide valuable
information on how components of biological systems interact with each other to
carry out vital cell functions. The behavior of complex biological systems can
only be understood by incorporating information on interactions among
components of the system and by analyzing the effect of biological pathways,
rather than individual components. The proposed method incorporates the
available network information and provides a flexible framework for analysis of
pathway effects.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this poster, we establish a connection between
Laplacian eigenmaps and principal components of the covariance matrix and
propose a dimension reduction method that directly incorporates the network
information. Using this framework, the significance of biological pathways can
then be analyzed by solving an eigenvalue problem on the graph with Neumann boundary
conditions. We hence reformulate the problem of analysis of biological pathways
as a principal component regression problem on the graph, a group-lasso penalty
to determine the significance of each subnetwork.</p>

<h3>Results</h3>

<p class="abresults">We evaluate the performance of the proposed method using
simulated data, as well as real gene expression data on yeast Galactose
Utilization. The findings of this analysis suggest that the proposed method
offers better efficiency compared to Gene Set Enrichment Analysis (GSEA) and is
a computationally efficient alternative to full network models for analysis of
biological pathways, such as our previously proposed NetGSA method.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The proposed method offers a systematic approach for
dimension reduction in networks, with a priori defined subnetworks of interest.
It can also incorporate both weighted and unweighted adjacency matrices and can
be easily extended for analysis of complex experimental conditions using the
framework of generalized linear models. This method can also be used to assess
the effect of biological pathways in longitudinal and time-course studies. Our
simulation studies, and the real data examples suggest that the method offers
significant improvements over methods of gene set enrichment analysis.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ali Shojaie (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58940 = 'sh&#111;j&#97;&#105;&#101;' + '&#64;';
 addy58940 = addy58940 + '&#117;m&#105;ch' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy58940 + suffix + '\'' + attribs + '>' );
 document.write( addy58940 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy75879 = 'sh&#111;j&#97;&#105;&#101;' + '&#64;';
 addy75879 = addy75879 + '&#117;m&#105;ch' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy75879 + suffix + '\'' + attribs + '>' );
 document.write( addy75879 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60453 = 'sh&#111;j&#97;&#105;&#101;' + '&#64;';
 addy60453 = addy60453 + '&#117;m&#105;ch' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy60453 + suffix + '\'' + attribs + '>' );
 document.write( addy60453 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Michigan</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Statistics, University of Michigan (2)
Department of Electrical Engineering and Computer Science, University of
Michigan</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was partially supported by grant
1RC1CA145444-0110 from the US National Institute of Health (NIH). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection59">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745830">G-58. Predicting metabolic pathways from bacterial
operons and regulons</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Faust K<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Faust K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Croes D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Croes D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Dupont P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dupont P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), van Helden J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van Helden J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">The pathway extraction tool predicts metabolic pathways
from sets of functionally related enzyme-coding genes [Faust, et al.,
Bioinformatics 2010, 26:1211]. In contrast to pathway projection approaches,
pathway discovery does not rely on any assumption of pathway conservation. This
approach can detect variants, super-pathways, and “cross-map” paths. It can be
applied to organisms whose metabolism is unknown, but for which enzyme-coding
genes have been identified in the genome, and some information is available
about their functional grouping (co- expression, operons, gene fusion, ....).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To extract a pathway, we connect query compounds or
reactions in the metabolic network using various algorithms: the random walk
based kWalks algorithm, three approaches based on k-shortest path finding and
combinations of kWalks with the latter. When predicting pathways from genes, we
have to link enzyme-coding genes to reactions. In case of broad-specificity
enzymes, this yields a large number of reactions, only a few of which are
relevant in the pathway. To address this problem, reactions catalyzed by the
same EC number are merged into equivalence groups.</p>

<h3>Results</h3>

<p class="abresults">We evaluated the pathway extraction algorithms on 71 MetaCyc
pathways and found that a combination of kWalks with a shortest-paths based
approach yields the highest accuracy (77%). The pathway extraction tool has
been integrated in the Network Analysis Tools (NeAT,
http://rsat.ulb.ac.be/neat/), and can be accessed via a web interface, as stand-alone
application or as SOAP/WSDL Web services. The seed nodes for subgraph
extraction can be provided as reactions, (partial) compound names as well as EC
numbers or genes. Pathways can be extracted from KEGG, MetaCyc or custom
networks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We will present a selection of study cases illustrating
the way to combine operon prediction, phylogenetic footprint discovery and
pathway extraction in order to infer metabolic pathways from bacterial genomes.
In future, this strategy will be systematically applied on bacterial genomes in
the framework of the MICROME project (EU FP7).</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://rsat.ulb.ac.be/neat/" mce_href="http://rsat.ulb.ac.be/neat/">http://rsat.ulb.ac.be/neat/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jacques van Helden (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90194 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy90194 = addy90194 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy90194 + suffix + '\'' + attribs + '>' );
 document.write( addy90194 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy5035 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy5035 = addy5035 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy5035 + suffix + '\'' + attribs + '>' );
 document.write( addy5035 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26231 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy26231 = addy26231 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy26231 + suffix + '\'' + attribs + '>' );
 document.write( addy26231 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Bioinformatics and (Eco-)Systems Biology (BSB).
Vrije Universiteit Brussel, Pleinlaan 2. B-1050 Brussels, Belgium. Email:

 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44648 = 'kf&#97;&#117;st' + '&#64;';
 addy44648 = addy44648 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy44648 + suffix + '\'' + attribs + '>' );
 document.write( addy44648 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 2. Laboratoire de Bioinformatique des Génomes et des Réseaux
(BiGRe). Université Libre de Bruxelles, Campus Plaine, CP 263. Bld du Triomphe.
B-1050 Bruxelles, Belgium. Email: 
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28448 = 'J&#97;cq&#117;&#101;s.v&#97;n.H&#101;ld&#101;n' + '&#64;';
 addy28448 = addy28448 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy28448 + suffix + '\'' + attribs + '>' );
 document.write( addy28448 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script> 3. UCL Machine
Learning Group, Computing Science and Engineering Department, Universite ́
catholique de Louvain, B-1348 Louvain-la-Neuve, Belgium.

 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9525 = 'P&#105;&#101;rr&#101;.D&#117;p&#111;nt' + '&#64;';
 addy9525 = addy9525 + '&#117;cl&#111;&#117;v&#97;&#105;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9525 + suffix + '\'' + attribs + '>' );
 document.write( addy9525 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by the MICROME Collaborative
Project funded by the European Commission within its FP7 Programme, under the
thematic area "BIO-INFORMATICS - Microbial genomics and
bio-informatics", contract number 222886-2.". The BiGRe laboratory is
also supported by the Belgian Program on Interuniversity Attraction Poles,
initiated by the Belgian Federal Science Policy Office, project P6/25
(BioMaGNet). The contribution of Karoline Faust was supported by the Actions de
Recherches Concertées de la Communauté Française de Belgique (ARC grant number
04/09-307). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection60">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745831">G-59. Inferring translationally active RNA binding
proteins: mRNA interactions from polysomal profiling data with a Bayesian
inference approach</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tebaldi T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Tebaldi T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Sanguinetti G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sanguinetti G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Niranjan M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Niranjan M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Quattrone A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Quattrone A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Post-transcriptional control of gene expression is
strongly mediated by the action of RNA binding proteins (RBPs), capable of
uncoupling changes in the abundances of mRNAs from changes in their polysomal
access and therefore their availability to translation. Despite this, the
binding preferences of the majority of RBPs and the specific interactions
between individual RBPs and their mRNA targets are still poorly known.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We consider a linear model relating changes in the
polysomal levels of RNA binding proteins to changes in the polysomal loading
ratios of their putative target mRNAs. A Bayesian network is used to extract
putative RBP-mRNA connections. The joint posterior probability of the network
is estimated with a Gibbs Sampler. Clustering of RBPs takes place
contemporaneously in order to solve dimensionality problems.</p>

<h3>Results</h3>

<p class="abresults">The method has shown learning capability with synthetic data
and it has been applied to the prediction of post transcriptional networks in
yeast, starting from six experiments where both transcriptome and translatome
profiling microarray raw data were available.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Uncoupling between transcriptome and translatome changes
is a general and measurable phenomenon, but little is known about the
underlying molecular mechanisms which affect the fate of every mRNA molecule
once it is transcribed. The presented approach is a first step to untangle the
complexity of this problem and make reasonable functional predictions from
existing data.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Toma Tebaldi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78375 = 't&#101;b&#97;ld&#105;' + '&#64;';
 addy78375 = addy78375 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy78375 + suffix + '\'' + attribs + '>' );
 document.write( addy78375 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19577 = 't&#101;b&#97;ld&#105;' + '&#64;';
 addy19577 = addy19577 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy19577 + suffix + '\'' + attribs + '>' );
 document.write( addy19577 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43495 = 't&#101;b&#97;ld&#105;' + '&#64;';
 addy43495 = addy43495 + 'sc&#105;&#101;nc&#101;' + '&#46;' + '&#117;n&#105;tn' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy43495 + suffix + '\'' + attribs + '>' );
 document.write( addy43495 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Centre for Integrative Biology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Centre for Integrative Biology, University of
Trento. (2) Institute for Adaptive and Neural Computation, University of
Edinburgh (3) School of Electronics and Computer Science, University of
Southampton</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection61">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745832">G-60. DASS-GUI: a new data mining framework</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Hollunder J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Hollunder J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Van de Peer Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Peer Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Wilhelm T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wilhelm T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">“Data mining is the process of extracting patterns from
data. Data mining is becoming an increasingly important tool to transform these
data into information” (wikipedia). The growing importance of data mining is
especially evident in biology, where modern ‘omics’ technologies are producing
increasingly complex data. Traditional data mining techniques such as APRIORI
and CHARM are predominantly applied outside the realm of biology. However, the
closely related concept of biclustering has already found many biological
applications, such as gene expression analysis.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">DASS-GUI processes several steps including
preparing the data, applying data mining algorithms, and evaluating the
results. DASS-GUI runs in two modes namely calculation (pattern identification)
and analysis mode (evaluation of the identified patterns). The identification
step can be done by either our DASS-cs or one of two other implemented
state-of-the-art algorithms (LCM and FPclose). DASS-cs has three outstanding
features (a) analyzing multi-sets, (b) similarity pruning, and (c) significance
calculation.</p>

<h3>Results</h3>

<p class="abresults">The power and versatility of the DASS approach was already
demonstrated in a number of different biological applications, for instance for
the analysis of protein complexes and corresponding protein annotations
(Hollunder et al., 2005; Hollunder et al., 2007), the identification of
transcription factor binding site modules (Beyer et al., 2006), and the
analysis of multi-domain proteins as well as the identification of conserved
subnetworks in different species (Hollunder et al., 2007).</p>

<h3>Discussion</h3>

<p class="abdiscussion">Only DASS-cs allows similarity pruning during closed set
calculation to avoid redundant clusters. Next straightforward steps are to
integrate a pipeline preparing the data for directly working with expression
data as well as to apply the extracted information for predictions and
classifications. Generally, the identified patterns and rules can guide
decision making and forecast effects of interactions.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ifr.ac.uk/dass/gui/" mce_href="http://www.ifr.ac.uk/dass/gui/">http://www.ifr.ac.uk/dass/gui/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jens Hollunder (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy52363 = 'J&#101;ns.H&#111;ll&#117;nd&#101;r' + '&#64;';
 addy52363 = addy52363 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy52363 + suffix + '\'' + attribs + '>' );
 document.write( addy52363 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67083 = 'J&#101;ns.H&#111;ll&#117;nd&#101;r' + '&#64;';
 addy67083 = addy67083 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy67083 + suffix + '\'' + attribs + '>' );
 document.write( addy67083 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6453 = 'J&#101;ns.H&#111;ll&#117;nd&#101;r' + '&#64;';
 addy6453 = addy6453 + 'psb' + '&#46;' + 'v&#105;b-&#117;g&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy6453 + suffix + '\'' + attribs + '>' );
 document.write( addy6453 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Plant Systems Biology, VIB, Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Plant Systems Biology, VIB, Ghent
University, Technologiepark 927, 9052 Gent, Belgium (2) Theoretical Systems
Biology, Institute of Food Research, Norwich Research Park, Conley Norwich NR4
7UH, U.K.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge the support of Ghent University
(Multidisciplinary Research Partnership “Bioinformatics: from nucleotides to
networks”) and Biotechnology and Biological Sciences Research Council Core
Strategic Grant for the Institute of Food Research. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection62">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745833">G-61. EnrichNet: network-based gene set enrichment
analysis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Glaab E<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Glaab E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Baudot A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Baudot A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Krasnogor N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Krasnogor N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Valencia A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valencia A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Classical overlap-based gene set enrichment analysis is
often used to examine the significance of association between gene sets of interest
and cellular processes. However, functional pathway/process annotations are
still missing for many genes. Molecular interaction data might help to uncover
some of this missing information. Here, we propose to measure the association
between gene sets and functional pathways/processes based on the distances
between corresponding proteins in a protein-protein interaction network,
instead of only using the overlap of gene sets to estimate their association.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We present a new network-based enrichment analysis
method. The approach maps gene and protein sets of interest onto a protein
interaction network and computes the distribution of shortest path distances
between all protein pairs from different sets. This distribution is then compared
against the distribution across all pathway/process protein sets from a chosen
database (KEGG, BioCarta, Reactome, etc.) using the Xd-distance (Olmea et al.,
1999), which assigns higher weights to smaller distances, ensuring that
large-distance outliers cannot distort the results.</p>

<h3>Results</h3>

<p class="abresults">When applying our network-based gene set association scoring
method to search for pathways altered in cancer (calculating network distances
between known cancer genes, Futreal et al., 2004, and KEGG signalling pathways)
a high correlation of pathway ranks to classical enrichment ranks is obtained,
but our method additionally enables the ranking of non-overlapping gene sets.
Moreover, the approach provides a visual analysis of sub-networks corresponding
to the input gene/protein sets to identify interactions between proteins of
interest and potential linking proteins.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The network-based enrichment analysis method enables the
combination of experimental evidence from multiple sources, different
interaction networks (protein-protein, genetic interactions, etc.) and
experimental data on genes and proteins (DNA microarrays, Western blots, RNA
sequencing, gene mutation and methylation, etc.), helping to fill knowledge
gaps in single experiments and increasing the robustness and interpretability
of enrichment analysis.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.infobiotics.net/enrichnet" mce_href="http://www.infobiotics.net/enrichnet">http://www.infobiotics.net/enrichnet</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Enrico Glaab (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy2837 = '&#101;nr&#105;c&#111;.gl&#97;&#97;b' + '&#64;';
 addy2837 = addy2837 + 'cs' + '&#46;' + 'n&#111;tt' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy2837 + suffix + '\'' + attribs + '>' );
 document.write( addy2837 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90271 = '&#101;nr&#105;c&#111;.gl&#97;&#97;b' + '&#64;';
 addy90271 = addy90271 + 'cs' + '&#46;' + 'n&#111;tt' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy90271 + suffix + '\'' + attribs + '>' );
 document.write( addy90271 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35351 = '&#101;nr&#105;c&#111;.gl&#97;&#97;b' + '&#64;';
 addy35351 = addy35351 + 'cs' + '&#46;' + 'n&#111;tt' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy35351 + suffix + '\'' + attribs + '>' );
 document.write( addy35351 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Nottingham University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. School of Computer Science, Nottingham University,
Jubilee Campus,NG8 1BB Nottingham, UK 2. Structural Biology and Biocomputing
Program, CNIO, E-28029 Madrid, Spain</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge support by the Biotechnology and
Biological Sciences Research Council (BB/F01855X/1), the Spanish Science and
Innovation (MICINN) grant (BIO2007-66855, "functions for gene sets")
and Instituto de Salud Carlos III RTIC COMBIOMED network (RD07/0067/0014). AB
is supported by the Juan de la Cierva postdoctoral fellowship. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection63">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745834">G-62. Interacting copy number alterations in breast
cancer</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Canisius S<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Canisius S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Klijn C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Klijn C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Smid M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Smid M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Martens J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Martens J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Foekens J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Foekens J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Wessels L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wessels L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">One of the challenges faced by cancer research is finding
out which genes play a role in the formation and growth of tumours. While many
such genes have been found and described individually, it is known that
deregulation of a single gene does not cause cancer. It is therefore equally
important to reveal the interactions between genes that drive tumour
development. Since DNA copy number alterations are an important cause of gene
deregulation in cancer, we hypothesise that interactions between cancer genes
can be found by detecting co-occurring copy number changes in a set of tumours.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For a set of 216 breast tumours, we obtained copy
number data, generated with a high-density SNP array. An association matrix is
constructed that scores the dependency between pairs of genomic loci. The
estimates in this matrix are smoothed to account for the fact that copy number
alterations tend to involve continuous stretches of DNA rather than isolated
nucleotides. Given such a smoothed association matrix, regions of significant
association are detected by a genome-wide permutation test that corrects for
the massive amount of multiple testing.</p>

<h3>Results</h3>

<p class="abresults">Analyses for co-occurring gains, losses, and gain/loss
combinations revealed several genomic regions between which significant
dependencies are observed. A closer inspection of these co-occurrences showed
interesting links with known subtypes of breast cancer. With mRNA expression
for the same set of tumours, we confirmed a positive relation between gene
dosage and mRNA abundance. Moreover, protein interaction databases confirmed
the existence of interacting genes in the co-occurring regions. These findings
provide suggestions for the possible targets of the alterations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We present an overview of co-occurring copy number
alterations in breast cancer. We show that such co-occurrences are a frequent
phenomenon, and provide additional analyses that suggest an important
regulatory role in tumourigenic processes. The strong association we found
between copy numbers of genomic regions located on different chromosomes
illustrate that tumour development not only selects for individual genes, but
also for gene interactions. These interactions may be found to be the weak
links in the regulatory networks that drive tumour development.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor"><a href="/Rebholz-srv/eccb2010/Sander%20V.M.%20Canisius%20%
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93830 = '28S.C&#97;n&#105;s&#105;&#117;s' + '&#64;';
 addy93830 = addy93830 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy93830 + suffix + '\'' + attribs + '>' );
 document.write( addy93830 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>%29" mce_href="http://mce_host/Sander%20V.M.%20Canisius%20%
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93166 = '28S.C&#97;n&#105;s&#105;&#117;s' + '&#64;';
 addy93166 = addy93166 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy93166 + suffix + '\'' + attribs + '>' );
 document.write( addy93166 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>%29">Sander V.M. Canisius
(
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54756 = 'S.C&#97;n&#105;s&#105;&#117;s' + '&#64;';
 addy54756 = addy54756 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy54756 + suffix + '\'' + attribs + '>' );
 document.write( addy54756 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</a><br mce_bogus="1"></p>

<p class="presentingauthor">Netherlands Cancer Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Division of Molecular Biology, The Netherlands
Cancer Institute, Amsterdam, The Netherlands (2) Information and Communication
Theory Group, Delft University of Technology, Delft, The Netherlands (3)
Department of Medical Oncology, Erasmus Medical Center Rotterdam, Josephine
Nefkens Institute and Cancer Genomics Centre, Rotterdam, The Netherlands</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection64">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745835">G-63. Compression of mass spectral imaging data
using discrete wavelet transform guided by spatial information</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Verbeeck N<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Verbeeck N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Van de Plas R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van de Plas R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), De Moor B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Moor B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3), Waelkens E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Waelkens E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3)</p>

<p class="abmotivation">Mass Spectral Imaging is a relatively new molecular
imaging technology that makes it possible to detect thousands of molecules
throughout tissue simultaneously, ranging from low-mass metabolites to
high-mass proteins. This technology is of prime interest for the molecular
characterization of tissue in biomedical studies. In recent years, MSI data sets
have grown in size to such extent that it becomes more and more infeasible to
computationally analyze them in their raw form due to both memory and
calculation time constraints.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Previous research at ESAT by Van de Plas et al. has
shown solid results using Discrete Wavelet Transform (DWT) on mass spectra to
perform feature selection, thus reducing data size, dimensionality and noise.
Our newest method further improves on this approach by incorporating one of the
key aspects of MSI, spatial information, to better understand what part of the
data can truly be considered noise. By using this information, we can
selectively remove only those details that do not exhibit a spatial structure.</p>

<h3>Results</h3>

<p class="abresults">We demonstrate the performance of this new compression
method on a sagittal section of mouse brain and compare the results to the Van
de Plas et al. method and to direct analysis of the raw measurements. The
presented study focuses on neurodegenerative diseases that show spatially
specific behaviour. Examples of such diseases include Parkinson’s disease,
where dopamine producing brain nuclei such as the amygdala are affected, and
amyotrophic lateral sclerosis, where motor neuron regions in the brain are
affected.</p>

<h3>Discussion</h3>

<p class="abdiscussion">By retaining a small number of detail coefficients that
express spatial structure we can strongly improve reconstruction of the mass
spectra while still achieving considerable compression.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Nico Verbeeck (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88515 = 'n&#105;c&#111;.v&#101;rb&#101;&#101;ck' + '&#64;';
 addy88515 = addy88515 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy88515 + suffix + '\'' + attribs + '>' );
 document.write( addy88515 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74829 = 'n&#105;c&#111;.v&#101;rb&#101;&#101;ck' + '&#64;';
 addy74829 = addy74829 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy74829 + suffix + '\'' + attribs + '>' );
 document.write( addy74829 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91183 = 'n&#105;c&#111;.v&#101;rb&#101;&#101;ck' + '&#64;';
 addy91183 = addy91183 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy91183 + suffix + '\'' + attribs + '>' );
 document.write( addy91183 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Katholieke Universiteit Leuven, Dept. of Electrical
Engineering (ESAT), SCD-SISTA (BIOI), Kasteelpark Arenberg 10, B-3001 Leuven
(Heverlee), Belgium. 2. Katholieke Universiteit Leuven, Dept. of Molecular Cell
Biology, O&amp;N, Herestraat 49 - bus 901, B-3000 Leuven, Belgium. 3.
Katholieke Universiteit Leuven, ProMeta, Interfaculty Centre for Proteomics and
Metabolomics, O&amp;N2, Herestraat 49, B-3000 Leuven, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">* Research Council KUL: ProMeta, GOA Ambiorics, GOA
MaNet, CoE EF/05/007 SymBioSys, START 1, several PhD/postdoc &amp; fellow
grants * Flemish Government: o FWO: PhD/postdoc grants, projects , G.0318.05
(subfunctionalization), G.0553.06 (VitamineD), G.0302.07 (SVM/Kernel), research
communities (ICCoS, ANMMM, MLDM); G.0733.09 (3UTR); G.082409 (EGFR); o IWT: PhD
Grants, Silicos; SBO-BioFrame, SBO-MoKa, TBM-IOTA3 o FOD:Cancer plans * Belgian
Federal Science Policy Office: IUAP P6/25 (BioMaGNet, Bioinformatics and Modeling:
from Genomes to Networks, 2007-2011 * EU-RTD: ERNSI: European Research Network
on System Identification; FP7-HEALTH CHeartED</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection65">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745836">G-64. Learning ancestral polytrees for HIV-1
mutation pathways against nelfinavir</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Li GD<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Li GD&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*),
Beheydt G<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Beheydt G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bielza C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bielza C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Larrañaga P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Larrañaga P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Camacho RJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Camacho RJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4), Grossman Z<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Grossman Z&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5,6), Torti C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Torti C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7), Zazzi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zazzi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(8), Prosperi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Prosperi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(9,10), Kaiser R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kaiser R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(11), Van Laethem K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Laethem K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Maeyer M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Maeyer M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(12), Jansen M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Jansen M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(13), Vandamme AM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vandamme AM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">Probabilistic graphical models like Bayesian networks or
dependency trees reveal HIV-1 drug resistance pathways. However, the training
data usually originate from different geographic regions with varied viral
subtypes, resulting in sampling biases. Moreover, factors not being encoded in
graphical models (e.g.interactions between HIV inhibitors and host proteins)
might induce confounding effects. To take into account sampling biases and
confounding effects, we use ancestral polytrees to test their robust learning
capacity in the analysis of HIV-1 resistance against NFV, a well studied drug.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We extracted sequences from 3000 PI naïve and 1637
NFV experienced patients from a database containing data from Belgium, Germany,
Israel, Italy, Portugal, Stanford and Sweden. The most prevalent amino acid was
considered the wild type. Amino acids whose prevalence was above 1% were
selected. Wild type was only included if it appeared simultaneously with two
other amino acids. We trained ancestral polytrees using a mutual information
criterion, orientation principles and dependency analysis (Fisher’s exact
test). Robustness was assessed by a non-parametric bootstrap method (100
replicates).</p>

<h3>Results</h3>

<p class="abresults">The subtype distribution was: B (62.5%), G (8.2%), C (7.3%),
A1 (4.1%) and others. Among the selected 105 amino acids (40 wild types), 87
pairs had a high mutual information score (&gt;0.01), while NFV had high scores
only with two mutations, 30N (0.105) and 90M (0.029). NFV ancestral polytree
included 106 variables and 105 edges, of which 76 arcs had bootstrap supports
over 65%, and 47 over 90%. Only 30N (83%) and 90M (69%) were directly linked to
NFV, whereas other known resistance mutations like 10F/I, 46I/L, 71V/T and
88D/S were indirectly linked within 5 arcs away from NFV.</p>

<h3>Discussion</h3>

<p class="abdiscussion">NFV ancestral polytree found 30N and 90M as only
mutations directly linked with NFV, implying their importance. Besides, we
found that 10F/I, 46I/L, 71V/T and 88D/S were close to NFV in the ancestral
polytrees. Mutations at position 19, 36 and 41 do not cluster together while
other positions were highly clustered (bootstrap &gt;65%). Interestingly,
mutations close to NFV in the ancestral polytree were also found to be located
around the NFV binding pocket within 10 angstroms. Although there were major
agreements with mutagenetic trees and Bayesian networks, the differences will
need further study</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Guangdi Li (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67029 = 'g&#117;&#97;ngd&#105;.l&#105;' + '&#64;';
 addy67029 = addy67029 + 'r&#101;g&#97;' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy67029 + suffix + '\'' + attribs + '>' );
 document.write( addy67029 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41149 = 'g&#117;&#97;ngd&#105;.l&#105;' + '&#64;';
 addy41149 = addy41149 + 'r&#101;g&#97;' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy41149 + suffix + '\'' + attribs + '>' );
 document.write( addy41149 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy28560 = 'g&#117;&#97;ngd&#105;.l&#105;' + '&#64;';
 addy28560 = addy28560 + 'r&#101;g&#97;' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy28560 + suffix + '\'' + attribs + '>' );
 document.write( addy28560 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Laboratory for Clinical and Evolutionary Virology,
Rega Institute for Medical Research, Katholieke Universiteit Leuven, Leuven,
Belgium</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Laboratory for Clinical and Evolutionary Virology,
Rega Institute for Medical Research, Katholieke Universiteit Leuven, Leuven,
Belgium, 2 Departamento de Inteligencia Artificial, Universidad Politécnica de
Madrid, Madrid, Spain, 3 Centro de Malária e Outras Doenças Tropicais,
Instituto de Higiene e Medicina Tropical, Universidade Nova de Lisboa, Lisboa,
Portugal, 4 Laboratório de Biologia Molecular, Centro Hospitalar de Lisboa
Ocidental, Lisboa, Portugal, 5 National HIV Reference Lab, Central Virology,
Public Health Laboratories, MOH, Sheba Medical Center, Ramat-gan, Israel, 6
School of Public Health, Sackler Faculty of Medicine, Tel Aviv University,
Tel-Aviv, Israel, 7 Università degli Studi di Brescia, p.le Spedali Civili 1,
25123 Brescia, Italy, 8 Department of Molecular Biology, University of Siena,
Siena, Italy, 9 Catholic University of Sacred Heart, Clinic of Infectious
Diseases, Rome, Italy, 10 Informa SRL, Rome, Italy, 11 Institute of Virology,
University of Cologne, Cologne, Germany, 12 Laboratory for Pharmaceutical
Biology, Faculty of Pharmaceutical Sciences, Katholieke Universiteit Leuven,
Leuven, 13 Department of Mathematics, Katholieke Universiteit Leuven, Leuven,
Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Li GD<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Li GD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>is supported by CSC and K.U.Leuven. This study
was partially supported by the Interuniversity Attraction Poles Programme,
Belgian State, Belgian Science Policy (IAP-VI P6/41). The research leading to
the results has received funding from the European Community's Seventh
Framework Programme (FP7/2007-2013) under the project "Collaborative HIV
and Anti-HIV Drug Resistance Network (CHAIN)" - grant agreement n° 223131.
</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection66">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745837">G-65. Ranking of genes from RNAi perturbation data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Siebourg J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Siebourg J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Beerenwinkel N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Beerenwinkel N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Using the technique of RNA interference, genome wide
knock down screens have been performed. Finding the genes whose knock down has
the greatest impact on the inspected situation, leads to building an importance
ranking of genes according to their experimental values. Creating such a
ranking is a difficult task because several issues have to be faced. Often one
faces the problem of low numbers of replicates in the data that don't allow for
statistical robustness.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">There are a lot of different ranking techniques
available, such as RSA (König 2010) for example, but all lead to different
results. Another group of methods aim to assemble different rankings into a
single more robust one like the rank aggregation (Pihur 2010). We compare different
ranking methods in accessing their stability. For this we use stability
selection (Meinshausen 2009). We also investigate their differences using
distance based approaches (Fagin 2003). The results are validated using primary
and secondary RNAi screening data.</p>

<h3>Results</h3>

<p class="abresults">Single rankings can differ a lot in their individual
results. Especially when only a low number of replicates is available in the
data. We show that nevertheless some methods prove to be more stable then
others and not all are suitable for RNAi data. Since for a secondary validation
screen only a set of the most significant genes is important rather than the
absolute rank values of the genes, we retrieve stable subsets of the data
containing those genes irrespective of small changes in the ranks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Gene rankings are needed in various biological fields and
there is not yet a single perfect method available to find a best ranking. We
show which methods perform better when dealing with RNAi knock down screens and
give an estimation in how trustworthy the resulting ranked gene lists really
are.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Juliane R. E. Siebourg (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy45566 = 'j&#117;l&#105;&#97;n&#101;.s&#105;&#101;b&#111;&#117;rg' + '&#64;';
 addy45566 = addy45566 + 'bss&#101;' + '&#46;' + '&#101;thz' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy45566 + suffix + '\'' + attribs + '>' );
 document.write( addy45566 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy38150 = 'j&#117;l&#105;&#97;n&#101;.s&#105;&#101;b&#111;&#117;rg' + '&#64;';
 addy38150 = addy38150 + 'bss&#101;' + '&#46;' + '&#101;thz' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy38150 + suffix + '\'' + attribs + '>' );
 document.write( addy38150 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31341 = 'j&#117;l&#105;&#97;n&#101;.s&#105;&#101;b&#111;&#117;rg' + '&#64;';
 addy31341 = addy31341 + 'bss&#101;' + '&#46;' + '&#101;thz' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy31341 + suffix + '\'' + attribs + '>' );
 document.write( addy31341 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">ETH Zurich</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) D-BSSE, ETH Zurich</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection67">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745838">G-66. A network centered approach for genome wide
data integration applied to Alzheimers disease prediction</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">van den Akker E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van den Akker E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Heijmans B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Heijmans B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kok J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kok J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, (1,3), Slagboom P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Slagboom P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Reinders M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinders M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Medical research has shifted its focus from
mono-factorial diseases to the more challenging field of multi-factorial
diseases. Often, multiple types of genome wide data sources are created in
search for candidates underlying the complex etiology of disease phenotypes.
Furthermore, an increasing amount of knowledge on molecular interactions could
be employed to help interpret the results of these genome-wide screens. All of
this requires novel ways to analyze the available data that allows a joint
interpretation of multiple data sources, while taking known molecular
interactions into account.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We revisited SNP and expression data generated in a
case-control setting in the Late Onset Alzheimers Disease (LOAD) cohort. In the
methodology presented, we aim to identify subsets of the gene-gene interaction
network predictive of Alzheimers disease using both data sources. We seed a
subnetwork once at every node of the gene-gene interaction network and use a
wrapper approach to extent the network to network neighbors until no
improvement in classification performance is observed. Resulting subnetworks
are validated on an independent set by employing a double loop
cross-validation.</p>

<h3>Results</h3>

<p class="abresults">When analyzed separately, the network centered analysis of
gene expression data as well as the SNP data already yields biologically relevant
signatures for Late Onset Alzheimers Disease. Furthermore, we present a
comparison of predictive performance and biological interpretability of several
ways on how to combine the different data sources: an early integration scheme
that concatenates features, an intermediate scheme that combines distances in
the separate data domains, and a late integration scheme that overlays the
individually found networks.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The work presented here illustrates the benefits of a
network centered approach. By employing the data for identification of
subnetworks in the gene-gene interaction network, the biological interpretation
of genome wide data sources are not confined by arbitrarily set boundaries as
seen in the classical grouped data analysis approaches. The presented framework
is applicable to any type of genome-wide data source and depending on the
biological question asked, offers a great flexibility in different settings for
genome wide data integration.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Erik B. van den Akker (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy73538 = '&#101;r&#105;kv&#97;nd&#101;n&#97;kk&#101;r' + '&#64;';
 addy73538 = addy73538 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy73538 + suffix + '\'' + attribs + '>' );
 document.write( addy73538 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97090 = '&#101;r&#105;kv&#97;nd&#101;n&#97;kk&#101;r' + '&#64;';
 addy97090 = addy97090 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy97090 + suffix + '\'' + attribs + '>' );
 document.write( addy97090 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7220 = '&#101;r&#105;kv&#97;nd&#101;n&#97;kk&#101;r' + '&#64;';
 addy7220 = addy7220 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy7220 + suffix + '\'' + attribs + '>' );
 document.write( addy7220 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Delft BioInformatics Lab, Delft University of
Technology, Delft, The Netherlands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Molecular Epidemiology, Leiden University Medical
Centre, Leiden, The Netherlands (2) The Delft Bioinformatics Lab, Delft
University of Technology, Delft, The Netherlands (3) Algorithms, Leiden
Institute of Advanced Computer Science, University Leiden, Leiden, The
Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This project was funded by the Medical Delta. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection68">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745839">G-67. How to efficiently efficiently hunt the
disease causing gene</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tranchevent L-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tranchevent L-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Aerts S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aerts S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,4), Van Loo P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Loo P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3,4), Hassan BA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hassan BA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,4), Moreau Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moreau Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The development of advanced high-throughput technologies
such as array CGH has led to many breakthrough in human genetics by deciphering
disease-loci associations. One typical feature of array CGH is that the loci
found to be altered usually contain dozens of genes of which only a few are
really causative for the disease / phenotype of interest. The validation of all
the candidate genes is expensive and time consuming, therefore there is a need
for a method that identifies the most promising candidate genes to assay first.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">ENDEAVOUR, a web resource for the prioritization of
genes, indicates which genes are the most promising candidates [1]. It relies
on evidence that suggest that functionally related genes often cause similar
phenotypes [2]. ENDEAVOUR uses various data sources that describe the function
of the genes, their expression profiles, their genetic interactors, and their
regulation processes and integrate them to make one global prediction.</p>

<h3>Results</h3>

<p class="abresults">Our approach has been successfully validated by mean of an
extensive cross-validation on OMIM and MetaCore genetic disorders, and on GO,
MetaCore, and Ingenuity pathways. Furthermore, we have experimentally validated
it with a detailed study on DiGeorge syndrome [1], on neural development in
Drosophila [3], and on congenital heart defects [4].</p>

<h3>Discussion</h3>

<p class="abdiscussion">In conclusion, ENDEAVOUR is a gene prioritization
resource that was extensively validated and is publicly available at
http://www.esat.kuleuven.be/endeavour. 1- Aerts S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aerts S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>et al. Gene prioritization through genomic
data fusion. Nat. Biotechnol. 2006. 2- Jimenez-Sanchez G et al. Human disease
genes. Nature. 2001 3- Aerts et al. Integrating Computational Biology and
Forward Genetics in Drosophila. Plos Genetics. 2009. 4- Thienpont et al.
Haploinsufficiency of TAB2 causes congenital heart defects in humans. Am J Hum
Genet. 2010.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.esat.kuleuven.be/endeavour" mce_href="http://www.esat.kuleuven.be/endeavour">http://www.esat.kuleuven.be/endeavour</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Léon-Charles Tranchevent (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33991 = 'L&#101;&#111;n-Ch&#97;rl&#101;s.Tr&#97;nch&#101;v&#101;nt' + '&#64;';
 addy33991 = addy33991 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy33991 + suffix + '\'' + attribs + '>' );
 document.write( addy33991 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87284 = 'L&#101;&#111;n-Ch&#97;rl&#101;s.Tr&#97;nch&#101;v&#101;nt' + '&#64;';
 addy87284 = addy87284 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy87284 + suffix + '\'' + attribs + '>' );
 document.write( addy87284 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12254 = 'L&#101;&#111;n-Ch&#97;rl&#101;s.Tr&#97;nch&#101;v&#101;nt' + '&#64;';
 addy12254 = addy12254 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy12254 + suffix + '\'' + attribs + '>' );
 document.write( addy12254 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Electrical Engineering ESAT-SCD,
Katholieke Universiteit Leuven, Leuven, Belgium</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Electrical Engineering ESAT-SCD,
Katholieke Universiteit Leuven, Leuven, Belgium. 2 Laboratory of Neurogenetics,
Department of Molecular and Developmental Genetics, VIB, Leuven, Belgium. 3
Human Genome Laboratory, Department of Molecular and Developmental Genetics,
VIB, Leuven, Belgium. 4 Department of Human Genetics, Katholieke Universiteit
Leuven School of Medicine, Leuven, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Research Council KUL
[GOA AMBioRICS, CoE EF/05/007 SymBioSys, PROMETA]; the Flemish Government
[G.0241.04, G.0499.04, G.0232.05, G.0318.05, G.0553.06, G.0302.07, ICCoS,
ANMMM, MLDM, G.0733.09, G.082409, GBOU-McKnow-E, GBOU-ANA, TAD-BioScope-IT,
Silicos, SBO-BioFrame, SBO-MoKa, TBM-Endometriosis, TBM-IOTA3,
O&amp;O-Dsquare]; the Belgian Federal Science Policy Office [IUAP P6/25]; and
the European Research Network on System Identification (ERNSI) [FP6-NoE,
FP6-IP, FP6-MC-EST, FP6-STREP, FP7-HEALTH]. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection69">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745840">G-68. Study of the ultrasensitivity of the BCL-2
apoptotic switch</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tokár T<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Tokár T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Uličný J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Uličný J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Bcl-2 apoptotic switch is important control point of
apoptosis since regulating permeabilization of outer mitochondrial
membrane.Bcl-2 apoptotic switch is molecular mechanisms converting continuous
incoming signals to two mutually distinct outputs, ensuring transitions between
different cellular states. Necessary requirement to generate such behaviour is
ultrasensitive reaction mechanism in which the output response is more
sensitive to change in stimulus then hyperbolic response.There are two
competing hypotheses regarding functioning of Bcl-2 apoptotic switch, the
indirect and direct model.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Based on hypotheses of direct and indirect
activation, we constructed corresponding mathematical models. We introduced
third, hybrid model involving controversial interactions from both, models. In
this work we utilized ultrasensitivity of the Bcl-2 apoptotic switch models as
a criterion, against which we judge plausibility of its models. For each model
we analysed its robustness with respect to its ability to preserve
ultrasensitivity against variations of its parameters around estimated
reference values.We have analysed influence of the most debatable reactions on
the behaviour of our models</p>

<h3>Results</h3>

<p class="abresults">Results of robustness analysis show that it is very
improbable that the indirect model could act as biological switch mechanism.
Moreover, we have found that, while reaction specific for the direct model has
particularly beneficial effect on ultrasensitivity of the Bcl-2 apoptotic
switch, reactions proposed by the indirect model seems to reduce sensitivity in
very strong manner.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The direct model as we proposed in this work, can act as
biological switch, under wide range of parameter settings. We found alternative
variants as inappropriate, since unable to resemble requested behaviour.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tomas Tokar (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60221 = 't&#111;m&#97;st&#111;k&#97;r' + '&#64;';
 addy60221 = addy60221 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy60221 + suffix + '\'' + attribs + '>' );
 document.write( addy60221 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31931 = 't&#111;m&#97;st&#111;k&#97;r' + '&#64;';
 addy31931 = addy31931 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy31931 + suffix + '\'' + attribs + '>' );
 document.write( addy31931 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40701 = 't&#111;m&#97;st&#111;k&#97;r' + '&#64;';
 addy40701 = addy40701 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy40701 + suffix + '\'' + attribs + '>' );
 document.write( addy40701 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">PhD. student</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Biophysics, Faculty of Science,
University of P. J. Safarik, Kosice, Slovakia</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was financially supported by grants
VEGA-1/4019/07; APVV-0449-07 and VVGS PF 12/2009/F. This work could not be done
without scholarships granted to Tomas Tokar from National Scholarship Programme
of the Slovak Republic and ”Hlavicka” scholarship from Slovensky plynarensky
priemysel a.s., we wish to thank for this support. Tomas Tokar appreciate many
valuable discussions with colleagues from Institute for Systems Theory and
Automatic Control, University of Stuttgart, Germany. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection70">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745841">G-69. Improved genome-wide protein-protein
interaction prediction and analysis of biological process coordination in
Escherichia coli</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Muley VY<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Muley VY&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Ranjan A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ranjan A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Protein-protein interactions (PPI) play crucial role in
executing physiological function and to maintain the structural integrity of
cell in diverse envioronements. A number of computational methods have been
proposed for PPI prediction using genomic context. In this study, we check the
possibility of integration of gene expression, correlated mutation and genomic
context based method to enhance the reliability of PPI prediction. Furthermore,
We use panel of seven machine learning classifiers (MLCs) and compare their
performance.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">PPI prediction methods used for analysis are 1.
Phylogenetic profile 2. Minimum Distance Method 3. Gene Expression Similarity 4.
Improved Mirrortree Method 5. Gene Cluster Method 6. Gene Order Conservation We
compute the association score using above methods and feed them to seven MLCs
are as follows 1. Random Forest 2. Decision Tree 3. Naive Bayes 4. Bayesian
Network 5. Logistic Regression 6. Neural Network 7. Support Vector Machine The
gold standard positive dataset is derived from EcoCyc dataset whereas negative
dataset generated by indirect evidence</p>

<h3>Results</h3>

<p class="abresults">In this study, we build a genome-wide interaction map for E.
coli which has two main aspects that could be of potential use to researchers.
First, this work provides a interaction map that is reliable than previous
reports with 10-fold cross-validation balanced accuracy of 93%. Second, this
interaction map makes certain potentially interesting biological predictions:
an antagonistic link between purine catabolism and pyrimidine biosynthesis and
a coupling between cell division, lipopolysaccharide biosynthesis, replication
initiation, ATP synthesis, and colanic acid biosynthesis.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In the present study, we have introduced a consensus
approach for predicting genome-wide functional interactions by a combination of
seven MLCs.Functional analysis of consensus PPI network showed that gene
expression pattern of the predicted non-interacting protein pairs is not
similar even though they belong to the same biological pathway. Topological
properties of networks predicted by various MLCs are not similar. Analysis of
various biological pathways in context of gene expression shows novel
connection between various biological pathways.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.cdfd.org.in/ecofunppi/" mce_href="http://www.cdfd.org.in/ecofunppi/">http://www.cdfd.org.in/ecofunppi/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Vijaykumar Y Muley (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69745 = 'v&#105;j&#97;y.m&#117;l&#101;y' + '&#64;';
 addy69745 = addy69745 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy69745 + suffix + '\'' + attribs + '>' );
 document.write( addy69745 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10305 = 'v&#105;j&#97;y.m&#117;l&#101;y' + '&#64;';
 addy10305 = addy10305 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy10305 + suffix + '\'' + attribs + '>' );
 document.write( addy10305 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60277 = 'v&#105;j&#97;y.m&#117;l&#101;y' + '&#64;';
 addy60277 = addy60277 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy60277 + suffix + '\'' + attribs + '>' );
 document.write( addy60277 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Center for DNA Fingerprinting and Diagnostics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Computational and Functional Genomics Group, Center for
DNA Fingerprinting and Diagnostics, Bldg. 7, Gruhakalpa, 5-4-399 / B, Nampally,
Hyderabad - 500 001</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">CSIR-UGC Fellowship CDFD</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection71">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745842">G-70. A generic gene regulatory network
reconstruction method: application to Lactococcus lactis MG1363</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Theunissen D<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Theunissen D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,*), Brouwer R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brouwer R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,5), Kuipers O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kuipers O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,4,5), Hugenholtz J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Hugenholtz J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Siezen R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Siezen R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,5,6), Van Hijum S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Hijum S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,5,6)</p>

<p class="abmotivation">Gene regulatory networks (GRN) are used by prokaryotes to
adapt to changing environment by regulating gene expression levels. The basic
building blocks of GRN are regulons, which consist of a transcriptional
regulator and target genes. In prokaryotes these target genes are organized
into transcriptional units (TUs). We are developing a generic method to predict
gene regulatory networks for any prokaryote. As a model system for predicting
the GRN the diary bacterium Lactococcus lactis MG1363 was chosen.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our method consists of 4 steps: 1) training a
classification model on known regulatory interactions of E. coli and B.
subtilis. Any interaction occurring between genes part of the same regulonis
used. Features used to predict interactions between two genes include operons,
correlated expression, gene ontologies, metabolic pathways; 2) classifying
interactions between any two genes; 3) identifying quasi-cliques, or dense
sub-graphs of putative co-regulated genes; 4) identifying putative
transcription regulators that govern the expression of the genes part of a
quasi-clique.</p>

<h3>Results</h3>

<p class="abresults">The result of applying the four steps onto gene-pairs of L.
lactis MG1363 is a putative gene-regulatory network. Preliminary results
indicate favorable performance of our method: in general, 60% of the known L.
lactis regulatory interactions are correctly identified. When considering the
smaller regulons, with member sizes of 6 to 14 members, accuracy increases to
over 80%.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In future, we will further develop the method , one way
to do this is in implementing new predictive features. These new features can
help improve the accuracy of our method. Finally we will implement our method
into user-friendly software to accurately determine the GRN for any prokaryote.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Daniel H.J. Theunissen (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13240 = 'dth&#101;&#117;n&#105;s' + '&#64;';
 addy13240 = addy13240 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy13240 + suffix + '\'' + attribs + '>' );
 document.write( addy13240 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62668 = 'dth&#101;&#117;n&#105;s' + '&#64;';
 addy62668 = addy62668 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy62668 + suffix + '\'' + attribs + '>' );
 document.write( addy62668 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy27360 = 'dth&#101;&#117;n&#105;s' + '&#64;';
 addy27360 = addy27360 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy27360 + suffix + '\'' + attribs + '>' );
 document.write( addy27360 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">NIZO food research B.V.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Radboud University Medical Centre, Center for
Molecular and Biomolecular Informatics, Nijmegen Center for Molecular Life
Sciences, P.O. Box 9101, 6500 HB Nijmegen, the Netherlands 2 Kluyver Centre for
Genomics of Industrial Fermentation, P.O. Box 5057, 2600 GA Delft, the
Netherlands 3 NIZO food research, P.O. Box 20, 6710 BA Ede, the Netherlands 4
Molecular Genetics group, GBB, RUG, Kerklaan 30, Haren. 5 Netherlands
Bioinformatics Centre, 260 NBIC, P.O. Box 9101, 6500 HB Nijmegen, the
Netherlands 6 TI Food and Nutrition, P.O. Box 557, 6700 AN Wageningen, the
Netherlands</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection72">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269745843">G-71. Dissecting the specificity of E2-E3
interaction in the ubiquitination pathway by a molecular docking approach</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Melquiond ASJ<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Melquiond ASJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Rodrigues J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rodrigues J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bonvin AMJJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bonvin AMJJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Kinetic and thermodynamical properties of complexes
account for basic regulatory properties. The affinity with which a group of
proteins interacts allows for a certain degree of modulation of these
interactions. Enzyme-inhibitor complexes usually interact strongly, while
proteins involved in regulation or signaling, such as G-coupled receptors,
often interact more dynamically. Starting from proteomics data on ~8000 E2-E3
interactions, we used molecular modeling tools to dissect the forces which
drive the specificity of their interactions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Up to now, only 7 structures of E2-E3 complexes are
solved either by Xray crystallography or NMR spectroscopy. Starting from a set
of Yeast Two-Hybrid assays between 37 E2s and 227 E3s, we used our molecular
docking software HADDOCK to model more than 500 potential complexes. Structural
alignment of E2 and E3 models on the bound structures of the existent
complexes, followed by refinement in water and scoring with HADDOCK’s force
field and scoring function has been shown to discriminate between biologically
relevant complexes and artefacts.</p>

<h3>Results</h3>

<p class="abresults">The result reveal a high correlation between the HADDOCK
score of the predicted complexes and the proteomics data. HADDOCK is equally
fit to predict interactions starring any E3, whatever differences they may
have. Specific complexes show the importance of electrostatic interactions
between charged residues on the loop 1 of E2 and the ubiquitin ligase E3
protein</p>

<h3>Discussion</h3>

<p class="abdiscussion">Some new findings on the role of potential dynamical
intermolecular salt-bridges were considered interesting enough to encourage a
detailed inspection. They led to unexpected findings and proved that this
method can provide priceless insights on the molecular mechanisms that drive
the interaction.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Adrien S.J. Melquiond (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19693 = '&#97;.s.j.m&#101;lq&#117;&#105;&#111;nd' + '&#64;';
 addy19693 = addy19693 + '&#117;&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy19693 + suffix + '\'' + attribs + '>' );
 document.write( addy19693 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65505 = '&#97;.s.j.m&#101;lq&#117;&#105;&#111;nd' + '&#64;';
 addy65505 = addy65505 + '&#117;&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy65505 + suffix + '\'' + attribs + '>' );
 document.write( addy65505 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy17631 = '&#97;.s.j.m&#101;lq&#117;&#105;&#111;nd' + '&#64;';
 addy17631 = addy17631 + '&#117;&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy17631 + suffix + '\'' + attribs + '>' );
 document.write( addy17631 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Utrecht University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) NMR Spectroscopy research group, Bijvoet Center for
Biomolecular Research, Utrecht University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">NWO VICI to A.M.J.J Bonvin (grant no. 700.96.442) </p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="120-protein-interactions-molecular-networks-and-systems-biology.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>