<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Genomic Medicine</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Genomic Medicine			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Genomic Medicine</span><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:
normal !important;text-transform:uppercase"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Niko Beerenwinkel and Yves Moreau<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'>Â </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Genomic Medicine<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746519 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span><span style=""><o:p></o:p></span></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746520" mce_href="#_Toc269746520">H-1. DISCOVERY: a resource for the
rational selection of drug target proteins and leads for the malaria parasite,
Plasmodium falciparum<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746520 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">2</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746521" mce_href="#_Toc269746521">H-2. Variance estimators for t-test
ranking influence the stability and predictive performance of microarray gene
signatures<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746521 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">3</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746522" mce_href="#_Toc269746522">H-3. Selecting small subsets of
genes for predicting the outcome of chemotherapy treatments: a dynamic
programming approach<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746522 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">4</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746523" mce_href="#_Toc269746523">H-4. Networks of generalized top
scoring pairs for robust phenotype classification<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746523 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746524" mce_href="#_Toc269746524">H-5. Comparing network and pathway
based classification for breast cancer: network and pathway based classifiers
do not outperform single gene classifiers<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746524 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746525" mce_href="#_Toc269746525">H-6. Enterotypes of the human gut
microbiome<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746525 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746526" mce_href="#_Toc269746526">H-7. Identifying chemotherapy
resistance genes using outlier detection<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746526 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746527" mce_href="#_Toc269746527">H-8. Genome wide association study
of non-synonymous single nucleotide polymorphisms for seven common diseases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746527 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">9</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746528" mce_href="#_Toc269746528">H-9. Comparative bioinformatics
approach to multiple tumors reveals novel prognostic markers in breast cancer<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1"> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746528 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746529" mce_href="#_Toc269746529">H-10. Genomic and epigenomic
molecular signatures reveals network mechanisms associated with ovarian cancer
prognosis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746529 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500320039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746530" mce_href="#_Toc269746530">H-11. Inferring distributions of
trait-associated SNPs with application to genetic association studies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746530 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500330030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746531" mce_href="#_Toc269746531">H-12. Optimizing exon CGH array
designs for robust rearrangements detection<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746531 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500330031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746532" mce_href="#_Toc269746532">H-13. Interactive human brain atlas
for a better understanding of diseases<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746532 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500330032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746533" mce_href="#_Toc269746533">H-14. Clinical prognosis with
transcriptional association networks and regression trees<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746533 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500330033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746534" mce_href="#_Toc269746534">H-15. Prediction of clinical outcome
after cardiac arrest and induced therapeutic hypothermia<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:
windowtext;display:none;mso-hide:screen;text-decoration:none;text-underline:
none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746534 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500330034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="121-genomic-medicine.html#_Toc269746535" mce_href="#_Toc269746535">H-16. Allele-specific copy number
analysis of breast carcinomas<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746535 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003500330035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746520">H-1. DISCOVERY: a resource for the rational
selection of drug target proteins and leads for the malaria parasite,
Plasmodium falciparum</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Odendaal CJ<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Odendaal CJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Harrison CM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Harrison CM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Szolkiewicz MS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Szolkiewicz MS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Joubert F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Joubert F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1, *)</p>

<p class="abmotivation">In the past, the selection of drug targets and lead
compounds in malaria has been mostly based on serendipitous discoveries and
legacy compounds. Few rational approaches have been successfully followed in
especially the selection of promising drug target proteins in the parasite. The
emergence of widespread drug resistance, even against current drugs is making
the effective selection of new drug targets together with lead compounds
essential and urgent, requiring optimal approaches to be put in place for this
process.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The project is aimed at providing an informatics
resource where comprehensive information on the parasite and host proteins are
stored, together with the results from relevant 3rd-party investigations as
well as from our own high-throughput analysis. Data included in the resource is
aimed as wide as possible, including protein, gene-ontology, orthology,
metabolic, structural, expression, interactome and chemoinformatics
information. This is combined with an interface for researchers to perform the
selection of putative drug target protein and lead compounds.</p>

<h3>Results</h3>

<p class="abresults">Protein information includes data from the human, mosquito
and the various malaria genome projects. Chemical information is from PDB, KEGG
and DrugBank. Information includes basic annotations, motifs, domains, binding
sites, structural features, orthology information, ontology terms,
protein-protein interactions, protein-ligand interactions, pathogen-host
interactions and comparative genomics information. Chemical information
includes protein interactions and ADMET (Absorption, Distribution, Metabolism,
Excretion, and Toxicity) properties.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The researcher accessing the resource is able to perform
advanced searching and filtering of proteins and chemical compounds according
to possible interactions and the different types of properties described in the
database. This may be initiated either from the protein or chemical compound as
starting point. Additional work currently being performed includes the
development of more accurate statistical scoring methods for the predictions, a
literature mining component as well as the inclusion of additional chemical
data sources.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://malport.bi.up.ac.za" mce_href="http://malport.bi.up.ac.za"><span style="" mce_style="mso-no-proof:
yes">http://malport.bi.up.ac.za</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Fourie Joubert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy20584 = 'f&#111;&#117;r&#105;&#101;.j&#111;&#117;b&#101;rt' + '&#64;';
 addy20584 = addy20584 + '&#117;p' + '&#46;' + '&#97;c' + '&#46;' + 'z&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy20584 + suffix + '\'' + attribs + '>' );
 document.write( addy20584 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy68271 = 'f&#111;&#117;r&#105;&#101;.j&#111;&#117;b&#101;rt' + '&#64;';
 addy68271 = addy68271 + '&#117;p' + '&#46;' + '&#97;c' + '&#46;' + 'z&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy68271 + suffix + '\'' + attribs + '>' );
 document.write( addy68271 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy65308 = 'f&#111;&#117;r&#105;&#101;.j&#111;&#117;b&#101;rt' + '&#64;';
 addy65308 = addy65308 + '&#117;p' + '&#46;' + '&#97;c' + '&#46;' + 'z&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy65308 + suffix + '\'' + attribs + '>' );
 document.write( addy65308 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Pretoria</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics and Computational Biology Unit,
University of Pretoria, Pretoria, 0001, South Africa.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">National Bioinformatics Network, National Research
Foundation, Medical Research Council. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746521">H-2. Variance estimators for t-test ranking
influence the stability and predictive performance of microarray gene
signatures</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Touleimat N<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Touleimat N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, HernÃ¡ndez-Lobato D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;HernÃ¡ndez-Lobato D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Dupont P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dupont P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">A Student t-Test is a standard statistical approach for
ranking differentially expressed genes from microarray data. Shrinkage t-Test
and Window t-Test have been proposed to improve the variance estimates when
only very small data sets are available. The choice of variance estimator is
known to influence gene ranking but we study here the effect on the
classification performance of predictive models built from those gene
signatures. We further assess the stability of those gene selection methods
with respect to sampling variation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We consider seven datasets from different cancer
prognosis or diagnosis studies. Repeated 90% (training)-10% (test) resamplings
are performed from each microarray dataset. Gene signatures of increasing sizes
are estimated using the various t-Test ranking methods. Predictive models are
built from the selected signatures with a nearest-centroid classifier using
Pearson correlation. Comparative performances are reported in terms of the
average between specificity and sensitivity over the test samples while
selection stability is measured through the Kuncheva index.</p>

<h3>Results</h3>

<p class="abresults">Our results show that signatures built with the three
ranking methods provide models that have comparable predictive performances for
signature sizes of more than 50 genes. For smaller signature sizes, models
using the shrinkage t-Test offer better performance on 3 data sets (with no significant
difference in the other cases). Shrinkage and Student t-Test clearly outperform
the Window t-Test in terms of selection stability in most cases. Overall, the
shrinkage t-Test offers the best ranking method when looking both at the
classification performance and the stability of the selection.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We study the classification performance of predictive
models for cancer diagnosis or prognosis. A Student t-Test is commonly
performed to select differentially expressed genes from microarray data. Due to
the limited sample size, correcting the variance estimate may be beneficial in
terms of predictive performance. We show that, for signature size with fewer
than 50 genes, the shrinkage t-Test globally outperforms its competitors. We
also observe that the optimal window size for the Window t-Test should be as
small as possible, thus questioning the interest of this variant.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Daniel HernÃ¡ndez-Lobato (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86265 = 'd&#97;n&#105;&#101;l.h&#101;rn&#97;nd&#101;z-l&#111;b&#97;t&#111;' + '&#64;';
 addy86265 = addy86265 + '&#117;cl&#111;&#117;v&#97;&#105;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86265 + suffix + '\'' + attribs + '>' );
 document.write( addy86265 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14902 = 'd&#97;n&#105;&#101;l.h&#101;rn&#97;nd&#101;z-l&#111;b&#97;t&#111;' + '&#64;';
 addy14902 = addy14902 + '&#117;cl&#111;&#117;v&#97;&#105;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy14902 + suffix + '\'' + attribs + '>' );
 document.write( addy14902 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy83583 = 'd&#97;n&#105;&#101;l.h&#101;rn&#97;nd&#101;z-l&#111;b&#97;t&#111;' + '&#64;';
 addy83583 = addy83583 + '&#117;cl&#111;&#117;v&#97;&#105;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy83583 + suffix + '\'' + attribs + '>' );
 document.write( addy83583 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Machine Learning Group, ICTEAM Institute, UniversitÃ©
catholique de Louvain</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Machine Learning Group, ICTEAM Institute, UniversitÃ©
catholique de Louvain, Belgium. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746522">H-3. Selecting small subsets of genes for predicting
the outcome of chemotherapy treatments: a dynamic programming approach</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Natowicz R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Natowicz R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Moraes Pataro C-D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Moraes Pataro C-D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Incitti R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Incitti R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Costa M-A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Costa M-A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Cela A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cela A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Souza T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Souza T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Braga A-P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Braga A-P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Rouzier R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rouzier R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4)</p>

<p class="abmotivation">Accurate and robust predictors of the outcomes of the
chemotherapy treatments are important for clinical purposes, for a better
understanding of the biological mechanisms of the responses to the treatments,
and for the developpement of new treatments dedicated to the patients who are
non responders to the treatments that are presently available. The clinical
trials provide tens of thousands of expression levels measured on very limited
numbers of patient cases, while the expected property of robustness asks the
predictors to be made out of very small sets of relevant genes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">133 patients with stage I-III breast cancer were
included in a clinical trial. The gene pretreatment expression profiling was
performed by Affymetrix U133A microarrays. A responder was a case with no
histopathologic evidence of any residual invasive cancer cells in the breast. The
genes were ranked by a univariate statistics then paired. Let Sn be the first n
pairs, S a subset of Sn, and d(S) the minimum distance between any two
responder and non responder cases relative to S. We search for subsets S* that
maximize this minimum distance over the subsets of Sn.</p>

<h3>Results</h3>

<p class="abresults">Upto n=4950 pairs of genes (made out of the 100 top ranked
genes in the univariate statistics) no more than 24 genes were part of Sn (3-fold
cross validation). In external validation, the predictors made out of these
genes outperformed the best predictors reported so far on the same dataset
(sensitivity = 0.92, specificity = 0.86). An example of optimal subset S* is :
THRAP2 MAPT FBP1 AMFR JMJD2B ATPIF1 MELK HDC GREM1 THUMPD2 X61079 RLN2 SYNCRIP
ZNF131 SLC43A3 IL21 SPDEF</p>

<h3>Discussion</h3>

<p class="abdiscussion">Up to now, no predictor of chemotherapy outcomes can be
said to be used in clinical routine. To this end, the reliability of the
predictors is a central concern. We think that high levels of robustness can be
achieved by predictors made out of very small sets of highly informative and
weakly correlated genes. In this work, we have modeled the selection of such
small sets of genes by a combinatorial optimization approach, and we have
proposed an efficient dynamic programming process to compute them. The results
that we have obtained so far seem to be promising.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.esiee.fr/%7Enatowicr/" mce_href="http://www.esiee.fr/~natowicr/"><span style="">http://www.esiee.fr/~natowicr/</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">RenÃ© Natowicz (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76263 = 'r&#101;n&#101;.n&#97;t&#111;w&#105;cz' + '&#64;';
 addy76263 = addy76263 + '&#117;n&#105;v-p&#97;r&#105;s-&#101;st' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy76263 + suffix + '\'' + attribs + '>' );
 document.write( addy76263 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6825 = 'r&#101;n&#101;.n&#97;t&#111;w&#105;cz' + '&#64;';
 addy6825 = addy6825 + '&#117;n&#105;v-p&#97;r&#105;s-&#101;st' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy6825 + suffix + '\'' + attribs + '>' );
 document.write( addy6825 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16902 = 'r&#101;n&#101;.n&#97;t&#111;w&#105;cz' + '&#64;';
 addy16902 = addy16902 + '&#117;n&#105;v-p&#97;r&#105;s-&#101;st' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy16902 + suffix + '\'' + attribs + '>' );
 document.write( addy16902 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">UniversitÃ© Paris-Est - Esiee-Paris</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) ESIEE-Paris, UniversitÃ© Paris-Est, Paris, France (2)
Universidade Federal de Minas Gerais, Escola de Engenharia, Belo Horizonte,
Brazil (3) INSERM IFR 10, UniversitÃ© de Paris Val-de-Marne, HÃ´pital Henri
Mondor, CrÃ©teil, France (4) INSERM UPRES 4053, UniversitÃ©
Pierre-et-Marie-Curie, HÃ´pital Tenon, Paris, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is supported by CAPES-COFECUB, a
French-Brazilian cooperation program. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746523">H-4. Networks of generalized top scoring pairs for
robust phenotype classification</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Popovici V<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Popovici V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Budinska E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Budinska E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Delorenzi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Delorenzi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Recently, the top scoring pairs classification method has
been proposed for building simple, yet robust, decision rules, based on the
comparison of pairs of genes. While this simple approach works well on a number
of cases, it suffers from the constraint interaction imposed on the variables.
However, by exploring the graph structure of the relationships induced by the
top pairs, we noticed that improved decision rules can be constructed by
considering cliques of genes, all sharing a similar topology. Finally, we are
interested in generalizing the bivariate model used.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use the breast cancer dataset from the MAQC-II
project, on which two endpoints are to be predicted: the estrogen receptor
status and the pathologic complete response. The dataset consists of
independent training and validation sets. We use two different version of the
TSP algorithm: as originally proposed, and our generalized approach (bivariate
logistic regression models). The top N scoring pairs induce a partial ordering
of the genes, that can be represented as an oriented graph with special clique
topologies. These cliques are used for building stronger decision rules.</p>

<h3>Results</h3>

<p class="abresults">A generalized version of the top scoring pairs algorithm is
proposed and implemented. Also, a novel method for combining individual pairs
of variables is introduced. Its performance, assessed by stratified 5-fold
cross-validation, compares favorably with other, more complex classifiers. Finally,
the new algorithm was implemented in a C++ library and designed to take
advantage of the multi-core/multi-processor architectures, reducing drastically
the execution time.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The top N scoring pairs induce a partial ordering among
all the variable. This ordering can be represented as a directed more or less
large directed graph, depending on the difficulty of the classification
problem. This graph has a particular structure that enables us building more
powerful decision rules. However, these rules retain the orignal
interpretability, which makes the proposed approach extremely interesting.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Vlad Popovici (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69734 = 'vl&#97;d.p&#111;p&#111;v&#105;c&#105;' + '&#64;';
 addy69734 = addy69734 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy69734 + suffix + '\'' + attribs + '>' );
 document.write( addy69734 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3161 = 'vl&#97;d.p&#111;p&#111;v&#105;c&#105;' + '&#64;';
 addy3161 = addy3161 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy3161 + suffix + '\'' + attribs + '>' );
 document.write( addy3161 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23548 = 'vl&#97;d.p&#111;p&#111;v&#105;c&#105;' + '&#64;';
 addy23548 = addy23548 + '&#105;sb-s&#105;b' + '&#46;' + 'ch';
 document.write( '<a ' + path + '\'' + prefix + addy23548 + suffix + '\'' + attribs + '>' );
 document.write( addy23548 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Swiss Institute of Bioinformatics</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics Core Facility, Swiss Institute of
Bioinformatics</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">VP and MD acknowledge the support of Swiss National
Science Foundation NCCR Molecular Oncology. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746524">H-5. Comparing network and pathway based
classification for breast cancer: network and pathway based classifiers do not
outperform single gene classifiers</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Staiger C<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Staiger C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2, *), Kooter R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kooter R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Dittrich M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dittrich M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), MÃ¼ller T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;MÃ¼ller T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Klau GW<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Klau GW&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 5), Wessels L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wessels L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2, 3, 5)</p>

<p class="abmotivation">Recently many methods for the classification of cancer
patients based on primary (mRNA) and secondary data (network data derived from
protein-protein interaction (PPI) data or pathway data) have appeared. All
these methods claim to improve the classification of breast cancer patients
compared to single gene classifiers. So far there no study compared all of the
methods against each other and against classification based on single genes
only in an unbiased fashion. We selected three of the methods and compared them
against single gene classifier.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We analysed the prediction of patient outcome of
six breast cancer studies with distant metastasis free survival (DMFS) and
overall survival (OS) as endpoints. We compare the approaches proposed by 1)
Lee et al. which finds predictive subsets of genes from predefined gene sets
contained in e.g. MSigDB; 2) Chuang et al. which is employs a greedy search to
find predictive subnetwork from PPI data and 3) Heinz (Dittrich et al.), an
approach based on an optimal search procedure to find the most predictive PPI
subnetwork and 4) a single gene classifier. As secondary data we used HPRD and
KEGG.</p>

<h3>Results</h3>

<p class="abresults">We find that none of the three methods perform better than
the single gene classifier on the breast cancer data. In contrast we show that
the single gene classifier outperforms all the other network or pathway based
approaches. Furthermore we do not find that the network or pathway based
feature selection methods are more stable across datasets. The features found
by Heinz have slightly higher average overlap as measured by the Jaccard index
but also the variance of the overlap across datasets is higher.</p>

<h3>Discussion</h3>

<p class="abdiscussion">In contrast to Chuang et al. and Lee et al. we find that
their approaches do not improve on single gene classifiers in prediction of
breast cancer outcome. In both studies it was claimed that single gene
classifiers achieved a lower AUC value than in our study. Another network based
approach, Heinz, does not show an improvement over single genes either. In
contrast to Chuang et al., we do not find that the network based methods
stabilize the signatures. Taken together, our results suggest that the employed
network-based approaches do not provide significant benefit in outcome
prediction.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Christine Staiger (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39382 = 'c.st&#97;&#105;g&#101;r' + '&#64;';
 addy39382 = addy39382 + 'cw&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy39382 + suffix + '\'' + attribs + '>' );
 document.write( addy39382 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76036 = 'c.st&#97;&#105;g&#101;r' + '&#64;';
 addy76036 = addy76036 + 'cw&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy76036 + suffix + '\'' + attribs + '>' );
 document.write( addy76036 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32632 = 'c.st&#97;&#105;g&#101;r' + '&#64;';
 addy32632 = addy32632 + 'cw&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy32632 + suffix + '\'' + attribs + '>' );
 document.write( addy32632 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">(1, 2)</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Life Sciences group, CWI, Science Park 123, 1098 XG
Amsterdam, Netherlands (2) Netherlands Cancer Institute, Division of Molecular
Biology, Plesmanlaan 121 1066 CX Amsterdam (3) Delft Bioinformatics Lab,
Faculty of Electrical Engineering, Mathematics and Computer Science, Delft, The
Netherlands (4) Department of Bioinformatics, Biocenter, Am Hubland, D-97074
University of WÃ¼rzburg, Germany (5) Shared last authorship</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746525">H-6. Enterotypes of the human gut microbiome</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Raes J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Raes J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,6,*), Arumugam M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Arumugam M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,6), The Metahit Consortium<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;The Metahit Consortium&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Dore J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dore J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Weissenbach J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weissenbach J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Ehrlich SD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ehrlich SD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Bork P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bork P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,5)</p>

<p class="abmotivation">Our knowledge on species and function composition of the
human gut microbiome is rapidly increasing, but it is still based on very few
cohorts and little is known about their variation across the world.
Computational analysis of gut metagenomics data allows first insights in the
role of our flora in health and disease.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Here, we analysed the fecal metagenomes of 39
individuals from 6 countries at both functional and phylogenetic level.</p>

<h3>Results</h3>

<p class="abresults">We identified several robust clusters (enterotypes
hereafter) that are not nation or continent-specific. This further indicates
the existence of a limited number of well-balanced host-microbial symbiotic
states that might respond differently to diet and drug intake. The enterotypes
are mostly driven by species composition, but the abundance of molecular
functions detected therein does not necessarily correlate with the known
abundant species, highlighting the importance of a functional analysis for a
community understanding.</p>

<h3>Discussion</h3>

<p class="abdiscussion">While individual host properties such as disease state,
age, or gender cannot explain the observed enterotypes, data-driven marker
species, genes or pathways can be identified for each of these host properties.
For example, Eubacterium abundance is linked to nationality, three functional
modules significantly correlate with the body mass index and 11 genes change in
abundance with age, hinting at a diagnostic potential of microbial markers.
This stud shows the power of combined metagenomic and computational approaches
in molecular medicine.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jeroen Raes (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87865 = 'j&#101;r&#111;&#101;n.r&#97;&#101;s' + '&#64;';
 addy87865 = addy87865 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy87865 + suffix + '\'' + attribs + '>' );
 document.write( addy87865 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56481 = 'j&#101;r&#111;&#101;n.r&#97;&#101;s' + '&#64;';
 addy56481 = addy56481 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy56481 + suffix + '\'' + attribs + '>' );
 document.write( addy56481 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21454 = 'j&#101;r&#111;&#101;n.r&#97;&#101;s' + '&#64;';
 addy21454 = addy21454 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy21454 + suffix + '\'' + attribs + '>' );
 document.write( addy21454 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB - Vrije Universiteit Brussel</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 European Molecular Biology Laboratory,
Meyerhofstrasse 1, 69117 Heidelberg, Germany. 2 VIBâVrije Universiteit Brussel,
1050 Brussels, Belgium. 3 Institut National de la Recherche Agronomique, 78350
Jouy en Josas, France. 4 Commissariat Ã  lâEnergie Atomique, Genoscope, 91000
Evry, France. 5 Max DelbrÃ¼ck Centre for Molecular Medicine, D-13092 Berlin,
Germany. 6 Contributed equally</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The research leading to these results has received
funding from the European Community's Seventh Framework Programme
(FP7/2007-2013): MetaHIT, grant agreement HEALTH-F4-2007-201052. JR is
supported by the Odysseus programme of the Fund for Scientific Research
Flanders (FWO). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746526">H-7. Identifying chemotherapy resistance genes
using outlier detection</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">de Ronde J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;de Ronde J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Mulder L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mulder L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Lips E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lips E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Rodenhuis S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rodenhuis S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Wessels L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wessels L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Breast cancer is a heterogeneous disease and although
gene expression analysis or immuno histo chemistry can be used to subclassify
tumors into distinct subgroups, even within these subgroups there exist
significant differences at the molecular level. When it comes to chemotherapy
resistance, it appears highly unlikely that all resistant tumors share the same
mechanism of action. Classical approaches have been unsuccessful in finding
reliable biomarkers for therapy resistance and this may be due to the fact that
these approaches are not sensitive to changes in small subgroups of samples.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Traditional analyses approaches, like a t-test or
using the SAM approach, compare the two groups of samples for each gene, and
identify genes with a significant difference in expression between the two
groups. If only a small subset of the resistant tumors would show aberrant
expression indicative of a resistance mechanism, then this would not be picked
up by such approaches. Using a novel algorithm we try to circumvent this
problem and specifically aim to find relatively small subgroups of tumors
within the resistant group that show differential expression compared to the
sensitive group.</p>

<h3>Results</h3>

<p class="abresults">Using a positive control set, where we mixed in a small set
of HER2-positive tumors in a larger HER2-negative set and by generating an
artificial dataset we show that our algorithm is able to pick up all positive
controls. These controls are not picked up by either a t-test or the SAM
algorithm. Next, we applied the algorithm on a gene expression set of 195
patients that were neoadjuvantly treated and identified a number of genes that
show an expression pattern that suggests a role in chemotherapy resistance in
ER+ tumors. We validate our findings on a separate set of 90 patients.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have developed a novel algorithm that allows the
identification of genes that show aberrant expression, relative to the
reference group, in a small subset of the samples. Using this approach we
identified a number of genes that are linked to chemotherapy resistance. The
algorithm can be used in any type of analysis that involves the detection of
small subsets of samples within one of the labeled classes, where the small
subset shows behavior different from the average behavior of the remaining
samples. Our approach can therefore be applied to a wide range of problems.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jorma J. de Ronde (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy21657 = 'j.d.r&#111;nd&#101;' + '&#64;';
 addy21657 = addy21657 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy21657 + suffix + '\'' + attribs + '>' );
 document.write( addy21657 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy78976 = 'j.d.r&#111;nd&#101;' + '&#64;';
 addy78976 = addy78976 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy78976 + suffix + '\'' + attribs + '>' );
 document.write( addy78976 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14210 = 'j.d.r&#111;nd&#101;' + '&#64;';
 addy14210 = addy14210 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy14210 + suffix + '\'' + attribs + '>' );
 document.write( addy14210 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Netherlands Cancer Institute, Amsterdam, The
Netherlands</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Bioinformatics and Statistics, The
Netherlands Cancer Institute, Amsterdam, The Netherlands 2. Department
Experimental Therapy, The Netherlands Cancer Institute, Amsterdam, The
Netherlands 3. Department of Medical Oncology, The Netherlands Cancer Institute,
Amsterdam, The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This study was performed within the framework of CTMM,
the Center for Translational Molecular Medicine (www.ctmm.nl), project Breast
CARE (grant 03O-104). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746527">H-8. Genome wide association study of
non-synonymous single nucleotide polymorphisms for seven common diseases</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Surendran P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Surendran P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Shields D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Shields D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Stanton A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Stanton A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Associations of several Single Nucleotide Polymorphisms
(SNPs) with seven common diseases were identified in a study conducted by the
Wellcome Trust Case Control Consortium. We hypothesize that there are more
chances of finding association of rare SNPs with diseases by refined analysis
of non synonymous SNPs (nsSNPs) in genome wide association studies and expect
better results in WTCCC data by using larger SNPs datasets which were published
after the original WTCCC study. In the present study we analyzed the
association of 12,660 nsSNPs using a case control study in the WTCCC
population.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We simulated the genotypes at 10,798 nsSNP loci
studied by the Stage 2 HapMap project using the genotype information from WTCCC
for all 14,000 individuals studied for seven diseases and in 3000 controls. We
performed these simulations or imputations of missing SNPs using two widely
used programs called IMPUTE and MACH after standard quality control checks
using programs in PLINK genome wide analysis package. Subsequent case control
association of 10,798 imputed nsSNPs and 1,862 genotyped nsSNPs was performed
using an additive model and genotype model in a frequentist and bayesian framework.</p>

<h3>Results</h3>

<p class="abresults">We found 4 nsSNPs associated with Crohn's Disease, 8 with
Rheumatoid Arthritis, 5 with Type 1 Diabetes and 1 with Type 2 Diabetes. In
total, 18 new associations with the seven diseases (p &lt; 5 x 10<span style="font-family: &quot;Cambria Math&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Cambria Math&quot;,&quot;serif&quot;;">â</span>6)
studied by WTCCC. We also developed a pipeline which summarizes quality control
measures which should be considered to minimize false associations in genome
wide association studies.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Among the 18 SNPs which are found associated with
diseases, 5 SNPs are found in genetic loci which were not identified in the
previous WTCCC study. All the other SNPs are found adjacent to the SNPs
previously identified and are in high linkage disequilibrium with the
significant SNPs. These non synonymous SNPs can play a significant role in
modulating protein-protein interactions with substitutions in interacting
regions of proteins like in domains, motifs or disordered regions of proteins.
Such relations are of critical importance in direct or indirect disease causation.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Praveen Surendran (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32153 = 'pr&#97;v&#101;&#101;n.s&#117;r&#101;ndr&#97;n' + '&#64;';
 addy32153 = addy32153 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy32153 + suffix + '\'' + attribs + '>' );
 document.write( addy32153 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9283 = 'pr&#97;v&#101;&#101;n.s&#117;r&#101;ndr&#97;n' + '&#64;';
 addy9283 = addy9283 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9283 + suffix + '\'' + attribs + '>' );
 document.write( addy9283 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4726 = 'pr&#97;v&#101;&#101;n.s&#117;r&#101;ndr&#97;n' + '&#64;';
 addy4726 = addy4726 + '&#117;cd' + '&#46;' + '&#105;&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy4726 + suffix + '\'' + attribs + '>' );
 document.write( addy4726 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University College Dublin</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 School of Medicine and Medical Science, University
College Dublin, Belfield, Dublin 4, Ireland. 2Department of Clinical
Pharmacology, Royal College of Surgeons in Ireland, Dublin<span style="font-family: &quot;Cambria Math&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Cambria Math&quot;,&quot;serif&quot;;">â</span>1,
Ireland.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was funded by Embark Initiative, Graduate
Research Education Program of Irish Research Council for Science Engineering
and Technology. Computational resources were provided by ICHEC, Ireland's
High-Performance Computing Centre. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746528">H-9. Comparative bioinformatics approach to
multiple tumors reveals novel prognostic markers in breast cancer</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Krupp M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Krupp M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Marquardt J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marquardt J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Maas T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Maas T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Galle P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Galle P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tresch A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tresch A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Teufel A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Teufel A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Over the last decade microarray studies have extensively
been applied in tumor research and the obtained results have lead to a more
systemic view on tumor development. However until now only partial efforts have
been performed to analyze this data in context of comparative genomics between
multiple tumor entities to unravel similarities in the gene expression profiles
and subsequently their biological behaviour. Ultimately, such investigation may
lead to a deeper understanding and extended application of target therapeutic
approaches as well as to the discovery of potential prognosis markers.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">1402 tumor associated microarrays were downloaded
from the Standford Microarray Database and analysed for enriched pathway
modules obtained from the Kyoto Encyclopaedia of Genes and Genomes. Furthermore
we adopted survival data on 99 breast cancer patients with a follow up of 6.1
years. This data comprises 7650 probes which can be assigned to 33 patients
with bad and 66 with good outcome. Determination of possible pathways
associated with patients survival was calculated by applying the gene set
enrichment analysis (GSEA) computational method to the data.</p>

<h3>Results</h3>

<p class="abresults">We analysed a dataset of 649 tumor associated microarrays,
corresponding to 16 tumor entities. Analyzing these data, we found several signaling
pathways referenced by KEGG to be significantly enriched. Two pathway
categories were remarkably conserved in the tumors leading to the assumption
that they are essential key player in tumor progression. On this basis we
adopted breast cancer prognostic data to our dataset and the performed GSEA to
the differentially expressed genes located in those conserved pathway
categories has shown new prognostic markers to this tumor entity.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Together, we provide a comparative analysis of pathways enriched
in multiple tumor entities and disclosed new prognostic markers by applying gen
set enrichment analysis to pathway groups highly conserved among the tumors.
Furthermore we summarize the multiple tumor pathway relations in a clearly
structured tumor pathway map enabling scientist to get a systematic overview
about tumor affinity and central tumor mechanisms.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Markus Krupp (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60257 = 'kr&#117;ppm' + '&#64;';
 addy60257 = addy60257 + '&#117;n&#105;-m&#97;&#105;nz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy60257 + suffix + '\'' + attribs + '>' );
 document.write( addy60257 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53765 = 'kr&#117;ppm' + '&#64;';
 addy53765 = addy53765 + '&#117;n&#105;-m&#97;&#105;nz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy53765 + suffix + '\'' + attribs + '>' );
 document.write( addy53765 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57334 = 'kr&#117;ppm' + '&#64;';
 addy57334 = addy57334 + '&#117;n&#105;-m&#97;&#105;nz' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy57334 + suffix + '\'' + attribs + '>' );
 document.write( addy57334 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Medicine I, Johannes Gutenberg University,
Mainz, Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Medicine I, Johannes Gutenberg
University, Mainz, Germany 2 National Cancer Institute, National Institutes of
Health, United States 3 Gene Center Munich, Department of Chemistry and
Biochemistry, Ludwig-Maximilians-University, Munich, Germany</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746529">H-10. Genomic and epigenomic molecular signatures
reveals network mechanisms associated with ovarian cancer prognosis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ben-Hamo R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Ben-Hamo R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, Efroni S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Efroni S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Epithelial ovarian cancer causes more deaths than any
other female gynecologic cancer. A better understanding of the molecular
mechanisms in advanced ovarian cancer may improve patient treatment.
Identification of molecular interactions that stratify prognosis may be the key
for such novel treatments. Gene methylation, copy number variation and gene
expression are characterizing factors in malignancies. The Cancer Genome Atlas
(http://cancergenome.nih.gov/), a large multi center coordinated effort, has
recently made available the molecular characteristics of more than 200 patients.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">All data obtained from The TCGA and validation data
obtained from the Duke Cancer Center. Data consists of CNV, Methylation, Gene
expression and clinical annotation. Pathway Consistency and pathway Activity
metrics, which overlay expression data over network knowledge, were calculated
according to (Efroni,et al,2007). Pathway targeting by genomic and epigenomic
alterations were calculated according to Fisher's omnibus test. Metrics for the
set of these highly significant pathways were then clustered to stratify
patients into two groups. Survival analyses was done using Kaplan-Meier.</p>

<h3>Results</h3>

<p class="abresults">Using this multi analyte study and a set of computational
algorithms we have recently developed, we were able to identify subnetworks
that significantly stratify survival rates. Expression levels of single genes
do not explain the prognostic stratification. Only when we apply network
effects phenotypically distinct groups emerge. Our results demonstrate specific
pathway interactions whose synthetic genetic and epigenetic interactions drive
prognosis in ovarian cancer through combined alterations in expression. These
interactions are associated with disease outcome.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Here, by combining CNV, Methylation and gene expression
features we were able to efficiently stratify prognosis. The selection of a set
of subnetworks whose transcriptional behavior successfully stratifies the
prognosis can identify the most influential molecular agents in the network and
the key interactions. By isolating specific subnetworks, we were able to handle
the NP-hard numeracy of network interactions. Further analysis revealed
specific interactions at the core of the phenotypic clustering.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Rotem Ben-Hamo (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63703 = 'r&#111;t&#101;m830' + '&#64;';
 addy63703 = addy63703 + 'w&#97;ll&#97;' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy63703 + suffix + '\'' + attribs + '>' );
 document.write( addy63703 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy70343 = 'r&#111;t&#101;m830' + '&#64;';
 addy70343 = addy70343 + 'w&#97;ll&#97;' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy70343 + suffix + '\'' + attribs + '>' );
 document.write( addy70343 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39922 = 'r&#111;t&#101;m830' + '&#64;';
 addy39922 = addy39922 + 'w&#97;ll&#97;' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy39922 + suffix + '\'' + attribs + '>' );
 document.write( addy39922 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Mina and Everard Goodman Faculty of Life Science,
Bar Ilan University, Ramat-Gan, 52900, Israel.</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">The Mina and Everard Goodman Faculty of Life Science,
Bar Ilan University, Ramat-Gan, 52900, Israel. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746530">H-11. Inferring distributions of trait-associated
SNPs with application to genetic association studies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Neuvirth H<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Neuvirth H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Aharoni E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aharoni E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Azencott C-A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Azencott C-A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Farkash A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Farkash A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Landau D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Landau D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Rosen-Zvi M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rosen-Zvi M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Geiger D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Geiger D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,4)</p>

<p class="abmotivation">Hundreds of recent genetic association studies provide an
increasingly more viable opportunity to derive generalizing principles regarding
SNPs' association potential and utilize these inferences for improving new GWAS
studies.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Several features of SNPs have been identified as
being significantly over-represented in trait-associated SNPs (TASs) based on
published data for 19 diseases. A regression model based on these features and
the published data has been constructed. For each SNP in a given SNP panel, the
model provides the probability of being relevant to a trait under study, before
the SNP data of this study is examined. The resulting probabilities are used as
weights that increase or decrease the needed threshold for the p-value of each
SNP using standard association methods.</p>

<h3>Results</h3>

<p class="abresults">For 16 out of the 19 diseases, SNPs judged to be associated
with the disease in previous studies have been promoted by the computed
weights, and were practically unchanged for the remaining disease. Repeating a real
study analysis of type-2 diabetes showed an increase in the number of relevant
SNPs found for every sample size in this study.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This work is the first effort to perform the full
supervised analysis. It applies large scale learning models that generalize a
sizeable collection of studies in order to generate an a-priori prioritization of
all SNPs on a given chip and consequently improve the outcomes of new GWAS
studies. Our method allows a smaller sample to be collected for GWAS studies
and enhances the statistical power for a fixed sample size.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://srv-rimon.haifa.il.ibm.com:8080/rimon_web/snpWeights.jsp" mce_href="http://srv-rimon.haifa.il.ibm.com:8080/rimon_web/snpWeights.jsp"><span style="">http://srv-rimon.haifa.il.ibm.com:8080/rimon_web/snpWeights.jsp</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Michal Rosen-Zvi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64690 = 'r&#111;s&#101;n' + '&#64;';
 addy64690 = addy64690 + '&#105;l' + '&#46;' + '&#105;bm' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy64690 + suffix + '\'' + attribs + '>' );
 document.write( addy64690 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90926 = 'r&#111;s&#101;n' + '&#64;';
 addy90926 = addy90926 + '&#105;l' + '&#46;' + '&#105;bm' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy90926 + suffix + '\'' + attribs + '>' );
 document.write( addy90926 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy8192 = 'r&#111;s&#101;n' + '&#64;';
 addy8192 = addy8192 + '&#105;l' + '&#46;' + '&#105;bm' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy8192 + suffix + '\'' + attribs + '>' );
 document.write( addy8192 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Machine Learning and Data Mining group, IBM Haifa
research labs</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Machine Learning and Data Mining Group, IBM
Research Laboratory in Haifa, Israel. (2) School of Information and Computer
Sciences, Institute for Genomics and Bioinformatics, University of California, Irvine,
California 92697-3435 (3) IT for Healthcare and Life Sciences Group, IBM
Research Laboratory in Haifa, Israel. (4) Technion - Israel Institute of
Technology, Computer Science Department, Haifa, 36000, Israel</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The research leading to these results has received
funding from the European Community's Seventh Framework Program FP7/2007-2013
under the project HYPERGENES, grant agreement nÂ° 201550. This study makes use
of data generated by the Wellcome Trust Case-Control Consortium. A full list of
the investigators who contributed to the generation of the data is available
from www.wtccc.org.uk. Funding for the project was provided by the Wellcome
Trust under award 076113 and 085475. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746531">H-12. Optimizing exon CGH array designs for robust
rearrangements detection</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Gambin T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Gambin T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Sykulski M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sykulski M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Gambin A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gambin A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2)</p>

<p class="abmotivation">Array-comparative genomic hybridization (aCGH) technology
enables rapid, high-resolution analysis of genomic rearrangements. With the use
of it, genome copy number changes and rearrangement breakpoints can be detected
and analyzed at resolutions down to a few kilobases. An exon array CGH approach
proposed recently accurately measures copy-number changes of individual exons
in the human genome. The crucial and highly non-trivial starting task is the
design of an array, i.e. the choice of appropriate set of oligos. The success
of the whole high-level analysis depends on the design quality.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The dataset, we used in our study, come from 60
arrays hybridized with DNA from subjects with epilepsy, autism, heart defects
and mental disorders. Each experiment was performed on the 180 K exon targeted
oligonucleotide array. Based on those data we generated several smaller
(optimized) designs by selecting various subsets of oligos from original one.
Aiming in testing the robustness of segmentation we enhance the DNAcopy method
by incorporating parametrized noise model. Finally, we used this algorithm to
perform the comparison among reduced designs.</p>

<h3>Results</h3>

<p class="abresults">For each probe we computed, using a set of
Kolmogorov-Smirnov tests, cumulative properties, which reflects the oligo
suitability in the context of its surrounding. To investigate the influence of
design optimization strategy on segmentation robustness several approaches for
probes selection were tested, including uniform sampling and most/least
suitable oligo removal. Some of those methods reduced the nr of probes with a
little loss of segmentation robustness. One can benefit from this strategy
especially for targeted arrays used for the diagnosis of specific chromosomal
aberrations.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The investigation shows that while optimizing the design
it is crucial to find a tradeoff between keeping uniform distribution and
selecting the best performing probes. We discovered that the results of designs
comparisons greatly depends on the definition of distance between two
segmentations. Finally, we found new robustness measure very useful in
evaluation of optimized design performance of rearrangement detection, and its
resistance to the noise, in comparison to the original array.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="mailto:http://bioputer.mimuw.edu.pl/papers/robustness/index.php" mce_href="mailto:http://bioputer.mimuw.edu.pl/papers/robustness/index.php"><span style="">http://bioputer.mimuw.edu.pl/papers/robustness/index.php</span></a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tomasz Gambin (
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29998 = 'tg&#97;mb&#105;n' + '&#64;';
 addy29998 = addy29998 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy29998 + suffix + '\'' + attribs + '>' );
 document.write( addy29998 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>)</p>

<p class="presentingauthor">Institute of Computer Science, Warsaw University of
Technology, ul. Nowowiejska 15/19, 00-665 Warsaw, Poland</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Institute of Computer Science, Warsaw University of
Technology, ul. Nowowiejska 15/19, 00-665 Warsaw, Poland (2) Institute of Informatics,
University of Warsaw, ul. Banacha 2, 02-097 Warsaw, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is supported in part by Polish Ministry
of Science and Educations grants N301 065236 and N206 356036. It is also
supported by the Foundation for Polish Science and the European Social Fund and
the State Budget from the Integrated Regional Operational Program, Action 2.6
"Regional Innovation Strategies and Knowledge Transfer", the project
of Mazovia Voivodeship "Mazovia Doctoral Scholarship".</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746532">H-13. Interactive human brain atlas for a better
understanding of diseases</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kremer A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kremer A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*), Gaenzler C (2) and van der Spek P (1)</p>

<p class="abmotivation">The bioinformatics department within the Erasmus MC is
supporting many large scale clinical research projects. One major challenge is
the integration of genomics, proteomics and cytogenetic data with medical
imaging data to identify genes associated with neuro-development,
neuro-degeneration and neuro-oncology. The aim of our research is to gain
insight in the molecular and cellular mechanisms driving the development of the
face and the brain.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The ever increasing complexity and amount of
research data require new approaches to data analysis and visualization. Here
we describe an application which was developed together with TIBCO Spotfire
that facilitates the integration and visualization of gene expression data in
the context of the brain anatomy.</p>

<h3>Results</h3>

<p class="abresults">These new brain maps are useful research tools and are an
example for the successful integration of medical and biological data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Linking molecular with imaging data opens new avenues for
image guided diagnosis and intervention.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Andreas Kremer (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77191 = '&#97;.kr&#101;m&#101;r' + '&#64;';
 addy77191 = addy77191 + '&#101;r&#97;sm&#117;smc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy77191 + suffix + '\'' + attribs + '>' );
 document.write( addy77191 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23094 = '&#97;.kr&#101;m&#101;r' + '&#64;';
 addy23094 = addy23094 + '&#101;r&#97;sm&#117;smc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy23094 + suffix + '\'' + attribs + '>' );
 document.write( addy23094 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13580 = '&#97;.kr&#101;m&#101;r' + '&#64;';
 addy13580 = addy13580 + '&#101;r&#97;sm&#117;smc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy13580 + suffix + '\'' + attribs + '>' );
 document.write( addy13580 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Erasmus MC</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Erasmus Medical Center, Department Bioinformatics (2)
TIBCO Software GmbH</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746533">H-14. Clinical prognosis with transcriptional
association networks and regression trees</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nepomuceno I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nepomuceno I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Azuaje F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Azuaje F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Devaux Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Devaux Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Nazarov PV<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nazarov PV&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Muller A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Muller A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Aguilar-Ruiz JS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aguilar-Ruiz JS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Wagner DR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wagner DR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,5)</p>

<p class="abmotivation">The application of information encoded in molecular
networks for prognostic purposes is a crucial objective of systems biomedicine.
This approach has not been widely investigated in the cardiovascular research
area where the prediction of clinical outcomes after suffering a heart attack
would represent a significant step forward. We developed a new supervised
prediction method for this prognostic problem based on the discovery of
clinically-relevant gene association networks. The method integrates model
trees and clinical class-specific networks. It can be applied to other clinical
domains.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our method consists of two main steps. The first
involves the inference of class-specific networks. Each network is inferred
from gene expression of patients with the same clinical class. In order to do
it, each gene is analyzed by taking into account the remaining genes as inputs
to M5P model tree algorithm. M5P focus on localized similarities. The second
step involves predicting the clinical class of a new patient through the
inferred networks. The prediction is based on the relative error between the
true and predicted expression values of those genes involved in the inferred
networks.</p>

<h3>Results</h3>

<p class="abresults">We assessed the predictive performance of our approach on a
benchmark dataset and we compared the predictive capability against other
classifiers applied on the network information inferred from several techniques
based on co-expressed gene modules. The predictive strength is reflected in a high
AUC value (0,85). Furthermore, we analyzed the gene expression profiles of
patients with acute MI that is divided into good and bad prognosis groups. We
also detected biological processes which may be used to characterise and
possibly treat the development of ventricular dysfunction after MI.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our method discovered small biologically-meaningful
networks, which facilitate human expert interpretation. Furthermore, we
predicted processes that may represent novel therapeutic targets for heart disease,
such as the synthesis of leucine and isoleucine. These results encouraged us to
investigate our method as a new strategy to discover potential biomarkers of
clinical outcome after MI and one of the strengths of this method is that it
can be applied to other prognostic problems. As drawback of this work, we can
mention the limited number of gene expression profiles under analysis.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Isabel Nepomuceno (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy53453 = '&#105;n&#101;p&#111;m&#117;c&#101;n&#111;' + '&#64;';
 addy53453 = addy53453 + '&#117;s' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy53453 + suffix + '\'' + attribs + '>' );
 document.write( addy53453 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29919 = '&#105;n&#101;p&#111;m&#117;c&#101;n&#111;' + '&#64;';
 addy29919 = addy29919 + '&#117;s' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy29919 + suffix + '\'' + attribs + '>' );
 document.write( addy29919 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy30482 = '&#105;n&#101;p&#111;m&#117;c&#101;n&#111;' + '&#64;';
 addy30482 = addy30482 + '&#117;s' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy30482 + suffix + '\'' + attribs + '>' );
 document.write( addy30482 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universidad de Sevilla</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Dpt. Lenguajes y Sistemas InformÃ¡ticos, Universidad
de Sevilla,Spain 2 Laboratory of Cardiovascular Research, CRP-SantÃ©, Luxembourg
3 Microarray Center, CRP-SantÃ©, Luxembourg 4 School of Engineering, Pablo de
Olavide University, Spain 5 Division of Cardiology, Centre Hospitalier,
Luxembourg</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">In Luxembourg this research was in part supported by:
Fonds National de la Recherche (Luxembourg), SociÃ©tÃ© pour la Recherche sur les
Maladies Cardiovasculaires, and MinistÃ©re de la Culture, de l'Enseignement
SupÃ©rieur et de la Recherche. In Spain, it was in part supported by the Spanish
Ministry of Science and Innovation under Grant TIN2007--68084--C02--00, and by
the Plan Propio of the University of Seville. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746534">H-15. Prediction of clinical outcome after cardiac
arrest and induced therapeutic hypothermia</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nepomuceno I<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nepomuceno I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Azuaje F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Azuaje F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Devaux Y<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Devaux Y&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Stammet P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Stammet P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Aguilar-Ruiz JS<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aguilar-Ruiz JS&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Wagner DR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wagner DR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,5)</p>

<p class="abmotivation">Standard approaches to biomarker discovery are based on
the identification of differentially expressed genes. However, it is known that
known biomarkers may be encoded by genes that are not highly differentially
expressed across control and disease patients. Network-based prognostic
approaches have not been widely investigated in the cardiovascular research
area where the prediction of clinical outcome would represent a significant
contribution to translational research. We developed a new supervised
prediction method based on the discovery of clinically-relevant gene
association networks.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Our method consists of two main steps. The first
involves the inference of class-specific networks. Each network is inferred
from gene expression profiles of patients with the same clinical class. In
order to do it, each gene is analyzed by taking into account the remaining
genes as inputs to M5P model tree algorithm. The second step involves
predicting the clinical class of a new patient through the inferred
transcriptional networks. The prediction is based on the relative error between
the true and predicted expression values of those genes involved in the
inferred networks.</p>

<h3>Results</h3>

<p class="abresults">We analyzed gene expression profiles of blood cells obtained
from 35 CA patients treated with therapeutic hypothermia: 21 patients had a
normal brain function and 14 patients showed signs of brain dysfunction after
hypothermia. Our method discovered small biologically-meaningful networks,
which facilitate human expert interpretation. An average classification
accuracy of 75% was obtained after leave-one-out cross-validation. We also
detected biological processes which may be used to characterize and possibly
prevent the adverse neurological outcome in comatose survivors.</p>

<h3>Discussion</h3>

<p class="abdiscussion">A key outcome of this investigation was the discovery of
new potential biomarkers for predicting neurological outcome after CA. Although
the method has been used to predict favourable and unfavourable prognosis, it
can be extended to infer more than two clinical outcomes.. The predictions
reported here will require to be validated in larger and independent patient
cohorts. Further in vitro investigations will be necessary to test the
functional significance of the biomarkers discovered in this study and to
determine their involvement in the appearance of neurological sequels.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Isabel Nepomuceno (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy23187 = '&#105;n&#101;p&#111;m&#117;c&#101;n&#111;' + '&#64;';
 addy23187 = addy23187 + '&#117;s' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy23187 + suffix + '\'' + attribs + '>' );
 document.write( addy23187 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33080 = '&#105;n&#101;p&#111;m&#117;c&#101;n&#111;' + '&#64;';
 addy33080 = addy33080 + '&#117;s' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy33080 + suffix + '\'' + attribs + '>' );
 document.write( addy33080 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy54029 = '&#105;n&#101;p&#111;m&#117;c&#101;n&#111;' + '&#64;';
 addy54029 = addy54029 + '&#117;s' + '&#46;' + '&#101;s';
 document.write( '<a ' + path + '\'' + prefix + addy54029 + suffix + '\'' + attribs + '>' );
 document.write( addy54029 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universidad de Sevilla</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Dpt. Lenguajes y Sistemas InformÃ¡ticos, Universidad
de Sevilla,Spain 2 Laboratory of Cardiovascular Research, CRP-SantÃ©, Luxembourg
3 Dpt. Anaesthesia and Intensive Care, Centre Hospitalier, Luxembourg 4 School
of Engineering, Pablo de Olavide University, Spain 5 Division of Cardiology,
Centre Hospitalier, Luxembourg</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">In Luxembourg this research was in part supported by:
Fonds National de la Recherche (Luxembourg), SociÃ©tÃ© pour la Recherche sur les
Maladies Cardiovasculaires, and MinistÃ©re de la Culture, de l'Enseignement
SupÃ©rieur et de la Recherche. In Spain, it was in part supported by the Spanish
Ministry of Science and Innovation under Grant TIN2007--68084--C02--00, and by
the Plan Propio of the University of Seville. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746535">H-16. Allele-specific copy number analysis of
breast carcinomas</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Van Loo P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Van Loo P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,3,12,*), Nordgard SH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nordgard SH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,4,12), LingjÃ¦rde OC<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;LingjÃ¦rde OC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5), Russnes HG<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Russnes HG&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,6,7), Rye IH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rye IH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Sun W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sun W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,8), Weigman VJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weigman VJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Marynen P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Marynen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Zetterberg A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zetterberg A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(9), Naume B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Naume B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(10), Perou CM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Perou CM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), BÃ¸rresen-Dale A-L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;BÃ¸rresen-Dale A-L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,7,13), Kristensen VN<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kristensen VN&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,7,11,13)</p>

<p class="abmotivation">Whole genome SNP genotyping is an expanding technology to
measure genomic aberrations in an allele-specific manner. However, to
accurately index all genomic aberrations in a cancer sample, both the ploidy of
the cancer cells and the infiltration of non-aberrant cells need to be
accounted for in the analysis. We infer tumor ploidy, non-aberrant cell admixture
and genome-wide allele-specific copy-number profiles from genome-wide SNP data
of breast cancers and identify specific signatures of aberrations in breast
carcinoma and breast carcinoma subtypes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We performed genotyping of 112 breast carcinoma
samples using Illumina 109K SNP arrays and constructed an algorithm (ASCAT,
Allele-Specific Copy number Analysis of Tumors) to estimate the fraction of
aberrant cells and the tumor ploidy, and to index all genomic aberrations
taking both properties into account. This allows calculation of âASCAT
Profilesâ (genome-wide allele-specific copy-number profiles) from which gains,
losses, copy-number-neutral events and LOH can accurately be determined.</p>

<h3>Results</h3>

<p class="abresults">We present the first allele-specific copy number analysis of
the in vivo breast cancer genome. We obtained ASCAT Profiles for 91 of the
breast carcinomas (81 %). We observe aneuploidy (&gt;2.7n) in 45% of the cases
and an average non-aberrant cell admixture of 49%. We obtain first-time genome-wide
views of LOH and copy-number-neutral events in breast cancer. In addition, the
ASCAT Profiles reveal differences in aberrant tumor cell fraction, ploidy,
gains, losses, LOH and copy-number-neutral events between the five previously
identified molecular breast cancer subtypes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Basal-like breast carcinomas have a significantly higher
frequency of LOH compared to other subtypes, and their ASCAT Profiles show
large-scale loss of genomic material during tumor development, followed by a
whole-genome duplication, resulting in near-triploid genomes. Finally, from the
ASCAT Profiles, we can construct a genome-wide map of allelic skewness in
breast cancer, indicating loci where one allele is preferentially lost while
the other allele is preferentially gained. We hypothesize that these
alternative alleles have a different influence on breast carcinoma development.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Peter Van Loo (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62568 = 'P&#101;t&#101;r.V&#97;nL&#111;&#111;' + '&#64;';
 addy62568 = addy62568 + 'm&#101;d' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy62568 + suffix + '\'' + attribs + '>' );
 document.write( addy62568 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9116 = 'P&#101;t&#101;r.V&#97;nL&#111;&#111;' + '&#64;';
 addy9116 = addy9116 + 'm&#101;d' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9116 + suffix + '\'' + attribs + '>' );
 document.write( addy9116 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86661 = 'P&#101;t&#101;r.V&#97;nL&#111;&#111;' + '&#64;';
 addy86661 = addy86661 + 'm&#101;d' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86661 + suffix + '\'' + attribs + '>' );
 document.write( addy86661 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VIB and K.U.Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Department of Genetics, Institute for Cancer
Research, Oslo University Hospital Radiumhospitalet, Oslo, Norway; 2.
Department of Molecular and Developmental Genetics, VIB, Leuven, Belgium; 3.
Department of Human Genetics, University of Leuven, Leuven, Belgium; 4.
Department of Genetics, University of North Carolina, Chapel Hill, NC, USA; 5.
Department of Informatics, University of Oslo, Oslo, Norway; 6. Department of
Pathology, Oslo University Hospital Radiumhospitalet, Oslo, Norway; 7. Institute
for Clinical Medicine, University of Oslo, Oslo, Norway; 8. Department of
Biostatistics, University of North Carolina, Chapel Hill, NC, USA; 9.
Department of OncologyâPathology, Karolinska Institutet, Stockholm, Sweden; 10.
The Cancer Clinic, Oslo University Hospital Radiumhospitalet, Oslo, Norway; 11.
Institute for Clinical Epidemiology and Molecular Biology (EpiGen), University
of Oslo, Oslo, Norway; 12. These authors contributed equally to this work; 13.
These authors share senior authorship.</p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="121-genomic-medicine.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>