<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" dir="ltr" >
<head>
  <base href="" />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ECCB10, ECCB2010, Computational Biology, Bioinformatics, Conference, Ghent" />
  <meta name="description" content="ECCB10 is the key European computational biology event in 2010 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <title>Other Bioinformatics Applications</title>
  <link rel="stylesheet" href="../../../templates/eccb10/css/template.css.php%3Fcolor=blue&amp;styleswitcherFont=1&amp;styleswitcherWidth=0&amp;widthThinPx=780&amp;widthWidePx=940&amp;widthFluidPx=0.9.css" type="text/css" />
  <link rel="stylesheet" href="../../../templates/eccb10/lib/js/lightbox/css/shadowbox.css" type="text/css" />
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/mootools.js.php"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../plugins/content/phocagalleryslideshow.js"></script>
  <script type="text/javascript">
var YtSettings = { tplurl: '/Rebholz-srv/eccb2010/templates/eccb10', color: 'blue', layout: 'left', fontDefault: 'font-medium', widthDefault: 'width-wide', widthThinPx: 780, widthWidePx: 940, widthFluidPx: 0.9 };
  </script>
  <script type="text/javascript" src="../../../templates/eccb10/lib/js/template.js.php"></script>

<link rel="apple-touch-icon" href="../../../templates/eccb10/apple_touch_icon.png" />
</head>

<body id="page" class="font-medium width-wide left  blue">
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-15308130-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

	
	<div id="page-body">
		<div class="wrapper floatholder">
			<div class="wrapper-tl">
				<div class="wrapper-tr">
					<div class="wrapper-bl">
						<div class="wrapper-br">

							<div id="header">
								<div class="header-l">
									<div class="header-r">
								
										<div id="toolbar">
											<div class="floatbox ie_fix_floats">
											
																								<div id="date">
													Friday, 01 April 2011												</div>
																							
												
																								<div id="styleswitcher">
																																							<a id="switchfontlarge" href="javascript:void(0)" title="Increase font size"></a>
													<a id="switchfontmedium" href="javascript:void(0)" title="Default font size"></a>
													<a id="switchfontsmall" href="javascript:void(0)" title="Decrease font size"></a>
																									</div>
																								
											</div>
										</div>
										
										<div id="headerbar">
											<div class="floatbox ie_fix_floats">

												
											</div>
										</div>
										
																				
																				
																				
																		
									</div>
								</div>
							</div>
							<!-- header end -->
					
														
							<div id="middle">
								<div class="background">
				
																		<div id="left">
										<div id="left_container" class="clearfix">

													<div class="module_menu">
			<div>
				<div>
					<div>
													<h3>Main Menu</h3>
											<ul class="menu"><li class="level1 item1 first"><a href="../../../index.html" class="level1 item1 first"><span>Home</span></a></li><li class="level1 item2 parent"><a href="../../eccb10info.html" class="level1 item2 parent"><span>ECCB10 Info</span></a></li><li class="level1 item5"><a href="../../schedule.html" class="level1 item5"><span>Schedule</span></a></li><li class="level1 item7"><a href="../../keynotes.html" class="level1 item7"><span>Keynotes</span></a></li><li class="level1 item9 active current"><a href="../../proceedings.html" class="level1 item9 active current"><span>Proceedings</span></a></li><li class="level1 item10"><a href="../../best-posters.html" class="level1 item10"><span>Best Posters</span></a></li><li class="level1 item11"><a href="http://escs.iscbsc.org" target="_blank" class="level1 item11"><span>ISCB Student Symposium</span></a></li><li class="level1 item13 parent"><a href="../../workshops.html" class="level1 item13 parent"><span>Workshops</span></a></li><li class="level1 item16 parent"><a href="../../tutorials.html" class="level1 item16 parent"><span>Tutorials</span></a></li><li class="level1 item18"><a href="../../registration.html" class="level1 item18"><span>Registration</span></a></li><li class="level1 item19 parent"><a href="../../travel.html" class="level1 item19 parent"><span>Travel</span></a></li><li class="level1 item20 parent"><a href="../../guidelines.html" class="level1 item20 parent"><span>Guidelines</span></a></li><li class="level1 item22"><a href="../../fellowships.html" class="level1 item22"><span>Fellowships</span></a></li><li class="level1 item23 parent"><a href="../../sponsoring-eccb10.html" class="level1 item23 parent"><span>Sponsoring ECCB10</span></a></li><li class="level1 item24"><a href="../../committees.html" class="level1 item24"><span>Committees</span></a></li><li class="level1 item25"><a href="../../contact-us.html" class="level1 item25"><span>Contact Us</span></a></li><li class="level1 item26"><a href="../../credits.html" class="level1 item26"><span>Credits</span></a></li><li class="level1 item29 last"><a href="../../photos.html" class="level1 item29 last"><span>Photos</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Follow us on</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a mce_href="http://www.facebook.com/group.php?gid=252053815757" href="http://www.facebook.com/group.php?gid=252053815757" target="_blank"><img alt="Facebook" src="../../../images/photos/facebook.png" mce_src="/Rebholz-srv/eccb2010/images/photos/facebook.png" width="50" height="50"></a>
<a mce_href="http://www.linkedin.com/groupRegistration?gid=2662253" href="http://www.linkedin.com/groupRegistration?gid=2662253"><img src="../../../images/photos/linkedin.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/linkedin.jpg" alt="linkedin" width="52" height="52"> </a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Platinum Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.ibm.com" mce_href="http://www.ibm.com" target="_blank"><img alt="IBM" src="../../../images/photos/sponsors/IBM_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/IBM_logo.jpg"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Gold Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.nbic.nl" mce_href="http://www.nbic.nl" target="_blank"><img alt="NBIC" src="../../../images/photos/sponsors/nbic_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/nbic_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Silver Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.enfin.org" mce_href="http://www.enfin.org" target="_blank"><img alt="ENFIN" src="../../../images/photos/sponsors/ENFIN_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ENFIN_logo.jpg"></a>
<a href="https://www.ebi.ac.uk/" mce_href="https://www.ebi.ac.uk/" target="_blank"><img alt="EMBL-EBI" src="../../../images/photos/sponsors/EMBL-EBI_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/spon&lt;mce:script type=" text="" javascript"=""></a>
<a href="http://www.illumina.com/" mce_href="http://www.illumina.com/" target="_blank"><img alt="Illumina" src="../../../images/photos/sponsors/Illumina_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Illumina_logo.png"></a>

<a href="http://www.elixir-europe.org/" mce_href="http://www.elixir-europe.org/" target="_blank"><img alt="Elixir" src="../../../images/photos/sponsors/elixir_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/elixir_logo.jpg"></a>
<br>
<a href="http://www.conveycomputer.com/" mce_href="http://www.conveycomputer.com/" target="_blank"><img alt="Convey Computer" src="../../../images/photos/sponsors/Convey_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Convey_logo.png"></a>
<br>
<a href="http://www.clcbio.com/" mce_href="http://www.clcbio.com/" target="_blank"><img alt="CLC Bio" src="../../../images/photos/sponsors/CLCbio_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CLCbio_logo.png"></a>
<a href="http://www.mygrid.org.uk/" mce_href="http://www.mygrid.org.uk/" target="_blank"><img alt="myGrid" src="../../../images/photos/sponsors/mygrid_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/mygrid_logo.png" height="41" width="125"></a><br><a href="http://www.omixon.com/omixon/" mce_href="http://www.omixon.com/omixon/"><img src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg" alt="Omixon" mce_src="http://homes.esat.kuleuven.be/~asifrim/ECCB10/omixon-JPG.jpg"></a></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Fellowships</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.iscb.org/" mce_href="http://www.iscb.org/" target="_blank"><img alt="ISCB" src="../../../images/photos/sponsors/ISCB_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/ISCB_logo.jpg"></a>
</p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Other Sponsors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www1.frs-fnrs.be/" mce_href="http://www1.frs-fnrs.be/" target="_blank"><img alt="FNRS" src="../../../images/photos/sponsors/FNRS-logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FNRS-logo.png"></a>

<a href="http://www.fwo.be/" mce_href="http://www.fwo.be/" target="_blank"><img alt="FWO" src="../../../images/photos/sponsors/FWO_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/FWO_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.kuleuven.be/" mce_href="http://www.kuleuven.be/" target="_blank"><img alt="kuleuven" src="../../../images/photos/sponsors/KULeuven_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/KULeuven_logo.jpg"></a>
</p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.belspo.be/" mce_href="http://www.belspo.be/" target="_blank"><img alt="Belspo" src="../../../images/photos/sponsors/belspo_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/belspo_logo.png"></a><br mce_bogus="1">

<a href="http://www.microsoft.com/" mce_href="http://www.microsoft.com" target="_blank"><img alt="Microsoft" src="../../../images/photos/sponsors/microsoft_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/microsoft_logo.png"></a><br mce_bogus="1">

<a href="http://www.tibco.com/" mce_href="http://www.tibco.com" target="_blank"><img alt="TIBCO" src="../../../images/photos/sponsors/tibco_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/tibco_logo.png"></a><br mce_bogus="1"></p>

<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.keygene.com" mce_href="http://www.keygene.com" target="_blank"><img alt="Keygene" src="../../../images/photos/sponsors/keygene_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/keygene_logo.png"></a><br mce_bogus="1">

<a href="http://www.baseclear.com/" mce_href="http://www.baseclear.com/" target="_blank"><img alt="BaseClear" src="../../../images/photos/sponsors/baseclear_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/baseclear_logo.png"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
													<h3>Exhibitors</h3>
											<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.crcpress.com/" mce_href="http://www.crcpress.com/" target="_blank"><img alt="CRC Press" src="../../../images/photos/sponsors/CRC_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CRC_logo.jpg"></a>
<br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.oup.co.uk" mce_href="http://www.oup.co.uk" target="_blank"><img alt="Oxford University Press" src="../../../images/photos/sponsors/OUP_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/OUP_logo.jpg"></a><br></p>
<p style="text-align: center;" mce_style="text-align: center;">
<a href="http://www.springer.com" mce_href="http://www.springer.com" target="_blank"><img src="../../../images/photos/sponsors/Springer_logo.jpg" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Springer_logo.jpg" alt="Springer"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.cambridge.org" mce_href="http://www.cambridge.org" target="_blank"><img src="../../../images/photos/sponsors/CUP_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/CUP_logo.png" alt="Cambridge University Press"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.applied-maths.com/" mce_href="/Rebholz-srv/eccb2010/http//www.applied-maths.com/" target="_blank"><img src="../../../images/photos/sponsors/applied_maths_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/applied_maths_logo.png" alt="Applied Maths NV"></a><br mce_bogus="1"><br>
<a href="http://www.korilog.com/" mce_href="http://www.korilog.com/" target="_blank"><img src="../../../images/photos/sponsors/Korilog_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/Korilog_logo.png" alt="Korilog"></a><br mce_bogus="1">
<a href="http://www.sciengines.com/joomla/index.php" mce_href="http://www.sciengines.com/joomla/index.php" target="_blank"><img src="../../../images/photos/sponsors/sciengines_logo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/sponsors/sciengines_logo.png" alt="Korilog"></a><br mce_bogus="1"></p>					</div>
				</div>
			</div>
		</div>
			<div class="module">
			<div>
				<div>
					<div>
											<p>

<map name="embo">
<area shape="rect" mce_shape="rect" coords="15,3,67,99" mce_coords="15,3,67,99" href="http://2011.the-embo-meeting.org/" mce_href="http://2011.the-embo-meeting.org/" alt="EMBO 2011" target="_blank">
<area shape="rect" mce_shape="rect" coords="75,29,170,109" mce_coords="75,29,170,109" href="http://www.the-embo-meeting.org/" mce_href="http://www.the-embo-meeting.org/" target="_blank" alt="EMBO 2010">
</map>
</p>

<p style="text-align: center;" mce_style="text-align: center;"><a href="http://www.biomedcentral.com" mce_href="http://www.biomedcentral.com" target="_blank"><img src="../../../images/photos/logo_bmc.png" mce_src="/Rebholz-srv/eccb2010/images/photos/logo_bmc.png" alt="BioMed Central"></a><br mce_bogus="1"></p>
<p style="text-align: center;" mce_style="text-align: center;"><img src="../../../images/photos/embo.png" mce_src="/Rebholz-srv/eccb2010/images/photos/embo.png" alt="EMBO meeting 2010 (4-7 September, Barcelona) &amp; EBMO Meeting 2011 (Vienna)" usemap="#embo"></p>					</div>
				</div>
			</div>
		</div>
	

										</div>
									</div>
									<!-- left end -->
														
									<div id="main">
										<div id="main_container" class="clearfix">
			
											
											<div id="mainmiddle" class="floatbox">

																				
												<div id="content">
													<div id="content_container" class="clearfix">

																		
																						
														<div class="floatbox">
															
															<table class="contentpaneopen">
<tr>
		<td class="contentheading" width="100%">
					Other Bioinformatics Applications			</td>
				
		
					</tr>
</table>

<table class="contentpaneopen">



<tr>
<td valign="top">
<h1><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant:normal !important;
text-transform:uppercase">Other Bioinformatics Applications</span><span style="font-variant: normal ! important; text-transform: uppercase;" mce_style="font-variant: normal ! important; text-transform: uppercase;"><o:p></o:p></span></h1>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="">Chairs:
Martin Vingron and Hans-Peter Lenhof<o:p></o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoToc1" style="" mce_style="tab-stops:right dotted 450.85pt"><!--[if supportFields]><span
style='font-family:"Times New Roman","serif"'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span><span
style='mso-spacerun:yes'> </span>TOC \o &quot;1-2&quot; \h \z \u <span
style='mso-element:field-separator'></span></span><![endif]--><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><span class="MsoHyperlink"><span style="">Other Bioinformatics
Applications<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746918 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">1</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900310038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></span></span><span style=""><o:p></o:p></span></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746919" mce_href="#_Toc269746919">I-1. Universal virtual screening<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746919 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">3</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900310039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746920" mce_href="#_Toc269746920">I-2. Accuracy and computational
efficiency of a graphical modeling approach to linkage disequilibrium
estimation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746920 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">4</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746921" mce_href="#_Toc269746921">I-3. Stepwise classifier for
heterogeneous genomic data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746921 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">5</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746922" mce_href="#_Toc269746922">I-4. Fitness differences as an
explanation for difference between chronic myeloid leukemia therapies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746922 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">6</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746923" mce_href="#_Toc269746923">I-5. ALADYN: a web server for
aligning proteins by matching their large-scale motion<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746923 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">7</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746924" mce_href="#_Toc269746924">I-6. Stochastic extinction of
leukemic stem cells provides a road to cure chronic myeloid leukemia<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746924 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">8</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746925" mce_href="#_Toc269746925">I-7. sscMap Perturbation: unbiased
candidate therapeutic ranking in connectivity mapping<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746925 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">9</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746926" mce_href="#_Toc269746926">I-8. In silico study of expression
profile correlation between microRNAs and cancerous genes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746926 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">10</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746927" mce_href="#_Toc269746927">I-9. Harry Plotter: a user friendly
program to visualize genome and genetic map features<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746927 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">11</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746928" mce_href="#_Toc269746928">I-10. Inhibitory activity of
thiadiazoles on protein kinase PKnB from Mycobacterium tuberculosis: a virtual
screening and molecular docking study<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746928 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">12</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746929" mce_href="#_Toc269746929">I-11. A microarray format
standardization study: meaningful structures<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746929 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">13</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900320039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746930" mce_href="#_Toc269746930">I-12. Normalization of proteomic
ratio data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746930 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">14</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746931" mce_href="#_Toc269746931">I-13. BioCatalogue: the curated
catalogue of life science web services<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:
none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746931 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">15</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746932" mce_href="#_Toc269746932">I-14. Regression system for
prediction of errors in the data on gene expression in situ obtained from
confocal images<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746932 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">16</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746933" mce_href="#_Toc269746933">I-15. Impact of genetic variations
on phosphorylation sites<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:
1 dotted">. </span></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-begin'></span></span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"> PAGEREF
_Toc269746933 \h </span><span style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;display:none;mso-hide:
screen;text-decoration:none;text-underline:none"><span style="mso-element:field-separator" mce_style="mso-element:field-separator"></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">17</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746934" mce_href="#_Toc269746934">I-16. FuSiGroups: grouping gene
products by functional similarity<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746934 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">18</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746935" mce_href="#_Toc269746935">I-17. A file system strategy for
high-performance sequence interval queries of very large datasets<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746935 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">19</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746936" mce_href="#_Toc269746936">I-18. Evaluation of multivariate
data analysis strategies for high-content screening<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746936 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">20</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746937" mce_href="#_Toc269746937">I-19. Studies of the binding
capacity of cyclooxygenase II and biodistribution of phenols contained in
natural products<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746937 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">21</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746938" mce_href="#_Toc269746938">I-20. Finding logic networks using
the Dutch Life Science Grid: handling 15 million jobs<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746938 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">22</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746939" mce_href="#_Toc269746939">I-21. A detailed view on several
Model-Based Multifactor Dimensionality Reduction methods for detecting
gene-gene interactions in case-control data in the absence and presence of
noise<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746939 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">23</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900330039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746940" mce_href="#_Toc269746940">I-22. Quality ranking of 16S
sequences: an approach based on poset theory<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746940 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">24</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746941" mce_href="#_Toc269746941">I-23. Clustering and kernel methods
using R<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746941 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">25</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746942" mce_href="#_Toc269746942">I-24. Ensuring data integrity in
large modern integrative genomic studies<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746942 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">26</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746943" mce_href="#_Toc269746943">I-25. Testing branches in
dendrograms representing species abundance data.<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746943 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">27</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746944" mce_href="#_Toc269746944">I-26. Reliable identification of
hundreds of proteins without peptide fragmentation<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746944 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">28</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746945" mce_href="#_Toc269746945">I-27. Exploring unassigned peaks in
protein fragment mass spectra with frequent itemset mining techniques<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746945 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">29</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746946" mce_href="#_Toc269746946">I-28. Benchmarking a new semantic
similarity measure using reference sets and clustering : evaluation and
interpretation for the discovery of missing information<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746946 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">30</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746947" mce_href="#_Toc269746947">I-29. Utopia:GPCRDB: a
domain-specific PDF reader<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style=""> </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746947 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">31</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746948" mce_href="#_Toc269746948">I-30. Locally optimal structures of
an RNA<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;text-decoration:
none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">.. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746948 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">32</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746949" mce_href="#_Toc269746949">I-31. Anesthetics and the role of
the biological membrane in the molecular regulation of K2P potassium channels<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746949 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">33</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900340039000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746950" mce_href="#_Toc269746950">I-32. Comparison of validation
methods for merging cancer microarray data sets<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746950 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">34</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350030000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746951" mce_href="#_Toc269746951">I-33. Mapping insertions to putative
target genes<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746951 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">35</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350031000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746952" mce_href="#_Toc269746952">I-34. The zebrafish embryo in
toxicology: a combination of toxicogenomics and other screening techniques<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1">&nbsp; </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746952 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">36</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350032000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746953" mce_href="#_Toc269746953">I-35. Disease gene prediction based
on tissue-specific conserved coexpression<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746953 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">37</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350033000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746954" mce_href="#_Toc269746954">I-36. Improved classification by
integrating multiple patient data sets with literature information using
co-inertia analysis<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746954 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">38</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350034000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746955" mce_href="#_Toc269746955">I-37. Time-resolved signatures from
weighted visibility graph representation of time-series data<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746955 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">39</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350035000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746956" mce_href="#_Toc269746956">I-38. Genome sequence of the edible
cyanobacterium Arthrospira sp. PCC 8005<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746956 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">40</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350036000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746957" mce_href="#_Toc269746957">I-39. Moa: managing command line
bioinformatics<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;mso-hide:screen;
text-decoration:none;text-underline:none"><span style="" mce_style="mso-tab-count:1 dotted">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746957 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">42</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350037000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoToc2" style="" mce_style="tab-stops:right dotted 450.85pt"><span class="MsoHyperlink"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-no-proof:yes=""><a href="122-other-bioinformatics-applications.html#_Toc269746958" mce_href="#_Toc269746958">I-40. SAAPdb: structural effects of
single amino acid polymorphisms<span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><span style="">. </span></span><!--[if supportFields]><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span></span><span
style='color:windowtext;display:none;mso-hide:screen;text-decoration:none;
text-underline:none'> PAGEREF _Toc269746958 \h </span><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-separator'></span></span><![endif]--><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color: windowtext; display: none; text-decoration: none;">43</span><span style="color: windowtext; display: none; text-decoration: none;" mce_style="color:windowtext;display:none;
mso-hide:screen;text-decoration:none;text-underline:none"><!--[if gte mso 9]><xml>
 <w:data>08D0C9EA79F9BACE118C8200AA004BA90B02000000080000000E0000005F0054006F0063003200360039003700340036003900350038000000</w:data>
</xml><![endif]--></span><!--[if supportFields]><span style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none" mce_style="color:windowtext;
display:none;mso-hide:screen;text-decoration:none;text-underline:none"><span
style='mso-element:field-end'></span></span><![endif]--></a></span></span><span style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Times New Roman&quot;,&quot;serif&quot;;"><o:p></o:p></span></p>

<p class="MsoNormal"><!--[if supportFields]><span style="font-family:"Times New Roman","serif"" mce_style="font-family:"Times New Roman","serif""><span
style='mso-element:field-end'></span></span><![endif]--><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p>

<p class="MsoNormal"><span style="" mce_style="font-family:" times="" new="" roman="" ,="" serif=""><o:p>&nbsp;</o:p></span></p><p>



<span style="font-size: 11pt; line-height: 115%;" mce_style="font-size:11.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa=""><br style="page-break-before: always;" mce_style="page-break-before:always;
mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection2">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746919">I-1. Universal virtual screening</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Onodera K<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Onodera K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), Kamijo S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kamijo S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Structure-based virtual screening is one of the most
promising technologies in drug discovery. Many molecular docking programs and
scoring functions have been developed, but more improvements are required for
practical usages. One way to improve the accuracy of the screening is the
consensus scoring method. It combines docking scores from various scoring
functions without considering characteristics of the docking scores.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this study, we use the concepts of the consensus
scoring to improve the docking scores from docking programs (DOCK, FRED, and
GOLD) in virtual screening. While those docking scores are simple sum of each
score component, we introduced weight factors of the score components for
scoring calculations, and adjusted for better predictions of true actives
during virtual screening. After several optimization processes, we found the
best optimized scores using a wide variety of 113 target proteins with over
2000 diverse decoys.</p>

<h3>Results</h3>

<p class="abresults">The success rates finding true actives were improved by up
to 52.4% (e.g. from 36.8% to 56.1% in GOLD) for the test set. Also, the
combination of the optimized scores from GOLD and FRED improved success rate by
77.2%. In total, the combination of optimized scores now predict about 70% of
true actives for target proteins in the test set with 20 times enrichment.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Universal optimization of docking scores certainly
reduces chances to enrich non-specific compounds in the hit list erroneously.
At the same time, it increases chances to acquire novel drugs for drug
discovery. Our optimizing methods enhance performance of virtual screening
using molecular docking programs currently available in our community.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Kenji Onodera (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy59329 = '&#111;n&#111;d&#101;r&#97;k' + '&#64;';
 addy59329 = addy59329 + '&#105;&#105;s' + '&#46;' + '&#117;-t&#111;ky&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy59329 + suffix + '\'' + attribs + '>' );
 document.write( addy59329 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy42615 = '&#111;n&#111;d&#101;r&#97;k' + '&#64;';
 addy42615 = addy42615 + '&#105;&#105;s' + '&#46;' + '&#117;-t&#111;ky&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy42615 + suffix + '\'' + attribs + '>' );
 document.write( addy42615 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36625 = '&#111;n&#111;d&#101;r&#97;k' + '&#64;';
 addy36625 = addy36625 + '&#105;&#105;s' + '&#46;' + '&#117;-t&#111;ky&#111;' + '&#46;' + '&#97;c' + '&#46;' + 'jp';
 document.write( '<a ' + path + '\'' + prefix + addy36625 + suffix + '\'' + attribs + '>' );
 document.write( addy36625 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">IIS, University of Tokyo</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Institute of Industrial Science, University of Tokyo,
4-6-1 Komaba, Meguro, Tokyo 153-8505, Japan.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by grant-in-aids for Scientific
Research (19710173 and 22651079). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection3">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746920">I-2. Accuracy and computational efficiency of a
graphical modeling approach to linkage disequilibrium estimation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Abel H J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Abel H J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Thomas A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thomas A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*)</p>

<p class="abmotivation">The increasing availability of genome-wide dense
single-nucleotide polymorphism (SNP) data provides an abundance of information
for gene mapping studies. However, along with the benefits this new information
provides come the difficulties posed by allelic associations or linkage
disequilibrium (LD). Existing methods for modeling LD take time and storage of
O(nnm) and O(nm) where n is the number of individuals in the reference sample
and m is the number of SNPs being considered. These methods are not usable for
current data where n and m are around 1000 and 100,000.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We use a Markov chain Monte Carlo search method to
fit a graphical model. Parameter values were chosen by cross validation to
maximize the predictive accuracy when imputing masked genotypes. We used data
sets with 60 individuals genotyped at between 20 and 200,000 SNP loci to check
that the programs scale linearly with the number of loci. We used data
simulated on up to 1000 individuals to check for linear scaling with sample
size.</p>

<h3>Results</h3>

<p class="abresults">While our approach to model estimation and imputation is
marginally less accurate than existing methods, it is considerably more
computationally efficient as shown by several systematic benchmarking experiments.
For a reference set of n individuals genotyped at m markers the time and
storage required for fitting a graphical model are O(nm) and O(n+m)
respectively. To impute the phases and missing data on n individuals using an
already fitted graphical model requires O(nm) time and O(m) storage.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Two important features contribute to the computational
efficiency: separating the fitting and imputation processes into different
programs, and holding in memory only the data within a window of loci during fitting.
Note that while the time for fitting and imputation are both O(nm) imputation
is considerably faster, thus, once a model is estimated from a reference data
set, the marginal cost of phasing and imputing further samples is very low. Previous
methods combine the estimation and imputation stages in implementations
requiring time of O(nnm) and storage of O(nm).</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Alun Thomas (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy46320 = 'Al&#117;n.Th&#111;m&#97;s' + '&#64;';
 addy46320 = addy46320 + '&#117;t&#97;h' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy46320 + suffix + '\'' + attribs + '>' );
 document.write( addy46320 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49184 = 'Al&#117;n.Th&#111;m&#97;s' + '&#64;';
 addy49184 = addy49184 + '&#117;t&#97;h' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy49184 + suffix + '\'' + attribs + '>' );
 document.write( addy49184 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89834 = 'Al&#117;n.Th&#111;m&#97;s' + '&#64;';
 addy89834 = addy89834 + '&#117;t&#97;h' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy89834 + suffix + '\'' + attribs + '>' );
 document.write( addy89834 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Division of Genetic Epidemiology, University of Utah</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Division of Genetic Epidemiology, University of
Utah</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by grants NIH R01 GM81417, and
DOD W81XWH-07-01-0483 to Alun Thomas. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection4">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746921">I-3. Stepwise classifier for heterogeneous genomic
data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Wubulikasimu A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Wubulikasimu A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*), van de Wiel MA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;van de Wiel MA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">Combining heterogeneous data types and produce accuracy
which cannot be attained by either one of these data types alone is a hot topic
in classification problem. However, there is a shortage of effective and
efficient statistical and bioinformatic tools for truly integrative data
analysis. Existing integrative classifiers have two main disadvantages:
Firstly, coarsely combination in which the potential subtle contributions of
one data type likely to be overcome by more obvious contributions of other data
types. Secondly, huge computational costs because of sophisticated algorithms.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We aim to capture the distinct prediction power of
each data type in such a way that they are complementary rather than redundant
to each other. We first use the easy to get or economically cheaper data in
first stage and only turn to more expensive and noisy data type when local
error rate of samples below certain threshold. Local error rate and optimal
threshold value will be calculated in the training phase and apply for test
samples. By this way we can save considerable amount of the samples reclassify
by expensive data in second stage, at the same time keep accuracy as high as
possible.</p>

<h3>Results</h3>

<p class="abresults">Potential of our method is illustrated on publicly available
data sets. For the setting with aCGH and GE data we show that including GE to
aCGH data can enhance the prediction accuracy significantly, although a fair
proportion of the samples need not to be reclassified, and hence no GE measurements
would be needed for those. We come to a similar conclusion for a setting with
data sets containing both clinical variables and genomic ones, which implies a
serious potential economical and practical benefit.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The Stepwise classification method we introduced is
practically very attractive. It dos not require specific type of classification
algorithm, this give user more freedom in terms of algorithm. Experimental
results show that even if we choose wrong algorithm, because we don’t know beforehand
which algorithm is suitable for the problem at hand, which produce very worse
result in first or second stage, we will not get dramatic decrease in our
stepwise approach. This is because it will pass more samples to the data type
which produces better result by adjusting the threshold value.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Askar Wubulikasimu (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86234 = '&#97;sk&#97;r.w&#117;b&#117;l&#105;k&#97;s&#105;m&#117;' + '&#64;';
 addy86234 = addy86234 + 'v&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy86234 + suffix + '\'' + attribs + '>' );
 document.write( addy86234 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy56691 = '&#97;sk&#97;r.w&#117;b&#117;l&#105;k&#97;s&#105;m&#117;' + '&#64;';
 addy56691 = addy56691 + 'v&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy56691 + suffix + '\'' + attribs + '>' );
 document.write( addy56691 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6767 = '&#97;sk&#97;r.w&#117;b&#117;l&#105;k&#97;s&#105;m&#117;' + '&#64;';
 addy6767 = addy6767 + 'v&#117;mc' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy6767 + suffix + '\'' + attribs + '>' );
 document.write( addy6767 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Epidemiology and Biostatistics ,VU
University medical center</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Department of Epidemiology and Biostatistics VU
University medical center Amsterdam The Netherlands</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research was supported by the Center for
Translational Molecular Medicine (DeCoDe project) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection5">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746922">I-4. Fitness differences as an explanation for
difference between chronic myeloid leukemia therapies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lenaerts T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Lenaerts T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Castagnetti F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Castagnetti F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Traulsen A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Traulsen A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Pacheco JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pacheco JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5,6), Rosti G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rosti G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Dingli D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dingli D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7)</p>

<p class="abmotivation">Since the successful introduction of Imatinib as the
treatment of chronic myeloid leukemia (CML), several more potent kinase
inhibitors (TKI) like Nilotinib have been developed. Nilotinib binds with a
higher affinity to the BCR-ABL oncoprotein and it also is able to block many
Imatinib-resistant mutations. Despite these molecular effects, in vitro studies
suggest that there is no difference in the inhibition of signaling downstream
of BCR-ABL. How can one explain then the faster and deeper response observed in
the clinical data [1] ?</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have developed a computational model of the
hematopoietic system in which we can study the population dynamics of different
types of cell: healthy, cancer and TKI treated cells. Normal hematopoiesis is
represented by a hierarchical model in dynamic equilibrium where cells move
along the hematopoietic tree as they become increasingly differentiated. In
this work we use clinical data [1] to predict the overall amount of cells
affected by the treatment and the effect of the drug on the differentiation
rate of the cancer cells.</p>

<h3>Results</h3>

<p class="abresults">We show that the fraction of CML cells responding to
nilotinib is slightly higher than for imatinib (8.5% versus 5%), capturing the
higher affinity of Nilotinib for BCR-ABL. Yet this in itself does not explain
the deeper response for this therapeutic agent. By expressing the difference
between the two drugs in terms of their fitness in relation to the fitness of
normal hematopoietic cells [2] (1.0) we show the fitness of Nilotinib treated
cells (0.49) is lower than that for Imatinib treated cells (0.63). As such,
Nilotinib treated cells will disappear more quickly from the overall population</p>

<h3>Discussion</h3>

<p class="abdiscussion">Since Nilotinib treated cells will disappear more quickly
from the overall population of cells than Imatinib treated cells, the response
of the patient will reach lower disease levels. This explanation can only be
observed by studying the interplay between the different cell populations in
the hematopoietic system, providing an illustration of the power of our model
to study the dynamics of tumor growth and response to therapy. [1] G. Rosti et
al (2009) Blood 114(24):4933-4938 [2] D. Dingli, A. Traulsen, T. Lenaerts and
J.M. Pacheco (2010) Genes &amp; Cancer 1(4):309-315</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ulb.ac.be/di/map/tlenaert/" mce_href="http://www.ulb.ac.be/di/map/tlenaert/">http://www.ulb.ac.be/di/map/tlenaert/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tom Lenaerts (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy91522 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy91522 = addy91522 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy91522 + suffix + '\'' + attribs + '>' );
 document.write( addy91522 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57500 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy57500 = addy57500 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy57500 + suffix + '\'' + attribs + '>' );
 document.write( addy57500 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66705 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy66705 = addy66705 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy66705 + suffix + '\'' + attribs + '>' );
 document.write( addy66705 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">MLG, DI, Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 MLG, Département d’Informatique, Université Libre de
Bruxelles, Boulevard du Triomphe CP212, 1050 Brussels, Belgium. 2 Computer
Science Department, Vrije Universiteit Brussel, Pleinlaan 2, 1050 Brussels,
Belgium. 3 Department of Hematology and Oncology, "L. and A.
Seràgnoli", St Orsola University Hospital, Via Massarenti, 9 - 40138
Bologna 4 Evolutionary Dynamics Group, Max-Planck-Institute for Evolutionary
Biology, August-Thienemann-Str. 2, 24306 Plön, Germany. 5 Departamento de
Matematica e Aplicações, Universidade do Minho, 4710-057 Braga, Portugal. 6
ATP-Group, Centro de Matemática e Aplicações Fundamentais, Complexo Interdisciplinar,
1649-003 Lisboa codex, Portugal. 7 Division of Hematology, Mayo Clinic College
of Medicine, 200 First Street SW, Rochester, MN 55905, USA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">TL acknowledges the support of the FNRS (Belgium), JMP
acknowledges the support of FCT (Portugal), AT is supported by the Emmy-Noether
program of the German Research Foundation and DD is supported by an Early
Career Development Award. We all acknowledge the GIMEMA CML working party for
providing the data. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection6">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746923">I-5. ALADYN: a web server for aligning proteins by
matching their large-scale motion</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Potestio R<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Potestio R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Aleksiev T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aleksiev T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Pontiggia F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pontiggia F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Cozzini S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cozzini S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,1), Micheletti C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Micheletti C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">The characterization of proteins and enzymes has largely
benefited from the large-scale application of sequence- and structure-based
alignment methods. These tools proved extremely valuable to identify salient
features shared by proteins differing at the level of primary sequence or
three-dimensional organisation. Our group has recently shown that the
comparison of proteins' internal dynamics can be used to pin-point
functionally-oriented relationships that would otherwise be elusive to
sequence- or structure-based alignments.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">To detect these common dynamical features we align
proteins by matching their large-scale internal motion. The latter is reliably
and efficiently computed using elastic network models. The dynamics-based
alignment method is offered as a web-server, named ALADYN and is available at:
http://aladyn.escience-lab.org . The server takes as input a pair of protein
structures and presents an interactive visualization of the amino acids that
are in good structural and dynamical correspondence.</p>

<h3>Results</h3>

<p class="abresults">We discuss the application of the dynamics-based alignment
to various members of enzymatic superfamilies, such as hydrolases. In
particular we illustrate the case of HIV1-PR/Beta-secretase and exonuclease
III/human adenovirus proteinase.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The dynamics-based alignment sever lends naturally to be
used as a quantitative tool for investigating the role of dynamics in protein
function, and establish nontrivial relations among proteins performing similar
functions in spite of different structural organisation.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://aladyn.escience-lab.org" mce_href="http://aladyn.escience-lab.org">http://aladyn.escience-lab.org</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Raffaello Potestio (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32438 = 'p&#111;t&#101;st&#105;&#111;' + '&#64;';
 addy32438 = addy32438 + 's&#105;ss&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy32438 + suffix + '\'' + attribs + '>' );
 document.write( addy32438 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51404 = 'p&#111;t&#101;st&#105;&#111;' + '&#64;';
 addy51404 = addy51404 + 's&#105;ss&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy51404 + suffix + '\'' + attribs + '>' );
 document.write( addy51404 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16198 = 'p&#111;t&#101;st&#105;&#111;' + '&#64;';
 addy16198 = addy16198 + 's&#105;ss&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy16198 + suffix + '\'' + attribs + '>' );
 document.write( addy16198 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">SISSA</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1) Scuola Internazionale Superiore di Studi Avanzati
and eLab, via Bonomea 265, 34136 Trieste, Italy 2) Department of Biochemistry,
Howard Hughes Medical Institute, Brandeis University, Waltham, Massachusetts -
02454, USA 3) Democritos CNR-IOM, via Bonomea 265, 34136 Trieste, Italy</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">CNR-IOM Democritos and the Italian Ministry of
Education. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection7">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746924">I-6. Stochastic extinction of leukemic stem cells
provides a road to cure chronic myeloid leukemia</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lenaerts T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Lenaerts T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Traulsen A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Traulsen A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Pacheco JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pacheco JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,5), Dingli D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dingli D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6)</p>

<p class="abmotivation">Tyrosine kinase inhibitors (TKI), of which Imatinib is a
first, play a pivotal role in the treatment of chronic myeloid leukemia (CML). Notwithstanding,
their therapeutic success, none of the TKI are considered to be curative. This
view follows from the experimental observation that they do not affect leukemic
stem cells (LSC), which are at the origin of the disease. As a consequence it
is assumed that only by removing the LSC, one can cure a patient (e.g.
bone-marrow transplant).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have developed a computational model of the
hematopoietic system in which we can study the population dynamics of different
types of cell: healthy, cancer and TKI treated cells. Normal hematopoiesis is
represented by a hierarchical model in dynamic equilibrium where cells move
along the hematopoietic tree as they become increasingly differentiated. The
parameters of the model are derived from molecular and clinical literature. This
model is now split into a stochastic and deterministic part since the
populations of stem cells and early progenitors are limited in size.</p>

<h3>Results</h3>

<p class="abresults">Through this model we showed in [1] that this view has to be
revisited when one considers that stochastic effects at the level of the stem
and early progenitor cells will cause the LSC to disappear from the pool of
hematopoietic stem cells (HSC). Our simulations involving a large cohort of virtual
patients show that in the majority of these patients (~84%) the LSC have
disappeared, making the progenitors and not the stem cells the drivers of the
disease. Since these progenitors are susceptible to TKI treatment, the
different inhibitors may actually cure the disease.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our predictions show that early diagnosis in combination
with long-term TKI therapy opens the road to CML eradication when the response
of the patients is sufficiently lowered. The results of are simulations are
supported by recent clinical studies on long-term Imatinib treatments of CML-CP
patients. Overall, our results are fundamental for the understanding of
acquired HSC disorders and other derived diseases. [1] T. Lenaerts, J.M.
Pacheco, A. Traulsen and D. Dingli (2010) Haematologica 95(6):900-907.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.ulb.ac.be/di/map/tlenaert/" mce_href="http://www.ulb.ac.be/di/map/tlenaert/">http://www.ulb.ac.be/di/map/tlenaert/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tom Lenaerts (<a href="/Rebholz-srv/eccb2010/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19100 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy19100 = addy19100 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy19100 + suffix + '\'' + attribs + '>' );
 document.write( addy19100 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="http://mce_host/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64723 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy64723 = addy64723 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy64723 + suffix + '\'' + attribs + '>' );
 document.write( addy64723 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57455 = 'tl&#101;n&#97;&#101;rt' + '&#64;';
 addy57455 = addy57455 + '&#117;lb' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy57455 + suffix + '\'' + attribs + '>' );
 document.write( addy57455 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">MLG, DI, Université Libre de Bruxelles</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 MLG, Département d’Informatique, Université Libre de
Bruxelles, Boulevard du Triomphe CP212, 1050 Brussels, Belgium. 2 Computer
Science Department, Vrije Universiteit Brussel, Pleinlaan 2, 1050 Brussels,
Belgium. 3 Evolutionary Dynamics Group, Max-Planck-Institute for Evolutionary
Biology, August-Thienemann-Str. 2, 24306 Plön, Germany. 4 Departamento de
Matematica e Aplicações, Universidade do Minho, 4710-057 Braga, Portugal. 5
ATP-Group, Centro de Matemática e Aplicações Fundamentais, Complexo Interdisciplinar,
1649-003 Lisboa codex, Portugal. 6 Division of Hematology, Mayo Clinic College
of Medicine, 200 First Street SW, Rochester, MN 55905, USA</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">TL acknowledges the support of the FNRS (Belgium), JMP
acknowledges the support of FCT (Portugal), AT is supported by the Emmy-Noether
program of the German Research Foundation and DD is supported by an Early
Career Development Award. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection8">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746925">I-7. sscMap Perturbation: unbiased candidate
therapeutic ranking in connectivity mapping</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">McArt D<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;McArt D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Zhang S-D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zhang S-D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Different biological states have their own characteristic
gene-expression profiles. Connectivity mapping is a technique for discovering
the underlying biological connections between states, based on gene-expression
similarities. The sscMap method has been shown to provide enhanced sensitivity
in mapping meaningful connections leading to testable biological hypotheses,
and in identifying drug candidates with particular pharmacological and/or
toxicological properties. Challenges remain, however, as how to prioritize the
large number of significant connections in an unbiased manner.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We examined a perturbation approach on datasets
from the GEO website. We attained microarray datasets for cervical cancer,
acute myeloid leukemia and breast cancer(letrozole treated). For each casestudy
the raw data were downloaded and expression analysed using the Bioconductor
suite and analyzed with gene-signature perturbation (removal of a single gene
systematically from the signature and measuring its impact). We aimed to test
whether an identified connection between a disease gene-signature and a
particular set of drug-induced reference profiles is stable.</p>

<h3>Results</h3>

<p class="abresults">The perturbation approach helps to identify meaningful
biological connections that have been ranked according to the ability to be
robust to systematic perturbation and thus retrieves the most influential candidate
drugs. In the case of AML, we found that the prevalent drugs were retinoic
acids and PPARγ activators. For cervical cancer, our results suggested that
potential candidates are likely to involve the EGFR pathway, and with the
breast cancer dataset, we identified candidates that are involved in
prostaglandin inhibition.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Thus the gene-signature perturbation approach offers a
convenient and effective approach to the connectivity mapping process. It
allows for increased specificity in the identification of possible therapeutic
candidates. The 'strength' of each candidate derived from checking for
robustness against perturbation offers a viable unbiased method of candidate
selection for further experimentation. Diseases that have poor prognosis and
treatments that require non-invasive criteria will benefit from timely and
unbiased therapeutic selections.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Darragh G. McArt (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44889 = 'd.mc&#97;rt' + '&#64;';
 addy44889 = addy44889 + 'q&#117;b' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy44889 + suffix + '\'' + attribs + '>' );
 document.write( addy44889 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51534 = 'd.mc&#97;rt' + '&#64;';
 addy51534 = addy51534 + 'q&#117;b' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy51534 + suffix + '\'' + attribs + '>' );
 document.write( addy51534 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88068 = 'd.mc&#97;rt' + '&#64;';
 addy88068 = addy88068 + 'q&#117;b' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy88068 + suffix + '\'' + attribs + '>' );
 document.write( addy88068 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Queens University Belfast</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Cancer Bioinformatics Group, Centre for Cancer
Research and Cell Biology (CCRCB), Queen's University Belfast, 97 Lisburn Road,
Belfast BT9 7BL, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the Department for
Employment and Learning through its "Strengthening the all-Island Research
Base" initiative</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection9">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746926">I-8. In silico study of expression profile
correlation between microRNAs and cancerous genes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Ng K-L<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Ng K-L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Weng C-W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weng C-W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Huang C-H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Huang C-H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">We investigate the possibility that microRNA can act as
an oncogene or tumor suppressor gene. Experimentally verified microRNA target
genes information (TarBase) are integrated with microRNA and mRNA expression
data (NCI-60) to study this hypothesis, in which the Pearson correlation and
Spearman rank coefficients are used to quantify these relations for nine cancer
tissues.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In order to investigate the regulatory role of
mirna in cancer, we study the expression profiles correlation between mirna and
cancer-related genes, in particular the OCG and TSG targets. Mirna target pairs
with PCC/SRC or both below a given threshold are filtered for further
investigation. These pairs suggest a regulatory relationship between the mirna
and its targets. The TAG dataset is used in order to sort out the mirna-OCG and
mirna-TSG pairs. These pairs are further annotated by using the OMIM, GO and
KEGG terms. Filtered results are further validated by OMIM, KEGG and
miR2Diseas.</p>

<h3>Results</h3>

<p class="abresults">Correlation coefficients with negative values are used to
filter out miRNA targets. Biological annotations of the targets are supplied by
using the TAG, GO and KEGG records. Highly correlated cancer-related miRNAs are
validated by the OMIM, miR2Disease and KEGG databases. Our analysis indicated
the usefulness of the correlation approach in testing the hypothesis. The above
information is utilized to provide a platform in identifying potential cancer
related miRNAs. A web based interface is set up for information query,
http://ppi.bioinfo.asia.edu.tw/mirna_target/index.html</p>

<h3>Discussion</h3>

<p class="abdiscussion">Recent studies indicate that mirna could possibly play an
important role in human cancer, where mirna targets TSG or OCG. Experimentally
verified mirna targeted genes information are ob-tained from TarBase, which are
integrated with the mirna and mRNA expression data from NCI-60 to study this
hypothesis. Two correlation coefficients, PCC and SRC, are used to quantify the
correlation between mirna and its targets expression profiles. The predicted
results are evaluated with reference to the OMIM, KEGG and miR2Disease data
sets.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://ppi.bioinfo.asia.edu.tw/mirna_target/index.html" mce_href="http://ppi.bioinfo.asia.edu.tw/mirna_target/index.html">http://ppi.bioinfo.asia.edu.tw/mirna_target/index.html</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ka-Lok Ng (<a href="/Rebholz-srv/eccb2010/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24531 = 'pp&#105;dd&#105;' + '&#64;';
 addy24531 = addy24531 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy24531 + suffix + '\'' + attribs + '>' );
 document.write( addy24531 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="http://mce_host/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33038 = 'pp&#105;dd&#105;' + '&#64;';
 addy33038 = addy33038 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy33038 + suffix + '\'' + attribs + '>' );
 document.write( addy33038 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93095 = 'pp&#105;dd&#105;' + '&#64;';
 addy93095 = addy93095 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy93095 + suffix + '\'' + attribs + '>' );
 document.write( addy93095 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Dept. of Bioinformatics, Asia University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Bioinformatics, Asia University, 500
Lioufeng Road, Wufeng Shiang, Taichung, Taiwan 41354</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Ka-Lok Ng and Chien-Hung Huang works are supported by
the National Science Council of Taiwan. under the grants of NSC 98-2221-E-468-013
and NSC. 98-2221-E-150-062 respectively. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection10">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746927">I-9. Harry Plotter: a user friendly program to
visualize genome and genetic map features</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Moretto M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Moretto M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Cestaro A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Cestaro A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Troggio M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Troggio M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Costa F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Costa F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Velasco R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Velasco R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Analyzing a genome requires a lot of different software
and tools, which also includes, especially for biologists, those that inspect,
visualize and print the data. Visualization of the correspondence among genome
sequence and a genetic map is very useful to show the coverage and the correctness
of the assembled genome. Moreover there are a lot of genomic feature
distribution that could benefit of a graphical representation. For instance,
gene density, repeated elements distribution, CpG abundance, and QTL
significance could be all better understood with a multi-colored gradient
representation</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Harry Plotter is a stand-alone program written in
Java which can run easily on all the platforms for which a Java Virtual Machine
is provided, like Microsoft Windows, Linux, MacOSX. It has a Graphical User
Interface (GUI) and its use only requires input text files in the correct
tab-delimited format. No other software or databases are needed.</p>

<h3>Results</h3>

<p class="abresults">Harry Plotter has already been used for the Vitis, Malus and
Fragaria sequencing genome projects to visually show genomic scaffolds anchored
to chromosomes. It has also been used in Malus to show different QTL
significance along chromosomes with a multi-colored gradient. Harry Plotter
allows the user to adjust the image size and quality and to export images in
PNG or JPEG file format.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Most of the software used to visualize genomic
information like genome sequence anchored scaffolds and DNA features are
usually not public scripts or require a complex setup in order to work properly;
while most of the time just a simple image that synthesize all the information
is needed. Harry Plotter is a small, easy to use program, that does not require
installation and has a graphical user interface. The input files are simple
tab-delimited text files with the minimum data required to create the images.
Following new users requests, we are currently adding new functionalities.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Marco M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Marco M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->oretto (<a href="/Rebholz-srv/eccb2010/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7027 = 'm&#97;rc&#111;.m&#111;r&#101;tt&#111;' + '&#64;';
 addy7027 = addy7027 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy7027 + suffix + '\'' + attribs + '>' );
 document.write( addy7027 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="http://mce_host/
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82483 = 'm&#97;rc&#111;.m&#111;r&#101;tt&#111;' + '&#64;';
 addy82483 = addy82483 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy82483 + suffix + '\'' + attribs + '>' );
 document.write( addy82483 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40447 = 'm&#97;rc&#111;.m&#111;r&#101;tt&#111;' + '&#64;';
 addy40447 = addy40447 + '&#105;&#97;sm&#97;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy40447 + suffix + '\'' + attribs + '>' );
 document.write( addy40447 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Fondazione Edmund Mach - Istituto Agrario di San
Michele all'Adige</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Fondazione Edmund Mach - Istituto Agrario di San
Michele all'Adige</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Provincia Autonoma di Trento</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection11">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746928">I-10. Inhibitory activity of thiadiazoles on
protein kinase PKnB from Mycobacterium tuberculosis: a virtual screening and
molecular docking study</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Raj U<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Raj U&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*),
Singh VB<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE
&quot;Singh VB&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1), Swati<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Swati&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Srivastava A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Srivastava A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Naqvi SAH<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Naqvi SAH&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Lots of deaths are caused by Tuberculosis per year hence
it has become a global threat. In order to develop efficient therapeutic
strategies it is vital to understand the physiology of the causative organism,
Mycobacterium tuberculosis. Protein Kinase B (PKnB) from Mycobacterium
tuberculosis is a crucial receptor-like protein kinase involved in signal
transduction. M. tuberculosis PKnB is a trans-membrane Ser/Thr protein kinase
(STPK) highly conserved in Gram-positive bacteria and apparently essential for
mycobacterial viability.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have attempted with the help of virtual
screening and docking approach to expound the extent of specificity of protein
kinase B towards different classes of Thiadiazoles(an anti-tubercular agent).
Total number of Thiadiazoles was 1800 in number with the minimum binding energy
of -11.29 kcal/mol. Three different databases were searched for molecules with
a Thiadiazole pharmacophore. The selected Thiadiazoles were chosen on the basis
of the structural specificity to the enzyme towards its substrate and
inhibitors. Later on ADME &amp; Tox studies were conducted to produce much promising
results.</p>

<h3>Results</h3>

<p class="abresults">The docking result of the study of 1800 Thiadiazoles
demonstrated that the binding energies were in the range of -11.29 kcal/mol to
-3.61 kcal/mol, with 15 molecules showing hydrogen bonds with the active site
residue. Later, ADME &amp; Toxixicty studies of these 15 molecules were
conducted and 6 molecules showed excellent results.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The protein kinase B peptide contains two types of
structural elements (Valine 95, Arginine 97) and basic residue ring constituted
of glycine rich residue. The structure of the protein-ligand complex reveals
that Thiadiazoles partially occupies the adenine-binding pocket in PKnB,
providing a framework for the design of compounds with potential therapeutic
applications. The study provides hints for the future design of new derivatives
with higher potency and specificity.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Utkarsh Raj (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy24062 = 'r&#97;j&#97;m&#105;ty1' + '&#64;';
 addy24062 = addy24062 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy24062 + suffix + '\'' + attribs + '>' );
 document.write( addy24062 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77201 = 'r&#97;j&#97;m&#105;ty1' + '&#64;';
 addy77201 = addy77201 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy77201 + suffix + '\'' + attribs + '>' );
 document.write( addy77201 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55730 = 'r&#97;j&#97;m&#105;ty1' + '&#64;';
 addy55730 = addy55730 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy55730 + suffix + '\'' + attribs + '>' );
 document.write( addy55730 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Amity Institute of Biotechnology, Amity Unversity</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)Amity University (Lucknow,IN) (2)Bio Discovery-
Solutions for future (Agra,IN) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection12">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746929">I-11. A microarray format standardization study:
meaningful structures</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kocabas F<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kocabas F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Can T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Can T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Baykal N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Baykal N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">There is an increasing demand for microarray experiment.
Putting into biological context for interpretation makes microarray data
exceptionally larger. Microarray data will be difficult to manage if standards
are not developed. Although there are standardization initiatives on content,
format, and model, repositories have isolated information space and have
proprietary formats. At present, they can not exchange and their integration is
costly. There is a need for format and data management standards to make
disparate microarray data visible, usable, and understandable for scientific community.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have studied on records on GEO. MINiML file
which is a descriptive file for each experiment has been extended in structure
(metadata card) and semantics within a metadata framework. Two semantic nets
are developed in RDF format (Experimenter: encoded in FOAF/RDF; Summary: in
RuleML Datalog). SPARQL has been used for information access. They are
meaningful structures which are extendable, queriable, portable, integrable,
and machine processable. The metadata card and semantic nets can be exchanged
with major repositories. This framework can also be used for all biomedical
repositories.</p>

<h3>Results</h3>

<p class="abresults">With the proposed framework the benefits are, • Producer can
deposit meaningful data and consumer can get the intended meaning • backlog is
reduced due to automation • Ambiguity and redundancy is reduced with standard
format and additional semantics • Visibility, understandability, and usability
is enforced • Users can use W3C tools on public domain • The framework promotes
standardization leading to optimum structured reporting through use of more
controlled vocabularies (Countries, Date Time Group, Names etc) not only to
annotate but to encode the metadata and data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Metadata card and semantic nets are format standard
contributions. They do not replace any existing work. However, if adopted, they
can be a focus for discovery and integration. There is up to %3 monthly
increase in records at GEO which will increase the current backlogs (currently
20% in Series records and 80% in GEO Dataset creation). Such standardization
studies to promote machine understandability and semantic interoperability are
needed. We believe that once such standardization efforts become adopted, the
required tools and detailed guidance will follow.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Fahri S Kocabas (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6407 = 'f&#97;hr&#105;' + '&#64;';
 addy6407 = addy6407 + '&#105;&#105;' + '&#46;' + 'm&#101;t&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy6407 + suffix + '\'' + attribs + '>' );
 document.write( addy6407 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy36529 = 'f&#97;hr&#105;' + '&#64;';
 addy36529 = addy36529 + '&#105;&#105;' + '&#46;' + 'm&#101;t&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy36529 + suffix + '\'' + attribs + '>' );
 document.write( addy36529 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98345 = 'f&#97;hr&#105;' + '&#64;';
 addy98345 = addy98345 + '&#105;&#105;' + '&#46;' + 'm&#101;t&#117;' + '&#46;' + '&#101;d&#117;' + '&#46;' + 'tr';
 document.write( '<a ' + path + '\'' + prefix + addy98345 + suffix + '\'' + attribs + '>' );
 document.write( addy98345 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">PhD student, 1Department of Health Informatics,
Middle East Technical University (METU) Informatics Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Middle East Technical University (METU) Informatics
Institute, Health Informatics Department, 06531 Ankara, Turkey 2 NATO HQ C3S,
Information Services Branch, Blvd Leopold III B1110 Brussels, Belgium 3 Middle
East Technical University (METU) Computer E ngineering Department, 06531 Ankara,
Turkey</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection13">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746930">I-12. Normalization of proteomic ratio data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Sherman J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Sherman J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Molloy M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Molloy M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,3), Descallar J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Descallar J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Lance B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lance B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Uitto P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Uitto P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Wood G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wood G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,*)</p>

<p class="abmotivation">Quantitative mass spectrometry techniques are commonly
used for comparative proteomic analysis in order to provide relative
quantitation between samples. To account for variable loading between samples a
normalization procedure is required. A standard approach to normalization is to
use internal standards, proteins that are assumed to display only minimal
changes in abundance between the samples under comparison. A normalization
procedure allows adjustment of the data, so enabling relative quantitation to
be reported, for example, as ratios of the two samples. This research examines
current normalization practice and offers improved extensions.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Normalization is determined by centring symmetrised
data. A generic symmetrising transformation is defined and a range of centring
properties considered; these together provide a set of possible normalizations,
with each normalization being reciprocally invariant, a needed property. This
presents the opportunity for alternative normalization procedures. When testing
for departure of a ratio from unity we use a t-test, so normality of the
symmetrised observations is desirable. The customary logarithm symmetrisation
takes lognormally distributed ratios to normality. A symmetrising function
which takes a ratio of normal distributions to normality is developed and its
properties examined.</p>

<h3>Results</h3>

<p class="abresults">This research makes two contributions to an understanding of
normalization. First, the customary centring of logarithmically transformed
ratios is shown to attend not only to centring but also to minimisation of the
spread of the symmetrised data. Second, the normalization problem is set in a
broad context, allowing normalization to be achieved based on a symmetrisation
which carries the ratios to approximate normality, so at times increasing the
power with which differentially expressed samples can be detected. The improved
power of the new method is illustrated using simulated data and then applied to
real proteomic data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Current practice is seen to be robust. The alternative
normalization method proposed requires the user to input (or estimate, from
inputted internal standards data) the mean and variance of the normal
distributions of the numerator and denominator of the internal standard ratios.
Use of such a tailored symmetrisation can increase the power with which we can
detect differentially regulated ratios. The method can be extended to handle
alternative assumptions concerning the distribution of the numerator and
denominator.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Graham R. Wood (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43031 = 'gr&#97;h&#97;m.w&#111;&#111;d' + '&#64;';
 addy43031 = addy43031 + 'mq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy43031 + suffix + '\'' + attribs + '>' );
 document.write( addy43031 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82849 = 'gr&#97;h&#97;m.w&#111;&#111;d' + '&#64;';
 addy82849 = addy82849 + 'mq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy82849 + suffix + '\'' + attribs + '>' );
 document.write( addy82849 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy47528 = 'gr&#97;h&#97;m.w&#111;&#111;d' + '&#64;';
 addy47528 = addy47528 + 'mq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#97;&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy47528 + suffix + '\'' + attribs + '>' );
 document.write( addy47528 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Statistics, Macquarie University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Department of Pharmaceutical Chemistry, University
of California, San Francisco, CA 94143, USA. (2) Australian Proteome Analysis
Facility, Macquarie University, NSW 2109, Australia. (3) Department of
Chemistry and Biomolecular Sciences, Macquarie University, NSW 2109, Australia.
(4) Department of Statistics, Macquarie University, NSW 2109, Australia.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">J. Sherman was supported by NIH NCRR P41RR001614 and J.
Descallar by a Macquarie University Safety Net Grant. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection14">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746931">I-13. BioCatalogue: the curated catalogue of life
science web services</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Tanoh F<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Tanoh F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bhagat J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bhagat J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Nzuobontane E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nzuobontane E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Laurent T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laurent T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Wolstencroft K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wolstencroft K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Stevens R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Stevens R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Pettifer S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pettifer S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Lopez R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lopez R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Goble C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Goble C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Web services have gained a momentum as a means for
packaging existing data and computational resources in a form that is amenable
for use and composition by third party applications. Many institutions and
organisations such as the EBI, NCBI and DDBJ provide Web Services to access and
analyse their resources. However, one of the main issues that hinders the wide
adoption and use of web services is the difficulty in locating the
“appropriate” web service. The descriptions of available web services are often
poor providing little information to the scientist about their usefulness.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Web Service and annotations in the BioCatalogue are
added by its members or automatically harvested from existing registry such as
the Embrace registry http://www.embraceregistry.net/), seekda
(http://webservices.seekda.com/) and BioMoby (http://www.biomoby.org/).</p>

<h3>Results</h3>

<p class="abresults">The BioCatalogue can be accessed via its Web site or via a
Web Service API for programmatic access. It currently holds over 1650 Life
Science Web Services and has more than 300 registered users. It has had more
than 15000 visits since its launch in 2009.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The BioCatalogue has established itself as a
one-stop-shop for the Life Science community to locate and use Web service that
implement the analysis relevant for their scientific experiment. The next phase
of the development concentrates on extending the content, improving the quality
and coverage of service annotation and integration with other tools.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.biocatalogue.org/" mce_href="http://www.biocatalogue.org/">http://www.biocatalogue.org/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Steve Pettifer (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32865 = 'st&#101;v&#101;.p&#101;tt&#105;f&#101;r' + '&#64;';
 addy32865 = addy32865 + 'm&#97;nch&#101;st&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy32865 + suffix + '\'' + attribs + '>' );
 document.write( addy32865 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69082 = 'st&#101;v&#101;.p&#101;tt&#105;f&#101;r' + '&#64;';
 addy69082 = addy69082 + 'm&#97;nch&#101;st&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy69082 + suffix + '\'' + attribs + '>' );
 document.write( addy69082 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy4218 = 'st&#101;v&#101;.p&#101;tt&#105;f&#101;r' + '&#64;';
 addy4218 = addy4218 + 'm&#97;nch&#101;st&#101;r' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy4218 + suffix + '\'' + attribs + '>' );
 document.write( addy4218 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Manchester</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) School of Computer Science, University of
Manchester, UK (2) EMBL European Bioinformatics Institute, Cambridge, UK</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">The BioCatalogue project is funded by the BBSRC [BB/F01046X/1,
BB/F010540/1], with additional funding from the European Commission via the
EMBRACE project [LHSG-CT-2004-512092]. Development on Search By Data was funded
by EMBO [ASTF 338.00-2009]. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection15">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746932">I-14. Regression system for prediction of errors in
the data on gene expression in situ obtained from confocal images</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Myasnikova E<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Myasnikova E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Surkova S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Surkova S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Samsonova M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Samsonova M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In the recently published paper (Myasnikova et al.,
Bioinformatics, 2009) we presented an algorithm for estimation of errors in the
data on gene expression in situ extracted from confocal images. The application
of the method is limited by the requirement of additional information that is
not available from the standard procedure of data acquisition, and the special
design of experiments is needed. The aim of the work is to create a learning
system for the prediction of error size in the data obtained from a confocal
image based on the information about the parameters of the microscope.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The common way to reduce the noise in confocal
images is averaging of multiple frames that however leads to the biased data in
case of clipped single frames. For the estimation and correction of this kind
of errors a method based on censoring technique is used, which requires the
availability of all the confocal scans along with the averaged image. To
predict error size in the data extracted from the averaged image we developed a
regression system. The learning sample is composed of images obtained at
different combinations of microscope parameters, and for each image all the
scans are saved.</p>

<h3>Results</h3>

<p class="abresults">The regression system was applied to the data on
segmentation gene expression in Drosophila blastoderm stored in the FlyEx
database (http://urchin.spbcas.ru/flyex/). The predicted errors proved to be of
small size not exceeding 5-7% of the mean intensity level in the embryo
nucleus.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The high quality of confocal images allows to extract high-precision
information at cellular resolution. However the data accuracy is limited due to
experimental errors that arise in course of confocal scanning. High values of
errors are usually caused by improper choice of the microscope settings used in
order to increase the image brightness and contrast. An important application
of the current work is the possibility to accurately correct this kind of
errors thereby allowing to obtain images of the higher dynamical range and thus
to extract more detailed quantitative information from them.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://urchin.spbcas.ru/downloads/step/step.htm" mce_href="http://urchin.spbcas.ru/downloads/step/step.htm">http://urchin.spbcas.ru/downloads/step/step.htm</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Ekaterina Myasnikova (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy39631 = 'my&#97;sn&#105;k&#111;v&#97;' + '&#64;';
 addy39631 = addy39631 + 'spbc&#97;s' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy39631 + suffix + '\'' + attribs + '>' );
 document.write( addy39631 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60603 = 'my&#97;sn&#105;k&#111;v&#97;' + '&#64;';
 addy60603 = addy60603 + 'spbc&#97;s' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy60603 + suffix + '\'' + attribs + '>' );
 document.write( addy60603 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61718 = 'my&#97;sn&#105;k&#111;v&#97;' + '&#64;';
 addy61718 = addy61718 + 'spbc&#97;s' + '&#46;' + 'r&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy61718 + suffix + '\'' + attribs + '>' );
 document.write( addy61718 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">St.Petersburg State Polytechnical University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">St.Petersburg State Polytechnical University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is supported by NIH grant RR07801, GAP award
RUB1-1578 and RFBR grants 08-04-00712-a, 08-01-00315-a and 09-04-01590-a. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection16">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746933">I-15. Impact of genetic variations on
phosphorylation sites</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Via A<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Via A&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1,*), Le
Pera L<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span> XE &quot;Le
Pera L&quot; <![endif]--><!--[if supportFields]><span style="mso-element:field-end" mce_style="mso-element:field-end"></span><![endif]--><span style="">&nbsp;</span>(1), Ferré F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ferré F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Tramontano A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tramontano A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Most genes produce a wide range of different products
that might change in different tissues/cells, conditions, individuals, and in
time. Each product might have different features and roles: genomic variations
can affect the protein or transcript functional properties, regulation,
efficiency, and specificity of action. In order to investigate how the
environment created by genome variations affects functional sites, we are
carrying out a systematic analysis of the mutual interactions between
phosphorylation sites (Psites), polymorphisms (SNPs), deleterious mutations,
and splice variants.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We mapped Mendelian Inheritance in Man (MIM)
mutations, cancer-related mutations and SNPs, and Psites (from Phospho.ELM) on
Ensembl56 alternative splicing (AS) isoforms. Control sites were defined for
each case to evaluate statistically significant inter-dependence of these
features. Single-nucleotide resolution level expression data were retrieved
from publicly available Illumina Genome Analyzer RNA-seq datasets from adult
human tissues to measure when and how much individual sites are spliced.</p>

<h3>Results</h3>

<p class="abresults">Our study highlights that the co-occurrence of genome
variations and functional sites is not randomly distributed, providing clues
about the context in which a mutation can be deleterious, and suggesting
possible scenarios of protein evolution. In particular, we found that
cancer-related mutations are overrepresented, compared with a control set of
non-Psites, in the [-5,+5] surrounding of pS and pT and not pY. No
statistically significant differences were observed for SNPs. Additionally,
Psites are less spliceable than expected in Ensembl isoforms, and less spliced
in human tissues.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We addressed important questions such as how and why an
important class of post-translational modification sites is expressed and
spliced. Interestingly, AS does not seem to be extensively used to modulate the
presence of Psites, suggesting that other regulation forms are more appropriate
for transient interactions such as phosphorylation. Additionally, our findings
provide clues on the inter-relation between cancer-associated mutations and
Psites.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Allegra Via (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy6336 = '&#97;ll&#101;gr&#97;.v&#105;&#97;' + '&#64;';
 addy6336 = addy6336 + '&#117;n&#105;r&#111;m&#97;1' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy6336 + suffix + '\'' + attribs + '>' );
 document.write( addy6336 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93041 = '&#97;ll&#101;gr&#97;.v&#105;&#97;' + '&#64;';
 addy93041 = addy93041 + '&#117;n&#105;r&#111;m&#97;1' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy93041 + suffix + '\'' + attribs + '>' );
 document.write( addy93041 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy13122 = '&#97;ll&#101;gr&#97;.v&#105;&#97;' + '&#64;';
 addy13122 = addy13122 + '&#117;n&#105;r&#111;m&#97;1' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy13122 + suffix + '\'' + attribs + '>' );
 document.write( addy13122 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Department of Biochemical Sciences "A.
Rossi-Fanelli" Sapienza University of Rome</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Department of Biochemical Sciences "A.
Rossi-Fanelli" Sapienza University of Rome 2 Istituto Pasteur, Fondazione
Cenci Bolognetti, Sapienza University of Rome</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection17">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746934">I-16. FuSiGroups: grouping gene products by
functional similarity</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Welter DN<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Welter DN&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Gray WA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gray WA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Kille P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Kille P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Gene product annotations have become important in the
discovery of functionally related gene products in large datasets. Several
approaches to quantify annotation similarity have been developed. However, few
efforts have addressed the potential applications of these similarity
approaches. We propose a novel algorithm to group gene products based on their
functional similarity.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The FuSiGroups algorithm calculates the functional
similarity between all gene products in a list based on the semantic similarity
between the GO terms they are annotated with. A number of different semantic
and functional similarity measures are available. Then gene products are sorted
into groups that represent their different functional aspects. Each group has a
“deﬁnition” of one or more GO terms that link the gene products. This makes it
easy to determine the functional aspect(s) that a set of gene products share.</p>

<h3>Results</h3>

<p class="abresults">We determined the minimum recommended semantic and
functional thresholds for different measures using ROC curves of three
different types of datasets, namely gene expression, protein interaction and
phenotypes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">An advantage of our algorithm is that it reﬂects
biological reality better than traditional hierarchical clustering as each gene
product can be in multiple groups at the same time. The approach also excludes
gene products with insufficient functional similarity to all other gene
products so that only gene products with common functionality are grouped
together. In summary, we exploit existing semantic and functional similarity
measures in a new approach for grouping gene products based on their Gene Ontology
annotation, which enables easy discovery of functional links between gene
products.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Danielle N.C. Welter (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22533 = 'd.n.w&#101;lt&#101;r' + '&#64;';
 addy22533 = addy22533 + 'cs' + '&#46;' + 'cf' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy22533 + suffix + '\'' + attribs + '>' );
 document.write( addy22533 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12140 = 'd.n.w&#101;lt&#101;r' + '&#64;';
 addy12140 = addy12140 + 'cs' + '&#46;' + 'cf' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy12140 + suffix + '\'' + attribs + '>' );
 document.write( addy12140 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77844 = 'd.n.w&#101;lt&#101;r' + '&#64;';
 addy77844 = addy77844 + 'cs' + '&#46;' + 'cf' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy77844 + suffix + '\'' + attribs + '>' );
 document.write( addy77844 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Cardiff University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Cardiff School of Computer Science and Informatics,
Cardiff University 2 Cardiff School of Biosciences, Cardiff University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work is funded by AFR grant "TR-PHD
BFR06-102" from the "Fonds National de la Recherche (Luxembourg"</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection18">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746935">I-17. A file system strategy for high-performance
sequence interval queries of very large datasets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Karcz SR<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Karcz SR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, Links MG<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Links MG&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->, Parkin IAP<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Parkin IAP&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The huge numbers of variant sequence features arising
from genome re-sequencing projects jeopardizes our ability to efficiently
query, visualize and make decisions with the information. The critical
scalability issue in current storage and query systems is sequence interval
query performance. We developed a scalable file system strategy optimized for
high performance range queries and a simple method for visualizing the data
using the open source genome browser JBrowse. Our goal was to enable efficient
utilization of large datasets without extensive computational infrastructure.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Using the Arabidopsis thaliana genome as a basis,
we populated a mysql database and a GF2S file system with identical simulated
SNP feature data ranging in numbers from 32 to 1024 million and measured
population time, disk usage and sequence interval query times for both systems.
These performance metrics provided a comparison between a RAID-5 spinning disk
solution and a solid state device. A 256 million feature GF2S was used as a
dynamic data source for the JBrowse genome browser. All tests were run on a
Dell R900 server (Centos 5.4, 64GB RAM) using only free and open source
software.</p>

<h3>Results</h3>

<p class="abresults">Our write tests showed that GF2S can be populated at a rate
5 times faster than mysql and can write 512 million features overnight (16
hours). Sequence interval query tests demonstrated that GF2S could return
feature data at a rate 10-20 times faster than mysql using sequence intervals
from 1 kb to 100 kb and for dataset sizes ranging from 32 to 256 million
features. The file system can store 1024 million features in under 600 GB.
JBrowse was also able to be used without modifying its source code to
dynamically receive JSON data from the GF2S without noticeable latency.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We have demonstrated that the standard XFS filesystem can
be efficiently utilized for high performance sequence interval queries on
datasets ranging up to 1 billion SNP features. This was accomplished without
resorting to any strategies for caching, multi-node clustering, virtualization
or cloud computing. The relative ease of adapting the file system to serve data
to a standard genome browser application also suggests that the scalability and
speed of the GF2S could be made available to other "interval
query-centric" visualization applications in computational biology.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Steven R. Karcz (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy79987 = 'k&#97;rczs' + '&#64;';
 addy79987 = addy79987 + '&#97;gr' + '&#46;' + 'gc' + '&#46;' + 'c&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy79987 + suffix + '\'' + attribs + '>' );
 document.write( addy79987 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57028 = 'k&#97;rczs' + '&#64;';
 addy57028 = addy57028 + '&#97;gr' + '&#46;' + 'gc' + '&#46;' + 'c&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy57028 + suffix + '\'' + attribs + '>' );
 document.write( addy57028 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29378 = 'k&#97;rczs' + '&#64;';
 addy29378 = addy29378 + '&#97;gr' + '&#46;' + 'gc' + '&#46;' + 'c&#97;';
 document.write( '<a ' + path + '\'' + prefix + addy29378 + suffix + '\'' + attribs + '>' );
 document.write( addy29378 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Agriculture and AgriFood Canada</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Agriculture and AgriFood Canada 107 Science Place Saskatoon,
SK, Canada S7N 0X2</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We gratefully acknowledge funding provided by the
Canadian Crop Genomics Initiative</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection19">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746936">I-18. Evaluation of multivariate data analysis
strategies for high-content screening</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Kümmel A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Kümmel A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Parker C N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Parker C N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Gabriel D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Gabriel D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Biological measurements are increasingly becoming
multiparametric. In particular, high content screening (HCS) can monitor
multiple phenotypic readouts of e.g. cell morphology or protein levels and
localization. To analyze such results a multitude of computational methods is
available. However, there has been no published comparison of the performance
and applicability of such methods. The aim of this study has been to establish
a generic and modular data processing pipeline that enables a systematic review
of alternative data processing and analysis methods to assess HCS data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Multivariate HCS data from single cells monitoring
cell cycle was analyzed using alternative methods for dimension reduction and
well summary. (a) The dimensionality was reduced using either factor analysis
or a successive elimination of the parameter with highest correlation to
others. (b) The cell populations of different wells were compared using either
the median, 5 percentile values or the KS statistic. The alternatives were
evaluated on the ability of the resulting data sets to distinguish between
controls using classification accuracy or the Z’ factor using linear
discriminant analysis.</p>

<h3>Results</h3>

<p class="abresults">If parameters are eliminated that are highly correlated to
others the ability to discrimination between positive and negative controls is
not significantly reduced but only leads to a limited reduction in
dimensionality. Factor analysis reduces the discrimination ability in general
but often performs better if a low number of dimensions is chosen to represent
the data. Using the KS statistic to compare control samples is not more
sensitive than a comparison of the medians. Taking 5 percentile values to
summarize a well improves the discrimination in comparison to only the well
median.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Using the data analysis pipeline presented here,
alternative strategies were evaluated for analyzing a HCS data set. The
alternatives for reducing the dimensionality or summarizing the cell population
were quantitatively evaluated based on positive control samples. Obviously,
other methods to those applied here are available. E.g., the cells within each
well can be summarized by fractions of subpopulations. The logical and modular
assembly of the data analysis pipeline allows for additional methods to be
tested when establishing a suitable strategy for a particular data set.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anne Kummel (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy68055 = '&#97;nn&#101;.k&#117;&#101;mm&#101;l' + '&#64;';
 addy68055 = addy68055 + 'n&#111;v&#97;rt&#105;s' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy68055 + suffix + '\'' + attribs + '>' );
 document.write( addy68055 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy81559 = '&#97;nn&#101;.k&#117;&#101;mm&#101;l' + '&#64;';
 addy81559 = addy81559 + 'n&#111;v&#97;rt&#105;s' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy81559 + suffix + '\'' + attribs + '>' );
 document.write( addy81559 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62415 = '&#97;nn&#101;.k&#117;&#101;mm&#101;l' + '&#64;';
 addy62415 = addy62415 + 'n&#111;v&#97;rt&#105;s' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy62415 + suffix + '\'' + attribs + '>' );
 document.write( addy62415 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Novartis Institues of Biomedical Research</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Novartis Institutes for Biomedical Research</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Postdoc fellowship is funded by the NIBR Education
Office</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection20">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746937">I-19. Studies of the binding capacity of
cyclooxygenase II and biodistribution of phenols contained in natural products</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Paulino Z-M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Paulino Z-M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2,*), Aguilera M-S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Aguilera M-S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), García Jenifer García<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;García Jenifer García&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Iribarne F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Iribarne F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Phenolic compounds, widely distributed and abundant in
natural products such as red wine and propolis, are known for their antioxidant
properties. Their biochemical traits are also of great interest in relation to
the interaction with mediators of inflammatory processes such as cyclooxygenase
II (COX-II). In a previous work, we identified phenols in Uruguayan propolis
(M. Paulino Z et al. (2010), Portugal (S. Falcão. Et al. (2010)) and wine
grapes (K Ali et al (2009)). The detected structures included phenolic acids,
flavonols, flavones, flavanones, flavanols, anthocyanins and stilbenes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">In this paper the molecular interactions of these
phenols with mouse COX-II (Kiefer R et al (2000)) as well as their
biodistribution properties were studied using in silico techniques. All
calculations were performed with MOE 2007.09 simulation package. Phenol
structures were modeled and optimized. The three-dimensional structure of
COX-II was obtained from PDB (Protein Data Bank, code 1CVU). All possibilites
for binding sites were examined. Flexible docking was performed at all binding
sites identified, using for each compound a conformational basis built in
advance.</p>

<h3>Results</h3>

<p class="abresults">Docking energies (scores) varied from -18 to -8 kcal/mol and
were divided into three ranges: high (-18 to -14), medium (-13 to -10 ) and low
(-10 to -8). For the best scoring poses, phenols were sorted as follows:
anthocyanins, flavonols and flavanols (high and medium scores); flavones and
flavanones (medium scores); stilbenes (medium and low scores) and phenolic
acids (low scores). Additionally, for each ligand, the best score conformations
we analyzed in terms of the relationship among the binding energy, the number
of hydroxylations and the number of contacts.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Finally, given that compound biodistribution may define
the quality of a candidate to be developed as an anti-inflammatory drug, a QSAR
study was conducted to identify which properties associated with structure
(hydrophobicity, molecular weight, accessible polar area, number of rotatable
bonds and drug-ability) could affect compound bioavailability.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Margot Paulino Zunini (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy61149 = 'm&#97;rg&#111;t' + '&#64;';
 addy61149 = addy61149 + 'fq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#117;y';
 document.write( '<a ' + path + '\'' + prefix + addy61149 + suffix + '\'' + attribs + '>' );
 document.write( addy61149 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy88585 = 'm&#97;rg&#111;t' + '&#64;';
 addy88585 = addy88585 + 'fq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#117;y';
 document.write( '<a ' + path + '\'' + prefix + addy88585 + suffix + '\'' + attribs + '>' );
 document.write( addy88585 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy44898 = 'm&#97;rg&#111;t' + '&#64;';
 addy44898 = addy44898 + 'fq' + '&#46;' + '&#101;d&#117;' + '&#46;' + '&#117;y';
 document.write( '<a ' + path + '\'' + prefix + addy44898 + suffix + '\'' + attribs + '>' );
 document.write( addy44898 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Facultad de Química</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Departamento de Química y Farmacia. Departamento de
Física. Facultad de Ciencias. Universidad Católica del Norte. Avda. Angamos
0610. Antofagasta. Chile. 2. LaBioFarMol, DETEMA, Facultad de Química, UdelaR,
General Flores 2124, 11600, Montevideo, Uruguay</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Funding: Minera Escondida Limitada (Chile).
Acknowledgments: Pablo Carmona Venegas, undergraduate student, UCN. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection21">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746938">I-20. Finding logic networks using the Dutch Life
Science Grid: handling 15 million jobs</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bot J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Bot J&quot; <![endif]--><!--[if supportFields]><span style="mso-element:
field-end" mce_style="mso-element:
field-end"></span><![endif]--><span style="" mce_style="mso-spacerun:yes">&nbsp;</span>(1, 2, *),
de Ridder J<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;de Ridder J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2), Reinders M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reinders M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, 2)</p>

<p class="abmotivation">Using insertional mutagenesis one can get insight on how
a tumor develops, here we try to explain an observed gene expression with a
logical combination of observed insertions. Finding such logic networks is
computationally intensive. When using grid computing, finding one network can
seen as one job whose runtime is unpredictable and ranges from 2 seconds to 14
hours. The number of possible combinations, networks and the need for
permutations results in more than 15 Million jobs which the grid middleware is
unable to schedule and whose output cannot be stored on standard grid facilities.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We developed a method built upon the ToPoS pilot
job framework to schedule and keep track of the jobs. For dealing with the
outputs we wrote our own XML-RPC based web-service with database back-end. We
also implemented clients to easily communicate between the grid nodes and the
ToPoS schedular as well as with the database. The web-service receives both the
outputs of the individual runs as well as the errors generated while executing.
This makes sure that the user can quickly find out whether a job ran
successfully or inspect problems on the grid.</p>

<h3>Results</h3>

<p class="abresults">Our method was able to deal with the 15 million jobs (all
generating output) that ran on 5000+ cores. The algorithm ran for about a month
on the Dutch Life Science Grid doing about a century's worth of CPU work. The
debugging facilities were very useful as they provided insight in the different
errors and irregularities of the grid. The proposed method also allowed us to
easily add a desktop cluster to our compute facilities.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The method itself is generic so it is able to deal with
other bioinformatics applications, in different domains, and with other data
types. However, the current implementation needs be revised to make it easier
for layman end-users to work with.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jan J. Bot (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy1596 = 'j.j.b&#111;t' + '&#64;';
 addy1596 = addy1596 + 't&#117;d&#101;lft' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy1596 + suffix + '\'' + attribs + '>' );
 document.write( addy1596 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy12647 = 'j.j.b&#111;t' + '&#64;';
 addy12647 = addy12647 + 't&#117;d&#101;lft' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy12647 + suffix + '\'' + attribs + '>' );
 document.write( addy12647 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy22098 = 'j.j.b&#111;t' + '&#64;';
 addy22098 = addy22098 + 't&#117;d&#101;lft' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy22098 + suffix + '\'' + attribs + '>' );
 document.write( addy22098 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Delft University of Technology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Delft Bioinformatics Lab, Delft University of
Technology (2) Netherlands Bioinformatics Centre</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection22">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746939">I-21. A detailed view on several Model-Based
Multifactor Dimensionality Reduction methods for detecting gene-gene
interactions in case-control data in the absence and presence of noise</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Cattaert T<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Cattaert T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Van Lishout F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Lishout F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Mahachie John JM<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mahachie John JM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Van Steen K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Steen K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">When searching for epistasis, parametric approaches have
severe limitations. Alternatively, the non-parametric Multifactor
Dimensionality Reduction (MDR) method can be applied. It handles the
dimensionality problem by pooling multi-locus genotypes into two groups of
risk: High Risk and Low Risk. However, MDR involves computationally intensive
cross-validations because it is based on prediction. Moreover, MDR is
restricted to univariate, dichotomous traits, and it is not possible to
flexibly adjust for lower-order effects and confounders.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Recently, Calle et al. proposed the Model-Based MDR
(MB-MDR) method. Association tests are applied to identify risk cells and to
test the final one-dimensional construct, and a third risk category, that of ‘No
Evidence for Risk’, is introduced. We evaluate the empirical power of MB-MDR to
detect gene-gene interactions in the absence of any noise and in the presence
of genotyping error, missing data, phenocopy, and genetic heterogeneity. We
also explore the potential of alternative definitions for risk cell
identification, based on ranking of the case to control ratios.</p>

<h3>Results</h3>

<p class="abresults">Our simulation results illustrate that MB-MDR has increased
power over MDR to identify gene-gene interactions for most considered genetic
models, in particular in the presence of genetic heterogeneity, phenocopy, or
low minor allele frequencies. Preliminary results indicate that the alternative
risk cell definitions based on ranking further improve MB-MDR power. Type-I
error rates have been found to be close to the nominal level. Finally, MB-MDR
is much less computationally intensive than MDR.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Using association tests allows multiple models to be
proposed, no longer requires cross-validations, and flexibly deals with
different outcome types and covariates. Hence, MB-MDR offers a flexible
framework that naturally deals with different outcome types (e.g. categorical,
continuous or survival type) and allows for lower order effects corrections or
adjustments for important confounding factors. Furthermore, it can easily
handle (a combination of) unrelated individuals and families of any size and
structure.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Tom Cattaert (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy57325 = 't&#111;m.c&#97;tt&#97;&#101;rt' + '&#64;';
 addy57325 = addy57325 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy57325 + suffix + '\'' + attribs + '>' );
 document.write( addy57325 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy19053 = 't&#111;m.c&#97;tt&#97;&#101;rt' + '&#64;';
 addy19053 = addy19053 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy19053 + suffix + '\'' + attribs + '>' );
 document.write( addy19053 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58626 = 't&#111;m.c&#97;tt&#97;&#101;rt' + '&#64;';
 addy58626 = addy58626 + '&#117;lg' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy58626 + suffix + '\'' + attribs + '>' );
 document.write( addy58626 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Liege</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Montefiore Institute, University of Liege, Belgium (2)
GIGA-Research, University of Liege, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">T. Cattaert is a Postdoctoral Researcher of the Fonds
de la Recherche Scientifique - FNRS. T. Cattaert, F. Van Lishout, J. M. Mahachie
John and K. Van Steen acknowledge research opportunities offered by the Belgian
Network BioMAGNet (Bioinformatics and Modelling: from Genomes to Networks),
funded by the Interuniversity Attraction Poles Programme (Phase VI/4),
initiated by the Belgian State, Science Policy Office. Their work was also
supported in part by the IST Programme of the European Community, under the
PASCAL2 Network of Excellence (Pattern Analysis, Statistical Modelling and
Computational Learning), IST-2007-216886. In addition, F. Van Lishout
acknowledges support by Alma in Silico, funded by the European Commission and
Walloon Region through the Interreg IV Program. The scientific responsibility
for this work rests with its authors. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection23">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746940">I-22. Quality ranking of 16S sequences: an approach
based on poset theory</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Smet W<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Smet W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Verslyppe B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Verslyppe B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Loof K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Loof K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Vos P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Vos P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Baets B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Baets B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Dawyndt P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dawyndt P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In the field of microbial taxonomy, the gene sequences
coding for the 16S ribosomal RNA (rRNA) are now an integral part of many taxonomic
studies. Because of their usefulness in divining the evolutionary past, 16S
rRNA sequences of many taxa have proliferated in the International Sequence
Database Collaboration (INSDC) databases. Increasingly, several sequences are
available for any one species and tools become necessary to support researchers
who want to quickly and easily gather available sequences and assess their
quality.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">The StrainInfo project (http://www.straininfo.net/)
already provides a useful resource to microbiological researchers, by
integrating information about microbiological cultures, available in Biological
Resource Centers (BRCs) around the world, on one single strain passport.
Included is taxonomic and sequence data that can then be used to automate the
sequence selection process. We used a ranking algorithm based on the theory of
partially ordered sets to select an appropriate sequence and compare the
results with those of the All-Species Living Tree Project.</p>

<h3>Results</h3>

<p class="abresults">Exploration of the results of this comparison show some
limitations in the sequence retrieval process, caused by a lack of usable
annotations. Comparison with the results of a manually curated data set reveal
several controversial or surprising picks, depending on the quality criteria
used. The approach used shows promise as a way to quickly explore available
sequences for particular genera and visualize quality differences.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Despite some limitations, automated retrieval often finds
enough sequences to rank, visualize and build a first approximate phylogenetic
tree of any genus with. Leveraging data available within StrainInfo we can make
this a single step process, helping researchers to waste less time searching
for sequences and more time doing research. Visualization of the poset by the
various used criteria is an especially promising way to quickly get an overview
of available sequences and their relative merits.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Wim P.P. De Smet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy55669 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy55669 = addy55669 + 'UG&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy55669 + suffix + '\'' + attribs + '>' );
 document.write( addy55669 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy62084 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy62084 = addy62084 + 'UG&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy62084 + suffix + '\'' + attribs + '>' );
 document.write( addy62084 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41475 = 'W&#105;m.D&#101;Sm&#101;t' + '&#64;';
 addy41475 = addy41475 + 'UG&#101;nt' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy41475 + suffix + '\'' + attribs + '>' );
 document.write( addy41475 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Ghent University</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Ghent University</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Research carried out by the Interuniversity Attraction
Poles programme funded by the Belgian Science Policy</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection24">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746941">I-23. Clustering and kernel methods using R</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Adefioye A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Adefioye A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->*, De Moor B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Moor B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--></p>

<p class="abmotivation">The kernel methods are advantageous, since they are able
to take the non-linear aspect of a given dataset into consideration. The
ability of the methods are tested on compounds represented either as bit string
representations of molecular structures using fingerprints or as physiochemical
descriptors. The performances of the kernel based methods are compared to each
other as well as to more traditional methods such as those based on
partitioning around mediods. The kernel based methods give a better cluster
representation of the compound set.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Represented the chemical compounds using six
different descriptor types ( Fingerprints: Estate, Extended, MACCS, Graph,
Standard and physiochemical descriptors). Clustered the compounds using these
representation, with the hiearachical Ward clustering algorithm. Determined k. With
k=4, clustered all the different types of compound representation using k
K-Mean, spectral clustering and pam clustering. The clusters are checked using
cluster validity methods.</p>

<h3>Results</h3>

<p class="abresults">The kernel based methods give a better cluster
representation of the compound set.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Clustering can answer important questions when using
machine learning approaches for chemoinformatics purposes. Initially, the
hierarchical clustering algorithm is able to indicate several clusters these
clusters, in this case 4, can be used to decide what training set one will use,
and which set of compounds will be in the test set. Here we focus on the
usefulness of the hierarchical cluster in selecting the number of k’s, for the
k k-Means run. The clustering done by the Ward hierarchical method has some
notable mistakes, in effect showing the necessity for more superior methods.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Adeshola A Adefioye (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3197 = 't&#117;nd&#101;.&#97;d&#101;f&#105;&#111;y&#101;' + '&#64;';
 addy3197 = addy3197 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy3197 + suffix + '\'' + attribs + '>' );
 document.write( addy3197 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy94948 = 't&#117;nd&#101;.&#97;d&#101;f&#105;&#111;y&#101;' + '&#64;';
 addy94948 = addy94948 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy94948 + suffix + '\'' + attribs + '>' );
 document.write( addy94948 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy10556 = 't&#117;nd&#101;.&#97;d&#101;f&#105;&#111;y&#101;' + '&#64;';
 addy10556 = addy10556 + '&#101;s&#97;t' + '&#46;' + 'k&#117;l&#101;&#117;v&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy10556 + suffix + '\'' + attribs + '>' );
 document.write( addy10556 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Katholieke Universiteit Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">Katholieke Universiteit Leuven</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Katholieke Universiteit Leuven</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection25">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746942">I-24. Ensuring data integrity in large modern
integrative genomic studies</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Lynch A<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Lynch A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Dunning M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dunning M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Chin S-F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Chin S-F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Curtis C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Curtis C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">This is an age of large, multi-dimensional genomic
studies with clinical applications (e.g. TCGA, METABRIC). As a consequence,
samples may be drawn in from many clinics, prepared and plated several times for
different technologies, and distributed to several laboratories. As a result of
these complications, it is possible for errors to occur in the mapping of
sample IDs to resultant data, with detrimental consequences for downstream
analyses.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Where there exist strong relationships between
recorded genotypes and recorded expression (be that due to genuine eQTLs or
technical biases) then we can use a record of such relationships to establish a
score for the agreement between a set of genotypes and a set of expression
intensities.</p>

<h3>Results</h3>

<p class="abresults">We present a simple iterative algorithm to validate and
correct sample ID mappings for experiments running expression microarrays and
also generating genotyping data. Its performance in identifying ID mapping
errors is exceptional in large studies, and can pick out systematic changes in
relatively modest experiments.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The approach is naturally dependent on the quality of the
data arising from technologies, so may demonstrate a high-false positive rate
for poor quality samples - although it is no bad thing to have one's attention
drawn to such samples. If the data are then to be used to search for new eQTLs
then there are some subtleties and potential biases for which one should be
prepared, but these pale in comparison to the difficulties that mis-identified
samples would bring.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Andy G. Lynch (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy7415 = '&#97;ndy.lynch' + '&#64;';
 addy7415 = addy7415 + 'c&#97;nc&#101;r' + '&#46;' + '&#111;rg' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy7415 + suffix + '\'' + attribs + '>' );
 document.write( addy7415 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy34579 = '&#97;ndy.lynch' + '&#64;';
 addy34579 = addy34579 + 'c&#97;nc&#101;r' + '&#46;' + '&#111;rg' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy34579 + suffix + '\'' + attribs + '>' );
 document.write( addy34579 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy71159 = '&#97;ndy.lynch' + '&#64;';
 addy71159 = addy71159 + 'c&#97;nc&#101;r' + '&#46;' + '&#111;rg' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy71159 + suffix + '\'' + attribs + '>' );
 document.write( addy71159 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Cambridge</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 - Department of Oncology, University of Cambridge, Li
Ka Shing Centre, Robinson Way, Cambridge, CB2 0RE 2 - Cancer Research UK
Cambridge Research Institute, Li Ka Shing Centre Robinson Way, Cambridge, CB2
0RE</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We acknowledge the support of The University of Cambridge,
Cancer Research UK and Hutchison Whampoa Limited. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection26">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746943">I-25. Testing branches in dendrograms representing
species abundance data.</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Calkiewicz J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Calkiewicz J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1*), Wlodarska-Kowalczuk M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wlodarska-Kowalczuk M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Wrobel B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wrobel B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,3)</p>

<p class="abmotivation">Hierarchical clustering is one of the most popular
techniques to represent ecological data, eg. species abundances in samples.
Usually, however, the resulting dendrograms are presented without support for
branches. In this work we aim to investigate if some methods devised to find
support for branches in phylogenetic trees are useful in the analysis of
ecological data. We use BP (bootstrap proportion, Felsenstein test), AU
(Shimodaira approximate unbiased test), and two methods that ask a question if
the branch length is significantly longer than zero: the Dopazo and WLS-LRT.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We reanalysed data from 3 published studies
(Wlodarska-Kowalczuk et al. 2004. Deep-Sea Res I51:1903; Wlodarska-Kowalczuk
and Pearson 2004. Polar Biol 27:155; Wlodarska-Kowalczuk and Weslawski 2008.
Mar Ecol Prog Ser 356:215). Species that occurred in only one sample were
removed, and data were transformed using square root, double root or log. We
used 2 distance measures (Bray-Curtis or Euclidean), 4 linkage criteria
(single, complete, average linkage or neighbor-joining). For all 4 approaches
to test branches, we bootstrapped the data by random sampling the species with
replacement.</p>

<h3>Results</h3>

<p class="abresults">For the datasets we used here, biologically relevant
clustering was observed most easily for average linkage and data transformed
with double root. Regardless of the method used, the obtained dendrograms
contained branches that separated clusters that had biological relevance.
However, these branches were judged significant only using the Dopazo test and
WLS-LRT. The support for branches obtained using the BP and AU was mostly below
0.70. This indicates that some approaches originally proposed for phylogenetic
trees can be applied in the analysis of species abundance data.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The fact that the BP and AU approaches result in low
support for branches may explain why these methods are rarely used: if branches
are not judged significant even for high quality data, it is understandable
that a method does not reach acceptance. However, it seems that the methods
that are based on asking a question if a particular branch separating clusters
is longer than zero can be useful for determining if the clusters observed
using a particular distance measure or linkage method have statistical support.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.evosys.org/" mce_href="http://www.evosys.org/">http://www.evosys.org/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Joanna Calkiewicz (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy69132 = 'c&#97;lk&#105;&#101;w&#105;cz' + '&#64;';
 addy69132 = addy69132 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy69132 + suffix + '\'' + attribs + '>' );
 document.write( addy69132 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy40914 = 'c&#97;lk&#105;&#101;w&#105;cz' + '&#64;';
 addy40914 = addy40914 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy40914 + suffix + '\'' + attribs + '>' );
 document.write( addy40914 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64199 = 'c&#97;lk&#105;&#101;w&#105;cz' + '&#64;';
 addy64199 = addy64199 + '&#105;&#111;p&#97;n' + '&#46;' + 'gd&#97;' + '&#46;' + 'pl';
 document.write( '<a ' + path + '\'' + prefix + addy64199 + suffix + '\'' + attribs + '>' );
 document.write( addy64199 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Computational Biology Group, Institute of Oceanology,
Polish Academy of Sciences, Sopot, Poland</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Computational Biology Group, Institute of
Oceanology, Polish Academy of Sciences, Sopot, Poland (2) Marine Ecosystems
Laboratory, Institute of Oceanology, Polish Academy of Sciences, Sopot, Poland (3)
Laboratory of Bioinformatics, Adam Mickiewicz University, Poznan, Poland</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Polish Ministry of Science and Education (projects N303
291234 and N N519 402037) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection27">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746944">I-26. Reliable identification of hundreds of
proteins without peptide fragmentation</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bochet P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bochet P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Rügheimer F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Rügheimer F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Guina T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Guina T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Brooks P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Brooks P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4,5), Goodlett D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Goodlett D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Clote P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Clote P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7,8,9), Schwikowski B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schwikowski B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">One of the most common approaches for large scale protein
identification is Hight Performance Liquid Chromatography, followed by Mass
Spectrometry (HPLC-MS). If more that a few proteins have to be identified, the additional
fragmentation of individual peptides has been considered essential. Here we
present evidence that, by combining high-precision mass measurement and modern
retention time prediction algorithms (Krokhin et al 2006) with a robust scoring
scheme, hundreds of proteins can be identified without having to rely on
peptide fragmentation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">For each candidate protein, taken from a relevant
protein sequence database, we predict the peptides resulting from the digestion
of the protein and sort them according to their computed HPLC retention times.
Corresponding peaks are then searched in the same order in the series of
spectra from HPLC-MS. For this purpose, a set of peaks with masses
corresponding to those of the predicted peptides is searched by a dynamic
programming algorithm maximizing an alignment score. Using quantile regression,
this alignment score is compared to those obtained for proteins from a suitable
decoy database.</p>

<h3>Results</h3>

<p class="abresults">The method was tested on HPLC-MS data obtained from the
pathogenic bacteria Francisella tularensis, for which HPLC-MS/MS fragmentation
spectra were also available. Out of 1719 possible proteins in F. tularensis, we
were able to detect 257 proteins with a FDR of 5.7% (The FDR was estimated from
the score distribution of the proteins in the decoy database). This is 59% of
the number of proteins detected by applying the Mascot tool to the
fragmentation spectra from the same sample. 31 additional proteins were seen
that had not been detected in the fragmentation data by Mascot.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Previous works describe the identification of proteins
from MS and measured retention times (Strittmatter et al, 2003) or use the
predicted retention times of peptides to filter false identifications from
MS/MS (Pfeifer et al. 2009). Relying only on MS and predicted retention times,
we detect many more proteins than Palmblad et al 20004 using the same inputs.
Using the high accuracy now available in MS, and eliminating the need for the
measurement of the retention times of peptides, this method could be an
alternative to peptide fragmentation for the protein identification in complex
samples.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Pascal F. Bochet (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy82253 = 'p&#97;sc&#97;l.b&#111;ch&#101;t' + '&#64;';
 addy82253 = addy82253 + 'p&#97;st&#101;&#117;r' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy82253 + suffix + '\'' + attribs + '>' );
 document.write( addy82253 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy63446 = 'p&#97;sc&#97;l.b&#111;ch&#101;t' + '&#64;';
 addy63446 = addy63446 + 'p&#97;st&#101;&#117;r' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy63446 + suffix + '\'' + attribs + '>' );
 document.write( addy63446 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy76339 = 'p&#97;sc&#97;l.b&#111;ch&#101;t' + '&#64;';
 addy76339 = addy76339 + 'p&#97;st&#101;&#117;r' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy76339 + suffix + '\'' + attribs + '>' );
 document.write( addy76339 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CNRS, Institut Pasteur</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 Institut Pasteur, System Biology Laboratory, Dept
Génomes et Génétique, Paris, France 2 CNRS, URA2171, Paris, France 3 University
of Washington, Dept of Pediatrics, Seattle, WA, USA 4 CNRS, UMR7592, Paris,
France 5 Université Paris 7, Paris, France 6 University of Washington, Dept of
Medicinal Chemistry, Seattle, WA, USA 7 Biology Dept, Boston College, Boston,
MA,USA 8 Ecole Polytechnique, LIX, Palaiseau, France 9 LRI, Université Paris
XI,Orsay, France</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">Work of P. Clote was partially funded by the Digiteo
Foundation, as well as by the National Science Foundation grants DBI-0543506
and DMS-0817971. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection28">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746945">I-27. Exploring unassigned peaks in protein
fragment mass spectra with frequent itemset mining techniques</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vu T-N<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Vu T-N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Valkenborg D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Valkenborg D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(3), Eeckhout D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Eeckhout D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), De Jaeger G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Jaeger G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(2), Goethals B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Goethals B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Witters E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Witters E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(3), Lemière F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Lemière F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(4), Laukens K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Laukens K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1)</p>

<p class="abmotivation">Mass spectra generated in proteomics experiments are rich
sources of information. Often, only a minor fraction of this information is
used for protein identification. Of the remaining peaks, a fraction may be
assigned to common contaminants, but usually a significant fraction of the
peaks remains unexplained. Some of these unassigned masses may represent
interesting fragments with hidden valuable knowledge. We employed frequent
itemset mining techniques to exploit potentially useful patterns. The patterns
may serve as a basis to generate hypotheses that can be validated by a
spectrometrist.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Original data from over 20.000 “Peptide Mass
Fingerprint”(PMF) MALDI TOF measurements of the higher plant "Arabidopsis
thaliana" were compared against different sequence databases including
TAIR9 and SwissProt on a Mascot server, and unassigned peaks where retained.
The corresponding masses, discretized to their lower integer, were considered
as the items, and the spectra in which they occur as transactions. Frequent
itemset mining and error tolerant itemset mining algorithms were applied. The
results are evaluated towards their potential to reveal novel insights.</p>

<h3>Results</h3>

<p class="abresults">Transactions were subjected to the frequent itemset mining
with different support thresholds, and ranked according to support. Similarly,
error tolerant itemset mining algorithm was also applied on the transactions.
This approach is less strict than the previous one because transactions that
contain an "almost" complete item set are now taken into account. In
follow-up experiments, redundant items, that appear in most frequent item sets,
were removed from all transactions prior to error tolerant itemset mining.
Retrieved patterns are evaluated.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This study demonstrates both the feasibility of frequent
itemset mining in mass spectrometry (MS) data analysis and its pitfalls. The
analysis of unassigned peaks in large numbers of PMF spectra reveals several
highly frequent masses. Frequent itemset mining techniques yields a large
number of peaks that frequently co-occur in unassigned peak lists of individual
MS spectra. Prioritization, redundancy removal and assessment of statistical
significance are remaining challenges in MS-based frequent itemset mining. To
evaluate retrieved patterns, we will use existing and new tandem-MS analyses.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Vu Trung Nghia (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy60097 = 'Tr&#117;ngNgh&#105;&#97;.V&#117;' + '&#64;';
 addy60097 = addy60097 + '&#117;&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy60097 + suffix + '\'' + attribs + '>' );
 document.write( addy60097 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy43433 = 'Tr&#117;ngNgh&#105;&#97;.V&#117;' + '&#64;';
 addy43433 = addy43433 + '&#117;&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy43433 + suffix + '\'' + attribs + '>' );
 document.write( addy43433 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy33366 = 'Tr&#117;ngNgh&#105;&#97;.V&#117;' + '&#64;';
 addy33366 = addy33366 + '&#117;&#97;' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy33366 + suffix + '\'' + attribs + '>' );
 document.write( addy33366 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">University of Antwerp, dept. Mathematics &amp;
Computer Science, Antwerp, Belgium</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1)University of Antwerp, dept. Mathematics &amp;
Computer Science, Antwerp, Belgium; (2)VIB - Plant Systems Biology / University
of Ghent, Gent, Belgium; (3)Flemish Institute for Technological Research,
Antwerp, Belgium;(4) University of Antwerp, dept. Chemistry, Antwerp, Belgium</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">IWT SBO Project "Bioframe" and University of
Antwerp Interdisciplinary BOF</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection29">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746946">I-28. Benchmarking a new semantic similarity measure
using reference sets and clustering : evaluation and interpretation for the
discovery of missing information</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Benabderrahmane S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Benabderrahmane S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Smaïl-Tabbone M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Smaïl-Tabbone M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Napoli A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Napoli A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Poch O<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Poch O&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Devignes MD<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Devignes MD&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Calculating similarity between genes is an important task
in post-genomics. Recently, semantic similarity measures were introduced as a
complement to sequence similarity measures. Those measures use GO annotations which
are organized into three hierarchical controlled vocabularies (molecular
function, biological process, cellular component). Moreover, the GO annotation
process is traced by evidence codes. Existing semantic similarity measures do
not take into account all these GO features, and few studies have been
conducted for their evaluation and comparison.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We have defined a semantic similarity measure
(IntelliGO) based on representing genes in a new vector space model and taking
into account the GO terms frequency, their organization in the GO graph, and
the annnotation evidence codes. The comparison with other measures involves two
collections of reference sets of genes (KEGG pathways, PFAM clans) and inter-
and intra-set global similarity measures. The measure was used for functional clustering.
Resulting clusters were compared with reference sets using the F-Score method.
Overlap analysis leads to possible discovery of missing information.</p>

<h3>Results</h3>

<p class="abresults">The IntelliGO semantic similarity measure is the first
described measure that allows differential weighting of annotation evidence
codes. When compared with three other measures, the IntelliGO intra-set
similarity values reflect at best the biological cohesion of refrerence sets of
genes. The inter-set values express stronger discrimination between distinct
sets of genes. This is confirmed by clustering evaluation which is better with IntelliGO
than with other measures. Mismatches between clusters and reference sets are
analyzed and exploited for improving set composition or gene annotation.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The methodology for comparing semantic similarity
measures relies on defining collections of reference sets of genes that share
common annotations. KEGG pathways are relevant for testing biological process
GO annotations, Pfam clans are relevant for testing molecular function GO
annotations. Other sets can be proposed for completing the benchmark. The
method itself relates to well-known concepts in data and cluster analysis. Interestingly
this method has lead to analyze the overlaps between clusters and reference
sets as a mean for discovering missing information.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://intelligo.loria.fr" mce_href="http://intelligo.loria.fr">http://intelligo.loria.fr</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sidahmed Benabderrahmane (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89474 = 'b&#101;n&#97;bds&#105;' + '&#64;';
 addy89474 = addy89474 + 'l&#111;r&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy89474 + suffix + '\'' + attribs + '>' );
 document.write( addy89474 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy11487 = 'b&#101;n&#97;bds&#105;' + '&#64;';
 addy11487 = addy11487 + 'l&#111;r&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy11487 + suffix + '\'' + attribs + '>' );
 document.write( addy11487 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy14924 = 'b&#101;n&#97;bds&#105;' + '&#64;';
 addy14924 = addy14924 + 'l&#111;r&#105;&#97;' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy14924 + suffix + '\'' + attribs + '>' );
 document.write( addy14924 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Loria, Nancy Université</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1): LORIA UMR7503, CNRS-Nancy niversité, Orpailleur
Team, Nancy. (2):IGBMC UMR 7104, CNRS-INSERM-Université Louis Pasteur,
Strasbourg.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the French National
Institute of Cancer (INCa) and by the Region Lorraine program for research and
technology (CPER MISN). Benabderrahmane S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Benabderrahmane S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->. was recipient of an INCa
doctoral fellowship. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection30">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746947">I-29. Utopia:GPCRDB: a domain-specific PDF reader</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Vroling B<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Vroling B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Thorne D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thorne D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), McDermott P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;McDermott P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Pettifer S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Pettifer S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vriend G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vriend G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The amount of data and information present in literature
and databases is growing at an ever-increasing pace. There is so much
information available that we no longer know what information exists and
finding what we would like to know has become an increasingly difficult task.
The knowledge we seek is often fragmentary and disconnected, spread thinly
across a vast sea of literature and databases.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">As a step towards a solution for this enormous
challenge, we present the domain-specific PDF reader Utopia:GPCRDB. This reader
dynamically links concepts in scientific articles to relevant entities in the
GPCRDB such as proteins and mutations. The GPCRDB is a molecular class-specific
information system that contains a large amount of heterogeneous data on G
protein-coupled receptors (GPCRs) and functions as a one-stop resource from
GPCR researchers.</p>

<h3>Results</h3>

<p class="abresults">With Utopia:GPCRDB it is now possible to retrieve the
knowledge stored in the GPCRDB and integrate it with the article you are
currently reading, all with the single click of a button.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This integration of scientific articles with a
domain-specific database greatly enhances the process of turning data into
knowledge and offers new ways of thinking about how to deal with information
stored in literature and databases.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Bas Vroling (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy51889 = 'bvr&#111;l&#105;ng' + '&#64;';
 addy51889 = addy51889 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy51889 + suffix + '\'' + attribs + '>' );
 document.write( addy51889 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy72636 = 'bvr&#111;l&#105;ng' + '&#64;';
 addy72636 = addy72636 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy72636 + suffix + '\'' + attribs + '>' );
 document.write( addy72636 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy3509 = 'bvr&#111;l&#105;ng' + '&#64;';
 addy3509 = addy3509 + 'cmb&#105;' + '&#46;' + 'r&#117;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy3509 + suffix + '\'' + attribs + '>' );
 document.write( addy3509 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">CMBI, NCMLS, Radboud University Medical Centre</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 CMBI, NCMLS, Radboud University Nijmegen Medical
Centre, Nijmegen, The Netherlands 2 School of Computer Science, University of
Manchester, Manchester, M13 9PL, UK</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection31">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746948">I-30. Locally optimal structures of an RNA</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Saffarian A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Saffarian A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Giraud M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Giraud M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Touzet H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Touzet H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">The folding space of an RNA provides a rich knowledge
about the structures and functions of the RNA molecule. Partition functions or
sample of optimal and suboptimal structures give some insight on this folding
space. An alternative description of this space can also be achieved by
studying so-called locally optimal secondary structures. They are structures
maximal for inclusion: no base pair can be added without creating a pseudoknot
or a base triplet (Clote, 2005). As far as we know, there were no tool to
construct all such structures.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We defined an algorithm that computes all locally
optimal secondary structures of an RNA sequence. Our algorithm works with any
folding model, including those that take into account adjacent nucleotide
interactions. The input is a sequence together with a set of putative helices. As
the number of locally optimal secondary structures could be exponential, we
show that the problem divides in two substeps. First, we apply the
juxtaposition operations, by dynamic programming, then we build on-the-fly the
full structures using the nesting operation.</p>

<h3>Results</h3>

<p class="abresults">The algorithm is implemented in a freely available software
called REGLISS. We compared REGLISS to MFOLD and showed that REGLISS better
describes the folding landscape, finding some structures missed by MFOLD. Our
results also illustrate that for some families, the structural RNA has a
different folding landscape than random sequences with the same dinucleotide
frequency. Finally, we successfully compared our approach to RNALOSS (Clote,
2005). In RNALOSS, the notion of locally optimal structure is reduced to the
Nussinov-Jacobson model.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our work shows that all the locally optimal secondary
structures of a given RNA can effectively be computed. These structures can
also be filtered out using some post-processing criterium such as the free
energy or the shape of the structure. This is a fruitful alternative to MFOLD
suboptimal structures. Another advantage of the method is that the user can
provide its own set of helices, based on the thermodynamic Nearest Neighbor
model or on any other model.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://bioinfo.lifl.fr/RNA/regliss/index.php" mce_href="http://bioinfo.lifl.fr/RNA/regliss/index.php">http://bioinfo.lifl.fr/RNA/regliss/index.php</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Azadeh Saffarian (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy96786 = '&#97;z&#97;d&#101;h.s&#97;ff&#97;r&#105;&#97;n' + '&#64;';
 addy96786 = addy96786 + 'l&#105;fl' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy96786 + suffix + '\'' + attribs + '>' );
 document.write( addy96786 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74231 = '&#97;z&#97;d&#101;h.s&#97;ff&#97;r&#105;&#97;n' + '&#64;';
 addy74231 = addy74231 + 'l&#105;fl' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy74231 + suffix + '\'' + attribs + '>' );
 document.write( addy74231 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy16155 = '&#97;z&#97;d&#101;h.s&#97;ff&#97;r&#105;&#97;n' + '&#64;';
 addy16155 = addy16155 + 'l&#105;fl' + '&#46;' + 'fr';
 document.write( '<a ' + path + '\'' + prefix + addy16155 + suffix + '\'' + attribs + '>' );
 document.write( addy16155 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Laboratoire d'Informatique Fondamentale de Lille
(LIFL), Lille 1 University, France</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Laboratoire d'Informatique Fondamentale de Lille
(LIFL), UMR CNRS 8022, Lille 1 University, France 2. INRIA Lille Nord-Europe,
France</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection32">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746949">I-31. Anesthetics and the role of the biological membrane
in the molecular regulation of K2P potassium channels</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Bernardi RC<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Bernardi RC&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Treptow W<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Treptow W&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Klein ML<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Klein ML&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2)</p>

<p class="abmotivation">Detailing the action mechanism of anesthetics have been
the purpose of several studies, since this is not well established. TREK-1, a
member of the K2P family, was identified as one key molecular component
required for anesthetic effects. The recent atomistic description of the
membrane-bound state of TREK-1 (Treptow &amp; Klein, JACS 2010, in press)
provide us with an unique opportunity to decipher the anesthetic action on
membrane bound species. We have investigated the structure of K2P channels in a
membrane environment using MD simulation.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Two distinct fully atomistic models for the most
studied K2P channel, namely, the TWIK-related (TREK)-1 channel have been built.
These constructs were then inserted into a fully hydrated zwitterionic lipid
bilayer, and each relaxed by means of MD simulations spanning <span style="font-family: &quot;Cambria Math&quot;,&quot;serif&quot;;" mce_style="font-family: &quot;Cambria Math&quot;,&quot;serif&quot;;">∼</span>0.3
μs. Further multi-ns simulations of the channel in presence of isoflurane were
also carried out. The simulations were performed in the NPT ensemble using the
program NAMD2. The simulations used the CHARMM22-CMAP force field with
torsional cross-terms for the protein lipids.</p>

<h3>Results</h3>

<p class="abresults">The MD study reveals a direct coupling of the C-terminus to
the intracellular membrane surface. As a domain structure physically linked to
the channel's pore and energetically coupled to the bilayer, the C-terminus
appears to form a robust device able to gate the channel in response to
membrane stimulation. In presence of isoflurane, the C-terminal coupling onto
the membrane increases with anesthetic partition into the bilayer and we
observe the partial opening of the channel’s pore after membrane swelling.</p>

<h3>Discussion</h3>

<p class="abdiscussion">An atomistic structural description for TREK-1 is
provided, which rationalizes the plethora of experimental findings on this
channel. We point to an anesthetic action mechanism in which the agonist
affects indirectly the channel’s pore gating through modulation of the
C-terminal membrane coupling. Given this, further structural and experimental
studies aimed at the investigation of the dynamical processes underlying the
anesthetic induced protein activation are within reach and should contribute to
the deciphering of the key molecular mechanisms underlying the function of K2P
channels.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Rafael C. Bernardi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18883 = 'b&#101;rn&#97;rd&#105;' + '&#64;';
 addy18883 = addy18883 + 'b&#105;&#111;f' + '&#46;' + '&#117;frj' + '&#46;' + 'br';
 document.write( '<a ' + path + '\'' + prefix + addy18883 + suffix + '\'' + attribs + '>' );
 document.write( addy18883 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31556 = 'b&#101;rn&#97;rd&#105;' + '&#64;';
 addy31556 = addy31556 + 'b&#105;&#111;f' + '&#46;' + '&#117;frj' + '&#46;' + 'br';
 document.write( '<a ' + path + '\'' + prefix + addy31556 + suffix + '\'' + attribs + '>' );
 document.write( addy31556 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy35208 = 'b&#101;rn&#97;rd&#105;' + '&#64;';
 addy35208 = addy35208 + 'b&#105;&#111;f' + '&#46;' + '&#117;frj' + '&#46;' + 'br';
 document.write( '<a ' + path + '\'' + prefix + addy35208 + suffix + '\'' + attribs + '>' );
 document.write( addy35208 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Universidade Federal do Rio de Janeiro</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Laboratório de Biologia Teórica e Computacional,
Universidade de Brasilia – Brasília, DF, Brazil. (2) Institute for
Computational Molecular Science, College of Science and Technology, Temple
University - Philadelphia, PA, USA.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">We thanks the Brazilian support agencies: CNPq, CAPES and
FAPERJ. Our research support came in part from the National Institutes of
Health and the National Science Foundation through TeraGrid resources. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection33">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746950">I-32. Comparison of validation methods for merging
cancer microarray data sets</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Taminau J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Taminau J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Meganck S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Meganck S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2), Weiss-Solis DY<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Weiss-Solis DY&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3,4), Van Staveren WCG<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Staveren WCG&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Dom G<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Dom G&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Venet D<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Venet D&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Bersini H<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bersini H&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Detours V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Detours V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Nowé A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Nowé A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">There are ~75,000 publicly available gene expression
profiles assayed on the human Affymetrix platforms and their reproducibility
and validity are increasingly accepted. Whereas the activity of all 20K genes
can be measured, there are much fewer samples per-study, limiting statistical
power. A solution is to combine multiple studies into larger meta-studies, but
no existing methods remove the technical biases in general, eg, different
experimental and bioinformatics protocols. Here, we seek a general method by
benchmarking existing methods applied to cases of increasing biological
complexity.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We compare five merging techniques: RAW (do
nothing) NORM (gene standardization), BMC (batch mean centering), DWD (Distance
Weighted Discrimination) and XPN (Cross Platform Normalization). We examine
their results on three test cases of increasing biological complexity. NCI60: a
collection of cell lines of nine different types of cancer. THYROID: two
publicly available and two in-house thyroid data sets. BREAST: a collection of
eight data sets, which is the most complicated case since breast cancer is
known as being a very heterogeneous disease.</p>

<h3>Results</h3>

<p class="abresults">MDS was used to inspect the clustering with respect to
samples’ study of origin, technical replicates and biological type. In the
NCI60 and THYROID cases the study-bias could not be removed with the simplest
methods in contrast to the BREAST case. DWD and XPN showed that samples
belonging to the same biological class were clustered together. Comparison of
average expressions of housekeeping genes was consistent with the MDS plots. We
analyzed cross-study classification accuracy. For this validation NORM and BMC
performed best, which is in contradiction with the previous validation
outcomes.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Most merging techniques have previously only been
validated in one specific setting and on specific data. The added value of our
work is the validation on three cases of increasing biological complexity. Our
results show that no single algorithm exists that outperforms the others in all
tasks. Depending on the desired result it is often possible to select some
validation criteria that works best for that specific technique. This opens the
perspective of the development of an objective method that can be applied
generically for merging microarray data.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://como.vub.ac.be/%7Ejtaminau/ECCB2010/" mce_href="http://como.vub.ac.be/~jtaminau/ECCB2010/">http://como.vub.ac.be/~jtaminau/ECCB2010/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jonatan Taminau (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy77509 = 'jt&#97;m&#105;n&#97;&#117;' + '&#64;';
 addy77509 = addy77509 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy77509 + suffix + '\'' + attribs + '>' );
 document.write( addy77509 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy87225 = 'jt&#97;m&#105;n&#97;&#117;' + '&#64;';
 addy87225 = addy87225 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy87225 + suffix + '\'' + attribs + '>' );
 document.write( addy87225 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97292 = 'jt&#97;m&#105;n&#97;&#117;' + '&#64;';
 addy97292 = addy97292 + 'v&#117;b' + '&#46;' + '&#97;c' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy97292 + suffix + '\'' + attribs + '>' );
 document.write( addy97292 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">VUB</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) CoMo, Vrije Universiteit Brussel, Pleinlaan 2, 1050
Brussels, Belgium. (2) ETRO, Vrije Universiteit Brussel, Pleinlaan 2, 1050
Brussels, Belgium. (3) IRIDIA, Université Libre de Bruxelles, Avenue F. D.
Roosevelt 50, 1050 Brussels, Belgium. (4) IRIBHM, Université Libre de
Bruxelles, Route de Lennik 808, 1070 Brussels, Belgium.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This research is partially funded by the Institute for
the encouragement of Scientiﬁc Research and Innovation of Brussels (IRSIB) </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection34">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746951">I-33. Mapping insertions to putative target genes</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">De Jong J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;De Jong J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), De Ridder J (1,2), Sun N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sun N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Van Uitert M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Van Uitert M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Adams DJ<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Adams DJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Wessels L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wessels L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->FA<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wessels LFA&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">Large-scale retrovirus-based, and more recently also
transposon-based, insertional mutagenesis screens are of great use in the
identification of new cancer genes and in gene therapy. However useful, there
are some limitations. One such limitation lies in the difficulties in
predicting which genes in the surrounding DNA are affected by insertions, since
the range across which insertions influence target genes is unknown. Here, a
knowledge-based approach is presented for mapping insertions to such putative
target genes.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Two sets of same-sample insertion-expression
datasets were used: 1) ~2000 MuLV insertions and expression data for 97 tumors;
2) ~10000 Sleeping Beauty insertions and expression data from 138 tumors. For
the MuLV dataset, we compared an existing rule-based approach to mapping each
insertion to the nearest gene. Then we investigated the effect of insertions on
putative target genes. The range of the influence of insertions was estimated
by the association of the insertion and the expression data for increasing
window sizes around target genes, for both sense and antisense insertions.</p>

<h3>Results</h3>

<p class="abresults">For MuLV, the existing rule-based approach produced superior
association of insertions with gene expression than the nearest gene approach.
For varying window sizes, the largest effect was seen for upstream-antisense
and downstream-sense insertions. For SB, this was for sense insertions,
although generally the effect was less pronounced. Furthermore, the effect was
more local, and for single insertions per gene, there seemed to be a depletion
of upregulated genes for ~20kb to ~200mb windows. For both systems, requiring
multiple insertions per gene led to a substantial increase in effect.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We believe that automated insertion mapping can be a useful
complement to CIS analysis, which generally disregards individual properties of
insertions. Some limitations of the analysis include the assumption that
insertions have a positive effect on gene expression. Furthermore, the
association measure and distance-based approach detect mostly direct
insertion-expression associations. Ideally, in the future, more insertional
mutagenesis systems will be included, and also other data types, such as
chromatin lamina association. This will result in more accurate insertion to
gene mappings.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Johann de Jong (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy18983 = 'j.d.j&#111;ng' + '&#64;';
 addy18983 = addy18983 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy18983 + suffix + '\'' + attribs + '>' );
 document.write( addy18983 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy90421 = 'j.d.j&#111;ng' + '&#64;';
 addy90421 = addy90421 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy90421 + suffix + '\'' + attribs + '>' );
 document.write( addy90421 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy92239 = 'j.d.j&#111;ng' + '&#64;';
 addy92239 = addy92239 + 'nk&#105;' + '&#46;' + 'nl';
 document.write( '<a ' + path + '\'' + prefix + addy92239 + suffix + '\'' + attribs + '>' );
 document.write( addy92239 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The Netherlands Cancer Institute</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Bioinformatics and Statistics, The Netherlands
Cancer Institute, Plesmanlaan 121, 1066CX Amsterdam, The Netherlands (2) Delft
Bioinformatics Lab, Faculty of EEMCS, TU Delft, Delft, The Netherlands (3)
Experimental Cancer Genetics, Wellcome Trust Sanger Institute, Hinxton, United
Kingdom. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection35">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746952">I-34. The zebrafish embryo in toxicology: a
combination of toxicogenomics and other screening techniques</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Legradi J<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Legradi J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Yang L<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Yang L&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Alshut R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alshut R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Ho N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ho N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1), Klüver N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Klüver N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Scholz S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Scholz S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Mikut R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mikut R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Reischl M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Reischl M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Liebel U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Liebel U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Strähle U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Strähle U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">In our work, we aim at obtaining further insights into
the mode of action of toxicants. Our new system should allow us to study
genotypic and phenotypic toxicological effects chemicals have. With our
approach we hope to get a better understanding of how chemicals act in whole
organisms, especially during the embryonic development.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We used the commercially available Agilent
zebrafish arrays and the Agilent eArray system for designing custom expression
arrays. Gene expression studies where performed using 10 toxicants, covering
several different mode of actions. The arrays where analysed using our Matlab
platform Gait-CAD and MeV. Publicly available databases like GO, KEGG, CTD were
used for further analysis of obtained gene lists. As high throughput imaging
platform, we utilized the standard Olympus Scan R system and several in-house
developed microscope systems.</p>

<h3>Results</h3>

<p class="abresults">30 Agilent microarray experiments were performed for finding
genes acting like biosensors. To better understand the gene regulation, we
further developed a microarray system based on transcription factors. In the
second part, we developed an automated high throughput microscope screening
platform for zebrafish. This enabled us to study concentration and time
depended toxicological effects of our compounds.</p>

<h3>Discussion</h3>

<p class="abdiscussion">We studied the different mode of actions of 10 Chemicals
via a microarray-based toxicogenomics in combination with a phenotype-based
microscope approach. As zebrafish presents a good model organism for studying
toxicological effects, we think that our approach is suitable for obtaining a
better understanding of the different mode of actions of toxicants.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Jessica Legradi (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy29539 = 'j&#101;ss&#105;c&#97;.l&#101;gr&#97;d&#105;' + '&#64;';
 addy29539 = addy29539 + 'k&#105;t' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy29539 + suffix + '\'' + attribs + '>' );
 document.write( addy29539 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy97836 = 'j&#101;ss&#105;c&#97;.l&#101;gr&#97;d&#105;' + '&#64;';
 addy97836 = addy97836 + 'k&#105;t' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy97836 + suffix + '\'' + attribs + '>' );
 document.write( addy97836 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy26818 = 'j&#101;ss&#105;c&#97;.l&#101;gr&#97;d&#105;' + '&#64;';
 addy26818 = addy26818 + 'k&#105;t' + '&#46;' + '&#101;d&#117;';
 document.write( '<a ' + path + '\'' + prefix + addy26818 + suffix + '\'' + attribs + '>' );
 document.write( addy26818 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Strähle Group, Institute of Toxicology and Genetics,
Karlsruhe Institute of Technology</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1 ITG - Institute of Toxicology and Genetics, Karlsruhe
Institute of Technology 2 IAI - Institute of Applied Informatics, Karlsruhe
Institute of Technology 3 Helmholtz-Zentrum für Umweltforschung UFZ, Leipzig</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection36">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746953">I-35. Disease gene prediction based on
tissue-specific conserved coexpression</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Piro RM<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Piro RM&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->(1,*), Ala U<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Ala U&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Molineris I<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Molineris I&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Grassi E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Grassi E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Damasco C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Damasco C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Bracco C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Bracco C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Provero P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Provero P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Di Cunto F<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Di Cunto F&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Even with the success of next generation sequencing
techniques, the identification of genes involved in human hereditary disease
remains a demanding task that can be significantly aided by computational
predictions. We discuss a method for disease gene prediction based on
high-throughput microarray expression data that uses the conservation of
coexpression as a powerful filter for biological significance and allows to
specifically focus on tissue-specific relationships between disease and candidate
genes, providing novel high-confidence candidates for several genetic
disorders.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We build both multi-tissue and tissue-specific
conserved coexpression networks (CCNs) and exploit the gene coexpression
clusters they provide to uncover global and tissue-specific relationships
between candidate genes and “reference genes” known to be involved in the given
phenotype or similar disorders. Through an explicit integration of phenomics,
in particular the concept of phenotype similarity, we can apply our method also
to disease phenotypes with so far unknown molecular basis.</p>

<h3>Results</h3>

<p class="abresults">We show that the novelty of our approach, the
tissue-specificity, provides disease-related information that is highly
complementary to the information obtained from multi-tissue coexpression. This
leads to biologically meaningful predictions that could not be derived from
multi-tissue expression profiles. We discuss the obtained results and present a
user-friendly web tool for custom analysis.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The conservation of coexpression between human and mouse
allows to focus on biologically relevant relationships that have been preserved
during evolution. The analysis of tissue-specific coexpression profiles takes
into account the dynamic nature of such relationships that may vary over
different tissues, cell types and/or conditions. The integration of both
concepts, the tissue-specific conserved coexpression, can provide important
clues about molecular mechanisms underlying human hereditary disorders and thus
provide a powerful tool for disease gene prediction.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.cbu.mbcunito.it/ts-coexp/" mce_href="http://www.cbu.mbcunito.it/ts-coexp/">http://www.cbu.mbcunito.it/ts-coexp/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Rosario M. Piro (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy698 = 'r&#111;s&#97;r&#105;&#111;.p&#105;r&#111;' + '&#64;';
 addy698 = addy698 + '&#117;n&#105;t&#111;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy698 + suffix + '\'' + attribs + '>' );
 document.write( addy698 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy66968 = 'r&#111;s&#97;r&#105;&#111;.p&#105;r&#111;' + '&#64;';
 addy66968 = addy66968 + '&#117;n&#105;t&#111;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy66968 + suffix + '\'' + attribs + '>' );
 document.write( addy66968 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy67732 = 'r&#111;s&#97;r&#105;&#111;.p&#105;r&#111;' + '&#64;';
 addy67732 = addy67732 + '&#117;n&#105;t&#111;' + '&#46;' + '&#105;t';
 document.write( '<a ' + path + '\'' + prefix + addy67732 + suffix + '\'' + attribs + '>' );
 document.write( addy67732 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Molecular Biotechnology Center, University of Torino,
Italy</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Molecular Biotechnology Center, University of
Torino, Italy. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection37">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746954">I-36. Improved classification by integrating
multiple patient data sets with literature information using co-inertia analysis</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Thomas M<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Thomas M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Daemen A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Daemen A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), De Moor B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;De Moor B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">Previous studies on tumor classification have shown that
classification can be improved by fusing microarray data sets. In our study, we
will proceed one step further by incorporating prior literature knowledge
specific to each data set gathered from patients.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Method Co-inertia Analysis: a multivariate method
used in combination with principal component analysis to couple two or more
data sets. Datasets Patient data sets: Micro array data sets of DNA and RNA of
prostate cancer patients. Document data sets: Each entries on the matrix,
corresponds to number of PUBMED abstract in which a gene and cancer related
term co-occurs.</p>

<h3>Results</h3>

<p class="abresults">a) Principal component analysis of the Patients data sets
gives the classification error as 0.4033 b) Co-inertia analysis of patient data
sets and document data sets gives the classification error as 0.3578</p>

<h3>Discussion</h3>

<p class="abdiscussion">This study shows that classification accuracy can be
improved with addition of literature information as prior to microarray data
sets. Further research is being undertaken to investigate the effect of
integration of additional text information from PUBMED as prior to microarray
and clinical data sets.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Thomas M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Thomas M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->inta (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy64897 = 'm&#105;nt&#97;82' + '&#64;';
 addy64897 = addy64897 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy64897 + suffix + '\'' + attribs + '>' );
 document.write( addy64897 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy49221 = 'm&#105;nt&#97;82' + '&#64;';
 addy49221 = addy49221 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy49221 + suffix + '\'' + attribs + '>' );
 document.write( addy49221 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy31177 = 'm&#105;nt&#97;82' + '&#64;';
 addy31177 = addy31177 + 'gm&#97;&#105;l' + '&#46;' + 'c&#111;m';
 document.write( '<a ' + path + '\'' + prefix + addy31177 + suffix + '\'' + attribs + '>' );
 document.write( addy31177 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">ESAT, K U Leuven</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. ESAT, KU Leuven,Belgium</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection38">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746955">I-37. Time-resolved signatures from weighted
visibility graph representation of time-series data</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Nikoloski Z<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Nikoloski Z&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2,*), Grimbs S<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Grimbs S&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Schäfer R<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Schäfer R&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Sers C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Sers C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Selbig J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Selbig J&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,2)</p>

<p class="abmotivation">High-throughput technologies could be employed to
generate a scope of data necessary for understanding the complexities of living
organisms, particularly if the data capture the kinetic resolution of
investigated processes. Despite the decreasing costs of the omics technologies,
experimental studies are bound to yield relatively short time series due to the
inherent problems of gathering sufficient sample material and designing
complex, large-scale perturbation experiments. Consequently, there is a
pressing need for development of novel methods for analysis of short
time-series data.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">We provide a representation of time series via
directed weighted visibility graphs. The proposed representation proves to be an
effective lossless approach that both captures the important properties of
short time series and is invariant to vertical rescaling and translation. Our
contributions include: (1) definition of a polynomially computable distance
measure for time series represented by weighted visibility graphs, (2) proof
that the proposed distance measure is a generalized metric, and (3) algorithms
for finding time-resolved representatives in large-scale experiments based on
the proposed measure.</p>

<h3>Results</h3>

<p class="abresults">We demonstrate how the proposed representation and measure
can be employed in statistical tests that account for the dependencies between
not only the time points but also the entities (i.e., genes, proteins,
metabolites) from which time-series data were obtained. In addition, we devise
algorithms by which we demonstrate that our methods can be used to efficiently
determine time-resolved signatures on a gene expression data set from
colorectal cancer cell lines.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Our weighted visibility graph representation and the
proposed generalized metric are particularly suitable for statistical testing which
takes into account the dependencies between not only the time points but also
the biological entities considered in the study. We demonstrated that, on a
transcriptomics data set from colorectal cancer cell lines, our approach finds
ontologically related gene profiles whose temporal behaviour is captured by the
corresponding cluster representatives. The latter can serve as a time-resolved
label which could be employed for classification of new data. In our view, this
is a first necessary step towards statistically sound discovery of temporal
biomarkers.</p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Sergio Grimbs (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy41235 = 'gr&#105;mbs' + '&#64;';
 addy41235 = addy41235 + 'mp&#105;mp-g&#111;lm' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy41235 + suffix + '\'' + attribs + '>' );
 document.write( addy41235 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy9317 = 'gr&#105;mbs' + '&#64;';
 addy9317 = addy9317 + 'mp&#105;mp-g&#111;lm' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy9317 + suffix + '\'' + attribs + '>' );
 document.write( addy9317 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74610 = 'gr&#105;mbs' + '&#64;';
 addy74610 = addy74610 + 'mp&#105;mp-g&#111;lm' + '&#46;' + 'mpg' + '&#46;' + 'd&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy74610 + suffix + '\'' + attribs + '>' );
 document.write( addy74610 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Biochemistry and Biology, University Of
Potsdam, Potsdam, Germany</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Max-Planck Institute of Molecular Plant Physiology,
Potsdam, Germany (2) Institute of Biochemistry and Biology, University of
Potsdam, Potsdam, Germany (3) Laboratory of Molecular Tumorpathology, Institute
of Pathology, University Medicine Charité, Berlin, Germany</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">ZN and JS are supported by the GoFORSYS project funded
by the Federal Ministry of Education and Research, Grant no. 0313924. SG, RS,
CS and JS are supported by the ColoNET project funded by the Federal Ministry
of Education and Research, Grant no. 0215417F. ZN is supported by the
Max-Planck Society. CS and RS are supported by IBB (PROFIT). </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection39">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746956">I-38. Genome sequence of the edible cyanobacterium
Arthrospira sp. PCC 8005</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Janssen P<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Janssen P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->J<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Janssen PJ&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Morin N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Morin N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Monsieurs P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Monsieurs P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1,*), Mergeay M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Mergeay M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1), Leroy B<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Leroy B&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Wattiez<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wattiez&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(2), Vallaeys T<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Vallaeys T&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(3), Waleron K<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Waleron K&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Waleron M<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Waleron M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(4), Wilmotte A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Wilmotte A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(5), Quillardet P<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Quillardet P&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Tandeau de Marsac N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Tandeau de Marsac N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(6), Talla E<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Talla E&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7), Zhang C-C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Zhang C-C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(7), Médigue C<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Médigue C&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(8), Barbe V<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Barbe V&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(9), N. Leys (1).</p>

<p class="abmotivation">Arthrospira are nonheterocystous filamentous
cyanobacteria that typically reside in alkaline lakes. They have a high protein
content, are rich in carbohydrates and essential fatty acids, and produce a
variety of minerals, vitamins, and nutritional pigments such as beta-carotene.
Unsurprisingly, they have a long history of human consumption and are used
worldwide as industrial feed. PCC 8005 strain was selected by the European
Space Agency (ESA) as an oxygen producer and as a nutritional endproduct of the
life support system MELiSSA (Micro-Ecological Life Support System Alternative).</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Whole-genome shotgun sequencing of strain PCC 8005
was performed using 454 pyrosequencing technology (amounting to 400,000 reads)
supplemented with Sanger sequencing (up to 96,000 longer reads), leading to a
final assembly in 16 scaffolds. The PCC 8005 genome is most likely organised in
a single replicon without current evidence for plasmids. These scaffolds were
processed by the MaGe annotation platform. Final assembly was facilitated by
optical genome mapping and long-PCR. Genome comparison with A. platensis
NIES-39 and A. maxima CS-328 were performed using the GeneRage clustering
algorithm.</p>

<h3>Results</h3>

<p class="abresults">The genome (6,279,260 bases) is highly repetitive, with &gt;
300 kb present as tandem sequences, and contains four CRISPR elements, at least
140 complete IS elements, and eight copies of a putative genomic island. Genes
for the production of beta-carotene and two essential fatty acids linoleic acid
and gamma-linolenic acid were identified. Genome data confirm the inability of
PCC 8005 to fix nitrogen as it lacks essential nif genes and indicate that
nitrogen metabolism in PCC 8005 follows classic routes utilizing nitrate,
nitrite, urea, and ammonium, with NtcA as the global nitrogen regulator.</p>

<h3>Discussion</h3>

<p class="abdiscussion">This draft genome sequence presents 119 contigs rendering 16 scaffolds by assembly. The MaGe annotation system could assign 63%
of the CDSs to one or more functional COG groups and currently reports 1,704
conserved hypothetical and 884 hypothetical proteins. Although gap closure and
final assembly are very cumbersome due to extended regions of highly repetitive
DNA, we were able to place nine scaffolds on a single optical genome map and we
soon hope to fully close the genome by multiplex long-PCR and adaptor-assisted
genome walking. GenBank submission ADDH00000000. MaGe data available July 2010.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="https://www.genoscope.cns.fr/agc/microscope/about/collabprojects.php?P_id=49" mce_href="https://www.genoscope.cns.fr/agc/microscope/about/collabprojects.php?P_id=49">https://www.genoscope.cns.fr/agc/microscope/about/collabprojects.php?P_id=49</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Peter Monsieurs (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy74601 = 'pm&#111;ns&#105;&#101;&#117;rs' + '&#64;';
 addy74601 = addy74601 + 'sckc&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy74601 + suffix + '\'' + attribs + '>' );
 document.write( addy74601 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy98791 = 'pm&#111;ns&#105;&#101;&#117;rs' + '&#64;';
 addy98791 = addy98791 + 'sckc&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy98791 + suffix + '\'' + attribs + '>' );
 document.write( addy98791 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy86097 = 'pm&#111;ns&#105;&#101;&#117;rs' + '&#64;';
 addy86097 = addy86097 + 'sckc&#101;n' + '&#46;' + 'b&#101;';
 document.write( '<a ' + path + '\'' + prefix + addy86097 + suffix + '\'' + attribs + '>' );
 document.write( addy86097 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Belgian Nuclear Research Center</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">(1) Molecular and Cellular Biology - Unit of
Microbiology, Institute for Environment, Health and Safety , Belgian Nuclear
Research Centre SCK•CEN, B-2400 Mol, Belgium; (2) Department of Proteomics and
Protein Biochemistry, Université de Mons-Hainaut, B-7000 Mons, Belgium; (3)
Laboratoire ECOLAG, UMR CNRS-UMII 5119, Université Montpellier II, cc 093,
Place E. Bataillon, F-34095 Montpellier Cedex 5, France; (4) Intercollegiate
Faculty of Biotechnology, University of Gdańsk &amp; Medical University of
Gdańsk, Kładki 24, Gdańsk, Poland; (5) Centre for Protein Engineering,
Institute of Chemistry B6, Sart Tilman, University of Liège, B-4000 Liège, Belgium;
(6) Unité des Cyanobactéries, CNRS-URA2172, Institut Pasteur, F-75015, Paris,
France; (7) Laboratoire de Chimie Bactérienne, CNRS-UPR9043, Université
d'Aix-Marseille, 31 chemin Joseph A<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Joseph A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]-->iguier, F-13402 Marseille
cedex 20, France; (8) Laboratoire d'Analyses Bioinformatiques pour la Génomique
et le Métabolisme (LABGeM), Centre National de la Recherche Scientifique -
UMR8030 et CEA/DSV/IG/Genoscope, F-91057 Evry, France; (9) Génoscope, Centre
national de séquençage, CEA/DSV/IG/Genoscope, 2 rue Gaston Cremieux, F-91057
Evry Cedex, France.</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">This work was supported by the European Space Agency
ESA/ESTEC under MELGEN-2 contract No. 20397/06/NL/SFe in frame of the MELiSSA
project. </p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: always;" mce_style="page-break-before:
always;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection40">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746957">I-39. Moa: managing command line bioinformatics</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Fiers M<!--[if supportFields]><span style="mso-element:field-begin" mce_style="mso-element:field-begin"></span>
XE &quot;Fiers M&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(*)</p>

<p class="abmotivation">Bioinformatics projects often consist of many separate,
mutually dependent steps. A range of advanced software tools exists to organise,
maintain and automate such projects. However, in many cases bioinformaticians
use the most basic approach possible: the command line and adhoc scripts. This
approach is often the most flexible and rapid route to results. A downside to
this method is the substantial effort necessary to prevent a project from
becoming disorganised and impractical to maintain.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">Many reliable tools are available that can assist
in the organization of a command line project. Rather than re-implementing
functionality, Moa incorporates established software into a single integrated tool.
To this end Moa has an extensible architecture, with most functionality
implemented as a plugin. At the core Moa uses Gnu Make to describe, schedule
and execute jobs.</p>

<h3>Results</h3>

<p class="abresults">Moa provides a framework to: create, use and share building
blocks; build, maintain and execute analysis pipelines; track pipeline history;
generate documentation; and provide easy access for non-bioinformaticians. A
command line interface provides uniform access to all Moa functionality.</p>

<h3>Discussion</h3>

<p class="abdiscussion">Moa aims to be a compromise between large automated
frameworks (such as Galaxy) and adhoc scripting pipelines. In comparison to the
larger frameworks Moa is less user friendly. This is offset by Moa's greater flexibility.
Adhoc scripting does allow flexibility, but requires boilerplate code and does
not enforce any structure. Moa offers the best of both worlds, to both
bioinformaticians and capable non-bioinformaticians.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://mfiers.github.com/Moa/" mce_href="http://mfiers.github.com/Moa/">http://mfiers.github.com/Moa/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Mark W.E.J. Fiers (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy89525 = 'm&#97;rk.f&#105;&#101;rs' + '&#64;';
 addy89525 = addy89525 + 'pl&#97;nt&#97;ndf&#111;&#111;d' + '&#46;' + 'c&#111;' + '&#46;' + 'nz';
 document.write( '<a ' + path + '\'' + prefix + addy89525 + suffix + '\'' + attribs + '>' );
 document.write( addy89525 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy50680 = 'm&#97;rk.f&#105;&#101;rs' + '&#64;';
 addy50680 = addy50680 + 'pl&#97;nt&#97;ndf&#111;&#111;d' + '&#46;' + 'c&#111;' + '&#46;' + 'nz';
 document.write( '<a ' + path + '\'' + prefix + addy50680 + suffix + '\'' + attribs + '>' );
 document.write( addy50680 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy58732 = 'm&#97;rk.f&#105;&#101;rs' + '&#64;';
 addy58732 = addy58732 + 'pl&#97;nt&#97;ndf&#111;&#111;d' + '&#46;' + 'c&#111;' + '&#46;' + 'nz';
 document.write( '<a ' + path + '\'' + prefix + addy58732 + suffix + '\'' + attribs + '>' );
 document.write( addy58732 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">The New Zealand Institute for Plant &amp; Food
Research Limited</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">The New Zealand Institute for Plant &amp; Food Research
Limited, Private Bag 4704, Christchurch 8140</p>

</div><p>

<span style="font-size: 9pt; line-height: 115%;" mce_style="font-size:9.0pt;line-height:115%;font-family:" times="" new="" roman="" ,="" serif="" ;="" mso-fareast-font-family:calibri;color:black;mso-ansi-language:en-us;mso-fareast-language:="" en-us;mso-bidi-language:ar-sa;mso-no-proof:yes=""><br style="page-break-before: auto;" mce_style="page-break-before:
auto;mso-break-type:section-break" clear="all">
</span></p>

<div class="WordSection41">

<div style="border-width: 2.25pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 2.25pt;
padding:1.0pt 0in 0in 0in;background:#C6D9F1">

<h2><a class="mceItemAnchor" name="_Toc269746958">I-40. SAAPdb: structural effects of single amino
acid polymorphisms</a><br mce_bogus="1"></h2>

</div>

<p class="abauthors">Baresic A<!--[if supportFields]><span style="mso-element:
field-begin" mce_style="mso-element:
field-begin"></span> XE &quot;Baresic A&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Alnumair N<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Alnumair N&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1, *), Martin ACR<!--[if supportFields]><span
style='mso-element:field-begin'></span> XE &quot;Martin ACR&quot; <![endif]--><!--[if supportFields]><span
style='mso-element:field-end'></span><![endif]--><span style="">&nbsp;</span>(1)</p>

<p class="abmotivation">The SAAPdb database was created with two main goals in
mind: to collect and unify data on neutral and pathogenic missense mutations;
and automatically to process them in terms of their structural effects. This is
supposed to fill the gap between mutation annotations at the sequence level,
and their pathogenicity/phenotype annotation. This leads to enhanced
understanding of protein evolution and will contribute to pharmacogenomics.</p>

<h3>Materials and Methods</h3>

<p class="abmaterialsmethods">SNPs, assumed largely to be phenotypically neutral,
are obtained from dbSNP and disease-associated mutations (DAMs) from OMIM and
several locus-specific mutation databases and mapped to protein structures,
where available. An automated analysis tests mutations against a range of
structural effects disrupting protein folding, binding, function and stability,
yielding an 'structural effects vector' for every mutation. Several subsets of
mutations, e.g. cancer-associated kinase mutations are analysed regarding their
preference for specific structural effects.</p>

<h3>Results</h3>

<p class="abresults">SNPs and pathogenic mutations show clear differences in
frequencies of some structural effects, and in their amino acid propensities.
Similarly, compensated DAMs on average have a milder effect on the structure,
when compared with uncompensated DAMs. Compensated mutations are also
preferentially located on the protein surface and in less conserved structural
neighbourhoods. A second analysis shows family-specific features of mutations in
kinase domains.</p>

<h3>Discussion</h3>

<p class="abdiscussion">The work presented here covers several aspects of
analysis of the structural effects of SAAPs. First, we address data collection,
and the issue of numerous databases in different formats. Furthermore, this
update contains an addition to the spectrum of structural analyses: by adding
predicted protein-protein interface residues we compensate for the discrepancy
in the number of protein complex structures available. Finally, we propose an
alternative method for inferring likely structural effects when structure is
not available.</p>

<h3>URL</h3>

<p class="abwebsite"><a href="http://www.bioinf.org.uk/saap/db/" mce_href="http://www.bioinf.org.uk/saap/db/">http://www.bioinf.org.uk/saap/db/</a><br mce_bogus="1"></p>

<div style="border-width: 1pt medium medium; border-style: solid none none; border-color: windowtext -moz-use-text-color -moz-use-text-color; padding: 1pt 0in 0in; background: none repeat scroll 0% 0% rgb(198, 217, 241);" mce_style="mso-element:para-border-div;border:none;border-top:solid windowtext 1.0pt;
mso-border-top-alt:solid windowtext .5pt;padding:1.0pt 0in 0in 0in;background:
#C6D9F1">

<p class="H3presentingauthor">Presenting Author</p>

</div>

<p class="presentingauthor">Anja Baresic (<a href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy93033 = '&#97;.b&#97;r&#101;s&#105;c' + '&#64;';
 addy93033 = addy93033 + '&#117;cl' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy93033 + suffix + '\'' + attribs + '>' );
 document.write( addy93033 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>" mce_href="mailto:
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy47465 = '&#97;.b&#97;r&#101;s&#105;c' + '&#64;';
 addy47465 = addy47465 + '&#117;cl' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy47465 + suffix + '\'' + attribs + '>' );
 document.write( addy47465 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script>">
 <script language='JavaScript' type='text/javascript'>
 <!--
 var prefix = 'm&#97;&#105;lt&#111;:';
 var suffix = '';
 var attribs = '';
 var path = 'hr' + 'ef' + '=';
 var addy32963 = '&#97;.b&#97;r&#101;s&#105;c' + '&#64;';
 addy32963 = addy32963 + '&#117;cl' + '&#46;' + '&#97;c' + '&#46;' + '&#117;k';
 document.write( '<a ' + path + '\'' + prefix + addy32963 + suffix + '\'' + attribs + '>' );
 document.write( addy32963 );
 document.write( '<\/a>' );
 //-->
 </script> <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '<span style=\'display: none;\'>' );
 //-->
 </script>This e-mail address is being protected from spambots. You need JavaScript enabled to view it
 <script language='JavaScript' type='text/javascript'>
 <!--
 document.write( '</' );
 document.write( 'span>' );
 //-->
 </script></a>)</p>

<p class="presentingauthor">Institute of Structural and Molecular Biology,
Division of Biosciences, University College London</p>

<h3>Author Affiliations</h3>

<p class="abaffiliations">1. Institute of Structural and Molecular Biology Division
of Biosciences University College London Gower Street London WC1E 6BT</p>

<h3>Acknowledgements</h3>

<p class="abaffiliations">AB - Overseas Research Awards Scheme; UCL Graduate
School Research Scheme NA - The King Faisal Specialist Hospital and Research
Center; Royal Embassy of Saudi Arabia Cultural Bureau</p></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

														</div>
				
														
													</div>
												</div>
												<!-- content end -->
				
											</div>
											<!-- mainmiddle end -->
		
											
										</div>
									</div>
									<!-- main end -->
						
								</div>
							</div>
							<!-- middle end -->
							
												
						</div>	
					</div>	
				</div>	
			</div>	
		</div>		
	</div>
	<!-- page-body end -->
	
	<div id="page-footer">
		<div class="wrapper floatholder">
	
						<div id="footer">
				<a class="anchor" href="122-other-bioinformatics-applications.html#page">&nbsp;</a>
				<div>Copyright &#169; 2011 eccb2010. All Rights Reserved.</div>

			</div>
						
			
			
		</div>
	</div>
	<!-- page-footer end -->
	
</body>
</html>