<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/ismbeccb2011-program/895-special-session-details by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:32:56 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/ismbeccb2011-program/895-special-session-details.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:15 GMT -->
<meta http-equiv="content-type" content="text/html;charset=utf-8"><!-- /Added by HTTrack -->
<head>
  <base  />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISCB, bioinformatics, cheminformatics, biocomputing, algorithm, biology, chemistry, computer" />
  <meta name="title" content="Special Session Details" />
  <meta name="description" content="ISCB -  International Society for Computational Biology" />
  <meta name="generator" content="Joomla! 1.5 - Open Source Content Management" />
  <title>Special Session Details</title>
  <link href="../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../media/system/js/modal.js"></script>
  <script type="text/javascript" src="../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">

		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->


 
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-24246804-1");
pageTracker._trackPageview();
</script>

</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">
                                                    <a id="ismbeccb2011" href="../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../index.html" title="ISCB Home"></a>
                                                </div>
						
					</div>
				</div>
			</div>
			
			<div id="search">
				<form action="http://www.iscb.org/ismbeccb2011-program/index.php" method="post">
	<div class="search">
		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="690" />
</form>
               
			</div>
				
			<div id="tabarea">


						<div id="tabmenu">
                        
						<table  class="pill">
							<tr>
								
								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>
								
							</tr>
							</table>
						</div>
 			</div>

			
	
			<div class="clr"></div>

			<div id="whitebox">
           
				<div id="whitebox_t"> 
                
					<div id="whitebox_tl"> 
						<div id="whitebox_tr"></div>
					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area"> 
                   
									

						<div id="leftcolumn">
                         
															<div class="module">
			<div>
				<div>
					<div>
											
<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);
	
	});
</script>

<ul class="menuacc" id="accordionMenu"><li class="item677"><a href="../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li><li class="item1099"><a href="../ismbeccb2011-general-info/conference-press.html"><span>Conference Press</span></a></li></ul></li><li id="current" class="parent active item690"><a href="../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="../cms_addon/conferences/ismbeccb2011/schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="../cms_addon/conferences/ismbeccb2011/artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="../cms_addon/conferences/ismbeccb2011/highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="../cms_addon/conferences/ismbeccb2011/latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="../cms_addon/conferences/ismbeccb2011/posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="../cms_addon/conferences/ismbeccb2011/oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="../cms_addon/conferences/ismbeccb2011/proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="../cms_addon/conferences/ismbeccb2011/technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="workshops.html"><span>Workshops</span></a></li><li class="item1097"><a href="ismbeccb2011-award-winners.html"><span>Award Winners</span></a></li></ul></li><li class="parent item692"><a href="../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="../cms_addon/conferences/ismbeccb2011/sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="../cms_addon/conferences/ismbeccb2011/birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../ismbeccb2011-committee-organizing/ismbeccb2011-conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../ismbeccb2011-committee-organizing/ismbeccb2011-committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="../cms_addon/conferences/ismbeccb2011/presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
	
												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">
							
							<img id="sig" width="148px" src="../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="satellite-meetings.html">
							Satellite Meetings 
							
							</a>
						</div>
						</div>                                                                                                                 
						
						

												<div id="maincolumn">
													
									<div id="nopad">
										
<table class="contentpaneopen">



<tr>
<td valign="top">
<h3>
	Special Session Details</h3>
<p>
	ISMBECCB 2011 features Special Session presentations throughout the conference July 17 - July 19. Special Sessions have the purpose of introducing the scientific community to relevant scientific issues and topics that are typically not within the focus of the conference. Preliminary program information on Special Sessions is noted below. (Schedule subject to change)</p>
<p class="MsoNoSpacing">
	<strong><span lang="EN-CA"><a name="session1"></a></span></strong></p>
<p>
	<span style="color: rgb(178, 34, 34);"><strong>Special Session 1</strong><strong>:</strong> <span class="Apple-style-span" style="font-weight: bold;">BioLINK: Integration across Genomics and Medicine</span></span></p>
<div>
	<strong><span>Organizer(s):</span></strong></div>
<div>
	Christian Blaschke, Bioalma, Madrid, Spain<br />
	Lynette Hirschman, Mitre, Bedford, United States<br />
	Hagit Shatkay, University of Delaware, United States<br />
	Alfonso Valencia, Spanish National Cancer Research Centre, Madrid, Spain</div>
<div>
	<br />
	<strong><span>Date:</span></strong><strong> </strong> Sunday, July 17<br />
	<strong>Start Time:</strong><strong> </strong> 10:45 a.m. - 12:40 p.m.<br />
	<strong>Room:</strong> Hall F2</div>
<div>
	 </div>
<div>
	<div>
		<b><span style="style">Description:</span></b></div>
	<p>
		This year, the BioLINK meeting reaches out to the diverse ISMB community with a highly relevant interdisciplinary topic: data integration and interoperability across the computational, biological and medical fields. Instead of the traditional one-day SIG before ISMB, which typically focused on various aspects of text mining, this year we take a broad focus at the interface of bioinformatics and medicine, and we are holding the meeting as a special session within the conference. <br />
		<br />
		Much information in both biology and medicine exists in the form of text, including reports, published literature, database entries and medical patient records. Hence, text mining methods (which have traditionally been the focus of BioLINK) remain relevant to both the biological and the medical communities, and to the interdisciplinary research between the two. <br />
		However, many other types of relevant data, ranging from genomic mutations to chemical reactions and vital signs in patients are not textual. They come in the form of time series, images, genomic and proteomic sequences, among others.<br />
		<br />
		Translating biological advances into practical medical solutions is the focus of much current research in several communities, and discussions of the needs and the challenges have recently been taking place at AMIA, particularly at the Summit on Translational Bioinformatics, as well as in the EU, at the workshop on Bridging the Gap in Biomedical Genetics (October, 2010, Cambridge, UK). This special session is specifically centered on the role and challenges of multiple types of data and models, and the way such diverse sources of information can be integrated and used effectively.</p>
	<p>
		<u><strong>Talks and Speakers:<br />
		</strong></u></p>
	<table border="1" cellpadding="1" cellspacing="1" style="width: 609px; height: 397px;">
		<tbody>
			<tr>
				<td>
					<p>
						10:45-10:50</p>
				</td>
				<td>
					<p>
						 Session Introduction</p>
				</td>
			</tr>
			<tr>
				<td valign="top">
					<p>
						10:50-11:10</p>
				</td>
				<td valign="top">
					<p>
						<b>Speaker:</b> <i>Soren Brunak, PhD.<br />
						<span>                   Technical University of Denmark &<br />
						                    University of Copenhagen</span></i></p>
					<p>
						<span><b>Title:</b></span><br />
						<b><i>Integrating phenotypic data from electronic patient <span>records with molecular level systems biology</span></i></b></p>
				</td>
			</tr>
			<tr>
				<td valign="top">
					<p>
						11:10-11:15</p>
				</td>
				<td valign="top">
					<p>
						Transition to parallel tracks</p>
				</td>
			</tr>
			<tr>
				<td style="vertical-align: top;">
					<p>
						11:15-11:40<span lang="EN-US" new="New" roman="roman" style="font-size: 11pt;"> </span></p>
				</td>
				<td style="vertical-align: top;">
					<p>
						<b>Speaker:</b> <i>Russ Altman, MD, PhD.<br />
						<span>                        Stanford University</span></i></p>
					<p>
						<span><b>Title:<br />
						<i>Issues in annotating whole-genome data with clinically relevant pharmacogenomics knowledge</i></b></span></p>
				</td>
			</tr>
			<tr>
				<td style="vertical-align: top;">
					<p>
						 11:40-11:45<span lang="EN-US" new="New" roman="roman" style="font-size: 11pt;"> </span></p>
				</td>
				<td style="vertical-align: top;">
					<p>
						Transition to parallel tracks</p>
				</td>
			</tr>
			<tr>
				<td style="vertical-align: top;">
					<p>
						11:45-12:10</p>
				</td>
				<td style="vertical-align: top;">
					<p>
						<b>Speaker:</b> <em>Yves Moreau</em><i><em>, PhD.</em> </i></p>
					<p>
						<i>                   Katholieke Universiteit Leuven<span>                 </span></i><span>         <br />
						</span></p>
					<p>
						<span><b>Title:</b></span>  <b><br />
						<span><i>Instrumenting the Healthcare System to Define the True Names of Disease</i></span></b></p>
				</td>
			</tr>
			<tr>
				<td style="vertical-align: top;">
					<p>
						 12:10-12:15</p>
				</td>
				<td style="vertical-align: top;">
					<p>
						Transition to parallel tracks</p>
				</td>
			</tr>
			<tr>
				<td style="vertical-align: top;">
					<p>
						12:15-12:40</p>
				</td>
				<td style="vertical-align: top;">
					<p>
						Panel Discussion (Drs. Altman, Brunak and Moreau)<br />
						<span>Interoperability across Information Extraction and Computational Bio-medicine Platforms </span></p>
					<p>
						<span>Led by <i>Florian Leitner, CNIO, Spain,</i></span><br />
						<i><span>and Karin Verspoor, PhD, University of Colorado, Denver </span></i></p>
				</td>
			</tr>
		</tbody>
	</table>
	<p style="margin-left: 80px;">
		 </p>
	<p>
		<strong>Talk Abstracts and Speakers Bios:</strong></p>
</div>
<div>
	<div>
		<u><b>Integrating Phenotypic Data from Electronic Patient Records with Molecular Level Systems Biology</b></u></div>
	<div>
		<strong>Speaker</strong>:Søren Brunak, Technical University of Denmark & University of Copenhagen</div>
	<div>
		<b><i> </i></b></div>
	<div>
		<a href="http://www.cbs.dtu.dk/staff/brunak"><b><i>http://www.cbs.dtu.dk/staff/brunak</i></b></a></div>
	<div>
		 </div>
	<div>
		<b>Abstract:</b> Electronic patient records remain a rather unexplored, but potentially rich data source for discovering correlations between diseases. We describe a general approach for gathering phenotypic descriptions of patients from medical records in a systematic and non-cohort dependent manner. By extracting phenotype information from the free-text in such records we demonstrate that we can extend the information contained in the structured record data, and use it for producing fine-grained patient stratification and disease co-occurrence statistics. The approach uses a dictionary based on the International Classification of Disease ontology and is therefore in principle language independent. As a use case we show how records from a Danish psychiatric hospital lead to the identification of disease correlations, which subsequently are mapped to systems biology frameworks.</div>
	<div>
		 </div>
	<div>
		<b><i>BIO:</i></b></div>
	<div>
		Søren Brunak, Ph.D., is professor of Bioinformatics at the Technical University of Denmark and professor of Disease Systems Biology at the University of Copenhagen. Prof. Brunak is the founding Director of the Center for Biological Sequence Analysis, which was formed in 1993 as a multi-disciplinary research group of molecular biologists, biochemists, medical doctors, physicists, and computer scientists. Søren Brunak has been highly active within data integration, where machine learning techniques often have been used to integrate predicted or experimentally established functional genome and proteome annotation. His current research does combine molecular level systems biology and healthcare sector data such as electronic patient records and biobank questionnaires . The aim is to group and stratify patients not only from their genotype, but also phenotypically based on the clinical descriptions in the medical records.</div>
	<div>
		Prof. Brunak completed his early studies in physics at the Niels Bohr Institute, University of Copenhagen, and his doctoral work in Computational Biology at the Technical University of Denmark. Brunak has published more than 175 papers with peer-review, altogether having more than 18,500 citations. His most cited paper (from 1997) has more than 3,800 citations, h-factor 53. Søren Brunak is member of the Danish Academy for the Natural Sciences (1997), The Danish Academy of Technical Sciences (2002), The Danish Royal Society of Science and Letters (2004) and of EMBO since 2009. More about his research group can be found at <a href="http://www.cbs.dtu.dk/" target="_blank">www.cbs.dtu.dk</a>.</div>
	<div>
		 </div>
	<div>
		<b> </b></div>
	<div>
		<u><b>Issues in Annotating Whole-Genome Data with Clinically Relevant Pharmacogenomics Knowledge</b></u></div>
	<div>
		<strong>Speaker</strong>: Russ Biagio Altman. Professor of Bioengineering, Genetics, & Medicine; Chairman of the Bioengineering Department. Stanford University.</div>
	<div>
		 </div>
	<div>
		<a href="http://helix-web.stanford.edu/people/altman/" target="_blank"><b><i>http://helix-web.stanford.edu/people/altman/</i></b></a></div>
	<div>
		<b><i> </i></b></div>
	<div>
		<b>Abstract:</b>  The mission of the Pharmacogenomics Knowledgebase (PharmGKB, <a href="http://www.pharmgkb.org/" target="_blank">http://www.pharmgkb.org/</a>) is to collect and encode all available information about how human genetic variation impacts drug response.  We have recently reported using the PharmGKB to annotate the whole genome of an individual.  This initial annotation was done manually and taught us what would be required for routine markup of genomes in a relatively high-throughput fashion.   We have therefore revisited much of the PharmGKB contents and re-curated the data in order to make clinical annotation more facile and accurate.   I will discuss the issues associated with annotating whole human genomes, particularly focusing on the challenges of annotating rare or novel mutations that have never been (and may never be) studied, but which look potentially important for the individual whose genome is being annotated. </div>
	<div>
		 </div>
	<div>
		<b><i>BIO:</i></b></div>
	<div>
		Russ Biagio Altman is professor of bioengineering, genetics, & medicine (and of computer science, by courtesy) and chairman of the Bioengineering Department at Stanford University. His primary research interests are in the application of computing technology to basic molecular biological problems of relevance to medicine. He particularly interested in informatics methods for advancing pharmacogenomics, the study of how human genetic variation impacts drug response (e.g. <a href="http://www.pharmgkb.org/" target="_blank">http://www.pharmgkb.org</a>/). Other work focuses on the analysis of functional sites within macromolecules  and the application of algorithms for determining the structure, dynamics and function of biological macromolecules (<a href="http://features.stanford.edu/" target="_blank">http://features.stanford.edu/</a>). Dr. Altman holds an M.D. from Stanford Medical School, a Ph.D. in Medical Information Sciences from Stanford, and an A.B. from Harvard College. He has been the recipient of the U.S. Presidential Early Career Award for Scientists and Engineers and a National Science Foundation CAREER Award. He is a fellow of the American College of Physicians, the American College of Medical Informatics, and the American Institute of Medical and Biological Engineering. He is a past-president and founding board member of the International Society for Computational Biology, and an organizer of the annual Pacific Symposium on Biocomputing. He leads one of seven NIH-supported National Centers for Biomedical Computation, focusing on physics-based simulation of biological structures (<a href="http://simbios.stanford.edu/" target="_blank">http://simbios.stanford.edu/</a>). He won the Stanford Medical School graduate teaching award in 2000. He is a member of the Institute of Medicine of the National Academies</div>
	<div>
		 </div>
	<div>
		 </div>
	<div>
		<u><b>Instrumenting the Healthcare System to Define the True Names of Disease</b></u></div>
	<div>
		<p>
			<b>Speaker:</b> <em>Yves Moreau</em><i><em>,</em> Katholieke Universiteit Leuven<span>                 </span></i><span>         <br />
			</span></p>
	</div>
	<br />
	<div>
		<b>Abstract:</b>  Most disciplines of medicine, with the notable exception of infectious disease, continue to classify and treat disease at the phenomenological level. The promise of genomic medicine includes personalizing diagnoses and therapies but also of redefining diseases based on molecular markers to bring us closer to an etiological classification. The availability of commodity-priced genome-scale assays and large public data sets makes the goal of finding these &ldquo;true names&rdquo; of diseases now feasible. Several such approaches are described ranking from cancer to autism. Also discussed are several structural impediments achieving this goal including: A) the growth of the Incidentalome, the tsunami of false positives that inevitably result from application of massively parallel tests; B) the lack of systematic interpretations of genomic tests and evaluation of their performance C) the absence of a mechanism to transfer the growing knowledge of genomics to the physician at the point of care. The opportunity in instrumenting the healthcare enterprise for discovery research to overcome these impediments is reviewed.</div>
	<br />
	<div>
		 </div>
	<div>
		<strong>About the Organizers:<br />
		</strong></div>
	<div>
		 </div>
	<div>
		<b>Christian Blaschke</b> is technical coordinator of the Spanish Institute of Bioinformatics (INB), Madrid, Spain. He obtained his PhD in the Group of Alfonso Valencia and was part of the organization of BioLINK SIG meetings starting in 2002. His academic work has been focused on text mining applied to molecular biology and biomedicine where he contributed in the areas of protein-protein interactions, DNA array analysis and automatic ontology learning. Christian Blaschke was also one of the organizers of the BioCreAtIvE (Critical Assessment of Information Extraction for Biology) challenge carried out in 2004.</div>
	<div>
		 </div>
	<div>
		 </div>
	<div>
		<b>Lynette Hirschman</b> is Director, Biomedical Informatics for the Information Technology Center at the MITRE Corporation in Bedford, MA, USA. She has been an organizer of the BioLINK SIG on Text Mining in Biology for ISMB starting in 2002. and a founder of BioCreAtIvE, (Critical Assessment of Information Extraction for Biology), the first Challenge Evaluation for text mining applied to biological applications. Her research focuses on the application of natural language processing technology to biomedical applications, including removal of personally identifiable information (PII) for protection of privacy, automated fact extraction for medical records and on the coupling of genomic and clinical data to support translational medicine. She is a member of the board of the Genomic Standards Consortium, working on metadata capture for metagenomics and a member of the organizing team for the i2b2 2011 challenge evaluation for information extraction from medical records.</div>
	<div>
		<b><font size="4"> </font></b></div>
	<div>
		<span style="font-size: 12px;"><strong>Hagit Shatkay</strong> is an Associate Professor at the Dept. of Computer and Information Sciences at the University of Delaware, with cross-appointments at the Biomedical Engineering Program, and at the Delaware Biotechnology Institute. Her research is in the area of machine learning as it applies to biomedical data mining. Among her many organizational roles, she has been on the organizing committee of the BioLINK SIG at ISMB since 2005, on the steering committee for TREC Genomics, and multiple times chair of the text mining area at ISMB.</span></div>
	<div>
		<span style="font-size: 12px;"> </span></div>
	<div>
		<span style="font-size: 12px;">She is an active member of the bio-text research community since its early days (1999), where her work focuses primarily on biomedical information retrieval, and more recently on integrating text, image and sequence data within the biological data mining process. Recent major projects in her lab include work on understanding and predicting heart disease using multiple sources of information, document retrieval through image and text data, SherLoc &ndash; protein subcellular location prediction by integrating text and sequence data, and F-SNP &ndash; a comprehensive system for ranking SNPs by potential deleterious effects.  </span></div>
	<div>
		<span style="font-size: 12px;"><br />
		</span></div>
	<div>
		<span style="font-size: 12px;"><strong> Alfonso Valencia</strong> is the director of the Structural and Computational Biology Programme of the Spanish National Cancer Research Centre and Director of the Spanish Bioinformatics Institute. His group works in the development and application of Computational Biology methods for the study of problems related with the evolution of molecular interactions. In the field of text mining his group developed new approaches for the detection of protein function and interactions in biological text and a number of applications combining text mining with other genomic resources.</span></div>
	<div>
		<span style="font-size: 12px;"><br />
		Dr. Valencia is co-executive editor of the journal "Bioinformatics", member of EMBO, co-organizer of ECCB 05, chair of the text mining/Biological networks areas of ISMB (03-07) and founder, former VP, and member of the board of directors of ISCB. Dr. Valencia has been part of the organization of BioLINK (Text Mining in Biology) since 2001 and co-organizer of the Critical Assessment of Information Extraction for Biology (BioCreAtIvE 2003-4, 2006-07, 2009-2012).</span></div>
	<p>
		 </p>
</div>
<div>
	 </div>
<p>
	<span style="font-size: 10px;"><a href="#up">TOP<br />
	</a></span></p>
<hr />
<p>
	 </p>
<div>
	<span style="color: rgb(178, 34, 34);"><strong><a name="session2"></a>Special Session 2: Insights & outlook for individual genome interpretation: lessons from the Critical Assessment of Genome Interpretation</strong></span><br />
	<strong><span>Organizer(s):</span></strong> <br />
	John Moult, Baltimore, University of Maryland, United States <br />
	Susanna Repo, University of California, Berkeley, United States<br />
	Steven E. Brenner, University of California, Berkeley, United States</div>
<div>
	 </div>
<div>
	<b>Date:  </b>Sunday, July 17</div>
<div>
	<strong>Start Time: </strong>2:30 p.m. &ndash; 4:25 p.m.<br />
	<strong><span>Room: </span></strong> Hall F2<br />
	<br />
	<strong>Description:</strong></div>
<div>
	This session will provide an overview and assessment of the state-of-the art in prediction of phenotype from genotype. The session is motivated by the burgeoning availability of individuals' genomes, and the desire to interpret them for research and clinical applications. <br />
	<br />
	The Critical Assessment of Genome Interpretation (CAGI, \'kā-jē\), is a community experiment to assess the current methods of predicting phenotypic impacts of genomic variation. Based on the same model as CASP (Critical Assessment of Structure Prediction), the experimental goals of CAGI are to evaluate the current methods to make useful predictions of phenotypes from genomics data, to identify bottlenecks in genome interpretation that suggest especially critical areas of future research, and to engage and connect researchers from the diverse disciplines whose expertise is essential to methods for genome interpretation. <br />
	<br />
	The first round of CAGI, which was organized during the latter half of 2010, included predictions of phenotype for two sets of cancer mutations, a set of mutations in a monogenic disease related enzyme, 10 personal genomes, and identification of the molecular mechanisms underlying the results of genome wide association studies. In total, 108 prediction submissions from 17 research groups in 8 different countries were submitted to the first CAGI experiment. <br />
	<br />
	In this session, an overview of the initial CAGI experiment will be presented. We will discuss the broad successes and failures, and describe the prediction challenges of the different datasets of CAGI. Current methods for predicting phenotype from genomic data will be presented and the limitations of these methods discussed. The future directions for CAGI and genome interpretation will also be discussed.<br />
	<div>
		 </div>
	<div>
		 </div>
	<div>
		<b>Finalized Schedule </b></div>
	<div>
		 </div>
	<div>
		<b>Timeline:</b></div>
	<div>
		 </div>
	2:30PM <strong>Introduction to the Critical Assessment of Genome Interpretation (CAGI) experiment</strong><br />
	2:50PM Discussion<br />
	2:55PM Transition to parallel tracks<br />
	3:00PM <strong>Results from CAGI 2010: successes and challenges</strong><br />
	3:20PM Discussion<br />
	3:25PM Transition to parallel tracks<br />
	3:30PM <strong>Methods to predict individuals&rsquo; medical phenotypes from genomic data</strong><br />
	3:50PM Discussion<br />
	3:55PM Transition to parallel tracks<br />
	4:00PM <strong>Future directions for CAGI and Genome Interpretation, Panel Discussion</strong><br />
	4:20PM Conclusions on panel discussion<br />
	4:25PM Transition to parallel tracks <br />
	<div>
		 </div>
	<div>
		<b> </b></div>
	<div>
		<b>Session descriptions:</b></div>
	<div>
		<b>  <br />
		</b></div>
	<div>
		<b>Introduction to the Critical Assessment of Genome Interpretation (CAGI<span>)  experiment </span></b></div>
	<div>
		 </div>
	<div>
		<strong>Speakers:</strong> <br />
		Steven E. Brenner, University of California Berkeley<br />
		Susanna Repo, University of California Berkeley</div>
	<div>
		<b> </b></div>
	<div>
		In this session, <span>CAGI organizers will provide an introduction to the experiment and survey the mechanisms by which genetic variation can yield phenotypic impacts and will define the different levels of phenotypes &ndash; organismal, cellular and molecular &ndash; that are predicted in the <span>CAGI effort. </span>The speakers will discuss variation phenomena, observation methods and resulting data, and exemplary biological phenotypes that result from each. The first CAGI experiment will also be described, with a description of the prediction datasets of that experiment. </span></div>
	<div>
		<b> </b></div>
	<div>
		<b>Results from CAGI <span>2010: successes and challenges </span></b></div>
	<div>
		<br />
		<p class="MsoNormal">
			<b><span lang="EN-US">Speaker:<span style="mso-tab-count:1"> </span></span></b><span lang="EN-US" style="mso-bidi-font-weight:bold">Pauline Ng, Genome Institute of Singapore</span><b> <br />
			</b></p>
	</div>
	<div>
		Pauline Ng, an assessor for CAGI , will present an overview of results of the initial experiment. She will discuss broad successes and failures, and describe the prediction challenges of the different CAGI datasets, focusing on the single nonsynonymous variations. She will identify bottlenecks of the current methods and datasets and suggest future research directions.</div>
	<div>
		 </div>
	<br />
	<div>
		<p class="MsoNormal" style="tab-stops:360.8pt">
			<b style="mso-bidi-font-weight:normal"><span lang="EN-US">Methods to predict individuals&rsquo; medical phenotypes from genomic data<span style="mso-bidi-font-weight:bold"> <span style="mso-tab-count:1">           </span></span></span></b></p>
		<p class="MsoNormal">
			<b><span lang="EN-US"> Speakers:<span style="mso-tab-count:1">       </span></span></b><span lang="EN-US" style="mso-bidi-font-weight:bold"><br />
			Sean Mooney, Buck Institute<b><span style="mso-spacerun:yes"> <br />
			</span></b></span><span lang="EN-US" style="mso-bidi-font-weight:bold">Rachel Karchin, Johns Hopkins University<br />
			Hannah Carter, Johns Hopkins University</span></p>
		<p class="MsoNormal">
			<span lang="EN-US" style="mso-bidi-font-weight:bold"> </span></p>
		<p class="MsoNormal">
			<span lang="EN-US" style="mso-bidi-font-weight:bold">One of the most challenging CAGI datasets involved predicting the phenotypes of individuals from the Personal Genome Project.<span style="mso-spacerun:yes">  </span>These individuals have made many of their genomes and medical characteristics publicly available.<span style="mso-spacerun:yes">  </span>We solicited dozens of additional as-yet unpublished phenotypes from these individuals and presented these to predictors.<span style="mso-spacerun:yes">  </span>The assessor for this dataset, Sean Mooney, will present an overview of the challenge and the nature of the results.<span style="mso-spacerun:yes">  </span>Hannah Cater and Rachel Karchin will be shown in a video explaining their approach to making predictions for this dataset.<span style="mso-spacerun:yes">  </span></span></p>
	</div>
	<div>
		 </div>
	<div>
		<b>Future directions for CAGI <span>and Genome Interpretation, Panel Discussion </span></b></div>
	<div>
		 </div>
	<div>
		<strong>Panel members: <br />
		</strong></div>
	<div>
		Russ Altman, Stanford University<br />
		Rita Casadio, University of Bologna<br />
		Iddo Friedberg, Miami University<br />
		Mauno Vihinen, University of Tampere<br />
		John Moult, University of Maryland (Discussion moderator)</div>
	<div>
		<b> </b></div>
	<div>
		In this last session, panelists will discuss the future directions for CAGI and genome interpretation. In order to facilitate the participation of the ISMB community in the discussion, we will collect questions, comments, and suggestions from the audience throughout the special session using electronic communications.</div>
</div>
<div>
	 </div>
<div>
	 </div>
<div>
	<a href="#up"><span style="font-size: 10px;">TOP</span></a></div>
<div>
	 </div>
<hr />
<div>
	 </div>
<div>
	<span style="color: rgb(178, 34, 34);"><strong><a name="session3"></a>Special Session 3: RNA structure: from genomes to nanotechnology </strong></span><br />
	<strong><span>Organizer(s):</span></strong><br />
	Peter Clote, Boston College, United States<br />
	Ivo L. Hofacker, University of Vienna, Austria<br />
	David Mathews, Rochester Medical Center, New York, United States<span lang="EN-US" style="style"><br />
	Peter Schuster, Professor emeritus, University of Vienna, Austria</span></div>
<div>
	<br />
	<strong>Date:</strong><strong> </strong> Monday, July 18<br />
	<strong><span>Start Time:</span></strong><strong> </strong> 10:45 a.m. &ndash; 12:40 p.m.<br />
	<strong>Room:</strong><strong> </strong> Hall F2</div>
<div>
	<strong><br />
	Description:<br />
	</strong></div>
<div>
	This Special Session includes both computational and experimental methods at the cutting edge of RNA research. The interdisciplinary aspects are focusing on theory and practice to engineer and control RNA molecules for performing computations, for controlling genes, for self-assembly, for binding as aptamers to specific targets, and ultimately for serving as pharmaceuticals.</div>
<div>
	 </div>
<div>
	<strong>Speakers</strong>:</div>
<ul>
	<li>
		Niles Pierce, Caltech</li>
	<li>
		Rhiju Das, Stanford</li>
	<li>
		Susan Gottesman, NIH</li>
	<li>
		Peter Stadler, Leipzig</li>
</ul>
<p>
	<strong>Title:<br />
	</strong></p>
<p>
	<span>Engineering Nucleic Acid Devices</span></p>
<p>
	<strong><span>Speaker:</span><br />
	</strong></p>
<p>
	Niles Pierce, Caltech</p>
<p>
	<strong>Abstract:<br />
	</strong></p>
<p>
	The programmable chemistry of nucleic acid base-pairing provides a versatile framework for engineering molecular devices and systems. This talk will describe efficient algorithms for nucleic acid sequence design and illustrate their use in engineering programmable molecular amplifiers for multiplexed imaging of mRNA expression within intact vertebrate embryos.</p>
<p>
	<br />
	<strong>Title:</strong><br />
	Atomic accuracy and blind predictions through enumerative 3D RNA modeling</p>
<p>
	<strong>Speaker</strong>:<br />
	Prof Rhiju Das, Stanford<br />
	<br />
	<strong>Abstract:</strong><br />
	High-resolution 3D structure modeling of RNAs and other macromolecular systems is a long sought but still unachieved goal of computational biology. Inspired by successful protein modeling approaches, we recently showed that the Rosetta framework permits modeling and design of small non-canonical RNA motifs at near-atomic accuracy. Nevertheless, a major bottleneck remains: the difficulty of comprehensively sampling these molecules&rsquo; many degrees of freedom. Towards eliminating fundamental issues in this sampling bottleneck, we have postulated a &lsquo;StepWise Ansatz&rsquo; (SWA) for constructing well-packed models in small steps, enumerating several million conformations for each nucleotide and covering all possible build-up paths. We present results on non-canonical RNA motifs as well as highly irregular protein loops that have been intractable for prior fragment assembly or analytic loop closure approaches. In all cases, the enumerative SWA method either reaches atomic accuracy or exposes flaws in Rosetta&rsquo;s high-resolution energy function, suggesting that the approach is ready for more rigorous tests. Over the last year, we have therefore helped organize and participated in blind de novo modeling trials on RNAs with previously unsolved 3D structures. For a few cases, chemical mapping and crystallographic data have become available, and we report high accuracy de novo modeling in at least two non-canonical motifs.</p>
<p>
	 </p>
<p>
	<strong>Title:</strong><br />
	Bacterial small RNAs in regulatory networks</p>
<p>
	<strong>Speaker:</strong><br />
	Susan Gottesman, National Cancer Institute, NIH</p>
<p>
	<br />
	<strong>Abstract:</strong><br />
	Bacteria such as E. coli contain on the order of 100 small regulatory RNAs. A large number of these regulate gene expression by pairing with target mRNAs to activate or repress gene expression; expression of the sRNAs are themselves subject to stringent regulation of their transcription. One challenge has been to identify all the targets of a given sRNA, although both experimental and computational methods continue to improve. In an alternative approach, we have developed rapid methods to query the pairing sRNAs for their effects on a given target. The combined outcome of experiments in our lab and others demonstrates an important role for sRNAs in connecting transcriptional networks, in feedback regulation of networks, and in mediating multiple regulatory inputs to many targets.</p>
<p>
	 </p>
<p>
	<strong>Title:<br />
	</strong></p>
<p>
	Folding Algorithms for RNAs with Pseudoknots</p>
<p>
	<strong>Speaker:</strong><strong><br />
	</strong>Peter Stadler, Boston College</p>
<p>
	<strong>Abstract:</strong><br />
	A large variety of dynamics programming algorithms for RNA structures with pseudoknots have been proposed that differ dramatically from one another in the classes of structures considered. Mostly, these are motivated by considerations of algorithmic tractability. An alternative is the use of the natural topological classification of RNA structures in terms of irreducible components that are embeddable in surfaces of fixed genus. For genus-1 structures, this adds four additional building blocks to the conventional secondary structures.<br />
	With a single exception the resulting class of structures encompasses all known pseudoknotted structured. A corresponding unambiguous multiple context free grammar provides an efficient dynamic programming approach for energy minimization, partition function, and stochastic sampling. The topology-based approach is not only appealing from a theoretical point of view: it admits a parametrization of pseudoknot penalties that depends on topological complexity, which increases sensitivity and positive predictive value for base pairs by 10-20%.</p>
<p>
	Reidys CM, Huang FWD, Andersen JE, Penner RC, Stadler PF, Nebel ME<br />
	Topology and prediction of RNA pseudoknots. Bioinformatics 27: 1076-1085 (2011)<br />
	<strong><br />
	</strong></p>
<p>
	 </p>
<div>
	 </div>
<p class="MsoNoSpacing">
	<span style="font-size: 10px;"><a href="#up">TOP</a></span></p>
<hr />
<p>
	 </p>
<div>
	<span style="color: rgb(178, 34, 34);"><strong><a name="session4"></a>Special Session 4: ELIXIR </strong></span><br />
	<strong><span>Organizer(s):</span></strong> <br />
	Janet Thornton, European Bioinformatics Institute, United Kingdom<br />
	<br />
	<strong>Date:  </strong> Monday, July 18<br />
	<strong>Start Time: </strong> 2:30 p.m. - 4:25 p.m.<br />
	<strong>Room: </strong> Hall F2</div>
<div>
	 </div>
<div>
	<strong>Description:</strong> The mission of ELIXIR is to build a sustainable European infrastructure for biological information supporting life science research and its translation to:</div>
<ul>
	<li>
		medicine,</li>
	<li>
		the environment,</li>
	<li>
		the bioindustries, and</li>
	<li>
		society</li>
</ul>
<div>
	<br />
	ELIXIR has completed a three year, European wide consultation involving academic and industrial users, data providers and international collaborators, including three stakeholders meetings, two surveys, and fourteen work packages.<br />
	<br />
	ELIXIR will be a distributed infrastructure arranged as a hub and nodes, with the hub at the European Molecular Biology Laboratory&rsquo;s European Bioinformatics Institute (EMBL-EBI) in Hinxton, UK.<br />
	<br />
	The ELIXIR Steering Committee recently issued a Request for Suggestions for ELIXIR nodes, seeking input from organisations that are interested in hosting one of its &lsquo;nodes&rsquo;, in order to help shape ELIXIRs construction. Organisations wishing to contribute their input were requested to consider how they might contribute: data resources; bio-computing capacity; infrastructure for data integration; and services for the research community, including training and standards development.<br />
	<br />
	The infrastructure is currently moving forward towards construction.</div>
<div>
	 </div>
<div>
	<strong>The ELIXIR special session will comprise: </strong><br />
	<br />
	<div>
		2:30 p.m. - 2:55 p.m. Building ELIXIR presented by Janet Thornton (ELIXIR Project Coordinator) to introduce the audience to ELIXIR and the latest developments towards its construction<br />
		<br />
		3:00 p.m. - 3:25 p.m. Two short presentations on:</div>
	<div>
		'ELIXIR and the other ESFRI BMS Projects' presented by ELIXIR Project Manager, Andrew Lyall<br />
		and<br />
		'ELIXIR and Industry' presented by Dominic Clark, EBI Industry Programme Manager.</div>
	<div>
		 </div>
	<div>
		3:30 p.m. - 3:55 p.m. A panel session during which representatives from the ELIXIR Steering Committee will talk about how their country is working towards a pan European infrastructure and how ELIXIR has fostered the construction of an internal network within their country.</div>
	<div>
		 </div>
	<div>
		 </div>
	<div>
		<strong>Chaired by: Janet Thornton (ELIXIR Project Coordinator)</strong></div>
	<div>
		<strong>Panellists: </strong><br />
		<br />
		Anna Tramontano (University of Rome) representing Italy<br />
		Alfonso Valencia (Spanish National Cancer Research Centre- CNIO) representing Spain<br />
		Søren Brunak (Technical University of Denmark- DTU) representing Denmark<br />
		Bengt Persson (Linköping University) representing Sweden<br />
		Tim Hubbard (Wellcome Trust Sanger Institute) representing the UK</div>
	<div>
		<br />
		Time will be allowed after each presentation and panel session for discussion.<br />
		4:00 p.m. - 4:25 p.m. Panel Continues</div>
	<div>
		 </div>
	Website: <a href="http://www.elixir-europe.org/" target="_blank">www.elixir-europe.org</a></div>
<div>
	 </div>
<p>
	<a href="#up"><span style="font-size: 10px;">TOP</span></a></p>
<hr />
<p>
	 </p>
<div>
	<span style="color: rgb(178, 34, 34);"><strong><a name="session5"></a>Special Session 5: Bioinformatics for Synthetic Biology: Scaling to Complex Biological Systems</strong></span><br />
	<strong><span>Organizer(s):</span></strong> <br />
	C. Forbes Dewey, Jr., Massachusetts Institute of Technology, Cambridge, United States<br />
	Richard I. Kitney, Imperial College, London, United Kingdom</div>
<div>
	 </div>
<div>
	<strong><span>Date:  </span></strong> Tuesday, July 19<br />
	<strong>Start Time: </strong> 10:45 a.m. - 12:40 p.m.<br />
	<strong>Room: </strong> Hall F2</div>
<div>
	 </div>
<div>
	<strong>Description:<br />
	</strong></div>
<div>
	Synthetic Biology is an emerging discipline that learns from natural biological systems and attempts to construct novel living devices that are based on biological principles. These constructs are self-replicating systems that have special functions. Making ethanol from switchgrass would be an example. Many of the constructs resemble electrical circuit elements such as and/nor gates, adders and multipliers, and feedback loops, so one can conceive of crating &ldquo;living&rdquo; entities that have complex decision capabilities as well as useful functions.</div>
<div>
	 </div>
<div>
	This Special Session will be devoted to exploring the role of bioinformatics in synthetic biology. One of the key concepts driving current research is the ability to combine multiple synthetic biological &ldquo;parts&rdquo; to achieve complex goals. With the publication and curation of these synthetic biology constructs, one can create a database of Standard Biological Parts. If the curation includes computable descriptions (e.g. ODEs that include known rate constants written in an accepted machine-readable format such as SBML), then one could predict the complex behavior of collections of these modules working as a single entity.</div>
<div>
	 </div>
<div>
	<strong>Presentations </strong>(in presentation order):</div>
<ol>
	<li>
		Richard I. Kitney. <strong>Systematic design challenges in synthetic biology.</strong></li>
	<li>
		Ron Weiss. <strong>Synthetic biology: from parts to modules to therapeutic systems.</strong></li>
	<li>
		Christopher Voigt. <strong>Refactoring complex gene clusters in bacteria.</strong></li>
	<li>
		C. Forbes Dewey, Jr. <strong>Managing the "BRICKS" of synthetic biology.</strong></li>
</ol>
<div>
	 </div>
<div>
	<strong>Abstract - Systematic design challenges in synthetic biology.</strong></div>
<div>
	<p class="MsoNormal" style="text-align: justify;">
		<span lang="EN-GB">The paper will discuss the systematic design approach to synthetic biology. Key elements of this approach are the design cycle and the use of modularity. The paper will examine these principles in relation to two examples: biological logic gate design and a biosensor for urinary track infection. </span></p>
</div>
<div>
	 </div>
<div>
	<strong>Abstract - Synthetic biology: from parts to modules to therapeutic systems.</strong></div>
<div>
	Synthetic biology is revolutionizing how we conceptualize and approach the engineering of biological systems. Recent advances in the field are allowing us to expand beyond the construction and analysis of small gene networks towards the implementation of complex multicellular systems with a variety of applications. In this talk I will describe our integrated computational / experimental approach to engineering complex behavior in living systems ranging from bacteria to stem cells. In our research, we appropriate design principles from electrical engineering and other established fields. These principles include abstraction, standardization, modularity, and computer aided design. But we also spend considerable effort towards understanding what makes synthetic biology different from all other existing engineering disciplines and discovering new design and construction rules that are effective for this unique discipline. We will briefly describe the implementation of genetic circuits and modules with finely-tuned digital and analog behavior and the use of artificial cell-cell communication to coordinate the behavior of cell populations. The first system to be presented is an RNAi-based logic circuit that can detect and destroy specific cancer cells based on their microRNA expression profiles. We will also discuss preliminary experimental results for obtaining precise spatiotemporal control over stem cell differentiation for tissue engineering applications. We will conclude by discussing the design and preliminary results for creating an artificial tissue homeostasis system where genetically engineered stem cells maintain indefinitely a desired level of pancreatic beta cells despite attacks by the autoimmune response, relevant for diabetes.</div>
<div>
	 </div>
<div>
	<strong>Abstract -</strong> <strong>Managing the "BRICKS" of synthetic biology.</strong></div>
<div>
	One of the most exciting opportunities in synthetic biology is to assemble individual biological parts to achieve new functionality. Because the behavior of each part is, in general, highly nonlinear, simple methods of combination will not be productive. We examine current efforts to define the semantics of the &ldquo;BRICKS&rdquo; (Clotho, SBOL, MIRIAM, DICOM) and what is required of the underlying mathematical models that carry the quantitative information on temporal behavior.</div>
<div>
	<b> </b></div>
<div>
	<a href="#up"><span style="font-size: 10px;">TOP<br />
	</span></a></div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

										
                                   	</div>
						</div>
						
						<div class="clr"></div>
					</div>
					<div class="clr"></div>
				</div>

				<div id="whitebox_b">
					<div id="whitebox_bl">
						<div id="whitebox_br"></div>
					</div>
				</div>
			</div>

			<div id="footerspacer"></div>
		</div>

		
	</div>
</div>


</body>

<!-- Mirrored from www.iscb.org/ismbeccb2011-program/895-special-session-details by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:32:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8"><!-- /Added by HTTrack -->

<!-- Mirrored from www.iscb.org/archive/conferences/iscb/ismbeccb2011-program/895-special-session-details.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:15 GMT -->
</html>
