<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/ismbeccb2011-program/workshops by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:32:44 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/ismbeccb2011-program/workshops.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:55 GMT -->
<meta http-equiv="content-type" content="text/html;charset=utf-8"><!-- /Added by HTTrack -->
<head>
  <base  />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISCB, bioinformatics, cheminformatics, biocomputing, algorithm, biology, chemistry, computer" />
  <meta name="title" content="Workshops" />
  <meta name="description" content="ISCB -  International Society for Computational Biology" />
  <meta name="generator" content="Joomla! 1.5 - Open Source Content Management" />
  <title>Workshops</title>
  <link href="../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../media/system/js/modal.js"></script>
  <script type="text/javascript" src="../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">

		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->


 
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-24246804-1");
pageTracker._trackPageview();
</script>

</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">
                                                    <a id="ismbeccb2011" href="../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../index.html" title="ISCB Home"></a>
                                                </div>
						
					</div>
				</div>
			</div>
			
			<div id="search">
				<form action="http://www.iscb.org/ismbeccb2011-program/index.php" method="post">
	<div class="search">
		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="909" />
</form>
               
			</div>
				
			<div id="tabarea">


						<div id="tabmenu">
                        
						<table  class="pill">
							<tr>
								
								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>
								
							</tr>
							</table>
						</div>
 			</div>

			
	
			<div class="clr"></div>

			<div id="whitebox">
           
				<div id="whitebox_t"> 
                
					<div id="whitebox_tl"> 
						<div id="whitebox_tr"></div>
					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area"> 
                   
									

						<div id="leftcolumn">
                         
															<div class="module">
			<div>
				<div>
					<div>
											
<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);
	
	});
</script>

<ul class="menuacc" id="accordionMenu"><li class="item677"><a href="../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li><li class="item1099"><a href="../ismbeccb2011-general-info/conference-press.html"><span>Conference Press</span></a></li></ul></li><li class="parent active item690"><a href="../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="../cms_addon/conferences/ismbeccb2011/schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="../cms_addon/conferences/ismbeccb2011/artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="../cms_addon/conferences/ismbeccb2011/highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="../cms_addon/conferences/ismbeccb2011/latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="../cms_addon/conferences/ismbeccb2011/posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="../cms_addon/conferences/ismbeccb2011/oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="../cms_addon/conferences/ismbeccb2011/proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="../cms_addon/conferences/ismbeccb2011/technologytrack.html"><span>Technology Track</span></a></li><li id="current" class="active item909"><a href="workshops.html"><span>Workshops</span></a></li><li class="item1097"><a href="ismbeccb2011-award-winners.html"><span>Award Winners</span></a></li></ul></li><li class="parent item692"><a href="../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="../cms_addon/conferences/ismbeccb2011/sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="../cms_addon/conferences/ismbeccb2011/birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../ismbeccb2011-committee-organizing/ismbeccb2011-conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../ismbeccb2011-committee-organizing/ismbeccb2011-committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="../cms_addon/conferences/ismbeccb2011/presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>
				</div>
			</div>
		</div>
	
												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">
							
							<img id="sig" width="148px" src="../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="satellite-meetings.html">
							Satellite Meetings 
							
							</a>
						</div>
						</div>                                                                                                                 
						
						

												<div id="maincolumn">
													
									<div id="nopad">
										
<table class="contentpaneopen">



<tr>
<td valign="top">
<h2>
	Workshops</h2>
<p>
	 </p>
<p>
	<a href="#w1">Workshop 1: Bioinformatics Core Facilities</a><br />
	<a href="#w2">Workshop 2: Navigating the Granting Jungle</a><br />
	<a href="#w3">Workshop 3: Data Visualization and User Interfaces</a><br />
	<a href="#w4">Workshop 4: Unifying Bio-Resources Descriptors</a><br />
	<a href="#w5">Workshop 5: Workshop on Education in Bioinformatics (WEB 2011)</a><br />
	<a href="#w6">Workshop 6: Genomics for Non-Model Organisms</a></p>
<p>
	 </p>
<div class="subheading1">
	<b><a name="w1"></a>Workshop 1: </b><b>Bioinformatics Core Facilities</b></div>
<div>
	<b> </b></div>
<div>
	<b>Organizer(s):<br />
	</b>Simon Andrews, Babraham Institute, United Kingdom<br />
	Fran Lewitter, Whitehead Institute, United States<br />
	Brent Richter, Partners Healthcare, United States<br />
	David Sexton, Vanderbilt University, United States<b><br />
	</b></div>
<div>
	<a href="http://www.bioteam.net/" target="_blank"><br />
	</a></div>
<p>
	<strong>Date:</strong> Sunday, July 17<br />
	<strong>Time:</strong> 10:45 a.m. &ndash; 12:40 p.m.<br />
	<strong>Room:</strong> Hall N/O</p>
<div>
	<b> </b></div>
<div>
	<b> </b></div>
<div>
	<strong>Workshop Overview</strong></div>
<div>
	 </div>
<div>
	<div>
		This workshop is organized by an international organisation (http://www.bioinfo-core.org) that brings together computational biologists working in and running bioinformatics core facilities. The group was formed in 2002 and draws its membership from over 200 facilities in 23 countries.</div>
	<div>
		 </div>
	<div>
		Bioinformatics core facilities undertake work encompassing the full breadth of modern computational biology. Exposure to varied datasets from many different sources allows us to critically evaluate software and methods from a viewpoint not normally available to research groups.<br />
		 </div>
	<div>
		<span style="color: rgb(0, 0, 255);"><span style="font-size: 11px;"><b>Part A: Analysis Pipelines for high throughput sequencing &ndash; what can we learn, and what do we miss?  (25 minutes of talks, 25 minutes of open discussion moderated by Brent Richter, Partners Healthcare)</b></span></span></div>
	<div>
		 </div>
	<div>
		A major goal of the bioinformatics core facility is to deliver high-quality data and tools to the researcher. Standardised pipelines coming either from instrument manufacturers or dedicated bioinformatics groups can provide a simple way to provide analysed data to researchers. However the unsupervised nature of pipelines leads to the possibility that useful and relevant information may be missed. We will explore the potential benefits of automated analysis, but will also investigate what insights might be missed by relying only on standard pipelines. We will address questions such as:</div>
	<div>
		<span>·<span>         </span></span>Under what circumstances can delivering only the results of a standard pipeline be misleading?</div>
	<div>
		 </div>
	<div>
		<span>·<span>         </span></span>What is the potential value of the intermediate data calculated in larger pipelines?</div>
	<div>
		 </div>
	<div>
		<span>·<span>         </span></span>How can we best balance the value of in-depth investigation with the value derived from this extra effort?</div>
	<div>
		 </div>
	<div>
		<span>·<span>         </span></span>Can pipelines be given to non-bioinformaticians to use in their own data analysis without instruction on the caveats of their use?</div>
	<div>
		 </div>
	<div>
		<span>·<span>         </span></span>Can pipelines be shared within the community without the need to adapt them to each individual environment?</div>
	<div>
		 </div>
	<div>
		<span style="font-size: 12px;"><b>Speakers</b></span></div>
	<div>
		<div>
			<b>Simon Andrews</b> (Head of Bioinformatics, Babraham Institute) will present a series of case studies derived from two tools developed in his group; FastQC &ndash; a tool which analyses raw high throughput sequence data to identify trends and biases which might not appear in later analysis of mapped data, and a repeat mapping pipeline which provides a mechanism by which aggregate data on multiply mapping reads to be collected and analysed.</div>
		<div>
			 </div>
	</div>
	<div>
		<b>Jim Cavalcoli</b> (Bioinformatics Core Director, University of Michigan) will present the results of two RNA-Seq studies, where data that were originally rejected as unmappable was reanalysed to reveal insights about transposable elements and non-standard splicing mechanisms.</div>
	<div>
		 </div>
	<div>
		<b><font size="5"> </font></b></div>
	<div>
		<span style="color: rgb(0, 0, 255);"><span style="font-size: 11px;"><b>Part B: Practical aspects of running a bioinformatics core facility (25 minutes of talks, 25 minutes of open discussion moderated by Fran Lewitter, Whitehead Institute)</b></span></span></div>
	<div>
		 </div>
	<div>
		We will explore the practicalities of providing a bioinformatics service to a large group of researchers. It will look at how such groups are structured, funded, staffed and monitored. The aim of the session is to share best practices and to stimulate discussions on how these facilities can achieve the best scientific results in the most efficient way.<br />
		<span style="font-size: 12px;"><br />
		</span></div>
	<div>
		<span style="font-size: 12px;"><b>Speakers</b></span></div>
	<div>
		 </div>
	<div>
		<b>Matthew Eldridge</b> (Head, Bioinformatics Core, Cancer Research UK, Cambridge Research Institute). The CRI Bioinformatics Core provides computational and statistical support to 21 research groups, with approximately 270 scientists, whose interests&rsquo; range from pure basic research questions to clinical applications with practical benefits for cancer patients.</div>
	<div>
		 </div>
	<div>
		<b>Simon Lin</b> (Bioinformatics Consulting Core Director, Northwestern University) The Northwestern Bioinformatics Core Facility provides analysis, support and design for next generation sequencing, microarrays, proteomics, clinical trial informatics as well as custom web-based database development for basic science and clinical projects. The goal is to provide researchers with in-house expertise of bioinformatics in order to produce studies that ultimately result in publications and grants.</div>
	<div>
		 </div>
</div>
<div>
	<b> </b></div>
<div>
	<span style="font-size: 9px;"><a href="#up">top<br />
	</a></span></div>
<div>
	<b> </b></div>
<div class="subheading1">
	<b><a name="w2"></a></b><strong>Workshop 2: Navigating the Granting Jungle</strong></div>
<p>
	 </p>
<p>
	<strong>Organizer(s): </strong><br />
	Yana Bromberg, Rutgers University, United States<br />
	Magali Michaut, University of Toronto, Canada<br />
	Venkata P. Satagopam, EMBL, Germany<br />
	Andrea Schafferhans, TU Münich, Germany<strong><br />
	</strong></p>
<p>
	<strong>Date:</strong> <span class="Apple-style-span" style="font-weight: normal;">Sunday, July 17</span><br />
	<strong> Time:</strong> <span class="Apple-style-span" style="font-weight: normal;">2:30 p.m. &ndash; 4:25 p.m.</span><br />
	<strong>Room: Hall N/O</strong><strong><b><br />
	</b></strong></p>
<div>
	<strong> </strong></div>
<div>
	<strong>Workshop Overview <br />
	</strong></div>
<div>
	<strong><b> </b></strong>
	<div>
		Many different agencies provide funding for research activities. These exist on national levels, but also fund bi-national or international co-operation. This workshop will help current and future principle investigators from the field of bioinformatics and computational biology to get an overview of funding options and information resources available to find such funding sources. In addition, the workshop will help select those funding schemes that are promising as fitted to the individual situation (career stage, collaborators, etc).</div>
	<div>
		 </div>
	<div>
		The proposed workshop in Grant writing is meant to complement the Tutorial on Grant writing to be given on July 15<sup>th</sup>, 2011. While the tutorial focuses on how to write a grant application, the primary aim of the workshop will be to point researchers to the available funding options. The workshop will consist of a panel discussion and 6 invited presentations (12 min each with time for questions) on the following topics:</div>
	<div>
		<span>A)<span>     </span></span>Overview of funding options (individual grants, group funding, integrated projects, etc.)</div>
	<div>
		<span>B)<span>      </span></span>Experiences and recommendations from the reviewers&rsquo; point of view</div>
	<div>
		<span>C)<span>      </span></span>How to find applicable grants and how to choose which ones to apply for</div>
	<div>
		 </div>
	<div>
		<u> </u></div>
	<div>
		<u>Speakers</u></div>
	<ul type="disc">
		<li>
			<u>Gary Bader</u> is an Assistant Professor at the University of Toronto</li>
		<li>
			<u>Phil Irving</u> is the Head of Grants Services, European Molecular Biology Laboratory (EMBL)</li>
		<li>
			<u>Andrew L. Hufton</u> works at EMBO as an Editor for Molecular Systems Biology Journal</li>
		<li>
			<u>Lawrence Hunter</u> is the Director of the Centre for Computational Pharmacology & Computational Bioscience Program</li>
		<li>
			<u>Michal Linial</u> is the Director of the Sudarsky Center for Computational Biology,  the Hebrew University of Jerusalem, Israel</li>
		<li>
			<u>Alfonso Valencia </u>is the Director of the Spanish National Bioinformatics Institute (INB)</li>
	</ul>
	<div>
		<u>Programme Time Table</u></div>
	<div>
		 </div>
	<table border="1" cellpadding="0" cellspacing="0" style="width: 605px; height: 196px;">
		<tbody>
			<tr>
				<td valign="top">
					<div>
						<b>Time</b></div>
				</td>
				<td valign="top" width="222">
					<div>
						<b>Topic</b></div>
				</td>
				<td valign="top" width="312">
					<div>
						<b>Referent</b></div>
				</td>
			</tr>
			<tr>
				<td valign="top">
					<div>
						2:30 p.m. - 2:55 p.m.</div>
				</td>
				<td valign="top">
					<div>
						Funding availability</div>
				</td>
				<td valign="top">
					<div>
						Andrew L. Hufton, Phil Irving</div>
				</td>
			</tr>
			<tr>
				<td colspan="3" valign="top">
					<div align="center">
						5 min break</div>
				</td>
			</tr>
			<tr>
				<td valign="top">
					<div>
						3:00 p.m. - 3:25 p.m.</div>
				</td>
				<td valign="top">
					<div>
						Reviewer&rsquo;s recommendations</div>
				</td>
				<td valign="top">
					<div>
						Michal Linial, Alfonso Valencia</div>
				</td>
			</tr>
			<tr>
				<td colspan="3" valign="top">
					<div align="center">
						5 min break</div>
				</td>
			</tr>
			<tr>
				<td valign="top">
					<div>
						3:30 p.m. - 3:55 p.m.</div>
				</td>
				<td valign="top">
					<div>
						Speaking from experience</div>
				</td>
				<td valign="top">
					<div>
						Lawrence Hunter, Gary Bader</div>
				</td>
			</tr>
			<tr>
				<td colspan="3" valign="top">
					<div align="center">
						5 min break</div>
				</td>
			</tr>
			<tr>
				<td valign="top">
					<div>
						4:00 p.m. - 4:25 p.m.</div>
				</td>
				<td valign="top">
					<div>
						Finding/selecting funding schemes</div>
				</td>
				<td valign="top">
					<div>
						Panel discussion with the previous speakers</div>
				</td>
			</tr>
		</tbody>
	</table>
	<div>
		 </div>
	<div>
		<u>Qualification of the submitter(s) to organize the workshop</u></div>
	<div>
		 </div>
	<div>
		<b>Yana Bromberg</b> is an assistant professor at Rutgers University, School of Environmental and Biological Sciences. She is the chair of the ISMB poster session, co-organizer of the SNP-SIG, and an active participant on a number of scientific conference and journal review boards. In her 10 years in the field, Yana had been funded through traditional (T15 and R01 mechanisms of the NIH/NLM) and through more exotic sources (SBIR R43, Private funding). In her new position she is looking forward to learning more about the available resources that will support her ongoing research.</div>
	<div>
		 </div>
	<div>
		<b>Magali Michaut</b> is a postdoctoral researcher at the University of Toronto. She has been highly involved with the ISCB Student Council for 4 years, in particular in the organization of the Student Council Symposium (reviewing committee 2008, speaker chair 2009, symposium co-chair 2010, ESCS1 chair 2010). She was one of the elected members (secretary) of the Student Council in 2009, initiated the RSG Europe (secretary) and created and chaired the RSG France.</div>
	<div>
		<br />
		<div>
			<b>Venkata P. Satagopam</b> is a Senior Bioinformatics Scientist at EMBL. He is involved with ISCB student council since formed in 2004, particularly in the organization of several student council symposiums, initiated internships for students from developing nations etc. He is also involved in the organization of the Bioinformatics workshops at EMBL and organizing committee member of the &lsquo;Bioinformatics to Systems Biology 2010 conference&rsquo;.</div>
	</div>
	<div>
		 </div>
	<div>
		<b>Andrea Schafferhans</b> is a postdoctoral researcher at the TU Munich. In her academic and industrial research positions, she has been involved in applications for national and European funding. As a researcher in the transition to independence she is gathering experience in navigating the funding agency jungle &ndash; with a special focus on non-standard careers including industry and family phases.</div>
	<br />
	<div>
		 </div>
	<div>
		<a href="#up"><span style="font-size: 9px;">top<br />
		</span></a></div>
</div>
<div>
	<strong><b> </b></strong></div>
<div class="subheading1">
	<b><a name="w3"></a>Workshop 3: </b><b>Data Visualization and User Interfaces</b></div>
<p>
	 </p>
<p>
	<strong>Organizer(s):</strong><br />
	Nils Gehlenborg, Harvard Medical School, United States<br />
	Lawrence Hunter, University of Colorado, United Kingdom<br />
	Seán O'Donoghue, European Molecular Biology Laboratory, Germany<br />
	James Procter, University of Dundee, United Kingdom</p>
<p>
	<strong>Date:</strong> Monday, July 18<strong><br />
	Time: </strong>10:45 a.m. &ndash; 12:40 p.m.<strong><br />
	Room:</strong> Hall N/O</p>
<div>
	<strong> </strong></div>
<div>
	<strong>Workshop Overview<br />
	</strong></div>
<div>
	<strong> </strong></div>
<div>
	<p>
		This workshop is aimed at ISMB attendees who develop interactive tools or methods to help biologists gain insight from data. Three expert speakers will cover principles, illustrative examples, and resources that can help to improve the effectiveness of data visualization, to improve the usability of user interfaces, and finally to help ensure that users find your tool truly helpful in their research. As biology is becoming increasingly data-driven, and as data volume and complexity increase, the above topics are becoming more relevant and urgent for bioinformatics tool developers. The workshop will consist of three 30 minute talks followed by a 30 minute discussion session, as outlined below.</p>
	<p>
		<span style="color:#0000cd;"><strong>Part A: 10:45 - 11:10    User Interface Design</strong></span> <i>(Speaker: <a href="http://www.cgl.ucsf.edu/home/scooter/">Scooter Morris</a>)</i>. For many computational systems, including bioinformatics tools, ease-of-use can greatly influence the extent to which the end-user community either adopts the tool, or ignores it. This talk will outline key principles that can improve user interfaces, and present several case studies highlighting these principles.</p>
	<p>
		<span style="color:#0000cd;"><strong>Part B: 11:15 - 11:40    Data Visualization</strong></span> <i>(Speaker: <a href="http://www.gehlenborg.com/">Nils Gehlenborg</a>)</i>. Data visualization can be a powerful tool to explore large and complex data sets for hypothesis generation. This talk will outline strategies for the design and implementation of interactive visualizations and highlight successful applications of visualization in bioinformatics. The talk will also address the basic principles of data visualization and discuss some of the challenges that developers face in practice.</p>
	<p>
		<span style="color:#0000cd;"><strong>Part C: 11:45 - 12:10    Visual Analytics</strong></span> <i>(Speaker: <a href="http://www.cc.gatech.edu/~goerg/">Carsten Görg</a>)</i>. The interpretation of biological data is a complex process, involving many different data analysis techniques, often applied in multiple different combinations. Visual analytics tools combine the process of analysis and visualization, and this talk will highlight key tools that enable the researcher to visually analyse and identify significant aspects of their results.</p>
	<p>
		<span style="color:#0000cd;"><strong>Part D: 12:15 - 12:40    Discussion Session</strong></span> <i>(Moderator: <a href="http://odonoghuelab.org/">Seán O&rsquo;Donoghue</a>)</i>. Workshop participants will have the opportunity to discuss practical aspects of data visualization, interface design and visual analytics tools with the speakers. In addition to the above speakers, several developers of widely-used software tools have been invited to be part of this session to answer questions from the audience and to share their insights.</p>
</div>
<div>
	<strong><b> </b></strong></div>
<div>
	<span style="font-size: 9px;"><a href="#up">top<br />
	</a></span></div>
<div>
	<strong><b> </b></strong></div>
<div class="subheading1">
	<a name="w4"></a> <b>Workshop 4: </b><strong>Unifying Bio-Resources Descriptors</strong></div>
<p>
	<br />
	<strong>Organizer(s):</strong><br />
	Dawn Field, NERC, NEBC, United Kingdom<br />
	Pascale Gaudet, Swiss Institute of Bioinformatics, Switzerland<br />
	Susanna-Assunta Sansone, University of Oxford, United Kingdom</p>
<div>
	<strong>Date:</strong>  Monday, July 18</div>
<div>
	<strong>Time:</strong> 2:30 p.m. &ndash; 4:25 p.m.</div>
<div>
	<strong>Room: </strong>Hall N/O</div>
<div>
	<strong><b> </b></strong></div>
<p>
	<strong>Workshop Overview:</strong></p>
<div>
	This workshop will bring together developers, curators, journal editors and researchers to discuss on the growing number of (closely related efforts) developing to <b>catalogues of tools, databases, related data and publications</b>. The focus on the workshop is a strawman uniform system for describing these bio-resources, in particular, indicating in a consistent manner which <b>community-defined</b> <b>standards</b> <span>(</span>minimal information checklists, terminologies and exchange formats<span>) </span>they implement.</div>
<div>
	Such uniform system will i) assist the research and bioinformatics communities to locate and access the information distributed in these bio-resources and ii) inform journal editors and funders, implementing <span><b>data preservation, management and sharing</b> <strong>policies</strong><strong>, when recommending or requiring that certain standards are met and that data is deposited to a public database.</strong></span></div>
<div>
	 </div>
<div>
	<b>Background</b></div>
<div>
	<span>High-throughput approaches in genomics and functional genomics bioscience domain have become ubiquitous in the past decade. Several </span><strong>policies</strong><span> have emerged in response to the increased quantity of data available and the correspondingly large variability in their storage and analysis</span><sup>1</sup><span>. </span>The need to store and share this data helps explaining the explosion in the number and variety of tools and databases that cater to the needs of biological research. <span>Many of those resources have contributed to the development of community-level</span><strong> standards</strong><span> that support the harmonization of the annotation and storing process, so that different data can be comprehensible (in principle), reproducible, but also easily shared, analyzed, compared and integrated. The next challenge is to better integrate those policies and standards and link them to the relevant </span>tools and databases<span> implementing them. The goal is to present researchers with a portfolio of standards and resources to enable use and promote adoption. The grand vision is to ease data management and enable data exchange, shielding researchers from unnecessary complexity, and ultimately facilitate data analysis.</span></div>
<div>
	The <a href="http://biocurator.org/"><span>International Society for Biocuration (ISB)</span></a> and the <a href="http://biosharing.org/"><span>BioSharing</span></a> initiative have already taken steps in this direction by i) <span>producing <b>BioDBcore,</b> a community-defined, uniform, generic description of the core attributes of biological databases and ii) releasing a first prototype of &ldquo;</span>one-stop shop&rdquo; <b>BioSharing </b>catalogue for those seeking data sharing policy documents and information about the standards. BioDBcore has been developed and published with a wider list of contributors, including NAR and DATABASE editors and published in both journals<sup>2</sup> <span>(</span>Gaudet <i>et al.</i> 2011). The catalogue of policies and standards, an outcome of a Special Interest Group (SIG)<sup>3</sup> at ISMB 2010, works to link to exiting complementary portals, such as MIBBI<sup>4</sup>, but also open access resources, such as BMC Research Notes and Nature Preceding<span>s, with documents or publications on standards, but also standards-compliant systems and research data. </span></div>
<div>
	Combined, BioDBcore and BioSharing efforts will enable existing and emerging resources - cataloguing tools and database - to have a uniform system for describing the biological databases and indicating what standards they implement.</div>
<div>
	 </div>
<div>
	<b>Workshop Objectives</b></div>
<div>
	At the workshop we will provide updates as to the status of the progresses of how BioDBcore core attributes and the BioSharing catalogues can be used in combination, to unify the description of bio-resources and seek input as to the future of these resources. In addition, the mechanism to exchange the descriptors will be discussed, providing an opportunity for the many interested participants to engage in community assessment and planning for these new efforts. To engage key stakeholders, we will open discussion to emerging efforts working to incorporate BioDBcore descriptors and link to relevant standards catalogued by BioSharing.</div>
<div>
	 </div>
<div>
	<b>References</b></div>
<div>
	1. Field, Sansone <i>et al</i>. &lsquo;Omics Data Sharing, <a href="http://www.ncbi.nlm.nih.gov/pubmed/19815759"><i><span>Science</span></i> 9, 234</a> (2009).</div>
<div>
	<span style="font-size: 12px;">2. Gaudet, Bairoch, Field, Sansone <i>et al</i>. Towards BioDBcore: a community-defined information specification for biological databases. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21205783"><i>Database</i> 4 2011:baq027</a>; and <a href="http://www.ncbi.nlm.nih.gov/pubmed/21097465"><i>Nucleic Acids Res</i>, 39(Database issue):D7-10</a> (2011).</span></div>
<div>
	<em>3. Field, Sansone et al. </em>Meeting Report: BioSharing at ISMB 2010<b>,</b><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304729" target="_blank"><i><span>Standards in Genomic Sciences</span></i>, 3</a> (2010).</div>
<div>
	4. <a href="http://www.nature.com/nbt/journal/v26/n8/pdf/nbt.1411.pdf"><span>Taylor, Field, Sansone <i>et al</i>. </span></a>MIBBI: A Minimum Information Checklist Resource, <a href="http://www.ncbi.nlm.nih.gov/pubmed/18688244" target="_blank"><i><span>Nat. Biotechnol</span></i>. 26, 889</a> (2008).</div>
<div>
	 </div>
<div>
	<b>Agenda</b></div>
<div>
	<table border="1" cellpadding="1" cellspacing="1" style="width: 617px; height: 371px;">
		<tbody>
			<tr>
				<td>
					<strong>Time</strong></td>
				<td>
					<strong>Presentation Title<br />
					</strong></td>
				<td>
					<strong>Presenting Author<br />
					</strong></td>
			</tr>
			<tr>
				<td>
					2:30 p.m. - 3:00 p.m.</td>
				<td>
					Unifying Bio-Resources Descriptors - Introduction & BioSharing [<a href="#unifying">abstract</a>]</td>
				<td>
					<p>
						Susanna-Assunta Sansone, Sansone, <i>University of Oxford</i></p>
					<p>
						Dawn Field, <i>UK NERC Bioinformatics Data Center</i></p>
				</td>
			</tr>
			<tr>
				<td>
					3:00 p.m. - 3:15 p.m.</td>
				<td>
					BioDBcore: a community-defined information specification for biological databases</td>
				<td>
					Pascale Gaudet, <i>Swiss Institute of Bioinformatics</i></td>
			</tr>
			<tr>
				<td>
					3:15 p.m. - 3:30 p.m.</td>
				<td>
					BioDBcore: moving toward implementation</td>
				<td>
					Philippe Rocca-Serra, <i>University of Oxford</i></td>
			</tr>
			<tr>
				<td>
					3:30 p.m. - 4:25 p.m.</td>
				<td>
					Unifying Bio-Resources Descriptors - Panel Discussion</td>
				<td>
					- Nicolas Le Novère, <a href="http://www.ebi.ac.uk/miriam/main" target="_blank">MIRIAM resource</a><br />
					- Naomi Attar<span>, <i>BioMedCentral<br />
					</i></span>- Myles Axton, <i>Nature Publishing Group<br />
					- </i>Adriaan Klinkenberg, <i>Elsevier</i><br />
					- Rebecca Lawrence, <a href="http://f1000.com/" target="_blank"><i>F1000</i> </a><br />
					- Francis <span>Ouellette</span><span>, </span><a href="http://www.bioinformatics.ca/links_directory/" target="_blank"><span><i>DATABASE</i> and <i>Bioinformatics Link Directory </i></span></a><br />
					- Jeffrey Grethe, <a href="http://www.neuinfo.org/"><i>Neuroscience Information Network</i> </a></td>
			</tr>
		</tbody>
	</table>
</div>
<div>
	 </div>
<div>
	<strong><a name="unifying"></a>Unifying Bio-Resources Descriptors</strong></div>
<div>
	This workshop will bring together developers, curators, journal editors and researchers to discuss on the growing number of (closely related efforts) developing to catalogues of tools, databases, related data and publications. The focus on the workshop is a strawman uniform system for describing these bio-resources, in particular, indicating in a consistent manner which community-defined standards (minimal information checklists, terminologies and exchange formats) they implement.<br />
	<br />
	Such uniform system will i) assist the research and bioinformatics communities to locate and access the information distributed in these bio-resources and ii) inform journal editors and funders, implementing data preservation, management and sharing policies, when recommending or requiring that certain standards are met and that data is deposited to a public database.<br />
	<br />
	BioSharing: <a href="http://biosharing.org/">http://biosharing.org</a><br />
	bioDBcore: <a href="http://biocurator.org/biodbcore.shtml">http://biocurator.org/biodbcore.shtml </a></div>
<div>
	 </div>
<p>
	<span style="font-size: 9px;"><a href="#up">top<br />
	</a></span></p>
<div class="subheading1">
	<strong> <b><a name="w5"></a></b>Workshop 5: Workshop on Education in Bioinformatics (WEB 2011)</strong></div>
<p>
	 </p>
<div>
	<strong>Organizer(s):<span><br />
	</span></strong></div>
<div>
	Michelle Brazas, <span>Ontario Institute for Cancer Research, Canada</span></div>
<div>
	Fran Lewitter, <span>Whitehead Institute for Biomedical Research, United States</span></div>
<div>
	Andrea Schafferhans, TU München, Germany</div>
<div>
	Vicky Schneider, EMBL-EBI, United Kingdom</div>
<div>
	<strong><b> </b></strong></div>
<div>
	<strong>Date: </strong> Tuesday, July 19</div>
<div>
	<strong>Time: </strong>10:45 a.m. &ndash; 12:40 p.m.</div>
<div>
	<strong>Room:</strong>  Hall N/O</div>
<div>
	<strong><b> </b></strong></div>
<div>
	<strong>Workshop Overview:</strong></div>
<div>
	<strong> </strong></div>
<div>
	<div>
		The Workshop on Education in Bioinformatics (WEB 2011) is for ISMB attendees who are involved in training, education and teaching. The workshop will provide a platform to listen from experienced experts and discuss the following topics relevant to the continued advancement of this field:</div>
	<div>
		1)<span>         The key elements required of a good bioinformatics tutorial;</span></div>
	<div>
		2)<span>         How bioinformatics tutorials and training materials can be used in remote training;</span></div>
	<div>
		3)<span>         Sharing of training related materials. </span></div>
	<div>
		 </div>
	<div>
		<div>
			 </div>
		<div>
			<b>Time Table</b></div>
		<table border="1" cellpadding="0" cellspacing="0" style="width: 565px; height: 162px;">
			<tbody>
				<tr>
					<td valign="top" width="130">
						<div>
							<b>Time </b></div>
					</td>
					<td valign="top" width="385">
						<div>
							<b>Speakers/Panellists</b></div>
					</td>
				</tr>
				<tr>
					<td valign="top" width="130">
						<div>
							<strong>Part A:<br />
							</strong></div>
						<div>
							10:45 a.m. - 11:10 a.m.</div>
					</td>
					<td valign="top" width="385">
						Dr. Anna Tramontano<br />
						Dr. Gary Bader</td>
				</tr>
				<tr>
					<td valign="top" width="130">
						<div>
							<strong>Part B:<br />
							</strong></div>
						<div>
							11:15 a.m. - 11:40 a.m.</div>
					</td>
					<td valign="top" width="385">
						<div>
							 </div>
						<div>
							Panel discussion lead by Dr. Fran Lewitter</div>
					</td>
				</tr>
				<tr>
					<td valign="top" width="130">
						<div>
							<strong>Part C:<br />
							</strong></div>
						<div>
							11:45 a.m. - 12:10 p.m.</div>
					</td>
					<td valign="top" width="385">
						<div>
							Dr. Aidan Budd</div>
						<div>
							Dr. Alex Bateman</div>
						<div>
							Dr. Vicky Schneider</div>
					</td>
				</tr>
				<tr>
					<td valign="top" width="130">
						<div>
							<strong>Part D:<br />
							</strong></div>
						<div>
							12:15 p.m. - 12:40 p.m.</div>
					</td>
					<td valign="top" width="385">
						<div>
							Panel discussion lead by Dr. Michelle Brazas (Panel composed of all WEB11 contributing speakers)</div>
					</td>
				</tr>
			</tbody>
		</table>
		<div>
			<strong> </strong></div>
		<div>
			 </div>
	</div>
	<div>
		<span style="color: rgb(0, 0, 255);"><strong>Part A: Effective Bioinformatics Training: what makes a good tutorial</strong></span></div>
	<div>
		Bioinformatics Training is challenging not only because of the interdisciplinary nature of the field and audience, but also because the current rapid technological advances demand new and increasingly complex bioinformatics tools. Educational materials must be constantly updated and produced to meet training needs. Solutions that increase development and dissemination of useful and effective learning tools, and increase access and uptake of bioinformatics training would benefit the bioinformatics community as a whole.</div>
	<div>
		 </div>
	<div>
		Increased exposure of bioinformatics training content in peer-reviewed journals would benefit bioinformatics training efforts as well as the scientific community at large. Experts in Bioinformatics training will present their views. This will be followed by a discussion of what constitutes a &lsquo;good tutorial&rsquo; which would help define the landscape for the development and dissemination of effective bioinformatics training material in the future.</div>
	<div>
		 </div>
	<div>
		<span style="color: rgb(0, 0, 255);"><strong>Part B: Panel Discussion, Gary Bader, Anna Tramontano, Andrea Schafferhans, Moderator: Fran Lewitter</strong></span></div>
	<div>
		 </div>
	<div>
		<span style="color: rgb(0, 0, 255);"><strong>Part C: Effective sharing of training efforts: from the materials to the live session</strong></span></div>
	<div>
		As effective training materials are developed and disseminated, such materials may be used to expand bioinformatics training to researchers in more remote locations where face-to-face training is costly and prohibitive. Speakers will discuss their own experiences with remote training, as well as using Wikipedia and other databases to share training materials. A discussion of how technology enables such training would follow. The aim will be to define the landscape for expanding bioinformatics education and awareness to the wider scientific community.</div>
	<div>
		 </div>
	<div>
		<span style="color: rgb(0, 0, 255);"><strong>Part D: Panel Discussion, Gary Bader, Alex Bateman, Aidan Budd, Vicky Schneider, Anna Tramontano, Moderator: Michelle Brazas</strong></span></div>
	<div>
		 </div>
	<div>
		<strong><br />
		</strong></div>
	<div>
		<strong>Panellist Chairs:</strong></div>
	<div>
		<b>Dr. Fran Lewitter</b></div>
	<div>
		<b>Dr. Michelle Brazas</b></div>
	<div>
		<b> </b></div>
	<div>
		<strong>Confirmed speakers include:</strong></div>
	<div>
		<b>Dr. Gary Bader </b>(University of Toronto)</div>
	<div>
		<b>Dr. Alex Bateman</b> (Wellcome Trust Sanger Institute)</div>
	<div>
		<b>Dr. Aidan Budd</b> (EMBL-Heidelberg)</div>
	<div>
		<b>Dr. Anna Tramontano </b>(Universita La Sapienza)</div>
	<div>
		<b>Dr. Vicky Schneider</b> (EMBL-EBI)</div>
	<div>
		<b><br />
		</b></div>
</div>
<div>
	<a>top</a></div>
<div>
	 </div>
<div>
	<div class="subheading1">
		<b><a name="w6"></a>Workshop 6: </b><strong>Genomics for Non-Model Organisms</strong></div>
	<br />
	<div>
		<strong>Organizer(s):</strong></div>
	Dave Clements, Emory University, United States
	<div>
		<strong> </strong></div>
	<div>
		<strong>Date:  </strong>Tuesday, July 19</div>
	<div>
		<strong>Time:</strong> 2:30 p.m. &ndash; 4:25 p.m.</div>
	<div>
		<strong>Room:  </strong>Hall N/O</div>
	<div>
		<strong><br />
		</strong></div>
	<div>
		<strong>Workshop Overview:<br />
		</strong></div>
	<div>
		<strong> </strong></div>
	<p>
		This workshop will demonstrate implemented, successful approaches to several common challenges when working with genomic data from non-model organisms. Early genomics work focused on well established model organisms and built on the availability of robust resources such as high-quality annotated reference genomes and manually curated online model organism databases.  This workshop will highlight recent advances in working with genomic data when you don't have access to the rich resources that come with well-established model organisms.<br />
		<br />
		<span class="Apple-style-span" style="border-collapse: collapse; font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 15px; ">2:30 p.m. - 2:55 p.m.</span>  Using RAD-seq for genomic and transcriptomic studies of non-model organisms, Julian Catchen<br />
		<span class="Apple-style-span" style="border-collapse: collapse; font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 15px; ">3:00 p.m. - 3:25 p.m.</span> Using RNA-seq for gene annotation, quantitation, and function in non-model organisms, Jeremy Goecks<br />
		<span class="Apple-style-span" style="border-collapse: collapse; font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 15px; ">3:30 p.m. - 3:55 p.m</span>.  Development of a workflow for SNP detection in grapevine species: MAPHiTS, Marc Bras<br />
		<span class="Apple-style-span" style="border-collapse: collapse; font-family: Helvetica, Arial, sans-serif; font-size: 12px; line-height: 15px; ">4:00  p.m. - 4:25 p.m.</span>  Repeatable plant pathology bioinformatic analysis: not everything is NGS data, Peter Cock<br />
		<br />
		<br />
		<span style="color:#0000cd;"><b>Part A: Using RAD-seq for genomic and transcriptomic studies of non-model organisms<br />
		</b></span>Julian Catchen (presenting) and William Cresko, University of Oregon, United States<br />
		<br />
		We have developed novel next generation sequencing protocols for high-throughput genotyping and allele-specific profiling in organisms with or without prior genomic resources. Our technique, Restriction-site Associated DNA sequencing (RAD-seq), focuses high-throughput sequencing effort on regions of the genome adjacent to restriction enzyme recognition sites. RAD-seq enables the construction of genetic maps, genome contig tiling, QTL and association mapping, and studies of evolutionary processes in natural populations. We will discuss 1) the suite of RAD molecular biology techniques with examples from non-model organisms; 2) analytical and computational tools that we have developed, encapsulated in the software package Stacks, for the analysis of RAD-seq; and 3) the major remaining challenges for applying techniques like RAD to studies in non-model organisms.<b><br />
		<span style="color:#0000cd;"><br />
		</span></b><span style="color:#0000cd;"><span><b>Part B: </b></span><b>Using RNA-seq for gene annotation, quantitation, and Function in non-model organisms<br />
		</b></span>Jeremy Goecks, Emory University, United States<br />
		<br />
		A primary use of RNA-seq data from a non-model organism is to gain insight into the organism's genes.  Specifically, it is often useful to annotate a genome assembly with gene locations; also, regardless of whether a genome assembly is available, it is useful to quantitate the expression of genes and predict their likely function. Using multiple tools, including Cufflinks, Trans-ABySS, and Galaxy, we have developed methods and pipelines for performing gene annotation, quantitation, and predicting function. We will discuss a case study where we have analyzed RNA-seq data from four related organisms to characterize the similarities and differences in gene expression amongst the organisms.</p>
	<p>
		<br />
		<font><span style="color: rgb(0, 0, 205);"><b>Part C: Development of a workflow for SNP detection in grapevine species: MAPHiTS</b></span><br />
		Marc Bras, INRA-URGI, Centre de Recherche de Versailles-Grignon, France<br />
		<br />
		A Single-Nucleotide Polymorphism (SNP) is a DNA sequence variation. It can be used as a marker to characterize genetic variations. They can be used to detect complex traits such as those involved in diseases resistance or agronomical performance. The URGI platform developed a workflow for SNPs detection from short reads (MAPHiTS: Mapping Analysis Pipeline for High-Throughput Sequences), integrated in the Galaxy workflow manager. Galaxy allows, through a web page, to chain different tools graphically. In addition, a large number of workflows can be built and shared. MAPHiTS workflow is able to deliver all SNPs and small indels found in the data set and to filter them according to various parameters such as the genome coverage, the allele frequency and pValue.</font><br />
		<br />
		<span style="color:#0000cd;"><span><b>Part D: </b></span></span><span style="color:#0000cd;"><b>Repeatable plant pathology bioinformatic analysis: not everything is NGS data.<br />
		</b></span>Peter Cock (presenting), Leighton Pritchard, James Hutton Institute (formerly SCRI, Scottish Crop Research Institute), United Kingdom<br />
		<br />
		The James Hutton Institute's Plant Pathology programme studies a wide range of biological kingdoms and diverse pathogens of great social impact.  We are interested in high throughput sequencing data, but our principal use for Galaxy is to provide a framework to deliver standardised, reusable components for repeatable bioinformatic analyses of assembled sequences and gene sets. We aim to empower bench scientists to carry out their own complex bioinformatic analyses and construct workflows.  We present examples of the  integration of bioinformatic tools using the Galaxy framework, and common workflows for plant pathology, such as identifying candidate proteins that are delivered by pathogens into their hosts - an important mechanism for pathogens and parasites.</p>
	<p>
		 </p>
	<p>
		<a href="#up"><span style="font-size: 9px;">top<br />
		</span></a></p>
</div>

</td>
</tr>

</table>
<span class="article_separator">&nbsp;</span>

										
                                   	</div>
						</div>
						
						<div class="clr"></div>
					</div>
					<div class="clr"></div>
				</div>

				<div id="whitebox_b">
					<div id="whitebox_bl">
						<div id="whitebox_br"></div>
					</div>
				</div>
			</div>

			<div id="footerspacer"></div>
		</div>

		
	</div>
</div>


</body>

<!-- Mirrored from www.iscb.org/ismbeccb2011-program/workshops by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:32:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8"><!-- /Added by HTTrack -->

<!-- Mirrored from www.iscb.org/archive/conferences/iscb/ismbeccb2011-program/workshops.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:55 GMT -->
</html>
