
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb">


<!-- Mirrored from eccb14.loria.fr/program/workshops/rcam?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:49:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <script type="text/javascript">
var siteurl='../../index.html';
var tmplurl='../../templates/ja_elastica/index.html';
var isRTL = false;
</script>

  <base  />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="keywords" content="ECCB'14, ECCB2014, Computational Biology, Bioinformatics, Conference, Strasbourg, ECCB, European Conference on Computational Biology, ECCB14" />
  <meta name="author" content="Adrien" />
  
  <meta name="description" content="ECCB'14 is the key European computational biology event in 2014 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <meta name="generator" content="Joomla! - Open Source Content Management" />
  <title>ECCB'14 - W12 - Recent Computational Advances in Metagenomics</title>
  <link href="http://eccb14.loria.fr/program/workshops/rcam?tmpl=component&amp;print=1&amp;page=" rel="canonical" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/media/jui/css/icomoon.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/." type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/addons.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/usertools.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/print.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/menu/mega.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/typo.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/modules.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile.css" type="text/css" media="only screen and (max-width:719px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile-port.css" type="text/css" media="only screen and (max-width:479px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-tablet.css" type="text/css" media="only screen and (min-width:720px) and (max-width: 985px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-normal.css" type="text/css" media="only screen and (min-width:986px) and (max-width: 1235px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-wide.css" type="text/css" media="only screen and (min-width:1236px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/menu/mega.css" type="text/css" />
  <script src="http://eccb14.loria.fr/media/system/js/mootools-core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/mootools-more.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/anchor.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/mega.rjd.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery/jquery-1.8.3.min.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.noconflict.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.masonry.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.mediaqueries.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/menu/mega.js" type="text/javascript"></script>
  <script type="text/javascript">
jQuery.noConflict();
  </script>

<!--[if ie]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/templates/ja_elastica/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 


<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=3.0, user-scalable=yes"/>
<meta name="HandheldFriendly" content="true" />


<link href="http://eccb14.loria.fr/templates/ja_elastica/images/favicon.ico" rel="shortcut icon" type="image/x-icon" /></head>

<body id="bd" class="fs3 com_content contentpane">
    
<div id="system-message-container">
<div id="system-message">
</div>
</div>
    <div class="item-page clearfix">
	<h2 class="contentheading">
			W12 - Workshop on Recent Computational Advances in Metagenomics		</h2>

	
<div class="article-tools clearfix">
	<ul class="actions">
			<li>
		<a href="#" onclick="window.print();return false;"><span class="icon-print"></span>&#160;Print&#160;</a>		</li>
		
	</ul>



	
</div>
	<p>The workshop will take place on September 7 <span>at the </span><a href="https://www.google.com/maps/place/Palais+de+la+Musique+et+des+Congrès/@48.597588,7.758298,17z/data=!3m1!4b1!4m2!3m1!1s0x4796c85e13ab47a5:0x50b56a5f14ae00d4" target="_blank">PMC</a><span> (Palais de la Musique et des Congrès, Strasbourg Convention Centre, place de Bordeaux) in the room "<strong>Salon Orangerie</strong>".</span></p>
<p><span style="color: #000000;"><span style="line-height: 1.3em;"><span style="text-decoration: underline;"><strong>Preliminary program</strong> </span>:</span></span></p>
<p><span style="color: #000000;"><span style="line-height: 1.3em;">09:00 - 09:10 Welcome</span></span></p>
<p><span style="color: #000000;"><span style="line-height: 1.3em;">09:10 - 10:10 Keynote </span></span><a href="http://phylodiversity.net/skembel/" target="_blank" style="line-height: 1.3em;">Steven Kembel</a><span style="line-height: 1.3em;"> (University of Québec at Montréal): "</span><strong style="line-height: 1.3em;">Using metagenomics to model community assembly in the plant microbiome</strong><span style="line-height: 1.3em;">"</span></p>
<p><span style="line-height: 1.3em;">10:10 - 10:45 <span style="color: #ff6600;">Pierre Péricard</span>: "</span><strong style="line-height: 1.3em;"><a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Pericard_ECCB_2014.pdf">SortMeRNA 2: ribosomal RNA classification for taxonomic assignation</a>" </strong>(see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">10:45 - 11:15 Coffee break</span></p>
<p><span style="line-height: 1.3em;">11:15 - 11:50 <span style="color: #ff6600;">Simon Foucart</span>: "</span><strong style="line-height: 1.3em;"><a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Foucart_ECCB_2014.pdf">Quikr &amp; WGSQuikr: Rapid Bacterial Community Reconstruction Via Compressive Sensing</a>" </strong> (see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">11:50 - 12:25 <span style="color: #ff6600;">Kévin Vervier</span>: "</span><strong style="line-height: 1.3em;"><a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Vervier_ECCB_2014.pdf">Towards Large-scale Machine Learning for Metagenomics Sequence Classification</a>" </strong> (see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">12:25 - 13:00 <span style="color: #ff6600;">Frédéric Mahé</span>: "</span><strong style="line-height: 1.3em;"><a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Mahe_ECCB_2014.pdf">Swarm: robust and fast clustering method for amplicon-based studies</a>" </strong> (see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">13:00 - 14h15 Lunch</span></p>
<p><span style="line-height: 1.3em;">14:14 - 15:15 Keynote </span><a href="http://www.uts.edu.au/staff/aaron.darling" target="_blank" style="line-height: 1.3em;">Aaron Darling</a><span style="line-height: 1.3em;"> (University of Technology, Sydney): "</span><strong style="line-height: 1.3em;">Toward resolving the fine scale genetic structure of microbial </strong><strong style="line-height: 1.3em;">populations: a metagenomic Hi-C approach</strong><span style="line-height: 1.3em;">"</span></p>
<p><span style="line-height: 1.3em;">15:15 - 15:50 <span style="color: #ff6600;">Edi Prifti</span>: "</span><strong style="line-height: 1.3em;"><a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Prifti_ECCB_2014.pdf">Quantitative metagenomics: form reads to biomarkers</a>" </strong>(see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">15:50 - 16:20 Coffee break</span></p>
<p><span style="line-height: 1.3em;">16:20 - 16:55 <span style="color: #ff6600;">Frédéric Plewniak</span>: "</span><strong style="line-height: 1.3em;"><a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Plewniak_ECCB_2014.pdf">Metagenome-scale metabolic network reconstruction</a>" </strong> (see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">16:55 - 17:30 <span style="color: #ff6600;">Clovis Galiez</span>: </span><strong style="line-height: 1.3em;">"<a href="http://genome.jouy.inra.fr/~mmariadasso/downloads/eccb14/Galliez_ECCB_2014.pdf">Identifying distant homologous viral sequences in metagenomes using protein structure information</a>" </strong> (see abstract and co-authors below)</p>
<p><span style="line-height: 1.3em;">17:30 Closing remarks</span></p>
<p> </p>
<p><strong style="line-height: 1.3em;"> </strong></p>
<p><strong style="line-height: 1.3em;">Objectives</strong><span style="text-decoration: underline; line-height: 1.3em;">:</span></p>
<p>This workshop aims at promoting discussions and collaborations between biologists (modelers), computer scientists and applied-mathematicians involved in metagenomics and/or metatranscriptomics studies, either in the bioinformatics or statistical aspects of such analysis.</p>
<p>Metagenomics studies refer to analyses based on high-throughput sequencing of environmental samples and microbial ecosystems. Both marker-gene (16S, 18S, ITS, ...) and whole-genome strategies will be adressed to cover a wide array of question ranging from quantifying the microbial diversity in order to find structuring factors to assessing the functional role of microbial communities.</p>
<p>The workshop will provide an overview of the state-of-the-art methods currently used in metaomics including comparative metagenomics and metatranscriptomics. At the other end of the spectrum, case-studies will illustrate how these methods produce insightful biological knowledge.</p>
<p><span style="text-decoration: underline;"><strong>Scope</strong>:</span></p>
<p>The program of this workshop will consist mainly of presentations of refereed papers (30 minutes) and 2 invited talks (50 minutes). Contributions are welcomed on all aspects about bioinformatics and statistical analyses of metaomics datasets, including, but not limited to:</p>
<ul>
<li><span style="line-height: 1.3em;">data reduction,</span></li>
<li><span style="line-height: 1.3em;">comparative metaomics,</span></li>
<li><span style="line-height: 1.3em;">models for community assembly,</span></li>
<li><span style="line-height: 1.3em;">informal methods (that could be candidate to formalization),</span></li>
<li><span style="line-height: 1.3em;">challenges for tackling the complexity of metaomics datasets.</span></li>
</ul>
<p><span style="text-decoration: underline;"><strong>Keynote speakers</strong></span>:</p>
<ul>
<li><a href="http://phylodiversity.net/skembel/" target="_blank">Steven Kembel</a> (University of Québec at Montréal): "<strong>Using metagenomics to model community assembly in the plant microbiome</strong>"</li>
<li><a href="http://www.uts.edu.au/staff/aaron.darling" target="_blank">Aaron Darling</a> (University of Technology, Sydney): "<strong>Toward resolving the fine scale genetic structure of microbial </strong><span><strong>populations: a metagenomic Hi-C approach</strong>"</span></li>
</ul>
<p><span><span style="text-decoration: underline;"><strong>Accepted contributions:</strong></span></span></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">Quikr &amp; WGSQuikr: Rapid Bacterial Community Reconstruction Via Compressive Sensing</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">D. Koslicki, S. Foucart and G. Rosen</span></p>
<p style="line-height: 1.3em; padding-left: 30px;"><span style="line-height: 1.3em;">Short abstact : </span></p>
<p style="line-height: 1.3em; padding-left: 30px;"><em><span style="line-height: 1.3em;">Many metagenomic studies compare hundreds to thousands of environmental and health-related samples by extracting and sequencing their DNA. However, one of the first steps - to determine what bacteria are actually in the sample - can be a computationally time-consuming task since most methods rely on computing the classification of each individual read out of tens to hundreds of thousands of reads. We introduce Quikr: a QUadratic, K-mer based, Iterative, Reconstruction method which computes a vector of taxonomic assignments and their proportions in the sample using an optimization technique motivated from the mathematical theory of compressive sensing. On both simulated and actual biological data, we demonstrate that Quikr is typically more accurate as well as typically orders of magnitude faster than the most commonly utilized taxonomic assignment techniques for both whole genome techniques (Metaphyler, Metaphlan) and 16S rRNA techniques (the Ribosomal Database Project's Naive Bayesian Classifier). We also show that in general nonnegative L1 minimization can be reduced to a simple nonnegative least squares problems. </span></em><span style="line-height: 1.3em;"> </span></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">Towards Large-scale Machine Learning for Metagenomics Sequence Classification</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">K. Vervier, J.-P. Vert, M.Tournoud, J.-B. Veyrieras and P.Mahé</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Short abstract : </span></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">We investigated the potential of modern, large-scale SVM implementations for taxonomic binning and compared them to similarity-based approaches, like BLAST and TMAP. In practice, compositional approaches offer significant gain in terms of classification time over similarity-based approaches while achieving comparable performances in terms of classification accuracy. Compositional approaches must be trained on a set of sequences with known taxonomic labels, typically obtained by sampling fragments from reference genomes.</span></em></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">We considered a reference database with 356 complete genome sequences from 52 bacterial species to mimic the expected flora of a Human respiratory sample and simulated test sets of around 130k Roche 454 and IonTorrent PGM reads. We observed that increasing the number of training fragments (up to five millions) and longer k-mers (up to k=15) improved the accuracy of SVM models to values similar to TMAP. In terms of speed, the best SVM was at least 17 times faster than TMAP and took no more than 3 minutes to classify the 130k test sequences on a single core. These first results demonstrate the potential of SVM-based methods with massive training set for sequence classification in metagenomics.</span></em></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">Swarm: robust and fast clustering method for amplicon-based studies</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"> F. Mahé, T. Rognes, C. Quince, C. de Vargas and M. Dunthorn</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Short abstact : </span></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">Popular de novo amplicon clustering methods suffer from two fundamental flaws: arbitrary global clustering thresholds, and input-order dependency induced by centroid selection. Swarm was developed to address these issues by first clustering nearly identical amplicons iteratively using a local threshold, and then by using clusters' internal structure and amplicon abundances to refine its results. This fast, scalable, and input-order independent approach reduces the influence of clustering parameters and produces robust operational taxonomic units, improving the amount of meaningful biological information that can be extracted from amplicon-based studies.</span></em></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">SortMeRNA 2: ribosomal RNA classification for taxonomic assignation</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"> E. Kopylova, L. Noé, P. Pericard, M. Salson and H. Touzet</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Short abstact : </span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"><em><span>We have developed SortMeRNA, a software designed to filter ribosomal reads from metatranscriptomic or metagenomic data. It is capable of handling large data sets and sorting out all fragments matching to a database of annotated ribosomal RNA sequences from the three-domain system with high sensitivity and a low running time. <br /><br />We propose a new version, SortMeRNA2, with extended functionalities for improved data analysis. Most importantly, it can now perform sequence alignments to any ribosomal RNA database, which allows the user to study the taxonomic content of a microbial sample. For that, we have developed an alignment strategy based on approximate seeds and seed extension using a variant of the Longest Increasing Subsequence. SortMeRNA2 also applies statistical analysis to evaluate the significance of an alignment, based on the E-value, which confers a great accuracy to the program. </span></em></span></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">Quantitative metagenomics: form reads to biomarkers</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">E. Prifti, E.Le Chatelier, N. Pons, M. Almeida, A. Ghozlane, F. Plaza, N. Aram Gaye, P. Leonard, J.-M. Batto and D. Erlich</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Short abstact : </span></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">The study of complex microbial ecosystems has been increasingly improved with the advent of metagenomics. The role of gut microbiota in human health and disease has received unprecedented attention over the past few years and several complex chronic diseases have been associated with gut microbiota. Even though the number of metagenomics data-mining tools is growing many issues concerning data processing and statistical analyses are still to be tackled. Here we discuss our experience with different data processing techniques and analytical approaches that have been proposed or adapted to explore quantitative metagenomics data in identifying gut microbial biomarkers associated with complex diseases. We illustrate the application of these different approaches, implemented in a suite of tools (Meteor Studio, MetaOMineR and Metaprof), to real gut microbiome data in different studies.</span></em><span style="line-height: 1.3em;"> </span></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">Identifying distant homologous viral sequences in metagenomes using protein structure information</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"> M. Boccara, M. Carpentier, J. Chomilier, F. Coste, C. Galiez, J. Pothier and A. Veluchamy</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Short abstact : </span></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">It is estimated that marine viruses daily kill about 20% of the ocean biomass. Identifying them in water samples is thus a biological issue of great importance. The metagenomic approach for virus identification is a challenging task: standard homology searches usualy fail to identify them especially because of the large amount of mutations in their sequences.<br /> The PEPS VAG project aims at establishing a novel methodology that uses protein structures as extra-information in order to annotate metagenomes without relying only on sequence homology.<br /> In the context of the first experiments made on one metagenome of the TARA Ocean Project, we use the structures of capsid proteins to infer signatures in the amino-acid sequence that are characteristic of their fold. We will present here the methodology, the first experiments and the on-going improvements</span></em><span style="line-height: 1.3em;"> </span></p>
<ul>
<li><strong style="line-height: 1.3em;"><span style="line-height: 1.3em;">Metagenome-scale metabolic network reconstruction</span></strong></li>
</ul>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"> F. Guktnecht, G. Collet, J. Cambefort, J. Andres, S. Prigent, D. Eveillard, A. Siegel, F. Plewniak</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Short abstact : </span></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">Genome-scale metabolic networks have been successfully used to understand a wide range of organism's physiology from </span>Escherichia coli<span style="line-height: 1.3em;"> to </span>Homo sapiens<span style="line-height: 1.3em;">. </span><span style="line-height: 1.3em;">However, despite their interest, individual genomes remain difficult to extract from their natural environment, which seriously limits our ability to model the metabolism and metabolic interactions within natural ecosystems. We present herein the reconstruction of a metabolic network model of the microbial community of Carnoulès Acid Mine Drainage. This community was shown to be intricate and dominated by seven bacterial strains and a photosynthetic protist, </span>Euglena mutabilis<span style="line-height: 1.3em;">.</span></em></p>
<p style="padding-left: 30px;"><em><span style="line-height: 1.3em;">The complete network was built from a combination of transcriptomic, metabolomic, metagenomic and metatranscriptomic data allowing investigating all potential interactions between the partners by the sake of introduction of hypothetical exchange reactions to link the non-produced reactants of every partner to the corresponding metabolite produced in others. </span><span style="line-height: 1.3em;">Although more elaborate analyses still remain to be done in order to identify in detail the actual exchanges at stake, preliminary observations suggest that metabolic networks reconstructed at the scale of a whole microbial community from metagenomic data may provide insights into the potential interactions between its members.</span></em></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"> </span></p>
<p><span><span style="text-decoration: underline;"><strong>Submission</strong></span>:</span></p>
<p><span style="font-family: tahoma, arial, helvetica, sans-serif;">Please <span style="font-size: small;">send</span> the abstract to 
 <script type='text/javascript'>
 <!--
 var prefix = '&#109;a' + 'i&#108;' + '&#116;o';
 var path = 'hr' + 'ef' + '=';
 var addy89169 = 'm&#101;t&#97;g&#101;n&#111;m&#105;cs-&#101;ccb14' + '&#64;';
 addy89169 = addy89169 + 'j&#111;&#117;y' + '&#46;' + '&#105;nr&#97;' + '&#46;' + 'fr';
 document.write('<a ' + path + '\'' + prefix + ':' + addy89169 + '\'>');
 document.write(addy89169);
 document.write('<\/a>');
 //-->\n </script><script type='text/javascript'>
 <!--
 document.write('<span style=\'display: none;\'>');
 //-->
 </script>This email address is being protected from spambots. You need JavaScript enabled to view it.
 <script type='text/javascript'>
 <!--
 document.write('</');
 document.write('span>');
 //-->
 </script></span></p>
<p><span style="font-family: tahoma, arial, helvetica, sans-serif;">All submitted abstracts will be peer-reviewed by</span> the program committee.</p>
<p><span style="line-height: 1.3em;">Submitted abstracts should not exceed 2 pages including bibliography,</span></p>
<p>Abstracts must be written and presented in English. They may describe original work that has been published or that is simultaneously<br />submitted to a journal, conference, or workshop with refereed proceedings.</p>
<p><strong><span style="text-decoration: underline;">Program co-chairs</span>:</strong></p>
<ul>
<li><span style="line-height: 1.3em;">Sophie Schbath (INRA/MIG, Jouy-en-Josas, France)</span></li>
<li><span style="line-height: 1.3em;">Valentin Loux (INRA/MIG, Jouy-en-Josas, France)</span></li>
<li><span style="line-height: 1.3em;">Mahendra Mariadassou (INRA/MIG, Jouy-en-Josas, France)</span></li>
</ul>
<p><span> <span style="text-decoration: underline;"><strong>Sponsor</strong></span>:</span></p>
<ul>
<li><em>Meta-omics of Microbial Ecosystems (M2E) </em>metaprogramme from <a href="http://inra.fr/en" target="_blank">INRA</a>, the french National Institute for Agricultural Research</li>
</ul>
		

	</div>
</body>


<!-- Mirrored from eccb14.loria.fr/program/workshops/rcam?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:49:19 GMT -->
</html>