<html>
<head>
<title>List of Accepted Papers</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">

<!--
<link type="text/css" rel="StyleSheet" href="submission_accepted.cgi_files/easy.css">
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"></head><body bgcolor="white">
<script src="submission_accepted.cgi_files/easy.js"> </script><script>easyState.user='g71ufxin4Gsq19zHP4Bf';easyState.role='C';easyState.track=1;</script><div class="menu"><span class="menuheading">ECCB 2006 (chair)</span><br> <a onmouseover="top.status='list of submissions';return true;" href="http://www.easychair.org/ECCB06/submissions_show.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Submissions</a> | <a onmouseover="top.status='reviews for all papers';return true;" href="http://www.easychair.org/ECCB06/review_all.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">All&nbsp;Reviews</a> | <a onmouseover="top.status='current status of all papers';return true;" href="http://www.easychair.org/ECCB06/status.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Status</a> | <a onmouseover="top.status='assign papers to PC members for reviewing';return true;" href="http://www.easychair.org/ECCB06/assignment.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Assign&nbsp;Reviews</a> | <a onmouseover="top.status='view or update information about PC members';return true;" href="http://www.easychair.org/ECCB06/pc_show.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">PC</a> | <a onmouseover="top.status='list of recent (all) events';return true;" href="http://www.easychair.org/ECCB06/events.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Events</a> | <a onmouseover="top.status='list of recent (all) emails sent by EasyChair';return true;" href="http://www.easychair.org/ECCB06/email_show_all.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Letters</a> | <a onmouseover="top.status='other facilities';return true;" href="http://www.easychair.org/ECCB06/other.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Other</a> | <a onmouseover="top.status='view or update EasyChair configuration';return true;" href="http://www.easychair.org/ECCB06/configure.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Configure</a> | <a onmouseover="top.status='your account information and updates';return true;" href="http://www.easychair.org/ECCB06/account.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">My&nbsp;Account</a> | <a onmouseover="top.status='login with a different role';return true;" href="http://www.easychair.org/ECCB06/login_as.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Login&nbsp;As</a> | <a onmouseover="top.status='exit EasyChair';return true;" href="http://www.easychair.org/ECCB06/logout.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;"><font color="orange">Logout</font></a></div><h2 class="blue">List of Accepted Papers</h2><form method="post" action="submission_accepted.cgi" enctype="multipart/form-data">
<input name="eu" value="g71ufxin4Gsq19zHP4Bf" type="hidden">
<input name="es" value="C1" type="hidden">
<input tabindex="1" name="submit" value="Hide Abstracts" class="redbutton" type="submit">
<div></div></form>
--><br>
					<br>
					<table border="0" cellpadding="3" cellspacing="0"><tbody>
							<tr>
								<td bgcolor="#d9d9d9">
									<p><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Sequence Analysis I,</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Sun., January 21, 20:30-21:30</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br>
											</b></font></p>
								</td>
							</tr>
							<tr>
								<td><a id="36" name="36"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Multiple Alignment by Sequence Annealing<br/>
													</b><a target="_top" href="http://www.cs.berkeley.edu/%7Esariel">Ariel S. Schwartz</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">EECS Computer Science Division, UC Berkeley, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												<a target="_top" href="http://math.berkeley.edu/%7Elpachter/">Lior Pachter</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Mathematics, UC Berkeley, USA</font><br>
								</td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
										We introduce a novel approach to multiple alignment that is based on an algorithm for rapidly checking whether single matches are consistent with a partial multiple alignment. This leads to a sequence annealing algorithm, which is an incremental method for building multiple sequence alignments one match at a time. Our approach improves significantly on the standard progressive alignment approach to multiple alignment.<br>
										<br/>
											Results:<br/>
												The sequence annealing algorithm performs well on benchmark test sets of protein sequences. It is not only sensitive, but also specific, drastically reducing the number of incorrectly aligned residues in comparison to other programs. The method allows for adjustment of the sensitivity/specificity tradeoff, and can be used to reliably identify homologous regions among protein sequences.<br/>
												<br/>
											Availability:<br/>
												An implementation of the sequence annealing algorithm is available at <a href="http://bio.math.berkeley.edu/amap/" target="_blank">http://bio.math.berkeley.edu/amap/</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="21" name="21"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Tandem Repeats over the Edit Distance<br/>
													</b><a target="_top" href="http://www.sci.brooklyn.cuny.edu/%7Esokol">Dina Sokol</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Justin Tojeira, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Brooklyn College of the City University of New York, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												Gary Benson, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Boston University, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>A tandem repeat in DNA is a sequence of two or more contiguous, approximate copies of a pattern of nucleotides. Tandem repeats occur in the genomes of both eukaryotic and prokaryotic organisms. They are important in numerous fields including disease diagnosis, mapping studies, human identity testing (DNA fingerprinting), sequence homology, and population studies. Although tandem repeats have been used by biologists for many years, there are few tools available for performing an exhaustive search for all tandem repeats in a given sequence. In this paper we describe an efficient algorithm for finding all tandem repeats within a sequence, under the edit distance measure.<br>
										The
contributions of this paper are two-fold: theoretical and practical. We
present a precise definition for tandem repeats over the edit distance,
and an efficient, deterministic algorithm for finding these repeats.
The algorithm has been implemented in C++, and a website for the
program is under construction. The use of this tool will assist
biologists in discovering new ways that tandem repeats affect both the
structure and function of DNA and protein molecules.<br/>
												<br/>
										<hr>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Molecular Recognition and Computer Aided Drug Design,</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Mon.,  January 22, 10:00-11:30<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="7" name="7"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Analysis of Binding Site Similarity, Small-Molecule Similarity and Experimental Binding Profiles in the Human Cytosolic Sulfotransferase Family</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br>
										</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>
													</b><a target="_top" href="https://www.ebi.ac.uk/%7Erafi">Rafael J. Najmanovich</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">European Bioinformatics Institute (EMBL-EBI), UK and Structural Genomics Consortium,  University of Toronto, Canada</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
										Richard J. Morris </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Janet M. Thornton, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">European Bioinformatics Institute (EMBL-EBI), UK</font><br/>
										<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Abdellah Allali-Hassani, Ludmila Dombrovsky </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Masoud Vedadi, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Structural Genomics Consortium, University of Toronto, Canada</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Patricia W. Pan, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Medical Biophysics Department and Structural Genomics Consortium, University of Toronto, Canada</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Alexander N. Plotnikov, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Physiology Department and Structural Genomics Consortium, University of Toronto, Canada</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												Aled Edwards </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Cheryl Arrowsmith</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Structural Genomics Consortium and Banting &amp; Best Department of Medical Research, University of Toronto, Canada<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/> In the present work we combine computational analysis and experimental data to explore the extent to which binding site similarities between members of the human cytosolic sulfotransferase family correlate with small-molecule binding profiles. Conversely, from a small-molecule point of view, we explore the extent to which structural similarities between small--molecules correlate to protein binding profiles.<br>
										<br/>
											Results:<br/>
										 The comparison of binding site structural similarities and small-molecule binding profiles shows that proteins with similar small-molecule binding profiles tend to have a higher degree of binding site similarity but the later is not sufficient to predict small-molecule binding patterns, highlighting the difficulty of predicting small-molecule binding patterns from sequence or structure. Likewise, from a small-molecule perspective, small-molecules with similar protein binding profiles tend to be topologically similar but topological similarity is not sufficient to predict their protein binding patterns. These observations have important consequences for function prediction and drug design</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="14" name="14"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Electrostatic Potentials of Proteins in Water: A Structured Continuum Approach<br/>
													</b>Andreas Hildebrandt </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.bioinf.uni-sb.de/HPL/">Hans-Peter Lenhof</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Center for Bioinformatics, Saarland University, Germany</font><br/>
											<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Ralf Blossey, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Interdisciplinary Research Institute Lille, Germany</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										Sergej Rjasanow, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Dept. of Mathematics, Saarland University, Germany</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										Oliver Kohlbacher, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Dept. for Simulation of Biological Systems, University T&#252;bingen, Germany</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Electrostatic interactions play a crucial role in many biomolecular processes, including molecular recognition and binding. Biomolecular electrostatics is modulated to a large extent by the water surrounding the molecules. Here, we present a novel approach to the computation of electrostatic potentials which allows the inclusion of water structure into the classical theory of continuum electrostatics. Based on our recent purely differential formulation of nonlocal electrostatics (Hildebrandt et al., Phys. Rev. Lett. {\bf 93} (2004) 108104) we have developed a new algorithm for its efficient numerical solution. The key component of this algorithm is a boundary element solver, having the same computational complexity as established boundary element methods for local continuum electrostatics.<br>
										<br/>
											This allows, for the first time, the computation of electrostatic potentials and interactions of large biomolecular systems immersed in water including effects of the solvent's structure in a continuum description. We illustrate the applicability of our approach with two examples, the enzymes trypsin and acetylcholinesterase (AChE).<br>
										<br/>
											The approach is applicable to all problems requiring precise prediction of electrostatic interactions in water, such as protein-ligand and protein-protein docking, folding, and chromatin regulation. Initial results indicate that this approach may shed a new light on biomolecular electrostatics and on aspects of molecular recognition that classical local electrostatics cannot reveal.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="11" name="11"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>EBIMed - Text Crunching to Gather Facts for Proteins from Medline<br/>
													</b><a target="_top" href="https://www.ebi.ac.uk/Rebholz">Dietrich Rebholz-Schuhmann</a>, Harald Kirsch, </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Miguel Arregui, </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Sylvain Gaudan, </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Mark Riethoven, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">  </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Peter Stoehr, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">European Bioinformatics Institute, UK<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>To allow efficient and systematic retrieval of statements from Medline, we have developed EBIMed, a service that combines document retrieval with co-occurrence-based analysis of Medline abstracts. Upon keyword query, EBIMed retrieves the abstracts from EMBL-EBI's installation of Medline and filters for sentences that contain biomedical terminology maintained in public bioinformatics resources. The extracted sentences and terminology are used to generate an overview table on proteins, GO annotations, drugs and species used in the same biological context. All terms in retrieved abstracts and extracted sentences are linked to their entries in biomedical databases.<br>
											We assessed the quality of the identification of terms and relations in the retrieved sentences. More than 90% of the protein names found indeed represented a protein. According to the analysis of 4 protein/protein pairs from the Wnt pathway we estimated that 37% of the statements containing such a pair mentioned a meaningful interaction and clarified the interaction of Dkk with LRP.<br>
										We conclude that
EBIMed improves access to information where proteins and drugs are
involved in the same biological process, e.g. statements with GO
annotations of proteins, protein/protein interactions and effects of
drugs on proteins.<br>
											Access: <a href="https://www.ebi.ac.uk/Rebholz-srv/ebimed" target="_blank">https://www.ebi.ac.uk/Rebholz-srv/ebimed</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Computational Genomics I, </b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Mon., January 22, 12:00-14:00</font><br>
								</td>
							</tr>
							<tr>
								<td><a name="1"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Optimization of Probe Coverage for High-Resolution Oligonucleotide aCGH</b></font><br/>
											<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www.cs.technion.ac.il/%7Edlipson/">Doron Lipson</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Computer Science Dept., Technion, Israel<br/>
												</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Zohar Yakhini, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Agilent Technologies, Israel<br/>
												</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Yonatan Aumann, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Computer Science Dept., Bar-Ilan University, Israel</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:</b><br>
										Motivation:<br/>
										The resolution at which genomic alterations can be mapped by means of oligonucleotide aCGH (array-based Comparative Genomic Hybridization) is limited by two factors: the availability of high-quality probes for the the target genomic sequence and the array real-estate. Optimization of the probe selection process is required for arrays that are designed to probe specific genomic regions in very high resolution without compromising probe quality constraints.<br>
										<br/>
											Results:<br/>
										In this paper we describe a well-defined optimization problem associated with the problem of probe selection for high-resolution aCGH arrays. We propose the &quot;whenever possible epsilon-cover&quot; as a formulation that faithfully captures the requirement of probe selection problem, and provide a fast randomized algorithm that solves the optimization problem in O(n log n) time, as well as a deterministic algorithm with the same asymptotic performance. We apply the method in a typical high-definition array design scenario and demonstrate its superiority with respect to alternative approaches. <br/>
										<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="8" name="8"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Simultaneous Alignment and Annotation of cis-Regulatory Regions<br/>
													</b>Abha Singh Bais, Steffen Grossmann, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Martin Vingron, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Max Planck Institute for Molecular Genetics, Germany<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation: <br>
											Current methods that annotate conserved transcription factor binding sites in an alignment of two regulatory regions perform the alignment and annotation step separately and combine the results in the end. If the site descriptions are weak or the sequence similarity is low, the local gap structure of the alignment poses a problem in detecting the conserved sites. It is therefore desirable to have an approach that is able to simultaneously consider the alignment as well as possibly matching site locations.<br>
										<br>
											Results:<br>
										With SimAnn we have
developed a tool that serves exactly this purpose. By combining the
annotation step and the alignment of the two sequences into one
algorithm, it detects conserved sites more clearly. It has the
additional advantage that all parameters are calculated based on
statistical considerations.&nbsp;&nbsp;This allows for its successful
application with any binding site model of interest.&nbsp;&nbsp;We
present the algorithm and the approach for parameter selection and
compare its performance with that of other, non-simultaneous methods on
both simulated and real data.<br>
										<br>
											Availability:<br>
										A command-line
based C++ implementation of SimAnn is available from the authors upon
request. Additionally, we provide Perl scripts for calculating the
input parameters based on statistical considerations.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="10" name="10"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Genetic Code Symmetry and Efficient Design of GC-Constrained Coding Sequences<br/>
													</b>Matan Gavish, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Physics, Tel Aviv University, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										 Amnon Peled, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Goldyne Savad Gene Therapy Institute, Hadassah, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										 Benny Chor, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science, Tel Aviv University, Israel<br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation: <br>
										Cloning
of long DNA sequences (40-60 bases) into phage display libraries using
PCR is a low efficiency process, in which PCR is used to incorporate a
DNA insert, coding for a certain peptide, into the amplified sequence.
The PCR efficiency in this process is strongly affected by the
distribution of G-C bases in the amplified sequence. As any DNA insert
coding for the target peptide may be attempted, there is a flexibility
in choosing part of the amplified sequence. Since the number of inserts
coding for the same peptide is exponential in the peptide length, a
computational problem naturally arises - that of efficiently finding an
insert, whose parameters are optimal for PCR cloning.<br>
										<br/>
										Results: <br>
										The
GC distribution requirements are formulated as a search problem. We
developed an efficient, linear time &#8220;one pass&#8221; algorithm for this
problem. Interestingly, our algorithm strongly relies on a novel
symmetry, which we discovered in the standard genetic code. Most
nonstandard genetic codes examined possess this symmetry as well, yet
some do not. We generalize the search problem and consider the case of
a nonstandard, or arbitrary, genetic code where this symmetry does not
necessary hold. We solve the generalized problem in polynomial, but
non-linear, time.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="33" name="33"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Discovering Tightly Regulated and Differentially Expressed Gene Sets in Whole Genome Expression Data<br/>
												 </b>Chun Ye </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Eleazar Eskin, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of California, San Diego, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
												Recently, a new type of expression data is being collected which aims to measure variation of pathways due to genetic variation. In these datasets, expression profiles are constructed for multiple strains of the same model organism under the same condition. The goal of analyses of this data is to find differences in regulatory patterns due to genetic variation between strains, often without a phenotype of interest in mind. We present a new method based on notions of correlated expression and differential expression to look for significant sets of genes.<br>
										<br>
											Results:<br/>
												When we use categorical phenotype information, as in the Alzheimer's and diabetes datasets, our method finds many of the same gene sets as GSEA. But the notion of correlated gene sets allows us to focus our efforts on biological processes subjected to tight regulation. In murine hematopoietic stem cells, we are able to discover significant gene sets independent of a phenotype of interest. Some of these gene sets are associated with several blood related phenotypes.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9">
									<p><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Networks, Pathways and Systems,</b></font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Mon., January 22, 15:30-18:00<br>
										</font></p>
								</td>
							</tr>
							<br>
							<tr>
								<td><a id="28" name="28"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Biological Network Comparison using Graphlet Degree Distribution<br/>
													</b><a target="_top" href="http://www.cs.utoronto.ca/%7Enatasha/">Natasa Przulj</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Computer Science Department, University of California, Irvine, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br>
											Analogous to biological sequence comparison, comparing cellular networks is an important problem that could provide insight into biological understanding and therapeutics. For technical reasons, comparing large networks is computationally infeasible, and thus heuristics, such as the degree distribution, clustering coefficient, diameter, and relative graphlet frequency distribution have been sought. It is easy to demonstrate that two networks are different by simply showing a short list of properties in which they differ. It is much harder to show that two networks are similar, as it requires demonstrating their similarity in all of their exponentially many properties. Clearly, it is computationally prohibitive to analyze all network properties, but the larger the number of constraints we impose in determining network similarity, the more likely it is that the networks will truly be similar.<br>
										<br>
										Results: <br>
										We introduce a new systematic measure of a network's local structure that imposes a large number of similarity constraints on networks being compared. In particular, we generalize the degree distribution, which measures the number of nodes &quot;touching'' k edges, into distributions measuring the number of nodes &quot;touching'' k &quot;graphlets'', where graphlets are small connected non-isomorphic subgraphs of a large network. Our new measure of network local structure consists of 73 </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">&quot;graphlet degree distributions'' of graphlets with 2, 3, 4, and 5 nodes, but it is easily extendible to a greater number of constraints (i.e, graphlets), if necessary, and the extensions are limited only by the available CPU. Furthermore, we show a way to combine the 73 graphlet degree distributions into a network &quot;agreement'' measure which is a number between 0 and 1, where 1 means that networks have identical distributions and 0 means that they are far apart. Based on this new network agreement measure, we show that almost all of the fourteen eukaryotic PPI networks, including human, resulting from various high-throughput experimental techniques, as well as from curated databases, are better modeled by geometric random graphs than by Erdos-Renyi, random scale-free, or Barabasi-Albert scale-free networks.<br>
										<br>
											Availability: Software executables are available upon request.<br/>
												<br/>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="17" name="17"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Identification of Conserved Protein Complexes Based on a Model of Protein Network Evolution<br/>
													</b>Eitan Hirsh </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Roded Sharan, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science, Tel Aviv University, Israel<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation: <br>
											Data on protein-protein interactions are increasing exponentially. To date, large scale protein interaction networks are available for human and most model species. The arising challenge is to organize these networks into models of cellular machinery. As in other biological domains, a comparative approach provides a powerful basis for addressing this challenge.<br>
										<br>
										Results: <br>
											We develop a probabilistic model for protein complexes that are conserved across two species. The model describes the evolution of conserved protein complexes from an ancestral species by protein interaction attachment and detachment and gene duplication events. <br/>
											We apply our model to search for conserved protein complexes within the protein-protein interaction networks of yeast and fly, which are the largest networks in public databases. We detect 150 conserved complexes that match well known complexes in yeast and are coherent in their functional annotations both in yeast and in fly. In comparison to two previous approaches, our model yields higher specificity and sensitivity levels in protein complex detection.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="2" name="2"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Phylogenetic Reconstruction from Non-Genomic Data<br/>
													</b>Jose C. Clemente </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Kenji Satou, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Japan Advanced Institute of Science and Technology, Japan</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www.lsi.upc.edu/%7Evaliente/">Gabriel Valiente</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Technical University of Catalonia, Spain<br/>
												</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>Recent results related to horizontal gene transfer suggest that phylogenetic reconstruction cannot be determined conclusively from sequence data, resulting in a shift from approaches based on polymorphism information in DNA or protein sequence to studies aimed at understanding the evolution of complete biological processes. The increasing amount of available information on metabolic pathways for several species makes it of greater relevance to understand the similarities and differences among such pathways. These similarities can then be used to infer phylogenetic trees not based exclusively in sequence data, therefore avoiding the previously mentioned problems.<br>
										<br>
										In this paper, we
present a method to assess the structural similarity of metabolic
pathways for several organisms. Our algorithms work by using one of
three possible enzyme similarity measures (hierarchical, information
content, gene ontology), and one of two clustering methods
(neighbor-joining, unweighted pair group method with arithmetic mean),
to produce a phylogenetic tree both in Newick and graphic format. The
web server implementing our algorithms is optimized to answer queries
in linear time.<br>
										<br>
										The software is available for free public use on a web server, at the contact address. It is available on demand in source code form for research use to educational institutions, non-profit research institutes, government research laboratories, and individuals, for non-exclusive use, without the right of the licensee to further redistribute the source code.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="9" name="9"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Similarities and Differences of Gene Expression in Yeast Stress Conditions<br/>
													</b>Oleg Rokhlenko </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Ydo Wexler, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Computer Science Dept., Technion, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Zohar Yakhini, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Agilent Technologies, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation and Methods: <br>
											All living organisms and the survival of all cells critically depend on their ability to sense and quickly adapt to changes in the environment and to other stress conditions. We study stress response mechanisms in Saccharomyces cerevisiae by identifying genes that, according to very stringent criteria, have persistent co-expression under a variety of stress conditions. This is enabled through a fast clique search method applied to the intersection of several co-expression graphs calculated over the data of Gasch et al. [3]. This method exploits the topological characteristics of these graphs.<br>
										<br/>
										Results: <br>
											We observe cliques in the intersection graphs that are much larger than expected under a null model of changing gene identities for different stress conditions but maintaining the co-expression topology within each one. Persistent cliques are analyzed to identify enriched function as well as enriched regulation by a small number of TFs. These TFs therefore characterize a universal and persistent reaction to stress response. We further demonstrate that the vertices (genes) of many cliques in the intersection graphs are colocalized in the yeast genome, to a degree far beyond the random expectation. Co-localization can hypothetically contribute to a quick co-ordinated response. We propose the use of persistent cliques in further study of properties of co-regulation.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="30" name="30"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Efficient Inference on Phylogenetic Trees Using Poisson Regression<br/>
													</b>Saharon Rosset, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">IBM T. J. Watson Research Center, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>We suggest the use of Poisson regression for time inference and hypothesis testing on a bifurcating Phylogenetic tree with known topology. This method is computationally simple and naturally accommodates variable substitution rates across different sites, without requiring estimation of these rates. We identify the assumptions under which this is a maximum likelihood inference approach and show that in some realistic situations --- in particular, when the probability of repeated mutation within each branch of the tree is small --- these assumptions hold with high probability. Our motivating domain is human mitochondrial DNA trees, and we illustrate our method on a problem of estimating the time to most recent common ancestor of all non-African mtDNA, using publicly available data. We test for molecular clock violations using multiple comparisons, and conclude that the global molecular clock hypothesis cannot be rejected based on this data.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Structural Bioinformatics I,</b></font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tue., January 23, 10:00-11:30<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="13" name="13"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Vorolign - Fast Structural Alignment Using Voronoi Contacts<br/>
													</b><a target="_top" href="http://cgi.bio.ifi.lmu.de/%7Ebirzele/">Fabian Birzele</a>, Jan Erik Gewehr, Gergely Csaba, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://-/">Ralf Zimmer</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Practical Informatics and Bioinformatics Group, LMU Munich, Germany<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract: <br/>
										</b>Summary:<br/>
										Vorolign, a fast and flexible structural alignment method for two or more protein structures is introduced. The method aligns protein structures using double dynamic programming and measures the similarity of two residues based on the evolutionary conservation of their corresponding Voronoi-contacts in the protein structure. This similarity function allows to align protein structures even in cases where structural flexibilities exist. Multiple structural alignments are generated from a set of pairwise alignments using a consistency-based, progressive multiple alignment strategy.<br>
										<br>
										Results:<br/>
										The performance of Vorolign is evaluated for several different applications of protein structure comparison, including automatic family detection as well as pairwise and multiple structure alignment. Vorolign accurately detects the correct family, superfamily or fold of a protein with respect to the SCOP classification on a set of difficult target structures. A scan against a database of 4358 proteins takes on average 1 minute per target. The performance of Vorolign in calculating pairwise and multiple alignments is found to be comparable to other pairwise and multiple protein structure alignment methods.<br>
										<br>
										Availability:<br/>
										Vorolign is freely available for academic users as a Web server at <a href="http://www.bio.ifi.lmu.de/Vorolign" target="_blank">http://www.bio.ifi.lmu.de/Vorolign</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="34" name="34"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Prediction and Simulation of Motion in Pairs of Transmembrane Alpha-Helices<br/>
													</b>Angela Enosh, Sarel J. Fleishman, Nir Ben-Tal, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Dan Halperin, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tel Aviv University, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Motivation:<br>
											Motion in transmembrane-proteins plays an essential role in a variety of biological phenomena, including signal transduction, transport of metabolites, ligand binding and molecular recognition of hormones and neurotransmitters. Thus, developing an automated method for predicting and simulating motion in this class of proteins should result in an increased level of understanding of their biochemical mechanisms. We have developed an algorithm for predicting and simulating motion in transmembrane (TM) proteins.<br>
											We used probabilistic motion-planning techniques to sample possible motion paths. For each conformation we computed the buried-surface area between the helices, providing a rudimentary assessment of the free-energy of helix association. Our algorithm considers a wide range of degrees of freedom (dofs) involved in the motion, including external and internal moves. However, in order to handle the vast dimensionality of the problem, we employed some relaxations on these dofs in a way that is unlikely to demolish the native motion that the protein undergoes.<br>
										<br/>
											Results:<br>
											Overexpression of the receptor tyrosine kinase (RTK) ErbB2 was implicated in causing a variety of human cancers. Recently, a molecular mechanism for rotation-coupled activation of the receptor was suggested. We applied our algorithm to investigate the TM domain of this protein, and compared our results to this mechanism. Our algorithm yielded several motion pathways and among them a motion similar to the proposed mechanism was ranked first. Our algorithm simulates the motion including all the dofs involved in this process and can automatically produce movies that demonstrate such motions.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="3" name="3"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Rediscovering Secondary Structures as Network Motifs - An Unsupervised Learning Approach</b><br/> Barak Raveh, Ofer Rahat, Ronen Basri, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Gideon Schreiber, <font size="2">Weizmann Institute of Science, Israel<br>
										</font></font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation: <br/>
											Secondary structures are key descriptors of a protein fold and its topology. In recent years, they facilitated intensive computational tasks for finding structural homologues, fold prediction and protein design. Their popularity stems from an appealing regularity in patterns of geometry and chemistry. However, the definition of secondary structures is of subjective nature. An unsupervised de-novo discovery of these structures would shed light on their nature, and improve the way we use these structures in algorithms of structural bioinformatics.<br>
										<br/>
											Methods:<br/>
										We developed a new method for unsupervised partitioning of undirected graphs, based on patterns of small recurring network motives. Our input was the network of all H-bonds and covalent interactions of protein backbones. This method can be also used for other biological and non-biological networks.<br>
										<br/>
											Results:<br/>
												In a fully unsupervised manner, and without assuming any explicit prior knowledge, we were able to rediscover the existence of conventional alpha-helices, parallel beta-sheets, anti-parallel sheets and loops, as well as various non-conventional hybrid structures. The relation between connectivity and crystallographic temperature factors establishes the existence of novel secondary structures.<br>
											Contact: <a href="mailto:barak.raveh@weizmann.ac.il">barak.raveh@weizmann.ac.il</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Structural Bioinformatics II,</b></font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tue., January 23, 12:00-14:00<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="19" name="19"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>A Tale of Two Tails: Why are Terminal Residues of Proteins Exposed?</b><br/>
											 Etai Jacob </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://faculty.biu.ac.il/%7Eunger/">Ron Unger</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Faculty of Life Sciences, Bar Ilan University, Israel<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/> It is widely known that terminal residues of proteins (i.e. the N and C termini) are predominantly located on the surface of proteins and exposed to the solvent. However, there is no good explanation as to the forces driving this phenomenon. The common explanation that terminal resi-dues are charged, and charged residues prefer to be on the surface, can not explain the magnitude of the phenomenon. Here, we survey a large number of proteins from the PDB in order to explore, quantitatively, this phenomenon, and then we use a lattice model to study the mechanisms involved.<br>
										<br/>
											Results:<br/>
										 The location of the termini was examined for 425 small monomeric proteins (50-200 AA) and it was found that the average solvent accessibility of termini residues is 87.1% compared with 49.2% of charged residues and 35.9% of all residues. Using a cutoff of 50% of the maximal possible exposure, 80.3% of the N-terminal and 86.1% of the C-terminal residues are exposed compared to 32% for all residues. In addition, terminal residues are much more distant from the center of mass of their proteins than other residues. Using a 2D lattice, a large population of model proteins was studied on three levels: structural selection of compact structures, thermodynamic selection of conformations with a pronounced energy gap, and kinetic selection of fast folding proteins using Monte-Carlo simulations. Progressively, each selection raises the proportion of proteins with termini on the surface, resulting in similar proportions to those observed for real proteins.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="23" name="23"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>A Novel Pattern Recognition Algorithm to Classify Membrane Protein Unfolding Pathways with High-Throughput Single Molecule Force Spectroscopy<br/>
													</b>Annalisa Marsico, Dirk Labudde, Tanuj Sapra, Daniel Muller, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Michael Schroeder</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Biotechnologisches Zentrum der TU Dresden, Germany<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br>
											Misfolding of membrane proteins plays an important role in many human diseases such as retinitis pigmentosa, hereditary deafness, and diabetes insipidus. Little is known about membrane proteins as there are only a very few high-resolution structures. Single molecular force spectroscopy is a novel technique, which measures the force necessary to pull a protein out of a membrane. Such force curves contain valuable information on the protein's structure, conformation, and inter- and intra-molecular forces. High-throughput force spectroscopy experiments generate hundreds of force curves including spurious ones and good curves, which correspond to different unfolding pathways.<br>
											Manual analysis of these data is a bottleneck and source of inconsistent and subjective annotation.<br>
										<br/>
											Results:<br>
											We propose a novel algorithm for the identification of spurious curves and curves representing different unfolding pathways. Our algorithm proceeds in three stages: First, we reduce noise in the curves by applying dimension reduction; second, we align the curves with dynamic programming and compute pairwise distances; third, we cluster the curves based on these distances.<br>
											We apply our method to a hand curated dataset of 135 force curves of bacteriorhodopsin mutant P50A. Our algorithm achieves a success rate of 81% distinguishing spurious from good curves and a success rate of 76% classifying unfolding pathways. As a result, we discuss five different unfolding pathways of bacteriorhodopsin including three main unfolding events and several minor ones. Finally, we link folding barriers to the degree of conservation of residues.<br>
											Overall, the algorithm tackles the force spectroscopy bottleneck and leads to more consistent and reproducible results paving the way for high-throughput analysis of structural features of membrane proteins.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="27" name="27"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Using an Alignment of Fragment Strings for Comparing Protein Structures<br/>
													</b><a target="_top" href="http://iddo-friedberg.org/">Iddo Friedberg</a>, Tim Harder, Zhanwen Li, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://godziklab.org/">Adam Godzik</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Burnham Institute for Medical Research, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
										Rachel Kolodny, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Columbia University and Howard Hughes Medical Institute, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												Einat Sitbon, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Weizmann Institute of Science, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
										Most methods which are used to compare protein structures use 3D structural information to do so. At the same time, it has been shown that a 1D string representation of local protein structure retains a degree of structural information. This type of representation can be a powerful tool for protein structure comparison and classification, given the arsenal of sequence comparison tools developed by computational biology. However, in order to do so, there is a need to first understand how much information is contained in various possible 1D representations of protein structure.<br>
										<br>
										Results:<br/>
											 Here we describe the use of a particular structure fragment library, denoted here as KL strings, for the 1D representation of protein structure. Using KL strings, we develop an infrastructure for comparing protein structures with a 1D representation. This study focuses on the added value gained from such a description. We show the new local structure language adds resolution to the traditional three state (helix, strand and coil) secondary structure description, and provides a high degree of accuracy in recognizing structural similarities when used with a pairwise alignment benchmark. The results of this study have immediate applications towards fast structure recognition, and for fold prediction and classification.<br>
										<br>
											Supplementary material: <a href="http://iddo-friedberg.org/ECCB06-supplement" target="_blank">http://iddo-friedberg.org/ECCB06-supplement</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="29" name="29"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>ISIS: Interaction Sites Identified from Sequence<br/>
													</b>Yanay Ofran </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.rostlab.org/">Burkhard Rost</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Columbia University, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
										Large-scale experiments reveal pairs of inter-acting proteins but leave the residues involved in the inter-actions unknown. These interface residues are essential for understanding the mechanism of interaction and are often desired drug targets. Reliable identification of residues that reside in protein-protein interface typically requires analysis of protein structure. Therefore, for the vast majority of pro-teins, for which there is no high resolution structure, there is no effective way of identifying interface residues.<br>
										<br/>
											Results:<br/>
										Here, we present a knowledge-based method that identifies interacting residues from sequence. The method is based on a detailed characterization of protein-protein inter-faces in known high-resolution complexes. We use pre-dicted structural features and evolutionary information, combined with the sequence of the protein. Our strongest predictions reached above 90% accuracy in a cross-validation experiment. Our results suggested that despite the significant diversity in the nature of protein-protein interac-tions, they all share common basic principles identifiable even at the sequence level.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Sequence Analysis II, </b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tue., January 23, 15:30-17:30<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="35" name="35"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Incremental Window-Based Protein Sequence Alignment Algorithms<br/>
													</b><a target="_top" href="http://www.cs.umn.edu/%7Erangwala">Huzefa Rangwala</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www.cs.umn.edu/%7Ekarypis">George Karypis</a></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Department of Computer Science and Engineering, University of Minnesota-Twin Cities, USA<br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
										Protein sequence alignment plays a critical role in computational biology as it is an integral part in many analysis tasks designed to solve problems in compartive genomics, structure and function prediction, and homology modeling.<br>
										<br>
											Methods:<br/>
										We have developed novel sequence alignment algorithms that compute the alignment between a pair of sequences based on short fixed- or variable-length highscoring subsequences. Our algorithms build the alignments by repeatedly selecting the highest scoring pairs of subsequences and using themto construct small portions of the final alignment. We utilize PSI-BLAST generated sequence profiles and employ a profile-to-profile scoring scheme derived from PICASSO.<br>
										<br>
											Results:<br/>
												We evaluated the performance of the computed alignments on two recently published benchmark datasets and compared them against the alignments computed by existing state-of-the-art dynamic programming-based profile-to-profile local and global sequence alignment algorithms. Our results show that the new algorithms achieve alignments that are comparable or better to those achieved by existing algorithms. Moreover, our results also showed that these algorithms can be used to provide better information as to which of the aligned positions are more reliable - a critical piece of information for comparative modeling applications.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="25" name="25"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Designing Patterns for Profile HMM Search<br/>
													</b><a target="_top" href="http://www.cse.wustl.edu/%7Eyanni">Yanni Sun</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.cse.wustl.edu/%7Ejbuhler">Jeremy Buhler</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Washington University in Saint Louis, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br>
										Profile HMMs are a powerful tool for modeling conserved motifs in proteins. These models are widely used by search tools to classify new protein sequences into families based on domain architecture. However, the proliferation of known motifs and new proteomic sequence data poses a computational challenges for search, potentially requiring days of CPU time to annotate an organism's proteome.<br>
										<br>
											Results:<br>
										We use PROSITE-like patterns as a filter to speed up the comparison between protein sequence and profile HMM. A set of patterns is designed starting from the HMM, and only sequences matching one of these patterns are compared to the HMM by full dynamic programming. We give an algorithm to design patterns with maximal sensitivity subject to a bound on the false positive rate. Experiments show that our patterns typically retain at least 90% of the sensitivity of the source HMM while accelerating search by an order of magnitude.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="5" name="5"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Simulating Multiplexed SNP Discovery Rates using Base-Specific Cleavage and Mass Spectrometry</b><br/>
												</font><a href="http://gi.cebitec.uni-bielefeld.de/people/boecker"><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Sebastian B&ouml;cker</a><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Bielefeld University, Germany<br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Single Nucleotide Polymorphisms (SNPs) are believed to contribute strongly to the genetic variability in living beings, and SNP and mutation discovery is of great interest in today's Life Sciences. A comparatively new method to discover such polymorphisms is based on base-specific cleavage, where resulting cleavage products are analyzed by mass spectrometry. One particular advantage of this method is the possibility of multiplexing the biochemical reactions, that is, examining multiple genomic regions in parallel.<br>
										<br>
											Simulations can help estimating the performance of a method for polymorphism discovery, and allow us to evaluate the influence of method parameters on the discovery rate, and also to investigate whether the method is well-suited for a certain genomic region.<br>
										<br>
										We show
how to efficiently conduct such simulations for polymorphism discovery
using base-specific cleavage and mass spectrometry.<br>
										<br>
										Simulating multiplexed polymorphism discovery leads us to the problem of uniformly drawing a multiplex: Given a multiset of natural numbers we want to uniformly draw a subset of fixed cardinadinality so that the elements sum up to some fixed total length. We show how to enumerate multiplex layouts using dynamic programming what allows us to uniformly draw a multiplex.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="32" name="32"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Learning Probabilistic Models of cis-Regulatory Modules that Represent Logical and Spatial Aspects<br/>
													</b>Keith Noto </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Mark Craven, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of Wisconsin, Madison, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
												The process of transcription is controlled by systems of factors which bind in a specific arrangement, called a {\em cis-regulatory module} (CRM), in a set of promoter regions. We present a discriminative learning algorithm which simultaneously learns the DNA binding site motifs as well as the logical structure and spatial aspects of a CRM.<br>
										<br>
											Results:<br/>
												Our results on yeast data sets show better predictive accuracy than a current state-of-the-art approach on the same data sets. Our results on yeast, fly, and human data sets show that the inclusion of logical and spatial aspects improve the predictive accuracy of our learned models.<br>
										<br>
											Availability:<br/>
												Source code will be made available at: <a href="http://www.biostat.wisc.edu" target="_blank">http://www.biostat.wisc.edu</a><br>
										<br>
											Contact: <a href="mailto:noto@cs.wisc.edu">noto@cs.wisc.edu</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Proteomics,</b></font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Wed., January 24, 10:00-11:30<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="6" name="6"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>TOPP - The OpenMS Proteomics Pipeline<br/>
													</b><a target="_top" href="http://www-bs.informatik.uni-tuebingen.de/">Oliver Kohlbacher</a>, <a target="_top" href="http://www-bs.informatik.uni-tuebingen.de/Members/npfeifer/contact">Nico Pfeifer</a></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www-bs.informatik.uni-tuebingen.de/Members/sturm/contact">Marc Sturm</a></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Universit&#228;t T&#252;bingen, Germany</font><br/>
											<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www.inf.fu-berlin.de/inst/ag-bio/file.php?p=ROOT/People/1_Staff/001_Reinert.person.htm">Knut Reinert</a>, </font><a href="http://www.inf.fu-berlin.de/%7egroepl" target="_top"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Clemens Gr&ouml;pl</font></a><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www.inf.fu-berlin.de/inst/ag-bio/file.php?p=ROOT/People/1_Staff/050_Lange.person.htm">Eva Lange</a></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Freie Universit&#228;t Berlin, Germany</font><br/>
											<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://page.mi.fu-berlin.de/%7Etrieglaf/">Ole Schulz-Trieglaff</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Freie Universit&#228;t Berlin / Max Planck Research School for Computational Biology, Germany</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br/>
											 Experimental techniques in proteomics have seen rapid development over the last few years. Volume and complexity of the data have both been growing at a similar rate. Accordingly, data management and analysis are one of the major challenges in proteomics. Flexible algorithms are required to handle changing experimental setups and to assist in developing and validating new methods. In order to facilitate these studies, it would be desirable to have a flexible 'toolbox' of versatile and user-friendly applications allowing for rapid construction of computational workflows in proteomics.<br>
										<br>
											Results:<br/>
												We describe a set of tools for proteomics data analysis -- TOPP, The OpenMS Proteomics Pipeline. TOPP provides a set of computational tools which can be easily combined into analysis pipelines even by non-experts and can be used in proteomics workflows. These applications range from useful utilities (file format conversion, peak picking) over wrapper applications for known applications (e.g. Mascot) to completely new algorithmic techniques for data eduction and data analysis. We anticipate that TOPP will greatly facilitate rapid prototyping of proteomics data evaluation pipelines. As such, we describe the basic concepts and the current abilities of TOPP and illustrate these concepts in the context of two example applications: the identification of peptides from a raw data set through database search and the complex analysis of a standard addition experiment for the absolute quantitation of biomarkers. The latter example demonstrates TOPP's ability to construct flexible analysis pipelines in support of complex experimental setups.<br>
										<br>
											Availability:<br/>
												The TOPP components are available as open-source software under the lesser GNU public license (LGPL). Source code is available from the project web site at <a href="http://www.OpenMS.de" target="_blank">www.OpenMS.de</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="15" name="15"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Difference Detection in LC-MS Data for Protein Biomarker Discovery<br/>
													</b><a target="_top" href="http://www.cs.toronto.edu/%7Ejenn">Jennifer Listgarten</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Sam T. Roweis, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Computer Science, University of Toronto, Canada</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										Radford M. Neal, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Statistics and Department of Computer Science, University of Toronto, Canada</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Peter Wong </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Andrew Emili, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Banting and Best Department of Medical Research, University of Toronto, Canada<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract: <br/>
										</b>Motivation:<br>
											There is a pressing need for improved proteomic screening methods allowing for earlier diagnosis of disease, systematic monitoring of physiological responses, and the uncovering of fundamental mechanisms of drug action. The combined platform of LC-MS (Liquid-Chromatography-Mass-Spectrometry) has shown promise in moving toward a solution in these areas. In this paper we present a technique for discovering differences in protein signal between two classes of samples of LC-MS serum proteomic data without use of tandem mass spectrometry, gels, or labelling. This method works on data from a lower-precision MS instrument, the type routinely used by and available to the community at large today. We test our technique on a controlled (spike-in) but realistic (serum biomarker discovery) experiment which is therefore verifiable. We also develop a new method for helping to assess the difficulty of a given spike-in problem. Lastly, we show that the problem of class prediction, sometimes mistaken as a solution to biomarker discovery, is actually a much simpler problem.<br>
										<br>
											Results:<br>
											Using precision-recall curves with experimentally extracted ground truth, we show that i) our technique has good performance using 7 replicates from each class, ii) performance degrades with decreasing number of replicates, iii) the signal that we are teasing out is not trivially available (i.e., the differences are not so large that the task is easy). Lastly, we easily obtain perfect classification results for data in which the problem of extracting differences does not produce absolutely perfect results. This emphasizes the different nature of the two problems and also their relative difficulties.<br>
										<br>
											Availability<br>
											Our data is publically available as a benchmark for further studies of this nature, along with supplementary information, at <a href="http://www.cs.toronto.edu/%7ejenn/LCMS.html" target="_blank">http://www.cs.toronto.edu/~jenn/LCMS.html</a>.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="12" name="12"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Identifying HLA Supertypes by Learning Distance Functions<br/>
													</b><a target="_top" href="http://www.cs.huji.ac.il/%7Etomboy">Tomer Hertz</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.cs.huji.ac.il/%7Echeny">Chen Yanover</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Hebrew University of Jerusalem, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>The
development of epitope-based vaccines crucially relies on the ability
to classify Human Leukocyte Antigen (HLA) molecules into sets that have
similar peptide binding specificities, termed supertypes. In their
seminal work, Sette and Sidney defined 9 HLA class I supertypes, and
claimed that&nbsp;&nbsp;these provide an almost perfect coverage of the
entire repertoire of HLA class I molecules.<br>
										HLA alleles are highly polymorphic and polygenic and therefore experimentally classifying each of these molecules to supertypes is at present an impossible task. Recently, a number of computational methods have been proposed for this task. These methods are based on defining protein similarity measures, derived from analysis of binding peptides or from analysis of the proteins themselves. In this paper we define both peptide derived and protein derived similarity measures, which are based on learning distance functions. The peptide driven measure is defined using a peptide-peptide distance function, which is learnt using information about known binding and non-binding peptides. The protein similarity measure is defined using a protein-protein distance function, which is learnt using information about alleles previously classified into supertypes by Sette and Sidney. We compare the classification obtained by these two complimentary methods to previously suggested classification methods. In general, our results are in excellent agreement with the classifications suggested by Sette and Sidney and with those reported by Lund et. al.<br>
										There are two important advantages of our proposed distance-based approach. First, it makes use of two different and important immunological sources of information -- HLA alleles and peptides that are known to bind or not bind to these alleles. Second, since each of our distance measures is trained using a different source of information, their combination can provide a more confident classification of alleles into supertypes.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Comparative and Computational Genomics,</b></font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Wed., January 24, 12:00-14:00<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="22" name="22"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>A Comparative Genome Approach to Marker Ordering<br/>
													</b>Thomas Faraut, Simon de Givry, Patrick Chabrier, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a target="_top" href="http://www.inra.fr/mia/T/schiex">Thomas Schiex</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">INRA, France</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										 </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Thomas Derrien, Francis Galibert, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www-recomgen.univ-rennes1.fr/doggy.html">Christophe Hitte</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">CNRS-Universit&eacute; de Renes 1, France</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br>
											Genome maps are fundamental to the study of an organism and essential in the process of genome sequencing which in turn provides the ultimate map of the genome. The increased number of genomes being sequenced offers new opportunities for the mapping of closely related organisms. We propose here an algorithmic formalization of a genome comparison approach to marker ordering.<br>
										<br>
											Results:<br>
											In order to integrate a comparative mapping approach in the algorithmic process of map construction and selection, we propose to extend the usual statistical model describing the experimental data, here radiation hybrids (RH) data, in a statistical framework that models additionally the evolutionary relationships between a proposed map and a reference map: an existing map of the corresponding orthologous genes or markers in a closely related organism. This has concretely the effect of exploiting, in the process of map selection, the information of marker adjacencies in the related genome when the information provided by the experimental data is not conclusive for the purpose of ordering. In order to compute efficiently the map, we proceed to a reduction of the maximum likelihood estimation to the Traveling Salesman Problem. Experiments on simulated RH data sets as well as on a real RH data set from the canine RH project show that maps produced using the likelihood defined by the new model are significantly better than maps built using the traditional RH model.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="26" name="26"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Family Relationships: Should Consensus Reign?-Consensus Clustering for Protein Families<br/>
													</b><a target="_top" href="http://dept-info.labri.fr/%7Emacha/">Macha Nikolski</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">David James Sherman, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">CNRS/LaBRI, Universit&eacute; Bordeaux 1, France<br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br>
											Reliable identification of protein families is key to phylogenetic analysis, functional annotation and the exploration of protein function diversity in a given phylogenetic branch. As more and more complete genomes are sequenced, there is a need for powerful and reliable algorithms facilitating protein families fabrication.<br>
										<br>
											Results:<br>
											We have formulated the problem of protein families construction as an instance of consensus clustering, for which we designed a novel algorithm that is computationally efficient in practice and produces high quality results. Our algorithm uses an election method to construct consensus families from competing clustering computations.<br>
										<br>
											Our consensus clustering algorithm is tailored to serve the specific needs of comparative genomics projects. First, it provides a robust means to incorporate results from different and complementary clustering methods, thus avoiding the need for an a priori choice that may introduce computational bias in the results. Second, it is suited to large-scale projects due to the practical efficiency. And third, it produces high quality results where families tend to represent groupings by biological function.<br>
										<br>
											Availability:<br>
											This method has been used for Genolevures project to compute protein families of Hemiascomycetous yeasts. The data is available online at <a href="http://cbi.labri.fr/Genolevures/fam/" target="_blank">http://cbi.labri.fr/Genolevures/fam/</a><br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="24" name="24"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Merging Microarray Cell Synchronization Experiments through Curve Alignment<br/>
													</b><a target="_top" href="http://www.psb.ugent.be/cbd/">Filip Hermans</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.psb.ugent.be/cbd/">Elena Tsiporkova</a></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Dept. of Plant Systems Biology, Flanders Institute for Biotechnology, Belgium</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>The
validity of periodic cell-cycle regulation studies in plants is
seriously compromised by the relatively poor quality of cell synchrony
that is achieved for plant suspension cultures in comparison to yeast
and mammals. The present state-of-the-art plant synchronization
techniques cannot offer a complete cell-cycle coverage and moreover a
considerable loss of cell synchrony may occur toward the end of the
sampling. One possible solution is to consider combining multiple data
sets, produced by different synchronization techniques and thus
covering different phases of the cell cycle, in order to arrive at a
better cell-cycle coverage. <br>
										<br>
										We propose a method that enables
pasting expression profiles from different plant cell synchronization
experiments and results in an expression curve that spans more than one
cell cycle. The optimal pasting overlap is determined via a dynamic
time warping alignment. Consequently, the different expression time
series are merged together by aggregating the corresponding expression
values lying within the overlap area. We demonstrate that the periodic
analysis of the merged expression profiles produces more reliable
p-values for periodicity. Subsequent Gene Ontology analysis of the
results confirms that merging synchronization experiments is a more
robust strategy for the selection of potentially periodic genes.
Additional validation of the proposed algorithm on yeast data is also
presented.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="18" name="18"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>A Supervised Approach for Identifying Discriminating Genotype Patterns and its Application to Breast Cancer Data<br/>
													</b><a target="_top" href="http://www.cs.tau.ac.il/%7Eniryosef/">Nir Yosef</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Roded Sharan, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science, Tel Aviv University, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
											 Zohar Yakhini </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Anya Tsalenko, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Agilent Technologies, Israel</font><br/>
											<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Vessela Kristensen, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Genetics, Institute of Cancer Research, Rikshospitalet-Radiumhospitalet Medical Center, Montebello, Norway</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
										Anne-Lise Børresen-Dale, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Genetics, Institute of Cancer Research, Rikshospitalet-Radiumhospitalet Medical Center, Montebello, Norway and Medical Faculty, University of Oslo, Norway</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
										Eytan Ruppin, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science and School of Medicine, Tel Aviv University, Israel<br>
									</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation: <br>
											Large scale association studies, investigating the genetic determinants of a phenotype of interest, are producing increasing amounts of genomic variation data on human cohorts. A fundamental challenge in these studies is the detection of genotypic patterns that discriminate individuals exhibiting the phenotype under study from individuals that do not posses it. The difficulty stems from the large number of single nucleotide polymorphism(SNP) combinations that have to be tested. The discrimination problem becomes even more involved when additional high-throughput data, such as gene expression data, is available for the same cohort.<br>
										<br>
										Results: <br>
											We have developed a graph theoretic approach for identifying discriminating patterns for a given phenotype in a genotyped population. The method is based on representing the SNP data as a bipartite graph of individuals and their SNP states, and identifying fully connected subgraphs of this graph that relate individuals enriched for a given phenotypic group. The method can handle additional data types such as expression profiles of the genotyped population. It is reminiscent of biclustering approaches with the crucial difference that its search process is guided by the phenotype under consideration in a supervised manner.<br>
											We tested our approach in simulations and on real data. In simulations, our method was able to retrieve planted patterns with high success rate. We then applied our approach to a data set of 72 breast cancer patients with available gene expression profiles, genotyped over 695 SNPs. We detected several discriminating patterns thatwere highly significant with respect to various clinical phenotypes, and investigated the groups of patients and the groups of genes they defined.<br>
											We found the patient groups to be highly enriched for other phenotypes and to display expression coherency among their profiles. The gene groups displayed functional coherency and involved genes with known role in cancer, providing additional support to their involvement.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Evolution and Phylogenetics,</b></font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Wed., January 24, 15:30-17:30<br>
									</font></td>
							</tr>
							<tr>
								<td><a id="4" name="4"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Inferring Phylogeny from Whole Genomes</b><br/>
												<a target="_top" href="http://www.mimuw.edu.pl/%7Egorecki">Pawel Gorecki</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.mimuw.edu.pl/%7Etiuryn">Jerzy Tiuryn</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Warsaw University, Poland<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract: <br/>
										</b>Motivation:<br/>
										Inferring species phylogenies with a history of gene losses and duplications is a challenging and important task in computational biology. This problem can be solved by duplication-loss models in which the primary step is to reconcile a rooted gene tree with a rooted species tree. Most modern methods of phylogenetic reconstruction (from sequences) produce unrooted gene trees. This limitation leads to the problem of transforming unrooted gene tree into a rooted tree, and then reconciling rooted trees. The main questions are ``What about biological interpretation of choosing rooting?'', ``Can we find efficiently the optimal rootings?'', ``Is the optimal rooting unique?''.<br>
										<br>
											Results:<br>
											In this paper we present a model of reconciling unrooted gene tree with a rooted species tree, which is based on a concept of choosing rooting which has minimal reconciliation cost. Our analysis leads to the surprising property that all the minimal rootings have identical distributions of gene duplications and gene losses in the species tree. It implies, in our opinion, that the concept of an optimal rooting is very robust, and thus biologically meaningful. Also, it has nice computational properties. We present a linear time and space algorithm for computing optimal rooting(s). This algorithm was used in two different ways to reconstruct the optimal species phylogeny of five known yeast genomes from approx. 4700 gene trees. Moreover, we determined locations (history) of all gene duplications and gene losses in the final species tree. It is interesting to notice that the top 5 species trees are the same for both methods.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="16" name="16"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Using Median Sets for Inferring Phylogenetic Trees<br/>
										</b>Matthias<b>
													</b>Bernt, Daniel Merkle, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Martin Middendorf, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of Leipzig, Germany</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Motivation:<br>
											Algorithms for phylogenetic tree reconstruction based on gene-order data typically repeatedly solve instances of the Reversal Median problem (RMP) which is to find for three given gene sequences a fourth sequence (called median) with a minimal sum of reversal distances. All existing algorithms of this type consider only one median for each RMP instance even when a large number of medians exist. A careful selection of one of the medians might lead to better phylogenetic trees.<br>
										<br>
											Results:<br>
											We propose a heuristic algorithm amGRP for solving the Multiple Genome Rearrangement Problem (MGRP) by repeatedly solving instances of the RMP taking all medians into account. Algorithm amGRP uses a branch-and-bound method that branches over medians from a selected subset of all medians for each RMP instance. Different heuristics for selecting the subsets have been investigated. To show that the medians for RMP vary strongly with respect to different properties that are likely to be relevant for phylogenetic tree reconstruction, the set of all medians have been investigated for artificial data sets and mitochondrial DNA (mtDNA) sequences. Phylogenetic trees have been computed for a large set of random data sequences and two sets of mtDNA gene-order data for different animal taxa with amGRPg and with two standard algorithms (GRAPPA-DCM and MGR). The results show that amGRP outperforms both other algorithms with respect to solution quality and computation time on the test data.<br>
										<br>
											Availability:<br>
										The source code of amGRP, additional results, and the test instances used in this paper are freely available from the authors of the paper.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="31" name="31"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Efficient Parsimony-Based Methods for Phylogenetic Network Reconstruction<br/>
													</b>Guohua Jin </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Luay Nakhleh, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Rice University, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Sagi Snir, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of California, Berkeley, USA</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
										Tamir Tuller, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tel Aviv University, Israel</font><br/>
								</td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Phylogenies---the evolutionary histories of groups of organisms---play a major role in representing relationships among biological entities. Although many biological processes can be effectively modeled as a tree-like relationships, others, such as hybrid speciation, and horizontal gene transfer (HGT) result in networks of relationships rather than trees of relationships. Hybrid speciation is a significant evolutionary mechanism in plants, fish, and other groups of species. HGT plays a major role in bacterial genome diversification, and is a significant mechanism by which bacteria develop resistance to antibiotics.<br>
											Maximum parsimony (MP) is one of the most commonly used criteria for phylogenetic tree inference. Roughly speaking, inference based on this criterion seeks the tree that minimizes the amount of evolution. In 1990, Jotun Hein proposed using this criterion for inferring the evolution of sequences subject to recombination. Preliminary results on small synthetic data sets (Nakhleh et al., 2005) demonstrated the criterion's application to phylogenetic network reconstruction in general, and HGT detection in particular. However, the naive algorithms used by the authors are inapplicable to large data sets due to their demanding computational requirements. Further, no rigorous theoretical analysis of computing the criterion was given, nor was it tested on biological data.<br>
										<br>
											In this work, we prove that the problem of scoring the parsimony of a phylogenetic network is NP-hard, and provide an improved fixed parameter tractable algorithm for it. Further, we devise an efficient heuristics for parsimony-based reconstruction of phylogenetic networks. We test our methods on both synthetic and biological data (rbcL gene in bacteria), and obtain very promising results.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="20" name="20"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Phylogeny Reconstruction: Increasing the Accuracy of Pairwise Distance Estimation using Bayesian Inference of Evolutionary Rates<br/>
													</b><a target="_top" href="http://www.cs.huji.ac.il/%7Eninio/matan/">Matan Ninio</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.cs.huji.ac.il/%7Enir/">Nir Friedman</a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Hebrew University of Jerusalem, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												<a target="_top" href="http://www.tau.ac.il/%7Eprivmane/">Eyal Privman</a> </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> <a target="_top" href="http://www.tau.ac.il/%7Etalp/">Tal Pupko</a></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Tel Aviv University, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Distance-based
methods for phylogeny reconstruction are the fastest and easiest to
use, and their popularity is accordingly high. They are also the only
known methods that can cope with huge datasets of thousands of
sequences. These methods rely on evolutionary distance estimation, and
are sensitive to errors in such estimations. In this study, a novel
Bayesian method for estimation of evolutionary distances is developed.
The proposed method enables the use of a sophisticated evolutionary
model that better accounts for among-site rate variation (ASRV),
thereby improving the accuracy of distance estimation. Rate variations
are estimated within a Bayesian framework by extracting information
from the entire dataset of sequences, unlike standard methods that can
only use one pair of sequences at a time. We compare the accuracy of a
cascade of distance estimation methods, starting from commonly used
methods and moving towards the more sophisticated novel method.
Simulation studies show significant improvements in the accuracy of
distance estimation by the novel method over the commonly used ones. We
demonstrate the effect of the improved accuracy on tree reconstruction
using both real and simulated protein sequence alignments.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td bgcolor="#d9d9d9"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Late Breaking Presentations<br>
										</b></font></td>
							</tr>
							<tr>
								<td><a id="41" name="41"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Algorithms on Splicing Graphs for Identifying and Aligning Alternative Splicing Events</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Michael Sammeth </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Sylvain Foissac, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Genome Bioinformatics Laboratory, Centre for Genomic Regulation, Catalonia, EU</font><br>
								</td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>The complexity of molecular mechanisms involved in alternative splicing (AS) in literature grows continuously with new studies being published. This data requires an universal way, to describe AS events and to flexibly correlate them with each other. Herein, we describe a method to universally characterize AS events from gene annotations, and an algorithm to align gene structures in order to study the changes of AS events across homologous genes. An online visualization of the proposed methods AStaLaVista is being made available at: http://genome.imim.es/astalavista.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="42" name="42"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Inferring Mechanistic Explanations for Transcription Factor Binding Changes</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br/>
										</b>Kenzie MacIsaac </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Ernest Fraenkel, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Massachusetts Institute of Technology, USA</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>We present an algorithm that determines the mechanisms underlying changes in transcriptional regulatory protein binding.  The algorithm, called Bluebird, uses a probabilistic approach based on biophysical principles to jointly analyze high-throughput chromatin immunoprecipitation (ChIP-chip) data for multiple proteins in multiple conditions.  Bluebird predicts how the observed binding changes are best explained:  Do the binding specificities of the proteins change in different conditions?   Are binding differences best explained by changes in the nuclear concentration of the proteins?  Do several proteins interact?  By probabilistically analyzing the raw ChIP-chip data, Bluebird avoids the use of arbitrary thresholds to define bound and unbound genes.  Analysis of ChIP-chip data for 37 different yeast transcription factors gave motif predictions consistent with the known binding specificity for all factors.  In addition, results for several factors yielded predictions about the underlying mechanism of binding changes between conditions that were in agreement with previously reported experimental results.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="43" name="43"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Reconstructing Sibling Relationships from Microsatellite Data<br/>
											 </b>Saad Sheikh, Tanya Berger-Wolf</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Bhaskar Dasgupta, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Mary Ashley, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of Illinois at Chicago, USA<br>
									</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Wanpracha Chaovalitwongse, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Rutgers University, USA</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>Reconstruction of sibling relationships from genetic data is an important component of many biological applications. Most current sibship reconstruction methods use statistical and heuristic techniques that rely on a priori knowledge about various parameter distributions. We present a deterministic algorithm that parsimoniously reconstructs sibling groups. We use Mendelian laws of inheritance to generate potential sibling groups. We then optimally select a minimum set of sibling groups necessary to explain the data. We validate our approach using both simulated and real biological data.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="44" name="44"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Refinement and Expansion of Signaling Pathways: The Osmotic Response Network in Yeast<br>
										</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Irit Gat-Viks</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tel Aviv University, Israel<br>
									</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">		</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>The analysis of large scale genome-wide experiments carries the promise of dramatically broadening our understanding on biological networks. The challenge of systematic integration of experimental results with established biological knowledge on a pathway is still unanswered. Here we present a methodology that attempts to answer this challenge when investigating signaling pathways. We formalize existing qualitative knowledge as a probabilistic model that depicts known interactions between molecules (genes, proteins etc.) as a network and known regulatory relations as logics. We present algorithms that analyze experimental results (e.g., transcription profiles) vis-a-vis the model, and propose improvements to the model based on the fit to the experimental data. These algorithms refine the relations between model components, as well as expand the model to include new components that are regulated by components of the original network.<br>
										<br>
										Using our methodology, we have modeled together the knowledge on four established signaling pathways related to osmotic shock response in S. cerevisiae. Using over 100 published transcription profiles, our refinement methodology revealed three cross-talks in the network. The expansion procedure identified with high confidence large groups of genes that are co-regulated by transcription factors from the original network via a common logic. The results reveal a novel delicate repressive effect of the HOG pathway on many transcriptional target genes, and suggest an unexpected alternative functional mode of the MAP kinase Hog1. The analysis also predicts novel feed-forward and feedback loops in the regulatory network, which probably support cellular adaptation to osmotic stress. These results demonstrate that by integrated analysis of data and of well-defined knowledge on signaling pathways, one can generate concrete biological hypotheses about signaling cascades and their downstream regulatory programs.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="45" name="45"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>PROMO: A Method for Identifying Modules in Protein Interaction Networks<br/>
										</b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Omer Tamuz, Yaron Singer, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Roded Sharan</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tel Aviv University, Israel<br>
									</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">		</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>A major goal of systems biology is the dissection of protein machineries within the cell. The recent availability of genome-scale protein interaction networks provides a key resource for addressing this challenge. Current approaches to the problem are based on devising a scoring scheme for putative protein modules and applying a heuristic search for high-scoring modules.<br>
										<br>
										Here we develop a branch and bound approach to perform an exhaustive scan of the search space.We show that such a search is possible and enables detecting modules that are missed by previous approaches. The modules we identify are shown to be significantly coherent in their functional annotations and expression patterns. Our algorithm, PROMO, is shown to outperform the state-of-the-art MCODE and to provide results that are more in line with current biological knowledge.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="46" name="46"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Temporal Aspects of Histone-Modifying Proteins Revealed by Sub-network Analysis<br/>
													</b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Sabine Dietmann, Michael Mader, Hans-Werner Mewes, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Martin Münsterkötter, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Institute for Bioinformatics, Germany</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">		</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>Network analysis has been recently applied to model not only static topological properties, but also temporal aspects of protein complexes on a genomic scale, e.g. during the yeast cell cycle. The topology of transcription factor networks, in particular, has been studied extensively. In eukaryotic genomes, the chromosomal context is crucial for transcriptional activity because of the regulatory effects of histones, around which DNA are wrapped to form the nucleosomes and further the chromatin. The properties of chromatin can be altered either by nucleosome remodeling or by covalently modifying the N-terminal tails of histones. Here, we present an integrative approach that extends the yeast transcription factor network by sub-networks of histone-modifying proteins and addresses temporal aspects of potentially recruiting transcription factors and their behavior under different cellular conditions. We provide a thorough benchmark of sub-networks extracted form protein-protein and protein-DNA interaction data sets against genome-wide location data sets.<br/>
												<br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
						</tbody></table>
	</body></html>