<html>
<head>
<title>Poster Abstracts E</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category E: Evolution & phylogenetics</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="E01" name="E01"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E01</b><br>
    <b>Keeping a Watchful Eye for Mammalian mtDNA Recombination</b><br>
    Joanna Elson (1), Robert Lightowlers (1), Neil Howell (2,3)<br>
    (1) University Newcastle upon Tyne; (2) MitoKor Inc. (now Migenix Corp.), San Diego, US; (3) University Texas (UTMB)
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The standard paradigm postulates that mammalian mtDNA (mitochondrial DNA) is inherited clonally. Thus, mtDNA analyses are held to be free of the complexities introduced by bi-parental recombination. This important assumption has been tested a number of times the results have been the subject of intense debate. 
<br>Complete mtDNA sequences provide the best starting point for robust tests of recombination.  I propose to set up an internet based repository of mtDNA sequences that will allow detection of recombination candidates and then carry out laboratory follow-up to confirmation any observations.
<br><br>Contact: j.l.lelson [at] ncl.ac.uk
<br><br>Keywords: mtDNA, Recombination, Database
</font><hr></td></tr>
<tr>
  <td><a id="E02" name="E02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E02</b><br>
    <b>Molecular Recognition and Evolution in the Olfactory Receptor Gene Family</b><br>
    Tsviya Olender, Ester Feldmesser, Ronny Aloni, Doron Lancet<br>
    Department of Molecular Genetics, Weizmann Institute of Science, Rehovot 76100, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Whole-genome comparison of five olfactory receptors (ORs) mammalian repertoires including the marsupial opossum, was performed. We identified 48 CLICs (CLusters In Conservation) containing most of the mammalian ORs. Human and mouse OR expression data were collected from a multitude of sources. Data showed an unexpectedly high degree of ectopic expression in non-olfactory tissues. 
<br><br>Contact: tsviya.olender [at] weizmann.ac.il
<br><br>Keywords: Olfaction, Ectopic Expression, Orthologs
</font><hr></td></tr>
<tr>
  <td><a id="E03" name="E03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E03</b><br>
    <b>Genomic Coevolution of Syntenic Genes in Rhizobiales</b><br>
    Humberto Peralta, Gabriela Guerrero, Alejandro Aguilar, Jaime Mora<br>
    Centro de Ciencias Genomicas-UNAM
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Syntenic genes (with conserved chromosomal position) represent the ancestral gene set and we consider they are constrained to rearrangement. To assess the coevolution of syntenic (and non-syntenic) genes in Rhizobiales we evaluated the phylogeny of these genes and their relation with sequence conservation, functional role and prediction of network linkage. 
<br><br>Contact: peralta [at] ccg.unam.mx
<br><br>Keywords: Rhizobiales, Coevolution, Synteny
</font><hr></td></tr>
<tr>
  <td><a id="E04" name="E04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E04</b><br>
    <b>Analysis of Probabilities of Evolutionary Changes in Protein Structures</b><br>
    Juris Viksna (1,2), David Gilbert (2)<br>
    (1) Institute of Mathematics and Computer Science, University of Latvia; (2) Bioinformatics Research Centre, University of Glasgow 
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The aim of the work was to estimate the probabilities with which different structural changes in proteins might occur. We have analyzed protein structures using representation with information about the SSEs and grouping of strands in beta-sheets, and several types of structural changes have been considered. The evolutionary distances between all pairs of beta-topologies from CATH database were compared with sequence similarities between proteins. Statistically significant probability distribution was observed for a transformation of a loop into a sheet of two unparallel strands.
<br><br>Contact: juris.viksna [at] mii.lu.lv
<br><br>Keywords: Protein Structures, Protein Evolution
</font><hr></td></tr>
<tr>
  <td><a id="E05" name="E05"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E05</b><br>
    <b>Lineage Tree Analysis and the Dynamics of FL and DLBCL</b><br>
    Neta S. Zuckerman (1), Katy McCann (2), Christian H. Ottensmeier (2), Freda S. Stevenson (2), Ramit Mehr (1)<br>
    (1) Bar Ilan University, Israel; (2) University of Southampton, UK
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Much information about the dynamics of hypermutation and antigen-driven clonal selection during the humoral immune response is contained in the shape of mutational lineage trees deduced from the responding clones. We have designed a novel algorithm for quantifying the shape properties of mutational lineage trees and applied it to trees from several B cell lymphomas. This analysis suggests that the diseased clones have a higher intraclonal diversity, a longer diversification time and impaired selection compared to normal B cell clones.
<br><br>Contact: tsukern [at] mail.biu.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="E06" name="E06"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E06</b><br>
    <b>Trees and Forests: A Genome-Wide Reconstruction of Orthologous Groups and their Evolution</b><br>
    Ilan Wapinski (1,2), Nir Friedman (3), Avi Pfeffer (2), Aviv Regev (1)<br>
    (1) Bauer Center for Genomics Research, Harvard University; (2) Division of Engineering and Applied Sciences, Harvard University; (3) School of Computer Science and Engineering, Hebrew University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Gene duplication and loss are major evolutionary forces.  Despite the growing number of fully sequenced genomes, methods for investigating these events on a genome-wide scale are still in their infancy.  As a result, comparative genomics studies are yet to realize their full potential for shedding light on how biological processes behave across evolutionary timespans.  Here we examine the orthology and paralogy relations between the entire set of genes from 17 fully sequenced yeast genomes, covering an evolutionary range comparable to that of the entire phylum of chordates. 
<br><br>Contact: ilan [at] eecs.harvard.edu
<br><br>Keywords: Orthology, Comparative Genomics, Duplication
</font><hr></td></tr>
<tr>
  <td><a id="E08" name="E08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E08</b><br>
    <b>Periodic Distribution of Hydrophobic Amino-Acids Use for Distantly Related Proteins Alignment and Homology Search</b><br>
    Julie Baussand, Alessandra Carbone<br>
    INSERM U511 Universite Pierre et Marie Curie-Paris 6
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We developed the first automatic alignment tool which uses hydrophobic blocks (hb) as a basic pattern to align remote homologues. Hb are detected by a prescreening of the amino-acids sequence, and are then integrated in a classical alignment procedure thanks to parameters specific to hb context. Tests on 3 reference alignment datasets (334 pairs of proteins of <30% identity) confirms the accuracy of the use of hb for the alignment of distantly related proteins.  New criteria based on the evaluation of hb are proposed for the detection of remote homologues on large databases.
<br><br>Contact: baussand [at] infop6.jussieu.fr
<br><br>Keywords: Remote Homologue, Alignment, Hydrophobic Block
</font><hr></td></tr>
<tr>
  <td><a id="E09" name="E09"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E09</b><br>
    <b>A Covarion-like Evolutionary Model for the Detection of Rate Shifts</b><br>
    Osnat Penn, Tal Pupko<br>
    Tel Aviv University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Functional changes in a protein are reflected by shifts in the evolutionary rate. For instance, a decrease in the rate of a site may indicate that it underwent a functional shift, and now has higher functional significance. We present a novel evolutionary based method for the detection of specific lineages and sites where an evolutionary rate shift occurred. The method is based on a covarion model which allows the site-specific evolutionary rate to vary among lineages. Several biological datasets were analyzed with the aim of detecting lineages and sites which display covarion-like evolution.
<br><br>Contact: zomerosn [at] post.tau.ac.il
<br><br>Keywords: Covarion, Site-Specific Rate Variation
</font><hr></td></tr>
<tr>
  <td><a id="E10" name="E10"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E10</b><br>
    <b>The Evolution of the Human Genome: Prevalence of Tandem Duplications on Small Scales</b><br>
    Philipp W. Messer, Peter F. Arndt<br>
    Max-Planck-Institute for Molecular Genetics, Ihnestr. 63-73, 14195 Berlin, Germany
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Comparing the human and chimp genome we analyze recent insertions and deletions in the human lineage of up to 100 bp in length. We can distinguish insertions from deletions by taking rhesus as an outgroup and find that the majority of insertions are tandem duplications of adjacent sequence segments. While such a duplication process is well expected to occur in low complexity DNA, it also affects high complexity regions. Small tandem duplications can constitute an important process for the rapid evolution of complex regulatory functions encoded in the promoter and enhancer regions of genes.
<br><br>Contact: philipp.messer [at] molgen.mpg.de
</font><hr></td></tr>
<tr>
  <td><a id="E11" name="E11"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster E11</b><br>
    <b>Converting Empirical Amino-Acid Replacement Matrices into Codon-Based Substitution Matrices</b><br>
    Adi Doron-Faigenboim, Tal Pupko<br>
    Tel Aviv University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Evolutionary selection forces acting on proteins are inferred by contrasting the rate of synonymous to non-synonymous substitutions. This inference is based on evolutionary codon models. Most models ignore the fact that amino acids differ in their replacement rates, and are based on theoretical assumptions only. We develop a model that allows integration of different empirical amino-acid replacement probabilities into a codon substitution matrix, thus allowing the creation of "context dependent codon models". We show that for most datasets the novel model is superior to the mechanistic models.
<br><br>Contact: doronadi [at] post.tau.ac.il
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
