<html>
<head>
<title>Poster Abstracts C</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category C: Computer-aided drug design</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="C02" name="C02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster C02</b><br>
    <b>An Alternative Method for the Evaluation of Docking Performance: Real Space R Value (RSR) vs. RMSD</b><br>
    Dilimulati Yusufujiangaili<br>
    Institute for Theoretical Chemistry, University of Vienna
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The evaluation of the docking programs is usually based on the root mean square deviation (RMSD) between the nonhydrogen atoms positions of ligand's crystallographic model and those predicted by docking programs. Since the drawbacks of RMSD based evaluation causes much bias, an alternative evaluation method is presented in this paper. The new method is based on ligand's real space fit (RSR) to experimental density map. In most cases, RMSD and RSR do not have general agreement on quality of docked models. RSR based method gives a fairer evaluation on docked models.
<br><br>Contact: dilmurat [at] tbi.univie.ac.at
<br><br>Keywords: Docking Evalution, RMSD, RSR
</font><hr></td></tr>
<tr>
  <td><a id="C03" name="C03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster C03</b><br>
    <b>A Probabilistic Approach to the Design of Structural Selectivity of Proteins</b><br>
    Menachem Fromer, Chen Yanover, Julia Shifman, Yair Weiss<br>
    The Hebrew University of Jerusalem
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The multiple-state protein design problem strives to predict the protein sequence(s) best suited to selectively fold to one and only one of a set of competing (related) protein structures.  We have formulated the problem using probabilistic graphical models.  Using this formulation, we performed computational multiple-state protein design on relevant biological examples.  The results have been computationally assessed through comparison with competing state-of-the-art methods (and using exact energy computations, where feasible).
<br><br>Contact: fromer [at] cs.huji.ac.il
<br><br>Keywords: Protein Design, Probabilistic, Multi-state
</font><hr></td></tr>
<tr>
  <td><a id="C04" name="C04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster C04</b><br>
    <b>Analysis of Antibiotic Binding Pockets on the Ribosome Large Subunit using Structure-Based Networks</b><br>
    Hilda David, Yael Mandel-Gutfreund<br>
    Technion-Israel Institute of Technology
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Clinically important antibiotics inhibit the activity of bacterial ribosome. Prominent among them are the MLSBK antibiotics which target the ribosome large subunit. We analyzed the 23S rRNA of bacteria and archea using mathematical graphs (networks) where nucleotides are represented as nodes and the intermolecular interactions as edges. For each individual node in the network we calculated different centrality measures. Our results show that the betweenness and the degree measures help to better understand the structural properties of the MLSBK antibiotics binding pockets.
<br><br>Contact: hildad [at] tx.technion.ac.il
<br><br>Keywords: Ribosome Structure Analysis, Network Analysis
</font><hr></td></tr>
<tr>
  <td><a id="C05" name="C05"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster C05</b><br>
    <b>Computational Design of RNA Structural Switches from Building Blocks</b><br>
    Assaf Avihoo, Danny Barash<br>
    Department of Computer Science, Ben-Gurion University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We aim at devising a robust computational procedure for designing RNA structural switches from building blocks with favorable properties. To achieve maximal throughput for genetic control purposes, future designer RNA switches can be assembled based on a computerized buildup of the constituent domains, namely the aptamer and the expression platform in the case of a synthetic riboswitch. Initially, simulations can produce a list of short sequences that switch between two conformers when trigerred by point mutations. Consequently, one of the known aptamers is attached and screening is performed.
<br><br>Contact: dbarash [at] cs.bgu.ac.il
<br><br>Keywords: Energy Minimization Methods, RNA Switches
</font><hr></td></tr>
<tr>
  <td><a id="C07" name="C07"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster C07</b><br>
    <b>Mapping Protein-Protein Interfaces using Combinatorial Libraries</b><br>
    Itay Mayrose (1), Tomer Shlomi (2), Nimrod D. Rubinstein (1), Jonathan M. Gershoni (1), Eytan Ruppin (2), Roded Sharan (2), Tal Pupko (1)<br>
    (1) Department of Cell Research and Immunology, Tel Aviv University; (2) School of Computer Science, Tel-Aviv University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    A phage-display library of random peptides is a combinatorial technique for obtaining random peptides that bind an antibody with high affinity. These peptides can be used to infer the antibody's corresponding epitope. Here we present 'PepSurf', an algorithm for mapping a set of affinity-selected peptides onto the 3D structure of the antigen. This is done by aligning each peptide to a graph representing the antigen's surface. The most significant paths are then clustered into predicted epitopes. We show that PepSurf accurately predicts the known epitopes in 3 antibody-antigen complexes.
<br><br>Contact: itaymay [at] post.tau.ac.il
<br><br>Keywords: Epitope Mapping, Combinatorial Libraries
</font><hr></td></tr>
<tr>
  <td><a id="C08" name="C08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster C08</b>&nbsp;&nbsp;&nbsp;<font color="#ff0000"><b><i>Late-Breaking Results</i></b></font><br>
    <b>Revealing the Interaction Network of the Pro-apoptotic Protein ASPP2: Combination of Docking and Experiment</b><br>
    Hadar Benyamini, Chen Katz, Shahar Rotem, Assaf Friedler<br>
    Hebrew University, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The ASPP2 protein specifically enhances the ability of the tumor suppressor p53 to cause apoptosis. ASPP2 is part of a protein interactions network and its C-terminal ankyrin and SH3 domains mediate its interactions with numerous other proteins, including the apoptosis regulators Bcl-2 and NF-KB. Our research aim was to characterize the interactions of ASPP2 with its partner proteins, to gain insight into its mechanism of action and regulation.
<br>We have explored the interactions of ASPP2 using a combination of experimental and computational methods. We used peptide mapping to identify the ASPP2-binding sites on its partner proteins: a membrane-bound peptide array, consisting of overlapping peptides derived from various ASPP2-binding proteins was screened for binding various ASPP2 constructs. The experimental binding results served as a basis for docking analysis in which peptide binding information was used to improve the ranking and selection of docking models for the complexes of ASPP2 with its binding proteins. The structure of the C terminal part of ASPP2 containing the ankyrin repeats and SH3 domain was docked to structures of various Bcl-2 family proteins.
<br>ASPP2 was found to bind the anti-apoptotic proteins Bcl-2, Bcl-W and Bcl-XL through three alpha-helices.  The N-terminal alpha-helix corresponds to the BH4 ("Bcl-2 homology" 4) region, which is known to be sufficient to prevent apoptosis in cells. The other two alpha-helices correspond to the BH1 and BH3 regions, which are components of a conserved groove that is also a site for Bcl-2 regulation by binding pro-apoptotic Bcl-2 family members. Residues within these regions are also known to be disrupted upon binding of Bcl-XL to p53. The peptide binding data and docking models suggest that at least one way by which ASPP2 promotes apoptosis is by inhibiting anti-apoptotic Bcl-2 proteins in a similar manner by which they are inhibited by p53 and pro-apoptotic Bcl-2 family members.
<br><br>Contact: hadar [at] chem.ch.huji.ac.il
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
