<html>
<head>
<title>Poster Abstracts Q</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category Q: Other</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="Q01" name="Q01"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q01</b><br>
    <b>Investigations on the Systems Biology of Aspergillus niger</b><br>
    Andreas Grote (1,2), Alex Dalpiaz (2), Guido Melzer (2), Ursula Rinas (3), Bernd Northemann (2), Petra Dersch (1), Dietmar C. Hempel (2), Dieter Jahn (1)<br>
    (1) Institute for Microbiology, TU Braunschweig; (2) Institute of Biochemical Engineering, TU Braunschweig; (3) Structural Biology, German Research Center for Biotechnology
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The investigation of the systems biology of the spore forming fungus Aspergillus niger is one of the main objectives of the SFB578. For this purpose the transcriptome, proteome and metabolome are investigated. It is intended to optimize the product synthesis in the cell on different levels by applying bioinformatics algorithms. As a first tool  the "Java Codon Adaptation Tool" was developed to offer the possibility to optimize heterologous protein production on a transcriptional level.
<br><br>Contact: andreas.grote [at] tu-bs.de
<br><br>Keywords: Systems Biology, Aspergillus niger, Codon Usage
</font><hr></td></tr>
<tr>
  <td><a id="Q02" name="Q02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q02</b><br>
    <b>The Stability of the Ancestral Eukaryotic Protein Complexome in Extant Eukaryotes and its Implications</b><br>
    Hugo Ceulemans, Lijs Beke, Mathieu Bollen<br>
    Katholieke Universiteit Leuven, Department of Molecular Cell Biology
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We present a study of the ancestral eukaryotic protein complexome, which we defined as the set of protein complexes that extant eukaryotes inherited from their last common ancestor. The ancestral complexome constitutes nearly half of the ancestral proteome and at 90 % conservation of the included 1400 proteins in all studied organisms, proves highly stable. This observation endorses the potential of (sub)complexes as the units of choice for consistent systems modelling across various model eukaryotes.
<br><br>Contact: Hugo.Ceulemans [at] med.kuleuven.be
<br><br>Keywords: Complexome Orthology
</font><hr></td></tr>
<tr>
  <td><a id="Q03" name="Q03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q03</b><br>
    <b>A Computational Framework for Analysis of Dynamic Social Structures</b><br>
    Tanya Y. Berger-Wolf (1), Ilya R. Fischhoff (2,3), Daniel I. Rubenstein (2,3), Jared Saia (4), Siva R. Sundaresan (2,3)<br>
    (1) Department of Computer Science, University of Illinois, Chicago; (2) Department of Ecology & Evolutionary Biology, Princeton University; (3) Mpala Research Centre, Nanyuki, Kenya; (4) Department of Computer Science, University of New Mexico, Albuquerque
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Analysis of many processes in animal populations (disease spread, dominance, reproduction) requires analysis of the network of individual interactions. Until now such analysis was essentially aggregate and static. We show that it may produce incorrect answers and propose a novel conceptual and computational framework for analysis of dynamic interaction networks.
<br><br>Contact: tanyabw [at] cs.uic.edu
<br><br>Keywords: computational population biology, social networks 
</font><hr></td></tr>
<tr>
  <td><a id="Q04" name="Q04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q04</b><br>
    <b>Towards Verified Biological Models</b><br>
    Avital Sadot (1), Jasmin Fisher (2), Dan Barak (1), Yishai Admanit (1), Michael J. Stern (3), E. Jane Albert Hubbard (4), David Harel (1)<br>
    (1) Weizmann Institute of Science; (2) Swiss Federal Institute of Technology (EPFL); (3) Yale University School of Medicine; (4) New York University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Using tools that were originally developed for non-biological system design, we constructed a model of the early lineage of the somatic gonad of C. elegans and focused on an interaction between two cells within this lineage. We tested the model by showing that the simulations were consistent with the set of experimental scenarios. This small-scale modular example demonstrates the potential for using similar approaches for verification by exhaustive testing of models. It also shows the advantages of these approaches for modeling biology.
<br><br>Contact: avitals [at] weizmann.ac.il
<br><br>Keywords: Modeling, C. elegans, Verification
</font><hr></td></tr>
<tr>
  <td><a id="Q05" name="Q05"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q05</b><br>
    <b>Evaluating the Design of Biological Cells Using a Computer Workbench </b><br>
    Tessa E. Pronk (1,2) , Simon Polstra (2), Andy D. Pimentel (2), Marco Roos (1), Timo M. Breit (1)<br>
    (1) Integrative BioInformatics Unit, University of Amsterdam; (2) Computer Systems Architecture, University of Amsterdam
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    For embedded computer systems as well as for biological cell systems, design is one feature that defines their identity. The assembly of different components in designs of both systems can vary widely. In contrast to the cell, embedded computer systems in general are well-mapped systems and their behavior can effectively be characterized. 
<br>Our aim is to construct a framework that focuses on understanding the design options and their consequences within biological cells, based upon ideas from the field of embedded computer systems. 
<br><br>Contact: tepronk [at] science.uva.nl
<br><br>Keywords: Architecture, Design, Computer, Cell Systems
</font><hr></td></tr>
<tr>
  <td><a id="Q08" name="Q08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q08</b><br>
    <b>BioFrame: A Java-based Software Development framework for Bioinformatics</b><br>
    Fernando Martinez, Junaid Gamieldien, Darren Otgaar, Dan Jacobson<br>
    National Bioinformatics Network, South Africa
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Bioframe is an Open Source, Java-based, rapid application development (RAD) framework for Bioinformatics and Computational Biology. Bioframe integrates mature open source tools and frameworks like Biojava, Hibernate, and others, in a robust J2EE environment. The framework is extensible, thread safe, and among other services, provides security, logging, reusable components, templates and wizards to help create and test new web applications, as well as seamless integration between the different layers of an application.
<br><br>Contact: fernando [at] nbn.ac.za
<br><br>Keywords: Bioinformatics, Java, Software Development
</font><hr></td></tr>
<tr>
  <td><a id="Q09" name="Q09"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q09</b><br>
    <b>Sequence Dependence of DNA Conformation and Deformation Calculated by Normal Mode Dynamics</b><br>
    Michael Levitt (1), Peter S. Stern (2)<br>
    (1) Stanford University; (2) Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In this work, we calculate the stiffness of the DNA double helix using a novel method that depends on the use of normal mode dynamics to calculate the frequencies of the overall bending and twisting motions of a series of DNA sequences.  We investigate the dependence of DNA deformability on the base sequence.  Small differences in the bending and twisting stiffness are detected by calculating the normal modes of segments of DNA of closely related sequence.  The results show that sequences rich in A and T are more easily bent than those rich in G and C.
<br><br>Contact: Peter.Stern [at] weizmann.ac.il
<br><br>Keywords: DNA bending, Normal Modes
</font><hr></td></tr>
<tr>
  <td><a id="Q11" name="Q11"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q11</b><br>
    <b>Building Bioinformatics across Africa</b><br>
    Dan Jacobson (1), Nicky Mulder (2), Trushar Shah (3)<br>
    (1) National Bioinformatics Network (NBN), South Africa; (2) ASBCB & University of Cape Town NBN Node; (3) Biosciences Eastern and Central Africa (BECA)
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    "Modern biotechnology is opening up new opportunities to address challenges of increasing food production, stemming environmental degradation, fighting such diseases as malaria, HIV/AIDS and tuberculosis, and enhancing industrialization in Africa." (1)  In order for Africa to develop to the point where it can meaningfully participate in the Biosciences it must have the capacity to engage in Bioinformatics.  The application of Bioinformatics therefore has great potential to underpin development of solutions to Africa's problems in health and agriculture, and thus to promote economic development.
<br><br>Contact: dan [at] nbn.ac.za
<br><br>Keywords: NBN, BECA, Africa, NEPAD
</font><hr></td></tr>
<tr>
  <td><a id="Q18" name="Q18"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster Q18</b>&nbsp;&nbsp;&nbsp;<font color="#ff0000"><b><i>Late-Breaking Results</i></b></font><br>
    <b>G3DV: A New 3D Genome Browser and Experimental Data Viewer</b><br>
    Michael J. Moorhouse (1), Bert Eussen (2), Tobias A. Knoch (3), Michael Lesnussa (3), Albert D. M. E. Osterhaus (1), Frank Grosveld (3), Annelies deKlein (2)<br>
    (1) Department of Virology, Erasmus MC; (2) Department of Clinical Genetics, Erasmus MC; (3) Department of Cell Biology and Genetics, Erasmus MC
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    With the increasing complexity of biological studies and sophistication of molecular diagnostic tests, new visualization methods are needed to display experimental results in an appropriate genomic context so that they can be properly understood.  We describe a new type of genome browser, the GLOBE 3D Viewer (G3DV) that presents genomics experimental data alongside public annotation in an interactive 3D virtual space to give a more "holistic" representation of the genome.  We demonstrate its utility for the selection of BACs in a repetitive element-rich region of the human genome that is suitable for use in medical diagnostic tests using classical FISH techniques.
<br><br>Contact: m.moorhouse [at] erasmusmc.nl
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
