<html>
<head>
<title>Poster Abstracts J</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category J: Proteomics</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="J01" name="J01"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J01</b><br>
    <b>Comparative Proteomics of Different Morphological Types of Streptococcus pneumoniae</b><br>
    M. Portnoi (1), A. Adawi (1), D. Kafka (1), N. Porat (1,2), R. Dagan (1), Y. Mizrachi-Nebenzahl (1,2)<br>
    (1) Pediatric Infectious Disease Unit, Soroka University Medical Center; (2) Department of Microbiology and Immunology, Ben-Gurion University of Negev
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Most clinical isolates of S. pneumoniae consist of heterogeneous populations of at least two colony phenotypes, opaque and transparent. Bacterial load in the different tissue of the opaque phenotype were higher in comparison to transparent phenotype. The survival rates of the mice which were infected with the opaque phenotype were 30% and 10%. Expression pattern of proteins extracted from the cell wall of opaque variant were differ in comparison to transparent variant. Using proteomics, we have identified several proteins that differ between opague and transparent population.
<br><br>Contact: maximp [at] bgu.ac.il
<br><br>Keywords: S. pneumoniae, Protein, Virulence 
</font><hr></td></tr>
<tr>
  <td><a id="J02" name="J02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J02</b><br>
    <b>A Pseudo Probabilistic Context-Free Grammar for the Detection of Binding Sites from a Protein Sequence</b><br>
    Witold Dyrka (1), Jean-Christophe Nebel (2)<br>
    (1) Wroclaw University of Technology, Poland; (2) Kingston University, UK
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Analysis of protein sequences to predict their functions is a very challenging problem where pattern recognition techniques based on HMMs have proved to be the most efficient. However HMMs have limitations. According to formal language theory, their expressive power is similar to probabilistic regular grammars. Here, we propose a pattern recognition method based on a more powerful grammar. We developed a pseudo probabilistic context-free grammar to detect protein regions that are involved in binding sites. We then present results showing the potential of our method to predict protein function.
<br><br>Contact: 108770 [at] student.pwr.wroc.pl
<br><br>Keywords: Probabilistic Grammar, CFG, GA, Binding Site
</font><hr></td></tr>
<tr>
  <td><a id="J03" name="J03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J03</b><br>
    <b>Evolutionary Conservation of Domain-Domain Interactions</b><br>
    Zohar Itzhaki, Eyal Akiva, Yael Altuvia, Hanah Margalit<br>
    Department of Molecular Genetics and Biotechnology, The Hebrew University of Jerusalem
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We apply computational tools to map structurally-derived domain-domain interactions (DDIs) onto protein-protein interaction (PPI) networks of different organisms. We demonstrate that there is a catalogue of domain pairs that is used for mediating various interactions in the cell. Comparison of the repertoires of DDIs in the networks of E.coli, yeast, worm, fly and human, shows that they are evolutionary conserved. This suggests that different organisms use the same "building blocks" for PPIs and that the functionality of the DDIs as mediating protein interactions is maintained in evolution.
<br><br>Contact: zohari [at] md.huji.ac.il
<br><br>Keywords: Protein Domains, Protein-Protein Interactions
</font><hr></td></tr>
<tr>
  <td><a id="J04" name="J04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J04</b><br>
    <b>Comparative Assessment of Large-Scale Maps of the Human Protein Interactome</b><br>
    Matthias E. Futschik (1), Gautam Chaurasia (1,2), Erich Wanker (2), Hanspeter Herzel (1)<br>
    (1) Institute for Theoretical Biology, Charite-Medical Division, Humboldt-University; (2) Max-Delbruck-Centrum, Berlin, Germany
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We present here a first comparative assessment of eight different large-scale human protein-protein interaction networks including over 57000 interactions between 10000 unique proteins. This analysis shows that current maps have only a small, but significant overlap. We detected intrinsic tendencies which are important to take into consideration in future applications of these maps. We also observed that some previous findings for network structures in lower eukaryotes cannot be reproduced for current human interaction maps and a reevaluation of network concepts might be warranted.
<br><br>Contact: m.futschik [at] biology.hu-berlin.de
<br><br>Keywords: Proteomics, Interactome, Protein  Networks
</font><hr></td></tr>
<tr>
  <td><a id="J06" name="J06"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J06</b><br>
    <b>PROMPT: A Protein Mapping and Comparison Tool</b><br>
    Thorsten Schmidt, Dmitrij Frishman<br>
    Technische Universitat Munchen, Department of Genome Oriented Bioinformatics
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Comparison of large protein datasets has become a standard task in bioinformatics. Typically researchers wish to know whether one group of proteins is significantly enriched in certain annotation attributes or sequence properties compared to another group, PROMPT is a comprehensive bioinformatics software environment which enables the user to compare arbitrary protein sequence sets, revealing statistically significant differences in their annotation features. A Java API, scripting capabilities and a Graphical User Interface make it accessible to bioinformaticians and biologically-oriented users.
<br><br>Contact: t.schmidt [at] wzw.tum.de
<br><br>Keywords: Comparative Proteomics, Statistics, Generic
</font><hr></td></tr>
<tr>
  <td><a id="J07" name="J07"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J07</b><br>
    <b>Sequence to Function Analysis of New Protein-Splicing Domains</b><br>
    Bareket Dassa, Shmuel Pietrokovski<br>
    Department of Molecular Genetics, Weizmann Institute of Science, Rehovot, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Inteins and related protein domains post-translationally process their own molecules by protein-splicing, cleavage and ligation. Integrating computational and experimental approaches we in silico identified new bacterial intein-like domains (BILs) in hyper-variable proteins of diverse bacteria (including pathogens), and in vivo studied their autocatalytic activities. Our research reveals new mechanisms for generating protein variability and new ways of protein maturation, and has biotechnological implications.
<br><br>Contact: bareket [at] wicc.weizmann.ac.il
<br><br>Keywords: Intein, Protein Splicing, Autocatalysis
</font><hr></td></tr>
<tr>
  <td><a id="J08" name="J08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J08</b><br>
    <b>Functional Identification of Protein Signatures</b><br>
    Iris Bahir, Michal Linial<br>
    Hebrew University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Short signatures are hard to detect through routine methods due to their size. Most of what is known about short signatures to date was generated by time consuming experiments. We tested the conservation and functional significance of short signature in protein termini by applying a positional search method. Our results show that short signatures are conserved at protein termini; such short signature can typify a group of proteins in terms of their function. The collection of significant signature group of proteins was archived and advanced search options are available at http://www.proteus.cs.huji.ac.il. 
<br><br>Contact: irisss [at] pob.huji.ac.il
<br><br>Keywords: Signatures, Proteins, Termini
</font><hr></td></tr>
<tr>
  <td><a id="J09" name="J09"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J09</b><br>
    <b>Computer Simulation in Tritium Planigraphy</b><br>
    Alexey Dolgov, Elena Bogacheva, Alexey Chulichkov, Alexander Shishkov, Ekaterina Vedeniapina<br>
    Semenov Institute of Chemical Physics, Russian Academy of Science 
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The algorithm of computer simulation of bombardment of a macromolecule by an anisotropic  beam of atoms is developed for interpretation in structural terms of experiment data received by this method. The results of computer simulating allow to propose the 3D structures model of a macromolecule on the basis of experiment data and to define its possible orientations relatively to a interface of phases and a degree of immersion into one of phases. The developed algorithm is approved on model adsorption layers and membrane protein M1 of an influenza A virus.
<br><br>Contact: dolgov48 [at] nm.ru, ben@chph.ras.ru
<br><br>Keywords: Computer Simulation, Proteins, Tritium
</font><hr></td></tr>
<tr>
  <td><a id="J13" name="J13"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J13</b><br>
    <b>A Geometric Approach to Integration of Site-Directed FTIR Data on Transmembrane Proteins as a Base for Modeling</b><br>
    Daniel K Shenfeld, Itamar Kass, Joshua Manor, Isaiah T Arkin<br>
    The Hebrew University of Jerusalem, The Department of Biological Chemistry
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Site-directed FTIR (Fourier Transform Infrared) spectroscopy measurements provide data on various bond angles in transmembrane (TM) alpha-helices. Here we introduce a mathematical model which is used to numerically calculate the helix director from FTIR measurements. We then construct a helix whose director matches the calculated one and conforms to the measured data. Finally, a molecular dynamics (MD) protocol is used to extend the model. Information from the resulting structure can then be compared with biochemical and biophysical data. 
<br><br>Contact: danielshenf [at] gmail.com
<br><br>Keywords: Molecular Modeling, FTIR, Molecular Dynamics
</font><hr></td></tr>
<tr>
  <td><a id="J14" name="J14"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J14</b><br>
    <b>Peptide Biomarker Discovery for the Diagnosis of Renal Allograft Rejection</b><br>
    Ole Schulz-Trieglaff (1,2), Clemens Gropl (2), Joachim Thiemann (3), Knut Reinert (2), Hartmut Schlueter (3)<br>
    (1) Int. Max Planck Research School for Computational Biology and Scientific Computing Berlin; (2) Free University Berlin, Department of Computer Science ; (3) Core Facility Protein Analysis, Charite - University Medicine Berlin
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The rejection of the allograft is a major problem after renal transplantation. It has been shown that these rejections are accompanied by a change in proteolytic activities and consequently by a change of the peptide pattern in plasma and urine. 
<br>We summarize our computational workflow which builds on top of OpenMS, our software framework for computational proteomics. Furthermor we describe the computational challenges arising from the complex data in LC/MS experiments and how we intend to surmount them.
<br><br>Contact: trieglaf [at] inf.fu-berlin.de
<br><br>Keywords: Biomarker, Allograft Rejection, Proteomics
</font><hr></td></tr>
<tr>
  <td><a id="J15" name="J15"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J15</b><br>
    <b>A Novel Method for Model-Based Feature Extraction in Mass Spectra</b><br>
    Karin Noy (1,3), Anne-Katrin Emde (2), Daniel Fasulo (3)<br>
    (1) Ben Gurion University, Beer-Sheva, Israel; (2) Free University, Berlin, Germany; (3) Siemens Corporate Research, Princeton, NJ, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Motivation: Mass spectrometry (MS) is being used increasingly for biomedical research. The typical analysis of MS data consists of several complex steps. Feature extraction is a crucial step since subsequent analyses are performed only on the detected features. Current methodologies, in which features are peaks or wavelet functions, are parameter-sensitive and inaccurate. Depending on many factors, a single molecule may appear either as a single or multiple peaks; spectra can also contain complex, overlapping signals and degenerate peak shapes due to noise and resolution limitations. 
<br>Results: In this paper, we suggest a model-based approach to feature extraction in which spectra are decomposed into a mixture of distributions derived from peptide models. Our model is parameterized by chemical and physical properties and are therefore applicable to different MS techniques and instruments. We apply our methods to both high and low resolution real MS data and show that the performance is higher than commonly used tools. In addition, we examine the performance of our method on simulated data where we show high sensitivity and no false positives.
<br><br>Contact: karin.noy.ext [at] siemens.com
<br><br>Keywords: Mass Spectrometry, Feature Extraction
</font><hr></td></tr>
<tr>
  <td><a id="J16" name="J16"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J16</b><br>
    <b>De Novo Peptide Sequencing Using Exhaustive Enumeration of Peptide Composition and MALDI TOF/TOF</b><br>
    Jonathan A. Epstein, Matthew T. Olson, Alfred L. Yergey<br>
    NIH, Bethesda, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Novel aspect: A new potentially general approach to de novo sequencing has been developed.
<br>Protein identification typically relies on established database searching routines, but there is a need for reliable peptide sequencing because of incomplete or inadequate databases. De novo peptide sequencing algorithms abound, but objective assessments of de novo sequencing capabilities with unknowns indicate that much work remains. We introduce a peptide composition lookup table indexed by residual mass and number of amino acids (LIPCUT) for de novo sequencing of polypeptides using exhaustive compositional analysis. Using mass tolerances of 0.035 or 0.050 Da, peaks are referenced against the exhaustive amino acid composition lookup table to obtain a complete profile of amino acid combinations present. Concatenating the differences between successive entries yields peptide sequences which can be ranked by signal intensities.
<br>LIPCUT was generated as an exhaustive list of ALL amino acid sequences to 1750 Da and by using using composition and residue monoisotopic mass was indexed into a 35GB database. Peptides, 0.3uL, ~1pmole/uL, were applied to an ABI 192 well plate, and matrix (CHCA, 5mg/mL in 900uL 1:1 ACN:0.1% TFA + 100uL 0.1M AmPhos) added. MS/MS spectra acquired using ABI 4700 Proteomics Analyzer: 3500 shots/spectrum, collision gas off. Peak lists generated from the spectra, formatted, and filtered using frequency of peak S/N. Peaks are encoded to yield residue masses assuming they are both y- or b-series ions. One of several algorithms generates sequences using either extension or matching approaches.
<br>All spectra investigated were the result of unimolecular decompositions, i.e., PSD, rather than the result of CID measurements. These spectra are less complex than those produced by CID, yet generally yield complete fragmentations. LIPCUT was produced using 19 amino acids with the isobaric pair Ile and Leu being assigned the symbol X. The sequencing operation consists principally of using LIPCUT as a lookup table, although the extension types of algorithms do this less than the matching algorithms. For well fragmented spectra we find the extension algorithms to be more efficient than the matching algorithms. 23 peptides were sequenced and yielded correct sequences with maximum of one missed fragment, typically at the N-terminal; that is the correct residues were found, but their order could not be completely resolved. The minimal missed fragments are a likely consequence of the instrumentation used. In a number of the cases of missed fragmentations subsequent manual inspection of the spectra revealed internal ions that could be used to resolve misses. In general, the near isobaric residues, K and Q were correctly identified despite a mass tolerance in the algorithm that would suggest this is not possible.
<br>Most recently (summer 2006) we have incorporated the use of internal fragment ions as a tiebreaking mechanism for similarly-scored polypeptides.
<br><br>Contact: Jonathan_Epstein [at] nih.gov
<br><br>Keywords: Amino Acids, Peptide Fragmentation, MALDI, Mass Spectrometry, Time of Flight
</font><hr></td></tr>
<tr>
  <td><a id="J17" name="J17"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J17</b>&nbsp;&nbsp;&nbsp;<font color="#ff0000"><b><i>Late-Breaking Results</i></b></font><br>
    <b>Protein Function Prediction of a Lassa Virus Glycoprotein from a Newly Extracted Lassa Virus Gene</b><br>
    Lawrence Okoror, Bayo Momodu, Ibiwumi Oloye<br>
    Ambrose Alli University, Ekpoma, Nigeria
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Lassa virus is the cause of morbidity and high mortality in Ekpoma Nigeria. Over the years lassa virus has led to tremendous epidemics in this locality with little or no hope for vaccine. Recently, two new strains of the virus were identified. The genes were deposited in the GenBank. We presume that genes of similar sequence have similar function. Using one of the newly isolated gene sequence, we ran the blast tool and wu- blast there was a significant hits using matrix PAM250, PAM100, and Blosum62. However there was no significant hit at PAM50. The ORfs was determined using the tool at expasy and the six ORFs was determined. The glycoprotein gene product is said to be antigenic and at the same time virulent. Hence using bioinformatics tools the virulent part of the protein could be removed and then synthesized and used as a vaccine since it will stimulate the production of antibodies. We determine the function of the protein by running it on blastp with different parameters. However not all the parameters produced significant hits even when the e-values were adjusted. Other functions used in determining the function of the protein included, hydrophobicity, leader sequence, transmembrane helices, using tools at expasy and the protein predict server. The prints and block data bases were also searched. While the print produced significant hits, the block database produced no statistically significant hits. GeneDoc and ClutalW was used to remove the myristoylation part of the protein which is the virulence part of the protein leaving the carbohydrate and the kinase part which could trigger antibody production and hence could be synthesized for vaccine production.
<br><br>Contact: Larison86 [at] yahoo.com
</font><hr></td></tr>
<tr>
  <td><a id="J18" name="J18"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster J18</b>&nbsp;&nbsp;&nbsp;<font color="#ff0000"><b><i>Late-Breaking Results</i></b></font><br>
    <b>Networks of Functional Coupling in Eukaryotes</b><br>
    Andrey Alexeyenko, Erik Sonnhammer<br>
    Stockholm Bioinformatics Center, Sweden
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    FunCoup is a statistical framework of data integration for finding functional coupling (FC) between proteins. It is capable of transferring information from model organisms via orthologs. Data of different sources and various natures (contacts of whole proteins and individual domains, mRNA co-expression, protein co-occurrence, phylogenetic profiles etc.) are collected and probabilistically evaluated in a Bayesian network, trained on sets of known FC cases  vs. sets of randomly picked protein pairs as background reference. FunCoup was optimized to address known drawbacks of Bayesian estimators. Compared to the previous framework configurations of this sort, the net gain in performance is tens of percentage points in both sensitivity and specificity. The number of simultaneously used model organisms (5-8) and individual datasets (50-60) has been estimated as maximal for practical purposes. 
<br>FunCoup is a self-consistent framework that easily incorporates nearly any kind of data from various data sources without curation. It has thus been possible to generate networks for several organisms (human, mouse, rat, worm, fly, Arabidopsis, and yeast) in respect of different types of functional coupling. A network for Ciona intestinalis, which had neither training sets nor sources of its own data, was created as well.  The networks are available at the FunCoup website http://FunCoup.sbc.su.se.
<br><br>Contact: Andrey.Alexeyenko [at] cgb.ki.se
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
