<html>
<head>
<title>List of Accepted Software Demos</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">

<!--
<link type="text/css" rel="StyleSheet" href="submission_accepted.cgi_files/easy.css">
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"></head><body bgcolor="white">
<script src="submission_accepted.cgi_files/easy.js"> </script><script>easyState.user='g71ufxin4Gsq19zHP4Bf';easyState.role='C';easyState.track=1;</script><div class="menu"><span class="menuheading">ECCB 2006 (chair)</span><br> <a onmouseover="top.status='list of submissions';return true;" href="http://www.easychair.org/ECCB06/submissions_show.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Submissions</a> | <a onmouseover="top.status='reviews for all papers';return true;" href="http://www.easychair.org/ECCB06/review_all.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">All&nbsp;Reviews</a> | <a onmouseover="top.status='current status of all papers';return true;" href="http://www.easychair.org/ECCB06/status.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Status</a> | <a onmouseover="top.status='assign papers to PC members for reviewing';return true;" href="http://www.easychair.org/ECCB06/assignment.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Assign&nbsp;Reviews</a> | <a onmouseover="top.status='view or update information about PC members';return true;" href="http://www.easychair.org/ECCB06/pc_show.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">PC</a> | <a onmouseover="top.status='list of recent (all) events';return true;" href="http://www.easychair.org/ECCB06/events.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Events</a> | <a onmouseover="top.status='list of recent (all) emails sent by EasyChair';return true;" href="http://www.easychair.org/ECCB06/email_show_all.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Letters</a> | <a onmouseover="top.status='other facilities';return true;" href="http://www.easychair.org/ECCB06/other.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Other</a> | <a onmouseover="top.status='view or update EasyChair configuration';return true;" href="http://www.easychair.org/ECCB06/configure.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Configure</a> | <a onmouseover="top.status='your account information and updates';return true;" href="http://www.easychair.org/ECCB06/account.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">My&nbsp;Account</a> | <a onmouseover="top.status='login with a different role';return true;" href="http://www.easychair.org/ECCB06/login_as.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Login&nbsp;As</a> | <a onmouseover="top.status='exit EasyChair';return true;" href="http://www.easychair.org/ECCB06/logout.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;"><font color="orange">Logout</font></a></div><h2 class="blue">List of Accepted Papers</h2><form method="post" action="submission_accepted.cgi" enctype="multipart/form-data">
<input name="eu" value="g71ufxin4Gsq19zHP4Bf" type="hidden">
<input name="es" value="C1" type="hidden">
<input tabindex="1" name="submit" value="Hide Abstracts" class="redbutton" type="submit">
<div></div></form>
-->
<table border="0" cellpadding="3" cellspacing="0"><tbody>
							<tr>
								<td><br>
									<br>
									<hr>
									<br>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Advanced Micro Devices<br>
											<hr>
										</b></font></td>
							</tr>
							<tr>
								<td><a id="12" name="12"></a><br>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Large Scale Pedigree Haplotyper (LSPH)<br/>
													</b>Eyal Seroussi, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Dept.of Quantitative and Molecular Genetics, Institute of Animal Science, The Agricultural Research Organization (ARO),Volcani Center, Israel<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>The genotyping data of diploid organisms obtained by current laboratory techniques provides unordered allele pairs for each marker. Reconstruction of haplotypes from this data is a crucial step in many applications. Haplotypes of tightly linked markers provide insight into old and rare recombination events, and thus are more informative than single markers. In particular, haplotypes are essential for linkage disequilibrium (LD) based gene mapping. Approaches to reconstruction of haplotypes by genotype inference can be divided into statistical methods and non-statistical methods. The first approach applies computational or statistical inference to find the most likely haplotype configuration consistent with the observed genotypic data. In the non-statistical approach, every possible trio of two parents and an offspring is examined. The haplotypes are resolved by forward inference of Mendelian rules from the parental genotypes to the offspring genotype and backward inference from offspring genotype to his parents. As the pedigree size increases, the number of computations for most rule-based algorithms increases exponentially. Thus, analysis of pedigrees consisting of hundreds of individuals would require several hours or even days of computations.<br>
										<br>
										LSPH (Baruch et.al. 2006 Genetics.172:1757-1765) is a rule based program for inference of haplotypes from genotypes of large animal pedigree or population for Windows OS. It assumes nothing about the haplotype frequency in the population, but assumes zero recombination events.The application reads the raw genotype file which consists of rows. Each row contains the gender and identities of an individual and its parents followed by the available genotypes of this individual. Then it verifies that the structure of the families is free from errors. The result of LSPH run is a file which describes resolved haplotypes for each of the pedigree members. It also records some statistical estimation about the amount of resolved haplotypes in a log file. Output is presented in a user friendly XML format allowing the use of Excel as a tool for viewing the resolved haplotype. The parental and maternal haplotypes are colored in blue and red, respectively and genotype errors can be traced by viewing the type and description of conflicts which are displayed as Excel comments indicated with red triangles at right corners of the allele cells.<br>
									</font>
									<hr/>
								</td>
							</tr>
							<tr>
								<td><a id="13" name="13"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Grid Computing for Bioinformatics</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br/>
													</b>David Horn </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Assaf Gottlieb, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Physics and Astronomy, Tel Aviv University, Israel<br>
									</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Edward Aronovich</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science, Tel Aviv University, Israel<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>We shall demonstrate the use of a simple web interface to perform common bioinformatics tasks and show the power of parallel computing on grid to reduce the running time of these tasks.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">We offer researchers a tool thorough which they can transform their proprietary software from a batch mode task into an interactive task. This enables researchers to expose their software capabilities through the web without distribution of their source code/ binaries.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The system is based on Gilda and Genius portal which are part of Enabling Grids for E-science (EGEE II) project. The system benefits from the idle time of a large number of computers.</font><br>
									<br>
									<font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Israel Academic Grid (IAG) team will demonstrate the system. Further details may be found at <a href="http://iag.iucc.ac.il/" target="_blank">http://iag.iucc.ac.il/<br>
										</a></font>
									<hr size="2"/>
								</td>
							</tr>
							<tr>
								<td><a id="10" name="10"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Selecton: A Web Server for the Detection of Site-Specific Positive Darwinian Selection and Purifying Selection<br/>
													</b>Adi Stern, Adi Doron-Faigenboim, Eran Bacharach, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Tal Pupko, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Cell Research and Immunology</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Tel Aviv University, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>The ratio of non-synonymous to synonymous substitutions, known as the Ka/Ks ratio, is used to estimate both purifying and positive Darwinian selection. A Ka/Ks ratio significantly greater than 1 is indicative of positive selection, whereas values significantly smaller than 1 are indicative of purifying selection. We present an algorithmic web-based tool which calculates the Ka/Ks ratio for each codon site in a codon-based multiple sequence alignment (Doron-Faigenboim et al. 2005; Stern et al. 2006).Selecton implements both an empirical Bayesian algorithm (Yang et al. 2000) as well as a maximum-likelihood algorithm (Goldman and Yang 1994), which the user may choose (the default algorithm is the Bayesian one). On the one hand, through its user-friendly interface Selecton enables simplicity of use for non-expert users. The minimal input to the server consists of merely a file of homologous DNA coding sequences. On the other hand, Selecton further implements a wide variety of user options which enable maximal fine-tuning of the server to the user's needs. These include control over the different parameters implemented in the Bayesian algorithm, computation of the Ka/Ks scores under different genetic codes, and input of a user-built phylogenetic tree. Following calculation of Ka/Ks ratios at each site of the protein, these ratios are converted into a discrete color scale, and projected onto one of the homologous sequences specified by the user. If a 3-dimensional structure of the protein is available, the scores will also be projected onto the Van-der-Waals surface of the protein.<br>
										<br>To exemplify its use, we used Selecton to analyze the HIV-1 protease for mutations that confer resistance to the Ritonavir drug (Doron-Faigenboim et al. 2005).The analysis was done on 70 sequences obtained from patients who were treated with the drug. Selecton projected the Ka/Ks scores onto the tertiary structure of the protease, enabling us to detect several patches of positive selection and purifying selection<br/>
										<br>
											References<br>Doron-Faigenboim A, Stern A, Mayrose I, Bacharach E, Pupko T (2005) Selecton: a server for detecting evolutionary forces at a single amino-acid site. Bioinformatics 21:2101-3<br>Goldman N, Yang Z (1994) A codon-based model of nucleotide substitution for protein-coding DNA sequences. Mol Biol Evol 11:725-36<br>Stern A, Doron-Faigenboim A, Bacharach E, Pupko T (2006) Selecton 2006: detecting positive Darwinian selection and purifying selection in proteins using a Bayesian approach. Submitted to BMC Evolutionary Biology<br>
										Yang Z, Nielsen R, Goldman N, Pedersen AM (2000) Codon-substitution models for heterogeneous selection pressure at amino acid sites. Genetics 155:431-49<br>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="9" name="9"></a><br>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Text Mining as Web Services Provided from the EBI<br>
										</b>Dietrich Rebholz-Schuhmann, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">EBI, Wellcome Trust Genome Campus, UK</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Whatizit is a modular text processing system that is openly available through EBI's Web pages. It allows processing of any type of text to identify named entities and to annotate the named entities with links to bioinformatics databases. In addition, Whatizit enables retrieval of Medline abstracts either by keyword query, by UniProtKB/Swiss-Prot accession key or by PubMed ID. Retrieved abstracts can be processed automatically via single modules or complex text processing pipelines. Whatizit is also available as 1) a webservice and as 2) a streamed servlet. The Web service allows enriching content of individual Web sites in a similar way as in wikipedia. The streamed servlet allows processing large amounts of text.<br>
										<a href="https://www.ebi.ac.uk/Rebholz-srv/whatizit/pipe" target="_blank">
										https://www.ebi.ac.uk/Rebholz-srv/whatizit/pipe</a><br>
										<br>EBIMed is a web application that combines Information Retrieval and Extraction from Medline.EBIMed retrieves Medline abstracts in a way similar to PubMed. In addition it analyses them to offer a complete overview on associations between UniProtKB/Swiss-Prot protein/gene names, GO annotations, Drugs and Species. The results are represented in a table that displays all the associations and links to the sentences that support them and to the original abstracts. By selecting relevant sentences and highlighting the biomedical terminology EBIMed enhances your ability to acquire knowledge, relate facts, discover implications and, overall, have a good overview economizing the effort in reading. Design principles and implementation of EBIMed will be presented in the text mining session of the ECCB 2006.<br>
										<a href="https://www.ebi.ac.uk/Rebholz-srv/ebimed" target="_blank">https://www.ebi.ac.uk/Rebholz-srv/ebimed</a><br/>
											
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="8" name="8"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>EVEREST: A Collection of Evolutionary Conserved Protein Domains<br/>
													</b>Elon Portugaly, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science and Engineering, The Hebrew University of Jerusalem, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
											Nathan Linial, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">School of Computer Science and Engineering, The Hebrew University of Jerusalem, Israel<br>
									</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Michal Linial, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Department of Biological Chemistry, Institute of Life Sciences, The Hebrew University of Jerusalem, Israel</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Proteins are comprised of one or several domains that are evolutionary recurrent. We developed a computational tool to identify protein domains and to classify them into their families. Our tool and database, is combined in the website called EVEREST. We have applied EVEREST (Ver1.1) to SwissProt database (Ver 1.1;133,000 proteins). EVEREST is a collection of 13,569 domain families. These families include over one million domains, and jointly cover 83%of the amino acids in the Swiss-Prot database. The average (median) size of a domain family in this collection is 81 (41) amino acids and the average length of the domains is 117 (76) amino acids.<br>
										<br>The process underlying EVEREST begins by constructing a database of protein segments that emerge in an all vs. all pairwise sequence comparison. It then proceeds to cluster these segments, choosing the best clusters using machine learning techniques, and creating a statistical model for each of the these clusters. This procedure is then iterated: The aforementioned statistical models are used to scan all protein sequences, to recreate a segment database and to cluster them again. Performance tests show that EVEREST recovers 63% of Pfam families and 40% of SCOP families with high accuracy, and suggests new families with about 40% fidelity. EVEREST domains are frequently a combination of domains as defined by Pfam or SCOP and are also frequently subdomains of such domains.<br>
										<br>EVEREST is the only web based tool that provides in depth analysis of major domain based classifications including SCOP and CATH (structurally based) and Pfam (sequence based). It is designed to allow users to test their own protein sequence against all HMM models. It is a very rich resource for domains that will be essential for structural genomics initiatives (for domain boundary definition), for evolutionary based view on proteins, and as a source for new discoveries.<br>
										<br>EVEREST presents one statistical model per domain family. The website provides rich navigating tools at the level of the proteins, domain families and the entire system. EVEREST is available at <a href="http://www.everest.cs.huji.ac.il" target="_blank">http://www.everest.cs.huji.ac.il</a><br>In the presentation, EVEREST navigating tools will be demonstrated. Specifically, we will demonstrate the usefulness of EVEREST for navigating in the domain space through phyletic groups. In addition, the use of EVEREST as a discovery tool and as an assessment tool will be demonstrated by testing several biological questions.<br/>
											
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="25" name="25"></a><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Sun Microsystems, Inc.</b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br>
										</b></font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
										</b>In the new global economy, speed-to-market is an essential component in getting, and staying, ahead of the competition. High Performance Computing (HPC) concentrates on this critical need. It uses compute power to help solve highly complex problems, perform business critical analysis, or run computationally intensive workloads all faster and more efficiently. The outcome is faster time to results.<br>
										<b><br>
											<hr>
										</b></font></td>
							</tr>
							<tr>
								<td><a id="3" name="3"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Blast2GO v2 </b><br/>
									</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Ana Conesa</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Centro de Genomica, Instituto Valenciano de Investigaciones, Agrarias, Spain<br>
									</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Stefan Goetz</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Grupo de Informatica Biomedica, Instituto de Aplicaciones de las Tecnologias de la Informacion y de las Comunicaciones Avanzadas, Universidad Politecnica de Valencia, Spain<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>Blast2GO (B2G) is bioinformatics software developed with the aim of providing biologists with a user-friendly tool that joins in one application similarity search-based functional annotation of (novel) genes or proteins, and the functional analysis of genomics data. Basically, B2G performs blast to find sequences similar to a query set and assigns GO annotation based on an annotation rule that takes into consideration multiplicity of homologues,amount of similarity and the annotation quality of the hit sequences. Blast searches can be performed remotely against the NCBI using default parameters of locally on specific databases. B2G offers the possibility of direct statistical analysis on gene function information and visualization of relevant functional features on an interactive and highlighted GO direct acyclic graph (DAG). Powerful navigation algorithms with interactive functions have been incorporated for explorative analysis. The application includes various descriptive statistics features for summarizing results obtained at blasting,GO-mapping, annotation as well as functional data mining functions such as significance analysis for GO-term over or under-representation of sets of sequences.<br>
										<br>Processing steps are configurable and data import and export are supported at any stage. The application has been endowed with an array of data input possibilities. Not only raw sequences can be processed, but also, pre-existing blast or annotation files are accepted and taken to subsequent steps. While blast results can be imported in XML format, existing sequences annotation are supported as Gene-Symbol or Accession identifier lists.<br>
										<br>Improvements have been recently incorporated at different stages of the annotation and analysis process. The blast module has been enhanced by various effective possibilities to perform massive blast searches in less time, also permitting the search against multiple DB sources. A species filter can be applied to select results obtained from extensive DB's like the non redundant (nr) NCBI sequence DB. During the species independent annotation mining, blast results hits are now matched with Gene Ontology terms, enzyme code, KEGG pathways and in near future InterPro domains. The automatic annotation module offers the possibility of manual curation making use of various linked web resources. Two extra functionalities are available for modulating the intensity of annotation: the &quot;Second Layer&quot;approach that enhances GO annotation by considering GO term cross-relationships,and GO-Slim simplification which is supported for various species-specific &quot;slim&quot;mappings.<br>
										<br>Blast2GO is a freely available desktop Java application that can be started by Java Web Start technology. B2G is platform-independent, installable with minimal requirements and automatically updated via internet. Additionally,the annotation module is also amenable to annotation pipelines through the B2G4Pipe command line interface.<br>
										<br>
											The software demo will include the following aspects:<br>
											-Sequence annotation using diversity of databases<br>-Manual curation<br>
											-Modulation of annotation intensity and diversity<br>
											-Formats and alternative ways for data import and export<br>
											-Explorative analysis through interactive combined GO Graphs<br>
											-Generation of annotation statistics and visualization<br>
											-Statistical analysis of GO term distributions in functional genomics data<br/>
											
										<hr/>
									</font></td>
							</tr>
							<!--
							<tr>
								<td><a id="14" name="14"></a><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>What's new with ArrayExpress</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br/>
										</b>Misha Kapushesky, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">EBI, UK</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>ArrayExpress is a public repository for microarray data, which is aimed at storing MIAME-compliant data in accordance with MGED recommendations. The ArrayExpress Data Warehouse stores gene-indexed expression profiles from a curated subset of experiments in the repository. We will demonstrate a new interface to  the ArrayExpress database, new workflow and web service capabilities  of its data analysis tools, REST interfaces for automated database queries, novel functionality for the gene expression warehouse and  advanced data submission tools</font>.<br>
									<hr>
								</td>
							</tr>
							-->
							<tr>
								<td>
									<p><a id="15" name="15"></a><br>
										<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Agilentâ€™s CGH Analytics Solution for Comparative Genomic Hybridization Studies</b><br>
												Ewan Hunter, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Agilent Technologies</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
										</font></p>
								</td>
							</tr>
							<tr>
								<td>
									<p><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Agilent CGH Analytics software provides an intuitive user interface for visually exploring, detecting and analyzing aberration patterns from multiple Comparative Genomic Hybridization (CGH) microarray profiles. It includes several aberration detection algorithms and mechanisms as well as methods that detect common aberrations and correlations with expression data.<br>
										</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">This demonstration is dedicated to live data analysis of a data set in CGH analytics with relevance to transcription data, allowing the study of associations between DNA copy number and gene expression profiles.<br>
										</font></p>
									<hr/>
								</td>
							</tr>
							<tr>
								<td><a id="7" name="7"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>CFinder: Locating Cliques and Overlapping Modules in Biological Networks<br>
										</b>Gergely Palla<b>, </b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Biological Physics Research Group of the Hungarian Academy of Sciences, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
										
Budapest, Hungary</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">High-throughput experimental techniques, e.g., protein-protein interaction </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">(PPI) and mRNA expression methods, have largely advanced our knowledge </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">about the functioning of the cell. Gene (protein) association networks integrate </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">the broadest possible set of evidence -- including high-throughput data </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">-- on protein linkages: they provide an integrated list of binary interactions </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[1, 2] and allow the discovery of previously uncharacterized cellular systems </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[3]. One major goal of current research efforts is to elucidate how the observed </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">behaviors of an entire cell can be understood in terms of the interactions of its </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">protein modules. To identify such modules, a common approach is to search </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">for groups of densely interconnected nodes in the cell's protein association </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">network [4, 5]. Note, however, that modules strongly overlap. According to </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">the CYGD database [6], in Saccharomyces cerevisiae the number of proteins </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">in known protein complexes (modules where the participating proteins physically </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">interact at the same time) vs. the sum of the sizes of these complexes </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">is 2750/8932. Thus, most protein modules probably share many of their </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">proteins with other modules.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">We introduce CFinder [7], a platform-independent application locating </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">overlapping groups of densely interconnected nodes in graphs, and illustrate </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">its use on biological networks. Generic graph visualization and analysis programs </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[8] are frequently used for the layout and structural analysis of networks. <br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Recent bioinformatics software platforms [9], on the other hand, enable </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">the user to integrate many different types of data, e.g., PPI, expression </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">levels, and annotation information. CFinder reads a list of binary interactions, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">performs a search for dense subgraphs (groups), and -- unlike several </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">currently used algorithms [10] -- it allows for any node to belong to more </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">than one group.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Due to its algorithm and implementation, CFinder is e&#177;cient for networks </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">with millions of nodes and, as a byproduct of its search, the full clique </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">overlap matrix of the network is determined. Below we will show that in gene </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">association networks CFinder's results can be used to predict novel modules </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and novel individual protein functions.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Overview of CFinder<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The input of CFinder is a file containing strings and numbers ordered into </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">three columns; in each row the first two strings correspond to the two end </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">points of a link and the third item is the weight of this link.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The computational core of CFinder was implemented in C++, while the </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">visualization and analysis components were written in Java. The search algorithm uses the Clique Percolation Method (CPM, see [11]) to locate the </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">k-clique percolation clusters of the network that we interpret as modules. A k-</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">clique is a complete subgraph on k nodes (k = 3; 4; : : :), and two k-cliques are </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">said to be adjacent, if they share exactly k-1 nodes. A k-clique percolation </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">cluster consists of (i) all nodes that can be reached via chains of adjacent </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">k-cliques from each other and (ii) the links in these cliques. Note that larger </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">values of k correspond to a higher stringency during the identification of </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">dense groups and provide smaller groups with a higher density of links inside </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">them. For both local and global analysis in a network, we suggest using </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">such a value of k (typically between 4 and 6) that provides the user with the </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">richest group structure (see [12]). In the presence of link weights CFinder </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">can apply lower and upper cutoff values to keep only the set of connections </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">judged to be significant by the user.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The user interface of CFinder offers several views of the analyzed network </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and its module structure. As an example, Fig. 1. shows the modules of the </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">protein Pwp2 in the DIP &quot;yeast core&quot; network [2] at clique size k = 4. Alternative </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">views currently available in CFinder are &quot;Communities&quot; (displaying </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">the identified modules), &quot;Cliques&quot;, &quot;Stats&quot; (statistics of, e.g., module and </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">overlap sizes) and \Graph of communities&quot;. The special buttons &quot;forward&quot;, &quot;back&quot;, &quot;zoom&quot; and &quot;walk&quot; allow a quick navigation between the views. A </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">wide variety of visualization settings can be adjusted in the &quot;Tools&quot; menu.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">In Fig. 2. we display the network of modules produced by CFinder (k = 4) </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">in the DIP &quot;yeast full&quot; data set. In the complete map (a) each node represents </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">a module, the area of a node is proportional to the number of proteins in </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">the corresponding module, and the width of a link is proportional to the </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">number of proteins shared by the two modules. Panel (b) shows a previously </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">known complex identified by CFinder. Panels (c) and (d) both display a </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">known complex grouped together with one additional protein (Msh2 and </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Vps8, respectively), leading to an improved functional annotation of that </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">protein. In panel (e) Eeb1 (function currently unknown) is grouped together </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">with proteins participating in vesicle-mediated transport, thus, we predict </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">this to be a key function of Eeb1. Proteins in the marked dark blue and </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">brown groups of panel (e) cooperate on the establishment of cell polarity, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">a function performed by a total of 103 proteins in the cell. We anticipate </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">that these two groups are biologically meaningful, novel modules within that </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">larger set of 103 proteins. (Gene names and annotations were handled with </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Perl tools, e.g., GO::TermFinder [13].)<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">References<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[1] von Mering,C. et al. (2005) STRING: known and predicted protein-protein associations, integrated and transferred across organisms. Nucleic Acids Res. 33, D433-D437.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[2] Salwinski,L. et al. (2004) The Database of Interacting Proteins: 2004 update. Nucleic Acids Res., 32, D449-451.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[3] Date,S.V. and Marcotte,E.M. (2003) Discovery of uncharacterized cellular systems by genome-wide analysis of functional linkages. Nat. Biotechnol., 21, 1055-1062.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[4] Bader,G.D. and Hogue,C.W.V. (2003) An automated method for finding molecular complexes in large protein interaction networks. BMC Bioinformatics, 4, 2.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[5] Rives,A.W. and Galitski,T. (2003) Modular organization of cellular networks. Proc. Natl. Acad. Sci. U S A, 100, 1128-1133.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[6] Guldener,U. et al. (2005) CYGD: the Comprehensive Yeast Genome Database. Nucleic Acids Res., 33, D364-D368.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[7] Adamcsek, B. et al. (2006) CFinder: Locating cliques and overlapping modules in biological networks Bioinformatics, 22, 1021-1023.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[8] Batagelj,V. and Mrvar,A. (1998) Pajek - program for large network analysis. Connections, 21, 47-57.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[9] Shannon,P. et al. (2003) Cytoscape: A software environment for integrated models of biomolecular interaction networks. Genome Res., 13, 2498-2504.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[10] Newman,M.E.J. (2004) Detecting community structure in networks. Eur. Phys. J. B, 38, 321-330.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[11] Derenyi,I. et al. (2005) Clique percolation in random networks. Phys. Rev. Lett., 94, 160202.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[12] Palla,G. et al. (2005) Uncovering the overlapping community structure of complex networks in nature and society. Nature, 435, 814-818.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">[13] Boyle,E.I. et al. (2004) GO::TermFinder - open source software for accessing Gene Ontology information and finding significantly enriched Gene Ontology terms associated with a list of genes. Bioinformatics, 20, 3710-3715.<br>
									</font>
									<hr/>
								</td>
							</tr>
							<tr>
								<td><a id="6" name="6"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>SABIO-RK (System for the Analysis of Biochemical Pathways - Reaction Kinetics)<br>
										</b>Olga Krebs<b>,</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Scientific Databases and Visualization Group, EML Research gGmbH,</font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Germany</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The simulation of biochemical reaction networks requires experimental data about substance concentrations, metabolic fluxes, enzyme activities, or reaction kinetics, describing the dynamics of the reactions with their respective parameters determined under certain experimental conditions. These data are widely scattered through various publications and described in many different formats.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">To address these needs we have developed an integrated biological pathway database system, SABIO-RK (System for the Analysis of Biochemical Pathways - Reaction Kinetics), to store and offer access to information about biochemical reactions and their kinetics in a comprehensive and standardised manner. It stores the fundamental information about biochemical pathways, like reactions and their participants (enzymes, compounds, modifiers) and descriptive information about proteins, protein complexes and genes, all this linked to organism (including strains) and to biochemical reactions (in the case of enzymes). Most of the reactions, their associations with biochemical pathways, and their enzymatic classifications (EC number of the International Union of Biochemistry and Molecular Biology) are extracted from the KEGG database (Kyoto Encyclopedia of Genes and Genomes).<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The kinetic data is mainly manually extracted from published scientific articles and then verified by curators. It includes data about the organism, tissue, and cellular location where the reaction takes place, as well as the type of the kinetic law and the reaction's rate equation. The latter is shown with its parameters (Km, Vmax, concentration, etc.) and the experimental conditions (e.g. pH, temperature, buffer) under which the parameters were determined.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">During the curation process, the data is unified and structured consistently in order to facilitate the comparison of the kinetic data extracted from different sources, since they are usually obtained under different experimental conditions or from different organisms, tissues etc.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The data integration process is supported by a web-based input interface which allows users to load and store information about reactions and enzyme kinetics. It matches in most points with the current recommendations for standards by the STRENDA commission (<a href="http://www.strenda.org/" target="_blank">http://www.strenda.org/</a>) especially with respect to the kinetics of enzymes and reactions.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">SABIO RK has a web-based user interface that enables the user to search for biochemical reactions and their kinetics, based on the characteristics of the reactions and on the environmental conditions under which its kinetics were obtained. For example the user can define a pathway, e.g. glycolysis; or can specify reaction participants (compounds or enzymes), organisms, tissues, or cell types in which the reaction is reported to occur. Additional search terms include cellular locations, environmental conditions (pH and temperature), or publications in which kinetic data are reported. The interface enables the export of data in SBML (Systems Biology Mark-Up Language) format that can then be used as the basis to create a simulation model of a biochemical network</font>.
									<br>
									<hr size="2"/>
								</td>
							</tr>
							<tr>
<td><a id="1" name="1"></a><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>The</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b> </b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>SimpAT Package: Integrating SIMAP into its own Applications</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br>
										</b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Roland Arnold, Thomas Rattei, Volker Stumpflen,</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> and </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Hans-Werner Mewes</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
											MIPS/IBI Institute for Bioinformatics at the GSF - National Research Center for Environment and Health, GmbH, Germany<br>
									</font></td>
</tr>
							<tr>
<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
<b>Abstract:</b><br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Similarity searches of a query protein between a set of other proteins is one of the basic procedures in many in silico analyses. They are used to detect homologous sequences in order to transfer annotation, detect members of a protein family, find putative orthologs, create phylogenetic profiles and in many more applications. Very often, this similarity searches query for sequences which are already known in public databases and are therefore redone again and again by different scientists or even different rounds of one analysis. BLAST (1) is therefore the most prominent heuristic for similarity searches, because of its speed.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">SIMAP(2), the Similarity Matrix of Proteins, is a database containing pre-calculated similarity searches of protein sequences based on the Fasta 3 (3) procedure. The datapoints in SIMAP are saved as ordered lists of hits which can be retrieved very rapidly. The database covers a comprehensive list of public databases as SWISSPROT/UNIPROT (4) or the NCBI gene-bank (6) and contains almost every sequenced genome in the public domain. Taxonomic information is added to each protein-entry.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">SIMAP is accessible via the Web Service (<a href="http://www.w3.org/2002/ws/" target="_blank">http://www.w3.org/2002/ws/</a>) technology. To simplify the integration of the SIMAP Web Service into own applications, we developed SimpAT, the SIMAP Access Tool. SimpAT is a Java package which can be used without any knowledge about Web Service technology. This integration can be done either directly in the Java code or in any other program via a wrapper application. This wrapper application can also be used as stand-alone client. Since SimpAT makes use of Web Service technology to invoke SIMAP via the internet, no parts of SIMAP itself have to be downloaded and the data fetched is always up-to-date.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">SimpAT retrieves the result as a comprehensive XML document. This document is automatically parsed into convenient java classes, so no own parsing is necessary.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The query scheme is very flexible and based on the selection of appropriate cut-offs and search-spaces. These parameters are evaluated on the server-side to minimize download-traffic. Queries to SIMAP use the unique md5 key (5) of a amino-acid sequence, or the sequence itself, or a protein identifier in combination with a specific database, avoiding identifier problems across different databases. The search-space can be defined by the selection of (several) individual datasets or types of datasets (as 'complete genomes') or by including and excluding branches of the taxonomy or a combination of them. Therefore queries like &quot;return all hits with an E-value better than 10e-5 in UNIPROT but only of eukaryotic origin which are not human&quot; can be formulated in a few lines of code.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The result attributes include taxonomic information for each protein entry returned referencing the NCBI taxonomic database (5) and a link-out to the database of origin as HTML-link. Also, synonymous protein entries in other databases are returned. It also contains E-values, similarity scores as the Smith-Waterman alignment score, the bit-score, the identity, the coordinates of the Smith-Waterman alignment for both sequences (for details see (2)).<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">SimpAT also directly supports access to data of putative orthologs defined by bi-directional hits and putative in-paralogs.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">In the software demonstration, we will perform a live implementation of two typical use-cases.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The first part of the demonstration will show how to retrieve a hit-list of a certain sequence. We will demonstrate how to define customized search-spaces and the usage of taxonomic filtering and discuss the possible cut-off parameters. We will implement an example application which answers questions like &quot;which sequences of a bunch of input-sequences have close homologs only in bacteria but not in eukaryota&quot; in a few lines of code.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The second part will demonstrate how to use SIMAP for the prediction of orthologs. As examples, we will discuss a very short implementation to create phylogenetic profiles and a function which returns the number of putative gene duplications after a speciation event, both for a complete protein set of a genome by one rapid whole genome query.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The presentation targets on software developers in projects dealing with large amounts of similarity searches but also on non-programmers since SIMAP can be accessed using the wrapper application without further programming. Knowledge of Java is therefore desired but not recommended.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">1. Altschul, S. F., Gish, W., Miller, W., Myers, G. and Lipman, D. J. (1990) A basic local alignment search tool. J. Mol. Biol., 215, 403410<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">2. Arnold R, Rattei T, Tischler P, Truong MD, Stumpflen V, Mewes W. (2005) SIMAP--The similarity matrix of proteins. Bioinformatics. Sep 1;21 Suppl 2:ii42-ii46<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">3. Pearson, W. R. (2000) Flexible sequence similarity searching with the FASTA3 program package. Methods Mol. Biol. 132, 185-219<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">4. Bairoch A, Apweiler R, Wu CH, Barker WC, Boeckmann B, Ferro S, Gasteiger E, Huang H, Lopez R, Magrane M, Martin MJ, Natale DA, O'Donovan C, Redaschi N, Yeh LS. (2005) The Universal Protein Resource (UniProt). Nucleic Acids Res. 2005 Jan 1;33(Database issue):D154-9<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">5. Smith M, Kunin V, Goldovsky L, Enright AJ, Ouzounis CA. (2005) MagicMatch--cross-referencing sequence identifiers across databases. Bioinformatics, Aug 15;21(16):3429-30.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">6. Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, Chetvernin V, Church DM, DiCuccio M, Edgar R, Federhen S, Geer LY, Helmberg W, Kapustin Y, Kenton DL, Khovayko O, Lipman DJ, Madden TL, Maglott DR, Ostell J, Pruitt KD, Schuler GD, Schriml LM, Sequeira E, Sherry ST, Sirotkin K, Souvorov A, Starchenko G, Suzek TO, Tatusov R, Tatusova TA, Wagner L, Yaschenko E. (2006) Database resources of the National Center for Biotechnology Information. Nucleic Acids Res. 2006 Jan 1;34(Database issue):D173-80.<br>
									</font>
									<hr/>
								</td>
</tr>
<!--<br>-->
							<tr>
								<td><a id="5" name="5"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>COLOMBO/SIGI-HMM: Prediction of Genomic Islands in Procaryotic Genomes using HMMs<br>
										</b>Oliver Keller</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Institut fur Informatik,</font> <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Goettingen</font>, <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Germany<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>The process of acquiring genes from foreign species, with or without a closer taxonomical relation to the host, is called horizontal gene transfer. This phenomenon is frequent among microbial species and is considered one major means of rapid adaptation to changing environmental demands. Researchers are seeking for algorithms that detect anomalies characterising, for example, horizontally transferred genes, often found in larger contiguous regions called genomic islands. We have implemented the algorithm SIGI-HMM that predicts GIs in Procaryotic Genomes, as well as the putative source of each individual alien gene.<br>
										<br>
											Benchmark tests showed the output of SIGI-HMM in agreement with known findings, consistent both with annotated GIs and predictions generated by other methods. It can be concluded that SIGI-HMM is a valuable tool for the identification of GIs in microbial genomes. It is publicly available and ready to use. It processes annotated genomic sequences given in EMBL format, and can as well produce GFF output. <br/>
										<hr/>
									</font></td>
							</tr>
							<tr>
								<td><a id="4" name="4"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>The Scriptome: Protocols</b></font><font size="1" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b> </b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>for Manipulating Biological Data</b></font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br/>
												</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Amir Karger</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Bauer Center for Genomics Research, Harvard, USA<br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract: <br/>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Scriptome tools perform simple analysis and formatting of biological data. The focus is on basic biological data manipulation, not complex statistics or algorithms. There are currently six tool categories: Calculate, Change, Choose, Fetch, Merge, and Sort. Popular tools include:<br>
											- Choosing lines where the value in a given column exceeds a certain threshold. This can be especially useful with files larger than Excel's limit of 65535 lines.<br>
											- Merging files together based on shared values in certain columns. This tool essentially performs a SQL join.<br>
											- Changing FASTA files to a tabular format. The output can be viewed in Excel, or filtered with other Scriptome tools.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Tool implementation<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Each tool is a short Perl script embedded in a one-line shell command. These tools can be cut and pasted from the website onto the command line. The tools' simplicity makes it possible to develop tools rapidly, to keep up with biologists' changing needs.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>User interface<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The interface is extremely lightweight. Users browse the web site's hierarchical table of contents to find a tool, then cut and paste the tool text onto the command line. They edit filenames and parameters, which are highlighted in red on the website, and then hit Enter to run the tool.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Building Protocols<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Once biologists can use single tools, they may begin the more difficult process of chaining tools together to form protocols (workflows). In the command-line interface, protocols become batch scripts; alternatively, some users have chosen to save the tools in a Word document. Users can therefore create reproducible, commented protocols.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Benefits<br>
											<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Ease of installation<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Scriptome requires no installation at all on UNIX and Macintosh, since Perl is standard on those platforms. Windows users need only a one-click installation of Perl from ActiveState, <a href="http://www.activestate.com/Products/ActivePerl" target="_blank">http://www.activestate.com/Products/ActivePerl</a>. The Scriptome has been tested for compatibility with versions of Perl as old as 5.005_03, released in 1999. A few tools also require Bioperl, <a href="http://www.bioperl.org/" target="_blank">http://www.bioperl.org</a>.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Ease of learning</b><br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Scriptome avoids introducing a new, graphical user interface that will require user training. The tools perform simple tasks requiring only a few parameters. The website features a hierarchical, English table of contents (so that users do not need to memorize command names), along with concise documentation and examples. Most users can therefore begin using tools after only a few minutes of training.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>High-throughput data analysis without programming<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">By using the Scriptome, experienced programmers can avoid rewriting commonly used tools. Novice programmers can also learn from its short, working, relevant Perl examples. But most importantly, the Scriptome allows <i>non</i>-programmers to perform high-throughput analysis without learning any programming at all.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Some argue that all biologists should learn how to program. However, many biologists focus on performing laboratory experiments, spending only a limited time analyzing data. There would not be enough return on the investment of time needed to become effective programmers. In addition, in the weeks or months before the next analysis, they could lose their programming skills.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Conclusion<br>
										</b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Despite the wide range of available software tools and programming languages, most biologists still rely on older methods for high-throughput data analysis. When confronted with large data sets in incompatible formats, the average biologist will hand-edit in Excel, depend on bioinformaticists, or just give up. The most important factor behind this problem is the barrier to entry: biologists are simply too busy keeping up with the rapid development of their own field to learn how to program or operate complicated software. The Scriptome focuses on ease of learning and installation to surmount this initial barrier. In doing so, it deliberately sacrifices the scope and complexity of more sophisticated software tools or full fledged programming.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The Scriptome's early users have quickly recognized that the Scriptome empowers them. On their own, without extensive training, they can filter and reformat data, perform high-throughput analysis, and explore their data in new ways.</font><br>
									<hr/>
								</td>
							</tr>
							<tr>
								<td><a id="11" name="11"></a><br/>
									<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>DeltaProt: Molecular Comparison of Proteins Based on Sequence Alignments<br>
												A Matlab(c) Companion Toolbox<br>
											<a href="http://www.math.uit.no/bi/deltaprot/" target="_blank">http://www.math.uit.no/bi/deltaprot/</a><br/>
													</b>Steinar Thorvaldsen, Tor FlÃ¥, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">and</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> Nils P. Willassen, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of Tromso and Norwegian Structural Biology Centre, Tromso, Norway</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font></td>
							</tr>
							<tr>
								<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
													</b>We present statistical methods, trend-tests and visualisations that are useful when the protein sequences in alignments can be divided into two or more populations based on known phenotypic traits such as preference of temperature, pH, salt concentration or pressure. The algorithms have been successfully applied in the research on extremophile organisms.<br>
										<br>
											References:<br>
										S. Thorvaldsen, T. Fl&aring; and N. P. Willassen: Extracting molecular diversity between populations through sequence alignments. Lecture Notes in Bioinformatics, Vol. 3745, Springer-Verlag 2005, pp. 317-328.<br>
											S. Thorvaldsen, E. Ytterstad and T. Fla: Property-dependent analysis of aligned proteins from two or more populations. Proceedings of the 4th Asia-Pacific Bioinformatics Conference (Eds.: T. Jiang et al.). Imperial College Press 2006, pp. 169-178.<br>
										<br>
											Download Matlab code of the DeltaProt Toolbox <a href="http://www.math.uit.no/bi/deltaprot/deltaprot.zip" target="_blank">here</a>! <br>
											This software can be used freely for academic, non-profit purposes.<br>
											Note: Matlab Statistical Toolbox is used in some functions.<br>
									</font>
									<hr/>
								</td>
							</tr>
							<tr>
								<td><a id="16" name="16"></a><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Agilent's &quot;Gene Spring&quot; Platforms for Gene Expression, Metabolite, &amp; Biomarker Studies<br>
										</b>Ewan Hunter, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Agilent Technologies</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font></td>
							</tr>
							<tr>
								<td>
									<p><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">GeneSpring GX - flexible software that enables statistical analysis, biological exploration, visual interpretations, hypothesis generation and hypothesis testing of microarray results from any platform. The Gene Spring GX platform supports several statistical methods for clustering, differential expression, enrichment analysis and more. It offers easy to use and intuitive user-interface and a variety of tools to support the generation of publication grade graphical representation of results.<br>
											<br>
										</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">GeneSpring MS software enables the rapid discovery of protein and metabolite biomarkers through the analysis of mass spectrometry data. Researchers can easily import, analyze and visualize GC/MS and LC/MS data from large sample sets and complex experimental designs. Using a comprehensive array of powerful statistical methods, GeneSpring MS can profile proteins or small molecules associated with changes in cellular function, enabling the rapid discovery of biomarkers and thus possibly detect disease or drug toxicity.<br>
											<br>
										</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">This demonstration is dedicated to live data analysis, using GS GX &amp; GS MS of the data reported in Xie Lu et al â€œGenomic and Proteomic Analysis of Mammary Tumors Arising in Transgenic Miceâ€ Journal of Proteome Research 2005, 4, 2088-2098</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
										</font></p>
									<hr/>
								</td>
							</tr>
							<tr><td><a id="2" name="2"></a><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>CoryneRegNet: An Integrative Bioinformatics Platform for the Analysis of Transcription Factors and Regulatory Networks<br/>
										</b>Jan Baumbach, Sven Rahmann,</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> and </font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Andreas Tauch</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Bielefeld University,Germany<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
									</font></td>
</tr>
							<tr>
<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/></b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Background: Recently several whole-genome sequencing projects have generated huge amounts of data related to various organisms including gene sequences and their functional annotations. Since the gene activity varies under different conditions the goal is to understand the process of their (transcriptional) regulation. The application of post-genomic analysis techniques to bacterial genome sequences provides knowledge to encoded proteins involved in the gene regulation. This data along with literature-derived knowledge on the regulation of gene expression has opened the way for genome-wide reconstruction of transcriptional regulatory networks. These large-scale reconstructions can be converted into in silico models of corynebacterial cells that allow systematic analysis of network behaviour in response to changing environmental conditions. Besides pathogenic corynebacterial species of medical importance, like C. diphtheriae that causes the upper respiratory tract illness diptheria and C. jeikeium, other corynebacteria like C. glutamicum and C. efficiens are traditionally used in biotechnological production processes particularly of amino acids.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Description: CoryneRegNet [1] is an ontology-based data warehouse designed to facilitate the genome-wide reconstruction of transcriptional regulatory networks of corynebacteria relevant in biotechnology and human medicine. CoryneRegNet is based on a multi-layered, hierarchical and modular concept of transcriptional regulation and was implemented by using an ontology-based data structure. We integrated PoSSuMsearch [2], a fast and statistically sound method to detect transcription factor binding site motifs within and across species. CoryneRegNet provides an user-friendly interface to PoSSuMsearch that is the only available software package that is fast enough to provide interactive response times for large-scale Position Specific Scoring Matrix (PSSM) searches and at the same time integrates exact statistics for p-value computations. Reconstructed regulatory networks can be visualized on a web interface and as graphs. Special graph layout algorithms have been developed and implemented to facilitate the comparison of gene regulatory networks across species and to assist biologists with the evaluation of predicted and graphically visualized networks.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Conclusion: CoryneRegNet allows a pertinent data management of regulatory interactions along with the genome-scale reconstruction of transcriptional regulatory networks. These models can further be combined with metabolic networks to build integrated models of cellular function.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">The public release of CoryneRegNet is freely accessible at<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a href="https://www.cebitec.uni-bielefeld.de/groups/gi/software/coryneregnet/" target="_blank">https://www.cebitec.uni-bielefeld.de/groups/gi/software/coryneregnet/</a>. The final slash (/) is mandatory. In order to use the GraphVis feature, Java (at least version 1.4.2) is required.<br>
										<br>
										References<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">1. Baumbach J, Brinkrolf K, Czaja LF, Rahmann S, Tauch A: CoryneRegNet: An ontology-based data warehouse of corynebacterial transcription factors and regulatory networks. BMC Genomics 2006, 7(1):24.<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">2. Beckstette M, Strothmann D, Homann R, Giegerich R, Kurtz S: PoSSuMsearch: Fast and Sensitive Matching of Position Specific Scoring Matrices using Enhanced Suffix Arrays. GI Lecture Notes in Informatics 2004:53-64.<br>
									</font>
									<hr size="1"/>
								</td></tr>
							<tr>
								<td><br>
								</td>
							</tr>
							<tr>
								<td><br>
								</td>
							</tr>
						</tbody></table>
	</body></html>