<html>
<head>
<title>Poster Abstracts P</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category P: Transcriptomics</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="P01" name="P01"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P01</b><br>
    <b>A Conserved Genetic Signature of Gene Expression Divergence</b><br>
    Itay Tirosh, Adina Weinberger, Naama Barkai<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Only little is known about the principles driving the evolution of gene expression. Here we describe the differential transcriptional response of four closely related yeast species to a variety of environmental stresses.  Genes containing a TATA-box in their promoters show an increased inter-species variability in expression, independently of their functional association. Examining additional datasets, we find that this enhanced expression divergence of TATA-containing genes is consistent across all eukaryotes studied to date.
<br><br>Contact: itay.tirosh [at] weizmann.ac.il
<br><br>Keywords: Gene Expression, Microarrays, Evolution
</font><hr></td></tr>
<tr>
  <td><a id="P02" name="P02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P02</b><br>
    <b>Derivation of Valid Biological Data from Cross-Species Hybridizations (CSHs) to cDNA Microarrays</b><br>
    Carmiya Bar-Or (1,2), Eugene Novikov (3), Anat Reiner (4), Henryk Czosnek (2), Hinanit Koltai (1)<br>
    (1) Department of Ornamental Horticulture, ARO Volcani Center, Bet Dagan, Israel; (2) Faculty of Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel; (3) Service Bioinformatique, Institute Curie, Paris, France; (4) Department of Physics of Complex Systems, Weizmann Institute, Rehovot, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    CSHs have become a powerful tool for studying diversity due to the lack of available microarrays of most complex species. Previous CSH studies resulted in contradicting results, questioning the ability of this technology to reflect biological phenomenon. The present study examined public data of CSH experiments and detected raw data features that were correlated with the phylogenetic distance between the species providing the RNA and the species providing the microarray probes. Utilization of these features for filtration of CSH data greatly enhanced the validity of the biological results.
<br><br>Contact: carmiya [at] agri.gov.il
<br><br>Keywords: Cross-Species Hybridization, cDNA Microarray
</font><hr></td></tr>
<tr>
  <td><a id="P03" name="P03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P03</b><br>
    <b>Expression Profiling of Drosophila Heart Remodelling during Metamorphosis</b><br>
    Bruno Zeitouni, Bruno Monier, Michel Semeriva, Laurent Perrin<br>
    CNRS UMR 6216, Institut de Biologie du Developpement de Marseille-Luminy (IBDML)
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    To understand the molecular mechanisms of Drosophila heart remodelling during metamorphosis, we analyzed by a global genomic approach, the kinetics of the modifications of gene expression during this process. Transcriptome analysis at 8 different time-points covering cardiac tube remodelling revealed a highly dynamic transcriptional profile with a dozen clusters of genes involved in dedicated functions such as myogenesis or specific signalling pathways. Our study provides a genome-wide basis for understanding how a mechanism of cell reprogramming is genetically controlled.
<br><br>Contact: zeitouni [at] ibdml.univ-mrs.fr
<br><br>Keywords: Drosophila, Remodelling, Expression Profiling
</font><hr></td></tr>
<tr>
  <td><a id="P04" name="P04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P04</b><br>
    <b>Regulating Signals on Alu Play a Significant Role in Development</b><br>
    Paz Polak (1), Eytan Domany (2)<br>
    (1) Department of Computational Biology, Max Planck Institute for Molecular Gentics, Berlin; (2) Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    More than half a million putative transcription factors binding sites (TFBSs), residing on Alu repeats located in the region 5Kbp upstream to the transcription start sites (TSS) of 14000 genes. Many of these TFs are transcriptional regulators of development. 
<br>The Alu content in promoters of genes was found to be correlated with their biological role. This finding led us to propose a novel mechanism, which takes advantage on the developmental regulatory signals that reside on Alu, to regulate differentiation and proliferation.
<br><br>Contact: paz.polak [at] weizmann.ac.il
<br><br>Keywords: Alu, Development
</font><hr></td></tr>
<tr>
  <td><a id="P07" name="P07"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P07</b><br>
    <b>Embryonic Genes are Reactivated in Human Cancers</b><br>
    Roi Gilat, Dorit Shweiki<br>
    Bioinformatics Program, School of Computer Science, The Academic College of Tel Aviv-Yaffo, Tel Aviv, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Embryogenesis and tumorogenesis share mutual patterns of gene expression, and a significant change in genomic DNA methylation patterning. ESTs-based study was designed in order to identify human embryonic genes which are reactivated in human cancers. Gene Ontology (GO) categories clustering revealed a significant enrichment of genes which play a role in nucleic-related processes (transcription, DNA replication, cell cycle, etc).  CpG promoter analysis of those genes implies on a possible complex involvement of DNA methylation in embryonic cancer-reactivated genes. 
<br><br>Contact: dorits [at] mta.ac.il
<br><br>Keywords: Embryonic Genes, Cancer, DNA Methylation, GO
</font><hr></td></tr>
<tr>
  <td><a id="P09" name="P09"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P09</b><br>
    <b>Expression Pattern Analysis of Housekeeping Genes Across Large Number of Microarray Experiments</b><br>
    Levent Carkacioglu (1), Ozlen Konu (2), Tolga Can (1), Volkan Atalay (1), Rengul Cetin-Atalay (2)<br>
    (1) Middle East Technical University Department of Computer Engineering; (2) Bilkent University Department of Molecular Biology and Genetics
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Recent advances in large scale transcriptome analysis result in data accumulation at a large scale. Transcriptome data must be comparable although they are generated from different experiments. Our basic aim is to compare gene expression data across different experiments. We have characterized expression patterns of a published set of housekeeping genes across large number of microarray experiments from NCBI-GEO. Our study has supported the claim that housekeeping gene expression is less variable across different experiment sets when compared with randomly selected gene sets.
<br><br>Contact: leventc [at] ceng.metu.edu.tr
<br><br>Keywords: Transcriptome, Microarray, Houkeeping Genes
</font><hr></td></tr>
<tr>
  <td><a id="P10" name="P10"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P10</b><br>
    <b>Effect of Functional Variants on Gene Expression in Human Brain</b><br>
    V Strumba (1), T Blackwell (1), E Sliwerska (1), R Bernard (1), J Li (2), AJ Schatzberg (2), EG Jones (3), WE Bunney (4), RM Myers (2), H Akil (1), SJ Watson (1), M Burmeister (1)<br>
    (1) University of Michigan; (2) Stanford University; (3) UC Davis; (4) UC Irvine
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Many polymorphisms have been found associated with behavior or psychiatric disorders. However, the mechanism of how genetic variants lead to phenotypic differences is usually not known. Towards this goal, we test for association between functional variants in candidate genes and expression levels of thousands of genes. Specifically, we evaluated variants in the SLC6A4 (5-HTT), COMT and MAOA genes and expression levels measured by Affymetrix microarrays performed on mRNAs isolated from postmortem brains. Genes identified as changed are then analyzed for the biological pathways involved.
<br><br>Contact: vresnik [at] umich.edu
<br><br>Keywords: Genetical Genomics, Microarray, COMT, 5-HTTLPR
</font><hr></td></tr>
<tr>
  <td><a id="P11" name="P11"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P11</b><br>
    <b>Elucidating Factors Influencing Probes Sensitivity and Specificity in High Density Micro Arrays Using Affymetrix Setup</b><br>
    Hadar Less, Gad Galili<br>
    Department of Plant Sciences, Weizmann institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Affymetrix technology uses probe pairs of perfect match (PM) and mismatch (MM) to monitor gene products. In this study we used the PM/MM ratio as a measure to better understand some of the various factors that influence probe pairs sensitivity and specificity under the competitive conditions of high density micro array. Our results indicate that the three most important factors are the expression level of the monitored genes, the 13th mismatch nucleotide in which pyrimidine swapping is advantages upon purine swapping and the present of short sequence similarities to non target mRNA molecules.
<br><br>Contact: hadar.less [at] weizmann.ac.il
<br><br>Keywords: Affymetrix, Probe Level Analysis
</font><hr></td></tr>
<tr>
  <td><a id="P12" name="P12"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P12</b><br>
    <b>Over Represented and Functional Human Transcription Factor Binding Motifs Appear Mainly Within 200bp from the TSS</b><br>
    Yuval Tabach (1,2), Anat Reiner (1), Ran Brosh (2), Assif Yitzhaky (1), Varda Rotter (2), Eytan Domany (1)<br>
    (1) Department of Physics of Complex Systems, Weizmann Institute of Science, Rehovot, Israel; (2) Department of Molecular Cell Biology, Weizmann Institute of Science, Rehovot, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    One of the major challenges in systems biology is to predict gene expression by solving the promoters sequences. Taking into account positional bias and using a novel scoring method based on groups of putative co-regulated genes, we create a data base of motif over representation, in all GO groups, for 414 known transcription factors. Surprisingly, almost all motifs were found to be over represented nearly exclusively within 200bp from the TSS. These bioinformatic results were validated for cell cycle motifs and for NFKB using expression data, and for myocardin in a direct in-vitro experiment.
<br><br>Contact: yuval.tabach [at] weizmann.ac.il
<br><br>Keywords: Transcription Factors, NFKB, Cell Cycle
</font><hr></td></tr>
<tr>
  <td><a id="P13" name="P13"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P13</b><br>
    <b>A Thermodynamic Model for Understanding Spatio-Temporal Control of Gene Expression</b><br>
    Tali Sadka (1), Ulrike Gaul (2), Eran Segal (1)<br>
    (1) Weizmann Institute of Science, Rehovot, Israel; (2) Rockefeller University, New York, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Formation of complex spatio-temporal expression patterns is central to many cellular processes, yet mechanistic understanding of the underlying transcriptional regulation is still missing.  We present a novel thermodynamic model that integrates DNA sequence with spatio-temporal expression patterns of regulators, in an attempt to mimic cellular computations and predict gene expression patterns. We present results from an application of our model to the segmentation network of Drosophila. Our model accurately predicts expression patterns for many cis-regulatory modules using only their sequence.
<br><br>Contact: tali.sadka [at] weizmann.ac.il
<br><br>Keywords: Transcription, Thermodynamic Model, Learning 
</font><hr></td></tr>
<tr>
  <td><a id="P14" name="P14"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P14</b><br>
    <b>Analysis of Regulatory Element and Functional Class Associations: Comparison in Arabidopsis and Other Plants</b><br>
    Ananyo Choudhury, Ansuman Lahiri<br>
    University of Calcutta
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Using a site overrepresentation estimation based computational approach at the genomic level we identified ABRE associated functional classes (GO and metabolic pathway) in arabidopsis. The predicted associations are in excellent agreement with expression data and other experimental evidences wherever available. We analyzed the association of ABREs to the predicted functional classes phylogenetically. We also compared regulatory element enrichment among biosynthesis-degradation pathway pairs, interrelated pathway groups and ABA treatment induced expression groups in arabidopsis.
<br><br>Contact: ananyo.c [at] rediffmail.com
<br><br>Keywords: TFBS, GO, Pathway, ABRE
</font><hr></td></tr>
<tr>
  <td><a id="P16" name="P16"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P16</b><br>
    <b>A Hidden Markov Model Mixture for Estimating Human Endogenous Retrovirus Activities from Expressed Sequence Databases</b><br>
    Merja Oja (1,2), Jaakko Peltonen (2,1), Samuel Kaski (2)<br>
    (1) University of Helsinki, Finland; (2) Helsinki University of Technology, Finland
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Human endogenous retroviruses (HERVs) are remnants of ancient retrovirus infections and now reside within the human DNA. Recently HERV expression has been detected in both diseased patients and normal tissues. However, the expression levels of individual HERV sequences are mostly unknown. In this work we introduce a generative mixture model, based on Hidden Markov Models, for estimating the activities of the individual HERV sequences from databases of expressed sequences. We determined the relative expression levels of 91 HERVs; the majority  of their activities were previously unknown.
<br><br>Contact: merja.oja [at] hut.fi
<br><br>Keywords: HMM Mixture, Expression, HERV, EST Sequences
</font><hr></td></tr>
<tr>
  <td><a id="P17" name="P17"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P17</b><br>
    <b>Thousands of Samples are Needed to Generate a Robust Gene List for Predicting Outcome in Cancer</b><br>
    Or Zuk, Liat Ein-Dor, Eytan Domany<br>
    Department of Physics of Complex Systems, Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Predicting prognosis and metastatic potential of cancer is a major challenge in clinical research. Numerous studies searched for gene expression signatures outperforming clinical parameters in outcome prediction. Several groups published lists of genes with good predictive performance yet lists obtained by different groups had very few genes in common. The main source of the problem is the small sample size used to generate these lists. We evaluate the robustness of such lists, and calculate for several published datasets the sample size needed to achieve a desired stability level.
<br><br>Contact: or.zuk [at] weizmann.ac.il
<br><br>Keywords: Microarray, Outcome Prediction, Robustness
</font><hr></td></tr>
<tr>
  <td><a id="P18" name="P18"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P18</b><br>
    <b>Clustering Algorithms Optimizer - A Framework for Large Datasets</b><br>
    Roy Varshavsky (1),  David Horn (2), Michal Linial (3)<br>
    (1) School of Computer Science and Engineering, The The Hebrew University of Jerusalem, Israel; (2)School of Physics and Astronomy, Tel Aviv University, Israel; (3) Dept of Biological Chemistry,  The Hebrew University of Jerusalem, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Most clustering algorithms for gene-expression data use predetermined knowledge on the expected number of groups and often employ nondeterministic steps leading to inconsistent outcomes. We present a two steps framework that overcomes these shortcomings. The first step is SVD-based dimensional reduction and the second step is an automated tuning of the algorithm's parameter according to Bayesian Information Criterion. This framework can incorporate most clustering algorithms and improve their performance. We demonstrate the success of the framework for several gene-expression benchmarks.
<br><br>Contact: royke [at] cs.huji.ac.il
<br><br>Keywords: Gene Expression, Clustering Optimization
</font><hr></td></tr>
<tr>
  <td><a id="P19" name="P19"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P19</b><br>
    <b>Genome-Wide mRNA Decay Profiles Modeled by First-Principles Kinetics</b><br>
    Ophir Shalem (1,2), Yitzhak Pilpel (1), Eran Segal (2)<br>
    (1) Department of Molecular Genetics, Weizmann Institute of Science; (2) Department of Applied Mathematics and Computer Sciences, Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Steady state mRNA expression levels, widely used for high throughput genomic analyses, reflect a balance between two highly regulated processes: transcription initiation and mRNA decay. Although most of the attention is usually directed towards transcription initiation, mRNA degradation is known to have a significant impact on steady state levels. We have developed a computational framework based on first principles kinetics that allows for both the analysis of gene specific degradation and the identification of groups of genes whose degradation might be controlled by shared protein factors.
<br><br>Contact: ophir.shalem [at] weizmann.ac.il
<br><br>Keywords: Kinetic Model, mRNA Degradation
</font><hr></td></tr>
<tr>
  <td><a id="P20" name="P20"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P20</b><br>
    <b>Integrating Functional Knowledge during Sample Clustering for Microarray Data using Unsupervised Decision Trees</b><br>
    Henning Redestig (1), Dirk Repsilber (2), Florian Sohler (3), Joachim Selbig (1,2)<br>
    (1) Max Planck Institute for Molecular Plant Physiology; (2) University of Potsdam; (3) {Institute for Informatics, Ludwig Maximilian's University, Muenchen
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Sample-wise clustering of microarrays can reveal novel subclasses of  diseases and treatment responses. Classical methods for doing this depend on filtering genes for high-variance thus potentially losing information in relatively low-variant genes. Furthermore, classical clustering methods do not facilitate the biological interpretation of the results. In order to address these problems we propose to  integrate gene annotations directly in the clustering algorithm for  dividing samples on a one-feature-at-a-time basis. The proposed method  performed well on both real and simulated data.
<br><br>Contact: redestig [at] mpimp-golm.mpg.de
<br><br>Keywords: clustering, microarray
</font><hr></td></tr>
<tr>
  <td><a id="P22" name="P22"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P22</b><br>
    <b>Seeing the Forest for Hierarchical Subtrees: Improving Cluster Coherence and Functional Prediction by Tree Snipping</b><br>
    Dikla Dotan Cohen (1), Avraham A. Melkman (1), Simon Kasif (2,3,4)<br>
    (1) Department of Computer Science, Ben Gurion University, Beer Sheva, Israel; (2) Department of Biomedical Engineering, Boston University, MA; (3) Center for Advanced Genomic Technology, Boston University, MA; (4) Bioinformatics Program, Boston University, MA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In order to improve the qualitative interpretation and the biological significance of hierarchical clustering procedures, we develop a new framework of tree snipping which constrains the partition of the tree obtained from hierarchical clustering to be maximally consistent with partially available background knowledge in the form of functional gene classifications. We present two algorithms for judiciously partitioning the tree to optimize the functional enrichment of the resulting clusters.  One important application of the new framework is the problem of predicting biological function.
<br><br>Contact: dotna [at] cs.bgu.ac.il
<br><br>Keywords: Hierarchical Clustering,Functional Prediction
</font><hr></td></tr>
<tr>
  <td><a id="P23" name="P23"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster P23</b><br>
    <b>Wall-Guidance and Attraction in Mice Correlate With Gene-Expression Patterns in Specific Brain Areas</b><br>
    Guy Horev (1), Greg I. Elmer (2), Ilan Golani (1), Neri Kafkafi (2), Norman H. Lee (3), Anat Reiner (4), Daniel Yekutieli (4), Yoav Benjamini (4)<br>
    (1) Department of Zoology, Tel-Aviv University; (2) Department of Psychiatry, Maryland Psychiatric Research Center, University of Maryland Baltimore; (3) Department of Functional Genomics, The Institute for Genomic Research, Rockville, Maryland; (4) Department of Statistics and Operation Research, Tel Aviv University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Exploration studies use the path traced by a mouse to reveal genetic effects. Two influences the wall has on exploration even when the mouse is distant from it are: Guidance--the tendency to progress in parallel to the wall and Attraction--the tendency to move along the wall and to rapidly return to it. Seven heritable measures (end-points) of guidance and attraction were correlated to few hundred gene expression patterns measured in 5 brain regions, using an FDR controlled procedure. These results suggest that guidance and attraction reflect processes taking place in different brain areas.
<br><br>Contact: horevguy [at] post.tau.ac.il
<br><br>Keywords: Phenotype, Microarray, Multiplicity-Problem
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
