<html>
<head>
<title>Poster Abstracts M</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category M: Structural bioinformatics</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="M03" name="M03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M03</b><br>
    <b>A Computational Study of Substrate Specificity in the Family of Short-Chain Dehydrogenases</b><br>
    Angelo Favia (1), Janet Thornton (1), Rafael Najmanovich (1), Fabian Glaser (1), Irene Nobeli (2)<br>
    (1) European Bioinformatics Institute; (2) King's College London
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    This project examines the extent and physicochemical origin of substrate-binding specificity in short-chain dehydrogenases, a large family whose members are known to catalyse reactions involving a wide variety of substrates. We employ bioinformatics and chemoinformatics methods including molecular docking to assess the binding preferences of these proteins. Ultimately, we aim to assist structural genomics projects with the functional characterisation of newly-crystallised proteins, by suggesting a few small molecule scaffolds that are likely to be good binders for a given binding site.
<br><br>Contact: irilenia.nobeli [at] kcl.ac.uk
<br><br>Keywords: Substrate Specificity, Docking
</font><hr></td></tr>
<tr>
  <td><a id="M05" name="M05"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M05</b><br>
    <b>Scan2S: Regular Expression Scan Algorithm for Detection of Type II Restriction Endonucleases</b><br>
    Masha Y. Niv (1), Lucy Skrabanek (1), Richard J. Roberts (2), Harold Scheraga (3), Harel Weinstein (1)<br>
    (1) Weill Medical College of Cornell University; (2) Cornell University; (3) New England Biolabs
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Structure-based sequence alignment of highly dissimilar sequences of Type II Restriction Endonucleases is analyzed to derive a conserved motif. The motif is constructed based on positions of conserved physicochemical properties and optional secondary structure constraints and is used in a regular expression matching algorithm. A successful application to REase detection in a bacterial genome is presented. Additionally, motifs based on REases with specificity towards particular DNA sequences successfully retrieve REases of the same specificity. The Scan2S approach is readily applicable to other protein families in the "twilight zone" of sequence similarity.
<br>
<br><br>Contact: man2016 [at] gmail.com
<br><br>Keywords: Secondary Structure Physochemical Motif Scan
</font><hr></td></tr>
<tr>
  <td><a id="M06" name="M06"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M06</b><br>
    <b>Evolutionary Cluster Conservation of Proteins</b><br>
    Ofer Rahat, Gideon Schreiber<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We suggest that proteins binding sites have internal order, and are composed of small modules. We extract these modules, and show that they are conserved among homologs. This concept can be applied to evaluation of mutagenesis studies, used widely for force-field calibration.
<br><br>Contact: ofer.rahat [at] weizmann.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="M08" name="M08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M08</b><br>
    <b>Scoring Protein-Protein Complexes using MD-Generated Ensembles</b><br>
    Marcin Krol (1,2), Paul A. Bates (1)<br>
    (1) Biomolecular Modelling Laboratory, Cancer Research UK London Research Institute; (2) Departament of Bioinformatics and Telemedicine, Collegium Medicum, Jagiellonian Univ.
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    MD simulations are used to generate ensembles of structures around near-native and false-positive docked solutions generated by the rigid-body docking program, FTDOCK. We assess the scoring potential of different energy terms and entropy estimates for 25 non-obligate protein-protein complexes. The highest percentage of true positives are obtained from the sum of the van der Waals and scaled electrostatic interaction energy terms. Moreover, ensemble-averaged energy terms improve ranking compared to energy terms calculated on a single, minimized structure.
<br><br>Contact: mykrol [at] cyf-kr.edu.pl
<br><br>Keywords: Protein-Protein Complex, Scoring Function
</font><hr></td></tr>
<tr>
  <td><a id="M09" name="M09"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M09</b><br>
    <b>The ProCKSi-Server: A Server for Similarity Comparisons of Protein Structures</b><br>
    Daniel Barthel (1), Jonathan D. Hirst (2), Natalio Krasnogor (1)<br>
    (1) ASAP, School of Computer Science and IT, University of Nottingham, UK; (2) School of Chemistry, University of Nottingham, UK
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    ProCKSi is a web server for "Protein Comparison using Kolmogorov Similarity" that additionally serves as a "proxy" to established protein comparison and alignment methods. It implements a protocol for protein structure comparison using the Universal Similarity Metric (USM), and harvests and incorporates results from other sources in order to collate these in one place. Analysis tools have been developed for the visualisation, comparison and integration of multiple similarity measures, so as to give a consensus picture of the protein datasets submitted.
<br>Web-Address: http://www.procksi.net
<br><br>Contact: daniel.barthel [at] cs.nott.ac.uk
<br><br>Keywords: Protein, Similarity, Comparison, Web Server
</font><hr></td></tr>
<tr>
  <td><a id="M10" name="M10"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M10</b><br>
    <b>Computational Investigations of the Reversible Interactions of G-proteins with Membranes Surfaces</b><br>
    Mickey Kosloff (1), Barry Honig (1,2)<br>
    (1) Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY, USA; (2) Howard Hughes Medical Institute
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Heterotrimeric G-proteins are key molecular transducers in signal transduction cascades and, like many soluble signaling proteins, they are peripherally attached to the membrane. Here we employ a quantitative approach to characterize their interaction with the membrane surface, combining coarse global sampling using the finite-difference Poisson-Boltzmann method with the effect of covalently attached lipid anchors. Our results provide novel mechanistic insights into the dynamic interactions of G-proteins with the membrane, protein partners and into their intracellular translocation.
<br><br>Contact: mk2417 [at] columbia.edu
<br><br>Keywords: Electrostatics, 3D, Membrane, Localization
</font><hr></td></tr>
<tr>
  <td><a id="M11" name="M11"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M11</b><br>
    <b>Sequence-Similar, Structure-Dissimilar Protein Pairs in the PDB</b><br>
    Mickey Kosloff (1), Rachel Kolodny (1,2), Barry Honig (1,2)<br>
    (1) Department of Biochemistry and Molecular Biophysics, Columbia University, New York, NY, USA; (2) Howard Hughes Medical Institute
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    A common assumption is that almost all proteins in the PDB with high sequence identity will have similar structures. Here, we report many pairs of proteins in the PDB with high sequence identity and significant structural dissimilarity. To detect structural dissimilarity we use sequence-based structure superpositioning rather than structural alignment, as the latter underestimates dissimilarity. Our results suggest that using structural information in addition to sequence can result in more representative non-redundant PDB sets and can enhance template selection in comparative modeling.
<br><br>Contact: mk2417 [at] columbia.edu
<br><br>Keywords: PDB, Structure, Sequence, Align, Dissimilar
</font><hr></td></tr>
<tr>
  <td><a id="M12" name="M12"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M12</b><br>
    <b>Hierarchical Structure Induction by Compression in Graphs</b><br>
    Leon Peshkin (1), Virginia Savova (2)<br>
    (1) Harvard Medical School; (2) Johns Hopkins University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    This work is motivated by the necessity to automate the discovery of structure in vast and ever-growing collection of relational data in general and genomic networks in particular. A novel algorithm for structure induction by graph compression is presented and illustrated by a clear and broadly known case of nested structure in a DNA molecule.
<br><br>Contact: savova [at] jhu.edu
<br><br>Keywords: Structure Discovery, Network
</font><hr></td></tr>
<tr>
  <td><a id="M13" name="M13"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M13</b><br>
    <b>Protein-Protein Interactions: De-Novo Rational Protein-Interface Design</b><br>
    Mati Cohen, Dana Reichmann, Gideon Schreiber<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The assembly of proteins into protein complexes is part of most biological processes. In the last decade rational protein design evolved from the design of the hydrophobic core, considering basic interactions, to the design of a completely new protein. In the field of protein interactions the success of rational design is more modest. We are developing a knowledge based potential function, based on non-redundant protein interface database, to describe the various interface interaction types.
<br><br>Contact: mati.cohen [at] weizmann.ac.il
<br><br>Keywords: Rational Design, Protein-Inteface Database
</font><hr></td></tr>
<tr>
  <td><a id="M15" name="M15"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M15</b><br>
    <b>Systematic Discovery of Functional RNA Structural Motifs</b><br>
    Michal Rabani, Michael Kertesz<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    There is growing evidence for the important role of RNA secondary structure in gene regulation, cell localization, RNA editing and other cellular processes. Here, we develop a computational framework for representing the building blocks of RNA structural motifs and apply it to comprehensively identify all structural elements encoded in RNAs.
<br><br>Contact: michal.rabani [at] weizmann.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="M16" name="M16"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M16</b><br>
    <b>Refining Intra-Protein Contact Predictions using Graph Analysis</b><br>
    Milana Frenkel-Morgenstern, Shmuel Pietrokovski<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Accurate prediction of residue contacts within proteins from sequence data will allow prediction of protein structures. Currently correlated mutation analyses are usually followed by neural network refinement. Here, we present a graph analysis refinement of contact predictions methods (GARP). It uses weighted graphs, where edges correspond to contact predictions and nodes to contacting positions, and analyzes graph topology and relative primary positions of predicted contacts. GARP significantly improves various contact prediction methods, reaching 27% of the accuracy level.
<br><br>Contact: milana.frenkel [at] weizmann.ac.il
<br><br>Keywords: Protein Contact Prediction, 3D Structure
</font><hr></td></tr>
<tr>
  <td><a id="M17" name="M17"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M17</b><br>
    <b>"Learning to Learn" HLA-Peptide Binding Specificity using Supertypes</b><br>
    Chen Yanover (1), Tomer Hertz (1,2),  Ora Schueler-Furman (3)<br>
    (1) School of Computer Science and Engineering, The Hebrew University of Jerusalem, Israel; (2) Interdisciplinary Center for Neural Computation, The Hebrew University of Jerusalem, Israel; (3) Department of Molecular Genetics and Biotechnology, Hadassah Medical School, The Hebrew University of Jerusalem, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    MHC-peptide interactions are at the heart of the cellular immune response, as they are responsible for the presentation of pathogen-derived peptides on the surface of infected cells. MHC alleles can be grouped into sets of alleles that bind to similar peptides called supertypes. We show here that information extracted at the supertype level can surprisingly well describe previously uncharacterized alleles of a defined supertype. Based on this insight, we train several prediction methods on supertype data and demonstrate their effectiveness in binding predictions on novel alleles.
<br><br>Contact: cheny [at] cs.huji.ac.il
<br><br>Keywords: HLA, Immunoinformatics, "Learning to Learn"
</font><hr></td></tr>
<tr>
  <td><a id="M18" name="M18"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M18</b><br>
    <b>ProMateus: An Open Research Approach to Protein Binding Sites Analysis</b><br>
    Hani Neuvirth (1,2), David Birnbaum (1), Uri Heinemann (1), Naftali Tishby (1), Gideon Schreiber (2)<br>
    (1) The Hebrew University; (2) Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Information about the binding site location on a protein is extremely valuable when examining chemical mechanisms. In this work, the thriving open source approach is adapted to its research parallel. We extend ProMate, a protein binding site prediction server, into a generic protein binding sites research tool. ProMateus is a web tool, based on ProMate's infrastructure and insights, which enables the easy exploration and incorporation of new features potentially capable of identifying protein binding sites. The analysis is demonstrated on ProMate and on a database of protein-DNA interactions.
<br><br>Contact: neuvirth [at] cs.huji.ac.il
<br><br>Keywords: Protein, Structure, Interface, DNA, Open Research
</font><hr></td></tr>
<tr>
  <td><a id="M19" name="M19"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M19</b><br>
    <b>A WebServer Integrative Approach for the Combination and Validation of Inter-residue Contacts</b><br>
    Javier Garcia-Garcia, Baldomero Oliva<br>
    Universitat Pompeu Fabra
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    An increasing number of approaches for the prediction and validation of residue-residue contacts have been developed in the last decade: for example, correlated mutation or conservation analysis methods. Every time a new method is described, a different benchmarck is used to validate it. This difficults the comparison between methods. Here, we propose the creation of a WebServer for the validation of new prediction methods. In this way, it will be very easy to compare new methods with the old ones. A structural filter is proposed in order to improve the results.
<br><br>Contact: boliva [at] imim.es
<br><br>Keywords: Inter-residue Contact Predictions, Validation
</font><hr></td></tr>
<tr>
  <td><a id="M20" name="M20"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M20</b><br>
    <b>Analysis of Sequence and Structure Features of H/ACA Like Box SnoRNA in Trypanosomatids</b><br>
    Inna Myslyuk, Avihay Apatoff, Avraham Hury, Ron Unger, Shulamit Michaeli<br>
    The Mina and Everard Goodman Faculty of Life Sciences, Bar-Ilan University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The family of H/ACA small nucleolar RNA guides pseudouridylation on rRNA and snRNAs. Until now, only third of the expected number of H/ACA molecules were found in trypanosomes. Finding additional H/ACA's in these organisms is a real challenge because of their short rRNA recognition motif. We performed sequence and secondary structure analysis of homologous H/ACA molecules in trypanosomes using bioinformatic tools in order to determine the conserved features of these molecules. Using these features, a genomic search algorithm is being developed and 6 novel H/ACA's were identified in L.major.
<br><br>Contact: myslyuk [at] hotmail.com
<br><br>Keywords: H/ACA, snoRNA, Noncoding RNA, Tripanosomatid
</font><hr></td></tr>
<tr>
  <td><a id="M21" name="M21"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M21</b><br>
    <b>Scaling in Biology: Exploring Surface-to-Volume Ratios and other Properties within Thousands of X-ray Structures</b><br>
    Hector Romero (1), Martin Grana (2), Pablo Dans (3), Hugo Naya (1,4)<br>
    (1) Laboratorio de Organizacion y Evolucion del Genoma, Seccion Biomatematicas, Facultad de Ciencias; (2) Unite de Biochimie Structurale, Institut Pasteur Paris; (3) Laboratorio de Quimica Teorica y Computacional, Inst. Quimica Biologica, Facultad de Ciencias ; (4) Unidad de Bioinformatica, Institut Pasteur de Montevideo
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Allometric relationships pervade Biology. Classical examples include metabolic rates, time-scales and sizes, scaling with typical simple powers of 1/4. We analyzed the surface to volume relationship on 7698 X-ray protein structures extracted from the PDB database and studied the allometric properties of these variables. We confirmed that proteins are not isometric, because their allometric coefficient is around 0.8 and not around 2/3. We discuss the possible implications of these findings and suggest a mechanism that could explain our results.
<br><br>Contact: eletor [at] fcien.edu.uy
<br><br>Keywords: Allometric Scaling, PDB, Protein Structure
</font><hr></td></tr>
<tr>
  <td><a id="M22" name="M22"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M22</b><br>
    <b>Funnel Hunting in a Rough Terrain: Learning and Discriminating Energy Funnels</b><br>
    Nir London (1), Ora Furman Schueler (2)<br>
    (1) Computational biology, The Hebrew University, Jerusalem Israel; (2) Dept. of Molecular Genetics and Biotechnology, Hebrew University, Hadassah Medical School
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Modeling of protein complex structures with RosettaDock often results in a deep energy funnel near the native conformation, and the global minimum energy model at the funnel tip usually represents a prediction of near-crystallographic quality. More intensive sampling however indicates that funnels can appear in other regions.  In this study we characterize funnels and use learning algorithms to improve the distinction of the correct from incorrect funnels. In addition, models at different locations in the funnel can shed light on its relevance to the process of protein-protein association.
<br><br>Contact: fora [at] md.huji.ac.il
<br><br>Keywords: Energy Landscape, Docking, Machine Learning
</font><hr></td></tr>
<tr>
  <td><a id="M23" name="M23"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M23</b><br>
    <b>Effects of Protein Sequence Conservation on Protein Secondary Structure Patterns</b><br>
    Einat Sitbon, Shmuel Pietrokovski<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We analyzed patterns of consecutive secondary structure elements (SSEs) in conserved sequence regions. The SSE patterns in these regions differ significantly from their occurrence in other protein regions. Their occurrence also significantly differed in each protein structural class. In addition, pairs of inverted patterns occurred differently. We discuss the significance of this large-scale analysis for understanding the evolution and functionality of protein structure. SSE patterns can be used for predicting structural features and functional sites in proteins.
<br><br>Contact: einat.sitbon [at] weizmann.ac.il
<br><br>Keywords: Conservation, Motifs, Secondary Structure
</font><hr></td></tr>
<tr>
  <td><a id="M25" name="M25"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M25</b><br>
    <b>High Affinity and Binding Specificity of Computationally Redesigned Protein Interfaces</b><br>
    Vladimir Potapov (1), Dana Reichmann (2), Vladimir Sobolev (1), Marvin Edelman (1), Gideon Schreiber (2)<br>
    (1) Department of Plant Sciences, Weizmann Institute of Science; (2) Department of Biological Chemistry, Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We demonstrate an approach for computational redesign of protein-protein interfaces combining a scoring function and natural template fragments from resolved structures to produce novel putative complexes. Experimentally validating with the TEM1-BLIP interface, the top-scoring design yielded a complex showing affinity exceeding that of wild type and specific binding.
<br><br>Contact: vladimir.potapov [at] weizmann.ac.il
<br><br>Keywords: Protein-Protein Interface Design
</font><hr></td></tr>
<tr>
  <td><a id="M26" name="M26"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M26</b><br>
    <b>Predictions of Single Point Mutations that Substantially Alter the RNA Secondary Structure</b><br>
    Alexander Churkin (1), Ofer Peleg (2), Danny Barash (1,2)<br>
    (1) Department of Computer Science, Ben-Gurion University; (2) Genome Diversity Center, Institute of Evolution, University of Haifa
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Previously, it has already been calculated that there is some probability that even a single point mutation can substantially alter the RNA secondary structure. Such point mutations are interesting to explore because of the likelihood that by a conformational rearrangement of the secondary structure, they may cause a change in function. We have developed a Java application program called RNAMute (available at http://www.cs.bgu.ac.il/~RNAMute to predict rearranging point mutations. RNAMute can also be used for the analysis of ovelapping messages in genomes.
<br><br>Contact: dbarash [at] cs.bgu.ac.il
<br><br>Keywords: RNA Folding Predictions, RNA Mutations
</font><hr></td></tr>
<tr>
  <td><a id="M27" name="M27"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M27</b><br>
    <b>A Machine Learning Approach for Predicting RNA-Binding Proteins from Three Dimensional Structure</b><br>
    Shula Shazman, Yael Mandel-Gutfreund<br>
    Technion - Israel Institute of Technology
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We apply a machine learning approach for predicting RNA-binding proteins from 3D structures. The method is based on characterizing unique properties of electrostatic patches on the protein surface. It does not rely on sequence or structural homology and could be applied to novel proteins with unique folds and/or binding motifs. 
<br><br>Contact: shulas [at] techunix.technion.ac.il
<br><br>Keywords: RNA-Binding, SVM, Function Prediction
</font><hr></td></tr>
<tr>
  <td><a id="M28" name="M28"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M28</b><br>
    <b>In Search of Short RNA Thermoswitches Using Computational and Experimental Analyses</b><br>
    Idan Gabdank (1), Assaf Avihoo (1), Danny Barash (1), Michal Shapira (2)<br>
    (1) Ben Gurion University, Computer Science Department; (2) Ben Gurion University, Life Science Department
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Temperature is one of the physical parameters under constant vigilance in living cells. Avihoo et al. computationally designed temperature driven short RNA thermoswitches restricted to physical temperatures[1]. We have successfully synthesized and tested those thermosensor candidates in vitro. Experimental results encouraged us to hypothesize that this kind of control may also participate in initiating the heat-shock response. Potential thermosensing elements were identified in the 5' UTRs of several heat-shock mRNAs. We are currently pursuing this hypothesis by an experimental approach. 
<br><br>Contact: gabdank [at] cs.bgu.ac.il
<br><br>Keywords: RNA Structure, Thermo-Switches, Heat-Shock
</font><hr></td></tr>
<tr>
  <td><a id="M29" name="M29"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M29</b><br>
    <b>Detection of Out-Of-Register Errors in Protein Crystal Structures</b><br>
    Marian Novotny (1,2), Henrik Hansson (1), Helena Strombergsson (2), Gerard J. Kleywegt (1)<br>
    (1) Department of Cell and Molecular Biology, Uppsala University; (2) Linnaeus Centre for Bioinformatics, Uppsala University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    An out-of-register error is characterized by the fact that one or typically more amino acids that were built in a wrong place often as a result of poor electron density in that region. We want to find out how widely these errors occur in the PDB and to develop a diagnostic tool that could show regions likely to contain register shifts. We are currently gathering various structure-quality related statistics for known register shifts and we will employ the "rough sets" method to find a set of statistics that could diagnose probable register errors in the proteins.
<br>
<br><br>Contact: marian [at] xray.bmc.uu.se
<br><br>Keywords: Out-of-Register Error, Structure, Rough Sets
</font><hr></td></tr>
<tr>
  <td><a id="M30" name="M30"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M30</b><br>
    <b>Are There More Structural Interaction Types than Previously Thought?</b><br>
    Benjamin Schuster-Boeckler, Robert D. Finn, Alex Bateman<br>
    Wellcome Trust Sanger Institute
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Protein interactions are thought to be mediated by a limited set of reusable structural interaction modules or domains. To try and estimate the number of unique domain-domain interaction types in nature, we used a merged set of experimentally verified protein interactions together with iPfam, the database of structurally interacting Pfam domains. Our results disagree with previously published figures. We show that only a small number of interaction types has been structurally characterised.
<br><br>Contact: bsb [at] sanger.ac.uk
<br><br>Keywords: Protein Interaction Networks, Protein Domains
</font><hr></td></tr>
<tr>
  <td><a id="M32" name="M32"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M32</b><br>
    <b>Efficient Prediction of Metal Binding Sites in Apo-Protein Structures</b><br>
    Mariana Babor, Sergey Gerzon, Barak Raveh, Vladimir Sobolev, Marvin Edelman<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    An algorithm and interactive web site for predicting transition metal binding sites in apo (and holo) protein structures is presented (http://ligin.weizmann.ac.il/ched). Two types of predictions are made: maximum sensitivity (maximum true positives) or maximum selectivity (minimum false positives). Test results achieved on non-redundant data sets indicate levels of ~80% sensitivity and ~95% selectivity for our algorithm.
<br><br>Contact: vladimir.sobolev [at] weizmann.ac.il
<br><br>Keywords: Bioinformatics, Structural Genomics
</font><hr></td></tr>
<tr>
  <td><a id="M33" name="M33"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M33</b><br>
    <b>Fast Protein Structure Alignment by Joining Similar Substructure Pair</b><br>
    Chan-Yong Park (1), Sung-Hee Park (1), Dae-Hee Kim (1), Soo-Jun Park (1), Man-Kyu Sung (1), Hongro-Lee (2), Chi-Jung Hwang (2)<br>
    (1) ETRI (Electronics and Telecommunications Research Institute); (2) Chung Nam University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    This paper proposes a novel fast protein structure alignment algorithm and its application. In this paper, we propose a 3D chain code representation for fast measuring the local geometric similarity of protein and introduce a backtracking algorithm for joining a similar local substructure efficiently. We have designed and implemented a protein structure alignment system based on our protein visualization software. These experiments show rapid alignment with precise results.
<br><br>Contact: cypark [at] etri.re.kr
<br><br>Keywords: Protein Structure Alignment, 3D Chain Code
</font><hr></td></tr>
<tr>
  <td><a id="M34" name="M34"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M34</b><br>
    <b>Inherent Limitations in Protein-Protein Docking Procedures</b><br>
    Noga Kowalsman, Miriam Eisenstein<br>
    The Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Analysis of a large dataset of protein-protein docking results showed that the distinction between nearly correct models and decoys depends on the size of the interface to be predicted thus setting a limit to the prediction ability of docking procedures, particularly those in which the geometric complementarity descriptor is dominant. Grid-based docking procedures carry a large statistical error which further reduces the distinction between nearly correct models and decoys. The distinction is improved when the docking models are ranked by statistically equivalent scores.
<br><br>Contact: miriam.eisenstein [at] weizmann.ac.il
<br><br>Keywords: Statistically Equivalent Scores, Docking
</font><hr></td></tr>
<tr>
  <td><a id="M35" name="M35"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M35</b><br>
    <b>Prediction of Symmetric Assemblies by Docking</b><br>
    Dina Schneidman (1), Yuval Inbar (1), Ruth Nussinov (2,3), Haim Wolfson (1)<br>
    (1) School of Computer Science, Tel Aviv University; (2) Sackler Institute of Molecular Medicine, Sackler Faculty of Medicine, Tel Aviv University; (3) Center for Cancer Research, Nanobiology Program, National Cancer Institute, Frederick, MD, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The majority of proteins within the cell function as symmetrical oligomeric complexes. Symmetry gives several advantages to protein function, such as formation of large protein structures, stability against denaturation and folding efficiency. There are many types of symmetry in nature. We present efficient docking methods for predicting symmetric assemblies starting from monomeric unit for different types of symmetry: cyclic, dihedral, cubic and helical. All the algorithms a priori restrict their transformational search space only to symmetric transformations, and thus gain both in efficiency and performance.
<br><br>Contact: duhovka [at] tau.ac.il
<br><br>Keywords: Docking, Assembly, Symmetry, Structure
</font><hr></td></tr>
<tr>
  <td><a id="M36" name="M36"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M36</b><br>
    <b>Pharmacophore Detection: Multiple Flexible Alignment of Drug-Like Molecules</b><br>
    Oranit Dror (1),  Yuval Inbar (1),  Dina Schneidman (1), Ruth Nussinov (2,3), Haim Wolfson (1)<br>
    (1) School of Computer Science, Tel Aviv University; (2) Sackler Institute of Molecular Medicine, Sackler Faculty of Medicine, Tel Aviv University; (3) Center for Cancer Research, Nanobiology Program, National Cancer Institute, Frederick, MD, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We present a novel highly efficient method for detecting a pharmacophore, which is the 3D arrangement of essential features that enable ligand molecules to bind to the same protein. The method superimposes multiple input ligands with a preference on aligning chemically similar features that are probably the most important for ligand-receptor interactions. The ligand flexibility is considered explicitly by combining the conformational search within the pattern identification process. The method was tested successfully on the FlexS dataset. 
<br><br>Contact: oranit [at] tau.ac.il
<br><br>Keywords: Drug Design, Pharmacophore, Structure, CADD
</font><hr></td></tr>
<tr>
  <td><a id="M38" name="M38"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M38</b><br>
    <b>FILTREST3D: A Simple Method for Discrimination of Multiple Structural Models against Spatial Restraints</b><br>
    Michal J. Gajda (1), Marta Kaczor (1), Alan M. Friedman (2), Chris Bailey-Kellogg (3), Janusz M. Bujnicki (1)<br>
    (1) International Institute of Molecular and Cell Biology, Warsaw, Poland; (2) Department of Biological Sciences, Purdue University, West Lafayette, USA; (3) Department of Computer Science, Dartmouth College, Hanover, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Automatic methods for protein structure prediction produce large sets of alternative models, which often include native-like structures, but with scores that are indistinguishable from false positives. Native-like models can be often easily identified based on data from relatively simple experimental analyses that can be encoded as spatial restraints (cross-linking, chemical modification, mutagenesis etc). FILTREST3D (http://filtrest3d.genesilico.pl/) is a new method for discrimination of a large number of alternative models of protein structure against a set of restraints. 
<br><br>Contact: iamb [at] genesilico.pl
<br><br>Keywords: Model Discrimination, Spatial Restraints
</font><hr></td></tr>
<tr>
  <td><a id="M39" name="M39"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M39</b><br>
    <b>3D Segmentation of Low and Intermediate Resolution EM Density Maps</b><br>
    Yuval Inbar (1), Ruth Nussinov (2), Haim J. Wolfson (1)<br>
    (1) School of Computer Science, Tel Aviv University; (2) Sackler Institute of Molecular Medicine, Sackler Faculty of Medicine, Tel Aviv University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Electron Microscopy is becoming a powerful tool for structural studies of multimolecular complexes. Due to physical and technological barriers most solved structures are obtained at medium and low resolution. We present here a novel and efficient algorithm for 3D segmentation of low and intermediate resolution density maps. The algorithm was tested on different maps with structures varying in their shape, architecture, resolution, and voxel size. In most cases, the algorithm distinguished between different chains. In some cases, even different domains of the same chain were detected.
<br><br>Contact: inbaryuv [at] tau.ac.il
<br><br>Keywords: EM, Protein Complexes, Segmentation
</font><hr></td></tr>
<tr>
  <td><a id="M40" name="M40"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M40</b><br>
    <b>Optimizing Scoring Matrices for Alignment Ranking</b><br>
    Jan E. Gewehr, Fabian Birzele, Ralf Zimmer<br>
    Institut fur Informatik, LMU Munchen
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Sequence-structure alignments are an important intermediate step for protein structure prediction based on sequence homology. We present a genetic algorithm for optimizing scoring matrices for the approximation of structure-based benchmark scores like the TM-Score. Comparison is done against well-known scoring matrices on a large set of alignments between structural domains.
<br><br>Contact: jan.gewehr [at] ifi.lmu.de
<br><br>Keywords: alignment scoring, genetic algorithm
</font><hr></td></tr>
<tr>
  <td><a id="M43" name="M43"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M43</b><br>
    <b>Contact Prediction using Secondary Structure Pairings</b><br>
    Michal J. Gajda (1,2), Janusz M. Bujnicki (1), Chris Bailey-Kellogg (3)<br>
    (1) International Institute for Molecular and Cell Biology, Warsaw, Poland; (2) Warsaw Technical University, Poland; (3) Dartmouth College, Hanover, US
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We generalize notion of beta-strand pairings and use it to discover which contacts are likely to be robust with respect to alignment and modelling errors. The most reliable of contacts found in metaserver models are then combined into consensus prediction.
<br><br>Contact: mgajda [at] genesilico.pl
<br><br>Keywords: Contact Prediction, Protein Structure
</font><hr></td></tr>
<tr>
  <td><a id="M44" name="M44"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M44</b><br>
    <b>Accurate Prediction of Enzyme Mutant Activity based on a Multibody Statistical Potential</b><br>
    Majid Masso, Iosif I. Vaisman<br>
    George Mason University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We describe a machine learning approach for inferring the activity levels of all unexplored single point mutants of an enzyme, based on a training set of such mutants with experimentally measured activity.  Based on a Delaunay tessellation-derived four-body statistical potential function, a perturbation vector measuring environmental changes relative to wild type at every residue position uniquely characterizes each enzyme mutant for model development and prediction. Models developed using training sets of these perturbation vectors can predict mutant activity with 80% accuracy.
<br><br>Contact: ivaisman [at] gmu.edu
<br><br>Keywords: Computational Mutagenesis
</font><hr></td></tr>
<tr>
  <td><a id="M45" name="M45"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M45</b><br>
    <b>A Novel Approach to Protein-Water Interaction Characteristics Using Computational Geometry </b><br>
    Gregory M. Reck, Iosif I. Vaisman<br>
    George Mason University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    A new topological strategy is presented for characterizing the relationship between a globular protein and the surrounding water environment, using the results of Delaunay tessellation of computationally hydrated proteins. Topological parameters are especially informative in developing knowledge-based methods for analyzing and predicting protein characteristics, such as stability and functionality.  The new parameter is based on the extent of inclusion of simulated water molecules into the Delaunay tetrahedra surrounding each residue. 
<br><br>Contact: ivaisman [at] gmu.edu
<br><br>Keywords: Protein Hydration, Delaunay Tessellation
</font><hr></td></tr>
<tr>
  <td><a id="M46" name="M46"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M46</b><br>
    <b>Protein Structural Domain Assignment Using a Delaunay-Tessellation Derived Lattice</b><br>
    Todd Taylor, Iosif I. Vaisman<br>
    George Mason University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We describe a new method of protein structural domain assignment based on Delaunay and Potts algorithms (DePot). Each residue is represented as a site in an irregular three-dimensional lattice derived as a result of the Delaunay tessellation of the protein structure.  Domain membership is represented by a spin value and each site has a spin which can change under the influence of its neighbors. Spins are allowed to interact subject to an Ising ferromagnetic-like energy function until clusters of like spins emerge and these clusters define domains. 
<br><br>Contact: ivaisman [at] gmu.edu
<br><br>Keywords: Protein Topology, Domain Assignment
</font><hr></td></tr>
<tr>
  <td><a id="M47" name="M47"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M47</b><br>
    <b>A Simplified Representation of Electrostatic Model Surfaces for Addressing Protein-Protein Interaction Problems</b><br>
    Dietlind Gerloff (1), Shakir Ali (2), Xueping Quan (2), Rupert Koenig (3)<br>
    (1) University of California, Santa Cruz; (2) University of Edinburgh, School of Informatics; (3) University of Applied Sciences, Bingen
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We have developed a novel way of simplifying the comparison of the electrostatic molecular surfaces of proteins to the comparison of 1-D "electrostatic surface profiles" based on the same information. While the simplified representation will necessarily mean a "neglect" of more fine-grained information, the profile format offers many advantages over the classic 3-D format of electrostatic potential surfaces. Here we show examples of how comparisons of the model surfaces of homologous proteins may be useful in protein-protein interaction questions, such as binding site and partner prediction.
<br><br>Contact: rupert.koenig [at] gmx.net
<br><br>Keywords: Protein Interactions, Comparative Modelling
</font><hr></td></tr>
<tr>
  <td><a id="M48" name="M48"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster M48</b><br>
    <b>A Cooperative Energy Term for Hydrogen Bonds</b><br>
    Ami Levy-Moonshine, Tatiana Maximova, Chen Keasar<br>
    Department of Computer Science, Ben Gurion University of the Negev
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Secondary structure (SS) elements constitute a major part of protein structures. Thus their assembly is an essential step in most of the protein structure prediction processes. In this work, we present a cooperative energy term for hydrogen bonds which takes advantage of the characteristic hydrogen bond patterns of SS. Optimizing under this energy term, results in models that contain native-like SS. The cooperative energy term for hydrogen bonds is implemented within the freely available, molecular modeling package [Bioinformatics 21:3931-3932].
<br><br>Contact: amilev [at] cs.bgu.ac.il
<br><br>Keywords: Protein Structure Prediction, Hydrogen Bonds
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
