<html>
<head>
<title>Poster Abstracts N</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category N: Text mining</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="N01" name="N01"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster N01</b><br>
    <b>Association Rule Mining and Protein Databases: Exceptions Point to Annotation Errors</b><br>
    Irena I. Artamonova (1), Goar Frishman (1), Mikhail S. Gelfand (3), Dmitrij Frishman (1,2)<br>
    (1) Institute for Bioinformatics, GSF-National Research Center for Environment and Health; (2) Department of Genome Oriented Bioinformatics, Technische Universitat Munchen; (3) Institute of Information Transmission Problems RAS
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The Association Rule Mining technique was applied to identification of possible errors in protein annotations in the Swiss-Prot and PEDANT databases. The errors were identified as feature combinations constituting exceptions from strong rules found by the algorithm. Suspicious features were marked for manual curation, omitted features were added to the annotation. Tests demonstrated that most exceptions indeed were caused by annotation errors.
<br><br>Contact: irena.artamonova [at] gsf.de
<br><br>Keywords: Protein Annotation, Association Rule Mining
</font><hr></td></tr>
<tr>
  <td><a id="N02" name="N02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster N02</b><br>
    <b>Finding Optimal Microarray Platform to Analyze High-Risk Pathologies</b><br>
    David Alarcon Gallego (1), Baldomero Oliva (2)<br>
    (1) Insitut Municipal d'Investigacions Mediques (IMIM); (2)Universitat Pompeu-Fabra (UPF)
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The risk of a population to suffer pathologies as schizophrenia and hypertension is analysed by means of data mining and network analysis of the phenotype in microarrays. This work is aimed to determine the minimal and most significant microarray to detect these pathologies. An initial set of the most important genes involved in these pathologies was obtained by data mining. PIANA included genes interacting with others in the initial set. The resulting set is used to determine the optimal platforms: GPL136 for schizophrenia and GPL549 for hypertension from the GEO database.
<br><br>Contact: boliva [at] imim.es
<br><br>Keywords: Microarray, Schizophrenia, Hypertension, GEO
</font><hr></td></tr>
<tr>
  <td><a id="N04" name="N04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster N04</b><br>
    <b>Cross-Discipline Knowledge Integration</b><br>
    Evangelos Pafilis<br>
    European Molecular Biology Laboratory, EMBL
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The increasing number of publications and the limited scope of literature which  scientists follow, result in relevant and potentially important information being lost. By augmenting text mining with biological database information and providing it  with web-based corpora, originating from biology, chemistry and medicine, we aim at delivering knowledge associations that would not be visible otherwise. To deal with the variety and heterogeneity of data types and formats we are exposing the software components as services and managing them via workflow engines.
<br><br>Contact: pafilis [at] embl.de
<br><br>Keywords: Text Mining, Web Services, Data Integration
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
