<html>
<head>
<title>Poster Abstracts H</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category H: Pathways, networks & systems</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="H01" name="H01"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H01</b><br>
    <b>System Level Analysis of the Cell Death Network via RNAi Knock-Down Experiments</b><br>
    Einat Zalckvar, Adi Kimchi<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The molecular network underlying programmed cell death (PCD) comprises approximately 100-150 proteins (the network's nodes), organized along several biochemical/genetic pathways. Several phenotypic outcomes of PCD have been characterized, including type I apoptosis, type II autophagic cell death, and programmed necrosis.  Recently, a switch between the various phenotypes has been documented in response to a given stress signal, which depends on the functional status of the different proteins in the network. In order to provide a dynamic insight into the molecular network underlying PCD, we developed a new method that measures the functional weight of the network's individual nodes. This is achieved by applying single and double silencing perturbations, in different combinations, using RNA interference (RNAi), in human HEK293 cells, and then, measuring the impact of this silencing on responses to different stress signals. The method was scaled up to enable high throughput data collection with maximal accuracy and sensitivity.  The numeric data that assess the system performance is combined with a molecular analysis in order to measure the extent to which the various perturbations affect inter and intra-modular connections between different nodes in the network. Several novel concepts concerning the PCD network have already emerged from experiments using a limited combination of perturbations. This type of analysis has the potential to provide novel principles in protein networks' function in mammalian cells.
<br><br>Contact: einatz [at] weizmann.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="H02" name="H02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H02</b><br>
    <b>Genome-Wide Adaptive Reprogramming of Gene Regulation in Yeast</b><br>
    Shay Stern (1), Tali Dror (1), Elad Stolovicki (2), Naama Brenner (1), Erez Braun (2)<br>
    (1) Department of Chemical Engineering, Technion; (2) Department of Physics, Technion
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In this study we measure the genome-wide expression profiles in a yeast population during adaptation to a synthetic gene recruitment event, where HIS3 was linked to the GAL system. We show that adaptation is characterized by large-scale changes in gene expression, involving hundreds of genes. We identify significant gene-clusters whose kinetics are correlated with physiological attributes of adaptation. We show that correlations among genes are strongly context-dependent, in particular, some metabolic groups of genes become highly "intra-correlated" upon increasing environmental pressure.
<br><br>Contact: shayst [at] tx.technion.ac.il
<br><br>Keywords: Gene Recruitment, Adaptation, Regulatory Network
</font><hr></td></tr>
<tr>
  <td><a id="H03" name="H03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H03</b><br>
    <b>Structure and Function of Transcriptional Networks: Rewiring and Crosstalk in the Yeast Osmolarity Pathway</b><br>
    Andrew Capaldi (1), Tommy Kaplan (3), Ying Liu (1), Kristen Cook (1), Naomi Habib (3), Yoseph Barash (3), Nir Friedman (3), Aviv Regev (2), Erin K O'Shea (1)<br>
    (1) Department of Molecular and Cellular Biology, Harvard University; (2) Bauer Center for Genomics Research, Harvard University; (3) School of Computer Science and Engineering, The Hebrew University of Jerusalem
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    To gain insights into transcriptional network structure and function we study the HOG pathway in budding yeast. We combine genomics, molecular and computational approaches and examine how one kinase, Hog1, responds to hyper-osmotic stress, modulates the activity of several transcription factors, and controls the expression of over than 300 genes in different transcriptional modules.
<br>Our findings reveal dense overlapping regulatory network architecture, and shed light on how transcriptional programs can be re-wired and re-utilized to control diverse responses to varying conditions.
<br><br>Contact: tommy [at] cs.huji.ac.il
<br><br>Keywords: Design of Transcriptional Regulatory Networks
</font><hr></td></tr>
<tr>
  <td><a id="H04" name="H04"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H04</b><br>
    <b>Decomposition of Metabolic Networks for Distributed Computation of Elementary Modes</b><br>
    Axel von Kamp, Stefan Schuster<br>
    Department of Bioinformatics, Friedrich-Schiller-University Jena
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    One problem of elementary modes analysis is that the number of modes usually rises exponentially with network complexity. Consequently the calculation of all modes for large metabolic networks is currently intractable. Recently a general procedure for the decomposition of metabolic networks was outlined. In order to use it, reactions from the network have to be selected on which to base the decomposition. Here a heuristics is presented that automatically determines which reactions to use. This allows the calculation of elementary modes with reduced computational effort for the whole network.
<br><br>Contact: kamp [at] minet.uni-jena.de
<br><br>Keywords: Elementary Flux Modes, Metabolic Networks
</font><hr></td></tr>
<tr>
  <td><a id="H05" name="H05"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H05</b><br>
    <b>Understanding the Integrated Network of Signal Transduction and Protein-Protein Interaction using Network Analysis Tools</b><br>
    Gali Niv, Hanah Margalit<br>
    Department of Molecular Genetics and Biotechnology, Faculty of Medicine, The Hebrew University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In this study we used network analysis tools to analyze the integrated network of protein-protein interactions (PPI) and kinase-substrate interactions (KSI) in human. We defined the building blocks of the network and their biological significance. We also compared the integrated network of PPI and KSI to the previously studied network of transcription regulation interactions (TRI) and PPI. We compared both basic network properties and the building blocks that appear in each network. We found several motifs that seem to be unique for the PPI-KSI in comparison to the TRI-PPI networks.
<br><br>Contact: galin [at] md.huji.ac.il
<br><br>Keywords: Network, Motif, Phosphorylation, PPI, Interaction
</font><hr></td></tr>
<tr>
  <td><a id="H06" name="H06"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H06</b><br>
    <b>A Systematic Analysis of Chromatin Modifiers that Act as Transcriptional Co-Factors in the Yeast Genome</b><br>
    Israel Steinfeld (1,2), Ron Shamir (2), Martin Kupiec (1)<br>
    (1) Dept. of Molecular Microbiology and Biotechnology, Tel Aviv University; (2) The School of Computer Science, Tel Aviv University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We present a new methodology that tests for statistically significant expression change of a gene group in a specific experiment. We use this method to search for transcription factors that depend on chromatin modification (CM) factors to carry out their transcriptional programs. 
<br>We employ our methodology on a large data set of S. cerevisiae and demonstrate our method on a well-characterized example. We also manage to uncover many novel putative genetic interactions between transcription factors and CM factors and dissect the mechanisms by which they cooperate to control gene expression.
<br><br>Contact: steinfe [at] post.tau.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="H07" name="H07"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H07</b><br>
    <b>ENRICH: A Computational Aid Tool for Finding Enrichment in Various Data Sets</b><br>
    Tali Goren, Ohad Manor, Tommy Kaplan, Nir Friedman<br>
    School of Computer Science and Engineering, The Hebrew University of Jerusalem
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Which Transcription Factor controls stress response in yeast? Do mitochondrial genes have a stronger response to heat shock than peripheral genes?  
<br>The amount of available Biological data is rapidly increasing. Gene annotations, localization and expression are overwhelming in their richness of information, but how does one extract meaningful Biological conclusions from it?
<br>Toward this end we created ENRICH, a computational aid tool to upload data, manipulate it, and perform various statistical tests and clustering techniques in order to extract a meaningful biological conclusion.
<br><br>Contact: omanor [at] gmail.com
<br><br>Keywords: Transcription, Annotation, Expression, Enrich
</font><hr></td></tr>
<tr>
  <td><a id="H08" name="H08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H08</b><br>
    <b>A Novel Bayesian Probabilistic Method for Comparing Transcription Factor Binding Site Profiles</b><br>
    Naomi Habib (1,2), Tommy Kaplan (1,2), Hanah Margalit (2), Nir Friedman (1)<br>
    (1) School of Computer Science, The Hebrew University of Jerusalem; (2) Faculty of Medicine, The Hebrew University of Jerusalem, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    An important step in understanding transcription regulation is identifying transcription factors DNA binding sites. Various tools exist for identifying binding sites, which output a redundant set of DNA motifs. Overcoming this redundancy and associating DNA motifs with their binding factors is crucial for understanding the regulatory program. We present a method for comparing and clustering DNA motifs, based on Information Theory and Bayesian probabilistic reasoning. We show that our method is more accurate and sensitive than the current methods.
<br><br>Contact: naomih [at] cs.huji.ac.il
<br><br>Keywords: Transcription Factor Bayesian Binding Site
</font><hr></td></tr>
<tr>
  <td><a id="H09" name="H09"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H09</b><br>
    <b>On the Construction of a Hypothetical Metabolic Network: Analysis of the Intersection with the Interactome</b><br>
    Joan Planas-Iglesias (1), Daniel Aguilar (2), Baldomero Oliva (1)<br>
    (1) Universitat Pompeu Fabra (UPF); (2) Institut Municipal d'Investigacions Mediques (IMIM)
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We propose a new method for the construction of the metabolic network overcoming problems related to the arbitrary exclusion of a number of chemical compounds or related with the a priori assumption of the topological properties of the network. The method relies on an score based on the inverse of the frequency of metabolites shared by two enzymes involved in consecutive metabolic reactions. The intersection of the obtained metabolic network and interactome aids inferring similar protein-protein relationships in other organisms than the few from which the actual experiments were performed.
<br><br>Contact: boliva [at] imim.es
<br><br>Keywords: Biochemical Network Pathway Annotation
</font><hr></td></tr>
<tr>
  <td><a id="H10" name="H10"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H10</b><br>
    <b>Genome-Scale Identification of Nucleosome Positions during Cell Cycle Reveals Insights on Transcriptional Regulation</b><br>
    Moran Yassour (1), Tommy Kaplan (1,2), Ariel Jaimovich (1,2), Chih Long Liu (4), Guocheng Yuan (3), Oliver J Rando (3), Nir Friedman (1)<br>
    (1) School of Computer Science, The Hebrew University of Jerusalem; (2) Faculty of Medicine, The Hebrew University of Jerusalem; (3) Bauer Center for Genomics Research, Harvard University; (4) Dept. of Chemistry & Chemical Biology, Harvard University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Transcriptional regulation involves changes in nucleosome positions and occupancy, allowing differential accessibility to functional sites along the DNA. We use a model-based Bayesian algorithm to analyze high-resolution measurements of nucleosomal positioning in S. cerevisiae along the cell cycle, and present a genome-scale mapping of nucleosome positions. Surprisingly, we find that chromatin remodeling at promoters of cell-cycle regulated genes is mainly obtained via changes in nucleosome occupancy rather than shifts in nucleosome positions.
<br><br>Contact: morani [at] cs.huji.ac.il
<br><br>Keywords: Nucleosomes, Bayesian, Graphic Models
</font><hr></td></tr>
<tr>
  <td><a id="H11" name="H11"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H11</b><br>
    <b>A Multi-way Strategy for the Integrative Analysis of High Dimensionality </b><br>
    Ana Conesa (1), Jose Manuel Prats-Montalban (2), Maria Jose Nueda (3), Manuel Talon (1), Alberto Ferrer (2)<br>
    (1) Centro de Genomica, IVIA, Moncada, Spain; (2) Departamento de Estadistica e Investigacion Operativa Aplicadas y Calidad, UPV, Valencia, Spain; (3) Departamento de Estadistica e Investigacion Operativa, UA, Alicante, Spain
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In the work we present our results on the application of multi-way methods for the analysis of complex data structures that are generated in extensive functional genomics studies. Multi-way methods are able to take into account the different levels of data organization and analyze the underlying components of variability that affect different types of biological variables. They are therefore suitable for explorative and variable selection analysis of Systems Biology data. We discuss the potentials of this approach and comment some considerations for its successful application.
<br><br>Contact: aconesa [at] ivia.es
<br><br>Keywords: Multivariate Analysis, Data Integration
</font><hr></td></tr>
<tr>
  <td><a id="H12" name="H12"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H12</b><br>
    <b>Modeling Evolution of Gene and Protein Interactions</b><br>
    Todd A. Gibson (1), Debra S. Goldberg (2)<br>
    (1) University of Colorado Health Sciences Center; (2) University of Colorado
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Current protein network analyses either use abstract evolutionary models lacking gene context, or model current-day protein interactions without a dynamic evolutionary component.  We present a generalizable method for evolving an organism's putative ancestral protein interaction network to its current-day interactions.  We compare evolutionary parameters and topology of distinct protein families.
<br><br>Contact: Todd.Gibson [at] uchsc.edu
<br><br>Keywords: Networks, Evolution, Genome Duplication
</font><hr></td></tr>
<tr>
  <td><a id="H15" name="H15"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H15</b><br>
    <b>Derepression of the follicle stimulating hormone beta subunit gene by GnRH involves CaMKI, calcineurin and Nur77 </b><br>
    Philippa Melamed (1), Stefan Lim (2), Min Luo (1), Mingshi Koh (1)<br>
    (1) Dept of Biological Sciences, National University of Singapore; (2) NUS Graduate School for Integrative Sciences and Engineering, National University of Singapore
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The study aims to understand the molecular mechanisms through which GnRH overcomes repression of gonadotropin gene expression to re-initiate reproductive activity. We propose that the actions of GnRH to derepress the FSH-beta gene in the immature gonadotrope target both Nur77 and FSH-beta genes, involving CaMKI-phosphorylation of class IIa HDACs causing disruption of the repressive complexes. In addition, GnRH activation of calcineurin stimulates Nur77 gene expression which activates the FSH-beta gene likely by facilitating, together with MEF2D, the recruitment of co-activators. 
<br><br>Contact: dbsmp [at] nus.edu.sg
<br><br>Keywords: HDAC, Deacetylation, Histone, Hormone
</font><hr></td></tr>
<tr>
  <td><a id="H17" name="H17"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H17</b><br>
    <b>Biological Implications of Anti-motifs in Transcriptional Regulation Networks</b><br>
    Ying Wang (1), Natasa Przulj (2)<br>
    (1) Dev and Cell Biology Dept., School of Biological Sciences, University of California, Irvine; (2) Department of Computer Science, University of California, Irvine
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    A network motif is defined as a subgraph occurring at a significantly higher frequency than expected in a randomized network. Motifs are believed to reveal evolutionary selection for specific patterns in biological networks. The biological significance of network motifs has been studied extensively. However, biological meaning of under-represented subgraphs, called network anti-motifs, has not been examined. We analyze both motifs and anti-motifs, demonstrate their inter-dependence, and use them to uncover transcriptional regulation differences amongst multicellular and unicellular organisms.
<br><br>Contact: natasha [at] igor.ics.uci.edu
<br><br>Keywords: Anti-motif, Motif, Triad, Multicellular, TRN
</font><hr></td></tr>
<tr>
  <td><a id="H18" name="H18"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H18</b><br>
    <b>Structural Similarity is a Prominent Feature of Genetic Interactions</b><br>
    Alexandra Shulman-Peleg (1), Oranit Dror (1), Dina Schneidman (1), Roded Sharan (1), Ruth Nussinov (2,3), Haim Wolfson (1)<br>
    (1) School of Computer Science, Tel Aviv University; (2) Sackler Institute of Molecular Medicine, Sackler Faculty of Medicine, Tel Aviv University; (3) Center for Cancer Research, Nanobiology Program, National Cancer Institute, Frederick, MD, USA
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The recent availability of large scale genetic interaction networks in yeast and worm allows the investigation of the biological mechanisms underlying these interactions at a global scale. Here, we perform the first genome-scale structural comparison among protein pairs in the two species. We show that significant fractions of genetic interactions involve structurally similar proteins, spanning from 7% and 14% of all known interactions in yeast and worm, respectively. We identify several structural features that are predictive of genetic interactions and we suggest putative mechanisms for genetic interactions among structurally similar proteins.
<br><br>Contact: shulmana [at] tau.ac.il
<br><br>Keywords: Genetic Interactions, Structural Alignment
</font><hr></td></tr>
<tr>
  <td><a id="H21" name="H21"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H21</b><br>
    <b>preCLUE: PREdiction system for CLUstering and regulatory rElation</b><br>
    Ho-Youl Jung, Myung-Guen Chung, Minho Kim, Pora Kim, Seon-Hee Park, Soo-Jun Park<br>
    Bioinformatics Team, Electronics and Telecommunications Research Institute
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Our system provides an integrated procedure for the gene expression profile analysis. Users can do clustering and also predict the regulatory relation between transcription factor and target gene using single platform. In the clustering process, we provide conventional clustering algorithms. In the predicting module, regulatory relations are inferred by the SVM (Support Vector Machine) or graph optimization method based-on local alignment.
<br><br>Contact: hoyoul.jung [at] etri.re.kr
<br><br>Keywords: Regulatory Relation, Gene Clustering
</font><hr></td></tr>
<tr>
  <td><a id="H24" name="H24"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H24</b><br>
    <b>Comprehensive Gene Pathway Analysis - A Tool for Pathway Analysis of CGH and Microarray Data</b><br>
    Felix Engel, Nicolas Delhomme, Natalia Becker, Frederic Blond, Peter Lichter, Grischa Toedt<br>
    German Cancer Research Center
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We developed a system which performs pathway analysis on CGH and microarray data. It identifies the most discriminating pathways between two tumour types from a selection of pathway databases (KEGG, BioCarta, BioCyc and GOTerms). We use random permutations and significance tests to score the pathways. The comparison results are annotated and visualized.
<br><br>Contact: f.engel [at] dkfz.de
<br><br>Keywords: Pathway Analysis, CGH, Microarray, Expression
</font><hr></td></tr>
<tr>
  <td><a id="H27" name="H27"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H27</b><br>
    <b>Combining Inference from Differential Expression and Variability in Microarray Data Analysis</b><br>
    Joan Valls (1,2), Monica Grau (1), Xavier Sole (1), Pilar Hernandez (1), David Montaner (3), Joaquin Dopazo (3), Miguel A Peinado (4), Gabriel Capella (1), Miguel Angel Pujana (1), Victor Moreno (1,2)<br>
    (1) Bioinformatics and Biostatistics Unit, Catalan Institute of Oncology, IDIBELL; (2) Autonomous University of Barcelona; (3) Bioinformatics Department, Centro de Investigacion Principe Felipe; (4) Cancer Research Institute, IDIBELL
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We propose a new procedure for the analysis of microarray data by combining inference from differential expression and variability (CLEAR-test). Analysis of public DNA microarray datasets reveal the performance of the CLEAR-test relative to the t-test and alternative methods for differential expression. Empirical and simulated data analyses demonstrate the greater reproducibility and statistical power of the CLEAR-test with respect to current alternative methods.
<br><br>Contact: mapujana [at] iconcologia.net
</font><hr></td></tr>
<tr>
  <td><a id="H28" name="H28"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H28</b><br>
    <b>Gene Expression Profiles of AML Derived Stem Cells; Similarity to Hematopoietic Stem Cells</b><br>
    Hilah Gal (1,2), Ninette Amariglio (2), Luba Trakhtenbrot (2), Jasmine Jacob-Hirsh (2), Ofer Margalit (2), Abraham Avigdor (2), Arnon Nagler (2), Sigal Tavor (3), Liat Ein-Dor (1),  Tsvee Lapidot (1), Eytan Domany (1), Gideon Rechavi (2), David Givol (1)<br>
    (1) Weizmann Institute of Science, Israel; (2) Chaim Sheba Medical Center, Tel-Hashomer, Israel; (3) Sourasky Medical Center, Tel-Aviv, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In our work, we fractionated AML stem cells, CD34+CD38- (LSC), and compared their gene expression profile to the AML differentiated cells (CD34+CD38+). We found 409 modulated genes in LSC, showing under-expression of cell cycle and DNA repair genes, consistent with the quiescent state of stem cells. A large portion of the modulated genes in LSC (34%) were also found to be modulated in normal HSC. We identified the Notch pathway as important in LSC self-renewal. The inhibition of this pathway reduced LSC colony formation. Identification of additional genes that regulate LSC self-renewal may provide new targets for therapy.
<br><br>Contact: hilahg [at] weizmann.ac.il
<br><br>Keywords: Leukemia, Stem Cells, Microarray, Notch
</font><hr></td></tr>
<tr>
  <td><a id="H29" name="H29"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H29</b><br>
    <b>SPIKE - A Database, Visualization and Analysis Tool for Signaling Pathways</b><br>
    Ran Elkon (1), Rita Vesterman (1), Nira Amit (1), Igor Ulitsky (2), Mali Weisz (1), Nir Orlev (1), Giora Sternberg (1), Ran Blekhman (1), Jackie Assa (2), Yosef Shiloh (1), Ron Shamir (2)<br>
    (1) The David and Inez Myers Laboratory for Genetic Research, Department of Molecular Genetics and Biochemistry, Sackler School of Medicine, Tel Aviv University, Israel; (2) School of Computer Science, Tel Aviv University, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Our realization of the complexity of signaling networks that regulate cellular physiology is growing commensurate with the rapid expansion in biological knowledge. It is now clear that biological pathways that govern cellular development and responses to environmental challenges are not linear, parallel, and independent, but instead form a web of interlocking processes. Given the complexity of these networks, the assimilation and interpretation of the wealth of data collected on signaling pathways become an acute bioinformatics challenge. To cope with this challenge, we are developing SPIKE (Signaling Pathway Integrated Knowledge Engine). SPIKE is a knowledge base of signaling pathways, which can be utilized to analyze any signaling network. At present, we focus our efforts on populating SPIKE with data on networks induced by DNA damage in human cells. 
<br>SPIKE contains three main software components: 1) a database of cellular signaling pathways; 2) a visualization package that allows interactive graphic representations of regulatory interactions; 3) an algorithmic inference engine that analyzes the networks, aiming to discover novel functional interplays between network components.
<br>SPIKE's database contains extensive and highly curated data on pathways induced by DNA damage, such as cell-cycle checkpoints, apoptosis and other stress responses. Our vision is that the database will be populated by a distributed and collaborative effort undertaken by multiple groups, with quality supervised by SPIKE's curators.
<br><br>Contact: ulitskyi [at] post.tau.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="H31" name="H31"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster H31</b><br>
    <b>Functional Profiling of Microarray Experiments Using Text-Mining Derived Bioentities</b><br>
    Pablo Minguez, Fatima Al-Shahrour, David Montaner, Joaquin Dopazo<br>
    Department of Bioinformatics, Centro de Investigacion Principe Felipe, Autopista del Saler 16, 46013 Valencia, Spain
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The proper functional interpretation of genome-scale experiments remains a fundamental challenge. We propose a new algorithm MarmiteScan (a variant of FatiScan), that tests the behaviour of blocks of functionally-related genes (instead of single genes) to find connections between experimental results and terms with biological meaning. We focus on chemical compounds and disease-associated terms. 
<br>We applied this methodology to a set of Acute Myeloid Leukemia (AML) samples treated with a panel of compounds inducing, with different success, their differentiation to mature cells. The gene expression data of each AML sample treated with a compound was compared to the expression data of the negative controls, AML cells and AML cells treated with compounds that do not alter gene expression. The output was a set of lists of genes, sorted by the importance in the difference between the compound action versus AML status. Then, we applied to each list the MarmiteScan, a variant of the FatiScan algorithm, to extract blocks of functionally related genes. From the thirteen experiments analysed, eight showed chemical products significantly associated to high expression in either AML+compound, AML control samples or both.
<br><br>Contact: pminguez [at] cipf.es
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
