<html>
<head>
<title>Poster Abstracts K</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
  <td>
    <font size="4" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <br><b>Poster Abstracts for Category K: Sequence analysis</b><br><br>
  </td>
</tr>
<tr>
  <td><a id="K02" name="K02"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K02</b><br>
    <b>A Computer Program to Predict Gene Position and Reveal Inconsistent Designation of Similar Genes in Different Organisms</b><br>
    Marjanca Starcic Erjavec (1), Marko Erjavec (2), Darja Zgur-Bertok (1)<br>
    (1) Department of Biology, Biotechnical Faculty, University of Ljubljana; (2) UnistarLC
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    The DNA sequence is the foundation upon which the structure and function of an organism to a large degree depends. To find the most similar sequences the computer program BLAST is used. The presented computer program links the GenBank data with BLAST results and draws a figure with information including the sequence name, the percentage of similarity, gene positions and designations, that can be used to unravel the positions of genes on the examined DNA sequence and the different designations for genes with same function in different organisms. Examples of the computer program use are given.
<br><br>Contact: Marjanca.Starcic.Erjavec [at] bf.uni-lj.si
<br><br>Keywords: Gene Position and Designation, BLAST, GenBank
</font><hr></td></tr>
<tr>
  <td><a id="K03" name="K03"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K03</b><br>
    <b>Detection and Reduction of Evolutionary Noise in Correlated Mutation Analysis</b><br>
    Orly Noivirt, Miriam Eisenstein, Amnon Horovitz<br>
    Weizmann Institute of Science
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Direct or indirect inter-residue interactions in proteins are often reflected by mutations at one site that compensate for mutations at another site. Correlated mutation analysis for non-interacting proteins showed that the signal due to real interactions is of similar magnitude to the noise that arises from other evolutionary processes. A new method for detecting correlated mutation is presented that reduces the evolutionary noise by considering the evolutionary distances within the protein family.
<br><br>Contact: orly.noivirt [at] weizmann.ac.il
<br><br>Keywords: Coordinated Mutations, Co-evolving Residues
</font><hr></td></tr>
<tr>
  <td><a id="K06" name="K06"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K06</b><br>
    <b>Mammalian microRNA Prediction using a Support Vector Machine Model of Sequence and Structure</b><br>
    Ying Sheng, Par G. Engstrom, Boris Lenhard<br>
    Computational Biology Unit, Bergen Computational Center of Science, University of Bergen, Norway
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    MicroRNAs are endogenous small noncoding RNAs with important regulatory roles in animals and plants. We present an efficient microRNA prediction method that uses sequence conservation profiles and secondary structure characteristics. The method predicts an extensive set of potential human and mouse microRNAs. We expect our predictions to contribute a significant number of new candidate miRNAs for experimental verification.
<br><br>Contact: ying.sheng [at] bccs.uib.no
<br><br>Keywords: microRNA, Support Vector Machine, Prediction
</font><hr></td></tr>
<tr>
  <td><a id="K08" name="K08"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K08</b><br>
    <b>Classifying High-Level Protein Functionality: Toxins and Toxin-Like Proteins</b><br>
    Noam Kaplan (1,2), Michal Linial (2)<br>
    (1) Weizmann Institute; (2) The Hebrew University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Toxins are short proteins that appear in animal venom and are extremely varied in sequence, structure and function. We have developed a toxin classifier based on extracting sequence-derived features that are related to the notion of structural stability. Application of the classifier to the honey bee genome and to a set of recently-discovered mouse sequences reveals 3 novel toxin-like proteins. Remarkably, these proteins seem to be expressed in the brain rather than in the venom. We suggest that the novel endogenous toxin-like proteins may function as neuromodulators via toxin-like mechanisms.
<br><br>Contact: noam.kaplan [at] weizmann.ac.il
<br><br>Keywords: Toxins, Machine Learning, Function Prediction
</font><hr></td></tr>
<tr>
  <td><a id="K12" name="K12"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K12</b><br>
    <b>Built-in Switches Allow Versatility in Domain-Domain Interactions</b><br>
    Eyal Akiva, Hanah Margalit<br>
    Department of Molecular Genetics and Biotechnology, Faculty of Medicine, The Hebrew University
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Certain domains have been identified as mediating homo-dimerization. As these domains are found also in proteins that function as monomers, a question arises as to what determines the oligomeric state of proteins having such domains. By comparing multiple sequence alignments of known dimers and monomers that contain such domains, we identify the domain segments that are responsible for this versatility. Analysis of our results in view of relevant solved structures provides insight into the molecular basis of this phenomenon and has implications for predicting the oligomeric states of proteins.
<br><br>Contact: akiva2 [at] md.huji.ac.il
<br><br>Keywords: Protein, Interaction, Domain, Dimer, Monomer
</font><hr></td></tr>
<tr>
  <td><a id="K13" name="K13"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K13</b><br>
    <b>Augmenting Protein Sequence Alignment of Remote Homologues using Secondary Structure Confidence Scores</b><br>
    Yaniv Loewenstein, Elon Portugaly, Michal Linial<br>
    Hebrew University of Jerusalem, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Incorporating sequence together with protein secondary structure has a synergistic affect on alignment quality and remote homology detection, without affecting the alignment speed. Yet, synergism vanishes when noisy structure prediction is used. Using Bayesian confidence levels in the alignment scoring function, we overcome this inherent problem, and reproduce substantially better performance than current methods, even when no experimental structure is at hand. Our method is applicable to BLAST searches, better sequence profiles, and large-scale unsupervised classification of protein domains.
<br><br>Contact: lonshy [at] cs.huji.ac.il
<br><br>Keywords: Alignment, Confidence, Homology Detection
</font><hr></td></tr>
<tr>
  <td><a id="K14" name="K14"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K14</b><br>
    <b>Using Co-occurrence of Transcription Factor Binding Sites for the Assessment of Regulatory Potential</b><br>
    Holger Klein, Martin Vingron<br>
    Max Planck Institute for Molecular Genetics
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We present an approach for the detection of co-occurrence of transcription factor binding sites within known regulatory sequences. We annotate a set of upstream regions of human genes with predicted TFBSs based on a set of representative position weight matrices. We count co-occurring pairs of TFBSs using a sliding window and calculate a log-odds score of observed vs. expected number of pairs to identify significant combinations. To assess the co-occurrence scores we use known interactions of TFs. A way to use the introduced co-occurrence scores for promoter prediction is outlined.
<br><br>Contact: holger.klein [at] molgen.mpg.de
<br><br>Keywords: Gene Regulation, Co-occurrence of TFBS
</font><hr></td></tr>
<tr>
  <td><a id="K15" name="K15"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K15</b><br>
    <b>Olfactory Receptors in Peptide Space</b><br>
    Assaf Gottlieb (1), Tsviya Olender (2), Doron Lancet (2) , David Horn (1)<br>
    (1) School of Physics and Astronomy, Tel Aviv University, Israel; (2) Dept of Molecular Genetics, Weizmann Institute of Science, Israel
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We use the Motif EXtraction algorithm (MEX) to extract specific peptides from Olfactory Receptor (OR) proteins of seven vertebrate species. The peptides trace well evolutionary trends. 567 peptides are found to be common to all tetrapod species, probably related to shared structure and function. While mammals share most of their peptides (3016), many peptides in non-mammals species are species-specific. Within mammals 1874 peptides are OR-family specific. Peptides that lie on the extra-cellular part of the ORs may be related to specific OR functions.
<br><br>Contact: assafgot [at] post.tau.ac.il
<br><br>Keywords: Olfactory Receptors, Motif Extraction
</font><hr></td></tr>
<tr>
  <td><a id="K17" name="K17"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K17</b><br>
    <b>SIGI-HMM: Score-Based Prediction of Genomic Islands in Prokaryotic Genomes using HMMs </b><br>
    Oliver Keller (1), Katharina Surovcik (1), Thomas Brodag (1), Rainer Merkl (2),  Roman Asper (1), Carsten Damm (3), Stephan Waack (1)<br>
    (1) Institut fur Informatik, Universitat Gottingen; (2) Institut fur Biophysik und Physikalische Biochemie, Universitat Regensburg; (3) Institut fur Numerische und Angewandte Mathematik, Universitat Gottingen
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Horizonal gene transfer, i.e. the process of acquiring genes from foreign species, is a frequent phenomenon among microbial species. It is considered a strong evolutionary force allowing rapid adaptation to changing environmental demands. The transferred pieces of DNA often comprise a large number of genes that are found in contiguous regions called Genomic Islands (GIs).
<br>We have implemented the program SIGI-HMM that predicts GIs and the putative donor of each  individual alien gene, by the use of a HMM that takes into account codon usage.
<br><br>Contact: keller [at] cs.uni-goettingen.de
<br><br>Keywords: Genomic Islands, Hgt, HMM, Codon Usage
</font><hr></td></tr>
<tr>
  <td><a id="K18" name="K18"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K18</b><br>
    <b>A New Approach for Estimation of Statistical Significance in Sequence Profile-Profile Comparisons</b><br>
    Mindaugas Margelevicius, Ceslovas Venclovas<br>
    Institute of Biotechnology, Lithuania
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Sequence profile-profile comparison is one of the most sensitive techniques for distant homology detection. We propose a new approach to estimate statistical significance of the profile-profile alignments. The exhaustive testing showed that this approach combined with a newly developed comparison procedure compares favorably to a number of other existing methods.
<br><br>Contact: minmar [at] ibt.lt
<br><br>Keywords: Profile Comparisons, Statistical Significance
</font><hr></td></tr>
<tr>
  <td><a id="K20" name="K20"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K20</b><br>
    <b>An Algorithm for the Identification of Active Site Residues Using Pfam Alignments</b><br>
    Jaina Mistry, Alex Bateman, Rob Finn<br>
    Wellcome Trust Sanger Institute
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Over 10% of the 8300 Pfam families are enzymatic, but within these families only a small fraction of the sequences have been biochemically characterized.  We have developed an algorithm that takes experimentally determined active site residues contained within a family, and identifies the presence of these active site residue patterns within other family members.  We compare our results to UniProt and the Catalytic Site Atlas.  Using this algorithm we have predicted 606110 active site residues and have increased the active site annotations in Pfam by over 200 times.
<br><br>Contact: jm14 [at] sanger.ac.uk
<br><br>Keywords: Catalytic, Active Site, Annotation, Function
</font><hr></td></tr>
<tr>
  <td><a id="K23" name="K23"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K23</b><br>
    <b>Information Content and Sequence Periodicity</b><br>
    Alexander Bolshoy<br>
    Genome Diversity Center, University of Haifa
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Here we report a novel information content based method for sequence analysis. The method can conveniently indicate all kinds of periodicity and repeat-related features in a set of genomic DNA sequences. We illustrate the power of the method by studying the nucleosomal database of Ioshikhes et al. and intergenic regions of E. coli.
<br><br>Contact: bolshoy [at] research.haifa.ac.il
</font><hr></td></tr>
<tr>
  <td><a id="K24" name="K24"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K24</b><br>
    <b>DeltaProt: Toolbox for Molecular Comparison of Proteins based on Sequence Alignments</b><br>
    Steinar Thorvaldsen (1), Tor Fla (1), Nils P. Willassen (2)<br>
    (1) University of Tromso, Faculty of Science, Tromso, Norway; (2) Norwegian Structural Biology Centre, Tromso, Norway
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    We present statistical methods, trend-tests and visualisations that are useful when the protein sequences in alignments can be divided into two or more populations based on known phenotypic traits such as preference of temperature, pH, salt concentration or pressure. The algorithms have been successfully applied in the research on extremophile organisms. 
<br><br>Contact: steinart [at] math.uit.no
<br><br>Keywords: Physicochemical Properties, Extremophiles
</font><hr></td></tr>
<tr>
  <td><a id="K25" name="K25"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K25</b><br>
    <b>Assessment of Protein Domain Classifications: Automatic Sequence-Based Method and Methods based on 3D Structures</b><br>
    Elon Portugaly (1), Nathan Linial (1), Michal Linial (2)<br>
    (1) School of Computer Science and Engineering, The Hebrew University of Jerusalem; (2) Dept. of Biochemistry, Inst. of Life Sciences, The Hebrew University of Jerusalem
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    EVEREST is an automatic system that identifies and classifies domains within a database of protein sequences. We show that the set of EVEREST families is as similar to the set of CATH families and to the set of SCOP families as the latter two sets are similar to each other.
<br><br>Contact: elonp-eccb06 [at] cs.huji.ac.il
<br><br>Keywords: Automatic Annotation, Protein Domains
</font><hr></td></tr>
<tr>
  <td><a id="K27" name="K27"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K27</b><br>
    <b>Enhancing Transmembrane Beta-Barrel Topology Prediction by Information Encoded in Multiple Sequence Alignments</b><br>
    Costas Tsirigos (1), Pantelis Bagos  (1), Vasilis Promponas (1,2),  Stavros Hamodrakas (1)<br>
    (1) Department of Cell Biology & Biophysics, Faculty of Biology, University of Athens, Greece; (2) Department of Biological Sciences, University of Cyprus, Nicosia, Cyprus
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    In this work we demonstrate that some common pitfalls of predictive algorithms (for example broken strands, false positives or dipping loops) may be resolved by using the positional conservation and gap distribution of multiple sequence alignments. Our results indicate that it is possible to use information encoded in multiple sequence alignments to post-process and enhance topology prediction for transmembrane beta-barrels. We also have preliminary data suggesting that positions in the alignment mainly occupied by gaps may accurately indicate shifted predictions.
<br><br>Contact: vprobon [at] ucy.ac.cy
<br><br>Keywords: TM Beta-Barrel, Multiple Alignment, Prediction
</font><hr></td></tr>
<tr>
  <td><a id="K28" name="K28"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K28</b><br>
    <b>On the Kinetics of Prokaryotic Transcription Initiation</b><br>
    Johanna Weindl (1), Pavol Hanus (1), Zaher Dawy (2), Juergen Zech (3), Joachim Hagenauer (1), Jakob C. Mueller (3,4)<br>
    (1) Institute for Communications Engineering (LNT), Technical University of Munich (TUM); (2) Department of Electrical and Computer Engineering (ECE), American University of Beirut; (3) Institute for Medical Statistics and Epidemiology, Technical University of Munich (TUM); (4) Hertie-Institute for Clinical Brain Research, University Clinic Tuebingen
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Transcription initiation in prokaryotes has been extensively studied in the past decades. However, little is known about the kinetics involved in promoter detection by the RNA polymerase (RNAP) and its sigma subunit. We present an approach to relate the binding energy between the sigma factor and the DNA to the kinetics of promoter detection during the first step of transcription. Results suggest that the sequence surrounding the promoters contains important information to guide the RNAP and its sigma subunit in a way to increase the probability of transcription initiation.
<br><br>Contact: jweindl [at] tum.de
<br><br>Keywords: Kinetics, Dynamics, Transcription Initiation
</font><hr></td></tr>
<tr>
  <td><a id="K29" name="K29"></a>
    <br>
    <font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Poster K29</b>&nbsp;&nbsp;&nbsp;<font color="#ff0000"><b><i>Late-Breaking Results</i></b></font><br>
    <b>Towards Alignment Independent Quantitative Assessment of Homology Detection Methods</b><br>
    Avihay Apatoff (1), Eddo Kim (1), Yossef Kliger (2)<br>
    (1) The Mina and Everard Goodman Faculty of Life Sciences, Bar-Ilan University; (2) Compugen Ltd.
    </font></td></tr>
<tr>
  <td>
    <font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
    <b>Abstract:</b><br>
    Identification of homologous proteins provides a basis for protein annotation. Sequence alignment tools reliably identify homologs sharing high sequence similarity. However, identification of homologs that share low sequence similarity remains a challenge. Lowering the cutoff value could enable the identification of diverged homologs, but also introduces numerous false hits. Methods are being continuously developed to minimize this problem. Estimation of the fraction of homologs in a set of protein alignments can help in the assessment and development of such methods. Herein, we present a computational approach that estimates the amount of homologs in a set of protein pairs. The method requires a prevalent and detectable protein feature that is conserved between homologs. By analyzing the feature prevalence in a set of pairwise protein alignments, the method can estimate the number of homologs pairs in the set, independently of the alignments quality. Using the HomoloGene database as a standard of truth, we implemented this approach in a proteome-wide analysis. The results revealed that this approach works well for estimating the number of homologous proteins in a wide range of homology values. In summary, the presented method can accompany homology searches and method development, provides validation to search results, and allows tuning of tools and methods.
<br><br>Contact: kliger [at] compugen.co.il
</font><hr></td></tr>
</tbody>
</table>
</body>
</html>
