<html>
<head>
<title>List of Keynote Abstracts</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
		<!-- ImageReady Preload Script (basic.psd) --><!-- End Preload Script -->
		
<script language="JavaScript">
<!--
function checkMaxInput(obj,maxLen) {
if (obj.value.length > maxLen) obj.value = obj.value.substring(0, maxLen);
}
// -->
</script>

</head>
<!--<body bgcolor="#FFFFFF" leftmargin="0" topmargin="0" marginwidth="0" marginheight="0">
		<div align="center">-->

<body bgcolor="#FFFFFF" text="#333333" link="#990000" leftmargin="0" marginheight="0" marginwidth="0" topmargin="0">
		<table id="Table_01" width="1425" border="0" cellspacing="0" cellpadding="0" height="197">
			<tr>
				<td><img src="eccb_logo_reg.jpg" alt="" height="259" width="735" border="0"></td>
				<td></td>
			</tr>
		</table>
		<table width="731" border="0" cellspacing="0" cellpadding="0">
		  <tr>
			<td width="731">

<!--
<link type="text/css" rel="StyleSheet" href="submission_accepted.cgi_files/easy.css">
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"></head><body bgcolor="white">
<script src="submission_accepted.cgi_files/easy.js"> </script><script>easyState.user='g71ufxin4Gsq19zHP4Bf';easyState.role='C';easyState.track=1;</script><div class="menu"><span class="menuheading">ECCB 2006 (chair)</span><br> <a onmouseover="top.status='list of submissions';return true;" href="http://www.easychair.org/ECCB06/submissions_show.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Submissions</a> | <a onmouseover="top.status='reviews for all papers';return true;" href="http://www.easychair.org/ECCB06/review_all.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">All&nbsp;Reviews</a> | <a onmouseover="top.status='current status of all papers';return true;" href="http://www.easychair.org/ECCB06/status.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Status</a> | <a onmouseover="top.status='assign papers to PC members for reviewing';return true;" href="http://www.easychair.org/ECCB06/assignment.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Assign&nbsp;Reviews</a> | <a onmouseover="top.status='view or update information about PC members';return true;" href="http://www.easychair.org/ECCB06/pc_show.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">PC</a> | <a onmouseover="top.status='list of recent (all) events';return true;" href="http://www.easychair.org/ECCB06/events.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Events</a> | <a onmouseover="top.status='list of recent (all) emails sent by EasyChair';return true;" href="http://www.easychair.org/ECCB06/email_show_all.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Letters</a> | <a onmouseover="top.status='other facilities';return true;" href="http://www.easychair.org/ECCB06/other.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Other</a> | <a onmouseover="top.status='view or update EasyChair configuration';return true;" href="http://www.easychair.org/ECCB06/configure.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Configure</a> | <a onmouseover="top.status='your account information and updates';return true;" href="http://www.easychair.org/ECCB06/account.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">My&nbsp;Account</a> | <a onmouseover="top.status='login with a different role';return true;" href="http://www.easychair.org/ECCB06/login_as.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;">Login&nbsp;As</a> | <a onmouseover="top.status='exit EasyChair';return true;" href="http://www.easychair.org/ECCB06/logout.cgi?es=C1;eu=g71ufxin4Gsq19zHP4Bf" onmouseout="top.status='';return true;"><font color="orange">Logout</font></a></div><h2 class="blue">List of Accepted Papers</h2><form method="post" action="submission_accepted.cgi" enctype="multipart/form-data">
<input name="eu" value="g71ufxin4Gsq19zHP4Bf" type="hidden">
<input name="es" value="C1" type="hidden">
<input tabindex="1" name="submit" value="Hide Abstracts" class="redbutton" type="submit">
<div></div></form>
-->
<table border="0" cellpadding="3" cellspacing="0"><tbody>
<tr>
<td>
<hr/>
									<a id="1" name="1"></a><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Principles of Gene Expression Evolution</b></font><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><a href="http://barkai-serv.weizmann.ac.il/GroupPage/index.htm" target="_blank"><b>Prof. Naama Barkai</b></a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Weizmann Institute of Science, Israe</font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">l</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
									</font>
</td>
</tr>

<tr>
<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
<b>Abstract:</b><br>
										It is widely believed that evolution of gene expression plays a key role in the generation of phenotypic diversity. Still, now much is known about the priciples which drive inter-species differences in gene expression. I will describe recent results from our lab addressing this issue.</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br></font>
									<hr/>
								</td>
</tr>
<!--<br>-->

<tr>
<td>
									<p><a id="2" name="2"></a><br>
										<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Structural Bioinformatics, Cancer and Drug Discovery<br>
											</b></font><a href="http://www-cryst.bioc.cam.ac.uk/%7etom/" target="_blank"><b><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Prof. Sir Tom Blundell, FRS</font></b></a><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, University of Cambridge, UK<br>
										</font></p>
									<font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">
</font></td>
</tr>

<tr>
<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/></b>Structural bioinformatics - using the knowledge of the three-dimensional structures of protein targets now emerging from genomic data - has the potential to accelerate greatly drug discovery, but technical challenges and time constraints have traditionally limited use primarily to target validation and lead optimization. Application are now being extended into new approaches for lead discovery (for reviews see Blundell <i>et al.,</i> 2002; Congreve <i>et al</i>., 2005). New informatics approaches are changing the way we look at protein structures and emphasising the importance of protein ensembles. Virtual screening coupled with high throughput X-ray crystallography is focused on identifying one or more weakly binding small-molecule fragments from compound libraries consisting of hundreds of small-molecule fragments. The high-resolution definition of this binding interaction provides an information-rich starting point for medicinal chemistry. I will give examples of using this approach in protein kinase targets important in oncology.<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">But many important targets for cancer are multiprotein assemblies. Structural bioinformatics can contribute to predicting protein interactions, but a major challenge for drug discovery arises from the very large surfaces that are characteristic of many of the protein complexes, for example those involved in receptor recognition and signal transduction (see for example, Pellegrini <i>et al.,</i> 2000). This is especially true of complexes that are assembled from preformed globular domains. Not only is it difficult to bind a small molecule to the large, relatively flat surfaces of such proteins involved in protein interactions, but it is also difficult to disrupt the interaction entirely even if one did. It remains to be seen whether the emerging lead discovery approaches discussed in this lecture will prove suitable for these systems. However, recent analyses of multiprotein systems involved in cell regulation and signalling have identified a large number in which one component involves a flexible or unstructured region of the polypeptide chain. An example involves the complex of the human recombinase, Rad51, and the product of the breast cancer associated gene, BRCA2 (Pellegrini <i>et al.,</i> 2003), which is not only scientifically revealing but offers an encouraging and perhaps more druggable site of interaction that could be used to target agents that would be helpful during chemo- or radio-therapy. We suggest that proteins forming interactions with a ligand that comprises a continuous region of flexible peptide may be more druggable targets than where complexes are formed from preformed globular protein structures.<br>
										<br>
										1. </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Blundell TL, Sibanda BL, Wander Montalvao R, Brewerton S, Chelliah V, Worth CL, Harmer NJ, Davies O, Burke D (2006) Structural biology and bioinformatics in drug design: opportunities and challenges for target identification and lead discovery. Phil. Trans. R. Soc. B 361: 413-423<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">2. Blundell, T.L., Jhoti, H. and Abell, C. (2002). High-Throughput crystallography for lead discovery in drug design. Nature Reviews Drug Discovery. 1, 45-54<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">3. Furnham N, Blundell TL, DePristo MA, Terwilliger T (2006) Is one solution sufficient? Nature Structural Molecular Biology. 13, 184-185<br>
										<br>
									</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">4. Pellegrini L., Burke D.F., von Delft F., Mulloy B., Blundell TL (2000) Crystal Structure of fibroblast growth factor receptor ectodomain bound to ligand and heparin. <i>Nature </i>407, 1029-1034<br>
										<br>
											5. </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Pellegrini L, Yu DS, Lo T, Anand S, Lee M, Blundell TL, Venkitaraman AR (2002) Insights into DNA recombination from the structure of a RAD51-BRCA2 complex. Nature 420, 287-293<br>
										<br>
										6. </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Congreve M, Murray CW and Blundell TL (2005) Structural Biology and Drug Discovery. Drug Discovery Today 10, 895-907<br>
										<br>
									</font>
									<hr/>
								</td>
</tr> 

<tr>
<td>
<p><a id="3" name="3"></a><br/>
<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Combinatorial Algorithms for Regulatory Network Analysis and Pathway Reconstruction<br>
</b><a href="http://www.cs.berkeley.edu/People/Faculty/Homepages/karp.html" target="_blank"><b>Prof. Richard Karp</b></a>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">University of California, Berkeley, USA</font><a href="http://www.cs.berkeley.edu/People/Faculty/Homepages/karp.html" target="_blank"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><br>
</font></a></p>
<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font></td></tr><tr><td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
</b>This talk will survey new and old algorithms for identifying coherently interacting sets of proteins conserved in two or more species and revealing the causal relationships and detailed logic of regulatory pathways.<br>
<br>
Specific topics will include:<br>
(1) Finding dense subgraphs of large graphs;<br>
(2) Finding structures indicative of signal transduction pathways or molecular complexes occurring  in the protein interaction graph of a single species or conserved across the protein interaction graphs of two or more species;<br>
(3) Determining causal relationships among proteins using single-cell flow cytometry measurements of protein activity.<br>
<br>
Joint work with Joseph Dale, Eran Halperin, Trey Ideker, Mani Narayanan, Ron Shamir and Roded Sharan.<br>
											
<hr/>
</font>
</td>
</tr> 

<tr>
<td><a id="4" name="4"></a><br/>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Prediction of Protein Structure, Function and Druggability on a Proteomic Scale</b></font><br>
									<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><a href="http://cssb.biology.gatech.edu/skolnick/" target="_blank">Prof. Jeffrey Skolnick</a></b>, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Georgia Institute of Technology, USA<br>
</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font>
</td>
</tr>

<tr>
<td><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract: <br/>
										</b>A novel method for the prediction of protein structure, function and druggability based on the &quot;sequence to structure to function&quot; paradigm has been developed. We first show recent results that suggest that for compact single domain proteins, the PDB is most likely complete and that the completeness can be explained by the packing of compact, hydrogen bonded, secondary structural elements. We next present results from the application of our structure prediction algorithm, TASSER to all GPCRs in the human genome. Based on confidence criteria, 90% should have approximately correct structures, and clustering shows that structurally similar GPCRs have similar function even when their sequences are diverse. We then describe our multimeric structure prediction algorithm, MULTITASSER, and its application to the prediction of protein-protein interactions. Next, a newly developed method for the accurate inference of protein biochemical function is presented and results of the comprehensive analysis of all sequenced genomes and the automated assignment of proteins to metabolic pathways are shown. Finally, we combine these approaches into a pathway-based method for the prediction of druggable protein targets and apply the resulting methodology to the human genome.<br>
									</font>

									<hr/>
</td>
</tr> 

<tr>
<td>
									<p><a id="5" name="5"></a><br/>
										<font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Evolution Teaches Protein Function Prediction</b></font><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b><br>
											</b></font><b><a href="http://www.rostlab.org/" target="_blank"><font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Prof. </font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Burkhard Rost</font></a></b><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, </font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Columbia University, New York, USA<br>
										</font></p>
								</td></tr>
										<tr><td>
									<p><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:<br/>
											</b>The function of most proteins in the human genome remains obscure and target for extensive and expansive investigations. Protein function is a rather intuitive concept, yet it is neither a well-defined term, nor is there a generic descriptor for all aspects of function. This has hampered the progress in the development of general methods that predict function directly from sequence. Our group has been developing many, unique template-based and <i>de novo</i> prediction methods. Our special edge is the combination of evolutionary information and machine learning. One particular challenge that we have been addressing over the last years with our CS colleague Yechiam Yemini is the design of a semantic modeling server to integrate access and manipulations of diverse, distributed biological resources. This modeling server is based on an extended object-relationship network model for correlating and manipulating biological resources.<br>
											<br>
										</font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Our long-term goal is the contribution to the development of a comprehensive system that models three aspects of function particularly in multi-cellular organisms: where is a molecule located most of the time, what does it interact with, and when does the interaction occur. In my talk, I will focus on three particular topics pertaining to the first of these three objectives. Firstly, I will describe one of our methods for the</font><i><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> de novo</font></i><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"> prediction subcellular localization. Secondly, I will sketch our first steps toward <i>de novo</i> predictions of protein-protein and protein-DNA interactions. Lastly, I will briefly sketch the idea behind our most recent, preliminary improved method for database searches employed for the selection of targets in the context of large-scale structural genomics. One goal of structural genomics is to determine one experimental protein structure for each representative protein family. Experimental structures have been determined for over 200 of the proteins selected by us by our experimental colleagues. In this context, I shall present some of our recent findings about the organization of sequence/structure space.<br>
										</font></p>
									<font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font>
									<hr/>
								</td></tr> <tr><td>
									<p><a id="6" name="6"></a><br/>
										<font size="3" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Markov Models in Protein Evolution: The Resolvent Method and Family-Specific Rates<br>
												<a href="http://cmb.molgen.mpg.de/" target="_blank">Prof. Martin Vingron</a></b></font><font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">, Max Planck Institute for Molecular Genetics, Germany<br>
										</font><font face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">		</font></p>
								</td>
								</tr>
								
<tr>
<td>
<font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"><b>Abstract:</b></font><br/>
									<font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif">Markov models for amino acid exchanges are the basis for evolutionary studies on proteins. This talk will shortly summarize the corresponding theory and introduce an estimation procedure for amino exchange matrices. We will further explore a simple parameterization to explore family specific rates in protein evolution. As expected, one can see that, e.g., enzymes tend to have low family specific rates while extracellular proteins show high rates. Likewise, one can observe a certain correlation between this rate and whether a protein is essential. A possible link between the family specific rate and the age of a protein family will also be discussed.<br>
										<br>
										 References:<br>
										T. Muller and M. Vingron, Modeling amino acid replacement. J.Comput.Biol. 7, 761-776, 2000.<br>
										H. Luz and M. Vingron, Family specific rates of evolution. Bioinformatics, Advance online publication March 2006.</font><br>
									<br>

<hr/>
<font size="2" face="Helvetica, Geneva, Arial, SunSans-Regular, sans-serif"></font>
</td>
</tr>
</tbody>
</table>

</body>
</html>