
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Tutorial 2 | ECCB'12</title>
<link rel="icon" href="http://eccb12.org/eccb12/static/images/eccb12-icon.png" type="image/png"> 
<link href="eccb12/static/images/style.css" rel="stylesheet" type="text/css" />
<script type="text/javascript" src="eccb12/static/images/protoculous.js"></script>
<script type="text/javascript" src="eccb12/static/images/carousel.js"></script>
<script type="text/javascript" src="eccb12/static/images/swfobject.js"></script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-8667194-6']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();


</script>
</head>

<body>
<a name="top"></a>

<div id="header"></div>
<div id="content">
<div class="sidebar"><img src="eccb12/static/images/news_button.gif" width="140" height="33" /><br/><br/>
<h3>ECCB'12 KEYNOTES:</h3>
<p><strong><br />S&oslash;ren Brunak</strong><br />Technical University of Denmark</p>
<p>&nbsp;</p>
<p><strong>Aaron Ciechanover</strong><br /> Technion - Israel Institute of Technology</p>
<p>&nbsp;</p>
<p><strong>Barry Honig</strong></p>
<p>Columbia University, New York</p>
<p>&nbsp;</p>
<p><strong>Laurent Keller</strong></p>
<p>University of Lausanne</p>
<p>&nbsp;</p>
<p><strong>Gene Myers</strong><br />MPI of Molecular Cell Biology and Genetics, Dresden</p>
<p>&nbsp;</p>
<p><strong>Mihaela Zavolan</strong><br />Biozentrum University of Basel &amp; SIB</p>
<p>&nbsp;</p>
<h2>CONFERENCE CHAIRS:</h2>
<p>Torsten Schwede (chair)</p>
<p>Dagmar Iber (co-chair)</p>
<p>&nbsp;</p>
<div class="line"></div>
<strong>ECCB'12 is organized by:</strong><br />
<a href="sib.html"><img src="eccb12/static/images/sib_logo_small.png"/></a></br></br>
<a href="biozentrum.html"><img src="eccb12/static/images/bioz_logo_small.png"/></a></br></br>
<a href="systemsx.html"><img src="eccb12/static/images/systemsx_logo_small.png"/></a></br></br>
</div>
<br />
<div class="menue">
	
		<a class="" href="index.html">Home</a>
		
	
		<a class="" href="schedule.html">Schedule</a>
		
	
		<a class="" href="registration-info.html">Registration</a>
		
	
		<a class="" href="keynotes.html">Keynotes</a>
		
	
		<a class="" href="program.html">Program</a>
		
	
		<a class="" href="tutorials.html">Tutorials</a>
		
			
					
						<a class="submenue " href="T1.html">Tutorial 1</a>
					
			
					
						<a class="submenue active" href="T2.html">Tutorial 2</a>
					
			
					
						<a class="submenue " href="T3.html">Tutorial 3</a>
					
			
					
						<a class="submenue " href="T4.html">Tutorial 4</a>
					
			
					
						<a class="submenue " href="T5.html">Tutorial 5</a>
					
			
					
						<a class="submenue " href="T6.html">Tutorial 6</a>
					
			
		
	
		<a class="" href="workshops.html">Workshops</a>
		
	
		<a class="" href="travel.html">Travel+Venue</a>
		
	
		<a class="" href="committees.html">Committees</a>
		
	
		<a class="" href="eccb12-info.html">ECCB&#39;12 Info</a>
		
	
		<a class="" href="exhibition.html">Exhibition</a>
		
	
		<a class="" href="sponsoring.html">Sponsoring</a>
		
	
		<a class="" href="leisure.html">Leisure</a>
		
	
<div class="menue_social">
	<a href="https://www.maillist.unibas.ch/mailman/listinfo/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/email.png"/></a>
	<a href="http://www.facebook.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/facebook.png"/></a>
	<a href="http://twitter.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/twitter.png"/></a>
	<a href="https://plus.google.com/u/0/115411355857863872059" target="_blank"><img src="eccb12/static/images/social/PNG/24px/google-plus.png" /></a>
</div>
</div>
<div class="middle">
        	<h1>Tutorial 2: Protein evolution</h1>
		<p>&nbsp;</p>
<h2>Protein Evolution: From Sequence to Structure to Function</h2>
<p>&nbsp;</p>
<p><strong>Christine Orengo</strong>, UCL, London, UK</p>
<p><strong>Romain Studer</strong>, UCL, London, UK</p>
<p><strong>Nicholas Furnham</strong>, EMBL-EBI, Cambridge, UK</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<table border="0" align="left">
<tbody>
<tr>
<td><strong>Date:</strong></td>
<td>Sunday, 9 September, 2012</td>
</tr>
<tr>
<td><strong>Time:</strong></td>
<td>9:00 - 17:00</td>
</tr>
<tr>
<td><strong>Registration: &nbsp;&nbsp; </strong></td>
<td><a href="venue.html" target="_self">Congress Center Basel</a>, Messeplatz 21</td>
</tr>
<tr>
<td><strong>Venue:&nbsp;&nbsp;&nbsp;</strong></td>
<td><strong>Room "Geneva 3",</strong> Hotel Ramada, Messeplatz 12<br /><br /></td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Motivation</h1>
<p>Proteins are major components of organisms and as such are subject to natural selection, which promotes novel adaptive features or removes deleterious mutations. These adaptations, which can be to both internal and external environments, range from single-point amino acid substitutions to quite large insertions and deletions of residues and can sometimes result in the additions of significant structural embellishments to the evolutionary conserved cores of structural domains.</p>
<p>&nbsp;</p>
<p>Recent studies have presented many interesting novel views on protein evolution (ie <a href="http://dx.doi.org/10.1073/pnas.1015994108" target="_blank">T&oacute;th-Petr&oacute;czy PNAS 2011</a>, <a href="http://dx.doi.org/10.1371/journal.pone.0008591" target="_blank">Chakrabarti PloS One 2010</a>, <a href="http://dx.doi.org/10.1016/j.str.2010.08.017" target="_blank">Dessailly Structure 2010</a>) and have described various tools to estimate and understand the evolutionary forces that act on proteins. When functional divergence has occurred, positive selection can be detected at the nucleotide level. These adaptations can occur at single positions or in clusters of sites, in a co-evolutionary manner. For example, it has been observed that amino acids in the core of the protein tend to evolve if the surface evolves too, as if they are connected. At a higher lever, a large number of variations are observed between remote homologs in some superfamilies. Whilst, substrate specificity can be diverse across a superfamily, the reaction chemistry is more often maintained. In many enzyme superfamilies, the position of catalytic residues may vary despite performing equivalent functional roles in related proteins. The implications of functional diversity within superfamilies is a focus for some of the structural genomics projects will be discussed in this workshop.</p>
<p>&nbsp;</p>
<p>Understanding the mechanisms that drive protein evolution can help in understanding how the genomes are shaped by natural selection and more practically, can help in predicting the effect of mutations on proteins targeted by drugs.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Overall Goals</h1>
<p>This tutorial will present recent concepts regarding the evolution and adaptation of protein sequences and protein structures. It will be divided into three sections, in which we will present methods relating to different levels of protein evolution. Each section will have an introduction explaining the concepts underlying any analysis methods, and a discussion of the power and limitations of different methods and tools used to explore these concepts and which participants will learn how to use during the practical for that section.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Prerequisites and intended audience</h1>
<p>&nbsp;</p>
<p><strong>Intended audience:</strong> Evolutionary biologists, biochemists, computational biologists, structural biologists.</p>
<p>&nbsp;</p>
<p><strong>Possible prerequisites:</strong> Unix command line, R (for some statistical tests).</p>
<p>&nbsp;</p>
<p><strong>Level:</strong> Introductory</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Tutorial Outline</h1>
<table border="0" cellspacing="5" cellpadding="5">
<tbody>
<tr>
<td>Time</td>
<td>&nbsp;</td>
<td>Session Details</td>
</tr>
<tr>
<td>9:00</td>
<td>&nbsp;</td>
<td>
<p><strong>Introduction: Detection of evolutionary shifts in protein sequences <br />(Romain Studer &ndash; UCL)</strong><br /><br />The first part will focus on tools to detect adaptation in protein sequences. It will start with a brief introduction on multiple alignment and phylogenetic trees followed by a more detailed presentation of tools available to detect adaptation in protein sequences. A common measure to estimate evolutionary forces is to compute the dN/dS ratio, where dN represents the rate of fixation of amino acid and dS the rate of nucleotides substitution. A dN/dS &lt; 1 indicates purifying selection while a dN/dS &gt; 1 indicates positive selection, suggesting adaptation. Different codon models can estimate different dN/dS values between sites and between clades. These models have been implemented in <a href="http://abacus.gene.ucl.ac.uk/software/paml.html" target="_blank">CodeML / PAML</a> (&gt;4,000 citations, Ziheng Yang, UCL). When the estimation of the dN/dS measure is not reliable (due to nucleotide substitution saturation), other evolutionary measures can deal directly with amino acids (<a href="http://www.southampton.ac.uk/~re1u06/software/badasp/index.html" target="_blank">BADASP</a>, <a href="http://dx.doi.org/10.1093/bioinformatics/bti678" target="_blank">Edwards and Shields 2005</a>), TDG09 (<a href="http://dx.doi.org/10.1371/journal.pcbi.1000564" target="_blank">Tamuri, Dos Reis, Hay, Goldstein 2009</a>), FunDi (<a href="http://dx.doi.org/10.1093/bioinformatics/btr470" target="_blank">Gaston, Susko, Roger 2011</a>). They can detect shift in evolutionary rates (sites that evolve faster in a clade than in another clade (heterotachy, covarion-like)) and shift in biochemical properties of sites (an alanine conserved in a clade mutated in a glutamic acid in the other clade). If the number of shifts are statistically significant, we can infer functional divergence due to environmental adaptation.</p>
<p>&nbsp;</p>
<p><strong>References:</strong></p>
<p>Large-scale analysis of orthologs and paralogs under covarion-like and constant-but-different models of amino acid evolution. Studer RA, Robinson-Rechavi M. <a href="http://dx.doi.org/10.1093/molbev/msq149" target="_blank">Mol Biol Evol. 2010 Nov;27(11):2618-27. </a></p>
<p>&nbsp;</p>
<p>Pervasive positive selection on duplicated and nonduplicated vertebrate protein coding genes. Studer RA, Penel S, Duret L, Robinson-Rechavi M. <a href="http://dx.doi.org/10.1101/gr.076992.108" target="_blank">Genome Res. 2008 Sep;18(9):1393-402.</a><br /><br /></p>
</td>
</tr>
<tr>
<td>9:45</td>
<td>&nbsp;</td>
<td><strong>Practical: Detection of evolutionary shifts in protein sequences <br />(Romain Studer &ndash; UCL)</strong><br /><br />The participants will have the opportunity to apply various methods on different dataset of protein sequences (provided by the organisers or by the participants themselves). They will start from raw sequences, learning to build alignments and phylogenetic trees and finally analysing these protein families with some of the methods presented in the introduction, such as CodeML (Null model, Site-model, Branch-model, Branch-site model), BADASP (detect shift in evolutionary rate and shift in biochemical properties), or FunDi (detect sites under functional divergence).<br /><br /></td>
</tr>
<tr>
<td>10:30</td>
<td>&nbsp;</td>
<td><strong><em>Coffee break</em></strong></td>
</tr>
<tr>
<td>11:00</td>
<td>&nbsp;</td>
<td><strong>Practical: Detection of evolutionary shifts in protein sequences </strong><br />Practical continues.</td>
</tr>
<tr>
<td>12:30</td>
<td>&nbsp;</td>
<td><em><strong>Lunch</strong></em></td>
</tr>
<tr>
<td>13:30</td>
<td>&nbsp;</td>
<td>
<p><strong>Afternoon session: Evolution of functions in domain structure superfamilies<br /><br />Introduction: Exploring structural evolution and its impact on function in CATH superfamilies (Christine Orengo &ndash; UCL)</strong></p>
<p><br />The second part of the tutorial will examine structural data in the <a href="http://www.cathdb.info/" target="_blank">CATH database </a>and learn about tools used to explore the evolution of protein domains in CATH superfamilies and functional families (&ldquo;funfams&rdquo;). This will focus particularly on how structural changes across relatives in a superfamily affect the functions of the domains.</p>
<p>&nbsp;</p>
<p>CATH is database of domain superfamilies initially identified using structural data and then expanded with predicted domain structures in genome sequences. Structural similarities are identified using the CATHEDRAL algorithm (<a href="http://dx.doi.org/10.1371/journal.pcbi.0030232" target="_blank">Redfern, Orengo, 2007</a>) and homologous relationships are then confirmed using sequence based HMM-HMM approaches (eg HHpred, Hildebrand 2009). CATH is a widely used resource (currently more than 10,000 unique visitors per month and &gt;2,000 citations) containing 110,000 domain structures and nearly 16 million domain sequences classified into 2,600 domain superfamilies.</p>
<p>&nbsp;</p>
<p>Recently, each domain superfamily has been sub-classified into functional families (funfams) using an iterative profile-profile based approach (<a href="http://dx.doi.org/10.1093/nar/gkp1049" target="_blank">Rentzsch, Lee, Orengo, 2010</a>). Functional families are annotated with information from GO, EC and other functional resources and are being used to explore the evolution of function in domain superfamilies, highlighting structural changes and shifts in conserved residues which, for example, mediate changes in chemistry or substrate specificity in different functional families. CATH also provides information on the protein interactions and associations in which the domains participate and displays functional networks.</p>
<p>&nbsp;</p>
<p>Recent analyses (<a href="http://dx.doi.org/10.1016/j.str.2010.08.017" target="_blank">Dessailly, Orengo, 2010</a>) of a very highly populated CATH domain superfamily, the HUP superfamily, have shown the mechanisms by which structural changes, occurring during evolution modify, the actives sites of enzyme relatives to change chemistry and specificity and alter the surface features of relatives promoting diverse protein interactions and oligmerisations which can also affect function.</p>
<p>&nbsp;</p>
<p><em><strong>References:</strong></em></p>
<p>Gene3D: a domain-based resource for comparative genomics, functional annotation and protein network analysis. Lees J, Yeats C, Perkins J, Sillitoe I, Rentzsch R, Dessailly BH, Orengo C. <a href="http://dx.doi.org/10.1093/nar/gkr1181" target="_blank">Nucleic Acids Res. 2012 Jan;40(Database issue):D465-71. </a></p>
<p>&nbsp;</p>
<p>Extending CATH: increasing coverage of the protein structure universe and linking structure with function. Cuff AL, Sillitoe I, Lewis T, Clegg AB, Rentzsch R, Furnham N, Pellegrini-Calace M, Jones D, Thornton J, Orengo CA. <a href="http://dx.doi.org/10.1093/nar/gkq1001" target="_blank">Nucleic Acids Res. 2011 Jan;39(Database issue):D420-6. </a><strong></strong></p>
<p>&nbsp;</p>
</td>
</tr>
<tr>
<td>14:15</td>
<td>&nbsp;</td>
<td>
<p><strong>Practical: Exploring structural evolution and its impact on function in CATH superfamiles (Christine Orengo &ndash; UCL)</strong></p>
<p><br /> The participants will be guided through a CATH tutorial which exposes the user to in-house structure and sequence based comparison and prediction tools (eg CATHEDRAL, funfampred) and illustrates structural mechanisms of protein evolution in selected, functionally diverse superfamilies (eg the HUPS, which contains class I aminoacyl tRNA synthetase, USPA, ETFP, photolyase, and PP-ATPase nucleotide-binding domains). The analysis will involve the exploration of structural variation, functional diversity, species distribution, protein-protein interactions.</p>
<p><br /> We will demonstrate new functional family sub-classification in CATH and methods that predict function from structure and from sequence. We will also demonstrate methods for identifying putative interaction partners for a given query protein and for ranking target genes for a given biological system of interest. Participants will also be able to submit their own sequences or structures of and explore the structural relationships between this protein and others classified in the same CATH superfamily.</p>
</td>
</tr>
<tr>
<td>15:00</td>
<td>&nbsp;</td>
<td><strong><em>Coffee break<br /><br /></em></strong></td>
</tr>
<tr>
<td>15:30</td>
<td>&nbsp;</td>
<td>
<p><strong>Introduction: Exploring the functional evolution of structurally defined enzyme superfamilies. Nicholas Furnham (EMBL-EBI)</strong><br /><br /> <a href="https://www.ebi.ac.uk/thornton-srv/databases/FunTree/" target="_blank">FunTree </a>is a new resource, developed in the laboratory of Prof. Janet Thornton at the European Bioinformatics Institute, in collaboration with Prof. Christine Orengo at UCL. It brings together sequence, structure, phylogenetic, chemical and mechanistic information for structurally defined enzyme superfamilies. Gathering together this range of data into a single resource allows the investigation of how novel enzyme functions have evolved within a structurally defined superfamily as well as providing a means to analyse trends across many superfamilies. This is done not only within the context of an enzyme&rsquo;s sequence and structure but also the relationships of their reactions. Developed in tandem with the CATH database, it currently comprises 276 superfamilies covering approximately 1800 (70%) of sequence assigned enzyme reactions.</p>
<p>&nbsp;</p>
<p><strong>References:</strong></p>
<p>&nbsp;</p>
<p>Exploring the evolution of novel enzyme functions within structurally defined protein superfamilies. Furnham N, Sillitoe I, Holliday GL, Cuff AL, Laskowski RA, Orengo CA, Thornton JM. <a href="http://dx.doi.org/10.1371/journal.pcbi.1002403%20" target="_blank">PLoS Comput Biol. 2012 Mar;8(3):e1002403</a>.</p>
<p>&nbsp;</p>
<p>FunTree: a resource for exploring the functional evolution of structurally defined enzyme superfamilies. Furnham N, Sillitoe I, Holliday GL, Cuff AL, Rahman SA, Laskowski RA, Orengo CA, Thornton JM. <a href="http://dx.doi.org/10.1093/nar/gkr852" target="_blank">Nucleic Acids Res. 2012 Jan;40(Database issue):D776-82.</a></p>
<p>&nbsp;</p>
</td>
</tr>
<tr>
<td>16h15</td>
<td>&nbsp;</td>
<td>
<p><strong>Practical: Exploring the functional evolution of structurally defined enzyme superfamilies. Nicholas Furnham (EMBL-EBI)</strong></p>
<p><br /> The last part of the tutorial will explore the capabilities of FunTree to analyse the evolution of enzyme function within structurally defined superfamilies. We will show how the wide variety of data captured in FunTree from CATH/CATH-Gene3D, PDB, UniProtKB, ArchSchema, CSA, MACiE, ChEBI and KEGG is gathered, analysed and displayed. We will also, by example, show how common traits within a superfamily can be detected as well as how differences in function are distinguished. In addition we will illustrate how proteins (either as sequence or structure) with unknown function can be placed into its functional context to give clues to its possible function. Participants will be able to bring their own protein sequences / structures that are of interest to them to be characterised by FunTree. By the end of this session attendees will be able to independently use the resource to evaluate the evolution of enzyme function with respect to a specific superfamily or individual protein sequence or structure.</p>
</td>
</tr>
<tr>
<td>17:00</td>
<td>&nbsp;</td>
<td>End of Workshop</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Tutors</h1>
<table>
<tbody>
<tr>
<td>
<p>&nbsp;</p>
<h2>Christine Orengo</h2>
<p><img class="floatimage" src="eccb12/static/images/Christine_Orengo.png" border="0" alt="Christine Orengo" title="Christine Orengo" width="194" height="259" />Christine Orengo is a Professor of Bioinformatics in the Structural and Molecular Biology Department at University College, London. After completing a PhD in the mathematical modelling of enzyme reactions, her research interests focused on the development of algorithms for classifying protein structures and sequences into evolutionary families and for predicting protein structures and functions. Together with Professor Janet Thornton she established the <a href="http://www.cathdb.info/" target="_blank">CATH domain structure database</a> in 1994. This was later expanded, in 2002, to include predicted domain structures in completed genomes and now also provides information on protein interactions/associations and functional networks. She has presented CATH at many international conferences and has considerable experience in teaching bioinformatics algorithms at UCL and in demonstrating CATH at workshops and tutorials, including presentations at EMBO workshops on protein structure (at the EBI, UK) and comparative genomics (Brazil). She has also given workshops in the EU funded Biosapiens network for structural and functional annotation of genomes and the Bologna workshop on protein structures and functions. She is on the Board of Directors for the International Society for Computational Biology (ISCB) and is a member of the Scientific Advisory Board of the Swiss Institute of Bioinformatics (SIB). She has edited a textbook on Bioinformatics and contributed many chapters on protein structure classifications to books on protein evolution and structural bioinformatics.<br /><br /></p>
</td>
</tr>
<tr>
<td>
<p>&nbsp;</p>
<h2>Nicholas Furnham</h2>
<img class="floatimage" src="eccb12/static/images/Nicolas_Furnham.png" border="0" alt="Nicolas Furnham" title="Nicolas Furnham" width="173" height="259" />Nicholas Furnham is a post-doctoral research scientist in the laboratory of Prof. Janet Thornton at the European Bioinformatics Institute developing a new resource for understanding the evolution of novel enzyme functions in structure based protein superfamilies (<a href="https://www.ebi.ac.uk/thornton-srv/databases/FunTree/" target="_blank">FunTree</a>). This has involved bringing together large and diverse sets of data, as well as developing novel tools and algorithms to analyze and present the information. He obtained his PhD in computational biology under the supervision of Prof. Sir Tom Blundell at the University of Cambridge, where he developed novel methods for performing protein structure prediction based and methods for solving structures from crystallographic experimental data, especially at low resolution. These components formed part of the RAPPER software package, which he integrated into the CCP4 software suite used by many thousands of researches in academic and industrial laboratories worldwide He has been invited to give a number of talks at international conferences, national meetings and to industry groups. He has also been invited to teach on a number of graduate and post-graduate courses including a international PhD workshop in protein crystallography at the University of Copenhagen, the Canadian Light Source Summer School in Saskatoon, Wellcome Trust Advance courses in proteomics bioinformatics, as well as EMBO and EMBRACE courses in protein structure and function.
<p>&nbsp;</p>
</td>
</tr>
<tr>
<td>
<h2>&nbsp;</h2>
<h2>Romain Studer</h2>
<img class="floatimage" src="eccb12/static/images/Romain_Studer.png" border="0" alt="Romain Studer" title="Romain Studer" width="194" height="259" />Romain Studer is a senior post-doc research scientist in the laboratory of Prof. Christine Orengo. He obtained is PhD in the laboratory of Prof. Marc Robinson-Rechavi (Lausanne, Switzerland). His PhD work in evolutionary bioinformatics focused on protein evolution in the context of whole-genome duplication. He specialised in various methods to identify adaptation at genetic levels, such as CodeML/PAML. Using CodeML and other software, he studied the prevalence of positive selection in Vertebrates and the characteristics of shifts in evolutionary rate in Animals proteins. While he detected positive selection in many vertebrate genes, he found no difference between orthologous and paralogous genes. Based on the CodeML pipeline he built for his own research, he participated in the development of the web resource <a href="http://selectome.unil.ch/" target="_blank">Selectome</a>, a database of positive selection, which is currently supported by the HP2C initiative. During his PhD, he participated in two studies revealing the relationship between structure and evolution, both in a nuclear receptor in insects and in an MHC family in birds. Romain was assistant-lecturer for bioinformatics practicals (B.Sc/M.Sc levels) at University of Lausanne, as well as involved in the EMBnet course of phylogeny. He now continues to work on the topic of protein evolution in Vertebrates with respect to the 3D structures.</td>
</tr>
</tbody>
</table>
</div>
<div class="clear"></div>
<div class="outerfooter">
	<a href="sponsoring.html">
		<div id="footer">ECCB'12 is supported by:<br /><br />
			<div class="carousel-box" id="carousel">
				    <div class="carousel-middle" style="margin-left:auto; margin-right:auto;">
					<div class="carousel-inner">
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/1_baselarea.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/2_ibm.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_ncbic.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_convey.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/3_roche.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_hp.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_embo.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/6_bbrc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/4_interpharma.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_fmi.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/7_bioalps.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/5_cccs.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/8_novartis.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/9_syngenta.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/10_basf.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/11_merckserono.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/12_swissfoundation.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/13_iscb.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/14_unibasel.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/15_bits.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/16_swiss.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/17_bsculture.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/18_schlossbottmingen.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/19_safran.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/20_loewenzorn.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/21_myswitzerland.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/22_akos.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/23_cup.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/24_crc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/25_elixir.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/26_embl.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/28_sbvimprover.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_oxford.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/29_ion.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/99_lorilog.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_psi.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_springer.png" alt="" /></div>
					</div>
				    </div>
			</div>
			<script type="text/javascript">
			    new Carousel($('carousel').down('.carousel-middle'), $('carousel').down('.carousel-inner').select('img'), $('carousel').select('a'), {
				auto: true,
				duration: 0.5,
				frequency: 1.5,
				transition: 'sinoidal',
				visibleSlides: 1,
				circular: true
			    });
			</script>
		</div>
	</a>
</div>
<div class="link_footer">
	Looking for the <a href="http://www.eccb2012.org" target="_blank">European Congress of Conservation Biology (ECCB2012)</a>?
</div>
</body>
</html>

