
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Tutorial 4 | ECCB'12</title>
<link rel="icon" href="http://eccb12.org/eccb12/static/images/eccb12-icon.png" type="image/png"> 
<link href="eccb12/static/images/style.css" rel="stylesheet" type="text/css" />
<script type="text/javascript" src="eccb12/static/images/protoculous.js"></script>
<script type="text/javascript" src="eccb12/static/images/carousel.js"></script>
<script type="text/javascript" src="eccb12/static/images/swfobject.js"></script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-8667194-6']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();


</script>
</head>

<body>
<a name="top"></a>

<div id="header"></div>
<div id="content">
<div class="sidebar"><img src="eccb12/static/images/news_button.gif" width="140" height="33" /><br/><br/>
<h3>ECCB'12 KEYNOTES:</h3>
<p><strong><br />S&oslash;ren Brunak</strong><br />Technical University of Denmark</p>
<p>&nbsp;</p>
<p><strong>Aaron Ciechanover</strong><br /> Technion - Israel Institute of Technology</p>
<p>&nbsp;</p>
<p><strong>Barry Honig</strong></p>
<p>Columbia University, New York</p>
<p>&nbsp;</p>
<p><strong>Laurent Keller</strong></p>
<p>University of Lausanne</p>
<p>&nbsp;</p>
<p><strong>Gene Myers</strong><br />MPI of Molecular Cell Biology and Genetics, Dresden</p>
<p>&nbsp;</p>
<p><strong>Mihaela Zavolan</strong><br />Biozentrum University of Basel &amp; SIB</p>
<p>&nbsp;</p>
<h2>CONFERENCE CHAIRS:</h2>
<p>Torsten Schwede (chair)</p>
<p>Dagmar Iber (co-chair)</p>
<p>&nbsp;</p>
<div class="line"></div>
<strong>ECCB'12 is organized by:</strong><br />
<a href="sib.html"><img src="eccb12/static/images/sib_logo_small.png"/></a></br></br>
<a href="biozentrum.html"><img src="eccb12/static/images/bioz_logo_small.png"/></a></br></br>
<a href="systemsx.html"><img src="eccb12/static/images/systemsx_logo_small.png"/></a></br></br>
</div>
<br />
<div class="menue">
	
		<a class="" href="index.html">Home</a>
		
	
		<a class="" href="schedule.html">Schedule</a>
		
	
		<a class="" href="registration-info.html">Registration</a>
		
	
		<a class="" href="keynotes.html">Keynotes</a>
		
	
		<a class="" href="program.html">Program</a>
		
	
		<a class="" href="tutorials.html">Tutorials</a>
		
			
					
						<a class="submenue " href="T1.html">Tutorial 1</a>
					
			
					
						<a class="submenue " href="T2.html">Tutorial 2</a>
					
			
					
						<a class="submenue " href="T3.html">Tutorial 3</a>
					
			
					
						<a class="submenue active" href="T4.html">Tutorial 4</a>
					
			
					
						<a class="submenue " href="T5.html">Tutorial 5</a>
					
			
					
						<a class="submenue " href="T6.html">Tutorial 6</a>
					
			
		
	
		<a class="" href="workshops.html">Workshops</a>
		
	
		<a class="" href="travel.html">Travel+Venue</a>
		
	
		<a class="" href="committees.html">Committees</a>
		
	
		<a class="" href="eccb12-info.html">ECCB&#39;12 Info</a>
		
	
		<a class="" href="exhibition.html">Exhibition</a>
		
	
		<a class="" href="sponsoring.html">Sponsoring</a>
		
	
		<a class="" href="leisure.html">Leisure</a>
		
	
<div class="menue_social">
	<a href="https://www.maillist.unibas.ch/mailman/listinfo/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/email.png"/></a>
	<a href="http://www.facebook.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/facebook.png"/></a>
	<a href="http://twitter.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/twitter.png"/></a>
	<a href="https://plus.google.com/u/0/115411355857863872059" target="_blank"><img src="eccb12/static/images/social/PNG/24px/google-plus.png" /></a>
</div>
</div>
<div class="middle">
        	<h1>Tutorial 4: Genetic Diversity</h1>
		<h2>&nbsp;</h2>
<h2>Inferring Genetic Diversity from Next-generation Sequencing Data: Computational Methods and Biomedical Applications</h2>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong>Niko Beerenwinkel</strong>, ETH Zurich and SIB.<br /><strong>Karin J. Metzner</strong>, University Hospital Zurich.<br /><strong>Volker Roth</strong>, University of Basel</p>
<p style="text-align: center;"><em><br /></em></p>
<p style="text-align: center;"><em><br /></em></p>
<table border="0">
<tbody>
<tr>
<td><strong>Date:</strong></td>
<td>9 September 2012</td>
</tr>
<tr>
<td><strong>Time:</strong></td>
<td>9:00-17:00</td>
</tr>
<tr>
<td><strong>Registration:&nbsp;</strong></td>
<td><a href="venue.html" target="_self">Congress Center Basel</a>,<strong> </strong>Messeplatz 21</td>
</tr>
<tr>
<td><strong>Venue:</strong></td>
<td>Room "Helvetia 1", Swissotel Le Plaza, Messeplatz 25</td>
</tr>
</tbody>
</table>
<p><strong></strong>&nbsp;</p>
<p>&nbsp;</p>
<p style="text-align: center;">&nbsp;<em>[<a href="eccb12/static/images/ECCB12-T4-slides.pdf" target="_blank">Download tutorial slides here</a>]</em></p>
<p><strong><br /></strong></p>
<p>&nbsp;</p>
<p>Genetic diversity is a hallmark of evolution and it plays a key role in the pathogenesis and treatment of rapidly evolving cancer cells and pathogens, such as viruses and bacteria. With high-coverage next-generation sequencing (NGS), the genetic diversity of mixed samples can be probed at an unprecedented level of detail in a cost-effective manner. However, reads are erroneous and relatively short, complicating the detection of low-frequency variants and the reconstruction of long haplotype sequences. In this tutorial, we will introduce new computational biology problems associated with genetic diversity estimation from NGS data. We discuss computational and statistical approaches to their solution based on probabilistic graphical models and combinatorial optimization, and we present several real-world biomedical applications, including HIV and cancer. The tutorial addresses both, potential or actual method developers as well as users of software for inferring genetic diversity from NGS data.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Motivation</h1>
<p>Genetic diversity is abundant between species, among individuals of the same species, and also among the cells of single multicellular organisms. It plays a particularly important role in pathogen populations, where it often drives disease progression and the development of drug resistance. For example, tumors are large ensembles of genetically inhomogeneous cancer cells and the pre-existing diversity is a major factor for resistance to chemotherapy. Similarly, infectious agents, including bacteria and viruses, exist as heterogeneous populations. Prime examples are retroviruses, which include HIV, where genetic diversity is fostered by high mutation and recombination rates, resulting in so-called viral quasispecies. The diversity of viral quasispecies has been linked to disease progression, virulence, immune escape, and drug resistance.</p>
<p>&nbsp;</p>
<p>In the past, assessing the diversity of a mixed sample has been very difficult and labor-intensive, relying either on cloning individual genomes followed by Sanger sequencing or on targeted PCR- or hybridization-based technologies. With recent NGS technologies, however, heterogeneous samples can be probed at an unprecedented level of detail in a cost-effective manner. This approach is based on direct, high-coverage sequencing of the mixed sample followed by computational and statistical analysis of the resulting read data. The data analysis and interpretation step is critical, because all NGS platforms produce reads that (i) contain errors, which limits the ability to detect low-frequency variants, and (ii) are relatively short, which complicates inference of long haplotypes and the detection of co-occurring variants.</p>
<p>&nbsp;</p>
<p>Thus, computational and statistical methods for error correction and haplotype reconstruction are essential for inferring genetic diversity from NGS data. The development of such methods and their implementation in software tools is an emerging topic in bioinformatics. It is methodologically challenging and involves concepts from probabilistic graphical models, Markov chains, and combinatorial optimization. The tutorial aims at introducing these new computational biology problems and approaches to their solution to the bioinformatics community, including both developers and users. We will discuss experimental procedures, computational and statistical methods, and biomedical applications by means of selected case studies.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Overall Goals</h1>
<p>The tutorial will provide an introduction to computational methods for inferring genetic diversity from NGS data. Specifically, participants will</p>
<p style="padding-left: 30px;"><br />&bull; become acquainted with important biomedical applications of genetic diversity estimation, including the somatic evolution of cancer (intra-tumor diversity) and the evolution of viral quasispecies (intra-patient viral diversity);<br /> &bull; understand the different tasks associated with diversity estimation, including inference of low-frequency single nucleotide variants (SNVs) and haplotype reconstruction;<br /> &bull; appreciate the opportunities and limitations of different NGS technologies for diversity inference;<br /> &bull; survey existing approaches and software; and<br /> &bull; understand the basic computational and statistical principles underlying error correction and haplotype inference.</p>
<p>&nbsp;<br />To follow the tutorial, basic statistics and algorithms are required on the level covered by any bioinformatics or systems biology Master program.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Tutorial Outline</h1>
<table border="0" cellspacing="5" cellpadding="5">
<tbody>
<tr>
<td>Time</td>
<td>&nbsp;</td>
<td>Session Details</td>
</tr>
<tr>
<td>9:00</td>
<td>&nbsp;</td>
<td><strong>Introduction: Motivation, Case Studies, NGS Technology</strong><br />In the first session, we start by discussing the role of genetic diversity in several biomedical systems. Two specific case studies will be analyzed in more detail, namely the genetic diversity of tumors and the intra-patient genetic diversity of HIV. They will serve as running examples throughout the tutorial. Next, we briefly review sample preparation and the basic principles of some of the most widely used NGS platforms (454/Roche and Illumina), and compare characteristic sequencing error patterns and quality scores.</td>
</tr>
<tr>
<td>10:30</td>
<td>&nbsp;</td>
<td><strong><em>Coffee break</em></strong></td>
</tr>
<tr>
<td>11:00</td>
<td>&nbsp;</td>
<td><strong>Local Diversity Estimation</strong><br />In this session, we first review strategies for aligning reads and filtering or correcting sequencing errors, including k-mer?based indexing methods and pairwise and multiple sequence alignment. Then, we discuss methods for inferring genetic diversity at single sites, i.e., for detecting single nucleotide variants (SNVs), followed by diversity estimation across short genomic segments, i.e., local reconstruction of haplotypes. For this purpose, probabilistic clustering methods are presented that operate either on flow grams (raw 454 data) or on DNA sequences after base calling.</td>
</tr>
<tr>
<td>12:30</td>
<td>&nbsp;</td>
<td><em><strong>Lunch</strong></em></td>
</tr>
<tr>
<td>13:30</td>
<td>&nbsp;</td>
<td><strong>Global Diversity Estimation</strong><br />In this session, we introduce computational methods for diversity estimation across longer genomic segments that are not covered by individual reads, i.e., for global read clustering or, equivalently, assembly into multiple global haplotypes. Specifically, we present probabilistic graphical models (finite and infinite mixture models, hidden Markov models), read graph based approaches (relying on combinatorial optimization techniques such as minimal path cover, maximum flows, graph colorings), and de novo assembly methods.</td>
</tr>
<tr>
<td>15:00</td>
<td>&nbsp;</td>
<td><strong><em>Coffee break</em></strong></td>
</tr>
<tr>
<td>15:30</td>
<td>&nbsp;</td>
<td><strong>Comparative Assessment of Methods, Demonstration of Case Studies </strong><br />During this session, we will compare the presented methods for diversity estimation based on general properties, such as identifiability, sample size calculations, and the trade-off between read length and depth of coverage. Existing software implementations will be surveyed in the context of successful biomedical applications. Finally, we return to the two case studies and discuss estimation of tumor diversity and intra-patient diversity of HIV using the methods for SNV detection and local and global haplotype reconstruction.</td>
</tr>
<tr>
<td>17:00</td>
<td>&nbsp;</td>
<td>End of Workshop</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Tutors</h1>
<table>
<tbody>
<tr>
<td>
<p>&nbsp;</p>
<h2>Niko Beerenwinkel</h2>
<p><img class="floatimage" src="eccb12/static/images/Niko_Beerenwinkel.png" border="0" alt="Niko Beerenwinkel" title="Niko Beerenwinkel" width="194" height="259" />Niko Beerenwinkel is Assistant Professor for Computational Biology at the Department of Biosystems Science and Engineering of ETH Zurich in Basel. His research is at the interface of mathematics, statistics, and computer science with biology and medicine. His interests range from mathematical foundations of biostatistical models to clinical applications, including the genetic progression of cancer and the development of drug resistance in HIV. His honors include the Otto Hahn Medal of the Max Planck Society and the Emmy Noether Fellowship of the German National Science Foundation. Beerenwinkel has pioneered the development of computational methods for diversity estimation in RNA virus populations from next-generation sequencing data. He is a co-developer of several software packages for SNV detection, local and global haplotype inference.<br /><br /></p>
</td>
</tr>
<tr>
<td>
<p>&nbsp;</p>
<h2>Karin J. Metzner</h2>
<p><img class="floatimage" src="eccb12/static/images/Karin_J_Metzner.png" border="0" alt="Karin Metzner" title="Karin Metzner" width="194" height="259" />Karin Metzner received her MD at the Goethe University in Frankfurt and is a virologist at the University Hospital Zurich, Division of Infectious Diseases and Hospital Epidemiology. She has a broad and long-standing experience in the research field of HIV drug resistance, genetics, and molecular biology. She has pioneered detection of minority variants harboring drug resistance mutations in HIV and is using next-generation sequencing to study virus evolution in HIV-infected individuals<br /><br /></p>
</td>
</tr>
<tr>
<td>
<h2>&nbsp;</h2>
<h2>Volker Roth</h2>
<p><img class="floatimage" src="eccb12/static/images/Volker_Roth.png" border="0" alt="Volker Roth" title="Volker Roth" width="194" height="259" />Volker Roth received his diploma degree in physics from the University of Bonn in 1997 and his Ph.D. in Computer Science in 2001. From 2001 to 2003 he was a postdoctoral researcher at the Computer Vision and Pattern Recognition Group at the Institute of Computer Science in Bonn. Before joining University of Basel as Assistant Professor for Biomedical Data Analysis in 2007, he was affiliated with the Machine Learning Group at ETH Zurich, where he received the Venia Legendi (Habilitation) for the field Computational Science in 2005. Over the last years, Roth has developed machine learning methods for analyzing next-generation sequencing data. He is the main developer of the &lsquo;PredictHaplo&rsquo; software package for global haplotype reconstruction.</p>
</td>
</tr>
</tbody>
</table>
</div>
<div class="clear"></div>
<div class="outerfooter">
	<a href="sponsoring.html">
		<div id="footer">ECCB'12 is supported by:<br /><br />
			<div class="carousel-box" id="carousel">
				    <div class="carousel-middle" style="margin-left:auto; margin-right:auto;">
					<div class="carousel-inner">
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/1_baselarea.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/2_ibm.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_ncbic.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_convey.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/3_roche.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_hp.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_embo.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/6_bbrc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/4_interpharma.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_fmi.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/7_bioalps.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/5_cccs.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/8_novartis.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/9_syngenta.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/10_basf.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/11_merckserono.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/12_swissfoundation.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/13_iscb.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/14_unibasel.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/15_bits.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/16_swiss.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/17_bsculture.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/18_schlossbottmingen.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/19_safran.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/20_loewenzorn.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/21_myswitzerland.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/22_akos.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/23_cup.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/24_crc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/25_elixir.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/26_embl.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/28_sbvimprover.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_oxford.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/29_ion.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/99_lorilog.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_psi.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_springer.png" alt="" /></div>
					</div>
				    </div>
			</div>
			<script type="text/javascript">
			    new Carousel($('carousel').down('.carousel-middle'), $('carousel').down('.carousel-inner').select('img'), $('carousel').select('a'), {
				auto: true,
				duration: 0.5,
				frequency: 1.5,
				transition: 'sinoidal',
				visibleSlides: 1,
				circular: true
			    });
			</script>
		</div>
	</a>
</div>
<div class="link_footer">
	Looking for the <a href="http://www.eccb2012.org" target="_blank">European Congress of Conservation Biology (ECCB2012)</a>?
</div>
</body>
</html>

