
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Tutorial 5 | ECCB'12</title>
<link rel="icon" href="http://eccb12.org/eccb12/static/images/eccb12-icon.png" type="image/png"> 
<link href="eccb12/static/images/style.css" rel="stylesheet" type="text/css" />
<script type="text/javascript" src="eccb12/static/images/protoculous.js"></script>
<script type="text/javascript" src="eccb12/static/images/carousel.js"></script>
<script type="text/javascript" src="eccb12/static/images/swfobject.js"></script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-8667194-6']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();


</script>
</head>

<body>
<a name="top"></a>

<div id="header"></div>
<div id="content">
<div class="sidebar"><img src="eccb12/static/images/news_button.gif" width="140" height="33" /><br/><br/>
<h3>ECCB'12 KEYNOTES:</h3>
<p><strong><br />S&oslash;ren Brunak</strong><br />Technical University of Denmark</p>
<p>&nbsp;</p>
<p><strong>Aaron Ciechanover</strong><br /> Technion - Israel Institute of Technology</p>
<p>&nbsp;</p>
<p><strong>Barry Honig</strong></p>
<p>Columbia University, New York</p>
<p>&nbsp;</p>
<p><strong>Laurent Keller</strong></p>
<p>University of Lausanne</p>
<p>&nbsp;</p>
<p><strong>Gene Myers</strong><br />MPI of Molecular Cell Biology and Genetics, Dresden</p>
<p>&nbsp;</p>
<p><strong>Mihaela Zavolan</strong><br />Biozentrum University of Basel &amp; SIB</p>
<p>&nbsp;</p>
<h2>CONFERENCE CHAIRS:</h2>
<p>Torsten Schwede (chair)</p>
<p>Dagmar Iber (co-chair)</p>
<p>&nbsp;</p>
<div class="line"></div>
<strong>ECCB'12 is organized by:</strong><br />
<a href="sib.html"><img src="eccb12/static/images/sib_logo_small.png"/></a></br></br>
<a href="biozentrum.html"><img src="eccb12/static/images/bioz_logo_small.png"/></a></br></br>
<a href="systemsx.html"><img src="eccb12/static/images/systemsx_logo_small.png"/></a></br></br>
</div>
<br />
<div class="menue">
	
		<a class="" href="index.html">Home</a>
		
	
		<a class="" href="schedule.html">Schedule</a>
		
	
		<a class="" href="registration-info.html">Registration</a>
		
	
		<a class="" href="keynotes.html">Keynotes</a>
		
	
		<a class="" href="program.html">Program</a>
		
	
		<a class="" href="tutorials.html">Tutorials</a>
		
			
					
						<a class="submenue " href="T1.html">Tutorial 1</a>
					
			
					
						<a class="submenue " href="T2.html">Tutorial 2</a>
					
			
					
						<a class="submenue " href="T3.html">Tutorial 3</a>
					
			
					
						<a class="submenue " href="T4.html">Tutorial 4</a>
					
			
					
						<a class="submenue active" href="T5.html">Tutorial 5</a>
					
			
					
						<a class="submenue " href="T6.html">Tutorial 6</a>
					
			
		
	
		<a class="" href="workshops.html">Workshops</a>
		
	
		<a class="" href="travel.html">Travel+Venue</a>
		
	
		<a class="" href="committees.html">Committees</a>
		
	
		<a class="" href="eccb12-info.html">ECCB&#39;12 Info</a>
		
	
		<a class="" href="exhibition.html">Exhibition</a>
		
	
		<a class="" href="sponsoring.html">Sponsoring</a>
		
	
		<a class="" href="leisure.html">Leisure</a>
		
	
<div class="menue_social">
	<a href="https://www.maillist.unibas.ch/mailman/listinfo/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/email.png"/></a>
	<a href="http://www.facebook.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/facebook.png"/></a>
	<a href="http://twitter.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/twitter.png"/></a>
	<a href="https://plus.google.com/u/0/115411355857863872059" target="_blank"><img src="eccb12/static/images/social/PNG/24px/google-plus.png" /></a>
</div>
</div>
<div class="middle">
        	<h1>Tutorial 5: RNA-seq</h1>
		<p>&nbsp;</p>
<h2>Reads to Biological Patterns: End-to-End Differential Expression Analysis of RNA Sequencing Data Using Bioconductor</h2>
<p>&nbsp;</p>
<p><strong>Simon Anders</strong>, EMBL</p>
<p><strong>Wolfgang Huber</strong>, EMBL</p>
<p><strong>Michal Okoniewski</strong>, ETHZ</p>
<p><strong>Mark Robinson</strong>, University of Zurich</p>
<p>&nbsp;</p>
<table border="0">
<tbody>
<tr>
<td><strong>Date:</strong></td>
<td>9 September 2012</td>
</tr>
<tr>
<td><strong>Time:</strong></td>
<td>9:00-17:00</td>
</tr>
<tr>
<td><strong>Venue:&nbsp;&nbsp; <br /></strong></td>
<td>Room "<strong>Singapore</strong>",<a href="venue.html" target="_self">Congress Center Basel</a>,<strong> </strong>Messeplatz 21</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Motivation</h1>
<p>RNA sequencing (RNA-seq) is a powerful technique for characterizing RNA transcripts and comparative analyses of their abundances. It is used in many areas of biology including functional genomics, developmental biology and cancer biology. However, the data analysis can be challenging. The computational and statistical methods for analyzing RNA-seq data are recent, and best practices are not yet clearly established. In this tutorial, we will explain the state-of-the-art of RNA-seq analysis.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Overall Goals</h1>
<p>Our tutorial will consist of three main components: (i) a hands-on demonstration, walking through an end-to-end analysis of a typical RNA-seq data set from raw reads to biological interpretation, focusing on the use of packages from the Bioconductor project for quality control, counting of features, statistical analysis of differences and downstream interpretive analyses, and discussing each step in detail; (ii) basic lectures on the statistical concepts for count data and on the design and quality assessment of RNA-seq experiments, and an advanced lecture on recent developments, including adjustment for covariates ('normalization'), variance stabilizing transformation, gene set enrichment type analyses; (iii) After each lecture, time is planned in for extensive discussions, where participants can raise questions not only on the content of the lectures but also on implications for their own projects. We finish with an open panel discussion, to address any further topics and questions that participants may have encountered in their work. &nbsp;<br /><br />We aim to give attendees not just a recipe book for differential expression analysis, but a discussion on the motivation and fundamentals underlying the methods, the strengths and limitations, and insight into the latest research.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Prerequisites and intended audience</h1>
<p>Basic knowledge of sequencing technology and basic statistics is assumed, as well as a general understanding of the central dogma of molecular biology and alternative splicing.&nbsp; Attendees will benefit from prior experience using R and Bioconductor, and from some first exposure to high-throughput sequencing data.&nbsp; The intended audience includes: Computational biologists, statisticians and genome biologists with an interest in data analysis and statistical methods.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Tutorial Outline</h1>
<table border="0" cellspacing="5" cellpadding="5">
<tbody>
<tr>
<td>Time</td>
<td>&nbsp;</td>
<td>Session Details</td>
</tr>
<tr>
<td>9:00</td>
<td>&nbsp;</td>
<td><strong>Introduction (Simon Anders)</strong><br />- overview on the typical work-flow of an RNA-Seq analysis: alignment, counting, testing, downstream analysis<br />- noise in RNA-Seq data: Poisson noise and overdispersion; implications for experimental design<br />- alignment software, Bioconductor packages for counting and testing</td>
</tr>
<tr>
<td>10:00</td>
<td>&nbsp;</td>
<td>
<p><strong>Demo: A simple but complete analysis</strong></p>
<p>- aligning reads to the genome<br />- counting reads mapping to genes<br />- making quality checks<br />- performing a simple analysis with DESeq or edgeR<br />- assessing signal and noise from MA plots<br />- exporting results<strong><br /></strong></p>
</td>
</tr>
<tr>
<td>10:30</td>
<td>&nbsp;</td>
<td><strong><em>Coffee break</em></strong></td>
</tr>
<tr>
<td>11:00</td>
<td>&nbsp;</td>
<td><strong>Statistical models for count data analysis (Mark Robinson)</strong><br />&nbsp; - the negative binomial distribution<br />&nbsp; - estimation of dispersion and information sharing<br />&nbsp; - normalization considerations<br />&nbsp; - exact test and likelihood-ratio test<br />&nbsp; - generalized linear models<br />&nbsp; - implementation differences between edgeR and DESeq [with Simon]</td>
</tr>
<tr>
<td>12:00</td>
<td>&nbsp;</td>
<td><strong>Demo: A more complex analysis<br /></strong>&nbsp; - use of GLMs to treat a multi-factor design<strong></strong></td>
</tr>
<tr>
<td>12:30</td>
<td>&nbsp;</td>
<td><em><strong>Lunch</strong></em></td>
</tr>
<tr>
<td>13:30</td>
<td>&nbsp;</td>
<td><strong>Alternative isoform regulation (Simon Anders)</strong><br />&nbsp; - quantifying isoform and exon usage<br />&nbsp; - testing for differential exon usage with DEXSeq<br />&nbsp; - testing for alternative isoform expression on the gene level [with Mark]</td>
</tr>
<tr>
<td>14:15</td>
<td>&nbsp;</td>
<td><strong>Downstream analysis (Mark Robinson)&nbsp; <br />&nbsp;</strong> - normalization to account for sample-specific CG biases, etc.<br />&nbsp; - biases due to dependence of power on expression strength, transcript length, etc.<br />&nbsp; - problems caused by this when using downstream analysis methods developed for microarrays<br />&nbsp; - methods for enrichment analyses developed specifically for RNA-Seq data<br />&nbsp; - variance-stabilizing transformation of count data<strong></strong></td>
</tr>
<tr>
<td>15:00</td>
<td>&nbsp;</td>
<td><strong><em>Coffee break</em></strong></td>
</tr>
<tr>
<td>15:30</td>
<td>&nbsp;</td>
<td><strong>Demo: Downstream analysis</strong><br />- Finding gene categories enriched for differentially expressed genes</td>
</tr>
<tr>
<td>15:45</td>
<td>&nbsp;</td>
<td><strong>Nucleotide coverage profiles (Michal Okoniewski)<br /></strong>&nbsp; - coverage-based local normalizations and differential-expression measures<br />&nbsp; - a data processing pipeline for this methodology<strong></strong></td>
</tr>
<tr>
<td>16:15</td>
<td>&nbsp;</td>
<td><strong>Open panel session: Ask the "expert"</strong></td>
</tr>
<tr>
<td>17:00</td>
<td>&nbsp;</td>
<td>End of Workshop</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>Since RNA sequencing is probably the most sophisticated high-throughput technique for transcriptome analysis, there are numerous aspects that can influence the data, processing and interpretation of the outcome in a biological context.&nbsp; Our final session is an open discussion moderated by the tutorial authors.&nbsp; Participants will be invited to join and share their experience, as well as concerns, about the analysis of RNA-seq data.&nbsp; Attendees are encouraged to submit a list of questions for discussion over the course of the first sessions.&nbsp; Possible discussion items may, for example, include:<br /><br />- Experimental design, replication, advanced experimental setups<br />- Throughput of sequencers and coverage level considerations<br />- Tuning the read counting and statistics to solve specific biological problems<br />- Alternative splicing analysis<br />- Directed statistics versus unsupervised discovery in RNA sequencing<br />- Visualization of results<br />- Integration with microarrays, quantitative proteomics, new genome assemblies, functional analysis</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h1>Tutors</h1>
<table>
<tbody>
<tr>
<td>
<p>&nbsp;</p>
<h2>Simon Anders</h2>
<p><img class="floatimage" src="eccb12/static/images/Simon_Anders.png" border="0" alt="Simon Anders" title="Simon Anders" width="194" height="259" />Simon Anders is a Postdoctoral Fellow in Wolfgang Huber's group at EMBL Heidelberg and author of the widely-used DESeq package for differential expression analysis and the DEXSeq package for differential splicing analyses.&nbsp; He studied physics at the Technische Universit&auml;t M&uuml;nchen (Munich, Germany), did his PhD in Innsbruck, and then switched to bioinformatics, first at EMBL-EBI in Hincton/Cambridge and now EMBL in Heidelberg. His current work focuses on statistical methods for the analysis of high-throughput sequencing data.</p>
</td>
</tr>
<tr>
<td>
<p>&nbsp;</p>
<h2>Wolfgang Huber</h2>
<img class="floatimage" src="eccb12/static/images/Wolfgang_Huber.png" border="0" alt="Wolfgang Huber" title="Wolfgang Huber" width="163" height="259" />Wolfgang Huber is group leader and senior scientist at the European Molecular Biology Laboratory (EMBL).&nbsp; His research group develops computational and statistical methods to design and analyse novel experimental approaches in genetics and cell biology. He studied theoretical physics in Freiburg, did a postdoc in cheminformatics at IBM Research (Almaden) and a postdoc in bioinformatics at the German Cancer Research Centre (DKFZ Heidelberg). In 2004, he joined EMBL-EBI in Cambridge and in 2009, EMBL's Genome Biology unit in Heidelberg.<br /><br />
<p>&nbsp;</p>
</td>
</tr>
<tr>
<td>
<h2>&nbsp;</h2>
<h2>Michal Okoniewski</h2>
<p><img class="floatimage" src="eccb12/static/images/Michal_Okoniewski.png" border="0" alt="Michal Okoniewski" title="Michal Okoniewski" width="194" height="259" />Michal Okoniewski is a Systems Expert in Genomics and Transcriptomics at the Functional Genomics Centre Zurich, the core facility operated jointly by University of Zurich and ETH Zurich. Michal is originally a computer scientist who switched his research interests to data analysis and knowledge discovery in transcriptome data.</p>
<p>He has research experience spanning nearly 10 years, including positions at the University of Antwerp in Belgium and Cancer Research UK Paterson Institute in Manchester. Currently, he combines the work on customized analysis of over 20 next-generation sequencing projects a year with his own research on the analysis of RNA sequencing data. His input to Bioconductor includes the packages exonmap/xmapcore and rnaSeqMap.&nbsp;</p>
</td>
</tr>
<tr>
<td>
<h2>&nbsp;</h2>
<h2>Mark Robinson</h2>
<p><img class="floatimage" src="eccb12/static/images/Mark_Robinson.png" border="0" alt="Mark Robinson" title="Mark Robinson" width="194" height="259" />Mark Robinson joined the University of Zurich in 2011 as a new Assistant Professor at Institute of Molecular Life Sciences and is co-author of the widely-used edgeR package.&nbsp; He studied Applied Mathematics (BSc, Uni. Guelph) and Statistics (MSc, Uni. British Columbia) and did a PhD in statistical bioinformatics with Gordon Smyth and Terry Speed at the University of Melbourne; he has predoctoral experience at the Banting and Best Department of Medical Research (Uni. Toronto) and postdoctoral experience in the Sue Clark's Cancer Epigenomics group at the Garvan Institute in Sydney. The Robinson group at UZH develops statistical methods for interpreting high-throughput sequencing and other genomics technologies in the context of genome sequencing, gene expression and regulation and analysis of epigenomes.<br /><br /></p>
</td>
</tr>
</tbody>
</table>
</div>
<div class="clear"></div>
<div class="outerfooter">
	<a href="sponsoring.html">
		<div id="footer">ECCB'12 is supported by:<br /><br />
			<div class="carousel-box" id="carousel">
				    <div class="carousel-middle" style="margin-left:auto; margin-right:auto;">
					<div class="carousel-inner">
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/1_baselarea.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/2_ibm.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_ncbic.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_convey.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/3_roche.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_hp.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_embo.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/6_bbrc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/4_interpharma.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_fmi.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/7_bioalps.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/5_cccs.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/8_novartis.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/9_syngenta.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/10_basf.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/11_merckserono.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/12_swissfoundation.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/13_iscb.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/14_unibasel.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/15_bits.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/16_swiss.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/17_bsculture.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/18_schlossbottmingen.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/19_safran.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/20_loewenzorn.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/21_myswitzerland.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/22_akos.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/23_cup.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/24_crc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/25_elixir.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/26_embl.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/28_sbvimprover.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_oxford.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/29_ion.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/99_lorilog.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_psi.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_springer.png" alt="" /></div>
					</div>
				    </div>
			</div>
			<script type="text/javascript">
			    new Carousel($('carousel').down('.carousel-middle'), $('carousel').down('.carousel-inner').select('img'), $('carousel').select('a'), {
				auto: true,
				duration: 0.5,
				frequency: 1.5,
				transition: 'sinoidal',
				visibleSlides: 1,
				circular: true
			    });
			</script>
		</div>
	</a>
</div>
<div class="link_footer">
	Looking for the <a href="http://www.eccb2012.org" target="_blank">European Congress of Conservation Biology (ECCB2012)</a>?
</div>
</body>
</html>

