
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>TechTrack | ECCB'12</title>
<link rel="icon" href="http://eccb12.org/eccb12/static/images/eccb12-icon.png" type="image/png"> 
<link href="eccb12/static/images/style.css" rel="stylesheet" type="text/css" />
<script type="text/javascript" src="eccb12/static/images/protoculous.js"></script>
<script type="text/javascript" src="eccb12/static/images/carousel.js"></script>
<script type="text/javascript" src="eccb12/static/images/swfobject.js"></script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-8667194-6']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();


</script>
</head>

<body>
<a name="top"></a>

<div id="header"></div>
<div id="content">
<div class="sidebar"><img src="eccb12/static/images/news_button.gif" width="140" height="33" /><br/><br/>
<h3>ECCB'12 KEYNOTES:</h3>
<p><strong><br />S&oslash;ren Brunak</strong><br />Technical University of Denmark</p>
<p>&nbsp;</p>
<p><strong>Aaron Ciechanover</strong><br /> Technion - Israel Institute of Technology</p>
<p>&nbsp;</p>
<p><strong>Barry Honig</strong></p>
<p>Columbia University, New York</p>
<p>&nbsp;</p>
<p><strong>Laurent Keller</strong></p>
<p>University of Lausanne</p>
<p>&nbsp;</p>
<p><strong>Gene Myers</strong><br />MPI of Molecular Cell Biology and Genetics, Dresden</p>
<p>&nbsp;</p>
<p><strong>Mihaela Zavolan</strong><br />Biozentrum University of Basel &amp; SIB</p>
<p>&nbsp;</p>
<h2>CONFERENCE CHAIRS:</h2>
<p>Torsten Schwede (chair)</p>
<p>Dagmar Iber (co-chair)</p>
<p>&nbsp;</p>
<div class="line"></div>
<strong>ECCB'12 is organized by:</strong><br />
<a href="sib.html"><img src="eccb12/static/images/sib_logo_small.png"/></a></br></br>
<a href="biozentrum.html"><img src="eccb12/static/images/bioz_logo_small.png"/></a></br></br>
<a href="systemsx.html"><img src="eccb12/static/images/systemsx_logo_small.png"/></a></br></br>
</div>
<br />
<div class="menue">
	
		<a class="" href="index.html">Home</a>
		
	
		<a class="" href="schedule.html">Schedule</a>
		
	
		<a class="" href="registration-info.html">Registration</a>
		
	
		<a class="" href="keynotes.html">Keynotes</a>
		
	
		<a class="" href="program.html">Program</a>
		
			
					
						<a class="submenue " href="keynote-lectures.html">Keynotes</a>
					
			
					
						<a class="submenue " href="proceedings-talks.html">Proceedings</a>
					
			
					
						<a class="submenue " href="highlights.html">Highlights</a>
					
			
					
						<a class="submenue " href="posters.html">Posters</a>
					
			
					
						<a class="submenue active" href="TechTrack.html">TechTrack</a>
					
			
					
						<a class="submenue " href="art-and-science.html">Art+Science</a>
					
			
		
	
		<a class="" href="tutorials.html">Tutorials</a>
		
	
		<a class="" href="workshops.html">Workshops</a>
		
	
		<a class="" href="travel.html">Travel+Venue</a>
		
	
		<a class="" href="committees.html">Committees</a>
		
	
		<a class="" href="eccb12-info.html">ECCB&#39;12 Info</a>
		
	
		<a class="" href="exhibition.html">Exhibition</a>
		
	
		<a class="" href="sponsoring.html">Sponsoring</a>
		
	
		<a class="" href="leisure.html">Leisure</a>
		
	
<div class="menue_social">
	<a href="https://www.maillist.unibas.ch/mailman/listinfo/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/email.png"/></a>
	<a href="http://www.facebook.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/facebook.png"/></a>
	<a href="http://twitter.com/eccb12" target="_blank"><img src="eccb12/static/images/social/PNG/24px/twitter.png"/></a>
	<a href="https://plus.google.com/u/0/115411355857863872059" target="_blank"><img src="eccb12/static/images/social/PNG/24px/google-plus.png" /></a>
</div>
</div>
<div class="middle">
        	<h1>Technology and Software</h1>
		<p style="text-align: justify;"><strong><a name="top" target="_self"></a></strong></p>
<p style="text-align: justify;"><strong>T1: Verification of Systems Biology Research in the Age of Collaborative - Competition.</strong> Julia Hoeng Ph.D., Manager Computational Disease Biology, Philip Morris International, Research &amp; Development, Neuch&acirc;tel, Switzerland</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T2: Developing intelligent software</strong><span><strong> for storing, retrieving and analyzing biological data is the core of bioinformatics &ndash; Are these assets protectable?</strong> Julian W&uuml;rmser, </span>German and European Patent Attorney, Meissner Bolte &amp; Partner, M&uuml;nchen, Germany.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T3: Calling Pathogenic Variants from Exome Sequencing Data.</strong> Frank Schacherer, CTO, BIOBASE GmbH, Wolfenb&uuml;ttel, Germany.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T4: Better Computing for Better Bioinformatics.</strong> George Vacek, Director, Life Sciences, Convey Computer Corporation, Richardson, TX, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T5: Using globally integrated public expression data to investigate <em>your own</em> hypotheses.</strong> Dr. Philip Zimmermann, CEO, NEBION.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T6: The SBKB Structural Biology Knowledgebase and the Protein Model Portal: a one-stop-resource to structure information.</strong> J&uuml;rgen Haas, SIB Swiss Institute of Bioinformatics &amp; Yi-Ping Wendy Tao, RCSB PDB.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T7: Ion Torrent: Open, Accessible, Enabling.</strong> Matt Dyer, Ion Torrent Associate Director, South San Francisco, CA, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T8: ELIXIR - working towards a pan-European research infrastructure for biological information.</strong> S&oslash;ren Brunak, Technical University of Denmark, Chair of the Interim ELIXIR Board.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T9: Putting IBM Watson to Work In Healthcare.</strong> Bill Rapp, Chief Architect Watson Solutions Development, IBM Software Group, Rochester, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T10: Pathway Studio is a platform for integrating predicted and measured interactions, enabling pathway and network analysis of high-throughput molecular profiling data</strong>. Anton Yuryev, Sales Development Director, Pathway Studio, Elsevier.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>T11: Finding unusual peptides on the Internet using plain three letter sequence codes.</strong> <span>Hans-Juergen Himmler, AKos Consulting &amp; Solutions Deutschland GmbH, Steinen, Germany.</span></p>
<p style="text-align: justify;"><span><br /></span></p>
<p style="text-align: justify;"><strong>T12:</strong> <strong>IBM Watson For Healthcare - the Technology</strong> Bill Rapp, Chief Architect - Watson Solutions Development, Rochester, MN, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<hr />
<p>&nbsp;</p>
<h1>Abstracts</h1>
<p>&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T1" target="_self"></a>T1: Verification of Systems Biology Research in the Age of Collaborative - Competition.</strong> Julia Hoeng Ph.D., Manager Computational Disease Biology, Philip Morris International, Research &amp; Development, Neuch&acirc;tel, Switzerland</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Modern society demands greater scrutiny of the potential health risks and benefits of long-term, and sometimes lifelong, exposure to drugs, chemicals, and substances found in consumer products and the environment.</p>
<p style="text-align: justify;">Organizations such as companies and academic consortia conduct large multi-year scientific studies that entail the collection and analysis of thousands of data points. The individual experiments are often conducted over many physical sites and with internal and outsourced components. To extract maximum value, the interested parties need to verify the accuracy and reproducibility of automated collection and analysis workflows in systems biology before the initiation of large multi-year studies.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Traditional verification using the peer-review process has shortcomings, such as lack of scalability, which renders it insufficient for the assessment of high throughout research. A team of researchers at PMI and IBM, whose aim is to improve the effectiveness of scientific studies and verification of scientific findings propose a scheme called IMPROVER, for Industrial Methodology for Process Verification of Research. This methodology evaluates a research program by dividing its workflow into smaller building blocks, whereby the verification of each building block can be done internally or externally via challenge-based `crowd-sourcing` to a research community.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Scientific challenges will be broadcast to potential stakeholders in the form of an open call for participation with the intention of providing the community with the opportunity to test their computational methods on new data as well as to partake in a collaborative effort whose ultimate goal could contribute to solving a grand scientific problem.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Considering cancer as the leading cause of death worldwide, we formulate the Diagnostics Signature Challenge to evaluate novel approaches for the identification of robust and predictive signatures for this disease. The goal of a Diagnostics Signature Challenge is to verify that transcriptomics data contains enough information for the determination and prognosis of certain human disease states that could profit from better diagnostics signatures.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Here we will describe the approach, the necessary operational steps, and how we intend to engage the wider scientific community to assess the applicability of the IMPROVER approach to molecular diagnostics (i.e., genomic signatures).</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><em>Speaker:</em></p>
<p style="text-align: justify;"><em><br /></em></p>
<p style="text-align: justify;">Julia Hoeng Ph.D., Manager Computational Disease Biology, Philip Morris International, Research &amp; Development, Neuch&acirc;tel, Switzerland. SBV 3.0 IMPROVER Project (Industrial Methodology for Process Verification in Research) is a scientific collaboration between Philip Morris International (PMI) and IBM&rsquo;s Thomas J. Watson Research Center on a project funded by PMI</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T2" target="_self"></a>T2: Developing intelligent software</strong><span><strong> for storing, retrieving and analyzing biological data is the core of bioinformatics &ndash; Are these assets protectable?</strong> Julian W&uuml;rmser, </span>German and European Patent Attorney, Meissner Bolte &amp; Partner, M&uuml;nchen, Germany.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">It is often said that the sequencing of the human genome was an achievement in computing rather than in molecular biology. Since the development of the technology, a number of bioinformatics companies have been active in creating new algorithms and software for storing, retrieving and analyzing biological data. <br /> <br /> While the United States Patent Office is very accepting of patent applications in this field, the European Patent Office, and many national patent offices in the European Union, take a somewhat more restrictive approach. Nevertheless, with little public attention, the European Patent Office has granted numerous patents for bioinformatics inventions. Due to misinformation and legal uncertainties, European companies are disregarding available options for protecting their assets.<br /> <br /> In the session, the available means for legal protection of software based inventions in the field of bioinformatics will be presented. Risks and remedies to these risks are illustrated and discussed.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><em>Speaker:</em><br /> <br /> Julian W&uuml;rmser graduated from the Technical University of Munich with a Masters degree (Dipl.-Inf.) in Computer Science. He worked several years as a software designer focusing on medical imaging and developing algorithms in the field of structural biology before qualifying as German and European Patent Attorney. Julian W&uuml;rmser is also admitted as a European Trademark Attorney. Julian W&uuml;rmser is working at Meissner, Bolte &amp; Partner, one of Germany&rsquo;s most prominent intellectual property law firms. His work focuses on patents for software-implemented inventions, and the enforcement of respective rights.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp; [<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong>&gt;<a name="T3" target="_self"></a>T3: Calling Pathogenic Variants from Exome Sequencing Data.</strong> Frank Schacherer, CTO, BIOBASE GmbH, Wolfenb&uuml;ttel, Germany.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Next Generation Sequencing offers powerful capabilities to screen multiple candidate genes for disease causing variants in parallel. This talk will review how mutations with pathological significance can be identified in whole exome data, and will provide you with an overview on a variety of existing approaches.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">* Direct application of known genotype-phenotype associations from supporting literature evidence, such as the Human Gene Mutation Database (HGMD)</p>
<p style="text-align: justify;"><br /> * Functional assessment of novel variants by filtering with common SNPs and known disease-associated genes</p>
<p style="text-align: justify;"><br /> * Prediction of pathologic effects for missense variants, splice-site or nonsense mutations</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T4" target="_self"></a>T4: Better Computing for Better Bioinformatics.</strong> George Vacek, Director, Life Sciences, Convey Computer Corporation, Richardson, TX, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Advances in sequencing technology have significantly increased data generation, requiring similar computational advances for bioinformatics analysis. Advanced architectures based on reconfigurable computing can reduce application run times from hours to minutes, while addressing problems unapproachable with commodity servers. The increased capability also improves research quality by allowing more accurate, previously impractical approaches. This work describes the use of Convey&rsquo;s Hybrid-Core (HC) computing architecture, which combines a traditional x86 environment with a reconfigurable coprocessor, to solve a data-intensive problem of next-generation sequencing analysis such as reference mapping, de novo assembly, functional annotation, variant analysis and RNA expression profiling.</p>
<p style="text-align: justify;">&nbsp;</p>
<p>Convey has developed a personality that improves the performance of the aln step of the BWA processing pipeline, and a parallelized version of the samse and sampe processing steps, that allow Convey systems to dramatically reduce time to solution and increase throughput more than 18x for a full BWA paired-end mapping of GoNL data. Integrated BAM file generation leads to additional workflow optimization.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Graph Constructor for Velvet reduces not only run time, but also required memory, making it capable of larger assemblies. Additional performance and workflow optimizations will be discussed, including a fast kmer counting tool that allows quick identification of optimal kmer length and coverage cutoffs for de novo assembly.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">SWSearch, a search and alignment program using the Smith-Waterman algorithm, dramatically reduces the time to perform large numbers of local alignments. SWSearch on an HC-2ex server is 15x faster than the fastest software implementation on a commodity x86 system. When searching Illumina reads against a database of protein sequences SWSearch on an HC-2ex server is more than 7 times faster than NCBI BLASTx. Furthermore, BLAST uses a heuristic filter and matches only about 1/3 as many as found by the full Smith-Waterman approach. Since matches indicate a read that is part of a gene of interest, any miss could be significant.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T5" target="_self"></a>T5: Using globally integrated public expression data to investigate <em>your own</em> hypotheses.</strong> Dr. Philip Zimmermann, CEO, NEBION.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">During the past 10 years, a huge amount of expression data has been generated and made publicly available. The biological information contained in this data is largely under-explored, and most experiments have been analyzed individually. Integrating this bulk of data on a large scale provides new means of biological investigation and hypothesis testing. This presentation will show an example of public data integration and creation of a high-performance search engine - the Genevestigator platform. Currently, Genevestigator contains manually curated and globally normalized expression data from 14 organisms, including more than 170 human diseases from 29 disease areas. Scientists can use the Genevestigator online tools to check the expression of genes against thousands of experimental conditions, tissue types, cancers, and across development, or to search for genes with particular expression characteristics. In parallel, high quality aggregated datasets from the Genevestigator compendium will soon be made publicly available for bioinformaticians to serve as baseline/reference datasets and for testing novel methods and algorithms on real-life data. This presentation will discuss these resources and how they can be used to speed up your research projects.</p>
<p>&nbsp;</p>
<p><em>Speaker:</em></p>
<p><em><br /></em></p>
<p style="text-align: justify;">Dr. Philip Zimmermann studied Agronomy at ETH Zurich, with a focus on plant breeding and physiology. His Masters at Texas A&amp;M dealt with drought resistance in cotton. He then pursued a PhD in plant molecular biology at ETH Zurich. As a post-doc and senior scientist at ETH Zurich, his work focused on modeling gene regulatory networks and systems biology in mammalian and plant species. Together with his team, he developed the Genevestigator platform as an online search engine for gene expression. He currently serves as CEO at NEBION and coordinates several research projects in collaboration with academic and commercial partners.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T6" target="_self"></a>T6: The SBKB Structural Biology Knowledgebase and the Protein Model Portal: a one-stop-resource to structure information.</strong> J&uuml;rgen Haas, SIB Swiss Institute of Bioinformatics &amp; Yi-Ping Wendy Tao, RCSB PDB.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">The Protein Structure Initiative Structural Biology Knowledgebase (SBKB, http://sbkb.org) is the scientific web portal that integrates biological, experimental, and structural data about proteins. SBKB delivers unique and comprehensive information, including 3D structures from the Protein Data Bank, theoretical models available in the Protein Model Portal, target history and protocols from PSI TargetTrack, dna clones from PSI Material Repository, annotations from 140+ open biological resources, technology reports from the PSI Technology Portal, PSI articles from the PSI Publications Portal, a series on featured structures by David Goodsell and research and technical highlights from the Nature Publishing Group.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">The talk will illustrate the benefits from searching the SBKB in detail and also highlight the structural coverage within the Protein Model Portal. Visit us at <a href="http://sbkb.org" target="_blank">sbkb.org</a>!</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T7" target="_self"></a>T7: Ion Torrent: Open, Accessible, Enabling.</strong> Matt Dyer, Ion Torrent Associate Director, South San Francisco, CA, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Ion Torrent has pioneered an entirely new approach to sequencing that enables a direct connection between chemical and digital information and leverage decades of semiconductor technology advances. The result is the first commercial sequencing technology that does not use light, and as a result delivers unprecedented speed, scalability, accuracy, and low cost. In just the first year the Ion Torrent Personal Genome Machine (TM) has become the fastest selling sequencing platform. The throughput scaled 100X, from 10Mb to 1Gb, in just the first year and will scale another 100X in the next year with the new Proton (TM) sequencer, which will enable the single day $1000 human genome. Automated data analysis is driven by Torrent Suite, an open-source software suite that provides a simple and intuitive interface to streamline data analysis and provide results in minutes to hours, not days. Built on top of Torrent Suite is a flexible SDK that allows users to expand the analysis capabilities through the development and utilization of plugins and APIs.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T8" target="_self"></a>T8: ELIXIR - working towards a pan-European research infrastructure for biological information.</strong> S&oslash;ren Brunak, Technical University of Denmark, Chair of the Interim ELIXIR Board.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">The mission of ELIXIR is to build a sustainable European infrastructure for biological information supporting life science research and its translation too: medicine, the environment, the bioindustries, and society.<br /> <br /> ELIXIR will be a distributed infrastructure arranged as a Hub and Nodes, with the Hub at the European Molecular Biology Laboratory&rsquo;s European Bioinformatics Institute (EMBL-EBI) in Hinxton, UK and Nodes located throughout Europe.<br /> <br /> An Interim ELIXIR Board, consisting of&nbsp; Scientific and Administrative representatives from countries that have indicated interested in joining ELIXIR by signing a Memorandum of Understanding with ELIXIR, has been established and is currently moving ELIXIR towards construction, working with those institutes interested in hosting&nbsp; one of the it's &lsquo;Nodes&rsquo; contributing data resources; bio-computing capacity; infrastructure for data integration; and services for the research community, including training and standards development.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><em>Speakers:</em><br /> <br /> The Chair of the Interim ELIXIR Board, S&oslash;ren Brunak, of the Technical University of Denmark, will present an introduction to ELIXIR and the latest developments made towards its construction. Delegates will then be welcomed to put questions to ELIXIR representatives.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T9" target="_self"></a>T9: Putting IBM Watson to Work In Healthcare.</strong> Bill Rapp, Chief Architect Watson Solutions Development, IBM Software Group, Rochester, USA.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">The challenge in building a computer system like Watson lies in developing its ability to understand the language and intent of a question, scour millions of lines of human language, and return a single, precise answer - in less than three seconds. This discussion will outline the efforts going on inside IBM to commercialize the Watson engine for use in various industries, starting with Healthcare, and describe how Watson is being adapted to answer complex medical questions.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T10" target="_self"></a>T10: Pathway Studio is a platform for integrating predicted and measured interactions, enabling pathway and network analysis of high-throughput molecular profiling data</strong>. Anton Yuryev, Sales Development Director, Pathway Studio, Elsevier.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">Pathway Studio includes a comprehensive and accurate knowledgebase of molecular findings with a powerful analytical framework. Pathway Studio knowledgebase consists of regulatory and physical interaction relations accurately extracted from peer-reviewed scientific publications by MedScan, natural processing technology from peer reviewed scientific publications. Pathway Studio allows import of interaction data from high-throughput experiments or from predictions based on protein sequence similarity. Thus, Pathway Studio is an efficient tool for validation of new interaction data by comparison with published knowledge. It also can also be a source of interaction for the training sets of algorithms predicting molecular interaction. All interactions imported into Pathway Studio knowledgebase can be used to analyze gene expression microarray, GWAS and NGS datasets. This analysis can be used for further validation of new interactions as wells to obtain new insights into the biological mechanism using previously published molecular profiling results. Pathway Studio offers application programming interfaces that can be used for integrating novel statistical algorithms into the Pathway Studio analytical framework.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><strong><a name="T11" target="_self"></a>T11: Finding unusual peptides on the Internet using plain three letter sequence codes.</strong> <span>Hans-Juergen Himmler, AKos Consulting &amp; Solutions Deutschland GmbH, Steinen, Germany.</span></p>
<p style="text-align: justify;"><span><br /></span></p>
<p style="text-align: justify;">Finding peptides with modified Amino acids is difficult or impossible when you use plain three letter sequence codes and BLAST. You can find those peptides when you use the structure as a query, but drawing the structure correctly is rather difficult for non-chemists.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">We developed CWM Global Search with Proteax. This is an Internet search engine that allows scientists such as biologists to input plain three letter sequence codes and subsequently search the corresponding chemical structures on the Internet including Substructure searches and structure similarity searches.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">The results are mapped back to three letter sequence codes if possible. This makes the interpretation of the search results much easier than trying to interpret the structures or systematic names normally provided in the databases. Results can also be easily compared using Proteax for Spreadsheets.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">We demonstrate how to input the plain three letter sequence codes in the Proteax editor and the easy and powerful interpretation of the results performing substructure and structure similarity searches for peptides in PubChem and ChEBI.</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;"><span><br /></span></p>
<p style="text-align: justify;"><strong><a name="T12" target="_self"></a>T12:</strong> <strong>IBM Watson For Healthcare - the Technology</strong> Bill Rapp, Chief Architect - Watson Solutions Development, Rochester, MN, USA.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">This interactive technical discussion will take a look behind the scenes at how Watson works and how it is being trained to answer medical questions. We will use our experience in leveraging Watson to recommend treatment options for oncology patients and then generalize for other healthcare domains. <br /> <span style="font-family: sans-serif; font-size: x-small;"></span></p>
<p style="text-align: justify;">&nbsp;</p>
<p style="text-align: justify;">&nbsp;[<a href="TechTrack.html#top" target="_self">top</a>]</p>
</div>
<div class="clear"></div>
<div class="outerfooter">
	<a href="sponsoring.html">
		<div id="footer">ECCB'12 is supported by:<br /><br />
			<div class="carousel-box" id="carousel">
				    <div class="carousel-middle" style="margin-left:auto; margin-right:auto;">
					<div class="carousel-inner">
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/1_baselarea.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/2_ibm.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_ncbic.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_convey.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/3_roche.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_hp.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_embo.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/6_bbrc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/4_interpharma.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_fmi.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/7_bioalps.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/5_cccs.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/8_novartis.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/9_syngenta.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/10_basf.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/11_merckserono.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/12_swissfoundation.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/13_iscb.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/14_unibasel.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/15_bits.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/16_swiss.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/17_bsculture.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/18_schlossbottmingen.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/19_safran.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/20_loewenzorn.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/21_myswitzerland.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/22_akos.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/23_cup.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/24_crc.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/25_elixir.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/26_embl.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/28_sbvimprover.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_oxford.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/29_ion.png" alt="" /></div>
					    <div class="carousel-image-border-middle"><img class="carousel-images" src="eccb12/static/images/sponsors/99_lorilog.png" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/00_psi.jpg" alt="" /></div>
					    <div class="carousel-image-border"><img class="carousel-images" src="eccb12/static/images/sponsors/99_springer.png" alt="" /></div>
					</div>
				    </div>
			</div>
			<script type="text/javascript">
			    new Carousel($('carousel').down('.carousel-middle'), $('carousel').down('.carousel-inner').select('img'), $('carousel').select('a'), {
				auto: true,
				duration: 0.5,
				frequency: 1.5,
				transition: 'sinoidal',
				visibleSlides: 1,
				circular: true
			    });
			</script>
		</div>
	</a>
</div>
<div class="link_footer">
	Looking for the <a href="http://www.eccb2012.org" target="_blank">European Congress of Conservation Biology (ECCB2012)</a>?
</div>
</body>
</html>

