
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb">


<!-- Mirrored from eccb14.loria.fr/program/satellite-meetings/jpi?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:49:19 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <script type="text/javascript">
var siteurl='../../index.html';
var tmplurl='../../templates/ja_elastica/index.html';
var isRTL = false;
</script>

  <base  />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="keywords" content="ECCB'14, ECCB2014, Computational Biology, Bioinformatics, Conference, Strasbourg, ECCB, European Conference on Computational Biology, ECCB14" />
  <meta name="author" content="Adrien" />
  
  <meta name="description" content="ECCB'14 is the key European computational biology event in 2014 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <meta name="generator" content="Joomla! - Open Source Content Management" />
  <title>ECCB'14 - S02 - Junior Principal Investigator</title>
  <link href="http://eccb14.loria.fr/program/satellite-meetings/jpi?tmpl=component&amp;print=1&amp;page=" rel="canonical" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/media/jui/css/icomoon.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/." type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/addons.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/usertools.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/print.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/menu/mega.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/typo.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/modules.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile.css" type="text/css" media="only screen and (max-width:719px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile-port.css" type="text/css" media="only screen and (max-width:479px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-tablet.css" type="text/css" media="only screen and (min-width:720px) and (max-width: 985px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-normal.css" type="text/css" media="only screen and (min-width:986px) and (max-width: 1235px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-wide.css" type="text/css" media="only screen and (min-width:1236px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/menu/mega.css" type="text/css" />
  <script src="http://eccb14.loria.fr/media/system/js/mootools-core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/mootools-more.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/anchor.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/mega.rjd.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery/jquery-1.8.3.min.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.noconflict.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.masonry.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.mediaqueries.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/menu/mega.js" type="text/javascript"></script>
  <script type="text/javascript">
jQuery.noConflict();
  </script>

<!--[if ie]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/templates/ja_elastica/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 


<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=3.0, user-scalable=yes"/>
<meta name="HandheldFriendly" content="true" />


<link href="http://eccb14.loria.fr/templates/ja_elastica/images/favicon.ico" rel="shortcut icon" type="image/x-icon" /></head>

<body id="bd" class="fs3 com_content contentpane">
    
<div id="system-message-container">
<div id="system-message">
</div>
</div>
    <div class="item-page clearfix">
	<h2 class="contentheading">
			S02 - Junior Principal Investigator		</h2>

	
<div class="article-tools clearfix">
	<ul class="actions">
			<li>
		<a href="#" onclick="window.print();return false;"><span class="icon-print"></span>&#160;Print&#160;</a>		</li>
		
	</ul>



	
</div>
	<p><span>This satelite meeting has been cancelled.</span></p>
<p><span>Apologies for the inconvenience this may cause. </span></p>
<p><span><strong><span><em> </em></span></strong></span></p>
<p><span><strong><span><em>Page Last Updated: Sept. 1st, 2014</em></span></strong></span></p>
<p> </p>
<p> </p>
<p> </p>
<p> </p>
<p> </p>
<p> </p>
<p> </p>
<p>The Junior Principal Investigator (JPI) satellite meeting aims to help junior PIs to anticipate and manage the challenges of running a research group and to learn from successful examples of established young PIs. Indeed, the JPI workshop has two main goals. The first goal is to build a strong network of peers that they can draw on for advice, support and collaboration in the future. The workshop will therefore allow for focused interaction between the attendees. The second goal is to learn directly from the experience of young but well-established PIs about the way they manage their groups. This will be achieved through focused talks and a panel discussion.</p>
<p> </p>
<h2>Organization of the workshop</h2>
<p>The workshop is planned in four sections as follows bellow. Speaker invitations are ongoing. The meeting is planned for <strong>Sunday September 7th</strong>. Here is a tentative schedule (which is subject to change).</p>
<table style="width: 640px;" border="1" cellspacing="1" cellpadding="0" align="left">
<tbody>
<tr>
<td> </td>
<td>Section</td>
<td>Speakers</td>
<td>Session</td>
<td>Details of organization</td>
</tr>
<tr>
<td>
<p>08:00-09-00</p>
</td>
<td>
<p>REGISTRATION</p>
</td>
<td> </td>
<td> </td>
<td> </td>
</tr>
<tr>
<td><span style="color: #0000ff;">9:00-9:05</span></td>
<td><span style="color: #0000ff;">Introduction</span></td>
<td><span style="color: #0000ff;">Chair</span></td>
<td><span style="color: #0000ff;">Introduction</span></td>
<td><span style="color: #0000ff;"> </span></td>
</tr>
<tr>
<td><span style="color: #ff0000;">9:05-10:00</span></td>
<td><span style="color: #ff0000;"> </span></td>
<td><span style="color: #ff0000;">Attendees</span></td>
<td><span style="color: #ff0000;">Speed-dating</span></td>
<td><span style="color: #ff0000;">5min rouns with each other</span></td>
</tr>
<tr>
<td><span style="color: #008000;">10:00-10:30</span></td>
<td><span style="color: #008000;">Talk</span></td>
<td><span style="color: #008000;">Speaker 1</span></td>
<td><span style="color: #008000;">Position talk 1</span></td>
<td><span style="color: #008000;">10min talk + discussion</span></td>
</tr>
<tr>
<td><span style="color: #000000;">10:30-10:45</span></td>
<td><span style="color: #000000;"> </span></td>
<td><span style="color: #000000;"> </span></td>
<td><span style="color: #000000;">Coffee break</span></td>
<td> </td>
</tr>
<tr>
<td><span style="color: #008000;">10:45-11:10</span></td>
<td><span style="color: #008000;">Talk</span></td>
<td><span style="color: #008000;">Speaker 2</span></td>
<td><span style="color: #008000;">Position talk 2</span></td>
<td><span style="color: #008000;">10min talk + discussion</span></td>
</tr>
<tr>
<td><span style="color: #008000;">11:10-11:35</span></td>
<td><span style="color: #008000;">Talk</span></td>
<td><span style="color: #008000;">Speaker 3 </span></td>
<td><span style="color: #008000;">Position talk 3</span></td>
<td><span style="color: #008000;">10min talk + discussion</span> </td>
</tr>
<tr>
<td><span style="color: #008000;">11:35-12:00</span></td>
<td><span style="color: #008000;">Talk</span></td>
<td><span style="color: #008000;">Speaker 4</span></td>
<td><span style="color: #008000;">Position talk 4<br /></span></td>
<td><span style="color: #008000;">10min talk + discussion</span></td>
</tr>
<tr>
<td><span style="color: #000000;">12:00-1:30</span></td>
<td><span style="color: #000000;"> </span></td>
<td><span style="color: #000000;"> </span></td>
<td><span style="color: #000000;">Lunch</span></td>
<td> </td>
</tr>
<tr>
<td><span style="color: #ff0000;">1:30-3:00</span></td>
<td><span style="color: #ff0000;">Discussion</span></td>
<td><span style="color: #ff0000;">5 groups with attendees and 1 speaker<br /></span></td>
<td><span style="color: #ff0000;">Group discussion 1</span></td>
<td><span style="color: #ff0000;"><span>5 groups, topics to be collected at the registration</span></span></td>
</tr>
<tr>
<td><span style="color: #008000;">3:00-4:00</span></td>
<td><span style="color: #008000;">Talk</span></td>
<td><span style="color: #008000;">Speaker 5</span></td>
<td><span style="color: #008000;">Long talk</span></td>
<td><span style="color: #008000;">30 min talk + discussion</span></td>
</tr>
<tr>
<td><span style="color: #000000;">4:00-4:20</span></td>
<td><span style="color: #000000;"> </span></td>
<td><span style="color: #000000;"> </span></td>
<td><span style="color: #000000;">Coffee break</span></td>
<td> </td>
</tr>
<tr>
<td><span style="color: #ff0000;">4:20-5:50</span></td>
<td><span style="color: #ff0000;">Discussion</span></td>
<td><span style="color: #ff0000;">5 groups with attendees and 1 speaker</span></td>
<td><span style="color: #ff0000;">Group discussion 2</span></td>
<td><span style="color: #ff0000;">5 groups, topics to be collected at the registration</span></td>
</tr>
<tr>
<td><span style="color: #0000ff;">5:50-6:00</span></td>
<td><span style="color: #0000ff;"> </span></td>
<td><span style="color: #0000ff;">Chair</span></td>
<td><span style="color: #0000ff;">Conclusions</span></td>
<td><span style="color: #0000ff;">Meeting conclusion and wrap-up</span></td>
</tr>
</tbody>
</table>
<p> </p>
<p> </p>
<p> </p>
<p> </p>
<p> </p>
<p>Seats at 9am are assigned randomly. At every break (<span style="color: #000000;">in black</span>), attendees will by shuffled by changing the seats randomly with the aim to provide more opportunities for face-to-face communication.</p>
<p>As highlighted<span style="color: #ff0000;"> in red</span>, the workshop aims to promote interactions. It will start off with a networking opportunity through speed-dating introductions. Then, small discussion groups are dispersed during the day. Finally, the meeting is concluded through an open panel discussion, in which the questions and issues raised by the discussion groups are presented and discussed by the speakers. A final wrap-up by the workshop chair then summarizes and closes the event.</p>
<p>By mixing a few short position talks with an interactive format that is aimed at providing excellent networking and experience-sharing opportunities, the workshop will thus deliver an exciting platform for young researchers to learn how to successfully run a group.</p>
<p> </p>
<p> </p>
<h2>Speakers</h2>
<h3><span>Sampsa Hautaniemi<br /></span></h3>
<ul>
<li>M.Sc. (engineering), Department of Automation, Tampere University of Technology, Finland, 1995-2000</li>
<li>D.Tech, Department of Computer Science, Tampere University of Technology, Finland, 2001-2003</li>
<li>Visiting scientist, National Human Genome Research Institute, National Institutes of Health, USA 2001-2002</li>
<li>Post-doctoral researcher, Department of Biological Engineering, Massachusetts Institute of Technology, USA, 2004-2006</li>
<li>Principal Investigator of the Systems Biology group, Faculty of Medicine, University of Helsinki, Finland, since 2006</li>
<li>Professor of Systems Biology, Faculty of Medicine, University of Helsinki, Finland, since 2013</li>
<li><a href="http://research.med.helsinki.fi/gsb/hautaniemi/">Group website</a></li>
</ul>
<p class="Normal"><span style="text-decoration: underline;">Research Interests</span></p>
<p class="Normal"><span class="Normal__Char">The research in the Systems Biology group concentrates on two topics. Firstly, in order to efficiently analyze large-scale and multilevel molecular data, we have developed a computational ecosystem called Anduril. Our approach provides a modular and open source workflow framework for data analysis, and it is the backbone that enables efficient analysis and interpretation of biomedical data. Secondly, we actively develop and apply approaches to integrate genetics, transcriptomics, proteomics, epigenetics and clinical data to gain understanding of cancer progression and resistance to anti-cancer treatments. Currently, we focus on lymphoma, ovarian cancer and breast cancer.</span></p>
<h3><span><span>Lennart Martens</span></span></h3>
<ul>
<li>Software Engineer and Framework Architect, Sydney-Tristar Development Company, Belgium, 2000-2002</li>
<li>Ph.D. in Sciences: Biotechnology in 2006, Ghent University, Belgium </li>
<li>Senior Software Developer and Group Coordinator, PRIDE team, EMBL-EBI, Cambridge, UK, 2006-2009</li>
<li>Professor of Systems Biology at Ghent University, Belgium, since 2009</li>
<li>Group Leader of the Computational Omics and Systems Biology group (CompOmics), VIB, Belgium, since 2009</li>
<li>Member of the Young Academy of Belgium, since 2013</li>
<li><a href="http://compomics.com/">Group website</a></li>
</ul>
<p class="Normal"><span style="text-decoration: underline;"><span class="Normal__Char">Research Interests</span></span></p>
<p class="Normal"><span class="Normal__Char">The CompOmics group is specialized in the management, storage, analysis and interpretation of large-scale omics data. The group has developed a large collection of algorithms and free and open end-user applications for these tasks, particularly in the field of proteomics. Recently, the group also published its extensive hands-on tutorials for proteomics informatics, illustrating the key goal of empowering end-users through education and production-grade, state-of-the-art software tools. One of the main areas of interest is the (orthogonal) reuse of publicly available omics data in meta-analyses that span thousands of individual experiments.</span></p>
<h3>Oliver Kohlbacher</h3>
<ul>
<li>Ph.D. in Computer Science 2001, Saarland University </li>
<li>Group Leader 2000-03, Center for Bioinformatics, Saarbrücken</li>
<li>Post-doctoral training 2001-2002, Celera Genomics, Rockville, Maryland  </li>
<li>University of Tübingen: Professor of Applied Bioinformatics since 2003; Director, Quantitative Biology Center since 2012</li>
<li><a href="http://abi.inf.uni-tuebingen.de/">Group website</a></li>
</ul>
<p class="Normal"><span style="text-decoration: underline;"><span class="Normal__Char">Research Interests</span></span></p>
<p class="Normal"><span>The research of our group is focused on method development in computational biology and their application in systems biology, drug design, and immunology. Recently we have focussed on the development of methods for the analysis of high-throughput metabolomics and proteomics data, vaccines design, and virtual high-throughput screening. The group is also known for its open-source software development efforts, most notably OpenMS (computational mass spectrometry), BALL (structural bioinformatics), and BN++/BiNA (systems biology).</span></p>
<h3>Sebastian Schultheiss</h3>
<ul>
<li><span>2000-2007 Bioinformatics undergraduate training at University of Tübingen and University of Michigan</span></li>
<li><span>2001-2006 Freelance work at IT start-up company Intra2net AG, Tübingen</span></li>
<li><span>2007-2011 PhD in Bioinformatics at Max Planck Institute for Developmental Biology and Friedrich Miescher Laboratory of the Max Planck Society, working with plant NGS data</span> </li>
<li><span>Since 2012 Co-founder and Managing Director of Computomics, a bioinformatics service company for crop research</span></li>
<li><a href="http://www.computomics.com/">Group website</a></li>
</ul>
<p class="Normal"><span style="text-decoration: underline;"><span class="Normal__Char">Current focus</span></span></p>
<p class="Normal"><span>Computomics is a team of world-leading experts in plant research and bioinformatics, offering turnkey next-generation sequencing (NGS) analyses for agriculture biotech companies and crop scientists. Computomics helps navigate the complexities of cereal, vegetable, fruit, and other genomes, starting from the experimental design and leading up to complex interpretations. Our turnkey bioinformatics services provide our clients with NGS data interpretation using state-of-the-art genomics tools and technologies.</span></p>
<p class="Normal"><span> </span></p>
<h3>Alice C. McHardy </h3>
<ul>
<li>1995 – 2000: Studied biochemistry, Department of Chemistry, Bielefeld University, Germany</li>
<li>2000 : <em>Diplom</em> degree in biochemistry</li>
<li>2004 : Doctoral dissertation: Gene finding and the evaluation of synonymous codon usage features in microbial genomes</li>
<li>2005-2007 : Postdoc / Research Staff, IBM Research, Yorktown Heights, New York, USA</li>
<li>2007-2010 : Independent Max-Planck-Research Group Leader at the Max-Planck Institute for Informatics</li>
<li>since 2010 : Chair of Algorithmic Bioinformatics at Heinrich-Heine University Duesseldorf</li>
<li><a href="http://www.cs.hhu.de/en/research-groups/algorithmic-bioinformatics/home-chair.html">Group website</a></li>
</ul>
<p class="Normal"><span>Current focus</span></p>
<p class="Normal"><span><span>The research of the group focuses on the data-driven analysis of biological questions, as well as method development to solve prediction problems for large biological data sets. To address problems of either medical or biotechnological relevance we are using pattern recognition techniques and phylodynamic methods. An example  is the development of phylodynamic techniques, which combine phylogenetic and epidemiological information to infer different aspects of the evolutionary dynamics of rapidly evolving populations. We apply these techniques to analyze genomic data of microbial communities (also known as metagenomic data), of influenza viruses and of cancer cells.</span></span></p>
<p class="Normal"><span> </span></p>
<h2>Sponsors &amp; fundings</h2>
<p class="Normal"><span> </span></p>
<p class="Normal"><span> <img src="http://eccb14.loria.fr/images/GDR-BIM.jpg" alt="" /></span></p>
<p class="Normal"><span> </span></p>
<p class="Normal"><span><img src="http://eccb14.loria.fr/images/Banner-for-ECCB.GIF" alt="" /></span></p>
<h2> </h2>
<h2>Organizers</h2>
<ul>
<li><span style="line-height: 1.3em;"><a href="https://www.linkedin.com/profile/view?id=4884359">Virginie Bernard</a>, Curie Institute, Paris, France</span></li>
<li><span style="line-height: 1.3em;"><a href="https://www.linkedin.com/in/magalimichaut">Magali Michaut</a>, Netherlands Cancer Institute, Amsterdam, the Netherlands</span></li>
<li><span style="line-height: 1.3em;"><a href="https://www.linkedin.com/profile/view?id=17688495">Lennart Martens</a>, Ghent University and VIB, Belgium</span></li>
<li><span style="line-height: 1.3em;"><a href="https://www.linkedin.com/profile/view?id=267691192">Theodore Alexandrov</a>, University of Bremen, Germany; University of California San Diego, USA</span></li>
</ul>
<p><span style="line-height: 1.3em;"> </span></p>
		

	</div>
</body>


<!-- Mirrored from eccb14.loria.fr/program/satellite-meetings/jpi?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:49:19 GMT -->
</html>