<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD>
<TITLE>ECCB2002 Invited Speakers</TITLE>
<meta name="author" content="Andreas Moll, Marc Sturm">
<meta content="text/html; charset=windows-1252">
<link rel="stylesheet" type="text/css" href="../style.css">
</HEAD>
<body bgcolor="White" text="Black" link="Blue" vlink="Navy" alink="#000088">
<br><br>
<table align="center" width="88%" cellspacing="2" cellpadding="0" border="0">
<tr>
    	<td colspan=2><b>Biological information: making it accessible and integrated (and trying to make sense of it). </b></td></tr>
<tr>
	<td colspan=2><A HREF="javascript:if(confirm('http://predict.sanger.ac.uk/th/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://predict.sanger.ac.uk/th/'">Tim Hubbard</A>, Sanger Institute, Cambridgeshire, United Kingdom</td></tr>
<tr>
	<td width=40></td>
    	<td>The availability of the genome sequences of human and mouse, human 
sequence variation data and other large genetic data sets will lead 
to a revolution in understanding of the human machine and the 
treatment of its diseases.  The success of the international genome 
sequencing consortiums shows what can be achieved by well coordinated 
large scale public domain projects and the benefits of data access to 
all.  It is already clear that the availability of this sequence is 
having a huge impact on research worldwide.
Complete genome sequences provide a framework to pull all biological 
data together such that each piece has the potential to say something 
about biology as a whole.  Biology is too complex for any 
organisation to have a monopoly of ideas or data, so the collection, 
analysis and access to this data can be contributed to by research 
institutes around the world.  However, although it is possible for 
all this data to be accessible to all through the internet, the more 
organisations provide data or analysis separately, the harder it 
becomes for anyone to collect and integrate the results.
To address these problems of intergration of data, open standards for 
biological data exchange, such as the "Distributed Annotation System" 
(DAS) are being developed and bioinformatics as a whole is now being 
strongly driven by the open source software (OSS) model for 
collaborative software development (1).  The leading provider of 
human genome annotation, the Ensembl project (www.ensembl.org), is 
entirely an OSS project and has been widely adopted by academic and 
commerical organisations alike (2).
Accurate automatic annotation of features such as genes in vertebrate 
genomes currently relies on supporting evidence in the form of 
homologies to mRNAs, ESTs or protein.  However, it appears that 
sufficient high quality experimentally curated annotation now exists 
to be used as a substrate for machine learning algorithms to create 
effective models of biological signal sequences (3).  Is there hope 
for ab initio prediction methods after all?<br><br>
1.		 Hubbard & Birney, Open annotation offers a democratic 
solution to genome sequencing (1999) Nature, 403, 825.<br>
2.		 Hubbard, T., et al., The Ensembl genome database 
project 
(2002). NAR, 30, 38-41.<br>
3.		 Down, T.A. and Hubbard, T.J., Computational detection 
and 
location of transcription start sites in mammalian genomic DNA. 
Genome Research (2002), 12, 458-61.<br>
</td></tr>
</table>  
</BODY></HTML>
