<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD>
<TITLE>ECCB2002 Invited Speakers</TITLE>
<meta name="author" content="Andreas Moll, Marc Sturm">
<meta content="text/html; charset=windows-1252">
<link rel="stylesheet" type="text/css" href="../style.css">
</HEAD>
<body bgcolor="White" text="Black" link="Blue" vlink="Navy" alink="#000088">
<br><br>
<table align="center" width="88%" cellspacing="2" cellpadding="0" border="0">
<tr>
    	<td colspan=2><b>Selecting Targets for Therapeutic Validation Through Differential Protein Expression using Chromatography-Mass Spectrometry </b></td></tr>
<tr>
	<td colspan=2><A HREF="javascript:if(confirm('http://www.celera.com/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.celera.com/'">Scott Patterson</A> ,Celera, Rockville, United States</td></tr>
<tr>
	<td width=40></td>
    	<td>
The identification of potential targets for therapeutic intervention 
can be accomplished on a systematic basis by a variety of techniques 
that include quantitative analysis of gene-specific mRNA levels and 
expressed proteins in normal and diseased cells. Differences in the 
expression levels of nucleic acid and protein gene products could 
suggest protein drug targets that are directly causative of disease, 
or reveal biochemical pathways that could be modulated by therapeutic 
molecules. Any effort based on mRNA or protein expression level 
comparisons could be confounded by a number of factors: 
<ul>
<li>
level in steady-state may not be correlated with actual encoded 
protein levels;
<li> differentially expressed protein levels might 
be a result of disease process, and not causative of the process, 
and therapeutic intervention based on such a difference will be 
unproductive and
<li> the differential expression of mRNA or protein 
may be the result of biological variation unrelated to the disease 
process under study. 
</ul>
In order to address these possibly confounding 
factors, it is necessary to validate potential targets by establishing 
their firm association with disease, and their minimal distribution 
in non-diseased tissues of any type. This requirement suggests that 
emphasis on true and reproducible quantitation of protein expression 
levels in a variety of samples will be an effective and highly efficient 
method of generating drug targets with a high degree of utility. To 
achieve this aim, we have established an industrial-scale proteomics-based 
discovery platform consisting of cell biology, protein chemistry, 
and mass spectrometry technical groups together with bioinformatics 
groups. The analytical method used for quantitation employs isotope 
labeling for differential analysis (ICAT<sup>TM</sup>, Applied Biosystems, Inc.). 
With this technique, tryptic peptides are generated from labeled proteins 
that have been specifically captured from various subcellular locations 
or protein families. The resulting peptides are identified and 
quantified by mass spectrometry. To evaluate this approach on a 
large-scale, we have applied it to a study of continuous cell lines 
derived from human pancreatic adenocarcinomas. We have been able to 
establish processes for target discovery for small molecule drug 
targets as well as therapeutic antibody target identification for 
cell surface proteins. In addition, we have developed a process for 
identification of serum markers of this disease based upon standardized 
fractionation procedures. The results of these analyses will be presented 
together with the some of the issues from both the wet and dry (computational) 
lab that need to be addressed in such an undertaking.</td></tr>
</table>  
</BODY></HTML>
