<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="content-type" content="
   text/html; charset=ISO-8859-1" />
<title>ECCB08</title>
<script src="themes/default/script/equalcolumns.js" type="text/javascript"><!--
//--></script>
<script src="menu/menu.js" type="text/javascript"><!--
//--></script>
<script type="text/javascript">
<!--
  function showMenu(sectionId) {
  //  hideMenu();
    id = parseInt(sectionId2pos[sectionId]);
    if (id>0 && id <13) 
      document.getElementById('item'+id).className = 'item'+id+'sel';
     if (typeof(startList) != "undefined") startList();
  }
  function hideMenu() {
    for (var i=1;i<13;i++)
document.getElementById('item'+i).className = 'item'+i;
  }
//--></script>
<style type="text/css" media="all">
	@IMPORT url("themes/default/stili/stile-v3.css");
	@IMPORT url("themes/default/stili/menu.css");
</style>
<!--[if lte IE 6]>
<style type="text/css" media="all">
@IMPORT url("themes/default/stili/ie6stile.css");
</style>
<script src="themes/default/script/drop_down.js" type="text/javascript"></script>
<![endif]-->
<!--[if lte IE 7]>
<style type="text/css" media="all">
@IMPORT url("themes/default/stili/iestile.css");
</style>
<![endif]-->
</head>
   
<body onload="showMenu('245')">
<div id="document">
<div id="header" onClick="document.location='index.php'">
<h1><a href="index.php_pageId=1.html" title="Link to Home Page"><span>European Conference on Computational Biology - September 22nd - September 26th - Cagliari, Sardinia - Italy</span></a></h1>
<div class="barra"></div>
<h2 class="bits"><a href="javascript:if(confirm('http://www.bioinformatics.it  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.bioinformatics.it'"><span>Bits Meeting</span></a></h2>
</div>
<div id="leftcolumn">
<div id="menu">
<ul id="nav">
<li id="item1" class="item1"><a href="index_5.html"><span>ECCB'08 Info</span></a>
<ul>
<li><a href="index_0.html">ECCB Conference Format</a></li>
<li><a href="index_25.html">Location</a></li>
<li><a href="index_15.html">Key Dates</a></li>
<li><a href="index_9.html">FAQ</a></li>
<li><a href="index_17.html">Social Event</a></li>
<li><a href="index_20.html">Newsletter subscription</a></li>
<li><a href="index_21.html">ECCB08 Posters</a></li>
<li><a href="index_26.html">Satellite meetings</a></li>
<li><a href="index_28.html">Pictures</a></li>
</ul>
</li>
<li id="item2" class="item2"><a href="index_1.html"><span>Program</span></a>
<ul>
<li><a href="index_30.html">Workshops</a></li>
<li><a href="index_23.html">Tutorials</a></li>
<li><a href="index_29.html">Papers</a></li>
<li><a href="index_31.html">Posters</a></li>
</ul>
</li>
<li id="item3" class="item3"><a href="index_2.html"><span>Schedule</span></a>
</li>
<li id="item4" class="item4"><a href="index_7.html"><span>Registration</span></a>
<ul>
<li><a href="index_27.html">Cancellation Policies</a></li>
</ul>
</li>
<li id="item5" class="item5"><a href="index_12.html"><span>Guidelines</span></a>
<ul>
<li><a href="index_14.html">Papers</a></li>
<li><a href="index_13.html">Posters</a></li>
<li><a href="index_16.html">Fellowships</a></li>
<li><a href="index_19.html">Workshops</a></li>
<li><a href="index_18.html">Tutorials</a></li>
</ul>
</li>
<li id="item6" class="item6"><a href="index_4.html"><span>Submissions</span></a>
</li>
<li id="item7" class="item7"><a href="index_6.html"><span>Conference Sponsors</span></a>
</li>
<li id="item8" class="item8"><a href="index_3.html"><span>Travel</span></a>
<ul>
<li><a href="index_22.html">Accommodation</a></li>
</ul>
</li>
<li id="item9" class="item9"><a href="index_11.html"><span>Sponsoring ECCB'08</span></a>
</li>
<li id="item10" class="item10"><a href="index_8.html"><span>Committees</span></a>
</li>
<li id="item11" class="item11"><a href="index_10.html"><span>About ECCB</span></a>
</li>
<li id="item12" class="item12"><a href="index_32.html"><span>Contact us</span></a>
</li>
</ul>
</div>
</div>
<div id="content">
<div class="topbg">
<h2 class="head"><span>Biology by other means</span></h2>
<div class="section">
<h3 class="title"><span>Tutorials</span></h3>
<div>
<h2><a name="top"></a>ECCB08 Tutorials&nbsp;  <a accesskey="1" onclick="javascript:Clickheretoprint();return false;" href="#"><img width="53" height="19" border="0" align="absmiddle" src="upload/Image/button_printable.gif" alt="printable version" /></a>  <br />
</h2>
ECCB08 features an introductory to advanced tutorials session. The tutorials will be given on <strong>Monday, September 22, 2008</strong> at the conference site.<br />
The minimum number of participants for each tutorial is 15.</div>
<div>
<br />
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;"> <strong><a href="#TUT1">- </a><a href="#TUT1">From sequence to structure and function: Structural modeling of proteins and protein-DNA complexes by </a><br />
&nbsp; <a href="#TUT1">comparative methods</a></strong><strong><a href="#TUT1"> <em><br />
(Jan Kosinski)</em></a><br />
</strong> </fieldset><br />
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;"> <a href="#TUT2"><strong>- Microarray data analysis and quality assessment, ArrayExpress and BioConductor<em> <br />
(</em><em>Audrey Kauffmann</em>, <em>Gabriella Rustici)<br />
</em></strong></a> </fieldset><br />
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;"> <a href="#TUT3"><strong>- Exploration and analysis of Protein-Protein Interactions (PPIs) <em><br />
(Javier De Las Rivas)<br />
</em></strong></a> </fieldset><br />
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;"> <a href="#TUT4"><strong>- Interoperability of bioinformatics software and databases <em><br />
(Gert Vriend)</em></strong></a><br />
</fieldset> <br />
<h3><a name="TUT1"></a>&nbsp; From sequence to structure and function: <br />
</h3>
<h3>&nbsp;Structural modeling of proteins and protein-DNA complexes by comparative methods</h3>
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;">
<ul>
    <li><strong>Synopsis:</strong><br />
    Structural modeling of biological macromolecules and their complexes is a powerful tool in understanding their function and mechanism of action.<br />
    This tutorial will provide detailed and exhaustive instructions for constructing comparative structural models of proteins and protein-DNA complexes. The state-of-the-art modeling methodology will be presented step-by-step using exemplary &ldquo;real life&rdquo; problem - modeling of BcnI restriction nuclease in complex with its cognate DNA. Special emphasis will be put on how the sequence analysis of the whole protein family can be combined with structure prediction methods for building accurate 3D models. <br />
    The tutorial materials will include all instructions and files necessary for exercising all presented steps by the attendees after the course as well as applying the methodology in their own research.<br />
    <br />
    </li>
    <li><strong>Intended audience: </strong><br />
    The tutorial is especially targeted to the audience with general knowledge about protein and nucleic acid structure (and some experience in using structure viewers and alignment editors), but little or no experience in protein structure modeling. <br />
    Basic understanding of sequence similarity and database searches will be also advantageous, although not critical. We invite particularly those researchers who would like to apply structural modeling of proteins and protein-DNA complexes in their own work. Thus, the tutorial is most appropriate for postgraduate students, postdocs and other researchers with background in structural biology or with basic training in structural bioinformatics.     </li>
    <br />
    <li><strong>Web site:&nbsp;</strong><font color="#0000ff"><a href="javascript:if(confirm('http://genesilico.pl/ECCB2008_tutorial  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://genesilico.pl/ECCB2008_tutorial'">http://genesilico.pl/ECCB2008_tutorial</a></font><a href="javascript:if(confirm('https://www.ebi.ac.uk/Rebholz-srv/aimm.html  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/Rebholz-srv/aimm.html'">                	 	 	       <br />
    <br />
    </a>   </li>
    <li><strong>Organizer:</strong> <br />
    <p align="justify" style="margin-bottom: 0cm;">Jan Kosinski, MSc<br />
    <br />
    Laboratory of Bioinformatics and Protein Engineering,<br />
    International Institute of Molecular and Cell Biology,<br />
    Trojdena 4, 02-109 Warsaw, Poland<br />
    <br />
    home page: <font color="#0000ff"><a href="javascript:if(confirm('http://genesilico.pl/JanKosinski  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://genesilico.pl/JanKosinski'">http://genesilico.pl/JanKosinski</a></font><br />
    e-mail: <a href="mailto:kosa@genesilico.pl?subject=ECCB08%20Tutorial">kosa@genesilico.pl</a>      </p>
    <p align="justify" style="margin-bottom: 0cm;"><br />
    </p>
    </li>
    <li><strong>Fee: </strong>100 &euro;<br />
    </li>
</ul>
</fieldset>
<p><a href="#top">top</a></p>
<h3><a name="TUT2"></a>&nbsp;Microarray data analysis and quality assessment, ArrayExpress and BioConductor </h3>
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;">
<ul>
    <li><strong>Synopsis:<br />
    </strong>This tutorial aims to instruct the attendees on how to retrieve biologically meaningful information from microarray data using EBI resources and newly developed BioConductor packages. Participants will learn how to browse and retrieve data from the ArrayExpress public repository of microarray data. They will then learn to assess microarray data quality using the arrayQualityMetrics BioConductor package and will perform some fundamental steps in the analysis of gene expression data (normalization, identification of differentially expressed genes, clustering,...) using Expression Profiler and BioConductor. Training material and an extensive bibliography will be provided to the participants of this tutorial.<br />
    <br />
    </li>
    <li><strong>Intended audience: </strong><br />
    This tutorial is aimed at biologists working with expression data and will be of use to those starting to do microarray analysis as well as users experienced with this technique. At each step, an attempt will be made to describe the relevant biological and statistical assumptions made. No particular background is required although some preliminary basic knowledge of statistics (t-test, multiple testing correction,...) and of the basic steps involved in microarray data analysis ( data normalization, clustering, ...) would be helpful. Familiarity with R will be beneficial but is not mandatory. <br />
    <br />
    </li>
    <li><strong>Organizers:</strong> <br />
    <p style="margin-bottom: 0cm;">Audrey Kauffmann<br />
    EMBL-EBI<br />
    Wellcome Trust Genome Campus<br />
    Hinxton CB10<br />
    1SD<br />
    UK<br />
    <br />
    email: <font color="#000080"><a href="mailto:audrey@ebi.ac.uk?subject=ECCB08%20Tutorial">audrey@ebi.ac.uk</a></font><br />
    <br />
    Gabriella Rustici<br />
    EMBL-EBI<br />
    Wellcome Trust Genome Campus<br />
    Hinxton CB10<br />
    1SD<br />
    UK<br />
    email: <font color="#000080"><a href="mailto:gabry@ebi.ac.uk?subject=ECCB08%20Tutorial">gabry@ebi.ac.uk<br />
    <br />
    </a></font></p>
    </li>
    <li>  <strong>Fee: </strong>100 &euro;<br />
    </li>
</ul>
</fieldset>
<p><a href="#top">top</a></p>
<h3><a name="TUT3"></a>&nbsp; Exploration and analysis of Protein-Protein Interactions (PPIs)&nbsp;</h3>
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;">
<ul>
    <li><strong>Synopsis:<br />
    </strong>                	 	 	The advancement of genome and proteome-wide experimental technologies have introduced modern biology in the high complexity of living cells, where thousands of biomolecules work together in an interactive way. The compendium of all protein-protein interactions (PPIs) present in a given cell is called the &quot;interactome&quot; network.<br />
    This TUTORIAL will present the current state of the research on the interactomes and will show a selected set of bioinformatic tools and strategies useful to explore and analyse PPIs. A short outline below shows up the flow of the tutorial: <br />
    1- Introduction to protein-protein interactions and interactomes: definition, types, experimental methods and major papers. <br />
    2- Protein-protein interactions main databases and public repositories: data comparison, information exchange and integration. <br />
    3- Query, find and explore interactions for a given protein: bioinformatic tools available and useful. <br />
    4- Quality control and quality assessment of protein-protein interactions: score schemes. <br />
    5- Visualize and analyze protein interaction networks with functional and topological parameters.<br />
    The focus and examples within the tutorial will correspond to human proteins, although the tools and strategies presented will be applicable to proteins from many other species.<br />
    <br />
    </li>
    <li><strong>Intended audience: </strong><br />
    The tutorial is aimed to both biologist and bioinformaticians. For biologist: to learn how to find and explore the interactome for a given query set of proteins, using major current bioinformatic and computational databases and tools about PPIs. For bioinformaticians: to learn what is the biological meaning of a protein-protein interaction, which is the minimal and the critical information that has to be contained in an interaction file and what are the major current bioinformatic and computational databases and tools about PPIs. </li>
    <br />
    <li><strong>Web site: </strong><a class="moz-txt-link-freetext" href="javascript:if(confirm('http://bioinfow.dep.usal.es/apid/tutorial.htm  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://bioinfow.dep.usal.es/apid/tutorial.htm'">http://bioinfow.dep.usal.es/apid/tutorial.htm</a><a href="javascript:if(confirm('https://www.ebi.ac.uk/Rebholz-srv/aimm.html  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/Rebholz-srv/aimm.html'">       <br />
    <br />
    </a>   </li>
    <li><strong>Organizers:</strong> <br />
    <br />
    Javier De Las Rivas <br />
    email: <a class="moz-txt-link-abbreviated" href="mailto:jrivas@usal.es?subject=ECCB08%20Tutorial">jrivas@usal.es</a> <br />
    <br />
    Carlos Prieto<br />
    email: <a class="moz-txt-link-abbreviated" href="mailto:cprietos@usal.es?subject=ECCB08%20Tutorial">cprietos@usal.es</a> <br />
    <br />
    Bioinformatics and Functional Genomics Research Group <br />
    Cancer Research Center (CIC-IBMCC, CSIC-USAL) <br />
    Campus Miguel de Unamuno <br />
    E37007 Salamanca - Spain <br />
    <br />
    Tel:&nbsp; +34 923 294819 <br />
    Fax: +34 923 294743 <br />
    <br />
    </li>
    <li><strong>Fee: </strong>100 &euro;<br />
    </li>
</ul>
</fieldset>
<p><a href="#top">top</a></p>
<p><a href="#top"><br />
</a></p>
<h3><a name="TUT4"></a>&nbsp; Interoperability of bioinformatics software and databases: how to access databases and algorithms <br />
</h3>
<h3>&nbsp;using the Distributed Annotation System (DAS) and web-service technology.</h3>
<fieldset style="background: rgb(255, 255, 255) none repeat scroll 0% 50%; -moz-background-clip: -moz-initial; -moz-background-origin: -moz-initial; -moz-background-inline-policy: -moz-initial;">
<ul>
    <li><strong>Synopsis:<br />
    </strong>                	 	 	Properly annotating a gene or a protein sequence implies running many, many computer programs. DAS technology provides a way to get easy and rapid access to hundreds of sequence annotation facilities, and the DAS clients can visualize all the information in a compact, comprehensible format. <br />
    Web-services are the (near) future for most bioinformatics. They allow bioinformaticians access to hundreds and hundreds of computational facilities ranging from simple sequence operations to highly complex three dimensional protein structure calculations. <br />
    The Taverna system allows the combination of many web services into complex work flows. <br />
    This tutorial will provide three master classes: 1) The DAS system; 2) The Taverna system; 3) Web-service technology. <br />
    The tutorial will be supplemented by the possibility of &ldquo;tutor sessions&rdquo; throughout the conference. <br />
    <br />
    </li>
    <li><strong>Intended audience: </strong><br />
    The tutorial is targeted at bioinformaticians. While the specific tools and concepts (DAS, Taverna, etc) will be introduced, familiarity with standard web technologies as well as Perl and/or Java is assumed. <br />
    </li>
    <br />
    <li><strong>Web site: </strong><a class="moz-txt-link-freetext" href="javascript:if(confirm('http://swift.cmbi.ru.nl/ECCB_workshop/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://swift.cmbi.ru.nl/ECCB_workshop/'">http://swift.cmbi.ru.nl/ECCB_workshop/<br />
    <br />
    </a> <a href="javascript:if(confirm('https://www.ebi.ac.uk/Rebholz-srv/aimm.html  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='https://www.ebi.ac.uk/Rebholz-srv/aimm.html'"> </a>   </li>
    <li><strong>Organizers:</strong> <br />
    <br />
    Gert Vriend, CMBI<br />
    email: <a class="moz-txt-link-abbreviated" href="mailto:vriend@cmbi.ru.nl?subject=ECCB08%20Tutorial">vriend@cmbi.ru.nl</a> <br />
    <br />
    Henning Hermjakob, EMBL-EBI<br />
    email: <a class="moz-txt-link-abbreviated" href="mailto:hhe@ebi.ac.uk?subject=ECCB08%20Tutorial">hhe@ebi.ac.uk</a> <br />
    <br />
    </li>
    <li><strong>Fee: </strong>100 &euro;<br />
    </li>
</ul>
</fieldset>
<p><a href="#top">top</a></p>
</div>
</div>
 
</div>
</div>
<div id="rightcolumn">
<div class="topbg">
<div class="section">
&nbsp;</div>
</div>
</div>
<div id="footer">
<p>
Site developed by <a href="javascript:if(confirm('http://www.sciencedev.com  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.sciencedev.com'">Sciencedev</a> and based on EasyManager
</p>
</div>
</div>
</body>
</html>
