
<!doctype html>
<html lang="en-GB">
<head>
	
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, viewport-fit=cover">
	<link rel="profile" href="https://gmpg.org/xfn/11">

	<title>Poster presentations &#8211; ECCB2022</title>
<meta name='robots' content='max-image-preview:large'>
<link rel='dns-prefetch' href='//fonts.googleapis.com'>
<link rel="alternate" type="application/rss+xml" title="ECCB2022 &raquo; Feed" href="../feed/index.htm">
<link rel="alternate" type="application/rss+xml" title="ECCB2022 &raquo; Comments Feed" href="../comments/feed/index.htm">
<link rel='stylesheet' id='blocksy-dynamic-global-css' href='../wp-content/uploads/blocksy/css/global.css?ver=20706' media='all'>
<link rel='stylesheet' id='ugb-style-css-css' href='../wp-content/plugins/stackable-ultimate-gutenberg-blocks-premium/dist/frontend_blocks.css?ver=3.1.1' media='all'>
<style id='ugb-style-css-inline-css'>
:root {--stk-block-width-default-detected: 1000px;}
</style>
<link rel='stylesheet' id='ugb-style-css-responsive-css' href='../wp-content/plugins/stackable-ultimate-gutenberg-blocks-premium/dist/frontend_blocks_responsive.css?ver=3.1.1' media='all'>
<link rel='stylesheet' id='wp-block-library-css' href='../wp-includes/css/dist/block-library/style.min.css?ver=6.2' media='all'>
<link rel='stylesheet' id='ugb-style-css-premium-css' href='../wp-content/plugins/stackable-ultimate-gutenberg-blocks-premium/dist/frontend_blocks__premium_only.css?ver=3.1.1' media='all'>
<link rel='stylesheet' id='carousel-block-slick-css' href='../wp-content/plugins/carousel-block/dist/assets/vendor/slick/slick.min.css?ver=1681832604' media=''>
<link rel='stylesheet' id='carousel-block-css' href='../wp-content/plugins/carousel-block/dist/blocks.style.build.css?ver=1681832604' media='all'>
<link rel='stylesheet' id='fontawesome-free-css' href='../wp-content/plugins/getwid/vendors/fontawesome-free/css/all.min.css?ver=5.5.0' media='all'>
<link rel='stylesheet' id='slick-css' href='../wp-content/plugins/getwid/vendors/slick/slick/slick.min.css?ver=1.9.0' media='all'>
<link rel='stylesheet' id='slick-theme-css' href='../wp-content/plugins/getwid/vendors/slick/slick/slick-theme.min.css?ver=1.9.0' media='all'>
<link rel='stylesheet' id='fancybox-css' href='../wp-content/plugins/getwid/vendors/fancybox/jquery.fancybox.min.css?ver=3.5.7' media='all'>
<link rel='stylesheet' id='getwid-blocks-css' href='../wp-content/plugins/getwid/assets/css/blocks.style.css?ver=1.8.2' media='all'>
<style id='getwid-blocks-inline-css'>
.wp-block-getwid-section .wp-block-getwid-section__wrapper .wp-block-getwid-section__inner-wrapper{max-width: 1000px;}
</style>
<link rel='stylesheet' id='classic-theme-styles-css' href='../wp-includes/css/classic-themes.min.css?ver=6.2' media='all'>
<style id='global-styles-inline-css'>
body{--wp--preset--color--black: #000000;--wp--preset--color--cyan-bluish-gray: #abb8c3;--wp--preset--color--white: #ffffff;--wp--preset--color--pale-pink: #f78da7;--wp--preset--color--vivid-red: #cf2e2e;--wp--preset--color--luminous-vivid-orange: #ff6900;--wp--preset--color--luminous-vivid-amber: #fcb900;--wp--preset--color--light-green-cyan: #7bdcb5;--wp--preset--color--vivid-green-cyan: #00d084;--wp--preset--color--pale-cyan-blue: #8ed1fc;--wp--preset--color--vivid-cyan-blue: #0693e3;--wp--preset--color--vivid-purple: #9b51e0;--wp--preset--color--palette-color-1: var(--paletteColor1, #ffffff);--wp--preset--color--palette-color-2: var(--paletteColor2, #f7b032);--wp--preset--color--palette-color-3: var(--paletteColor3, #d38e30);--wp--preset--color--palette-color-4: var(--paletteColor4, #0081a7);--wp--preset--color--palette-color-5: var(--paletteColor5, #1a1a1a);--wp--preset--color--palette-color-6: var(--paletteColor6, #f2f5f7);--wp--preset--color--palette-color-7: var(--paletteColor7, #FAFBFC);--wp--preset--color--palette-color-8: var(--paletteColor8, #ffffff);--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple: linear-gradient(135deg,rgba(6,147,227,1) 0%,rgb(155,81,224) 100%);--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan: linear-gradient(135deg,rgb(122,220,180) 0%,rgb(0,208,130) 100%);--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange: linear-gradient(135deg,rgba(252,185,0,1) 0%,rgba(255,105,0,1) 100%);--wp--preset--gradient--luminous-vivid-orange-to-vivid-red: linear-gradient(135deg,rgba(255,105,0,1) 0%,rgb(207,46,46) 100%);--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray: linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%);--wp--preset--gradient--cool-to-warm-spectrum: linear-gradient(135deg,rgb(74,234,220) 0%,rgb(151,120,209) 20%,rgb(207,42,186) 40%,rgb(238,44,130) 60%,rgb(251,105,98) 80%,rgb(254,248,76) 100%);--wp--preset--gradient--blush-light-purple: linear-gradient(135deg,rgb(255,206,236) 0%,rgb(152,150,240) 100%);--wp--preset--gradient--blush-bordeaux: linear-gradient(135deg,rgb(254,205,165) 0%,rgb(254,45,45) 50%,rgb(107,0,62) 100%);--wp--preset--gradient--luminous-dusk: linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%);--wp--preset--gradient--pale-ocean: linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%);--wp--preset--gradient--electric-grass: linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%);--wp--preset--gradient--midnight: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%);--wp--preset--gradient--juicy-peach: linear-gradient(to right, #ffecd2 0%, #fcb69f 100%);--wp--preset--gradient--young-passion: linear-gradient(to right, #ff8177 0%, #ff867a 0%, #ff8c7f 21%, #f99185 52%, #cf556c 78%, #b12a5b 100%);--wp--preset--gradient--true-sunset: linear-gradient(to right, #fa709a 0%, #fee140 100%);--wp--preset--gradient--morpheus-den: linear-gradient(to top, #30cfd0 0%, #330867 100%);--wp--preset--gradient--plum-plate: linear-gradient(135deg, #667eea 0%, #764ba2 100%);--wp--preset--gradient--aqua-splash: linear-gradient(15deg, #13547a 0%, #80d0c7 100%);--wp--preset--gradient--love-kiss: linear-gradient(to top, #ff0844 0%, #ffb199 100%);--wp--preset--gradient--new-retrowave: linear-gradient(to top, #3b41c5 0%, #a981bb 49%, #ffc8a9 100%);--wp--preset--gradient--plum-bath: linear-gradient(to top, #cc208e 0%, #6713d2 100%);--wp--preset--gradient--high-flight: linear-gradient(to right, #0acffe 0%, #495aff 100%);--wp--preset--gradient--teen-party: linear-gradient(-225deg, #FF057C 0%, #8D0B93 50%, #321575 100%);--wp--preset--gradient--fabled-sunset: linear-gradient(-225deg, #231557 0%, #44107A 29%, #FF1361 67%, #FFF800 100%);--wp--preset--gradient--arielle-smile: radial-gradient(circle 248px at center, #16d9e3 0%, #30c7ec 47%, #46aef7 100%);--wp--preset--gradient--itmeo-branding: linear-gradient(180deg, #2af598 0%, #009efd 100%);--wp--preset--gradient--deep-blue: linear-gradient(to right, #6a11cb 0%, #2575fc 100%);--wp--preset--gradient--strong-bliss: linear-gradient(to right, #f78ca0 0%, #f9748f 19%, #fd868c 60%, #fe9a8b 100%);--wp--preset--gradient--sweet-period: linear-gradient(to top, #3f51b1 0%, #5a55ae 13%, #7b5fac 25%, #8f6aae 38%, #a86aa4 50%, #cc6b8e 62%, #f18271 75%, #f3a469 87%, #f7c978 100%);--wp--preset--gradient--purple-division: linear-gradient(to top, #7028e4 0%, #e5b2ca 100%);--wp--preset--gradient--cold-evening: linear-gradient(to top, #0c3483 0%, #a2b6df 100%, #6b8cce 100%, #a2b6df 100%);--wp--preset--gradient--mountain-rock: linear-gradient(to right, #868f96 0%, #596164 100%);--wp--preset--gradient--desert-hump: linear-gradient(to top, #c79081 0%, #dfa579 100%);--wp--preset--gradient--ethernal-constance: linear-gradient(to top, #09203f 0%, #537895 100%);--wp--preset--gradient--happy-memories: linear-gradient(-60deg, #ff5858 0%, #f09819 100%);--wp--preset--gradient--grown-early: linear-gradient(to top, #0ba360 0%, #3cba92 100%);--wp--preset--gradient--morning-salad: linear-gradient(-225deg, #B7F8DB 0%, #50A7C2 100%);--wp--preset--gradient--night-call: linear-gradient(-225deg, #AC32E4 0%, #7918F2 48%, #4801FF 100%);--wp--preset--gradient--mind-crawl: linear-gradient(-225deg, #473B7B 0%, #3584A7 51%, #30D2BE 100%);--wp--preset--gradient--angel-care: linear-gradient(-225deg, #FFE29F 0%, #FFA99F 48%, #FF719A 100%);--wp--preset--gradient--juicy-cake: linear-gradient(to top, #e14fad 0%, #f9d423 100%);--wp--preset--gradient--rich-metal: linear-gradient(to right, #d7d2cc 0%, #304352 100%);--wp--preset--gradient--mole-hall: linear-gradient(-20deg, #616161 0%, #9bc5c3 100%);--wp--preset--gradient--cloudy-knoxville: linear-gradient(120deg, #fdfbfb 0%, #ebedee 100%);--wp--preset--gradient--soft-grass: linear-gradient(to top, #c1dfc4 0%, #deecdd 100%);--wp--preset--gradient--saint-petersburg: linear-gradient(135deg, #f5f7fa 0%, #c3cfe2 100%);--wp--preset--gradient--everlasting-sky: linear-gradient(135deg, #fdfcfb 0%, #e2d1c3 100%);--wp--preset--gradient--kind-steel: linear-gradient(-20deg, #e9defa 0%, #fbfcdb 100%);--wp--preset--gradient--over-sun: linear-gradient(60deg, #abecd6 0%, #fbed96 100%);--wp--preset--gradient--premium-white: linear-gradient(to top, #d5d4d0 0%, #d5d4d0 1%, #eeeeec 31%, #efeeec 75%, #e9e9e7 100%);--wp--preset--gradient--clean-mirror: linear-gradient(45deg, #93a5cf 0%, #e4efe9 100%);--wp--preset--gradient--wild-apple: linear-gradient(to top, #d299c2 0%, #fef9d7 100%);--wp--preset--gradient--snow-again: linear-gradient(to top, #e6e9f0 0%, #eef1f5 100%);--wp--preset--gradient--confident-cloud: linear-gradient(to top, #dad4ec 0%, #dad4ec 1%, #f3e7e9 100%);--wp--preset--gradient--glass-water: linear-gradient(to top, #dfe9f3 0%, white 100%);--wp--preset--gradient--perfect-white: linear-gradient(-225deg, #E3FDF5 0%, #FFE6FA 100%);--wp--preset--duotone--dark-grayscale: url('#wp-duotone-dark-grayscale');--wp--preset--duotone--grayscale: url('#wp-duotone-grayscale');--wp--preset--duotone--purple-yellow: url('#wp-duotone-purple-yellow');--wp--preset--duotone--blue-red: url('#wp-duotone-blue-red');--wp--preset--duotone--midnight: url('#wp-duotone-midnight');--wp--preset--duotone--magenta-yellow: url('#wp-duotone-magenta-yellow');--wp--preset--duotone--purple-green: url('#wp-duotone-purple-green');--wp--preset--duotone--blue-orange: url('#wp-duotone-blue-orange');--wp--preset--font-size--small: 13px;--wp--preset--font-size--medium: 20px;--wp--preset--font-size--large: 36px;--wp--preset--font-size--x-large: 42px;--wp--preset--spacing--20: 0.44rem;--wp--preset--spacing--30: 0.67rem;--wp--preset--spacing--40: 1rem;--wp--preset--spacing--50: 1.5rem;--wp--preset--spacing--60: 2.25rem;--wp--preset--spacing--70: 3.38rem;--wp--preset--spacing--80: 5.06rem;--wp--preset--shadow--natural: 6px 6px 9px rgba(0, 0, 0, 0.2);--wp--preset--shadow--deep: 12px 12px 50px rgba(0, 0, 0, 0.4);--wp--preset--shadow--sharp: 6px 6px 0px rgba(0, 0, 0, 0.2);--wp--preset--shadow--outlined: 6px 6px 0px -3px rgba(255, 255, 255, 1), 6px 6px rgba(0, 0, 0, 1);--wp--preset--shadow--crisp: 6px 6px 0px rgba(0, 0, 0, 1);}:where(.is-layout-flex){gap: 0.5em;}body .is-layout-flow > .alignleft{float: left;margin-inline-start: 0;margin-inline-end: 2em;}body .is-layout-flow > .alignright{float: right;margin-inline-start: 2em;margin-inline-end: 0;}body .is-layout-flow > .aligncenter{margin-left: auto !important;margin-right: auto !important;}body .is-layout-constrained > .alignleft{float: left;margin-inline-start: 0;margin-inline-end: 2em;}body .is-layout-constrained > .alignright{float: right;margin-inline-start: 2em;margin-inline-end: 0;}body .is-layout-constrained > .aligncenter{margin-left: auto !important;margin-right: auto !important;}body .is-layout-constrained > :where(:not(.alignleft):not(.alignright):not(.alignfull)){max-width: var(--wp--style--global--content-size);margin-left: auto !important;margin-right: auto !important;}body .is-layout-constrained > .alignwide{max-width: var(--wp--style--global--wide-size);}body .is-layout-flex{display: flex;}body .is-layout-flex{flex-wrap: wrap;align-items: center;}body .is-layout-flex > *{margin: 0;}:where(.wp-block-columns.is-layout-flex){gap: 2em;}.has-black-color{color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-color{color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-color{color: var(--wp--preset--color--white) !important;}.has-pale-pink-color{color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-color{color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-color{color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-color{color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-color{color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-color{color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-color{color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-color{color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-color{color: var(--wp--preset--color--vivid-purple) !important;}.has-black-background-color{background-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-background-color{background-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-background-color{background-color: var(--wp--preset--color--white) !important;}.has-pale-pink-background-color{background-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-background-color{background-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-background-color{background-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-background-color{background-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-background-color{background-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-background-color{background-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-background-color{background-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-background-color{background-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-background-color{background-color: var(--wp--preset--color--vivid-purple) !important;}.has-black-border-color{border-color: var(--wp--preset--color--black) !important;}.has-cyan-bluish-gray-border-color{border-color: var(--wp--preset--color--cyan-bluish-gray) !important;}.has-white-border-color{border-color: var(--wp--preset--color--white) !important;}.has-pale-pink-border-color{border-color: var(--wp--preset--color--pale-pink) !important;}.has-vivid-red-border-color{border-color: var(--wp--preset--color--vivid-red) !important;}.has-luminous-vivid-orange-border-color{border-color: var(--wp--preset--color--luminous-vivid-orange) !important;}.has-luminous-vivid-amber-border-color{border-color: var(--wp--preset--color--luminous-vivid-amber) !important;}.has-light-green-cyan-border-color{border-color: var(--wp--preset--color--light-green-cyan) !important;}.has-vivid-green-cyan-border-color{border-color: var(--wp--preset--color--vivid-green-cyan) !important;}.has-pale-cyan-blue-border-color{border-color: var(--wp--preset--color--pale-cyan-blue) !important;}.has-vivid-cyan-blue-border-color{border-color: var(--wp--preset--color--vivid-cyan-blue) !important;}.has-vivid-purple-border-color{border-color: var(--wp--preset--color--vivid-purple) !important;}.has-vivid-cyan-blue-to-vivid-purple-gradient-background{background: var(--wp--preset--gradient--vivid-cyan-blue-to-vivid-purple) !important;}.has-light-green-cyan-to-vivid-green-cyan-gradient-background{background: var(--wp--preset--gradient--light-green-cyan-to-vivid-green-cyan) !important;}.has-luminous-vivid-amber-to-luminous-vivid-orange-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-amber-to-luminous-vivid-orange) !important;}.has-luminous-vivid-orange-to-vivid-red-gradient-background{background: var(--wp--preset--gradient--luminous-vivid-orange-to-vivid-red) !important;}.has-very-light-gray-to-cyan-bluish-gray-gradient-background{background: var(--wp--preset--gradient--very-light-gray-to-cyan-bluish-gray) !important;}.has-cool-to-warm-spectrum-gradient-background{background: var(--wp--preset--gradient--cool-to-warm-spectrum) !important;}.has-blush-light-purple-gradient-background{background: var(--wp--preset--gradient--blush-light-purple) !important;}.has-blush-bordeaux-gradient-background{background: var(--wp--preset--gradient--blush-bordeaux) !important;}.has-luminous-dusk-gradient-background{background: var(--wp--preset--gradient--luminous-dusk) !important;}.has-pale-ocean-gradient-background{background: var(--wp--preset--gradient--pale-ocean) !important;}.has-electric-grass-gradient-background{background: var(--wp--preset--gradient--electric-grass) !important;}.has-midnight-gradient-background{background: var(--wp--preset--gradient--midnight) !important;}.has-small-font-size{font-size: var(--wp--preset--font-size--small) !important;}.has-medium-font-size{font-size: var(--wp--preset--font-size--medium) !important;}.has-large-font-size{font-size: var(--wp--preset--font-size--large) !important;}.has-x-large-font-size{font-size: var(--wp--preset--font-size--x-large) !important;}
.wp-block-navigation a:where(:not(.wp-element-button)){color: inherit;}
:where(.wp-block-columns.is-layout-flex){gap: 2em;}
.wp-block-pullquote{font-size: 1.5em;line-height: 1.6;}
</style>
<link rel='stylesheet' id='popup-maker-site-css' href='../wp-content/uploads/pum/pum-site-styles.css?generated=1663251559&#038;ver=1.16.7' media='all'>
<link rel='stylesheet' id='blocksy-fonts-font-source-google-css' href='../css2?family=Inter:wght@400;500;600;700&#038;display=swap' media='all'>
<link rel='stylesheet' id='ct-main-styles-css' href='../wp-content/themes/blocksy/static/bundle/main.min.css?ver=1.8.14' media='all'>
<link rel='stylesheet' id='ct-page-title-styles-css' href='../wp-content/themes/blocksy/static/bundle/page-title.min.css?ver=1.8.14' media='all'>
<link rel='stylesheet' id='ct-getwid-styles-css' href='../wp-content/themes/blocksy/static/bundle/getwid.min.css?ver=1.8.14' media='all'>
<link rel='stylesheet' id='ct-stackable-styles-css' href='../wp-content/themes/blocksy/static/bundle/stackable.min.css?ver=1.8.14' media='all'>

			<!-- Global site tag (gtag.js) - Google Analytics v4 -->
			<script async="" src="../gtag/js?id=G-9JDW1SQX9T"></script>
			<script>
			window.dataLayer = window.dataLayer || [];
			function gtag(){dataLayer.push(arguments);}
			gtag('js', new Date());

			gtag('config', 'G-9JDW1SQX9T');
			</script>
			<!-- End Google Analytics v4 -->

			<script id='ugb-block-frontend-js-js-extra'>
var stackable = {"restUrl":"https:\/\/eccb2022.org\/wp-json\/"};
</script>
<script src='../wp-includes/js/jquery/jquery.min.js?ver=3.6.3' id='jquery-core-js'></script>
<script src='../wp-includes/js/jquery/jquery-migrate.min.js?ver=3.4.0' id='jquery-migrate-js'></script>
<link rel="https://api.w.org/" href="../wp-json/index.htm"><link rel="alternate" type="application/json" href="../wp-json/wp/v2/pages/2905"><link rel="EditURI" type="application/rsd+xml" title="RSD" href="../xmlrpc.php?rsd">
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="../wp-includes/wlwmanifest.xml">
<meta name="generator" content="WordPress 6.2">
<link rel="canonical" href="index.htm">
<link rel='shortlink' href='index.htm?p=2905'>
<link rel="alternate" type="application/json+oembed" href="../wp-json/oembed/1.0/embed-44?url=https%3A%2F%2Feccb2022.org%2Fposter-presentations%2F">
<link rel="alternate" type="text/xml+oembed" href="../wp-json/oembed/1.0/embed-45?url=https%3A%2F%2Feccb2022.org%2Fposter-presentations%2F&#038;format=xml">
<noscript><link rel='stylesheet' href='../wp-content/themes/blocksy/static/bundle/no-scripts.min.css' type='text/css'></noscript>
<link rel="icon" href="../wp-content/uploads/2022/01/cropped-Favicon-03-32x32.png" sizes="32x32">
<link rel="icon" href="../wp-content/uploads/2022/01/cropped-Favicon-03-192x192.png" sizes="192x192">
<link rel="apple-touch-icon" href="../wp-content/uploads/2022/01/cropped-Favicon-03-180x180.png">
<meta name="msapplication-TileImage" content="https://eccb2022.org/wp-content/uploads/2022/01/cropped-Favicon-03-270x270.png">
		<style id="wp-custom-css">
			/* Cover home */

.cover-home h1
{
	font-size: 32px;
	margin-bottom: 15px;
}

.cover-home p
{
	font-size: 24px;
	line-height: 30px;
	margin-bottom: 30px;
}

/* Tarjetas con botón */

.tarjeta-boton .stk-inner-blocks
{
	display: flex;
justify-content: space-between !important;
	height: 100%;
}

.tarjeta-boton p
{
	line-height: 18px;
	line-height: 23px;
	margin-bottom: 25px !important;
}

.tarjeta-boton strong
{
	font-size: 24px;
	line-height: 30px;
	font-weight: 400;
	display: block;
}

/* Welcome home */

.welcome-home p
{
	font-size: 32px;
	line-height: 36px !important;	
}


/* Key dates home */



.key-dates-home .listado p
{

	font-size: 24px;
	line-height: 30px;
}




@media (min-width: 768px)
{
	.key-dates-home .listado p
{
	
	background-image: url("../wp-content/uploads/2022/01/flecha-01.svg");
	background-repeat: no-repeat;
	
background-size: 25px auto;
	background-position: 0 6px;
	padding-left: 35px;
}
	
	.key-dates .wp-block-stackable-column
	{
		padding-right: 30px
	}
	
	.key-dates p
	{
			padding-left: 30px;
	}
	
		.key-dates p:first-child
	{
		background-image: url("../wp-content/uploads/2022/01/Flecha-02.svg");
		background-size: 18px auto;
	background-position: 0 6px;
	padding-left: 30px;
			background-repeat: no-repeat;
	}
	
}

/* Fondos */

.cover-home
{
			background-image: url("../wp-content/uploads/2022/01/Fondo-cover-05.png");
	background-repeat: no-repeat;
	background-size: contain;
	background-position: 102% 100%;
}

.destacados-home
{
		background-image: url("../wp-content/uploads/2022/01/Helice-06.svg");
	background-repeat: no-repeat;
	background-position: 40% 50%;
	background-size: auto 65px;
}

.welcome-home
{
	background-image: url("../wp-content/uploads/2022/01/Helice-05.svg");
	background-repeat: no-repeat;
	background-position: 40% 50%;
	background-size: auto 65px;
}

.key-dates-home
{
	background-image: url("../wp-content/uploads/2022/01/Helice-03.svg");
	background-repeat: no-repeat;
	background-size: 65px auto;
	background-position: 90% 50%;
}

@media (max-width: 768px) {
.welcome-home
{
	background-image: url("../wp-content/uploads/2022/01/Helice-01.svg");
	background-repeat: no-repeat;
	background-position: 40% 90%;
	background-size: auto 65px;
}
	
	.key-dates-home
	{
		background-image: none !important;
	}

}

.hero-section 
{
		background-image: url("../wp-content/uploads/2022/01/Helice-01.svg");
	background-repeat: no-repeat;
	background-position: 40% 100%;
	background-size: auto 65px;
}

.texto-destacado
{
	font-size: 28px;
}

/* COMITÉS */

.comites p
{
	margin-bottom: 10px;
	font-size: 15px;
}

.comites p:first-child
{
	font-size: 22px;
}

.tabla-comite td
{
	vertical-align: top;
	font-size: 16px;
	line-height: 22px;
}

.tabla-comite td:first-child
{
	width: 25% !important;
}

.tabla-comite td:last-child
{
	width: 15%;
}





.tabla-pres td
{
	font-size: 16px;
	vertical-align: top;
	padding-bottom: 16px;
	padding-top: 16px;
}

.tabla-pres td:first-child
{
	width: 5% !important;
}

.tabla-pres td:nth-child(2)
{
	width: 12% !important;
}



.tabla-pres tr:first-child td
{
	color: #0081A7;
}



@media (max-width: 600px) {
.tabla-comite td
{
	font-size: 13px;
	}
	
	.tabla-pres td
{
	font-size: 12px;
	line-height: 15px;
	padding: 15px 7px !important;
	}

}

.tabla-comite-p
{
	margin-bottom: 40px !important;
}


.tabla-key-dates td,
.tabla-topics td
{
	vertical-align: top;
}

.tabla-key-dates td:first-child
{
	width: 35%;
}


.comites .wp-block-stackable-column 
{
	padding-right: 20px;
	/*
background-image:
	url("https://eccb2022.org/wp-content/uploads/2022/01/Borde-01.png");
		background-repeat: no-repeat;
	background-position: 0 100%;
	background-size: 50px 2px;
	padding-bottom: 10px;
	*/

}

/* PIE */

.ct-footer #block-8 img
{
	width: 200px !important;
	height: auto;
}

.ct-footer strong
{
	font-weight: 400;
	color: #f7b032;
}

.ct-footer *
{
	font-size: 13px !important
}

.logos-org
{
	display: flex;
	flex-direction: row;
	flex-wrap: wrap;
	align-items: flex-end;
}

.logos-org img
{
	margin-right: 30px; 
	margin-bottom: 30px;
}

.logos-org .logo-1 img
{
	height: 41px;
	width: auto;
}

.logos-org .logo-2 img
{
	height: 51px;
	width: auto;
}

.logos-org .logo-3 img
{
	height: 42px;
	width: auto;
}


/* LISTADOS */

#main ul
{
	padding: 0;
	list-style: none;
}

#main ul li
{
background-image: url("../wp-content/uploads/2022/01/Flecha-02.svg");
		background-size: 18px auto;
	background-position: 0 6px;
	padding-left: 30px;
			background-repeat: no-repeat;
	margin-bottom: 10px;
}

#main li
{

	margin-bottom: 10px
}

/* BOLD */

#main strong
{
	/*
	font-weight: 400;
	color: #0081a7;
	*/
}

/* PROGRAMA CALLS */

.programa-calls th
{
	text-transform: uppercase;
	font-size: 15px;
}

/* FICHAS */

.fichas
{
	background-image: none !important;
}

/* MARGEN DEL HASHTAG */

.ct-header-socials
{
	margin-right: -5px !important
}

/* iframe */

.icontainer {
  position: relative;
  overflow: hidden;
  width: 100%;
  padding-top: 56.25%; /* 16:9 Aspect Ratio (divide 9 by 16 = 0.5625) */
	 padding-top: 80%;
}

.iresponsive-iframe {
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
  width: 100%;
  height: 65%;
	border: 0;
}

@media (max-width: 768px)
{

.iresponsive-iframe
	{
		  height: 90%;
	}
}

.stk-block-accordion__heading 
{
	border-bottom: 1px solid #f7b032 !important;
	padding-left: 0 !important;
}

.boton-suscripcion img
{
	width: 38px;
	height: 38px;
	
}
/* TABLA NTB */

.tabla-ntb td,
.tabla-ntb th,
.tabla-ntb table
{
	border: 0 !important;
	padding-left: 0;
	vertical-align: top !important;
}

.tabla-ntb td:first-child
{
	width: 140px !important;
}

.tabla-ntb th
{
	text-transform: uppercase;
	font-size: 15px;
}

/*
.tabla-ntb td:nth-child(2),
.tabla-ntb th:nth-child(2)
{
	vertical-align: top;
}

.tabla-ntb td:nth-child(3),
.tabla-ntb th:nth-child(3)
{
	vertical-align: top;	
}

.tabla-ntb td:nth-child(4),
.tabla-ntb th:nth-child(4)
{
	vertical-align: top;	
}

.tabla-ntb td:nth-child(5),
.tabla-ntb th:nth-child(5)
{
	vertical-align: top;	
}

.tabla-ntb td:nth-child(6),
.tabla-ntb th:nth-child(6)
{
	vertical-align: top;	
}

.tabla-ntb td:nth-child(7),
.tabla-ntb th:nth-child(7)
{
	vertical-align: top;	
}

.tabla-ntb td:nth-child(8),
.tabla-ntb th:nth-child(8)
{
	vertical-align: top;	
}

*/


/*OCULTO */

.oculto
{
	display: none !important;
}

/* Botones en Registration */

.botones-linea .wp-block-stackable-button
{
	width: 49% !important;
}

.botones-linea .wp-block-stackable-button a
{
	width: 100%;
	text-align: center;
	padding-top: 15px !important;
	padding-bottom: 15px !important;
}

@media (max-width: 780px)
{
	
.botones-linea .wp-block-stackable-button
{
	width: 100% !important;
	margin-right: 0 !important;
}
}

.anclas .wp-block-stackable-button 
{
	margin-right: 7px !imaportant;
}

.anclas .wp-block-stackable-button a
{
	font-size: 15px !important;
	padding: 10px 12px !important;

}



		</style>
			</head>

<body data-rsssl="1" class="page-template-default page page-id-2905 wp-custom-logo wp-embed-responsive stk--is-blocksy-theme ct-loading" data-link="type-2" data-prefix="single_page" data-header="type-1" data-footer="type-1" itemscope="itemscope" itemtype="http://schema.org/WebPage">

<a class="skip-link show-on-focus" href="#main">
	Skip to content</a>

<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-dark-grayscale"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0 0.49803921568627"></fefuncr><fefuncg type="table" tablevalues="0 0.49803921568627"></fefuncg><fefuncb type="table" tablevalues="0 0.49803921568627"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-grayscale"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0 1"></fefuncr><fefuncg type="table" tablevalues="0 1"></fefuncg><fefuncb type="table" tablevalues="0 1"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-purple-yellow"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0.54901960784314 0.98823529411765"></fefuncr><fefuncg type="table" tablevalues="0 1"></fefuncg><fefuncb type="table" tablevalues="0.71764705882353 0.25490196078431"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-blue-red"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0 1"></fefuncr><fefuncg type="table" tablevalues="0 0.27843137254902"></fefuncg><fefuncb type="table" tablevalues="0.5921568627451 0.27843137254902"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-midnight"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0 0"></fefuncr><fefuncg type="table" tablevalues="0 0.64705882352941"></fefuncg><fefuncb type="table" tablevalues="0 1"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-magenta-yellow"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0.78039215686275 1"></fefuncr><fefuncg type="table" tablevalues="0 0.94901960784314"></fefuncg><fefuncb type="table" tablevalues="0.35294117647059 0.47058823529412"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-purple-green"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0.65098039215686 0.40392156862745"></fefuncr><fefuncg type="table" tablevalues="0 1"></fefuncg><fefuncb type="table" tablevalues="0.44705882352941 0.4"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg><svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 0 0" width="0" height="0" focusable="false" role="none" style="visibility: hidden; position: absolute; left: -9999px; overflow: hidden;"><defs><filter id="wp-duotone-blue-orange"><fecolormatrix color-interpolation-filters="sRGB" type="matrix" values=" .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 .299 .587 .114 0 0 "></fecolormatrix><fecomponenttransfer color-interpolation-filters="sRGB"><fefuncr type="table" tablevalues="0.098039215686275 1"></fefuncr><fefuncg type="table" tablevalues="0 0.66274509803922"></fefuncg><fefuncb type="table" tablevalues="0.84705882352941 0.41960784313725"></fefuncb><fefunca type="table" tablevalues="1 1"></fefunca></fecomponenttransfer><fecomposite in2="SourceGraphic" operator="in"></fecomposite></filter></defs></svg>
<div id="main-container">
	<header id="header" class="ct-header" data-id="type-1" itemscope="" itemtype="https://schema.org/WPHeader"><div data-device="desktop"><div data-row="middle" data-column-set="2"><div class="ct-container-fluid"><div data-column="start" data-placements="1"><div data-items="primary">
<div class="site-branding" data-id="logo" itemscope="itemscope" itemtype="https://schema.org/Organization">

			<a href="../index.htm" class="site-logo-container" rel="home"><img width="112" height="15" src="../wp-content/uploads/2022/01/Logo-02.svg" class="default-logo" alt="ECCB 2022"></a>	
	</div>

</div></div><div data-column="end" data-placements="1"><div data-items="primary">
<nav id="header-menu-1" class="header-menu-1" role="navigation" data-id="menu" data-interaction="hover" data-menu="type-1" data-dropdown="type-1:simple" data-responsive="no" itemscope="" itemtype="http://schema.org/SiteNavigationElement">

	<ul id="menu-general" class="menu"><li id="menu-item-122" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-122 animated-submenu"><a href="#" class="ct-menu-link">About<span class="ct-toggle-dropdown-desktop"><svg class="ct-icon" width="8" height="8" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M2.1,3.2l5.4,5.4l5.4-5.4L15,4.3l-7.5,7.5L0,4.3L2.1,3.2z"></path></svg></span></a><button class="ct-toggle-dropdown-desktop-ghost" aria-label="Expand dropdown menu" aria-expanded="false"></button>
<ul class="sub-menu">
	<li id="menu-item-209" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-209"><a href="../welcome/index.htm" class="ct-menu-link">Welcome</a></li>
	<li id="menu-item-246" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-246"><a href="../committees/index.htm" class="ct-menu-link">Committees</a></li>
	<li id="menu-item-310" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-310"><a href="../key-dates/index.htm" class="ct-menu-link">Key dates</a></li>
</ul>
</li>
<li id="menu-item-2630" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-2630 animated-submenu"><a href="#" class="ct-menu-link">Sponsors<span class="ct-toggle-dropdown-desktop"><svg class="ct-icon" width="8" height="8" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M2.1,3.2l5.4,5.4l5.4-5.4L15,4.3l-7.5,7.5L0,4.3L2.1,3.2z"></path></svg></span></a><button class="ct-toggle-dropdown-desktop-ghost" aria-label="Expand dropdown menu" aria-expanded="false"></button>
<ul class="sub-menu">
	<li id="menu-item-319" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-319"><a href="../sponsors/index.htm" class="ct-menu-link">Become a sponsor</a></li>
	<li id="menu-item-2629" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2629"><a href="../confirmed-sponsors-exhibitors/index.htm" class="ct-menu-link">Confirmed sponsors &#038; exhibitors</a></li>
</ul>
</li>
<li id="menu-item-1170" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1170 animated-submenu"><a href="#" class="ct-menu-link">Programme<span class="ct-toggle-dropdown-desktop"><svg class="ct-icon" width="8" height="8" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M2.1,3.2l5.4,5.4l5.4-5.4L15,4.3l-7.5,7.5L0,4.3L2.1,3.2z"></path></svg></span></a><button class="ct-toggle-dropdown-desktop-ghost" aria-label="Expand dropdown menu" aria-expanded="false"></button>
<ul class="sub-menu">
	<li id="menu-item-313" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-313"><a href="../final-programme/index.htm" class="ct-menu-link">Final programme</a></li>
	<li id="menu-item-908" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-908"><a href="../preliminary-programme-at-a-glance/index.htm" class="ct-menu-link">Programme at a glance</a></li>
	<li id="menu-item-2386" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2386"><a href="../keynote-speakers/index.htm" class="ct-menu-link">Keynote Speakers</a></li>
	<li id="menu-item-1799" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1799"><a href="../workshop-and-tutorials/index.htm" class="ct-menu-link">Workshops and tutorials</a></li>
	<li id="menu-item-2200" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2200"><a href="../satellite-meetings/index.htm" class="ct-menu-link">Satellite Meetings</a></li>
	<li id="menu-item-3202" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3202"><a href="../birds-of-a-feather/index.htm" class="ct-menu-link">Birds of a Feather</a></li>
	<li id="menu-item-3452" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3452"><a href="../eccb2022-best-poster-awards/index.htm" class="ct-menu-link">ECCB2022 Best Poster Awards</a></li>
</ul>
</li>
<li id="menu-item-919" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-919"><a href="../registration/index.htm" class="ct-menu-link">Registration</a></li>
<li id="menu-item-1163" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1163 animated-submenu"><a href="#" class="ct-menu-link">Practicals<span class="ct-toggle-dropdown-desktop"><svg class="ct-icon" width="8" height="8" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M2.1,3.2l5.4,5.4l5.4-5.4L15,4.3l-7.5,7.5L0,4.3L2.1,3.2z"></path></svg></span></a><button class="ct-toggle-dropdown-desktop-ghost" aria-label="Expand dropdown menu" aria-expanded="false"></button>
<ul class="sub-menu">
	<li id="menu-item-1167" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1167"><a href="../venue/index.htm" class="ct-menu-link">Venue</a></li>
	<li id="menu-item-1165" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1165"><a href="../travel/index.htm" class="ct-menu-link">Travel</a></li>
	<li id="menu-item-1166" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1166"><a href="../hotels/index.htm" class="ct-menu-link">Hotels</a></li>
	<li id="menu-item-1164" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1164"><a href="../sitges/index.htm" class="ct-menu-link">Sitges</a></li>
	<li id="menu-item-3168" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3168"><a href="../app/index.htm" class="ct-menu-link">Official APP</a></li>
	<li id="menu-item-3200" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3200"><a href="../safety-protocols-and-accesibilty-statement/index.htm" class="ct-menu-link">Safety protocols and accessibility information</a></li>
</ul>
</li>
<li id="menu-item-459" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-459 animated-submenu"><a href="../calls/index.htm" class="ct-menu-link">Calls<span class="ct-toggle-dropdown-desktop"><svg class="ct-icon" width="8" height="8" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M2.1,3.2l5.4,5.4l5.4-5.4L15,4.3l-7.5,7.5L0,4.3L2.1,3.2z"></path></svg></span></a><button class="ct-toggle-dropdown-desktop-ghost" aria-label="Expand dropdown menu" aria-expanded="false"></button>
<ul class="sub-menu">
	<li id="menu-item-3073" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3073"><a href="../guidelines/index.htm" class="ct-menu-link">Presentations guidelines</a></li>
	<li id="menu-item-3072" class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-2905 current_page_item menu-item-3072"><a href="index.htm" aria-current="page" class="ct-menu-link">Guidelines for poster presentations</a></li>
</ul>
</li>
<li id="menu-item-643" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-643"><a href="../code-of-conduct/index.htm" class="ct-menu-link">Code of Conduct</a></li>
</ul></nav>


<div class="ct-header-socials " data-id="socials">

	
		<div class="ct-social-box" data-icon-size="custom" data-color="custom" data-icons-type="rounded:solid">
			
			
							
				<a href="https://twitter.com/ECCBinfo" data-network="twitter" aria-label="Twitter" target="_blank" rel="noopener">
					<span class="ct-icon-container">
				<svg class="ct-icon" width="20px" height="20px" viewbox="0 0 20 20" aria-label="Twitter Icon">
					<path d="M20,3.8c-0.7,0.3-1.5,0.5-2.4,0.6c0.8-0.5,1.5-1.3,1.8-2.3c-0.8,0.5-1.7,0.8-2.6,1c-0.7-0.8-1.8-1.3-3-1.3c-2.3,0-4.1,1.8-4.1,4.1c0,0.3,0,0.6,0.1,0.9C6.4,6.7,3.4,5.1,1.4,2.6C1,3.2,0.8,3.9,0.8,4.7c0,1.4,0.7,2.7,1.8,3.4C2,8.1,1.4,7.9,0.8,7.6c0,0,0,0,0,0.1c0,2,1.4,3.6,3.3,4c-0.3,0.1-0.7,0.1-1.1,0.1c-0.3,0-0.5,0-0.8-0.1c0.5,1.6,2,2.8,3.8,2.8c-1.4,1.1-3.2,1.8-5.1,1.8c-0.3,0-0.7,0-1-0.1c1.8,1.2,4,1.8,6.3,1.8c7.5,0,11.7-6.3,11.7-11.7c0-0.2,0-0.4,0-0.5C18.8,5.3,19.4,4.6,20,3.8z"></path>
				</svg>
			</span><span class="ct-label" hidden="">Twitter</span>				</a>
			
			
					</div>

	
</div>
</div></div></div></div></div><div data-device="mobile"><div data-row="middle" data-column-set="2"><div class="ct-container-fluid"><div data-column="start" data-placements="1"><div data-items="primary">
<div class="site-branding" data-id="logo" itemscope="itemscope" itemtype="https://schema.org/Organization">

			<a href="../index.htm" class="site-logo-container" rel="home"><img width="112" height="15" src="../wp-content/uploads/2022/01/Logo-02.svg" class="default-logo" alt="ECCB 2022"></a>	
	</div>

</div></div><div data-column="end" data-placements="1"><div data-items="primary">
<button data-toggle-panel="#offcanvas" class="ct-header-trigger ct-toggle" data-design="simple" data-label="right" aria-label="Menú" data-id="trigger">

	<span class="ct-label ct-hidden-sm ct-hidden-md ct-hidden-lg">Menú</span>

	<svg class="ct-icon" width="18" height="14" viewbox="0 0 18 14" aria-label="Off Canvas trigger icon" data-type="type-1">

		<rect y="0.00" width="18" height="1.7" rx="1"></rect>
		<rect y="6.15" width="18" height="1.7" rx="1"></rect>
		<rect y="12.3" width="18" height="1.7" rx="1"></rect>
	</svg>
</button>
</div></div></div></div></div></header>
	<main id="main" class="site-main hfeed">

		
<div class="hero-section" data-type="type-2">
	
	<header class="entry-header ct-container">
		<h1 class="page-title" title="Poster presentations" itemprop="headline">Poster presentations</h1>	</header>
</div>



	<div class="ct-container-full" data-content="normal" data-vertical-spacing="top:bottom">

		
		
	<article id="post-2905" class="post-2905 page type-page status-publish hentry">

		
		
		
		<div class="entry-content">
			
<div class="wp-block-stackable-heading stk-block-heading stk-block stk-a31bb54" data-block-id="a31bb54"><style>.stk-a31bb54 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">General information for posters</h2><div class="stk-block-heading__bottom-line"></div></div>



<ul><li>The&nbsp; two poster sessions scheduled within the ECC2022 programme will take place on Monday, 19 Sept. and Tuesday, 20 Sept., from 17:30h to 19:30h.&nbsp;</li></ul>



<ul><li>All posters will be displayed at the room Tramuntana 3 (groundfloor). The presenting authors are required to give their presentation in-person from the conference venue during the session where they have been assigned.</li></ul>



<ul><li>Material to fix the posters on the panels will be available in the poster area (double-sided tape).</li></ul>



<ul><li>Each poster will be displayed on the day of the poster session only, (either on Monday or Tuesday). The posters should be installed in the morning and removed before 21:00h. Please make sure to take away your poster yourself; leftover posters will be disposed of.</li></ul>



<ul><li>The poster boards are adapted to standard A0 poster size / portrait (A0 is 84.1 x 118.9cm ; or 33.1 x 46.8 inches). The maximum size of the poster must be 90 cm wide and 120 cm high.&nbsp;</li></ul>



<ul><li>The posters must include the title, the name of the authors, the affiliations and a contact address. It is recommended to use a font size of at least 24 points to ensure its legibility. All posters must be written in English.</li></ul>



<ul><li>The poster authors are invited to upload the PDF of their poster and share it with the rest of the delegates through the conference APP. Further information will be sent to you by email very soon.</li></ul>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-1e167d1" data-block-id="1e167d1"><style>.stk-1e167d1 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">ECCB2022 Best Poster Awards</h2><div class="stk-block-heading__bottom-line"></div></div>



<ul><li>The ECCB2022 Best Poster Awards recognises the <strong>scientific quality and the communication effectiveness </strong>of the posters presented at ECCB2022. The <strong>Global Organisation for Bioinformatics Learning, Education and Training (GOBLET) </strong>will award the best posters selected by the Posters Committee.&nbsp;</li></ul>



<ul><li>The evaluation criteria considered by the PostersCommittee are:</li></ul>



<ol><li>Scientific quality and contribution of the results in computational biology and bioinformatics.</li><li>Visual communication effectiveness (visual attractiveness and structural clarity).</li><li>Data sources are open and clearly cited.</li><li>Engaging and well-structured presentation pitch.</li><li>Participants voting results.</li></ol>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-22fc178" data-block-id="22fc178"><style>.stk-22fc178 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Print shops</h2><div class="stk-block-heading__bottom-line"></div></div>



<p>Please note that ECCB2022 organisers are not offering printing services. However, posters can be printed at the following copy shops from Sitges, both are at 15 min by walk from Hotel Melià Sitges: </p>



<p><strong>Altraera</strong><br>Address: Av. Camí dels Capellans, 20, 08870 Sitges<br>Phone: +34 93 522 22 93<br>Opening hours: from Monday to Friday (between 9:00-18:00h)<br>Web: <a href="https://www.altraera.com/en/computer-shop-sitges/">https://www.altraera.com</a><br>Email: copisteria@altraera.com</p>



<p></p>



<p><strong>El pati</strong><br>Address: Carrer Illa de Cuba, 10, 08870 Sitges<br>Phone: +34 93 894 76 76<br>Opening hours: from Monday to Friday, between 9:30-13:30h and 16:30-20:00h; Saturday: between 10:00-13:30h<br>Email: el.pati.sitges@telefonica.net</p>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-b680d6d" data-block-id="b680d6d"><style>.stk-b680d6d .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Themes</h2><div class="stk-block-heading__bottom-line"></div></div>



<p>Posters will be clustered by theme:</p>



<ul><li>Data</li><li>Genes</li><li>Genomes</li><li>Proteins</li><li>Systems</li><li>Climate Crisis and Health</li><li>Applications</li><li>Training</li></ul>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-035a52d" data-block-id="035a52d"><style>.stk-035a52d .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">List of posters</h2><div class="stk-block-heading__bottom-line"></div></div>



<ul><li>The date and time of the poster session assigned to you, and the final submission reference number are indicated in the list below.</li></ul>



<ul><li>The final reference number of the poster identifies the poster board and the poster session day (for example, P001-T means poster panel 1 and poster session on Tuesday (T) and P032-M means poster panel 032 and poster session on Monday (M).</li></ul>



<ul><li>Posters are ordered by topic and then by title. To search for your poster, you need to press the key “Ctrl + F” and introduce your last name, abstract title or EasyChair reference code.</li></ul>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-fcb0ea2" data-block-id="fcb0ea2"><style>.stk-fcb0ea2{margin-top:50px !important}.stk-fcb0ea2 .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-8faf321" data-block-id="8faf321"><style>.stk-8faf321 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-8faf321 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-8faf321 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-8faf321 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-8faf321 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-8faf321 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-8faf321 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-527d292" data-block-id="527d292"><style>.stk-527d292 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-527d292 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-527d292 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-527d292 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-527d292 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-527d292 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-527d292 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-ff82369" data-block-id="ff82369"><style>.stk-ff82369 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-ff82369 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-ff82369 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-ff82369 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-ff82369 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-ff82369 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-ff82369 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-1373865" data-block-id="1373865"><style>.stk-1373865 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-1373865 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-1373865 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-1373865 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-1373865 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-1373865 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-1373865 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-a0d468c" data-block-id="a0d468c"><style>.stk-a0d468c .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-a0d468c .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-a0d468c .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-a0d468c .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-a0d468c .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-a0d468c .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-a0d468c .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-6cc2851" data-block-id="6cc2851"><style>.stk-6cc2851 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-6cc2851 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-6cc2851 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-6cc2851 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-6cc2851 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-6cc2851 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-6cc2851 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-400ba11" data-block-id="400ba11"><style>.stk-400ba11 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-400ba11 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-400ba11 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-400ba11 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-400ba11 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-400ba11 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-400ba11 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-1c0bc6b" data-block-id="1c0bc6b"><style>.stk-1c0bc6b .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-1c0bc6b .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-1c0bc6b .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-1c0bc6b .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-1c0bc6b .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-1c0bc6b .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-1c0bc6b .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-f5058ae" id="h-data" data-block-id="f5058ae"><style>.stk-f5058ae .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Data</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>EasyChair<br>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>489</td><td>P001-M</td><td>A benchmark of library-size bias in correlation for single-cell expression data</td><td>Suzanne Jin, Nacho Molina, Ionas Erb and Cedric Notredame</td></tr><tr><td>8780</td><td>P002-M</td><td>A graph-based knowledge base of multi-layer clinical information for the heart failure disease</td><td>Irina-Afrodita Balaur, Shaman Narayanasamy, Nikolaus Berndt, Muhammad Shoaib, Sarah Nordmeyer, Titus Kühne and Venkata Satagopam</td></tr><tr><td>9753</td><td>P003-M</td><td>A linear optimization approach for normalization of SARS-CoV-2 vaccination data</td><td>Simon Cyrani, Lukas Wenner, Ferdous Nasri and Bernhard Y Renard</td></tr><tr><td>3677</td><td>P004-M</td><td>A Named Entity Recognition Pipeline for Custom Anonymisation of Clinical Free-text Notes</td><td>Andreia Rogerio, Christopher Boustred, Anna Need and Francisco Azuaje</td></tr><tr><td>8585</td><td>P005-M</td><td>A simulation-based approach to normalize county-level vaccination data</td><td>Lukas Wenner, Simon Cyrani, Simon Knott, Ferdous Nasri and Bernhard Y Renard</td></tr><tr><td>9027</td><td>P006-M</td><td>A sustainable workflow for MPRA data analysis: MPRAsnakeflow</td><td>Pyaree Mohan Dash, Martin Kircher and Max Schubach</td></tr><tr><td>2652</td><td>P007-M</td><td>A Text Mining approach for retrieving gene::drug::cancer associations from full-text biomedical literature</td><td>Elissavet Zacharopoulou, Giorgos Skoufos, Spiros Tastsoglou and Artemis Hatzigeorgiou</td></tr><tr><td>6903</td><td>P008-M</td><td>A tissue specific post-translational modification (PTM) map of human proteome</td><td>Pathmanaban Ramasamy, Hanne Devos, Lennart Martens and Wim Vranken</td></tr><tr><td>9475</td><td>P009-M</td><td>Agnodice: indexing experimentally supported bacterial sRNA-RNA interactions</td><td>Vasiliki Kotsira, Giorgos Skoufos, Athanasios Alexiou, Filippos S. Kardaras, Maria Zioga, Spyros Tastsoglou, Theodosia Charitou, Zacharopoulou Elissavet, Nikos Perdikopanis and Artemis G. Hatzigeorgiou</td></tr><tr><td>3397</td><td>P010-M</td><td>AN INTEGRATIVE BIOINFORMATICS PIPELINE FOR THE ANALYSIS OF A COMPREHENSIVE NGS DIAGNOSTICS PAN-CANCER PANEL</td><td>Raúl Marín, Ania Alay, Sara Hijazo-Pechero, Ernest Nadal, Víctor Moreno and Xavier Solé</td></tr><tr><td>5437</td><td>P011-M</td><td>Analysis of phenotypes using ontology and word embedding</td><td>Canh Nguyen-Duc, Gregor Bucher and Jürgen Dönitz</td></tr><tr><td>9368</td><td>P012-M</td><td>Analyzing socio-demographic biases in cellphone mobility data used for COVID-19 contact predictions</td><td>Conrad Halle, Ferdous Nasri, Jeremias Dötterl, Steven Schulz, Sten Rüdiger and Bernhard Y Renard</td></tr><tr><td>8471</td><td>P013-M</td><td>Assessing SARS-CoV-2 evolution through the analysis of emerging mutations</td><td>Anastasios Mitsigkolas, Nikolaos Pechlivanis and Fotis Psomopoulos</td></tr><tr><td>9601</td><td>P014-M</td><td>ASTERICS: A Tool for the ExploRation and Integration of omiCS data</td><td>Élise Maigné, Céline Noirot, Jérôme Mariette, Yaa Adu Kesewaah, Sébastien Déjean, Camille Guilmineau, Julien Henry, Arielle Krebs, Laurence Liaubet, Fanny Mathevet, Hyphen-Stat, Christine Gaspin and Nathalie Vialaneix</td></tr><tr><td>2335</td><td>P015-M</td><td>Benchmarking methods for the identification of mislabeled data in genomics</td><td>Lusine Nazaretyan, Martin Kircher and Ulf Leser</td></tr><tr><td>9367</td><td>P016-M</td><td>BEstimate: a python tool for in silico analysis of potential edits with CRISPR base editors</td><td>Cansu Dincer, Matthew Coelho and Mathew Garnett</td></tr><tr><td>6790</td><td>P017-M</td><td>BLOOD LIPID PROFILE AS AN INDICATOR OF PSYCHIATRIC DISORDERS</td><td>Anna Tkachev, Anna Morozova, Yana Zorkina, Alexander Reznik, Denis Andreyuk, Elena Stekolshchikova, Nickolay Anikanov, Georgiy Kostyuk and Philipp Khaitovich</td></tr><tr><td>5598</td><td>P018-M</td><td>Breast Cancer Detection in UKBiobank Data Using Mutation Based Gene Weighting and Deep Learning</td><td>Nuriye Özlem Özcan Şimşek, Fikret Gurgen and Arzucan Ozgur</td></tr><tr><td>887</td><td>P019-M</td><td>CABiNet – Biclustering and joint cell-gene visualization of single-cell transcriptomics</td><td>Yan Zhao, Clemens Kohl and Martin Vingron</td></tr><tr><td>1222</td><td>P020-M</td><td>Can we trust probabilities in deep drug activity models? A comparative calibration study.</td><td>Hannah Rosa Friesacher, Lewis Mervin H, Ola Engkvist, Yves Moreau and Adam Arany</td></tr><tr><td>7934</td><td>P021-M</td><td>ClustAssess: tools for assessing the robustness of single-cell clustering</td><td>Irina Mohorianu, Arash Shahsavari and Andi Munteanu</td></tr><tr><td>9217</td><td>P022-M</td><td>Concordant or not? Performance and concordance rates analysis of ten prediction algorithms on clinically relevant variants from the BRCA1 and BRCA2 genes.</td><td>Erda Qorri, Bertalan Takács, Alexandra Gráf, Márton Z. Enyedi, Lajos Pintér, Ernő Kiss and Lajos Haracska</td></tr><tr><td>6651</td><td>P023-M</td><td>Covid-19 Mutation Incidence Forecasting using Spatio-Temporal Graph Neural Networks</td><td>Larissa Hoffaeller, Athar Khodabakhsh and Bernhard Renard</td></tr><tr><td>9622</td><td>P024-M</td><td>&#8211;</td><td>NOT REGISTERED</td></tr><tr><td>1477</td><td>P025-M</td><td>Cross-Mapper for Structural Domains (CroMaSt): A workflow for domain family curation through cross-mapping of structural instances between protein domain databases</td><td>Hrishikesh Dhondge, Isaure Chauvot de Beauchêne and Marie-Dominique Devignes</td></tr><tr><td>6701</td><td>P026-M</td><td>Data and Computing Platform to facilitate NCER-PD (National Center of Excellence in Research in Parkinson&#8217;s disease) project</td><td>Rajesh Rawal, Carlos Vega, Soumyabrata Ghosh, Sascha Herzinger, Kirsten Roomp, Peter Banda, Piotr Gawron, Rejko Krüger, Jens C. Schwamborn and Venkata P. Satagopam</td></tr><tr><td>7111</td><td>P027-M</td><td>Data integration analysis for profiling host-microbiome interactions in non-responsive Celiac disease patients.</td><td>Laura Judith Marcos-Zambrano, Blanca LaCruz, Viviana Loria-Kohen, Ana Ramirez de Molina and Enrique Carrillo de Santa Pau</td></tr><tr><td>516</td><td>P028-M</td><td>Data Management in Galaxy</td><td>Beatriz Serrano-Solano and Bjoern Gruening</td></tr><tr><td>5126</td><td>P029-M</td><td>Dereplication with DEREPer: a tool for High Throughput Metabolomics with LC-MS/MS</td><td>Simone Zorzan, Kjell Sergeant and Sophie Charton</td></tr><tr><td>8941</td><td>P030-M</td><td>Designing a Cloud and HPC Based Model&amp;Simulation platform to Investigate Diseases Mechanisms</td><td>Maria Paola Ferri, Laia Codó Tarraubella, Josep Lluis Gelpi Buchaca, Dimitrios Lialios and Francesco Gualdi</td></tr><tr><td>6694</td><td>P031-M</td><td>DiASPora: A Natural Language Processing platform for automated extraction of microbial phenotype information from scientific articles</td><td>Arindam Halder, Ziyad Ziyad and Konrad U Förstner</td></tr><tr><td>3446</td><td>P032-M</td><td>Digital Heart (DHART) Research: A multi-omics resource portal for the cardiac community</td><td>Etienne Boileau, Christoph Dieterich, Harald Wilhelmi and Enio Gjerga</td></tr><tr><td>3069</td><td>P033-M</td><td>digitalDLSorteR: An R package to deconvolute bulk RNA-Seq from scRNA-Seq data</td><td>Diego Mañanes Cayero, Carlos Torroja, Carlos Relaño Ruperez, Inés Rivero García and Fátima Sánchez-Cabo</td></tr><tr><td>2820</td><td>P034-M</td><td>Disentangling spatial and non-spatial transcriptomic signals: A modified variational auto encoder model</td><td>Loïc Chadoutaud, Andrei Zinovyev and Emmanuel Barillot</td></tr><tr><td>3428</td><td>P035-M</td><td>Efficient and rapid genome profiling of SARS-CoV-2 by covSonar</td><td>Alice Wittig, Kunaphas Kongkitimanon and Stephan Fuchs</td></tr><tr><td>3176</td><td>P036-M</td><td>Evaluation of the bioinformatics tumour-control approach in a next-sequencing panel in pediatric leukemia</td><td>Beatriz Ruz-Caracuel, Carlos Rodríguez-Antolín, Carmen Rodríguez Jiménez, Sonia Rodríguez Novoa, Victoria Eugenia Fernández Montaño, Victoria Gómez del Pozo, Inmaculada Ibáñez, Javier de Castro, Adela Escudero López and Antonio Pérez-Martínez</td></tr><tr><td>2313</td><td>P037-M</td><td>Event segmentation in highly modified Nanopore direct RNA sequencing</td><td>Wiep van der Toorn, Patrick Bohn, Liuwei Wang, Redmond Smyth and Max von Kleist</td></tr><tr><td>2776</td><td>P038-M</td><td>Exploiting Pretrained Biochemical Language Models for Targeted Drug Design</td><td>Gökçe Uludoğan, Elif Olmez, Nilgün Karalı, Kutlu Ö. Ülgen and Arzucan Ozgur</td></tr><tr><td>2008</td><td>P039-M</td><td>Exploring Chemical Diversity with the Chemical Checker</td><td>Martino Bertoni and Patrick Aloy</td></tr><tr><td>4497</td><td>P040-M</td><td>Exploring the development of innate immune responses to infection in early life: A call for data.</td><td>Mary McCabe, Dr Guillermo Lopez Campos, Dr Helen Groves and Professor Ultan Power</td></tr><tr><td>5250</td><td>P041-M</td><td>Extending the potential of plasmiR in the cancer microRNA biomarker landscape</td><td>Marios Miliotis, Dimitris Grigoriadis, Spyros Tastsoglou, Nikos Perdikopanis, Athanasios Alexiou and Artemis Hatzigeorgiou</td></tr><tr><td>1824</td><td>P042-M</td><td>FAIR by Design &#8211; FAIRification Strategy in Large Data Centric Projects</td><td>Soumyabrata Ghosh, Irina-Afrodita Balaur, Basile Rommes, Kavita Rege, Hanna Ćwiek-Kupczyńska, Wei Gu and Venkata Pardhasaradhi Satagopam</td></tr><tr><td>3133</td><td>P043-M</td><td>Far tail approximation of non-standard test statistic distribution</td><td>Krzysztof Mnich, Wojciech Lesiński and Witold Rudnicki</td></tr><tr><td>9537</td><td>P044-M</td><td>Forest-Guided Clustering &#8211; Explainability for Random Forest Models</td><td>Lisa Barros de Andrade E Sousa, Dominik Thalmeier, Helena Pelin and Marie Piraud</td></tr><tr><td>6682</td><td>P045-M</td><td>From climate defined ecological niches to microbiome diversity and intra-community synergies</td><td>Dagmara Błaszczyk, Witold Wydmański, Krzysztof Mnich, Valentyn Bezshapkin, Michał B. Kowalski, Alina Frolova, Witold Rudnicki and Paweł P. Łabaj</td></tr><tr><td>5192</td><td>P046-M</td><td>Gene co-occurrence analysis for prediction of unknown gene function</td><td>Bartosz Baranowski and Krzysztof Pawłowski</td></tr><tr><td>3657</td><td>P047-M</td><td>GenOptics: An intuitive platform of visual analytics for integrative analysis of large-scale multi-omics data</td><td>Konstantinos A. Kyritsis, Nikolaos Pechlivanis, Angeliki Magklara, Anastasia Kougioumtzi, Panagiotis Dafopoulos, Eleni Ntzioni, Efstathia Tsarouchi, Dimitris Sakellariou, Marios Kotoulas, Stratos Arampatzis, Panagiotis Chatzikamaris, Elisavet Siomou, Maria Argyraki, Dimitrios Botskaris, Iannis Talianidis, Fotis Psomopoulos and Vasileios Vasileiou</td></tr><tr><td>7430</td><td>P048-M</td><td>GWAS Central: a resource for the discovery and comparison of summary-level genome-wide association study data</td><td>Tim Beck, Tom Shorter, Thomas Rowlands and Anthony Brookes</td></tr><tr><td>4582</td><td>P049-M</td><td>IDEAS: Integrative and Differential Expression and Alternative Splicing Analysis</td><td>Leonie Pohl, Armin Hadziahmetovic, Alexandra Schubö and Ralf Zimmer</td></tr><tr><td>7425</td><td>P050-M</td><td>Identifying Viral miRNAs with Text Mining at the example of SARS-CoV-2 (VIM-TM)</td><td>Markus Joppich, Armin Hadziahmetovic, Alexandra Schubö and Ralf Zimmer</td></tr><tr><td>735</td><td>P051-M</td><td>Imaging Data: Adding a new type of annotations to 3DBionotes-WS</td><td>Carolina Simón Guerrero, Jose Ramon Macias, Erney Ramírez-Aportela, Jose Luis Vilas Prieto, Marta Martinez Gonzalez, Carlos Oscar Sanchez Sorzano and Jose Maria Carazo</td></tr><tr><td>2650</td><td>P052-M</td><td>Impact of preprocessing in data integration of single-cell RNA-seq data</td><td>Youngjun Park and Anne-Christin Hauschild</td></tr><tr><td>5575</td><td>P053-M</td><td>IMPaCT-Data&nbsp;Biomedical&nbsp;Cloud: An initial iteration for a&nbsp;federated virtual computing environment in the context of Precision Medicine&nbsp;in Spain.</td><td>María Chavero-Díez, Jose María Fernández, Laia Codó, Lidia Lopez, Salvador Capella-Gutierrez, Josep Lluis Gelpi and Impact-Data Wp Two</td></tr><tr><td>4179</td><td>P001-T</td><td>Implementing best practices for setting a bioinformatics core facility</td><td>Pau Marc Muñoz Torres, Merce Alemany-Chavarria and Lara Nonell</td></tr><tr><td>668</td><td>P002-T</td><td>Indication expansion via geometric scattering-based knowledge graph embedding</td><td>Dhananjay Bhaskar, Sergio Picart-Armada and Smita Krishnaswamy</td></tr><tr><td>4669</td><td>P003-T</td><td>InSoLiTo: The research software graph-based network from OpenEBench</td><td>Sergi Aguiló-Castillo, José M. Fernández, Josep Ll. Gelpí and Salvador Capella-Gutierrez</td></tr><tr><td>7551</td><td>P004-T</td><td>Integration of multi-omics data using graph neural networks to identify and contextualize biomarker genes for psychiatric disorders</td><td>Svitlana Oleshko, Francesc Fernández-Albert, Matthias Heinig, Sergio Picart-Armada and Annalisa Marsico</td></tr><tr><td>4287</td><td>P005-T</td><td>Integrative network analysis interweaves the missing links in cardiomyopathy diseasome</td><td>Pankaj Chauhan and Ramanathan Sowdhamini</td></tr><tr><td>9486</td><td>P006-T</td><td>Introducing the National Research Data Infrastructure for the Research of Microbiota (NFDI4Microbiota)</td><td>Adrian Fritz, Anke Becker, Peer Bork, Thomas Clavel, Ulisses Nunes da Rocha, Konrad U. Förstner, Alexander Goesmann, Barbara Götz, Manja Marz, Jörg Overmann, Carmen Paulmann, Kristin Sauerland, Alexander Sczyrba, Jens Stoye and Alice C. McHardy</td></tr><tr><td>3603</td><td>P007-T</td><td>It’s FLAN time! Summing feature-wise latent representations for interpretability</td><td>An-Phi Nguyen, Stefania Vasilaki, Maria Rodriguez Martinez and Alice Driessen</td></tr><tr><td>9983</td><td>P008-T</td><td>Latent Variable Random Forest for Enhanced Feature Importance</td><td>Melpomeni Kasapi, Kexin Xu, James S. Ware, Declan P. O&#8217;Regan, Timothy M.D. Ebbels and Joram M. Posma</td></tr><tr><td>2496</td><td>P009-T</td><td>LCRBert: word embedding to support detection of articles containing low-complexity regions</td><td>Sylwia Szymańska and Aleksandra Gruca</td></tr><tr><td>6322</td><td>P010-T</td><td>Light-weight alignment enables fast and accurate metagenome and metatranscriptome quantification</td><td>Shuba Varshini Alampalli, Eva Weiss and Lars Barquist</td></tr><tr><td>1524</td><td>P011-T</td><td>Low Complexity Regions in kinases</td><td>Joanna Ziemska-Legięcka, Aleksandra Gruca and Marcin Grynberg</td></tr><tr><td>2618</td><td>P012-T</td><td>Machine learning for extraction of biochemical reactions from the scientific literature​</td><td>Blanca Cabrera Gil, Anne Morgat, Venkatesh Muthukrishnan, Elisabeth Coudert, Kristian Axelsen, Nicole Redaschi and Alan Bridge</td></tr><tr><td>2392</td><td>P013-T</td><td>Machine learning identifies weak prognostic signal in colorectal polyp&#8217;s transcriptome</td><td>Russell Hung, Simon Fisher, Ditte Andersen, Gerard Lynch, Noori Maka, Jennifer Hay, Jakub Jawny, William Sloan, Stephen McSorley, Joanne Edwards and Ian Poole</td></tr><tr><td>3126</td><td>P014-T</td><td>Microbial co-occurrence network reveals climate and geographic patterns for soil diversity on the planet</td><td>Nikolaos Pechlivanis, George Karakatsoulis, Stefanos Sgardelis, Ilias Kappas and Fotis Psomopoulos</td></tr><tr><td>9758</td><td>P015-T</td><td>Mitigating failure modes of molecular optimization using expert constraints and ensembles</td><td>Adam Arany, Jaak Simm, Natalia Dyubankova, Jonas Verhoeven, Martijn Oldenhof and Yves Moreau</td></tr><tr><td>4336</td><td>P016-T</td><td>MTLSurv: Predicting Breast Cancer Patients’ Survival with Multimodal Deep Neural Networks and Modality-Specific Transfer Learning</td><td>Sören Richard Stahlschmidt, Göran Falkman, Benjamin Ulfenborg and Jane Synnergren</td></tr><tr><td>391</td><td>P017-T</td><td>Multi-omics lung cancer subtyping by machine learning informs biomarker-guided drug development</td><td>Sven-Eric Schelhorn</td></tr><tr><td>1284</td><td>P018-T</td><td>Namco: A microbiome explorer</td><td>Monica Matchado, Alexander Dietrich, Maximilian Zwiebel, Benjamin Ölke, Michael Lauber, Ilias Lagkouvardos, Beate Brandl, Thomas Skurk, Jan Baumbach, Hans Hauner, Dirk Haller, Sandra Reitmeier and Markus List</td></tr><tr><td>344</td><td>P019-T</td><td>nf-core as the standard for BovReg reference pipelines</td><td>Jose Espinosa-Carrasco, Björn E. Langer, Philip A. Ewels, Harshil Patel, Peter Harrison and Cedric Notredame</td></tr><tr><td>3007</td><td>P020-T</td><td>Nightingale, visualizing biology on the web following standards</td><td>Aurélien Luciani, Gustavo Salazar, Daniel Rice, Swaathi Kandasaamy and Maria Martin</td></tr><tr><td>8104</td><td>P021-T</td><td>Omics data integration with Correlation guided Network Integration (CoNI)</td><td>José Manuel Monroy Kuhn, Sonja C. Schriever, Viktorian Miok, Andreas Peter, Martin Heni, Paul T. Pfluger and Dominik Lutter</td></tr><tr><td>4024</td><td>P022-T</td><td>Panomicon, allowing heterogeneous multi-omics analysis on the web</td><td>Rodolfo Allendes, Johan T. Nystroem-Persson, Yuji Kosugi, Kenji Mizuguchi and Yayoi Natsume-Kitatani</td></tr><tr><td>3840</td><td>P023-T</td><td>PascalX: Detecting shared genes and pathways between pairs of GWAS traits</td><td>Daniel Krefl and Sven Bergmann</td></tr><tr><td>7557</td><td>P024-T</td><td>Pilot: Making Research Data FAIR</td><td>Christina Yung, Moyez Dharsee, Fan Dong, Kenneth Evans, Susan Evans, Tom Gee, Mojib Javadi, Stephane Pollentier and Shahab Shahnazari</td></tr><tr><td>9515</td><td>P025-T</td><td>Predicting Gene Dependencies using Machine Learning from Proteomic Data</td><td>Robert Nkwo, Shirin Khorsandi and Pedro Cutillas</td></tr><tr><td>4977</td><td>P026-T</td><td>PROMPT: Toward PRecisiOn Medicine for the Prediction of Treatment response in major depressive disorder through stratification of combined clinical and -omics</td><td>Júlia Perera-Bel, Alessandra Minelli, Johannes Zang, Britta Kelch, María Martínez de Lagrán, Mara Dierssen, Bernardo Carpiniello, Massimo Gennarelli, Filip Rybakowski, Marie-Claude Potier, Ferran Sanz and Bernhard T Baune</td></tr><tr><td>6121</td><td>P027-T</td><td>RDMkit: the ELIXIR Research Data Management Kit</td><td>Laura Portell Silva and Munazah Andrabi</td></tr><tr><td>514</td><td>P028-T</td><td>Recommendations on training VAEs on TCGA transcriptome data</td><td>Mostafa Eltager, Tamim Abdelaal, Mohammed Charrout, Ahmed Mahfouz, Marcel Reinders and Stavros Makrodimitris</td></tr><tr><td>1565</td><td>P029-T</td><td>Rhea, a FAIR resource of expert curated biochemical and transport reaction data.</td><td>Parit Bansal, Anne Morgat, Kristian Axelsen, Venkatesh Muthukrishnan, Elisabeth Coudert, Lucila Aimo, Nevila Hyka-Nouspikel, Elisabeth Gasteiger, Arnaud Kerhornou, Teresa Neto, Monica Pozzato, Marie-Claude Blatter, Nicole Redaschi and Alan Bridge</td></tr><tr><td>415</td><td>P030-T</td><td>ROLE OF RNA-SEQ PREPROCESSING STEPS IN MOLECULAR SUBTYPE CLASSIFICATION IN MUSCLE-INVASIVE BLADDER CANCER</td><td>Ariadna Acedo-Terrades, Júlia Perera-Bel, Joaquim Bellmunt and Lara Nonell</td></tr><tr><td>7249</td><td>P031-T</td><td>scDAVIS: Single-cell Data Analysis and VISualization</td><td>Carlos Torroja, Daniel Jimenez Carretero, Jon E. Sicilia, Juan L. Onieva Zafra, Jorge G. Garcia Gomez, Celia Centeno Tundidor and Fatima Sanchez-Cabo</td></tr><tr><td>4681</td><td>P032-T</td><td>Short read de novo transcriptome assembly by means of ant colony optimization</td><td>Karl Johan Westrin, Olof Emanuelsson and Henric Zazzi</td></tr><tr><td>5590</td><td>P033-T</td><td>Single-cell transcriptomic analyses reveal distinct B cell subsets and their class-switch recombination dynamics</td><td>Joseph Ng, Alexander Stewart, Deborah Dunn-Walters and Franca Fraternali</td></tr><tr><td>1319</td><td>P034-T</td><td>Software Observatory</td><td>Eva Martin del Pico, Salvador Capella-Gutiérrez and Josep Ll Gelpi</td></tr><tr><td>8028</td><td>P035-T</td><td>SQANTI3: how to curate a Long Read-defined transcriptome in 3 steps.</td><td>Francisco J. Pardo-Palacios, Ángeles Arzalluz Luque and Ana Conesa</td></tr><tr><td>4554</td><td>P036-T</td><td>SQANTISIM: a simulator of controlled novelty and degradation of transcripts sequenced by long-reads</td><td>Jorge Mestre-Tomás, Francisco J. Pardo-Palacios and Ana Conesa</td></tr><tr><td>8216</td><td>P037-T</td><td>Standardized analysis of complex RNAseq experiments using Snakemake</td><td>Christian M. Heyer, Ashik Ahmed Abdul Pari, Hellmut G. Augstin and Matthias Schlesner</td></tr><tr><td>9659</td><td>P038-T</td><td>Systematic benchmarking and error evaluation of basecallers for Nanopore Direct RNA-seq</td><td>Wang Liu-Wei, Patrick Bohn, Wiep van der Toorn, Redmond Smyth and Max von Kleist</td></tr><tr><td>1507</td><td>P039-T</td><td>Temporal changes in microbiome composition after FMT in subject with Clostridioides difficile infection: a network perspective</td><td>Marco Cappellato, Massimo Bellato, Giacomo Baruzzo, Sonia Facchin, Luisa Barzon, Valeria Besutti, Paola Brun, Simone Del Favero, Luca Schenato, Edoardo Vincenzo Savarino, Ignazio Castagliuolo and Barbara Di Camillo</td></tr><tr><td>9141</td><td>P040-T</td><td>Text mining resources for extracting genetic and phenotypic data from scientific publication full-texts and tables</td><td>Thomas Rowlands, Janet Pinero, Pablo Accuosto, Tom Shorter, Joram Posma, Laura Furlong and Tim Beck</td></tr><tr><td>5144</td><td>P041-T</td><td>The Wisdom of the Crowd: comparing individual and aggregated solutions for metagenomics-based inflammatory bowel disease diagnostics in the scope of the sbv IMPROVER MEDIC challenge</td><td>Lusine Khachatryan, Carine Poussin, Yang Xiang, Adrian Stan, James Battey, Giuseppe Lo Sasso, Stephanie Boue, Nicolas Sierro, Nikolai Ivanov and Julia Hoeng</td></tr><tr><td>6028</td><td>P042-T</td><td>tolKG: A Linked Knowledge Graph using NLP for Immune Regulatory Cells &nbsp;</td><td>Ayesha Sahar, Phillip Lord, Catharien Hilkens, Michael Hughes and Lee Harland</td></tr><tr><td>3534</td><td>P043-T</td><td>Towards a knowledge graph for pre-/probiotics and microbiota-gut-brain axis diseases</td><td>Ting Liu, Gongjin Lan, K. Anton Feenstra, Zhisheng Huang and Jaap Heringa</td></tr><tr><td>2669</td><td>P044-T</td><td>Towards a more inductive world: A review on graph embedding methods for drug repurposing approaches</td><td>Jesús De la Fuente, Guillermo Serrano, Uxía Veleiro, Mikel Casals, Oier Azurmendi-Senar, Antonio Pineda-Lucena, Idoia Ochoa, Silve Vicent, Olivier Gevaert and Mikel Hernaez</td></tr><tr><td>4041</td><td>P045-T</td><td>Towards interpretable machine learning applications in human microbiome via information theory-guided feature selection</td><td>Valentyn Bezshapkin, Witold Wydmański, Krzysztof Mnich, Michał Kowalski, Dagmara Błaszczyk, Tomasz Kościółek, Witold Rudnicki and Paweł Łabaj</td></tr><tr><td>7217</td><td>P046-T</td><td>Transmorph: A computational framework for dataset integration</td><td>Aziz Fouché, Loïc Chadoutaud and Andrei Zinovyev</td></tr><tr><td>9854</td><td>P047-T</td><td>tRNAstudio: facilitating the study of human tRNA-seq datasets</td><td>Marina Murillo Recio, Adrian Gabriel Torres and Lluís Ribas de Pouplana</td></tr><tr><td>214</td><td>P048-T</td><td>Using Statistical and Machine Learning Models to Identify Features Contributing to Class Switch Recombination</td><td>Lutecia Servius, Joseph Chi-Fung Ng, Davide Pigoli and Franca Fraternali</td></tr><tr><td>3523</td><td>P049-T</td><td>VIROMEdash: Global Virome Sequence Metadata Visualizer</td><td>Eyyüb Ünlü and Mohammad A. Khan</td></tr><tr><td>5367</td><td>P050-T</td><td>WorkflowHub: a FAIR registry for workflows</td><td>Carole Goble, Finn Bacall, Stian Soiland-Reyes, Stuart Owen, Alan Williams, Ignacio Eguinoa, Bert Droesbeke, Hervé Ménager, Laura Rodríguez Navas, José Maria Fernández González, Salvador Capella-Gutierrez, Michael R. Crusoe, Björn Grüning, Simone Leo, Luca Pireddu, Johan Gustafsson, Phil Ewels and Frederik Coppens</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-a51c0a9" data-block-id="a51c0a9"><style>.stk-a51c0a9{margin-top:50px !important}.stk-a51c0a9 .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-4c0fd41" data-block-id="4c0fd41"><style>.stk-4c0fd41 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-4c0fd41 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-4c0fd41 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-4c0fd41 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-4c0fd41 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-4c0fd41 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-4c0fd41 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-a865503" data-block-id="a865503"><style>.stk-a865503 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-a865503 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-a865503 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-a865503 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-a865503 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-a865503 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-a865503 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-5730896" data-block-id="5730896"><style>.stk-5730896 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-5730896 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-5730896 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-5730896 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-5730896 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-5730896 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-5730896 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-04c39ba" data-block-id="04c39ba"><style>.stk-04c39ba .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-04c39ba .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-04c39ba .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-04c39ba .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-04c39ba .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-04c39ba .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-04c39ba .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-6469365" data-block-id="6469365"><style>.stk-6469365 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-6469365 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-6469365 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-6469365 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-6469365 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-6469365 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-6469365 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-2a3125b" data-block-id="2a3125b"><style>.stk-2a3125b .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-2a3125b .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-2a3125b .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-2a3125b .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-2a3125b .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-2a3125b .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-2a3125b .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-641b7d0" data-block-id="641b7d0"><style>.stk-641b7d0 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-641b7d0 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-641b7d0 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-641b7d0 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-641b7d0 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-641b7d0 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-641b7d0 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-a4749a2" data-block-id="a4749a2"><style>.stk-a4749a2 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-a4749a2 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-a4749a2 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-a4749a2 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-a4749a2 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-a4749a2 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-a4749a2 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-4ceda4a" id="h-genes" data-block-id="4ceda4a"><style>.stk-4ceda4a .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Genes</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>4005</td><td>P054-M</td><td>A gene expression signature for survival and risk prediction of Breast Cancer that improves the signatures used in clinical genomic platforms</td><td>Santiago Bueno-Fortes, Alberto Berral-Gonzalez, Jose M Sanchez-Santos, Manuel Martin-Merino and Javier De Las Rivas</td></tr><tr><td>4414</td><td>P055-M</td><td>A knowledge graph approach for interpretable prediction of pathogenic genetic interactions</td><td>Alexandre Renaux, Chloé Terwagne, Michael Cochez, Ilaria Tiddi, Ann Nowé and Tom Lenaerts</td></tr><tr><td>1633</td><td>P056-M</td><td>A multi cohort analysis workflow for RNA-seq data</td><td>Xinhui Wang, Soumyabrata Ghosh and Venkata Satagopam</td></tr><tr><td>1429</td><td>P057-M</td><td>A New Method for Discovering Drivers Made of Epistatic Gene Pairs in Cancer Tumors</td><td>Jairo Rocha, Jaume Sastre Tomas, Victor Asensio-Landa, Emilia Amengual-Cladera, Jessica Hernandez-Rodriguez, Damià Heine-Suñer and Emidio Capriotti</td></tr><tr><td>571</td><td>P058-M</td><td>A spatial and single-cell transcriptomics approach to investigate scleroderma in human fibroblasts.</td><td>Till Baar, Ann-Helen Rosendahl, Katrin Schönborn, Niklas Kleinenkuhnen, Beate Eckes, Pia Moinzadeh, Thomas Krieg and Achim Tresch</td></tr><tr><td>9492</td><td>P059-M</td><td>A state-of-the-art and easy-to-use Python framework for plant phenotype prediction</td><td>Florian Haselbeck, Maura John and Dominik Grimm</td></tr><tr><td>3485</td><td>P060-M</td><td>acorde unravels functionally interpretable networks of isoform co-usage from single cell data</td><td>Ángeles Arzalluz-Luque, Sonia Tarazona and Ana Conesa</td></tr><tr><td>5350</td><td>P061-M</td><td>Allele-specific copy-number based deconvolution of bulk tumour RNA sequencing data from the TRACERx study</td><td>Carla Castignani, Jonas Demeulemeester, Elizabeth Larose Cadieux, Robert E. Hynds, Stefan C. Dentro, Tracerx Consortium, Charles Swanton and Peter Van Loo</td></tr><tr><td>4791</td><td>P062-M</td><td>Analysis of differential cellular communication from single cell RNA-seq data with scSeqComm</td><td>Giulia Cesaro, Giacomo Baruzzo and Barbara Di Camillo</td></tr><tr><td>8446</td><td>P063-M</td><td>Analysis of transposable elements in R and Bioconductor with atena</td><td>Beatriz Calvo-Serra and Robert Castelo</td></tr><tr><td>923</td><td>P064-M</td><td>APPRIS: selecting functionally important isoforms</td><td>Daniel Cerdán-Vélez, Jose Manuel Rodríguez, Fernando Pozo, Tomás Di Domenico, Jesús Vázquez and Michael Tress</td></tr><tr><td>7258</td><td>P065-M</td><td>BamQuery: A new proteogenomic tool to explore the Immunopeptidome</td><td>Maria Virginia Ruiz Cuevas, Marie-Pierre Hardy, Anca Apavaloaei, Sebastien Lemieux, Claude Perreault and Gregory Ehx</td></tr><tr><td>9004</td><td>P066-M</td><td>Benchmarking methods for detecting differential states between conditions from multi-subject single-cell RNA-seq data</td><td>Sini Junttila, Johannes Smolander and Laura Elo</td></tr><tr><td>3777</td><td>P067-M</td><td>Beyondcell 2.0: dissecting therapeutic heterogeneity in spatial single-cell RNA-seq data</td><td>María José Jiménez-Santos, Coral Fustero-Torre, Santiago García-Martín, Carlos Carretero-Puche, Luis García-Jimeno, Tomás Di Domenico, Gonzalo Gómez-López and Fátima Al-Shahrour</td></tr><tr><td>8338</td><td>P068-M</td><td>Cancer signatures for reproducible gene expression analysis data: the computational way to achieve precision medicine</td><td>Stefania Pirrotta, Laura Masatti, Fabiola Pedrini, Chiara Romualdi and Enrica Calura</td></tr><tr><td>7003</td><td>P069-M</td><td>Cell type specificity of Hi-C interaction network topology and gene expression patterns</td><td>Juris Viksna, Lelde Lace, Gatis Melkus, Peteris Rucevskis, Edgars Celms, Sandra Silina and Andrejs Sizovs</td></tr><tr><td>7568</td><td>P070-M</td><td>Cell type-specific gene co-expression modules and expression signatures define tumor heterogeneity in melanoma patients</td><td>Lars Bosshard, Franziska Singer and Michael Prummer</td></tr><tr><td>8917</td><td>P071-M</td><td>Comparative study of dynamic changes in gene expression profiles induced by PPARa ligands</td><td>Yayoi Natsume-Kitatani, Ken-Ichi Aisaki, Satoshi Kitajima and Jun Kanno</td></tr><tr><td>3318</td><td>P072-M</td><td>Comparison of Deconvolution Techniques for Spatially Resolved Transcriptomics Data</td><td>Carolin Walter, Sarah Sandmann, Ada Anike Pohlmann, Alina Sophie Disch, Kornelius Kerl and Julian Varghese</td></tr><tr><td>5089</td><td>P073-M</td><td>Computational analysis of differential splicing and transcript alternations in severe COVID-19 infection</td><td>Sunanda Biswas Mukherjee and Milana Frenkel-Morgenstern</td></tr><tr><td>7921</td><td>P074-M</td><td>Computational Mapping of the Human-SARS-CoV-2 Protein-RNA Interactome</td><td>Marc Horlacher, Svitlana Oleshko, Yue Hu, Giulia Cantini, Patrick Schinke, Mahsa Ghanbari, Ernesto Elorduy Vergara, Florian Bittner, Nikola Mueller, Uwe Ohler, Lambert Moyon and Annalisa Marsico</td></tr><tr><td>2063</td><td>P075-M</td><td>Computational method prioritising gene-specific cis-regulatory elements reveals new aspects of transcriptional regulation in B cells</td><td>Amber Emmett, Matthew Care, Amel Saadi, Gina Doody, Reuben Tooze and David Westhead</td></tr><tr><td>6206</td><td>P076-M</td><td>consICA: a package for reference-free deconvolution reveals oncogenic processes in “omics” data and highlights the benefits of multimodal data analysis for patient stratification</td><td>Maryna Chepeleva, Tony Kaoma, Aliaksandra Kakoichankava, Yue Zhang, Reka Toth and Petr Nazarov</td></tr><tr><td>9701</td><td>P077-M</td><td>Context-Aware Transcript Discovery and Quantification from Long Read RNA-Seq data with Bambu</td><td>Ying Chen, Andre Sim, Yuk Kei Wan, Keith Yeo, Michael Love and Jonathan Goeke</td></tr><tr><td>9675</td><td>P078-M</td><td>Curare and GenExVis: A toolkit for analyzing and visualizing RNA-Seq data</td><td>Patrick Blumenkamp, Raphael Müller and Alexander Goesmann</td></tr><tr><td>5308</td><td>P079-M</td><td>De novo gene evolution in bacteria: a case study of taxonomically restricted genes in Bacillus</td><td>Wojciech Karlowski, Deepti Varshney and Andrzej Zielezinski</td></tr><tr><td>2017</td><td>P080-M</td><td>decoupleR: ensemble of computational methods to infer biological activities from omics data</td><td>Pau Badia i Mompel and Julio Saez-Rodriguez</td></tr><tr><td>2421</td><td>P081-M</td><td>Diana &#8211; ROOT: A categorization tool related to the Region Of Origin biotype.</td><td>Nikos Perdikopanis, Georgios Georgakilas and Artemis Hatzigeorgiou</td></tr><tr><td>8223</td><td>P082-M</td><td>Differential Abundance analysis of microbiome data: which tool and how to choose it?</td><td>Marco Cappellato, Giacomo Baruzzo and Barbara Di Camillo</td></tr><tr><td>6704</td><td>P083-M</td><td>Disentangling Cellular Heterogeneity with Multimodal single-cell Integration</td><td>Jules Samaran, Gabriel Peyré and Laura Cantini</td></tr><tr><td>4897</td><td>P084-M</td><td>Dissecting the impact of genetic variants on transcriptional and post-transcriptional gene regulation</td><td>Anneke Brümmer and Sven Bergmann</td></tr><tr><td>4266</td><td>P085-M</td><td>Diving into tumor heterogeneity</td><td>Marina Mendiburu-Eliçabe, Adrián Blanco-Gomez, Diego Alonso-Lopez, Roberto Corchado-Cobos, Natalia García-Sancha, Sonia Castillo-Lluva, Andrés Castellanos-Martín, María del Mar Sáez-Freire, Julie Milena Galvis-Jiménez, Alejandro Jiménez-Navas, Manuel Jesús Pérez-Baena, Asunción García-Sánchez, María Isidoro Martín, Martin Pérez-Andrés, Alberto Orfao, Jian-Hua Mao, Javier De Las Rivas and Jesús Pérez Losada</td></tr><tr><td>785</td><td>P086-M</td><td>EmbedPVP: Prioritizing causative variants by integrating functional embedding and biological annotations for genes</td><td>Azza Althagafi and Robert Hoehndorf</td></tr><tr><td>5199</td><td>P087-M</td><td>EmpiReS: Differential Analysis of Gene Expression and Alternative Splicing</td><td>Gergely Csaba, Evi Berchtold, Armin Hadziahmetovic, Markus Gruber, Constantin Ammar and Ralf Zimmer</td></tr><tr><td>776</td><td>P088-M</td><td>EvolclustDB: exploring eukaryotic gene clusters with evolutionary conserved synteny.</td><td>Uciel Chorostecki, Marina Marcet-Houben, Ismael Collado-Cala, Andrés Garisoain-Zafra, Diego Fuentes Palacios and Toni Gabaldón</td></tr><tr><td>4617</td><td>P089-M</td><td>Exploring mRNA isoform diversity in mouse</td><td>Agata Muszyńska, Ryszard Przewłocki and Paweł P. Łabaj</td></tr><tr><td>6340</td><td>P090-M</td><td>Exploring the functional landscape of soft tissue sarcoma</td><td>Miriam Payá-Milans, María Peña-Chilet, Carlos Loucera, Marina Esteban-Medina and Joaquín Dopazo</td></tr><tr><td>8920</td><td>P091-M</td><td>GAZE: A single-cell gene regulatory inference framework from transcriptomics and epigenomics data</td><td>Shamim Ashrafiyan, Fatemeh Behjati Ardakani, Dennis Hecker and Marcel Schulz</td></tr><tr><td>6749</td><td>P092-M</td><td>Genetic determinants of blood transcript splicing and impact on molecular phenotypes in 4732 healthy individuals</td><td>Alex Tokolyi, Elodie Persyn, Katie Burnham, Artika Nath, Jonathan Marten, David Roberts, Emanuele Di Angelantonio, John Danesh, Adam Butterworth, Mike Inouye, Dirk Paul and Emma Davenport</td></tr><tr><td>4093</td><td>P093-M</td><td>Genome-wide high-resolution identification of regulatory small RNAs in bacteria based on hybrid transcriptome sequencing data</td><td>Muhammad Elhossary, Konrad Förstner, Lauren Walling, Kai Papenfort and Gisela Storz</td></tr><tr><td>9501</td><td>P094-M</td><td>High-dimensional Differential Explorer</td><td>Felix Offensperger, Markus Joppich and Ralf Zimmer</td></tr><tr><td>6723</td><td>P095-M</td><td>Identification and characterization of aneuploid cells applied to 5q deleted Myelodysplastic Syndromes</td><td>Guillermo Serrano, Aintzane Díaz, Nerea Berastegui, Marina Ainciburu, Sofia Huerga, José María Lamo de Espinosa, Mikel San Julián, Pamela Acha, Tamara Jimenez, Antonieta Molero, Maria J. Montoro, María Díez-Campelo, David Valcarcel, Francesc Sole, Teresa Ezponda, Mikel Hernaez and Felipe Prosper</td></tr><tr><td>7881</td><td>P096-M</td><td>Identification of biomarkers associated with cisplatin resistance in testicular germ cell tumours</td><td>Dominik Hadzega, Xavier Tominisec Vandoren, Katarina Kalavska, Silvia Schmidtova, Lucia Kucerova, Michal Chovanec, Petra Nemcova, Gabriel Minarik, Lubos Klucar and Michal Mego</td></tr><tr><td>3222</td><td>P097-M</td><td>Identification of novel RNA Switches using inverse RNA folding</td><td>Sumit Mukherjee and Danny Barash</td></tr><tr><td>9605</td><td>P051-T</td><td>Improving genome annotations with RNA-seq data: a scalable and reproducible workflow for Transcripts And Genes Assembly, Deconvolution, Analysis (TAGADA).</td><td>Cyril Kurylo, Cervin Guyomar, Sarah Djebali and Sylvain Foissac</td></tr><tr><td>5166</td><td>P052-T</td><td>Inferring aberrant expression dynamics across early myeloid differentiation to discover potential therapeutic targets in myelodysplastic syndromes</td><td>Aintzane Díaz-Mazkiaran, Jesús De la Fuente, Guillermo Serrano, Paula Garcia-Olloqui, Nerea Berastegui, Marina Ainciburu, Ana Alfonso, Amaia Vilas-Zornoza, Patxi San Martin, Jose Lamo de Espinosa, Mikel San-Julian, Pamela Acha, Francesc Solé, Tamara Jimenez, Félix López, María Díez-Campelo, Antonieta Molero, María Julia Montoro, David Valcarcel, Teresa Ezponda, Mikel Hernaez and Felipe Prósper</td></tr><tr><td>2516</td><td>P053-T</td><td>Influence of Topologically Associating Domains on gene expression variation</td><td>Patrycja Rosa and Aleksander Jankowski</td></tr><tr><td>3597</td><td>P054-T</td><td>Integrating multi-omics and biophysical data to explore cellular responses against phenolic compounds in yeast</td><td>Natalia Coutuouné, Rafael Boni, Cleyton Biffe, Ingrid Barcelos, Silvana Rocco, George Jackson de Moraes Rocha and Leandro Vieira dos Santos</td></tr><tr><td>4222</td><td>P055-T</td><td>Investigating the Transcriptome of Adenomatous Colorectal Polyps in the Context of Metachronous Recurrence</td><td>Simon Fisher, Russell Hung, Ditte Andersen, Gerard Lynch, Noori Maka, Jennifer Hay, Jakub Jawny, William Sloan, Stephen McSorley, Joanne Edwards and Ian Poole</td></tr><tr><td>7490</td><td>P056-T</td><td>IsoAnnot: a pipeline for functional annotation of isoforms</td><td>Alessandra Martínez Martín, Francisco Jose Pardo-Palacios, Pedro Salguero, Ángeles Arzalluz-Luque, Lorena de la Fuente Lorente and Ana Conesa</td></tr><tr><td>6084</td><td>P057-T</td><td>K-nearest neighbour correction for confounding effects in gene expression measurements</td><td>Franco Simonetti, Macarena Alonso, Cristina Marino Buslje, Saikat Banerjee and Johannes Soeding</td></tr><tr><td>2598</td><td>P058-T</td><td>Lung tissue multi-layer network in Chronic Obstructive Pulmonary Disease</td><td>Núria Olvera, Jon Sánchez, Iker Núñez, Guillaume Noell, Sandra Casas, Alejandra Lopez, Angela Guirao, Rosalba Lepore, Davide Cirillo, Àlvar Agustí, Alfonso Valencia and Rosa Faner</td></tr><tr><td>7319</td><td>P059-T</td><td>Machine leaning on whole blood RNAseq identifies interferon-regulated genes as key drivers in thrombotic primary Antiphospholipid syndrome</td><td>Kleio-Maria Verrou, Petros Sfikakis and Maria Tektonidou</td></tr><tr><td>6414</td><td>P060-T</td><td>Machine learning for improved immune cell type classification as a basis to better understand effects of cell type specific interferon stimulation</td><td>Bogac Aybey, Benedikt Brors, Sheng Zhao and Eike Staub</td></tr><tr><td>5385</td><td>P061-T</td><td>The landscape of expression and alternative splicing variation across human traits</td><td>Raquel Garcia-Perez, Jose-Miguel Ramirez and Marta Mele</td></tr><tr><td>249</td><td>P062-T</td><td>Macrophage subpopulations identification using single-cell RNA sequencing data</td><td>Mercè Alemany-Chavarria, Marta Lalinde, Joaquín Arribas and Lara Nonell</td></tr><tr><td>5511</td><td>P063-T</td><td>Molecular and functional atlas of sex-differences in multiple sclerosis subtypes analising single cell and single nucleus transcriptomic data</td><td>Irene Soler-Sáez, Zoraida Andreu, José Francisco Català-Senent, Rubén Grillo-Risco, Adolfo López-Cerdán, Almudena Neva-Alejo, Borja Gómez, Héctor Carceller, María de la Iglesia-Vayá, Marta R. Hidalgo and Francisco García-García</td></tr><tr><td>2566</td><td>P064-T</td><td>Molecular mechanisms governing CAR-T cell response in MM patients at single cell level</td><td>Lorea Jordana, Guillermo Serrano, María Erendira Calleja, Patxi San Martín-Úriz, Amaia Vilas-Zornoza, Asier Ullate-Agote, Aintzane Zabaleta, Diego Alignani, Aina Oliver-Caldes, Marta Español-Rego, Mariona Pascal, Teresa Lozano, Bruno Paiva, Susana Inoges, Ascension Lopez-Diaz de Cerio, Juan Jose Lasarte, Carlos Fernadez de Larrea, Mikel Hernaez, Juan Roberto Rodriguez-Madoz and Felipe Prosper</td></tr><tr><td>7780</td><td>P065-T</td><td>NetRank Recovers Known Cancer Hallmark Genes as Universal Biomarker Signature for Cancer Outcome Prediction</td><td>Ali Al-Fatlawi and Michael Schroeder</td></tr><tr><td>2273</td><td>P066-T</td><td>New genes and splicing isoforms revealed with native RNA in fission yeast</td><td>José Carlos Montañés, M. Mar Albà, Marta Huertas, Simone G. Moro, Williarm R. Blevins, Mercè Carmona, José Ayte and Elena Hidalgo</td></tr><tr><td>8488</td><td>P067-T</td><td>Optimal Transport improves cell-cell similarity inference in single-cell omics data</td><td>Geert-Jan Huizing, Gabriel Peyre and Laura Cantini</td></tr><tr><td>2295</td><td>P068-T</td><td>Power analysis of cell-type deconvolution across human tissues</td><td>Anna Vathrakokoili Pournara, Zhichao Miao and Irene Papatheodorou</td></tr><tr><td>2797</td><td>P069-T</td><td>Predicting off-target effects of antisense oligomers targeting bacterial mRNAs with the MASON webserver</td><td>Jakob Jung, Linda Popella, Phuong Thao Do, Patrick Pfau, Jörg Vogel and Lars Barquist</td></tr><tr><td>9109</td><td>P070-T</td><td>Predicting Transcription Factors Biological Roles in P. vulgaris</td><td>Liudmyla Kondratova, Eduardo Vallejos and Ana Conesa</td></tr><tr><td>7406</td><td>P071-T</td><td>Rational design of a novel respiratory syncytial virus reporter of early events of its viral cycle</td><td>Marcio Andrés De Ávila, Jose Luis Villarreal Camacho, Christian Cadena, Leidy Hurtado Gómez, Laura Piñeres Santos, Amner Muñoz Acevedo and Homero San Juan Vergara</td></tr><tr><td>2065</td><td>P072-T</td><td>RBPNet: Predicting Protein-RNA Interaction via CLIP-Seq Sequence-to-Signal Learning</td><td>Marc Horlacher, Nils Wagner, Marco Salvatore, Julien Gagneur, Lambert Moyon, Ole Winther and Annalisa Marsico</td></tr><tr><td>3809</td><td>P073-T</td><td>ReMap 2022: a database of Human, Mouse, Drosophila and Arabidopsis regulatory regions from an integrative analysis of DNA-binding sequencing experiments</td><td>Fayrouz Hammal, Benoit Ballester, Pierre De Langen, Fabrice Lopez and Aurélie Bergon</td></tr><tr><td>256</td><td>P074-T</td><td>rfPhen2Gen: A machine learning based association study of brain imaging phenotypes to genotypes</td><td>Muhammad Ammar Malik, Alexander Selvikvåg Lundervold and Tom Michoel</td></tr><tr><td>8576</td><td>P075-T</td><td>RNA-seq analysis: primary breast tumour and circulating tumour cells</td><td>Dominik Hadzega, Gabriel Minarik, Andrea Soltysova, Petra Nemcova, Katarina Kalavska, Marian Karaba, Juraj Benca, Tatiana Sedlackova, Daniel Pindak, Lubos Klucar and Michal Mego</td></tr><tr><td>6040</td><td>P076-T</td><td>Sex differences in the molecular basis of multiple sclerosis: meta-analysis of transcriptomic data</td><td>Jose Francisco Català-Senent, Zoraida Andreu, Marta R. Hidalgo, Roig Francisco José, Natalia Yanguas-Casás, Almudena Neva-Alejo, Adolfo López-Cerdán, Irene Soler-Sáez, María de la Iglesia-Vayá and Francisco García-García</td></tr><tr><td>6670</td><td>P077-T</td><td>Shigella-type IpaH ubiquitin ligases in non-human-host enteroinvasive Escherichia</td><td>Olga Bochkareva, Natalia Dranenko and Maria Tutukina</td></tr><tr><td>3503</td><td>P078-T</td><td>SNEEP: SNP exploration and functional analysis using epigenomics data</td><td>Nina Baumgarten, Chaonan Zhu, Ting Yuan, Meiqian Wu, Minh Duc Pham, Despina Stefanoska, Thorsten Kessler, Stefanie Dimmeler, Jaya Krishnan and Marcel H. Schulz</td></tr><tr><td>5307</td><td>P079-T</td><td>Statistical test to detect equivalence between feature lists based on the Sorensen-Dice index and GO term enrichment</td><td>Pablo Flores, Jordi Ocaña Rebull, Miquel Salicrú Pages and Alex Sanchez</td></tr><tr><td>8741</td><td>P080-T</td><td>Studying the dynamics of relative RNA localization &#8211; From nucleus to the cytoplasm</td><td>Vasilis F. Ntasis and Roderic Guigó Serra</td></tr><tr><td>7485</td><td>P081-T</td><td>T-cell receptors as biomarkers for early diagnosis of colorectal cancer in Lynch Syndrome</td><td>Maria S. Benitez-Cantos, Sonia Garcia-Rodriguez, Marta Cuadros, Carlos Cano, Antonio Poyatos-Andújar, Carmen Sánchez-Toro, Pilar Carrasco-Salas, Ana M. Serrano-Mira, Carmelo Diéguez-Castillo, Ana Delgado-Maroto and Paul Lizardi</td></tr><tr><td>5828</td><td>P082-T</td><td>The adapted Activity-By-Contact model for enhancer-gene assignment and its application to single-cell data</td><td>Dennis Hecker, Fatemeh Behjati Ardakani and Marcel Schulz</td></tr><tr><td>2675</td><td>P083-T</td><td>The Enrichment of Genes Associated with T-cell Proliferation and Memory Formation of B-cell Co-stimulatory Domain Potentiate CD19CAR-T cell Functions</td><td>Socheatraksmey Ung, Jakrawadee Julamanee, Wannakorn Khopanlert, Kajornkiat Maneechai, Surasak Sangkhathat and Pongsakorn Choochuen</td></tr><tr><td>4499</td><td>P084-T</td><td>The landscape of expression and alternative splicing variation across human traits</td><td>José Miguel Ramírez Cardeñosa, Raquel Garcia Perez and Marta Mele Messeguer</td></tr><tr><td>5863</td><td>P085-T</td><td>The least diverged orthologue conjecture</td><td>Alex Warwick Vesztrocy</td></tr><tr><td>2083</td><td>P086-T</td><td>The Multilayer Community Structure of Medulloblastoma</td><td>Iker Núñez Carpintero, Marianyela Petrizzelli, Andrei Zinovyev, Davide Cirillo and Alfonso Valencia</td></tr><tr><td>1813</td><td>P087-T</td><td>topRegNet: an analysis workflow for detecting enriched regulatory elements in RNA-seq datasets</td><td>Anshupa Sahu, Sugirthan Sivalingam, Farhad Shakeri, Svetozar Nesic and Andreas Buness</td></tr><tr><td>4341</td><td>P088-T</td><td>Towards sex-specific polygenic risk scores</td><td>María Morales Martínez, Hira Shahid, Kathleen Imbach and Eduard Porta Pardo</td></tr><tr><td>884</td><td>P089-T</td><td>Transcription start site signal profiling improves transposable element expression analysis at loci level</td><td>Natalia Savytska, Peter Heutink and Vikas Bansal</td></tr><tr><td>1608</td><td>P090-T</td><td>TranSNPs: A class of functional SNPs affecting mRNA translation potential revealed by fraction-based allelic imbalance</td><td>Samuel Valentini, Caterina Marchioretti, Alessandra Bisio, Annalisa Rossi, Sara Zaccara, Meriem Hadjer Hamadou, Elisa Pettinà, Giacomo Fantoni, Alberto Inga and Alessandro Romanel</td></tr><tr><td>6238</td><td>P091-T</td><td>Using multi-omics to identify predictive markers of aggressive neuroendocrine cancer</td><td>Dimitria Brempou, Louise Izatt, Cynthia Andoniadou and Rebecca Oakey</td></tr><tr><td>9960</td><td>P092-T</td><td>Variability of nonsense-mediated mRNA decay (NMD) pathway efficiency in human cancers</td><td>Guillermo Palou Márquez and Fran Supek</td></tr><tr><td>1583</td><td>P093-T</td><td>WASP &#8211; A versatile, web-accessible single cell RNA-seq processing platform</td><td>Andreas Hoek, Katharina Maibach, Ebru Özmen, Torsten Hain, Susanne Herold and Alexander Goesmann</td></tr><tr><td>274</td><td>P094-T</td><td>κ-velo improves single-cell RNA-velocity estimation</td><td>Valérie Marot-Lassauzaie, Brigitte Bouman, Fearghal Donaghy and Laleh Haghverdi</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-853365c" data-block-id="853365c"><style>.stk-853365c{margin-top:50px !important}.stk-853365c .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-64078c1" data-block-id="64078c1"><style>.stk-64078c1 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-64078c1 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-64078c1 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-64078c1 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-64078c1 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-64078c1 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-64078c1 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-eacaacb" data-block-id="eacaacb"><style>.stk-eacaacb .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-eacaacb .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-eacaacb .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-eacaacb .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-eacaacb .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-eacaacb .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-eacaacb .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-8d39392" data-block-id="8d39392"><style>.stk-8d39392 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-8d39392 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-8d39392 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-8d39392 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-8d39392 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-8d39392 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-8d39392 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-968e60f" data-block-id="968e60f"><style>.stk-968e60f .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-968e60f .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-968e60f .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-968e60f .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-968e60f .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-968e60f .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-968e60f .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-b92272f" data-block-id="b92272f"><style>.stk-b92272f .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-b92272f .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-b92272f .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-b92272f .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-b92272f .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-b92272f .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-b92272f .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-d194cf1" data-block-id="d194cf1"><style>.stk-d194cf1 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-d194cf1 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-d194cf1 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-d194cf1 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-d194cf1 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-d194cf1 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-d194cf1 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-34fa2fb" data-block-id="34fa2fb"><style>.stk-34fa2fb .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-34fa2fb .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-34fa2fb .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-34fa2fb .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-34fa2fb .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-34fa2fb .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-34fa2fb .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-ea607df" data-block-id="ea607df"><style>.stk-ea607df .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-ea607df .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-ea607df .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-ea607df .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-ea607df .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-ea607df .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-ea607df .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-d0f5355" id="h-genomes" data-block-id="d0f5355"><style>.stk-d0f5355 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Genomes</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>628</td><td>P098-M</td><td>3D modeling of genome reorganization upon senescence</td><td>Vasiliki Varamogianni Mamatsi and Argyris Papantonis</td></tr><tr><td>6213</td><td>P099-M</td><td>A computational model of double strand breaks and repair explaining cut-and-paste structural variants</td><td>Bingxin Lu and Chris Barnes</td></tr><tr><td>5333</td><td>P100-M</td><td>A database of copy number variant frequencies in the Spanish population</td><td>Daniel López-López, Gema Roldan, Jose Luis Fernandez-Rueda, Rosario Carmona, Virginia Aquino, Maria Peña-Chillet, Ruben Garcia, Rocio Nuñez, Anna Gonzalez, Guillermo Pita and Joaquin Dopazo</td></tr><tr><td>4794</td><td>P101-M</td><td>A phylum-wide genomic screen in cyanobacteria for RNA structure motifs adjacent to orthologues genes.</td><td>Adrian Geissler, Elena Carrasquer-Álvarez, Niels-Ulrik Frigaard, Jan Gorodkin and Stefan E. Seemann</td></tr><tr><td>8215</td><td>P102-M</td><td>A universal protein-coding gene finder</td><td>Ferriol Calvet, Jaume Reig, Emilio Righi, Francisco Camara and Roderic Guigó</td></tr><tr><td>2625</td><td>P103-M</td><td>Accurate de novo identification of biosynthetic gene clusters</td><td>Laura Carroll, Martin Larralde, Jonas Fleck, Ruby Ponnudurai, Alessio Milanese, Elisa Cappio and Georg Zeller</td></tr><tr><td>6008</td><td>P104-M</td><td>Aggregated genomic data as cohort-specific allelic frequencies can boost variants and genes prioritization in non-solved cases of inherited retinal dystrophies</td><td>Ionut Florin Iancu, Irene Perea-Romero, Gonzalo Nuñez-Moreno, Lorena de la Fuente, Raquel Romero, Almudena Ávila-Fernandez, Maria Jose Trujillo-Tiebas, Rosa Riveiro-Álvarez, Berta Almoguera, Inmaculada Martín-Mérida, Marta Del Pozo-Valero, Alejandra Damián-Verde, Marta Cortón, Carmen Ayuso and Pablo Minguez</td></tr><tr><td>3435</td><td>P105-M</td><td>An evaluation of pipelines for DNA variant detection can guide a reanalysis protocol to increase the diagnostic ratio of genetic diseases</td><td>Raquel Romero, Lorena de la Fuente, Marta Del Pozo-Valero, Rosa Riveiro-Álvarez, María José Trujillo-Tiebas, Inmaculada Martín-Mérida, Almudena Ávila-Fernández, Ionut-Florin Iancu, Irene Perea-Romero, Gonzalo Núñez-Moreno, Alejandra Damián, Cristina Rodilla, Berta Almoguera, Marta Cortón, Carmen Ayuso and Pablo Mínguez</td></tr><tr><td>1333</td><td>P106-M</td><td>An Interactive Workflow for Exploratory Detection of Genomic Insertions on a Population Scale</td><td>Kedi Cao, Nourhan Elfaramawy, Matthias Weidlich and Birte Kehr</td></tr><tr><td>4581</td><td>P107-M</td><td>An Orthogonal Approach for Genome Assembly Curation: Examples on Vertebrate Genome Assemblies</td><td>Fernando Cruz, Lucía Álvarez-González, Rut Lara-Hernández, Jèssica Gómez-Garrido, Margarida Barcelo-Serra, Laura Baldo, Aurora Ruiz-Herrera, José Antonio Godoy and Tyler Alioto</td></tr><tr><td>5908</td><td>P108-M</td><td>Analysis of new immune systems using Deep Learning</td><td>Sven Hauns, Omer Alkhnbashi and Rolf Backofen</td></tr><tr><td>3856</td><td>P109-M</td><td>Analysis of SNPs in E. Coli populations in murine gut metagenomes</td><td>Francisco Cerqueira, João Costa, Massimo Amicone and Isabel Gordo</td></tr><tr><td>8209</td><td>P110-M</td><td>Assessment of supervised methods for lncRNA function prediction from expression data</td><td>Fatemeh Behjati, Hicham Saddiki, Matthias S. Leisegang, Theresa Gimbel, Frederike Boos, Stefanie Dimmeler, Ilka Wittig, Reinier Boon, Ralf P. Brandes and Marcel H. Schulz</td></tr><tr><td>2824</td><td>P111-M</td><td>Assessment of the structural differences of the JHEH from closely related insects as potential targets for safe pesticide design</td><td>Weronika Bagrowska, Tomasz Skalski, Maria Bzówka and Artur Góra</td></tr><tr><td>8684</td><td>P112-M</td><td>Automating the genome assembly process with Snakemake</td><td>Jèssica Gómez-Garrido, Fernando Cruz, Marc Palmada-Flores, Laura Baldo and Tyler Alioto</td></tr><tr><td>5852</td><td>P113-M</td><td>Beacon Network goes v2</td><td>Dmitry Repchevsky, Sergi Aguiló-Castillo, Dominik Bruchner, Teemu Kataja, Ville Muilu, Juha Törmroos, Babita Singh, Jordi Rambla, J. Dylan Spalding, Michael Baudis, Salvador Capella-Gutierrez and Josep Lluís Gelpí</td></tr><tr><td>5369</td><td>P114-M</td><td>Benchmarking and improving imputation approaches for recurrent inversions in the human genome</td><td>Illya Yakymenko, Jon Lerga-Jaso and Mario Cáceres</td></tr><tr><td>7102</td><td>P115-M</td><td>Cell type specific drug response prediction from single cell data</td><td>Nikoletta Katsaouni and Marcel Schulz</td></tr><tr><td>7904</td><td>P116-M</td><td>CESIM – A Simulator for Clonal Evolution</td><td>Sarah Sandmann, Silja Richter, Xiaoyi Jiang and Julian Varghese</td></tr><tr><td>3302</td><td>P117-M</td><td>Characterising human cell types using TF footprinting</td><td>Aybuge Altay, Yufei Zhang and Martin Vingron</td></tr><tr><td>3497</td><td>P118-M</td><td>Characterization of the role of regulatory variants for trait pleiotropy</td><td>Aitor González</td></tr><tr><td>6347</td><td>P119-M</td><td>CimpleG: Simple CpG methylation signatures</td><td>Tiago Maié, Marco Schmidt, Wolfgang Wagner and Ivan G. Costa</td></tr><tr><td>5031</td><td>P120-M</td><td>CloneTracer enables the identification of healthy and leukemic cells from single-cell transcriptomic data</td><td>Sergi Beneyto-Calabuig, Anne Kathrin Merbach, Jonas Alexander Kniffka, Chelsea Szu-Tu, Magdalena Antes, Michael Scherer, Simon Raffel, Carsten Müller-Tidow and Lars Velten</td></tr><tr><td>2727</td><td>P121-M</td><td>Cluster-based conserved non-coding element (CNE) characterization</td><td>Silvia Fibi-Smetana and Leila Taher</td></tr><tr><td>9164</td><td>P122-M</td><td>CoLoRd: Compressing long reads</td><td>Marek Kokot, Adam Gudyś, Heng Li and Sebastian Deorowicz</td></tr><tr><td>1403</td><td>P123-M</td><td>Combining Epigenome and Genetic Mutation Data to Study Disease-Relevant Cell Types and Gene Sets</td><td>Ahlam Mallak, Dennis Hecker and Marcel Schulz</td></tr><tr><td>8467</td><td>P124-M</td><td>Comparative Analysis of Long Versus Short-Read Sequencing Technologies in Terms of Structural Variants Inference for Trios</td><td>Sachin Gadakh, Mateusz Chiliński, Karolina Jodkowska, Jan Gawor and Dariusz Plewczynski</td></tr><tr><td>7503</td><td>P125-M</td><td>Comparison of Chromatin Contacts Detectability in Human ESC-H1 Mapped Using GAM or Hi-C</td><td>Teresa Szczepińska, Christoph Thieme, Sachin Gadakh, Alexander Kukalev, Warren Winick-Ng, Rieke Kempfer, Thomas Sparks, Miao Yu, Bing Ren, Dariusz Plewczynski and Ana Pombo</td></tr><tr><td>4588</td><td>P126-M</td><td>Comprehensive analysis of genomic patterns in different types of dengue virus</td><td>Myeongji Cho, Xianglan Min and Hyeon S. Son</td></tr><tr><td>7652</td><td>P127-M</td><td>Containerised Pipelines for Sensitive Sequencing Data</td><td>Tina Visnovska, Sadia Saeed, Lars la Cour Poulsen, Torunn Rønningen, Mai Britt Dahl, Junbai Wang, Tom Mala, Jon A. Kristinsson, Jens Kristoffer Hertel, Jøran Hjelmesæth, Matthias Blüher, Tone Gretland Valderhaug and Yvonne Böttcher</td></tr><tr><td>2826</td><td>P128-M</td><td>Contaminant Detection using Differentiating k-mers</td><td>Alessia Petescia, Martina Nebohacova, Jozef Nosek, Brona Brejova and Tomas Vinar</td></tr><tr><td>6548</td><td>P129-M</td><td>Copy number-aware methylation deconvolution analysis of cancers from whole genome bisulfite sequencing data</td><td>Nana Mensah, Elizabeth Larose Cadieux, Tom Lesluyes, Jonas Demeulemeester and Peter Van Loo</td></tr><tr><td>2939</td><td>P130-M</td><td>Deciphering the etiology and role in oncogenic transformation of the CpG island methylator phenotype: a pan-cancer analysis</td><td>Josephine Yates and Valentina Boeva</td></tr><tr><td>9088</td><td>P131-M</td><td>Detection and evaluation of variable number tandem repeats</td><td>Maryam Ghareghani, Hossein Moeinzadeh and Martin Vingron</td></tr><tr><td>6110</td><td>P132-M</td><td>Development of a new structural variant detection software based on graph clustering algorithms from long reads</td><td>Nicolas Gustavo Gaitan Gomez and Jorge Duitama</td></tr><tr><td>9259</td><td>P133-M</td><td>Differential methylation analysis signature evaluation using Hobotnica metric</td><td>Anna Budkina, Alexey Stupnikov and Yulia Medvedeva</td></tr><tr><td>997</td><td>P134-M</td><td>Discovering Significant Evolutionary Trajectories in Cancer Phylogenies</td><td>Leonardo Pellegrina and Fabio Vandin</td></tr><tr><td>4206</td><td>P135-M</td><td>DNA-DDA predicts genome wide 3D chromatin structure directly from the reference sequence</td><td>Xenia Lainscsek and Leila Taher</td></tr><tr><td>2804</td><td>P136-M</td><td>Evidence-driven annotation of the Trichechus manatus latirostris genome using long-reads</td><td>Alejandro Paniagua, Francisco J. Pardo-Palacios and Ana Conesa</td></tr><tr><td>1106</td><td>P137-M</td><td>FrameRate: predicting coding frames direct from unassembled DNA reads</td><td>Wang Liu-Wei, Robert Hoehndorf, Wayne Aubrey, Christopher Creevey, Amanda Clare and Nicholas Dimonaco</td></tr><tr><td>2935</td><td>P138-M</td><td>Genetic diversity in Amaranthaceae crops</td><td>Felix Leopold Wascher, Nancy Stralis-Pavese, Heinz Himmelbauer and Juliane C. Dohm</td></tr><tr><td>5847</td><td>P139-M</td><td>Genome sequencing and comparative genomic analysis of turmeric plant provide insights into adaptive evolution of its medicinal properties</td><td>Abhisek Chakraborty, Shruti Mahajan, Shubham K. Jaiswal and Vineet K. Sharma</td></tr><tr><td>9607</td><td>P140-M</td><td>Germline genetics correlates with aberrant signaling pathways in cancer</td><td>Davide Dalfovo, Riccardo Scandino, Marta Paoli, Samuel Valentini and Alessandro Romanel</td></tr><tr><td>4322</td><td>P141-M</td><td>GPU-Accelerated Pairwise Sequence Alignment using the Wavefront Algorithm</td><td>Quim Aguado-Puig, Santiago Marco-Sola, Juan Carlos Moure, Christos Matzoros, David Castells-Rufas, Antonio Espinosa and Miquel Moreto</td></tr><tr><td>6579</td><td>P142-M</td><td>GWAS of vessel diameter, number of bifurcations, and main temporal angles identifies genetic variants in OCA2, HERC2 and other genes to modulate these traits</td><td>Sofia Ortin Vela, Michael Johannes Beyeler, Mattia Tomasoni, Olga Trofimova, Florence Hoogewoud and Sven Bergmann</td></tr><tr><td>3623</td><td>P143-M</td><td>Haplotype-resolved assembly of a tetraploid potato genome using long reads and low-depth offspring data</td><td>Rebecca Serra Mari, Sven Schrinner, Richard Finkers, Paul Arens, Maximilian H.-W. Schmidt, Björn Usadel, Gunnar W. Klau and Tobias Marschall</td></tr><tr><td>8830</td><td>P144-M</td><td>Human glycosylation gene variants as possible contenders to explain differences in SARS-CoV-2 disease severity in Indian sub populations.</td><td>Bithika Chatterjee and Shekhar Mande</td></tr><tr><td>327</td><td>P145-M</td><td>Identification and characterization of the microbiome, resistome, and mobilome in the wastewater treatment plant</td><td>Stephanie Pillay, Ramin Shirali Hossein Zade and Thomas Abeel</td></tr><tr><td>2754</td><td>P146-M</td><td>Identification of epivariations in rare diseases from a single patient perspective.</td><td>Robin Grolaux, Alexis Hardy and Matthieu Defrance</td></tr><tr><td>9612</td><td>P147-M</td><td>Identification of genes under positive selection in E. coli: Focusing on antibiotic resistance</td><td>Negin Malekian Boroujeni</td></tr><tr><td>6453</td><td>P148-M</td><td>Identification of genome edited cells using CRISPRnano</td><td>Thach Nguyen, Haribaskar Ramachandran, Soraia Martins, Jean Krutmann and Andrea Rossi</td></tr><tr><td>7863</td><td>P149-M</td><td>Identification of regions with high evolutionary variability on the SARS-CoV-2 S1 and S2 surface glycoprotein.</td><td>Katrina Norwood, Alice McHardy, Susanne Reimering and Thorsten Klingen</td></tr><tr><td>1927</td><td>P150-M</td><td>Identification of viral miRNAs</td><td>Alexandra Schubö, Armin Hadziahmetovic, Leonie Pohl, Markus Joppich and Ralf Zimmer</td></tr><tr><td>2188</td><td>P151-M</td><td>Impacts of COVID-19 on the Microbiome: A Bioinformatics and Machine Learning Study</td><td>Bertalan Takács, Zoltán Gyuris, Lajos Pintér, Ádám Visnyovszki, Nóra Éva Nagy, Márton Zsolt Enyedi, Edit Hajdú and Lajos Haracska</td></tr><tr><td>758</td><td>P152-M</td><td>Improving multiplicity assignments in de Bruijn graphs using Loopy Belief Propagation inference on Conditional Random Fields.</td><td>Aranka Steyaert, Pieter Audenaert and Jan Fostier</td></tr><tr><td>2284</td><td>P153-M</td><td>Inferring mutational signatures from distinct copy-number events</td><td>Tom L Kaufmann and Roland F Schwarz</td></tr><tr><td>4377</td><td>P154-M</td><td>Influence of motif interactions on post-hoc attribution methods in genomic CNNs</td><td>Marta S. Lemanczyk, Jakub M. Bartoszewicz and Bernhard Y. Renard</td></tr><tr><td>1536</td><td>P155-M</td><td>Interpretable deep learning for phage life cycle prediction</td><td>Melania Nowicka, Jakub M Bartoszewicz and Bernhard Y Renard</td></tr><tr><td>8586</td><td>P156-M</td><td>Intrinsic linking of chromatin in human cells</td><td>Maciej Borodzik, Michał Denkiewicz, Krzysztof Spaliński, Kamila Winnicka, Sevastianos Korsak, Kaustav Sengupta, Marcin Pilipczuk, Michał Pilipczuk, Yijun Ruan and Dariusz Plewczynski</td></tr><tr><td>3015</td><td>P157-M</td><td>JLOH: Extracting Loss of Heterozygosity Blocks from Short-Read Sequencing Data</td><td>Matteo Schiavinato</td></tr><tr><td>9113</td><td>P095-T</td><td>k-mer and GWAS approaches to identify host-specific genomic determinants in Klebsiella pneumoniae</td><td>Konstantinos A. Kyritsis, Georgios-Nikolaos Kartanos, Victoria Siarkou and Fotis Psomopoulos</td></tr><tr><td>6078</td><td>P096-T</td><td>Leveraging patient-level metatranscriptomics data and phase variation predictions to prioritize bacterial antigens</td><td>Bart Cuypers, Pieter Meysman, Alessandro Brozzi, Normand Blais, Christophe Lambert and Kris Laukens</td></tr><tr><td>1150</td><td>P097-T</td><td>LIVE-DREAM: Live analysis of NGS data using an optimized hierarchical inter-leaved Bloom Filter index</td><td>Ferdous Nasri, Tobias Loka and Bernhard Y Renard</td></tr><tr><td>9739</td><td>P098-T</td><td>Long non-coding RNAs and novel transcripts as the principal source of potential novel neoantigens in hepatocellular carcinoma</td><td>Marta E. Camarena, Júlia Perera-Bel and M. Mar Albà</td></tr><tr><td>8956</td><td>P099-T</td><td>Looking at the overall picture: development of a metagenomic analyses pipeline for highly diverse microbial communities in heritage science</td><td>Monika Waldherr, Laura Rabbachin, Johannes Tichy, Guadalupe Piñar, Martin Ortbauer, Beate Sipek and Alexandra Graf</td></tr><tr><td>5102</td><td>P100-T</td><td>Lossless Approximate Pattern Matching on Pan-genome de Bruijn Graphs</td><td>Lore Depuydt, Luca Renders, Thomas Abeel and Jan Fostier</td></tr><tr><td>8600</td><td>P101-T</td><td>Metagenetics versus metagenomics, a dual approach to reach strain-level resolution for starter culture monitoring</td><td>Cristian Díaz-Muñoz, Hannes Decadt, Luc De Vuyst and Stefan Weckx</td></tr><tr><td>1314</td><td>P102-T</td><td>metagWGS: a workflow to analyse short and long HiFi metagenomic reads</td><td>Joanna Fourquet, Jean Mainguy, Maïna Vienne, Céline Noirot, Vincent Darbot, Pierre Martin, Olivier Bouchez, Adrien Castinel, Sylvie Combes, Carole Iampietro, Christine Gaspin, Denis Milan, Cécile Donnadieu, Geraldine Pascal and Claire Hoede</td></tr><tr><td>1769</td><td>P103-T</td><td>MetaProFi: An ultrafast chunked Bloom filter for storing and querying protein and nucleotide sequence data for accurate identification of functionally relevant genetic variants</td><td>Sanjay Kumar Srikakulam, Sebastian Keller, Fawaz Dabbaghie, Robert Bals and Olga Kalinina</td></tr><tr><td>5368</td><td>P104-T</td><td>Modeling dynamic interactions within human gut microbiome using statistical approach</td><td>Zuzanna Karwowska, Marcin Możejko, Paweł Szczerbiak, Ewa Szczurek and Tomasz Kościółek</td></tr><tr><td>4426</td><td>P105-T</td><td>Multi-scale phase separation by explosive percolation with single chromatin loop resolution</td><td>Kaustav Sengupta, Michał Denkiewicz, Mateusz Chiliński, Sevastianos Korsak, Teresa Szczepinska, Ayatullah Faruk Mollah, Raissa D&#8217;Souza, Yijun Ruan and Dariusz Plewczynski</td></tr><tr><td>8524</td><td>P106-T</td><td>Mutation Clonality, Neoantigens, and Immune Biomarkers relate to Immunotherapy Response in Bladder Cancer patients</td><td>Lilian Marie Boll, Júlia Perera-Bel, Oriol Arpí, Ana Rovira, Núria Juanpere Rodero, Silvia Hernández-Llodrà, Josep Lloreta, Alejo Rodriguez-Vida, M. Mar Albà and Joaquim Bellmunt</td></tr><tr><td>7537</td><td>P107-T</td><td>Mutational signatures are markers of drug sensitivity of cancer cells</td><td>Fran Supek, Jurica Levatić, Marina Salvadores and Francisco Fuster-Tormo</td></tr><tr><td>1373</td><td>P108-T</td><td>NETWORK ANALYSIS OF MULTI-OMICS MICROBIOME DATA TO IDENTIFY KEYSTONE UNKNOWN TAXA</td><td>Rocío Amorín de Hegedüs, Ana Conesa and Jamie Foster</td></tr><tr><td>1715</td><td>P109-T</td><td>NGSEP 4: EFFICIENT AND ACCURATE IDENTIFICATION OF ORTHOGROUPS AND WHOLE-GENOME ALIGNMENT</td><td>Laura Natalia González García, Daniel Tello, Juan Camilo Zuluaga-Monares, Ricardo Angel, Daniel Mahecha, Nicolas Cardozo, Camilo Escobar, Jorge Gomez, Mario Linares-Vasquez and Jorge Duitama</td></tr><tr><td>6198</td><td>P110-T</td><td>OMArk: Quality assessment of protein-coding gene repertoires</td><td>Yannis Nevers, Victor Rossier and Christophe Dessimoz</td></tr><tr><td>9432</td><td>P111-T</td><td>Omics approaches and artificial intelligence strategies applied to industrial Saccharomyces cerevisiae yeasts for corn and second-generation ethanol production processes</td><td>Marcelo Falsarella Carazzolle, Beatriz Vargas, Larissa Escalfi Tristao, Thais Oliveira Secches, Jade Ribeiro dos Santos, Juliana Jose, Fellipe da Silveira Bezerra Mello and Gonçalo Amarante Guimarães Pereira</td></tr><tr><td>3332</td><td>P112-T</td><td>PerSVade: Personalized Structural Variation detection in your species of interest</td><td>Miquel Àngel Schikora Tamarit and Toni Gabaldón</td></tr><tr><td>6507</td><td>P113-T</td><td>PhyloCloud: an online platform for making sense of phylogenomic data</td><td>Ziqi Deng, Jorge Botas, Carlos P Cantalapiedra, Ana Hernández-Plaza, Jordi Burguet-Castell and Jaime Huerta-Cepas</td></tr><tr><td>8677</td><td>P114-T</td><td>PhylomeDB v5: An updated site to browse and mine genome-wide catalogs of gene phylogenies</td><td>Diego Fuentes Palacios, Manu Molina, Uciel Chorostecki, Salvador Capella-Gutiérrez, Marina Marcet-Houben and Toni Gabaldón</td></tr><tr><td>5259</td><td>P115-T</td><td>Predicting Virulence of Listeria Monocytogenes using Whole Genome Sequencing and Machine Learning</td><td>Alexander Gmeiner, Patrick Murigu Kamau Njage, Lisbeth Truelstrup Hansen and Pimlapas Leekitcharoenphon</td></tr><tr><td>9313</td><td>P116-T</td><td>Prevalence and Specificity of Chemoreceptor Profiles in Plant-Associated Bacteria</td><td>Claudia Sanchis-López, Jean Paul Cerna-Vargas, Saray Santamaría-Hernando, Cayo Ramos, Tino Krell, Pablo Rodríguez-Palenzuela, Emilia López-Solanilla, Jaime Huerta and José J. Rodríguez-Herva</td></tr><tr><td>8005</td><td>P117-T</td><td>Proton and alpha radiation-induced mutational profiles in human cells.</td><td>Tiffany Delhomme, Manuela Buonanno, Grilj Veljko, Josep Biayna and Fran Supek</td></tr><tr><td>5346</td><td>P118-T</td><td>Quantifying microbial dark matter and its impact on metagenomic analyses</td><td>Elizabeth Yuu, Vitor Piro and Bernhard Renard</td></tr><tr><td>7040</td><td>P119-T</td><td>Read-level GC bias correction for improved cell-free DNA signal processing</td><td>Sebastian Röner, Benjamin Spiegl, Michael R. Speicher and Martin Kircher</td></tr><tr><td>1619</td><td>P120-T</td><td>Recurrent positive selection of lipid trafficking genes in Clupeiformes</td><td>Jorge Langa, Yuri Rueda, Aitor Albaina, Martin Huret, Darrell Conklin and Andone Estonba</td></tr><tr><td>3783</td><td>P121-T</td><td>Redefining Promoter DNA Methylation Change in Cancer</td><td>Richard Heery and Martin Schaefer</td></tr><tr><td>5450</td><td>P122-T</td><td>Relation between genome mutability, variant pathogenicity and vertical ionization potential of nucleobase motifs</td><td>Cyril Karamaoun, Pauline Hermans, Fabrizio Pucci and Marianne Rooman</td></tr><tr><td>5615</td><td>P123-T</td><td>Resolution of deep nodes and new solid backbone phylogeny in Ophioglossaceae ferns</td><td>Darina Koubínová, Li-Yaung Kuo and Jason Grant</td></tr><tr><td>6082</td><td>P124-T</td><td>Sample demultiplexing and doublets removal from single-nuclei RNA sequencing data</td><td>Kwong Leong Wong, Lena Jassowicz, Peter Lichter, Christel Herold-Mende, Martina Seiffert and Marc Zapatka</td></tr><tr><td>1667</td><td>P125-T</td><td>scTAM-seq enables targeted high-confidence analysis of DNA methylation in single cells</td><td>Agostina Bianchi, Michael Scherer, Roser Zaurin, Kimberly Quililan, Lars Velten and Renee Beekman</td></tr><tr><td>8914</td><td>P126-T</td><td>Shared genetic architecture between tobacco smoking and iron concentration in the brain’s dorsal striatum</td><td>Olga Trofimova and Sven Bergmann</td></tr><tr><td>2410</td><td>P127-T</td><td>Single-cell guided deconvolution of bulk AML transcriptomics recapitulates FAB landscape and CD14+ Monocyte percentage predicts Venetoclax resistance</td><td>Emin Onur Karakaslar, Jeppe Severens, Elena Sanchez Lopez, Peter van Balen, Heendrik Veelken, Marcel Jt Reinders, Marieke Griffioen and Erik van den Akker</td></tr><tr><td>6762</td><td>P128-T</td><td>Single-cell resolution unravels spatial alterations in metabolism, transcriptome and epigenome of ageing liver</td><td>Chrysa Nikopoulou, Niklas Kleinenkuhnen, Swati Parekh, Tonantzi Sandoval, Farina Schneider, Patrick Giavalisco, Mihaela Bozukova, Anna Juliane Vesting, Janine Altmüller, Thomas Wunderlich, Vangelis Kondylis, Achim Tresch and Peter Tessarz</td></tr><tr><td>638</td><td>P129-T</td><td>SonicParanoid enhanced by machine learning allows fast de novo orthology inference of huge MAG datasets</td><td>Salvatore Cosentino and Wataru Iwasaki</td></tr><tr><td>7525</td><td>P130-T</td><td>Sounds trivial &#8211; but it&#8217;s not! Automatic reference detection and web service independent reconstruction of influenza A and B genomes</td><td>Katja Winter, Oliver Drechsel, Marianne Wedde, Ralf Dürrwald, Thorsten Wolff and Stephan Fuchs</td></tr><tr><td>902</td><td>P131-T</td><td>Statistical and data mining analysis of stop codon triplets in introns in view of the stop-to-stop ORF definition</td><td>Valentin Wesp and Stefan Schuster</td></tr><tr><td>6709</td><td>P132-T</td><td>Strengths and weaknesses of metabarcoding long read.</td><td>Jean Mainguy, Adrien Castinel, Olivier Bouchez, Sylvie Combes, Carole Iampietro, Christine Gaspin, Denis Milan, Cécile Donnadieu, Claire Hoede and Geraldine Pascal</td></tr><tr><td>7488</td><td>P133-T</td><td>Structural comparison of chromatin interaction networks generated from Hi-C data</td><td>Gatis Melkus, Lelde Lace, Peteris Rucevskis, Sandra Silina, Andrejs Sizovs, Edgars Celms and Juris Viksna</td></tr><tr><td>4984</td><td>P134-T</td><td>Synggen: fast and data-driven generation of synthetic heterogeneous NGS cancer data</td><td>Riccardo Scandino, Federico Calabrese and Alessandro Romanel</td></tr><tr><td>5987</td><td>P135-T</td><td>Taking the prediction of pathogenic variant-combinations to the next level with VarCoPP2.0</td><td>Nassim Versbraegen, Barbara Gravel, Charlotte Nachtegael, Alexandre Renaux, Emma Verkinderen, Ann Nowé, Tom Lenaerts and Sofia Papadimitriou</td></tr><tr><td>3491</td><td>P136-T</td><td>Target-Oriented miRNA Discovery (TOMiD) Analysis of qCLASH Ribonomics Experiments</td><td>Daniel Stribling, Nicholas Hiers, Mingyi Xie and Rolf Renne</td></tr><tr><td>6989</td><td>P137-T</td><td>The additive effects of Next Generation Sequencing error sources on the quality of de novo genome assembly</td><td>Alex Váradi, Zoltán Rádai, Nikoletta A Nagy, Péter Takács, Gábor Kardos and Levente Laczkó</td></tr><tr><td>4924</td><td>P138-T</td><td>The genome of Belgium: whole genome sequencing of the Belgian population to enable public health genomics</td><td>Thomas Delcourt, Charlotte De Vogelaere, Emilie Cauët, Nina Van Goethem, Johan Van der Heyden, Stefaan Demarest, Karin De Ridder, Nancy Roosens, Marc Van Den Bulcke and Kevin Vanneste</td></tr><tr><td>7141</td><td>P139-T</td><td>The Spatial Organization Of Enhancers Around Promoter Regions Within Chromatin Contact Domains For Selected Human Cell Lines: Structural Regulatory Landscape</td><td>Abhishek Agarwal and Dariusz Plewczyński</td></tr><tr><td>9193</td><td>P140-T</td><td>Towards an atlas of copy-number conditional selection on somatic mutations in cancer driver genes</td><td>Elizaveta Besedina and Fran Supek</td></tr><tr><td>7646</td><td>P141-T</td><td>TP53-dependent toxicity of CRISPR/Cas9 cuts is differential across genomic loci and can confound genetic screening</td><td>Miguel M Álvarez, Josep Biayna and Fran Supek</td></tr><tr><td>7435</td><td>P142-T</td><td>Training DeepSignal models to call CpG methylation in pig and quail ONT reads</td><td>Paul Terzian, Céline Vandecasteele, Christine Gaspin, Cécile Donnadieu, Denis Milan, Rémi-Félix Serre and Christophe Klopp</td></tr><tr><td>707</td><td>P143-T</td><td>Transformer Language Models for Genomic Sequences</td><td>Vlastimil Martinek, David Cechak, Petr Simecek and Panagiotis Alexiou</td></tr><tr><td>5181</td><td>P144-T</td><td>Transgenerational epigenetics in quail: whole genome DNA methylation analysis</td><td>Chloé Cerutti, Sophie Leroux, Paul Terzian, David Gourichon, Frederique Pitel and Guillaume Devailly</td></tr><tr><td>3093</td><td>P145-T</td><td>Unbiased discovery of diversity-generating mechanisms and mobile genetic elements</td><td>Jordi Abante and Julia Salzman</td></tr><tr><td>4973</td><td>P146-T</td><td>Uncovering signatures of mutational processes in SARS-CoV-2</td><td>Kieran Lamb</td></tr><tr><td>795</td><td>P147-T</td><td>Untangling Costa Rican cocoa bean fermentation processes using a combined shotgun metagenomics, metatranscriptomics, and meta-metabolomics approach</td><td>Stefan Weckx, Marko Verce and Luc De Vuyst</td></tr><tr><td>3384</td><td>P148-T</td><td>Unveiling epigenetic profiles of transposable elements in ChIP-seq data using T3E</td><td>Michelle Almeida da Paz and Leila Taher</td></tr><tr><td>4624</td><td>P149-T</td><td>Uplifting trimAl for handling thousands of sequences</td><td>Nicolás Díaz Roussel and Salvador Capella Gutiérrez</td></tr><tr><td>5775</td><td>P150-T</td><td>Using AlphaFold2-generated structural information to improve detection of evolutionary adaptations in proteins</td><td>Sophie-Luise Heidig, Ravy Leon Foun Lin, Danny Ionescu, Jean-François Flot and Wim Vranken</td></tr><tr><td>7600</td><td>P151-T</td><td>Variable DNA methylation underlies mutation rate variability at the mesoscale in human somatic cells</td><td>David Mas-Ponte and Fran Supek</td></tr><tr><td>3965</td><td>P152-T</td><td>VARIANT PRIORITIZATION IN PLASMA WHOLE-EXOME SEQUENCING FOR THE IDENTIFICATION OF POTENTIAL THERAPEUTIC TARGETS IN RELAPSING COLON CANCER PATIENTS</td><td>Jorge Martín-Arana, Francisco Gimeno-Valiente, Roberto Tébar-Martínez, Blanca García-Micó, Valentina Gambardella, Marisol Huerta, Carolina Martínez-Ciarpaglini, Juan Antonio Carbonell-Asins, Manuel Cabeza-Segura, José Martín-Arévalo, David Casado, Vicente Pla, Leticia Pérez, Pilar Rentero-Garrido, Sheila Zúñiga-Trejos, Susana Roselló, Tania Fleitas, Josefa Castillo, Desamparados Roda, Andrés Cervantes and Noelia Tarazona</td></tr><tr><td>9752</td><td>P153-T</td><td>Vessel type-specific GWAS of Retinal Vessel Tortuosity Identifies 173 Novel Loci Revealing Genes and Pathways Associated with Vascular Pathomechanisms and Diseases</td><td>Michael Beyeler, Sofia Ortin Vela, Mattia Tomasoni and Sven Bergmann</td></tr><tr><td>2212</td><td>P154-T</td><td>What is the reality? – Influence of the sequencing technology and the approach on microbial community composition estimation</td><td>Dedan Githae, Agata Jarosz, Kinga Herda, Kamila Marszałek, Wojciech Branicki and Paweł Łabaj</td></tr><tr><td>1671</td><td>P155-T</td><td>Whole genome of a biparental beetle species, Anoplotrupes stercorosus</td><td>Nikoletta A Nagy, Levente Laczkó and Zoltán Barta</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-b628f36" data-block-id="b628f36"><style>.stk-b628f36{margin-top:50px !important}.stk-b628f36 .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-6eae5fe" data-block-id="6eae5fe"><style>.stk-6eae5fe .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-6eae5fe .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-6eae5fe .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-6eae5fe .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-6eae5fe .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-6eae5fe .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-6eae5fe .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-f51834c" data-block-id="f51834c"><style>.stk-f51834c .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-f51834c .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-f51834c .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-f51834c .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-f51834c .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-f51834c .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-f51834c .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-e0ff84e" data-block-id="e0ff84e"><style>.stk-e0ff84e .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-e0ff84e .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-e0ff84e .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-e0ff84e .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-e0ff84e .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-e0ff84e .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-e0ff84e .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-70fe11a" data-block-id="70fe11a"><style>.stk-70fe11a .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-70fe11a .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-70fe11a .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-70fe11a .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-70fe11a .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-70fe11a .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-70fe11a .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-dab5086" data-block-id="dab5086"><style>.stk-dab5086 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-dab5086 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-dab5086 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-dab5086 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-dab5086 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-dab5086 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-dab5086 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-8c78bbc" data-block-id="8c78bbc"><style>.stk-8c78bbc .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-8c78bbc .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-8c78bbc .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-8c78bbc .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-8c78bbc .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-8c78bbc .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-8c78bbc .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-3319595" data-block-id="3319595"><style>.stk-3319595 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-3319595 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-3319595 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-3319595 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-3319595 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-3319595 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-3319595 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-8f3eb05" data-block-id="8f3eb05"><style>.stk-8f3eb05 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-8f3eb05 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-8f3eb05 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-8f3eb05 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-8f3eb05 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-8f3eb05 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-8f3eb05 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-06ccba7" id="h-proteins" data-block-id="06ccba7"><style>.stk-06ccba7 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Proteins</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>6465</td><td>P158-M</td><td>3DBionotes COVID-19 Structural Hub: a central resource for validation information and refined models</td><td>Jose Ramon Macias Gonzalez, Carolina Simon Guerrero, Erney Ramírez Aportela, Jose Luis Vilas Prieto, Marta Martinez Gonzalez, Carlos Oscar Sanchez Sorzano and Jose Maria Carazo</td></tr><tr><td>5334</td><td>P159-M</td><td>A graph based method for identifying and clustering short tandem repeats</td><td>Patryk Jarnot, Joanna Ziemska-Legięcka, Marcin Grynberg, Aleksandra Gruca and Vasilis Promponas</td></tr><tr><td>8454</td><td>P160-M</td><td>A graph-based algorithm for detecting rigid domains in protein structures</td><td>Linh Dang, Thach Nguyen and Michael Habeck</td></tr><tr><td>3233</td><td>P161-M</td><td>Alternative to combat superbugs – dynamic determinants of quorum quenching enzymes and their engineering towards efficient antibacterial potency</td><td>Bartlomiej Surpeta, Michal Grulich, Andrea Palyzová, Helena Marešová and Jan Brezovsky</td></tr><tr><td>6439</td><td>P162-M</td><td>AmyloGraph: A comprehensive database of amyloid-amyloid interactions</td><td>Michal Burdukiewicz, Dominik Rafacz, Agnieszka Barbach, Katarzyna Hubicka, Laura Bąkała, Anna Lassota, Jakub Stecko, Natalia Szymańska, Jakub Wojciechowski, Dominika Kozakiewicz, Natalia Szulc, Jarosław Chilimoniuk, Izabela Jęśkowiak, Marlena Gąsior-Głogowska and Malgorzata Kotulska</td></tr><tr><td>7693</td><td>P163-M</td><td>Analysis of fragment screening datasets in context with Human genetic variation</td><td>Javier Sánchez Utgés, Callum Ives, Stuart MacGowan and Geoff Barton</td></tr><tr><td>8789</td><td>P164-M</td><td>Annotating the regeneration transcriptome of Cloeon dipterum.</td><td>Patricia Medina-Burgos, Israel Barrios-Núñez, Fernando Casares and Ana Rojas</td></tr><tr><td>8539</td><td>P165-M</td><td>Annotation of biologically relevant ligands in UniProtKB using ChEBI</td><td>Elisabeth Coudert, Sebastien Gehant, Edouard de Castro, Monica Pozzato, Christian Sigrist, Delphine Baratin, Teresa Neto, Nicole Redaschi and Alan Bridge</td></tr><tr><td>9620</td><td>P166-M</td><td>APPRIS Principal Isoforms and MANE Select Transcripts Define Reference Splice Variants</td><td>Fernando Pozo, Laura Martinez Gomez, Jose Manuel Rodriguez, Jesús Vázquez and Michael Tress</td></tr><tr><td>7552</td><td>P167-M</td><td>ATHENA: Analysis of Tumor Heterogeneity from Spatial Omics Measurements</td><td>Adriano Martinelli, Pushpak Pati and Maria Anna Rapsomaniki</td></tr><tr><td>5457</td><td>P168-M</td><td>ATMision: a web portal for the in silico annotation of ATM missense variants beyond pathogenicity predictions</td><td>Natàlia Padilla Sirera, Luz Marina Porras, Alejandro Moles-Fernandez, Lidia Feliubadaló, Marta Santamariña-Pena, Alysson T. Sánchez, Anael López-Novo, Ana Blanco, Miguel de la Hoya, Ignacio J. Molina, Ana Osorio, Marta Pineda, Daniel Rueda, Clara Ruiz-Ponte, Ana Vega, Conxi Lázaro, Orland Díez, Sara Gutiérrez-Enríquez and Xavier de la Cruz</td></tr><tr><td>3336</td><td>P169-M</td><td>Automated System for Mechanistic Analysis and Interpretation of Genetic Variants</td><td>Ana C. González-Álvarez, Francisco J. Guzmán-Vega, Kelly J. Cardona-Londoño, Karla A. Peña-Guerra and Stefan T. Arold</td></tr><tr><td>1421</td><td>P170-M</td><td>Benchmarking of current conformational B-cell epitope prediction methods</td><td>Gabriel Cia, Fabrizio Pucci and Marianne Rooman</td></tr><tr><td>4693</td><td>P171-M</td><td>Biases and generalizability in predictions of protein-protein binding affinity changes upon mutations</td><td>Matsvei Tsishyn, Marianne Rooman and Fabrizio Pucci</td></tr><tr><td>7589</td><td>P172-M</td><td>CanProSite: Predicting potential residues associated with lung cancer using deep neural network</td><td>Medha Pandey and M. Michael Gromiha</td></tr><tr><td>3366</td><td>P173-M</td><td>Characterization of rare and novel AlphaFold structural space</td><td>Janani Durairaj, Mehmet Akdel, Pedro Beltrão and Torsten Schwede</td></tr><tr><td>1310</td><td>P174-M</td><td>Characterizing and explaining impact of disease-associated mutations in proteins without known structures or structural homologues</td><td>Neeladri Sen, Ivan Anishchanka, Nicola Bordin, Ian Sillitoe, Sameer Velankar, David Baker and Christine Orengo</td></tr><tr><td>6984</td><td>P175-M</td><td>COCOMAPS-2: an improved web tool for characterizing the interface of protein-protein and protein-nucleic acids complexes</td><td>Tiziana Ricciardelli, Mohit Chawla, Luigi Cavallo and Romina Oliva</td></tr><tr><td>2713</td><td>P176-M</td><td>Comparative clustering of eukaryote complexomes identifies novel taxon-specific protein complexes and interactors</td><td>Joeri van Strien, Felix Evers, Madhurya Lutikurti, Alfredo Cabrera Orefice, Ulrich Brandt, Taco W.A. Kooij and Martijn Huynen</td></tr><tr><td>7873</td><td>P177-M</td><td>Computational approach for the recognition of small molecule inhibitors for Toll-like receptor</td><td>Shailya Verma and Ramanathan Sowdhamini</td></tr><tr><td>2217</td><td>P178-M</td><td>Computational approach to identify multifunction by metamorphism in proteins</td><td>Israel Barrios Núñez and Ana Rojas M.</td></tr><tr><td>5151</td><td>P179-M</td><td>Computational function prediction in UniProt &amp; AI/ML community engagement</td><td>Vishal Joshi, Hermann Zellner and Maria Martin</td></tr><tr><td>2095</td><td>P180-M</td><td>Computational prediction of epitope-specific paratopes using convolutional neural networks</td><td>Dong Li, Marianne Rooman and Fabrizio Pucci</td></tr><tr><td>8237</td><td>P181-M</td><td>Conformine, a predictor of protein Conformational Variability from amino acid sequence</td><td>Jose Gavalda-Garcia and Wim Vranken</td></tr><tr><td>8964</td><td>P182-M</td><td>Conservation and evolution of Roquin-1 binding sites of known and novel targets in the T cell transcriptome</td><td>Giulia Cantini, Taku Ito-Kureha, Elaine H. Wong, Gesine Behrens, Lambert Moyon, Annalisa Marsico and Vigo Heissmeyer</td></tr><tr><td>3698</td><td>P183-M</td><td>Darwin: a side-chain positioning program with electron density map constraints based on an exact optimization framework</td><td>Nadege Polette, Mikael Grialou and David Allouche</td></tr><tr><td>8977</td><td>P184-M</td><td>DDGun: an untrained predictor of protein stability changes upon amino acid variants</td><td>Ludovica Montanucci, Emidio Capriotti, Giovanni Birolo, Silvia Benevenuta, Corrado Pnacotti, Dennis Lal and Piero Fariselli</td></tr><tr><td>3634</td><td>P185-M</td><td>De novo proteins from rice have a potential for forming structured entities</td><td>Francisco J. Guzmán-Vega, Yuanmin Zheng, Afaque A. Momin and Stefan T. Arold</td></tr><tr><td>7709</td><td>P186-M</td><td>Deciphering protein secretion from brain to CSF for biomarker discovery</td><td>Katharina Waury, Renske De Wit and Sanne Abeln</td></tr><tr><td>6002</td><td>P187-M</td><td>Deciphering the RRM-RNA recognition code: A computational analysis</td><td>Joel Roca-Martinez, Hrishikesh Dhondge and Wim Vranken</td></tr><tr><td>2400</td><td>P188-M</td><td>Deep-learning protein structure predictions suggest likely molecular functions for two uncharacterised polytopic membrane proteins from the P. falciparum apicoplast</td><td>David Murphy, Daniel Rigden, Shahram Mesdaghi, Filomeno Sanchez Rodriguez, Adam Simpkin and J Javier Burgos-Mármol</td></tr><tr><td>4524</td><td>P189-M</td><td>Designing next-generation kinase inhibitors using machine learning of structural and chemical features</td><td>Nicholas Clark, Ratul Chowdhury, Clemens Hug, Caitlin Mills, Peter Sorger and Mohammed Alquraishi</td></tr><tr><td>9196</td><td>P190-M</td><td>Development of a free interactive web portal for cytometry data gating</td><td>Robin Cohen, Gabor Beke, Lubos Klucar, Dana Cholujova and Jana Jakubikova</td></tr><tr><td>&#8211;</td><td>P191-M</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>4249</td><td>P192-M</td><td>Discovering novel genes in bacteria: gene function prediction using protein embeddings and synteny</td><td>Aysun Urhan, Bianca-Maria Cosma and Thomas Abeel</td></tr><tr><td>3271</td><td>P193-M</td><td>DistilProtBert: A distilled protein language model used to distinguish between real proteins and their randomly shuffled counterparts</td><td>Yaron Geffen, Yanay Ofran and Ron Unger</td></tr><tr><td>6785</td><td>P194-M</td><td>Drug repurposing for identification of potential spike inhibitors for SARS-CoV-2 using molecular docking and molecular dynamics simulations</td><td>Michal Lazniewski, Doni Dermawan and Dariusz Plewczynski</td></tr><tr><td>1377</td><td>P195-M</td><td>Elucidating important structural features for the binding affinity of spike &#8211; SARS-CoV-2 neutralizing antibody complexes</td><td>Divya Sharma and M. Michael Gromiha</td></tr><tr><td>8123</td><td>P196-M</td><td>Enzyme and transporter annotation in UniProtKB using Rhea and ChEBI</td><td>Lionel Breuza, Lucila Aimo, Ghislaine Argoud-Puy, Kristian Axelsen, Emmanuel Boutet, Cristina Casals-Casas, Elisabeth Coudert, Marc Feuermann, Nadine Gruaz-Gumowski, Damien Lieberherr, Michele Magrane, Anne Morgat, Nevila Hyka-Nouspikel, Lucille Pourcel, Sylvain Poux, Catherine Rivoire, Shyamala Sundaram, Rossana Zaru and Alan Bridge</td></tr><tr><td>9416</td><td>P197-M</td><td>E-SNPs&amp;GO: Embedding of protein sequence and function improves the prediction of human pathogenic variants</td><td>Matteo Manfredi, Castrense Savojardo, Pier Luigi Martelli and Rita Casadio</td></tr><tr><td>7308</td><td>P198-M</td><td>Expression regulation of protein complex partners as a compensatory mechanism in aneuploid tumors</td><td>Gokce Senger, Stefano Santaguida and Martin Schaefer</td></tr><tr><td>3691</td><td>P199-M</td><td>Features derived from protein 3D structure improve prediction of variant effects in novel proteins</td><td>Alexander Gress and Olga V. Kalinina</td></tr><tr><td>9407</td><td>P156-T</td><td>Foldseek: fast and accurate protein structure search</td><td>Michel van Kempen, Stephanie Kim, Charlotte Tumescheit, Milot Mirdita, Cameron Gilchrist, Johannes Soding and Martin Steinegger</td></tr><tr><td>7943</td><td>P157-T</td><td>FrustraEvo: Assessing Protein Families Divergence In The Light Of Sequence and Energetic Constraints</td><td>Victoria Ruiz-Serra, Maria Freiberger, Camila Pontes, Miguel Romero, Pablo Galaz-Davison, Cesar Ramirez-Sarmiento, Rodrigo Gonzalo Parra and Alfonso Valencia</td></tr><tr><td>4867</td><td>P158-T</td><td>Functional Impacts of Copy Number Imbalance in Aneuploid Cancer</td><td>Elle Loughran, Emmanuel Barillot, Andrei Zinovyev, Nicolas Servant and Aoife McLysaght</td></tr><tr><td>5684</td><td>P159-T</td><td>Gaussian accelerated molecular dynamics for the convergence of allosterism</td><td>Oriol Gracia i Carmona, Franca Fraternalli and Chris Oostenbrink</td></tr><tr><td>7011</td><td>P160-T</td><td>Highly significant improvement of protein sequence alignments with AlphaFold2</td><td>Leila Mansouri, Athanasios Baltzis, Suzanne Jin, Björn E. Langer, Ionas Erb and Cedric Notredame</td></tr><tr><td>2580</td><td>P161-T</td><td>High-throughput analyses of internal voids in biomolecules and ligand transport through them with TransportTools library</td><td>Jan Brezovsky, Carlos Eduardo Sequeiros-Borja, Aravind Selvaram Thirunavukarasu, Bartlomiej Surpeta, Nishita Mandal and Dheeraj Kumar Sarkar</td></tr><tr><td>7986</td><td>P162-T</td><td>Identification of protein-protein interaction hubs in multiple sclerosis</td><td>Gözde Yazıcı, Burcu Kurt Vatandaşlar, Bilal Kerman, Emre Karakoç and Can Alkan</td></tr><tr><td>&#8211;</td><td>P163-T</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>2821</td><td>P164-T</td><td>Improving Tandem Repeats Proteins annotation and classification in RepeatsDB</td><td>Martina Bevilacqua, Damiano Clementel, Alexander Monzon, Jiachen Lu, Paula Arrias and Silvio Tosatto</td></tr><tr><td>8661</td><td>P165-T</td><td>Investigation of Bacterial Fibrillar Adhesins and their Binding Characteristics</td><td>Vivian Monzon and Alex Bateman</td></tr><tr><td>1940</td><td>P166-T</td><td>Lower-order Statistics Facilitate Decoy-free FDR Control in Shotgun Proteomics</td><td>Dominik Madej and Henry Lam</td></tr><tr><td>1976</td><td>P167-T</td><td>Machine learning applications for the classification of serial ED data</td><td>Senik Matinyan, Burak Demir and Jan Pieter Abrahams</td></tr><tr><td>6279</td><td>P168-T</td><td>MobiDB: intrinsically disordered proteins in 2022</td><td>Alexander Monzon, Damiano Piovesan and Silvio Tosatto</td></tr><tr><td>526</td><td>P169-T</td><td>ModCRE: a structure homology-modeling approach to predict TF binding in cis-regulatory elements</td><td>Baldo Oliva, Oriol Fornes, Alberto Meseguer, Joaquim Aguirre-Plans, Patrick Gohl, Patricia-Mirela Bota, Ruben Molina-Fernandez, Altair Chinchilla, Ferran Pegenaute, Oriol Gallego, Narcis Fernandez-Fuentes and Jaume Bonet</td></tr><tr><td>6055</td><td>P170-T</td><td>nf-core/proteinfold: a bioinformatics best-practice pipeline for protein 3D structure prediction</td><td>Athanasios Baltzis, Jose Espinosa-Carrasco, Luisa Santus, Martin Steinegger, Harshil Patel and Cedric Notredame</td></tr><tr><td>9899</td><td>P171-T</td><td>Novel ADP-ribosyltransferase families in the Legionella genus</td><td>Marianna Krysińska, Krzysztof Pawłowski and Marcin Gradowski</td></tr><tr><td>3776</td><td>P172-T</td><td>Novel binding site descriptors built upon inverse virtual screening</td><td>Arnau Comajuncosa-Creus, Miquel Duran-Frigola, Xavier Barril and Patrick Aloy</td></tr><tr><td>9776</td><td>P173-T</td><td>Origin and evolution of Cas9 and Cas12 proteins</td><td>Darius Kazlauskas, Lukas Valančauskas and Česlovas Venclovas</td></tr><tr><td>8378</td><td>P174-T</td><td>PACT – Prediction of Amyloid Cross-interactions by Threading</td><td>Jakub Wojciechowski and Małgorzta Kotulska</td></tr><tr><td>8921</td><td>P175-T</td><td>PanPA: Construction and Alignments of Panproteome Graphs</td><td>Fawaz Dabbaghie, Sanjay Srikakulam, Olga Kalinina and Tobias Marschall</td></tr><tr><td>5271</td><td>P176-T</td><td>PrankWeb3 &#8211; ligand binding site prediction for PDB and AlphaFold structures</td><td>David Jakubec, Petr Škoda, Radoslav Krivak, Marian Novotny and David Hoksza</td></tr><tr><td>5389</td><td>P177-T</td><td>Predicting the binding affinity of antibodies with antigens</td><td>Nahin Khan, Hossam Almeer, Joao Palotti, Sanjay Chawla and Ehsan Ullah</td></tr><tr><td>213</td><td>P178-T</td><td>Predictive and comparative analysis of interaction hotspots of ACE2 orthologs in potential intermediate hosts for SARS-CoV-2</td><td>Myeongji Cho, Nara Been and Hyeon S. Son</td></tr><tr><td>1706</td><td>P179-T</td><td>ProtVar: Protein coding variant annotation</td><td>Rizwan Ishtiaq, James Stephenson, Alok Mishra and Maria Jesus Martin</td></tr><tr><td>6740</td><td>P180-T</td><td>Reduced structural flexibility of eplet amino acids in HLA proteins</td><td>Diego Amaya, Romain Lhotte, Magali Devriese, Constantin Hays, Jean-Luc Taupin and Marie-Dominique Devignes</td></tr><tr><td>1079</td><td>P181-T</td><td>Schistosoma mansoni MEG family proteins in the environment of host-parasite interactions</td><td>Stepanka Nedvedova, Kristyna Peterkova, Vojtech Vacek, Petr Mateju, Lukas Konecny, Jan Dvorak, Adriana Erica Miele, Francesca Fiorini Tregouët and Maggy Hologne</td></tr><tr><td>4751</td><td>P182-T</td><td>Single-cell map of Acute Myeloid Leukaemia</td><td>Alice Driessen, Susanne Unger, An-Phi Nguyen, Burkhard Becher and Maria Rodriguez Martinez</td></tr><tr><td>4296</td><td>P183-T</td><td>SparseChem: Fast and accurate machine learning model for small molecules</td><td>Adam Arany, Jaak Simm, Martijn Oldenhof and Yves Moreau</td></tr><tr><td>7467</td><td>P184-T</td><td>Spatial relationships in the urothelial cancer microenvironment, the potential for immune-based subtyping and immunotherapy response prediction</td><td>Alberto Gil-Jimenez, Nick van Dijk, Yoni Lubeck, Maurits L. van Montfoort, Erik Hooijberg, Annegien Broeks, Bas van Rhijn, Daniel J. Vis, Michiel S. van der Heijden and Lodewyk F. A. Wessels</td></tr><tr><td>1636</td><td>P185-T</td><td>Structural and functional insights into P protein</td><td>Shahram Mesdaghi</td></tr><tr><td>5173</td><td>P186-T</td><td>Structural modeling and computational analysis of uncharacterized anti-CRISPR Cas proteins</td><td>Lukas Valančauskas, Darius Kazlauskas and Česlovas Venclovas</td></tr><tr><td>431</td><td>P187-T</td><td>Structural modelling of odorant receptors from Aedes aegypti and search for natural repellents</td><td>Vikas Tiwari and Ramanathan Sowdhamini</td></tr><tr><td>2500</td><td>P188-T</td><td>Structure-activity relationships generated for peptaibols produced by Trichoderma: via accelerated MD simulations</td><td>Dóra Balázs, Chetna Tyagi, Tamás Marik, András Szekeres, Csaba Vágvölgyi and László Kredics</td></tr><tr><td>4272</td><td>P189-T</td><td>Studying the effect of phosphorylations on protein backbone dynamics</td><td>David Bickel and Wim Vranken</td></tr><tr><td>2408</td><td>P190-T</td><td>Superfamily analysis of rice protein structures reveals a variety of stress tolerance mechanisms</td><td>Fatima Shahid, Nicola Bordin, Christine Anne Orengo and Su Datt Lam</td></tr><tr><td>1322</td><td>P191-T</td><td>Targeting SARS-CoV-2 Endoribonuclease: A structure-based virtual screening supported by in vitro analysis</td><td>Ibrahim Mohamed, Abdo Elfiky, Mohamed Fathy, Sara Mahmoud and Mahmoud Elhefnawi</td></tr><tr><td>6099</td><td>P192-T</td><td>The alteration of structural network by transient association between proteins</td><td>Vasam Manjveekar Prabantu, Himani Tandon, Sankaran Sandhya and Narayanaswamy Srinivasan</td></tr><tr><td>1675</td><td>P193-T</td><td>The clinical importance of tandem exon duplication-derived substitutions</td><td>Laura Martinez-Gomez, Fernando Pozo Ocampo, Thomas A. Walsh, Federico Abascal and Michael Tress</td></tr><tr><td>9209</td><td>P194-T</td><td>The importance of protein-protein interactions in Toll-like receptor 8 functioning</td><td>Maria Bzówka, Weronika Bagrowska and Artur Góra</td></tr><tr><td>4827</td><td>P195-T</td><td>Development of continuous, protein-specific predictors of the impact of protein sequence variants</td><td>Selen Ozkan, Natàlia Padilla Sirera and Xavier de la Cruz</td></tr><tr><td>1026</td><td>P196-T</td><td>What is hidden in the darkness? A large-scale approach to make sense of all natural unknown proteins</td><td>Joana Pereira and Torsten Schwede</td></tr><tr><td>1937</td><td>P197-T</td><td>Where do they come from, where do they go? Prediction of protein subplastid localization and origin with PlastoGram.</td><td>Katarzyna Sidorczuk, Przemyslaw Gagat, Jakub Kała, Henrik Nielsen, Filip Pietluch, Paweł Mackiewicz and Michal Burdukiewicz</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-e9ef4db" data-block-id="e9ef4db"><style>.stk-e9ef4db{margin-top:50px !important}.stk-e9ef4db .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-85a8539" data-block-id="85a8539"><style>.stk-85a8539 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-85a8539 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-85a8539 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-85a8539 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-85a8539 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-85a8539 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-85a8539 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-b13c86b" data-block-id="b13c86b"><style>.stk-b13c86b .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-b13c86b .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-b13c86b .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-b13c86b .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-b13c86b .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-b13c86b .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-b13c86b .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-26169b5" data-block-id="26169b5"><style>.stk-26169b5 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-26169b5 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-26169b5 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-26169b5 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-26169b5 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-26169b5 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-26169b5 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-0fbf828" data-block-id="0fbf828"><style>.stk-0fbf828 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-0fbf828 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-0fbf828 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-0fbf828 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-0fbf828 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-0fbf828 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-0fbf828 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-bd1eb10" data-block-id="bd1eb10"><style>.stk-bd1eb10 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-bd1eb10 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-bd1eb10 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-bd1eb10 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-bd1eb10 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-bd1eb10 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-bd1eb10 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-2f85b00" data-block-id="2f85b00"><style>.stk-2f85b00 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-2f85b00 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-2f85b00 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-2f85b00 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-2f85b00 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-2f85b00 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-2f85b00 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-1d0fb8c" data-block-id="1d0fb8c"><style>.stk-1d0fb8c .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-1d0fb8c .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-1d0fb8c .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-1d0fb8c .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-1d0fb8c .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-1d0fb8c .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-1d0fb8c .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-47b1c7b" data-block-id="47b1c7b"><style>.stk-47b1c7b .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-47b1c7b .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-47b1c7b .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-47b1c7b .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-47b1c7b .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-47b1c7b .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-47b1c7b .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-c747678" id="h-systems" data-block-id="c747678"><style>.stk-c747678 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Systems</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>&#8211;</td><td>P200-M</td><td>&#8211;</td><td>&#8211;</td></tr><tr><td>8972</td><td>P201-M</td><td>A functional analysis of omic network embedding spaces reveals key altered functions in cancer.</td><td>Sergio Doria-Belenguer, Alexandros Xenos, Gaia Ceddia, Noel Malod and Natasa Przulj</td></tr><tr><td>1656</td><td>P202-M</td><td>A mathematical model for strigolactone biosynthesis in plants</td><td>Abel Lucido, Ester Vilaprinyo and Rui Alves</td></tr><tr><td>5069</td><td>P203-M</td><td>A Mechanistic Cellular Atlas of the Rheumatic Joint</td><td>Naouel Zerrouk, Sahar Aghakhani, Vidisha Singh, Franck Augé and Anna Niarakis</td></tr><tr><td>8238</td><td>P204-M</td><td>A user-friendly strain design pipeline enabling the reproducible identification of Knock-Out strategies for overproducing metabolites of interest.</td><td>Álvaro Gargantilla Becerra, David San León Granado and Juan Nogales Enrique</td></tr><tr><td>2450</td><td>P205-M</td><td>An agent-based model of tumor-associated macrophage differentiation in chronic lymphocytic leukemia</td><td>Nina Verstraete, Malvina Marku, Marcin Domagala, Helene Arduin, Jean-Jacques Fournié, Loic Ysebaert, Mary Poupot and Vera Pancaldi</td></tr><tr><td>6862</td><td>P206-M</td><td>An interpretable Graph Convolutional Network for predicting disease-causing genes involved in SARS-CoV-2 infection.</td><td>Samuele Firmani, Valter Bergant, Christoph Ogris, Annalisa Marsico and Andreas Pichlmair</td></tr><tr><td>7242</td><td>P207-M</td><td>Between viral targets and differentially expressed genes in viral infections: the sweet spot of disease mechanisms for therapeutic intervention</td><td>Carme Zambrana, Sam Windels, Noel Malod-Dognin and Nataša Pržulj</td></tr><tr><td>9762</td><td>P208-M</td><td>Bioinformatic analysis of prescription patterns and drug combinations of patients under peritoneal dialysis treatment</td><td>Michail Evgeniou, Fabian Eibensteiner, Klaus Kratochwill and Paul Perco</td></tr><tr><td>5292</td><td>P209-M</td><td>Biologically informed neural network identifies Unfolded Protein Response as key pathway in critical COVID-19</td><td>George Gavriilidis, Stella Dimitsaki, Fotis Psomopoulos and Vasileios Vasileiou</td></tr><tr><td>6265</td><td>P210-M</td><td>Changing terpenoid biosynthesis in rice through synthetic biology</td><td>Oriol Basallo, Rui Alves and Ester Vilaprinyo</td></tr><tr><td>1234</td><td>P211-M</td><td>Characterising Alternative Splicing Effects to Protein Interaction Networks with LINDA</td><td>Enio Gjerga and Christoph Dieterich</td></tr><tr><td>1680</td><td>P212-M</td><td>Comparison of methods and resources for cell-cell communication inference from single-cell RNA-Seq data</td><td>Daniel Dimitrov and Julio Saez-Rodriguez</td></tr><tr><td>6047</td><td>P213-M</td><td>Composing and organizing metabolic model perturbations and constraint systems with COBREXA.jl</td><td>Miroslav Kratochvíl, St Elmo Wilken, Laurent Heirendt, Reinhard Schneider, Christophe Trefois and Wei Gu</td></tr><tr><td>9793</td><td>P214-M</td><td>Computational prediction of time-course drug transcriptomic responses</td><td>Michio Iwata and Yoshihiro Yamanishi</td></tr><tr><td>8268</td><td>P215-M</td><td>Construction of the cystic fibrosis biological network from the meta-analysis of transcriptomic studies</td><td>Matthieu Najm, Loredana Martignetti, Matthieu Cornet, Mairead Aubert, Isabelle Sermet-Gaudelus, Laurence Calzone and Véronique Stoven</td></tr><tr><td>437</td><td>P216-M</td><td>Context-specific investigations of combined miRNA effects with DIANA-miRPath v4.0</td><td>Spyros Tastsoglou, Giorgos Skoufos, Marios Miliotis, Dimitra Karagkouni, Ioannis Koutsoukos, Anna Karavangeli, Filippos Kardaras and Artemis Hatzigeorgiou</td></tr><tr><td>8628</td><td>P217-M</td><td>Coupling of poro-aniso-hyperelastic and solute transport finite element models in a High-Performance Computing framework, for the study of Intervertebral Disc Degeneration</td><td>Dimitrios Lialios, Mariano Vazquez, Beatriz Eguzkitza, Eva Casoni and Maria Paola Ferri</td></tr><tr><td>5451</td><td>P218-M</td><td>DEPI v3: A systems biology and artificial intelligence based patient stratification and drug positioning platform for neurodevelopmental disorders</td><td>Laura Pérez-Cano, Francesco Sirci, Igor Ariz-Extreme, Daniel Boloc, Sara Azidane, José Hidalgo, Rubén Sabido, Lynn Durham and Emre Guney</td></tr><tr><td>4796</td><td>P219-M</td><td>DrDimont: Explainable drug response prediction from differential analysis of multi-omics networks</td><td>Pauline Hiort, Julian Hugo, Justus Zeinert, Nataniel Müller, Spoorthi Kashyap, Jagath C. Rajapakse, Francisco Azuaje, Bernhard Y. Renard and Katharina Baum</td></tr><tr><td>3375</td><td>P220-M</td><td>Estimating Strengths of Causal Interactions for Gene Regulatory Networks in Yeast</td><td>Adriaan Ludl, Tom Michoel and Mariyam Khan</td></tr><tr><td>4239</td><td>P221-M</td><td>Ethanol-induced sex-based differences in the extracellular vesicles lipidome</td><td>Carla Perpiñá-Clérigues, José F. Català-Senent, Susana Mellado, Consuelo Guerri, María Pascual and Francisco García-García</td></tr><tr><td>2978</td><td>P222-M</td><td>Evaluation of machine-learning methods using diverse representations and feature selection techniques for cell type annotation in single-cell transcriptomics data</td><td>Hyojin Kim, Rafael Kramann and Sikander Hayat</td></tr><tr><td>6941</td><td>P223-M</td><td>Flexible nets to optimize antibody production in chinese hamster ovary cells</td><td>Teresa Joven, Jorge Lázaro, Jorge Júlvez, Nicole Borth, Diana Széliová and Jürgen Zanghellini</td></tr><tr><td>7611</td><td>P224-M</td><td>GALLANT: A standardized workflow for multi strain Genome-scale metabolic modeling.</td><td>David San León Granado and Juan Nogales Enrique</td></tr><tr><td>6429</td><td>P225-M</td><td>Gene-essentiality based drug signature helps repurposing non-cancer drugs</td><td>Jing Tang and Wenyu Wang</td></tr><tr><td>4550</td><td>P226-M</td><td>Guiding CAR T cell experimental design using probabilistic graphical models</td><td>Alice Driessen, Rocío Castellanos Rueda, Constance le Gac, Nicolas Deutschman, Sai Reddy and Maria Rodriguez Martinez</td></tr><tr><td>7673</td><td>P227-M</td><td>Hybrid system based gene regulation models of circadian cycles</td><td>Lelde Lace, Gatis Melkus, Karlis Cerans and Juris Viksna</td></tr><tr><td>1478</td><td>P228-M</td><td>Identification of transcriptional network disruptions associated to drug resistance in cancer with TraRe</td><td>Charles Blatti, Jesús de la Fuente, Huanyao Gao, Irene Marín Goñi, Zikun Chen, Sihai Zao, Winston Tan, Richard Weinshilboum, Krishna Kalari, Liewei Wang and Mikel Hernaez</td></tr><tr><td>9741</td><td>P229-M</td><td>Identifying key multifunctional components shared by critical cancer and normal liver pathways via sparseGMM</td><td>Shaimaa Bakr, Kevin Brennan, Pritam Mukherjee, Josepmaria Argemi, Mikel Hernaez and Olivier Gevaert</td></tr><tr><td>3509</td><td>P230-M</td><td>Image-based simulation of morphogen gradient formation during zebrafish epiboly.</td><td>Justina Stark, Rohit Krishnan Harish, Michael Brand and Ivo F. Sbalzarini</td></tr><tr><td>9046</td><td>P231-M</td><td>Implementation of cellular transport mechanisms within a multiscale simulation framework</td><td>Othmane Hayoun-Mya, Arnau Montagud, Miguel Ponce-de-Leon and Alfonso Valencia</td></tr><tr><td>5320</td><td>P232-M</td><td>Improving xylose-fermenting yeast for 2G ethanol production via constraint-based modeling combined with omics data analysis</td><td>Lucas Carvalho, Vítor Pereira, Guido Araujo, Marcelo Carazzolle, Gonçalo A. G. Pereira and Miguel Rocha</td></tr><tr><td>4036</td><td>P233-M</td><td>Inferring biologically relevant molecular tissue substructures by agglomerative clustering of digitized spatial transcriptomes with multilayer</td><td>Marco Antonio Mendoza Parra and Julien Moehlin</td></tr><tr><td>6455</td><td>P234-M</td><td>Inferring pathway activities from gene expression data using perturbation transcription profiles of the LINCS-L1000 dataset</td><td>Bence Szalai, Szabolcs Hetey, Péter Szikora, Kristóf Szalay and Dániel V Veres</td></tr><tr><td>8148</td><td>P235-M</td><td>In-silico perturbation of transcription factors in a deep model of gene expression can predict tissue-specific regulation</td><td>Yuhu Liang, Viktoria Schuster, Thilde Terkelsen and Anders Krogh</td></tr><tr><td>8328</td><td>P236-M</td><td>Integrating and mining time-dependent single-cell RNA-seq data: Parkinson&#8217;s disease application</td><td>Katarina Mihajlovic, Gaia Ceddia, Noël Malod-Dognin, Gabriela Novak, Alexander Skupin, Dimitrios Kyriakis and Nataša Pržulj</td></tr><tr><td>2151</td><td>P237-M</td><td>ISMARA: completely automated inference of gene regulatory networks from high-throughput data.</td><td>Piotr Balwierz, Mikhail Pachkov, Phil Arnold, Andreas Gruber, Mihaela Zavolan and Erik van Nimwegen</td></tr><tr><td>5700</td><td>P238-M</td><td>Isoform-level quantification for single-cell RNA sequencing</td><td>Lu Pan, Huy Dinh, Yudi Pawitan and Trung Nghia Vu</td></tr><tr><td>5565</td><td>P239-M</td><td>Learning and reasoning with Bayesian networks to reconstruct cell signalling networks and automate hypothesis generation from phosphoproteomics data</td><td>Magdalena Huebner, Conrad Bessant and Pedro Cutillas</td></tr><tr><td>8993</td><td>P198-T</td><td>Leveraging multi-omic network embeddings to attain mechanistic insights into acute-on-chronic liver failure</td><td>Matthias Fabian Meyer-Bender, Pia Erdoesi, Maren Büttner, Ioannis Deligiannis, Rizqah Kamies, Ersin Karatayli, Matthias Ebert, Frank Lammert, Christine von Törne, Stefanie Hauck, Celia P. Martinez-Jimenez, Nikola Mueller, Steven Dooley, Michael Menden, Seddik Hammad and Christoph Ogris</td></tr><tr><td>5526</td><td>P199-T</td><td>Leveraging systems biology and machine learning for automatic drug repurposing in the rare disease landscape</td><td>Marina Esteban, Carlos Loucera, Maria Peña-Chilet and Joaquín Dopazo</td></tr><tr><td>4218</td><td>P200-T</td><td>Longitudinal analysis of biological age (Phenoage) in UKBiobank participants reveals key factors driving ageing trajectories</td><td>Laura Bravo, Victor Cardoso, John William, Dominic Russ, Samantha Pendleton, Furqan Aziz, Archana Sharma-Oates, Animesh Acharjee, Georgios Gkoutos and Janet Lord</td></tr><tr><td>4759</td><td>P201-T</td><td>Machine learning for functional reconstruction and analysis of Biochemical interactions controlling Colorectal cancer phenotypes</td><td>Victor Olorunshola, Ian Overton, Sandra Van Schaeybroeck, Alan Murphy and Erola Pairo-Castineira</td></tr><tr><td>9295</td><td>P202-T</td><td>Mathematically mapping the network of cells in the tumor microenvironment</td><td>M. van Santvoort, Ó. Lapuente-Santana, F. Finotello, W.L.F. van der Hoorn and F. Eduati</td></tr><tr><td>8530</td><td>P203-T</td><td>Metabolic Atlas &#8211; exploration and visualization of metabolic networks for model organisms</td><td>Mihail Anton, Nanjiang Shu, Ingrid Hyltander, Malin Klang, Per Johnsson, Shan Huang, L. Thomas Svensson and Jens Nielsen</td></tr><tr><td>161</td><td>P204-T</td><td>Metabolic Reprogramming in Rheumatoid Arthritis Synovial Fibroblast: a Hybrid Modeling Approach</td><td>Sahar Aghakhani, Sylvain Soliman and Anna Niarakis</td></tr><tr><td>7952</td><td>P205-T</td><td>miRarmature: a time series analysis pipeline for paired miRNA and RNA-seq data reveals new regulatory dynamics</td><td>Ranjan Kumar Maji, Ariane Fischer, Martin C. Simon, Stefanie Dimmeler and Marcel H. Schulz</td></tr><tr><td>3537</td><td>P206-T</td><td>Modeling the metabolic switch in Ewing Sarcoma from single-cell transcriptomic profiles</td><td>Marianyela Petrizzelli and Andrei Zinovyev</td></tr><tr><td>1213</td><td>P207-T</td><td>Molecular cross-talk communication between Intermuscular Adipose Tissue and Skeletal Muscle under progressing Insulin Resistance</td><td>Amare Wolide and Dominik Lutter</td></tr><tr><td>8056</td><td>P208-T</td><td>Multi-modal based predictions of vaccine-induced immune responses.</td><td>Fabio Affaticati, Abdulkader Azouz, Esther Bartholomeus, Benson Ogunjimi, Stanislas Goriely, Kris Laukens and Pieter Meysman</td></tr><tr><td>1194</td><td>P209-T</td><td>Multimodal Synthetic Lethality Prediction in Cancer</td><td>Yasin Tepeli, Colm Seale and Joana P. Gonçalves</td></tr><tr><td>9944</td><td>P210-T</td><td>Multi-Omics Visible Drug Activity prediction, interpreting the biological processes underlying drug sensitivity</td><td>Luigi Ferraro, Giovanni Scala and Michele Ceccarelli</td></tr><tr><td>3395</td><td>P211-T</td><td>Multiscale model of the different modes of invasion</td><td>Marco Ruscone, Arnau Montagud, Philippe Chavrier, Olivier Destaing, Andrei Zinovyev, Emmanuel Barillot, Vincent Noel and Laurence Calzone</td></tr><tr><td>5883</td><td>P212-T</td><td>Network analysis of Aicardi-Goutières syndrome-related genes &#8211; a systems biology approach</td><td>Gerda Cristal Villalba Silva and Shiaw-Yih Lin</td></tr><tr><td>4921</td><td>P213-T</td><td>Network medicine-based gene prioritization in Intervertebral Disc Degeneration (IDD)</td><td>Francesco Gualdi, Janet Piñero and Baldomero Oliva</td></tr><tr><td>9712</td><td>P214-T</td><td>Network-based driver identification with GoNetic</td><td>Simon Isphording, Giles Miclotte and Kathleen Marchal</td></tr><tr><td>1973</td><td>P215-T</td><td>Novel systemic approach using comorbid phenotype clusters to propose disease-causing genes in undiagnosed patients</td><td>José Córdoba-Caballero, Pedro Seoane-Zonjic, James R. Perkins, Elena Rojano and Juan A. G. Ranea</td></tr><tr><td>4592</td><td>P216-T</td><td>Numerical approaches for the rapid analysis of prophylactic efficacy against HIV</td><td>Lanxin Zhang, Junyu Wang and Max von Kleist</td></tr><tr><td>5411</td><td>P217-T</td><td>Omic fold changes clustering and network inference to study the radiation response of endothelial cells</td><td>Polina Arsenteva, Vincent Paget, Olivier Guipaud, Fabien Milliat, Hervé Cardot and Mohamed Amine Benadjaoud</td></tr><tr><td>8074</td><td>P218-T</td><td>Patient stratification reveals the molecular basis of comorbidities</td><td>Beatriz Urda-García, Jon Sánchez-Valle, Rosalba Lepore and Alfonso Valencia</td></tr><tr><td>8785</td><td>P219-T</td><td>Predicting EDC mode of action from toxicogenomics data using EDTox</td><td>Arindam Ghosh, Amirhossein Sakhteman, Raghavendra Mysore, Einari Niskanen, Thomas Darde, Pierre Daligaux, Christophe Chesné, Jorma Palvimo and Vittorio Fortino</td></tr><tr><td>687</td><td>P220-T</td><td>Prediction Of Shared Intratumor Transcriptional Heterogeneity From Bulk Cancer Transcriptomic Data</td><td>Agnieszka Kraft and Valentina Boeva</td></tr><tr><td>4320</td><td>P221-T</td><td>Reactome disease association overlay</td><td>Eliot Ragueneau, Deng Chuan, Krishna Tiwari, Chuqiao Gong, Guilherme Viteri and Henning Hermjakob</td></tr><tr><td>6186</td><td>P222-T</td><td>RedRedundance: A web app to summarize the results of functional enrichment analyzes for one or multiple gene lists</td><td>Mireia Ferrer, Esther Camacho, Berta Miró, Angel Blanco and Alex Sanchez</td></tr><tr><td>3146</td><td>P223-T</td><td>scRNA-Seq based drug repurposing for targeting alveolar regeneration in idiopathic pulmonary fibrosis</td><td>Anika Liu, Joo-Hyeon Lee, Andreas Bender and Namshik Han</td></tr><tr><td>1674</td><td>P224-T</td><td>Simulation of ground truth interaction networks from microbial community model</td><td>Ada Rossato, Marco Cappellato, Nora Nikoloska, Giacomo Baruzzo and Barbara Di Camillo</td></tr><tr><td>3390</td><td>P225-T</td><td>Single-cell multi-omics heterogenous multilayer network to infer gene regulatory mechanisms</td><td>Ina Maria Deutschmann, Rémi Trimbour and Laura Cantini</td></tr><tr><td>6756</td><td>P226-T</td><td>spongEffects: ceRNA modules offer patient-specific insights into the miRNA regulatory landscape</td><td>Fabio Boniolo, Markus Daniel Hoffmann, Norman Roggendorf, Bahar Tercan, Jan Baumbach, Mauro Antônio Castro, A. Gordon Robertson, Dieter Saur and Markus List</td></tr><tr><td>5329</td><td>P227-T</td><td>Stochastic Model of Intra-Tumor Heterogeneity (SMITH)</td><td>Adam Streck, Tom Kaufmann and Roland F. Schwarz</td></tr><tr><td>9633</td><td>P228-T</td><td>STOCHASTIC SIMULATION OF SARS-COV-2 SPREADING AND EVOLUTION ACCOUNT FOR WHITHIN HOST DYNAMICS (SIMPLICITY).</td><td>Pietro Gerletti, Max Von Kleist, Sofia Paraskevopoulou, Matthew Huska and Stephan Fuchs</td></tr><tr><td>4613</td><td>P229-T</td><td>Systematic Analysis of Alternative Splicing in Time Course Data of SARS-Cov2 infection development using Spycone</td><td>Chit Tong Lio, Zakaria Louadi, Amit Fenn, Jan Baumbach, Olga Tsoy, Tim Kacprowski and Markus List</td></tr><tr><td>1218</td><td>P230-T</td><td>The sample size value in Network Medicine: an application of gene co-expression networks</td><td>Joaquim Aguirre-Plans, Bingsheng Chen, Deisy Morselli Gysi and Albert-Laszlo Barabasi</td></tr><tr><td>9847</td><td>P231-T</td><td>Topological analysis as a tool for detection of abnormalities in protein-protein interaction data</td><td>Alicja Nowakowska and Malgorzata Kotulska</td></tr><tr><td>4430</td><td>P232-T</td><td>Towards a community-driven benchmark: PerMedCoE prepares a three agent-based modelling frameworks comparison.</td><td>Thaleia Ntiniakou, Arnau Montagud and Alfonso Valencia</td></tr><tr><td>9434</td><td>P233-T</td><td>Towards a data-driven network inference model of interactions between immune and cancer cells in Chronic Lymphocytic Leukaemia</td><td>Malvina Marku, Hugo Chenel, Julie Bordenave, Nina Verstraete, Leila Khajavi, Flavien Raynal and Vera Pancaldi</td></tr><tr><td>4707</td><td>P234-T</td><td>Tracking pathway activity in pseudotime</td><td>Priyansh Srivastava, Stefan Götz and Ana Conesa</td></tr><tr><td>378</td><td>P235-T</td><td>Tumor microenvironment evolution simulated through a hybrid Multi-Agent Spatio-Temporal model informed using sequencing data</td><td>Mikele Milia, Giulia Cesaro, Giacomo Baruzzo, Giovanni Finco, Francesco Morandini, Alessio Lazzarini, Piergiorgio Alotto, Noel Filipe da Cunha Carvalho de Miranda, Zlatko Trajanoski, Francesca Finotello and Barbara Di Camillo</td></tr><tr><td>9578</td><td>P236-T</td><td>Using neural networks to decipher non-equilibrium states of cell differentiation process</td><td>Susan Ghaderi, Alexander Skupin and Yves Moreau</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-7656bab" data-block-id="7656bab"><style>.stk-7656bab{margin-top:50px !important}.stk-7656bab .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-00b42bf" data-block-id="00b42bf"><style>.stk-00b42bf .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-00b42bf .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-00b42bf .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-00b42bf .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-00b42bf .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-00b42bf .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-00b42bf .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-02e128a" data-block-id="02e128a"><style>.stk-02e128a .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-02e128a .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-02e128a .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-02e128a .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-02e128a .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-02e128a .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-02e128a .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-60cc953" data-block-id="60cc953"><style>.stk-60cc953 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-60cc953 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-60cc953 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-60cc953 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-60cc953 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-60cc953 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-60cc953 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-9f9a811" data-block-id="9f9a811"><style>.stk-9f9a811 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-9f9a811 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-9f9a811 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-9f9a811 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-9f9a811 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-9f9a811 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-9f9a811 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-4ce0ca4" data-block-id="4ce0ca4"><style>.stk-4ce0ca4 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-4ce0ca4 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-4ce0ca4 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-4ce0ca4 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-4ce0ca4 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-4ce0ca4 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-4ce0ca4 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-cde10af" data-block-id="cde10af"><style>.stk-cde10af .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-cde10af .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-cde10af .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-cde10af .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-cde10af .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-cde10af .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-cde10af .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-af993ab" data-block-id="af993ab"><style>.stk-af993ab .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-af993ab .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-af993ab .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-af993ab .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-af993ab .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-af993ab .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-af993ab .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-dd27910" data-block-id="dd27910"><style>.stk-dd27910 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-dd27910 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-dd27910 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-dd27910 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-dd27910 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-dd27910 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-dd27910 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-8de27df" id="h-climate" data-block-id="8de27df"><style>.stk-8de27df .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Climate Crisis and Health</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>8710</td><td>P240-M</td><td>A metabolomic approach to understand the metabolic link between obesity and type 2 diabetes</td><td>Qiuling Dong, Sapna Sharma and Harald Grallert</td></tr><tr><td>4359</td><td>P241-M</td><td>APPLYING HUMAN EPIGENETIC INTER-VARIABILITY FOR PERSONALIZED NUTRITION STRATEGIES IN CANCER PATIENTS</td><td>Teresa Laguna, Marco Garranzo, Marta Gómez de Cedrón, Ana Ramírez de Molina and Enrique Carrillo de Santa Pau</td></tr><tr><td>7907</td><td>P242-M</td><td>Automated Diagnostic System for Medical Treatment of Infectious Diseases using Causal transfer learning and biological knowledge graph embedding</td><td>Sakhaa Alsaedi, Katsuhiko Mineta, Xin Gao and Takashi Gojobori</td></tr><tr><td>8386</td><td>P243-M</td><td>Characterization Of The Latent Resistome In External And Host-associated Environments</td><td>Juan Inda-Díaz, David Lund, Anna Johnning, Marcos Parras-Moltó, Johan Bengtsson-Palme and Erik Kristiansson</td></tr><tr><td>5869</td><td>P244-M</td><td>DeclaraMID: A Declarative Framework for Modeling Infectious Diseases</td><td>Sebastiaan Weytjens, Ann Nowé, Niel Hens and Pieter Libin</td></tr><tr><td>2209</td><td>P245-M</td><td>Deep Learning to differentiate between images of burnt and normal skin</td><td>Kirsty Smith, Sharmila Jivan, Ajay Mahajan, Khaled Jumah and Krzysztof Poterlowicz</td></tr><tr><td>9800</td><td>P246-M</td><td>EDAM and EDAM Geo: an ontology for data-intensive, interdisciplinary geo- and biosciences</td><td>Lucie Lamothe, Mads Kierkegaard, Melissa Black, Hager Eldakroury, Veit Schwämmle, Hamish Struthers, Bryan Brancotte, Kessy Abarenkov, Olga Silantyeva, Jean Iaquinta, Jon Ison, Jonathan Karr, Anne Fouilloux, Alban Gaignard, Hervé Ménager, Matúš Kalaš and And The Edam Community</td></tr><tr><td>8910</td><td>P237-T</td><td>Evaluating COVID-19 vaccine allocation policies using Bayesian m-top exploration</td><td>Alexandra Cimpean, Lander Willem, Timothy Verstraeten, Niel Hens, Ann Nowé and Pieter Libin</td></tr><tr><td>3303</td><td>P238-T</td><td>Exploring the Pareto front of multi-objective COVID-19 mitigation policies using reinforcement learning</td><td>Mathieu Reymond, Conor F Hayes, Lander Willem, Roxana Radulescu, Steven Abrams, Diederik M. Roijers, Enda Howley, Patrick Mannion, Niel Hens, Ann Nowé and Pieter Libin</td></tr><tr><td>5817</td><td>P239-T</td><td>Feed Your Model More Protein: Novel features based on viral protein composition and predicted interaction improve and slimline models of viral zoonoses</td><td>Matt Arnold, Simon Crouzet, Daniel Streicker and Simon Babayan</td></tr><tr><td>7396</td><td>P240-T</td><td>FOODRUGS: AN EXPLORATORY MINING WEB APPLICATION FOR FOOD &#8211; DRUG INTERACTIONS</td><td>Marco Garranzo, Teresa Laguna Lobo, David Pérez Serrano, Blanca Lacruz Pleguezuelos, María Isabel Espinosa, Ana Ramírez de Molina and Enrique Carrillo-De Santa Pau</td></tr><tr><td>3186</td><td>P241-T</td><td>Providing a dashboard for monitoring effects of contact behavior on the spread of SARS-CoV-2</td><td>Paul Chevelev, Richard Schiemenz, Ferdous Nasri, Richard Pastor, Bernhard Y Renard and Sten Rüdiger</td></tr><tr><td>3293</td><td>P242-T</td><td>Rooting virus phylogenies using UNREST and the general Markov model</td><td>Jose Nikhil Teja Dasari, Prabhav Kalaghatgi and Martin Vingron</td></tr><tr><td>7293</td><td>P243-T</td><td>Similarity network approach on transcriptomic data of patients with systemic infectious diseases.</td><td>Francesco Messina, Carolina Venditti, Carla Nisii, Carla Fontana and Alessandro Capone</td></tr><tr><td>345</td><td>P244-T</td><td>TCR-epitope recognition models to untangle unique and cross-reactive T-cell immunity in COVID-19 patients</td><td>Anna Postovskaya, Alexandra Vujkovic, Tessa de Block, Lida van Petersen, Maartje van Frankenhuijsen, Isabel Brosius, Emannuel Bottieau, Christophe Van Dijck, Caroline Theunissen, Sabrina van Ierssel, Erika Vlieghe, Esther Bartholomeus, Wim Adriaensen, Guido Vanham, Benson Ogunjimi, Koen Vercauteren, Kris Laukens and Pieter Meysman</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-4557ad2" data-block-id="4557ad2"><style>.stk-4557ad2{margin-top:50px !important}.stk-4557ad2 .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-3e4e0bd" data-block-id="3e4e0bd"><style>.stk-3e4e0bd .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-3e4e0bd .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-3e4e0bd .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-3e4e0bd .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-3e4e0bd .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-3e4e0bd .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-3e4e0bd .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-9afd41e" data-block-id="9afd41e"><style>.stk-9afd41e .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-9afd41e .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-9afd41e .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-9afd41e .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-9afd41e .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-9afd41e .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-9afd41e .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-9adfc02" data-block-id="9adfc02"><style>.stk-9adfc02 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-9adfc02 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-9adfc02 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-9adfc02 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-9adfc02 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-9adfc02 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-9adfc02 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-14f5328" data-block-id="14f5328"><style>.stk-14f5328 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-14f5328 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-14f5328 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-14f5328 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-14f5328 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-14f5328 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-14f5328 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-509ff69" data-block-id="509ff69"><style>.stk-509ff69 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-509ff69 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-509ff69 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-509ff69 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-509ff69 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-509ff69 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-509ff69 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-be1dd75" data-block-id="be1dd75"><style>.stk-be1dd75 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-be1dd75 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-be1dd75 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-be1dd75 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-be1dd75 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-be1dd75 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-be1dd75 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-74e1ae8" data-block-id="74e1ae8"><style>.stk-74e1ae8 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-74e1ae8 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-74e1ae8 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-74e1ae8 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-74e1ae8 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-74e1ae8 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-74e1ae8 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-6b779fa" data-block-id="6b779fa"><style>.stk-6b779fa .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-6b779fa .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-6b779fa .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-6b779fa .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-6b779fa .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-6b779fa .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-6b779fa .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-5c648ee" id="h-applications" data-block-id="5c648ee"><style>.stk-5c648ee .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Applications</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>9350</td><td>P247-M</td><td>Alternative splicing analysis benchmark with DICAST</td><td>Amit Fenn, Olga Tsoy, Tim Faro, Fanny Roessler, Alexander Dietrich, Johannes Kersting, Zakaria Louadi, Chit Tong Lio, Uwe Völker, Jan Baumbach, Tim Kacprowski and Markus List</td></tr><tr><td>8514</td><td>P248-M</td><td>Attention-based Variation Graph Autoencoder for DTA Prediction on Multiplex Heterogeneous Network</td><td>Dimitrios Papadopoulos, Bin Liu, David Čechák and Grigorios Tsoumakas</td></tr><tr><td>2281</td><td>P249-M</td><td>cellAssign: a small toolset to identify the cell types associated with methylation components</td><td>Reka Toth, Pavlo Lutsik, Christoph Plass and Petr V. Nazarov</td></tr><tr><td>3857</td><td>P250-M</td><td>Cell-type identification via functional enrichment analysis of Single-cell RNA-seq data</td><td>Marta Benegas Coll and Stefan Götz</td></tr><tr><td>4354</td><td>P251-M</td><td>ChIA-BERT: prediction of CTCF-mediated chromatin loops identified by Chromatin Interaction Analysis with Paired-End Tag (ChIA-PET) from DNA sequence</td><td>Mateusz Chiliński and Dariusz Plewczynski</td></tr><tr><td>8613</td><td>P252-M</td><td>Choosing variant interpretation tools for clinical applications: context matters</td><td>Xavier de la Cruz, Natàlia Padilla, Josu Aguirre, Selen Özkan, Casandra Riera and Lidia Feliubadaló</td></tr><tr><td>700</td><td>P253-M</td><td>Combined Quality Assessment of Different Long-Reads Sequencing Methods</td><td>Enrique Presa-Díez, Adolfo López-Cerdán and Stefan Götz</td></tr><tr><td>4478</td><td>P254-M</td><td>Continuous reference color scheme for genetic ancestry using CIELAB</td><td>Felix Pacheco, Mikaela Koutrouli, Karina Banasik, Lars Juhl Jensen and Søren Brunak</td></tr><tr><td>3671</td><td>P255-M</td><td>CROSTA: CROss-Species Transmissibility Analyser for Pathogen Sequences</td><td>Shan Tharanga, Eyyüb Selim Ünlü, Esra Büşra Işık, Muhammad Farhan Sjaugi and Mohammad Asif Khan</td></tr><tr><td>4052</td><td>P256-M</td><td>Current activities of the ELIXIR Machine Learning Focus Group</td><td>Fotis Psomopoulos, Emidio Capriotti, Núria Queralt Rosinach, Machine Learning Focus Group Tasks Participants, Leyla Jael Castro and Silvio Tostto</td></tr><tr><td>7403</td><td>P257-M</td><td>DA4LT: an efficient computational method for cross-species label transfer from single cell data</td><td>Rawan Olayan, Sergio Picart-Armada, Gregorio Alanis-Lobato, Francesc Fernandez-Albert, Timothy B Sackton and Stefano Patassini</td></tr><tr><td>6882</td><td>P258-M</td><td>fCAT &#8211; Assessing gene set completeness using domain-architecture aware targeted ortholog searches</td><td>Vinh Tran and Ingo Ebersberger</td></tr><tr><td>6199</td><td>P259-M</td><td>FrankenSeq: A user-friendly and modular machine learning package for Single-cell RNA-Seq Cluster Analysis</td><td>Edward Agboraw and Quan Gu</td></tr><tr><td>3471</td><td>P260-M</td><td>Functional Enrichment Analysis of Regulatory Elements with GeneCodis4</td><td>Adrian Garcia-Moreno, Raul Lopez-Dominguez, Juan Antonio Villatoro-Garcia, Samuel Perez-Fernandez and Pedro Carmona-Saez</td></tr><tr><td>1741</td><td>P261-M</td><td>GeneHTracker: improving reproducibility and reusability of datasets based on gene identifiers.</td><td>Hugo Guillen-Ramirez, Daniel Sanchez-Taltavull and Rory Johnson</td></tr><tr><td>5168</td><td>P262-M</td><td>Implications of phenotypic and functional heterogeneity in cancer cells</td><td>Asadullah and Shamik Sen</td></tr><tr><td>7281</td><td>P263-M</td><td>Mathematica as a platform to develop tools for education, data representation, and analysis in epidemiology</td><td>Rui Alves, Ester Vilaprinyo, Alberto Marin-Sanguino and Albert Sorribas</td></tr><tr><td>9084</td><td>P245-T</td><td>MicroNAP-web: A novel framework to characterize genomic and transcriptomic changes in engineered microbial strains based on Next Generation Sequencing data.</td><td>Veronika Schusterbauer, Daniel Degreif, Christoph Schiklenk, Anton Glieder and Gerhard Thallinger</td></tr><tr><td>4055</td><td>P246-T</td><td>MinSizeML: An R Package to Estimate the Minimum Sample Size in Supervised Learning for Classification</td><td>Guillermo Prol Castelo and Jose Luis Mosquera</td></tr><tr><td>4616</td><td>P247-T</td><td>Multi-Omics Analysis and Metabolic Network Construction in suberinization of cork oak.</td><td>Nuria Mauri, Maria Verdum, Patrícia Jové, Ignacio Ontañon, Jordi Roselló, Vicente Ferreira and David Caparrós-Ruíz</td></tr><tr><td>9890</td><td>P248-T</td><td>New algorithms for accurate and efficient de-novo genome assembly from long DNA sequencing reads</td><td>David Guevara-Barrientos, Laura Natalia González García, Daniela Lozano, Juanita Gil, Maria Camila Hoyos, Christian Chavarro, Natalia Guayazan, Luis Chica, Maria Camila Buitrago, Edwin Bautista, Juan Camilo Bojacá, Miller Andrés Trujillo Achury and Jorge Duitama</td></tr><tr><td>2358</td><td>P249-T</td><td>Organisation-wide data discovery at the European Bioinformatics Institute</td><td>Matthew Pearce, Prasad Basutkar, Ossama Edbali, Anton Kolesnikov, Henning Hermjakob and Rodrigo Lopez</td></tr><tr><td>8039</td><td>P250-T</td><td>PaintOmics 4: new tools for the integrative analysis of multi-omics datasets supported by multiple pathway databases</td><td>Tianyuan Liu, Pedro Salguero, Marko Petek, Carlos Martinez-Mira, Leandro Balzano-Nogueira, Živa Ramšak, Lauren McIntyre, Kristina Gruden, Sonia Tarazona and Ana Conesa</td></tr><tr><td>9080</td><td>P251-T</td><td>Predicting clinical response to immunotherapy in advanced melanoma</td><td>Matthieu Genais, Bruno Ségui, Anne Montfort and Vera Pancaldi</td></tr><tr><td>6768</td><td>P252-T</td><td>Secured and annotated execution of workflows with WfExS-backend</td><td>José M. Fernández, Laura Rodríguez-Navas and Salvador Capella-Gutierrez</td></tr><tr><td>1139</td><td>P253-T</td><td>SMASCH: Facilitating multi-appointment scheduling in longitudinal clinical research studies and care programs.</td><td>Carlos Vega, Piotr Gawron, Jacek Lebioda, Valentin Grouès, Rejko Krüger, Reinhard Schneider and Venkata Satagopam</td></tr><tr><td>7424</td><td>P254-T</td><td>The Bioinfo4Women Research Working Group on Sex and Gender Bias in Healthcare and AI</td><td>Atia Cortés Martínez, Davide Cirillo, Maria José Rementeria and Nataly Buslon</td></tr><tr><td>9077</td><td>P255-T</td><td>The ELIXIR::GA4GH Cloud</td><td>Alexander Kanitz, Justin Clark-Casey, Michael Crusoe, Gavin Farrell, Alvaro Gonzalez, Thanasis Vergoulis and Jonathan Tedds</td></tr><tr><td>9980</td><td>P256-T</td><td>The Job Dispatcher sequence analysis tools services from EMBL-EBI in 2022</td><td>Fábio Madeira, Joon Lee, Adrian Tivey, Nandana Madhusoodanan, Sarah Butcher and Rodrigo Lopez</td></tr><tr><td>1283</td><td>P257-T</td><td>The Service Collections of the ELIXIR Rare Disease Community</td><td>Emidio Capriotti, David Salgado, Friederike Ehrhart, Allegra Via, Claudio Carta, Marko Vidak, Rare Disease Community Contributors And Work Package Leaders, Marco Roos and Sergi Beltran</td></tr><tr><td>6207</td><td>P258-T</td><td>VaRHC: an R package for semi-automate ACMG/AMP variant classification on hereditary cancer genes according to gene-specific ClinGen guidelines</td><td>Elisabet Munté, Lidia Feliubadaló, Marta Pineda, Eva Tornero, Maria Isabel González-Acosta, Carla Roca, Jose Marcos Moreno-Cabrera, Concepción Lázaro and Jose Luis Mosquera</td></tr><tr><td>4360</td><td>P259-T</td><td>Whole-genome sequencing analysis of food enzyme products reveals contaminations with genetically modified microorganism of related origin</td><td>Jolien D&#8217;Aes, Marie-Alice Fraiture, Bert Bogaerts, Sigrid C.J. De Keersmaecker, Nancy H.C. Roosens and Kevin Vanneste</td></tr><tr><td>9788</td><td>P260-T</td><td>WiNGS: Widely integrated NGS platform for federated genome analysis</td><td>Haleh Chizari, Nasim Lalani Shabani, Nishkala Sattanathan, Geert Vandeweyer and Yves Moreau</td></tr></tbody></table></figure>



<div class="wp-block-stackable-button-group anclas stk-block-button-group stk-block stk-12a36c4" data-block-id="12a36c4"><style>.stk-12a36c4{margin-top:50px !important}.stk-12a36c4 .stk-button-group{flex-wrap:wrap !important}</style><div class="stk-row stk-inner-blocks has-text-align-left stk-block-content stk-button-group">
<div class="wp-block-stackable-button stk-block-button stk-block stk-7a54e11" data-block-id="7a54e11"><style>.stk-7a54e11 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-7a54e11 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-7a54e11 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-7a54e11 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-7a54e11 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-7a54e11 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-7a54e11 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-data"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Data</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-2b5f9c7" data-block-id="2b5f9c7"><style>.stk-2b5f9c7 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-2b5f9c7 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-2b5f9c7 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-2b5f9c7 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-2b5f9c7 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-2b5f9c7 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-2b5f9c7 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-38ef707" data-block-id="38ef707"><style>.stk-38ef707 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-38ef707 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-38ef707 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-38ef707 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-38ef707 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-38ef707 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-38ef707 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-genomes"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Genomes</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-fae3bdf" data-block-id="fae3bdf"><style>.stk-fae3bdf .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-fae3bdf .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-fae3bdf .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-fae3bdf .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-fae3bdf .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-fae3bdf .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-fae3bdf .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-proteins"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Proteins</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-62f7ad0" data-block-id="62f7ad0"><style>.stk-62f7ad0 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-62f7ad0 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-62f7ad0 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-62f7ad0 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-62f7ad0 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-62f7ad0 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-62f7ad0 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-systems"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Systems</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-e1b3d69" data-block-id="e1b3d69"><style>.stk-e1b3d69 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-e1b3d69 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-e1b3d69 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-e1b3d69 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-e1b3d69 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-e1b3d69 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-e1b3d69 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-climate"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Climate Crisis and Health</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-35f7798" data-block-id="35f7798"><style>.stk-35f7798 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-35f7798 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-35f7798 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-35f7798 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-35f7798 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-35f7798 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-35f7798 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-applications"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Applications</span></a></div>



<div class="wp-block-stackable-button stk-block-button stk-block stk-841c9a0" data-block-id="841c9a0"><style>.stk-841c9a0 .stk-button{background:var(--paletteColor1,#ffffff) !important}.stk-841c9a0 .stk-button:hover:after{background:var(--paletteColor2,#f7b032) !important;opacity:1 !important}.stk-841c9a0 .stk-button{border-radius:5px !important;overflow:hidden !important}.stk-841c9a0 .stk-button:before{border-style:solid !important;border-color:var(--paletteColor4,#0081a7) !important;border-top-width:1px !important;border-right-width:1px !important;border-bottom-width:1px !important;border-left-width:1px !important}.stk-841c9a0 .stk-button:hover:before{border-color:var(--paletteColor2,#f7b032) !important}.stk-841c9a0 .stk-button__inner-text{color:var(--paletteColor4,#0081a7) !important}.stk-841c9a0 .stk-button:hover .stk-button__inner-text{color:var(--paletteColor1,#ffffff) !important}</style><a class="stk-link stk-button stk--hover-effect-darken" href="#h-training"><span class="has-text-color has-palette-color-4-color stk-button__inner-text">Training</span></a></div>
</div></div>



<div class="wp-block-stackable-heading stk-block-heading stk-block stk-efba7f3" id="h-training" data-block-id="efba7f3"><style>.stk-efba7f3 .stk-block-heading__bottom-line{height:2px !important;width:50px !important;background-color:var(--paletteColor4,#0081a7) !important;margin-top:20px !important}</style><h2 class="stk-block-heading__text">Training</h2><div class="stk-block-heading__bottom-line"></div></div>



<figure class="wp-block-table is-style-stripes tabla-pres"><table><tbody><tr><td>Ref.</td><td>Poster Nº</td><td>Title</td><td>Authors</td></tr><tr><td>9898</td><td>P264-M</td><td>Bioinformatics core facility management training &#8211; availability and challenges</td><td>Eva Alloza, Ezgi Karaca, Salvador Capella-Gutiérrez, Cath Brooksbank and Patricia Carvajal-López</td></tr><tr><td>1053</td><td>P265-M</td><td>Community building and training in the EMBL Bio-IT project</td><td>Renato Alves and Lisanna Paladin</td></tr><tr><td>5405</td><td>P266-M</td><td>ELIXIR-hCNV: Galaxy workflows and training</td><td>Khaled Jumah, Katarzyna Kamienicka, Wolfgang Maier, David Salgado, Christophe Béroud, Michael Baudis, Steven Laurie, Tim Beck, Salvador Capella-Gutierrez, Björn Grüning and Krzysztof Poterlowicz</td></tr><tr><td>1143</td><td>P267-M</td><td>ELIXIR-UK DaSH: A Fellowship of data stewards</td><td>Xenia Perez Sitja, Robert Andrews, Sara Khalil, Branka Franicevic, Catherine Knox, Munazah Andrabi, Shoaib Sufi, Neil Hall, Susanna-Assunta Sansone, Carole Goble and Krzysztof Poterlowicz</td></tr><tr><td>9922</td><td>P268-M</td><td>GOBLET: Unite, inspire and equip Bioinformatics Trainers worldwide</td><td>Javier De Las Rivas, Eija Korpelainen, Annette McGrath, Asif M. Khan and Celia W.G. van Gelder</td></tr><tr><td>145</td><td>P261-T</td><td>Investigating Student Sense of Belonging in Biology and Computer Science</td><td>Shamima Runa, Brett Becker and Catherine Mooney</td></tr><tr><td>4223</td><td>P262-T</td><td>The Bioinfo4Women International Pilot Mentoring Programme for Young Scientists</td><td>Alba Jene-Sanz, Olfat Khannous-Lleiffe, Othmane Hayoun-Mya, Maria Sopena-Rios, Mireia Codina-Tobías, Eva Alloza, Àtia Cortés and Maria José Rementeria</td></tr><tr><td>2690</td><td>P263-T</td><td>The EOSC-Life Training programme as a tool for bringing together a community of practice</td><td>Daniel Thomas-Lopez, Rebecca Ludwig and Vera Matser</td></tr><tr><td>369</td><td>P264-T</td><td>The new Community of Practice for Data Management/Data Stewardship Training</td><td>Helena Schnitzer, Nils-Christian Lübke, Irena Maus, Tanja Dammann-Kalinowski, Robert Andrews, Alexia Cardona, Fatima Nazeefa, Celia van Gelder, Mijke Jetten, Brane L. Leskosek, Jessica Lindvall, Ana Portugal Melo, Wolmar Nyberg Åkerström, Krzysztof Poterlowicz, Olivier Sand, Andreas Tauch, Marko Vidak and Daniel Wibberg</td></tr><tr><td>3188</td><td>P265-T</td><td>The new de.NBI / ELIXIR-DE training platform in 2022</td><td>Daniel Wibberg, Nils Christian Lübke, Andreas Tauch and Helena Schnitzer</td></tr></tbody></table></figure>



<div style="height:50px" aria-hidden="true" class="wp-block-spacer"></div>
		</div>

		
		
		
		
	</article>

	
		
			</div>

	</main>

	<footer class="ct-footer" data-id="type-1" itemscope="" itemtype="https://schema.org/WPFooter"><div data-row="bottom"><div class="ct-container"><div data-column="widget-area-1"><div class="ct-widget widget_block widget_media_image" id="block-8">
<figure class="wp-block-image size-full"><a href="https://www.bcocongresos.com" target="_blank"><img decoding="async" loading="lazy" width="400" height="316" src="../wp-content/uploads/2022/01/Logo-Bco-footer-2.png" alt="" class="wp-image-362" srcset="../wp-content/uploads/2022/01/Logo-Bco-footer-2.png 400w, ../wp-content/uploads/2022/01/Logo-Bco-footer-2-300x237.png 300w" sizes="(max-width: 400px) 100vw, 400px"></a></figure>
</div></div><div data-column="widget-area-2"><div class="ct-widget widget_block widget_text" id="block-13">
<p><strong>TECHNICAL SECRETARIAT</strong></p>
</div><div class="ct-widget widget_block widget_text" id="block-14">
<p>Plaça Europa 17-19, 1st floor<br>08908 L´Hospitalet de Llobregat, Barcelona (Spain)</p>
</div><div class="ct-widget widget_block widget_text" id="block-12">
<p><a href="tel:+34938823878">+34 93 882 38 78</a><br><a href="mailto:eccb2020@bcocongresos.com">eccb2022@bcocongresos.com</a></p>
</div></div><div data-column="text">
<div class="ct-header-text " data-id="text">
	<div class="entry-content">
		<p><strong>ORGANISED BY</strong></p>
<div class="logos-org"><a class="logo-1" href="https://www.bsc.es/" target="_blank" rel="noopener"> <img src="../wp-content/uploads/2022/02/logo-bsc.png"> </a> <a class="logo-2" href="https://inb-elixir.es/" target="_blank" rel="noopener"> <img src="../wp-content/uploads/2022/02/logo-inb.svg"> </a> <a class="logo-3" href="http://inb-elixir.es" target="_blank" rel="noopener"> <img src="../wp-content/uploads/2022/02/logo-elixir.svg"> </a></div>	</div>
</div>
</div></div></div></footer></div>

<div class="ct-drawer-canvas"><div id="offcanvas" class="ct-panel ct-header" data-behaviour="right-side"><div class="ct-panel-inner">
		<div class="ct-panel-actions">
			<button class="ct-toggle-close" data-type="type-1" aria-label="Close drawer">
				<svg class="ct-icon" width="12" height="12" viewbox="0 0 15 15">
					<path d="M1 15a1 1 0 01-.71-.29 1 1 0 010-1.41l5.8-5.8-5.8-5.8A1 1 0 011.7.29l5.8 5.8 5.8-5.8a1 1 0 011.41 1.41l-5.8 5.8 5.8 5.8a1 1 0 01-1.41 1.41l-5.8-5.8-5.8 5.8A1 1 0 011 15z"></path>
				</svg>
			</button>
		</div>
		<div class="ct-panel-content" data-device="desktop"></div><div class="ct-panel-content" data-device="mobile">
<nav class="mobile-menu has-submenu" role="navigation" data-id="mobile-menu" data-interaction="click" data-toggle-type="type-1">
	<ul id="menu-general-1" class=""><li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-122"><span class="ct-sub-menu-parent"><a href="#" class="ct-menu-link">About</a><button class="ct-toggle-dopdown-mobile" aria-label="Expand dropdown menu" aria-expanded="false"><svg class="ct-icon toggle-icon-1" width="15" height="15" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M3.9,5.1l3.6,3.6l3.6-3.6l1.4,0.7l-5,5l-5-5L3.9,5.1z"></path></svg></button></span>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-209"><a href="../welcome/index.htm" class="ct-menu-link">Welcome</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-246"><a href="../committees/index.htm" class="ct-menu-link">Committees</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-310"><a href="../key-dates/index.htm" class="ct-menu-link">Key dates</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-2630"><span class="ct-sub-menu-parent"><a href="#" class="ct-menu-link">Sponsors</a><button class="ct-toggle-dopdown-mobile" aria-label="Expand dropdown menu" aria-expanded="false"><svg class="ct-icon toggle-icon-1" width="15" height="15" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M3.9,5.1l3.6,3.6l3.6-3.6l1.4,0.7l-5,5l-5-5L3.9,5.1z"></path></svg></button></span>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-319"><a href="../sponsors/index.htm" class="ct-menu-link">Become a sponsor</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2629"><a href="../confirmed-sponsors-exhibitors/index.htm" class="ct-menu-link">Confirmed sponsors &#038; exhibitors</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1170"><span class="ct-sub-menu-parent"><a href="#" class="ct-menu-link">Programme</a><button class="ct-toggle-dopdown-mobile" aria-label="Expand dropdown menu" aria-expanded="false"><svg class="ct-icon toggle-icon-1" width="15" height="15" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M3.9,5.1l3.6,3.6l3.6-3.6l1.4,0.7l-5,5l-5-5L3.9,5.1z"></path></svg></button></span>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-313"><a href="../final-programme/index.htm" class="ct-menu-link">Final programme</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-908"><a href="../preliminary-programme-at-a-glance/index.htm" class="ct-menu-link">Programme at a glance</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2386"><a href="../keynote-speakers/index.htm" class="ct-menu-link">Keynote Speakers</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1799"><a href="../workshop-and-tutorials/index.htm" class="ct-menu-link">Workshops and tutorials</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-2200"><a href="../satellite-meetings/index.htm" class="ct-menu-link">Satellite Meetings</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3202"><a href="../birds-of-a-feather/index.htm" class="ct-menu-link">Birds of a Feather</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3452"><a href="../eccb2022-best-poster-awards/index.htm" class="ct-menu-link">ECCB2022 Best Poster Awards</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-919"><a href="../registration/index.htm" class="ct-menu-link">Registration</a></li>
<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-has-children menu-item-1163"><span class="ct-sub-menu-parent"><a href="#" class="ct-menu-link">Practicals</a><button class="ct-toggle-dopdown-mobile" aria-label="Expand dropdown menu" aria-expanded="false"><svg class="ct-icon toggle-icon-1" width="15" height="15" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M3.9,5.1l3.6,3.6l3.6-3.6l1.4,0.7l-5,5l-5-5L3.9,5.1z"></path></svg></button></span>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1167"><a href="../venue/index.htm" class="ct-menu-link">Venue</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1165"><a href="../travel/index.htm" class="ct-menu-link">Travel</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1166"><a href="../hotels/index.htm" class="ct-menu-link">Hotels</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-1164"><a href="../sitges/index.htm" class="ct-menu-link">Sitges</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3168"><a href="../app/index.htm" class="ct-menu-link">Official APP</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3200"><a href="../safety-protocols-and-accesibilty-statement/index.htm" class="ct-menu-link">Safety protocols and accessibility information</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-post_type menu-item-object-page current-menu-ancestor current-menu-parent current_page_parent current_page_ancestor menu-item-has-children menu-item-459"><span class="ct-sub-menu-parent"><a href="../calls/index.htm" class="ct-menu-link">Calls</a><button class="ct-toggle-dopdown-mobile" aria-label="Expand dropdown menu" aria-expanded="false"><svg class="ct-icon toggle-icon-1" width="15" height="15" viewbox="0 0 15 15" aria-label="Menu dropdown icon"><path d="M3.9,5.1l3.6,3.6l3.6-3.6l1.4,0.7l-5,5l-5-5L3.9,5.1z"></path></svg></button></span>
<ul class="sub-menu">
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-3073"><a href="../guidelines/index.htm" class="ct-menu-link">Presentations guidelines</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page current-menu-item page_item page-item-2905 current_page_item menu-item-3072"><a href="index.htm" aria-current="page" class="ct-menu-link">Guidelines for poster presentations</a></li>
</ul>
</li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-643"><a href="../code-of-conduct/index.htm" class="ct-menu-link">Code of Conduct</a></li>
</ul></nav>


<div class="ct-header-socials " data-id="socials">

	
		<div class="ct-social-box" data-icon-size="custom" data-color="custom" data-icons-type="rounded:solid">
			
			
							
				<a href="https://twitter.com/ECCBinfo" data-network="twitter" aria-label="Twitter" target="_blank" rel="noopener">
					<span class="ct-icon-container">
				<svg class="ct-icon" width="20px" height="20px" viewbox="0 0 20 20" aria-label="Twitter Icon">
					<path d="M20,3.8c-0.7,0.3-1.5,0.5-2.4,0.6c0.8-0.5,1.5-1.3,1.8-2.3c-0.8,0.5-1.7,0.8-2.6,1c-0.7-0.8-1.8-1.3-3-1.3c-2.3,0-4.1,1.8-4.1,4.1c0,0.3,0,0.6,0.1,0.9C6.4,6.7,3.4,5.1,1.4,2.6C1,3.2,0.8,3.9,0.8,4.7c0,1.4,0.7,2.7,1.8,3.4C2,8.1,1.4,7.9,0.8,7.6c0,0,0,0,0,0.1c0,2,1.4,3.6,3.3,4c-0.3,0.1-0.7,0.1-1.1,0.1c-0.3,0-0.5,0-0.8-0.1c0.5,1.6,2,2.8,3.8,2.8c-1.4,1.1-3.2,1.8-5.1,1.8c-0.3,0-0.7,0-1-0.1c1.8,1.2,4,1.8,6.3,1.8c7.5,0,11.7-6.3,11.7-11.7c0-0.2,0-0.4,0-0.5C18.8,5.3,19.4,4.6,20,3.8z"></path>
				</svg>
			</span><span class="ct-label" hidden="">Twitter</span>				</a>
			
			
					</div>

	
</div>

<div class="ct-header-cta" data-id="button">

	<a href="https://barcelo.eventsair.com/bbdd-eccb2022/keepmeupdated/Site/Register" class="ct-button" data-size="small" target="_blank" rel="noopener noreferrer">
		Keep me updated	</a>
</div>

</div></div></div></div><script>requestAnimationFrame(() => document.body.classList.add( "stk--anim-init" ))</script><div id="pum-2358" class="pum pum-overlay pum-theme-1538 pum-theme-lightbox popmake-overlay click_open" data-popmake="{&quot;id&quot;:2358,&quot;slug&quot;:&quot;maria-rodriguez-martinez&quot;,&quot;theme_id&quot;:1538,&quot;cookies&quot;:[],&quot;triggers&quot;:[{&quot;type&quot;:&quot;click_open&quot;,&quot;settings&quot;:{&quot;extra_selectors&quot;:&quot;&quot;,&quot;cookie_name&quot;:null}}],&quot;mobile_disabled&quot;:null,&quot;tablet_disabled&quot;:null,&quot;meta&quot;:{&quot;display&quot;:{&quot;stackable&quot;:false,&quot;overlay_disabled&quot;:false,&quot;scrollable_content&quot;:false,&quot;disable_reposition&quot;:false,&quot;size&quot;:&quot;medium&quot;,&quot;responsive_min_width&quot;:&quot;0%&quot;,&quot;responsive_min_width_unit&quot;:false,&quot;responsive_max_width&quot;:&quot;100%&quot;,&quot;responsive_max_width_unit&quot;:false,&quot;custom_width&quot;:&quot;640px&quot;,&quot;custom_width_unit&quot;:false,&quot;custom_height&quot;:&quot;380px&quot;,&quot;custom_height_unit&quot;:false,&quot;custom_height_auto&quot;:false,&quot;location&quot;:&quot;center top&quot;,&quot;position_from_trigger&quot;:false,&quot;position_top&quot;:&quot;100&quot;,&quot;position_left&quot;:&quot;0&quot;,&quot;position_bottom&quot;:&quot;0&quot;,&quot;position_right&quot;:&quot;0&quot;,&quot;position_fixed&quot;:false,&quot;animation_type&quot;:&quot;fade&quot;,&quot;animation_speed&quot;:&quot;350&quot;,&quot;animation_origin&quot;:&quot;center top&quot;,&quot;overlay_zindex&quot;:false,&quot;zindex&quot;:&quot;1999999999&quot;},&quot;close&quot;:{&quot;text&quot;:&quot;&quot;,&quot;button_delay&quot;:&quot;0&quot;,&quot;overlay_click&quot;:false,&quot;esc_press&quot;:false,&quot;f4_press&quot;:false},&quot;click_open&quot;:[]}}" role="dialog" aria-hidden="true">

	<div id="popmake-2358" class="pum-container popmake theme-1538 pum-responsive pum-responsive-medium responsive size-medium">

				

				

		

				<div class="pum-content popmake-content" tabindex="0">
			<div id="page" class="hfeed site">
<div id="main" class="site-main">
<div class="fw-page-builder-content">
<section class="fw-main-row-custom auto fw-section-position-back tf-sh-ad8832c2d698d969c636f897079bd7cc">
<div class="fw-container">
<div class="fw-row">
<div id="column-5f946a9167305" class="fw-col-sm-12 tf-sh-c2095f383fe977187eb4ac3a23d13077">
<div class="fw-wrap-col-inner clearfix ">
<div class="fw-col-inner">
<div class="fw-text-box tf-sh-80452394785117c2c3036894f2cc2ed0 ">
<div class="fw-text-inner">
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<h4 class="fw-special-title">MARÍA RODRÍGUEZ-MARTÍNEZ</h4>
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<p>IBM Research Europe<br>
Technical Leader of Systems Biology at IBM Research Europe (Switzerland), associated member of the Department of Biology at ETH and editor for ImmunoInformatics and Frontiers in Systems Biology.</p>
<p class="p2"><b>Tuesday, 20 Sept.<span class="Apple-converted-space"> | </span></b><b>09:00 &#8211; 10:00h<span class="Apple-converted-space"> </span></b></p>
</div>
<h5 class="gmail_default">Interpretable AI for cancer personalized medicine</h5>
<p>In recent years, deep learning models have resulted in outstanding breakthrough performances. However, many models behave as black boxes that can hide data biases, incorrect hypotheses or even software errors. In this talk, I will illustrate how interpretable deep learning models can achieve both high prediction accuracy and transparency.</p>
<p>First, I will introduce multi-modal deep learning models that predict drug response while highlighting the genetic and chemical patterns that were more informative to make a prediction. I will also discuss how reinforcement learning approaches can facilitate the early phases of drug discovery and support the personalised design of new candidate compounds.</p>
<p>Focusing next on T cell-based immunotherapies, I will present a model to predict the binding of T cell receptors and epitopes. This model can be coupled with an easy-to-use interpretable pipeline to extract the binding rules governing the T cell binding. These approaches are a first step towards the design and engineering of receptors of improved affinity.</p>
<p>Finally, I will discuss how the integration of AI and mechanistic models is necessary to tackle many current computational challenges and enable the personalized design of new therapeutic interventions.</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
		</div>


				

				            <button type="button" class="pum-close popmake-close" aria-label="Close">
			×            </button>
		
	</div>

</div>
<div id="pum-2503" class="pum pum-overlay pum-theme-1538 pum-theme-lightbox popmake-overlay click_open" data-popmake="{&quot;id&quot;:2503,&quot;slug&quot;:&quot;mar-alba&quot;,&quot;theme_id&quot;:1538,&quot;cookies&quot;:[],&quot;triggers&quot;:[{&quot;type&quot;:&quot;click_open&quot;,&quot;settings&quot;:{&quot;extra_selectors&quot;:&quot;&quot;,&quot;cookie_name&quot;:null}}],&quot;mobile_disabled&quot;:null,&quot;tablet_disabled&quot;:null,&quot;meta&quot;:{&quot;display&quot;:{&quot;stackable&quot;:false,&quot;overlay_disabled&quot;:false,&quot;scrollable_content&quot;:false,&quot;disable_reposition&quot;:false,&quot;size&quot;:&quot;medium&quot;,&quot;responsive_min_width&quot;:&quot;0%&quot;,&quot;responsive_min_width_unit&quot;:false,&quot;responsive_max_width&quot;:&quot;100%&quot;,&quot;responsive_max_width_unit&quot;:false,&quot;custom_width&quot;:&quot;640px&quot;,&quot;custom_width_unit&quot;:false,&quot;custom_height&quot;:&quot;380px&quot;,&quot;custom_height_unit&quot;:false,&quot;custom_height_auto&quot;:false,&quot;location&quot;:&quot;center top&quot;,&quot;position_from_trigger&quot;:false,&quot;position_top&quot;:&quot;100&quot;,&quot;position_left&quot;:&quot;0&quot;,&quot;position_bottom&quot;:&quot;0&quot;,&quot;position_right&quot;:&quot;0&quot;,&quot;position_fixed&quot;:false,&quot;animation_type&quot;:&quot;fade&quot;,&quot;animation_speed&quot;:&quot;350&quot;,&quot;animation_origin&quot;:&quot;center top&quot;,&quot;overlay_zindex&quot;:false,&quot;zindex&quot;:&quot;1999999999&quot;},&quot;close&quot;:{&quot;text&quot;:&quot;&quot;,&quot;button_delay&quot;:&quot;0&quot;,&quot;overlay_click&quot;:false,&quot;esc_press&quot;:false,&quot;f4_press&quot;:false},&quot;click_open&quot;:[]}}" role="dialog" aria-hidden="true">

	<div id="popmake-2503" class="pum-container popmake theme-1538 pum-responsive pum-responsive-medium responsive size-medium">

				

				

		

				<div class="pum-content popmake-content" tabindex="0">
			<div id="page" class="hfeed site">
<div id="main" class="site-main">
<div class="fw-page-builder-content">
<section class="fw-main-row-custom auto fw-section-position-back tf-sh-ad8832c2d698d969c636f897079bd7cc">
<div class="fw-container">
<div class="fw-row">
<div id="column-5f946a9167305" class="fw-col-sm-12 tf-sh-c2095f383fe977187eb4ac3a23d13077">
<div class="fw-wrap-col-inner clearfix ">
<div class="fw-col-inner">
<div class="fw-text-box tf-sh-80452394785117c2c3036894f2cc2ed0 ">
<div class="fw-text-inner">
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<h4 class="fw-special-title">MAR ALBÀ</h4>
<p>ICREA-IMIM, Barcelona, ICREA Research Professor</p>
<p class="p2"><b>Wednesday, 21 Sept<span class="Apple-converted-space"> | </span></b><b>11:30 &#8211; 12:30h<span class="Apple-converted-space"> </span></b></p>
<h5 class="gmail_default">The emerging small proteome</h5>
<p>Research in recent years has uncovered the existence of a large non-canonical proteome that mostly consists of proteins smaller than 100 amino acids. In humans alone, thousands of non-canonical proteins have been discovered using ribosome profiling and proteomics techniques. These proteins are translated from alternative open reading frames or from regions previously believed to be non-coding, such as long non-coding RNAs (lncRNAs) and untranslated regions (UTRs). A subset of the small proteins show strong phylogenetic conservation and are involved in fundamental cellular functions. Another large fraction is species- or lineage-specific, and represents a much more rapidly evolving part of the proteome than remains poorly characterized. Using studies in yeast, we provide evidence that these proteins are important for short evolutionary time scale adaptations.</p>
<p>We also show that they can provide plenty of raw material for de novo gene birth, a process by which proteins with completely new sequences emerge from previously non-coding parts of the genome.</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
		</div>


				

				            <button type="button" class="pum-close popmake-close" aria-label="Close">
			×            </button>
		
	</div>

</div>
<div id="pum-2822" class="pum pum-overlay pum-theme-1538 pum-theme-lightbox popmake-overlay click_open" data-popmake="{&quot;id&quot;:2822,&quot;slug&quot;:&quot;raul-rabadan&quot;,&quot;theme_id&quot;:1538,&quot;cookies&quot;:[],&quot;triggers&quot;:[{&quot;type&quot;:&quot;click_open&quot;,&quot;settings&quot;:{&quot;extra_selectors&quot;:&quot;&quot;,&quot;cookie_name&quot;:null}}],&quot;mobile_disabled&quot;:null,&quot;tablet_disabled&quot;:null,&quot;meta&quot;:{&quot;display&quot;:{&quot;stackable&quot;:false,&quot;overlay_disabled&quot;:false,&quot;scrollable_content&quot;:false,&quot;disable_reposition&quot;:false,&quot;size&quot;:&quot;medium&quot;,&quot;responsive_min_width&quot;:&quot;0%&quot;,&quot;responsive_min_width_unit&quot;:false,&quot;responsive_max_width&quot;:&quot;100%&quot;,&quot;responsive_max_width_unit&quot;:false,&quot;custom_width&quot;:&quot;640px&quot;,&quot;custom_width_unit&quot;:false,&quot;custom_height&quot;:&quot;380px&quot;,&quot;custom_height_unit&quot;:false,&quot;custom_height_auto&quot;:false,&quot;location&quot;:&quot;center top&quot;,&quot;position_from_trigger&quot;:false,&quot;position_top&quot;:&quot;100&quot;,&quot;position_left&quot;:&quot;0&quot;,&quot;position_bottom&quot;:&quot;0&quot;,&quot;position_right&quot;:&quot;0&quot;,&quot;position_fixed&quot;:false,&quot;animation_type&quot;:&quot;fade&quot;,&quot;animation_speed&quot;:&quot;350&quot;,&quot;animation_origin&quot;:&quot;center top&quot;,&quot;overlay_zindex&quot;:false,&quot;zindex&quot;:&quot;1999999999&quot;},&quot;close&quot;:{&quot;text&quot;:&quot;&quot;,&quot;button_delay&quot;:&quot;0&quot;,&quot;overlay_click&quot;:false,&quot;esc_press&quot;:false,&quot;f4_press&quot;:false},&quot;click_open&quot;:[]}}" role="dialog" aria-hidden="true">

	<div id="popmake-2822" class="pum-container popmake theme-1538 pum-responsive pum-responsive-medium responsive size-medium">

				

				

		

				<div class="pum-content popmake-content" tabindex="0">
			<div id="page" class="hfeed site">
<div id="main" class="site-main">
<div class="fw-page-builder-content">
<section class="fw-main-row-custom auto fw-section-position-back tf-sh-ad8832c2d698d969c636f897079bd7cc">
<div class="fw-container">
<div class="fw-row">
<div id="column-5f946a9167305" class="fw-col-sm-12 tf-sh-c2095f383fe977187eb4ac3a23d13077">
<div class="fw-wrap-col-inner clearfix ">
<div class="fw-col-inner">
<div class="fw-text-box tf-sh-80452394785117c2c3036894f2cc2ed0 ">
<div class="fw-text-inner">
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<h4 class="fw-special-title">RAÚL RABADÁN</h4>
<p>Raul Rabadan is the Gerald and Janet Carrus Professor in the Departments of Systems Biology, Biomedical Informatics and Surgery at Columbia University. He is the director of the Program for Mathematical Genomics at Columbia University and he was the Director of the NCI Center for Topology of Cancer Evolution and Heterogeneity at Columbia University (2015-2021). From 2001 to 2003, Dr. Rabadan was a fellow at the Theoretical Physics Division at CERN, the European Organization for Nuclear Research, in Geneva, Switzerland. In 2003 he joined the Physics Group of the School of Natural Sciences at the Institute for Advanced Study. Previously, Dr. Rabadan was the Martin A. and Helen Chooljian Member at The Simons Center for Systems Biology at the Institute for Advanced Study in Princeton, New Jersey. He has been named one of Popular Science&#8217;s Brilliant 10 (2010), a Stewart Trust Fellow (2013), and he received the Harold and Golden Lamport Award at Columbia University (2014) and the Diz Pintado award (2018). Dr. Rabadan’s current interest focuses on uncovering patterns of evolution in biological systems through the lens of genomics. His recent interests include the development of mathematical approaches to uncover the evolution of cancer and infectious diseases, including topological data analysis and Random Matrix Theory, among others.</p>
<p class="p2"><b>Monday, 19 Sept<span class="Apple-converted-space"> | </span></b><b>09:00 &#8211; 10:00h<span class="Apple-converted-space"> </span></b></p>
<h5><strong>Some mysteries about microbes and cancer</strong></h5>
<p>At least 20% of all tumors in the world are linked to pathogens. Viral related tumors present very unique characteristics including unusual age, sex and geographical distributions. For instance some of these tumors like Burkitt Lymphomas occur commonly in young kids in Africa but not in other parts of the world.  Others in some populations in South America and Japan. As more genomic studies illuminate the distinct mutational spectrum of these tumors some common patterns are emerging.</p>
<p>&nbsp;</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
		</div>


				

				            <button type="button" class="pum-close popmake-close" aria-label="Close">
			×            </button>
		
	</div>

</div>
<div id="pum-2276" class="pum pum-overlay pum-theme-1538 pum-theme-lightbox popmake-overlay click_open" data-popmake="{&quot;id&quot;:2276,&quot;slug&quot;:&quot;ana-teresa-freitas&quot;,&quot;theme_id&quot;:1538,&quot;cookies&quot;:[],&quot;triggers&quot;:[{&quot;type&quot;:&quot;click_open&quot;,&quot;settings&quot;:{&quot;extra_selectors&quot;:&quot;&quot;,&quot;cookie_name&quot;:null}}],&quot;mobile_disabled&quot;:null,&quot;tablet_disabled&quot;:null,&quot;meta&quot;:{&quot;display&quot;:{&quot;stackable&quot;:false,&quot;overlay_disabled&quot;:false,&quot;scrollable_content&quot;:false,&quot;disable_reposition&quot;:false,&quot;size&quot;:&quot;medium&quot;,&quot;responsive_min_width&quot;:&quot;0%&quot;,&quot;responsive_min_width_unit&quot;:false,&quot;responsive_max_width&quot;:&quot;100%&quot;,&quot;responsive_max_width_unit&quot;:false,&quot;custom_width&quot;:&quot;640px&quot;,&quot;custom_width_unit&quot;:false,&quot;custom_height&quot;:&quot;380px&quot;,&quot;custom_height_unit&quot;:false,&quot;custom_height_auto&quot;:false,&quot;location&quot;:&quot;center top&quot;,&quot;position_from_trigger&quot;:false,&quot;position_top&quot;:&quot;100&quot;,&quot;position_left&quot;:&quot;0&quot;,&quot;position_bottom&quot;:&quot;0&quot;,&quot;position_right&quot;:&quot;0&quot;,&quot;position_fixed&quot;:false,&quot;animation_type&quot;:&quot;fade&quot;,&quot;animation_speed&quot;:&quot;350&quot;,&quot;animation_origin&quot;:&quot;center top&quot;,&quot;overlay_zindex&quot;:false,&quot;zindex&quot;:&quot;1999999999&quot;},&quot;close&quot;:{&quot;text&quot;:&quot;&quot;,&quot;button_delay&quot;:&quot;0&quot;,&quot;overlay_click&quot;:false,&quot;esc_press&quot;:false,&quot;f4_press&quot;:false},&quot;click_open&quot;:[]}}" role="dialog" aria-hidden="true">

	<div id="popmake-2276" class="pum-container popmake theme-1538 pum-responsive pum-responsive-medium responsive size-medium">

				

				

		

				<div class="pum-content popmake-content" tabindex="0">
			<div id="page" class="hfeed site">
<div id="main" class="site-main">
<div class="fw-page-builder-content">
<section class="fw-main-row-custom auto fw-section-position-back tf-sh-ad8832c2d698d969c636f897079bd7cc">
<div class="fw-container">
<div class="fw-row">
<div id="column-5f946a9167305" class="fw-col-sm-12 tf-sh-c2095f383fe977187eb4ac3a23d13077">
<div class="fw-wrap-col-inner clearfix ">
<div class="fw-col-inner">
<div class="fw-text-box tf-sh-80452394785117c2c3036894f2cc2ed0 ">
<div class="fw-text-inner">
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<h4 class="fw-special-title">ANA TERESA FREITAS</h4>
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<p>CSE Department, INESC-ID/IST Technical University of Lisbon (Portugal)<br>
Senior research member (PI) at the Information and Decision Support Systems Group/INESC-ID</p>
<p class="p2"><b>Monday, 19 Sept.<span class="Apple-converted-space"> | </span></b><b>16:30 &#8211; 17:30h<span class="Apple-converted-space"> </span></b></p>
</div>
<h5 class="gmail_default">Personalized medicine in the era of artificial intelligence</h5>
<p>Like in many other industries, the healthcare sector routinely generates vast amounts of data from many different sources ranging from biochemical exams, electronic medical records, vital signs, patient-reported outcomes, health surveys, clinical trials, insurance claims, administrative data, and more recently omics. These days, large volumes of data associated with the new technologies of artificial intelligence are promising to create the foundations for a new paradigm of medicine focused on the individuality of each person.</p>
<p>In this talk I will discuss different approaches that are being tested in EU hospitals with the goal to transform healthcare from reactive disease care to care that is patient or person-centered and focused on disease prevention. A special focus will be given to the use of polygenic risk scoring models in the implementation of genetic panels that can support health professionals in disease prevention. Additionally, I will discuss how the field of pharmacogenomics is being able to make its way from research to clinical practice, which could become, in the short term, the first pillar of the democratization of preventive and personalized medicine.</p>
<p>Since artificial intelligence is becoming a disruptive technology in the healthcare sector, it is also crucial to address the ethical and legal challenges imposed by this new technological advance.</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
		</div>


				

				            <button type="button" class="pum-close popmake-close" aria-label="Close">
			×            </button>
		
	</div>

</div>
<div id="pum-2698" class="pum pum-overlay pum-theme-1538 pum-theme-lightbox popmake-overlay click_open" data-popmake="{&quot;id&quot;:2698,&quot;slug&quot;:&quot;cesar-hidalgo&quot;,&quot;theme_id&quot;:1538,&quot;cookies&quot;:[],&quot;triggers&quot;:[{&quot;type&quot;:&quot;click_open&quot;,&quot;settings&quot;:{&quot;extra_selectors&quot;:&quot;&quot;,&quot;cookie_name&quot;:null}}],&quot;mobile_disabled&quot;:null,&quot;tablet_disabled&quot;:null,&quot;meta&quot;:{&quot;display&quot;:{&quot;stackable&quot;:false,&quot;overlay_disabled&quot;:false,&quot;scrollable_content&quot;:false,&quot;disable_reposition&quot;:false,&quot;size&quot;:&quot;medium&quot;,&quot;responsive_min_width&quot;:&quot;0%&quot;,&quot;responsive_min_width_unit&quot;:false,&quot;responsive_max_width&quot;:&quot;100%&quot;,&quot;responsive_max_width_unit&quot;:false,&quot;custom_width&quot;:&quot;640px&quot;,&quot;custom_width_unit&quot;:false,&quot;custom_height&quot;:&quot;380px&quot;,&quot;custom_height_unit&quot;:false,&quot;custom_height_auto&quot;:false,&quot;location&quot;:&quot;center top&quot;,&quot;position_from_trigger&quot;:false,&quot;position_top&quot;:&quot;100&quot;,&quot;position_left&quot;:&quot;0&quot;,&quot;position_bottom&quot;:&quot;0&quot;,&quot;position_right&quot;:&quot;0&quot;,&quot;position_fixed&quot;:false,&quot;animation_type&quot;:&quot;fade&quot;,&quot;animation_speed&quot;:&quot;350&quot;,&quot;animation_origin&quot;:&quot;center top&quot;,&quot;overlay_zindex&quot;:false,&quot;zindex&quot;:&quot;1999999999&quot;},&quot;close&quot;:{&quot;text&quot;:&quot;&quot;,&quot;button_delay&quot;:&quot;0&quot;,&quot;overlay_click&quot;:false,&quot;esc_press&quot;:false,&quot;f4_press&quot;:false},&quot;click_open&quot;:[]}}" role="dialog" aria-hidden="true">

	<div id="popmake-2698" class="pum-container popmake theme-1538 pum-responsive pum-responsive-medium responsive size-medium">

				

				

		

				<div class="pum-content popmake-content" tabindex="0">
			<div id="page" class="hfeed site">
<div id="main" class="site-main">
<div class="fw-page-builder-content">
<section class="fw-main-row-custom auto fw-section-position-back tf-sh-ad8832c2d698d969c636f897079bd7cc">
<div class="fw-container">
<div class="fw-row">
<div id="column-5f946a9167305" class="fw-col-sm-12 tf-sh-c2095f383fe977187eb4ac3a23d13077">
<div class="fw-wrap-col-inner clearfix ">
<div class="fw-col-inner">
<div class="fw-text-box tf-sh-80452394785117c2c3036894f2cc2ed0 ">
<div class="fw-text-inner">
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<h4 class="fw-special-title">CÉSAR HIDALGO</h4>
<p>Cesar Hidalgo directs the Center for Collective Learning at ANITI, University of Toulouse. He directed the Collective Learning group at MIT, holds a PhD in Physics from the University of Notre Dame, and is the author of dozens of papers &amp; three books.</p>
<p class="p1"><strong>Sunday, 18 Sept. | 18:30 &#8211; 19:30h</strong></p>
<h5 class="gmail_default">How humans judge machines</h5>
<p>How would you feel about losing your job to a machine? How about a tsunami alert system that fails? Would you react differently to acts of discrimination performed by a machine or a human? How about public surveillance? <i>How Humans Judge Machines</i> compares people&#8217;s reactions to actions performed by humans and machines. Using data collected in dozens of experiments, this book reveals the biases that permeate human-machine interactions. Are there conditions in which we judge machines unfairly? Is our judgment of machines affected by the moral dimensions of a scenario? Is our judgment of machines correlated with demographic factors, such as education or gender?<br>
Hidalgo and colleagues use hard science to take on these pressing technological questions. Using randomized experiments, they create revealing counterfactuals and build statistical models to explain how people judge A.I. and whether we do it fairly or not. Through original research, they bring us one step closer to understanding the ethical consequences of artificial intelligence. How Humans Judge Machines can be read for free at <a href="https://www.judgingmachines.com/" target="_blank" rel="noopener" data-saferedirecturl="https://www.google.com/url?q=https://www.judgingmachines.com/&amp;source=gmail&amp;ust=1661326127293000&amp;usg=AOvVaw2mT7fcz8wqXwRiDWqoIuDf">https://www.<wbr>judgingmachines.com/</a> (in print with MIT Press).</p>
<p>&nbsp;</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
		</div>


				

				            <button type="button" class="pum-close popmake-close" aria-label="Close">
			×            </button>
		
	</div>

</div>
<div id="pum-2871" class="pum pum-overlay pum-theme-1538 pum-theme-lightbox popmake-overlay click_open" data-popmake="{&quot;id&quot;:2871,&quot;slug&quot;:&quot;graciela-gonzalez-hernandez&quot;,&quot;theme_id&quot;:1538,&quot;cookies&quot;:[],&quot;triggers&quot;:[{&quot;type&quot;:&quot;click_open&quot;,&quot;settings&quot;:{&quot;extra_selectors&quot;:&quot;&quot;,&quot;cookie_name&quot;:null}}],&quot;mobile_disabled&quot;:null,&quot;tablet_disabled&quot;:null,&quot;meta&quot;:{&quot;display&quot;:{&quot;stackable&quot;:false,&quot;overlay_disabled&quot;:false,&quot;scrollable_content&quot;:false,&quot;disable_reposition&quot;:false,&quot;size&quot;:&quot;medium&quot;,&quot;responsive_min_width&quot;:&quot;0%&quot;,&quot;responsive_min_width_unit&quot;:false,&quot;responsive_max_width&quot;:&quot;100%&quot;,&quot;responsive_max_width_unit&quot;:false,&quot;custom_width&quot;:&quot;640px&quot;,&quot;custom_width_unit&quot;:false,&quot;custom_height&quot;:&quot;380px&quot;,&quot;custom_height_unit&quot;:false,&quot;custom_height_auto&quot;:false,&quot;location&quot;:&quot;center top&quot;,&quot;position_from_trigger&quot;:false,&quot;position_top&quot;:&quot;100&quot;,&quot;position_left&quot;:&quot;0&quot;,&quot;position_bottom&quot;:&quot;0&quot;,&quot;position_right&quot;:&quot;0&quot;,&quot;position_fixed&quot;:false,&quot;animation_type&quot;:&quot;fade&quot;,&quot;animation_speed&quot;:&quot;350&quot;,&quot;animation_origin&quot;:&quot;center top&quot;,&quot;overlay_zindex&quot;:false,&quot;zindex&quot;:&quot;1999999999&quot;},&quot;close&quot;:{&quot;text&quot;:&quot;&quot;,&quot;button_delay&quot;:&quot;0&quot;,&quot;overlay_click&quot;:false,&quot;esc_press&quot;:false,&quot;f4_press&quot;:false},&quot;click_open&quot;:[]}}" role="dialog" aria-hidden="true">

	<div id="popmake-2871" class="pum-container popmake theme-1538 pum-responsive pum-responsive-medium responsive size-medium">

				

				

		

				<div class="pum-content popmake-content" tabindex="0">
			<div id="page" class="hfeed site">
<div id="main" class="site-main">
<div class="fw-page-builder-content">
<section class="fw-main-row-custom auto fw-section-position-back tf-sh-ad8832c2d698d969c636f897079bd7cc">
<div class="fw-container">
<div class="fw-row">
<div id="column-5f946a9167305" class="fw-col-sm-12 tf-sh-c2095f383fe977187eb4ac3a23d13077">
<div class="fw-wrap-col-inner clearfix ">
<div class="fw-col-inner">
<div class="fw-text-box tf-sh-80452394785117c2c3036894f2cc2ed0 ">
<div class="fw-text-inner">
<div class="fw-heading fw-heading-left tf-sh-dd55a3a6fde3dc8b3e0cd0991715ba11">
<h4 class="fw-special-title">GRACIELA GONZÁLEZ-HERNÁNDEZ</h4>
<p>Dr Graciela Gonzalez-Hernandez is Vice Chair for Research and Education in the new Department of Computational Biomedicine at Cedars-Sinai Medical Center. Cedars-Sinai is a research hospital in Beverly Hills, California ranked #1 in California and #2 in the nation by US News and World Reports in 2022. Prior to joining Cedars-Sinai in May 2022, Dr Gonzalez-Hernandez was an Associate Professor of Informatics in the Department of Biostatistics, Epidemiology and Informatics (DBEI) of the Perelman School of Medicine, University of Pennsylvania. She transferred her Health Language Processing (HLP) Lab to Cedars-Sinai, which focuses on natural language processing (NLP) and machine learning for knowledge discovery, extracting unstructured information from clinical records, journal articles, and social media postings to elucidate data patterns, trends, and relationships that can aid the discovery process in areas such as pharmacoepidemiology, clinical research, or public health monitoring and surveillance.</p>
<p class="p2"><b>Tuesday, 20 Sept<span class="Apple-converted-space"> | </span></b><b>16:30 &#8211; 17:30h<span class="Apple-converted-space"> </span></b></p>
<h5><strong>Mining for Digital Epidemiology: Overcoming the Challenges of Real World Data</strong></h5>
<p>The use of Real World Data (RWD) such as social media or electronic health record data for large-scale epidemiological studies presents many challenges. From identifying the right cohort and reducing bias to finding key patterns or unique data points, RWD can be very valuable but is sometimes misused and frowned upon for &#8220;serious&#8221; studies. Systematic methods and validated approaches are necessary to turn &#8216;real world data&#8217; into ‘real world evidence.</p>
<p>&nbsp;</p>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
</div>
</div>
</div>
		</div>


				

				            <button type="button" class="pum-close popmake-close" aria-label="Close">
			×            </button>
		
	</div>

</div>
<script id='getwid-blocks-frontend-js-js-extra'>
var Getwid = {"settings":[],"ajax_url":"https:\/\/eccb2022.org\/wp-admin\/admin-ajax.php","isRTL":"","nonces":{"recaptcha_v2_contact_form":"4d0e33e8e9"}};
</script>
<script src='../wp-content/plugins/getwid/assets/js/frontend.blocks.js?ver=1.8.2' id='getwid-blocks-frontend-js-js'></script>
<script src='../wp-includes/js/jquery/ui/core.min.js?ver=1.13.2' id='jquery-ui-core-js'></script>
<script id='popup-maker-site-js-extra'>
var pum_vars = {"version":"1.16.7","pm_dir_url":"https:\/\/eccb2022.org\/wp-content\/plugins\/popup-maker\/","ajaxurl":"https:\/\/eccb2022.org\/wp-admin\/admin-ajax.php","restapi":"https:\/\/eccb2022.org\/wp-json\/pum\/v1","rest_nonce":null,"default_theme":"1537","debug_mode":"","disable_tracking":"","home_url":"\/","message_position":"top","core_sub_forms_enabled":"1","popups":[],"analytics_route":"analytics","analytics_api":"https:\/\/eccb2022.org\/wp-json\/pum\/v1"};
var pum_sub_vars = {"ajaxurl":"https:\/\/eccb2022.org\/wp-admin\/admin-ajax.php","message_position":"top"};
var pum_popups = {"pum-2358":{"triggers":[],"cookies":[],"disable_on_mobile":false,"disable_on_tablet":false,"atc_promotion":null,"explain":null,"type_section":null,"theme_id":"1538","size":"medium","responsive_min_width":"0%","responsive_max_width":"100%","custom_width":"640px","custom_height_auto":false,"custom_height":"380px","scrollable_content":false,"animation_type":"fade","animation_speed":"350","animation_origin":"center top","open_sound":"none","custom_sound":"","location":"center top","position_top":"100","position_bottom":"0","position_left":"0","position_right":"0","position_from_trigger":false,"position_fixed":false,"overlay_disabled":false,"stackable":false,"disable_reposition":false,"zindex":"1999999999","close_button_delay":"0","fi_promotion":null,"close_on_form_submission":false,"close_on_form_submission_delay":"0","close_on_overlay_click":false,"close_on_esc_press":false,"close_on_f4_press":false,"disable_form_reopen":false,"disable_accessibility":false,"theme_slug":"lightbox","id":2358,"slug":"maria-rodriguez-martinez"},"pum-2503":{"triggers":[],"cookies":[],"disable_on_mobile":false,"disable_on_tablet":false,"atc_promotion":null,"explain":null,"type_section":null,"theme_id":"1538","size":"medium","responsive_min_width":"0%","responsive_max_width":"100%","custom_width":"640px","custom_height_auto":false,"custom_height":"380px","scrollable_content":false,"animation_type":"fade","animation_speed":"350","animation_origin":"center top","open_sound":"none","custom_sound":"","location":"center top","position_top":"100","position_bottom":"0","position_left":"0","position_right":"0","position_from_trigger":false,"position_fixed":false,"overlay_disabled":false,"stackable":false,"disable_reposition":false,"zindex":"1999999999","close_button_delay":"0","fi_promotion":null,"close_on_form_submission":false,"close_on_form_submission_delay":"0","close_on_overlay_click":false,"close_on_esc_press":false,"close_on_f4_press":false,"disable_form_reopen":false,"disable_accessibility":false,"theme_slug":"lightbox","id":2503,"slug":"mar-alba"},"pum-2822":{"triggers":[],"cookies":[],"disable_on_mobile":false,"disable_on_tablet":false,"atc_promotion":null,"explain":null,"type_section":null,"theme_id":"1538","size":"medium","responsive_min_width":"0%","responsive_max_width":"100%","custom_width":"640px","custom_height_auto":false,"custom_height":"380px","scrollable_content":false,"animation_type":"fade","animation_speed":"350","animation_origin":"center top","open_sound":"none","custom_sound":"","location":"center top","position_top":"100","position_bottom":"0","position_left":"0","position_right":"0","position_from_trigger":false,"position_fixed":false,"overlay_disabled":false,"stackable":false,"disable_reposition":false,"zindex":"1999999999","close_button_delay":"0","fi_promotion":null,"close_on_form_submission":false,"close_on_form_submission_delay":"0","close_on_overlay_click":false,"close_on_esc_press":false,"close_on_f4_press":false,"disable_form_reopen":false,"disable_accessibility":false,"theme_slug":"lightbox","id":2822,"slug":"raul-rabadan"},"pum-2276":{"triggers":[],"cookies":[],"disable_on_mobile":false,"disable_on_tablet":false,"atc_promotion":null,"explain":null,"type_section":null,"theme_id":"1538","size":"medium","responsive_min_width":"0%","responsive_max_width":"100%","custom_width":"640px","custom_height_auto":false,"custom_height":"380px","scrollable_content":false,"animation_type":"fade","animation_speed":"350","animation_origin":"center top","open_sound":"none","custom_sound":"","location":"center top","position_top":"100","position_bottom":"0","position_left":"0","position_right":"0","position_from_trigger":false,"position_fixed":false,"overlay_disabled":false,"stackable":false,"disable_reposition":false,"zindex":"1999999999","close_button_delay":"0","fi_promotion":null,"close_on_form_submission":false,"close_on_form_submission_delay":"0","close_on_overlay_click":false,"close_on_esc_press":false,"close_on_f4_press":false,"disable_form_reopen":false,"disable_accessibility":false,"theme_slug":"lightbox","id":2276,"slug":"ana-teresa-freitas"},"pum-2698":{"triggers":[],"cookies":[],"disable_on_mobile":false,"disable_on_tablet":false,"atc_promotion":null,"explain":null,"type_section":null,"theme_id":"1538","size":"medium","responsive_min_width":"0%","responsive_max_width":"100%","custom_width":"640px","custom_height_auto":false,"custom_height":"380px","scrollable_content":false,"animation_type":"fade","animation_speed":"350","animation_origin":"center top","open_sound":"none","custom_sound":"","location":"center top","position_top":"100","position_bottom":"0","position_left":"0","position_right":"0","position_from_trigger":false,"position_fixed":false,"overlay_disabled":false,"stackable":false,"disable_reposition":false,"zindex":"1999999999","close_button_delay":"0","fi_promotion":null,"close_on_form_submission":false,"close_on_form_submission_delay":"0","close_on_overlay_click":false,"close_on_esc_press":false,"close_on_f4_press":false,"disable_form_reopen":false,"disable_accessibility":false,"theme_slug":"lightbox","id":2698,"slug":"cesar-hidalgo"},"pum-2871":{"triggers":[],"cookies":[],"disable_on_mobile":false,"disable_on_tablet":false,"atc_promotion":null,"explain":null,"type_section":null,"theme_id":"1538","size":"medium","responsive_min_width":"0%","responsive_max_width":"100%","custom_width":"640px","custom_height_auto":false,"custom_height":"380px","scrollable_content":false,"animation_type":"fade","animation_speed":"350","animation_origin":"center top","open_sound":"none","custom_sound":"","location":"center top","position_top":"100","position_bottom":"0","position_left":"0","position_right":"0","position_from_trigger":false,"position_fixed":false,"overlay_disabled":false,"stackable":false,"disable_reposition":false,"zindex":"1999999999","close_button_delay":"0","fi_promotion":null,"close_on_form_submission":false,"close_on_form_submission_delay":"0","close_on_overlay_click":false,"close_on_esc_press":false,"close_on_f4_press":false,"disable_form_reopen":false,"disable_accessibility":false,"theme_slug":"lightbox","id":2871,"slug":"graciela-gonzalez-hernandez"}};
</script>
<script src='../wp-content/uploads/pum/pum-site-scripts.js?defer&#038;generated=1663251559&#038;ver=1.16.7' id='popup-maker-site-js'></script>
<script id='ct-scripts-js-extra'>
var ct_localizations = {"ajax_url":"https:\/\/eccb2022.org\/wp-admin\/admin-ajax.php","nonce":"f88d404fb8","public_url":"https:\/\/eccb2022.org\/wp-content\/themes\/blocksy\/static\/bundle\/","rest_url":"https:\/\/eccb2022.org\/wp-json\/","search_url":"https:\/\/eccb2022.org\/search\/QUERY_STRING\/","show_more_text":"Show more","more_text":"More","expand_submenu":"Expand dropdown menu","collapse_submenu":"Collapse dropdown menu","dynamic_js_chunks":[{"id":"blocksy_pro_micro_popups","selector":".ct-popup","url":"https:\/\/eccb2022.org\/wp-content\/plugins\/blocksy-companion-pro\/framework\/premium\/static\/bundle\/micro-popups.js"},{"id":"blocksy_dark_mode","selector":"[data-id=\"dark-mode-switcher\"]","url":"https:\/\/eccb2022.org\/wp-content\/plugins\/blocksy-companion-pro\/static\/bundle\/dark-mode.js","trigger":"click"},{"id":"blocksy_sticky_header","selector":"header [data-sticky]","url":"https:\/\/eccb2022.org\/wp-content\/plugins\/blocksy-companion-pro\/static\/bundle\/sticky.js"},{"id":"blocksy_lazy_load","selector":".ct-lazy[class*=\"ct-image\"]","url":"https:\/\/eccb2022.org\/wp-content\/themes\/blocksy\/static\/bundle\/lazy-load.js"}],"dynamic_styles":{"lazy_load":"https:\/\/eccb2022.org\/wp-content\/themes\/blocksy\/static\/bundle\/non-critical-styles.min.css","search_lazy":"https:\/\/eccb2022.org\/wp-content\/themes\/blocksy\/static\/bundle\/non-critical-search-styles.min.css"},"dynamic_styles_selectors":[{"selector":"#account-modal","url":"https:\/\/eccb2022.org\/wp-content\/plugins\/blocksy-companion-pro\/static\/bundle\/account-lazy.min.css"}]};
var WP_DEBUG = {"debug":"1"};
</script>
<script src='../wp-content/themes/blocksy/static/bundle/main.js?ver=1.8.14' id='ct-scripts-js'></script>

</body>
</html>
