
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/highlights.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:48:45 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/highlights.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:24 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Highlights</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="http://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top">




<h3>Highlights Track Presentation Schedule</h3><div class=style1><div class=subheading2>Highlights Track: <strong></strong><a name="HL01" >HL01</a> <span style="float: right;">Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Analysis and design of RNA sequencing experiments for identifying isoform regulation</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Yarden Katz , Massachusetts Institute of Technology, United States<br><br><strong>Additional authors:</strong><br>Eric Wang, Harvard/MIT, United States<br>Edoardo Airoldi, Harvard, United States<br>
<br><strong>Area Session Chair: </strong>Janet Kelso    <br>
<br><strong>Presentation Overview:</strong> <br>Through alternative splicing, most human genes express multiple isoforms that often differ in function. To infer isoform regulation from high-throughput sequencing of cDNA fragments (RNA-seq), we developed the mixture-of-isoforms (MISO) model, a statistical model that estimates expression of alternatively spliced exons and isoforms and assesses confidence in these estimates. Incorporation of mRNA fragment length distribution in paired-end RNA-seq greatly improved estimation of alternative-splicing levels. MISO also detects differentially regulated exons or isoforms. Application of MISO implicated the RNA splicing factor hnRNP H1 in the regulation of alternative cleavage and polyadenylation, a role that was supported by UV cross-linking-immunoprecipitation sequencing (CLIP-seq) analysis in human cells. Our results provide a probabilistic framework for RNA-seq analysis, give functional insights into pre-mRNA processing and yield guidelines for the optimal design of RNA-seq experiments for studies of gene and isoform expression.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL02" >HL02</a> <span style="float: right;">Sunday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>The Central Human Proteome</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Jacques Colinge , Ce-M-M- Center for Molecular Medicine of the Austrian Academy of Science, Austria<br><br><strong>Additional authors:</strong><br>Keiryn Bennett, Ce-M-M- Center for Molecular Medicine of the Austrian Academy of Science, Austria<br>Giulio Superti-Furga, Ce-M-M- Center for Molecular Medicine of the Austrian Academy of Science, Austria<br>
<br><strong>Area Session Chair: </strong>Janet Kelso
    <br>
<br><strong>Presentation Overview:</strong> <br>We have obtained a first unbiased estimation of the repertoire of proteins commonly expressed by human cells through proteomics analysis of several cell lines. The bioinformatics analysis of this central human proteome (CHP) shows it has several features that confer it an augmented flexibility to adapt to multiple environments (more exons, more interactions, etc.). We shall discuss these results, extend them, and relate then to findings by other authors to show that the CHP is not a static machine but it participates in specialized tasks and can be “recruited” by diseases on top of its fundamental housekeeping tasks. Considering the central human interactome spanned by the CHP, we shall show it has global properties that synchronize translation with other biological processes and its topology supports a global presence facilitating interactions with any specialized process.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL03" >HL03</a> <span style="float: right;">Sunday, July 17: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>From revealing new insights into Human Tissue Development to Minimum Curvineality</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Carlo Cannistraci , King Abdullah University for Science and Technology (KAUST), Saudi Arabia<br><br><strong>Additional authors:</strong><br>Timothy Ravasi, King Abdullah University for Science and Technology (KAUST), Saudi Arabia<br>
<br><strong>Area Session Chair: </strong>Janet Kelso    <br>
<br><strong>Presentation Overview:</strong> <br>We will focus on the data-mining exploration of 32 human tissues determined by 1321 transcription factor (TF) expressions. Integrating the expressions with the physical TF interactions and performing machine learning (ML) analysis, we selected 6 expression-weighted-interactions - a homeobox-sub-network – as best discriminating features that unfolded the presence of the three developmental tissue germ-layer-classes (ectoderm, mesoderm, endoderm) with 82% accuracy. Then, we will reveal how starting only from the expressions, it was possible to provide a bi-dimensional data visualization that, evaluated by clustering, offered 84% accuracy. This was achieved by means of two unsupervised and parameter-free MLs: minimum-curvilinear-embedding for nonlinear-dimension-reduction, and minimum-curvilinear-affinity-propagation for non-spherical-clustering. <br>We will conclude with our two recent results: the presence of the germ-layers-classes is conserved in the exploration of several other human and mouse gene-expression-datasets; a novel unsupervised ML using just the expressions is able to identify a discriminative homeobox-sub-network that extends the one previously proposed.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL04" >HL04</a> <span style="float: right;">Sunday, July 17: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Five-Vertebrate ChIP-seq Reveals the Evolutionary Dynamics of Transcription Factor Binding</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Benoit Ballester , European Bioinformatics Institute (EMBL-EBI), United Kingdom<br><br><strong>Additional authors:</strong><br>Petra Schwalie, EBI-EMBL, United Kingdom<br>Paul Flicek, EBI-EMBL, United Kingdom<br>Dominic Schmidt, CRI, United Kingdom<br>Michael Wilson, CRI, United Kingdom<br>Duncan Odom, CRI, United Kingdom<br>Claudia Kutter, CRI, United Kingdom<br>Stephen Watt, CRI, United Kingdom<br>Aileen Marshall, CRI,Cambridge Hepatobiliary Service, United Kingdom<br>Celia Martinez-Jimenez, Biomedical Sciences Research Center Alexander Fleming, Greece<br>Iannis Talianidis, Biomedical Sciences Research Center Alexander Fleming, Greece<br>Sarah Mackay, CRI, United Kingdom<br>
<br><strong>Area Session Chair: </strong>Janet Kelso    <br>
<br><strong>Presentation Overview:</strong> <br>Transcription factors (TFs) direct gene expression by binding to DNA regulatory regions. To explore the evolution of gene regulation, we used chromatin immunoprecipitation with high-throughput sequencing (ChIP-seq) to determine experimentally the genome-wide occupancy of two TFs, CCAAT/enhancer-binding protein alpha and hepatocyte nuclear factor 4 alpha, in the livers of five vertebrates. Although each TF displays highly conserved DNA binding preferences, most binding is species-specific, and aligned binding events present in all five species are rare. Regions near genes with expression levels that are dependent on a TF are often bound by the TF in multiple species yet show no enhanced DNA sequence constraint. Binding divergence between species can be largely explained by sequence changes to the bound motifs. Among the binding events lost in one lineage, only half are recovered by another binding event within 10 kilobases. Our results reveal large interspecies differences in transcriptional regulation and provide insight into regulatory evolution.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL05" >HL05</a> <span style="float: right;">Sunday, July 17: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Protein Complexes are Central in the Yeast Genetic Landscape</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Magali Michaut , University of Toronto, Canada<br><br><strong>Additional authors:</strong><br>Anastasia Baryshnikova, University of Toronto, Canada<br>Michael Costanzo, University of Toronto, Canada<br>Chad L Myers, University of Minnesota, United States<br>Brenda J Andrews, University of Toronto, Canada<br>Charles Boone, University of Toronto, Canada<br>Gary D Bader, University of Toronto, Canada<br>
<br><strong>Area Session Chair: </strong>Burkhard Rost
    <br>
<br><strong>Presentation Overview:</strong> <br>Genetic interactions indicate functional dependencies between genes and are a powerful tool to predict gene function. Functionally related genes tend to have similar profiles of genetic interactions. Recently, global scale mapping of quantitative (positive and negative) genetic interactions has been performed. This data clearly shows groups of genes connected by predominantly positive or negative interactions, termed monochromatic groups. These groups often correspond to functional modules, such as biological processes or protein complexes, or connections between modules, but it is not yet known how these patterns globally relate to known functional modules. Here we systematically evaluate the monochromatic nature of known biological processes and their connections in yeast Saccharomyces cerevisiae. We find that 10% of biological processes and less than 1% of inter-process connections are monochromatic. Further, we show that protein complexes are responsible for a surprisingly large fraction of these monochromatic groups.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL06" >HL06</a> <span style="float: right;">Sunday, July 17: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Comparative Genomics Reveals Birth and Death of Fragile Regions in Mammalian Evolution</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Max Alekseyev , University of South Carolina, United States<br><br><strong>Additional authors:</strong><br>Pavel Pevzner, University of California, San Diego, United States<br>
<br><strong>Area Session Chair: </strong>Burkhard Rost    <br>
<br><strong>Presentation Overview:</strong> <br>An important question in genome evolution is whether there exist fragile regions (rearrangement hotspots) where chromosomal rearrangements are happening over and over again. Existence of fragile regions in mammalian genomes is postulated by Fragile Breakage Model (FBM) proposed in 2003 as a replacement of then widely accepted Random Breakage Model (RBM). While the rebuttal of RBM initially caused a controversy, nearly all recent studies support FBM. However, the most comprehensive phylogenomic analysis of mammals (Ma et al., 2006. Genome Res. 16: 1557–1565) revealed only a few fragile regions shared between different lineages.<br><br>Our study (2010, Genome Biology 11(11): R117) provided a refinement of FBM, reconciling it with the observed features of mammalian evolution. The newly proposed Turnover Fragile Breakage Model (TFBM) postulates that fragile regions are subject to a "birth and death" process, implying that fragility has a limited evolutionary lifespan. TFBM further implies that fragile regions migrate to different locations in different mammals, explaining why only a few fragile regions are shared between different lineages.<br><br>The "birth and death" of fragile regions reinforce the recently proposed hypothesis that rearrangements are promoted by matching segmental duplications (Zhao and Bourque, 2009, Genome Res. 19: 934-942) and suggest putative locations of the currently active fragile regions in the human genome.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL07" >HL07</a> <span style="float: right;">Sunday, July 17: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Survival of the Friendly - the Importance of Protein-Protein Interactions in the Evolution of Bacterial Genomes</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Yanay Ofran , Bar Ilan University, Israel<br><br><strong>Additional authors:</strong><br>Uri Gophna, Tel Aviv University, Israel<br>
<br><strong>Area Session Chair: </strong>Burkhard Rost
    <br>
<br><strong>Presentation Overview:</strong> <br>Bacterial genomes include many genes (up to 30% of the genome, by some accounts)that were not inherited vertically from ancestors, but were acquired laterally from the environment. This phenomenon of individual genes that are incorporated into an existing genome, poses an evolutionary and biological puzzle. Biological functions are not implemented by single genes but by complex and tightly regulated networks of interactions between multiple species of molecules. How could one element that was ripped out of such complex machine become functional on its own? Moreover, how can it be incorporated into an already well-controlled module in the new host? The widely accepted answer to these questions is the decade old “complexity hypothesis”, which postulates that Lateral gene transfer (LGT) occurs mostly in genes with low complexity, that is, in genes that act alone and don't have many interactions with other elements in the genome. Hundreds of follow up studies reiterated this hypothesis. In our study, however, we introduce evidence that the opposite is true: Genes with many interactions are actually more likely to be transferred than genes that have only a few interactions in their pre-transfer host. We show that proteins with more interactions have more interaction sites on their surfaces. Their sticky, or “friendly”, surface makes them more likely to establish new functional interaction after the transfer. These results underline the importance of interactions in the design of bacterial genome throughout evolution. They may provide useful principles for the attempt to design novel modules and genomes.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL08" >HL08</a> <span style="float: right;">Sunday, July 17
: 4:00 p.m. - 4:25 p.m.
</span></div><strong><em>Universal epitope prediction for class II MHC</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Andrew Bordner , Mayo Clinic, United States<br><br><strong>Additional authors:</strong><br>Hans Mittelmann, Arizona State University, United States<br>
<br><strong>Area Session Chair: </strong>Burkhard Rost
    <br>
<br><strong>Presentation Overview:</strong> <br>Predicting peptide-class II MHC binding affinities is a challenging problem due to MHC diversity and multiple binding modes but has many biomedical applications.  We recently developed a structure-based approach using peptide docking and machine learning to predict peptide-MHC binding affinities.  Unlike popular sequence-based methods, it is applicable to any MHC type because it relies on universal physical interactions rather than limited experimental data for specific MHC types.   Using a model trained only on DRB1*0101 binding data we were able to accurately predict peptide binding affinities for all human class II MHC loci (HLA-DP, DQ, and DR) and for two murine MHC types.  This provides the first demonstration that a single prediction model can be applied to diverse MHC types with completely different binding specificities.  In addition, we will review our RTA sequence-based prediction method, which outperformed more complicated competing methods, and discuss recent work.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL09" >HL09</a> <span style="float: right;">Monday, July 18: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Topological network alignment uncovers biological function and phylogeny</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Natasa Przulj , Imperial College London, United Kingdom<br><br><strong>Additional authors:</strong><br>Tijana Milenkovic, University of Notre Dame, United States<br>
<br><strong>Area Session Chair: </strong>Erik Bongcam-Rudloff    <br>
<br><strong>Presentation Overview:</strong> <br>There are thousands of genes in the human genome.  However, genes are just a means to an end: they produce different protein types that interact in complex networked ways and make our cells work.  Thus, network connectivity provides additional biological insight, over and above sequences of individual proteins. Hence, analogous to tools for aligning genetic sequences that have revolutionized biological understanding, network alignment tools are likely to have a similar groundbreaking impact.  We introduce a topology-based network alignment algorithm that exposes surprisingly large regions of network similarity even in distant species.  Substantial improvements are achieved when additional data sources (including sequence) are integrated with topology: surprisingly, 77.7% of yeast proteins participate in a connected subnetwork that is fully contained in the human network suggesting broad similarities in cellular wiring across all life on Earth.  Furthermore, we show that topology is a successful predictor of new cancer genes in melanogenesis-related pathways.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL10" >HL10</a> <span style="float: right;">Monday, July 18: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Systematic planning of genome-scale experiments in poorly studied species.</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Casey Greene , Princeton University, United States<br>
<br><strong>Area Session Chair: </strong>Nir Ben-Tal
    <br>
<br><strong>Presentation Overview:</strong> <br>The planning of genome-scale experiments in poorly studied species is in general based on the intuition of experts or heuristic trials. We propose that computational and systematic approaches can be applied to drive the experiment planning process in poorly studied species based on available data and knowledge in closely related model organisms. To this end, we use the data-rich functional genomics compendium of the model organism to quantify the accuracy of each dataset in predicting each specific biological process and the overlap in such coverage between different datasets. Our approach uses an optimized combination of these quantifications to recommend an ordered list of experiments for accurately annotating most proteins in the poorly studied related organisms to most biological processes, as well as a set of experiments that target each specific biological process. This experiment-planning framework could readily be adapted to the design of other types of large-scale experiments<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL11" >HL11</a> <span style="float: right;">Monday, July 18: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>A survey of genomic traces reveals a common sequencing error, RNA editing, and DNA editing.</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Erez Levanon , Bar-Ilan University, Israel<br><br><strong>Additional authors:</strong><br>Alexander Wait Zaranek, Harvard Medical School,, United States<br>Tomer Zecharia, Compugen LTD, Israel<br>Tom  Clegg, Scalable Computing Experts, United States<br>George Church, Harvard Medical School, United States<br>
<br><strong>Area Session Chair: </strong>Erik Bongcam-Rudloff    <br>
<br><strong>Presentation Overview:</strong> <br>Most biomedical, genomic research begins with the painstaking assembly of a ‘‘reference genome’’ for the organism of interest. Implicit in this process is an assumption that genomic information is constant throughout an organism. There are enzymes, however, that can change, or ‘‘edit,’’ genomic information so that variations from the reference can exist within a single organism. In this work, we analyze the raw data used to assemble the reference genomes of ten organisms to discover evidence for editing. We found candidates for DNA and RNA editing as well as a sequencing error that has become incorporated into commonly used genomic resources. Our analysis demonstrates the utility of raw genomic data for the discovery of some editing events and sets the stage for further analysis as sequencing costs continue to decrease exponentially.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL12" >HL12</a> <span style="float: right;">Monday, July 18: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Pi Release From Myosin: A Simulation Analysis of Possible Pathways</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Marco Cecchini , University of Strasbourg, France<br><br><strong>Additional authors:</strong><br>Martin Karplus, Harvard University, United States<br>Yuri Alexeev, Institute of Food Research, United Kingdom<br>
<br><strong>Area Session Chair: </strong>Nir Ben-Tal
    <br>
<br><strong>Presentation Overview:</strong> <br>The release of phosphate (Pi) is an important element in actomyosin function that has been shown to be accelerated by the binding of myosin to actin. To provide information about the structural elements important for Pi release, possible escape pathways from various isolated myosin II structures have been determined by molecular dynamics simulations designed for studying such slow processes. The residues forming the pathways were identified and their role evaluated by mutant simulations. Pi release is slow in the pre-powerstroke structure, an important element in preventing the powerstroke prior to actin binding, and is much more rapid for Pi modeled into the post-rigor and rigor-like structures. The backdoor route suggested by Yount et al. is dominant in the pre-powerstroke and post-rigor states, while a different path is most important in the rigor-like state. This finding suggests a novel mechanism for the actin-activated acceleration of Pi release.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL13" >HL13</a> <span style="float: right;">Monday, July 18: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Initial steps towards a production platform for DNA sequence analysis on the grid</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Barbera Van Schaik , Academic Medical Center, Netherlands<br><br><strong>Additional authors:</strong><br>Angela Luyf, Academic Medical Center, Netherlands<br>Michel de Vries, Academic Medical Center, Netherlands<br>Frank Baas, Academic Medical Center, Netherlands<br>Antoine van Kampen, Academic Medical Center, Netherlands<br>Silvia Olabarriaga, Academic Medical Center, Netherlands<br>
<br><strong>Area Session Chair: </strong>Erik Bongcam-Rudloff
    <br>
<br><strong>Presentation Overview:</strong> <br>Next generation sequencing confronts bioinformaticians with new challenges regarding data storage and analysis. Therefore it is needed to switch to other IT infrastructures. Grid and workflow technology can help to handle the data more efficiently and facilitate collaborations.<br><br>In this study we reused a platform that was developed for the analysis of medical images. Data transfer, workflow execution and monitoring are operated from one interface. We developed workflows for two sequence alignment tools for which the analysis time was significantly reduced. All workflows are available for the members of two Dutch virtual organizations and all components are open source.<br><br>The availability of in-house expertise and tools facilitates the usage of grids by new users. Our first results indicate that this is a practical, powerful and scalable solution. We currently adopt this methodology on a daily basis for DNA sequencing and other applications. More information and source code via http://www.bioinformaticslaboratory.nl/<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL14" >HL14</a> <span style="float: right;">Monday, July 18: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>The imprint of codons on protein structure</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Charlotte Deane , Oxford University, United Kingdom<br>
<br><strong>Area Session Chair: </strong>Nir Ben-Tal
    <br>
<br><strong>Presentation Overview:</strong> <br>The central dogma of molecular biology describes the unidirectional flow of interpretable data from genetic sequence to protein sequence. This has led to the idea that a protein’s structure is dependent only on its amino acid sequence. Analysing the input (mRNA) and output (protein) of translation, we find that local protein structure information is encoded in the mRNA nucleotide sequence. Using a detailed mapping between over 4000 solved protein structures and their mRNA we have carried out a comprehensive analysis of codon usage across many organisms. We found no evidence that domain boundaries are enriched with slow codons. In fact, genes seemingly avoid slow codons around structurally defined domain boundaries. Translation speed, however, does decrease at the transition into secondary structure. These results support the premise that codons encode more information than merely amino acids and give insight into the role of translation in protein folding.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL15" >HL15</a> <span style="float: right;">Monday, July 18: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>SlideSort: Fast and exact algorithm for Next Generation Sequencing data analysis</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Kana Shimizu , National Institute of Advanced Industrial Science and Technology, Japan<br><br><strong>Additional authors:</strong><br>Koji Tsuda, National Institute of Advanced Industrial Science and Technology, Japan<br>
<br><strong>Area Session Chair: </strong>Erik Bongcam-Rudloff
    <br>
<br><strong>Presentation Overview:</strong> <br>Next Generation Sequencing (NGS) technology calls for fast and accurate algorithms that can evaluate sequence similarity for a huge amount data. In this study, we designed and implemented exact algorithm SlideSort that finds all similar pairs whose edit-distance does not exceed a given threshold from NGS data, which helps many important analyses, such as de novo genome assembly, identification of frequently appearing sequence patterns and accurate clustering.<br>Using an efficient pattern growth algorithm, SlideSort discovers chains of common k-mers to narrow down the search. Compared to existing methods based on single k-mer, our method is more effective in reducing the number of edit-distance calculations. In comparison to state-of-the-art methods, our method is much faster in finding remote matches, scaling easily to tens of millions of sequences. Our software has an additional function of single link clustering, which is useful in summarizing NGS data for further processing.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL16" >HL16</a> <span style="float: right;">Monday, July 18: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Predicting genetic modifier loci using functional gene networks</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Insuk Lee , Yonsei University, Korea, Rep<br><br><strong>Additional authors:</strong><br>Ben  Lehner, EMBL-CRG Systems Biology Research Unit, Spain<br>Tanya  Vavouri, 2EMBL-CRG Systems Biology Research Unit, Spain<br>Junha Shin, Yonsei University, Korea, Rep<br>Andrew Fraser, University of Toronto, Canada<br>Edward Marcotte, University of Texas at Austin, United States<br>
<br><strong>Area Session Chair: </strong>Nir Ben-Tal    <br>
<br><strong>Presentation Overview:</strong> <br>Most phenotypes are genetically complex with contributions from mutations in many different genes. Mutations in more than one gene can combine synergistically to cause phenotypic change and systematic studies in model organisms show that these genetic interactions are pervasive. However, in human association studies such non-additive genetic interactions are very difficult to identify because of a lack of statistical power — simply put, the number of potential interactions is too vast. One approach to resolve this is to predict candidate modifier interactions between loci, and then to specifically test these for associations with the phenotype. Here we describe a general method for predicting genetic interactions based on the use of integrated functional gene networks. We show that in both S. cerevisiae and C. elegans a single high coverage, high quality functional network can successfully predict genetic modifiers for the majority of genes. We demonstrate how it is possible to rapidly expand the number of modifier loci known for a gene, predicting and validating new genetic interactions for each of three signal transduction genes. We propose that this approach, termed network-guided modifier screening, provides a general strategy for predicting genetic interactions. This work thus suggests that a high quality integrated human gene network will provide a powerful resource for modifier locus discovery in many different diseases.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL17" >HL17</a> <span style="float: right;">Monday, July 18: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Genome3D: a viewer-model framework for integrating and visualizing multi-scale epigenomic information within a three-dimensional genome</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> W Zheng , Medical University of South Carolina, United States<br><br><strong>Additional authors:</strong><br>Thomas Asbury, Sequenta, Inc, United States<br>Matt Mitman, Maxgaming Technologies, Inc, United States<br>Jijun Tang, University of South Carolina, United States<br>
<br><strong>Area Session Chair: </strong>Ivo Hofacker    <br>
<br><strong>Presentation Overview:</strong> <br>We have created the first model-view framework of eukaryotic genome, Genome3D, to enable integration and visualization of genomic and epigenomic data in a three-dimensional space.  Our model of the physical genome implicitly contains all levels of structure and hierarchy, and provides our underlying platform for integrating multi-scale structural and genomic information within three dimensions.  The viewer is designed to display data from multiple scales and uses a hierarchical model of the relative positions of all nucleotide atoms in the cell nucleus, i.e., the physical genome.  Genome3D does not intend to replace but rather works with UCSC genome browser, complementing its functionality by visualizing structural and epigenomic information in 3D space.  Genome3D can significantly advance genome research in inferring epigenomic knowledge, studying long range inter- and intra-chromosome interaction, and analyzing structural feature of genetic variations, and will have a profound impact on genome information integration and analysis.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL18" >HL18</a> <span style="float: right;">Monday, July 18: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>The Impact of Multifunctional Genes on "Guilt by Association" Analysis</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Jesse Gillis , University of British Columbia, Canada<br><br><strong>Additional authors:</strong><br>Paul Pavlidis, University of British Columbia, Canada<br>
<br><strong>Area Session Chair: </strong>Michal Linial
    <br>
<br><strong>Presentation Overview:</strong> <br>Many previous studies have shown that by using variants of “guilt-by-association”, gene function predictions can be made with high statistical confidence. In these studies, it is assumed that the “associations” in the data (e.g., protein interactions) of a gene are necessary in establishing “guilt”. Here we show that gene multifunctionality, rather than association, is a primary driver of gene function prediction. We first show that knowledge of the degree of multifunctionality alone can produce astonishingly strong performance when used as a predictor of gene function. We then demonstrate how multifunctionality is encoded in gene interaction data and feeds forward into function prediction. We find that high-quality gene function predictions can be made using data that possesses no information on which gene interacts with which. We suggest that this bias due to multifunctionality is important to control for, with widespread implications for the interpretation of genomics studies.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL19" >HL19</a> <span style="float: right;">Monday, July 18: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Structure determination of genomes and genomic domains by satisfaction of spatial restraints.</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Marc A. Marti-Renom , Prince Felipe Research Center, Spain<br><br><strong>Additional authors:</strong><br>Davide Bau, Prince Felipe Research Center, Spain<br>
<br><strong>Area Session Chair: </strong>Ivo Hofacker
    <br>
<br><strong>Presentation Overview:</strong> <br>The genome three-dimensional (3D) organization plays important, yet poorly understood roles in gene regulation.  Chromosomes assume multiple distinct conformations in relation to the expression status of resident genes and undergo dramatic alterations in higher order structure through the cell cycle.  Despite advances in microscopy, a general technique to determine the 3D conformation of chromatin has been lacking.  We developed a new method for the determination of the 3D conformation of chromatin domains in the interphase nucleus, which combines 5C experiments with the computational Integrative Modeling Platform (IMP).  The general approach of our method, which has been applied to study the 3D conformation of the ?-globin domain in the human genome [1] and the Caulobacter crescentus whole genome, opens the field for comprehensive studies of the 3D conformation of chromosomal domains and contributes to a more complete characterization of genome regulation.<br><br>[1] D. Baù et al. Nat Struct Mol Biol 18 (2011) 107.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL20" >HL20</a> <span style="float: right;">Monday, July 18: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Bringing order to protein disorder through comparative genomics and genetic interactions</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Philip Kim , University of Toronto, Canada<br><br><strong>Additional authors:</strong><br>Jeremy Bellay, University of Minnesota, United States<br>Sangjo Han, University of Toronto, Canada<br>Magali Michaut, University of Toronto, Canada<br>Taehyung Kim, University of Toronto, Canada<br>Michael Costanzo, University of Toronto, Canada<br>Charles Boone, University of Toronto, Canada<br>Gary Bader, University of Toronto, Canada<br>Chad Myers, University of Minnesota, Canada<br>
<br><strong>Area Session Chair: </strong>Michal Linial    <br>
<br><strong>Presentation Overview:</strong> <br>Intrinsically disordered regions are widespread, especially in proteomes of higher eukaryotes, and have been associated with a plethora of different cellular functions. Here, we attempt to better understand the different roles of disorder using a novel analysis that leverages both comparative genomics and genetic interactions. Strikingly, we find that disorder can be partitioned into three biologically distinct phenomena: regions where disorder is conserved but with quickly evolving amino acid sequences (“flexible disorder”), regions of conserved disorder with also highly conserved amino acid sequence (“constrained disorder”) and, lastly, non-conserved disorder. Flexible disorder is closest to canonical protein disorder and is associated with signaling pathways and multi-functionality. Conversely, constrained disorder has markedly different functional attributes and is involved in RNA binding and protein chaperones. Finally, non-conserved disorder appears largely non-functional. These distinctions provide both an informative division of disorder functionality and imply common underlying mechanisms that support these functions<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL21" >HL21</a> <span style="float: right;">Monday, July 18: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>A Unifying Theory for GC3 Biology in Plants and Animals</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Tatiana TATARINOVA , University of Glamorgan, United Kingdom<br><br><strong>Additional authors:</strong><br>Nickolai Alexandrov, Ceres, United States<br>John  Bouck, Ceres, United States<br>Kenneth Feldmann, University of Arizona , United States<br>
<br><strong>Area Session Chair: </strong>Ivo Hofacker    <br>
<br><strong>Presentation Overview:</strong> <br>There is a well-documented bias for cytosine and guanine at the third position in a subset of transcripts within a single organism; it is present in some plant species and warm-blooded vertebrates. We demonstrated that in certain organisms the amount of GC at the wobble position (GC3) can be used to distinguish two classes of genes. We highlight the following features of genes with high GC3content: they (1) provide more targets for methylation, (2) exhibit more variable expression, (3) more frequently possess certain transcription factor binding sites, (4) are predominant in certain classes of genes and (5) have a GC3 content that increases from 5'to 3'. These observations led us to formulate a hypothesis to explain GC3 bimodality in grasses and later extend it to other species. High levels of GC3 typify a class of genes regulated through DNA methylation or are a legacy of accelerated evolution through gene conversion.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL22" >HL22</a> <span style="float: right;">Monday, July 18: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Model-Based Learning for All SCOP Families</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Stefan Kramer , TU Muenchen, Germany<br><br><strong>Additional authors:</strong><br>Tobias Hamp, TU Muenchen, Germany<br>Fabian Buchwald, TU Muenchen, Germany<br>Fabian Birzele, Roche Deutschland GmbH, Germany<br>
<br><strong>Area Session Chair: </strong>Michal Linial
    <br>
<br><strong>Presentation Overview:</strong> <br>As the automated annotation of genomic and proteomic data is becoming increasingly important, big community efforts aim at computationally predicting their manual classification, as found for example in SCOP. These methods fall into two categories: instance-based (e.g. alignments of a target against templates, followed by the assignment of the class of the best template to the target) and model-based (e.g. a Neural Network) methods. In this context, model-based algorithms have supposedly been unfit for a full-scale application due to the large presence of extremely small classes. Only integration with instance-based methods could enable their universal use.In the talk, we show that it is, for SCOP, effectively impossible to find an integration that is guaranteed to outperform the instance-based-only counterpart. Further, we show that model-based-only classifiers can be applied to arbitrary class sizes and exhibit the so far best reported accuracy to predict the SCOP family of a protein.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL23" >HL23</a> <span style="float: right;">Monday, July 18: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Strengths and limitations of the federal guidance on synthetic DNA</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Jean Peccoud , Virginia Tech, United States<br><br><strong>Additional authors:</strong><br>Laura  Adam, Virginia Tech, United States<br>Michael Kozar, Virginia Tech, United States<br>Gaelle Letort, Virginia Tech, United States<br>Olivier Mirat, Virginia Tech, United States<br>Arunima Srivastava,, Virginia Tech, United States<br>Tyler Steward, Virginia Tech, United States<br>Mandy Wilson, Virginia Tech, United States<br>
<br><strong>Area Session Chair: </strong>Ivo Hofacker
    <br>
<br><strong>Presentation Overview:</strong> <br>An implementation of the sequence screening method recommended by the U.S. Government to prevent the misuse of gene synthesis highlights improvements over the protocols proposed by the industry. Since it does not rely on a database of curated sequences, its deployment is fast and inexpensive. Without resulting in an unacceptable computational cost, breaking sequences into 200 bp fragments translated in six frames precludes the hiding of sequences of concern within longer, benign sequences. A standardized dictionary of keywords used to interpret alignment results and a realistic suite of annotated test sequences are still needed to assess the performance of the screen software implementations. Beyond its biosecurity application, this screening algorithm can be used to enforce other policies and regulations affecting the biotechnology industry. It is also likely to find a variety of other applications such as partitioning sequencing reads by species in metagenomic samples, forensic, or for clinical diagnostic.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL24" >HL24</a> <span style="float: right;">Monday, July 18: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Next-generation genome alignment with LAST</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Paul Horton , AIST, Computational Biology Research Center, Japan<br><br><strong>Additional authors:</strong><br>Martin Frith, AIST, Japan<br>Raymond Wan, The University of Tokyo, Japan<br>Kengo Sato, The University of Tokyo, Japan<br>Szymon Kielbasa, Max Planck Institute for Molecular Genetics, Japan<br>
<br><strong>Area Session Chair: </strong>Michal Linial
    <br>
<br><strong>Presentation Overview:</strong> <br>We present LAST, an open-source software package to replace BLAST.<br><br>BLAST and related sequence similarity tools are arguably the most successful of all bioinformatics applications. However they are not fully adequate for important tasks such as mammalian genome-genome alignment and tera-scale mapping of sequence reads.<br><br>While BLAST searchers are based on fixed length exact match "seeds", LAST employs the concept of adaptive length seeds. We show that adaptive seeds are robust to highly repetitive (e.g. mammalian) and biased composition (e.g.  malaria) genomes. LAST also introduces improved methods for xeno-mapping, e.g. of mammoth reads to an elephant genome.<br><br>For the task of genome vs genome alignment, LAST is often 10-100 times faster than BLAST for similar levels of sensitivity. In fact, LAST is the first method that can sensitively compare giga-scale, repeat-rich sequences -- all previous methods either have low sensitivity e.g. DNA read mappers, or must heavily suppress repeats e.g. BLASTZ.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL25" >HL25</a> <span style="float: right;">Tuesday, July 19: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>A scalable approach for discovering conserved active subnetworks across species</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Raamesh Deshpande , University of Minnesota-Twin Cities, United States<br><br><strong>Additional authors:</strong><br>Shikha Sharma, University of Minnesota-Twin Cities, United States<br>Wei-Shou Hu, University of Minnesota-Twin Cities, United States<br>Catherine Verfaillie, Catholic University Leuven, United States<br>Chad Myers, University of Minnesota-Twin Cities, United States<br>
<br><strong>Area Session Chair: </strong>Yanay Ofran    <br>
<br><strong>Presentation Overview:</strong> <br>Overlaying differential changes in gene expression on protein interaction networks has proven to be a useful approach to interpreting the cell's dynamic response to a changing environment. We have extended this idea to enable the discovery of active subnetworks across species.  Specifically, we present a scalable, cross-species network search algorithm, neXus (Network - cross(X)-species - Search), that discovers conserved, active subnetworks based on parallel differential expression studies in multiple species. We applied our approach to identify conserved modules that are differentially active in stem cells relative to differentiated cells based on analogous gene expression studies and functional linkage networks from mouse and human. We find hundreds of conserved subnetworks enriched for stem cell-associated functions such as cell cycle, DNA repair, and chromatin modification processes. Furthermore, we demonstrate that a comparative approach to subnetwork discovery has many statistical advantages over the single-species formulation, which can enable more reliable module discovery.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL26" >HL26</a> <span style="float: right;">Tuesday, July 19: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Benchmarking Ontologies: Bigger or Better?</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Lixia Yao , Columbia University, United States<br><br><strong>Additional authors:</strong><br>Andrey Rzhetsky, University Of Chicago, United States<br>Anna Divoli, University Of Chicago, United States<br>Ilya Mayzus, University Of Chicago, United States<br>James Evans, University Of Chicago, United States<br>
<br><strong>Area Session Chair: </strong>Paul Horton    <br>
<br><strong>Presentation Overview:</strong> <br>An ontology represents the concepts and their interrelation within a knowledge domain. Many ontologies have been developed in biomedicine, providing standardized vocabularies to describe genes and proteins, anatomical structures, physiological phenotypes or diseases, and many other phenomena. Scientists use them to encode observations and experimental results, and to perform integrative analysis to discover new knowledge. A remaining challenge is to evaluate how well an ontology represents the underlying knowledge domain. We introduce a family of metrics, including breadth and depth, to capture the conceptual and relational coverage and parsimony of an ontology. We test these measures using four commonly used medical ontologies and seven popular English thesauri (ontologies of synonyms) with respect to text from medicine, news and novels. Results demonstrate that both medical ontologies and English thesauri have a small overlap in concepts and relations, and suggest further efforts to tighten the fit between ontologies and biomedical knowledge domain.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL27" >HL27</a> <span style="float: right;">Tuesday, July 19: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Network Modeling Identifies Molecular Functions Targeted by miR-204 to  Suppress Head and Neck Tumor Metastasis and Mechanisms of Therapeutic Resistance</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Yves Lussier , The University of Chicago, United States<br><br><strong>Additional authors:</strong><br>Mark Gerstein, Yale, United States<br>Rosie H Xing, University of Chicago, United States<br>Younghee  Lee, University of Chicago, United States<br>Xinan "Holly" Yang, University of Chicago, United States<br>Yong Huang, University of Chicago, United States<br>Qingbei Zhang, University of Chicago, United States<br>Jianrong Li, University of Chicago, United States<br>Hanli Fan, University of Chicago, United States<br>Rifat Hasina, University of Chicago, United States<br>Mark Lingen, University of Chicago, United States<br>Chao Cheung, Yale University , United States<br>Ralph Weichselbaum, University of Chicago, United States<br>
<br><strong>Area Session Chair: </strong>Yanay Ofran
    <br>
<br><strong>Presentation Overview:</strong> <br>Relevance: Accurately modeling microRNA regulation of oncogenic phenotype via its targets is relevant to a broad audience interest because of heightened interest in microRNA-directed therapy and the computational innovations that range from microRNA network models to the genetics of acquired oncogenic phenotypes. Previous studies lack in depth since only a few genes are biologically confirmed as microRNA targets in vitro and rarely in vivo. Additionally, key biological systems perturbed by altered microRNA functions in the context of cancer remain to be identified. This paper demonstrates how to bioinformatically integrate genetics knowledge, gene expression, and molecular network properties, to uncover previously unknown connections between microRNAs, their regulated genes, and their dynamics for streamlined and comprehensive  biological validations.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL28" >HL28</a> <span style="float: right;">Tuesday, July 19: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>miRGator v2.0 and the construction of miRNA-disease network</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Wankyu Kim , Ewha Womans University, Korea, Rep<br><br><strong>Additional authors:</strong><br>Sooyoung Cho, Ewha Womans University, Korea, Rep<br>Yukyung Jun, Ewha Womans University, Korea, Rep<br>Minjeong Ko, Ewha Womans University, Korea, Rep<br>Sanghyuk Lee, Ewha Womans University, Korea, Rep<br>
<br><strong>Area Session Chair: </strong>Paul Horton
    <br>
<br><strong>Presentation Overview:</strong> <br>miRGator is developed as an integrated database of microRNA-associated gene expression, target prediction, disease association and genomic annotation, in order to facilitate functional investigation of miRNAs (http://miRGator.kobic.re.kr). It contains (i) human miRNA expression profiles under various conditions, (ii) paired expression profiles of both mRNAs and miRNAs, (iii) gene expression profiles under miRNA-perturbation (e.g. miRNA knockout and overexpression), (iv) known/predicted miRNA targets and (v) miRNA-disease associations. In total, >8000 miRNA expression profiles, ?300 miRNA-perturbed gene expression profiles and ~2000 mRNA expression profiles are compiled with manual annotations on disease, tissue type and perturbation. Additionally, disease signature genes were extracted from ~12,000 gene expression profiles for ~100 human diseases. By integrating these data sets, a series of novel associations between human diseases and miRNAs is extracted by systematically comparing disease and target signature genes from various sources. Our approach correctly predicted known disease-miRNA associations with high accuracy as well as novel associations.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL29" >HL29</a> <span style="float: right;">Tuesday, July 19: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Mutation Impact Mining using SADI Semantic Web Services</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Christopher Baker , University of New Brunswick, Canada<br><br><strong>Additional authors:</strong><br>Alexandre Riazanov, University of New Brunswick, Canada<br>Jonas Laurila, National Food Administration, Sweden<br>
<br><strong>Area Session Chair: </strong>Yanay Ofran
    <br>
<br><strong>Presentation Overview:</strong> <br>We report on a platform for mining and integration of mutation impacts from the literature. Core features of this infrastructure are: a GATE pipeline for extracting impacts of mutations on proteins populating and OWL-DL mutation impact ontology, establishment of semantic database for storing the results of text mining, the SADI framework as a medium for publishing mutation impact software and data. Through multiple case studies we demonstrate the utility of SADI (a set of conventions for creating web services with semantic descriptions that facilitate automatic service discovery and workflow orchestration) to facilitate ad-hoc knowledge discovery through a single SPARQL interface (SHARE) to a registry of SADI services. We illustrate integration of mutation impact services with external SADI services providing information about related biological entities, such as proteins, pathways, SNPS and drugs. SADI provides an effective way of exposing our mutation impact data for reuse by a variety of stakeholders.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL30" >HL30</a> <span style="float: right;">Tuesday, July 19: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Fast and Efficient Dynamic Nested Effects Models</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Holger Fröhlich , University of Bonn, Bonn-Aachen International Center for IT, Germany<br><br><strong>Additional authors:</strong><br>Paurush Praveen, University of Bonn, Bonn-Aachen International Center for IT, Germany<br>Tresch Achim, Ludwig-Maximilians-University Muenchen, Gene Center Munich, Germany<br>
<br><strong>Area Session Chair: </strong>Paul Horton    <br>
<br><strong>Presentation Overview:</strong> <br>Reverse engineering of biological networks is a key for the understanding<br>of biological systems. The exact knowledge of interdependencies between<br>proteins in the living cell is crucial for the identification of drug<br>targets for various diseases. However, due to the complexity of the<br>system a complete picture with detailed knowledge of the behavior<br>of individual proteins is still out of reach. Nonetheless, the advent<br>of gene perturbation techniques like RNA interference (RNAi),<br>opened new perspectives for network reconstruction by boosting the<br>ability to subject organisms to well defined interventions.<br><br>Nested Effects Models (NEMs; Markowetz et al., Bioinformatics, 2005) have been introduced as a statistical<br>approach to estimate the upstream signal flow from the downstream nested<br>subset structure of high-dimensional perturbation effects (measured e.g. on microarrays). The method was substantially<br>extended later on by a number of authors and successfully applied to various<br>datasets (Markowetz et al., Bioinformatics, 2005; Tresch & Markowetz, Stat. Appl. Genome Biol., 2007; Froehlich et al., BMC Bioinformatics 2007; Froehlich et al., Bioinformatics, 2008; Froehlich et al., Biometrical Journal, 2009; Zeller et al., EURASIP J. on Bioinf. and Syst. Biol. 2009; Anchang et al., PNAS, 2009). The connection of NEMs to Bayesian Networks and factor graph models has been highlighted (Zeller et al., EURASIP J. on Bioinf. and Syst. Biol. 2009; Vaske et al., PLOS Comp. Biol., 2009). <br><br>Here we introcude a computationally attractive extension of NEMs that enables the analysis of perturbation time series data (measured e.g. on microarrays). It thus complements the attempt of Anchang et al. (PNAS, 2009) to extend static NEMs to the modeling of perturbation time series measurements. Most importantly, this allows for the resolution of feedback loops in the signaling cascade, as well as for the discrimination of direct and indirect signalling. In contrast to Anchang et al. the key idea in our model is to unroll the signal flow over time. This allows for a computation showing some similarity to Dynamic Bayesian Networks and naturally extends the classical NEM formulation. Our model circumvents the need for time consuming Gibbs sampling, which makes it also computationally attractive.<br><br>We performed extensive simulations of our model (also compared to a static NEM) to investigate its dependency on the length of time series, the sizes and architectures of the networks to be learned, and on the amount of available data. Our results indicate a very high specificity together with a good sensitivity of our method. The high specificity can be attributed to a special network structure prior favoring sparse networks here, but more generally could also incorporate prior beliefs on specific edges.<br><br>We applied our model to data investigating self-renewal in murine embryonic stem cell development in mice (Ivanova et al., Nature, 2006). We found a good accordance of our estimated network between 6 key proteins (5 transcription factors) and the biological literature. Moreover, our result generally agrees with the previous published one by Anchang et al., although being more sparse.<br><br>In summary we believe that our approach can serve as a useful tool to generate data driven hypotheses about signaling and/or transcriptional networks based on high-dimensional perturbation effects.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL31" >HL31</a> <span style="float: right;">Tuesday, July 19: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>The chicken or the egg problem in epidemiology: untangling the mutual impact of transmission network dynamic and epidemic spread.</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Christel Kamp , Paul-Ehrlich-Institut, Germany<br>
<br><strong>Area Session Chair: </strong>Yanay Ofran
    <br>
<br><strong>Presentation Overview:</strong> <br>Network models are well established in computational biology as they acknowledge that system dynamics is rarely determined by components alone but by interactions between components. In epidemiology, not individual behaviour but rather the structure and dynamics of the transmission network between hosts strongly influence the time scale and patterns of epidemics. Within a mathematical framework we further show that also the reverse is true and quantify the impact that epidemics have on the way contacts are made among healthy and infected individuals. This information is relevant for epidemic control as will be shown in a case study on HIV epidemics: The clustering and mixing of infected individuals depend both on the transmission network and stage of epidemic – as do the contributions of primarily and latently infected individuals to epidemic spread. In conclusion, the flexibility of the framework will be highlighted in combination with potential applications in epidemiology and beyond.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL32" >HL32</a> <span style="float: right;">Tuesday, July 19: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>C(alpha)-trace model of the transmembrane domain of human copper transporter 1, motion and functional implications</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Nir Ben-Tal , Tel-Aviv University, Israel<br><br><strong>Additional authors:</strong><br>Yariv  Barkan, Tel-Aviv University, Israel<br>Turkan Haliloglu, Bogazici University, Turkey<br>Nir Ben-Tal, Tel-Aviv University, Israel<br>
<br><strong>Area Session Chair: </strong>Paul Horton    <br>
<br><strong>Presentation Overview:</strong> <br>The human copper transporter 1 (hCTR1) is essential for copper uptake and is implicated in sensitivity to chemotherapy drugs. Using the hCTR1 cryoelectron microscopy (cryoEM) map and evolutionary data, we constructed a C?-trace model of the membrane region. Investigating the model's global dynamics through elastic network models, hCTR1’s MxxxM and GxxxG motifs were shown to have significant roles in the two slowest modes of motion. For example, in one of these modes the glycine residues of the GxxxG motif appeared to serve as hinge points and the copper-binding methionine residues of the MxxxM motif manifested cooperative rotational motion, possibly reflecting activation at the pore’s extracellular entrance. We suggest a molecular mechanism of copper transport in which this motif serves both as a gate and as a selectivity filter. We also suggest residues that are responsible for pH activation.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL33" >HL33</a> <span style="float: right;">Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Organization of mammalian genomes with respect to the nuclear lamina</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Wouter Meuleman , Netherlands Cancer Institute / Delft University of Technology, Netherlands<br><br><strong>Additional authors:</strong><br>Daan Peric-Hupkes, Netherlands Cancer Institute, Netherlands<br>Marcel Reinders, Delft University of Technology, Netherlands<br>Lodewyk Wessels, Netherlands Cancer Institute, Netherlands<br>Bas van Steensel, Netherlands Cancer Institute, Netherlands<br>
<br><strong>Area Session Chair: </strong>Terry Gaasterland
    <br>
<br><strong>Presentation Overview:</strong> <br>The three-dimensional organization of chromosomes within the nucleus is largely unknown. We present high-resolution maps of the interaction of human and mouse genomes with the nuclear lamina, providing detailed views of the spatial organization of interphase chromosomes.  These maps reveal substantial refolding of chromosomes during differentiation, involving hundreds of genes that collectively determine cellular identity. We illustrate the involvement of lamina-genome interactions in the control of gene expression programs.<br><br>We find that a substantial portion of the spatial chromosome organization is identical across all assayed cell types. This is even the case between species, despite extensive chromosomal rearrangements in mouse and human.  Using the genomic sequence alone, we can accurately predict these lamina-genome interactions in both mouse and human. This indicates that the organization of the genome is, to a large extent, hard-coded in the primary sequence. Based on this, we conclude with a mechanistic explanation of lamina-genome interactions.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL34" >HL34</a> <span style="float: right;">Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Multi-species integrative biclustering</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Peter Waltman , New York University, United States<br>
<br><strong>Area Session Chair: </strong>Donna Slonim
    <br>
<br><strong>Presentation Overview:</strong> <br>A key challenge in the analysis of functional genomics data is the identification of modules of genes with similar regulatory controls, a non-trivial problem due to the complexity of regulatory networks. Recent works that compare functional genomics datasets for closely related species reveal that many co-regulated gene groups are conserved across several species. This suggests that comparative analysis of multiple-species functional genomics datasets could prove powerful in accurately identifying those conserved modules. <br>We describe an extension of our cMonkey algorithm that allows for the simultaneous biclustering of heterogeneous data collections spanning multiple species. We present results from the multi-species biclustering of a group of Gram positive bacteria containing Bacillus subtilis, Bacillus anthracis, and Listeria monocytogenes. We identify conserved groups of orthologous genes, yielding evolutionary insights into the formation and surprisingly high degree of conservation of regulatory modules across these three species. We report a temporal difference between the two Bacillus species in the expression of a conserved bicluster of metabolic genes required for spore formation. In addition, we discuss the unexpected identification of a highly expressed flagellum assembly bicluster in non-motile B. anthracis.  Analysis of biclusters obtained revealed a large number of gene groups with conserved modularity and high biological significance as judged by several measures of cluster quality. We show that the method provides a framework that allows data and insights from well-studied organisms to complement the analysis of related but less well studied organisms.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL35" >HL35</a> <span style="float: right;">Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Model-based method for transcription factor target identification with limited data</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Magnus Rattray , University of Sheffield, United Kingdom<br><br><strong>Additional authors:</strong><br>Neil Lawrence, University of Sheffield, United Kingdom<br>Antti Honkela, University of Helsinki, Finland<br>Eileen Furlong, EMBL Heidelberg, Germany<br>Charles Girardot, EMBL Heidelberg, Germany<br>Hilary Gustafson, EMBL Heidelberg, Germany<br>Ya-Hsin Liu, National Cheng Kung University, Taiwan<br>Jonatan Ropponen, Aalto University, Finland<br>Pei Gao, University of Cambridge, United Kingdom<br>
<br><strong>Area Session Chair: </strong>Terry Gaasterland
    <br>
<br><strong>Presentation Overview:</strong> <br>A fundamental problem in systems biology is uncovering the structure and dynamics of gene regulatory networks. A first step is to identify the targets of regulatory molecules such as transcription factors (TFs). We introduce a computational method for inferring targets of transcription factors from gene expression time-series data. Our approach works by fitting a simple model of transcriptional regulation to gene expression data for each putative target gene. The model is used to identify targets of transcription factors of interest. We apply the method to identify targets of TFs regulating Drosophila mesoderm development. Using only wild-type microarray time course data we make predictions that are validated with high significance using ChIP-chip data from the same system. Our model-based approach gives better target predictions than knock-out data for the same TFs and we show how spatial expression data can improve the accuracy of predictions. The method is available as a Bioconductor package allowing easy parallel implementation.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL36" >HL36</a> <span style="float: right;">Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Inference of Signaling Network Architecture and Dynamics from High-throughput Combinatorial RNAi Screens</em></strong><br><strong>Room:</strong> Hall A<br /><strong>Presenting author:</strong> Chris Bakal , The Institute of Cancer Research, United Kingdom<br><br><strong>Additional authors:</strong><br>Oaz Nir, Massachusetts Institute of Technology, United States<br>Norbert Perrimon, Harvard Medical School, United States<br>Bonnie Berger, Massachusetts Institute of Technology, United States<br>
<br><strong>Area Session Chair: </strong>Donna Slonim
    <br>
<br><strong>Presentation Overview:</strong> <br>Biological signaling networks are highly complex systems that act to regulate fundamental cellular processes such as growth, proliferation, differentiation and migration. Because the actions of these networks underpin both health and disease, describing their architecture and dynamics is a fundamental challenge in biology. High-content image based RNAi screens provide a wealth of data regarding the contribution of single genes to different phenotypes, but mapping cellular networks from this data is a daunting challenge. Here we describe a systematic computational framework based on a classification model for network inference using high-dimensional single-cell morphological data. We applied this method to a dataset comprised of images generated in the course of a double RNAi screen and generated a network of RhoGTPases and RhoGTPase Activating Proteins – which are essential for cell shape migration and motility in all eukaryotic cells. Furthermore, we have recently extended these methods to model signaling network dynamics.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>Highlights Track: <strong></strong><a name="HL37" >HL37</a> <span style="float: right;">Tuesday, July 19: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Quantitative analysis of the Drosophila segmentation regulatory network using pattern generating potentials</em></strong><br><strong>Room:</strong> Hall E1<br /><strong>Presenting author:</strong> Saurabh Sinha , University of Illinois Urbana-Champaign, United States<br><br><strong>Additional authors:</strong><br>Saurabh Sinha, University of Illinois Urbana-Champaign, United States<br>Majid Kazemian, University of Illinois Urbana-Champaign, United States<br>Charles Blatti, University of Illinois Urbana-Champaign, United States<br>Adam Richards, University of Massachusetts Medical School, United States<br>Michael McCutchan, Arizona State University, United States<br>Noriko Wakabayashi-Ito, University of Massachusetts Medical School, United States<br>Ann Hammonds, Lawrence Berkeley National Laboratory, United States<br>Susan Celniker, Lawrence Berkeley National Laboratory, United States<br>Sudhir Kumar, Arizona State University, United States<br>Scot Wolfe, University of Massachusetts Medical School, United States<br>Michael Brodsky, University of Massachusetts Medical School, United States<br>
<br><strong>Area Session Chair: </strong>Terry Gaasterland
    <br>
<br><strong>Presentation Overview:</strong> <br>The developmental program specifying segmentation along the anterior-posterior axis of the Drosophila embryo is encoded in DNA segments called cis-regulatory “modules”. Previous work has identified some of these modules along with their related transcription factors. We present a novel computational framework that turns a qualitative and fragmented understanding of modules and factor-module interactions into a quantitative, systems-level view. Our model utilizes experimentally characterized binding specificities of transcription factors and gene expression patterns to describe how multiple transcription factors act together in a module to determine its regulatory activity. This logistic regression-based model can explain the expression patterns of known modules, infer factor-module interactions and identify novel modules of the regulatory network by quantifying their potential to drive a gene's expression. As databases of binding motifs and gene expression patterns grow, this new approach provides a general method to decode transcriptional regulatory sequences and networks.<br><a href="#up" style="font-size:9px;">TOP</a><br><br></div></td> 
</tr> 
</table> 
<span class="article_separator">&nbsp;</span> 
                                                      	</div> 
						</div> 
						<div class="clr"></div> 
					</div> 
					<div class="clr"></div> 
				</div> 
 
				<div id="whitebox_b"> 
					<div id="whitebox_bl"> 
						<div id="whitebox_br"></div> 
					</div> 
				</div> 
			</div> 
 
			<div id="footerspacer"></div> 
		</div> 
 
		
	</div> 
</div> 
 
 
</body> 

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/highlights.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:48:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->

<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/highlights.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:24 GMT -->
</html> 

