
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/latebreakingresearch.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:48:45 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/latebreakingresearch.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:24 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Late Breaking Research</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="http://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top">




<h3>Late Breaking Research Presentation Schedule</h3> <div class=style1><div class=subheading2><a name="LBR01" >LBR01</a> <span style="float: right;">Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Potential function of proteins encoded by chimeric transcripts</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Milana Frenkel-Morgenstern, Spanish National Cancer Research Centre (CNIO), Spain<br><br><strong>Additional authors:</strong><br>Milana Frenkel-Morgenstern, Spanish National Cancer Research Centre (CNIO), Spain<br>Iakes Ezkurdia, Spanish National Cancer Research Centre (CNIO), Spain<br>Michael Tress, Spanish National Cancer Research Centre (CNIO), Spain<br>Alfonso Valencia, Spanish National Cancer Research Centre (CNIO), Spain<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>Chimeric RNAs are produced by trans-splicing of independent RNA transcripts. These RNAs have been reported in many organisms, and they have been identified when analyzing ensembles of ESTs and data from high-throughput paired-end RNA-seq. However, to what extent the resulting chimeric RNAs have been selected during evolution in order to design new functional proteins currently remains unclear. <br>An extensive review of the composition of the proteins translated from such chimeric transcripts in human, mouse and fruit fly, emphasized the mixture of the missing and complete functional domains of the proteins assembling the chimera. In many cases, such chimeric proteins contain transmembrane domains and signal peptides that can potentially change their cellular location, altering the corresponding cellular processes. By reviewing the protein domains preserved in these chimeras, one can hypothesize that those resulting from trans-splicing may produce proteins with novel functions that in a relevant number of cases can be associated with dominant negative phenotypes. <br>Indeed, we report intriguing examples of potential chimeric proteins the existence of which was confirmed in mass-spectrometry experiments in human, mouse and fly. The exploration of a number of chimeras encoded by complementary DNA strands in humans provides additional evidence of the potentially complex molecular isoforms that can be generated by alternative transcription.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20187.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR02" >LBR02</a> <span style="float: right;">Sunday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Identification, classification and functional role of viral protein kinases.</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Nidhi Tyagi, Indian Institute of Science, India<br><br><strong>Additional authors:</strong><br>Nidhi Tyagi, Indian Institution of Science, in<br>Narayanaswamy  Srinivasan, Indian Institute of Science, in<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>Protein kinases encoded by viral genomes play a major role in infection, replication and survival of viruses. Our study focuses on identification, classification and recognition of functional role(s) of various protein kinases from viruses. Using traditional sequence homology detection tools, sequence alignment methods and phylogenetic approach, protein kinases were recognized. 9,82,857 protein sequences from 53,315 viral genomes (including strains) have been used in this analysis. Protein kinases are identified using a combination of profile-based search methods such as PSI-BLAST and RPS-BLAST and HMMER approach. 470 protein kinase domains in 466 proteins from 241 viral genomes have been identified. We classified these viral protein kinases in 30 subfamilies. Remarkably, more than 50% of subfamilies are reported viral-specific as sequences from various eukaryotic/prokaryotic sources sharing homology with these sequences could not be identified. We also hint at their potential biological roles by functional annotation. Surprisingly, some of the largely populated subfamilies share homology with well characterized eukaryotic protein kinase groups such as tyrosine, tyrosine kinase-like, Casein kinase 1, CAML (Ca2+/calmodulin-dependant protein kinase), CMGC (Cyclin-dependant kinases (CDK), Mitogen activated kinases and Glycogen synthase kinases and CDK-like kinases). Phylogenetic analysis of viral kinases along with members from various eukaryotic protein kinase groups suggest close homologous relationship among them indicating horizontal gene transfer. Such viral kinases represent the mutated form of proteins from carcinogenic cells. Some viral kinase homologues participate in metabolic pathways raising concerns for host-pathogen interaction (host protein mimicking).<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20204.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR03" >LBR03</a> <span style="float: right;">Sunday, July 17
: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>A structural and dynamical model of human telomerase</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Samuel Flores, Uppsala University, Sweden<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>Mutations in the telomerase complex disrupt either nucleic acid binding or catalysis, and are the cause of numerous human diseases.  Despite its importance, the structure of the human telomerase complex has not been observed crystallographically, nor are its dynamics understood in detail. Fragments of this complex from Tetrahymena thermophila and, more controversially,Tribolium castaneum have been crystallized. Biochemical probes provide important insight into dynamics. In this work we use available structural fragments to build a homology model of human TERT, and validate the result with functional assays. We then generate a trajectory of telomere elongation following a “typewriter” mechanism: the RNA template moves to keep the end of the growing telomere in the active site, disengaging after every 6-residue extension to execute a “carriage return” and go back to its starting position.  A hairpin can easily form in the telomere, from DNA residues leaving the telomere-template duplex. The trajectory is consistent with available experimental evidence and suggests focused biochemical experiments for further validation.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20217.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR04" >LBR04</a> <span style="float: right;">Sunday, July 17: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>The multiple Specificity Landscape of Peptide Recognition Domains</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>David Gfeller, Swiss Institute of Bioinformatics, Switzerland<br><br><strong>Additional authors:</strong><br>David Gfeller, Swiss Institute of Bioinformatics, Switzerland<br>Frank  Butty, University of Toronto, Canada<br>Marta Wierzbicka, University of Toronto, Canada<br>Erik Verschueren, CRG-Centre de Regulacio Genomica, Spain<br>Peter Vanhee, CRG-Centre de Regulacio Genomica, Spain<br>Haiming Huang, University of Toronto, Canada<br>Andreas Ernst, University of Toronto, Canada<br>Nisa Dar, University of Toronto, Canada<br>Igor Stagljar, University of Toronto, Canada<br>Luis Serrano, CRG-Centre de Regulacio Genomica, Spain<br>Sachdev S Sidhu, University of Toronto, Canada<br>Gary D Bader, University of Toronto, Canada<br>Philip M Kim, University of Toronto, Canada<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>Modular protein interaction domains form the building blocks of eukaryotic signaling pathways. Many of them, known as Peptide Recognition Domains, mediate protein interactions by recognizing short, linear amino acid stretches on the surface of their cognate partners with high specificity. Residues in these stretches are usually assumed to contribute independently to binding, which has led to a simplified understanding of protein interactions. Conversely, we observe in large binding peptide datasets that different residue positions display highly significant correlations for many domains in three distinct families (PDZ, SH3 and WW). These correlation patterns reveal a widespread occurrence of multiple binding specificity and give novel structural insights into protein interactions. We show that multiple specificity more accurately predicts protein interactions and experimentally validate some of the predictions for the human proteins DLG1 and SCRIB. Overall, our results reveal a rich specificity landscape in peptide recognition domains, suggesting new ways of encoding specificity in protein interaction networks.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20282.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR05" >LBR05</a> <span style="float: right;">Sunday, July 17: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Investigating The Evolution of Novel Enzyme Function And Chemistry Within Structurally Defined Protein Superfamilies</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Nicholas Furnham, European Bioinformatics Institute, United Kingdom<br><br><strong>Additional authors:</strong><br>Nicholas Furnham, European Bioinformatics Institute, United Kingdom<br>Gemma Holliday, European Bioinformatics Institute, United Kingdom<br>Ian Sillitoe, University College London, United Kingdom<br>Alison Cuff, University College London, United Kingdom<br>Christine Orengo, University College London, United Kingdom<br>Janet Thornton, European Bioinformatics Institute, United Kingdom<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>A significant proportion of gene products are annotated as having enzymatic functions, which, as biological catalysis, are essential for life. In addition, many of the targets of pharmaceutical drugs are acting to modify the behavior of enzymes. Thus, an understanding of how enzymes have evolved to undertake the wide variety of reactions they perform is essential to many studies in biology and medicinal chemistry. To unravel this problem requires the combination of protein three-dimensional structural, sequence, phylogenetic and chemistry information. We have combined this variety of data in an automatic pipeline for investigating enzyme functional evolution within structurally defined protein superfamilies. This has permitted us to analysis all enzymatic superfamilies cataloged by the CATH database. In addition to showing relationships between structures and sequences though phylogeny, we are able to show relationships of the small molecule metabolites the enzymes are acting on as well as similarities between the mechanisms by which an enzyme performs its overall reaction. This allows us to demonstrate the power of combining the range of information to show features across multiple superfamilies as well as unique qualities of specific enzyme superfamilies, thus providing a means to improve function prediction and contribute to the design of novel enzyme functions.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20192.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR06" >LBR06</a> <span style="float: right;">Sunday, July 17: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Quantitative analysis of cellular composition in primary breast tumours deconvolutes molecular signatures and elucidates impact of tumour microenvironment</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Yinyin Yuan, Cambridge Research Institute, United Kingdom<br><br><strong>Additional authors:</strong><br>Yinyin Yuan, Cancer Research UK, United Kingdom<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>A major obstacle in refining molecular cancer signatures is the heterogeneity of cellular composition in most tumour samples, which is a mixture of cancer, stromal, and immune cells. These cells form an integral part of the tumour microenvironment, but their contributions may obscure the “pure” cancer signal in genomic and transcriptomic data. We show how to extract cellular composition using an automated image analysis system and integrate it with DNA copy number and expression data. Quantifying the percentage of tumour cells allows improving the signal-to-noise ratio in copy-number profiles. We have applied our approach to 290 images of frozen breast tumour samples for which we have also obtained copy-number and gene-expression profiles. Our quantitative results are highly correlated to the qualitative results obtained by human experts on the same samples. On the other hand, we show that location of cancer cells allows distinguishing tumours of different complexity using spatial statistics. Also, 	quantifying the density of lymphocytes allows distinguishing cases with active immune system from cases with less active immune system. We show that lymphocyte density is predictive of prognosis in ER-negative subtype, which includes both an aggressive form of breast cancer but also an often over-treated patient subgroup.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20221.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR07" >LBR07</a> <span style="float: right;">Sunday, July 17: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Modulators of microRNA Activity Regulate Glioblastoma Pathogenesis</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Pavel Sumazin, Columbia Univesity, United States<br><br><strong>Additional authors:</strong><br>Xuerui Yang, Columbia University, United States<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>MicroRNAs (miRs) have been shown to drive pathogenesis and prognosis in cancer tumors, including glioblastoma, the most common and the most aggressive type of primary human brain tumor. MiR activity is modulated by miR-target abundance and by post-transcriptional factors that regulate miRISC-mediated mRNA degradation. MiR-activity modulators are potential key regulators of cancer, but the extent to which they regulate cancer remains unknown. We developed an algorithm to help assemble the repertory of tumor-specific miR-activity modulators as a step towards elucidating their regulatory effect on pathogenesis of cancer. We identified over a hundred glioblastoma miR-activity modulators, which activate or suppress miRISC-mediated regulation through protein interaction or act as miR decoys by titrating miRs away from other mRNAs. We identified dozens of oncogenes and tumor suppressors that are regulated by miR-activity regulators in glioblastoma, including master regulators RUNX1 and PTEN. We experimentally validated WIPF2 as a miRISC-mediated modulator of RUNX1, and PALB2 and WNT7A as miRISC-mediated modulators of PTEN. Moreover, a number of genes, including RUNX1, were identified as decoy modulators of PTEN. Interestingly, PTEN and its 7 modulators bear genetic alterations in 60% of 462 GBM samples. Both modulator silencing and PTEN-3’UTR transfection confirmed these 7 modulators as decoy modulators of PTEN, suggesting that genetic alterations at these genes post-transcriptionally regulate PTEN. In summary, our results suggest that miR modulators post-transcriptionally regulate the expression of master regulators of glioblastoma, thus playing a significant role in its tumorigenesis and progression.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20247.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR08" >LBR08</a> <span style="float: right;">Sunday, July 17: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Post-transcriptional regulators of microRNA biogenesis regulate pathogenesis of cancer</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Wei-Jen Chung, Columbia University, United States<br><br><strong>Additional authors:</strong><br>Pavel Sumazin, Columbia University, United States<br>
<br><strong>Area Session Chair: </strong>Curtis Huttenhower
    <br>
<br><strong>Abstract:</strong><br>MicroRNAs (miRs) have emerged as key regulators of both normal and pathologic phenotypes, including cancer, but fine grained regulation of their biogenesis is still poorly understood. In order to understand the extent and specificity of miR-biogenesis control, as well as the role of miR-biogenesis regulators in tumorigenesis and cancer progression, we set out to identify these regulators and profile their targets. We developed an algorithm for genome-wide inference of miR-biogenesis regulators, and identified regulators that are specific to individual miRs or miR families in glioma and ovarian cancer. Our algorithm identified known biogenesis regulators, including DGCR8, HNRNPA1, DDX5, LIN28 and SMAD family proteins, and predicted new miR-biogenesis regulators that are common to both cancers or specific to one tumor type. We validated miR-biogenesis regulators that are common to both cancers and target tumor- and prognosis-specific miRs, including oncomirs miR-218, miR-23b, and miR-155. We showed that miR-biogenesis regulators can act before or after cropping by DROSHA, and that they can alter expression of large sets of microRNAs. Our results suggest that miR biogenesis is a complex, context specific, and finely-regulated process, and that miR-biogenesis regulators may influence tumor initiation and progression by altering the expression of specific tumor-suppressor miRs and oncomirs or by modifying large miR programs.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20240.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR09" >LBR09</a> <span style="float: right;">Monday, July 18: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>SpliceGrapher: Predicting Splice Graphs from Diverse Evidence</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Asa Ben-Hur, Colorado State University, United States<br><br><strong>Additional authors:</strong><br>Asa Ben-Hur, Colorado State University, United States<br>Mark Rogers, Colorado State University, United States<br>Anireddy, S.N. Reddy, Colorado State University, United States<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>Deep transcriptome sequencing with next-generation sequencing (NGS) provides unprecedented opportunities for researchers to assess the extent of alternative splicing in many species.  Although it is inexpensive and easy to obtain whole transcriptome data in this manner, one limitation has been the lack of versatile methods to analyze the data.  We present a new method called SpliceGrapher, which is designed to enhance existing gene annotations on the basis of NGS reads and EST alignments.  SpliceGrapher predicts splice graphs which are a compact representation of all the ways in which a gene's exons may be assembled.  We demonstrate our approach using NGS read data from Arabidopsis and grape, and find that SpliceGrapher's predictions are better aligned with the existing annotations than those of other tools.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20226.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR10" >LBR10</a> <span style="float: right;">Monday, July 18: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>A Comprehensive Computational Model for Analyzing Gene Translation</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Tamir Tuller, Weizmann Institute of Science, Israel<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>We describe a model of gene translation that is based on all the physical and dynamical aspects of this process.  The Whole-cell Simulation Translation Model (WSTM) predicts fundamental features of the translation process, including translation rates, protein abundance-levels, ribosomal densities and the relation between all these variables, better than alternative ('non-physical') approaches.  In addition, we show that the WSTM model can be used for accurately inferring various variables that could not be inferred by previous predictors, such as genes’ initiation rates and cost of translation. <br>We find that increasing the number of available ribosomes (or equivalently the initiation rate) increases the genomic translation rate and the mean ribosome density only up to a certain point, beyond which both saturate. Strikingly, assuming that the translation system is tuned to work at the pre-saturation point leads to the best predictions of experimental data. This result may suggest that in all the organisms analyzed, ribosome allocation is optimized to the pre-saturation point. Remarkably, the gap between the performances the WSTM and the alternative predictors is strikingly large in the case of heterologous genes, testifying to the model’s promising biotechnological value in predicting the protein abundance of heterologous proteins before expressing them in the desired host. We find that the different local features of the coding sequence of a gene (codon bias, amino acid charge, and mRNA folding energy) affect its translation elongation (partial correlations of - 0.47, -0.22, and 0.31 respectively with ribosomal density; all p-values < 0.0015).<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20160.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR11" >LBR11</a> <span style="float: right;">Monday, July 18: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Differential expression with RNA-seq: a matter of Depth</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Ana Conesa, CIPF, Spain<br><br><strong>Additional authors:</strong><br>Ana Conesa, CIPF, Spain<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>We investigate the relationship between sequencing depth and differential expression detection in RNA-seq experiments. We show that current statistical approaches have a strong dependency of their differential expression calls on the amount of available reads and that this leads to a high number  false discoveries specially of genes of short length, with small fold-change differences including off-target non coding genes. We present a novel methodology -NOISeq- which is robust to this kind of biases. NOISeq, by adopting an empirical approach to model the null distribution of differential expression, captures better the shape of noise in RNA-seq data and obtains more accurate and consistent results.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20239.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR12" >LBR12</a> <span style="float: right;">Monday, July 18: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Inferring Gene Regulatory Networks from Expression Data using Tree-based Methods</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Van Anh Huynh-Thu, University of Liege, Belgium<br><br><strong>Additional authors:</strong><br>Van Anh Huynh-Thu, University of Liege, Belgium<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>One of the pressing open problems of computational systems biology is the elucidation of the topology of genetic regulatory networks (GRNs) using high throughput genomic data, in particular microarray gene expression data. In this work, we present GENIE3, an original algorithm for the inference of GRNs. GENIE3 decomposes the prediction of a regulatory network between p genes into p different regression problems. In each of the regression problems, the expression pattern of one of the genes (target gene) is predicted from the expression patterns of all the other genes (input genes), using tree-based ensemble methods. The importance of an input gene in the prediction of the target gene expression pattern is taken as an indication of a putative regulatory link. The whole network is then reconstructed by aggregating putative links over all genes. Our method was evaluated in the context of the Dialogue for Reverse Engineering Assessments and Methods (DREAM) challenge, which is an annual international competition aiming at the evaluation of GRN inference algorithms on benchmarks of simulated and real data. The GENIE3 method was best performer on the DREAM4 In Silico Multifactorial challenge in 2009 and on the DREAM5 Network Inference challenge in 2010. In addition, the resulting method is simple and generic, making it adaptable to other types of genomic data and interactions.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20190.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR13" >LBR13</a> <span style="float: right;">Monday, July 18: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Age matters:  Functional annotation tailored to human development improves our understanding of fetal transcriptional profiles</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Donna Slonim, Tufts University, United States<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>Recent successes in detecting fetal RNA in amniotic fluid and in profiling both fetal DNA and RNA from maternal blood suggest novel, comprehensive, and minimally-invasive approaches to prenatal diagnosis and treatment.  However, our ability to interpret transcriptional profiles meaningfully depends increasingly on prior knowledge of functionally related gene sets relevant to the subject of the study.  We established the DFLAT (Developmental FunctionaL Annotation at Tufts) project to identify developmentally-relevant gene sets that my help us better interpret transcriptional profiles of the living human fetus.  To assess the novel contributions of the current DFLAT collection of gene sets, we re-analyzed three of our previously-published data sets:  one that identifies fetal RNA in blood from pregnant mothers carrying healthy, full-term fetuses, and two comparing amniotic fluid of common autosomal aneuploid pregnancies to controls (in Trisomy 21 and Trisomy 18).  In all three cases, the DFLAT data has facilitated novel, valuable observations.   In some cases the new results clarify and refine existing knowledge; in others, they suggest new, early molecular causes for well-studied consequences of common aneuploidies.   These new results demonstrate that functional annotation focused on developmentally active proteins can indeed improve our ability to interpret developmental expression profiles.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20243.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR14" >LBR14</a> <span style="float: right;">Monday, July 18: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Analysis of bacterial genes with disrupted ORFs reveals frequent utilization of Programmed Ribosomal Frameshifting (PRF) and Programmed Transcriptional Realignment (PTR).</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Pavel Baranov, University College Cork, Ireland<br><br><strong>Additional authors:</strong><br>Pavel Baranov, University College Cork, Ireland<br>Virag Sharma, University College Cork, Ireland<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>Bacterial genome annotations contain a number of CDSs that in spite of disruption(s) of initial reading frame encode a single continuous polypeptide. Such disruptions have different origins: sequencing errors, frameshift or stop codon mutations, as well as instances of unconventional decoding utilization (Recoding). We have extracted over one thousand CDSs with annotated disruptions and found out that  about 75% of them can be clustered into 64 groups based on sequence similarity. Analysis of the clusters revealed deep phylogenetic conservation of ORF organization as well as presence of conserved sequence patterns that indicate likely utilization of the programmed ribosomal frameshifting (PRF) and the programmed transcriptional realignment (PTR). Further enrichment of these clusters with homologous nucleotide sequences revealed over six thousand candidate genes utilizing PRF or PTR. Analysis of the patterns of conservation apparently associated with non-triplet decoding revealed  presence of both previously characterized frameshift-prone sequences  and a few novel ones. Since the starting point of our analysis was a set of genes with already annotated disruptions, it is highly plausible that in this study we have identified only a fraction of all bacterial genes that utilize PRF and PTR. In addition to identification of a large number of alternatively decoded genes, a surprising observation as that about half of them are expressed via PTR - a mechanism that, in contrast with PRF, has not yet received substantial attention.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20234.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR15" >LBR15</a> <span style="float: right;">Monday, July 18: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Metagenomic Biomarker Discovery and Explanation</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Nicola Segata, Harvard School of Public Health, United States<br><br><strong>Additional authors:</strong><br>Jacques Izard, The Forsyth Institute, United States<br>Levi  Waldron, Harvard School of Public Health, United States<br>Dirk Gevers, Broad Institute, United States<br>Larisa Miropolsky, Harvard School of Public Health, United States<br>Wendy Garrett, Harvard School of Public Health, United States<br>Curtis Huttenhower, Harvard School of Public Health, United States<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>Understanding how and why biomolecular activity differs among environmental conditions or disease phenotypes is one of the central questions addressed by high-throughput biology. Biomarker discovery, the process of finding and explaining these differences, has proven to be both a methodological challenge in high-dimensional statistics and biologically challenging to interpret. Metagenomics provides a new avenue for biomarker discovery, since changes in the composition and functional activity of microbial communities can provide insight into ecological differences among communities or diagnostic or prognostic power when applied to the human microbiome. We propose the LDA Effect Size (LEfSe) algorithm to discover and explain microbial and functional biomarkers in the human microbiota and other microbiomes. We demonstrate this method to be effective in mining human microbiomes for metagenomic biomarkers associated with mucosal tissues and with different levels of oxygen availability. Similarly, when applied to 16S rRNA gene data describing a murine ulcerative colitis gut community, LEfSe confirms the key role played by Bifidobacterium in this disease and suggests the involvement of additional clades including the Clostridia and Metascardovia. Finally, we provide characterizations of microbial functional activity from metagenomic community sequencing, comparing environmental bacterial and viral microbiomes and distinguishing the infant gut microbiota from adult. A quantitative validation of LEfSe in comparison to existing microbial biomarker discovery methods and to standard statistical approaches (including synthetic data) highlights a lower false positive rate, consistent ranking of biomarkers’ relevance, and concise representations of taxonomic and functional shifts in microbial communities associated with environmental conditions or disease phenotypes.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20238.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR16" >LBR16</a> <span style="float: right;">Monday, July 18: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>The Human gut Ecosystem: Gut Microbiome and Host Transcriptome  in Breast-fed vs. Formula-fed Infants</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Iddo Friedberg, Miami University, United States<br><br><strong>Additional authors:</strong><br>Iddo Friedberg, Miami University, United States<br>Scott Schwartz, Texas A&M University, United States<br>Ivan Ivanov, Texas A&M University, United States<br>Laurie Davidson, Texas A&M University, United States<br>Jennifer Goldsby, Texas A&M University, United States<br>David  Dahl, Texas A&M University, United States<br>Edward Dougherty, Texas A&M University, United States<br>Damir Herman, University of Arkasas, United States<br>Sharon Donovan, University of Illinois Urbana-Champaign, United States<br>Robert Chapkin, Texas A&M University, United States<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>We present a study of the gut microbiome and gut transcriptome of breast-fed (BF) and formula-fed (FF) babies, at the age of three months. Here we have established the feasibility of non-invasive data collection to evaluate the impact of nutritional and other environmental exposures on the developing infant gut-microbial ecosystem. At the same time, we have developed a robust computational pipeline for revealing significant correlations between commensal microbiome and the host transcriptome. Two interesting findings are that Firmicutes and Actinobacter are bacterial phyla that differentially colonize FF and BF babies. Also, the gut microbiome of FF babies contains more matches to virulence genes. At the same time, FF babies seem to have a more activated innate immune system as evidenced by their gut transcriptome.<br>In the future, the statistical methodology presented here can be used to study other conditions that are associated with the gut microbiome both in infants and in adults, including obesity, inflammatory bowel diseases and colon cancer.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20231.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR17" >LBR17</a> <span style="float: right;">Tuesday, July 19: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>STAGR: Software To Annotate Genome Rearrangement</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Egor Dolzhenko, Princeton University, United States<br>
<br><strong>Area Session Chair: </strong>Predrag Radivojac
    <br>
<br><strong>Abstract:</strong><br>Non-evolutionary sequence rearrangements arise in a number of different <br>biological contexts, including somatic recombination, chromatin diminution, <br>chromothrypsis, trans-splicing of RNA transcripts, and development of the <br>product somatic genome from the precursor germline genome in the <br>binucleate ciliates. The algorithm that we have developed is capable of <br>finding and annotating both the rearranged and nonrearranged pieces in <br>the precursor genome and product genome or transcriptome for all of these <br>cases, and may be useful for identifying novel rearrangements where no <br>specialized software exists. The rearranged sequence segments in the scrambled <br>precursor genome of the ciliate Oxytricha present the most challenging case for <br>annotating DNA rearrangements. In Oxytricha, our lab model organism, long <br>noncoding RNA templates help piece together and rearrange DNA segments<br>derived from the precursor genome, while deleting intervening regions and <br>as much as 95% of the precursor genome.<br><br>In the product genome, the product segments contain short overlapping<br>regions called pointers. The ability to map and annotate the locations<br>of product and precursor segments is critical in understanding the complex<br>developmental processes through which the precursor germline genome develops<br>into the mature, product genome. We have developed and implemented a method<br>for annotating the precursor and product genomes that reduces the problem to<br>finding a maximal weight matching in a bipartite graph.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20246.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR18" >LBR18</a> <span style="float: right;">Tuesday, July 19: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Methods for Phylogenetic Inference of Multidomain Evolution</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Maureen Stolzer, Carnegie Mellon University, United States<br><br><strong>Additional authors:</strong><br>Maureen Stolzer, Carnegie Mellon University, United States<br>Minli Xu, Carnegie Mellon University, United States<br>Katherine Siewert, Carnegie Mellon University, United States<br>Benjamin Vernot, University of Washington, United States<br>Ravi Chinoy, Carnegie Mellon University, United States<br>Dannie Durand, Carnegie Mellon University, United States<br>
<br><strong>Area Session Chair: </strong>Chad Myers    <br>
<br><strong>Abstract:</strong><br>We present algorithms and software for phylogenetic analysis of multidomain protein families. These families evolve via domain shuffling; i.e., domain duplication, insertion, and deletion - processes not captured by current phylogenetic models. Phylogenetics is fundamental to evolutionary analysis, as well as biomedical applications such as function annotation, drug design, and model organism research. Phylogenetic methods for multidomain proteins are urgently needed because of their prevalence and their evolutionary and functional importance. Current phylogenetic methods cannot be applied to multidomain families with varied architectures. These methods rely on the implicit assumption that the entire sequence has the same evolutionary history. However, domains co-occurring in the same protein can have different histories; phylogenetic trees for these domains will have different topologies.<br><br>Our novel method exploits this phylogenetic incongruence to infer a most parsimonious history of domain duplications, losses, and insertions; the timing of these events; and the ancestral domain architectures. Key features of our approach include (1) explicit representation of domain shuffling events; (2) a model that captures sequence variation within domain families; and (3) algorithms to infer historical events and ancestral domain architectures from comparison of domain phylogenies.<br><br>We demonstrate the utility of our method with in-depth analyses of well-studied multidomain families. We further present a genome-scale analysis of all the domain families in human. Our results suggest that a remarkably greater amount of domain shuffling may have occurred than predicted by models that do not consider sequence variation and underscore the importance of accurate domain architecture reconstruction for homology-based function prediction.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20255.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR19" >LBR19</a> <span style="float: right;">Tuesday, July 19: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Epistasis Detection in Complex Traits: An Application to Bipolar Disorder</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Michael Mooney, Oregon Health & Science University, United States<br><br><strong>Additional authors:</strong><br>Michael Mooney, Oregon Health & Science University, United States<br>Beth Wilmot, Oregon Health & Science University, United States<br>Shannon McWeeney, Oregon Health & Science University, United States<br>
<br><strong>Area Session Chair: </strong>Chad Myers
    <br>
<br><strong>Abstract:</strong><br>In recent years, large genome-wide association studies have been successful in revealing a number of genes thought to play a role in complex diseases. Yet, in most cases, the genetic associations discovered have accounted for only a small portion of the “genetic component” of these diseases, as the analyses were focused primarily on the effects of individual SNPs. In an attempt to reveal a greater portion of the genetic component of one such disease, bipolar disorder, we have adapted a machine learning technique, known as a genetic algorithm (GA), to search for multi-locus associations in the context of a large-scale genome-wide association study (GWAS). The GA is guided by the structure of a gene interaction network and is able to find statistically significant multi-locus associations without performing a prohibitively high number of statistical tests. In a GWAS data set from the NIMH-sponsored Bipolar Genome Study, consisting of 2033 cases and 1420 controls, we found three SNP pairs that were jointly associated with the disease (p-values: 7.48E-10, 1.17E-7 and 3.26E-7). We are in the process of attempting to replicate these results in other data sets.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20173.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR20" >LBR20</a> <span style="float: right;">Tuesday, July 19: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>A rational approach for aiding directed evolution towards DNA specificity of site-specific recombinases</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Josephine Abi Ghanem, Biotechnology Center  of the Technische Universität Dresden, Germany<br>
<br><strong>Area Session Chair: </strong>Chad Myers
    <br>
<br><strong>Abstract:</strong><br>Cre recombinase has been recently evolved, by using substrate-linked protein evolution, to recognize a sequence present in the 5?-LTR and 3?-LTR (loxLTR) of an integrated HIV-1 provirus. In vitro evolution approaches are generally tedious and time consuming. In addition, they are prone to lead to a diversity of binding profiles with considerable plasticity and relaxed specificity. On the other hand, the Cre/loxP system has been intensively studied structurally. The bulk of information makes the Cre/loxP system an ideal case to dissect the molecular mechanisms responsible for the specificity in Cre/DNA complexes and to be able to accelerate engineering processes. In this work we make use of computational approaches together with large scale sequencing data obtained from in vitro evolution studies to engineer DNA specificity into a relaxed specificity recombinase.  From a large set of sequenced clones we first generated a “consensus recombinase”, C16 that contains 16 conserved mutations.  C16 shows a relaxed specificity recognizing loxP and loxLTR. By applying molecular modelling and simulation techniques, we generated a new recombinase, C18 that contains two additional residues in the N–terminal domain predicted to switch target specificity. Strikingly, C18 showed virtually no recombination on loxP, while fully recombining the loxLTR sequence, demonstrating that rational design linked to substrate-linked protein evolution can be used to design DNA specificity into evolved recombinases.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20184.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR21" >LBR21</a> <span style="float: right;">Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Co-expression network analysis reveals rewiring of maize transcriptome by domestication</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Roman Briskine, University of Minnesota, United States<br><br><strong>Additional authors:</strong><br>Roman Briskine, University of Minnesota, United States<br>Ruth Swanson-Wagner, University of Minnesota, United States<br>Robert  Schaefer, University of Minnesota, United States<br>Peter Tiffin, University of Minnesota, United States<br>Nathan Springer, University of Minnesota, United States<br>Chad Myers, University of Minnesota, United States<br>
<br><strong>Area Session Chair: </strong>Chad Myers
    <br>
<br><strong>Abstract:</strong><br>Traditional methods of comparative genomics exploit the abundance of sequence data to extend our knowledge about gene functions and evolutionary processes. However, gene expression profiling represents an attractive alternative for the development of complementary methods. One of the particularly interesting questions addressed by comparative genomics concerns the effects of domestication on wild progenitors. Standard sequence-based methods identified several thousand candidate genes that underlie the process of maize domestication from its wild ancestor, teosinte. Nevertheless, further research is necessary to narrow the list of candidates and extend the characterization of targeted genes. In this study, we built co-expression networks based on high-quality expression data from the diverse varieties of maize and teosinte. Global comparisons of the co-expression networks revealed evidence for transcriptome rewiring due to domestication. We also developed a novel bootstrapping approach to identify specific candidate genes that presumably were direct targets or downstream effects of selection during domestication. We show that our candidate list overlaps considerably with domestication genes identified through previous independent sequence-based analyses, an enrichment that was not obtained using existing approaches for comparative expression analysis. Our study demonstrates the utility of expression analysis applied to the question of maize domestication and provides new insights into robust methods for comparing expression patterns across species.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20244.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2><a name="LBR22" >LBR22</a> <span style="float: right;">Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Scalable metabolic reconstruction for metagenomic data and the human microbiome</em></strong><br><strong>Room: </strong>Hall F1<br /><strong>Presenting author: </strong>Curtis Huttenhower, Harvard School of Public Health, United States<br><br><strong>Additional authors:</strong><br>Sahar Abubucker, Washington University, United States<br>Nicola Segata, Harvard School of Public Health, United States<br>Alyxandria Schubert, University of Michigan, United States<br>Beltran Rodriguez-Mueller, San Diego State University, United States<br>Jeremy Zucker, Broad Institute, United States<br>Human Microbiome Project consortium  ,  , United States<br>Patrick Schloss, University of Michigan, United States<br>Dirk Gevers, Broad Institute, United States<br>Makedonka Mitreva, Washington University, United States<br>Curtis Huttenhower, Harvard School of Public Health, United States<br>
<br><strong>Area Session Chair: </strong>Chad Myers
    <br>
<br><strong>Abstract:</strong><br>Microbial communities carry out the majority of biochemical activity on the planet, and they play integral roles in metabolism and immune homeostasis in the human microbiome. Here, we describe HUMAnN, a methodology for determining the functional and metabolic potential of a microbial community by inferring pathways present or absent and their relative abundances directly from short metagenomic reads. We validated this methodology using a collection of four synthetic metagenomes, accurately determining the presence and abundance of pathways and outperforming standard best-hit approaches. Finally, we analyzed 741 samples drawn from 7 body sites on 103 individuals as part of the Human Microbiome Project (HMP), demonstrating the scalability of our methodology and the critical importance of microbial metabolism in the human microbiome. Previous studies have found that no organisms are present in all body sites or individuals; conversely, we find that 19 of 220 pathways are confidently present in every HMP community, and 53 are present in at least 90% of samples. This demonstrates a degree of functional consistency that is lacking at the organismal level - who's there varies, but what they're doing is more constant. Conversely, the relative abundances of most pathways vary among body sites but not individuals, suggesting that community function is dictated by microbial environment and less strongly by the host. This does not yet speak to host genetics, environment, or disease, as the HMP comprises a normal baseline of healthy individuals; each of these represents an additional area for future studies of microbial community function.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20253.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br></div></td> 
</tr> 
</table> 
<span class="article_separator">&nbsp;</span> 
                                                      	</div> 
						</div> 
						<div class="clr"></div> 
					</div> 
					<div class="clr"></div> 
				</div> 
 
				<div id="whitebox_b"> 
					<div id="whitebox_bl"> 
						<div id="whitebox_br"></div> 
					</div> 
				</div> 
			</div> 
 
			<div id="footerspacer"></div> 
		</div> 
 
		
	</div> 
</div> 
 
 
</body> 

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/latebreakingresearch.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:48:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->

<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/latebreakingresearch.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:24 GMT -->
</html> 