
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/oralposters.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:49:09 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/oralposters.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:30 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Oral Poster Presentations</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><h3>Oral Poster Presentations</h3> <div class=style1><p>
ISMB/ECCB 2011 introduces oral poster presentations for a select group of outstanding posters.
Works deemed exceptional in original review were passed on to three other reviewers each for further consideration.
24 posters were selected for oral presentations (8-minute talk) on the first day of the conference.
These Oral Poster Presentations will also be considered for "best poster" awards. <br> <br>
<strong>Note:</strong>
The presenting authors should be available to further discuss their poster content during regular poster-presentation times.
</p><br>
<div class=subheading2>

<a name="OPT01" >OPT01</a> <span style="float: right;">Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>ALISSA-An Automated Live Cell Imaging System for Signal Transduction Analyses</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Heinrich Huber, Royal College of Surgeons in Ireland, Ireland<br><br><strong>Additional authors:</strong><br>Heiko Dussmann, RCSI, Ireland<br>Perrine Paul, Institut Curie , France<br>Dimitris Kalamatianos, NUIM, Ireland<br>Endl Martina, Siemens Austria, Ireland<br>Jakub Wenus, NUIM, Ireland<br>Peter Wellstead, NUIM, Ireland<br>Jochen Prehn, RCSI, Ireland<br>

<br><strong>Abstract:</strong><br>Probe photo bleaching and the specimen’s sensitivity to photo toxicity severely limit the frequency and duration of exposures during time-lapse fluorescent microscopy experiments. Consequently, when a study of cellular processes requires measurements over hours or days, temporal resolution is limited, and spontaneous or rapid events may be missed. We have developed ALISSA, an automated live-cell imaging system for signal-transduction analysis. It allows an adaptation of imaging  modalities and laser resources tailored to the biological process, and extends temporal resolution from minutes to seconds. ALISSA employs online image analysis to detect cellular events that are then used to exercise microscope control. Our system can be integrated into standard fluorescence microscopes and applied to a large range of single cell experiments by using a graphical language.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT02" >OPT02</a> <span style="float: right;">Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>GenomeRNAi: A Phenotype Database for Large-scale RNAi Screens</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Esther Schmidt, German Cancer Research Center, Germany<br>

<br><strong>Abstract:</strong><br>RNA interference (RNAi) is a powerful method to generate loss-of-function phenotypes. It enables systematic genetic screens for every annotated gene in the genome. Genome-wide RNAi screens have been performed in human, mouse, drosophila and C.elegans and many RNAi libraries have become available to study phenotypes on a large scale.<br>With more and more datasets of RNAi-induced phenotypes becoming available, the systematic integration of functional information remains an important task. RNAi screens are performed using different types of assays from visible phenotypes to focused transcriptional readouts and provide a rich data source for functional annotation. Large-scale in vivo RNAi screens are now emerging, representing a new challenge in data representation and annotation, due to the complex nature of in vivo assays and phenotypes.<br>Currently, the GenomeRNAi database provides access to cell-based phenotypes in drosophila and human cells. It holds 97600 entries for human and 99700 for drosophila phenotypes extracted from 136 published RNAi screens. It also contains sequence and efficiency data for 300000 and 118000 RNAi reagents for human and drosophila, respectively. The database can be searched by phenotype, gene or RNAi probe and is publicly accessible at http://www.genomernai.org. <br>Our aim is comprehensive coverage of all published genome-wide RNAi screening data in GenomeRNAi, including large-scale data from drosophila in vivo screens. We have developed an annotation guide to ensure consistent data collection. Work to adapt it for in vivo data is ongoing. We will report on curation progress and present an update on new features on the GenomeRNAi website.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22429.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT03" >OPT03</a> <span style="float: right;">Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Annotare – a tool for annotating high-throughput biomedical investigations and resulting data</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Tony Burdett, European Bioinformatics Institute, United Kingdom<br><br><strong>Additional authors:</strong><br>Ravi Shankar, Stanford University, United States<br>Helen Parkinson, European Bioinformatics Institute, United Kingdom<br>Tony Burdett, European Bioinformatics Institute, United Kingdom<br>Emma Hastings, European Bioinformatics Institute, United Kingdom<br>Alvis Brazma, European Bioinformatics Institute, United Kingdom<br>Junmin Liu, IBM, United States<br>Michael  Miller, Institute for Systems Biology, United States<br>Chris Stoeckart, University of Pennsylvania School of Medicine , United States<br>Rashmi Srinivasa, 5am Solutions, United States<br>Joseph White, Dana Farber Cancer Institute, United States<br>Gavin Sherlock, Stanford University School of Medicine, United States<br>Catherine Ball, Stanford University School of Medicine, United States<br>

<br><strong>Abstract:</strong><br>Meta data describing high-throughput investigations enable unambiguous interpretation of experiments, experiment reproducibility, and meaningful searching and analysis of the resulting data. The microarray community has developed MAGE-TAB, an annotation format for microarray data acquisition and communication, and several public repositories of microarray data support microarray data submissions with MAGE-TAB annotations. In order to improve the volume, quality, and granularity of annotations, there is a compelling need for software that enables biologists to easily annotate such data. We have developed Annotare, a software tool that facilitates annotation of gene expression data in MAGE-TAB format. Using the tool, bench biologists can describe their investigations with the investigators’ contact details, experimental design, protocols that were employed and references to publications, and can also report on the relationships between biological materials, arrays, and experimental data produced in the investigation. Annotare features 1) a set of intuitive graphical user interface forms to create and modify annotations, 2) access to different biomedical ontologies that can be used as controlled vocabularies in the annotations, 3) a set of standard annotation templates indexed by different investigation elements, 4) a design wizard that solicits investigation details from the user and creates partial annotations that the user can then complete, and 5) a MAGE-TAB validator that checks for syntactic and semantic violations in the annotations. Annotare is a collaborative open-source software development effort involving different institutions. A version of the tool is available from Annotare’s project website: http://code.google.com/p/annotare/<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT04" >OPT04</a> <span style="float: right;">Sunday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Mapping the spread of HIV in Germany</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Glenn Lawyer, Max Planck Institute for Informatics, Germany
<br>

<br><strong>Abstract:</strong><br>The Resina Project tracks the spread of HIV-1 genotypes in German patients. To date, 35 participating clinics have gathered over 2700 genotypes, along with demographics including patient risk group and region of residence.<br><br>Genotypic similarity between viral samples can be converted into a graph linking related infections. This allows graph theoretical investigations of the data, such as measuring transmissions between various risk groups, the degree of interconnectivity within each group, and the structure and size of connected components. Additionally, the geographical information can be used to condense the graph to connections between regions. Measures on the condensed graph are also of interest, and the geographical structure allows the graph to be plotted over a map of Germany. <br><br>A conservative similarity threshold led to the following observations. Connected components in the full graph were mostly pairs or triplets. Only 5% of transmission chains had more than 10 patients. The homosexual risk group continues to be at the center of the epidemic, with all risk groups showing many connections to this group. This was surprising, given that two of the risk groups (intravenous drug users; people from endemic regions) were not expected to have strong overlap with the homosexual community. The reduced graph showed several interesting patterns. For example, two distinct networks of non-B HIV were found in the intravenous drug user risk group.<br><br>Graphical representation of genetic relationships reveals deep structure in the spread of HIV, both through mathematical analysis and visual display.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT05" >OPT05</a> <span style="float: right;">Sunday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Evolutionary meta-analysis reveals ancient constraints affecting missing heritability and reproducibility in disease association studies</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Joel Dudley, Stanford University, United States<br><br><strong>Additional authors:</strong><br>Rong Chen, Stanford University, United States<br>Atul Butte, Stanford University, United States<br>Sudhir Kumar, Arizona State University, United States<br>

<br><strong>Abstract:</strong><br>Genome-wide disease association studies contrast genetic variation between disease cohorts and healthy populations to discover single nucleotide polymorphisms (SNPs) and other genetic markers revealing underlying genetic architectures of human diseases. Despite many large efforts over the past decade, these studies are yet to identify many reproducible genetic variants that explain significant proportions of the heritable risk of common human diseases. Here, we report results from a multi-species comparative genomic meta-analysis of 6,720 risk variants for more than 420 disease phenotypes reported in 1,814 studies, which is aimed at investigating the role of evolutionary histories of genomic positions on the discovery, reproducibility, and missing heritability of disease associated SNPs (dSNPs) identified in association studies. We show that dSNPs are disproportionately discovered at conserved genomic loci in both coding and non-coding regions, as the effect size (odds ratio) of dSNPs relates strongly to the evolutionary conservation of their genomic positions. Our findings indicate that association studies are biased towards discovering rare variants, because strongly conserved positions only permit minor alleles with lowest frequencies. Using published data from a large case-control study, we demonstrate that the use of a straightforward multi-species evolutionary prior improves the power of association statistics to discover SNPs with reproducible genetic disease associations. Therefore, long-term evolutionary histories of genomic positions are poised to play a key role in reassessing data from existing disease association studies and in the design and analysis of future studies aimed at revealing the genetic basis of common human diseases.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT06" >OPT06</a> <span style="float: right;">Monday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Practical NGS Analysis: A Bioinformatician’s Perspective</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Abhishek Pratap, University of Maryland, Baltimore County, USA DOE-Joint Genome Institute, United States<br>

<br><strong>Abstract:</strong><br>The motivation for this poster comes from the questions I see across bioinfo mailing lists on how to handle NGS (Next Generation Sequencing) Data.  I plan to present various analysis methods and techniques that I have used or developed for analysis of data from Illumina class of sequencers in the last three years or so. <br><br>The focus remains on asking biologically pertinent questions efficiently in a timely manner from the raw data. More importantly making sure the conclusions make biological sense. I plan to address the following questions via this poster and engage in exciting discussions with my peers at ISMB.<br>1.	Data quality conversions. Why should you convert Illumina QScores to SangerQ?<br>2.	Error Corrections : Dynamic v/s Static trims<br>3.	Can I sequence more from the same library? : Evaluating Library Complexity<br>4.	Alignment: Which aligner is right for your data and question and why?<br>5.	Exome Capture : Knowing you captured the right region<br>6.	RNA Seq :  issues with read count and expression analysis<br>7.	SNP Calling :  Specificity v/s Sensitivity <br>8.	How to identify disease causing mutations with case and controls?<br>9.	Data exchange formats: How should I store and transfer my NGS data?<br><br>Many techniques that will be shown will use some of already available open source tools. My goal is to provide review for those and describe how I threaded different pipelines together to answer the questions that principal investigator’s asked of the data.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT07" >OPT07</a> <span style="float: right;">Sunday, July 17: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>A Novel Bioinformatics Pipeline for Identification and Characterization of Fusion Transcripts in 31 Breast Cancer and Normal Cell Lines</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Yan Asmann, Mayo Clinic, United States<br><br><strong>Additional authors:</strong><br>Asif Hossain, Mayo Clinic, United States<br>Brian Necela, Mayo Clinic, United States<br>Sumit Middha, Mayo Clinic, United States<br>Krishna Kalari, Mayo Clinic, United States<br>Zhifu Sun, Mayo Clinic, United States<br>High-Seng Chai, Mayo Clinic, United States<br>David Williamson, Illumina, United States<br>Derek Radisky, Mayo Clinic, United States<br>Gary Schroth, Illumina, United States<br>Jean-Pierre Kocher, Mayo Clinic, United States<br>Edith Perez, Mayo Clinic, United States<br>Aubrey Thompson, Mayo Clinic, United States<br>

<br><strong>Abstract:</strong><br>SnowShoes-FD, developed for fusion transcript detection in paired end mRNA-Seq data, employs multiple steps of false positive filtering to nominate fusion transcripts with near 100% confidence. Unique features include: (i) identification of multiple fusion isoforms from two gene partners; (ii) prediction of genomic rearrangements; (iii) identification of exon fusion boundaries; (iv) generation of a 5’ to 3’ fusion spanning sequence for PCR validation; (v) prediction of the protein sequences, including frame shift and amino acid insertions. We applied SnowShoes-FD to identify 50 fusion candidates in 22 breast cancer and 9 non-transformed cell lines. Five additional fusion candidates with two isoforms were confirmed. Thirty out of 55 fusion candidates had in-frame protein products. No fusion transcripts were detected in non-transformed cells. Consideration of the possible functions of a subset of predicted fusion proteins suggests several potentially important functions in transformation, including a possible new mechanism for overexpression of ERBB2 in a HER positive cell line. The source code of SnowShoes-FD is provided in two formats: one configured to run on the Sun Grid Engine for parallelization, and the other formatted to run on a single LINUX node. Executables in PERL are available for download from our website:  http://mayoresearch.mayo.edu/mayo/research/biostat/stand-alone-packages.cfm.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT08" >OPT08</a>  <font style='color: red; margin-left: 20px;'>Withdrawn</font> <span style="float: right;">Sunday, July 17: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Discovering Novel Patterns of DNA Methylation Landscape of Human Cancer Genome</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>G Reshmi, Rajiv Gandhi Centre for Biotechnology, India<br><br><strong>Additional authors:</strong><br>Priyanka James, Rajiv Gandhi Centre for Biotechnology, in<br>M.Radhakrishna  Pillai, Rajiv Gandhi Centre for Biotechnology, in<br>

<br><strong>Abstract:</strong><br>Differential DNA methylation is an essential epigenetic signal for gene regulation, development, and disease processes.  Aberrant methylation of CpG islands in promoter region is associated with tumor suppressor gene silencing in cancer cells. It is not known why some CpG islands are susceptible to aberrant methylation during tumor progression while others are not. To address this question we described a computational pattern recognition method that is used to predict the methylation landscape of human cancer genome. Using sequence-based rules derived from cancer cell methylation data has also been explored as a way to predict the pattern of aberrant methylation in cancer genome-wide. To gain more insights into how such epigenetic mechanism works in the human cells, we apply the best performing classifier the support vector machine approach. These studies have identified consensus sequences, proximity to repetitive elements and chromosomal location as potential factors influencing the likelihood of a CpG island becoming aberrantly methylated.Our results also shows that some of the gene promoters are highly co-methylated, demonstrating the evidence that genes are highly interactive epigenetically in human cells. Our data shed new light on the nature of methylation patterns in human cells, the sequence dependence of DNA methylation, and its function as epigenetic signal in gene regulation.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT09" >OPT09</a> <span style="float: right;">Sunday, July 17: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>CNAmet: an R package for integrating copy number, methylation and expression data</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Riku Louhimo, University of Helsinki, Finland<br><br><strong>Additional authors:</strong><br>Sampsa Hautaniemi, University of Helsinki, Finland<br>

<br><strong>Abstract:</strong><br>Genomic instability is a key enabling characteristic of tumorigenesis. Identification of regions with copy number alterations has uncovered deregulated tumor suppressors and oncogenes which play key roles in tumor progression and drug response. In addition to copy number changes, gene expression is regulated by DNA methylation. Gene copy number, methylation and expression can all be measured with microarrays which enables their computational integration. The goal of integrating copy number and expression data is to characterize genes essential to cancer progression. This is achieved by identifying genes that are both amplified and upregulated or deleted and downregulated. However, gene upregulation can also be caused by hypomethylation (decrease in methylation of cytosine and adenosine residues in DNA) and downregulation caused by hypermethylation. Since both copy number and methylation alterations can deregulate genes, integration of these three types of data should improve the characterization of essential genes in cancer. We introduce the CNAmet R package which integrates high-throughput copy number, methylation and expression data. Our primary goal is to identify genes that are simultaneously amplified, hypomethylated and upregulated, or deleted, hypermethylated and downregulated. To our knowledge CNAmet is the only software package for the three-way integration of copy number, methylation and expression. We illustrate the usefulness of CNAmet by analyzing copy number, methylation and expression data from 174 patients with glioblastoma multiforme, which is the most aggressive type of brain cancer, as well as 196 ovarian cancer patients.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT10" >OPT10</a> <span style="float: right;">Sunday, July 17: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Penalized regression elucidates aberration hotspots mediating subtype-specific transcriptional responses in breast cancer</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Yinyin Yuan, Cambridge Research Institute, United Kingdom<br><br><strong>Additional authors:</strong><br>Yinyin Yuan, Cancer Research UK, United Kingdom<br>Oscar Rueda, Cancer Research UK, United Kingdom<br>Christina  Curtis, Cancer Research UK, United Kingdom<br>Florian Markowetz, Cancer Research UK, United Kingdom<br>

<br><strong>Abstract:</strong><br>COPY(CNAs) associated with cancer are known to contribute to genomic instability and gene deregulation. Integrating copy-number data with gene expression helps to elucidate the mechanisms by which CNAs act and to identify the transcriptional Down-stream targets of copy-number changes. Such analyses can help to sort functional driver events from the many accompanying passenger alterations. However, the way CNAs affect gene expression can vary between different cellular contexts, for example between healthy tissue and tumour or between different subtypes of the same cancer. Thus it is important to develop computational approaches capable of inferring differential connectivity of regulatory networks in different cellular contexts. We propose a statistical deregulation model that integrates copy-number and expression data of different disease subtypes to jointly model common and differential regulatory relationships. Our model not only identifies copy-number alterations driving gene expression changes, but at the same time also predicts differences in regulation that distinguish one cancer subtype from the other. We implement our model in a penalized regression framework and demonstrate in a simulation study the feasibility and accuracy of our approach. On a real breast cancer dataset we show that we can identify both known and novel aspects of pathway deregulation in ER positive versus negative disease as well as crosstalk between pathways. Availability: The Bioconductor-compliant R package DANCE is available From www.markowetzlab.org/software/<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT11" >OPT11</a> <span style="float: right;">Sunday, July 17: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>The functional importance and detection of regulatory sequence variants</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Virginie Bernard, University of British Columbia, Canada<br><br><strong>Additional authors:</strong><br>Wyeth Wasserman, Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Canada<br>David Arenillas, Centre for Molecular Medicine and Therapeutics, Child and Family Research Institute, Canada<br>

<br><strong>Abstract:</strong><br>The convergence of high-throughput technologies for sequencing individual exomes and genomes and rapid advances in genome annotation are driving a neo-revolution in human genetics. This wave of family-based genetics analysis is revealing causal mutations responsible for striking phenotypes. By mapping the reads to the human genome reference and by searching for variations relative to the reference, a list of small nucleotide variations and structural variations is obtained. Analysis is required to reveal those variations most likely to contribute to a disease phenotype within a family.  Existing software score the severity of changes that arise in protein encoding exons.  However, most mutations within a family are situated in the 98% of the genome that controls the developmental and physiological profile of gene activity - the sequences that control when and where a gene will be active.  <br><br>Functional contributions of cis-regulatory sequence variations to human genetic disease are numerous. With full genome sequencing becoming accessible to medical researchers, the need to identify potential causal mutations in regulatory DNA is becoming imperative. We are implementing a software system to enable genetics researchers to characterize regulatory DNA changes within individual genome sequences. We are combining reference databases of known regulatory elements, experimental archives of protein-DNA interactions and computational predictions within an integrated analysis package. With our software, researchers will have greater capacity to identify variations potentially causal for disease.<br><br>The poster introduces the challenges and approaches of regulatory sequence variation analysis.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT12" >OPT12</a> <span style="float: right;">Sunday, July 17: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>SNPs&GO: predicting the deleterious effect of human mutations using functional annotation.</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Emidio Capriotti, Stanford University, United States<br><br><strong>Additional authors:</strong><br>Piero Fariselli, University of Bologna, it<br>Pier Luigi Martelli, University of Bologna, it<br>Rita Casadio, University of Bologna, it<br>

<br><strong>Abstract:</strong><br>High-throughput data from large-scale sequencing and genotyping techniques allow to analyze a huge amount of genetic variation from whole human genome. Single Nucleotide Polymorphisms (SNPs), which are the main cause of human genome variability, can also be involved in the insurgence of many diseases. In particular missense SNPs, occurring in coding regions and causing single amino acid polymorphisms (SAPs), can affect protein function and lead to genetic pathologies. <br>In this work, we present SNPs&GO (Calabrese at al., Human Mutation 2009), a new web server for the prediction of deleterious SAPs using protein functional annotation. We implemented two different SVM-based methods relying either on protein sequence or structure information. Both algorithms have been extensively tested on a large set of mutations extracted from SwissVar database (Mottaz et al., Bioinformatics 2010). Selecting a balanced dataset of SAPs, the sequence-based approach reaches 81% overall accuracy, 0.63 correlation coefficient and 0.89 area under the receiving operating characteristic curve (AUC). For the subset of mutations that can be mapped on protein structures known with atomic resolution (at the Protein Data Bank), the structure-based method results in 85% overall accuracy, a correlation coefficient of 0.70, and AUC of 0.92. In conclusion, SNPs&GO is a valuable tools that includes in unique framework information derived from protein sequence, structure, evolutionary profile, and protein function. In a recent study (Thusberg et al., Human Mutation 2011), SNPs&GO has been scored as one of the best algorithms for prediction of deleterious SAPs. <br>Avaliablity: http://snps.uib.es/snps-and-go<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT13" >OPT13</a> <span style="float: right;">Sunday, July 17: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Large-scale protein flexibility analysis of single nucleotide polymorphisms, using molecular dynamics simulations.</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Marc Offman, Technical University of Munich, Germany<br><br><strong>Additional authors:</strong><br>Burkhard Rost, Technical University of Munich, Germany<br>

<br><strong>Abstract:</strong><br>Proteins are intrinsically flexible molecules, thus function is often associated to flexibility. Experimental methods to determine protein flexibility are expensive and often time consuming. Over the past few years an efficient complementing method, molecular dynamics simulations, more and more proved to be a powerful tool to yield information on protein dynamics. We have recently proven that careful biology-driven MD simulations can be used to predict the impact of single amino acid mutations on protein flexibility and function, at a level of accuracy comparable to experimental techniques. The question remains whether it will be possible to fully automate this process in the context of a large-scale analysis, and to what extent additional structural information, beyond that derived by sequence analysis of single nucleotide polymorphisms (SNPs) only, is useful. For this we created several comprehensive datasets of non-synonymous SNPs mapped to high-resolution and above average quality crystal structures from the PDB. In the context of the European SCALALIFE project up to 28,000 different mutations found in 1,600 individual crystal structures are simulated in duplicates for 10 ns each, using the GROMACS package. A comprehensive analysis pipeline has been established, investigating protein flexibility and stability, alteration of hydrogen-bond networks, active site integrity, changes in global and local energy and other structural effects. This pipeline has previously been successfully applied in the context of clinically relevant proteins. The results of this study, the automatic protocol and set of analysis tools will help in the future to understand individual phenotypes in clinical contexts.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT14" >OPT14</a> <span style="float: right;">Sunday, July 17: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>A method to separate function and fold specific residues in a protein.</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Mohd Rehan, Jawaharlal Nehru University, India<br><br><strong>Additional authors:</strong><br>Andrew Lynn, Jawaharlal Nehru University, India<br>

<br><strong>Abstract:</strong><br>According to the neutral theory of molecular evolution, once a protein has evolved to a useful level of functionality, the majority of mutation are selectively neutral at the molecular level and do not affect the function and folding of the protein, whereas those mutations which are deleterious provide selection pressure for residue conservation. Thus, the conservation indicates the importance of residue for structure and function of the protein. Often, the mutations at a conservation site after the gene duplication leads to functional divergence. The residues of a protein at these sites called function specific residues when mutated changes the protein's function. For a class of protein, sequences are further grouped into subtypes contain subtype conserved residues indicative of functional differences among the subtypes. Here we present a method for finding function specific residues and fold specific residues in a protein family pre-classified into subtypes based on the function. Our method is based on RE (Relative Entropy) and a newly derived MI (Mutual Information) score. RE calculates the conservation over background distribution of amino acids whereas the derived MI tells about differential conservation among subtypes. This newly derived MI and the RE are combined in a way to give a new score, MIRE to rank the residues for fold and function. The methodology is implemented using HMM, validated on AGC-Protein Kinases and G-Protein Coupled receptors (GPCRs) and compared with other existing methods. The kinases have different peptide substrates, but  common protein kinase function whereas GPCRs have similar scaffold with different substrate binding sites.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT15" >OPT15</a> <span style="float: right;">Sunday, July 17: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Structural Biology Meets Systems Biology: A Structural Systems Biology Approach for Gauging the Systemic Effect of Single Nucleotide Polymorphisms</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Tammy Cheng, Cancer Research UK, United Kingdom<br><br><strong>Additional authors:</strong><br>Linda Jeffery, Cancer Research UK London Research Institute, United Kingdom<br>Lucas Goehring, Max Plank Institute, Germany<br>Yu-En Lu, University of Cambridge, United Kingdom<br>Jacqueline Hayles, Cancer Research UK London Research Institute, United Kingdom<br>Bela Novak, University of Oxford, United Kingdom<br>Paul Bates, Cancer Research UK London Research Institute, United Kingdom<br>

<br><strong>Abstract:</strong><br>Small changes in protein structure, such as non-synonymous single nucleotide polymorphisms, can have a large impact on cellular behaviour. To understand how a change at the protein structure level eventually affects a cell's phenotypic outcome is, however, not trivial. This is because complex, multi-scale, information needs to be considered in order to obtain analytical results with physiological meanings. To tackle this issue, we have developed a structural systems biology approach, PEPP (Phenotype Extrapolation via Pathway and Protein information), to effectively integrate protein structural information with pathway dynamics. <br><br>Here we demonstrate the new method by studying point mutations in two biological systems: (1) Regulation of the G2 to mitosis transition in Schizosaccharomyces pombe (fission yeast) and (2) the human ERK pathway. In the first system, we use mitosis-promoting factor as a reporter protein and studied 11 mutations that result in various extent of growth before yeast cells enter the mitosis stage. In the second system, we used ERK as a downstream reference to study 40 mutations that lead to phenotypically overlapping symptoms under the broad term 'neuro-cardio-facial-cutaneous syndrome'. By combining the information on both pathway and protein structure levels, we are able to gauge the phenotypic effect of the mutations – in the yeast model, PEPP reflects the general trend of cell sizes measured experimentally; in the human model, PEPP clusters the point mutations into subgroups that reflects their clinical symptoms.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/86/20194.pdf">Download Abstract</a><br> <br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT16" >OPT16</a> <span style="float: right;">Sunday, July 17: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Network-based gene prioritization from expression data by diffusing through protein interaction networks</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Daniela Nitsch, Belgium, KU Leuven<br><br><strong>Additional authors:</strong><br>Léon-Charles  Tranchevent, KU Leuven, Belgium<br>Joana Gonçalves, INESC-ID, Portugal<br>Yves Moreau, KU Leuven, Belgium<br>

<br><strong>Abstract:</strong><br>Discovering novel disease genes is challenging for diseases for which no prior knowledge is available. Performing genetic studies frequently result in large lists of candidate genes of which only few can be followed up for further investigation. In the past couple of years, several gene prioritization methods have been proposed. Most of them use a guilt- by - association concept, and are therefore not applicable when little is known about the phenotype or no disease genes are available.<br><br>We have proposed a method that overcomes this limitation by replacing prior knowledge about the biological process by experimental data on differential gene expression between affected and healthy individuals. At the core of the method are a protein interaction network and disease-specific expression data. Our approach propagates the expression data over the network using an extended Random Walk approach based on kernel methods, as the inclusion of indirect associations compensating for network sparsity and small world effect issues. It relies on the assumption that strong candidate genes tend to be surrounded by many differentially expressed neighboring genes in a protein interaction network.<br>We have benchmarked our approach, and results showed that it clearly outperforms other gene prioritization approaches with an average ranking position of 8 out of 100 genes, and an AUC value of 92.3%.<br><br>Recently, we have developed the web server PINTA implementing our gene prioritization approach to make it available for clinicians and other researchers.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT17" >OPT17</a> <span style="float: right;">Sunday, July 17: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Protein interaction partners revealed by their dynamical properties</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Jau-Ji Lin, National Chiao Tung University, Taiwan<br><br><strong>Additional authors:</strong><br>Jenn-Kang Hwang, National Chiao Tung University, Taiwan<br>

<br><strong>Abstract:</strong><br>The biological processes in cells are carried out by various kinds of protein molecules that perform their functions by interacting with other proteins. Identifying the interacting partners of proteins is crucial to the constructing of the biological pathways in a cell. Recently we developed a method called WCN (Lin et al., 2008) to calculate the protein dynamical properties from its folded structure directly and efficiently. In this work, we analyze the dynamical properties of interacting proteins. We find that these proteins display a “complementary” relationship in their profiles. Therefore we analyze the dynamical profiles of all protein structures in PDB and try to elucidate the relationships between them. The dynamical profiles reveal that proteins do have their specific “disposition” to interact with certain kinds of protein molecules. By inspecting the dynamical profiles between each pair of protein structures in PDB, we can construct a network with nodes and edges representing the proteins and the interactions between them. The resulting network is composed of several sub networks, and most of them agree with the available protein-protein interaction data. In addition, we use CELLO (Yu et al., 2006) to predict the subcellular localization of proteins being proposed as interacting partners and find most of them have the same subcellular localization. This finding confirms the space requirement of proteins to interact with each other. This study constitutes a novel methodology of constructing protein-protein interaction networks and provides a promising way to identify interacting partners of proteins of unknown function with structural information only.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT18" >OPT18</a> <span style="float: right;">Sunday, July 17: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Detection of nonlinear effects in gene expression pathways</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Andreas Mayr, Johannes Kepler University Linz, Austria<br><br><strong>Additional authors:</strong><br>Djork-Arné Clevert, Johannes Kepler University Linz, Austria<br>Sepp Hochreiter, Johannes Kepler University Linz, Austria<br>

<br><strong>Abstract:</strong><br>One of the main topics in systems biology is to model genetic pathways. Genes of a pathway, which show linear dependencies of their expression values, are easy to identify to belong to the pathway. However, if feedback loops or signal cascades are present, gene expression values of pathway genes can be nonlinearly dependent on the expression values of other genes in the pathway. In this situation such genes are hard to detect as belonging to the pathway because nonlinearity and noise must be distinguished.<br><br>We propose an algorithm to infer nonlinear network elements in pathways from microarray data. Our model assumes, that gene expression values, belonging to one pathway, are mainly driven by one single latent factor. We expect two groups of genes in a pathway: genes belonging to the first group are linearly dependent on the hidden factor, genes from the other group show a nonlinear dependence from the latent variable. The goal is to identify the kind of dependence from the hidden factor.<br><br>Our algorithm for detecting nonlinear effects is an extension of linear Gaussian factor analysis. Nonlinearities are modelled by the square of the latent variable weighted by specific coefficients. We derived a novel model selection method for this generalization of factor analysis. To avoid the interpretation of noise as nonlinearity, we determine p-values that measure the probability of a linear gene being detected by chance as nonlinear. <br><br>We apply our algorithm to microarray data of breast cancer samples, where we identified nonlinear dependencies of gene expression values in the p53  pathway.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT19" >OPT19</a> <span style="float: right;">Sunday, July 17: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Systematically mapping the druggable pathways of Saccharomyces cerevisiae</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Kristen  Fortney, University of Toronto, Canada<br><br><strong>Additional authors:</strong><br>Wing Xie, University of Toronto, Canada<br>Max Kotlyar, University of Toronto, Canada<br>Igor Jurisica, University of Toronto, Canada<br>

<br><strong>Abstract:</strong><br>Drug modes of action are complex and still poorly understood. The set of known drug targets is widely acknowledged to be biased and incomplete, and so gives only limited insight into the system-wide effects of drugs. But a high-throughput assay unique to yeast – barcode-based chemogenomic screens – can measure the individual drug response of every yeast deletion mutant in parallel. <br>We integrate the four largest S. cerevisiae chemogenomic experiments, which together comprise the responses of thousands of gene knockout strains to over 500 drug treatments, and develop a data-mining approach to investigate drug effects at the systems level. We apply our method to identify yeast pathways, functions, and phenotypes that are targeted by particular drugs. To demonstrate relevance to human disease, we collect groups of disease-associated human genes, map them to their homologs in yeast and apply our method; we recover drugs already prescribed for those diseases and propose several new drug candidates. We also develop methods for modeling the set of all significant pathway-drug connections as bipartite interaction networks. Our analyses of the structure of these networks reveal that while most pathways are targeted by few drugs, some are extremely druggable. Finally, we build YEDr, the YEast Drug database, a searchable interface to our data, methods, and results. Users can query YEDr with new gene groups (yeast, mouse, or human) and YEDr will retrieve the drugs that target them. Human targets are integrated with GeneCards, I2D, mirDIP and other resources.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT20" >OPT20</a> <span style="float: right;">Sunday, July 17: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Gene expression evolution on the emergence of pathogenicity in Ascomycetes</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Aminael Sanchez-Rodriguez, Catholic University of Leuven, Belgium<br><br><strong>Additional authors:</strong><br>Kristof Engelen, Catholic University of Leuven, Belgium<br>Riet De Smet, Ghent University, Belgium<br>Qiang Fu, Catholic University of Leuven, Belgium<br>Yan Wu, Catholic University of Leuven, Belgium<br>Kathleen Marchal, Catholic University of Leuven, Belgium<br>

<br><strong>Abstract:</strong><br>The Ascomycetes form the largest phylum in the fungal kingdom. They are of special interest due to their broad spectrum of life styles including both plant and human pathogens. In a previous study we showed that most of the protein-coding genes needed for pathogenicity were already present in an ancestor common to both pathogenic and non-pathogenic Ascomycetes. Based on this finding that at least some of the coding potential related to pathogenicity is common to both pathogens and non-pathogens, we studied whether alterations in expression behavior could be correlated with a pathogenic versus a non pathogenic lifestyle. Using coexpression networks derived from large scale expression compendia for the non-pathogen N. crassa and the pathogens M. grisea and F. graminearum, we compared the coexpression behavior of true orthologs from gene families related to pathogenicity and occurring in each of the species. We found that the direct neighbours, of the respective orthologs in the coexpression network of the pathogen was largely different from its neighbours in the non-pathogenic species, implying a considerable rewiring of the coexpression network of these common orthologs. In addition we also observed that the expression evolution of paralogs in gene families that are common to both pathogens and non pathogens is different. More specifically we detected that the expression behavior of paralogs belonging to gene families that are involved in pathogen-host interaction tend evolve faster in pathogens than in non-pathogens.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT21" >OPT21</a> <span style="float: right;">Sunday, July 17: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>The Darwinian Tree Of Life In Light Of Horizontal Gene Transfer (Is Still Sound)</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Adam Sardar, University of Bristol, United Kingdom<br>

<br><strong>Abstract:</strong><br>Horizontal gene transfer (HGT) is the process whereby nucleic acids are transmitted from the genome of one species to another without inheritance by descent (i.e parent to child). Some people believe that there is no prokaryotic tree of life, but in fact a ‘thicket’ of shared genetic material. It is therefore extremely important to understand HGT, as it is so divergent from assumptions made in some phylogenetic analysis , such as a direct ancestry of homologous proteins. Previous studies have focused on trying to predict precisely which genes have been horizontally transferred and where, based on multiple sequence alignments and searches for a ’footprint’ or biochemical signal peculiar to the parent genome. This is very difficult and the signal is quickly lost with evolutionary time. We propose avoiding to attempt prediction of every horizontal transfer event, but to take a global view. We look for proteins comprised of sets of structural-domains (domain architectures, as identified in the SUPERFAMILY database) that show disparity in their deletion rates across the tree of life, requiring an overwhelmingly improbable realisation of events in order for their current distribution in observed genomes. This does not provide concrete identification of individual HGT events, but creates an enriched set of possible contenders, acting as an indicator towards HGT across all all organisms.<br><br>Our research suggest an upper bound of 30% for the percentage of domain architectures in bacteria involved in HGT, suggesting that bacteria do indeed posses an underlying Darwinian tree.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT22" >OPT22</a> <span style="float: right;">Sunday, July 17: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Quality or Quantity of Structural Dynamics Information: That is the Question! (when improving performance of structure-based function prediction methods)</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Dariya Glazer, Stanford University, United States<br><br><strong>Additional authors:</strong><br>Grace  Tang, Stanford University, United States<br>Vijay  Pande, Stanford University, United States<br>Russ Altman, Stanford University, United States<br>

<br><strong>Abstract:</strong><br>Previously, we showed that incorporating structural dynamics information from molecular simulations improved performance of structure-based function prediction methods. This raised new intriguing questions. Does the choice of force field, the length of the simulation trajectories, or the presence of a ligand in the simulation systems have any effect? Are there any meaningful descriptors of the true positive results that can help discriminate those from the false positive ones?<br><br>To this end, we scaled up our efforts in molecular dynamics simulations with Folding@Home distributed computing. Using 5 force fields (Amber ’96, ‘99sb, '03, Gromos53a6, and OPLS-AA) we generated 14 multiples of 10 nanosecond trajectories (a total of 140 nanoseconds) with and without the presence of calcium ions for 11 pairs of structures, HOLO/APO for calcium binding. Resulting structural ensembles were evaluated for calcium binding sites using FEATURE.  <br><br>Our results indicate that different force fields explore somewhat different conformational space of the molecules with respect to the sensitivities of FEATURE. Longer simulation trajectories and inclusion of calcium ions in the simulation systems did not yield a greater improvement in FEATURE’s performance. We propose 6 descriptors of sites identified by FEATURE that can be intelligently combined by machine learning algorithms in order to maximize true positive and minimize false positive results. <br><br>Future efforts in structure-based function prediction, especially identifying calcium binding sites, could benefit from applying function prediction methods to structural ensembles of the molecules of interest obtained from several copies of short-scale molecular dynamics simulations employing at least two general force fields.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT23
" >OPT23
</a> <span style="float: right;">Sunday, July 17: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Nearest-Neighbor Approaches to Predict Protein Function by Sequence Homology Alone</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Tobias Hamp, TU Muenchen, Germany<br><br><strong>Additional authors:</strong><br>Rebecca Kassner, TU Muenchen, Germany<br>Stefan Seemayer, TU Muenchen, Germany<br>Esmeralda Vicedo, TU Muenchen, Germany<br>Christian Schaefer, TU Muenchen, Germany<br>Dominik Achten, TU Muenchen, Germany<br>Florian Auer, TU Muenchen, Germany<br>Ariane Boehm, TU Muenchen, Germany<br>Tatjana Braun, TU Muenchen, Germany<br>Maximilian Hecht, TU Muenchen, Germany<br>Mark Heron, TU Muenchen, Germany<br>Peter Hoenigschmid, TU Muenchen, Germany<br>Thomas Hopf, TU Muenchen, Germany<br>Stefanie Kaufmann, TU Muenchen, Germany<br>Michael Kiening, TU Muenchen, Germany<br>Denis Krompass, TU Muenchen, Germany<br>Cedric Landerer, TU Muenchen, Germany<br>Yannick Mahlich, TU Muenchen, Germany<br>Manfred Roos, TU Muenchen, Germany<br>Burkhard Rost, TU Muenchen, Germany<br>

<br><strong>Abstract:</strong><br>This year's Automated Function Prediction SIG meeting at ISMB/ECCB 2011 features, for the first time, the so-called CAFA challenge. <br>Participants were supposed to computationally predict and submit the Gene Ontology (GO) terms for over 40,000 so far functionally unannotated targets.<br>In this context, we have developed three different Nearest-Neighbor-based methods to derive the functions of a protein by sequence homology alone. Despite being conceptually the same, they largely differ in their technical details, ranging from an emphasis on superior ad-hoc performance to sophisticated and well-defined scoring schemes, allowing to direct either recall or precision into the desired reliability range. <br>All three methods were employed to predict all the CAFA targets mentioned before and are, in a way, supposed to act as controls for an expected wealth of more sophisticated function prediction machineries at the CAFA meeting.<br>In addition, we have created a meta-classifier to combine the three approaches and established new ways to generally capture the performance of function prediction tools with GO terms. The latter now enable us to not only estimate precision and recall as described in the official CAFA rules, but also to use the shapes of the GO subgraphs of a target protein to create an assessment that considers its biologically distinct functions.<br>An own pre-CAFA evaluation of our methods with 10,000 random targets from the SwissProt database already showed remarkable performances of up to 87% precision with 84% recall using the official standard protein centric measure and a greatly outperformed class of random prediction models.<br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class=subheading2>

<a name="OPT24
" >OPT24
</a> <span style="float: right;">Sunday, July 17: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Graph-based queries of Semantic-Web integrated biological data</em></strong><br><strong>Room: </strong>Hall E1<br /><strong>Presenting author: </strong>Marco Moretto, Edmund Mach Foundation, Italy<br><br><strong>Additional authors:</strong><br>Alessandro Cestaro, Edmund Mach Foundation, it<br>Riccardo Velasco, Edmund Mach Foundation, it<br>

<br><strong>Abstract:</strong><br>In the post-genomic era, life science researchers are faced with the need to manage and inspect a growing abundance of data and information. Data from different sources, both public and proprietary, have the most value when considered in the context of each other as they give more information. In order to answer questions that spans multiple fields in the biology domain without an integrated approach, a biologist needs to visit all data sources related to the problem and ?nd relevant data. In the last years we have become witnesses of a growing interest for the Semantic Web technologies to integrate and query biological data. Semantic Web technologies were designed to meet the challenges of reduce the complexity of combining data from multiple sources, resolve the lack of widely accepted standards and manage highly distributed and mutable resources. However, Semantic Web standard technologies do not provide any tools to query integrated knowledge bases from a graph perspective, that is defining graph traversal patterns. For example, it is not possible to ask a query like "are enzyme A and compound B related?" without knowing the complete structure of the knowledge base. After exploring different alternatives we come up with the use of a graph traversal programming language on top of a triplestore in order to perform several path traversal queries on an integrated graph. We tested the feasibility of the approach integrating Uniprot, Gene Ontology, Chebi and Kegg resources posing queries of different complexity.<br><a href="#up" style="font-size:9px;">TOP</a><br><br></div></td> 
</tr> 
</table> 
<span class="article_separator">&nbsp;</span> 
                                                      	</div> 
						</div> 
						<div class="clr"></div> 
					</div> 
					<div class="clr"></div> 
				</div> 
 
				<div id="whitebox_b"> 
					<div id="whitebox_bl"> 
						<div id="whitebox_br"></div> 
					</div> 
				</div> 
			</div> 
 
			<div id="footerspacer"></div> 
		</div> 
 
		
	</div> 
</div> 
 
 
</body> 

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/oralposters.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:49:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->

<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/oralposters.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:31 GMT -->
</html> 
