<h3>Agreement of PLoS Postcards Allowance </h3>PLoS Postcards-ISCB is partnering with PLoS Computational Biology to have all presentations at ISMB/ECCB 2011 potentially reported on by conference student and jr. scientist attendees and published in PLoS Computational Biology as part of an article summarizing the conference. This program is referred to as PLoS Postcards. Your permission is not a guarantee that the material will be published in PLoS Computational Biology.<br><br><!-- <strong>Total: </strong>491<br> -->The following submitters have agreed<br><br><h3>Posters</h3><table border=1><tr><td>20098</td><td></td><td>A02</td><td>Bayesian Networks to Decision Making in Lymph Node Metastasis of Colorectal Cancer</td>
	<td>Md. Aminul Hoque </td></tr><tr><td>22172</td><td></td><td>A04</td><td>A large scale analysis in the human proteome detects correlation among disease associated mutations and perturbation of protein stability</td>
	<td>Rita Casadio </td></tr><tr><td>22173</td><td></td><td>A05</td><td>SNPs&GO: predicting the deleterious effect of human mutations using functional annotation.</td>
	<td>Emidio Capriotti </td></tr><tr><td>20308</td><td></td><td>A07</td><td>Uncovering Network Control Mechanisms in Induced Oncogenic Transformation of Human Mammary Epithelial Cells through Whole Transcriptome Analyses</td>
	<td>Sol Efroni </td></tr><tr><td>20322</td><td></td><td>A09</td><td>siRna – A Comprehensive Analysis Workflow Towards Better Understanding of High-throughput Screening Data.</td>
	<td>Vidal Fey </td></tr><tr><td>20323</td><td></td><td>A10</td><td>miRNA levels are associated with immune network activity and associate with disease outcome in Ovarian Cancer. A possible miRNA – network control mechanism.</td>
	<td>Sol Efroni </td></tr><tr><td>20339</td><td></td><td>A11</td><td>A Novel Bioinformatics Pipeline for Identification and Characterization of Fusion Transcripts in 31 Breast Cancer and Normal Cell Lines</td>
	<td>Yan Asmann </td></tr><tr><td>20353</td><td></td><td>A12</td><td>Feature extraction from Spinal cord images</td>
	<td>Sejong Oh </td></tr><tr><td>20375</td><td></td><td>A14</td><td>Cross-species candidate gene prioritization with MerKator</td>
	<td>Leon-Charles Tranchevent </td></tr><tr><td>20377</td><td></td><td>A15</td><td>A guide to web tools to prioritize candidate genes</td>
	<td>Leon-Charles Tranchevent </td></tr><tr><td>20379</td><td></td><td>A16</td><td>History-aware models for HIV therapy screening</td>
	<td>Jasmina Bogojeska </td></tr><tr><td>20301</td><td></td><td>A17</td><td>Network-based gene prioritization from expression data by diffusing through protein interaction networks</td>
	<td>Daniela Nitsch </td></tr><tr><td>20391</td><td></td><td>A18</td><td>Detecting cancer driving mutations by whole genome sequencing in the ICGC PedBrain project</td>
	<td>Barbara Hutter </td></tr><tr><td>20168</td><td></td><td>A20</td><td>Mapping the spread of HIV in Germany</td>
	<td>Glenn Lawyer </td></tr><tr><td>20406</td><td></td><td>A21</td><td>SNP4Disease: a knowledge database to detect disease associated SNPs</td>
	<td>David John </td></tr><tr><td>20447</td><td></td><td>A25</td><td>Transcriptomic profiles and gene networks of human Mesenchymal Stem Cells (MSCs)</td>
	<td>Beatriz Rosón </td></tr><tr><td>20481</td><td></td><td>A27</td><td>Association Rule Mining with Prior Knowledge for Alzheimer's Disease</td>
	<td>Peter Li </td></tr><tr><td>20486</td><td></td><td>A29</td><td>An Integrative Bioinformatic Predictor of Protein Sub-Cellular Localisation in Malaria</td>
	<td>Ben Woodcroft </td></tr><tr><td>20530</td><td></td><td>A32</td><td>Deconvoluting Cancer Genomes using an Optimal Scaffolder</td>
	<td>Niranjan Nagarajan </td></tr><tr><td>20555</td><td></td><td>A34</td><td>Chipster 2.0: User-friendly analysis software for next generation sequencing data with interactive genome viewer.</td>
	<td>Massimiliano Gentile </td></tr><tr><td>20567</td><td></td><td>A37</td><td>AllergenNet: a database applied to the study of allergenic proteins</td>
	<td>Helen Arcuri </td></tr><tr><td>20570</td><td></td><td>A38</td><td>Integrative analysis of expression and copy number profiles to find causal genes in cancer</td>
	<td>Celia Fontanillo </td></tr><tr><td>20595</td><td></td><td>A39</td><td>GUILDify: A web server for phenotypic characterization of genes by means of protein-protein interactions and graph algorithms</td>
	<td>Emre Guney </td></tr><tr><td>20623</td><td></td><td>A41</td><td>Bioinformatics for large scale characterization and tracking of gene-modified hematopoietic stem cells</td>
	<td>Manfred Schmidt </td></tr><tr><td>20627</td><td></td><td>A42</td><td>Detecting Intrahost Emergence of Recombinant Viruses within HIV-1 infections using Ultra-Deep Sequencing</td>
	<td>Felix Feyertag </td></tr><tr><td>20656</td><td></td><td>A46</td><td>Structural comparison of Erwinase and E. coli L-Asparaginase to facilitate rational engineering of a cancer drug</td>
	<td>Mainá Bitar </td></tr><tr><td>20659</td><td></td><td>A47</td><td>Inhaled Toxicant Interactions with Transcriptional Regulatory Networks in Frontline Lung Cells</td>
	<td>George Acquaah-Mensah </td></tr><tr><td>20671</td><td></td><td>A48</td><td>Multi-Harmony: detecting functional specificity in HIV differential replication rates</td>
	<td>K. Anton Feenstra </td></tr><tr><td>20670</td><td></td><td>A49</td><td>Data Based Identification of Estimation Models for Tumor Markers Using Evolutionary Computation</td>
	<td>Stephan Winkler </td></tr><tr><td>20698</td><td></td><td>A50</td><td>Models for reverse engineering of transcriptional networks in Yersinia enterocolitica infection</td>
	<td>Olivia Prazeres da Costa </td></tr><tr><td>20704</td><td></td><td>A51</td><td>A framework for whole-exome deep sequening data management and analysis</td>
	<td>Angelo Nuzzo </td></tr><tr><td>20706</td><td></td><td>A52</td><td>Genome-Wide Association Study of TB in the South African Colored Population: Comprehensive Gene and pathway-based Association Study</td>
	<td>Emile RUGAMIKA CHIMUSA </td></tr><tr><td>20468</td><td></td><td>A54</td><td>Hunting cancer predisposition genes by whole exome sequencing in relatives from affected family</td>
	<td>Tatiana Popova </td></tr><tr><td>20738</td><td></td><td>A55</td><td>deepBlockAlign: aligning profiles from short blocks of reads</td>
	<td>Jan Gorodkin </td></tr><tr><td>20742</td><td></td><td>A56</td><td>Integrating network co-expression analysis and gene expression in the analysis of coronary artery disease GWAS: application to CARDIoGRAM data</td>
	<td>Seraya Maouche </td></tr><tr><td>20711</td><td></td><td>A57</td><td>A Novel Algorithm for Identifying Non-canonical Splicing Regions Using RNA-Seq Data</td>
	<td>Yongsheng Bai </td></tr><tr><td>20789</td><td></td><td>A59</td><td>Personalized Medicine for Cancer</td>
	<td>Miguel García </td></tr><tr><td>20817</td><td></td><td>A62</td><td>Bioinformatics and Biodiversity:  A holistic approach to enhancing drug discovery.</td>
	<td>Negusse Kitaba </td></tr><tr><td>20824</td><td></td><td>A63</td><td>Finding locally disrupted RNA structure from SNPs</td>
	<td>Jan Gorodkin </td></tr><tr><td>20827</td><td></td><td>A64</td><td>Heme Detoxification Protein: A promising antimalarial drug target.</td>
	<td>Dedan Githae </td></tr><tr><td>20829</td><td></td><td>A65</td><td>Network Analysis and validation of gene expression profiling of salivary gland from human and mice with primary Sjögren’s syndrome</td>
	<td>Hui Zhou </td></tr><tr><td>20850</td><td></td><td>A67</td><td>Candidate mutations for early-onset lung cancer by family genome sequencing</td>
	<td>Evangelos Simeonidis </td></tr><tr><td>22182</td><td></td><td>A68</td><td>Improving the detection of deleterious mutations integrating the predictions four well-tested methods</td>
	<td>Emidio Capriotti </td></tr><tr><td>20874</td><td></td><td>A72</td><td>Discovery of novel cancer pathway-activating genomic aberrations via a mutual information metric</td>
	<td>Olga Botvinnik </td></tr><tr><td>20929</td><td></td><td>A74</td><td>Prediction of miRNA gene targets – a combined computational and experimental approach</td>
	<td>Anastasis Oulas </td></tr><tr><td>22642</td><td></td><td>A77</td><td>Systematic Analysis of MicroRNAs Targeting the Androgen Receptor in Prostate Cancer Cells.</td>
	<td>Pekka Kohonen </td></tr><tr><td>22696</td><td></td><td>A80</td><td>Robust and accurate data enrichment statistics via distribution function of sum of weights</td>
	<td>Yi-Kuo Yu </td></tr><tr><td>22709</td><td></td><td>A81</td><td>Measuring the complexity of solving diseases</td>
	<td>Raul Rodriguez-Esteban </td></tr><tr><td>22677</td><td></td><td>A82</td><td>Nonparametric Estimation of an Unknown Probability Distribution Using Maximum Likelihood and Bayesian Approaches</td>
	<td>Alona Chubatiuk </td></tr><tr><td>22713</td><td></td><td>A83</td><td>Exploratory analysis of the structure and dynamics of healthy intestinal microbiota</td>
	<td>Leo Lahti </td></tr><tr><td>22720</td><td></td><td>A85</td><td>Identification of secondary targets for clinically used anti-tubercular drugs through binding site similarities</td>
	<td>Nagasuma Chandra </td></tr><tr><td>22727</td><td></td><td>A86</td><td>Re-visiting classification of HLA class-I molecules</td>
	<td>Nagasuma Chandra </td></tr><tr><td>22731</td><td></td><td>A87</td><td>A molecular based subtyping in juvenile idiopathic arthritis</td>
	<td>Margherita Squillario </td></tr><tr><td>20448</td><td></td><td>B2</td><td>Free energy and flexibility basis of SNP occurrence</td>
	<td>Luciana Oliveira </td></tr><tr><td>20465</td><td></td><td>B3</td><td>Pipeline for the training of NMR chemical shift prediction models</td>
	<td>Anna Katharina Dehof </td></tr><tr><td>20459</td><td></td><td>B4</td><td>Modeling of Actomyosin Interactions</td>
	<td>Cynthia Prudence </td></tr><tr><td>20798</td><td></td><td>B5</td><td>Fitting Comparative Models to X-ray Data with Kino-Geometric Sampling</td>
	<td>Henry van den Bedem </td></tr><tr><td>22690</td><td></td><td>B6</td><td>Insights to the Thrombospondin Signature Domain</td>
	<td>Thomas Haschka </td></tr><tr><td>22679</td><td></td><td>B7</td><td>Universal patterns of protein-ligand interactions: Gaussian Mixture Model analyses on three dimensional structure of proteins in Protein Data Bank.</td>
	<td>Kota Kasahara </td></tr><tr><td>22712</td><td></td><td>B9</td><td>Intrinsically disordered proteins/regions: an important factor in protein evolvability</td>
	<td>Akinori Sarai </td></tr><tr><td>20229</td><td></td><td>C01</td><td>Mining the FDA’s Adverse Event Reporting System (AERS) for Pathway Risk</td>
	<td>Mark Gosink </td></tr><tr><td>20354</td><td></td><td>C02</td><td>The use of virtual screening reveals a viral-like polymerase inhibitor, which complex with the M. perniciosa DNA polymerase</td>
	<td>Bruno Andrade </td></tr><tr><td>20382</td><td></td><td>C03</td><td>Consensus docking method</td>
	<td>Dariusz Plewczynski </td></tr><tr><td>20394</td><td></td><td>C04</td><td>Homology modeling and molecular docking studies of Mannosyl Transferase (pimB) in Mycobacterium tuberculosis</td>
	<td>Karthik Maddi </td></tr><tr><td>20743</td><td></td><td>C06</td><td>Homology Modeling and Molecular Dynamics Simulation of N-acetylglucosamine fosfomutase (AGM1) from Moniliphothora perniciosa</td>
	<td>Wagner Soares </td></tr><tr><td>20835</td><td></td><td>C07</td><td>EC-BLAST: A novel tool for searching similar enzymes based on chemical knowledge</td>
	<td>Syed Asad Rahman </td></tr><tr><td>20864</td><td></td><td>C09</td><td>A systems approach to characterization of molecular targets of imaging agents</td>
	<td>Ting Liu </td></tr><tr><td>20893</td><td></td><td>C10</td><td>Stereological and hormonal evidences of epithelial hyperplasia with improved fertility capacity induced by aqueous sesame leaves Intake in adult male sprague dawley rats accessory organs</td>
	<td>LUKEMAN ADELAJA JOSEPH shittu </td></tr><tr><td>20267</td><td></td><td>D01</td><td>Characterization of a mannose-binding Insecticidal lectin gene from Allium sativum (Garlic)</td>
	<td>Nusrah Afolabi-Balogun </td></tr><tr><td>20389</td><td></td><td>D04</td><td>Tools for comparative metagenome analysis</td>
	<td>Kathrin Aßhauer </td></tr><tr><td>20485</td><td></td><td>D05</td><td>Identification of transcription factors and their correlation with the high diversity of Stramenopiles.</td>
	<td>Francisco Buitrago </td></tr><tr><td>20644</td><td></td><td>D09</td><td>A new Perl toolkit for the detection and analysis of polymorphic loci and their application for bacterial typing</td>
	<td>Ralf KOEBNIK </td></tr><tr><td>20705</td><td></td><td>D11</td><td>GECKO2: A software tool for de novo gene cluster detection</td>
	<td>Katharina Jahn </td></tr><tr><td>20739</td><td></td><td>D12</td><td>Automated Segmentation of DNA Sequences</td>
	<td>Brona Brejova </td></tr><tr><td>20772</td><td></td><td>D13</td><td>Antimicrobial Resistance and susceptility profiling of four pathogenic organisms using UV irradiation.</td>
	<td>Olugbenga Taiwo </td></tr><tr><td>20790</td><td></td><td>D16</td><td>Protein sequences of the human genome compared to those of several model organisms</td>
	<td>Edda Kloppmann </td></tr><tr><td>22649</td><td></td><td>D19</td><td>QuartetS: a fast and accurate program for large-scale orthology detection</td>
	<td>Jaques Reifman </td></tr><tr><td>22681</td><td></td><td>D20</td><td>Computational analysis of metagenomes and metatranscriptomes: challenges and opportunities.</td>
	<td>Larisa Kiseleva </td></tr><tr><td>22719</td><td></td><td>D22</td><td>Species-specific open chromatin has evidence of branch-specific selection.</td>
	<td>Nathan Sheffield </td></tr><tr><td>22718</td><td></td><td>D23</td><td>Comparative Genomics for expression QTL</td>
	<td>Steffen Möller </td></tr><tr><td>19914</td><td></td><td>E01</td><td>Seqcrawler  – A cloud ready indexing platform</td>
	<td>Olivier Sallou </td></tr><tr><td>19916</td><td></td><td>E02</td><td>Creating a Concept Map Interface to Visualize and Interpret a Comparative Effectiveness Data Model</td>
	<td>David Fenstermacher </td></tr><tr><td>22615</td><td></td><td>E03</td><td>SolDB: A Database of Solanum lycopersicum and Solanum tuberosum Primers</td>
	<td>Hassan Tariq </td></tr><tr><td>22627</td><td></td><td>E05</td><td>miRGator v2.0 and the construction of miRNA-disease network</td>
	<td>Wankyu Kim </td></tr><tr><td>20209</td><td></td><td>E08</td><td>GWAS Central: an advanced database for the integration and comparative interrogation of genome-wide association study datasets</td>
	<td>Sirisha Gollapudi </td></tr><tr><td>20220</td><td></td><td>E09</td><td>ISA tools: standard-compliant reporting and curation at the community level</td>
	<td>Susanna-Assunta Sansone </td></tr><tr><td>20257</td><td></td><td>E10</td><td>DARNED (Database of RNA editing).</td>
	<td>Pavel Baranov </td></tr><tr><td>20264</td><td></td><td>E11</td><td>Graph-based queries of Semantic-Web integrated biological data</td>
	<td>Marco Moretto </td></tr><tr><td>20283</td><td></td><td>E12</td><td>TMPad: an integrated structural database for helix-packing folds in transmembrane proteins</td>
	<td>Cheng-Wei Cheng </td></tr><tr><td>20293</td><td></td><td>E13</td><td>EBI Search: a new way to explore biology</td>
	<td>Andrew Jenkinson </td></tr><tr><td>20355</td><td></td><td>E17</td><td>A web-based tool for functional investigation of microRNAs : miRGator v2.0</td>
	<td>Wankyu Kim </td></tr><tr><td>20439</td><td></td><td>E20</td><td>Sierra: A simple LIMS system for small sequencing facilities</td>
	<td>Simon Andrews </td></tr><tr><td>20457</td><td></td><td>E21</td><td>Soybean Knowledge Base (SoyKB) : A Web Resource for Soybean Translational Genomics</td>
	<td>Trupti Joshi </td></tr><tr><td>20546</td><td></td><td>E23</td><td>easyDAS: Automatic creation of DAS servers</td>
	<td>Bernat Gel Moreno </td></tr><tr><td>20554</td><td></td><td>E24</td><td>Wheat international sequence repository integrated with genomic and genetic data in GnpIS</td>
	<td>Michael Alaux </td></tr><tr><td>20598</td><td></td><td>E26</td><td>Virtual Fly Brain 3D Interaction Tool</td>
	<td>Nestor Milyaev </td></tr><tr><td>20611</td><td></td><td>E28</td><td>GenomeRNAi: A Phenotype Database for Large-scale RNAi Screens</td>
	<td>Esther Schmidt </td></tr><tr><td>20638</td><td></td><td>E29</td><td>PROSITE: Enhancing the functional characterization of proteins</td>
	<td>Nicolas Hulo </td></tr><tr><td>20643</td><td></td><td>E30</td><td>PHI-base 4.0: A Second Generation Pathogen-Host Interaction database</td>
	<td>Jacek Grzebyta </td></tr><tr><td>20690</td><td></td><td>E32</td><td>Recent developments to the Ensembl website</td>
	<td>Stephen Trevanion </td></tr><tr><td>20701</td><td></td><td>E33</td><td>The Bioinformatics Links Directory: A Community Curated Collection of Bioinformatics Links, Tools and Databases</td>
	<td>Michelle Brazas </td></tr><tr><td>20722</td><td></td><td>E34</td><td>Structural Biology Knowledgebase: An Online Portal to Proteins, Functions, Methods, and More</td>
	<td>Margaret Gabanyi </td></tr><tr><td>20347</td><td></td><td>E35</td><td>Development of a Small Portable Application for Plant DNA Barcoding</td>
	<td>M Mehedi Hassan </td></tr><tr><td>20773</td><td></td><td>E36</td><td>MobiDB: a database of intrinsically disordered proteins</td>
	<td>Tomas Di Domenico </td></tr><tr><td>20788</td><td></td><td>E39</td><td>Community Cyberinfrastructure for Advanced Microbial Ecology Research and Analysis - The CAMERA  Resource</td>
	<td>Shulei Sun </td></tr><tr><td>22653</td><td></td><td>E40</td><td>Embryonic Stem Cell Database</td>
	<td>Elena Nikolaeva </td></tr><tr><td>22628</td><td></td><td>E41</td><td>RiceXPro and Rice TOGO Browser: Integrated Databases for Rice Functional and Applied Genomics</td>
	<td>Hajime Ohyanagi </td></tr><tr><td>22699</td><td></td><td>E42</td><td>SABIO-RK: Improvement of search capacities and user-friendliness</td>
	<td>Renate Kania </td></tr><tr><td>22764</td><td></td><td>E45</td><td>Correlation between annotation in the human gene database H-InvDB</td>
	<td>Katsuhiko Murakami </td></tr><tr><td>22766</td><td></td><td>E46</td><td>A Ruby API to query the Ensembl database for genomic features</td>
	<td>Jan Aerts </td></tr><tr><td>20331</td><td></td><td>E47</td><td>Identifying knowledge contributors – motivating online sharing of research data</td>
	<td>Gudmundur Thorisson </td></tr><tr><td>22620</td><td></td><td>F01</td><td>Multi-species integrative biclustering</td>
	<td>Richard Bonneau </td></tr><tr><td>20273</td><td></td><td>F04</td><td>The Operonic Location of Auto-Transcriptional Repressors is Highly Conserved in Bacteria</td>
	<td>Tal Pupko </td></tr><tr><td>20407</td><td></td><td>F07</td><td>Phylogenetic analysis of mapped sequence reads</td>
	<td>Jean-François Dufayard </td></tr><tr><td>20418</td><td></td><td>F08</td><td>Transcriptomic and metabolomic investigation of thermal stress response through metabolic modulations in seagrasses</td>
	<td>Jenny Gu </td></tr><tr><td>20421</td><td></td><td>F09</td><td>Positive Feedback and Tradeoff as Sources for Complexity in Social Interaction System</td>
	<td>Tomoyuki Yamamoto </td></tr><tr><td>20538</td><td></td><td>F12</td><td>Distribution patterns of genomic elements in bacteria using genome linguistic methods.</td>
	<td>Oliver Bezuidt </td></tr><tr><td>20668</td><td></td><td>F13</td><td>Gene expression evolution on the emergence of pathogenicity in Ascomycetes</td>
	<td>Aminael Sanchez-Rodriguez </td></tr><tr><td>20630</td><td></td><td>F14</td><td>Incorporating protein structure into coevolution analysis reveals functional side chain interactions in ABC-C exporters</td>
	<td>Attila Gulyas-Kovacs </td></tr><tr><td>22689</td><td></td><td>F17</td><td>A Phylogenetic Approach to Germline Methylome Reconstruction</td>
	<td>Lars Feuerbach </td></tr><tr><td>22668</td><td></td><td>F18</td><td>Probabilistic reconstruction of protein interaction histories using a simple measure of interaction affinity.</td>
	<td>Ryan Topping </td></tr><tr><td>22672</td><td></td><td>F20</td><td>TimeTree2: Species divergence times from molecular data</td>
	<td>Sudhir Kumar </td></tr><tr><td>20648</td><td></td><td>F21</td><td>Identifying Patterns of Balancing Selection by Comparing the Bonobo, Chimpanzee and Human Genomes</td>
	<td>Kay Prüfer </td></tr><tr><td>20691</td><td></td><td>F22</td><td>The Darwinian Tree Of Life In Light Of Horizontal Gene Transfer (Is Still Sound)</td>
	<td>Adam Sardar </td></tr><tr><td>22171</td><td></td><td>G01</td><td>Large-Scale Computational Identification of Regulatory SNPs</td>
	<td>Alberto Riva </td></tr><tr><td>20188</td><td></td><td>G03</td><td>Annotating New Genes: From In Silico Screening to Validations by Experiments</td>
	<td>Shizuka Uchida </td></tr><tr><td>20349</td><td></td><td>G06</td><td>Mining of Transcriptome of a climacteric fruit species Diospyros kaki Thunb for functional genomics</td>
	<td>Farzana Rahman </td></tr><tr><td>20451</td><td></td><td>G10</td><td>Human gene coexpression landscape and network: location of house-keeping and tissue-specific genes</td>
	<td>Alberto Risueño </td></tr><tr><td>20526</td><td></td><td>G12</td><td>Genomes and Evolutionary Analysese of Robust Oil Production in Microalgae</td>
	<td>Kang Ning </td></tr><tr><td>20577</td><td></td><td>G14</td><td>Analysis of protein functional sites encoding features</td>
	<td>Irina Medvedeva </td></tr><tr><td>20660</td><td></td><td>G16</td><td>An open source library for miRNA target prediction with a novel comparative genomics approach</td>
	<td>Charles Vejnar </td></tr><tr><td>20674</td><td></td><td>G17</td><td>From data to disease: Identification of genes with cell lineage specific expression in the human kidney</td>
	<td>Casey Greene </td></tr><tr><td>20759</td><td></td><td>G22</td><td>Estimating differential expression of transcripts with RNA-seq by using Bayesian Inference</td>
	<td>Peter Glaus </td></tr><tr><td>20785</td><td></td><td>G23</td><td>ARH-Seq: Alternative Splicing Robust Prediction by Entropy on RNA-Seq data</td>
	<td>Axel Rasche </td></tr><tr><td>22726</td><td></td><td>G30</td><td>The Limpopo MAGE-TAB Parser</td>
	<td>Tony Burdett </td></tr><tr><td>22728</td><td></td><td>G31</td><td>The Functional Genomics Software Suite</td>
	<td>Tomasz Adamusiak </td></tr><tr><td>20834</td><td></td><td>G32</td><td>Novel Method for Removal of Sequence Biases in ChIp-seq Data</td>
	<td>Joanna Raczynska </td></tr><tr><td>20472</td><td></td><td>H2</td><td>Improving the Accuracy of Automated Prokaryotic Gene Calling wih the Intergenic Region Scanner</td>
	<td>Heather Kent </td></tr><tr><td>20484</td><td></td><td>H3</td><td>Computing the probability of RNA hairpin and multiloop formation</td>
	<td>Peter Clote </td></tr><tr><td>20522</td><td></td><td>H4</td><td>A fast stochastic score based rank fusion method: application on miRNA target prediction</td>
	<td>Debarka Sengupta </td></tr><tr><td>20189</td><td></td><td>I01</td><td>Integrated prediction of protein functions in Arabidopsis thaliana by Bayesian Markov Random Fields</td>
	<td>Yiannis Kourmpetis </td></tr><tr><td>20346</td><td></td><td>I06</td><td>Nonparametric Approach to Analysis of Data-Rich Biological Problems</td>
	<td>Tatiana TATARINOVA </td></tr><tr><td>20435</td><td></td><td>I08</td><td>Genome Scale Annotation Using Rosetta De Novo Structure Predictions</td>
	<td>Kevin Drew </td></tr><tr><td>20548</td><td></td><td>I09</td><td>From next generation sequencing to knowledge: A case study of functional genomics annotation in prokaryotes</td>
	<td>Carlos Prieto </td></tr><tr><td>20562</td><td></td><td>I10</td><td>Human annotation in Ensembl</td>
	<td>Amonida Zadissa </td></tr><tr><td>20413</td><td></td><td>I11</td><td>Collaborative action between Spain and Austria to enhance the application of High Performance Computing to High throughput technologies</td>
	<td>Noura Chelbat </td></tr><tr><td>20634</td><td></td><td>I12</td><td>CHIPEAN – CHIP Seq Peak Annotation</td>
	<td>Agnes Hotz-Wagenblatt </td></tr><tr><td>20647</td><td></td><td>I13</td><td>Comprehensive Homeodomain Gene Annotation in Human and Mouse</td>
	<td>Veronika Boychenko </td></tr><tr><td>20661</td><td></td><td>I14</td><td>APPRIS - Canonical Isoforms for the Human Genome</td>
	<td>Michael Tress </td></tr><tr><td>20814</td><td></td><td>I18</td><td>Automated curation and assignment of massive DNA sequences</td>
	<td>Duong Vu </td></tr><tr><td>20832</td><td></td><td>I19</td><td>Characterization of Long Noncoding RNAs</td>
	<td>Andrea Tanzer </td></tr><tr><td>22691</td><td></td><td>I21</td><td>YGAP: the Yeast automatic Genome Annotation Pipeline</td>
	<td>Estelle Proux Wera </td></tr><tr><td>22758</td><td></td><td>I22</td><td>A genome-wide, functional analysis of transcription initiation and promoter architecture in the budding yeast Saccharomyces cerevisiae</td>
	<td>R. Taylor Raborn </td></tr><tr><td>22695</td><td></td><td>I23</td><td>Human transcriptome data in Ensembl</td>
	<td>Thibaut Hourlier </td></tr><tr><td>20191</td><td></td><td>I24</td><td>Extending Metagenomic Functional Annotation Through Functional Inference</td>
	<td>Gregory Vey </td></tr><tr><td>19928</td><td></td><td>J01</td><td>Microsoft Biology Initiative</td>
	<td>Beatriz Diaz Acosta </td></tr><tr><td>19931</td><td></td><td>J02</td><td>NUPTs and NUMTs from algae to land plants - a genomic analysis in 12 different species</td>
	<td>Noa Sela </td></tr><tr><td>20259</td><td></td><td>J03</td><td>Dissecting a massive parallel sequencing workflow for quantitative miRNA expression analysis</td>
	<td>Raffaele Calogero </td></tr><tr><td>20284</td><td></td><td>J04</td><td>Boosting the Principle Component Analysis for explorative analysis of genome-wide expressions reveals a common developmental pattern in human and mouse datasets</td>
	<td>Carlo Cannistraci </td></tr><tr><td>20400</td><td></td><td>J09</td><td>Rice-Map: a new-generation rice genome browser</td>
	<td>Ge Gao </td></tr><tr><td>20440</td><td></td><td>J11</td><td>The immense genomic variability of human olfactory receptor genes</td>
	<td>Tsviya Olender </td></tr><tr><td>20495</td><td></td><td>J16</td><td>Interactive Microbial Analysis and Visualization with GView and GView Server</td>
	<td>Aaron Petkau </td></tr><tr><td>20541</td><td></td><td>J19</td><td>SARUMAN - Using GPU programming for short read mapping</td>
	<td>Jochen Blom </td></tr><tr><td>20534</td><td></td><td>J20</td><td>Chipster genome browser: Smooth and interactive visualization of next generation sequencing data</td>
	<td>Aleksi Kallio </td></tr><tr><td>20593</td><td></td><td>J22</td><td>Associating Chromatin Features with Gene Expression</td>
	<td>Thomas Sakoparnig </td></tr><tr><td>20604</td><td></td><td>J23</td><td>Whole genome sequencing, assembly and annotation of the Saccharomyces cerevisiae strain CEN.PK 113-7D</td>
	<td>Dick de Ridder </td></tr><tr><td>20607</td><td></td><td>J24</td><td>Genometa - Accurate, flexible and fast taxonomic assignment for ultra-short metagenomic reads</td>
	<td>Colin Davenport </td></tr><tr><td>20619</td><td></td><td>J25</td><td>Comparing long maximal exact repeats in four human genome assemblies</td>
	<td>Sara Garcia </td></tr><tr><td>20632</td><td></td><td>J26</td><td>Revealing multiple nucleotide polymorphysms and indels in homopolymeric regions in human exome data</td>
	<td>Szilveszter Juhos </td></tr><tr><td>20650</td><td></td><td>J27</td><td>Double nucleotide polymorphisms in human pathogen Propionibacterium acnes strains</td>
	<td>Krisztina Rigo </td></tr><tr><td>20672</td><td></td><td>J28</td><td>Detecting homogenous methylation-domains within whole genome methylation maps</td>
	<td>Michael Hackenberg </td></tr><tr><td>20744</td><td></td><td>J30</td><td>Next generation sequencing and its application to study breast tumor subtypes</td>
	<td>Krishna Kalari </td></tr><tr><td>20761</td><td></td><td>J31</td><td>Bayesian Evaluation of de novo Genome Assembly</td>
	<td>Sergey Nikolenko </td></tr><tr><td>20783</td><td></td><td>J32</td><td>Extracting relationships from integrated functional genomics data with the Ontological Discovery Environment</td>
	<td>Jeremy Jay </td></tr><tr><td>20822</td><td></td><td>J34</td><td>Role of Conserved 3’-UTR Cis-elements in the Regulation of Human MECP2 and MAPT Genes Involved in Mental Disorders</td>
	<td>Joetsaroop Bagga </td></tr><tr><td>20833</td><td></td><td>J35</td><td>GARM - Genome Assembly, Reconciliation and Merging</td>
	<td>Fidel Sanchez-Flores </td></tr><tr><td>20809</td><td></td><td>J38</td><td>A-Bruijn graph approach to de novo genome assembly</td>
	<td>Mikhail Dvorkin </td></tr><tr><td>20878</td><td></td><td>J39</td><td>BLESS: mapping DNA double-strand breaks by next-generation sequencing</td>
	<td>Maga Rowicka </td></tr><tr><td>20884</td><td></td><td>J41</td><td>Identifying the Dynamic States of the 3D Genome Organization</td>
	<td>Andrzej Kudlicki </td></tr><tr><td>20811</td><td></td><td>J42</td><td>Instant-Seq - an integrated and fast Web-based program for analysis of (timecourse) ChIP-Seq data</td>
	<td>Abhishek Mitra </td></tr><tr><td>20924</td><td></td><td>J43</td><td>Statistical Tests for Detecting Differential RNA-Transcript Expression from Read Counts</td>
	<td>Philipp Drewe </td></tr><tr><td>22711</td><td></td><td>J45</td><td>Multi-Image Genome Viewer (MIG)</td>
	<td>Stephen Taylor </td></tr><tr><td>22683</td><td></td><td>J47</td><td>Resource-Efficient de novo Assembly of Eukaryotic Genomes</td>
	<td>Bryan Beresford-Smith </td></tr><tr><td>22667</td><td></td><td>J48</td><td>The SMALT Software for the Efficient and Accurate Alignment of DNA Sequencing Reads</td>
	<td>Hannes Ponstingl </td></tr><tr><td>22743</td><td></td><td>J49</td><td>SeqGI: Sequence Read Enrichment at Genomic Intervals</td>
	<td>Ines de Santiago </td></tr><tr><td>22752</td><td></td><td>J50</td><td>Genome3D: A viewer-model framework for integrating and visualizing multi-scale epigenomic information within a three-dimensional genome</td>
	<td>Wenjin Zheng </td></tr><tr><td>19899</td><td></td><td>K1</td><td>Partial Purification of Cellulase obtained from Tomato (Lycopersicon lycopersicum Karst) Fruits deteriorated by Aspergillus flavus Linn</td>
	<td>Adesola Ajayi </td></tr><tr><td>20291</td><td></td><td>K2</td><td>The EMBL-EBI User Survey</td>
	<td>Jennifer Cham </td></tr><tr><td>20325</td><td></td><td>K3</td><td>Identification of NMR Resonances Using EM Algorithms</td>
	<td>Martin Krone </td></tr><tr><td>20046</td><td></td><td>K4</td><td>In-silico analysis and prevention of Coksackie virus        induced Insulin Dependent Diabetes Mellitus in human</td>
	<td>Ambuj Kumar </td></tr><tr><td>20602</td><td></td><td>K5</td><td>The Hellenic Society for Computational Biology and Bioinformatics</td>
	<td>Christos Ouzounis </td></tr><tr><td>20703</td><td></td><td>K6</td><td>A Need for Specific, Advanced Topic Bioinformatics Training:  The Canadian Bioinformatics Workshop Series</td>
	<td>Michelle Brazas </td></tr><tr><td>22656</td><td></td><td>K7</td><td>Spanish National Bioinformatics Institute (INB)</td>
	<td>Allan Orozco </td></tr><tr><td>20049</td><td></td><td>L01</td><td>GENS2: a simulator for multiloci/whole-genome case-control studies with gene-environment interactions</td>
	<td>Roberto Amato </td></tr><tr><td>22636</td><td></td><td>L02</td><td>Universal epitope prediction for class II MHC</td>
	<td>Andrew Bordner </td></tr><tr><td>20336</td><td></td><td>L06</td><td>Characterization of compound gene regulation mechanisms</td>
	<td>Reiko Tanaka </td></tr><tr><td>20380</td><td></td><td>L09</td><td>Unraveling protein-protein interactions by a semi-automated high-throughput platform</td>
	<td>Dominik Mertens </td></tr><tr><td>20399</td><td></td><td>L10</td><td>Discovery of a 3D Motif Essential to LDLR-Protein Complexes: Application to docking prediction evaluation</td>
	<td>Reyhaneh Esmaielbeiki </td></tr><tr><td>20420</td><td></td><td>L11</td><td>Coevolution and Predicted Solvent Accessibility</td>
	<td>David Ochoa </td></tr><tr><td>20499</td><td></td><td>L13</td><td>On sequential conservation space of RNA-recognizing residues in protein sequences</td>
	<td>Li Xue </td></tr><tr><td>20753</td><td></td><td>L14</td><td>Functional assessment of topological characterization using graphlet degrees in PPI networks</td>
	<td>Marco Mina </td></tr><tr><td>20560</td><td></td><td>L15</td><td>SNP-SNP synergy discovery with SNPsyn</td>
	<td>Tomaz Curk </td></tr><tr><td>20854</td><td></td><td>L16</td><td>RISEARCH: Fast RNA Interaction Search</td>
	<td>Jan Gorodkin </td></tr><tr><td>22631</td><td></td><td>L17</td><td>Predicting conserved interactions and structures of two multiple alignments of RNA sequences.</td>
	<td>Stefan Seemann </td></tr><tr><td>22682</td><td></td><td>L20</td><td>A novel ultra-fast comparison method for known and potential ligand-binding sites of proteins</td>
	<td>Kentaro Tomii </td></tr><tr><td>22572</td><td></td><td>L21</td><td>New methods for finding common insertion sites and co-occurring common insertion sites in transposon- and virus-based genetic screens</td>
	<td>Tracy Bergemann </td></tr><tr><td>22744</td><td></td><td>L23</td><td>An alternative approach to protein docking: Complexes via homology modeling</td>
	<td>Ozlem Tastan Bishop </td></tr><tr><td>22619</td><td></td><td>M01</td><td>Fast and Efficient Dynamic Nested Effects Models</td>
	<td>Holger Fröhlich </td></tr><tr><td>20035</td><td></td><td>M02</td><td>Prediction of bioluminescent proteins based on Discrete Wavelet Transform and Support Vector Machine</td>
	<td>Krishna Kumar kandaswamy </td></tr><tr><td>22609</td><td></td><td>M03</td><td>A new approach for discovering knowledge from a large mass of data and optimisation of the numbers of variables before learning Bayesian Networks structure: Application in genetic studies of complex g</td>
	<td>Heni Bouhamed </td></tr><tr><td>20265</td><td></td><td>M04</td><td>R-value: A new measure for dataset evaluation</td>
	<td>Sejong Oh </td></tr><tr><td>20321</td><td></td><td>M05</td><td>Transcription Regulation: models for combinatorial regulation and functional specificity</td>
	<td>David Thomas </td></tr><tr><td>20343</td><td></td><td>M06</td><td>Contrasting the Molecular Basis of Human and Mouse Embryonic Stem Cell Self-Renewal Using Bayesian Network Machine Learning Methods</td>
	<td>Karen Dowell </td></tr><tr><td>20350</td><td></td><td>M07</td><td>A new feature selection method based on classes overlap</td>
	<td>Sejong Oh </td></tr><tr><td>20395</td><td></td><td>M08</td><td>Protein subnuclear localization using a spectrum kernel minimum distance probability method</td>
	<td>Esteban Vegas </td></tr><tr><td>20409</td><td></td><td>M09</td><td>SVM-based prediction of granzyme B cleavage sites</td>
	<td>Lawrence Wee </td></tr><tr><td>20425</td><td></td><td>M10</td><td>Genome-wide enhancer discovery from chromatin methylation marks using Genetic Algorithm-optimized Support Vector Machines</td>
	<td>Michael Fernandez </td></tr><tr><td>20449</td><td></td><td>M11</td><td>Machine Learning Approaches to Identify and Characterize Effector Proteins in Pathogenic Bacteria</td>
	<td>David Burstein </td></tr><tr><td>20576</td><td></td><td>M13</td><td>Assessment of protein interaction prediction from sequence data using n-gram analysis</td>
	<td>Guray Kuzu </td></tr><tr><td>20575</td><td></td><td>M14</td><td>Model selection for predictive and prognostic gene signatures</td>
	<td>Miika Ahdesmaki </td></tr><tr><td>20662</td><td></td><td>M16</td><td>Dealing with Uncertainty in Support Vector Machines Prediction</td>
	<td>Sorin Draghici </td></tr><tr><td>20665</td><td></td><td>M17</td><td>Identification of SH2-peptide interactions using support vector machine</td>
	<td>Kousik Kundu </td></tr><tr><td>20688</td><td></td><td>M18</td><td>A Support Vector Machine approach for rapid genome-wide epistasis detection</td>
	<td>Barbara Rakitsch </td></tr><tr><td>20717</td><td></td><td>M19</td><td>Time series classifier model for miR-mRNA interaction</td>
	<td>Jovan Rebolledo-Mendez </td></tr><tr><td>20719</td><td></td><td>M20</td><td>mTiM: margin-based transcript mapping from RNA-seq</td>
	<td>Nico Görnitz </td></tr><tr><td>20855</td><td></td><td>M21</td><td>A Computational Paradigm for More Specific TFBS Detection</td>
	<td>Heike Sichtig </td></tr><tr><td>20896</td><td></td><td>M22</td><td>Combined Approach to Anaphora Resolution for Disease-Determinants Ontology Learning from Web Sources</td>
	<td>Jae-Hong Eom </td></tr><tr><td>20156</td><td></td><td>N01</td><td>Power and minimal sample size for multivariate analysis of microarrays</td>
	<td>Maarten Iterson </td></tr><tr><td>20344</td><td></td><td>N05</td><td>caArray: make microarray data easily accessible and exchangeable</td>
	<td>Xiaopeng Bian </td></tr><tr><td>20351</td><td></td><td>N06</td><td>A combination technique for microarray dataset to improve classification accuracy</td>
	<td>Sejong Oh </td></tr><tr><td>20462</td><td></td><td>N09</td><td>ASPIRE: An improved tool for the analysis of alternative splicing using AltSplice microarrays</td>
	<td>Melis Kayikci </td></tr><tr><td>20498</td><td></td><td>N10</td><td>Meta-analysis of gene expression microarrays with missing replicates</td>
	<td>Fan Shi </td></tr><tr><td>20505</td><td></td><td>N12</td><td>Feature selection for specifying experimental conditions that activate gene transcriptions</td>
	<td>Sho Ohsuga </td></tr><tr><td>20387</td><td></td><td>N13</td><td>A Method for Inference of Gene Regulatory Network based on Bayesian Network with Uniting of Partial Problems</td>
	<td>Yukito Watanabe </td></tr><tr><td>20550</td><td></td><td>N15</td><td>Prediction of specific versus non-specific RBP-RNA interaction</td>
	<td>Eleonora Kulberkyte </td></tr><tr><td>20561</td><td></td><td>N16</td><td>cn.FARMS: a latent variable model to detect copy number variations in microarray data with a low false discovery rate</td>
	<td>Djork-Arné Clevert </td></tr><tr><td>20608</td><td></td><td>N18</td><td>COLOMBOS: Access Port for Cross-Platform Bacterial Expression Compendia</td>
	<td>Kristof Engelen </td></tr><tr><td>20666</td><td></td><td>N24</td><td>Simulation strategies to evaluate the performance of pathway-based gene expression analysis methods</td>
	<td>Shailesh Tripathi </td></tr><tr><td>20747</td><td></td><td>N25</td><td>Signaling Pathway Coupling Phenomena</td>
	<td>Michele Donato </td></tr><tr><td>20699</td><td></td><td>N27</td><td>Alternative Splicing in Malignant Lymphoma</td>
	<td>Karin Zimmermann </td></tr><tr><td>20823</td><td></td><td>N29</td><td>Flanking sequence effects on oligonucleotide hybridization</td>
	<td>D. Andrew Carr </td></tr><tr><td>20872</td><td></td><td>N30</td><td>Elimination of spatial autocorrelation in microarray data</td>
	<td>Sébastien Lemieux </td></tr><tr><td>22647</td><td></td><td>N31</td><td>A Computational Study of Carotenoid Gene Expression using Micro-array Data</td>
	<td>Firdous Khan </td></tr><tr><td>22662</td><td></td><td>N32</td><td>A survey of batch effects in public gene expression data</td>
	<td>Paul Pavlidis </td></tr><tr><td>22735</td><td></td><td>N36</td><td>Using R through a Tiki web interface to implement bioinformatics pipelines</td>
	<td>Xavier Pedro Puente </td></tr><tr><td>22750</td><td></td><td>N37</td><td>Combining omics data and biological knowledge  using multiple factor analysis</td>
	<td>Jose Luis Mosquera </td></tr><tr><td>20287</td><td></td><td>O1</td><td>An integrated approach to the alignment and classification of 3D neuronal morphology in the Drosophila brain</td>
	<td>chao-chun chuang </td></tr><tr><td>20569</td><td></td><td>O2</td><td>Effect of Leaky Channels in Calcium Homeostasis</td>
	<td>Suhas Vasaikar </td></tr><tr><td>22618</td><td></td><td>P01</td><td>Benchmarking Ontologies: Bigger or Better?</td>
	<td>Lixia Yao </td></tr><tr><td>20032</td><td></td><td>P03</td><td>A Modular Annotation Framework for Mathematical Models</td>
	<td>Michael Cooling </td></tr><tr><td>20327</td><td></td><td>P04</td><td>An ontology of bioinformatics tool functions, data types and formats</td>
	<td>Jon Ison </td></tr><tr><td>20328</td><td></td><td>P05</td><td>Multiple Ontology Based System for Integrating and Display Rat Phenotype Data</td>
	<td>Mary Shimoyama </td></tr><tr><td>20473</td><td></td><td>P06</td><td>ontoCAT R package for ontologies traversal and search</td>
	<td>Natalja Kurbatova </td></tr><tr><td>20527</td><td></td><td>P07</td><td>What does semantic similarity mean in terms of protein function comparison? The analysis of relation between functional and structural similarity.</td>
	<td>Bogumil Konopka </td></tr><tr><td>20566</td><td></td><td>P08</td><td>A Semantic Web Infrastructure for Bioinformatics of Staphylococcus aureus</td>
	<td>Cândida Delgado </td></tr><tr><td>20581</td><td></td><td>P09</td><td>A standard for cooperative interactions</td>
	<td>Kim Van Roey </td></tr><tr><td>20801</td><td></td><td>P11</td><td>BioPAX – A Community Standard for Pathway Data Sharing</td>
	<td>Nadia Anwar </td></tr><tr><td>20299</td><td></td><td>Q01</td><td>Signatures of positive selection on genes of the N-Glycosylation pathway concentrate on genes with regulatory positions</td>
	<td>Giovanni Marco Dall'Olio </td></tr><tr><td>20417</td><td></td><td>Q03</td><td>cn.MOPS: Mixture of Poissons for Discovering Copy Number Variations in Next Generation Sequencing Data</td>
	<td>Günter Klambauer </td></tr><tr><td>20490</td><td></td><td>Q04</td><td>Deep-sequencing reveals phage-driven diversification of Pseudomonas aerugionsa biofilms</td>
	<td>Kerensa McElroy </td></tr><tr><td>20810</td><td></td><td>Q07</td><td>INRICH: improved enrichment testing method that integrates an unbiased GWAS strategy with prior biological knowledge: application to Type 2 diabetes</td>
	<td>Phil Lee </td></tr><tr><td>20363</td><td></td><td>R01</td><td>Statistical approach to absolute protein quantification</td>
	<td>Sarah Gerster </td></tr><tr><td>20633</td><td></td><td>R03</td><td>Chimeras knocking at the door</td>
	<td>Milana Frenkel-Morgenstern </td></tr><tr><td>20676</td><td></td><td>R04</td><td>Bioinformatics for Proteomics: developing new tools for the analysis of experimental data</td>
	<td>Angelo Facchiano </td></tr><tr><td>20707</td><td></td><td>R05</td><td>Predict subcellular localization for proteins in all kingdoms</td>
	<td>Tatyana Goldberg </td></tr><tr><td>20637</td><td></td><td>R06</td><td>Quantitative mass spectrometry data based pathway analysis</td>
	<td>Philipp Gormanns </td></tr><tr><td>20752</td><td></td><td>R08</td><td>PRIDE, the PRoteomics IDEntifications database</td>
	<td>Johannes Griss </td></tr><tr><td>20756</td><td></td><td>R09</td><td>Lost Data – The Consequences of Changing Protein Accession</td>
	<td>Johannes Griss </td></tr><tr><td>20768</td><td></td><td>R10</td><td>Proteomics based genome annotation using conceptually translated ESTs</td>
	<td>Paul Blakeley </td></tr><tr><td>20825</td><td></td><td>R11</td><td>Data mining on a leukemia phosphoproteome dataset: insights on chemoresistant cell signaling</td>
	<td>Renato Milani </td></tr><tr><td>22659</td><td></td><td>R12</td><td>Identification of protein expression dynamics during germination in Streptomyces</td>
	<td>Eva Strakova </td></tr><tr><td>22748</td><td></td><td>R13</td><td>Tracing Evolutionary Events Over Ancestral Proteomes</td>
	<td>Matt Oates </td></tr><tr><td>22767</td><td></td><td>R14</td><td>Viral proteins acquired from a host converge to simplified domain architectures</td>
	<td>Nadav Rappoport </td></tr><tr><td>20174</td><td></td><td>S01</td><td>Gene Array Analyzer (GAA): Alternative usage of gene arrays to study alternative splicing events</td>
	<td>Pascal Gellert </td></tr><tr><td>20271</td><td></td><td>S02</td><td>Cell specific gene expression from histome modifications and transcription factor binding</td>
	<td>Thaís Rêgo </td></tr><tr><td>20311</td><td></td><td>S05</td><td>Induction of Effectors in Diuraphis noxia after Feeding on Preference and Non-preference Wheat Hosts</td>
	<td>Nicolaas Burger </td></tr><tr><td>20370</td><td></td><td>S06</td><td>ConTra v2: a tool to identify transcription factor binding sites                 across species, update 2011</td>
	<td>Stefan Broos </td></tr><tr><td>20535</td><td></td><td>S08</td><td>Unveiling combinatorial regulation through the combination of ChIP information and in silico cis-regulatory module detection</td>
	<td>Kathleen Marchal </td></tr><tr><td>20542</td><td></td><td>S09</td><td>Transcriptional cross-regulation as survival mechanism in bacteria</td>
	<td>Pieter Monsieurs </td></tr><tr><td>20558</td><td></td><td>S10</td><td>A probabilistic model for prediction of Transcription Start Regions in both intra- and inter-genic miRNAs</td>
	<td>Annalisa Marsico </td></tr><tr><td>20587</td><td></td><td>S11</td><td>Analysis of the structure and evolutionary conservation of regulatory networks in prokaryotes.</td>
	<td>Richard Münch </td></tr><tr><td>20657</td><td></td><td>S12</td><td>PRODORIC - a platform for the elucidation of gene regulatory networks in prokaryotes - from data to models</td>
	<td>Richard Münch </td></tr><tr><td>20876</td><td></td><td>S14</td><td>Identifying CNVs of regulatory elements which impact overall survival in serous cystadenocarcinoma</td>
	<td>Geoff Macintyre </td></tr><tr><td>20889</td><td></td><td>S16</td><td>Improved in silico regulatory SNP detection facilitates large scale SNP scanning</td>
	<td>Geoff Macintyre </td></tr><tr><td>22737</td><td></td><td>S18</td><td>Identification of shortened 3’untranslated regions by Affymetrix genechip arrays</td>
	<td>Loredana Martignetti </td></tr><tr><td>22757</td><td></td><td>S19</td><td>Integrative approach at building and analyzing gene regulatory networks</td>
	<td>Julia Ponomarenko </td></tr><tr><td>20493</td><td></td><td>T1</td><td>A Framework for the Evaluation of Bioinformatics Methods</td>
	<td>José M. Fernández </td></tr><tr><td>20612</td><td></td><td>T2</td><td>eBiomics: an e-Learning Environment on Bioinformatics for Life Scientists</td>
	<td>Patrick Koks </td></tr><tr><td>20873</td><td></td><td>T3</td><td>Practical NGS Analysis: A Bioinformatician’s Perspective</td>
	<td>Abhishek Pratap </td></tr><tr><td>20891</td><td></td><td>T4</td><td>A Review of Software for Visualization and Interpretation of Genome Wide Association Studies Results</td>
	<td>Armita Zarnegar </td></tr><tr><td>22626</td><td></td><td>U03</td><td>Accurate quantification of transcriptome from RNA-Seq data by effective length normalization</td>
	<td>Sanghyuk Lee </td></tr><tr><td>20183</td><td></td><td>U04</td><td>Application of whole genome resequencing to detect sequence polymorphisms underlying boar taint</td>
	<td>Rahul Agarwal </td></tr><tr><td>20186</td><td></td><td>U05</td><td>High-throughput protein analysis in the cloud</td>
	<td>Guy Yachdav </td></tr><tr><td>20197</td><td></td><td>U06</td><td>Evolutionary Conservation and Genetic Variability of Hepatitis C Virus Core Protein (HCVc)</td>
	<td>Muhammad Ilyas </td></tr><tr><td>20256</td><td></td><td>U07</td><td>A method to separate function and fold specific residues in a protein.</td>
	<td>Andrew Lynn </td></tr><tr><td>20298</td><td></td><td>U10</td><td>EMBOSS: EuropeanMolecular Biology Open Software Suite</td>
	<td>Peter Rice </td></tr><tr><td>20383</td><td></td><td>U11</td><td>SKM: A String Kernel Motifs Classifier for Nucleic Acid and Protein Sequences</td>
	<td>Hugh Shanahan </td></tr><tr><td>20392</td><td></td><td>U12</td><td>P-value based motif identification using positional weight matrices</td>
	<td>Holger Hartmann </td></tr><tr><td>20450</td><td></td><td>U14</td><td>Toolbox for the analysis of Roche 454 Sequencing data</td>
	<td>Christian Ruckert </td></tr><tr><td>20460</td><td></td><td>U15</td><td>Comrad: Detection of expressed rearrangements by integrated analysis of RNA-Seq and low coverage genome sequence data</td>
	<td>Andrew McPherson </td></tr><tr><td>20470</td><td></td><td>U16</td><td>The Interlaboratory Robustness of Next-Generation Sequencing (IRON) Study</td>
	<td>Hans-Ulrich Klein </td></tr><tr><td>20502</td><td></td><td>U20</td><td>Improved performance of sequence search algorithms in remote homology detection</td>
	<td>Ramanathan Sowdhamini </td></tr><tr><td>20513</td><td></td><td>U21</td><td>New method of detecting pathogenic viruses using high-throughput sequencing data</td>
	<td>Shumpei Ishikawa </td></tr><tr><td>20520</td><td></td><td>U22</td><td>Perfect Hamming Code as Hash key for Fast Genome Mapping</td>
	<td>Yoichi Takenaka </td></tr><tr><td>20524</td><td></td><td>U24</td><td>fastapl -- a utility for versatile and easy processing of multifasta format data</td>
	<td>Paul Horton </td></tr><tr><td>20539</td><td></td><td>U25</td><td>Behind the significance in NGS reads assembly</td>
	<td>Antonio Muñoz </td></tr><tr><td>20559</td><td></td><td>U26</td><td>Computational Analysis of DNA Replicases in Double-Stranded DNA Viruses: Relationship with the Genome Size</td>
	<td>Darius Kazlauskas </td></tr><tr><td>20565</td><td></td><td>U27</td><td>A novel miRNA prediction method based on next generation sequencing technology</td>
	<td>Kui Qian </td></tr><tr><td>20580</td><td></td><td>U28</td><td>Sequence alignment of coiled coil proteins: adapting to restricted sequence variation</td>
	<td>Michael Kuhn </td></tr><tr><td>20592</td><td></td><td>U29</td><td>Computational prediction of transposon insertion sites in bacterial genomes</td>
	<td>Michael Steinert </td></tr><tr><td>20606</td><td></td><td>U30</td><td>Improving SNV calling sensitivity by probabilistically combining SNVs reported by different calling tools</td>
	<td>Alejandro Balbin </td></tr><tr><td>20547</td><td></td><td>U31</td><td>Integrating Protein Domain Prediction in  Multiple Sequence Alignment</td>
	<td>Layal Al-Ait </td></tr><tr><td>20663</td><td></td><td>U32</td><td>A new algorithm for sequencing by hybridization based on non-classical approach</td>
	<td>Marcin Radom </td></tr><tr><td>20682</td><td></td><td>U34</td><td>Computational challenges in the production and analysis of Illumina Genome Analyzer data</td>
	<td>Martin Kircher </td></tr><tr><td>20685</td><td></td><td>U35</td><td>Simulating Quality and Depth of Sequence for Genotype Accuracy</td>
	<td>Iain Bancarz </td></tr><tr><td>20730</td><td></td><td>U37</td><td>Using next-generation sequencing “dark matter” to identify a new transposable element location in the mouse genome.</td>
	<td>James Cavalcoli </td></tr><tr><td>20750</td><td></td><td>U39</td><td>A Conditional Random Fields Model for RNA Conformation Sampling</td>
	<td>Jinbo Xu </td></tr><tr><td>20763</td><td></td><td>U40</td><td>A new approach to suboptimal pairwise sequence alignment</td>
	<td>Feng Lou </td></tr><tr><td>20758</td><td></td><td>U41</td><td>Support Vector Machines for finding deletions and short insertions using paired-end short reads</td>
	<td>Dominik Grimm </td></tr><tr><td>20770</td><td></td><td>U42</td><td>A combination of amino acid composition and matching for conotoxin superfamily classification</td>
	<td>Nazar Zaki </td></tr><tr><td>20781</td><td></td><td>U43</td><td>Ranking and filtering translocations in paired-end sequencing data using BLAT realignment</td>
	<td>Christoph Bartenhagen </td></tr><tr><td>20808</td><td></td><td>U45</td><td>Distributed Web Services for Bioinformatics:  Multiple Sequence Alignment</td>
	<td>Peter Troshin </td></tr><tr><td>20806</td><td></td><td>U46</td><td>Pythoscape: A Software Framework for Generation of Large Protein Similarity Networks</td>
	<td>Alan Barber </td></tr><tr><td>20596</td><td></td><td>U47</td><td>LaTcOm: A web server for visualizing rare codon clusters</td>
	<td>Athina Theodosiou </td></tr><tr><td>20830</td><td></td><td>U48</td><td>SRS 3D - Integrating structures, sequences and annotations</td>
	<td>Andrea Schafferhans </td></tr><tr><td>20680</td><td></td><td>U49</td><td>Multiobjective sequence alignment: Formulation and Algorithms</td>
	<td>Maryam Abbasi </td></tr><tr><td>20838</td><td></td><td>U50</td><td>An improved RIP-Seq Toolkit to Reveal Genome-wide Exon Junction Complex Landscape</td>
	<td>Alper Kucukural </td></tr><tr><td>20870</td><td></td><td>U54</td><td>Visualizing the next generation of sequencing data with GenomeView</td>
	<td>Thomas Abeel </td></tr><tr><td>22643</td><td></td><td>U55</td><td>SlideSort: Fast and exact algorithm for Next Generation Sequencing data analysis</td>
	<td>Kana Shimizu </td></tr><tr><td>22706</td><td></td><td>U56</td><td>Who & How: Investigating miRNA Regulation in Ranked Gene Lists using miRnaPursuit.</td>
	<td>Teresa Colombo </td></tr><tr><td>22736</td><td></td><td>U60</td><td>Performing Functional Site Prediction on Protein Sequences Generated by De Novo Transcriptome Assembly</td>
	<td>Chien-Yu Chen </td></tr><tr><td>22671</td><td></td><td>U61</td><td>MEGA5: Molecular Evolutionary Genetics Analysis using Maximum Likelihood, Evolutionary Distance, and Maximum Parsimony Methods</td>
	<td>Sudhir Kumar </td></tr><tr><td>22739</td><td></td><td>U62</td><td>Twenty-Five Fold Diversity Overestimation Revealed by Analyzing a MOCK  Sample using 454 Amplicon Pyrosequencing</td>
	<td>Sebastian Jünemann </td></tr><tr><td>22654</td><td></td><td>U64</td><td>ADDAPTS: A Data-Driven Automated Pipeline and Tracking System</td>
	<td>Risha Narayan </td></tr><tr><td>20048</td><td></td><td>V1</td><td>P.R.E.S.S.--- An R-package for Exploring Residual-Level Protein Structural Statistics</td>
	<td>Yuanyuan huang </td></tr><tr><td>22630</td><td></td><td>V2</td><td>DIBER: protein, dna or both?</td>
	<td>Grzegorz Chojnowski </td></tr><tr><td>20736</td><td></td><td>V6</td><td>The Biochemical Algorithms Library (BALL) - Rapid Application Development in Structural Bioinformatics</td>
	<td>Andreas Hildebrandt </td></tr><tr><td>20745</td><td></td><td>V7</td><td>SymD server: a platform for detecting internally symmetric protein structures</td>
	<td>Chin-Hsien (Emily) Tai </td></tr><tr><td>19881</td><td></td><td>W01</td><td>A Time-Dependent Framework for Solving the Degeneracy Problem in RNA Secondary Structure Deleterious Mutation Prediction</td>
	<td>Danny Barash </td></tr><tr><td>22613</td><td></td><td>W02</td><td>Correcting protein alignment errors using a novel multiple-template threading method</td>
	<td>Jinbo Xu </td></tr><tr><td>22602</td><td></td><td>W03</td><td>All-atom knowledge-based potential for RNA structure prediction and assessment</td>
	<td>Emidio Capriotti </td></tr><tr><td>22601</td><td></td><td>W04</td><td>A structural and dynamical model of human telomerase</td>
	<td>Samuel Flores </td></tr><tr><td>20359</td><td></td><td>W05</td><td>Optimization of residual electrostatic environment to enhance protein thermal stability</td>
	<td>Chi-Wen Lee </td></tr><tr><td>22616</td><td></td><td>W06</td><td>Complex networks govern coiled coil oligomerization - predicting and profiling by means of a machine learning approach</td>
	<td>Ulrich Bodenhofer </td></tr><tr><td>22634</td><td></td><td>W08</td><td>The imprint of codons on protein structure</td>
	<td>Charlotte Deane </td></tr><tr><td>20154</td><td></td><td>W09</td><td>PyRy3D: a software tool for modelling of large macromolecular complexes</td>
	<td>Joanna Kasprzak </td></tr><tr><td>20177</td><td></td><td>W10</td><td>Planar shape representation based Protein function prediction</td>
	<td>Heng Yang </td></tr><tr><td>20212</td><td></td><td>W12</td><td>MODORAMA - a web application for comparative protein structure modeling</td>
	<td>Jan Kosinski </td></tr><tr><td>20275</td><td></td><td>W13</td><td>Mean Scaled RMSD: A Size Independent Measure For Superimposition of Protein Structures</td>
	<td>Seyed Shahriar Arab </td></tr><tr><td>20286</td><td></td><td>W14</td><td>Probabilistic model for secondary structure prediction from protein chemical shifts</td>
	<td>Martin Mechelke </td></tr><tr><td>20294</td><td></td><td>W15</td><td>Multi-LZerD: Multiple protein docking for asymmetric complexes</td>
	<td>Juan Esquivel-Rodríguez </td></tr><tr><td>20297</td><td></td><td>W16</td><td>Modelling Protein Domain Binding to  Kinase Structures altered by Phosphorylation</td>
	<td>Christoph Gernert </td></tr><tr><td>20345</td><td></td><td>W17</td><td>Full atom comparative three-dimensional modeling of protein-DNA complexes</td>
	<td>Francisco Melo </td></tr><tr><td>20352</td><td></td><td>W18</td><td>Identification of cavities on protein surface using multiple compu-tational approaches for drug binding site prediction</td>
	<td>Bingding Huang </td></tr><tr><td>20364</td><td></td><td>W19</td><td>Physical contacts-based score for evaluation of protein three-dimensional models</td>
	<td>Kliment Olechnovic </td></tr><tr><td>20373</td><td></td><td>W21</td><td>The influence of the local sequence environment on RNA loop structures</td>
	<td>Dirk Walther </td></tr><tr><td>20402</td><td></td><td>W23</td><td>Peptides modulating conformational changes: From in silico design to preclinical proof of concept</td>
	<td>Itamar Borukhov </td></tr><tr><td>20412</td><td></td><td>W24</td><td>Spritz: Intrinsic protein disorder prediction for genomes and insights into protein function.</td>
	<td>Ian Walsh </td></tr><tr><td>20433</td><td></td><td>W26</td><td>Slip symmetry – a new type of symmetry in protein structures</td>
	<td>Dukka KC </td></tr><tr><td>20474</td><td></td><td>W27</td><td>A multiple-template approach to protein threading</td>
	<td>Jinbo Xu </td></tr><tr><td>20483</td><td></td><td>W28</td><td>A novel type of methyltransferase encoded by a conserved domain of GRAS</td>
	<td>Magdalena Dymecka </td></tr><tr><td>20461</td><td></td><td>W29</td><td>Implementation of the Protein Fluorescence And Structural Toolkit (PFAST)</td>
	<td>Cynthia Prudence </td></tr><tr><td>20504</td><td></td><td>W30</td><td>Maximal Use of Information Found in All Protein Structure Data</td>
	<td>Armando Solis </td></tr><tr><td>20557</td><td></td><td>W33</td><td>Protein function prediction using clustering method of Protein-Protein Interaction network</td>
	<td>Sejong Oh </td></tr><tr><td>20588</td><td></td><td>W34</td><td>Comprehensive study of HOX transcription factors DNA binding specificity</td>
	<td>Katarzyna Kokoszynska </td></tr><tr><td>20597</td><td></td><td>W36</td><td>The MULTICOM Toolbox for Protein Structure Prediction</td>
	<td>Jianlin Cheng </td></tr><tr><td>20601</td><td></td><td>W37</td><td>Comprehensive structure-to-function analysis of transcription factor IID (TFIID) complex</td>
	<td>Maja Małkowska </td></tr><tr><td>20616</td><td></td><td>W38</td><td>Structure-based classification of the plant non-specific lipid transfer protein superfamily towards its functional characterization</td>
	<td>Cécile Fleury </td></tr><tr><td>20635</td><td></td><td>W39</td><td>A comparative study on filtering protein secondary structure prediction</td>
	<td>Chris Christodoulou </td></tr><tr><td>20645</td><td></td><td>W40</td><td>firestar in 2011</td>
	<td>Michael Tress </td></tr><tr><td>20675</td><td></td><td>W42</td><td>BOCTOPUS: Topology prediction of Transmembrane beta barrel proteins</td>
	<td>Sikander Hayat </td></tr><tr><td>20679</td><td></td><td>W43</td><td>Analysis of Functional Islands using Structural Clustering</td>
	<td>Anne-Christin Hauschild </td></tr><tr><td>20712</td><td></td><td>W45</td><td>Predicting the 3d DNA structure of the human genome</td>
	<td>Sven Bilke </td></tr><tr><td>20734</td><td></td><td>W46</td><td>Binding Ligand Prediction by Comparing Local Surface Patches of Potential Pocket Regions</td>
	<td>Daisuke Kihara </td></tr><tr><td>20764</td><td></td><td>W48</td><td>ProBiS: A web server for detection of structurally similar protein binding sites</td>
	<td>Janez Konc </td></tr><tr><td>20478</td><td></td><td>W49</td><td>Protein conformational movements modeling based on the average action principle</td>
	<td>Alexander Krass </td></tr><tr><td>20767</td><td></td><td>W50</td><td>Evaluation of transmembrane helix (TMH) packing prediction methods on successive in sequence TMHs</td>
	<td>Vasilis Promponas </td></tr><tr><td>20792</td><td></td><td>W52</td><td>Are Protein Disordered Regions Equal to Loops?</td>
	<td>Esmeralda Vicedo </td></tr><tr><td>20695</td><td></td><td>W53</td><td>Environment Specific Substitution Tables Improve Membrane Protein Alignment</td>
	<td>Jamie Hill </td></tr><tr><td>20819</td><td></td><td>W54</td><td>Quality or Quantity of Structural Dynamics Information: That is the Question! (when improving performance of structure-based function prediction methods)</td>
	<td>Dariya Glazer </td></tr><tr><td>20837</td><td></td><td>W55</td><td>Nearest-Neighbor Approaches to Predict Protein Function by Sequence Homology Alone</td>
	<td>Tobias Hamp </td></tr><tr><td>20578</td><td></td><td>W58</td><td>Is the biological hydrophobicity scale correct?</td>
	<td>Christoph Peters </td></tr><tr><td>20892</td><td></td><td>W60</td><td>A comprehensive review of protein fold space and the correlation of structure with function</td>
	<td>Spencer Bliven </td></tr><tr><td>20897</td><td></td><td>W61</td><td>Protein interaction partners revealed by their dynamical properties</td>
	<td>Jau-Ji Lin </td></tr><tr><td>22693</td><td></td><td>W65</td><td>Is It Biologically Relevant? An Evolutionary Method for Distinguishing Biological Interfaces from Crystal Contacts</td>
	<td>Guido Capitani </td></tr><tr><td>22670</td><td></td><td>W66</td><td>SAHG, a comprehensive database of annotated structure models for human proteins</td>
	<td>Chie Motono </td></tr><tr><td>22721</td><td></td><td>W67</td><td>Training bidirectional recurrent neural networks with Conjugate gradient-type algorithms for protein secondary structure prediction</td>
	<td>Chris Christodoulou </td></tr><tr><td>22759</td><td></td><td>W68</td><td>Protein Sequence Design Using Ensemble-Based Energetic Information</td>
	<td>James Wrabl </td></tr><tr><td>22761</td><td></td><td>W69</td><td>Predicting Functions of the Universal Stress Proteins in Metabolically Versatile Rhodopseudomonas palustris</td>
	<td>Shaneka Simmons </td></tr><tr><td>22663</td><td></td><td>W70</td><td>The Impact of Multifunctional Genes on "Guilt by Association" Analysis</td>
	<td>Jesse Gillis </td></tr><tr><td>22717</td><td></td><td>W71</td><td>Understanding protein-protein interactions: Efficient classification of physiological and non-physiological protein complexes and unexpected conservation in interaction interfaces</td>
	<td>Dmitry Korkin </td></tr><tr><td>20622</td><td></td><td>W72</td><td>Protein Structural Domain Assignment Using Delaunay Tessellation</td>
	<td>Todd Taylor </td></tr><tr><td>19952</td><td></td><td>X01</td><td>A regulatory network model of CD4+ lymphocytes</td>
	<td>Luis Mendoza </td></tr><tr><td>20054</td><td></td><td>X03</td><td>ALISSA-An Automated Live Cell Imaging System for Signal Transduction Analyses</td>
	<td>Heinrich Huber </td></tr><tr><td>22605</td><td></td><td>X04</td><td>Interactive, multiscale navigation of large and complicated biological networks</td>
	<td>Thanet Praneenararat </td></tr><tr><td>22637</td><td></td><td>X05</td><td>Finding out what happened -- Highlighting the essence of a biological experiment</td>
	<td>Georg Fuellen </td></tr><tr><td>22639</td><td></td><td>X06</td><td>Topological network alignment uncovers biological function and phylogeny</td>
	<td>Natasa Przulj </td></tr><tr><td>20159</td><td></td><td>X07</td><td>Inferring the role of alternative splicing on biological pathways: Studies using cancer transcriprome data</td>
	<td>Murlidharan Nair </td></tr><tr><td>20258</td><td></td><td>X08</td><td>Genetic Design Through Branch and Bound</td>
	<td>Dennis Egen </td></tr><tr><td>20276</td><td></td><td>X09</td><td>Bridging clinical knowledge to molecular events through integration of imaging biomarkers and protein interactions in Alzheimer’s disease</td>
	<td>Erfan Younesi </td></tr><tr><td>20281</td><td></td><td>X10</td><td>Network-based approach towards candidate gene prioritisation using TargetMine</td>
	<td>Lokesh Tripathi </td></tr><tr><td>20285</td><td></td><td>X11</td><td>Integrated cell cycle and DNA repair signalling network modelling for identification of key molecular regulators in basal-like breast cancer</td>
	<td>Inna Kuperstein </td></tr><tr><td>20292</td><td></td><td>X12</td><td>An interface analysis of human E2-E3 complexes: structural insights into specificity</td>
	<td>Gozde Kar </td></tr><tr><td>20296</td><td></td><td>X13</td><td>How little do we actually know? – Towards the size of gene regulatory networks.</td>
	<td>Richard Röttger </td></tr><tr><td>20367</td><td></td><td>X16</td><td>A Directed Graphical Gaussian Model for Inferring Gene Regulatory Networks</td>
	<td>Tomoshige Ohno </td></tr><tr><td>20368</td><td></td><td>X18</td><td>Fast and Efficient Dynamic Nested Effects Models</td>
	<td>Holger Fröhlich </td></tr><tr><td>20378</td><td></td><td>X20</td><td>bioCompendium: high-throughput experimental data analysis platform</td>
	<td>Venkata Satagopam </td></tr><tr><td>20424</td><td></td><td>X22</td><td>Detection of nonlinear effects in gene expression pathways</td>
	<td>Andreas Mayr </td></tr><tr><td>20434</td><td></td><td>X23</td><td>Network topology complements sequence as a source of biological information</td>
	<td>Tijana Milenkovic </td></tr><tr><td>20436</td><td></td><td>X24</td><td>Understanding temporal patterns in cell cycle regulation through visualization</td>
	<td>Maria Secrier </td></tr><tr><td>20444</td><td></td><td>X25</td><td>From WIPA (Web Interface Protein Atlas) to Systems Biology Analysis of Protein Atlas Tissue Microarrays (TMAs)</td>
	<td>Hans-Juergen Thiesen </td></tr><tr><td>20341</td><td></td><td>X27</td><td>Mapping Genetic Interactions in Various Contexts Provides Complementary Information</td>
	<td>Magali Michaut </td></tr><tr><td>20496</td><td></td><td>X30</td><td>Construction of a drug-target-disease network and its application to drug repositioning and mode of action analysis</td>
	<td>Charny Park </td></tr><tr><td>20511</td><td></td><td>X31</td><td>Finding sets of shortest elementary flux modes that decompose genome-scale metabolic flux distributions</td>
	<td>Siu Hung Joshua Chan </td></tr><tr><td>20521</td><td></td><td>X34</td><td>Different regulatory networks of SOX2 in embryonic and neural stem cells</td>
	<td>Sanghyuk Lee </td></tr><tr><td>20528</td><td></td><td>X35</td><td>Interspecies translation of disease networks increases robustness and predictive accuracy</td>
	<td>Yahya Anvar </td></tr><tr><td>20553</td><td></td><td>X38</td><td>An Integrated Approach To Identify Human MicroRNA-Targets.</td>
	<td>Kim Wan Kyu </td></tr><tr><td>20582</td><td></td><td>X42</td><td>Cholesterol-Responsive Gene Network Reconstruction from Hypercholesterolemic Mice Transcriptome Data</td>
	<td>Gustavo Hime </td></tr><tr><td>20477</td><td></td><td>X47</td><td>Integrated Inference and Analysis of Regulatory Networks Using Multi-level Measurements</td>
	<td>Christopher Poultney </td></tr><tr><td>20651</td><td></td><td>X49</td><td>Topological features predict misannotations in metabolic network</td>
	<td>Rodrigo Liberal </td></tr><tr><td>20654</td><td></td><td>X51</td><td>Candidate gene prediction using integrative and network-based approaches - Mining Pig QTL</td>
	<td>Keywan Hassani-Pak </td></tr><tr><td>20673</td><td></td><td>X52</td><td>Organizational structure of the peripheral gene regulatory network in B-cell lymphoma</td>
	<td>Ricardo de Matos Simoes </td></tr><tr><td>20713</td><td></td><td>X56</td><td>A Novel Reverse Engineering Method for Gene/Protein Network Reconstruction - Divergence Weighted Independence Graphs</td>
	<td>Yang Xiang </td></tr><tr><td>20714</td><td></td><td>X57</td><td>Joint Dynamic Networks of miRNA and Transcripton Factor Regulation</td>
	<td>Marcel Schulz </td></tr><tr><td>20740</td><td></td><td>X58</td><td>KeyPathwayMiner - Detecting case-specific biological pathways by using expression data</td>
	<td>Hande Kucuk </td></tr><tr><td>20749</td><td></td><td>X59</td><td>Finding Consistent Explanations for Observations from Genome-Wide Mutant Assays</td>
	<td>Deborah Chasman </td></tr><tr><td>20786</td><td></td><td>X61</td><td>Delineating signaling network from cancer related pathways</td>
	<td>Chandrajit Lahiri </td></tr><tr><td>20796</td><td></td><td>X63</td><td>A network topology-based machine learning approach to extract relevant cancer-related signaling subnetworks</td>
	<td>Marcio Acencio </td></tr><tr><td>20802</td><td></td><td>X65</td><td>HotRegion: A web server of cooperative hot spots</td>
	<td>Engin Cukuroglu </td></tr><tr><td>20803</td><td></td><td>X66</td><td>GraphCrunch 2 Network Analysis Tool and MI-GRAAL Network Aligner</td>
	<td>Natasa Przulj </td></tr><tr><td>20804</td><td></td><td>X67</td><td>Minimum Curvilinearity to address high-throughput protein-protein interaction experiments</td>
	<td>Gregorio Alanis-Lobato </td></tr><tr><td>20828</td><td></td><td>X69</td><td>Aligning multiple time-series experiments to learn regulatory models: An application to modeling the core network in stem cell differentiation</td>
	<td>Irene Ong </td></tr><tr><td>20841</td><td></td><td>X70</td><td>Structural Analysis of Correlation Networks in Bioinformatics:  Graph Theoretic Substructures and the Corresponding Cellular Functions</td>
	<td>Kathryn Dempsey </td></tr><tr><td>20863</td><td></td><td>X71</td><td>Systematic, comparative network analysis related to human disease</td>
	<td>Serene Wong </td></tr><tr><td>20871</td><td></td><td>X72</td><td>Systematically mapping the druggable pathways of Saccharomyces cerevisiae</td>
	<td>Kristen Fortney </td></tr><tr><td>20868</td><td></td><td>X73</td><td>Global gene network alignment among species by using graph summarization</td>
	<td>Aika Terada </td></tr><tr><td>20851</td><td></td><td>X74</td><td>A systems approach to learning the regulatory network of T helper-17 differentiation</td>
	<td>Aviv Madar </td></tr><tr><td>20880</td><td></td><td>X76</td><td>Modeling the Spatiotemporal Pattern of Gene Expression During Somitogenesis with Maximum Entropy Deconvolution.</td>
	<td>Andrzej Kudlicki </td></tr><tr><td>20848</td><td></td><td>X77</td><td>Characterization of transcription factor and microRNA regulatory networks involved in myelination</td>
	<td>Li-Wei Chang </td></tr><tr><td>20931</td><td></td><td>X78</td><td>Extending the Yeast Metabolic Network Using Integrated System Biology Approach</td>
	<td>Paul Fisher </td></tr><tr><td>22658</td><td></td><td>X79</td><td>An Epistatic Profile of the Cell Envelope Biogenesis in Escherichia coli Evidences Extensive Genetic and Functional Crosstalk Between Processes</td>
	<td>Juan Diaz-Mejia </td></tr><tr><td>22686</td><td></td><td>X84</td><td>Correction for ascertainment bias in S. cerevisiae interactions: an insight</td>
	<td>Jonathan Dickerson </td></tr><tr><td>22688</td><td></td><td>X85</td><td>Using community structure for complex network layout</td>
	<td>Oliver Duerr </td></tr><tr><td>22722</td><td></td><td>X86</td><td>Quantifying the Response of a Biological System using Network Perturbation Amplitudes</td>
	<td>Sam Ansari </td></tr><tr><td>22741</td><td></td><td>X88</td><td>Causal Reasoning on Biological Networks: Interpreting transcriptional changes</td>
	<td>Daniel Ziemek </td></tr><tr><td>22760</td><td></td><td>X91</td><td>A multi-level Bayesian model for identifying predictive combinations of signaling network modules, proteins, and phosphopeptides for cancer treatments</td>
	<td>Ann Chen </td></tr><tr><td>22646</td><td></td><td>X94</td><td>Generative Probabilistic Models for Protein-Protein Interaction Networks – The Biclique Perspective</td>
	<td>Regev Schweiger </td></tr><tr><td>22607</td><td></td><td>Y01</td><td>Biomedical Natural Language figure Processing for Intelligent Figure Search</td>
	<td>Hong Yu </td></tr><tr><td>20365</td><td></td><td>Y04</td><td>ANDCell-ANDVisio system: knowledge extraction using text-mining in the systems biology</td>
	<td>Pavel Demenkov </td></tr><tr><td>20366</td><td></td><td>Y05</td><td>Self-Training Improves Robustness of Protein-Protein Interaction Extraction from Text</td>
	<td>Philippe Thomas </td></tr><tr><td>20385</td><td></td><td>Y06</td><td>Extraction of protein cellular localization using the Locminer text mining system</td>
	<td>Martin Krallinger </td></tr><tr><td>20438</td><td></td><td>Y07</td><td>Reflect: a browser plugin for life scientist</td>
	<td>Janos Binder </td></tr><tr><td>20573</td><td></td><td>Y08</td><td>Biomedical text mining from multiple views: information fusion and vertical</td>
	<td>xinhai Liu </td></tr><tr><td>20677</td><td></td><td>Y09</td><td>Keyword Clustering in Biomedical Information Retrieval Using Evolutionary Algorithms</td>
	<td>Viktoria Dorfer </td></tr><tr><td>20715</td><td></td><td>Y10</td><td>Automating host-pathogen interaction discovery: an HIV case study</td>
	<td>Daniel Jamieson </td></tr><tr><td>20754</td><td></td><td>Y11</td><td>Relation-mining workflows to extract animal host-microorganism interactions in free text</td>
	<td>BalaKrishna Kolluru </td></tr><tr><td>22747</td><td></td><td>Y12</td><td>Overview of the second CALBC Challenge</td>
	<td>Senay Kafkas </td></tr><tr><td>22751</td><td></td><td>Y13</td><td>An exercise in kidney factomics: From article titles to RDF Knowledgebase</td>
	<td>James Eales </td></tr><tr><td>22622</td><td></td><td>Z01</td><td>BOOST: A Boolean Representation-based Method for Detecting SNP-SNP Interactions in Genome-wide Association Studies</td>
	<td>Xiang Wan </td></tr><tr><td>22573</td><td></td><td>Z02</td><td>Linguistic analysis of genomic islands reveals a recent acquisition of genetic materials by Mycobacterium tuberculosis from alpha-Proteobacteria</td>
	<td>Oleg Reva </td></tr><tr><td>22562</td><td></td><td>Z07</td><td>Structural Biology Meets Systems Biology: A Structural Systems Biology Approach for Gauging the Systemic Effect of Single Nucleotide Polymorphisms</td>
	<td>Tammy Cheng </td></tr><tr><td>22560</td><td></td><td>Z08</td><td>In silico prediction of the calpain degradome</td>
	<td>Lawrence Wee </td></tr><tr><td>22559</td><td></td><td>Z09</td><td>A scalable signaling map identifies cross-talk between gene modules inferred from multi-parametric RNAi phenotypes</td>
	<td>Xin Wang </td></tr><tr><td>22557</td><td></td><td>Z11</td><td>PINALOG: a novel approach to align protein interaction networks – implications for function prediction and complex detection</td>
	<td>Hang Phan </td></tr><tr><td>22553</td><td></td><td>Z14</td><td>Comparing nucleotide-based and codon-based alignment of human next-gen sequencing exome data</td>
	<td>Szilveszter Juhos </td></tr><tr><td>22575</td><td></td><td>Z15</td><td>RIBER and DIBER: a software suite for crystal content analysis  in the studies of protein-nucleic acid complexes.</td>
	<td>Grzegorz Chojnowski </td></tr><tr><td>22576</td><td></td><td>Z16</td><td>SpliceGrapher: Predicting Splice Graphs from Diverse Evidence</td>
	<td>Asa Ben-Hur </td></tr><tr><td>22578</td><td></td><td>Z17</td><td>Topological features predict misannotations in metabolic networks</td>
	<td>Rodrigo Liberal </td></tr><tr><td>22579</td><td></td><td>Z18</td><td>A Theoretical Approach to Identify Indispensable Proteins of the Type III Secretion Systems of Salmonella enterica Serovar Typhimurium Strain LT2</td>
	<td>Chandrajit Lahiri </td></tr><tr><td>22583</td><td></td><td>Z19</td><td>Analysis of bacterial genes with disrupted ORFs reveals frequent utilization of Programmed Ribosomal Frameshifting (PRF) and Programmed Transcriptional Realignment (PTR).</td>
	<td>Pavel Baranov </td></tr><tr><td>22584</td><td></td><td>Z20</td><td>An Event-driven Approach to Study Operon Evolution</td>
	<td>Iddo Friedberg </td></tr><tr><td>22586</td><td></td><td>Z21</td><td>RTA and MultiRTA: Simple yet Accurate Methods for Predicting Class II MHC Epitopes</td>
	<td>Andrew Bordner </td></tr><tr><td>22587</td><td></td><td>Z22</td><td>Metagenomic Biomarker Discovery and Explanation</td>
	<td>Curtis Huttenhower </td></tr><tr><td>22585</td><td></td><td>Z25</td><td>RDyNet: bridging functional analyses and network visualization</td>
	<td>Mauro Alves Castro </td></tr><tr><td>22593</td><td></td><td>Z26</td><td>Modulators of microRNA Activity Regulate Glioblastoma Pathogenesis</td>
	<td>Xuerui Yang </td></tr><tr><td>22599</td><td></td><td>Z27</td><td>Scalable metabolic reconstruction for metagenomic data and the human microbiome</td>
	<td>Curtis Huttenhower </td></tr><tr><td>22600</td><td></td><td>Z28</td><td>Systems-level insights from modular decomposition of the yeast genetic interaction network</td>
	<td>Jeremy Bellay </td></tr><tr><td>22592</td><td></td><td>Z29</td><td>Large scale merging of microarrays. From curation to meta-curation</td>
	<td>Colin Molter </td></tr><tr><td>20312</td><td></td><td>Z30</td><td>INBIOMEDvision: Promoting and Monitoring Biomedical Informatics in Europe</td>
	<td>Victoria López Alonso </td></tr><tr><td>20525</td><td></td><td>Z31</td><td>Conveyor - A strong type workflow library</td>
	<td>Burkhard Linke </td></tr><tr><td>20591</td><td></td><td>Z32</td><td>New types of services in Mobyle 1.0</td>
	<td>Herve Menager </td></tr><tr><td>20620</td><td></td><td>Z33</td><td>An integrated core facility to go from biology to data and answers</td>
	<td>Laurent Gautier </td></tr><tr><td>20888</td><td></td><td>Z34</td><td>Toward Genomic Data Sharing Directly at the Lab Site in the New Agent-Based Infrastructure OpenKnowledge</td>
	<td>Dietlind Gerloff </td></tr><tr><td>22608</td><td></td><td>Z35</td><td>New tools for RNA 3D structure modeling: inspirations from the protein world</td>
	<td>Janusz Bujnicki </td></tr><tr><td>22604</td><td></td><td>Z36</td><td>REPAIRtoire - a database of DNA repair pathways.</td>
	<td>Kaja Milanowska </td></tr><tr><td>22603</td><td></td><td>Z37</td><td>FABIA: Factor Analysis for Bicluster Acquisition</td>
	<td>Djork-Arné Clevert </td></tr><tr><td>22621</td><td></td><td>Z38</td><td>Malaria Endemicity among Pregnant Women in Urban and Semi-Urban Area in Southwest, Nigeria</td>
	<td>Tony Ojiezeh </td></tr><tr><td>22641</td><td></td><td>Z39</td><td>Strengths and limitations of the federal guidance on synthetic DNA</td>
	<td>Jean Peccoud </td></tr><tr><td>22701</td><td></td><td>Z41</td><td>Extraction protocol optimization for the LC-MS based global metabolite profiling of grapes</td>
	<td>Matthias Scholz </td></tr><tr><td>22666</td><td></td><td>Z43</td><td>Jalview 2.7</td>
	<td>James Procter </td></tr><tr><td>22753</td><td></td><td>Z52</td><td>The reference species tree of (sequenced) life</td>
	<td>Julian Gough </td></tr></table><br><br><h3>LBR</h3><table border=1><tr><td>LBR01</td><td>Milana Frenkel-Morgenstern</td><td>Potential function of proteins encoded by chimeric transcripts</td><td>Sunday, July 17 10:45 a.m. - 11:10 a.m. </td>
<td></td></tr><tr><td>LBR04</td><td>David Gfeller</td><td>The multiple Specificity Landscape of Peptide Recognition Domains</td><td>Sunday, July 17 4:00 p.m. - 4:25 p.m. </td>
<td></td></tr><tr><td>LBR06</td><td>Yinyin Yuan</td><td>Quantitative analysis of cellular composition in primary breast tumours deconvolutes molecular signatures and elucidates impact of tumour microenvironment</td><td>Sunday, July 17 3:00 p.m. - 3:25 p.m. </td>
<td></td></tr><tr><td>LBR08</td><td>Wei-Jen Chung</td><td>Post-transcriptional regulators of microRNA biogenesis regulate pathogenesis of cancer</td><td>Sunday, July 17 4:00 p.m. - 4:25 p.m. </td>
<td></td></tr><tr><td>LBR10</td><td>Tamir Tuller</td><td>A Comprehensive Computational Model for Analyzing Gene Translation</td><td>Monday, July 18 11:15 a.m. - 11:40 a.m. </td>
<td></td></tr><tr><td>LBR11</td><td>Ana Conesa</td><td>Differential expression with RNA-seq: a matter of Depth</td><td>Monday, July 18 11:45 a.m. - 12:10 p.m. </td>
<td></td></tr><tr><td>LBR12</td><td>Van Anh Huynh-Thu</td><td>Inferring Gene Regulatory Networks from Expression Data using Tree-based Methods</td><td>Monday, July 18 12:15 p.m. - 12:40 p.m. </td>
<td></td></tr><tr><td>LBR14</td><td>Pavel Baranov</td><td>Analysis of bacterial genes with disrupted ORFs reveals frequent utilization of Programmed Ribosomal Frameshifting (PRF) and Programmed Transcriptional Realignment (PTR).</td><td>Monday, July 18 3:00 p.m. - 3:25 p.m. </td>
<td></td></tr><tr><td>LBR15</td><td>Nicola Segata</td><td>Metagenomic Biomarker Discovery and Explanation</td><td>Monday, July 18 3:30 p.m. - 3:55 p.m. </td>
<td></td></tr><tr><td>LBR17</td><td>Egor Dolzhenko</td><td>STAGR: Software To Annotate Genome Rearrangement</td><td>Tuesday, July 19 10:45 a.m. - 11:10 a.m. </td>
<td></td></tr><tr><td>LBR21</td><td>Roman Briskine</td><td>Co-expression network analysis reveals rewiring of maize transcriptome by domestication</td><td>Tuesday, July 19 2:30 p.m. - 2:55 p.m. </td>
<td></td></tr></table><br><br><h3>Tech Track</h3><table border=1><tr><td>TT01</td><td>Michael Schuster</td>
<td>Ensembl Genome Browser: Harnessing the Next Generation Sequencing Wave</td>
	<td>Sunday, July 17 10:45 a.m. - 11:10 a.m. </td><td></td></tr><tr><td>TT02</td><td>Fan Meng and  Barbara Mirel</td>
<td>PubAnatomy: A Rich Internet Application for Integrated Literature and Data Exploration</td>
	<td>Sunday, July 17 10:45 a.m. - 11:10 a.m. </td><td></td></tr><tr><td>TT03</td><td>Rachel Harte</td>
<td>UCSC Genome Browser:  Visualization tools for high-throughput sequencing</td>
	<td>Sunday, July 17 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT04</td><td>Michael Reich</td>
<td>GenomeSpace</td>
	<td>Sunday, July 17 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT06</td><td>Hienke Sminia</td>
<td>Bioinformatics@school</td>
	<td>Sunday, July 17 11:45 a.m. - 12:10 p.m. </td><td></td></tr><tr><td>TT07</td><td>Zhong Li</td>
<td>Supporting Inter-connected Biomedical Open-source Software</td>
	<td>Sunday, July 17 12:15 p.m. - 12:40 p.m. </td><td></td></tr><tr><td>TT08</td><td>Sanghyuk Lee</td>
<td>MONGKIE: Modular Network Generation and Visualization Platform with Knowledge Integration Environment</td>
	<td>Sunday, July 17 2:30 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT09</td><td>Attila Berces</td>
<td>Cloud Computing for Computational Biology</td>
	<td>Sunday, July 17 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT11</td><td>George Vacek</td>
<td>Efficient Graph Based Assembly of Short-Read Sequences on a Hybrid Core Architecture</td>
	<td>Sunday, July 17 3:30 p.m. - 4:25 p.m. </td><td></td></tr><tr><td>TT13</td><td>Scott Markel,  Ruth Dunn</td>
<td>Analyzing Next Gen Sequencing Data with Pipeline Pilot</td>
	<td>Monday, July 18 10:45 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT14</td><td>Janez Konc</td>
<td>ProBiS: a web server for detection of protein binding sites</td>
	<td>Monday, July 18 10:45 a.m. - 11:10 a.m. </td><td></td></tr><tr><td>TT15</td><td>Katy Wolstencroft</td>
<td>RightField: The Semantic Annotation of Experimental Data using Spreadsheets</td>
	<td>Monday, July 18 10:45 a.m. - 11:10 a.m. </td><td></td></tr><tr><td>TT16</td><td>Jo McEntyre</td>
<td>UK PubMed Central: a database of life science research articles.</td>
	<td>Monday, July 18 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT17</td><td>Paul Fisher</td>
<td>Software for the data-driven researcher: Annotation of Next Generation Sequencing SNP Data using scientific workflows</td>
	<td>Monday, July 18 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT18</td><td>Irina Dalah</td>
<td>GeneCards: The Human Gene ReSearcher</td>
	<td>Monday, July 18 11:45 a.m. - 12:10 p.m. </td><td></td></tr><tr><td>TT19</td><td>Elissa Chesler</td>
<td>Integrating Heterogeneous Data using the Ontological Discovery Environment</td>
	<td>Monday, July 18 11:45 a.m. - 12:10 p.m. </td><td></td></tr><tr><td>TT22</td><td>Julian Gough</td>
<td>SUPERFAMILY</td>
	<td>Monday, July 18 12:15 p.m. - 12:40 p.m. </td><td></td></tr><tr><td>TT23</td><td>Kostas Karasavvas</td>
<td>Enacting Taverna Workflows through Galaxy</td>
	<td>Monday, July 18 12:15 p.m. - 12:40 p.m. </td><td></td></tr><tr><td>TT24</td><td>Peter Rice</td>
<td>EMBOSS: New developments and extended data access</td>
	<td>Monday, July 18 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT25</td><td>Guy Yachdav</td>
<td>PredictProtein sequence analysis in the cloud</td>
	<td>Monday, July 18 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT26</td><td>Peter Rose</td>
<td>Become an Expert User of the RCSB Protein Data Bank Website and Web Services</td>
	<td>Monday, July 18 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT27</td><td>Allan Orozco</td>
<td>The Spanish National Bioinformatics Institute (INB)</td>
	<td>Monday, July 18 3:00 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT28</td><td>Michael Remmert</td>
<td>HHblits: Lightning-fast iterative sequence searching by HMM-HMM comparison</td>
	<td>Monday, July 18 3:00 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT30</td><td>Mickael Goujon</td>
<td>Using the EMBL-EBI services through the JDispatcher framework</td>
	<td>Monday, July 18 3:30 p.m. - 3:55 p.m. </td><td></td></tr><tr><td>TT31</td><td>Eija Korpelainen</td>
<td>Chipster 2.0: User-friendly NGS data analysis software with interactive genome browser</td>
	<td>Monday, July 18 3:30 p.m. - 3:55 p.m. </td><td></td></tr><tr><td>TT32</td><td>Esther Schmidt</td>
<td>GenomeRNAi: A Phenotype Database for Large-scale RNAi Screens</td>
	<td>Monday, July 18 3:30 p.m. - 3:55 p.m. </td><td></td></tr><tr><td>TT33</td><td>Mickael Goujon</td>
<td>EBI Search: a new way to explore biology</td>
	<td>Monday, July 18 4:00 p.m. - 4:25 p.m. </td><td></td></tr><tr><td>TT34</td><td>Ray Fergerson</td>
<td>Introduction to National Center for Biomedical Ontology (NCBO) Technology</td>
	<td>Monday, July 18 4:00 p.m. - 4:25 p.m. </td><td></td></tr><tr><td>TT35</td><td>Jean Peccoud</td>
<td>GenoCAD: A Web-Based Open Source Application to Design Synthetic DNA Molecules</td>
	<td>Tuesday, July 19 3:00 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT36</td><td>Benoit Bely</td>
<td>Overview of The Universal Protein Resource (UniProt)</td>
	<td>Tuesday, July 19 10:45 a.m. - 11:10 a.m. </td><td></td></tr><tr><td>TT37</td><td>Daniel Blankenberg</td>
<td>NGS Best Practices through Galaxy: Cloud-based variant discovery with visual analytics</td>
	<td>Tuesday, July 19 10:45 a.m. - 11:10 a.m. </td><td></td></tr><tr><td>TT39</td><td>Jinbo Xu</td>
<td>RaptorX: an integrated web server for protein sequence-structure alignment and protein structure prediction</td>
	<td>Tuesday, July 19 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT40</td><td>Matthew Bellgard</td>
<td>YABI: A sophisticated Internet-based cross–omics analytic environment</td>
	<td>Tuesday, July 19 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT41</td><td>Desmond Higgins</td>
<td>Clustal Omega for Protein Multiple Sequence Alignment</td>
	<td>Tuesday, July 19 11:15 a.m. - 11:40 a.m. </td><td></td></tr><tr><td>TT42</td><td>Craig McAnulla</td>
<td>Introducing InterProScan 5</td>
	<td>Tuesday, July 19 11:45 a.m. - 12:10 p.m. </td><td></td></tr><tr><td>TT43</td><td>Matus Kalas</td>
<td>EDAM ontology and BioXSD: The unifying semantics and syntax for bioinformatics tools</td>
	<td>Tuesday, July 19 11:45 a.m. - 12:10 p.m. </td><td></td></tr><tr><td>TT44</td><td>Seán O'Donoghue</td>
<td>Killer App Award</td>
	<td>Tuesday, July 19 11:45 a.m. - 12:40 p.m. </td><td></td></tr><tr><td>TT45</td><td>Shulei Sun</td>
<td>CAMERA: Metagenomic Resources and Tools</td>
	<td>Tuesday, July 19 12:15 p.m. - 12:40 p.m. </td><td></td></tr><tr><td>TT46</td><td>Keywan Hassani-Pak</td>
<td>Knowledge-driven QTL analysis and network-based candidate gene discovery with Ondex</td>
	<td>Tuesday, July 19 12:15 p.m. - 12:40 p.m. </td><td></td></tr><tr><td>TT47</td><td>Fady Mohareb</td>
<td>SORF, a web application for heterogeneous data management and visualisation</td>
	<td>Tuesday, July 19 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT49</td><td>Patricia Whetzel</td>
<td>NCBO Web Services and Development of Semantic Applications</td>
	<td>Tuesday, July 19 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT50</td><td>Arek Kasprzyk</td>
<td>OncoPortal, a data management and analysis framework for cancer research</td>
	<td>Tuesday, July 19 2:30 p.m. - 2:55 p.m. </td><td></td></tr><tr><td>TT51</td><td>Peter Karp</td>
<td>Recent Developments in the Pathway Tools Software and BioCyc Databases</td>
	<td>Tuesday, July 19 3:00 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT52</td><td>Pål Puntervoll</td>
<td>eSysbio - an online system for interdisciplinary life science research</td>
	<td>Tuesday, July 19 3:00 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT54</td><td>Junjun Zhang</td>
<td>BioMart: A data federation framework for large collaborative projects</td>
	<td>Tuesday, July 19 3:00 p.m. - 3:25 p.m. </td><td></td></tr><tr><td>TT55</td><td>Rajarajan Swaminathan</td>
<td>Database on Potential “Antibiotics” From Flowering Plants</td>
	<td>Tuesday, July 19 3:30 p.m. - 3:55 p.m. </td><td></td></tr><tr><td>TT56</td><td>Geoffrey Barton</td>
<td>Jalview 2.7</td>
	<td>Tuesday, July 19 3:30 p.m. - 3:55 p.m. </td><td></td></tr><tr><td>TT57</td><td>Claire O'Donovan</td>
<td>The International Society for Biocuration and the efforts of its members</td>
	<td>Tuesday, July 19 3:30 p.m. - 3:55 p.m. </td><td></td></tr><tr><td>TT58</td><td>Nicolas Alcaraz</td>
<td>KeyPathwayMiner - Detecting case-specific biological pathways by using expression data</td>
	<td>Tuesday, July 19 4:00 p.m. - 4:25 p.m. </td><td></td></tr><tr><td>TT59</td><td>Jeremy Goecks</td>
<td>The Galaxy Track Browser: Transforming the Genome Browser from Visualization Tool to Analysis Tool</td>
	<td>Tuesday, July 19 4:00 p.m. - 4:25 p.m. </td><td></td></tr></table>
