
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=B by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist3a9a.html?cat=B by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:17 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'B'- Biophysics '</b></td>
 </tr>
</table>

<a name='B1'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B1</strong></td>
 </tr> <tr> 
  <td><b>Structure-based Prediction of Natural Residue Covariation Using Computational Protein Design</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Noah Ollikainen <i>University of California, San Francisco</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Residue covariation is a key feature of naturally occurring protein sequences, where, in the simplest pairwise case, the likelihood of observing a specific amino acid residue at a given position is dependent on the identity of the amino acid at a second position. Previous, large-scale computational studies have used information theory to detect residue covariation in multiple sequence alignments, revealing statistically significant instances of residue covariation in hundreds of protein families. Despite the clear occurrence of residue covariation in nature, it has not been systematically tested whether computational protein design methods can capture this property of natural sequences. Here we evaluate state-of-the-art protein design methods based on their ability to recapitulate natural covariation for a large set of protein domains. We find that modeling backbone flexibility significantly improves the accuracy of predicting residue covariation when compared to fixed backbone protein design and we explore how the magnitude and mechanism of structural variation affects covariation prediction. Successful prediction of residue covariation by structure-based simulations both demonstrates accurate modeling of residue interactions and reveals pairs of residues that have likely undergone coevolution in order to maintain protein stability. Interestingly, pairs of residues that covary within sets of designed sequences but not in natural sequences suggest candidate regions for engineering proteins with novel residue interactions whose sequences may be inaccessible via natural protein evolutionary pathways. Modeling novel residue interactions is critical for protein design applications such as enzyme design or altering the specificity of protein-protein interactions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B2'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B2</strong></td>
 </tr> <tr> 
  <td><b>Free energy and flexibility basis of SNP occurrence</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Luciana Oliveira <i>Universidade Federal de Ouro Preto</i></td>
 </tr>
 <tr> 
  <td>  Gerald Weber (Universidade Federal de Minas Gerais, Departamento de Física); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Why do some types of SNPs occur much more frequently than others? Clearly, the fact that transitions, where a nucleotide is replaced by another of the same type, make out 2/3 of all SNPs is a sure sign that there is a molecular basis for SNP occurrence. It is also known that the immediate vicinity of the point mutation plays a role in SNP occurrence. But which other physical parameters of oligonucleotides would correlate positively with mutation rates? Here we analyse all major organisms for which a large number SNPs were sequenced and deposited in SNPdb, and group them in order of Gibbs free energy and DNA flexibility. The physical parameters used in this case are those of the mismatch base pairs assumed to be involved in causing a given SNP. The flexibilities for mismatched base pairs were calculated following our recent work on matched DNA [Weber et al., Nature Physics v.5, p.769, 2009]. We found that for some organisms there is a linear correlation of SNP occurrence and free energy, in some cases with higher free energies giving rise to twice as many SNPs than for lower energies. Yet, some other species do not display any significant correlation at all suggesting that the detailed distribution of SNPs can not be attributed solely to the nature of the DNA molecule. We also determined that for many organisms SNPs appear much more often when the mismatched base pair is much more rigid than undamaged DNA. Support: CNPq, Capes and Fapemig.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B3'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B3</strong></td>
 </tr> <tr> 
  <td><b>Pipeline for the training of NMR chemical shift prediction models</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Anna Katharina Dehof <i>Saarland University</i></td>
 </tr>
 <tr> 
  <td>  Andreas Hildebrandt (Johannes-Gutenberg Universität Mainz, Computer Science); Hans-Peter Lehof (Saarland University, Center for Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: NMR chemical shift prediction plays an important role in many applications<br>in computational biology. Among others, structure determination, structure optimization, and the scoring of docking results can profit from an efficient and accurate chemical shift estimation from a three-dimensional model of the molecule under consideration.<br>The development of novel prediction techniques is a challenging task. The required information is spread over several databases and stored in hard-to-parse file formats which sometimes contain serious errors. In addition, the computation of physical terms or of molecular features for a heuristic approach requires complex molecular datastructures and algorithms.<br><br>Here, we present a pipeline for developing novel hybrid NMR chemical<br>shift prediction methods that combine physical terms -- approximations to quantum mechanical effects -- with a statistical model. The pipeline allows the simple import of data from diverse sources, such as the BMRB and the PDB. Several semi-classical terms for shift prediction are implemented and readily available. As of now, we include<br>random coil contributions, aromatic ring current effects, electric field contributions, and hydrogen bonding effects. The feature set for the training of the statistical term encompasses sequential, structural (angles, surface, and density),<br>force-field based, and experimental properties. All features are computed using our open source library BALL, and can be easily extended.<br><br>For the statistical contribution we propose a random forest model which has demonstrated in our experiments to yield very accurate and stable results. In general, however, the pipeline is model-agnostic and can be used with any regression technique implemented in R.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B4'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B4</strong></td>
 </tr> <tr> 
  <td><b>Modeling of Actomyosin Interactions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Cynthia Prudence <i>University of Rhode Island</i></td>
 </tr>
 <tr> 
  <td>  Vladislav Markin (UT Southwester Medical Center, Department of Neurology); Yana Reshetnyak (University of Rhode Island, Physics Department); Oleg Andreev (University of Rhode Island, Physics Department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Muscle contraction is the result of the interaction of myosin with actin and ATP. The formation of this actomyosin interface may play a role in the generation of force in muscles. Kinetic studies of binding between myosin sub-fragment 1 and F-actin revealed a two-step binding process. Chemical cross-linking of S1 and F-actin demonstrated that the myosin head initially binds via the loop 635-647 to the N-terminus of one actin and then via the loop 567-574 to the N-terminus of the second actin. Depending on degree of saturation of F-actin with S1s, two structurally different complexes are formed: at complete saturation each S1 binds only one actin and its cleft is closed while at partial saturation S1 interacts with two actins and its cleft is opened. The transition between one- and two-actin binding states of myosin accompanying with opening the cleft in central domain of S1 might be associated with force generation. Early computational docking of S1 with F-actin demonstrated that both actin monomers are located in the same strand of F-actin. As such, we theorize that the formation of the actomyosin complex is a sequential multi-step process that begins with myosin weakly binding to the N-terminus of one actin then rotating towards the barbed end of F-actin to create a second stronger bond to the N-terminus of the second actin on the same strand. To investigate the formation of this complex, we will use experimental data to guide protein-protein docking techniques.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B5'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B5</strong></td>
 </tr> <tr> 
  <td><b>Fitting Comparative Models to X-ray Data with Kino-Geometric Sampling</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Henry van den Bedem <i>SLAC National Accelerator Center</i></td>
 </tr>
 <tr> 
  <td>  Peggy Yao (Stanford University, Computer Science Department); Liangjun Zhang (Stanford University, Computer Science Department); Ashley Deacon (SLAC National Accelerator Laboratory, Stanford Synchrotron Radiation Lightsource); Jean-Claude Latombe (Stanford University, Computer Science Department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: For the PSI:Biology initiative in the USA, which uses high throughput structural genomics to address significant problems in biomedicine, the proteins of interest are often large, multi-domain macromolecules or multi-protein assemblies. Such large systems are more difficult to crystallize and often yield poorly ordered crystals that diffract to limited resolution. Fortunately, structural coverage of the protein universe is reaching a point at which leveraging structural similarities can augment traditional structure determination techniques, especially in the presence of partial experimental data sets.<br><br>We present the application of a fast algorithm that samples the folded state of a protein to fitting a comparative model into experimental data. The Kino-Geometric Sampler with X-ray data, (KGSx) encodes dominant energy terms implicitly by kinematic and geometric constraints. Key features of KGSx are a robotics-inspired null-space method to simultaneously deform a large number of interdependent kinematic cycles without breaking covalent or favorable hydrogen bonds and a strategy to sample the folded state uniformly. These features result in a large radius of convergence for the optimization algorithm. We will furthermore show how the algorithm is used to build an accurate  consensus model in a high throughput structure determination pipeline by combining it with output from traditional structure determination techniques.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B6'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B6</strong></td>
 </tr> <tr> 
  <td><b>Insights to the Thrombospondin Signature Domain</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Thomas Haschka <i>Université de Reims Champagne Ardennes</i></td>
 </tr>
 <tr> 
  <td>  Laurent Martiny (CNRS, UMR 6237); Manuel Dauchez (CNRS, UMR 6237); Cathrine Etchebest (INSERM, UMRS 665); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Thrombospondin's C-terminal region plays a key role in PSACH disease, as well as in cancer research. Mutations in the calcium binding stalk was shown to be responsible for PSACH a decade ago. Using molecular dynamics we are modeling the deformation of the protein in cases of calcium depletion/repletion. The poster highlights the key features of the signature domain as found by our molecular dynamics simulations, such as integrin binding site accessibility, deformation and flexibility.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B7'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B7</strong></td>
 </tr> <tr> 
  <td><b>Universal patterns of protein-ligand interactions: Gaussian Mixture Model analyses on three dimensional structure of proteins in Protein Data Bank.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kota Kasahara <i>Tohoku University</i></td>
 </tr>
 <tr> 
  <td>  Kengo Kinoshita (Tohoku University, Graduate School of Information Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Illuminating mechanisms of protein-ligand interaction is one of the most important subjects in the structural biology, and many analyses have been carried out by using known complex structures in PDB, but universal rule of interactions have not been elucidated. In particular, universal mechanism to achieve the selective recognition of ligand was not clarified. Here, we present an analysis focusing on “universality” of molecular interactions in PDB. At first, we divided proteins and ligands into successive three atoms, called fragments, and then patterns of interactions were investigated by using a pattern recognition technique based on the Gaussian mixture model. Finally, frequency of each pattern was analyzed by considering the diversity of amino-acid sequences and that of ligand chemistries in each pattern.<br>As a result, 18,608 patterns were obtained from 2,970 and 936 types of protein- and ligand-fragments, respectively. According to the analyses of sequence- and ligand-diversity, some distinct features were found; For example, when we focused on atom types in ligand-fragments, patterns of aromatic carbon atoms in ligand-fragments were distributed among diverse ligand types and small number of protein families, but those of oxygen atom in a ligand-fragment were exhibited in wide variety of proteins and relatively small variation of ligand types. However, when we focused on atom types in protein-fragments, such differences were not observed. This example implies that, three-atom ligand-fragment can select partner of interactions, but protein-fragment can’t do it. The selectivity of local interactions is arisen from larger part of protein structure, such as a combination of amino-acids.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B8'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B8</strong></td>
 </tr> <tr> 
  <td><b>Theoretical study for charge separate state of artificial photosynthesis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ryota JONO <i>The university of Tokyo</i></td>
 </tr>
 <tr> 
  <td>  Koichi Yamashita (The university of Tokyo, Chemical System Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Photo-induced charge separation (CS) between donor and acceptor is a key process for photo-energy conversion systems such a photosynthesis and solar cells. It is important to elucidate the mechanism of CS procedure for understanding the life and solving the sustainable energy problem. Chlorophyll and quinone are the most principle actor in photosynthesis as electron donor and acceptor. A chemical species ZnP-BQ which is consisted of zinc-porphyrin and benzoquinone is designed by using the abundant parts in biochemical systems and is known as an artificial photosynthesis molecule. It was observed that the CS state of ZnP-BQ is generated from relaxation of the Q-state of zinc-porphyrin moiety by time-resolved transient absorption spectra in benzene solution. The energy of CS state and Q-state were 1.53 and 2.08 eV, respectively. However, quantum chemical calculation of ZnP-BQ shows that the energy of CS states were higher than Q state by 2 eV. Therefore the electron transfer from Q state to CT state was inhibited in this optimized structure. This results may be due to the long distance between donor and acceptor (about 12.7 Å). In this paper, we will introduce potential energy curves with respect to four dihedral angles and distance between zinc and oxygen using various density functional and symmetry-adapted-cluster configuration interaction method, and show the conclusion that a dimer creation is necessary to reproduce the experimentally determined energy of CS states.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='B9'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster B9</strong></td>
 </tr> <tr> 
  <td><b>Intrinsically disordered proteins/regions: an important factor in protein evolvability</b><br/><br/></td>
 </tr>
 <tr> 
 <td>He Huang <i>Kyushu Institute of Technology</i><br></td>
 </tr>
 <tr> 
  <td>Akinori Sarai (Kyushu Institute of Technology) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: More and more studies have revealed that the protein evolvability is not only restricted by functional and structural importance, but also affected by other factors, such as gene duplication, protein stability and organism’s robustness. There is now an increasing need to better understand the protein evolvability. Here, we examine an emerging concept that intrinsically disorder proteins (IDPs)/regions (IDRs) may play a role to facilitate protein evolution. In this regard, we have made a systematic comparative analysis of IDPs/IDRs. Evolutionary analysis shows that recently emerged IDRs have high evolutionary rates relaxing more functional constraints (or experiencing more positive selection), and this may have caused accelerated evolution in the flanking regions and the whole protein. In addition, we carried out a systematic analysis of observed stability changes due to single amino acid mutations in IDRs and ordered regions. Our results show that most mutations induce a destabilizing effect to proteins and the mutations in IDRs cause less stability changes than in the ordered regions. The weaker impact of mutations in IDRs to the protein stability may have advantage for the evolvability to gain new functions. Our study will provide insight into better understanding the role of IDPs/IDRs in the mechanisms of protein evolability and functional diversification.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




