
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=S by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist5c37.html?cat=S by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:19 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'S'- Regulation'</b></td>
 </tr>
</table>

<a name='S01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S01</strong></td>
 </tr> <tr> 
  <td><b>Gene Array Analyzer (GAA): Alternative usage of gene arrays to study alternative splicing events</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pascal Gellert Max-Planck-Institute <i>Max-Planck-Institute Bad Nauheim</i><br></td>
 </tr>
 <tr> 
  <td>Pascal Gellert (Max-Planck-Institute Bad Nauheim) Mizue Teranishi (Max-Planck-Institute Bad Nauheim, Cardiac Development and Remodelling); Katharina Jenniches (Max-Planck-Institute Bad Nauheim, Cardiac Development and Remodelling); Piera De Gaspari (Max-Planck-Institute Bad Nauheim, Cardiac Development and Remodelling); David John (Max-Planck-Institute Bad Nauheim, Cardiac Development and Remodelling); Karsten grosse Kreymborg (University of Giessen, Lung Center); Thomas Braun (Max-Planck-Institute Bad Nauheim, Cardiac Development and Remodelling); Shizuka Uchida (Max-Planck-Institute Bad Nauheim, Cardiac Development and Remodelling); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A recent study indicates that nearly every gene has at least one alternative isoform [1]. Although microarrays are widely used in the research community, so-called 3'-arrays are not capable of detecting alternatively spliced genes. To circumvent this problem, Affymetrix, Inc. introduced a new type of arrays called "GeneChip Exon 1.0 ST Array (exon arrays)", which can be analyzed by our web interface  called "Exon Array Analyzer (EAA)" [2]. The same company released another type of arrays called "GeneChip Gene 1.0 ST Array (gene arrays)", which covers all the known exons at the half of the price of an exon array. Here, we introduce an extension to EAA called "Gene Array Analyzer (GAA)". GAA enables to analyze gene arrays at both gene and exon levels and is equipped with various filters to reduce false positive results and graphical viewers to visualize splicing events. To test the applicability of GAA, both gene- and exon-level analyses of heart development and ES-derived cardiomyocytes were performed to show gene and isoform switches focused on heart. Furthermore, re-analyses of published gene arrays indicated that some of the previously identified splicing events reoccur under pathological conditions. GAA is a valuable tool for biologists to analyze their gene array experiments for alternative splicing without any additional costs. The web interface of GAA is user friendly, without a need for set up and freely available at http://GAA.mpi-bn.mpg.de.<br><br><br>References<br><br>[1] Wang, E.T. et al., Nature 456: 470-6.<br>[2] Gellert, P. et al., Bioinformatics 25: 3323-4.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S02</strong></td>
 </tr> <tr> 
  <td><b>Cell specific gene expression from histome modifications and transcription factor binding</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Thaís Rêgo <i>UFPE</i></td>
 </tr>
 <tr> 
  <td>  Ivan Costa (UFPE, Centro de Informática); Francisco Carvalho (UFPE, Centro de Informática); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In eukaryotes, chromosomes are arranged in chains of nucleosomes, which consist of the DNA strand folded around octamer of four histone proteins. Modifications of these structures are involved in many biological processes including transcriptional regulation, key event in the process of differentiation of stem cells to form different cell types. Thus, this work proposes models that explain the process of differentiation of CD4 + cells from gene expression data and promoter regions of genes, as well as modification and positioning of nucleosome  and motifs of transcriptional regulators.Therefore, we analyzed the behavior of gene expression in four different stages in the differentiation of CD4 + cells: hematopoietic stem cell (HSC), multipotent progenitor (MPP), megakaryocite / erytrhocyte progenitors (PreMegE) and T cells differentiated under different modifications their histones: H3K4me3, H3K79me2, H3ac, H3K9me3, H3K27me3 and binding of RNA polymerase II in Mus musculus based on the work of Holger (2009). Our approach aims to find a mixture of linear regression model, which explain cell specific gene expression from histome modifications and transcription factor binding. We found several relationships between the regulatory factors in specific stages of the process of cell differentiation, indicating their role in transcriptional regulation of the formation of these cells.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S03</strong></td>
 </tr> <tr> 
  <td><b>Analyzing the Cardiac Transcription Network Driven by the Interplay of Transcription Factors, Histone Modifications and microRNAs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marcel Grunert <i>Max Planck Institute for Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>  Markus Schueler (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Jenny Schlesinger (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Ilona Dunkel (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Silke R. Sperling (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The transcriptome is regulated by the interaction between different molecular levels, involving epigenetic, transcriptional and post-transcriptional mechanisms. However, many previous studies investigated each of these levels individually and little is known about their interdependency. <br><br>We present a systems-level analysis integrating microarray and next-generation sequencing data of DNA-binding events of four key cardiac transcription factors (Gata4, Mef2a, Nkx2.5 and Srf), activating histone modifications and mRNA as well as genome-wide microRNA profiles obtained in wildtype and RNAi mediated knockdown. We confirmed conclusions primarily obtained in cardiomyocyte cell culture in a time-series of cardiac maturation in mouse around birth.<br><br>We provide insights for the combinatorial regulation by transcription factors and show that they can in part compensate each other’s function. Genes regulated by multiple transcription factors are less likely differentially expressed in RNAi knockdown of one respective factor. Further, we show that histone 3 acetylation has an significant impact on the regulation of Srf and Gata4 dependent genes, and moreover buffers the downregulation of Srf target gene expression in its knockdown. We found that the overwhelming proportion of differentially expressed genes in our RNAi experiments were indirect targets of the respective transcription factor. We analyzed microRNAs as major regulatory factors for this indirect regulation and show on the example of Srf, that they have the potential to explain a large fraction of the altered mRNA profiles.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S04</strong></td>
 </tr> <tr> 
  <td><b>Dynamics of histone modifications and transcription factor binding during cardiac maturation in mice</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Markus Schueler <i>Max Planck Institute for Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>  Qin Zhang (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Jenny Schlesinger (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Martje Tönjes (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); Silke R. Sperling (Max Planck Institute for Molecular Genetics, Vertebrate Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The adaptation of the cellular network to functional changes, timing and patterning of gene expression is regulated by binding of transcription factors to gene promoters. This binding and the activity of each transcription factor is in turn dependent on chromatin state, which is determined by the presence of histone modifications. These two layers of regulation can influence each other, enabling a further level of regulatory fine-tuning.<br><br>We analyzed the dependency between the enrichment of histone 3 acetylation (H3ac) and histone 3 lysine 4 (H3K4me2) dimethylation on binding changes of the key transcription factors Srf and the histone acetyltransferase p300 in an in vivo model using chromatin immunoprecipitation (ChIP) in a time-series of cardiac maturation in mice.<br><br>In line with high regulatory interdependencies, we found strong correlation between changes in the enrichment of the two histone modifications, binding of the two transcription factors as well as between these two regulatory levels. Using linear modeling techniques we show that each of the transcription factors contribute individually as well as conjointly to the regulation of H3ac, probably aided by accompanying H3K4me2. We further demonstrate that changes in gene expression during cardiac maturation are attended by changes of the analyzed regulators while revealing a high variability of combinatorial regulation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S05</strong></td>
 </tr> <tr> 
  <td><b>Induction of Effectors in Diuraphis noxia after Feeding on Preference and Non-preference Wheat Hosts</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nicolaas Burger <i>University of Stellenbosch</i></td>
 </tr>
 <tr> 
  <td>  Anna-Maria Botha-Oberholster (University of Stellenbosch, Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Russian Wheat Aphid (RWA), Diuraphus noxia, is a major insect pest of wheat in all wheat producing countries except Australia.  Feeding of RWA on wheat induces chloratic streaking, leaf rolling and death in susceptible varieties, whereas localized necrosis is observed in resistant varieties as a result of the Hypersensitivity Response (HR) being induced.  Evidence supports that this host/pest interaction works on a gene for gene model, which signifies that there should be a unique elicitor in RWA saliva driving this aberrant interaction.  Activation of the HR is usually associated with bacterial and fungal infections so identifying the RWA elicitor that provokes this unusual behaviour is of the utmost importance for breeding resistant cultivars.  Two different South African RWA biotypes were transferred between 15 varying host plants, either susceptible (preference host) or resistant (non-preference host), to elicit a change in their salivary enzyme composition.  Aphid heads were sampled at two distinct time intervals correlating with that of the initial recognition response and that of systemic acquired resistance.  From this, RNA was extracted and cDNA synthesized which was analysed using the cDNA-Amplified Fragment Length Polymerism (AFLP) technique.  cDNA-AFLP technology is extremely powerful in analysing thousands of transcripts between samples to ascertain differential expression profiles.  Transcript derived fragments  (TDFs)obtained after cDNA-AFLP  profiling were excised,  cloned and then sequenced and analysed using bioinformatic tools for the identification of putative virulence effectors, as well as associated structural changes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S06</strong></td>
 </tr> <tr> 
  <td><b>ConTra v2: a tool to identify transcription factor binding sites                 across species, update 2011</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Stefan Broos <i>Ghent University</i></td>
 </tr>
 <tr> 
  <td>  Paco Hulpiau (Ghent University, Department of Biomedical Molecular Biology); Jeroen Galle (Ghent University, Department of Biomedical Molecular Biology); Bart Hooghe (Ghent University, Department of Biomedical Molecular Biology); Frans van Roy (Ghent University, Department of Biomedical Molecular Biology); Pieter De Bleser (Ghent University, Department of Biomedical Molecular Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Transcription factors are important gene regulators with distinctive roles in development, cell signaling, and cell cycling, and they have been associated with many diseases. The ConTra v2 web server allows easy visualization and exploration of predicted transcription factor binding sites in any genomic region surrounding coding or non-coding genes. In this new version, users can choose from nine reference organisms ranging from human to yeast. ConTra v2 can analyze promoter regions, 5’UTRs, 3’UTRs and introns or any other genomic region of interest. Hundreds of position weight matrices are available to choose from, but the user can also upload any other matrices for detecting specific binding sites. A typical analysis is run in four simple steps of choosing the gene, the transcript, the region of interest and then selecting one or more transcription factor binding sites. The ConTra v2 web server is freely available at http://bioit.dmbr.ugent.be/contrav2/index.php.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S07</strong></td>
 </tr> <tr> 
  <td><b>Pseudogenes as post-transcriptional regulators, their role in disease.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Enrique Muro <i>Max-Delbrück-Centrum für Molekulare Medizin</i></td>
 </tr>
 <tr> 
  <td>  Miguel  Andrade (Max-Delbrück-Centrum für Molekulare Medizin, Computational Biology and Data Mining); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Pseudogenes have been considered evolutionary relics. However, new evidence indicate that some pseudogenes are functional and have a role as post-transcriptional regulators, even with consequences in the translation of the parental gene. Two new concepts are now in the eye of the hurricane in the field of research of pseudogene function. First, siRNAs are expressed from pseudogenes as demonstrated genome wide experimentally in mouse oocytes [PMID:18404146 and 18404147] and computationally for human in our recent work [PMID:21047404]. Second, pseudogene transcripts can act as a decoy of post-transcriptional regulators of the parental gene, like miRNAs. In some cases, even subtle changes in the level of transcription of the pseudogene can have consequences on the expression of the parental gene. This has been experimentally demonstrated for both the tumor suppressor PTEN and the oncogene KRAS [PMID:20577206]. The observation that some pseudogenes are aberrantly transcribed in tumors suggests that this mechanism is relevant in pathogenesis. Here we add computational evidence of this second function of pseudogenes, transcriptional decoys, by analysis of the human EST/cDNA sequences from the dbEST database: we evaluate which pseudogenes are transcribed and in which tissues and conditions; the same will be done for their parental genes. We will focus on parental genes with an annotated implication in pathogenesis. The result will be a table of pseudogenes which can be potential regulators of their corresponding parental genes and can have a role in disease.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S08</strong></td>
 </tr> <tr> 
  <td><b>Unveiling combinatorial regulation through the combination of ChIP information and in silico cis-regulatory module detection</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kathleen Marchal <i>Katholieke Universiteit Leuven</i></td>
 </tr>
 <tr> 
  <td>  Hong Sun (KULeuven , Microbial and molecular systems); Tias Guns (KULeuven , Computer science); Siegfried Nijssen (KULeuven , Computer science); Kathleen Marchal (KULeuven , Microbial and molecular systems); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Nowadays with high-throughput chromatin-immunoprecipitation technologies becoming increasingly popular for the genome-wide identification of TF binding sites, cis-regulatory module detection (CRM) can be used in combination with ChIP information to computationally predict with which other TFs a ChIP-assayed TF potentially interacts. In contrast to gene centered methods, ChIP information allows reducing largely the regions in which the motif of the assayed TF should be located (typically 500 bp instead of thousands of bp). However, as the binding site of the assayed TF often not coincides with the peak location, searching for CRMs in ChIP-Seq defined regions still boils down to a combinatorial search problem. In addition, as it is not known in advance with which other TF the assayed one interacts, the CRM detection approach needs to be able to search for a CRM that can include any of the known motifs.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S09</strong></td>
 </tr> <tr> 
  <td><b>Transcriptional cross-regulation as survival mechanism in bacteria</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pieter Monsieurs <i>Belgian Nuclear Research Center (SCK•CEN)</i></td>
 </tr>
 <tr> 
  <td>  Kristel Mijnendonckx (Belgian Nuclear Research Center (SCK•CEN), Molecular and Cellular Biology); Natalie Leys (Belgian Nuclear Research Center (SCK•CEN), Molecular and Cellular Biology); Rob Van Houdt (Belgian Nuclear Research Center (SCK•CEN), Molecular and Cellular Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The high number of metal resistance genes in the soil bacterium Cupriavidus metallidurans CH34 makes it an interesting model organism to study microbial heavy metal responses. In order to reconstruct the transcriptional regulatory networks underlying heavy metal resistance, genomewide expression experiments were performed to investigate the full stress response of C. metallidurans CH34 when challenged to a variety of heavy metals including zinc, copper, cadmium, and lead. Certain heavy metal response gene clusters showed similar expression profiles when cells were found to be exposed to varying combinations of heavy metals, thus pointing to complex cross-talk at the transcriptional level between the different heavy metal resistance mechanisms. Our results could partially explain this cross-talk by identification and similarity analysis of transcription factor binding sites in the promoter region of metal resistance genes. <br>This hypothesis is further confirmed by a directed evolution experiment, exposing the bacterium to toxic concentrations of silver, resulting in mutants with an increased resistance towards this metal. Subsequent Illumina sequencing of two of these mutant strains points towards an inactivation of the sensory component of the two-component regulatory system AgrR/S. A phylogenetic footprinting approach of all already known or recently identified metalloregulators predicts a AgrR regulatory motif which resembles the regulatory motifs found in a subset of other metal resistance regions. This way, the AgrR transcription factor not only activates its own dedicated metal efflux pump AgrCBA, but also other metal resistance regions containing a similar transcription factor binding site, which indeed could be confirmed by microarray data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S10</strong></td>
 </tr> <tr> 
  <td><b>A probabilistic model for prediction of Transcription Start Regions in both intra- and inter-genic miRNAs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Annalisa Marsico <i>Max Planck for Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>  Martin Vingron (Max Planck Institute for Molecular Genetics, Computational Molecular Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Despite the fact that miRNAs regulate more than one third of all protein coding genes, it is still largely unknown how miRNAs themselves are regulated. Several studies showed that in mammals miRNA primary transcripts (pri-miRNAs) are mainly transcribed from RNA-polymeraseII, and that Transcription Start Regions (TSRs) can be located up to several kb upstream of the mature miRNA. Cap Analysis Gene Expression (CAGE) allows high-throughput identification of sequence tags corresponding to genes’ TSRs. <br><br> In this work we map CAGE tags from 36 different tissues to the human genome, in order to define tag clusters corresponding to putative TSRs in both inter- and intra-genic miRNAs. We have developed a probabilistic mixture model to determine the probability of a sequence to be a real miRNA TSR compared to random intergenic sequences. Our model combines the number of tags in both TSRs and background sequences together with the presence of CpG islands and conserved sequence elements. The results show that miRNA TSRs can be dominated by a narrow tag peak or be very broad. Our method confirmed the few known pri-miRNA transcripts annotated in Ensembl and most of the promoters predicted by means of the epigenetic mark H3K4me3 (Barski et al. 2009). In addition, we characterize novel miRNA TSRs located up to 40 kb upstream of the mature miRNA. Tag clusters located upstream or downstream of the TSR of the host gene support the hypothesis of alternative promoters for intra-genic miRNAs. The knowledge of the TSR locations allowed us to analyze putative miRNA promoters for enriched transcription binding signals.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S11</strong></td>
 </tr> <tr> 
  <td><b>Analysis of the structure and evolutionary conservation of regulatory networks in prokaryotes.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Svitlana Yarmolinets <i>University of Braunschweig</i><br></td>
 </tr>
 <tr> 
  <td>Richard Münch (University of Braunschweig) Isam  Haddad (University of Braunschweig, Institute for Microbiology ); Ida  Retter (University of Braunschweig, Institute for Microbiology ); Johannes  Klein (University of Braunschweig, Institute for Microbiology ); Ilona Biegler (University of Braunschweig, Institute for Microbiology ); Dieter  Jahn (University of Braunschweig, Institute for Microbiology ); Richard Münch (University of Braunschweig, Institute for Microbiology ); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One of the important feature of bacterial pathogens is its adaptation to grow under highly different conditions. For example Pseudomonas aeruginosa grows under both aerobic and anaerobic conditions at diffrent stages of the host-colonization. Thereby, the metabolism is adapted to denitri?cation under unaerobic conditions. Our goal is to investigate the structure and evolutionary conservation of this and related regulatory pathways of metabolic adaptation during infection. <br><br>The data set used in this study was provided by PRODORIC's (Prokaryotic database of gene-regulation) manually curated collection of transcription factor binding sites (TFBSs). <br>In a new comparative approach the regulatory networks from different species are analyzed by transfering the evolutionary conserved TFBSs to the less investigated genomes. <br><br>Changes in the structure of transcriptional regulatory networks can be very dynamic based on single point mutations within the transcription factor binding site. For this reason a high variation of regulatory interactions between related species is observed. The comparison of different species enables the generation of phylogenetic trees based on the differences of its regulatory networks. Moreover, we investigate the evolution of operons depending on changes of regulatory networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S12</strong></td>
 </tr> <tr> 
  <td><b>PRODORIC - a platform for the elucidation of gene regulatory networks in prokaryotes - from data to models</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Johannes Klein <i>TU Braunschweig</i><br></td>
 </tr>
 <tr> 
  <td>Richard Münch (TU Braunschweig) Louisa Roselius (TU Braunschweig, Institute of Microbiology); Svitlana Yarmolinets (TU Braunschweig, Institute of Microbiology); Karin Münch (TU Braunschweig, Institute of Microbiology); Dieter Jahn (TU Braunschweig, Institute of Microbiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The availability of nearly 1500 completely sequenced and annotated prokaryotic genomes offers a variety of predictive approaches on genome-scale. However, the elucidation of the corresponding gene regulatory networks is still a challenging process that requires extensive analyses on multiple levels. We present a platform for the retrieval, prediction visualization and modeling of gene regulatory networks in prokaryotes that enables an integrated and interactive analysis. The framework consists of the PRODORIC database, the Virtual Footprint DNA binding site prediction tool and the ProdoNet visualization tool. PRODORIC, the Prokaryotic Database Of GeneRegulation, provides a comprehensive source of manually curated information on regulation of gene expression in prokaryotes. The database provides integrated data of transcriptional regulators and their corresponding binding sites, gene expression patterns and related information. The pattern matching tool Virtual Footprint complements the data obtained from experimental evidence by the predictionof regulatory interactions and whole regulons in bacterial genomes. For the mapping of prokaryotic genes and corresponding proteins to common gene regulatory and metabolic networks, ProdoNetprovides an intuitive tool for the visualization of experimental and predicted data. The derived information is suitable for dierent approaches to create models for prokaryotic gene regulatory networks.In this way, we exemplary modeled and simulated the anaerobic adaptation of Pseudomonas aeruginosa using a deterministic approach. In conclusion, the ow of data is turned into information and knowledge in a process of data mining, prediction, visualization and mathematical modeling. The PRODORIC platform is accessible at http://www.prodoric.de.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S13</strong></td>
 </tr> <tr> 
  <td><b>Epigenetic regulation of microRNAs during embryonic stem cell differentiation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Wieslawa Mentzen <i>CRS4 Bioinformatics Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Matteo Floris (CRS4 , Bioinformatics Laboratory); Paolo Uva (CRS4 , Bioinformatics Laboratory); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Embryonic stem cells (ESCs) are a unique class of cells characterized by self-renewal capability and the potential to differentiate in a wide range of specialized cell types. However, the regulatory mechanisms driving the ESC differentiation are still unclear.<br>Among the mechanisms contributing to the ESC differentiation, epigenetic modifications play an essential role by altering the chromatin density and accessibility of the DNA, thus affecting the transcription regulation of the underlying genes. Aberrant patterns of the epigenetic profile have been associated to several diseases.<br>Recently, the ability of ESCs to self-renew and differentiate has also been associated to microRNAs (miRNAs). Interestingly, recent evidences have shown the direct connections between the epigenetic regulation (DNA methylation and histone modifications) and the miRNAs. While efforts to identify the impact of altered epigenetic profiles of miRNAs in cancer have been undertaken, only a few studies have explored the epigenetic regulation of miRNAs in ESCs.<br>In this study we have integrated single-base resolution DNA methylation maps, genome-wide histone modification profiles and miRNA expression data in human ESCs and fetal lung fibroblasts to identify the potential interactions between epigenetic modifications and miRNAs during ESC differentiation.<br>First, we identified the regions enriched in each chromatin modification in ESCs and differentiated cells. Then, we mapped those regions to the miRNA promoters and precursors. <br>We observed that both DNA methylation and histone modifications are able to regulate the expression of microRNAs involved in cellular differentiation.<br>Our results provide new insights into the connection between microRNAs and epigenetics during cellular differentiation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S14</strong></td>
 </tr> <tr> 
  <td><b>Identifying CNVs of regulatory elements which impact overall survival in serous cystadenocarcinoma</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Geoff Macintyre <i>University of Melbourne</i></td>
 </tr>
 <tr> 
  <td>  Adam Kowalczyk (NICTA VRL, Diagnostic Genomics); James Bailey (University of Melbourne, Computer Science and Software Engineering); Izhak Haviv (Baker IDI Heart and Diabetes Institute, Bioinformatics and Systems Integration); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We hypothesize that copy number variations (CNVs) encompassing a regulatory element can affect the expression of a target gene which is not itself part of the CNV. By incorporating CNV and gene expression data in serous cystadenocarcinoma from The Cancer Genome Atlas, we demonstrate that a statistical trend test (Jonckheere-Terpstra (Lehmann, 1975)) can be employed to identify regulatory CNVs. The trend test is used to find those individuals showing a loss, normal, or gain in copy number at the regulatory element, and lowered, normal or elevated expression, respectively, of the target gene. We derived a set of regulatory element and target gene pairs (RE/TG) from expression profiling and ChIP-SEQ of STAT1 and ERalpha in IFNgamma treated HelaS3 cells and estrogen treated MCF7 cells, respectively. While these RE/TG come from a different cell type to the CNV and expression profiling, the HelaS3 and K562 cell lines are cancer cell lines, and both STAT1 and ERalpha show evidence of activity in ovarian carcinoma (Zhang, 2010; Burges,2010). Therefore, we speculate that a subset of RE/TG interactions will occur in serous cystadenocarcinoma. Our analysis identified two cases, where a gain in CN of STAT1 binding sites increased expression of GEMIN4, and a gain in CN of ERalpha sites increased expression of DDX39. Individuals with the gain in regulatory element CN, had significantly decreased overall survival (Kaplan-Meier, Wilcoxon test p<0.01) compared to loss/normal CN. Both genes play a role in cancer progression (Sugiura, 2007,Horikawa, 2008), suggesting this approach can be used broadly across TCGA studies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S15</strong></td>
 </tr> <tr> 
  <td><b>Predicting cell type-specific transcription factor co-binding</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Aleksander Jankowski <i>Genome Institute of Singapore</i></td>
 </tr>
 <tr> 
  <td>  Jerzy Tiuryn (University of Warsaw, Faculty of Mathematics, Informatics and Mechanics); Shyam Prabhakar (Genome Institute of Singapore, Computational and Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Transcription factors are essential for the regulation of gene expression. Their activating effect is achieved by binding to the specific DNA sequence fragments in the regulatory regions of the genome. Numerous experiments have acknowledged that transcription factors tend to bind close to each other, and their combinatorial effect is highly cooperative.<br><br>We are making use of DNase-seq, a relatively new high-throughput method that identifies DNase I hypersensitive sites across the whole genome, corresponding to the regions of open chromatin. We take advantage of the DNase-seq datasets from the ENCODE project, which are currently provided for over 100 cell types.<br><br>The sets of DNase hypersensitive sites differ between cell types, being a mixture of ubiquitous cell type-specific ones. In our study, we first restrict ourselves to the cell type-specific hypersensitive sites. Then, we consider all the pairs of transcription factor motifs available in databases like TRANSFAC.<br><br>For each of the hundreds of thousands of motif pairs, we estimate the number of their occurrences, taking into account the motif layout and mutual orientation. In the cell type-specific hypersensitive sites, we are able to identify over-represented motif pairs, which could not be noticed in the whole genome analysis. These pairs could correspond to the pairs of co-binding transcription factors.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S16</strong></td>
 </tr> <tr> 
  <td><b>Improved in silico regulatory SNP detection facilitates large scale SNP scanning</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Adam Kowalczyk <i>University of Melbourne</i><br></td>
 </tr>
 <tr> 
  <td>Geoff Macintyre (University of Melbourne) Geoff Macintyre (NICTA VRL, Diagnostic Genomics); James Bailey (University of Melbourne, Computer Science and Software Engineering); Izhak Haviv (Baker IDI Heart and Diabetes Research Institute, Bioinformatics and Systems Integration); John Markham (NICTA Victoria Research Laboratories, Diagnostic Genomics); Qiao Wang (NICTA Victoria Research Laboratories, Diagnostic Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One possible functional role for non-coding disease associated SNPs is that they are<br>regulatory SNPs (rSNPs). That is, they alter the binding affinity of a transcription factor<br>to the DNA, which in turn alters the expression of certain genes, consequently contributing to the disease phenotype. Finding these rSNPs is difficult, since they can be<br>obscured by the potentially large number of SNPs present in a linkage-disequilibrium (LD)<br>block, and the experimental procedures involved can be expensive and labour intensive.<br>Therefore in silico approaches for identifying candidate rSNPs are required to improve<br>the efficiency of interpretation of non-coding disease associated SNPs. We previously designed an in silico rSNP predictor called is-rSNP (Macintyre, et al. 2010). Shown to be successful at identifying rSNPs using a gold standard set of 41 known rSNPs, the approach, however, can only scan in the order of 1000s of SNPs before suffering from prohibitive multiple testing correction penalty. We have therefore designed improved statistics for calculating the significance of a base change on TF binding affinity. Our new statistics result in a 15% improvement on median true positive rank, across the gold standard dataset. Furthermore, a two-order magnitude improvement in significance scores facilitates scanning of 100,000s of SNPs at once. This allows is-rSNP to be used to screen entire LD blocks for candidate rSNPs. Finally, the introduction of matrix clustering to the algorithm output removes redundancy. Our enhancements to is-rSNP make the tool more practical and accessible for interpretation of disease associated SNPs.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S17</strong></td>
 </tr> <tr> 
  <td><b>Granzyme G, a serine protease, is expressed in the two-cell stage mouse embryo and is required for the maternal-zygotic transition.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chuan-Mu Chen <i>National Chung Hsing University</i></td>
 </tr>
 <tr> 
  <td>  Tung-Chou  Tsai (National Chung Hsing University, Department of Life Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Detailed knowledge of the molecular and cellular mechanisms that direct spatial and temporal gene expression in pre-implantation embryos is critical for understanding the control of the maternal-zygotic transition and cell differentiation in early embryonic development. In this study, twenty-three clones, expressed at different stages of early mouse development, were identified using differential display reverse transcription-PCR(DDRT-PCR). One of these clones, which is expressed in 2-cell stage embryos at 48 hr post-hCG injection, shows a perfect sequence homology to the gene encoding the granzyme G protein. The granzyme family members are serine proteases that are present in the secretory granules of cytolytic T lymphocytes. However, the pattern of granzyme G expression and its function in early mouse embryos are entirely unknown. Upon the introduction of an antisense morpholino (2 mM) against granzyme G to knock-down endogenous gene function, all embryos were arrested at the 2- to 4-cell stages of egg cleavage, and the de novo synthesis of zygotic RNAs was decreased. The embryonic survival rate was dramatically decreased at the late 2-cell stage when serine protease-specific inhibitors, 0.1 mM 3,4-dichloroisocoumarin (3,4-DCI), and 2 mM phenyl methanesulphonyl fluoride (PMSF), were added to the in vitro embryonic culture medium. Survival was not affected by the addition of 0.5 mM EDTA, a metalloproteinase inhibitor. We characterized for the first time the expression and function of granzyme G during early stage embryogenesis. Our data suggest that granzyme G is an important factor in early mouse embryonic development and may play a novel role in the elimination</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S18</strong></td>
 </tr> <tr> 
  <td><b>Identification of shortened 3’untranslated regions by Affymetrix genechip arrays</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Loredana Martignetti <i>Institut Curie / INSERM U900 / Mines Paris Tech</i></td>
 </tr>
 <tr> 
  <td>  Karine Laud-Duval (Institut Curie/ INSERM U830, -None-); Franck  Tirode (Institut Curie/ INSERM U830, -None-); Emmanuel Barillot (Institut Curie/ INSERM U900/ Mines Paris Tech, -None-); Olivier Delattre (Institut Curie/ INSERM U830, -None-); Andrei Zinovyev (Institut Curie / INSERM U900 / Mines Paris Tech, -None-); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Cancer cells have been recently shown to express high level of short 3' UTR isoforms that  escape miRNA-mediated regulation. We present here a computational procedure for systematically identifying shortened 3'UTRs by Affymetrix GeneChip microarrays. The advantage of this technology compared  with more recent and promising ones such as exon arrays and RNA-Seq is that, giving the relatively small cost, a typical study includes a considerably higher number of experiments. Moreover, the design of Affymetrix Gene Chips is well-suited for 3'UTR analysis of a large number of genes. Initially, GeneChip individual probes are regrouped into customized probe sets mapping specifically the CDS or the 3'UTR of the transcript, according to NCBI RefSeq  annotation (Release 45). A high value of CDS:3'UTR expression ratio for a given transcript is indicative of variation in the expression between the CDS and the 3'UTR and it can be interpreted as an event of short 3'UTR isoform expression. The analysis has been applied to expression data from 75 samples of Ewing's sarcoma patients as well as time series profiles from Ewing cell lines silenced for EWS-FLI1 oncogene. We identify 266 genes showing short 3'UTR expression in at least 10% of patients. Among them, 74 genes are confirmed to have alternative polyadenylation signals in their 3'UTR according to ATD database (Pv ~ 10-9), confirming the interest of the proposed procedure.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='S19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster S19</strong></td>
 </tr> <tr> 
  <td><b>Integrative approach at building and analyzing gene regulatory networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Julia Ponomarenko <i>University of California, San Diego</i></td>
 </tr>
 <tr> 
  <td>  Sergey Kozhenkov (University of California, San Diego, San Diego Supercomputer Center); Mihail Bailtaluk (University of California, San Diego, San Diego Supercomputer Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Gene regulatory networks provide insight into the mechanisms of differential gene expression at a system level. However, methods for inferring regulatory module network require from the user collecting heterogeneous data – gene expression data, pathways, functional annotations, TF binding sites, motifs and gene regulators – from many sources and using numerous bioinformatics tools. The necessity to use different tools for visualization and module functional analysis further complicates the procedure, making it expensive and time consuming.<br>We extended the BiologicalNetworks application with a module of new tools for inferring gene regulatory networks using a query-driven, unsupervised, integrative, module inference method. The backend database of the application integrates public data on gene expression, pathways, gene and protein sequences from multiple species and contains the comprehensive collection of public data on TF binding sites. Thus, all data required for regulatory modules discovery are automatically mined during the network generation. Additional data to mine can be integrated in the database by the user. The generated networks, regulatory modules and binding sites can be visualized and further analysed within the same application. The network can be inferred from either the gene expression experiment submitted by the user or multiple experiments available in GEO and ArrayExpress. <br>The developed tools were tested on two use cases: inferring the OCT4 regulatory network using public data on gene expression in human and mouse ES cells and inferring modules from expression data obtained in mouse model of asthma. <br>The tools are available within the BiologicalNetworks application freely downloadable at www.biologicalnetworks.org.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




