
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=R by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist27ad.html?cat=R by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:18 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'R'- Proteomics'</b></td>
 </tr>
</table>

<a name='R01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R01</strong></td>
 </tr> <tr> 
  <td><b>Statistical approach to absolute protein quantification</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sarah Gerster <i>Eidgenössische Technische Hochschule Zürich</i></td>
 </tr>
 <tr> 
  <td>  Peter Bühlmann (Eidgenössische Technische Hochschule Zürich, Seminar für Statistik (Mathematik)); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A major goal in proteomics is the comprehensive and accurate description of a proteome. Proteomics provides additional insights into biological systems that cannot be provided by genomic or transcriptomic approaches (Aebersold and Mann, 2003). In particular, proteomics holds great promise for the identification of biomarkers capable of accurately predicting disease already at a very early stage.  <br><br>The method of choice for the analysis of complex protein mixtures is shotgun proteomics. Proteins are identified and quantified based on experimentally measured peptides. While several probabilistic models exist for the identification of proteins, label-free quantification is often done in a deterministic way.<br><br>We propose a statistical approach to protein quantification with three main advantages. (i) Peptide intensities are modeled as random quantities, allowing to account for the uncertainty of these measurements. (ii) Our Markovian-type model for bipartite graphs ensures transparent propagation of the uncertainties and reproducible results. (iii) The problem of peptides mapping to several protein sequences (often neglected in other models) is addressed automatically according to our statistical model.<br><br>The performance of our model is shown on two synthetic control datasets and compared to the results of two common approaches for protein quantification (Silva et al., 2006; Lu et al., 2007). <br><br>References: <br>[1] Aebersold and Mann, (2003), Mass spectrometry-based proteomics. Nature 422(6928):198-207.<br>[2] Silva et al., (2006), Absolute quantification of proteins by LCMSE: a virtue of parallel MS acquisition. Mol. Cell. Proteomics 5:144–156.<br>[3] Lu et al., (2007), Absolute protein expression profiling estimates the relative contributions of transcriptional and translational regulation. Nat Biotechnol. 25(1):117-24.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R02</strong></td>
 </tr> <tr> 
  <td><b>Accurate Retention Time Prediction for Post-translationally Modified Peptides</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Luminita Moruz <i>Stockholm University</i></td>
 </tr>
 <tr> 
  <td>  An Staes (University of Ghent, Department of Medical Protein Research); Evy Timmerman (University of Ghent, Department of Medical Protein Research); Lennart Martens (University of Ghent, Department of Medical Protein Research); Lukas Käll (Stockholm University, Department of Biochemistry and Biophysics); Maria Hatzou (Stockholm University, Department of Biochemistry and Biophysics); Joe Foster (European Bioinformatics Institute, Wellcome Trust Genome Campus); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Predictions of peptide retention times in liquid chromatography have proven to be a valuable tool in mass spectrometry-based proteomics. In shotgun experiments, differences between predicted and observed retention times are used to remove incorrect peptide identifications. In targeted proteomics such predictions are employed for designing efficient mass spectrometry experiments. While extensive work has been put into developing accurate predictors for unmodified peptides, little efforts have been directed towards post-translationally modified peptides. In a previous study we proposed ELUDE, a machine learning-based retention time predictor. We showed that ELUDE outperforms state-of-the-art predictors, while including a special work-flow for dealing with small datasets. Here we extended our software to peptides modified by arbitrary post-translational modifications. We evaluated the new version of ELUDE by applying it on five types of modifications that shift the retention time of the peptides. Our results suggest that ELUDE yields equally good performances for modified and unmodified peptides. Furthermore, our predictor can handle peptides with more than one modification. ELUDE is fully portable to new chromatographic conditions and can be easily applied for other types of post-translational modifications. Our software can be downloaded under Apache License at www.per-colator.com or applied via an web-interface at http://elude.sbc.su.se.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R03</strong></td>
 </tr> <tr> 
  <td><b>Chimeras knocking at the door</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Milana Frenkel-Morgenstern <i>Spanish National Research Centre (CNIO)</i></td>
 </tr>
 <tr> 
  <td>  Alfonso Valencia (Spanish National Cancer Research Centre  (CNIO), Structural Biology and BioComputing); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Chimeric RNAs are distinct from conventional alternatively spliced isoforms; they may result from the trans-splicing of pre-mRNAs, or alternatively, may be the product of gene fusion following translocations or rearrangements. Chimeric transcripts can contribute to the complex completion of distinct cellular processes, permitting a combinatorial increase in the gene products available. So far, only a limited number of chimeric transcripts and/or their associated protein products have been characterized, most of which result from chromosomal translocations and many of which are associated with cancer. Therefore, it is important to extend these observations in order to catalog the chimeric transcripts that exist and to study the potential functions of their corresponding chimeric proteins. <br>Here, we used the dataset of more than 900 known chromosomal breakpoints, which are characterized by production of chimeric or fusion proteins in cancers. Using this dataset, we identified important consequences of chromosomal translocations in a context of the chromatin structure. Intriguing examples of specific translocations are discussed in details. Finally, we found constitutive heterochromatin rearrangements, which can contribute to the cancer progression by changing gene expression via long range epigenetic mechanisms.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R04</strong></td>
 </tr> <tr> 
  <td><b>Bioinformatics for Proteomics: developing new tools for the analysis of experimental data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Angelo Facchiano <i>Italian National Research Council</i></td>
 </tr>
 <tr> 
  <td>  Anna Marabotti (Italian National Research Council , Institute of Biomedical Technologies, Segrate (MI)); Francesco Facchiano (Istituto Superiore di Sanità, Dept. of Haematology, Oncology and Molecular Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Bioinformatics and computational tools for analysing data obtained in proteomics studies have been developed in our labs over the last years in order to help the complex analysis of the experimental data obtained in this field of research. A dedicated web site has been created to host these tools and it is planned to be implemented with additional tools required by our present and future users. The main tools concern the analysis of mass spectrometry data in terms of spectra alignment and comparison, the search for specific protein features in PubMed archive by means of an automatic search engine, the analysis of protein structure at different levels (primary, secondary and tertiary structure), the analysis of lists of protein sequences, or their accession numbers, to simulate proteolysis with common proteases and  predict features of the expected mixture of fragments, for comparison to experimental results by proteomics and peptidomics studies. Additional tools are also available for the editing of lists and tables, so that experimental data can be arranged in the suitable format for the tools, although these editing features can be also used for other purposes. Links to our previous tools for protein and peptide investigation are also provided.<br>The web site for accessing the tools is at the URL: http://www.bioinformatics.org/bioinfo-af-cnr/proteomics_tools/</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R05</strong></td>
 </tr> <tr> 
  <td><b>Predict subcellular localization for proteins in all kingdoms</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tatyana Goldberg <i>Technische Universität München</i></td>
 </tr>
 <tr> 
  <td>  Burkhard Rost (Technische Universität München, Department for Bioinformatics and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The prediction of protein subcellular localization is an important step towards understanding its function. Here, we present a new approach to predict localization in all five taxonomic kingdoms namely bacteria, archaea, fungi, plants and animals. We use a hierarchical system based on support vector machines (SVMs). The SVMs allow prediction of protein subcellular localization by mimicking the cascading mechanism of cellular sorting. The method is trained on a non-redundant data set of proteins of known localization from SWISS-PROT. We target more than 20 groups including cell wall, extracellular space, nucleus, cytosol, endoplasmic reticulum, golgi apparatus and melanosome. Input information is an amino acid sequence and a taxonomic classification. First, TMHMM (Krogh et al.) predicts transmembrane helices. Subsequently, the amino acid sequence is processed by SVMs in order to detect compartment-specific patterns. Finally, the protein is assigned to one of the subcellular classes and a confidence of the prediction is reported. Our program is useful for the subcellular localization prediction of newly discovered proteins and its results may be applied to other problems in biology.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R06</strong></td>
 </tr> <tr> 
  <td><b>Quantitative mass spectrometry data based pathway analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Philipp Gormanns <i>Max-Planck-Institut für Psychiatrie</i></td>
 </tr>
 <tr> 
  <td>  Simone Wolf (Max-Planck-Institut für Psychiatrie, Proteomics und Biomarker); Chris Turck (Max-Planck-Institut für Psychiatrie, Proteomics und Biomarker); Ralf Zimmer (Ludwig-Maximilians-Universität München, Lehrstuhl für Praktische Informatik und Bioinformatik); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Shotgun mass spectrometry in combination with stable isotope labeling methods enables the determination of relative protein abundance between two physiological conditions. Based on multiple peptides quantitative data protein abundance ratios are estimated which can subsequently be analyzed by pathway enrichment methods to identify affected networks. We developed an approach which predicts interesting subnetworks using the relative peptide abundance ratios. Density estimation is applied to pathway modules which consist of the corresponding peptides to discover pathway hotspots. Using peptide instead of protein abundance ratios for the pathway analysis assures that no information is lost from the experimental data to the pathway analysis process. Thus, a sophisticated data model is created which is suitable for successful subnetwork inference. The inferred probabilities directly indicate potential dysregulations of subnetworks. Therefore, it allows the detection of the most prominent subnetworks affected by the physiological condition under investigation. The method has been applied to datasets derived from trait anxiety mouse line specimens. Interesting candidate networks have been identified which will be further validated.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R07</strong></td>
 </tr> <tr> 
  <td><b>Protein Identification Using Top-Down Spectra</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yakov Sirotkin <i>St. Petersburg Academic University, Algorithmic Biology Laboratory</i><br></td>
 </tr>
 <tr> 
  <td>Xiaowen Liu (University of California, San Diego) Yufeng Shen (PNNL, Biological Science Division); Gordon Anderson (PNNL, Biological Science Division); Yihsuan Tsai (University of Washington, Department of Medicinal Chemistry); Ying Ting (University of Washington, Department of Medicinal Chemistry); David Goodlett (University of Washington, Department of Medicinal Chemistry); Richard Smith (PNNL, Biological Science Division); Vineet  Bafna (University of California, San Diego, Department of Computer Science and Engineering); Pavel Pevzner (University of California, San Diego, Department of Computer Science and Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In the last two years, due to advances in mass spectrometry, top-down mass spectrometry moved from analyzing single proteins to analyzing complex samples and identifying hundreds and even thousands of proteins. However, computational tools for database search of top-down spectra against protein databases are still in infancy. We describe MS-Align+, a fast algorithm for top-down protein identification based on spectral alignment that enables searches for unexpected post-translational modifications (PTMs). Given a top-down deconvoluted spectrum, we use a dynamic programming algorithm to find the best spectral alignment between the spectrum and each protein in the protein database and report the best-scoring protein. To speed up the protein identification process, we apply a spectral convolution approach to quickly filter out the proteins that can not have good alignments. In addition, we estimate the E-value of each identified protein-spectrum-match(PrSM) using a generating function approach. We developed a web-based software tool for MS-Align+. It has a user-friendly interface and interactive web pages for showing identification results on spectrum, protein species and protein levels.<br><br>We tested MS-Align+ on two top-down data sets from Saccharomyces cerevisiae(SC 8,468 deconvoluted spectra) and Salmonella typhimurium(ST 3,582 deconvoluted spectra). We identified 2,455 and 1,490 MS/MS spectra with E-value threshold 0.0001 from the SC and ST data sets respectively. On the SC data set, MS-Align+ identified 123 new proteins compared to SEQUEST. On the ST data set, MS-Align+ identified 86 new proteins compared to ProSightPTM, and 35 proteins compared to PIITA.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R08</strong></td>
 </tr> <tr> 
  <td><b>PRIDE, the PRoteomics IDEntifications database</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Johannes Griss <i>EMBL-European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Melih Birim (EMBL, European Bioinformatics Institute); Richard Côté (EMBL, European Bioinformatics Institute); Attila Csordas (EMBL, European Bioinformatics Institute); Joe Foster (EMBL, European Bioinformatics Institute); Gavin O'Kelly (EMBL, European Bioinformatics Institute); David Ovelleiro (EMBL, European Bioinformatics Institute); Daniel  Ríos (EMBL, European Bioinformatics Institute); Florian Reisinger (EMBL, European Bioinformatics Institute); Rui Wang (EMBL, European Bioinformatics Institute); Juan A Vizcaino (EMBL, European Bioinformatics Institute); Henning Hermjakob (EMBL, European Bioinformatics Institute); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The PRoteomics IDEntifications database (PRIDE, http://www.ebi.ac.uk/pride) at the European Bioinformatics Institute (EBI) has become one of the main repositories of mass spectrometry (MS) derived proteomics data. PRIDE stores three different kinds of information: MS and MS/MS mass spectra as peak lists, the derived peptide and protein identifications, and any associated metadata.<br>A major aim for PRIDE is the support of the peer-review process in the proteomics community. Therefore, we developed a new open source application called PRIDE Inspector (http://code.google.com/p/pride-toolsuite/). It allows researchers to examine their own data sets before the actual submission to PRIDE is performed, or access data already in PRIDE for data mining purposes. It can also be used by journal’s editors/reviewers since it facilitates the thorough review of submitted data at the pre-publication stage. Different views of the data are provided: metadata, spectrum, peptide and protein centric. Additionally, the major strength of the tool is the possibility to perform a first assessment on data quality, since a variety of charts based on the data are automatically generated.<br>Finally, PRIDE serves as the entry point for the ProteomeXchange consortium (http://www.proteomexchange.org), which aims at establishing a regular data exchange between major proteomics repositories.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R09</strong></td>
 </tr> <tr> 
  <td><b>Lost Data – The Consequences of Changing Protein Accession</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Johannes Griss <i>EMBL-European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Richard  Côté (EMBL, European Bioinformatics Institute); Christopher Gerner (Medical University of Vienna, Department of Medicine I); Henning Hermjakob (EMBL, European Bioinformatics Institute); Juan A  Vizcaino (EMBL, European Bioinformatics Institute); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Proteomics data is produced in constantly growing quantities. The storage of digital data for long periods of time always comes with the question of how long it will be possible to read the data. Protein identifications are reported and stored using an unstable reference system: protein identifiers. These identifiers are created by every protein database individually and can change or even be deleted over time. <br>Therefore, we analyzed the changes of the reported protein identifiers from all public experiments from the PRIDE database (http://www.ebi.ac.uk/pride) by November 2010. We mapped every protein identifier to a currently active entry using two independent approaches. The first one used the PICR service at the EBI, which is based on 100% sequence identity. The second one (termed the logical mapping algorithm) accessed the source databases and retrieved the current status of the reported identifier. <br>Our analysis showed great differences between the most commonly used protein databases (IPI, UniProtKB, NCBI nr database and Ensembl), in respect to identifier stability. In this analysis UniProtKB proved to be the most stable resource. On the other hand, experiments using IPI already contained 20% of deleted identifiers after only two years. In addition, we furthermore investigated the proportion of peptide identifications no longer fitting the current protein sequence. Up to our knowledge, this is the first analysis that quantifies the effect of changing protein sequences on the long-term storage of proteomics data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R10</strong></td>
 </tr> <tr> 
  <td><b>Proteomics based genome annotation using conceptually translated ESTs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Paul Blakeley <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  Jennifer  Siepen  (University of Manchester, Faculty of Life Science); Simon Hubbard  (University of Manchester, Faculty of Life Science ); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Mass Spectrometry (MS) based proteomics is a powerful approach to gain evidence for protein-coding genes and splice variants predicted from genome sequences. However, protein and peptide identification is critically dependant on searching the MS data against a database of amino acid sequences. Alternative strategies include searching against theoretical translations of the “raw” genome sequence, de novo gene predictions and transcriptome-derived data. These ‘proteogenomic’ approaches enable the high-throughput discovery of proteins that are not present in existing sequence datasets (e.g. Ensembl), as well as validation of those that are. Most proteogenomic studies to date have searched MS data against a 6-frame translation of a genome sequence and have facilitated improvements in gene models and gene discovery. However, this approach remains challenging owing to size of the database, high content of false positives and difficulties associated with estimating error rates. As an alternative we have investigated different methodologies that can be used overcome this using transcriptome data, thereby reducing the size of the database, improving the conceptual translation of nucleotide sequences, and applying appropriate measures of statistical significance. We demonstrate this approach using the chicken genome as a model system, searching MS data against protein sequences predicted from assembled Expressed Sequence Tags (ESTs). This approach enabled the validation of 2526 known Ensembl proteins, as well as 145 single amino acid polymorphisms, and 275 candidate novel genes illustrated via the discovery of aldolase c, which was previously unannotated in the chicken genome.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R11</strong></td>
 </tr> <tr> 
  <td><b>Data mining on a leukemia phosphoproteome dataset: insights on chemoresistant cell signaling</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Renato Milani <i>University of Campinas</i></td>
 </tr>
 <tr> 
  <td>  Carmen Ferreira (University of Campinas, Department of Biochemistry); Eduardo Galembeck (University of Campinas, Department of Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Biologists in the bench have been generating ever-growing amounts of data from high-throughput techniques. However, there is often a shortage of means to analyze these data. Reversible protein modification by phosphorylation is a ubiquitous mechanism to control signal transduction networks governing biological processes such as chemoresistance and cancer itself. Techniques like phosphoproteomics enable researchers to evaluate whether and where a phosphorylation reaction takes place in the cell. Here we present a leukemia chemoresistance phosphoproteome dataset generated by our own research group. Our objective was to establish whether a phosphorylation event activates or inhibits a particular enzyme for each protein in the dataset. We built an automated tool, called PhosphoActivity, to extract this information from trusted online databases and submitted 4.257 phosphorylation entries to it. With this information, we could sort between active and inhibited enzymes that may play a role on chemoresistance. Thus, we were able to propose novel signaling networks related to chemoresistance and to identify essential components of multidrug resistance myeloid leukemia, as well as suggest new avenues for addressing this issue.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R12</strong></td>
 </tr> <tr> 
  <td><b>Identification of protein expression dynamics during germination in Streptomyces</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Eva Strakova <i>Institute of Microbiology, Czech Academy of Sciences</i></td>
 </tr>
 <tr> 
  <td>  Alice Zikova (Institute of Microbiology, Czech Academy of Sciences, Laboratory of Bioinformatics); Pavel Rehulka (University of Defence, Institute of Molecular Pathology); Jiri Vohradsky (Institute of Microbiology, Czech Academy of Sciences, Laboratory of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A comprehensive insight into the mechanisms controlling Streptomyces germination is of primary importance both for practical applications, as Streptomyces species are important producers of antibiotics and other secondary metabolites, and for fundamental studies of cell development. To understand initialization of germination in Streptomyces we employed proteomic based approach in a time-dependent manner.  Germination is a transition process when dormant spores, under favourable conditions, trigger physiological processes essential for active growth and development to vegetative cells. We observed dynamic changes in proteome in 13 time points in the initial 6 hours of germination of Streptomyces coelicolor using 2D gel electrophoresis to examine both protein concentration level (florescence staining) and newly expressed proteins (in vivo radioisotope labelling) in each time point.  Proteins with increased expression level were identified by mass spectrometry. Analysis of protein kinetic profiles revealed biochemical pathways associated with initiation and other phases of germination.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R13</strong></td>
 </tr> <tr> 
  <td><b>Tracing Evolutionary Events Over Ancestral Proteomes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Matt Oates <i>University of Bristol</i></td>
 </tr>
 <tr> 
  <td>  Julian Gough (University of Bristol, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: With an abundance of well covered proteomes in the public domain we can begin to explore possible ancestral objects relating them. Important historic events such as the rise of animals and vertebrates occurred in coincidence with large-scale domain shuffling, and novel architecture creation that is visible in protein domain annotations. <br>Presented is an online resource for exploring the evolution of the protein repertoire, from a protein domain architecture perspective; describing the functional relevance of domain level changes along an ancestral trace of the tree of life. <br>Using Dollo parsimony-based ancestral reconstructions of domain architecture from the SUPERFAMILY database, we create domain architecture networks for every internal node of a phylogenetic tree. These networks provide the basis of comparison along each edge of the tree. Using a domain centric GO annotation available per domain, a description is made for each branch using topic allocation, similar to TF-IDF from natural language processing. This provides a descriptive trace along each path of the tree.<br>A user can query based on a set of gene IDs or functional keywords, resulting in a map of relevant domains being generated automatically over the tree. This can be viewed in relation to the background events of each branch to find the evolutionary story that has given rise to the system of interest, in any available taxa.<br>This tool will be of use to biologists wishing to explore divergent protein systems over a span of phylogeny.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='R14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster R14</strong></td>
 </tr> <tr> 
  <td><b>Viral proteins acquired from a host converge to simplified domain architectures</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nadav Rappoport <i>Hebrew University of Jerusalem</i></td>
 </tr>
 <tr> 
  <td>  Michal Linial (Hebrew University of Jerusalem, Department of Biological Chemistry Institute of Life Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Viruses’ infection cycle creates multiple opportunities for the exchange of genetic material with the host. As part of their life cycle, some viruses, due to faulty excision, may acquire host sequences. In most instances, such sequences will accumulate mutations or be deleted. However, in rare instances, sequences acquired from a host become beneficial for the virus. We searched for unexpected sequence similarity among the 900,000 viral proteins and all cellular organism sequences. Here we focus on viruses that infect specifically metazoa. Our analysis was carried out at two levels of sequence conservation: we consider (using UniRef90) highly conserved sequences found in viruses and their hosts. Our low-conservation sequence analysis utilizes the Pfam family collection. The high-conservation analysis yielded 187 viral-host sequences of which 80% are viral sequences that were inserted into their host, and 20% in which a virus hijacked and maintained host sequences. From the 12,000 statistical models archived in Pfam, 670 families are composed of viral-metazoan proteins. We show that in about 75% of these families, the viral proteins are significantly shorter than their metazoan counterparts. The phenomena that lead to shorter viral proteins are a substantial elimination of domains, a reduction in the length of the protein tails and the linkers between domains. We conclude that throughout viral evolution, host-originated sequences were shaped toward simplified domain compositions. We postulate that such shorter proteins may act by interfering with fundamental functions of the host. We demonstrate such functions in intracellular signaling, post-translational modification, protein-protein interaction and cellular trafficking.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




