
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=D by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist2f7c.html?cat=D by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:17 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'D'- Comparative Genomics'</b></td>
 </tr>
</table>

<a name='D01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D01</strong></td>
 </tr> <tr> 
  <td><b>Characterization of a mannose-binding Insecticidal lectin gene from Allium sativum (Garlic)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nusrah Afolabi-Balogun <i>Agriculture research Council-Vegetable and Ornamental Plant Institute</i></td>
 </tr>
 <tr> 
  <td>  Hajia Inuwa  (Ahmadu Bello University , Biochemistry); Lynelle  Van Emmenes (Agriculture Research Council , Plant Breeding); Patrick Adeboola (Agriculture Research Council , Plant Breeding); Ibrahim Sanni (Ahmadu Bello University, Biochemistry); Andrew Nok (Ahmadu Bello University, Biochemistry); ishiyaku Mohammed (Ahmadu Bello University, plant Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The full-length cDNA of monocot mannose-binding insecticidal agglutinin was synthesized using with Reverse 5’-TTGTAGAAACTAGTAGGAACCCTAT-3’ and Forward 5’-ATGGCCAGGAACCTACTGACGAACG-3’. Nucleotide-nucleotide BLAST of the ASAi full-length cDNA sequence on NCBI website (http://www.ncbi.nlm.nih.gov) revealed 362 bp, and had a 327, 216 and 162 bp open reading frame (ORF) when considered from frame +1, -3 and -1and positions 25 to 351, 85 to 300 and 18 to 182 encoding 108aa, 54aa and 71aa  respectively. Multiple alignments of ASAi amino acids with those of seven other MMBLs revealed three highly conserved domains for sugar-binding [Gln (Q), Asp (D), Asn (N) and Tyr (Y)] namely QDNY <br>Signal peptide prediction on website (http://www.cbs.dtu.dk/services/singalP) showed that ASAi, like most other mannose-binding lectins from Allium species (Van Damme et al., 1995; Yao et al., 2003; Zhao et al., 2003), encoded a lectin precursor with two signal peptide of 12aa and 1-22aa, and Most likely cleavage site between positions 22 and 23. <br>Phylogenetic analysis indicated that 20 MMBLs including ASAi belonged to an extendedPhylogenetic analysis indicated that 20 MMBLs including ASAi belonged to an extended superfamily. Gene Ontology analysis indicate one biological process with GO ID: 0006952 representing defense response, with secondary IDs GO: 0002217 GO: 0042829.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D02</strong></td>
 </tr> <tr> 
  <td><b>Bacterial Phylogenomics</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Robert Stones <i>Food and Environment Research Agency</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Foodborne pathogens such Escherichia coli and Salmonella strains are of serious concern to public health. It is estimated that there are from 2 to 4 million cases of salmonellosis occur in the U.S. annually, and is rising.  Many methods have been developed for detection and analysis of foodborne pathogens.  Conventional culture methods require many days for presumptive results; several rapid methods are available but still require days to confirm results. These methods are slow to develop new diagnostics for closely related bacterial strains. And there is the increasing need to have a greater understanding of the evolution and biology of pathogens, which are becoming more and more of a concern to public health.<br><br>However, advances both in rapid next generation DNA sequencing technology, and phylogenetic analytical methods of large datasets across many taxa, has enabled the potential to compare whole genomes of bacteria; important to both clinical and regulatory science agencies.  Enabling a better understanding and in much greater detail, the biology and evolution of bacterial strains, and how they are increasingly becoming a danger to public health.<br><br>The ability to construct whole phylogenies of complete bacterial genomes, through advances in rapid genome annotation is becoming more apparent.  Development of novel gene clustering and analytical pipelines, coupled with rapid phylogenetic analysis presented in this work, will enable a better understanding of bacterial genomics, and help protect public health.  This work presents recent results and methods for constructing phylogenies from entire genomes of foodborne bacteria, namely Bacterial Phylogenomics.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D03</strong></td>
 </tr> <tr> 
  <td><b>Detecting rare copy number variations (CNVs) with sparse coding</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andreas Mitterecker <i>Johannes Kepler University</i></td>
 </tr>
 <tr> 
  <td>  Djork-Arné Clevert (Johannes Kepler University, Institute of Bioinformatics); Andreas Mayr (Johannes Kepler University, Institute of Bioinformatics); An De Bondt (Janssen Pharmaceutica, Research & Development); Willem Talloen (Janssen Pharmaceutica, Research & Development); Hinrich Göhlmann (Janssen Pharmaceutica, Research & Development); Sepp Hochreiter (Johannes Kepler University, Institute of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: High-density oligonucleotide genotyping microarrays, especially Affymetrix SNP6 chips, are widely used for high-resolution copy number analysis. In order to identify CNVs more reliable, we have proposed a Maximum a posteriori factor analysis model called cn.FARMS. The latent variable, the factor, captures the simultaneous increase or decrease of DNA amount at neighboring chromosome locations measured by the intensity of oligonucleotide probes. This increase or decrease indicates amplification or deletion of a DNA region that is a CNV. cn.FARMS considerably reduces the false discovery rate (FDR) by combining adjacent chromosome locations to an ensemble voting (agreement of multiple measurements) instead of relying on a single measurement as other methods do.<br><br>Standard factor analysis assumes a Gaussian factor distribution which, however, is a wrong assumption for CNVs. Redon et al. 2006 showed that most CNVs affect less than three individuals out of 270 HapMap samples. These rare events are hard to detect by cn.FARMS as they would be interpreted as noise. Therefore we propose a factor analysis model with a Laplacian prior, which leads to a sparse factor distribution. But now we face another problem: the likelihood becomes analytically intractable. We tackled this problem applying an algorithm that employs a variational expectation maximization algorithm to the sparse prior, which optimizes a lower bound on the likelihood.<br><br>We have applied the Laplacian cn.FARMS model on the HapMap dataset to detect CNVs. We could verify most of published copy number variable regions and found new ones. However many known CNVs seem to be false positives.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D04</strong></td>
 </tr> <tr> 
  <td><b>Tools for comparative metagenome analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kathrin Aßhauer <i>University of Göttingen</i></td>
 </tr>
 <tr> 
  <td>  Thomas Lingner (University of Göttingen, Department of Bioinformatics); Peter Meinicke (University of Göttingen, Department of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Metagenomics provides an approach to the analysis of microbial communities from environmental and clinical samples. In this context, the  advances of next-generation sequencing technologies allow insights into the enormous taxonomic and functional diversity even of complex microbial communities (Daniel, Nat Rev Microbiol, 2005). However,  this development demands new bioinformatics tools which can efficiently deal with metagenomic data sets on a large-scale.<br><br>We developed Taxy and CoMet, two freely available tools that allow to quickly determine the phylogenetic and functional composition of large metagenomic data sets. In contrast to currently existing approaches for taxonomic profiling, which rely on sequencing read classification, Taxy is based on mixture modeling of the overall nucleotide distribution of a metagenomic sample. This completely novel approach is insensitive to read length variation and thus provides a better comparability of samples across different sequencing platforms. Taxy is available as a Matlab/Octave toolbox as well as an interactive tool for Windows.<br><br>The web-server CoMet focuses on the comparative functional profiling based on assignments of metagenomic reads to Pfam protein domains. CoMet combines a speed-optimized implementation of the Orphelia gene prediction engine (Hoff et al., Nucleic Acids Res, 2009) with Pfam domain detection using the UFO approach (Meinicke, BMC Genomics, 2009). Based on the resulting domain frequency profiles comparative statistical analyses are performed to identify and to visualize functional differences of metagenomic samples. The CoMet pipeline is accessible via an easy-to-use web interface at http://comet.gobics.de.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D05</strong></td>
 </tr> <tr> 
  <td><b>Identification of transcription factors and their correlation with the high diversity of Stramenopiles.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Francisco/Javier Buitrago/Florez <i>Universidad de los Andes</i><br></td>
 </tr>
 <tr> 
  <td>Francisco Buitrago (Universidad de los Andes) Diego Mauricio Riaño-Pachon (Universidad de los Andes, Biological Sciences); Silvia Restrepo (Universidad de los Andes, Biological Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Transcription factors (TFs) regulate spatial and temporal gene expression by binding to DNA and either activating or repressing the action of RNA polymerases; in addition to TFs, other Transcriptional regulators (TRs) participate in transcriptional modulation. With the availability of genome sequences for several organisms and computational strategies for gene functional annotation, the entire set of TFs and TRs can be identified, described, and compared between species and lineages. The diversity among Stramenopiles is striking; they range from large multicellular seaweeds to tiny unicellular species, they are present in freshwater, marine and terrestrial habitats and embrace many ecologically important algal (e.g. diatoms, brown algae, chrysophytes), and heterotrophic (e.g., Oomycetes) groups. In order to find TF and TR genes in the deduced proteomes of Stramenopiles, we followed and extended the approach developed in Perez et al. 2010. Briefly, it exploits the presence of protein domains and their combinations, in the form of boolean rules, that are specific for different families of TFs and TRs. We applied an enlarged set of rules to the deduced proteomes of 9 different Stramenopiles, identifying more than 400 different regulatory genes in each species belonging to up to 126 different gene families. The identification of this class of regulatory genes will constitute and important resource that could be exploited in gene functional characterization and evolutionary analyses. All TFs/TRs families found will be publicly released via a web database.<br>References.<br>Pérez-Rodríguez P, Riaño-Pachón DM, Corrêa LG, Rensing SA, Kersten B, Mueller-Roeber B. 2010. Nucleic Acids Res. 38(Database issue):D822-7</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D06</strong></td>
 </tr> <tr> 
  <td><b>SeqXML and OrthoXML: standards for sequence and orthology information</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Thomas Schmitt <i>Stockholm University</i></td>
 </tr>
 <tr> 
  <td>  David Messina (Stockholm University, Department of Biochemistry and Biophysics); Fabian Schreiber (Stockholm University, Department of Biochemistry and Biophysics); Erik LL Sonnhammer (Stockholm University, Department of Biochemistry and Biophysics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Today’s orthology databases lack standards. Users must contend with different ortholog representations from each provider, and the providers themselves must independently gather and parse the input sequence data. These burdensome and redundant procedures make data comparison and integration difficult. We have designed two XML-based formats, SeqXML and OrthoXML, to solve these problems.<br><br>SeqXML is a lightweight format for sequence records – the input for orthology prediction. It stores the same sequence and meta data as typical FASTA format records, but overcomes common problems such as unstructured metadata in the header and erroneous sequence content. XML provides validation to prevent data integrity problems that are frequent in FASTA files. The range of applications for SeqXML is broad, and we provide read/write functions for BioJava, BioPerl, and Biopython.<br><br>OrthoXML was designed to represent orthologs from any source in a consistent and structured way, yet cater to specific needs such as scoring schemes or meta information. A unified format is particularly valuable for ortholog consumers that want to combine lots of sources, e.g. for gene annotation projects.<br><br>Reference proteomes for 61 organisms are already available in SeqXML, and 10 orthology databases have signed on to OrthoXML. Adoption by the entire field would substantially facilitate exchange and quality control of sequence and orthology information.<br><br>Please visit http://SeqXML.org or http://OrthoXML.org for more details.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D07</strong></td>
 </tr> <tr> 
  <td><b>A Practical Algorithm for Ancestral Rearrangement Reconstruction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jakub Kovac <i>Comenius University</i></td>
 </tr>
 <tr> 
  <td>  Brona Brejova (Comenius University, Department of Computer Science); Tomas Vinar (Comenius University, Department of Applied Informatics ); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 85<br><br>Motivation: Genome rearrangements are a valuable source of information about early evolution, as well as an important factor in speciation processes. Reconstruction of ancestral gene orders on a phylogeny is thus one of the crucial tools contributing to understanding of evolution of genome organization. For most models of evolution, this problem is NP-hard. Existing heuristic approaches impose restrictions on the used model or input genome architectures.<br>Results: We have developed a universal method for reconstruction of ancestral gene orders by parsimony (PIVO) using an iterative local optimization procedure. Our method can be applied to different rearrangement models. Combined with a sufficiently rich model, such as the double cut and join (DCJ), it can support a mixture of different chromosomal architectures in the same tree. We show that PIVO can outperform previously used steinerization framework and achieves better results on real data than previously published methods.<br>Availability and Implementation: Datasets, reconstructed histories, and the software can be downloaded at http://compbio.fmph.uniba.sk/pivo/.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D08</strong></td>
 </tr> <tr> 
  <td><b>Curiosities from the Drosophila proteome</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Michael Tress <i>Centro Nacional de Investigaciones Oncologicas</i></td>
 </tr>
 <tr> 
  <td>  Bernd  Bodenmiller (ETH Zurich, Institute of Molecular Systems Biology); Ruedi  Aebersold (ETH Zurich, Institute of Molecular Systems Biology); Alfonso Valencia (Centro Nacional de Investigaciones Oncologicas, Structural and Comptational Biology Programme); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Mature mRNA is generated from primary precursor mRNA transcripts (pre-mRNA) by the joining of exonic sequences. In the standard procedure (cis-splicing) exons from the same pre-mRNA molecule are joined. However, splicing can also occur between two separately transcribed pre-mRNA molecules. This process is termed “trans-splicing”.<br>There is concrete evidence that trans-splicing is important to the function of at least two Drosophila genes, modifier of mdg4 (mod(mdg4)) and longitudinals lacking (lola). A recent study has confirmed that the majority of lola and mod(mdg4) splice variants are indeed generated through trans-splicing. <br>We have analysed two large-scale studies of the Drosophila melangaster proteome. The experiments showed that a surprising number of the variants from lola and mod(mdg4) are indeed expressed as stable proteins in vivo. The transcripts that gave rise to these isoforms had all been shown to be generated almost entirely by trans-splicing. The large numbers of expressed protein isoforms detected for lola and mod(mdg4) suggests that trans-splicing may play an even greater role in Drosophila alternative splicing than previously thought.  <br>Transcripts that generate proteins with premature stop codons are generally believed to be degraded by the nonsense mediated decay (NMD) pathway. Another recent study investigated down-regulation by NMD in Drosophila. As part of the paper the authors provided a high confidence set of more than 50 NMD targets. Analysis of the two large-scale Drosophila proteomics studies showed that at least four of the isoforms tagged as targeted for NMD were also expressed as proteins.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D09</strong></td>
 </tr> <tr> 
  <td><b>A new Perl toolkit for the detection and analysis of polymorphic loci and their application for bacterial typing</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Luis M Rodriguez-R <i>Institut de Recherche pour le Développement</i><br></td>
 </tr>
 <tr> 
  <td>Ralf KOEBNIK (Institut de Recherche pour le Développement) Luis M Rodriguez-R (Institut de Recherche pour le Développement, UMR Résistance des Plantes aux Bioagresseurs); Boris Szurek (Institut de Recherche pour le Développement, UMR Résistance des Plantes aux Bioagresseurs); Christine Pourcel (Université Paris-Sud, Institut de Génétique et Microbiologie); Ralf Koebnik (Institut de Recherche pour le Développement, UMR Résistance des Plantes aux Bioagresseurs); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Since the introduction of DNA fingerprinting during the 1990s, molecular typing has been extensively employed in human medicine, veterinary medicine and plant pathology for a wide range of pathogens and based on distinct polymorphic loci. The explosion of genomic data during the last decade empowered the rational design and development of typing techniques enabling the prediction and detection of specific loci with different key features, like sequence similarity, repetitive regions or domains architecture. Loci employed for bacterial typing include Clustered Regularly Interspaced Short Palindromic Repeats (CRISPR), Variable Number of Tandem Repeats (VNTR) and Simple Sequence Repeats (SSR), among others. Hitherto, several tools have been developed for the detection and comparison of such loci and some information systems have been developed either locus-specific or organism-specific. Here, we present a Perl library and a set of scripts for the analysis of variable loci used for bacterial typing based on highly customizable rules. The library allows the detection, grouping and group extension and comparison of any locus definable with the supported rules. We successfully employed the library for the analysis of VNTRs as a test case on genomes of different qualities from the genus Xanthomonas, a group of plant pathogens belonging to the gamma division of Proteobacteria. We used the library to reproduce and automate a previous workflow that required high human intervention, without losing results quality and reducing the risk of human error. The tool is accessible at http://bionfo-prod.mpl.ird.fr/xantho/ and is being used for the development of laboratory genotyping tools of xanthomonads.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D10</strong></td>
 </tr> <tr> 
  <td><b>Protein Intrinsic Disorder in Viruses and Virus-Host Interactions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yu-An Dong <i>Technische Universität München</i></td>
 </tr>
 <tr> 
  <td>  Burkhard Rost (Technische Universität München, Informatics); Markus Schmidberger  (Technische Universität München, Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein intrinsic disorder plays an important role in signaling and regulation and thus is more prevalent in higher eukaryotes than bacteria. Many viral genomes, however, exhibit surprisingly high disorder content. Furthermore, there is significant variation of disorder content among viruses. Several herpesviruses, which are large DNA viruses, are compared with several common RNA viruses, for both their own disorder content and that of their human interaction partners. RNA viruses are less disordered, but their human protein interaction targets are more disordered. Distribution of disorder is correlated with phylogeny and viral life cycle. Patterns of disorder-order interactions are investigated in both human and virus-human context.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D11</strong></td>
 </tr> <tr> 
  <td><b>GECKO2: A software tool for de novo gene cluster detection</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Katharina Jahn <i>Universität Bielefeld</i></td>
 </tr>
 <tr> 
  <td>  Leon Kuchenbecker (Universität Bielefeld, Technische Fakultät); Sebastian Böcker (Friedrich-Schiller-Universitaet Jena, Institut fuer Informatik ); Jens Stoye (Universität Bielefeld, Technische Fakultät); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Co-localized genes that remain clustered across multiple genomes are known to provide highly informative signals for functional analysis. Incomplete conservation caused by micro-rearrangements, gene losses, gene insertions, as well as difficulties in establishing gene homologies turn the detection of such gene clusters into a computationally hard problem.<br><br>We developed a flexible platform for de novo detection of conserved gene clusters in multiple genomes. The underlying models increase robustness against incomplete conservation patterns essentially over the widely used max-gap model.<br>Our software constitutes a reasoned trade-off between algorithmic efficiency and model complexity<br>that allows for an efficient analysis of large genome sets.<br>The significance of the predicted clusters is evaluated in a statistical framework integrated into our software.<br>To facilitate the analysis, we offer different search scenarios and provide interactive visualization of the results.<br>The combination of these features presents a valuable improvement over existing approaches<br>to the systematic analysis of gene cluster conservation in multiple genomes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D12</strong></td>
 </tr> <tr> 
  <td><b>Automated Segmentation of DNA Sequences</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Brona Brejova <i>Comenius University Bratislava</i></td>
 </tr>
 <tr> 
  <td>  Michal Burger (Comenius University, Computer Science); Tomas Vinar (Comenius University, Applied Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 143.<br><br>Most algorithms for reconstruction of evolutionary histories involving<br>large-scale events such as duplications, deletions or rearrangements,<br>work on sequences of predetermined markers. Typically, protein coding<br>genes or other functional elements are used as makers. However,<br>markers defined in this way ignore information included in non-coding<br>sequences, are prone to errors in annotation, and may even introduce<br>artifacts due to partial gene copies or chimeric genes.<br><br>We propose the problem of sequence segmentation where the goal is to<br>automatically select suitable markers based on sequence homology alone.<br>We design an algorithm for this problem which can tolerate certain<br>amount of inaccuracies in the input alignments and still produce<br>segmentation of the sequence to markers with high coverage and accuracy. <br>We test our algorithm on several artificial and real data sets<br>representing complex clusters of segmental duplications.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D13</strong></td>
 </tr> <tr> 
  <td><b>Antimicrobial Resistance and susceptility profiling of four pathogenic organisms using UV irradiation.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Olugbenga Taiwo <i>Covenant University</i></td>
 </tr>
 <tr> 
  <td>  Solomon Oranusi (Covenant University, Biological Sciences); Enejo Abah (Covenant University, Biological Sciences); Conrad omonhinmin (Covenant University, Biological Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Antimicrobial Resistance and Susceptibility Profiling of four Pathogenic Organisms Using UV Irradiation.<br>1Taiwo, O.S., 1Oranusi U.S., 2Omonhinmin A. C., and 1Abah, S.E.<br>1Microbiology Unit, Department of Biological Sciences,<br>College of Science & Technology, Covenant University, Ota.<br>2Applied Biology & Biotechnology Unit, Department of Biological Sciences,<br>College of Science & Technology, Covenant University, Ota.<br>ABSTRACT<br>This poster is based on proceedings submission of Studies on antimicrobial resistance and susceptibility profile and effects of UV irradiation in conferring susceptibility and resistance was carried out on two gram positive (Staphylococcus aureus, Enterococcus faecalis ATCC29212); and two gram negative organisms (Escherichia coli ATCC25922,Shigella dysenteriae 006). The test organisms were subjected to UV irradiation for 0hr, 1hr, 3hrs,5hrs and 7hrs,and check afterward for antimicrobial sensitivity.Antimicrobial sensitivity test showed E.coli ATCC 25922, initially susceptible to COT and COL acquired resistance to both with increased exposure to UV irradiation. S. aureus initially resistant to COT became susceptible after 5hrs UV exposure.Statistical analyses showed a marked difference in the susceptibility of E. coli ATCC25922,E.faecalis  ATCC29212 and S. aureus to antibiotics but not with the exposure time regime (? = 0.05). Conversely, S.dysenteriae 006 recorded a significant change (? = 0.05) in the exposure time without a noticeable change amongst the antibiotics.The trend line analyses of the mean susceptibility of the various organisms to the time of exposure showed a possibility of high increase in susceptibility in S. dysenteriae 006, a slim possibility of increase in E. faecalis ATCC29212 and S. aureus, and a decrease for E.coli.keyword:UVIrradiation,Resistance,Susceptibility and Induced Mutation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D14</strong></td>
 </tr> <tr> 
  <td><b>Structure-based Whole Genome Realignment Reveals Putative Non-coding RNAs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sebastian Will <i>Massachusetts Institute of Technology</i></td>
 </tr>
 <tr> 
  <td>  Michael Yu (Massachusetts Institute of Technology, CSAIL); Bonnie Berger (Massachusetts Institute of Technology, Mathematics, CSAIL); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 233<br><br>Motivation: Current whole genome alignment approaches align according<br>to sequence-similarity but do not explicitly consider molecular<br>structure.  Therefore, non-coding RNAs (ncRNAs) with high structural<br>but low sequence conservation are typically poorly aligned.  Such<br>non-coding RNAs often remain hidden from genome-wide de novo<br>prediction screens that require visible structural conservation in the<br>whole genome alignment.  In consequence, ncRNA prediction by recent<br>genomic screens is strongly biased towards high sequence conservation.<br>Even when the alignment quality is sufficient for the identification<br>of a ncRNA, improving the alignment can be advantageous for<br>further comparative analysis.<br><br>Results: We present an original approach for realigning whole genomes<br>to reveal previously undetected ncRNAs.  Our realignment pipeline<br>combines de-novo prediction of structural RNAs and a novel<br>structure-based realignment algorithm.  For the latter we introduce a<br>novel banding technique in the dynamic programming algorithm of the<br>established RNA alignment method LocARNA.  This limits the deviation<br>from the original alignment.  The benefits of this strategy are<br>two-fold.  First, the runtime of this otherwise costly computation on a<br>genomic scale is significantly reduced.  Second, controlling the<br>deviation from the original alignment ensures a conservative<br>realignment, increasing the confidence in predictions from such<br>alignments.<br><br>We apply the pipeline to a whole genome alignment of the twelve<br>assembled Drosophilids genomes.  Due to our realignment strategy, we<br>predict thousands of high confidence putative ncRNAs that could not be<br>identified from the whole genome alignment.  Particularly we improve<br>the prediction of ncRNAs with low sequence conservation which are<br>intrinsically more challenging for de novo prediction.<br><br>Availability: The deviation-limited realignment algorithm is freely<br>available as part of the open-source package LocARNA at<br>http://www.bioinf.uni-freiburg.de/Software/LocARNA/.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D15</strong></td>
 </tr> <tr> 
  <td><b>Degree of synteny between the genomes of radiation-resistant species of Deinococcus does not reflect their extreme dsDNA break repair capabilities</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jelena Repar <i>Rudjer Boskovic Institute</i></td>
 </tr>
 <tr> 
  <td>  Ksenija Zahradka (Rudjer Boskovic Institute, Division of Molecular Biology); Davor Zahradka (Rudjer Boskovic Institute, Division of Molecular Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Deinococcus radiodurans, a highly radiation resistant bacterium, is able to accurately reassemble it's genome after it has been shattered into hundreds of pieces. Extreme radiation resistance is a typical attribute of members of Deinococcus genus implicating similar DNA repair capabilities. Such fidelity of dsDNA break repair would imply high degree of gene order conservation across genomes of deinococci. We have investigated the degree of synteny between 5 currently available completely sequenced genomes of genus Deinococcus. Whole genome alignments were performed by open source MUMer software implementing a suffix tree alignment algorithm. Maximal unique matching subsequences were detected for each genome pair and visualized as dot plots. Notably, the dot plot analysis revealed very low degree of synteny in all genome comparisons showing, therefore, that the ability of these deinococci to accurately reassemble DNA after extreme doses of gamma radiation is not mirrored in the degree of synteny between their genomes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D16</strong></td>
 </tr> <tr> 
  <td><b>Protein sequences of the human genome compared to those of several model organisms</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Edda Kloppmann <i>TU Munich</i></td>
 </tr>
 <tr> 
  <td>  Marco Punta (TU Munich and IAS, Informatics, Bioinformatics); Burkhard Rost (TU Munich, Bioinformatics and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In the last decade, the (nearly) complete DNA sequences of the human genome and numerous other organisms have become available. With this information in hand, large-scale comparative analysis has become feasible. Here, we present a sequence level comparison (PSI-Blast) between the annotated human protein sequences and those of well-known model organisms such as, among others, mouse, zebrafish, rice and yeast.<br><br>For assessing alignment similarity, we used different combinations of E-value, alignment length and sequence identity. We discuss differences and similarities in relation to the occurrence of several protein structural features, including transmembrane, coiled-coil and disordered regions. We found that even closely related organisms show significant diversity at the protein sequence level.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D17</strong></td>
 </tr> <tr> 
  <td><b>Integrating multiple species to identify essential promoter elements</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Julia Lasserre <i>Max-Planck Institute for Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>  Ho-Ryun Chung (Max-Planck Institute for Molecular Genetics, Computational Biology); Klaus-Robert Müller (Technische Universität, Machine Learning); Martin Vingron (Max-Planck Institute for Molecular Genetics, Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Regulation of gene expression encoded in the promoter is an essential characteristic of eukaryotes. Existing methods for extracting promoter features traditionally involve motif finding along a sequence in a species-specific manner. However, some of the detectable features arise from transcriptional mutational biases. Such biases occur as a consequence of the transcriptional process and differ between species. They do not constitute regulatory signals, yet they overcome them. This project aims at identifying generic regulatory sequence signals of polymerase II promoters across metazoan genomes. We therefore study promoters of several evolutionarily distant species together. By doing so, the goal is to emphasize the regulatory signals and to isolate them from a background of process-induced features. We use multi-task feature learning, a method that allows to extract features that are common to related datasets. We identify a set of words that are useful for the detection of promoters in all species.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D18</strong></td>
 </tr> <tr> 
  <td><b>OPE - Optimal Pairwise Epistasis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Benjamin Goudey <i>NICTA and University of Melbourne</i></td>
 </tr>
 <tr> 
  <td>  Armita Zarnegar (NICTA VRL, Australian Mathematical Society) Eder  Kikianty (NICTA VRL) Dave  Rawlinson (NICTA VRL) Qiao  Wang (NICTA) John Markham (NICTA) Richard Campbell (NICTA, University of Melbourne, Department of Electrical and Electronic Engineering); Michael  Inouye (The Walter and Eliza Hall Institute of Medical Research) Geoff Macintyre (NICTA, University of Melbourne,  COmputer Science and Software Enginering); Gad Abraham (NICTA, University of Melbourne,  Computer Science and Software Enginering); Izhak Haviv (Baker IDI Heart & Diabetes Institute) Adam Kowalczyk (NICTA) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motivation: Most genome-wide association studies (GWAS) to date use a single-locus analysis strategy, essentially ignoring the existence of interaction between SNPs. Previously published methods that account for interactions do not scale to real datasets, typically taking days or weeks of computation on non-commodity systems. In addition, most interaction methods are limited by only considering SNPs with strong marginal effects (statistically significant 1st order association with a phenotype).<br><br>We have developed a novel approach to assess all possible pairwise SNP interactions using rigorous computation of extreme tails of dedicated statistics. Analysis of a dataset with 300K SNPs takes <6 hour on a PC, with further improvements anticipated. <br><br>Results: We tested our approach on two independent GWAS studies of Celiac disease. In one study, one million statistically significant interactions were found, and over 98% were independently replicated in the second. The major histo-compatibility complex (MHC) and chromosome X were significantly enriched for interactions. Among replicated interactions, 1,000 had no significant marginal effect. Of autosomal, interacting SNPs with no marginal effect, 145 (from 151 pairs) were expression quantitative trait loci. Top putative interactions supported known mechanisms for T-cell attenuation and clinical targets for immunological control. Many interactions included SNPs not described previously as affecting Celiac disease risk.<br><br>Conclusion: We show that the calculation of higher order SNP interactions is feasible in reasonable time and that interaction is both pervasive in the genome and biologically relevant.  In particular, this makes  a systematic exploration of higher order interactions practical, especially with cluster or cloud computing.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D19</strong></td>
 </tr> <tr> 
  <td><b>QuartetS: a fast and accurate program for large-scale orthology detection</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jaques Reifman <i>US Army Medical Research and Materiel Command</i></td>
 </tr>
 <tr> 
  <td>  Chenggang Yu (Biotechnology HPC Software Applications Institute ) Nela Zavaljevski (Biotechnology HPC Software Applications Institute ) Valmik Desai (Biotechnology HPC Software Applications Institute ) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The unparalleled growth in the availability of genomic data offers both a challenge to develop orthology detection methods that are simultaneously accurate and high throughput and an opportunity to improve orthology detection by leveraging evolutionary evidence in the accumulated sequenced genomes. Here, we report a novel orthology detection method, termed QuartetS, that exploits evolutionary evidence in a computationally efficient manner. Based on the well-established evolutionary concept that gene duplication events can be used to discriminate homologous genes, QuartetS uses an approximate phylogenetic analysis of quartet gene trees to infer the occurrence of duplication events and discriminate paralogous from orthologous genes. We used function- and phylogeny-based metrics to perform a large-scale, systematic comparison of the orthology predictions of QuartetS with those of four other methods [bi-directional best hit (BBH), outgroup, OMA, and QuartetS-C (QuartetS followed by clustering)], involving 624 bacterial genomes and >2 million genes. We found that QuartetS slightly, but consistently, outperformed the highly specific OMA method and that, while consuming only 0.5% additional computational time, QuartetS predicted 50% more orthologs with a 50% lower false positive rate than the widely used BBH method. We conclude that, for large-scale phylogenetic and functional analysis, QuartetS and QuartetS-C should be preferred, respectively, in applications where high accuracy and high throughput are required.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D20</strong></td>
 </tr> <tr> 
  <td><b>Computational analysis of metagenomes and metatranscriptomes: challenges and opportunities.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Larisa Kiseleva <i>Okinawa Institute of Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  Igor Goryanin (Okinawa Institute of Science and Technology) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Recent advances in ultra-high throughput sequencing technologies, which do not require cloning or PCR amplification, and can produce huge numbers of DNA reads at an affordable cost, have boosted the number and scope of metagenomic sequencing projects. Now we can read DNA directly from environmental samples and compare the biological diversity and the functional activity of different microbial communities. While metagenomics provides information on the gene content, metatranscriptomic aims at understanding gene expression patterns of microbial population. Such meta- data has enormous potential in variety of areas: revealing novel sequences, genes, pathways; discovering potentially important enzymes; studying early events in the evolution of gene families etc.<br><br>Along with the progress in molecular sequencing methods, new powerful tools for computational analysis of obtained sequence data have been developed. This review will describe new methods for interactively exploring, analyzing and comparing multiple metagenomic datasets. We also apply our experimental data on several publicly available annotation pipelines and sequence analysis tools (1,2,3) and compare the obtained results.<br><br>References:<br>1. Gerlach et al., WebCARMA: a web application for the functional and taxonomic classification of unassembled metagenomic reads. BMC Bioinformatics 2009, 10:430.<br>2. Huson et al., MEGAN analysis of metagenomic data. Genome research 2007, 17(3):337-386.<br>3. Meyer et al., The metagenomics RAST server - a public resource for the automatic phylogenetic and functional analysis of metagenomes. BMC Bioinformatics 2008, 9:386.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D21</strong></td>
 </tr> <tr> 
  <td><b>A strategy integrating genomic and metabolic contexts to propose candidate genes for orphan enzymes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Adam SMITH <i>CEA/Genoscope</i></td>
 </tr>
 <tr> 
  <td>  Marcel SALANOUBAT (CNRS/UMR8030, CEA/Genoscope); Claudine MEDIGUE (CNRS/UMR8030, CEA/Genoscope); David VALLENET (CNRS/UMR8030, CEA/Genoscope); Alain VIARI (INRIA, Grenoble); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Despite an ongoing experimental and bibliographical effort, approximately 27% of all enzymatic activities recognised by the IUBMB are still orphan enzymes (i.e. metabolic reactions with no known protein/nucleic sequences associated) in the UniProt database. Sequence homology-based methods are traditionally used to transfer functional annotations to newly sequenced genes, and are obviously of no help in this case. Only integrative, context-based similarity-free methods can address this problem. Here, we present a new two-step method explicitly combining genomic and metabolic contextual information in order to propose candidate genes for orphan enzymes. A first step involves locating Genomic Metabolons (i.e. groups of co-localised genes coding proteins catalysing reactions belonging to a same metabolic process, allowing for gene and reaction gaps) using an exact algorithm, genome by genome. The second step integrates metabolons over 860 available prokaryotic genomes, in order to propose the best candidates for orphan enzymes. We have benchmarked our method computationally, as well as validating it by locating a candidate Escherichia coli K12 gene for an orphan enzyme involved in allantoin degredation. A web interface is under development to proffer these results to bio-analyst users of MicroScope, the prokaryotic comparative genomics platform. The method will thus help bioanalysts to discover new genes corresponding to orphan enzymes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D22</strong></td>
 </tr> <tr> 
  <td><b>Species-specific open chromatin has evidence of branch-specific selection.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nathan Sheffield <i>Duke</i></td>
 </tr>
 <tr> 
  <td>  Yoichiro Shibata (Duke, Institute for Genome Sciences and Policy); Olivier Fedrigo (Duke, Institute for Genome Sciences and Policy); Greg Wray (Duke, Institute for Genome Sciences and Policy); Greg Crawford (Duke, Institute for Genome Sciences and Policy); Terrence Furey (UNC, Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There is considerable evidence for differential gene expression among humans and other primates. These expression differences are likely probably partially caused by unknown differences in gene regulation. Mapping DNaseI hypersensitivity sites is a general measure of chromatin accessibility or openness, and is used to identify all types of regulatory elements, including promoters, enhancers, silencers, and insulators. We used DNase-Seq to identify all DNaseI hypersensitive sites in the same cell type (fibroblasts) isolated from human, chimp, and macaque. We identified approximately 1000 human- and chimp-specific increases and decreases in chromatin openness, which we polarize using the macaque outgroup. For the same samples, we also quantitated expression using DGE-seq. We find examples of chromatin differences that correspond to expression differences more often than expected. For example, presence of a species-specific DNase hypersensitive site is often correlated with species-specific expression of a nearby gene. We also tested regions of differential chromatin for sequence conservation and evidence of branch-specific positive selection; we found that DNase hypersensitive sites with species differences more often have evidence for positive selection on the differing branch.  This is true for both human-specific and chimp-specific differential open chromatin. Similarly, open chromatin that is shared across species contains more conserved elements than open chromatin that differs across species. These results suggest that sequence evolution corresponds with chromatin structure differences, which in turn may lead to altered gene expression. Our results also highlight specific cases with the testable hypothesis that differential gene expression is caused by functional mutations in regulatory sequence.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D23</strong></td>
 </tr> <tr> 
  <td><b>Comparative Genomics for expression QTL</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Steffen Moeller <i>University of Lübeck</i><br></td>
 </tr>
 <tr> 
  <td>Steffen Möller (University of Lübeck) Michael Brehler (University of Lübeck, Department of Dermatology); Georg Zeplin (University of Lübeck, Department of Dermatology); Saleh Ibrahim (University of Lübeck, Department of Dermatology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In a series of projects the genotype information of individuals is now combined with molecular phenotypes. When applied with transcriptomics or proteomics data, this allows to determine those chromosomal regions that are associated with the variation of a gene expression level. Such expression QTL, especially those located within QTL for clinical phenotypes, help formulating hypotheses for the etiology of polygenic diseases.<br><br>We have seen comparisons of expression QTL for different tissues of the same individuals and with the advent of recombinant inbred strains we also have the data to compare genetically identical individuals in different environments. This work provides the technical infrastructure for allowing such comparative analyses and, thinking this further, also supports the comparison across species. This was motivated by earlier work showing the strong connections between Multiple Sclerosis and its rodent animal model EAE.<br><br>Firstly, we were interested in orthologous genes that have expression QTL in both species. This is similar  to the direct comparison of differentially expressed genes between projects, but here we also have the direct notion of a location within a disease-associated chromosomal region in at least one of the species. Furthermore, we are interested in such controlling regions that are syntenic to each other. This would be indicative of the location of a molecular mechanism conserved between two animal modles. Gene homologies and genomic syntenies are contributed by Ensembl Compara. The system, an extension to the Interactive QTL System (TISQ), provides searches for syntenic regions controlling orthologuous genes both graphically and with statistical summaries.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='D24'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster D24</strong></td>
 </tr> <tr> 
  <td><b>Comparative genome analysis of completely sequenced Thermus scotoductus SA-01 and Thermus thermophilus HB27 and HB8</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Benjamin Kumwenda <i>University of Pretoria</i></td>
 </tr>
 <tr> 
  <td>  Derrick Litthauer (University of Free State, Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Although studies of thermophilic organisms dates back to 1960’s with over 50 species isolated world-wide, only three strains Thermus scotoductus SA-01, Thermus thermophilus HB8 and HB27 have been completely sequenced so far. It was impossible until now to do a comprehensive comparative genomic study of thermus species to gain more understanding on their natural transformation which is considerd to be their major way of survival in high temperature environments. This work does a comprehensive comparative genomic study and further investigates genome rearrangement and its effect on metabolic network clustering.<br> <br>Complete chromosome global alignment was analysed using M-GCAT software. Metabolic pathways were reconstructed using pathway tools software. Genome organisation was investigated by identifying breakpoints in SA-01 in comparison with HB8 and HB27. A cross-clustering coefficient was used to analyse levels of metabolic networks clustering. Genomic islands were predicted by SeqWord Sniffer, IslandViewer, IslandPick and IslandPath. Statistical parameters of accummulation of mutation in homologs as determined by BLAST were analysed.<br><br>Homologous genes in T. scotoductus have been found in both chromosomes and plasmids of HB8 and HB27. The three genomes are closely related and highly conserved with mutations around 2-3 nucleotides per 100 bp. Insignificant differences in the number of breakpoints were observed in  SA-01 with reference to HB8 and HB27, with HB27 having fewer breakpoints. This finding agrees with clustering results where HB27 is well clustered seconded by HB8. SA-01 was found to have the least number of genomic islands.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




