
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=C by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlistf77a.html?cat=C by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:17 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'C'- Chemical and Pharmaceutical Informatics'</b></td>
 </tr>
</table>

<a name='C01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C01</strong></td>
 </tr> <tr> 
  <td><b>Mining the FDA’s Adverse Event Reporting System (AERS) for Pathway Risk</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Mark Gosink <i>Pfizer Inc.</i></td>
 </tr>
 <tr> 
  <td>  Kjell Johnson (Pfizer Inc., Biostatistics); Cen Guo (University of Michigan, Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Enormous informatic efforts have gone into understanding the pathways of disease etiology. This work has led to a better understanding of the pathways and processes leading to disease and has suggested new therapeutic approaches. However, numerous compounds still fail in the clinic due to safety issues. To identify these safety issues, the Federal Drug Administration maintains a voluntary reporting database, AERS, of adverse events associated with drugs. Here, we describe our efforts to mine this data together with drug-to-gene and gene-to-pathway mappings to better understand the etiology of risk. We utilized a novel algorithm to identify drugs with higher than expected numbers of events for a given category of adversity. Next, the genes targeted by drugs in the highest risk category are mapped to their corresponding pathway. To avoid the potential bias which would arise for those drugs which map to multiple gene targets (i.e. gene families), a single drug could only map to a single pathway one time. Next pathway-adverse event pairs with higher than expected frequency are defined as high risk pathways. As might be expected, we found that serotonin signaling and dopamine signaling pathways have a high risk potential for neurologically-related events. We also identified immune-related pathways as contributing to hepatotoxicity as has been suggested in some recent reports.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C02</strong></td>
 </tr> <tr> 
  <td><b>The use of virtual screening reveals a viral-like polymerase inhibitor, which complex with the M. perniciosa DNA polymerase</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Bruno Andrade <i>State University of Feira de Santana</i></td>
 </tr>
 <tr> 
  <td>  Aristóteles Góes-Neto (State University of Feira de Santana, Biological Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The filamentous fungus Moniliophthora perniciosa (Stahel) Aime & Phillips-Mora is a Basidiomycota that causes witches' broom disease of cocoa (Theobroma cacao L.). In some cases, polymerases coded by mitochondrial plasmids may change the aging time of some fungal species. The mitochondrial DNA polymerase of M. perniciosa is classified within the B family of DNA polymerases, which can be found in viruses and organelles. The structure of this polymerase was previously constructed using a homology modeling approach. Using a virtual screening processes, accessing Kegg, ZINC and SEA databases we selected the 15 best probable nucleoside viral-like polymerase inhibitors, to test against DNA polymerase. Autodock Vina was used to perform docking calculations for each molecule, and returning energy numbers in several ligand conformations. After, we used Pymol to check presence or absence of hydrogen, stereochemistry of chiral carbons, substructure, superstructure, number of rotable bonds, number of rings, number of donor groups, and hydrogen bonding receptors. As a result we selected the Entecavir Hydrate, a hepatitis B polymerase inhibitor, and then AMBER 10 was used to describe the behavior of polymerase-entecavir complex after a set of 3500 ps of simulation up to 300 K in water. This calculation returned a graph of RMS x Energy during the time of simulation, and showed that the ligand remains inside the active site after this time with a final energy of -612587.4214 Kcal/Mol. Furthermore we can accept that Entecavir Hydrate could be a good inhibitor to be tested in vitro and in vivo against M. perniciosa.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C03</strong></td>
 </tr> <tr> 
  <td><b>Consensus docking method</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dariusz Plewczynski <i>University of Warsaw</i></td>
 </tr>
 <tr> 
  <td>  Michal Lazniewski (Medical University of Warsaw, Dpartment of Physical Chemistry, Faculty of Pharmacy); Krzysztof Ginalski (University of Warsaw, ICM); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We present the recent advances in our consensus docking approach to predict both protein-ligand complex structure and its binding affinity. VoteDock method uses as the input the results from seven independent docking programs (Surflex, LigandFit, Glide, GOLD, FlexX, eHiTS and AutoDock) that are widely used by community. The approach were evaluated on the extensive dataset of 1300 protein-ligands pairs from refined set of PDBbind database, for which the structural and binding affinity data are available. We compared independently ability of proper scoring by calculating Pearson correlation between docking score and experimental binding affinities, and posing by measuring RMSD of obtained conformations with native structure. <br><br>Consensus-based method is able to dock properly about 20% of pairs more than docking methods in average, and more than 10 % of pairs more than single best program. Also drop in RMSD of top scored conformation can be observed, with value 0.5Å lower than that of best individual program, namely GOLD. Similar increase in overall docking accuracy can be also observed for subsets created based on PDBbind, that explore various physico-chemical properties of ligands, their size or hydrophobic potential. Finally, we are able to boost the Pearson correlation of the predicted binding affinity in comparison to the experimental value up to 0.5, although scoring functions used as an input could not exceed 0.4 value.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C04</strong></td>
 </tr> <tr> 
  <td><b>Homology modeling and molecular docking studies of Mannosyl Transferase (pimB) in Mycobacterium tuberculosis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Karthik Maddi <i>Karolinska Institute</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Tuberculosis is caused by Mycobacterium tuberculosis (M.tb) and affects 2-3 million every year. First line of drugs prescribed for Tuberculosis include Isoniazid, Rifampcin, Ethambutanol, etc. targeting different enzymes involved in fatty acid synthesis and transcription.  With time, M.tb has developed many strategies to evade from these drugs by developing point mutations in the drug target enzymes leading to inefficient binding of the inhibitor and finally the development of extreme multidrug resistant strains. Multi drug resistant M.tb displays metabolic shifts to compensate for the above drug target enzymes. Drug target proteins which cannot be metabolically compensated are in high need of today. Among such targets, Mannosyl transferase is known to be crucial enzyme involved in Lipoarabinomannan (LAM) synthesis. LAM is a critical cell wall component, which is mannosylated by Mannosyl transferase.  The presence of Mannose caps on LAM is known to inhibit production of immune regulatory cytokines like IL-12 and TNF-?. For years, these enzymes have been ignored due to its complexity in structural characterization. Structural studies and inhibitors against these enzymes would enable us to treat increasing multidrug resistant strains of M.tb.Here we present the homology modeling and virtual screening studies of Mannosyl transferases, which can lead to development of a lead inhibitor.  ISMB has always been a common platform to exchange and share different scientific perspectives and knowledge. This poster at the conference might open the gate to develop Mannosyl transferases as good drug targets.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C05</strong></td>
 </tr> <tr> 
  <td><b>Metabolite Fate Prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chris Sinclair <i>Food and Environment Research Agency</i></td>
 </tr>
 <tr> 
  <td>  Robert Stones (Food and Environment Research Agency, Statstics and Informatics); Karen  Tiede (Food and Environment Research Agency, Ecochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There is a need to develop techniques that lead to a reduction in animals used in toxicity testing studies.  When organisms are exposed to chemicals in the environment, the chemical may be bioconcentrated, but subsequently some of these chemicals are metabolised. Ability of chemicals to be metabolised varies from easy (irrespective of species), to those that may be metabolised by specific species/taxa or are non-metabolisable. Objective of this study was to develop new software tools to validate this concept. Identify chemicals that are susceptible to biotransformation, and help improve the design of bioconcentration studies, leading to reduced animal testing.  Bioconcentration and biotransformation datasets in both mammalian and aquatic organisms have been collated from public literature.<br><br>Molecular fragment analysis of metabolic pathways has identified sub-structural moieties susceptible to biotransformation from in vivo and in vitro studies. We are investigating whether these are generically susceptible to biotransformation or restricted to individual taxa.  A standalone application has been developed which incorporates 2D molecular substructure matching. This allows the user to search for susceptible fragments within query molecules. <br><br>Methods of this study include:<br>•??Fragmenting molecules in each reaction step. Discard fragments that do not contain the reaction centre. Store fragment library in a database.<br>•??2D structural matching algorithm is then used to count the prevalence of unique fragments.<br>•??Stepwise discriminant analysis has been used to identify molecular properties and descriptors associated with chemicals that biotransform<br>•??Displaying biotransformation pathways (stored in database) for query molecule matches.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C06</strong></td>
 </tr> <tr> 
  <td><b>Homology Modeling and Molecular Dynamics Simulation of N-acetylglucosamine fosfomutase (AGM1) from Moniliphothora perniciosa</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Wagner Soares <i>State University of Southwest of Bahia</i></td>
 </tr>
 <tr> 
  <td>  Bruno Andrade (State University of Feira de Santana, Biological Sciences); Catiane Souza (State University of Feira de Santana, Biological Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: N-acetylglucosamine fosfomutase (AGM1) is directly related enzyme to the synthesis of cellular wall in Fungi. Due to its capacity to produce UDP-N-acetylglucosamine (UDP-GlcNAc) this is classified in the ?-D-fosfohexomutase superfamily. UDP-GlcNAc is a UDP (Uridine Diphosphate) sugar that serves as a biosynthetic precursor of glycoproteins, mucopolysaccharides and cell wall of bacteria and fungi. AGM1 catalyzes the conversion of N-acetylglucosamine-6-phosphate (GlcNAc-6-P) in N-acetylglucosamine-1-phosphate (GlcNAc-1-P). In addition, this enzyme can be used as a specific molecular target for drugs that could inhibit the growth and development of the pathogenic fungi Moniliophthora perniciosa, by blocking cell wall formation. After obtaining the protein sequence, an initial model was constructed using Homology Modeling approach. The best homologous template, 2DKA, presented a sequence with 83% identity with the target. Then, the first model was constructed by SWISS-MODEL, and refined by a set of Molecular Mechanics (MM) and Molecular Dynamics (MD) calculation, both using ff99 force field in Amber 10.0. The quality of resultant model was evaluated by PROCHECK 3.0, ANOLEA, and MD simulations. Ramachandran plot and MD simulations showed that the model has 96% of residues in the most favored regions with thermodynamic stability after 3000ps of simulation in water.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C07</strong></td>
 </tr> <tr> 
  <td><b>EC-BLAST: A novel tool for searching similar enzymes based on chemical knowledge</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Syed Asad Rahman <i>EMBL-European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Janet Thornton (EMBL-EBI, Thornton Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Enzymes can be classified by the kind of chemical reaction they catalyze. One such scheme of enzyme classification is defined by IUBMB. There have been efforts to classify enzymes using computational knowledge based on the existing enzymes classification data. Some methods have exploited enzymes sequence to achieve this goals while other have exploited the reaction type encoded by these enzymes. <br><br>We present a fast and efficient method called “EC-BLAST” to search and compare enzymes across various enzyme classes. This tool exploits the reactions’ substrate specificity and bond changes while comparing enzyme similarity. A user can choose an enzyme search for similar enzymes based on the conserved bond changes or similar substrate specificity or a combination of these queries. The results immediately highlight the conserved and variable regions in the enzymatic reactions, complementing knowledge obtained from protein sequence/structure based studies. This knowledge is then passed into our scoring function, which ranks the solution based on scores resulting from this function. The knowledge of annotated reaction centers from the processed reactions and further used to predict the possible enzymes, which can transform or act on a query molecules.<br><br>This tool will find application in various life science fields, and can be used to find alternate/missing enzymes in metabolic pathways, in drug discovery and toxicity based studies that not only require alternate pathways but also seek enzyme re-engineering. It can potentially provide a basic framework for both Academic and Industrial research and may eventually lead to a new classification system for enzymes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C08</strong></td>
 </tr> <tr> 
  <td><b>ChemMine Tools: an online service for analyzing and clustering small molecules</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Thomas Girke <i>University of California, Riverside</i></td>
 </tr>
 <tr> 
  <td>  Tyler Backman (University of California, Riverside, Botany and Plant Sciences); Yiqun Cao (University of California, Riverside, Computer Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: ChemMine Tools is an online service for small molecule data analysis. It provides a web interface to a set of cheminformatics and data mining tools that are useful for various analysis routines performed in chemical genomics and drug discovery. The service also offers programmable access options via the R library ChemmineR. The primary functionalities of ChemMine Tools fall into five major application areas: data visualization, structure comparisons, similarity searching, compound clustering and prediction of chemical properties. First, users can upload compound data sets to the online Compound Workbench.  Numerous utilities are provided for compound viewing, structure drawing and format interconversion. Second, pairwise structural similarities among compounds can be quantified. Third, interfaces to ultra-fast structure similarity search algorithms are available to efficiently mine the chemical space in the public domain. These include fingerprint and embedding/indexing algorithms.  Fourth, the service includes a Clustering Toolbox that integrates cheminformatic algorithms with data mining utilities to enable systematic structure and activity based analyses of custom compound sets. Fifth, physicochemical property descriptors of custom compound sets can be calculated. These descriptors are important for assessing the bioactivity profile of compounds in silico and QSAR analyses. ChemMine Tools is available at: http://chemmine.ucr.edu.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C09</strong></td>
 </tr> <tr> 
  <td><b>A systems approach to characterization of molecular targets of imaging agents</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ting Liu <i>Stanford University</i></td>
 </tr>
 <tr> 
  <td>  David  Paik (Stanford University, Biomedical Informatics & Radiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The development of novel imaging agents and the identification of their targets are areas of intense research and have a wide range of applications in both biomedical research and clinics. Here we show we can use a computational approach to characterizing the molecular target of an imaging agent. We quantitatively clustered proteins based on topological similarity of their imaging agents. To our knowledge, this is the first study that applies this approach to imaging agent data, although previous studies have validated this approach on pharmacological data. Our dataset consists of 400 small molecular imaging agents annotated into sets for over 80 protein targets. First, we demonstrated the feasibility of this computational approach on imaging agents by performing unsupervised clustering based on topology of imaging agents. Our result shows that ligands for the same protein target cluster together in a heatmap. We calculated a pairwise similarity score between every two sets of ligands. The similarity scores were expressed in a network with nodes being the targets. We further tested the model on an external set of EGFR imaging agents, and EGFR was returned as the top hit of a ranked list of molecular protein targets. Although experimental validation is needed, this systems approach can be used for effective characterization and identification of molecular targets of imaging agents.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C10</strong></td>
 </tr> <tr> 
  <td><b>Stereological and hormonal evidences of epithelial hyperplasia with improved fertility capacity induced by aqueous sesame leaves Intake in adult male sprague dawley rats accessory organs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>LUKEMAN ADELAJA JOSE shittu <i>Benue state university</i><br></td>
 </tr>
 <tr> 
  <td>LUKEMAN ADELAJA JOSEPH shittu (Benue state university) Remilekun Keji shittu (jireh international Laboratory, Gwagwalada, Abuja, microbiology units); Joseph A Olayode (ladoke akintola university, medical school, ogbomoso, anatomy); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: the study aimed to explore the phytoestrogenic lignans impact on fertility and on the male seminal vesicle, an androgeic organ contributing to about 60% seminal fluid and also being implicated in having immunoprotective activity, which could be a target that can be used to prevent antisperm antibodies,which is the main cause of male infertility with a rising prevalence in our environment. Thirty adult male rats were divided into 3 groups of 10 rats each. The treated groups received 28.0 mg /g bwt/day and 14mg/kg bwt of aqueous extract of sesame leaves via oral garvage respectively, while control group received equal volume of 0.9% normal saline per day for 6 weeks. Five microns ( 5um ) of uniformly random sections of processed seminal vesicular tissues were analyzed using an un-biased stereological study. SPSS analysis of data were generated with comparison using student`s t-test and Mann-whitney t-test.  P< 0.05 was considered statistically significant. Result:The mean animal weights and seminal vesicles weights including the volume fractions of  epithelial lining and lumen with exception of that stroma in the treated were significantly (P<0.05) higher than the control in a dose related manner. Thus, the mean volume fractions of epithelial lining and lumen of the seminal vesicle were 76% (P< 0.05) and 22% (P< 0.05) respectively in the low dose compared to control. Moreover, the testosterone level in the treated was significantly higher in the high dose than the control.Conclusion:Sesame intake increased seminal secretion and improve quality of semen in a dose related manner.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C11</strong></td>
 </tr> <tr> 
  <td><b>Curcumin reduces pulmonary tumorigenesis in human vascular endothelial growth factor A165 (hVEGF A165)-overexpressing transgenic mice</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hsaio-Ling Chen <i>Da-Yeh University</i></td>
 </tr>
 <tr> 
  <td>  Chuan-Mu Chen (National Chunag Hsing University,  Department of Life Sciences); Yu-Tang  Tung (National Chunag Hsing University,  Department of Life Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Curcumin is a major active polyphenol compound of turmeric and exhibits remarkable effects on cancers such as brain, colon, bladder, breast, prostate, and cervical cancers. In this study, a strain of transgenic mice carrying human vascular endothelial growth factor A165 (hVEGF-A165) gene to induce pulmonary tumor was used as an in vivo cancer therapy model. We found that curcumin significantly eliminated hVEGF-A165 overexpression to normal, specifically in clara cells of the lungs of transgenic mice, and suppressed the formation of tumors. In addition, we demonstrated a relationship between curcumin treatment and the expression of VEGF, EGFR, ERK2, and Cyclin A at the transcriptional and translational levels. We also noticed a reduction of Cyclin A and Cyclin B after curcumin treatment that was an effect on the cell cycle. Curcumin-induced inhibition of Cyclin A and Cyclin B likely results in decreased progression through the S and G2/M phases. These results demonstrated that the expression of proteins involved in the S to M phase transition in transgenic mice is suppressed by curcumin. Data suggest that a blockade of the cell cycle may be a critical mechanism for the observed effects on vasculogenesis and angiogenesis following treatment with curcumin.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='C12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster C12</strong></td>
 </tr> <tr> 
  <td><b>Drug-Induced Transcriptional Modules Across Human Cell Lines and Rat Liver</b><br/><br/></td>
 </tr>
 <tr> 
 <td>murat iskar <i>European Molecular Biology Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Monica Campillos (Helmholtz Zentrum, Institute of Bioinformatics and Systems Biology); Georg Zeller (EMBL, structural and computational biology); Vera  van Noort (EMBL, structural and computational biology); Kasia Kaminska (The International Institute of Molecular and Cell Biology in Warsaw, Laboratory of Bioinformatics and Protein Engineering); Peer Bork (EMBL, structural and computational biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Treating cells with drugs leads to complex transcriptional responses to these perturbations. The Connectivity Map (CMap) resource contains an extensive repository of gene-expression profiles from human cell lines treated with a large and diverse set of compounds. It was generated to reveal connections between genes, drugs and their mechanism of action. Analyzing this data, we have systematically studied the transcriptional modules induced upon drug treatment. These modules were identified with an unsupervised biclustering approach from the CMap data as well as from drug-induced gene-expression profiles from rat. To increase signal-to-noise ratio and reduce biases such as batch effects, we developed a pipeline including quality filtering and state-of-the-art normalization steps. In the normalized data, between 25 and 40 drug-induced transcriptional modules were discovered in human cell lines and rat liver. We found that more than 70% of these transcriptional modules were common to multiple cell lines. Subsequently, we systematically characterized these modules with the aim to link the regulated genes to the drug-centric information, such as drug-target interactions and side effects. For example, in a transcriptional module driven by antipsychotic drugs, the pathways of sterol and cholesterol biosynthesis were significantly altered. Interestingly, this connection between antipsychotic drugs and metabolic pathways may be an explanation for the metabolic side effects reported for these drugs. Thus, our work not only quantifies to which extent transcriptional reprogramming depends on the (cell-line or organismal) context, but also identifies novel associations between transcriptional modules, drug targets and side effects.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




