
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=X by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist8ba3.html?cat=X by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:19 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'X'- Systems Biology and Networks'</b></td>
 </tr>
</table>

<a name='X01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X01</strong></td>
 </tr> <tr> 
  <td><b>A regulatory network model of CD4+ lymphocytes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Luis Mendoza <i>Instituto de Investigaciones Biomédicas, UNAM</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There is a vast amount of molecular information regarding the differentiation of CD4+ lymphocytes, in particular regarding in vitro experimental treatments that modify the differentiation process. This publicly available information was used to infer the regulatory network that controls the differentiation process of CD4+ cells in mice. The network consist of 36 nodes and 62 regulatory interactions, representing the main signaling circuits established between molecules and molecular complexes regulating the differentiation of CD4+ cells. The network was converted into a continuous dynamical system in the form of a set of coupled ordinary differential equations and its dynamical behavior was studied. With the aid of numerical methods, five fixed point attractors were found for the dynamical system. These attractors correspond to the activation states observed experimentally for the precursor (naive), and effector (Th1, Th2, Th17 and Treg) CD4+ lymphocytes. The model is also able to describe the differentiation from a precursor to an effector type due to specific molecular signals. Moreover, taking a network instance as a representation for a single cell, I constructed a model of a cell culture where cells produce, and respond to, different molecular signals. The dynamical behavior of this virtual cell culture was analyzed, showing that it leads to cultures containing subsets of undifferentiated and differentiated cells in appropriate proportions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X02</strong></td>
 </tr> <tr> 
  <td><b>A nucleosomal approach to reconstructing histone modification network</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tu Le <i>Japan Advanced Institute of Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Post-translational modifications (PTMs) of histone proteins play critical roles in establishing functionally different domains on chromatin and regulating important biological processes, such as transcription. These modifications often act in cooperative manner, forming complicated “histone codes”. Elucidation of functional relationships among them will, therefore, significantly increase our understanding of cell differentiation, development, and cancer pathogenesis. Biological evidence has shown that nucleosome positioning can provide invaluable information about interactive effects of PTMs. However, to our knowledge, none of previous works has exploited this information in the reconstruction of histone modification networks.<br>We propose a computational approach based on Bayesian Network to reconstruct a dependence network representing functional relationships among histone modifications. Our approach employed the search-and-score method to infer the network structure using interactive information of histone modifications, which is measured by the correlation between each modification with nucleosome positioning. When applied on human CD4+ T cell ChIP-Seq dataset, containing 38 different histone modifications and binding information of three other proteins, H2A.Z, PolII and CTCF, our method not only outperformed previous approaches in recovering known relationships but also suggested many new ones, confirming its validity and efficiency. Our unbiased method for inferring the network structure can also be applied to reconstruct interaction networks of other epigenetic factors as well.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X03</strong></td>
 </tr> <tr> 
  <td><b>ALISSA-An Automated Live Cell Imaging System for Signal Transduction Analyses</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Heinrich Huber <i>Royal College of Surgeons in Ireland</i></td>
 </tr>
 <tr> 
  <td>  Heiko Dussmann (RCSI, Physiology); Perrine Paul (Institut Curie , UMR 144 ); Dimitris Kalamatianos (NUIM, Hamilton Institute ); Endl Martina (Siemens Austria, Central Technology); Jakub Wenus (NUIM, Hamilton Institute); Peter Wellstead (NUIM, Hamilton Institute); Jochen Prehn (RCSI, Physiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Probe photo bleaching and the specimen’s sensitivity to photo toxicity severely limit the frequency and duration of exposures during time-lapse fluorescent microscopy experiments. Consequently, when a study of cellular processes requires measurements over hours or days, temporal resolution is limited, and spontaneous or rapid events may be missed. We have developed ALISSA, an automated live-cell imaging system for signal-transduction analysis. It allows an adaptation of imaging  modalities and laser resources tailored to the biological process, and extends temporal resolution from minutes to seconds. ALISSA employs online image analysis to detect cellular events that are then used to exercise microscope control. Our system can be integrated into standard fluorescence microscopes and applied to a large range of single cell experiments by using a graphical language.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X04</strong></td>
 </tr> <tr> 
  <td><b>Interactive, multiscale navigation of large and complicated biological networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Thanet Praneenararat <i>The University of Tokyo</i></td>
 </tr>
 <tr> 
  <td>  Toshihisa Takagi (The University of Tokyo, Computational Biology); Wataru Iwasaki (The University of Tokyo, Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Biological networks in the post-genomic era often appear as “hair-balls” with a large number of extremely tangled edges, and thus cannot be visually interpreted. Therefore, a Google Maps-like (i.e., interactive and multiscale) navigation method is desperately needed for discovering knowledge from large and complicated biological networks. We present a navigation method that can rapidly and automatically abstract any portion of a large network to an immediately interpretable extent and enable researchers to interactively investigate regions of interest by zooming in and panning out, with an application to real yeast protein network data. The method relies on an ultrafast graph clustering and a biological-property-based clustering that utilizes biological information often provided for biological entities (e.g., Gene Ontology terms). We believe that this method would aid modern biologists faced with post-genomic datasets and lead to development of broad application in the emerging field of interactive interpretation of abundant data.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20079.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='X05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X05</strong></td>
 </tr> <tr> 
  <td><b>Finding out what happened -- Highlighting the essence of a biological experiment</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Georg Fuellen <i>University Rostock</i></td>
 </tr>
 <tr> 
  <td>  Gregor Warsow (Universität Rostock, Institute for Biostatistics and Informatics in Medicine and Ageing Research); Steffi Falk (Universität Rostock, Institute for Biostatistics and Informatics in Medicine and Ageing Research); Clemens Harder (Universität Rostock, Institute for Biostatistics and Informatics in Medicine and Ageing Research); Marcin Siatkowski (Universität Rostock, Institute for Biostatistics and Informatics in Medicine and Ageing Research); Som Anup (Universität Rostock, Institute for Biostatistics and Informatics in Medicine and Ageing Research); Paudel Yogesh (Universität Rostock, Institute for Biostatistics and Informatics in Medicine and Ageing Research); Boris Greber (MPI Münster, Department of Cell and Developmental Biology,); Hans Schöler (MPI Münster, Department of Cell and Developmental Biology,); Sandra Schordan (University Greifswald, Institute for Anatomy and Cell Biology); Nicole Endlich (University Greifswald, Institute for Anatomy and Cell Biology); Karlhans Endlich (University Greifswald, Institute for Anatomy and Cell Biology); Dirk Repsilber (Leibniz Institute for Farm Animal Biology, Biomathematics and Bioinformatics); Clemens Cap (University Rostock, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: To understand the response of cells and organisms to internal or external change of  conditions, experimentalists are generating an ever increasing plethora of differential and time series data. This data is often investigated by application of statistical or bioinformatics techniques yielding a set of candidate genes that may explain the observed changes. To improve the comprehension of the mechanisms behind the changes, we suggest to combine the data with existing knowledge from pathways or protein interaction networks, and developed ExprEssence, an easy-to-use Open-source Cytoscape plugin.  The plugin uses gene expression data to condense a large network by identifying the interactions most heavily influenced by the experiment. Furthermore, to obtain a better understanding of the mechanisms behind induction of pluripotency, we curated a protein interaction network called the PluriNetWork, available at WikiPathways. ExprEssence analysis of experiments in the context of this network allowed predictions that were  confirmed later on.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20128.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='X06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X06</strong></td>
 </tr> <tr> 
  <td><b>Topological network alignment uncovers biological function and phylogeny</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Natasa Przulj <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  Tijana Milenkovic (University of Notre Dame, Computer Science and Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There are thousands of genes in the human genome.  However, genes are just a means to an end: they produce different protein types that interact in complex networked ways and make our cells work.  Thus, network connectivity provides additional biological insight, over and above sequences of individual proteins. Hence, analogous to tools for aligning genetic sequences that have revolutionized biological understanding, network alignment tools are likely to have a similar groundbreaking impact.  We introduce a topology-based network alignment algorithm that exposes surprisingly large regions of network similarity even in distant species.  Substantial improvements are achieved when additional data sources (including sequence) are integrated with topology: surprisingly, 77.7% of yeast proteins participate in a connected subnetwork that is fully contained in the human network suggesting broad similarities in cellular wiring across all life on Earth.  Furthermore, we show that topology is a successful predictor of new cancer genes in melanogenesis-related pathways.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20130.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='X07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X07</strong></td>
 </tr> <tr> 
  <td><b>Inferring the role of alternative splicing on biological pathways: Studies using cancer transcriprome data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Murlidharan Nair <i>Indiana University South Bend</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The central goal of cancer biology is to understand how the acquired genetic changes become responsible for deregulated cell growth and differentiation at a molecular level. Cancer is not one disease but a family of diseases that makes early detection extremely difficult. Rapid development in high-throughput molecular techniques has resulted in the accumulation of vast amounts of cancer transcriptome data. Information from these data sets are capable of revealing the disruption of normal cellular processes, and help compare the molecular picture of pathways in a tumor cell versus normal cell. Since genes in eukaryotes are split into exons and introns, with splice variants displaying different and even antagonistic biological functions, demonstrating the presence or absence of a particular splice variant in the cancer transcriptome data is vital to understanding its influence on the cellular process that it mediates. Aberrant splicing events have been associated with several cancers and there is increasing evidence that they could serve as markers for profiling the diseased state. In this poster, we present the preliminary work directed to extract information on signaling pathway deregulation, expression modules, and to derive isoforms of genes involved in the deregulated pathways. The measure of pathway activity from transcriptome data has been approached by the statistical analysis of the enrichment of binding motifs for transcription factors involved in the pathway. Pathways are characterized by the transcription factors that mediate it.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X08</strong></td>
 </tr> <tr> 
  <td><b>Genetic Design Through Branch and Bound</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dennis Egen <i>Rutgers University</i></td>
 </tr>
 <tr> 
  <td>  Desmond Lun (Rutgers University, Computer Science Camden); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Since the advent of genome scale models of metabolic networks, several computational methods have been developed that yield engineered strains overproducing certain metabolic targets. Many of these methods seek to find gene deletion (knockout) strategies that result in overproduction while maximizing the cell's biological objective.  Finding such gene knockout strategies is a complex problem in which computational time can grow exponentially as the number of possible knockouts or manipulations increases. <br><br>In this work, we introduce Genetic Design through Branch and Bound (GDBB), a novel heuristic approach to finding favorable knockout strategies. As in previous approaches to finding knockout strategies, GDBB poses the problem as a bi-level program, which is converted into a single level mixed integer linear program (MILP). GDBB uses a novel truncated branch and bound approach to find solutions significantly faster than previous approaches. In many cases, GDBB improves on the computational time of previous methods by several orders of magnitude--finding in seconds solutions that would otherwise take hours or days or more.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X09</strong></td>
 </tr> <tr> 
  <td><b>Bridging clinical knowledge to molecular events through integration of imaging biomarkers and protein interactions in Alzheimer’s disease</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Erfan Younesi <i>Fraunhofer Institute for Algorithms and Scientific Computing</i></td>
 </tr>
 <tr> 
  <td>  Martin Hofmann-Apitius (Fraunhofer Institute for Algorithms and Scientific Computing, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Numerous studies have applied imaging techniques and have investigated molecular mechanisms in the field of neurodegenerative diseases but the challenge is to validate imaging outcomes by elucidating relationships between imaging biomarkers and the corresponding pathophysiological pathways. We have tackled this problem by integrating imaging and molecular biomarkers of Alzheimer’s disease (AD), extracted from literature, and mapping them onto a human brain protein interaction network annotated to 16 brain regions. <br>With the help of state-of-the-art text-mining technologies, we retrieved PubMed abstracts in the context of diagnostic imaging (including MRI, DTI, PET) in relation to AD and extracted the information of affected brain regions, and assigned those regions to three stages of the disease, namely mild cognitive impairment, early Alzheimer’s, and acute Alzheimer’s. Over-representation of affected temporal lobe in all stages led us to generate a subnetwork specific to temporal lobe region out of the whole brain protein interactome. <br>This subnetwork was subjected to pathway analysis using Alzheimer’s molecular biomarkers as anchors and gene expression data under the disease condition. Gene Set Enrichment Analysis (GSEA) was performed to match enriched biomarkers to their corresponding pathways when upregulated genes were mapped onto these dissected pathways. The results indicate that, in contrast to widely accepted dominant hypothesis of the amyloid pathway in AD, particular pathways not appreciated before to be causally related pathways in the molecular etiology of AD are upregulated in the affected temporal lobe including NEF signaling, apoptosis, and FAS signaling pathways.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X10</strong></td>
 </tr> <tr> 
  <td><b>Network-based approach towards candidate gene prioritisation using TargetMine</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Lokesh Tripathi <i>National Institute of Biomedical Innovation</i></td>
 </tr>
 <tr> 
  <td>  Yi-An Chen (National Institute of Biomedical Innovation, Bioinformatics Project); Kenji Mizuguchi (National Institute of Biomedical Innovation, Bioinformatics Project); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Prioritising suitable targets (genes, proteins etc.) for further experimental characterisation is a challenging task in understanding gene function, drug discovery and biomedical research. The biological role of a gene is determined by its sequence, structure, when and where it is expressed and its interactions with other biomolecules in cellular networks. Therefore, an integrated investigative approach that combines information from multiple data types is best suited for optimal target discovery. In view of the merits of an integrated data repository, we have developed TargetMine, an integrated data warehouse for selection of target genes and proteins for experimental characterisation and drug discovery. TargetMine is based on the flexible InterMine framework and thus effectively combines biological data from several public resources. It enables biological data gathering and data analysis in a single user-friendly interface and thus, a useful tool to assist in candidate gene selection. TargetMine has been effectively employed for target selection in a protein interaction network-based investigation of Hepatitis C virus (HCV) pathogenesis. Follow-up experimental investigations on selected candidates validated three novel potentially useful targets for anti-HCV strategies. The planned inclusion of additional data sources and analytical tools will further enhance the ability of TargetMine to mine biological repositories for better target discovery. TargetMine academic version is available at http://targetmine.nibio.go.jp/.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X11</strong></td>
 </tr> <tr> 
  <td><b>Integrated cell cycle and DNA repair signalling network modelling for identification of key molecular regulators in basal-like breast cancer</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Inna Kuperstein <i>INSERM - Mines ParisTech - Institut Curie</i></td>
 </tr>
 <tr> 
  <td>  Paola Vera-Licona (INSERM - Mines ParisTech - Institut Curie, Bioinformatics and Computational Systems Biology of Cancer); Andrei Zinovyev (INSERM - Mines ParisTech - Institut Curie, Bioinformatics and Computational Systems Biology of Cancer); Gordon  Tucker ( Servier, Institut de Recherches); Thierry Dubois (INSERM - Mines ParisTech - Institut Curie, Département de Transfert); Emmanuel Barillot (INSERM - Mines ParisTech - Institut Curie, Bioinformatics and Computational Systems Biology of Cancer); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Basal-like breast cancer (BLC) is associated with a poor prognosis and there is a lack of targeted therapy. Pathways involved in cell cycle and DNA repair are highly perturbed in BLC. To understand orchestration between cell cycle and DNA repair molecular mechanisms, we used a systems biology approach to represent biological processes as comprehensive models based on experimental data retrieved from literature and transcriptomic data. <br>The network is created using the CellDesigner software, which is adapted to further mathematical modelling of signalling network dynamics. We have constructed an integrated cell cycle and DNA repair molecular signalling network composed of three interconnected layers. The first layer represents core cell cycle pathways and checkpoint proteins. The second layer includes DNA repair pathways. The third layer is composed of common regulators and modulator enzymes for cell cycle and DNA repair that ensure reciprocal influence between these processes. We further integrated transcriptomic data from breast tumours into the network and highlighted specific pathways modified in the disease. To verify the network, we simulated, in silico, the familial BRCA1-negative phenotype and inhibition of the base excision repair protein PARP to prove that our model recapitulates some well-described data. <br>A comprehensive reconstruction of the cell cycle and DNA repair signalling network allows integration of multiple crosstalks between DNA repair and cell cycle. Mathematical modelling of the network will bring a better understanding of dynamic regulatory circuits. The network will be used for discovering key players in breast cancer progression to induce synthetic lethality of malignant cells.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X12</strong></td>
 </tr> <tr> 
  <td><b>An interface analysis of human E2-E3 complexes: structural insights into specificity</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gozde Kar <i>Koc University</i></td>
 </tr>
 <tr> 
  <td>  Ozlem  Keskin (Koc University, Computational Biology & Bioinformatics); Attila Gursoy (Koc University, Computational Biology & Bioinformatics); Ruth Nussinov (Basic Research Program, SAIC-Frederick, Inc., Center for Cancer Research Nanobiology Program); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Ubiquitination is crucial for protein degradation in eukaryotic cells. Ubiquitin attachment takes place via a sequential enzymatic cascade involving ubiquitin-activation (by E1 enzymes), ubiquitin-conjugation (by E2 enzymes), and ubiquitin substrate-tagging (by E3 enzymes). E3 ligases mediate ubiquitin transfer from E2s to substrates and as such confer substrate specificity. Although E3s can interact and function with numerous E2s, it is still not clear how they choose which E2 to use. Identifying all E2 partners of an E3 is essential to infer principles guiding E2 selection by an E3. Here, we model the interactions of E3 and E2 proteins in a large, proteome-scale strategy based on interface structural motifs, which allows elucidation of both which E3s interact with which E2s in the human ubiquitination pathway and how they interact. Interface analysis of E2-E3 complexes reveals that loop L1 of E2s is critical in binding; the sixth position residue in loop L1 is structurally conserved and indispensible for E2 interactions. Loop L1 residues also confer specificity to the E2-E3 interactions; we observe diverse binding behavior of E3 families; HECT E3s utilize loop L1 of E2s distinctively from RING finger type E3s. These findings might be important in finding the details of specific binding regions of drug candidates targeting E3s, several of which have been implicated in disease processes, including cancer.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X13</strong></td>
 </tr> <tr> 
  <td><b>How little do we actually know? – Towards the size of gene regulatory networks.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Richard Röttger <i>Max Planck Institute for Informatics</i></td>
 </tr>
 <tr> 
  <td>  Jan Baumbach (Max Planck Institute for Informatics, Computational Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The NCBI recently announced "1,000 prokaryotic genomes are now completed and available in the Genome database". Knowing the genes' locations and their function only is just part of the whole task. Equally important is to decipher the regulatory mechanisms that control how cells survive, reproduce and adapt their behavior according to changing environmental conditions. One major control mechanism is transcriptional gene regulation. Here, even the few model organisms, for which experimental data on their gene regulations exist, are far away from being complete. But how much do we really know, how much is still missing?<br><br>We basically estimate the size of the nature given entire network by calculating the edge probabilities of the known part of the network. We differentiate between three different kinds of regulatory connections: (1) transcriptional factors (TF)->TF, (2) self loops of TFs and (3) TF->non-TF genes. With these edge-probabilities we are able to predict the size of the underlying regulatory network. To validate the correctness of our approach, we generated sub-subnetworks of different sizes and used these samples to calculate the size of the original, known subnetwork. Furthermore we facilitated bootstrap methods in order to give approximate confidence intervals for our estimations.<br><br>Our results show that Escherichia Coli has about 9k regulatory connections, whereas Bacillus Subtilis possesses only 5.5k of these regulations which results in a known-to-unknown ratio of 35% and 25%, respectively. Complex mammals like Homo Sapiens or Mus Musculus possess networks with estimated 500k and above regulations what corresponds to a ratio below 1%.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X14</strong></td>
 </tr> <tr> 
  <td><b>Identifying Differentially Regulated Subnetworks from Phosphoproteomic Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Martin Klammer <i>Evotec Munich</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: INTRODUCTION<br>Protein phosphorylation is one of the most important posttranslational modifications in a living cell, particularly in signal transduction. To identify phosphorylation sites (phosphosites) on a large scale, mass spectrometry (MS) has become an increasingly important technology.<br>Integrating data from global quantification technologies with protein networks should make it possible to identify subnetworks that are significantly regulated. In particular, processing mass spectrometry-based phosphoproteomic data in this manor may expose signal transduction pathways and reveal a drug's mode of action.<br><br>METHODS<br>Here, we introduce SubExtractor, an algorithm that combines phosphoproteomic data with protein network information from STRING to identify differentially regulated subnetworks. The method is based on a Bayesian probabilistic model combined with a genetic algorithm and rigorous significance testing. The Bayesian model accounts for information about both differential regulation and network topology (i.e. neighborhood information). Significance testing is based on a global rank test, which has been proven to reliably identify even low fold-changed significant entities. <br><br>RESULTS<br>The method was tested with artificial data and subsequently applied to a comprehensive phosphoproteomics study investigating the mode of action of sorafenib, a small molecule kinase inhibitor. The results of this study led to creating new hypotheses about the mode of action of sorafenib in prostate cancer PC3 cells, including a possible effect on the mTor pathway.<br><br>CONCLUSION<br>SubExtractor reliably identifies differentially regulated subnetworks from phosphoproteomic data by integrating protein networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X15</strong></td>
 </tr> <tr> 
  <td><b>Biological Networks Analysis and Graph Edit Distance</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Rashid Ibragimov <i>Max-Planck-Institut für Informatik</i></td>
 </tr>
 <tr> 
  <td>  Jan Baumbach (Max-Planck-Institut für Informatik, Computational Systems Biology); Jiong Guo (Max-Planck-Institut für Informatik, Cluster of Excellence Multimodal Computing and Interaction); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Many processes taking place in a cell or in an organism can be modeled as networks. Data for building such biological networks is provided by different sources and represents different types of interactions, for example, protein complexes, protein-protein interactions, gene regulations, etc. The ability to compare and analyze these networks is of very high impact in Computational Systems Biology. It is important to identify similar subnetworks, in order to perform inter-species knowledge transfers. This kind of network similarity analysis becomes even more challenging when one deals with noisy and diverse data.<br>We model this problem as Graph Edit Distance (GED) problem, i.e. finding the minimal amount of necessary distortions for transforming one graph into another. However, the GED problem is of high computational complexity. Here, we first show that it is non-fixed parameter tractable and APX, which means that in practice it is impossible to find exact solution. Afterwards, we will introduce an approximative method for solving GED and first applications to protein complex data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X16</strong></td>
 </tr> <tr> 
  <td><b>A Directed Graphical Gaussian Model for Inferring Gene Regulatory Networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tomoshige Ohno <i>Osaka University</i></td>
 </tr>
 <tr> 
  <td>  Kiyoshi Yoshizawa (Osaka University, Department of Bioinformatic Engineering, Graduate School of Information Science and Technology); Shigeto Seno (Osaka University, Department of Bioinformatic Engineering, Graduate School of Information Science and Technology); Yoichi Takenaka (Osaka University, Department of Bioinformatic Engineering, Graduate School of Information Science and Technology); Hideo Matsuda (Osaka University, Department of Bioinformatic Engineering, Graduate School of Information Science and Technology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Inference of the gene regulatory networks from gene expression profiles is a significant area in the field of bioinformatics. Graphical Gaussian models (GGMs) are utilized as an approach to inferring gene regulatory networks. The GGMs measure likelihood of regulations using partial correlation coefficients in order to determine regulatory relationships between two genes. The conventional GGMs, however, are not capable of estimating a directed network. To address this issue, we propose an advanced method based on the GGM. The objective of this study is to enable the GGM to infer directed gene regulatory network from time-series gene expression profiles. <br>The proposed method consists of three steps. At first genes are conceptually duplicated into multiple nodes based on the number of detention in order to quantify time-lagged regulations. Secondly, an undirected network model is estimated using a conventional GGM. Finally, the direction of each edge is determined according to the time lag. Nodes corresponding to an identical gene are united by contraction. Thus a directed gene regulatory network model can be obtained.<br>To evaluate the effectiveness of our method, we validated the performance with time-series gene expression profiles of yeast cell cycle data. The results showed the proposed method is capable of inferring directed gene regulatory networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X17</strong></td>
 </tr> <tr> 
  <td><b>Sensitivity Analysis on Caspase-1 Pathway</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Cemre Kocahakimoglu <i>Koc University</i></td>
 </tr>
 <tr> 
  <td>  Ahmet Gul (Istanbul University, Division of Rheumatology, Department of Internal Medicine, Istanbul Faculty of Medicine); Burak Erman (Koc University, Chemical and Biological Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There have been many studies related to caspase-1 (Interleukin converting enzyme, ICE) upstream and downstream interactions. Among its several roles, caspase-1 acts as a thiol protease that cleaves proIL-1 beta to produce its active IL-1 beta form. Caspase-1 is processed from procaspase-1 in NLRP1 (NACHT, LRR and PYD-containing protein 1), NLRP2, NLRP3, NLRC4 (CARD, LRR, and NACHT-containing protein) or AIM2 inflammasomes. Each inflammasome is triggered by different stimulants but activates the same enzyme, which is caspase-1. Inflammasome stimulating factors can vary from various bacterial antigens to different crystals or endogenous danger signals. NF-kappa B has an important role in the ICE pathway by inducing the production of proIL-1 beta. In addition to NF-kappa B; RIG-I, TLRs (Toll like receptors), inhibitors such as Iceberg, and pyrin/marenostrin are some of the proteins involved in the ICE pathway. In this study, all interactions in the ICE pathway taken from literature data are organized in a connectivity Kirchhoff matrix and the properties of the resulting graph are analyzed using the Gaussian Network Model (GNM). A sensitivity analysis on the caspase-1 pathway is performed using GNM. Affinities between ith and jth proteins (<deltaR_ij>^2) are considered in the sensitivity analysis. In previous studies related to GNM, residue based interactions in a protein are considered. However, protein level interactions through affinities in a pathway have not been analyzed before by GNM. The present study extends the use of GNM to the analysis of  interactions in the ICE pathway.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X18</strong></td>
 </tr> <tr> 
  <td><b>Fast and Efficient Dynamic Nested Effects Models</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Holger Fröhlich <i>University of Bonn, Bonn-Aachen Int. Center for IT</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Targeted interventions in combination with the measurement of secondary effects can be used to computationally reverse engineer features of upstream non-transcriptional signaling cascades. Nested effect models (NEMs; Markowetz et al., 2005) have been introduced as a statistical approach to estimate the upstream signal flow from downstream nested subset structure of perturbation effects. The method was substantially extended later on by several authors and successfully applied to various datasets. The connection of NEMs to Bayesian Networks and factor graph models has been highlighted. <br><br>Results:<br>Here we introcude a computationally attractive extension of NEMs that enables the analysis of perturbation time series data, hence allowing to discriminate between direct and indirect signaling and to resolve feedback loops. It complements the method of Anchang et al. (2009) to extend static NEMs to the modeling of perturbation time series measurements. In contrast to Anchang et al. the key idea in our model is to unroll the signal flow over time. This allows for a computation showing some similarity to Dynamic Bayesian Networks and naturally extends the classical NEM formulation introduced by Markowetz et al. In our model we circumvent the need for any time consuming Gibbs sampling, which makes it also computationally attractive.<br><br>Extensive simulations highlight the robustness of our approach. Results on a dataset from murine stem cell development in mice (Invanova et al., 2006) show high consistency with the literature.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X19</strong></td>
 </tr> <tr> 
  <td><b>Semantic signature: comparative interpretation of gene expres-sion microarray experiments on a semantic space</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jihun Kim <i>Seoul National University</i></td>
 </tr>
 <tr> 
  <td>  Keewon Kim (SNUBI, College of medicine); Sun Min Yun (SNUBI, College of Medicine, SNU); Chan Hee Park (SNUBI, College of Medicine, SNU); Ju Han Kim (Systems Biomedical Informatics Research Center, Div. of Niomedical Informatics, College of Medicine, SNU); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motivation: Comparative analysis is important for evaluating causality and association and benefiting from the growing number of microarray data. Methodologies, developed so far for comparative microarray data analysis, have limitation of only relative comparison of given information with other datasets and are restricted by platforms and methods applied. <br>Results: We have constructed a semantic space based on the relative semantic distances among molecular signature gene sets, which served as a frame of reference for comparative microarray data analysis. Mapping concept lattices created from microarray data on the semantic space as the frame of reference enables comparative studies for the different contexts of microarray experiments. We extracted distinctive patterns that are repeatedly observed in the concept lattices of associated experiments. Visualization of concept lattices on the semantic space provides an ‘executive summary’ of the whole contexts of microarray experiments. Comparing different geographies of concept lattices from different categories of microarray experiments revealed ‘the semantic signatures’: specific geography of concept lattice on the semantic space disclosing its phenotype.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X20</strong></td>
 </tr> <tr> 
  <td><b>bioCompendium: high-throughput experimental data analysis platform</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Venkata Satagopam <i>European Molecular Biology Laboratory (EMBL)</i></td>
 </tr>
 <tr> 
  <td>  Jean-Karim Heriche, (EMBL, Cell Biology and Biophysics); Reinhard  Schneider (EMBL, Structural and Computational Biology  ); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We have developed a high throughput experimental data analysis platform, 'bioCompendium'. The basic input to the system are one or more gene lists, which are typically the basis for an experiment or are the outcome of a wide range of experiments like gene expression analysis, RNAi screening results, proteomics experiment and 'cross-species comparison' experiments.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X21</strong></td>
 </tr> <tr> 
  <td><b>A systems-level study of giant regulation in Drosophila melanogaster</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Astrid Hoermann <i>Centro de Regulación Genómica</i></td>
 </tr>
 <tr> 
  <td>  Hilde Janssens (Centro de Regulación Genómica, EMBL/CRG Research Unit in Systems Biology); Johannes Jaeger (Centro de Regulación Genómica, EMBL/CRG Research Unit in Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Eukaryotic transcription is very complex, and we are far from a satisfactory biochemical understanding of it. We aim to use data-driven mathematical modeling to investigate how different binding sites form a regulatory element, and how these elements then together establish the expression pattern of the endogenous gene. These questions are addressed by quantitative analysis and mathematical modeling of the expression pattern of the gap gene giant (gt) in the Drosophila melanogaster blastoderm. It is expressed in a posterior and an anterior domain, which refines into two stripes over time, and finally also expression at the anterior tip is visible. The regulatory region of gt contains binding sites for the maternal activators Bicoid and Cadual, and repressor sites for other gap genes, such as Krueppel and Hunchback. Seven partially overlapping cis-regulatory elements (CRE) for gt were previously predicted through detection of clusters of binding sites. I will quantify their expression patterns with a high-throughput method for extracting concentrations from confocal microscope images and then feed this data into a mathematical model of transcriptional control in order to predict spatio-temporal expression patterns of CREs from sequence. The model gives the combinations of binding sites required for correct expression as output. It can predict expression patterns for all kinds of modifications and combinations within the regulatory regions, and the most interesting ones will then be verified in vivo with constructs harbouring mutations in binding sites, or by testing a CRE in a mutant background lacking a certain transcription factor.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X22</strong></td>
 </tr> <tr> 
  <td><b>Detection of nonlinear effects in gene expression pathways</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andreas Mayr <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>  Djork-Arné Clevert (Johannes Kepler University Linz, Institute of Bioinformatics); Sepp Hochreiter (Johannes Kepler University Linz, Institute of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One of the main topics in systems biology is to model genetic pathways. Genes of a pathway, which show linear dependencies of their expression values, are easy to identify to belong to the pathway. However, if feedback loops or signal cascades are present, gene expression values of pathway genes can be nonlinearly dependent on the expression values of other genes in the pathway. In this situation such genes are hard to detect as belonging to the pathway because nonlinearity and noise must be distinguished.<br><br>We propose an algorithm to infer nonlinear network elements in pathways from microarray data. Our model assumes, that gene expression values, belonging to one pathway, are mainly driven by one single latent factor. We expect two groups of genes in a pathway: genes belonging to the first group are linearly dependent on the hidden factor, genes from the other group show a nonlinear dependence from the latent variable. The goal is to identify the kind of dependence from the hidden factor.<br><br>Our algorithm for detecting nonlinear effects is an extension of linear Gaussian factor analysis. Nonlinearities are modelled by the square of the latent variable weighted by specific coefficients. We derived a novel model selection method for this generalization of factor analysis. To avoid the interpretation of noise as nonlinearity, we determine p-values that measure the probability of a linear gene being detected by chance as nonlinear. <br><br>We apply our algorithm to microarray data of breast cancer samples, where we identified nonlinear dependencies of gene expression values in the p53  pathway.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X23</strong></td>
 </tr> <tr> 
  <td><b>Network topology complements sequence as a source of biological information</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tijana Milenkovic <i>University of Notre Dame</i></td>
 </tr>
 <tr> 
  <td>  Natasa Przulj (Imperial College London, Computing); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Individual genes are just a means to an end: they produce proteins that interact in complex networked ways and make our cells work. Thus, network topology provides biological insight over and above sequences of individual proteins. We argue that sequence and topology uncover complementary biological information, which sometimes corroborate each other, but sometimes do not. The advancement relies on developing sophisticated graph-theoretic methods for extracting biological knowledge from topology before being integrated with other biological data types (e.g., sequence). Since many graph-theoretic problems are computationally infeasible, heuristic algorithms for network analysis are sought.<br><br>Analogous to sequence alignments, alignments of biological networks will impact biological understanding. Our topology-based network alignments expose surprisingly large regions of network similarity even in distant species.  Substantial improvements are achieved by integrating additional data sources (including sequence) with topology: surprisingly, 77.7% of yeast proteins participate in a connected subnetwork that is fully contained in the human network, suggesting broad similarities in cellular wiring across all life on Earth. We extract from our alignments protein function and species phylogeny. Furthermore, we find that aging, cancer, pathogen-interacting, drug-target and genes involved in signaling pathways are topologically "central" in the network, occupying dense network regions and "dominating" other genes in the network. Also, we show that cancer and non-cancer genes have different "topological signatures" and when we integrate topological with functional-genomics data, we successfully predict new cancer and melanogenesis-related pathways.  Hence, topology, an invaluable source of biological information, can suggest novel drug targets and impact therapeutics and health care.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X24'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X24</strong></td>
 </tr> <tr> 
  <td><b>Understanding temporal patterns in cell cycle regulation through visualization</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Maria Secrier <i>European Molecular Biology Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Reinhard Schneider (European Molecular Biology Laboratory, Structural and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Highly time-dependent biological processes like the cell cycle pose interesting questions in terms of visual representation of temporal events and the overall understanding of dynamic aspects of regulation. We look at the projection of the phenotypic space of cell division defects unto the temporal landscape of cell cycle regulation and then place it in the larger context of evolution.<br>The temporal succession of phenotypes in cell populations upon knockdown of genes essential to cell division is described. By visualizing patterns of these time-driven events in relation to overrepresented GO terms, we can get substantial insight into the key triggers of phenotypic changes and generate hypotheses about gene co-activation/co-regulation or participation in the same pathway. This will in turn enable potential linking of diseases that have common regulators and the discovery of connections among drugs that trigger similar phenotypes.<br>To get an evolutionary perspective of timing in the cell cycle, we analyze how transcriptional activation levels compare in different organisms. This allows us to understand how the human homolog landscape maps to the cell cycle events. By comparing the degree of conservation of genes that are being actively transcribed during the cell cycle, one can identify temporal hotspots of novel activation events in human.<br>Visualization tools to help capture all these aspects are presented.<br>The ultimate goal is to create a time-encoded mapping of genotype-phenotype interactions in the context of the cell cycle, for a better interpretation of how a system’s dynamics can shape the morphology and interaction topology of the cell.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X25'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X25</strong></td>
 </tr> <tr> 
  <td><b>From WIPA (Web Interface Protein Atlas) to Systems Biology Analysis of Protein Atlas Tissue Microarrays (TMAs)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hans-Juergen Thiesen <i>University of Rostock</i></td>
 </tr>
 <tr> 
  <td>  Bjoern Ziems (Gesellschaft für Individualisierte Medizin, Image Analysis); Guoqing Zhang (Shanghai Center for Bioinformation Technology, DatabaseDevelopment); Glen Kristiansen (University Hospital Zuerich, Institute of Clinical Pathology); Michael Kreutzer (University of Rostock, Institute of Immunology); Yixue Li (Shanghai Institutes for Biological Sciences, Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Introduction: The Protein Atlas database (www.proteinatlas.org) presents a unique reservoir of toponome data sets (13154 antibodies  staining 10118 proteins, see Proteinatlas Version 7.0). The WIPA interface (www.toponostics.org) recently established visualizes immunohistochemical (IHC) Protein Atlas protein expressions in 47 human tissues and 20 cancers with the intention to support innovative systems biology approaches. Hereto, protein expression levels of IHC Protein Atlas images quantitated in normal and cancer kidney tissues were found to be correlated with renal cancer type and tumor grade.<br>Method: Proteins differentially expressed in IHC images of human kidney tissues and cancer were quantitated and compared by making use of the mean, standard deviations (stdv) or the coefficient of variation (cv). Any combination of quantitative antibody expression levels were correlated to type and grade of individual renal cancer samples assessed by an expert in pathological routine diagnosis. <br>Results: Protein expression levels of 38674 images of normal kidney and 293038 renal cancer images were quantitated derived from 43 kidney tumor samples of which 2 displayed grade 1, 16 grade 2, 12 grade 3 and 1 grade 4 clear cell renal cellular carcinoma (ccRCC) besides 12 of 4 other renal cancer types. 12 ccRCC cancer tumors visualized by IHC of 2218 antibodies led to the identification of antibody sets that discriminate G1/G2 tumor grades from G3/G4 tumor samples.<br>Conclusion: These antibody sets are currently validated on reference renal cancer TMAs in clinical pathology. The final objective is to model protein networks in order to direct therapeutic decision making in clinical uro-pathology.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X26'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X26</strong></td>
 </tr> <tr> 
  <td><b>Gene Regulatory Network Reconstruction using Conditional Mutual Information</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gabriele Sales <i>University of Padua</i></td>
 </tr>
 <tr> 
  <td>  Chiara Romualdi (University of Padua, Department of Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Inferring gene regulatory networks (GRNs) is currently one of the most<br>challenging task in systems biology. Several approaches have been proposed to<br>reconstruct GRNs using experimental data such as Bayesian Networks (BN),<br>Relevance Networks (RN) and Graphical Gaussian Models (GGM). While BN and GGM<br>should be able to distinguish between direct and indirect edges, RN does<br>not. However BN and GGM hardly work in the case of thousands of genes and with a<br>small number of replicates, while RN have the ability to deal with them.<br><br>Here we describe a novel reconstruction method we have developed that improves<br>the RN approach in a number of ways. Our analysis is based on the Conditional<br>Mutual Information (CMI). Mutual information is a measure of the mutual<br>dependence of two variables: it can be interpreted as a generalized measure of<br>correlation, analogous to the Pearson correlation, but sensitive to any<br>functional relationship, not just to linear dependencies. Its Conditional<br>variant (CMI) is able to identify the relations that are specific to a pair of<br>variables and at the same time independent from others. This property allow us<br>to detect the presence of direct regulatory interactions from experimental data<br>in a more direct way than possible with other RN methods such as ARACNE<br>(DPI), CLR and MRNET. We have, moreover, implemented a CMI estimator using<br>k-nearest neighbor distances (following the work by Kraskov and colleagues)<br>that attains much lower levels of distortion and higher precision than<br>alternatives based on kernel densities and variable discretization.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X27'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X27</strong></td>
 </tr> <tr> 
  <td><b>Mapping Genetic Interactions in Various Contexts Provides Complementary Information</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Magali Michaut <i>University of Toronto</i></td>
 </tr>
 <tr> 
  <td>  Gary D. Bader (University of Toronto, The Donnelly Centre); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Mapping Genetic Interactions in Various Contexts Provides Complementary Information<br><br>Goals<br>Genetic interactions are useful to study biological processes and their functional relationships. A genetic interaction is a deviation from the expectation for a double mutation in terms of a certain phenotypic readout in a given environment. In Saccharomyces cerevisiae, most genetic interaction studies assess a growth defect in standard laboratory conditions. Recently some studies were performed in DNA damage conditions or were based on a different phenotypic readout (e.g endocytosis instead of growth defect). How different are genetic interaction networks obtained in different conditions or using different phenotypic readouts and what can we learn from them?<br><br>Method<br>We compare here various genetic interaction maps, and in particular how they overlap. As a control we use networks obtained with the same phenotypic readout. Since the networks are subject to systematic and stochastic errors, we also assess the overlap expected by chance based on error rates and compare it with observed overlaps.<br><br>Results<br>We find that networks obtained with different phenotypic readouts provide much unique and complementary information. The quantitative interaction scores for these networks are not correlated (r=0.05) whereas they are for the control network comparison (r=0.38). The overlap between them is lower than in the control (0.09 vs 0.17) and lower than expected whereas it is higher than expected in the control (ratio 0.6 vs 1.5). Finally the networks show a lower agreement than in the control in terms of type of genetic interaction (positive/negative) in the overlap (0.64 vs 0.94).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X28'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X28</strong></td>
 </tr> <tr> 
  <td><b>Systems Biology of Seasonal Influenza Vaccination in Humans</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Helder Nakaya <i>Emory University</i></td>
 </tr>
 <tr> 
  <td>  Jens Wrammert (Emory University, Emory Vaccine Center); Eva Lee (Georgia Institute of Technology, Center for Operations Research in Medicine and HealthCare); Luigi Racioppi (Federico II University of Naples, Department of Molecular and Cellular Biology and Pathology); Stefanie Marie-Kunze (Emory University, Emory Vaccine Center); Sudhir Kasturi (Emory University, Emory Vaccine Center); Shuzhao Li (Emory University, Emory Vaccine Center); Rivka Elbein (Emory University, Emory Transplant Center); Aneesh Mehta (Emory University, Emory Transplant Center); Bali Pulendran (Emory University, Emory Vaccine Center); Rafi Ahmed (Emory University, Emory Vaccine Center); Anthony Means (Duke University, Department of Pharmacology and Cancer Biology); Nicholas  Haining (Dana-Farber Cancer Institute, Department of Pediatric ); Alan Aderem (Institute for Systems Biology, Institute for Systems Biology); Kanta Subbarao (National Institutes of Health, National Institute of Allergy and Infectious Diseases); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Systems vaccinology has emerged as an interdisciplinary field that combines systems wide measurements and network and predictive modeling applied to vaccinology. Here it was used to study the molecular mechanisms underlying the innate responses to the trivalent inactivated influenza (TIV) and live attenuated influenza (LAIV) vaccination in humans, and to identify early gene signatures that predict the magnitude of the antibody responses to influenza vaccination. Healthy adults were vaccinated with TIV or LAIV, and blood samples collected at days 0, 3, 7 and 28. Innate and adaptive responses were studied using systems biological approaches. Our analyses revealed that LAIV but not TIV induced the expression of several interferon related genes. A meta-analysis of publicly available microarray data of peripheral blood mononuclear cell types revealed that TIV induces a plasma B cell signature, consistent with the enhancement of plasma cell numbers after vaccination with TIV. Gene signatures of FACS sorted subsets of dendritic cells, monocytes, and B cells, demonstrated influenza vaccination induced cell type specific signatures. Furthermore, we identified gene signatures that were capable of predicting the magnitude of the antibody responses in two independent flu season trials. Of note, the expression of the gene CamkIV was inversely correlated with the antibody response. Strikingly, vaccination of CamkIV deficient mice with TIV induced enhanced antigen-specific antibody titers, demonstrating an unappreciated role for CamkIV in the regulation of antibody responses. Together, these results demonstrate the utility of systems biology not only in predicting vaccine immunogenicity, but also in offering novel insights into the molecular mechanism of influenza vaccines.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X29'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X29</strong></td>
 </tr> <tr> 
  <td><b>Computational analysis of stress dependency and differential BCL2-protein activity to Bax and Bak activation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andreas Lindner <i>Royal College of Surgeons in Ireland</i></td>
 </tr>
 <tr> 
  <td>  Jochen Prehn (Royal College of Surgeons in Ireland, Department of Physiology and Medical Physics); Heinrich Huber (Royal College of Surgeons in Ireland , Department of Physiology and Medical Physics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Apoptosis is a form of genetically programmed cell death and its deregulations have been implicated in cancer, development disorders and neurodegenerative diseases. To kill cancer cells, chemotherapeutic drugs often induce genotoxic stress conditions such as DNA strand breaks or cell cycle arrest. Genotoxic stress induced pro-apoptotic proteins of the BCL-2 family eventually causing mitochondrial outer membrane permeabilisation (MOMP), believed to be the key event in most apoptotic pathways. To prevent MOMP, pro-survival proteins of the same BCL-2 family neutralise their pro-apoptotic family members. However, the interaction of BCL2 family proteins is complex with several pro-survival and pro-apoptotic proteins currently known and with different protein expressions in different cell lines. Based on extensive screening of biochemical literature, we constructed a computational model to investigate the differential role of the BCL2-proteins and to understand the apoptotic machinery upstream to MOMP. The model allows comparison of two competing scenarios of MOMP activation known as the direct and indirect activation model. Model calculations for the indirect activation model predicted apoptosis even in the absence of stress stimuli and suggested the direct model as more robust against low genotoxic stress. The direct activation model was then applied to study the administration of chemotherapeutic drugs that mimic the effect of pro-apoptotic BCL2 family proteins and that are currently under clinical trial. Our model aims to stimulate inter-working between in-silico, wet-lab and clinical research scientist and eventually targets to be a clinical tool for predicting apoptotic drug response to different cells, tissues and stimuli.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X30'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X30</strong></td>
 </tr> <tr> 
  <td><b>Construction of a drug-target-disease network and its application to drug repositioning and mode of action analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Charny Park <i>Ewha Womans University</i></td>
 </tr>
 <tr> 
  <td>  Hyelin Park (Ewha Womans University, Division of life and pharmaceutical sciences); Sanghyuk Lee (Ewha Womans University, Division of life and pharmaceutical sciences); Wankyu  Kim (Ewha Womans University, Division of life and pharmaceutical sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Many drugs show polypharmacology i.e. a drug having multiple targets. With increasing cost of developing novel drugs, drug polypharmacology has been actively sought aiming at drug repositioning. For efficient discovery of novel indications for known drugs, it is critical to understand drug mode-of-action and its implication in various human diseases. Gene expression profiles are widely used as molecular signature, suggesting a specific cellular state or perturbation such as disease or chemical treatment. Therefore, similar or inverse expression patterns may suggest common mode-of-action involving the same molecular pathway or regulatory perturbation.<br>Here, we integrated chemically-perturbed and disease-associated gene expression profiles as well as existing knowledge on drug-target and gene-disease relationship. Our data set includes ~12,000 disease, and ~7000 disease gene expression profiles for ~300 drugs and ~100 human diseases respectively. All the data on drug-target and gene-disease association are integrated using standard terms and identifiers, which are manually checked and curated where necessary.<br>We performed all-against-all comparison among the data sets including drug targets, drug-perturbed genes and disease signature genes, resulting in a comprehensive network among drugs, targets and human diseases. This network reveals hidden associations among drugs and human diseases as well as previously known cases, which may share similar mode-of-action or involve common molecular pathways.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X31'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X31</strong></td>
 </tr> <tr> 
  <td><b>Finding sets of shortest elementary flux modes that decompose genome-scale metabolic flux distributions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Siu Hung Joshua Chan <i>The Hong Kong Polytechnic University</i></td>
 </tr>
 <tr> 
  <td>  Ping Ji (The Hong Kong Polytechnic University, Industrial and Systems Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Metabolic network consisting of a set of biochemical reactions and compounds is an important biological system which has successfully described cellular metabolism. One particular use of metabolic networks is to analyze metabolic pathways.<br>Elementary flux mode (EFM) is an important concept in metabolic pathway analysis. It is a minimal set of reactions maintaining metabolism at steady state. EFMs have been applied to study network structures, predict new pathways, reconstruct networks, design rational strain, etc. One important interpretation of EFMs is that every metabolic flux distribution is composed of EFMs. Finding such decompositions is desirable. However, traditional methods require full sets of EFMs which are intractable for huge networks due to combinatorial explosion.<br>Based on an optimization model to find the shortest EFMs, we derived an algorithm to decompose flux distributions into EFMs without first finding the full set of EFMs. The subset of EFMs found by the algorithm has a nice property that each EFM has a unique weight contributing to the flux distribution. The role and cooperation between pathways and enzymes are thus uniquely defined. This leads to a more exact biological interpretation.<br>Two cases are examined by the proposed algorithm: the growth in Escherichia coli and the mouse cardiac muscle. Metabolic pathways consistent with literature are located. This gives a closer look on cooperation between metabolic pathways in realistic metabolism, e.g. the CAC cycle and the fatty acid oxidation. This is also the first step towards investigating flux distribution by EFMs in present-day genome-scale networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X32'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X32</strong></td>
 </tr> <tr> 
  <td><b>MONGKIE: Modular Network Generation  and Visualization  Platform with Knowledge Integration Environment</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yeongjun Jang <i>Korea Research Institute of Bioscience and Biotechnology</i><br></td>
 </tr>
 <tr> 
  <td>Sanghyuk Lee (Korea Research Institute of Bioscience and Biotechnology) Jihae Seo (Korea Research Institute of Bioscience and Biotechnology, Korean BioInformation Center); Namhee Yu (Korea Research Institute of Bioscience and Biotechnology, Korean BioInformation Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: MONGKIE is an integrated network visualization platform which allows us to explore and analyze interconnected biological data in an interactive manner with knowledge integration environment. It is optimized for exploring protein-protein interaction data and pathway data from KoPath(http://kopath.kobic.re.kr), Integrated Human Pathway Database. But it can be easily applied to any biological data which can be modeled as network structures, such as gene-drug-disease interaction or association network. To represent diverse types of biological entities, interactions and controls, MONGKIE supports various types of nodes, edges and arrows such as Complex node, Super-node, hierarchically decomposed compound node representing adjacency and inclusion relationships, (Un)Directed edge, Multi-edge, Hyper-edge and Self interaction edge. Also users can customize visual properties of them through the integrated visual editor UI. MONGKIE offers automatic graph layouts and graph manipulation techniques to increase the usability. Users can explore networks through dynamic expansion, filtering, grouping, search functionalities. MONGKIE provides knowledge integration and analysis modules such as Import/Export Gene List and Network, Gene ID Conversion, Expression Overlay and Analysis, Network Clustering and Ranking, Gene Set Enrichment Analysis, Pathway Integration and Visualization. In addition, to generate insight on the biological meaning, commonly used network analysis processes are provided in the form of pipeline consisting of these integration and analysis modules. The network can be exported as XML format(GraphML/GML/XGMML), Comma-Separated Values(CSV), Encapsulated PostScript(EPS), high-resolution Scalable Vector Graphics(SVG) and various bitmap images. MONGKIE is a java-based application built on top of NetBeans(http://netbeans.org) platform that supports modular(plug-in) architecture, thus being platform-independent and easily extendable with additional functionalities.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X33'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X33</strong></td>
 </tr> <tr> 
  <td><b>Hierarchical high-dimensional regression-based sensitivity analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kristin Tøndel <i>Norwegian University of Life Sciences</i></td>
 </tr>
 <tr> 
  <td>  Harald Martens (Norwegian University of Life Sciences, Centre for Integrative Genetics (CIGENE)); Jon Olav Vik (Norwegian University of Life Sciences, Centre for Integrative Genetics (CIGENE)); Ulf G. Indahl (Norwegian University of Life Sciences, Centre for Integrative Genetics (CIGENE)); Stig W. Omholt (Norwegian University of Life Sciences, Centre for Integrative Genetics (CIGENE)); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Dynamic models describing complex biological systems often contain a large number of parameters and output variables, and the causal relations between parameter variation and model output  can be very complex. Hence, comprehensive sensitivity analyses of such models across the whole biologically relevant parameter space are not trivial. In particular, more efficient and generic ways to obtain a solid understanding of how the sensitivity to each parameter is dependent on the parameter background are sorely needed. We report a generic methodology for global sensitivity analysis based on Hierarchical Cluster-based Partial Least Squares Regression (HC-PLSR), that allows separate handling of different regions in the parameter space and therefore more accurate description of a model’s sensitivity to parameters conditional on the span of the other parameters than prevailing methods. More specifically, HC-PLSR automatically separates the observations into groups based on model behavior. Each group is then analyzed separately with PLSR, resulting in one global regression model comprising all observations, as well as regional regression models within each group. Both the global and regional regression models provide regression coefficients that are used as measures of model sensitivity to the different parameters and parameter settings. We illustrate the HC-PLSR approach on a complex, nonlinear dynamic model of the mouse ventricular myocyte.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X34'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X34</strong></td>
 </tr> <tr> 
  <td><b>Different regulatory networks of SOX2 in embryonic and neural stem cells</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jihae Seo <i>Korea Research Institute of Bioscience and Biotechnology,KRIBB</i><br></td>
 </tr>
 <tr> 
  <td>Sanghyuk Lee (Korea Research Institute of Bioscience and Biotechnology,KRIBB) Yeongjun Jang Jang (Korea Research Institute of Bioscience and Biotechnology,KRIBB, Korean Bioinformation Center,KOBIC); Jihyun Lee (Seoul National University, Information Center for Bio-pharmacological Network College of Pharmacy); Jinah Park (Korea Research Institute of Bioscience and Biotechnology,KRIBB, Korean Bioinformation Center,KOBIC); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: It is well known that the transcriptional factor SOX2 is an essential TF for early development as well as for the propagation of undifferentiated embryonic stem cells(ESC). In addition, SOX2 has an essential role in development of neural stem cell(NSC). In an effort to elucidate the difference in regulatory mechanisms of SOX2 in ESC and NSC, we performed the ChIP-chip experiment to identify SOX2 target genes in human NSC. The result was compared to the equivalent data in human ESC publicly available. Target genes were significantly different between ESC and NSC. Gene set analysis showed that target genes were enriched in different categories of GO and pathways. We hypothesized that there must be cell type specific cofactors for Sox2, and it’s verified using random permutation test. Several transcription factors which are well-known essential factors in ESC, NSC are included in those cell type specific cofactors. We also constructed cell type specific Sox2 target gene networks and expanded it using Protein-Protein interactions with cofactors. And then, network clustering analysis was performed. Cell type specific clusters were observed. In ESC network cluster, TGF beta signaling highly related with SMAD3, SMAD4 and SP1 was significant. Whereas, in NSC network cluster, several genes(CREB, TP53, HDAC1, YY1, SMAD2, and STAT3) popped out and it looks like EP300 plays role as a network hub. Finally, we have analyzed the gene expression profiles of cell type specific network clusters. The result would provide useful information to understand the role of SOX2 in differentiation of ESC to NSC.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X35'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X35</strong></td>
 </tr> <tr> 
  <td><b>Interspecies translation of disease networks increases robustness and predictive accuracy</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yahya Anvar <i>Leiden University Medical Center</i></td>
 </tr>
 <tr> 
  <td>  Allan Tucker (Brunel University, Department of Information Systems and Computing); Andrea Venema (Leiden University Medical Center, Center for Human and Clinical Genetics); Gert-Jan van Ommen (Leiden University Medical Center, Center for Human and Clinical Genetics); Silvere van der Maarel (Leiden University Medical Center, Center for Human and Clinical Genetics); Vered Raz (Leiden University Medical Center, Center for Human and Clinical Genetics); Peter 't Hoen (Leiden University Medical Center, Center for Human and Clinical Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The derivation of gene regulatory networks from high-throughput expression data via machine learning strategies is problematic. Their reliability is often compromised by limited and highly variable samples, heterogeneity in transcript isoforms, noise, and other artifacts. Hence, a probabilistic approach is needed that can represent complex stochastic relationships, integrate different types of data, weight the reliability of the data, and accommodate noise and missing values. We developed a novel algorithm, dubbed Dandelion, in which we construct and train intraspecies Bayesian networks which are translated and examined on independent test sets from other species in a reiterative procedure. The interspecies disease networks are subjected to multi-layers of analysis and evaluation criteria which lead to the identification of the most consistent relationships within the network structure. We demonstrate the performance of our algorithms on datasets from animal models of oculopharyngeal muscular dystrophy (OPMD) and patient materials. We show that the interspecies proteasome networks provide highly accurate predictions on gene expression levels and disease outcome whilst the stability of these networks increases after interspecies translation. Unlike existing modeling approaches, our algorithms do not require notoriously difficult one-to-one mappings of protein orthologues or alternative transcripts and can deal with missing data. We show that the OPMD-association of potential key regulators can be reproduced and validated on an unseen and independent model system. This study presents a state-of-the-art strategy in constructing interspecies disease networks that provide crucial information on functional regulatory relationships among genes, leading to better understanding of the molecular mechanisms underlying the disease.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X36'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X36</strong></td>
 </tr> <tr> 
  <td><b>Discovery and characterization of genotype-selective drugs in cancers</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ningning He <i>Sookmyung Women's University</i><br></td>
 </tr>
 <tr> 
  <td>Sukjoon Yoon (Sookmyung Women's University) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission Most cell lines recapitulated known tumor-associated genotypes and genetically defined cancer subsets, irrespective of tissue types. Drug treatment on many different cell lines provides an important preclinical model for early clinical applications of novel targeted inhibitors. The NCI60 is a program developed by the NCI/NIH aiming the discovery of new chemotherapeutical agents to treat cancer. Here we present a novel statistical method, CLEA (Cell Line Enrichment Analysis) to quantitatively correlate the genotype with gene expression signatures and drug sensitivity in cancer cell lines. The results provided us new insights on genotype-dependent gene expression signatures, cancer pathways and chemical sensitivity. It will have applications in predicting and optimizing therapeutic windows of anti-cancer agents.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X37'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X37</strong></td>
 </tr> <tr> 
  <td><b>Comprehensive map of cell survival signalling</b><br/><br/></td>
 </tr>
 <tr> 
 <td>David Cohen <i>Institut Curie, ISERM, Mines ParisTech</i></td>
 </tr>
 <tr> 
  <td>  Inna Kupperstein (Institut Curie, ISERM, Mines ParisTech, Bioinformatics and Computational Systems Biology of Cancer); Andrei Zinovyev (Institut Curie, ISERM, Mines ParisTech, Bioinformatics and Computational Systems Biology of Cancer); Gordon Tucker (Institut de Recherches Servier (IdRS), Cancer Research & Drug Discovery); Thierry Dubois (Institut Curie, ISERM, Mines ParisTech, Translational Research ); Emmanuel Barillot (Institut Curie, ISERM, Mines ParisTech, Bioinformatics and Computational Systems Biology of Cancer); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Malignant cells are characterised by a high survival capacity. There is a number of cellular mechanisms that can contribute to cell survival like apoptosis suppression, modulation of cell cycle checkpoints, accelerated proliferation mechanisms. Pathways involved in these processes are deregulated in the majority of cancers, and therefore, some of their components represent attractive drug targets to stop malignant propagation. To understand how these molecular mechanisms are orchestrated, we used a systems biology approach for the representation of biological processes as comprehensive models.<br>We constructed a network of cell survival signalling pathways. This network was built using the CellDesigner software for visualisation of cell processes as a comprehensive map. The map has a modular structure and is composed of a collection of interconnected sub-maps that include PI3K-AKT-mTOR, Sonic Hedgehog, Wnt and Ras-Raf-Mek signalling pathways. The map shows proteins involved in the pathways, protein-protein and protein-DNA interaction, complex formation, sub-cellular localisations and transport of signalling components. To further understand the cross-talks between the cellular processes involved in a malignant propagation, we developed a method for merging/decomposition of signalling networks. The map will be merged with existing maps such as cell cycle-DNA repair or an apoptosis maps, a process which allow further investigations on the major players involved in cancer.<br>The map is applicable for data integration and mathematical modelling for the discovery of key players in cancers and for the search of synthetic lethal partners.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X38'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X38</strong></td>
 </tr> <tr> 
  <td><b>An Integrated Approach To Identify Human MicroRNA-Targets.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Haeseung Lee <i>Ewha Research Center for Systems Biology</i><br></td>
 </tr>
 <tr> 
  <td>Kim Wan Kyu (Ewha Research Center for Systems Biology) Hyung-Seok Choi (Ewha Research Center for Systems Biology,  Future IT R&D Lab.,LG Electronics Inc., Bio & Health Group); Sanghyuk Lee (Ewha Womans University, Division of life and pharmaceutical sciences, Bio & Health Group); Wankyu Kim (Ewha Womans University, Division of life and pharmaceutical sciences, Bio & Health Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: MicroRNAs are post-transcriptional regulators that bind to complementary sequences on target mRNA, usually causing translational repression and gene silencing. Identifying microRNA targets and microRNA-mRNA regulatory modules are a pivotal step for understanding microRNA function. There are a number of miRNA target predictions methods available. However, the functional analysis of microRNAs is still challenging because target predictions are not sufficiently reliable and validated targets are sparse. Here we propose an integrative computational approach to infer reliable miRNA targets using a number of expression data and various miRNA target prediction methods. These data include (i) mRNA expression profiles under miRNA-perturbed conditions (e.g. knockout or over-expression), (ii) miRNA and mRNA expression profiles simultaneously measured for the same sample, (iii) miRNA target predictions, and (iv) experimentally validated miRNA targets. All the data sets are integrated using a standard scoring scheme based on Bayesian probability. Our integrated approach shows significantly improved accuracy compared to individual target prediction method and any single data set.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X39'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X39</strong></td>
 </tr> <tr> 
  <td><b>The human nucleic acid interactome</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gerhard Dürnberger <i>CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences</i></td>
 </tr>
 <tr> 
  <td>  Tilmann Bürckstümmer (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Tobias Doerks (EMBL Heidelberg) Kilian Huber (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Evren Karayel (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Thomas Burkard (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Melanie Planyavsky (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Andre Müller (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Keiryn Bennett (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Peer Bork (EMBL Heidelberg) Giulio Superti-Furga (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) Jacques Colinge (CeMM - Reserach Center for Molecular Medicine of the Austrian Academy of Sciences) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Interactions between proteins and nucleic acids are fundamental to many biological processes ranging from gene regulation, replication to viral sensing. Current proteomic pulldown approaches allow to get an unbiased nucleic acid centric view on protein-nucleic acid interactions. Here we present a first systematic effort towards this goal.<br><br>To achieve a wide coverage across the nucleic acid binding protein space a diverse set of 28 nucleic acid baits was designed and used to affinity purify proteins from three human cell lines. These cell lines were chosen from the different germ layers to maximize diversity in the results.<br><br>The so-obtained data set is comprised of more than 6000 interactions between the nucleic acid baits and 963 unique proteins. Bioinformatics analysis of the data set yielded the identification of new potential nucleic acid binding domains and proteins with affinity to nucleic acid sub-categories.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X40'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X40</strong></td>
 </tr> <tr> 
  <td><b>Statistical assessment of gene group crosstalk enrichment in networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Oliver Frings <i>Stockholm University</i></td>
 </tr>
 <tr> 
  <td>  Theodore McCormack (Stockholm University, Stockholm Bioinformatics Center); Andrey Alexeyenko (Science for Life Laboratory, Stockholm); Erik Sonnhammer (Stockholm University, Stockholm Bioinformatics Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Analyzing sets of functionally coupled genes or proteins in the context of global interaction networks has become an important bioinformatical analysis. Typically, one wants to analyze the crosstalk, i.e. the total connectivity between or within functional groups. This is however only meaningful if the statistical significance of the measured crosstalk enrichment is assessed.<br><br>We developed CrossTalkZ, a statistical method and software to assess the significance of crosstalk enrichment between pairs of gene or protein groups in large biological networks. The statistic we propose applies the standard z-score, calculated by comparing the observed number of network links to the distribution of the expected number of links estimated by network randomizations. CrossTalkZ implements four different network randomization strategies including a novel algorithm that preserves higher-order topology. <br><br>We demonstrate that 1) all the four algorithms randomize biological networks well and 2) the standard z-score is generally an appropriate and unbiased statistic. We further evaluated the ability of the different methods to recover crosstalk within known biological pathways and estimated the confidence of the findings. We conclude that the methods preserving the topology properties of the original network performed the best. We do however offer the users to choose among all four methods since different types of analysis might require different types of randomization.<br><br>The software implementation (available at http://sonnhammer.sbc.su.se/download/CrossTalkZ/) is fast, accepts simple input, and produces a number of basic and supplementary statistics, including p-value, false discovery rate, and test of normality for the random distribution.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X41'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X41</strong></td>
 </tr> <tr> 
  <td><b>System-level analysis of cell line data for targeted drug discovery</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nayoung Kim <i>Sookmyung Women's University</i><br></td>
 </tr>
 <tr> 
  <td>Sukjoon Yoon (Sookmyung Women's University) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission The lineage of cell lines was usually considered for a discovery of cancer drugs, but recently targeted drugs are highly regarded in cancer therapeutics. Each cell line has particular and multiple tumor-associated genotypes. Some mutations of genes known as oncogene or tumor suppressor can be a good point for developing targeted anticancer drug. Especially TP53 is an important gene in cancer and it regulates the cell cycle arrest, apoptosis, senescence, DNA repair or changes in metabolism. Here we represent that analysis of cell line data in system-level for discovery of targeted cancer drug. Firstly, we selected genotypes to play a major role in cancer development. And it was again categorized along mutated state of TP53. Therefore each cancer cell line is largely separated to two subgroups -co-mutation with TP53 and single mutation without TP53. We could get the meaningful patterning of drug response and gene expression in specific genotypes. Through network analysis, important mechanisms related to genotypes were indicated. System-level analysis based on genotype enabled us to discover targeted drug and suggest its mechanism.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X42'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X42</strong></td>
 </tr> <tr> 
  <td><b>Cholesterol-Responsive Gene Network Reconstruction from Hypercholesterolemic Mice Transcriptome Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gustavo Hime <i>Freiburg Institute For Advanced Studies</i></td>
 </tr>
 <tr> 
  <td>  Josefin Skogsberg (Karolinska Institutet, Computational Medicine Group); Tom Michoel (Freiburg Institute for Advanced Studies, School of Life Sciences); Johan Björnkegren (Karolinska Institutet, Computational Medicine Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Atherosclerosis is a condition that results from fat deposition on the arterial  walls and subsequent inflammation. The lesions can be asymptomatic for years and  develop into life threatening clinical manifestations very quickly. Its incidence  in the population is unknown, since it is usually only diagnosed at an already  advanced stage, through invasive procedures.  Prophylactically, it is now common  practice and recommendation to control the levels of lipoproteins (cholesterol)  in the blood. Associated treatment is through cholesterol-lowering drugs. Although the relationship between atherosclerotic lesions and high  levels of lipoproteins has for long been well known and scrutinized, it was not  until recently that the organism's response at trascriptome level was put under  investigation. We analyzed  transcriptomes of the tissue from atherosclerotic aortic arches of transgenic hypercholesterolemic mice, obtained through microarrays. The experimental data consist of time profiles of gene expression along the  lifetime of the mice as they develop atheromatous plaques, and also of mice which,  at different ages, were subjected to genetic cholesterol reducing knock-out.   The experimental data indicate the disease progresses through two distinct phases.  The first allows for full regression to health, while the second implies permanent  damage to the artery. We developed  a semi-automated analysis pipeline with which we perform gene clustering according to patterns mined from the data itself, ontology matching to previously published regulatory pathways, and finally Bayesian network inference to identify key genes and dysregulated networks that correlate separately to these  two phases of the disease development, both for diagnostic and therapeutic purposes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X43'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X43</strong></td>
 </tr> <tr> 
  <td><b>Evolution of domain-linear motif interactions shapes the modular architecture of protein-protein interaction networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Inhae Kim <i>Pohang university of science and technology</i></td>
 </tr>
 <tr> 
  <td>  Seong Kyu Han (Pohang university of science and technology, Divsion of Molecular and Life Sciences); Jinho Kim (Pohang university of science and technology, Divsion of Molecular and Life Sciences); Jihye Hwang (Pohang university of science and technology, Divsion of Molecular and Life Sciences); Sanguk Kim (Pohang University of Science and Technology, Divsion of IT Convergence Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein-protein interactions (PPIs) are essential to achieve proper functions of living cells. PPIs are usually built by two components, domains and linear motifs, which make two types of contacts in protein interfaces. Rather strong and stable interactions are mediated by domain-domain interactions (DDIs), whereas weak and transient interactions are mediated by domain-linear motif interactions (DLIs). The prevalence of DLIs in PPI network has been recently emphasized, however, DLI’s role in the network structure and evolution is still elusive. Here, we analyzed all possible linear-motif-mediated interactions of human protein interactome and mapped 4,120 DLIs and 5,612 DDIs in the network. We discovered that DLIs bridge protein interactions between modules, whereas DDIs cluster proteins into individual modules. The modular architectures of PPI network correspond to protein complexes, subcellular compartments, functional groups, or similar phenotypes. Moreover, linear motifs emerged to create novel PPIs to facilitate interactions between evolutionarily unrelated proteins in the network. Particularly, DLIs arose concurrently with the appearance of metazoan species, which showed a remarkable impact on the growth of network complexity. Linear motifs have contributed to the evolution of the modular architecture of PPI network and provided efficient way to achieve biological innovations by reuse of network modules in novel combinations.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X44'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X44</strong></td>
 </tr> <tr> 
  <td><b>Consensus Modules - Confidence Values for Integrated Biologial Network Analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daniela Beisser <i>University of Wuerzburg</i><br></td>
 </tr>
 <tr> 
  <td>Tobias Mueller (University of Wuerzburg) Thomas Dandekar (University of Wuerzburg, Bioinformatics); Gunnar Klau (Science Park Amsterdam, Centrum Wiskunde & Informatica); Marcus Dittrich (University of Wuerzburg, Bioinformatics); Tobias Mueller (University of Wuerzburg, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: High-throughput techniques like microarrays, RNA-Seq or mass spectrometry provide a wealth of information for integrated systems-level analyses. The large-scale integration and analysis of various data sources is nowadays often accomplished by network approaches. Several approaches exist that allow to identify functional modules, pathways or gene signatures in biological networks, e.g. modules, containing significantly differentially expressed genes in the context of a protein-protein interaction network. Recently, an exact approach was introduced by Dittrich and co-workers that resolves the subwork-finding problem to optimality using integer linear programming.<br><br>The objective of the presented study is to assess the accuracy and variability of the identified functional modules. Therefore, we propose a novel concept of a consensus module based on jackknife resampling. The consensus module summarizes possible subnetworks and displays support values for its nodes and edges. <br><br>Since our objective is not only to find a module which obtains a good accuracy but also yields results that are robust to minor changes in the integrated data, we assess the robustness and variability of the obtained solutions in an extensive simulation study.<br><br>In addition, the resampling procedure is applied to two biological microarray data sets on diffuse large B-cell lymphomas and acute lymphoblastic leukemia.  A future anaylsis will focus on the identification of functional modules in an integrated metabolite network on Salmonella typhimurium during infection.<br><br>The algorithms will be implemented in the R package BioNet available from http://bionet.bioapps.biozentrum.uni-wuerzburg.de and the Bioconductor project.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X45'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X45</strong></td>
 </tr> <tr> 
  <td><b>A NEW NETWORK ATTACK STRATEGY MIMICKING DRUG MECHANISM: INTERFACE ATTACK</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Billur Engin <i>Koc University</i></td>
 </tr>
 <tr> 
  <td>  Ozlem Keskin (Koc University, Center for Computational Biology and Bioinformatics); Attila Gursoy (Koc University, Center for Computational Biology and Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Molecular networks are robust to random hits and vulnerable under targeted attacks because of their scale-free architecture. Although hub nodes are critical elements of these networks, removing a small number of edges may damage the system more than a hub attack. The most powerful attack strategy is hitting the set of most harming edges (distributed edge attack) via a greedy algorithm. <br>Breaking an edge between two nodes can be seen as the graphical representation of a drug blocking two proteins’ interaction. So, during an attack, rather than determining the target edge set randomly or according to their topological importance, choosing the party which is targeted by a specific drug, is a more physical and realistic approach. <br>Here we propose a network attack strategy, interface attack, where protein pairs that interact via structurally similar interfaces are targeted. If surface regions of any given protein pairs are alike considering their geometry and evolutionary conservation of hot spots, they are assumed to interact via similar interface regions. <br>Interface attacks are performed on p53 network that is derived from literature. Using different kinds of target selection strategies, such as attacking most frequent interface, targeting interface connecting hub nodes or hitting interface with highest betweenness centrality, network’s robustness is analyzed. Results imply that; since interface attack is a more physical kind of distributed attack, it is better than attacking nodes but not as efficient as distributed attack. Besides, likewise drug mechanism, interface attack doesn’t collapse the system while it disturbs the communication between target nodes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X46'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X46</strong></td>
 </tr> <tr> 
  <td><b>Enriched computational models as a knowledge base for Systems Biology</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nicolas Le Novere <i>EMBL-EBI</i></td>
 </tr>
 <tr> 
  <td>   BioModels.net  project (  ) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The use of computational modelling in the description and analysis of biological systems is at the heart of Systems Biology. There is a growing need to complement model structures, simulation descriptions and numerical results, encoded in standard description formats, with additional semantic information in a standards-compliant, computer readable form. Semantically enriched models facilitate model classification, analysis and processing, efficient search strategies and accurate conversion between formats, including graphical ones.<br><br>We present several community efforts which significantly contribute to the enrichment of the entire model life cycle. They cover the rules that descriptions must comply with (minimum standards), classify domain knowledge (ontologies), prescribe an identification system for model components (URIs), and finally provide a sharing strategy for enriched models (BioModels Database).<br><br>The Minimal Information Required In the Annotation of Models (MIRIAM) and the Minimal Information About a Simulation Experiment (MIASE) define what is needed to exchange models, and reuse them in simulations. The MIRIAM standard describes how model constituents should reference external resources in a stable and persistent manner (MIRIAM URIs), while MIRIAM Resources provides numerous services in support of this URI system. The Systems Biology Ontology (SBO), the Kinetic Simulation Algorithm Ontology (KiSAO) and the TErminology for the Description of DYnamics (TEDDY) introduce semantic information needed for the model, simulation and result description, respectively. BioModels Database is a repository which delivers enriched peer-reviewed, published computational models to the community.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X47'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X47</strong></td>
 </tr> <tr> 
  <td><b>Integrated Inference and Analysis of Regulatory Networks Using Multi-level Measurements</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Christopher Poultney <i>New York University</i></td>
 </tr>
 <tr> 
  <td>  Alex Greenfield (New York University, Computational Biology Program, Sackler School of Medicine); Colin Flinders (University of California, Los Angeles, Department of Biological Chemistry); Shannon Mumenthaler (University of Southern California, Center for Applied Molecular Medecine); Parag Mallick (Stanford University / University of Southern California, Canary Center / Center for Applied Molecular Medecine); Richard Bonneau (New York University, Center for Genomics and Systems Biology / Courant Institute for Mathematical Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Analyses that integrate proteomics, expression and signaling data are essential for making sense of how cells make decisions (translating genotype to phenotype, or predicting how cells respond to their microenvironment). Using data from several consortia that aim to assemble multi-level (protein, signaling, expression, environment) sets of measurements suitable for automatically learning large scale network models that span these informational levels, we describe our early developments towards a multi-level genomics analysis and visualization pipeline. Example applications include learning networks controlling: metastasis in breast cancer cell lines, lymphoma response to drugs, and white blood cell differentiation. We use an differential-equation-based inference pipeline and a Gaggle, Cytoscape and Sungear based suite of analysis and visualization tools to infer regulatory networks that utilize microarray, proteomics, and mass cytometry data. Our code, the Inferelator, uses microarray and proteomics data to learn systems of ordinary differential equations describing the rate of change in each gene’s rate of expression and translation as a function of transcription factors and other relevant predictors.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X48'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X48</strong></td>
 </tr> <tr> 
  <td><b>Using Ondex to constuct a predicted protein-protein interaction network for Fusarium graminearum</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Catherine Canevet <i>Rothamsted Research</i></td>
 </tr>
 <tr> 
  <td>  Michael Defoin Platel (Rothamsted Research, Bioinformatics); Artem Lysenko (Rothamsted Research, Bioinformatics); Jan Taubert (Rothamsted Research, Bioinformatics); Martin Urban (Rothamsted Research, Plant Pathology); Kim Hammond-Kosack (Rothamsted Research, Plant Pathology); Chris Rawlings (Rothamsted Research, Bioinformatics); Mansoor Saqi (Rothamsted Research, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A commonly used approach for protein-protein interaction network prediction involves the integration of information on known interacting proteins in other organisms with information on orthologous relationships between these organisms and the organism of interest. The available data on known interactors is contained in multiple sources and is based on many different types of biological experiments. The construction and analysis of protein-protein interaction networks involves several steps of data integration as well as manual inspection and filtering of the predicted networks based on data source and evidence.<br>Here we use Ondex a data integration and visualisation platform [1] (http://ondex.org/), to construct a predicted protein-protein interaction network for Fusarium graminearum, a major pathogen of wheat. The approach involves integrating data from five major protein interaction databases (HPRD, MINT, DIP, IntAct and BioGrid) for seven species together with orthology information from  InParanoid (http://inparanoid.sbc.su.se/). We import these datasets into Ondex (transforming them from their original data format to the OXL format [2]) and map them per species using data alignment techniques. We show how Ondex can be used to construct a predicted interaction network for Fusarium graminearum and how the visualisation tools facilitate exploration and analysis of the data.<br><br>[1] Lysenko A, et al. Data integration for plant genomics--exemplars from the integration of Arabidopsis thaliana database;. Brief Bioinform. 2009 Nov;10(6):676-93.<br>[2] Taubert J , et al. The OXL format for the exchange of integrated datasets; Journal of Integrative Bioinformatics, 4(3):62, 2007.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X49'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X49</strong></td>
 </tr> <tr> 
  <td><b>Topological features predict misannotations in metabolic network</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Rodrigo Liberal <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Misannotation in sequence databases has been a recognised problem for more than a decade. This issue is an important obstacle for the success of automated tools for gene function annotation, which rely extensively on comparison to sequences with known function. To improve current annotations and prevent future propagation of errors, tools are therefore needed to assist in the identification of misannotated gene products. The aim of this work is to predict misannotated proteins using only the topological information from a metabolic network, i.e. independently of the protein sequences.<br><br>In the case of enzymatic functions, each functional assignment implies the existence of a reaction within the organism’s metabolic network; a first approximation to a genome-scale metabolic model can be obtained directly from an automated genome annotation. Any obvious problems in the network, such as dead-end or disconnected reactions, can therefore be strong indications of misannotations. A decision tree trained on curated sets of correct and incorrect enzyme assignments was found to have an accuracy of up to 80% in cross-validation experiments. The resulting predictor was applied to an automated genome annotation of the human malaria parasite, Plasmodium falciparum, achieving an accuracy of 62% when validated against a recent genome-scale metabolic model.<br><br>In conclusion, we demonstrate that machine learning using only network topological features can predict the validity of enzyme annotations. We expect this method to be of use in the process of refining metabolic network models and in the detection of erroneous functional assignments within sequence databases.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X50'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X50</strong></td>
 </tr> <tr> 
  <td><b>A comprehensive map of cell death regulation and energy metabolism</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Simon Fourquet <i>Institut Curie - INSERM U900 - Mines ParisTech</i></td>
 </tr>
 <tr> 
  <td>  Laurence Calzone (Institut Curie - INSERM U900 - Mines ParisTech, Bioinformatics unit); Emmanuel Barillot (Institut Curie - INSERM U900 - Mines ParisTech, Bioinformatics unit); Andrei Zinovyev (Institut Curie - INSERM U900 - Mines ParisTech, Bioinformatics unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Cell death or survival pathways and energy metabolism are frequently deregulated in cancer cells. We present a comprehensive review of the molecular mechanisms underlying these processes and their cross-talks, as a map of biochemical reactions. This map is constructed by manual literature curation, fitting to standard SBGN format as implemented in CellDesigner. Around 1700 species and 1200 reactions are included and thoroughly annotated with literature references and external databases identifiers. Two important criteria for inclusion of results in the map are a sufficient level of description of the underlying biochemistry, and the existence of functional results highlighting the importance of these biochemical events in cellular responses. However, in the reductionist approach of describing biological processes as a mere collection of biochemical reactions, this notion of functional pathway vanishes. For this reason, the reaction network is semi-automatically mapped to a directed signed graph that represents local influences between species. Path description onto this graph is straightforward and provides reconstructed information about long-distance influences, thus re-introducing the notion of functional pathways spanning over several reactions. Identification of paths that had no experimental support reported in the publications used for the construction of the map, but for which experimental support was identified in the literature later on, provides a proof of principle that path searching on this graph allows to formulate reasonable hypotheses. We also apply a simple method based on path analysis to identify candidate deregulated species or reactions from expression data. For instance, BIM pro-apoptotic protein activity is proposed to be significantly inhibited in basal-like breast cancer.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X51'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X51</strong></td>
 </tr> <tr> 
  <td><b>Candidate gene prediction using integrative and network-based approaches - Mining Pig QTL</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Keywan Hassani-Pak <i>Rothamsted Research</i></td>
 </tr>
 <tr> 
  <td>  Tanja Kunej (University of Ljubljana, Department of Animal Science); Peter Dovc (University of Ljubljana, Department of Animal Science); Chris Rawlings (Rothamsted Research, Centre for Mathematical and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Correct identification of functional candidate genes for an important agronomic trait can be very valuable for informing the development of improved varieties by breeding with marker assisted selection methods.  Forward (classical) genetic approaches to gene discovery are designed to identify regions (loci) of the genome that are linked with a particular trait (phenotype). Presented with a list of positional candidate genes for a particular trait, the evidence users would need in order to evaluate their potential functional candidacy and whether any might have a role in that trait would include functional annotations, role in biochemical pathways, evidence of expression of tissue of interest, comparative information from related organisms, links to related trait information, the scientific literature and other resources that might be specific to the domain of interest. The assembly and analysis of such diverse information for a biologist is a challenging task, which makes automated and systematic methods essential.<br>We demonstrated the potential of such an integrative and network-based approach to study fat deposition in pigs by identifying obesity related genes in fatness QTL. The freely available Ondex system (www.ondex.org) was used to build a comprehensive knowledge network for pig which includes genome information from Ensembl, trait information from AnimalQTLdb, cross-species information from UNIPROTKB, annotations from the Gene Ontology, diseases from OMIM, pathways from KEGG, and phenotype information extracted from MEDLINE abstracts. Trait-based search facilities of Ondex combined with network inference and visualisation techniques facilitate the identification of candidate genes for complex phenotypes such as obesity.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X52'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X52</strong></td>
 </tr> <tr> 
  <td><b>Organizational structure of the peripheral gene regulatory network in B-cell lymphoma</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ricardo de Matos Simoes <i>Queen's University Belfast</i></td>
 </tr>
 <tr> 
  <td>  Shailesh Tripathi (Queen's University Belfast, Center for Cancer Research and Cell Biology); Frank Emmert-Streib (Queen's University Belfast, Center for Cancer Research and Cell Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 146.<br>The periphery of a cell is described by signaling pathways which are triggered by transmembrane proteins and receptors that are sentinels to control the whole gene regulatory network of a cell. To date knowledge of gene regulatory mechanisms that are governed from extracellular signals is limited. We infer the peripheral region of the regulatory network of B-cell lymphoma from a large-scale expression dataset (Basso 2005) by using the C3NET  algorithm (Altay 2010) and perform a functional and structural analysis of the largest  connected component of this network. Further, we analyze the hierarchical organization of  network components of the whole peripheral B-cell network by introducing a Bootstrap approach  which exploits the variability within the data as well as the inferential  characteristics of C3NET. We identify a variety of highly connected transmembrane proteins in the  B-cell lymphoma gene network such as ion channel complexes and signalling receptors. Our study allows to highlight the peripheral regulatory network of B cells that is centered at the physical periphery of the cell.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X53'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X53</strong></td>
 </tr> <tr> 
  <td><b>Identifying functional modules in protein-protein interaction networks based on semantic similarity using an exact approach</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Santosh Nilla <i>University Würzburg</i><br></td>
 </tr>
 <tr> 
  <td>Marcus Dittrich (University Würzburg) Desislava Boyanova (University of Wuerzburg, Bioinformatics); Daniela Beisser (University of Wuerzburg, Bioinformatics); Gunnar Klau (CWI, Life Sciences Group); Thomas Dandekar (University of Wuerzburg, Bioinformatics); Tobias Mueller (University of Wuerzburg, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The increasing amount of large scale data (transcriptomic, proteomic) provides the need for designing novel computational techniques for integrated network analysis. Many algorithmic approaches have been introduced, including an exact approach (heinz) based on integer linear programming. At the same time, the concept of semantic similarities between two genes using Gene Ontology (GO) annotations has become an important basis for many analytical approaches in bioinformatics. Assuming that a higher number of semantically similar gene functional annotations reflect biologically more relevant interactions, we devised an edge score for functional network analysis. Bringing these two approaches together, the edge score, based on the GO similarity, and the node score, based on the expression of the proteins in the analyzed cell type (e.g. data from proteomic studies), we search for the functional module as a maximum-scoring subnetwork in large PPI networks. We apply our method to various proteome datasets (different types of blood cells, embryonic stem cells) to identify protein modules that functionally characterize the respective cell type. This scalable method allows a smooth integration of data from various sources and retrieves biologically relevant signaling modules.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X54'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X54</strong></td>
 </tr> <tr> 
  <td><b>Topology of Functional Networks Predicts Physical Binding of Proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ömer Sarac <i>TU Dresden</i></td>
 </tr>
 <tr> 
  <td>  Andreas Beyer (TU Dresden, BIOTEC); Vera Pancaldi (University College London, Department of Genetics, Evolution & Environment and UCL Cancer Institute); Jürg Bähler (University College London, Department of Genetics, Evolution & Environment and UCL Cancer Institute); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: "This poster is based on Proceedings Submission 123"<br><br>It has been recognized that the topology of molecular networks provides information about the certainty and nature of individual interactions. Thus, network motives have been used for predicting missing links in biological networks or for removing false positives. However, a systematic analysis assessing a range of different network features with respect to their predictive power has not been published so far.<br>Here, we present a systematic assessment of seven different network features extracted from the topology of functional genetic networks and we quantify their ability to predict underlying physical protein interactions. Using machine learning we compare the predictive power of network features vs. ‘classic’ features and the combination of the two types of features. We demonstrate the utility of network features based on human and budding yeast networks; we show that network features can distinguish different sub-types of physical protein associations and we apply the framework to fission yeast, which has a much sparser known physical interactome than the other two species. <br>Our analysis shows that network features are at least as predictive for the tasks we tested as traditional machine learning features. However, we also found significant differences between features with respect to both, coverage of the network and predictive power. Also, feature importance varies between species owing to biological differences and different experimental methods used for obtaining the reference interactomes. The application to fission yeast shows that small maps of physical interactomes can be extended based on functional networks, which are often more readily available.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X55'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X55</strong></td>
 </tr> <tr> 
  <td><b>Integrated analysis of cellular network using phosphoproteome data to identify active signaling modules</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Desislava Boyanova <i>University Würzburg</i><br></td>
 </tr>
 <tr> 
  <td>Marcus Dittrich (University Würzburg) Santosh Nilla (University of Wuerzburg, Bioinformatics); Daniela Beisser (University of Wuerzburg, Bioinformatics); Gunnar Klau (CWI, Life Sciences Group); Thomas Dandekar (University of Wuerzburg, Bioinformatics); Tobias Mueller (University of Wuerzburg, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The increasing amount of proteomic and phosphoproteomic data has motivated the development of new approaches in integrative network analysis. Differences in protein abundance and phosphorylation levels represent changes in cellular signaling which are fully comprehensible only in a systems biological manner. Here, we present an algorithm to detect signaling modules in large-scale networks. In contrast to previous methods which mainly relied on gene expression data, our approach focuses on the analysis of cell-wide phosphorylation patterns. The integrated analysis combine protein-protein interaction (PPI) networks along with phosphoproteomic data to functionally describe signaling pathways and the change of information flow during various states of stimulation. To this end we use quantitative phosphoproteome data for node scoring in networks derived from PPI data as well as kinase-substrate relationships. Subsequently, we search for the maximum-scoring subnetwork using an exact algorithm (heinz) to identify differentially phosphorylated signaling modules in cellular networks.<br>Using this approach we analyze networks under various conditions (including time-series experiments), thereby characterizing system states in a network context. Furthermore, we performed this integrated analysis on various cell types (e.g. human platelets) to identify characteristic signaling modules. We believe that the integrative network analysis of phosphoproteome data can help to reconstruct cellular signaling pathways of biological importance and describe differences in the network state under specific conditions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X56'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X56</strong></td>
 </tr> <tr> 
  <td><b>A Novel Reverse Engineering Method for Gene/Protein Network Reconstruction - Divergence Weighted Independence Graphs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yang Xiang <i>Philip Morris International Research & Development</i></td>
 </tr>
 <tr> 
  <td>  Julia Hoeng (Philip Morris International R&D, Department of Biological System Research); Peter Sperisen (Philip Morris International R&D, Department of Biological System Research); Vincenzo Belcastro (Philip Morris International R&D, Department of Biological System Research); Joe Whittaker (Lancaster University, Department of Mathematics and Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Identification of the interactions between molecular entities within cells is the key to understanding the biological processes involved. Unfortunately, it is difficult to identify these interactions entirely by experiments. Although numerous methods have been developed for inferring gene/protein regulatory networks from expression data, reliable network inference from gene/protein expression data remains an unsolved problem. Recently, a novel method, divergence weighted independence graphs (DWIG), was developed. A simulated data set with 160 virtual animals was generated from the mathematical model of insulin signaling pathway to evaluate the performance of DWIG.  This simulated data characterized both the within-individual and between-individual variabilities, which other widely used public simulation data, such as the DREAM challenge, did not mimic fully. The performance of three reverse engineering methods, ARACNE (Algorithm for the Reconstruction of Accurate Cellular Networks), Banjo (Bayesian Network with Java Objects), and DWIG, were compared based on these simulated data. The area under curve (AUC) of receiver operating characteristic (ROC) curve showed that DWIG outperformed ARACNE and Banjo. ARACNE uses the marginal mutual information, while DWIG uses the conditional mutual information, which could be the reason for DWIG’s superior performance over ARACNE.  After prefiltering some weak links out by ARACNE, DWIG was applied to a protein dataset consisting of cytokines and chemokines that were measured in bronchoalveolar lavage fluid (BALF) of female A/J mice exposed to cigarette mainstream smoke for 3 and 5 months. An association network with 25 cytokines/chemokines was built.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X57'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X57</strong></td>
 </tr> <tr> 
  <td><b>Joint Dynamic Networks of miRNA and Transcripton Factor Regulation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marcel Schulz <i>Carnegie Mellon University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Understanding the regulation of  gene expression is one of the key issues to understand mechanisms that lead to diseases. A common approach is to model the interdependencies of regulatory relationships in form of a regulatory network (RN). In most cases RNs are learned as static snapshots but we are concerned with constructing the RN dynamic to recognize regulatory relationships that change over time. <br>The dynamic regulatory events miner (DREM)[1] was previously developed to include static transcription factor (TF) binding data (ChIP-Chip or ChIP-seq) and gene expression time series data to produce a regulatory map. In this work we show how to extend the DREM approach to model the down-regulating effect of miRNAs by including miRNA time series expression data. The new approach called DREMmir is able to predict joint regulatory events of TF-miRNA gene regulation.  We show the effect of inclusion of miRNA down-regulation by application to mouse time series data, sheding new light on the impact of miRNA regulation.<br>References :<br>[1] Jason Ernst, Oded Vainas, Cristopher T. Harbison, Itamar Simon, and Ziv Bar-Joseph. Reconstructing Dynamic Regulatory Maps. Molecular Systems Biology, 3:74, 2007</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X58'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X58</strong></td>
 </tr> <tr> 
  <td><b>KeyPathwayMiner - Detecting case-specific biological pathways by using expression data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hande Kucuk <i>Max Planck Institute Informatics</i></td>
 </tr>
 <tr> 
  <td>  Nicolas Alcaraz Millman (Max Planck Institute Informatics, Computational Systems Biology); Mayank Kumar (Max Planck Institute Informatics, Computational Systems Biology); Jan Baumbach (Max Planck Institute Informatics, Computational Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Computational systems biology methods today help life scientists with exploring the masses of available data. Different methods and tools have been developed in an attempt to serve the need of extraction, exploration, and deeper analysis of the data. For instance, statistical and machine learning methods have been successfully used in identifying pattern signatures. However, these methods fail to give deeper insights into gene expression data. In this work, we introduce Key-Pathway-Miner, a Cytoscape plug-in that allows extracting and visualizing sub-pathways that may be of interest given the results of a series of gene expression studies. We aim to detect “highly-connected” sub-networks where most genes show “similar” expression behavior. In particular, given network and gene expression study data, those maximal sub-networks are identified where all but n nodes of the network are expressed similarly on all but m cases of the user specified gene expression study data. As finding such modules is computationally intense, we developed and implemented heuristics algorithms based on Ant Colony Optimization. Here, we present the plug-in as well as the initial results and first evaluations for the Huntington ’s Disease (HD) data set.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X59'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X59</strong></td>
 </tr> <tr> 
  <td><b>Finding Consistent Explanations for Observations from Genome-Wide Mutant Assays</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Deborah Chasman <i>University of Wisconsin-Madison</i></td>
 </tr>
 <tr> 
  <td>  Brandi Gancarz (University of Wisconsin-Madison, Institute for Molecular Virology); David Berry (University of Wisconsin-Madison, Genetics); Linhui Hao (University of Wisconsin-Madison, Institute for Molecular Virology); Audrey  Gasch (University of Wisconsin-Madison, Genetics); Paul Ahlquist (University of Wisconsin-Madison, Institute for Molecular Virology; Howard Hughes Medical Institute); Mark Craven (University of Wisconsin-Madison, Biostatistics and Medical Informatics; Computer Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We present a method to explain observations from genome-wide mutant assays. For example, consider investigating the genes involved in host-virus interactions by measuring the level of virus replication in each mutant in a yeast knockout library. While many mutants may yield a significant effect on replication, some implicated genes may not directly interact with viral components. Instead, they may be upstream of a smaller set of host factors that do. From an interaction network among yeast genes and small molecules, we select interactions that may consistently account for the experimental observations, suggesting a causal narrative. The network includes protein-protein and protein-nucleic acid interactions, protein complexes, and metabolic pathways. An explanation consists of a subnetwork of interactions and a prediction about which gene or small molecule is most likely to interact with a viral component. A subnetwork may also make conjectures about the roles of unobserved genes. Some challenging aspects of this task include noise in experimental observations, an incomplete interaction network, and redundancy in yeast pathways, which may obscure the effects of a single mutant.<br><br>We apply our method to observations from two assays investigating host-virus interactions, and another measuring mutant fitness under hydrogen peroxide exposure (Kushner et al., 2003; in preparation, Gancarz et al., Hao et al., and Berry et al.). Our ground-truth knowledge is limited: for each phenotype, the mechanism has been determined for only a small number of genes. So, we conduct permutation tests to evaluate the statistical significance of the subnetworks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X60'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X60</strong></td>
 </tr> <tr> 
  <td><b>Biological Connection Markup Language (BCML): A dynamic system for the annotation, visualization and analysis of biological pathways</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Enrica Calura <i>University of Padova</i></td>
 </tr>
 <tr> 
  <td>  Luca Beltrame (Univerity of Florence, Pharmacology); Razvan Popovici (Wayne State University, Department of Computer Science); Lisa Rizzetto (University of Firenze, Pharmacology); Damariz Rivero Guedez (University of Firenze, Pharmacology); Michele Donato (Wayne State University, Computer Science); Sorin Draghici (Wayne State University, Computer Science); Duccio Cavalieri (University of Firenze, Pharmacology); Chiara Romualdi (University of Padova, Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Many models and analysis of signaling pathways have been proposed. However, neither of them takes into account that a biological pathway is not a fixed system, but instead it depends on the organism, tissue, and cell type as well as on physiological, pathological and experimental conditions. The need for dynamism is even more emphasized when studing signalling pathway. As formal descriptions of the signaling processes by which a cell converts certain signals into other, they involve interconnected and finely regulated structures that may present a high level of redundancy. With the intention to be more informative, pathway analysis research moved from the analysis of gene lists to more complex algorithms able to exploit the topology of networks (Kashtan, et al., 2004; Alves, et al., 2006; Draghici, et al., 2007; Massa, et al., 2010). The extraction of the topological information from a biological pathway and their interpretation to obtain a network are not a trivial tasks and are still extremely dependent on the level of details provided by the data format.<br>We propose the Biological Connection Markup Language (BCML) as a format to describe, visualize and analyse pathways. BCML can be automatically converted into data formats suitable for pathway analysis and into a fully compliant System Biology Graphical Notation. BCML is constructed to be dynamic, enabling the storage of multiple information on a single element or reaction, permitting a selective view of the pathway as it exists and behave in specific organisms, tissues or cells.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X61'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X61</strong></td>
 </tr> <tr> 
  <td><b>Delineating signaling network from cancer related pathways</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chandrajit Lahiri <i>Technische Universität München</i></td>
 </tr>
 <tr> 
  <td>  Ashraf Md. Izhar (Institute of Mathematical Sciences, Theoretical Physics); Radhakrishnan Sabarinathan (Indian Institute of Science, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Cancer is a phenomenon of abnormal growth, invasion and translocation of a group of cells. The reasons of cancer being the abnormalities in the genetic material, the ways it can happen could be many. A number of pathways have been shown to be affected wherein numerous proteins are known to be involved. While several reports exist on the role and importance of these individual proteins, the most indispensable of these playing major roles in evoking such abnormalities has not been worked out in detail. We have adopted a theoretical approach to build a protein interaction network (PIN) of these and other associated signal transduction proteins. From such PIN we find a minimum spanning tree forming the skeleton of the network delineating those which could be thought to be the most basic ones for signalling pathways leading to cancer. The backbone comprises EGFR, CDK2, RAF1, RB1, STAT3, CTNNB1, CDKN1B, EP300, SMAD2, TGFB1, PCNA and SOS1 all of which are directly reported to be involved in regulating cell growth and/or adhesion between cells. Again, by a k-core analysis we have found out the most indispensable ones like AKT1, BAD, FOXO3A, BCL2  CASP3, CYCS, BCL2L1, TP53, BCL2L11, MCL1, BBC3 & BAX involved in regulating apoptosis and tumorigenesis. Our method is the first of its kind to figure out, albeit theoretically, potential proteins encoded by these signaling pathways of cancer for therapeutic targets.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X62'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X62</strong></td>
 </tr> <tr> 
  <td><b>The MAP Interactome in Drosophila: an integrative systems biology quest for new mitotic proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Faisal Khan <i>University of Oxford</i></td>
 </tr>
 <tr> 
  <td>  Waqar  Ali (University of Oxford, Statistics); James  Wakefield  (University of Exeter, Biosciences); Charlotte  Deane (University of Oxford, Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We have created an integrated network of microtubule-associated proteins (MAPs), by extending a binary protein-protein interaction network for a set of biochemically identified MAPs in Drosophila. This was done by bringing in homologues and interologs from similar biochemical MAP datasets from four other model organisms, to give a clustered network with numerous, previously uncharacterised, putative MAPs. A preliminary functional analysis of some of these putative MAPs using in vitro RNAi supports their role in mitotic microtubule organisation.<br> <br>Next, we ‘layered’ this extended network with different experimental and bioinformatics data. We added genetic interactions, gene expression data, along with data from a compilation of genome-wide mitotic RNAi screens, and also their domain compositions from the CDD database. All these layers of information were compiled as features and were used for fitting a prediction model that scored these MAPs based on the likelihood of their involvement in mitosis.<br> <br>Many proteins on the network assemble into sub-clusters based on cellular function suggesting similar roles for putative MAPs that are connected to them. By looking at the top 100 proteins from our list of potential MAPs from our model in the Drosophila interactome, different sub-clusters can be seen which correspond to particular stages of mitosis, i.e.  G1/S transition, entry into mitosis, centrosome duplication and the spindle assembly check-point. Many high-scoring potential MAPs can be seen connecting these steps of mitosis, which we now aim to validate and further characterize in vivo using an array of methods from biochemistry, cell biology and genetics.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X63'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X63</strong></td>
 </tr> <tr> 
  <td><b>A network topology-based machine learning approach to extract relevant cancer-related signaling subnetworks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marcio Acencio <i>Universidade Estadual Paulista</i></td>
 </tr>
 <tr> 
  <td>  Ney Lemke (Universidade Estadual Paulista, Physics and Biophysics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In this work we present a network topology-based machine learning approach to extract relevant cancer-related signaling subnetworks from an integrated network of human genes interactions (INGHI). This approach, called graph2sig, is twofold: in addition to predict novel potential oncogenic interactions, it also extracts the candidate oncogenic signaling subnetwork between two genes of interest in the INGHI. By training an ensemble of learning algorithms on known oncogenic interactions, the first step of graph2sig is to construct a predictor that assigns an oncogenic potential (onco_pot) to each interaction in the INGHI. The second step of graph2sig, considering the onco_pot as interaction weight, is to use a path finding algorithm to uncover the candidate oncogenic pathways that will be finally used to build the relevant cancer-related signaling subnetwork. Regarding the prediction of novel putative oncogenic interactions, graph2sig can recover ~84% of known oncogenic interactions with a precision of ~81%. Moreover, the graph2sig is able to assign onco_pot values above 0.7 to ~79% of known oncogenic interactions. With respect of extraction of relevant cancer-related signaling subnetworks, we tested graph2sig by attempting to reconstruct the entire cancer related signaling subnetwork formed by the whole set of known oncogenic interactions. About 62% of known oncogenic interactions could be recovered by graph2sig with a precision of ~20%. Albeit the low precision, hypergeometric distribution–based overrepresentation analysis of this reconstructed subnetwork showed that all cancer-related pathways in KEGG PATHWAY database are significantly overrepresented.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X64'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X64</strong></td>
 </tr> <tr> 
  <td><b>Bottleneck Analysis in Protein – Interface – Protein Interaction Networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Halil Peynirci <i>Koc University</i></td>
 </tr>
 <tr> 
  <td>  Ozlem Keskin (Koc University, Chemical and Biological Engineering); Attila Gursoy (Koc University, Computer Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Topological properties of protein interaction networks are studied to determine important or essential proteins/interactions in the network. In these studies, proteins with high degrees are called as “hubs” and those proteins are considered as essential proteins. Proteins or interactions with high betweenness are called as “bottlenecks” and they are considered as key connector members of the network. However, such studies analyze regular interaction networks and structures of proteins are not taken into consideration. In this study, we propose a “bottleneck” analysis which takes structures of proteins into consideration. In a previous study, substructure on a particular protein that is used through a particular interaction was defined as “interface”. With addition of interface information into regular interaction network a new network definition, Protein – Interface – Protein Interaction Network (PIPN), is made and for case study p53 pathway interaction network is used. Upon analysis of interfaces used in this network, it is found that some interfaces are being used through several interactions and some interfaces overlap with other interfaces. Overlapping and shared interfaces provided a mutually exclusive relationship between interactions and this affected edge betweenness values of interactions. %70 of bottlenecks found in PIPN differed from bottlenecks found in regular p53 pathway network. It is found that bottlenecks found in PIPN are observed mostly between hub proteins whereas they are between hub and/or nonhub proteins in interaction network. Additionally, bottlenecks which are dominated in PIPN connected proteins with DNA related functions such as transcription factors, kinases, replication proteins and oncogene products.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X65'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X65</strong></td>
 </tr> <tr> 
  <td><b>HotRegion: A web server of cooperative hot spots</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Engin Cukuroglu <i>Koc University</i></td>
 </tr>
 <tr> 
  <td>  Attila Gursoy (Koc University, Department of Computer Engineering); Ozlem Keskin (Koc University, Department of Chemical and Biological Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One of the interesting features of interfaces is the degree of contribution of an amino acid to the binding free energy between two proteins. It is well known that not all residues contribute to the same extent in the binding, some are more important and these residues are called hot spots. Experimentally, a hot spot can be detected by alanine scanning mutagenesis. If the binding free energy change is more than 2kcal/mol, the residue is flagged as a hot spot. Further, these hot spots are not randomly distributed in the interfaces but rather they are clustered. The assemblies of hot spots are located within densely packed regions. Within an assembly, the tightly packed hot spots form networks of interactions. These modular assembly regions are called hot regions. An interface may contain none, single, or multiple hot regions. The tight, networked hot spot organization may imply that the contribution of the hot spots to the stability of the protein–protein complex within a hot region is cooperative. This binding site organization rationalizes how a given protein molecule may bind to different protein partners. Here, we present a web server, HotRegion, which finds the organization of hot spots in the interfaces. The input to the HotRegion server is a PDB complex which has an interface between two defined chains. The server provides the hot region distribution results and an interactive 3D visualization of the complex with hot regions highlighted.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X66'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X66</strong></td>
 </tr> <tr> 
  <td><b>GraphCrunch 2 Network Analysis Tool and MI-GRAAL Network Aligner</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Natasa Przulj <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  Oleksii Kuchaiev (Microsoft, Research Software Development); Wayne Hayes (UC Irvine, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Sequence comparison and alignment have had an enormous impact on biological understanding. However, proteins rarely act alone; instead they form complex interaction networks which make the cells work. High-throughput methods for detecting such interactions have produced large biological network data sets with more yet to come. Hence, the problems of biological network modeling, comparison, alignment and clustering are becoming important. We introduce GraphCrunch 2, the only software that simultaneously implements methods to address all of these problems based solely on network topology. <br><br>Network alignment is a way to make sense of protein-protein interaction (PPI) networks.  We introduce a novel network alignment algorithm, called Matching-based Integrative GRAph ALigner (MI-GRAAL), which can integrate any number and type of similarity measures between network nodes including, but not limited to, any topological network similarity measure, sequence similarity, functional similarity, and structural similarity. MI-GRAAL exposes the largest functional, connected regions of PPI network similarity to date: surprisingly, it reveals that 77.7% of proteins in the baker’s yeast high-confidence PPI network participate in such a subnetwork that is fully contained in the human high-confidence PPI network. This is the first demonstration that species as diverse as yeast and human contain large, continuous regions of global network similarity. We apply MI-GRAAL’s alignments to predict functions of un-annotated proteins in yeast, human and bacteria. Furthermore, using network alignment scores for PPI networks of different herpes viruses, we reconstruct their phylogenetic relationship. This is the first time that phylogeny is exactly reconstructed from purely topological alignments of PPI networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X67'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X67</strong></td>
 </tr> <tr> 
  <td><b>Minimum Curvilinearity to address high-throughput protein-protein interaction experiments</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gregorio Alanis-Lobato <i>King Abdullah University of Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  Carlo Cannistraci (King Abdullah University of Science and Technology, Red Sea Laboratory for Integrative Systems Biology); Timothy Ravasi (King Abdullah University of Science and Technology, Red Sea Laboratory for Integrative Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motivation: The most significant functions within the cell emerge thanks to protein-protein-interactions (PPIs), but their high-throughput experimental determination is both expensive and inefficient, producing high amounts of false positives. Computational prediction of reliable PPIs based on prior-network-topology is a valuable strategy to assist in the design of high-throughput PPI studies, pointing towards which interactions the investigation should be addressed. <br>Methods: Topology of PPI-networks is based upon nonlinear distances shaped by some biological properties of a bio-system in a high-dimensional proteomic space. Starting from this idea, we apply Minimum Curvilinear Embedding (MCE) to four Yeast PPI-networks (with different degrees of sparseness) to emphasize relations between close proteins over the network topology embedded in a low-dimensional space. Protein-pairs closer to each other in the reduced space represent potential interactions. We compare the degree of Functional Homogeneity and Localization Coherence of the PPIs predicted by MCE against those predicted by another embedding technique (ISOMAP) and several well-established topological methods (FSWeight, CD-Dist, IRAP, IG2, and IG1).<br>Results: MCE turned out to be a very sound method for PPI prediction. Despite its extreme computational simplicity, MCE performed better than or comparable to the other methods.<br>Conclusion: Minimum Curvilinearity is a valuable nonlinear framework, which we applied in network-systems-biology to unsupervisedly provide a prediction of novel PPIs. The central idea is that prior biological knowledge is hidden in the ‘nonlinear-relations’ of the network topology and might be used for prediction. The predicted PPIs indicate potential true positive interactions to be tested on PPI high-throughput experiments.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X68'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X68</strong></td>
 </tr> <tr> 
  <td><b>iDiscover: an intelligent assistant for integrative analysis of transcriptome data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tomaz Curk <i>University of Ljubljana</i></td>
 </tr>
 <tr> 
  <td>  Crtomir Gorup (University of Ljubljana, Faculty of computer and information science); Gregor Rot (University of Ljubljana, Faculty of computer and information science); Jernej Ule (MRC Laboratory of Molecular Biology, Structural studies division); Blaz Zupan (University of Ljubljana, Faculty of computer and information science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Next-generation high-throughput RNA-Seq and ChIP-Seq technologies produce enormous quantities of data. Even after filtering, reduction and summarization the user is confronted with an overwhelmingly large quantity of results that require tedious manual sifting before identification of potentially novel regulatory patterns. <br> <br>iDiscover is a knowledge-based, interactive web application that enables an efficient and effective scientific discovery through workflows that guide the researcher to the most interesting and unexpected patterns in transcriptome data. As an input it accepts data and results from RNA-Seq and ChIP-Seq analyses performed by iCount - a computational pipeline that we have implemented for the analysis of iCLIP, ncRNA-Seq, mRNA-Seq, CLIP, CLAP, HITS-CLIP, and similar data. The main output of iCount is a nucleotide resolution and precisely quantified genome map of RNA-protein binding sites and mRNA expression. iCount also reports  on quality of reads and mapping, summarizes statistics of expressed and bound regions, and provides detailed genome annotation of binding sites, peak analysis, binding sequence motifs, differential binding, RNA maps, gene expression, and differential expression. While important on their own, all these results require the user to consider many different combinations and take an overwhelming large number of steps to identify interesting patterns in the data. Instead, iDiscover infers rules that relate the experimentally identified transcriptome elements in an experiment and combines the results and patterns from all experiments. Candidate patterns identified in this way are then ranked, allowing the user to focus on best candidates for novel hypotheses on regulatory rules.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X69'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X69</strong></td>
 </tr> <tr> 
  <td><b>Aligning multiple time-series experiments to learn regulatory models: An application to modeling the core network in stem cell differentiation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Irene Ong <i>University of Wisconsin</i></td>
 </tr>
 <tr> 
  <td>  Adam Smith (Oregon Health and Science University, Behavioral Neuroscience); Mark Craven (University of Wisconsin, Biostatistics and Medical Informatics); Ron Stewart (Morgridge Institute for Research, Regenerative Biology); James Thomson (Morgridge Institute for Research, Regenerative Biology); David Page (University of Wisconsin, Biostatistics and Medical Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The discovery that human skin cells can be genetically reprogrammed to create cells that are virtually indistinguishable from embryonic stem (ES) cells (Yu et al., 2007, Takahashi et al., 2007, Okita et al., 2007) motivated our goal of modeling the core regulatory network in stem cell differentiation.  Given five time-series microarray experiments of ES cells undergoing differentiation into different cell states, the task we consider is to find the master regulators of the human ES cell gene network.  To avoid overfitting due to the sparsity of the time-series data, which have non-uniform sampling rates, we align and aggregate the data before learning a dynamic Bayesian network that reflects regulator and regulatee relationships.<br>We use bootstrapping and the assumption that master regulators are transcription factors (TFs) that have the highest number of connections (from regulator-regulatee relationships learned above) in common to construct a cohesive network of 31 TFs.  This network recovers OCT4, SOX2, and NANOG, three TFs involved in reprogramming somatic cells to the ES state (Yu et al., 2007).  15 of the 31 (48%) genes' roles are supported by the literature.  A second method for arriving at core ES genes is to find genes that cluster most tightly with a quantitative measure of commitment of the population of cells as they differentiate.  Only 10 of these 40 (25%) genes are supported by literature.  A third method for identifying ES genes involves finding rapidly down-regulated genes; 10 of these 38 (26%) genes have literature support.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X70'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X70</strong></td>
 </tr> <tr> 
  <td><b>Structural Analysis of Correlation Networks in Bioinformatics:  Graph Theoretic Substructures and the Corresponding Cellular Functions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kathryn Dempsey <i>University of Nebraska Medical Center</i></td>
 </tr>
 <tr> 
  <td>  Hesham Ali (University of Nebraska at Omaha, College of IS&T); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Correlation networks provide an excellent instrument to represent relationships between gene products over entire genomes. Serving as a powerful tool for modeling systems from high-throughput data; this network model can handle a large number of relationships, and thus is ideal for analysis of current and future high-throughput assays. Previous studies applying graph theory toward the identification of network structures reveals multiple layers within a network: high-degree nodes have been found to be critical for network stability; semi-cliques can represent clusters of genes working toward some discrete function; and the interconnectedness of the network itself represents the manifestation of the evolutionary need for maintenance of cellular homeostasis. Due to the relative novelty of the network model in bioinformatics, disparities exist between network creation and modeling standards. In addition, clear connections between the graph theoretic concepts and the corresponding biological functions need to be further defined. Continued use of this model among the scientific community requires these network characteristics be defined, such that basic concepts and the obtained results can be readily communicated for future work supported by these concepts. Using murine studies in aging obtained from NCBI's Gene Expression Omnibus, we establish more refined definitions for several network structures, and give evidence to support that these structures correspond to critical functions and subsystems within the cell itself. Further, we use the identified structural properties along with their ties to biological functions to propose some previously unidentified mechanisms behind aging in the mouse brain.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X71'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X71</strong></td>
 </tr> <tr> 
  <td><b>Systematic, comparative network analysis related to human disease</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Serene Wong <i>York University</i></td>
 </tr>
 <tr> 
  <td>  Igor Jurisica (University of Toronto, Computer Science and Medical Biophysics); Nick Cercone (York University, Computer Science and Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Comparing network structures that characterize healthy and disease state is an important problem as it provides insights to the underlying mechanisms and treatments for complex disease.  However, it is infeasible to compare all aspects of large networks as it requires solving the sub-graph isomorphism problem, which is NP-complete.  We develop a heuristic to compare healthy and disease networks based on local network properties, and the neighborhoods of their correlation difference networks.  The heuristic identifies areas of difference between “healthy” and “disease networks” through their correlation difference networks, and provides a divide and conquer focus, which makes sub-graph enumeration of large networks feasible.  Analysis of several datasets with healthy and lung tumor samples shows the ability to identify differences between healthy and lung tumor networks that is consistent across multiple datasets.  Furthermore, both global and local network structures are observed.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X72'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X72</strong></td>
 </tr> <tr> 
  <td><b>Systematically mapping the druggable pathways of Saccharomyces cerevisiae</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kristen Fortney <i>University of Toronto</i></td>
 </tr>
 <tr> 
  <td>  Wing Xie (University of Toronto, Medical Biophysics); Max Kotlyar (University of Toronto, Medical Biophysics); Igor Jurisica (University of Toronto, Medical Biophysics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Drug modes of action are complex and still poorly understood. The set of known drug targets is widely acknowledged to be biased and incomplete, and so gives only limited insight into the system-wide effects of drugs. But a high-throughput assay unique to yeast – barcode-based chemogenomic screens – can measure the individual drug response of every yeast deletion mutant in parallel. <br>We integrate the four largest S. cerevisiae chemogenomic experiments, which together comprise the responses of thousands of gene knockout strains to over 500 drug treatments, and develop a data-mining approach to investigate drug effects at the systems level. We apply our method to identify yeast pathways, functions, and phenotypes that are targeted by particular drugs. To demonstrate relevance to human disease, we collect groups of disease-associated human genes, map them to their homologs in yeast and apply our method; we recover drugs already prescribed for those diseases and propose several new drug candidates. We also develop methods for modeling the set of all significant pathway-drug connections as bipartite interaction networks. Our analyses of the structure of these networks reveal that while most pathways are targeted by few drugs, some are extremely druggable. Finally, we build YEDr, the YEast Drug database, a searchable interface to our data, methods, and results. Users can query YEDr with new gene groups (yeast, mouse, or human) and YEDr will retrieve the drugs that target them. Human targets are integrated with GeneCards, I2D, mirDIP and other resources.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X73'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X73</strong></td>
 </tr> <tr> 
  <td><b>Global gene network alignment among species by using graph summarization</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Aika Terada <i>Ochanomizu University</i></td>
 </tr>
 <tr> 
  <td>  Jun Sese (Ochanomizu University, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Biological networks, such as protein-protein interactions and gene co-expression network, express molecular relationships and the structures represent molecular functions. As with the conservation of gene sequences between genes having similar functions, finding similar network structures over different species may tell us the functional conservations between species. Advantage of such network comparison over sequence comparison is that network comparison may discover functional similarity even if they have little sequence conservations.<br> One existing network comparison method is network alignment, which focuses on structures of small parts of a large network and is limited to  compare between tightly related molecules such as protein complexes. Therefore, this method is difficult to find conservation of networks about large impact on cellular mechanism such as growth factors and cellular cycles because their networks are large and complicated.<br> To discover such large network conservation over species, we developed a novel method to align the global structure across large networks. Our method has three steps. First, we generate gene clusters on two species network independently. Second, we adjust the clusters such that homologous genes of genes in a cluster tend to be in the same cluster. Third, we modify the clusters such that brief structures of the two networks are similar to each other. We repeat the second and third steps until convergence.<br> We applied our method to protein-protein interactions dataset of C.elegans and D.melanogaster, we found that network structure related to growth are highly conserved while networks related to high order function have little conservation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X74'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X74</strong></td>
 </tr> <tr> 
  <td><b>A systems approach to learning the regulatory network of T helper-17 differentiation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Aviv Madar <i>New York University</i></td>
 </tr>
 <tr> 
  <td>  Maria Ciofani (New York University Medical School, Molecular Pathogenesis Program); Dan Littman (New York University Medical School, Molecular Pathogenesis Program); Richard Bonneau (New York University, Center for genomics and systems biology, Biology department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Sustained inflammation can lead to autoimmune diseases.  T helper 17 (Th17) cells, a pro-inflammatory subset of T cells that normally functions to control mucosal pathogens, are key contributors to multiple autoimmune diseases, e.g. multiple sclerosis.  We use a systems approach to investigate the regulatory events responsible for the differentiation of Th17 cells from naïve T helper cells.  Our integrated approach combines multiple data types: transcription factor (TF) binding data (ChIPseq), RNAseq of TF knockouts and of Th17 differentiation time series, and public microarray data from over 150 other immune lineages (Immunological Genome Project data).  These datasets are complimentary: ChIPseq can identify TF binding sites, RNAseq of TF-knockouts can suggest which of these binding events are regulating a gene, Th17 differentiation time series can resolve the temporal order of regulatory events, and comparison with other immune lineages can suggest which regulatory events are specific to Th17.  Our approach successfully tackles the challenge of combining multiple genomics datatypes (a key problem in systems biology) and is comprised of two main steps: 1) scoring each possible regulatory interaction using dedicated methods to individual datatypes, and 2) integrating the scores of the same regulatory interaction (from these different datatypes) into a single confidence score using the Integrator, a novel quantile based method.  The result is a comprehensive regulatory network for Th17 differentiation, which not only captures all of the previously characterized regulatory events as top hits, but also predicts many novel regulatory interactions with a high accuracy.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X75'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X75</strong></td>
 </tr> <tr> 
  <td><b>Mapping and Simulating Flow-dependent Signaling Pathways in Endothelial Cells</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrew Koo <i>Massachusetts Institute of Technology</i></td>
 </tr>
 <tr> 
  <td>  C Forbes Dewey (Massachusetts Institute of Technology, Mechanical Engineering); David Nordsletten (Massachusetts Institute of Technology, Mechanical Engineering); Beracah Yankama (Massachusetts Institute of Technology, Electrical Engineering and Computer Science); Renato Umeton (Massachusetts Institute of Technology, Mechanical Engineering); Shiva Ayyadurai (Massachusetts Institute of Technology, Biological Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Nitric Oxide (NO) produced by endothelial cells plays multiple roles in vascular stasis including being an anti-oxidant, a mediator of inflammation, and a potent vasodilator.  Not surprisingly, NO production is complexly regulated by multiple pathways.  In order to understand the rich diversity of responses that have been observed experimentally, it is necessary to account for an ensemble of these pathways acting simultaneously- a systems biology problem.<br>We have assembled four different quantitative molecular pathways appearing in the literature that have been proposed for shear stress-induced NO production. In these pathways, endothelial nitric oxide synthase (eNOS) is activated (a) via calcium influx, (b) via phosphorylation reactions, via enhanced eNOS protein expression through (d) the MAP kinase pathway, and (d) the NFkB pathway.  To these we added a fifth pseudo-pathway describing the actual NO production from different calcium-bound or phosphorylated states of eNOS.   All five components were combined using Cytosolve, a new computational environment for combining independent pathway calculations to create complexes of simultaneous biological reactions.  <br>The integrated model is able to describe the changes in NO concentration with time following the application of fluid shear stress to endothelial cells.  The complex time history, arising from interaction between various pathways, is computed.  The results agree favorably with experimental data.  The complete model can also be used to predict the specific effects on NO production following interventional pharmacological and genetic changes to the cell.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X76'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X76</strong></td>
 </tr> <tr> 
  <td><b>Modeling the Spatiotemporal Pattern of Gene Expression During Somitogenesis with Maximum Entropy Deconvolution.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Bernard Fongang <i>University of Texas Medical Branch</i><br></td>
 </tr>
 <tr> 
  <td>Andrzej Kudlicki (University of Texas Medical Branch) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Modeling the Spatiotemporal Pattern of Gene Expression During Somitogenesis with Maximum Entropy Deconvolution.<br><br>Somitogenesis is the process of forming somites, the anterioposterior segmentation of the vertebrate body.<br>According to the clock and wave front model of somitogenesis the spatial periodicity is created by spatiotemporal waves of transcriptional activity, which were first observed by in-situ hybridization. Recent genomic studies suggest the involvement of dozens of genes, mostly from the Wnt, Notch, and FGF signaling pathways.<br><br>We determine the precise sequence of transcriptional events in these pathways in space and time by analyzing the published mouse whole-genome time-course data [Dequeant et al 2006]. This is achieved by applying a filtering algorithm based on the principle of Maximum Entropy Deconvolution [Silver et al 1990]. The main prerequisite for this approach is that a model of synchrony loss can be derived from the design of the experiment; here this requirement is satisfied because of the known geometry of the system. Our results confirm the existing model of the expression wave that is the driving force of somitogenesis.<br><br>We present the temporal and spatial delays between the expression patterns of the genes involved and discuss the results in context of the biophysical models of propagation of the waves along the embryo.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X77'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X77</strong></td>
 </tr> <tr> 
  <td><b>Characterization of transcription factor and microRNA regulatory networks involved in myelination</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Li-Wei Chang <i>Washington University School of Medicine</i></td>
 </tr>
 <tr> 
  <td>  Andreu Viader (Washington University School of Medicine, Genetics); Jeffrey  Milbrandt (Washington University School of Medicine, Genetics); Rakesh Nagarajan (Washington University School of Medicine, Pathology and Immunology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Myelination is the formation of myelin sheath around axons. This process is critical to neural development and in the peripheral nervous system is under the control of Schwann cells. Disruption of myelination is associated with a number of neural diseases.  While several genes have been shown to play important roles in myelination, including myelin proteins Pmp22, Mbp and Mpz, and transcription factors Nab1, Sox10, and Egr2, the genetic programs that govern the regulation of these myelination genes are still unclear.  Furthermore, while microRNAs have emerged as critical regulators of many biological functions and complex human diseases, their role in the coordination of myelination remains largely unexplored. To solve these problems, we have performed mRNA and miRNA expression profiling experiments and developed an integrated, bioinformatic approach to infer the transcriptional and miRNA regulatory networks involved in myelination.  Using a previously developed statistical model for genome-wide regulatory sequence analysis, regulatory targets of transcription factors and microRNAs were characterized based on the enrichment of regulatory sequence elements. Furthermore, promoter sequences and transcriptional regulators of primary microRNAs were also predicted. These computational predictions were combined with mRNA and miRNA expression profiles of myelinating peripheral nerves and highly confident regulatory interactions between transcription factors, microRNAs, and genes were identified and summarized in regulatory networks.  Key regulators in this network were farther validated by experimental methods. Together, these results provide detailed information about the transcriptional and miRNA regulation in Schwann cells.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X78'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X78</strong></td>
 </tr> <tr> 
  <td><b>Extending the Yeast Metabolic Network Using Integrated System Biology Approach</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Paul Fisher <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  Robert Stevens (University of Manchester, Computer Science); Paul Dobson (University of Manchester, Computer Science); BalaKrishna  Kolluru  (National Centre for Text Mining , Data Mining, Pattern Classification); Christian  Brenninkmeijer  (University of Manchester, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Yeast metabolic network is a comprehensive, manually annotated SBML model of metabolism in budding yeast [PMID:18846089]. To extend this network, researchers manually examine publications about yeast to find if any details can extend the SBML model. This manual curation is expensive, tedious and prone to error. A combination of bioinformatics tools are required to generate a semi-automatic approach that can assist in the faster curation of the metabolic network, providing experts with suggested publications, and novel correlations between data entities. <br><br>We present a methodology, using an integrated systems biology approach through the combination of Ondex and Taverna, into a platform that shows how bioinformatics data integration can be created to provide a powerful data analysis platform for systems biology research. <br><br>To assist in the network extension, we have developed Ondex workflows on this platform, capable of running within Taverna, that allow researchers to identify and fill gaps within the yeast metabolism network. These workflows integrate publications into the current metabolic network, allowing Ondex to provide correlations between previously disjoint concepts that can be reviewed by experts for decisions on their inclusion in the network.  By constructing these workflows we have developed a system that reduces the time needed by the curators of the Yeast Jamboree to identify evidence that links new metabolites into the yeast metabolic network, and this has led to progress in extending the Jamboree model [PMID: 21029416].</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X79'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X79</strong></td>
 </tr> <tr> 
  <td><b>An Epistatic Profile of the Cell Envelope Biogenesis in Escherichia coli Evidences Extensive Genetic and Functional Crosstalk Between Processes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>J. Javier Diaz-Mejia <i>University of Toronto</i><br></td>
 </tr>
 <tr> 
  <td>Juan Diaz-Mejia (University of Toronto) Mohan Babu (University of Toronto, Donelly CCBR); Jack Greenblatt (University of Toronto, Donelly CCBR); Gabriel Moreno-Hagelsieb (University of Toronto, Donelly CCBR); Andrew Emili (University of Toronto, Donelly CCBR); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The bacterial cell envelope represents one of the most important targets of evolutionary, physiological and pharmacological studies. Analytical studies in Escherichia coli have delineated ~400 genes/proteins participating in the core biogenesis and maintenance of the envelope. Complementary, genetic interaction studies provide valuable information to determine functional relationship between biological processes. Nevertheless, in prokaryotes the number of known genetic interactions is notoriously underrepresented compared with model eukaryotes and, given the large evolutionary distance between these clades, inference by homology is often challenging. In this study, we attempted to delineate the network of genetic interactions among known and predicted components of the biogenesis and maintenance of the cell envelope in E. coli. We used our recently developed protocol called E. coli Synthetic Genetic Array analysis (eSGA) and the resulting network of genetic interactions was integrated with protein physical interactions and genomic context inferences uncovering unanticipated induced essentiality between processes. For example, disruption of pathways involved in colanic acid biosynthesis shows significant enrichment for genetic interactions with peptidoglycan and lipopolysaccharide assembly, resulting in synthetic lethality and functional crosstalk at pathways level. We also assigned specific roles to genes poorly characterized based on genetic interaction patterns and independent complementary assays. Our study provides a genetic and evolutionary blueprint of the functional crosstalk between and within biological processes occurring during the biogenesis and maintenance of the bacterial cell envelope.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X80'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X80</strong></td>
 </tr> <tr> 
  <td><b>Disease sytems chemical biology of flavors.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>karine audouze <i>Technical University of Denmark / Center for Biological Sequence Analysis</i><br></td>
 </tr>
 <tr> 
  <td>Karine Audouze (Technical University of Denmark / Center for Biological Sequence Analysis) Anne-Marie Le Bon (Centre des sciences du gout et de l alimentation, UMR 1324 INRA); Olivier Taboureau (Technical University of Denmark / Center for Biological Sequence Analysis, Systems Biology); Søren Brunak (Technical University of Denmark / Center for Biological Sequence Analysis, Systems Biology); Rasmus Koefoed Petersen (University of Copenhagen, Biology); Karsten Kristiansen (University of Copenhagen, Biology); Anne  Tromelin (Centre des sciences du gout et de l alimentation, UMR 1324 INRA); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Although, the human olfactory system consists of around 350 odorant receptors (ORs) with diverse sensitivity to flavor molecules, the human odor perception and how odorants might play a major role in our systems biology remain largely unknown.<br>Here, we present a global mapping of flavor molecules in the pharmacological space. Based on a chemogenomic database called ChemProt,, we developed an odorant-target matrix to explore the relationships between chemical structures, biological targets and diseases susceptibility. To validate our approach, we tested seven compounds for the peroxisome proliferator-activated receptor gamma (PPAR?). Six showed PPAR? agonist activities, suggesting potential therapeutic effect for diabetes and inflammation. <br>In a second step, we explored the complexity of the OR-flavor relationships in human defined as odorome. We developed a protein-protein association network (P-PAN) in order to identify potential novel OR-flavor relationships not yet annotated. The P-PAN is based on a recently published computational systems biology method. The P-PAN is generated under the assumption that if two proteins are affected with two chemicals, then both proteins are deemed associating in chemical space. The developed human odorome will help to understand the underlying molecular mechanisms of flavors and the biological pathways they perturb by integrating protein-protein interaction data, protein-disease annotations and functional annotation of proteins.<br>With the proposed computational systems biology approach, identification of disease-gene associations within the human odorome are of potential interest, especially with the fact that many neuropsychiatric disorders might be accompanied by a decrease or increase in odor detection.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X81'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X81</strong></td>
 </tr> <tr> 
  <td><b>Systems-level inference and dissection of drug-target interaction networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Francisco Azuaje <i>Public Research Centre for Health</i></td>
 </tr>
 <tr> 
  <td>  Lu  Zhang (Public Research Centre for Health, Cardiovascular Research); Yvan Devaux (Public Research Centre for Health, Cardiovascular Research); Daniel Wagner (Centre Hospitalier, Division of Cardiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The systems-level characterization of the set of drug-target interactions in specific clinical domains is crucial towards the realization of a P4 (Predictive, Preventive, Personalized, and Participatory) medicine era. There is a lack of such investigations, especially in areas in which a relatively small number of drugs are used in the routine clinical setting. We discuss a new integrative computational strategy to assembling and mining large, unbiased networks of drug-target interactions. The proposed strategy is based on the integration of different types of annotated drug-drug, drug-target and protein-protein interaction information, and extends beyond the incorporation of approved drugs. The network inference phase is followed by extensive topological and biological information analyses. The predictive value of the resulting resources is estimated by statistically linking local and global network properties to specific biological processes, regulatory control mechanisms and disease phenotypes. To demonstrate the validity of this approach and its potential clinical relevance, we introduce My-DTOme: Myocardial infarction drug-target interactome. Based on the elicitation of its modularity properties, we describe how approved and other drugs can be strongly interrelated and affect different clinically-meaningful molecular responses. This involves direct and indirect interactions involving seemingly unrelated biological processes and pathways. We point to areas that deserve new investigations for target discovery and drug repositioning in the specific context of myocardial infarction treatment. My-DTOme inference and mining strategy represents a first step toward the systematic characterization of drug-target interactions in this clinical area.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X82'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X82</strong></td>
 </tr> <tr> 
  <td><b>PhD student</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Mohammad Sadeh <i>Institute for Functional Genomics University of Regensburg</i></td>
 </tr>
 <tr> 
  <td>  Gussi Moffa  (Institute for Functional Genomics University of Regensburg, Statistical Bioinformatics); Rainer  Spang  (Institute for Functional Genomics University of Regensburg, Statistical Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Nested Effects Models (NEMs) are a class of graphical models introduced to analyze the results of gene perturbation screens. NEMs explore noisy subset relations between the high dimensional outputs of phenotyping studies, e.g. the downstream effects observed in gene expression profiles or morphological features of the perturbed cell. A problem with nested effect models is the existence of hidden factors, which are not observed, yet interact with several of the observed variables. Furthermore, NEM inference can become challenging when the distribution of downstream effects is unbalanced. This can make network reconstruction impossible in practice. In such scenarios we propose instead an approach aiming at excluding models which are somehow incompatible with the observed data. This seems more realistic than focusing on the reconstruction of a topology which would almost surely be inaccurate. We call this approach "network exclusion". Extensive simulations on artificial data and an application on wnt signalling, a key pathway in corectal cancer, shows that our approach yields sensible results.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X83'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X83</strong></td>
 </tr> <tr> 
  <td><b>Assessment of the regulatory mechanisms of different protein post-translational modifications by studying their role within a realistic in silico model of the cell.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pablo Minguez <i>European Molecular Biology Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Peer Bork (European Molecular Biology Laboratory, Structural and Computational Biology Unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We study the different modus operandi of protein Post-Translational Modification (PTM) types (e.g. phosphorylation, acetylation, methylation, ubiquitination, glycosylation, etc.) within a multi-molecular network that mimic the cell functionality. The aim is to characterize PTMs by their role within this network to be able to predict new functions in similar not annotated patterns. We generated an in silico model of the cell which includes several molecular types and layers of information such as regulation, protein-protein interactions (ppi) or metabolic relationships including their elements and relationships. These relationships are taken from nearly 20 public resources such as pathways, ppi and PTM databases, that already collect high-throughput experiments, but also from the scientific literature (PubMed abstracts) by means of an in-house text-mining pipeline. This novel network has been annotated by a novel ontology of actions describing molecular relationships. Besides that, we have collected more than 300,000 different PTMs in a total of 300 species that are mapped in the network.<br><br>In here, we present the generation of such a novel network and its annotation which for the first time is done using a common ontology describing the edges (i.e. relationships). Our intention is to describe such a complex network in terms of topology and focus on the role of the PTM events to be able to extract common features (including network motifs) that describe their role within this highly annotated interactome. This framework will set up the bases for functional prediction in other still not described events with similar patterns.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X84'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X84</strong></td>
 </tr> <tr> 
  <td><b>Correction for ascertainment bias in S. cerevisiae interactions: an insight</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jonathan Dickerson <i>The University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  David Robertson (The University of Manchester, Faculty of Life Sciences); John Pinney (Imperial College London, Division of Molecular Biosciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Network biology is concerned with representing cellular interactions and associated biological systems as networks. Substantial amounts of interaction information are curated from the scientific literature. The interests and expertise of researchers, their peers and collaborators, funding streams, public interest and nascent protocols and equipment all guide contemporary research. As such, there is undoubtedly bias in scientific research and hence the resulting literature. Robust analyses are reliant upon accurate data, for instance, in calculating the degree of a particular protein in a protein-interaction network. However, it is feasible that this protein, and hence its interactions, are affected by ascertainment bias. Specifically, if highly studied, it is likely that more interactions involving the protein will be known, thus distorting the observed network structure. We have previously evaluated node-level bias based on the proteins’ publication counts in PubMed. Here we expand this work and present a novel method to evaluate ascertainment bias on interaction networks as a whole. We use rejection sampling to generate randomised sets of protein-protein interactions with the same pattern of bias as the literature-curated yeast interaction network. Applying this method to evaluate classic network properties, we find that rejection sampling performs favourably compared to naïve random sampling. Ultimately, this method will not only greatly facilitate the assessment of bias in literature-based interactomes, it will also enable a more rigorous insight into the significance of biological network properties.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X85'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X85</strong></td>
 </tr> <tr> 
  <td><b>Using community structure for complex network layout</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Arnd Brandenburg <i>Genedata</i><br></td>
 </tr>
 <tr> 
  <td>Oliver Duerr (Genedata) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The approximate calculation of the community structure for large networks can be efficiently performed via the approach of Blondel at al. [1]. This algorithm optimizes the modularity in an iterative process which yields community structures at different scales, by successively aggregating nodes into meta-nodes. We exploit this multi-scale approach for solving two problems related to the layout of networks.<br><br>First, we use the different scales to iteratively generate the layout of the network. Using a standard force-directed layout algorithm, we minimize the energy of the network built from the meta-nodes at the coarsest scale. The procedure is then iterated until the meta-nodes are identical to the nodes of the original network. This results in a drastic reduction of the computational effort for a minimal energy layout.<br> <br>Second, in complex large networks like the protein-protein interaction (PPI) networks, the community structure is not evident  in the layout. By stiffening the springs inside a community, the resulting layouts show a pronounced community structure.<br><br>[1] V.D. Blondel, J.L. Guillaume, R. Lambiotte and E. Lefebvre, J. Stat. Mech. (2008) P10008. DOI : 10.1088/1742-5468/2008/10/P10008</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X86'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X86</strong></td>
 </tr> <tr> 
  <td><b>Quantifying the Response of a Biological System using Network Perturbation Amplitudes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Vincenzo Belcastro <i>Philip Morris Research Laboratories GmbH</i><br></td>
 </tr>
 <tr> 
  <td>Sam Ansari (Philip Morris Research Laboratories GmbH) Florian Martin (Philip Morris Products S.A., Philip Morris International R&D); Alain Sewer (Philip Morris Products S.A., Philip Morris International R&D); Carole Mathis (Philip Morris Products S.A., Philip Morris International R&D); Julia Hoeng (Philip Morris Products S.A., Philip Morris International R&D); Manuel Peitsch (Philip Morris Products S.A., Philip Morris International R&D); Sam Ansari (Philip Morris Research Laboratories GmbH, Philip Morris International R&D); Ty Thomson (Selventa) Dexter Pratt (Selventa) David Drubin (Selventa) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: BACKGROUND:<br>Holistic approaches such as gene expression profiling (GEP), complemented with prior knowledge captured in gene regulatory networks, provide an efficient way of identifying molecular processes systems-wide.<br><br>PURPOSE:<br>The purpose of this study was to develop an approach to quantify the global impact of a treatment  applied to a biological sub-system. This approach is  termed “Network Perturbation Amplitude” (NPA). <br><br>METHODS:<br>Network: A fully curated NF-kB signaling network model was constructed from the Selventa(TM) Knowledge Assembly. The nodes of the network correspond to causally related biological processes that directly regulate the expression of specific sets of genes. <br>Perturbation applied: GEPs were collected from two experiments where Normal Human Bronchial Epithelial (NHBE) cells were exposed to TNFa at different concentrations and times. <br>Algorithm: To score the TNFa perturbation amplitude, all genes regulated by the NF-kB network model nodes were pooled. Four methods were then developed to score the aggregated network effect based on the individual gene response and the overall directionality of the network regulation.<br><br>RESULTS:<br>The four methods were concordant when applied to the NF-kB network and GEPs from the NHBE perturbation experiments. They all correctly indicated dose-dependent responses to TNFa. In order to validate the methods, NPA results for identical treatments (same concentration and exposure time) between two experiments were also compared.<br><br>CONCLUSION:<br>The NPA method provides an objective metric to quantify the global impact of external perturbations on a biological system by combining the knowledge contained in causal network models and systems response profiles such as gene expression.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X87'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X87</strong></td>
 </tr> <tr> 
  <td><b>Query and visualization of RDF resources in Ondex</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrea Splendiani <i>Rothamsted Research</i></td>
 </tr>
 <tr> 
  <td>  Artem Lysenko (Rothamsted Research, Biomathematics and Bioinformatics); Catherine Canevet (Rothamsted Research, Biomathematics and Bioinformatics); Chris Rawlings (Rothamsted Research, Biomathematics and Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As more and more biomedical information is available on the Semantic Web, the focus is shifting from the publication of resources on the Semantic Web to their exploitation. This is a critical step in the adoption of this technological framework, especially because in the Semantic Web  information consumers cannot rely on a consistent information schema, but need to “inspect” what is present in a Semantic Web knowledge base.<br>Tools for the visualization of information, and in particular tools which rely on a graph-based representation of information, are ideal candidates to provide interfaces for the exploration and visualization of RDF resources.<br>Ondex is a data integration and analysis resource which is based on a graph-based representation of information, which is annotated through ontologies. It provides a modular set of data integration and manipulation tools, based on a plugin architecture, and an interactive visualization and analysis tools.<br>We have been studying how Ondex can be adapted to operate on RDF knowledge bases and we have implemented a prototype which offers RDF import functionalities. Given the nature of Ondex, the importer is able to resolve some low level RDF representations (e.g.: reification) and translate it into a more intuitive representation in Ondex.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X88'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X88</strong></td>
 </tr> <tr> 
  <td><b>Causal Reasoning on Biological Networks: Interpreting transcriptional changes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daniel Ziemek <i>Pfizer Inc.</i></td>
 </tr>
 <tr> 
  <td>  Leonid Chindelevitch (Pfizer Inc., Computational Sciences CoE); Ahmed Enayetallah (Pfizer Inc., Compound Safety Prediction); Ranjit Randhawa (Pfizer Inc., Computational Sciences CoE); Benjamin Sidders (Pfizer Inc., eBiology); Christoph Brockel (Pfizer Inc., Research Business Technologies); Enoch Huang (Pfizer Inc., Computational Sciences CoE); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The interpretation of high-throughput datasets has remained one of the central challenges of computational biology over the past decade. Furthermore, as the amount of biological knowledge increases, it becomes more and more difficult to integrate this large body of knowledge in a meaningful manner.<br><br>We recently presented an approach to integrate available biological knowledge by constructing a causal network of molecular interactions. The resulting causal graph can then be queried to suggest causal upstream molecular hypotheses that could explain the variations observed in a particular high-throughput gene expression experiment. Our main contributions are the definition of a scoring function for detecting putative upstream regulators, accompanied by a novel statistical significance metric as well as robustness analysis of the causal graph with respect to noise (Chindelevitch et al., 2011).<br><br>In this poster, we will explain the underlying statistical approach as well as give more details on use cases from our research work leading to testable biological hypotheses for gene expression data sets relevant to drug discovery.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X89'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X89</strong></td>
 </tr> <tr> 
  <td><b>Inferring Regulatory Networks by Combining Perturbation Screens and Steady State Gene Expression Profiles</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ali Shojaie <i>University of Washington</i></td>
 </tr>
 <tr> 
  <td>  Alexandra Jauhiainen (University of Michigan, Statistics); Michael Kallitsis (University of Michigan, Statistics); George Michailidis (University of Michigan, Statistics and EECS); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Reconstructing a transcriptional regulatory network is an important task in functional genomics. Data obtained from experiments that perturb genes by knock-outs or RNA interference contain useful information for addressing the reconstruction problem. However, such data can be limited in size and/or expensive to acquire. On the other hand, observational data of the organism in steady state are more readily available, but their informational content inadequate for the task at hand. We develop a computational approach to appropriately utilize both data sources for estimating a regulatory network. The proposed method offers significant advantages over existing techniques. We develop a three-step algorithm to estimate the underlying directed acyclic regulatory network that uses as input both perturbation screens and steady state gene expression data. In the first step, the algorithm determines causal orderings of the genes that are consistent with the perturbation data. In the second step, for each ordering, a regulatory network is estimated using a penalized likelihood based method, while in the third step a consensus network is constructed from the highest scored ones. The algorithm offers two options for determining all possible causal orderings: an exhaustive search that becomes prohibitive for larger scale problems and a fast heuristic that couples a Monte Carlo technique with a fast search algorithm. Further, it is established that the algorithm produces a consistent estimate of the regulatory network. Numerical results show that the algorithm performs well in uncovering the underlying network and clearly outperforms competing approaches that rely only on a single data source.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X90'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X90</strong></td>
 </tr> <tr> 
  <td><b>Pathway Enrichment Analysis using P-value Selection</b><br/><br/></td>
 </tr>
 <tr> 
 <td>George Michailidis <i>University of Michigan</i></td>
 </tr>
 <tr> 
  <td>  Ali Shojaie (University of Michigan, Statistics); Moulinath Banerjee (University of Michigan, Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Assessing the enrichment of biological pathways is a critical problem in systems biology, which has received much attention in recent years. While simple enrichment scores are easy to interpret and provide useful information about significant elements within each pathway, they fail to incorporate the complex interaction mechanism among the genes. In addition, these methods require a pre-determined cutoff for determining the significance of individual genes. On the other hand, methods of gene set enrichment analysis do not provide any information about the subset of significant genes in each pathway, and as a result, their performance relies on the choice of the gene sets. We propose a new methodology for assessing the significance of biological pathways, which uses a perturbed version of regular p-values, and results in simultaneous reduction in false positive and false negative rates. In addition, the proposed method offers a systematic treatment of correlation among genes and hence provides a reliable inference framework for pathway enrichment analysis. The proposed method has desirable asymptotic, as well as small sample properties, which are confirmed on a variety of simulated and real data examples in comparison to available methodologies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X91'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X91</strong></td>
 </tr> <tr> 
  <td><b>A multi-level Bayesian model for identifying predictive combinations of signaling network modules, proteins, and phosphopeptides for cancer treatments</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ann Chen <i>Moffitt Cancer Center</i></td>
 </tr>
 <tr> 
  <td>  Guolin Zhang (Moffitt Cancer Center, Experimental Therapeutic); Eric Welsh (Moffitt Cancer Center, bioinformatics); Steven Eschrich (Moffitt Cancer Center, Bioinformatics); John Koomen (Moffitt Cancer Center, Molecular Oncology); Eric Haura (Moffitt Cancer Center, Experimental Therapeutic); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There is increasing recognition of the role that signaling networks play in cancer biology.  The state of the art mass-spectrometry based phosphoproteomics profiling technology has been shown to identify and quantify signaling proteins accurately, and further enable protein network mapping.  However, association of expressions of each level of markers (e.g., phosphopeptides, proteins, or pathways) with outcomes of interest [such as drug response or disease status (normal vs. control)] is typically assessed only at the single marker level.   Taking the traditional pathway analysis for example, association of a pathway with an outcome would be performed after summarizing the group behavior of genes within a pathway, but typically the effects of subsets of genes, pathways, and their combinations are not accessed.  We have developed a Bayesian method to integrate experimental proteomics data with existing biological pathway and network information and select combinations of predictive markers across multi-levels of network modules/pathways, proteins, and phosphopeptides simultaneously through a structured stochastic search procedure.  We illustrate this method with an application to phosphoproteomic data generated using liquid chromatography-mass spectrometry (LC-MS/MS) from cellline A204 treated with two different cancer drugs, Dasatinib and Imatinib, at three different concentrations with duplicates each. A total of 485 unique phosphopeptides and 340 unique proteins were identified.  Our results have shown that integration of existing biological knowledge with experimental data illuminates the differences of combinations of signaling modules, kinases, and phosphopeptides between the two treatments.  Our approach also provides insights into combinational therapeutic strategies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X92'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X92</strong></td>
 </tr> <tr> 
  <td><b>Improving Network Completeness of Yeast Transcriptional Interaction Network by Predicted TF-TF Interactions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Mei-Ju Chen <i>National Taiwan University</i></td>
 </tr>
 <tr> 
  <td>  Chien-Yu Chen (National Taiwan University, Bio-Industrial Mechatronics Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Gene regulation involves complicated interactions of several transcription factors (TFs). For investigating TF-TF interactions, our previous work identified 221 interacting TF pairs directly from ChIP-chip data by incorporating motif discovery. The proposed method shows a great improvement in predicting accuracy as compared with other methods using alternative strategies to derive target genes for inferring TF-TF interactions. However, the annotation of several predicted TF pairs remain unclear. This poster aims at utilizing network analysis to investigate the role of the predicted interactions in influencing the network structure of the known interaction network. Three network properties are considered: degree, clustering coefficient and betweenness. We investigate the networks constructed from the following three lists: the predicted pairs, the annotated interactions from databases or literature, and a combined set of the previous two lists. P values for estimating the statistical significance of these networks are derived by performing permutation tests. Background distribution for each property is generated from 10,000 simulated networks of randomly chosen TFs in an equal size of the total TFs present any of the three lists. The results reveal that incorporating the predicted pairs into the annotated interaction network efficiently increases the significances on all of the three properties. For instance, the network with the predicted pairs takes a great advantage on clustering coefficient and thus improves the compactness of the original annotated interacting network. We concluded that the prediction of TF-TF interactions helps to improve the network structure of the current annotated interacting network which might be still considerably incomplete.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X93'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X93</strong></td>
 </tr> <tr> 
  <td><b>A novel Hierarchical clutering method for detecting overlapping clusters</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yunku Yeu <i>University of Yonsei</i></td>
 </tr>
 <tr> 
  <td>  Youngmi Yoon (Gachon university of Medicine & Science, Division of Information Technology); Sanghyun Park (University of Yonsei, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Identifying communities is an essential step for understanding complex system and investigating hidden information. Network clustering is most popular procedure for finding communities in the network. Various kinds of clustering methods have adapted to network clustering.<br>In many real networks, communities are overlapped with each other. It means a node could be assigned into more than one communities. Also, many communities have hierarchical structure. In other word, many small communities join together and make up larger community. This characteristic is obvious in biological domain.<br>In this paper, we propose a novel hierarchical clustering method that finds overlapping communities and constructs their hierarchical structure. We merges more than two nodes at single step of hierarchical clustering. Thus, a smaller community could be merged with many other communities, and that common cluster becomes overlap area among result clusters. <br>We apply our method to PPI(Protein-Protein Interaction) network of Saccharomyces cerevisiae, we found clusters and their overlaps that are involved with biological function significantly.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X94'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X94</strong></td>
 </tr> <tr> 
  <td><b>Generative Probabilistic Models for Protein-Protein Interaction Networks – The Biclique Perspective</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Regev Schweiger <i>Hebrew University in Jerusalem</i></td>
 </tr>
 <tr> 
  <td>  Michal Linial (Hebrew University in Jerusalem, Dept. of Biological Chemistry, Institute of Life Sciences); Nathan Linial (Hebrew University in Jerusalem, School of Computer Science and Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 16. Much of the large-scale molecular data from living cells can be represented in terms of networks. Such networks occupy a central position in cellular systems biology. In the protein-protein interaction (PPI) network, nodes represent proteins and edges represent connections between them, based on experimental evidence. PPI networks are rich and complex, so that a mathematical model is sought to capture their properties and shed light on PPI evolution. The mathematical literature contains various generative models of random graphs. It is a major, still largely open question, which of these models (if any) can properly reproduce various biologically-interesting networks. We are trying to distinguishing between a model family which performs a process of copying neighbors, represented by the Duplication-Divergence (DD) model, and models which do not copy neighbors, with the Barabási-Albert (BA) preferential attachment model as a leading example.<br><br>The property of the network that we observe is the distribution of maximal bicliques in the graph. This is a novel criterion to distinguish between models in this area. It is particularly appropriate for this purpose, since it reflects the graph’s growth pattern under either model. This test clearly favors the DD model. Our results, based on the biclique perspective, conclusively show that a naïve unmodified DD model can capture a key aspect of PPI networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='X95'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X95</strong></td>
 </tr> <tr> 
  <td><b>A Principled Approach to Rewiring the Transcriptome in Yeast to Evolve Networks with Enhanced Heterologous Expression Capability</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Oliver Windram <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  Thomas Thorne (co-author, Centre for Bioinformatics); Travis Bayer (Last Author, Centre for Synthetic Biology and Innovation); Oliver Windram (First Author-Speaker, Centre for Synthetic Biology and Innovation); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Yeasts represent useful eukaryote single cell expression factories with the potential to express foreign proteins with complex post-translational modifications. Indeed, host species such as Saccharomyces cerevisiae and Pichia pastoris are currently used for several industrial scale protein expression platforms for biopharmaceuticals and food additives. With this said however, heterologous protein expression presents a highly unnatural cellular state that invokes a number of responses that can have adverse effects on quality and quantity of the desired product. <br><br>We are seeking to evolve rewired transcriptome networks that facilitate enhanced heterologous protein expression in yeast. New connections are being selected using two methodical approaches: one will use biological intuition of researchers garnered from knowledge obtained from current literature. The other will use network analysis techniques to objectively select components based on proposed function of the native regulator within the transcriptome network. <br><br>Artificial rewired networks which exhibit enhanced protein productivity will be assessed using high-resolution time series expression profiling of transcriptional regulators using real time PCR spanning the period of protein production. Dynamic Bayesian network inference will then be used to model the rewired network to observe the effect that novel edge incorporation has had on network structure. This learned data will be used to inform future attempts to improve productivity of expression hosts. Machine learning tools developed in this study could potentially form the basis of architectural software that could eventually help guide researchers in synthetic network construction in order to optimize the desired outcome.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20233.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='X96'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X96</strong></td>
 </tr> <tr> 
  <td><b>The whole-organism heavy chain B cell repertoire from Zebrafish self-organizes into distinct network features</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sol Efroni <i>Bar Ilan University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The adaptive immune system is based on selected populations of molecularly distinct individual B and T cell clones. However, it has not been possible to characterize these clones in a comprehensive and informatics manner to date; attempts have been limited by the number of cells in the adaptive immune system and an inability to quantify them. Recently, using the Zebrafish (ZF) Danio rerio as a model organism and parallel sequencing as the quantifying technology, Weinstein et al. overcame this major hurdle and quantified the entire heavy chain B-cell repertoire in ZF. Here, we present a novel network analysis of the data from the Weinstein group, providing new insights into the network structure of the B-cell repertoire. <br>Using a collection of computational methods, the IgM sequences from 14 fish were analyzed. This analysis demonstrated that the B-cell repertoire of the ZF is structured along similar lines to those previously detected in limited parts of the human B-cell immune system. The analysis confirms the validity of the global data and the evolutionary placement of the ZF based on known sequence motifs. Recombination events in the repertoire were quantified, and demonstrated a lack of shared recombined V, J groups across fish. Nevertheless, it was demonstrated that a similar network architecture is shared among fish. However, the network analysis identified two distinct populations within the group; these findings are compatible with the occurrence of an immune response in a subset of the fish. The emerging connectivity network was demonstrated and quantified, and mutation drifts within the groups were characterized. Dissection of sequence data revealed common network features of the B-cell repertoire as well as individual differences. <br>The ZF B-cell repertoire reveals an underlying order that is compatible with self-organization representing every portion of the sequence-based network. This pattern varies in individual specimens, perhaps as a response to an immune challenge. However, a sequence-non-specific network that maintains a common architecture of sequence diversity was detected. <br>The common feature among different individuals can be captured by the network architecture and characteristics, rather than specific clones. We believe that further study of the dynamics of this network could provide insight into modes of operation of the immune system.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21050.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='X97'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster X97</strong></td>
 </tr> <tr> 
  <td><b>Protein interaction networks at the thresholds for the appearance of subgraphs?</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tiago Rito <i>University of Oxford</i></td>
 </tr>
 <tr> 
  <td>  Charlotte  Deane (University of Oxford, Statistics); Gesine  Reinert (University of Oxford, Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Large datasets describing Protein Interaction Networks (PINs) are increasingly available. An efficient and biologically meaningful method to compare these PINs is needed. Recent bottom-up strategies for network comparison have become popular, these, count and compare the occurrences of small subgraphs within the networks.<br><br>By comparing graphs from the same model across a wide range of graph density, we have shown that scores based on subgraph counts tend to neglect their instability in the graph density region relevant to PINs. We have previously hypothesized that this instability is due to the fact that PPIs are at the thresholds for the appearance of subgraphs.<br><br>In order to further investigate this behaviour we have identified Exponential Random Graphs as a well-specified stochastic model for PINs. We have also examined threshold detection through the analysis of the empirical distributions of subgraph counts. We illustrate this using a Chi-squared goodness-of-fit test with the null hypothesis that counts of a certain subgraph under a given model come from a Poisson distribution.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




