
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=P by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlistafd9.html?cat=P by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:18 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'P'- Ontologies'</b></td>
 </tr>
</table>

<a name='P01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P01</strong></td>
 </tr> <tr> 
  <td><b>Benchmarking Ontologies: Bigger or Better?</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Lixia Yao <i>Columbia University</i></td>
 </tr>
 <tr> 
  <td>  Andrey Rzhetsky (University Of Chicago, Dept of Medicine); Anna Divoli (University Of Chicago, Dept of Medicine); Ilya Mayzus (University Of Chicago, Dept of Medicine); James Evans (University Of Chicago, Sociology Department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: An ontology represents the concepts and their interrelation within a knowledge domain. Many ontologies have been developed in biomedicine, providing standardized vocabularies to describe genes and proteins, anatomical structures, physiological phenotypes or diseases, and many other phenomena. Scientists use them to encode observations and experimental results, and to perform integrative analysis to discover new knowledge. A remaining challenge is to evaluate how well an ontology represents the underlying knowledge domain. We introduce a family of metrics, including breadth and depth, to capture the conceptual and relational coverage and parsimony of an ontology. We test these measures using four commonly used medical ontologies and seven popular English thesauri (ontologies of synonyms) with respect to text from medicine, news and novels. Results demonstrate that both medical ontologies and English thesauri have a small overlap in concepts and relations, and suggest further efforts to tighten the fit between ontologies and biomedical knowledge domain.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/19919.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='P02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P02</strong></td>
 </tr> <tr> 
  <td><b>Functional coherence assessment for protein groups and its application to pathway assignment</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Meghana Chitale <i>Purdue University</i></td>
 </tr>
 <tr> 
  <td>  Daisuke Kihara (Purdue University, Departmen of Biological Sciences, Department of Computer Science); Shriphani  Palakodety (Purdue University, Department of Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: With the availability of large scale omics data, computational function elucidation of proteins needs to take a step further to understand the systems level interplay of proteins. Traditional analyses had mainly focused on analyzing single gene function at a time, but to understand the governing biological principles behind the functional units, we would like to identify functionally coherent groups of proteins working in co-ordination. <br><br>We have developed and analyzed two coherence measures for identifying biologically relevant protein groups. Our scores quantify functional coherence of proteins by considering association of Gene Ontology (GO) terms observed in two actual biological contexts, co-occurrences in protein annotations and co-mentions in literature. The first score is named the Co-occurrence Association Score (CAS) while the second one is named the PubMed Association Score (PAS). Importantly, functional coherence is not identical with functional similarity because proteins with different function work for a same pathway in a coordinated fashion. Our scores have shown to have the superior ability for accurately discriminating biologically relevant protein groups, such as proteins in the same pathway, proteins in the same complex, and proteins which co-localize in a cell when compared with the other state-of-the-art methods. Our scores were also effectively applied to detect physically interacting pairs of proteins. Remarkably, our scores were successful in assigning 90-94% of proteins to correct pathways within the top ranks based on their coherence.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P03</strong></td>
 </tr> <tr> 
  <td><b>A Modular Annotation Framework for Mathematical Models</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Michael Cooling <i>University of Auckland</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The last ten years has seen the development of model encoding schemes for documenting and sharing mathematical models. Now hundreds of published computational biology models are available for free, online, encoded in languages such as the XML-based CellML. <br>Having succeeded in availability, a next challenge is to increase utility through the addition of semantic meaning to model code. The kinds of metadata that would be immediately useful include: biological (allowing computer-assisted model composition from smaller models, and intelligent search and retrieval of repositories), authorship (so one knows who to credit, and/or contact with queries), citation (describing the connections between the model and academic works), licensing and general comments, among others.<br>In collaboration with the CellML team (http://www.cellml.org/community/team), I have developed a metadata specification framework to better enable the annotation of CellML models with metadata. The framework consists of a core specification describing, in general terms, how annotations should be attached using RDF/XML, and how satellite specifications drawing on existing orthogonal ontologies should be described. Satellite specifications for the kinds of annotation described above have been developed using elements from the Dublin Core, FOAF (Friend-Of-A-Friend), BIBO (Bibliographic Ontology) and Biomodels Qualifiers. The development of this framework allows the attribution of semantic meaning to sharable, reusable mathematical models</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P04</strong></td>
 </tr> <tr> 
  <td><b>An ontology of bioinformatics tool functions, data types and formats</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jon Ison <i>EMBL-EBI</i></td>
 </tr>
 <tr> 
  <td>  Peter Rice (EMBL-EBI, Rice Group); Matus Kalas (University of Bergen, Bergen Center for Computational Science); James Malone (EMBL-EBI, Functional Genomics Team); Hamish McWilliam (EMBL-EBI, External Services); Steve Pettifer (The University of Manchester, Advanced Interfaces Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Advances in the discovery, comparison and connection of bioinformatics resources require consistent machine-understandable descriptions of the underlying tools and data. A comprehensive controlled vocabulary of common concepts that is broadly applicable is therefore urgently required.<br><br>EDAM is a simple ontology of bioinformatics tool functions, data types, formats and identifiers.  It provides a controlled vocabulary of over 2000 defined terms for annotating diverse entities such as Web services, standalone tools, data schema, data objects and databases.  EDAM provides coarse coverage of Web services and data from BioCatalogue and is ready for pilot annotations.<br><br>EDAM is available in OBO (Open Biomedical Ontologies) format from http://sourceforge.net/projects/edamontology/files/.  For documentation and license see http://edamontology.sourceforge.net/.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P05</strong></td>
 </tr> <tr> 
  <td><b>Multiple Ontology Based System for Integrating and Display Rat Phenotype Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Mary Shimoyama <i>Medical College of Wisconsin</i></td>
 </tr>
 <tr> 
  <td>  Rajni Nigam (Medical College of Wisconsin, Rat Genome Database, Human and Molecular Genetics Center); Melinda Dwinell (Medical College of Wisconsin, Rat Genome Database, Human and Molecular Genetics Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The strength of the rat as a model organism lies in its utility in pharmacology, biochemistry, disease, and physiology research. Data resulting from such studies is difficult to represent in databases and the creation of user-friendly data mining tools has proven difficult.  Typically clinical measurement data for model organisms such as the rat is housed by experiment or combined only when researchers follow identical protocols. This prevents researchers from accessing and analyzing the vast amount of valuable phenotype data that has been generated for the rat. To address this issue, multiple ontologies and data formats were developed at the Rat Genome Database to standardize this data.  The ontologies include the Clinical Measurement Ontology, Measurement Method Ontology, Experimental Condition Ontology and the Rat Strain Ontology.  All are available at RGD (rgd.mcw.edu) or the National Center for Biomedical Ontology (bioportal.bioontology.org/). These ontologies and accompanying data fields were used to create standardized formats for the major components of phenotype records: who was measured, what was measured, how was it measured and under what conditions was it measured. The resulting PhenoMiner system (rgd.mcw.edu/phenotypes/) consists of three components. The first is an ontology based database which has incorporated more than 14,000 records from multiple sources.  A second component consists of data mining and visualization tools and the third component is the ontology based curation software for creating and editing the data. The PhenoMiner system provides an outstanding resource for the rat research community.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P06</strong></td>
 </tr> <tr> 
  <td><b>ontoCAT R package for ontologies traversal and search</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Natalja Kurbatova <i>European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Natalja Kurbatova (European Bioinformatics Institute, Functional Genomics Group); Tomasz Adamusiak (European Bioinformatics Institute, Functional Genomics Group); Misha Kapushesky (European Bioinformatics Institute, Functional Genomics Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: An R package called ''ontoCAT'' provides a simple interface to any ontology described in OWL or OBO formats. The ontoCAT R package supports these two formats widely used for ontology creation and provides unified, format independent access to the ontology terms and ontology hierarchy.<br><br>Our package's focus is the basic methods that allows ontology traversal and search: search of the terms, getting paticular term's parents and/or children, showing paths to the term from the root element of the ontology, showing flat tree representations of the ontology hierarchy, list and search of relations, etc. <br><br>Additional functionality of the ontoCAT R package allows to work with group of ontologies and with the major ontology repositories like OLS and Bioportal, namely search terms within batch of ontologies, list available ontologies and load selected ontology for futher analysis when needed.<br><br>The ontoCAT R package has been used in a number of projects at EBI and has proven its usefulness.<br><br>Considering all mentioned above we believe the ontoCAT R package can be extermly useful for the bioinformatics society working in the R environment.<br><br>The package is based on the Ontology Common API Tasks Java library (http://www.ontocat.org).<br><br>We are providing two versions of the ontoCAT R package:<br>- Light ontoCAT package version is available in Bioconductor starting from the release 2.7. It includes all functionality except methods to work with batch of ontologies, since Bioconductor package size is limited with 2 MB.  <br>- Full ontoCAT package version includes batch methods and is available from the ontoCAT website  http://www.ontocat.org/wiki/r</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P07</strong></td>
 </tr> <tr> 
  <td><b>What does semantic similarity mean in terms of protein function comparison? The analysis of relation between functional and structural similarity.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Bogumil Konopka <i>Wroclaw University of Technology</i></td>
 </tr>
 <tr> 
  <td>  Tomasz Golda (Wroclaw University of Technology, Faculty of Fundamental Problems of Technology, Institute of Biomedical Engineering and Instrumentation); Pawel Wozniak (Wroclaw University of Technology, Faculty of Fundamental Problems of Technology, Institute of Biomedical Engineering and Instrumentation); Malgorzata Kotulska (Wroclaw University of Technology, Faculty of Fundamental Problems of Technology, Institute of Biomedical Engineering and Instrumentation); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Functions of a substantial number of proteins in  wwPDB and the Uniprot databases are  represented by Gene Ontology terms. The hierarchical structure of GO allows to quantify the similarity of protein functions by applying semantic similarity algorithms. Their result can be used to  discover evolutionary relationships between proteins or aid proper protein classification. However its usefulness is limited since relation between the structural similarity and functional similarity of proteins, described by GO terms, has not been yet thoroughly investigated. The aim of this work is to define the exact meaning of protein function similarity given by measures  of semantic similarity.<br>In order to estimate the actual value of a result of functional comparison between two proteins, we  calculated the similarities between sets of GO terms randomly chosen from the ontology. We also calculated pairwise similarities between randomly chosen proteins.  We propose a Z-score statistic to measure the significance of pairwise similarities by relating the score to the distribution of results of those random comparisons. Using the Z-score metric we ran a systematic analysis of functional similarity of proteins within SCOP classes at different levels of structural similarity. <br>Results of functional comparison of random proteins show a distribution that could be approximated with an extreme values distribution. The study showed that on the same level of structural similarity there are SCOP classes where function is better conserved. This heterogeneity should be taken into consideration in development of techniques that benefit from the structure?function relationship.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P08</strong></td>
 </tr> <tr> 
  <td><b>A Semantic Web Infrastructure for Bioinformatics of Staphylococcus aureus</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Cândida Delgado <i>Instituto de Tecnologia Química e Biológica</i></td>
 </tr>
 <tr> 
  <td>  Jonas Almeida (University of Alabama at Birmingham, Department of Pathology); Maria Miragaia (	 Instituto de Tecnologia Química e Biológica, Laboratório de Genetica Molecular); Hermínia de Lencastre (The Rockefeller University, Laboratory of Microbiology); Study Group on Molecular Epidemiology of Staphylococcus (Instituto de Tecnologia Química e Biológica, Laboratório de Genetica Molecular); Helena F Deus (Digital Enterprise Research Institute, Health Care and Life Sciences Domain); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Staphylococcus aureus, a leading cause of human bacterial infections worldwide, can cause a wide range of diseases. In Europe, hospital-acquired methicillin-resistant Staphylococcus aureus infections are estimated to affect more than 150,000 patients annually. Although much is known about this pathogen, the efficient sharing and linking of knowledge is still hampered by scarce bioinformatics infrastructures.<br>Linked Data consists of a set of best practices for enabling standardized access and query over aggregated data. This integrative potential can be employed in the development stage of bioinformatics apparatus that support molecular epidemiology research. Furthermore, recent developments have made JavaScript a powerful prototype-based object-oriented scripting language, ideal for web-based user interfaces. In this report we exploit the Simple Sloppy Semantic Database (S3DB), as the basis for an extensible, usage-oriented, bioinformatics infrastructure geared towards the study of Staphylococcus aureus epidemiology.<br>The key outcomes of this work are an Ontology model for Staphylococcus epidemiology and a JavaScript system designed to simplify visualization and interpretation of high-throughput datasets submitted to S3DB (S3DB Ontology Analytics). Both were primarily designed based on requirement assembly and regular evaluation by its ultimate users, the biology experts. A core infrastructure that will integrate multiple views over the same ontologies and support user-friendly interaction with the dataset (viewing, editing or creating) is under development. This system will be reusable in various molecular domains, including cancer research, as a model for publishing and consuming Linked Data. This design model has been shown to be proficient in the management of multiple biomedical datasets.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P09</strong></td>
 </tr> <tr> 
  <td><b>A standard for cooperative interactions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kim Van Roey <i>European Molecular Biology Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Toby Gibson (European Molecular Biology Laboratory, Structural and Computational Biology Unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Cells must continuously monitor external and internal cues, integrate the wide variety of signals they perceive, and translate these inputs into proper outputs. This process of signal transduction depends on the formation of large, dynamic macromolecular complexes. Assembly of these signaling complexes depends on multiple transient, low-affinity interactions. Many of these interactions are mediated by short linear peptide motifs, some of which can be additionally regulated by post-translational modifications, and have been observed to be highly cooperative. Such cooperative interactions provide the dynamic plasticity that is required for cells to integrate multiple input signals, robustly and reliably transmit information, and rapidly generate appropriate responses.<br><br>Despite the importance of cooperative interactions in molecular biology, formalisms that can adequately capture and represent such interactions are currently lacking. By means of a literature survey, we will collect cooperative interaction data and use it to define classes of information to be stored. From this analysis, we will develop a data exchange format and create a controlled vocabulary that can describe the relevant features of the collected data set. Once we can computationally describe cooperative interactions, a relational database will be developed, and a prototype will be populated with cooperative interaction data extracted from the literature. As a guideline for experimentalists to unambiguously describe cooperative interactions, the Minimum Information About a Cooperative Interaction (MIACI) will be defined. Together, these tools will facilitate the systematic capture, comparison, exchange and verification of cooperative interaction data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P10</strong></td>
 </tr> <tr> 
  <td><b>Mining Gene Ontology data with AGENDA</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Guvanch Ovezmyradov <i>University of Goettingen</i></td>
 </tr>
 <tr> 
  <td>  Martin Goepfert (University of Goettingen, Cellular neurobiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: With the emergence of novel genetic techniques and the exponential accumulation of genomic data, the need for powerful bioinformatics tools is growing accordingly. The Gene Ontology (GO) database is one of the most essential information sources in biology. Here we present AGENDA (Application for mining Gene Ontology data), a novel web-based tool that provides practical access to the GO database. AGENDA can be used to efficiently explore genetic information within and across species. The application comes with a user friendly interface and offers novel possibilities for the user. AGENDA provides diverse search options and offers possibilities to bookmark or download the results. The program undergoes active development to suit the needs of the research community. We anticipate that this tool will provide novel insights into the genetic basis of diverse biological processes and facilitate comparisons in-between. AGENDA is open source software that is freely available for non-commercial use. AGENDA can be queried online at http://bioagenda.uni-goettingen.de.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P11</strong></td>
 </tr> <tr> 
  <td><b>BioPAX – A Community Standard for Pathway Data Sharing</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nadia Anwar <i>Memorial Sloan-Kettering Cancer Center</i></td>
 </tr>
 <tr> 
  <td>  Emek Demir (MSKCC, cBio); Gary Bader (University of Toronto, The Donnelly Centre); Igor Rodchenkov (University of Toronto, The Donnelly Centre); Ozgun  Babur (MKSCC, cBio); Chris Sander (MKSCC, cBio); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The BioPAX ontology (www.biopax.org) is a standard language for formally representing biological pathways and is available to the biological community to enable exchange and integration of pathway data. BioPAX level 3 supports the representation of metabolic pathways, signal transduction pathways, protein-protein interaction networks, gene regulatory networks and genetic interactions. <br><br>Data exchange and integration continues to be a challenge given the complex nature of both pathway data and data sources. Biological pathways are constructs that biologists use to represent relationships between and within chains of cellular events. BioPAX was developed to address these issues and to ease the access, use, exchange and aggregation of pathway data.<br><br>This poster will outline data representation in BioPAX and the use of the BioPAX ontology in integration, analysis and visualization of pathway data, which enables more efficient use and reuse of these data. We will outline progress on the further work highlighted by the BioPAX community, including: visualization and exchange of co-ordinates, semantic web applications, enhancing the use of external taxonomies and controlled vocabularies within BioPAX, aligning BioPAX to SBML for quantitative modeling and the continued development of the core ontology to support novel data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='P12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster P12</strong></td>
 </tr> <tr> 
  <td><b>Infer Novel Genes and Pathways that Modulate Sphingolipid Pathway from a Novel Yeast Gene Network Derived from Ontology Fingerprints</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tingting Qin <i>Medical University of South Carolina</i><br></td>
 </tr>
 <tr> 
  <td>Wenjin Zheng (Medical University of South Carolina) Tingting Qin (Medical University of South Carolina, Biochemistry); Lam Tsoi (University of Michigan, Biostatistics); Nabil Matmati (Medical University of South Carolina, Biochemistry & Molecular Biology); Bidyut Mohanty (Medical University of South Carolina, Biochemistry & Molecular Biology); Andrew Lawson (Medical University of South Carolina, Medicine); Yusuf Hannun (Medical University of South Carolina, Biochemistry & Molecular Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We integrated biomedical literature, ontology, network analysis and experimental investigation to infer novel genes that can modulate yeast sphingolipid pathway.  Such modulations may play important roles in regulating the cellular functions of bioactive lipids.  We first constructed novel gene networks by performing pairwise comparison of all yeast genes’ Ontology Fingerprints—a set of ontology terms overrepresented in the PubMed abstracts linked to a gene along with their corresponding enrichment p-value. The comparison generated a weighted undirected gene network where genes are nodes and the similarity scores between genes are weighted edges.  The network was further refined by applying a Bayesian hierarchical model to distinguish biologically relevant connections from those that are not. To infer novel genes potentially involved in sphingolipid pathway modulation, we identified a subnetwork of well known yeast sphingolipid genes together with non-sphingolipid genes that have no “sphingo” prefix or “ceramide” in PubMed abstracts and descriptions associated with these gene.  These non-sphingolipid genes are considered as candidate genes that can modulate sphingolipid pathway.  We tested the Myriocin sensitivity of the deletion strains of the top and bottom ranked candidate genes.  The proportion of top ranked candidates genes whose deletion showing altered sphingolipid pathway activity (Myriocin sensitivity) is significantly higher than that among lower ranked ones, and the lipidomic profiles of these deletion strains are significantly different from that of wide type. Our novel network analysis provides a powerful tool to study pathway modulation and can be applied to study human disease related pathways.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




