
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=Z by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist8001.html?cat=Z by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:19 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'Z'- Other'</b></td>
 </tr>
</table>

<a name='Z01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z01</strong></td>
 </tr> <tr> 
  <td><b>BOOST: A Boolean Representation-based Method for Detecting SNP-SNP Interactions in Genome-wide Association Studies</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Xiang Wan <i>The Hong Kong University of Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  Can Yang (The Hong Kong University of Science and Technology, Electronic and Computer Engineering); Qiang  Yang (The Hong Kong University of Science and Technology, Computer Science and Engineering); Hong Xue (The Hong Kong University of Science and Technology, Life Science); Xiaodan Fan (The Chinese University of Hong Kong , Statistics); Nelson Tang (The Chinese University of Hong Kong , Chemical Pathology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Gene-gene interactions have long been recognized to be fundamentally important for understanding genetic causes of complex disease traits. At present, identifying gene-gene interactions from genome-wide case-control studies is computationally and methodologically challenging. In this paper, we introduce a simple but powerful method, named ‘‘BOolean Operation-based Screening and Testing’’<br>(BOOST). For the discovery of unknown gene-gene interactions that underlie complex diseases, BOOST allows examination of all pairwise interactions in genome-wide case-control studies in a remarkably fast manner. We have carried out interaction analyses on seven data sets from the Wellcome Trust Case Control Consortium (WTCCC). Each analysis took less than 60 hr to completely evaluate all pairs of roughly 360,000 SNPs on a standard 3.0 GHz desktop with 4G memory running the Windows XP system. The interaction patterns identified from the type 1 diabetes data set display significant difference from those identified from the rheumatoid arthritis data set, although both data sets share a very similar hit region in the WTCCC report. BOOST has also identified some disease-associated interactions between genes in the major histocompatibility complex region in the type 1 diabetes data set. We believe that our method can serve as a computationally and statistically useful tool in the coming era of large-scale interaction mapping in genome-wide case-control studies.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/19882.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z02</strong></td>
 </tr> <tr> 
  <td><b>Linguistic analysis of genomic islands reveals a recent acquisition of genetic materials by Mycobacterium tuberculosis from alpha-Proteobacteria</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Oleg Reva <i>University of Pretoria</i></td>
 </tr>
 <tr> 
  <td>  Oliver Bezuidt (University of Pretoria, Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Important genes may flux among bacteria by horizontal transfer and trigger substantial evolutionary changes. Novel genome linguistic approaches helped to identify several genomic islands in Mycobacterium tuberculosis and trace them down to alpha-Proteobacteria origin. Identified genomic islands are distinctive from prophage inserts common for Mycobacteria.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/19932.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z03</strong></td>
 </tr> <tr> 
  <td><b>Domain architecture conservation in orthologs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kristoffer Forslund <i>Stockholm Bioinformatics Centre</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As orthologous proteins are expected to retain function more often than other homologs, they are often used for functional annotation transfer between species. While the assumption of higher similarity in various respects of orthologs over paralogs at the same evolutionary separation is widespread and supported by many isolated examples, it has not been systematically evaluated, and some recent publications have questioned its general validity. Because of this, we set out to consistently validate the specific effect of orthologous versus paralogous evolution on the conservation of domain architectures in homologous proteins. <br><br>We carried out a large-scale study of domain architecture conservation or change on a large-scale dataset. We designed a score to measure domain architecture similarity and used it to analyze differences between orthologs and close outparalogs as defined by InParanoid. To compensate for differences between the pair types in average evolutionary divergence, we divided the protein comparisons into bins based on Jukes-Cantor corrected protein sequence identity, and analyzed the difference between ortholog and paralog pairs within each bin separately.<br><br>The analysis shows that orthologs exhibit significantly greater domain architecture conservation than non-orthologous (paralogous) homologs, even when differences in evolutionary divergence are compensated for. We interpret this as an indication of a stronger selective pressure on orthologs than on paralogs to retain the domain architecture required to perform a specific function. This supports the notion that orthologs are functionally more similar than other types of homologs at the same evolutionary distance.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20157.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z04</strong></td>
 </tr> <tr> 
  <td><b>CADDSuite: A flexible and open framework and workflow system for computer-aided drug design</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marcel Schumann <i>University of Tuebingen</i></td>
 </tr>
 <tr> 
  <td>  Marc Röttig (University of Tuebingen, Div. for Applied Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Virtual Screening and molecular docking usually require a large number of diverse and often incompatible programs. Problems arise when different programs require dissimilar file formats, are not available due to license issues, or have to be used in very different kind of ways. Therefore, in practice many potentially important steps are often omitted or cannot be sufficiently adapted to the user's needs and obtained results are often unreproducible.<br><br>Here we present a comprehensive framework,  based on the Biochemical Algorithms Library (BALL), that provides a broad range of functionality, covering most common areas of interest for computer-aided drug design. It contains, among others, procedures for preparing receptor and ligand structures, for scoring receptor-ligand complexes, for evaluating results as well as our own algorithms for performing QSAR analysis and molecular docking.<br><br>Using the 40 benchmark data sets supplied by DUD (http://dud.docking.org), we will show that our docking algorithm, used in conjunction with preparation tools supplied by our framework, overall performs very compatitively to well-known docking programs and on several of those data sets also out-performs the latter.<br><br>Furthermore, all tools of our framework have been integrated into the workflow management system galaxy. This way, the programs can be used directly from a webbrowser, without any need for programming skills or installation of tools on the part of the user. Workflows can be easily generated, stored and reused, which makes successive application of tools much easier.<br><br>CADDSuite, including its integration into galaxy, will be made available free of charge as open-source.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20170.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z05</strong></td>
 </tr> <tr> 
  <td><b>Inferring Gene Regulatory Networks from Expression Data using Tree-based Methods</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Van Anh Huynh-Thu <i>University of Liege</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One of the pressing open problems of computational systems biology is the elucidation of the topology of genetic regulatory networks (GRNs) using high throughput genomic data, in particular microarray gene expression data. In this work, we present GENIE3, an original algorithm for the inference of GRNs. GENIE3 decomposes the prediction of a regulatory network between p genes into p different regression problems. In each of the regression problems, the expression pattern of one of the genes (target gene) is predicted from the expression patterns of all the other genes (input genes), using tree-based ensemble methods. The importance of an input gene in the prediction of the target gene expression pattern is taken as an indication of a putative regulatory link. The whole network is then reconstructed by aggregating putative links over all genes. Our method was evaluated in the context of the Dialogue for Reverse Engineering Assessments and Methods (DREAM) challenge, which is an annual international competition aiming at the evaluation of GRN inference algorithms on benchmarks of simulated and real data. The GENIE3 method was best performer on the DREAM4 In Silico Multifactorial challenge in 2009 and on the DREAM5 Network Inference challenge in 2010. In addition, the resulting method is simple and generic, making it adaptable to other types of genomic data and interactions.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20190.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z06</strong></td>
 </tr> <tr> 
  <td><b>Investigating The Evolution of Novel Enzyme Function And Chemistry Within Structurally Defined Protein Superfamilies</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nicholas Furnham <i>European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Nicholas Furnham (European Bioinformatics Institute, -); Gemma Holliday (European Bioinformatics Institute, -); Ian Sillitoe (University College London, Institute of Structural and Molecular Biology); Alison Cuff (University College London, Institute of Structural and Molecular Biology); Christine Orengo (University College London, Institute of Structural and Molecular Biology); Janet Thornton (European Bioinformatics Institute, -); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A significant proportion of gene products are annotated as having enzymatic functions, which, as biological catalysis, are essential for life. In addition, many of the targets of pharmaceutical drugs are acting to modify the behavior of enzymes. Thus, an understanding of how enzymes have evolved to undertake the wide variety of reactions they perform is essential to many studies in biology and medicinal chemistry. To unravel this problem requires the combination of protein three-dimensional structural, sequence, phylogenetic and chemistry information. We have combined this variety of data in an automatic pipeline for investigating enzyme functional evolution within structurally defined protein superfamilies. This has permitted us to analysis all enzymatic superfamilies cataloged by the CATH database. In addition to showing relationships between structures and sequences though phylogeny, we are able to show relationships of the small molecule metabolites the enzymes are acting on as well as similarities between the mechanisms by which an enzyme performs its overall reaction. This allows us to demonstrate the power of combining the range of information to show features across multiple superfamilies as well as unique qualities of specific enzyme superfamilies, thus providing a means to improve function prediction and contribute to the design of novel enzyme functions.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20192.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z07</strong></td>
 </tr> <tr> 
  <td><b>Structural Biology Meets Systems Biology: A Structural Systems Biology Approach for Gauging the Systemic Effect of Single Nucleotide Polymorphisms</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tammy Cheng <i>Cancer Research UK London Research Institute</i></td>
 </tr>
 <tr> 
  <td>  Tammy Cheng (Cancer Research UK London Research Institute, Biomolecular Modelling Laboratory); Linda Jeffery (Cancer Research UK London Research Institute, Cell Cycle Laboratory); Lucas Goehring (Max Plank Institute, Dynamics and Self-Organization); Yu-En Lu (University of Cambridge, Computer Laboratory); Jacqueline Hayles (Cancer Research UK London Research Institute, Cell Cycle Laboratory); Bela Novak (University of Oxford, Department of Biochemistry); Paul Bates (Cancer Research UK London Research Institute, Biomolecular Modelling Laboratory); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Small changes in protein structure, such as non-synonymous single nucleotide polymorphisms, can have a large impact on cellular behaviour. To understand how a change at the protein structure level eventually affects a cell's phenotypic outcome is, however, not trivial. This is because complex, multi-scale, information needs to be considered in order to obtain analytical results with physiological meanings. To tackle this issue, we have developed a structural systems biology approach, PEPP (Phenotype Extrapolation via Pathway and Protein information), to effectively integrate protein structural information with pathway dynamics. <br><br>Here we demonstrate the new method by studying point mutations in two biological systems: (1) Regulation of the G2 to mitosis transition in Schizosaccharomyces pombe (fission yeast) and (2) the human ERK pathway. In the first system, we use mitosis-promoting factor as a reporter protein and studied 11 mutations that result in various extent of growth before yeast cells enter the mitosis stage. In the second system, we used ERK as a downstream reference to study 40 mutations that lead to phenotypically overlapping symptoms under the broad term 'neuro-cardio-facial-cutaneous syndrome'. By combining the information on both pathway and protein structure levels, we are able to gauge the phenotypic effect of the mutations – in the yeast model, PEPP reflects the general trend of cell sizes measured experimentally; in the human model, PEPP clusters the point mutations into subgroups that reflects their clinical symptoms.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20194.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z08</strong></td>
 </tr> <tr> 
  <td><b>In silico prediction of the calpain degradome</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Lawrence Wee <i>Institute for Infocomm Research</i></td>
 </tr>
 <tr> 
  <td>  Hwee Meng Low (Genome Institute of Singapore, NextGen Sequencing); Joo Chuan Tong (Instituite for Infocomm Research, Data Mining); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Calpains belong to a family of calcium-dependent cysteine proteases which are implicated in a myriad of pathologies such as cancer and neurodegeneration. Despite extensive experimental work on calpain activity, the precise mechanisms of substrate recognition and cleavage remain to be determined. In addition, many more substrates are expected to be discovered and characterized. We extracted 341 unique, experimentally-verified calpain cleavage sites from literature and analyzed the data for unique sequence signatures. We observed previously unknown conservation of amino acids in the immediate vicinity of the cleavage site which could be exploited for cleavage site discrimination. We trained a series of SVM classifiers incorporating different window segments enclosing the cleavage site and two different modes of feature encoding (simple binary and Bayes Feature). The trained SVM classifiers achieved AROC of 0.79-0.83 and 0.84-0.93 on independent test sets using simple binary and Bayes Feature encoding respectively. The best performing SVM classifier was applied on the entire human proteome and a proteome-wide atlas of potential calpain cleavage sites was constructed. In addition, we analyzed predicted cleavage sites on proteins from the family of receptor tyrosine kinases (RTKs) and observed co-localization with caspase cleavage sites at critical regulatory domains in several RTKs. As caspase cleavage was found to modulate downstream signaling of RTK, these results suggest a novel function of calpains in regulating RTK function.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20200.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z09</strong></td>
 </tr> <tr> 
  <td><b>A scalable signaling map identifies cross-talk between gene modules inferred from multi-parametric RNAi phenotypes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Xin Wang <i>Cancer Research UK Cambridge Research Institute</i></td>
 </tr>
 <tr> 
  <td>  Xin Wang (Cancer Research UK Cambridge Research Institute, University of Cambridge Department of Oncology); Roland Schwarz (Cancer Research UK Cambridge Research Institute,  University of Cambridge Department of Oncology); Florian Markowetz (Cancer Research UK Cambridge Research Institute,  University of Cambridge Department of Oncology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Recently, there is an increasing interest in combining large-scale RNA interference (RNAi) screens with automated imaging to describe and quantify the impact of gene perturbations on multiparametric cell morphologies. Analyzing multiparametric phenotypes is a very challenging task, because they provide richer information than individual reporters (for which specialized approaches exist) but less than high-dimensional phenotypes like microarrays. Multiparametric phenotypic data are often represented as association networks clustering together genes with similar phenotypes. However, association networks are based on symmetric similarity measures between phenotypic profiles and cannot infer directed hierarchies of information flow between gene modules and pathways.<br>Here, we propose to build networks from asymmetric association measures that allow inferring directed graphs. Our approach builds on Nested Effects Models (NEMs), a group of graphical models for reconstruction of cellular signaling hierarchies from subset relationships of downstream perturbation effects. We extend NEMs to accommodate multiparametric morphological screens by scaling network resolution to the richness of phenotypic information.  Our method consists of two steps: First, we cluster genes into modules based on subset relations. Second, we infer directed relationships between the modules. We propose an efficient algorithm that makes use of the fact that internal interactions of modules do not affect estimation of signal flow between modules.<br>The algorithm was applied to the multiparametric morphological RNAi screens conducted by Fuchs et al. Using their clustering results the signaling map yields inconsistencies and unexpected cross-talk between ‘simple gene modules’ that regulate single morphological phenotypes. In contrast, our subset-based clustering method minimizes these inconsistencies.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20201.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z10</strong></td>
 </tr> <tr> 
  <td><b>Stability of domain folds in Multi-domain proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ramachandra Bhaskara <i>Indian Institute of Science</i></td>
 </tr>
 <tr> 
  <td>  Ramachandra M Bhaskara (Indian Institute of Science, Molecular Biophysics Unit); Narayanaswamy Srinivasan (Indian Institute of Science, Molecular Biophysics Unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Many proteins comprise of multiple domain and their abundance is dependent on the complexity of the organism. In spite of humongous work done on protein folding and stability, the precise role of domain-domain interactions in multi-domain proteins and its relationship to independent stable existence of domains remain obscure. Using extensive empirical energy computations we find that domains of multi-domain proteins are less stable independently owing to their large hydrophobic inter-domain interface which gets exposed to water during independent existence. Their independently stable single-domain homologues are characterized by specific residue changes at the sites equivalent to domain-domain interface of multi-domain proteins. Our results suggest biophysical explanations for the two optimal solutions employed by nature to stabilize protein domains. Stability of domains having a large hydrophobic patch on the surface of the fold is influenced either by tethering them to other domains resulting in a multi-domain protein or by them being part of obligate functional complexes. The stable single-domain homologues are often results of divergence of duplicated sequences which have undergone positive selection of preferential stabilizing mutations in the hydrophobic patch to avoid forming misfolded aggregates and inclusion bodies. We provide safe guidelines for mutational studies on protein domains for modulating protein stability. Our analysis of nsSNPs resulting in mutations at the domain-domain interfaces of human proteins suggests that destability in the disease causing variant is often associated with replacement of apolar residues in the interface by polar residues. Such changes affect communication between domains leading to improper folding and non-functional proteins followed by disease manifestation.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20202.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z11</strong></td>
 </tr> <tr> 
  <td><b>PINALOG: a novel approach to align protein interaction networks – implications for function prediction and complex detection</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hang Phan <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Analysis of protein interaction networks (PINs) at the system level has become increasingly important in understanding biological processes. Comparison of the interactomes of different species not only provides a better understanding of species evolution but also helps with detecting conserved functional components and function prediction. Here we have developed a PIN alignment method, called PINALOG, which combines information available for the proteins in the networks, including sequence, function and topological information. Alignment of human and yeast PINs from the IntAct database reveals several conserved subnetworks between the two species that participate in similar biological processes, notably the proteasome and transcription related processes. We also systematically quantify the power of function prediction based on the resulting alignment by direct transfer functions of mapped proteins. With a test set composing of proteins with low PSI-BLAST sequence percent identity, cross validation has demonstrated the improved performance of function prediction made by PINALOG over PSI-BLAST, where we obtained much better recall at a similar level of precision. PINALOG was used to predict functions for unannotated human proteins which have no PSI-BLAST hit.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20205.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z12</strong></td>
 </tr> <tr> 
  <td><b>Exploring structure-function and evolutionary relationships in proteins of TIM fold</b><br/><br/></td>
 </tr>
 <tr> 
 <td>GARIMA AGARWAL <i>Indian Institute of Science</i></td>
 </tr>
 <tr> 
  <td>  GARIMA AGARWAL (PhD student, Molecular Biophysics Unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: TIM fold presents a classical example of “one fold many functions” in protein space. The fold serves as an appropriate model to understand the origin of catalytic diversity through identification of functional determinants as well as gain evolutionary insights. Although, the members of this fold have been studied well, evolutionary relationships among them are still unclear. Structure of a protein being more conserved than sequences, a structure-based phylogenetic study has been performed to discern complex structure-function and evolutionary relationships not obvious from sequence comparisons. A non-redundant dataset of over 270 domains, corresponding to 86 families in 17 superfamilies of the TIM fold has been analysed. Based on a dissimilarity measure that quantifies the extent of global structural dissimilarity, members in many superfamilies, co-cluster highlighting their evolutionary relatedness. The members within each of these clusters were analyzed in detail to identify the distinguishing features common to members within a cluster. For the superfamily members that do not fall in a single clade, the domains performing similar functions were found to cluster. Each case was separately explored to understand the basis of clustering to identify unifying features.  <br><br>Additionally, co-clustering of hypothetical proteins with members of known function has been analyzed which gives clues to predicting function of functions of uncharacterized members whose structures are available through structural genomics initiatives.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20206.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z13</strong></td>
 </tr> <tr> 
  <td><b>Essentiality of proteins on the basis of their membership of protein complexes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>David Talavera <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  David Robertson (University of Manchester, Michael Smith Building); Simon Lovell (University of Manchester, Michael Smith Building); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Gene essentiality has challenged researchers for years. Duplication events have been used to explain gene essentiality, the suggestion being that many singletons are likely to be essential, whereas gene duplication can generate synthetic-lethal pairs. However, most research has focused on single-gene essentiality and has considered synthetic lethal pairs separately. By contrast, in this work we consider the latter as essential pairs of genes. We have analysed the similarities between both types of essential genes and conclude that paralogy alone cannot determine gene essentiality. Indeed, we show that functional singularity is a better explanation of essentiality; i.e.: some pairs can functionally compensate each other, even if they are not evolutionary related, whereas the function of other proteins cannot be compensate by any other protein. Furthermore we demonstrate the existence of a subnetwork of essential interactions that includes essential proteins from both types. Previous research had generated a debate on the role of essential proteins in the interactome. Our approach can reconcile current opposing hypotheses. Finally, we show that functional convergence accounts for some cases of loss of single-gene essentiality without the emergence of duplicates. We advocate for a redefinition of essentiality according to our results: essentiality is a feature of the functional complexes instead of being inherent in protein themselves.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20207.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z14</strong></td>
 </tr> <tr> 
  <td><b>Comparing nucleotide-based and codon-based alignment of human next-gen sequencing exome data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Szilveszter Juhos <i>Omixon</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Software used to align reads from next-gen sequencers is rarely using the underlying information that the reads are the result of a biological process. When analyzing data from exome sequencing, even more information is available as the sequences are corresponding to proteins and some mutations in codons are more likely than others. We are presenting a method for SOLiD color-space read alignment in where we incorporated DNA (F84) and protein (Goldman-Yang 1994) evolution rules to achieve more precise alignment. We found that with these extensions we are more likely to find SNPs and MNPs with less false positives. Furthermore, microindels of up to 10-20 nucleotides long can be reliably found even in homopolimer or repetitive regions.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20211.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z15</strong></td>
 </tr> <tr> 
  <td><b>RIBER and DIBER: a software suite for crystal content analysis  in the studies of protein-nucleic acid complexes.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Grzegorz Chojnowski <i>International Institute of Molecular and Cell Biology in Warsaw</i></td>
 </tr>
 <tr> 
  <td>  Janusz M. Bujnicki (International Institute of Molecular and Cell Biology in Warsaw, Laboratory of Bioinformatics and Protein Engineering); Matthias Bochtler (Cardiff University, Schools of Chemistry and Biosciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In co-crystallization experiments with protein and nucleic acids, it is not always clear whether the crystal contains the complex or one component alone. In this study w present a new program RIBER, which provides as easy way to judge RNA content of a crystal based solely on a diffraction data, before the structure is solved. RIBER, and a previously developed DIBER for detecting DNA, form a complete suite of programs for preliminary analysis of crystal content in the crystallographic studies of protein-nucleic acid complexes. The use of the machine learning techniques makes reliable, quantitative prediction of the crystal content available for non-expert users and high throughput crystallography.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20223.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z16</strong></td>
 </tr> <tr> 
  <td><b>SpliceGrapher: Predicting Splice Graphs from Diverse Evidence</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Asa Ben-Hur <i>Colorado State University</i></td>
 </tr>
 <tr> 
  <td>  Mark Rogers (Colorado State University, Computer Science); Anireddy, S.N. Reddy (Colorado State University, Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Deep transcriptome sequencing with next-generation sequencing (NGS) provides unprecedented opportunities for researchers to assess the extent of alternative splicing in many species.  Although it is inexpensive and easy to obtain whole transcriptome data in this manner, one limitation has been the lack of versatile methods to analyze the data.  We present a new method called SpliceGrapher, which is designed to enhance existing gene annotations on the basis of NGS reads and EST alignments.  SpliceGrapher predicts splice graphs which are a compact representation of all the ways in which a gene's exons may be assembled.  We demonstrate our approach using NGS read data from Arabidopsis and grape, and find that SpliceGrapher's predictions are better aligned with the existing annotations than those of other tools.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20226.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z17</strong></td>
 </tr> <tr> 
  <td><b>Topological features predict misannotations in metabolic networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Rodrigo Liberal <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  Rodrigo Liberal (Imperial College London, Centre for Bioinformatics, Division of Molecular Biosciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Misannotation in sequence databases has been a recognised problem for more than a decade. Recent studies have confirmed that this problem is still a reality, identifying over-prediction and error propagation as the main error sources. This issue is an important obstacle for the success of automated tools for gene function annotation, which rely extensively on comparison to sequences with known function. To improve current annotations and prevent future propagation of errors, tools are therefore needed to assist in the identification of misannotated gene products. <br>In the case of enzymatic functions, each functional assignment implies the existence of a reaction within the organism’s metabolic network; a first approximation to a genome-scale metabolic model can be obtained directly from an automated genome annotation. Any obvious problems in the network, such as dead-end or disconnected reactions, can therefore be strong indications of misannotations. In this work, we demonstrate that machine learning can be used to predict misannotated proteins using only the topological information from a metabolic network, i.e. independently of the protein sequences. A decision tree trained on gold standard sets of correct and incorrect EC number assignments for 331 species in KEGG is able to successfully separate correct annotations from incorrect ones, achieving an accuracy of 80% in ten-fold cross-validation. We expect this method to be of use in the process of refining metabolic network models and in the detection of erroneous functional assignments within sequence</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20228.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z18</strong></td>
 </tr> <tr> 
  <td><b>A Theoretical Approach to Identify Indispensable Proteins of the Type III Secretion Systems of Salmonella enterica Serovar Typhimurium Strain LT2</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chandrajit Lahiri <i>Technische Universität München</i></td>
 </tr>
 <tr> 
  <td>  Chandrajit Lahiri (Technische Universität München, Microbiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: An array of proteins of Type III secretion systems are used by the pathogenic bacteria Salmonella enterica serovar Typhimurium strain LT2 to inject into the host cells to start with the process of systemic infection. While several reports exist on the role and importance of these individual proteins of Salmonella pathogenecity islands (SPI), the most indispensable of them and their hierarchical role has not been worked out in detail.<br>We have adopted a theoretical approach to build a network of these and other associated signal transduction proteins from where we propose a hierarchy of proteins which comes into action in the process of invasion and colonization and thus becomes indispensable.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20230.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z19</strong></td>
 </tr> <tr> 
  <td><b>Analysis of bacterial genes with disrupted ORFs reveals frequent utilization of Programmed Ribosomal Frameshifting (PRF) and Programmed Transcriptional Realignment (PTR).</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pavel Baranov <i>University College Cork</i></td>
 </tr>
 <tr> 
  <td>  Virag Sharma (University College Cork, Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Bacterial genome annotations contain a number of CDSs that in spite of disruption(s) of initial reading frame encode a single continuous polypeptide. Such disruptions have different origins: sequencing errors, frameshift or stop codon mutations, as well as instances of unconventional decoding utilization (Recoding). We have extracted over one thousand CDSs with annotated disruptions and found out that  about 75% of them can be clustered into 64 groups based on sequence similarity. Analysis of the clusters revealed deep phylogenetic conservation of ORF organization as well as presence of conserved sequence patterns that indicate likely utilization of the programmed ribosomal frameshifting (PRF) and the programmed transcriptional realignment (PTR). Further enrichment of these clusters with homologous nucleotide sequences revealed over six thousand candidate genes utilizing PRF or PTR. Analysis of the patterns of conservation apparently associated with non-triplet decoding revealed  presence of both previously characterized frameshift-prone sequences  and a few novel ones. Since the starting point of our analysis was a set of genes with already annotated disruptions, it is highly plausible that in this study we have identified only a fraction of all bacterial genes that utilize PRF and PTR. In addition to identification of a large number of alternatively decoded genes, a surprising observation as that about half of them are expressed via PTR - a mechanism that, in contrast with PRF, has not yet received substantial attention.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20234.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z20</strong></td>
 </tr> <tr> 
  <td><b>An Event-driven Approach to Study Operon Evolution</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Iddo Friedberg <i>Miami University</i></td>
 </tr>
 <tr> 
  <td>  David Ream (Miami University, Microbiology); Mark Wohlever (Miami University, Mathematics & Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Operons and functional gene neighborhoods are characteristic of bacterial genomes, although not <br>exclusively limited to bacteria. Many protein complexes, metabolic pathways and information-processing pathways are encoded by genes in operons. Indeed operons are a critical component of bacterial adaptation, and it is estimated that  5-25% of bacterial genes reside in operons. Understanding how operons are formed and dissolved in evolutionary time would provide insight to one of the most basic and prevalent bacterial genomic structures.<br>The construction and/or destruction of clusters of co-transcribed genes can be described as a sequential series of defined events. By casting these events as attributes we can employ statistical learning to classify evolutionary paths of operons, and connect these paths with biological function.<br><br>Using a set of 33 proteobacteria and 38 different E. coli operons, we have established  attributes for the study of operon evolution. These attributes enable us to explore the evolutionary trajectory of a single operon from a reference organism within a phylogenetic tree. Some attributes correlate with evolutionary distance between taxa, while others do not. Which attributes correlate with phylogenetic distance depends on the operon, although some attributes, such as global migration, almost always correlate with phylogenetic distance. Using principal component analysis and k-means clustering we have shown that operons with similar cellular roles may cluster together based on their evolutionary trajectories. This clustering is not  based on common evolutionary descent (homology), but on common evolutionary events that occur in the history of operons which may also indicate a functional relationship.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20235.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z21</strong></td>
 </tr> <tr> 
  <td><b>RTA and MultiRTA: Simple yet Accurate Methods for Predicting Class II MHC Epitopes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrew Bordner <i>Mayo Clinic</i></td>
 </tr>
 <tr> 
  <td>  Andrew Bordner (Mayo Clinic, Department of Molecular Pharmacology and Experimental Therapeutics); Hans Mittelmann (Arizona State University, School of Mathematical and Statistical Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The binding of antigen fragments to class II MHC proteins is crucial for generating a helper T cell response.  Knowledge of which peptide bind a particular MHC protein can aid in vaccine design and in discovering new therapeutics for autoimmune diseases and allergies.  However, the large number of MHC types combined with the enormous number of possible peptide sequences precludes comprehensive experimental characterization..  We have developed a new prediction method called Regularized Thermodynamic Average, or RTA, that utilizes available binding data to identify novel epitopes.  This method has the following features: (1) it accounts for multiple binding conformations, (2) it includes a regularization constraint to reduce overfitting, (3) it is simpler than most competing methods. and (4) its has physically interpretable parameters.  RTA was found to outperform all other recent prediction methods on common data sets.  We also developed an extension of the RTA method, called MultiRTA, for predicting epitopes for multiple MHC types.  The MultiRTA model for HLA-DR achieved comparable or better prediction performance than other methods, even while being considerably simpler.  We also fit the first HLA-DP prediction model and discovered new peptide binding motifs.  In addition, we explored two extensions to RTA that further improved its accuracy, namely including additional terms for pairs of neighboring peptide residues and for peptide residues flanking the core binding segment.  Finally, we discuss two applications of these models: combining them with structure-based methods to develop a universal prediction model and discovering promiscuous epitopes for vaccine development.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20237.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z22</strong></td>
 </tr> <tr> 
  <td><b>Metagenomic Biomarker Discovery and Explanation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Curtis Huttenhower <i>Harvard School of Public Health</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Understanding how and why biomolecular activity differs among environmental conditions or disease phenotypes is one of the central questions addressed by high-throughput biology. Biomarker discovery, the process of finding and explaining these differences, has proven to be both a methodological challenge in high-dimensional statistics and biologically challenging to interpret. Metagenomics provides a new avenue for biomarker discovery, since changes in the composition and functional activity of microbial communities can provide insight into ecological differences among communities or diagnostic or prognostic power when applied to the human microbiome. We propose the LDA Effect Size (LEfSe) algorithm to discover and explain microbial and functional biomarkers in the human microbiota and other microbiomes. We demonstrate this method to be effective in mining human microbiomes for metagenomic biomarkers associated with mucosal tissues and with different levels of oxygen availability. Similarly, when applied to 16S rRNA gene data describing a murine ulcerative colitis gut community, LEfSe confirms the key role played by Bifidobacterium in this disease and suggests the involvement of additional clades including the Clostridia and Metascardovia. Finally, we provide characterizations of microbial functional activity from metagenomic community sequencing, comparing environmental bacterial and viral microbiomes and distinguishing the infant gut microbiota from adult. A quantitative validation of LEfSe in comparison to existing microbial biomarker discovery methods and to standard statistical approaches (including synthetic data) highlights a lower false positive rate, consistent ranking of biomarkers’ relevance, and concise representations of taxonomic and functional shifts in microbial communities associated with environmental conditions or disease phenotypes.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20238.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z23</strong></td>
 </tr> <tr> 
  <td><b>Differential expression with RNA-seq: a matter of Depth</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ana Conesa <i>CIPF</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We investigate the relationship between sequencing depth and differential expression detection in RNA-seq experiments. We show that current statistical approaches have a strong dependency of their differential expression calls on the amount of available reads and that this leads to a high number  false discoveries specially of genes of short length, with small fold-change differences including off-target non coding genes. We present a novel methodology -NOISeq- which is robust to this kind of biases. NOISeq, by adopting an empirical approach to model the null distribution of differential expression, captures better the shape of noise in RNA-seq data and obtains more accurate and consistent results.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20239.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z24'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z24</strong></td>
 </tr> <tr> 
  <td><b>Post-transcriptional regulators of microRNA biogenesis regulate pathogenesis of cancer</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pavel Sumazin <i>Columbia</i></td>
 </tr>
 <tr> 
  <td>  Pavel Sumazin (Columbia University, Systems biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: MicroRNAs (miRs) have emerged as key regulators of both normal and pathologic phenotypes, including cancer, but fine grained regulation of their biogenesis is still poorly understood. In order to understand the extent and specificity of miR-biogenesis control, as well as the role of miR-biogenesis regulators in tumorigenesis and cancer progression, we set out to identify these regulators and profile their targets. We developed an algorithm for genome-wide inference of miR-biogenesis regulators, and identified regulators that are specific to individual miRs or miR families in glioma and ovarian cancer. Our algorithm identified known biogenesis regulators, including DGCR8, HNRNPA1, DDX5, LIN28 and SMAD family proteins, and predicted new miR-biogenesis regulators that are common to both cancers or specific to one tumor type. We validated miR-biogenesis regulators that are common to both cancers and target tumor- and prognosis-specific miRs, including oncomirs miR-218, miR-23b, and miR-155. We showed that miR-biogenesis regulators can act before or after cropping by DROSHA, and that they can alter expression of large sets of microRNAs. Our results suggest that miR biogenesis is a complex, context specific, and finely-regulated process, and that miR-biogenesis regulators may influence tumor initiation and progression by altering the expression of specific tumor-suppressor miRs and oncomirs or by modifying large miR programs.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20240.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z25'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z25</strong></td>
 </tr> <tr> 
  <td><b>RDyNet: bridging functional analyses and network visualization</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Mauro Alves Castro <i>Cancer Reseach UK - Cambridge Reseach Institute</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Graph visualization and statistical analysis of gene networks are essential parts of scientific studies that make use of systems biology approaches. However, in general it requires a constant switch between different applications in order to run a complete analysis. Here we present RDyNet, a software to bridge the gap between network analysis and visualization. RDyNet is an R based package combined with a powerful Java engine that incorporates the major technologies already validated for the analysis of gene networks together with the ability to dynamically visualize network behavior and its changes over time. More specifically, RDyNet establishes a new level of customization: for R Developers, it allows the development of Java plug-ins exclusively using R codes; for Java Users, it runs R methods implemented in a stand-alone application, and for R Users RDyNet interactively displays R graphs using a robust Java graphic engine embed in R.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20236.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z26'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z26</strong></td>
 </tr> <tr> 
  <td><b>Modulators of microRNA Activity Regulate Glioblastoma Pathogenesis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Xuerui Yang <i>Columbia Univesity</i></td>
 </tr>
 <tr> 
  <td>  Xuerui Yang (Columbia University, Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: MicroRNAs (miRs) have been shown to drive pathogenesis and prognosis in cancer tumors, including glioblastoma, the most common and the most aggressive type of primary human brain tumor. MiR activity is modulated by miR-target abundance and by post-transcriptional factors that regulate miRISC-mediated mRNA degradation. MiR-activity modulators are potential key regulators of cancer, but the extent to which they regulate cancer remains unknown. We developed an algorithm to help assemble the repertory of tumor-specific miR-activity modulators as a step towards elucidating their regulatory effect on pathogenesis of cancer. We identified over a hundred glioblastoma miR-activity modulators, which activate or suppress miRISC-mediated regulation through protein interaction or act as miR decoys by titrating miRs away from other mRNAs. We identified dozens of oncogenes and tumor suppressors that are regulated by miR-activity regulators in glioblastoma, including master regulators RUNX1 and PTEN. We experimentally validated WIPF2 as a miRISC-mediated modulator of RUNX1, and PALB2 and WNT7A as miRISC-mediated modulators of PTEN. Moreover, a number of genes, including RUNX1, were identified as decoy modulators of PTEN. Interestingly, PTEN and its 7 modulators bear genetic alterations in 60% of 462 GBM samples. Both modulator silencing and PTEN-3’UTR transfection confirmed these 7 modulators as decoy modulators of PTEN, suggesting that genetic alterations at these genes post-transcriptionally regulate PTEN. In summary, our results suggest that miR modulators post-transcriptionally regulate the expression of master regulators of glioblastoma, thus playing a significant role in its tumorigenesis and progression.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20247.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z27'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z27</strong></td>
 </tr> <tr> 
  <td><b>Scalable metabolic reconstruction for metagenomic data and the human microbiome</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Curtis Huttenhower <i>Harvard School of Public Health</i></td>
 </tr>
 <tr> 
  <td>  Sahar Abubucker (Washington University, The Genome Center); Nicola Segata (Harvard School of Public Health, Biostatistics); Alyxandria Schubert (University of Michigan, Microbiology and Immunology); Beltran Rodriguez-Mueller (San Diego State University, Biology); Jeremy Zucker (Broad Institute) Human Microbiome Project consortium   ( ) Patrick Schloss (University of Michigan, Microbiology and Immunology); Dirk Gevers (Broad Institute) Makedonka Mitreva (Washington University, The Genome Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Microbial communities carry out the majority of biochemical activity on the planet, and they play integral roles in metabolism and immune homeostasis in the human microbiome. Here, we describe HUMAnN, a methodology for determining the functional and metabolic potential of a microbial community by inferring pathways present or absent and their relative abundances directly from short metagenomic reads. We validated this methodology using a collection of four synthetic metagenomes, accurately determining the presence and abundance of pathways and outperforming standard best-hit approaches. Finally, we analyzed 741 samples drawn from 7 body sites on 103 individuals as part of the Human Microbiome Project (HMP), demonstrating the scalability of our methodology and the critical importance of microbial metabolism in the human microbiome. Previous studies have found that no organisms are present in all body sites or individuals; conversely, we find that 19 of 220 pathways are confidently present in every HMP community, and 53 are present in at least 90% of samples. This demonstrates a degree of functional consistency that is lacking at the organismal level - who's there varies, but what they're doing is more constant. Conversely, the relative abundances of most pathways vary among body sites but not individuals, suggesting that community function is dictated by microbial environment and less strongly by the host. This does not yet speak to host genetics, environment, or disease, as the HMP comprises a normal baseline of healthy individuals; each of these represents an additional area for future studies of microbial community function.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20253.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z28'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z28</strong></td>
 </tr> <tr> 
  <td><b>Systems-level insights from modular decomposition of the yeast genetic interaction network</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jeremy Bellay <i>University of Minnesota</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genetic interactions provide a powerful perspective into biological processes that is fundamentally different from other high-throughput genome-wide studies. Recently, Synthetic Genetic Array technology was used to produce the first genome scale map of digenic genetic interactions, which covered ~5.4 million genetic interactions or about ~30% of all possible gene pairs in yeast. This provides a first opportunity for a global, unbiased assessment of the structure of the genetic interaction network and the relationship between this structure and individual gene function.  We developed a data mining approach based on association rule learning to exhaustively discover all block structures within the yeast genetic interaction network, producing a complete modular decomposition of the network. We find that genetic interaction hubs can be clearly differentiated into distinct classes of hubs based on their modular structure.  Moreover, module membership provides a specific and unbiased assessment of the prevalence of multi-functionality among genes:  we find that genes participate in far more functions or contexts than was previously appreciated. In addition, we find that genetic interactions contained within structured modules exhibit strikingly different functional properties relative to isolated interactions, providing insight into the evolution and functional divergence of duplicate genes.  Finally, we used module membership to explore the evolution of module redundancy within the cell and find that while there is often age coherency within modules, modules that buffer each other often arise at different times indicating that much cellular redundancy is a product of subsequent diversification.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20254.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z29'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z29</strong></td>
 </tr> <tr> 
  <td><b>Large scale merging of microarrays. From curation to meta-curation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Colin Molter <i>University of Brussels</i></td>
 </tr>
 <tr> 
  <td>  Colin Molter (university of Brussels, CoDE-IRIDIA); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As new technologies are developed and studies remain underpowered, gene expression microarray data is abundantly and freely available. As such, merging microarray data from several studies can be used as primary, larger, datasets; and to increase the sample size and diversity, as well as to validate, a proprietary genomewide study.   While technically challenging, recent results have proved promising results by integrating expression data from different studies at the expression value level after transforming the expressions to numerically comparable measures. Combined with our web-based collaborative platform for re-annotation of studies and of merged meta-studies, we provide a generic means of generating gene expression compendia on-the-fly.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/20245.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z30'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z30</strong></td>
 </tr> <tr> 
  <td><b>INBIOMEDvision: Promoting and Monitoring Biomedical Informatics in Europe</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Miguel Mayer <i>Universitat Pompeu Fabra INBIOMEDvision Consortium (INBIOMEDvision Consortium, European Coordination and Support Action)</i><br></td>
 </tr>
 <tr> 
  <td>Victoria López Alonso (Instituto de Salud Carlos III) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: INBIOMEDvision is a European Support Action funded by the FP7-ICT. The main goal of the project is to promote Biomedical Informatics by means of the permanent monitoring of the scientific state-of-the-art and existing activities in the area, prospective analysis of the emerging challenges and opportunities, and dissemination of the knowledge in the field by means of a collaborative effort of experts with complementary views and experiences. Biomedical Informatics deals with the integrative management and synergic exploitation of wide-ranging and inter-related scope of information generated and needed in healthcare settings, biomedical research institutions and health-related industry. INBIOMEDvision operational objectives:•To compile the existing knowledge on genotype and phenotype data resources providing an overview of methods and models that connect biological systems at the molecular level with the clinical physiopathology. •To consolidate a BMI community of researchers by congregating and promoting the interaction between them. •To develop and carry out activities to train new generations of scientists and professionals having a BMI perspective and skills.•To widely disseminate the BMI knowledge and resources.•To devise sustainability measures ensuring long-term maintenance of the INBIOMEDvision activities after the EU funds ending. INBIOMEDvision activities in development to achieve objectives:•Generation of a permanently updated and electronically accessible catalogue of initiatives and resources in the field. •Production of periodic state-of-the-art reviews on the matter. •Execution of prospective analyses on the theme. •Intensive organisation of community building activities (website, newsletter, think-tanks, scientific meetings and training activities).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z31'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z31</strong></td>
 </tr> <tr> 
  <td><b>Conveyor - A strong type workflow library</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Burkhard Linke <i>University of Bielefeld</i></td>
 </tr>
 <tr> 
  <td>  Alexander Goesmann (University of Bielefeld, CeBiTec); Robert Giegerich (University of Bielefeld, Pratical Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Workflows have become an important tool for data processing and data analysis in the field of Bioinformatics. Various tools like Taverna, Galaxy or Kepler offer workflow based processing, with a wide range of available functionality.<br>Conveyor is a new workflow engine with a number of unique features. In contrast to other workflow systems, it is built as a library, suitable for integration into other applications. This allows application designers to implement processing logic as workflows, making it easily extensible and maintainable.<br>The most important difference to other workflow systems is the type system Conveyor is built upon. Instead of relying on files with textual representation of data or simple primitive types, Conveyor uses a full featured object oriented data model, including inheritance, interfaces and generic types. This greatly improves type safety, allows easy extension and introduction of new types, and makes existing functionality reusable.<br><br>Conveyor's design allows developers to easily extend its functionality. A generic core library is separated from plugins providing application specific data types and processing. Implementing new functionality is just a matter of writing a new plugin.<br>Conveyor is developed as .NET application using Mono, a freely available .NET runtime for Unix systems. A large number of programming languages is available for .NET, simplifying integration of existing functionality or access to legacy applications. More information and the engine, as simple web service application and a designer for workflows are available at http://conveyor.CeBiTec.Uni-Bielefeld.DE</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z32'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z32</strong></td>
 </tr> <tr> 
  <td><b>New types of services in Mobyle 1.0</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Herve Menager <i>Institut Pasteur</i></td>
 </tr>
 <tr> 
  <td>  Vivek Gopalan (National Institute of Allergy and Infectious Diseases, National Institute of Health, Bioinformatics and Computational Biosciences Branch); Bertrand Neron (Institut Pasteur, Projets et Developpements en Bioinformatique); Sandrine Larroude (Institut Pasteur, Projets et Developpements en Bioinformatique); Pierre Tuffery (Université Paris Diderot / INSERM- MTi, RPBS); Yentram Huyen (National Institute of Allergy and Infectious Diseases, National Institute of Health, Bioinformatics and Computational Biosciences Branch); Bernard Caudron (Institut Pasteur, Projets et Developpements en Bioinformatique); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Performing bioinformatics analyses requires the selection and combination of tools and data to answer a given scientific question. Many bioinformatics applications are command-line only and researchers are often hesitant to use them based on installation issues and complex command requirements. Mobyle is a framework and web portal specifically aimed at the integration of bioinformatics software and databanks. It allows to run bioanalyses through a web interface without installing anything locally.<br><br>In addition to command-line tools, the latest release of Mobyle offers the possibility to execute predefined workflows. The data model has been extended to define a workflow as a dataflow-based coordination of programs that run successive and/or parrallel tasks to perform an analysis. Similarly to programs, workflows are viewed as services, sharing most of their description with programs, with the exception of the execution, which consists of a coordination of subtasks rather than the generation and execution of a command line.<br><br>Furthermore, we now offer a solution to overcome the limitations of results pre-visualization in the portal, which is not adapted to potentially large and/or complex text files. Viewers are a way to embed type-dependant visualization components for the data displayed in the Mobyle Portal. As opposed to programs and workflows, viewers are not executed on the server side, but rather rely entirely on browser-embedded code.<br><br>The Mobyle package is open-source and freely available at: http://projets.pasteur.fr/wiki/mobyle/download. More information about this project is available at http://projets.pasteur.fr/projects/show/mobyle.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z33'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z33</strong></td>
 </tr> <tr> 
  <td><b>An integrated core facility to go from biology to data and answers</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Laurent Gautier <i>Technical University of Denmark</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The DTU Multi-Assay Core (DMAC) is a core facility with a strong scientific focus. Unlike service-only facilities, the core is giving much importance to projects for which no clear methodology is established, either in the wet lab or on the data analysis and bioinformatics side, and is working in concert with investigators to support biology with cutting-edge or novel methodologies. We outline here how our model is working.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z34'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z34</strong></td>
 </tr> <tr> 
  <td><b>Toward Genomic Data Sharing Directly at the Lab Site in the New Agent-Based Infrastructure OpenKnowledge</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jonathan Magasin <i>University of California, Santa Cruz</i><br></td>
 </tr>
 <tr> 
  <td>Dietlind Gerloff (University of California, Santa Cruz) Adrian de Pinninck (Intelligent Pharma, -); Marco Schorlemmer (Artificial Intelligence Research Institute, Multiagent Systems); Dave Robertson (University of Edinburgh, School of Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Over the next decade functional genomics and metagenomics will produce increasingly massive data sets by the thousands. Such data volumes will preclude centralized, repository-based access to the raw data as the sole solution. The Cloud will help but storage and transfer costs make it likely that the producing laboratories will only upload selected data. The early-stage, peer-to-peer infrastructure OpenKnowledge (www.openk.org) could enable controlled access to data sets which might otherwise remain hidden, through searches directly at the lab sites. This is in addition to searching other data sets at centralized repositories, i.e. the currently prevailing access option is retained by the new infrastructure.  Data-sharing interactions in OpenKnowledge (OK) are implemented as simple query-answering protocols written in OK’s interaction language, LCC (Lightweight Coordination Calculus).  While LCC protocols and OK can also support more advanced, customized data analyses this poster focuses on raw data sharing using OK.  Two sharing scenarios have been implemented to date, for genomic and proteomic data.  Though the search and user interface software are customized for these scenarios, the underlying interaction protocols can be reused to share data of any type between labs. For example, sharing raw Sanger sequencing "trace" chromatograms using new software, SeqDoC+, helps investigate polymorphic sites in closely related bacterial strains.  In this new OK-demo we envision a researcher who searches a remote lab site for similar DNA sequences to his query sequence, and then compares the corresponding traces directly using an analysis pipeline based on the SeqDoC algorithm (research.imb.uq.edu.au/seqdoc/).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z35'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z35</strong></td>
 </tr> <tr> 
  <td><b>New tools for RNA 3D structure modeling: inspirations from the protein world</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Janusz Bujnicki <i>Intl. Institute of Molecular and Cell Biology (IIMCB)</i></td>
 </tr>
 <tr> 
  <td>  Kristian Rother (Adam Mickiewicz University, Faculty of Biology); Magdalena Rother (Adam Mickiewicz University, Faculty of Biology); Tomasz Puton (Adam Mickiewicz University, Faculty of Biology); Michal Boniecki (International Institute of Molecular and Cell Biology in Warsaw, Laboratory of Bioinformatics and Protein Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA is a large group of functionally important biomacromolecules. In striking analogy to proteins, the function of RNA depends on its structure and dynamics, which in turn is encoded in the linear sequence. However, while there are numerous methods for automated computational prediction of protein three-dimensional (3D) structure from sequence, with comparative modeling being the most reliable approach, there are very few such methods for RNA. We have developed ModeRNA, a software tool for comparative modeling of RNA 3D structures. As an input, ModeRNA requires a 3D structure of a template RNA molecule, and a sequence alignment between the target and the template. ModeRNA can model post-transcriptional modifications, a functionally important feature analogous to post-translational modifications in proteins. ModeRNA also offers a scripting interface that enables more complex manipulations, such as recombination of fragments taken from unrelated structures. Windows and UNIX implementations of ModeRNA with comprehensive documentation and a tutorial are freely available at http://iimcb.genesilico.pl/moderna/.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21054.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z36'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z36</strong></td>
 </tr> <tr> 
  <td><b>REPAIRtoire - a database of DNA repair pathways.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kaja Milanowska <i>University of Adam Mickiewicz</i></td>
 </tr>
 <tr> 
  <td>  Kaja Milanowska (University of Adam Mickiewicz, Institute of Molecular Biology and Biotechnology ); Joanna Krwawicz ( Institute of Biochemistry and Biophysics Polish Academy of Sciences, Department of Molecular Biology); Grzegorz Papaj (International Institute of Molecular and Cell Biology, Laboratory of Bioinformatics and Protein Engineering); Jan  Kosi?ski (International Institute of Molecular and Cell Biology, Laboratory of Bioinformatics and Protein Engineering); Katarzyna Poleszak (International Institute of Molecular and Cell Biology, Laboratory of Bioinformatics and Protein Engineering); Justyna Lesiak (International Institute of Molecular and Cell Biology, Laboratory of Bioinformatics and Protein Engineering); Ewelina  Osi?ska (International Institute of Molecular and Cell Biology, Laboratory of Bioinformatics and Protein Engineering); Kristian Rother (University of Adam Mickiewicz, Institute of Molecular Biology and Biotechnology); Janusz Bujnicki (International Institute of Molecular and Cell Biology, Laboratory of Bioinformatics and Protein Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: REPAIRtoire is the first database of DNA repair pathways. Although it is devoted to repair pathways it gathers information not only about reactions but also about enzymes that take part in DNA repair, their structures and genes, processed damages and diseases correlated with them. The database provides also links to other databases and literature information. The current list of <br>damages is comprehensive, while the dataset of pathways is limited to eight pathways from three model organisms: E.coli, S.cerevisiae and H.sapiens. Molecules and diseases from other organisms will be included in the near future. REPAIRtoire can be queried by the name of pathway, protein, enzymatic complex, RNA molecule, damage, reaction or disease and also using keywords. Options for data representation include graphs of pathways and tabular forms with enzymes and literature data. For enzymes with known structures, links are provided to the corresponding PDB entries. The contents of REPAIRtoire can be accessed at http://repairtoire.genesilico.pl/.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21058.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z37'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z37</strong></td>
 </tr> <tr> 
  <td><b>FABIA: Factor Analysis for Bicluster Acquisition</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Djork-Arné Clevert <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>  Ulrich Bodenhofer (Johannes Kepler University Linz, Institute of Bioinformatics); Martin Heusel (Johannes Kepler University Linz, Institute of Bioinformatics); Andreas Mayr (Johannes Kepler University Linz, Institute of Bioinformatics); Andreas Mitterecker (Johannes Kepler University Linz, Institute of Bioinformatics); Adetayo Kasim (Durham University, Wolfson Research Institute); Tatsiana Khamiakova (Hasselt University, Institute for Biostatistics and Statistical Bioinformatics); Suzy Van Sanden (  ) Dan Lin (Hasselt University, nstitute for Biostatistics and Statistical Bioinformatics); Willem Talloen (Johnson Pharmaceutical Research & Development, Division of Janssen Pharmaceutica); Luc Bijnens (Johnson Pharmaceutical Research & Development, Division of Janssen Pharmaceutica); Hinrich Göhlmann (Johnson Pharmaceutical Research & Development, Division of Janssen Pharmaceutica); Ziv Shkedy (Hasselt University, Institute for Biostatistics and Statistical Bioinformatics); Sepp Hochreiter (Johannes Kepler University Linz, Institute of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Biclustering is an unsupervised analysis technique, which is emerging as a standard tool for extracting knowledge from high-dimensional genomic data by clustering genes and samples simultaneously.<br>We present a novel generative approach for biclustering called "FABIA: Factor Analysis for Bicluster Acquisition". FABIA is based on a multiplicative model, which accounts for linear dependencies between gene expression and conditions, and also captures heavy-tailed distributions as observed in real-world transcriptomic data. The generative framework allows to utilize well-founded model selection methods and to apply Bayesian techniques.<br>In this talk, we will present:<br>a) a new generative model for biclustering which additionally allows to rank biclusters according to their information content;<br>b) a rigorous assessment with 11 competitors;<br>c) results from a high-throughput compound screening;<br>d) results of biclustering copy number variations (CNVs).</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21059.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z38'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z38</strong></td>
 </tr> <tr> 
  <td><b>Malaria Endemicity among Pregnant Women in Urban and Semi-Urban Area in Southwest, Nigeria</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tony Ojiezeh <i>Achievers University, Owo</i></td>
 </tr>
 <tr> 
  <td>  Ibeh Isaiah (University of Benin,, Microbiology Department); Opedun Doyin (LUTH, Department of Microbiology and Parasitology); Udoh Jack. S (OAUTH, Department of Microbiology and Parasitology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Abstract: The comparative study on malaria endemicity among pregnant women of various age groups at two different locations (urban and semi urban) in Southwest of Nigeria was carried out in antenatal clinics at LUTH(Idi - Araba) and OAUTH (Ile - Ife) to determine prevalence rate and areas of higher endemicity. Both thick and thin blood films were made and stained using parasitological standard procedures. A face-to-face questionnaire relating to their health status was distributed and completed before being recruited into the study. 178 of 250 women examined had significant malaria parasites in their blood films and the frequency was higher in Idi -Araba which recorded 93 (52.2 %) with mean parasite density of 700 dl against Ile-Ife that had 85 (47.7 %) withmean parasite density of 640 dl for Ile - Ife. The percentage infection rate was the same in both locations at 41.2 %, but at different age groups: in LUTH the prevalence was higher among age group of 26-30 years while in OAUTH, it was higher among age group of 40 years and above. Those that lived under bad environmental conditions had the highest mean parasite density of 1200 dlG1 with prevalent rate of 81.3 %. The use of mosquito proof nets was also looked into in which those that did not use mosquito net had the highest mean parasite density of 740 dl, with infection rate of 68.8 %. However, the prevalence of malaria parasite in these two geographic locations is statistically significant at P < 0.025.Key words: Malaria infection % Endemicity % Pregnant women % Parasite density</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21041.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z39'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z39</strong></td>
 </tr> <tr> 
  <td><b>Strengths and limitations of the federal guidance on synthetic DNA</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jean Peccoud <i>Virginia Tech</i></td>
 </tr>
 <tr> 
  <td>  Laura  Adam (Virginia Tech, Bioinformatics); Michael Kozar (Virginia Tech, Bioinformatics); Gaelle Letort (Virginia Tech, Bioinformatics); Olivier Mirat (Virginia Tech, Bioinformatics); Arunima Srivastava, (Virginia Tech, Bioinformatics); Tyler Steward (Virginia Tech, Bioinformatics); Mandy Wilson (Virginia Tech, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: An implementation of the sequence screening method recommended by the U.S. Government to prevent the misuse of gene synthesis highlights improvements over the protocols proposed by the industry. Since it does not rely on a database of curated sequences, its deployment is fast and inexpensive. Without resulting in an unacceptable computational cost, breaking sequences into 200 bp fragments translated in six frames precludes the hiding of sequences of concern within longer, benign sequences. A standardized dictionary of keywords used to interpret alignment results and a realistic suite of annotated test sequences are still needed to assess the performance of the screen software implementations. Beyond its biosecurity application, this screening algorithm can be used to enforce other policies and regulations affecting the biotechnology industry. It is also likely to find a variety of other applications such as partitioning sequencing reads by species in metagenomic samples, forensic, or for clinical diagnostic.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21082.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Z40'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z40</strong></td>
 </tr> <tr> 
  <td><b>Screening a repressor library to build orthogonal circuits</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Brynne Stanton <i>University of California San Francisco</i></td>
 </tr>
 <tr> 
  <td>  Christopher Voigt (University of California San Francisco, Pharmaceutical Chemistry); Alvin Tamsir (University of California San Francisco, Pharmaceutical Chemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genetic circuits are often constructed using prokaryotic repressor proteins. Currently, only a few well-characterized repressors are commonly implemented within circuit designs, which severely limits the number and complexity of programs that result. To expand the toolbox of programmable orthoganol operator-repressor pairs, we are characterizing a library of repressor proteins belonging to the TetR family. Collectively, the repressors originate from 45 distinct prokaryotic organisms and were synthesized by GENEART. Using protein-binding arrays in conjunction with in vivo reporter assays, we are determining the sequences bound by each purified repressor protein, building reporters, and screening the orthogonality of each newly determined binding sequence against our library in vivo. Initial results demonstrate that repression is highly specific for the properly matched repressor, and that our library exhibits orthogonal behavior. Complete characterization of the transcriptional repressor library will drastically increase the number of well-characterized parts available for use in genetic programs, and will thereby enhance the complexity of circuits that may be constructed.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z41'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z41</strong></td>
 </tr> <tr> 
  <td><b>Extraction protocol optimization for the LC-MS based global metabolite profiling of grapes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Matthias Scholz <i>Edmund Mach Foundation, Research and Innovation Center</i></td>
 </tr>
 <tr> 
  <td>  Georgios Theodoridis (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Helen Gika (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Pietro Franceschi (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Lorenzo Caputi (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Panagiotis Arapitsas (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Domenico Masuero (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Ron Wehrens (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Urska Vrhovsek (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); Fulvio Mattivi (Edmund Mach Foundation, Research and Innovation Center, Food Quality and Nutrition Department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Optimal solvent conditions for grape sample preparation were investigated for the purpose of metabolite profiling studies, with the aim of obtaining as many features as possible with the best analytical repeatability. Mixtures of water, methanol and chloroform in different combinations were studied as solvents for the extraction of ground grapes. The experimental design used a two stage study to find the optimum extraction medium. The extracts obtained were further purified using solid phase extraction and analysed using a UPLC full scan TOF MS with both reversed phase and hydrophilic interaction chromatography. The data obtained were processed using data extraction algorithms and advanced statistical software for data mining. The results obtained indicated that a fairly broad optimal area for solvent composition could be identified, containing approximately equal amounts of methanol and chloroform and up to 20% water. Since the water content of the samples was variable, the robustness of the optimal conditions suggests these are appropriate for large scale profiling studies for characterisation of the grape metabolome.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z42'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z42</strong></td>
 </tr> <tr> 
  <td><b>Protein-protein docking with random partners: implications for binding region prediction.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Juliette Martin <i>Centre National de la Recherche Scientifique</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: When one attempts to dock a protein with a non-native partner -here called "random partner"-, the residues of the experimental binding site tend to be part of the binding site generated by docking.  This fact has been observed on data sets of 3, 6 and 56 protein complexes (Fernandez-Recio J et al, 2004, Sacquin-Mora S et al, 2008, Wass M N et al,2011). In this work, I explore this property in further details.I consider a non-redundant set of 198 protein chains with structures available in bound and unbound forms, and the Nh3D data set as source of random partners. Thanks to the docking software hex, running on GPU, which can dock two proteins in less than a minute, I performed a great number of aspecific docking experiments in order to understand the process of aspecific docking and the nature of binding sites identified in this way.I address the following points. Does aspecific docking with several random partners converge toward specific regions of the surface of the target protein ? Does it depend on the size of the partners ? Is it sensitive to conformational change (i.e. bound versus unbound structures) ? Can we relate aspecific docking observations with binding affinity data ? Does aspecific docking actually help to detect binding sites and is it applicable in practice ?</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z43'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z43</strong></td>
 </tr> <tr> 
  <td><b>Jalview 2.7</b><br/><br/></td>
 </tr>
 <tr> 
 <td>James Procter <i>University of Dundee</i></td>
 </tr>
 <tr> 
  <td>  Petr Troshin (University of Dundee, College of Life Sciences); David Martin (University of Dundee, College of Life Sciences); Geoffrey Barton (University of Dundee, College of Life Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Jalview 2.7 is the latest release of this widely used multiple sequence alignment, analysis, annotation and visualisation system. The system consists of a web-based applet, and a desktop application capable of retrieving data from online databases and distributed annotation system (DAS) servers, and accessing web services for multiple alignment, analysis and protein secondary structure prediction. Recent developments include improved support for visualising linked structure superpositions based on domain and protein interface alignments, RNA secondary structure display, and extended web based alignment visualisation functionality. We also describe our ongoing outreach program, including user training courses and participation in the Google Summer of Code. Finally, we introduce some future goals, including current progress towards Jalview version 3.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z44'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z44</strong></td>
 </tr> <tr> 
  <td><b>Protein structure prediction aided by consensus-derived restraints.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marcin Skwark <i>Stockholm University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Predicting 3D protein structures is an important task of contemporary<br>computational biology, due to the high cost and low throughput of experimental<br>methods. Recent CASP (Moult, 2009) experiments have proven that consensus-based approaches (utilizing diverse prediction methods and selecting the most suitable model) are superior to the single-approach methods. Though, pure consensus method is only able to perform as well as the best of its compound methods i.e. it is not able to devise a model better than any of the ones obtained from the input predictors. Taking into account the amount of information contained in the input models, it should be possible to create a method, that would not only select the best model, but rather identify the prevalent structural features and produce a superior model basing on them.<br><br>This work proposes that quality of structure predictions can be improved by using information derived from the individual compound methods and presents the method of deriving and applying<br>such constraints. PconsD, a method based on the presented assumptions, consistently outperforms<br>other methods operating on the same model ensemble, both in terms of model completeness and stereochemical feasibilty. This approach is particularly useful for multidomain proteins and proteins for which no homologous protein of known structure can be easily identified.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z45'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z45</strong></td>
 </tr> <tr> 
  <td><b>Hierarchichal Network Model for unsupervised segmentation of Cryo-EM Density Maps</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Virginia Burger <i>University of Pittsburgh School of Medicine</i><br></td>
 </tr>
 <tr> 
  <td>Chakra Chennubhotla (University of Pittsburgh School of Medicine) Ivet Bahar (University of Pittsburgh , Computational and Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Challenge: Cryogenic Electron Microscopy (Cryo-EM) experiments yield low resolution (3-30A) 3D-density maps of proteins, viruses, and macromolecules. These density maps are segmented to identify subregions of structurally distinct proteins or protein domains, useful for inferring protein motions and functional properties, as well as for fitting available high-resolution atomistic structures (NMR or X-ray).  EM-map segmentation has traditionally required tedious and subjective manual partitioning or supervised computational methods, while validation of resulting segmentations has remained an open problem in this field.<br>Approach: We propose an unsupervised hierarchical network model for segmenting EM-density maps into functionally-distinct subregions. To our knowledge, this is the first input-parameter-free EM-map segmentation method.  Our soft-partitioning approach models EM-maps as graphs, where graph nodes correspond to map voxels and edges express distances and intensities. These graphs are described by Markov transition matrices. At each hierarchy level, distant node connections are revealed by diffusing the Markov matrix, and a coarsened subset of nodes is selected according to node connectivity and map symmetry to carry on to the next hierarchy level. To validate resulting segmentations, we have rendered a set of synthetic ground-truth map segmentations by smoothing atomistic structures with respect to map resolution and available B-factors.  Segmented experimental maps are aligned with these corresponding synthetic maps and compared by established shape-matching metrics. These ground-truth maps can be used for scoring EM-map segmentations in benchmark comparisons across the Cryo-EM modeling community. <br>Impact: Our unsupervised hierarchical network model for EM-map segmentation efficiently and unbiasedly partitions Cryo-EM maps into functionally-relevant subregions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z46'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z46</strong></td>
 </tr> <tr> 
  <td><b>Integrative analysis of gene coexpression modules in glioma based on WGCNA algorithm</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Alexander Ivliev <i>Lomonosov Moscow State University</i></td>
 </tr>
 <tr> 
  <td>  Peter-Bram 't Hoen (Leiden University Medical Center, Center for Human and Clinical Genetics); Marina Sergeeva (Lomonosov Moscow State University, A.N. Belozersky Institute of Physico-Chemical Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Weighted gene coexpression networks analysis (WGCNA) represents an effective tool for exploring transcriptome structure by identification of gene coexpression modules that are highly robust to noise [1]. For computational reasons WGCNA is typically used to construct networks of a limited size, while not covering the entire human genome. Here we report a genome-scale analysis of gene coexpression relationships in glioma based on a heuristic approach that extends gene composition of WGCNA modules from a seed set of genes up to the genome scale. The analysis was performed on the largest open-access glioma dataset (GSE16011) and included module validation in 4 independent reference datasets (Affymetrix microarrays, a total of 790 glioma samples). We determined a genome-scale gene composition for 20 coexpression modules that were shared by the 5 datasets and associated with proliferation, angiogenesis, hypoxia, immune response, genomic alterations, and other features of glioma. Expression profiles of specific modules were found correlated with each other, thus forming a higher-order coexpression structure of the transcriptome. Two novel coexpression modules were detected that identified a novel “proastrocytic” group of favorable prognosis markers and suggested that EGF signaling in glioma may be a subject to regulation by proteins of the Sprouty family. This study highlights importance of performing gene coexpression analysis at a genome scale and provides insight into the structure of the glioma transcriptome [2]. This work is partially supported by grant RFBR 10-04-01385-?.<br><br>[1] Langfelder P et al, BMC Bioinformatics, 2008. [2] Ivliev A et al, Cancer Res, 2010.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z47'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z47</strong></td>
 </tr> <tr> 
  <td><b>Web-based management and visualisation of heterogenous experimental data using SORF</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Fady Mohareb <i>Cranfield University</i></td>
 </tr>
 <tr> 
  <td>  Conrad Bessant (Cranfield University, Bioinformatics Group - Cranfield Health); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster presents the Symbiosis Online Research Framework (SORF) - a web application for data management, exchange, visualisation and analysis. This framework supports a wide variety of experimental platforms thanks to its generic entity-attribute-value (EAV) database backend. The generic nature of the EAV allows the system to accommodate heterogeneous data types without the need to provide separate storage modules for every experimental platform applied. The SORF makes data management straightforward despite the complex relationships typically found in systems biology studies – one experiment can include more than one dataset, and individual samples may be represented in many different datasets. The recently developed SORF front end uses the latest web technologies to allow users to upload, query, browse, and visualise data through a series of data views, exploratory data analysis tools and relational visualisations. SORF has been developed as a central data repository for the Symbiosis-EU project, which aims to find new ways to assess freshness in meat products. Symbiosis-EU serves as a perfect case study for SORF as it involves the application of various metabolic and proteomic techniques, such as HPLC, FTIR, machine olfaction, microarrays and mass spectrometry. In addition, all acquired analytical data is accompanied by experimental metadata and results from traditional microbiological methods such as bacterial counts and sensory scores. An implementation of SORF is available online via the project web site (www.symbiosis-eu.net).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z48'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z48</strong></td>
 </tr> <tr> 
  <td><b>Identification of the transcriptional gene network in 2D and 3D human normal bronchial epithelial cells culture systems</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Vincenzo Belcastro <i>Philip Morris International R&D</i></td>
 </tr>
 <tr> 
  <td>  Carole Mathis (Philip Morris International R&D, BSR); Carine Poussin (Philip Morris International R&D, BSR); Dirk Weisensee (Philip Morris International R&D, GmbH); Julia Hoeng (Philip Morris International R&D, BSR); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Microarray gene expression technology has become standard practice in systems biology to analyze transcriptomic changes at the cell, tissue, and organism level, enabling the reverse-engineering of condition-specific gene networks.<br>To better understand how transcriptional gene networks of biologically close cellular systems, differ, two specific networks were reverse-engineered from 2D and 3D human normal bronchial epithelial cell cultures.<br>100 gene expression profiles for both Normal Human Bronchial Epithelial (NHBE) cells and organotypically differentiated bronchial cells (AIR100 culture) were collected under normal culture conditions. Gene networks were reconstructed by correlating gene expression profiles at a False Discovery Rate (FDR, t-statistic and Benjamini-Hochberg correction) < 5E-6. A message-passing-based algorithm published by BJ Frey in Science 2007 was applied to discover the modularity within the network topologies.<br>84% of genes were commonly expressed in NHBE cells and AIR100 culture. Genes were grouped in 262 communities in NHBE cells and 249 in AIR100 culture (FDR<0.01) with 50% of conservation between the two biological systems (FDR<0.01), representing normal cell physiology (e.g., translational elongation, mitochondrial functions). The transcriptional specificity of AIR100 culture resides in the genes associated with “immune response” (FDR<8.7E-4), which may be attributed to the maturity of the 3D tissue-like culture.<br>By using this approach, two reference in silico models were created, allowing the identification of the transcriptional similarities and specificities of the two cellular systems analyzed. These networks represent a resource to evaluate the response of differential perturbations collected in NHBE cells or AIR100 cultures in in vitro assays.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z49'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z49</strong></td>
 </tr> <tr> 
  <td><b>Structure-from-FRET: Statistical inference of structural dynamics from FRET data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrej Savol <i>University of Pittsburgh School of Medicine</i><br></td>
 </tr>
 <tr> 
  <td>Chakra Chennubhotla (University of Pittsburgh School of Medicine) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Challenge: Experimental and computational techniques make opposite compromises in trying to reveal the mechanistic underpinnings of molecular movements. In particular, atomistically accurate simulations suffer from temporal sampling deficits, whereas single-molecule methods necessarily sacrifice spatial detail. For example, single-molecule FRET (smFRET) reveals the one-dimensional instantaneous distance between only two residues within a protein over time. Here we suggest an approach, structure-from-FRET (SFF), for merging representative computational and experimental techniques: MD and smFRET. SFF addresses the non-trivial (ill-posed) problem of determining backbone-resolution three-dimensional protein motions from one-dimensional inter-residue distance constraints.  <br><br>Approach: We perform extensive MD simulations to first construct an ensemble of representative conformations for a protein also studied by smFRET. For each MD conformer, we compute the inter-residue distance analogous to the experimentally-determined distance. Following dimensionality reduction, we model the distance distributions as mixtures of Gaussians within a Bayesian framework. We condition these distributions on the experimentally determined distance, which enables subsequent sampling from the posterior distribution. Each vector so sampled, when mapped to full conformational space, constitutes a synthesized conformer in agreement with a smFRET data point. Performed sequentially, a three-dimensional protein trajectory explaining the smFRET trace is generated. <br><br>Impact: We apply SFF to the model hinge protein adenylate kinase to produce millisecond-resolution trajectories representing extended temporal (seconds) and conformational (tens of Angstroms) sampling without perturbations between reference states. In bridging atomistic detail with experimental timescales, our approach constitutes both a stepping-stone toward a predictive model of protein dynamics and a visualization tool for single-molecule biophysicists.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z50'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z50</strong></td>
 </tr> <tr> 
  <td><b>Identification of binding pockets in protein structures using a knowledge-based potential derived from local structural similarities</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pier Federico Gherardini <i>University of Rome Tor Vergata</i><br></td>
 </tr>
 <tr> 
  <td>Manuela Helmer-Citterich (University of Rome Tor Vergata) Valerio Bianchi (University of Rome Tor Vergata, Biology); Gabriele Ausiello (University of Rome Tor Vergata, Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We have developed PDBinder, a novel method for the prediction of ligand binding sites in protein structures. Our approach is based on the observation that unrelated binding sites share small structural motifs that bind the same chemical fragments irrespective of the nature of the ligand molecule as whole. PDBinder compares a query protein against a library of binding and non-binding protein surface regions derived from the PDB. The results of the comparison are used to derive a propensity value for each residue which is correlated with the likelihood that the residue is part of a ligand binding site.<br>PDBinder has been trained on a non-redundant set of 1356 high-quality protein-ligand complexes and tested on a set of 267 holo and apo complex pairs. We obtained an MCC of 0.327 on the holo set with a PPV of 0.436 while on the apo set we achieved an MCC of 0.289 and a PPV of 0.402. The good performance of PDBinder on the unbound proteins is extremely important for real-world applications where the location of the binding site is unknown. <br>Finally we show that the performance of PDBinder is superior to that of other methods both in the prediction of specifc binding residues and in the identification of which cavity in the structure is most likely to host a ligand binding site. Moreover, since our innovative approach is orthogonal to those used in existing methods, the propensity value assigned by PDBinder can be integrated in other algorithms futher increasing the final performance.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z51'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z51</strong></td>
 </tr> <tr> 
  <td><b>Integration of Standardized Cloning Methodologies and Sequence Handling to Support Synthetic Biology Studies</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kevin Clancy <i>Life Technologies</i></td>
 </tr>
 <tr> 
  <td>  Maurice Ling (Life Technologies, Synthetic Biology); Angela Jean (Life Technologies, Synthetic Biology); Dunqiang Liao (Life Technologies, Synthetic Biology); Kok Hien Gan (Life Technologies, Synthetic Biology); Kin Chung Sam (Life Technologies, Synthetic Biology); YongMing Chen (Life Technologies, Synthetic Biology); Lian Seng Loh (Life Technologies, Synthetic Biology); Stephen Cheng (Life Technologies, Synthetic Biology); Todd Peterson (Life Technologies, Synthetic Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The assembly and downstream transformation of genetic constructs has been a fundamental scientific technology for the last thirty years. Synthetic biology is an engineering based approach to molecular biology as emphasizing the standardized assembly of characterized DNA fragments. The standards promoted by the BioBricks™ Foundation have enabled novel constructs to be developed based upon the expected function of these constructs. However scientists need a software environment that enables them to curate large collections of parts and assemblies, combined with appropriate tools to facilitate quick creation of constructs and identification of potential design issues in silico.  In this paper, we present the implementation of BioBrick™® and GENEART® Assembly tools as a means to develop collections of parts and deveices. We then developed the necessary means to both automate import of parts collections and means of managing and developing such collections in an ongoing basis. We used this tool to both simulate and guide the design and development of novel clones ranging froma few kilobases up to 120 kb. Integration of these tools and data is a step towards implementation of BioCAD™, a computer based design approach to facilitate development of complex circuit based perturbation of cellular systems.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Z52'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Z52</strong></td>
 </tr> <tr> 
  <td><b>The reference species tree of (sequenced) life</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Julian Gough <i>University Of Bristol</i></td>
 </tr>
 <tr> 
  <td>  Ralph Pethica (University of Bristol, Computer Science); Matt Oates (University of Bristol, Computer Science); Hai Fang (University of Bristol, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Essential to our understanding of life, evolution and the diversity of biological organisms is the underlying Darwinian species tree. There are now about 1,500 completely sequenced genomes, and the impending impact of next generation sequencing is yet to be felt. The challenging side to this anticipated growth is to keep up with the cataloging, indexing and analysis of the genomes. We present a high quality reference species tree of all completely sequenced organisms, which will grow continually as new genomes become available.<br><br>The underlying data from which the tree is constructed are the domain architectures from the SUPERFAMILY database. Based on protein structure, these contain information on relationships over far greater evolutionary distances than can be detected by BLAST or traditional phylogenetics of multiple sequence alignments. Furthermore the single reference tree combines the information from every sequence in every genome. A maximum likelihood algorithm (RAxML) using the Gamma model of rate heterogeneity is used. Expert knowledge from the literature is added via constraint to the NCBI taxonomy, reducing the computational complexity and further improving the RAxML contribution.<br><br>Such a tree of sequenced genomes, not previously available, is crucial to any genomic study reliant on evolutionary context. To assist such studies, in addition to the tree itself, we provide analysis and sophisticated visualisation for the tree. E.g. a browsable, annotated tree can be automatically drawn showing the evolution of a given GO term throughout evolution. We ourselves have analysed the evolution of the calcium signalling toolkit in detail.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




