
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=Q by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlistedce.html?cat=Q by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:18 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'Q'- Population Genetics and Variation'</b></td>
 </tr>
</table>

<a name='Q01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q01</strong></td>
 </tr> <tr> 
  <td><b>Signatures of positive selection on genes of the N-Glycosylation pathway concentrate on genes with regulatory positions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Giovanni Marco Dall'Olio <i>Pompeu Fabra University (UPF) - IBE-CEXS</i></td>
 </tr>
 <tr> 
  <td>  Ludovica Montanucci (Pompeu Fabra University (UPF), IBE-CEXS); Pierre Luisi (Pompeu Fabra University (UPF), IBE-CEXS); Jaume Bertranpetit (Pompeu Fabra University (UPF), IBE-CEXS); Hafid Laayouni (Pompeu Fabra University (UPF), IBE-CEXS); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Asparagine N-Glycosylation is one of the most important forms of protein modification in eukaryotes. Estimates on the UniProt database predict that almost 50% of the transmembrane proteins known are potentially glycosylated. This form of modification is required in order to achieve proper folding for the proteins in the secretory pathway, and the most complex forms of glycosylation are involved in cell-to-cell interactions, innate immunity and development.<br>Here we present an analysis of the signatures of positive selection that affected the genes of the N-Glycosylation pathway among human populations. While the earlier part of the pathway is evolutionary conserved among among all eukaryotes, the latter part is complex and variable among organisms; therefore, studying signatures of positive selection on these two parts allows to compare genes belonging to the same pathway but exposed to different functional constraints. Results show a signature of positive selection in the GCS1 (MOGS) and the ALG12 gene in European populations and in MGAT3 in Eastern Asian populations; all these events occurred in genes, that given their position in the pathway, can be expected to be exposed to strong functional constraints.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q02</strong></td>
 </tr> <tr> 
  <td><b>Detecting imbalanced allele pair frequencies (ImAP) in family genotype data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marit Ackermann <i>TU Dresden</i></td>
 </tr>
 <tr> 
  <td>  Andreas Beyer (TU Dresden, Biotechnology Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 108. Epistatic genetic interactions are key for understanding the genetic contribution to complex traits. Epistasis is always defined with respect to some trait such as growth rate or fitness. Whereas most existing epistasis screens explicitly test for some trait, it is also possible to implicitly test for fitness traits by searching for the over- or under-representation of allele pairs in a given population. Such analysis of imbalanced allele pair frequencies of distant loci has not been exploited yet on a genome-wide scale mostly due to statistical difficulties such as the multiple testing problem.<br>We propose a new approach called ImAP for inferring epistatic interactions that is exclusively based on DNA sequence information. Our approach is based on genome-wide SNP data sampled from a population with known family structure. We make use of genotype information of parent-child trios and inspect 3 x 3 contingency tables for detecting pairs of alleles from different genomic positions that are over- or underrepresented in the population. We also developed a simulation setup which mimics the pedigree structure by simultaneously assuming independence of the markers.<br>When applied to mouse SNP data, our method detected 566 imbalanced allele pairs, which is substantially more than what we got in simulations assuming no interactions. We could validate a significant number of the interactions with external data and we found that interacting loci are enriched for genes involved in developmental processes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q03</strong></td>
 </tr> <tr> 
  <td><b>cn.MOPS: Mixture of Poissons for Discovering Copy Number Variations in Next Generation Sequencing Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Günter Klambauer <i>Johannes Kepler University Linz</i></td>
 </tr>
 <tr> 
  <td>  Sepp Hochreiter (Johannes Kepler University Linz, Institute of Bioinformatics); Karin  Schwarzbauer (Johannes Kepler University Linz, Institute of Bioinformatics); Andreas Mayr (Johannes Kepler University Linz, Institute of Bioinformatics); Djork-Arné  Clevert (Johannes Kepler University Linz, Institute of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Next generation sequencing (NGS) is now one of the key technologies in molecular biology for genotyping and genome assembly.  Still challenging is NGS' application to quantitative analysis like the detection of copy number variations (CNVs) in a cohort of DNA samples. Current methods detect CNV regions by a variation of read densities along chromosomes.  However they possess a high false discovery rate (FDR), because of read count variations along chromosomes,  even aften correcting for read biases like the GC bias. Furthermore most of these methods do not provide integer copy number estimates.<br><br>We propose "Copy Number estimation by a Mixture Of PoissonS" (cn.MOPS) for  CNV detection with a low FDR and integer copy number estimation. Since cn.MOPS constructs for each genomic position a model across samples, it is not  affected by read count variations along chromosomes. cn.MOPS decomposes read variations across samples into integer copy numbers and noise by its mixture components and Poisson distributions, respectively. Model selection in a Bayesian framework  is based on maximizing the posterior by an expectation maximization algorithm. The model incorporates the linear dependency between copy number and average read count.  Most importantly, a Dirichlet prior on the mixture components prefers constant copy number two for all samples and  thereby controls the FDR in detecting CNVs.<br><br>We construct artificial and real world benchmarking data sets for CNV detection methods. Cn.mops  significantly outperforms previously published methods with respect to precision/recall on benchmark data  and to the concordance with published copy number tables based on microarray data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q04</strong></td>
 </tr> <tr> 
  <td><b>Deep-sequencing reveals phage-driven diversification of Pseudomonas aerugionsa biofilms</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kerensa McElroy <i>University of New South Wales</i></td>
 </tr>
 <tr> 
  <td>  Fabio Luciani (University of New South Wales, Infection and Inflammation Research Centre); Janice Hu (University of New South Wales, Centre for Marine Bioinnovation); Jerry Woo (University of New South Wales, Centre for Marine Bioinnovation); Staffan Kjelleberg (University of New South Wales, Centre for Marine Bioinnovation); Scott Rice (University of New South Wales, Centre for Marine Bioinnovation); Torsten Thomas (University of New South Wales, Centre for Marine Bioinnovation); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Pseudomonas aeruginosa infection is the leading cause of death for Cystic Fibrosis patients. Antibiotic resistance is rife, possibly due to high colonising population diversity. Our lab has replicated phenotypic diversification in P. aeruginosa PAO1 biofilm models of lung infection. To reveal underlying genetic variants, we deep-sequenced samples from a PAO1 biofilm, after four and 11 days of growth. All bioinformatic methods, including variant detection and haplotype reconstruction, were validated on simulated data created with GemSIM. <br>Surprisingly, the PAO1 genome contained only two SNPs, with frequencies around 10%. Both were within a large hypothetical outer-membrane protein postulated to be involved in biofilm formation and antibiotic resistance. Both SNPs were silent; however converted rare codons to more common ones, which likely increases gene expression levels. These SNPs may reflect early biofilm lifestyle adaptation. <br>In contrast to the negligible genetic diversity of PAO1, the genome of its associated bacteriophage Pf4 revealed ongoing diversification, characterised by an increase in Shannon’s entropy between days four and 11 and an explosion in phage population size. All phage SNPs were within or upstream from the putative Repressor C gene. This gene is implicated in Pf4 superinfectivity, which results in loss of host resistance and conversion from a lysogenic form to a lethal, lytic lifecyle. In total, nine Pf4 haplotypes emerged by day 11, with frequencies from 1% to 44%, while the original haplotype dropped to 9%. These results suggest that variation of the phage genome, and specifically the emergence of superinfective haplotypes, drives diversification within PAO1 biofilms.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q05</strong></td>
 </tr> <tr> 
  <td><b>Computational methods as first-pass filter for missense mutations in ATM gene- A road towards pharmacogenomic approach</b><br/><br/></td>
 </tr>
 <tr> 
 <td>George Priya Doss C <i>VIT University</i></td>
 </tr>
 <tr> 
  <td>  Ujjwal Shah (VIT University, School of Biosciences and Technology); Srajan Jain (VIT University, School of Biosciences and Technology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: With the recent availability of the complete genome sequence and the accumulation of variation data, determining the effects of missense variants will be the next challenge in mutation research. In addition to the molecular approaches, which are laborious and time-consuming, it is now possible to apply computational approaches to filter out mutations that are unlikely to affect protein function. Alternatively, bioinformatics approaches, based on the biochemical severity of the amino acid substitution, and the protein sequence and structural information, can offer a more feasible means for phenotype prediction. Deleterious missense mutations of ATM gene are accountable for various forms of cancer associated disease. Yet, distinguishing deleterious mutations of ATM from the massive number of non-functional variants that occur within a single genome is a considerable challenge. In this approach, we present the use of computational methods to explore the mutation-structure-function relationship. In other respects, we attempted these methods to work as first-pass filter to identify the missense mutations worth pursuing for further experimental research. This review surveys and compares variation databases and in silico prediction programs that assess the effects of deleterious functional variants on protein functions. We also introduce a combinatorial approach that uses machine learning algorithms to improve prediction performance.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q06</strong></td>
 </tr> <tr> 
  <td><b>Inference of Kinship Coefficient from Korean SNP genotyping data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Seong-Jin Park <i>Korea Research Institute of Bioscience & Biotechnology</i></td>
 </tr>
 <tr> 
  <td>  Jin ok  Yang (Korea Research Institute of Bioscience & Biotechnology, Korean Bioinformation Center); Sanghyuk  Lee (Korea Research Institute of Bioscience & Biotechnology, Korean Bioinformation Center); Byungwook Lee (Korea Research Institute of Bioscience & Biotechnology, Korean Bioinformation Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genome-wide association studies (GWAS) have been widely used to identify common variants that contribute to variation in complex human phenotypes and diseases. Identification of kinship coefficient in the family pedigree and relatedness between individuals in a family are crucial in GWAS of common complex diseases. We present a method to infer close inter-familial relationship from SNP genotyping data and provide the reference standards of 1st ~ 6th degree of kinship in Korean families. Blood samples were obtained from 43 pure Korean individuals in two families. SNP information was obtained by using the manufacturer's instructions for the Affymetrix Genome-wide Human SNP array 6.0 and Illumina Human 1M-Duo chip. To measure kinship coefficient with SNP genotyping data, we considered all the possible pairs of individuals in each family. Genetic distance between two individuals in a pair was constructed by using the allele sharing distance method. We assigned these genetic distance values to the standard of the 1st ~ 6th degree of kinship. We have proposed a method to infer close relationship between two individuals using high-density genotyping data and developed the reference standards of degree of kinship in Korean family. Our approach is the first attempt to identify the genetic distance of very close individuals related by blood. The kinship coefficient can be used to verify relationships, to reconstruct pedigree, to detect pedigree errors, to analyze forensic DNA data, and to indentify unknown relationships in the family members.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q07</strong></td>
 </tr> <tr> 
  <td><b>INRICH: improved enrichment testing method that integrates an unbiased GWAS strategy with prior biological knowledge: application to Type 2 diabetes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Phil Lee <i>Massachusetts General Hospital</i></td>
 </tr>
 <tr> 
  <td>  Colm O'Dushlaine (MGH, Center for Human Genetics); Shaun Purcell (MGH, Center for Human Genetics Research); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Over the past years, pathway analysis has been widely employed in genome-wide association studies (GWAS), providing new insights into the genetic basis of various complex disorders.  However, several limitations still remain in currently available methods [1]. Here we describe INRICH, a new pathway analysis method for GWAS that tests for the enrichment signals of predefined gene sets within fixed genomic intervals. INRICH consists of three major steps: i) LD-based interval data generation, where unique regions of association are identified; ii) empirical enrichment calculation using an interval-based permutation strategy; and iii) bootstrap-based multiple testing correction at the pathway level. Extensive simulation studies were conducted using Hapmap III CEU data and Gene Ontology (GO), and the analysis results confirm less than 1% Type I error rates. We also applied INRICH to SNPs that showed genome-wide significance in recent Type II diabetes (T2D) GWAS [2]. Consistent with established knowledge [3], T2D-associated intervals were most significantly enriched for genes involved in glucose homeostasis (GO:0042593; p=0.0014). A number of additional pathways of interest (e.g. insulin secretion), that did not survive our stringent corrections, have also been implicated in the pathogenesis of T2D. INRICH represents a significant step forward for pathway-based GWAS mining in terms of wide applicability, fast running time, and its robustness to potential biases (such as gene size and distribution) and other confounding factors.<br><br>References<br>1.	Wang et al. 2010 Nature Reviews Genet 11:843-852<br>2.	Hindorff et al. 2009 Proc Natl Acad Sci USA 106(23):9362-9367<br>3.	Dupuis et al. 2010 Nature Genet 42:105-116</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q08</strong></td>
 </tr> <tr> 
  <td><b>A Knowledge-based Approach to Genome-Wide Association Studies for Complex Diseases</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Alireza Nazarian <i>University of Florida</i></td>
 </tr>
 <tr> 
  <td>  Heike Sichtig (University of Florida, Genetics Institute and Department of Molecular Genetics and Microbiology); Alberto Riva (University of Florida, Genetics Institute and Department of Molecular Genetics and Microbiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genome-Wide Association Studies (GWAS) have become the method of choice for large scale analysis of genotype-phenotype relationships. Despite the continuous advances of genotyping technology, GWAS have so far shown limited success in comprehensively revealing the genetic architecture of complex diseases. This can be explained by their inability to reliably detect small risk contributions, possibly distributed over a large number of genetic factors.<br>In this study we propose a hypothesis-based approach to perform genome-wide association analysis <br>for complex diseases. Instead of examining each marker (typically SNPs) individually, our method allows the user to define models consisting of a set of markers, to combine their genotypes into a single variable, and to evaluate the model’s ability to correctly classify cases and controls. Hypothesis testing and refinement are performed using a Genetic Algorithm (GA). The method was successfully tested on case-control genotype data provided by the WTCCC containing two healthy control groups, and seven disease groups. As an example, in one test the GA was able to find a combination of ten SNPs (out of 1140 SNPs belonging to genes in the “Insulin Signaling” pathway) able to distinguish the Type I Diabetes group (T1D) from both control groups and all other six disease groups with a highly significant p-value (p < 9.24 X 10 -9). The p-value of any pairwise comparison between the other eight groups was non-significant. Interestingly, there is no evidence of association between T1D and any of these ten SNPs individually, indicating their joint contribution to T1D.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q09</strong></td>
 </tr> <tr> 
  <td><b>Testing for Joint Association of All SNP Pairs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ronald Schuyler <i>University of Colorado Denver</i></td>
 </tr>
 <tr> 
  <td>  Lawrence Hunter (University of Colorado Denver, Pharmacology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Despite the success of genome-wide association studies (GWAS) in providing insight into mechanisms of disease, the associated loci usually account for only a fraction of the expected heritability of each condition studied. This implies that more may be learned from GWAS data by going beyond the one-SNP-at-a-time association approach. In studying complex traits, it is useful to look for associations with pairs of loci. The standard logistic regression test for joint effects requires iterative methods for determining maximum likelihood estimates (MLEs), which makes testing all possible pairwise combinations from common high-throughput genotyping platforms extremely computationally demanding, and limits the wider application of this approach.<br><br>Loglinear categorical data analysis methods with closed-form solutions for MLEs are well known and have recently been applied to GWAS data, reducing computation time and making it feasible to test all locus pairs. We have proposed a refinement to this method which reduces the number of computations by nearly two thirds. We used likelihood ratio tests of loglinear models with a step-wise model selection procedure to test all 150 billion possible two-locus pairs of a 550k SNP study for joint association with generalized vitiligo.<br><br>Using a stringent multiple testing adjustment, we detected a small number of significant pairs where the two SNPs of every pair detected are in close proximity to one another. SNPs in most regions detected showed no trend toward significance in single locus tests, but these loci have clear biological relevance to the condition studied.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q10</strong></td>
 </tr> <tr> 
  <td><b>Predicting the Functional Impact of Protein Mutations: Application to Cancer Genomics</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Boris Reva <i>Memorial Sloan-Kettering Cancer</i></td>
 </tr>
 <tr> 
  <td>  Yevgeniy Antipin (MSKCC, Computational Biology Center ); Chris  Sander (MSKCC, Computational Biology Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As large scale re-sequencing of genomes reveals many protein mutations, especially in human cancer tissues, prediction of their likely functional impact becomes an important practical goal. Here, we introduce a new functional impact score (FIS) for amino acid residue changes using conservation patterns. The evolutionary information in these patterns is derived from aligned families and sub-families of sequence homologs within and between species using a combinatorial entropy formalism. We tested the score on a large set of human protein mutations for its ability to separate disease-associated variants (~19,200), assumed to be strongly functional, from common polymorphisms (~35,600), assumed to be weakly functional, and obtained an area under the receiver-operating-characteristic curve of ~0.86. From analysis of ~10,000 cancer mutations of COSMIC, we conclude that  recurrent mutations, mutations in multiply mutated genes and mutations annotated as cancer genes tend to have significantly higher functional impact scores than control sets. We report a ranked list of ~1000 top human cancer genes frequently mutated in one or more cancer types; and, an additional ~1000 candidate cancer genes with rare but likely functional mutations. In addition, we estimate that ~5% of cancer-relevant mutations involve switch of function, rather than simply loss or gain of function.  The computational protocol is implemented as a public server: http://mutationassessor.org. The server provides links to multiple sequence alignment and 3D structures, to various biological and cancer annotations. The service is built to process output of sequencing machines, and it is capable of quickly processing thousands of variants (through WEBAPI). </td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Q11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Q11</strong></td>
 </tr> <tr> 
  <td><b>Compare H. sapiens + H. neanderthalensis by predicting SNPs effects</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Shaila Rössle <i>Technical University Munich</i></td>
 </tr>
 <tr> 
  <td>  Dominik Achten (Technical University Munich, Department for Bioinformatics and Computational Biology); Martin Kircher (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); Janet Kelso (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); Svante Pääbo (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); Burkhard Rost (Technical University Munich, Department for Bioinformatics and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We analyzed features that are unique to human with respect to Neandertal.We started from a set of 78 nucleotide substitutions (nsSNPs) where modern humans are fixed for a derived state and where the Neandertal carry the ancestral state. We applied SNAP to these mutations in order to predict changes that affect protein function and infer phenotypic differences between humans and Neandertals. In particular, for each nsSNP encountered in 72 human, chimpanzee and Neandertal proteins, we predicted the effect of all non-native mutations. <br>Our results are still very preliminary; we focus on very few cases of proteins for which the predicted sensitivity to change differed substantially, proteins that show high sequence identity between human and chimpanzee, and that SNP shows different functional effects among the organisms predicted by SNAP. Neandertal proteins with very high similarity between human and chimpanzee could show phenotypic differences. Sometimes resemble human proteins in terms of their sensitivity to mutations and sometimes they seem to be more chimpanzee proteins. For example SSH2, a protein phosphatase, differs from chimpanzee in its interaction hotspots. On the other hand ACCN4, an acid sensing ion channel implicated in synaptic transmission, pain perception as well as mechanoperception, by our analysis is more similar to the human protein.<br>Our major challenge is getting further remain the incompleteness of Neandertal: 10% have been sequenced. This number implies that not a single Neanderthal protein is properly known. Should we use human to fill white spaces or chimpanzee? Or a hybrid? Next, we will address these</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




