
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=Y by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist718c.html?cat=Y by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:19 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'Y'- Text Mining'</b></td>
 </tr>
</table>

<a name='Y01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y01</strong></td>
 </tr> <tr> 
  <td><b>Biomedical Natural Language figure Processing for Intelligent Figure Search</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hong Yu <i>University of Wisconsin-Milwaukee</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Figures represent important knowledge in biomedical literature. An intelligent biomedical figure search engine can assist biocuration, provide biomedical scientists with targeted evidence search, and provide a method for automatic validation of genome-wide high-throughput predictions. Currently, neither natural language processing (NLP) nor image processing algorithms address the unique complexities inherent to accurate biomedical figure searching. Most NLP approaches focus on text, and, for the most part, ignore figures altogether. On the other hand, traditional image search processing approaches frequently ignore important associated text. No approach takes advantage of the semantics that relate in-text content to figures that are uniquely encased in biomedical publications. We open a new research direction entitled “biomedical Natural Language figure Processing” (bNLfP) that combines traditional NLP and image processing by using the semantic relationships that exist between published biomedical text and figures. I will present two innovative bNLfP algorithms: figure ranking and figure text extraction.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20074.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Y02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y02</strong></td>
 </tr> <tr> 
  <td><b>Differing structure and content of abstracts and full text journal articles</b><br/><br/></td>
 </tr>
 <tr> 
 <td>K. Bretonnel Cohen <i>U. Colorado School of Medicine</i></td>
 </tr>
 <tr> 
  <td>  Helen Johnson (U. Colorado School of Medicine, Pharmacology); Karin Verspoor (U. Colorado School of Medicine, Pharmacology); Christophe Roeder (U. Colorado School of Medicine, Pharmacology); Lawrence Hunter (U. Colorado School of Medicine, Pharmacology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Abstracts have historically been the main input to biomedical text mining, but a recent paradigm shift is seeing a major move towards mining the full text of journal articles.  This paper demonstrates that a number of differences with consequences for biomedical text mining exist between abstracts and full text articles, and lays out a roadmap for adapting to the new challenges presented to us by full text.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20090.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='Y03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y03</strong></td>
 </tr> <tr> 
  <td><b>Implicit conceptual links in the literature: a large untapped reservoir of gene discovery</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Herman van Haagen <i>University of Leiden</i></td>
 </tr>
 <tr> 
  <td>  Martijn Schuemie (Erasmus University of Rotterdam, Biosemantics); Peter-Bram 't Hoen (University of Leiden, Human Genetics); Marco Roos (University of Leiden, Human Genetics); Barend Mons (University of Leiden, Human Genetics); Johan den Dunnen (University of Leiden, Human Genetics); Gert-Jan van Ommen (University of Leiden, Human Genetics); Erik Schultes (University of Leiden, Human Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The published literature and bioinformatic data are growing faster than individual researchers can track. Text- and data-mining tools have been developed to retrieve explicit links between pairs of concepts (e.g. genes and disease). However, in addition to explicit links, text also contains a complex network of indirect (implied) links containing useful information. We applied concept profiling to probe this vast network of implied information. In a retrospective analysis of 18 gene-disease relationships, implicit information alone could prioritize the causative gene on average within the top 13 out of 200 genes located in a specified linkage interval, at least one year before publication of the landmark paper. The concepts shared between gene and disease enabled the evaluation of the plausibility of the inferred relationship. Of the 40,404,412 possible gene disease pairs, 120,246 (47%, p < 0.003) arose exclusively from implicit relations. These results reveal an enormous untapped discovery potential in the implied information of biomedical literature .</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y04</strong></td>
 </tr> <tr> 
  <td><b>ANDCell-ANDVisio system: knowledge extraction using text-mining in the systems biology</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pavel Demenkov <i>Institute of Cytology and Genetics</i></td>
 </tr>
 <tr> 
  <td>  Ewgenia Yarkova (Institute of Cytology and Genetics, Siberian Branch of the Russian Academy of Sciences); Timofey Ivanisenko (Institute of Cytology and Genetics, Siberian Branch of the Russian Academy of Sciences); Vladimir Ivanisenko (Institute of Cytology and Genetics, Siberian Branch of the Russian Academy of Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Work with scientific literature is required for research in every knowledge area. The PubMed database currently contains about 19 millions of abstracts. Analysis of this bulky unstructured information is very time-consuming. Furthermore, the modern approaches to analysis of the unstructured information make obligatory reference to another important source, the factographic databases for molecular biology and genetics.<br><br>The method for automated extraction of information about molecular-genetic interactions from PubMed abstracts was developed using the text-mining approach. For text-mining, we used the previously developed thesauruses for the names of proteins, genes, microRNAs, metabolites, biological pathways, diseases, cells, and organisms. To recognize facts describing molecular genetic interactions in abstract texts we created more than 4000 patterns or decision rules. These templates provide an automated extraction of knowledge from PubMed abstracts about molecular genetic interactions, gene regulations, catalytic processes and other associations between facts and their representation as semantic association networks. Extracted information was integrated through reconstruction of networks for semantic associations joining literary facts about molecular-genetics regulations, physical interactions, also about associations between molecular-genetic objects, biological processes, and diseases.<br><br>The ANDCell system contains the knowledge base and the ANDVisio program for associative network reconstruction. The ANDCell knowledge base contains more than 5 millions of molecular genetic interaction facts. The ANDVisio program allows the user to access the database and represents the results in a graphic form as associative networks. The system is provided with a user's friendly interface implemented links to the molecular-genetic databases and articles from information was extracted.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y05</strong></td>
 </tr> <tr> 
  <td><b>Self-Training Improves Robustness of Protein-Protein Interaction Extraction from Text</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Philippe Thomas <i>Humboldt University of Berlin</i></td>
 </tr>
 <tr> 
  <td>  Illés Solt Solt (Budapest University of Technology and Economics, 2Department of Telecommunications and Media Informatics); Ulf Leser (Humboldt University of Berlin, Knowledge Management in Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motivation: Automated extraction of protein-protein interactions (PPI) from the literature is a key challenge in current biomedical text-mining. The past years have seen a certain increase in the performances reported for novel methods. However, these results are typically obtained using corpus-wise cross-validation, which, as shown by a number of more recent works, tends to be overly optimistic when trying to extrapolate the performance on new text. <br>Result: In this work, we present a general method for increasing the robustness of PPI extraction methods that is based on self-training. Self-training is a semi-supervised learning technique where a classifier is first trained, then applied, and then retrained on the results of the first application. We achieve consistent improvements in both extrinsic performance (0.8-6.6% in F1) and robustness (the F1 gap to cross-validation reduced by 41%) compared to the over-optimistic estimates on extrinsic performance provided by cross-corpus evaluation. Notably, our method is agnostic to the particular information that is extracted and can thus be used to improve performance in many tasks in biomedical text-mining.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y06</strong></td>
 </tr> <tr> 
  <td><b>Extraction of protein cellular localization using the Locminer text mining system</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Martin Krallinger <i>Spanish National Cancer Research Centre</i></td>
 </tr>
 <tr> 
  <td>  Alfonso Valencia (Spanish National Cancer Research Centre, Structural Biology and Biocomputing Programme ); Ashish Tendulkar (Indian Institute of Technology, Madras, Department of Computer Science and Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: To determine the subcellular location of proteins is crucial to understand their biological and functional properties. We implemented the Locminer system, which explores semantic-syntactic frames for extracting fine-grained associations between proteins and subcellular locations from sentences. This tool is based on localization frames, an set of trigger verbs relevant for expressing locative relations and a location sentence classifier. A comprehensive subcellular location dictionary integrating keywords/synonyms from SwissProt, Cellular Component terms from Gene Ontology, and an in house location dictionary was created. After detecting protein names co-occurring with location terms, a total of 1,288 sentences were manually revised to derive 396 hand crafted location frames, covering mainly binary protein-localization relations, in addition to associations to multiple (alternative) locations. We expanded semi-automatically locative expressions selecting 220 location and motion-relevant verb roots by including verb synonyms, English spelling variants, generation of affix variants (using 14 general English affixes such as de-, sub-, co- and trans-, and 5 domain specific affixes such as immuno- and radioimmuno-), generation of hyphenation variants as well as inflectional variants and nominalized forms. This resulted in 11,544 expanded forms and a dictionary of 161,616 potential location. These variants were filtered based on their instantiation on the whole PubMed database (remaining a total of 6,436 location triggers). To prioritize location words/expressions for subcellular relevance we considered the fraction of cellular location term co-occurrences with respect to their total frequency in PubMed, as well as a score provided by a SVM sentence classifier trained on documents annotated as location relevant.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y07</strong></td>
 </tr> <tr> 
  <td><b>Reflect: a browser plugin for life scientist</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Janos Binder <i>European Molecular Biology Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Reinhard Schneider (European Molecular Biology Laboratory, Structural and Computational Biology Unit); Sean  O'Donoghue (European Molecular Biology Laboratory, Structural and Computational Biology Unit); Lars Juhl Jensen (University of Copenhagen, NNF Center for Protein Research); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: During the past decade, extensibility of web browsers became a killer feature. We have built a browser plugin called Reflect which augments life science browsing experience. By tagging an article with a single click, any user can access further information related to proteins, genes or chemicals in a popup window.<br>Reflect was widely used during the past years, and by accessing its API different websites have incorporated its features e.g.  it can be switched on ScienceDirect website by highlighting keywords. We aim to keep it simple and user friendly, without having to access any complex databases and retrieving relevant information about the selected terms. This year we are empowering our software with new features like sub-cellular and tissue localization of proteins, while keeping the current visualization abilities e.g.: sequence, structure and interaction network. The software is freely downloadable at: http://www.reflect.ws<br><br>References:<br><br>O’Donoghue et al: Reflect: A practical approach to web semantics, Journal of Web Semantics (2010), Special Issue: Sp. Iss. SI 2-3 (8) 182-189<br><br>Pafilis et al: Reflect: augmented browsing for the life scientist, Nat Biotechnol. 2009 Jun;27(6):508-10.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y08</strong></td>
 </tr> <tr> 
  <td><b>Biomedical text mining from multiple views: information fusion and vertical</b><br/><br/></td>
 </tr>
 <tr> 
 <td>xinhai Liu <i>Katholieke Universiteit Leuven & IBBT</i></td>
 </tr>
 <tr> 
  <td>  Olivier Gevaert (1. Katholieke Universiteit Leuven & IBBT , ESAT-SCD & Future Health Department); Léon-Charles Tranchevent (1. Katholieke Universiteit Leuven & IBBT, ESAT-SCD & Future Health Department); Bart De Moor (1. Katholieke Universiteit Leuven & IBBT, ESAT-SCD & Future Health Department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Biomedical literature contains rich information which could be observed from different point of views and is expected to provide better or exact knowledge about biomedical process. Thus we propose a novel strategy to provide text prior information from multi-view perspectives. The strategy is implemented by text mining on MEDLINE database. Our strategy can be applied to do information fusion by integrating multi-view data or provide certain knowledge from a small vertical perspective.  A Web application of our strategy is developed for gene retrieval. The multiple views can be different controlled vocabularies, weighting schemes, publishing time periods and biomedical subjects. As a series of human related genes are input, based on the multi-view selection, the outputs are the gene-by-term profile, which is illustrated by term cloud and gene-by-gene similariy matrix, which is visualized by color map. In addition, the hierarchical clustering of these queried genes are available as well.    We employ a set of genes which belong to different diseases to test our multi-view gene retrieval system. Firstly, we investigate the dis-similarity of multiple views by calculating the cosine similarity among them and the results demonstrate that multiple views vary with each other, in other words, each of them is able to provide complementary information to certain extend. Secondly, we carry out the hybrid clustering by integrating multi-view text mining data. The clustering results show that integrating multi-view controlled vocabularies and weighting schemes is able to boost clustering performance.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y09</strong></td>
 </tr> <tr> 
  <td><b>Keyword Clustering in Biomedical Information Retrieval Using Evolutionary Algorithms</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Viktoria Dorfer <i>Upper Austria University of Applied Sciences</i></td>
 </tr>
 <tr> 
  <td>  Stephan Winkler (Upper Austria University of Applied Sciences, Medical and Bioinformatics); Thomas Kern (Upper Austria University of Applied Sciences, Research & Development); Sophie Blank (Upper Austria University of Applied Sciences, Research & Development); Gerald Petz (Upper Austria University of Applied Sciences, Marketing & Electronic Business); Patrizia Faschang (Upper Austria University of Applied Sciences, Research & Development); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As the amount of available data in the field of life sciences grows exponentially, intelligent search strategies are necessary to help people in information retrieval. We here describe the use of a new keyword clustering method: Based on a set of documents (D), keyword clusters are optimized so that the identified groups of keywords consist of keywords that often occur in combination in D. The so generated keyword clusters shall in the near future serve as a solid base for a new PubMed search tool based on query extension, using also user feedback to optimize the search process.<br>We have defined several important characteristics for clustering candidates, including the data set coverage, the cluster confidence (measuring the ratio of clustered keywords that are found in the same documents), and the document confidence (measuring the amount of equal keywords in the documents assigned to a cluster through their keywords). Evolutionary algorithms have been applied for solving this optimization task, amongst others evolution strategies (ES) and a multi-objective genetic algorithm (NSGA-II, used because the optimization objectives are partially contradictory).<br>For testing this approach we have used data published for the TREC-9 conference containing 36,890 entries. Out of this data set we extracted the most significant keywords for clustering using tf-idf weighting. Analyzing first optimization results we see that the best result obtained with 10+1 ES provides 23.5% data set coverage, 45.2% cluster confidence, and 23.4% document confidence; using the NSGA-II we for example got results with respective values 71%, 56% and 37%.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y10</strong></td>
 </tr> <tr> 
  <td><b>Automating host-pathogen interaction discovery: an HIV case study</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daniel Jamieson <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  Jonathan Dickerson (University of Manchester, Bioinformatics); Martin  Gerner (University of Manchester, Computer Sciences); Farzaneh Sarafraz (University of Manchester, Computer Sciences); Goran  Nenadic (University of Manchester, Computer Sciences); David  Robertson (University of Manchester, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: HIV’s exploitation of the human host for replication involves highly specific interactions with the cellular system. The persistence of the virus is dependent on an intricate network of biological interactions. There is presently extensive knowledge of molecular interactions within the scientific literature. However, manual curation of these would be an enormously time-consuming task. Here we present a procedure for automated host-pathogen focused data extraction. We focus on mining HIV-1-host protein interactions, using the gold-standard HIV-1 Human Protein Interaction Database, containing 5,127 interactions manually extracted from 14,312 references, as a benchmark for evaluation. Our process involves employing a combination of named entity recognisers (such as BANNER) and the highest performing event extraction tools (e.g. from the BioNLP 2009 shared task). We replicated the original dataset to an acceptable degree, however with compromises devised to conform to the design of tools presently available. Notably, event extraction tools are limited to obtaining interactions from 9 event types, which are not compatible with all of the interactions in HIV biology. Thus, we have developed an appropriate ontology to encompass the full spectrum of host-pathogen interactions and demonstrate improved coverage. As a result, we demonstrate the effective use of sophisticated text mining tools to functionally recreate a large database of host-pathogen based interactions. This provides a platform for full-scale automated extraction of host-pathogen interactions from all the available literature (20 million+ online references). Not only will such an approach improve our understanding of host-pathogen biology, it may also permit informed development of pharmaceutical interventions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y11</strong></td>
 </tr> <tr> 
  <td><b>Relation-mining workflows to extract animal host-microorganism interactions in free text</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sophia Ananiadou <i>University of Manchester</i><br></td>
 </tr>
 <tr> 
  <td>BalaKrishna Kolluru (University of Manchester) Sirintra Nakjang (Univerisity of Newcastle, Institute for Cell and Molecular Biosciences); Robert, P. Hirt (Univerisity of Newcastle, Institute for Cell and Molecular Biosciences); Anil Wipat (Univerisity of Newcastle, Institute for Cell and Molecular Biosciences); Sophia Ananiadou (University of Manchester, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission. We have developed a relation-mining workflow to extract sentences with a particular focus on animal host-microorganisms pairs from free text such as journal papers. The workflow uses three different component, one each to convert pdf to raw text, to process sentences from this raw text and a CRF-based relation miner. The CRF-based component uses, lexical features and bag-of-words as features to estimate the "relation" aspect in a sentence.  The results indicate that the precision of our approach is around 85%; a significant number of errors reported by the workflow are due to the erroneous conversion from pdf to text. We are currently working both at improving the net coverage of the machine-learning component using syntactic relationship between the main entities and their connecting words. To this end, we are working on reducing the noise in pdf-to-text conversion. We show that accuracy in automatic extraction of relations is a boost to creation of comprehensive databases.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y12</strong></td>
 </tr> <tr> 
  <td><b>Overview of the second CALBC Challenge</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chen Li <i>EMBL Outstation Hinxton</i><br></td>
 </tr>
 <tr> 
  <td>Senay Kafkas (EMBL Outstation Hinxton) Erik van Mulligen (Erasmus University Medical Center, Department of Medical Informatics); Jan Kors (Erasmus Medical Center, Department of Medical Informatics); Udo Hahn (Friedrich-Schiller-Universität Jena, Jena University Language & Information Engineering (JULIE) Lab); David Milward (Linguamatics, -); Dietrich Rebholz-Schuhmann (EMBL Outstation Hinxton, European Bioinformatics Institute); Ian Lewin (EMBL Outstation Hinxton, European Bioinformatics Institute); Chen Li (EMBL Outstation Hinxton, European Bioinformatics Institute); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The aim of the CALBC project is to generate a large scale Silver Standard Corpus (SSC) (about 1 million Medline abstracts) annotated with diverse biomedical entities by automatically harmonizing different solutions from the community. The second CALBC challenge attracted 16 teams from all around the world. The results were presented in a workshop held in Hinxton, UK, March 16-18, 2011. <br>The 2nd challenge required participants to annotate a much greater quantity of data (175k and, optionally 714k documents) and offered the opportunity to annotate a much larger number of entity types (up to 16). An annotated training corpus (100k documents)  for 4 entity types was generated automatically using the harmonization scheme derived from the first challenge, applied to the 4 project partners. Challenge submissions were then a) used to generate a new SSC through an improved character-based voting mechanism b) evaluated against it. As with the smaller scale 1st challenge evaluations (1k documents), some participants, especially trained solutions, outperform the partners over a partner-only SSC; and this is true for all 4 entity types. In contrast to the 1st challenge, we find that an all-participant SSC requires different voting thresholds for different entity types.  <br>In a new departure, we have also evaluated our SSCs against published Gold standard annotations (where available).  Furthermore, several solutions have been developed for normalizing the annotated biomedical entities to terminological resources (e.g. UMLS). The SSC has been converted to RDF format for public exploitation via the Semantic Web.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y13</strong></td>
 </tr> <tr> 
  <td><b>An exercise in kidney factomics: From article titles to RDF Knowledgebase</b><br/><br/></td>
 </tr>
 <tr> 
 <td>James Eales <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  George Demetriou (University of Sheffield, Department of Computer Science); Robert Stevens (University of Manchester, School of Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: There are many existing resources that integrate data between databases; they do this either semantically by the use of RDF and triplestores (e.g. Bio2RDF), or with web links and ID mapping services (e.g. PICR, eUtils).  However results declared in the literature are only rarely interlinked with existing databases and even more rarely interlinked with each other.  We describe a method to take factual statements reported in the literature and turn them into semantic networks of RDF triples.  We use a method based on finding titles of papers that contain a positive, direct statements about the outcome of a biomedical investigation. We then use dependency parsing and an ontological perspective to create and combine graphs of knowledge about a domain.  Our aim in this work is to collect knowledge from the literature for inclusion in the Kidney and Urinary Pathways Knowledge Base (KUPKB), which will be used in the e-LICO project to illustrate the utility of data-mining methods for biomarker discovery and pathway modelling.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='Y14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster Y14</strong></td>
 </tr> <tr> 
  <td><b>AUTOMATIC PREDICTION OF GENE FUNCTION USING LITERATURE PROFILING</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Roney S. Coimbra <i>Center for Excellence in Bioinformatics (CEBio), FIOCRUZ-Minas</i><br></td>
 </tr>
 <tr> 
  <td>Francislon Oliveira (Center for Excellence in Bioinformatics (CEBio), FIOCRUZ-Minas) Guilherme Corrêa Oliveira (Center for Excellence in Bioinformatics (CEBio), FIOCRUZ-Minas, LPCM); Raul Torrieri (Center for Excellence in Bioinformatics (CEBio), FIOCRUZ-Minas, LPCM); Roney Santos Coimbra (Center for Excellence in Bioinformatics (CEBio), FIOCRUZ-Minas, LPCM); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We propose a new tool for the automatic prediction of bacterial gene function based on literature profiles. For this purpose we developed and used LitProf, an implementation of the Chaussabel & Sher algorithm, to identify the minimum vocabulary required to describe the function of a given gene from a collection of abstracts in Pubmed. Our initial dataset consisted of ~50 K canonical gene names randomly picked from genomes representing all bacterial phylogenetic branches at the JCVI Resource. This gene set was screened to eliminate homonymous orthologs, and genes assigned to imprecise or to more than one functional categories of the JCVI ontology. To further reduce redundancy, genes were grouped by Hierarchical Clustering based on the similarities of word-frequency vectors generated by Litprof from their gene-specific text corpora of PubMed abstracts. One gene was randomly chosen to represent each cluster of highly related genes (> 0.99 correlation). From the text corpora of these selected genes, LitProf disclosed a minimum informative vocabulary and produced the word-frequency vectors representing each gene. We used these vectors to train a Support Vector Machine (SVM) gene classifier. In a 100X cross-validation the classifier showed 80±3% average precision, and 60±3% average recall. In an independent set of 4,000 JCVI annotated genes, the classifier achieved up to 81% precision with 73% recall.<br>	Through this method we propose functional categories to 24,104 unclassified genes with abstracts in PubMed. For confidence thresholds higher than 0.7, 4,927 genes (> 20%) were unambiguously assigned to one functional category of the JCVI ontology.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




