
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=G by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist4238.html?cat=G by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:18 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'G'- Functional Genomics'</b></td>
 </tr>
</table>

<a name='G01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G01</strong></td>
 </tr> <tr> 
  <td><b>Large-Scale Computational Identification of Regulatory SNPs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Alberto Riva <i>University of Florida</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: rSNP-MAPPER is a web-based tool to identify SNPs that potentially affect a Transcription Factor Binding Site to a significant extent (regulatory SNPs, or rSNPs). rSNP-MAPPER builds on MAPPER, a previously developed application for the computational detection of TFBSs in DNA sequences. We have provided MAPPER with the ability to analyze two variants of the same sequence (each containing one allele of a SNPs), determining whether the substitution results in a significant change in the TFBS predictive score.<br><br>The application's provides an intuitive and flexible interface. The user may search for potential rSNPs in the promoters of one or more genes, that can be specified as a list, or chosen from the members of a pathway. Alternatively, the user may specify a set of SNPs to be analyzed by uploading a list of SNP identifiers or providing the coordinates of a genomic region; rSNP-MAPPER will determine which SNPs lie within a TFBS and compute the corresponding score changes. Finally, the use can provide two alternative sequences (wildtype and mutant): the system will analyze them to determine the location of variants, to identify potential TFBSs, and to determine the effect of the variants on the TFBS scores. rSNP-MAPPER is optimized to efficiently perform all these operations on a large scale, allowing for the fast annotation of thousands of SNPs.<br><br>We present the architecture of rSNP-MAPPER, and we describe its usage through several examples that demonstrate its ability to correctly identify previously known rSNPs, or to predict new ones with high confidence.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G02</strong></td>
 </tr> <tr> 
  <td><b>Systematic functional characterization of cancer genomes using large scale pooled shRNA screening</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Barbara Weir <i>The Broad Institute</i></td>
 </tr>
 <tr> 
  <td>  Shuba Gopal (The Broad Institute, RNAi platform); Jesse Boehm (The Broad Institute, Cancer Program); Glenn Cowley (The Broad Institute, RNAi platform); Hiu Wing Cheung (The Broad Institute, Cancer Program); Terence Wong (The Broad Institute, Cancer Program); Pablo Tamayo (The Broad Institute, Cancer Program); Nicolas Stransky (The Broad Institute, Cancer Program); Jordi Barretina (The Broad Institute, Cancer Program); Aviad Tsherniak (The Broad Institute, Cancer Program); Josh Gould (The Broad Institute, Cancer Program); Scott  Rusin (The Broad Institute, RNAi platform); Justin Scott (The Broad Institute, RNAi platform); Aravind Subramanian (The Broad Institute, Cancer Program); Levi Garraway (The Broad Institute, Cancer Program); Matthew Meyerson (The Broad Institute, Cancer Program); Jill Mesirov (The Broad Institute, Computational Biology and Bioinformatics Program); David Root (The Broad Institute, RNAi platform); William Hahn (The Broad Institute, Cancer Program); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A wealth of information is being generated with the goal of characterizing all somatic alterations in cancer genomes across various tumor types.  A complementary approach to these cancer genomic efforts involves the systematic functional assessment of genes and their requirement for producing malignant phenotypes.  This project uses a genome scale loss-of-function screening approach in a diverse panel of cancer cell lines to identify genes involved in cancer pathogenesis.  We developed a complete pipeline of computational methods for the processing and analysis of RNAi pooled screening data.  Viability data obtained from Affymetrix custom barcode array measurements were processed at the shRNA level, then combined into gene scores and utilized in higher level analyses.  We demonstrated that the relative effects on proliferation, as measured in a retest of 350 individual shRNAs in low throughput, agreed well with the quantification of shRNA pooled screen data.  A class-discrimination feature selection method was used to identify and rank shRNAs by their ability to distinguish specified classes, including lineage of origin or mutational status.  Specifically, we applied our analyses to the ovarian lineage and revealed genes, including PAX8, that were both essential in ovarian cancer cells and reside in recurrently amplified regions found by analyses of ovarian cancer genomes from The Cancer Genome Atlas project.  This scheme of integrating structural and functional genomics efforts provides a general and efficient course to identifying cancer drivers, novel targets for cancer therapeutics and previously unknown dependencies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G03</strong></td>
 </tr> <tr> 
  <td><b>Annotating New Genes: From In Silico Screening to Validations by Experiments</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Shizuka Uchida <i>Max-Planck-Institute for Heart and Lung Research</i></td>
 </tr>
 <tr> 
  <td>  Pascal Gellert (Max-Planck-Institute for Heart and Lung Research, Dept. of Cardiac Development and Remodelling); Katharina Jenniches (Max-Planck-Institute for Heart and Lung Research, Dept. of Cardiac Development and Remodelling); Mizue Teranishi (Max-Planck-Institute for Heart and Lung Research, Dept. of Cardiac Development and Remodelling); David John (Max-Planck-Institute for Heart and Lung Research, Dept. of Cardiac Development and Remodelling); Piera De Gaspari (Max-Planck-Institute for Heart and Lung Research, Dept. of Cardiac Development and Remodelling); Thomas Braun (Max-Planck-Institute for Heart and Lung Research, Dept. of Cardiac Development and Remodelling); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In recent years, the advent of high-throughput analytical techniques, such as microarrays and serial analysis of gene expression (SAGE), has led to a rapid accumulation of biological data. The large size of databases precludes manual analysis and renders unsystematic approaches obsolete. To cope with these new challenges and to facilitate efficient data analyses, numerous academic and commercial software packages and databases have been developed. Yet, genes to which no biological function has been assigned compromise the usability of these data. To facilitate functionally annotating these so-called ‘novel genes’, an in silico screening of such genes has been developed by focusing especially on their expression patterns, namely their “tissue-enrichment” and a knowledge database called “C-It” (http://C-It.mpi-bn.mpg.de) has been developed. Emphasis has also been placed on “tissue-specific” isoforms by developing a tool to analyze Affymetrix’s Exon Array, called “Exon Array Analyzer (EAA)” (http://EAA.mpi-bn.mpg.de/). <br>With these algorithms and tools, ~1,000 genes are currently being annotated, which are enriched in a tissue but have not been characterized. To this end, a project called “1,000 Genes Project” has been initiated to functionally annotate these evolutionary-conserved, tissue-enriched genes with unknown functions using various model organisms (mouse, chicken and zebrafish) and in vitro models (ES cells). In order to benefit from this project, we are also screening for such genes with possible relations to human diseases by incorporating SNPs information. In this conference, we would like to share some of our preliminary data on evolutionary-conserved, tissue-enriched genes with unknown function.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G04</strong></td>
 </tr> <tr> 
  <td><b>Maximizing the number of aligned reads from RNA-Seq data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Thomas Bonfert <i>Ludwig-Maximilians-University Munich</i></td>
 </tr>
 <tr> 
  <td>  Gergely Csaba (Ludwig-Maximilians-University Munich, Institute for Informatics); Ralf Zimmer (Ludwig-Maximilians-University Munich, Institute for Informatics); Caroline C. Friedel (Ruprecht-Karls-University Heidelberg, Institute for Molecular Biotechnology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA sequencing by next generation sequencing technologies (RNA-Seq) provides a novel way to characterize the transcriptome of different cell types. One important application is the detection of different transcripts from the same genetic locus.<br>For this purpose, the position of reads on the transcriptome has to be determined first. If an organism with a sequenced and well-annotated genome is investigated, a common approach is to align the generated reads against a reference transcriptome. Based on the alignment, the reads are mapped onto exons and exon-exon junctions of annotated genes. This mapping then provides the basis for quantification and analysis of the transcriptome. Unfortunately, depending on the quality of the sequenced reads, this approach generally fails to identify the origin for a significant large amount of reads.<br>In order to address this problem, we have developed a pipeline which maximizes the number of reads whose origin can be explained by investigating several alternative possibilities. This pipeline is suitable for RNA-Seq data for any organism with available transcriptome and genome annotation. In a first step, a transcriptome mapping is performed as described above. Second, alignment to the genome identifies both incompletely spliced transcripts as well as novel splicing events. Finally, novel transcripts are predicted and potential sources of contaminations are analyzed. Our evaluation results show that using this advanced read processing pipeline, the number of aligned reads can be increased considerably compared to alternative approaches.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G05</strong></td>
 </tr> <tr> 
  <td><b>Trypanosoma cruzi gene expression analysis in response to gamma radiation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Priscila Grynberg <i>Universidade Federal de Minas Gerais</i></td>
 </tr>
 <tr> 
  <td>  Danielle Passos-Silva (Universidade Federal de Minas Gerais, Departamento de Bioquímica e Imunologia); Marina Mourão (Centro de Pesquisas René Rachou/FIOCRUZ, Grupo de Genômica e Biologia Computacional); Roberto Hirata-Jr (Universidade de São Paulo, Instituto de Matemática e Estatística); Andrea Macedo (Universidade Federal de Minas Gerais, epartamento de Bioquímica e Imunologia); Carlos Machado (Universidade Federal de Minas Gerais, epartamento de Bioquímica e Imunologia); Daniella Bartholomeu (Universidade Federal de Minas Gerais, epartamento de Parasitologia); Gloria Franco (Universidade Federal de Minas Gerais, epartamento de Bioquímica e Imunologia); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Among the various types of damages caused by ionizing radiation, the lesions of greatest biological importance are the double-stranded DNA breaks. The damage tolerance requires an efficient apparatus for recognition and repair of lesions and these characteristics define the degree of resistance of various organisms to this type of stress. Trypanosoma cruzi is highly resistant to gamma rays, considering that a 500 Gy dose induces a genomic DNA fragmentation, but the karyotype is gradually repaired and the chromosomal bands pattern is restored in less than 48 hours. In this study, we aimed to compare T. cruzi epimastigotes gene expression from cells irradiated or not with a sub-lethal dose of 500 Gy, through microarray experiments. Our results showed that irradiation caused an arrest in cell growth, but did not significantly affect the integrity of RNA molecules, contrary to what was seen in DNA molecules. The gene expression was affected in a time-dependent manner, because the peak of down-regulated genes (composed mostly of genes with known function) and up-regulated genes (composed mostly of genes of unknown function) occurred after 4 and 96 hours, respectively. Four maxicircle genes and those coding for retrotransposons hot spot protein (RHSs) were strongly induced after 48 hours. However, genes related to basal metabolic functions presented a decrease in the expression levels. There was no induction in the expression of DNA repair genes, excepted for the tyrosil-DNA phosphodiesterase 1 gene.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G06</strong></td>
 </tr> <tr> 
  <td><b>Mining of Transcriptome of a climacteric fruit species Diospyros kaki Thunb for functional genomics</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Farzana Rahman <i>University of Glamorgan</i></td>
 </tr>
 <tr> 
  <td>  Dr. Gaurav Sablok (Huazhong Agricultural University, Shizishan, Wuhan, China 430070, Key Lab of Horticultural Plant Biology(MOE)); Dr. Tatiana Tatarinova (University of Glamrogan,Pontypridd, Wales, UK, Computational Biology Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Our current research focuses on developing computational resources for analysis of genome of persimmon (Diospyros kaki). Persimmon is an economically important climacteric fruit species, which has promising medicinal values. The genus Diospyros is widely distributed in Asia, Africa, and America and is characterised by complex ploidy level. <br>In order to provide an insight into the developmental stages of D. kaki, we conducted a comparative analysis of developmental stage specific ESTs libraries. A total of 2,529 putative tentative unigenes were identified in the Mature Fruit (MF) library whereas   3,775 tentative unigenes were identified in the Ovary and Young Fruit (OYF) library.<br>Comparative mining of SSRs between two ESTs libraries showed 325 EST-SSRs in 296 putative unigenes which were detected in the MF library showing an occurrence of 11.7% with a frequency of 1 SSR/3.16 kb and the OYF library showed an EST-SSRs occurrence of 10.8% with 407 EST- SSRs in the 352 putative unigenes with a frequency of 1 SSR/2.92kb. SNPs and indels were also identified and compared among the two cDNA libraries using redundancy and co-segregation based approach. Overall, the OYF library has 20.94 SNPs/indels per 100 bp, whereas the MF library has only 0.74 SNPs/indels per 100 bp. We detected 506 SSRs primer pairs that can be utilized to infer genetic diversity in persimmon. We also found potentially conserved 159 miRNA family expressed during the developmental stages in OYF library which correlates the potential involvement of miR159 family in development and relative distribution of SNP and SSR-FDMs markers.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G07</strong></td>
 </tr> <tr> 
  <td><b>An integrated analysis of molecular aberrations in NCI-60 cell lines</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chen-Hsiang Yeang <i>Academia Sinica</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Cancer is a complex disease where various types of molecular aberrations drive the development and progression of malignancies.  As large-scale screenings of multiple types of molecular aberrations become increasingly common, a computational model integrating multiple types of information is essential for the analysis of the comprehensive data.  We propose an integrated modeling framework to identify the statistical and putative causal relations of various molecular aberrations and gene expressions in cancer.  We sequentially applied three layers of logistic regression models with increasing complexity and uncertainty regarding the possible mechanisms connecting molecular aberrations and gene expressions.  We applied the layered models to the integrated datasets of NCI-60 cancer cell lines and validated the results with large-scale statistical and biological analysis.  Furthermore, we discovered/reaffirmed the following prominent links: (1)Protein expressions are generally consistent with mRNA expressions.  (2)Several gene expressions are modulated by composite local aberrations.  (3)Amplification of chromosome 6q in leukemia elevates the expression of MYB, and the downstream targets of MYB on other chromosomes are up-regulated accordingly.  (4)Amplification of chromosome 3p and hypo-methylation of PAX3 together elevate MITF expression in melanoma, which up-regulates the downstream targets of MITF.  (5)Mutations of TP53 are negatively associated with its direct target genes.  These validated analysis results justify the utility of the layered models for the incoming flow of cancer genomic data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G08</strong></td>
 </tr> <tr> 
  <td><b>CNAmet: an R package for integrating copy number, methylation and expression data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Riku Louhimo <i>University of Helsinki</i></td>
 </tr>
 <tr> 
  <td>  Sampsa Hautaniemi (University of Helsinki, Institute of Biomedicine); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genomic instability is a key enabling characteristic of tumorigenesis. Identification of regions with copy number alterations has uncovered deregulated tumor suppressors and oncogenes which play key roles in tumor progression and drug response. In addition to copy number changes, gene expression is regulated by DNA methylation. Gene copy number, methylation and expression can all be measured with microarrays which enables their computational integration. The goal of integrating copy number and expression data is to characterize genes essential to cancer progression. This is achieved by identifying genes that are both amplified and upregulated or deleted and downregulated. However, gene upregulation can also be caused by hypomethylation (decrease in methylation of cytosine and adenosine residues in DNA) and downregulation caused by hypermethylation. Since both copy number and methylation alterations can deregulate genes, integration of these three types of data should improve the characterization of essential genes in cancer. We introduce the CNAmet R package which integrates high-throughput copy number, methylation and expression data. Our primary goal is to identify genes that are simultaneously amplified, hypomethylated and upregulated, or deleted, hypermethylated and downregulated. To our knowledge CNAmet is the only software package for the three-way integration of copy number, methylation and expression. We illustrate the usefulness of CNAmet by analyzing copy number, methylation and expression data from 174 patients with glioblastoma multiforme, which is the most aggressive type of brain cancer, as well as 196 ovarian cancer patients.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G09</strong></td>
 </tr> <tr> 
  <td><b>Identification of transcription factor binding sites from ChIP-seq data at high-resolution</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Anaïs Bardet <i>Institute of Molecular Pathology</i></td>
 </tr>
 <tr> 
  <td>  Jonas Steinmann (Institute of Molecular Biotechnology, Knoblich Group); Alexander Stark (Institute of Molecular Pathology, Stark Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Gene expression is mainly regulated at the transcriptional level and achieved through the binding of transcription factors to enhancers. Chromatin immunoprecitation followed by massive parallel sequencing (ChIP-seq) enables the determination of transcription factor binding sites in vivo. Computational tools are available to predict binding sites as regions with significantly enriched read-counts. In our experience however, available tools often merge closely spaced binding sites that frequently occur due to homotypic clusters of binding sites into larger regions.<br><br>Here, we present a new computational tool, Peakzilla, which aims at identifying individual transcription factor binding sites, even when closely spaced. It achieves an improved spatial resolution, pushed to half the lengths of the sonicated DNA fragments, by empirically modeling this fragment size and the bimodal distribution of reads from both strands. <br><br>Peakzilla frequently splits large peak regions identified by other methods into multiple individual regions. These are bona fide transcription factor binding sites as we find them to be enriched in sequence motifs of the respective transcription factor. Peakzilla’s increased spatial resolution also improves the precision of peak-summit locations, as assessed by their distance to the nearest motif, outperforming other methods. It also provides an option to conveniently visualize the genome-wide score profile (similar to the ChIP read density) and to estimate if the ChIP-seq data allow peak-calling at saturating conditions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G10</strong></td>
 </tr> <tr> 
  <td><b>Human gene coexpression landscape and network: location of house-keeping and tissue-specific genes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Alberto Risueño <i>Fundacion de Investigacion del Cancer de la Universidad de Salamanca</i></td>
 </tr>
 <tr> 
  <td>  Celia Fontanillo (Consejo Superior de Investigaciones Cientificas (CSIC), Bioinformatics and Functional Genomics Group); Javier De Las Rivas (Consejo Superior de Investigaciones Cientificas (CSIC), Bioinformatics and Functional Genomics Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Gene coexpression analyses provide information of co-regulated genes throughout different conditions, making possible to identify transcriptional relationships between genes. Such relationships may be related to common gene evolution and usually correspond to involvement in common functional processes at cellular or physiological level. Microarray technology mesuring genome-wide expression has been shown very suitable for coexpression studies. In order to explore the coexpression landscape of human genes in a systemic way, covering most of the human body transcription events, we develop a bioinformatic analysis using a large set of microarrays from different human tissues. Within this set, we select samples from 32 different tissues to keep a balanced body-wide coverage, each one including 3 biological replicates. This selection provided a perfect clustering in a non-supervised test. Next, coexpression between all possible gene pairs was performed in two ways by combination of parametric/non-parametric signal- and correlation- algorithms: MAS5-Spearman and RMA-Pearson. This provides a robust collection of binary gene-to-gene correlations that allows the construction of a gene coexpression network. In parallel to this approach we analyse all ESTs data from UniGene libraries to build a database reflecting the presence/absence of ESTs per gene and tissue. In this way, both the microarray data and the EST data allow to know if a pair of genes is coexpressed in a given tissue. Finally, we analyse both datasets to identify the human house-keeping and tissue-specific genes. We compare our results with two existing datasets previously published, and we locate both types of genes in the coexpression network.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G11</strong></td>
 </tr> <tr> 
  <td><b>Functional Annotation of Regulatory Variants: A Systems Biology Approach to Translational Bioinformatics</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Konrad Karczewski <i>Stanford University</i></td>
 </tr>
 <tr> 
  <td>  Joel Dudley (Stanford University, Systems Medicine); Nicholas Tatonetti (Stanford University, Biomedical Informatics); Stephen Landt (Stanford University, Genetics); Atul Butte (Stanford University, Systems Medicine); Michael Snyder (Stanford University, Genetics); Russ Altman (Stanford University, Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Many genomic variants are discovered outside of genes, where their functional consequences are more difficult to characterize. However, as many of these variants are associated with disease, it is likely that they affect molecular physiology at the level of gene regulation. We investigate the role of variants in regulatory regions, on both transcription factor cooperativity as well as disease pathophysiology. First, we developed the Allele Binding Cooperativity (ABC) test and the ALPHABIT pipeline, which utilizes variation in transcription factor binding among individuals to discover combinations of factors and their targets. Second, we demonstrate a systematic approach to combine disease association, transcription factor binding, and gene expression data to assess the functional consequences of variants associated with hundreds of human diseases. In this systematic approach, we close a major loop in biological context-free association studies and assign putative function to many disease-associated SNPs. In this way, we apply findings from systems biology in a translational approach.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G12</strong></td>
 </tr> <tr> 
  <td><b>Genomes and Evolutionary Analysese of Robust Oil Production in Microalgae</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kang Ning <i>Qingdao Institute of Bioenergy and Bioprocess Technology, Chinese Academy of Sciences</i></td>
 </tr>
 <tr> 
  <td>  Dongmei Wang (Qingdao Institute of Bioenergy and Bioprocess Technology, Chinese Academy of Sciences, Functional Genomics); Jian Xu (Qingdao Institute of Bioenergy and Bioprocess Technology, Chinese Academy of Sciences, Functional Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Microalgae are considered as promising biodiesel feedstock. However the genetic diversity and genomic evolution of microalgal traits underlying robust oil production remain ill defined. In this work, we have studied seven-member phylogenome of Nannochloropsis, a phylogenetic distinct group of unicellular microalgae capable of rapid growth and robust neutral-lipid production in large-scale cultivation. <br><br>We have first assembled the draft genome of these seven Nannochloropsis from next generation sequencing (both 454 and Solexa platforms), with results indicating their small genome size (25~40M bp). To explore the evolutionary history of Nannochloropsis, we compared its proteome to a serials of evolutionarily representative organisms, Arabidopsis (land plant), Cyanidioschyzon (red algae), Chlamydomonas (Green algae) separately. The phylogenetic analyses indicate Nannochloropsis’ unique evolutionary position in microalgae as well as in eukaryote. Additionally, we have analyzed the horizontal gene transfer (HGT) events within the seven Nannochloropsis strain, and discovered a couple of HGT events introduced in different stages (before the division of Nannochloropsis and after that). Furthermore, the analyses on protein-coding genes have shown that the biological process “response to stress” have highest non-synonymous/synonymous ratio, which is consistent with Nannochloropsis’ dynamic phenotype under different conditions (highlight, low nitrogen, etc.). <br><br>The seven-member phylogenomes, the only microalgal phylogenome and Eustigmatophyceae genomes available so far, revealed for the first time a comprehensive genomic hierarchy of an eukaryotic algal genus. Our research showed the striking diversity of oil producing modes in microalgae and provided novel sequence-based and functional markers for high-throughput genotype-based screening and development of oil-production traits in microalgae.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G13</strong></td>
 </tr> <tr> 
  <td><b>Orange Bioinformatics: Data Analytics through Visual Programming</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marko Toplak <i>University of Ljubljana</i></td>
 </tr>
 <tr> 
  <td>  Ales Erjavec (University of Ljubljana, Faculty of computer and information science); Janez Demsar (University of Ljubljana, Faculty of computer and information science); Minca Mramor (University of Ljubljana, Faculty of computer and information science); Crtomir Gorup (University of Ljubljana, Faculty of computer and information science); Gregor Rot (University of Ljubljana, Faculty of computer and information science); Gad Shaulsky (Baylor College of Medicine, Department of Molecular and Human Genetics); Tomaz Curk (University of Ljubljana, Faculty of computer and information science); Blaz Zupan (University of Ljubljana, Faculty of computer and information science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Data analysis is becoming the bottleneck of scientific discovery. Vast amounts of data generated by high-throughput experiments need to be analysed in context of internal or publicly available data and knowledge, and background knowledge can aid analysis immensely.<br><br>Orange Bioinformatics framework (http://orange.biolab.si) aims to decrease the barrier between the experimentalist and the data, reducing the time from data acquisition to the first biomedical insights gained through visual analytics. The experimentalist works in a visual programming environment where computational units called widgets are connected into a data flow schema. Orange widgets are high level operations or visualizations, specially tailored to the needs of molecular biologists. For instance, they implement gene selection, enrichment analysis, or exploration of genes from the data set in KEGG pathways. They also provide access to publicly available data, like GEO data sets, Biomart, GO, KEGG, Atlas, and ArrayExpress. While most widgets are specific enough to be used independently, the real magic lies in their interconnection. Furthermore, the experimentalist can directly, in the same interface, use powerful visualization, network exploration and data mining techniques from the Orange data mining framework.<br><br>Orange widgets are crafted in Python, and could in principle combine other libraries, like SciPy and BioPython. Widgets interfacing to R are in development. The framework is freely available and open source.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G14</strong></td>
 </tr> <tr> 
  <td><b>Analysis of protein functional sites encoding features</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Irina Medvedeva <i>Institute of Cytology and Genetics SB RAS</i></td>
 </tr>
 <tr> 
  <td>  Vladimir Ivanisenko (Institute of Cytology and Genetics SB RAS, Sector of Computer Proteomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Earlier we constructed the computer system SitEx (http://www-bionet.sscc.ru/sitex/) that includes database mapping the protein functional sites (FS) positions on the exon-intron structure of encoding gene and BLAST and 3DExonScan search. Testing this system on the 955 proteins that possess known PDB structures reveal that exons encoding FS are more likely to be found and are more conservative than those not encoding.<br><br>We have shown that the FS discontinuity through exon structure is significantly less than expected by chance and the protein FS tends to be encoded by single or neighboring exons. Than we analyzed codon usage and discovered that the exons coding FS amino acids on the 5`-end possess the less optimal codon usage. We also found that the frequency of codons encoding FS on the exon border in phases 1 and 2 is significantly greater than it is for the others. It could be the result of the unification of the genes that encode the single FS.<br><br>The extended version contains the information about SNP in FS, and the FS positions projection on homologous and paralogous genes. Based on this information we found that SNP frequency in functional sites amino acids is less than in surroundings. We observed the same effect analyzing FS area and protein domain area. The analysis of physical and chemical properties of the amino acid changes in SNP reveals that acidity changes more often that it could be expected by chance and the mass and volume are changing quite significantly more often that it could be expected.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G15</strong></td>
 </tr> <tr> 
  <td><b>Improved hit selection in phenotypic screens by combining the information about phenotype strength and specificity</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nikolay Samusik <i>MPI-CBG</i></td>
 </tr>
 <tr> 
  <td>  Martin Stoeter (Max Planck Institute of Cell Biology and Genetics, TDS); Michael Vanlandewijck (Ludwig Institute for Cancer Research, Uppsala Branch); Aristidis Moustakas (Ludwig Institute for Cancer Research, Uppsala Branch); Yannis Kalaidzidis (Max Planck Institute of Cell Biology and Genetics, -); Marino Zerial (Max Planck Institute of Cell Biology and Genetics, -); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Image-based multiparametric RNAi and chemical screens became an important tool in systems biology and drug discovery. A typical output of a screen is a set of multidimensional vectors normalized to the negative control, so that the length of the vector corresponds to the phenotype strength, and the direction corresponds to the kind of the phenotype. The standard hit selection approach, chi2, takes only vector length into account, omitting the phenotype direction. However the number of observed phenotypes is typically limited, vectors tend to group around few distinct phenotypic directions. These groups can be identified by clustering and cluster centers would estimate the phenotypic directions. We propose a model in which every measured vector is a linear combination of a specific phenotypic component that is collinear to the center of a respective cluster, and a normally distributed noise component with covariance matrix that is estimated from negative control set. Since the multidimensional space is sparse, it is unlikely that a random noise vector with null specific phenotype component will fall close to one of the phenotypic directions. Based on this, we propose a novel scoring measure Phenoscore that is as a probability that the length of the vector specific phenotypic component (collinear to cluster center) is higher than the expected length of a random noise vector on this given direction. We validate the performance of Phenoscore on the dataset of chemical screen for TGFb inhibitors and show up to 70% improvement of ROC AUC at FPR <1%, compared to chi2.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G16</strong></td>
 </tr> <tr> 
  <td><b>An open source library for miRNA target prediction with a novel comparative genomics approach</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Charles Vejnar <i>University of Geneva</i></td>
 </tr>
 <tr> 
  <td>  David Gatfield (University of Lausanne, Biology); Ueli Schibler (University of Geneva, Biology); Evgeny Zdobnov (University of Geneva, GEDEV); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Post-transcriptional regulation via microRNA (miRNA) contributes to fundamental processes such as cell growth, development, and differentiation, yet miRNA target prediction remains a challenging problem. As there is no publicly available complete target prediction software, we implemented three approaches to predict miRNA targets in an open source library covering the full range of published prediction models. These approaches include the thermodynamics of RNA-RNA interactions, the target site sequence features (i.e. the TargetScan context score), and a probabilistic model. Additionally we introduced a novel comparative genomics approach.<br><br>Our novel comparative genomics approach measures differences in substitution rates in miRNA target sites that test for alternative modes of selection pressure (acceleration or conservation) on these target sites. This quantification of evolutionary pressures, together with target sites functionality prediction, and with miRNA age estimates, we have gained a new insight into miRNA regulatory network evolution.<br><br>Our all-in-one implementation of miRNA target prediction (manuscript in preparation) allowed us to compare these methods and test their efficiency without any mapping biases (as prediction sets are usually produced by different research groups). This comparison was done within the framework of knocking down an endogenously highly expressed miRNA (miR-122 molecules represent about 70% of miRNAs in hepatocytes), and with published expression data on mRNA and protein levels. Together with our open source target prediction software, this study enables the research community to efficiently use all state-of-art target prediction methods.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G17</strong></td>
 </tr> <tr> 
  <td><b>From data to disease: Identification of genes with cell lineage specific expression in the human kidney</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Casey Greene <i>Princeton University</i></td>
 </tr>
 <tr> 
  <td>  Wenjun Ju (University of Michigan, Department of Nephrology); Felix Eichinger (University of Michigan, Department of Nephrology); Matthias Kretzler (University of Michigan, Department of Nephrology); Olga Troyanskaya (Princeton University, Lewis-Sigler Institute for Integrative Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Knowledge of tissue-specific gene action is critical to understanding the biology of complex metazoans such as humans.  For many human tissues, cell lineage specific high throughput experiments are infeasible due to the difficulty of obtaining such samples and genomic aberrations caused by immortalization processes.  We have developed an iterative machine learning methodology for discovering cell-lineage specific gene expression from high-throughput non-cell-lineage resolved data.  We have implemented this methodology in the PILGRM (Platform for Interactive Learning by Genomics Results Mining) system and applied this strategy to the discovery of genes with kidney podocyte (visceral epithelial cells) specific expression patterns.  Podocytes are important cells in the kidney glomerulus that play a key role in the kidney’s function.  Mutations or alterations of podocyte specific proteins underlie the hereditary proteinuric syndromes and many acquired glomerular diseases. Identification of novel podocyte-specific genes will help us understand the regulation of physiological properties of the podocyte and the mechanisms of its cellular response to disease or injury.  We use multiple validation strategies including high throughput experiments in model organisms and low throughput (double immunofluorescence of human kidney biopsies and immunohistochemical staining) approaches.  We show that our predictions are highly enriched for podocyte-specific genes.  Furthermore, genes that we predict to be podocyte expressed are associated with important clinical outcomes including glomerular flow rate, an important clinical measure of kidney function.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G18</strong></td>
 </tr> <tr> 
  <td><b>Identification and classification of protein subfamilies using top-down phylogenetic tree reconstruction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Eduardo Costa <i>Katholieke Universiteit Leuven</i></td>
 </tr>
 <tr> 
  <td>  Celine Vens (Katholieke Universiteit Leuven, Department of Computer Science); Hendrik Blockeel (Katholieke Universiteit Leuven, Department of Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Identifying protein subfamilies is an important step in gene function prediction, which remains a challenge in the post-genomic era. Phylogenomic analysis has been used to tackle this task, as an alternative to homology-based methods. Current phylogenomic methods first build a complete phylogenetic tree, then extract clusters from it that hopefully correspond to protein subfamilies. We propose a novel phylogenomic method that differs from the existing methods in two important ways: (1) it builds the phylogenetic tree top-down, rather than bottom-up, and stops when clusters are found; thus it avoids constructing the whole tree; (2) it associates particular mutations with each division into subclusters. The novel method identifies subfamilies with comparable accuracy to that of existing methods, but is much more efficient (and can therefore identify subfamilies in much larger sets of proteins), and identifies key mutations by which different subfamilies can be recognized, allowing easy classification of new proteins.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G19</strong></td>
 </tr> <tr> 
  <td><b>Exploring muscle hypertrophy transcriptional signature through knowledge-based gene association networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Francesca Mulas <i>University of Pavia</i></td>
 </tr>
 <tr> 
  <td>  Angelo Nuzzo (University of Pavia, Centre for Tissue Engineering); Flavio Ronzoni (University of Pavia, Di.Me.S., Human Anatomy Dept.); Maurilio Sampaolesi (University of Pavia, Di.Me.S., Human Anatomy Dept.); Blaž Zupan (University of Ljubljana, Faculty of Computer and Information Science); Bellazzi Riccardo (University of Pavia, Dipartimento di Informatica e Sistemistica); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: With the increasing number of biological knowledge bases, a variety of approaches have been developed in computational biology for the inclusion of prior knowledge in the data analysis process. For instance, various functional gene annotations contained in biological repositories, such as the Gene Ontology and the biomedical literature, can be exploited to visualize gene similarities by means of graphs called gene association networks. These networks represent genes in nodes and connect a pair of genes if their annotation profile is similar. We present a procedure for the construction of association networks for a set of 'focus genes' based on their annotations in Gene Ontology and the Medical Subject Headings from NCBI's PubMed. While most of the proposed strategies associate genes on the basis of co-citation in the publications or shared biological functions, our method uses approaches from text mining to characterize each gene by highlighting its most significant annotation terms. <br>The proposed procedure has been applied to a transcriptional analysis of myogenic precursor cells derived from wild type and Magic-F1 (met activating genetically improved chimeric factor 1) transgenic mice. MAGIC-F1 is an engineered protein that was previously demonstrated to induce muscle hypertrophy. Starting from a list of muscle specific genes and muscle growth regulators, we obtained networks that emphasize the results of global transcriptome analysis for the focus genes and their annotation-neighbours. In addition our strategy allowed us to identify novel gene candidates potentially involved in muscle hypertrophy.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G21</strong></td>
 </tr> <tr> 
  <td><b>Analyzing high resolution nucleosome structure by DNase-seq</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Deborah Winter <i>Duke University</i></td>
 </tr>
 <tr> 
  <td>  Lingun Song (Duke University, IGSP); Sayan Mukherjee (Duke University, IGSP); Terrence Furey (University of North Carolina, Chapel Hill, Department of Genetics); Gregory Crawford (Duke University, IGSP); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The majority of the human genome is wrapped around histone proteins to form nucleosomes that can be packaged into higher-order structures. These chromatin structures contribute to the compaction of DNA within the nucleus and their arrangement plays a role in gene transcription. Previously, we used the high-throughput assay, DNase-seq, to identify DNaseI hypersensitive sites (DHS), or nucleosome depleted regions, across the genome. Over 30 years ago, gel electrophoresis showed that DNaseI tends to digest DNA, outside of DHS, with approximately 10.4bp periodicity: this matches the bending of the DNA helix around the histone octamer. We detected this pattern even in aggregate DNase-seq data from multiple cell types, suggesting precise nucleosome positioning is relatively conserved. From this high-resolution data, we discovered periodicity within individual loci and located regions of stable nucleosome organization. As anticipated, Fourier analysis established that this data set is dominated by 10-11bp signals. We confirmed these findings across the genome through a convolution operation where fittings representing the 10.4bp pattern demonstrated better agreement than fittings with alternatively spaced patterns. When we equivalently analyzed data from FAIRE-seq – another high-throughput assay for open chromatin but whose detection model we expect would be independent of nucleosome position – we did not observe similar results. On a broader level, we can identify higher-order compaction, such as 30nm fibre, by exploring the pattern of nucleosomes exposed to DNA digestion. Our goal is to use this data to understand the 3D organizational program of chromatin and its impact on regulation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G22</strong></td>
 </tr> <tr> 
  <td><b>Estimating differential expression of transcripts with RNA-seq by using Bayesian Inference</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Peter Glaus <i>The University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  Antti  Honkela (University of Helsinki , Helsinki Institute for Information Technology HIIT); Magnus Rattray (The University of Sheffield, The Sheffield Institute for Translational Neuroscience); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: High-throughput sequencing enables expression analysis at the level of individual transcripts. <br>The analysis of transcriptome expression levels and differential expression estimation requires a probabilistic approach to properly account for the ambiguity caused by shared exons and finite read sampling as well as the intrinsic biological variance of transcript expression.<br>Another important factor are the biological sources of variance, which, as we show in our analysis, can be substantial and may dependent on the transcript expression level.<br>To avoid false positive differential expression calls, one has to anticipate the intrinsic variance of the transcript expression levels using empirical prior knowledge and information from replicates where they exist.<br><br>We present a Bayesian approach for estimation of transcript expression level from RNA-seq experiments. <br>Inferred relative expression is in the form of a probability distribution represented by samples of the distribution obtained from a Markov chain Monte Carlo inference method applied to a generative model of the read data.<br>Additionally to implementation of the regular Gibbs sampling algorithm, we provide a comparison with Collapsed Gibbs sampling in which some of the parameters are marginalized in order to obtain faster convergence.<br><br>We propose a novel method for differential expression analysis across replicates which propagates uncertainty from the sample-level model while modelling biological variance using an expression-level dependent prior.<br>We demonstrate the advantages of our method using a RNA-seq dataset (Xu G. et al., RNA 2010) with technical and biological replication for both studied conditions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G23</strong></td>
 </tr> <tr> 
  <td><b>ARH-Seq: Alternative Splicing Robust Prediction by Entropy on RNA-Seq data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Axel Rasche <i>MPI Molecular Genetics</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Several experimental platforms for the analysis of alternative splice variants exist such as RNA-Seq, microarrays, and EST libraries. Advanced high-throughput technologies demand adequate methods for the analysis of the generated data. However development of the statistical analysis still lags behind the progress in analysis techniques. Especially method performance has not been evaluated systematically for different methods and technologies. This is surprising as splicing prediction poses a stronger challenge than differential expression due to different expression levels for exons and genes and a coupling is proposed for transcription and splicing.<br>The focus is on de novo differential splicing predictions allowing the detection of splicing events not known from the transcript libraries as possible in diseases. A statistical model is developed to analyse splicing events based on a modified entropy function (ARH - Alternative splicing Robust prediction by Entropy) in order to judge significance of alternative splicing. This method covers different technologies like transcriptome sequencing or exon array data sets. Especially the RNA-Seq data allows to integrate different information ressources like exon and junction expression. Rather than combining separate predictions on the two ressource, ARH-Seq is able to fuse the measurements of both aspects.<br>The major drawback in developing and comparing new splicing prediction methods is the lack of controlled data sets. Results from alternative splicing prediction methods are compared on different Illumina sequencing tissue data sets with fixed preprocessing and confirmed splicing events from manually curated ressources.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G24'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G24</strong></td>
 </tr> <tr> 
  <td><b>Expression analysis of intronic non-coding RNAs revealed novel androgen-regulated ncRNAs in prostate cancer cell line</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Vinicius Maracaja-Coutinho <i>Universidade de São Paulo</i></td>
 </tr>
 <tr> 
  <td>  Felipe Beckedorff (Universidade de São Paulo, Departamento de Biquímica, Instituto de Química); Murilo Amaral (Universidade de São Paulo, Departamento de Biquímica, Instituto de Química); Yuri Moreira (Universidade de São Paulo, Departamento de Biquímica, Instituto de Química); João Setubal (Virginia Tech, Virginia Bioinformatics Institute); Eduardo Reis (Universidade de São Paulo, Departamento de Biquímica, Instituto de Química); Sergio Verjovski-Almeida (Universidade de São Paulo, Departamento de Biquímica, Instituto de Química); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Recent evidence shows that non-coding RNAs (ncRNAs) transcribed from intronic and intergenic genomic regions play important roles in transcriptional regulation. We applied here next-generation sequencing and a custom-built strand-specific 44k intronic-exonic oligoarray to reveal the patterns of expression of non-coding and protein-coding transcripts of LNCaP prostate cancer cell line with or without androgen stimulus. The 454 RNA-seq showed that most of intronic ncRNAs (56%) identified in both androgen-treated and control cells are novel ESTs. We identified a set of 69 intronic non-coding transcripts and 1,416 protein-coding genes differentially regulated by androgen (FDR<5%). We also explored LNCaP cells expression profile with the 44k oligoarray at seven different times of exposure to androgen (t= 0, 1, 3, 6, 12, 18 and 24 hours). This analysis reveals 507 intronic non-coding transcripts and 2,783 protein-coding genes differentially expressed (FDR<5%) over time under androgen treatment in comparison to vehicle-treated cells. Out of the 69 RNA-seq differentially expressed transcripts, 51 (73%) were represented by intronic contigs probed on the oligoarray. Of these, 39 (76%) were identified as expressed in both approaches. Finally, when cross-referenced to public datasets of epigenetic markers for active chromatin (H3K4me1, H3K4me2, H3K4me3) and androgen receptor motifs identified by ChIP-seq, a set of 291 (57%) intronic transcripts, out of 507 identified by our oligoarray analysis, showed enrichment (p-value<0.05) of these marker elements in the genome within up to ~16kb upstream of the androgen-regulated ncRNAs. These results show a set of new, hormone and epigenetically-regulated intronic ncRNAs. <br>Supported by FINEP, FAPESP, CNPq and CAPES.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G25'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G25</strong></td>
 </tr> <tr> 
  <td><b>Visualizing RNA-Seq fragment alignments</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Harold Pimentel <i>University of California, Berkeley</i></td>
 </tr>
 <tr> 
  <td>  Adam Roberts (University of California, Berkeley, Computer Science); Cole Trapnell (Broad Institute and Harvard University, Pathology); Lior Pachter (University of California, Berkeley, Mathematics, Molecular Cell Biology and Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The increasing use of sequencing technology for functional genomics has spurred the development of visualization tools that allow users to view fragment alignments to reference genomes. Examples of such tools include new browsers such as IGV, and modifications to existing systems such as the UCSC browser. In terms of visualizing RNA-Seq data, these and other browsers generally employ the same approach. Fragments are distributed according to the genomic coordinates of their alignments and displayed as “tracks”, usually above transcript annotations. The information in the tracks is then the most direct way to verify estimates of transcript abundances that are based on fragment counts. The interpretation of results hinges on the ability to evaluate the quality of the alignments, and the alignments to a reference genome are also helpful in validating transcript exon- intron structures via “spliced alignments”.<br><br>We present a tool that helps in the interpretation of fragment alignments to a reference genome by coloring reads according to transcript compatibility and abundance. This reveals the likely origin of fragments, and facilitates the assessment of RNA-Seq experiments. Spurious fragment alignments can be easily identified as the fragments are assigned to transcripts by inverting a recent probabilistic assignment technique via maximization of likelihood functions whose parameters represent the transcript abundances. Errors in mapping or annotation can also be flagged for further investigation. The probabilistic assignment of fragments to transcripts requires a non-trivial computation, yet our approach is efficient while allowing for seamless integration with existing next-generation sequence visualization tools.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G26'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G26</strong></td>
 </tr> <tr> 
  <td><b>Impact of copy number alterations on gene expression of triple negative breast cancer patients</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Catalin Barbacioru <i>Life Technologies</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Triple negative breast cancer (TNBC) is characterized by the absence of expression of estrogen receptor, progesterone receptor and Her2-neu, accounting for 15% of all breast cancer diagnosis. Targeted therapies have a reasonable likelihood of improving the cure rate of early stage TNBC, and promising therapies discovered in the metastatic setting are rapidly advancing through clinical trials. We present integrated analysis of matched normal and tumor whole genome data, along with tumor transcriptome sequencing data of multiple individuals with metastatic chemo-resistant TNBC.<br><br>In each case, two independent 1.5kb Mate-pair libraries were generated for both tumor and germ-line derived genomic DNA and sequenced using SOLiD version 4.0 paired 50mers to a target of 30x depth. The tumor transcriptome was sequenced on four replicates and compared to transcriptome sequencing from ethnicity-matched population-based control hyperplastic breast tissue.  Genome analysis was performed using multiple aligners and variant callers. Transcriptome alignment was performed using Life Technologies Bioscope pipeline, and differential expression analysis was performed using EdgeR and DESeq. We prioritized annotated germline and somatic variants by integrative analysis with differential expression results. Several striking examples of intronic events correlating with either altered splicing, differential expression or allelic imbalance were observed in genes relevant to cancer treatment, suggesting that transcriptomic data may have high value in interpreting somatic events that fall outside of coding regions. Final integration of data was validated through knowledge mining and convergence of somatic events and expression.<br><br>For Research Use Only.  Not intended for any animal or human therapeutic or diagnostic use.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G27'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G27</strong></td>
 </tr> <tr> 
  <td><b>GeneProf: Integrated Analysis of High-Throughput Sequencing Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Florian Halbritter <i>University of Edinburgh</i></td>
 </tr>
 <tr> 
  <td>  Simon Tomlinson (University of Edinburgh, MRC Centre for Regenerative Medicine / Institute for Stem Cell Research); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The promise of Next-Generation Sequencing (NGS) to deliver large volumes of inexpensive data is now becoming a reality. Unfortunately, many biologists and non-specialist users can only make limited use of available data, because they lack the expertise required to analyse and interpret these data. This skills-gap is likely to become wider in future as technologies and analysis methods rapidly develop and diversify.<br><br>We sought to improve accessibility of NGS data to a diverse range of users by developing a graphical software suite, called GeneProf. GeneProf consists of two components: A Java web application front-end and a custom-built job-scheduling system operating a network of high-performance compute nodes. This architecture relieves users of the need to set up specialised software and hardware and provides intuitive, tiered access to state-of-the-art analysis methods: Novice users with a bare minimum of data analysis experience can easily and quickly construct complex pipelines using an array of step-by-step wizards, while more advanced users benefit from GeneProf's versatile workflow designer to define highly customised procedures. Computer programmers can extend the application by implementing additional workflow components using a well-documented API. Data analyses tightly couple the research data at hand with the processing workflows and all steps are fully transparent, traceable and reproducible. Completed analyses may be shared and linked in research publications.<br><br>We have used GeneProf to re-analyse a wealth of published data. All results are readily accessible and can be explored by any visitor of the website. Additionally, registered users can straightforwardly integrate public data in their own experiments.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G28'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G28</strong></td>
 </tr> <tr> 
  <td><b>Allele specific analysis of gene expression in mouse embryonic stem cell line</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Robert Ivanek <i>Friedrich Miescher Institute for Biomedical Research</i></td>
 </tr>
 <tr> 
  <td>  Dirk Schübeler (Friedrich Miescher Institute for Biomedical Research, Epigenetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The extent to which genetic variation (single nucleotide polymorphisms) may influence the transcription factor binding and chromatin structure, and thus contribute to variation in gene expression and phenotypes, is still not well understood. To address this question we performed a whole genome sequencing of the murine embryonic stem cell (ES) line, which was derived from mouse with mixed genetic background 129xC57BL6/J. By intersecting our sequencing data with publicly available data from Sanger Mouse Genomes Project we found out, that approximately one fifth of the genome of the ES cell line is heterozygous and eligible for allele specific analysis.  For the same ES cell line we generated ChIP-seq profiles of several chromatin marks and transcription factors as well as unbiased transcriptome maps using strand specific total RNA sequencing. We established an integrated approach for analysis of allele specific distribution of chromatin marks, transcription factors and their effects on gene expression levels, which could be used as guide to similar studies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G29'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G29</strong></td>
 </tr> <tr> 
  <td><b>Condition-specific networks of transcriptional responses in human immune cells from multivariate analysis of microarray data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Wanseon Lee <i>EMBL_EBI</i></td>
 </tr>
 <tr> 
  <td>  Leo  Lahti (Aalto University School of Science and Technology, Department of Information and Computer Science); Misha Kapushesky (EMBL-EBI, Functional Genomics); Johan Rung (EMBL-EBI, Functional Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The human immune system responds to stress and stimuli through a complex regulatory system, provided through a number of specialized cell types. The response pattern on the gene expression level reflects this complexity and is still not well understood, since it is largely condition- and cell specific, but only a minority of genes are expressed exclusively in any single type of immune cell. <br>We present a multivariate analysis of gene expression networks across a substantial number of cell types and conditions to investigate the regulatory mechanisms behind unique and shared responses of the immune system. Gene expression data from 30 experiments on immune cells from blood of over 600 healthy individuals were manually collected from the ArrayExpress database and additional curation of annotation was performed.<br>To investigate condition-specific behaviors of the interaction network the probabilistic NetResponse algorithm (Lahti et al. 2011) was applied. The algorithm detects subnetworks with distinct transcriptional responses in subsets of conditions. It is independent of predefined classifications for genes or conditions, providing tools for unsupervised analysis of network activation patterns. It performs an agglomerative network search, iteratively merging interacting genes with coordinated expression changes. A variational infinite Gaussian mixture model is used to quantify such dependency, and to detect and characterize the condition-specific responses in each subnetwork. <br>The results provide condition-specific activation patterns in immune cells as well as a global view of interaction networks in the immune response across diverse physiological conditions, with potential for further investigation into the function of immune system.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G30'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G30</strong></td>
 </tr> <tr> 
  <td><b>The Limpopo MAGE-TAB Parser</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tony Burdett <i>European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Joe White (Dana Faber Cancer Institute, Dana Faber Cancer Institute); Niran Abeygunawardena (European Bioinformatics Institute, Functional Genomics); Vincent Xue (European Bioinformatics Institute, Functional Genomics); Helen Parkinson (European Bioinformatics Institute, Functional Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: MAGE-TAB is a widely used tab-delimited text document format designed for exchange and submission of gene expression data.  It aims to be easy for biologists to use, create and edit and consists of a group of related text ?les that are both “human-readable” and can be constructed using a spreadsheet or text editor.<br><br>The MAGE-TAB speci?cation [1] imposes a strict series of rules on the ?les comprising a MAGE-TAB document.  These cannot be easily checked by a human reading the document or detected by a simple text parser; programmatic validation is required. To address these challenges we present the Limpopo MAGE-TAB parser, a standard canonical java parser for MAGE-TAB format.<br>Limpopo contains the parser, a speci?cation-compliant object model of the MAGE-TAB structure written in Java, and an API for accessing the data contained within a MAGE-TAB document, to support applications that consume MAGE-TAB.  It also includes a plugin framework for extending the parser to add custom validation rules (on top of the basic syntactic validation provided) or the ability to write data from the Limpopo object model, for example to store MAGE-TAB data in a database.<br><br>Limpopo is being used in several projects, including ArrayExpress, the Gene Expression Atlas, and Annotare.  These three projects all use validator extensions to add custom validation rules, and the first two also use the write functionality to store data in their respective databases.  <br><br>[1] Rayner, T. et al. A simple spreadsheet-based, MIAME-supportive format for microarray data: MAGE-TAB. BMC Bioinformatics 2006, 7:489Jo</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G31'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G31</strong></td>
 </tr> <tr> 
  <td><b>The Functional Genomics Software Suite</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tomasz Adamusiak <i>European Bioinformatics Institute</i></td>
 </tr>
 <tr> 
  <td>  Tony Burdett (European Bioinformatics Institute, Functional Genomics); Anna Farne (European Bioinformatics Institute, Functional Genomics); Adam Faulconbridge (European Bioinformatics Institute, Functional Genomics); Emma Hastings (European Bioinformatics Institute, Functional Genomics); Natalja Kurbatova (European Bioinformatics Institute, Functional Genomics); James Malone (European Bioinformatics Institute, Functional Genomics); Ravensara Travillian (European Bioinformatics Institute, Functional Genomics); Eleanor Williams (European Bioinformatics Institute, Functional Genomics); Helen Parkinson (European Bioinformatics Institute, Functional Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Functional Genomics Production Team at the European Bioinformatics Institute have produced a large suite of software for data submissions, working with the MAGE-TAB format, annotating data against the Experimental Factor Ontology and automating curation processes of microarray and high throughput sequencing data.  Here we present an overview of our publicly available tools.<br><br>Tab2MAGE is a series of Perl modules that facilitate data submissions to ArrayExpress.  It includes support for parsing and validating a variety of formats, including MAGE-TAB, MAGE-ML and MIAMExpress.  It also contains a lightweight process tracking system and taxon specific template generation system implemented in Ruby on Rails with a MySQL backend.<br><br>Limpopo is a MAGE-TAB parser written in Java.  It provides functionality for validating MAGE-TAB with custom rules and exporting MAGE-TAB data to any other format (including your own database).<br><br>OntoCAT is a Java library that abstracts a number of common ontology-related tasks for interacting with ontology resources, including local files and public repositories.  OntoCAT R is an R-package designed to enable ontology traversal and search in R environment.  ZOOMA is a tool that provides optimal ontology mappings for user supplied terms, facilitating automatic data annotation to preferred ontologies.<br><br>OntologyMapper is a Perl package providing fuzzy concept recognition and lexical matching based on a list of terms and any target ontology.<br><br>Bubastis is an ontology diff tool used to compare two ontologies (typically versions of the same ontology) in OWL or OBO format for logical differences in classes and identifying new or deleted classes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='G32'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster G32</strong></td>
 </tr> <tr> 
  <td><b>Novel Method for Removal of Sequence Biases in ChIp-seq Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Joanna Raczynska <i>UT Southwestern Medical Center</i></td>
 </tr>
 <tr> 
  <td>  Dominika Borek (UT Southwestern Medical Center, Biochemistry); Zbyszek Otwinowski (UT Southwestern Medical Center, Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Chromatin immunoprecipitation followed by high?throughput sequencing (ChIp?seq) is an increasingly popular method to study protein?DNA interactions in a genome?wide manner. Yet, the enrichment of sequenced fragments at a particular genomic location does not directly correspond to the desired signal (i.e. protein binding) due to various biases. These are: underrepresentation of sequences having extreme GC?composition (either GC or AT?rich) or uneven base frequencies at particular positions within the sequenced fragments. In some ChIp?seq datasets this effect can be very large leading to uninterpretable results or worse, false biological conclusions. Therefore, correction of the biases is necessary for any meaningful interpretation, especially when looking for correlations between different datasets.<br>Some effort has already been made to account for the described effects [Schwartz et al, 2011, PLoS One 6(1)]. However, none of the existing procedures takes into account the base frequencies at individual positions outside of the sequenced fragment which, as we have found, also significantly influence the bias.<br>We developed a new method to normalize ChIp?seq data with respect to the reference genome. It takes into account the general base composition (like GC?content) of each read together with its genomic context. Also, it accounts for different base frequencies at individual positions relative to the starting position of a read, not only within the actual read, but also outside of it. The method is based on a Poisson?regression procedure to find a set of coefficients that then allow for calculation of weights for each read based on the sequence of its genomic context.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




