
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=W by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:27 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlistdc81.html?cat=W by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:19 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'W'- Structure and Function Prediction'</b></td>
 </tr>
</table>

<a name='W01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W01</strong></td>
 </tr> <tr> 
  <td><b>A Time-Dependent Framework for Solving the Degeneracy Problem in RNA Secondary Structure Deleterious Mutation Prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Danny Barash <i>Ben-Gurion University</i></td>
 </tr>
 <tr> 
  <td>  Alexander Churkin (Ben-Gurion University, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The problem of predicting which single point mutation in a given RNA sequence will induce a conformational rearrangement in its secondary structure may often encounter multiple solutions. The simultaneous integration of the chemical master equation by the Gillespie algorithm for RNA mutant sequences in realistic times may solve this problem.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/83/19881.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W02</strong></td>
 </tr> <tr> 
  <td><b>Correcting protein alignment errors using a novel multiple-template threading method</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jinbo Xu <i>Toyota Technological Institute at Chicago</i></td>
 </tr>
 <tr> 
  <td>  Jian Peng (Toyota Technological Institute at Chicago, NA); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Template-based protein modeling (TBM) suffers greatly from alignment errors especially when the sequence and template are distantly-related. We present a novel multiple-template threading method to address this problem, which employs a novel probabilistic-consistency algorithm to accurately thread a single sequence simultaneously to multiple templates. Multiple-template methods have been studied by several groups before, but none has showed significant improvement in alignment accuracy through a large-scale blind test (e.g., CASP). Our CASP9 results indicate that 1) using multiple templates we can significantly improve alignment accuracy; 2) our multi-template models consistently excel models built from the best single templates; and 3) without an accurate alignment modeling improvement is not guaranteed by simply using multiple templates to increase alignment coverage. According to the CASP9 official evaluation, our method generated the best alignments for the 50 hardest TBM targets. Our method can also be applied to alignment of multiple protein/RNA sequences and structures.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20041.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W03</strong></td>
 </tr> <tr> 
  <td><b>All-atom knowledge-based potential for RNA structure prediction and assessment</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Emidio Capriotti <i>Stanford University</i></td>
 </tr>
 <tr> 
  <td>  Tomas Norambuena (Pontificia Universidad Católica de Chile, Departamento de Genética Molecular y Microbiología); Marc A. Marti-Renom (Centro de Investigacion Principe Felipe, Department of Bioinformatics and Genetics); Francisco Melo (Pontificia Universidad Católica de Chile, Departamento de Genética Molecular y Microbiología); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The vision that RNA simply serves as information transfer molecules has dramatically changed. The study of the sequence/structure/function relationships in RNA is becoming more important.  Therefore, new methods for assessing the accuracy of RNA structure models are needed. We introduce an all-atom knowledge-based potential for the assessment of RNA three-dimensional structures. We have benchmarked our new potential, called RASP, with two different decoy datasets composed of near-native RNA structures. In one of the benchmark sets, RASP was able to rank the native structure as the best and within the top 10 models for ~93% and ~95% of decoys, respectively. The average correlation coefficient between model accuracy, calculated as the RMSD and GDT-TS measures of C3’ atoms, and the RASP score was 0.85 and 0.89, respectively. Based on a recently released benchmark dataset RASP scoring function compared favorably to previously developed methods in the selection of accurate models.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20083.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W04</strong></td>
 </tr> <tr> 
  <td><b>A structural and dynamical model of human telomerase</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Samuel Flores <i>Uppsala University</i></td>
 </tr>
 <tr> 
  <td>  Christina Waldsich (University of Vienna, MFPL); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As described below, the human Telomerase complex has tremendous consequences in human health and longevity. In this presentation I will show how we predicted the structure of much of the complex from fragments crystallized from other organisms, and by combining biochemical evidence using RNABuilder, our recently completed modeling code.  I will also show how we combined further evidence on the dynamics of telomere elongation to create an exciting movie of the process.  I discuss the implications of the model and propose focused experiments for further validation.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20086.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W05</strong></td>
 </tr> <tr> 
  <td><b>Optimization of residual electrostatic environment to enhance protein thermal stability</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chi-Wen Lee <i>National Chiao Tung University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Electrostatic interaction is known to be one of the most important factors to affect protein thermal stability. This related research is not systemically used to apply the thermal stable protein engineering. Here we present a new computational method to optimize the electrostatic interaction by predicting mutation to increase the thermal stability, which are re-engineered by experimental mutagenesis. The basic idea is to create the novel salt bridge in the proteins to enhance the thermal stability. We applied our method to ?-glucosidase A, which is a glycoside hydrolase that catalyzes the hydrolytic cleavage of the glycosidic bonds, and were able to produce more of stable variants with   up to 15.8 ºC. The finial results indicate that the inter-subunit of the electrostatic interaction is more important than the intra-subunit interface to enhance the thermal stability. This method is highly efficient and easy to implement that can provide a useful tool to design more thermostable mutants, and extensive apply to find the hot spot residues that responsible for enhancing thermostability of protein-protein interaction via oligomerization.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W06</strong></td>
 </tr> <tr> 
  <td><b>Complex networks govern coiled coil oligomerization - predicting and profiling by means of a machine learning approach</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ulrich Bodenhofer <i>Johannes Kepler University</i></td>
 </tr>
 <tr> 
  <td>  Carsten Mahrenholz (Charite Medical School, Institute of Medical Immunology); Ingrid Abfalter (Johannes Kepler University, Institute of Bioinformatics); Rudolf Volkmer (Charite Medical School, Institute of Medical Immunology); Sepp Hochreiter (Johannes Kepler University, nstitute of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Understanding the relationship between protein sequence and structure is one of the great challenges in biology. In the case of the ubiquitous coiled coil motif, structure and occurrence have been described in extensive detail, but there is a lack of insight into the rules that govern oligomerization, i.e., how many alpha-helices form a given coiled coil. To shed new light on the formation of two- and three-stranded coiled coils, we developed a machine learning approach to identify rules in the form of weighted amino acid patterns. These rules form the basis of our classification tool PrOCoil, which also visualizes the contribution of each individual amino acid to the overall oligomeric tendency of a given coiled coil sequence. We discovered that sequence positions previously thought irrelevant to direct coiled coil interaction have an undeniable impact on stoichiometry. Our rules also demystify the oligomerization behavior of the yeast transcription factor GCN4, which can now be described as a hybrid - part dimer and part trimer - with both theoretical and experimental justification.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/19955.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W07</strong></td>
 </tr> <tr> 
  <td><b>Combining inference and integration for multiple ranked data from genomic studies</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Michael Schimek <i>Medical University of Graz</i></td>
 </tr>
 <tr> 
  <td>  Alena Mysickova (Max Planck Institute for Molecular Genetics, Bioinformatics); Eva Budinska (Swiss Institute of Bioinformatics, Lausanne, Bioinformatics Core Facility); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Current experimental molecular research typically produces a large amount of data from rather small studies, many of them addressing similar questions. Public data repositories make such results easily accessible. A key problem is the combination of data across studies that differ in characteristics such as laboratory technology, quantification of measurements, and size. Only highly conforming subsets of these data are relevant for integration. Here, rank-based methods are most useful. We introduce an approach to deal with several rankings of the same set of genes or proteins that combines the estimation of the lengths of informative sublists with their stochastic aggregation. The goal is to obtain a smaller set of consolidated genes or proteins in a new rank order. Our approach does not only allow for huge data, but also for irregular and incomplete rankings due to random and missing assignments. Its advantages are exemplified on the metaanalysis of microarray experiments.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20088.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W08</strong></td>
 </tr> <tr> 
  <td><b>The imprint of codons on protein structure</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Charlotte Deane <i>Oxford University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The central dogma of molecular biology describes the unidirectional flow of interpretable data from genetic sequence to protein sequence. This has led to the idea that a protein’s structure is dependent only on its amino acid sequence. Analysing the input (mRNA) and output (protein) of translation, we find that local protein structure information is encoded in the mRNA nucleotide sequence. Using a detailed mapping between over 4000 solved protein structures and their mRNA we have carried out a comprehensive analysis of codon usage across many organisms. We found no evidence that domain boundaries are enriched with slow codons. In fact, genes seemingly avoid slow codons around structurally defined domain boundaries. Translation speed, however, does decrease at the transition into secondary structure. These results support the premise that codons encode more information than merely amino acids and give insight into the role of translation in protein folding.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20120.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='W09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W09</strong></td>
 </tr> <tr> 
  <td><b>PyRy3D: a software tool for modelling of large macromolecular complexes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Joanna Kasprzak <i>Adam Mickiewicz University in Pozna?</i></td>
 </tr>
 <tr> 
  <td>  Wojciech Potrzebowski (International Institute of Molecular and Cell Biology in Warsaw, ul. Ks. Trojdena 4, 02-109 Warsaw, POLAND , Laboratory of Bioinformatics and Protein Engineering); Janusz M. Bujnicki (International Institute of Molecular and Cell Biology in Warsaw, ul. Ks. Trojdena 4, 02-109 Warsaw, POLAND , Laboratory of Bioinformatics and Protein Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One of the major challenges in structural biology is to determine the structures of macromolecular complexes and to understand their function and mechanism of action. However, compared to structure determination of the individual components, structural characterization of macromolecular assemblies is very difficult. To maximize completeness, accuracy and efficiency of structure determination for large macromolecular complexes, a hybrid computational approach is required that will be able to incorporate spatial information from a variety of experimental methods (like X-ray, NMR, cryo-EM, cross-linking and mass spectrometry, etc.) into modeling procedure. For many biological complexes such an approach might become the only possibility to retrieve structural details essential for planning further experiments e.g. in order to explain mechanism of action.<br>	We developed PyRy3D, a method for building and visualizing low-resolution models of large macromolecular complexes. The components can be represented as rigid bodies (e.g. macromolecular structures determined by X-ray crystallography or NMR, theoretical models, or abstract shapes) or as flexible shapes (e.g. disordered regions or parts of protein or nucleic acid sequence with unknown structure). Spatial restraints are used to identify components interacting with each other, and to pack them tightly into contours of the entire complex (e.g. cryoEM density maps or ab initio reconstructions from SAXS or SANS methods). Such an approach enables creation of low-resolution models even for very large macromolecular complexes with components of unknown 3D structure. Our model building procedure applies Monte Carlo approach to sample the space of solutions fulfilling experimental restraints.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W10</strong></td>
 </tr> <tr> 
  <td><b>Planar shape representation based Protein function prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Heng Yang <i>Drexel Univerisity</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We present a protein function prediction method which flattens 3D protein structures into 2D planes. The 2D planar grid encodes the 3D protein surface from different projection angles.  The Dimension Reduction Method (DRM) is used to reduce the dimensionality of a given protein.  28 different DRMs are applied to each 3D protein structure and evaluated by a voting system to pick the best reduction result.<br>The method is divided into 3 parts: 3D protein surface generation, dimension reduction, and surface alignment.   The 3D protein surface is generated for a given PDB file and the vertex coordinates are scanned into a matrix.  Electrostatic properties are assigned to the vertices as color values with the help of APBS tool.  The surface matrix is fed to the DRM system and the results are ranked.  Our method requires 20 planes.  Grid points on the projected planes represent vertices of the original protein surface, and the color value of each point indicates the vertex’s electrostatic property.  Corresponding planes between the test protein and the known protein are compared to detect the surface structure similarity of the two proteins.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W11</strong></td>
 </tr> <tr> 
  <td><b>The conserved structures of RNA-dependent RNA polymerases</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Heli Mönttinen <i>University of Helsinki</i><br></td>
 </tr>
 <tr> 
  <td>Minna Poranen (University of Helsinki) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA-dependent RNA polymerases (RdRp) are responsible for the replication and transcription of RNA viruses. Like DNA-dependent DNA polymerase I and RNA-dependent DNA polymerases RdRps are structurally right-handed including palm, finger and thumb motifs. The catalytic site is situated in the palm motif. In bacteriophage phi6 at a position ~6 Å from catalytic ions there is a Mn2+ binding site. This noncatalytic ion is needed for the correct coordination of nucleotides for catalysis and for the dynamic functions of the RdRp. The noncatalytic ion binding site is conserved in a number of RdRps including RdRps of poliovirus and Dengue virus. Many other RdRps are highly stimulated in the presence of Mn2+ ions. However, the noncatalytic ion has not been detected in all RdRps. This might reflect different crystallization conditions applied in solving the RdRp structures. In this project we have identified the noncatalytic ion binding sites by means of structural alignments and distance calculations. Our preliminary results suggest that the noncatalytic ion binding site is conserved in the RdRps of positive-stranded and double-stranded RNA viruses and putatively also in the reverse transcriptases of retroviruses. These results give insights into structural and functional co-evolution of RdRps. The studies of structural co-evolution will be extended by using computational methods (for instance identifying the sequence fingerprint of RdRps on the basis of structural alignment and predicting protein-ligand interactions).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W12</strong></td>
 </tr> <tr> 
  <td><b>MODORAMA - a web application for comparative protein structure modeling</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jan Kosinski <i>Sapienza University of Rome</i></td>
 </tr>
 <tr> 
  <td>  Alessandro Barbato (Sapienza University of Rome, Department of Biochemical Sciences); Anna Tramontano (Sapienza University of Rome, Department of Physics); Pascal Benkert (University of Basel, Biozentrum); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Building protein models maximally useful for practical biological analyses is still a difficult task. Thus, we developed a new modeling platform, Modorama, which both greatly facilitates critical steps in comparative modeling and allows for easy integration of biological knowledge. <br><br>In Modorama, users can either go through the full modeling procedure starting from a sequence, or discriminate between a user-defined set of alignments and templates, or just refine an existing target-template alignment. <br><br>The best modeling templates can be selected according to automatic scores as well as various structural and functional annotations using an intuitive tabular interface. This interface easily allows to search for e.g. “best scoring, best quality template solved in the presence of ATP and DNA”. <br><br>Target-template alignments can be manually refined prior to modeling. Utilizing an interactive alignment editor, users can analyze real-time changes in alignment quality without actually building a model. This alignment quality can be evaluated in a similar way to typical expert analysis, which includes evaluation of sequence conservation, secondary structure and solvent accessibility agreement, or a residue-level QMEAN energy profile. <br><br>Importantly, the alignment evaluations are automatically performed also for representative target homologs, giving an estimate of how the whole target family “fits” to a given template. Also, potential errors are automatically detected and highlighted. <br><br>The interactive interfaces of Modorama such as template selection tables or the alignment editor have been implemented using JavaScript and they can be used in any browser without the need of installing any plug-ins. <br><br>Modorama is available at http://biocomputing.it/Modorama</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W13</strong></td>
 </tr> <tr> 
  <td><b>Mean Scaled RMSD: A Size Independent Measure For Superimposition of Protein Structures</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Seyed Shahriar Arab <i>Institute for Research in Fundamental Sciences (IPM)</i></td>
 </tr>
 <tr> 
  <td>  Rosa Aghdam (Institute for Research in Fundamental Sciences (IPM), Bioinformatics); Masoud Rahimi Ghazi Kalayeh (Institute for Research in Fundamental Sciences (IPM), Computer Science); Changiz Eslahchi (Shahid Beheshti University, Mathematics); Mehdi Sadeghi (National Institute of Genetic Engineering and Biotechnology, Biochemistry); Hamid Pezeshk (University of Tehran, Center of Excellence in Biomathematics, College of Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: To compare three-dimensional structures of two molecules, biologists mostly use the RMSD (root mean square deviation) of the atomic coordinates after optimal rigid body superimposition as the similarity measure. In the same size segments, the smaller the RMSD is, the more similar the two structures are. What is not so clear is comparing of RMSD values for segments in different lengths. <br>In this work, we introduce a new measure, mRMSD (mean-scaled RMSD) of structural similarity based on RMSD that is independent of the sizes of the segments.<br>We show how mRMSD fits can be used to identify protein domains and protein mobility. Lastly we investigate the use of mRMSD to evaluate predicted protein structures.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W14</strong></td>
 </tr> <tr> 
  <td><b>Probabilistic model for secondary structure prediction from protein chemical shifts</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Martin Mechelke <i>MPI for Developmental Biology</i></td>
 </tr>
 <tr> 
  <td>  Michael Habeck (Department of Protein Evolution, MPI for Developmental Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: (This poster is based on Proceedings Submission 142)<br>Motivation:<br>Protein chemical shifts are long known to encode valuable structural information that it is difficult and computationally costly to describe at a fundamental level. Statistical and machine learning approaches that infer the relationship between chemical shifts and secondary structure have a long tradition in biomolecular NMR. These methods range from simple chemical shift index to complex neural networks. In the most successful prediction algorithms, the relation between secondary structure and chemical shift is obscured by the complexity and missing transparency of the prediction device that often involves many parameters.<br>Results: We present hidden Markov models with Gaussian and non-Gaussian emission probabilities to model the dependence between protein chemical shifts and secondary structure. The continuous emission probabilities are modeled as conditional probabilities for given amino acid and secondary structure type. We estimated and tested several multivariate probability distributions including uncorrelated and fully correlated Gaussian distributions. To capture asymmetries and outliers in the chemical shift distributions, we also estimated multivariate Normal inverse Gaussian distributions that are more flexible than Gaussians and can account for skewness and heavy tails. Using these distributions as outputs of first and second order HMMs we achieve a prediction accuracy of 82.7% maximum, which is competitive with existing methods for predicting secondary structure from protein chemical shifts.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W15</strong></td>
 </tr> <tr> 
  <td><b>Multi-LZerD: Multiple protein docking for asymmetric complexes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Juan Esquivel-Rodríguez <i>Purdue University</i></td>
 </tr>
 <tr> 
  <td>  Daisuke Kihara (Purdue University, Biological Sciences/Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Binary and multimeric protein complexes are involved in many biological processes mediating diverse important cellular functions. The tertiary structures of protein complexes provide a crucial insight about the molecular mechanisms that regulate their functions and assembly. However, solving protein complex structures by experimental methods is often more difficult than single protein structures. Computational prediction methods are expected to make significant contributions in this area. Until now most of the protein docking prediction methods focus only on pairwise docking. There exist a handful of methods for the prediction of multimeric complexes, but almost all of them assume specific properties such as homomericity or symmetry. Considering a substantial number of multimeric complexes of diverse kinds exist in a cell, there is an urgent need for the development of a multiple protein docking prediction method that does not target a specific type of complexes. We have developed a novel multiple protein docking algorithm, Multi-LZerD, that builds models of multimeric complexes by effectively reusing pairwise docking predictions of component proteins. A genetic algorithm is applied to explore the conformational space followed by a structure refinement procedure. Benchmark on seven hetero-multimeric complexes resulted in near native conformations for all of them (a root mean square deviation smaller than 2Å). Our framework was able to predict near native structures for multimeric complexes of various topologies</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W16</strong></td>
 </tr> <tr> 
  <td><b>Modelling Protein Domain Binding to  Kinase Structures altered by Phosphorylation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Christoph Gernert <i>Helmholtz Centre for Infection Research</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 76.<br><br>In my previous work I connected experimental mass spectrometry (MS) data with a protein interaction database to recognize most likely active signaling pathways. I utilized the information gained from quantitative MS experiments and especially changes in protein modifications to highlight the "flow" of information from one protein to another. The resulting interaction network represents proteins and their interactions, reflecting known posttranslational modifications like phosphorylation. The spanned network of highlighted edges easily shows temporal or spatial changes which reveal the likely state of any signal transduction network.<br>Through mass spectrometry experiments phosphorylations at different protein kinases are detected, with no clue of their interaction partners. To check potential partners for a posttranslational modified protein kinase, an available 3D model of the protein from the Protein Data Bank (PDB) is extended by a phosphate molecule at the corresponding site. The modified structure is involved in an in silico docking process consecutively with the ten possible phosphoprotein-binding domains, 14-3-3, BRCT, C2, FHA, MH2, PBD, PTB, SH2, WD-40 and WW. In this process various combinations of the two structures are tested whether they fit together and a protein interaction is given. Proteins that include a domain which returned a positive result in the docking process are potential interaction partners.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W17</strong></td>
 </tr> <tr> 
  <td><b>Full atom comparative three-dimensional modeling of protein-DNA complexes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Francisco Melo <i>P. Universidad Catolica de Chile</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: No free computer tools are currently available to build full atom three-dimensional (3D) models of protein-DNA complexes. Existing tools are only able to model either full atom DNA duplexes (ie. 3DNA), full atom proteins (ie. MODELLER) or coarse grain protein-DNA complexes (ie. Rosetta). Among these, when duplex DNA molecules need to be modeled, Rosetta modeling software suite and 3DNA software are the most widely used. The major problem with currently exiting tools is their incapacity to properly model the DNA sugar-phosphate backbone. Unfortunately, about 70% of the atomic contacts that DNA make with proteins involve DNA atoms that belong to the sugar-phosphate backbone. Therefore, the accurate modeling of the complete DNA duplex molecule is required if the final objective is to model the 3D structure of protein-DNA complexes.<br><br>In this work, new software that has been developed in our laboratory to build full atom 3D models of protein-DNA complexes is presented. The software was written in Python and it can be described as a new module of MODELLER software. The generated 3D models are consistent with CHARMM molecular topology and thus have proper stereochemistry. A detailed description of the method is provided, as well as an assessment of its performance, along with a comparison with models produced with other software.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W18</strong></td>
 </tr> <tr> 
  <td><b>Identification of cavities on protein surface using multiple compu-tational approaches for drug binding site prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Bingding Huang <i>Techinical University of Dresden</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein-ligand binding sites are the active sites on protein sur-face that perform protein functions. Thus the identification of those binding sites is often the first step to study protein functions and structure-based drug design. There are many computational algo-rithms and tools developed in recent decades, such as LIGSITEcs/c, PASS, Q-SiteFinder, SURFNET and so on. In our previous work MetaPocket, we have proved that it is possible to combine the results of many methods together to improve the prediction result. Here we continue our previous work by adding four more methods Fpocket, GHECOM, ConCavity and POCASA to further improve the prediction success rate. The new method MetaPocket 2.0 and the individual approaches are all tested on two datasets of 48 un-bound/bound and 210 bound structures as used before. The results show that the average success rate has been raised 5% at the top1 prediction compared to previous work. Moreover, we construct a non-redundant data-set of drug-target complexes with known structure from DrugBank, DrugPort and PDB database and apply MetaPocket 2.0 to this data-set to predict drug-binding sites. As a result, more than 74% drug-binding sites on protein target are cor-rectly identified at the top 3 prediction, and it is 12% better than the best individual approach.  MetaPocket 2.0 is freely available at two web sites: http://projects.biotec.tu-dresden.de/metapocket/ and http://sysbio.zju.edu.cn/metapocket.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W19</strong></td>
 </tr> <tr> 
  <td><b>Physical contacts-based score for evaluation of protein three-dimensional models</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kliment Olechnovic <i>Vilnius University Institute of Biotechnology</i></td>
 </tr>
 <tr> 
  <td>  Eleonora Kulberkyte (Vilnius University Institute of Biotechnology, Department of Bioinformatics); Ceslovas Venclovas (Vilnius University Institute of Biotechnology, Department of Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The ability to effectively score protein structure models against the reference structure is at<br>the heart of protein structure prediction methods comparison and development. Currently,<br>Global Distance Test (GDT) and the template modeling score (TM-score), are among the most<br>popular scores. They are able to rank models of different level of completeness as is often the<br>case when different prediction methods are used. However, both GDT and TM-score suffer<br>from at least two major weaknesses. First, both scores consider only the main chain and are<br>not able to assess the side chain prediction accuracy. Second, both scores perform poorly on<br>multidomain proteins as they are based on the rigid-body comparison.<br><br>We have developed a new scoring method for the model evaluation against the reference<br>protein structure. The method is based on physical contacts derived from the Voronoi diagram<br>of atoms. Therefore, the method considers real interatomic interactions within protein structure<br>and does not use any arbitrary parameters or cutoffs. The new score assigns the model quality<br>in the {0;1} range depending on the agreement with the reference structure. For single domain<br>structures our contact-based score shows a strong correlation (over 0.9) with both GDT and<br>TM-score, at the same time providing a significantly better resolution for models having<br>relatively accurate main chain trace. Moreover, the new score does not require structure<br>superposition, and it works equally well on single-, multi-domain and even multi-chain protein<br>models. We implemented the new method both as a standalone program and as a web server.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W20</strong></td>
 </tr> <tr> 
  <td><b>A Novel Approach for Predicting Disordered Regions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Meijing Li <i>Chungbuk National University</i></td>
 </tr>
 <tr> 
  <td>  Jung Kwang Su (Korea National Institution of Health, Division of Bio-Medical Informatics); Keun Ho  Ryu (Chungbuk National University, Database/Bioinformatics laboratory); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The study of disordered reigns in proteins applying data mining techniques is recently becoming important as a research topic in Bioinformatics. Until now, a number of published predictors for these protein regions are based on various algorithms and disordered protein sequence properties. In this study, we propose a novel approach for predicting disordered protein regions using emerging subsequence mining without using the characteristics of disordered proteins. An emerging subsequence is a part of a protein sequence that has a higher frequency occurrence in target class than in source class. We first adapt the new approach to generate emerging protein subsequences on public protein sequence data. Second, the disordered and ordered regions in a protein sequence were predicted by searching generated emerging protein subsequence with sliding window, which tends to overlap. Third, the scores of the overlapping regions were calculated based on support and growth_rate values in both classes. Finally, the score of predicted regions in the target class were compared with the score of the source class, and the class having a higher score was selected.<br>In this experiment, disordered sequence data is extracted from DisProt 4.9, ordered sequence data is from PDB, both used as training data. The test data came from CASP 7 with known disordered and ordered regions. Comparing with several published predictors, the results of the experiment showed higher accuracy rate than other existing methods.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W21</strong></td>
 </tr> <tr> 
  <td><b>The influence of the local sequence environment on RNA loop structures</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dirk Walther <i>Max Planck Institute for Molecular Plant Physiology</i></td>
 </tr>
 <tr> 
  <td>  Christian Schudoma (Max Planck Institute for Molecular Plant Physiology, Bioinformatics); Abdelhalim Larhlimi (Max Planck Institute for Molecular Plant Physiology, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA folding is assumed to be a hierarchical process. The secondary structure of an RNA molecule, signified by base pairing and stacking interactions between the paired bases, is formed first. Subsequently, the RNA molecule adopts an energetically favourable 3D conformation in the structural space determined mainly by the rotational degrees of freedom associated with the backbone of regions<br>of unpaired nucleotides (loops). To what extent the backbone conformation of RNA loops also results from interactions within the local sequence context or rather follows global optimisation constraints alone has not been addressed yet. Because the majority of base stacking interactions are exerted locally, a critical influence of local sequence on local structure appears plausible. Thus, local loop structure ought to be predictable, at least in part, from the local sequence context alone. To test this hypothesis, we used Random Forests on a non-redundant dataset of unpaired nucleotides extracted from 97 X-ray structures from the Protein Data Bank (PDB) to predict discrete backbone angle<br>conformations given by the discretised ?/?- pseudo-torsional space. Predictions on balanced sets with<br>4 to 6 conformational classes using local sequence information yielded average accuracies of up to<br>55%, thus significantly better than expected by chance (17%-25%). Bases close to the central nucleotide appear to be most tightly linked to its conformation. Our results suggest that RNA loop structure does not only depend on long-range base-pairing interactions. Instead it appears that local sequence context exerts a significant influence on the formation of the local loop structure.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W22</strong></td>
 </tr> <tr> 
  <td><b>Structure Prediction of CRISPR repeats in Prokaryotic Immunity</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sita Lange <i>University of Freiburg</i></td>
 </tr>
 <tr> 
  <td>  Dominic Rose (University of Freiburg, Bioinformatics); Omer Alkhnbashi (University of Freiburg, Bioinformatics); Rolf Backofen (University of Freiburg, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Prokaryotes are known to acquire immunity against phages and viruses through a widely conserved RNA-based gene silencing pathway. Fragments of the foreign DNA are initially integrated into clusters of regularly interspaced short palindromic repeats (CRISPRs). During a new invasion these fragments are extracted from the CRISPR array as mature CRISPR-RNAs (crRNAs), which target the viral DNA for degradation. A small hairpin structure has been proposed to guide an endoribonuclease to the cleavage site and there is recent evidence of direct interaction between the stem-loop and the Cas6, one of the many CRISPR-associated (Cas) proteins. Based on sequence similarity, Kunin et al. (2007) reported 12 major families of CRISPR repeats and claimed that only six exhibit the typical hairpin motif. We have revisited structural properties of the CRISPR system on a genome-wide scale. Our results show that the hairpin motif is present in almost all families. Additionally, some sequences are misplaced and a few clusters ought to be subdivided into further families. Since repeats are able to form multiple stem-loop structures, we have developed an approach to predict the functional hairpin of a single CRISPR array by folding the repeat structures within the context of all spacer sequences. We show that the most probable hairpin in such a context is not always the minimum free energy (MFE) structure.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W23</strong></td>
 </tr> <tr> 
  <td><b>Peptides modulating conformational changes: From in silico design to preclinical proof of concept</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Itamar Borukhov <i>Compugen Ltd.</i></td>
 </tr>
 <tr> 
  <td>  Yossef  Kliger (Compugen Ltd., Research & Development); Ofer Levy (Compugen Ltd., Research & Development); Anat Oren (Compugen Ltd., Research & Development); Haim Ashkenazy (Compugen Ltd. and The Mina and Everard Goodman Faculty of Life Sciences, Research & Development); Zohar  Tiran (Compugen Ltd., Research & Development); Amit Novik (Compugen Ltd., Research & Development); Avi Rosenberg (Compugen Ltd., Research & Development); Anat Amir (Compugen Ltd., Research & Development); Assaf Wool (Compugen Ltd., Research & Development); Amir Toporik (Compugen Ltd., Research & Development); Ehud Schreiber (Compugen Ltd., Research & Development); Dani Eshel (Compugen Ltd., Research & Development); Zurit Levine (Compugen Ltd., Research & Development); Yossi Cohen (Compugen Ltd., Research & Development); Claudia Nold-Petry (University of Colorado.) Charles  A. Dinarello (University of Colorado.) Itamar Borukhov (Compugen Ltd., Research & Development); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Blocking conformational changes in biologically active proteins holds therapeutic promise. Inspired by the susceptibility of viral entry to inhibition by synthetic peptides that block the formation of helix–helix interactions in viral envelope proteins, we developed a computational approach for predicting interacting helices. Using this approach, which combines correlated mutations analysis and Fourier transform, we designed peptides that target gp96 and Clusterin, two  secreted chaperones known to shift between inactive and active conformations. The gp96-derived peptide and the Clusterin-derived peptide exhibit anti-inflammatory and anti-tumor activity, respectively, both in-vitro and in-vivo. Furthermore, the predicted mode of action of the active peptides was experimentally verified: both peptides bound to their parent proteins, and their biological activity was abolished in the presence of the peptides corresponding to the counterpart helices. These data demonstrate a novel method for rational design of protein antagonists.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W24'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W24</strong></td>
 </tr> <tr> 
  <td><b>Spritz: Intrinsic protein disorder prediction for genomes and insights into protein function.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ian Walsh <i>University of Padua</i></td>
 </tr>
 <tr> 
  <td>  Alberto J. Martin Martin (University of Padua, Department of Biology); Tomas Di Domenico (University of Padua, Department of Biology); Silvio Tosatto (University of Padua, Department of Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: An alternative view is emerging with respect to non-folding regions which suggest a reassessment of the structure-to-function paradigm. These non-folding flexible regions within a protein are known as disordered regions. This work is concerned with two algorithms for the prediction of this important protein feature:<br><br><b>Genomes</b>: Protein disordered regions are key for the function of numerous processes within an organism. Experimental annotations for disorder remain low (e.g. Disprot and the PDB) while current genomic efforts cause sequence databases to grow very quickly. ESpritz is an accurate and efficient method which can process entire organisms in hours (human genome in approximately 6hrs). Espritz uses the Bi-directional Recurrent Neural Network algorithm. We prove that ESpritz is state-of-the-art with respect to Disopred and IUPred on X-ray, Disprot and NMR type disorder. <br>Server availability: http://protein.bio.unipd.it/espritz/. Executables can be downloaded from http://protein.bio.unipd.it/download/<br><br><b>Disorder and function</b>: Disorder, secondary structure and possible functional amino acid patterns (linear motifs) are particularly useful for determining the biological role of a protein. Our server CSpritz allows the analysis of these three pieces of information. Motif patterns are found from ELM(http://elm.eu.org/) and secondary structure from Porter(http://distill.ucd.ie/porter/). The algorithm for disorder prediction is based on averaging three orthogonal systems developed in-house. CSpritz is less efficient but more accurate than ESpritz. It is more suited to the examination of a small number of proteins which may be of biological interest. We benchmark CSpritz on the recent CASP9 dataset and prove that it is state-of-the-art with respect to the other participating groups at CASP9.<br>Server availability: http://protein.bio.unipd.it/cspritz/</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W25'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W25</strong></td>
 </tr> <tr> 
  <td><b>TargetP 2.0: Improved subcellular localization prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daniel Edsgärd <i>Royal Institute of Technology</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Determining the subcellular localization of a protein is an important first step toward understanding its function. The translocation of a protein within the cell is governed by intrinsic signals of the proteins, and bioinformatics has become an important tool for the characterization and prediction of these protein-encoded sorting signals. TargetP is a neural-network based subcellular localization tool which provides prediction of chloroplast, mitochondrial and secretory pathway translocation, based on N-terminal sequence motifs. It can be applied to newly identified proteins and to characterize the proteome-wide localization patterns of a species protein content. Here we report a comprehensive update of TargetP with regard to two major aspects. First, TargetP is extended to supply specific predictions for three phyla; plant, fungi and metazoa; rather than only for plants and non-plants. Second, new protein compartments are added. TargetP is currently available as a web-server at http://www.cbs.dtu.dk/services/TargetP, where also the new version will be released.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W26'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W26</strong></td>
 </tr> <tr> 
  <td><b>Slip symmetry – a new type of symmetry in protein structures</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dukka KC <i>NIH-NCI</i></td>
 </tr>
 <tr> 
  <td>  Todd  Taylor (NIH, NCI); Byungkook Lee (NIH, NCI); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We have recently developed a procedure called SymD to determine whether a protein is symmetric or not (Kim et al., BMC Bioinfo., 2010).  Upon applying SymD on ASTRAL 40 dataset we obtained 10 to 15 % of proteins to be symmetric according to our criteria.  <br><br>Upon detailed investigation of the results, we have realized that there are proteins that have high similarity score (Z-score) but are different from regular symmetric proteins. These proteins have what we call a ‘slip symmetry’, i.e. the structure superimposes well with itself when translated by a small amount (slipped) along a certain direction. Structural alignment between two proteins with slip symmetry can produce two or more different structure-based sequence alignments that have similar scores.<br><br>We have developed a procedure to determine whether a protein has a ‘slip symmetry’ using the Z-score, symmetry angle, and the sequence alignment that SymD produces. The method is based on the value of the average serial number difference between the aligned residues of the protein with high z-score and small symmetry angle.<br><br>We have applied this algorithm to the whole dataset of Astral SCOP 1.75 that has around 110000 domains and have obtained nearly 700 domains that have a slip symmetry. These include a large number of helix bundles but also some in Tim-barrel (c.1), Rossmann (c.2), and other folds.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W27'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W27</strong></td>
 </tr> <tr> 
  <td><b>A multiple-template approach to protein threading</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jian Peng <i>Toyota Technological Institute at Chicago</i><br></td>
 </tr>
 <tr> 
  <td>Jinbo Xu (Toyota Technological Institute at Chicago) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Due to the increasing number of solved structures, a protein without solved structure is very likely to have more than one similar template structures. Therefore, a natural question to ask is if we can improve modeling accuracy using multiple templates. This work describes a new multiple-template threading method to answer this question. At the heart of this multiple-template threading method is a novel probabilistic-consistency algorithm that can accurately align a single protein sequence simultaneously to multiple templates. Experimental results indicate that our multiple-template method can improve pairwise sequence-template alignment accuracy and generate models with better quality than single-template models even if they are built from the best single templates while many popular multiple sequence/structure alignment tools fail to do so. The underlying reason is that our probabilistic-consistency algorithm can generate accurate multiple sequence/template alignments. In another word, without an accurate multiple sequence/template alignment the modeling accuracy cannot be improved by simply using multiple templates to increase alignment coverage. According to the CASP9 official evaluation, our method outperforms almost all other CASP9 servers and our method generated the best alignments for the 50 hardest template-based modeling targets. Our method was also voted by the CASP9 community as one of the most innovative and interesting methods. Our method will have greater potential in the near future when many more templates are available due to the increasing number of solved structures. We can further improve alignment accuracy by extending our algorithm to simultaneously thread multiple homologous sequences to multiple templates.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W28'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W28</strong></td>
 </tr> <tr> 
  <td><b>A novel type of methyltransferase encoded by a conserved domain of GRAS</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Magdalena Dymecka <i>M. Sk?odowska-Curie Cancer Center and Institute of Oncology</i></td>
 </tr>
 <tr> 
  <td>  Katarzyna Kokoszy?ska (M. Sk?odowska-Curie Cancer Center and Institute of Oncology, -); Leszek Rychlewski (Bioinfobank Institute, -); Cordelia Bolle (Ludwig-Maximilians-Universitat, -); Lucjan Wyrwicz (M. Sk?odowska-Curie Cancer Center and Institute of Oncology, -); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The GRAS proteins, expressed exclusively in plants, play an important role in plant growth and development and take part in such processes as signal transduction, gibberellic acid (GA) signaling and meristem maintenance. As the exact molecular mechanism of GRAS remains undiscovered, a detailed bioinformatics study was performed.<br>The GRAS proteins sequence analysis showed the presence of two regions - the DELLA domain and the second domain (also known as GRAS domain). The previous reports on the GRAS domain based on its nuclear localization and the presence of five characteristic sequence motifs suggested that these proteins may belong to the family of transcription factors. <br>Here we present the results of a detailed bioinformatics study which reveal the presence of a novel methyltransferase fold within the GRAS domain. Top-of-the-line bioinformatics (identification of homology via meta-profile methods as well as mapping of distant protein homology for homology modeling) served to prepare a structural model of the GRAS domain.<br>The presented results shed new light on the putative function of the GRAS protein. The mapping of the identified functional regions allowed to build a hypothesis on its molecular action, still the analyses require further experimental verification.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W29'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W29</strong></td>
 </tr> <tr> 
  <td><b>Implementation of the Protein Fluorescence And Structural Toolkit (PFAST)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Cynthia Prudence <i>University of Rhode Island</i></td>
 </tr>
 <tr> 
  <td>  Yana Reshetnyak (University of Rhode Island, Physics Department); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The major goal in the application of tryptophan fluorescence spectroscopy is to interpret fluorescence properties in terms of structural parameters and to predict of the structural changes in the protein. We have developed methods for the mathematical analysis of fluorescence spectra of multitryptophan proteins aimed at revealing the spectral components of individual tryptophan or clusters of tryptophan residues located close to each other (Burstein et al., 2001, Biophys J., 81, 1699-1709; Reshetnyak and Burstein, 2001, Biophys. J., 81, 1710-1734). Also, we have created an algorithm for the structural analysis of the tryptophan environment in 3D atomic structures of proteins from PDB (Reshetnyak et al, 2001, Biophys. J., 81, 1735-1758). The successful design of the methods of spectral and structural analysis opened an opportunity for establishing a relationship between the spectral and structural properties of a protein. We have integrated the developed software modules, introduced new programs for the assignment of tryptophan residues to spectral-structural classes, and created a web-based toolkit PFAST: Protein Fluorescence and Structural Toolkit (Shen et al., 2008, Proteins, 71, 1744-1754). PFAST contains 3 modules: 1) FCAT - fluorescence-correlation analysis tool, which decomposes protein fluorescence spectra and assigns spectral components to one of five previously established spectral-structural classes. 2) SCAT - structural-correlation analysis tool for the calculation of the structural parameters of the environment of tryptophan residues from the atomic structures of the proteins from the PDB, and for the assignment of tryptophan residues to one of five spectral-structural classes. 3) The last module is a PFAST database.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W30'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W30</strong></td>
 </tr> <tr> 
  <td><b>Maximal Use of Information Found in All Protein Structure Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Armando Solis <i>New York City College of Technology, The City University of New York</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The universal mechanism of transforming occurrence frequencies in structural data into “energies” or scores, for use as knowledge-based potentials, requires a well-defined way of assembling a comprehensive structural data set from the PDB. My ongoing strategy, to link information-theoretic notions to the nature and action of empirical potentials, continues to explore ways to maximize the information that can be extracted from still limited data. I will report on the latest results of one particular application of this strategy. Specifically, I am advancing an improved method to use all high-resolution structures in the PDB to build more informative potentials. This is in contrast to the commonly employed strategy of using a non-redundant subset of structures, which limits database bias, but also leaves out potentially valuable knowledge. This approach is supported by fundamental information-theoretic formulations, with the consistent goal of maximizing discrimination between native and decoy conformations. I present my most current results, which appear to demonstrate that such an approach can potentially produce potentials that show markedly improved performance in fold recognition tests.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W31'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W31</strong></td>
 </tr> <tr> 
  <td><b>Investigating the effects of pH variations on structure and stability of maltotriose-binding protein from Thermus thermophilus using a combined approach of Fluorescence correlation spectroscopy and Mo</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Anna Marabotti <i>Italian National Research Council</i></td>
 </tr>
 <tr> 
  <td>  Antonio Varriale (Italian National Research Council, Institute of Protein Biochemistry); Maria  Staiano (Italian National Research Council, Institute of Protein Biochemistry); Luisa Iozzino (Italian National Research Council, Institute of Protein Biochemistry); Luciano Milanesi (Italian National Research Council, Institute of Biomedical Technologies); Sabato D'Auria (Italian National Research Council, Institute of Protein Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We used a combination of a computational technique (Molecular Dynamics, MD) and of an experimental technique (Fluorescence Correlation Spectroscopy, FCS) to study the conformational variability of Maltotriose binding protein from T. thermophilus (MalE2) in different conditions of pH. The structure of the unliganded form of the protein was modelled and simulations were conducted in three different pH conditions by changing the protonation state of the protic residues according to their predicted pKa.<br>Results show that at extreme pH values multiple conformations of the protein are visible, indicating that the tertiary structure of the protein is highly unstable. This effect is maximum at acidic pH, indicating that probably the protein undergoes a pH-induced unfolding transition. On the contrary, at neutral pH the protein shows only one cluster of tertiary structures with a low RMSF. A PCA analysis identified some regions of the protein with increased flexibility. The diffusion coefficient calculated from MD simulations is in excellent agreement with the experimental one obtained by FCS. It is interesting to note that these two investigation techniques are both focused on single molecules, but their timescale are completely different since FCS explores the dynamics of a protein in the microsecond time range.<br>This work provides the evidence that MD simulations are able to anticipate the phenomena occurring at a longer timescale, supporting the hypothesis that at these two extreme pH conditions this protein shows a conformational instability on its tertiary structure that can evolve into unfolded or less stable states.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W32'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W32</strong></td>
 </tr> <tr> 
  <td><b>Pathogenic-Or-Not-Pipeline (PON-P): Integration of predictors for disease relevance</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Mauno Vihinen <i>University of Tampere</i></td>
 </tr>
 <tr> 
  <td>  Ayodeji Olatubosun (University of Tampere, Institute of Biomedical Technology); Jouni Väliaho (University of Tampere, Institute of Biomedical Technology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This work deals with the prediction of whether a non-synonymous single nucleotide polymorphism leads to disease or not. This is an important scientific problem, with potential applications in such areas as molecular diagnosis, prioritization of experiments and screening of variations. This work utilizes a unique approach in an effort to improve on the current performance limitations of predictors in this problem domain. The importance of this work to this particular conference is further highlighted by the fact that the SNP-special interest group one-day meeting is organized specifically to address the same problem dealt with in this work, which will also be the focus of the second special session in this meeting. The results of this work, and their implications would be of high interest to researchers in many fields.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W33'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W33</strong></td>
 </tr> <tr> 
  <td><b>Protein function prediction using clustering method of Protein-Protein Interaction network</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ji Hye Choi <i>University of Dankook</i><br></td>
 </tr>
 <tr> 
  <td>Sejong Oh (University of Dankook) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Assigning functions to novel proteins is one of the most important problems in the postagenomic era. Several approaches have been applied to this problem, including the analysis of gene expression patterns, phylogenetic profiles, protein fusions and protein-protein interaction. In this study, we develop a novel approach to infer a protein's function using clustering method on protein-protein interaction network. In protein-protein interaction network, nodes can represent each protein and edges can represent relationshils of interaction. If two nodes are interacted, the nodes are functionally similar. It is representative feature of protein-protein interaction. And we can detect protein complexes by clustering protein-protein interaction network. In same protein complex, each protein is functionally associated to other proteins. So if we use result of clustering, we can infer functions of unknown proteins. We gained data of protein-protein interaction in MIPS. And used clustering method is ClusterONE of Cytoscape software. From our experiments, we confirmed improvement of prediction accuracy. Particularly ratio of 'true positive(TP)' is highly increased. Problem of other methods like neighbor counting and chi-square was low accuracy of TP. Proposed method promises to improve the accuracy of protein function prediction.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W34'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W34</strong></td>
 </tr> <tr> 
  <td><b>Comprehensive study of HOX transcription factors DNA binding specificity</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Katarzyna Kokoszynska <i>Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</i></td>
 </tr>
 <tr> 
  <td>  Maja  Ma?kowska (Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Laboratory of Bioinformatics and Systems Biology); Leszek  Rychlewski (BioInfoBank Institute, -); Lucjan Wyrwicz (Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology, Laboratory of Bioinformatics and Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The homeodomain-containing HOX protein family forms an important evolutionary network of transcription factors regulating the development and differentiation in Metazoa. Though the high level of homeodomain conservation within the protein family, particular HOX proteins are characterized by specific regulation of target genes. Previous structural studies of HOX representatives pointed that the mechanism of DNA fragment recognition by the third alpha-helix of the homeodomain occurs via interactions with the major groove. The additional region responsible for the specific  DNA recognition - so called N-terminal arm - is involved in DNA minor groove interaction, yet very little is known how the N-terminal arm achieves its specificity. <br>For the purpose of better understanding the specific mechanism of transcription regulation by HOX proteins, we performed a detailed structural study of protein-DNA interaction specificity. Using homology modeling methods we prepared 3D models of the complete set of human HOX proteins putting it into an open-access repository of HOX structures. The summary of the experimental data (source: TRANSFAC database) together with the analysis of evolutionary protein sequence conservation and structure conformation study of HOX monomers was performed. Separate analyses were made for the HOX proteins in  heterodimer conformations with their cofactors playing a major role in the complex binding specificity (TALE homeodomain proteins). <br><br>The obtained results are discussed along with the detailed analyses of expression pattern of HOX proteins and their target genes, giving a new insight into the potential mechanism of the DNA recognition specificity and processes regulated by these factors.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W35'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W35</strong></td>
 </tr> <tr> 
  <td><b>Structure Based Analysis of the Interactions between PAS Domain Containing Circadian Clock Proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>SERAP BELDAR <i>KOÇ UNIVERSITY</i><br></td>
 </tr>
 <tr> 
  <td>Serap Beldar (KOÇ UNIVERSITY) Özlem Keskin (KOÇ UNIVERSITY, Chemical and Biological Engineering); Halil Kavakli (KOÇ UNIVERSITY, Chemical and Biological Engineering); Attila  Gürsoy (KOÇ UNIVERSITY, Computer Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The mammalian BMAL1 and CLOCK are the transcription factors that bind E-box and regulate the expression of clock controlled genes. On the other hand, the PER2 and CRYPTOCHROME (CRY) proteins act as negative regulators of such transcription. PER2 and CRY form ternary complexes with casein kinase I? (in the cytoplasm) and translocates into the nucleus, where they form complex with BMAL1/CLOCK and suppress expression of clock controlled genes. Many experimental studies have been performed to figure out the interactions between these clock proteins. However, there is no study related with how clock proteins form such a complex including all clock proteins. To understand the nature of such a complex, we performed structure based analysis on protein-protein interactions with clock proteins (CLOCK, BMAL1 and PER2) that contains PAS domain. This complex was analyzed by using structural data and efficient structural comparison algorithms to predict potential interactions. Since there are no available atomic structures for clock proteins, homology models are used as structural data. In our model, BMAL1 and CLOCK interacts with each other through their PAS domain and the PER2 interacts with this dimer through the PAS domain of CLOCK. This is the first time that we showed how the clock complexes form at the molecular level. This study is not only important to understand clock mechanism at molecular level but also will allow us to develop drugs against clock-regulated diseases like Jet-Lag and some form of depressions. Currently we are carrying out experimental work to prove this model.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W36'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W36</strong></td>
 </tr> <tr> 
  <td><b>The MULTICOM Toolbox for Protein Structure Prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jianlin Cheng <i>University of Missouri Columbia</i></td>
 </tr>
 <tr> 
  <td>  Jilong Li (University of Missouri, Columbia, Computer Science); Zheng Wang (University of Missouri, Columbia, Computer Science); Jesse Eickholt (University of Missouri, Columbia, Computer Science); Xin Deng (University of Missouri, Columbia, Computer Science); Jianfeng He (University of Missouri, Columbia, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As genome sequencing is becoming a routine in biomedical research, the total number of protein sequences is increasing exponentially, reaching over 108 million recently. However, to date, only a tiny portion of these proteins (e.g. ~72,000) have known tertiary structures determined by experimental techniques. The gap between protein sequence and structure still enlarges rapidly as the throughput of genome sequencing is much higher than that of protein structure determination. Computational software tools of predicting protein structure and structural features are crucial to make use of this vast repository of protein resources. To meet the need, we developed a comprehensive MULTICOM toolbox consisting of a set of protein structure and structural feature prediction tools, including secondary structure prediction, solvent accessibility prediction, disorder region prediction, domain boundary prediction, contact map prediction, disulfide bond prediction, beta-sheet topology prediction, fold recognition, multiple template combination and alignment, template-based tertiary structure modeling, protein model quality assessment, and mutation stability prediction. These tools had been rigorously tested by many users in the last several years and/or during the last three rounds of the Critical Assessment of Techniques for Protein Structure Prediction (CASP7-9) from 2006 to 2010, achieving the state-of-the-art or close performance. In order to facilitate bioinformatics research and technology development in the field, we make the MULTICOM toolbox freely available for academic use and scientific research at http://sysbio.rnet.missouri.edu/multicom_toolbox/.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W37'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W37</strong></td>
 </tr> <tr> 
  <td><b>Comprehensive structure-to-function analysis of transcription factor IID (TFIID) complex</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Maja Ma?kowska <i>Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology</i><br></td>
 </tr>
 <tr> 
  <td>Maja Małkowska (Maria Sklodowska-Curie Memorial Cancer Center and Institute of Oncology) Katarzyna Kokoszy?ska (Maria Sklodowska- Curie Memorial Cancer Center and Institute of Oncology, Laboratory of Bioinformatics and Systems Biology); Leszek Rychlewski (BioInfoBank Institute, BioInfoBank Institute); Lucjan Wyrwicz (Maria Sklodowska- Curie Memorial Cancer Center and Institute of Oncology, Laboratory of Bioinformatics and Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The transcription of protein genes into mRNA in Eukaryotes is<br>mediated by RNA polymerase II (Pol II). The initiation of this process<br>demands cooperation of many transcription factors (TFIIs). The main<br>component of preinitiation complex is TFIID. It recognizes and binds the<br>gene promoter, forms a scaffold for other transcription factors (TFIIA,<br>TFIIB, TFIIE, TFIIF, TFIIH) and provides the proper position of Pol<br>II. TFIID is a large macromolecular complex composed of TATA-box<br>binding protein (TBP) and a group of 14 conserved TBP-associated<br>factors (TAFs). TAFs are known to regulate transcription at various<br>levels - mediating transcription via interaction with activators, histone<br>modifications, recognition and binding to promoters, being a platform for<br>other transcription factors and RNA polymerase II.<br><br>Despite numerous previous studies of the TFIID complex still the<br>knowledge on the structure of its components and thus the exact mechanism<br>of its function, remains undetermined. To perform an in-depth analysis of<br>TFIID we performed the state-of-the-art structural bioinformatic analysis<br>of the TFIID complex. Homology modeling of particular TAFs allowed to<br>determine a potential structure of TAF2 as M1 aminopeptidase-like protein,<br>and facilitated mapping of previously not fully characterized structural<br>domains in well-studied TAF proteins (including full histone-like domains<br>of TAF4 and 12 or TAF3 and 8 bromodomain associated structure). In the<br>study we provided the detailed structural models for all elements of human<br>and major model species analyzed in the context of TFIID activity with<br>indication of structural alterations within TFIID in various model species.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W38'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W38</strong></td>
 </tr> <tr> 
  <td><b>Structure-based classification of the plant non-specific lipid transfer protein superfamily towards its functional characterization</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Cécile Fleury <i>INRA/CIRAD</i></td>
 </tr>
 <tr> 
  <td>  Marie-Françoise GAUTIER (INRA, UMR AGAP); Franck MOLINA (SysDiag CNRS/Bio-Rad, UMR 3145); Frédéric de LAMOTTE (INRA, UMR AGAP); Manuel RUIZ (CIRAD, UMR AGAP); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The non specific Lipid Transfer Proteins (nsLTPs) show large variations in their sequences, biological roles, quaternary associations and the nature of bound hydrophobic ligands. Besides, they are involved in a large number of biological processes relative to plant development and defense However they share a conserved eight-cysteine-residue pattern which plays an important role in the structural scaffold. For these reasons, the nsLTP superfamily constitutes an interesting case of study to validate a method designed for the investigation of protein structure-function relationships. According to new identification criteria, 800 mature amino acid sequences belonging to more than 100 plant species have been selected and submitted to comparative analysis. Given the overall fold conservation and the good quality of the available experimental structures, reliable models were obtained for all nsLTPs using homology modeling method, although the identity rate observed among all nsLTP sequences is only up from 25%. Sequence- and structure-based phylogenies have been performed independently, and both resulting classifications have been compared. Using the evolutionary trace method, we observed that only the eight cysteines are strictly conserved among the whole family members. Other less conserved residues appear to be class-specific and localized inside the hydrophobic cavity which accommodates the ligand. A high variability is observed on the external loop which is supposed to be structurally unimportant but probably playing a role in driving the ligand into the cavity. We assume that this region may be responsible for ligand binding diversity. Further investigation will focus on the molecular determinants of this variability.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W39'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W39</strong></td>
 </tr> <tr> 
  <td><b>A comparative study on filtering protein secondary structure prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Petros Kountouris <i>University of Cyprus</i><br></td>
 </tr>
 <tr> 
  <td>Chris Christodoulou (University of Cyprus) Michalis Agathocleous (University of Cyprus, Computer Science); Vasilis Promponas (University of Cyprus, Biological Sciences); Georgia Christodoulou (University of Cyprus, Computer Science); Simos Hadjicostas (University of Cyprus, Computer Science); Vassilis Vassiliades (University of Cyprus, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Over the past 20 years, machine learning techniques and evolutionary information have significantly boosted the quality of protein secondary structure prediction methods. Moreover, filtering of the final predictions have been shown to improve and smooth the predictions, providing more realistic results. Both machine learning techniques and empirical rules have been used to filter the sequence-to-structure secondary structure prediction. Despite being employed widely, to the best of our knowledge, no study has been carried out to find the most suitable filtering technique. In this paper, we perform a comparative study on the challenging problem of filtering, utilising both widely used empirical smoothing rules and machine learning techniques. In particular, we predicted the secondary structure using an ensemble of Bidirectional Recurrent Neural Networks and  employed the WEKA software package to evaluate filtering with the following algorithms: Naive Bayes, Classification And Regression Tree, AdaBoost, Logistic function, Radial Basis Function (RBF) Neural Network, k-Nearest Neighbour, Multilayer Perceptron, J48 decision trees, Random Forest and combinations of the above using several voting schemes. In addition, we filtered the predictions using Support Vector Machines (SVM) by utilising the LibSVM software package. For this task, we also implemented a Hidden Markov Model and an RBF Network, which is initialised by a Self-Organising Map. Different local window sizes were tested to select the optimal one for each approach. Notably, the Logistic function and the SVM have been found to be superior to the tested methods in terms of both predictive accuracy (Q3) and the Segment Overlap score.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W40'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W40</strong></td>
 </tr> <tr> 
  <td><b>firestar in 2011</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Paolo Maietta <i>Centro Nacional de Investigaciones Oncologicas</i><br></td>
 </tr>
 <tr> 
  <td>Michael Tress (Centro Nacional de Investigaciones Oncologicas) Gonzalo Lopez (Columbia University, C2B2); Jose Manuel Rodriguez (Centro Nacional de Investigaciones Oncologicas, INB); Alfonso Valencia (Centro Nacional de Investigaciones Oncologicas, Structural and Comptational Biology Programme); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Here we present the new developments in firestar, an expert system for predicting functional residues in protein sequences based on information extracted from the PDB. firestar extrapolates from the large inventory of catalytic and biologically relevant small molecule ligand binding residues that are organized in the FireDB database and makes predictions for those validated functional residues from FireDB that are supported by local sequence conservation in the query sequence.<br><br>Several new features have been incorporated into firestar to improve the quality and coverage of the predictions. Functional residues in the FireDB repository are now classified in terms of their biological relevance using evolutionary information, structural data and lists of known cognate ligands and we have implemented HHsearch as an additional, more powerful method of detecting remotely related templates. <br><br>Previous versions of firestar required human interpretation of the results. Now, the whole process has been automated and a new web interface has been made available. Additionally, the server is able to produce high quality results in a high throughput mode by using sequences as the only input.<br><br>firestar has been benchmarked against the CASP7, CASP8 and CASP9 ligand binding prediction targets. The server was able to detect ligand binding residues for 94% of the sites over all three CASP editions. During CASP8 firestar outperformed all officially participating groups in CASP8, showing that firestar is a state of the art method and initial results for the CASP9 experiment suggested that firestar outperformed all the other 15 servers on targets with biological ligands.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W41'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W41</strong></td>
 </tr> <tr> 
  <td><b>MinkoFit3D: Modeling of macromolecular structures into density maps by 3D Minkowski sum analysis.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Wojciech Potrzebowski <i>International Institute of Molecular and Cell Biology in Warsaw</i></td>
 </tr>
 <tr> 
  <td>  Janusz M. Bujnicki (International Institute of Molecular and Cell Biology in Warsaw, Laboratory of Bioinformatics and Protein Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Experimental advances in producing electron microscopy data on structures of macromolecular assemblies have paved the way for theoretical methods that dock atomic models into low-resolution electron density maps. Although considerable progress has recently been made in building pseudo-atomic models based on combination of cryo-EM and X-ray crystallography, putting together macromolecular assemblies consisting of tens or even hundreds of components remains a challenging and still unsolved problem. The same is true for combining structures with partial information e.g. with known molecular shape. <br>We developed a new method for fitting atomic models or any user-defined shapes into cryo-EM maps by means of reduced mesh representation. Our program achieves this goal by calculating the Minkowski sum of polyhedrons of the structure or any user-defined shape and the map. The position of the fitted component is obtained by finding “tight passages” in the configuration space defined by the Minkowski sum. This method has been inspired by an approach used in mobile robotic navigation control in obstacle environment. The fit inferred from Minkowski sum is subject to local refinement in the electron density map. This procedure applies iterative superposition of the atomic structure on the calculated density centers of the reference electron density map. Selected or all subunits forming assembly may undergo rotational and translational movement to optimize the correlation coefficient and minimize number of inter-subunit steric clashes until convergence.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W42'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W42</strong></td>
 </tr> <tr> 
  <td><b>BOCTOPUS: Topology prediction of Transmembrane beta barrel proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sikander Hayat <i>Stockholm University, Sweden</i></td>
 </tr>
 <tr> 
  <td>  Arne Elofsson (Stockholm University, Department of Biophysics and Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Transmembrane beta barrel proteins (TMBs) constitute 2-3% of the genome of Gram-Negative bacteria and are found in the outer membrane of Gram-Negative Bacteria, Chloroplast and Mitochondria where they play a major role in the translocation machinery. TMBs also perform functions such as pore formation, membrane anchoring, ion exchange and are candidate molecular targets for development of antimicrobial drugs and vaccines.<br><br>We present here BOCTOPUS, a computational method for the topology prediction of TMBs. BOCTOPUS is based on ideas from computational methods that have previously been used for topology prediction of Helical membrane proteins and work by combining local and global predictions [1, 2]. BOCTOPUS is benchmarked using a non-redundant dataset of 37 TMBs with known structures. BOCTOPUS employs Support Vector Machines (SVMs) and a Hidden Markov Model (HMM)  and uses position specific scoring matrix (PSSM) as the input parameter. In the first stage, three separate SVMs are used to predict: strand/non-strands, in/not-in, out/not-out regions and the output probabilities are used as the input for the HMM.<br><br>Preliminary results show that the SOV, Q3 of BOCTOPUS based on a leave-one-out cross-validation test are 92.8 and 87.8, respectively. Moreover, BOCTOPUS can predict 526 out of 559 (94.1%) strands in the dataset within +/- 3 residues. The BOCTOPUS method outperforms existing methods from the literature.<br><br>1. Improving the accuracy of transmembrane protein topology prediction using evolutionary information, Jones DT, Bioinformatics, 2007<br>2. OCTOPUS: improving topology prediction by two-track ANN-based preference scores and an extended topological grammar, Viklund H, Elofsson A, Bioinformatics, 2008</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W43'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W43</strong></td>
 </tr> <tr> 
  <td><b>Analysis of Functional Islands using Structural Clustering</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Anne-Christin Hauschild <i>Max-Planck-Institute for Informatics</i></td>
 </tr>
 <tr> 
  <td>  Ingolf Sommer (Max-Planck-Institute for Informatics, Computational Biology & Applied Algorithmics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motivation: Over the past years protein structure databases have been growing at a rapid pace. Using structural comparison of proteins one can embed the proteins into a - not necessarily metric - space. Using available function annotations, one can analyze local conservation of function in structure space. Considerable differences in the local conservation of different molecular functions have been reported. Here we present a clustering method, useful to analyze and visualize the landscape of proteins in this space.<br>Method: Structure space can be represented as a weighted, directed graph: vertices represent proteins, connected by edges if the proteins are structurally similar. The edges are weighted by structural similarities.<br>The clustering method works as follows. Two connected vertices are assigned to the same cluster if they share a similar neighborhood, which means, that two vertices  share the same adjacent vertices. We discuss, how this strategy relates to established clustering methods.<br>Result: We applied the clustering method to a representative set of protein domains annotated with GO molecular function terms. We visualize clusterings for various molecular functions. Essentially, we observe two classes of clusterings: I) all proteins are located in one cluster, in these cases the structural similarity infers the same GO function. II) multiple subclusters occur, which means even though the proteins in different clusters are annotated with the same molecular function, they have distinct structural origin. This approach provides new insights into the relation between protein structures and the annotated GO molecular function terms.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W44'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W44</strong></td>
 </tr> <tr> 
  <td><b>The i-Patch Server: Prediction of inter-protein contact sites</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Konrad Krawczyk <i>University of Oxford</i></td>
 </tr>
 <tr> 
  <td>  Charlotte Deane (University of Oxford, Statistics); Rebecca Hamer (University of Oxford, Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We present i-Patch - an interactive software tool for the prediction of inter protein contact sites. i-Patch is currently one of the most successful methods for the prediction of residues involved in protein protein interactions (REF).  It achieves 59% precision with 20% recall on a blind testing set of 31 proteins. The method uses the sequence alignments of the homologues of two proteins which are known to interact alongside their three dimensional structures. It produces a contact-site likelihood score for each of the surface-exposed residues in the two proteins. This score is calculated by treating the complex of two proteins as a network with amino acids as nodes and distances between them as edges. Our tool allows the user to enter their two proteins of interest and gives back protein structures which are highlighted by the i-Patch score. This allows the user to identify the residues and patches of residues that potentially constitute the binding site between the two proteins.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W45'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W45</strong></td>
 </tr> <tr> 
  <td><b>Predicting the 3d DNA structure of the human genome</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sven Bilke <i>National Cancer Institute</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Evidence for a non-random spatial 3d organization of the cells DNA content and its relevance for gene regulation has been accumulating in recent years. In a recent study [1], Dekker and co-workers introduced a novel method, HiC, allowing for an unbiased genome wide study of 3d conformations producing a "probability map" of DNA-DNA contacts in an ensemble of cells. Here we aim to identify genomic parameters correlating with the 3d-structure described in [1]. <br><br>We successfully developed a model based exclusively on DNA sequence related observables and a set of mixing parameters. Using Monte Carlo optimization techniques, we identify two major sequence features contributing to the contact matrix. The resulting model faithfully reproduces the empirical consensus contact probability map described in [1] with Pearson's correlation r > 0.71. For comparison, intra-experiment correlation of the data in [1] ranges from 0.55 to 0.89.  Only 12 out of 32 intra-experimental correlation are larger than 0.71.<br><br>Additional experimental validation of our findings has started at time of writing this abstract. <br><br>[1] Lieberman-Aiden,E., Dekker,J. et al, Comprehensive mapping of<br>long-range interactions reveals folding principles of the human<br>genome. Science, 326(5950) 289-293 (2009).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W46'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W46</strong></td>
 </tr> <tr> 
  <td><b>Binding Ligand Prediction by Comparing Local Surface Patches of Potential Pocket Regions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daisuke Kihara <i>Dept of Biological Sci/Computer Sci</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Function of a protein, specifically, the type of ligand that bind to a protein, can be predicted by finding similar local surface regions of known proteins. We developed an alignment free local surface comparison method for predicting ligand molecule which binds to a query protein. The algorithm, named Patch-Surfer, represents a binding pocket as a combination of segmented surface patches, each of which is characterized by its shape, the electrostatic potential, the hydrophobicity, and concaveness. Representing a pocket by a set of patches is effective to absorb difference of global pocket shape while capturing local similarity of pockets. The shape and the physicochemical properties of surface patches are represented using the 3D Zernike descriptor, which is a series expansion of mathematical 3D function. Two pockets are compared using a modified weighted bipartite matching algorithm, which matches similar patches from the two pockets. <br>The benchmark studies showed that Patch-Surfer’s prediction performance is superior to existing methods, including Pocket-Surfer developed previously by us (Chikhi, Sael, Kihara, Proteins 2010), which represents a pocket as a whole. On a dataset of 100 non-homologous proteins, 84.0% of the binding ligands were predicted correctly within the top three scores using the shape and pocket size information. The performance was further improved to 87.0% when physicochemical properties were added. Overall, we show that proposed method is powerful in predicting the type of ligand a protein binds even in the absence of homologous proteins in the database. (Sael & Kihara Int J Mol Sci, 2010).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W47'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W47</strong></td>
 </tr> <tr> 
  <td><b>Prediction of binding preferences of RNA-binding proteins using a novel graph representation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daniel Maticzka <i>University of Freiburg</i><br></td>
 </tr>
 <tr> 
  <td>Rolf Backofen (University of Freiburg) Fabrizio Costa (University of Freiburg, Chair for Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA-binding proteins (RBPs) are involved in a number of cellular processes as diverse as regulation of post-transcriptional gene expression, splicing or nuclear export. RBPs recognize their various RNA binding partners by a combination of sequence and secondary structure context of the binding site. In order to accurately model binding preferences of RBPs, this structural context has to be considered in addition to the nucleotide sequence. Structural context has been used by the MEMERIS motif finder to guide the search towards single-stranded regions. The more recent RNAcontext method is able to discover both sequence and structural preferences. Here we propose a novel graph representation of RNA sequence and secondary structure information suitable for the elicitation of RBP binding preferences. In this graph each nucleotide can be annotated with information about its predicted structural context such as being paired, unpaired or being unpaired while being part of a certain secondary structure. Using NSPDK (neighbourhood subgraph pairwise distance kernel) these graphs can then be mapped to explicit features. The resulting sparse feature vectors are suitable for support vector machine learning methods. We successfully applied our graph representation in combination with support vector regression to the prediction of RNA-binding preferences of 9 RBPs as determined by RNAcompete, an in vitro assay for the estimation of binding affinities of RBPs.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W48'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W48</strong></td>
 </tr> <tr> 
  <td><b>ProBiS: A web server for detection of structurally similar protein binding sites</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Janez Konc <i>National Institute of Chemistry</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A web server, ProBiS, freely available at http://probis.cmm.ki.si, is presented. This provides access to the program ProBiS (Protein Binding Sites), which detects protein binding sites<br>based on local structural alignments. Given a structure of a protein with unknown binding sites, ProBiS suggests the regions on its surface which may be involved in binding with small ligands,<br>proteins, or DNA/RNA. Alternatively, given a protein with an identified binding site, ProBiS finds other proteins with structurally or physicochemically similar binding sites. If used as a pairwise structure alignment program, ProBiS detects and superimposes similar functional sites in a pair of submitted protein structures, even when these do not have similar folds.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W49'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W49</strong></td>
 </tr> <tr> 
  <td><b>Protein conformational movements modeling based on the average action principle</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Alexander Krass <i>ElectroMedOborudovanie</i></td>
 </tr>
 <tr> 
  <td>  Eugene Stepanov (St. Petersburg State University of Information Technologies, Mechanics and Optics, Department of Mathematical Modelling); Sergey Nikolenko (Academic University, Laboratory of Algorithmic Biology); Yuri Porozov (Academic University, Department of Mathematics and Computer Science Technologies); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: At present, protein structural biology has several modeling methods for macromolecular conformational motions. In this work, we introduce and implement a new modeling method for predicting the conformational motion of protein molecules. It allows to both significantly reduce computation time as compared to molecular dynamics methods and provide better accuracy than the Mixed Elastic Network Model and Cartesian coordinates linear interpolation.<br>We have developed a new mathematical model where a protein is represented as a graph whose nodes mark the atoms of the backbone chain. We compute the weights of side chains for each amino acid and transport them to the corresponding graph node. The basic functional is minimized with gradient descent. An additional problem arises here: we have to ‘factor’ all molecule's movements. This has been taken into account both in our modeling method and in the software that we have developed.<br>The proposed method has shown good results in predicting the correct geometric structure of a protein molecule. We plan to extend this model in further research. This mathematical model and results found with its help can be useful for researchers working in pharmacology and signal biology fields.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W50'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W50</strong></td>
 </tr> <tr> 
  <td><b>Evaluation of transmembrane helix (TMH) packing prediction methods on successive in sequence TMHs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Maria Xenophontos <i>University of Cyprus</i><br></td>
 </tr>
 <tr> 
  <td>Vasilis Promponas (University of Cyprus) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In this work, we evaluate the performance of two state-of-the-art, freely available, SVM-based TMH packing methods (MEMPACK and TMhit), in the special case of predicting interacting successive in sequence TMHs (SSTMHs). Along these lines, we derive information on interacting TMHs, from a non-redundant set of PDB entries of polytopic TM subunits. Locations of TMHs are inferred from the structural data and interaction is defined requiring a minimum of residue-pairs in contact, based on a distance criterion. Additionally, we compute the ground truth of interacting TMH pairs by implementing the same criteria used in the training of the aforementioned methods. <br>Both MEMPACK and TMhit require for input an amino acid sequence, along with the location of TMHs (optionally, TMhit also accepts topology data). Unfortunately, experimental information on TM protein topology is rare. Therefore, in a real-life scenario, users would invoke TMH packing methods with topologies predicted by one of the numerable algorithms that have been developed to accurately predict TM protein topology. Thus, ongoing work investigates how SSTMH interaction prediction quality is affected when using topologies predicted by different widely used methods. Particularly, we have selected a set of diverse freely available TM protein topology prediction tools. <br>Our results so far indicate that MEMPACK predicts SSTMH interactions slightly more accurate than TMhit. Moreover, MEMPACK exhibits higher accuracy, when PSI-BLAST profiles are generated against UniRef90, instead of NCBI-NR. Importantly, both methods show a tendency for higher specificity than sensitivity.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W51'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W51</strong></td>
 </tr> <tr> 
  <td><b>Computing Page Number of RNA Pseudoknotted Structures</b><br/><br/></td>
 </tr>
 <tr> 
 <td>IVAN DOTU <i>BOSTON COLLEGE</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Let S denote the set of (possibly noncanonical) base pairs {i, j} of an RNA tertiary structure; i.e. {i, j} belongs to S if there is a hydrogen bond between the ith and jth nucleotide. The page number of S, denoted P(S), is the minimum number k such that S can be decomposed into a disjoint union of k secondary structures. Finding the page number of an RNA tertiary structure is recently proven to be NP-complete. Here, we present a Constraint Programming (CP)<br>approach to solving the page number problem that yields optimal solutions. The approach is divided into three phases: first, we collapse the set of base pairs into a set of helices; second, we construct a graph where helices are the nodes and vertices correspond to pairs of helices that<br>are involved in a pseudoknot; finally, we apply CP to solve a Minimum Coloring problem on the mentioned graph which corresponds, in fact, to the page number of the given RNA. We present results for the set of RNAs considered in [1] for which their genus number was calculated and we find that the maximum page number among all these RNAs is 4. The motivation for our work is to <br>provide additional computational data relevant to defining a reasonable pseudoknot energy penalty when computing minimum energy pseudoknotted structures.<br><br>[1] M. Bon, G. Vernizzi, H. Orland and A. Zee, J. Mol. Biol. 2008 Jun 13;379(4):900-11.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W52'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W52</strong></td>
 </tr> <tr> 
  <td><b>Are Protein Disordered Regions Equal to Loops?</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Esmeralda Vicedo <i>Technische Universität München</i></td>
 </tr>
 <tr> 
  <td>  Avner Schlessinger (University of California, Bioengineering and Therapeutic Sciences); Markus Schmidberger (Technische Universität München , I12 - Informatik); Marco  Punta (Technische Universität München , I12 - Informatik); Burkhard Rost (Technische Universität München , I12 - Informatik); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One common definition of regions of  “disorder” in proteins is that they do not adopt a regular three-dimensional  (3D) structure in isolation (i. e., when not bound to other molecules) on their own. These disordered regions are in contrast to regions that are well structured or “ordered”. Notably, there is a great variety of “flavors” of disorder: some adopt a unique regular 3D secondary structure only upon binding; others, for example loops, remain irregular; some proteins are almost entirely disordered and others have only short disordered regions.<br>Numerous computational methods exist that predict disorder based on a variety of concepts. One of these methods, NORSnet, has been developed in our group. NORSnet aims to predict disordered regions of the “loopy” type. Here, we predict secondary structure and disorder for all completely sequenced organisms (~ 4 million proteins, UNIPROT, December 2010). We report two observations: (1) Most of the predicted disorder regions are also predicted as loops as expected by the design of the method. (2) Only half of the long loops are actually predicted as disorder.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W53'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W53</strong></td>
 </tr> <tr> 
  <td><b>Environment Specific Substitution Tables Improve Membrane Protein Alignment</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jamie Hill <i>University of Oxford</i></td>
 </tr>
 <tr> 
  <td>  Sebastian Kelm (University of Oxford, Statistics); Jiye Shi (UCB Celltech, Research & Development); Charlotte Deane (University of Oxford, Statistics); Jiye Shi (Shanghai University, Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 80.<br><br>Motivation: Membrane proteins are both abundant and important in cells, but the small number of solved structures restricts our understanding of them. Here we consider whether membrane proteins undergo different substitutions from their soluble counterparts and whether these can be used to improve membrane protein alignments, and therefore improve prediction of their structure.<br><br>Results: We construct environment-specific substitution tables for membrane proteins. These are markedly different from tables for soluble proteins. For example, substitution preferences in lipid tail-contacting parts of membrane proteins are found to be distinct from all environments in soluble proteins, including buried residues. From a principal component analysis of the tables, the most important determinant of substitution preferences is hydrophobicity, followed by secondary structure. We demonstrate the use of our tables in sequence-to-structure alignments of membrane proteins using the FUGUE alignment program. On average, in the 10 - 25% sequence identity range, alignments are improved by 28 correctly aligned residues compared with the default FUGUE tables. Coordinate generation from our alignments yields improved structure models.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W54'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W54</strong></td>
 </tr> <tr> 
  <td><b>Quality or Quantity of Structural Dynamics Information: That is the Question! (when improving performance of structure-based function prediction methods)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dariya Glazer <i>Stanford University</i></td>
 </tr>
 <tr> 
  <td>  Grace  Tang (Stanford University, Bioengineering); Vijay  Pande (Stanford University, Chemistry, Structural Biology, and Computer Science); Russ Altman (Stanford University, Bioengineering, Genetics, and Medicine); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Previously, we showed that incorporating structural dynamics information from molecular simulations improved performance of structure-based function prediction methods. This raised new intriguing questions. Does the choice of force field, the length of the simulation trajectories, or the presence of a ligand in the simulation systems have any effect? Are there any meaningful descriptors of the true positive results that can help discriminate those from the false positive ones?<br><br>To this end, we scaled up our efforts in molecular dynamics simulations with Folding@Home distributed computing. Using 5 force fields (Amber ’96, ‘99sb, '03, Gromos53a6, and OPLS-AA) we generated 14 multiples of 10 nanosecond trajectories (a total of 140 nanoseconds) with and without the presence of calcium ions for 11 pairs of structures, HOLO/APO for calcium binding. Resulting structural ensembles were evaluated for calcium binding sites using FEATURE.  <br><br>Our results indicate that different force fields explore somewhat different conformational space of the molecules with respect to the sensitivities of FEATURE. Longer simulation trajectories and inclusion of calcium ions in the simulation systems did not yield a greater improvement in FEATURE’s performance. We propose 6 descriptors of sites identified by FEATURE that can be intelligently combined by machine learning algorithms in order to maximize true positive and minimize false positive results. <br><br>Future efforts in structure-based function prediction, especially identifying calcium binding sites, could benefit from applying function prediction methods to structural ensembles of the molecules of interest obtained from several copies of short-scale molecular dynamics simulations employing at least two general force fields.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W55'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W55</strong></td>
 </tr> <tr> 
  <td><b>Nearest-Neighbor Approaches to Predict Protein Function by Sequence Homology Alone</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tobias Hamp <i>TU Muenchen</i></td>
 </tr>
 <tr> 
  <td>  Rebecca Kassner (TU Muenchen, Institut fuer Informatik); Stefan Seemayer (TU Muenchen, Institut fuer Informatik); Esmeralda Vicedo (TU Muenchen, Institut fuer Informatik); Christian Schaefer (TU Muenchen, Institut fuer Informatik); Dominik Achten (TU Muenchen, Institut fuer Informatik); Florian Auer (TU Muenchen, Institut fuer Informatik); Ariane Boehm (TU Muenchen, Institut fuer Informatik); Tatjana Braun (TU Muenchen, Institut fuer Informatik); Maximilian Hecht (TU Muenchen, Institut fuer Informatik); Mark Heron (TU Muenchen, Institut fuer Informatik); Peter Hoenigschmid (TU Muenchen, Institut fuer Informatik); Thomas Hopf (TU Muenchen, Institut fuer Informatik); Stefanie Kaufmann (TU Muenchen, Institut fuer Informatik); Michael Kiening (TU Muenchen, Institut fuer Informatik); Denis Krompass (TU Muenchen, Institut fuer Informatik); Cedric Landerer (TU Muenchen, Institut fuer Informatik); Yannick Mahlich (TU Muenchen, Institut fuer Informatik); Manfred Roos (TU Muenchen, Institut fuer Informatik); Burkhard Rost (TU Muenchen, Institut fuer Informatik); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This year's Automated Function Prediction SIG meeting at ISMB/ECCB 2011 features, for the first time, the so-called CAFA challenge. <br>Participants were supposed to computationally predict and submit the Gene Ontology (GO) terms for over 40,000 so far functionally unannotated targets.<br>In this context, we have developed three different Nearest-Neighbor-based methods to derive the functions of a protein by sequence homology alone. Despite being conceptually the same, they largely differ in their technical details, ranging from an emphasis on superior ad-hoc performance to sophisticated and well-defined scoring schemes, allowing to direct either recall or precision into the desired reliability range. <br>All three methods were employed to predict all the CAFA targets mentioned before and are, in a way, supposed to act as controls for an expected wealth of more sophisticated function prediction machineries at the CAFA meeting.<br>In addition, we have created a meta-classifier to combine the three approaches and established new ways to generally capture the performance of function prediction tools with GO terms. The latter now enable us to not only estimate precision and recall as described in the official CAFA rules, but also to use the shapes of the GO subgraphs of a target protein to create an assessment that considers its biologically distinct functions.<br>An own pre-CAFA evaluation of our methods with 10,000 random targets from the SwissProt database already showed remarkable performances of up to 87% precision with 84% recall using the official standard protein centric measure and a greatly outperformed class of random prediction models.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W56'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W56</strong></td>
 </tr> <tr> 
  <td><b>Efficiently Represent RNA Ensemble</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Reazur Rahman <i>Purdue University</i></td>
 </tr>
 <tr> 
  <td>  Michael Gribskov (Purdue University, Biological Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Minimum Free Energy (MFE) based approaches to RNA structure prediction provide structures that contain only a fraction of the biologically relevant stems, due to limitations of the algorithm and inaccuracies in energy parameters. One can analyze the ensemble of near MFE structures to find other biologically relevant stems, including pseudoknots, that are not found in the MFE predicted structure. Unfortunately, with increasing free energy difference from the predicted MFE structure, the number of suboptimal structures increases rapidly along with the number of stems. Conserved stems can be identified by comparing ensembles of structures represented as RNA graphs, where graph nodes are stems and the relationships between stems are different types of edges. Since graph matching is an NP-complete problem, large graphs make the task of comparison less tractable.<br><br>Our aim is to reduce the complexity of these graphs by removing similar stems while preserving biologically important stems. First, we theoretically enumerate all possible stems that can be formed with respect to a given stem. We identify cases with topologically similar stems that can be combined to reduce the graph size without significantly altering the overall ensemble of topological relationships. This provides the basis for developing a set of heuristics to contract RNA graphs. We apply our heuristic rules on a well curated dataset, containing Rnase P, Group I Intron, tmRNA and tRNA sequences, and report the effectiveness of each heuristic.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W57'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W57</strong></td>
 </tr> <tr> 
  <td><b>Chemical shift prediction using graph kernels</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nino Shervashidze <i>Max Planck Institutes Tübingen</i></td>
 </tr>
 <tr> 
  <td>  Michael Habeck (Max Planck Institutes Tübingen, Computational Structural Biology); Karsten Borgwardt (Max Planck Institutes Tübingen, Machine Learning and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In this work we consider the problem of  learning to predict chemical shifts of different atomic nuclei in a protein based on the tertiary structure of the protein. <br>We tackle the problem by regressing the chemical shift value of a nucleus based on its local neighborhood.<br>In more detail, our approach proceeds as follows: We first represent the local neighborhood of each such nucleus as a graph. This is done by identifying the nodes of each graph with the atoms within a given distance threshold from the considered nucleus and assigning them discrete labels according to the types of the corresponding atoms; The distances between the nodes are determined from the tertiary structure of the protein.<br>Second, we represent these graphs in a feature space using a variant of the Weisfeiler-Lehman kernel by Shervashidze and Borgwardt (2009). In the end, we predict the chemical shift values using a regression method on these representations of the graphs.<br>In this poster, we will describe our approach in detail, as well as present and discuss the empirical results of our study in the light of results given by other state-of-the-art chemical shift prediction methods.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W58'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W58</strong></td>
 </tr> <tr> 
  <td><b>Is the biological hydrophobicity scale correct?</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Christoph Peters <i>Stockholm University</i></td>
 </tr>
 <tr> 
  <td>  Arne Elofsson (Stockholm University, Dep of Biochemistry and Biophysics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Alpha helical membrane proteins are inserted into the membrane by the Sec-translocon. When sufficiently hydrophobic segments pass through the translocon they are recognized and passed into the membrane. This process can be described by a simple equilibrium process. In a set of experiments Gunnar von Heijne and co-workers performed direct measurements of the contribution of individual amino acids to the free energy of insertion (Hessa, 2005, 2007 Nature). These measures where then used to create a "biological hydrophobicity scale". In general this scale correlates quite well with other hydrophobicity scales but it differs in detail. This discrepancy has caused some discussion in the recent literature (Johansson & Lindahl PNAS 2009; Gumbart, Chipot & Schulten PNAS 2011).<br><br>We have recently showed that using the biological hydrophobicity scale it is possible to develop a method for membrane-protein topology prediction that can challenge the most advances machine learning algorithms. Scampi is based on a very simple hidden-Markov model like algorithm and only has two parameters that can be optimized. We assumed that this excellent performance was due to the high quality of the biological hydrophobicity scale. <br><br>Here, we want to test if the biological hydrophobicity scale really provides a more accurate picture of the insertion mechanism than other hydrophobicity scales, by optimizing the SCAMPI algorithm using different hydrophobicity scales. Preliminary results show that other hydrophobicity scales lead to a decrease in the topology prediction accuracy of Scampi, i.e. the biological hydrophobicity scale does indeed describe the recognition of transmembrane segments by the translocon better than other scales.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W59'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W59</strong></td>
 </tr> <tr> 
  <td><b>A computational analysis to identify putative residues responsible for specific activity variation in Sucrose Phosphate Phosphatase (SPP)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Divya P Syamala Devi <i>Sugarcane Breeding Institute, India ; Currently at National Centre for Biological Sciences, Bangalore</i><br></td>
 </tr>
 <tr> 
  <td>Divya Syamala Devi (Sugarcane Breeding Institute, India ; Currently at National Centre for Biological Sciences, Bangalore) Neethi Jayaraman (Sugarcane Breeding Institute, Division of Crop Improvement); Subramonian N (Sugarcane Breeding Institute, Division of Crop Improvement); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: This poster is based on Proceedings Submission 218<br>Sucrose phosphate synthase(SPP) is one of the key enzymes in the sucrose biosynthesis pathway. Sequence and structural comparison between the orthologs of this enzyme identified crucial residue changes that might have contributed to the activity variation between them.  We report minute structural variations in active site regions as deduced from GDT scores calculated for fragments containing reported active site residues. The analysis suggests the role of two residues - one phosphate binding and another sucrose binding- in causing variation in catalytic activity among orthologs of the enzyme. We also report a substitution observed in rice SPP that can disrupt stabilizing interaction between sucrose and the enzyme, and thereby contributing to a relatively higher specific activity of rice SPP. Phylogenetic studies based on sequence and structure reveal that the evolutionary relationship of SPP orthologs depends on the over all sequence as well as architectural differences.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W60'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W60</strong></td>
 </tr> <tr> 
  <td><b>A comprehensive review of protein fold space and the correlation of structure with function</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Spencer Bliven <i>University of California, San Diago</i></td>
 </tr>
 <tr> 
  <td>  Andreas Prlic (San University of California, San Diego, San Diego Supercomputer Center); Philip Bourne (University of California, San Diego, Skaggs School of Pharmacy and Pharmaceutical Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Numerous studies have attempted to characterize protein fold space.  Existing classification schemes such as SCOP and CATH tend to view protein structures as belong to a finite number of discrete protein folds. Others prefer to view protein space as continuum of possible structures, of which only a few have been sampled.  However, previous debate about the topic has been primarily qualitative and philosophical, with examples being cited in support of both discrete and continuous viewpoints [1-4].  In this study, with the aid of specialized hardware, we comprehensively compare protein structures, with the goal of quantitatively assessing the continuity or discreteness of fold space.<br><br>Protein fold space was mapped for the complete non-redundant Protein Data Bank (PDB) using the FATCAT algorithm for structural comparison [5] on the OpenScienceGrid compute cluster, yielding 189,031,127 pairwise comparisons. Conceptually, the pairwise protein comparisons form a large graph, where the edges correspond to the similarity between proteins.  By analyzing this graph it is possible to quantitatively measure the similarity between protein folds.  We find that this graph is highly connected, providing support to the hypothesis that fold space is continuous. Results of mapping the graph to SCOP and to functional annotations will be reported.<br><br>[1]Sadreyev et al. Curr Opin Struct Biol (2009) vol.19 (3) pp. 321-8<br>[2]Sippl. Curr Opin Struct Biol (2009) vol.19 (3) pp. 312-20<br>[3]Govindarajan et al. Proteins (1999) vol.35 (4) pp. 408-14<br>[4]Shindyalov and Bourne. Proteins (2000) vol. 38 (3) pp. 247-60<br>[5]Prlic et al. Bioinformatics (2010) vol. 26 (23) pp. 2983-2985</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W61'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W61</strong></td>
 </tr> <tr> 
  <td><b>Protein interaction partners revealed by their dynamical properties</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jau-Ji Lin <i>National Chiao Tung University</i></td>
 </tr>
 <tr> 
  <td>  Jenn-Kang Hwang (National Chiao Tung University, Institute of Bioinformatics and Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The biological processes in cells are carried out by various kinds of protein molecules that perform their functions by interacting with other proteins. Identifying the interacting partners of proteins is crucial to the constructing of the biological pathways in a cell. Recently we developed a method called WCN (Lin et al., 2008) to calculate the protein dynamical properties from its folded structure directly and efficiently. In this work, we analyze the dynamical properties of interacting proteins. We find that these proteins display a “complementary” relationship in their profiles. Therefore we analyze the dynamical profiles of all protein structures in PDB and try to elucidate the relationships between them. The dynamical profiles reveal that proteins do have their specific “disposition” to interact with certain kinds of protein molecules. By inspecting the dynamical profiles between each pair of protein structures in PDB, we can construct a network with nodes and edges representing the proteins and the interactions between them. The resulting network is composed of several sub networks, and most of them agree with the available protein-protein interaction data. In addition, we use CELLO (Yu et al., 2006) to predict the subcellular localization of proteins being proposed as interacting partners and find most of them have the same subcellular localization. This finding confirms the space requirement of proteins to interact with each other. This study constitutes a novel methodology of constructing protein-protein interaction networks and provides a promising way to identify interacting partners of proteins of unknown function with structural information only.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W62'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W62</strong></td>
 </tr> <tr> 
  <td><b>Predicting peptidase substrate profiles using computational methods</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Haslina Hashim <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Peptidases are ubiquitous enzymes which hydrolyze peptide bonds and constitute around 2% of the genomes of all organisms. With an estimated 14% of the 500 human peptidases under investigation as drug targets from the ~550 active and putative peptidases in the human genome, they are important for human health. Given their abundance and pharmaceutical importance, there are still many peptidases whose function is not well characterised. According to the MEROPS peptidase database, out of the 95,150 known peptidase sequences only 415 have known structures, yet this covers the majority of MEROPS families. Hence, comparative modelling is a viable proposition to predict peptidase structure and specificity for most of these enzymes. We have used MODELLER and FoldX to build peptidase structures interacting with candidate substrates, using inhibitor structures as templates, in order to predict subsite specificity from FoldX interaction energies. We have evaluated the protocol using a well characterised test system, trypsin, where structures of 11 variants at the P1 position and associated binding free energies are known. The results show promise in predicting the known specificity profile of trypsin at the P1 subsite, using the actual enzyme structure as a template. Similarly, in a second example, the method can also predict the P4 specificity of Pepsin A with success. However, we will also present results which show how performance is affected when using modelled enzyme structures at a range of similarities, highlighting the true utility of comparative modelling in predicting biological function.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W63'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W63</strong></td>
 </tr> <tr> 
  <td><b>SNPeffect 4.0: Molecular and structural phenotyping of human SNPs and disease mutations</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Greet De Baets <i>Flanders Institute for Biotechnology</i></td>
 </tr>
 <tr> 
  <td>  Joost Van Durme (VIB, SWITCH); Frederic  Rousseau (VIB, SWITCH); Joost Schymkowitz (VIB, SWITCH); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Single nucleotide polymorphisms (SNPs) are, together with copy number variation, the primary source of variation in the human genome and are associated with altered response to drug treatment, susceptibility to disease, and other phenotypic variation. Linking structural effects of non-synonymous SNPs to functional outcomes is a major issue in structural bioinformatics, and many tools and studies have shown that specific structural properties such as stability and residue burial can be used to distinguish neutral variations and disease associated mutations.<br>The SNPeffect database uses sequence- and structure-based bioinformatics tools to predict the effect on the molecular phenotype of proteins. It integrates Tango (an aggregation predictor, http://tango.crg.es/); Waltz (a predictor of amyloid forming sequences, http://waltz.switchlab.org/); Limbo (a predictor for chaperone specificity, http://limbo.switchlab.org/); and FoldX (http://foldx.switchlab.org/) that reports the ??G, the change in free energy upon mutation. In that way, FoldX predicts the effect of SNPs in two categories of functional properties: (1) structural and thermodynamic properties affecting protein dynamics and stability and (2) the integrity of functional and binding sites.<br>The database already contains the annotations for the UniProt set of human disease and polymorphism mutations, but users can also submit their own set of mutations for analysis.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W64'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W64</strong></td>
 </tr> <tr> 
  <td><b>canSAR 3D: Building a 3D Structural Map of Cancer</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Krishna C Bulusu <i>Institute of Cancer Research</i><br></td>
 </tr>
 <tr> 
  <td>Krishna Bulusu (Institute of Cancer Research) Bissan Al-Lazikani (Institute of Cancer Research, CRUK Cancer Therapeutics Unit); Mark Halling-Brown (Institute of Cancer Research, CRUK Cancer Therapeutics Unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein 3D structural data is a very powerful tool in drug design and development. Understanding the rules governing protein conformational movement is vital in understanding the structure-function correlation. This research is aimed at studying the structural basis of function, pathology and drug binding in Cancer-associated protein families using integrative in-silico techniques. 3D structural data available in the public domain and also within the Institute is integrated with chemogenomic data available to understand structural variation within a protein family and how it influences the protein’s role in cancer pathways. This is achieved by performing structural comparisons using superpositions and family-based clustering to analyse the effect of conformational changes on function. Ligand-binding footprints will be derived by studying the binding sites and mapped to the chemical structures of small molecule ligands. The knowledge derived will be utilised to build a library of curated and validated 3D models of Cancer-specific proteins.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W65'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W65</strong></td>
 </tr> <tr> 
  <td><b>Is It Biologically Relevant? An Evolutionary Method for Distinguishing Biological Interfaces from Crystal Contacts</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Guido Capitani <i>Paul Scherrer Institut</i></td>
 </tr>
 <tr> 
  <td>  Jose Duarte (Paul Scherrer Institut, Biomolecular research); Schärer Martin (Paul Scherrer Institut, Biomolecular research); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Thanks to technical and conceptual advances, macromolecular crystallography is nowadays a technique that can achieve the structure determination of very complex molecular objects. Such complexity makes it increasingly difficult to distinguish by visual inspection the two different types of interfaces found in protein crystals: biologically relevant ones and non-specific ones, corresponding to crystal lattice contacts. A need thus exists for computational tools capable of assigning a given interface as either “biological” or as “crystal contact”. To this end we devised a novel indicator called CRK (1) and a software tool to compute it. The CRK indicator compares the selection pressure acting, on average, on interface residues that are fully buried upon protein-protein interface formation (the "core" residues) and on those that are only partially buried (the "rim" residues). For a biologically relevant interface, the average selection pressure should be stronger on the core residues than on the rim residues, while in a crystal contact interface it is assumed not to differ significantly between the rim and core set. The CRK software evaluates selection pressure at the single residue level by calculating Ka/Ks ratios with SELECTON (2), or also by sequence entropy.<br>CRK distinguishes the two types of interfaces very effectively. It can also be used for structure validation purposes. A new version of the CRK software is being written, with the goal of being user-friendly and easy to install.<br> (1) Schaerer et al (2010), Proteins 78:2707-2713<br> (2) Stern et al (2007), Nucleic Acids Res 35:W506-W511</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W66'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W66</strong></td>
 </tr> <tr> 
  <td><b>SAHG, a comprehensive database of annotated structure models for human proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Chie Motono <i>National Institute of Advanced Industrial Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  Ryotaro Koike (Nagoya University, Graduate School of Information Science); Kana Shimizu  (National Institute of Advanced Industrial Science and Technology, Computational Biology Research Center); Matsuyuki Shirota (Tohoku University, Graduate School of Information Science); Takayuki Amemiya (Nagoya University, Graduate School of Information Science); Kentaro Tomii (Institution National Institute of Advanced Industrial Science and Technology , Computational Biology Research Center); Nozomi Nagano (Institution National Institute of Advanced Industrial Science and Technology , Computational Biology Research Center); Hidekazu Hiroaki (Kobe University, Graduate School of Medicine); Tsuyoshi Shirai (Nagahama Institute of Bioscience and Technology, Department of Bioscience); Kengo Kinoshita (Tohoku University, Graduate School of Information Science); Tamotsu Noguchi (National Institute of Advanced Industrial Science and Technology, Computational Biology Research Center); Motonori Ota (Nagoya University, Graduate School of Information Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We have constructed a novel database, SAHG, Structural Atlas of Human Genome (http://bird.cbrc.jp/sahg/), which exhibits protein structure models encoded in the human genome. All of the Open Reading Frames in the human genome are subjected to a fully automated, exhaustive protein-structure-prediction pipeline to generate protein structure models. SAHG contains 42,577 domain-structure models in ~24800 unique human protein sequences from the Refseq database. We believe that the SAHG database provides an up-to-date collection of annotated protein 3D models of reliable quality to users.<br>Compared to other existing databases of protein structure models, SAHG is distinct in that the pipeline is focus on proteins of higher organisms, and in that conformational changes of proteins were predicted and displayed as animated image.<br>To achieve the features, we developed the structure prediction methods suitable for the multi-domain proteins with considerable intrinsically disordered (ID) regions by the combination of local alignment methods (BLAST, PSI-BLAST, and the Smith-Waterman profile-profile alignment), global alignment method (FORTE and the probabilistic profile-profile alignment), prediction tool for ID regions, and MODELLER. Conformational changes of protein models upon ligand binding are predicted by the simultaneous modeling using the templates of apo and holo forms. When there were no suitable templates for holo forms, we prepared holo models using prediction methods for ligand-binding and conformational change (the elastic network model and the linear response theory). Models are displayed as animated images, which provide significant information for protein function when they are shown with functional residues and ligands.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W67'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W67</strong></td>
 </tr> <tr> 
  <td><b>Training bidirectional recurrent neural networks with Conjugate gradient-type algorithms for protein secondary structure prediction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Michalis Agathocleous <i>University of Cyprus</i><br></td>
 </tr>
 <tr> 
  <td>Chris Christodoulou (University of Cyprus) Petros Kountouris (University of Cyprus, Computer Science ); Vasilis Promponas (University of Cyprus, Biological Sciences ); Georgia Christodoulou (University of Cyprus, Computer Science  ); Vassilis Vassiliades (University of Cyprus, Computer Science  ); Chris Christodoulou (University of Cyprus, Computer Science  ); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein Secondary Structure Prediction (PSSP) is an important step to predict the protein 3D structure from an amino acid sequence, which is essential for the exploration of protein functions. Recent PSSP research has managed to achieve an overall result around 80% prediction accuracy, utilizing various machine learning techniques. One approach uses a Bidirectional Recurrent Neural Network (BRNN) architecture and an extension of the backpropagation through structure algorithm with the error propagated through time in both directions of the BRNN. The BRNN has proved to be a very efficient architecture for PSSP, since it can capture both upstream and downstream information. In order to increase the prediction capabilities of the BRNN architecture and achieve better results for the PSSP problem, we have implemented a second-order learning algorithm to replace the conventional backpropagation through time algorithm. It has been widely proven that second-order learning algorithms are superior to the backpropagation algorithm in training multilayer networks. In particular, a variant of the traditional conjugate gradient (CG) algorithm, the scaled-conjugate gradient (SCG) has been shown to be very effective when used on large datasets and faster than the standard backpropagation algorithm. Even though the standard CG has been developed for simple recurrent neural networks, to the best of our knowledge it has never been developed and implemented for BRNN architectures. In this paper, we theoretically develop and implement the SCG for the BRNN and apply to PSSP. Our preliminary results both on learning speed and on PSSP accuracy are very promising.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W68'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W68</strong></td>
 </tr> <tr> 
  <td><b>Protein Sequence Design Using Ensemble-Based Energetic Information</b><br/><br/></td>
 </tr>
 <tr> 
 <td>James Wrabl <i>Johns Hopkins University</i></td>
 </tr>
 <tr> 
  <td>  Shayer Chowdhury (Johns Hopkins University, Biology); Vincent Hilser (Johns Hopkins University, Biology and Biophysics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Despite advances in computational technology and theoretical understanding, rational selection of amino acid sequences that adopt a desired protein structure is still a formidable challenge.  This work explores a preliminary approach to this design problem using information derived from a previously reported (and experimentally validated) statistical mechanical ensemble description of globular protein thermodynamics.  Information about both the native and denatured state energetics of the design target is taken into account, as we hypothesize that the denatured state in particular cryptically encodes necessary negative design information.  An algorithm encoding the strategy is developed that automatically generates amino acid sequences thought to be energetically compatible with an arbitrary target structure.  These designed sequences indeed demonstrate primary, secondary, and tertiary structure properties similar to the target in silico.  Interestingly, although designed sequences exhibit twilight-zone levels of sequence identity to the target, no designed amino acid sequence has yet been observed to identically match a known entry in publicly available databanks.  Results of experimental overexpression, purification, and biophysical characterization of sequence designs for the specific example of SH3 domain are presented.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W69'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W69</strong></td>
 </tr> <tr> 
  <td><b>Predicting Functions of the Universal Stress Proteins in Metabolically Versatile Rhodopseudomonas palustris</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Shaneka Simmons <i>Jackson State University</i></td>
 </tr>
 <tr> 
  <td>  Alexander  Ropelewski (Carnegie Mellon University, Pittsburgh Supercomputing Center); Hari Cohly (Jackson State University, Biology); Hugh Nicholas (Carnegie Mellon University, Pittsburgh Supercomputing Center); Raphael Isokpehi (Jackson State University, Center for Bioinformatics & Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genes encoding the universal stress protein (USP) domain (Pfam PF00582) are induced during a plethora of environmental stress stimuli. The USPs of Methanococcus jannaschii MJ0577 and Haemophilus influenzae have been well characterized as asymmetric dimers with a tertiary alpha/beta fold and predicted to be associated with ATP binding or non-ATP binding motifs, respectively. However, the biochemical mechanisms and structural architecture defining the diverse functional roles USPs in other bacterial species remains largely unknown. Public availability of finished genome sequences housed at Integrated Microbial Genome (IMG) system, a comprehensive microbial database, allows for investigation of microbial genes encoding proteins containing the PF00582 domain. The metabolically versatile, bioenergy producing microbe, Rhodopseudomonas palustris, has 6 finished genome sequences from strains BisA53, BisB5, BisB18, CGA009, HaA2 and TIE-1, representing the highest number of USP-containing genomes and genes in microbes identified as important to the Department of Energy’s mission for bioenergy production. USPs within finished R. palustris genome sequences were retrieved from IMG system and analyzed to characterize the functional and biochemical roles of the USP domain. Fifty-one R. palustris USPs were grouped according to amino acid length. Twenty sequences ranging from 274 aa to 293 aa were aligned with Clustal X Gonnet series protein weight matrix and segregated into conserved familial sequence motifs using the MEME motif analysis algorithm. Phylogenetic analysis revealed four R. palustris Usp-containing clades. Annotations based on INTERPRO domains identified evidence of Rossmann-like fold topology within each sequence. We are currently assessing the ATP-binding capabilities of the 20 prioritized proteins.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W70'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W70</strong></td>
 </tr> <tr> 
  <td><b>The Impact of Multifunctional Genes on "Guilt by Association" Analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jesse Gillis <i>University of British Columbia</i></td>
 </tr>
 <tr> 
  <td>  Paul Pavlidis (University of British Columbia, Psychiatry and Centre for High-throughput Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Many previous studies have shown that by using variants of “guilt-by-association”, gene function predictions can be made with very high statistical confidence. In these studies, it is assumed that the “associations” in the data (e.g., protein interaction partners) of a gene are necessary in establishing “guilt”. In this work we show that multifunctionality, rather than association, is a primary driver of gene function prediction. We first show that knowledge of the degree of multifunctionality alone can produce astonishingly strong performance when used as a predictor of gene function. We then demonstrate how multifunctionality is encoded in gene interaction data (such as protein interactions and coexpression networks) and how this can feed forward into gene function prediction algorithms. We find that high-quality gene function predictions can be made using data that possesses no information on which gene interacts with which. By examining a wide range of networks from mouse, human and yeast, as well as multiple prediction methods and evaluation metrics, we provide evidence that this problem is pervasive and does not reflect the failings of any particular algorithm or data type. We propose computational controls that can be used to provide more meaningful control when estimating gene function prediction performance. We suggest that this source of bias due to multifunctionality is important to control for, with widespread implications for the interpretation of genomics studies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W71'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W71</strong></td>
 </tr> <tr> 
  <td><b>Understanding protein-protein interactions: Efficient classification of physiological and non-physiological protein complexes and unexpected conservation in interaction interfaces</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dmitry Korkin <i>University of Missouri-Columbia</i></td>
 </tr>
 <tr> 
  <td>  Nan Zhao (University of Missouri-Columbia, Dept. of Computer Science and Informatics Institute); Bin Pang (University of Missouri-Columbia, Dept. of Computer Science and Informatics Institute); Chi-Ren Shyu (University of Missouri-Columbia, Dept. of Computer Science and Informatics Institute); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Detailed structural knowledge about protein-protein interactions can provide insights to the basic processes underlying cell function. In this work, we address two problems concerning protein-protein interactions. The first problem is to determine whether an interaction formed by a pair of proteins is physiological or it is an artifact of computational or experimental methods. Solving this problem is crucial for detection of crystal-packing interfaces and accurate protein-protein docking. We introduce two feature-based classifiers that are trained using (i) supervised and (ii) semi-supervised learning approaches. All classifiers employ a universal set of features extracted from the structures of protein-protein interfaces. Our classifiers reach 93% accuracy when classifying physiological crystal-packing interactions in PDB and 85% when classifying native and decoy interactions from protein docking. <br><br>Our second problem is concerned with understanding the role of charges residues in protein-protein interactions.  To address the problem, we propose a simple approach to study structural conservation of the charged residue pairs by analyzing the homologous binary complexes.  As a result, we find a novel conservation pattern, which we call the correlated reappearance of charged residues. The analysis of conservation patterns across different superkingdoms as well as structural classes of proteins has revealed that the correlated is by far the most prevalent conservation pattern. By linking our findings with the electrostatic steering mechanism, we propose that often it is not a specific location, but rather a mere presence of charged residue pairs in the protein interface that needs to be conserved to form a protein-protein interaction.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='W72'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster W72</strong></td>
 </tr> <tr> 
  <td><b>Protein Structural Domain Assignment Using Delaunay Tessellation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Todd Taylor <i>NIH</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: We describe a fully automated method of protein structural domain assignment using a Potts model that we call DePot (an abbreviation for Delaunay-Potts). Each amino acid residue is represented as a site in an irregular lattice derived from the Delaunay tessellation of the protein structure. Domain membership is represented by a spin value and each site has a spin that can change under the influence of its neighbors. Neighboring spins on the lattice are allowed to interact subject to an Ising ferromagnetic-like energy function until clusters of like spins emerge and these clusters define domains. DePot is simple and easy to implement and the assignments agree very well with previously published methods.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




