
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=F by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist4158.html?cat=F by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:17 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'F'- Evolution'</b></td>
 </tr>
</table>

<a name='F01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F01</strong></td>
 </tr> <tr> 
  <td><b>Multi-species integrative biclustering</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Richard Bonneau <i>New York University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A key challenge in the analysis of functional genomics data is the identification of modules of genes with similar regulatory controls, a non-trivial problem due to the complexity of regulatory networks. Recent works that compare functional genomics datasets for closely related species reveal that many co-regulated gene groups are conserved across several species. This suggests that comparative analysis of multiple-species functional genomics datasets could prove powerful in accurately identifying those conserved modules. <br>We describe an extension of our cMonkey algorithm that allows for the simultaneous biclustering of heterogeneous data collections spanning multiple species. We present results from the multi-species biclustering of a group of Gram positive bacteria containing Bacillus subtilis, Bacillus anthracis, and Listeria monocytogenes. We identify conserved groups of orthologous genes, yielding evolutionary insights into the formation and surprisingly high degree of conservation of regulatory modules across these three species. We report a temporal difference between the two Bacillus species in the expression of a conserved bicluster of metabolic genes required for spore formation. In addition, we discuss the unexpected identification of a highly expressed flagellum assembly bicluster in non-motile B. anthracis.  Analysis of biclusters obtained revealed a large number of gene groups with conserved modularity and high biological significance as judged by several measures of cluster quality. We show that the method provides a framework that allows data and insights from well-studied organisms to complement the analysis of related but less well studied organisms.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/19887.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='F02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F02</strong></td>
 </tr> <tr> 
  <td><b>Evolutionary comparisons between sequential datasets with horizontal dependencies infer tumor evolution</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Roland Schwarz <i>University of Cambridge</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In Schwarz et al 2010 we showed that rational kernels based on<br>finite-state transducers can perform pairwise sequence comparisons<br>with horizontal dependencies and extract accurate distance information<br>from divergent sets of sequences. We demonstrated the power of our<br>approach on the ambiguous phylogeny of the Chlorophyceae and are<br>currently extending it to infer tumor evolution within a single cancer<br>patient.<br><br>Copy number variation profiling is a powerful tool for measuring the<br>heterogeneity in cancer samples directly linked to patient<br>survival. Reconstructing the evolutionary history of a cancer within<br>one patient allows to discover driver events for resistance<br>development. We have obtained multiple Ovarian cancer samples from the<br>same patient and have developed a new distance measure based on the<br>method published in Schwarz et al 2010 to reconstruct the evolutionary<br>tree and the unsampled internal nodes including the cancer<br>precursor. Our model includes horizontal dependencies,<br>i.e. amplification and deletion events of arbitrary length over the<br>full CNV profiles.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20117.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='F03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F03</strong></td>
 </tr> <tr> 
  <td><b>Domain architecture conservation in orthologs</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kristoffer Forslund <i>Stockholm Bioinformatics Centre</i></td>
 </tr>
 <tr> 
  <td>  Erik Sonnhammer (Stockholm University, Stockholm Bioinformatics Centre); Isabella Pekkari (Stockholm University, Stockholm Bioinformatics Centre); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: As orthologous proteins are expected to retain function more often than other homologs, they are often used for functional annotation transfer between species. While the assumption of higher similarity in various respects of orthologs over paralogs at the same evolutionary separation is widespread and supported by many isolated examples, it has not been systematically evaluated, and some recent publications have questioned its general validity. Because of this, we set out to consistently validate the specific effect of orthologous versus paralogous evolution on the conservation of domain architectures in homologous proteins. <br><br>We carried out a large-scale study of domain architecture conservation or change on a large-scale dataset. We designed a score to measure domain architecture similarity and used it to analyze differences between orthologs and close outparalogs as defined by InParanoid. To compensate for differences between the pair types in average evolutionary divergence, we divided the protein comparisons into bins based on Jukes-Cantor corrected protein sequence identity, and analyzed the difference between ortholog and paralog pairs within each bin separately.<br><br>The analysis shows that orthologs exhibit significantly greater domain architecture conservation than non-orthologous homologs, even when differences in evolutionary divergence are compensated for. We interpret this as an indication of a stronger selective pressure on orthologs than on paralogs to retain the domain architecture required to perform a specific function. This supports the notion that orthologs are functionally more similar than other types of homologs at the same evolutionary distance.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F04</strong></td>
 </tr> <tr> 
  <td><b>The Operonic Location of Auto-Transcriptional Repressors is Highly Conserved in Bacteria</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tal Pupko <i>Tel-Aviv University</i></td>
 </tr>
 <tr> 
  <td>  Nimrod Rubinstein (Tel-Aviv Univeristy, Cell Research and Immunology); David Zeevi (Tel-Aviv Univeristy, Cell Research and Immunology); Yaara Oren (Tel-Aviv Univeristy, Cell Research and Immunology); Gil Segal (Tel-Aviv University, Molecular Microbiology and Biotechnology Department ); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Bacterial genes are commonly encoded in clusters, known as operons, which share transcriptional regulatory control and often encode functionally related proteins that take part in certain biological pathways. Regulatory related operons are known to colocalize in the genome, suggesting that their spatial organization is under selection for efficient expression regulation. However, the internal order of genes within operons is believed to be poorly conserved and hence expression requirements are claimed to be too weak to oppose gene rearrangements. In light of these opposing views we set out to investigate whether the internal location of the regulatory genes within operons is under selection. Our analysis shows that transcription factors are preferentially encoded either first or last in their operons, in the two diverged model bacteria Escherichia coli and Bacillus subtilis. In a higher resolution, we find that transcription factors that repress transcription of the operon in which they are encoded (autorepressors), contribute most of this signal by specific preference of the first operon position. We show that this trend is strikingly conserved throughout highly diverged bacterial phyla. Moreover, these autorepressors regulate operons that carry out highly diverse biological functions. We propose a model according to which autorepressors are selected to be located first in their operons in order to optimize transcription regulation. Specifically, the first operon position helps autorepressors to minimize leaky transcription of the operon structural genes, thus minimizing energy waste. Our robust statistical evidence corroborated with our suggested model thus reveals a paradigm of bacterial operon expression control.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F05</strong></td>
 </tr> <tr> 
  <td><b>Evolution of transition-metal binding domains</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Arye (Arik) Harel <i>Rutgers University</i><br></td>
 </tr>
 <tr> 
  <td>Arik Harel (Rutgers University) Arik Harel (Rutgers, Institute of Marine and Coastal Science & the Department of Biochemistry and Microbiology); Cheong Xin   Chan (Rutgers, Department of Ecology, Evolution and Natural Resources); Debashish Bhattacharya (Rutgers, Department of Ecology, Evolution and Natural Resources); Paul Falkowski (Rutgers, Institute of Marine and Coastal Science); Yana Bromberg (Rutgers, Department of Biochemistry and Microbiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein redox reactions are among the most important biochemical reactions due to their association with key biological processes like energy homeostasis and metabolism. Their basic machinery evolved in microbes early in our planet's history and has remained conserved throughout time. As amino acids are weak electron transfer reaction mediators, most of these reactions are performed using cofactors such as organic ligands and transition-metals. Recently, it has been suggested that the differential availability of different metal types in the Achaean ocean influenced both the evolution of metal-binding architectures and the selection of preferred binders. That is, iron, manganese, and molybdenum were preferentially used before the Great Oxidation Event (GOE), while copper and zinc utilization evolved after the GOE.<br>To further study the evolution of transition-metal binding domains and its relationship to the environmental trends, we extracted from Swiss-Prot 35,446 Oxidoreductases (Enzyme Commission group 1) and analyzed them for transition-metal binding. 298 InterPro transition-metal binding domains were identified in 7,227 proteins. As InterPro domains overlap, we further manually filtered this set to the most representative subset of 106 domains (median size 129 residues)  in 7,201 proteins. 86% of these sequences contain annotations confirming the presence of the metal binding residues. In order of decreasing abundance, domains binding iron, copper, molybdenum, manganese, and vanadium are present in our dataset. Here we describe our efforts to understand the origins of these domains, their relationships, and respective function evolution, independent of carrier organisms, using phylogenetic analysis and functional data reported in the literature.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F06</strong></td>
 </tr> <tr> 
  <td><b>Transcriptomics in an ecological key species: Adaptive variations in seagrass responses to rising water temperatures</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Susanne Franssen <i>Leibniz-Institute for Marine Sciences</i><br></td>
 </tr>
 <tr> 
  <td>Thorsten Reusch (Leibniz-Institute for Marine Sciences) Jenny Gu (University of Münster, Institute for Evolution and Biodiversity); Nina Bergmann (Leibniz-Institute for Marine Sciences, Evolutionary Ecology of Marine Fishes); Gidon Winters (University of Münster, Institute for Evolution and Biodiversity); Ulrich Klostermeier (Christian-Albrechts-University Kiel, Institute of Clinical Molecular Biology); Philip Rosenstiel (Christian-Albrechts-University Kiel, nstitute of Clinical Molecular Biology); Erich Bornberg-Bauer (University of Münster, Institute for Evolution and Biodiversity); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA-seq offers the opportunity to perform global transcription profiling of a key ecological species, predicting ecologically relevant responses under global warming. The seagrass Zostera marina, occurring along a thermal cline, provides the unique opportunity to assess temperature effects on gene expression as a function of their long term adaptation to local temperature regimes.<br><br>Here, natural populations from cold and warm adapted seagrass populations (Denmark, Italy) of Zostera marina were exposed to a realistic heat wave scenario in a common stress garden setup, capturing a two-point thermal reaction norm. Transcriptomic responses were obtained by RNA-seq of eight cDNA libraries, each comprising ~125 000 reads. The expression profiles were assessed subsequent to transcriptome de novo assembly and gene identification via orthologous plant genes.<br><br>Expression profiles revealed similar acute stress responses of both populations, with a focus on heat shock proteins. Population differences, however, became apparent at immediate heat recovery, characterised by a convergence to control expression of the warm adapted populations, while profiles of cold adapted genotypes diverged further from controls as well as acute heat responses. This divergent expression was characterized by a diverse set of gene functions dominated by protein degradation and RNA transcription regulation.<br><br>Results implicate that ecological experiments addressing gene expression differences of locally adapted populations may be misleading when only acute stress responses are considered. Moreover, we propose expressional flexibility to return to control expression as an important measure to predict the plasticity of individuals and hence, the fate of local populations under increasing climatic extremes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F07</strong></td>
 </tr> <tr> 
  <td><b>Phylogenetic analysis of mapped sequence reads</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jean-François Dufayard <i>CIRAD Montpellier</i></td>
 </tr>
 <tr> 
  <td>  Manuel Ruiz (CIRAD Montpellier, Hérault); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The aim of the project is to document variation in genomic patterns among a wide range of crops and wild relative species, during the domestication process and at a larger evolutionary scale. We analyse  transcriptomes of several individuals for eleven different crops. For each crop, we assemble a transcriptome of one individual, and then we map the sequence reads of the different individuals on this reference.<br>The main weakness of this method resides in the analyses of mapping results: most frequently, the only mapping relationships taken into account are the 1:1 ones. It means that many paralogy, polymorphism or allelism problems may break-off the analysis.<br>We propose a methodological improvement in three steps:<br>-a global clustering of consensus sequences from the individuals, and from the reference,<br>-a phylogenetic analysis of each cluster (MAFFT, GBlocks, PhyML + branch support statistics),<br>-a mapping based on tree topologies and branch supports.<br>The mapping of transcripts on the reference using phylogenetical methods permits a rigorous analysis of multigenic, multi-individual or allelic data.<br>For each mapping we are able to provide additional information:<br>-the number of paralogies in the family,<br>-the maximum or the mean divergence of the family,<br>-the maximum or the mean divergence of the mapped transcripts with the reference,<br>-the presence of highly supported incongruences in the topologies.<br>Such methods are able to infer rich mapping relationships, and important information to detect even individual-specific evolutionary events.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F08</strong></td>
 </tr> <tr> 
  <td><b>Transcriptomic and metabolomic investigation of thermal stress response through metabolic modulations in seagrasses</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jenny Gu <i>University of Muenster</i></td>
 </tr>
 <tr> 
  <td>  Susanne Franssen (University of Muenster, Institute for Evolution and Biodiversity); Katrin Weber (Heinrich-Heine University, Plant Biochemistry); Elisabeth Klemp (Heinrich-Heine University, Plant Biochemistry); Andreas  Weber (Heinrich-Heine University, Plant Biochemistry); Erich Bornberg-Bauer (University of Muenster, Institute for Evolution and Biodiversity); Gidon Winters (University of Muenster, Institute for Evolution and Biodiversity); Thorsten Reusch (Leibniz-Institute for Marine Sciences, Marine Ecology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: While chaperones are known to stabilize proteins against denaturing effects, metabolic adjustments modulating compatible osmolytes may prove to be a more viable strategy to counteract chronic thermal stress. A recent proteomic analysis of extremophiles with eScape, a protein stability predicting algorithm, suggests that global fitness of organisms to thermal stress may be achieved via interactions between the metabolome and the proteome.  This hypothesis is further investigated in seagrasses (Zostera marina and Zostera noltii) which have been subjected to longer heat waves with the recent climate change.  Metabolic adaptation may be the key alternate strategy for many organisms, particularly those that are sessile in aqueous environments and are therefore subjected to longer dosage effects of thermal stress due to the heat capacity of water compared to terrestrial organisms.  Metabolomics and transcriptomic studies using next generation sequencing for seagrasses collected from two populations (Denmark and Italy) were conducted.  Seagrasses were subjected to an ecologically relevant heat regime controlled within a mesocosm.  Significant metabolic adjustment during the thermal stress response of these two populations with different thermal tolerance has been observed, particularly with sucrose and fructose.<br><br>References:<br>Jenny Gu and Vincent J. Hilser.  Mol. Biol. Evol. 26(10):2217-2227.  2009</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F09</strong></td>
 </tr> <tr> 
  <td><b>Positive Feedback and Tradeoff as Sources for Complexity in Social Interaction System</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tomoyuki Yamamoto <i>Osaka University</i></td>
 </tr>
 <tr> 
  <td>  Yuya Yoshimura (Osaka University, Engineering Science); Hiroshi Ishiguro (Osaka University, Grad. School of Engineering Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A theoretical model of social system is investigated for the origin complexity of the social interaction system.<br>While "small world" theory seems to suggest existence of a positive feedback phenomenon in social systems, the phenomenon implies winner-takes-all situation that leads to simplification of the system falling into minimization problem on a energy landscape. <br>It does not correspond to a real social system, such as human society.<br>There must be another feature that originates diversity.<br><br>We hypothesized that the diversity of society is a tradeoff feature between social strength parameters, like diversion aspects of strength and weakness form a personality. <br><br>The simulation model is based on "DODOM" interaction model introduced by Hogeweg.<br>In our model, DOM value is extended to 2 or 3 dimensional vector. <br><br>While this interaction is a two-person game with a dilemma, a randomness and a positive feedback, our extension is adaptation of tradeoff between DOM components.<br>The interaction is done in only one component and other updated inversely; if an individual wins, DOM component used for the interaction increase and one of the rest decrease.<br><br>As the results, we found grouping and oscillating phenomenon in 2D system. <br>The grouping, differentiation into two groups, is maintained both of inter- and intra- group interaction, where cutoff of one of them results in reunification. <br>Also, in 3D, we obtained clustering and dynamical interchanging of members in addition to above results. We believe this self organizing hierarchical structure is obtained, as a source of complexity of society.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F10</strong></td>
 </tr> <tr> 
  <td><b>Novel phosphorylation modification sites in human proteins that originated after the human-chimpanzee divergence</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yoonsoo Hahn <i>Chung-Ang University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Phosphorylation modifications of specific protein residues are involved in a wide range of biological processes such as modulation of intracellular signal networks. Here, we present the development and application of a bioinformatics procedure for systematic identification of human-specific phosphorylation sites in proteins that may have occurred after the human-chimpanzee divergence. We collected annotated human phosphorylation sites and compared each site to orthologous mammalian proteins across taxa including chimpanzee, orangutan, rhesus macaque, marmoset, mouse, dog, cow, elephant, opossum, and platypus. We identified 39 human-specific gain of annotated phosphorylation sites in 37 proteins: 24 serines, 12 threonines, and three tyrosines. In many cases, the novel phosphorylation sites are situated in highly conserved segments of the protein. Proteins with novel phosphorylation sites are involved in crucial biological processes such as cell division (AURKB, CASC5, MKI67, and PDCD4) and chromatin remodeling (HIRA, HIRIP3, HIST1H1T, NAP1L4, and LRWD1). Modified phosphorylatable residues produce novel target sites for protein kinases such as cyclin-dependent kinases and casein kinases, possibly resulting in rewiring of phosphorylation regulatory networks. We propose that the gain of phosphorylation sites during human evolution may have altered protein interaction networks and resulted in the development of novel human-specific phenotypes.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F11</strong></td>
 </tr> <tr> 
  <td><b>Large tandem, higher order repeats (HOR) and regularly dispersed repeat units contribute substantially to divergence between human and chimpanzee Y chromosomes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Matko Glun?i? <i>University of Zagreb, Faculty of Science</i><br></td>
 </tr>
 <tr> 
  <td>Matko Glunčić (University of Zagreb, Faculty of Science) Vladimir Paar (Faculty of Science, University of Zagreb, Theoretical Physics Dept. ); Ivan Basar (Faculty of Science, University of Zagreb, Theoretical Physics Dept. ); Marija Rosandi? (University Hospital Rebro, Department of Internal Medicine); Mislav Cvitkovi? (Faculty of Science, University of Zagreb, Theoretical Physics Dept.); Petar Paar (Faculty of Electrical Engineering and Computing, Theoretical Dept.); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Comparison of human and chimpanzee genomes has received much attention recently, because of paramount role for understanding the evolutionary step distinguishing us from our closest living relative. In order to contribute to insight into Y chromosome evolutionary history, we study and compare tandems, HORs and regularly dispersed repeats in human and chimpanzee Y chromosome contigs, using robust Global Repeat Map (GRM) algorithm. We find Human Accelerated HOR Regions (HAHORs). In peripheral domains of 35mer human alphoid HORs we find riddled features containing ten additional repeat monomers. In chimpanzee we identify 30mer alphoid HOR. We construct alphoid HOR schemes showing significant human-chimpanzee difference, revealing rapid evolution after human-chimpanzee separation. We identify and analyze over twenty other large repeat units, most of them reported here for the first time: chimpanzee and human ~1.6 kb 3mer secondary repeat unit (SRU) and ~23.5 kb tertiary repeat unit (for ~0.55 kb primary repeat unit (PRU)); human 10848, 15775, 20309, 60910, and 72140 bp PRUs; human 3mer SRU (for ~2.4 kb PRU); 715mer and 1123mer SRUs (for 5mer PRU); chimpanzee 5096, 10762, 10853, 60523 bp PRUs; chimpanzee 64624 bp SRU (for 10853 bp PRU) and so on. We show that substantial human-chimpanzee differences are concentrated in large repeat structures, at the level of as much as ~70% divergence, sizably exceeding previous numerical estimates for some selected noncoding sequences. Smeared over the whole sequenced assembly (25 Mb) this gives ~14% human-chimpanzee divergence. This is significantly higher estimate of divergence between human and chimpanzee than previous estimates.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F12</strong></td>
 </tr> <tr> 
  <td><b>Distribution patterns of genomic elements in bacteria using genome linguistic methods.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Oliver Bezuidt <i>University of Pretoria</i></td>
 </tr>
 <tr> 
  <td>  Oleg Reva (University of Pretoria, Biochemistry); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: DNA molecules encoding functional enzymes, transcriptional regulators and virulence factors are fluxing through the bacterial taxonomic walls. They endow environmental and clinical strains of bacteria with new unexpected properties. Lateral genetic exchange, particularly of drug tolerance genes has been recognized for a long time; however the ontology of genomic islands and their donor-recipient relations remain generally obscure because of methodological problems. Prediction of insertion sites of these elements in host chromosomes still remain to be a challenge. Horizontally transferred genes are highly mutable and the mobilome entities having been inserted into host chromosomes undergo multiple events of fragmentation, partial duplications and deletions. This work is based on novel genome linguistics methods to study and visualize intrinsic relationships between mobile genetic elements in bacteria.  The method also determines the  relative insertion time of genomic islands in bacterial chromosomes using analytical stratigraphic approaches.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F13</strong></td>
 </tr> <tr> 
  <td><b>Gene expression evolution on the emergence of pathogenicity in Ascomycetes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Aminael Sanchez <i>Catholic University of Leuven</i><br></td>
 </tr>
 <tr> 
  <td>Aminael Sanchez-Rodriguez (Catholic University of Leuven) Aminael Sanchez-Rodriguez (Catholic University of Leuven, Microbial and Molecular Systems); Kristof Engelen (Catholic University of Leuven, Microbial and Molecular Systems); Riet De Smet (Ghent University, Plant Systems Biology); Qiang Fu (Catholic University of Leuven, Microbial and Molecular Systems); Yan Wu (Catholic University of Leuven, Microbial and Molecular Systems); Kathleen Marchal (Catholic University of Leuven, Microbial and Molecular Systems); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Ascomycetes form the largest phylum in the fungal kingdom. They are of special interest due to their broad spectrum of life styles including both plant and human pathogens. In a previous study we showed that most of the protein-coding genes needed for pathogenicity were already present in an ancestor common to both pathogenic and non-pathogenic Ascomycetes. Based on this finding that at least some of the coding potential related to pathogenicity is common to both pathogens and non-pathogens, we studied whether alterations in expression behavior could be correlated with a pathogenic versus a non pathogenic lifestyle. Using coexpression networks derived from large scale expression compendia for the non-pathogen N. crassa and the pathogens M. grisea and F. graminearum, we compared the coexpression behavior of true orthologs from gene families related to pathogenicity and occurring in each of the species. We found that the direct neighbours, of the respective orthologs in the coexpression network of the pathogen was largely different from its neighbours in the non-pathogenic species, implying a considerable rewiring of the coexpression network of these common orthologs. In addition we also observed that the expression evolution of paralogs in gene families that are common to both pathogens and non pathogens is different. More specifically we detected that the expression behavior of paralogs belonging to gene families that are involved in pathogen-host interaction tend evolve faster in pathogens than in non-pathogens.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F14</strong></td>
 </tr> <tr> 
  <td><b>Incorporating protein structure into coevolution analysis reveals functional side chain interactions in ABC-C exporters</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Attila Gulyas-Kovacs <i>postdoc/Rockefeller Universiy</i></td>
 </tr>
 <tr> 
  <td>  David C. Gadsby (Professor/Rockefeller University, Laboratory of Cardiac/Membrane Physiology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Prediction of coevolving amino acid positions from aligned sequences provides detailed insight into conserved molecular mechanisms in a protein family.  However, the accuracy of prediction is limited by the non-independence of sequence homologs and the variability of the rate of amino acid substitution across positions.  To mitigate these factors, a new approach is introduced, which takes advantage of protein structure to optimize the prediction of side chain contacts in three steps. One step tackles sequence non-independence by adjusting phylogenetic filtering of the alignment. Another addresses substitution rate heterogeneity by partitioning position pairs into rate classes followed by class weighting.  Separately or in combination, these optimization steps improved three out of four tested coevolution prediction methods. A third step, which weights and combines methods, yielded further improvement relative to individual methods.  These results were obtained using an alignment of ABC-C exporters, which underlie multidrug resistance of cancerous cells, insulin secretion, and cystic fibrosis disease. ABC-C proteins are characterized by highly asymmetric nucleotide binding domains, whose ATP-hydrolytic activity is signaled to the transport pathway gates via bundles of long helices that form intracellular linkers extending into the transmembrane domains. The new approach was harnessed to predict functional side chain interactions within and between the nucleotide binding domains and linker, or transmembrane, helices. Comparing outward- and inward-facing transporter conformations suggests that many interactions provide persistent stability to the helix bundles, whereas others occur transiently in a particular conformational state.  Intriguingly, some identified positions are implicated in cystic fibrosis.<br>[NIH-DK51767]</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F15</strong></td>
 </tr> <tr> 
  <td><b>Evolutionary meta-analysis reveals ancient constraints affecting missing heritability and reproducibility in disease association studies</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Joel Dudley <i>Stanford University</i></td>
 </tr>
 <tr> 
  <td>  Atul Butte (Stanford University, Systems Medicine); Sudhir Kumar (Arizona State University, Center for Evolutionary Medicine and Informatics); Rong Chen (Stanford University, Systems Medicine); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genome-wide disease association studies contrast genetic variation between disease cohorts and healthy populations to discover single nucleotide polymorphisms (SNPs) and other genetic markers revealing underlying genetic architectures of human diseases. Despite many large efforts over the past decade, these studies are yet to identify many reproducible genetic variants that explain significant proportions of the heritable risk of common human diseases. Here, we report results from a multi-species comparative genomic meta-analysis of 6,720 risk variants for more than 420 disease phenotypes reported in 1,814 studies, which is aimed at investigating the role of evolutionary histories of genomic positions on the discovery, reproducibility, and missing heritability of disease associated SNPs (dSNPs) identified in association studies. Our findings indicate that association studies are biased towards discovering rare variants, because strongly conserved positions only permit minor alleles with lowest frequencies. Using published data from a large case-control study, we demonstrate that the use of a straightforward multi-species evolutionary prior improves the power of association statistics to discover SNPs with reproducible genetic disease associations.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F16</strong></td>
 </tr> <tr> 
  <td><b>Evolutionary meta-analysis reveals ancient constraints affecting missing heritability and reproducibility in disease association studies</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Joel Dudley <i>Stanford University</i></td>
 </tr>
 <tr> 
  <td>  Rong Chen (Stanford University, Systems Medicine and Pediatrics); Atul Butte (Stanford University, Systems Medicine and Pediatrics); Sudhir Kumar (Arizona State University, Center for Evolutionary Medicine and Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genome-wide disease association studies contrast genetic variation between disease cohorts and healthy populations to discover single nucleotide polymorphisms (SNPs) and other genetic markers revealing underlying genetic architectures of human diseases. Despite many large efforts over the past decade, these studies are yet to identify many reproducible genetic variants that explain significant proportions of the heritable risk of common human diseases. Here, we report results from a multi-species comparative genomic meta-analysis of 6,720 risk variants for more than 420 disease phenotypes reported in 1,814 studies, which is aimed at investigating the role of evolutionary histories of genomic positions on the discovery, reproducibility, and missing heritability of disease associated SNPs (dSNPs) identified in association studies. We show that dSNPs are disproportionately discovered at conserved genomic loci in both coding and non-coding regions, as the effect size (odds ratio) of dSNPs relates strongly to the evolutionary conservation of their genomic positions. Our findings indicate that association studies are biased towards discovering rare variants, because strongly conserved positions only permit minor alleles with lowest frequencies. Using published data from a large case-control study, we demonstrate that the use of a straightforward multi-species evolutionary prior improves the power of association statistics to discover SNPs with reproducible genetic disease associations. Therefore, long-term evolutionary histories of genomic positions are poised to play a key role in reassessing data from existing disease association studies and in the design and analysis of future studies aimed at revealing the genetic basis of common human diseases.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F17</strong></td>
 </tr> <tr> 
  <td><b>A Phylogenetic Approach to Germline Methylome Reconstruction</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Lars Feuerbach <i>Max Planck Institut Informatik</i></td>
 </tr>
 <tr> 
  <td>  Rune B. Lyngsoe (Oxford University, Department for Statistics); Thomas Lengauer (Max Planck Institute Informatik, Computational Biology and Applied Algorithmics); Jotun Hein (Oxford University, Department for Statistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: DNA methylation of cytosine (C) is a molecular mechanism that establishes an epigenetic imprint, which can be maintained across cell division. Furthermore, it introduces a mutational burden that facilitates the transition of 5-methylcytosine (5mC) to thymine (T). Methylation in mammals is mainly targeted to cytosines that are followed by a guanine (CpG) and is considered the default state of this dinucleotide as, for instance, 70-80% of all CpGs are methylated in human tissues. As spontaneous deamination of 5mC is a relatively frequent chemical event, over the course of evolution genomic CpGs are being significantly depleted. This CpG decay has resulted in a six fold underrepresentation of CpGs in the human genome. In fact, the CpG to TpG/CpA mutations appear about 48 times more frequently than normal transversions. It can be assumed that CpG containing sequences of biological relevance are preferentially unmethylated in the germline to escape this mutational burden. Conversely, genome regions that have been persistently unmethylated in the germline over several million years acquire a notable footprint in their dinucleotide composition. In consequence, in unmethylated regions phenomena such as CpG island formation are observable.<br>We here propose a phylogeny based approach, which exploits the substantial differences in the mutation rates of methylated and unmethylated DNA. In simulation studies, the underlying algorithm robustly reconstructs the methylation state of the applied evolutionary constraints from homologous sequences. One of the intended applications of the approach is an enhanced identification of epigenetically active candidate biomarkers in methylation-based genome-wide association studies.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F18</strong></td>
 </tr> <tr> 
  <td><b>Probabilistic reconstruction of protein interaction histories using a simple measure of interaction affinity.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ryan Topping <i>Imperial College London</i></td>
 </tr>
 <tr> 
  <td>  John Pinney (Imperial College London, Molecular Biosciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The increasing availability of sequence and structural data describing protein complexes across species has encouraged attempts to understand the evolutionary histories of these complexes. However, previous attempts have lacked a thorough phylogenetic underpinning, attempting to model the re-wiring of protein-protein interactions whilst neglecting our knowledge of sequence evolution.  The inclusion of detailed phylogenetic information within a probabilistic model of interaction evolution will provide a better appreciation of uncertainties and permit the prediction of assembled complex structures in species for which we have no interaction data.<br><br>We present a probabilistic approach for the reconstruction of interaction histories in a complex of paralogous subunits across several species. The approach incorporates a maximum a posteriori reconciled gene tree and a model for interaction gain and loss (re-wiring events) during protein sequence evolution. In order to predict re-wiring events, we propose using a simple measure of binding affinity that takes into account the coevolution occurring at the protein-protein interface. Preliminary results describing an application of the method to reconstruct the history of the proteasome show improvements using this measure compared to previously used measures. Using a training set of protein x-ray crystal structures, we then fit a generalised model relating changes at a protein-protein interface to re-wiring events during evolution.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F19</strong></td>
 </tr> <tr> 
  <td><b>Next Generation Sequencing, Assembly, and Analysis of Influenza A (H1N1)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Simon Watson <i>Wellcome Trust Sanger Institute</i></td>
 </tr>
 <tr> 
  <td>  Eve Coulter (Wellcome Trust Sanger Institute, Viral Genomics); Gregory Baillie (Wellcome Trust Sanger Institute, Viral Genomics); Rachael Wash (Wellcome Trust Sanger Institute, Viral Genomics); Astrid Gall (Wellcome Trust Sanger Institute, Viral Genomics); Anne Palser (Wellcome Trust Sanger Institute, Viral Genomics); Monica Galiano (Health Protection Agency, Center for Infections); Maria Zambon (Health Protection Agency, Center for Infections); Oliver Pybus (University of Oxford, Department of Zoology); Andrew Rambaut (University of Edinburgh, Institute of Evolutionary Biology); Paul Kellam (Wellcome Trust Sanger Institute, Viral Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Viral gene sequencing and phylogenetics can be used to study the epidemiological dynamics of rapidly-evolving viruses. With complete genome data, it's possible to identify and trace individual transmission chains of viruses, like influenza, during the course of an epidemic. We have developed methods for next generation sequencing of influenza A and sequenced 26 influenza A H1N1 genomes from the 2009 pandemic. Samples were sequenced on an Illumina GAII and the resultant reads quality controlled and assembled against a reference sequence using a pipeline developed here called QUASR.<br>By comparing to other influenza A H1N1 2009 genomes from the UK, we demonstrate that the UK epidemic was composed of many co-circulating lineages, of which at least 12 were exclusively or predominantly UK clusters. The estimated divergence times of two clusters predate the detection of H1N1pdm in the UK, suggesting H1N1pdm was circulating in the UK before the first clinical case. Crucially, three clusters contain isolates from the second wave of infections in the UK, two of which represent chains of transmission that have persisted within the UK between the first and second waves.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F20</strong></td>
 </tr> <tr> 
  <td><b>TimeTree2: Species divergence times from molecular data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sudhir Kumar <i>The Biodesign Institute, Arizona State University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Scientists and non-scientists often need to know times of divergence between species but rarely can locate that information because it is buried in the scientific literature, usually in a format that is inaccessible to text search engines. We have developed a public knowledgebase that enables data-driven access to the collection of peer reviewed publications in molecular evolution and phylogenetics that have reported estimates of time of divergence between species. Users can query the TimeTree resource by providing two names of organisms (common or scientific) that can correspond to species or groups of species. The current TimeTree web resource (TimeTree2) contains timetrees reported from molecular clock analyses in 910 published studies and 17,341 species that span the diversity of life. TimeTree2 interprets complex and hierarchical data from these studies for each user query, which can be launched using a specially designed iPhone application, in addition to the website. Published time estimates are now readily accessible to the scientific community and general public, without requiring knowledge of evolutionary nomenclature.<br>TimeTree2 is accessible from the URL http://www.timetree.org, with an iPhone app available from iTunes (http://itunes.apple.com/us/app/timetree/id372842500?mt=8) and a YouTube tutorial (http://www.youtube.com/watch?v=CxmshZQciwo).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F21</strong></td>
 </tr> <tr> 
  <td><b>Identifying Patterns of Balancing Selection by Comparing the Bonobo, Chimpanzee and Human Genomes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kay Prüfer <i>Max-Planck-Institute for Evolutionary Anthropology</i></td>
 </tr>
 <tr> 
  <td>  Aida Andres (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); Kasper Munch (Aarhus University, Bioinformatics Research Centre (BiRC) ); Thomas  Mailund (Aarhus University, Bioinformatics Research Centre (BiRC) ); Asger Hobolth (Aarhus University, Bioinformatics Research Centre (BiRC) ); Mikkel Schierup (Aarhus University, Bioinformatics Research Centre (BiRC) ); The Bonobo Genome Consortium (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); Janet Kelso (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); Svante  Pääbo (Max Planck Institute for Evolutionary Anthropology, Evolutionary Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Balancing selection is a process by which advantageous diversity in populations is maintained. It is, for instance, known to play a vital role in maintaining variation in some immunity-related genes. Here, we describe a new method to identify regions with the genetic signatures of long-standing balancing selection acting in the chimpanzee and bonobo, and potentially the human, lineages. <br>Alleles evolving under long-standing balancing selection may trace their ancestry back to a common ancestor older than the common ancestor of bonobo, chimpanzee and human and can appear as regions of incomplete lineage sorting (i.e. the gene tree may differ from the species tree for this region). We therefore applied a coalescent hidden Markov model approach to the bonobo, chimpanzee and human genomes to identify regions of incomplete lineage sorting. To enrich for candidate regions with a high likelihood of being true targets of selection, we used Illumina resequencing data for 16 chimpanzee and three bonobo individuals. Within this data we investigated several other signatures that can derive from the action of balancing selection: high diversity in natural chimpanzee and bonobo populations, and a high fraction of shared SNPs between each of the three pairs of species. We identified several regions that constitute candidate targets of balancing selection.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='F22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster F22</strong></td>
 </tr> <tr> 
  <td><b>The Darwinian Tree Of Life In Light Of Horizontal Gene Transfer (Is Still Sound)</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Adam Sardar <i>University of Bristol</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Horizontal gene transfer (HGT) is the process whereby nucleic acids are transmitted from the genome of one species to another without inheritance by descent (i.e parent to child). Some people believe that there is no prokaryotic tree of life, but in fact a ‘thicket’ of shared genetic material. It is therefore extremely important to understand HGT, as it is so divergent from assumptions made in some phylogenetic analysis , such as a direct ancestry of homologous proteins. Previous studies have focused on trying to predict precisely which genes have been horizontally transferred and where, based on multiple sequence alignments and searches for a ’footprint’ or biochemical signal peculiar to the parent genome. This is very difficult and the signal is quickly lost with evolutionary time. We propose avoiding to attempt prediction of every horizontal transfer event, but to take a global view. We look for proteins comprised of sets of structural-domains (domain architectures, as identified in the SUPERFAMILY database) that show disparity in their deletion rates across the tree of life, requiring an overwhelmingly improbable realisation of events in order for their current distribution in observed genomes. This does not provide concrete identification of individual HGT events, but creates an enriched set of possible contenders, acting as an indicator towards HGT across all all organisms.<br><br>Our research suggest an upper bound of 30% for the percentage of domain architectures in bacteria involved in HGT, suggesting that bacteria do indeed posses an underlying Darwinian tree.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




