
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=E by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlist1685.html?cat=E by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:17 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'E'- Databases'</b></td>
 </tr>
</table>

<a name='E01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E01</strong></td>
 </tr> <tr> 
  <td><b>Seqcrawler  – A cloud ready indexing platform</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Olivier Sallou <i>IRISA</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Seqcrawler is an open source platform to index biological metadata, from various sources, to store raw data (dna, ...), and to browse/retrieve them from a web interface or in a programmatic way with REST API. It is fully scalable (components isolation) and cloud ready.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/83/19914.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='E02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E02</strong></td>
 </tr> <tr> 
  <td><b>Creating a Concept Map Interface to Visualize and Interpret a Comparative Effectiveness Data Model</b><br/><br/></td>
 </tr>
 <tr> 
 <td>David Fenstermacher <i>Moffitt Cancer Center</i></td>
 </tr>
 <tr> 
  <td>  Rodrigo Carvajal (Moffitt Cancer Center, Biomedical Informatics); Scott Morgan (Moffitt Cancer Center, Biomedical Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Comparative Effectiveness Research (CER) requires the design of new information technologies to utilize large observational patient-based data. Concept Maps are being developed to graphically organize and represent the CER data model. The result is an interactive interface revealing a comprehensive data dictionary and exposing the metadata of each data element.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/83/19916.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='E03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E03</strong></td>
 </tr> <tr> 
  <td><b>SolDB: A Database of Solanum lycopersicum and Solanum tuberosum Primers</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hassan Tariq <i>Government College University Faisalabad</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: SolDB is the Database of Solanaceae Family. It is an interactive, free online specialized database for Solanaceae family. Currently, it spans complete nucleotide sequences of expressed genes of Solanum lycopersicum and Solanum tuberosum along with their annotation. We have designed PCR oligonucleotide primer sequences for each gene, with their features and conditions given. This feature alone greatly facilitates researchers in PCR amplification of genes sequences, especially in cloning experiments. We also provided chloroplast genome section which gives access to fully sequenced plastid genomes and their annotations. Flexible database design, easy expandability, and easy retrieval of information are the main features of SolDB. The Database is publicly available at www.soldb.pakbiz.org.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/19961.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='E04'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E04</strong></td>
 </tr> <tr> 
  <td><b>PheMaDB: A solution for management, visualization, and analysis of OmniLog Phenotypic Microarray data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Wenling Chang <i>MITRE Corporation</i></td>
 </tr>
 <tr> 
  <td>  Brandon Higgs (MITRE Corporation, Biotechnology); Timothy Read (Emory University, Human Genetics); Keri Sarver (MITRE Corporation, Innovative Information Engineering and Biometrics); Nichole Nolan (FDA, FDA); Carol Chapman (Naval Medical Research Center, Genomics); Kimberly Bishop-Lilly (Naval Medical Research Center, Genomics); Shanmuga Sozhamannan (Naval Medical Research Center, Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Phenotype Microarray DataBase (PheMaDB) is a web-based relational database system that can be utilized to manage and to analyze data from OmniLog phenotypic microarrays (PM). PheMaDB has seven analytical modules which include outlier analysis, negative control analysis, phenotype barplot, correlation matrix, phenotype profile search, k-mean clustering, and heatmap analysis. These tools can provide users with a better understanding of cell growth patterns among different samples as they respond to various environmental conditions. The web-based GUI also allows easy data sharing between registered users. PheMaDB is open-sourced which allows users to freely modify or add new features according to their interests. Finally, demonstrations of the application capabilities are presented with experimental test data.  The source code can be downloaded at http://phemadb.sourceforge.net.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E05</strong></td>
 </tr> <tr> 
  <td><b>miRGator v2.0 and the construction of miRNA-disease network</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Wankyu Kim <i>Ewha Womans University</i></td>
 </tr>
 <tr> 
  <td>  Sooyoung Cho (Ewha Womans University, Ewha Research Center for Systems Biology); Yukyung Jun (Ewha Womans University, Ewha Research Center for Systems Biology); Minjeong Ko (Ewha Womans University, Ewha Research Center for Systems Biology); Sanghyuk Lee (Ewha Womans University, Ewha Research Center for Systems Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: miRGator is developed as an integrated database of microRNA-associated gene expression, target prediction, disease association and genomic annotation, in order to facilitate functional investigation of miRNAs (http://miRGator.kobic.re.kr). It contains (i) human miRNA expression profiles under various conditions, (ii) paired expression profiles of both mRNAs and miRNAs, (iii) gene expression profiles under miRNA-perturbation (e.g. miRNA knockout and overexpression), (iv) known/predicted miRNA targets and (v) miRNA-disease associations. In total, >8000 miRNA expression profiles, ?300 miRNA-perturbed gene expression profiles and ~2000 mRNA expression profiles are compiled with manual annotations on disease, tissue type and perturbation. Additionally, disease signature genes were extracted from ~12,000 gene expression profiles for ~100 human diseases. By integrating these data sets, a series of novel associations between human diseases and miRNAs is extracted by systematically comparing disease and target signature genes from various sources. Our approach correctly predicted known disease-miRNA associations with high accuracy as well as novel associations.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20103.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='E06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E06</strong></td>
 </tr> <tr> 
  <td><b>Comprehensive nsSNP database with predictions and annotations of impact</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Christian Schaefer <i>Technical University Munich</i></td>
 </tr>
 <tr> 
  <td>  Yana Bromberg (Rutgers University, Department of Biochemistry and Microbiology ); Burkhard Rost (Technical University Munich, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In the past few years, a plethora of databases emerged containing single nucleotide polymorphisms (SNPs); dbSNP may be the most popular one. Non-synonymous SNPs (nsSNPs) change the amino acid in the gene product and either could have an effect on protein structure and/or protein function or are neutral in that sense. Most nsSNPs, however, lack experimental annotations about their functional impact and most human nsSNPs also have no annotation about their disease-associated role. Here, we introduce a rich database based on dbSNP. While data from human are clearly the most prevalent, we also report nsSNPs from 13 other organisms. Apart from data taken from dbSNP, we also provide nsSNPs from other databases like SwissProt, OMIM and PMD. Amino acid annotations from SwissProt around observed nsSNPs augment the available information. The impact on function of each nsSNP is predicted using SIFT and SNAP. The modular design of database and management scripts allows for easy integration of further databases and predictors, a web front end offers convenient retrieval of information.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E07</strong></td>
 </tr> <tr> 
  <td><b>bEXOpedia - a comprehensive proteome database for bacterial exosomes</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dae-Kyum Kim <i>Pohang University of Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  Byeongsoo Kang (Pohang University of Science and Technology, School of Interdisciplinary Bioscience and Bioengineering); Dong-Sic  Choi (Pohang University of Science and Technology, Department of Life Science and Division of Molecular and Life Sciences); Jaewook Lee (Pohang University of Science and Technology, Department of Life Science and Division of Molecular and Life Sciences); Jun-Pyo  Choi (Pohang University of Science and Technology, Department of Life Science and Division of Molecular and Life Sciences); Seng Jin  Choi (Pohang University of Science and Technology, Department of Life Science and Division of Molecular and Life Sciences); Yong Song  Gho (Pohang University of Science and Technology, Department of Life Science and Division of Molecular and Life Sciences); Daehee Hwang (Pohang University of Science and Technology, School of Interdisciplinary Bioscience and Bioengineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The secretion of exosomes is a universal cellular process occurring from simple organisms to complex multicellular organisms, including bacteria. Throughout evolution, both prokaryotic and eukaryotic cells have adapted to manipulate exosomes for intercellular communication via outer membrane vesicles (OMVs), membrane vesicles, and microvesicles or exosomes in Gram-negative bacteria, Gram-positive bacteria, and eukaryotic cells, respectively. A number of studies have shown that bacterial exosomes are involved in virulence-related processes including biofilm formation and host-pathogen interaction. Many proteomic studies have been performed to investigate exosomal proteins and their functions. However, comparative analysis of these proteomic data from various bacterial strains has been hampered because it requires manual data acquisition and parsing and application of bioinformatic tools. Here, we present bEXOpedia, a comprehensive database of exosomal proteins from 24 proteomic studies for 17 bacterial strains. bEXOpedia provides an array of functionalities for comparative analysis of bacterial exosomal proteome: (i) search for a set of proteins from 24 proteome datasets, (ii) several bioinformatic tools including set analysis for comparison of multiple sets of proteins, GO analysis, and network analysis for understanding functions of the sets of proteins, (iii) gene expression explorer for integration of mRNA data with proteomic data, and (iv) experimental details including TEM images of exosomes, SDS-PAGE of exosomal fractions, and procedures for mass spectrometry-based proteomics in the individual studies. Comprehensive proteomic data in bEXOpedia together with an array of bioinformatic tools can serve as useful resources to improve functional roles of bacterial exosomes in various contexts.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E08</strong></td>
 </tr> <tr> 
  <td><b>GWAS Central: an advanced database for the integration and comparative interrogation of genome-wide association study datasets</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sirisha Gollapudi <i>University of Leicester</i></td>
 </tr>
 <tr> 
  <td>  Robert Free (University of Leicester, Department of Genetics); Robert  Hastings (University of Leicester, Department of Genetics); Owen Lancaster (University of Leicester, Department of Genetics); Tim Beck (University of Leicester, Department of Genetics); Gudmundur Thorisson (University of Leicester, Department of Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Comprehensive genome-wide association study (GWAS) data-sets are rarely published in journals or databases, and in the case of negative findings often not reported at all. Consequently, comparing different studies is difficult, and it is impossible to examine a full and unbiased picture of all the data that exist. To address this deficit, GWAS Central (www.gwascentral.org) (previously HGVbaseG2P) was constructed – representing a free and open access resource for the interrogation of summary-level GWAS data, ultimately combining the features of a database and a scientific journal.<br>GWAS Central employs powerful graphical and text based data presentation methods for discovery, visualisation and co-examination of many studies, at genome-wide and region-specific levels. Studies of interest can be identified using chromosomal regions/genes and markers. There is also the facility for researchers to securely view their own uploaded datasets alongside many published studies.<br>Current content includes top p-values from collections; supplementary data; direct researcher submissions; and publicly available data. Consequently, the database now hosts >21 million p-values and 708 studies (vs 3,948 p-values and 798 studies in the NHGRI GWAS catalog), representing an estimated ~5% of all such data yet produced. <br>A version of GWAS Central will soon be released for research groups to install on their own servers allowing them to manage, analyse and control access to their own data. These installations will be interoperable and searchable as a federated network.<br>This research received funding from the European Community's Seventh Framework Programme (FP7/2007-2013) under grant agreement number 200754, GEN2PHEN.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E09</strong></td>
 </tr> <tr> 
  <td><b>ISA tools: standard-compliant reporting and curation at the community level</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Eamonn Maguire <i>University of Oxford</i><br></td>
 </tr>
 <tr> 
  <td>Susanna-Assunta Sansone (University of Oxford) Philippe  Rocca-Serra (University of Oxford, Oxford e-Research Centre); Eamonn  Maguire (University of Oxford, Oxford e-Research Centre); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Shared, annotated research data afford significant discovery opportunities and can prevent unnecessary repetition of work. Preserving and sharing research data, however, produce little value if it is not sufficiently well annotated to be comprehensible and (in principle) reproducible. A growing number of funders and journals also encourage the enrichment and standardization of experimental details (Field, et al. 2009 ‘Omics Data Sharing, Science, 9, 234-236;). <br><br>ISA tools (http://www.isa-tools.org) are the first general-purpose format and open source desktop software suite that contribute to this grand vision by (i) regularizing the local management of experimental details and curation at the source, (ii) supporting graduated adherence to community-defined minimal information reporting standards and ontologies (http://biosharing.org), and (iii) facilitating submission to a growing number of international public repositories. The novelty is in the modular nature of the ISA software suite that separates conformance to standards from the reporting of experimental details, shielding researchers from unnecessary complexity.<br><br>With an active pool of geographically-distributed community of developers, the ISA tools address real case studies of service providers that, in turn, assist life science research communities. Currently, the ISA tools power several public and internal systems managing studies as diverse as environmental health, stem cell genomics, toxicogenomics, environmental genomics, plant metabolomics and metagenomics (e.g. http://discovery.hsci.harvard.edu/, http://sidr-dr.inist.fr).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E10</strong></td>
 </tr> <tr> 
  <td><b>DARNED (Database of RNA editing).</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Pavel Baranov <i>University Cllege Cork</i></td>
 </tr>
 <tr> 
  <td>  Anmol Kiran (University College Cork, Biochemistry); John O'Mahony (University College Cork, Biochemistry); Komal Sanjeev (Banaras Hindu University, Institute of technology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The identity of particular nucleotides in RNA can be enzymatically changed in the process termed RNA editing. The stochastic nature of this process results in considerable diversification of the transcriptome. Regulation of this process allows generation of transcriptomes that could be specific for particular tissues, development stages, individual cells or their compartments.<br><br>Recently, due to developments in the sequencing technologies, we have witnessed explosion of information on RNA editing in humans.  The rapid growth of information on RNA editing has demanded generation of a centralized depository for these data. <br><br>The database of RNA editing (DARNED) has been launched in 2010 and is available at http://darned.ucc.ie. Initially DARNED provided access to the data on RNA editing mapped to the reference human genome. The current version of DARNED also provides information on RNA editing in fruit flies. In current version the data can be accessed in three different ways. 1. By sequence similarity searching. In such a case a user can verify whether a user-provided sequence may have known sites of RNA editing. 2. By sequence entity (gene, RNA, EST) reference number to explore associated RNA editing information. 3. By specification of reference genome coordinates when a goal is to investigate RNA editing corresponding to a particular genomic location.<br><br>The data can be explored either within DARNED itself or in the environment of popular genome browsers (UCSC & Ensembl). RNA editing cases that result in recoding of protein sequences are annotated in a community based manner. Corresponding annotations are available through Wikipedia.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E11</strong></td>
 </tr> <tr> 
  <td><b>Graph-based queries of Semantic-Web integrated biological data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marco Moretto <i>Edmund Mach Foundation</i></td>
 </tr>
 <tr> 
  <td>  Alessandro Cestaro (Edmund Mach Foundation, Computational Biology); Riccardo Velasco (Edmund Mach Foundation, Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: In the post-genomic era, life science researchers are faced with the need to manage and inspect a growing abundance of data and information. Data from different sources, both public and proprietary, have the most value when considered in the context of each other as they give more information. In order to answer questions that spans multiple fields in the biology domain without an integrated approach, a biologist needs to visit all data sources related to the problem and ?nd relevant data. In the last years we have become witnesses of a growing interest for the Semantic Web technologies to integrate and query biological data. Semantic Web technologies were designed to meet the challenges of reduce the complexity of combining data from multiple sources, resolve the lack of widely accepted standards and manage highly distributed and mutable resources. However, Semantic Web standard technologies do not provide any tools to query integrated knowledge bases from a graph perspective, that is defining graph traversal patterns. For example, it is not possible to ask a query like "are enzyme A and compound B related?" without knowing the complete structure of the knowledge base. After exploring different alternatives we come up with the use of a graph traversal programming language on top of a triplestore in order to perform several path traversal queries on an integrated graph. We tested the feasibility of the approach integrating Uniprot, Gene Ontology, Chebi and Kegg resources posing queries of different complexity.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E12</strong></td>
 </tr> <tr> 
  <td><b>TMPad: an integrated structural database for helix-packing folds in transmembrane proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Cheng-Wei Cheng <i>Academia Sinica</i></td>
 </tr>
 <tr> 
  <td>  Allan Lo (British Columbia Cancer Agency, Michael Smith Genome Sciences Centre); Yi-Yuan Chiu (National Chiao Tung University, Institute of Bioinformatics and Systems Biology); Ting-Yi Sung (Academia Sinica, Institute of Information Science); Wen-Lian Hsu (Academia Sinica, Institute of Information Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Alpha-Helical transmembrane (TM) proteins play an important role in many critical and diverse biological processes, and specific associations between TM helices are important determinants for membrane protein folding, dynamics and function. In order to gain insights into the above phenomena, it is necessary to investigate different types of helix-packing modes and interactions. However, such information is difficult to obtain because of the experimental impediment and a lack of a well-annotated source of helix-packing folds in TM proteins. We have developed the TMPad (TransMembrane Protein Helix- Packing Database) which addresses the above issues by integrating experimentally observed helix–helix interactions and related structural information of membrane proteins. Specifically, the TMPad offers pre-calculated geometric descriptors at the helix-packing interface including residue backbone/side-chain contacts, interhelical distances and crossing angles, helical translational shifts and rotational angles. The TMPad also includes the corresponding sequence, topology, lipid accessibility, ligand-binding information and supports structural classification, schematic diagrams and visualization of the above structural features of TM helix-packing. Through detailed annotations and visualizations of helix-packing, this online resource can serve as an information gateway for deciphering the relationship between helix–helix interactions and higher levels of organization in TM protein structure and function. The website of the TMPad is freely accessible to the public at http://bio-cluster.iis.sinica.edu.tw/TMPad.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E13</strong></td>
 </tr> <tr> 
  <td><b>EBI Search: a new way to explore biology</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrew Jenkinson <i>EMBL-EBI</i></td>
 </tr>
 <tr> 
  <td>  Mickael  Goujon (EMBL-EBI, External Services); Jennifer Cham (EMBL-EBI, User Experience Analyst); Rodrigo Lopez (EMBL-EBI, External Services); Ewan Birney (EMBL-EBI, PANDA); Graham Cameron (EMBL-EBI, Associate Director); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Life science researchers in all disciplines are increasingly incorporating computational approaches in their work, often accessing the vast (and freely available) resources at the European Bioinformatics Institute (EMBL-EBI). Because accessibility is our top priority, we carried out large-scale user experience analysis with a view to improving the searchability of our resources. The result is a unique, intuitive search service that makes it simpler for users to explore the data. The service, which indexes and updates more than 300 million entries a day, uses Apache Lucene as the internal search engine and REST web services to retrieve the data. ‘Biologically Aware’ search methods are based on an algorithm that identifies key relationships between biological concepts, namely between genes, gene expression profiles, proteins and 3D protein structures.  The service is available at www.ebi.ac.uk</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E14</strong></td>
 </tr> <tr> 
  <td><b>Creating a curated resource for translational profiling data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ruth Spriggs <i>MRC Toxicology Unit</i></td>
 </tr>
 <tr> 
  <td>  Anne Willis (MRC Toxicology Unit, MRC Toxicology Unit); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Polysome profiling experiments, used to investigate the post-transcriptional regulation of gene expression, have been performed within our group to look at different types of cell stress. Other groups are also generating polysome profiling data, and there is a need to systematically compare the gene lists produced by these different experiments and different groups. The aim of an analysis would be to identify how, and which, specific subsets of mRNAs are translationally regulated in unison.<br>In order to enable this type of meta analysis, and to make all these data available to the translational community, we are creating a resource to collate polysome profiling data. Initially, data sets obtained within our group will be incorporated into the resource and analysed to identify sequence motifs common to groups of coordinately regulated genes. The database, and data within, will be made freely available on the internet. Data from external groups will be incorporated and included in the analysis as they become available to us.<br>The relational database underpinning the resource has been designed and implemented, the development of a semi-automated process of data input is complete, and an initial search interface has been created. Data sets from within the group have been input and are being analysed; the results of this analysis, together with details of the underlying database, will be described.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E15</strong></td>
 </tr> <tr> 
  <td><b>CyMoBase and diArk - Resources for cytoskeletal and motor protein sequence information and eukaryotic genome research</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Björn Hammesfahr <i>Max-Planck-Institut für biophysikalische Chemie</i></td>
 </tr>
 <tr> 
  <td>  Martin Kollmar (Max-Planck-Institut für biophysikalische Chemie, NMR-basierte Strukturbiologie); Marcel Hellkamp (Max-Planck-Institut für biophysikalische Chemie, NMR-basierte Strukturbiologie); Florian Odronitz (Max-Planck-Institut für biophysikalische Chemie, NMR-basierte Strukturbiologie); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motor proteins are involved in processes like cellular transport, muscle contraction, and cell division. Three motors are myosin, dynein, and kinesin. They convert chemical energy (ATP) into mechanical work (movement). Protein sequences are the bases for many biochemical and cell biological experiments, as well as bioinformatical analyses.<br>	We implemented two web applications, called CyMoBase and diArk. Both are working with the same database that contains 48 proteins, 24000 sequences with a total sequence length of 23 million amino acids, 174 domains, 922 species, 635 publications, 1911 projects, 101 sequencing centres, and more than 1 Terra-Byte of genome files. Using CyMoBase (www.cymobase.org), manually annotated sequence data of motor proteins and corresponding information and analyses can be searched. diArk (www.diark.org) is designed to reach the data and information connected to the sequencing projects of eukaryotic species and the genome files, in addition providing many statistical analyses. Both applications have different search modules that can be combined to reach the data of interest. Furthermore, they provide different result views to display the results of the users search.<br>	The size of the database, the kind of annotation, the possibility to use and combine different search modules, and the amount of information, options, and analyses offered by the web interface make CyMoBase and diArk the reference applications for cytoskeletal and motor proteins, and for eukaryotic sequencing data, respectively.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E16</strong></td>
 </tr> <tr> 
  <td><b>A Time Machine for miRBase</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Raoul Bonnal <i>Istituto Nazionale Genetica Molecolare</i></td>
 </tr>
 <tr> 
  <td>  Riccardo Rossi (Istituto Nazionale Genetica Molecolare, Integrative Biology Program); Valeria Ranzani (Istituto Nazionale Genetica Molecolare, Integrative Biology Program); Massimiliano Pagani (Istituto Nazionale Genetica Molecolare, Integrative Biology Program); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: miRNAs are key players in regulation of many cellular mechanisms. Data associated with miRNA and their expression have grown tumultuously in the last years. miRBase is the reference database for miRNA annotation in over 140 species and it is updated regularly. A new miRBase version adds novel miRNAs and overhaul the annotation of those already present, including also sequence and name variations. All these updates are recorded as well, in a static archives (i.e. log-table) but from the main site is not possible to explore directly the history of a specific miRNA. The platforms exploited for high-throughput miRNA analysis use miRNA annotations that are not often aligned with the most recent miRBase's release and could induce mistakes or wrong associations between miRNA names and their sequences. Here we present a web application which allows users to browse trough different releases of miRBase searching for a set of miRNAs and/or by species as well. Furthermore we include the annotation of the most popular platforms used for HT miRNA analyses (i.e. Applied Biosystems, Exiqon, Illumina). This web-application is extremely useful for cross comparison between data generated from different platforms/miRBase-releases. Currently we support miRBase v.10 up to v.16, which are the versions most platforms refer to. This application is distributed with an embedded SQLite database, which includes all the miRNAs data. The code was written using BioRuby and Ruby on Rails, two powerful libraries to work with biological data and complex databases.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E17</strong></td>
 </tr> <tr> 
  <td><b>A web-based tool for functional investigation of microRNAs : miRGator v2.0</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Yukyung Jun <i>Ewha Womans University</i><br></td>
 </tr>
 <tr> 
  <td>Wankyu Kim (Ewha Womans University) SooYoung Cho (Ewha Womans University, ERCSB); Sanghyun Lee (Ewha Womans University, ERCSB); Hyung-Seok Choi (Ewha Womans University and LG Electronics, ERCSB and Bio & Health Group); Sungchul Jung (Ewha Womans University, ERCSB); Youngjun Jang (Korean Bioinformation Center, KOBIC); Charny Park (Ewha Womans University, ERCSB); Sangok Kim (Ewha Womans University, ERCSB); Sanghyuk Lee (Ewha Womans University and Korean Bioinformation Center, ERCSB and KOBIC); Wankyu Kim (Ewha Womans University, ERCSB); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: miRGator is an integrated database of microRNA (miRNA)-associated gene expression, target prediction, disease association and genomic annotation, which aims to facilitate functional investigation of miRNAs. The recent version of miRGator v2.0 contains information about (i) human miRNA expression profiles under various experimental conditions, (ii) paired expression profiles of both mRNAs and miRNAs, (iii) gene expression profiles under miRNA-perturbation (e.g. miRNA knockout and overexpression), (iv) known/predicted miRNA targets and (v) miRNA-disease associations. In total, >8000 miRNA expression profiles, ?300 miRNA-perturbed gene expression profiles and ?2000 mRNA expression profiles are compiled with manually curated annotations on disease, tissue type and perturbation. By integrating these data sets, a series of novel associations (miRNA–miRNA, miRNA–disease and miRNA–target) is extracted via shared features. For example, differentially expressed genes (DEGs) after miRNA knockout were systematically compared against miRNA targets. Likewise, differentially expressed miRNAs (DEmiRs) were compared with disease-associated miRNAs. Additionally, miRNA expression and disease-phenotype profiles revealed miRNA pairs whose expression was regulated in parallel in various experimental and disease conditions. Complex associations are readily accessible using an interactive network visualization interface. The miRGator v2.0 serves as a reference database to investigate miRNA expression and function (http://miRGator.kobic.re.kr).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E18</strong></td>
 </tr> <tr> 
  <td><b>The Leukemia Gene Atlas - A public platform to support leukemia research</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Sören Gröttrup <i>University of Münster</i></td>
 </tr>
 <tr> 
  <td>  Katja Hebestreit (University of Münster, Institute of Medical Informatics); Hans-Ulrich Klein (University of Münster, Institute of Medical Informatics); Christian Ruckert (University of Münster, Institute of Medical Informatics); Christoph Bartenhagen (University of Münster, Institute of Medical Informatics); Martin Dugas (University of Münster, Institute of Medical Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A major challenge for researchers is to keep track of the fast increasing number of analyzed and published molecular data sets regarding leukemia. One reason for this is that common cancer repositories like GEO or ArrayExpress do not provide leukemia specific molecular and biological annotations and hence are not adequate for research on this topic. The Leukemia Gene Atlas (LGA) is aimed to close this gap.<br><br>The centerpiece of the LGA is a PostgreSql database containing molecular and clinical data as well as results, e.g. top tables of significant genes, from published studies. This database is able to store many different data types, like gene expression, methylation, next-generation sequencing or copy-number. These data and their samples are leukemia specific annotated and structured. The LGA database can be accessed via a web front end, which is implemented using the open source project Google Web Toolkit and the Smart GWT library. Besides wide search functions this web front end provides various analysis functions and visualization tools for the stored data, including extended barplots, in which all phenotypes for each selected sample are displayed, simplifying the identification of clusters in the data. These analysis functions are implemented in R/Bioconductor. Another useful feature of the LGA is the possibility to search for studies and groups of samples whose gene expression for certain genes of interest differ significantly.<br><br>Currently the LGA comprises about 5000 samples and 290 result tables from 10 studies, available online at www.leukemia-gene-atlas.org</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E19</strong></td>
 </tr> <tr> 
  <td><b>Eukaryotic Linear Motif Resource - ELM</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Holger Dinkel <i>European Molecular Biology Laboratory</i></td>
 </tr>
 <tr> 
  <td>  Francesca Diella (European Molecular Biology Laborator, Structural and Computational Biology); Toby Gibson (European Molecular Biology Laborator, Structural and Computational Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Linear motifs are short stretches of protein sequence that provide regulatory functions without structural context. Many intracellular signalling processes depend heavily on interactions mediated by short linear motifs. The ELM resource at http://elm.eu.org/ provides an expanding knowledge base, currently covering  more than 150 known motifs classes, with extensive annotation that includes >1500 experimentally reported instances. ELM is also a server for detecting new candidates of known linear motifs in proteins of interest. It employs a variety of different filters that take into account protein structure, native disorder, domain architecture as well as cellular and taxonomic contexts in order to reduce or deprecate false positive matches. Results are displayed graphically as well as in tabular format with extensive links to a number of remote resources. Using these links, researchers can explore the motifs, proteins, complex structures and associated literature to evaluate whether candidate motifs might be worth experimental investigation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E20</strong></td>
 </tr> <tr> 
  <td><b>Sierra: A simple LIMS system for small sequencing facilities</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Simon Andrews <i>The Babraham Institute</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The decreasing cost of high throughput sequencing systems has allowed many groups to consider setting up a small scale sequencing facility.  Whilst these systems will provide a software pipeline to automate the collection and primary analysis of sequencing runs they may not provide the tools required to track the samples being run or distribute results back to the sample owners.  Manual curation of samples and copying of data can be an onerous task in a small facility.<br>Sierra provides a simple web-based system which will provide sample tracking through a sequencing pipeline.  It uses a small database for sample metadata and can extract results from a simple collection of sequencer run folders.  It is able to provide a controlled way to let users see the progress of their samples through a sequencing pipeline, share their samples with others,  and to collect sequences, mapping results and QC reports for their samples. <br>Sierra is initially configured for connection to an Illumina sequencing pipeline, but could easily be extended to other platforms.<br>Sierra is free software released under the GPLv3. <br>You can find more information about Sierra at www.bioinformatics.bbsrc.ac.uk/projects/sierra/</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E21</strong></td>
 </tr> <tr> 
  <td><b>Soybean Knowledge Base (SoyKB) : A Web Resource for Soybean Translational Genomics</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Trupti Joshi <i>University of Missouri-Columbia</i></td>
 </tr>
 <tr> 
  <td>  Kapil Patil (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); Levi D. Franklin  (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); Michael  Fitzpatrick  (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); Qiuming  Yao  (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); Zheng  Wang (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); Marc   Libault (Division of Plant Sciences, University of Missouri, Columbia, MO 65211-2060); Laurent   Brechenmacher (Division of Plant Sciences, University of Missouri, Columbia, MO 65211-2060); Babu  Valliyodan  (Division of Plant Sciences, University of Missouri, Columbia, MO 65211-2060); Xiaolei  Wu  (Division of Plant Sciences, University of Missouri, Columbia, MO 65211-2060); Jianlin  Cheng  (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); Gary  Stacey  (Division of Plant Sciences, University of Missouri, Columbia, MO 65211-2060); Henry Nguyen  (Division of Plant Sciences, University of Missouri, Columbia, MO 65211-2060); Dong Xu (Department of Computer Science , University of Missouri, Columbia, MO 65211-2060); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Soybean Knowledge Base (SoyKB) is a comprehensive all-inclusive web resource for translational genomics in soybean. SoyKB is designed to handle the storage and integration of the genomics, microarray, transcriptomics, proteomics and metabolomics data along with biological pathway information. It includes a mySQL database at the back end that incorporates and integrates all the soybean omics data from various experiments. It is designed to contain information on 4 different entities namely genes, miRNAs, metabolites and SNPs. SoyKB has a user-friendly web interface together with genome browser and pathway viewer. <br><br>SoyKB has four tiers of registration options which control the different levels of access to the public and private data. It allows users of certain levels to share their expertise by adding comments to the data. It provides an informatics-based social network system to build connections among soybean researchers, producers and consumers. SoyKB is also linked to other soybean databases such as Soybase to allow easy and seamless navigation between the two. <br><br>SoyKB has many useful tools including protein 3D structure viewer, Affymetrix probe id search, gene family search, multiple gene/metabolite search, as well as download and upload capacity for experimental data and genomic information. Many other new tools are also currently under development including breeder toolbox for QTL and traits information, comparison against G. soja, coexpression analysis for multiple gene/metabolite searches and motif prediction.<br><br>SoyKB can be accessed at http://soykb.org/</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E22</strong></td>
 </tr> <tr> 
  <td><b>HumanVariome: A High Quality Human Variation study and Resource for Rare Variant Detection and Validation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andreas Kremer <i>Erasmus Medical Center Rotterdam</i></td>
 </tr>
 <tr> 
  <td>  Andrew Stubbs (Erasmus MC, Bioinformatics); Bas Horsman (Erasmus MC, Bioinformatics); Joke  Reumers (Vlaams Institute for Biotechnology, Vesalius Research Center); Stephan Nouwens (Erasmus MC, Bioinformatics); Jules P.P.  Meijerink (Erasmus MC, Pediatric Oncology/Hematology); Ivo Palli (Erasmus MC, Bioinformatics); Daphne Heijsman (Erasmus MC, Bioinformatics); Sigrid M.A. Swagemakers (Erasmus MC, Bioinformatics); Anton Koning (Erasmus MC, Bioinformatics); Diether  Lambrechts (Vlaams Institute for Biotechnology, Vesalius Research Center); Peter J. van der Spek           (Erasmus MC, Bioinformatics); Elizabeth McClellan (Erasmus MC, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Next-generation DNA sequencing has recently empowered scientists to identify genetic variations associated with human disease at higher resolution and greater sensitivity than previously possible. <br>Our HumanVariome project goes beyond other projects in both sequencing depth of the sequencing and size of the cohort analysed at high depth. We have utilized the latest Oracle 11i technology, which is centered on a database model designed to store genomic variations and annotations detected using the Complete Genomics (CG) sequencing3 platform. Currently, the database contains almost 100 genomes from Erasmus MC and V.I.B of which a subset, 41 genomes (“Unaffected” cohort), have been used initially.  The application has been developed to be file format independent, to allow for enriched variation reporting and includes additional 60 genomes, made publicly by CG. <br>The total content of the HumanVariome project comprises 248 Gb, 202 Gb and 246 Gb of mapped reads for “Affected”, “Unaffected” and “Normal Tissues from Cancer Patients” cohorts, respectively, representing an average of > 80x coverage for the three groups. In total, nearly 1 million novel SNVs have been identified for the “Unaffected” cohort (70% have complete calling of both alleles).<br>In summary, we present a resource for prioritizing rare SNVs identified with NGS technology. In contrast with other projects, only high coverage genome sequences are used and no imputation has been used to infer unsequenced variants. In addition, our resource has successfully been used to prioritize candidate cancer targets and genomic variations detected in familial congenital malformations.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E23</strong></td>
 </tr> <tr> 
  <td><b>easyDAS: Automatic creation of DAS servers</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Bernat Gel Moreno <i>Institut de Medicina Predictiva i Personalitzada del Càncer</i></td>
 </tr>
 <tr> 
  <td>  Andrew Jenkinson (EBI, Proteomics Services); Rafael Jimenez (EBI, Proteomics Services); Xavier Messeguer Peypoch (UPC, LSI); Henning Hermjakob (EBI, Proteomics Services); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Distributed Annotation System (DAS) has proven to be a successful way to publish and share biological data. Although more than 1000 servers from around 50 organizations are currently registered, setting up a DAS server comprises a fair amount of work, making it difficult for many research groups to share their biological annotations. Given the clear advantage that the generalized sharing of relevant biological data is for the research community it would be desirable to facilitate the sharing process.<br>easyDAS is a web-based system enabling anyone to publish biological annotations with just some clicks. With a wizard-like simple interface, the system is capable of reading different standard data file formats (including GFF and CSV), process the data and create a new publicly available DAS source in a completely automated way. easyDAS is available at available at http://www.ebi.ac.uk/panda-srv/easydas. The created sources are hosted on the EBI systems and can take advantage of its high storage capacity and network connection, freeing the data provider from any network management work. easyDAS is an open source project under the GNU LGPL license.<br>easyDAS can help many researchers in sharing their biological data, potentially increasing the amount of relevant biological data available to the scientific community.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E24'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E24</strong></td>
 </tr> <tr> 
  <td><b>Wheat international sequence repository integrated with genomic and genetic data in GnpIS</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Michael Alaux <i>INRA</i></td>
 </tr>
 <tr> 
  <td>  Véronique Jamilloux (INRA, URGI); Françoise Alfama (INRA, URGI); Daphné Verdelet (INRA, URGI); Aminah-Olivia Keliet (INRA, URGI); Sébastien Reboux (INRA, URGI); Nacer Mohellibi (INRA, URGI); Sophie Durand (INRA, URGI); Erik Kimmel (INRA, URGI); Isabelle Luyten (INRA, URGI); Delphine Steinbach (INRA, URGI); Hadi Quesneville (INRA, URGI); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: URGI is a research unit in genomics and bioinformatics at INRA (French National institute for Agricultural Research), dedicated to plants and crop parasites. We develop and maintain a genomic and genetic information system called GnpIS, especially for wheat.<br>In the frame of the IWGSC (International Wheat Genome Consortium) sequence survey initiative, we set up a sequence repository, which allow to download and blast sequences: http://urgi.versailles.inra.fr/index.php/urgi/Species/Wheat/Sequence-Repository. <br>This repository is also integrated in GnpIS to have access to wheat genomic and genetic data such as physical and genetic mapping, genomic annotations, polymorphism and genetic resources data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E25'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E25</strong></td>
 </tr> <tr> 
  <td><b>gsGator : an integrated platform for gene set analysis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hyunjung Kang <i>Ewha Womans University</i></td>
 </tr>
 <tr> 
  <td>  Youngjun Jang (Korea Research Institute of Bioscience and Biotechnology (KRIBB), Korean Bioinformation Center (KOBIC)); Sooyoung Cho (Ewha Womans University, Ewha Research Center for Systems Biology); Sanghyuk  Lee (Korea Research Institute of Bioscience and Biotechnology (KRIBB), Ewha Womans University, Korean Bioinformation Center (KOBIC), Ewha Research Center for Systems Biology,Division of Life and Pharmaceutical Sciences); Wan Kyu Kim (Korea Research Institute of Bioscience and Biotechnology (KRIBB), Ewha Womans University, Korean Bioinformation Center (KOBIC), Ewha Research Center for Systems Biology,Division of Life and Pharmaceutical Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: High-throughput experiments e.g. microarray and mass spectrometry typically generate gene lists. Gene set analysis (GSA) is a powerful method to deduce biological meaning for an a priori defined set of genes. Reflecting the utility of GSA, numerous tools have been developed to test statistical enrichment or depletion of e.g. pathways or gene ontology (GO) terms.<br>gsGator is a web-based platform for GSA with many novel features, where extensive data sets are integrated related to molecular network, pathway, miRNA target, disease, drug and phenotype. gsGator allows simultaneous analysis of multiple gene sets and the user-defined gene sets are conveniently stored on our server for further analysis. These gene sets can be made publicly available so that the pre-defined gene sets can grow by community effort in a similar way to Wikipedia. Particularly, gsGator supports cross-species GSA, where orthology is automatically mapped among different organisms including human, mouse, fly, worm and yeast. Cross-species GSA may lead to the discovery of conserved gene modules between different species and greatly extends the scope of GSA. gsGator provides virtually fully-automated analysis for any gene sets with the modules for management, retrieval and statistical analysis being tightly integrated. To facilitate the exploration of GSA results, the relationships among gene sets are presented in various ways such as table, heatmap and a dedicated network viewer.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E26'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E26</strong></td>
 </tr> <tr> 
  <td><b>Virtual Fly Brain 3D Interaction Tool</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nestor Milyaev <i>University of Edinburgh</i></td>
 </tr>
 <tr> 
  <td>  David Osumi-Sutherland (University of Cambridge, Department of Genetics); Simon Reeve (University of Cambridge, Department of Genetics); Michael Ashburner (University of Cambridge, Department of Genetics); Douglas Armstrong (University of Edinburgh, School of Informatics); Nicolas  Burton (MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine); Zsolt Husz (MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine); Richard Baldock (MRC Human Genetics Unit, Institute of Genetics and Molecular Medicine); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Complex 3D datasets of biological information, for example gene expression, are most commonly handled as series of sections. These data are expensive to generate and thus their sharing and distribution is important. The ideal distribution and sharing is via web interfaces but data bandwidth is a major challenge especially considering recent imaging technologies are now routinely generating stacks that are several gigabytes and increasing. To make such image data useful, mechanisms for linking it with underlying organism ontology and experimental databases are also needed.<br><br>We have been developing a generic web application for browsing images (such as confocal microscopy stacks) integrated with querying of organism phenotype databases.<br><br>The client-facing part is a configurable AJAX-based web interface that runs in any modern web browser. It allows the user to specify what section of a 3D stack to display, what biological domains to include in the view (as coloured overlays), and to execute queries to underlying organism anatomy and gene expression databases. Since all elements are seamlessly integrated the user can easily navigate between the imaging and data layers and to execute the queries in a “point-and-click” fashion.<br><br>The multi-purpose server processes user queries and provides the delivery of a series of web optimised image tiles as specified by user request. The image tiles produced from a compound 3D object representing both the overall structure and individual domains.<br><br>We will demonstrate this tool using FlyBase datasets. All the software developed is open-source. The application is available online at http://www.virtualflybrain.org/</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E27'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E27</strong></td>
 </tr> <tr> 
  <td><b>An Interactive Map Viewer and its Application for Visualization of Cattle Candidate Loci for Milk Production and Mastitis</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Crtomir Gorup <i>University of Ljubljana</i></td>
 </tr>
 <tr> 
  <td>  Tanja Kunej (University of Ljubljana, Department of animal science); Jernej Ogorevc (University of Ljubljana, Department of animal science); Tomaz Curk (University of Ljubljana, Laboratory for bioinformatics); Blaz Zupan (University of Ljubljana, Laboratory for bioinformatics); Peter Dovc (University of Ljubljana, Department of animal science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Lactation and udder health are two important research areas in animal genetics. However, due to the absence of classic research tools (e.g., targeted knock-outs, large population studies) candidate gene detection in farm animals is possible only by combining different sources of evidence. <br><br>We have developed an interactive web-based genome browser called DairyVis (http://dairyvis.fri.uni-lj.si). Its main feature is a map-driven integration of biological data (QTL, genes, microRNAs, etc.) associated with milk production and mastitis. Users can easily locate overlapping genome regions and identify candidate genes related to milk production. DairyVis uses a local database which is manually curated by the authors and automatically updated by quering freely available biological databases. DairyVis integrates the data on different species that originate from AnimalQTLdb, miRbase and our manual literature curation. Automatic aggregation of the latest available biological databases frees the users from looking at the individual sources and allows them to spend more time discovering new and annotated QTLs.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E28'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E28</strong></td>
 </tr> <tr> 
  <td><b>GenomeRNAi: A Phenotype Database for Large-scale RNAi Screens</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Esther Schmidt <i>German Cancer Research Center</i></td>
 </tr>
 <tr> 
  <td>  Thomas Horn (German Cancer Research Center, Div. Signaling and Functional Genomics); Oliver Pelz (German Cancer Research Center, Div. Signaling and Functional Genomics); Svetlana Mollova (German Cancer Research Center, Div. Signaling and Functional Genomics); Arunraj Dhamodaran (German Cancer Research Center, Div. Signaling and Functional Genomics); Klaus Yserentant (German Cancer Research Center, Div. Signaling and Functional Genomics); Grainne Kerr (German Cancer Research Center, Div. Signaling and Functional Genomics); Michael Boutros (German Cancer Research Center, Div. Signaling and Functional Genomics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: RNA interference (RNAi) is a powerful method to generate loss-of-function phenotypes. It enables systematic genetic screens for every annotated gene in the genome. Genome-wide RNAi screens have been performed in human, mouse, drosophila and C.elegans and many RNAi libraries have become available to study phenotypes on a large scale.<br>With more and more datasets of RNAi-induced phenotypes becoming available, the systematic integration of functional information remains an important task. RNAi screens are performed using different types of assays from visible phenotypes to focused transcriptional readouts and provide a rich data source for functional annotation. Large-scale in vivo RNAi screens are now emerging, representing a new challenge in data representation and annotation, due to the complex nature of in vivo assays and phenotypes.<br>Currently, the GenomeRNAi database provides access to cell-based phenotypes in drosophila and human cells. It holds 97600 entries for human and 99700 for drosophila phenotypes extracted from 136 published RNAi screens. It also contains sequence and efficiency data for 300000 and 118000 RNAi reagents for human and drosophila, respectively. The database can be searched by phenotype, gene or RNAi probe and is publicly accessible at http://www.genomernai.org. <br>Our aim is comprehensive coverage of all published genome-wide RNAi screening data in GenomeRNAi, including large-scale data from drosophila in vivo screens. We have developed an annotation guide to ensure consistent data collection. Work to adapt it for in vivo data is ongoing. We will report on curation progress and present an update on new features on the GenomeRNAi website.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E29'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E29</strong></td>
 </tr> <tr> 
  <td><b>PROSITE: Enhancing the functional characterization of proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Nicolas Hulo <i>Swiss Institute of Bioinformatics</i></td>
 </tr>
 <tr> 
  <td>  Béatrice Cuche (Swiss Institute of Bioinformatics, Swiss-Prot group); Lorenzo Cerutti  (Swiss Institute of Bioinformatics, Swiss-Prot group); Lydie Bougueleret  (Swiss Institute of Bioinformatics, Swiss-Prot group); Ioannis Xenarios  (Swiss Institute of Bioinformatics, Swiss-Prot group); Christian Sigrist (Swiss Institute of Bioinformatics, Swiss-Prot group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: PROSITE consists of documentation entries describing protein domains, families and functional sites, as well as associated patterns and profiles to identify them. It is complemented by ProRule, a collection of rules based on profiles and patterns, which increase the discriminatory power of these profiles and patterns by providing additional information about functionally and/or structurally critical amino acids. PROSITE is largely used for the annotation of domain features of UniProtKB/Swiss-Prot entries. PROSITE current developments focus on further characterizing protein functions. One of the strategies under investigation to achieve this lies in dividing functionally heterogeneous families into subfamilies of proteins with homologous functions. PROSITE is also being adapted to support Next Generation Sequencing projects. These recent developments were applied on the newly sequenced fire ant genome and have allowed the functional annotation of more than 50% of the predicted proteins. The latest version of PROSITE (Release 20.71, of 08-Mar-2011)<br>contains  1308 patterns, 920 profiles and 915 ProRule. PROSITE is accessible<br>at: http://www.expasy.org/prosite/.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E30'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E30</strong></td>
 </tr> <tr> 
  <td><b>PHI-base 4.0: A Second Generation Pathogen-Host Interaction database</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jacek Grzebyta <i>Rothamsted Research</i></td>
 </tr>
 <tr> 
  <td>  Andrea Splendiani (Rothamsted Research , Biomathematics and Bioinformatics); Martin Urban (Rothamsted Research , Biomathematics and Bioinformatics); Kim Hammond-Kosack (Rothamsted Research , Biomathematics and Bioinformatics); Chris J. Rawlings (Rothamsted Research , Biomathematics and Bioinformatics); Mansoor Saqi (Rothamsted Research , Biomathematics and Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: PHI-base, the Pathogen-Host Interaction database (www.phi-base.org), is an open access internet resource which provides information on pathogenicity, virulence and effector genes from fungal, oomycete and bacterial pathogens, where the contribution of the genes to pathogenicity has been experimentally tested.  It is based on manually curated information retrieved from the peer-reviewed scientific literature, and went on-line in 2005. Two developments have necessitated a redesign of PHI-base, namely the need to depict more complex host-pathogen relationships as suggested by emerging new experimental methods and the need to facilitate community curation. <br>PHI-base 4.0 has a new schema which describes pathogen host interactions in terms of a system perturbation, which provides the basis for better integration with external resources by clarifying the semantics of an interaction. Additionally a community tool for submission of data by domain experts is provided, which will employ mechanisms of checking data quality and editorial control tools. Source code modularisation is achieved using several external tools such as OpenCms (content management system), Spring WebFlow and Hibernate implementation of Java Persistence API (JPA) for communication with the database. This redesign brings several significant benefits to the ease of database service/upgrade and the development of new functionalities. The provisional version shows the promise of this approach.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E31'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E31</strong></td>
 </tr> <tr> 
  <td><b>The EBI  Metagenomics Portal</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Craig McAnulla <i>European Bioinformatics Institute</i><br></td>
 </tr>
 <tr> 
  <td>Christopher Hunter (European Bioinformatics Institute) Matt Corbett  (European Bioinformatics Institute, InterPro); Matthew  Fraser (European Bioinformatics Institute, InterPro); Phil  Jones (European Bioinformatics Institute, InterPro); Alex Mitchell (European Bioinformatics Institute, InterPro); Sebastien Pesseat (European Bioinformatics Institute, InterPro); Antony Quinn (European Bioinformatics Institute, InterPro); Maxim Scheremetjew (European Bioinformatics Institute, InterPro); Sarah Hunter (European Bioinformatics Institute, InterPro); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The study of all genomes present in any given environment without the need for prior individual identification or amplification is termed metagenomics.<br><br>The ENA (Sequence Read Archive and EMBL-Bank), UniProt, InterPro, Ensembl Genomes and Gene Ontology (GO) resources, based at the EBI, are all used for metagenomic analysis. Here we present a new web resource targeted at metagenomic researchers offering a user friendly interface to these (and other) services. Currently released in beta, we intend to make rapid and frequent updates to improve the interfaces and services provided.<br><br>The EBI Metagenomics portal enables submission of raw data to the SRA, followed by protein coding sequence predictions (pCDS). InterProScan then scans the pCDS against predictive protein signatures in the InterPro databases. The in-depth annotations resulting from this, including GO terms, can then be used to determine the make-up of the metagenome. In future iterations we intend to offer comparison to complete reference genomes, metabolic pathway analysis, search facilities for both sequence and contextual data, and robust statistical analyses that can be compared across samples.<br><br>Presently, analysis is restricted to "long" (average reads lengths over 250nt), unassembled random shotgun sequence reads, i.e. Roche 454 sequences, from metagenomic or metatranscriptomic samples.<br><br>Additional tools and analyses will become available during the next few months and we actively encourage users to make requests for the tools that they wish to see incorporated.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E32'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E32</strong></td>
 </tr> <tr> 
  <td><b>Recent developments to the Ensembl website</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Stephen Trevanion <i>Wellcome Trust Sanger Institute</i></td>
 </tr>
 <tr> 
  <td>  Ridwan Amode (Wellcome Trust Sanger Institute, Informatics); Simon Brent (Wellcome Trust Sanger Institute, Informatics); Bethan Pritchard (Wellcome Trust Sanger Institute, Informatics); Anne Parker (Wellcome Trust Sanger Institute, Informatics); Harpreet Riat (Wellcome Trust Sanger Institute, Informatics); Steve Searle (Wellcome Trust Sanger Institute, Informatics); Maurice Hendrix (Wellcome Trust Sanger Institute, Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Ensembl genome browser (www.ensembl.org) is updated four to five times a year, presenting both new data and also enhancements to the website. Recent developments include: (i) Individual tracks on location-based views can be quickly configured by clicking on the track label, options available include  displaying only protein coding transcripts and tagging tracks as 'Favourites'. (ii) The vertical ordering of tracks on location-based views can be altered using drag and drop. (iii) Tables can be sorted by any column, and the rows visible can be filtered. (iv) For logged in users a history of the last five genes (and locations and transcripts) is available. (v) User data upload supports BAM and BigWig file formats, and multiple user datasets can be viewed and compared in a single multispecies page. (vi) The search engine supporting the site has been changed from a proprietary solution to the open source Lucene project. As well as allowing us to more easily develop search, for example static content and help documentation are now included, this will also makes it easier for external developers. Ongoing developments on the website include (i) development of a unified ticketing system for Ensembl tools such as the Variant Effect Predictor and BLAST, and (ii) the development of a modular help system that will allow for the presentation of context sensitive help information to users, and will also allow sites based on Ensembl to modify, rather than having to rewrite, existing Ensembl help documentation.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E33'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E33</strong></td>
 </tr> <tr> 
  <td><b>The Bioinformatics Links Directory: A Community Curated Collection of Bioinformatics Links, Tools and Databases</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Michelle Brazas <i>Ontario Institute for Cancer Research</i></td>
 </tr>
 <tr> 
  <td>  David Yim (Ontario Institute for Cancer Research, Informatics and Biocomputing); Joseph Yamada (Ontario Institute for Cancer Research, Informatics and Biocomputing); Raffi Melkon (Ontario Institute for Cancer Research, Informatics and Biocomputing); Winston Yeung (Ontario Institute for Cancer Research, Informatics and Biocomputing); Francis Ouellette (Ontario Institute for Cancer Research, Informatics and Biocomputing); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Bioinformatics Links Directory (bioinformatics.ca/links_directory) is a compendium of useful bioinformatics resources, tools and databases organized in an intuitive, research-task hierarchy. The directory, begun in 2002, is maintained by the Canadian Bioinformatics Workshops and is constantly enhanced by community suggestions and input. <br><br>The Bioinformatics Links Directory has recently been upgraded to allow for a better user experience and to provide more informative content. These upgrades include:<br>?	Inclusion of NAR Database content in addition to the NAR Web Server issue content;<br>?	Better content management to facilitate user input of useful resources, tools and databases;<br>?	Introduction of metadata to allow for more meaningful content searches;<br>?	Introduction of tags from user input and PubMed MeSH terms for better content curation;<br>?	Development of user groups to encourage collaborative content curation and maintenance within a group or an institution;<br>?	Ability to add documentation and supplementary educational content to a tool or database;<br><br>Continued community input regarding useful bioinformatics links and addition of supporting documentation and educational materials, will ensure that the Bioinformatics Links Directory remains an invaluable bioinformatics resource for everyone.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E34'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E34</strong></td>
 </tr> <tr> 
  <td><b>Structural Biology Knowledgebase: An Online Portal to Proteins, Functions, Methods, and More</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Torsten Schwede <i>Rutgers University</i><br></td>
 </tr>
 <tr> 
  <td>Margaret Gabanyi (Rutgers University) John Westbrook (Rutgers University, Chemistry and Chemical Biology); Wendy Y.-P. Tao (Rutgers University, Chemistry and Chemical Biology); Raship Shah (Rutgers University, Chemistry and Chemical Biology); David Micallef (Rutgers University, Chemistry and Chemical Biology); Konstantin Arnold (University at Basel, Swiss Institute for Bioinformatics and Biozentrum); Juergen Kopp (University at Basel, Swiss Institute for Bioinformatics and Biozentrum); Lorenza Bordoli (University at Basel, Swiss Institute for Bioinformatics and Biozentrum); Paul D. Adams (Lawrence Berkeley Laboratory, Physical Biosciences); Wladek Minor (Univ of Virginia Medical School, Molecular Physiology and Biological Physics); Helen Berman (Rutgers University, Chemistry and Chemical Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Structural Biology Knowledgebase (SBKB, URL: http://sbkb.org) is a free online resource designed to combine many protocols and results of the structural genomics and structural biology efforts with information from the biological community in order to have a better understanding living systems and disease. We will present examples of how to navigate the SBKB and how to use it to enable biological research. For example, a protein sequence or PDB ID search will provide a list of protein structures from the Protein Data Bank, associated biological descriptions (annotations), homology models, structural genomics protein target information, experimental protocols, and the ability to order available DNA clones. A text search will find technology reports and publications that were created by the Protein Structure Initiative's high-throughput research efforts. Web tools that aid in bench top research, such as the Sequence Comparison and Analysis tool for protein construct design, will also be demonstrated. Created in collaboration with the Nature Publishing Group, the Structural Biology Knowledgebase Gateway provides a research library, editorials about new research advances, news, and an events calendar also present a broader view of structural genomics and structural biology.<br>The SBKB is funded by the Protein Structure Initiative/NIGMS.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E35'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E35</strong></td>
 </tr> <tr> 
  <td><b>Development of a Small Portable Application for Plant DNA Barcoding</b><br/><br/></td>
 </tr>
 <tr> 
 <td>M Mehedi Hassan <i>University of Glamorgan, Pontypridd, Wales, UK</i></td>
 </tr>
 <tr> 
  <td>  Dr. Gaurav Sablok (Huazhong Agricultural University, Shizishan,Wuhan 430070,, Key Lab of Horticultural Plant Biology (MOE)); Farzana Rahman (University of Glamorgan, Pontypridd, Wales, UK, Computational Biology Group); Dr. Tatiana Tatarinova (University of Glamorgan, Pontypridd, Wales, UK, Computational Biology Group); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: During the last quarter of twentieth century, we have seen the rise in the inference of the phylogeography of species and species discrimination using the chloroplast genes as DNA barcodes. With growing demand for sequencing the chloroplast genes as DNA barcodes and for the effective management of the sequences in accordance with species timeliness, we developed a relational database model which, not only serves for the effective management tool for the biodiversity conservation but also serves as a database for the systematic storing and reporting of the DNA barcodes related to the species. A species-specific form along with primer information and the most robust DNA combination tool form are also made available.<br>The potential of this application relies in its portability. The term ‘portability’ has wider implication viz. small to medium size lab-specific data can be stored and manipulated. Furthermore, the application can be accessed any time without remote connection. The user application cum database is user friendly and can be manipulated to store large data. Initially as per CBOL (Consortium on DNA barcoding of Life), we have included mainly rbcL, matK and trnH-psbA as potential coding and non-coding barcodes. The user can further manipulate to add more potential barcodes with the primer sequence and the robust primer identification for species-specific study.<br><br>In our poster, we will demonstrate the basic design and construction of the database model and its implications in phylogeny and effective data management using a portable application.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E36'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E36</strong></td>
 </tr> <tr> 
  <td><b>MobiDB: a database of intrinsically disordered proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tomas Di Domenico <i>University of Padua</i></td>
 </tr>
 <tr> 
  <td>  Alberto Martin (University of Padova, Biology); Ian Walsh (University of Padova, Biology); Silvio Tosatto (University of Padova, Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: MobiDB is a web accessible relational database of protein disorder. It provides a centralized source for the analysis of data on structurally mobile regions in protein structures, with an option to access non-redundant representative subsets. Two different sources of raw data and three different definitions of disorder are included and available: PDB structures resolved by X-ray cristallography, where the standard binary mobility definition of intrinsic disorder in X-ray crystallographic structures was used; PDB structures resolved by nuclear magnetic resonance, where mobility was determined by processing the structures using the MOBI web server; and disorder data obtained from the DisProt database. In addition, MobiDB features a subset of associated data providing information on mobile residue patterns, obtained by analyzing the relationship between protein sequence and intrinsic disorder. These patterns provide insight on the physical reasons for disordered structures.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E37'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E37</strong></td>
 </tr> <tr> 
  <td><b>Codon usage database in the cloud</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Denis Shestakov <i>Aalto University</i></td>
 </tr>
 <tr> 
  <td>  Tommi Lindholm (Aalto University) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Most amino acids can be coded by more than one triplet of nucleotides  <br>(codon). Codons encoding the same amino acid (known as synonymous) are not used with equal frequencies: their usage is different in different genomes and even within the same genome. Bias in codon usage is an essential feature of both bacterial and eukaryotic genomes. Among the important uses of different patterns in codon usage for different organisms are to identify highly expressed genes and to detect horizontal gene transfer events.<br>Currently there are no up-to-date comprehensive database of codon usage patterns available. The existing codon usage repositories are either based on  <br>outdated sequence data or cover only prokaryotic organisms. Our motivation is to provide researchers involved in codon usage studies with a comprehensive database of codon usage patterns for all organisms with completed genomes. In this poster, we present a comprehensive codon usage repository for all completed<br>genomes (calculated based on the latest EMBL Nucleotide Sequence Database release). For every protein-coding sequence several codon usage measures (e.g., relative synonymous codon usage) and indices (e.g., effective number of codons) are available and can be easily accessed. We also describe a workflow for scalable<br>codon usage calculations that was used for building the proposed codon usage database. Our workflow is based on a large-scale distributed batch processing  <br>framework called Hadoop and deployed using cloud computing services and thus could be of high interest to bioinformaticians involved in analyzing large amounts of nucleotide sequence data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E38'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E38</strong></td>
 </tr> <tr> 
  <td><b>BioMart Central Portal – an open database network for biological community</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Junjun Zhang <i>Ontario Institute for Cancer Research</i><br></td>
 </tr>
 <tr> 
  <td>Arek Kasprzyk (Ontario Institute for Cancer Research) Joachim  Baran (Ontario Institute for Cancer Research, Informatics and Bio-computing); Anthony Cros (Ontario Institute for Cancer Research, Informatics and Bio-computing); Jonathan Guberman (Ontario Institute for Cancer Research, Informatics and Bio-computing); Syed Haider (University of Cambridge, Computer Laboratory); Jack  Hsu (Ontario Institute for Cancer Research, Informatics and Bio-computing); Yong Liang (Ontario Institute for Cancer Research, Informatics and Bio-computing); Elena Rivkin (Ontario Institute for Cancer Research, Informatics and Bio-computing); Jianxin Wang (Ontario Institute for Cancer Research, Informatics and Bio-computing); Long Yao (Ontario Institute for Cancer Research, Informatics and Bio-computing); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: BioMart Central Portal (http://www.biomart.org) is a first-of-its-kind, community-driven access point comprising dozens of biological databases, spanning genomics, proteomics, model organisms, cancer data, and more. All these databases are developed and maintained independently by groups of experts in their respective field. Because all databases in the portal share a common interface, the need for users to learn multiple systems is substantially reduced. Most importantly, by allowing datasets to be linked together, the portal simplifies cross-database searches and allows resources to be combined in novel ways, perhaps revealing unexpected connections or suggesting new avenues of inquiry. For advanced querying, data can be accessed programmatically using several methods, including a Java API, REST, and SOAP queries, as well as accessed via third-party tools such as BioConductor. Furthermore, several integrated tools are available to streamline common tasks, such as converting ID formats and finding homologues. <br>The combination of a wide variety of databases, an easy-to-use interface, robust programmatic access, and the array of tools make BioMart Central Portal a one-stop shop for biological data querying. Any database can easily be converted to a BioMart source using a suite of data transformation, configuration, and deployment tools, and can then be added to BioMart Central Portal by its administrator.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E39'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E39</strong></td>
 </tr> <tr> 
  <td><b>Community Cyberinfrastructure for Advanced Microbial Ecology Research and Analysis - The CAMERA  Resource</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Shulei Sun <i>University of CA, San Diego</i></td>
 </tr>
 <tr> 
  <td>  Jing Chen (University of CA, San Diego, Center for Research in Biological Systems); Weizhong Li (University of CA, San Diego, Center for Research in Biological Systems); Eric Allen (University of CA, San Diego, Center for Research in Biological Systems); Karen Stocks (University of CA, San Diego, Center for Research in Biological Systems); Abel Lin (University of CA, San Diego, Center for Research in Biological Systems); Ilkay Altintas (University of CA, San Diego, Center for Research in Biological Systems); Steven Peltier (University of CA, San Diego, Center for Research in Biological Systems); Mark Ellisman (University of CA, San Diego, Center for Research in Biological Systems); John Wooley (University of CA, San Diego, Center for Research in Biological Systems); Jeffrey Grethe (University of CA, San Diego, Center for Research in Biological Systems); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Community Cyberinfrastructure for Advanced Microbial Ecology Research and Analysis (CAMERA, http://camera.calit2.net/) is a database and associated computational infrastructure that provides a single system for depositing, locating, analyzing, visualizing and sharing data about microbial biology through an advanced web-based analysis portal. CAMERA collects and links metadata relevant to environmental metagenome datasets with annotation in a semantically-aware environment allowing users to write expressive semantic queries against the database.  To meet the needs of the research community, users are able to query metadata categories such as habitat, sample type, time, location, and other environmental physicochemical parameters. CAMERA is compliant with the standards promulgated by the Genomic Standards Consortium (GSC), and sustains a role within the GSC in extending standards for content and format of the metagenomic data and metadata and its submission to the CAMERA repository. To ensure wide, ready access to data and annotation, CAMERA also provides data submission tools to allow researchers to share and forward data to other metagenomics sites. It has multiple interfaces for easy submission of large or complex datasets, and supports pre-registration of samples for sequencing. CAMERA integrates a growing list of bioinformatic applications, tools and viewers for querying, analyzing, annotating, and comparing publically available and user-provided metagenome and genome data. All tools are organized through a collaborative scientific workflow system with provenance. This data-oriented view of an analysis enables communication (to collaborators) of what has occurred within a given workflow, and allows for the exchange and reproducibility of the computation itself.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E40'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E40</strong></td>
 </tr> <tr> 
  <td><b>Embryonic Stem Cell Database</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Elena Nikolaeva <i>University of Tartu</i></td>
 </tr>
 <tr> 
  <td>  Hedi Peterson (University of Tartu, Institute of Molecular and Cell Biology); Jaak Vilo (University of Tartu, Institute of Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Embryonic Stem Cell Database (ESCDb) http://biit.cs.ut.ee/escd/ offers a summarized view of multiple pluripotency related datasets. The database provides easy access to transcription factor binding data together with various perturbation experiments. ESCDb gathers mainly two types of data – chromatin immunoprecipitation array-based data on transcription factor targets and gene specific knockdown of pluripotency associated factors.<br>	An aggregate understanding of the transcriptional regulation of embryonic stem cells is essential for exploiting these cells in regenerative medicine. We are in an era of high-throughput functional genomics and systems biology-driven research where large datasets are usually needed and provided as supplementary tables in most publications. Though useful, such tables in isolation are of limited use for making cross-references across other related datasets. <br>	We have developed a specialized database, which enables rapid and convenient access and comparisons between published datasets related to embryonic stem cell biology to help overcome this shortfall. <br>	Due to the difficulties in understanding large quantities of data, information visualization techniques have become an attractive option for the field of bioinformatics. As part of ongoing research, we developed further the ESCDb that now allows displaying the promoter region together with chromatin immunoprecipitation data for genes listed in the ESCDb. We believe this visualization will prove particularly valuable for biologists who wish to visualize the promoter regions and generate figures for publications.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E41'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E41</strong></td>
 </tr> <tr> 
  <td><b>RiceXPro and Rice TOGO Browser: Integrated Databases for Rice Functional and Applied Genomics</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hajime Ohyanagi <i>Mitsubishi Space Software Co., Ltd.</i></td>
 </tr>
 <tr> 
  <td>  Yutaka Sato (National Institute of Agrobiological Sciences, Genome Resource Center); Baltazar A. Antonio (National Institute of Agrobiological Sciences, Genome Resource Center); Akio Miyao (National Institute of Agrobiological Sciences, Genome Resource Center); Nobukazu Namiki (Mitsubishi Space Software Co., Ltd., Tsukuba Division); Hinako Takehisa (National Institute of Agrobiological Sciences, Genome Resource Center); Jun-ichi Yonemaru (National Institute of Agrobiological Sciences, QTL Genomics Research Center); Hiroshi Minami (Mitsubishi Space Software Co., Ltd., Tsukuba Division); Kaori Kamatsuki (Mitsubishi Space Software Co., Ltd., Tsukuba Division); Kazuhiko Sugimoto (National Institute of Agrobiological Sciences, QTL Genomics Research Center); Kan Shimura (Mitsubishi Space Software Co., Ltd., Tsukuba Division); Hiroshi Ikawa (Mitsubishi Space Software Co., Ltd., Tsukuba Division); Yuji Shimizu (Mitsubishi Space Software Co., Ltd., Tsukuba Division); Hirohiko Hirochika (National Institute of Agrobiological Sciences, Division of Genome Research and Biodiversity); Yoshiaki Nagamura (National Institute of Agrobiological Sciences, Genome Resource Center); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: One of the ultimate goals in cereal genomics is to elucidate the function of all predicted genes in rice and ensure the development of improved varieties that will sustain an expanding world population. We present two bioinformatics platforms with potential applications in various rice genomics strategies. RiceXPro (http://ricexpro.dna.affrc.go.jp/) is a gene expression database that provides an overview of the transcriptional changes throughout the growth of the rice plant in the field. It contains gene expression profiles of various organs and tissues derived from spatiotemporal and continuous profiling from transplanting until harvesting. The Rice TOGO Browser (http://agri-trait.dna.affrc.go.jp/) is an online public resource designed to facilitate integration and visualization of mapping data of BAC/PAC clones, genes, RFLP/SSR markers, and phenotype data represented as QTLs onto the genome sequence. Both database platforms provide for more efficient utilization of genome and gene expression information from the point of view of functional and applied genomics.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21067.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='E42'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E42</strong></td>
 </tr> <tr> 
  <td><b>SABIO-RK: Improvement of search capacities and user-friendliness</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Renate Kania <i>Heidelberg Institute for Theoretical Studies</i></td>
 </tr>
 <tr> 
  <td>  Lei Shi (Heidelberg Institute for Theoretical Studies, SDBV); Enkhjargal Algaa  (Heidelberg Institute for Theoretical Studies, SDBV); Andreas Weidemann (Heidelberg Institute for Theoretical Studies, SDBV); Ulrike Wittig (Heidelberg Institute for Theoretical Studies, SDBV); Martin Golebiewski (Heidelberg Institute for Theoretical Studies, SDBV); Maja Rey (Heidelberg Institute for Theoretical Studies, SDBV); Lenneke Jong (Heidelberg Institute for Theoretical Studies, SDBV); Isabel Rojas (Heidelberg Institute for Theoretical Studies, SDBV); Wolfgang Müller (Heidelberg Institute for Theoretical Studies, SDBV); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The mission of SABIO-RK is not only to provide the scientific community with a comprehensive, high-quality resource of kinetic parameters and the conditions under which they were measured, but also to make these data easy searchable and accessible.<br><br>With respect to previous versions we have further reduced the number of clicks necessary to obtain a result. At the same time the interface has become more understandable for users. Furthermore, query processing speed has been increased by orders of magnitude enabling precise estimations of the result size even while typing the query.<br><br>Additionally the following alternative searches have been implemented:<br><br>1. NCBI taxonomy tree based search for closely related organisms (e.g. Primates)<br>2. The possibility to search for entries inserted at a given date.<br>3. External identifier search (e.g. KeggID)<br>4. Reaction participant based parameter type search (e.g. Km for D-Fructose)<br><br>SABIO-RK is produced by the Scientific Databases and Visualization group from the Heidelberg Institute for Theoretical Studies (HITS).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E43'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E43</strong></td>
 </tr> <tr> 
  <td><b>Towards Linked Open Gene Mutations Data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Paolo Romano <i>National Cancer Research Institute</i></td>
 </tr>
 <tr> 
  <td>  Achille Zappa (University of Genoa, DIST); Andrea Splendiani (Rothamsted Research, Bioinformatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Semantic Web technologies are enough mature to offer a viable solution for data integration. A requirement for this is the conversion into RDF of data stored in relational databases. Although the human variation analysis is now one of the biggest issues, there is no mutation data available in RDF.<br>In this poster, we present a first prototype of a server implementing an RDF representation of the IARC TP53 Mutation Database. This prototype server was developed with the aim of studying all issues related to the publication of linked mutation data. It was developed by using the D2RQ platform. Automatic mappings were first generated. Later a fine tuned, manual revision of mappings was carried out in order to incorporate proper relationships making reference to ontologies widely adopted in the field, e.g., bibo (bibliographic ontology), and to link to external systems by the use of standardized URIs.<br>A prototype D2R server is now available on-line at http://ml370.istge.it:7777/. It includes somatic mutations, i.e., data on observed mutations, gene variations, summarizing effects of known mutations, and related bibliographic references. An HTML view and a SPARQL endpoint are available starting from this address. Linked Data views can be obtained from other tools.<br>This prototype demonstrates that an RDF representation of mutation data can already be easily set up. The main difficulty lies, as usual, on the identification of a shared, semantically meaningful, ontology-based representation of variation information. A revised version of the prototype including more shared concepts and the full IARC database is under development.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E44'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E44</strong></td>
 </tr> <tr> 
  <td><b>The BioGRID: a database of literature-curated protein and genetic interactions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrew Chatr-aryamontri <i>University of edinburgh</i></td>
 </tr>
 <tr> 
  <td>  Andrew Winter (University of Edinburgh, Life Sciences); Julie Nixon (University of Edinburgh, Life Sciences); Mike Tyers (University of Edinburgh, Life Sciences); Teresa Reguly (Samuel Luenfeld Research Institute, Life Sciences); Lorrie Boucher (Samuel Luenfeld Research Institute, Life Sciences); Bobby-Joe Breitkreutz (Samuel Luenfeld Research Institute, Life Sciences); Chris Stark (Samuel Luenfeld Research Institute, Life Sciences); Rose Oughtred (Princeton University, Lewis-Singler Institute); Michael Livstone (Princeton University, Lewis-Singler Institute); Jennifer Rust (Princeton University, Lewis-Singler Institute); Kara Dolinski (Princeton University, Lewis-Singler Institute); Kimberly Van Auken (California Institute of Technology, Division of Biology); Xiadong Wang (California Institute of Technology, Division of Biology); Xiaoqi Shi (Ontario Institute For Cancer Research, Division of Biology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Biological General Repository for Interaction Datasets (BioGRID) aims to systematically curate protein-protein and genetic interactions from the available literature for major eukaryotic model organisms. These literature-curated datasets serve as a convenient look-up source for biological interactions, a resource for network analysis and as a benchmark for high-throughput interaction studies. Curation of all physical and genetic interactions for the model yeasts S. cerevisiae (Reguly et al, J Biology 2006 5: 11-39) and S. pombe has been completed and forms the reference interaction data set for these organisms. Curation of physical interactions has been completed for A. thaliana and is updated on a monthly basis, and curation of interactions from C. elegans, D. melanogaster and H. sapiens on-going. <br><br>BioGRID version 3.1.75 contains 385,884 interactions from 26,523 publications. The new BioGRID3.0 web interface (www.thebiogrid.org) can be searched using gene names and identifiers. Each page represents a gene and its interactors, detailing names, links to other gene resources, and Gene Ontology terms. Evidence codes supporting the interaction from a total of 16 physical and 11 genetic evidence types are present, along with publications supporting the interaction (with links to PubMed). Interaction data is available to download in BioGRID custom tabular files, PSI MITAB and PSI-MI XML. BioGRID data in the MITAB format is also available on the PSICQUIC web service. In addition to our own Osprey network visualization tool, a plugin for the Cytoscape has been developed allowing easy access and visualization of BioGRID data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E45'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E45</strong></td>
 </tr> <tr> 
  <td><b>Correlation between annotation in the human gene database H-InvDB</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Katsuhiko Murakami <i>National Institute of Advanced Industrial Science and Technology</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Because of the widespread use of "Gene Set Enrichment Analysis (GSEA)", characterizing a given set of genes through identification of significantly enriched annotations has become an important task. The significant annotations can be a clue to explain why the gene expression increased. However, the interpretation as a set of annotations is often difficult for the user. The annotations are shown as if they were independent, despite that some annotations are actually correlated each other. To elucidate the relationships among the annotations, we comprehensively examined how much each annotation is correlated each other through genes. We selected eight gene annotation features (Gene Family, Gene Ontology, InterPro, KEGG pathway, OMIM, SCOP, Tissue specificity of gene expression, and Wolf-PSORT) in the integrated human gene database, H-InvDB. For each pair of individual annotations (e.g. GO:0004252 and IPR001627), we counted the number of genes with/without the two annotations respectively, resulting a 2x2 contingency table. We then performed Fisher's exact (one-side) test. The correlations of a pair of the annotation features were evaluated using the p-value with Bonferroni correction. As a result, we found many significant pairs. For example, 27,708 annotation pairs (20%) showed significant p-values when we compare the annotations in InterPro and Wolf-PSORT. The relationships between annotations may include yet unexplored relationships (hypothesis), as well as known relationships known. With the relationships, one can understand the annotations much deeply, and interpret the annotations, especially for the results of GSEA. Furthermore, the correlation map for each annotation will provide the bird-eye view of annotation features.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E46'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E46</strong></td>
 </tr> <tr> 
  <td><b>A Ruby API to query the Ensembl database for genomic features</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Jan Aerts <i>Leuven University</i></td>
 </tr>
 <tr> 
  <td>  Francesco Strozzi (Parco Tecnologico Padano, IDRA Lab); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The Ensembl database makes genomic features available via its genome browser. It is also possible to access the underlying data through a Perl API for advanced querying. We have developed a full-featured Ruby API to the Ensembl core and variation databases, providing the same functionality as the Perl interface (including data retrieval and coordinate system projections). However, this library also provides several advantages compared to the Perl API. First, a single Ruby API is used to access all releases of the Ensembl databases and is also able to query multi-species databases; the library code is independent of release number. Second, the Ruby Ensembl API includes a powerful interactive shell environment. Third, this API has - we believe - a more useful implementation of the Slice concept, whereby the Slice of a feature (e.g. gene) is delineated by the boundaries of that feature rather than comprises the complete sequence region (e.g. chromosome). Finally, the extensive metaprogramming and introspection capabilities of the Ruby programming language and ActiveRecord framework allow for a very small code-base which is easily maintainable while providing full functionality.<br><br>The API is available through the RubyGem system and can be installed with the command "gem install ruby-ensembl-api". Development is managed from http://github.com/jandot/ruby-ensembl-api and a community feedback website is available at http://rubyensemblapi.userecho.com</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='E47'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster E47</strong></td>
 </tr> <tr> 
  <td><b>Identifying knowledge contributors – motivating online sharing of research data</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gudmundur Thorisson <i>University of Leicester</i></td>
 </tr>
 <tr> 
  <td>  Owen Lancaster (University of Leicester, Genetics); Anthony Brookes (University of Leicester, Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Contributor identification is a core challenge in data publication. As in scholarly communication more generally, non-unique person names and the current lack of a global identification infrastructure for producers of scholarly content makes it difficult to establish the identity of authors and other contributors. This in turn makes it difficult to accurately attribute datasets published via online digital repositories to their creators – one of several key requirements for including these important outputs in the scholarly record.<br><br>In the GEN2PHEN project (http://www.gen2phen.org) we are developing a series of novel web-based systems and processes for online dissemination of genetic variation and other research data. The core aim is that of ensuring that data creators are recognized and rewarded for publishing data. This work builds on and integrates with recently launched international initiatives to i) extend and adapt the existing DOI infrastructure for identifying, locating and citing online datasets (DataCite: http://www.datacite.org), and ii) create a global registry of unique identifiers for authors and other contributors (ORCID: http://www.orcid.org).<br><br>The technical approach we are exploring in this pilot project utilizes this emerging global data citation and contributor identification framework, in order to allow published datasets to be discovered, cited in a scholarly context and unambiguously attributed. We argue that, along with other measures, such an incentive-based approach is key to motivating the sharing of data and other types of digital research outputs in the life sciences.<br><br><br>GEN2PHEN is funded by the European Community's Seventh Framework Programme (FP7/2007-2013) under grant agreement 200754.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




