
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php?cat=L by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:54:26 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/posterlistfdc3.html?cat=L by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:37:18 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Posters</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top"><br><br>
<h3>Accepted Posters</h3>



<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;">
  <a name="cat"></a>
<b>Category 'L'- Interactions'</b></td>
 </tr>
</table>

<a name='L01'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L01</strong></td>
 </tr> <tr> 
  <td><b>GENS2: a simulator for multiloci/whole-genome case-control studies with gene-environment interactions</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Roberto Amato <i>Universita' di Napoli "Federico II"</i></td>
 </tr>
 <tr> 
  <td>  Giovanni Scala (Universita' degli Studi di Napoli "Federico II", Dipartimento di Scienze Fisiche); Michele Pinelli (Universita' degli Studi di Napoli "Federico II", DBPCM "L. Califano"); Sergio Cocozza (Universita' degli Studi di Napoli "Federico II", DBPCM "L. Califano"); Gennaro Miele (Universita' degli Studi di Napoli "Federico II", Dipartimento di Scienze Fisiche); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Complex diseases represent the vast majority of human diseases and include those with largest prevalence and mortality, like cancer, obesity, etc. They are determined by both genetic and environmental factors, often interacting in a non-linear, non-additive way.<br>Despite an even increasing amount of information has been collected about both genetic and environmental risk factors, there are still few examples of studies on their interactions in literature. One reason for this lack can be found in the incomplete knowledge of the power of statistical methods designed to search for interacting risk factors in these data sets.<br>A possible strategy is to challenge the different statistical methods against data sets where the underlying phenomenon is completely known and fully controllable, like for example the simulated ones. In this direction, we developed a mathematical model (Multi-Logistic Model, MLM) able to describe a wide variety of gene-environment interactions. We also implemented the MLM in a user-friendly tool (Gene-Environment iNteraction Simulator, GENS) where a knowledge-based approach reduces the complexity of the mathematical model by using reasonable biological constraints and make the simulation more understandable in biological terms, and a Monte Carlo process allows random variability. <br>Here we present GENS2, an extension of our tool that allows for easy-to-define epistatic interactions and whole-genome data sets with realistic linkage disequilibrium patterns.<br>As a whole, this allows the user to produce even more realistic data set for the assessment of novel statistical methods or for the evaluation of the statistical power while designing a study.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L02'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L02</strong></td>
 </tr> <tr> 
  <td><b>Universal epitope prediction for class II MHC</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Andrew Bordner <i>Mayo Clinic</i></td>
 </tr>
 <tr> 
  <td>  Hans Mittelmann (Arizona State University, School of Mathematical and Statistical Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Predicting peptide-class II MHC binding affinities is a challenging problem due to MHC diversity and multiple binding modes but has many biomedical applications.  We recently developed a structure-based approach using peptide docking and machine learning to predict peptide-MHC binding affinities.  Unlike popular sequence-based methods, it is applicable to any MHC type because it relies on universal physical interactions rather than limited experimental data for specific MHC types.   Using a model trained only on DRB1*0101 binding data we were able to accurately predict peptide binding affinities for all human class II MHC loci (HLA-DP, DQ, and DR) and for two murine MHC types.  This provides the first demonstration that a single prediction model can be applied to diverse MHC types with completely different binding specificities.  In addition, we will review our RTA sequence-based prediction method, which outperformed more complicated competing methods, and discuss recent work.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/81/20126.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='L03'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L03</strong></td>
 </tr> <tr> 
  <td><b>Specificity and Redundancy in Protein-Protein Interaction Interfaces</b><br/><br/></td>
 </tr>
 <tr> 
 <td>David Talavera <i>University of Manchester</i></td>
 </tr>
 <tr> 
  <td>  Simon Williams (University of Manchester, Faculty of Life Sciences); Matthew Norris (University of Manchester, Faculty of Life Sciences); David Robertson (University of Manchester, Faculty of Life Sciences); Simon Lovell (University of Manchester, Faculty of Life Sciences); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein-protein interactions are involved in almost all biological processes. Their importance may suggest that there is strong evolutionary constraint on protein-protein interaction interfaces. However the degree of sequence conservation in interfaces is relatively weak. This lack of conservation cannot be explained by coevolution in interfaces; although coevolution is observed in some cases, it is by no means ubiquitous and is often undetectable. An alternative explanation for the lack of sequence conservation in protein-protein interaction interfaces may be some degree of redundancy, i.e., some interacting residues being replaceable by other types. Here we introduce the concept of “residue-type independent” interactions. We define an interaction as being independent of residue type if it is made only through the protein main-chain, the beta-carbon, or associated hydrogen atoms. Since all residues have main-chain atoms, and all except glycine have a ?-carbon and at least one ?-hydrogen, we hypothesize that residues that make interactions only through these atoms may be more interchangeable than those that make interactions through other side-chain atoms. We find that nearly a quarter of residues in protein interaction interfaces make inter-chain contacts that are not dependent on residue type. These residues are less structurally constrained than residues making interactions that are residue-type specific. We propose therefore that residue-type independent interactions are an important source of functional redundancy in protein-protein interaction interfaces. This redundancy not only explains the interfaces’ lack of conservation, but also provides interfaces with evolutionary robustness and the ability to take part in a range of functional roles.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L05'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L05</strong></td>
 </tr> <tr> 
  <td><b>Applications of Domain-Domain Interactions – Deriving Cancer Specific Domain-Domain Interactions, Differential Protein-Protein Interactions due to Alternative Splicing in Liver Cancer</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ka-Lok Ng <i>Asia University</i></td>
 </tr>
 <tr> 
  <td>  Chi-Ying  Huang (National Yang-Ming University, Institute of Clinical Medicine ); Jie-Wei  Wang (Asia University, Department of Biomedical Informatics); Wen-Chin  Chiang (Asia University, Department of Biomedical Informatics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Defects in protein-protein interaction (PPI) can possibly induced diseases, such as cancer. Behind PPI there are protein domains interacting with each others to perform the necessary functions. The first application of domain-domain interaction (DDI) is to derive a set of DDI pairs specifically for onco-proteins (OCP), and tumor suppressor proteins (TSP) based on their PPI. It is suggested that PPI, which is mediated by DDI, may be affected by domain removal due to alternative splicing. The second application of DDI in this work is to study the effects of liver cancer isoforms’ PPI, caused by domain removal. <br>Three cancer specific sets of DDI, i.e. OCP-OCP, TSP-TSP, and OCP-TSP, are derived from PPI. Ten-fold cross-validation tests are utilized to evaluate the performance. It is found that the OCP-OCP set of DDI pairs achieves a sensitivity value of 80.5%. After removing overlapping DDI, the prediction sensitivity drops to 76.6%, however, the false positive rate can be significantly lowered. <br>The second application is to study the effects of domain removal on isoforms’ PPI where liver cancer proteins is selected as a case study. Certain liver cancer-related isoforms are found to have differential PPI. Alternative splicing of some proteins have been observed in liver cancer, and although still scarce, certain isoforms are found to have differential PPI. The available data suggest that splicing defects may have a role in hepato-carcinogenesis.<br><br>This poster is based on Proceedings Submission 96.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L06'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L06</strong></td>
 </tr> <tr> 
  <td><b>Characterization of compound gene regulation mechanisms</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Elisa Domínguez Hüttinger <i>Imperial College London</i><br></td>
 </tr>
 <tr> 
  <td>Reiko Tanaka (Imperial College London) Mauricio Barahona (Imperial College London, Bioengineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Genes are transcribed in response to multiple environmental signals. This context-dependent cellular response is achieved by interactions among multiple signal-specific transcription factors (TFs) and the promoter of the target gene, and is termed here Compound Gene Regulation (CGR). <br>CGR has previously been studied by evaluating the effect of multiple environmental changes on the expression levels of specific genes. In these studies, intermediate processes between the stimulation and the genetic expression are either neglected or restricted to the particular gene under consideration, hindering systematic characterisation and comparison of different CGR mechanisms. However, a detailed mechanistic understanding of biochemical interactions underlying CGR is essential for the analysis and design of genetic networks. <br>We develop here a theoretical framework to characterize different CGR mechanisms in a unified way. The four main classes of CGR mechanisms are: no interactions, structural interactions (overlap of DNA binding sites), physical interactions (formation of a TF heteromer), and functional interactions (modification of the biochemical properties of the target gene promoter). We propose a mathematical model to describe these CGR mechanisms, and further develop indices to quantitatively characterize and compare their compound effects. Our results suggest that each CGR mechanism is represented by a characteristic transcriptional response curve, constituting a particular biological strategy of signal integration. These theoretical results provide a framework for the study and characterisation of CGR.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L07'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L07</strong></td>
 </tr> <tr> 
  <td><b>Structure-Derived Dynamic Properties of Protein-protein Interface</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tsun-Tsao Huang <i>National Chiao Tung University</i></td>
 </tr>
 <tr> 
  <td>  </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein-protein interaction sites (interface) use specific chemical and physical characteristics to perform molecular recognition. It has been studied that interfaces and surfaces are various in amino acid composition, hydrophobicity, size, shape, complementary and conservation; however, these major features of interfaces are diverse among proteins. In this study, we analyze protein surface residues by the weighted contact number (WCN), a structure-derived dynamics property reflecting the crowdedness and rigidity of a residue. In comparison with non-interface residues, our results showed that the core interface residues, which are thoroughly buried on complexation, have larger WCN; whereas the peripheral interface residues, which are partially exposed to solvent on complexation, have smaller WCN. These trends occur in both obligate and transient interactions; moreover, these trends are also found in enzyme/inhibitor and antibody/antigen interfaces. Our results imply that protein interface consist of rigid core residues surrounded by flexible residues, and this property might be useful in protein-protein interaction prediction.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L08'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L08</strong></td>
 </tr> <tr> 
  <td><b>Modeling the two-way relation in protein interaction network by global-local density based quasi-biclique</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Juan Liu <i>Wuhan University</i></td>
 </tr>
 <tr> 
  <td>  Tao Zeng (Wuhan University, School of Computer); Fei Luo (Wuhan University, School of Computer); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Scientific Justification: This poster is based on Proceedings Submission 184  Two-way relation indicates the relations between nodes from two groups. Detecting the two-way relations have been absorbing many attentions in both academic studies and actual applications of biology, for they are usually used to reflect the connections between genes/proteins in biological network. The quasi-bicliques are usually used as the general models of two-way relations in the analysis of biological network. To mine much more biological significant quasi-bicliques, we first a novel quasi-biclique GLDQB (global-local density based quasi-biclique) based on local density (inner density) and global density (outer density), and then develop an efficient heuristic algorithm GLDQBM to mine GLDQBs, based on four significant criteria as follows: size maximum, inner coherence, outer sparse and node balance. By comparing with LQBs (large quasi-biclique) mined by PPIExtend on protein-protein interaction network, the GLDQBs mined by GLDQBM are indeed more significant. Similar conclusions can also be obtained on the analysis of protein lethal interaction network.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L09'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L09</strong></td>
 </tr> <tr> 
  <td><b>Unraveling protein-protein interactions by a semi-automated high-throughput platform</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Dominik Mertens <i>Albert-Ludwigs-University Freiburg</i></td>
 </tr>
 <tr> 
  <td>  Anke Becker (Albert-Ludwigs-University Freiburg , Chair of Molecular Genetics ); Enrico Schmidt (Albert-Ludwigs-University Freiburg, Regulatory Networks); Javier Serrania (Albert-Ludwigs-University Freiburg , Chair of Molecular Genetics); Bernadette Boomers (Albert-Ludwigs-University Freiburg, Regulatory Networks); Ralf Baumeister (Albert-Ludwigs-University Freiburg, Bioinformatics and Molecular Genetics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: A myriad of biological processes is affected by protein-protein interactions (PPI), turning into research focus during the last decade. However, without further validation processes high-throughput PPI-screening techniques like yeast-two-hybrid (Y2H) come along with both high false-positive and high false-negative rates. <br>We have developed a half-automatic platform aiming to identify high-quality PPIs by saturated screening based on the successor of the Y2H, the split-ubiquitin system, which enables to identify protein interactions at the physiological site. The platform covers a robot to carry out pipeting steps, plate organization and storage via barcode identification, optical density (OD) readouts and colony picking. In order to circumvent the high false-positive rate, our analysis pipeline includes several verification steps to check if the single steps known as error-prone are successfully accomplished. Due to the high data gathering during the single analysis and verification steps which have to be carried out, we have developed an advanced laboratory information management system (LIMS) called PIE to handle robot control as well as data supervision. PIE processes output files created by the robot like OD readout growth curves to identify candidates to go on working and creates robot control files to perform the next steps in the analysis pipeline. <br>Final PPI results are stored in a PSI-MI compliant and secured database to overview the results and to gather further information about the interacting proteins by using web services. The adaption of PSI-MI enables the user to continue data analysis with software tools like Cytoscape supporting this widely used standard format.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L10'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L10</strong></td>
 </tr> <tr> 
  <td><b>Discovery of a 3D Motif Essential to LDLR-Protein Complexes: Application to docking prediction evaluation</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Reyhaneh Esmaielbeiki <i>Kingston University London</i></td>
 </tr>
 <tr> 
  <td>  Declan Naughton (Kingston University London, Life Sciences); Jean-Christophe Nebel (Kingston University London, Faculty of Computing, Information Systems and Mathematics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The low-density lipoprotein receptor (LDLR) family is capable of interacting with a wide variety of human, e.g. glycoprotein, apolipoprotein and alpha-2-MRAP, and virion proteins. This interaction relies on three LDLR acidic residues involved in calcium ion binding which create hydrogen bonds with a basic residue of the partner protein. Following the study of all available receptor-protein complex structures, an atomic 3D motif describing those interactions was produced. <br>The 3D motif was applied to the evaluation of docking predictions. This was performed using leave-one-complex-out cross validation. First, a resolved 3D complex involving LDLR is selected. Secondly, a 3D motif is produced using all the other available LDLR complexes. Thirdly, the complex is predicted using ClusPro 2.0 – the docking software which won the CAPRI2009 competition-. Then, the fitting of the 3D motif is used to score each of the generated putative complex models. Finally, the produced ranked list is compared with the list of models ranked according to their quality as expressed by their RMSD with respect to the actual resolved structure.<br>Result shows that, consistently, our method is able to discover among the 9 models with highest scores the 3 models which display the best quality. This indicates that the quality scores associated to the proposed 3D motif are useful indicators for the evaluation of docking models involving LDLR. Using this approach, the structure of a complex involving a human antimicrobial peptide (HNP1) is proposed. Its analysis supports earlier reports of its putative interaction with LDLR.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L11'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L11</strong></td>
 </tr> <tr> 
  <td><b>Coevolution and Predicted Solvent Accessibility</b><br/><br/></td>
 </tr>
 <tr> 
 <td>David Ochoa <i>National Center for Biotechnology (CNB-CSIC)</i></td>
 </tr>
 <tr> 
  <td>  Ponciano García (National Center for Biotechnology (CNB-CSIC), Systems Biology Program); David Juan (National Cancer Research Institute(CNIO), Structural & Bioinformatics Unit); Florencio Pazos (National Center for Biotechnology (CNB-CSIC), Systems Biology Program); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: To complement experimental approaches for detecting protein interactions (PPIs), many computational methods have been developed which explore different solutions to predict protein interactions based on sequence information. One of the most successful approaches is based on the idea that interacting proteins tend to share evolutionary forces and this fact is reflected on the protein sequences. The widely used mirrortree family of methods has already shown a considerable accuracy and coverage exploiting this co-evolution to predict PPIs. <br><br>In order to asses the effect of the characteristics of the alignment positions used for building the phylogenetic trees on the performance of this methodology, we filtered the multiple sequence alignments of the proteome of E.coli according with different thresholds of sequence conservation and predicted solvent accessibility (based on PROF). The trees generated from these filtered alignments were used for predicting interacting pairs of proteins using: Mirrortree, Coevolutionary Profiles and ContextMirror, and the predictions were validated against gold standards of multimeric complexes and binary physical interactions obtained from EcoCyc and MPIDB respectively.<br><br>Although restricting to conserved positions doesn’t seem to affect the global performance, we show that the prediction of physical interactions could be improved using only the positions predicted as accessible. We illustrate this observation with the recombinase BCD complex: restricting the tree generation to the alignment positions predicted as accessible has a double effect: it raises the co-evolutionary signal between these interacting proteins and lowers that of the non-interacting ones.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L12'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L12</strong></td>
 </tr> <tr> 
  <td><b>HIPPIE: a web tool for querying the human interactome and evaluating experimental PPI screens</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Martin Schaefer <i>Max Delbrück Center for Molecular Medicine</i></td>
 </tr>
 <tr> 
  <td>  Tiago J.S. Lopes (JST ERATO KAWAOKA, Infection-induced Host Responses Project); Jenny Russ (Max Delbrück Center for Molecular Medicine, Proteomics and Molecular Mechanisms of Neurodegenerative Diseases); Jean-Fred Fontaine (Max Delbrück Center for Molecular Medicine, Computational Biology and Data Mining); Arunachalam Vinayagam (Harvard Medical School, Department of Genetics); Pablo Porras-Millan (Max Delbrück Center for Molecular Medicine, Proteomics and Molecular Mechanisms of Neurodegenerative Diseases); Hiroaki Kitano (JST ERATO KAWAOKA, Infection-induced Host Responses Project); Erich E. Wanker (Max Delbrück Center for Molecular Medicine, Proteomics and Molecular Mechanisms of Neurodegenerative Diseases); Miguel A. Andrade-Navarro (Max Delbrück Center for Molecular Medicine, Computational Biology and Data Mining); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: HIPPIE (Human Integrated Protein-Protein Interaction rEference) is a versatile web tool which allows for querying and analyzing the human interactome. We integrated various public human protein-protein interaction (PPI) data sources and developed a continuous scoring scheme reflecting the experimental evidence supporting each interaction. HIPPIE offers several ways to access the human interactome: a distinctive feature of the HIPPIE web tool enables researchers to upload a list of proteins from which a tissue specific subnetwork is generated.<br><br>The annotation functionality of HIPPIE is particularly useful for experimentalists conducting PPI screens: lists of interactions can be uploaded which then get annotated with literature information (in case they have been observed before) or an estimate of their reliability (in case they were measured for the first time) which allows to select high-confidence interactions for validation screens or functional assays. This confidence score is calculated by integrating various features commonly applied to estimate the reliability of a measured interaction (functional similarity, domain composition, coexpression and distance in a PPI network of the two proteins). Additionally, we developed a novel score which estimates the probability of an edge in the PPI network conditional on functional properties of the protein neighborhood of the interactors. We integrate these different features with a Support Vector Machine and compare the performance with other methods.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L13'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L13</strong></td>
 </tr> <tr> 
  <td><b>On sequential conservation space of RNA-recognizing residues in protein sequences</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Li Xue <i>Iowa State University</i></td>
 </tr>
 <tr> 
  <td>  Rasna Walia (Iowa State University, Computer Science); Yasser  EL-Manzalawy (Iowa State University, Computer Science); Drena Dobbs (Iowa State University, Genetics, Development & Cell Biology); Vasant  Honavar (Iowa State University, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Protein-RNA interactions play an important role in cellular process like protein synthesis, RNA processing, and gene expression regulation. Reliable identification of RNA-recognizing residues in protein sequences is essential for comprehending the mechanism and the functional implications of RNA-protein interactions and provides a valuable guide for rational drug discovery and drug design. <br><br>In this study, we systematically study the protein-RNA interface conservation in putative sequence homologs on a representative non-redundant dataset of known prorein-RNA binding interfaces. A total of 8,970 query protein-homolog sequence alignment pairs are studied. Our analysis shows that: 1) RNA-recognizing residues in protein sequences are conserved in putative sequence homologs; 2) The conservation space clearly suggest three zones: Safe Zone (where interfaces are very well conserved in homologs), Twilight Zone (where interfaces are middle-level conserved) and Dark Zone(where interfaces are not well conserved); 3) The sequential conservation space can be represented as a function of six alignment statistics derived from BLAST alignment results : Local Alignment Length (LAL), BLAST E-Val, Sequence Identity, BLAST Positives, LAL/Query_length, LAL/Homolog_length. Based on our analysis we define a scoring function, Interface Conservation score IC-score, based on the alignment statistics for ranking the similarity of interfaces of homologs to that of a query protein. Our preliminary results suggest that the proposed function of conservation space can be effectively used for the transcriptome-wide prediction of the degree to which the interfaces of a putative sequence homolog can be used to transfer to a query protein.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L14'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L14</strong></td>
 </tr> <tr> 
  <td><b>Functional assessment of topological characterization using graphlet degrees in PPI networks</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Marco Mina <i>University of Padova</i></td>
 </tr>
 <tr> 
  <td>  Tiziana Sanavia (University of Padova, Department of Information Engineering); Barbara Di Camillo (University of Padova, Department of Information Engineering); Gianna Toffolo (University of Padova, Department of Information Engineering); Concettina Guerra (University of Padova, Department of Information Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Motivation<br><br>Topological analysis of PPI networks is useful to understand biological relationships between proteins. A recent approach (Milenkovic and Przulj, 2008) characterizes each node in terms of “graphlet degrees” counting the number of subgraphs (graphlets) that the node touches. The concept of “automorphism orbit” is used to distinguish the position of the node with respect to the graphlet.<br>This work investigates to what extent graphlet degrees characterize the functional role of proteins in Yeast and Human PPI networks.<br><br>Methods<br><br>Two types of analysis were performed:<br> <br>1. we considered groups of proteins belonging to the same complex (CYC2008 and HPRD database) and tested if they have similar graphlet degrees for at least one specific orbit. Statistical significance was assessed by comparison with random sets of proteins;<br><br>2. we considered groups of proteins with similar graphlet degrees (higher than the 99th percentile) for at least one specific orbit and tested if they share enriched functional annotations in terms of Gene Ontology (GO) molecular functions. Fisher’s Exact Test was used to assess statistical significance and information content was used to investigate the specificity of GO annotations.<br><br>Results<br><br>Proteins belonging to the same complex showed significant results in terms of participation to highly connected graphlets, i.e. cliques. Enriched GO terms were observed only for cliques and cliques with one missing edge. These GO terms were also characterized by high information content, above the 85th percentile. Results highlighted that only few specific types of subgraphs provide a topological characterization which is significantly related to biological functions in Yeast and Human PPI networks.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L15'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L15</strong></td>
 </tr> <tr> 
  <td><b>SNP-SNP synergy discovery with SNPsyn</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Gregor Rot <i>University of Ljubljana</i><br></td>
 </tr>
 <tr> 
  <td>Tomaz Curk (University of Ljubljana) Blaz Zupan (University of Ljubljana, Faculty of computer and information science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: SNPsyn (http://snpsyn.biolab.si) is an interactive web-based application for the discovery of synergistic pairs of single nucleotide polymorphisms (SNPs) in large genome-wide case-control association studies (GWAS) of complex diseases. SNPsyn is both a stand-alone C++/Flash application and a web server. The computationally intensive part is implemented in C++ and can run in parallel on a dedicated cluster or grid. We have developed the graphical user interface in Adobe Flash Builder 4 so that it can run in standard web browsers or as a stand-alone application.<br><br>The SNPsyn web server receives GWAS data submissions, invokes the interaction analysis and serves the results for rendering at the client site. Synergies of pairs of SNPs are estimated through interaction analysis, an information-theoretic approach. Synergy occurs when a combination of SNPs carries more information than the sum of information provided by individual SNPs. Several heuristics and GO term-based selection approaches have been implemented to limit the number of investigated SNPs and speed-up the analysis.<br><br>SNPsyn web application has a graphical visual analytics interface. Users can view the synergy and informativity scores, select a subset of most synergistic pairs, browse through the detailed lists of SNPs that are also linked to NCBI and HapMap, perform GO term enrichment analysis on selected pairs, and interact with the constructed SNP synergy network. SNPsyn aims to complement the present set of gene interaction analysis programs. Its simple and intuitive graphical interface should allow biomedical researchers to effortlessly upload, analyze and gain insight into their own data.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L16'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L16</strong></td>
 </tr> <tr> 
  <td><b>RISEARCH: Fast RNA Interaction Search</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Anne Wenzel <i>University of Copenhagen</i><br></td>
 </tr>
 <tr> 
  <td>Jan Gorodkin (University of Copenhagen) </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Non-coding RNAs typically function by interaction with other molecules, often other RNAs. For example, regulatory RNAs, such as eukaryotic microRNAs or bacterial sRNAs, form duplexes with their mRNA target sites. Thus, RNA-RNA interactions are a good place to start searching for putative functionality of predicted structured RNAs, and recent in silico screens have generated thousands of such candidates. The demand for genome-wide scale prediction of putative RNA-RNA interactions is therefore high. Algorithms that identify short stretches of perfect complementary sequences, such as BLAST, are typically fast but lack sensitivity. However, current thermodynamic models for RNA base pairing, as implemented in e.g. RNAcofold, are too complex to be used in larger scale screens. Therefore, methods with simplified interaction models need to be employed to pre-filter for regions of interest. Current programs that neglect intramolecular base pairing but still make use of thermodynamics, such as RNAplex, achieve a runtime of O(n·m). Here we present RISEARCH, which further reduces the runtime by simplifying the thermodynamic model. We employ a 25x25 scoring matrix, which approximates the Turner nearest neighbor energies, within a di-nucleotide Smith-Waterman-like algorithm. This significantly reduces the runtime of the interaction prediction and we show in a benchmark that we achieve a two- to eightfold speed improvement in comparison to RNAplex, the currently fastest method. We achieve a prediction accuracy comparable to that of RNAplex on two datasets of known bacterial sRNA-mRNA and eukaryotic microRNA-mRNA interactions.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L17'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L17</strong></td>
 </tr> <tr> 
  <td><b>Predicting conserved interactions and structures of two multiple alignments of RNA sequences.</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Stefan Seemann <i>University of Copenhagen</i></td>
 </tr>
 <tr> 
  <td>  Andreas Richter (University of Freiburg, Bioinformatics Group); Tanja Gesell (Center for Integrative Bioinformatics Vienna, University of Vienna); Rolf Backofen (Bioinformatics Group, University of Freiburg); Jan Gorodkin (Center for non-coding RNA in Technology and Health, University of Copenhagen); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The function of non-coding RNA genes largely depends on their<br>secondary structure and the interaction with other molecules. Thus, an<br>accurate prediction of the secondary structure and RNA-RNA interaction<br>is essential to understand biological roles and pathways associated<br>with a specific RNA gene. The RNA structure is constrained by<br>thermodynamic energies and evolutionary conservation patterns. Here,<br>both information sources are unified in one scoring scheme by the PET<br>(Probabilistic Evolutionary and Thermodynamic) model for analyzing<br>multiple aligned RNA sequences for common RNA structure and for RNA<br>interaction sites. We present the predictor PETcofold, an extension of<br>PETfold for RNA-RNA interactions, and its application on bacterial<br>small RNAs, and snoRNAs and their targets.  Additionally, we provide a<br>web server with direct access to annotated RNA alignments to overcome<br>the main drawback of the current methods that the prediction accuracy<br>is largely dependent from the quality of the input alignment(s).</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/93/21071.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='L18'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L18</strong></td>
 </tr> <tr> 
  <td><b>Identifying Conservative Subnetworks by NASCENT</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Daniel Banky <i>Eotvos University</i><br></td>
 </tr>
 <tr> 
  <td>Vince Grolmusz (Eotvos University) Daniel  Banky (Eotvos University, Computer Science); Rafael Ordog (Eotvos University, Computer Science); Balazs Szerencsi (Eotvos University, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: The NASCENT protein-protein interaction (PPI) generating tool (http://nascent.pitgroup.org) was originally designed to build PPI networks for non-model organisms, using the networks of evolutionary close model organisms. The second version of NASCENT uses automatical search in sequential databases for the best aligned  protein correspondence for the inter-species PPI mapping. <br><br>In the present work we demonstrate a completely different use of NASCENT: by predicting a PPI network „U” for a model organism (i.e., with a known PPI network „V”) from another model organism (i.e., with another known PPI network „W”), one can compare the the measured „V” and the predicted „U” networks, and find conservative subnetworks, inherited from „W” in „V” and „U”.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L19'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L19</strong></td>
 </tr> <tr> 
  <td><b>Protein Interaction Network Generated by NASCENT Using  Geometric Hashing</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Balazs Szerencsi <i>Eotvos Univeristy</i><br></td>
 </tr>
 <tr> 
  <td>Vince Grolmusz (Eotvos Univeristy) Rafael Ordog (Eotvos University, Computer Science); Daniel Banky (Eotvos University, Computer Science); Vince Grolmusz (Eotvos University, Computer Science); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: NASCENT, our automatic protein-protein interaction (furthermore referred to as PPI) network generation tool for non-model organisms, has been introduced previously. NASCENT<br> constructs the protein-protein physical interaction network for any chosen non-model species<br> using the available PPI network of a given well-researched model species. The mapping of<br> physical interactions is performed based on identical gene names of expressed proteins and/or sequence alignment in the<br> two organisms<br>. In our poster we present a new approach to predict PPI networks based on geometric hashing of crystallographically determined spatial protein structures. <br><br>NASCENT’s calculations are based on protein interaction data of model organisms, retrieved<br> from PPI databases (e.g. IntAct, MINT, DIP). Available protein information<br> corresponding to the source and target species were picked from the SwissProt and/or TrEMBL database. The three-dimensional structure of proteins are retrieved from RCSB Protein Data Bank. We use a correspondence between the UniProtKB sequences and the PDB data files. The entire Protein Data Bank is converted into a geometric hash that can identify the similar molecules. Above a similarity threshold proteins are considered having the same function in all species. We use this relation to refine our previously used and presented PPI prediction methods.<br><br>To process readily accessible<br> biological data sets from the Internet, our software is also capable of assigning frequently used<br> values to proteins of the predicted PPI graph, such as subspecies name, amino acid sequence,<br> and references to several external databases (e.g. PubMed, RefSeq).</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L20'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L20</strong></td>
 </tr> <tr> 
  <td><b>A novel ultra-fast comparison method for known and potential ligand-binding sites of proteins</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Kentaro Tomii <i>National Institute of Advanced Industrial Science and Technology (AIST)</i></td>
 </tr>
 <tr> 
  <td>  Jun-ichi Ito (University of Tokyo, Graduate School of Frontier Science); Yasuo Tabei (ERATO, Minato Discrete Structure Manipulation System Project); Kana Shimizu (National Institute of Advanced Industrial Science and Technology (AIST), Computational Biology Research Center (CBRC)); Koji Tsuda (National Institute of Advanced Industrial Science and Technology (AIST), Computational Biology Research Center (CBRC)); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Most proteins exhibit their functions through interactions with other molecules (so-called ligands), and their functions can be characterized by ligand-binding sites. A large number of ligand-binding sites can be found in the Protein Data Bank (PDB) now. In addition to these known sites, several computational methods are available for predicting potential ligand-binding sites. Exhaustive pairwise comparison of such known and potential ligand-binding sites is computationally demanding, but useful in elucidating the biological functions and evolutionary relationships of proteins.<br>We propose a tremendously fast alignment-free method for comparing huge number of ligand-binding sites, in which binding sites are mapped as vectors onto high-dimensional feature spaces based on their physicochemical and geometrical properties. Once binding sites are converted to bit strings, called structural sketches, which is obtained by random projections of feature vectors, a multiple sorting method is applied to the enumeration of all similar pairs in terms of the Hamming distance. With 1.2 million known and potential ligand-binding sites, the proposed method found 88 million similar binding site pairs within approximately 30 h on a single-core CPU machine. We report several remarkable analogous pairs of ligand-binding sites shared across distinct proteins. In particular, we succeeded in finding highly plausible functions of several potential ligand-binding sites via strong structural analogies. These suggest that our method would be useful for finding new structural motifs relevant to protein functions and for screening of target proteins in drug discovery.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L21'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L21</strong></td>
 </tr> <tr> 
  <td><b>New methods for finding common insertion sites and co-occurring common insertion sites in transposon- and virus-based genetic screens</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Tracy Bergemann <i>University of Minnesota</i></td>
 </tr>
 <tr> 
  <td>  Tracy Bergemann (University of Minnesota, Division of Biostatistics); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Insertional mutagenesis screens identify genes that drive tumorigenesis. Cancer genes reside near common insertion sites (CIS) or co-occurring common insertions (CCI) with transposon or retroviral insertions in combinations of regions. We describe new methods to detect CIS and CCI, the Poisson Regression Insertion Model, and show improvement upon previous methods.</td>
 </tr> <tr> <td><br/><b>Long Abstract</b>: <a href='http://www.iscb.org/uploaded/css/86/19956.pdf'>Click Here</a></td>
 </tr>
 <tr> 
  <td><b><br/></b></td>
 </tr>
</table>
<a name='L22'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L22</strong></td>
 </tr> <tr> 
  <td><b>Analysis of the interactions in the PIDDosome</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Hyun Ho Park <i>Yeungnam University</i></td>
 </tr>
 <tr> 
  <td>  Ju-Hong  Jeon (Seoul National University, Physiology); Dongseop Kwon (Myongji University, Computer Engineering); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Caspase-2 is critical for genotoxic stress induced apoptosis and is activated by formation of the PIDDosome, an oligomeric caspase-2 activating complex. The PIDDosome comprises three protein components, PIDD, RAIDD and caspase-2. RAIDD contains both a death domain (DD) and a caspase recruitment domain (CARD). It acts as the bridge to recruit PIDD using the DD: DD interaction and to recruit caspase-2 via the CARD: CARD interaction. Here we report biochemical characterization and in vitro reconstitution of the core interactions in the PIDDosome. We show that RAIDD CARD and RAIDD DD interact with their binding partners, caspase-2 CARD and PIDD DD, respectively. However, full-length RAIDD fails to interact with either caspase-2 CARD or PIDD DD under a physiological buffer condition. We reveal that this lack of interaction of full-length RAIDD is not due to its tendency to aggregate under the physiological buffer condition, as decreasing full-length RAIDD aggregation using high salt or high pH is not able to promote complex formation. Instead, full-length RAIDD forms both binary and ternary complexes under a low salt condition. Successful in vitro reconstitution of the ternary complex provides a basis for further structural studies of the PIDDosome</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<a name='L23'></a>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>   <td class='subheading1'><strong>Poster L23</strong></td>
 </tr> <tr> 
  <td><b>An alternative approach to protein docking: Complexes via homology modeling</b><br/><br/></td>
 </tr>
 <tr> 
 <td>Ozlem Tastan Bishop <i>Rhodes University</i></td>
 </tr>
 <tr> 
  <td>  Matthys Kroon (Rhodes University, Rhodes University Bioinformatics (RUBi), Department of Biochemistry, Microbiology, Biotechnology); </td>
 </tr>
<tr><td>&nbsp;</td></tr> <tr> 
  <td><b>Short Abstract</b>: Reliable prediction of protein complexes is not easy and current protein-protein docking methods are computationally expensive. Homology modeling is a promising new approach for predicting protein complex structures. This study develops and evaluates large scale calculation of 3D structures of protein complexes by homology modeling. As a case study, Clan CA, family C1 cysteine proteases and their protein inhibitors were used. Cysteine proteases play numerous roles in human and parasitic metabolism, and some of them have been identified as drug targets. The study was performed in two parts. The purpose of the first part (evaluation set) was to evaluate the modeling accuracy. Nine crystal structure complexes were selected, and each of these complexes was modeled in a variety of ways using data from the remaining eight structures. In total 1325 homology models of known complexes were rebuilt, allowing an analysis of the factors influencing the accuracy of the models. In the second part (study set), the findings of the evaluation set were used to select appropriate templates to model novel cysteine protease-inhibitor complexes from human and malaria parasites Plasmodium falciparum and Plasmodium vivax. The energy functions of the models were evaluated and, taking into account the correlations found in the first part, it follows that the models are of high accuracy.</td>
 </tr> <tr><td> &nbsp;</td></tr></table>
<h3>Accepted Posters</h3>
<div class="style1">
<a href="posterlist.html#prep">Preparing your Poster - Information and Poster Size</a><br />
<!--<a href="#video">Poster presentation video taped for posting to the SciVee website Information</a> -->
<a href="posterlist.html#schedule">Poster Schedule</a><br />

<a href="posterlist.html#print">Vienna Poster Printing Services</a><br />


<a href="posterlist.html#cat">Poster Categories</a><br />
<a href="posterlist.html#search1">Search for a Poster</a><br />

<br />
</div>




<div align=left class=style1>
<br><a name="prep"></a>
<strong>Attention Poster Authors</strong>:

The ideal poster size should be max. 1.30 m (130 cm) high x 0.90 m (90 cm) wide.
Fasteners (Velcro / double sided tape) will be provided at the site, please
<strong>DO NOT</strong> bring tape, tacks or pins.
View a diagram of the the poster board <a href="../../../ismbeccb2011-submission-details/856-ismbeccb-2011-poster-board.html">here</a>

<br><br>
<hr>

<!--
<a name="video"></a>
<div class="style1">
<strong>Poster presentation video taped for posting to the SciVee website Information</strong><br />
A special opportunity to have your poster presentation video taped for posting to the SciVee website is now available on a first come, first serve basis.
Please view the letter (<a target="_blank" href="./downloads/ISMBECCB_posterinvite.doc">Poster invite letter</a>) for full details and the email address to sign up.
</div> <br>
-->
<br><br>
<a name="schedule"></a>
<div class="subheading1"><strong style="font-size: 16px;">Posters Display Schedule:</strong></div>
<br>
<div class="style1">



 <span style="font-size: 14px;"><strong><u>Odd</u></strong> Numbered posters:</span><br>
 <ul>
      <li style="line-height: 1.5em;"> Set-up timeframe:			Sunday, July 17, 7:30 a.m. - 10:00 a.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations:	Monday, July 18, 12:40 p.m. - 2:30 p.m.</li>
	  <li style="line-height: 1.5em;">Removal timeframe: Monday, July 18, 2:30 p.m. - 3:30 p.m.*</li>

 </ul>

<span style="font-size: 14px;"><strong><u>Even</u></strong> Numbered posters:</span><br>
<ul>
  <li style="line-height: 1.5em;"> Set-up timeframe: Monday, July 18, 3:30 p.m. - 4:30 p.m.</li>
  <li style="line-height: 1.5em;"> Author poster presentations: Tuesday, July 19, 12:40 p.m. - 2:30 p.m.</li>
  <li style="line-height: 1.5em;">Removal timeframe:	 Tuesday, July 19, 2:30 p.m. - 4:00 p.m.*</li>
</ul>
* Posters that are not removed by the designated time may be taken down by the organizers and discarded. Please be sure to remove your poster within the stated timeframe.

<br><br>

<div class="subheading1"><strong style="font-size: 16px;">
Delegate Posters Viewing Schedule</strong></div>
<br/>
<p style="line-height: 1.5em;">
    <strong style="font-size: 14px;">Odd Numbered posters:</strong><br>

On display Sunday, July 17, 10:00 a.m. through Monday, June 18, 2:30 p.m.<br>
Author presentations will take place Monday, July 18: 12:40 p.m.-2:30 p.m.<br>
</p>
<p style="line-height: 1.5em;">
<strong style="font-size: 14px;">
Even Numbered posters:</strong><br>
On display Monday, July 18, 4:30 p.m. through Tuesday, June 19, 2:30 p.m.<br>
Author presentations will take place Tuesday, July 19: 12:40 p.m.-2:30 p.m<br>
</p>

</div>
<br><br>
<hr>
<a name="print"></a>
<br>

<div class="subheading1"><strong style="font-size: 16px;">
Want to print a poster in Vienna - try these options:</strong></div>
<br>
<p>
    
   <a target="_blank" href="http://www.repacopy.at/index.php?id=334">Repacopy</a>- next to the congress venue  <a target="_blank" href="http://www.repacopy.at/index.php?id=334">link</a>   <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/95vgq/17-repa-copy-wien-izd-tower/karte/">[MAP]            </a> </li>
   <br><br>
   Also at Karlsplatz is in the Ring Center, K&auml;rntner Str. 42, <a target="_blank" href="http://www.repacopy.at/index.php?id=335">link</a> <a target="_blank" href="http://www.herold.at/gelbe-seiten/wien/9pZZt/18-repa-copy-wien-ringcenter/karte/">[MAP]</a>
           </p>
           <br>
   <p>
       If you need your poster on a thicker material, you may also use a plotter service next to Karlsplatz: <a href="http://schiessling.at/portfolio/">http://schiessling.at/portfolio/</a>
 </p>



<br>

<hr>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>View Posters By Category</b></td>
 </tr>
</table>
<a name="cat"></a>
<ul>
<li class="style1"><a href="posterlist950c950c.html?cat=A">A) Bioinformatics of Health and Disease</a></li>
<li class="style1"><a href="posterlist3a9a3a9a.html?cat=B">B) Biophysics </a></li>
<li class="style1"><a href="posterlistf77af77a.html?cat=C">C) Chemical and Pharmaceutical Informatics</a></li>
<li class="style1"><a href="posterlist2f7c2f7c.html?cat=D">D) Comparative Genomics</a></li>
<li class="style1"><a href="posterlist16851685.html?cat=E">E) Databases</a></li>
<li class="style1"><a href="posterlist41584158.html?cat=F">F) Evolution</a></li>
<li class="style1"><a href="posterlist42384238.html?cat=G">G) Functional Genomics</a></li>
<li class="style1"><a href="posterlistbb6abb6a.html?cat=H">H) Gene Prediction</a></li>
<li class="style1"><a href="posterlist711f711f.html?cat=I">I) Genome Annotation</a></li>
<li class="style1"><a href="posterlist39b439b4.html?cat=J">J) Genomics</a></li>
<li class="style1"><a href="posterlist7fcd7fcd.html?cat=K">K) Institutional Research</a></li>
<li class="style1"><a href="posterlistfdc3fdc3.html?cat=L">L) Interactions</a></li>
<li class="style1"><a href="posterlistb2c2b2c2.html?cat=M">M) Machine Learning</a></li>
<li class="style1"><a href="posterlista34ca34c.html?cat=N">N) Microarrays</a></li>
<li class="style1"><a href="posterlist53f753f7.html?cat=O">O) Neuroscience</a></li>
<li class="style1"><a href="posterlistafd9afd9.html?cat=P">P) Ontologies</a></li>
<li class="style1"><a href="posterlistedceedce.html?cat=Q">Q) Population Genetics and Variation</a></li>
<li class="style1"><a href="posterlist27ad27ad.html?cat=R">R) Proteomics</a></li>
<li class="style1"><a href="posterlist5c375c37.html?cat=S">S) Regulation</a></li>
<li class="style1"><a href="posterliste35de35d.html?cat=T">T) Reviews</a></li>
<li class="style1"><a href="posterlistd227d227.html?cat=U">U) Sequence analysis</a></li>
<li class="style1"><a href="posterlista946a946.html?cat=V">V) Structural Genomics</a></li>
<li class="style1"><a href="posterlistdc81dc81.html?cat=W">W) Structure and Function Prediction</a></li>
<li class="style1"><a href="posterlist8ba38ba3.html?cat=X">X) Systems Biology and Networks</a></li>
<li class="style1"><a href="posterlist718c718c.html?cat=Y">Y) Text Mining</a></li>
<li class="style1"><a href="posterlist80018001.html?cat=Z">Z) Other</a></li>
</ul>
<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td height="25" style="border-bottom: 1px dashed #B2B2B2;"><b>Search Posters:</b></td>
 </tr>
</table>

<form action="https://www.iscb.org/cms_addon/conferences/ismbeccb2011/posterlist.php" method="POST">
<input type="hidden" name="search" value="1">

<table width="100%" border="0" cellspacing="0" cellpadding="0" class="style1">
 <tr>
  <td>Poster Number</td>
  <td>Matches</td>
  <td><input type="text" style="width: 300px;" name="slot"></td>
 </tr>
 <tr>
  <td>Last Name</td>
  <td><select name="match_ln"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="last_name"></td>
 </tr>
 <tr>
  <td>Co-Authors</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="co_auth"></td>
 </tr>
 <tr>
  <td>Title</td>
  <td><select name="match_pt"><option value="match">Matches</option><option value="contain">Contains</option></select></td>
  <td><input type="text" style="width: 300px;" name="pres_title"></td>
 </tr>
 <tr>
  <td>Abstract</td>
  <td>Contains</td>
  <td><input type="text" style="width: 300px;" name="abstr_text"></td>
 </tr>
 <tr><td colspan=3 align="top"><input type="submit" value="search"></td></tr>

</table>
<br>
<a name="search1"></a>
<br>
</form>

<br />
<br />
<br />
<br />
  <a href="#up" style="font-size:9px">&uarr; TOP</a><br /><br />
<br />
<br />




