
  
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb" >

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/technologytrack.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:50:15 GMT -->
<!-- Added by HTTrack -->
<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/technologytrack.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:55 GMT -->
<meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->
<head>
 
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="robots" content="index, follow" />
  <meta name="keywords" content="ISMB/ECCB 2011, ismbeccb2011, ismb, eccb, bioinformatics, chemoinformatics, cheminformatics, biocomputing, algorithm" />
   <meta name="description" content="ISCB -  International Society for Computational Biology" />

  <title>ISMB/ECCB 2011 Technology Track</title>
  <link href="../../../templates/ismb2011/favicon.ico" rel="shortcut icon" type="image/x-icon" />
  <link rel="stylesheet" href="../../../media/system/css/modal.css" type="text/css" />
  <script type="text/javascript" src="../../../includes/js/joomla.javascript.html"></script>
  <script type="text/javascript" src="../../../media/system/js/mootools.js"></script>
  <script type="text/javascript" src="../../../media/system/js/caption.js"></script>
  <script type="text/javascript" src="../../../media/system/js/modal.js"></script>

  <script type="text/javascript" src="../../../modules/mod_accordionmenu/assets/accordionmenu.js"></script>
  <script type="text/javascript">
		window.addEvent('domready', function(){ var JTooltips = new Tips($$('.hasTip'), { maxTitleChars: 50, fixed: false}); });
		window.addEvent('domready', function() {

			SqueezeBox.initialize({});

			$$('a.modal').each(function(el) {
				el.addEvent('click', function(e) {
					new Event(e).stop();
					SqueezeBox.fromElement(el);
				});
			});
		});
  </script>

<link rel="stylesheet" href="../../../templates/system/css/system.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/system/css/general.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/template.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white.css" type="text/css" />
<link rel="stylesheet" href="../../../templates/ismb2011/css/white_bg.css" type="text/css" />
<!--[if IE 7]>
<link href="/templates/ismb2011/css/ie7only.css" rel="stylesheet" type="text/css" />
<![endif]-->
<!--[if lte IE 6]>
<link href="/templates/ismb2011/css/ieonly.css" rel="stylesheet" type="text/css" />
<![endif]-->
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-24246804-1']);
  _gaq.push(['_setDomainName', '.iscb.org']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body id="page_bg" class="color_white bg_white width_small">
<a name="up" id="up"></a>
<div class="center" align="center">
	<div id="wrapper">
		<div id="wrapper_r">
			<div id="header">
				<div id="header_l">
					<div id="header_r">
						<div id="logo">

                                                    <a id="ismbeccb2011" href="../../../ismbeccb2011.html" title="ISMB2011 Home"></a>
                                                    <a id="iscb" href="../../../index.html" title="ISCB Home"></a>
                                                </div>

					</div>
				</div>
			</div>
			<div id="search">
				<form action="http://www.iscb.org/cms_addon/conferences/ismbeccb2011/index.php" method="post">
	<div class="search">

		<input name="searchword" id="mod_search_searchword" maxlength="20" alt="Search" class="inputbox" type="text" size="20" value="search..."  onblur="if(this.value=='') this.value='search...';" onfocus="if(this.value=='search...') this.value='';" />	</div>
	<input type="hidden" name="task"   value="search" />
	<input type="hidden" name="option" value="com_search" />
	<input type="hidden" name="Itemid" value="677" />
</form>

			</div>

			<div id="tabarea">


						<div id="tabmenu">


						<table  class="pill">
							<tr>

								<td class="pill_m">
								<div id="pillmenu">
									<ul id="mainlevel"><li><a href="http://www.iscb.org/iscb-membership" class="mainlevel" >JOIN ISCB</a></li><li><a href="../../../ismbeccb2011-keydates.html" class="mainlevel" >KEY DATES</a></li><li><a href="../../../ismbeccb2011-news.html" class="mainlevel" >NEWS</a></li><li><a href="../../../ismbeccb2011-registration.html" class="mainlevel" >REGISTER</a></li></ul>
								</div>
								</td>


							</tr>
							</table>
						</div>
 			</div>



			<div class="clr"></div>

			<div id="whitebox">

				<div id="whitebox_t">

					<div id="whitebox_tl">
						<div id="whitebox_tr"></div>

					</div>
				</div>

				<div id="whitebox_m">
					<div id="sidenotes" class="sidenotes">
    	            	19th Annual International Conference on<br /> Intelligent Systems for Molecular Biology and<br /> 10th European Conference on Computational Biology
	                </div>
					<div id="area">



						<div id="leftcolumn">


															<div class="module">
			<div>
				<div>
					<div>

<script type="text/javascript">
	window.addEvent('domready', function(){
	accordionMenu(
		"accordionMenu", // Menu Tag ID (this id is assigned to menu parent ul tag)
		"", //path to image used to expand menu item
		"", //path to image used to collapse menu item
		{duration:400, transition:Fx.Transitions.Quart.easeOut}, // (optional argument) custom accordion options to override defaults, use null if you want to set next argument without changing this
		200, // (optional argument) hover delay in milliseconds over "parent menu item" to open its sublevels, default is 200
		1 // (optional argument) enable/disable opening of submenus on hovering
	);

	});
</script>

<ul class="menuacc" id="accordionMenu"><li id="current" class="active item677"><a href="../../../ismbeccb2011.html"><span>HOME</span></a></li><li class="parent item739"><a href="../../../ismbeccb2011-general-info.html"><span>GENERAL INFO</span></a><ul><li class="item757"><a href="../../../ismbeccb2011-general-info/about-ismb.html"><span>About ISMB</span></a></li><li class="item902"><a href="../../../ismbeccb2011-general-info/awards.html"><span>Awards</span></a></li><li class="item825"><a href="../../../ismbeccb2011-general-info/air-travel.html"><span>Air Travel</span></a></li><li class="item1077"><a href="../../../ismbeccb2011-general-info/ismbeccb2011-airport-train.html"><span>Airport Train</span></a></li><li class="item1074"><a href="../../../ismbeccb2011-general-info/child-care.html"><span>Child Care</span></a></li><li class="item906"><a href="../../../ismbeccb2011-general-info/faq.html"><span>FAQ</span></a></li><li class="item907"><a href="../../../ismbeccb2011-general-info/housing.html"><span>Housing</span></a></li><li class="item1076"><a href="../../../ismbeccb2011-general-info/official-journals.html"><span>Official Journals</span></a></li><li class="item826"><a href="../../../ismbeccb2011-general-info/promotion.html"><span>Promotion</span></a></li><li class="item1086"><a href="../../../ismbeccb2011-general-info/ismbeccb-2011-speaker-information.html"><span>Speaker Information</span></a></li><li class="item851"><a href="../../../ismbeccb2011-general-info/vienna-public-transit-pass.html"><span>Vienna Public Transit Pass</span></a></li><li class="item938"><a href="../../../ismbeccb2011-general-info/volunteers.html"><span>Volunteers</span></a></li></ul></li><li class="parent item690"><a href="../../../ismbeccb2011-program.html"><span>PROGRAM</span></a><ul><li class="item1040"><a href="schedule/alldays.html"><span>Schedule (Detailed)</span></a></li><li class="item704"><a href="../../../ismbeccb2011-program/schedule-brief.html"><span>Schedule Overview</span></a></li><li class="item1041"><a href="artscience.html"><span>Art &amp; Science</span></a></li><li class="item1087"><a href="../../../ismbeccb2011-program/birds-of-a-feather.html"><span>Birds of a Feather</span></a></li><li class="item700"><a href="../../../ismbeccb2011-program/keynotes.html"><span>Keynotes</span></a></li><li class="item948"><a href="highlights.html"><span>Highlights</span></a></li><li class="item949"><a href="latebreakingresearch.html"><span>Late Breaking Research</span></a></li><li class="item1037"><a href="posterlist.html"><span>Posters</span></a></li><li class="item1043"><a href="oralposters.html"><span>Oral Poster Presentations</span></a></li><li class="item936"><a href="proceedingsBrief.html"><span>Proceedings</span></a></li><li class="item730"><a href="../../../ismbeccb2011-program/satellite-meetings.html"><span>SIGs &amp; Satellite Meetings</span></a></li><li class="item728"><a href="../../../ismbeccb2011-program/social-events.html"><span>Social Events</span></a></li><li class="item729"><a href="../../../ismbeccb2011-program/special-sessions.html"><span>Special Sessions</span></a></li><li class="item1082"><a href="../../../ismbeccb2011-program/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item726"><a href="../../../ismbeccb2011-program/tutorials.html"><span>Tutorials</span></a></li><li class="item736"><a href="../../../ismbeccb2011-program/plos-postcards.html"><span>PLoS Postcards</span></a></li><li class="item1039"><a href="technologytrack.html"><span>Technology Track</span></a></li><li class="item909"><a href="../../../ismbeccb2011-program/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item692"><a href="../../../ismbeccb2011-exhibitors-sponsors.html"><span>SPONSORS</span></a><ul><li class="item698"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsors.html"><span>Sponsors</span></a></li><li class="item705"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsorshipopportunities.html"><span>Sponsorship Opportunities</span></a></li><li class="item706"><a href="../../../ismbeccb2011-exhibitors-sponsors/past-sponsors.html"><span>Past Sponsors</span></a></li><li class="item707"><a href="sponsor_signup.html"><span>Sponsor/Exhibitor Sign up</span></a></li><li class="item716"><a href="../../../ismbeccb2011-exhibitors-sponsors/industry-poster.html"><span>Industry Poster</span></a></li><li class="item717"><a href="../../../ismbeccb2011-exhibitors-sponsors/additional-sponsor-opps.html"><span>Other Sponsor Opps</span></a></li><li class="item718"><a href="../../../ismbeccb2011-exhibitors-sponsors/tech-track-presentations.html"><span>Tech Track</span></a></li><li class="item719"><a href="../../../ismbeccb2011-exhibitors-sponsors/sponsor-benefits.html"><span>Sponsor Benefits</span></a></li><li class="item720"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-package.html"><span>Exhibitor Package</span></a></li><li class="item727"><a href="../../../ismbeccb2011-exhibitors-sponsors/floorplan.html"><span>Floor Plan</span></a></li><li class="item738"><a href="../../../ismbeccb2011-exhibitors-sponsors/exhibitor-info.html"><span>Exhibitor Info</span></a></li></ul></li><li class="parent item693"><a href="../../../ismbeccb2011-submission-details.html"><span>SUBMISSION</span></a><ul><li class="item714"><a href="../../../ismbeccb2011-submission-details/artandscienceexhibition.html"><span>Art and Science</span></a></li><li class="item1038"><a href="birds_signup.html"><span>Birds of a Feather</span></a></li><li class="item713"><a href="../../../ismbeccb2011-submission-details/highlights.html"><span>Highlights</span></a></li><li class="item724"><a href="../../../ismbeccb2011-submission-details/late-breaking-research.html"><span>Late Breaking Research</span></a></li><li class="item722"><a href="../../../ismbeccb2011-submission-details/call-for-posters.html"><span>Posters</span></a></li><li class="item734"><a href="../../../ismbeccb2011-submission-details/late-posters.html"><span>Posters - Late</span></a></li><li class="item711"><a href="../../../ismbeccb2011-submission-details/call-for-proceedings.html"><span>Proceedings</span></a></li><li class="item712"><a href="../../../ismbeccb2011-submission-details/special-interest-groups.html"><span>Special Interest Groups</span></a></li><li class="item699"><a href="../../../ismbeccb2011-submission-details/call-for-special-sessions.html"><span>Special Sessions</span></a></li><li class="item731"><a href="../../../ismbeccb2011-submission-details/student-council-symposium.html"><span>Student Council Symposium</span></a></li><li class="item710"><a href="../../../ismbeccb2011-submission-details/technology-track.html"><span>Technology Track</span></a></li><li class="item735"><a href="../../../ismbeccb2011-submission-details/travel-fellowship.html"><span>Travel Fellowship</span></a></li><li class="item715"><a href="../../../ismbeccb2011-submission-details/call-for-tutorials.html"><span>Tutorials</span></a></li><li class="item721"><a href="../../../ismbeccb2011-submission-details/workshops.html"><span>Workshops</span></a></li></ul></li><li class="parent item694"><a href="../../../ismbeccb2011-committee-organizing.html"><span>COMMITTEES</span></a><ul><li class="item702"><a href="../../../ismbeccb2011-committee-organizing/conference-chairs.html"><span>Conference Chairs</span></a></li><li class="item725"><a href="../../../ismbeccb2011-committee-organizing/committees.html"><span>Committees</span></a></li></ul></li><li class="item695"><a href="../../../ismbeccb2011-contacts.html"><span>CONTACT</span></a></li><li class="item696"><a href="../../../ismbeccb2011-promotion.html"><span>PROMOTION</span></a></li><li class="item697"><a href="presspass.html"><span>PRESSPASS</span></a></li></ul>					</div>

				</div>
			</div>
		</div>

												<!-- <img id="sc" src="/templates/ismb2011/images/sclogo.jpg"> -->
						<div class="miniAd" style="margin-top: -10px;">
							<a target=_blank href="http://symposium.iscbsc.org/">

							<img id="sig" width="148px" src="../../images/stories/ismbeccb2011/misc/scs7button.png">
							</a>
                            <a style="margin-top: 5px;" target=_blank href="../../../ismbeccb2011-program/satellite-meetings.html">

							Satellite Meetings

							</a>
						</div>
						</div>



												<div id="maincolumn">

									<div id="nopad">

<table class="contentpaneopen">



<tr>
<td valign="top">




<h3>Technology Track Presentation Schedule</h3> <div class=style1><div class='subheading2'><a name="TT01" >TT01</a><span style='float: right;'>Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Ensembl Genome Browser: Harnessing the Next Generation Sequencing Wave</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Michael Schuster, EMBL-European Bioinformatics Institute<br><br>
    <strong>Abstract:</strong><br>Recent developments of Ensembl include visualisation of custom next-generation sequencing data and their seamless integration into genome annotation provided by the project. A Variant Effect Predictor allows classification of genetic variants by their overlap with genome annotation. Annotation of regulatory elements has been extended with cell type-specific data sets.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22432.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT02" >TT02</a><span style='float: right;'>Sunday, July 17: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>PubAnatomy: A Rich Internet Application for Integrated Literature and Data Exploration</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Fan Meng and  Barbara Mirel, University of Michigan<br><br>
    <strong>Abstract:</strong><br>PubAnatomy is a novel Flash-based literature exploration solution that provides  rich interactivity and bi-directional data communication with other Flash RIA in different browser windows  to facilitate the integrated exploration of neurobiology literature in relevant biological contexts such as brain structures, pathophysiological processes, gene expression levels and protein-protein interactions<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22482.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT03" >TT03</a><span style='float: right;'>Sunday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>UCSC Genome Browser:  Visualization tools for high-throughput sequencing</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Rachel Harte, University of California, Santa Cruz<br><br>
    <strong>Abstract:</strong><br>The UCSC Genome Browser and Ensembl offer two presentations featuring Web-based tools for high-throughput sequencing visualization.  The Browser allows interactive viewing of BAM files as Custom Tracks.  The new Personal Genome SNP format displays variants and the Session tool provides an easy way to share results with others.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22419.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT04" >TT04</a><span style='float: right;'>Sunday, July 17: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>GenomeSpace</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Michael Reich, Broad Institute<br><br>
    <strong>Abstract:</strong><br>GenomeSpace is a platform that supports an open community of genomics analysis tools. Using GenomeSpace, researchers can construct analyses requiring communication and data transfer between diverse tools such as Web applications and client-side tools. GenomeSpace supports cloud-based storage, handles file format compatibility issues automatically, and enables reproducibility of multi-tool analyses.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22475.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT05" >TT05</a><span style='float: right;'>Sunday, July 17: 11:45 a.m. - 12:40 p.m.</span></div><strong><em>Enhancing researcher productivity through integrated Bio-medical workflow solutions on SciVerse Platform</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Vishal Gupta,  Adriaan Klinkenberg, Elsevier<br><br>
    <strong>Abstract:</strong><br><p>
SciVerse is a platform for trusted scientific content and innovative applications that enhance information search and discovery of researchers by adding semantic search functionality, text and data mining annotation to visualize data and relationships, or integrating data with ontologies and repositories. Using the SciVerse framework, researchers and external developers can build customized workflow solutions that can appear alongside full text articles, abstracts and search results within the SciVerse suite. 
</p><p>
This session explores the SciVerse platform that enables collaboration with the scientific community and demonstrates two workflow solutions- NCBI Genome Viewer and Health-mash. 
</p>
<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/tt05.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT06" >TT06</a><span style='float: right;'>Sunday, July 17: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Bioinformatics@school</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Hienke Sminia, Netherlands Bioinformatics Centre<br><br>
    <strong>Abstract:</strong><br>- Bringing bioinformatics to the classroom - <br>Bioinformatics is virtually unknown to most high school pupils. Since  they are potentially tomorrow's bioinformaticians, the Netherlands  Bioinformatics Centre (NBIC) and the Centre for Molecular and  Biomolecular Informatics (CMBI) are bringing bioinformatics to the  classroom using hands-on student workshops and teacher trainings.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22540.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT07" >TT07</a><span style='float: right;'>Sunday, July 17: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Supporting Inter-connected Biomedical Open-source Software</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Zhong Li, Columbia University / Broad Institute<br><br>
    <strong>Abstract:</strong><br>One of the unmet needs for open source biomedical software is support. The Molecular Analysis Tools Knowledge Center (MATKC), sponsored by NCI, offers support, expertise, and knowledge resources on selected tools in molecular data management and analysis (caArray, caIntegrator, geWorkbench, and GenePattern). A vibrant online community has been established.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/21092.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT08" >TT08</a><span style='float: right;'>Sunday, July 17: 2:30 p.m. - 3:25 p.m.</span></div><strong><em>MONGKIE: Modular Network Generation and Visualization Platform with Knowledge Integration Environment</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Sanghyuk Lee, Korean Bioinformation Center<br><br>
    <strong>Abstract:</strong><br>MONGKIE is an integrated network visualization platform which allows us to explore and analyze interconnected biological data in an interactive manner with knowledge integration environment providing functional modules such as Import/Export Network, Expression Overlay, Network Clustering/Ranking, Pathway Integration/Visualization, Gene Set Enrichment Analysis(GO/Pathway) and Analysis Pipelines consisting of other modules.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22431.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT09" >TT09</a><span style='float: right;'>Sunday, July 17: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Cloud Computing for Computational Biology</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Attila Berces, Omixon<br><br>
    <strong>Abstract:</strong><br>We demonstrate some free utilities that help biologist to take advantage of cloud computing. In addition we demonstrate next generation sequencing data analysis on the Omixon portal starting with short reads and ending with annotated variants and aligned reads in a genome browser. <br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22462.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT10" >TT10</a><span style='float: right;'>Sunday, July 17: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Accurate Genome Variant Detection</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Attila Berces, Omixon<br><br>
    <strong>Abstract:</strong><br>We demonstrate Omixon Variant Toolkit, an easy-TO-USE NEXT generation sequencing DATA analysis pipeline especially designed FOR the most accurate genomic variant detection. We shall demonstrate several biologically relevant variants FOUND only BY this tool but NOT BY the most popular tools AND were subsequently validate BY Sanger sequencing. <br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22462.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT11" >TT11</a><span style='float: right;'>Sunday, July 17: 3:30 p.m. - 4:25 p.m.</span></div><strong><em>Efficient Graph Based Assembly of Short-Read Sequences on a Hybrid Core Architecture</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>George Vacek, Convey Computer<br><br>
    <strong>Abstract:</strong><br>Convey’s Graph Constructor takes advantage of increased random-access memory bandwidth to dramatically reduce run time and memory usage of Velvet for short-read, de novo assembly.  Results will be presented for a variety of datasets.  Performance improvements facilitate multiple runs with varying parameters, ultimately resulting in higher quality assemblies.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22420.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT12" >TT12</a><span style='float: right;'>Sunday, July 17: 3:30 p.m. - 4:25 p.m.</span></div><strong><em>Functional Synergy Analysis of OMICs Data of Different Types in Cancer Research</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Yuri Nikolsky, Thomson Reuters<br><br>
    <strong>Abstract:</strong><br>We developed and applied several methods of functional data analysis for revealing of functional correlation between the datasets of different types (NGS, expression, epigenetics) regardless of gene content overlap between them. I will describe the basic techniques of pathway, network and interactome – based correlation and summarize the results of collaborative studies on cancer.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22468.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT13" >TT13</a><span style='float: right;'>Monday, July 18: 10:45 a.m. - 11:40 a.m.</span></div><strong><em>Analyzing Next Gen Sequencing Data with Pipeline Pilot</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Scott Markel,  Ruth Dunn, Accelrys<br><br>
    <strong>Abstract:</strong><br>Pipeline Pilot® provides a software environment where algorithms and data come together into a data-pipelining model that allows quick and customizable application integration, data analysis, and reporting of results.  Pipeline Pilot already integrates chemistry, text analytics, statistics, and sequence analysis.  Now next generation sequencing is supported.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22473.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT14" >TT14</a><span style='float: right;'>Monday, July 18: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>ProBiS: a web server for detection of protein binding sites</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Janez Konc, National Institute of Chemistry<br><br>
    <strong>Abstract:</strong><br>We present new features of the recently enhanced ProBiS 2.0 web server, and provide a brief tutorial of its usage on an example of protein function prediction.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/21087.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT15" >TT15</a><span style='float: right;'>Monday, July 18: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>RightField: The Semantic Annotation of Experimental Data using Spreadsheets</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Katy Wolstencroft, University of Manchester<br><br>
    <strong>Abstract:</strong><br>RightField is an open source application for embedding ontology annotation in spreadsheets. Cells can be restricted to ranges of allowed classes or instances from chosen ontologies and presented as a simple drop-down list in the spreadsheet. The result is “semantic annotation by stealth” and consistently annotated standards compliant data.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22426.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT16" >TT16</a><span style='float: right;'>Monday, July 18: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>UK PubMed Central: a database of life science research articles.</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Jo McEntyre, EMBL-EBI<br><br>
    <strong>Abstract:</strong><br>UK PubMed Central is a database of about two million full-text life science research articles and 25 million abstracts (from CiteXplore), overlaid with semantic enrichment, database links, and citation networks. It is free to use, and the open access articles are made available in full.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/21094.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT17" >TT17</a><span style='float: right;'>Monday, July 18: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Software for the data-driven researcher: Annotation of Next Generation Sequencing SNP Data using scientific workflows</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Paul Fisher, University of Manchester<br><br>
    <strong>Abstract:</strong><br>In this demonstration we will showcase the Taverna 2 workflow engine and Taverna Server, and illustrate how scientific workflows deployed on the cloud can provide a means of addressing the data analysis requirements for the annotation of SNP from Next Generation Sequencing.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22427.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT18" >TT18</a><span style='float: right;'>Monday, July 18: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>GeneCards: The Human Gene ReSearcher</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Irina Dalah, Weizmann Istitute of Science<br><br>
    <strong>Abstract:</strong><br>GeneCards (www.genecards.org) is a comprehensive, searchable compendium of annotative information about human genes, used worldwide for >15 years.<br>Our presentation will demonstrate how GeneCards facilitates life-science research, showcasing the powerful search, with sortable results shown as expandable, structured minicards, and with gene-sets exportable to GeneALaCart and GeneDecks, enabling systems-biology insights.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22433.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT19" >TT19</a><span style='float: right;'>Monday, July 18: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Integrating Heterogeneous Data using the Ontological Discovery Environment</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Elissa Chesler, The Jackson Laboratory<br><br>
    <strong>Abstract:</strong><br>The Ontological Discovery Environment (ODE) is an approach to large-scale data analysis of heterogeneous life science data using biclique enumeration and highlights the advantages of unifying discretized data into graph structures. We will present its work flow processes by analyzing a diverse set of use cases in complex trait disorders.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/21396.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT20" >TT20</a><span style='float: right;'>Monday, July 18: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>End user tools for data interoperability</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Christine Chichester, NBIC<br><br>
    <strong>Abstract:</strong><br>In the NBIC interoperability task force, we work towards a sustainable future for a large-scale, community editable store  of disambiguated scientific assertions, exemplifying a new  paradigm in life sciences data accumulation. We will demonstrate  some of the tools we have created as end user applications.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22541.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT21" >TT21</a><span style='float: right;'>Monday, July 18: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>PRIDE Inspector</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Florian Reisinger, EMBL-EBI<br><br>
    <strong>Abstract:</strong><br>Public data deposition and distribution has become a necessity in proteomics and is increasingly requested by journals. As a MS proteomics repository, PRIDE (http://www.ebi.ac.uk/pride) provides valuable tools to the community which support submission and visualization/validation of proteomics findings. PRIDE Inspector (http://code.google.com/p/pride-toolsuite/wiki/PRIDEInspector) is the latest addition to the PRIDE tool suite.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22414.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT22" >TT22</a><span style='float: right;'>Monday, July 18: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>SUPERFAMILY</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Julian Gough, University of Bristol<br><br>
    <strong>Abstract:</strong><br>We will be: giving an overview of SUPERFAMILY and a practical demonstration of how to use it; inviting feedback and requests from existing users and questions from potential users; and showcasing many powerful analysis features, currently under-exploited, such as: domain-centric GO, tree of sequenced life, evolutionary analyses and comparative genomics.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22362.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT23" >TT23</a><span style='float: right;'>Monday, July 18: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Enacting Taverna Workflows through Galaxy</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Kostas Karasavvas, NBIC - LUMC<br><br>
    <strong>Abstract:</strong><br>We demonstrate making Galaxy and Taverna, two widely used bioinformatics pipeline platforms, more interoperable. We show how Taverna workflows can be wrapped as Galaxy tools automatically, via myExperiment, using our generator. We then show how the Taverna workflow becomes part of the Galaxy pipeline with an example.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22437.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT24" >TT24</a><span style='float: right;'>Monday, July 18: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>EMBOSS: New developments and extended data access</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Peter Rice, European Bioinformatics Institute<br><br>
    <strong>Abstract:</strong><br>EMBOSS is a mature package of software tools developed for the molecular biology community. We have made a major effort to add new features and applications for this release. EMBOSS 6.4.0 includes ontologies, taxonomy, automated access to public data resources, a new query language and new data types.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22410.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT25" >TT25</a><span style='float: right;'>Monday, July 18: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>PredictProtein sequence analysis in the cloud</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Guy Yachdav, Technical University Munich<br><br>
    <strong>Abstract:</strong><br>In this technology track we report the <br>release of a new freely available protein feature prediction machine image for high-throughput cloud computing and includes the most popular prediction methods from our group, including PredictProtein. We will show how to use the image and some of the methods integrated.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22405.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT26" >TT26</a><span style='float: right;'>Monday, July 18: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Become an Expert User of the RCSB Protein Data Bank Website and Web Services</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Peter Rose, University of California San Diego<br><br>
    <strong>Abstract:</strong><br>We will demonstrate the latest features of the redesigned RCSB PDB website and provide tips how to effectively use the website and web services. Focus will be on query refinement, data export, sequence and structural neighbors, and programmatic access of data through web services and web widgets.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22440.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT27" >TT27</a><span style='float: right;'>Monday, July 18: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>The Spanish National Bioinformatics Institute (INB)</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Allan Orozco, Spanish National Bioinformatics Institute<br><br>
    <strong>Abstract:</strong><br>Our main goal with the Technology Track will cover aspects related to the bioinformatics tools, systems, desktop applications, data bases, web services and bioinformatics methods developed by the INB (Spanish National Bioinformatics Institute) and used by biologists and bioinformatics that work in areas that require complex data.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22481.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT28" >TT28</a><span style='float: right;'>Monday, July 18: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>HHblits: Lightning-fast iterative sequence searching by HMM-HMM comparison</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Michael Remmert, Gene Center Munich<br><br>
    <strong>Abstract:</strong><br>HHblits is the first iterative search method based on pairwise comparison of profile Hidden Markov Madols (HMMs). Using a very fast profile-profile based prefilter it achieves better runtimes than the frequently-used methods PSI-BLAST and HMMER3 and greatly improves upon these methods in terms of sensitivity/selectivity and alignment quality.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22287.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT29" >TT29</a><span style='float: right;'>Monday, July 18: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>ENCODE Genome-Wide Data on the UCSC Genome Browser</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Melissa Cline, UCSC<br><br>
    <strong>Abstract:</strong><br>The ENCODE data on the UCSC Genome Browser is a rich resource for bioinformatics research, with experimental designs that promote integrative, cross-species analysis.  This presentation covers the types of ENCODE data, how to obtain the data you want efficiently, and relevant Genome Browser functionality that simplifies ENCODE data analysis.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22441.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT30" >TT30</a><span style='float: right;'>Monday, July 18: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Using the EMBL-EBI services through the JDispatcher framework</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Mickael Goujon, EMBL-EBI<br><br>
    <strong>Abstract:</strong><br>The JDispatcher framework has been developed to encourage users to consume a broad range of analytical services provided by the EMBL-EBI. It focuses on improving accessibility and user experience for both the scientist and the software developer. It offers some unique features to assist the users during the results analysis.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22422.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT31" >TT31</a><span style='float: right;'>Monday, July 18: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Chipster 2.0: User-friendly NGS data analysis software with interactive genome browser</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Eija Korpelainen, CSC - IT Center for Science<br><br>
    <strong>Abstract:</strong><br>The open source software Chipster (http://chipster.csc.fi/) enables biologists to analyze and visualize NGS data and construct workflows. Tools cover quality control, preprocessing, alignment, normalization, differential expression, peak calling, motif discovery and pathway analysis. Reads and results can be viewed interactively in their genomic context with the in-built genome browser.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22401.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT32" >TT32</a><span style='float: right;'>Monday, July 18: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>GenomeRNAi: A Phenotype Database for Large-scale RNAi Screens</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Esther Schmidt, German Cancer Research Center (DKFZ)<br><br>
    <strong>Abstract:</strong><br>RNA interference (RNAi) allows systematic screening for loss-of-function phenotypes on a genome-wide scale, providing a rich source of functional annotation. The GenomeRNAi database (www.genomernai.org) holds RNAi-induced phenotype data from published screens for human and drosophila. Data on RNAi reagents are provided, as well as phenotype download options and GBrowse visualization.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22429.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT33" >TT33</a><span style='float: right;'>Monday, July 18: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>EBI Search: a new way to explore biology</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Mickael Goujon, EMBL-EBI<br><br>
    <strong>Abstract:</strong><br>The European Bioinformatics Institute provides free and unrestricted access to vast resources of biomolecular data. In order to improve accessibility and usability large-scale user experience testing (UX analysis) was used to review and improve the institute’s search service and make it simpler for users to explore these data.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22423.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT34" >TT34</a><span style='float: right;'>Monday, July 18: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>Introduction to National Center for Biomedical Ontology (NCBO) Technology</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Ray Fergerson, Stanford University<br><br>
    <strong>Abstract:</strong><br>Ontologies are increasingly used to integrate heterogeneous information sources. NCBO provides a public repository for biomedical ontologies and tools to use them to annotate text and to search biomedical datasets. This demo is an overview of current and planned NCBO technology, covering the BioPortal application, Web services, and widgets.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22444.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT35" >TT35</a><span style='float: right;'>Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><font color='red'>TT35 re-scheduled to: Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</font><br/><strong><em>GenoCAD: A Web-Based Open Source Application to Design Synthetic DNA Molecules</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Jean Peccoud, Virginia Tech<br><br>
    <strong>Abstract:</strong><br>GenoCAD is an open source web-based system that provides a streamlined, rule-driven process for designing genetic sequences. GenoCAD provides a graphical interface that allows users to design sequences consistent with formalized design strategies specific to a domain, organization, or project.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/21096.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT36" >TT36</a><span style='float: right;'>Tuesday, July 19: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>Overview of The Universal Protein Resource (UniProt)</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Benoit Bely, The European Bioinformatics Institute<br><br>
    <strong>Abstract:</strong><br>1. A description of UniProt, including new developments.<br>2. Accessing UniProt using simple query syntax, including helpful suggestions and hints. <br>3. Exploration of sequence similarity searches, alignments and ID mapping tools.<br>4. Accessing UniProt data programmatically.<br><br>This demonstration is intended to be interactive and user participation and feedback is welcomed.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22438.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT37" >TT37</a><span style='float: right;'>Tuesday, July 19: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>NGS Best Practices through Galaxy: Cloud-based variant discovery with visual analytics</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Daniel Blankenberg, Penn State University<br><br>
    <strong>Abstract:</strong><br>Here, we present a collection of Galaxy workflows for the discovery of variants in re-sequencing studies.  In addition to highlighting a new set of polymorphism discovery workflows, our demonstration will be performed in the Amazon Cloud and will utilize a newly implemented visual analytics framework.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22442.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT38" >TT38</a><span style='float: right;'>Tuesday, July 19: 10:45 a.m. - 11:10 a.m.</span></div><strong><em>New Interactive Approach to the Visual Analysis of Protein Structure and Function</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Nadezhda Doncheva, Max Planck Institute for Informatics<br><br>
    <strong>Abstract:</strong><br>Residue interaction networks derived from protein structures are very useful for understanding the structural and functional roles of residues. Therefore, we developed the software RINalyzer that offers novel visual analytics tools and supports the simultaneous viewing and exploration of a residue network in 2D and the corresponding 3D protein structure.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22443.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT39" >TT39</a><span style='float: right;'>Tuesday, July 19: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>RaptorX: an integrated web server for protein sequence-structure alignment and protein structure prediction</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Jinbo Xu, Toyota Tech Inst at Chicago<br><br>
    <strong>Abstract:</strong><br>RaptorX is a well-known software package for protein modeling and has been consistently ranked very top in the past several CASP events. We will demonstrate how to use RaptorX as an analytic and visualization tool to align protein sequences and structures and to predict 3D structures of a protein sequence.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22445.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT40" >TT40</a><span style='float: right;'>Tuesday, July 19: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>YABI: A sophisticated Internet-based cross–omics analytic environment</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Matthew Bellgard, Murdoch University<br><br>
    <strong>Abstract:</strong><br>We will describe the design and features of YABI, a sophisticated bioinformatics system that enables transparent access to supercomputing infrastructure, flexible analysis workflow creation/reuse and support of external workflow environments. Yabi is available at http://ccg.murdoch.edu.au/yabi/.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22451.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT41" >TT41</a><span style='float: right;'>Tuesday, July 19: 11:15 a.m. - 11:40 a.m.</span></div><strong><em>Clustal Omega for Protein Multiple Sequence Alignment</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Desmond Higgins, Conway Institute<br><br>
    <strong>Abstract:</strong><br>Clustal Omega is a high capacity high accuracy protein multiple alignment package.  It is one of the most accurate aligners, based on standard benchmark datasets and it can routinely align 100,000 or more sequences on a single processor. This demonstration will give an overview of its features.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22453.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT42" >TT42</a><span style='float: right;'>Tuesday, July 19: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>Introducing InterProScan 5</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Craig McAnulla, EMBL-EBI<br><br>
    <strong>Abstract:</strong><br>InterProScan is used extensively for automated sequence analysis of protein and genomic sequence data.  Completely re-written, InterProScan is a flexible application designed to be easy to install and use.  We will describe the benefits of the new system for biologists, bioinformaticians and bioinformatics service providers.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22457.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT43" >TT43</a><span style='float: right;'>Tuesday, July 19: 11:45 a.m. - 12:10 p.m.</span></div><strong><em>EDAM ontology and BioXSD: The unifying semantics and syntax for bioinformatics tools</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Matus Kalas, University of Bergen<br><br>
    <strong>Abstract:</strong><br>The EDAM (EMBRACE Data And Methods) ontology and the BioXSD XML Schema will contribute to more efficient integration of bioinformatics tools and resources into analytical workflows. EDAM is an ontology of bioinformatics operations, types of data, formats, and topics. BioXSD is a common format for sequences, alignments, and feature annotations.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22460.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT44" >TT44</a><span style='float: right;'>Tuesday, July 19: 11:45 a.m. - 12:40 p.m.</span></div><strong><em>Killer App Award</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Seán O'Donoghue, EMBL<br>Syed &nbsp;Asad Rahman,&nbsp;EBI&nbsp; <br>Sirisha &nbsp;Gollapudi,&nbsp;University of Leicester&nbsp; <br>Joanna&nbsp;M. Kasprzak,&nbsp;Adam Mickiewicz University&nbsp; <br><br><br>
    <strong>Abstract:</strong><br> <p>The ISMB 'Killer App Award' (http://killerapp.iscb.org/2011.html) will be awarded for the
 individual or team with a tool or system judged to be of most
 practical benefit to biochemists and/or molecular biologists. To
 be eligible, a system or tool must be fully functional, and must
 be presented at the ISMB conference as a poster. Three finalists
 have been chosen and invited to give a short talk about their
 application during the Killer App session. The winner will be
 announced in the closing ceremony, and will receive a
 certificate from the ISMB/ISCB plus an Apple iPad. The three
 finalists are:</p>

<p>
 ECBlast: a novel tool for searching similar enzymes based on
 chemical knowledge" by Syed Asad Rahman, Franz Fenninger,
 Gillleain Torrence, Lorenzo Baldacci, Nimish Gopal, Gemma L
 Holliday, and Janet M Thornton.
</p><p>
 GWAS Central: a database for the integration and comparative
 interrogation of genome-wide association study datasets" by
  Robert C. Free, Robert K. Hastings, Tim Beck, 
 Sirisha Gollapudi, Gudmundur A. Thorisson, and Anthony J. Brookes.
</p><p>
 PyRy3D: a tool for modeling of large macromolecular
 complexes" by Joanna M. Kasprzak, Wojciech Potrzebowski, Mateusz
 Dobrychłop and Janusz M. Bujnicki.</p><br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22480.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT45" >TT45</a><span style='float: right;'>Tuesday, July 19: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>CAMERA: Metagenomic Resources and Tools</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Shulei Sun, University of California, San Diego<br><br>
    <strong>Abstract:</strong><br>CAMERA provides a single system for depositing, locating, analyzing, visualizing and sharing data about microbial biology. Its computational resources include rich and distinctive bioinformatics tools and a flexible collaborative workflow environment that makes it possible for researchers to analyze large and complex sequencing data generated by NGS technology.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22465.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT46" >TT46</a><span style='float: right;'>Tuesday, July 19: 12:15 p.m. - 12:40 p.m.</span></div><strong><em>Knowledge-driven QTL analysis and network-based candidate gene discovery with Ondex</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Keywan Hassani-Pak, Rothamsted Research<br><br>
    <strong>Abstract:</strong><br>Integrative and automated tools are required for analyzing QTL and selecting candidate genes. We introduce a knowledge-driven approach based on the Ondex (www.ondex.org) framework for data integration and network visualization. We present a web-based tool which links classical genetics views with modern network views to explain functional candidacy of genes.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22476.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT47" >TT47</a><span style='float: right;'>Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>SORF, a web application for heterogeneous data management and visualisation</em></strong><br><br /><strong>Room: </strong>Hall F2<br /><br /><strong>Presenting Author: </strong>Fady Mohareb, The Bioinformatics Group - Cranfield University<br><br>
    <strong>Abstract:</strong><br>SORF is a web application for heterogeneous data management and visualisation, compatible with a wide variety of experimental data formats. This demonstration will guide the attendants through the procedures of defining a new experiment, uploading experimental datasets, as well as an overview of the visualisation tools the system currently provides.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22472.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT48" >TT48</a><span style='float: right;'>Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>Oqtans: A Galaxy-integrated workflow for Quantitative Transcriptome Analysis from NGS data</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Géraldine Jean and  Sebastian J. Schultheiss, Friedrich Miescher Laboratory of Max Planck Society<br><br>
    <strong>Abstract:</strong><br>We demonstrate how a complete quantitative RNA-seq analysis can be performed easily and effectively in <20 minutes with our Machine-Learning-powered workflow. It's integrated in the easy-to-use Galaxy framework and is accessible locally or in the cloud (http://galaxy.fml.mpg.de). We developed a modular toolsuite that performs better or as well as the state-of-the-art for short-read alignments, transcript identification/quantification and differential expression analysis.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22477.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT49" >TT49</a><span style='float: right;'>Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>NCBO Web Services and Development of Semantic Applications</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Patricia Whetzel, Stanford University / National Center for Biomedical Ontology<br><br>
    <strong>Abstract:</strong><br>Ontologies provide domain knowledge to drive data integration and information retrieval. The National Center for Biomedical Ontology provides a suite of ontology-based Web services for the development of semantically aware software applications. This presentation will provide an overview of these Web services and their use within software applications.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22478.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT50" >TT50</a> <font style='color: red; margin-left: 20px;'>Withdrawn</font> <span style='float: right;'>Tuesday, July 19: 2:30 p.m. - 2:55 p.m.</span></div><strong><em>OncoPortal, a data management and analysis framework for cancer research</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Arek Kasprzyk, Ontario Institute for Cancer Research<br><br>
    <strong>Abstract:</strong><br>To address the needs of the increasingly-collaborative field of cancer research, we are creating OncoPortal. Based on the BioMart software, OncoPortal is a set of tools that allows cancer research projects to quickly and easily allow users to access their data, analyse them, and integrate them with external annotation sources.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22467.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT51" >TT51</a><span style='float: right;'>Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Recent Developments in the Pathway Tools Software and BioCyc Databases</em></strong><br><br /><strong>Room: </strong>Hall F2<br /><br /><strong>Presenting Author: </strong>Peter Karp, SRI International<br><br>
    <strong>Abstract:</strong><br>Recent developments include growth of the BioCyc database collection<br>to 1,100 genomes, the ability of Pathway Tools to automatically<br>generate flux-balance models from its Pathway/Genome databases in a<br>manner that drastically reduces model-development time, and <br>upgrading of the web metabolic map diagram.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22479.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT52" >TT52</a><span style='float: right;'>Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>eSysbio - an online system for interdisciplinary life science research</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Pål Puntervoll, Uni Research<br><br>
    <strong>Abstract:</strong><br>eSysbio provides a virtual workspace for project-oriented data sharing and analyses. Researchers get instant access to remote Web service tools and R-script/packages through automatically generated  web-based graphical user interfaces. Workflows can be manually constructed from the services and R-scripts, and the history of the composite analysis is recorded.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22463.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT53" >TT53</a><span style='float: right;'>Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>Interfacing with Industry: The EMBL-EBI Industry Programme and SME Forum</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Dominic Clark, EMBL-EBI<br><br>
    <strong>Abstract:</strong><br>This presentation will provide an overview and demonstration of some of the means by which the European Molecular Biology Laboratory-European Bioinformatics Institute (EMBL-EBI) interacts with industry stakeholders, who already comprise a significant and growing proportion of EMBL-EBI’s user base.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22417.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT54" >TT54</a><span style='float: right;'>Tuesday, July 19: 3:00 p.m. - 3:25 p.m.</span></div><strong><em>BioMart: A data federation framework for large collaborative projects</em></strong><br><br /><strong>Room: </strong>Lounge 1<br /><br /><strong>Presenting Author: </strong>Junjun Zhang, Ontario Institute for Cancer Research<br><br>
    <strong>Abstract:</strong><br>To manage the data generated by large, collaborative biological projects, we have created BioMart data federation system. BioMart seamlessly presents geographically-disparate data sources in a single access point, thereby enabling the creation of distributed, databases ideally suited for presenting the complex data generated by multi-institution projects.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22466.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT55" >TT55</a><span style='float: right;'>Tuesday, July 19: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Database on Potential “Antibiotics” From Flowering Plants</em></strong><br><br /><strong>Room: </strong>Hall F2<br /><br /><strong>Presenting Author: </strong>Rajarajan Swaminathan, Presidency College (Aut)<br><br>
    <strong>Abstract:</strong><br>The database provides details about phytochemical compounds whose potential antimicrobial activities have been experimentally verified by in vitro/in vivo/clinical trials with feisty details if any and provides instant access via the www.bifcpresidency.tn.gov.in.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22483.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT56" >TT56</a><span style='float: right;'>Tuesday, July 19: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>Jalview 2.7</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Geoffrey Barton, University of Dundee<br><br>
    <strong>Abstract:</strong><br>Jalview 2.7 is the latest version of this freely available, and widely used Java based multiple sequence alignment, annotation, visualization and editing tool. In this technology track presentation, we will give a tour of its core strengths, and how they will be developed in the future.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22491.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT57" >TT57</a><span style='float: right;'>Tuesday, July 19: 3:30 p.m. - 3:55 p.m.</span></div><strong><em>The International Society for Biocuration and the efforts of its members</em></strong><br><br /><strong>Room: </strong>J565<br /><br /><strong>Presenting Author: </strong>Claire O'Donovan, EMBL - EBI<br><br>
    <strong>Abstract:</strong><br>In this technical track, I would like to introduce the participants to the International Society for Biocuration and report on the diverse and valuable work of its members around the world as well as highlighting the potential careers in this field.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22493.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT58" >TT58</a><span style='float: right;'>Tuesday, July 19: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>KeyPathwayMiner - Detecting case-specific biological pathways by using expression data</em></strong><br><br /><strong>Room: </strong>Hall F2<br /><br /><strong>Presenting Author: </strong>Nicolas Alcaraz, Max Planck Institute for Informatics<br><br>
    <strong>Abstract:</strong><br>We introduce KeyPathwayMiner, a tool that allows for extracting and visualizing active<br>key-pathways in a given series of gene expression studies. We detect sub-networks where most genes are differentially expressed in a particular disease. KeyPathwayMiner<br>identifies maximal connected sub-graphs where all but k nodes are differentially<br>expressed in all but l cases.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/20949.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br><div class='subheading2'><a name="TT59" >TT59</a><span style='float: right;'>Tuesday, July 19: 4:00 p.m. - 4:25 p.m.</span></div><strong><em>The Galaxy Track Browser: Transforming the Genome Browser from Visualization Tool to Analysis Tool</em></strong><br><br /><strong>Room: </strong>J241<br /><br /><strong>Presenting Author: </strong>Jeremy Goecks, Emory University<br><br>
    <strong>Abstract:</strong><br>The Galaxy Track Browser (GTB) is a Web-based genome browser integrated into the Galaxy platform. GTB leverages the Galaxy platform to combine data visualization and data analysis, enabling users to (a) specify parameters and run tools to produce and visualize new data and (b) dynamically filter visualized data.<br><br><strong>Presentation PDF:</strong> <a target=tt href="http://www.iscb.org/uploaded/css/82/22435.pdf">Download PDF</a><br><a href="#up" style="font-size:9px;">TOP</a><br><br></div></td> 
</tr> 
</table> 
<span class="article_separator">&nbsp;</span> 
                                                      	</div> 
						</div> 
						<div class="clr"></div> 
					</div> 
					<div class="clr"></div> 
				</div> 
 
				<div id="whitebox_b"> 
					<div id="whitebox_bl"> 
						<div id="whitebox_br"></div> 
					</div> 
				</div> 
			</div> 
 
			<div id="footerspacer"></div> 
		</div> 
 
		
	</div> 
</div> 
 
 
</body> 

<!-- Mirrored from www.iscb.org/cms_addon/conferences/ismbeccb2011/technologytrack.php by HTTrack Website Copier/3.x [XR&CO'2010], Mon, 27 Feb 2012 05:50:15 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=UTF-8"><!-- /Added by HTTrack -->

<!-- Mirrored from www.iscb.org/archive/conferences/iscb/cms_addon/conferences/ismbeccb2011/technologytrack.html by HTTrack Website Copier/3.x [XR&CO'2014], Thu, 08 Aug 2024 12:34:55 GMT -->
</html> 
