
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb">


<!-- Mirrored from eccb14.loria.fr/program/highlight-papers?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:53:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <script type="text/javascript">
var siteurl='../index.html';
var tmplurl='../templates/ja_elastica/index.html';
var isRTL = false;
</script>

  <base  />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="keywords" content="ECCB'14, ECCB2014, Computational Biology, Bioinformatics, Conference, Strasbourg, ECCB, European Conference on Computational Biology, ECCB14" />
  <meta name="author" content="Adrien" />
  <meta name="robots" content="noindex, nofollow" />
  <meta name="description" content="ECCB'14 is the key European computational biology event in 2014 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <meta name="generator" content="Joomla! - Open Source Content Management" />
  <title>ECCB'14 - Highlight Papers</title>
  <link href="http://eccb14.loria.fr/program/highlight-papers?tmpl=component&amp;print=1&amp;page=" rel="canonical" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/media/jui/css/icomoon.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/." type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/addons.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/usertools.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/print.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/menu/mega.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/typo.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/modules.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile.css" type="text/css" media="only screen and (max-width:719px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile-port.css" type="text/css" media="only screen and (max-width:479px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-tablet.css" type="text/css" media="only screen and (min-width:720px) and (max-width: 985px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-normal.css" type="text/css" media="only screen and (min-width:986px) and (max-width: 1235px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-wide.css" type="text/css" media="only screen and (min-width:1236px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/menu/mega.css" type="text/css" />
  <script src="http://eccb14.loria.fr/media/system/js/mootools-core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/mootools-more.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/anchor.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/mega.rjd.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery/jquery-1.8.3.min.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.noconflict.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.masonry.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.mediaqueries.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/menu/mega.js" type="text/javascript"></script>
  <script type="text/javascript">
jQuery.noConflict();
  </script>

<!--[if ie]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/templates/ja_elastica/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 


<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=3.0, user-scalable=yes"/>
<meta name="HandheldFriendly" content="true" />


<link href="http://eccb14.loria.fr/templates/ja_elastica/images/favicon.ico" rel="shortcut icon" type="image/x-icon" /></head>

<body id="bd" class="fs3 com_content contentpane">
    
<div id="system-message-container">
<div id="system-message">
</div>
</div>
    <div class="item-page clearfix">
	<h2 class="contentheading">
			Selected Highlight Papers		</h2>

	
<div class="article-tools clearfix">
	<ul class="actions">
			<li>
		<a href="#" onclick="window.print();return false;"><span class="icon-print"></span>&#160;Print&#160;</a>		</li>
		
	</ul>



	
</div>
	<p><strong><strong>DOWNLOAD: </strong></strong><a href="http://eccb14.loria.fr/programme/higlight_papers_abstracts.pdf">[All the Abstracts of Highlight Papers in pdf file]</a></p>
<p><strong> </strong></p>
<p><strong><a name="HP01"></a>HP01: </strong><span>Wiring miRNAs to pathways: a topological approach to integrate miRNA and mRNA expression profiles</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span>Enrica Calura, Paolo Martini, Gabriele Sales and Chiara Romualdi</span></p>
<p style="padding-left: 30px; text-align: justify;"><strong>Abstract:</strong> The production rate of gene expression data is nothing less than astounding. However, with the benefit of hindsight we can assert that, since we completely ignored the non-coding part of the transcriptome, we spent the last decade to study cell mechanisms having few data in our hands. In this scenario, microRNAs, which are key post-trascriptional regulators, deserve special attention. Currently, miRNA and gene circuits are identified through the combination of binding prediction and expression correlation analyses, MAGIA, the web tool we developed, is an example to feel this aim (Sales et al NAR 2010, Bisognin et al NAR 2012). Although effective in many cases the simple correlation does not imply a causal relationship and a lot of false positive miRNA-mRNA interactions are still found. Moreover, miRNA and target genes are characterized by many-to-many relationships and they should be considered as part of a much more complex system of cellular interactions. Recently, to analyze the cellular circuits we developed a new web tool dedicated to topological pathway analyses called Graphite Web (Sales et al NAR 1013). Given the state of knowledge about the biogenesis of miRNAs, their mechanisms of action and the numerous experimentally validated target genes, miRNAs are also gradually appearing in the formal pathway representations such as KEGG and Reactome maps. However, the number of miRNAs annotated in pathway maps is very small and pathway analyses exploiting this new regulatory layer are still lacking. To fill these gaps, we developed micrographite a new pipeline to perform topological pathway analysis integrating gene and miRNA expression profiles. Micrographite analysis of gene and miRNA integrated transcriptome is used to study and dissect the epithelial ovarian cancer gene complexity and miRNA transcriptome defining and validating a new regulatory circuits.</p>
<p style="padding-left: 30px;"><strong>Original Publications: </strong></p>
<p style="padding-left: 30px;"><span>Calura E, Fruscio R, Paracchini L, Bignotti E, Ravaggi A, Martini P, Sales G, Beltrame L, Clivio L, Ceppi L, Di Marino M, Fuso Nerini I, Zanotti L, Cavalieri D, Cattoretti G, Perego P, Milani R, Katsaros D, Tognon G, Sartori E, Pecorelli S, Mangioni C, D'Incalci M, Romualdi C, Marchini S. MiRNA landscape in stage I epithelial ovarian cancer defines the histotype specificities. <em>Clin Cancer Res.</em> 2013 Aug 1;19(15):4114-23.</span></p>
<p style="padding-left: 30px;"><span>Calura E, Martini P, Sales G, Beltrame L, Chiorino G, D'Incalci M, Marchini S, Romualdi C. Wiring miRNAs to pathways: a topological approach to integrate miRNA and mRNA expression profiles.</span><em> Nucleic Acids Res.</em><span> 2014;42(11):e96.</span></p>
<p style="padding-left: 30px;"><strong>Presenter:</strong> Enrica Calura, Department of Biology, University of Padova, Italy</p>
<p style="padding-left: 30px;"><strong>Area:</strong>  C. Pathways and molecular networks</p>
<p><strong> </strong></p>
<p><strong><a name="HP02"></a>HP02: </strong><span style="line-height: 1.3em;">Comparison of mapping algorithms used in high-throughput sequencing: application to Ion Torrent data</span></p>
<p style="padding-left: 30px;"><strong><span style="line-height: 1.3em;">Authors: </span></strong><span style="line-height: 1.3em;">Ségolène Caboche, Christophe Audebert, Yves Lemoine and David Hot</span></p>
<p style="padding-left: 30px; text-align: justify;"><span style="line-height: 1.3em;"><strong>Abstract:</strong> A fundamental step in High-throughput sequencing (HTS) data analysis is the mapping of reads onto reference sequences. Choosing a suitable mapper is a subtle task because of the difficulty of evaluating mapping algorithms. We present a benchmark procedure to compare mappers using both real and simulated datasets and considering computational resource and time requirements, robustness of mapping, ability to report positions for reads in repetitive regions, and ability to retrieve true genetic variation positions. To measure robustness, a new definition for a correctly mapped read was introduced. We developed CuReSim, a read simulator, and CuReSimEval, a tool to evaluate the mapping quality of the simulated reads. The benchmark procedure was applied to evaluate mappers in the context of whole genome sequencing of small genomes with Ion Torrent data. These results were used to develop a pipeline to quickly and automatically characterize pathogens during an episode of infection.</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"><strong>Original Publication: </strong>Caboche S, Audebert C, Lemoine Y, Hot D. Comparison of mapping algorithms used in high-throughput sequencing: application to Ion Torrent data. <em>BMC Genomics</em>. 2014 Apr 5;15:264.</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"><strong>Presenter:</strong> Ségolène Caboche, FRE 3642 Molecular and Cellular Medecine, CNRS, Institut Pasteur de Lille and University of Lille Nord de France, France</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;"><strong>Area:</strong>  A. Sequencing and sequence analysis for genomics </span></p>
<p> </p>
<p><strong><a name="HP03"></a>HP03: </strong><span>Patterns of positive selection in seven ant genomes</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span>Julien Roux, Eyal Privman, Sébastien Moretti, Josephine Daub, Marc Robinson-Rechavi and Laurent Keller</span></p>
<p style="padding-left: 30px; text-align: justify;"><strong>Abstract:</strong> The evolution of ants is marked by remarkable adaptations that allowed the development of very complex social systems. To identify how ant-specific adaptations are associated with patterns of molecular evolution, we searched for signs of positive selection on amino-acid changes in proteins. We identified 24 functional categories of genes which were enriched for positively selected genes in the ant lineage. We also reanalyzed genome-wide datasets in bees and flies with the same methodology, to check whether positive selection was specific to ants or also present in other insects. Notably, genes implicated in immunity were enriched for positively selected genes in the three lineages, ruling out the hypothesis that the evolution of hygienic behaviors in social insects caused a major relaxation of selective pressure on immune genes. Our scan also indicated that genes implicated in neurogenesis and olfaction started to undergo increased positive selection before the evolution of sociality in Hymenoptera. Finally, the comparison between these three lineages allowed us to pinpoint molecular evolution patterns that were specific to the ant lineage. In particular, there was ant-specific recurrent positive selection on genes with mitochondrial functions, suggesting that mitochondrial activity was improved during the evolution of this lineage. This might have been an important step toward the evolution of extreme lifespan that is a hallmark of ants.</p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong>Roux J, Privman E, Moretti S, Daub JT, Robinson-Rechavi M, Keller L. Patterns of positive selection in seven ant genomes. <em>Mol Biol Evol.</em> 2014 Jul;31(7):1661-85.</p>
<p style="padding-left: 30px;"><strong>Presenter:</strong> <strong> </strong>Julien Roux, University of Lausanne, Switzerland<span style="line-height: 1.3em;"> </span></p>
<p style="padding-left: 30px;"><strong>Area:</strong>  F. Evolution and population genomics</p>
<p> </p>
<p><strong><a name="HP04"></a>HP04: </strong><span>Comprehensive analysis of DNA polymerase III alpha subunits and their homologs in bacterial genomes</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span>Kęstutis Timinskas, Monika Balvočiūtė, Albertas Timinskas and Česlovas Venclovas</span></p>
<p style="padding-left: 30px; text-align: justify;"><strong>Abstract:</strong> <span>Bacteria, unlike archaea and eukaryotes, use distinct C-family DNA polymerases for genome replication. Unfortunately, except for a few species, bacterial genome replication is poorly characterized. It is not known whether all bacteria use C-family DNA polymerases for DNA replication, how many distinct C-family groups are there, and how many different replication systems they form. In order to address these questions, we performed extensive computational analysis of C-family polymerases in nearly 2000 complete bacterial genomes. We found that all the genomes without exception encode at least one C-family polymerase implying the universal use of this polymerase family for bacterial DNA replication. Our analysis revealed four distinct groups of C-family polymerases. Based on their properties and distribution in genomes we discovered a novel, so far experimentally uncharacterized, replication system in Clostridia. Computational results also indicated that one of the C-family groups might be responsible for shaping genomic G+C content.</span><span style="line-height: 1.3em;"> </span></p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong>Timinskas K, Balvočiūtė M, Timinskas A, Venclovas Č. Comprehensive analysis of DNA polymerase III α subunits and their homologs in bacterial genomes. <em>Nucleic Acids Res.</em> 2014 Feb;42(3):1393-413.<span style="line-height: 1.3em;"> </span></p>
<p style="padding-left: 30px;"><strong>Presenter:</strong> Česlovas Venclovas, Institute of Biotechnology, Vilnius University, Lithuania.<span style="line-height: 1.3em;"> </span></p>
<p style="padding-left: 30px;"><strong>Area:  </strong>E. Structural bioinformatics</p>
<p><span style="line-height: 1.3em;"> </span></p>
<p><strong><a name="HP05"></a>HP05: </strong><span>High-dimensional Bayesian parameter estimation: Case study for a model of JAK2/STAT5 signaling</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span>Sabine Hug, Andreas Raue, Jan Hasenauer, Julie Bachmann, Ursula Klingmüller, Jens Timmer and Fabian J. Theis</span></p>
<p style="padding-left: 30px; text-align: justify;"><strong>Abstract:</strong> <span>Mechanistic dynamical models are nowadays commonly used for the analysis for complex datasets. Dynamical models depend however on many unknown parameters which have to be inferred from experimental data. The statistical inference in a high-dimensional parameter space is however conceptually and computationally challenging. In this paper we provide a proof of principle that the rigorous statistical analysis is also feasible in these demanding situations. </span><span>To ensure rigorous assessment of model and prediction uncertainties we take advantage of both a profile posterior approach and Markov chain Monte Carlo sampling. </span><span>We analyzed a dynamical model of the JAK2/STAT5 signaling pathway containing more than hundred parameters. The profile posterior reveals that the corresponding posterior distribution is bimodal. To nevertheless guarantee efficient mixing we applied a multi-chain sampling approach. The Bayesian parameter estimation enables the assessment of prediction uncertainties and the design of additional experiments enhancing the explanatory power of the model.</span><span style="line-height: 1.3em;"> </span></p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong><span>Hug S, Raue A, Hasenauer J, Bachmann J, Klingmüller U, Timmer J, Theis FJ. High-dimensional Bayesian parameter estimation: case study for a model of JAK2/STAT5 signaling. <em>Math Biosci.</em> 2013 Dec;246(2):293-304.</span></p>
<p style="padding-left: 30px;"><strong>Presenter:</strong> Sabine Hug, Institute of Computational Biology, Helmholtz Zentrum München,<span> Germany</span></p>
<p style="padding-left: 30px;"><strong>Area:</strong>  D. Computational systems biology</p>
<p style="padding-left: 30px;"> </p>
<p><strong><a name="HP06"></a>HP06: </strong><span>Shaping the interaction landscape of bioactive molecules</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span>David Gfeller, Aurelien Grosdidier, Matthias Wirth, Antoine Daina, Olivier Michielin and Vincent Zoete</span></p>
<p style="padding-left: 30px; text-align: justify;"><strong>Abstract:</strong> Bioactive small molecules, such as drugs or metabolites, interact with proteins targets to modulate their activity, which in turn results in the observed phenotypic effects. However, for most bioactive compounds the list of targets is only partially known. Therefore computational predictions of bioactive molecule targets are powerful to narrow down the number of potential targets and to rationalize side effects of known molecules. Here, we introduce a new computational approach to accurately predict the targets of bioactive small molecules based on a combination of 2D and 3D similarity measures with known ligands. The method is trained on a large dataset of 280,381 small molecules interacting with 2686 targets from the ChEMBL database. Predictions can be carried out in five different organisms, and mapping predictions by homology within and between different species is enabled for close paralogs and orthologs. The method is accessible free of charge at <a href="http://www.swisstargetprediction.ch/">http://www.swisstargetprediction.ch</a>.</p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong>Gfeller D, Michielin O, Zoete V. Shaping the interaction landscape of bioactive molecules. <em>Bioinformatics</em>. 2013 Dec 1;29(23):3073-9.</p>
<p style="padding-left: 30px;"><strong>Presenter:</strong> David Gfeller, Swiss Institute of Bioinformatics, Lausanne, Switzerland</p>
<p style="padding-left: 30px;"><strong>Area:</strong>  H. Biological knowledge discovery from data, texts and bio-images</p>
<p style="padding-left: 30px;"> </p>
<p><strong><a name="HP07"></a>HP07: </strong><span>Key regulators control distinct transcriptional programmes in blood and progenitor and mast cells</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span>Felicia Ng, Fernando Calero-Nieto, Nicola Wilson, Rebecca Hannah, Victoria Moignard, Ana Leal-Cervantes, Isabel Jimenez-Madrid, Evangelia Diamanti, Lorenz</span><span> Wernisch and Berthold Gottgens.</span></p>
<p style="padding-left: 30px; text-align: justify;"><strong>Abstract:</strong> <span>Despite major advances in the generation of genome-wide binding maps, the mechanisms by which transcription factors (TFs) regulate cell type identity have remained largely obscure. Through comparative analysis of 10 key haematopoietic TFs in both mast cells and blood progenitors, we demonstrate that the largely cell type-specific binding profiles are not opportunistic, but instead contribute to cell type-specific transcriptional control, because (i) mathematical modelling of differential binding of shared TFs can explain differential gene expression, (ii) consensus binding sites are important for cell type-specific binding and (iii) knock-down of blood stem cell regulators in mast cells reveals mast cell-specific genes as direct targets. Finally, we show that the known mast cell regulators Mitf and c-fos likely contribute to the global reorganisation of TF binding profiles. Taken together therefore, our study elucidates how key regulatory TFs contribute to transcriptional programmes in several distinct mammalian cell types.</span></p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong>Calero-Nieto FJ, Ng FS, Wilson NK, Hannah R, Moignard V, Leal-Cervantes AI, Jimenez-Madrid I, Diamanti E, Wernisch L, Göttgens B. Key regulators control distinct transcriptional programmes in blood  progenitor and mast cells. <em>EMBO J.</em> 2014 Jun 2;33(11):1212-26.</p>
<p style="padding-left: 30px;"><strong>Presenter: </strong><span>Felicia Ng, </span><span>Cambridge Institute for Medical Research, United Kingdom.</span></p>
<p style="padding-left: 30px;"><strong>Area:</strong>  B. Gene expression</p>
<p style="padding-left: 60px;"> </p>
<p><strong><a name="HP08"></a>HP08: </strong><span style="line-height: 1.3em;">Chromatin Position Effects Quantified from Thousands of Reporters Integrated in Parallel</span><span style="line-height: 1.3em;"> </span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span style="line-height: 1.3em;">Lodewyk Wessels, Waseem Akhtar, Johann de Jong, Alex Pindyurin, Ludo Pagie, Wouter Meuleman, Jeroen de Ridder, Anton Berns, Maarten van Lohuizen and Bas van Steensel</span></p>
<p style="padding-left: 30px; text-align: justify;"><span><strong>Abstract: </strong></span><span style="text-align: justify; line-height: 1.3em;">Reporter genes integrated into the genome are a powerful tool to reveal effects of regulatory elements and local chromatin context on gene expression. However, such assays have been low throughput. Here, we describe an approach to monitor transcriptional activity of thousands of randomly integrated reporters. Computational analyses of more than 27,000 distinct reporter integrations in mouse embryonic stem cells reveal the following. First, lamina associated domains act as attenuators of transcription, likely by reducing access of transcription factors to binding sites. Second, chromatin compaction as derived from HiC data is predictive of reporter activity. Third, we find evidence of cross-talk between neighbouring genes and estimate that enhancers can influence gene expression on average over ~20 kb. Most importantly, the richness and size of the datasets opens up the opportunity for additional extensive and robust computational analyses. We will showcase the utility with recent analyses shedding new light on gene regulation.</span></p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong>Akhtar W, de Jong J, Pindyurin AV, Pagie L, Meuleman W, de Ridder J, Berns A, Wessels LF, van Lohuizen M, van Steensel B. Chromatin position effects assayed by thousands of reporters integrated in parallel. <em>Cell</em>. 2013 Aug 15;154(4):914-27.</p>
<p style="padding-left: 30px;"><strong>Presenter: </strong><span style="line-height: 1.3em;">Lodewyk Wessels, Netherlands Cancer Institute, Amsterdam, The Netherlands</span></p>
<p style="padding-left: 30px;"><span><strong>Area: </strong> B. Gene expression</span></p>
<p><span> </span></p>
<p><strong><a name="HP09"></a>HP09: </strong><span style="line-height: 1.3em;">Novel Developments In Computational Clinical Breath Analysis and Biomarker Detection</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span style="line-height: 1.3em;">Anne-Christin Hauschild, Jörg Ingo Baumbach and Jan Baumbach.</span></p>
<p style="padding-left: 30px; text-align: justify;"><span><strong>Abstract:</strong> The volatolom is the sum of volatile organic compounds that are emitted by all living cells and tissues. <br /> We seek to non-invasively “sniff'” biomarker molecules that are predictive for the biomedical fate of individual patients. This promises great hope to move the therapeutic windows to earlier stages of disease progression. While portable devices for breathomics measurement exist, we face the traditional biomarker research barrier: a lack of robustness hinders translation to the world outside laboratories. To move from biomarker discovery to validation, from separability to predictability, we have developed several bioinformatics methods for computational breath analysis, which have the potential to redefine non-invasive biomedical decision making by rapid and cheap matching of decisive medical patterns in exhaled air. We aim to provide a supplementary diagnostic tool complementing classic urine, blood and tissue samples. The presentation will review the state of the art, highlight existing challenges and introduce new data mining methods for identifying breathomics biomarkers.</span></p>
<p style="padding-left: 30px;"><strong>Original Publications:</strong></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Hauschild AC, Kopczynski D, D'Addario M, Baumbach JI, Rahmann S, Baumbach J. Peak detection method evaluation for ion mobility spectrometry by using machine learning approaches. <em>Metabolites</em>. 2013 Apr 16;3(2):277-93.</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Eckel SP, Baumbach J, Hauschild AC. On the importance of statistics in breath analysis--hope or curse? <em>J Breath Res</em>. 2014 Mar;8(1):012001.</span></p>
<p style="padding-left: 30px;"><span style="line-height: 1.3em;">Maurer F, Hauschild AC, Eisinger K, Baumbach J, Mayor A and Baumbach JI. MIMA - a software for analyte identification in MCC/IMS chromatograms by mapping accompanying GC/MS measurements. <em>Int. J. Ion Mobil. Spec.</em> 2014 Apr:17:95–101.</span></p>
<p style="padding-left: 30px;">Smolinska A, Hauschild AC, Fijten RR, Dallinga JW, Baumbach J, van Schooten FJ. Current breathomics-a review on data pre-processing techniques and machine learning in metabolomics breath analysis. <em>J Breath Res</em>. 2014 Jun;8(2):027105.</p>
<p style="padding-left: 30px;"><span><strong>Presenter:</strong> Jan Baumbach, </span><span>University of Southern Denmark, Denmark</span></p>
<p style="padding-left: 30px;"><span><strong>Area:</strong>  G. Bioinformatics of health and disease </span></p>
<p><span> </span><span style="line-height: 1.3em;"> </span></p>
<p><strong><a name="HP10"></a>HP10: </strong><span style="line-height: 1.3em;">Text mining technologies for database curation</span></p>
<p style="padding-left: 30px;"><strong>Authors: </strong><span style="line-height: 1.3em;">Fabio Rinaldi, Simon Clematide, Simon Hafner, Gerold Schneider, Gintare Grigonyte, Martin Romacker and Therese Vachon</span></p>
<p style="padding-left: 30px; text-align: justify;"><span><strong>Abstract:</strong> Although human curation for life science databases offers the best guarantee of high quality annotations, it suffers from severe bottlenecks which have long been recognized in the curation community. The most pressing problem is that of efficiency of the process: it is impossible for human curators to keep up with the growing pace of publication. Text mining technologies, coupled with advanced user interfaces, offer the potential to partially alleviate this bottleneck. We survey the results of several recent competitive evaluations of text mining technologies, discuss how text mining systems can be integrated in a curation workflow, and illustrate our approach to assisted curation, which has been tested in collaboration with major databases.</span></p>
<p style="padding-left: 30px;"><strong>Original Publication: </strong>Rinaldi F, Clematide S, Hafner S, Schneider G, Grigonyte G, Romacker M, Vachon T. Using the OntoGene pipeline for the triage task of BioCreative 2012. <em>Database (Oxford)</em>. 2013 Feb 9;2013:bas053.</p>
<p style="padding-left: 30px;"><span><strong>Presenter:</strong> Fabio Rinaldi, </span><span style="line-height: 1.3em;">Institute of Computational Linguistics, University of Zurich, Switzerland</span></p>
<p style="padding-left: 30px;"><span><strong>Area:</strong>  H. Biological knowledge discovery from data, texts and bio-images</span></p>
<p style="padding-left: 30px;"> </p>
<p><strong><strong>DOWNLOAD: </strong></strong><a href="http://eccb14.loria.fr/programme/higlight_papers_abstracts.pdf">[All the Abstracts of Highlight Papers in pdf file]</a></p>
		

	</div>
</body>


<!-- Mirrored from eccb14.loria.fr/program/highlight-papers?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:53:24 GMT -->
</html>