
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-gb" lang="en-gb">


<!-- Mirrored from eccb14.loria.fr/program/keynotes?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:48:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <script type="text/javascript">
var siteurl='../index.html';
var tmplurl='../templates/ja_elastica/index.html';
var isRTL = false;
</script>

  <base  />
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="keywords" content="ECCB'14, ECCB2014, Computational Biology, Bioinformatics, Conference, Strasbourg, ECCB, European Conference on Computational Biology, ECCB14" />
  <meta name="author" content="Emmanuel" />
  <meta name="robots" content="noindex, nofollow" />
  <meta name="description" content="ECCB'14 is the key European computational biology event in 2014 uniting scientists working in a broad range of disciplines, including bioinformatics, computational biology, biology, medicine, and systems biology." />
  <meta name="generator" content="Joomla! - Open Source Content Management" />
  <title>ECCB'14 - Keynote speakers</title>
  <link href="http://eccb14.loria.fr/program/keynotes?tmpl=component&amp;print=1&amp;page=" rel="canonical" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/media/jui/css/icomoon.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/." type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/addons.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/usertools.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/print.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/css/menu/mega.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/typo.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template-j30.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/template.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/modules.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile.css" type="text/css" media="only screen and (max-width:719px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-mobile-port.css" type="text/css" media="only screen and (max-width:479px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-tablet.css" type="text/css" media="only screen and (min-width:720px) and (max-width: 985px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-normal.css" type="text/css" media="only screen and (min-width:986px) and (max-width: 1235px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/layout-wide.css" type="text/css" media="only screen and (min-width:1236px)"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/css3.css" type="text/css" />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/component.css" type="text/css" media="all"  />
  <link rel="stylesheet" href="http://eccb14.loria.fr/templates/ja_elastica/css/menu/mega.css" type="text/css" />
  <script src="http://eccb14.loria.fr/media/system/js/mootools-core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/mootools-more.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/media/system/js/anchor.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/plugins/system/jat3/jat3/base-themes/default/js/core.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/mega.rjd.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery/jquery-1.8.3.min.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.noconflict.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.masonry.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/jquery.mediaqueries.js" type="text/javascript"></script>
  <script src="http://eccb14.loria.fr/templates/ja_elastica/js/menu/mega.js" type="text/javascript"></script>
  <script type="text/javascript">
jQuery.noConflict();
  </script>

<!--[if ie]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/plugins/system/jat3/jat3/base-themes/default/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 
<!--[if ie 7]><link href="/templates/ja_elastica/css/template-ie7.css" type="text/css" rel="stylesheet" /><![endif]--> 


<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=3.0, user-scalable=yes"/>
<meta name="HandheldFriendly" content="true" />


<link href="http://eccb14.loria.fr/templates/ja_elastica/images/favicon.ico" rel="shortcut icon" type="image/x-icon" /></head>

<body id="bd" class="fs3 com_content contentpane">
    
<div id="system-message-container">
<div id="system-message">
</div>
</div>
    <div class="item-page clearfix">
	<h2 class="contentheading">
			Keynote speakers		</h2>

	
<div class="article-tools clearfix">
	<ul class="actions">
			<li>
		<a href="#" onclick="window.print();return false;"><span class="icon-print"></span>&#160;Print&#160;</a>		</li>
		
	</ul>



	
</div>
	<p><img src="http://eccb14.loria.fr/images/keynotes/Jean-Marie_Lehn.jpg" width="188" height="232" style="float: left; margin-left: 5px; margin-right: 5px;" /></p>
<p><strong style="font-size: medium; line-height: 1.3em;"><span style="font-size: x-large;">Jean-Marie Lehn</span>   <span style="font-size: small;"><strong><a href="http://www-isis.u-strasbg.fr/supra/start" target="_blank">Home page</a></strong></span></strong></p>
<p><strong style="text-align: right; line-height: 1.3em;"><span style="font-size: small;">University of Strasbourg, Strasbourg, France</span></strong> <br />Nobel Laureate in Chemistry (1987)</p>
<p style="text-align: left;"><span style="font-size: large; line-height: 1.3em;"><a href="http://eccb14.loria.fr/program/keynotes/jean-marie-lehn"><strong><strong><strong><strong>Perspectives in Chemistry: </strong><strong><strong>From Supramolecular Chemistry towards Adaptive Chemistry</strong></strong></strong></strong></strong></a></span></p>
<p style="text-align: justify;"> </p>
<p style="text-align: justify;"><strong>Bio:</strong> Jean-Marie Lehn was born in Rosheim, France in 1939. In 1970 he became Professor of Chemistry at the Université Louis Pasteur in Strasbourg and from 1979 to 2010 he was Professor at the Collège de France in Paris. He is presently Professor Emeritu<span style="line-height: 1.3em;">s at the University of Strasbourg. He shared the Nobel Prize in Chemistry in 1987 for his studies on the chemical basis of “molecular recognition” (i.e. the way in which a receptor molecule recognizes and selectively binds a substrate), which also plays a fundamental role in biological processes.</span> Over the years his work led him to the definition of a new field of chemistry, which he has proposed calling “supramolecular chemistry” as it deals with the complex entities formed by the association of two or more chemical species held together by non-covalent intermolecular forces, whereas molecular chemistry concerns the entities constructed from atoms linked by covalent bonds. Subsequently, the area developed into the chemistry of "self-organization" processes and more recently towards "adaptive chemistry", dynamic networks and complex systems. <br />Author of more than 900 scientific publications, Lehn is a member of many academies and institutions. He has received numerous international honours and awards.</p>
<p style="text-align: justify;"> <span style="line-height: 1.3em;"> </span><span style="line-height: 1.3em;"> </span></p>
<p><img src="http://eccb14.loria.fr/images/keynotes/patrick_aloy.jpg" width="189" height="246" style="line-height: 1.3em; float: left; margin-left: 5px; margin-right: 5px;" /></p>
<p><span style="font-size: x-large;"><strong>Patrick Aloy   <span style="font-size: small;"><strong><a href="http://sbnb.irbbarcelona.org/people/patrick_aloy" target="_blank">Home page</a></strong></span><br /></strong></span></p>
<p class="p1"><span style="font-size: small;"><strong>Institute for Research in Biomedicine, Barcelona, Spain</strong></span> <br /><span style="font-size: small;">ICREA Research Professor</span><span style="line-height: 1.3em;"> </span></p>
<p class="p1"><a href="http://eccb14.loria.fr/program/keynotes/patrick-aloy"><span style="line-height: 1.3em; font-size: large;"><strong>A network biology approach to novel therapeutic strategies</strong></span></a></p>
<p class="p1"> </p>
<p style="text-align: justify;"><strong>Bio:</strong> Born in Barcelona in 1972. He obtained his BSc in Biochemistry (1996), MSc in Biotechnology (1998) and PhD in Biochemistry &amp; Molecular Biology (2000) from the Autonomous University of Barcelona. He then joined the Structural Bioinformatics Group at the European Molecular Biology Laboratory (2001) where he did his postdoctoral training. Since April 2006, Patrick Aloy is an ICREA Research Professor and Principal Investigator of the Structural Bioinformatics Lab in the Institute for Research in Biomedicine (IRB Barcelona). For fifteen years, Dr Aloy has been developing and implementing new technologies and algorithms, applying state-of-the-art methods to specific problems and bridging the gap between theoretical models and experiments in different disciplines. Nowadays, the main goal of the SB&amp;NB lab is to combine computational and structural biology with interaction discovery experiments to unveil the basic wiring architecture of physio-pathological pathways. It is our believe that a deeper knowledge of the global topology of interactome networks related to human disease will have important bearings in the discovery of new drug targets and biomarkers, optimization of preclinical models and understanding how biological networks change from the healthy state to disease.</p>
<p class="p1"><span style="font-size: small;"><strong> </strong></span><strong style="font-size: small; line-height: 1.3em;"> </strong></p>
<p><span style="font-size: x-large;"><strong>Ewan Birney   <span style="font-size: small;"><strong><a href="https://www.ebi.ac.uk/~birney/" target="_blank">Home page</a></strong></span></strong></span><img src="http://eccb14.loria.fr/images/keynotes/EwanBirney-large.jpg" width="191" height="142" style="float: left; margin-left: 5px; margin-right: 5px;" /></p>
<p><span style="font-size: small;"><strong>European Bioinformatics Institute, Hinxton, United Kingdom</strong></span><br /><span style="font-size: small;">Associate Director</span></p>
<p><strong><span style="font-size: small;"><span style="font-size: large;"><a href="http://eccb14.loria.fr/85">Big Data in Biology</a></span> </span></strong></p>
<p style="text-align: justify;"><strong>Bio:</strong><span> </span>Ewan Birney, a British biochemists and bioinformatitian born in 1972, is a leader in the computing and analysis of the genome. <br />After leaving Eton College and before enrolling at the University of Oxford, Birney interned at Cold Spring Harbor Laboratory, under the supervision of James Watson - a discoverer of the structure of DNA. Later, as a PhD student at the Wellcome Trust Sanger Institute in Cambridge, Birney was involved in the Human Genome Project, which was led by James Watson. Within the project, he helped to assembly the genome database. Shortly after receiving his PhD Birney was employed by the EMBL European Bioinformatics Institute in Cambridge, and soon started to coordinate large-scale collaborative projects. Among others, two big public databases resulted from these projects: Ensembl Genome Browser, with vertebrate genomes and gene annotations, and ENCODE (Encyclopedia of DNA Elements), a database of functional elements in the human genome. Ewan Birney and his team also developed a number of open-source and wildly-used bioinformatics tools. For his outstanding achievements, Birney was awarded with various awards such as the Francis Crick Lecture from the Royal Society (2003), Chris Overton Prize from the international Society for Computational Biology (2005) and the Benjamin Franklin Award (2005) for contributions in Open Source Bioinformatics from Bioinformatics.org in association with BioIT. The current research of Birney’s group focuses on sequence algorithms to store digital data in DNA molecules, and on associating natural genome variations with cellular phenotypes. Since 2012, Ewan Birney also serves as an Associate Director of EMBL-EBI. Ewan Birney maintains a blog, Bioinformatician at Large, and is active on Twitter with over 5000 followers.</p>
<p style="text-align: justify;"> </p>
<p class="p1"><img src="http://eccb14.loria.fr/images/keynotes/doron_lancet.jpg" width="190" height="248" style="line-height: 1.3em; float: left; margin-left: 5px; margin-right: 5px;" /></p>
<p class="p1"><strong style="line-height: 1.3em;"><span style="font-size: medium;"> </span></strong><strong style="line-height: 1.3em;"><span style="font-size: x-large;">Doron Lancet</span>   <span style="font-size: small;"><strong><a href="http://www.weizmann.ac.il/molgen/Lancet/home" target="_blank">Home page</a></strong></span></strong></p>
<p class="p1"><span style="font-size: small;"><strong><strong>The Weizmann Institute of Science, Rehovot, Israel</strong></strong></span> <br /><span style="font-size: small;">Professor in the Department of Molecular Genetics, Head of the Crown Human Genome Center</span></p>
<p class="p1"><span style="font-size: large;"><a href="http://eccb14.loria.fr/program/keynotes/doron-lancet"><strong>Rational confederation of genes and diseases</strong></a></span></p>
<p class="p1"> </p>
<p style="text-align: justify;"><strong>Bio:</strong><span> </span>Prof. Lancet earned his BSc in chemistry from the Hebrew University of Jerusalem, and his PhD is in Chemical Immunology from the Weizmann Institute of Science. Following postdoctoral training at Harvard and Yale, he joined the Weizmann Institute faculty in 1981, where he is currently the incumbent of the Silver Professorial Chair in Human Genomics and Head of the Crown Human Genome Center. Prof. Lancet has played a key role in establishing genome research in Israel, and is presently involved in the establishment of a National Center for Personalized Medicine at Weizmann. A pioneer of research on the sense of smell, he discovered key protein components of this sensory mechanism. He now studies the population genetics of human olfaction, showing that each human individual has a “personal nose”.  In parallel, Lancet runs several programs for disease gene discovery, including an international collaborative study on genome sequencing for rare monogenic diseases. In the realm of bioinformatics, Lancet developed <a href="http://www.genecards.org/">GeneCards</a>, a world-renown automatically-mined web compendium of human genes. Recently, he developed the companion database <a href="http://malacards.weizmann.ac.il/">MalaCards</a>, a comprehensive web tool for human diseases. Lancet was awarded the first international Takasago Award in Japan on olfactory research (1986), the Wright Award in the USA (1998) and the Landau Prize in Human Genetics in Israel (2008). He is member of the European Molecular Biology Organization since 1996 and was Council member of the Human Genome Organization (2007-2012). Lancet wrote a science column in the major Israeli daily <em>Haaretz</em>, and often delivers public lectures and interviews on the human genome project.</p>
<p class="p1"><strong style="line-height: 1.3em;"> </strong><strong style="line-height: 1.3em;"> </strong><strong style="line-height: 1.3em;"> </strong></p>
<p><strong><strong><img src="http://eccb14.loria.fr/images/keynotes/nada5.jpg" width="190" height="255" style="float: left; margin-left: 5px; margin-right: 5px;" /></strong></strong></p>
<p class="p1"><span style="font-size: medium;"><strong><strong> </strong></strong></span><strong style="font-size: medium; line-height: 1.3em;"><strong><span style="font-size: x-large;">Nada Lavrač   </span><span style="font-size: small;"><strong><strong><a href="http://www-ai.ijs.si/NadaLavrac/" target="_blank">Home page</a></strong></strong></span></strong></strong></p>
<p class="p1"><span style="font-size: small;"><strong><strong>Jožef Stefan Institute, Ljubljana and University of Nova Gorica, Nova Gorcia, Slovenia</strong></strong></span> <br /><span style="font-size: small;">Professor, Head of Department of Knowledge Technologies</span></p>
<p class="p1"><strong><a href="http://eccb14.loria.fr/program/keynotes/nada-lavrac"><span style="font-size: large;">Advances in data mining for biomedical research</span></a></strong></p>
<p class="p1"><span style="font-size: small;"><strong> </strong></span></p>
<p class="p1" style="text-align: justify;"><strong>Bio:</strong><span> </span>Nada Lavrač is Head of Department of Knowledge Technologies at Jozef Stefan Institute, Ljubljana, Slovenia.  She is also Professor at the Jozef Stefan International Postgraduate School in Ljubljana and at the University of Nova Gorica. Her main research interests are in Knowledge Technologies, with particular interests in machine learning, data mining, text mining, knowledge management and computational creativity. She is author of several books, including the recently published Foundations of Rule Learning, Springer 2012. Her special interest is in supervised descriptive rule induction, where the research goal is to automatically induce rules from class labeled data, stored either in simple tabular format or in complex relational databases. Areas of her applied research include data mining applications in medicine, health care and bioinformatics.</p>
<p class="p1" style="text-align: justify;"> </p>
<p class="p1"><img src="http://eccb14.loria.fr/images/keynotes/alie_mchardy_3.jpg" width="190" height="246" style="float: left; margin-left: 5px; margin-right: 5px;" /></p>
<p class="p1"><strong><span style="font-size: medium;"> </span></strong><strong style="font-size: x-large; line-height: 1.3em;">Alice Carolyn McHardy   <span style="font-size: small;"><strong><a href="http://www.cs.hhu.de/en/research-groups/algorithmic-bioinformatics/our-team/team/mchardy.html" target="_blank">Home page</a></strong></span></strong></p>
<p class="p1"><strong><span style="line-height: 1.3em; font-size: small;">HHU Düsseldorf and <span>Helmholtz Center <strong><span>for Infection Biology in Braunschweig</span></strong><strong><span>, Germany</span></strong></span></span></strong> <br /><span style="line-height: 1.3em; font-size: small;">Chair of the Department for Algorithmic Bioinformatics at HHU, Head of the Computational Biology for Infection Research Group at the Helmholtz Center</span></p>
<p class="p1"><a href="http://eccb14.loria.fr/program/keynotes/alice-carolyn-mchardy"><span style="font-size: large;"><strong>Gaining Insight into the Uncultured Microbial World by Computational Metagenome Analysis</strong></span></a></p>
<p class="p1" style="text-align: justify;"><strong> </strong></p>
<p class="p1" style="text-align: justify;"><strong>Bio:</strong><span> </span>Alice McHardy’s <span>research </span><span>interest is the design of computational methods to solve problems from </span><span>the medical and biotechnological domains using next generation sequence </span><span>data. A particular focus is the study of the evolution of influenza A </span><span>viruses and characterizing the taxonomic composition, functions and </span><span>evolution of microbial communities using metagenome sequence samples. </span><span>She has a PhD in bioinformatics and a master’s degree in biochemistry, </span><span>both from Bielefeld University. From 2005 to 2007 she worked as a </span><span>postdoctoral researcher, then as a permanent staff member in the </span><span>Bioinformatics &amp; Pattern Discovery Group at the IBM T.J. Watson Research </span><span>Center in Yorktown Heights, NY. From 2007 to 2012 she led the Max-Planck </span><span>research group for Computational Genomics &amp; Epidemiology at the </span><span>Max-Planck institute for Informatics in Saarbruecken, Germany.</span></p>
<p class="p1"><strong style="line-height: 1.3em;"><span style="line-height: 1.3em; font-size: small;"> </span></strong></p>
<p class="p1"><strong><span><img src="http://eccb14.loria.fr/images/keynotes/eric_westhof.jpeg" width="190" height="151" style="float: left; margin-left: 5px; margin-right: 5px;" /></span></strong></p>
<p class="p1"><span style="font-size: x-large;"><strong style="line-height: 1.3em;">Eric Westhof  <span style="font-size: small;"><strong><a href="http://www-ibmc.u-strasbg.fr/arn/Westhof/" target="_blank">Home page</a></strong></span> </strong></span></p>
<p class="p1"><strong><span><span style="font-size: small;"><strong><span>University of Strasbourg, Strasbourg, France</span></strong></span></span></strong> <br /><span style="font-size: small;">Professor, Director of the Institute of Molecular and Cellular Biology, CNRS</span></p>
<p class="p1"><strong><span style="font-size: large;"><a href="http://eccb14.loria.fr/program/keynotes/eric-westhof">The Detection of Architectural Modules in RNA sequences and the <em>RNA-Puzzles</em> Modeling Contest</a></span></strong></p>
<p style="text-align: justify;"><strong>Bio:</strong><span> </span>Eric Westhof received his Ph.D. in Biophysics in 1974 (Liège University, Belgium) after graduate work at Regensburg University, Germany. In 1977, as a <em>FULBRIGHT-HAYS</em> Research Fellow, he joined the Department of Biochemistry, University of Wisconsin (Madison, USA) to work with M. Sundaralingam in crystallography of nucleic acids. In 1981, with a <em>EMBO</em> post-doctoral fellowship, he moved to Strasbourg to work with Dino Moras on transfer RNA crystals. In 1988, he became Professor of Structural Biochemistry at the University of Strasbourg. Since 2006, he is Director of the <em>Institut de Biologie Moléculaire et Cellulaire</em> and head of the unit <em>Architecture et Réactivité de l’ARN</em> of the CNRS. He is an executive editor of <em>RNA Journal</em> and <em>Nucleic Acids Research</em> and a member of <em>EMBO</em>, <em>Deutsche Akademie der Naturforscher LEOPOLDINA</em>, <em>Academia Europaea</em>, the <em>Académie des Sciences</em>. <br />His research activities are centered on the relationships between sequences, architectures, evolution and functions of RNA molecules, especially those with catalytic activity. With his collaborators, he develops and studies RNA sequence alignments in the light of RNA structures and architectures in order to identify RNA modules and to develop rules for predicting RNA folds and functions. The rules are transformed into algorithms for manipulating and assembling RNA architectures <em style="line-height: 1.3em;">ab initio</em><span style="line-height: 1.3em;"> or in density maps. The tools used are X-ray crystallography, bioinformatics, sequence comparisons, three-dimensional modeling and molecular dynamics simulations.</span></p>
<p class="p1"><strong><span> </span></strong></p>
<p class="p1"><strong><span style="line-height: 1.3em; font-size: small;"> </span></strong></p>
		

	</div>
</body>


<!-- Mirrored from eccb14.loria.fr/program/keynotes?tmpl=component&print=1&page= by HTTrack Website Copier/3.x [XR&CO'2014], Sun, 12 Jul 2015 15:48:52 GMT -->
</html>