<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD>
<TITLE>ECCB2002 Invited Speakers</TITLE>
<meta name="author" content="Andreas Moll, Marc Sturm">
<meta content="text/html; charset=windows-1252">
<link rel="stylesheet" type="text/css" href="style.css">
</HEAD>
<body bgcolor="White" text="Black" link="Blue" vlink="Navy" alink="#000088">
<br>
<H2>Accepted Papers:<H2>
<br>
<table align=center width=95% border="1">
<tr><td valign=top width=40%>
M. Arita & K. Tsuda</td><td valign=top>	Modeling splicing sites with pairwise correlations
</td></tr>
<tr><td valign=top>
P. Nicodeme, T. Doerks, M. Vingron</td><td valign=top>	Proteome analysis based on motif statistics
</td></tr>
<tr><td valign=top>
K. Ellrott, Ch. Yang, F. Sladek, T. Jiang</td><td valign=top>	Identifying transcription factor binding sites through Markov chain optimization
</td></tr>
<tr><td valign=top>
J. Pomarenko, Ph. Bourne, I. Shindyalov	</td><td valign=top>Building an automated classification of DNA-binding protein domains
</td></tr>
<tr><td valign=top>
M. Salmenkivi, J. Kere, H. Mannila	</td><td valign=top>Genome segmentation using piecewise constant intensity models and reversible jump MCMC
</td></tr>
<tr><td valign=top>
J. Taylor, R. King, Th. Altmann, O. Fiehn	</td><td valign=top>Application of metabolomics to plant genotype discrimination using statistics and machine learning
</td></tr>
<tr><td valign=top>
A. Gambin, S. Lasota, R. Szklarczyk, J. Tiuryn, J. Tyszkiewicz	</td><td valign=top>Contextual alignment of biological sequences
</td></tr>
<tr><td valign=top>
J. Rung, Th. Schlitt, A. Brazma, K. Freivalds, J. Vilo	</td><td valign=top>Building and analysing genome-wide gene disruption networks
</td></tr>
<tr><td valign=top>
R. Steuer, J. Kurths, C. Daub, J. Weise, J. Selbig	</td><td valign=top>The mutual information: Detecting and evaluating dependencies between variables
</td></tr>
<tr><td valign=top>
S. Degroeve, B. De Baets, Y. Van de Peer, P. Rouzé	</td><td valign=top>Feature subset selection for splice site prediction
</td></tr>
<tr><td valign=top>
St. Boué, M. Vingron, I. Koch	</td><td valign=top>Theoretical analysis of alternative splice forms using computational methods
</td></tr>
<tr><td valign=top>
Ho-Youl Jung, Hwan-Gue Cho	</td><td valign=top>An automatic block and spot indexing with k-nearest neighbors graph for microarray image analysis
</td></tr>
<tr><td valign=top>
U. Mückenstein, I. Hofacker, P. F. Stadler	</td><td valign=top>Stochastic pairwise alignments
</td></tr>
<tr><td valign=top>
J. Waldispühl, B. Behzadi, J.-M. Steyaert	</td><td valign=top>An approcimate matching algorithm for finding (sub-)optimal sequences in S-attributed grammars
</td></tr>
<tr><td valign=top>
Gary Benson	</td><td valign=top>A new distance measure for comparing sequence profiles based on path lengths along an entropy surface
</td></tr>
<tr><td valign=top>
E. Althaus, A. Caprara, H.-P. Lenhof, K. Reinert</td><td valign=top>	Multiple sequence alignments with arbitrary gap costs: Computing an optimal solution using polyhedral combinatorics
</td></tr>
<tr><td valign=top>
J. Freudenberg, P. Propping	</td><td valign=top>A similarity-based method for genome-wide prediction of disease-relevant human genes
</td></tr>
<tr><td valign=top>
O. Elemento, O. Gascuel	</td><td valign=top>An efficient and accurate distance based algorithm to reconstruct tandem duplication trees
</td></tr>
<tr><td valign=top>
P. Pipenbacher, A. Schliep, S. Schneckener, A. Schönhuth, D. Schomburg, R. Schrader	</td><td valign=top>ProClust: Improved clustering of protein sequences with an extended graph-based approach
</td></tr>
<tr><td valign=top>
K. Palin, E. Ukkonen, A. Brazma, J. Vilo	</td><td valign=top>Correlating gene promoters and expression in gene disruption experiments
</td></tr>
<tr><td valign=top>
Chr. Dieterich, B. Cusack, H. Wang, K. Rateitschak, A. Krause, M. Vingron	</td><td valign=top>Annotating regulatory DNA based on man-mouse genomic comparison
</td></tr>
<tr><td valign=top>
A. Bazzan, S. Ceroni, L. Schroeder, P. M. Engel	</td><td valign=top>Automated annotation of keywords for proteins related to Mycoplasmataceae using machine learning techniques
</td></tr>
<tr><td valign=top>
J.Gramm, R. Niedermeier	</td><td valign=top>Breakpoint medians and breakpoint phylogenies: A fixed-parameter approach
</td></tr>
<tr><td valign=top>
M. Sirava, T. Schäfer, M. Eiglsperger, M. Kaufmann, O. Kohlbacher, E. Bornberg-Bauer, H.-P. Lenhof	</td><td valign=top>BioMiner-modeling, analysing, and visualizing biochemical pathways and networks
</td></tr>
<tr><td valign=top>
M. Apaydin, C. Guestrin, Ch. Varma, D. Brutlag	</td><td valign=top>Stochastic roadmap simulation for the study of ligand-protein interactions
</td></tr>
<tr><td valign=top>
A. Bergeron, S. Heber, J. Stoye	</td><td valign=top>Common intervals and sorting by reversals: A marriage of necessity
</td></tr>
</table>
</BODY></HTML> 
