<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML>

<HEAD>
  <TITLE>ECCB2002 Posters Abstracts</TITLE>
  <meta name="author" content="Andreas Moll, Marc Sturm">
  <link rel="stylesheet" type="text/css" href="style.css">
  <META content="text/html; charset=windows-1252">
</head>

<body text="000000" bgcolor="FFFFFF" topmargin="1" marginheight="1"><!-- BEGIN WAYBACK TOOLBAR INSERT -->
  <b>
    <font size="4">ECCB 2002 Poster</font>
  </b>
  <hr shade="noshade" size="1">
  <table cellspacing="2" cellpadding="4">
    <tbody>
      <tr>
        <th nowrap="" align="left"><b>
            <font face="Arial">P#</font>
          </b></th>

        <th nowrap="" align="left"><b>
            <font face="Arial">Title and Authors</font>
          </b></th>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">1</font>
        </td>
        <td>
          <font face="Arial">The
              Particle Simulation Toolkit. A toolkit for rapid development of massively parallel particle method
              simulations<br>
              Adalsteinsson, Helgi</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">2</font>
        </td>
        <td>
          <font face="Arial">Analysis
              of the genetic and amino acid sequence diversity of antigenic determinants in the Plasmodium falciparum
              C-terminal merozoite surface protein-1 encoding a 19-kDa protein<br>
              Adoro, Stanley; Nwuba, Roseangela; Anumudu, Chiaka; Omosun, Yusuf; Nwagwu, Mark</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">3</font>
        </td>
        <td>
          <font face="Arial">Toucan:
              A Workbench for Regulatory Sequence Analysis<br>
              Aerts, Stein; Staes, Mik; Coessens, Bert; Thijs, Gert; Moreau, Yves; De Moor, Bart</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">4</font>
        </td>
        <td>
          <font face="Arial">Computational
              analysis of metabolic pathways with the program MING<br>
              Aguilar, Daniel; Oliva, Baldomero; Aviles, Francesc-Xavier; Querol, Enrique</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">5</font>
        </td>
        <td>
          <font face="Arial">Generating
              Rules to Predict P. gingivalis Protein Functional Class from Sequence<br>
              Al-Shahib, A. (1); Gilbert, David (1); Saqi, M. (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">6</font>
        </td>
        <td>
          <font face="Arial">Towards
              a Pattern-Based "Composite Homology Analysis and SEarch" (CHASE) Tool<br>
              Alam, Intikhab; Dress, Andreas; Fuellen, Georg</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">7</font>
        </td>
        <td>
          <font face="Arial">Protein
              Structure and Function Prediction of the Machado-Joseph Disease Gene Product Ataxin-3<br>
              Albrecht, Mario (1); Schnaible, Volker (2); Hoffmann, Daniel (2); Evert, Bernd O. (3); Schmitt, Ina (3);
              Wüllner, Ullrich (3); Lengauer, Thomas (1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">8</font>
        </td>
        <td>
          <font face="Arial">Application
              of Bioinformatics in Plant Genome and Post-Genome Research<br>
              Altschmied, Lothar; Hege, Hans-Christian; Paul, Rolf; Posch, Stefan; Scheel, Dierk</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">9</font>
        </td>
        <td>
          <font face="Arial">A
              classical dynamics study of the unbinding of ATP to the F1-ATPase<br>
              Antes, Iris (1); Oster, George (2); Chandler, David (3)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">10</font>
        </td>
        <td>
          <font face="Arial">Biological
              context and the analysis of gene expression data<br>
              Apostolakis, Joannis; Güttler, Daniel; Sohler, Florian; Zimmer, Ralf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">11</font>
        </td>
        <td>
          <font face="Arial">Sequence
              families of classified enzymes - Correlation with the function<br>
              aus dem Spring, Christian; Schomburg, Dietmar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">12</font>
        </td>
        <td>
          <font face="Arial">Biological
              Significance of Jumping Alignments<br>
              Bannert, Constantin; Stoye, Jens</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">13</font>
        </td>
        <td>
          <font face="Arial">A
              new model for an inhibitor of human cathepsin G by bacterial surface display library screening and
              protein-peptide docking<br>
              Betscheider, Dirk; Ray, Somak; Zangen, Dirk; Schultheiss, Eva; Lengauer, Thomas; Jose, Joachim</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">14</font>
        </td>
        <td>
          <font face="Arial">Molecular
              Class-Specific Information System<br>
              Bettler, Emmanuel; Folkertsma, Simon; Van Durme, Joost; Joosten, Henk-Jan; Fleuren, Wilco; Vriend,
              Gert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">15</font>
        </td>
        <td>
          <font face="Arial">"ab
              initio'' metabolic pathway reconstruction<br>
              Boyer, Frédéric (1,2); Morgat, Anne (1); Trilling, Laurent (2); Viari, Alain (1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">16</font>
        </td>
        <td>
          <font face="Arial">Paracel
              TranscriptAssembler: Identification of Alternative Splice Forms and polyA Sites from EST and mRNA Data<br>
              Boysen, Cecilie; Qian, Jun; Gill, Tristan; Messenger, Richard J.; Mo, Yi; Sievers, Michael; Zhu, Lingyan;
              Borkowski, Joseph A.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">17</font>
        </td>
        <td>
          <font face="Arial">Compact
              Gene Index : Clustering public protein coding sequences for microarray specific oligonucleotide design<br>
              Brett, David; Weber, Jaqueline; Hoerster, Andrea; Bell, Robert; Drescher,Bernd</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">18</font>
        </td>
        <td>
          <font face="Arial">Evolutionary
              analysis of protein domain families<br>
              Bru, Catherine; Kahn, Daniel</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">19</font>
        </td>
        <td>
          <font face="Arial">The
              Center for Bioinformatics (CBI), Saarbrücken<br>
              CBI Consortium</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">20</font>
        </td>
        <td>
          <font face="Arial">Symbolic
              model checking of biological systems<br>
              Chabrier, Nathalie; Fages, François</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">21</font>
        </td>
        <td>
          <font face="Arial">Protein-protein
              docking of cytochrome c2 to the bacterial reaction centre<br>
              Chandran, Vidya; Floeck, Dagmar; Helms, Volkhard</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">22</font>
        </td>
        <td>
          <font face="Arial">Metabolic
              Pathway Alignment and Alternative Pathways Identification<br>
              Chen, Ming</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">23</font>
        </td>
        <td>
          <font face="Arial">Berlin
              Center for Genome Based Bioinformatics (BCB)<br>
              Cordes, Frank(1); Klipp, Edda(2); Koch, Ina(3); Leser, Ulf (4); Meisel, Christian(5); Preissner,
              Robert(6); Spang, Rainer(2); Steinke, Thomas (1); Wiehe, Thomas(7)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">24</font>
        </td>
        <td>
          <font face="Arial">A
              conserved genomic region between vertebrates and insects<br>
              Danchin, Etienne G. J.; Abi-Rached, Laurent; Gilles, André; Pontarotti, Pierre</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">25</font>
        </td>
        <td>
          <font face="Arial">Seqan:
              a modular sequence analysis system<br>
              De Rijk, Peter; Glassée, Wim; Weckx, Stefan; Del-Favero, Jurgen; Van Broeckhoven, Christine</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">26</font>
        </td>
        <td>
          <font face="Arial">Development
              of Annotation Tools in the W3H-Task System<br>
              del Val, Coral; Arunachalam, Vinayagam; Glatting, Karl-Heinz; Suhai, Sándor</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">27</font>
        </td>
        <td>
          <font face="Arial">From
              putative promoter sequence to genomic context : biological data collection on the web using a generic
              application (Xprom).<br>
              Devignes, Marie-Dominique (1); Norsa, Yvan (1); Smaïl, Malika (1); Collet, Philippe (2); Domenjoud, Lionel
              (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">28</font>
        </td>
        <td>
          <font face="Arial">ThoR
              --- A Domain Discovery and Curation Tool<br>
              Dickens, Nicholas J.; Ponting, Chris P.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">29</font>
        </td>
        <td>
          <font face="Arial">Detection
              of degenerated repeated sequences by means of genomic signature in human chromosomes<br>
              Dufraigne, Christine; Fertil, Bernard; Giron, Alain; Deschavanne, Patrick</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">30</font>
        </td>
        <td>
          <font face="Arial">Dynamic
              generation and qualitative analysis of metabolic pathways by a joint database / graph theoretical
              approach<br>
              Ehrentreich, F.; Schomburg, D.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">31</font>
        </td>
        <td>
          <font face="Arial">IMGT/PhyloGene
              : an online software package for phylogenetic analysis of IG and TR genes<br>
              Elemento, Olivier; Lefranc, Marie-Paule</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">32</font>
        </td>
        <td>
          <font face="Arial">Implementation
              of a generic ANOVA model for the analysis of gene expression data<br>
              Engelen, Kristof; Coessens, Bert; Van Hummelen, Paul; De Brabanter, Jos; De Moor, Bart; Marchal,
              Kathleen</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">33</font>
        </td>
        <td>
          <font face="Arial">Bioinformatics
              for the Functional Analysis of Mammalian Genomes (BFAM)<br>
              Fellenberg, Matthias (1); Fries, Ruedi (2); Frishman, Dmitrij (3) et al.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">34</font>
        </td>
        <td>
          <font face="Arial">The
              Lipase Engineering Database<br>
              Fischer, Markus; Henke, Erik; Schmid, Rolf D; Pleiss, Jürgen</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">35</font>
        </td>
        <td>
          <font face="Arial">Iterative
              Blast Machine<br>
              Fleuren, Wilco; Bettler, Emmanuel; Vriend, Gert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">36</font>
        </td>
        <td>
          <font face="Arial">Entropy-variability
              analysis of the nuclear receptor ligand binding domain<br>
              Folkertsma, Simon; Bettler, Emmanuel; van Durme, Joost; Joosten, Henk-Jan; Oliveira, Laerte; de Vlieg,
              Jacob; Vriend, Gert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">37</font>
        </td>
        <td>
          <font face="Arial">Analysis
              of the maximal regulatory region of all human nuclear receptor genes<br>
              Frank, Christian; Dunlop, Thomas W.; Matilainen, Merja; Carlberg, Carsten</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">38</font>
        </td>
        <td>
          <font face="Arial">Object-oriented
              Modelling, Integration and Analysis of Gene Controlled Metabolic Networks<br>
              Freier, Andreas; Hofestädt, Ralf; Lange, Matthias</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">39</font>
        </td>
        <td>
          <font face="Arial">Accurate
              Base Calling from Multiple Electropherograms<br>
              Freschi, Valerio; Bogliolo, Alessandro</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">40</font>
        </td>
        <td>
          <font face="Arial">De
              novo protein design: combination of computational and chemical screening methods<br>
              Fritzemeier, Kai; Renner, Steffen; Drepper, Friedel; Haehnel, Wolfgang</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">41</font>
        </td>
        <td>
          <font face="Arial">A
              comparative study of Amino Acids binary codes<br>
              Fu, Huaiguo; Bouita, Belkhacem; Mephu Nguifo, Engelbert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">42</font>
        </td>
        <td>
          <font face="Arial">Evolution
              of the mitochondrial ancestor proteome<br>
              Gabaldon, Toni; Huynen, Martijn</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">43</font>
        </td>
        <td>
          <font face="Arial">Guided
              Gene Regulation Data Analysis Within the Helmholtz Network for Bioinformatics<br>
              Gailus-Durner, Valérie (1); Gößling, Frank (2); Crass, Torsten (2,*)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">44</font>
        </td>
        <td>
          <font face="Arial">HAMAP:
              High Quality Automated Annotation of Microbial Proteomes<br>
              Gattiker, Alexandre; Coudert, Elisabeth; Michoud, Karine; Rivoire, Catherine; Auchincloss, Andrea; Lima,
              Tania; Lachaize, Corinne; Pagni(1), Marco; Bairoch, Amos</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">45</font>
        </td>
        <td>
          <font face="Arial">Comparative
              DNA Sequence Analysis for the Detection of Regulatory Elements<br>
              Gausmann, Ulrike (1); Jahn, Niels (1); Platzer, Cornelia (2); Szafranski, Karol (1); Platzer, Matthias
              (1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">46</font>
        </td>
        <td>
          <font face="Arial">Integration
              and Distribution of Biological Databases using CORBA<br>
              Gnanasekaran, Thoppae; Mewes, Werner</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">47</font>
        </td>
        <td>
          <font face="Arial">Development
              and Evaluation of a Knowledge-Based-Potential for the Scoring of Protein-Protein-Docking Results<br>
              Grimm, V.; Schomburg, D.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">48</font>
        </td>
        <td>
          <font face="Arial">Simple
              and Efficient Secondary Structure Prediction<br>
              Guimarães, Katia; Melo, Jeane; Cavalcanti, George</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">49</font>
        </td>
        <td>
          <font face="Arial">A
              new model for Sequencing by Hybridization<br>
              Guinand, Frédéric; Mouchard, Laurent; Rabia, Aurélia</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">50</font>
        </td>
        <td>
          <font face="Arial">Integration
              of Multiple Alignment and Phylogeny Reconstruction<br>
              Guinand, Frédéric; Parmentier, Gilles; Trystram, Denis</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">51</font>
        </td>
        <td>
          <font face="Arial">Prediction
              of N-Glycosylation sites in Human Proteins<br>
              Gupta, Ramneek; Jung, Eva; Brunak, Soren</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">52</font>
        </td>
        <td>
          <font face="Arial">Predicting
              "Good" drug targets based on Metabolic Control Analysis<br>
              Gupta, Shobhit</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">53</font>
        </td>
        <td>
          <font face="Arial">Discovery
              of Differentially Expressed Genes using Fuzzy Technology<br>
              Guthke, Reinhard; Scherf, Uwe</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">54</font>
        </td>
        <td>
          <font face="Arial">Expression
              Analysis in Inflammatory Bowel Disease <br>
              Hahn, Andreas(1); Costello, Christine(2); Lu, Tim(2); Lengauer, Thomas(3); Schreiber, Stefan(2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">55</font>
        </td>
        <td>
          <font face="Arial">Co-clustering
              of Biological Networks and Gene Expression Data<br>
              Hanisch, Daniel; Zien, Alexander; Zimmer, Ralf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">56</font>
        </td>
        <td>
          <font face="Arial">Non-Gumbel
              behavior of the statistics of local sequence alignment<br>
              Hartmann, Alexander K.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">57</font>
        </td>
        <td>
          <font face="Arial">Automatic
              construction and simulation of Petri nets for a whole cell<br>
              Hartmann, Kai; Schomburg, Dietmar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">58</font>
        </td>
        <td>
          <font face="Arial">Time
              series microarray data analysis using ISA grouping biologically related functions<br>
              Hei Jin, Kim</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">59</font>
        </td>
        <td>
          <font face="Arial">The
              Helmholtz Network for Bioinformatics - A User-friendly Integrated Bioinformatics Web Portal<br>
              HNB Consortium</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">60</font>
        </td>
        <td>
          <font face="Arial">Gene
              expression profiles in disease: From gene expression in isolated cells to gene expression in whole tissues
              and vice versa<br>
              Hoffmann, Martin (1); Pohlers, Dirk (2); Kinne, Raimund W.(2); Kroll, Torsten (3); Wölfl, Stefan (3)</a>
          </font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">61</font>
        </td>
        <td>
          <font face="Arial">Supervised
              Classification of Pathological States in Gene and Protein Expression Data <br>
              Hoffmann, Martin; Lau, Stephan; von Eggeling, Ferdinand; Junker, Kerstin; Guthke, Reinhard</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">62</font>
        </td>
        <td>
          <font face="Arial">Mining
              the literature for enzyme-disease associations<br>
              Hofmann, Oliver; Schomburg, Dietmar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">63</font>
        </td>
        <td>
          <font face="Arial">Prediction
              of Protein Thermostability<br>
              Hoppe, C.; Schomburg, D.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">64</font>
        </td>
        <td>
          <font face="Arial">Linguistic
              Complexity Profiles of Prokaryotic Genomic Sequences Assist in Detection and Classification of
              Terminators<br>
              Hosid, Sergey; Bolshoy, Alexander</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">65</font>
        </td>
        <td>
          <font face="Arial">XX_frag:
              design of cDNA sequences for micro-arrays application to comparative Genomic Hybridization<br>
              Hubans, Christine; Kerckaert, Jean Pierre; Van Hoecke, Marie Pierre</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">66</font>
        </td>
        <td>
          <font face="Arial">XX_frag:
              design of cDNA sequences for microarrays-application to CGH<br>
              Hubans, Christine; Kerkaert, Jean Pierre; Van Hoecke, Marie Pierre</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">67</font>
        </td>
        <td>
          <font face="Arial">Variance
              stabilization applied to stochastic modeling of DNA microarray measurements, calibration, and the
              quantification of differential expression<br>
              Huber, Wolfgang (1); von Heydebreck, Anja (2); Sueltmann, Holger(1); Poustka, Annemarie (1); Vingron,
              Martin (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">68</font>
        </td>
        <td>
          <font face="Arial">matriXarray
              platform: Workflow for Experimental Planning, Data Extraction, Data Interpretation and Data Handling of
              Microarray Experiments<br>
              Hurlebaus, Jochen (1); Renzing, Jochen(1); Fellenberg, Matthias(2); Beichele, Sebastian(2); Anderson,
              Brooke P.(3); Frey, Bruno(1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">69</font>
        </td>
        <td>
          <font face="Arial">E.coli
              Heat Shock Response Control Mechanism in Silicon Analysis<br>
              Hu, Bin; Matsuzaiki,Yuri; Tomita, Masaru; Hiroki, Kitano</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">70</font>
        </td>
        <td>
          <font face="Arial">Improved
              gene selection for classification of microarrays <br>
              Jaeger, Jochen; Sengupta, Rimli; Ruzzo, Walter L.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">71</font>
        </td>
        <td>
          <font face="Arial">Automated
              Model Builder (AMB) for nuclear receptors<br>
              Joosten, Henk-Jan; Bettler, Emmanuel; Vriend, Gert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">72</font>
        </td>
        <td>
          <font face="Arial">IMGT/3Dstructure-DB
              for immunoglobulin, T cell receptor and MHC structural data<br>
              Kaas, Quentin; Lefranc, Marie-Paule</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">73</font>
        </td>
        <td>
          <font face="Arial">SBEprimer:
              Primer Design for Multiplexed Genotyping<br>
              Kaderali, Lars; Deshpande, Alina; White, P. Scott</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">74</font>
        </td>
        <td>
          <font face="Arial">Mapping
              sequence properties to biological classification by machine learning algorithms<br>
              Kanapin, Alexander; Soinov, Lev; Krestyaninova, Maria</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">75</font>
        </td>
        <td>
          <font face="Arial">Identification
              of ORFs from Organelle Genomes: A Data Mining approach using C4.5 Machine Learning Algorithm<br>
              Kannan, Sivakumar; Boucher, Genevieve; Burger, Gertraud</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">76</font>
        </td>
        <td>
          <font face="Arial">Automatic
              construction of genetic networks using artificial neural networks and natural language processing<br>
              Karopka, Thomas; Scheel, Thomas; Bansemer, Sven; Glass, Änne</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">77</font>
        </td>
        <td>
          <font face="Arial">Finding
              and Characterizing Novel Cancer Related Genes in Genomic Sequences Using IBM Life Sciences Framework<br>
              Kelkar, Bhooshan P.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">78</font>
        </td>
        <td>
          <font face="Arial">Optimized
              recognition of errors in protein structures using knowledge based potentials: general findings and
              application to ProSAII<br>
              Kienberger, Ferry; Wiederstein, Markus; Lackner, Peter; Sippl, Manfred J.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">79</font>
        </td>
        <td>
          <font face="Arial">The
              Effects of the Transcription Factor on Binding Site Information are Constrained by Genetic Autonomy<br>
              Kim, Jan T.; Martinetz, Thomas; Polani, Daniel</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">80</font>
        </td>
        <td>
          <font face="Arial">Kinetic
              modeling of metabolic changes in Down Syndrome<br>
              Klipp, Edda</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">81</font>
        </td>
        <td>
          <font face="Arial">Integration
              and Visualisation of Experimental Data<br>
              Knäble, Hella; Ratsch, Esther; Kania, Renate; Wittig, Ulrike; Rojas, Isabel</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">82</font>
        </td>
        <td>
          <font face="Arial">Process
              Management for the Chip Fabrication<br>
              Kokocinski,Felix; Wrobel,Gunnar; Lichter,Peter</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">83</font>
        </td>
        <td>
          <font face="Arial">Doublet
              frequency maps of DNA sequences: A tool for quantitative visualization<br>
              of segment homologies<br>
              Köhler, Michael (1); Wölfl, Stefan (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">84</font>
        </td>
        <td>
          <font face="Arial">The
              SYSTERS Protein Family Web Server<br>
              Krause, Antje; Meinel, Thomas; Stoye, Jens; Schmidt, Heiko A.; Luz, Hannes; Vingron, Martin</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">85</font>
        </td>
        <td>
          <font face="Arial">A
              2nd Order Polynomial Normalization for Competitive Microarray Experiments<br>
              Kroll, Torsten; Wölfl, Stefan</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">86</font>
        </td>
        <td>
          <font face="Arial">Analysis
              of the lipopolysaccharide (LPS)-stimulated innate immune response with TRANSPATH®<br>
              Krull, Mathias; Choi, Claudia; Pistor, Susanne; Voss, Nico; Wingender, Edgar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">87</font>
        </td>
        <td>
          <font face="Arial">CodeProbe
              - coding frame detection tool<br>
              Kumanduri, Vasudev; Schweizer, Patrick</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">88</font>
        </td>
        <td>
          <font face="Arial">Algebraic
              Concepts for Data Domains in Life Science<br>
              Lange, Matthias(1); Köhler, Jacob(2); Schweizer, Patrick(1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">89</font>
        </td>
        <td>
          <font face="Arial">Tools
              for comparative Genome Analysis by low redundant sequencing<br>
              Lehmann, Rüdiger; Romualdi, Alessandro; Siddiqui, Roman; Szafranski, Karol; Glöckner, Gernot; Sühnel,
              Jürgen; Platzer, Matthias</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">90</font>
        </td>
        <td>
          <font face="Arial">Gene
              duplications at the base of chordate ancestry<br>
              Leveugle, Magalie; Prat, Karine; Coulier, François; Birnbaum, Daniel</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">91</font>
        </td>
        <td>
          <font face="Arial">Methylation
              analysis by direct sequencing of PCR products<br>
              Lewin, Jörn; Schmitt, Armin O.; Genc, Bülent; Hildmann, Thomas; Novic, Karen L.; Beck, Stephan</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">92</font>
        </td>
        <td>
          <font face="Arial">A
              linear theory of optimal regulation relates gene expression to function<br>
              Liebermeister, Wolfram; Schuster, Stefan; Heinrich, Reinhard</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">93</font>
        </td>
        <td>
          <font face="Arial">Comparative
              genomics study of inverted repeats in bacteria<br>
              Lillo, Fabrizio; Basile, Salvatore; Mantegna,, Rosario</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">94</font>
        </td>
        <td>
          <font face="Arial">Research
              and teaching at the Cologne University Bioinformatics Center (CUBIC)* Molecular Networks in Organisms<br>
              Lohmann, Mark; Schomburg, Dietmar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">95</font>
        </td>
        <td>
          <font face="Arial">Gene
              Ontology Driven Classification of Gene Expression Patterns<br>
              Lottaz, Claudio; Bentink, Stefan; Spang, Rainer</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">96</font>
        </td>
        <td>
          <font face="Arial">A
              hidden Markov model for progressive multiple alignment<br>
              Löytynoja, Ari; Milinkovitch, Michel</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">97</font>
        </td>
        <td>
          <font face="Arial">Interpreting
              Patterns of Codon Usage in Bacterial Genomes<br>
              Mahony, Shaun; Golden, Aaron; McInerney, James</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">98</font>
        </td>
        <td>
          <font face="Arial">Structural
              Infectomics: Identification and Characterization of Potential Virulence Factors in Legionella
              pneumophila<br>
              Mann, Martin; Heyne, Steffen; Pal, Debnath; Baharuddin, Aida; Vogel, Andre'; Hilgenfeld, Rolf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">99</font>
        </td>
        <td>
          <font face="Arial">Trying
              to reconstruct the history of genes families<br>
              Marangoni, Roberto; Pisanti, Nadia; Ferragina, Paolo; Frangioni, Antonio; Luccio, Fabrizio</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">100</font>
        </td>
        <td>
          <font face="Arial">A
              Statistical Learning Approach to Predicting Protein-DNA-Binding Sites<br>
              Martinetz, Thomas; Gewehr, Jan E.; Kim, Jan T.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">101</font>
        </td>
        <td>
          <font face="Arial">Development
              and application of functional surface representations of proteins using GRID: A simple but effective
              post-filter for geometric rigid body protein-protein docking<br>
              Martin, Oliver; Wohlfahrt, Gerd; Schomburg, Dietmar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">102</font>
        </td>
        <td>
          <font face="Arial">A
              Graph-based Pathway Searching System over a Signal Transduction Database<br>
              May, Gerhard H.W.; Gilbert, David</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">103</font>
        </td>
        <td>
          <font face="Arial">Analysis
              of the hierarchical structure of large scale metabolic networks constructed from genome data<br>
              Ma, H.-W.; Aeng, A.-P.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">104</font>
        </td>
        <td>
          <font face="Arial">"VIRTUAL2D"
              - A web accessible predictive database for proteomic analysis<br>
              Medjahed, Djamel; Smythers, Gary; Powel,l Douglas; Lemkin, Peter; Munroe, David</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">105</font>
        </td>
        <td>
          <font face="Arial">Self-association
              of isoguanine nucleobases and molecular recognition of alkaline ions: Tetrad versus pentad structures <br>
              Meyer, M.; Suehnel, J.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">106</font>
        </td>
        <td>
          <font face="Arial">TSM,
              the TRANSFAC Saccharomyces Module<br>
              Michael, Holger; Christensen, Maik; Goessling, Ellen; Wingender, Edgar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">107</font>
        </td>
        <td>
          <font face="Arial">GeneRule:
              A Tool for Extraction and Validation of Knowledge for Rule - Based Expert Systems<br>
              Monossov, Vladimir; Guthke, Reinhard</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">108</font>
        </td>
        <td>
          <font face="Arial">Biclustering
              microarray data by Gibbs sampling<br>
              Moreau, Yves; Qizheng, Sheng; De Moor, Bart</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">109</font>
        </td>
        <td>
          <font face="Arial">4DiCeS:
              A Workbench for the Simulation of Intracellular Signalling<br>
              Möller, Mark; Oleson, Björn E.; Prank, Klaus</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">110</font>
        </td>
        <td>
          <font face="Arial">Advantages
              of a global optimization approach for the cluster analysis of gene expression data<br>
              Möller, Ulrich; Thies, Frank</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">111</font>
        </td>
        <td>
          <font face="Arial">PRODORIC:
              Prokaryotic Database Of Gene Regulation<br>
              Münch, Richard; Hiller, Karsten; Schobert, Max; Wingender, Edgar; Jahn, Dieter</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">112</font>
        </td>
        <td>
          <font face="Arial">Genomic
              Object Net in JAVA: A Platform for Biopathway Modeling and Simulation<br>
              Nagasaki, Masao; Doi, Atsushi; Sasaki, Makiko; Savoie, Christopher J.; Matsuno, Hiroshi; Miyano,
              Satoru</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">113</font>
        </td>
        <td>
          <font face="Arial">ElMaR:
              A Protein Docking System using Flexibility Information<br>
              Neumann, Steffen; Zoellner, Frank; Koch, Kerstin; Kummert, Franz; Sagerer, Gerhard</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">114</font>
        </td>
        <td>
          <font face="Arial">ASPIC
              Application-Service-Providing for integrated simulations and visualization in science and engineering on
              high performance pc-clusters<br>
              Niedworok, Sebastian; Nollert, Hans-Peter</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">115</font>
        </td>
        <td>
          <font face="Arial">A
              new method of finding similarity regions in DNA sequences<br>
              Noé, Laurent; Kucherov, Gregory</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">116</font>
        </td>
        <td>
          <font face="Arial">4DiCeS:
              Simulating Diffusion of Signalling Molecules within a Cell<br>
              Oleson, Björn E.; Möller, Mark; Prank, Klaus</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">117</font>
        </td>
        <td>
          <font face="Arial">The
              nest/egg motif in proteins - a comprehensive structural bioinformatics study<br>
              Pal, Debnath; Suehnel, Juergen; Weiss, Manfred S.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">118</font>
        </td>
        <td>
          <font face="Arial">Hidden
              messages in the nef gene of HIV-1suggest a novel RNA secondary structure<br>
              Peleg, Ofer; Trifonov, Edward; Bolshoy, Alexander</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">119</font>
        </td>
        <td>
          <font face="Arial">SWISS-PROT
              Format Developments<br>
              Phan, Isabelle; Redaschi, Nicole; Roland, Pascal; Runte, Kai; Jain, Eric; Bairoch, Amos</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">120</font>
        </td>
        <td>
          <font face="Arial">A
              New Database on Alternative Splice Forms<br>
              Pospisil, Heike; Herrmann, Alexander; Reich, Jens</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">121</font>
        </td>
        <td>
          <font face="Arial">Modeling
              the Architecture of Signal Transduction Networks: Insulin Receptor Signaling Pathways<br>
              Potapov, Anatolij; Wingender, Edgar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">122</font>
        </td>
        <td>
          <font face="Arial">Detection
              of a Sequence of Manipulated Experiments for Bayesian Learning<br>
              Pournara, Iosifina; Wernisch, Lorenz</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">123</font>
        </td>
        <td>
          <font face="Arial">PACS:
              A novel methodology and graphical representation for understanding protein structure from an engineer's
              perspective<br>
              Prasad, B.V.L.S.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">124</font>
        </td>
        <td>
          <font face="Arial">WILMA
              - a platform for the annotation of protein sequences<br>
              Prlic, Andreas; Domingues, Francisco S.; Lackner, Peter; Sippl, Manfred J.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">125</font>
        </td>
        <td>
          <font face="Arial">The
              Proteome Analysis Database in 2002<br>
              Pruess, Manuela; Kanapin, Alexander; Karavidopoulou, Youla; Kersey, Paul; Kriventseva, Evgenia; Mittard,
              Virginie; Mulder, Nicola; Phan, Isabelle; Apweiler, Rolf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">126</font>
        </td>
        <td>
          <font face="Arial">Hybrid
              clustering for microarray image analysis combining intensity and shape features<br>
              Rahnenfueher, Joerg</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">127</font>
        </td>
        <td>
          <font face="Arial">The
              Jena Centre for Bioinformatics <br>
              Rahn, Kerstin; Suehnel, Juergen</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">128</font>
        </td>
        <td>
          <font face="Arial">Statistical
              analysis of consistent repeating patterns in a set of 16S rDNA sequences<br>
              Raje,D.V.; Purohit, H.J.; Singh, R.N.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">129</font>
        </td>
        <td>
          <font face="Arial">Unifying
              Data Mining Tools for Automated Annotation on TrEMBL<br>
              Rakow, Astrid; Hackmann, Andre; Kretschmann, Ernst; O'Rourke, John; Apweiler, Rolf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">130</font>
        </td>
        <td>
          <font face="Arial">Improved
              Prediction of RNA Secondary Structures Including Pseudoknots<br>
              Reeder, Jens; Giegerich, Robert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">131</font>
        </td>
        <td>
          <font face="Arial">Using
              a Biochemical Pathways Database for Deriving Transition-State Drug Analogs<br>
              Reitz, Martin; Sacher, Oliver; Gasteiger ,Johann</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">132</font>
        </td>
        <td>
          <font face="Arial">The
              moss transcriptome<br>
              Rensing, Stefan; Lang, Daniel; Reski, Ralf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">133</font>
        </td>
        <td>
          <font face="Arial">Inferring
              Coarse-Grained Models of Regulatory Gene Networks from Dynamic Expression Data<br>
              Repsilber, Dirk; Kim, Jan T.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">134</font>
        </td>
        <td>
          <font face="Arial">Reversible
              scaling of dihedral angle barriers during molecular dynamics to improve structure prediction of cyclic
              peptides and protein loops <br>
              Riemann, Nico; Zacharias, Martin</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">135</font>
        </td>
        <td>
          <font face="Arial">Probability
              occurrence of structured motifs and application to candidates promoters<br>
              Robin, Stéphane; Daudin, Jean-Jacques; Richard, Hugues; Sagot, Marie-France; Schbath, Sophie</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">136</font>
        </td>
        <td>
          <font face="Arial">Building
              a Protein Structure Workbench with PyMOL<br>
              Rother, Kristian; Froemmel ,C.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">137</font>
        </td>
        <td>
          <font face="Arial">Comparing
              Whole Genomes with the Artemis Comparison Tool (ACT)<br>
              Rutherford, Kim</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">138</font>
        </td>
        <td>
          <font face="Arial">Feature
              Subset Selection for Splice Site Prediction by Estimation of Distribution Algorithms<br>
              Saeys, Yvan; Degroeve, Sven; Aeyels, Dirk; Van de Peer, Yves; Rouzé, Pierre</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">139</font>
        </td>
        <td>
          <font face="Arial">Bilateral
              inverse symmetry in whole bacterial chromosomes<br>
              Sanchez, Joaquin (1); Jose Marco, V. (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">140</font>
        </td>
        <td>
          <font face="Arial">Analysis
              of topological representations of transcriptional regulatory regions<br>
              Sand, Olivier; Vu, Tien Dung; Gilbert, David; Viksna, Juris</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">141</font>
        </td>
        <td>
          <font face="Arial">Searching
              for biologically meaningful groupings in microarray clustering data<br>
              Schacherer, Frank; Scheel, Hartmut; Hofmann, Kay</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">142</font>
        </td>
        <td>
          <font face="Arial">Data
              mining for protein interaction domains<br>
              Scheel, Hartmut; Hofmann, Kay</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">143</font>
        </td>
        <td>
          <font face="Arial">TRANSFAC®
              Database on Eukaryotic Gene Transcription Regulation: Innovations and Improvements in Content, Structure
              and Bioinformatic Tools<br>
              Scheer, Maurice P.; Matys, Volker; Fricke, Ellen; Land, Sigrid; Thiele, Susanne; Michael, Holger; Gößling,
              Ellen; Hornischer, Klaus; Reuter, Ingmar; Kel, Alexander E.; Kel-Margoulis, Olga V.; Wingender, Edgar</a>
          </font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">144</font>
        </td>
        <td>
          <font face="Arial">GHMM
              &amp; HMMed: A comprehensive HMM toolkit<br>
              Schliep, Alexander</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">145</font>
        </td>
        <td>
          <font face="Arial">A
              Novel Binary Clustering Algorithm to Analyze Gene Expression Time Series<br>
              Schmidt-Heck, Wolfgang; Guthke, Reinhard; Reischer, Helga; Dürrschmid, Karin; Bayer, Karl</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">146</font>
        </td>
        <td>
          <font face="Arial">A
              Structural Model for NDH-2: Membrane Interaction and Function<br>
              Schmid, Ralf; Gerloff, Dietlind L.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">147</font>
        </td>
        <td>
          <font face="Arial">Hidden
              Markov Model based clustering of gene expression data<br>
              Schoenhuth, Alexander; Schliep, Alexander; Mueller, Tobias; Steinhoff, Christine</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">148</font>
        </td>
        <td>
          <font face="Arial">BRENDA
              - the Enzyme and Metabolic Information System<br>
              Schomburg, D.; Schomburg, I.; Chang, A.; Hofmann, O.; Ebeling, C.; Ehrentreich, F.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">149</font>
        </td>
        <td>
          <font face="Arial">Interface
              targetring inhibitors of HIV-1 Protease<br>
              Schramm, H.J.; König, S.; Büttner, J.; Schramm, W.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">150</font>
        </td>
        <td>
          <font face="Arial">Composing
              a promoter model for antibacterial response of epithelial cells<br>
              Shelest, Ekaterina; Kel, Alexander; Gößling, Ellen; Wingender, Edgar</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">151</font>
        </td>
        <td>
          <font face="Arial">Noise
              in gene expression systems with monomer and dimer autoregulation<br>
              Shibata, Tatsuo; Mikhailov, Alexander S.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">152</font>
        </td>
        <td>
          <font face="Arial">Multiple
              Local Sequence Structure Alignment of RNA Sequences<br>
              Siebert, Sven</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">153</font>
        </td>
        <td>
          <font face="Arial">Efficient
              Stochastic Simulation of Intracellular Signalling<br>
              Singh, Kapil; Prank, Klaus</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">154</font>
        </td>
        <td>
          <font face="Arial">Predicting
              structure and function by comparing local sequence motifs<br>
              Sitbon, Einat; Pietrokovski, Shmuel</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">155</font>
        </td>
        <td>
          <font face="Arial">Discovery
              of protein complexes in the network of protein interactions<br>
              Spirin, Victor; Zhao, Dacheng; Mirny, Leonid</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">156</font>
        </td>
        <td>
          <font face="Arial">BLASTing
              Proteomes, Yielding Phylogenies<br>
              Spitzer, Michael(1); Lorkowski, Stefan(2); Cullen, Paul(3); Fuellen, Georg(1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">157</font>
        </td>
        <td>
          <font face="Arial">Clustering
              of differentially expressed genes in the human genome<br>
              Steigele, Stephan; Dietzsch, Janko; Khaitovich, Philipp; Huson, Daniel; Pääbo, Svante; Nieselt-Struwe,
              Kay</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">158</font>
        </td>
        <td>
          <font face="Arial">Comparison
              of Various Normalisation Strategies for Microarray Analysis <br>
              Steinhoff, C.; Nuber, U.A.; Vingron, Martin</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">159</font>
        </td>
        <td>
          <font face="Arial">Modeling
              the influence of feedback loops on the G1/S transition<br>
              Swat, Maciej; Kel, Alexander; Kel-Margoulis, Olga; Deineko, Oleg; Herzel, Hanspeter</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">160</font>
        </td>
        <td>
          <font face="Arial">Structure
              prediction of protein segments with low target/template sequence identity based on a reduced protein
              model<br>
              Szymoszek, Andrzej; Zacharias, Martin</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">161</font>
        </td>
        <td>
          <font face="Arial">AGenDA:
              A WWW Server for Gene Recognition by Comparative Sequence Analysis<br>
              Taher, Leila (1); Rinner, Oliver (2); Garg, Saurabh (1); Scyrba, Alexander (3); Brudno, Mike (4);
              Batzouglou, Serafim (5); Morgenstern, Burkhard (1)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">162</font>
        </td>
        <td>
          <font face="Arial">Hierarchical
              Machine Learning and Knowledge Discovery in Characterising Protein Families<br>
              Tan, Aik Choon; Gilbert, David</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">163</font>
        </td>
        <td>
          <font face="Arial">Evaluation
              of methods for the searching RNA motifs task<br/>
              Thebault, Patricia; Allouche, David; Gaspin, Christine</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">164</font>
        </td>
        <td>
          <font face="Arial">Predicting
              protein-protein interactions for the C. elegans interactome project<br>
              Thierry-Mieg, Nicolas</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">165</font>
        </td>
        <td>
          <font face="Arial">MotifScanner:
              a novel probabilistic approach to screen DNA sequences for predefined regulatory elements <br>
              Thijs, Gert; Aerts, Stein; Moreau, Yves; Marchal, Kathleen; De Moor, Bart</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">166</font>
        </td>
        <td>
          <font face="Arial">Exploiting
              weak sequence similarities for protein functional prediction<br>
              Tomiuk, Stefan; Scheel, Hartmut; Hofmann, Kay</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">167</font>
        </td>
        <td>
          <font face="Arial">ProToGO
              - evaluating biological features for a set of proteins using GO annotations<br>
              Ulanovsky, Hagit (1); Ron, Shany (1); Kifer, Ilona (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">168</font>
        </td>
        <td>
          <font face="Arial">Nuclear
              receptor information management: application with a nuclear receptor mutation database<br>
              van Durme, Joost; Bettler, Emmanuel; Folkertsma, Simon; Vriend, Gert</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">169</font>
        </td>
        <td>
          <font face="Arial">Incorporating
              Sequence and Biochemical Information in Topological Models of Protein Structure Towards the Structural and
              Functional Genomics<br>
              Veeramalai, Mallika (1); Gilbert, David (1); Westhead, David (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">170</font>
        </td>
        <td>
          <font face="Arial">Similarity
              based approach to protein domain architecture prediction<br>
              Vlahovicek, Kristian; Kajan, Laszlo; Pongor, Sandor</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">171</font>
        </td>
        <td>
          <font face="Arial">Homology
              modeling and molecular dynamics simulation of the human androgen receptor ligand binding domain<br>
              von Langen, Johannes; Diekmann, Stephan; Hillisch, Alexander</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">172</font>
        </td>
        <td>
          <font face="Arial">Oligonucleotide
              Selection for Molecular Taxa Classification via Microarray Technology Using Special Types of Multiple
              Sequence Alignments<br>
              von Rohr, Peter (1); Roth, Alexander (1); Frey, Jürgen (2)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">173</font>
        </td>
        <td>
          <font face="Arial">Alignments
              of sequences with repeats<br>
              Wagner, Holger</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">174</font>
        </td>
        <td>
          <font face="Arial">An
              Approach to Support the Comparison of Microbial Genomic DNA Sequences with Spatial Knowledge of Genomic
              Structures<br>
              Wetjen, Tom</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">175</font>
        </td>
        <td>
          <font face="Arial">The
              Influence of Environment on Horizontal Gene Transfer<br>
              Whitehead, Dion; Dress, Andreas</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">176</font>
        </td>
        <td>
          <font face="Arial">Molecular
              dynamics simulations on the free and complexed SHP2-SH2 domain<br>
              Wieligmann, Karin; Zacharias, Martin</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">177</font>
        </td>
        <td>
          <font face="Arial">Quantitative
              analysis of the large-scale organization of the protein-protein interaction network in yeast<br>
              Wilhelm, Thomas; Nasheuer, Heinz-Peter; Suehnel, Juergen</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">178</font>
        </td>
        <td>
          <font face="Arial">TrEMBL
              protein sequence database: a guide to the proteomic world<br>
              Williams, Allyson; Martin, Maria Jesus; O'Donovan, Claire; Barrell, Daniel; Fedotov, Alexander; Apweiler,
              Rolf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">179</font>
        </td>
        <td>
          <font face="Arial">Protein
              Structure Prediction In The Face-Centered-Cubic Lattice<br>
              Will, Sebastian; Backofen, Rolf</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">180</font>
        </td>
        <td>
          <font face="Arial">ZIGIA-DB:
              A Public Database to Access Arabidopsis thaliana Knockout Plants<br>
              Wingen, Luzie U.; Saedler, Heinz; Dekker, Koen</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">181</font>
        </td>
        <td>
          <font face="Arial">Choosing
              transition prior to maximize accuracy in homologue searches using Hidden Markov Models<br>
              Wistrand, Markus; Sonnhammer, Erik L.L</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">182</font>
        </td>
        <td>
          <font face="Arial">Applying
              Data Mining Methods to SELDI-TOF Analysed Renal Cell Carcinoma Samples to Identify Relevant Tumor
              Markers<br>
              Woetzel, Dirk; Driesch, Dominik; Pfaff, Michael; von Eggeling, Ferdinand; Junker, Kerstin; Guthke,
              Reinhard</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">183</font>
        </td>
        <td>
          <font face="Arial">SNP
              Database Integration Project<br>
              Wong, Marie; Choo, Keng Wah; Liu, Jianjun</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">184</font>
        </td>
        <td>
          <font face="Arial">Towards
              standardized microarray experiments: complete data management <br>
              Wölfl, Stefan; Kube, Steffen</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">185</font>
        </td>
        <td>
          <font face="Arial">rChip:
              Documenting microarray data analysis<br>
              Wrobel, Gunnar; Radlwimmer, Bernhard; Lichter, Peter</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">186</font>
        </td>
        <td>
          <font face="Arial">Information
              Management of Zebrafish Full-length cDNA project<br>
              Yuyu, Kuang; Oh, Tania; Mathavan, S.; Le Ber, Pierre Louis; Dowts, Heidi; Kolatkar, Prasanna R.</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">187</font>
        </td>
        <td>
          <font face="Arial">SigNetRouter
              - Computer Program for Detecting the Routes between Factors and Targets in Signaling Networks<br>
              Zevedei-Oancea, Ionela; Schuster, Stefan</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">188</font>
        </td>
        <td>
          <font face="Arial">Prediction
              of transcription factors that activate promoters with E2F in a synergistic manner<br>
              Zhong, Sheng; Wong, Wing Hung</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">189</font>
        </td>
        <td>
          <font face="Arial">Quantitative
              Characterization and Scaling of Human Signal Transduction Networks <br>
              Ziagos, Sotirios (1); Hoffmann, Martin (2); Wilhelm, Thomas (3)</font>
        </td>
      </tr>

      <tr valign="top">
        <td>
          <font face="Arial">190</font>
        </td>
        <td>
          <font face="Arial">Calculating
              Residue Flexibility Information from Statistics and Energy based Prediction<br>
              Zoellner, Frank; Koch, Kerstin; Neumann, Steffen; Kummert, Franz; Sagerer, Gerhard</font>
        </td>
      </tr>
    </tbody>
  </table>




</body>

</HTML>