<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD>
<TITLE>ECCB2002 Invited Speakers</TITLE>
<meta name="author" content="Andreas Moll, Marc Sturm">
<meta charset="UTF-8">
<link rel="stylesheet" type="text/css" href="style.css">
</HEAD>
<body bgcolor="White" text="Black" link="Blue" vlink="Navy" alink="#000088">
<center>
<h2>Program Schedule:</h2>
<table border="0" width="80%">
<tbody><tr><td>
<br>
<font size="+2"><b> Sunday, October 6, 2002</b></font>
<table border="1" cellpadding="3" cellspacing="4" width="100%">
	<tbody><tr><td width="100">
	<b>18.00 - 20.00</b>
	</td><td bgcolor="#B4E07F">
	<b>Registration and Reception</b>
	</td></tr>
</tbody></table>
<br>
<br>
<font size="+2"><b> Monday, October 7, 2002</b></font>
<table border="1" cellpadding="3" cellspacing="4" width="100%">
	<tbody><tr><td width="100">
	<b>08:30</b>
	</td><td bgcolor="#B4E07F">
	<b>Opening: Thomas Lengauer</b>
	</td></tr>
	<tr><td valign="top">
	<b>08:40</b>
	</td><td bgcolor="#B4E07F">
	<b>Welcome Address: Jürgen Schreier, Minister of Education, Culture and Science</b>
	</td></tr>
	
	<tr><td valign="top">
	<b>08:50</b>
	</td><td bgcolor="#B4E07F">
	<b>Welcome Address: Dr. jur. Hartwig Cremers, Chancellor of Saarland University</b>
	</td></tr>	
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Thomas Lengauer, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>09:00</b>
	</td><td bgcolor="#D0E07F">
	Selecting targets for therapeutic validation through differential protein expression using chromatography-mass spectrometry<br>
        <i>Scott Patterson, Celera, Rockville, United States</i>
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Protein networks</b>,<i> Soren Brunak , Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>09:50</b>
	</td><td bgcolor="#9BBEDD">
	Building and analysing genome-wide gene disruption networks
	<br><i>J. Rung, Th. Schlitt, A. Brazma, K. Freivalds, J. Vilo</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>10:15</b>
	</td><td bgcolor="#9BBEDD">
	BioMiner-modeling, analyzing, and visualizing biochemical pathways and networks
	<br><i>M. Sirava, T. Schäfer, M. Eiglsperger, M. Kaufmann, O. Kohlbacher, E. Bornberg-Bauer, H.-P. Lenhof</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>10:40</b>
	</td><td>
	Coffee Break
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Expression data</b>,<i> Soren Brunak , Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:00</b>
	</td><td bgcolor="#9BBEDD">
	An automatic block and spot indexing with k-nearest neighbors graph for microarray image analysis
	<br><i>Ho-Youl Jung, Hwan-Gue Cho</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:25</b>
	</td><td bgcolor="#9BBEDD">
	The mutual information: Detecting and evaluating dependencies between variables
	<br><i>R. Steuer, J. Kurths, C. Daub, J. Weise, J. Selbig</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:50</b>
	</td><td bgcolor="#9BBEDD">
	Correlating gene promoters and expression in gene disruption experiments
	<br><i>K. Palin, E. Ukkonen, A. Brazma, J. Vilo</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>12:15</b>
	</td><td>
	Lunch Break
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Alvis Brazma, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>13:50</b>
	</td><td bgcolor="#D0E07F">
	Biological information: making it accessible and integrated (and trying to make sense of it)<br>
        <i>Tim Hubbard, Sanger Institute, Cambridgeshire, United Kingdom</i>
	</td></tr>
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: DNA sequence analysis</b>, <i>Knut Reinert, Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>14:40</b>
	</td><td bgcolor="#9BBEDD">
	Modeling splicing sites with pairwise correlations
	<br><i>M. Arita &amp; K. Tsuda</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>15:05</b>
	</td><td bgcolor="#9BBEDD">
	Identifying transcription factor binding sites through Markov chain optimization
	<br><i>K. Ellrott, Ch. Yang, F. Sladek, T. Jiang</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>15:30</b>
	</td><td>
	Coffee Break
	</td></tr>
	
	<tr><td valign="top">
	<b>15:50</b>
	</td><td bgcolor="#9BBEDD">
	Feature subset selection for splice site prediction
	<br><i>S. Degroeve, B. De Baets, Y. Van de Peer, P. Rouzé</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>16:15</b>
	</td><td bgcolor="#9BBEDD">
	Theoretical analysis of alternative splice forms using computational methods
	<br><i>St. Boué, M. Vingron, I. Koch</i>
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Martin Vingron, Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>16:40</b>
	</td><td bgcolor="#D0E07F">
	Estonian Genome Project-before the take-off and take-off
	<br><i>Andres Metspalu</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>17:30</b>
	</td><td bgcolor="#B4E07F">
	<b>Posters</b>
	</td></tr>
	
	<tr><td valign="top">
	<b>19:00 - 20:00</b>
	</td><td bgcolor="#B4E07F">
	<b>Business Meeting</b>
	</td></tr>	
</tbody></table>
<br>
<br>
<font size="+2"><b> Tuesday, October 8, 2002</b></font>
<table border="1" cellpadding="3" cellspacing="4" width="100%">
	<tbody><tr><td valign="top" width="100">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Martin Vingron, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>08:30</b>
	</td><td bgcolor="#D0E07F">
	Comparative genomics of microbial pathogens and symbionts<br>
        <i>Siv Andersson, Uppsala University, Uppsala, Sweden</i>
	</td></tr>
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Sequence algorithms</b>, <i>David Gilbert , Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>09:20</b>
	</td><td bgcolor="#9BBEDD">
	Multiple sequence alignments with arbitrary gap costs: Computing an optimal solution using polyhedral combinatorics
	<br><i>E. Althaus, A. Caprara, H.-P. Lenhof, K. Reinert</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>09:45</b>
	</td><td bgcolor="#9BBEDD">
	An approximate matching algorithm for finding (sub-)optimal sequences in S-attributed grammars
	<br><i>J. Waldispühl, B. Behzadi, J.-M. Steyaert</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>10:10</b>
	</td><td bgcolor="#9BBEDD">
	Contextual alignment of biological sequences
	<br><i>A. Gambin, S. Lasota, R. Szklarczyk, J. Tiuryn, J. Tyszkiewicz</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>10:35</b>
	</td><td>
	Coffee Break
	</td></tr>
	
	<tr><td valign="top">
	<b>11:00</b>
	</td><td bgcolor="#9BBEDD">
	Stochastic pairwise alignments
	<br><i>U. Mückenstein, I. Hofacker, P. F. Stadler</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:25</b>
	</td><td bgcolor="#9BBEDD">
	A new distance measure for comparing sequence profiles based on path lengths along an entropy surface
	<br><i>Gary Benson</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:50</b>
	</td><td bgcolor="#9BBEDD">
	Common intervals and sorting by reversals: A marriage of necessity
	<br><i>A. Bergeron, S. Heber, J. Stoye</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>12:15</b>
	</td><td>
	Lunch Break
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Marie-France Sagot, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>14:00</b>
	</td><td bgcolor="#D0E07F">
	Detecting genomic features under weak selective pressure: the example of codon usage in animals and plants
        <br><i>Laurent Duret, Laboratoire de Biométrie et Biologie Évolutive, Lyon, France</i>
	</td></tr>
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Phylogeny</b>, <i>Marie-France Sagot, Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>14:50</b>
	</td><td bgcolor="#9BBEDD">
	Breakpoint medians and breakpoint phylogenies: A fixed-parameter approach
	<br><i>J.Gramm, R. Niedermeier</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>15:15</b>
	</td><td bgcolor="#9BBEDD">
	An efficient and accurate distance based algorithm to reconstruct tandem duplication trees
	<br><i>O. Elemento, O. Gascuel</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>15:40</b>
	</td><td>
	Coffee Break
	</td></tr>
	
	<tr><td valign="top">
	<b>16:00</b>
	</td><td bgcolor="#B4E07F">
	<b>Excursion</b>
	</td></tr>
</tbody></table>
<br>
<br>
<font size="+2"><b> Wednesday, October 9, 2002</b></font>
<table border="1" cellpadding="3" cellspacing="4" width="100%">
	<tbody><tr><td valign="top" width="100">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Thomas Lengauer, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>08:30</b>
	</td><td bgcolor="#D0E07F">
	Using robotics to fold proteins and dock ligands<br>
        <i>Douglas L. Brutlag, Department of Biochemistry, Stanford University, USA</i>
	</td></tr>
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Gene and protein function</b>, <i>Ralf Zimmer , Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>9:20</b>
	</td><td bgcolor="#9BBEDD">
	Stochastic roadmap simulation for the study of ligand-protein interactions
	<br><i>M. Apaydin, C. Guestrin, Ch. Varma, D. Brutlag</i>
	</td></tr>	
	
	<tr><td valign="top">
	<b>9:45</b>
	</td><td bgcolor="#9BBEDD">
	Application of metabolomics to plant genotype discrimination using statistics and machine learning
	<br><i>J. Taylor, R. King, Th. Altmann, O. Fiehn</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>10:10</b>
	</td><td bgcolor="#9BBEDD">
	A similarity-based method for genome-wide prediction of disease-relevant human genes
	<br><i>J. Freudenberg, P. Propping</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>10:35</b>
	</td><td>
	Coffee Break
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Protein classification</b>, <i>Ralf Zimmer , Chair</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:00</b>
	</td><td bgcolor="#9BBEDD">
	Building an automated classification of DNA-binding protein domains
	<br><i>J. Pomarenko, Ph. Bourne, I. Shindyalov</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:25</b>
	</td><td bgcolor="#9BBEDD">
	ProClust: Improved clustering of protein sequences with an extended graph-based approach
	<br><i>P. Pipenbacher, A. Schliep, S. Schneckener, A. Schönhuth, D. Schomburg, R. Schrader</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>11:50</b>
	</td><td bgcolor="#9BBEDD">
	Automated annotation of keywords for proteins related to Mycoplasmataceae using machine learning techniques
	<br><i>A. Bazzan, S. Ceroni, L. Schroeder, P. M. Engel</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>12:15</b>
	</td><td>
	Lunch Break
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Esko Ukkonen, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>14:00</b>
	</td><td bgcolor="#D0E07F">
	Principles of protein-protein interactions
        <br><i>Sarah A. Teichmann, MRC Laboratory of Molecular Biology, Cambridge, United Kingdom</i>
	</td></tr>
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#9BBEDD">
	<b>Session: Genome analysis</b>, <i>Alfonso Valencia, Chair</i>
	</td></tr>
	<tr><td valign="top">
	<b>14:50</b>
	</td><td bgcolor="#9BBEDD">
	Genome segmentation using piecewise constant intensity models and reversible jump MCMC
	<br><i>M. Salmenkivi, J. Kere, H. Mannila</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>15:15</b>
	</td><td bgcolor="#9BBEDD">
	Annotating regulatory DNA based on man-mouse genomic comparison
	<br><i>Chr. Dieterich, B. Cusack, H. Wang, K. Rateitschak, A. Krause, M. Vingron</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>15:40</b>
	</td><td>
	Coffee Break
	</td></tr>
	
	<tr><td valign="top">
	<b>16:00</b>
	</td><td bgcolor="#9BBEDD">
	Proteome analysis based on motif statistics
	<br><i>P. Nicodeme, T. Doerks, M. Vingron</i>
	</td></tr>
	
	<tr><td valign="top">
	<b></b>
	</td><td bgcolor="#D0E07F">
	<b>Invited talk:</b> <i>Alfonso Valencia, Chair</i>
	</td></tr>
	
	
	<tr><td valign="top">
	<b>16:25</b>
	</td><td bgcolor="#D0E07F">
	Comparative analysis of protein interaction networks<br>
        <i>Peer Bork, EMBL, Heidelberg, Germany</i>
	</td></tr>
	
	<tr><td valign="top">
	<b>17:15</b>
	</td><td bgcolor="#B4E07F">
	<b>Posters</b>
	</td></tr>
		
	<tr><td valign="top">
	<b>18:45</b>
	</td><td bgcolor="#B4E07F">
	<b>End</b>
	</td></tr>
		
	
</tbody></table>
</td>
</tr>
</tbody></table>
</center>
 
</body>
</HTML> 
