 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><!-- #BeginTemplate "/Templates/modelepage2.dwt" -->
<head>
<!-- #BeginEditable "doctitle" --> 
<title>Scientific Program</title>
<!-- #EndEditable --> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<meta name="author" content="INRA -Comité d'organisation ECCB 2003">
<style type="text/css">                           
<!--
.titre {font-family:"Arial, sans serif"; color:yellow; font-size: 26px;}
.baratin {font-family:"Arial, sans serif"; color:yellow; font-size: 16px;}
#ssmenu{position:absolute; z-index:10;}           
.MenuPrinc{font-family:"Arial, sans serif"; color:black; font-size: 16px;text-decoration:none;}  
.SsMenu{font-family:" Arial, sans serif"; color:black; font-size:14px;text-decoration:none;}         
A:hover.MenuPrinc{color:"orange"; text-decoration:none;}            
//-->
</style>
</head>
<body bgcolor="#343A90">
<div id="entete" style="position:absolute; visibility:visible; top:0px; left:80px; width:750px; z-index:2; ">
  <table border=0 cellspacing="1" cellpadding="0" align="center" width="100%">
    <tr> 
      <td align="center" ><a href="index.htm"><img src="Templates/images/logo22-11.gif" width="210" height="130" border=0 alt="Retour page accueil"></a></td>
      <td align="center" ><a href="javascript:if(confirm('http://www.cite-sciences.fr/english/indexFLASH.htm  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.cite-sciences.fr/english/indexFLASH.htm'"><img src="Templates/images/lavillette.jpg" width="200" height="130" border=0 alt="Cité des Sciences - La Villette"></a></td>
      <td align="center" ><img src="Templates/images/notredame.jpg" width="180" height="130" border=0 alt="Notre-Dame de Paris"></td>
      <td align="center" ><img src="Templates/images/louvre.jpg" width="170" height="130" border=0 alt="Le Louvre"></td>
    </tr>
  </table>
</div>
<div id="barremenu" style="position:absolute; visibility:visible; height:25px; width:600px; z-index:10;">
<table bgcolor="#01274C" border=0 cellspacing="0" cellpadding="0" >
  <tr>
    <td>
       <table border=0 width="100%">
         <tr>
            <td align="center" bgcolor="#9999FF" width=110 height=25 > 
              <A class="MenuPrinc" href="#" onMouseOver="affiche_Menu(tab[0],2);return false">ECCB'2003 </A></td>
          </tr>
       </table> 
    </td>
    <td>
       <table border=0   width="100%">
         <tr>
            <td align="center" bgcolor="#66FF66" width=110 height=25> 
              <A class="MenuPrinc" href="#" onMouseOver="affiche_Menu(tab[1],115);return false">Program</A></td>
          </tr>
        </table>
    </td>
 <td>
       <table border=0  width="100%">
         <tr>
            <td align="center" bgcolor="#FFFF99" width=110  height=25> 
               <A class="MenuPrinc" href="#" onMouseOver="affiche_Menu(tab[2],230);return false">Call 
              for papers</A> </td>
          </tr>
        </table>
    </td>
 <td>
       <table border=0  width="100%">
         <tr>
            <td align="center"  bgcolor="#FFFFFF" width=110  height=25> 
              <A class="MenuPrinc" href="#" onMouseOver="affiche_Menu(tab[3],347);return false">Registration</A> 
            </td>
          </tr>
        </table>
    </td>
 <td>
       <table border=0  width="100%">
         <tr>
            <td align="center" bgcolor="#9999FF" width=110  height=25> 
               <A class="MenuPrinc" href="#" onMouseOver="affiche_Menu(tab[4],460);return false">Practical 
              Info</A> </td>
          </tr>
        </table>
    </td>
   </tr>
</table>
</div>
<div style="position:absolute; top:190px; left:80px; width:780px; z-index:2;">
<!-- #BeginEditable "truc" --> 
  <table width="100%" border="0" cellpadding="10" bgcolor="#FFFFCC">
    <tr> 
      <td colspan="2"> 
        <div align="center"> 
          <p><font size="5" face="Arial, Helvetica, sans-serif"><b>ECCB'2003 European 
            Conference on Computational Biology<br>
            </b></font><font face="Arial, Helvetica, sans-serif" size="3"><b>September 
            27-30, 2003, Paris, FRANCE</b></font></p>
          <p><b><font size="4" face="Arial, Helvetica, sans-serif">SCIENTIFIC 
            PROGRAM</font></b><br>
            <font face="Arial, Helvetica, sans-serif" size="3"><br>
            <font size="2"><b> <a href="invspeak.htm">Invited papers (abstracts),</a><br>
            Selected papers (long papers, short papers and flash): click <a href="accepap.htm">here</a>,</b><br>
            <b><a href="posters..htm">ECCB'2003: posters abstracts</a></b><br>
            <font color="#FF0000"><b><font color="#333333">Final program</font> 
            <a href="progdef.pdf"> in PDF</a></b></font></font></font></p>
          <hr>
        </div>
        <p align="left"><font face="Arial, Helvetica, sans-serif" size="3" color="#0000CC"><b><font color="#993399"><a name="saturday"></a></font></b></font><font face="Arial, Helvetica, sans-serif" size="3" color="#0000CC"><b><font color="#993399">Saturday 
          September 27&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="2"><b><font face="Arial, Helvetica, sans-serif" size="3"><b><font size="2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sunday">sunday 
          28th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#monday">monday 
          29th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tuesday">tuesday 
          30th</a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b><a href="progched.htm"><img src="images/retour.gif" width="29" height="29" border="0"></a></b></font></font></b></font><br>
          <font size="3" color="#993399"> </font></b></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"> 
            9h30 Registration opens, welcome coffee</font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2" color="#0000FF"><i><b><font face="Comic Sans MS">12h-13h 
            Light buffet </font></b></i></font></font></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">13h15-13h30 
            Welcoming speeches </font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">13h30-14h20 
            <b><font color="#33CC99">Invited talk<i> <br>
            Chair: Shoshana Wodak</i></font><i><br>
            Thomas Lengauer</i> </b><i>&nbsp;&nbsp; </i>Analyzing resistance phenomena 
            in HIV with bioinformatics methods </font></font></li>
        </ul>
        <font face="Arial, Helvetica, sans-serif" size="3"><font size="2" color="#00FF00"><b><font color="#FF0000">Contributed 
        talks - Genetic networks </font></b></font></font> <br>
        <font color="#FF0000"><i><b>Chair: David Gilbert</b></i></font> 
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">14h20-14h45 
            A description of dynamical graphs associated to elementary regulatory 
            circuits<font size="3" face="Times New Roman, Times, serif"></font><i><font size="3" face="Times New Roman, Times, serif"> 
            </font></i></font></font><br>
            <font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><i>Elisabeth 
            Rémy, Brigitte Mossé, Claudine Chaouiya, Denis Thieffry</i></font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"> 
            14h45-15h00 Using ChIP data to decipher regulatory logic of MBF and 
            SBF during the Yeast cell cycle <br>
            <i>Feng Gao, Harmen Bussemaker</i></font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">15h00-15h15 
            Validation of noisy dynamical system models of gene regulation inferred 
            from time-course gene expression data at arbitrary time intervals<i> 
            Michiel de Hoon, Sascha Ott, Seiya Imoto, Satoru Miyano </i></font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">15h15-15h30 
            Accuracy of the models for gene regulation - a comparison of two modeling 
            methods<br>
            <i>Kimmo Palin</i> </font></font></li>
        </ul>
        <ul>
          <li><font size="3" face="Times New Roman, Times, serif"><i><font color="#0000FF"><b><font face="Comic Sans MS" size="2">15h30-16h10 
            Break</font></b></font></i></font><font face="Comic Sans MS" size="2"><i><font color="#0000FF"><b> 
            </b></font></i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="3"><font size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Genetic networks and gene expression</font><br>
          <i><font color="#FF0000">Chair: St&eacute;phane Robin</font></i></b></font></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">16h10-16h35 
            Extracting active pathways from gene expression data<br>
            <i>Jean-Philippe Vert, Minoru Kanehisa</i> </font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">16h35-17h00 
            Gene networks inference using dynamic Bayesian networks<br>
            <i>Bruno-Edouard Perrin, Liva Ralaivola, Florence d'Alché-Buc, Samuele 
            Bottani, Aurélien Mazurie </i></font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">17h00-17h25 
            Estimating gene networks from gene expression data by combining Bayesian 
            network model with promoter element detection <br>
            <i>Yoshinori Tamada, Sunyong Kim, Hideo Bannai, Seiya Imoto, Kousuke 
            Tashiro, Satoru Kuhara, Satoru Miyano </i></font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">17h25-17h50 
            Biclustering microarray data by Gibbs sampling<br>
            <i>Qizheng Sheng, Yves Moreau, Bart De Moor </i></font></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="3"><font size="2">17h50-18h15 
            Discover significant rules for classifying cancer Diagnosis Data<br>
            <i>Jinyan Li, Huiqing Liu, See-Kiong Ng, Limsoon Wong</i></font></font><font face="Arial, Helvetica, sans-serif" size="2"><br>
            </font></li>
        </ul>
        <ul>
          <li><font color="#0000FF"><b><i><font face="Comic Sans MS" size="2">19h-22h 
            Cocktail at &quot;Le Palais de la Découverte&quot; </font><font face="Times New Roman, Times, serif" size="3"><br>
            </font></i></b><font face="Times New Roman, Times, serif" size="3"><i><font color="#000000"><b><font face="Comic Sans MS" size="2">(Avenue 
            Franklin Roosevelt near the Champs Elysées)</font><br>
            </b></font></i></font></font> <font face="Times New Roman, Times, serif" size="3" color="#0000FF"><b><font color="#993399"><a name="sunday"></a></font></b></font> 
            <hr>
            <font color="#0000FF"><i><font face="Arial, Helvetica, sans-serif" size="2"><font size="1" color="#000000"><b> 
            </b></font></font></i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#0000FF"><b><font size="3" color="#993399">Sunday 
          September 28&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="2"><b><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#saturday">saturday 
          27th</a></b></font><b><font size="2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#monday">monday 
          29th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tuesday">tuesday 
          30th</a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b><a href="progched.htm"><img src="images/retour.gif" width="29" height="29" border="0"></a></b></font></font></b></font><br>
          <font size="3" color="#993399"> </font></b></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2"> 9h-9h50<font color="#FF0000"> 
            <b><font color="#33CC99">Invited talk</font></b></font><font color="#33CC99"><b><br>
            <i>Chair: Satoru Miyano</i></b></font><b><i><br>
            Nir Friedman</i></b> &nbsp;&nbsp;&nbsp;Models for identifying regulation 
            networks </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Gene expression and motifs</font></b></font><font face="Arial, Helvetica, sans-serif" size="2"><br>
          <i><b><font color="#FF0000">Chair: Alvis Brazma</font></b></i></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">9h50-10h15 The 
            shortest common supersequence problem in a microarray production setting<br>
            <i>Sven Rahmann </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h15-10h40 Exploring 
            potential target genes of signaling pathways by predicting conserved 
            transcription factor binding sites<br>
            <i>Christoph Dieterich, Ralf Herwig, Martin Vingron </i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#CC66CC">Flash 
          presentations - Genetic networks and gene expression</font><br>
          <font color="#CC66CC"><i>Chair: Alvis Brazma</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h40-10h45 Graph 
            theory based methodology for comparing interaction networks between 
            genes across organisms<br>
            <i>Gaëlle Lelandais, Pierre Vincens, Claude Jacq, Stéphane Vialette</i> 
            </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h45-10h50 Spotting 
            effects in cDNA experiments<br>
            <i>T. Mary-Huard, S. Robin, J.-J. Daudin</i> </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h50-10h55 Design 
            of a clinical microarray chip<br>
            <i>Jochen Jäger, Rainer Spang </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h55-11h00 Classification 
            of cancers by gene expression profiles from peripheral blood<br>
            <i>Andrey Loboda, Michael Nebozhyn, Steven W. Johnson, Peter J. O'Dwyer, 
            Calen Nichols, Linda Alila, Louise C. Showe, Michael K. Showe</i> 
            </font></li>
        </ul>
        <ul>
          <li><b><i><font face="Comic Sans MS" size="2" color="#0000FF">11h00-11h45 
            Posters and break</font></i></b></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Gene expression and motifs<br>
          <i>Chair: Philipp Bucher</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">11h45-12h00 Modular 
            decision system and information integration for improved disease outcome 
            prediction<br>
            <i>Matthias Futschik, Mi Sullivan, Anthony Reeve, Nikola Kasabov</i> 
            </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h00-12h15 Genomic 
            distribution of short motifs involved in DNA repair in pathogenic 
            and non pathogenic Escherichia coli <br>
            <i>Isabelle Bourgait, Hélène Chiapello, Christelle Hennequet-Antier, 
            Stéphane Robin, Sophie Schbath, Alexandra Gruss, Meriem El Karoui 
            </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h15-12h40 Searching 
            for statistically significant regulatory modules<br>
            <i>Timothy Bailey, William Noble </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h40-13h05 Computational 
            detection of cis-regulatory modules <br>
            <i>Stein Aerts, Peter Van Loo, Gert Thijs, Yves Moreau, Bart De Moor 
            </i></font></li>
        </ul>
        <ul>
          <li><font color="#0000FF"><i><b><font face="Comic Sans MS" size="2">13h05-14h30 
            Lunch </font></b></i></font></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">14h30-15h20 <b><font color="#33CC99">Invited 
            talk<br>
            <i>Chair: James McInerney</i></font><i><br>
            Hiroshi Akashi</i></b> &nbsp;&nbsp;Metabolic economics and microbial 
            proteome evolution </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2"><b><font color="#FF0000">Contributed 
          talks - Gene expression and motifs<br>
          <i>Chair: Michal Linial</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">15h20-15h45 Finding 
            subtle motifs by branching from sample strings<br>
            <i>Alkes Price, Sriram Ramabhadran, Pavel Pevzner </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">15h45-16h10 Finding 
            optimal degenerate patterns in DNA sequences<br>
            <i>Daisuke Shinozaki, Tatsuya Akutsu, Osamu Maruyama</i> </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#CC66CC">Flash 
          presentations - Motifs, protein-protein interactions and proteomics<br>
          <i>Chair: Michal Linial</i></font></b> </font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h10-16h15 Automatic 
            procedures for compilation of promoter sequences and their evaluation 
            based on signal content and positional distributions <i>Christoph 
            S. Schmid, Viviane Praz, Mauro Delorenzi, Rouaida Perier, Philipp 
            Bucher </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h15-16h20 Modeling 
            the Rho dependent transcription termination sites in the bacterium 
            Helicobacter pylori<br>
            <i>Lisa Petersen, Anders Krogh </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h20-16h25 GENOFRAG: 
            A software to design primers optimized for whole genome scanning by 
            long-range PCR amplification. Application to the study of Staphylococcus 
            aureaus genome plasticity<br>
            <i>Nouri Ben Zakour, Michel Gautier, Rumen Andonov, Dominique Lavenier, 
            Philippe Veber, Alexeï Sorokin, Yves Le Loir </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h25-16h30 ISYMOD: 
            A Knowledge Base for integrated system modeling<br>
            <i>Julie Chabalier, Yves Quentin, Cécile Capponi, Gwennaele Fichant 
            </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h30-16h35 PARIS: 
            a system for 2-DE based proteomic data management<br>
            <i>Juhui Wang, Christophe Caron, Michel-Yves Mistou, Christophe Gitton, 
            Alain Trubuil </i></font></li>
        </ul>
        <ul>
          <li><i><font face="Comic Sans MS" size="2" color="#0000FF"><b>16h35-17h15 
            Posters and break</b></font><b><font face="Times New Roman, Times, serif" size="3" color="#0000FF"> 
            </font></b></i></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Proteomics and repeats<br>
          <i>Chair: Graziano Pesole</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">17h15-17h40<i> 
            </i>A Suffix Tree approach to the interpretation of tandem mass spectra: 
            Applications to peptides of nonspecific digestion and post-translational 
            modifications <i>Bingwen Lu, Ting Chen</i> </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">17h40-18h05 Inferring 
            strengths of protein-protein interactions from experimental data using 
            linear programming<br>
            <i>Morihiro Hayashida, Nobuhisa Ueda, Tatsuya Akutsu </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">18h05-18h20 Overview 
            of a software pipeline dedicated to automatic MS/MS data analysis<br>
            <i>Erwan Reguer, Estelle Nugues, Romain Cahuzac </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">18h20-18h35 Resources 
            for bacterial strain identification Using polymorphic tandem repeats<br>
            <i>France Denoeud, Gilles Vergnaud </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">18h35-18h50 Detecting 
            short inverted segments in a biological sequence<i><br>
            David Robelin, Bernard Prum</i></font></li>
        </ul>
        <ul>
          <li><font color="#0000FF"><i><b><font face="Comic Sans MS" size="2">19h30-dawn 
            Gala Diner</font><font face="Times New Roman, Times, serif" size="3"> 
            </font><font face="Arial, Helvetica, sans-serif" size="2"><br>
            </font></b></i></font> <font face="Arial, Helvetica, sans-serif" size="2"><font color="#993399"><b><font face="Arial, Helvetica, sans-serif" size="2"><font face="Arial, Helvetica, sans-serif" size="2"><font color="#993399"><b><font face="Arial, Helvetica, sans-serif" size="2"><font color="#993399"><b><font size="3"><a name="monday"></a></font></b></font></font></b></font></font><font color="#993399"><b></b></font></font></b></font></font> 
            <hr>
            <font color="#0000FF"><i><b><font face="Arial, Helvetica, sans-serif" size="2"> 
            </font></b></i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2"><font color="#993399"><b><font face="Arial, Helvetica, sans-serif" size="2"><font color="#993399"><b></b></font></font><font size="3">Monday 
          September 29&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="2"><b><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#saturday">saturday 
          27th</a></b></font><b><font size="2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sunday">sunday 
          28th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tuesday">tuesday 
          30th</a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b><a href="progched.htm"><img src="images/retour.gif" width="29" height="29" border="0"></a></b></font></font></b></font><br>
          <font size="3" color="#993399"> </font></b></font><font size="3"> </font></b></font></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2"> 9h-9h50 <b><font color="#33CC99">Invited 
            talk<br>
            <i>Chair: Anna tramontano</i></font><i><br>
            Janet Thornton</i></b> &nbsp;&nbsp;The proteome and the metabolome 
            </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Metabolic networks and RNAs<br>
          <i>Chair: Martin Vingron</i></font></b></font><font face="Arial, Helvetica, sans-serif" size="2"> 
          </font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">9h50-10h15 Ab 
            initio reconstruction of metabolic pathways<i><br>
            Frédéric Boyer, Alain Viari</i> </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h15-10h40 A 
            rapid method for detection of putative RNAi target genes in genomic 
            data<i><br>
            Yair Horesh, Amihood Amir, Shulamit Michaeli, Ron Unger </i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#CC66CC">Flash 
          presentations - Systems biology and RNAs<br>
          <i>Chair: Martin Vingron</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h40-10h45 The 
            Biochemical Abstract Machine BIOCHAM<br>
            <i>Nathalie Chabrier, François Fages </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h45-10h50 Relevance 
            of the secondary structure of messenger RNA: a statistical study<br>
            <i>Fabrice Thalmann, Hervé Isambert</i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2"> 10h50-10h55 
            Searching for ncRNAs in protist genomes<i><br>
            Lesley J. Collins, Thomas J. Macke, David Penny </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h55-11h00 Computational 
            detection of MicroRNAs in animal genomes<i><br>
            Matthieu Legendre, Daniel Gautheret </i></font></li>
        </ul>
        <ul>
          <li><i><b><font face="Comic Sans MS" size="2" color="#0000FF">11h-11h45 
            Posters and break </font></b></i></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - RNAs and gene finding<br>
          <i>Chair: S<font size="1">&Oslash;</font>ren Brunak</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">11h45-12h00 Annotating 
            animal mitochondrial tRNAs: an experimental evaluation of four methods<br>
            <i>Stacia Wyman, Jeffrey Boore </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h00-12h15 Searching 
            the Saccharomyces cerevisiae genome for -1 frameshifting sites<br>
            <i>Michaël Bekaert, Jean-Paul Forest, Laure Bidou, Alain Denise, Guillemette 
            Duchateau-Nguyen, Céline Fabret, Christine Froidevaux, Isabelle Hatin, 
            Jean-Pierre Rousset, Michel Termier </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h15-12h40 Fast 
            feature selection using a simple Estimation of Distribution Algorithm 
            : A case study on splice site prediction&nbsp;<i>Yvan Saeys, Sven 
            Degroeve, Dirk Aeyels, Yves Van de Peer, Pierre Rouzé </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h40-13h05 HMM 
            sampling and applications to gene finding and alternative splicing<br>
            <i>Simon Cawley, Lior Pachter</i></font></li>
        </ul>
        <ul>
          <li><font color="#0000FF"><i><b><font face="Comic Sans MS" size="2">13h05-14h30 
            Lunch</font><font face="Times New Roman, Times, serif" size="3"> </font></b></i></font></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">14h30-15h20<font color="#CC0000"> 
            <b><font color="#33CC99">Invited talk<br>
            <i>Chair: Daniel Gautheret</i></font></b></font><b><i><br>
            Jürgen Brosius</i></b> How significant is 98.5% "junk" in mammalian 
            genomes ? </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Sequence analysis and gene finding<br>
          <i>Chair: Roderic Guig<font size="1">&Oacute;</font></i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">15h20-15h45 Modelling 
            sequencing errors by combining Hidden Markov Models<br>
            <i>Claudio Lottaz, Christian Iseli, C. Victor Jongeneel, Philipp Bucher 
            </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">15h45-16h10 Gene 
            prediction with a Hidden Markov Model and new intron submodel <br>
            <i>Mario Stanke, Stephan Waack </i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#CC66CC">Flash 
          presentations - Functional genomics<br>
          <i>Chair: Roderic Guig<font size="1">&Oacute;</font></i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h10-16h15 Compositional 
            analysis of non-coding regions in eukaryotic genomes<br>
            <i> Emanuele Bultrini, Paolo Del Giudice, Elisabetta Pizzi </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h15-16h20 Inferring 
            site-specific evolutionary rates: Bayesian methods are superior<br>
            <i>Itay Mayrose, Dan Graur, Tal Pupko </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h20-16h25 TIGERSearch 
            attacks Proteins<i><br>
            Jasmin Saric, Uwe Reyle </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h25-16h30 Protein 
            function prediction: Application of a propositional rules learning 
            system to a set of human protein sequences<i>&nbsp;&nbsp;Manuel J. 
            Gómez, Francisco Javier Guijarro, Ramón P. Otero, Lars J. Jensen, 
            Søren Brunak, Alfonso Valencia</i></font></li>
        </ul>
        <ul>
          <li><i><b><font face="Comic Sans MS" size="2" color="#0000FF">16h30-17h15 
            Posters and break </font></b></i></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">17h15-18h05<font color="#33CC99"> 
            <b>Invited talk<br>
            <i>Chair: Esko Ukkonen</i></b></font><b><i><br>
            Pedro Mendes</i></b> &nbsp;&nbsp;Simulation meets data analysis: the 
            development of objective tests for functional genomics data analysis 
            algorithms </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Functional genomics and evolution<br>
          <i>Chair: Alfonso Valencia</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">18h05-18h30 Predicting 
            gene function in Saccharomyces cerevisiae<br>
            <i>Amanda Clare, Ross King </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">18h30-18h45 Rules 
            extraction in linkage disequilibrium mapping with an adaptive genetic 
            algorithm<i><br>
            Laetitia Jourdan, Clarisse Dhaenens, El-Ghazali Talb </i></font></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2" color="#0000FF"><b><i><font face="Comic Sans MS">18h50-19h50 
            Business meeting at the Parc de la Villette and Poster session <br>
            for those not wishing to attend the Business meeting</font><br>
            </i></b></font> <font face="Arial, Helvetica, sans-serif" size="2"><b><font size="3" color="#993399"><a name="tuesday"></a></font></b></font> 
            <hr>
            <font face="Arial, Helvetica, sans-serif" size="2"><b><font size="3" color="#993399"><br>
            </font></b></font><font face="Arial, Helvetica, sans-serif" size="2"><b><font size="3" color="#993399">Tuesday 
            September 30 &nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="2"><b><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#saturday">saturday 
            27th</a></b></font><b><font size="2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sunday">sunday 
            28th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#monday">monday 
            29th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b><a href="progched.htm"><img src="images/retour.gif" width="29" height="29" border="0"></a></b></font></font></b></font><br>
            <font size="3" color="#993399"> </font></b></font></b></font></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">9h-9h50<font color="#CC0000"> 
            <b><font color="#33CC99">Invited talk<br>
            <i>Chair: Mike Hendy</i></font></b></font><b><i><br>
            Andy Waters</i></b> &nbsp;&nbsp;Comparative genomics of malaria parasites 
            and its exploitation in a rodent malaria model </font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Protein structure<br>
          <i>Chair: Jerzy Tiuryn</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">9h50-10h15 A 
            novel approach to fold recognition using sequence-derived properties 
            from sets of structurally similar local fragments of proteins<i>&nbsp;&nbsp;Torgeir 
            R. Hvidsten, Andriy Kryshtafovych, Jan Komorowski, Krzysztof Fidelis</i> 
            </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h15-10h40 Flexible 
            structure alignment by chaining aligned fragment pairs allowing twists<i><br>
            Yuzhen Ye, Adam Godzik </i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#CC66CC">Flash 
          presentations - Protein structure<br>
          <i>Chair: Jerzy Tiuryn</i></font> </b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h40-10h45 Structural 
            similarity search in databases: YAKUSA<br>
            <i>Mathilde Carpentier, Sophie Brouillet, Joël Pothier </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h45-10h50 A 
            new 3D statistical potential to predict protein-ligand interactions 
            based on atomic interaction patterns in PDB<i><br>
            Ernesto Moreno, Luis A. Diego</i> </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h50-10h55 An 
            automatic procedure for the search and identification of new unbounded 
            docking examples<i><br>
            Frank Steinacker, Oliver Martin, Philipp Heuser, Dietmar Schomburg 
            </i> </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">10h55-11h00 Classification 
            of Fourier spectra of short protein sequences compared to their corresponding 
            structural classification<i><br>
            Manoj Tyagi, Henri Ralambondranny, Frédéric Cadet, Philippe Charton, 
            Bernard Offmann </i></font></li>
        </ul>
        <ul>
          <li><font color="#0000FF" size="3" face="Times New Roman, Times, serif"><i><b><font face="Comic Sans MS" size="2">11h-11h45 
            Posters and break </font></b></i></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Multiple alignment and phylogeny<br>
          <i>Chair: Daniel Huson</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">11h45-12h10 Divide-and-Conquer 
            multiple alignment with segment-Based constraints<i><br>
            Michael Sammeth Burkhard Morgenstern, Jens Stoye </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h10-12h35 Upper 
            bounds on maximum likelihood for phylogenetic trees<i><br>
            Michael Hendy, Barbara Holland </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h35-12h50 A 
            Randomized Linear-time Majority Tree Algorithm<i><br>
            Nina Amenta, Fredericke Clarke, Katherine St. John </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">12h50-13h05 Whole 
            genome-based prokaryotic phylogeny <br>
            <i>Stefan Henz, Alexander Auch, Daniel Huson, Kay Nieselt-Struwe, 
            Stephan Schuster</i></font></li>
        </ul>
        <ul>
          <li><i><b><font face="Comic Sans MS" size="2" color="#0000FF">13h05-14h30 
            Lunch </font></b></i></li>
        </ul>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">14h30-15h20 <b><font color="#33CC99">Invited 
            talk<br>
            <i>Chair: Eduardo Rocha<font size="3"> </font></i></font><i><font size="3"><br>
            <font size="2">François Taddei</font></font></i></b><font size="2"> 
            </font><font face="Arial, Helvetica, sans-serif" size="2">&nbsp;&nbsp;Sources 
            of genetic and phenotypic variability </font></font></li>
        </ul>
        <p><font face="Arial, Helvetica, sans-serif" size="2" color="#CC0000"><b><font color="#FF0000">Contributed 
          talks - Comparative genomics <br>
          <i>Chair: Bernard Prum</i></font></b></font></p>
        <ul>
          <li><font face="Arial, Helvetica, sans-serif" size="2">15h20-15h45 Genome 
            rearrangements in tumor cells: Reconstructing tumor genome architecture<br>
            <i>Pavel Pevzner, Ben Raphael</i> </font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">15h45-16h10 MCMC 
            genome rearrangement<br>
            <i>István Miklós </i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2">16h10-16h25 Single 
            step reconciliation algorithm for duplication, loss and horizontal 
            gene transfer model<br>
            <i>Pawel Gorecki</i></font></li>
          <li><font face="Arial, Helvetica, sans-serif" size="2"> 16h25-16h40 
            Data retrieval and handling tools for the PBIL gene family databases<i><br>
            Guy Perrière, Jean-François Dufayard, Simon Penel, Julien Grassot, 
            Laurent Duret, Manolo Gouy</i></font></li>
        </ul>
        <ul>
          <li><b><font face="Arial, Helvetica, sans-serif" size="2" color="#FF0000">16h45-17h 
            Closing remarks </font></b></li>
        </ul>
        <ul>
          <li><font face="Comic Sans MS" size="2" color="#0000FF"><b><i>17h-18h 
            Final posters session for those not needing to catch an early train 
            or flight</i></b></font></li>
        </ul>
        <hr>
        <font face="Arial, Helvetica, sans-serif" size="2"> </font> 
        <div align="left"> 
          <p align="center"><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b>&nbsp;&nbsp;<a href="#saturday">saturday 
            27th</a></b></font><b><font size="2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#sunday">sunday 
            28th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#monday">monday 
            29th</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="#tuesday">tuesday 
            30th</a> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</font><font face="Arial, Helvetica, sans-serif" size="3"><font size="2"><b><a href="progched.htm"><img src="images/retour.gif" width="29" height="29" border="0"></a></b></font></font></b></font></p>
          <p><font color="#0000FF"><i><font color="#993399"><b>Welcome to ECCB'2003</b></font><b><font color="#0000FF"><i><font color="#993399">, 
            see you soon in Paris</font></i></font><font color="#993399">!</font></b></i></font><b><a href="mailto:cch@jouy.inra.fr"></a></b><a href="mailto:cch@jouy.inra.fr"><br>
            Catherine Christophe</a></p>
        </div>
      </td>
    </tr>
    <tr>
      <td colspan="2">&nbsp;</td>
    </tr>
  </table>
  <!-- #EndEditable --> 
</div>
<layer name="ssmenu" id="ssmenu"></layer>
<nolayer>
<div id="ssmenu" style="position:absolute; z-index:2;" >
  <div id="ptimenu" style="position:absolute; z-index:10;"  >
  </div>
</div>
</nolayer>
 
<script type="text/javascript">
<!--
tab = new Array;
tab[0] = new Array;
tab[1] = new Array;
tab[2] = new Array;tab[3] = new Array;tab[4] = new Array;
tab[0][0] = '<A HREF="eccb.htm" class="SsMenu">The Conference</A>';
tab[0][1] = '<A HREF="committees.htm" class="SsMenu">Committees</A>';
tab[0][2] = '<A HREF="keydat.htm" class="SsMenu">Key Dates</A>';
tab[0][3] = '<A HREF="prevmeet.htm" class="SsMenu">Previous meetings</A>';
tab[0][4] = '<A HREF="partn.htm" class="SsMenu">Partners</A>';
tab[1][0] = '<A HREF="geninfo.htm" class="SsMenu">Program Info</A>';
tab[1][1] = '<A HREF="keydat.htm" class="SsMenu">Key Dates</A>';
tab[1][2] = '<A HREF="invspeak.htm" class="SsMenu">Invited speakers</A>';
tab[1][3] = '<A HREF="progched.htm" class="SsMenu">Program schedule</A>';
tab[1][4] = '<A HREF="particip.htm" class="SsMenu">Participants</A>';
tab[1][5] = '<A HREF="fest.htm" class="SsMenu">Festivities</A>';
tab[2][0] = '<A HREF="geninfo.htm" class="SsMenu">Program Info</A>';
tab[2][1] = '<A HREF="callpap.htm" class="SsMenu">Papers Submissions</A>';
tab[2][2] = '<A HREF="callpost.htm" class="SsMenu">Posters Submissions</A>';
tab[2][3] = '<A HREF="accepap.htm" class="SsMenu">Accepted papers</A>';
tab[2][4] = '<A HREF="proceed.htm" class="SsMenu">Proceedings</A>';
tab[3][0] = '<A HREF="registfees.htm" class="SsMenu">Registration fees</A>';
tab[3][1] = '<A HREF="registform.htm" class="SsMenu">Registration forms</A>';
tab[4][0] = '<A HREF="locat.htm" class="SsMenu">Location</A>';
tab[4][1] = '<A HREF="travel.htm" class="SsMenu">Travel info</A>';
tab[4][2] = '<A HREF="accomod.htm" class="SsMenu">Accommodation</A>';
tab[4][3] = '<A HREF="tourism.htm" class="SsMenu">Tourism info</A>';
 
 
var user_agent=navigator.userAgent.toLowerCase();
var nav_version = navigator.appVersion.toLowerCase();
var nav_minor = parseFloat(nav_version);
var nav_major = parseInt(nav_minor);
 
var iePos = nav_version.indexOf('msie');
  if (iePos !=-1) {
    nav_minor = parseFloat(nav_version.substring(iePos+5,nav_version.indexOf(';',iePos)))
    nav_major = parseInt(nav_minor);
  }
var nav6Pos = user_agent.indexOf('netscape6');
  if (nav6Pos !=-1) {
    nav_minor = parseFloat(user_agent.substring(nav6Pos+10))
    nav_major = parseInt(nav_minor)
  }
 
var nav_ns = ((user_agent.indexOf('mozilla')!=-1) && (user_agent.indexOf('spoofer')==-1)
    && (user_agent.indexOf('compatible') == -1) && (user_agent.indexOf('opera')==-1)
    && (user_agent.indexOf('webtv')==-1));
var nav_ns4up = (nav_ns && (nav_major >= 4));
var nav_ns6up = (nav_ns && nav_minor >= 6);
 
var nav_ie = (nav_version.indexOf('msie')!=-1);
var nav_ie4up = (nav_ie && nav_minor >= 4);
var nav_ie5up = (nav_ie && nav_minor >= 5);
var nav_konqueror = (user_agent.indexOf("onqueror") != -1);
var nav_mozilla = (user_agent.indexOf("ozilla") != -1);
var nav_opera = (user_agent.indexOf("opera") != -1);
var nav_win = ( (user_agent.indexOf("win")!=-1) || (user_agent.indexOf("16bit")!=-1) );
var nav_mac = (user_agent.indexOf("mac")!=-1);
var nav_linux = (user_agent.indexOf("inux")!=-1);
if (document.layers) { 
     
      document.barremenu.top= 140;
      document.barremenu.left = (window.innerWidth/5);
              }  
 
   else if (nav_ns6up||nav_ie5up||nav_mozilla||nav_konqueror||nav_opera) {
      
      document.getElementById("barremenu").style.top=140;
      topbarre=document.getElementById("barremenu").style.top;
      
   if ((nav_ie5up)&&!(nav_opera)){
       
         document.getElementById("barremenu").style.left= (document.body.clientWidth/5);
         }
     else {
         
          document.getElementById("barremenu").style.left=(window.innerWidth/5);
        
     }
   leftbarre=document.getElementById("barremenu").style.left;
}
function affiche_Menu(tableau,posX){
 
if (document.layers) {
    document.ssmenu.visibility ="hide";    
             }
  
   else if (nav_ns6up||nav_ie5up||nav_mozilla||nav_konqueror||nav_opera) { 
   
     document.getElementById("ssmenu").style.visibility = "hidden";
     }
     
   
i=0;
var content = "<table bgcolor=#66FF66 border=0 CELLPADDING=1 CELLSPACING=1 align=center><tr><td>";
while (i < tableau.length) {
  content += "<table CELLPADDING=1 CELLSPACING=1 border=0><tr><td width=100 align=center>"+tableau[i]+"</td></tr></table>";
   i++;
   }
content += "</td></tr></table>";
 
if (document.layers) {
               
                document.ssmenu.document.write(content);
                document.ssmenu.document.close();
                document.ssmenu.top= document.barremenu.top+35;
                document.ssmenu.left= posX-document.barremenu.left-22;
                document.ssmenu.visibility="show";   
                }
 
  else if (nav_ns6up||nav_ie5up||nav_mozilla||nav_konqueror||nav_opera) { 
     
     document.getElementById("ptimenu").innerHTML=content; 
     document.getElementById("ptimenu").style.width=100;
     document.getElementById("ptimenu").style.height=i*25;
     document.getElementById("ptimenu").style.left=parseInt(leftbarre)+posX;
     if ((nav_ie5up)&&!(nav_opera)) {
        document.getElementById("ptimenu").style.top=parseInt(topbarre)+20;
     }
     else document.getElementById("ptimenu").style.top=parseInt(topbarre)+30;
     document.getElementById("ptimenu").style.visibility = "visible";
     }
 
flag=true;
if (document.layers) {
         var timer=null;
         timer=setTimeout("cache_Menu()", 7000); 
         }
else if (nav_ie5up) {
        ptimenu.onmouseover=stop_Timer;
        ptimenu.onmouseout=PtiMenu_Out2;
       
        if (flag) {
           barremenu.onmouseover=stop_Timer;
           barremenu.onmouseout=PtiMenu_Out2;
        }
}
else  if (nav_ns6up||nav_mozilla||nav_konqueror||nav_opera) {
       
        document.getElementById("ptimenu").onmouseover=stop_Timer;
        document.getElementById("ptimenu").onmouseout=PtiMenu_Out2;
         if (flag) {
                 document.getElementById("barremenu").onmouseover=stop_Timer;
                 document.getElementById("barremenu").onmouseout=PtiMenu_Out2;
         }
       }
        
 
}
var timer=null;   
function PtiMenu_Out2(){
timer=setTimeout("cache_Menu() ", 2000);
}
function cache_Menu () {
if(document.layers) {
     document.ssmenu.visibility="hide";
     clearTimeout(timer);
     }   
else if (nav_ns6up||nav_ie5up||nav_mozilla||nav_konqueror||nav_opera) { 
    document.getElementById("ptimenu").style.visibility = "hidden";
 }
 
}
function stop_Timer(){
clearTimeout(timer);
}
var NS4sIW = window.innerWidth;
var NS4sIH = window.innerHeight;
if (window.onresize) {
    
    NS4resize();
 }
function NS4resize() {
    if (NS4sIW < window.innerWidth || 
       NS4sIW > window.innerWidth || 
       NS4sIH > window.innerHeight || 
       NS4sIH < window.innerHeight ) {
           window.location.reload();
     }
}
//-->
 
</SCRIPT>
 
</body>
<!-- #EndTemplate --></html>
