<HTML>
<HEAD><TITLE>Sequence Databases and Ontologies Workshop</TITLE></HEAD>
<BODY BGCOLOR="#FFFFFF">
<CENTER>
<H1>Sequence Databases and Ontologies Workshop</H1>
<FONT SIZE=+2>
<B>
Organized by
<A HREF="javascript:if(confirm('http://www.impg.prd.fr/  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.impg.prd.fr/'">IMPG</A>
and
<A HREF="javascript:if(confirm('http://pbil.univ-lyon1.fr  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://pbil.univ-lyon1.fr'">PBIL</A>
</FONT>
</B>
<H3>September 26, 2003</H3>
<FONT SIZE=+1>
Institut Pasteur
<BR>
Amphith&eacute;&acirc;tre Jacques Monod
<BR>
Paris, France
</FONT>
</CENTER>
<P>
<HR WIDTH=50% ALIGN=center>
<P ALIGN=justify>
As the number of biological sequences and their associated data is
exponentially growing, the need to structure and organize huge amounts
of information is stronger than ever. Even if hundreds of databases
are already available, many problems remain open: how to represent,
structure and integrate heterogeneous data such as sequences,
alignments, gene families, 3D structures or expression tags? What kind
of curation systems is it possible to develop to ensure the
maintenance and the reliability of the information stored? How to
provide efficient ways to query and retrieve relevant data subsets?
</P>
<P ALIGN=justify>
In the framework of the European Conference on Computational Biology
(2003) we organize a mini-satellite  event on sequence databases and
ontologies. From the submissions we received, ten have been selected
for 20 minutes talks, the others being selected as posters that will
be presented in the ECCB poster sessions. All the abstracts we
received will appear in the workshop proceedings which will be printed
locally and distributed to the registered participants.
</P>
<HR WIDTH=50% ALIGN=center>
<CENTER>
<H2>REGISTRATION</H2>
</CENTER>
<P ALIGN=justify>
If you wish to attend to this workshop, you only need to send an e-mail
to <a HREF="mailto:perriere@biomserv.univ-lyon1.fr">perriere@biomserv.univ-lyon1.fr</a>.
No registration fees are required but the number of attendants will
be limited to 100 due to space restrictions.
</P>
<P ALIGN=justify>
<B>Important:</B> registration is now <FONT COLOR=red><B>closed</B></FONT>,
as we reached the maximum number of seats available in the Jacques Monod
lecture hall.
</P>
<HR WIDTH=50% ALIGN=center>
<CENTER>
<H2>ACCESS</H2>
</CENTER>
<P ALIGN=justify>
The workshop will be held in Jacques Monod lecture hall at
Institut Pasteur, 25 rue du Docteur Roux, 75724 Paris. This
lecture hall is located in the Jacques Monod building (see
the <A HREF="javascript:if(confirm('http://www.pasteur.fr/pratique/plan/plan25.gif  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.pasteur.fr/pratique/plan/plan25.gif'">map</A>
of the campus). Information on how to get to the Institut Pasteur
is available
<A HREF="javascript:if(confirm('http://www.pasteur.fr/pratique/plan/index-en.html  \n\nThis file was not retrieved by BackStreet browser, because it is addressed on a domain or path outside the boundaries set for its Project.  \n\nDo you want to open it from the server?'))window.location='http://www.pasteur.fr/pratique/plan/index-en.html'">here</A>.
</P>
<HR WIDTH=50% ALIGN=center>
<CENTER>
<H2>PROGRAM</H2>
</CENTER>
<P ALIGN=justify>
Here is the final program for this event. Note that some last minute changes
can still happen, so check regularly te schedule on this web page.
</P>
<TABLE CELLPADDING=10>
<TR>
<TD VALIGN=top>9h00-9h30</TD>
<TD>Registration</TD>
</TR>
<TR>
<TD VALIGN=top>9h30-9h40</TD>
<TD><B>Welcoming speech</B></TD>
</TR>
<TR>
<TD VALIGN=top>9h40-10h30</TD>
<TD><B><FONT COLOR=blue>Invited talk:</FONT></B> Protein domain databases and the use of<BR>
large scale analysis of protein modularity - <I>D. Kahn</I></TD>
</TR>
<TR>
<TD VALIGN=top>10h30-10h50</TD>
<TD>Standardization of format and content in the SWISS-PROT<BR>
protein knowledgebase - <I>A. Gattiker, E. Gasteiger,<BR>
A. Bairoch</I></TD>
</TR>
<TR>
<TD VALIGN=top>10h50-11h10</TD>
<TD><B>Coffee break</B></TD>
</TR>
<TR>
<TD VALIGN=top>11h10-11h30</TD>
<TD>Databases of homologous gene families: New developments<BR>
and Web interfaces - <I>S. Penel, J. Grassot, L. Duret, M.<BR>
Gouy, G. Perri&egrave;re</I></TD>
</TR>
<TR>
<TD VALIGN=top>11h30-11h50</TD>
<TD>HOPPSIGEN: A database dedicated to comparative analysis<BR>
of homologous processed pseudogenes - <I>A. Khelifi, L. Duret,<BR>
D. Mouchiroud</I></TD>
</TR>
<TR>
<TD VALIGN=top>11h50-12h10</TD>
<TD>Databases on eukaryotic gene regulation: TRANSFAC&#174;,<BR>
TRANSCompel&#174;, and TRANSPATH&#174;. Annotation, information<BR>
processing, and application for analysis of gene expression<BR>
data - <I>O. Kel-Margoulis, V. Matys, C. Choi, I. Reuter, E.<BR>
Goessling, N. Voss, M. Krull, S. Pistor., S. Thiele, E. Fricke,<BR>
S. Land, A. Potapov, D. Tchekmenev, B. Lewicki-Potapov,<BR>
A. Kel, E. Wingender</I></TD>
</TR>
<TR>
<TD VALIGN=top>12h10-12h30</TD>
<TD>A Human-machine platform for 3D visualization and<BR>
management of genomic databases - <I>P.E. Gros, J. H&eacute;risson,<BR>
R. Gherbi</I></TD>
</TR>
<TR>
<TD VALIGN=top>12h30-14h00</TD>
<TD><B>Lunch break</B></TD>
</TR>
<TD VALIGN=top>14h00-14h50</TD>
<TD><B><FONT COLOR=blue>Invited talk:</FONT></B> Ensembl: An
information infrastructure for<BR>
genomes - <I>E. Birney</I></TD>
</TR>
<TR>
<TD VALIGN=top>14h50-15h10</TD>
<TD>Mouse phenotype ontology development and application -<BR>
<I>E.C.J. Green, G.V. Gkoutos, A.M. Mallon, S. Greenaway,<BR>
A. Blake, M. Simon, J.M. Hancock, D. Davidson</I></TD>
</TR>
<TR>
<TD VALIGN=top>15h10-15h30</TD>
<TD>GeMCore: A mammalian comparative mapping database -<BR>
<I>V. Navratil, G. Bronner, B. Spataro, T. Faraut, D.<BR>
Mouchiroud, C. Gautier</I></TD>
</TR>
<TD>15h30-15h50</TD>
<TD><B>Coffee break</B></TD>
</TR>
<TR>
<TD VALIGN=top>15h50-16h10</TD>
<TD>ISYMOD: A knowledge base for integrated system modeling -<BR>
<I>J. Chabalier, Y. Quentin, C. Capponi, G. Fichant</I></TD>
</TR>
<TR>
<TD VALIGN=top>16h10-16h30</TD>
<TD>IMGT-ONTOLOGY and IMGT-ML for immunogenetics and<BR>
immunoinformatics - <I>D. Chaume, V. Giudicelli, K. Combres,<BR>
 M.P. Lefranc</I></TD>
</TR>
<TR>
<TD VALIGN=top>16h30-16h50</TD>
<TD>MADCAP: MicroArray data characterization and profiling.<BR>
A tool for profiling lists of genes with different expression<BR>
patterns - <I>J. Lozach, R. Sasik, S. Ogawa, C.K. Glass</I></TD><BR>
</TR>
<TR>
<TD VALIGN=top>16h50-17h00</TD>
<TD><B>Closing remarks</B></TD>
</TR>
</TABLE>
<P>
<HR WIDTH=50% ALIGN=center>
</BODY>
</HTML>
